{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Library design for CTP02, \n",
    "\n",
    "## chromosome 21 tracing, a small set to test neighboring points\n",
    "by Pu Zheng\n",
    "\n",
    "This code is to generate the library according to Xiaowei's idea"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#minimum imports:\n",
    "import time,os,sys,glob\n",
    "import cPickle as pickle\n",
    "import numpy as np\n",
    "import khmer\n",
    "sys.path.append(r'/n/home13/pzheng/Documents/python-functions/python-functions-library')\n",
    "\n",
    "from LibraryConstruction import fastaread,fastawrite,fastacombine\n",
    "import LibraryDesigner as ld\n",
    "import LibraryConstruction as lc"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": true
   },
   "source": [
    "## 0. (skipped) no need to build index again"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "# human genome\n",
    "master_dir = r'/n/boslfs/LABS/zhuang_lab/User/pzheng'\n",
    "genome_folder = master_dir + os.sep + r'Genomes/human/hg38/chroms';"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 1. (skipped) genome parsing is identical to SI-14/CTP-01 chr21 small"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "master_dir = r'/n/boslfs/LABS/zhuang_lab/User/pzheng';\n",
    "region_dir = r'Libraries/SI-14/chr21';\n",
    "region_folder = master_dir + os.sep + region_dir"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 2. (skipped) prode design is identical to SI-14/CTP-01 chr21 small"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "# You can continue here!\n",
    "region_folder=r'/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21';\n",
    "\n",
    "# dic for region -> tad\n",
    "if not 'new_id_dic' in vars():\n",
    "    new_id_dic = pickle.load(open(region_folder+os.sep+'region_to_TAD.pkl','r'))\n",
    "reg_id_dic = new_id_dic\n",
    "\n",
    "# dic for region -> it's length\n",
    "if not 'reg_len_dic' in vars():\n",
    "    reg_len_dic = pickle.load(open(region_folder+os.sep+'region_length.pkl','r'))\n",
    "reg_size_dic = reg_len_dic"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 3. Design Encoding"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def Design_Neighboring_Scheme(reg_id_dic, reg_size_dic, \n",
    "                              barcode_source, \n",
    "                              barcode_order,\n",
    "                              size_threshold=200, \n",
    "                              n_color=3, n_split=2, \n",
    "                              save=True, save_dir=r'/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/CTP-02/chr21', \n",
    "                              verbose=True):\n",
    "    '''Design neighboring encoding scheme\n",
    "    Inputs:\n",
    "        reg_id_dic: region -> TAD dictionary, dic\n",
    "        reg_size_dic: region -> number of probe dictionary, dic\n",
    "        barcode_source: dictionary to determine the source of barcodes, dictionary\n",
    "        barcode_order: list to determine numbering order of barcodes, list\n",
    "        size_threshold: lower bound for number of probes in each region, int\n",
    "        n_color: number of colors, int\n",
    "        n_split: split the whole region set equally into two, \n",
    "        save: whether save final result, bool\n",
    "        save_dir: save directory, string\n",
    "        verbose: whether say something!, bool\n",
    "    Output:\n",
    "        reg_encodings: encoding scheme, reg->'''\n",
    "    \n",
    "    import numpy as np\n",
    "    def _assign_unique_barcode(reg_encodings, _verbose=verbose):\n",
    "        '''Assign unique barcodes according to sequential order'''\n",
    "        i = 0;\n",
    "        if _verbose:\n",
    "            print \"-- number of total regions:\", len(reg_encodings)\n",
    "        for key, value in sorted(reg_encodings.items()):\n",
    "            reg_encodings[key]['id'] = i;\n",
    "            reg_encodings[key]['bc_unique'] = i;\n",
    "            i += 1;\n",
    "        return reg_encodings\n",
    "    \n",
    "    def _assign_split_barcode_and_color(reg_encodings, _n_split=n_split, _n_color=n_color, _verbose=verbose):\n",
    "        '''Assign split(common) barcodes according to sequential order\n",
    "            Assign color at the same time'''\n",
    "        # splitting into sequential subsets\n",
    "        _total_regions = len(reg_encodings);\n",
    "        _sub_regions = int(_total_regions / _n_split)\n",
    "        if _total_regions > _sub_regions * _n_split: # if regions cannot be splitted evenly\n",
    "            _sub_regions += 1;\n",
    "        if _verbose:\n",
    "            print \"-- number of region in each subset:\", _sub_regions\n",
    "        # assign split id\n",
    "        _keys = list(reg_encodings.keys())\n",
    "        for i in range(_sub_regions):\n",
    "            _current_keys = _keys[i::_sub_regions]\n",
    "            for _k in _current_keys:\n",
    "                reg_encodings[_k]['split_id'] = i;\n",
    "                reg_encodings[_k]['bc_split'] = i; # save split barcode\n",
    "                reg_encodings[_k]['color'] = i % _n_color # save color info\n",
    "        return reg_encodings\n",
    "    \n",
    "    def _assign_tad_barcode(reg_encodings, _reg_id_dic=reg_id_dic, _verbose=verbose):\n",
    "        '''Assign TAD barcode'''\n",
    "        if _verbose:\n",
    "            print \"-- number of total TADs:\", max(_reg_id_dic.values())+1\n",
    "        for key, value in sorted(reg_encodings.items()):\n",
    "            reg_encodings[key]['bc_tad'] = _reg_id_dic[key];\n",
    "            \n",
    "        return reg_encodings\n",
    "\n",
    "    def _numbering_barcodes(_encodings, _barcode_source=barcode_source, _barcode_order=barcode_order,\n",
    "                            _verbose=verbose):\n",
    "        '''Redo numbering of different types of barcodes, make sure they are not overlapping'''\n",
    "        if _verbose:\n",
    "            print \"-- Re-numbering barcodes according to their source and order\";\n",
    "        # step 1 inverting barcode_source map\n",
    "        _inv_bs = {};\n",
    "        for _bc in _barcode_order:\n",
    "            if _barcode_source[_bc] not in _inv_bs.keys():\n",
    "                _inv_bs[_barcode_source[_bc]] = [_bc];\n",
    "            else:\n",
    "                _inv_bs[_barcode_source[_bc]].append(_bc);\n",
    "        print \"--- inversed source dictionary\\n\",_inv_bs;\n",
    "        \n",
    "        # step 2 for each group of barcodes, sort\n",
    "        for _type, _bcs in _inv_bs.items():\n",
    "            _start = 0; # counter, make sure barcodes in the same type won't overlap\n",
    "            # for each type of barcode from the same source\n",
    "            for _bc in _bcs:\n",
    "                _bc_list = [v[_bc] for k,v in sorted(_encodings.items())];\n",
    "                for k,v in sorted(_encodings.items()):\n",
    "                    _encodings[k][_bc] = _encodings[k][_bc] - min(_bc_list) + _start;\n",
    "                # update starting point, +1 to avoid overlap\n",
    "                _start += max(_bc_list) + 1;\n",
    "        \n",
    "        return _encodings;\n",
    "    \n",
    "    \n",
    "   # Initialize\n",
    "    if verbose:\n",
    "        print \"- Initializing\";\n",
    "        print \"-- number of splitted subsets:\", n_split\n",
    "        print \"-- number of colors:\", n_color\n",
    "    reg_encodings = {};\n",
    "    for key, value in sorted(reg_id_dic.items()):\n",
    "        if value >= 0 and reg_size_dic[key] >= size_threshold: \n",
    "            reg_encodings[key] = {'TAD':reg_id_dic[key], \n",
    "                                  'size':reg_size_dic[key],\n",
    "                                  'color':None,\n",
    "                                  'split_id':None, \n",
    "                                  'id': None, \n",
    "                                  'bc_split':None,\n",
    "                                  'bc_unique':None,\n",
    "                                  'bc_tad':None}\n",
    "    # Assign unique ids\n",
    "    if verbose:\n",
    "        print \"- Assigning unique ids\"\n",
    "    reg_encodings = _assign_unique_barcode(reg_encodings);\n",
    "    \n",
    "    # Assign split ids\n",
    "    if verbose:\n",
    "        print \"- Assigning split ids and color\"\n",
    "    reg_encodings = _assign_split_barcode_and_color(reg_encodings);\n",
    "    \n",
    "    # Assign tad barcodes\n",
    "    if verbose:\n",
    "        print \"- Assigning tad barcodes\"\n",
    "    reg_encodings = _assign_tad_barcode(reg_encodings);\n",
    "    \n",
    "    # Re-number barcodes\n",
    "    if verbose:\n",
    "        print \"- Renumbering barcodes\";\n",
    "    reg_encodings = _numbering_barcodes(reg_encodings);\n",
    "    \n",
    "    if save:\n",
    "        if verbose:\n",
    "            print \"- Saving encodings and barcode infos.\";\n",
    "        import cPickle as pickle\n",
    "        import os\n",
    "        \n",
    "        # make dir if not exist\n",
    "        if not os.path.exists(save_dir):\n",
    "            os.makedirs(save_dir)\n",
    "        \n",
    "        # save encodings\n",
    "        save_filename = save_dir + os.sep + 'total_encoding.pkl';\n",
    "        with open(save_filename, 'w') as output_handle:\n",
    "            if verbose:\n",
    "                print \"-- Save encoding scheme to file:\", save_filename\n",
    "            pickle.dump(reg_encodings, output_handle)\n",
    "        \n",
    "        # save barcode infos\n",
    "        save_bc_source = save_dir + os.sep + 'barcode_source.pkl';\n",
    "        save_bc_order = save_dir + os.sep + 'barcode_order.pkl';\n",
    "        with open(save_bc_source, 'w') as output_handle:\n",
    "            if verbose:\n",
    "                print \"-- Save barcode_source to file:\", save_bc_source\n",
    "            pickle.dump(barcode_source, output_handle)\n",
    "        with open(save_bc_order, 'w') as output_handle:\n",
    "            if verbose:\n",
    "                print \"-- Save barcode_order to file:\", save_bc_order\n",
    "            pickle.dump(barcode_order, output_handle)\n",
    "            \n",
    "    return reg_encodings"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Initializing\n",
      "-- number of splitted subsets: 2\n",
      "-- number of colors: 3\n",
      "- Assigning unique ids\n",
      "-- number of total regions: 651\n",
      "- Assigning split ids and color\n",
      "-- number of region in each subset: 326\n",
      "- Assigning tad barcodes\n",
      "-- number of total TADs: 34\n",
      "- Renumbering barcodes\n",
      "-- Re-numbering barcodes according to their source and order\n",
      "--- inversed source dictionary\n",
      "{'stv': ['bc_split'], 'ndb': ['bc_tad', 'bc_unique']}\n",
      "- Saving encodings and barcode infos.\n",
      "-- Save encoding scheme to file: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/CTP-02/chr21/total_encoding.pkl\n",
      "-- Save barcode_source to file: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/CTP-02/chr21/barcode_source.pkl\n",
      "-- Save barcode_order to file: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/CTP-02/chr21/barcode_order.pkl\n"
     ]
    }
   ],
   "source": [
    "# Important inputs for patching barcodes\n",
    "barcode_source = {'bc_unique':'ndb',\n",
    "                  'bc_tad': 'ndb',\n",
    "                  'bc_split':'stv'};\n",
    "barcode_order = ['bc_split', 'bc_tad','bc_unique'];\n",
    "\n",
    "reg_encodings = Design_Neighboring_Scheme(reg_id_dic, reg_size_dic, barcode_source, barcode_order)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "34"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "min([v['bc_unique'] for k,v in reg_encodings.items()])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def Sub_Encoding_Scheme(reg_encodings, \n",
    "                        starting_tad=0,\n",
    "                        reg_per_split=100,\n",
    "                        n_color=3, n_split=2,\n",
    "                        report_dir = r'/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged',\n",
    "                        save=True, save_dir=r'/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/CTP-02/chr21',\n",
    "                        verbose=True):\n",
    "    '''Based on the total region encoding scheme, pick a small set for testing\n",
    "    Inputs:\n",
    "        reg_encodings: total encoding scheme designed by previous function, dic\n",
    "        starting_tad: starting picking sub scheme by this tad, int\n",
    "        reg_per_split: number of regions kept in each spiltted subset, int\n",
    "        n_color: number of colors, int\n",
    "        n_split: split the whole region set equally into subsets, int \n",
    "        report_dir: directory for probe reports, string\n",
    "        save: whether save final result, bool\n",
    "        save_dir: save directory, string\n",
    "        verbose: whether say something!, bool\n",
    "        '''\n",
    "    import numpy as np;\n",
    "    import glob;\n",
    "    def _extract_subset(_reg_encodings=reg_encodings, _starting_tad=starting_tad, \n",
    "                        _reg_per_split=reg_per_split, \n",
    "                        _n_color=n_color, _n_split=n_split,\n",
    "                        _verbose=verbose):\n",
    "        '''Extract defined subset of all regions in encoding scheme, return a sub encoding scheme'''\n",
    "                # splitting into sequential subsets\n",
    "        _total_num = len(_reg_encodings);\n",
    "        _sub_num = int(_total_num / _n_split)\n",
    "        if _total_num > _sub_num * _n_split: # if regions cannot be splitted evenly\n",
    "            _sub_num += 1;\n",
    "        if _verbose:\n",
    "            print \"-- number of region in each subset:\", _sub_num\n",
    "        # save \"bc_split\" as a reference\n",
    "        _split_ids = [];\n",
    "        # count number of regions in each split\n",
    "        _ct = 0; \n",
    "        for _i, (_key, _value) in enumerate(sorted(_reg_encodings.items(), key=lambda (k,v):v['id'])):\n",
    "            if _value['TAD'] >= _starting_tad and _value['split_id'] not in _split_ids:\n",
    "                _split_ids.append(_value['split_id']);\n",
    "            if len(_split_ids) >= min(_sub_num, _reg_per_split):\n",
    "                break\n",
    "        print \"-- Selected split_ids:\\n\", _split_ids;\n",
    "        # extract sub_encodings\n",
    "        _sub_encodings = {}\n",
    "        for _i, (_key, _value) in enumerate(sorted(_reg_encodings.items(), key=lambda (k,v):v['id'])):\n",
    "            if _value['split_id'] in _split_ids:\n",
    "                _sub_encodings[_key] = _value;\n",
    "        print \"-- Selected region ids:\\n\", [k for k,v in sorted(_sub_encodings.items())]       \n",
    "        return _sub_encodings;\n",
    "    \n",
    "\n",
    "    def _reassign_barcodes(_reg_encodings, _sub_encodings, _n_color=n_color, _n_split=n_split, _verbose=verbose):\n",
    "        '''Re-assign unique and spllit barcodes'''\n",
    "        # extract existing _unique and _split barcodes\n",
    "        _uniques = [v['bc_unique'] for k,v in sorted(_sub_encodings.items())];\n",
    "        _splits = [v['bc_split'] for k,v in sorted(_sub_encodings.items())];\n",
    "        # extract total _unique and _split barcodes\n",
    "        _total_uniques = [v['bc_unique'] for k,v in sorted(_reg_encodings.items())];\n",
    "        _total_splits = [v['bc_split'] for k,v in sorted(_reg_encodings.items())];\n",
    "        ct_unique = min(_total_uniques);\n",
    "        print ct_unique\n",
    "        for _key, _value in sorted(_sub_encodings.items(), key= lambda (k,v):v['id']):\n",
    "            _sub_encodings[_key]['bc_split'] -= min(_splits) - min(_total_splits);\n",
    "            _sub_encodings[_key]['bc_unique'] = ct_unique;\n",
    "            ct_unique += 1;\n",
    "            \n",
    "        print \"-- Updated split barcodes:\\n\", [v['bc_split'] for k,v in sorted(_sub_encodings.items())] ;\n",
    "        print \"-- Updated unique barcodes:\\n\", [v['bc_unique'] for k,v in sorted(_sub_encodings.items())] ;\n",
    "        print \"-- TAD barcodes:\\n\", [v['bc_tad'] for k,v in sorted(_sub_encodings.items())] ;\n",
    "        return _sub_encodings;\n",
    "        \n",
    "    sub_encodings = _extract_subset(reg_encodings.copy());\n",
    "    sub_encodings = _reassign_barcodes(reg_encodings.copy(), sub_encodings);\n",
    "    \n",
    "    if save:\n",
    "        if verbose:\n",
    "            print \"- Saving encodings and barcode infos.\";\n",
    "        import cPickle as pickle\n",
    "        import os\n",
    "        \n",
    "        # make dir if not exist\n",
    "        if not os.path.exists(save_dir):\n",
    "            os.makedirs(save_dir)\n",
    "        \n",
    "        # save encodings\n",
    "        save_filename = save_dir + os.sep + 'sub_encoding.pkl';\n",
    "        with open(save_filename, 'w') as output_handle:\n",
    "            if verbose:\n",
    "                print \"-- Save sub-encoding scheme to file:\", save_filename\n",
    "            pickle.dump(sub_encodings, output_handle)\n",
    "    \n",
    "    return sub_encodings\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- number of region in each subset: 326\n",
      "-- Selected split_ids:\n",
      "[41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99, 100]\n",
      "-- Selected region ids:\n",
      "[318, 319, 320, 321, 322, 323, 324, 325, 326, 327, 328, 329, 330, 331, 332, 333, 334, 335, 336, 337, 338, 339, 340, 341, 342, 343, 348, 349, 350, 351, 352, 353, 354, 355, 356, 357, 358, 359, 360, 361, 362, 363, 364, 365, 366, 367, 368, 369, 370, 371, 372, 373, 374, 375, 376, 377, 378, 379, 380, 381, 650, 651, 652, 653, 654, 655, 656, 657, 658, 659, 660, 661, 662, 663, 664, 665, 666, 667, 668, 669, 670, 671, 672, 673, 674, 675, 676, 677, 678, 679, 680, 681, 682, 683, 684, 685, 686, 687, 688, 689, 690, 691, 692, 693, 694, 695, 696, 697, 698, 699, 700, 701, 702, 703, 704, 705, 706, 707, 708, 709]\n",
      "34\n",
      "-- Updated split barcodes:\n",
      "[0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59]\n",
      "-- Updated unique barcodes:\n",
      "[34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99, 100, 101, 102, 103, 104, 105, 106, 107, 108, 109, 110, 111, 112, 113, 114, 115, 116, 117, 118, 119, 120, 121, 122, 123, 124, 125, 126, 127, 128, 129, 130, 131, 132, 133, 134, 135, 136, 137, 138, 139, 140, 141, 142, 143, 144, 145, 146, 147, 148, 149, 150, 151, 152, 153]\n",
      "-- TAD barcodes:\n",
      "[2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 13, 13, 13, 13, 13, 13, 14, 14, 14, 14, 14, 14, 14, 14, 14, 14, 14, 14, 14, 14, 14, 15, 15, 15, 15, 15, 15, 15, 15, 16, 16, 16, 16, 16, 16, 16, 16, 16, 16, 17, 17, 17, 17, 17, 18, 18, 18, 18, 18, 18, 18, 18, 18, 18, 18, 18, 18, 18, 18, 18]\n",
      "- Saving encodings and barcode infos.\n",
      "-- Save sub-encoding scheme to file: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/CTP-02/chr21/sub_encoding.pkl\n"
     ]
    }
   ],
   "source": [
    "sub_encodings = Sub_Encoding_Scheme(reg_encodings,\n",
    "                                    starting_tad=2, reg_per_split=60)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 4. Patch Barcodes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "# minimal imports for biopython\n",
    "from Bio import SeqIO\n",
    "from Bio.Seq import Seq\n",
    "from Bio.Alphabet import IUPAC\n",
    "from Bio.SeqRecord import SeqRecord\n",
    "import os,glob,time\n",
    "import numpy as np"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 4.1 Read barcode Sequences"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Barcodes loaded: Stv: 75, NDB: 1052\n"
     ]
    }
   ],
   "source": [
    "# read all Stv barcodes\n",
    "barcode_dir = r'/n/boslfs/LABS/zhuang_lab/User/pzheng/Barcodes';\n",
    "\n",
    "#stv_adaptor = [1,2,17,62,77,78,79,80,81,82,83,84] # barcodes saved for adaptors\n",
    "#stv_bad = [34,38,41] # barcodes performed badly\n",
    "#stv_mask = stv_adaptor + stv_bad \n",
    "stv_mask = []\n",
    "\n",
    "with open(barcode_dir+os.sep+'top_Stvs.fasta', \"rU\") as handle:\n",
    "    stv_barcodes = [];\n",
    "    for record in SeqIO.parse(handle, \"fasta\"):\n",
    "        if int(record.id.split('_')[1]) not in stv_mask:\n",
    "            stv_barcodes.append(record);\n",
    "\n",
    "# read all NDB barcodes\n",
    "ndb_mask = [];\n",
    "\n",
    "with open(barcode_dir+os.sep+'NDBs.fasta', \"rU\") as handle:\n",
    "    ndb_barcodes = [];\n",
    "    for record in SeqIO.parse(handle, \"fasta\"):\n",
    "        if int(record.id.split('_')[1]) not in ndb_mask:\n",
    "            ndb_barcodes.append(record);\n",
    "print \"Barcodes loaded: Stv: \"+str(len(stv_barcodes))+\", NDB: \"+str(len(ndb_barcodes));"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 4.2 Read all PCR primers"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Primers loaded: forward: 12, reverse: 9\n"
     ]
    }
   ],
   "source": [
    "primer_dir = r'/n/boslfs/LABS/zhuang_lab/User/pzheng/Primers';\n",
    "fwd_primer_filename = 'forward_primers_keep.fasta';\n",
    "rev_primer_filename = 'reverse_primers_keep.fasta';\n",
    "\n",
    "# read all forward primers\n",
    "with open(primer_dir+os.sep+fwd_primer_filename, \"rU\") as handle:\n",
    "    fwd_primers = [];\n",
    "    for record in SeqIO.parse(handle, \"fasta\"):\n",
    "        fwd_primers.append(record);\n",
    "# read all forward primers\n",
    "with open(primer_dir+os.sep+rev_primer_filename, \"rU\") as handle:\n",
    "    rev_primers = [];\n",
    "    for record in SeqIO.parse(handle, \"fasta\"):\n",
    "        rev_primers.append(record);\n",
    "print \"Primers loaded: forward: \"+str(len(fwd_primers))+\", reverse: \"+str(len(rev_primers));        "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 4.3 read all probe reports and generate primary probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- forward primer: ID: W1A01_primer_0\n",
      "Name: W1A01_primer_0\n",
      "Description: W1A01_primer_0\n",
      "Number of features: 0\n",
      "Seq('CGGCTCGCAGCGTGTAAACG', SingleLetterAlphabet())\n",
      "- reverse primer: ID: W1A02_primer_1\n",
      "Name: W1A02_primer_1\n",
      "Description: W1A02_primer_1\n",
      "Number of features: 0\n",
      "Seq('TAATACGACTCACTATAGGGCATTTCAGGATCACCGGCGG', SingleLetterAlphabet())\n"
     ]
    }
   ],
   "source": [
    "# master directory\n",
    "master_dir =r'/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/CTP-02/chr21';\n",
    "\n",
    "# primers\n",
    "fprimer = fwd_primers[0];\n",
    "print '- forward primer:', fprimer\n",
    "rprimer = rev_primers[0];\n",
    "print '- reverse primer:', rprimer\n",
    "\n",
    "# Important inputs for patching barcodes\n",
    "if not 'barcode_source' in vars():\n",
    "    import cPickle as pickle\n",
    "    print 'loading barcode_source'\n",
    "    barcode_source = pickle.load(open(master_dir+os.sep+'barcode_source.pkl','r'))\n",
    "if not 'barcode_order' in vars():\n",
    "    import cPickle as pickle\n",
    "    print 'loading barcode_order'\n",
    "    barcode_order = pickle.load(open(master_dir+os.sep+'barcode_order.pkl','r'))\n",
    "\n",
    "# reload dic for region -> tad\n",
    "if not 'sub_encodings' in vars():\n",
    "    import cPickle as pickle\n",
    "    print 'loading sub_encodings'\n",
    "    sub_encodings = pickle.load(open(master_dir+os.sep+'sub_encoding.pkl','r'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def Patch_Barcodes(reg_encodings, \n",
    "                   fwd_primer,rev_primer,\n",
    "                   barcode_source, \n",
    "                   barcode_order,\n",
    "                   stv_barcodes, ndb_barcodes, barcode_starts={'stv':1,'ndb':1},\n",
    "                   report_folder=r'/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged',\n",
    "                   save_folder=r'/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/CTP-02/chr21/final_probes',\n",
    "                   add_rand_gap=0,\n",
    "                   save=True, verbose=True):\n",
    "    '''Function to patch barcodes to designed probes\n",
    "    Inputs:\n",
    "        reg_encodings: encoding scheme for the barcode, dictionary(generated previously)\n",
    "        fwd_primer: forward primer,20mer, biopython SeqRecord\n",
    "        rev_primer: reverse primer,40mer(rc), last 20mer-rc should be used\n",
    "        barcode_source: dictionary to determine the source of barcodes, dictionary\n",
    "        barcode_order: list to determine numbering order of barcodes, list\n",
    "        stv_barcodes: old barcodes,30mer, biopython SeqRecord list\n",
    "        ndb_barcodes: new barcodes,30mer, biopython SeqRecord list\n",
    "        barcode_starts: id of the first unused barcode, dictionary\n",
    "        report_folder: directory for probe reports, string\n",
    "        save_folder: directory for save files, string\n",
    "        add_rand_gap: whether adding (or length) of random gaps between barcodes, int\n",
    "        save: whether save, bool\n",
    "        verbose: whether say something, bool\n",
    "    Outputs:\n",
    "        total library SeqRecord\n",
    "        '''\n",
    "    # minimal imports\n",
    "    from Bio import SeqIO\n",
    "    from Bio.Seq import Seq\n",
    "    from Bio.Alphabet import IUPAC\n",
    "    from Bio.SeqRecord import SeqRecord \n",
    "    import numpy as np;\n",
    "    import glob, os, sys, time\n",
    "    import LibraryDesigner as ld\n",
    "    \n",
    "    # check inputs:\n",
    "    if verbose:\n",
    "        print \"- Check inputs\"\n",
    "    # check barcode_source\n",
    "    barcode_types = reg_encodings.values()[0].keys();\n",
    "    for k, v in barcode_source.iteritems():\n",
    "        if k not in barcode_types:\n",
    "            raise ValueError('wrong barcode_source input!');\n",
    "    # check barcode_order\n",
    "    for _name in barcode_order:\n",
    "        if _name not in barcode_types:\n",
    "            raise ValueError('wrong barcode_order input!');\n",
    "            \n",
    "    # filter stv_barcodes and ndb_barcodes\n",
    "    if verbose:\n",
    "        print \"- check barcode starts: \", barcode_starts\n",
    "    _stv_barcodes, _ndb_barcodes = [],[];\n",
    "    for record in stv_barcodes:\n",
    "        if not int(record.id.split('_')[1]) < barcode_starts['stv']:\n",
    "            _stv_barcodes.append(record)\n",
    "    for record in ndb_barcodes:\n",
    "        if not int(record.id.split('_')[1]) < barcode_starts['ndb']:\n",
    "            _ndb_barcodes.append(record)\n",
    "    \n",
    "    def _generating_file_encoding(_report_folder=report_folder, \n",
    "                                  _reg_encodings=reg_encodings, _verbose=verbose):\n",
    "        '''Convert region id encoding scheme into filename encoding scheme, change keys\n",
    "        Inputs: \n",
    "            report_folder\n",
    "            reg_encodings\n",
    "            verbose\n",
    "        Output:\n",
    "            pb_files\n",
    "            file_encodings'''\n",
    "        # load probe reports:\n",
    "        _pb_files = [fl for fl in glob.glob(_report_folder+os.sep+r'*.pbr') if int(os.path.basename(fl).split('_')[1].split('.')[0]) in _reg_encodings.keys()]\n",
    "        if _verbose:\n",
    "            print \"- Load probe reports, total_num:\", len(_pb_files);\n",
    "        # save to file_encodings\n",
    "        _file_encodings = {};\n",
    "        for fl in _pb_files:\n",
    "            _file_encodings[fl] = _reg_encodings[int(os.path.basename(fl).split('_')[1].split('.')[0])];\n",
    "        \n",
    "        return _pb_files, _file_encodings;\n",
    "    \n",
    "\n",
    "    \n",
    "    def _patch_barcode_per_file(_file, _file_encodings, \n",
    "                                _fwd_primer=fwd_primer, _rev_primer=rev_primer,\n",
    "                                _barcode_source=barcode_source, _stv_barcodes=stv_barcodes, _ndb_barcodes=ndb_barcodes,\n",
    "                                _add_rand_gap=add_rand_gap, _verbose=verbose):\n",
    "        from random import choice\n",
    "        import os\n",
    "        if _verbose:\n",
    "            print \"-- patch barcodes for:\", _file\n",
    "        # load probe report\n",
    "        _pb = ld.pb_reports_class()\n",
    "        _pb.load_pbr(_file)\n",
    "        \n",
    "        # extract encoding info:\n",
    "        _encoding = _file_encodings[_file];\n",
    "        \n",
    "        # initialize, save all infos here\n",
    "        _plist = [];\n",
    "        _precords = [];\n",
    "        for _info in _pb.pb_reports_keep.values():\n",
    "            _tmp_info = _info.copy();\n",
    "\n",
    "            # extract all encoding info from reg_encodings\n",
    "            _tmp_info['reg_index'] = _encoding['id']\n",
    "            _tmp_info['color'] = _encoding['color']\n",
    "            if 'gene' in _encoding.keys():\n",
    "                _tmp_info['gene'] = _encoding['gene']\n",
    "\n",
    "            # extract barcode info\n",
    "            _islist = False; # variable used for later design\n",
    "            for _k,_v in _barcode_source.iteritems():\n",
    "                if isinstance(_encoding[_k], list):\n",
    "                    _islist = _k; # variable used for later design\n",
    "                    _bcs = [];\n",
    "                    for _bid in _encoding[_k]:\n",
    "                        if _v == 'stv':\n",
    "                            _bcs.append(_stv_barcodes[_bid]);\n",
    "                        elif _v == 'ndb':\n",
    "                            _bcs.append(_ndb_barcodes[_bid]);\n",
    "                    _tmp_info[_k] = _bcs;\n",
    "                else:\n",
    "                    if _v == 'stv':\n",
    "                        _tmp_info[_k] =_stv_barcodes[_encoding[_k]];\n",
    "                    elif _v == 'ndb':\n",
    "                        _tmp_info[_k] =_ndb_barcodes[_encoding[_k]];\n",
    "            # extract primer info:\n",
    "            _tmp_info['fwd_primer'] = _fwd_primer;\n",
    "            _tmp_info['rev_primer'] = _rev_primer;\n",
    "\n",
    "            ## generate_whole sequence\n",
    "            # fwd_primer(20)\n",
    "            # barcode 1 [from list, 1], (reverse-complement of last 20)\n",
    "            # barcode 2, (reverse-complement of last 20)\n",
    "            # target sequence\n",
    "            # barcode 3, (reverse-complement of last 20)\n",
    "            # barcode 4 [from list, 1], (reverse-complement of last 20)\n",
    "            # rev_primer, (reverse-complement of last 20)\n",
    "            _seq_list = []; # start\n",
    "            _seq_list.append(_tmp_info['fwd_primer'].seq) # fwd primer\n",
    "            if _islist:\n",
    "                _seq_list += [_bc.seq[-20:].reverse_complement() for _bc in _tmp_info[_islist]]; # list barcodes, usually for decoding\n",
    "                for _k,_v in _barcode_source.iteritems():\n",
    "                    if _k != _islist:\n",
    "                        _seq_list.insert(-1, _tmp_info[_k].seq[-20:].reverse_complement()) # other barcodes\n",
    "                _seq_list.insert(-2, Seq(_tmp_info['seq']) ) # target sequence in the middle\n",
    "            else:\n",
    "                for _k,_v in _barcode_source.iteritems():\n",
    "                    _seq_list.append(_tmp_info[_k].seq[-20:].reverse_complement()) # other barcodes\n",
    "                _seq_list.insert(-2, Seq(_tmp_info['seq']) ) # target sequence in the middle\n",
    "\n",
    "            _seq_list.append(_tmp_info['rev_primer'].seq[-20:].reverse_complement()) # reverse primer\n",
    "            # result\n",
    "            dna_alphabet = ['A','A','C','G','T','T']; # used for adding random gap, if needed\n",
    "            _total_seq = Seq('');\n",
    "            for j in range(len(_seq_list)):\n",
    "                _seq = _seq_list[j]\n",
    "                _total_seq += _seq;\n",
    "                if j > 0 and j < len(_seq_list)-2:\n",
    "                    _total_seq += ''.join([choice(dna_alphabet) for i in range(_add_rand_gap)]);\n",
    "            _tmp_info['total_seq'] = _total_seq;\n",
    "\n",
    "            ## Generate total_name:\n",
    "            # chr21:10350001-10400001_reg_208_gene_chr21_pb_41577 (from base name)\n",
    "            # primer_[4,11]\n",
    "            # barcodes_75,109,[]\n",
    "\n",
    "            # base name\n",
    "            _total_name = _tmp_info['name'].split('reg_')[0] + 'reg_'+str(_tmp_info['reg_index']);\n",
    "            if 'gene' in _tmp_info['name']:\n",
    "                _total_name += '_gene' + _tmp_info['name'].split('gene')[1]\n",
    "            elif 'gene' in _tmp_info.keys():\n",
    "                _total_name += '_gene_'+_tmp_info['gene'];\n",
    "            # primer name\n",
    "            _primer_sets = [int(_tmp_info['fwd_primer'].id.split('_')[-1]), int(_tmp_info['rev_primer'].id.split('_')[-1])]\n",
    "            _total_name += '_primer_'+str(_primer_sets).replace(' ','')\n",
    "            # barcode name\n",
    "            _barcode_sets = [];\n",
    "            if _islist:\n",
    "                _barcode_sets.append([rec.id for rec in _tmp_info[_islist]]);\n",
    "                for _k,_v in _barcode_source.iteritems():\n",
    "                    if _k != _islist:\n",
    "                        _barcode_sets.append(_tmp_info[_k].id);\n",
    "            else:\n",
    "                for _k,_v in _barcode_source.iteritems():\n",
    "                    _barcode_sets.append(_tmp_info[_k].id);        \n",
    "            _total_name += '_barcodes_'+str(_barcode_sets).replace(' ','')\n",
    "            # color\n",
    "            _total_name += '_color_'+str(_tmp_info['color'])\n",
    "            \n",
    "            ## save\n",
    "            _tmp_info['total_name'] = _total_name;\n",
    "            ## Append\n",
    "            _plist.append(_tmp_info) # to plist\n",
    "            _precords.append(SeqRecord(_total_seq, id=_total_name, description='', name=_total_name)); # to seq record\n",
    "\n",
    "        return _plist, _precords    \n",
    "    \n",
    "    # generate file encoding\n",
    "    _pb_files, _file_encodings = _generating_file_encoding();\n",
    "\n",
    "    # initialize\n",
    "    _pb_lists, _pb_records = [],[];\n",
    "    # loop through all files\n",
    "    for _fl in sorted(_pb_files, key=lambda fl:int(fl.split('_')[-1].split('.')[0])):\n",
    "        _list, _records = _patch_barcode_per_file(_fl, _file_encodings);\n",
    "        _pb_lists.append(_list);\n",
    "        _pb_records += _records\n",
    "    \n",
    "    # save:\n",
    "    if save:\n",
    "        if not os.path.exists(save_folder):\n",
    "            os.makedirs(save_folder)\n",
    "        list_savefile = save_folder + os.sep + 'list.pkl';\n",
    "        pb_savefile = save_folder + os.sep + 'candidate_probes.fasta';\n",
    "        if verbose:\n",
    "            print \"- Saving list to:\", list_savefile\n",
    "        pickle.dump(_pb_lists, open(list_savefile,'w'));\n",
    "        if verbose:\n",
    "            print \"- Saving probes to:\", pb_savefile\n",
    "        with open(pb_savefile, 'w') as output_handle:\n",
    "            SeqIO.write(_pb_records, output_handle, 'fasta');\n",
    "        \n",
    "    return _pb_lists, _pb_records"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Check inputs\n",
      "- check barcode starts:  {'stv': 1, 'ndb': 1}\n",
      "- Load probe reports, total_num: 120\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_318.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_319.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_320.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_321.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_322.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_323.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_324.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_325.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_326.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_327.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_328.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_329.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_330.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_331.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_332.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_333.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_334.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_335.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_336.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_337.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_338.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_339.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_340.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_341.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_342.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_343.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_348.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_349.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_350.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_351.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_352.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_353.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_354.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_355.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_356.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_357.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_358.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_359.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_360.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_361.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_362.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_363.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_364.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_365.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_366.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_367.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_368.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_369.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_370.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_371.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_372.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_373.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_374.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_375.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_376.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_377.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_378.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_379.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_380.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_381.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_650.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_651.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_652.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_653.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_654.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_655.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_656.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_657.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_658.pbr\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_659.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_660.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_661.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_662.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_663.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_664.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_665.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_666.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_667.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_668.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_669.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_670.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_671.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_672.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_673.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_674.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_675.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_676.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_677.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_678.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_679.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_680.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_681.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_682.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_683.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_684.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_685.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_686.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_687.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_688.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_689.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_690.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_691.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_692.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_693.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_694.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_695.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_696.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_697.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_698.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_699.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_700.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_701.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_702.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_703.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_704.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_705.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_706.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_707.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_708.pbr\n",
      "-- patch barcodes for: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/SI-14/chr21/reports/centered_merged/reg_709.pbr\n",
      "- Saving list to: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/CTP-02/chr21/final_probes/list.pkl\n",
      "- Saving probes to: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/CTP-02/chr21/final_probes/candidate_probes.fasta\n"
     ]
    }
   ],
   "source": [
    "pb_lists, pb_records = Patch_Barcodes(reg_encodings=sub_encodings,\n",
    "                                      fwd_primer=fprimer, rev_primer=rprimer, \n",
    "                                      barcode_source=barcode_source, barcode_order=barcode_order, \n",
    "                                      stv_barcodes=stv_barcodes, ndb_barcodes=ndb_barcodes)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 5. Check probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- forward primer: ID: W1A01_primer_0\n",
      "Name: W1A01_primer_0\n",
      "Description: W1A01_primer_0\n",
      "Number of features: 0\n",
      "Seq('CGGCTCGCAGCGTGTAAACG', SingleLetterAlphabet())\n",
      "- reverse primer: ID: W1A02_primer_1\n",
      "Name: W1A02_primer_1\n",
      "Description: W1A02_primer_1\n",
      "Number of features: 0\n",
      "Seq('TAATACGACTCACTATAGGGCATTTCAGGATCACCGGCGG', SingleLetterAlphabet())\n"
     ]
    }
   ],
   "source": [
    "import cPickle as pickle\n",
    "\n",
    "master_dir =r'/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/CTP-02/chr21';\n",
    "pb_dir = r'final_probes';\n",
    "\n",
    "# primers\n",
    "fprimer = fwd_primers[0];\n",
    "print '- forward primer:', fprimer\n",
    "rprimer = rev_primers[0];\n",
    "print '- reverse primer:', rprimer\n",
    "\n",
    "# dic for region -> tad\n",
    "if not 'sub_encodings' in vars():\n",
    "    print 'loading sub_encodings'\n",
    "    sub_encodings = pickle.load(open(master_dir+os.sep+'sub_encoding.pkl','r'))\n",
    "if not 'pb_records' in vars():\n",
    "    print '- loading all probes'\n",
    "    with open(master_dir+os.sep+pb_dir+os.sep+'candidate_probes.fasta', \"rU\") as handle:\n",
    "        pb_records = [];\n",
    "        for record in SeqIO.parse(handle, \"fasta\"):\n",
    "            pb_records.append(record);\n",
    "if not 'pb_lists' in vars():\n",
    "    print '- loading pb_lists'\n",
    "    pb_lists = pickle.load(open(master_dir+os.sep+pb_dir+os.sep+'list.pkl', \"rU\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def Check_Probes(pb_records, pb_lists, reg_encodings, master_dir, \n",
    "                 fwd_primer,rev_primer,\n",
    "                 stv_barcodes, ndb_barcodes, barcode_starts={'stv':1,'ndb':1},\n",
    "                 report_dir=r'reports/centered_merged',save_dir=r'final_probes',\n",
    "                 add_rand_gap=0, total_bc=4, barcode_len=20, target_len=42,  \n",
    "                 word_size=17, max_internal_hits=5, max_genome_hits=200,\n",
    "                 index_dir=r'/n/boslfs/LABS/zhuang_lab/User/pzheng/Indeces/human/hg38',\n",
    "                 save=True, verbose=True):\n",
    "    # imports\n",
    "    import os,glob,sys\n",
    "    sys.path.append(r'/n/home13/pzheng/Documents/python-functions/python-functions-library')\n",
    "    from LibraryConstruction import fastaread,fastawrite,fastacombine\n",
    "    import LibraryDesigner as ld\n",
    "    import numpy as np\n",
    "    \n",
    "    def _check_primer_usage(pb_records=pb_records, fwd_primer=fwd_primer, rev_primer=rev_primer,\n",
    "                            _verbose=verbose):\n",
    "        '''Check whether forward or reverse primer are used in all probes'''\n",
    "        if _verbose:\n",
    "            print \"-- Checking primer usage, total probes:\", len(pb_records)\n",
    "        fwd_len = len(fwd_primer.seq);\n",
    "        rev_len = len(rev_primer.seq[-20:].reverse_complement());\n",
    "        \n",
    "        for record in pb_records:\n",
    "            if record.seq[:fwd_len] != fwd_primer.seq:\n",
    "                if _verbose:\n",
    "                    print \"--- Forward primer incorrect!\"\n",
    "                return False\n",
    "            if record.seq[-rev_len:] != rev_primer.seq[-20:].reverse_complement():\n",
    "                if _verbose:\n",
    "                    print \"--- Forward primer incorrect!\"\n",
    "                return False\n",
    "        return True # if no error applies\n",
    "    \n",
    "    def _check_region_size(pb_records=pb_records, pb_lists=pb_lists):\n",
    "        '''Generate a dirctionary '''\n",
    "        # get original region size\n",
    "        _reg_size_dic = {}\n",
    "        for lst in pb_lists:\n",
    "            _reg_size_dic[lst[0]['reg_index']] = len(lst);\n",
    "        # get region size from probe names\n",
    "        _size_from_rec = {}\n",
    "        for record in pb_records:\n",
    "            reg_id = int(record.id.split('_reg_')[1].split('_')[0]);\n",
    "            if reg_id not in _size_from_rec.keys():\n",
    "                _size_from_rec[reg_id] = 1; # if not in key, create\n",
    "            else:\n",
    "                _size_from_rec[reg_id] += 1; # otherwise, add count\n",
    "        # compare\n",
    "        _match = True;\n",
    "        for k,v in sorted(_size_from_rec.items()):\n",
    "            if k not in _reg_size_dic.keys():\n",
    "                print \"region list and region id in probes not match for\", k\n",
    "                _match = False\n",
    "                break\n",
    "            else:\n",
    "                if v != _reg_size_dic[k]:\n",
    "                    print \"region size doesn't match for:\", k\n",
    "                    _match = False\n",
    "                    break\n",
    "        \n",
    "        return _reg_size_dic, _match;\n",
    "    \n",
    "    def _check_gene_size():\n",
    "        pass\n",
    "    \n",
    "\n",
    "    def _check_region_to_barcode(pb_records=pb_records, stv_barcodes=stv_barcodes, ndb_barcodes=ndb_barcodes,\n",
    "                                 total_bc=total_bc):\n",
    "        '''Generate map from region id to barcodes used in this region'''\n",
    "        import re\n",
    "        _reg_to_barcode = {}\n",
    "        for record in pb_records:\n",
    "            # region id\n",
    "            reg_id = int(record.id.split('_reg_')[1].split('_')[0]);\n",
    "            if reg_id not in _reg_to_barcode.keys():\n",
    "                # barcode ids\n",
    "                stv_matches = re.findall('\\'Stv_(.+?)\\'', record.id, re.DOTALL)\n",
    "                ndb_matches = re.findall('\\'NDB_(.+?)\\'', record.id, re.DOTALL)\n",
    "                stv_names = ['Stv_'+str(stv_id) for stv_id in stv_matches]\n",
    "                ndb_names = ['NDB_'+str(ndb_id) for ndb_id in ndb_matches]\n",
    "                _reg_to_barcode[reg_id] = stv_names+ndb_names\n",
    "        \n",
    "        ## barcode check\n",
    "        _barcode_check = True;\n",
    "        # barcode names\n",
    "        bc_names = [stv.id for stv in stv_barcodes] + [ndb.id for ndb in ndb_barcodes]\n",
    "        # search through previous dictionary\n",
    "        for reg,bcs in sorted(_reg_to_barcode.items()):\n",
    "            for bc in bcs:\n",
    "                if len(bcs) != total_bc:\n",
    "                    print \"-- Error in barcode number for region:\", reg\n",
    "                    _barcode_check = False\n",
    "                    break\n",
    "                if bc not in bc_names:\n",
    "                    print \"-- Wrong barcode name for barcode: \"+str(bc)+\", region: \"+str(reg)\n",
    "                    _barcode_check = False\n",
    "                    break\n",
    "        \n",
    "        return _reg_to_barcode, _barcode_check;\n",
    "        \n",
    "    def _parsing_probe_sequence(record, fwd_primer=fwd_primer, rev_primer=rev_primer,\n",
    "                                add_rand_gap=add_rand_gap, barcode_len=barcode_len, target_len=target_len):\n",
    "        '''parse a probe sequence to acquire all barcode binding sites'''\n",
    "        # take in a seq record, parse the sequence and return a list of all included barcodes (20mer,RC)\n",
    "        barcode_list = [];\n",
    "        _main_seq = record.seq[len(fwd_primer.seq):-20];\n",
    "        \n",
    "        \n",
    "        # trim last 2 barcodes\n",
    "        for i in range(2):\n",
    "            barcode_list.append(_main_seq[-barcode_len:]);\n",
    "            _main_seq = _main_seq[:-(barcode_len+add_rand_gap)];\n",
    "        # trim all barcodes from the beginning\n",
    "        while len(_main_seq) > target_len:\n",
    "            barcode_list.append(_main_seq[:barcode_len]);\n",
    "            _main_seq = _main_seq[(barcode_len+add_rand_gap):];\n",
    "        \n",
    "        return barcode_list;\n",
    "    \n",
    "    def _finding_barcode_name(barcode_list, stv_barcodes=stv_barcodes, ndb_barcodes=ndb_barcodes, \n",
    "                              barcode_len=barcode_len, total_bc=total_bc):\n",
    "        '''Given barcode list generated by parsing probe, return a list of barcode names'''\n",
    "        _name_list = [];\n",
    "        for bc_site in barcode_list:\n",
    "            for bc in stv_barcodes+ndb_barcodes:\n",
    "                if bc.seq[-barcode_len:] == bc_site.reverse_complement():\n",
    "                    _name_list.append(bc.id);\n",
    "                    break;\n",
    "        \n",
    "        if len(_name_list) < total_bc:\n",
    "            print \"-- Failed in finding some barcodes.\"\n",
    "            return False\n",
    "        return _name_list;\n",
    "    \n",
    "    def _check_barcode_to_gene():\n",
    "        pass\n",
    "    \n",
    "    def _check_barcode_to_region(reg_to_barcode, \n",
    "                                 pb_records=pb_records, stv_barcodes=stv_barcodes, ndb_barcodes=ndb_barcodes):\n",
    "        '''Generate map from barcode id to region id'''\n",
    "        _barcode_to_reg = {}\n",
    "        _reg_id_exists = []\n",
    "        for record in pb_records:\n",
    "            reg_id = int(record.id.split('_reg_')[1].split('_')[0]);\n",
    "            if reg_id in _reg_id_exists:\n",
    "                continue;\n",
    "            else:\n",
    "                _barcode_list = _parsing_probe_sequence(record)\n",
    "                _name_list = _finding_barcode_name(_barcode_list)\n",
    "                for _n in _name_list:\n",
    "                    if _n not in _barcode_to_reg.keys(): # create if not in dic\n",
    "                        _barcode_to_reg[_n] = [reg_id]\n",
    "                    else: # otherwise, append\n",
    "                        _barcode_to_reg[_n].append(reg_id)\n",
    "            _reg_id_exists.append(reg_id)\n",
    "        ## check region distribution\n",
    "        # invert dic from reg_to_barcode\n",
    "        _inv_dic = {}\n",
    "        for reg,bcs in sorted(reg_to_barcode.items()):\n",
    "            for bc in bcs:\n",
    "                if bc not in _inv_dic.keys():\n",
    "                    _inv_dic[bc] = [reg];\n",
    "                else:\n",
    "                    _inv_dic[bc].append(reg);\n",
    "        # compare\n",
    "        _region_check=True\n",
    "        for bc, regs in sorted(_inv_dic.items()):\n",
    "            if bc not in _barcode_to_reg.keys():\n",
    "                print \"-- \"+str(bc)+\" not in barcode_to_region dic!\"\n",
    "                _region_check = False\n",
    "                break\n",
    "            else:\n",
    "                if sorted(regs) != sorted(_barcode_to_reg[bc]):\n",
    "                    print \"-- \"+str(bc)+\" and region\"+str(regs)+\" not compatible with barcode_to_region dic!\"\n",
    "                    _region_check = False\n",
    "                    break\n",
    "                    \n",
    "        return _barcode_to_reg, _region_check\n",
    "    \n",
    "    def _check_barcode_to_color(pb_records=pb_records, stv_barcodes=stv_barcodes, ndb_barcodes=ndb_barcodes, \n",
    "                                stv_color=True, ndb_color=False,\n",
    "                                _save=save, master_dir=master_dir, save_dir=save_dir):\n",
    "        '''If multi_color is applied, generate a barcode_to_color dic for adaptor design'''\n",
    "        if 'color' not in str(pb_records[0].id):\n",
    "            print \"-- color check not applied\";\n",
    "            return False\n",
    "        elif not stv_color and not ndb_color:\n",
    "            print \"-- color check turned off in both stv and ndb\";\n",
    "            return False\n",
    "        else:\n",
    "            # get barcodes\n",
    "            _barcode_names = []\n",
    "            if stv_color: # if stv has multi-color\n",
    "                _barcode_names += [bc.id for bc in stv_barcodes];\n",
    "            if ndb_color: # if ndb has multi-color\n",
    "                _barcode_names += [bc.id for bc in ndb_barcodes];\n",
    "            # initialize color dic\n",
    "            _barcode_to_color = {};\n",
    "            _exist_regs = [];\n",
    "            # search through all probes\n",
    "            for record in pb_records:\n",
    "                _reg_id = int(record.id.split('_reg_')[1].split('_')[0]); \n",
    "                if _reg_id in _exist_regs:\n",
    "                    continue\n",
    "                else: \n",
    "                    _exist_regs.append(_reg_id);\n",
    "                _color = int(str(record.id).split('color_')[1])\n",
    "                _barcode_list = _parsing_probe_sequence(record)\n",
    "                _name_list = _finding_barcode_name(_barcode_list)\n",
    "                \n",
    "                for _name in _name_list:\n",
    "                    if _name in _barcode_names:\n",
    "                        if _name not in _barcode_to_color.keys():\n",
    "                            _barcode_to_color[_name] = [_color]\n",
    "                        else:\n",
    "                            _barcode_to_color[_name].append(_color);\n",
    "            # keep the unique colors\n",
    "            _barcode_to_unique_color = {}\n",
    "            for k,v in sorted(_barcode_to_color.items()):\n",
    "                _barcode_to_unique_color[k] = np.unique(v)\n",
    "            if _save:\n",
    "                import csv\n",
    "                # mkdir if not exist for this region\n",
    "                if not os.path.exists(master_dir+os.sep+save_dir):\n",
    "                    os.makedirs(master_dir+os.sep+save_dir)\n",
    "                with open(master_dir+os.sep+save_dir+os.sep+'color-usage.csv','w') as output_handle:\n",
    "                    fieldnames = ['barcode', 'color']\n",
    "                    writer = csv.DictWriter(output_handle, fieldnames=fieldnames)\n",
    "                    writer.writeheader()\n",
    "                    for _barcode, _color in sorted(_barcode_to_unique_color.items(), key=lambda (k,v):int(k.split('_')[1])):\n",
    "                        writer.writerow({'barcode': _barcode, 'color': _color})\n",
    "                \n",
    "        return _barcode_to_unique_color\n",
    "                            \n",
    "    \n",
    "    def _construct_internal_map(master_dir=master_dir, save_dir=save_dir, word_size=word_size):\n",
    "        '''Using functions in LibraryDesign, compute an internal khmer map'''\n",
    "        _int_map = khmer.Countgraph(word_size, 1e9, 2) \n",
    "        _int_map.set_use_bigcount(True)\n",
    "        _nms,_seqs = fastaread(master_dir+os.sep+save_dir+os.sep+'candidate_probes.fasta')\n",
    "        for _seq in _seqs:\n",
    "            _int_map.consume(_seq.upper())\n",
    "        return _int_map\n",
    "    \n",
    "    def _check_barcode_in_probes(barcode_to_reg, reg_size_dic, int_map, \n",
    "                                 stv_barcodes=stv_barcodes, ndb_barcodes=ndb_barcodes,\n",
    "                                 barcode_len=barcode_len, max_internal_hits=max_internal_hits):\n",
    "        '''Check barcode appearance in probes, whether that match barcode_to_region scheme'''\n",
    "        _barcode_in_probes = {}\n",
    "        for bc_name, regs in sorted(barcode_to_reg.items()):\n",
    "            bc = None\n",
    "            for _bc in stv_barcodes+ndb_barcodes:\n",
    "                if bc_name == _bc.id:\n",
    "                    bc = _bc\n",
    "                    break\n",
    "            bc_hits = int_map.get_kmer_counts( str(bc.seq[-barcode_len:].reverse_complement()).upper());\n",
    "            if max(bc_hits) - min(bc_hits) > max_internal_hits:\n",
    "                print \"-- Barcode: \"+str(bc)+\" has more off-target in different part of itself!\"\n",
    "                return False\n",
    "            else:\n",
    "                regs,reg_cts = np.unique(regs, return_counts=True);\n",
    "                bc_in_probe = 0;\n",
    "                for reg,ct in zip(regs,reg_cts):\n",
    "                    bc_in_probe += reg_size_dic[reg] * ct;\n",
    "                if max(bc_hits) - bc_in_probe > max_internal_hits:\n",
    "                    print \"-- Barcode: \"+str(bc)+\" has more off-target than threshold!\"\n",
    "                    return False\n",
    "            _barcode_in_probes[bc_name] = bc_in_probe;\n",
    "        return _barcode_in_probes, True\n",
    "    \n",
    "    def _check_between_probes(int_map, pb_lists=pb_lists, pb_records=pb_records):\n",
    "        pass \n",
    "    \n",
    "    def _check_against_genome(pb_records=pb_records, max_genome_hits=max_genome_hits, index_dir=index_dir):\n",
    "        '''Use Khmer to compare probe against genome'''\n",
    "        hg38 = khmer.load_countgraph(index_dir+os.sep+'full_word17_.kmer')\n",
    "        _failed_num = 0;\n",
    "        _keep_pb_records = [];\n",
    "        for record in pb_records:\n",
    "            _kmer_hits = hg38.get_kmer_counts(str(record.seq).upper());\n",
    "            if sum(_kmer_hits) > max_genome_hits:\n",
    "                print '-- Max_genome_hits is: '+str(max_genome_hits)+\", this seq got hits: \"+ str(sum(_kmer_hits))\n",
    "                _failed_num += 1;\n",
    "            else:\n",
    "                _keep_pb_records.append(record);\n",
    "                \n",
    "        return _keep_pb_records, _failed_num # if nothing goes wrong\n",
    "    \n",
    "    def _plot_info():\n",
    "        pass\n",
    "            \n",
    "    ## check primers\n",
    "    primer_usage = _check_primer_usage()\n",
    "    if verbose:\n",
    "        print \"\\n- 1.Passing primer usage check? -\", primer_usage\n",
    "    \n",
    "    ## check region size\n",
    "    reg_size_dic, size_match = _check_region_size()\n",
    "    if verbose:\n",
    "        print \"\\n- 2.Passing region size check? -\", size_match    \n",
    "        for k,v in sorted(reg_size_dic.items()):\n",
    "            print k,':',v\n",
    "        \n",
    "    ## check region to barcode\n",
    "    reg_to_barcode, reg2bc = _check_region_to_barcode()\n",
    "    if verbose:\n",
    "        print \"\\n- 3.Passing region to barcode mapping check? -\", reg2bc    \n",
    "        for k,v in sorted(reg_to_barcode.items(), key=lambda (k,v):k):\n",
    "            print k,':',v\n",
    "        \n",
    "    ## check barcode to region (this step must be run after step 3) \n",
    "    barcode_to_reg, bc2reg = _check_barcode_to_region(reg_to_barcode)\n",
    "    if verbose:\n",
    "        print \"\\n- 4.Passing barcode to region mapping check? -\", bc2reg    \n",
    "        for k,v in sorted(barcode_to_reg.items(), key=lambda (k,v):[k[0],int(k.split('_')[1])]):\n",
    "            print k,':',v\n",
    "    \n",
    "    ## check barcode to region (this step must be run after step 3) \n",
    "    barcode_to_color = _check_barcode_to_color()\n",
    "    if verbose:\n",
    "        print \"\\n- 5.Calculating barcode to color dictionary.\"\n",
    "        for k,v in sorted(barcode_to_color.items(), key=lambda (k,v):[k[0],int(k.split('_')[1])]):\n",
    "            print k,':',v    \n",
    "    \n",
    "    \n",
    "    ## Construct an internal map\n",
    "    int_map = _construct_internal_map();\n",
    "    if verbose:\n",
    "        print \"\\n- 6.Constructing internal khmer map\";\n",
    "    \n",
    "    ## Check barcodes total counts in probes\n",
    "    barcode_in_probes, _bc_counting = _check_barcode_in_probes(barcode_to_reg, reg_size_dic, int_map)\n",
    "    if verbose:\n",
    "        print \"\\n- 7.Passing if counting barcode appearance times in probes\", _bc_counting;    \n",
    "\n",
    "    ## Check against each other    \n",
    "    \n",
    "    ## Check against genome\n",
    "    kept_records, failed_num = _check_against_genome();\n",
    "    if verbose:\n",
    "        print \"\\n- 8.Probes not passing through genome filter:\", failed_num;  \n",
    "    \n",
    "    # check region size for kept probes\n",
    "    _size_from_rec = {}\n",
    "    for record in pb_records:\n",
    "        reg_id = int(record.id.split('_reg_')[1].split('_')[0]);\n",
    "        if reg_id not in _size_from_rec.keys():\n",
    "            _size_from_rec[reg_id] = 1; # if not in key, create\n",
    "        else:\n",
    "            _size_from_rec[reg_id] += 1; # otherwise, add count\n",
    "    if verbose:\n",
    "        print \"--  re-check region size:\"\n",
    "        for k,v in sorted(_size_from_rec.items()):\n",
    "            print k,':',v\n",
    "        print \"--- total number of probes:\", len(pb_records);\n",
    "    if save:\n",
    "        pb_savefile = master_dir + os.sep + save_dir + os.sep + 'filtered_probes.fasta';\n",
    "        if verbose:\n",
    "            print \"\\n- 9.Saving probes to:\", pb_savefile\n",
    "        with open(pb_savefile, 'w') as output_handle:\n",
    "            SeqIO.write(kept_records, output_handle, 'fasta');  \n",
    "        \n",
    "    return kept_records, _size_from_rec"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- Checking primer usage, total probes: 57479\n",
      "\n",
      "- 1.Passing primer usage check? - True\n",
      "\n",
      "- 2.Passing region size check? - True\n",
      "41 : 500\n",
      "42 : 395\n",
      "43 : 500\n",
      "44 : 500\n",
      "45 : 500\n",
      "46 : 477\n",
      "47 : 500\n",
      "48 : 500\n",
      "49 : 500\n",
      "50 : 500\n",
      "51 : 500\n",
      "52 : 500\n",
      "53 : 500\n",
      "54 : 500\n",
      "55 : 500\n",
      "56 : 500\n",
      "57 : 500\n",
      "58 : 500\n",
      "59 : 500\n",
      "60 : 500\n",
      "61 : 495\n",
      "62 : 433\n",
      "63 : 499\n",
      "64 : 500\n",
      "65 : 500\n",
      "66 : 435\n",
      "67 : 412\n",
      "68 : 500\n",
      "69 : 475\n",
      "70 : 500\n",
      "71 : 500\n",
      "72 : 283\n",
      "73 : 202\n",
      "74 : 205\n",
      "75 : 500\n",
      "76 : 500\n",
      "77 : 500\n",
      "78 : 500\n",
      "79 : 500\n",
      "80 : 500\n",
      "81 : 494\n",
      "82 : 500\n",
      "83 : 500\n",
      "84 : 500\n",
      "85 : 500\n",
      "86 : 500\n",
      "87 : 500\n",
      "88 : 483\n",
      "89 : 500\n",
      "90 : 500\n",
      "91 : 500\n",
      "92 : 500\n",
      "93 : 500\n",
      "94 : 497\n",
      "95 : 488\n",
      "96 : 500\n",
      "97 : 424\n",
      "98 : 500\n",
      "99 : 500\n",
      "100 : 500\n",
      "367 : 500\n",
      "368 : 500\n",
      "369 : 500\n",
      "370 : 500\n",
      "371 : 500\n",
      "372 : 500\n",
      "373 : 410\n",
      "374 : 385\n",
      "375 : 500\n",
      "376 : 500\n",
      "377 : 500\n",
      "378 : 500\n",
      "379 : 500\n",
      "380 : 500\n",
      "381 : 500\n",
      "382 : 500\n",
      "383 : 500\n",
      "384 : 500\n",
      "385 : 500\n",
      "386 : 291\n",
      "387 : 213\n",
      "388 : 495\n",
      "389 : 500\n",
      "390 : 462\n",
      "391 : 500\n",
      "392 : 490\n",
      "393 : 500\n",
      "394 : 500\n",
      "395 : 500\n",
      "396 : 500\n",
      "397 : 500\n",
      "398 : 500\n",
      "399 : 500\n",
      "400 : 500\n",
      "401 : 500\n",
      "402 : 500\n",
      "403 : 500\n",
      "404 : 500\n",
      "405 : 480\n",
      "406 : 297\n",
      "407 : 259\n",
      "408 : 500\n",
      "409 : 500\n",
      "410 : 500\n",
      "411 : 500\n",
      "412 : 500\n",
      "413 : 500\n",
      "414 : 500\n",
      "415 : 500\n",
      "416 : 500\n",
      "417 : 500\n",
      "418 : 500\n",
      "419 : 500\n",
      "420 : 500\n",
      "421 : 500\n",
      "422 : 500\n",
      "423 : 500\n",
      "424 : 500\n",
      "425 : 500\n",
      "426 : 500\n",
      "\n",
      "- 3.Passing region to barcode mapping check? - True\n",
      "41 : ['Stv_3', 'NDB_35', 'NDB_3']\n",
      "42 : ['Stv_4', 'NDB_36', 'NDB_3']\n",
      "43 : ['Stv_5', 'NDB_37', 'NDB_3']\n",
      "44 : ['Stv_6', 'NDB_38', 'NDB_3']\n",
      "45 : ['Stv_7', 'NDB_39', 'NDB_3']\n",
      "46 : ['Stv_8', 'NDB_40', 'NDB_3']\n",
      "47 : ['Stv_9', 'NDB_41', 'NDB_3']\n",
      "48 : ['Stv_10', 'NDB_42', 'NDB_3']\n",
      "49 : ['Stv_11', 'NDB_43', 'NDB_3']\n",
      "50 : ['Stv_12', 'NDB_44', 'NDB_3']\n",
      "51 : ['Stv_13', 'NDB_45', 'NDB_3']\n",
      "52 : ['Stv_14', 'NDB_46', 'NDB_3']\n",
      "53 : ['Stv_16', 'NDB_47', 'NDB_3']\n",
      "54 : ['Stv_19', 'NDB_48', 'NDB_3']\n",
      "55 : ['Stv_20', 'NDB_49', 'NDB_3']\n",
      "56 : ['Stv_21', 'NDB_50', 'NDB_3']\n",
      "57 : ['Stv_22', 'NDB_51', 'NDB_3']\n",
      "58 : ['Stv_23', 'NDB_52', 'NDB_3']\n",
      "59 : ['Stv_25', 'NDB_53', 'NDB_3']\n",
      "60 : ['Stv_26', 'NDB_54', 'NDB_3']\n",
      "61 : ['Stv_27', 'NDB_55', 'NDB_3']\n",
      "62 : ['Stv_28', 'NDB_56', 'NDB_3']\n",
      "63 : ['Stv_29', 'NDB_57', 'NDB_3']\n",
      "64 : ['Stv_30', 'NDB_58', 'NDB_3']\n",
      "65 : ['Stv_31', 'NDB_59', 'NDB_3']\n",
      "66 : ['Stv_32', 'NDB_60', 'NDB_3']\n",
      "67 : ['Stv_33', 'NDB_61', 'NDB_3']\n",
      "68 : ['Stv_35', 'NDB_62', 'NDB_3']\n",
      "69 : ['Stv_36', 'NDB_63', 'NDB_3']\n",
      "70 : ['Stv_37', 'NDB_64', 'NDB_3']\n",
      "71 : ['Stv_39', 'NDB_65', 'NDB_3']\n",
      "72 : ['Stv_40', 'NDB_66', 'NDB_3']\n",
      "73 : ['Stv_42', 'NDB_67', 'NDB_3']\n",
      "74 : ['Stv_44', 'NDB_68', 'NDB_3']\n",
      "75 : ['Stv_45', 'NDB_69', 'NDB_3']\n",
      "76 : ['Stv_46', 'NDB_70', 'NDB_3']\n",
      "77 : ['Stv_48', 'NDB_71', 'NDB_3']\n",
      "78 : ['Stv_50', 'NDB_72', 'NDB_3']\n",
      "79 : ['Stv_53', 'NDB_73', 'NDB_3']\n",
      "80 : ['Stv_54', 'NDB_74', 'NDB_3']\n",
      "81 : ['Stv_59', 'NDB_75', 'NDB_4']\n",
      "82 : ['Stv_60', 'NDB_76', 'NDB_4']\n",
      "83 : ['Stv_61', 'NDB_77', 'NDB_4']\n",
      "84 : ['Stv_63', 'NDB_78', 'NDB_4']\n",
      "85 : ['Stv_64', 'NDB_79', 'NDB_4']\n",
      "86 : ['Stv_65', 'NDB_80', 'NDB_4']\n",
      "87 : ['Stv_86', 'NDB_81', 'NDB_4']\n",
      "88 : ['Stv_87', 'NDB_82', 'NDB_4']\n",
      "89 : ['Stv_88', 'NDB_83', 'NDB_4']\n",
      "90 : ['Stv_90', 'NDB_84', 'NDB_4']\n",
      "91 : ['Stv_91', 'NDB_85', 'NDB_4']\n",
      "92 : ['Stv_92', 'NDB_86', 'NDB_4']\n",
      "93 : ['Stv_94', 'NDB_87', 'NDB_4']\n",
      "94 : ['Stv_95', 'NDB_88', 'NDB_4']\n",
      "95 : ['Stv_99', 'NDB_89', 'NDB_4']\n",
      "96 : ['Stv_100', 'NDB_90', 'NDB_4']\n",
      "97 : ['Stv_101', 'NDB_91', 'NDB_4']\n",
      "98 : ['Stv_104', 'NDB_92', 'NDB_4']\n",
      "99 : ['Stv_105', 'NDB_93', 'NDB_4']\n",
      "100 : ['Stv_106', 'NDB_94', 'NDB_4']\n",
      "367 : ['Stv_3', 'NDB_95', 'NDB_14']\n",
      "368 : ['Stv_4', 'NDB_96', 'NDB_14']\n",
      "369 : ['Stv_5', 'NDB_97', 'NDB_14']\n",
      "370 : ['Stv_6', 'NDB_98', 'NDB_14']\n",
      "371 : ['Stv_7', 'NDB_99', 'NDB_14']\n",
      "372 : ['Stv_8', 'NDB_100', 'NDB_14']\n",
      "373 : ['Stv_9', 'NDB_101', 'NDB_15']\n",
      "374 : ['Stv_10', 'NDB_102', 'NDB_15']\n",
      "375 : ['Stv_11', 'NDB_103', 'NDB_15']\n",
      "376 : ['Stv_12', 'NDB_104', 'NDB_15']\n",
      "377 : ['Stv_13', 'NDB_105', 'NDB_15']\n",
      "378 : ['Stv_14', 'NDB_106', 'NDB_15']\n",
      "379 : ['Stv_16', 'NDB_107', 'NDB_15']\n",
      "380 : ['Stv_19', 'NDB_108', 'NDB_15']\n",
      "381 : ['Stv_20', 'NDB_109', 'NDB_15']\n",
      "382 : ['Stv_21', 'NDB_110', 'NDB_15']\n",
      "383 : ['Stv_22', 'NDB_111', 'NDB_15']\n",
      "384 : ['Stv_23', 'NDB_112', 'NDB_15']\n",
      "385 : ['Stv_25', 'NDB_113', 'NDB_15']\n",
      "386 : ['Stv_26', 'NDB_114', 'NDB_15']\n",
      "387 : ['Stv_27', 'NDB_115', 'NDB_15']\n",
      "388 : ['Stv_28', 'NDB_116', 'NDB_16']\n",
      "389 : ['Stv_29', 'NDB_117', 'NDB_16']\n",
      "390 : ['Stv_30', 'NDB_118', 'NDB_16']\n",
      "391 : ['Stv_31', 'NDB_119', 'NDB_16']\n",
      "392 : ['Stv_32', 'NDB_120', 'NDB_16']\n",
      "393 : ['Stv_33', 'NDB_121', 'NDB_16']\n",
      "394 : ['Stv_35', 'NDB_122', 'NDB_16']\n",
      "395 : ['Stv_36', 'NDB_123', 'NDB_16']\n",
      "396 : ['Stv_37', 'NDB_124', 'NDB_17']\n",
      "397 : ['Stv_39', 'NDB_125', 'NDB_17']\n",
      "398 : ['Stv_40', 'NDB_126', 'NDB_17']\n",
      "399 : ['Stv_42', 'NDB_127', 'NDB_17']\n",
      "400 : ['Stv_44', 'NDB_128', 'NDB_17']\n",
      "401 : ['Stv_45', 'NDB_129', 'NDB_17']\n",
      "402 : ['Stv_46', 'NDB_130', 'NDB_17']\n",
      "403 : ['Stv_48', 'NDB_131', 'NDB_17']\n",
      "404 : ['Stv_50', 'NDB_132', 'NDB_17']\n",
      "405 : ['Stv_53', 'NDB_133', 'NDB_17']\n",
      "406 : ['Stv_54', 'NDB_134', 'NDB_18']\n",
      "407 : ['Stv_59', 'NDB_135', 'NDB_18']\n",
      "408 : ['Stv_60', 'NDB_136', 'NDB_18']\n",
      "409 : ['Stv_61', 'NDB_137', 'NDB_18']\n",
      "410 : ['Stv_63', 'NDB_138', 'NDB_18']\n",
      "411 : ['Stv_64', 'NDB_139', 'NDB_19']\n",
      "412 : ['Stv_65', 'NDB_140', 'NDB_19']\n",
      "413 : ['Stv_86', 'NDB_141', 'NDB_19']\n",
      "414 : ['Stv_87', 'NDB_142', 'NDB_19']\n",
      "415 : ['Stv_88', 'NDB_143', 'NDB_19']\n",
      "416 : ['Stv_90', 'NDB_144', 'NDB_19']\n",
      "417 : ['Stv_91', 'NDB_145', 'NDB_19']\n",
      "418 : ['Stv_92', 'NDB_146', 'NDB_19']\n",
      "419 : ['Stv_94', 'NDB_147', 'NDB_19']\n",
      "420 : ['Stv_95', 'NDB_148', 'NDB_19']\n",
      "421 : ['Stv_99', 'NDB_149', 'NDB_19']\n",
      "422 : ['Stv_100', 'NDB_150', 'NDB_19']\n",
      "423 : ['Stv_101', 'NDB_151', 'NDB_19']\n",
      "424 : ['Stv_104', 'NDB_152', 'NDB_19']\n",
      "425 : ['Stv_105', 'NDB_153', 'NDB_19']\n",
      "426 : ['Stv_106', 'NDB_154', 'NDB_19']\n",
      "\n",
      "- 4.Passing barcode to region mapping check? - True\n",
      "NDB_3 : [41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80]\n",
      "NDB_4 : [81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99, 100]\n",
      "NDB_14 : [367, 368, 369, 370, 371, 372]\n",
      "NDB_15 : [373, 374, 375, 376, 377, 378, 379, 380, 381, 382, 383, 384, 385, 386, 387]\n",
      "NDB_16 : [388, 389, 390, 391, 392, 393, 394, 395]\n",
      "NDB_17 : [396, 397, 398, 399, 400, 401, 402, 403, 404, 405]\n",
      "NDB_18 : [406, 407, 408, 409, 410]\n",
      "NDB_19 : [411, 412, 413, 414, 415, 416, 417, 418, 419, 420, 421, 422, 423, 424, 425, 426]\n",
      "NDB_35 : [41]\n",
      "NDB_36 : [42]\n",
      "NDB_37 : [43]\n",
      "NDB_38 : [44]\n",
      "NDB_39 : [45]\n",
      "NDB_40 : [46]\n",
      "NDB_41 : [47]\n",
      "NDB_42 : [48]\n",
      "NDB_43 : [49]\n",
      "NDB_44 : [50]\n",
      "NDB_45 : [51]\n",
      "NDB_46 : [52]\n",
      "NDB_47 : [53]\n",
      "NDB_48 : [54]\n",
      "NDB_49 : [55]\n",
      "NDB_50 : [56]\n",
      "NDB_51 : [57]\n",
      "NDB_52 : [58]\n",
      "NDB_53 : [59]\n",
      "NDB_54 : [60]\n",
      "NDB_55 : [61]\n",
      "NDB_56 : [62]\n",
      "NDB_57 : [63]\n",
      "NDB_58 : [64]\n",
      "NDB_59 : [65]\n",
      "NDB_60 : [66]\n",
      "NDB_61 : [67]\n",
      "NDB_62 : [68]\n",
      "NDB_63 : [69]\n",
      "NDB_64 : [70]\n",
      "NDB_65 : [71]\n",
      "NDB_66 : [72]\n",
      "NDB_67 : [73]\n",
      "NDB_68 : [74]\n",
      "NDB_69 : [75]\n",
      "NDB_70 : [76]\n",
      "NDB_71 : [77]\n",
      "NDB_72 : [78]\n",
      "NDB_73 : [79]\n",
      "NDB_74 : [80]\n",
      "NDB_75 : [81]\n",
      "NDB_76 : [82]\n",
      "NDB_77 : [83]\n",
      "NDB_78 : [84]\n",
      "NDB_79 : [85]\n",
      "NDB_80 : [86]\n",
      "NDB_81 : [87]\n",
      "NDB_82 : [88]\n",
      "NDB_83 : [89]\n",
      "NDB_84 : [90]\n",
      "NDB_85 : [91]\n",
      "NDB_86 : [92]\n",
      "NDB_87 : [93]\n",
      "NDB_88 : [94]\n",
      "NDB_89 : [95]\n",
      "NDB_90 : [96]\n",
      "NDB_91 : [97]\n",
      "NDB_92 : [98]\n",
      "NDB_93 : [99]\n",
      "NDB_94 : [100]\n",
      "NDB_95 : [367]\n",
      "NDB_96 : [368]\n",
      "NDB_97 : [369]\n",
      "NDB_98 : [370]\n",
      "NDB_99 : [371]\n",
      "NDB_100 : [372]\n",
      "NDB_101 : [373]\n",
      "NDB_102 : [374]\n",
      "NDB_103 : [375]\n",
      "NDB_104 : [376]\n",
      "NDB_105 : [377]\n",
      "NDB_106 : [378]\n",
      "NDB_107 : [379]\n",
      "NDB_108 : [380]\n",
      "NDB_109 : [381]\n",
      "NDB_110 : [382]\n",
      "NDB_111 : [383]\n",
      "NDB_112 : [384]\n",
      "NDB_113 : [385]\n",
      "NDB_114 : [386]\n",
      "NDB_115 : [387]\n",
      "NDB_116 : [388]\n",
      "NDB_117 : [389]\n",
      "NDB_118 : [390]\n",
      "NDB_119 : [391]\n",
      "NDB_120 : [392]\n",
      "NDB_121 : [393]\n",
      "NDB_122 : [394]\n",
      "NDB_123 : [395]\n",
      "NDB_124 : [396]\n",
      "NDB_125 : [397]\n",
      "NDB_126 : [398]\n",
      "NDB_127 : [399]\n",
      "NDB_128 : [400]\n",
      "NDB_129 : [401]\n",
      "NDB_130 : [402]\n",
      "NDB_131 : [403]\n",
      "NDB_132 : [404]\n",
      "NDB_133 : [405]\n",
      "NDB_134 : [406]\n",
      "NDB_135 : [407]\n",
      "NDB_136 : [408]\n",
      "NDB_137 : [409]\n",
      "NDB_138 : [410]\n",
      "NDB_139 : [411]\n",
      "NDB_140 : [412]\n",
      "NDB_141 : [413]\n",
      "NDB_142 : [414]\n",
      "NDB_143 : [415]\n",
      "NDB_144 : [416]\n",
      "NDB_145 : [417]\n",
      "NDB_146 : [418]\n",
      "NDB_147 : [419]\n",
      "NDB_148 : [420]\n",
      "NDB_149 : [421]\n",
      "NDB_150 : [422]\n",
      "NDB_151 : [423]\n",
      "NDB_152 : [424]\n",
      "NDB_153 : [425]\n",
      "NDB_154 : [426]\n",
      "Stv_3 : [41, 367]\n",
      "Stv_4 : [42, 368]\n",
      "Stv_5 : [43, 369]\n",
      "Stv_6 : [44, 370]\n",
      "Stv_7 : [45, 371]\n",
      "Stv_8 : [46, 372]\n",
      "Stv_9 : [47, 373]\n",
      "Stv_10 : [48, 374]\n",
      "Stv_11 : [49, 375]\n",
      "Stv_12 : [50, 376]\n",
      "Stv_13 : [51, 377]\n",
      "Stv_14 : [52, 378]\n",
      "Stv_16 : [53, 379]\n",
      "Stv_19 : [54, 380]\n",
      "Stv_20 : [55, 381]\n",
      "Stv_21 : [56, 382]\n",
      "Stv_22 : [57, 383]\n",
      "Stv_23 : [58, 384]\n",
      "Stv_25 : [59, 385]\n",
      "Stv_26 : [60, 386]\n",
      "Stv_27 : [61, 387]\n",
      "Stv_28 : [62, 388]\n",
      "Stv_29 : [63, 389]\n",
      "Stv_30 : [64, 390]\n",
      "Stv_31 : [65, 391]\n",
      "Stv_32 : [66, 392]\n",
      "Stv_33 : [67, 393]\n",
      "Stv_35 : [68, 394]\n",
      "Stv_36 : [69, 395]\n",
      "Stv_37 : [70, 396]\n",
      "Stv_39 : [71, 397]\n",
      "Stv_40 : [72, 398]\n",
      "Stv_42 : [73, 399]\n",
      "Stv_44 : [74, 400]\n",
      "Stv_45 : [75, 401]\n",
      "Stv_46 : [76, 402]\n",
      "Stv_48 : [77, 403]\n",
      "Stv_50 : [78, 404]\n",
      "Stv_53 : [79, 405]\n",
      "Stv_54 : [80, 406]\n",
      "Stv_59 : [81, 407]\n",
      "Stv_60 : [82, 408]\n",
      "Stv_61 : [83, 409]\n",
      "Stv_63 : [84, 410]\n",
      "Stv_64 : [85, 411]\n",
      "Stv_65 : [86, 412]\n",
      "Stv_86 : [87, 413]\n",
      "Stv_87 : [88, 414]\n",
      "Stv_88 : [89, 415]\n",
      "Stv_90 : [90, 416]\n",
      "Stv_91 : [91, 417]\n",
      "Stv_92 : [92, 418]\n",
      "Stv_94 : [93, 419]\n",
      "Stv_95 : [94, 420]\n",
      "Stv_99 : [95, 421]\n",
      "Stv_100 : [96, 422]\n",
      "Stv_101 : [97, 423]\n",
      "Stv_104 : [98, 424]\n",
      "Stv_105 : [99, 425]\n",
      "Stv_106 : [100, 426]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "- 5.Calculating barcode to color dictionary.\n",
      "Stv_3 : [2]\n",
      "Stv_4 : [0]\n",
      "Stv_5 : [1]\n",
      "Stv_6 : [2]\n",
      "Stv_7 : [0]\n",
      "Stv_8 : [1]\n",
      "Stv_9 : [2]\n",
      "Stv_10 : [0]\n",
      "Stv_11 : [1]\n",
      "Stv_12 : [2]\n",
      "Stv_13 : [0]\n",
      "Stv_14 : [1]\n",
      "Stv_16 : [2]\n",
      "Stv_19 : [0]\n",
      "Stv_20 : [1]\n",
      "Stv_21 : [2]\n",
      "Stv_22 : [0]\n",
      "Stv_23 : [1]\n",
      "Stv_25 : [2]\n",
      "Stv_26 : [0]\n",
      "Stv_27 : [1]\n",
      "Stv_28 : [2]\n",
      "Stv_29 : [0]\n",
      "Stv_30 : [1]\n",
      "Stv_31 : [2]\n",
      "Stv_32 : [0]\n",
      "Stv_33 : [1]\n",
      "Stv_35 : [2]\n",
      "Stv_36 : [0]\n",
      "Stv_37 : [1]\n",
      "Stv_39 : [2]\n",
      "Stv_40 : [0]\n",
      "Stv_42 : [1]\n",
      "Stv_44 : [2]\n",
      "Stv_45 : [0]\n",
      "Stv_46 : [1]\n",
      "Stv_48 : [2]\n",
      "Stv_50 : [0]\n",
      "Stv_53 : [1]\n",
      "Stv_54 : [2]\n",
      "Stv_59 : [0]\n",
      "Stv_60 : [1]\n",
      "Stv_61 : [2]\n",
      "Stv_63 : [0]\n",
      "Stv_64 : [1]\n",
      "Stv_65 : [2]\n",
      "Stv_86 : [0]\n",
      "Stv_87 : [1]\n",
      "Stv_88 : [2]\n",
      "Stv_90 : [0]\n",
      "Stv_91 : [1]\n",
      "Stv_92 : [2]\n",
      "Stv_94 : [0]\n",
      "Stv_95 : [1]\n",
      "Stv_99 : [2]\n",
      "Stv_100 : [0]\n",
      "Stv_101 : [1]\n",
      "Stv_104 : [2]\n",
      "Stv_105 : [0]\n",
      "Stv_106 : [1]\n",
      "\n",
      "- 6.Constructing internal khmer map\n",
      "\n",
      "- 7.Passing if counting barcode appearance times in probes True\n",
      "-- Max_genome_hits is: 200, this seq got hits: 579\n",
      "-- Max_genome_hits is: 200, this seq got hits: 278\n",
      "-- Max_genome_hits is: 200, this seq got hits: 237\n",
      "-- Max_genome_hits is: 200, this seq got hits: 260\n",
      "-- Max_genome_hits is: 200, this seq got hits: 6626\n",
      "-- Max_genome_hits is: 200, this seq got hits: 220\n",
      "-- Max_genome_hits is: 200, this seq got hits: 942\n",
      "-- Max_genome_hits is: 200, this seq got hits: 234\n",
      "-- Max_genome_hits is: 200, this seq got hits: 487\n",
      "-- Max_genome_hits is: 200, this seq got hits: 251\n",
      "-- Max_genome_hits is: 200, this seq got hits: 208\n",
      "-- Max_genome_hits is: 200, this seq got hits: 229\n",
      "-- Max_genome_hits is: 200, this seq got hits: 237\n",
      "-- Max_genome_hits is: 200, this seq got hits: 3172\n",
      "-- Max_genome_hits is: 200, this seq got hits: 362\n",
      "-- Max_genome_hits is: 200, this seq got hits: 355\n",
      "-- Max_genome_hits is: 200, this seq got hits: 410\n",
      "-- Max_genome_hits is: 200, this seq got hits: 451\n",
      "-- Max_genome_hits is: 200, this seq got hits: 206\n",
      "-- Max_genome_hits is: 200, this seq got hits: 237\n",
      "-- Max_genome_hits is: 200, this seq got hits: 206\n",
      "-- Max_genome_hits is: 200, this seq got hits: 215\n",
      "-- Max_genome_hits is: 200, this seq got hits: 207\n",
      "-- Max_genome_hits is: 200, this seq got hits: 204\n",
      "-- Max_genome_hits is: 200, this seq got hits: 247\n",
      "-- Max_genome_hits is: 200, this seq got hits: 303\n",
      "-- Max_genome_hits is: 200, this seq got hits: 261\n",
      "-- Max_genome_hits is: 200, this seq got hits: 3432\n",
      "-- Max_genome_hits is: 200, this seq got hits: 588\n",
      "-- Max_genome_hits is: 200, this seq got hits: 2538\n",
      "-- Max_genome_hits is: 200, this seq got hits: 265\n",
      "-- Max_genome_hits is: 200, this seq got hits: 475\n",
      "-- Max_genome_hits is: 200, this seq got hits: 213\n",
      "-- Max_genome_hits is: 200, this seq got hits: 312\n",
      "-- Max_genome_hits is: 200, this seq got hits: 12479\n",
      "-- Max_genome_hits is: 200, this seq got hits: 302\n",
      "-- Max_genome_hits is: 200, this seq got hits: 5428\n",
      "-- Max_genome_hits is: 200, this seq got hits: 217\n",
      "-- Max_genome_hits is: 200, this seq got hits: 216\n",
      "-- Max_genome_hits is: 200, this seq got hits: 279\n",
      "-- Max_genome_hits is: 200, this seq got hits: 632\n",
      "-- Max_genome_hits is: 200, this seq got hits: 600\n",
      "-- Max_genome_hits is: 200, this seq got hits: 205\n",
      "-- Max_genome_hits is: 200, this seq got hits: 412\n",
      "-- Max_genome_hits is: 200, this seq got hits: 274\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1471\n",
      "-- Max_genome_hits is: 200, this seq got hits: 724\n",
      "-- Max_genome_hits is: 200, this seq got hits: 366\n",
      "-- Max_genome_hits is: 200, this seq got hits: 204\n",
      "-- Max_genome_hits is: 200, this seq got hits: 9001\n",
      "-- Max_genome_hits is: 200, this seq got hits: 228\n",
      "-- Max_genome_hits is: 200, this seq got hits: 203\n",
      "-- Max_genome_hits is: 200, this seq got hits: 659\n",
      "-- Max_genome_hits is: 200, this seq got hits: 341\n",
      "-- Max_genome_hits is: 200, this seq got hits: 597\n",
      "-- Max_genome_hits is: 200, this seq got hits: 302\n",
      "-- Max_genome_hits is: 200, this seq got hits: 279\n",
      "-- Max_genome_hits is: 200, this seq got hits: 226\n",
      "-- Max_genome_hits is: 200, this seq got hits: 430\n",
      "-- Max_genome_hits is: 200, this seq got hits: 459\n",
      "-- Max_genome_hits is: 200, this seq got hits: 358\n",
      "-- Max_genome_hits is: 200, this seq got hits: 577\n",
      "-- Max_genome_hits is: 200, this seq got hits: 548\n",
      "-- Max_genome_hits is: 200, this seq got hits: 316\n",
      "-- Max_genome_hits is: 200, this seq got hits: 352\n",
      "-- Max_genome_hits is: 200, this seq got hits: 354\n",
      "-- Max_genome_hits is: 200, this seq got hits: 214\n",
      "-- Max_genome_hits is: 200, this seq got hits: 258\n",
      "-- Max_genome_hits is: 200, this seq got hits: 572\n",
      "-- Max_genome_hits is: 200, this seq got hits: 201\n",
      "-- Max_genome_hits is: 200, this seq got hits: 228\n",
      "-- Max_genome_hits is: 200, this seq got hits: 590\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1284\n",
      "-- Max_genome_hits is: 200, this seq got hits: 283\n",
      "-- Max_genome_hits is: 200, this seq got hits: 251\n",
      "-- Max_genome_hits is: 200, this seq got hits: 589\n",
      "-- Max_genome_hits is: 200, this seq got hits: 241\n",
      "-- Max_genome_hits is: 200, this seq got hits: 671\n",
      "-- Max_genome_hits is: 200, this seq got hits: 347\n",
      "-- Max_genome_hits is: 200, this seq got hits: 209\n",
      "-- Max_genome_hits is: 200, this seq got hits: 236\n",
      "-- Max_genome_hits is: 200, this seq got hits: 455\n",
      "-- Max_genome_hits is: 200, this seq got hits: 514\n",
      "-- Max_genome_hits is: 200, this seq got hits: 234\n",
      "-- Max_genome_hits is: 200, this seq got hits: 228\n",
      "-- Max_genome_hits is: 200, this seq got hits: 210\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1286\n",
      "-- Max_genome_hits is: 200, this seq got hits: 352\n",
      "-- Max_genome_hits is: 200, this seq got hits: 657\n",
      "-- Max_genome_hits is: 200, this seq got hits: 227\n",
      "-- Max_genome_hits is: 200, this seq got hits: 201\n",
      "-- Max_genome_hits is: 200, this seq got hits: 282\n",
      "-- Max_genome_hits is: 200, this seq got hits: 535\n",
      "-- Max_genome_hits is: 200, this seq got hits: 242\n",
      "-- Max_genome_hits is: 200, this seq got hits: 214\n",
      "-- Max_genome_hits is: 200, this seq got hits: 292\n",
      "-- Max_genome_hits is: 200, this seq got hits: 201\n",
      "-- Max_genome_hits is: 200, this seq got hits: 349\n",
      "-- Max_genome_hits is: 200, this seq got hits: 286\n",
      "-- Max_genome_hits is: 200, this seq got hits: 207\n",
      "-- Max_genome_hits is: 200, this seq got hits: 319\n",
      "-- Max_genome_hits is: 200, this seq got hits: 300\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1903\n",
      "-- Max_genome_hits is: 200, this seq got hits: 227\n",
      "-- Max_genome_hits is: 200, this seq got hits: 302\n",
      "-- Max_genome_hits is: 200, this seq got hits: 333\n",
      "-- Max_genome_hits is: 200, this seq got hits: 249\n",
      "-- Max_genome_hits is: 200, this seq got hits: 748\n",
      "-- Max_genome_hits is: 200, this seq got hits: 253\n",
      "-- Max_genome_hits is: 200, this seq got hits: 333\n",
      "-- Max_genome_hits is: 200, this seq got hits: 246\n",
      "-- Max_genome_hits is: 200, this seq got hits: 550\n",
      "-- Max_genome_hits is: 200, this seq got hits: 313\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1007\n",
      "-- Max_genome_hits is: 200, this seq got hits: 205\n",
      "-- Max_genome_hits is: 200, this seq got hits: 360\n",
      "-- Max_genome_hits is: 200, this seq got hits: 238\n",
      "-- Max_genome_hits is: 200, this seq got hits: 245\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1029\n",
      "-- Max_genome_hits is: 200, this seq got hits: 203\n",
      "-- Max_genome_hits is: 200, this seq got hits: 233\n",
      "-- Max_genome_hits is: 200, this seq got hits: 316\n",
      "-- Max_genome_hits is: 200, this seq got hits: 269\n",
      "-- Max_genome_hits is: 200, this seq got hits: 295\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1025\n",
      "-- Max_genome_hits is: 200, this seq got hits: 348\n",
      "-- Max_genome_hits is: 200, this seq got hits: 266\n",
      "-- Max_genome_hits is: 200, this seq got hits: 340\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1694\n",
      "-- Max_genome_hits is: 200, this seq got hits: 221\n",
      "-- Max_genome_hits is: 200, this seq got hits: 246\n",
      "-- Max_genome_hits is: 200, this seq got hits: 275\n",
      "-- Max_genome_hits is: 200, this seq got hits: 442\n",
      "-- Max_genome_hits is: 200, this seq got hits: 329\n",
      "-- Max_genome_hits is: 200, this seq got hits: 3147\n",
      "-- Max_genome_hits is: 200, this seq got hits: 232\n",
      "-- Max_genome_hits is: 200, this seq got hits: 283\n",
      "-- Max_genome_hits is: 200, this seq got hits: 258\n",
      "-- Max_genome_hits is: 200, this seq got hits: 206\n",
      "-- Max_genome_hits is: 200, this seq got hits: 250\n",
      "-- Max_genome_hits is: 200, this seq got hits: 510\n",
      "-- Max_genome_hits is: 200, this seq got hits: 287\n",
      "-- Max_genome_hits is: 200, this seq got hits: 220\n",
      "-- Max_genome_hits is: 200, this seq got hits: 969\n",
      "-- Max_genome_hits is: 200, this seq got hits: 384\n",
      "-- Max_genome_hits is: 200, this seq got hits: 464\n",
      "-- Max_genome_hits is: 200, this seq got hits: 344\n",
      "-- Max_genome_hits is: 200, this seq got hits: 530\n",
      "-- Max_genome_hits is: 200, this seq got hits: 616\n",
      "-- Max_genome_hits is: 200, this seq got hits: 333\n",
      "-- Max_genome_hits is: 200, this seq got hits: 423\n",
      "-- Max_genome_hits is: 200, this seq got hits: 340\n",
      "-- Max_genome_hits is: 200, this seq got hits: 708\n",
      "-- Max_genome_hits is: 200, this seq got hits: 208\n",
      "-- Max_genome_hits is: 200, this seq got hits: 336\n",
      "-- Max_genome_hits is: 200, this seq got hits: 457\n",
      "-- Max_genome_hits is: 200, this seq got hits: 293\n",
      "-- Max_genome_hits is: 200, this seq got hits: 2001\n",
      "-- Max_genome_hits is: 200, this seq got hits: 310\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- Max_genome_hits is: 200, this seq got hits: 1101\n",
      "-- Max_genome_hits is: 200, this seq got hits: 234\n",
      "-- Max_genome_hits is: 200, this seq got hits: 837\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1639\n",
      "-- Max_genome_hits is: 200, this seq got hits: 228\n",
      "-- Max_genome_hits is: 200, this seq got hits: 454\n",
      "-- Max_genome_hits is: 200, this seq got hits: 213\n",
      "-- Max_genome_hits is: 200, this seq got hits: 205\n",
      "-- Max_genome_hits is: 200, this seq got hits: 321\n",
      "-- Max_genome_hits is: 200, this seq got hits: 254\n",
      "-- Max_genome_hits is: 200, this seq got hits: 295\n",
      "-- Max_genome_hits is: 200, this seq got hits: 227\n",
      "-- Max_genome_hits is: 200, this seq got hits: 206\n",
      "-- Max_genome_hits is: 200, this seq got hits: 280\n",
      "-- Max_genome_hits is: 200, this seq got hits: 246\n",
      "-- Max_genome_hits is: 200, this seq got hits: 381\n",
      "-- Max_genome_hits is: 200, this seq got hits: 685\n",
      "-- Max_genome_hits is: 200, this seq got hits: 306\n",
      "-- Max_genome_hits is: 200, this seq got hits: 254\n",
      "-- Max_genome_hits is: 200, this seq got hits: 262\n",
      "-- Max_genome_hits is: 200, this seq got hits: 504\n",
      "-- Max_genome_hits is: 200, this seq got hits: 895\n",
      "-- Max_genome_hits is: 200, this seq got hits: 298\n",
      "-- Max_genome_hits is: 200, this seq got hits: 243\n",
      "-- Max_genome_hits is: 200, this seq got hits: 738\n",
      "-- Max_genome_hits is: 200, this seq got hits: 374\n",
      "-- Max_genome_hits is: 200, this seq got hits: 269\n",
      "-- Max_genome_hits is: 200, this seq got hits: 641\n",
      "-- Max_genome_hits is: 200, this seq got hits: 213\n",
      "-- Max_genome_hits is: 200, this seq got hits: 906\n",
      "-- Max_genome_hits is: 200, this seq got hits: 355\n",
      "-- Max_genome_hits is: 200, this seq got hits: 224\n",
      "-- Max_genome_hits is: 200, this seq got hits: 357\n",
      "-- Max_genome_hits is: 200, this seq got hits: 439\n",
      "-- Max_genome_hits is: 200, this seq got hits: 570\n",
      "-- Max_genome_hits is: 200, this seq got hits: 305\n",
      "-- Max_genome_hits is: 200, this seq got hits: 855\n",
      "-- Max_genome_hits is: 200, this seq got hits: 500\n",
      "-- Max_genome_hits is: 200, this seq got hits: 201\n",
      "-- Max_genome_hits is: 200, this seq got hits: 268\n",
      "-- Max_genome_hits is: 200, this seq got hits: 238\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1092\n",
      "-- Max_genome_hits is: 200, this seq got hits: 823\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1092\n",
      "-- Max_genome_hits is: 200, this seq got hits: 292\n",
      "-- Max_genome_hits is: 200, this seq got hits: 206\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1624\n",
      "-- Max_genome_hits is: 200, this seq got hits: 209\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1186\n",
      "-- Max_genome_hits is: 200, this seq got hits: 212\n",
      "-- Max_genome_hits is: 200, this seq got hits: 561\n",
      "-- Max_genome_hits is: 200, this seq got hits: 341\n",
      "-- Max_genome_hits is: 200, this seq got hits: 291\n",
      "-- Max_genome_hits is: 200, this seq got hits: 304\n",
      "-- Max_genome_hits is: 200, this seq got hits: 226\n",
      "-- Max_genome_hits is: 200, this seq got hits: 222\n",
      "-- Max_genome_hits is: 200, this seq got hits: 53676\n",
      "-- Max_genome_hits is: 200, this seq got hits: 253\n",
      "-- Max_genome_hits is: 200, this seq got hits: 226\n",
      "-- Max_genome_hits is: 200, this seq got hits: 294\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1523\n",
      "-- Max_genome_hits is: 200, this seq got hits: 226\n",
      "-- Max_genome_hits is: 200, this seq got hits: 401\n",
      "-- Max_genome_hits is: 200, this seq got hits: 280\n",
      "-- Max_genome_hits is: 200, this seq got hits: 251\n",
      "-- Max_genome_hits is: 200, this seq got hits: 318\n",
      "-- Max_genome_hits is: 200, this seq got hits: 205\n",
      "-- Max_genome_hits is: 200, this seq got hits: 696\n",
      "-- Max_genome_hits is: 200, this seq got hits: 380\n",
      "-- Max_genome_hits is: 200, this seq got hits: 237\n",
      "-- Max_genome_hits is: 200, this seq got hits: 428\n",
      "-- Max_genome_hits is: 200, this seq got hits: 282\n",
      "-- Max_genome_hits is: 200, this seq got hits: 255\n",
      "-- Max_genome_hits is: 200, this seq got hits: 254\n",
      "-- Max_genome_hits is: 200, this seq got hits: 291\n",
      "-- Max_genome_hits is: 200, this seq got hits: 265\n",
      "-- Max_genome_hits is: 200, this seq got hits: 254\n",
      "-- Max_genome_hits is: 200, this seq got hits: 643\n",
      "-- Max_genome_hits is: 200, this seq got hits: 499\n",
      "-- Max_genome_hits is: 200, this seq got hits: 206\n",
      "-- Max_genome_hits is: 200, this seq got hits: 331\n",
      "-- Max_genome_hits is: 200, this seq got hits: 335\n",
      "-- Max_genome_hits is: 200, this seq got hits: 594\n",
      "-- Max_genome_hits is: 200, this seq got hits: 445\n",
      "-- Max_genome_hits is: 200, this seq got hits: 290\n",
      "-- Max_genome_hits is: 200, this seq got hits: 487\n",
      "-- Max_genome_hits is: 200, this seq got hits: 930\n",
      "-- Max_genome_hits is: 200, this seq got hits: 354\n",
      "-- Max_genome_hits is: 200, this seq got hits: 311\n",
      "-- Max_genome_hits is: 200, this seq got hits: 449\n",
      "-- Max_genome_hits is: 200, this seq got hits: 553\n",
      "-- Max_genome_hits is: 200, this seq got hits: 430\n",
      "-- Max_genome_hits is: 200, this seq got hits: 438\n",
      "-- Max_genome_hits is: 200, this seq got hits: 302\n",
      "-- Max_genome_hits is: 200, this seq got hits: 421\n",
      "-- Max_genome_hits is: 200, this seq got hits: 208\n",
      "-- Max_genome_hits is: 200, this seq got hits: 355\n",
      "-- Max_genome_hits is: 200, this seq got hits: 326\n",
      "-- Max_genome_hits is: 200, this seq got hits: 211\n",
      "-- Max_genome_hits is: 200, this seq got hits: 323\n",
      "-- Max_genome_hits is: 200, this seq got hits: 202\n",
      "-- Max_genome_hits is: 200, this seq got hits: 204\n",
      "-- Max_genome_hits is: 200, this seq got hits: 225\n",
      "-- Max_genome_hits is: 200, this seq got hits: 228\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1352\n",
      "-- Max_genome_hits is: 200, this seq got hits: 207\n",
      "-- Max_genome_hits is: 200, this seq got hits: 416\n",
      "-- Max_genome_hits is: 200, this seq got hits: 215\n",
      "-- Max_genome_hits is: 200, this seq got hits: 502\n",
      "-- Max_genome_hits is: 200, this seq got hits: 252\n",
      "-- Max_genome_hits is: 200, this seq got hits: 267\n",
      "-- Max_genome_hits is: 200, this seq got hits: 246\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1233\n",
      "-- Max_genome_hits is: 200, this seq got hits: 382\n",
      "-- Max_genome_hits is: 200, this seq got hits: 529\n",
      "-- Max_genome_hits is: 200, this seq got hits: 270\n",
      "-- Max_genome_hits is: 200, this seq got hits: 697\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1693\n",
      "-- Max_genome_hits is: 200, this seq got hits: 276\n",
      "-- Max_genome_hits is: 200, this seq got hits: 213\n",
      "-- Max_genome_hits is: 200, this seq got hits: 228\n",
      "-- Max_genome_hits is: 200, this seq got hits: 328\n",
      "-- Max_genome_hits is: 200, this seq got hits: 206\n",
      "-- Max_genome_hits is: 200, this seq got hits: 404\n",
      "-- Max_genome_hits is: 200, this seq got hits: 213\n",
      "-- Max_genome_hits is: 200, this seq got hits: 276\n",
      "-- Max_genome_hits is: 200, this seq got hits: 211\n",
      "-- Max_genome_hits is: 200, this seq got hits: 329\n",
      "-- Max_genome_hits is: 200, this seq got hits: 217\n",
      "-- Max_genome_hits is: 200, this seq got hits: 604\n",
      "-- Max_genome_hits is: 200, this seq got hits: 743\n",
      "-- Max_genome_hits is: 200, this seq got hits: 243\n",
      "-- Max_genome_hits is: 200, this seq got hits: 211\n",
      "-- Max_genome_hits is: 200, this seq got hits: 246\n",
      "-- Max_genome_hits is: 200, this seq got hits: 272\n",
      "-- Max_genome_hits is: 200, this seq got hits: 208\n",
      "-- Max_genome_hits is: 200, this seq got hits: 289\n",
      "-- Max_genome_hits is: 200, this seq got hits: 448\n",
      "-- Max_genome_hits is: 200, this seq got hits: 289\n",
      "-- Max_genome_hits is: 200, this seq got hits: 318\n",
      "-- Max_genome_hits is: 200, this seq got hits: 206\n",
      "-- Max_genome_hits is: 200, this seq got hits: 334\n",
      "-- Max_genome_hits is: 200, this seq got hits: 268\n",
      "-- Max_genome_hits is: 200, this seq got hits: 946\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1892\n",
      "-- Max_genome_hits is: 200, this seq got hits: 10393\n",
      "-- Max_genome_hits is: 200, this seq got hits: 8939\n",
      "-- Max_genome_hits is: 200, this seq got hits: 643\n",
      "-- Max_genome_hits is: 200, this seq got hits: 305\n",
      "-- Max_genome_hits is: 200, this seq got hits: 306\n",
      "-- Max_genome_hits is: 200, this seq got hits: 288\n",
      "-- Max_genome_hits is: 200, this seq got hits: 285\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1129\n",
      "-- Max_genome_hits is: 200, this seq got hits: 425\n",
      "-- Max_genome_hits is: 200, this seq got hits: 785\n",
      "-- Max_genome_hits is: 200, this seq got hits: 615\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1305\n",
      "-- Max_genome_hits is: 200, this seq got hits: 686\n",
      "-- Max_genome_hits is: 200, this seq got hits: 366\n",
      "-- Max_genome_hits is: 200, this seq got hits: 992\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1860\n",
      "-- Max_genome_hits is: 200, this seq got hits: 497\n",
      "-- Max_genome_hits is: 200, this seq got hits: 329\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1442\n",
      "-- Max_genome_hits is: 200, this seq got hits: 259\n",
      "-- Max_genome_hits is: 200, this seq got hits: 660\n",
      "-- Max_genome_hits is: 200, this seq got hits: 317\n",
      "-- Max_genome_hits is: 200, this seq got hits: 337\n",
      "-- Max_genome_hits is: 200, this seq got hits: 221\n",
      "-- Max_genome_hits is: 200, this seq got hits: 598\n",
      "-- Max_genome_hits is: 200, this seq got hits: 211\n",
      "-- Max_genome_hits is: 200, this seq got hits: 617\n",
      "-- Max_genome_hits is: 200, this seq got hits: 289\n",
      "-- Max_genome_hits is: 200, this seq got hits: 245\n",
      "-- Max_genome_hits is: 200, this seq got hits: 300\n",
      "-- Max_genome_hits is: 200, this seq got hits: 441\n",
      "-- Max_genome_hits is: 200, this seq got hits: 674\n",
      "-- Max_genome_hits is: 200, this seq got hits: 205\n",
      "-- Max_genome_hits is: 200, this seq got hits: 251\n",
      "-- Max_genome_hits is: 200, this seq got hits: 361\n",
      "-- Max_genome_hits is: 200, this seq got hits: 269\n",
      "-- Max_genome_hits is: 200, this seq got hits: 510\n",
      "-- Max_genome_hits is: 200, this seq got hits: 640\n",
      "-- Max_genome_hits is: 200, this seq got hits: 440\n",
      "-- Max_genome_hits is: 200, this seq got hits: 201\n",
      "-- Max_genome_hits is: 200, this seq got hits: 3172\n",
      "-- Max_genome_hits is: 200, this seq got hits: 251\n",
      "-- Max_genome_hits is: 200, this seq got hits: 432\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1046\n",
      "-- Max_genome_hits is: 200, this seq got hits: 246\n",
      "-- Max_genome_hits is: 200, this seq got hits: 435\n",
      "-- Max_genome_hits is: 200, this seq got hits: 234\n",
      "-- Max_genome_hits is: 200, this seq got hits: 206\n",
      "-- Max_genome_hits is: 200, this seq got hits: 224\n",
      "-- Max_genome_hits is: 200, this seq got hits: 363\n",
      "-- Max_genome_hits is: 200, this seq got hits: 307\n",
      "-- Max_genome_hits is: 200, this seq got hits: 358\n",
      "-- Max_genome_hits is: 200, this seq got hits: 371\n",
      "-- Max_genome_hits is: 200, this seq got hits: 287\n",
      "-- Max_genome_hits is: 200, this seq got hits: 206\n",
      "-- Max_genome_hits is: 200, this seq got hits: 225\n",
      "-- Max_genome_hits is: 200, this seq got hits: 304\n",
      "-- Max_genome_hits is: 200, this seq got hits: 228\n",
      "-- Max_genome_hits is: 200, this seq got hits: 247\n",
      "-- Max_genome_hits is: 200, this seq got hits: 215\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- Max_genome_hits is: 200, this seq got hits: 252\n",
      "-- Max_genome_hits is: 200, this seq got hits: 281\n",
      "-- Max_genome_hits is: 200, this seq got hits: 282\n",
      "-- Max_genome_hits is: 200, this seq got hits: 232\n",
      "-- Max_genome_hits is: 200, this seq got hits: 682\n",
      "-- Max_genome_hits is: 200, this seq got hits: 321\n",
      "-- Max_genome_hits is: 200, this seq got hits: 271\n",
      "-- Max_genome_hits is: 200, this seq got hits: 395\n",
      "-- Max_genome_hits is: 200, this seq got hits: 468\n",
      "-- Max_genome_hits is: 200, this seq got hits: 543\n",
      "-- Max_genome_hits is: 200, this seq got hits: 364\n",
      "-- Max_genome_hits is: 200, this seq got hits: 207\n",
      "-- Max_genome_hits is: 200, this seq got hits: 224\n",
      "-- Max_genome_hits is: 200, this seq got hits: 293\n",
      "-- Max_genome_hits is: 200, this seq got hits: 488\n",
      "-- Max_genome_hits is: 200, this seq got hits: 653\n",
      "-- Max_genome_hits is: 200, this seq got hits: 460\n",
      "-- Max_genome_hits is: 200, this seq got hits: 5042\n",
      "-- Max_genome_hits is: 200, this seq got hits: 433\n",
      "-- Max_genome_hits is: 200, this seq got hits: 444\n",
      "-- Max_genome_hits is: 200, this seq got hits: 235\n",
      "-- Max_genome_hits is: 200, this seq got hits: 968\n",
      "-- Max_genome_hits is: 200, this seq got hits: 206\n",
      "-- Max_genome_hits is: 200, this seq got hits: 202\n",
      "-- Max_genome_hits is: 200, this seq got hits: 283\n",
      "-- Max_genome_hits is: 200, this seq got hits: 487\n",
      "-- Max_genome_hits is: 200, this seq got hits: 997\n",
      "-- Max_genome_hits is: 200, this seq got hits: 299\n",
      "-- Max_genome_hits is: 200, this seq got hits: 557\n",
      "-- Max_genome_hits is: 200, this seq got hits: 595\n",
      "-- Max_genome_hits is: 200, this seq got hits: 296\n",
      "-- Max_genome_hits is: 200, this seq got hits: 257\n",
      "-- Max_genome_hits is: 200, this seq got hits: 408\n",
      "-- Max_genome_hits is: 200, this seq got hits: 232\n",
      "-- Max_genome_hits is: 200, this seq got hits: 218\n",
      "-- Max_genome_hits is: 200, this seq got hits: 244\n",
      "-- Max_genome_hits is: 200, this seq got hits: 209\n",
      "-- Max_genome_hits is: 200, this seq got hits: 201\n",
      "-- Max_genome_hits is: 200, this seq got hits: 404\n",
      "-- Max_genome_hits is: 200, this seq got hits: 6946\n",
      "-- Max_genome_hits is: 200, this seq got hits: 224\n",
      "-- Max_genome_hits is: 200, this seq got hits: 320\n",
      "-- Max_genome_hits is: 200, this seq got hits: 448\n",
      "-- Max_genome_hits is: 200, this seq got hits: 365\n",
      "-- Max_genome_hits is: 200, this seq got hits: 531\n",
      "-- Max_genome_hits is: 200, this seq got hits: 265\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1436\n",
      "-- Max_genome_hits is: 200, this seq got hits: 315\n",
      "-- Max_genome_hits is: 200, this seq got hits: 232\n",
      "-- Max_genome_hits is: 200, this seq got hits: 688\n",
      "-- Max_genome_hits is: 200, this seq got hits: 328\n",
      "-- Max_genome_hits is: 200, this seq got hits: 215\n",
      "-- Max_genome_hits is: 200, this seq got hits: 388\n",
      "-- Max_genome_hits is: 200, this seq got hits: 217\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1096\n",
      "-- Max_genome_hits is: 200, this seq got hits: 268\n",
      "-- Max_genome_hits is: 200, this seq got hits: 211\n",
      "-- Max_genome_hits is: 200, this seq got hits: 414\n",
      "-- Max_genome_hits is: 200, this seq got hits: 224\n",
      "-- Max_genome_hits is: 200, this seq got hits: 304\n",
      "-- Max_genome_hits is: 200, this seq got hits: 858\n",
      "-- Max_genome_hits is: 200, this seq got hits: 220\n",
      "-- Max_genome_hits is: 200, this seq got hits: 238\n",
      "-- Max_genome_hits is: 200, this seq got hits: 373\n",
      "-- Max_genome_hits is: 200, this seq got hits: 232\n",
      "-- Max_genome_hits is: 200, this seq got hits: 714\n",
      "-- Max_genome_hits is: 200, this seq got hits: 215\n",
      "-- Max_genome_hits is: 200, this seq got hits: 23106\n",
      "-- Max_genome_hits is: 200, this seq got hits: 312\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1125\n",
      "-- Max_genome_hits is: 200, this seq got hits: 2021\n",
      "-- Max_genome_hits is: 200, this seq got hits: 829\n",
      "-- Max_genome_hits is: 200, this seq got hits: 301\n",
      "-- Max_genome_hits is: 200, this seq got hits: 655\n",
      "-- Max_genome_hits is: 200, this seq got hits: 277\n",
      "-- Max_genome_hits is: 200, this seq got hits: 579\n",
      "-- Max_genome_hits is: 200, this seq got hits: 333\n",
      "-- Max_genome_hits is: 200, this seq got hits: 294\n",
      "-- Max_genome_hits is: 200, this seq got hits: 384\n",
      "-- Max_genome_hits is: 200, this seq got hits: 209\n",
      "-- Max_genome_hits is: 200, this seq got hits: 304\n",
      "-- Max_genome_hits is: 200, this seq got hits: 224\n",
      "-- Max_genome_hits is: 200, this seq got hits: 257\n",
      "-- Max_genome_hits is: 200, this seq got hits: 298\n",
      "-- Max_genome_hits is: 200, this seq got hits: 611\n",
      "-- Max_genome_hits is: 200, this seq got hits: 327\n",
      "-- Max_genome_hits is: 200, this seq got hits: 231\n",
      "-- Max_genome_hits is: 200, this seq got hits: 241\n",
      "-- Max_genome_hits is: 200, this seq got hits: 210\n",
      "-- Max_genome_hits is: 200, this seq got hits: 246\n",
      "-- Max_genome_hits is: 200, this seq got hits: 269\n",
      "-- Max_genome_hits is: 200, this seq got hits: 263\n",
      "-- Max_genome_hits is: 200, this seq got hits: 219\n",
      "-- Max_genome_hits is: 200, this seq got hits: 425\n",
      "-- Max_genome_hits is: 200, this seq got hits: 13895\n",
      "-- Max_genome_hits is: 200, this seq got hits: 359\n",
      "-- Max_genome_hits is: 200, this seq got hits: 233\n",
      "-- Max_genome_hits is: 200, this seq got hits: 480\n",
      "-- Max_genome_hits is: 200, this seq got hits: 366\n",
      "-- Max_genome_hits is: 200, this seq got hits: 312\n",
      "-- Max_genome_hits is: 200, this seq got hits: 204\n",
      "-- Max_genome_hits is: 200, this seq got hits: 555\n",
      "-- Max_genome_hits is: 200, this seq got hits: 215\n",
      "-- Max_genome_hits is: 200, this seq got hits: 247\n",
      "-- Max_genome_hits is: 200, this seq got hits: 218\n",
      "-- Max_genome_hits is: 200, this seq got hits: 224\n",
      "-- Max_genome_hits is: 200, this seq got hits: 307\n",
      "-- Max_genome_hits is: 200, this seq got hits: 212\n",
      "-- Max_genome_hits is: 200, this seq got hits: 365\n",
      "-- Max_genome_hits is: 200, this seq got hits: 209\n",
      "-- Max_genome_hits is: 200, this seq got hits: 307\n",
      "-- Max_genome_hits is: 200, this seq got hits: 228\n",
      "-- Max_genome_hits is: 200, this seq got hits: 220\n",
      "-- Max_genome_hits is: 200, this seq got hits: 382\n",
      "-- Max_genome_hits is: 200, this seq got hits: 765\n",
      "-- Max_genome_hits is: 200, this seq got hits: 460\n",
      "\n",
      "- 8.Probes not passing through genome filter: 480\n",
      "--  re-check region size:\n",
      "41 : 500\n",
      "42 : 395\n",
      "43 : 500\n",
      "44 : 500\n",
      "45 : 500\n",
      "46 : 477\n",
      "47 : 500\n",
      "48 : 500\n",
      "49 : 500\n",
      "50 : 500\n",
      "51 : 500\n",
      "52 : 500\n",
      "53 : 500\n",
      "54 : 500\n",
      "55 : 500\n",
      "56 : 500\n",
      "57 : 500\n",
      "58 : 500\n",
      "59 : 500\n",
      "60 : 500\n",
      "61 : 495\n",
      "62 : 433\n",
      "63 : 499\n",
      "64 : 500\n",
      "65 : 500\n",
      "66 : 435\n",
      "67 : 412\n",
      "68 : 500\n",
      "69 : 475\n",
      "70 : 500\n",
      "71 : 500\n",
      "72 : 283\n",
      "73 : 202\n",
      "74 : 205\n",
      "75 : 500\n",
      "76 : 500\n",
      "77 : 500\n",
      "78 : 500\n",
      "79 : 500\n",
      "80 : 500\n",
      "81 : 494\n",
      "82 : 500\n",
      "83 : 500\n",
      "84 : 500\n",
      "85 : 500\n",
      "86 : 500\n",
      "87 : 500\n",
      "88 : 483\n",
      "89 : 500\n",
      "90 : 500\n",
      "91 : 500\n",
      "92 : 500\n",
      "93 : 500\n",
      "94 : 497\n",
      "95 : 488\n",
      "96 : 500\n",
      "97 : 424\n",
      "98 : 500\n",
      "99 : 500\n",
      "100 : 500\n",
      "367 : 500\n",
      "368 : 500\n",
      "369 : 500\n",
      "370 : 500\n",
      "371 : 500\n",
      "372 : 500\n",
      "373 : 410\n",
      "374 : 385\n",
      "375 : 500\n",
      "376 : 500\n",
      "377 : 500\n",
      "378 : 500\n",
      "379 : 500\n",
      "380 : 500\n",
      "381 : 500\n",
      "382 : 500\n",
      "383 : 500\n",
      "384 : 500\n",
      "385 : 500\n",
      "386 : 291\n",
      "387 : 213\n",
      "388 : 495\n",
      "389 : 500\n",
      "390 : 462\n",
      "391 : 500\n",
      "392 : 490\n",
      "393 : 500\n",
      "394 : 500\n",
      "395 : 500\n",
      "396 : 500\n",
      "397 : 500\n",
      "398 : 500\n",
      "399 : 500\n",
      "400 : 500\n",
      "401 : 500\n",
      "402 : 500\n",
      "403 : 500\n",
      "404 : 500\n",
      "405 : 480\n",
      "406 : 297\n",
      "407 : 259\n",
      "408 : 500\n",
      "409 : 500\n",
      "410 : 500\n",
      "411 : 500\n",
      "412 : 500\n",
      "413 : 500\n",
      "414 : 500\n",
      "415 : 500\n",
      "416 : 500\n",
      "417 : 500\n",
      "418 : 500\n",
      "419 : 500\n",
      "420 : 500\n",
      "421 : 500\n",
      "422 : 500\n",
      "423 : 500\n",
      "424 : 500\n",
      "425 : 500\n",
      "426 : 500\n",
      "--- total number of probes: 57479\n",
      "\n",
      "- 9.Saving probes to: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/CTP-02/chr21/final_probes/filtered_probes.fasta\n"
     ]
    }
   ],
   "source": [
    "kept_records, kept_size_dic = Check_Probes(pb_records, pb_lists, sub_encodings, master_dir, \n",
    "                                           total_bc=3, save_folder = pb_dir,\n",
    "                                        fwd_primer=fprimer, rev_primer=rprimer,\n",
    "                                        stv_barcodes=stv_barcodes, ndb_barcodes=ndb_barcodes)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "56999"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(kept_records)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 2",
   "language": "python",
   "name": "python2"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.13"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
