{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Library design for CTP02, \n",
    "\n",
    "## PCDH single molecule FISH check\n",
    "by Pu Zheng\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#minimum imports:\n",
    "import time,os,sys,glob\n",
    "import cPickle as pickle\n",
    "import numpy as np\n",
    "import khmer\n",
    "sys.path.append(r'/n/home13/pzheng/Documents/python-functions/python-functions-library')\n",
    "\n",
    "from LibraryConstruction import fastaread,fastawrite,fastacombine\n",
    "import LibraryDesigner as ld\n",
    "import LibraryConstruction as lc\n",
    "\n",
    "from Bio import SeqIO\n",
    "from Bio.Seq import Seq\n",
    "from Bio.Alphabet import IUPAC\n",
    "from Bio.SeqRecord import SeqRecord "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 1. Load previously designed probes, generate pb_records and pb_lists"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- Save pb_lists\n",
      "-- Save pb_records\n"
     ]
    }
   ],
   "source": [
    "# dir\n",
    "master_dir = r'/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/CTP-02/PCDH_smFISH';\n",
    "# input filename\n",
    "pb_filename = 'PCDH_mRNA.csv';\n",
    "\n",
    "pb_records, gene_pb_dic = [],{}; # initialize\n",
    "\n",
    "import csv\n",
    "with open(master_dir+os.sep+pb_filename,'rU') as handle:\n",
    "    _csvreader = csv.reader(handle, delimiter=',') # reader\n",
    "    # check if has header\n",
    "    _has_header = csv.Sniffer().has_header(handle.readline())\n",
    "    handle.seek(0)  # Rewind.\n",
    "    if _has_header:\n",
    "        next(_csvreader)  # Skip header row.\n",
    "    for row in _csvreader:\n",
    "        _name = row[0];\n",
    "        _seq = row[1].upper()\n",
    "        pb_records.append(SeqRecord(Seq(_seq,alphabet=IUPAC.unambiguous_dna),id=_name, name=_name,description=''))\n",
    "        #reg_id = int(name.split('reg_')[1].split(\"_\")[0])\n",
    "        gene_id = _name.split('gene_')[1].split(\"_\")[0]\n",
    "        pb_info = {'reg_index':gene_id, 'total_seq':_seq, 'total_name':_name};\n",
    "        if gene_id not in gene_pb_dic.keys():\n",
    "            gene_pb_dic[gene_id] = [pb_info];\n",
    "        else:\n",
    "            gene_pb_dic[gene_id].append(pb_info);\n",
    "# define pb_lists, match format\n",
    "pb_lists = gene_pb_dic.values()\n",
    "\n",
    "# save\n",
    "save_dir = 'final_probes'\n",
    "if not os.path.exists(master_dir+os.sep+save_dir):\n",
    "    os.makedirs(master_dir+os.sep+save_dir)\n",
    "print \"-- Save pb_lists\"\n",
    "pickle.dump(pb_lists, open(master_dir+os.sep+save_dir+os.sep+'list.pkl', 'w'));\n",
    "print \"-- Save pb_records\"\n",
    "with open(master_dir+os.sep+save_dir+os.sep+'candidate_probes.fasta', \"w\") as output_handle:\n",
    "    SeqIO.write(pb_records, output_handle, 'fasta');"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 2. Load other inforamtion for quality check"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 2.1 Read barcode Sequences"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Barcodes loaded: Stv: 201, NDB: 1052\n"
     ]
    }
   ],
   "source": [
    "# read all Stv barcodes\n",
    "barcode_dir = r'/n/boslfs/LABS/zhuang_lab/User/pzheng/Barcodes';\n",
    "\n",
    "#stv_adaptor = [1,2,17,62,77,78,79,80,81,82,83,84] # barcodes saved for adaptors\n",
    "#stv_bad = [34,38,41] # barcodes performed badly\n",
    "#stv_mask = stv_adaptor + stv_bad \n",
    "stv_mask = []\n",
    "\n",
    "with open(barcode_dir+os.sep+'top_Stvs.fasta', \"rU\") as handle:\n",
    "    stv_barcodes = [];\n",
    "    for record in SeqIO.parse(handle, \"fasta\"):\n",
    "        if int(record.id.split('_')[1]) not in stv_mask:\n",
    "            stv_barcodes.append(record);\n",
    "\n",
    "# read all NDB barcodes\n",
    "ndb_mask = [];\n",
    "\n",
    "with open(barcode_dir+os.sep+'NDBs.fasta', \"rU\") as handle:\n",
    "    ndb_barcodes = [];\n",
    "    for record in SeqIO.parse(handle, \"fasta\"):\n",
    "        if int(record.id.split('_')[1]) not in ndb_mask:\n",
    "            ndb_barcodes.append(record);\n",
    "print \"Barcodes loaded: Stv: \"+str(len(stv_barcodes))+\", NDB: \"+str(len(ndb_barcodes));"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 2.2 Read PCR primers"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Primers loaded: forward: 12, reverse: 9\n",
      "- forward primer: ID: W1B09_primer_20\n",
      "Name: W1B09_primer_20\n",
      "Description: W1B09_primer_20\n",
      "Number of features: 0\n",
      "Seq('TAGGCGTGTCGGCCAACCAG', SingleLetterAlphabet())\n",
      "- reverse primer: ID: W1B10_primer_21\n",
      "Name: W1B10_primer_21\n",
      "Description: W1B10_primer_21\n",
      "Number of features: 0\n",
      "Seq('TAATACGACTCACTATAGGGCGGGTCGATCATCGGCTTTG', SingleLetterAlphabet())\n"
     ]
    }
   ],
   "source": [
    "primer_dir = r'/n/boslfs/LABS/zhuang_lab/User/pzheng/Primers';\n",
    "fwd_primer_filename = 'forward_primers_keep.fasta';\n",
    "rev_primer_filename = 'reverse_primers_keep.fasta';\n",
    "\n",
    "# read all forward primers\n",
    "with open(primer_dir+os.sep+fwd_primer_filename, \"rU\") as handle:\n",
    "    fwd_primers = [];\n",
    "    for record in SeqIO.parse(handle, \"fasta\"):\n",
    "        fwd_primers.append(record);\n",
    "# read all forward primers\n",
    "with open(primer_dir+os.sep+rev_primer_filename, \"rU\") as handle:\n",
    "    rev_primers = [];\n",
    "    for record in SeqIO.parse(handle, \"fasta\"):\n",
    "        rev_primers.append(record);\n",
    "print \"Primers loaded: forward: \"+str(len(fwd_primers))+\", reverse: \"+str(len(rev_primers));    \n",
    "\n",
    "# primers\n",
    "fprimer = fwd_primers[10];\n",
    "print '- forward primer:', fprimer\n",
    "rprimer = rev_primers[8];\n",
    "print '- reverse primer:', rprimer"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def Check_Probes(pb_records, pb_lists, master_dir, \n",
    "                 fwd_primer,rev_primer,\n",
    "                 stv_barcodes, ndb_barcodes,\n",
    "                 RNA=False,RNA_padding=6,\n",
    "                 add_rand_gap=0, total_bc=3, barcode_len=20, \n",
    "                 target_len=42, word_size=17, \n",
    "                 max_internal_hits=5, max_genome_hits=200,\n",
    "                 index_folder=r'/n/boslfs/LABS/zhuang_lab/User/pzheng/Indeces/human/hg38',\n",
    "                 save_dir=r'final_probes', save=True, verbose=True):\n",
    "    '''Functions to check probe quality\n",
    "    Inputs:\n",
    "        pb_records: probe information in biopython format, list of SeqRecords\n",
    "        pb_lists: list of lists of probe information dictionary, list of list\n",
    "        master_dir: directory of this sub library, string\n",
    "        fwd_primer: forward primer in biopython format, SeqRecord\n",
    "        rev_primer: reverse primer in biopython format, SeqRecord\n",
    "        stv_barcodes: stv_barcodes in biopython format, list of SeqRecord\n",
    "        ndb_barcodes: ndb_barcodes in biopython format, list of SeqRecord\n",
    "        (optional)\n",
    "        RNA: whether this library is for RNA or not, bool(false)\n",
    "        add_rand_gap: number of random bases between barcodes, int(0)\n",
    "        total_bc: number of barcodes per probe, int(3)\n",
    "        barcode_len: length of barcode binding site on probe, int(20)\n",
    "        target_len: length of targeting region on probe, int(42)\n",
    "        word_size: word size for indeces and for probe designer, int(17)\n",
    "        max_internal_hit: maximal internal k-mer allowed, int(5)\n",
    "        max_genome_hits: maximal genome k-mer allowed, int(200)\n",
    "        index_folder: full directory to indeces, string\n",
    "        save_dir: sub directory for saving, under master_dir, string\n",
    "        save: whether save, bool(True)\n",
    "        verbose: say something during checking probes, bool(True)\n",
    "    Output:\n",
    "        kept_records: all seq records that passed filter\n",
    "        _size_from_rec: size of each region/gene, dic\n",
    "        '''\n",
    "    # imports\n",
    "    import os,glob,sys\n",
    "    sys.path.append(r'/n/home13/pzheng/Documents/python-functions/python-functions-library')\n",
    "    from LibraryConstruction import fastaread,fastawrite,fastacombine\n",
    "    import LibraryDesigner as ld\n",
    "    import numpy as np\n",
    "    \n",
    "    def _check_primer_usage(pb_records=pb_records, fwd_primer=fwd_primer, rev_primer=rev_primer,\n",
    "                            _RNA=RNA, _RNA_padding=RNA_padding,\n",
    "                            _verbose=verbose):\n",
    "        '''Check whether forward or reverse primer are used in all probes'''\n",
    "        if _verbose:\n",
    "            print \"-- Checking primer usage, total probes:\", len(pb_records)\n",
    "        fwd_len = len(fwd_primer.seq);\n",
    "        rev_len = len(rev_primer.seq[-20:].reverse_complement());\n",
    "        \n",
    "        for record in pb_records:\n",
    "            if _RNA:\n",
    "                fp = record.seq[_RNA_padding : _RNA_padding+fwd_len];\n",
    "                rp = record.seq[-rev_len-_RNA_padding: -RNA_padding];\n",
    "            else:\n",
    "                fp = record.seq[:fwd_len].upper();\n",
    "                rp = record.seq[-rev_len:]\n",
    "            # checking\n",
    "            if fp != fwd_primer.seq:\n",
    "                if _verbose:\n",
    "                    print \"--- Forward primer incorrect!\"\n",
    "                return False\n",
    "            if rp != rev_primer.seq[-20:].reverse_complement():\n",
    "                if _verbose:\n",
    "                    print \"--- Forward primer incorrect!\"\n",
    "                return False\n",
    "        return True # if no error applies\n",
    "    \n",
    "    def _check_region_size(pb_records, pb_lists=pb_lists, _RNA=RNA):\n",
    "        '''Generate a dirctionary '''\n",
    "        # get original region size\n",
    "        _reg_size_dic = {}\n",
    "        for lst in pb_lists:\n",
    "            _reg_size_dic[lst[0]['reg_index']] = len(lst);\n",
    "        # get region size from probe names\n",
    "        _size_from_rec = {}\n",
    "        for record in pb_records:\n",
    "            if _RNA:\n",
    "                reg_id = record.id.split('gene_')[1].split('_')[0];\n",
    "            else:\n",
    "                reg_id = int(record.id.split('reg_')[1].split('_')[0]);\n",
    "            if reg_id not in _size_from_rec.keys():\n",
    "                _size_from_rec[reg_id] = 1; # if not in key, create\n",
    "            else:\n",
    "                _size_from_rec[reg_id] += 1; # otherwise, add count\n",
    "        # compare\n",
    "        _match = True;\n",
    "        for k,v in sorted(_size_from_rec.items()):\n",
    "            if k not in _reg_size_dic.keys():\n",
    "                print \"region list and region id in probes not match for\", k\n",
    "                _match = False\n",
    "                break\n",
    "            else:\n",
    "                if v != _reg_size_dic[k]:\n",
    "                    print \"region size doesn't match for:\", k\n",
    "                    _match = False\n",
    "                    break\n",
    "    \n",
    "        return _reg_size_dic, _match;\n",
    "    \n",
    "\n",
    "    def _check_region_to_barcode(pb_records=pb_records, stv_barcodes=stv_barcodes, ndb_barcodes=ndb_barcodes,\n",
    "                                 total_bc=total_bc, _RNA=RNA):\n",
    "        '''Generate map from region id to barcodes used in this region'''\n",
    "        import re\n",
    "        _reg_to_barcode = {}\n",
    "        for record in pb_records:\n",
    "            # region id\n",
    "            if _RNA:\n",
    "                reg_id = record.id.split('gene_')[1].split('_')[0];\n",
    "            else:\n",
    "                reg_id = int(record.id.split('reg_')[1].split('_')[0]);\n",
    "            # put reg_id into reg_to_barcode dic\n",
    "            if reg_id not in _reg_to_barcode.keys():\n",
    "                # barcode ids\n",
    "                stv_matches = re.findall('\\'STV_(.+?)\\'', record.id.upper(), re.DOTALL)\n",
    "                ndb_matches = re.findall('\\'NDB_(.+?)\\'', record.id.upper(), re.DOTALL)\n",
    "                stv_names = ['Stv_'+str(stv_id) for stv_id in stv_matches]\n",
    "                ndb_names = ['NDB_'+str(ndb_id) for ndb_id in ndb_matches]\n",
    "                _reg_to_barcode[reg_id] = stv_names+ndb_names\n",
    "        \n",
    "        ## barcode check\n",
    "        _barcode_check = True;\n",
    "        # barcode names\n",
    "        bc_names = [stv.id for stv in stv_barcodes] + [ndb.id for ndb in ndb_barcodes]\n",
    "        # search through previous dictionary\n",
    "        for reg,bcs in sorted(_reg_to_barcode.items()):\n",
    "            for bc in bcs:\n",
    "                if bc not in bc_names:\n",
    "                    print \"-- Wrong barcode name for barcode: \"+str(bc)+\", region: \"+str(reg)\n",
    "                    _barcode_check = False\n",
    "                    break\n",
    "        \n",
    "        return _reg_to_barcode, _barcode_check;\n",
    "        \n",
    "    def _parsing_probe_sequence(record, fwd_primer=fwd_primer, rev_primer=rev_primer,\n",
    "                                add_rand_gap=add_rand_gap, barcode_len=barcode_len, target_len=target_len,\n",
    "                                _RNA=RNA, _RNA_padding=RNA_padding):\n",
    "        '''parse a probe sequence to acquire all barcode binding sites'''\n",
    "        # take in a seq record, parse the sequence and return a list of all included barcodes (20mer,RC)\n",
    "        barcode_list = [];\n",
    "        if RNA:\n",
    "            _main_seq = record.seq[_RNA_padding+len(fwd_primer.seq):-20-_RNA_padding];\n",
    "        else:\n",
    "            _main_seq = record.seq[len(fwd_primer.seq):-20];\n",
    "        \n",
    "        \n",
    "        # trim first 2 barcodes\n",
    "        for i in range(2):\n",
    "            barcode_list.append(_main_seq[:barcode_len]);\n",
    "            _main_seq = _main_seq[(barcode_len+add_rand_gap):];\n",
    "        # trim all barcodes from the end\n",
    "        while len(_main_seq) > target_len:\n",
    "            barcode_list.append(_main_seq[-barcode_len:]);\n",
    "            _main_seq = _main_seq[:-(barcode_len+add_rand_gap)];\n",
    "        \n",
    "        return barcode_list;\n",
    "    \n",
    "    def _finding_barcode_name(barcode_list, stv_barcodes=stv_barcodes, ndb_barcodes=ndb_barcodes, \n",
    "                              barcode_len=barcode_len, total_bc=total_bc):\n",
    "        '''Given barcode list generated by parsing probe, return a list of barcode names'''\n",
    "        _name_list = [];\n",
    "        for bc_site in barcode_list:\n",
    "            for bc in stv_barcodes+ndb_barcodes:\n",
    "                if bc.seq[-barcode_len:] == bc_site.reverse_complement():\n",
    "                    _name_list.append(bc.id);\n",
    "                    break;\n",
    "        \n",
    "        if len(_name_list) < total_bc:\n",
    "            print \"-- Failed in finding some barcodes.\"\n",
    "            print barcode_list, _name_list\n",
    "        return _name_list;\n",
    "    \n",
    "    \n",
    "    def _check_barcode_to_region(reg_to_barcode, \n",
    "                                 pb_records=pb_records, stv_barcodes=stv_barcodes, ndb_barcodes=ndb_barcodes, _RNA=RNA):\n",
    "        '''Generate map from barcode id to region id'''\n",
    "        _barcode_to_reg = {}\n",
    "        _reg_id_exists = []\n",
    "        for record in pb_records:\n",
    "            # region id\n",
    "            if _RNA:\n",
    "                reg_id = record.id.split('gene_')[1].split('_')[0];\n",
    "            else:\n",
    "                reg_id = int(record.id.split('reg_')[1].split('_')[0]);\n",
    "\n",
    "            if reg_id in _reg_id_exists:\n",
    "                continue;\n",
    "            else:\n",
    "                _barcode_list = _parsing_probe_sequence(record)\n",
    "                _name_list = _finding_barcode_name(_barcode_list)\n",
    "                for _n in _name_list:\n",
    "                    if _n not in _barcode_to_reg.keys(): # create if not in dic\n",
    "                        _barcode_to_reg[_n] = [reg_id]\n",
    "                    else: # otherwise, append\n",
    "                        _barcode_to_reg[_n].append(reg_id)\n",
    "            _reg_id_exists.append(reg_id)\n",
    "        ## check region distribution\n",
    "        # invert dic from reg_to_barcode\n",
    "        _inv_dic = {}\n",
    "        for reg,bcs in sorted(reg_to_barcode.items()):\n",
    "            for bc in bcs:\n",
    "                if bc not in _inv_dic.keys():\n",
    "                    _inv_dic[bc] = [reg];\n",
    "                else:\n",
    "                    _inv_dic[bc].append(reg);\n",
    "        # compare\n",
    "        _region_check=True\n",
    "        for bc, regs in sorted(_inv_dic.items()):\n",
    "            if bc not in _barcode_to_reg.keys():\n",
    "                print \"-- \"+str(bc)+\" not in barcode_to_region dic!\"\n",
    "                print '---',bc\n",
    "                _region_check = False\n",
    "                break\n",
    "            else:\n",
    "                for reg in regs:\n",
    "                    if reg not in _barcode_to_reg[bc]:\n",
    "                        print \"-- some barcode in \"+str(regs)+\" not compatible with \"+str(bc)+\" in barcode_to_region dic!\"\n",
    "                        _region_check = False\n",
    "                        break\n",
    "                for reg in _barcode_to_reg[bc]:\n",
    "                    if reg not in regs:\n",
    "                        print \"-- some barcode in \"+str(bc)+\" not compatible with \"+str(regs)+\" in barcode_to_region dic!\"\n",
    "                        _region_check = False\n",
    "                        break\n",
    "                    \n",
    "        return _barcode_to_reg, _region_check\n",
    "    \n",
    "    def _check_barcode_to_color(pb_records=pb_records, stv_barcodes=stv_barcodes, ndb_barcodes=ndb_barcodes, \n",
    "                                stv_color=True, ndb_color=False,\n",
    "                                _save=save, master_dir=master_dir, save_dir=save_dir):\n",
    "        '''If multi_color is applied, generate a barcode_to_color dic for adaptor design'''\n",
    "        if 'color' not in str(pb_records[0].id):\n",
    "            print \"-- color check not applied\";\n",
    "            return False\n",
    "        elif not stv_color and not ndb_color:\n",
    "            print \"-- color check turned off in both stv and ndb\";\n",
    "            return False\n",
    "        else:\n",
    "            # get barcodes\n",
    "            _barcode_names = []\n",
    "            if stv_color: # if stv has multi-color\n",
    "                _barcode_names += [bc.id for bc in stv_barcodes];\n",
    "            if ndb_color: # if ndb has multi-color\n",
    "                _barcode_names += [bc.id for bc in ndb_barcodes];\n",
    "            # initialize color dic\n",
    "            _barcode_to_color = {};\n",
    "            _exist_regs = [];\n",
    "            # search through all probes\n",
    "            for record in pb_records:\n",
    "                # reg_id\n",
    "                if _RNA:\n",
    "                    reg_id = record.id.split('gene_')[1].split('_')[0];\n",
    "                else:\n",
    "                    reg_id = int(record.id.split('reg_')[1].split('_')[0]);\n",
    "                \n",
    "                if _reg_id in _exist_regs:\n",
    "                    continue\n",
    "                else: \n",
    "                    _exist_regs.append(_reg_id);\n",
    "                _color = int(str(record.id).split('color_')[1])\n",
    "                _barcode_list = _parsing_probe_sequence(record)\n",
    "                _name_list = _finding_barcode_name(_barcode_list)\n",
    "                \n",
    "                for _name in _name_list:\n",
    "                    if _name in _barcode_names:\n",
    "                        if _name not in _barcode_to_color.keys():\n",
    "                            _barcode_to_color[_name] = [_color]\n",
    "                        else:\n",
    "                            _barcode_to_color[_name].append(_color);\n",
    "            # keep the unique colors\n",
    "            _barcode_to_unique_color = {}\n",
    "            for k,v in sorted(_barcode_to_color.items()):\n",
    "                _barcode_to_unique_color[k] = np.unique(v)\n",
    "            if _save:\n",
    "                import csv\n",
    "                # mkdir if not exist for this region\n",
    "                if not os.path.exists(master_dir+os.sep+save_dir):\n",
    "                    os.makedirs(master_dir+os.sep+save_dir)\n",
    "                with open(master_dir+os.sep+save_dir+os.sep+'color-usage.csv','w') as output_handle:\n",
    "                    fieldnames = ['barcode', 'color']\n",
    "                    writer = csv.DictWriter(output_handle, fieldnames=fieldnames)\n",
    "                    writer.writeheader()\n",
    "                    for _barcode, _color in sorted(_barcode_to_unique_color.items(), key=lambda (k,v):int(k.split('_')[1])):\n",
    "                        writer.writerow({'barcode': _barcode, 'color': _color})\n",
    "                \n",
    "        return _barcode_to_unique_color\n",
    "                            \n",
    "    \n",
    "    def _construct_internal_map(master_dir=master_dir, save_dir=save_dir, word_size=word_size):\n",
    "        '''Using functions in LibraryDesign, compute an internal khmer map'''\n",
    "        _int_map = khmer.Countgraph(word_size, 1e9, 2) \n",
    "        _int_map.set_use_bigcount(True)\n",
    "        _nms,_seqs = fastaread(master_dir+os.sep+save_dir+os.sep+'candidate_probes.fasta')\n",
    "        for _seq in _seqs:\n",
    "            _int_map.consume(_seq.upper())\n",
    "        return _int_map\n",
    "    \n",
    "    def _check_barcode_in_probes(barcode_to_reg, reg_size_dic, int_map, \n",
    "                                 stv_barcodes=stv_barcodes, ndb_barcodes=ndb_barcodes,\n",
    "                                 barcode_len=barcode_len, max_internal_hits=max_internal_hits):\n",
    "        '''Check barcode appearance in probes, whether that match barcode_to_region scheme'''\n",
    "        _barcode_in_probes = {}\n",
    "        for bc_name, regs in sorted(barcode_to_reg.items()):\n",
    "            bc = None\n",
    "            for _bc in stv_barcodes+ndb_barcodes:\n",
    "                if bc_name == _bc.id:\n",
    "                    bc = _bc\n",
    "                    break\n",
    "            bc_hits = int_map.get_kmer_counts( str(bc.seq[-barcode_len:].reverse_complement()).upper());\n",
    "            if max(bc_hits) - min(bc_hits) > max_internal_hits:\n",
    "                print \"-- Barcode: \"+str(bc)+\" has more off-target in different part of itself!\"\n",
    "                return False\n",
    "            else:\n",
    "                regs,reg_cts = np.unique(regs, return_counts=True);\n",
    "                bc_in_probe = 0;\n",
    "                for reg,ct in zip(regs,reg_cts):\n",
    "                    bc_in_probe += reg_size_dic[reg] * ct;\n",
    "                if max(bc_hits) - bc_in_probe > max_internal_hits:\n",
    "                    print \"-- Barcode: \"+str(bc)+\" has more off-target than threshold!\"\n",
    "                    return False\n",
    "            _barcode_in_probes[bc_name] = bc_in_probe;\n",
    "        return _barcode_in_probes, True\n",
    "    \n",
    "    def _check_between_probes(int_map, pb_lists=pb_lists, pb_records=pb_records):\n",
    "        pass \n",
    "    \n",
    "    def _check_against_genome(pb_records=pb_records, max_genome_hits=max_genome_hits, index_folder=index_folder):\n",
    "        '''Use Khmer to compare probe against genome'''\n",
    "        hg38 = khmer.load_countgraph(index_folder+os.sep+'full_word17_.kmer')\n",
    "        _failed_num = 0;\n",
    "        _keep_pb_records = [];\n",
    "        for record in pb_records:\n",
    "            _kmer_hits = hg38.get_kmer_counts(str(record.seq).upper());\n",
    "            if sum(_kmer_hits) > max_genome_hits:\n",
    "                print '-- Max_genome_hits is: '+str(max_genome_hits)+\", this seq got hits: \"+ str(sum(_kmer_hits))\n",
    "                _failed_num += 1;\n",
    "            else:\n",
    "                _keep_pb_records.append(record);\n",
    "                \n",
    "        return _keep_pb_records, _failed_num # if nothing goes wrong\n",
    "    \n",
    "    def _plot_info():\n",
    "        pass\n",
    "            \n",
    "    ## check primers\n",
    "    primer_usage = _check_primer_usage()\n",
    "    if verbose:\n",
    "        print \"\\n- 1.Passing primer usage check? -\", primer_usage\n",
    "    \n",
    "    ## check region size\n",
    "    reg_size_dic, size_match = _check_region_size(pb_records)\n",
    "    if verbose:\n",
    "        print \"\\n- 2.Passing region size check? -\", size_match    \n",
    "        for k,v in sorted(reg_size_dic.items()):\n",
    "            print k,':',v\n",
    "        \n",
    "    ## check region to barcode\n",
    "    reg_to_barcode, reg2bc = _check_region_to_barcode()\n",
    "    if verbose:\n",
    "        print \"\\n- 3.Passing region to barcode mapping check? -\", reg2bc    \n",
    "        for k,v in sorted(reg_to_barcode.items(), key=lambda (k,v):k):\n",
    "            print k,':',v\n",
    "        \n",
    "    ## check barcode to region (this step must be run after step 3) \n",
    "    barcode_to_reg, bc2reg = _check_barcode_to_region(reg_to_barcode)\n",
    "    if verbose:\n",
    "        print \"\\n- 4.Passing barcode to region mapping check? -\", bc2reg    \n",
    "        for k,v in sorted(barcode_to_reg.items(), key=lambda (k,v):[k[0],int(k.split('_')[1])]):\n",
    "            print k,':',v\n",
    "    \n",
    "    ## check barcode to region (this step must be run after step 3) \n",
    "    barcode_to_color = _check_barcode_to_color()\n",
    "    if verbose and barcode_to_color:\n",
    "        print \"\\n- 5.Calculating barcode to color dictionary.\"\n",
    "        for k,v in sorted(barcode_to_color.items(), key=lambda (k,v):[k[0],int(k.split('_')[1])]):\n",
    "            print k,':',v    \n",
    "    \n",
    "    \n",
    "    ## Construct an internal map\n",
    "    int_map = _construct_internal_map();\n",
    "    if verbose:\n",
    "        print \"\\n- 6.Constructing internal khmer map\";\n",
    "    \n",
    "    ## Check barcodes total counts in probes\n",
    "    barcode_in_probes, _bc_counting = _check_barcode_in_probes(barcode_to_reg, reg_size_dic, int_map)\n",
    "    if verbose:\n",
    "        print \"\\n- 7.Passing if counting barcode appearance times in probes\", _bc_counting;    \n",
    "\n",
    "    ## Check against each other    \n",
    "    \n",
    "    ## Check against genome\n",
    "    kept_records, failed_num = _check_against_genome();\n",
    "    if verbose:\n",
    "        print \"\\n- 8.Probes not passing through genome filter:\", failed_num;  \n",
    "    \n",
    "    # check region size for kept probes\n",
    "    kept_size_dic, kept_match = _check_region_size(kept_records);\n",
    "    if verbose:\n",
    "        print \"\\n- 9.Re-check region size:\"\n",
    "        for k,v in sorted(kept_size_dic.items()):\n",
    "            print k,':',v\n",
    "        print \"--- total number of probes:\", len(kept_records);\n",
    "        \n",
    "    if save:\n",
    "        pb_savefile = master_dir + os.sep + save_dir + os.sep + 'filtered_probes.fasta';\n",
    "        if verbose:\n",
    "            print \"\\n- 10.Saving probes to:\", pb_savefile\n",
    "        with open(pb_savefile, 'w') as output_handle:\n",
    "            SeqIO.write(kept_records, output_handle, 'fasta');  \n",
    "        \n",
    "    return kept_records, kept_size_dic"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- Checking primer usage, total probes: 3560\n",
      "\n",
      "- 1.Passing primer usage check? - True\n",
      "\n",
      "- 2.Passing region size check? - True\n",
      "PCDHA : 111\n",
      "PCDHA1 : 66\n",
      "PCDHA10 : 77\n",
      "PCDHA11 : 77\n",
      "PCDHA12 : 76\n",
      "PCDHA13 : 72\n",
      "PCDHA2 : 66\n",
      "PCDHA3 : 62\n",
      "PCDHA4 : 67\n",
      "PCDHA5 : 58\n",
      "PCDHA6 : 52\n",
      "PCDHA7 : 53\n",
      "PCDHA8 : 72\n",
      "PCDHA9 : 88\n",
      "PCDHAC1 : 75\n",
      "PCDHAC2 : 95\n",
      "PCDHB1 : 72\n",
      "PCDHB10 : 50\n",
      "PCDHB11 : 48\n",
      "PCDHB12 : 46\n",
      "PCDHB13 : 54\n",
      "PCDHB14 : 51\n",
      "PCDHB15 : 52\n",
      "PCDHB16 : 72\n",
      "PCDHB2 : 43\n",
      "PCDHB3 : 45\n",
      "PCDHB4 : 44\n",
      "PCDHB5 : 52\n",
      "PCDHB6 : 55\n",
      "PCDHB7 : 60\n",
      "PCDHB8 : 28\n",
      "PCDHB9 : 31\n",
      "PCDHG : 93\n",
      "PCDHGA1 : 58\n",
      "PCDHGA10 : 70\n",
      "PCDHGA11 : 70\n",
      "PCDHGA12 : 87\n",
      "PCDHGA2 : 64\n",
      "PCDHGA3 : 60\n",
      "PCDHGA4 : 71\n",
      "PCDHGA5 : 76\n",
      "PCDHGA6 : 73\n",
      "PCDHGA7 : 68\n",
      "PCDHGA8 : 79\n",
      "PCDHGA9 : 63\n",
      "PCDHGB1 : 68\n",
      "PCDHGB2 : 63\n",
      "PCDHGB3 : 76\n",
      "PCDHGB4 : 55\n",
      "PCDHGB5 : 69\n",
      "PCDHGB7 : 86\n",
      "PCDHGC3 : 87\n",
      "PCDHGC4 : 76\n",
      "PCDHGC5 : 78\n",
      "\n",
      "- 3.Passing region to barcode mapping check? - True\n",
      "PCDHA : ['NDB_137']\n",
      "PCDHA1 : ['NDB_85']\n",
      "PCDHA10 : ['NDB_94']\n",
      "PCDHA11 : ['NDB_95']\n",
      "PCDHA12 : ['NDB_96']\n",
      "PCDHA13 : ['NDB_97']\n",
      "PCDHA2 : ['NDB_86']\n",
      "PCDHA3 : ['NDB_87']\n",
      "PCDHA4 : ['NDB_88']\n",
      "PCDHA5 : ['NDB_89']\n",
      "PCDHA6 : ['NDB_90']\n",
      "PCDHA7 : ['NDB_91']\n",
      "PCDHA8 : ['NDB_92']\n",
      "PCDHA9 : ['NDB_93']\n",
      "PCDHAC1 : ['NDB_132']\n",
      "PCDHAC2 : ['NDB_133']\n",
      "PCDHB1 : ['NDB_98']\n",
      "PCDHB10 : ['NDB_107']\n",
      "PCDHB11 : ['NDB_108']\n",
      "PCDHB12 : ['NDB_109']\n",
      "PCDHB13 : ['NDB_110']\n",
      "PCDHB14 : ['NDB_111']\n",
      "PCDHB15 : ['NDB_112']\n",
      "PCDHB16 : ['NDB_113']\n",
      "PCDHB2 : ['NDB_99']\n",
      "PCDHB3 : ['NDB_100']\n",
      "PCDHB4 : ['NDB_101']\n",
      "PCDHB5 : ['NDB_102']\n",
      "PCDHB6 : ['NDB_103']\n",
      "PCDHB7 : ['NDB_104']\n",
      "PCDHB8 : ['NDB_105']\n",
      "PCDHB9 : ['NDB_106']\n",
      "PCDHG : ['NDB_138']\n",
      "PCDHGA1 : ['NDB_114']\n",
      "PCDHGA10 : ['NDB_123']\n",
      "PCDHGA11 : ['NDB_124']\n",
      "PCDHGA12 : ['NDB_125']\n",
      "PCDHGA2 : ['NDB_115']\n",
      "PCDHGA3 : ['NDB_116']\n",
      "PCDHGA4 : ['NDB_117']\n",
      "PCDHGA5 : ['NDB_118']\n",
      "PCDHGA6 : ['NDB_119']\n",
      "PCDHGA7 : ['NDB_120']\n",
      "PCDHGA8 : ['NDB_121']\n",
      "PCDHGA9 : ['NDB_122']\n",
      "PCDHGB1 : ['NDB_126']\n",
      "PCDHGB2 : ['NDB_127']\n",
      "PCDHGB3 : ['NDB_128']\n",
      "PCDHGB4 : ['NDB_129']\n",
      "PCDHGB5 : ['NDB_130']\n",
      "PCDHGB7 : ['NDB_131']\n",
      "PCDHGC3 : ['NDB_134']\n",
      "PCDHGC4 : ['NDB_135']\n",
      "PCDHGC5 : ['NDB_136']\n",
      "\n",
      "- 4.Passing barcode to region mapping check? - True\n",
      "NDB_85 : ['PCDHA1', 'PCDHA1', 'PCDHA1']\n",
      "NDB_86 : ['PCDHA2', 'PCDHA2', 'PCDHA2']\n",
      "NDB_87 : ['PCDHA3', 'PCDHA3', 'PCDHA3']\n",
      "NDB_88 : ['PCDHA4', 'PCDHA4', 'PCDHA4']\n",
      "NDB_89 : ['PCDHA5', 'PCDHA5', 'PCDHA5']\n",
      "NDB_90 : ['PCDHA6', 'PCDHA6', 'PCDHA6']\n",
      "NDB_91 : ['PCDHA7', 'PCDHA7', 'PCDHA7']\n",
      "NDB_92 : ['PCDHA8', 'PCDHA8', 'PCDHA8']\n",
      "NDB_93 : ['PCDHA9', 'PCDHA9', 'PCDHA9']\n",
      "NDB_94 : ['PCDHA10', 'PCDHA10', 'PCDHA10']\n",
      "NDB_95 : ['PCDHA11', 'PCDHA11', 'PCDHA11']\n",
      "NDB_96 : ['PCDHA12', 'PCDHA12', 'PCDHA12']\n",
      "NDB_97 : ['PCDHA13', 'PCDHA13', 'PCDHA13']\n",
      "NDB_98 : ['PCDHB1', 'PCDHB1', 'PCDHB1']\n",
      "NDB_99 : ['PCDHB2', 'PCDHB2', 'PCDHB2']\n",
      "NDB_100 : ['PCDHB3', 'PCDHB3', 'PCDHB3']\n",
      "NDB_101 : ['PCDHB4', 'PCDHB4', 'PCDHB4']\n",
      "NDB_102 : ['PCDHB5', 'PCDHB5', 'PCDHB5']\n",
      "NDB_103 : ['PCDHB6', 'PCDHB6', 'PCDHB6']\n",
      "NDB_104 : ['PCDHB7', 'PCDHB7', 'PCDHB7']\n",
      "NDB_105 : ['PCDHB8', 'PCDHB8', 'PCDHB8']\n",
      "NDB_106 : ['PCDHB9', 'PCDHB9', 'PCDHB9']\n",
      "NDB_107 : ['PCDHB10', 'PCDHB10', 'PCDHB10']\n",
      "NDB_108 : ['PCDHB11', 'PCDHB11', 'PCDHB11']\n",
      "NDB_109 : ['PCDHB12', 'PCDHB12', 'PCDHB12']\n",
      "NDB_110 : ['PCDHB13', 'PCDHB13', 'PCDHB13']\n",
      "NDB_111 : ['PCDHB14', 'PCDHB14', 'PCDHB14']\n",
      "NDB_112 : ['PCDHB15', 'PCDHB15', 'PCDHB15']\n",
      "NDB_113 : ['PCDHB16', 'PCDHB16', 'PCDHB16']\n",
      "NDB_114 : ['PCDHGA1', 'PCDHGA1', 'PCDHGA1']\n",
      "NDB_115 : ['PCDHGA2', 'PCDHGA2', 'PCDHGA2']\n",
      "NDB_116 : ['PCDHGA3', 'PCDHGA3', 'PCDHGA3']\n",
      "NDB_117 : ['PCDHGA4', 'PCDHGA4', 'PCDHGA4']\n",
      "NDB_118 : ['PCDHGA5', 'PCDHGA5', 'PCDHGA5']\n",
      "NDB_119 : ['PCDHGA6', 'PCDHGA6', 'PCDHGA6']\n",
      "NDB_120 : ['PCDHGA7', 'PCDHGA7', 'PCDHGA7']\n",
      "NDB_121 : ['PCDHGA8', 'PCDHGA8', 'PCDHGA8']\n",
      "NDB_122 : ['PCDHGA9', 'PCDHGA9', 'PCDHGA9']\n",
      "NDB_123 : ['PCDHGA10', 'PCDHGA10', 'PCDHGA10']\n",
      "NDB_124 : ['PCDHGA11', 'PCDHGA11', 'PCDHGA11']\n",
      "NDB_125 : ['PCDHGA12', 'PCDHGA12', 'PCDHGA12']\n",
      "NDB_126 : ['PCDHGB1', 'PCDHGB1', 'PCDHGB1']\n",
      "NDB_127 : ['PCDHGB2', 'PCDHGB2', 'PCDHGB2']\n",
      "NDB_128 : ['PCDHGB3', 'PCDHGB3', 'PCDHGB3']\n",
      "NDB_129 : ['PCDHGB4', 'PCDHGB4', 'PCDHGB4']\n",
      "NDB_130 : ['PCDHGB5', 'PCDHGB5', 'PCDHGB5']\n",
      "NDB_131 : ['PCDHGB7', 'PCDHGB7', 'PCDHGB7']\n",
      "NDB_132 : ['PCDHAC1', 'PCDHAC1', 'PCDHAC1']\n",
      "NDB_133 : ['PCDHAC2', 'PCDHAC2', 'PCDHAC2']\n",
      "NDB_134 : ['PCDHGC3', 'PCDHGC3', 'PCDHGC3']\n",
      "NDB_135 : ['PCDHGC4', 'PCDHGC4', 'PCDHGC4']\n",
      "NDB_136 : ['PCDHGC5', 'PCDHGC5', 'PCDHGC5']\n",
      "NDB_137 : ['PCDHA', 'PCDHA', 'PCDHA']\n",
      "NDB_138 : ['PCDHG', 'PCDHG', 'PCDHG']\n",
      "-- color check not applied\n",
      "\n",
      "- 6.Constructing internal khmer map\n",
      "\n",
      "- 7.Passing if counting barcode appearance times in probes True\n",
      "-- Max_genome_hits is: 200, this seq got hits: 969\n",
      "-- Max_genome_hits is: 200, this seq got hits: 337\n",
      "-- Max_genome_hits is: 200, this seq got hits: 568\n",
      "-- Max_genome_hits is: 200, this seq got hits: 216\n",
      "-- Max_genome_hits is: 200, this seq got hits: 270\n",
      "-- Max_genome_hits is: 200, this seq got hits: 2603\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1247\n",
      "\n",
      "- 8.Probes not passing through genome filter: 7\n",
      "region size doesn't match for: PCDHG\n",
      "\n",
      "- 9.Re-check region size:\n",
      "PCDHA : 111\n",
      "PCDHA1 : 66\n",
      "PCDHA10 : 77\n",
      "PCDHA11 : 77\n",
      "PCDHA12 : 76\n",
      "PCDHA13 : 72\n",
      "PCDHA2 : 66\n",
      "PCDHA3 : 62\n",
      "PCDHA4 : 67\n",
      "PCDHA5 : 58\n",
      "PCDHA6 : 52\n",
      "PCDHA7 : 53\n",
      "PCDHA8 : 72\n",
      "PCDHA9 : 88\n",
      "PCDHAC1 : 75\n",
      "PCDHAC2 : 95\n",
      "PCDHB1 : 72\n",
      "PCDHB10 : 50\n",
      "PCDHB11 : 48\n",
      "PCDHB12 : 46\n",
      "PCDHB13 : 54\n",
      "PCDHB14 : 51\n",
      "PCDHB15 : 52\n",
      "PCDHB16 : 72\n",
      "PCDHB2 : 43\n",
      "PCDHB3 : 45\n",
      "PCDHB4 : 44\n",
      "PCDHB5 : 52\n",
      "PCDHB6 : 55\n",
      "PCDHB7 : 60\n",
      "PCDHB8 : 28\n",
      "PCDHB9 : 31\n",
      "PCDHG : 93\n",
      "PCDHGA1 : 58\n",
      "PCDHGA10 : 70\n",
      "PCDHGA11 : 70\n",
      "PCDHGA12 : 87\n",
      "PCDHGA2 : 64\n",
      "PCDHGA3 : 60\n",
      "PCDHGA4 : 71\n",
      "PCDHGA5 : 76\n",
      "PCDHGA6 : 73\n",
      "PCDHGA7 : 68\n",
      "PCDHGA8 : 79\n",
      "PCDHGA9 : 63\n",
      "PCDHGB1 : 68\n",
      "PCDHGB2 : 63\n",
      "PCDHGB3 : 76\n",
      "PCDHGB4 : 55\n",
      "PCDHGB5 : 69\n",
      "PCDHGB7 : 86\n",
      "PCDHGC3 : 87\n",
      "PCDHGC4 : 76\n",
      "PCDHGC5 : 78\n",
      "--- total number of probes: 3553\n",
      "\n",
      "- 10.Saving probes to: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/CTP-02/PCDH_smFISH/final_probes/filtered_probes.fasta\n"
     ]
    }
   ],
   "source": [
    "# master directory\n",
    "master_dir = r'/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/CTP-02/PCDH_smFISH';\n",
    "\n",
    "kept_records, kept_size_dic = Check_Probes(pb_records, pb_lists, master_dir, target_len=30,\n",
    "                                           RNA=True,RNA_padding=6,\n",
    "                                        fwd_primer=fprimer, rev_primer=rprimer,\n",
    "                                        stv_barcodes=stv_barcodes, ndb_barcodes=ndb_barcodes)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 2",
   "language": "python",
   "name": "python2"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.13"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
