{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": true
   },
   "source": [
    "# Library design for CTP02, \n",
    "\n",
    "## EMT single molecule FISH check\n",
    "by Pu Zheng\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#minimum imports:\n",
    "import time,os,sys,glob\n",
    "import cPickle as pickle\n",
    "import numpy as np\n",
    "import khmer\n",
    "sys.path.append(r'/n/home13/pzheng/Documents/python-functions/python-functions-library')\n",
    "\n",
    "from LibraryConstruction import fastaread,fastawrite,fastacombine\n",
    "import LibraryDesigner as ld\n",
    "import LibraryConstruction as lc\n",
    "\n",
    "from Bio import SeqIO\n",
    "from Bio.Seq import Seq\n",
    "from Bio.Alphabet import IUPAC\n",
    "from Bio.SeqRecord import SeqRecord "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 1. Load previously designed probes, generate pb_records and pb_lists"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- Save pb_lists\n",
      "-- Save pb_records\n"
     ]
    }
   ],
   "source": [
    "# dir\n",
    "master_dir = r'/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/CTP-02/EMT_smFISH';\n",
    "# input filename\n",
    "pb_filename = 'EMT-smFISH.csv';\n",
    "\n",
    "pb_records, gene_pb_dic = [],{}; # initialize\n",
    "\n",
    "import csv\n",
    "with open(master_dir+os.sep+pb_filename,'rU') as handle:\n",
    "    _csvreader = csv.reader(handle, delimiter=',') # reader\n",
    "    # check if has header\n",
    "    _has_header = csv.Sniffer().has_header(handle.readline())\n",
    "    handle.seek(0)  # Rewind.\n",
    "    if _has_header:\n",
    "        next(_csvreader)  # Skip header row.\n",
    "    for row in _csvreader:\n",
    "        _name = row[0];\n",
    "        _seq = row[1].upper()\n",
    "        pb_records.append(SeqRecord(Seq(_seq,alphabet=IUPAC.unambiguous_dna),id=_name, name=_name,description=''))\n",
    "        #reg_id = int(name.split('reg_')[1].split(\"_\")[0])\n",
    "        gene_id = _name.split('gene_')[1].split(\"_\")[0]\n",
    "        pb_info = {'reg_index':gene_id, 'total_seq':_seq, 'total_name':_name};\n",
    "        if gene_id not in gene_pb_dic.keys():\n",
    "            gene_pb_dic[gene_id] = [pb_info];\n",
    "        else:\n",
    "            gene_pb_dic[gene_id].append(pb_info);\n",
    "# define pb_lists, match format\n",
    "pb_lists = gene_pb_dic.values()\n",
    "\n",
    "# save\n",
    "save_dir = 'final_probes'\n",
    "if not os.path.exists(master_dir+os.sep+save_dir):\n",
    "    os.makedirs(master_dir+os.sep+save_dir)\n",
    "print \"-- Save pb_lists\"\n",
    "pickle.dump(pb_lists, open(master_dir+os.sep+save_dir+os.sep+'list.pkl', 'w'));\n",
    "print \"-- Save pb_records\"\n",
    "with open(master_dir+os.sep+save_dir+os.sep+'candidate_probes.fasta', \"w\") as output_handle:\n",
    "    SeqIO.write(pb_records, output_handle, 'fasta');"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Read barcode Sequences"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Barcodes loaded: Stv: 75, NDB: 1052\n"
     ]
    }
   ],
   "source": [
    "# read all Stv barcodes\n",
    "barcode_dir = r'/n/boslfs/LABS/zhuang_lab/User/pzheng/Barcodes';\n",
    "\n",
    "#stv_adaptor = [1,2,17,62,77,78,79,80,81,82,83,84] # barcodes saved for adaptors\n",
    "#stv_bad = [34,38,41] # barcodes performed badly\n",
    "#stv_mask = stv_adaptor + stv_bad \n",
    "stv_mask = []\n",
    "\n",
    "with open(barcode_dir+os.sep+'top_Stvs.fasta', \"rU\") as handle:\n",
    "    stv_barcodes = [];\n",
    "    for record in SeqIO.parse(handle, \"fasta\"):\n",
    "        if int(record.id.split('_')[1]) not in stv_mask:\n",
    "            stv_barcodes.append(record);\n",
    "\n",
    "# read all NDB barcodes\n",
    "ndb_mask = [];\n",
    "\n",
    "with open(barcode_dir+os.sep+'NDBs.fasta', \"rU\") as handle:\n",
    "    ndb_barcodes = [];\n",
    "    for record in SeqIO.parse(handle, \"fasta\"):\n",
    "        if int(record.id.split('_')[1]) not in ndb_mask:\n",
    "            ndb_barcodes.append(record);\n",
    "print \"Barcodes loaded: Stv: \"+str(len(stv_barcodes))+\", NDB: \"+str(len(ndb_barcodes));"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Read PCR primers"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Primers loaded: forward: 12, reverse: 9\n",
      "- forward primer: ID: W1A09_primer_8\n",
      "Name: W1A09_primer_8\n",
      "Description: W1A09_primer_8\n",
      "Number of features: 0\n",
      "Seq('TTGTTGAGGCGGCGGAAGTC', SingleLetterAlphabet())\n",
      "- reverse primer: ID: W1A08_primer_7\n",
      "Name: W1A08_primer_7\n",
      "Description: W1A08_primer_7\n",
      "Number of features: 0\n",
      "Seq('TAATACGACTCACTATAGGGCCCTTTCGCGCTGCATCGAC', SingleLetterAlphabet())\n"
     ]
    }
   ],
   "source": [
    "primer_dir = r'/n/boslfs/LABS/zhuang_lab/User/pzheng/Primers';\n",
    "fwd_primer_filename = 'forward_primers_keep.fasta';\n",
    "rev_primer_filename = 'reverse_primers_keep.fasta';\n",
    "\n",
    "# read all forward primers\n",
    "with open(primer_dir+os.sep+fwd_primer_filename, \"rU\") as handle:\n",
    "    fwd_primers = [];\n",
    "    for record in SeqIO.parse(handle, \"fasta\"):\n",
    "        fwd_primers.append(record);\n",
    "# read all forward primers\n",
    "with open(primer_dir+os.sep+rev_primer_filename, \"rU\") as handle:\n",
    "    rev_primers = [];\n",
    "    for record in SeqIO.parse(handle, \"fasta\"):\n",
    "        rev_primers.append(record);\n",
    "print \"Primers loaded: forward: \"+str(len(fwd_primers))+\", reverse: \"+str(len(rev_primers));    \n",
    "\n",
    "# primers\n",
    "fprimer = fwd_primers[4];\n",
    "print '- forward primer:', fprimer\n",
    "rprimer = rev_primers[3];\n",
    "print '- reverse primer:', rprimer"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def Check_Probes(pb_records, pb_lists, master_dir, \n",
    "                 fwd_primer,rev_primer,\n",
    "                 stv_barcodes, ndb_barcodes,\n",
    "                 RNA=False,RNA_padding=6,\n",
    "                 add_rand_gap=0, total_bc=3, barcode_len=20, \n",
    "                 target_len=42, word_size=17, \n",
    "                 max_internal_hits=5, max_genome_hits=200,\n",
    "                 index_folder=r'/n/boslfs/LABS/zhuang_lab/User/pzheng/Indeces/human/hg38',\n",
    "                 save_dir=r'final_probes', save=True, verbose=True):\n",
    "    '''Functions to check probe quality\n",
    "    Inputs:\n",
    "        pb_records: probe information in biopython format, list of SeqRecords\n",
    "        pb_lists: list of lists of probe information dictionary, list of list\n",
    "        master_dir: directory of this sub library, string\n",
    "        fwd_primer: forward primer in biopython format, SeqRecord\n",
    "        rev_primer: reverse primer in biopython format, SeqRecord\n",
    "        stv_barcodes: stv_barcodes in biopython format, list of SeqRecord\n",
    "        ndb_barcodes: ndb_barcodes in biopython format, list of SeqRecord\n",
    "        (optional)\n",
    "        RNA: whether this library is for RNA or not, bool(false)\n",
    "        add_rand_gap: number of random bases between barcodes, int(0)\n",
    "        total_bc: number of barcodes per probe, int(3)\n",
    "        barcode_len: length of barcode binding site on probe, int(20)\n",
    "        target_len: length of targeting region on probe, int(42)\n",
    "        word_size: word size for indeces and for probe designer, int(17)\n",
    "        max_internal_hit: maximal internal k-mer allowed, int(5)\n",
    "        max_genome_hits: maximal genome k-mer allowed, int(200)\n",
    "        index_folder: full directory to indeces, string\n",
    "        save_dir: sub directory for saving, under master_dir, string\n",
    "        save: whether save, bool(True)\n",
    "        verbose: say something during checking probes, bool(True)\n",
    "    Output:\n",
    "        kept_records: all seq records that passed filter\n",
    "        _size_from_rec: size of each region/gene, dic\n",
    "        '''\n",
    "    # imports\n",
    "    import os,glob,sys\n",
    "    sys.path.append(r'/n/home13/pzheng/Documents/python-functions/python-functions-library')\n",
    "    from LibraryConstruction import fastaread,fastawrite,fastacombine\n",
    "    import LibraryDesigner as ld\n",
    "    import numpy as np\n",
    "    \n",
    "    def _check_primer_usage(pb_records=pb_records, fwd_primer=fwd_primer, rev_primer=rev_primer,\n",
    "                            _RNA=RNA, _RNA_padding=RNA_padding,\n",
    "                            _verbose=verbose):\n",
    "        '''Check whether forward or reverse primer are used in all probes'''\n",
    "        if _verbose:\n",
    "            print \"-- Checking primer usage, total probes:\", len(pb_records)\n",
    "        fwd_len = len(fwd_primer.seq);\n",
    "        rev_len = len(rev_primer.seq[-20:].reverse_complement());\n",
    "        \n",
    "        for record in pb_records:\n",
    "            if _RNA:\n",
    "                fp = record.seq[_RNA_padding : _RNA_padding+fwd_len];\n",
    "                rp = record.seq[-rev_len-_RNA_padding: -RNA_padding];\n",
    "            else:\n",
    "                fp = record.seq[:fwd_len].upper();\n",
    "                rp = record.seq[-rev_len:]\n",
    "            # checking\n",
    "            if fp != fwd_primer.seq:\n",
    "                if _verbose:\n",
    "                    print \"--- Forward primer incorrect in\", record\n",
    "                return False\n",
    "            if rp != rev_primer.seq[-20:].reverse_complement():\n",
    "                if _verbose:\n",
    "                    print \"--- Forward primer incorrect in\", record\n",
    "                return False\n",
    "        return True # if no error applies\n",
    "    \n",
    "    def _check_region_size(pb_records, pb_lists=pb_lists, _RNA=RNA):\n",
    "        '''Generate a dirctionary '''\n",
    "        # get original region size\n",
    "        _reg_size_dic = {}\n",
    "        for lst in pb_lists:\n",
    "            _reg_size_dic[lst[0]['reg_index']] = len(lst);\n",
    "        # get region size from probe names\n",
    "        _size_from_rec = {}\n",
    "        for record in pb_records:\n",
    "            if _RNA:\n",
    "                reg_id = record.id.split('gene_')[1].split('_')[0];\n",
    "            else:\n",
    "                reg_id = int(record.id.split('reg_')[1].split('_')[0]);\n",
    "            if reg_id not in _size_from_rec.keys():\n",
    "                _size_from_rec[reg_id] = 1; # if not in key, create\n",
    "            else:\n",
    "                _size_from_rec[reg_id] += 1; # otherwise, add count\n",
    "        # compare\n",
    "        _match = True;\n",
    "        for k,v in sorted(_size_from_rec.items()):\n",
    "            if k not in _reg_size_dic.keys():\n",
    "                print \"region list and region id in probes not match for\", k\n",
    "                _match = False\n",
    "                break\n",
    "            else:\n",
    "                if v != _reg_size_dic[k]:\n",
    "                    print \"region size doesn't match for:\", k\n",
    "                    _match = False\n",
    "                    break\n",
    "    \n",
    "        return _reg_size_dic, _match;\n",
    "    \n",
    "\n",
    "    def _check_region_to_barcode(pb_records=pb_records, stv_barcodes=stv_barcodes, ndb_barcodes=ndb_barcodes,\n",
    "                                 total_bc=total_bc, _RNA=RNA):\n",
    "        '''Generate map from region id to barcodes used in this region'''\n",
    "        import re\n",
    "        _reg_to_barcode = {}\n",
    "        for record in pb_records:\n",
    "            # region id\n",
    "            if _RNA:\n",
    "                reg_id = record.id.split('gene_')[1].split('_')[0];\n",
    "            else:\n",
    "                reg_id = int(record.id.split('reg_')[1].split('_')[0]);\n",
    "            # put reg_id into reg_to_barcode dic\n",
    "            if reg_id not in _reg_to_barcode.keys():\n",
    "                # barcode ids\n",
    "                stv_matches = re.findall('\\'STV_(.+?)\\'', record.id.upper(), re.DOTALL)\n",
    "                ndb_matches = re.findall('\\'NDB_(.+?)\\'', record.id.upper(), re.DOTALL)\n",
    "                stv_names = ['Stv_'+str(stv_id) for stv_id in stv_matches]\n",
    "                ndb_names = ['NDB_'+str(ndb_id) for ndb_id in ndb_matches]\n",
    "                _reg_to_barcode[reg_id] = stv_names+ndb_names\n",
    "        \n",
    "        ## barcode check\n",
    "        _barcode_check = True;\n",
    "        # barcode names\n",
    "        bc_names = [stv.id for stv in stv_barcodes] + [ndb.id for ndb in ndb_barcodes]\n",
    "        # search through previous dictionary\n",
    "        for reg,bcs in sorted(_reg_to_barcode.items()):\n",
    "            for bc in bcs:\n",
    "                if bc not in bc_names:\n",
    "                    print \"-- Wrong barcode name for barcode: \"+str(bc)+\", region: \"+str(reg)\n",
    "                    _barcode_check = False\n",
    "                    break\n",
    "        \n",
    "        return _reg_to_barcode, _barcode_check;\n",
    "        \n",
    "    def _parsing_probe_sequence(record, fwd_primer=fwd_primer, rev_primer=rev_primer,\n",
    "                                add_rand_gap=add_rand_gap, barcode_len=barcode_len, target_len=target_len,\n",
    "                                _RNA=RNA, _RNA_padding=RNA_padding):\n",
    "        '''parse a probe sequence to acquire all barcode binding sites'''\n",
    "        # take in a seq record, parse the sequence and return a list of all included barcodes (20mer,RC)\n",
    "        barcode_list = [];\n",
    "        if RNA:\n",
    "            _main_seq = record.seq[_RNA_padding+len(fwd_primer.seq):-20-_RNA_padding];\n",
    "        else:\n",
    "            _main_seq = record.seq[len(fwd_primer.seq):-20];\n",
    "        \n",
    "        \n",
    "        # trim first 2 barcodes\n",
    "        for i in range(2):\n",
    "            barcode_list.append(_main_seq[:barcode_len]);\n",
    "            _main_seq = _main_seq[(barcode_len+add_rand_gap):];\n",
    "        # trim all barcodes from the end\n",
    "        while len(_main_seq) > target_len:\n",
    "            barcode_list.append(_main_seq[-barcode_len:]);\n",
    "            _main_seq = _main_seq[:-(barcode_len+add_rand_gap)];\n",
    "        \n",
    "        return barcode_list;\n",
    "    \n",
    "    def _finding_barcode_name(barcode_list, stv_barcodes=stv_barcodes, ndb_barcodes=ndb_barcodes, \n",
    "                              barcode_len=barcode_len, total_bc=total_bc):\n",
    "        '''Given barcode list generated by parsing probe, return a list of barcode names'''\n",
    "        _name_list = [];\n",
    "        for bc_site in barcode_list:\n",
    "            for bc in stv_barcodes+ndb_barcodes:\n",
    "                if bc.seq[-barcode_len:] == bc_site.reverse_complement():\n",
    "                    _name_list.append(bc.id);\n",
    "                    break;\n",
    "        \n",
    "        if len(_name_list) < total_bc:\n",
    "            print \"-- Failed in finding some barcodes.\"\n",
    "            print barcode_list, _name_list\n",
    "        return _name_list;\n",
    "    \n",
    "    \n",
    "    def _check_barcode_to_region(reg_to_barcode, \n",
    "                                 pb_records=pb_records, stv_barcodes=stv_barcodes, ndb_barcodes=ndb_barcodes, _RNA=RNA):\n",
    "        '''Generate map from barcode id to region id'''\n",
    "        _barcode_to_reg = {}\n",
    "        _reg_id_exists = []\n",
    "        for record in pb_records:\n",
    "            # region id\n",
    "            if _RNA:\n",
    "                reg_id = record.id.split('gene_')[1].split('_')[0];\n",
    "            else:\n",
    "                reg_id = int(record.id.split('reg_')[1].split('_')[0]);\n",
    "\n",
    "            if reg_id in _reg_id_exists:\n",
    "                continue;\n",
    "            else:\n",
    "                _barcode_list = _parsing_probe_sequence(record)\n",
    "                _name_list = _finding_barcode_name(_barcode_list)\n",
    "                for _n in _name_list:\n",
    "                    if _n not in _barcode_to_reg.keys(): # create if not in dic\n",
    "                        _barcode_to_reg[_n] = [reg_id]\n",
    "                    else: # otherwise, append\n",
    "                        _barcode_to_reg[_n].append(reg_id)\n",
    "            _reg_id_exists.append(reg_id)\n",
    "        ## check region distribution\n",
    "        # invert dic from reg_to_barcode\n",
    "        _inv_dic = {}\n",
    "        for reg,bcs in sorted(reg_to_barcode.items()):\n",
    "            for bc in bcs:\n",
    "                if bc not in _inv_dic.keys():\n",
    "                    _inv_dic[bc] = [reg];\n",
    "                else:\n",
    "                    _inv_dic[bc].append(reg);\n",
    "        # compare\n",
    "        _region_check=True\n",
    "        for bc, regs in sorted(_inv_dic.items()):\n",
    "            if bc not in _barcode_to_reg.keys():\n",
    "                print \"-- \"+str(bc)+\" not in barcode_to_region dic!\"\n",
    "                print '---',bc\n",
    "                _region_check = False\n",
    "                break\n",
    "            else:\n",
    "                for reg in regs:\n",
    "                    if reg not in _barcode_to_reg[bc]:\n",
    "                        print \"-- some barcode in \"+str(regs)+\" not compatible with \"+str(bc)+\" in barcode_to_region dic!\"\n",
    "                        _region_check = False\n",
    "                        break\n",
    "                for reg in _barcode_to_reg[bc]:\n",
    "                    if reg not in regs:\n",
    "                        print \"-- some barcode in \"+str(bc)+\" not compatible with \"+str(regs)+\" in barcode_to_region dic!\"\n",
    "                        _region_check = False\n",
    "                        break\n",
    "                    \n",
    "        return _barcode_to_reg, _region_check\n",
    "    \n",
    "    def _check_barcode_to_color(pb_records=pb_records, stv_barcodes=stv_barcodes, ndb_barcodes=ndb_barcodes, \n",
    "                                stv_color=True, ndb_color=False,\n",
    "                                _save=save, master_dir=master_dir, save_dir=save_dir):\n",
    "        '''If multi_color is applied, generate a barcode_to_color dic for adaptor design'''\n",
    "        if 'color' not in str(pb_records[0].id):\n",
    "            print \"-- color check not applied\";\n",
    "            return False\n",
    "        elif not stv_color and not ndb_color:\n",
    "            print \"-- color check turned off in both stv and ndb\";\n",
    "            return False\n",
    "        else:\n",
    "            # get barcodes\n",
    "            _barcode_names = []\n",
    "            if stv_color: # if stv has multi-color\n",
    "                _barcode_names += [bc.id for bc in stv_barcodes];\n",
    "            if ndb_color: # if ndb has multi-color\n",
    "                _barcode_names += [bc.id for bc in ndb_barcodes];\n",
    "            # initialize color dic\n",
    "            _barcode_to_color = {};\n",
    "            _exist_regs = [];\n",
    "            # search through all probes\n",
    "            for record in pb_records:\n",
    "                # reg_id\n",
    "                if _RNA:\n",
    "                    reg_id = record.id.split('gene_')[1].split('_')[0];\n",
    "                else:\n",
    "                    reg_id = int(record.id.split('reg_')[1].split('_')[0]);\n",
    "                \n",
    "                if _reg_id in _exist_regs:\n",
    "                    continue\n",
    "                else: \n",
    "                    _exist_regs.append(_reg_id);\n",
    "                _color = int(str(record.id).split('color_')[1])\n",
    "                _barcode_list = _parsing_probe_sequence(record)\n",
    "                _name_list = _finding_barcode_name(_barcode_list)\n",
    "                \n",
    "                for _name in _name_list:\n",
    "                    if _name in _barcode_names:\n",
    "                        if _name not in _barcode_to_color.keys():\n",
    "                            _barcode_to_color[_name] = [_color]\n",
    "                        else:\n",
    "                            _barcode_to_color[_name].append(_color);\n",
    "            # keep the unique colors\n",
    "            _barcode_to_unique_color = {}\n",
    "            for k,v in sorted(_barcode_to_color.items()):\n",
    "                _barcode_to_unique_color[k] = np.unique(v)\n",
    "            if _save:\n",
    "                import csv\n",
    "                # mkdir if not exist for this region\n",
    "                if not os.path.exists(master_dir+os.sep+save_dir):\n",
    "                    os.makedirs(master_dir+os.sep+save_dir)\n",
    "                with open(master_dir+os.sep+save_dir+os.sep+'color-usage.csv','w') as output_handle:\n",
    "                    fieldnames = ['barcode', 'color']\n",
    "                    writer = csv.DictWriter(output_handle, fieldnames=fieldnames)\n",
    "                    writer.writeheader()\n",
    "                    for _barcode, _color in sorted(_barcode_to_unique_color.items(), key=lambda (k,v):int(k.split('_')[1])):\n",
    "                        writer.writerow({'barcode': _barcode, 'color': _color})\n",
    "                \n",
    "        return _barcode_to_unique_color\n",
    "                            \n",
    "    \n",
    "    def _construct_internal_map(master_dir=master_dir, save_dir=save_dir, word_size=word_size):\n",
    "        '''Using functions in LibraryDesign, compute an internal khmer map'''\n",
    "        _int_map = khmer.Countgraph(word_size, 1e9, 2) \n",
    "        _int_map.set_use_bigcount(True)\n",
    "        _nms,_seqs = fastaread(master_dir+os.sep+save_dir+os.sep+'candidate_probes.fasta')\n",
    "        for _seq in _seqs:\n",
    "            _int_map.consume(_seq.upper())\n",
    "        return _int_map\n",
    "    \n",
    "    def _check_barcode_in_probes(barcode_to_reg, reg_size_dic, int_map, \n",
    "                                 stv_barcodes=stv_barcodes, ndb_barcodes=ndb_barcodes,\n",
    "                                 barcode_len=barcode_len, max_internal_hits=max_internal_hits):\n",
    "        '''Check barcode appearance in probes, whether that match barcode_to_region scheme'''\n",
    "        _barcode_in_probes = {}\n",
    "        for bc_name, regs in sorted(barcode_to_reg.items()):\n",
    "            bc = None\n",
    "            for _bc in stv_barcodes+ndb_barcodes:\n",
    "                if bc_name == _bc.id:\n",
    "                    bc = _bc\n",
    "                    break\n",
    "            bc_hits = int_map.get_kmer_counts( str(bc.seq[-barcode_len:].reverse_complement()).upper());\n",
    "            if max(bc_hits) - min(bc_hits) > max_internal_hits:\n",
    "                print \"-- Barcode: \"+str(bc)+\" has more off-target in different part of itself!\"\n",
    "                return False\n",
    "            else:\n",
    "                regs,reg_cts = np.unique(regs, return_counts=True);\n",
    "                bc_in_probe = 0;\n",
    "                for reg,ct in zip(regs,reg_cts):\n",
    "                    bc_in_probe += reg_size_dic[reg] * ct;\n",
    "                if max(bc_hits) - bc_in_probe > max_internal_hits:\n",
    "                    print \"-- Barcode: \"+str(bc)+\" has more off-target than threshold!\"\n",
    "                    return False\n",
    "            _barcode_in_probes[bc_name] = bc_in_probe;\n",
    "        return _barcode_in_probes, True\n",
    "    \n",
    "    def _check_between_probes(int_map, pb_lists=pb_lists, pb_records=pb_records):\n",
    "        pass \n",
    "    \n",
    "    def _check_against_genome(pb_records=pb_records, max_genome_hits=max_genome_hits, index_folder=index_folder):\n",
    "        '''Use Khmer to compare probe against genome'''\n",
    "        hg38 = khmer.load_countgraph(index_folder+os.sep+'full_word17_.kmer')\n",
    "        _failed_num = 0;\n",
    "        _keep_pb_records = [];\n",
    "        for record in pb_records:\n",
    "            _kmer_hits = hg38.get_kmer_counts(str(record.seq).upper());\n",
    "            if sum(_kmer_hits) > max_genome_hits:\n",
    "                print '-- Max_genome_hits is: '+str(max_genome_hits)+\", this seq got hits: \"+ str(sum(_kmer_hits))\n",
    "                _failed_num += 1;\n",
    "            else:\n",
    "                _keep_pb_records.append(record);\n",
    "                \n",
    "        return _keep_pb_records, _failed_num # if nothing goes wrong\n",
    "    \n",
    "    def _plot_info():\n",
    "        pass\n",
    "            \n",
    "    ## check primers\n",
    "    primer_usage = _check_primer_usage()\n",
    "    if verbose:\n",
    "        print \"\\n- 1.Passing primer usage check? -\", primer_usage\n",
    "    \n",
    "    ## check region size\n",
    "    reg_size_dic, size_match = _check_region_size(pb_records)\n",
    "    if verbose:\n",
    "        print \"\\n- 2.Passing region size check? -\", size_match    \n",
    "        for k,v in sorted(reg_size_dic.items()):\n",
    "            print k,':',v\n",
    "        \n",
    "    ## check region to barcode\n",
    "    reg_to_barcode, reg2bc = _check_region_to_barcode()\n",
    "    if verbose:\n",
    "        print \"\\n- 3.Passing region to barcode mapping check? -\", reg2bc    \n",
    "        for k,v in sorted(reg_to_barcode.items(), key=lambda (k,v):k):\n",
    "            print k,':',v\n",
    "        \n",
    "    ## check barcode to region (this step must be run after step 3) \n",
    "    barcode_to_reg, bc2reg = _check_barcode_to_region(reg_to_barcode)\n",
    "    if verbose:\n",
    "        print \"\\n- 4.Passing barcode to region mapping check? -\", bc2reg    \n",
    "        for k,v in sorted(barcode_to_reg.items(), key=lambda (k,v):[k[0],int(k.split('_')[1])]):\n",
    "            print k,':',v\n",
    "    \n",
    "    ## check barcode to region (this step must be run after step 3) \n",
    "    barcode_to_color = _check_barcode_to_color()\n",
    "    if verbose and barcode_to_color:\n",
    "        print \"\\n- 5.Calculating barcode to color dictionary.\"\n",
    "        for k,v in sorted(barcode_to_color.items(), key=lambda (k,v):[k[0],int(k.split('_')[1])]):\n",
    "            print k,':',v    \n",
    "    \n",
    "    \n",
    "    ## Construct an internal map\n",
    "    int_map = _construct_internal_map();\n",
    "    if verbose:\n",
    "        print \"\\n- 6.Constructing internal khmer map\";\n",
    "    \n",
    "    ## Check barcodes total counts in probes\n",
    "    barcode_in_probes, _bc_counting = _check_barcode_in_probes(barcode_to_reg, reg_size_dic, int_map)\n",
    "    if verbose:\n",
    "        print \"\\n- 7.Passing if counting barcode appearance times in probes\", _bc_counting;    \n",
    "\n",
    "    ## Check against each other    \n",
    "    \n",
    "    ## Check against genome\n",
    "    kept_records, failed_num = _check_against_genome();\n",
    "    if verbose:\n",
    "        print \"\\n- 8.Probes not passing through genome filter:\", failed_num;  \n",
    "    \n",
    "    # check region size for kept probes\n",
    "    kept_size_dic, kept_match = _check_region_size(kept_records);\n",
    "    if verbose:\n",
    "        print \"\\n- 9.Re-check region size:\"\n",
    "        for k,v in sorted(kept_size_dic.items()):\n",
    "            print k,':',v\n",
    "        print \"--- total number of probes:\", len(kept_records);\n",
    "        \n",
    "    if save:\n",
    "        pb_savefile = master_dir + os.sep + save_dir + os.sep + 'filtered_probes.fasta';\n",
    "        if verbose:\n",
    "            print \"\\n- 10.Saving probes to:\", pb_savefile\n",
    "        with open(pb_savefile, 'w') as output_handle:\n",
    "            SeqIO.write(kept_records, output_handle, 'fasta');  \n",
    "        \n",
    "    return kept_records, kept_size_dic"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- Checking primer usage, total probes: 2095\n",
      "\n",
      "- 1.Passing primer usage check? - True\n",
      "\n",
      "- 2.Passing region size check? - True\n",
      "AlphaCat : 56\n",
      "AlphaVIntegrin : 90\n",
      "BLACAT1 : 47\n",
      "BetaCat : 35\n",
      "CD133 : 47\n",
      "CD24 : 19\n",
      "CD44 : 114\n",
      "CDH1 : 13\n",
      "CDH2 : 47\n",
      "CDH3 : 79\n",
      "DDR2 : 57\n",
      "ERK2 : 60\n",
      "ESRP1 : 54\n",
      "ESRP2 : 68\n",
      "FN1 : 101\n",
      "GRHL2 : 24\n",
      "HOTAIR : 36\n",
      "KRT14 : 15\n",
      "KRT18 : 41\n",
      "MALAT1 : 88\n",
      "MMP2 : 65\n",
      "MMP9 : 45\n",
      "NEAT1 : 98\n",
      "OCLN : 75\n",
      "OVOL2 : 32\n",
      "RBFOX2 : 95\n",
      "SMAD4 : 100\n",
      "SNAI1 : 34\n",
      "SNAI2 : 34\n",
      "SNAI3 : 26\n",
      "SRSF1 : 27\n",
      "TCF3 : 26\n",
      "TWIST1 : 17\n",
      "VIM : 79\n",
      "ZEB1 : 49\n",
      "ZEB2 : 143\n",
      "ZO1 : 59\n",
      "\n",
      "- 3.Passing region to barcode mapping check? - True\n",
      "AlphaCat : ['NDB_301']\n",
      "AlphaVIntegrin : ['NDB_302']\n",
      "BLACAT1 : ['NDB_303']\n",
      "BetaCat : ['NDB_304']\n",
      "CD133 : ['NDB_305']\n",
      "CD24 : ['NDB_306']\n",
      "CD44 : ['NDB_307']\n",
      "CDH1 : ['NDB_308']\n",
      "CDH2 : ['NDB_309']\n",
      "CDH3 : ['NDB_310']\n",
      "DDR2 : ['NDB_311']\n",
      "ERK2 : ['NDB_312']\n",
      "ESRP1 : ['NDB_313']\n",
      "ESRP2 : ['NDB_314']\n",
      "FN1 : ['NDB_315']\n",
      "GRHL2 : ['NDB_316']\n",
      "HOTAIR : ['NDB_317']\n",
      "KRT14 : ['NDB_318']\n",
      "KRT18 : ['NDB_319']\n",
      "MALAT1 : ['NDB_320']\n",
      "MMP2 : ['NDB_321']\n",
      "MMP9 : ['NDB_322']\n",
      "NEAT1 : ['NDB_323']\n",
      "OCLN : ['NDB_324']\n",
      "OVOL2 : ['NDB_325']\n",
      "RBFOX2 : ['NDB_326']\n",
      "SMAD4 : ['NDB_327']\n",
      "SNAI1 : ['NDB_328']\n",
      "SNAI2 : ['NDB_329']\n",
      "SNAI3 : ['NDB_330']\n",
      "SRSF1 : ['NDB_331']\n",
      "TCF3 : ['NDB_332']\n",
      "TWIST1 : ['NDB_333']\n",
      "VIM : ['NDB_334']\n",
      "ZEB1 : ['NDB_335']\n",
      "ZEB2 : ['NDB_336']\n",
      "ZO1 : ['NDB_337']\n",
      "\n",
      "- 4.Passing barcode to region mapping check? - True\n",
      "NDB_301 : ['AlphaCat', 'AlphaCat', 'AlphaCat']\n",
      "NDB_302 : ['AlphaVIntegrin', 'AlphaVIntegrin', 'AlphaVIntegrin']\n",
      "NDB_303 : ['BLACAT1', 'BLACAT1', 'BLACAT1']\n",
      "NDB_304 : ['BetaCat', 'BetaCat', 'BetaCat']\n",
      "NDB_305 : ['CD133', 'CD133', 'CD133']\n",
      "NDB_306 : ['CD24', 'CD24', 'CD24']\n",
      "NDB_307 : ['CD44', 'CD44', 'CD44']\n",
      "NDB_308 : ['CDH1', 'CDH1', 'CDH1']\n",
      "NDB_309 : ['CDH2', 'CDH2', 'CDH2']\n",
      "NDB_310 : ['CDH3', 'CDH3', 'CDH3']\n",
      "NDB_311 : ['DDR2', 'DDR2', 'DDR2']\n",
      "NDB_312 : ['ERK2', 'ERK2', 'ERK2']\n",
      "NDB_313 : ['ESRP1', 'ESRP1', 'ESRP1']\n",
      "NDB_314 : ['ESRP2', 'ESRP2', 'ESRP2']\n",
      "NDB_315 : ['FN1', 'FN1', 'FN1']\n",
      "NDB_316 : ['GRHL2', 'GRHL2', 'GRHL2']\n",
      "NDB_317 : ['HOTAIR', 'HOTAIR', 'HOTAIR']\n",
      "NDB_318 : ['KRT14', 'KRT14', 'KRT14']\n",
      "NDB_319 : ['KRT18', 'KRT18', 'KRT18']\n",
      "NDB_320 : ['MALAT1', 'MALAT1', 'MALAT1']\n",
      "NDB_321 : ['MMP2', 'MMP2', 'MMP2']\n",
      "NDB_322 : ['MMP9', 'MMP9', 'MMP9']\n",
      "NDB_323 : ['NEAT1', 'NEAT1', 'NEAT1']\n",
      "NDB_324 : ['OCLN', 'OCLN', 'OCLN']\n",
      "NDB_325 : ['OVOL2', 'OVOL2', 'OVOL2']\n",
      "NDB_326 : ['RBFOX2', 'RBFOX2', 'RBFOX2']\n",
      "NDB_327 : ['SMAD4', 'SMAD4', 'SMAD4']\n",
      "NDB_328 : ['SNAI1', 'SNAI1', 'SNAI1']\n",
      "NDB_329 : ['SNAI2', 'SNAI2', 'SNAI2']\n",
      "NDB_330 : ['SNAI3', 'SNAI3', 'SNAI3']\n",
      "NDB_331 : ['SRSF1', 'SRSF1', 'SRSF1']\n",
      "NDB_332 : ['TCF3', 'TCF3', 'TCF3']\n",
      "NDB_333 : ['TWIST1', 'TWIST1', 'TWIST1']\n",
      "NDB_334 : ['VIM', 'VIM', 'VIM']\n",
      "NDB_335 : ['ZEB1', 'ZEB1', 'ZEB1']\n",
      "NDB_336 : ['ZEB2', 'ZEB2', 'ZEB2']\n",
      "NDB_337 : ['ZO1', 'ZO1', 'ZO1']\n",
      "-- color check not applied\n",
      "\n",
      "- 6.Constructing internal khmer map\n",
      "\n",
      "- 7.Passing if counting barcode appearance times in probes True\n",
      "-- Max_genome_hits is: 200, this seq got hits: 16203\n",
      "-- Max_genome_hits is: 200, this seq got hits: 539\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1019\n",
      "-- Max_genome_hits is: 200, this seq got hits: 2989\n",
      "-- Max_genome_hits is: 200, this seq got hits: 354\n",
      "-- Max_genome_hits is: 200, this seq got hits: 221\n",
      "-- Max_genome_hits is: 200, this seq got hits: 38299\n",
      "-- Max_genome_hits is: 200, this seq got hits: 468\n",
      "-- Max_genome_hits is: 200, this seq got hits: 327\n",
      "-- Max_genome_hits is: 200, this seq got hits: 237\n",
      "-- Max_genome_hits is: 200, this seq got hits: 464\n",
      "-- Max_genome_hits is: 200, this seq got hits: 218\n",
      "-- Max_genome_hits is: 200, this seq got hits: 276\n",
      "-- Max_genome_hits is: 200, this seq got hits: 494\n",
      "-- Max_genome_hits is: 200, this seq got hits: 11961\n",
      "-- Max_genome_hits is: 200, this seq got hits: 586\n",
      "-- Max_genome_hits is: 200, this seq got hits: 4397\n",
      "-- Max_genome_hits is: 200, this seq got hits: 214\n",
      "-- Max_genome_hits is: 200, this seq got hits: 56462\n",
      "-- Max_genome_hits is: 200, this seq got hits: 2523\n",
      "-- Max_genome_hits is: 200, this seq got hits: 123804\n",
      "-- Max_genome_hits is: 200, this seq got hits: 293\n",
      "-- Max_genome_hits is: 200, this seq got hits: 217\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1407\n",
      "-- Max_genome_hits is: 200, this seq got hits: 398\n",
      "-- Max_genome_hits is: 200, this seq got hits: 218\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1402\n",
      "-- Max_genome_hits is: 200, this seq got hits: 2643\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1405\n",
      "-- Max_genome_hits is: 200, this seq got hits: 2158\n",
      "-- Max_genome_hits is: 200, this seq got hits: 306\n",
      "-- Max_genome_hits is: 200, this seq got hits: 581\n",
      "-- Max_genome_hits is: 200, this seq got hits: 715\n",
      "-- Max_genome_hits is: 200, this seq got hits: 254\n",
      "-- Max_genome_hits is: 200, this seq got hits: 895\n",
      "-- Max_genome_hits is: 200, this seq got hits: 239\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1281\n",
      "-- Max_genome_hits is: 200, this seq got hits: 305\n",
      "-- Max_genome_hits is: 200, this seq got hits: 462\n",
      "-- Max_genome_hits is: 200, this seq got hits: 47335\n",
      "-- Max_genome_hits is: 200, this seq got hits: 3014\n",
      "-- Max_genome_hits is: 200, this seq got hits: 513\n",
      "-- Max_genome_hits is: 200, this seq got hits: 2981\n",
      "-- Max_genome_hits is: 200, this seq got hits: 22140\n",
      "-- Max_genome_hits is: 200, this seq got hits: 8703\n",
      "-- Max_genome_hits is: 200, this seq got hits: 12110\n",
      "-- Max_genome_hits is: 200, this seq got hits: 907\n",
      "-- Max_genome_hits is: 200, this seq got hits: 23642\n",
      "-- Max_genome_hits is: 200, this seq got hits: 30723\n",
      "-- Max_genome_hits is: 200, this seq got hits: 10209\n",
      "-- Max_genome_hits is: 200, this seq got hits: 2338\n",
      "-- Max_genome_hits is: 200, this seq got hits: 363\n",
      "-- Max_genome_hits is: 200, this seq got hits: 270\n",
      "-- Max_genome_hits is: 200, this seq got hits: 208\n",
      "-- Max_genome_hits is: 200, this seq got hits: 332\n",
      "-- Max_genome_hits is: 200, this seq got hits: 209\n",
      "-- Max_genome_hits is: 200, this seq got hits: 373\n",
      "-- Max_genome_hits is: 200, this seq got hits: 332\n",
      "-- Max_genome_hits is: 200, this seq got hits: 320\n",
      "-- Max_genome_hits is: 200, this seq got hits: 312\n",
      "-- Max_genome_hits is: 200, this seq got hits: 336\n",
      "-- Max_genome_hits is: 200, this seq got hits: 413\n",
      "-- Max_genome_hits is: 200, this seq got hits: 885\n",
      "-- Max_genome_hits is: 200, this seq got hits: 29496\n",
      "-- Max_genome_hits is: 200, this seq got hits: 2471\n",
      "-- Max_genome_hits is: 200, this seq got hits: 775\n",
      "-- Max_genome_hits is: 200, this seq got hits: 136819\n",
      "-- Max_genome_hits is: 200, this seq got hits: 15438\n",
      "-- Max_genome_hits is: 200, this seq got hits: 992\n",
      "-- Max_genome_hits is: 200, this seq got hits: 69755\n",
      "-- Max_genome_hits is: 200, this seq got hits: 234\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1978\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1422\n",
      "-- Max_genome_hits is: 200, this seq got hits: 2115\n",
      "-- Max_genome_hits is: 200, this seq got hits: 11002\n",
      "-- Max_genome_hits is: 200, this seq got hits: 344\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1706\n",
      "-- Max_genome_hits is: 200, this seq got hits: 1822\n",
      "-- Max_genome_hits is: 200, this seq got hits: 2691\n",
      "-- Max_genome_hits is: 200, this seq got hits: 420\n",
      "-- Max_genome_hits is: 200, this seq got hits: 2960\n",
      "-- Max_genome_hits is: 200, this seq got hits: 4902\n",
      "-- Max_genome_hits is: 200, this seq got hits: 901\n",
      "\n",
      "- 8.Probes not passing through genome filter: 83\n",
      "region size doesn't match for: AlphaCat\n",
      "\n",
      "- 9.Re-check region size:\n",
      "AlphaCat : 56\n",
      "AlphaVIntegrin : 90\n",
      "BLACAT1 : 47\n",
      "BetaCat : 35\n",
      "CD133 : 47\n",
      "CD24 : 19\n",
      "CD44 : 114\n",
      "CDH1 : 13\n",
      "CDH2 : 47\n",
      "CDH3 : 79\n",
      "DDR2 : 57\n",
      "ERK2 : 60\n",
      "ESRP1 : 54\n",
      "ESRP2 : 68\n",
      "FN1 : 101\n",
      "GRHL2 : 24\n",
      "HOTAIR : 36\n",
      "KRT14 : 15\n",
      "KRT18 : 41\n",
      "MALAT1 : 88\n",
      "MMP2 : 65\n",
      "MMP9 : 45\n",
      "NEAT1 : 98\n",
      "OCLN : 75\n",
      "OVOL2 : 32\n",
      "RBFOX2 : 95\n",
      "SMAD4 : 100\n",
      "SNAI1 : 34\n",
      "SNAI2 : 34\n",
      "SNAI3 : 26\n",
      "SRSF1 : 27\n",
      "TCF3 : 26\n",
      "TWIST1 : 17\n",
      "VIM : 79\n",
      "ZEB1 : 49\n",
      "ZEB2 : 143\n",
      "ZO1 : 59\n",
      "--- total number of probes: 2012\n",
      "\n",
      "- 10.Saving probes to: /n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/CTP-02/EMT_smFISH/final_probes/filtered_probes.fasta\n"
     ]
    }
   ],
   "source": [
    "# master directory\n",
    "master_dir = r'/n/boslfs/LABS/zhuang_lab/User/pzheng/Libraries/CTP-02/EMT_smFISH';\n",
    "\n",
    "kept_records, kept_size_dic = Check_Probes(pb_records, pb_lists, master_dir, target_len=30,\n",
    "                                           RNA=True,RNA_padding=6,\n",
    "                                        fwd_primer=fprimer, rev_primer=rprimer,\n",
    "                                        stv_barcodes=stv_barcodes, ndb_barcodes=ndb_barcodes)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 2",
   "language": "python",
   "name": "python2"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.13"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
