{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "deadly-absence",
   "metadata": {},
   "source": [
    "# Design CTP-10 Aire DNA-MERFISH library for all super enhancers\n",
    "\n",
    "by Pu Zheng\n",
    "\n",
    "2021.4.5\n",
    "\n",
    "Super-enhancers are called by ..."
   ]
  },
  {
   "cell_type": "markdown",
   "id": "delayed-belgium",
   "metadata": {},
   "source": [
    "# Table of contents\n",
    "\n",
    "\n",
    "> 0. [Minimum required packages and settings](#0)\n",
    ">>\n",
    ">> 0.1: [import required packages](#0.1)\n",
    ">\n",
    "> 1. [Extract region sequences](#1)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "fuzzy-reasoning",
   "metadata": {},
   "source": [
    "<a id='0.1'></a>\n",
    "## 0.1 load required packages"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "packed-worse",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "21036\n"
     ]
    }
   ],
   "source": [
    "%run \"..\\..\\Startup_py3.py\"\n",
    "sys.path.append(r\"..\\..\\..\\..\\Documents\")\n",
    "\n",
    "import ImageAnalysis3 as ia\n",
    "%matplotlib notebook\n",
    "\n",
    "from ImageAnalysis3 import *\n",
    "print(os.getpid())\n",
    "\n",
    "# library design specific tools\n",
    "from ImageAnalysis3.library_tools import LibraryDesigner as ld\n",
    "from ImageAnalysis3.library_tools import LibraryTools as lt\n",
    "# biopython imports\n",
    "from Bio import SeqIO\n",
    "from Bio.Seq import Seq\n",
    "from Bio.SeqRecord import SeqRecord\n",
    "from Bio.Blast.Applications import NcbiblastnCommandline\n",
    "from Bio.Blast import NCBIXML"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "severe-obligation",
   "metadata": {},
   "source": [
    "<a id='1'></a>\n",
    "# 1 Extract region sequences"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "single-palmer",
   "metadata": {},
   "outputs": [],
   "source": [
    "## Some folders\n",
    "# human genome\n",
    "reference_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl'\n",
    "genome_folder = os.path.join(reference_folder, 'Genome')\n",
    "# Library directories\n",
    "pool_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "sunset-queen",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- library_folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\n",
      "-- sequence_folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\n",
      "-- report_folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\reports\n"
     ]
    }
   ],
   "source": [
    "resolution = 0\n",
    "flanking = 10000\n",
    "# folder for sub-pool\n",
    "library_folder = os.path.join(pool_folder, f'Genes_TSS_DNA')\n",
    "if not os.path.exists(library_folder):\n",
    "    print(f\"create library folder: {library_folder}\")\n",
    "    os.makedirs(library_folder)\n",
    "# folder for fasta sequences\n",
    "sequence_folder = os.path.join(library_folder, 'sequences')\n",
    "if not os.path.exists(sequence_folder):\n",
    "    print(f\"create sequence folder: {sequence_folder}\")\n",
    "    os.makedirs(sequence_folder)\n",
    "# folder to save result probes\n",
    "report_folder = os.path.join(library_folder, 'reports')\n",
    "if not os.path.exists(report_folder):\n",
    "    print(f\"create report folder: {report_folder}\")\n",
    "    os.makedirs(report_folder)\n",
    "    \n",
    "print(f\"-- library_folder: {library_folder}\")\n",
    "print(f\"-- sequence_folder: {sequence_folder}\")\n",
    "print(f\"-- report_folder: {report_folder}\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "improved-cabinet",
   "metadata": {},
   "source": [
    "## 1.1 load gene list"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "willing-award",
   "metadata": {},
   "outputs": [],
   "source": [
    "gene_list_folder = os.path.join(pool_folder, 'Gene_list')\n",
    "gene_list_filename = os.path.join(gene_list_folder, 'uniqued_clustered_genes_for_yuan_2021-04-22.txt')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "careful-stroke",
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "gene_df = pd.read_csv(gene_list_filename, delimiter = \"\\t\", header=None)\n",
    "gene_df.columns = ['Cluster', 'Gene']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "employed-channel",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Cluster</th>\n",
       "      <th>Gene</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Immature MEC</td>\n",
       "      <td>Ccl21a</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Immature MEC</td>\n",
       "      <td>Krt14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Immature MEC</td>\n",
       "      <td>Krt5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Immature MEC</td>\n",
       "      <td>Col6a1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Immature MEC</td>\n",
       "      <td>Lifr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>204</th>\n",
       "      <td>Aire-stage</td>\n",
       "      <td>Ltf</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>205</th>\n",
       "      <td>Aire-stage</td>\n",
       "      <td>Clps</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>206</th>\n",
       "      <td>Aire-stage</td>\n",
       "      <td>Col1a1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>207</th>\n",
       "      <td>Aire-stage</td>\n",
       "      <td>Gpx6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>208</th>\n",
       "      <td>Aire-stage</td>\n",
       "      <td>Plvap</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>209 rows Ã— 2 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "          Cluster    Gene\n",
       "0    Immature MEC  Ccl21a\n",
       "1    Immature MEC   Krt14\n",
       "2    Immature MEC    Krt5\n",
       "3    Immature MEC  Col6a1\n",
       "4    Immature MEC    Lifr\n",
       "..            ...     ...\n",
       "204    Aire-stage     Ltf\n",
       "205    Aire-stage    Clps\n",
       "206    Aire-stage  Col1a1\n",
       "207    Aire-stage    Gpx6\n",
       "208    Aire-stage   Plvap\n",
       "\n",
       "[209 rows x 2 columns]"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "np.where(gene_df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "id": "registered-involvement",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- loading from save_file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Transcriptome\\Mus_musculus.GRCm38.102.chr.pkl\n",
      "opening ref_file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Transcriptome\\Mus_musculus.GRCm38.102.chr.gff3\n"
     ]
    }
   ],
   "source": [
    "# load gene reference\n",
    "reload(library_tools.references)\n",
    "reload(library_tools.sequences)\n",
    "ref_filename = os.path.join(reference_folder, 'Transcriptome', 'Mus_musculus.GRCm38.102.chr.gff3')\n",
    "with library_tools.references.gff3_reader(ref_filename, auto_read=False, load_savefile=True) as ref_rd:\n",
    "    ## example commands\n",
    "    #infos = _handle.load_all()\n",
    "    #ref_rd._load_headers()\n",
    "    #gene_infos = ref_rd.load_gene_by_id('ENSMUSG00000064842')\n",
    "    #gene_dict = ref_rd.parse_gene_info(gene_infos)\n",
    "    #gene_info_dict = ref_rd._batch_parse_gene_info()\n",
    "    ref_save_dict = ref_rd._save_to_file(overwrite=False)\n",
    "# searching example\n",
    "matched_gene = ref_rd._search_gene_by_id('gene:ENSMUSG00000089613')\n",
    "print(matched_gene[0]['infos']['Name'])\n",
    "matched_gene = ref_rd._search_gene_by_name('Ccl21a')\n",
    "print(matched_gene[0]['infos']['ID'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 60,
   "id": "functional-holiday",
   "metadata": {},
   "outputs": [],
   "source": [
    "from tqdm.notebook import tqdm"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 64,
   "id": "humanitarian-carbon",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "20f3fc377e5a454196dc64b6ada6312d",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "  0%|          | 0/209 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "gene_dicts = []\n",
    "for _gene in tqdm(gene_df['Gene']):\n",
    "    if _gene == 'Lect1': # there is one gene that actually has different name\n",
    "        _gene = \"Cnmd\" \n",
    "    _gds = ref_rd._search_gene_by_name(_gene)\n",
    "    if len(_gds) == 1:\n",
    "        gene_dicts.append(_gds[0])\n",
    "    else:\n",
    "        print(_gene)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 111,
   "id": "valued-completion",
   "metadata": {},
   "outputs": [],
   "source": [
    "# convert reg_dict into reg_start_dict\n",
    "reg_dicts = [library_tools.sequences.gene_dict_2_reg_dict(_gd) for _gd in gene_dicts]\n",
    "tss_dicts = []\n",
    "reg_size = 10000\n",
    "for _rd in reg_dicts:\n",
    "    _tss_d = {_k:_v for _k,_v in _rd.items()}\n",
    "    if _tss_d['Strand'] == '+':\n",
    "        _tss_d[\"Start\"] = int(_rd['Start']) - int(reg_size/2)\n",
    "        _tss_d[\"End\"] = int(_rd['Start']) + int(reg_size/2)\n",
    "        _tss_d['Region'] = f\"{_rd['Chr']}:{_tss_d['Start']}-{_tss_d['End']}\"\n",
    "    else:\n",
    "        _tss_d[\"Start\"] = int(_rd['End']) - int(reg_size/2)\n",
    "        _tss_d[\"End\"] = int(_rd['End']) + int(reg_size/2)\n",
    "        _tss_d['Region'] = f\"{_rd['Chr']}:{_tss_d['Start']}-{_tss_d['End']}\"\n",
    "    # append\n",
    "    tss_dicts.append(_tss_d)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 112,
   "id": "environmental-street",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "({'Chr': '4',\n",
       "  'Start': '42772860',\n",
       "  'End': '42773993',\n",
       "  'Name': 'gene:ENSMUSG00000094686-Ccl21a',\n",
       "  'Gene': 'Ccl21a',\n",
       "  'Region': '4:42772860-42773993',\n",
       "  'Strand': '-'},\n",
       " {'Chr': '4',\n",
       "  'Start': 42768993,\n",
       "  'End': 42778993,\n",
       "  'Name': 'gene:ENSMUSG00000094686-Ccl21a',\n",
       "  'Gene': 'Ccl21a',\n",
       "  'Region': '4:42768993-42778993',\n",
       "  'Strand': '-'})"
      ]
     },
     "execution_count": 112,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "reg_dicts[0], tss_dicts[0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 86,
   "id": "assured-technology",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- load sequence: 1, size=195471971\n",
      "-- load sequence: 10, size=130694993\n",
      "-- load sequence: 11, size=122082543\n",
      "-- load sequence: 12, size=120129022\n",
      "-- load sequence: 13, size=120421639\n",
      "-- load sequence: 14, size=124902244\n",
      "-- load sequence: 15, size=104043685\n",
      "-- load sequence: 16, size=98207768\n",
      "-- load sequence: 17, size=94987271\n",
      "-- load sequence: 18, size=90702639\n",
      "-- load sequence: 19, size=61431566\n",
      "-- load sequence: 2, size=182113224\n",
      "-- load sequence: 3, size=160039680\n",
      "-- load sequence: 4, size=156508116\n",
      "-- load sequence: 5, size=151834684\n",
      "-- load sequence: 6, size=149736546\n",
      "-- load sequence: 7, size=145441459\n",
      "-- load sequence: 8, size=129401213\n",
      "-- load sequence: 9, size=124595110\n",
      "-- load sequence: MT, size=16299\n",
      "-- load sequence: X, size=171031299\n",
      "-- load sequence: Y, size=91744698\n",
      "-- load sequence: JH584299.1, size=953012\n",
      "-- load sequence: GL456233.1, size=336933\n",
      "-- load sequence: JH584301.1, size=259875\n",
      "-- load sequence: GL456211.1, size=241735\n",
      "-- load sequence: GL456350.1, size=227966\n",
      "-- load sequence: JH584293.1, size=207968\n",
      "-- load sequence: GL456221.1, size=206961\n",
      "-- load sequence: JH584297.1, size=205776\n",
      "-- load sequence: JH584296.1, size=199368\n",
      "-- load sequence: GL456354.1, size=195993\n",
      "-- load sequence: JH584294.1, size=191905\n",
      "-- load sequence: JH584298.1, size=184189\n",
      "-- load sequence: JH584300.1, size=182347\n",
      "-- load sequence: GL456219.1, size=175968\n",
      "-- load sequence: GL456210.1, size=169725\n",
      "-- load sequence: JH584303.1, size=158099\n",
      "-- load sequence: JH584302.1, size=155838\n",
      "-- load sequence: GL456212.1, size=153618\n",
      "-- load sequence: JH584304.1, size=114452\n",
      "-- load sequence: GL456379.1, size=72385\n",
      "-- load sequence: GL456216.1, size=66673\n",
      "-- load sequence: GL456393.1, size=55711\n",
      "-- load sequence: GL456366.1, size=47073\n",
      "-- load sequence: GL456367.1, size=42057\n",
      "-- load sequence: GL456239.1, size=40056\n",
      "-- load sequence: GL456213.1, size=39340\n",
      "-- load sequence: GL456383.1, size=38659\n",
      "-- load sequence: GL456385.1, size=35240\n",
      "-- load sequence: GL456360.1, size=31704\n",
      "-- load sequence: GL456378.1, size=31602\n",
      "-- load sequence: GL456389.1, size=28772\n",
      "-- load sequence: GL456372.1, size=28664\n",
      "-- load sequence: GL456370.1, size=26764\n",
      "-- load sequence: GL456381.1, size=25871\n",
      "-- load sequence: GL456387.1, size=24685\n",
      "-- load sequence: GL456390.1, size=24668\n",
      "-- load sequence: GL456394.1, size=24323\n",
      "-- load sequence: GL456392.1, size=23629\n",
      "-- load sequence: GL456382.1, size=23158\n",
      "-- load sequence: GL456359.1, size=22974\n",
      "-- load sequence: GL456396.1, size=21240\n",
      "-- load sequence: GL456368.1, size=20208\n",
      "-- load sequence: JH584292.1, size=14945\n",
      "-- load sequence: JH584295.1, size=1976\n",
      "- 66 sequences loaded.\n"
     ]
    }
   ],
   "source": [
    "reload(library_tools.references)\n",
    "reload(library_tools.sequences)\n",
    "seq_rd = library_tools.sequences.RNA_sequence_reader(genome_folder, flanking=flanking)\n",
    "seq_rd.load_sequences()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 113,
   "id": "fluid-ukraine",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- searching among 1 references\n",
      "-- a match found in record: 4.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 15.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 10.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 15.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 2.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 8.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 10.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 6.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 17.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 16.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 3.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 17.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 10.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 5.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 2.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 13.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 8.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 9.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 5.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 9.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 9.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 1.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 14.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 6.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 17.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 15.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 12.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 4.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 1.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 16.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 4.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 15.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 9.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 2.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 9.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 2.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 3.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 10.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 12.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 6.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 4.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 16.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 18.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 9.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 14.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 9.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 3.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 2.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 15.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 9.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 1.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 19.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 14.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 1.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 18.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 12.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 9.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 4.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 2.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 4.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 10.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 3.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 17.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 15.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 4.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 14.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 4.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 16.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 1.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 19.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 3.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 19.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 2.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 12.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 12.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 1.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 17.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 13.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 3.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 2.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 19.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 9.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 5.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 8.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 12.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 17.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 16.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 2.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 9.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 15.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 10.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 3.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 16.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 5.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 10.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 16.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 8.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 15.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 17.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 3.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 16.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 4.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 6.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 4.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 9.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 9.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 12.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 2.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 9.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 15.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 6.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 6.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 2.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 1.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 15.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 5.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 4.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 3.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 3.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 15.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 6.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 8.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 15.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 17.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 17.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 3.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 5.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 12.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 1.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 13.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 2.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: X.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 9.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 14.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 8.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 4.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 2.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 4.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 15.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 14.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 14.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 6.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 1.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 6.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 3.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 4.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 8.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 14.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 2.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 3.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 6.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 14.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 5.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 15.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 9.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 17.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 13.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 8.\n",
      "209\n"
     ]
    }
   ],
   "source": [
    "for _tss_d in tss_dicts:\n",
    "    seq_rd.find_sequence_for_region(_tss_d)\n",
    "print(len(seq_rd.seq_dict))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 116,
   "id": "revolutionary-niagara",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- saving sequences into folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ccl21a_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt14_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt5_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Col6a1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Lifr_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Itga6_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Col4a1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ascl1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cald1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt17_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\C3_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Trp63_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Dclk1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Trpm5_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ly6g6f_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Avil_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Alox5ap_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Hck_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Plk2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Pik3r5_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Agt_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Nrgn_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Plac8_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Fxyd6_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Pou2f3_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ccl20_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ccl9_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ccl6_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Rnase1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gp2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Fabp1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Sox8_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Spib_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Fxyd3_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Tnfrsf11b_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ifi27l2b_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Guca2a_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Pigr_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Muc13_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Saa1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Aldob_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Lgals2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gsta2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Lcn2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Apoa1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Hnf4a_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Hnf4g_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Igf1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Grhl1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Lypd3_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krtdap_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Tacstd2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt16_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Pdzk1ip1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Tmem45a_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt10_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gm94_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Acpp_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cnfn_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Fam25c_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Elovl4_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Efna3_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cebpb_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ly6d_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gsta4_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Serpinb11_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gabrp_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cyp2f2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gsto1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Anxa8_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Mlph_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cd14_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Foxa1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ttc36_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Foxj1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Fam183b_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ccdc30_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Meig1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Spag8_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Vpreb3_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Riiad1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Rsph1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Odf3b_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Car8_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Rnase4_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Tspan1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Stxbp5l_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Igfbp5_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cpn1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Rap1gap2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Celf3_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Syt7_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gpx3_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Foxa3_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Foxa2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ckb_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Meg3_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Myog_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ckm_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Jpt1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Mymx_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Dbn1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\S100a9_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Pyy_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Fabp6_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Slpi_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gal_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Crabp1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Spp1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Mt4_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Pomc_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Tff3_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Sst_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ptgds_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cck_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Psca_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Tac2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gstm3_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Pcp4_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Pla2g1b_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Nts_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Apod_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ctrb1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Lypd2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Pgc_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Rarres1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Retnla_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ins2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cga_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Iapp_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Nppa_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Camp_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cyp1a2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Dlk1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gm13539_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt13_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cilp_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Igfbp6_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Apoc2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Sparc_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Tac1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Npy_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Saa2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Mpo_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gatm_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Xcl1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\H19_reg_1.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ly6c2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cxcl13_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Penk_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ecm1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Sprr1b_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ly6c1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Hbb-y_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ccl1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Klk7_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Dppa3_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Hba-a2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Mt3_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt71_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cryaa_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Tff2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Timp2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Klk14_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Fgg_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Oas1d_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Rsad2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Apoa2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Nefh_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Idi2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Olfm1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Bgn_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ngp_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Car4_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Lgals3_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ccl2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Hp_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\C1qb_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Mal_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Alpl_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt77_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cnmd_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ang4_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt19_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Mgp_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Igf2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Alpi_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ghrl_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Hamp2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gm2a_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt9_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Reg4_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Isg15_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Calb2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Zg16_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Aldh3a1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Sftpd_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Lcn3_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Sprr2i_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Reg3b_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Sftpa1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Oas1h_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt76_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gsdma_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ltf_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Clps_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Col1a1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gpx6_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Plvap_reg_1.fasta\n"
     ]
    }
   ],
   "source": [
    "seq_rd.save_sequences(sequence_folder)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "infinite-musical",
   "metadata": {},
   "source": [
    "<a id='2'></a>\n",
    "# 2. Design probe targeting sequences by probe_designer\n",
    "\n",
    "run probe_designer, remeber to clear the memory usage because each table should take ~32GB"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "acting-religion",
   "metadata": {},
   "source": [
    "<a id='2.1'></a>\n",
    "## 2.1 Construct count table with all the 17-mers in the genome\n",
    "\n",
    "Only do this if you don't have pre-built 17-mer\n",
    "\n",
    "However you can do almost the same thing for your own library during quality check\n",
    "\n",
    "This library requires mm10 genome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 117,
   "id": "greatest-architecture",
   "metadata": {},
   "outputs": [],
   "source": [
    "overwrite_table = False"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "hollow-turner",
   "metadata": {},
   "source": [
    "### construct map for whole genome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 118,
   "id": "close-mount",
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "reload(library_tools.design)\n",
    " \n",
    "genome_table_file = os.path.join(reference_folder, 'GRCm38_genome_17w.npy')\n",
    "\n",
    "if not os.path.exists(genome_table_file) or overwrite_table:\n",
    "    # genome\n",
    "    _genome_filenames = [os.path.join(genome_folder, _fl) \n",
    "         for _fl in os.listdir(genome_folder) \n",
    "         if _fl.split(os.extsep)[-1]=='fasta' or _fl.split(os.extsep)[-1]=='fa']\n",
    "    print(len(_genome_filenames))\n",
    "\n",
    "    ct = library_tools.design.countTable(word=17,save_file=genome_table_file, \n",
    "                       sparse=False)\n",
    "    ct.verbose=True\n",
    "\n",
    "    ct.read(_genome_filenames) # read sequences from fasta files\n",
    "\n",
    "    ct.consume_loaded(num_threads=24) # convert sequences into integers\n",
    "\n",
    "    ct.complete(verbose=True)\n",
    "\n",
    "    ct.save()\n",
    "\n",
    "    # clear RAM if contructed countable \n",
    "    del(ct)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "immune-strike",
   "metadata": {},
   "source": [
    "### construct map for transcriptome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 119,
   "id": "opponent-explorer",
   "metadata": {},
   "outputs": [],
   "source": [
    "from tqdm import tqdm\n",
    "# transcriptome\n",
    "transcriptome_folder = os.path.join(reference_folder, 'Transcriptome')\n",
    "\n",
    "transcriptome_table_file = os.path.join(reference_folder, 'GRCm38_transcriptome_17w.npy')\n",
    "\n",
    "if not os.path.exists(transcriptome_table_file) or overwrite_table:\n",
    "    # transcriptome\n",
    "    _transcriptome_filenames = [os.path.join(transcriptome_folder, _fl) \n",
    "         for _fl in os.listdir(transcriptome_folder) \n",
    "         if _fl.split(os.extsep)[-1]=='fasta' or _fl.split(os.extsep)[-1]=='fa']\n",
    "    print(len(_transcriptome_filenames))\n",
    "    \n",
    "    ct = library_tools.design.countTable(word=17,save_file=transcriptome_table_file, \n",
    "                       sparse=False)\n",
    "    ct.verbose=True\n",
    "\n",
    "    ct.read(_transcriptome_filenames) # read sequences from fasta files\n",
    "\n",
    "    ct.consume_loaded(num_threads=24) # convert sequences into integers\n",
    "\n",
    "    ct.complete(verbose=True)\n",
    "\n",
    "    ct.save()\n",
    "    \n",
    "    # clear RAM if contructed countable \n",
    "    del(ct)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "psychological-router",
   "metadata": {},
   "source": [
    "### construct map for repeats from RepBase"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 120,
   "id": "statistical-killer",
   "metadata": {},
   "outputs": [],
   "source": [
    "from tqdm import tqdm\n",
    "# repeat\n",
    "repeat_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\Repbase'\n",
    "\n",
    "repeat_table_file = os.path.join(reference_folder, 'Repbase_v2603_repeat_17w.npy')\n",
    "\n",
    "if not os.path.exists(repeat_table_file) or overwrite_table:\n",
    "    # repeat\n",
    "    _repeat_filenames = [os.path.join(repeat_folder, _fl) \n",
    "         for _fl in os.listdir(repeat_folder) \n",
    "         if _fl.split(os.extsep)[-1]=='fasta' or _fl.split(os.extsep)[-1]=='fa']\n",
    "    print(len(_repeat_filenames))\n",
    "    \n",
    "    ct = library_tools.design.countTable(word=17,save_file=repeat_table_file, \n",
    "                       sparse=False)\n",
    "    ct.verbose=True\n",
    "\n",
    "    ct.read(_repeat_filenames) # read sequences from fasta files\n",
    "\n",
    "    ct.consume_loaded(num_threads=24) # convert sequences into integers\n",
    "\n",
    "    ct.complete(verbose=True)\n",
    "\n",
    "    ct.save()\n",
    "    \n",
    "    # clear RAM if contructed countable \n",
    "    del(ct)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "accomplished-transparency",
   "metadata": {},
   "source": [
    "<a id='2.2'></a>\n",
    "## 2.2 Design probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 121,
   "id": "binary-berlin",
   "metadata": {},
   "outputs": [],
   "source": [
    "# required parameters\n",
    "resolution = 0\n",
    "\n",
    "## required folders\n",
    "## Some folders\n",
    "# human genome\n",
    "reference_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl'\n",
    "genome_folder = os.path.join(reference_folder, 'Genome')\n",
    "# Library directories\n",
    "pool_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire'\n",
    "\n",
    "library_folder = os.path.join(pool_folder, f'Genes_TSS_DNA')\n",
    "sequence_folder = os.path.join(library_folder, 'sequences')\n",
    "report_folder = os.path.join(library_folder, 'reports')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 122,
   "id": "beautiful-bookmark",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "209 regions loaded to design probes.\n",
      "\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\reports\\merged_probes.pbr\n"
     ]
    }
   ],
   "source": [
    "# requires pre_defined genome_folder and library_folder\n",
    "# Indices\n",
    "genome_index = os.path.join(reference_folder, 'GRCm38_genome_17w.npy')\n",
    "transcriptome_index = os.path.join(reference_folder, 'GRCm38_transcriptome_17w.npy') \n",
    "repeat_index = os.path.join(reference_folder, 'Repbase_v2603_repeat_17w.npy')\n",
    "#ref_merfish_index = os.path.join(reference_folder, 'M1_meng_MERFISH_17w.npy') # merfish designed by Meng\n",
    "# get input files \n",
    "input_files = glob.glob(os.path.join(sequence_folder, '*.fasta'))\n",
    "\n",
    "print(f\"{len(input_files)} regions loaded to design probes.\")\n",
    "\n",
    "if not os.path.exists(report_folder):\n",
    "    os.makedirs(report_folder)\n",
    "    \n",
    "# filename to save probe reports\n",
    "save_file = os.path.join(report_folder, f'merged_probes.pbr')\n",
    "print(save_file)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "forbidden-provider",
   "metadata": {},
   "source": [
    "### create pb_designer class"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 123,
   "id": "massive-yield",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Probe designer derived from Bogdan Bintu:\n",
      "https://github.com/BogdanBintu/ChromatinImaging/blob/master/LibraryDesign/LibraryDesigner.py\n",
      "by Pu Zheng, 2020.11\n",
      "\n",
      "Major changes:\n",
      "    1. allow design of two strands\n",
      "    2. separate reverse_complement (rev_com) and from two strands (two_stranded) as \n",
      "    two different inputs for map_dic and sequence_dic\n",
      "    3. replace 'local_genome' with 'self_sequences' to be more explicit, and only \n",
      "    exclude the counts for the corresponding self_sequence within each input. \n",
      "\n",
      "Key information:\n",
      "    - number of input_sequence(s): 209\n",
      "    - save_file location: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\reports\\merged_probes.pbr\n",
      "\n",
      "- Fail to load from savefile: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\reports\\merged_probes.pbr, file doesn't exist.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "False"
      ]
     },
     "execution_count": 123,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "reload(library_tools)\n",
    "reload(library_tools.design)\n",
    "\n",
    "pb_designer = library_tools.design.pb_reports_class(\n",
    "    sequence_dic={'file':input_files,\n",
    "                  'rev_com':True, # design two strands\n",
    "                  'two_stranded':True},\n",
    "    map_dic={'genome':{'file':genome_index,'rev_com':False,'two_stranded':True},\n",
    "             'transcriptome':{'file':transcriptome_index,'rev_com':True,'two_stranded':False},\n",
    "             'rep_genome':{'file':repeat_index,'rev_com':False,'two_stranded':True},\n",
    "             'self_sequences':{'file':input_files,'force_list':True,'rev_com':False,'two_stranded':True},\n",
    "             #'ref_merfish':{'file':ref_merfish_index,'rev_com':False,'two_stranded':True},\n",
    "             },\n",
    "        save_file=save_file,\n",
    "    params_dic={'word_size':17,'pb_len':40,'buffer_len':1,'max_count':2**16-1,\n",
    "                'check_on_go': False, # whether automatically check probes\n",
    "                'auto': False, # whether automatically convert reference maps\n",
    "               },\n",
    "    check_dic={('genome','self_sequences'): 25,\n",
    "               'rep_genome': 0,\n",
    "               'transcriptome': 14,\n",
    "               #'ref_merfish': 14,\n",
    "               'gc':[0.25,0.85],'tm': 70,\n",
    "               }\n",
    "    )\n",
    "print(pb_designer)\n",
    "pb_designer.load_from_file(load_probes_only=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "incoming-texture",
   "metadata": {},
   "source": [
    "### calculate probe reports"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 124,
   "id": "weird-starter",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- setting attribute: map_genome\n",
      "--- finish map_genome in 89.868s.\n",
      "-- setting attribute: map_transcriptome\n",
      "--- finish map_transcriptome in 93.490s.\n",
      "-- setting attribute: map_rep_genome\n",
      "--- finish map_rep_genome in 88.290s.\n",
      "Time(s): 271.6564977169037\n",
      "- Designing targeting sequence for 209 regions\n",
      "-- designing region: 9:104322748-104352748_reg_Acpp:- 0, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Acpp_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.118s.\n",
      "in 15.720s.\n",
      "-- designing region: 8:124554706-124584706_reg_Agt:- 1, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Agt_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.150s.\n",
      "in 16.387s.\n",
      "-- designing region: 11:61192537-61222537_reg_Aldh3a1:+ 2, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Aldh3a1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.127s.\n",
      "in 16.249s.\n",
      "-- designing region: 4:49534546-49564546_reg_Aldob:- 3, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Aldob_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.124s.\n",
      "in 16.324s.\n",
      "-- designing region: 5:149249767-149279767_reg_Alox5ap:+ 4, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Alox5ap_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.118s.\n",
      "in 16.306s.\n",
      "-- designing region: 1:87086606-87116606_reg_Alpi:- 5, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Alpi_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.129s.\n",
      "in 16.445s.\n",
      "-- designing region: 4:137781384-137811384_reg_Alpl:- 6, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Alpl_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.130s.\n",
      "in 16.360s.\n",
      "-- designing region: 14:51758590-51788590_reg_Ang4:- 7, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ang4_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.135s.\n",
      "in 16.330s.\n",
      "-- designing region: 14:34070981-34100981_reg_Anxa8:+ 8, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Anxa8_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.334s.\n",
      "-- designing region: 9:46213580-46243580_reg_Apoa1:+ 9, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Apoa1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.118s.\n",
      "in 16.362s.\n",
      "-- designing region: 1:171210054-171240054_reg_Apoa2:+ 10, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Apoa2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.434s.\n",
      "-- designing region: 7:19662941-19692941_reg_Apoc2:- 11, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Apoc2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.122s.\n",
      "in 16.445s.\n",
      "-- designing region: 16:31299808-31329808_reg_Apod:- 12, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Apod_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.162s.\n",
      "in 16.400s.\n",
      "-- designing region: 10:87478660-87508660_reg_Ascl1:- 13, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ascl1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.218s.\n",
      "-- designing region: 10:126985709-127015709_reg_Avil:+ 14, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Avil_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.126s.\n",
      "in 16.403s.\n",
      "-- designing region: X:73468602-73498602_reg_Bgn:+ 15, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Bgn_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.128s.\n",
      "in 16.301s.\n",
      "-- designing region: 4:136871187-136901187_reg_C1qb:- 16, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\C1qb_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.204s.\n",
      "in 16.377s.\n",
      "-- designing region: 17:57213136-57243136_reg_C3:- 17, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\C3_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.132s.\n",
      "in 16.366s.\n",
      "-- designing region: 8:110153210-110183210_reg_Calb2:- 18, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Calb2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.145s.\n",
      "in 16.364s.\n",
      "-- designing region: 6:34583500-34613500_reg_Cald1:+ 19, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cald1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.141s.\n",
      "in 16.276s.\n",
      "-- designing region: 9:109834617-109864617_reg_Camp:- 20, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Camp_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.149s.\n",
      "in 16.278s.\n",
      "-- designing region: 11:84942786-84972786_reg_Car4:+ 21, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Car4_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.371s.\n",
      "-- designing region: 4:8224041-8254041_reg_Car8:- 22, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Car8_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.258s.\n",
      "-- designing region: 4:119400521-119430521_reg_Ccdc30:- 23, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ccdc30_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.433s.\n",
      "-- designing region: 9:121480694-121510694_reg_Cck:- 24, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cck_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.118s.\n",
      "in 16.252s.\n",
      "-- designing region: 11:82290690-82320690_reg_Ccl1:- 25, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ccl1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.133s.\n",
      "in 16.286s.\n",
      "-- designing region: 1:83101766-83131766_reg_Ccl20:+ 26, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ccl20_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "in 16.180s.\n",
      "-- designing region: 4:42758993-42788993_reg_Ccl21a:- 27, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ccl21a_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.118s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "in 16.367s.\n",
      "-- designing region: 11:82020571-82050571_reg_Ccl2:+ 28, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ccl2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.118s.\n",
      "in 16.342s.\n",
      "-- designing region: 11:83578087-83608087_reg_Ccl6:- 29, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ccl6_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.126s.\n",
      "in 16.296s.\n",
      "-- designing region: 11:83563636-83593636_reg_Ccl9:- 30, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ccl9_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.137s.\n",
      "in 16.400s.\n",
      "-- designing region: 18:36711798-36741798_reg_Cd14:- 31, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cd14_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.308s.\n",
      "-- designing region: 2:167673915-167703915_reg_Cebpb:+ 32, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cebpb_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.133s.\n",
      "in 16.306s.\n",
      "-- designing region: 3:94463295-94493295_reg_Celf3:+ 33, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Celf3_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.129s.\n",
      "in 16.262s.\n",
      "-- designing region: 4:34878779-34908779_reg_Cga:+ 34, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cga_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.272s.\n",
      "-- designing region: 9:65250180-65280180_reg_Cilp:+ 35, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cilp_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.124s.\n",
      "in 16.246s.\n",
      "-- designing region: 12:111657338-111687338_reg_Ckb:- 36, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ckb_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.347s.\n",
      "-- designing region: 7:19389776-19419776_reg_Ckm:+ 37, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ckm_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.118s.\n",
      "in 16.400s.\n",
      "-- designing region: 17:28545766-28575766_reg_Clps:- 38, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Clps_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.138s.\n",
      "in 16.363s.\n",
      "-- designing region: 7:25354724-25384724_reg_Cnfn:- 39, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cnfn_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "in 16.404s.\n",
      "-- designing region: 14:79647170-79677170_reg_Cnmd:- 40, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cnmd_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "in 16.350s.\n",
      "-- designing region: 11:94921224-94951224_reg_Col1a1:+ 41, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Col1a1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.139s.\n",
      "in 16.375s.\n",
      "-- designing region: 8:11297826-11327826_reg_Col4a1:- 42, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Col4a1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.120s.\n",
      "in 16.363s.\n",
      "-- designing region: 10:76711168-76741168_reg_Col6a1:- 43, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Col6a1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.128s.\n",
      "in 16.364s.\n",
      "-- designing region: 19:43971556-44001556_reg_Cpn1:- 44, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cpn1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.136s.\n",
      "in 16.374s.\n",
      "-- designing region: 9:54749748-54779748_reg_Crabp1:+ 45, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Crabp1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "in 16.430s.\n",
      "-- designing region: 17:31662807-31692807_reg_Cryaa:+ 46, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cryaa_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.158s.\n",
      "in 16.331s.\n",
      "-- designing region: 8:111676010-111706010_reg_Ctrb1:- 47, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ctrb1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.136s.\n",
      "in 16.521s.\n",
      "-- designing region: 5:95941951-95971951_reg_Cxcl13:+ 48, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cxcl13_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.140s.\n",
      "in 16.273s.\n",
      "-- designing region: 9:57668703-57698703_reg_Cyp1a2:- 49, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cyp1a2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.144s.\n",
      "in 16.392s.\n",
      "-- designing region: 7:27104909-27134909_reg_Cyp2f2:+ 50, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cyp2f2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.120s.\n",
      "in 16.297s.\n",
      "-- designing region: 13:55473111-55503111_reg_Dbn1:- 51, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Dbn1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.136s.\n",
      "in 24.623s.\n",
      "-- designing region: 3:55227364-55257364_reg_Dclk1:+ 52, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Dclk1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.133s.\n",
      "in 16.305s.\n",
      "-- designing region: 12:109437823-109467823_reg_Dlk1:+ 53, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Dlk1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.137s.\n",
      "in 16.350s.\n",
      "-- designing region: 6:122611410-122641410_reg_Dppa3:+ 54, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Dppa3_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.129s.\n",
      "in 16.238s.\n",
      "-- designing region: 3:95724569-95754569_reg_Ecm1:- 55, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ecm1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.251s.\n",
      "-- designing region: 3:89307965-89337965_reg_Efna3:- 56, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Efna3_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.135s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "in 16.232s.\n",
      "-- designing region: 9:83791277-83821277_reg_Elovl4:- 57, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Elovl4_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.259s.\n",
      "-- designing region: 6:71184827-71214827_reg_Fabp1:+ 58, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Fabp1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.150s.\n",
      "in 16.358s.\n",
      "-- designing region: 11:43586540-43616540_reg_Fabp6:- 59, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Fabp6_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.137s.\n",
      "in 16.270s.\n",
      "-- designing region: 11:58786960-58816960_reg_Fam183b:- 60, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Fam183b_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "in 16.211s.\n",
      "-- designing region: 14:34340433-34370433_reg_Fam25c:- 61, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Fam25c_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.420s.\n",
      "-- designing region: 3:82992724-83022724_reg_Fgg:+ 62, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Fgg_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "in 16.350s.\n",
      "-- designing region: 12:57531916-57561916_reg_Foxa1:- 63, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Foxa1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.310s.\n",
      "-- designing region: 2:148031969-148061969_reg_Foxa2:- 64, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Foxa2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.130s.\n",
      "in 16.340s.\n",
      "-- designing region: 7:19008538-19038538_reg_Foxa3:- 65, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Foxa3_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.149s.\n",
      "in 16.450s.\n",
      "-- designing region: 11:116320399-116350399_reg_Foxj1:- 66, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Foxj1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "in 16.396s.\n",
      "-- designing region: 7:31061704-31091704_reg_Fxyd3:- 67, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Fxyd3_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.119s.\n",
      "in 16.383s.\n",
      "-- designing region: 9:45355185-45385185_reg_Fxyd6:+ 68, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Fxyd6_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.130s.\n",
      "in 16.271s.\n",
      "-- designing region: 11:33563959-33593959_reg_Gabrp:- 69, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gabrp_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "in 16.236s.\n",
      "-- designing region: 19:3399544-3429544_reg_Gal:- 70, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gal_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "in 16.239s.\n",
      "-- designing region: 2:122596303-122626303_reg_Gatm:- 71, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gatm_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.244s.\n",
      "-- designing region: 6:113704880-113734880_reg_Ghrl:- 72, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ghrl_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.133s.\n",
      "in 16.269s.\n",
      "-- designing region: 2:25767257-25797257_reg_Gm13539:+ 73, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gm13539_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.137s.\n",
      "in 16.242s.\n",
      "-- designing region: 11:55083115-55113115_reg_Gm2a:+ 74, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gm2a_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "in 16.242s.\n",
      "-- designing region: 18:43777878-43807878_reg_Gm94:- 75, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gm94_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "in 16.189s.\n",
      "-- designing region: 7:119444285-119474285_reg_Gp2:- 76, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gp2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.128s.\n",
      "in 16.220s.\n",
      "-- designing region: 11:54887453-54917453_reg_Gpx3:+ 77, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gpx3_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "in 16.304s.\n",
      "-- designing region: 13:21297125-21327125_reg_Gpx6:+ 78, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gpx6_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.291s.\n",
      "-- designing region: 12:24557283-24587283_reg_Grhl1:+ 79, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Grhl1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.328s.\n",
      "-- designing region: 11:98649351-98679351_reg_Gsdma:+ 80, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gsdma_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.344s.\n",
      "-- designing region: 9:78340788-78370788_reg_Gsta2:- 81, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gsta2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.122s.\n",
      "in 16.296s.\n",
      "-- designing region: 9:78167774-78197774_reg_Gsta4:+ 82, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gsta4_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.137s.\n",
      "in 16.300s.\n",
      "-- designing region: 3:107954283-107984283_reg_Gstm3:- 83, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gstm3_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.118s.\n",
      "in 16.351s.\n",
      "-- designing region: 19:47839970-47869970_reg_Gsto1:+ 84, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gsto1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.140s.\n",
      "in 16.368s.\n",
      "-- designing region: 4:119622704-119652704_reg_Guca2a:+ 85, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Guca2a_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "in 16.247s.\n",
      "-- designing region: 7:142563143-142593143_reg_H19:- 86, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\H19_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.133s.\n",
      "in 16.397s.\n",
      "-- designing region: 7:30909681-30939681_reg_Hamp2:- 87, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Hamp2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.135s.\n",
      "in 16.415s.\n",
      "-- designing region: 11:32281489-32311489_reg_Hba-a2:+ 88, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Hba-a2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.127s.\n",
      "in 16.322s.\n",
      "-- designing region: 7:103838216-103868216_reg_Hbb-y:- 89, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Hbb-y_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.374s.\n",
      "-- designing region: 2:153093468-153123468_reg_Hck:+ 90, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Hck_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.125s.\n",
      "in 16.289s.\n",
      "-- designing region: 2:163491808-163521808_reg_Hnf4a:+ 91, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Hnf4a_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.350s.\n",
      "-- designing region: 3:3493030-3523030_reg_Hnf4g:+ 92, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Hnf4g_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.132s.\n",
      "in 16.312s.\n",
      "-- designing region: 8:109564172-109594172_reg_Hp:- 93, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Hp_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.128s.\n",
      "in 16.385s.\n",
      "-- designing region: 6:142283423-142313423_reg_Iapp:+ 94, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Iapp_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "in 16.397s.\n",
      "-- designing region: 13:8937863-8967863_reg_Idi2:+ 95, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Idi2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.138s.\n",
      "in 16.766s.\n",
      "-- designing region: 12:103442223-103472223_reg_Ifi27l2b:- 96, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ifi27l2b_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.129s.\n",
      "in 16.351s.\n",
      "-- designing region: 10:87843265-87873265_reg_Igf1:+ 97, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Igf1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.118s.\n",
      "in 16.320s.\n",
      "-- designing region: 7:142651816-142681816_reg_Igf2:- 98, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Igf2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.151s.\n",
      "in 16.437s.\n",
      "-- designing region: 1:72859884-72889884_reg_Igfbp5:- 99, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Igfbp5_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.139s.\n",
      "in 16.378s.\n",
      "-- designing region: 15:102129362-102159362_reg_Igfbp6:+ 100, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Igfbp6_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.148s.\n",
      "in 16.377s.\n",
      "-- designing region: 7:142728381-142758381_reg_Ins2:- 101, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ins2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.127s.\n",
      "in 26.236s.\n",
      "-- designing region: 4:156185796-156215796_reg_Isg15:- 102, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Isg15_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.321s.\n",
      "-- designing region: 2:71730616-71760616_reg_Itga6:+ 103, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Itga6_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.130s.\n",
      "in 16.306s.\n",
      "-- designing region: 11:115499387-115529387_reg_Jpt1:- 104, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Jpt1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.140s.\n",
      "in 16.272s.\n",
      "-- designing region: 7:43675418-43705418_reg_Klk14:+ 105, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Klk14_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "in 16.307s.\n",
      "-- designing region: 7:43796294-43826294_reg_Klk7:+ 106, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Klk7_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.129s.\n",
      "in 16.806s.\n",
      "-- designing region: 11:99374364-99404364_reg_Krt10:- 107, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt10_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.142s.\n",
      "in 17.567s.\n",
      "-- designing region: 11:100106566-100136566_reg_Krt13:- 108, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt13_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.140s.\n",
      "in 16.512s.\n",
      "-- designing region: 11:100192548-100222548_reg_Krt14:- 109, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt14_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.133s.\n",
      "in 16.423s.\n",
      "-- designing region: 11:100233902-100263902_reg_Krt16:- 110, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt16_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.137s.\n",
      "in 16.462s.\n",
      "-- designing region: 11:100246029-100276029_reg_Krt17:- 111, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt17_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.135s.\n",
      "in 16.467s.\n",
      "-- designing region: 11:100133665-100163665_reg_Krt19:- 112, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt19_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "in 16.530s.\n",
      "-- designing region: 15:101697898-101727898_reg_Krt5:- 113, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt5_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.123s.\n",
      "in 16.357s.\n",
      "-- designing region: 15:101728109-101758109_reg_Krt71:- 114, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt71_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.136s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "in 16.427s.\n",
      "-- designing region: 15:101877920-101907920_reg_Krt76:- 115, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt76_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.118s.\n",
      "in 16.401s.\n",
      "-- designing region: 15:101854705-101884705_reg_Krt77:- 116, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt77_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.517s.\n",
      "-- designing region: 11:100178246-100208246_reg_Krt9:- 117, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt9_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "in 16.596s.\n",
      "-- designing region: 7:30772896-30802896_reg_Krtdap:+ 118, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krtdap_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.479s.\n",
      "-- designing region: 2:32373252-32403252_reg_Lcn2:- 119, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Lcn2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.133s.\n",
      "in 16.490s.\n",
      "-- designing region: 2:25750569-25780569_reg_Lcn3:+ 120, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Lcn3_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.127s.\n",
      "in 16.477s.\n",
      "-- designing region: 15:78840529-78870529_reg_Lgals2:- 121, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Lgals2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.387s.\n",
      "-- designing region: 14:47352751-47382751_reg_Lgals3:+ 122, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Lgals3_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "in 16.499s.\n",
      "-- designing region: 15:7075614-7105614_reg_Lifr:+ 123, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Lifr_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.140s.\n",
      "in 16.555s.\n",
      "-- designing region: 9:111004271-111034271_reg_Ltf:+ 124, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ltf_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.128s.\n",
      "in 16.510s.\n",
      "-- designing region: 15:75033830-75063830_reg_Ly6c1:- 125, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ly6c1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.569s.\n",
      "-- designing region: 15:75096970-75126970_reg_Ly6c2:- 126, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ly6c2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.126s.\n",
      "in 16.566s.\n",
      "-- designing region: 15:74748620-74778620_reg_Ly6d:- 127, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ly6d_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.137s.\n",
      "in 16.562s.\n",
      "-- designing region: 17:35070610-35100610_reg_Ly6g6f:- 128, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ly6g6f_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.136s.\n",
      "in 16.357s.\n",
      "-- designing region: 15:74719329-74749329_reg_Lypd2:- 129, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Lypd2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "in 16.388s.\n",
      "-- designing region: 7:24621550-24651550_reg_Lypd3:+ 130, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Lypd3_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.323s.\n",
      "-- designing region: 2:127641695-127671695_reg_Mal:- 131, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Mal_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.271s.\n",
      "-- designing region: 12:109526001-109556001_reg_Meg3:+ 132, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Meg3_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.258s.\n",
      "-- designing region: 2:3407648-3437648_reg_Meig1:- 133, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Meig1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "in 16.381s.\n",
      "-- designing region: 6:136860823-136890823_reg_Mgp:- 134, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Mgp_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.130s.\n",
      "in 16.417s.\n",
      "-- designing region: 1:90900085-90930085_reg_Mlph:+ 135, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Mlph_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.136s.\n",
      "in 16.270s.\n",
      "-- designing region: 11:87778581-87808581_reg_Mpo:+ 136, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Mpo_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.275s.\n",
      "-- designing region: 8:94137607-94167607_reg_Mt3:+ 137, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Mt3_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.120s.\n",
      "in 16.281s.\n",
      "-- designing region: 8:94122197-94152197_reg_Mt4:+ 138, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Mt4_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.118s.\n",
      "in 16.247s.\n",
      "-- designing region: 16:33779037-33809037_reg_Muc13:+ 139, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Muc13_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.281s.\n",
      "-- designing region: 17:45587102-45617102_reg_Mymx:- 140, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Mymx_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.276s.\n",
      "-- designing region: 1:134274989-134304989_reg_Myog:+ 141, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Myog_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "in 16.271s.\n",
      "-- designing region: 11:4933064-4963064_reg_Nefh:- 142, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Nefh_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.278s.\n",
      "-- designing region: 9:110404747-110434747_reg_Ngp:+ 143, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ngp_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.136s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "in 16.270s.\n",
      "-- designing region: 4:147985722-148015722_reg_Nppa:+ 144, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Nppa_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.126s.\n",
      "in 16.270s.\n",
      "-- designing region: 6:49807710-49837710_reg_Npy:+ 145, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Npy_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "in 16.269s.\n",
      "-- designing region: 9:37537904-37567904_reg_Nrgn:- 146, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Nrgn_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.259s.\n",
      "-- designing region: 10:102475486-102505486_reg_Nts:- 147, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Nts_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.218s.\n",
      "-- designing region: 5:120899536-120929536_reg_Oas1d:+ 148, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Oas1d_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.263s.\n",
      "-- designing region: 5:120846421-120876421_reg_Oas1h:+ 149, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Oas1h_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.118s.\n",
      "in 16.268s.\n",
      "-- designing region: 15:89364587-89394587_reg_Odf3b:- 150, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Odf3b_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.258s.\n",
      "-- designing region: 2:28177992-28207992_reg_Olfm1:+ 151, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Olfm1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.259s.\n",
      "-- designing region: 16:96452606-96482606_reg_Pcp4:+ 152, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Pcp4_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.256s.\n",
      "-- designing region: 4:115073708-115103708_reg_Pdzk1ip1:+ 153, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Pdzk1ip1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.258s.\n",
      "-- designing region: 4:4123819-4153819_reg_Penk:- 154, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Penk_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.249s.\n",
      "-- designing region: 17:47711842-47741842_reg_Pgc:+ 155, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Pgc_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.270s.\n",
      "-- designing region: 1:130811684-130841684_reg_Pigr:+ 156, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Pigr_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.133s.\n",
      "in 16.269s.\n",
      "-- designing region: 11:68417121-68447121_reg_Pik3r5:+ 157, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Pik3r5_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.123s.\n",
      "in 16.249s.\n",
      "-- designing region: 5:115451262-115481262_reg_Pla2g1b:+ 158, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Pla2g1b_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.128s.\n",
      "in 16.271s.\n",
      "-- designing region: 5:100557245-100587245_reg_Plac8:- 159, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Plac8_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.254s.\n",
      "-- designing region: 13:110380046-110410046_reg_Plk2:+ 160, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Plk2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.133s.\n",
      "in 16.239s.\n",
      "-- designing region: 8:71496752-71526752_reg_Plvap:- 161, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Plvap_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.255s.\n",
      "-- designing region: 12:3939951-3969951_reg_Pomc:+ 162, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Pomc_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.136s.\n",
      "in 28.342s.\n",
      "-- designing region: 9:43195369-43225369_reg_Pou2f3:- 163, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Pou2f3_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "in 16.306s.\n",
      "-- designing region: 15:74699839-74729839_reg_Psca:+ 164, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Psca_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.132s.\n",
      "in 16.976s.\n",
      "-- designing region: 2:25455046-25485046_reg_Ptgds:- 165, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ptgds_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.152s.\n",
      "in 16.354s.\n",
      "-- designing region: 11:102092832-102122832_reg_Pyy:- 166, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Pyy_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.121s.\n",
      "in 16.298s.\n",
      "-- designing region: 11:74595915-74625915_reg_Rap1gap2:- 167, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Rap1gap2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.118s.\n",
      "in 16.269s.\n",
      "-- designing region: 3:67500523-67530523_reg_Rarres1:- 168, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Rarres1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.300s.\n",
      "-- designing region: 6:78355657-78385657_reg_Reg3b:+ 169, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Reg3b_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.298s.\n",
      "-- designing region: 3:98207156-98237156_reg_Reg4:+ 170, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Reg4_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.129s.\n",
      "in 16.272s.\n",
      "-- designing region: 16:48827551-48857551_reg_Retnla:+ 171, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Retnla_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.280s.\n",
      "-- designing region: 3:94465341-94495341_reg_Riiad1:- 172, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Riiad1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.118s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "in 16.289s.\n",
      "-- designing region: 14:51131785-51161785_reg_Rnase1:- 173, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Rnase1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.355s.\n",
      "-- designing region: 14:51076077-51106077_reg_Rnase4:+ 174, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Rnase4_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "in 16.271s.\n",
      "-- designing region: 12:26441452-26471452_reg_Rsad2:- 175, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Rsad2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.288s.\n",
      "-- designing region: 17:31262383-31292383_reg_Rsph1:- 176, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Rsph1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.288s.\n",
      "-- designing region: 3:90680721-90710721_reg_S100a9:- 177, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\S100a9_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.288s.\n",
      "-- designing region: 7:46727980-46757980_reg_Saa1:- 178, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Saa1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.128s.\n",
      "in 16.289s.\n",
      "-- designing region: 7:46736790-46766790_reg_Saa2:+ 179, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Saa2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.118s.\n",
      "in 16.398s.\n",
      "-- designing region: 1:107346198-107376198_reg_Serpinb11:+ 180, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Serpinb11_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.120s.\n",
      "in 16.263s.\n",
      "-- designing region: 14:41116782-41146782_reg_Sftpa1:+ 181, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Sftpa1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.293s.\n",
      "-- designing region: 14:41170149-41200149_reg_Sftpd:- 182, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Sftpd_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.289s.\n",
      "-- designing region: 2:164374095-164404095_reg_Slpi:- 183, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Slpi_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.287s.\n",
      "-- designing region: 17:25555686-25585686_reg_Sox8:- 184, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Sox8_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.118s.\n",
      "in 16.277s.\n",
      "-- designing region: 4:43638594-43668594_reg_Spag8:- 185, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Spag8_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.309s.\n",
      "-- designing region: 11:55408183-55438183_reg_Sparc:- 186, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Sparc_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.125s.\n",
      "in 16.286s.\n",
      "-- designing region: 7:44517071-44547071_reg_Spib:- 187, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Spib_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.136s.\n",
      "in 16.317s.\n",
      "-- designing region: 5:104420118-104450118_reg_Spp1:+ 188, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Spp1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.282s.\n",
      "-- designing region: 3:92423789-92453789_reg_Sprr1b:- 189, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Sprr1b_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.136s.\n",
      "in 16.251s.\n",
      "-- designing region: 3:92392991-92422991_reg_Sprr2i:+ 190, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Sprr2i_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.267s.\n",
      "-- designing region: 16:23875958-23905958_reg_Sst:- 191, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Sst_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.289s.\n",
      "-- designing region: 16:37369962-37399962_reg_Stxbp5l:- 192, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Stxbp5l_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.132s.\n",
      "in 16.249s.\n",
      "-- designing region: 19:10374090-10404090_reg_Syt7:+ 193, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Syt7_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.288s.\n",
      "-- designing region: 6:7539879-7569879_reg_Tac1:+ 194, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Tac1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.136s.\n",
      "in 16.251s.\n",
      "-- designing region: 10:127709478-127739478_reg_Tac2:+ 195, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Tac2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 17.116s.\n",
      "-- designing region: 6:67520796-67550796_reg_Tacstd2:- 196, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Tacstd2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.234s.\n",
      "-- designing region: 17:31129282-31159282_reg_Tff2:- 197, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Tff2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.258s.\n",
      "-- designing region: 17:31114646-31144646_reg_Tff3:- 198, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Tff3_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.119s.\n",
      "in 16.321s.\n",
      "-- designing region: 11:118340740-118370740_reg_Timp2:- 199, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Timp2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.259s.\n",
      "-- designing region: 16:56871166-56901166_reg_Tmem45a:- 200, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Tmem45a_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.248s.\n",
      "-- designing region: 15:54263484-54293484_reg_Tnfrsf11b:- 201, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Tnfrsf11b_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "in 16.230s.\n",
      "-- designing region: 16:25668763-25698763_reg_Trp63:+ 202, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Trp63_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.237s.\n",
      "-- designing region: 7:143079642-143109642_reg_Trpm5:- 203, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Trpm5_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.274s.\n",
      "-- designing region: 4:116152601-116182601_reg_Tspan1:- 204, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Tspan1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.136s.\n",
      "in 16.270s.\n",
      "-- designing region: 9:44788072-44818072_reg_Ttc36:- 205, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ttc36_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "in 16.251s.\n",
      "-- designing region: 10:75928057-75958057_reg_Vpreb3:+ 206, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Vpreb3_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.249s.\n",
      "-- designing region: 1:164920527-164950527_reg_Xcl1:- 207, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Xcl1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.118s.\n",
      "in 16.238s.\n",
      "-- designing region: 7:127072328-127102328_reg_Zg16:- 208, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Zg16_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.129s.\n",
      "in 16.248s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\reports\\merged_probes.pbr\n"
     ]
    }
   ],
   "source": [
    "reload(library_tools)\n",
    "reload(library_tools.design)\n",
    "pb_designer.computeOTmaps() # load the tables \n",
    "pb_designer.compute_pb_report() # design candidate probes"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "rapid-intermediate",
   "metadata": {},
   "source": [
    "### check probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 125,
   "id": "restricted-scientist",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:0 9:104322748-104352748_reg_Acpp:-, 58448 candidate probes\n",
      "--- 30346 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.513s, 422 probes kept.\n",
      "-- check region:1 8:124554706-124584706_reg_Agt:-, 53018 candidate probes\n",
      "--- 25225 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.080s, 352 probes kept.\n",
      "-- check region:2 11:61192537-61222537_reg_Aldh3a1:+, 59118 candidate probes\n",
      "--- 31353 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.407s, 432 probes kept.\n",
      "-- check region:3 4:49534546-49564546_reg_Aldob:-, 57512 candidate probes\n",
      "--- 28730 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.335s, 413 probes kept.\n",
      "-- check region:4 5:149249767-149279767_reg_Alox5ap:+, 59050 candidate probes\n",
      "--- 27984 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.346s, 397 probes kept.\n",
      "-- check region:5 1:87086606-87116606_reg_Alpi:-, 59514 candidate probes\n",
      "--- 24857 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.151s, 396 probes kept.\n",
      "-- check region:6 4:137781384-137811384_reg_Alpl:-, 58836 candidate probes\n",
      "--- 27888 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.315s, 393 probes kept.\n",
      "-- check region:7 14:51758590-51788590_reg_Ang4:-, 51186 candidate probes\n",
      "--- 7585 probes passed check_dic, GC and Tm selection.\n",
      "finish in 1.473s, 174 probes kept.\n",
      "-- check region:8 14:34070981-34100981_reg_Anxa8:+, 59610 candidate probes\n",
      "--- 36723 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.710s, 496 probes kept.\n",
      "-- check region:9 9:46213580-46243580_reg_Apoa1:+, 59446 candidate probes\n",
      "--- 31039 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.468s, 454 probes kept.\n",
      "-- check region:10 1:171210054-171240054_reg_Apoa2:+, 58882 candidate probes\n",
      "--- 22793 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.044s, 373 probes kept.\n",
      "-- check region:11 7:19662941-19692941_reg_Apoc2:-, 56036 candidate probes\n",
      "--- 19564 probes passed check_dic, GC and Tm selection.\n",
      "finish in 1.920s, 319 probes kept.\n",
      "-- check region:12 16:31299808-31329808_reg_Apod:-, 59030 candidate probes\n",
      "--- 27156 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.267s, 411 probes kept.\n",
      "-- check region:13 10:87478660-87508660_reg_Ascl1:-, 57926 candidate probes\n",
      "--- 28107 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.237s, 410 probes kept.\n",
      "-- check region:14 10:126985709-127015709_reg_Avil:+, 58794 candidate probes\n",
      "--- 29089 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.329s, 434 probes kept.\n",
      "-- check region:15 X:73468602-73498602_reg_Bgn:+, 59380 candidate probes\n",
      "--- 31537 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.375s, 474 probes kept.\n",
      "-- check region:16 4:136871187-136901187_reg_C1qb:-, 58528 candidate probes\n",
      "--- 25835 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.184s, 409 probes kept.\n",
      "-- check region:17 17:57213136-57243136_reg_C3:-, 59432 candidate probes\n",
      "--- 25667 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.193s, 404 probes kept.\n",
      "-- check region:18 8:110153210-110183210_reg_Calb2:-, 59190 candidate probes\n",
      "--- 35456 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.538s, 456 probes kept.\n",
      "-- check region:19 6:34583500-34613500_reg_Cald1:+, 57594 candidate probes\n",
      "--- 29706 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.451s, 400 probes kept.\n",
      "-- check region:20 9:109834617-109864617_reg_Camp:-, 56924 candidate probes\n",
      "--- 22485 probes passed check_dic, GC and Tm selection.\n",
      "finish in 1.964s, 329 probes kept.\n",
      "-- check region:21 11:84942786-84972786_reg_Car4:+, 59260 candidate probes\n",
      "--- 30669 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.508s, 429 probes kept.\n",
      "-- check region:22 4:8224041-8254041_reg_Car8:-, 57068 candidate probes\n",
      "--- 24267 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.113s, 369 probes kept.\n",
      "-- check region:23 4:119400521-119430521_reg_Ccdc30:-, 59082 candidate probes\n",
      "--- 24034 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.173s, 388 probes kept.\n",
      "-- check region:24 9:121480694-121510694_reg_Cck:-, 58886 candidate probes\n",
      "--- 27777 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.270s, 404 probes kept.\n",
      "-- check region:25 11:82290690-82320690_reg_Ccl1:-, 59606 candidate probes\n",
      "--- 31428 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.354s, 441 probes kept.\n",
      "-- check region:26 1:83101766-83131766_reg_Ccl20:+, 58752 candidate probes\n",
      "--- 29109 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.290s, 419 probes kept.\n",
      "-- check region:27 4:42758993-42788993_reg_Ccl21a:-, 58856 candidate probes\n",
      "--- 27 probes passed check_dic, GC and Tm selection.\n",
      "finish in 1.311s, 5 probes kept.\n",
      "-- check region:28 11:82020571-82050571_reg_Ccl2:+, 59788 candidate probes\n",
      "--- 32483 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.467s, 470 probes kept.\n",
      "-- check region:29 11:83578087-83608087_reg_Ccl6:-, 28754 candidate probes\n",
      "--- 13778 probes passed check_dic, GC and Tm selection.\n",
      "finish in 1.252s, 199 probes kept.\n",
      "-- check region:30 11:83563636-83593636_reg_Ccl9:-, 58104 candidate probes\n",
      "--- 24382 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.124s, 392 probes kept.\n",
      "-- check region:31 18:36711798-36741798_reg_Cd14:-, 57850 candidate probes\n",
      "--- 20715 probes passed check_dic, GC and Tm selection.\n",
      "finish in 1.922s, 352 probes kept.\n",
      "-- check region:32 2:167673915-167703915_reg_Cebpb:+, 58686 candidate probes\n",
      "--- 26017 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.191s, 410 probes kept.\n",
      "-- check region:33 3:94463295-94493295_reg_Celf3:+, 3988 candidate probes\n",
      "--- 1627 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.170s, 26 probes kept.\n",
      "-- check region:34 4:34878779-34908779_reg_Cga:+, 57418 candidate probes\n",
      "--- 21216 probes passed check_dic, GC and Tm selection.\n",
      "finish in 1.934s, 342 probes kept.\n",
      "-- check region:35 9:65250180-65280180_reg_Cilp:+, 59032 candidate probes\n",
      "--- 24579 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.182s, 396 probes kept.\n",
      "-- check region:36 12:111657338-111687338_reg_Ckb:-, 59058 candidate probes\n",
      "--- 29462 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.330s, 429 probes kept.\n",
      "-- check region:37 7:19389776-19419776_reg_Ckm:+, 58948 candidate probes\n",
      "--- 22768 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.115s, 391 probes kept.\n",
      "-- check region:38 17:28545766-28575766_reg_Clps:-, 57758 candidate probes\n",
      "--- 21852 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.023s, 327 probes kept.\n",
      "-- check region:39 7:25354724-25384724_reg_Cnfn:-, 59336 candidate probes\n",
      "--- 29514 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.377s, 453 probes kept.\n",
      "-- check region:40 14:79647170-79677170_reg_Cnmd:-, 57040 candidate probes\n",
      "--- 31791 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.406s, 440 probes kept.\n",
      "-- check region:41 11:94921224-94951224_reg_Col1a1:+, 59068 candidate probes\n",
      "--- 32998 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.480s, 471 probes kept.\n",
      "-- check region:42 8:11297826-11327826_reg_Col4a1:-, 59510 candidate probes\n",
      "--- 36095 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.577s, 467 probes kept.\n",
      "-- check region:43 10:76711168-76741168_reg_Col6a1:-, 59678 candidate probes\n",
      "--- 34176 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.507s, 462 probes kept.\n",
      "-- check region:44 19:43971556-44001556_reg_Cpn1:-, 59248 candidate probes\n",
      "--- 28182 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.254s, 392 probes kept.\n",
      "-- check region:45 9:54749748-54779748_reg_Crabp1:+, 59532 candidate probes\n",
      "--- 34304 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.488s, 465 probes kept.\n",
      "-- check region:46 17:31662807-31692807_reg_Cryaa:+, 58760 candidate probes\n",
      "--- 25729 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.227s, 389 probes kept.\n",
      "-- check region:47 8:111676010-111706010_reg_Ctrb1:-, 57908 candidate probes\n",
      "--- 24326 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.095s, 361 probes kept.\n",
      "-- check region:48 5:95941951-95971951_reg_Cxcl13:+, 58768 candidate probes\n",
      "--- 28545 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.312s, 391 probes kept.\n",
      "-- check region:49 9:57668703-57698703_reg_Cyp1a2:-, 59106 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 25355 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.105s, 400 probes kept.\n",
      "-- check region:50 7:27104909-27134909_reg_Cyp2f2:+, 58014 candidate probes\n",
      "--- 21187 probes passed check_dic, GC and Tm selection.\n",
      "finish in 1.990s, 348 probes kept.\n",
      "-- check region:51 13:55473111-55503111_reg_Dbn1:-, 59344 candidate probes\n",
      "--- 29821 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.319s, 433 probes kept.\n",
      "-- check region:52 3:55227364-55257364_reg_Dclk1:+, 59692 candidate probes\n",
      "--- 29249 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.369s, 441 probes kept.\n",
      "-- check region:53 12:109437823-109467823_reg_Dlk1:+, 59602 candidate probes\n",
      "--- 32565 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.497s, 466 probes kept.\n",
      "-- check region:54 6:122611410-122641410_reg_Dppa3:+, 56332 candidate probes\n",
      "--- 17343 probes passed check_dic, GC and Tm selection.\n",
      "finish in 1.831s, 276 probes kept.\n",
      "-- check region:55 3:95724569-95754569_reg_Ecm1:-, 59310 candidate probes\n",
      "--- 21913 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.054s, 402 probes kept.\n",
      "-- check region:56 3:89307965-89337965_reg_Efna3:-, 59680 candidate probes\n",
      "--- 29761 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.429s, 436 probes kept.\n",
      "-- check region:57 9:83791277-83821277_reg_Elovl4:-, 59490 candidate probes\n",
      "--- 27924 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.307s, 412 probes kept.\n",
      "-- check region:58 6:71184827-71214827_reg_Fabp1:+, 59242 candidate probes\n",
      "--- 25309 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.236s, 375 probes kept.\n",
      "-- check region:59 11:43586540-43616540_reg_Fabp6:-, 58550 candidate probes\n",
      "--- 25427 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.204s, 388 probes kept.\n",
      "-- check region:60 11:58786960-58816960_reg_Fam183b:-, 59244 candidate probes\n",
      "--- 23552 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.134s, 366 probes kept.\n",
      "-- check region:61 14:34340433-34370433_reg_Fam25c:-, 59158 candidate probes\n",
      "--- 28575 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.265s, 415 probes kept.\n",
      "-- check region:62 3:82992724-83022724_reg_Fgg:+, 59390 candidate probes\n",
      "--- 30190 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.407s, 438 probes kept.\n",
      "-- check region:63 12:57531916-57561916_reg_Foxa1:-, 59446 candidate probes\n",
      "--- 31775 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.446s, 445 probes kept.\n",
      "-- check region:64 2:148031969-148061969_reg_Foxa2:-, 59804 candidate probes\n",
      "--- 34921 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.527s, 464 probes kept.\n",
      "-- check region:65 7:19008538-19038538_reg_Foxa3:-, 58090 candidate probes\n",
      "--- 24314 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.101s, 402 probes kept.\n",
      "-- check region:66 11:116320399-116350399_reg_Foxj1:-, 59384 candidate probes\n",
      "--- 32343 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.481s, 468 probes kept.\n",
      "-- check region:67 7:31061704-31091704_reg_Fxyd3:-, 59668 candidate probes\n",
      "--- 25471 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.316s, 400 probes kept.\n",
      "-- check region:68 9:45355185-45385185_reg_Fxyd6:+, 59098 candidate probes\n",
      "--- 33507 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.458s, 462 probes kept.\n",
      "-- check region:69 11:33563959-33593959_reg_Gabrp:-, 58802 candidate probes\n",
      "--- 29208 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.316s, 406 probes kept.\n",
      "-- check region:70 19:3399544-3429544_reg_Gal:-, 59350 candidate probes\n",
      "--- 30590 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.387s, 452 probes kept.\n",
      "-- check region:71 2:122596303-122626303_reg_Gatm:-, 58774 candidate probes\n",
      "--- 28898 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.254s, 419 probes kept.\n",
      "-- check region:72 6:113704880-113734880_reg_Ghrl:-, 59344 candidate probes\n",
      "--- 27867 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.329s, 437 probes kept.\n",
      "-- check region:73 2:25767257-25797257_reg_Gm13539:+, 33266 candidate probes\n",
      "--- 14763 probes passed check_dic, GC and Tm selection.\n",
      "finish in 1.334s, 228 probes kept.\n",
      "-- check region:74 11:55083115-55113115_reg_Gm2a:+, 58900 candidate probes\n",
      "--- 27862 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.225s, 415 probes kept.\n",
      "-- check region:75 18:43777878-43807878_reg_Gm94:-, 58494 candidate probes\n",
      "--- 25446 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.216s, 388 probes kept.\n",
      "-- check region:76 7:119444285-119474285_reg_Gp2:-, 58928 candidate probes\n",
      "--- 30082 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.327s, 421 probes kept.\n",
      "-- check region:77 11:54887453-54917453_reg_Gpx3:+, 58856 candidate probes\n",
      "--- 26816 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.229s, 393 probes kept.\n",
      "-- check region:78 13:21297125-21327125_reg_Gpx6:+, 58982 candidate probes\n",
      "--- 30841 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.415s, 436 probes kept.\n",
      "-- check region:79 12:24557283-24587283_reg_Grhl1:+, 59038 candidate probes\n",
      "--- 31945 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.408s, 454 probes kept.\n",
      "-- check region:80 11:98649351-98679351_reg_Gsdma:+, 58042 candidate probes\n",
      "--- 16976 probes passed check_dic, GC and Tm selection.\n",
      "finish in 1.881s, 318 probes kept.\n",
      "-- check region:81 9:78340788-78370788_reg_Gsta2:-, 58128 candidate probes\n",
      "--- 14714 probes passed check_dic, GC and Tm selection.\n",
      "finish in 1.913s, 271 probes kept.\n",
      "-- check region:82 9:78167774-78197774_reg_Gsta4:+, 58390 candidate probes\n",
      "--- 19732 probes passed check_dic, GC and Tm selection.\n",
      "finish in 1.919s, 332 probes kept.\n",
      "-- check region:83 3:107954283-107984283_reg_Gstm3:-, 59490 candidate probes\n",
      "--- 24974 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.254s, 394 probes kept.\n",
      "-- check region:84 19:47839970-47869970_reg_Gsto1:+, 58954 candidate probes\n",
      "--- 23686 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.095s, 336 probes kept.\n",
      "-- check region:85 4:119622704-119652704_reg_Guca2a:+, 52096 candidate probes\n",
      "--- 20949 probes passed check_dic, GC and Tm selection.\n",
      "finish in 1.902s, 314 probes kept.\n",
      "-- check region:86 7:142563143-142593143_reg_H19:-, 59866 candidate probes\n",
      "--- 40678 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.728s, 501 probes kept.\n",
      "-- check region:87 7:30909681-30939681_reg_Hamp2:-, 57016 candidate probes\n",
      "--- 14554 probes passed check_dic, GC and Tm selection.\n",
      "finish in 1.741s, 238 probes kept.\n",
      "-- check region:88 11:32281489-32311489_reg_Hba-a2:+, 54118 candidate probes\n",
      "--- 24729 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.056s, 378 probes kept.\n",
      "-- check region:89 7:103838216-103868216_reg_Hbb-y:-, 58158 candidate probes\n",
      "--- 23907 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.051s, 393 probes kept.\n",
      "-- check region:90 2:153093468-153123468_reg_Hck:+, 48344 candidate probes\n",
      "--- 25150 probes passed check_dic, GC and Tm selection.\n",
      "finish in 1.965s, 356 probes kept.\n",
      "-- check region:91 2:163491808-163521808_reg_Hnf4a:+, 59306 candidate probes\n",
      "--- 25155 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.177s, 378 probes kept.\n",
      "-- check region:92 3:3493030-3523030_reg_Hnf4g:+, 58354 candidate probes\n",
      "--- 23431 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.123s, 365 probes kept.\n",
      "-- check region:93 8:109564172-109594172_reg_Hp:-, 59218 candidate probes\n",
      "--- 27538 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.256s, 442 probes kept.\n",
      "-- check region:94 6:142283423-142313423_reg_Iapp:+, 56302 candidate probes\n",
      "--- 18070 probes passed check_dic, GC and Tm selection.\n",
      "finish in 1.920s, 301 probes kept.\n",
      "-- check region:95 13:8937863-8967863_reg_Idi2:+, 41940 candidate probes\n",
      "--- 13108 probes passed check_dic, GC and Tm selection.\n",
      "finish in 1.486s, 244 probes kept.\n",
      "-- check region:96 12:103442223-103472223_reg_Ifi27l2b:-, 48848 candidate probes\n",
      "--- 11945 probes passed check_dic, GC and Tm selection.\n",
      "finish in 1.504s, 193 probes kept.\n",
      "-- check region:97 10:87843265-87873265_reg_Igf1:+, 59642 candidate probes\n",
      "--- 33089 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.461s, 467 probes kept.\n",
      "-- check region:98 7:142651816-142681816_reg_Igf2:-, 59670 candidate probes\n",
      "--- 30455 probes passed check_dic, GC and Tm selection.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "finish in 2.389s, 476 probes kept.\n",
      "-- check region:99 1:72859884-72889884_reg_Igfbp5:-, 59572 candidate probes\n",
      "--- 31685 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.407s, 454 probes kept.\n",
      "-- check region:100 15:102129362-102159362_reg_Igfbp6:+, 58934 candidate probes\n",
      "--- 25780 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.200s, 387 probes kept.\n",
      "-- check region:101 7:142728381-142758381_reg_Ins2:-, 56480 candidate probes\n",
      "--- 22308 probes passed check_dic, GC and Tm selection.\n",
      "finish in 1.993s, 327 probes kept.\n",
      "-- check region:102 4:156185796-156215796_reg_Isg15:-, 58976 candidate probes\n",
      "--- 28669 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.325s, 420 probes kept.\n",
      "-- check region:103 2:71730616-71760616_reg_Itga6:+, 59210 candidate probes\n",
      "--- 32143 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.411s, 459 probes kept.\n",
      "-- check region:104 11:115499387-115529387_reg_Jpt1:-, 58398 candidate probes\n",
      "--- 21333 probes passed check_dic, GC and Tm selection.\n",
      "finish in 1.921s, 343 probes kept.\n",
      "-- check region:105 7:43675418-43705418_reg_Klk14:+, 58680 candidate probes\n",
      "--- 22537 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.134s, 378 probes kept.\n",
      "-- check region:106 7:43796294-43826294_reg_Klk7:+, 59190 candidate probes\n",
      "--- 25016 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.147s, 388 probes kept.\n",
      "-- check region:107 11:99374364-99404364_reg_Krt10:-, 59524 candidate probes\n",
      "--- 30939 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.355s, 443 probes kept.\n",
      "-- check region:108 11:100106566-100136566_reg_Krt13:-, 54122 candidate probes\n",
      "--- 30760 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.237s, 439 probes kept.\n",
      "-- check region:109 11:100192548-100222548_reg_Krt14:-, 28148 candidate probes\n",
      "--- 13160 probes passed check_dic, GC and Tm selection.\n",
      "finish in 1.123s, 194 probes kept.\n",
      "-- check region:110 11:100233902-100263902_reg_Krt16:-, 23626 candidate probes\n",
      "--- 4940 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.775s, 108 probes kept.\n",
      "-- check region:111 11:100246029-100276029_reg_Krt17:-, 58810 candidate probes\n",
      "--- 31014 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.438s, 438 probes kept.\n",
      "-- check region:112 11:100133665-100163665_reg_Krt19:-, 58888 candidate probes\n",
      "--- 29792 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.418s, 427 probes kept.\n",
      "-- check region:113 15:101697898-101727898_reg_Krt5:-, 59662 candidate probes\n",
      "--- 31854 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.458s, 427 probes kept.\n",
      "-- check region:114 15:101728109-101758109_reg_Krt71:-, 58580 candidate probes\n",
      "--- 30569 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.348s, 429 probes kept.\n",
      "-- check region:115 15:101877920-101907920_reg_Krt76:-, 46270 candidate probes\n",
      "--- 23718 probes passed check_dic, GC and Tm selection.\n",
      "finish in 1.851s, 333 probes kept.\n",
      "-- check region:116 15:101854705-101884705_reg_Krt77:-, 59860 candidate probes\n",
      "--- 33916 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.503s, 476 probes kept.\n",
      "-- check region:117 11:100178246-100208246_reg_Krt9:-, 59090 candidate probes\n",
      "--- 29053 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.343s, 425 probes kept.\n",
      "-- check region:118 7:30772896-30802896_reg_Krtdap:+, 58290 candidate probes\n",
      "--- 27363 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.302s, 405 probes kept.\n",
      "-- check region:119 2:32373252-32403252_reg_Lcn2:-, 59100 candidate probes\n",
      "--- 26267 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.245s, 445 probes kept.\n",
      "-- check region:120 2:25750569-25780569_reg_Lcn3:+, 59752 candidate probes\n",
      "--- 30002 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.517s, 423 probes kept.\n",
      "-- check region:121 15:78840529-78870529_reg_Lgals2:-, 58844 candidate probes\n",
      "--- 27708 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.289s, 417 probes kept.\n",
      "-- check region:122 14:47352751-47382751_reg_Lgals3:+, 59298 candidate probes\n",
      "--- 29012 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.302s, 413 probes kept.\n",
      "-- check region:123 15:7075614-7105614_reg_Lifr:+, 59624 candidate probes\n",
      "--- 32041 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.365s, 451 probes kept.\n",
      "-- check region:124 9:111004271-111034271_reg_Ltf:+, 58258 candidate probes\n",
      "--- 27140 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.236s, 414 probes kept.\n",
      "-- check region:125 15:75033830-75063830_reg_Ly6c1:-, 47374 candidate probes\n",
      "--- 3810 probes passed check_dic, GC and Tm selection.\n",
      "finish in 1.163s, 124 probes kept.\n",
      "-- check region:126 15:75096970-75126970_reg_Ly6c2:-, 58210 candidate probes\n",
      "--- 4217 probes passed check_dic, GC and Tm selection.\n",
      "finish in 1.395s, 147 probes kept.\n",
      "-- check region:127 15:74748620-74778620_reg_Ly6d:-, 56848 candidate probes\n",
      "--- 28491 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.256s, 409 probes kept.\n",
      "-- check region:128 17:35070610-35100610_reg_Ly6g6f:-, 58932 candidate probes\n",
      "--- 22876 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.099s, 391 probes kept.\n",
      "-- check region:129 15:74719329-74749329_reg_Lypd2:-, 38682 candidate probes\n",
      "--- 23514 probes passed check_dic, GC and Tm selection.\n",
      "finish in 1.749s, 318 probes kept.\n",
      "-- check region:130 7:24621550-24651550_reg_Lypd3:+, 59096 candidate probes\n",
      "--- 23647 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.103s, 387 probes kept.\n",
      "-- check region:131 2:127641695-127671695_reg_Mal:-, 59118 candidate probes\n",
      "--- 30989 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.495s, 427 probes kept.\n",
      "-- check region:132 12:109526001-109556001_reg_Meg3:+, 59806 candidate probes\n",
      "--- 35567 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.539s, 487 probes kept.\n",
      "-- check region:133 2:3407648-3437648_reg_Meig1:-, 58532 candidate probes\n",
      "--- 28182 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.371s, 422 probes kept.\n",
      "-- check region:134 6:136860823-136890823_reg_Mgp:-, 58416 candidate probes\n",
      "--- 28329 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.330s, 413 probes kept.\n",
      "-- check region:135 1:90900085-90930085_reg_Mlph:+, 59078 candidate probes\n",
      "--- 31152 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.356s, 432 probes kept.\n",
      "-- check region:136 11:87778581-87808581_reg_Mpo:+, 59520 candidate probes\n",
      "--- 27039 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.491s, 436 probes kept.\n",
      "-- check region:137 8:94137607-94167607_reg_Mt3:+, 30150 candidate probes\n",
      "--- 11405 probes passed check_dic, GC and Tm selection.\n",
      "finish in 1.090s, 187 probes kept.\n",
      "-- check region:138 8:94122197-94152197_reg_Mt4:+, 59242 candidate probes\n",
      "--- 28810 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.285s, 413 probes kept.\n",
      "-- check region:139 16:33779037-33809037_reg_Muc13:+, 58042 candidate probes\n",
      "--- 23513 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.021s, 369 probes kept.\n",
      "-- check region:140 17:45587102-45617102_reg_Mymx:-, 58956 candidate probes\n",
      "--- 25409 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.153s, 391 probes kept.\n",
      "-- check region:141 1:134274989-134304989_reg_Myog:+, 59366 candidate probes\n",
      "--- 30594 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.382s, 452 probes kept.\n",
      "-- check region:142 11:4933064-4963064_reg_Nefh:-, 58996 candidate probes\n",
      "--- 27266 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.226s, 418 probes kept.\n",
      "-- check region:143 9:110404747-110434747_reg_Ngp:+, 58770 candidate probes\n",
      "--- 24462 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.122s, 405 probes kept.\n",
      "-- check region:144 4:147985722-148015722_reg_Nppa:+, 59394 candidate probes\n",
      "--- 27703 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.222s, 419 probes kept.\n",
      "-- check region:145 6:49807710-49837710_reg_Npy:+, 59690 candidate probes\n",
      "--- 32042 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.498s, 445 probes kept.\n",
      "-- check region:146 9:37537904-37567904_reg_Nrgn:-, 59582 candidate probes\n",
      "--- 31016 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.438s, 450 probes kept.\n",
      "-- check region:147 10:102475486-102505486_reg_Nts:-, 59361 candidate probes\n",
      "--- 24964 probes passed check_dic, GC and Tm selection.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "finish in 2.175s, 396 probes kept.\n",
      "-- check region:148 5:120899536-120929536_reg_Oas1d:+, 58068 candidate probes\n",
      "--- 13068 probes passed check_dic, GC and Tm selection.\n",
      "finish in 1.818s, 261 probes kept.\n",
      "-- check region:149 5:120846421-120876421_reg_Oas1h:+, 57490 candidate probes\n",
      "--- 17356 probes passed check_dic, GC and Tm selection.\n",
      "finish in 1.928s, 314 probes kept.\n",
      "-- check region:150 15:89364587-89394587_reg_Odf3b:-, 59686 candidate probes\n",
      "--- 27571 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.262s, 450 probes kept.\n",
      "-- check region:151 2:28177992-28207992_reg_Olfm1:+, 59524 candidate probes\n",
      "--- 32848 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.547s, 452 probes kept.\n",
      "-- check region:152 16:96452606-96482606_reg_Pcp4:+, 59466 candidate probes\n",
      "--- 31074 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.366s, 450 probes kept.\n",
      "-- check region:153 4:115073708-115103708_reg_Pdzk1ip1:+, 59480 candidate probes\n",
      "--- 32449 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.375s, 451 probes kept.\n",
      "-- check region:154 4:4123819-4153819_reg_Penk:-, 58636 candidate probes\n",
      "--- 26539 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.191s, 386 probes kept.\n",
      "-- check region:155 17:47711842-47741842_reg_Pgc:+, 59038 candidate probes\n",
      "--- 28724 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.214s, 424 probes kept.\n",
      "-- check region:156 1:130811684-130841684_reg_Pigr:+, 59814 candidate probes\n",
      "--- 36712 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.568s, 493 probes kept.\n",
      "-- check region:157 11:68417121-68447121_reg_Pik3r5:+, 59132 candidate probes\n",
      "--- 28592 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.304s, 421 probes kept.\n",
      "-- check region:158 5:115451262-115481262_reg_Pla2g1b:+, 58576 candidate probes\n",
      "--- 19993 probes passed check_dic, GC and Tm selection.\n",
      "finish in 1.962s, 332 probes kept.\n",
      "-- check region:159 5:100557245-100587245_reg_Plac8:-, 58798 candidate probes\n",
      "--- 24931 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.179s, 373 probes kept.\n",
      "-- check region:160 13:110380046-110410046_reg_Plk2:+, 59592 candidate probes\n",
      "--- 23462 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.124s, 384 probes kept.\n",
      "-- check region:161 8:71496752-71526752_reg_Plvap:-, 58214 candidate probes\n",
      "--- 21223 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.043s, 378 probes kept.\n",
      "-- check region:162 12:3939951-3969951_reg_Pomc:+, 58864 candidate probes\n",
      "--- 25280 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.102s, 410 probes kept.\n",
      "-- check region:163 9:43195369-43225369_reg_Pou2f3:-, 59132 candidate probes\n",
      "--- 26692 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.273s, 392 probes kept.\n",
      "-- check region:164 15:74699839-74729839_reg_Psca:+, 59642 candidate probes\n",
      "--- 31791 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.408s, 501 probes kept.\n",
      "-- check region:165 2:25455046-25485046_reg_Ptgds:-, 59554 candidate probes\n",
      "--- 35131 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.590s, 479 probes kept.\n",
      "-- check region:166 11:102092832-102122832_reg_Pyy:-, 59360 candidate probes\n",
      "--- 27946 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.314s, 404 probes kept.\n",
      "-- check region:167 11:74595915-74625915_reg_Rap1gap2:-, 58804 candidate probes\n",
      "--- 23671 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.103s, 358 probes kept.\n",
      "-- check region:168 3:67500523-67530523_reg_Rarres1:-, 58966 candidate probes\n",
      "--- 32104 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.479s, 431 probes kept.\n",
      "-- check region:169 6:78355657-78385657_reg_Reg3b:+, 57644 candidate probes\n",
      "--- 21198 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.001s, 347 probes kept.\n",
      "-- check region:170 3:98207156-98237156_reg_Reg4:+, 59424 candidate probes\n",
      "--- 27317 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.262s, 396 probes kept.\n",
      "-- check region:171 16:48827551-48857551_reg_Retnla:+, 58654 candidate probes\n",
      "--- 15502 probes passed check_dic, GC and Tm selection.\n",
      "finish in 1.891s, 245 probes kept.\n",
      "-- check region:172 3:94465341-94495341_reg_Riiad1:-, 59500 candidate probes\n",
      "--- 28913 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.340s, 439 probes kept.\n",
      "-- check region:173 14:51131785-51161785_reg_Rnase1:-, 56892 candidate probes\n",
      "--- 11554 probes passed check_dic, GC and Tm selection.\n",
      "finish in 1.669s, 227 probes kept.\n",
      "-- check region:174 14:51076077-51106077_reg_Rnase4:+, 59468 candidate probes\n",
      "--- 20845 probes passed check_dic, GC and Tm selection.\n",
      "finish in 1.970s, 348 probes kept.\n",
      "-- check region:175 12:26441452-26471452_reg_Rsad2:-, 59268 candidate probes\n",
      "--- 27891 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.254s, 433 probes kept.\n",
      "-- check region:176 17:31262383-31292383_reg_Rsph1:-, 56674 candidate probes\n",
      "--- 26877 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.204s, 372 probes kept.\n",
      "-- check region:177 3:90680721-90710721_reg_S100a9:-, 59278 candidate probes\n",
      "--- 25837 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.247s, 375 probes kept.\n",
      "-- check region:178 7:46727980-46757980_reg_Saa1:-, 17450 candidate probes\n",
      "--- 8315 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.748s, 135 probes kept.\n",
      "-- check region:179 7:46736790-46766790_reg_Saa2:+, 56998 candidate probes\n",
      "--- 27016 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.165s, 445 probes kept.\n",
      "-- check region:180 1:107346198-107376198_reg_Serpinb11:+, 59258 candidate probes\n",
      "--- 26097 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.293s, 406 probes kept.\n",
      "-- check region:181 14:41116782-41146782_reg_Sftpa1:+, 59110 candidate probes\n",
      "--- 27157 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.194s, 411 probes kept.\n",
      "-- check region:182 14:41170149-41200149_reg_Sftpd:-, 59674 candidate probes\n",
      "--- 26821 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.226s, 404 probes kept.\n",
      "-- check region:183 2:164374095-164404095_reg_Slpi:-, 59258 candidate probes\n",
      "--- 28708 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.511s, 422 probes kept.\n",
      "-- check region:184 17:25555686-25585686_reg_Sox8:-, 59338 candidate probes\n",
      "--- 33458 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.479s, 462 probes kept.\n",
      "-- check region:185 4:43638594-43668594_reg_Spag8:-, 59834 candidate probes\n",
      "--- 29521 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.316s, 466 probes kept.\n",
      "-- check region:186 11:55408183-55438183_reg_Sparc:-, 58954 candidate probes\n",
      "--- 26348 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.183s, 390 probes kept.\n",
      "-- check region:187 7:44517071-44547071_reg_Spib:-, 59566 candidate probes\n",
      "--- 29544 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.276s, 457 probes kept.\n",
      "-- check region:188 5:104420118-104450118_reg_Spp1:+, 59054 candidate probes\n",
      "--- 24574 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.084s, 378 probes kept.\n",
      "-- check region:189 3:92423789-92453789_reg_Sprr1b:-, 59420 candidate probes\n",
      "--- 19579 probes passed check_dic, GC and Tm selection.\n",
      "finish in 1.973s, 331 probes kept.\n",
      "-- check region:190 3:92392991-92422991_reg_Sprr2i:+, 59356 candidate probes\n",
      "--- 22085 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.093s, 373 probes kept.\n",
      "-- check region:191 16:23875958-23905958_reg_Sst:-, 59762 candidate probes\n",
      "--- 29713 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.339s, 434 probes kept.\n",
      "-- check region:192 16:37369962-37399962_reg_Stxbp5l:-, 59606 candidate probes\n",
      "--- 22090 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.012s, 388 probes kept.\n",
      "-- check region:193 19:10374090-10404090_reg_Syt7:+, 59614 candidate probes\n",
      "--- 35077 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.579s, 466 probes kept.\n",
      "-- check region:194 6:7539879-7569879_reg_Tac1:+, 58070 candidate probes\n",
      "--- 22682 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.093s, 368 probes kept.\n",
      "-- check region:195 10:127709478-127739478_reg_Tac2:+, 59792 candidate probes\n",
      "--- 32136 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.468s, 472 probes kept.\n",
      "-- check region:196 6:67520796-67550796_reg_Tacstd2:-, 58880 candidate probes\n",
      "--- 25604 probes passed check_dic, GC and Tm selection.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "finish in 2.161s, 380 probes kept.\n",
      "-- check region:197 17:31129282-31159282_reg_Tff2:-, 28996 candidate probes\n",
      "--- 14229 probes passed check_dic, GC and Tm selection.\n",
      "finish in 1.284s, 196 probes kept.\n",
      "-- check region:198 17:31114646-31144646_reg_Tff3:-, 59642 candidate probes\n",
      "--- 28686 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.276s, 431 probes kept.\n",
      "-- check region:199 11:118340740-118370740_reg_Timp2:-, 59702 candidate probes\n",
      "--- 30714 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.427s, 433 probes kept.\n",
      "-- check region:200 16:56871166-56901166_reg_Tmem45a:-, 59382 candidate probes\n",
      "--- 20952 probes passed check_dic, GC and Tm selection.\n",
      "finish in 1.963s, 325 probes kept.\n",
      "-- check region:201 15:54263484-54293484_reg_Tnfrsf11b:-, 59462 candidate probes\n",
      "--- 25487 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.295s, 378 probes kept.\n",
      "-- check region:202 16:25668763-25698763_reg_Trp63:+, 59652 candidate probes\n",
      "--- 22892 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.044s, 377 probes kept.\n",
      "-- check region:203 7:143079642-143109642_reg_Trpm5:-, 59890 candidate probes\n",
      "--- 33323 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.629s, 478 probes kept.\n",
      "-- check region:204 4:116152601-116182601_reg_Tspan1:-, 59714 candidate probes\n",
      "--- 26263 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.205s, 413 probes kept.\n",
      "-- check region:205 9:44788072-44818072_reg_Ttc36:-, 59852 candidate probes\n",
      "--- 24350 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.196s, 444 probes kept.\n",
      "-- check region:206 10:75928057-75958057_reg_Vpreb3:+, 59480 candidate probes\n",
      "--- 26120 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.176s, 390 probes kept.\n",
      "-- check region:207 1:164920527-164950527_reg_Xcl1:-, 59846 candidate probes\n",
      "--- 25252 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.144s, 408 probes kept.\n",
      "-- check region:208 7:127072328-127102328_reg_Zg16:-, 59334 candidate probes\n",
      "--- 21661 probes passed check_dic, GC and Tm selection.\n",
      "finish in 2.013s, 345 probes kept.\n"
     ]
    }
   ],
   "source": [
    "pbs, pb_scores = pb_designer.check_probes()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "occasional-charter",
   "metadata": {},
   "source": [
    "### save probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "id": "helpful-manual",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\reports\\merged_probes.pbr\n",
      "-- number of probes kept: 447331\n"
     ]
    }
   ],
   "source": [
    "overwrite_savefile = True \n",
    "if not os.path.exists(pb_designer.save_file) or overwrite_savefile:\n",
    "    #pb_designer.plots()\n",
    "    #pb_designer.save_csv()\n",
    "    pb_designer.save_to_file()\n",
    "print(f\"-- number of probes kept: {len(pb_designer.kept_probes)}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 130,
   "id": "completed-winter",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'ATGTTACATGTATATGTGAGGTACGGTATGCACACATGCA'"
      ]
     },
     "execution_count": 130,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 126,
   "id": "historical-license",
   "metadata": {},
   "outputs": [],
   "source": [
    "from Bio.SeqUtils import MeltingTemp\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 133,
   "id": "official-buying",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "73.4869025515149"
      ]
     },
     "execution_count": 133,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "MeltingTemp.Tm_NN(list(pb_designer.kept_probes.keys())[0].decode(), Na=330)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 134,
   "id": "computational-criticism",
   "metadata": {},
   "outputs": [],
   "source": [
    "from Bio.SeqUtils import GC\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 136,
   "id": "velvet-harassment",
   "metadata": {},
   "outputs": [],
   "source": [
    "?GC"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "eastern-empire",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.9"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
