{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "deadly-absence",
   "metadata": {},
   "source": [
    "# Design CTP-10 Aire DNA-MERFISH library for all super enhancers\n",
    "\n",
    "by Pu Zheng\n",
    "\n",
    "2021.4.5\n",
    "\n",
    "Super-enhancers are called by ..."
   ]
  },
  {
   "cell_type": "markdown",
   "id": "delayed-belgium",
   "metadata": {},
   "source": [
    "# Table of contents\n",
    "\n",
    "\n",
    "> 0. [Minimum required packages and settings](#0)\n",
    ">>\n",
    ">> 0.1: [import required packages](#0.1)\n",
    ">\n",
    "> 1. [Extract region sequences](#1)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "fuzzy-reasoning",
   "metadata": {},
   "source": [
    "<a id='0.1'></a>\n",
    "## 0.1 load required packages"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "packed-worse",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "41280\n"
     ]
    }
   ],
   "source": [
    "%run \"..\\..\\Startup_py3.py\"\n",
    "sys.path.append(r\"..\\..\\..\\..\\Documents\")\n",
    "\n",
    "import ImageAnalysis3 as ia\n",
    "%matplotlib notebook\n",
    "\n",
    "from ImageAnalysis3 import *\n",
    "print(os.getpid())\n",
    "\n",
    "# library design specific tools\n",
    "from ImageAnalysis3.library_tools import LibraryDesigner as ld\n",
    "from ImageAnalysis3.library_tools import LibraryTools as lt\n",
    "# biopython imports\n",
    "from Bio import SeqIO\n",
    "from Bio.Seq import Seq\n",
    "from Bio.SeqRecord import SeqRecord\n",
    "from Bio.Blast.Applications import NcbiblastnCommandline\n",
    "from Bio.Blast import NCBIXML"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "severe-obligation",
   "metadata": {},
   "source": [
    "<a id='1'></a>\n",
    "# 1 Extract region sequences"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "single-palmer",
   "metadata": {},
   "outputs": [],
   "source": [
    "## Some folders\n",
    "# human genome\n",
    "reference_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl'\n",
    "genome_folder = os.path.join(reference_folder, 'Genome')\n",
    "# Library directories\n",
    "pool_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "id": "sunset-queen",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- library_folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\n",
      "-- sequence_folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\n",
      "-- report_folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\reports\n"
     ]
    }
   ],
   "source": [
    "resolution = 10000\n",
    "flanking = 100000\n",
    "# folder for sub-pool\n",
    "library_folder = os.path.join(pool_folder, f'Promoter_enhancer_pairs')\n",
    "if not os.path.exists(library_folder):\n",
    "    print(f\"create library folder: {library_folder}\")\n",
    "    os.makedirs(library_folder)\n",
    "# folder for fasta sequences\n",
    "sequence_folder = os.path.join(library_folder, 'sequences')\n",
    "if not os.path.exists(sequence_folder):\n",
    "    print(f\"create sequence folder: {sequence_folder}\")\n",
    "    os.makedirs(sequence_folder)\n",
    "# folder to save result probes\n",
    "report_folder = os.path.join(library_folder, 'reports')\n",
    "if not os.path.exists(report_folder):\n",
    "    print(f\"create report folder: {report_folder}\")\n",
    "    os.makedirs(report_folder)\n",
    "    \n",
    "print(f\"-- library_folder: {library_folder}\")\n",
    "print(f\"-- sequence_folder: {sequence_folder}\")\n",
    "print(f\"-- report_folder: {report_folder}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "environmental-street",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Input region file is: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\Yuan_selected_pairs.txt\n",
      "- 3 regions loaded from file: Yuan_selected_pairs.txt\n"
     ]
    }
   ],
   "source": [
    "reload(library_tools.sequences)\n",
    "region_file = os.path.join(library_folder, 'Yuan_selected_pairs.txt')\n",
    "region_dicts = library_tools.sequences.read_region_file(region_file)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "id": "assured-technology",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- load sequence: 1, size=195471971\n",
      "-- load sequence: 10, size=130694993\n",
      "-- load sequence: 11, size=122082543\n",
      "-- load sequence: 12, size=120129022\n",
      "-- load sequence: 13, size=120421639\n",
      "-- load sequence: 14, size=124902244\n",
      "-- load sequence: 15, size=104043685\n",
      "-- load sequence: 16, size=98207768\n",
      "-- load sequence: 17, size=94987271\n",
      "-- load sequence: 18, size=90702639\n",
      "-- load sequence: 19, size=61431566\n",
      "-- load sequence: 2, size=182113224\n",
      "-- load sequence: 3, size=160039680\n",
      "-- load sequence: 4, size=156508116\n",
      "-- load sequence: 5, size=151834684\n",
      "-- load sequence: 6, size=149736546\n",
      "-- load sequence: 7, size=145441459\n",
      "-- load sequence: 8, size=129401213\n",
      "-- load sequence: 9, size=124595110\n",
      "-- load sequence: MT, size=16299\n",
      "-- load sequence: X, size=171031299\n",
      "-- load sequence: Y, size=91744698\n",
      "-- load sequence: JH584299.1, size=953012\n",
      "-- load sequence: GL456233.1, size=336933\n",
      "-- load sequence: JH584301.1, size=259875\n",
      "-- load sequence: GL456211.1, size=241735\n",
      "-- load sequence: GL456350.1, size=227966\n",
      "-- load sequence: JH584293.1, size=207968\n",
      "-- load sequence: GL456221.1, size=206961\n",
      "-- load sequence: JH584297.1, size=205776\n",
      "-- load sequence: JH584296.1, size=199368\n",
      "-- load sequence: GL456354.1, size=195993\n",
      "-- load sequence: JH584294.1, size=191905\n",
      "-- load sequence: JH584298.1, size=184189\n",
      "-- load sequence: JH584300.1, size=182347\n",
      "-- load sequence: GL456219.1, size=175968\n",
      "-- load sequence: GL456210.1, size=169725\n",
      "-- load sequence: JH584303.1, size=158099\n",
      "-- load sequence: JH584302.1, size=155838\n",
      "-- load sequence: GL456212.1, size=153618\n",
      "-- load sequence: JH584304.1, size=114452\n",
      "-- load sequence: GL456379.1, size=72385\n",
      "-- load sequence: GL456216.1, size=66673\n",
      "-- load sequence: GL456393.1, size=55711\n",
      "-- load sequence: GL456366.1, size=47073\n",
      "-- load sequence: GL456367.1, size=42057\n",
      "-- load sequence: GL456239.1, size=40056\n",
      "-- load sequence: GL456213.1, size=39340\n",
      "-- load sequence: GL456383.1, size=38659\n",
      "-- load sequence: GL456385.1, size=35240\n",
      "-- load sequence: GL456360.1, size=31704\n",
      "-- load sequence: GL456378.1, size=31602\n",
      "-- load sequence: GL456389.1, size=28772\n",
      "-- load sequence: GL456372.1, size=28664\n",
      "-- load sequence: GL456370.1, size=26764\n",
      "-- load sequence: GL456381.1, size=25871\n",
      "-- load sequence: GL456387.1, size=24685\n",
      "-- load sequence: GL456390.1, size=24668\n",
      "-- load sequence: GL456394.1, size=24323\n",
      "-- load sequence: GL456392.1, size=23629\n",
      "-- load sequence: GL456382.1, size=23158\n",
      "-- load sequence: GL456359.1, size=22974\n",
      "-- load sequence: GL456396.1, size=21240\n",
      "-- load sequence: GL456368.1, size=20208\n",
      "-- load sequence: JH584292.1, size=14945\n",
      "-- load sequence: JH584295.1, size=1976\n",
      "- 66 sequences loaded.\n"
     ]
    }
   ],
   "source": [
    "reload(library_tools.references)\n",
    "reload(library_tools.sequences)\n",
    "seq_rd = library_tools.sequences.RNA_sequence_reader(genome_folder, flanking=flanking, resolution=resolution)\n",
    "seq_rd.load_sequences()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "id": "nearby-receiver",
   "metadata": {},
   "outputs": [],
   "source": [
    "for _reg_d in region_dicts:\n",
    "    _reg_info = _reg_d['Region']\n",
    "    _reg_d['Chr'] = _reg_info.split(':')[0]\n",
    "    _reg_d['Start'] = int(_reg_info.split(':')[1].split('-')[0])\n",
    "    _reg_d['End'] = int(_reg_info.split(':')[1].split('-')[1])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "id": "transparent-savage",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[{'Gene': 'Camk1',\n",
       "  'Region': '6:113334124-113671161',\n",
       "  'Strand': '-',\n",
       "  'Genome': 'GRCm38',\n",
       "  'Chr': '6',\n",
       "  'Start': 113334124,\n",
       "  'End': 113671161},\n",
       " {'Gene': 'Hoxd10',\n",
       "  'Region': '2:74691924-75665333',\n",
       "  'Strand': '+',\n",
       "  'Genome': 'GRCm38',\n",
       "  'Chr': '2',\n",
       "  'Start': 74691924,\n",
       "  'End': 75665333},\n",
       " {'Gene': 'Spp2',\n",
       "  'Region': '1:88406961-89126294',\n",
       "  'Strand': '+',\n",
       "  'Genome': 'GRCm38',\n",
       "  'Chr': '1',\n",
       "  'Start': 88406961,\n",
       "  'End': 89126294}]"
      ]
     },
     "execution_count": 28,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "region_dicts"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "id": "fluid-ukraine",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- searching among 1 references\n",
      "-- a match found in record: 6.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 2.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 1.\n",
      "3\n"
     ]
    }
   ],
   "source": [
    "for _reg_d in region_dicts:\n",
    "    seq_rd.find_sequence_for_region(_reg_d)\n",
    "print(len(seq_rd.seq_dict))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "id": "revolutionary-niagara",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- saving sequences into folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_2.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_3.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_4.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_5.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_6.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_7.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_8.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_9.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_10.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_11.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_12.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_13.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_14.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_15.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_16.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_17.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_18.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_19.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_20.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_21.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_22.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_23.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_24.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_25.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_26.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_27.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_28.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_29.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_30.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_31.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_32.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_33.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_34.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_35.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_36.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_37.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_38.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_39.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_40.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_41.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_42.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_43.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_44.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_45.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_46.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_47.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_48.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_49.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_50.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_51.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_52.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_53.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_54.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_2.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_3.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_4.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_5.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_6.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_7.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_8.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_9.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_10.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_11.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_12.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_13.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_14.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_15.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_16.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_17.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_18.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_19.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_20.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_21.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_22.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_23.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_24.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_25.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_26.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_27.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_28.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_29.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_30.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_31.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_32.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_33.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_34.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_35.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_36.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_37.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_38.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_39.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_40.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_41.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_42.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_43.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_44.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_45.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_46.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_47.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_48.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_49.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_50.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_51.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_52.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_53.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_54.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_55.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_56.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_57.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_58.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_59.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_60.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_61.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_62.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_63.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_64.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_65.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_66.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_67.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_68.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_69.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_70.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_71.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_72.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_73.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_74.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_75.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_76.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_77.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_78.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_79.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_80.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_81.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_82.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_83.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_84.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_85.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_86.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_87.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_88.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_89.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_90.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_91.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_92.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_93.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_94.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_95.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_96.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_97.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_98.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_99.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_100.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_101.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_102.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_103.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_104.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_105.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_106.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_107.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_108.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_109.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_110.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_111.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_112.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_113.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_114.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_115.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_116.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_117.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_118.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_2.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_3.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_4.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_5.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_6.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_7.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_8.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_9.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_10.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_11.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_12.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_13.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_14.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_15.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_16.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_17.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_18.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_19.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_20.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_21.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_22.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_23.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_24.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_25.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_26.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_27.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_28.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_29.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_30.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_31.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_32.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_33.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_34.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_35.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_36.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_37.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_38.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_39.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_40.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_41.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_42.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_43.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_44.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_45.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_46.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_47.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_48.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_49.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_50.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_51.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_52.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_53.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_54.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_55.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_56.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_57.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_58.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_59.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_60.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_61.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_62.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_63.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_64.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_65.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_66.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_67.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_68.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_69.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_70.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_71.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_72.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_73.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_74.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_75.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_76.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_77.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_78.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_79.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_80.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_81.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_82.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_83.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_84.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_85.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_86.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_87.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_88.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_89.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_90.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_91.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_92.fasta\n"
     ]
    }
   ],
   "source": [
    "seq_rd.save_sequences(sequence_folder)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "infinite-musical",
   "metadata": {},
   "source": [
    "<a id='2'></a>\n",
    "# 2. Design probe targeting sequences by probe_designer\n",
    "\n",
    "run probe_designer, remeber to clear the memory usage because each table should take ~32GB"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "acting-religion",
   "metadata": {},
   "source": [
    "<a id='2.1'></a>\n",
    "## 2.1 Construct count table with all the 17-mers in the genome\n",
    "\n",
    "Only do this if you don't have pre-built 17-mer\n",
    "\n",
    "However you can do almost the same thing for your own library during quality check\n",
    "\n",
    "This library requires mm10 genome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "id": "greatest-architecture",
   "metadata": {},
   "outputs": [],
   "source": [
    "overwrite_table = False"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "hollow-turner",
   "metadata": {},
   "source": [
    "### construct map for whole genome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "id": "close-mount",
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "reload(library_tools.design)\n",
    " \n",
    "genome_table_file = os.path.join(reference_folder, 'GRCm38_genome_17w.npy')\n",
    "\n",
    "if not os.path.exists(genome_table_file) or overwrite_table:\n",
    "    # genome\n",
    "    _genome_filenames = [os.path.join(genome_folder, _fl) \n",
    "         for _fl in os.listdir(genome_folder) \n",
    "         if _fl.split(os.extsep)[-1]=='fasta' or _fl.split(os.extsep)[-1]=='fa']\n",
    "    print(len(_genome_filenames))\n",
    "\n",
    "    ct = library_tools.design.countTable(word=17,save_file=genome_table_file, \n",
    "                       sparse=False)\n",
    "    ct.verbose=True\n",
    "\n",
    "    ct.read(_genome_filenames) # read sequences from fasta files\n",
    "\n",
    "    ct.consume_loaded(num_threads=24) # convert sequences into integers\n",
    "\n",
    "    ct.complete(verbose=True)\n",
    "\n",
    "    ct.save()\n",
    "\n",
    "    # clear RAM if contructed countable \n",
    "    del(ct)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "immune-strike",
   "metadata": {},
   "source": [
    "### construct map for transcriptome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "id": "opponent-explorer",
   "metadata": {},
   "outputs": [],
   "source": [
    "from tqdm import tqdm\n",
    "# transcriptome\n",
    "transcriptome_folder = os.path.join(reference_folder, 'Transcriptome')\n",
    "\n",
    "transcriptome_table_file = os.path.join(reference_folder, 'GRCm38_transcriptome_17w.npy')\n",
    "\n",
    "if not os.path.exists(transcriptome_table_file) or overwrite_table:\n",
    "    # transcriptome\n",
    "    _transcriptome_filenames = [os.path.join(transcriptome_folder, _fl) \n",
    "         for _fl in os.listdir(transcriptome_folder) \n",
    "         if _fl.split(os.extsep)[-1]=='fasta' or _fl.split(os.extsep)[-1]=='fa']\n",
    "    print(len(_transcriptome_filenames))\n",
    "    \n",
    "    ct = library_tools.design.countTable(word=17,save_file=transcriptome_table_file, \n",
    "                       sparse=False)\n",
    "    ct.verbose=True\n",
    "\n",
    "    ct.read(_transcriptome_filenames) # read sequences from fasta files\n",
    "\n",
    "    ct.consume_loaded(num_threads=24) # convert sequences into integers\n",
    "\n",
    "    ct.complete(verbose=True)\n",
    "\n",
    "    ct.save()\n",
    "    \n",
    "    # clear RAM if contructed countable \n",
    "    del(ct)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "psychological-router",
   "metadata": {},
   "source": [
    "### construct map for repeats from RepBase"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "id": "statistical-killer",
   "metadata": {},
   "outputs": [],
   "source": [
    "from tqdm import tqdm\n",
    "# repeat\n",
    "repeat_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\Repbase'\n",
    "\n",
    "repeat_table_file = os.path.join(reference_folder, 'Repbase_v2603_repeat_17w.npy')\n",
    "\n",
    "if not os.path.exists(repeat_table_file) or overwrite_table:\n",
    "    # repeat\n",
    "    _repeat_filenames = [os.path.join(repeat_folder, _fl) \n",
    "         for _fl in os.listdir(repeat_folder) \n",
    "         if _fl.split(os.extsep)[-1]=='fasta' or _fl.split(os.extsep)[-1]=='fa']\n",
    "    print(len(_repeat_filenames))\n",
    "    \n",
    "    ct = library_tools.design.countTable(word=17,save_file=repeat_table_file, \n",
    "                       sparse=False)\n",
    "    ct.verbose=True\n",
    "\n",
    "    ct.read(_repeat_filenames) # read sequences from fasta files\n",
    "\n",
    "    ct.consume_loaded(num_threads=24) # convert sequences into integers\n",
    "\n",
    "    ct.complete(verbose=True)\n",
    "\n",
    "    ct.save()\n",
    "    \n",
    "    # clear RAM if contructed countable \n",
    "    del(ct)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "accomplished-transparency",
   "metadata": {},
   "source": [
    "<a id='2.2'></a>\n",
    "## 2.2 Design probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "id": "binary-berlin",
   "metadata": {},
   "outputs": [],
   "source": [
    "# required parameters\n",
    "resolution = 0\n",
    "\n",
    "## required folders\n",
    "## Some folders\n",
    "# human genome\n",
    "reference_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl'\n",
    "genome_folder = os.path.join(reference_folder, 'Genome')\n",
    "# Library directories\n",
    "pool_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire'\n",
    "\n",
    "library_folder = os.path.join(pool_folder, f'Promoter_enhancer_pairs')\n",
    "sequence_folder = os.path.join(library_folder, 'sequences')\n",
    "report_folder = os.path.join(library_folder, 'reports')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "id": "beautiful-bookmark",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "264 regions loaded to design probes.\n",
      "\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\reports\\merged_probes.pbr\n"
     ]
    }
   ],
   "source": [
    "# requires pre_defined genome_folder and library_folder\n",
    "# Indices\n",
    "genome_index = os.path.join(reference_folder, 'GRCm38_genome_17w.npy')\n",
    "transcriptome_index = os.path.join(reference_folder, 'GRCm38_transcriptome_17w.npy') \n",
    "repeat_index = os.path.join(reference_folder, 'Repbase_v2603_repeat_17w.npy')\n",
    "#ref_merfish_index = os.path.join(reference_folder, 'M1_meng_MERFISH_17w.npy') # merfish designed by Meng\n",
    "# get input files \n",
    "input_files = glob.glob(os.path.join(sequence_folder, '*.fasta'))\n",
    "\n",
    "print(f\"{len(input_files)} regions loaded to design probes.\")\n",
    "\n",
    "if not os.path.exists(report_folder):\n",
    "    os.makedirs(report_folder)\n",
    "    \n",
    "# filename to save probe reports\n",
    "save_file = os.path.join(report_folder, f'merged_probes.pbr')\n",
    "print(save_file)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "forbidden-provider",
   "metadata": {},
   "source": [
    "### create pb_designer class"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "id": "massive-yield",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Probe designer derived from Bogdan Bintu:\n",
      "https://github.com/BogdanBintu/ChromatinImaging/blob/master/LibraryDesign/LibraryDesigner.py\n",
      "by Pu Zheng, 2020.11\n",
      "\n",
      "Major changes:\n",
      "    1. allow design of two strands\n",
      "    2. separate reverse_complement (rev_com) and from two strands (two_stranded) as \n",
      "    two different inputs for map_dic and sequence_dic\n",
      "    3. replace 'local_genome' with 'self_sequences' to be more explicit, and only \n",
      "    exclude the counts for the corresponding self_sequence within each input. \n",
      "\n",
      "Key information:\n",
      "    - number of input_sequence(s): 264\n",
      "    - save_file location: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\reports\\merged_probes.pbr\n",
      "\n",
      "- Fail to load from savefile: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\reports\\merged_probes.pbr, file doesn't exist.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "False"
      ]
     },
     "execution_count": 41,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "reload(library_tools)\n",
    "reload(library_tools.design)\n",
    "\n",
    "pb_designer = library_tools.design.pb_reports_class(\n",
    "    sequence_dic={'file':input_files,\n",
    "                  'rev_com':True, # design two strands\n",
    "                  'two_stranded':True},\n",
    "    map_dic={'genome':{'file':genome_index,'rev_com':False,'two_stranded':True},\n",
    "             'transcriptome':{'file':transcriptome_index,'rev_com':True,'two_stranded':False},\n",
    "             'rep_genome':{'file':repeat_index,'rev_com':False,'two_stranded':True},\n",
    "             'self_sequences':{'file':input_files,'force_list':True,'rev_com':False,'two_stranded':True},\n",
    "             #'ref_merfish':{'file':ref_merfish_index,'rev_com':False,'two_stranded':True},\n",
    "             },\n",
    "        save_file=save_file,\n",
    "    params_dic={'word_size':17,'pb_len':40,'buffer_len':1,'max_count':2**16-1,\n",
    "                'check_on_go': False, # whether automatically check probes\n",
    "                'auto': False, # whether automatically convert reference maps\n",
    "               },\n",
    "    check_dic={('genome','self_sequences'): (40-17+1)+(30-17+1),\n",
    "               'rep_genome': 0,\n",
    "               'transcriptome': (30-17+1),\n",
    "               'gc':[0.25,0.75],\n",
    "               'tm': 37+0.62*50+5,\n",
    "               }\n",
    "    )\n",
    "print(pb_designer)\n",
    "pb_designer.load_from_file(load_probes_only=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "incoming-texture",
   "metadata": {},
   "source": [
    "### calculate probe reports"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "id": "weird-starter",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- setting attribute: map_genome\n",
      "--- finish map_genome in 89.389s.\n",
      "-- setting attribute: map_transcriptome\n",
      "--- finish map_transcriptome in 95.823s.\n",
      "-- setting attribute: map_rep_genome\n",
      "--- finish map_rep_genome in 88.500s.\n",
      "Time(s): 273.7111585140228\n",
      "- Designing targeting sequence for 264 regions\n",
      "-- designing region: 6:113234124-113244124_reg_Camk1-1:- 0, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.098s.\n",
      "in 5.397s.\n",
      "-- designing region: 6:113324124-113334124_reg_Camk1-10:- 1, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_10.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.094s.\n",
      "in 5.876s.\n",
      "-- designing region: 6:113334124-113344124_reg_Camk1-11:- 2, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_11.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.885s.\n",
      "-- designing region: 6:113344124-113354124_reg_Camk1-12:- 3, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_12.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.867s.\n",
      "-- designing region: 6:113354124-113364124_reg_Camk1-13:- 4, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_13.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.085s.\n",
      "in 5.873s.\n",
      "-- designing region: 6:113364124-113374124_reg_Camk1-14:- 5, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_14.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.896s.\n",
      "-- designing region: 6:113374124-113384124_reg_Camk1-15:- 6, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_15.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.884s.\n",
      "-- designing region: 6:113384124-113394124_reg_Camk1-16:- 7, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_16.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.088s.\n",
      "in 5.898s.\n",
      "-- designing region: 6:113394124-113404124_reg_Camk1-17:- 8, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_17.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.095s.\n",
      "in 5.912s.\n",
      "-- designing region: 6:113404124-113414124_reg_Camk1-18:- 9, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_18.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.894s.\n",
      "-- designing region: 6:113414124-113424124_reg_Camk1-19:- 10, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_19.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.088s.\n",
      "in 6.080s.\n",
      "-- designing region: 6:113244124-113254124_reg_Camk1-2:- 11, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_2.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.094s.\n",
      "in 6.406s.\n",
      "-- designing region: 6:113424124-113434124_reg_Camk1-20:- 12, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_20.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.094s.\n",
      "in 6.419s.\n",
      "-- designing region: 6:113434124-113444124_reg_Camk1-21:- 13, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_21.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.094s.\n",
      "in 6.479s.\n",
      "-- designing region: 6:113444124-113454124_reg_Camk1-22:- 14, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_22.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.908s.\n",
      "-- designing region: 6:113454124-113464124_reg_Camk1-23:- 15, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_23.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.094s.\n",
      "in 5.921s.\n",
      "-- designing region: 6:113464124-113474124_reg_Camk1-24:- 16, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_24.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.085s.\n",
      "in 5.911s.\n",
      "-- designing region: 6:113474124-113484124_reg_Camk1-25:- 17, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_25.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.932s.\n",
      "-- designing region: 6:113484124-113494124_reg_Camk1-26:- 18, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_26.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.093s.\n",
      "in 5.923s.\n",
      "-- designing region: 6:113494124-113504124_reg_Camk1-27:- 19, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_27.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.912s.\n",
      "-- designing region: 6:113504124-113514124_reg_Camk1-28:- 20, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_28.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.094s.\n",
      "in 5.914s.\n",
      "-- designing region: 6:113514124-113524124_reg_Camk1-29:- 21, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_29.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.096s.\n",
      "in 5.929s.\n",
      "-- designing region: 6:113254124-113264124_reg_Camk1-3:- 22, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_3.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.085s.\n",
      "in 5.909s.\n",
      "-- designing region: 6:113524124-113534124_reg_Camk1-30:- 23, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_30.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.918s.\n",
      "-- designing region: 6:113534124-113544124_reg_Camk1-31:- 24, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_31.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.906s.\n",
      "-- designing region: 6:113544124-113554124_reg_Camk1-32:- 25, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_32.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.914s.\n",
      "-- designing region: 6:113554124-113564124_reg_Camk1-33:- 26, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_33.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.090s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "in 5.917s.\n",
      "-- designing region: 6:113564124-113574124_reg_Camk1-34:- 27, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_34.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.920s.\n",
      "-- designing region: 6:113574124-113584124_reg_Camk1-35:- 28, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_35.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.089s.\n",
      "in 5.899s.\n",
      "-- designing region: 6:113584124-113594124_reg_Camk1-36:- 29, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_36.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.928s.\n",
      "-- designing region: 6:113594124-113604124_reg_Camk1-37:- 30, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_37.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.898s.\n",
      "-- designing region: 6:113604124-113614124_reg_Camk1-38:- 31, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_38.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.093s.\n",
      "in 5.919s.\n",
      "-- designing region: 6:113614124-113624124_reg_Camk1-39:- 32, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_39.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.910s.\n",
      "-- designing region: 6:113264124-113274124_reg_Camk1-4:- 33, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_4.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.923s.\n",
      "-- designing region: 6:113624124-113634124_reg_Camk1-40:- 34, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_40.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.910s.\n",
      "-- designing region: 6:113634124-113644124_reg_Camk1-41:- 35, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_41.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.094s.\n",
      "in 5.955s.\n",
      "-- designing region: 6:113644124-113654124_reg_Camk1-42:- 36, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_42.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.896s.\n",
      "-- designing region: 6:113654124-113664124_reg_Camk1-43:- 37, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_43.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.085s.\n",
      "in 5.902s.\n",
      "-- designing region: 6:113664124-113674124_reg_Camk1-44:- 38, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_44.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.893s.\n",
      "-- designing region: 6:113674124-113684124_reg_Camk1-45:- 39, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_45.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.085s.\n",
      "in 5.904s.\n",
      "-- designing region: 6:113684124-113694124_reg_Camk1-46:- 40, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_46.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 6.474s.\n",
      "-- designing region: 6:113694124-113704124_reg_Camk1-47:- 41, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_47.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.903s.\n",
      "-- designing region: 6:113704124-113714124_reg_Camk1-48:- 42, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_48.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.898s.\n",
      "-- designing region: 6:113714124-113724124_reg_Camk1-49:- 43, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_49.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.918s.\n",
      "-- designing region: 6:113274124-113284124_reg_Camk1-5:- 44, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_5.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.093s.\n",
      "in 5.912s.\n",
      "-- designing region: 6:113724124-113734124_reg_Camk1-50:- 45, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_50.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.908s.\n",
      "-- designing region: 6:113734124-113744124_reg_Camk1-51:- 46, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_51.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.912s.\n",
      "-- designing region: 6:113744124-113754124_reg_Camk1-52:- 47, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_52.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.909s.\n",
      "-- designing region: 6:113754124-113764124_reg_Camk1-53:- 48, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_53.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.085s.\n",
      "in 5.912s.\n",
      "-- designing region: 6:113764124-113774124_reg_Camk1-54:- 49, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_54.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.093s.\n",
      "in 5.915s.\n",
      "-- designing region: 6:113284124-113294124_reg_Camk1-6:- 50, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_6.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.088s.\n",
      "in 5.905s.\n",
      "-- designing region: 6:113294124-113304124_reg_Camk1-7:- 51, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_7.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.918s.\n",
      "-- designing region: 6:113304124-113314124_reg_Camk1-8:- 52, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_8.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.088s.\n",
      "in 5.905s.\n",
      "-- designing region: 6:113314124-113324124_reg_Camk1-9:- 53, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Camk1_reg_9.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.923s.\n",
      "-- designing region: 2:74591924-74601924_reg_Hoxd10-1:+ 54, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "in 6.620s.\n",
      "-- designing region: 2:74681924-74691924_reg_Hoxd10-10:+ 55, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_10.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.901s.\n",
      "-- designing region: 2:75581924-75591924_reg_Hoxd10-100:+ 56, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_100.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.919s.\n",
      "-- designing region: 2:75591924-75601924_reg_Hoxd10-101:+ 57, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_101.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.094s.\n",
      "in 5.910s.\n",
      "-- designing region: 2:75601924-75611924_reg_Hoxd10-102:+ 58, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_102.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.902s.\n",
      "-- designing region: 2:75611924-75621924_reg_Hoxd10-103:+ 59, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_103.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.910s.\n",
      "-- designing region: 2:75621924-75631924_reg_Hoxd10-104:+ 60, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_104.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.894s.\n",
      "-- designing region: 2:75631924-75641924_reg_Hoxd10-105:+ 61, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_105.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.100s.\n",
      "in 5.921s.\n",
      "-- designing region: 2:75641924-75651924_reg_Hoxd10-106:+ 62, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_106.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.903s.\n",
      "-- designing region: 2:75651924-75661924_reg_Hoxd10-107:+ 63, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_107.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.906s.\n",
      "-- designing region: 2:75661924-75671924_reg_Hoxd10-108:+ 64, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_108.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.906s.\n",
      "-- designing region: 2:75671924-75681924_reg_Hoxd10-109:+ 65, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_109.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.913s.\n",
      "-- designing region: 2:74691924-74701924_reg_Hoxd10-11:+ 66, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_11.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.919s.\n",
      "-- designing region: 2:75681924-75691924_reg_Hoxd10-110:+ 67, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_110.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.918s.\n",
      "-- designing region: 2:75691924-75701924_reg_Hoxd10-111:+ 68, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_111.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.092s.\n",
      "in 5.921s.\n",
      "-- designing region: 2:75701924-75711924_reg_Hoxd10-112:+ 69, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_112.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.900s.\n",
      "-- designing region: 2:75711924-75721924_reg_Hoxd10-113:+ 70, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_113.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.085s.\n",
      "in 6.912s.\n",
      "-- designing region: 2:75721924-75731924_reg_Hoxd10-114:+ 71, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_114.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.882s.\n",
      "-- designing region: 2:75731924-75741924_reg_Hoxd10-115:+ 72, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_115.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.894s.\n",
      "-- designing region: 2:75741924-75751924_reg_Hoxd10-116:+ 73, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_116.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.085s.\n",
      "in 5.891s.\n",
      "-- designing region: 2:75751924-75761924_reg_Hoxd10-117:+ 74, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_117.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.898s.\n",
      "-- designing region: 2:75761924-75771924_reg_Hoxd10-118:+ 75, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_118.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.898s.\n",
      "-- designing region: 2:74701924-74711924_reg_Hoxd10-12:+ 76, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_12.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.900s.\n",
      "-- designing region: 2:74711924-74721924_reg_Hoxd10-13:+ 77, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_13.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.088s.\n",
      "in 5.901s.\n",
      "-- designing region: 2:74721924-74731924_reg_Hoxd10-14:+ 78, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_14.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.905s.\n",
      "-- designing region: 2:74731924-74741924_reg_Hoxd10-15:+ 79, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_15.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.909s.\n",
      "-- designing region: 2:74741924-74751924_reg_Hoxd10-16:+ 80, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_16.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.088s.\n",
      "in 5.908s.\n",
      "-- designing region: 2:74751924-74761924_reg_Hoxd10-17:+ 81, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_17.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.904s.\n",
      "-- designing region: 2:74761924-74771924_reg_Hoxd10-18:+ 82, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_18.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "in 5.907s.\n",
      "-- designing region: 2:74771924-74781924_reg_Hoxd10-19:+ 83, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_19.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.098s.\n",
      "in 5.907s.\n",
      "-- designing region: 2:74601924-74611924_reg_Hoxd10-2:+ 84, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_2.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.907s.\n",
      "-- designing region: 2:74781924-74791924_reg_Hoxd10-20:+ 85, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_20.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.085s.\n",
      "in 5.897s.\n",
      "-- designing region: 2:74791924-74801924_reg_Hoxd10-21:+ 86, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_21.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.903s.\n",
      "-- designing region: 2:74801924-74811924_reg_Hoxd10-22:+ 87, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_22.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.095s.\n",
      "in 5.908s.\n",
      "-- designing region: 2:74811924-74821924_reg_Hoxd10-23:+ 88, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_23.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.089s.\n",
      "in 5.900s.\n",
      "-- designing region: 2:74821924-74831924_reg_Hoxd10-24:+ 89, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_24.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.085s.\n",
      "in 5.901s.\n",
      "-- designing region: 2:74831924-74841924_reg_Hoxd10-25:+ 90, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_25.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.095s.\n",
      "in 5.898s.\n",
      "-- designing region: 2:74841924-74851924_reg_Hoxd10-26:+ 91, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_26.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 7.028s.\n",
      "-- designing region: 2:74851924-74861924_reg_Hoxd10-27:+ 92, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_27.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.882s.\n",
      "-- designing region: 2:74861924-74871924_reg_Hoxd10-28:+ 93, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_28.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.894s.\n",
      "-- designing region: 2:74871924-74881924_reg_Hoxd10-29:+ 94, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_29.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.886s.\n",
      "-- designing region: 2:74611924-74621924_reg_Hoxd10-3:+ 95, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_3.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.897s.\n",
      "-- designing region: 2:74881924-74891924_reg_Hoxd10-30:+ 96, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_30.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.088s.\n",
      "in 5.899s.\n",
      "-- designing region: 2:74891924-74901924_reg_Hoxd10-31:+ 97, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_31.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.896s.\n",
      "-- designing region: 2:74901924-74911924_reg_Hoxd10-32:+ 98, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_32.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.898s.\n",
      "-- designing region: 2:74911924-74921924_reg_Hoxd10-33:+ 99, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_33.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.901s.\n",
      "-- designing region: 2:74921924-74931924_reg_Hoxd10-34:+ 100, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_34.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.904s.\n",
      "-- designing region: 2:74931924-74941924_reg_Hoxd10-35:+ 101, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_35.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.910s.\n",
      "-- designing region: 2:74941924-74951924_reg_Hoxd10-36:+ 102, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_36.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.900s.\n",
      "-- designing region: 2:74951924-74961924_reg_Hoxd10-37:+ 103, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_37.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.909s.\n",
      "-- designing region: 2:74961924-74971924_reg_Hoxd10-38:+ 104, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_38.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.905s.\n",
      "-- designing region: 2:74971924-74981924_reg_Hoxd10-39:+ 105, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_39.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.908s.\n",
      "-- designing region: 2:74621924-74631924_reg_Hoxd10-4:+ 106, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_4.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.897s.\n",
      "-- designing region: 2:74981924-74991924_reg_Hoxd10-40:+ 107, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_40.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.094s.\n",
      "in 5.917s.\n",
      "-- designing region: 2:74991924-75001924_reg_Hoxd10-41:+ 108, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_41.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.909s.\n",
      "-- designing region: 2:75001924-75011924_reg_Hoxd10-42:+ 109, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_42.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.907s.\n",
      "-- designing region: 2:75011924-75021924_reg_Hoxd10-43:+ 110, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_43.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "in 5.906s.\n",
      "-- designing region: 2:75021924-75031924_reg_Hoxd10-44:+ 111, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_44.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.908s.\n",
      "-- designing region: 2:75031924-75041924_reg_Hoxd10-45:+ 112, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_45.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.906s.\n",
      "-- designing region: 2:75041924-75051924_reg_Hoxd10-46:+ 113, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_46.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.901s.\n",
      "-- designing region: 2:75051924-75061924_reg_Hoxd10-47:+ 114, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_47.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.899s.\n",
      "-- designing region: 2:75061924-75071924_reg_Hoxd10-48:+ 115, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_48.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.088s.\n",
      "in 5.899s.\n",
      "-- designing region: 2:75071924-75081924_reg_Hoxd10-49:+ 116, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_49.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 7.336s.\n",
      "-- designing region: 2:74631924-74641924_reg_Hoxd10-5:+ 117, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_5.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.910s.\n",
      "-- designing region: 2:75081924-75091924_reg_Hoxd10-50:+ 118, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_50.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.910s.\n",
      "-- designing region: 2:75091924-75101924_reg_Hoxd10-51:+ 119, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_51.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.913s.\n",
      "-- designing region: 2:75101924-75111924_reg_Hoxd10-52:+ 120, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_52.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.906s.\n",
      "-- designing region: 2:75111924-75121924_reg_Hoxd10-53:+ 121, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_53.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.911s.\n",
      "-- designing region: 2:75121924-75131924_reg_Hoxd10-54:+ 122, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_54.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.092s.\n",
      "in 5.912s.\n",
      "-- designing region: 2:75131924-75141924_reg_Hoxd10-55:+ 123, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_55.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.910s.\n",
      "-- designing region: 2:75141924-75151924_reg_Hoxd10-56:+ 124, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_56.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.909s.\n",
      "-- designing region: 2:75151924-75161924_reg_Hoxd10-57:+ 125, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_57.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.911s.\n",
      "-- designing region: 2:75161924-75171924_reg_Hoxd10-58:+ 126, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_58.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.905s.\n",
      "-- designing region: 2:75171924-75181924_reg_Hoxd10-59:+ 127, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_59.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.095s.\n",
      "in 5.913s.\n",
      "-- designing region: 2:74641924-74651924_reg_Hoxd10-6:+ 128, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_6.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.906s.\n",
      "-- designing region: 2:75181924-75191924_reg_Hoxd10-60:+ 129, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_60.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.910s.\n",
      "-- designing region: 2:75191924-75201924_reg_Hoxd10-61:+ 130, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_61.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.088s.\n",
      "in 5.899s.\n",
      "-- designing region: 2:75201924-75211924_reg_Hoxd10-62:+ 131, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_62.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.913s.\n",
      "-- designing region: 2:75211924-75221924_reg_Hoxd10-63:+ 132, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_63.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.911s.\n",
      "-- designing region: 2:75221924-75231924_reg_Hoxd10-64:+ 133, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_64.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.908s.\n",
      "-- designing region: 2:75231924-75241924_reg_Hoxd10-65:+ 134, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_65.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.912s.\n",
      "-- designing region: 2:75241924-75251924_reg_Hoxd10-66:+ 135, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_66.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.085s.\n",
      "in 5.910s.\n",
      "-- designing region: 2:75251924-75261924_reg_Hoxd10-67:+ 136, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_67.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.906s.\n",
      "-- designing region: 2:75261924-75271924_reg_Hoxd10-68:+ 137, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_68.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.914s.\n",
      "-- designing region: 2:75271924-75281924_reg_Hoxd10-69:+ 138, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_69.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.088s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "in 5.915s.\n",
      "-- designing region: 2:74651924-74661924_reg_Hoxd10-7:+ 139, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_7.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.091s.\n",
      "in 5.908s.\n",
      "-- designing region: 2:75281924-75291924_reg_Hoxd10-70:+ 140, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_70.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.905s.\n",
      "-- designing region: 2:75291924-75301924_reg_Hoxd10-71:+ 141, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_71.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.088s.\n",
      "in 6.080s.\n",
      "-- designing region: 2:75301924-75311924_reg_Hoxd10-72:+ 142, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_72.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.901s.\n",
      "-- designing region: 2:75311924-75321924_reg_Hoxd10-73:+ 143, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_73.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.908s.\n",
      "-- designing region: 2:75321924-75331924_reg_Hoxd10-74:+ 144, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_74.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.898s.\n",
      "-- designing region: 2:75331924-75341924_reg_Hoxd10-75:+ 145, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_75.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.097s.\n",
      "in 5.915s.\n",
      "-- designing region: 2:75341924-75351924_reg_Hoxd10-76:+ 146, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_76.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.096s.\n",
      "in 5.908s.\n",
      "-- designing region: 2:75351924-75361924_reg_Hoxd10-77:+ 147, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_77.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.903s.\n",
      "-- designing region: 2:75361924-75371924_reg_Hoxd10-78:+ 148, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_78.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.901s.\n",
      "-- designing region: 2:75371924-75381924_reg_Hoxd10-79:+ 149, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_79.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 7.707s.\n",
      "-- designing region: 2:74661924-74671924_reg_Hoxd10-8:+ 150, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_8.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.895s.\n",
      "-- designing region: 2:75381924-75391924_reg_Hoxd10-80:+ 151, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_80.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.902s.\n",
      "-- designing region: 2:75391924-75401924_reg_Hoxd10-81:+ 152, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_81.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.900s.\n",
      "-- designing region: 2:75401924-75411924_reg_Hoxd10-82:+ 153, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_82.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.906s.\n",
      "-- designing region: 2:75411924-75421924_reg_Hoxd10-83:+ 154, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_83.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.897s.\n",
      "-- designing region: 2:75421924-75431924_reg_Hoxd10-84:+ 155, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_84.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.105s.\n",
      "in 5.921s.\n",
      "-- designing region: 2:75431924-75441924_reg_Hoxd10-85:+ 156, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_85.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.085s.\n",
      "in 5.897s.\n",
      "-- designing region: 2:75441924-75451924_reg_Hoxd10-86:+ 157, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_86.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.910s.\n",
      "-- designing region: 2:75451924-75461924_reg_Hoxd10-87:+ 158, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_87.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.900s.\n",
      "-- designing region: 2:75461924-75471924_reg_Hoxd10-88:+ 159, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_88.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.902s.\n",
      "-- designing region: 2:75471924-75481924_reg_Hoxd10-89:+ 160, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_89.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.891s.\n",
      "-- designing region: 2:74671924-74681924_reg_Hoxd10-9:+ 161, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_9.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.907s.\n",
      "-- designing region: 2:75481924-75491924_reg_Hoxd10-90:+ 162, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_90.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.088s.\n",
      "in 5.900s.\n",
      "-- designing region: 2:75491924-75501924_reg_Hoxd10-91:+ 163, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_91.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.905s.\n",
      "-- designing region: 2:75501924-75511924_reg_Hoxd10-92:+ 164, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_92.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.897s.\n",
      "-- designing region: 2:75511924-75521924_reg_Hoxd10-93:+ 165, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_93.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.095s.\n",
      "in 5.919s.\n",
      "-- designing region: 2:75521924-75531924_reg_Hoxd10-94:+ 166, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_94.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.088s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "in 5.914s.\n",
      "-- designing region: 2:75531924-75541924_reg_Hoxd10-95:+ 167, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_95.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.918s.\n",
      "-- designing region: 2:75541924-75551924_reg_Hoxd10-96:+ 168, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_96.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.088s.\n",
      "in 5.901s.\n",
      "-- designing region: 2:75551924-75561924_reg_Hoxd10-97:+ 169, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_97.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.088s.\n",
      "in 6.411s.\n",
      "-- designing region: 2:75561924-75571924_reg_Hoxd10-98:+ 170, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_98.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.097s.\n",
      "in 5.915s.\n",
      "-- designing region: 2:75571924-75581924_reg_Hoxd10-99:+ 171, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Hoxd10_reg_99.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.911s.\n",
      "-- designing region: 1:88306961-88316961_reg_Spp2-1:+ 172, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.090s.\n",
      "in 5.921s.\n",
      "-- designing region: 1:88396961-88406961_reg_Spp2-10:+ 173, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_10.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.094s.\n",
      "in 5.929s.\n",
      "-- designing region: 1:88406961-88416961_reg_Spp2-11:+ 174, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_11.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.909s.\n",
      "-- designing region: 1:88416961-88426961_reg_Spp2-12:+ 175, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_12.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.920s.\n",
      "-- designing region: 1:88426961-88436961_reg_Spp2-13:+ 176, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_13.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.909s.\n",
      "-- designing region: 1:88436961-88446961_reg_Spp2-14:+ 177, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_14.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.921s.\n",
      "-- designing region: 1:88446961-88456961_reg_Spp2-15:+ 178, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_15.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.089s.\n",
      "in 5.919s.\n",
      "-- designing region: 1:88456961-88466961_reg_Spp2-16:+ 179, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_16.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.922s.\n",
      "-- designing region: 1:88466961-88476961_reg_Spp2-17:+ 180, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_17.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.918s.\n",
      "-- designing region: 1:88476961-88486961_reg_Spp2-18:+ 181, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_18.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.914s.\n",
      "-- designing region: 1:88486961-88496961_reg_Spp2-19:+ 182, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_19.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.912s.\n",
      "-- designing region: 1:88316961-88326961_reg_Spp2-2:+ 183, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_2.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.919s.\n",
      "-- designing region: 1:88496961-88506961_reg_Spp2-20:+ 184, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_20.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.089s.\n",
      "in 5.913s.\n",
      "-- designing region: 1:88506961-88516961_reg_Spp2-21:+ 185, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_21.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.912s.\n",
      "-- designing region: 1:88516961-88526961_reg_Spp2-22:+ 186, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_22.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.917s.\n",
      "-- designing region: 1:88526961-88536961_reg_Spp2-23:+ 187, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_23.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.917s.\n",
      "-- designing region: 1:88536961-88546961_reg_Spp2-24:+ 188, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_24.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.094s.\n",
      "in 5.930s.\n",
      "-- designing region: 1:88546961-88556961_reg_Spp2-25:+ 189, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_25.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 8.200s.\n",
      "-- designing region: 1:88556961-88566961_reg_Spp2-26:+ 190, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_26.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.088s.\n",
      "in 5.922s.\n",
      "-- designing region: 1:88566961-88576961_reg_Spp2-27:+ 191, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_27.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.085s.\n",
      "in 5.905s.\n",
      "-- designing region: 1:88576961-88586961_reg_Spp2-28:+ 192, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_28.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.915s.\n",
      "-- designing region: 1:88586961-88596961_reg_Spp2-29:+ 193, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_29.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.915s.\n",
      "-- designing region: 1:88326961-88336961_reg_Spp2-3:+ 194, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_3.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "in 5.902s.\n",
      "-- designing region: 1:88596961-88606961_reg_Spp2-30:+ 195, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_30.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.097s.\n",
      "in 5.929s.\n",
      "-- designing region: 1:88606961-88616961_reg_Spp2-31:+ 196, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_31.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.093s.\n",
      "in 5.919s.\n",
      "-- designing region: 1:88616961-88626961_reg_Spp2-32:+ 197, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_32.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.094s.\n",
      "in 5.932s.\n",
      "-- designing region: 1:88626961-88636961_reg_Spp2-33:+ 198, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_33.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.917s.\n",
      "-- designing region: 1:88636961-88646961_reg_Spp2-34:+ 199, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_34.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.912s.\n",
      "-- designing region: 1:88646961-88656961_reg_Spp2-35:+ 200, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_35.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.910s.\n",
      "-- designing region: 1:88656961-88666961_reg_Spp2-36:+ 201, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_36.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.917s.\n",
      "-- designing region: 1:88666961-88676961_reg_Spp2-37:+ 202, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_37.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.913s.\n",
      "-- designing region: 1:88676961-88686961_reg_Spp2-38:+ 203, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_38.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.920s.\n",
      "-- designing region: 1:88686961-88696961_reg_Spp2-39:+ 204, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_39.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.924s.\n",
      "-- designing region: 1:88336961-88346961_reg_Spp2-4:+ 205, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_4.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.914s.\n",
      "-- designing region: 1:88696961-88706961_reg_Spp2-40:+ 206, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_40.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.088s.\n",
      "in 5.919s.\n",
      "-- designing region: 1:88706961-88716961_reg_Spp2-41:+ 207, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_41.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.916s.\n",
      "-- designing region: 1:88716961-88726961_reg_Spp2-42:+ 208, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_42.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.909s.\n",
      "-- designing region: 1:88726961-88736961_reg_Spp2-43:+ 209, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_43.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.919s.\n",
      "-- designing region: 1:88736961-88746961_reg_Spp2-44:+ 210, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_44.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.920s.\n",
      "-- designing region: 1:88746961-88756961_reg_Spp2-45:+ 211, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_45.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.924s.\n",
      "-- designing region: 1:88756961-88766961_reg_Spp2-46:+ 212, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_46.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.913s.\n",
      "-- designing region: 1:88766961-88776961_reg_Spp2-47:+ 213, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_47.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.094s.\n",
      "in 5.926s.\n",
      "-- designing region: 1:88776961-88786961_reg_Spp2-48:+ 214, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_48.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.912s.\n",
      "-- designing region: 1:88786961-88796961_reg_Spp2-49:+ 215, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_49.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.084s.\n",
      "in 5.914s.\n",
      "-- designing region: 1:88346961-88356961_reg_Spp2-5:+ 216, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_5.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.094s.\n",
      "in 5.922s.\n",
      "-- designing region: 1:88796961-88806961_reg_Spp2-50:+ 217, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_50.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.924s.\n",
      "-- designing region: 1:88806961-88816961_reg_Spp2-51:+ 218, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_51.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.096s.\n",
      "in 5.921s.\n",
      "-- designing region: 1:88816961-88826961_reg_Spp2-52:+ 219, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_52.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.910s.\n",
      "-- designing region: 1:88826961-88836961_reg_Spp2-53:+ 220, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_53.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.095s.\n",
      "in 5.920s.\n",
      "-- designing region: 1:88836961-88846961_reg_Spp2-54:+ 221, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_54.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.914s.\n",
      "-- designing region: 1:88846961-88856961_reg_Spp2-55:+ 222, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_55.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "in 5.912s.\n",
      "-- designing region: 1:88856961-88866961_reg_Spp2-56:+ 223, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_56.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.920s.\n",
      "-- designing region: 1:88866961-88876961_reg_Spp2-57:+ 224, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_57.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.916s.\n",
      "-- designing region: 1:88876961-88886961_reg_Spp2-58:+ 225, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_58.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.916s.\n",
      "-- designing region: 1:88886961-88896961_reg_Spp2-59:+ 226, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_59.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.085s.\n",
      "in 5.908s.\n",
      "-- designing region: 1:88356961-88366961_reg_Spp2-6:+ 227, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_6.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.916s.\n",
      "-- designing region: 1:88896961-88906961_reg_Spp2-60:+ 228, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_60.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.914s.\n",
      "-- designing region: 1:88906961-88916961_reg_Spp2-61:+ 229, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_61.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.922s.\n",
      "-- designing region: 1:88916961-88926961_reg_Spp2-62:+ 230, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_62.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.911s.\n",
      "-- designing region: 1:88926961-88936961_reg_Spp2-63:+ 231, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_63.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.917s.\n",
      "-- designing region: 1:88936961-88946961_reg_Spp2-64:+ 232, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_64.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.910s.\n",
      "-- designing region: 1:88946961-88956961_reg_Spp2-65:+ 233, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_65.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.901s.\n",
      "-- designing region: 1:88956961-88966961_reg_Spp2-66:+ 234, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_66.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.925s.\n",
      "-- designing region: 1:88966961-88976961_reg_Spp2-67:+ 235, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_67.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.912s.\n",
      "-- designing region: 1:88976961-88986961_reg_Spp2-68:+ 236, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_68.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.912s.\n",
      "-- designing region: 1:88986961-88996961_reg_Spp2-69:+ 237, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_69.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.089s.\n",
      "in 5.929s.\n",
      "-- designing region: 1:88366961-88376961_reg_Spp2-7:+ 238, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_7.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.088s.\n",
      "in 5.919s.\n",
      "-- designing region: 1:88996961-89006961_reg_Spp2-70:+ 239, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_70.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 8.741s.\n",
      "-- designing region: 1:89006961-89016961_reg_Spp2-71:+ 240, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_71.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.914s.\n",
      "-- designing region: 1:89016961-89026961_reg_Spp2-72:+ 241, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_72.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.915s.\n",
      "-- designing region: 1:89026961-89036961_reg_Spp2-73:+ 242, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_73.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.910s.\n",
      "-- designing region: 1:89036961-89046961_reg_Spp2-74:+ 243, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_74.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.905s.\n",
      "-- designing region: 1:89046961-89056961_reg_Spp2-75:+ 244, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_75.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.916s.\n",
      "-- designing region: 1:89056961-89066961_reg_Spp2-76:+ 245, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_76.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.904s.\n",
      "-- designing region: 1:89066961-89076961_reg_Spp2-77:+ 246, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_77.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.916s.\n",
      "-- designing region: 1:89076961-89086961_reg_Spp2-78:+ 247, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_78.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.917s.\n",
      "-- designing region: 1:89086961-89096961_reg_Spp2-79:+ 248, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_79.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.097s.\n",
      "in 5.934s.\n",
      "-- designing region: 1:88376961-88386961_reg_Spp2-8:+ 249, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_8.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.094s.\n",
      "in 5.910s.\n",
      "-- designing region: 1:89096961-89106961_reg_Spp2-80:+ 250, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_80.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "in 5.917s.\n",
      "-- designing region: 1:89106961-89116961_reg_Spp2-81:+ 251, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_81.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.918s.\n",
      "-- designing region: 1:89116961-89126961_reg_Spp2-82:+ 252, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_82.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.926s.\n",
      "-- designing region: 1:89126961-89136961_reg_Spp2-83:+ 253, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_83.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.914s.\n",
      "-- designing region: 1:89136961-89146961_reg_Spp2-84:+ 254, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_84.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.915s.\n",
      "-- designing region: 1:89146961-89156961_reg_Spp2-85:+ 255, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_85.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.088s.\n",
      "in 5.917s.\n",
      "-- designing region: 1:89156961-89166961_reg_Spp2-86:+ 256, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_86.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.916s.\n",
      "-- designing region: 1:89166961-89176961_reg_Spp2-87:+ 257, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_87.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.907s.\n",
      "-- designing region: 1:89176961-89186961_reg_Spp2-88:+ 258, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_88.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.093s.\n",
      "in 5.926s.\n",
      "-- designing region: 1:89186961-89196961_reg_Spp2-89:+ 259, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_89.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.907s.\n",
      "-- designing region: 1:88386961-88396961_reg_Spp2-9:+ 260, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_9.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.923s.\n",
      "-- designing region: 1:89196961-89206961_reg_Spp2-90:+ 261, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_90.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.087s.\n",
      "in 5.909s.\n",
      "-- designing region: 1:89206961-89216961_reg_Spp2-91:+ 262, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_91.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.914s.\n",
      "-- designing region: 1:89216961-89226961_reg_Spp2-92:+ 263, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\sequences\\Spp2_reg_92.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.086s.\n",
      "in 5.936s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\reports\\merged_probes.pbr\n"
     ]
    }
   ],
   "source": [
    "reload(library_tools)\n",
    "reload(library_tools.design)\n",
    "overwrite_cand_probes =False\n",
    "if len(pb_designer.cand_probes) == 0 or overwrite_cand_probes:\n",
    "    pb_designer.computeOTmaps() # load the tables \n",
    "    pb_designer.compute_pb_report() # design candidate probes"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "rapid-intermediate",
   "metadata": {},
   "source": [
    "### check probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "id": "restricted-scientist",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:0 6:113234124-113244124_reg_Camk1-1:-, 19658 candidate probes\n",
      "--- 10906 probes passed check_dic selection.\n",
      "finish in 0.935s, 149 probes kept.\n",
      "-- check region:1 6:113324124-113334124_reg_Camk1-10:-, 19378 candidate probes\n",
      "--- 10062 probes passed check_dic selection.\n",
      "finish in 0.901s, 140 probes kept.\n",
      "-- check region:2 6:113334124-113344124_reg_Camk1-11:-, 19836 candidate probes\n",
      "--- 9883 probes passed check_dic selection.\n",
      "finish in 0.963s, 162 probes kept.\n",
      "-- check region:3 6:113344124-113354124_reg_Camk1-12:-, 19010 candidate probes\n",
      "--- 6697 probes passed check_dic selection.\n",
      "finish in 0.772s, 109 probes kept.\n",
      "-- check region:4 6:113354124-113364124_reg_Camk1-13:-, 18780 candidate probes\n",
      "--- 7255 probes passed check_dic selection.\n",
      "finish in 0.799s, 110 probes kept.\n",
      "-- check region:5 6:113364124-113374124_reg_Camk1-14:-, 19734 candidate probes\n",
      "--- 10998 probes passed check_dic selection.\n",
      "finish in 0.918s, 158 probes kept.\n",
      "-- check region:6 6:113374124-113384124_reg_Camk1-15:-, 19784 candidate probes\n",
      "--- 11263 probes passed check_dic selection.\n",
      "finish in 0.980s, 154 probes kept.\n",
      "-- check region:7 6:113384124-113394124_reg_Camk1-16:-, 19780 candidate probes\n",
      "--- 11718 probes passed check_dic selection.\n",
      "finish in 0.972s, 159 probes kept.\n",
      "-- check region:8 6:113394124-113404124_reg_Camk1-17:-, 19694 candidate probes\n",
      "--- 10427 probes passed check_dic selection.\n",
      "finish in 0.960s, 151 probes kept.\n",
      "-- check region:9 6:113404124-113414124_reg_Camk1-18:-, 19892 candidate probes\n",
      "--- 11026 probes passed check_dic selection.\n",
      "finish in 0.944s, 154 probes kept.\n",
      "-- check region:10 6:113414124-113424124_reg_Camk1-19:-, 19672 candidate probes\n",
      "--- 9762 probes passed check_dic selection.\n",
      "finish in 0.897s, 148 probes kept.\n",
      "-- check region:11 6:113244124-113254124_reg_Camk1-2:-, 19724 candidate probes\n",
      "--- 10239 probes passed check_dic selection.\n",
      "finish in 0.944s, 140 probes kept.\n",
      "-- check region:12 6:113424124-113434124_reg_Camk1-20:-, 19674 candidate probes\n",
      "--- 9070 probes passed check_dic selection.\n",
      "finish in 0.855s, 143 probes kept.\n",
      "-- check region:13 6:113434124-113444124_reg_Camk1-21:-, 19698 candidate probes\n",
      "--- 8608 probes passed check_dic selection.\n",
      "finish in 0.833s, 136 probes kept.\n",
      "-- check region:14 6:113444124-113454124_reg_Camk1-22:-, 19692 candidate probes\n",
      "--- 8712 probes passed check_dic selection.\n",
      "finish in 0.882s, 141 probes kept.\n",
      "-- check region:15 6:113454124-113464124_reg_Camk1-23:-, 19554 candidate probes\n",
      "--- 7828 probes passed check_dic selection.\n",
      "finish in 0.840s, 136 probes kept.\n",
      "-- check region:16 6:113464124-113474124_reg_Camk1-24:-, 19770 candidate probes\n",
      "--- 11701 probes passed check_dic selection.\n",
      "finish in 1.014s, 170 probes kept.\n",
      "-- check region:17 6:113474124-113484124_reg_Camk1-25:-, 19852 candidate probes\n",
      "--- 12062 probes passed check_dic selection.\n",
      "finish in 0.950s, 171 probes kept.\n",
      "-- check region:18 6:113484124-113494124_reg_Camk1-26:-, 19648 candidate probes\n",
      "--- 8336 probes passed check_dic selection.\n",
      "finish in 0.877s, 141 probes kept.\n",
      "-- check region:19 6:113494124-113504124_reg_Camk1-27:-, 19828 candidate probes\n",
      "--- 8689 probes passed check_dic selection.\n",
      "finish in 0.805s, 147 probes kept.\n",
      "-- check region:20 6:113504124-113514124_reg_Camk1-28:-, 19248 candidate probes\n",
      "--- 7270 probes passed check_dic selection.\n",
      "finish in 0.795s, 113 probes kept.\n",
      "-- check region:21 6:113514124-113524124_reg_Camk1-29:-, 19856 candidate probes\n",
      "--- 11101 probes passed check_dic selection.\n",
      "finish in 0.952s, 153 probes kept.\n",
      "-- check region:22 6:113254124-113264124_reg_Camk1-3:-, 19604 candidate probes\n",
      "--- 13153 probes passed check_dic selection.\n",
      "finish in 1.062s, 162 probes kept.\n",
      "-- check region:23 6:113524124-113534124_reg_Camk1-30:-, 19658 candidate probes\n",
      "--- 7743 probes passed check_dic selection.\n",
      "finish in 0.787s, 138 probes kept.\n",
      "-- check region:24 6:113534124-113544124_reg_Camk1-31:-, 19462 candidate probes\n",
      "--- 5685 probes passed check_dic selection.\n",
      "finish in 0.707s, 103 probes kept.\n",
      "-- check region:25 6:113544124-113554124_reg_Camk1-32:-, 19752 candidate probes\n",
      "--- 8040 probes passed check_dic selection.\n",
      "finish in 0.828s, 132 probes kept.\n",
      "-- check region:26 6:113554124-113564124_reg_Camk1-33:-, 19618 candidate probes\n",
      "--- 10331 probes passed check_dic selection.\n",
      "finish in 0.936s, 135 probes kept.\n",
      "-- check region:27 6:113564124-113574124_reg_Camk1-34:-, 19426 candidate probes\n",
      "--- 9671 probes passed check_dic selection.\n",
      "finish in 0.850s, 126 probes kept.\n",
      "-- check region:28 6:113574124-113584124_reg_Camk1-35:-, 19240 candidate probes\n",
      "--- 9239 probes passed check_dic selection.\n",
      "finish in 0.880s, 131 probes kept.\n",
      "-- check region:29 6:113584124-113594124_reg_Camk1-36:-, 19660 candidate probes\n",
      "--- 9535 probes passed check_dic selection.\n",
      "finish in 0.891s, 133 probes kept.\n",
      "-- check region:30 6:113594124-113604124_reg_Camk1-37:-, 19060 candidate probes\n",
      "--- 8259 probes passed check_dic selection.\n",
      "finish in 0.811s, 131 probes kept.\n",
      "-- check region:31 6:113604124-113614124_reg_Camk1-38:-, 19712 candidate probes\n",
      "--- 9706 probes passed check_dic selection.\n",
      "finish in 0.883s, 142 probes kept.\n",
      "-- check region:32 6:113614124-113624124_reg_Camk1-39:-, 19678 candidate probes\n",
      "--- 9331 probes passed check_dic selection.\n",
      "finish in 0.877s, 142 probes kept.\n",
      "-- check region:33 6:113264124-113274124_reg_Camk1-4:-, 19764 candidate probes\n",
      "--- 13871 probes passed check_dic selection.\n",
      "finish in 1.048s, 168 probes kept.\n",
      "-- check region:34 6:113624124-113634124_reg_Camk1-40:-, 19760 candidate probes\n",
      "--- 7987 probes passed check_dic selection.\n",
      "finish in 0.850s, 136 probes kept.\n",
      "-- check region:35 6:113634124-113644124_reg_Camk1-41:-, 19318 candidate probes\n",
      "--- 8037 probes passed check_dic selection.\n",
      "finish in 0.790s, 121 probes kept.\n",
      "-- check region:36 6:113644124-113654124_reg_Camk1-42:-, 19600 candidate probes\n",
      "--- 11472 probes passed check_dic selection.\n",
      "finish in 0.877s, 139 probes kept.\n",
      "-- check region:37 6:113654124-113664124_reg_Camk1-43:-, 19458 candidate probes\n",
      "--- 10400 probes passed check_dic selection.\n",
      "finish in 0.943s, 132 probes kept.\n",
      "-- check region:38 6:113664124-113674124_reg_Camk1-44:-, 19604 candidate probes\n",
      "--- 9122 probes passed check_dic selection.\n",
      "finish in 0.784s, 133 probes kept.\n",
      "-- check region:39 6:113674124-113684124_reg_Camk1-45:-, 19648 candidate probes\n",
      "--- 10647 probes passed check_dic selection.\n",
      "finish in 0.963s, 144 probes kept.\n",
      "-- check region:40 6:113684124-113694124_reg_Camk1-46:-, 19686 candidate probes\n",
      "--- 8687 probes passed check_dic selection.\n",
      "finish in 0.734s, 139 probes kept.\n",
      "-- check region:41 6:113694124-113704124_reg_Camk1-47:-, 19692 candidate probes\n",
      "--- 9423 probes passed check_dic selection.\n",
      "finish in 0.901s, 141 probes kept.\n",
      "-- check region:42 6:113704124-113714124_reg_Camk1-48:-, 19636 candidate probes\n",
      "--- 10048 probes passed check_dic selection.\n",
      "finish in 0.928s, 145 probes kept.\n",
      "-- check region:43 6:113714124-113724124_reg_Camk1-49:-, 19802 candidate probes\n",
      "--- 11958 probes passed check_dic selection.\n",
      "finish in 1.014s, 165 probes kept.\n",
      "-- check region:44 6:113274124-113284124_reg_Camk1-5:-, 19838 candidate probes\n",
      "--- 12286 probes passed check_dic selection.\n",
      "finish in 0.957s, 162 probes kept.\n",
      "-- check region:45 6:113724124-113734124_reg_Camk1-50:-, 19896 candidate probes\n",
      "--- 10530 probes passed check_dic selection.\n",
      "finish in 0.880s, 149 probes kept.\n",
      "-- check region:46 6:113734124-113744124_reg_Camk1-51:-, 19878 candidate probes\n",
      "--- 11806 probes passed check_dic selection.\n",
      "finish in 1.014s, 164 probes kept.\n",
      "-- check region:47 6:113744124-113754124_reg_Camk1-52:-, 19922 candidate probes\n",
      "--- 10979 probes passed check_dic selection.\n",
      "finish in 0.977s, 168 probes kept.\n",
      "-- check region:48 6:113754124-113764124_reg_Camk1-53:-, 19834 candidate probes\n",
      "--- 14718 probes passed check_dic selection.\n",
      "finish in 1.111s, 173 probes kept.\n",
      "-- check region:49 6:113764124-113774124_reg_Camk1-54:-, 19542 candidate probes\n",
      "--- 13663 probes passed check_dic selection.\n",
      "finish in 0.996s, 168 probes kept.\n",
      "-- check region:50 6:113284124-113294124_reg_Camk1-6:-, 19836 candidate probes\n",
      "--- 9916 probes passed check_dic selection.\n",
      "finish in 0.950s, 158 probes kept.\n",
      "-- check region:51 6:113294124-113304124_reg_Camk1-7:-, 19784 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 9732 probes passed check_dic selection.\n",
      "finish in 0.926s, 139 probes kept.\n",
      "-- check region:52 6:113304124-113314124_reg_Camk1-8:-, 19592 candidate probes\n",
      "--- 11610 probes passed check_dic selection.\n",
      "finish in 0.946s, 164 probes kept.\n",
      "-- check region:53 6:113314124-113324124_reg_Camk1-9:-, 19862 candidate probes\n",
      "--- 10616 probes passed check_dic selection.\n",
      "finish in 0.987s, 172 probes kept.\n",
      "-- check region:54 2:74591924-74601924_reg_Hoxd10-1:+, 19648 candidate probes\n",
      "--- 7447 probes passed check_dic selection.\n",
      "finish in 0.757s, 121 probes kept.\n",
      "-- check region:55 2:74681924-74691924_reg_Hoxd10-10:+, 19826 candidate probes\n",
      "--- 11180 probes passed check_dic selection.\n",
      "finish in 0.960s, 163 probes kept.\n",
      "-- check region:56 2:75581924-75591924_reg_Hoxd10-100:+, 19830 candidate probes\n",
      "--- 9747 probes passed check_dic selection.\n",
      "finish in 0.880s, 135 probes kept.\n",
      "-- check region:57 2:75591924-75601924_reg_Hoxd10-101:+, 19714 candidate probes\n",
      "--- 10743 probes passed check_dic selection.\n",
      "finish in 0.884s, 138 probes kept.\n",
      "-- check region:58 2:75601924-75611924_reg_Hoxd10-102:+, 19834 candidate probes\n",
      "--- 11542 probes passed check_dic selection.\n",
      "finish in 0.941s, 144 probes kept.\n",
      "-- check region:59 2:75611924-75621924_reg_Hoxd10-103:+, 19748 candidate probes\n",
      "--- 8308 probes passed check_dic selection.\n",
      "finish in 0.861s, 118 probes kept.\n",
      "-- check region:60 2:75621924-75631924_reg_Hoxd10-104:+, 19572 candidate probes\n",
      "--- 7978 probes passed check_dic selection.\n",
      "finish in 0.838s, 118 probes kept.\n",
      "-- check region:61 2:75631924-75641924_reg_Hoxd10-105:+, 19040 candidate probes\n",
      "--- 5152 probes passed check_dic selection.\n",
      "finish in 0.659s, 73 probes kept.\n",
      "-- check region:62 2:75641924-75651924_reg_Hoxd10-106:+, 19478 candidate probes\n",
      "--- 6907 probes passed check_dic selection.\n",
      "finish in 0.740s, 108 probes kept.\n",
      "-- check region:63 2:75651924-75661924_reg_Hoxd10-107:+, 19826 candidate probes\n",
      "--- 8922 probes passed check_dic selection.\n",
      "finish in 0.829s, 126 probes kept.\n",
      "-- check region:64 2:75661924-75671924_reg_Hoxd10-108:+, 19678 candidate probes\n",
      "--- 5019 probes passed check_dic selection.\n",
      "finish in 0.668s, 107 probes kept.\n",
      "-- check region:65 2:75671924-75681924_reg_Hoxd10-109:+, 19456 candidate probes\n",
      "--- 8617 probes passed check_dic selection.\n",
      "finish in 0.834s, 132 probes kept.\n",
      "-- check region:66 2:74691924-74701924_reg_Hoxd10-11:+, 19816 candidate probes\n",
      "--- 10869 probes passed check_dic selection.\n",
      "finish in 0.851s, 160 probes kept.\n",
      "-- check region:67 2:75681924-75691924_reg_Hoxd10-110:+, 19770 candidate probes\n",
      "--- 10320 probes passed check_dic selection.\n",
      "finish in 0.910s, 142 probes kept.\n",
      "-- check region:68 2:75691924-75701924_reg_Hoxd10-111:+, 19802 candidate probes\n",
      "--- 12492 probes passed check_dic selection.\n",
      "finish in 1.019s, 154 probes kept.\n",
      "-- check region:69 2:75701924-75711924_reg_Hoxd10-112:+, 19922 candidate probes\n",
      "--- 10445 probes passed check_dic selection.\n",
      "finish in 0.899s, 151 probes kept.\n",
      "-- check region:70 2:75711924-75721924_reg_Hoxd10-113:+, 19712 candidate probes\n",
      "--- 9515 probes passed check_dic selection.\n",
      "finish in 0.850s, 129 probes kept.\n",
      "-- check region:71 2:75721924-75731924_reg_Hoxd10-114:+, 19628 candidate probes\n",
      "--- 8788 probes passed check_dic selection.\n",
      "finish in 0.790s, 129 probes kept.\n",
      "-- check region:72 2:75731924-75741924_reg_Hoxd10-115:+, 19684 candidate probes\n",
      "--- 11139 probes passed check_dic selection.\n",
      "finish in 0.927s, 141 probes kept.\n",
      "-- check region:73 2:75741924-75751924_reg_Hoxd10-116:+, 18428 candidate probes\n",
      "--- 7922 probes passed check_dic selection.\n",
      "finish in 0.778s, 113 probes kept.\n",
      "-- check region:74 2:75751924-75761924_reg_Hoxd10-117:+, 19904 candidate probes\n",
      "--- 11769 probes passed check_dic selection.\n",
      "finish in 0.971s, 150 probes kept.\n",
      "-- check region:75 2:75761924-75771924_reg_Hoxd10-118:+, 19744 candidate probes\n",
      "--- 11693 probes passed check_dic selection.\n",
      "finish in 0.888s, 150 probes kept.\n",
      "-- check region:76 2:74701924-74711924_reg_Hoxd10-12:+, 19764 candidate probes\n",
      "--- 9753 probes passed check_dic selection.\n",
      "finish in 0.867s, 147 probes kept.\n",
      "-- check region:77 2:74711924-74721924_reg_Hoxd10-13:+, 19786 candidate probes\n",
      "--- 11661 probes passed check_dic selection.\n",
      "finish in 0.956s, 157 probes kept.\n",
      "-- check region:78 2:74721924-74731924_reg_Hoxd10-14:+, 19922 candidate probes\n",
      "--- 12017 probes passed check_dic selection.\n",
      "finish in 0.965s, 169 probes kept.\n",
      "-- check region:79 2:74731924-74741924_reg_Hoxd10-15:+, 19854 candidate probes\n",
      "--- 15347 probes passed check_dic selection.\n",
      "finish in 1.040s, 174 probes kept.\n",
      "-- check region:80 2:74741924-74751924_reg_Hoxd10-16:+, 19838 candidate probes\n",
      "--- 9850 probes passed check_dic selection.\n",
      "finish in 0.872s, 167 probes kept.\n",
      "-- check region:81 2:74751924-74761924_reg_Hoxd10-17:+, 19566 candidate probes\n",
      "--- 10328 probes passed check_dic selection.\n",
      "finish in 0.790s, 151 probes kept.\n",
      "-- check region:82 2:74761924-74771924_reg_Hoxd10-18:+, 19644 candidate probes\n",
      "--- 10558 probes passed check_dic selection.\n",
      "finish in 0.920s, 144 probes kept.\n",
      "-- check region:83 2:74771924-74781924_reg_Hoxd10-19:+, 19824 candidate probes\n",
      "--- 7969 probes passed check_dic selection.\n",
      "finish in 0.793s, 128 probes kept.\n",
      "-- check region:84 2:74601924-74611924_reg_Hoxd10-2:+, 19574 candidate probes\n",
      "--- 5284 probes passed check_dic selection.\n",
      "finish in 0.645s, 84 probes kept.\n",
      "-- check region:85 2:74781924-74791924_reg_Hoxd10-20:+, 19616 candidate probes\n",
      "--- 8647 probes passed check_dic selection.\n",
      "finish in 0.824s, 127 probes kept.\n",
      "-- check region:86 2:74791924-74801924_reg_Hoxd10-21:+, 19536 candidate probes\n",
      "--- 7599 probes passed check_dic selection.\n",
      "finish in 0.753s, 116 probes kept.\n",
      "-- check region:87 2:74801924-74811924_reg_Hoxd10-22:+, 19598 candidate probes\n",
      "--- 7816 probes passed check_dic selection.\n",
      "finish in 0.831s, 120 probes kept.\n",
      "-- check region:88 2:74811924-74821924_reg_Hoxd10-23:+, 19900 candidate probes\n",
      "--- 7734 probes passed check_dic selection.\n",
      "finish in 0.804s, 119 probes kept.\n",
      "-- check region:89 2:74821924-74831924_reg_Hoxd10-24:+, 19456 candidate probes\n",
      "--- 8201 probes passed check_dic selection.\n",
      "finish in 0.769s, 123 probes kept.\n",
      "-- check region:90 2:74831924-74841924_reg_Hoxd10-25:+, 19834 candidate probes\n",
      "--- 7340 probes passed check_dic selection.\n",
      "finish in 0.807s, 118 probes kept.\n",
      "-- check region:91 2:74841924-74851924_reg_Hoxd10-26:+, 19868 candidate probes\n",
      "--- 7805 probes passed check_dic selection.\n",
      "finish in 0.816s, 118 probes kept.\n",
      "-- check region:92 2:74851924-74861924_reg_Hoxd10-27:+, 19854 candidate probes\n",
      "--- 5735 probes passed check_dic selection.\n",
      "finish in 0.724s, 105 probes kept.\n",
      "-- check region:93 2:74861924-74871924_reg_Hoxd10-28:+, 19922 candidate probes\n",
      "--- 5642 probes passed check_dic selection.\n",
      "finish in 0.662s, 109 probes kept.\n",
      "-- check region:94 2:74871924-74881924_reg_Hoxd10-29:+, 19784 candidate probes\n",
      "--- 7018 probes passed check_dic selection.\n",
      "finish in 0.716s, 125 probes kept.\n",
      "-- check region:95 2:74611924-74621924_reg_Hoxd10-3:+, 19888 candidate probes\n",
      "--- 8979 probes passed check_dic selection.\n",
      "finish in 0.843s, 126 probes kept.\n",
      "-- check region:96 2:74881924-74891924_reg_Hoxd10-30:+, 19856 candidate probes\n",
      "--- 11103 probes passed check_dic selection.\n",
      "finish in 0.982s, 153 probes kept.\n",
      "-- check region:97 2:74891924-74901924_reg_Hoxd10-31:+, 19836 candidate probes\n",
      "--- 11542 probes passed check_dic selection.\n",
      "finish in 0.996s, 148 probes kept.\n",
      "-- check region:98 2:74901924-74911924_reg_Hoxd10-32:+, 17836 candidate probes\n",
      "--- 8377 probes passed check_dic selection.\n",
      "finish in 0.792s, 119 probes kept.\n",
      "-- check region:99 2:74911924-74921924_reg_Hoxd10-33:+, 19372 candidate probes\n",
      "--- 9320 probes passed check_dic selection.\n",
      "finish in 0.843s, 130 probes kept.\n",
      "-- check region:100 2:74921924-74931924_reg_Hoxd10-34:+, 19916 candidate probes\n",
      "--- 13296 probes passed check_dic selection.\n",
      "finish in 1.075s, 165 probes kept.\n",
      "-- check region:101 2:74931924-74941924_reg_Hoxd10-35:+, 19870 candidate probes\n",
      "--- 11435 probes passed check_dic selection.\n",
      "finish in 0.880s, 149 probes kept.\n",
      "-- check region:102 2:74941924-74951924_reg_Hoxd10-36:+, 19912 candidate probes\n",
      "--- 11034 probes passed check_dic selection.\n",
      "finish in 0.974s, 146 probes kept.\n",
      "-- check region:103 2:74951924-74961924_reg_Hoxd10-37:+, 19524 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 11922 probes passed check_dic selection.\n",
      "finish in 1.006s, 157 probes kept.\n",
      "-- check region:104 2:74961924-74971924_reg_Hoxd10-38:+, 19730 candidate probes\n",
      "--- 13327 probes passed check_dic selection.\n",
      "finish in 0.992s, 158 probes kept.\n",
      "-- check region:105 2:74971924-74981924_reg_Hoxd10-39:+, 19920 candidate probes\n",
      "--- 12016 probes passed check_dic selection.\n",
      "finish in 0.998s, 159 probes kept.\n",
      "-- check region:106 2:74621924-74631924_reg_Hoxd10-4:+, 19814 candidate probes\n",
      "--- 6862 probes passed check_dic selection.\n",
      "finish in 0.774s, 118 probes kept.\n",
      "-- check region:107 2:74981924-74991924_reg_Hoxd10-40:+, 19580 candidate probes\n",
      "--- 9627 probes passed check_dic selection.\n",
      "finish in 0.879s, 124 probes kept.\n",
      "-- check region:108 2:74991924-75001924_reg_Hoxd10-41:+, 19728 candidate probes\n",
      "--- 14206 probes passed check_dic selection.\n",
      "finish in 1.118s, 170 probes kept.\n",
      "-- check region:109 2:75001924-75011924_reg_Hoxd10-42:+, 19862 candidate probes\n",
      "--- 12881 probes passed check_dic selection.\n",
      "finish in 1.001s, 158 probes kept.\n",
      "-- check region:110 2:75011924-75021924_reg_Hoxd10-43:+, 19824 candidate probes\n",
      "--- 12717 probes passed check_dic selection.\n",
      "finish in 1.010s, 155 probes kept.\n",
      "-- check region:111 2:75021924-75031924_reg_Hoxd10-44:+, 19922 candidate probes\n",
      "--- 12267 probes passed check_dic selection.\n",
      "finish in 0.949s, 157 probes kept.\n",
      "-- check region:112 2:75031924-75041924_reg_Hoxd10-45:+, 19772 candidate probes\n",
      "--- 12526 probes passed check_dic selection.\n",
      "finish in 0.992s, 159 probes kept.\n",
      "-- check region:113 2:75041924-75051924_reg_Hoxd10-46:+, 19674 candidate probes\n",
      "--- 10413 probes passed check_dic selection.\n",
      "finish in 0.821s, 119 probes kept.\n",
      "-- check region:114 2:75051924-75061924_reg_Hoxd10-47:+, 19710 candidate probes\n",
      "--- 12274 probes passed check_dic selection.\n",
      "finish in 1.030s, 155 probes kept.\n",
      "-- check region:115 2:75061924-75071924_reg_Hoxd10-48:+, 19900 candidate probes\n",
      "--- 12565 probes passed check_dic selection.\n",
      "finish in 0.949s, 155 probes kept.\n",
      "-- check region:116 2:75071924-75081924_reg_Hoxd10-49:+, 19626 candidate probes\n",
      "--- 12678 probes passed check_dic selection.\n",
      "finish in 1.032s, 154 probes kept.\n",
      "-- check region:117 2:74631924-74641924_reg_Hoxd10-5:+, 19692 candidate probes\n",
      "--- 8327 probes passed check_dic selection.\n",
      "finish in 0.838s, 127 probes kept.\n",
      "-- check region:118 2:75081924-75091924_reg_Hoxd10-50:+, 19922 candidate probes\n",
      "--- 12187 probes passed check_dic selection.\n",
      "finish in 0.989s, 156 probes kept.\n",
      "-- check region:119 2:75091924-75101924_reg_Hoxd10-51:+, 19690 candidate probes\n",
      "--- 11791 probes passed check_dic selection.\n",
      "finish in 0.916s, 153 probes kept.\n",
      "-- check region:120 2:75101924-75111924_reg_Hoxd10-52:+, 19862 candidate probes\n",
      "--- 10214 probes passed check_dic selection.\n",
      "finish in 0.897s, 143 probes kept.\n",
      "-- check region:121 2:75111924-75121924_reg_Hoxd10-53:+, 19790 candidate probes\n",
      "--- 12352 probes passed check_dic selection.\n",
      "finish in 1.036s, 155 probes kept.\n",
      "-- check region:122 2:75121924-75131924_reg_Hoxd10-54:+, 19801 candidate probes\n",
      "--- 9529 probes passed check_dic selection.\n",
      "finish in 0.885s, 134 probes kept.\n",
      "-- check region:123 2:75131924-75141924_reg_Hoxd10-55:+, 19762 candidate probes\n",
      "--- 11802 probes passed check_dic selection.\n",
      "finish in 0.938s, 158 probes kept.\n",
      "-- check region:124 2:75141924-75151924_reg_Hoxd10-56:+, 19892 candidate probes\n",
      "--- 12198 probes passed check_dic selection.\n",
      "finish in 1.020s, 154 probes kept.\n",
      "-- check region:125 2:75151924-75161924_reg_Hoxd10-57:+, 19860 candidate probes\n",
      "--- 12505 probes passed check_dic selection.\n",
      "finish in 1.038s, 159 probes kept.\n",
      "-- check region:126 2:75161924-75171924_reg_Hoxd10-58:+, 19876 candidate probes\n",
      "--- 11750 probes passed check_dic selection.\n",
      "finish in 0.973s, 150 probes kept.\n",
      "-- check region:127 2:75171924-75181924_reg_Hoxd10-59:+, 19864 candidate probes\n",
      "--- 12309 probes passed check_dic selection.\n",
      "finish in 0.976s, 166 probes kept.\n",
      "-- check region:128 2:74641924-74651924_reg_Hoxd10-6:+, 19868 candidate probes\n",
      "--- 12001 probes passed check_dic selection.\n",
      "finish in 0.966s, 157 probes kept.\n",
      "-- check region:129 2:75181924-75191924_reg_Hoxd10-60:+, 19192 candidate probes\n",
      "--- 9301 probes passed check_dic selection.\n",
      "finish in 0.874s, 126 probes kept.\n",
      "-- check region:130 2:75191924-75201924_reg_Hoxd10-61:+, 19922 candidate probes\n",
      "--- 10706 probes passed check_dic selection.\n",
      "finish in 0.881s, 144 probes kept.\n",
      "-- check region:131 2:75201924-75211924_reg_Hoxd10-62:+, 19612 candidate probes\n",
      "--- 10821 probes passed check_dic selection.\n",
      "finish in 0.936s, 134 probes kept.\n",
      "-- check region:132 2:75211924-75221924_reg_Hoxd10-63:+, 19922 candidate probes\n",
      "--- 12191 probes passed check_dic selection.\n",
      "finish in 1.011s, 160 probes kept.\n",
      "-- check region:133 2:75221924-75231924_reg_Hoxd10-64:+, 15136 candidate probes\n",
      "--- 6521 probes passed check_dic selection.\n",
      "finish in 0.640s, 80 probes kept.\n",
      "-- check region:134 2:75231924-75241924_reg_Hoxd10-65:+, 19154 candidate probes\n",
      "--- 9940 probes passed check_dic selection.\n",
      "finish in 0.906s, 133 probes kept.\n",
      "-- check region:135 2:75241924-75251924_reg_Hoxd10-66:+, 19826 candidate probes\n",
      "--- 11474 probes passed check_dic selection.\n",
      "finish in 0.948s, 156 probes kept.\n",
      "-- check region:136 2:75251924-75261924_reg_Hoxd10-67:+, 19808 candidate probes\n",
      "--- 11452 probes passed check_dic selection.\n",
      "finish in 0.969s, 150 probes kept.\n",
      "-- check region:137 2:75261924-75271924_reg_Hoxd10-68:+, 19780 candidate probes\n",
      "--- 13090 probes passed check_dic selection.\n",
      "finish in 1.042s, 159 probes kept.\n",
      "-- check region:138 2:75271924-75281924_reg_Hoxd10-69:+, 19522 candidate probes\n",
      "--- 10585 probes passed check_dic selection.\n",
      "finish in 0.898s, 141 probes kept.\n",
      "-- check region:139 2:74651924-74661924_reg_Hoxd10-7:+, 19858 candidate probes\n",
      "--- 10407 probes passed check_dic selection.\n",
      "finish in 0.914s, 157 probes kept.\n",
      "-- check region:140 2:75281924-75291924_reg_Hoxd10-70:+, 19854 candidate probes\n",
      "--- 12150 probes passed check_dic selection.\n",
      "finish in 0.944s, 164 probes kept.\n",
      "-- check region:141 2:75291924-75301924_reg_Hoxd10-71:+, 19892 candidate probes\n",
      "--- 10434 probes passed check_dic selection.\n",
      "finish in 0.950s, 143 probes kept.\n",
      "-- check region:142 2:75301924-75311924_reg_Hoxd10-72:+, 19882 candidate probes\n",
      "--- 10415 probes passed check_dic selection.\n",
      "finish in 0.940s, 152 probes kept.\n",
      "-- check region:143 2:75311924-75321924_reg_Hoxd10-73:+, 19724 candidate probes\n",
      "--- 10815 probes passed check_dic selection.\n",
      "finish in 0.903s, 164 probes kept.\n",
      "-- check region:144 2:75321924-75331924_reg_Hoxd10-74:+, 19860 candidate probes\n",
      "--- 10384 probes passed check_dic selection.\n",
      "finish in 0.904s, 145 probes kept.\n",
      "-- check region:145 2:75331924-75341924_reg_Hoxd10-75:+, 18804 candidate probes\n",
      "--- 8925 probes passed check_dic selection.\n",
      "finish in 0.791s, 117 probes kept.\n",
      "-- check region:146 2:75341924-75351924_reg_Hoxd10-76:+, 19630 candidate probes\n",
      "--- 9274 probes passed check_dic selection.\n",
      "finish in 0.859s, 119 probes kept.\n",
      "-- check region:147 2:75351924-75361924_reg_Hoxd10-77:+, 19922 candidate probes\n",
      "--- 12155 probes passed check_dic selection.\n",
      "finish in 0.970s, 154 probes kept.\n",
      "-- check region:148 2:75361924-75371924_reg_Hoxd10-78:+, 19790 candidate probes\n",
      "--- 11446 probes passed check_dic selection.\n",
      "finish in 0.953s, 151 probes kept.\n",
      "-- check region:149 2:75371924-75381924_reg_Hoxd10-79:+, 19922 candidate probes\n",
      "--- 12796 probes passed check_dic selection.\n",
      "finish in 1.020s, 162 probes kept.\n",
      "-- check region:150 2:74661924-74671924_reg_Hoxd10-8:+, 19730 candidate probes\n",
      "--- 10778 probes passed check_dic selection.\n",
      "finish in 0.917s, 154 probes kept.\n",
      "-- check region:151 2:75381924-75391924_reg_Hoxd10-80:+, 19856 candidate probes\n",
      "--- 10868 probes passed check_dic selection.\n",
      "finish in 0.890s, 143 probes kept.\n",
      "-- check region:152 2:75391924-75401924_reg_Hoxd10-81:+, 19792 candidate probes\n",
      "--- 10798 probes passed check_dic selection.\n",
      "finish in 0.966s, 152 probes kept.\n",
      "-- check region:153 2:75401924-75411924_reg_Hoxd10-82:+, 19918 candidate probes\n",
      "--- 12978 probes passed check_dic selection.\n",
      "finish in 0.993s, 162 probes kept.\n",
      "-- check region:154 2:75411924-75421924_reg_Hoxd10-83:+, 19786 candidate probes\n",
      "--- 11204 probes passed check_dic selection.\n",
      "finish in 0.940s, 150 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:155 2:75421924-75431924_reg_Hoxd10-84:+, 19672 candidate probes\n",
      "--- 10325 probes passed check_dic selection.\n",
      "finish in 0.930s, 134 probes kept.\n",
      "-- check region:156 2:75431924-75441924_reg_Hoxd10-85:+, 19798 candidate probes\n",
      "--- 11373 probes passed check_dic selection.\n",
      "finish in 0.947s, 148 probes kept.\n",
      "-- check region:157 2:75441924-75451924_reg_Hoxd10-86:+, 19894 candidate probes\n",
      "--- 12790 probes passed check_dic selection.\n",
      "finish in 0.983s, 163 probes kept.\n",
      "-- check region:158 2:75451924-75461924_reg_Hoxd10-87:+, 19880 candidate probes\n",
      "--- 11149 probes passed check_dic selection.\n",
      "finish in 0.918s, 151 probes kept.\n",
      "-- check region:159 2:75461924-75471924_reg_Hoxd10-88:+, 19792 candidate probes\n",
      "--- 11734 probes passed check_dic selection.\n",
      "finish in 0.967s, 152 probes kept.\n",
      "-- check region:160 2:75471924-75481924_reg_Hoxd10-89:+, 19890 candidate probes\n",
      "--- 11228 probes passed check_dic selection.\n",
      "finish in 0.927s, 142 probes kept.\n",
      "-- check region:161 2:74671924-74681924_reg_Hoxd10-9:+, 19914 candidate probes\n",
      "--- 12495 probes passed check_dic selection.\n",
      "finish in 1.008s, 173 probes kept.\n",
      "-- check region:162 2:75481924-75491924_reg_Hoxd10-90:+, 19782 candidate probes\n",
      "--- 12120 probes passed check_dic selection.\n",
      "finish in 1.013s, 151 probes kept.\n",
      "-- check region:163 2:75491924-75501924_reg_Hoxd10-91:+, 19340 candidate probes\n",
      "--- 9194 probes passed check_dic selection.\n",
      "finish in 0.837s, 109 probes kept.\n",
      "-- check region:164 2:75501924-75511924_reg_Hoxd10-92:+, 18690 candidate probes\n",
      "--- 7949 probes passed check_dic selection.\n",
      "finish in 0.777s, 106 probes kept.\n",
      "-- check region:165 2:75511924-75521924_reg_Hoxd10-93:+, 19910 candidate probes\n",
      "--- 13315 probes passed check_dic selection.\n",
      "finish in 1.027s, 166 probes kept.\n",
      "-- check region:166 2:75521924-75531924_reg_Hoxd10-94:+, 19812 candidate probes\n",
      "--- 9947 probes passed check_dic selection.\n",
      "finish in 0.830s, 134 probes kept.\n",
      "-- check region:167 2:75531924-75541924_reg_Hoxd10-95:+, 19748 candidate probes\n",
      "--- 11421 probes passed check_dic selection.\n",
      "finish in 0.937s, 143 probes kept.\n",
      "-- check region:168 2:75541924-75551924_reg_Hoxd10-96:+, 19896 candidate probes\n",
      "--- 9527 probes passed check_dic selection.\n",
      "finish in 0.815s, 132 probes kept.\n",
      "-- check region:169 2:75551924-75561924_reg_Hoxd10-97:+, 19820 candidate probes\n",
      "--- 11641 probes passed check_dic selection.\n",
      "finish in 0.951s, 150 probes kept.\n",
      "-- check region:170 2:75561924-75571924_reg_Hoxd10-98:+, 19848 candidate probes\n",
      "--- 12843 probes passed check_dic selection.\n",
      "finish in 0.963s, 153 probes kept.\n",
      "-- check region:171 2:75571924-75581924_reg_Hoxd10-99:+, 19398 candidate probes\n",
      "--- 9575 probes passed check_dic selection.\n",
      "finish in 0.860s, 134 probes kept.\n",
      "-- check region:172 1:88306961-88316961_reg_Spp2-1:+, 19810 candidate probes\n",
      "--- 12325 probes passed check_dic selection.\n",
      "finish in 0.997s, 160 probes kept.\n",
      "-- check region:173 1:88396961-88406961_reg_Spp2-10:+, 19660 candidate probes\n",
      "--- 12754 probes passed check_dic selection.\n",
      "finish in 0.920s, 149 probes kept.\n",
      "-- check region:174 1:88406961-88416961_reg_Spp2-11:+, 19834 candidate probes\n",
      "--- 12866 probes passed check_dic selection.\n",
      "finish in 1.036s, 163 probes kept.\n",
      "-- check region:175 1:88416961-88426961_reg_Spp2-12:+, 19886 candidate probes\n",
      "--- 12137 probes passed check_dic selection.\n",
      "finish in 1.015s, 160 probes kept.\n",
      "-- check region:176 1:88426961-88436961_reg_Spp2-13:+, 19616 candidate probes\n",
      "--- 10887 probes passed check_dic selection.\n",
      "finish in 0.970s, 139 probes kept.\n",
      "-- check region:177 1:88436961-88446961_reg_Spp2-14:+, 19822 candidate probes\n",
      "--- 11187 probes passed check_dic selection.\n",
      "finish in 0.912s, 159 probes kept.\n",
      "-- check region:178 1:88446961-88456961_reg_Spp2-15:+, 19636 candidate probes\n",
      "--- 10009 probes passed check_dic selection.\n",
      "finish in 0.808s, 140 probes kept.\n",
      "-- check region:179 1:88456961-88466961_reg_Spp2-16:+, 19738 candidate probes\n",
      "--- 11890 probes passed check_dic selection.\n",
      "finish in 0.966s, 148 probes kept.\n",
      "-- check region:180 1:88466961-88476961_reg_Spp2-17:+, 19908 candidate probes\n",
      "--- 13797 probes passed check_dic selection.\n",
      "finish in 1.044s, 159 probes kept.\n",
      "-- check region:181 1:88476961-88486961_reg_Spp2-18:+, 19882 candidate probes\n",
      "--- 11435 probes passed check_dic selection.\n",
      "finish in 0.964s, 147 probes kept.\n",
      "-- check region:182 1:88486961-88496961_reg_Spp2-19:+, 19892 candidate probes\n",
      "--- 12612 probes passed check_dic selection.\n",
      "finish in 1.015s, 154 probes kept.\n",
      "-- check region:183 1:88316961-88326961_reg_Spp2-2:+, 19692 candidate probes\n",
      "--- 13449 probes passed check_dic selection.\n",
      "finish in 1.061s, 168 probes kept.\n",
      "-- check region:184 1:88496961-88506961_reg_Spp2-20:+, 19718 candidate probes\n",
      "--- 12118 probes passed check_dic selection.\n",
      "finish in 0.958s, 164 probes kept.\n",
      "-- check region:185 1:88506961-88516961_reg_Spp2-21:+, 19678 candidate probes\n",
      "--- 12561 probes passed check_dic selection.\n",
      "finish in 0.958s, 157 probes kept.\n",
      "-- check region:186 1:88516961-88526961_reg_Spp2-22:+, 19380 candidate probes\n",
      "--- 10289 probes passed check_dic selection.\n",
      "finish in 0.861s, 127 probes kept.\n",
      "-- check region:187 1:88526961-88536961_reg_Spp2-23:+, 19308 candidate probes\n",
      "--- 9332 probes passed check_dic selection.\n",
      "finish in 0.851s, 135 probes kept.\n",
      "-- check region:188 1:88536961-88546961_reg_Spp2-24:+, 19892 candidate probes\n",
      "--- 11778 probes passed check_dic selection.\n",
      "finish in 1.013s, 152 probes kept.\n",
      "-- check region:189 1:88546961-88556961_reg_Spp2-25:+, 19532 candidate probes\n",
      "--- 8285 probes passed check_dic selection.\n",
      "finish in 0.810s, 120 probes kept.\n",
      "-- check region:190 1:88556961-88566961_reg_Spp2-26:+, 19898 candidate probes\n",
      "--- 12027 probes passed check_dic selection.\n",
      "finish in 0.978s, 156 probes kept.\n",
      "-- check region:191 1:88566961-88576961_reg_Spp2-27:+, 19250 candidate probes\n",
      "--- 6638 probes passed check_dic selection.\n",
      "finish in 0.713s, 99 probes kept.\n",
      "-- check region:192 1:88576961-88586961_reg_Spp2-28:+, 19768 candidate probes\n",
      "--- 11163 probes passed check_dic selection.\n",
      "finish in 0.926s, 145 probes kept.\n",
      "-- check region:193 1:88586961-88596961_reg_Spp2-29:+, 18658 candidate probes\n",
      "--- 11344 probes passed check_dic selection.\n",
      "finish in 0.865s, 136 probes kept.\n",
      "-- check region:194 1:88326961-88336961_reg_Spp2-3:+, 18642 candidate probes\n",
      "--- 10265 probes passed check_dic selection.\n",
      "finish in 0.900s, 145 probes kept.\n",
      "-- check region:195 1:88596961-88606961_reg_Spp2-30:+, 18960 candidate probes\n",
      "--- 7510 probes passed check_dic selection.\n",
      "finish in 0.769s, 93 probes kept.\n",
      "-- check region:196 1:88606961-88616961_reg_Spp2-31:+, 18980 candidate probes\n",
      "--- 8864 probes passed check_dic selection.\n",
      "finish in 0.835s, 115 probes kept.\n",
      "-- check region:197 1:88616961-88626961_reg_Spp2-32:+, 19862 candidate probes\n",
      "--- 12591 probes passed check_dic selection.\n",
      "finish in 1.014s, 158 probes kept.\n",
      "-- check region:198 1:88626961-88636961_reg_Spp2-33:+, 19908 candidate probes\n",
      "--- 11548 probes passed check_dic selection.\n",
      "finish in 0.952s, 139 probes kept.\n",
      "-- check region:199 1:88636961-88646961_reg_Spp2-34:+, 19738 candidate probes\n",
      "--- 10421 probes passed check_dic selection.\n",
      "finish in 0.908s, 144 probes kept.\n",
      "-- check region:200 1:88646961-88656961_reg_Spp2-35:+, 19888 candidate probes\n",
      "--- 12972 probes passed check_dic selection.\n",
      "finish in 0.926s, 155 probes kept.\n",
      "-- check region:201 1:88656961-88666961_reg_Spp2-36:+, 19886 candidate probes\n",
      "--- 11634 probes passed check_dic selection.\n",
      "finish in 0.858s, 152 probes kept.\n",
      "-- check region:202 1:88666961-88676961_reg_Spp2-37:+, 19756 candidate probes\n",
      "--- 9291 probes passed check_dic selection.\n",
      "finish in 0.834s, 134 probes kept.\n",
      "-- check region:203 1:88676961-88686961_reg_Spp2-38:+, 19812 candidate probes\n",
      "--- 14708 probes passed check_dic selection.\n",
      "finish in 1.142s, 172 probes kept.\n",
      "-- check region:204 1:88686961-88696961_reg_Spp2-39:+, 19720 candidate probes\n",
      "--- 13449 probes passed check_dic selection.\n",
      "finish in 1.070s, 158 probes kept.\n",
      "-- check region:205 1:88336961-88346961_reg_Spp2-4:+, 19758 candidate probes\n",
      "--- 10958 probes passed check_dic selection.\n",
      "finish in 0.903s, 147 probes kept.\n",
      "-- check region:206 1:88696961-88706961_reg_Spp2-40:+, 19858 candidate probes\n",
      "--- 10486 probes passed check_dic selection.\n",
      "finish in 0.910s, 153 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:207 1:88706961-88716961_reg_Spp2-41:+, 19558 candidate probes\n",
      "--- 12305 probes passed check_dic selection.\n",
      "finish in 0.967s, 151 probes kept.\n",
      "-- check region:208 1:88716961-88726961_reg_Spp2-42:+, 19706 candidate probes\n",
      "--- 10211 probes passed check_dic selection.\n",
      "finish in 0.941s, 144 probes kept.\n",
      "-- check region:209 1:88726961-88736961_reg_Spp2-43:+, 19612 candidate probes\n",
      "--- 10173 probes passed check_dic selection.\n",
      "finish in 0.899s, 131 probes kept.\n",
      "-- check region:210 1:88736961-88746961_reg_Spp2-44:+, 19874 candidate probes\n",
      "--- 13441 probes passed check_dic selection.\n",
      "finish in 1.025s, 167 probes kept.\n",
      "-- check region:211 1:88746961-88756961_reg_Spp2-45:+, 19884 candidate probes\n",
      "--- 11355 probes passed check_dic selection.\n",
      "finish in 0.995s, 150 probes kept.\n",
      "-- check region:212 1:88756961-88766961_reg_Spp2-46:+, 19880 candidate probes\n",
      "--- 11834 probes passed check_dic selection.\n",
      "finish in 0.997s, 149 probes kept.\n",
      "-- check region:213 1:88766961-88776961_reg_Spp2-47:+, 19752 candidate probes\n",
      "--- 11291 probes passed check_dic selection.\n",
      "finish in 0.966s, 142 probes kept.\n",
      "-- check region:214 1:88776961-88786961_reg_Spp2-48:+, 19734 candidate probes\n",
      "--- 12193 probes passed check_dic selection.\n",
      "finish in 0.931s, 143 probes kept.\n",
      "-- check region:215 1:88786961-88796961_reg_Spp2-49:+, 19898 candidate probes\n",
      "--- 11106 probes passed check_dic selection.\n",
      "finish in 0.938s, 139 probes kept.\n",
      "-- check region:216 1:88346961-88356961_reg_Spp2-5:+, 19922 candidate probes\n",
      "--- 12108 probes passed check_dic selection.\n",
      "finish in 1.001s, 163 probes kept.\n",
      "-- check region:217 1:88796961-88806961_reg_Spp2-50:+, 19774 candidate probes\n",
      "--- 11227 probes passed check_dic selection.\n",
      "finish in 0.906s, 149 probes kept.\n",
      "-- check region:218 1:88806961-88816961_reg_Spp2-51:+, 19918 candidate probes\n",
      "--- 11847 probes passed check_dic selection.\n",
      "finish in 0.981s, 150 probes kept.\n",
      "-- check region:219 1:88816961-88826961_reg_Spp2-52:+, 19662 candidate probes\n",
      "--- 10732 probes passed check_dic selection.\n",
      "finish in 0.963s, 144 probes kept.\n",
      "-- check region:220 1:88826961-88836961_reg_Spp2-53:+, 18982 candidate probes\n",
      "--- 4399 probes passed check_dic selection.\n",
      "finish in 0.673s, 53 probes kept.\n",
      "-- check region:221 1:88836961-88846961_reg_Spp2-54:+, 19874 candidate probes\n",
      "--- 13161 probes passed check_dic selection.\n",
      "finish in 1.044s, 160 probes kept.\n",
      "-- check region:222 1:88846961-88856961_reg_Spp2-55:+, 19692 candidate probes\n",
      "--- 6691 probes passed check_dic selection.\n",
      "finish in 0.783s, 87 probes kept.\n",
      "-- check region:223 1:88856961-88866961_reg_Spp2-56:+, 19664 candidate probes\n",
      "--- 10024 probes passed check_dic selection.\n",
      "finish in 0.903s, 126 probes kept.\n",
      "-- check region:224 1:88866961-88876961_reg_Spp2-57:+, 19702 candidate probes\n",
      "--- 12093 probes passed check_dic selection.\n",
      "finish in 0.975s, 149 probes kept.\n",
      "-- check region:225 1:88876961-88886961_reg_Spp2-58:+, 19860 candidate probes\n",
      "--- 11424 probes passed check_dic selection.\n",
      "finish in 0.874s, 146 probes kept.\n",
      "-- check region:226 1:88886961-88896961_reg_Spp2-59:+, 19854 candidate probes\n",
      "--- 8539 probes passed check_dic selection.\n",
      "finish in 0.830s, 120 probes kept.\n",
      "-- check region:227 1:88356961-88366961_reg_Spp2-6:+, 19890 candidate probes\n",
      "--- 12796 probes passed check_dic selection.\n",
      "finish in 0.966s, 154 probes kept.\n",
      "-- check region:228 1:88896961-88906961_reg_Spp2-60:+, 19784 candidate probes\n",
      "--- 10312 probes passed check_dic selection.\n",
      "finish in 0.945s, 139 probes kept.\n",
      "-- check region:229 1:88906961-88916961_reg_Spp2-61:+, 19872 candidate probes\n",
      "--- 11559 probes passed check_dic selection.\n",
      "finish in 1.005s, 147 probes kept.\n",
      "-- check region:230 1:88916961-88926961_reg_Spp2-62:+, 19838 candidate probes\n",
      "--- 11997 probes passed check_dic selection.\n",
      "finish in 0.868s, 147 probes kept.\n",
      "-- check region:231 1:88926961-88936961_reg_Spp2-63:+, 19730 candidate probes\n",
      "--- 11031 probes passed check_dic selection.\n",
      "finish in 0.950s, 144 probes kept.\n",
      "-- check region:232 1:88936961-88946961_reg_Spp2-64:+, 19676 candidate probes\n",
      "--- 11633 probes passed check_dic selection.\n",
      "finish in 0.944s, 156 probes kept.\n",
      "-- check region:233 1:88946961-88956961_reg_Spp2-65:+, 19901 candidate probes\n",
      "--- 9615 probes passed check_dic selection.\n",
      "finish in 0.883s, 131 probes kept.\n",
      "-- check region:234 1:88956961-88966961_reg_Spp2-66:+, 19920 candidate probes\n",
      "--- 12317 probes passed check_dic selection.\n",
      "finish in 0.967s, 154 probes kept.\n",
      "-- check region:235 1:88966961-88976961_reg_Spp2-67:+, 19670 candidate probes\n",
      "--- 9322 probes passed check_dic selection.\n",
      "finish in 0.784s, 125 probes kept.\n",
      "-- check region:236 1:88976961-88986961_reg_Spp2-68:+, 19850 candidate probes\n",
      "--- 10397 probes passed check_dic selection.\n",
      "finish in 0.906s, 140 probes kept.\n",
      "-- check region:237 1:88986961-88996961_reg_Spp2-69:+, 19812 candidate probes\n",
      "--- 11932 probes passed check_dic selection.\n",
      "finish in 0.978s, 154 probes kept.\n",
      "-- check region:238 1:88366961-88376961_reg_Spp2-7:+, 19844 candidate probes\n",
      "--- 12664 probes passed check_dic selection.\n",
      "finish in 0.940s, 156 probes kept.\n",
      "-- check region:239 1:88996961-89006961_reg_Spp2-70:+, 19908 candidate probes\n",
      "--- 10448 probes passed check_dic selection.\n",
      "finish in 0.959s, 133 probes kept.\n",
      "-- check region:240 1:89006961-89016961_reg_Spp2-71:+, 19828 candidate probes\n",
      "--- 11606 probes passed check_dic selection.\n",
      "finish in 0.959s, 156 probes kept.\n",
      "-- check region:241 1:89016961-89026961_reg_Spp2-72:+, 19866 candidate probes\n",
      "--- 10793 probes passed check_dic selection.\n",
      "finish in 0.912s, 142 probes kept.\n",
      "-- check region:242 1:89026961-89036961_reg_Spp2-73:+, 19922 candidate probes\n",
      "--- 10206 probes passed check_dic selection.\n",
      "finish in 0.938s, 143 probes kept.\n",
      "-- check region:243 1:89036961-89046961_reg_Spp2-74:+, 19882 candidate probes\n",
      "--- 11829 probes passed check_dic selection.\n",
      "finish in 0.979s, 147 probes kept.\n",
      "-- check region:244 1:89046961-89056961_reg_Spp2-75:+, 19914 candidate probes\n",
      "--- 11296 probes passed check_dic selection.\n",
      "finish in 0.995s, 151 probes kept.\n",
      "-- check region:245 1:89056961-89066961_reg_Spp2-76:+, 19791 candidate probes\n",
      "--- 8421 probes passed check_dic selection.\n",
      "finish in 0.816s, 132 probes kept.\n",
      "-- check region:246 1:89066961-89076961_reg_Spp2-77:+, 19820 candidate probes\n",
      "--- 13107 probes passed check_dic selection.\n",
      "finish in 1.051s, 158 probes kept.\n",
      "-- check region:247 1:89076961-89086961_reg_Spp2-78:+, 19922 candidate probes\n",
      "--- 15686 probes passed check_dic selection.\n",
      "finish in 1.038s, 171 probes kept.\n",
      "-- check region:248 1:89086961-89096961_reg_Spp2-79:+, 19674 candidate probes\n",
      "--- 14294 probes passed check_dic selection.\n",
      "finish in 0.994s, 158 probes kept.\n",
      "-- check region:249 1:88376961-88386961_reg_Spp2-8:+, 19828 candidate probes\n",
      "--- 10744 probes passed check_dic selection.\n",
      "finish in 0.882s, 148 probes kept.\n",
      "-- check region:250 1:89096961-89106961_reg_Spp2-80:+, 19838 candidate probes\n",
      "--- 7548 probes passed check_dic selection.\n",
      "finish in 0.779s, 83 probes kept.\n",
      "-- check region:251 1:89106961-89116961_reg_Spp2-81:+, 19816 candidate probes\n",
      "--- 14897 probes passed check_dic selection.\n",
      "finish in 1.088s, 170 probes kept.\n",
      "-- check region:252 1:89116961-89126961_reg_Spp2-82:+, 19922 candidate probes\n",
      "--- 15065 probes passed check_dic selection.\n",
      "finish in 1.132s, 177 probes kept.\n",
      "-- check region:253 1:89126961-89136961_reg_Spp2-83:+, 19906 candidate probes\n",
      "--- 14130 probes passed check_dic selection.\n",
      "finish in 1.075s, 164 probes kept.\n",
      "-- check region:254 1:89136961-89146961_reg_Spp2-84:+, 19922 candidate probes\n",
      "--- 12978 probes passed check_dic selection.\n",
      "finish in 0.990s, 170 probes kept.\n",
      "-- check region:255 1:89146961-89156961_reg_Spp2-85:+, 19920 candidate probes\n",
      "--- 13007 probes passed check_dic selection.\n",
      "finish in 1.055s, 163 probes kept.\n",
      "-- check region:256 1:89156961-89166961_reg_Spp2-86:+, 19748 candidate probes\n",
      "--- 10822 probes passed check_dic selection.\n",
      "finish in 0.971s, 141 probes kept.\n",
      "-- check region:257 1:89166961-89176961_reg_Spp2-87:+, 19872 candidate probes\n",
      "--- 10180 probes passed check_dic selection.\n",
      "finish in 0.869s, 128 probes kept.\n",
      "-- check region:258 1:89176961-89186961_reg_Spp2-88:+, 19894 candidate probes\n",
      "--- 11351 probes passed check_dic selection.\n",
      "finish in 0.955s, 144 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:259 1:89186961-89196961_reg_Spp2-89:+, 19922 candidate probes\n",
      "--- 11881 probes passed check_dic selection.\n",
      "finish in 0.991s, 152 probes kept.\n",
      "-- check region:260 1:88386961-88396961_reg_Spp2-9:+, 19762 candidate probes\n",
      "--- 11161 probes passed check_dic selection.\n",
      "finish in 0.932s, 156 probes kept.\n",
      "-- check region:261 1:89196961-89206961_reg_Spp2-90:+, 19200 candidate probes\n",
      "--- 6796 probes passed check_dic selection.\n",
      "finish in 0.710s, 88 probes kept.\n",
      "-- check region:262 1:89206961-89216961_reg_Spp2-91:+, 19384 candidate probes\n",
      "--- 9339 probes passed check_dic selection.\n",
      "finish in 0.896s, 130 probes kept.\n",
      "-- check region:263 1:89216961-89226961_reg_Spp2-92:+, 19594 candidate probes\n",
      "--- 3648 probes passed check_dic selection.\n",
      "finish in 0.653s, 50 probes kept.\n"
     ]
    }
   ],
   "source": [
    "pbs, pb_scores = pb_designer.check_probes()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "occasional-charter",
   "metadata": {},
   "source": [
    "### save probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "id": "helpful-manual",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\reports\\merged_probes.pbr\n",
      "-- number of probes kept: 37669\n"
     ]
    }
   ],
   "source": [
    "overwrite_savefile = True \n",
    "if not os.path.exists(pb_designer.save_file) or overwrite_savefile:\n",
    "    #pb_designer.plots()\n",
    "    #pb_designer.save_csv()\n",
    "    pb_designer.save_to_file()\n",
    "print(f\"-- number of probes kept: {len(pb_designer.kept_probes)}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "id": "completed-winter",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Time to release OTmaps: 9.256s. \n"
     ]
    }
   ],
   "source": [
    "# after finish designing, remove all loaded OT maps to freeup memory usage\n",
    "pb_designer.release_OTmaps()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "integrated-spencer",
   "metadata": {},
   "source": [
    "## 2.3 summarize into a pb_dict\n",
    "\n",
    "start over from here is possible"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "id": "running-undergraduate",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\reports\\merged_probes.pbr\n"
     ]
    }
   ],
   "source": [
    "# filename to save probe reports\n",
    "save_file = os.path.join(report_folder, f'merged_probes.pbr')\n",
    "print(save_file)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "id": "insured-situation",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Loading from savefile: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\reports\\merged_probes.pbr.\n",
      "37669\n"
     ]
    }
   ],
   "source": [
    "reload(library_tools)\n",
    "reload(library_tools.design)\n",
    "\n",
    "verbose=True\n",
    "cand_prob_nums = []\n",
    "pb_dict = {}\n",
    "# Parse segemented list\n",
    "_pbde = library_tools.design.pb_reports_class()\n",
    "_pbde.load_from_file(save_file)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "id": "amber-daughter",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- appending gene:Camk1-1 with 149 probes\n",
      "-- appending gene:Camk1-10 with 140 probes\n",
      "-- appending gene:Camk1-11 with 162 probes\n",
      "-- appending gene:Camk1-12 with 109 probes\n",
      "-- appending gene:Camk1-13 with 110 probes\n",
      "-- appending gene:Camk1-14 with 158 probes\n",
      "-- appending gene:Camk1-15 with 154 probes\n",
      "-- appending gene:Camk1-16 with 159 probes\n",
      "-- appending gene:Camk1-17 with 151 probes\n",
      "-- appending gene:Camk1-18 with 154 probes\n",
      "-- appending gene:Camk1-19 with 148 probes\n",
      "-- appending gene:Camk1-2 with 140 probes\n",
      "-- appending gene:Camk1-20 with 143 probes\n",
      "-- appending gene:Camk1-21 with 136 probes\n",
      "-- appending gene:Camk1-22 with 141 probes\n",
      "-- appending gene:Camk1-23 with 136 probes\n",
      "-- appending gene:Camk1-24 with 170 probes\n",
      "-- appending gene:Camk1-25 with 171 probes\n",
      "-- appending gene:Camk1-26 with 141 probes\n",
      "-- appending gene:Camk1-27 with 147 probes\n",
      "-- appending gene:Camk1-28 with 113 probes\n",
      "-- appending gene:Camk1-29 with 153 probes\n",
      "-- appending gene:Camk1-3 with 162 probes\n",
      "-- appending gene:Camk1-30 with 138 probes\n",
      "-- appending gene:Camk1-31 with 103 probes\n",
      "-- appending gene:Camk1-32 with 132 probes\n",
      "-- appending gene:Camk1-33 with 135 probes\n",
      "-- appending gene:Camk1-34 with 126 probes\n",
      "-- appending gene:Camk1-35 with 131 probes\n",
      "-- appending gene:Camk1-36 with 133 probes\n",
      "-- appending gene:Camk1-37 with 131 probes\n",
      "-- appending gene:Camk1-38 with 142 probes\n",
      "-- appending gene:Camk1-39 with 142 probes\n",
      "-- appending gene:Camk1-4 with 168 probes\n",
      "-- appending gene:Camk1-40 with 136 probes\n",
      "-- appending gene:Camk1-41 with 121 probes\n",
      "-- appending gene:Camk1-42 with 139 probes\n",
      "-- appending gene:Camk1-43 with 132 probes\n",
      "-- appending gene:Camk1-44 with 133 probes\n",
      "-- appending gene:Camk1-45 with 144 probes\n",
      "-- appending gene:Camk1-46 with 139 probes\n",
      "-- appending gene:Camk1-47 with 141 probes\n",
      "-- appending gene:Camk1-48 with 145 probes\n",
      "-- appending gene:Camk1-49 with 165 probes\n",
      "-- appending gene:Camk1-5 with 162 probes\n",
      "-- appending gene:Camk1-50 with 149 probes\n",
      "-- appending gene:Camk1-51 with 164 probes\n",
      "-- appending gene:Camk1-52 with 168 probes\n",
      "-- appending gene:Camk1-53 with 173 probes\n",
      "-- appending gene:Camk1-54 with 168 probes\n",
      "-- appending gene:Camk1-6 with 158 probes\n",
      "-- appending gene:Camk1-7 with 139 probes\n",
      "-- appending gene:Camk1-8 with 164 probes\n",
      "-- appending gene:Camk1-9 with 172 probes\n",
      "-- appending gene:Hoxd10-1 with 121 probes\n",
      "-- appending gene:Hoxd10-10 with 163 probes\n",
      "-- appending gene:Hoxd10-100 with 135 probes\n",
      "-- appending gene:Hoxd10-101 with 138 probes\n",
      "-- appending gene:Hoxd10-102 with 144 probes\n",
      "-- appending gene:Hoxd10-103 with 118 probes\n",
      "-- appending gene:Hoxd10-104 with 118 probes\n",
      "-- appending gene:Hoxd10-105 with 73 probes\n",
      "-- appending gene:Hoxd10-106 with 108 probes\n",
      "-- appending gene:Hoxd10-107 with 126 probes\n",
      "-- appending gene:Hoxd10-108 with 107 probes\n",
      "-- appending gene:Hoxd10-109 with 132 probes\n",
      "-- appending gene:Hoxd10-11 with 160 probes\n",
      "-- appending gene:Hoxd10-110 with 142 probes\n",
      "-- appending gene:Hoxd10-111 with 154 probes\n",
      "-- appending gene:Hoxd10-112 with 151 probes\n",
      "-- appending gene:Hoxd10-113 with 129 probes\n",
      "-- appending gene:Hoxd10-114 with 129 probes\n",
      "-- appending gene:Hoxd10-115 with 141 probes\n",
      "-- appending gene:Hoxd10-116 with 113 probes\n",
      "-- appending gene:Hoxd10-117 with 150 probes\n",
      "-- appending gene:Hoxd10-118 with 150 probes\n",
      "-- appending gene:Hoxd10-12 with 147 probes\n",
      "-- appending gene:Hoxd10-13 with 157 probes\n",
      "-- appending gene:Hoxd10-14 with 169 probes\n",
      "-- appending gene:Hoxd10-15 with 174 probes\n",
      "-- appending gene:Hoxd10-16 with 167 probes\n",
      "-- appending gene:Hoxd10-17 with 151 probes\n",
      "-- appending gene:Hoxd10-18 with 144 probes\n",
      "-- appending gene:Hoxd10-19 with 128 probes\n",
      "-- appending gene:Hoxd10-2 with 84 probes\n",
      "-- appending gene:Hoxd10-20 with 127 probes\n",
      "-- appending gene:Hoxd10-21 with 116 probes\n",
      "-- appending gene:Hoxd10-22 with 120 probes\n",
      "-- appending gene:Hoxd10-23 with 119 probes\n",
      "-- appending gene:Hoxd10-24 with 123 probes\n",
      "-- appending gene:Hoxd10-25 with 118 probes\n",
      "-- appending gene:Hoxd10-26 with 118 probes\n",
      "-- appending gene:Hoxd10-27 with 105 probes\n",
      "-- appending gene:Hoxd10-28 with 109 probes\n",
      "-- appending gene:Hoxd10-29 with 125 probes\n",
      "-- appending gene:Hoxd10-3 with 126 probes\n",
      "-- appending gene:Hoxd10-30 with 153 probes\n",
      "-- appending gene:Hoxd10-31 with 148 probes\n",
      "-- appending gene:Hoxd10-32 with 119 probes\n",
      "-- appending gene:Hoxd10-33 with 130 probes\n",
      "-- appending gene:Hoxd10-34 with 165 probes\n",
      "-- appending gene:Hoxd10-35 with 149 probes\n",
      "-- appending gene:Hoxd10-36 with 146 probes\n",
      "-- appending gene:Hoxd10-37 with 157 probes\n",
      "-- appending gene:Hoxd10-38 with 158 probes\n",
      "-- appending gene:Hoxd10-39 with 159 probes\n",
      "-- appending gene:Hoxd10-4 with 118 probes\n",
      "-- appending gene:Hoxd10-40 with 124 probes\n",
      "-- appending gene:Hoxd10-41 with 170 probes\n",
      "-- appending gene:Hoxd10-42 with 158 probes\n",
      "-- appending gene:Hoxd10-43 with 155 probes\n",
      "-- appending gene:Hoxd10-44 with 157 probes\n",
      "-- appending gene:Hoxd10-45 with 159 probes\n",
      "-- appending gene:Hoxd10-46 with 119 probes\n",
      "-- appending gene:Hoxd10-47 with 155 probes\n",
      "-- appending gene:Hoxd10-48 with 155 probes\n",
      "-- appending gene:Hoxd10-49 with 154 probes\n",
      "-- appending gene:Hoxd10-5 with 127 probes\n",
      "-- appending gene:Hoxd10-50 with 156 probes\n",
      "-- appending gene:Hoxd10-51 with 153 probes\n",
      "-- appending gene:Hoxd10-52 with 143 probes\n",
      "-- appending gene:Hoxd10-53 with 155 probes\n",
      "-- appending gene:Hoxd10-54 with 134 probes\n",
      "-- appending gene:Hoxd10-55 with 158 probes\n",
      "-- appending gene:Hoxd10-56 with 154 probes\n",
      "-- appending gene:Hoxd10-57 with 159 probes\n",
      "-- appending gene:Hoxd10-58 with 150 probes\n",
      "-- appending gene:Hoxd10-59 with 166 probes\n",
      "-- appending gene:Hoxd10-6 with 157 probes\n",
      "-- appending gene:Hoxd10-60 with 126 probes\n",
      "-- appending gene:Hoxd10-61 with 144 probes\n",
      "-- appending gene:Hoxd10-62 with 134 probes\n",
      "-- appending gene:Hoxd10-63 with 160 probes\n",
      "-- appending gene:Hoxd10-64 with 80 probes\n",
      "-- appending gene:Hoxd10-65 with 133 probes\n",
      "-- appending gene:Hoxd10-66 with 156 probes\n",
      "-- appending gene:Hoxd10-67 with 150 probes\n",
      "-- appending gene:Hoxd10-68 with 159 probes\n",
      "-- appending gene:Hoxd10-69 with 141 probes\n",
      "-- appending gene:Hoxd10-7 with 157 probes\n",
      "-- appending gene:Hoxd10-70 with 164 probes\n",
      "-- appending gene:Hoxd10-71 with 143 probes\n",
      "-- appending gene:Hoxd10-72 with 152 probes\n",
      "-- appending gene:Hoxd10-73 with 164 probes\n",
      "-- appending gene:Hoxd10-74 with 145 probes\n",
      "-- appending gene:Hoxd10-75 with 117 probes\n",
      "-- appending gene:Hoxd10-76 with 119 probes\n",
      "-- appending gene:Hoxd10-77 with 154 probes\n",
      "-- appending gene:Hoxd10-78 with 151 probes\n",
      "-- appending gene:Hoxd10-79 with 162 probes\n",
      "-- appending gene:Hoxd10-8 with 154 probes\n",
      "-- appending gene:Hoxd10-80 with 143 probes\n",
      "-- appending gene:Hoxd10-81 with 152 probes\n",
      "-- appending gene:Hoxd10-82 with 162 probes\n",
      "-- appending gene:Hoxd10-83 with 150 probes\n",
      "-- appending gene:Hoxd10-84 with 134 probes\n",
      "-- appending gene:Hoxd10-85 with 148 probes\n",
      "-- appending gene:Hoxd10-86 with 163 probes\n",
      "-- appending gene:Hoxd10-87 with 151 probes\n",
      "-- appending gene:Hoxd10-88 with 152 probes\n",
      "-- appending gene:Hoxd10-89 with 142 probes\n",
      "-- appending gene:Hoxd10-9 with 173 probes\n",
      "-- appending gene:Hoxd10-90 with 151 probes\n",
      "-- appending gene:Hoxd10-91 with 109 probes\n",
      "-- appending gene:Hoxd10-92 with 106 probes\n",
      "-- appending gene:Hoxd10-93 with 166 probes\n",
      "-- appending gene:Hoxd10-94 with 134 probes\n",
      "-- appending gene:Hoxd10-95 with 143 probes\n",
      "-- appending gene:Hoxd10-96 with 132 probes\n",
      "-- appending gene:Hoxd10-97 with 150 probes\n",
      "-- appending gene:Hoxd10-98 with 153 probes\n",
      "-- appending gene:Hoxd10-99 with 134 probes\n",
      "-- appending gene:Spp2-1 with 160 probes\n",
      "-- appending gene:Spp2-10 with 149 probes\n",
      "-- appending gene:Spp2-11 with 163 probes\n",
      "-- appending gene:Spp2-12 with 160 probes\n",
      "-- appending gene:Spp2-13 with 139 probes\n",
      "-- appending gene:Spp2-14 with 159 probes\n",
      "-- appending gene:Spp2-15 with 140 probes\n",
      "-- appending gene:Spp2-16 with 148 probes\n",
      "-- appending gene:Spp2-17 with 159 probes\n",
      "-- appending gene:Spp2-18 with 147 probes\n",
      "-- appending gene:Spp2-19 with 154 probes\n",
      "-- appending gene:Spp2-2 with 168 probes\n",
      "-- appending gene:Spp2-20 with 164 probes\n",
      "-- appending gene:Spp2-21 with 157 probes\n",
      "-- appending gene:Spp2-22 with 127 probes\n",
      "-- appending gene:Spp2-23 with 135 probes\n",
      "-- appending gene:Spp2-24 with 152 probes\n",
      "-- appending gene:Spp2-25 with 120 probes\n",
      "-- appending gene:Spp2-26 with 156 probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- appending gene:Spp2-27 with 99 probes\n",
      "-- appending gene:Spp2-28 with 145 probes\n",
      "-- appending gene:Spp2-29 with 136 probes\n",
      "-- appending gene:Spp2-3 with 145 probes\n",
      "-- appending gene:Spp2-30 with 93 probes\n",
      "-- appending gene:Spp2-31 with 115 probes\n",
      "-- appending gene:Spp2-32 with 158 probes\n",
      "-- appending gene:Spp2-33 with 139 probes\n",
      "-- appending gene:Spp2-34 with 144 probes\n",
      "-- appending gene:Spp2-35 with 155 probes\n",
      "-- appending gene:Spp2-36 with 152 probes\n",
      "-- appending gene:Spp2-37 with 134 probes\n",
      "-- appending gene:Spp2-38 with 172 probes\n",
      "-- appending gene:Spp2-39 with 158 probes\n",
      "-- appending gene:Spp2-4 with 147 probes\n",
      "-- appending gene:Spp2-40 with 153 probes\n",
      "-- appending gene:Spp2-41 with 151 probes\n",
      "-- appending gene:Spp2-42 with 144 probes\n",
      "-- appending gene:Spp2-43 with 131 probes\n",
      "-- appending gene:Spp2-44 with 167 probes\n",
      "-- appending gene:Spp2-45 with 150 probes\n",
      "-- appending gene:Spp2-46 with 149 probes\n",
      "-- appending gene:Spp2-47 with 142 probes\n",
      "-- appending gene:Spp2-48 with 143 probes\n",
      "-- appending gene:Spp2-49 with 139 probes\n",
      "-- appending gene:Spp2-5 with 163 probes\n",
      "-- appending gene:Spp2-50 with 149 probes\n",
      "-- appending gene:Spp2-51 with 150 probes\n",
      "-- appending gene:Spp2-52 with 144 probes\n",
      "-- appending gene:Spp2-53 with 53 probes\n",
      "-- appending gene:Spp2-54 with 160 probes\n",
      "-- appending gene:Spp2-55 with 87 probes\n",
      "-- appending gene:Spp2-56 with 126 probes\n",
      "-- appending gene:Spp2-57 with 149 probes\n",
      "-- appending gene:Spp2-58 with 146 probes\n",
      "-- appending gene:Spp2-59 with 120 probes\n",
      "-- appending gene:Spp2-6 with 154 probes\n",
      "-- appending gene:Spp2-60 with 139 probes\n",
      "-- appending gene:Spp2-61 with 147 probes\n",
      "-- appending gene:Spp2-62 with 147 probes\n",
      "-- appending gene:Spp2-63 with 144 probes\n",
      "-- appending gene:Spp2-64 with 156 probes\n",
      "-- appending gene:Spp2-65 with 131 probes\n",
      "-- appending gene:Spp2-66 with 154 probes\n",
      "-- appending gene:Spp2-67 with 125 probes\n",
      "-- appending gene:Spp2-68 with 140 probes\n",
      "-- appending gene:Spp2-69 with 154 probes\n",
      "-- appending gene:Spp2-7 with 156 probes\n",
      "-- appending gene:Spp2-70 with 133 probes\n",
      "-- appending gene:Spp2-71 with 156 probes\n",
      "-- appending gene:Spp2-72 with 142 probes\n",
      "-- appending gene:Spp2-73 with 143 probes\n",
      "-- appending gene:Spp2-74 with 147 probes\n",
      "-- appending gene:Spp2-75 with 151 probes\n",
      "-- appending gene:Spp2-76 with 132 probes\n",
      "-- appending gene:Spp2-77 with 158 probes\n",
      "-- appending gene:Spp2-78 with 171 probes\n",
      "-- appending gene:Spp2-79 with 158 probes\n",
      "-- appending gene:Spp2-8 with 148 probes\n",
      "-- appending gene:Spp2-80 with 83 probes\n",
      "-- appending gene:Spp2-81 with 170 probes\n",
      "-- appending gene:Spp2-82 with 177 probes\n",
      "-- appending gene:Spp2-83 with 164 probes\n",
      "-- appending gene:Spp2-84 with 170 probes\n",
      "-- appending gene:Spp2-85 with 163 probes\n",
      "-- appending gene:Spp2-86 with 141 probes\n",
      "-- appending gene:Spp2-87 with 128 probes\n",
      "-- appending gene:Spp2-88 with 144 probes\n",
      "-- appending gene:Spp2-89 with 152 probes\n",
      "-- appending gene:Spp2-9 with 156 probes\n",
      "-- appending gene:Spp2-90 with 88 probes\n",
      "-- appending gene:Spp2-91 with 130 probes\n",
      "-- appending gene:Spp2-92 with 50 probes\n"
     ]
    }
   ],
   "source": [
    "gene_names = np.sort(np.unique([_v['reg_name'].split('reg_')[1].split('_')[0].split(':')[0] for _s, _v in _pbde.kept_probes.items()]))\n",
    "for _gene_name in gene_names:\n",
    "    _sel_pbs = {_s:_v for _s, _v in _pbde.kept_probes.items() \n",
    "                if _v['reg_name'].split('reg_')[1].split('_')[0].split(':')[0]==_gene_name}\n",
    "\n",
    "    if len(_sel_pbs) > 0:\n",
    "        _region_gid = list(_sel_pbs.values())[0]['reg_name'].split('_')[-1]\n",
    "        cand_prob_nums.append(len(_sel_pbs))\n",
    "        if verbose:\n",
    "            print(f\"-- appending gene:{_gene_name} with {len(_sel_pbs)} probes\")\n",
    "        _splitted_pbde = ld.pb_reports_class()\n",
    "        _splitted_pbde.kept_probes = _sel_pbs\n",
    "        pb_dict[_gene_name] = _splitted_pbde"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "id": "juvenile-citation",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Screen probes by genome counts.\n",
      "-- filtering region:Camk1-1, directly append 149 probes\n",
      "-- filtering region:Camk1-10, directly append 140 probes\n",
      "-- filtering region:Camk1-11, screen genome to keep 150 probes\n",
      "-- filtering region:Camk1-12, directly append 109 probes\n",
      "-- filtering region:Camk1-13, directly append 110 probes\n",
      "-- filtering region:Camk1-14, screen genome to keep 150 probes\n",
      "-- filtering region:Camk1-15, screen genome to keep 150 probes\n",
      "-- filtering region:Camk1-16, screen genome to keep 150 probes\n",
      "-- filtering region:Camk1-17, screen genome to keep 150 probes\n",
      "-- filtering region:Camk1-18, screen genome to keep 150 probes\n",
      "-- filtering region:Camk1-19, directly append 148 probes\n",
      "-- filtering region:Camk1-2, directly append 140 probes\n",
      "-- filtering region:Camk1-20, directly append 143 probes\n",
      "-- filtering region:Camk1-21, directly append 136 probes\n",
      "-- filtering region:Camk1-22, directly append 141 probes\n",
      "-- filtering region:Camk1-23, directly append 136 probes\n",
      "-- filtering region:Camk1-24, screen genome to keep 150 probes\n",
      "-- filtering region:Camk1-25, screen genome to keep 150 probes\n",
      "-- filtering region:Camk1-26, directly append 141 probes\n",
      "-- filtering region:Camk1-27, directly append 147 probes\n",
      "-- filtering region:Camk1-28, directly append 113 probes\n",
      "-- filtering region:Camk1-29, screen genome to keep 150 probes\n",
      "-- filtering region:Camk1-3, screen genome to keep 150 probes\n",
      "-- filtering region:Camk1-30, directly append 138 probes\n",
      "-- filtering region:Camk1-31, directly append 103 probes\n",
      "-- filtering region:Camk1-32, directly append 132 probes\n",
      "-- filtering region:Camk1-33, directly append 135 probes\n",
      "-- filtering region:Camk1-34, directly append 126 probes\n",
      "-- filtering region:Camk1-35, directly append 131 probes\n",
      "-- filtering region:Camk1-36, directly append 133 probes\n",
      "-- filtering region:Camk1-37, directly append 131 probes\n",
      "-- filtering region:Camk1-38, directly append 142 probes\n",
      "-- filtering region:Camk1-39, directly append 142 probes\n",
      "-- filtering region:Camk1-4, screen genome to keep 150 probes\n",
      "-- filtering region:Camk1-40, directly append 136 probes\n",
      "-- filtering region:Camk1-41, directly append 121 probes\n",
      "-- filtering region:Camk1-42, directly append 139 probes\n",
      "-- filtering region:Camk1-43, directly append 132 probes\n",
      "-- filtering region:Camk1-44, directly append 133 probes\n",
      "-- filtering region:Camk1-45, directly append 144 probes\n",
      "-- filtering region:Camk1-46, directly append 139 probes\n",
      "-- filtering region:Camk1-47, directly append 141 probes\n",
      "-- filtering region:Camk1-48, directly append 145 probes\n",
      "-- filtering region:Camk1-49, screen genome to keep 150 probes\n",
      "-- filtering region:Camk1-5, screen genome to keep 150 probes\n",
      "-- filtering region:Camk1-50, directly append 149 probes\n",
      "-- filtering region:Camk1-51, screen genome to keep 150 probes\n",
      "-- filtering region:Camk1-52, screen genome to keep 150 probes\n",
      "-- filtering region:Camk1-53, screen genome to keep 150 probes\n",
      "-- filtering region:Camk1-54, screen genome to keep 150 probes\n",
      "-- filtering region:Camk1-6, screen genome to keep 150 probes\n",
      "-- filtering region:Camk1-7, directly append 139 probes\n",
      "-- filtering region:Camk1-8, screen genome to keep 150 probes\n",
      "-- filtering region:Camk1-9, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-1, directly append 121 probes\n",
      "-- filtering region:Hoxd10-10, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-100, directly append 135 probes\n",
      "-- filtering region:Hoxd10-101, directly append 138 probes\n",
      "-- filtering region:Hoxd10-102, directly append 144 probes\n",
      "-- filtering region:Hoxd10-103, directly append 118 probes\n",
      "-- filtering region:Hoxd10-104, directly append 118 probes\n",
      "-- filtering region:Hoxd10-105, directly append 73 probes\n",
      "-- filtering region:Hoxd10-106, directly append 108 probes\n",
      "-- filtering region:Hoxd10-107, directly append 126 probes\n",
      "-- filtering region:Hoxd10-108, directly append 107 probes\n",
      "-- filtering region:Hoxd10-109, directly append 132 probes\n",
      "-- filtering region:Hoxd10-11, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-110, directly append 142 probes\n",
      "-- filtering region:Hoxd10-111, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-112, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-113, directly append 129 probes\n",
      "-- filtering region:Hoxd10-114, directly append 129 probes\n",
      "-- filtering region:Hoxd10-115, directly append 141 probes\n",
      "-- filtering region:Hoxd10-116, directly append 113 probes\n",
      "-- filtering region:Hoxd10-117, directly append 150 probes\n",
      "-- filtering region:Hoxd10-118, directly append 150 probes\n",
      "-- filtering region:Hoxd10-12, directly append 147 probes\n",
      "-- filtering region:Hoxd10-13, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-14, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-15, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-16, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-17, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-18, directly append 144 probes\n",
      "-- filtering region:Hoxd10-19, directly append 128 probes\n",
      "-- filtering region:Hoxd10-2, directly append 84 probes\n",
      "-- filtering region:Hoxd10-20, directly append 127 probes\n",
      "-- filtering region:Hoxd10-21, directly append 116 probes\n",
      "-- filtering region:Hoxd10-22, directly append 120 probes\n",
      "-- filtering region:Hoxd10-23, directly append 119 probes\n",
      "-- filtering region:Hoxd10-24, directly append 123 probes\n",
      "-- filtering region:Hoxd10-25, directly append 118 probes\n",
      "-- filtering region:Hoxd10-26, directly append 118 probes\n",
      "-- filtering region:Hoxd10-27, directly append 105 probes\n",
      "-- filtering region:Hoxd10-28, directly append 109 probes\n",
      "-- filtering region:Hoxd10-29, directly append 125 probes\n",
      "-- filtering region:Hoxd10-3, directly append 126 probes\n",
      "-- filtering region:Hoxd10-30, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-31, directly append 148 probes\n",
      "-- filtering region:Hoxd10-32, directly append 119 probes\n",
      "-- filtering region:Hoxd10-33, directly append 130 probes\n",
      "-- filtering region:Hoxd10-34, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-35, directly append 149 probes\n",
      "-- filtering region:Hoxd10-36, directly append 146 probes\n",
      "-- filtering region:Hoxd10-37, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-38, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-39, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-4, directly append 118 probes\n",
      "-- filtering region:Hoxd10-40, directly append 124 probes\n",
      "-- filtering region:Hoxd10-41, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-42, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-43, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-44, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-45, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-46, directly append 119 probes\n",
      "-- filtering region:Hoxd10-47, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-48, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-49, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-5, directly append 127 probes\n",
      "-- filtering region:Hoxd10-50, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-51, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-52, directly append 143 probes\n",
      "-- filtering region:Hoxd10-53, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-54, directly append 134 probes\n",
      "-- filtering region:Hoxd10-55, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-56, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-57, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-58, directly append 150 probes\n",
      "-- filtering region:Hoxd10-59, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-6, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-60, directly append 126 probes\n",
      "-- filtering region:Hoxd10-61, directly append 144 probes\n",
      "-- filtering region:Hoxd10-62, directly append 134 probes\n",
      "-- filtering region:Hoxd10-63, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-64, directly append 80 probes\n",
      "-- filtering region:Hoxd10-65, directly append 133 probes\n",
      "-- filtering region:Hoxd10-66, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-67, directly append 150 probes\n",
      "-- filtering region:Hoxd10-68, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-69, directly append 141 probes\n",
      "-- filtering region:Hoxd10-7, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-70, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-71, directly append 143 probes\n",
      "-- filtering region:Hoxd10-72, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-73, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-74, directly append 145 probes\n",
      "-- filtering region:Hoxd10-75, directly append 117 probes\n",
      "-- filtering region:Hoxd10-76, directly append 119 probes\n",
      "-- filtering region:Hoxd10-77, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-78, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-79, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-8, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-80, directly append 143 probes\n",
      "-- filtering region:Hoxd10-81, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-82, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-83, directly append 150 probes\n",
      "-- filtering region:Hoxd10-84, directly append 134 probes\n",
      "-- filtering region:Hoxd10-85, directly append 148 probes\n",
      "-- filtering region:Hoxd10-86, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-87, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-88, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-89, directly append 142 probes\n",
      "-- filtering region:Hoxd10-9, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-90, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-91, directly append 109 probes\n",
      "-- filtering region:Hoxd10-92, directly append 106 probes\n",
      "-- filtering region:Hoxd10-93, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-94, directly append 134 probes\n",
      "-- filtering region:Hoxd10-95, directly append 143 probes\n",
      "-- filtering region:Hoxd10-96, directly append 132 probes\n",
      "-- filtering region:Hoxd10-97, directly append 150 probes\n",
      "-- filtering region:Hoxd10-98, screen genome to keep 150 probes\n",
      "-- filtering region:Hoxd10-99, directly append 134 probes\n",
      "-- filtering region:Spp2-1, screen genome to keep 150 probes\n",
      "-- filtering region:Spp2-10, directly append 149 probes\n",
      "-- filtering region:Spp2-11, screen genome to keep 150 probes\n",
      "-- filtering region:Spp2-12, screen genome to keep 150 probes\n",
      "-- filtering region:Spp2-13, directly append 139 probes\n",
      "-- filtering region:Spp2-14, screen genome to keep 150 probes\n",
      "-- filtering region:Spp2-15, directly append 140 probes\n",
      "-- filtering region:Spp2-16, directly append 148 probes\n",
      "-- filtering region:Spp2-17, screen genome to keep 150 probes\n",
      "-- filtering region:Spp2-18, directly append 147 probes\n",
      "-- filtering region:Spp2-19, screen genome to keep 150 probes\n",
      "-- filtering region:Spp2-2, screen genome to keep 150 probes\n",
      "-- filtering region:Spp2-20, screen genome to keep 150 probes\n",
      "-- filtering region:Spp2-21, screen genome to keep 150 probes\n",
      "-- filtering region:Spp2-22, directly append 127 probes\n",
      "-- filtering region:Spp2-23, directly append 135 probes\n",
      "-- filtering region:Spp2-24, screen genome to keep 150 probes\n",
      "-- filtering region:Spp2-25, directly append 120 probes\n",
      "-- filtering region:Spp2-26, screen genome to keep 150 probes\n",
      "-- filtering region:Spp2-27, directly append 99 probes\n",
      "-- filtering region:Spp2-28, directly append 145 probes\n",
      "-- filtering region:Spp2-29, directly append 136 probes\n",
      "-- filtering region:Spp2-3, directly append 145 probes\n",
      "-- filtering region:Spp2-30, directly append 93 probes\n",
      "-- filtering region:Spp2-31, directly append 115 probes\n",
      "-- filtering region:Spp2-32, screen genome to keep 150 probes\n",
      "-- filtering region:Spp2-33, directly append 139 probes\n",
      "-- filtering region:Spp2-34, directly append 144 probes\n",
      "-- filtering region:Spp2-35, screen genome to keep 150 probes\n",
      "-- filtering region:Spp2-36, screen genome to keep 150 probes\n",
      "-- filtering region:Spp2-37, directly append 134 probes\n",
      "-- filtering region:Spp2-38, screen genome to keep 150 probes\n",
      "-- filtering region:Spp2-39, screen genome to keep 150 probes\n",
      "-- filtering region:Spp2-4, directly append 147 probes\n",
      "-- filtering region:Spp2-40, screen genome to keep 150 probes\n",
      "-- filtering region:Spp2-41, screen genome to keep 150 probes\n",
      "-- filtering region:Spp2-42, directly append 144 probes\n",
      "-- filtering region:Spp2-43, directly append 131 probes\n",
      "-- filtering region:Spp2-44, screen genome to keep 150 probes\n",
      "-- filtering region:Spp2-45, directly append 150 probes\n",
      "-- filtering region:Spp2-46, directly append 149 probes\n",
      "-- filtering region:Spp2-47, directly append 142 probes\n",
      "-- filtering region:Spp2-48, directly append 143 probes\n",
      "-- filtering region:Spp2-49, directly append 139 probes\n",
      "-- filtering region:Spp2-5, screen genome to keep 150 probes\n",
      "-- filtering region:Spp2-50, directly append 149 probes\n",
      "-- filtering region:Spp2-51, directly append 150 probes\n",
      "-- filtering region:Spp2-52, directly append 144 probes\n",
      "-- filtering region:Spp2-53, directly append 53 probes\n",
      "-- filtering region:Spp2-54, screen genome to keep 150 probes\n",
      "-- filtering region:Spp2-55, directly append 87 probes\n",
      "-- filtering region:Spp2-56, directly append 126 probes\n",
      "-- filtering region:Spp2-57, directly append 149 probes\n",
      "-- filtering region:Spp2-58, directly append 146 probes\n",
      "-- filtering region:Spp2-59, directly append 120 probes\n",
      "-- filtering region:Spp2-6, screen genome to keep 150 probes\n",
      "-- filtering region:Spp2-60, directly append 139 probes\n",
      "-- filtering region:Spp2-61, directly append 147 probes\n",
      "-- filtering region:Spp2-62, directly append 147 probes\n",
      "-- filtering region:Spp2-63, directly append 144 probes\n",
      "-- filtering region:Spp2-64, screen genome to keep 150 probes\n",
      "-- filtering region:Spp2-65, directly append 131 probes\n",
      "-- filtering region:Spp2-66, screen genome to keep 150 probes\n",
      "-- filtering region:Spp2-67, directly append 125 probes\n",
      "-- filtering region:Spp2-68, directly append 140 probes\n",
      "-- filtering region:Spp2-69, screen genome to keep 150 probes\n",
      "-- filtering region:Spp2-7, screen genome to keep 150 probes\n",
      "-- filtering region:Spp2-70, directly append 133 probes\n",
      "-- filtering region:Spp2-71, screen genome to keep 150 probes\n",
      "-- filtering region:Spp2-72, directly append 142 probes\n",
      "-- filtering region:Spp2-73, directly append 143 probes\n",
      "-- filtering region:Spp2-74, directly append 147 probes\n",
      "-- filtering region:Spp2-75, screen genome to keep 150 probes\n",
      "-- filtering region:Spp2-76, directly append 132 probes\n",
      "-- filtering region:Spp2-77, screen genome to keep 150 probes\n",
      "-- filtering region:Spp2-78, screen genome to keep 150 probes\n",
      "-- filtering region:Spp2-79, screen genome to keep 150 probes\n",
      "-- filtering region:Spp2-8, directly append 148 probes\n",
      "-- filtering region:Spp2-80, directly append 83 probes\n",
      "-- filtering region:Spp2-81, screen genome to keep 150 probes\n",
      "-- filtering region:Spp2-82, screen genome to keep 150 probes\n",
      "-- filtering region:Spp2-83, screen genome to keep 150 probes\n",
      "-- filtering region:Spp2-84, screen genome to keep 150 probes\n",
      "-- filtering region:Spp2-85, screen genome to keep 150 probes\n",
      "-- filtering region:Spp2-86, directly append 141 probes\n",
      "-- filtering region:Spp2-87, directly append 128 probes\n",
      "-- filtering region:Spp2-88, directly append 144 probes\n",
      "-- filtering region:Spp2-89, screen genome to keep 150 probes\n",
      "-- filtering region:Spp2-9, screen genome to keep 150 probes\n",
      "-- filtering region:Spp2-90, directly append 88 probes\n",
      "-- filtering region:Spp2-91, directly append 130 probes\n",
      "-- filtering region:Spp2-92, directly append 50 probes\n"
     ]
    }
   ],
   "source": [
    "# filter by genome counts\n",
    "reload(library_tools.quality_check)\n",
    "\n",
    "num_probes_kept = 150 # max num of probes kept for each region\n",
    "filtered_pb_dict = library_tools.quality_check.Screen_probe_by_hit(library_folder, pb_dict, num_probes_kept,\n",
    "                                                                  hit_type='genome')\n",
    "\n",
    "## filter by minimum number of probes in each region\n",
    "min_num_probes = 50\n",
    "removed_regions = []\n",
    "for _rid, _pb_class in filtered_pb_dict.items():\n",
    "    if len(_pb_class.kept_probes) < min_num_probes:\n",
    "        print(f\"region: {_rid} removed with {len(_pb_class.kept_probes)} probes\")\n",
    "        removed_regions.append(_rid)\n",
    "for _rid in removed_regions:\n",
    "    del(filtered_pb_dict[_rid])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "id": "simple-hepatitis",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "/* Put everything inside the global mpl namespace */\n",
       "/* global mpl */\n",
       "window.mpl = {};\n",
       "\n",
       "mpl.get_websocket_type = function () {\n",
       "    if (typeof WebSocket !== 'undefined') {\n",
       "        return WebSocket;\n",
       "    } else if (typeof MozWebSocket !== 'undefined') {\n",
       "        return MozWebSocket;\n",
       "    } else {\n",
       "        alert(\n",
       "            'Your browser does not have WebSocket support. ' +\n",
       "                'Please try Chrome, Safari or Firefox ≥ 6. ' +\n",
       "                'Firefox 4 and 5 are also supported but you ' +\n",
       "                'have to enable WebSockets in about:config.'\n",
       "        );\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure = function (figure_id, websocket, ondownload, parent_element) {\n",
       "    this.id = figure_id;\n",
       "\n",
       "    this.ws = websocket;\n",
       "\n",
       "    this.supports_binary = this.ws.binaryType !== undefined;\n",
       "\n",
       "    if (!this.supports_binary) {\n",
       "        var warnings = document.getElementById('mpl-warnings');\n",
       "        if (warnings) {\n",
       "            warnings.style.display = 'block';\n",
       "            warnings.textContent =\n",
       "                'This browser does not support binary websocket messages. ' +\n",
       "                'Performance may be slow.';\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.imageObj = new Image();\n",
       "\n",
       "    this.context = undefined;\n",
       "    this.message = undefined;\n",
       "    this.canvas = undefined;\n",
       "    this.rubberband_canvas = undefined;\n",
       "    this.rubberband_context = undefined;\n",
       "    this.format_dropdown = undefined;\n",
       "\n",
       "    this.image_mode = 'full';\n",
       "\n",
       "    this.root = document.createElement('div');\n",
       "    this.root.setAttribute('style', 'display: inline-block');\n",
       "    this._root_extra_style(this.root);\n",
       "\n",
       "    parent_element.appendChild(this.root);\n",
       "\n",
       "    this._init_header(this);\n",
       "    this._init_canvas(this);\n",
       "    this._init_toolbar(this);\n",
       "\n",
       "    var fig = this;\n",
       "\n",
       "    this.waiting = false;\n",
       "\n",
       "    this.ws.onopen = function () {\n",
       "        fig.send_message('supports_binary', { value: fig.supports_binary });\n",
       "        fig.send_message('send_image_mode', {});\n",
       "        if (fig.ratio !== 1) {\n",
       "            fig.send_message('set_dpi_ratio', { dpi_ratio: fig.ratio });\n",
       "        }\n",
       "        fig.send_message('refresh', {});\n",
       "    };\n",
       "\n",
       "    this.imageObj.onload = function () {\n",
       "        if (fig.image_mode === 'full') {\n",
       "            // Full images could contain transparency (where diff images\n",
       "            // almost always do), so we need to clear the canvas so that\n",
       "            // there is no ghosting.\n",
       "            fig.context.clearRect(0, 0, fig.canvas.width, fig.canvas.height);\n",
       "        }\n",
       "        fig.context.drawImage(fig.imageObj, 0, 0);\n",
       "    };\n",
       "\n",
       "    this.imageObj.onunload = function () {\n",
       "        fig.ws.close();\n",
       "    };\n",
       "\n",
       "    this.ws.onmessage = this._make_on_message_function(this);\n",
       "\n",
       "    this.ondownload = ondownload;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_header = function () {\n",
       "    var titlebar = document.createElement('div');\n",
       "    titlebar.classList =\n",
       "        'ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix';\n",
       "    var titletext = document.createElement('div');\n",
       "    titletext.classList = 'ui-dialog-title';\n",
       "    titletext.setAttribute(\n",
       "        'style',\n",
       "        'width: 100%; text-align: center; padding: 3px;'\n",
       "    );\n",
       "    titlebar.appendChild(titletext);\n",
       "    this.root.appendChild(titlebar);\n",
       "    this.header = titletext;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._init_canvas = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var canvas_div = (this.canvas_div = document.createElement('div'));\n",
       "    canvas_div.setAttribute(\n",
       "        'style',\n",
       "        'border: 1px solid #ddd;' +\n",
       "            'box-sizing: content-box;' +\n",
       "            'clear: both;' +\n",
       "            'min-height: 1px;' +\n",
       "            'min-width: 1px;' +\n",
       "            'outline: 0;' +\n",
       "            'overflow: hidden;' +\n",
       "            'position: relative;' +\n",
       "            'resize: both;'\n",
       "    );\n",
       "\n",
       "    function on_keyboard_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.key_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    canvas_div.addEventListener(\n",
       "        'keydown',\n",
       "        on_keyboard_event_closure('key_press')\n",
       "    );\n",
       "    canvas_div.addEventListener(\n",
       "        'keyup',\n",
       "        on_keyboard_event_closure('key_release')\n",
       "    );\n",
       "\n",
       "    this._canvas_extra_style(canvas_div);\n",
       "    this.root.appendChild(canvas_div);\n",
       "\n",
       "    var canvas = (this.canvas = document.createElement('canvas'));\n",
       "    canvas.classList.add('mpl-canvas');\n",
       "    canvas.setAttribute('style', 'box-sizing: content-box;');\n",
       "\n",
       "    this.context = canvas.getContext('2d');\n",
       "\n",
       "    var backingStore =\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        this.context.webkitBackingStorePixelRatio ||\n",
       "        this.context.mozBackingStorePixelRatio ||\n",
       "        this.context.msBackingStorePixelRatio ||\n",
       "        this.context.oBackingStorePixelRatio ||\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        1;\n",
       "\n",
       "    this.ratio = (window.devicePixelRatio || 1) / backingStore;\n",
       "\n",
       "    var rubberband_canvas = (this.rubberband_canvas = document.createElement(\n",
       "        'canvas'\n",
       "    ));\n",
       "    rubberband_canvas.setAttribute(\n",
       "        'style',\n",
       "        'box-sizing: content-box; position: absolute; left: 0; top: 0; z-index: 1;'\n",
       "    );\n",
       "\n",
       "    // Apply a ponyfill if ResizeObserver is not implemented by browser.\n",
       "    if (this.ResizeObserver === undefined) {\n",
       "        if (window.ResizeObserver !== undefined) {\n",
       "            this.ResizeObserver = window.ResizeObserver;\n",
       "        } else {\n",
       "            var obs = _JSXTOOLS_RESIZE_OBSERVER({});\n",
       "            this.ResizeObserver = obs.ResizeObserver;\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.resizeObserverInstance = new this.ResizeObserver(function (entries) {\n",
       "        var nentries = entries.length;\n",
       "        for (var i = 0; i < nentries; i++) {\n",
       "            var entry = entries[i];\n",
       "            var width, height;\n",
       "            if (entry.contentBoxSize) {\n",
       "                if (entry.contentBoxSize instanceof Array) {\n",
       "                    // Chrome 84 implements new version of spec.\n",
       "                    width = entry.contentBoxSize[0].inlineSize;\n",
       "                    height = entry.contentBoxSize[0].blockSize;\n",
       "                } else {\n",
       "                    // Firefox implements old version of spec.\n",
       "                    width = entry.contentBoxSize.inlineSize;\n",
       "                    height = entry.contentBoxSize.blockSize;\n",
       "                }\n",
       "            } else {\n",
       "                // Chrome <84 implements even older version of spec.\n",
       "                width = entry.contentRect.width;\n",
       "                height = entry.contentRect.height;\n",
       "            }\n",
       "\n",
       "            // Keep the size of the canvas and rubber band canvas in sync with\n",
       "            // the canvas container.\n",
       "            if (entry.devicePixelContentBoxSize) {\n",
       "                // Chrome 84 implements new version of spec.\n",
       "                canvas.setAttribute(\n",
       "                    'width',\n",
       "                    entry.devicePixelContentBoxSize[0].inlineSize\n",
       "                );\n",
       "                canvas.setAttribute(\n",
       "                    'height',\n",
       "                    entry.devicePixelContentBoxSize[0].blockSize\n",
       "                );\n",
       "            } else {\n",
       "                canvas.setAttribute('width', width * fig.ratio);\n",
       "                canvas.setAttribute('height', height * fig.ratio);\n",
       "            }\n",
       "            canvas.setAttribute(\n",
       "                'style',\n",
       "                'width: ' + width + 'px; height: ' + height + 'px;'\n",
       "            );\n",
       "\n",
       "            rubberband_canvas.setAttribute('width', width);\n",
       "            rubberband_canvas.setAttribute('height', height);\n",
       "\n",
       "            // And update the size in Python. We ignore the initial 0/0 size\n",
       "            // that occurs as the element is placed into the DOM, which should\n",
       "            // otherwise not happen due to the minimum size styling.\n",
       "            if (fig.ws.readyState == 1 && width != 0 && height != 0) {\n",
       "                fig.request_resize(width, height);\n",
       "            }\n",
       "        }\n",
       "    });\n",
       "    this.resizeObserverInstance.observe(canvas_div);\n",
       "\n",
       "    function on_mouse_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.mouse_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousedown',\n",
       "        on_mouse_event_closure('button_press')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseup',\n",
       "        on_mouse_event_closure('button_release')\n",
       "    );\n",
       "    // Throttle sequential mouse events to 1 every 20ms.\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousemove',\n",
       "        on_mouse_event_closure('motion_notify')\n",
       "    );\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseenter',\n",
       "        on_mouse_event_closure('figure_enter')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseleave',\n",
       "        on_mouse_event_closure('figure_leave')\n",
       "    );\n",
       "\n",
       "    canvas_div.addEventListener('wheel', function (event) {\n",
       "        if (event.deltaY < 0) {\n",
       "            event.step = 1;\n",
       "        } else {\n",
       "            event.step = -1;\n",
       "        }\n",
       "        on_mouse_event_closure('scroll')(event);\n",
       "    });\n",
       "\n",
       "    canvas_div.appendChild(canvas);\n",
       "    canvas_div.appendChild(rubberband_canvas);\n",
       "\n",
       "    this.rubberband_context = rubberband_canvas.getContext('2d');\n",
       "    this.rubberband_context.strokeStyle = '#000000';\n",
       "\n",
       "    this._resize_canvas = function (width, height, forward) {\n",
       "        if (forward) {\n",
       "            canvas_div.style.width = width + 'px';\n",
       "            canvas_div.style.height = height + 'px';\n",
       "        }\n",
       "    };\n",
       "\n",
       "    // Disable right mouse context menu.\n",
       "    this.rubberband_canvas.addEventListener('contextmenu', function (_e) {\n",
       "        event.preventDefault();\n",
       "        return false;\n",
       "    });\n",
       "\n",
       "    function set_focus() {\n",
       "        canvas.focus();\n",
       "        canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    window.setTimeout(set_focus, 100);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'mpl-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'mpl-button-group';\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'mpl-button-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        var button = (fig.buttons[name] = document.createElement('button'));\n",
       "        button.classList = 'mpl-widget';\n",
       "        button.setAttribute('role', 'button');\n",
       "        button.setAttribute('aria-disabled', 'false');\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "\n",
       "        var icon_img = document.createElement('img');\n",
       "        icon_img.src = '_images/' + image + '.png';\n",
       "        icon_img.srcset = '_images/' + image + '_large.png 2x';\n",
       "        icon_img.alt = tooltip;\n",
       "        button.appendChild(icon_img);\n",
       "\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    var fmt_picker = document.createElement('select');\n",
       "    fmt_picker.classList = 'mpl-widget';\n",
       "    toolbar.appendChild(fmt_picker);\n",
       "    this.format_dropdown = fmt_picker;\n",
       "\n",
       "    for (var ind in mpl.extensions) {\n",
       "        var fmt = mpl.extensions[ind];\n",
       "        var option = document.createElement('option');\n",
       "        option.selected = fmt === mpl.default_extension;\n",
       "        option.innerHTML = fmt;\n",
       "        fmt_picker.appendChild(option);\n",
       "    }\n",
       "\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.request_resize = function (x_pixels, y_pixels) {\n",
       "    // Request matplotlib to resize the figure. Matplotlib will then trigger a resize in the client,\n",
       "    // which will in turn request a refresh of the image.\n",
       "    this.send_message('resize', { width: x_pixels, height: y_pixels });\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_message = function (type, properties) {\n",
       "    properties['type'] = type;\n",
       "    properties['figure_id'] = this.id;\n",
       "    this.ws.send(JSON.stringify(properties));\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_draw_message = function () {\n",
       "    if (!this.waiting) {\n",
       "        this.waiting = true;\n",
       "        this.ws.send(JSON.stringify({ type: 'draw', figure_id: this.id }));\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    var format_dropdown = fig.format_dropdown;\n",
       "    var format = format_dropdown.options[format_dropdown.selectedIndex].value;\n",
       "    fig.ondownload(fig, format);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_resize = function (fig, msg) {\n",
       "    var size = msg['size'];\n",
       "    if (size[0] !== fig.canvas.width || size[1] !== fig.canvas.height) {\n",
       "        fig._resize_canvas(size[0], size[1], msg['forward']);\n",
       "        fig.send_message('refresh', {});\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_rubberband = function (fig, msg) {\n",
       "    var x0 = msg['x0'] / fig.ratio;\n",
       "    var y0 = (fig.canvas.height - msg['y0']) / fig.ratio;\n",
       "    var x1 = msg['x1'] / fig.ratio;\n",
       "    var y1 = (fig.canvas.height - msg['y1']) / fig.ratio;\n",
       "    x0 = Math.floor(x0) + 0.5;\n",
       "    y0 = Math.floor(y0) + 0.5;\n",
       "    x1 = Math.floor(x1) + 0.5;\n",
       "    y1 = Math.floor(y1) + 0.5;\n",
       "    var min_x = Math.min(x0, x1);\n",
       "    var min_y = Math.min(y0, y1);\n",
       "    var width = Math.abs(x1 - x0);\n",
       "    var height = Math.abs(y1 - y0);\n",
       "\n",
       "    fig.rubberband_context.clearRect(\n",
       "        0,\n",
       "        0,\n",
       "        fig.canvas.width / fig.ratio,\n",
       "        fig.canvas.height / fig.ratio\n",
       "    );\n",
       "\n",
       "    fig.rubberband_context.strokeRect(min_x, min_y, width, height);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_figure_label = function (fig, msg) {\n",
       "    // Updates the figure title.\n",
       "    fig.header.textContent = msg['label'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_cursor = function (fig, msg) {\n",
       "    var cursor = msg['cursor'];\n",
       "    switch (cursor) {\n",
       "        case 0:\n",
       "            cursor = 'pointer';\n",
       "            break;\n",
       "        case 1:\n",
       "            cursor = 'default';\n",
       "            break;\n",
       "        case 2:\n",
       "            cursor = 'crosshair';\n",
       "            break;\n",
       "        case 3:\n",
       "            cursor = 'move';\n",
       "            break;\n",
       "    }\n",
       "    fig.rubberband_canvas.style.cursor = cursor;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_message = function (fig, msg) {\n",
       "    fig.message.textContent = msg['message'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_draw = function (fig, _msg) {\n",
       "    // Request the server to send over a new figure.\n",
       "    fig.send_draw_message();\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_image_mode = function (fig, msg) {\n",
       "    fig.image_mode = msg['mode'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_history_buttons = function (fig, msg) {\n",
       "    for (var key in msg) {\n",
       "        if (!(key in fig.buttons)) {\n",
       "            continue;\n",
       "        }\n",
       "        fig.buttons[key].disabled = !msg[key];\n",
       "        fig.buttons[key].setAttribute('aria-disabled', !msg[key]);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_navigate_mode = function (fig, msg) {\n",
       "    if (msg['mode'] === 'PAN') {\n",
       "        fig.buttons['Pan'].classList.add('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    } else if (msg['mode'] === 'ZOOM') {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.add('active');\n",
       "    } else {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Called whenever the canvas gets updated.\n",
       "    this.send_message('ack', {});\n",
       "};\n",
       "\n",
       "// A function to construct a web socket function for onmessage handling.\n",
       "// Called in the figure constructor.\n",
       "mpl.figure.prototype._make_on_message_function = function (fig) {\n",
       "    return function socket_on_message(evt) {\n",
       "        if (evt.data instanceof Blob) {\n",
       "            /* FIXME: We get \"Resource interpreted as Image but\n",
       "             * transferred with MIME type text/plain:\" errors on\n",
       "             * Chrome.  But how to set the MIME type?  It doesn't seem\n",
       "             * to be part of the websocket stream */\n",
       "            evt.data.type = 'image/png';\n",
       "\n",
       "            /* Free the memory for the previous frames */\n",
       "            if (fig.imageObj.src) {\n",
       "                (window.URL || window.webkitURL).revokeObjectURL(\n",
       "                    fig.imageObj.src\n",
       "                );\n",
       "            }\n",
       "\n",
       "            fig.imageObj.src = (window.URL || window.webkitURL).createObjectURL(\n",
       "                evt.data\n",
       "            );\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        } else if (\n",
       "            typeof evt.data === 'string' &&\n",
       "            evt.data.slice(0, 21) === 'data:image/png;base64'\n",
       "        ) {\n",
       "            fig.imageObj.src = evt.data;\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        var msg = JSON.parse(evt.data);\n",
       "        var msg_type = msg['type'];\n",
       "\n",
       "        // Call the  \"handle_{type}\" callback, which takes\n",
       "        // the figure and JSON message as its only arguments.\n",
       "        try {\n",
       "            var callback = fig['handle_' + msg_type];\n",
       "        } catch (e) {\n",
       "            console.log(\n",
       "                \"No handler for the '\" + msg_type + \"' message type: \",\n",
       "                msg\n",
       "            );\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        if (callback) {\n",
       "            try {\n",
       "                // console.log(\"Handling '\" + msg_type + \"' message: \", msg);\n",
       "                callback(fig, msg);\n",
       "            } catch (e) {\n",
       "                console.log(\n",
       "                    \"Exception inside the 'handler_\" + msg_type + \"' callback:\",\n",
       "                    e,\n",
       "                    e.stack,\n",
       "                    msg\n",
       "                );\n",
       "            }\n",
       "        }\n",
       "    };\n",
       "};\n",
       "\n",
       "// from http://stackoverflow.com/questions/1114465/getting-mouse-location-in-canvas\n",
       "mpl.findpos = function (e) {\n",
       "    //this section is from http://www.quirksmode.org/js/events_properties.html\n",
       "    var targ;\n",
       "    if (!e) {\n",
       "        e = window.event;\n",
       "    }\n",
       "    if (e.target) {\n",
       "        targ = e.target;\n",
       "    } else if (e.srcElement) {\n",
       "        targ = e.srcElement;\n",
       "    }\n",
       "    if (targ.nodeType === 3) {\n",
       "        // defeat Safari bug\n",
       "        targ = targ.parentNode;\n",
       "    }\n",
       "\n",
       "    // pageX,Y are the mouse positions relative to the document\n",
       "    var boundingRect = targ.getBoundingClientRect();\n",
       "    var x = e.pageX - (boundingRect.left + document.body.scrollLeft);\n",
       "    var y = e.pageY - (boundingRect.top + document.body.scrollTop);\n",
       "\n",
       "    return { x: x, y: y };\n",
       "};\n",
       "\n",
       "/*\n",
       " * return a copy of an object with only non-object keys\n",
       " * we need this to avoid circular references\n",
       " * http://stackoverflow.com/a/24161582/3208463\n",
       " */\n",
       "function simpleKeys(original) {\n",
       "    return Object.keys(original).reduce(function (obj, key) {\n",
       "        if (typeof original[key] !== 'object') {\n",
       "            obj[key] = original[key];\n",
       "        }\n",
       "        return obj;\n",
       "    }, {});\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.mouse_event = function (event, name) {\n",
       "    var canvas_pos = mpl.findpos(event);\n",
       "\n",
       "    if (name === 'button_press') {\n",
       "        this.canvas.focus();\n",
       "        this.canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    var x = canvas_pos.x * this.ratio;\n",
       "    var y = canvas_pos.y * this.ratio;\n",
       "\n",
       "    this.send_message(name, {\n",
       "        x: x,\n",
       "        y: y,\n",
       "        button: event.button,\n",
       "        step: event.step,\n",
       "        guiEvent: simpleKeys(event),\n",
       "    });\n",
       "\n",
       "    /* This prevents the web browser from automatically changing to\n",
       "     * the text insertion cursor when the button is pressed.  We want\n",
       "     * to control all of the cursor setting manually through the\n",
       "     * 'cursor' event from matplotlib */\n",
       "    event.preventDefault();\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (_event, _name) {\n",
       "    // Handle any extra behaviour associated with a key event\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.key_event = function (event, name) {\n",
       "    // Prevent repeat events\n",
       "    if (name === 'key_press') {\n",
       "        if (event.which === this._key) {\n",
       "            return;\n",
       "        } else {\n",
       "            this._key = event.which;\n",
       "        }\n",
       "    }\n",
       "    if (name === 'key_release') {\n",
       "        this._key = null;\n",
       "    }\n",
       "\n",
       "    var value = '';\n",
       "    if (event.ctrlKey && event.which !== 17) {\n",
       "        value += 'ctrl+';\n",
       "    }\n",
       "    if (event.altKey && event.which !== 18) {\n",
       "        value += 'alt+';\n",
       "    }\n",
       "    if (event.shiftKey && event.which !== 16) {\n",
       "        value += 'shift+';\n",
       "    }\n",
       "\n",
       "    value += 'k';\n",
       "    value += event.which.toString();\n",
       "\n",
       "    this._key_event_extra(event, name);\n",
       "\n",
       "    this.send_message(name, { key: value, guiEvent: simpleKeys(event) });\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onclick = function (name) {\n",
       "    if (name === 'download') {\n",
       "        this.handle_save(this, null);\n",
       "    } else {\n",
       "        this.send_message('toolbar_button', { name: name });\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onmouseover = function (tooltip) {\n",
       "    this.message.textContent = tooltip;\n",
       "};\n",
       "\n",
       "///////////////// REMAINING CONTENT GENERATED BY embed_js.py /////////////////\n",
       "// prettier-ignore\n",
       "var _JSXTOOLS_RESIZE_OBSERVER=function(A){var t,i=new WeakMap,n=new WeakMap,a=new WeakMap,r=new WeakMap,o=new Set;function s(e){if(!(this instanceof s))throw new TypeError(\"Constructor requires 'new' operator\");i.set(this,e)}function h(){throw new TypeError(\"Function is not a constructor\")}function c(e,t,i,n){e=0 in arguments?Number(arguments[0]):0,t=1 in arguments?Number(arguments[1]):0,i=2 in arguments?Number(arguments[2]):0,n=3 in arguments?Number(arguments[3]):0,this.right=(this.x=this.left=e)+(this.width=i),this.bottom=(this.y=this.top=t)+(this.height=n),Object.freeze(this)}function d(){t=requestAnimationFrame(d);var s=new WeakMap,p=new Set;o.forEach((function(t){r.get(t).forEach((function(i){var r=t instanceof window.SVGElement,o=a.get(t),d=r?0:parseFloat(o.paddingTop),f=r?0:parseFloat(o.paddingRight),l=r?0:parseFloat(o.paddingBottom),u=r?0:parseFloat(o.paddingLeft),g=r?0:parseFloat(o.borderTopWidth),m=r?0:parseFloat(o.borderRightWidth),w=r?0:parseFloat(o.borderBottomWidth),b=u+f,F=d+l,v=(r?0:parseFloat(o.borderLeftWidth))+m,W=g+w,y=r?0:t.offsetHeight-W-t.clientHeight,E=r?0:t.offsetWidth-v-t.clientWidth,R=b+v,z=F+W,M=r?t.width:parseFloat(o.width)-R-E,O=r?t.height:parseFloat(o.height)-z-y;if(n.has(t)){var k=n.get(t);if(k[0]===M&&k[1]===O)return}n.set(t,[M,O]);var S=Object.create(h.prototype);S.target=t,S.contentRect=new c(u,d,M,O),s.has(i)||(s.set(i,[]),p.add(i)),s.get(i).push(S)}))})),p.forEach((function(e){i.get(e).call(e,s.get(e),e)}))}return s.prototype.observe=function(i){if(i instanceof window.Element){r.has(i)||(r.set(i,new Set),o.add(i),a.set(i,window.getComputedStyle(i)));var n=r.get(i);n.has(this)||n.add(this),cancelAnimationFrame(t),t=requestAnimationFrame(d)}},s.prototype.unobserve=function(i){if(i instanceof window.Element&&r.has(i)){var n=r.get(i);n.has(this)&&(n.delete(this),n.size||(r.delete(i),o.delete(i))),n.size||r.delete(i),o.size||cancelAnimationFrame(t)}},A.DOMRectReadOnly=c,A.ResizeObserver=s,A.ResizeObserverEntry=h,A}; // eslint-disable-line\n",
       "mpl.toolbar_items = [[\"Home\", \"Reset original view\", \"fa fa-home icon-home\", \"home\"], [\"Back\", \"Back to previous view\", \"fa fa-arrow-left icon-arrow-left\", \"back\"], [\"Forward\", \"Forward to next view\", \"fa fa-arrow-right icon-arrow-right\", \"forward\"], [\"\", \"\", \"\", \"\"], [\"Pan\", \"Left button pans, Right button zooms\\nx/y fixes axis, CTRL fixes aspect\", \"fa fa-arrows icon-move\", \"pan\"], [\"Zoom\", \"Zoom to rectangle\\nx/y fixes axis, CTRL fixes aspect\", \"fa fa-square-o icon-check-empty\", \"zoom\"], [\"\", \"\", \"\", \"\"], [\"Download\", \"Download plot\", \"fa fa-floppy-o icon-save\", \"download\"]];\n",
       "\n",
       "mpl.extensions = [\"eps\", \"jpeg\", \"pdf\", \"png\", \"ps\", \"raw\", \"svg\", \"tif\"];\n",
       "\n",
       "mpl.default_extension = \"png\";/* global mpl */\n",
       "\n",
       "var comm_websocket_adapter = function (comm) {\n",
       "    // Create a \"websocket\"-like object which calls the given IPython comm\n",
       "    // object with the appropriate methods. Currently this is a non binary\n",
       "    // socket, so there is still some room for performance tuning.\n",
       "    var ws = {};\n",
       "\n",
       "    ws.close = function () {\n",
       "        comm.close();\n",
       "    };\n",
       "    ws.send = function (m) {\n",
       "        //console.log('sending', m);\n",
       "        comm.send(m);\n",
       "    };\n",
       "    // Register the callback with on_msg.\n",
       "    comm.on_msg(function (msg) {\n",
       "        //console.log('receiving', msg['content']['data'], msg);\n",
       "        // Pass the mpl event to the overridden (by mpl) onmessage function.\n",
       "        ws.onmessage(msg['content']['data']);\n",
       "    });\n",
       "    return ws;\n",
       "};\n",
       "\n",
       "mpl.mpl_figure_comm = function (comm, msg) {\n",
       "    // This is the function which gets called when the mpl process\n",
       "    // starts-up an IPython Comm through the \"matplotlib\" channel.\n",
       "\n",
       "    var id = msg.content.data.id;\n",
       "    // Get hold of the div created by the display call when the Comm\n",
       "    // socket was opened in Python.\n",
       "    var element = document.getElementById(id);\n",
       "    var ws_proxy = comm_websocket_adapter(comm);\n",
       "\n",
       "    function ondownload(figure, _format) {\n",
       "        window.open(figure.canvas.toDataURL());\n",
       "    }\n",
       "\n",
       "    var fig = new mpl.figure(id, ws_proxy, ondownload, element);\n",
       "\n",
       "    // Call onopen now - mpl needs it, as it is assuming we've passed it a real\n",
       "    // web socket which is closed, not our websocket->open comm proxy.\n",
       "    ws_proxy.onopen();\n",
       "\n",
       "    fig.parent_element = element;\n",
       "    fig.cell_info = mpl.find_output_cell(\"<div id='\" + id + \"'></div>\");\n",
       "    if (!fig.cell_info) {\n",
       "        console.error('Failed to find cell for figure', id, fig);\n",
       "        return;\n",
       "    }\n",
       "    fig.cell_info[0].output_area.element.on(\n",
       "        'cleared',\n",
       "        { fig: fig },\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_close = function (fig, msg) {\n",
       "    var width = fig.canvas.width / fig.ratio;\n",
       "    fig.cell_info[0].output_area.element.off(\n",
       "        'cleared',\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "    fig.resizeObserverInstance.unobserve(fig.canvas_div);\n",
       "\n",
       "    // Update the output cell to use the data from the current canvas.\n",
       "    fig.push_to_output();\n",
       "    var dataURL = fig.canvas.toDataURL();\n",
       "    // Re-enable the keyboard manager in IPython - without this line, in FF,\n",
       "    // the notebook keyboard shortcuts fail.\n",
       "    IPython.keyboard_manager.enable();\n",
       "    fig.parent_element.innerHTML =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "    fig.close_ws(fig, msg);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.close_ws = function (fig, msg) {\n",
       "    fig.send_message('closing', msg);\n",
       "    // fig.ws.close()\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.push_to_output = function (_remove_interactive) {\n",
       "    // Turn the data on the canvas into data in the output cell.\n",
       "    var width = this.canvas.width / this.ratio;\n",
       "    var dataURL = this.canvas.toDataURL();\n",
       "    this.cell_info[1]['text/html'] =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Tell IPython that the notebook contents must change.\n",
       "    IPython.notebook.set_dirty(true);\n",
       "    this.send_message('ack', {});\n",
       "    var fig = this;\n",
       "    // Wait a second, then push the new image to the DOM so\n",
       "    // that it is saved nicely (might be nice to debounce this).\n",
       "    setTimeout(function () {\n",
       "        fig.push_to_output();\n",
       "    }, 1000);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'btn-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'btn-group';\n",
       "    var button;\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'btn-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        button = fig.buttons[name] = document.createElement('button');\n",
       "        button.classList = 'btn btn-default';\n",
       "        button.href = '#';\n",
       "        button.title = name;\n",
       "        button.innerHTML = '<i class=\"fa ' + image + ' fa-lg\"></i>';\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    // Add the status bar.\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message pull-right';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "\n",
       "    // Add the close button to the window.\n",
       "    var buttongrp = document.createElement('div');\n",
       "    buttongrp.classList = 'btn-group inline pull-right';\n",
       "    button = document.createElement('button');\n",
       "    button.classList = 'btn btn-mini btn-primary';\n",
       "    button.href = '#';\n",
       "    button.title = 'Stop Interaction';\n",
       "    button.innerHTML = '<i class=\"fa fa-power-off icon-remove icon-large\"></i>';\n",
       "    button.addEventListener('click', function (_evt) {\n",
       "        fig.handle_close(fig, {});\n",
       "    });\n",
       "    button.addEventListener(\n",
       "        'mouseover',\n",
       "        on_mouseover_closure('Stop Interaction')\n",
       "    );\n",
       "    buttongrp.appendChild(button);\n",
       "    var titlebar = this.root.querySelector('.ui-dialog-titlebar');\n",
       "    titlebar.insertBefore(buttongrp, titlebar.firstChild);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._remove_fig_handler = function (event) {\n",
       "    var fig = event.data.fig;\n",
       "    if (event.target !== this) {\n",
       "        // Ignore bubbled events from children.\n",
       "        return;\n",
       "    }\n",
       "    fig.close_ws(fig, {});\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (el) {\n",
       "    el.style.boxSizing = 'content-box'; // override notebook setting of border-box.\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (el) {\n",
       "    // this is important to make the div 'focusable\n",
       "    el.setAttribute('tabindex', 0);\n",
       "    // reach out to IPython and tell the keyboard manager to turn it's self\n",
       "    // off when our div gets focus\n",
       "\n",
       "    // location in version 3\n",
       "    if (IPython.notebook.keyboard_manager) {\n",
       "        IPython.notebook.keyboard_manager.register_events(el);\n",
       "    } else {\n",
       "        // location in version 2\n",
       "        IPython.keyboard_manager.register_events(el);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (event, _name) {\n",
       "    var manager = IPython.notebook.keyboard_manager;\n",
       "    if (!manager) {\n",
       "        manager = IPython.keyboard_manager;\n",
       "    }\n",
       "\n",
       "    // Check for shift+enter\n",
       "    if (event.shiftKey && event.which === 13) {\n",
       "        this.canvas_div.blur();\n",
       "        // select the cell after this one\n",
       "        var index = IPython.notebook.find_cell_index(this.cell_info[0]);\n",
       "        IPython.notebook.select(index + 1);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    fig.ondownload(fig, null);\n",
       "};\n",
       "\n",
       "mpl.find_output_cell = function (html_output) {\n",
       "    // Return the cell and output element which can be found *uniquely* in the notebook.\n",
       "    // Note - this is a bit hacky, but it is done because the \"notebook_saving.Notebook\"\n",
       "    // IPython event is triggered only after the cells have been serialised, which for\n",
       "    // our purposes (turning an active figure into a static one), is too late.\n",
       "    var cells = IPython.notebook.get_cells();\n",
       "    var ncells = cells.length;\n",
       "    for (var i = 0; i < ncells; i++) {\n",
       "        var cell = cells[i];\n",
       "        if (cell.cell_type === 'code') {\n",
       "            for (var j = 0; j < cell.output_area.outputs.length; j++) {\n",
       "                var data = cell.output_area.outputs[j];\n",
       "                if (data.data) {\n",
       "                    // IPython >= 3 moved mimebundle to data attribute of output\n",
       "                    data = data.data;\n",
       "                }\n",
       "                if (data['text/html'] === html_output) {\n",
       "                    return [cell, data, j];\n",
       "                }\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "};\n",
       "\n",
       "// Register the function which deals with the matplotlib target/channel.\n",
       "// The kernel may be null if the page has been refreshed.\n",
       "if (IPython.notebook.kernel !== null) {\n",
       "    IPython.notebook.kernel.comm_manager.register_target(\n",
       "        'matplotlib',\n",
       "        mpl.mpl_figure_comm\n",
       "    );\n",
       "}\n"
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<img src=\"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAApQAAAFKCAYAAACjE8/+AAAgAElEQVR4XuxdB7QURdMtyTnnoOSco+QkURFUgoooKCpmRTGQMwoGTCiKgAqImD+UnCTnnB85p0fO8T+399W+2mZ2Z3Z3dnnyd53jkbfb09Nd0ztdXXXr1l03b968SUaMBowGjAaMBowGjAaMBowGjAZC1MBdxqAMUXPmMqMBowGjAaMBowGjAaMBowGlAWNQmoVgNGA0YDRgNGA0YDRgNGA0EJYGjEEZlvrMxUYDRgNGA0YDRgNGA0YDRgPGoDRrwGjAaMBowGjAaMBowGjAaCAsDRiDMiz1mYuNBowGjAaMBowGjAaMBowGjEFp1oDRgNGA0YDRgNGA0YDRgNFAWBowBmVY6jMXGw0YDRgNGA0YDRgNGA0YDRiD0qwBowGjAaMBowGjAaMBowGjgbA0YAzKsNRnLjYaMBowGjAaMBowGjAaMBowBqVZA0YDRgNGA0YDRgNGA0YDRgNhacAYlGGpz1xsNGA0YDRgNGA0YDRgNGA0YAxKswaMBowGjAaMBowGjAaMBowGwtKAMSjDUp+52GjAaMBowGjAaMBowGjAaMAYlGYNGA0YDRgNGA0YDRgNGA0YDYSlAWNQhqU+c7HRgNGA0YDRgNGA0YDRgNGAMSjNGjAaMBowGjAaMBowGjAaMBoISwPGoAxLfeZiowGjAaMBowGjAaMBowGjAWNQmjVgNGA0YDRgNGA0YDRgNGA0EJYGjEEZlvrMxUYDRgNGA0YDRgNGA0YDRgPGoDRrwGjAaMBowGjAaMBowGjAaCAsDRiDMiz1mYuNBowGjAaMBowGjAaMBowGjEFp1oDRgNGA0YDRgNGA0YDRgNFAWBowBmVY6jMXGw0YDRgNGA0YDRgNGA0YDRiD0qwBowGjAaMBowGjAaMBowGjgbA0YAzKsNRnLjYaMBowGjAaMBowGjAaMBowBqVZA0YDRgNGA0YDRgNGA0YDRgNhacAYlGGpz1xsNGA0YDRgNGA0YDRgNGA0YAxKswaMBowGjAaMBowGjAaMBowGwtKAMSjDUp+52GjAaMBowGjAaMBowGjAaMAYlGYNGA0YDRgNGA0YDRgNGA0YDYSlAWNQhqU+c7HRgNGA0YDRgNGA0YDRgNGAMSjNGjAaMBowGjAaMBowGjAaMBoISwPGoAxLfeZiowGjAaMBowGjAaMBowGjAWNQmjVgNGA0YDRgNGA0YDRgNGA0EJYGjEEZlvrMxUYDRgNGA0YDRgNGA0YDRgPGoDRrwGjAaMBowGjAaMBowGjAaCAsDRiDMiz1mYuNBowGjAaMBowGjAaMBowGjEFp1oDRgNGA0YDRgNGA0YDRgNFAWBowBmVY6ou/+NKlS7R+/XrKmjUrJUmSxKVeTTdGA0YDRgNGA0YDRgMJVQPXrl2jY8eOUenSpSlFihQJdZhRGZcxKF1S8/Lly6lKlSou9Wa6MRowGjAaMBowGjAa+K9oYNmyZVS5cuX/ynAjMk5jULqk1t27d1P+/PkJiypnzpwu9Wq6MRowGjAaMBowGjAaSKgaOHTokHIm7dq1i/Lly5dQhxmVcRmD0iU179+/n/LmzUv79u2jPHnyuNSr6cZowGjAaMBowGjAaCChasDs/fFPxhiULq1Ss6hcUqTpxmjAaMBowGjAaOA/ogGz9xuD0vWlahaV6yo1HRoNGA0YDRgNGA0kaA2Yvd8YlK4vULOoXFep6dBowGjAaMBowGggQWvA7P3GoHR9gZpF5bpKTYdGA0YDRgNGA0YDCVoDZu83BqXrC9QsKtdVajo0GjAaMBowGjAaSNAaMHu/MShdX6BmUbmuUtOh0YDRgNGA0YDRQILWgNn7jUHp+gI1i8p1lZoOjQaMBowGjAaMBhK0BszebwxK1xeoWVSuq9R0aDRgNGA0YDRgNJCgNWD2fmNQur5AzaJyXaWmQ6MBowGjAaMBo4EErQGz9xuD0vUFahaV6yo1HRoNGA0YDRgNGA0kaA2Yvd8YlK4vULOoXFep6dBowGjAaMBowGggQWvA7P3GoHR9gZpF5bpKTYdGA0YDRgNGA0YDCVoDZu83BqXrC9QsKtdVajo0GjAaMBowGjAaSNAaMHu/MShdX6BmUbmuUtOh0YDRgNGA0YDRQILWgNn771CDcsCAATRz5kyaO3eud4YdO3akMWPG+CzI3LlzExYB5MaNG9S3b18aOXIknTx5kmrWrEnDhw+nQoUKBbWIzaIKSl2msdGA0YDRgNGA0cB/XgNm778DDcphw4ZRly5dqHbt2j4GZcWKFalJkyb0yiuveGedOHFiypo1q/obxiQMyNGjRxMMzbfffpt27txJGzdupGTJkjle7GZROVaVaWg0YDRgNGA0YDRwR2jA7P13kEF54MAB6tSpE82fP5/y5s1L2bNn9xqU169fpzRp0tBPP/1ELVu2vGXxXrlyhbJkyUJDhgyhzp07q+9PnTpFuXLlolGjRtGjjz7qeMGbReVYVaah0YDRgNGA0YDRwB2hAbP330EG5aRJk5TBiHB3v379aPfu3V6DcsuWLVS8eHHaunUrFSlS5JbFu2zZMqpateot3yPsXaZMGeW5dCpmUTnVlGlnNGA0YDRgNGA0cGdowOz9d5BBKZdkhw4dfAzKX375hdq2bau8j1OmTKFEiRJRs2bNlPGZPn16+v333+mRRx6hCxcuUMqUKb1dtWnTRn32999/O17x0VpU+05coHX7T1PjktkpSeJEanwXr1ynSesO0pmLVy3Hm+iuu6hRyeyUJ2MqR/NZtfckrdpzUrWtWzQrFcqW9pbr5mw9SjuOnqMkie6iJqVyUo70KRz1HajRhgOnKfb8FapTxANHCEVu3rxJf687REfOXKLkSRNTs1I5KHOa5LR670laGTcnq37vyZyaGpbITlev36B/1h2i4+cuU9oUSej+MrkoTfIktGzXCUp0F1GlfJlCGZbPNSfOX6H5McfovuLZKXXyJLR4RyxtPHia8Jzw2d2ZrZ+TfM7pUiSlFuVzUfIkiWnW5iO06/h5v+Mqf3cGqnhP+OMONPHTF69655QiaWKfpucvX6OZm49QjUJZKEua5N7vpm88THtPXPD+nT5lUmpeNhfx9XgWk9cfomNnL1ve+q677qIGxbJRviyp6eCpizRt42G6fuOmt23tIlmpSPa0dPTsJZq64TBduXaD5HOesuEwVbonI+XKkJJ2HjtHc7YeI6yfQFI8Zzo1D102HzpDh09fonrFsnm/wnP+Z/0hunz1uvczfhZnLl2lv9ceogtXrlHWtMnp/tI5KXGiu9Qc9p+8SMmSJKJmpXP66AudXL52Xa1P9K1L+bszUsV7MhKexd/rDqr3AkvJXOmpWsHM3r/3n7xA0zceoRtivtULZqESudKptQ+9Q193Z0qlfhfQNeT0has0d9tRql8sG6VNkfSW54z7nr10jTKnSUb3l86l5oH1mTN9StU3fpeYI/fdqGQO9ZvD84Fu5Dtq5Z4Tqn+s3bOXrtL/1vrOiW/Oz5n/3nr4LB04dYHqF8uu7jN902GlF4zBiazZd4ouXL5G1cVzxrOduuEQXbtxkwpkTa36xrOAnmLPXaFUyZJQi3K51O954fbjhPWA33OD4tnUmmOR7yF93HJs6HvaxiPe9Yk54TeO9773WVy8Sv9uO6be0XgfBBIn71aMO1WyxIR1ZPWc8duYvP4wHTp9Uf1GsWYzpk5G6/afolMXrhLmo8uWw2do34mLdF/xbEp3/G7ldngfPFg2FyXCy5WIzl2+Rn+vPaj+byVYT01L5VS/GSl4zlh7+F3gvf1AGc+ziIZEa++PxlzCvcddN+3eoOHeIYrX6wZl7969aeDAgTRo0CBlSMbExNBbb72lQuOzZ8+m8ePHU/v27QmhcRibLE8++SQdPHhQJfj4kzNnzhD+Yzl06BBVqVKF9u3bR3ny5HF91jdu3FQ/unzv/qP67t+yFLW/9x7176//3UHvT9kS8J4FsqSmqa/XVi/4QIKNrvKAmXT52g3VLHeGlLTw3fo+l+Dl1njYPO9neKGN6VglrDlfunpd3fcsjI8utS2NWCc3wMbUeexKb9Oc6VNQi3K5acS8HWRjK9DvL1ZXRnLXX9d5r3+1fiF6qno+qjpoFmFPXdrtPsqU2jm21mrMb/2yln5duZ9eqV+IHqtyN9UaMsdrCJXKnY7+fqWW5VS/mruDPpga/5y7NCxCVfJnoke/WRJQNXjmi9+trwzrSEm/SZto1MJdVDZPevrr5Zo+t/l4xjb6bFYMpUyamH57oboyLGBEP/btrePu3qw4PVu7gLr+z9UH6PWf1wQcctHsaWnaG7Xp0W8W05KdHgOEJVva5LS0WwN6cdwqgvHI8tsL1SjmyDl69/f1dG+BTDThuWrU4KO5tOOYf6Ocr8UamNe1HuXNFG/0X7t+gwp1n6KaTHjuXrq3gMdw4+csx4RnsfS9BvT57O1KXyzD2pZTm2S7kUu9nzUtlYO+eqKiz5zGL91L3f5Yb6mT5Oi7WwP6aPo2+nHJHp822K8XvdtAHfyw+TYZNo92aoeQDKmS0vLu99HrE9YoQ1jqiw8kff63kcYs2k3P1ylA7zUtbvmc+cMhrcpQsRxp6cEvFqqPXq5XiH5atlcdGuP7rk67j5+nN39ZS5XzZaRfOldXX8HwevirRZT4rrvU8x05f5e61kqyp8Nzvk99hfdklUEz6fi5KzTp5Zq09chZ9RxqFspCYztVtbxefggDv2L/mXTp2nX696163sNd+++W0vyY496mf79Sk9YfOE3v/R7/LDpUz0ePVslLTT+d733XYP5470JgFFceOJMuXfW8W5MmvouGtipLLcvnvmVcPy7eTT3/2qje151rF6Cv/t1BV6/fpB+fqUK1CnsMN/7NPV+7AL3XzPdZyA7xvCsOmKEM/Rlv1KbC2W91EOw4do7u+/hfSpEkMa3seR8NnrxFraHOdQrSu02Lqe7mbDlKHccs93b9aOW81OfBklRpwExlAOrvbRwkS/aeptrjd78n9jx1mbj2lrn2eqAEPV0zv/r8w2lb6Ys52wM+pyYlc9DX7X1/Fxhrzz83eK97474i9Np9hW2ftxsNjEEZr8U72qCErXz69GnKkCGDd8ZLliyhatWqEf6PhdCqVStLD+Xly5fpr7/+8rve+vTpoxJ6dHHboJy4fB99MnObOvHjBZn/vcnqltLwwA8JPyhs2DCgdOGNo8f9xalTLc9m7U9wssYLUcraXo0ofar4E/DsLUfo6TErvE2sjM5gf6jyviPaV6TGJXME24Vq3+PP9TR2yV510r52/SZdue55eUMyp05G8ILpAi8ZTs8fPFKath89R9/Oj9/oHy6fm56plZ/u/2yBuux/L9egMnni11Mog2z99SJavvuk8sQ+We0eeub7eF3idL2+T2PLbnlu/CX0XuGejDRp7UFKnSwxZU/n/9mP7liZ6hWN956FMu5A1xTvOZUuxnnixj5TlWoWjvfivTRulddAwThhNExcsU8ZJrxmD52+pK5/qto91LdFKXWrz2fF0EcztqlNNU8GX+8SNrCjZy+reW/s14RqfjBbefaypElGSRMnIvQH2dK/CT08fBFtOhR/+MPvAAblzyv2UY50KWjRu/WpUPfJBOcmjJPUyW71bMCTtzvW402VRiP+hlekQv8Z6jsYTW81Lqr+3eqrRbRiz0mCoZYhZVLv9b92rkZDp22lpbviDWAYIxj7h9O3edWcLkUSWtOrkdd7gy+GTN1Cw+fuIBiPeP4Q+FTZQ/3Ts/fSwMmbaMOBM+rggz543BOfr6YOIN/O20kDJ29W1+KgefXGDeVFgsx5qy49OWqp92989tlj5ZUXCfLsDytoxqYj1KhEdvrmyUo+S+LVn1YrLyLLC3ULKoPytQm+h4IUSROp3yZ+c4MfLq0MyhHzdlLa5Elofd/GyiiEMQlPIQS/ERzALly57vN8sF74OW8d0ER56+FZK9tvurru6ycq0OZDZ+nTWTHKqzj7zbq2y/7AqYtU4/3Zqt2oDpWUJxJe79J9pqn7s3zxeHkVKfpm3k6f54WIBgxfGPDsLF/bu5F67+DdAqMNgnV7Pq6/j1qXpUcq+johcGjA4UEXHHC7NPKsrxfHrVQeQ3j1P3+svPoM3vznflyhDjXvNPEYgrHnLlPFAR7nyJePV6D7y+S8pV/sM2//5jlI48DU/c/1yoBGW1wDGT53Ow2ZutV7bdX8meijNmWp5gdz1Gc4FEnjGL/xt+MO572bl6CT56/QZ7O3e9cunBcw/LFG53atp9Zrl4lr6PdVByzfZ9weXvN5b9fzmYO+9mDsvv9IGdvn7UYDY1DGa/GONiitFsv58+dVos6vv/6qPJXAUG7fvp0KFizobQ4MZdmyZenLL7/0u96i5aH8ZcU+5TGDofFv13rejUt6Bd/5dZ3aHK1e8pgA/0jRx9y36gb0VCHsIb0kuB6n8VK503t1MWX9IXph3Crv39jwt/Zv4g3FhPIjlX1ik4HnLhSB52XL4bOEUzvCadgAseHDePuyXQUVvtYFBjQMWpxqtx87pww0FniJOtXKT498tVh9NKZjZaobpmHW+JN5ynNyT+ZUyss84B/P5g6BB2znoGaWuuz6y1r6ZeV+FYaUoWJchxd2xxqeU76U2kPmqLZdGxell+oFR4UVjP7bfL2Ylu32GEjwGv7zak0vJKPT98tp5uaj3u5gnMC7AyOIPV2dvl+hwuJtK+WlD1p5NgJ43eF9R1j61xc8nisWhE2f/3GlChNvH9iUKg+cpUK17z9cmorkSKuMSMjKHvcp42RPnDGIz7DZbDtyllbtPUUwbpZ1v4/K9PEYIT8/dy9VjfMwyvvByCnQzXOYY0ODv8c86n3ooSqDNwdeHUjDj/+lmKPn1MbeuU4BKtFrmjKaP2lblj6cto1gvLDUKJRZhbf/WnPQ5/lOe702Fc0R71FirxTaj+t0r/f6SgNmqM2574MlafCUzcoLhvu0KJvbayx/076iCv3WHTpXRQIeqZBHGQQIjRfvNVX1hcPcC2NXeo0hfPZh67LUKs7geWrUMhVmhWEKA1VKh9HLaO7WY96P8BuGQdn7fxu9n8GAH9WhMnX9dS1tPHiGXmtQmHbHnlfzhqzr04hmbz5q6ZnGs178Xn3KltZzcAKE5ZGvPM95Vc+GyiCRzwLz33LorDJW82ZKSfPf9o20+Aw+7g9AH+p/5DH6Bj1Umh6vejfBe9cg7jO+ZkDLUirUO3HFfiqZK506sMgICK5jgxBGPuAGgBLxupz+Rm16c+Ja9TsomzcD/fVSDZ/h8O+BP8R7Af0D4vFdh8rq4+d/XKHC4s1K56Dh7Tweu5Hzd3rfJ/Pf9njSAW9go6/nAyXomThvoLxh77820PeLPV5tvO+7/7mB1u47paANeF4QdlzwdXi2WD982MazfKNhfK4CH5zRvl+LkrTz2Hl1iKxVOAv9+ExVD0Tkw7nqN4H3IKJubBjiHYGDjBTeIwCz2jqgqfrts+gRBl7bVs/Y7c+MQRmv0TvaoHz88ccpNjaWpk3zuN0h4KisV68ebdq0iQoUKEDZsmWjjz/+mJ555hn1fULL8l6yM9Yb0hzfqSo9HhcSkz+Y1yespj/XHKQHyuSkL+JOk/JHA9wSfrg4YcsTrtUPCx4G/KhhJOJkjv9w0gdOksUqFKl7MYP90crTb6jGD06wZftOVy9ebJ4wKHFiBz4RIS/GnOpj440QGyBC3mwYoR0M9041C9AT33lCkfopPNh5on21wbOUZwUvxNYV89CE5fvUvxn/t7FvY0v8D79sgdWCtwMbMiRZYk+oE3gmXWAcINwLvBMM6kjJA5/PV14xlq+fqEhNSnm8zG1GLFYYVHjVAKXAhgDvFIQ325fHr1LYV8zt00c9GwlvcrwBybHP23aMnhy1TH20bUBTddCC1/LTR8sRcI6NPvFAMrCpPjR8kTI2cZhAmwp3Z1CGHkKAEGzu3B7/Bu7SSkr0mqp+Q/AGwSvEIg0FGBpPxEFREN7E+uMDUqNP/qVtR87Rqw0K0xezY5TRBsNw4fZYQngeBiUME3j2xi7eo4y+gQ+VonZVPdAWCEKs8IABkzbyKc9GD+GQf72iWRUWFMIHQegGhyqEoA+cvKg8dvCQwRuZLc6rzWvyiXvvVh5+KWxY4bPHvllCi3fGUpHsaWj6G3V82j00fCGt3uvxKkJwGIMxPGxmjGqPEHm5vBnUOn16zHKaveWoMu5hBLK3Fgb0C+NWKuOjRE6PocYCfPHIp+K9ohJ6w8aTfBYY99bDZ5ShBEN2SbcGls9Vfrjp4Blq9pknQoPnBFiJ952YOJEyTAGNeKtREVq7/7Ty1raplEf9nmVIHF6+1iMW0ZEzl4kjQzDEYZBDsGb/XHNAefCwLtf3aeRziGz++QJlbMJzXbtIFuVphVdbzoONTmBcv43zFkuDkscvPaPP1S5A3SzC4+xNx9jGP1uVevy5QT0DeCF/jjs4PDNmOc3aEn8wRDQM3lXek+RvVxrm6PPNhkUUxOKP1Qd83kWAwgASA9twZY+G6r6AWzxcITd93Kacz/MCDvSBzz2RIsCw2EMPmAJC63jvI+IBA7VluVw0LO49YvvQw2xgDMp4Bd7RBiWSah588EHq37+/ogDatm0bvfTSSyrkPW7cOKWF7t2704gRIxRNUL58+ahr164qsWf9+vUJgodSni5xAsRmAOlUMz/1eKCE+jeHPgKdyrr8vIZ+X31AJX3Il7L+WxqzcBf1mbRJ/VgBK0UoDC9EYK9+Xr6PsGHCyH3nN18cVzi4R4yBvaz499M18lOv5p65BSPSyIBnyilm8N3f1imjDqdxvHzh0eOQFTwxz9YqoDydEIn3CWZssi0bJvgMeoanCl4KeAQgMA6twtfskcAGhiQL9vzIsJQ+JhguCKPmy5xKhZUiJfU/nOuDyevTvAR1iPOYsgcYnpEfF+/xwhCwka7u1VCFqBlviKSDEe09RgN7ZOVnPH4YqDBUIfBqVeg3QxmpMGRL50nvDVtOea2W8gphk8GhYsH24wq7BjwaCw4fz/3owd0u69bAa2TpugJWjL2gjwoPusSWyQNHkR5TFF5xeLsKKsGGN+RqBTIrowyC3xZ7qHlcMIoR4oWBom+s/DvWnzl7j7gPeLQ29W1CKZMlJn423ZoVU4cQeAP169kg5fWI62HkwiCSz5IND3wHz64U3UuEecKghFdKxzCyYYwDG0LeHJbH4bXzWE/0A57gL2ZvV55kCB8S+Z7y3Tj19VpULEc6lQDEEBL8VpEUAi8i4C4reza0Xf7AbuIAAuGwKTy+I/7dqWBGgEPA+MX7FyFvHD5hpJXOnZ5e+Wm1ug7z/um5e71GM2AzH7ctpyIfaAOv+Jb+TRVOlO8Fz6tMGuLDCB9QFu04To9/6znU8ruNjXIcIkbHYdjlwTxX+hQ0/536KvrChhg8f6/fV1gdTJBIkzp5YurXohS1HbHYG4LHbwiGHdY65jXpFQ8mmn/HBbOmVkY1YEUwKDlaJfHTwHoD883ybK38ykCFQYqDOw5ZEDz7unHe/Zld6tDQaVuU1xXvuCGtyvo8LwlnkLATeYiAAYznE+idaLsIgmxgDMp4hd3RBiWm+dtvv6nEHFAIAUsJryWyvFOk8IRNkJDTrVs3RWx+8eJFRYyOUDeMy2AkUosKXquiPaaozVJuRDi5AhAN4ZCi/KHqY2c8WqFsaQg/XH/y8fStCudSJk969fLExgdcG4dDcBqGx6jXXxspY6qkdPKCJ7Mc3lOZFRmM7tCWvVj4d6inS07+AC5s9lv2eCke4ycztilDHR4RhLfgRQPmCi9A6AG4U3gHIXYeXrt5I6O1cFwCh2zbrurdNC4OM4Xng+ekC3tSER56s1ERZTQBhzWuU1XLzGNcL40deEH0zFy78Tr9vuqgmcr4YEGYF542COMbYZgAUwhPJESC69kgAjTh+6c9CV6MvXyofG76pK2vt2L9/tPU/AuPtwIYyOpxuDdcWy5PBi+ODmFZNjyBb7QC/CMRiDGFjMWzmnedoXNU6FwPG/6+ar832QBh1ofK5yEkmRXr6Qkj8/Nhjyu8/zA0IbPerHNLOBVwgZmbjirsNGARgLqwsMcZIWiEoll+WLxb/SZZZIiXPYcv1i2ovF4wVCVWFde89/s6+mnZPu/1MCxh+MBweK9pMXo+Loz/4BcLlCFlBXNhIwiGzMHTl5SnGGFR5ZUSWDzc5NOZMWp++B46ZfytfNfAg4qwNg4b8MzNf6eeOnywnLpwhcr182BXkWiFxCEY4mgPQaQD8BcYcsDprfODTZbPWkaEGFbECTmAY5y6eEUZPNA/Qt7wOOM+OCxhnQN2wB7sj6ZvVclXnDjGCVVsjCNJp3Qc1OKHp6t4s6SR5FW4xxTlcRv5ZCW6r0R2H2wot4WHHodo6cHXk1qQxIOM7NZfew5fMLjwTpM48cLZ0iiPPcvQVmWUQcnvQcaeIvoDBgE8S2RrQ/q3KKmShyDAiQIvCuHfCvcJ3e08fk5hx2VCl34oGDp16y1GJ/eBnAhAU+C5lzAMTsgBVrlh8ewKFmR1CJXP2c1/R2rvd3OM0errjjIoo6U0q/tEclExDg4veM4QBEYHIR0Iv/CkkamPkU/HCI9u7t/EB38i23b/Y70ybPAyxYsPp/sq+TJ5w8DYJHDixgYMows4GIQB4VVBNnWoUmXgTJVkAbEKcTrpt93IJSp8iDDyULHZ2l3LL3oOyaI9sEo4TeNlCwwle2QREhzQ0qP3UEQmcMjrgf1D1jEEeCp4LHXhcCM2Lxg1CAHBixAI03n0zCWqMmiW6oqTMkIZt901pXtPUy96Fml4l+s3XXlDsAlgTXGoGklQbSt7sLID/9mkNjkZYus4epkK38LYHhi31rn/mCNnqWFcWBsGGOO4gIFEohIb7UieeHm85zCAsCB7muV8GO8Gj8umfk38TpU9NAjfvdIgPoN01IJd1O/vTeo6bBBSwxIAACAASURBVMatK+VVv4t7B3v0zqFnGY7E59gAV/dsqGAaZ+LC7/AMbu7XhFbsPumFWSDzmmlSWCf6OpQeLPQtsW/sycI81+w9pcLIehbsiH930GDBFIFQ/InzV5V3S86XMcq4x6Z+jRVdDgt7ZJE5j4x7hEThoQSuUn+GE5btVetd/ubQDxs3gIAgoQrwCDA3ABdbMKvvIQtGOe4JwUEChxGZcAQWBYSKgc1FKBTvPTuZu/UodRjtyWSGsQsPNxJa8LsFDhAhcUQzADlAyBuQBoYEYE0irAssO6h9QDMEbyvmAhgLPLXABcPDNysuQYihBvKQAlqeaoM9iUESv84HMz6sPTFyqfK4s0cU7TkLn+eJqBVC0fybQ6RCORU2H1VGNq87qRf0z2wSeIaL32ug6K2AAYbAgEb4HQLPozROsZ5PXbzqxRTjYIMoFw6PMCjZAGc8t/47Qb+ABiARC55TXfg3KPGafBjCmr07U2pLSIjdcw/n+0ju/eGM63ZcawxKl7QeyUX1+LdLaNEOT4iMRXoo2LsXiD5C4k8WvFPPLycle0AQasufObXKspUZi8COFc2eRoVRYVxevHJNhask3UuwKpUvK1wLkPs/r1pT5/jrG6f6Mn2nK+MWxpkMSdqNR89aR3sYbd8t2EV5MqZU4S3AACDhhlJk0oAcFzYODkv58zg+PHyhCv/B0/R2XAan3dzwPYdq/SXuOOkjUBt4Dgp282RJs2Cj6X5/CcXrCEodeNqR8AHIBUL3h89cUkklnHXP3hwZNkMYDuEr7kuOAXysoFuCQF+cSMZZ+GzcYKPuH2fsQccwrvjgwv1VL5hZ/b7gWVv0nn+cHScZSCoV9MGecfyb154Vvk+ntOJwIhJLmCOVPZLAepbpM03pVGKYOTSt6wSGDTyELPJdwGFyJG+s2nNK6V7SjuEaTnLi62EAIjyOTGsYZm/GZRbX/2iu8txD4BkGhydEemThjUKSIIw4GJToQ2a/oz14bDvGGW5Wa8spRKNI9ykKQvFVuwrUtHROZQhxqBW/WyS/wSMLw3T7oGb0x+r96tAJ4xBRHxhgYD9gTKzUA5J8cFhh4w5e0OmbjqjwN5Kb4CXHvRnSoM9DrlEcEsGH+eWcHYpv848XPUk47Ax4rEpeGvywJxlNhsKXdW/gTUJiyAtj5fmAKemWGCYi1zYcDQzpgFMC3md4nvEe+X7Rbm+om69BBIRpp9izK3GYgCIwywc8gfDYsoB6DYcWHLCQBIrowg+L9xB+Y1g3WHsSZ4wDMd5PkD9fqqFC3ng+HWvko97NPRE4Kc/9sEI9AwnvYq85oAdYh7ifjHRYrS83P4vk3u/mOKPRlzEoXdJyJBeVxBfycGUWXMsvF97y4tenJcMrOq2LbMvGKV7GyOzWeQARVge9CUI5ANgDs4Uwhj+wtxP1SiA82vOpONC1ONliHJxoI1/CweI5kbTDHi6+J4dycI9nahbwntjxYhz/rCe7FkYsOPWs8I7+xg6cZIsvPbx8LDDY4T0p1XuawvbJhBbZ7v7P5qtNPliONc7M1cOkTp6NkzYyS5jDuexBk4cFgP1Bnm0lX87ZrrweHB5EG94oOLlAXicNKOmF5KQaTkSBYYUsXwgYDkCHgg1LCuMGAXmY/Jr/gwzrkTNS4UnKnjYF9Zm0UW1iEN4sdYwniKf1dcbZuYzhxfUy2YbZAIB5e/0+T/asv986DPfy/WcoTzCEPaX4N2eGw5MFw9XKCNK5ZYHtxMaNeUjjlL1k6BfGFrC8EOkJlx4sDn/rlGVW9GTymTjlti3fb7qC3fB8ZegeRjHmBZgFZNfgZl7aNYTWARdBiByy+/371f//WnPAh+YIxiJ4TOE53tCnsTK04GXkNYNr/K1rPBP2PsOLiTnjehCAI2wtn41kMmDPJoxgJO8w6TcnsDCkh9/V0kBlPD3rEu9oGGeSuokdBPjdLIg5rjyuUuDdRsIUBGOIGdhUGeXs5URCDFMrwamAebF83KasShBFKB7rGwluSMoC/hQGJQ78koZKhy2AlgiHSH/7Ca9lZhkAhAgJOfBWI0q2dt9pxe+qsyD4TNDlPyK597s81Ih3ZwxKl1QcyUXFiRVyqBJ/5i8Up0+NqUV074RsB540nEbfblJUhbpbxWFvuA08l8hGBf8awpP4N7LyQsU9ol9Um8BLm8WOhghVZtp/t8wHJ8NeImxgeOFxNQknj1fytKE9sKHgQgRuEtmwz9QqoIi5IZKomL25gQx0/f48dvk5vKAL3qlPvDlKfJDVs5HUNE7mx9yFePkjhOe2SC8Db7TsQZCGBnhUkTBjJRwOlphBTvKQGD6+Fh48GOAQrGcmNeZs31pDZqtQGzwkwPBBEDqG4YrQoxSmZJGHBasx8maN3wCSTEDSjM0alVKYfxG0PSDCRxUgeIWweW8f2EwZBcCfwcBg4U1ThsKl95MNBrm5csjZiglB0rTA2wNjAsKGiKSb0rk0pYcR1wC79/3i3cqQkFAaCU2RuOntR8/SfR97MuuBIeQEFdatvqbBSQgD2J8Egu/IayQ+F0lg7MVDG6xBJOUwGwKMMw6RwyjD82CSdTYoJXci+sAhDEYn81hyqJ7nhTaTX62lyPqthD3KMG5B9K2zcSBjHwky8NSv6dVQvbfgNUTCne4x54QjNm5xwIGxLBNn+NDDY8EB7ema+W5JosT38NijIg9Toslr4NllAa4YzB6A/eDdiN8RE/lLPk20x1rF7wsGHlgFzl26pryVcBIwZ6ikXpO/Y6xJYEAxJ+CvmUdT6nX0wl3Ud9Imr27kQQiOBEC0sDdZ0Vr5XWxhfhHJvT/MoUX9cmNQuqTySC4qK5oeiZHijRdZnM/VjufT1KfGGZoI5+bNmFLhn9pUzuvTjD07wLcBm4cKMVJAPwOvHZJ0cNLGixkvkHBOhDphLu63oW9jS85IfMeJQ/COggQcoPMWXyxQmKZAiUn+HjU4Bov2nOLN/IXRiBAf8HZ4ecNbyzghpu2Q1VFkyMluOaE8GGP6uC1nwLIRJLNqZX/8fbCha94kEcIDX5/bsjf2AtUe6gk/A/CPpA2mKZKhMngIUSbRSsYt3UPd/9igiKu56kn1wbNUcge8xe2r+SbJSf0D4wdoBoRDhGx4MS0PvgM28bdV+1XCAQR4Ykl8L/n8rMbImeg4zKHQALDGMNLxe2DKGPbEMX8sNuDVvTyJCko/faZ5cWs8L0knI0muZRIWjGZI3aFzFMTEik+QM6f13w8n7EjoilXFFNY3rsfmjGopwBFLvDbjYdEG3jt4iuElQrk+5oQE7ICr4/C8ObmE/4b3rmjPqd7kJF3f/n4DejvdwJbJfViDmw+f8YbogWPkyi26IcQGpV5xBThXeH35wMzeQzkOGfrXx8fPBJnYMBbh+ZPvqBW7T3gP7fDEIxPn8vUbKqyOAwEOBiwykoOscBzCQdMkD7n8jmf6HB2yI8eHdyz0ABgIfrNYy8Bk6roBLhLveE5chBdfx0xzv/JaMCbMizmukqSkAf7Hi9XVeoHoyWtDpm1VbBcSZiHHPHPTEer0wwp1UEOmPLDDjHnFgWHYzG2K+B+QBFTniYZEcu+PxvjdvIcxKF3SZiQXFWra6qdIacDZGSI8Rd4QQaGBkzl+5PCMMZ8XcG6oGILsQiQwIDGlWC/flz4+Q4IAwiTI+MZLD6FKgOlnBMge96dmgN1xGkfSEF5+qHYCAY+bv5rWb/+6Vp1EIUg0gfeAcTg6tYjTx4sQDhNNA38DqiDmngSFBd+PvacAl3P5SasSef7ua1U6j8PD7GkGx93L9W8tG8beIckL6GR+Vt4yJ9c5bSM3OuZB5AOPhCIEonL6beV+VX5PZorKZB4m1pZjKtx9sjoEgGaKyxhyJjtvrEhAgFGLDWjHoGYKgwrDB4kS4KMEXINFGk5Wc+eEBySNZUiVTK1ZjBdeVWzIEPamstdRxwIybAFtuXoReGKRwIPfHTzI2Ngh7AGXeDHOprdaA2wM6ZVEmEdRzsnqWTAmjyEYKMEIDlN5/5K9pnoxd8DgYfOGgKKHE5PgaePsa74nqgNVyudbT57fW2ijJ4g4LSDA+FP2aDGZPPrEGtxy6Iw6lEAwZ64Yoz9fNij1xClux1EdmQXO3+GgAnomK5GsGUhA0rOcZciXr+f3s15iUHrjgIMHCwIO0fLdywY21iSy59EXWCpkyVbcB5GlFT08tE8w7mHscjhZnwciPsNmbFPZ00w7J9+XaC8PK/ibK7mBp1NPhAOzASdYyYMhnvmH07cqPlt/sB54nJsM8/CE/tu1rjKoAcsCBRkMZI5UWZWAdfo+C7ZdJPf+YMdyu9sbg9KlJxDJRSXDhjxc6RVzamgwTk1OWVb9kKFfnO5wytP5BeFNg0GJMCLA4fBSgpwXJ3mUiQtG4Hl9Y+Iab4UJZCNyxQx5itX7ZLoMfI6wCLxaCD/CYwlvkFU1HLtxycQIJBXAc8seF7xEkSnKguxWJFhwXVqn4Tlcz3XXOfsRn7FXi6vNWNVJRjum7bAq1RZofst3n/DShnAZODt9BPO99LIgwx4bD2eeSm7QQJQ8oCF5afwqRUcDTwNET7jQx8ReErAOgGMVArwXqGV4jSAxAATmXNYPmyfqQmdLl1x5FRlDh2vtkp0YOoDfRerkSRRODAIPJehiIByKtkoywvcyJCuxvhjHuUtXvdydaMsVriQpPa8BYNUeruBbrg941eFzdiiWBFntRz4D9IuDJMLwstIIPmdvGkMwXpuwWv0eZWk/NuLRnrO58W8YnfD+wkOFjZ0TsfiZWVFhyRA93iOsT1wDY+GezNbebLkOOKmFaZAY1oM2WIMI3eLQCgEcghO59LXEBqXVOxJt2dCXBg0+t4PnyPA1KLswHh2ugIQ2Lmogx6W/VySsAAfuF8evVMZX/iypFUk9hI10rFHgZeGpRBia+Yu5f6uoCtOn6bqBNxthZngvOfu62afzfUjn4SUFRRMEv7nRHSqrA8TSnbHU9pslPl1K1gL8Hrmc8HdPVVLOCfTjr7iFrA8OqBF0goRJXrNMj2eHh9bnGM7fkdz7wxnX7bjWGJQuaT2SiwohWd1TKE9g7MmxMzR405ZThkflgbK5CLVcAe7m8AGHJ6XxhusAHsdmjBqy2EQeKJvTm60ZyGCwUjNvWLzJ4YXC2YP4d4Pi2S2fjqQuQRJDymRJlLfIDgMX6FFLMDsob1AZiCtm8MuZr4d3YPTC3SoLHCIzNO2WkzRKkHG5dOcJ+vvVmoqUmeldJMUK8KVIqAK9DtfLBpj+gTLxlVrs7ikpdpxu1HZ9yu9lyBYYQBjNHK5jfKw0FK36loTUKD2J2tmM0/LnrWLeQ/aKciYv+teTE6wqpQz4exONjHuGuMYOMsLGBjZPwCy41rScDyqrIHTZ668NKlFHp8CS9wQtDvrxJ0zhJYmrOXuds5qdPCfwJcoQtD/oA3uyOelIJ5aX5SeV4ZA8iZcqiqEOjPur2H+GF5+ItvCGwSsmBQcI5jN8qV5BFeZF5jWeo6IMEpyT/uYpvbgfti7jY8hiDcKrxxyXqMLDUQW9P3ivYWDzQUB+j3mu6d1IfS9pbtDGiuBdXisPSvAWAkeoQzhgyE9YvlfBh5B9zSK5XPGZrD4DAxIhbyTEsDGFNgxZgndz6sbDqiud2gefWSXo4X3GjAhyDgi7gykAlEiM39aZR5A4hux67A3fPVXZW23KKvlK3ycKdZvsLUoADyUiCoFw4jxHMCoga5yTfv5+pZa33rhVJScnv5VQ2kRy7w9lPLfzGmNQuqT9SC8qSdeBIUvcjL+ScPrUJHUQf4cXC8IOAItL4DSqjyAzVdZ4xTUAgOMlytgqGD+cIS3LYTlRK9PCwDB9p2lRRY/BxjEA3W0q+eI7uU+J48JpGKd7ZA/aGQSBxtR30kZlJEJQ7g7ZyFyfGS96vExZAGYf8M8mxbUHkSXH7OYtjQSUzgNoPX2qpOoyWVoR5QeRuFBp4Ew1P2wgwMqCRgZwBMANnMrRs5eoykAPFtYfx6XTvqzacY1dGI3gh5PZ2j8v36vA/PDirejhH78pa8gjhHj1xg1vfW1//JnsjYHhAOOOw14Yo06fIrn/eA66NyrQmsM1nBAA7zK8oEyfI3XCGelW3j20G7tkj8JwOmEyYG5OeAInPFeNpEEHD1C9YtkcPTZJX4MLAhU3wJpDtAEh0B5/rldlGNmgRQJH0R4esnZdGLPH7yW9ag7qresGIowXPpQhOx4GCWAn0uNmN0GJa0UNeJn0hIQUmVyCqAtHHfR+2bgfNHmzSuqQIg8FevKSZCWwGqv00HF5Vb18K7x0wGnuOXFBZfGz6F7oPbHnqc5QT814eLdfGrdazU+uJRTBACE5PInMPMDeYzk+Kw+gnpDE7ZF89fT3yxUHMvMNsyHPbYCRrVIgk/KISvL5g6cueosOoC1XCZJj4TF/+XgFFfIGtZrOCiDbM/sDqKiQ1CMrMTEPKSdR2a0fN76P9N7vxhij1YcxKF3SdKQXlZ69J38wHDIBz1/jkp76yVYiM+r4e3g6UeEEJz0WhI4RdsSmgoSLL+bEqLAh8FTA64AmB+EPZLgCu8TGSqAwtdV4YLDh5SENQTac/Z1Q9Rc69wvDEl5Vp+UW9fFIYmd4R0GHwsTUwMkhI5QFJMqvjF/lTa6QtXTtlhOyX+FNtTJCu/2xnoCxBE71uw6VFXE5c1OCmoRLr8mqGnb3w/dWBNBOrnPaRiagvFK/sMLSMY6PMWl2RgLCc7zZIwkAyTKcECbJneWYuDY2cIpIVJG4MJ3g2QpTxcYd92mHv4UX/+3f1ilcGsLGHOaWY+KwOf9edQJyeJtBIA7uQ7tDAePveOySgolroDt5RqhxjwojLE4zYBlTx3htST3m775ghpjYuZp6lsytyXAD/RpJQg6DBIYcShlKj6zd/CSuFUYpG1y4Dl5p+V5D1RiwQ1gJY2/1AzTa6ni+Yj2neAtM2OlSJqXxff1FX3BgwAGSQ/R69TFZVQbeVnh40T+ve3+JarK+O48Bxhs4daXoPKn8HSiZuv66Tv35S+dqVDlfJpJUV/JzXbf6nmPl0QVbA9rBWMVhFDj6QElZXBkOpTGxH8jMeT706Thiu3UUzveR3vvDGVu0rzUGpUsaj/SiYm8KZx1ymEMHNQeqmoKpIlkBCQStKuVRxgvCSwgzSZGZtvy59M7kTJdSvfgR3sRJF7gqdGG3Ict74FSOygsIR8kqO4yrYqoUeCyQGYksS2CQZEax7C/QidbJI5b8czBg8mZM5S3fp18vefbwXTAZ7mxoWFWCGDx5s+JM5E1Klk1kPjzcD9VgJEbOyfw4mSLcikZW92KcGJK7Xq5fSGHxgLMFVopxWZLaxKoPydGIDFZwW9b/6F/VdPabdaiAViUFn7Ongg1+GfoDQTJIpFlkNRH+TIeA2FUS4vZIrkACjcwQ5z6Zs5H5IhHK7dq4mJNHdEsbeOyQTMHhO1llKZjDmySXx02cJpGBbxHwBcbb6fRaVpPipA2uW4428rnIa2SyECilpmw4pJJ8rGii/CmQISRIsELmO9fGRntZ4hJ/g9+181hPzXZdwH4AKIAV569eaODeQbO8hiqq4nzzpKf2vJXI+tP8PRtlVu1fn7BaGUgQmbyCv2UFHVAVvTx+lYqcMH5d3ktW3sKBBMk7UqwOaYu2H6fHR3rqhUvBGubfEidL8ruK2zH/q36tvvaskjclNhweSsACJPm53ic/I8CdsHehEhMf3Dgxza5Igd8HFsIXkd77QxjSbbvEGJQuqT7Siwo/TLw8gLmDJ4s3bKfE0TxNtD9z8Rohs1Qn2OY2VnyFzJeGEyZCLHhBcfkrxrIhVNyu6j2ONCo5+SQnHictcGYp/83kzpIwGpmEDErHiR0bSKgi+0U2aJoUSfyG92QyAu4niYXt7s+Ghl45BNfpgHL2iOE7ZPXiuUMkx6Dd/fh7zspknkSn1zlpx7RPCP/BY42sS/ZKsZfLDt+KGuoN4gxIhPcBvmfv7JL3GlCO9CluGQpTxDAlidysdCoqNnRkJzLMjs+tqHRke7vqLmjLZTE5mS0cGMaYhbt8Eg5k+HDq67UU7tapSEyjVSlLq344Yxb4yP+9XPMW/KDVNeDo/LhNOXpz4lqVpAPxd5iQyVzAWGZKlYx2xZ5XWEKnPLKSEB+wGcZgW86nTVlvIp3+PShusqVLQWzQsTGKMPW63o1UEhaLxHAjgQ+hdn+C9zZwr2AjYAn07PhgizUNSjSZYCiTM2EQwkOJTG7+reHwzYTj8II+8/0KdUtmz4CjANEoq77RzgoShc9xmGcjl+nc9N8X689KD8yvi++s6Hw4kQpUdUOnbVPlZANVO+PDIqAu0Cpohvh9yvsU749Ofx/htIv03h/O2KJ9rTEoXdJ4tBbV76v2q5ci06volQYq3uNLzeFvelbhb27LvIjyWpQse+PntYreA+XWkIkH8vMX6xYipkKR9VXt1LrtyFlqFFeLWfITSt620R2rePvmMDB7NYDFQcWar+fuUEkQqOoTjgCnCHxjzvQp6bX7CisqDb2coL/+JZ7Vbgwc0rcyNPRwjSS0l15RSS1jdz/+PtQqO076BxnxF3O2K8P6+doFlReIE2T0xA5//cnNEN6X81eu2Wam6wlj0nBh7kW+nxXxvr6ByjJ3VuOUhw5/8+DMXE4cGPJImVu4Xp3oFG34QMH4U2l0B+L0tOpfYrBlnfVAY5FJSFNfr+03OiD74PlLfKSemMTt8ZsDThMQGq4E5FQ33E56x5EQBeopfyIJ8PU2XI62848rVTILwsGJ7rqLKiJRsUZ+n+ZMVo4PA5W75YukRxOfBeKtBE612+8bFBUa17vmfvQyhfBQIjzMuET5TpVcoKhRD4w5vHgXLl+nYjnTWvIVS4ymnDA81KA7klAo5o3ldoESzOoMnaMMX4gVnIGpsOCQwLtEVj6yepajFuxSsBoYynjPAO6i879GinPXajzR2vuD/W3cjvbGoHRJ69FaVEyMDfAzyvXpp9ZgDCvJJSZD37KsI6tHZusitCmB00zdEQx9jqwYI19GbJwwbowTL5AIMf/t+sS4K1lRxaVHeEs3jO2x6z8YvA7X1cZpHJnbUhiLyC9DiecCBICTBfyFgAONs93IJarkYDDPyG7e/D3j2HAQwTi5RBsofF4Zv1pt0JLL0KpfGU5FPeAzF696GQeYCki/TlLw4DvG7+HfzGvJ11h55aQRi3bADQfycvvz4MhxYdPu36KUN9vYXxlNJ7rlwxMMgk39mviUblzarUFQJT8lptEpabg3wSFLapr9Vl1F0cLVcPyNnw+V8jDEtaedzDnYNqBbQmIOwr7wUg34Z7PfLoDLRhjfSuAVB8634+hlNGfrMQrkxZUJKXomtlXfD3w+X0VSWBivGexcfZ0H1RWOGxybMPRiBjbzqQEO2AjXIOf72NFi+atexOUzJf5RFmgABGRrHNWX1Zyaf76A1h/whNytDnYyeoKQN/D6euKS7Fc6FZInSazw7VyJiYuAAFMPD280JFp7fzTmEu49jEEZrgbjro/WomJqD4RiQHUhMzj94Vj8TVEm+shMQCujY/aWI95wEmeDM9lvp+9XKJ7GYOhz2HiSRNYYpzx9gnSdK4sgTINKFx9P36aoXuzA8G48Vjb+uC/O0uS/uayfXQYzt1fVQXpMVdg7K9oXrsKBCi7bBjb1ElvjeuZ3xL+DzabHNUzRgjF/0racG+rx9sFE88CTPVu7gNezCKaAF8euUglcdoasDt0AHuyFcatUNRvowkok7RS+l54wPcHAypMkOe3An7ixX5OAetl9/DzV/dCTZetP8Bvofn8Jb1nIYJJn9D65KgjWPqiUVu1F4tJi1Wxtr0ZedgAnD1NiGp3iaNnLiwMk1pwksPd3T67gIyvOMA2Rk3EG24Z/MzCqcJiRuFm9L3hmP5u93fIWDHdgOhyGLlg1lvXCBz9cWlW+CSTyPStLcQY7V5lcBRwmPJQIYUNQpxwHRi7GgEM6qgJJfkumtPJ3X1Q8Ktx9yi1fM2m5zGhHDXC+lxXmXnbyxMil6h0AscKOy+pPcCgAV4/yneA/tRJUx+EERf6eKzEh4RGJj+xwCVbHobSP1t4fytiifY0xKF3SeLQW1dytR72eG1BxwN2P+tuQYMNgkhNPcrShlN0rKAMmRAK2ubwW06yEYqywB0PnC2MMEbwyG/o0poJxlXswFOCGkCTw97pDZOVFdelReruRlTzwIdepxr/hlXyzURF6bcIaYg+S3f31MmM1CmXxuUS+pMHV1u7bpaquLQQZwag6AbHi9LO7txWnod01Tr/n5w+C8adr5vdiH4GrQpUM4G3twqw6JQ4SUPTKOfp49ExTmSCh10y3qrwh8W1sNAWas6Rf8tcONFzA+9b8wFOKMlCdZzv9SownDARkTbeLS5oIlvOVSdJxT6e157luNePRQM0kaW3QF7xToKlhQUZw60p5FZMB1/P2V0bPbv5Ovpek7Vx3WxLNyz5QQlXyjsrv/nm1pmJ2eHj4QlVNKZA3j5OVcL0TPlCJJwVkaF2InjN5AALuHCVcEQaHYC+YufmoF24Crz4y+89evuadZvdmxdWBL5Aw161VG6avwnfr95+m5l8sUM3sOB8lJ6wVdpwpppCM9dH0berA/fUTFRQXsJXI0D5//9sL1QhwL3nAgNc2GhKtvT8acwn3HsagDFeDcddHa1HJ0xkqtuw+fsFLwI0wBzCAToXD2FxVgHF2Vi9Jq/KP7OngF6ZeCxmEysiWzpg62S1DYo47HV8lDQFg2piSCB180rasykwHnoezwJ3ONZR2TEvD13ISEP6G0V0iVzoFfIcHacfAZpQIR/kAYgdPkBs2EoOQsbr3hAd7xDgm9TLv00hlvAcjEsgu6wMH04e/thwmRLi3Q/X8PgcckLUjmczJZiYr4xw/f4V6/rmBAmVr6tRA8pABb97Dwxd5xfu99gAAIABJREFUh+yPBYATykrmSkf/vForoDrkhu6vITzAnWrl93KzBsLL2elep1LC36hjzB5Lp4kruI/ENDo1cnW8thWGFNnVYI1gpghmepCHI6u643Zzd/q91BEnyzHJut4HvMc/LdunPtZLBTI/K78DA3nzJMWYEw+0zIj2l/HuZL76gRQGO1MM4cABFgJ5COMKatw3R5QC3YvxjGij17qX7ACSbcOq6o68hzz4Wb0HZD12hLy59K8/Wi3JdsD34UpM0qu/a/D9TtQadpto7f1hDzQKHRiD0iUlR2tRScMONXOBZWSqjEC1kq2mCa8QMleR1Y1EG9BSbDl0VpVT1MuyWeHHGB/G/Ilcvxn3Yk+jP0A+PFfwuCGUO7R1We/w5H1Q/7fV154QHwReg0nrDtK+ExcDEt+69EipxRcLfOg2ZHY3DIXdsee9oRcY96mSxWeCWo1BL52m1yqXnHXwNqO2N1f5QAZzzNFzqls7rJ/VvZ3yQYaiO2/JSOAnq+fzZpoieQh4ymNnL5OT0GDpPtMUfgoHh6NnLiu+RitCch7j4CmbVXUVFtRcH9LKs5Z0L4a/rFGu/eyE+kmvFINQH+YmWbcQpoPhwiE5wDRkhnAw+pVVRhByRl10eKVCCecxgwDu7xR/yTg5TvrQs+LRFwxokHdzvWxmbJC/Y7sKXsHoRG8r61tzBKFxyew0bWN8qVS+BgcO4O/wbgPJ+YXL17w0OVxrXHrLnq9T0HJoTNaPL50kyEm+TSsGDafzlyFpcNHCoGR+XGRf/7FqP/X8a6P3EMY8v9w/4wwD3Q/RLryHIJwdzu1lrXtJUQT6HhRp8CfSoLbCjrMRD88+qt5AAhH343dYuMcUn3A+R21kBI+rHznVb6jtorX3hzq+aF5nDEqXtB2tRSVf1AgpwqDkWqmheK6cTt8KkD+6Y2VF0MxVZuSm/NDwhbR67ymVgYjKJ7o3hTkE9XCYTJTACxCAexZQv8yLOaaIuoMtP+h0nrIdV/Lhz2BAg1exbrGs9F7T4qo6C4cAnYSh9cNAhlS+nltQOTGZNzbmR0UNXJAXI7wVincK42fMaig11+10JzPI21e7R5V/gyC5Bpg0VNhw8rzYWwiKJOgC9YcD8VcOm7nNuwHhflzPGf/WE278YbKY91TWyw40X+bzRBtk92NDB28eC7z0zcvkUvhPJLoh9BiMJ1HeW2I24YGBQQmC6VAyWH9cvFsZGxCnBxLGa8Obt3Pw/SR5UZlWB1CGeTHHvWUo2fspsd2ByqjarS277yXZN7fFwRN8lrrA8EFYmA1kcJ0W7+Wp/IPCAaiOVfOD2SpzOhC9lsTnOokKMUMG7lM1fyb6+flqdtOy/N4HFtKxMr06frU3pA3nAryv4C1l+iy93rYVmbl+Iw7543MZFcHfMlyNsTAcyS7hTlaksop+gb4O1D9cthX3Agl9rcJZ/eqJ3xXcgCsxSc+4XWnTkB6CxUXR2vvdGm8k+zEGpUvajdaikp6X+W/XU14yrv7gdKMIZcp6+Tb5EsZLDCTMqPP96wvVSdaORjsQXAOHBUHYFx4dGJRHz16+hcBWvuRhbH4uQPRM6o5+2KMQylycXtNh9DJFmsvCYRX+W84TzyJvplQBu5YbstXpWYZU4dEDhRILkg7AZWeXUelvAPAGwyvsNDzvVEdox54QhJXBQ8qbNCoKISEBgn/XKeJ/g0Cb2kPmqBB/rwdKKOJoZLUHSr4CnlZm7crEG51Q2p/Hg0v32WXAsj7kRoaxwVuIWuYswHHCU//u7+tVRZ2VPf2Xm7TTsYRIgPh7zf5TtjAAf30y1jCYkKuO15YYvYJZ06iyfyDc/9+ag9660Uy/g/AsuC/PX7muygQWypbWbrohfS8zn7kDHB4Yvyk7Bbn94p2xxBnAsigEV5/i5xuIB3HL4TPUZNh8xRG5sud9hEzjQCJxnlZ8qMFMPN+7/6jmMNIxR9ABQXCgHbNwt6LvAj8jYC0oYsEYbL6mQfHAJVtlAhGy84FXZ9FhI0xI/nSN/NSreQm/05CHGZ0kHhcxAwES91BGEcIGvr9OEb2BBx8iKzEt2RnrPYgzZ2Yw+g2lbbT2/lDGFu1rjEHpksajtaik1wKVFHYdO69wVRBk+oXqDbFTA0J7eNlKQdgIRLXsKQLG8O9XapFeDxeeqgp3ZyR4E3BqBoifgfxWdam5tivChwD3W4kTA85uTnbfSzA52urZ1dI74oRommEA/krRIUmkQLfJCkP0bK389O38XbcMMVRQvySRDjZD2E5PvAnDCEY5tPzvTVaXyJJtvFYC9cWYVfCbHjh5kcYtja8jbXUdcxDyd5IHVc9Y9VddCOsaDAZIAADjgJ1wRirawTjo26IkLYw5Tkt2xdLvqw4oYxJeKITrQdANup1QRWb1oooPvDgDJ29WHIWz3wyuX3iUABcpkj2tgrg4kcU7Yumxb5eopoB0wKBEvXkkoYHncO2+09SyfG6Fz2RDQEZJ4JGPPXeFGgUoB+tkHIHayLKi3A4Z2w3jOG7ltVwxhlkZ8HvjtcpeVAm7eKh8Hr+3hpcSh2S8/+xEZscz8bvdNf6+5zK7wKrCoOT3KMj/ccCSda11nla9lKPVPbg0LL7Tk5j02uJ8ALRKeJN961XIdGo7jgRJjGugakLom6nq8G+mlMO/ZRTI7fecv2cSrb0/1DUTzeuMQemStqO1qGS1DOB3EPJ+EfQqSRKpUFakRK8HjPtw+S72FCHUMvm1WlRt8CyfOsefPVZeZWWD5kKeeNEHPC+l8/iSkuN6hBH9geuxMfzxYg3bJJhwdSEzY9EXl2fjfiU4nI3mQPfUaVis2vKGxuE5vU2oFSAkZCFYNgA7PeplHTlTFIbhkKlb1eV2VWjQhmEQIKrff+IC/b76AAUKRUssG67XOQFRoQQGh1yrdnOx+16GEWWojzlDgT+ueHdG+mTmNnKS6BPoftIoHtOxssqs/WjGNuIkOruxhvu9THhBSHXW5qOWmfdcNSUcSpxwxsrJXOgD+EiEOkv0mupTnQbfAY+749h5Velr8XsN1C35Ws4q5jXjJHvb6Zilp9mOPsuuT5m49uqE1d454oD92awY+mXlfgKGdET7SsQk7dynk3cU4+FxDYovDJocz9upe/lRSWnckj0q+hCI/1hWmOLSjXKezJHLNGz4zm6sb/y8hv5YfUB1I2ExOHRxBTgnMCQ7fTv5Plp7v5Ox3O42xqB06QlEa1Hp1RLgsVSl7iJM5GrlCeBQlqzwghAYl8xjeiFUeUF4sLVIsGG1W1Un4U0b3rgzl66pE+ipC1dVwga8oOOeuTcoDr5QHzFT7fD1OibHjgZIv69eecRqXNUHz1IJDvBuITtal2BClvJafd0gLOaGSCwVc8FxlRgZwvJXPlGOQSb3oLIGCNFBBQMsrZVIzwe+1wm7eRz4zi0jmseIPjvWyEe9m5dUQ+MSk8ARl8+b0Sf0GI6eC3efrIwGGDzIpgY20Kp8XTj38HetjteeteWoT512vo4xhU6olyIxTlnajwse8MFM3o/5c2VRBD4MIUyOwwsiBJBRHSpR/WKBw8NO5yJD63b0WXZ98mENmGR4izkhDOt76LSt9M/6QwT6ro/blqMuP69RhzIWJwlETImEKEq3+4v7wG6sSr7Cy2sXFWOYFrLGV/Vq6FNOEmPjMDuyyKdsOKyGCw94mTz+31ED/9nkjeDIxE+5Zp28c+z07eT7aO39TsZyu9sYg9KlJxCtRaWHwXYdP0fv/LaenJJrhzpdq1KEHHYGlQ9OtqikgBczJwkBUwkMD0IZyBwE3Q84y7Yd8WQRQkDYrNPt8ImV28BTiSoXyEBvd+/dtpilUOeoXyd5OvGdDimQOrEK3ev9DZm6xdYg0KmK9D4CZT0Hmrfu7apbNJsrapKE5EyhwpUvsEFjg4M4wTNxGAvJNTCm58cc90m00Qc8beNhev7Hld6Pddyb5BGVON5wJi5xtVxfHv0xzAPhbhjrI4D/zJeJJnYOLQHDu/b7TFOHKoQbUW1l1MJd5CQjPZw58rU6Xnv2lqPU+38bFR8r4B8sMJi+X7yHyuVNr7gAoy2cSIP7AqcLj5lOmYPvAGlAElWhbGkIeGgI4wCh32alc1KxnnFJOp2qUnWNJzacebHR64Q+K9B9uHoXKNvAgcuCw33/vzcrPC8TyTM1G7dxcqjiQy/out5pWsznHlbeRac6mbhin8IUW2E4mXQfcBGsMYgdtZXETwMahX0HIrP+owGLwj2jtfc71fXtbGcMSpe0H61FhVqvqLYCAUExDEpJFeHSdCy70UlvkWWeLV0Kknx1eNF1GL1cXc+YmLJ5wVV3SmEDcbIGNyXwPTULZyFkHuqih8b9UQ9Fcq7o+6PpW71JQf6oWuQLvkW53AGHxLyJgeYjsyytOgsn3MljDVTWLFidSrJv9iow9QpniSIUiiQkO0/Gcz+soOmbjhDof3YeO68OI8j8xGHCSmSyA77XK8Awvx2+A9NAymSBkyeczP2lcau8RjKSh0DkDuHDAg5R8KzA8KteMLOqNx+OyDrHMCh/WraX7GhawrmfvFbPMkeSDkobhosNdWt83I98zhxSZnyfvBeXl5XUPbIUauOSOahcPw9DARNluzXWd35dR3+tPUC/PF/9FohPMPfgymE6Awa8j+Btlb8ZHcvuxGMH5grUKkdkoE6RbCqRjwWVr9IFyX/rZG5cQhW/l0U7YtUldlXfuOQm2spKTHq9exjYSRMnIlAeRUqitfdHavxu9msMSpe0Ga1FJYHkCMtg48VLHnVoUY82kiJDS7gPg55BqItqKchAxqbeeewqVV/2rUZFVXKCFObKRAjdX91kvBhRuo3FKaWL23OXdBf+qFpkQopdCbauv6xVGKcmJXPQ1+0rWg5XB9Jzdjc3Ln93BoUfDUXYk+O0lrOTe+hGB7w/XLsYUAVwhjpNJEIIDzyBwNvCo73p0BkKRDCtE23rdbM5e9SpQetkvvwM0Vby+n08fasq6wfvJDBdWL/sLXPSr782MoMeyR0KV1omp+VBLJz7WF0r8drwGAELh7CqLMHn9j1D6U8ewno3L0Eda+Snxp/MU1noVoIDLojMIQwxGdCylKpGxbRdjA8PZTz+rgFEJkXS8A41/A7WWSAwXjAWbDl8lt5qVIRerl9YUZyBeovFaZIKj1NWRwuXAiuQHvmQBpJ8VCmCIOEUTAL+ROIyJfWcJFwH7pidG25FKKzGE6293821GKm+jEHpkmajuagkkBwg82i95O8dNEvRubAwpnDW5iOqYgwEoaMuE9cqTOcHj5RRCUMsTj0bAHsDYM4STI1wlx6n6obriuPf/vBhsg4t6v8GfHGOX6WqWQTK9JQeMPSFjF4cGlhk+bNg58qGnsyGDrYPvf3Gg6e9VWGYk4+5Hbn2uQwxBrqfrAkOEncknPmrcIN+ZPk3/K1TE7Fx7i+rPpS5y+o8Eubw6cwYlYgDjC8MSnACuuFJZHoUVGaCgQ2MWSBcaShz8ndN7LnLVHGAh9kB+DnQXtlxg7p5f6d9yYxf5sZlbkOrPmRlF/ZkwhBtUCw71R7qKZkZSaojp/Oyasce1X4tSlKvOF5RtPvjxeqEJB0c4Nhz/s28HT5JNcGW65Q8u+DBRZJLJOS1CavprzUHVbIZ1jjELrwusZKyEpM/HmMn4f5Q5xbNvT/UMUbrOmNQuqTpaC4qGWaFsYGXPHCG/3u5pkuzse5GUqagBeMfJZksvF99Jm0iVBEZ+WRlb71XtHe6EXKSD4/i+ToFFJF4tIVrGeO+/rCLcsPX65/r45UlCge0LG05HTl3cEYihDd6oYebDRKO1+uJkUtpwfbjqs8+D3qSScKV5btPeJOtOCQmN3j0X7NQFhrbqartrXr9tYF+WLxHVWracuiM4ikFybm/cJXOd6pTAzHtU450KWhJN09Wb7jCJSzRj7yfTLhCTejfVu0nWaou1Puyl/WFugWVTuZsPUYocelv/YR6H6vrzl66SqX7TFdfgbYIIe9oJgU5ncsLY1d6kznYcEDYdsnOE5ZdSAwqwzOQ0QxccaM4uqFo4e+czpHbMQQCBy1EplhAs4Os7tjzV2jII2WoTeW8ykuOaA8kFC+9/H3Z1esOdh6yPbNpyMOzTtGm93/49CW6d/As9bGsxCQz6qXRrXMIhzNe/dpo7v1ujjsSfd1RBuWAAQNo5syZNHfuXK+uJk2aRP369aPNmzdTlixZqHXr1urvlCk9Na/37NlD+fLlu0W33377LXXq1MmxzqO5qDh7dUirMsqLI0nFHQ84hIYSq4RQbMzAZqoXyXEI7xcM3HyZU6nQbPm4qiloF4gsWA5Hz95FlvhL9QqFMOLwLpHjYI5NvUe54YO2JpC0GbGYEKaVBNxW7ffEnqdzl68RvAIg7+WwDdqCOPubJyuFNDHAEuAhbVkuFw17tDwBPgGxwzYGupkMPXG1CsZE8XV2lTS4HWO+wAiw+eAZVQUkEN5TJ9vXM0M5PB1qIpPVvCUMQmbNSuos1HiHx0UmC4T0wIhIHgJAqA0jCRyl3e/3TyQd6r3063S8NgzKkQt2UThecrfGJvthcnqEZRE1SZI4EfHhzep+dYtmpTEdq6ivZB3p2oWzeg/AVuwTkRh7sH1yiF5ScqEPJMQhYQy8lFwR57eV+xXNEwSMGxv7NQnqdpJnN1CBgaA6tWgMfOnPK/apKBA8jBDG5/vrG5Cpkr091FBMSo+2ksoN72MU3YBMe702Fc0RGXL9aO794eo60tffMQblsGHDqEuXLlS7dm2vQTl//nyqV68ewdBs1aoVbd++nZ577jlq0KABjR49WukWBmebNm1o586dPhtr+vTpvUank4cQzUXFPI3A/cBD6VYCgN08ZW1rVIlA5i5Ehh6RoYuMT5Slw4YLDweMI4jTMJKebNG/ZSkFvI62yCxiGSaT42BvnBOvH4ec7YiAZf86TjAcI+XNiWuV5wzlAb94rAK1/WYxnbt8XYXLQsV2Sfzs1jge1NcnrKY/18QT0jutQsNQB3jbNx48o2r1jmhfkZAsYSU62b4O5OfwtJve+zELdykPPAT13HNl8BxMuVY6YB3FcqalyesPB4Q2OF3LXPO+baW8ChOIMKRertRpX8G20/HaqBrFHmRs4glFmAMUh9i5XeupYUmvpT5OYCUBV4A0/3wBrT9wmpCxX6NQFq+3fW3vRo6I7qOtA8ZB4x0CiAULQv0d45IhgR2Et3XqhkMKzw4JpWrTyfNXvA6BQLjvcHXA3JcYIzysEMbaB+obCWrw2iPkjUMERLKggKIJuGaIXdZ4OHOI5t4fzjijce1/3qA8cOCA8iTCeMybNy9lz57da1A+8cQTdPToUZo+3RO2gYwdO5Y6duxI586do+TJk9OgQYNo4sSJtGZNPAVDKIqP5qKqM3QOgacPWJmdx8/R2CWBK4qEMh+ra9jDpr+gJL0IEiqQWMHJI+wBQNnEVT0aOiIj17FxTIzu1jyc9iMNW3+Z2eyNQ2bykFa38iUiux24JhhxDT76V9HhBMIF6mPTdeEUNmA1R5Ry5Czhro2LUeNh81QzmVziVDfcDnQgb2v1pd/7fZ3CELIg9PRktVujAPq9mCAbhsHu2Aue3+szVRUbgJXgoAL4B4sepuT65W5icLlP3FNSIbGheXemVMoTglKXMAI/aFUmWJX6tJeJSvidIekimh57SaQ9L+ZYHDY0O418KjQveVjK8HPxxOX76O3f1qkqTe8/4tG3zsEoL5VJTZzQ81K9glStQBZ64rulqmmweMNIzMuqT373o642yiyySBohLkuLDGcufRoKR6j0UCPhEIlAkRA+EDBvMe7hNIFIH4+kMQOm/bsFnmpjVgU03JpLNPd+t8YcqX7+8wYlPIw//fST8kIilL17926vQbl69WpKlCgRlS0bv9HDoGzfvj0dO3ZMhcAfffRR1Wb8+PFh6Tiai6rhx/8SkhZAp7Lj6Dmf6ghhTcLmYpmBDJ6yRXHVJmRmHfBvMMSYMoWpYBoUy0bfdajsaHgy1IIL+MTt6GIXG0l8IFef0Lvnig1Wmbcg/a4wYIYiZUdWKTgTkdQ08KFSqua1E5EVbtC+XdW7aeBDob3Y+cUN4xiE3Pd9/K8aQqgGCsJLz3y/nFbvPeXDMtB30kYf3GcgL6PUASdBMaE9vgtUslESRqOtHqaEhw00Ivkyp/Z6MJzoPFAb9lqDpBlGB8MFGK+G30WRHGlVDXg3sI7v/raOJixHgk92wlqAoS2TEMKdj931Eq8NAwXlJXE4Gt7OmqXArr9IfI/njPchnjMzR8iKL/o9UZHlk7bl1McShoKwLicXRrKMbTg6qP/RXBWVAmwGXKcsEi/I3jj5/nKaGCfHBr0W7j6Frt24SU6jDKHMDaU7YfgxrRP62Ni3MaVOniTo7mQBDlnKEVGY8nfbl8kM+oaGh9JHZf95g1LOpkOHDj4Gpb44rly5QlWrVlUG5MqVHkLkkiVLUs6cOeny5cu0bds2Kly4MPXs2ZMaN/aEc/3JmTNnCP+xHDp0iKpUqUL79u2jPHn814ANZcHq19z/2XwVEgSlCjZMt/BadmPj8BvaSZqiI2cueek2uF4uG5DAVyLs8MZ9hR3/oM9fvkYlhefJrgyX3bhD/V5mEjLuUO+Lq+mAlHeUZjCfvnCVyvbzeMfBvwkPIar9BMMDKbMW0c/TNfJTr+ah4ee4ugQwcEjKaTJsvhobavb2eCC4PkGUjopG2MghgF88EQdLYE5G1hWMaVC12AkT5Mt2dtU9uJIMrpF1pO3uFer3F69cp7d+Xatq08usfnh+8XxB7o962Uh+kpV0Qr0fh+2R2BRz9CwdORM4USnU+/i7TuK1cVBE6VR/vwW37x1Of2ykoA8kt8XBhVWX0nPMGFU8q0r3ZPLSnzF8I5wxROJadiZgvDJZT2IqZ79ZhwpkTUOSgUGWJwxmXMx7GUxUJZj+0Xbw5M0+xjE+06uSOe1TwjQ4WoZr3eYVleOJpjPJqR5uV7v/NwbltWvXqF27dvTnn3/SvHnzlGEJAzN16tRUvnx5Gjp0KKVJk0aFxD/99FOaMWOGwlr6kz59+lDfvn1v+ToaBuVDwxcqrxDCNDitRotK5JWfVtOktR5sHDCSU1+vrf4tDSeE/PaeuBAWVx5eCkV7TvXWYbbjJIvUjwdciOzF8xc2lUbahOd8q6LIpBFkK/f8a4PCBX7TviI18oML1OciQeb4Dtm+dsk//vTBGcrge4NB+eAXC1XTQDRG/vqSdZ45q5Tbfj4rRtWcZnFCqIy2TJAv72lHH1K69zSVvAOJGdhUkRjfDuHQP3BgBbOlUclXz9UuQN2ahcdOwMY5nhkowlDpBdRcD1eI7KGVdSjx2vNjjtG0jR7ieSt4x+3Qu797fjhtqzckLD3eaC89x5y8A4w2YDqgPENFHWAoE6IwhAiRinFL93qHiL3gyzk71N+cIY2ETfCYQvxhwO3myHtNIOiJXR9230vmBG6LQgigHQtFCnabrN6z8OrP3HxEdaEzQITSr79rjEEZr5n/Fwbl2bNnVeLNnDlz6JdffqHmzZt7NXD+/HlKkiSJwlOyNGniCWVNmTLF77q7nR7KtiMW09JdnmxPbDIoVwVqFRgtkRRJ6gxP5F9xNEWypjW/vFtXzEND/dRgdjJGpsdA20iS0gYaiyR29udtkokkOm2T9HCCR5CNrGBezlK3GKss9+dEj7LNsJnbaNjMGMWV2Kd5SWoVV1vdyrtq17fkHtVf/pyggj6wJ2wb0NRRyHnK+kP0guAtdfLsmVge4bLtgzysA7dD/li9n9742WOMgP6ED3zAqoYjbJyjuguiEQjpfdWuAjUtnTOcbh1fK/HaMChBWxQO7MLxjcNs+MXsGPpwuudQI7OH8bf8LXPUBfhLVDjiErLLukeGczHMaamoALgaYdRPXLHf2x2SAscs8tCL8ftSRo5CrTaGAy0MUxxowmGDCDRvnXcYbcOBHBTpMUX9TmTlnfEul9KU8zEGZbw27niDEqHopk2b0q5du5R3ElnfdtK1a1eaOnUqrV+/3q6p9/toLirGMiKjGgal29yC/iYta8PKOsXAChboNtnnsier3UP9WpRyrD+9oaQoCjX8EfLN4y6UWY7+MEQj/t2hqgFZYZRkpQnpUQgmhA9vLXTLITt4J+GlDEU46QWVTkDk/PhITwKCrByCv3HPIdO2Kk8zvGHJk9xa3YMpSayq4Ixbuoe6/+HhvwMf6dJuzjZnEGd3HOMp2wkBk8C63o0CJnJxvW7JOhCKbsK9BoloSKDBOAAHQeawGwTybJyz5x/jHN2hMtUr5k4tdrt5S7w2DMqF22NdCeXb3Tfc7+WhBtEUJDOxSF5bLiQALz3Cwn0nbSLoet7b9vtEuGMM5foHv1hA6/afJuBA/1h9wNuFNDCZD1ZmPIdDNxbKOIO5Ro9ohHs4LNlrKp2/cl2919bu81Te+fGZKlSrcNZghuW4bTT3fseDuk0N72iD8uTJk1S9enWFdYSBWLq0rwdv3bp16nt8V7NmPCk4jE5ki0+YMMHxY4nmour0/QrlykcYdsfR87Rs9wmKBvn3gL83KR46iH7i5VOh96VduwC9F0a47/Fvl6i6rijneLvwTNI7CA+jFXH5j4t3q1rqVlmUkrYDmNJZW44q9QTLiVai11S6cOW6ulbWj3a8OOMa8iYLgwc4TKYZQYnE+W/X93a389g5qv+RJ2GHOe30e3Ff92RORf/GUbVwGxm6DoayZ9GO4/T4tx4jF+IkTNfok39p25FzirMzUpU8nOh58vpDqipUiqSJVHKIW9nYbJyj8hTwtxBwDlYrmNnJsMJuw3ht0NQs3H48au+acAcuDzVItgEEgUXSLjHFFfB24A99f8oWKpwtDc3oUifcIUTk+pZfLlTUUUgCBGUXi/ybD+Ayaa1FuVz06aPlIzKmcDvlgy73g9/Qlv5NQ+62bN9+BeNYAAAgAElEQVTpChqC58gYb66gFHKnAS6M5t4fifG72ecdbVAiSQcZ4DAYixf3xTJlzZpVufBhUCLs/dVXX6ms72+++YaGDx9Oy5cvv8UADaT4aC4qeapGlvfa/acJnFtdGhV1c23c0pfEuuhl5Ur3mebd8HBhuN4ZniOSHG5X+EkCvP2B0n9duV/V0AUt0ppevrgrpjOBPkrmSqcSqSDBVuGo2H+Gl58tmAxx/QH+sHi3KtcG4xcGJbLOISmTJqZN/Rp7Q1rIuERSA8RfqP+j6Vvp89nbSUIf+H4ydC05/+wW56q9J+nh4Yu8zZxwe7LHJk/GlLTgnXij2O5ebn/P2d/wrsDIRuTgvabF6Pk6oXmTeXwcSpfjjWTGqq4XiddesD1WeXyixYMZzjOShxqJpUOf8nDIpOioagS6J4aE/P1KrXBuH7FruZACflegpmKpVzSrgiNAZLiYD/qSUiligwux42/n7aSBk+Or/oQbbeD3pYQ6jHyyEt1XInuIIwx8WTT3/ohMwMVO71iD8saNGyrh5tKl+NrTUm8IgaNCDuiD3n33XZo8eTKdOnWKKlSoQO+//z7VqhXcCyWai4qpah4ok5OQOAJvyFuNitDL9Qu7uDRu7UqGJu4vnZO+bFfB24ixbPwBKI06h7GZcng9FLoLN5VQvOdUunj1uk8Ws+yfjSfQlQArKEW+KLOkSUbHzzkn7ZX9MJkxPhvaqgy1rpQ3pClyKUl48/o8WIJeHr/a248kcpalE/15GFHSDVQ5VqUgZfUcJDuAmN6JbDp4hpp95sk8h6ASVBubuTLty+32KklMKYenw/Emsw6kl9v72eu1qFiOdE5UGnYbideGQYnKTf689WHfzMUO2GOMLhHOBt2R1btJcrMWzp42alXHQp1qm68XKy8xqv2AmoqFvbCgs9o2MP49VK7fdEVb5uRwFuqYwr2O6cK4H6vDeTD3YPx9xlRJ6eSFq+pSp9RlwdyH20Zz7w9lfNG85o4yKKOpOP1e0VxUXKoKp1QA9ZHp3b1ZcXq2doGIqkAaSA+Xz00fx3G54abS6MHffR8sSU9Vtyez9jdgvhdO3qPjyqRFdHJ+Oq//4VxFRv7dU5WoQfFbT7goR8elEfXkFPbi6V0HiwllHBv6CYfknb02SBwBhhIZrSwIwyOBJkf6FFSx/0y6cv2G+gqZlqDjSZXMlxPu5fGrFIWMVShtyc5YevSbJer6YA46MisV1/7zak1CXexAwnjiUGlR3FpTkkQauFHQ+/RvUZLaOyB0DzQG2S+343rVbo3diX6B1wZW2y3Pa6THLvG4MmEF95U8nszNioMRkqlAxSNrfUd6nMH2/9g3S2jxzlg1RuBZWTgCkjZ5ElofV8EM39V4f7YqZ4jDPQ75CVEYNsRjw+F7RY+GIQ+V54xDPpJzIMPbVaBmEUpki+beH7JSonShMShdUnQ0F5X0DsFDiRdGuAacEzVwyBRtdRqdBh/NVZsNi04l46R/2QZclH+uOUCor5s3U6pgL3etPTwyyNYGTYsVjYUsjajzIPJmJQcDow2GZzAZk7LkZTgnbVA+gfoJIW4YlO/+Hp90xh6PHOlSKPJ1KVaYPfZiwtDoqyVfISza4ksPJVEwHtVDpy9StcGzvbeGx5eJqv09UCbOd4K3dG1RWHQEfGG7uCQnZjpwWrs+0Lj00pto65SGyY35Srw2DBgkarnheXVjbIH6WLwjlh771nOo0avKyFKuzFeJjOB8WVITuFCDKcIQ6Xno/TNvJpIi4alkAS4aBzK9xCJoz7BHhMMOEek56vyzwSTyWY2t7tA53kpb/H04B3G7+Udz77cby+3+3hiULj2BaC4qTo6pViAzbT92jlDTGGWxUB4rkiIxgbohweB9vv/nj5Un1J2+00VSAy3t1oCyp0vhnbJe0xpf6B4EJ/rhsCPahlM1iHF+MIxhUAJP6U9ALYRQ2f6TFy29jFwD2QorK0txBpNdKbPqMa7d799vq57XJqxWxP6h0qLY3sBhA+mVZc9IOCUt+bZyffFnoZalczgVn2YSr71oe6w6bEgS+1D6jMY1KMsJih3ADxqVzO6z1j94pDS1rex5VyIJ5+t/d6gEsLszpfbWuk9IlYCkvtgjLzOY8T2w5kfPXiZZwQyfg5Ln+0W71XsjUpViwn2ezOHK/YRSJlKOQXdu4LtP2palh8pHhrs1mnt/uLqO9PXGoHRJw9FcVEx2XPGejOr0GS2y47/WHKDXJnhqnuukzVwTl9UZSRC0S4/MlW5kRvT3T1chVDRhT+bTY5YrjlApoSQZMfky+gknw1diG7s1K0aDJm/xq4NXGxSmPbHnlbGGgwsOLEg2Yc8qQxxgmHaskd+nn+PnLhMwtaA6siMmlxeiCk3xXlPVR3rmub+B8uFKh2C48nCD6AQVoZjXky9DfeUW5XIH0cutTSW5Pr5Nmvgu2ti3ia3nNqybiotlaVF4/cBL6ATb6tb9w+kHdGaokvPz8n0+3nhpXHw8fauq5FUubwbKnTGlypy+3Wsp0Jz5XVAiZzrFR8nCdbBlBTP+DnpIFCJJeDj6d3qtXtDAijnCaV9oJynn+Do3Dnf+xhDNvT8YPdyOtsagdEnr0VxUTFAN3BjKsV26ekOV9nugTGQ9glM3HKbOYz2ZwXqmJ9P8sDrHdapKNQplcUm7Cbebw6cv0b2DZ3kHKPk5W321iFbsOekzeKsXvt3sXhy3kiavP6yaBcNhqfcrw7IwGD+bFePTBIB+eDIQSkNZwT9XH1CUSCySW5Qr1Pg7+ePwce36TXqkonOvgOQzRWbuyKcq2amGjp65pAyGluVz31ZoxOq9J+khkaGOgbtBQC7J9dEnohAw7qMlEq+9ZEesqkoUTOnQaI0z0H2YiYHbyHclfgPw4gGDmDN9Si8d2+CHyySEod8yhk7fL6eZm4/6UOKgEZeXBMfstDc8Fcz+K8IcrjxeYFlnv1k35OE/8Pl82nAg3thGR9IrHXLHfi6M5t7v9tjd7s8YlC5pNJqLShJUw6C8cZOCKucX6pRlAoqebCG9aOj/txeqEzyod7qgxBfC/UycjBf7zjiMpEymYT1g4/rn1eAYBN6cuFaF4iBOElX86Xz57hPUOq46DmqCj1ro4RRlgXd1bKeq3r9hzDQeNs9LB1Uwa2qa9WZdQh3vwt09VaTc5ndrN3IJrdl7SpX1vJ3Y2WDXrVVo+tsnKxES58IRHQYArk1k6UdLJF4bySBIcohkgkMk5iUjK+hf4pC/mruDPpi6hYpkT6PgKvNjjifojOjnf1yhyl/my5zqFpwg5hYM72skdB1KnzIjH9eHaxQzV6ccSzh0a3ZziubebzeW2/29MShdegLRXFRMKg3uPWDcIAi3IlMxkiKB7npWeecfV9LUjR4vGmTyq7UUUfD/BwGBMELDb/7iyZrmmtKyfCTrQXownepGViia2aWOqsoTishkmbaV8tLPK/b5dGPFtYkw9NglexRPHNN5IKRdacBMda3bnIjwUiLDPEXSW6vzhDLnaF2D5K2mn8ZTHuG+4eBdedww4MAlCLkd1C8Srw2DEvJfg7PoJT3lIYjfpQWypFaGerSKRIS6LjlaoZeT5P4q3ZORfn2heqjd35brpm88TM/FceJiAAjnT34tuEO3HLhVZMgNxgV/yonm3n9bHlAQNzUGZRDKCtQ0mosK/H/wHGCDR+IEJBxsnVMVyLBevxYl6UlBiaInoMx5q64qQff/RWSVF5CEg2aHOSylDqx4G+10NGjyZvpm3k7VLFhSdNm35HnUK22gnT9jVVKwwFjeE3uBkD0K+f/2nP09q5gjZ6nhJ/N8vnYL9oHsfGRXA7ecNHEiu+Xi6veM1waBPQooQKJxeHVzEjM3HaFOP6zwdimfCxJWev9vI+FwjgxpVSSiQWHq0rCIm0NwrS+m64Lxi4OdLgmZ8sifEmZvOUJPj4l/PlbFEoJR4KPfLKYlO+Mz4HGtFdY7mD4Tyt7v1pgj1Y8xKF3SbDQNSs62RlWOa4h3RynELL0wOiaFsVaszmhSm7j0CMPqZuWeE/TIV4tVH2t6NVQGJXuWZMfNSuegYDNIgfFivOOy7g0oW9r4TPJgBi0TPGDYguMQSUKx569Qqdzp6c8Xq1vSGa3ff5qaf7FA3QrPdf/JC94EFMw1Q6pkwQzjjmwrk7N4ghOfr0YgnP4vC+O1gWsD3y0kGodXN3Wmc3n+0rkaVc7neS5MWQOqmgwpk9HWI2fp7SZF6cW6hdwcgmt9MasBuGSRjKnL7ebtDWWi87YdI2Svs1S4OwP9/mKNULpS18jCDNyJv0pnId9EXBjNvd+N8UayD2NQuqTdaC4qJEu8/rMn25rl71dqKqMgkiI3TR2Yr3MuysorkRxTQul73f5T9OAXHu5FGH2J7rrLGxaWY2xVMQ8h4zAYYZwXrglHr/tOXKBaQ+aoWwPfunLPSULyy6CHS1G6FEn9hpllYgjWGRKR4PEBXnT7wGaW/JzBzO9OaLs39gLVHurRLUs4CVQJRSeM12ZaGozrv4aPXrT9OD0exxGK8f/5Ug2V1Q1hyhp4J9OkSKK875L4PKE8Bx5Hl4lrVNWfVMkS04Ur128ZXpOSOejr9hUT2rADjkdGd9AwFFiQvIGO58d3bpRB9TeJaO79Cf3BGoPSpScUzUWlg5gxhRlv1CaUDoukSMNCz2CVYVmMwQkpdSTHGu2+txw+Q02GeTB0C96pp5IX6n/kCQtLsSICtxsrh+XQLtgqO7JvmZFeLEdalUikl9C0Gsvla9epaA8PnQ9wgSgh6a9+ud1c7tTv9WxszDMah7xI65MxhtKAmfRyTSqdJ7KHVzfnJZPR0K9MbOPDedoUSSh5kkRqbUcyIzjceXX9ZS39snK/oo+6et0TnZLyYNlcqprWf0l08n6QzI9/9t6Qp8Bk/LKDro2L0kv1IuN1jubeH7JSonShMShdUnQ0F5WOCcIU/u1al+7JHFnMIjjoKvSfoTQ2qkMlql8sPoOV+dzwHXgYtw9sGlQ1GJcew23rBiUwG8QZkLPfrKPCUUwjg8o0qAcOCaUEmiSU3zU4uCo7UiHy+TGo/6HyuekTUULTnwLL9JlGZy5dU97VUxeu0IB/NqtM07ld6902nSekG4O+qMqgePoojG1qFGtuR0oXelk83AdlOovmiOzh1c35rNl3ipD5yzKzS20qlM0zfvBOvjR+laoedZNuRo2CLdT5vfvbOpqw3DeZTvbVumIeGhpkBCTUsbh13aq9J+lhQbkVbpGCF8aupCkb4hNEMU5gYoGNjYREc++PxPjd7DPBGJQXL16kxIkTU7JkyWjz5s30zz//UPXq1dV//wWJ5qLSMSfQTzQwi8j4LdF7qiKs1nFUX87ZTkOnbVWPKk3yJLRB1JP9Lzy/cMcow8nYcFFR5Kk4XBCysoFfhLzZsAi9EuSLjbNUselt7t8k5KGiekip3tPU9ZzQhWzvD1rZc+5xTXOEjs5eukZfzNlOqNbx10uhY51CnkgCvDD23GWqGJf5zsOb9WYdKpg1tIz8hDLFn5fvpXd+iy/RiXH91xKxNh48Tfd/5sEAQ+Thm6tHMY8jvnebCsvNZ9ntj/UK9+lP2lW9mwY+FD2eUjfmJuFC6K9+sWw0qkPlkLtGeVkkskmxqugV8g20C6O597s15kj1kyAMynnz5lHLli3p119/pRIlSlCRIkUoUaJEdO7cOfrpp5+odevWkZq/a/1Gc1Et3RlLbb/x1KllWdWzIWVKHfnkCID0YRzBqyUzTjk0hvEgAxF8ef+f5MiZS1Q1zkOFkODu2POqbjZChcC2IqwDCQWfhbrmqFiCBI9OtQqErFYZuuaErvb33kOobWwnbb5erChVnq2VX3lbxy7ZS6j/PaZjFbtL/198f/rCVSrbb7rPXMPJyE8oSrPCawPSkSdjqoQyRNtxyFKgaLzo3fqUK0NKdZ1kMOCOfu1cjSrFJe3Ydh7lBswL6u+24Jft1bxElEcV3u0k+wR6AncrOFxDlS4/r6HfVx/wuRz13N9qXDTULgNeF829PyITcLHTBGFQ1qxZUxmRn376KY0aNYqGDh1KMTEx6t8jR46k1atXuzjlyHQVzUVlVZUDHkF4Bm+XMJUR7u+0bN7tGmsk7isJqJG0gIz4Hn9uoBzpUhDqYqO6BeT9h0vToxGuue5vfjdv3qT87032+drpBsT8dyhLd/n6DRUqbFkuFw179L+F14rEs0ef0vvL91j8Xn1C9ZX/sljhtcNhGrgduth1/DzV+3Cu99aSHH5BzHF64rulPsOa8lotKp4zYXLo9vnfRhqzaLdfNYYCqbkdz0TeUzf4m5bKQV89EXpiEeNM5T0iqZdo7v23+1nZ3T9BGJSpUqWiDRs2UIECBahZs2aUN29eGjFiBO3Z83/tnQd4lMUWhj8Segu9d0LvNfTeVUBABBQQRKmCIqCAQOgg9yJcqYKAAqJUkY6IdELvGAiEplSpCtK5z5mwm82asuX/k9nsN89zHy/J/5+Zeefszpcp51xA4cKFIdvhupfYdCr7LRxhYwmmHVecLLclpX7JOrHxo5px1ZQ4qVdWEYu93E6W4wCHLt3G5+tPKRbFs/lZ/2I2Ir+zOx0sOHidChxuKY5+0VpWRuR8kxx52HFG74wi7jBy5d2HT56h8JCwi0uWEttZbVxpd0zvRHZe+8jQBvBLniimV7X5vYS5qjY+/Aa+baSEoNCbaGO326PzyvLI1Sfx9Y6IGa5sQZu5tWvWgNqH3HqtVDZ86cbFooHLj2HR3ojHAiSG66AmRUzpQmzO/aZ0wECjWgjKjBkzYvv27cibNy/Sp0+PuXPnqm3uPXv2oFmzZrh6NeIBWwP7b5ip2HSqM9f/Qr2J4UGUbdP9GdYhJw3Z5mP1xrN1thlNvu1cEbvO3sSMrWchmSsk3eI3uy8oonGdZaTY0PW4bxNuxNEgzpM3heCLTafVyo3E1pZcuR/WK4AP6+kZANpJ93X7ccmW5P8yHaXFmKcJr8ggRHZe+7cRjZAssedkMrK/MGUbKUFCZ7WcvitC1w8NqY+0sXB8yBWns4+mYW/DzNvMrrTXkXfsQ245elEwKttDVx7Hty+/by3POLoT40h77Z+JzbnflfbF5jtaCMrWrVurVch06dJh2bJluHLlCs6ePYuuXbuqVUs5R6l7iU2nsv8A5kqXHNsGxO1tW8vhdhmnSvnS4fv3K+s+ZIa2z3Y7WUTjL8HX1V/JdQtnUikov9x8RtX33XsBqJI/g6F1O2OszIiNuP0yu5K85+gEJOkXZQtf4hHK2dk/7vyDwNeK4p2qeZ2pPt4+G9lxAkvGJE/udGQreBLBIWEsZ+xxh6F9PvSzY8Jjp9oG7bfUoXPIs3HrgtUfqlEVMwN4uzMG0b1rH3LL3Zvqw1edwNydEY8FuBKuzdH+xubc72ib4uo5LQTljRs30K1bN4SGhmLYsGHqgs7HH3+sVijlok6WLFniio/D9camU9leAJEGfv9+JVTKl97htprx4JZT1/HO3H3KtCdmazCCiWU7WWJ0rj52RZ0zlL+2i2RNhTFrg1UVcitaVnDjqtjnF7fPyR5Vu9Yfv4puCw6okFCJfX3UxRz74PZx1Sdd6s0/aC2evcxcJW3SWZg4ysz+vLZPAiB07CuOvq7Fc389fIISgWEXpuzbb5+DXXz79OjGWrQ7skb8Z8MpFWEhqmJmzmqzoNivILetmBNjW8QceSKq9kS2ivt2pVwY1dyc2++xOfebNQZG2dVCUEbWmcePH6sQQp5SYtOpZDWk87x9uP7XI3XbuqDJAc0dGQPblQxX0gs6Uofuz0hIHrmcIecklx74HdtD/oT8ZSzbxJ8uDwu9EhsB6KPjVG38Zvx+O/xM8vCmxdCxSp4Y0dqmlrQ8rPPlhRg7ZMIDkmpTjj5YijsxQ01onksm7z18ojI+WfqVNJEPgkfqK7gi66Tt+Vb79tumI5V3JVKGRMzQtdimYY2sjZ+3LInWFXLq2vxI22UbH1cecFf8WfLP21bmrkiNDmhszv26D6w2glK2uWfNmoXg4GBMmjQJW7duRYkSJdSlHE8o3u5UtisZLcpmx8TWpT1h2Axto2U7eUKrkli45yIkoHLP2vlRNKufCp4sZeendSBBxeOq1PnvFmtOZmnDmNdLoF1Arhibc/7P+6hlc1NWbq/LLeYEcoCXRRGwPZ8qYZnOjGkSL8j0XnQIckZaimSUORbY0KP6JavGsnocWfsv3LyPmhPCb4DrHqHCcpY5qgHwxF0DSQJRanh4yK13quRBYNNiLvtYZKLb3W10CkrHhkMLQXnmzBkEBATAz88PIsxEVH7yySdYt24dNm7c6BHBzb1dUNpuHXlicF3HPi7RP2XZTh79enHM331BpTbs16Cg2uJu//Ve9XJcH/hvNGmbapelSOYbyS8eU7EPi9OmQk6Ma+n6tlRM9Xni7y3ZhKTt7gah16n/20NuWP1X2nV+nGdteUubLccRMqRMjP2fha9A2p/fk5Sk6z+soRP+CG2ZsjkE/9l4Osr22afE1bYjNg178Pgpig4NS7ggpUu1vPjsVddjaf7vlxCIqLQtZi5yePvcb8tZC0EpN7nlpresUKZOnRpHjhxB7ty58c477+DixYtqtVL34u1OZRv6wd0vBN3HOqr2WbaT5bKK3DIM/fM+5JC8bEE1nrRdxeeUkEJxuarXdMoOHP39rrULkvdX8v/GVOSYhYTFefRyS3fG22XRqHjWmF7zqt9LWlLZvpOSOmlCHPWwlbyoBst2hc9TBWWhz9Yp383qJyvrda1dvfHXI1QYvcn67wp50mJJN32zs03fchbj14edx46s2KfE9YQPoG3CBWmvo6HMourbtC1nVMg229KsdDZMNilmrrfP/doJShGTIholS06qVKmUoJTb3bJSKSuXd++GT4C6fkC83ank1m/VcZvV8PSu44++DczJSqDr+Eu7LNvJg5oUxje7Lqib0JZD8jIpy4WAuBST0sZW03dh/4XbVowz3i6HRsUdu/Qm4yt9ku3cQ0PrI1VSz4lFGBt+I8JEBIqU9CkS44DGZ/Gc5WEJbJ8xVRLsG+x5WbBKDNuAvx49hX1EDMlLX3rEz1Ycumd/+mrbWesFv8jGcMG7AahWIO6iSDjrV/L88+cvkO/lkQT5t7tZbWZtC8Xotb9FaMorJbNiaruyrjQvxne8fe7XTlCmTZtWxaEsXrx4BEEpt7wbN26MW7fC0tbpXLzdqWxzGQ9oVAg9avnrPFymtM2ynSyheObsOIeb9x9Dt0Py7WYFqRiZljL3nQqoXTiTQzyaTd2JI5fueGVYKEcAVR77C67cfagelTOmQYPCV8IceV/nZ/55/AzTt5xBzUIZUS53Op2bGmnbLKvH/plSYlPf8KQLtgkJ5EUzhYcR0GxT3EZmb0m3yqigadrI6Pqfd+AalTBBirvB2efuPIfhq05GqM7d7DvRtd3b537tBGWbNm1UmxYuXIg0adLg6NGjauu7RYsWyJQpE5YsWWLEZ9FUG97uVLZn7Ia9VhSdvDA+oWU7WYKFi6C03PhuVjq7qb7njPGOc/Zi6+kb1lcWdglAVX/HVjQsIUs88eC/M4xcfbb655tx6VbYDfocaZNhxyd1XDXF9wwmYDnfLBEXJDqBpdgmJJCf6X42ODKxZIsqrsOSuTpsthm85Nx5rzoFXDWF+bvPY8jKExHedzc/OAWlY8OhxRnKy5cvo1atWrh586ba3pab3ZJ2UQKdb9u2TZ2n1L14u6C0zRQSl/mq49JPLNvJ3Wvlh2y7PH3+As5sKcdG29/7dj9+PnnNWtXirpVRMa/jK05yI9MvGbe6IxsryRcteaOl5MuQApv71YqNIWUdDhCwnG+2z+JlH5D+3Wp5McSNCyEONMWtR77dfR5D7cSSrcH1H1ZH4Sx65iGPruNFh67Hg5cZvD5pVBjyHepqkYQSkn7RttQpnAlz3qngqslo3/P2ud8WjhaCUhr04MEDlRHn0KFDeP78udr+fvvtt9VKpaNl1KhR2LRpE7ZsCQ8DcfjwYfTp0wf79+9XaR179+6Nfv36WU1KXcOHD8fs2bNx+/ZtVKtWDdOmTYO/v3NbtnSq8JuUcZ2v2lF/Mfq5tl8FYXfoTUjYi3m7wjI1zOtUAbUKObalbHR7IrMn4Ysk4Lql/NizKkrHYaD12OhzbNVRb+JWSFxDKd6Yzz62OLtSj0XsR3bpxn/QWvXHnxRHU5G60gYj3lm45wIGrzgepanNH9dEvowpjagqVm2UCNyAvx4+VXU6mmwhqgYu2X8J/ZcejfDrmgUz4pvOFU3pE+f+cKzaCEpp0sOHD3H69GkkSZJEXcpJlMjxlRCJXdm3b1/UqFHDKihlxVNWOyXzjvwuKCgIPXr0UIKxU6dOioKISfm35A/Pnj07BgwYoDL2nDhxwqnA6nQqoPnUnTj6+x1s+LAGCmgQbN2Ub49ojFq2k1uWzYFlB39XT8qt7sr54zaLkW2T+/5wGMsP/WH90dre1VVqSBb3CdiGZJL87Wt6h2+tum+dFtwh0OCLrTh97W9UyZ8e371XKYKpIkPWq8xPRogZd9royLvf771oTZIQ2fNxHefWkT5E9oxthAR3j0z9eOgPfPjD4QjVVPPPgAVdAlxtXrTvce7XTFA+e/ZMrRrOmDEDkiFHSrJkyVQsys8++yzam7F//PEHunTpoi715MyZE5kzZ7YKyrFjx2Lq1Kk4f/48EiZMqOwOGjQIy5cvVzfIpa4MGTLg888/V6kfpdy5cwfZsmXDnDlzYDnb6YgX0qkAyUghW6KZUyd1BFm8e6bLN/ux6bdraFA0Mza+3FZe0aMKyuRKq01fP1l6FD/sv2Rtj1xQkIsKLO4TeOV/23Hi8j1lyH5r1X3rtOAOAcvYRHaL23Z1zNFA/+60xZ13F++/hAF2q2+29uQGvtzE97RiGyHB3fSRq45cxgeLDkVAUOPMw6IAACAASURBVClfOnz/fmVTsHDu10xQDh48WAm4kSNHqjBBIjBFIAYGBqJXr15qFTGqsmrVKrVVLtvdI0aMUOLRsuXdpEkTdcnnu+++C59AN21C/fr1ce3aNfWs1Hfq1CkULFjQ+oxse5csWVKtXDpa6FSOkoq/z/VceBBrjl1B5Xzp1da3FN1WAIf8eBzzgy5YB2H7gNrImS55/B2UWOyZ5Ra8VKl7PMNYxKJFVbJ7IpmrIrucUW7kzyoigxTdj+ssO/A7Pl5yJEqmRwMbILUHhvOqMvYXXH4ZIWFsixJoWzHm7F1RQVh//Aq6LQjLTGYpFfOkw+JuFJRmfxi12PKWVUXZchYBaFuWLVumBKWkZXSkSCB0W0EpolDCDo0fP976+smTJ1GsWDHs27dPBU1v2bKlOr8pK6KW0rp1a/Wz1atXO1KteoaC0mFU8fbBD78/hB8PX0bx7Klx/I+wlapfPq6J/BqdaRq5+iS+3nHOOgZBA+sii593rigb7Ygtp+/CgZcxPuWPikXvR9xaNbo+2nOcwBszdmHf+duRhgWqNOYXXL0XFu5J98DgKw//gT7fR9zOtaUQPLIRkibydRyMJk/aRkj4vFVJtC7vej7yTSevocu3+yP0rGyuNFjeo6opveXcH45VC0Epq4i7d+9GkSJFIgy4iD9ZQfzrr/BUcdF5hL2glIs17dq1UyuXliLnI/Pnz69WQEV8tm/fXq2I+vj4WJ/p0KED5Oa5XPCJqty7dw/yP0sR0VuxYkVcunQJOXLEnMrOFM+m0TglMGDpESze/ztyp0+OCzcfqLbodqZp3LpgzNh61srp4JD6SJcicZxyiy+Vt565G3vPhcXMrVEwI7416RJAfOEVm/2wxF99vUx2fPFm6QhVW26Ayw9/eL8SAvLpc+bZnpH9dq6vTwJI0gRLOTe2SZwnT3BlXOv8Z4vKLCZlYutSaFHW9Tl0y6nreGfuvgjNKJXDDyt7VXOlaTG+Q0EZjkgLQSm3sK9fv45vvvkmwkWYzp07q7OPX331VYyDKg/YC8oSJUqoVc/IVigPHjyoLt+0atUq0hXKR48eYeXKlVHWK9vxkW3FU1A6NFTx8qHBK45h4Z6LKkuKZQtt/2f1kCGlPmeaJMet5Lq1lOPDGyJlkrDzxSzuEbANGl+vSCbM7mhOmBL3Wumdb1suzLUunwOftyoVAYKtmFn9QTUUz+6nLaS1x66gx8Lw7dzkiX2t4XYS+/rg9OjG2rY9uobVn7gVIS8jJDiaDjYqeztC/sTbX++J8GvZNVr9gTmX5CgoNRCUdeqEB/19+vQpduzYgaxZs6JChQrw9fVV4YNEnEme76VLlzr0IbEXlCImJVTQ/Pnzre/LqqP9GcozZ86oVUtLkTOUpUqVUhd6oipcoXRoSLzqoeGrTmDuzvOQL/bHz56rvusm2Kb+egYTNoTnuT09qjESJwxfnfeqATO4s+2/3oPtIX8qq42KZcGM9uUMroHmXCVgOTscWdBsyw1wsb21fy3kTp/C1WpMf2/98avotuCAtZ60yRPh9oMn6t+pkiTEseENTW+DGRU0nrwdv10J2/Gb9lZZNCmR1eVqgkJvos1XQRHeL5wlFdZ/WMNlm9G9SEGpgaC0hO1xZITlfKUjxV5Qjhs3Tt0cP3v2rBKpUgYOHIgVK1aoW96yCimZeCZOnIh3331X/Z63vB0hzWciIzB27W+YuS00wq9CRjdGIl99BJt9LmBP3SLT0QM7z9uHzcHXVdNeK5UNX7Yto2MzvbJNkmJx7/lb6sKc/RlD29v5uu0o2A+WJCWQ5ASWkjl1Ely75/n541/7cgeO/XFXdeur9uXQoFgWl/30wIVbaDl9d4T3C2RKiZ9tUm66bDySFykoNRCURg6oxZa9oJRtdIlD2bRpUxVfcu/evejevbsSmR07dlSvyQ3zmTNnqlvmefLkQf/+/dXZymPHjjEOpRmDFI9t/nfjKXy5+Yy1hz4JgLNj9DrTNG/nOQS+zHMrK5OyQsliDAHbLEQtymTHRLuzesbUQitGE7DcABe7ul9q+TX4OjrNCz8fmCtdcly8FXZeO5tfUuwa6Jn541+fthOHLt5R/XD3YpTc5pcxtS35MqbA5o/NyVxFQamhoJRBkS1myeMtAc3lJnbXrl2RK5fj4QPsBaV0U25zS3Yc2UKXLfWPP/5Y3Ry3FLmQI7EpZRX0n3/+UYHRpR0iLp0pdCpnaMXPZ+VsopxRtBQ533RyRCOtOvvdnosYtCIsLZmcnZQteRZjCHRfcADrjl9Vxt4snxPjW5U0xjCtmErAcgM8oU8CyI5CggQJTK3PHeNbT9+AnAe1FFl5s5w9zJM+Obb0r+2O+Th71zIG0gDJaCOZbVwtx/+4i1e/3BHhdbkoudUkNpz7NROUx48fV0IuefLk6qa0iDxJlXj//n3s3LlTiUvdC51K9xEyv31ye1puUVuKnG86NLSB+RU7UYNtWjK53S23vFmMIdDru4NY/TKt5VsBuTD69RLGGKYVUwlYLlNJjvojw/T6vNp33P7CiWRksgTTL5Q5FTZ8ZM45QVMHAIAlba3U812XAFTxz+Bylaeu/oWGk7ZFeD97mmQq4oYZhXO/ZoJSYkWmSJECCxcuVGkXpUgaRsnlLf91Jh6kGQ7jiE06lSOU4vczc3acw4jVJ62dzOqXFLs124KyjWOXJXVSBA3yzC0yHT3JEodU2ib53AOb6v+HsI4cY7tNlhvgZooOo/q06+yfaDcr/AazxFc8+HKruER2P6z6wJzQOEa1Pyo7thfa3A3ddPbG36j7360RqjLzu45zv2aCMmXKlCrPdvHixSM4gWx/y8qlXJTRvdCpdB8h89snGWjkNqml6LgFZZtFQs5fbRvgmVtk5o+m8zX0W3IESw+E5XB/r3peDH6lqPNG+EasE+jyzT5s+u06PGGFT+KcSrxTS7HNylU+d1os7V4l1vkZUWGnuXvx66kbytSy7lVQLrfr6Wov3LyPmhO2RGiWpKOUtJRmFM79mglKOdsoq5DlykUMsyHnHyXEDwWlGR8D2jSawOJ9lzBg2VGrWR0nqF9+u4Z3vwm7JSo5vCWXN4sxBD5ddhTf7wvLk96jVn4MaFTYGMO0YioBy9lXM7OpGNUB+xvMctZQzlVKqeqfHgu7eGZ2pi7f7Mem366pfqzsWRWlcqZxGdkfd/5B1XGbI7xv5vEeCkrNBKVsbUtmGok3mS5dOtW6P//8E5ICUbLoLF++3GXniq0X6VSxRVrfelYc+h0f/RCeZ9fM7AyuUtgecgPtvw471F80a2qs7WNOsF9X2+fJ71kC20sf+tQtgI/qF/Tk7nhN23svOoSfjlxG9QIZMP/dAK37fejibbw+bZe1jQ2KZsbGk2FCrHahjJjbqaLW7Y+qcbYX2twNLn/93kNUHPOLqippIh88fPIcqZMmxNFAcy4gcu7XTFDKgFSpUgW3bt1CwYIF1S27U6dOIW3atNi6dSvy5cun/YeETqX9EJnewDVHr6Dnd+FZLCrmSYfF3SqbXq8zFewJvYk3Xwb9lVUAWQ1gMYbAsJXH8c3uC8pYZAG0jamFVowmMHTlcXy7+wJals2B/7aOmEXH6LrctXf09ztoOiU8JI7EO5V0jFI8OZi+7YW2DR/WQKEsqVxGdev+Y5Qd+bN6X1Ym5d9mRrTg3K+ZoHzx4oW60S0ZbeTGt/xb0iZKHm4/P33TYNl6PJ3K5c9/vHlx44mreH9+eBYLHVc8Dl68jRYvVzh0FLye7AwjVp3EnJ3nVBcGNi6MrjXDs295cr/ie9t/v/0AEk6rdfmcyJNB3yw5Mg4nLt/FK/8LD4kjInjZwbBzu01LZYOkLfTE8tEPh7Hi0B+q6XIMR47juFru/vMEpYZvVK/LRSvZApeVyuCR5sTc5dyvmaAsX768CixesqTnxm2jU7n68Y8/7205dR3vzA0POlyvSGbM7lheqw7axmir5p8BC7rovcWnFbwYGmObKWnIq0XxbrW8ntR8ttUDCARfvYdGk7ZbW9ouIJcSw1LeKJcDE97Qe4U1KsT9lxzBkpcX2rb0q+WWsH/w+CmKDt2gqhJheub630jkKzFGm5gywpz7NROUGTJkwJ49eyLk0zZl5E00SqcyEa6HmN515k+0mx0e0uOVklkxtV1ZrVofcu0v1P8iLEZbncKZMOedClq1z5Mb8/n6YEzbclZ1YWSzYmhf2bnkCJ7cd7Y9dgjYfn6lRglPNW/XeVW5J8c+Hbj8KBbtDbvQtuOT2siRNrnLQB8/fY6Cn61T75fM4Yejv9+FZC0LHfuKyzaje5Fzv2aCcsKECVizZo1Ke+jv749kyZJFGD9nsuWY4jEOGKVTOQApnj+y//wttJoRHtJDxzNZ5/+8j1r/CQup4clnrnR0pYkbT+F/L1Nvjm1RAm0rOp7lS8f+sE36EbCPsdilWl7M3XUez56/QOeqeTH0Nc8MVSXh1iTsmpSggXWRxS+py/CfP3+BfIPWqvcD8qbDnnO31P8PHdMEPqIsDS6c+zUTlD4+PtYW2aa9krOU8m/JnKN7oVPpPkLmt+/IpTtoZpNDVrajxmiWLeXynX9Q5WVIDU8+c2X+aDpfw+RNIfhiU1jqzf+8UQqtyuVw3gjfIIFoCNj+QSiPda2RTwlKWZXrVjM/Pm3smaGqAn86YV1p3f9ZPWRIGZbgxNWSb+AaPH8B1CqUEVtexreUtJqJfMO1hqu27d/j3K+ZoJSb3NGVmjX1j5VHpzLq4+m5dn67cg+NJ4efb9JxxeDPvx+h/KhNCrKOK6ieO/rA1F/PYMKGU6oLk9uURrPS2T25O2y7hgQu3XqA6p//am1Z91r58e2u87j/+JlHh6oaveYkZm0Pu9B2eGh9pEme2C36suUtIrtJiSxYe+yqshU8shGSJvJ1y25kL3Pu10xQ2g6ShA7y9fX1mNvdlrbTqQz/nHqcQTn8XW9i+B9H8mX/iWbBrW1vQMqWrGzNshhDYObWsxj7Mpf7tLfKokmJrMYYphUSeEnAdodBftSrtr/aKpbPdf+GhdCztr9Hshq3LhgztoadPz4W2ACpkiZyqx/Fhq5XItv2FvyJ4Q2RIklCt+xSUEaPL8EL2VfWoMg5ysmTJ+PKlSuqNXnz5sUnn3yC9957T4PWxdwECsqYGcX3J+xXDz6sVwAf1tMruPXDJ89QeMh6NRTMN22sR87eHopRa35TRmd1KI/6RTMbWwGteT2Ba/ceIuBl0G6B0btuASwMuoCb9x/js1eKoEt1/WM2RzaI/914Cl++PH9sxEria1/uwLE/7qrwXZY/8o4MawC/ZO4JVQpKDxCU48ePx4gRI9C7d29UrlwZz58/x86dOzFt2jR88cUXeP/997X/IqGg1H6ITG+g/Ze9nGeSc006FTm8n//lgfX3a+TDoCZFdGqeR7dl3s5zCFx1UvVhbqcKqF0ok0f3h43Xj8CNvx6hwuiwIytSPqpXEIv3X1KxFke/XhxvBeTWr9EOtGjSptOYtClEPWnEWcc7Dx7j99v/4NHT52g5PSyz0KEh9ZE2hXtb6RSUHiAoc+bMiTFjxqB9+/YRWiuxKceOHYuQkDBH07lQUOo8OrHTNtsMDVLjsNeKolNV/WIRiqAUYSnbZf0aFoodOF5Qi2w9ym1VKQu7BKCqfwYv6DW7GJsEbt9/jDIvs8BIvZKRKXWyRFh37Ko6t5spteu3o2OzH/Z12Z4/Pje2ibqMa0SxvSi5b3A9ZEzl3mUfCkoPEJQpU6bEwYMHVdpF2yJCslSpUnjw4IERvmWqDQpKU/F6hPG/Hz1F8WFhAXWl6Bo6psiQ9fjnyTO1utGnXgGPYOsJjVy09yIGLj+mmvrD+5UQkC+9JzSbbfQgArZnoKXZAxoVQo9annlu0ha7nJ+Uc5S+PglwdoxxAchtEzm4G44oKjfh3B9ORoszlB07dkSqVKkwZcqUCGM2fPhwlYpxyZIl2n/k6VTaD5HpDXz09BkKfRZ2PlHKxNal0KKsfqFjJC2ZTExyYUguDrEYQ2DJ/kvov/SoMra8RxWUzZXWGMO0QgIvCdj/0RpfUnxazh8nTuiD06OMS5Fom1lo56d1VCpGowvnfs0EZd++fdV5ySJFikBCBCVKlAj79u3D9u3b0axZM6RJk8baYtkG17HQqXQcldhtk9xvyzswLKCuFF1v+krYIAkfxPSAxvrHikO/46Mfjiijq3pVQ4kcfsZWQGteT+Cfx89QZGj4H62efBHHdjAt549TJPbFiRGNDBvnM9f/Qr2JYZnBtvWvjVzpXc/AE1WjOPdrJihr167tsAP9+mt4DC6HX4qFB+lUsQDZA6ooMHgtnjwLC5zwdcfyqFtEv5u+lhuQkhZS0kOyGEPgpyOX0XvRIWVs/YfVUThLamMM0woJvCRgvwsy9NWi6BwPcsYvCLqAz348jtRJE+JoYEPDxjv0xt+o89+wUG6bP66JfBlTGmbbYohzv2aC0vARjgODdKo4gK5hlZb4Z9K0Be8GoFoB/S5myJfs4Ut38FqpbKZkjtBwWGKlSWuPXUGPhQdVXZv61oR/JuMnr1jpCCvRloBtlAZp5PCmxdCxiufnjP9h30V8suwY0qdIjAND6hvG/+LNB6gxIWwRalPfGvDPlMow2xSU/0apxRlKw0c4DgxSUMYBdA2rLD1iI+48eKJatrRbZZTPk07DVrJJZhDYeOIq3p9/QJk2a3vNjHbTpucQsD9WM7J5cbSv5Jmhgmypi/B7Y+YuNC6eFYFNixk2IBJOqerLVLNm7Rpw7g8fLgpKg1yXTmUQSA83U3H0Jlz/65HqBc/RefhgOtn8zcHX0HnefvXWrk/rIJsJFwCcbBIfj4cELHmqpWtjXi+BdgG54kUvRSwbFS7IAuTq3YeoNPYX9c/VH1RD8ezGn2vm3E9BafgHkE5lOFKPNCh/DctfxVJ+/qgGCmQ2fovFI8F4QaO3nr6BjnP2qp6aFfPOCzCyizEQsD2nPb5lCbxZIX4ISjMG3jYQ/E+9qqJkjvALvkbVx7lfA0H5008/oUGDBkia1DMDsdo7I53KqI+nZ9up858tCP3zvuoEtz09eyydbf3OM3/irdl71GtHhjaAX3Lj07w52yY+H/8IFB6yDg+fPFcdm9CqJN4onzP+ddKgHtkGgjcrlBfnfg0EpQQzDw4ORo4cOZAvXz4VJih9es8NBEynMugbwMPNNJq0DcFX/1K92DOoLjJ7aOYKDx+GOGn+sd/v4rUpO5DY1wdHAxsgaSLfOGkHK43fBGwv/uka61aXEbANBL+kW2VUMOFMO+d+DQRlpkyZ0LVrV9SrVw8SNmjFihVImzbyQMA1atTQxT+jbAedSvshipUGWkLycJUqVnBrVYmcAZu+9awKntysdHat2sbGxB8CJQI34K+HT1WHJN0ifS3qsb3/6CmKvcxe9v37lVDJhOxVnPs1EJTjx4/HoEGDYDmIK/+NrMgh3WfPnmn/bUCn0n6IYqWBLafvwoELt1VdwSMbcZUqVqizEhLwHgK2kSS+bFtGhf9iiZzAwyfPUHhIWCD4hV0CUNXf+DBunPs1EJTShLt37+LOnTvImzcv9u7di4wZM0bqFblzux4WYcuWLWoFNLIi9YaGhqJTp06YN29ehEeyZ88OcRRHC53KUVLx+7k2X+1GUOgt1clzY5sYfmsxftNj70iABGIiUH7Uz/jz78fqMV2zccXUh9j6/ZNnz1Fg8DpV3TedK6Jmwcg1hjvt4dyviaC0NGPr1q2oWrUqEiZM6M64Rvru48ePcetW2ARvKceOHUOTJk0wffp0dOnSBeXKlUOjRo3wwQcfWJ/x9fWNUuBGVhGdyvCh80iDHebsxbbTN2B0TlqPhMFGkwAJGE7ANjTZjLfLoVHxLIbXEV8MPn/+AvkGhaXDnftOBdQunMnwrnHu10xQSnNWr16N0aNH4+jRoyqXd9GiRdG/f3+8/vrrhjrAkydPUKZMGRQvXhzff/+92k6XC0KLFi1C8+bNXa6LTuUyunj1Ypdv9mHTb9eRKmlCHDMwhVi8gsTOkAAJuEyg8thfcOXuQ/X+rA7lUb+ofuldXe6cCS9a4naaxYpzv2aCcuXKlWjRooUSj3IBR0Tetm3bsGrVKixfvhxNmzY1zM0mTZqEYcOGqRvmWbNmVf8tUqQITp06hYIFC7pcD53KZXTx6sUeCw9g7bGryJgqiYpFyEICJEACRhKwjXU7553yqFOYgjI6vpa4nTPeLotGxbMaORTKFud+zQRl2bJllZgcMmRIhMEeMWKEWrmU85VGlIcPHyJnzpzo0aMHhg8frkwuWbIEb775Jrp164Z169bBx8dHbYePGjUKfn5RR9W/d+8e5H+WcuXKFVSsWBGXLl1SoZBYvJNAn+8PYeXhy8iRNhl2fFLHOyGw1yRAAqYRqPH5r7h464GyP69TBdQqZPw2rmmNjwPDlridU9uVxSslKSjNHAItUi8mS5YMcq7R398/Ql9DQkJQqlQpPHgQ9uFxt8yaNQsfffQRLly4YI15KauVstU+ZswYJSSlzn79+inhuXnzZiUwIyuBgYFWUWr7ewpKd0fJs9/vv+QIlhz4Hf6ZUmJT35qe3Rm2ngRIQDsCtskTFrwbgGoFjL+5rF2n3WhQ8WEb8Pejp6aFWOIKZfjgaCEo8+fPjy+//FIJOtuyZs0avPfee7h8+bIb7hT+avXq1VUQ9W+++cb6QwlXJLfN06QJT8kUFBSEypUrQ/4bEBAQad1coTRkSOKdkUErjuG7PRdRLFtqrOldPd71jx0iARKIWwL1Jm7Fmet/q0Z8914AquSnoIxuREoGbsC9h09hVhB4CkrNBKVsdc+fP1/dupbb3lJ27NihtqabNWuGyZMnu/0JvnHjBrJkyaK20Bs3bhytvfv376uLOkuXLkXLli0dqptO5RCmeP9Q4E8nMG/XeZTNlQbLe4T5MgsJkAAJGEWg4RfbcOpaWDauH96vhAATgnUb1VYd7JQd+TNu3X9sWppKzv2aCUo529imTRtIfm8JZC5FVg5lxXLx4sVInjy5234pF39EHMrKoq29du3a4ebNm9iwYYO1DkvsypMnT6oLO44UOpUjlOL/M7O3h2LUmt/QrHQ2TG5TJv53mD0kARKIVQKNJ2/Hb1fCzu8v7VYZ5U1IJxirHTK5sgqjN+HGX48wrkUJtKmYy/DaOPdrJigtzZEb13KWUsRkiRIlHBZzjniIXPBZsGABTp8+HeFxWbGUW+QjR45UolZ+37NnT7XlvXDhQkdMq2foVA6jitcPSmaGLaeuo2Le9EiXInG87is7RwIkEPsEbNO7Lu9RBWVzRZ6yOPZbpmeNlcb8gqv3HmJU8+J4u5LrSVKi6h3nfk0FpZnuKNvnhw4dwu7du/9VzbJly9TFHBG0cpZSVi3llnfSpEkdbhKdymFUfJAESIAESMBFAs2m7sSRS3fU2yt7VkWpnOHn/100Ga9fs4RZGtGsGDpUzmN4Xzn3e6GgNNyL7AzSqcwmTPskQAIkQAItpu3EwYthgnL1B9VQPHvU4e1IC6g54VdcuPkAQ18tis7V8hqOhHM/BSWdynACNEgCJEACJGA2gTdm7MK+87dVNWt7V0fRbKnNrtKj7VvCLA1uUgTv1chneF8oKCko6VSGE6BBEiABEiABswm0+Wo3gkJvqWo2fFgDhbKkMrtKj7Zff+JWhFz/G580KozutfIb3hcKSgpKOpXhBGiQBEiABEjAbAJvzQ7CzjM3VTWb+taAfyYKyuiYN5q0DcFX/0L/hoXQs3bE5ClGjBUFpWaC8ty5c/jkk09w/PhxSAgh+xIaGmrEuJtqg05lKl4aJwESIAESAND+6z3YHvKnYrH545rIlzEluURD4JX/bceJy/fwUb2C6FOvgOGsOPdrJiirVasGyYXdunXrSG9WS3pE3QudSvcRYvtIgARIwPMJvDN3L7acuqE6srV/LeROn8LzO2ViD5pN2YEjv99F7zr+6NugkOE1ce7XTFCmSJEC27dvR9myZQ0f7NgySKeKLdKshwRIgAS8l8C78/bhl+DrCsD2AbWRM537iT/iM03LrfietfOjf8PChneVc79mglKE5KRJk1CjRg3DBzu2DNKpYos06yEBEiAB7yXw/rf7sfHkNQVg16d1kC1NMu+F4UDPLbfiu9bMh4GNHct854BZ6yOc+zUTlEFBQejevTt69+6NvHnzwsfHJ8J4eoLQpFM58xHksyRAAiRAAq4Q6L7gANYdv6pe3TOoLjKndjwBhyv1efo7llvxXarlxWevFjW8O5z7NROUX3/9tRKUT58+/ddgS27vZ8+eGe4ERhukUxlNlPZIgARIgATsCfT87iDWHL2ifrxvcD1kTJWEkKIh8PbsPdhx5k90qpoHw14rZjgrzv2aCcqcOXPi1VdfVSuUyZP/+zxI7tzG59802qvoVEYTpT0SIAESIAF7Ar0XHcJPRy6rHx8cUh/pUiQmpGgIdJizF9tO30CHyrkxollxw1lx7tdMUMqlnBMnTiBPHuPzbBruPVEYpFPFFmnWQwIkQALeS6DvD4ex/NAfCsCRoQ3glzyR98JwoOed5+3D5uDraBeQC2NeL+HAG849wrlfM0HZqlUrNGnSBJ07d3ZuJDV6mk6l0WCwKSRAAiQQTwn0W3IESw/8rnp3LLABUiWloIxuqN/7dj9+PnkNbSrkxLiWJQ33Cs79mgnKKVOmYODAgWjcuDEKFSqERIkifkCGDh1quBMYbZBOZTRR2iMBEiABErAn8MnSo/hh/yX145MjGiJ54oSEFA2BbvMPYP2Jq2hVLgf+80Ypw1lx7tdMUMrN7qiKXMphphzDPwM0SAIkQAIk4IEEBi4/hkV7L6qWB49shKSJfD2wF7HXZMslptfLZMcXb5Y2vGIKSs0EpeEjHAcG6VRxAJ1VkgAJkICXEfjsx2NYEBQmKENGN0Yi34hh9rwMR4zdga9llAAAHOJJREFU7fP9Iaw8fBlNS2XD/9qWifF5Zx/g3E9B6azPxPg8nSpGRHyABEiABEjATQKBP53AvF3nlZXQMU3g45PATYvx+/W+iw9j+cE/8EqJrJj6lvHZ+Dj3ayYoJZC5bG1HVRiHMn5/4Nk7EiABEiABxwiMWHUSc3aeUw+fH/eKYy958VP9lxzBkgO/o2GxzJjZvrzhJCgoNROU8+bNiyAonzx5gpCQEMjPJ06ciLfeestwJzDaIJ3KaKK0RwIkQAIkYE9g9JqTmLX9HHx9EuDsmCYEFAOBgcuPYtHeS6hXJDNmd6SgNNNhErx48eKFmRW4Y/u7777D/PnzsW7dOnfMxMq7FJSxgpmVkAAJkIBXExi77jfM3BqKxL4+OD26sVezcKTzljOntQtlxNxOFR15xalnOPdrtkIZ1ehduHABRYsWxf37950a4Lh4mE4VF9RZJwmQAAl4F4EJG4Ix9dezSJrIB8EjKShjGv1hK4/jm90XUL1ABsx/NyCmx53+Ped+DxGU48aNw9SpU3HpUljMLZ0LnUrn0WHbSIAESCB+EJi48RT+t/kMUiT2xYkRjeJHp0zsheXMaVX/9FjYpZLhNXHu10xQShxK20s5sgt/79493LlzB6NHj8ann35quBMYbZBOZTRR2iMBEiABErAnMGnTaUzaFIJUSRPiWGBDAoqBwJi1v+GrbaEIyJsOP3StbDgvzv2aCcrAwMB/3fJOnDgxqlatiho1ahjuAGYYpFOZQZU2SYAESIAEbAl8+UsI/vvzaaRJngiHhzYgnBgIjFsXjBlbz6J87rRY2r2K4bw492smKA0f4TgwSKeKA+iskgRIgAS8jMC0LWfw+fpTSJ8iMQ4Mqe9lvXe+u//ZcApTfj2DMrnSYEWPqs4biOENzv0aCsqdO3dix44dePz4MewvnjOXt+GfARokARIgARLwQAIzt57F2HXByJgqCfYNrueBPYjdJn/x82lM/iUEJXP44ade1QyvnIJSM0Epl28GDRqEtGnTInXq1BEGnLm8Dfd/GiQBEiABEvBQArO3h2LUmt+QJXVSBA2q66G9iL1m/++XEEz8+TSKZk2NtX2qG14xBaVmgjJHjhzo3LkzRowYYfhgS+ihPHny/MvurFmz0KVLFxw+fBh9+vTB/v37kT59evTu3Rv9+vVzuh10KqeR8QUSIAESIAEnCczZcQ4jVp9E9jTJsPPTOk6+7X2PT/31DCZsOIXCWVJh/YfG38ng3K+ZoEyePDmOHj0Kf39/w7191apVaN26NUJDQyNc/PHz88ODBw9QuHBhNG/eHH379kVQUBB69OiBadOmoVOnTk61hU7lFC4+TAIkQAIk4AKBoNCbaPNVEOoXzYxZHYzP/OJCk7R+xXJEwD9TSmzqW9PwtnLu10xQNmjQQAm4tm3bGj7YY8aMweLFi9VKpH0ZO3asinN5/vx5JEyYUP1att6XL1+O4OBgp9pCp3IKFx8mARIgARJwkcCVu/8gY8okSOjr46IF73nNckQgb4YU+LVfLcM7zrlfM0E5e/ZsDBgwQIlKWTFMkiRJhEHv0KGDy07Qpk0b+Pj4QNI42pcmTZogTZo0EX63adMm1K9fH9euXUOmTJkcrpdO5TAqPkgCJEACJEACsUJg7s5zGL7qJHKlS45tA2obXifnfs0EpQi+qIpcynn27JnLTlCsWDFkzZoVjx49wunTp1GgQAEMGTIEDRs2RMmSJdG4cWOMHz/eav/kyZOQd/bt24fy5R3fTqBTuTxEfJEESIAESIAETCEwf/d5DFl5wrQzp5z7NROUpngRoEIQpUiRAmXKlMGECROQMmVKLFiwAJMnT8bPP/+Mrl27ol27dhEuA8lZy/z582P79u2oVi3qEAOSyUf+ZylXrlxBxYoVVZpIuWTEQgIkQAIkQAIkELcEFu65gMErjiNz6iTYM8j4MEsUlF4iKKWb9+/fV+cjbbfRGzVqpC7oiCPItndkK5QHDx5UQjSqItl9hg8f/q9fU1DG7ZcHaycBEiABEiABC4HVRy9j4sbTyJAyCRZ3Y+pFMz0jwQv7KOJm1qaJ7f79+2P9+vXImTOnChU0f/58a8scPUPJFUpNBpPNIAESIAESIIE4IsAVSi9ZoZRQRFWqVFHi0Xb7unbt2sicOTNKly6NGTNm4OzZs/D19VVUBg4ciBUrVvCWdxx9OFktCZAACZAACXgKAQpKLxGUz58/V4JStr2nT5+ODBky4KuvvlJxJuXSjYhKuVXetGlTdct879696N69uxKZHTt2dMqf6VRO4eLDJEACJEACJODxBDj3e4mglG7euHEDn376KdauXYs7d+6gbNmykFSP1auHpWASYSnZcQ4dOqRug3/88cfo1auX005Op3IaGV8gARIgARIgAY8mwLnfiwRlbHkqnSq2SLMeEiABEiABEtCDAOd+CkrDPZFOZThSGiQBEiABEiABrQlw7qegNNxB6VSGI6VBEiABEiABEtCaAOd+CkrDHZROZThSGiQBEiABEiABrQlw7qegNNxB6VSGI6VBEiABEiABEtCaAOd+CkrDHZROZThSGiQBEiABEiABrQlw7qegNNxB6VSGI6VBEiABEiABEtCaAOd+CkrDHZROZThSGiQBEiABEiABrQlw7qegNNxB6VSGI6VBEiABEiABEtCaAOd+CkrDHZROZThSGiQBEiABEiABrQlw7qegNNxB6VSGI6VBEiABEiABEtCaAOd+CkrDHZROZThSGiQBEiABEiABrQlw7qegNNxB6VSGI6VBEiABEiABEtCaAOd+CkrDHZROZThSGiQBEiABEiABrQlw7qegNNxB6VSGI6VBEiABEiABEtCaAOd+CkrDHZROZThSGiQBEiABEiABrQlw7qegNNxB6VSGI6VBEiABEiABEtCaAOd+CkrDHZROZThSGiQBEiABEiABrQlw7qegNNxB6VSGI6VBEiABEiABEtCaAOd+CkrDHZROZThSGiQBEiABEiABrQlw7qegNNxB6VSGI6VBEiABEiABEtCaAOd+CkrDHZROZThSGiQBEiABEiABrQlw7qegNNxB6VSGI6VBEiABEiABEtCaAOd+CkrDHZROZThSGiQBEiABEiABrQlw7qegNNxB6VSGI6VBEiABEiABEtCaAOd+CkrDHZROZThSGiQBEiABEiABrQlw7qegNNxB6VSGI6VBEiABEiABEtCaAOd+CkrDHZROZThSGiQBEiABEiABrQlw7vciQXnr1i0MGjQIq1evxr1791CyZEmMGzcO1apVUxQ6deqEefPmRXDY7NmzQ5zEmUKncoYWnyUBEiABEiABzyfAud+LBGWDBg1w/fp1TJkyBRkzZsTUqVMxe/ZsHDx4EIULF0a5cuXQqFEjfPDBB1Yqvr6+6llnCp3KGVp8lgRIgARIgAQ8nwDnfi8RlGfOnEGBAgWwc+dOVKlSRfX6xYsXKFiwINq2bYthw4YhZcqUWLRoEZo3b+6WZ9Op3MLHl0mABEiABEjA4whw7vcSQXnnzh3s2rULdevWRZIkSay99vf3V6uSvXr1QpEiRXDq1CklMt0pdCp36PFdEiABEiABEvA8Apz7vURQRuaaS5YsQevWrfHTTz/h4cOHePPNN9GtWzesW7cOPj4+aNKkCUaNGgU/P79oPVvOY8r/LOXKlSuoWLEiLl26hBw5cnjep4ItJgESIAESIAEScIoABaWXCkrZ+paVSVmx/PHHH9WW9+jRozFmzBglJENCQtCvXz/kzJkTmzdvVgIzqhIYGIjhw4f/69cUlE59FvkwCZAACZAACXgsAQpKLxSUK1euRLt27VCpUiV14ztZsmTqPOXdu3eRJk0aK5GgoCBUrlwZ8t+AgIAonZwrlB77+WfDSYAESIAESMAQAhSUXiYo5YZ3nz590KJFCyxYsCDCeUp7j7p//766qLN06VK0bNnSYYejUzmMig+SAAmQAAmQQLwgwLnfiwTl9OnT0aNHD/Tu3RtffPFFhG1sWbG8efMmNmzYYCWyZcsW1K5dGydPnlQXdhwtdCpHSfE5EiABEiABEogfBDj3e4mgPH36NIoXL45XX30V06ZNi+C9suW9fft2NG3aFCNHjkSbNm0gz/fs2VNteS9cuNApb6dTOYWLD5MACZAACZCAxxPg3O8lglIu2wwePDhSh+3YsaPKkLNs2TJ1MSc4OFidpZRVS7nlnTRpUqccnU7lFC4+TAIkQAIkQAIeT4Bzv5cIytj0VDpVbNJmXSRAAiRAAiQQ9wQ491NQGu6FdCrDkdIgCZAACZAACWhNgHM/BaXhDkqnMhwpDZIACZAACZCA1gQ491NQGu6gdCrDkdIgCZAACZAACWhNgHM/BaXhDkqnMhwpDZIACZAACZCA1gQ491NQGu6gdCrDkdIgCZAACZAACWhNgHM/BaXhDkqnMhwpDZIACZAACZCA1gQ491NQGu6gdCrDkdIgCZAACZAACWhNgHM/BaXhDkqnMhwpDZIACZAACZCA1gQ491NQGu6gdCrDkdIgCZAACZAACWhNgHM/BaXhDkqnMhwpDZIACZAACZCA1gQ491NQGu6gdCrDkdIgCZAACZAACWhNgHM/BaXhDkqnMhwpDZIACZAACZCA1gQ491NQGu6gdCrDkdIgCZAACZAACWhNgHM/BaXhDkqnMhwpDZIACZAACZCA1gQ491NQGu6gdCrDkdIgCZAACZAACWhNgHM/BaXhDkqnMhwpDZIACZAACZCA1gQ491NQGu6gdCrDkdIgCZAACZAACWhNgHM/BaXhDkqnMhwpDZIACZAACZCA1gQ491NQGu6gdCrDkdIgCZAACZAACWhNgHM/BaXhDkqnMhwpDZIACZAACZCA1gQ491NQGu6gdCrDkdIgCZAACZAACWhNgHM/BaXhDkqnMhwpDZIACZAACZCA1gQ491NQGu6gdCrDkdIgCZAACZAACWhNgHM/BaXhDkqnMhwpDZIACZAACZCA1gQ491NQGu6gdCrDkdIgCZAACZAACWhNgHM/BaXhDkqnMhwpDZIACZAACZCA1gQ491NQRnDQ58+fY/jw4Zg9ezZu376NatWqYdq0afD393fYkelUDqPigyRAAiRAAiQQLwhw7qegjODIIiZFQM6dOxfZs2fHgAEDEBoaihMnTiBx4sQOOT2dyiFMfIgESIAESIAE4g0Bzv0UlFYCjx8/RoYMGfD555+jW7du6ud37txBtmzZMGfOHLRp08Yhx6dTOYSJD5EACZAACZBAvCHAuZ+C0kpg7969CAgIwKlTp1CwYEHrz2Xbu2TJkmrl0pFCp3KEEp8hARIgARIggfhDgHM/BaWVwPLly9GyZUs8ePAAyZIls/68devW6merV692yPPpVA5h4kMkQAIkQAIkEG8IcO6noLQSWLBgAdq3b49nz57Bx8fH+vMOHTrg8uXL2LRpU6SOf+/ePcj/LOXSpUuoUqUKZMUza9as8ebDwo6QAAmQAAmQAAlETuDKlSuoWLEizp07hzx58ng1pgQvXrx44c0Eli1bhlatWkW6Qvno0SOsXLkyUjyBgYHqZjgLCZAACZAACZCAdxOQxaQKFSp4NQSvF5SWM5RnzpxB/vz5rc4gZyhLlSqFqVOnOrRC+fDhQ8gqZd68eZEwYUJDncryFxBXPw3FajVGvuZwtbVKxuYyJl/yNZeA+dY91YefPn2KGzduoESJEkiaNKn5oDSuwesFpaxCZsqUCRMnTsS7776rhsqVW95mjjHPaJhJFyBfc/mKdTI2lzH5kq+5BMy3Th82n7HZNXi9oBTAgwcPxsyZM1WYIDkD0b9/f5w/fx7Hjh1zOA6lmQPFD5qZdCl2zKUbZp0+bC5l8iVfcwmYb50+bD5js2ugoATUhZxBgwapwOb//PMPatSooba6dTlgyw+auR8D8jWXLwUl+ZpPwNwa+B1hLl9+R5jPNzZqoKCMDcpu1iG3yWVLvm/fvkidOrWb1vi6PQHyNd8nyNhcxuRLvuYSMN86fdh8xmbXQEFpNmHaJwESIAESIAESIIF4ToCCMp4PMLtHAiRAAiRAAiRAAmYToKA0mzDtkwAJkAAJkAAJkEA8J0BBGc8HmN0jARIgARIgARIgAbMJUFCaTdgN+8+fP1fZeGbPno3bt29Dgq1PmzYN/v7+blj13lcvXLgQ6c39WbNmoUuXLjh8+DD69OmD/fv3I3369Ojduzf69evnvcCc6PmoUaNUmtItW7ZY34qJJ/3bCcAAImPcqVMnzJs3L4Kh7NmzqzBNUsg4esa3bt1SET5Wr16tUumWLFkS48aNU9+1UujDzvmo/dMx8aX/usdXt7cpKHUbEZv2iJgUASnhjGSSGDBgAEJDQ3HixAkt4mNqjC7Spq1atQqtW7dWDBMkSGB9xs/PT6XeLFy4MJo3b65u0wcFBaFHjx6Kv3zpsURNYNKkSYqZhNuyCMqbN2/GyJP+7bhXRcZY3i5XrhwaNWqEDz74wGrM19cXGTNmVP8m4+gZN2jQANevX8eUKVMUMwkXJ3/AHzx4UP07pu8E8nWdr7Cl/zr+HeAJT1JQajpKjx8/RoYMGfD555+jW7duqpW6ZfDRFF2UzRozZgwWL16sVh3sy9ixY9VkIgHtLakzZeVi+fLlCA4O9rSuxkp7//jjD7Wyu337duTMmROZM2e2CsqYeNK/HRui6BhL/NyUKVNi0aJF6g8h+0LG0TOWdLsFChTAzp07UaVKFfXwixcvULBgQbRt2xbJkiWL9juBfN3jO2zYMPqvY18DHvMUBaWmQ2XJMX7q1Cn1BWcpshUj2zKycsbiHIE2bdrAx8cH33333b9ebNKkCdKkSRPhd7KFW79+fVy7dk2l52SJSEBWfEXMyFbsiBEjlBi3rFDGxFOeDQgIAP07eq+KjrH8oVOkSJF/MbRY5HdI9GzlD/Rdu3ahbt26SJIkifVhOVIkq76ykxHddwJ92D2+vXr1ov/Gs0mFglLTAZWVsZYtW6qtWPlL2VJky1Z+Jmd+WJwjUKxYMWTNmhWSv/306dNqdWLIkCFo2LChEumNGzfG+PHjrUZPnjwJeWffvn0oX768c5V52dPvvPNOBEEZE8+LFy/Sv530EXvGS5YswZtvvql2MNatW6f+WBIhLwJfjnHwO8RJwACEqXzH/vTTTyolb3TfCfRh9/g+fPiQ/us8Qq3foKDUdHgWLFiA9u3bq7SQMlFYSocOHXD58mV1AYLFcQKyPZUiRQqUKVMGEyZMUFstwnjy5Mn4+eef0bVrV7Rr106ttFmKrFDkz59fbelaDuk7XqN3PWkvdmSVJzqesrpD/3bOR+wZy5bh6NGjIUc5REiGhISoS2Ry/GDz5s1qtZ2MHWcsW9+yMikrlj/++KO6/EgfdpxfTE/a86X/xkTM835PQanpmC1btgytWrWKdIVSVthWrlypacv1bdb9+/fV+Ujb7S2ZQOSCjtyKlUk5shVKOaAvQpQlagL2YqdEiRLR8hSxTv92zqPsGct5v7t376ptWUuRy2SVK1dWl8rEp8nYMcbyfSrisVKlSmr3R3aF6MOOsXPkqcj40n8dIedZz1BQajpelvNPcnBcVsksRVbKSpUqpQ6Ls7hPoH///li/fr1a1ZFQQfPnz7ca5RlKx/naix0R59HxtJw/o3+7zjiyN+WPJll9X7p0qfJpOadKxtEzlhveEi6sRYsWatfC8gcnfdhx34zuyaj40n+N4auTFQpKnUbDpi2yCikXQSZOnIh3331X/Ya3vF0frKNHj6qbnCIebbeva9eurW4nly5dGjNmzMDZs2chYVekDBw4ECtWrOAtbwew2wtKieUXHU/6twNQ7R6xZywrahKeacOGDdYn5VKU+LSc/82XLx+/Q2LAPH36dBUeTGLOfvHFFxGOF9GHnfdR+zei40v/dZ+vbhYoKHUbEZv2yKHwmTNnYs6cOSogt6ymycrOsWPHGIfSyXGTAM8iKGUFR77kJCTTV199pW7Ly6UbEZUSF61p06Yq3qesEHfv3l2Joo4dOzpZm/c9bi92JLZfTDzp3875iT1j2ZoVfx05ciQkgoFcNOvZs6fa8l64cKEyTsZRMxZexYsXx6uvvvqvqBmy5S1/9NCHnfNR26dj4itn0+m/rvPV8U0KSh1H5WWb5EKOxEKUwOb//POPChwtW90iLlmcJ3Djxg18+umnWLt2rVrtLVu2rMqKUb16dWVMhKWsVBw6dEjdBv/4448hoS1YYiZgL3Yc4Un/jpmr7RORMZaz1nIxR0IIyVlKWfWRW95JkyZVr5Jx1IzlMpMI7siK/BEpGYhi+k4gX/f40n+d+w7Q/WkKSt1HiO0jARIgARIgARIgAc0JUFBqPkBsHgmQAAmQAAmQAAnoToCCUvcRYvtIgARIgARIgARIQHMCFJSaDxCbRwIkQAIkQAIkQAK6E6Cg1H2E2D4SIAESIAESIAES0JwABaXmA8TmkQAJ6ENAIizIbevAwEBTGmWJI3nu3LkoozmY3QZTOkajJEAC8Z4ABWW8H2J2kARIwCgCEnpKYhRKNhoziuScv3XrFjJmzGgNsG9fDwWlGeRpkwRIwF0CFJTuEuT7JEACJBCLBCgoYxE2qyIBEnCYAAWlw6j4IAmQgKcQSJAggQpaLbnZJePJ1q1bkTdvXgwZMkTla757967KkjJixAg0aNDA2q2NGzeqlJuW1IUS3F5Sn1q2oO3F3Jo1a1SmmuPHjyN16tRo27atCjRuCSwu7ZCMTIsXL8aOHTuQLl06FSxf6ois2G95Szsl2P7KlStVdixJdDBp0iRTt909ZYzZThIgAb0IUFDqNR5sDQmQgAEERMhJes1169bh6dOnqFSpksoic+LECSXIcuTIgVWrVqnMSZKv/ZVXXsHhw4dRoUIFfPTRR0pEyr8l/ebt27cjFZQ//vgjWrZsqc5Ttm7dGiEhIer5ihUrQjKASJF2+Pn5YcqUKernInAlk822bdusGZpsu2svKBs2bIiLFy8qUZoqVSqVvWnz5s0YNmyYaec4DcBPEyRAAl5IgILSCwedXSaB+E5AhJwIw4kTJ6qunjlzBgUKFFCp9MqXL2/tvqTYu3DhAkTIyf+X/MO7d++2/v7LL79UK4SRrVAGBAQgZ86cWLp0qfV5EamSn1hWOIsUKaIEZZ8+fZSItRRZpezfv3+kq5S2gtKSS3rTpk2oW7euev3atWvInTu3EsJmXQyK777B/pEACZhDgILSHK60SgIkEIcERMhNmzZNrRhKWbJkiVpFTJEiRYRWPXnyROXAFqFWokQJ1K9f3ypC5UFZpSxTpkykgjJ58uSQfNAffvih1aZsUYu977//Hm+++aYSlDNmzEDXrl2tz8i2eadOndQqo32xFZT79+/HG2+8obbnZTvdUooWLar6QkEZhw7GqkmABP5FgIKSTkECJBDvCIiQmzt3rjprKOWHH35AmzZtcPDgQbV1bFt8fX3V+UoRjjVr1oywmhidoJTb3uPGjVMrkJYiN7TTp0+vtrxbtGihBKVtO+S56C7VRCYoZctdRKqllCpVCq+//joFZbzzWnaIBDybAAWlZ48fW08CJBAJAXshJ2cn5RLO6tWr1XlJS5GLOz4+PupijYjP4OBgBAUFWX8/c+ZMdOvWLdIVSjkTmStXrghb3nJ5pnnz5hG2vF0VlM+ePYO/v3+ENt+5cwfZs2dXW+ZcoaTrkwAJ6ESAglKn0WBbSIAEDCEQ2crga6+9prawp06dqsTl8uXLMWDAAHz99ddqC1puapcuXRr9+vVD586dlSh8//33IbEnz58/r84u2q4uyja6bGsPHz5c/VfOX8oWu9iQs5RS3FmhlLpka1vOfc6aNQtZsmRR5y7lotFnn31GQWmIp9AICZCAUQT+D1k6bHeQKE16AAAAAElFTkSuQmCC\" width=\"599.9999869953505\">"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "pb_num_dict = {_k:len(_v.kept_probes) for _k,_v in filtered_pb_dict.items()}\n",
    "\n",
    "pb_nums = np.array(list(pb_num_dict.values()))\n",
    "plt.figure(figsize=(6,3), dpi=100)\n",
    "plt.plot(pb_nums)\n",
    "#plt.plot(np.ones(len(pb_nums))*20)\n",
    "plt.ylabel(f\"num of probes\", labelpad=1)\n",
    "plt.xlabel(f\"region id\", labelpad=1)\n",
    "plt.ylim([0, num_probes_kept*1.1])\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "young-monthly",
   "metadata": {},
   "source": [
    "# 3. Load and asssemble probes"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "alternate-kitchen",
   "metadata": {},
   "source": [
    "## 3.1 design readout mapping dict (region_to_readout)\n",
    "\n",
    "1. parallel tracing\n",
    "2. sequential tracing"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 73,
   "id": "independent-melbourne",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "36637"
      ]
     },
     "execution_count": 73,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "np.sum(pb_nums)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 127,
   "id": "unlimited-concrete",
   "metadata": {},
   "outputs": [],
   "source": [
    "sorted_gene_list = []\n",
    "genes = np.unique([_nm.split('-'+(_nm.split('-')[-1]))[0] for _nm in filtered_pb_dict ])\n",
    "for _gene in genes:\n",
    "    _gene_regs = [_nm for _nm in filtered_pb_dict if _gene in _nm]\n",
    "    _gene_regs = [_nm for _nm in sorted(_gene_regs, key=lambda v:int(v.split('-')[-1]))]\n",
    "    sorted_gene_list.extend(_gene_regs)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 129,
   "id": "solved-lightning",
   "metadata": {},
   "outputs": [],
   "source": [
    "gene_2_readout_dict = {}\n",
    "for _uid, _gene in enumerate(sorted_gene_list):\n",
    "    _rds = [f'u{_uid}'] * 3\n",
    "    #_rds.append(f\"c{_gene.split('-')[-1]}\")\n",
    "    gene_2_readout_dict[_gene] = _rds"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 130,
   "id": "wound-separate",
   "metadata": {},
   "outputs": [],
   "source": [
    "# save\n",
    "pickle.dump(gene_2_readout_dict, open(os.path.join(library_folder, 'gene_2_readout.pkl'), 'wb'))"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "surrounded-tunnel",
   "metadata": {},
   "source": [
    "## 3.2 loading components"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "academic-height",
   "metadata": {},
   "source": [
    "### primers\n",
    "pick primer set 6,11"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 131,
   "id": "constitutional-welsh",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Picked primer: ID: W1B05_primer_16\n",
      "Name: W1B05_primer_16\n",
      "Description: W1B05_primer_16\n",
      "Number of features: 0\n",
      "Seq('CGGGTTTCGTTGCGCACACC')\n",
      "- Picked primer: ID: W1B04_primer_15\n",
      "Name: W1B04_primer_15\n",
      "Description: W1B04_primer_15\n",
      "Number of features: 0\n",
      "Seq('TAATACGACTCACTATAGGGCTTGTGCATCGCGCCAAAGA')\n"
     ]
    }
   ],
   "source": [
    "primer_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Primers'\n",
    "primers = library_tools.assemble.load_primers([16,15], _primer_folder=primer_folder)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "subsequent-settle",
   "metadata": {},
   "source": [
    "### readouts"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 132,
   "id": "excellent-charity",
   "metadata": {},
   "outputs": [],
   "source": [
    "# summarize total readout usage\n",
    "gene_2_readout_dict = pickle.load(open(os.path.join(library_folder, 'gene_2_readout.pkl'), 'rb'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 133,
   "id": "intermediate-productivity",
   "metadata": {},
   "outputs": [],
   "source": [
    "# collect types\n",
    "readout_usage_by_type = {}\n",
    "for _gene, _infos in gene_2_readout_dict.items():\n",
    "    for _info in _infos:\n",
    "        if _info[0] not in readout_usage_by_type:\n",
    "            readout_usage_by_type[_info[0]] = []\n",
    "        if int(_info[1:]) not in readout_usage_by_type[_info[0]]:\n",
    "            readout_usage_by_type[_info[0]].append(int(_info[1:]))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 134,
   "id": "surprising-collective",
   "metadata": {},
   "outputs": [],
   "source": [
    "readout_usage_dict = {_type:len(_v) for _type,_v in readout_usage_by_type.items()}\n",
    "readout_dict ={}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 135,
   "id": "auburn-organizer",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'u': 264}"
      ]
     },
     "execution_count": 135,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "readout_usage_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 136,
   "id": "entire-timber",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "save\n"
     ]
    }
   ],
   "source": [
    "reload(library_tools.assemble)\n",
    "overwrite_readout_usage = True\n",
    "readout_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Readouts'\n",
    "readout_start = 0\n",
    "num_color = 2\n",
    "\n",
    "unique_readouts = library_tools.assemble.load_readouts(250, \n",
    "                                                       _type='other_snr_after_630', \n",
    "                                                       _readout_folder=readout_folder, \n",
    "                                                       _num_colors=num_color, # 750, 647\n",
    "                                                       _start_id=readout_start,\n",
    "                                                       )\n",
    "append_unique_readouts = library_tools.assemble.load_readouts(readout_usage_dict['u']-250, \n",
    "                                                       _type='merfish_readouts', \n",
    "                                                       _readout_folder=readout_folder, \n",
    "                                                       _num_colors=num_color, # 750, 647\n",
    "                                                       _start_id=readout_start,\n",
    "                                                       )\n",
    "readout_dict['u'] = unique_readouts + append_unique_readouts\n",
    "\n",
    "# save\n",
    "readout_usage_file = os.path.join(library_folder, 'readout_usage.pkl')\n",
    "if not os.path.isfile(readout_usage_file) or overwrite_readout_usage:\n",
    "    print('save')\n",
    "    pickle.dump(readout_dict, open(readout_usage_file, 'wb'))"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "killing-reasoning",
   "metadata": {},
   "source": [
    "## 3.3 assemble probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 161,
   "id": "intermediate-international",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Assemble probes by given target sequences, readouts and primers.\n",
      "-- included readout types: ['u']\n",
      "--- assemblying 149 probes in region: Camk1-1\n",
      "--- assemblying 140 probes in region: Camk1-10\n",
      "--- assemblying 150 probes in region: Camk1-11\n",
      "--- assemblying 109 probes in region: Camk1-12\n",
      "--- assemblying 110 probes in region: Camk1-13\n",
      "--- assemblying 150 probes in region: Camk1-14\n",
      "--- assemblying 150 probes in region: Camk1-15\n",
      "--- assemblying 150 probes in region: Camk1-16\n",
      "--- assemblying 150 probes in region: Camk1-17\n",
      "--- assemblying 150 probes in region: Camk1-18\n",
      "--- assemblying 148 probes in region: Camk1-19\n",
      "--- assemblying 140 probes in region: Camk1-2\n",
      "--- assemblying 143 probes in region: Camk1-20\n",
      "--- assemblying 136 probes in region: Camk1-21\n",
      "--- assemblying 141 probes in region: Camk1-22\n",
      "--- assemblying 136 probes in region: Camk1-23\n",
      "--- assemblying 150 probes in region: Camk1-24\n",
      "--- assemblying 150 probes in region: Camk1-25\n",
      "--- assemblying 141 probes in region: Camk1-26\n",
      "--- assemblying 147 probes in region: Camk1-27\n",
      "--- assemblying 113 probes in region: Camk1-28\n",
      "--- assemblying 150 probes in region: Camk1-29\n",
      "--- assemblying 150 probes in region: Camk1-3\n",
      "--- assemblying 138 probes in region: Camk1-30\n",
      "--- assemblying 103 probes in region: Camk1-31\n",
      "--- assemblying 132 probes in region: Camk1-32\n",
      "--- assemblying 135 probes in region: Camk1-33\n",
      "--- assemblying 126 probes in region: Camk1-34\n",
      "--- assemblying 131 probes in region: Camk1-35\n",
      "--- assemblying 133 probes in region: Camk1-36\n",
      "--- assemblying 131 probes in region: Camk1-37\n",
      "--- assemblying 142 probes in region: Camk1-38\n",
      "--- assemblying 142 probes in region: Camk1-39\n",
      "--- assemblying 150 probes in region: Camk1-4\n",
      "--- assemblying 136 probes in region: Camk1-40\n",
      "--- assemblying 121 probes in region: Camk1-41\n",
      "--- assemblying 139 probes in region: Camk1-42\n",
      "--- assemblying 132 probes in region: Camk1-43\n",
      "--- assemblying 133 probes in region: Camk1-44\n",
      "--- assemblying 144 probes in region: Camk1-45\n",
      "--- assemblying 139 probes in region: Camk1-46\n",
      "--- assemblying 141 probes in region: Camk1-47\n",
      "--- assemblying 145 probes in region: Camk1-48\n",
      "--- assemblying 150 probes in region: Camk1-49\n",
      "--- assemblying 150 probes in region: Camk1-5\n",
      "--- assemblying 149 probes in region: Camk1-50\n",
      "--- assemblying 150 probes in region: Camk1-51\n",
      "--- assemblying 150 probes in region: Camk1-52\n",
      "--- assemblying 150 probes in region: Camk1-53\n",
      "--- assemblying 150 probes in region: Camk1-54\n",
      "--- assemblying 150 probes in region: Camk1-6\n",
      "--- assemblying 139 probes in region: Camk1-7\n",
      "--- assemblying 150 probes in region: Camk1-8\n",
      "--- assemblying 150 probes in region: Camk1-9\n",
      "--- assemblying 121 probes in region: Hoxd10-1\n",
      "--- assemblying 150 probes in region: Hoxd10-10\n",
      "--- assemblying 135 probes in region: Hoxd10-100\n",
      "--- assemblying 138 probes in region: Hoxd10-101\n",
      "--- assemblying 144 probes in region: Hoxd10-102\n",
      "--- assemblying 118 probes in region: Hoxd10-103\n",
      "--- assemblying 118 probes in region: Hoxd10-104\n",
      "--- assemblying 73 probes in region: Hoxd10-105\n",
      "--- assemblying 108 probes in region: Hoxd10-106\n",
      "--- assemblying 126 probes in region: Hoxd10-107\n",
      "--- assemblying 107 probes in region: Hoxd10-108\n",
      "--- assemblying 132 probes in region: Hoxd10-109\n",
      "--- assemblying 150 probes in region: Hoxd10-11\n",
      "--- assemblying 142 probes in region: Hoxd10-110\n",
      "--- assemblying 150 probes in region: Hoxd10-111\n",
      "--- assemblying 150 probes in region: Hoxd10-112\n",
      "--- assemblying 129 probes in region: Hoxd10-113\n",
      "--- assemblying 129 probes in region: Hoxd10-114\n",
      "--- assemblying 141 probes in region: Hoxd10-115\n",
      "--- assemblying 113 probes in region: Hoxd10-116\n",
      "--- assemblying 150 probes in region: Hoxd10-117\n",
      "--- assemblying 150 probes in region: Hoxd10-118\n",
      "--- assemblying 147 probes in region: Hoxd10-12\n",
      "--- assemblying 150 probes in region: Hoxd10-13\n",
      "--- assemblying 150 probes in region: Hoxd10-14\n",
      "--- assemblying 150 probes in region: Hoxd10-15\n",
      "--- assemblying 150 probes in region: Hoxd10-16\n",
      "--- assemblying 150 probes in region: Hoxd10-17\n",
      "--- assemblying 144 probes in region: Hoxd10-18\n",
      "--- assemblying 128 probes in region: Hoxd10-19\n",
      "--- assemblying 84 probes in region: Hoxd10-2\n",
      "--- assemblying 127 probes in region: Hoxd10-20\n",
      "--- assemblying 116 probes in region: Hoxd10-21\n",
      "--- assemblying 120 probes in region: Hoxd10-22\n",
      "--- assemblying 119 probes in region: Hoxd10-23\n",
      "--- assemblying 123 probes in region: Hoxd10-24\n",
      "--- assemblying 118 probes in region: Hoxd10-25\n",
      "--- assemblying 118 probes in region: Hoxd10-26\n",
      "--- assemblying 105 probes in region: Hoxd10-27\n",
      "--- assemblying 109 probes in region: Hoxd10-28\n",
      "--- assemblying 125 probes in region: Hoxd10-29\n",
      "--- assemblying 126 probes in region: Hoxd10-3\n",
      "--- assemblying 150 probes in region: Hoxd10-30\n",
      "--- assemblying 148 probes in region: Hoxd10-31\n",
      "--- assemblying 119 probes in region: Hoxd10-32\n",
      "--- assemblying 130 probes in region: Hoxd10-33\n",
      "--- assemblying 150 probes in region: Hoxd10-34\n",
      "--- assemblying 149 probes in region: Hoxd10-35\n",
      "--- assemblying 146 probes in region: Hoxd10-36\n",
      "--- assemblying 150 probes in region: Hoxd10-37\n",
      "--- assemblying 150 probes in region: Hoxd10-38\n",
      "--- assemblying 150 probes in region: Hoxd10-39\n",
      "--- assemblying 118 probes in region: Hoxd10-4\n",
      "--- assemblying 124 probes in region: Hoxd10-40\n",
      "--- assemblying 150 probes in region: Hoxd10-41\n",
      "--- assemblying 150 probes in region: Hoxd10-42\n",
      "--- assemblying 150 probes in region: Hoxd10-43\n",
      "--- assemblying 150 probes in region: Hoxd10-44\n",
      "--- assemblying 150 probes in region: Hoxd10-45\n",
      "--- assemblying 119 probes in region: Hoxd10-46\n",
      "--- assemblying 150 probes in region: Hoxd10-47\n",
      "--- assemblying 150 probes in region: Hoxd10-48\n",
      "--- assemblying 150 probes in region: Hoxd10-49\n",
      "--- assemblying 127 probes in region: Hoxd10-5\n",
      "--- assemblying 150 probes in region: Hoxd10-50\n",
      "--- assemblying 150 probes in region: Hoxd10-51\n",
      "--- assemblying 143 probes in region: Hoxd10-52\n",
      "--- assemblying 150 probes in region: Hoxd10-53\n",
      "--- assemblying 134 probes in region: Hoxd10-54\n",
      "--- assemblying 150 probes in region: Hoxd10-55\n",
      "--- assemblying 150 probes in region: Hoxd10-56\n",
      "--- assemblying 150 probes in region: Hoxd10-57\n",
      "--- assemblying 150 probes in region: Hoxd10-58\n",
      "--- assemblying 150 probes in region: Hoxd10-59\n",
      "--- assemblying 150 probes in region: Hoxd10-6\n",
      "--- assemblying 126 probes in region: Hoxd10-60\n",
      "--- assemblying 144 probes in region: Hoxd10-61\n",
      "--- assemblying 134 probes in region: Hoxd10-62\n",
      "--- assemblying 150 probes in region: Hoxd10-63\n",
      "--- assemblying 80 probes in region: Hoxd10-64\n",
      "--- assemblying 133 probes in region: Hoxd10-65\n",
      "--- assemblying 150 probes in region: Hoxd10-66\n",
      "--- assemblying 150 probes in region: Hoxd10-67\n",
      "--- assemblying 150 probes in region: Hoxd10-68\n",
      "--- assemblying 141 probes in region: Hoxd10-69\n",
      "--- assemblying 150 probes in region: Hoxd10-7\n",
      "--- assemblying 150 probes in region: Hoxd10-70\n",
      "--- assemblying 143 probes in region: Hoxd10-71\n",
      "--- assemblying 150 probes in region: Hoxd10-72\n",
      "--- assemblying 150 probes in region: Hoxd10-73\n",
      "--- assemblying 145 probes in region: Hoxd10-74\n",
      "--- assemblying 117 probes in region: Hoxd10-75\n",
      "--- assemblying 119 probes in region: Hoxd10-76\n",
      "--- assemblying 150 probes in region: Hoxd10-77\n",
      "--- assemblying 150 probes in region: Hoxd10-78\n",
      "--- assemblying 150 probes in region: Hoxd10-79\n",
      "--- assemblying 150 probes in region: Hoxd10-8\n",
      "--- assemblying 143 probes in region: Hoxd10-80\n",
      "--- assemblying 150 probes in region: Hoxd10-81\n",
      "--- assemblying 150 probes in region: Hoxd10-82\n",
      "--- assemblying 150 probes in region: Hoxd10-83\n",
      "--- assemblying 134 probes in region: Hoxd10-84\n",
      "--- assemblying 148 probes in region: Hoxd10-85\n",
      "--- assemblying 150 probes in region: Hoxd10-86\n",
      "--- assemblying 150 probes in region: Hoxd10-87\n",
      "--- assemblying 150 probes in region: Hoxd10-88\n",
      "--- assemblying 142 probes in region: Hoxd10-89\n",
      "--- assemblying 150 probes in region: Hoxd10-9\n",
      "--- assemblying 150 probes in region: Hoxd10-90\n",
      "--- assemblying 109 probes in region: Hoxd10-91\n",
      "--- assemblying 106 probes in region: Hoxd10-92\n",
      "--- assemblying 150 probes in region: Hoxd10-93\n",
      "--- assemblying 134 probes in region: Hoxd10-94\n",
      "--- assemblying 143 probes in region: Hoxd10-95\n",
      "--- assemblying 132 probes in region: Hoxd10-96\n",
      "--- assemblying 150 probes in region: Hoxd10-97\n",
      "--- assemblying 150 probes in region: Hoxd10-98\n",
      "--- assemblying 134 probes in region: Hoxd10-99\n",
      "--- assemblying 150 probes in region: Spp2-1\n",
      "--- assemblying 149 probes in region: Spp2-10\n",
      "--- assemblying 150 probes in region: Spp2-11\n",
      "--- assemblying 150 probes in region: Spp2-12\n",
      "--- assemblying 139 probes in region: Spp2-13\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- assemblying 150 probes in region: Spp2-14\n",
      "--- assemblying 140 probes in region: Spp2-15\n",
      "--- assemblying 148 probes in region: Spp2-16\n",
      "--- assemblying 150 probes in region: Spp2-17\n",
      "--- assemblying 147 probes in region: Spp2-18\n",
      "--- assemblying 150 probes in region: Spp2-19\n",
      "--- assemblying 150 probes in region: Spp2-2\n",
      "--- assemblying 150 probes in region: Spp2-20\n",
      "--- assemblying 150 probes in region: Spp2-21\n",
      "--- assemblying 127 probes in region: Spp2-22\n",
      "--- assemblying 135 probes in region: Spp2-23\n",
      "--- assemblying 150 probes in region: Spp2-24\n",
      "--- assemblying 120 probes in region: Spp2-25\n",
      "--- assemblying 150 probes in region: Spp2-26\n",
      "--- assemblying 99 probes in region: Spp2-27\n",
      "--- assemblying 145 probes in region: Spp2-28\n",
      "--- assemblying 136 probes in region: Spp2-29\n",
      "--- assemblying 145 probes in region: Spp2-3\n",
      "--- assemblying 93 probes in region: Spp2-30\n",
      "--- assemblying 115 probes in region: Spp2-31\n",
      "--- assemblying 150 probes in region: Spp2-32\n",
      "--- assemblying 139 probes in region: Spp2-33\n",
      "--- assemblying 144 probes in region: Spp2-34\n",
      "--- assemblying 150 probes in region: Spp2-35\n",
      "--- assemblying 150 probes in region: Spp2-36\n",
      "--- assemblying 134 probes in region: Spp2-37\n",
      "--- assemblying 150 probes in region: Spp2-38\n",
      "--- assemblying 150 probes in region: Spp2-39\n",
      "--- assemblying 147 probes in region: Spp2-4\n",
      "--- assemblying 150 probes in region: Spp2-40\n",
      "--- assemblying 150 probes in region: Spp2-41\n",
      "--- assemblying 144 probes in region: Spp2-42\n",
      "--- assemblying 131 probes in region: Spp2-43\n",
      "--- assemblying 150 probes in region: Spp2-44\n",
      "--- assemblying 150 probes in region: Spp2-45\n",
      "--- assemblying 149 probes in region: Spp2-46\n",
      "--- assemblying 142 probes in region: Spp2-47\n",
      "--- assemblying 143 probes in region: Spp2-48\n",
      "--- assemblying 139 probes in region: Spp2-49\n",
      "--- assemblying 150 probes in region: Spp2-5\n",
      "--- assemblying 149 probes in region: Spp2-50\n",
      "--- assemblying 150 probes in region: Spp2-51\n",
      "--- assemblying 144 probes in region: Spp2-52\n",
      "--- assemblying 53 probes in region: Spp2-53\n",
      "--- assemblying 150 probes in region: Spp2-54\n",
      "--- assemblying 87 probes in region: Spp2-55\n",
      "--- assemblying 126 probes in region: Spp2-56\n",
      "--- assemblying 149 probes in region: Spp2-57\n",
      "--- assemblying 146 probes in region: Spp2-58\n",
      "--- assemblying 120 probes in region: Spp2-59\n",
      "--- assemblying 150 probes in region: Spp2-6\n",
      "--- assemblying 139 probes in region: Spp2-60\n",
      "--- assemblying 147 probes in region: Spp2-61\n",
      "--- assemblying 147 probes in region: Spp2-62\n",
      "--- assemblying 144 probes in region: Spp2-63\n",
      "--- assemblying 150 probes in region: Spp2-64\n",
      "--- assemblying 131 probes in region: Spp2-65\n",
      "--- assemblying 150 probes in region: Spp2-66\n",
      "--- assemblying 125 probes in region: Spp2-67\n",
      "--- assemblying 140 probes in region: Spp2-68\n",
      "--- assemblying 150 probes in region: Spp2-69\n",
      "--- assemblying 150 probes in region: Spp2-7\n",
      "--- assemblying 133 probes in region: Spp2-70\n",
      "--- assemblying 150 probes in region: Spp2-71\n",
      "--- assemblying 142 probes in region: Spp2-72\n",
      "--- assemblying 143 probes in region: Spp2-73\n",
      "--- assemblying 147 probes in region: Spp2-74\n",
      "--- assemblying 150 probes in region: Spp2-75\n",
      "--- assemblying 132 probes in region: Spp2-76\n",
      "--- assemblying 150 probes in region: Spp2-77\n",
      "--- assemblying 150 probes in region: Spp2-78\n",
      "--- assemblying 150 probes in region: Spp2-79\n",
      "--- assemblying 148 probes in region: Spp2-8\n",
      "--- assemblying 83 probes in region: Spp2-80\n",
      "--- assemblying 150 probes in region: Spp2-81\n",
      "--- assemblying 150 probes in region: Spp2-82\n",
      "--- assemblying 150 probes in region: Spp2-83\n",
      "--- assemblying 150 probes in region: Spp2-84\n",
      "--- assemblying 150 probes in region: Spp2-85\n",
      "--- assemblying 141 probes in region: Spp2-86\n",
      "--- assemblying 128 probes in region: Spp2-87\n",
      "--- assemblying 144 probes in region: Spp2-88\n",
      "--- assemblying 150 probes in region: Spp2-89\n",
      "--- assemblying 150 probes in region: Spp2-9\n",
      "--- assemblying 88 probes in region: Spp2-90\n",
      "--- assemblying 130 probes in region: Spp2-91\n",
      "--- assemblying 50 probes in region: Spp2-92\n",
      "-- 36637 probes assembled in total.\n",
      "-- saving 36637 probes into file:\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\candidate_probes.fasta\n",
      "-- saving readout_summary into file:\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\readout_summary.pkl\n"
     ]
    }
   ],
   "source": [
    "reload(library_tools.assemble)\n",
    "from ImageAnalysis3.library_tools.assemble import Assemble_probes\n",
    "\n",
    "cand_probes, readout_summary = Assemble_probes(library_folder, filtered_pb_dict, \n",
    "                                               gene_2_readout_dict, readout_dict, primers, \n",
    "                                               primer_len=20, \n",
    "                                               num_readout_per_probe=3, \n",
    "                                               unique_readout_per_probe=True,\n",
    "                                               rc_targets=False, overwrite=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "dietary-juice",
   "metadata": {},
   "source": [
    "# 4.Check quality"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "printable-cincinnati",
   "metadata": {},
   "source": [
    "## 4.1 Load probes\n",
    "\n",
    "Checking procedures could restart from here"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 139,
   "id": "promising-circuit",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Total probe loaded: 36637\n"
     ]
    }
   ],
   "source": [
    "# folder for this library\n",
    "# candidate full-length probe filename\n",
    "candidate_basename = 'candidate_probes.fasta' # old version\n",
    "# load full probes\n",
    "full_records = []\n",
    "with open(os.path.join(library_folder, candidate_basename), 'r') as handle:\n",
    "    for record in SeqIO.parse(handle, \"fasta\"):\n",
    "        full_records.append(record)\n",
    "\n",
    "print(f\"Total probe loaded: {len(full_records)}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 140,
   "id": "animated-calendar",
   "metadata": {},
   "outputs": [],
   "source": [
    "num_readout_per_probe = 3\n",
    "primer_len=20\n",
    "target_len=40\n",
    "min_num_probes = 50"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "adapted-blank",
   "metadata": {},
   "source": [
    "### 4.1.1 check primer usage"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 141,
   "id": "functional-small",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- Checking primer usage, total probes: 36637\n",
      "True\n"
     ]
    }
   ],
   "source": [
    "import ImageAnalysis3.library_tools.quality_check as check\n",
    "reload(check)\n",
    "primer_check = check._check_primer_usage(full_records, primers[0], primers[1], primer_len=primer_len)\n",
    "print(primer_check)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "precious-robert",
   "metadata": {},
   "source": [
    "### 4.1.2 check region probe number"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 142,
   "id": "different-creator",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "gene: Camk1-1:- -> 149\n",
      "gene: Camk1-10:- -> 140\n",
      "gene: Camk1-11:- -> 150\n",
      "gene: Camk1-12:- -> 109\n",
      "gene: Camk1-13:- -> 110\n",
      "gene: Camk1-14:- -> 150\n",
      "gene: Camk1-15:- -> 150\n",
      "gene: Camk1-16:- -> 150\n",
      "gene: Camk1-17:- -> 150\n",
      "gene: Camk1-18:- -> 150\n",
      "gene: Camk1-19:- -> 148\n",
      "gene: Camk1-2:- -> 140\n",
      "gene: Camk1-20:- -> 143\n",
      "gene: Camk1-21:- -> 136\n",
      "gene: Camk1-22:- -> 141\n",
      "gene: Camk1-23:- -> 136\n",
      "gene: Camk1-24:- -> 150\n",
      "gene: Camk1-25:- -> 150\n",
      "gene: Camk1-26:- -> 141\n",
      "gene: Camk1-27:- -> 147\n",
      "gene: Camk1-28:- -> 113\n",
      "gene: Camk1-29:- -> 150\n",
      "gene: Camk1-3:- -> 150\n",
      "gene: Camk1-30:- -> 138\n",
      "gene: Camk1-31:- -> 103\n",
      "gene: Camk1-32:- -> 132\n",
      "gene: Camk1-33:- -> 135\n",
      "gene: Camk1-34:- -> 126\n",
      "gene: Camk1-35:- -> 131\n",
      "gene: Camk1-36:- -> 133\n",
      "gene: Camk1-37:- -> 131\n",
      "gene: Camk1-38:- -> 142\n",
      "gene: Camk1-39:- -> 142\n",
      "gene: Camk1-4:- -> 150\n",
      "gene: Camk1-40:- -> 136\n",
      "gene: Camk1-41:- -> 121\n",
      "gene: Camk1-42:- -> 139\n",
      "gene: Camk1-43:- -> 132\n",
      "gene: Camk1-44:- -> 133\n",
      "gene: Camk1-45:- -> 144\n",
      "gene: Camk1-46:- -> 139\n",
      "gene: Camk1-47:- -> 141\n",
      "gene: Camk1-48:- -> 145\n",
      "gene: Camk1-49:- -> 150\n",
      "gene: Camk1-5:- -> 150\n",
      "gene: Camk1-50:- -> 149\n",
      "gene: Camk1-51:- -> 150\n",
      "gene: Camk1-52:- -> 150\n",
      "gene: Camk1-53:- -> 150\n",
      "gene: Camk1-54:- -> 150\n",
      "gene: Camk1-6:- -> 150\n",
      "gene: Camk1-7:- -> 139\n",
      "gene: Camk1-8:- -> 150\n",
      "gene: Camk1-9:- -> 150\n",
      "gene: Hoxd10-1:+ -> 121\n",
      "gene: Hoxd10-10:+ -> 150\n",
      "gene: Hoxd10-100:+ -> 135\n",
      "gene: Hoxd10-101:+ -> 138\n",
      "gene: Hoxd10-102:+ -> 144\n",
      "gene: Hoxd10-103:+ -> 118\n",
      "gene: Hoxd10-104:+ -> 118\n",
      "gene: Hoxd10-105:+ -> 73\n",
      "gene: Hoxd10-106:+ -> 108\n",
      "gene: Hoxd10-107:+ -> 126\n",
      "gene: Hoxd10-108:+ -> 107\n",
      "gene: Hoxd10-109:+ -> 132\n",
      "gene: Hoxd10-11:+ -> 150\n",
      "gene: Hoxd10-110:+ -> 142\n",
      "gene: Hoxd10-111:+ -> 150\n",
      "gene: Hoxd10-112:+ -> 150\n",
      "gene: Hoxd10-113:+ -> 129\n",
      "gene: Hoxd10-114:+ -> 129\n",
      "gene: Hoxd10-115:+ -> 141\n",
      "gene: Hoxd10-116:+ -> 113\n",
      "gene: Hoxd10-117:+ -> 150\n",
      "gene: Hoxd10-118:+ -> 150\n",
      "gene: Hoxd10-12:+ -> 147\n",
      "gene: Hoxd10-13:+ -> 150\n",
      "gene: Hoxd10-14:+ -> 150\n",
      "gene: Hoxd10-15:+ -> 150\n",
      "gene: Hoxd10-16:+ -> 150\n",
      "gene: Hoxd10-17:+ -> 150\n",
      "gene: Hoxd10-18:+ -> 144\n",
      "gene: Hoxd10-19:+ -> 128\n",
      "gene: Hoxd10-2:+ -> 84\n",
      "gene: Hoxd10-20:+ -> 127\n",
      "gene: Hoxd10-21:+ -> 116\n",
      "gene: Hoxd10-22:+ -> 120\n",
      "gene: Hoxd10-23:+ -> 119\n",
      "gene: Hoxd10-24:+ -> 123\n",
      "gene: Hoxd10-25:+ -> 118\n",
      "gene: Hoxd10-26:+ -> 118\n",
      "gene: Hoxd10-27:+ -> 105\n",
      "gene: Hoxd10-28:+ -> 109\n",
      "gene: Hoxd10-29:+ -> 125\n",
      "gene: Hoxd10-3:+ -> 126\n",
      "gene: Hoxd10-30:+ -> 150\n",
      "gene: Hoxd10-31:+ -> 148\n",
      "gene: Hoxd10-32:+ -> 119\n",
      "gene: Hoxd10-33:+ -> 130\n",
      "gene: Hoxd10-34:+ -> 150\n",
      "gene: Hoxd10-35:+ -> 149\n",
      "gene: Hoxd10-36:+ -> 146\n",
      "gene: Hoxd10-37:+ -> 150\n",
      "gene: Hoxd10-38:+ -> 150\n",
      "gene: Hoxd10-39:+ -> 150\n",
      "gene: Hoxd10-4:+ -> 118\n",
      "gene: Hoxd10-40:+ -> 124\n",
      "gene: Hoxd10-41:+ -> 150\n",
      "gene: Hoxd10-42:+ -> 150\n",
      "gene: Hoxd10-43:+ -> 150\n",
      "gene: Hoxd10-44:+ -> 150\n",
      "gene: Hoxd10-45:+ -> 150\n",
      "gene: Hoxd10-46:+ -> 119\n",
      "gene: Hoxd10-47:+ -> 150\n",
      "gene: Hoxd10-48:+ -> 150\n",
      "gene: Hoxd10-49:+ -> 150\n",
      "gene: Hoxd10-5:+ -> 127\n",
      "gene: Hoxd10-50:+ -> 150\n",
      "gene: Hoxd10-51:+ -> 150\n",
      "gene: Hoxd10-52:+ -> 143\n",
      "gene: Hoxd10-53:+ -> 150\n",
      "gene: Hoxd10-54:+ -> 134\n",
      "gene: Hoxd10-55:+ -> 150\n",
      "gene: Hoxd10-56:+ -> 150\n",
      "gene: Hoxd10-57:+ -> 150\n",
      "gene: Hoxd10-58:+ -> 150\n",
      "gene: Hoxd10-59:+ -> 150\n",
      "gene: Hoxd10-6:+ -> 150\n",
      "gene: Hoxd10-60:+ -> 126\n",
      "gene: Hoxd10-61:+ -> 144\n",
      "gene: Hoxd10-62:+ -> 134\n",
      "gene: Hoxd10-63:+ -> 150\n",
      "gene: Hoxd10-64:+ -> 80\n",
      "gene: Hoxd10-65:+ -> 133\n",
      "gene: Hoxd10-66:+ -> 150\n",
      "gene: Hoxd10-67:+ -> 150\n",
      "gene: Hoxd10-68:+ -> 150\n",
      "gene: Hoxd10-69:+ -> 141\n",
      "gene: Hoxd10-7:+ -> 150\n",
      "gene: Hoxd10-70:+ -> 150\n",
      "gene: Hoxd10-71:+ -> 143\n",
      "gene: Hoxd10-72:+ -> 150\n",
      "gene: Hoxd10-73:+ -> 150\n",
      "gene: Hoxd10-74:+ -> 145\n",
      "gene: Hoxd10-75:+ -> 117\n",
      "gene: Hoxd10-76:+ -> 119\n",
      "gene: Hoxd10-77:+ -> 150\n",
      "gene: Hoxd10-78:+ -> 150\n",
      "gene: Hoxd10-79:+ -> 150\n",
      "gene: Hoxd10-8:+ -> 150\n",
      "gene: Hoxd10-80:+ -> 143\n",
      "gene: Hoxd10-81:+ -> 150\n",
      "gene: Hoxd10-82:+ -> 150\n",
      "gene: Hoxd10-83:+ -> 150\n",
      "gene: Hoxd10-84:+ -> 134\n",
      "gene: Hoxd10-85:+ -> 148\n",
      "gene: Hoxd10-86:+ -> 150\n",
      "gene: Hoxd10-87:+ -> 150\n",
      "gene: Hoxd10-88:+ -> 150\n",
      "gene: Hoxd10-89:+ -> 142\n",
      "gene: Hoxd10-9:+ -> 150\n",
      "gene: Hoxd10-90:+ -> 150\n",
      "gene: Hoxd10-91:+ -> 109\n",
      "gene: Hoxd10-92:+ -> 106\n",
      "gene: Hoxd10-93:+ -> 150\n",
      "gene: Hoxd10-94:+ -> 134\n",
      "gene: Hoxd10-95:+ -> 143\n",
      "gene: Hoxd10-96:+ -> 132\n",
      "gene: Hoxd10-97:+ -> 150\n",
      "gene: Hoxd10-98:+ -> 150\n",
      "gene: Hoxd10-99:+ -> 134\n",
      "gene: Spp2-1:+ -> 150\n",
      "gene: Spp2-10:+ -> 149\n",
      "gene: Spp2-11:+ -> 150\n",
      "gene: Spp2-12:+ -> 150\n",
      "gene: Spp2-13:+ -> 139\n",
      "gene: Spp2-14:+ -> 150\n",
      "gene: Spp2-15:+ -> 140\n",
      "gene: Spp2-16:+ -> 148\n",
      "gene: Spp2-17:+ -> 150\n",
      "gene: Spp2-18:+ -> 147\n",
      "gene: Spp2-19:+ -> 150\n",
      "gene: Spp2-2:+ -> 150\n",
      "gene: Spp2-20:+ -> 150\n",
      "gene: Spp2-21:+ -> 150\n",
      "gene: Spp2-22:+ -> 127\n",
      "gene: Spp2-23:+ -> 135\n",
      "gene: Spp2-24:+ -> 150\n",
      "gene: Spp2-25:+ -> 120\n",
      "gene: Spp2-26:+ -> 150\n",
      "gene: Spp2-27:+ -> 99\n",
      "gene: Spp2-28:+ -> 145\n",
      "gene: Spp2-29:+ -> 136\n",
      "gene: Spp2-3:+ -> 145\n",
      "gene: Spp2-30:+ -> 93\n",
      "gene: Spp2-31:+ -> 115\n",
      "gene: Spp2-32:+ -> 150\n",
      "gene: Spp2-33:+ -> 139\n",
      "gene: Spp2-34:+ -> 144\n",
      "gene: Spp2-35:+ -> 150\n",
      "gene: Spp2-36:+ -> 150\n",
      "gene: Spp2-37:+ -> 134\n",
      "gene: Spp2-38:+ -> 150\n",
      "gene: Spp2-39:+ -> 150\n",
      "gene: Spp2-4:+ -> 147\n",
      "gene: Spp2-40:+ -> 150\n",
      "gene: Spp2-41:+ -> 150\n",
      "gene: Spp2-42:+ -> 144\n",
      "gene: Spp2-43:+ -> 131\n",
      "gene: Spp2-44:+ -> 150\n",
      "gene: Spp2-45:+ -> 150\n",
      "gene: Spp2-46:+ -> 149\n",
      "gene: Spp2-47:+ -> 142\n",
      "gene: Spp2-48:+ -> 143\n",
      "gene: Spp2-49:+ -> 139\n",
      "gene: Spp2-5:+ -> 150\n",
      "gene: Spp2-50:+ -> 149\n",
      "gene: Spp2-51:+ -> 150\n",
      "gene: Spp2-52:+ -> 144\n",
      "gene: Spp2-53:+ -> 53\n",
      "gene: Spp2-54:+ -> 150\n",
      "gene: Spp2-55:+ -> 87\n",
      "gene: Spp2-56:+ -> 126\n",
      "gene: Spp2-57:+ -> 149\n",
      "gene: Spp2-58:+ -> 146\n",
      "gene: Spp2-59:+ -> 120\n",
      "gene: Spp2-6:+ -> 150\n",
      "gene: Spp2-60:+ -> 139\n",
      "gene: Spp2-61:+ -> 147\n",
      "gene: Spp2-62:+ -> 147\n",
      "gene: Spp2-63:+ -> 144\n",
      "gene: Spp2-64:+ -> 150\n",
      "gene: Spp2-65:+ -> 131\n",
      "gene: Spp2-66:+ -> 150\n",
      "gene: Spp2-67:+ -> 125\n",
      "gene: Spp2-68:+ -> 140\n",
      "gene: Spp2-69:+ -> 150\n",
      "gene: Spp2-7:+ -> 150\n",
      "gene: Spp2-70:+ -> 133\n",
      "gene: Spp2-71:+ -> 150\n",
      "gene: Spp2-72:+ -> 142\n",
      "gene: Spp2-73:+ -> 143\n",
      "gene: Spp2-74:+ -> 147\n",
      "gene: Spp2-75:+ -> 150\n",
      "gene: Spp2-76:+ -> 132\n",
      "gene: Spp2-77:+ -> 150\n",
      "gene: Spp2-78:+ -> 150\n",
      "gene: Spp2-79:+ -> 150\n",
      "gene: Spp2-8:+ -> 148\n",
      "gene: Spp2-80:+ -> 83\n",
      "gene: Spp2-81:+ -> 150\n",
      "gene: Spp2-82:+ -> 150\n",
      "gene: Spp2-83:+ -> 150\n",
      "gene: Spp2-84:+ -> 150\n",
      "gene: Spp2-85:+ -> 150\n",
      "gene: Spp2-86:+ -> 141\n",
      "gene: Spp2-87:+ -> 128\n",
      "gene: Spp2-88:+ -> 144\n",
      "gene: Spp2-89:+ -> 150\n",
      "gene: Spp2-9:+ -> 150\n",
      "gene: Spp2-90:+ -> 88\n",
      "gene: Spp2-91:+ -> 130\n",
      "gene: Spp2-92:+ -> 50\n",
      "True\n"
     ]
    }
   ],
   "source": [
    "reload(check)\n",
    "reg_size_dic, len_check = check._check_region_size(full_records, \n",
    "                                                   min_size=min_num_probes)\n",
    "print(len_check)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "native-remains",
   "metadata": {},
   "source": [
    "### 4.1.3 check readout usage for each gene"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 143,
   "id": "rural-glasgow",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "3"
      ]
     },
     "execution_count": 143,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "num_readout_per_probe"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 144,
   "id": "crazy-recording",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Camk1-1:- -> ['NDB_640']\n",
      "Camk1-10:- -> ['NDB_674']\n",
      "Camk1-11:- -> ['NDB_667']\n",
      "Camk1-12:- -> ['NDB_677']\n",
      "Camk1-13:- -> ['NDB_673']\n",
      "Camk1-14:- -> ['NDB_686']\n",
      "Camk1-15:- -> ['NDB_676']\n",
      "Camk1-16:- -> ['NDB_692']\n",
      "Camk1-17:- -> ['NDB_679']\n",
      "Camk1-18:- -> ['NDB_695']\n",
      "Camk1-19:- -> ['NDB_682']\n",
      "Camk1-2:- -> ['NDB_632']\n",
      "Camk1-20:- -> ['NDB_701']\n",
      "Camk1-21:- -> ['NDB_685']\n",
      "Camk1-22:- -> ['NDB_704']\n",
      "Camk1-23:- -> ['NDB_688']\n",
      "Camk1-24:- -> ['NDB_719']\n",
      "Camk1-25:- -> ['NDB_697']\n",
      "Camk1-26:- -> ['NDB_722']\n",
      "Camk1-27:- -> ['NDB_724']\n",
      "Camk1-28:- -> ['NDB_731']\n",
      "Camk1-29:- -> ['NDB_733']\n",
      "Camk1-3:- -> ['NDB_646']\n",
      "Camk1-30:- -> ['NDB_764']\n",
      "Camk1-31:- -> ['NDB_745']\n",
      "Camk1-32:- -> ['NDB_770']\n",
      "Camk1-33:- -> ['NDB_757']\n",
      "Camk1-34:- -> ['NDB_782']\n",
      "Camk1-35:- -> ['NDB_763']\n",
      "Camk1-36:- -> ['NDB_794']\n",
      "Camk1-37:- -> ['NDB_766']\n",
      "Camk1-38:- -> ['NDB_797']\n",
      "Camk1-39:- -> ['NDB_769']\n",
      "Camk1-4:- -> ['NDB_656']\n",
      "Camk1-40:- -> ['NDB_800']\n",
      "Camk1-41:- -> ['NDB_781']\n",
      "Camk1-42:- -> ['NDB_803']\n",
      "Camk1-43:- -> ['NDB_787']\n",
      "Camk1-44:- -> ['NDB_827']\n",
      "Camk1-45:- -> ['NDB_808']\n",
      "Camk1-46:- -> ['NDB_830']\n",
      "Camk1-47:- -> ['NDB_811']\n",
      "Camk1-48:- -> ['NDB_833']\n",
      "Camk1-49:- -> ['NDB_820']\n",
      "Camk1-5:- -> ['NDB_655']\n",
      "Camk1-50:- -> ['NDB_839']\n",
      "Camk1-51:- -> ['NDB_850']\n",
      "Camk1-52:- -> ['NDB_848']\n",
      "Camk1-53:- -> ['NDB_874']\n",
      "Camk1-54:- -> ['NDB_851']\n",
      "Camk1-6:- -> ['NDB_662']\n",
      "Camk1-7:- -> ['NDB_658']\n",
      "Camk1-8:- -> ['NDB_671']\n",
      "Camk1-9:- -> ['NDB_664']\n",
      "Hoxd10-1:+ -> ['NDB_877']\n",
      "Hoxd10-10:+ -> ['NDB_893']\n",
      "Hoxd10-100:+ -> ['NDB_1103']\n",
      "Hoxd10-101:+ -> ['NDB_1141']\n",
      "Hoxd10-102:+ -> ['NDB_1106']\n",
      "Hoxd10-103:+ -> ['NDB_1144']\n",
      "Hoxd10-104:+ -> ['NDB_1109']\n",
      "Hoxd10-105:+ -> ['NDB_1147']\n",
      "Hoxd10-106:+ -> ['NDB_1112']\n",
      "Hoxd10-107:+ -> ['NDB_1150']\n",
      "Hoxd10-108:+ -> ['NDB_1115']\n",
      "Hoxd10-109:+ -> ['NDB_1153']\n",
      "Hoxd10-11:+ -> ['NDB_916']\n",
      "Hoxd10-110:+ -> ['NDB_1118']\n",
      "Hoxd10-111:+ -> ['NDB_1156']\n",
      "Hoxd10-112:+ -> ['NDB_1121']\n",
      "Hoxd10-113:+ -> ['NDB_1159']\n",
      "Hoxd10-114:+ -> ['NDB_1124']\n",
      "Hoxd10-115:+ -> ['NDB_1162']\n",
      "Hoxd10-116:+ -> ['NDB_1127']\n",
      "Hoxd10-117:+ -> ['NDB_1165']\n",
      "Hoxd10-118:+ -> ['NDB_1130']\n",
      "Hoxd10-12:+ -> ['NDB_923']\n",
      "Hoxd10-13:+ -> ['NDB_919']\n",
      "Hoxd10-14:+ -> ['NDB_932']\n",
      "Hoxd10-15:+ -> ['NDB_937']\n",
      "Hoxd10-16:+ -> ['NDB_938']\n",
      "Hoxd10-17:+ -> ['NDB_943']\n",
      "Hoxd10-18:+ -> ['NDB_947']\n",
      "Hoxd10-19:+ -> ['NDB_952']\n",
      "Hoxd10-2:+ -> ['NDB_854']\n",
      "Hoxd10-20:+ -> ['NDB_950']\n",
      "Hoxd10-21:+ -> ['NDB_964']\n",
      "Hoxd10-22:+ -> ['NDB_953']\n",
      "Hoxd10-23:+ -> ['NDB_967']\n",
      "Hoxd10-24:+ -> ['NDB_956']\n",
      "Hoxd10-25:+ -> ['NDB_970']\n",
      "Hoxd10-26:+ -> ['NDB_959']\n",
      "Hoxd10-27:+ -> ['NDB_979']\n",
      "Hoxd10-28:+ -> ['NDB_962']\n",
      "Hoxd10-29:+ -> ['NDB_982']\n",
      "Hoxd10-3:+ -> ['NDB_886']\n",
      "Hoxd10-30:+ -> ['NDB_968']\n",
      "Hoxd10-31:+ -> ['NDB_985']\n",
      "Hoxd10-32:+ -> ['NDB_980']\n",
      "Hoxd10-33:+ -> ['NDB_991']\n",
      "Hoxd10-34:+ -> ['NDB_992']\n",
      "Hoxd10-35:+ -> ['NDB_997']\n",
      "Hoxd10-36:+ -> ['NDB_998']\n",
      "Hoxd10-37:+ -> ['NDB_1000']\n",
      "Hoxd10-38:+ -> ['NDB_1001']\n",
      "Hoxd10-39:+ -> ['NDB_1003']\n",
      "Hoxd10-4:+ -> ['NDB_857']\n",
      "Hoxd10-40:+ -> ['NDB_1004']\n",
      "Hoxd10-41:+ -> ['NDB_1006']\n",
      "Hoxd10-42:+ -> ['NDB_1007']\n",
      "Hoxd10-43:+ -> ['NDB_1009']\n",
      "Hoxd10-44:+ -> ['NDB_1010']\n",
      "Hoxd10-45:+ -> ['NDB_1012']\n",
      "Hoxd10-46:+ -> ['NDB_1019']\n",
      "Hoxd10-47:+ -> ['NDB_1015']\n",
      "Hoxd10-48:+ -> ['NDB_1022']\n",
      "Hoxd10-49:+ -> ['NDB_1021']\n",
      "Hoxd10-5:+ -> ['NDB_892']\n",
      "Hoxd10-50:+ -> ['NDB_1025']\n",
      "Hoxd10-51:+ -> ['NDB_1024']\n",
      "Hoxd10-52:+ -> ['NDB_1031']\n",
      "Hoxd10-53:+ -> ['NDB_1030']\n",
      "Hoxd10-54:+ -> ['NDB_1034']\n",
      "Hoxd10-55:+ -> ['NDB_1045']\n",
      "Hoxd10-56:+ -> ['NDB_1037']\n",
      "Hoxd10-57:+ -> ['NDB_1054']\n",
      "Hoxd10-58:+ -> ['NDB_1040']\n",
      "Hoxd10-59:+ -> ['NDB_1057']\n",
      "Hoxd10-6:+ -> ['NDB_884']\n",
      "Hoxd10-60:+ -> ['NDB_1043']\n",
      "Hoxd10-61:+ -> ['NDB_1069']\n",
      "Hoxd10-62:+ -> ['NDB_1046']\n",
      "Hoxd10-63:+ -> ['NDB_1078']\n",
      "Hoxd10-64:+ -> ['NDB_1049']\n",
      "Hoxd10-65:+ -> ['NDB_1081']\n",
      "Hoxd10-66:+ -> ['NDB_1052']\n",
      "Hoxd10-67:+ -> ['NDB_1084']\n",
      "Hoxd10-68:+ -> ['NDB_1055']\n",
      "Hoxd10-69:+ -> ['NDB_1087']\n",
      "Hoxd10-7:+ -> ['NDB_898']\n",
      "Hoxd10-70:+ -> ['NDB_1058']\n",
      "Hoxd10-71:+ -> ['NDB_1090']\n",
      "Hoxd10-72:+ -> ['NDB_1061']\n",
      "Hoxd10-73:+ -> ['NDB_1093']\n",
      "Hoxd10-74:+ -> ['NDB_1064']\n",
      "Hoxd10-75:+ -> ['NDB_1096']\n",
      "Hoxd10-76:+ -> ['NDB_1067']\n",
      "Hoxd10-77:+ -> ['NDB_1099']\n",
      "Hoxd10-78:+ -> ['NDB_1070']\n",
      "Hoxd10-79:+ -> ['NDB_1102']\n",
      "Hoxd10-8:+ -> ['NDB_890']\n",
      "Hoxd10-80:+ -> ['NDB_1073']\n",
      "Hoxd10-81:+ -> ['NDB_1108']\n",
      "Hoxd10-82:+ -> ['NDB_1076']\n",
      "Hoxd10-83:+ -> ['NDB_1114']\n",
      "Hoxd10-84:+ -> ['NDB_1079']\n",
      "Hoxd10-85:+ -> ['NDB_1117']\n",
      "Hoxd10-86:+ -> ['NDB_1082']\n",
      "Hoxd10-87:+ -> ['NDB_1120']\n",
      "Hoxd10-88:+ -> ['NDB_1085']\n",
      "Hoxd10-89:+ -> ['NDB_1123']\n",
      "Hoxd10-9:+ -> ['NDB_913']\n",
      "Hoxd10-90:+ -> ['NDB_1088']\n",
      "Hoxd10-91:+ -> ['NDB_1126']\n",
      "Hoxd10-92:+ -> ['NDB_1091']\n",
      "Hoxd10-93:+ -> ['NDB_1129']\n",
      "Hoxd10-94:+ -> ['NDB_1094']\n",
      "Hoxd10-95:+ -> ['NDB_1132']\n",
      "Hoxd10-96:+ -> ['NDB_1097']\n",
      "Hoxd10-97:+ -> ['NDB_1135']\n",
      "Hoxd10-98:+ -> ['NDB_1100']\n",
      "Hoxd10-99:+ -> ['NDB_1138']\n",
      "Spp2-1:+ -> ['NDB_1168']\n",
      "Spp2-10:+ -> ['NDB_1145']\n",
      "Spp2-11:+ -> ['NDB_1183']\n",
      "Spp2-12:+ -> ['NDB_1148']\n",
      "Spp2-13:+ -> ['NDB_1186']\n",
      "Spp2-14:+ -> ['NDB_1151']\n",
      "Spp2-15:+ -> ['NDB_1189']\n",
      "Spp2-16:+ -> ['NDB_1154']\n",
      "Spp2-17:+ -> ['NDB_1192']\n",
      "Spp2-18:+ -> ['NDB_1157']\n",
      "Spp2-19:+ -> ['NDB_1195']\n",
      "Spp2-2:+ -> ['NDB_1133']\n",
      "Spp2-20:+ -> ['NDB_1160']\n",
      "Spp2-21:+ -> ['NDB_1198']\n",
      "Spp2-22:+ -> ['NDB_1163']\n",
      "Spp2-23:+ -> ['NDB_1201']\n",
      "Spp2-24:+ -> ['NDB_1166']\n",
      "Spp2-25:+ -> ['NDB_1204']\n",
      "Spp2-26:+ -> ['NDB_1169']\n",
      "Spp2-27:+ -> ['NDB_1207']\n",
      "Spp2-28:+ -> ['NDB_1172']\n",
      "Spp2-29:+ -> ['NDB_1210']\n",
      "Spp2-3:+ -> ['NDB_1171']\n",
      "Spp2-30:+ -> ['NDB_1175']\n",
      "Spp2-31:+ -> ['NDB_1213']\n",
      "Spp2-32:+ -> ['NDB_1178']\n",
      "Spp2-33:+ -> ['NDB_1228']\n",
      "Spp2-34:+ -> ['NDB_1181']\n",
      "Spp2-35:+ -> ['NDB_1222']\n",
      "Spp2-36:+ -> ['NDB_1184']\n",
      "Spp2-37:+ -> ['NDB_1225']\n",
      "Spp2-38:+ -> ['NDB_1187']\n",
      "Spp2-39:+ -> ['NDB_1216']\n",
      "Spp2-4:+ -> ['NDB_1136']\n",
      "Spp2-40:+ -> ['NDB_1190']\n",
      "Spp2-41:+ -> ['NDB_1219']\n",
      "Spp2-42:+ -> ['NDB_1193']\n",
      "Spp2-43:+ -> ['NDB_1237']\n",
      "Spp2-44:+ -> ['NDB_1196']\n",
      "Spp2-45:+ -> ['NDB_1231']\n",
      "Spp2-46:+ -> ['NDB_1199']\n",
      "Spp2-47:+ -> ['NDB_1240']\n",
      "Spp2-48:+ -> ['NDB_1202']\n",
      "Spp2-49:+ -> ['NDB_1234']\n",
      "Spp2-5:+ -> ['NDB_1174']\n",
      "Spp2-50:+ -> ['NDB_1205']\n",
      "Spp2-51:+ -> ['NDB_1258']\n",
      "Spp2-52:+ -> ['NDB_1208']\n",
      "Spp2-53:+ -> ['NDB_1261']\n",
      "Spp2-54:+ -> ['NDB_1211']\n",
      "Spp2-55:+ -> ['NDB_1264']\n",
      "Spp2-56:+ -> ['NDB_1214']\n",
      "Spp2-57:+ -> ['NDB_1249']\n",
      "Spp2-58:+ -> ['NDB_1217']\n",
      "Spp2-59:+ -> ['NDB_1252']\n",
      "Spp2-6:+ -> ['NDB_1139']\n",
      "Spp2-60:+ -> ['NDB_1226']\n",
      "Spp2-61:+ -> ['NDB_1243']\n",
      "Spp2-62:+ -> ['NDB_1235']\n",
      "Spp2-63:+ -> ['NDB_1246']\n",
      "Spp2-64:+ -> ['NDB_1244']\n",
      "Spp2-65:+ -> ['NDB_1270']\n",
      "Spp2-66:+ -> ['NDB_1253']\n",
      "Spp2-67:+ -> ['NDB_1255']\n",
      "Spp2-68:+ -> ['NDB_1259']\n",
      "Spp2-69:+ -> ['NDB_1267']\n",
      "Spp2-7:+ -> ['NDB_1177']\n",
      "Spp2-70:+ -> ['NDB_1265']\n",
      "Spp2-71:+ -> ['NDB_1273']\n",
      "Spp2-72:+ -> ['NDB_1271']\n",
      "Spp2-73:+ -> ['NDB_1276']\n",
      "Spp2-74:+ -> ['NDB_1223']\n",
      "Spp2-75:+ -> ['NDB_1279']\n",
      "Spp2-76:+ -> ['NDB_1277']\n",
      "Spp2-77:+ -> ['NDB_1282']\n",
      "Spp2-78:+ -> ['NDB_1232']\n",
      "Spp2-79:+ -> []\n",
      "Spp2-8:+ -> ['NDB_1142']\n",
      "Spp2-80:+ -> []\n",
      "Spp2-81:+ -> []\n",
      "Spp2-82:+ -> []\n",
      "Spp2-83:+ -> []\n",
      "Spp2-84:+ -> []\n",
      "Spp2-85:+ -> []\n",
      "Spp2-86:+ -> []\n",
      "Spp2-87:+ -> []\n",
      "Spp2-88:+ -> []\n",
      "Spp2-89:+ -> []\n",
      "Spp2-9:+ -> ['NDB_1180']\n",
      "Spp2-90:+ -> []\n",
      "Spp2-91:+ -> []\n",
      "Spp2-92:+ -> []\n",
      "True\n"
     ]
    }
   ],
   "source": [
    "reload(check)\n",
    "check_readout_dict = pickle.load(open(os.path.join(library_folder, 'readout_usage.pkl'), 'rb'))\n",
    "\n",
    "reg_readout_dic, reg2readout_check = check._check_region_to_readouts(full_records, check_readout_dict, \n",
    "                                                                     total_readout_num=1)\n",
    "print(reg2readout_check)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 152,
   "id": "careful-telescope",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'Camk1-1:-': ['NDB_640'],\n",
       " 'Camk1-10:-': ['NDB_674'],\n",
       " 'Camk1-11:-': ['NDB_667'],\n",
       " 'Camk1-12:-': ['NDB_677'],\n",
       " 'Camk1-13:-': ['NDB_673'],\n",
       " 'Camk1-14:-': ['NDB_686'],\n",
       " 'Camk1-15:-': ['NDB_676'],\n",
       " 'Camk1-16:-': ['NDB_692'],\n",
       " 'Camk1-17:-': ['NDB_679'],\n",
       " 'Camk1-18:-': ['NDB_695'],\n",
       " 'Camk1-19:-': ['NDB_682'],\n",
       " 'Camk1-2:-': ['NDB_632'],\n",
       " 'Camk1-20:-': ['NDB_701'],\n",
       " 'Camk1-21:-': ['NDB_685'],\n",
       " 'Camk1-22:-': ['NDB_704'],\n",
       " 'Camk1-23:-': ['NDB_688'],\n",
       " 'Camk1-24:-': ['NDB_719'],\n",
       " 'Camk1-25:-': ['NDB_697'],\n",
       " 'Camk1-26:-': ['NDB_722'],\n",
       " 'Camk1-27:-': ['NDB_724'],\n",
       " 'Camk1-28:-': ['NDB_731'],\n",
       " 'Camk1-29:-': ['NDB_733'],\n",
       " 'Camk1-3:-': ['NDB_646'],\n",
       " 'Camk1-30:-': ['NDB_764'],\n",
       " 'Camk1-31:-': ['NDB_745'],\n",
       " 'Camk1-32:-': ['NDB_770'],\n",
       " 'Camk1-33:-': ['NDB_757'],\n",
       " 'Camk1-34:-': ['NDB_782'],\n",
       " 'Camk1-35:-': ['NDB_763'],\n",
       " 'Camk1-36:-': ['NDB_794'],\n",
       " 'Camk1-37:-': ['NDB_766'],\n",
       " 'Camk1-38:-': ['NDB_797'],\n",
       " 'Camk1-39:-': ['NDB_769'],\n",
       " 'Camk1-4:-': ['NDB_656'],\n",
       " 'Camk1-40:-': ['NDB_800'],\n",
       " 'Camk1-41:-': ['NDB_781'],\n",
       " 'Camk1-42:-': ['NDB_803'],\n",
       " 'Camk1-43:-': ['NDB_787'],\n",
       " 'Camk1-44:-': ['NDB_827'],\n",
       " 'Camk1-45:-': ['NDB_808'],\n",
       " 'Camk1-46:-': ['NDB_830'],\n",
       " 'Camk1-47:-': ['NDB_811'],\n",
       " 'Camk1-48:-': ['NDB_833'],\n",
       " 'Camk1-49:-': ['NDB_820'],\n",
       " 'Camk1-5:-': ['NDB_655'],\n",
       " 'Camk1-50:-': ['NDB_839'],\n",
       " 'Camk1-51:-': ['NDB_850'],\n",
       " 'Camk1-52:-': ['NDB_848'],\n",
       " 'Camk1-53:-': ['NDB_874'],\n",
       " 'Camk1-54:-': ['NDB_851'],\n",
       " 'Camk1-6:-': ['NDB_662'],\n",
       " 'Camk1-7:-': ['NDB_658'],\n",
       " 'Camk1-8:-': ['NDB_671'],\n",
       " 'Camk1-9:-': ['NDB_664'],\n",
       " 'Hoxd10-1:+': ['NDB_877'],\n",
       " 'Hoxd10-10:+': ['NDB_893'],\n",
       " 'Hoxd10-100:+': ['NDB_1103'],\n",
       " 'Hoxd10-101:+': ['NDB_1141'],\n",
       " 'Hoxd10-102:+': ['NDB_1106'],\n",
       " 'Hoxd10-103:+': ['NDB_1144'],\n",
       " 'Hoxd10-104:+': ['NDB_1109'],\n",
       " 'Hoxd10-105:+': ['NDB_1147'],\n",
       " 'Hoxd10-106:+': ['NDB_1112'],\n",
       " 'Hoxd10-107:+': ['NDB_1150'],\n",
       " 'Hoxd10-108:+': ['NDB_1115'],\n",
       " 'Hoxd10-109:+': ['NDB_1153'],\n",
       " 'Hoxd10-11:+': ['NDB_916'],\n",
       " 'Hoxd10-110:+': ['NDB_1118'],\n",
       " 'Hoxd10-111:+': ['NDB_1156'],\n",
       " 'Hoxd10-112:+': ['NDB_1121'],\n",
       " 'Hoxd10-113:+': ['NDB_1159'],\n",
       " 'Hoxd10-114:+': ['NDB_1124'],\n",
       " 'Hoxd10-115:+': ['NDB_1162'],\n",
       " 'Hoxd10-116:+': ['NDB_1127'],\n",
       " 'Hoxd10-117:+': ['NDB_1165'],\n",
       " 'Hoxd10-118:+': ['NDB_1130'],\n",
       " 'Hoxd10-12:+': ['NDB_923'],\n",
       " 'Hoxd10-13:+': ['NDB_919'],\n",
       " 'Hoxd10-14:+': ['NDB_932'],\n",
       " 'Hoxd10-15:+': ['NDB_937'],\n",
       " 'Hoxd10-16:+': ['NDB_938'],\n",
       " 'Hoxd10-17:+': ['NDB_943'],\n",
       " 'Hoxd10-18:+': ['NDB_947'],\n",
       " 'Hoxd10-19:+': ['NDB_952'],\n",
       " 'Hoxd10-2:+': ['NDB_854'],\n",
       " 'Hoxd10-20:+': ['NDB_950'],\n",
       " 'Hoxd10-21:+': ['NDB_964'],\n",
       " 'Hoxd10-22:+': ['NDB_953'],\n",
       " 'Hoxd10-23:+': ['NDB_967'],\n",
       " 'Hoxd10-24:+': ['NDB_956'],\n",
       " 'Hoxd10-25:+': ['NDB_970'],\n",
       " 'Hoxd10-26:+': ['NDB_959'],\n",
       " 'Hoxd10-27:+': ['NDB_979'],\n",
       " 'Hoxd10-28:+': ['NDB_962'],\n",
       " 'Hoxd10-29:+': ['NDB_982'],\n",
       " 'Hoxd10-3:+': ['NDB_886'],\n",
       " 'Hoxd10-30:+': ['NDB_968'],\n",
       " 'Hoxd10-31:+': ['NDB_985'],\n",
       " 'Hoxd10-32:+': ['NDB_980'],\n",
       " 'Hoxd10-33:+': ['NDB_991'],\n",
       " 'Hoxd10-34:+': ['NDB_992'],\n",
       " 'Hoxd10-35:+': ['NDB_997'],\n",
       " 'Hoxd10-36:+': ['NDB_998'],\n",
       " 'Hoxd10-37:+': ['NDB_1000'],\n",
       " 'Hoxd10-38:+': ['NDB_1001'],\n",
       " 'Hoxd10-39:+': ['NDB_1003'],\n",
       " 'Hoxd10-4:+': ['NDB_857'],\n",
       " 'Hoxd10-40:+': ['NDB_1004'],\n",
       " 'Hoxd10-41:+': ['NDB_1006'],\n",
       " 'Hoxd10-42:+': ['NDB_1007'],\n",
       " 'Hoxd10-43:+': ['NDB_1009'],\n",
       " 'Hoxd10-44:+': ['NDB_1010'],\n",
       " 'Hoxd10-45:+': ['NDB_1012'],\n",
       " 'Hoxd10-46:+': ['NDB_1019'],\n",
       " 'Hoxd10-47:+': ['NDB_1015'],\n",
       " 'Hoxd10-48:+': ['NDB_1022'],\n",
       " 'Hoxd10-49:+': ['NDB_1021'],\n",
       " 'Hoxd10-5:+': ['NDB_892'],\n",
       " 'Hoxd10-50:+': ['NDB_1025'],\n",
       " 'Hoxd10-51:+': ['NDB_1024'],\n",
       " 'Hoxd10-52:+': ['NDB_1031'],\n",
       " 'Hoxd10-53:+': ['NDB_1030'],\n",
       " 'Hoxd10-54:+': ['NDB_1034'],\n",
       " 'Hoxd10-55:+': ['NDB_1045'],\n",
       " 'Hoxd10-56:+': ['NDB_1037'],\n",
       " 'Hoxd10-57:+': ['NDB_1054'],\n",
       " 'Hoxd10-58:+': ['NDB_1040'],\n",
       " 'Hoxd10-59:+': ['NDB_1057'],\n",
       " 'Hoxd10-6:+': ['NDB_884'],\n",
       " 'Hoxd10-60:+': ['NDB_1043'],\n",
       " 'Hoxd10-61:+': ['NDB_1069'],\n",
       " 'Hoxd10-62:+': ['NDB_1046'],\n",
       " 'Hoxd10-63:+': ['NDB_1078'],\n",
       " 'Hoxd10-64:+': ['NDB_1049'],\n",
       " 'Hoxd10-65:+': ['NDB_1081'],\n",
       " 'Hoxd10-66:+': ['NDB_1052'],\n",
       " 'Hoxd10-67:+': ['NDB_1084'],\n",
       " 'Hoxd10-68:+': ['NDB_1055'],\n",
       " 'Hoxd10-69:+': ['NDB_1087'],\n",
       " 'Hoxd10-7:+': ['NDB_898'],\n",
       " 'Hoxd10-70:+': ['NDB_1058'],\n",
       " 'Hoxd10-71:+': ['NDB_1090'],\n",
       " 'Hoxd10-72:+': ['NDB_1061'],\n",
       " 'Hoxd10-73:+': ['NDB_1093'],\n",
       " 'Hoxd10-74:+': ['NDB_1064'],\n",
       " 'Hoxd10-75:+': ['NDB_1096'],\n",
       " 'Hoxd10-76:+': ['NDB_1067'],\n",
       " 'Hoxd10-77:+': ['NDB_1099'],\n",
       " 'Hoxd10-78:+': ['NDB_1070'],\n",
       " 'Hoxd10-79:+': ['NDB_1102'],\n",
       " 'Hoxd10-8:+': ['NDB_890'],\n",
       " 'Hoxd10-80:+': ['NDB_1073'],\n",
       " 'Hoxd10-81:+': ['NDB_1108'],\n",
       " 'Hoxd10-82:+': ['NDB_1076'],\n",
       " 'Hoxd10-83:+': ['NDB_1114'],\n",
       " 'Hoxd10-84:+': ['NDB_1079'],\n",
       " 'Hoxd10-85:+': ['NDB_1117'],\n",
       " 'Hoxd10-86:+': ['NDB_1082'],\n",
       " 'Hoxd10-87:+': ['NDB_1120'],\n",
       " 'Hoxd10-88:+': ['NDB_1085'],\n",
       " 'Hoxd10-89:+': ['NDB_1123'],\n",
       " 'Hoxd10-9:+': ['NDB_913'],\n",
       " 'Hoxd10-90:+': ['NDB_1088'],\n",
       " 'Hoxd10-91:+': ['NDB_1126'],\n",
       " 'Hoxd10-92:+': ['NDB_1091'],\n",
       " 'Hoxd10-93:+': ['NDB_1129'],\n",
       " 'Hoxd10-94:+': ['NDB_1094'],\n",
       " 'Hoxd10-95:+': ['NDB_1132'],\n",
       " 'Hoxd10-96:+': ['NDB_1097'],\n",
       " 'Hoxd10-97:+': ['NDB_1135'],\n",
       " 'Hoxd10-98:+': ['NDB_1100'],\n",
       " 'Hoxd10-99:+': ['NDB_1138'],\n",
       " 'Spp2-1:+': ['NDB_1168'],\n",
       " 'Spp2-10:+': ['NDB_1145'],\n",
       " 'Spp2-11:+': ['NDB_1183'],\n",
       " 'Spp2-12:+': ['NDB_1148'],\n",
       " 'Spp2-13:+': ['NDB_1186'],\n",
       " 'Spp2-14:+': ['NDB_1151'],\n",
       " 'Spp2-15:+': ['NDB_1189'],\n",
       " 'Spp2-16:+': ['NDB_1154'],\n",
       " 'Spp2-17:+': ['NDB_1192'],\n",
       " 'Spp2-18:+': ['NDB_1157'],\n",
       " 'Spp2-19:+': ['NDB_1195'],\n",
       " 'Spp2-2:+': ['NDB_1133'],\n",
       " 'Spp2-20:+': ['NDB_1160'],\n",
       " 'Spp2-21:+': ['NDB_1198'],\n",
       " 'Spp2-22:+': ['NDB_1163'],\n",
       " 'Spp2-23:+': ['NDB_1201'],\n",
       " 'Spp2-24:+': ['NDB_1166'],\n",
       " 'Spp2-25:+': ['NDB_1204'],\n",
       " 'Spp2-26:+': ['NDB_1169'],\n",
       " 'Spp2-27:+': ['NDB_1207'],\n",
       " 'Spp2-28:+': ['NDB_1172'],\n",
       " 'Spp2-29:+': ['NDB_1210'],\n",
       " 'Spp2-3:+': ['NDB_1171'],\n",
       " 'Spp2-30:+': ['NDB_1175'],\n",
       " 'Spp2-31:+': ['NDB_1213'],\n",
       " 'Spp2-32:+': ['NDB_1178'],\n",
       " 'Spp2-33:+': ['NDB_1228'],\n",
       " 'Spp2-34:+': ['NDB_1181'],\n",
       " 'Spp2-35:+': ['NDB_1222'],\n",
       " 'Spp2-36:+': ['NDB_1184'],\n",
       " 'Spp2-37:+': ['NDB_1225'],\n",
       " 'Spp2-38:+': ['NDB_1187'],\n",
       " 'Spp2-39:+': ['NDB_1216'],\n",
       " 'Spp2-4:+': ['NDB_1136'],\n",
       " 'Spp2-40:+': ['NDB_1190'],\n",
       " 'Spp2-41:+': ['NDB_1219'],\n",
       " 'Spp2-42:+': ['NDB_1193'],\n",
       " 'Spp2-43:+': ['NDB_1237'],\n",
       " 'Spp2-44:+': ['NDB_1196'],\n",
       " 'Spp2-45:+': ['NDB_1231'],\n",
       " 'Spp2-46:+': ['NDB_1199'],\n",
       " 'Spp2-47:+': ['NDB_1240'],\n",
       " 'Spp2-48:+': ['NDB_1202'],\n",
       " 'Spp2-49:+': ['NDB_1234'],\n",
       " 'Spp2-5:+': ['NDB_1174'],\n",
       " 'Spp2-50:+': ['NDB_1205'],\n",
       " 'Spp2-51:+': ['NDB_1258'],\n",
       " 'Spp2-52:+': ['NDB_1208'],\n",
       " 'Spp2-53:+': ['NDB_1261'],\n",
       " 'Spp2-54:+': ['NDB_1211'],\n",
       " 'Spp2-55:+': ['NDB_1264'],\n",
       " 'Spp2-56:+': ['NDB_1214'],\n",
       " 'Spp2-57:+': ['NDB_1249'],\n",
       " 'Spp2-58:+': ['NDB_1217'],\n",
       " 'Spp2-59:+': ['NDB_1252'],\n",
       " 'Spp2-6:+': ['NDB_1139'],\n",
       " 'Spp2-60:+': ['NDB_1226'],\n",
       " 'Spp2-61:+': ['NDB_1243'],\n",
       " 'Spp2-62:+': ['NDB_1235'],\n",
       " 'Spp2-63:+': ['NDB_1246'],\n",
       " 'Spp2-64:+': ['NDB_1244'],\n",
       " 'Spp2-65:+': ['NDB_1270'],\n",
       " 'Spp2-66:+': ['NDB_1253'],\n",
       " 'Spp2-67:+': ['NDB_1255'],\n",
       " 'Spp2-68:+': ['NDB_1259'],\n",
       " 'Spp2-69:+': ['NDB_1267'],\n",
       " 'Spp2-7:+': ['NDB_1177'],\n",
       " 'Spp2-70:+': ['NDB_1265'],\n",
       " 'Spp2-71:+': ['NDB_1273'],\n",
       " 'Spp2-72:+': ['NDB_1271'],\n",
       " 'Spp2-73:+': ['NDB_1276'],\n",
       " 'Spp2-74:+': ['NDB_1223'],\n",
       " 'Spp2-75:+': ['NDB_1279'],\n",
       " 'Spp2-76:+': ['NDB_1277'],\n",
       " 'Spp2-77:+': ['NDB_1282'],\n",
       " 'Spp2-78:+': ['NDB_1232'],\n",
       " 'Spp2-79:+': [],\n",
       " 'Spp2-8:+': ['NDB_1142'],\n",
       " 'Spp2-80:+': [],\n",
       " 'Spp2-81:+': [],\n",
       " 'Spp2-82:+': [],\n",
       " 'Spp2-83:+': [],\n",
       " 'Spp2-84:+': [],\n",
       " 'Spp2-85:+': [],\n",
       " 'Spp2-86:+': [],\n",
       " 'Spp2-87:+': [],\n",
       " 'Spp2-88:+': [],\n",
       " 'Spp2-89:+': [],\n",
       " 'Spp2-9:+': ['NDB_1180'],\n",
       " 'Spp2-90:+': [],\n",
       " 'Spp2-91:+': [],\n",
       " 'Spp2-92:+': []}"
      ]
     },
     "execution_count": 152,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "reg_readout_dic"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "later-worse",
   "metadata": {},
   "source": [
    "### 4.1.4 check readout usage in the library"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 158,
   "id": "modern-gibson",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CTTGTGCATCGCGCCAAAGAATCCGCGGTCATCGTTGTTAATCCGCGGTCATCGTTGTTAGCATGCATAAATTACTGTGACATGAATAGAGTGAGTTAGCATCCGCGGTCATCGTTGTTAGGTGTGCGCAACGAAACCCG\n"
     ]
    }
   ],
   "source": [
    "print(full_records[0].seq.reverse_complement())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 155,
   "id": "corporate-neighbor",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(Seq('TCACAGTAATTTATGCATGCTAACAACGATGACCGCGGAT'),\n",
       " [Seq('ATCCGCGGTCATCGTTGTTA'),\n",
       "  Seq('CATGAATAGAGTGAGTTAGC'),\n",
       "  Seq('ATCCGCGGTCATCGTTGTTA')],\n",
       " Seq('CGGGTTTCGTTGCGCACACC'),\n",
       " Seq('CTTGTGCATCGCGCCAAAGA'))"
      ]
     },
     "execution_count": 155,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "check._parsing_probe_sequence(full_records[0], target_len=target_len)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 171,
   "id": "simplified-audit",
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- extracting probes from >region:Camk1-1:-, region:Camk1-10:-, region:Camk1-11:-, region:Camk1-12:-, region:Camk1-13:-, region:Camk1-14:-, region:Camk1-15:-, region:Camk1-16:-, region:Camk1-17:-, region:Camk1-18:-, region:Camk1-19:-, region:Camk1-2:-, region:Camk1-20:-, region:Camk1-21:-, region:Camk1-22:-, region:Camk1-23:-, region:Camk1-24:-, region:Camk1-25:-, region:Camk1-26:-, region:Camk1-27:-, region:Camk1-28:-, region:Camk1-29:-, region:Camk1-3:-, region:Camk1-30:-, region:Camk1-31:-, region:Camk1-32:-, region:Camk1-33:-, region:Camk1-34:-, region:Camk1-35:-, region:Camk1-36:-, region:Camk1-37:-, region:Camk1-38:-, region:Camk1-39:-, region:Camk1-4:-, region:Camk1-40:-, region:Camk1-41:-, region:Camk1-42:-, region:Camk1-43:-, region:Camk1-44:-, region:Camk1-45:-, region:Camk1-46:-, region:Camk1-47:-, region:Camk1-48:-, region:Camk1-49:-, region:Camk1-5:-, region:Camk1-50:-, region:Camk1-51:-, region:Camk1-52:-, region:Camk1-53:-, region:Camk1-54:-, region:Camk1-6:-, region:Camk1-7:-, region:Camk1-8:-, region:Camk1-9:-, region:Hoxd10-1:+, region:Hoxd10-10:+, region:Hoxd10-100:+, region:Hoxd10-101:+, region:Hoxd10-102:+, region:Hoxd10-103:+, region:Hoxd10-104:+, region:Hoxd10-105:+, region:Hoxd10-106:+, region:Hoxd10-107:+, region:Hoxd10-108:+, region:Hoxd10-109:+, region:Hoxd10-11:+, region:Hoxd10-110:+, region:Hoxd10-111:+, region:Hoxd10-112:+, region:Hoxd10-113:+, region:Hoxd10-114:+, region:Hoxd10-115:+, region:Hoxd10-116:+, region:Hoxd10-117:+, region:Hoxd10-118:+, region:Hoxd10-12:+, region:Hoxd10-13:+, region:Hoxd10-14:+, region:Hoxd10-15:+, region:Hoxd10-16:+, region:Hoxd10-17:+, region:Hoxd10-18:+, region:Hoxd10-19:+, region:Hoxd10-2:+, region:Hoxd10-20:+, region:Hoxd10-21:+, region:Hoxd10-22:+, region:Hoxd10-23:+, region:Hoxd10-24:+, region:Hoxd10-25:+, region:Hoxd10-26:+, region:Hoxd10-27:+, region:Hoxd10-28:+, region:Hoxd10-29:+, region:Hoxd10-3:+, region:Hoxd10-30:+, region:Hoxd10-31:+, region:Hoxd10-32:+, region:Hoxd10-33:+, region:Hoxd10-34:+, region:Hoxd10-35:+, region:Hoxd10-36:+, region:Hoxd10-37:+, region:Hoxd10-38:+, region:Hoxd10-39:+, region:Hoxd10-4:+, region:Hoxd10-40:+, region:Hoxd10-41:+, region:Hoxd10-42:+, region:Hoxd10-43:+, region:Hoxd10-44:+, region:Hoxd10-45:+, region:Hoxd10-46:+, region:Hoxd10-47:+, region:Hoxd10-48:+, region:Hoxd10-49:+, region:Hoxd10-5:+, region:Hoxd10-50:+, region:Hoxd10-51:+, region:Hoxd10-52:+, region:Hoxd10-53:+, region:Hoxd10-54:+, region:Hoxd10-55:+, region:Hoxd10-56:+, region:Hoxd10-57:+, region:Hoxd10-58:+, region:Hoxd10-59:+, region:Hoxd10-6:+, region:Hoxd10-60:+, region:Hoxd10-61:+, region:Hoxd10-62:+, region:Hoxd10-63:+, region:Hoxd10-64:+, region:Hoxd10-65:+, region:Hoxd10-66:+, region:Hoxd10-67:+, region:Hoxd10-68:+, region:Hoxd10-69:+, region:Hoxd10-7:+, region:Hoxd10-70:+, region:Hoxd10-71:+, region:Hoxd10-72:+, region:Hoxd10-73:+, region:Hoxd10-74:+, region:Hoxd10-75:+, region:Hoxd10-76:+, region:Hoxd10-77:+, region:Hoxd10-78:+, region:Hoxd10-79:+, region:Hoxd10-8:+, region:Hoxd10-80:+, region:Hoxd10-81:+, region:Hoxd10-82:+, region:Hoxd10-83:+, region:Hoxd10-84:+, region:Hoxd10-85:+, region:Hoxd10-86:+, region:Hoxd10-87:+, region:Hoxd10-88:+, region:Hoxd10-89:+, region:Hoxd10-9:+, region:Hoxd10-90:+, region:Hoxd10-91:+, region:Hoxd10-92:+, region:Hoxd10-93:+, region:Hoxd10-94:+, region:Hoxd10-95:+, region:Hoxd10-96:+, region:Hoxd10-97:+, region:Hoxd10-98:+, region:Hoxd10-99:+, region:Spp2-1:+, region:Spp2-10:+, region:Spp2-11:+, region:Spp2-12:+, region:Spp2-13:+, region:Spp2-14:+, region:Spp2-15:+, region:Spp2-16:+, region:Spp2-17:+, region:Spp2-18:+, region:Spp2-19:+, region:Spp2-2:+, region:Spp2-20:+, region:Spp2-21:+, region:Spp2-22:+, region:Spp2-23:+, region:Spp2-24:+, region:Spp2-25:+, region:Spp2-26:+, region:Spp2-27:+, region:Spp2-28:+, region:Spp2-29:+, region:Spp2-3:+, region:Spp2-30:+, region:Spp2-31:+, region:Spp2-32:+, region:Spp2-33:+, region:Spp2-34:+, region:Spp2-35:+, region:Spp2-36:+, region:Spp2-37:+, region:Spp2-38:+, region:Spp2-39:+, region:Spp2-4:+, region:Spp2-40:+, region:Spp2-41:+, region:Spp2-42:+, region:Spp2-43:+, region:Spp2-44:+, region:Spp2-45:+, region:Spp2-46:+, region:Spp2-47:+, region:Spp2-48:+, region:Spp2-49:+, region:Spp2-5:+, region:Spp2-50:+, region:Spp2-51:+, region:Spp2-52:+, region:Spp2-53:+, region:Spp2-54:+, region:Spp2-55:+, region:Spp2-56:+, region:Spp2-57:+, region:Spp2-58:+, region:Spp2-59:+, region:Spp2-6:+, region:Spp2-60:+, region:Spp2-61:+, region:Spp2-62:+, region:Spp2-63:+, region:Spp2-64:+, region:Spp2-65:+, region:Spp2-66:+, region:Spp2-67:+, region:Spp2-68:+, region:Spp2-69:+, region:Spp2-7:+, region:Spp2-70:+, region:Spp2-71:+, region:Spp2-72:+, region:Spp2-73:+, region:Spp2-74:+, region:Spp2-75:+, region:Spp2-76:+, region:Spp2-77:+, region:Spp2-78:+, region:Spp2-79:+, region:Spp2-8:+, region:Spp2-80:+, region:Spp2-81:+, region:Spp2-82:+, region:Spp2-83:+, region:Spp2-84:+, region:Spp2-85:+, region:Spp2-86:+, region:Spp2-87:+, region:Spp2-88:+, region:Spp2-89:+, region:Spp2-9:+, region:Spp2-90:+, region:Spp2-91:+, region:Spp2-92:+,  > Done.\n",
      "-- inverting region_to_readout dict\n",
      "NDB_1000 -> ['Hoxd10-37:+']\n",
      "NDB_1001 -> ['Hoxd10-38:+']\n",
      "NDB_1003 -> ['Hoxd10-39:+']\n",
      "NDB_1004 -> ['Hoxd10-40:+']\n",
      "NDB_1006 -> ['Hoxd10-41:+']\n",
      "NDB_1007 -> ['Hoxd10-42:+']\n",
      "NDB_1009 -> ['Hoxd10-43:+']\n",
      "NDB_1010 -> ['Hoxd10-44:+']\n",
      "NDB_1012 -> ['Hoxd10-45:+']\n",
      "NDB_1015 -> ['Hoxd10-47:+']\n",
      "NDB_1019 -> ['Hoxd10-46:+']\n",
      "NDB_1021 -> ['Hoxd10-49:+']\n",
      "NDB_1022 -> ['Hoxd10-48:+']\n",
      "NDB_1024 -> ['Hoxd10-51:+']\n",
      "NDB_1025 -> ['Hoxd10-50:+']\n",
      "NDB_1030 -> ['Hoxd10-53:+']\n",
      "NDB_1031 -> ['Hoxd10-52:+']\n",
      "NDB_1034 -> ['Hoxd10-54:+']\n",
      "NDB_1037 -> ['Hoxd10-56:+']\n",
      "NDB_1040 -> ['Hoxd10-58:+']\n",
      "NDB_1043 -> ['Hoxd10-60:+']\n",
      "NDB_1045 -> ['Hoxd10-55:+']\n",
      "NDB_1046 -> ['Hoxd10-62:+']\n",
      "NDB_1049 -> ['Hoxd10-64:+']\n",
      "NDB_1052 -> ['Hoxd10-66:+']\n",
      "NDB_1054 -> ['Hoxd10-57:+']\n",
      "NDB_1055 -> ['Hoxd10-68:+']\n",
      "NDB_1057 -> ['Hoxd10-59:+']\n",
      "NDB_1058 -> ['Hoxd10-70:+']\n",
      "NDB_1061 -> ['Hoxd10-72:+']\n",
      "NDB_1064 -> ['Hoxd10-74:+']\n",
      "NDB_1067 -> ['Hoxd10-76:+']\n",
      "NDB_1069 -> ['Hoxd10-61:+']\n",
      "NDB_1070 -> ['Hoxd10-78:+']\n",
      "NDB_1073 -> ['Hoxd10-80:+']\n",
      "NDB_1076 -> ['Hoxd10-82:+']\n",
      "NDB_1078 -> ['Hoxd10-63:+']\n",
      "NDB_1079 -> ['Hoxd10-84:+']\n",
      "NDB_1081 -> ['Hoxd10-65:+']\n",
      "NDB_1082 -> ['Hoxd10-86:+']\n",
      "NDB_1084 -> ['Hoxd10-67:+']\n",
      "NDB_1085 -> ['Hoxd10-88:+']\n",
      "NDB_1087 -> ['Hoxd10-69:+']\n",
      "NDB_1088 -> ['Hoxd10-90:+']\n",
      "NDB_1090 -> ['Hoxd10-71:+']\n",
      "NDB_1091 -> ['Hoxd10-92:+']\n",
      "NDB_1093 -> ['Hoxd10-73:+']\n",
      "NDB_1094 -> ['Hoxd10-94:+']\n",
      "NDB_1096 -> ['Hoxd10-75:+']\n",
      "NDB_1097 -> ['Hoxd10-96:+']\n",
      "NDB_1099 -> ['Hoxd10-77:+']\n",
      "NDB_1100 -> ['Hoxd10-98:+']\n",
      "NDB_1102 -> ['Hoxd10-79:+']\n",
      "NDB_1103 -> ['Hoxd10-100:+']\n",
      "NDB_1106 -> ['Hoxd10-102:+']\n",
      "NDB_1108 -> ['Hoxd10-81:+']\n",
      "NDB_1109 -> ['Hoxd10-104:+']\n",
      "NDB_1112 -> ['Hoxd10-106:+']\n",
      "NDB_1114 -> ['Hoxd10-83:+']\n",
      "NDB_1115 -> ['Hoxd10-108:+']\n",
      "NDB_1117 -> ['Hoxd10-85:+']\n",
      "NDB_1118 -> ['Hoxd10-110:+']\n",
      "NDB_1120 -> ['Hoxd10-87:+']\n",
      "NDB_1121 -> ['Hoxd10-112:+']\n",
      "NDB_1123 -> ['Hoxd10-89:+']\n",
      "NDB_1124 -> ['Hoxd10-114:+']\n",
      "NDB_1126 -> ['Hoxd10-91:+']\n",
      "NDB_1127 -> ['Hoxd10-116:+']\n",
      "NDB_1129 -> ['Hoxd10-93:+']\n",
      "NDB_1130 -> ['Hoxd10-118:+']\n",
      "NDB_1132 -> ['Hoxd10-95:+']\n",
      "NDB_1133 -> ['Spp2-2:+']\n",
      "NDB_1135 -> ['Hoxd10-97:+']\n",
      "NDB_1136 -> ['Spp2-4:+']\n",
      "NDB_1138 -> ['Hoxd10-99:+']\n",
      "NDB_1139 -> ['Spp2-6:+']\n",
      "NDB_1141 -> ['Hoxd10-101:+']\n",
      "NDB_1142 -> ['Spp2-8:+']\n",
      "NDB_1144 -> ['Hoxd10-103:+']\n",
      "NDB_1145 -> ['Spp2-10:+']\n",
      "NDB_1147 -> ['Hoxd10-105:+']\n",
      "NDB_1148 -> ['Spp2-12:+']\n",
      "NDB_1150 -> ['Hoxd10-107:+']\n",
      "NDB_1151 -> ['Spp2-14:+']\n",
      "NDB_1153 -> ['Hoxd10-109:+']\n",
      "NDB_1154 -> ['Spp2-16:+']\n",
      "NDB_1156 -> ['Hoxd10-111:+']\n",
      "NDB_1157 -> ['Spp2-18:+']\n",
      "NDB_1159 -> ['Hoxd10-113:+']\n",
      "NDB_1160 -> ['Spp2-20:+']\n",
      "NDB_1162 -> ['Hoxd10-115:+']\n",
      "NDB_1163 -> ['Spp2-22:+']\n",
      "NDB_1165 -> ['Hoxd10-117:+']\n",
      "NDB_1166 -> ['Spp2-24:+']\n",
      "NDB_1168 -> ['Spp2-1:+']\n",
      "NDB_1169 -> ['Spp2-26:+']\n",
      "NDB_1171 -> ['Spp2-3:+']\n",
      "NDB_1172 -> ['Spp2-28:+']\n",
      "NDB_1174 -> ['Spp2-5:+']\n",
      "NDB_1175 -> ['Spp2-30:+']\n",
      "NDB_1177 -> ['Spp2-7:+']\n",
      "NDB_1178 -> ['Spp2-32:+']\n",
      "NDB_1180 -> ['Spp2-9:+']\n",
      "NDB_1181 -> ['Spp2-34:+']\n",
      "NDB_1183 -> ['Spp2-11:+']\n",
      "NDB_1184 -> ['Spp2-36:+']\n",
      "NDB_1186 -> ['Spp2-13:+']\n",
      "NDB_1187 -> ['Spp2-38:+']\n",
      "NDB_1189 -> ['Spp2-15:+']\n",
      "NDB_1190 -> ['Spp2-40:+']\n",
      "NDB_1192 -> ['Spp2-17:+']\n",
      "NDB_1193 -> ['Spp2-42:+']\n",
      "NDB_1195 -> ['Spp2-19:+']\n",
      "NDB_1196 -> ['Spp2-44:+']\n",
      "NDB_1198 -> ['Spp2-21:+']\n",
      "NDB_1199 -> ['Spp2-46:+']\n",
      "NDB_1201 -> ['Spp2-23:+']\n",
      "NDB_1202 -> ['Spp2-48:+']\n",
      "NDB_1204 -> ['Spp2-25:+']\n",
      "NDB_1205 -> ['Spp2-50:+']\n",
      "NDB_1207 -> ['Spp2-27:+']\n",
      "NDB_1208 -> ['Spp2-52:+']\n",
      "NDB_1210 -> ['Spp2-29:+']\n",
      "NDB_1211 -> ['Spp2-54:+']\n",
      "NDB_1213 -> ['Spp2-31:+']\n",
      "NDB_1214 -> ['Spp2-56:+']\n",
      "NDB_1216 -> ['Spp2-39:+']\n",
      "NDB_1217 -> ['Spp2-58:+']\n",
      "NDB_1219 -> ['Spp2-41:+']\n",
      "NDB_1222 -> ['Spp2-35:+']\n",
      "NDB_1223 -> ['Spp2-74:+']\n",
      "NDB_1225 -> ['Spp2-37:+']\n",
      "NDB_1226 -> ['Spp2-60:+']\n",
      "NDB_1228 -> ['Spp2-33:+']\n",
      "NDB_1231 -> ['Spp2-45:+']\n",
      "NDB_1232 -> ['Spp2-78:+']\n",
      "NDB_1234 -> ['Spp2-49:+']\n",
      "NDB_1235 -> ['Spp2-62:+']\n",
      "NDB_1237 -> ['Spp2-43:+']\n",
      "NDB_1240 -> ['Spp2-47:+']\n",
      "NDB_1243 -> ['Spp2-61:+']\n",
      "NDB_1244 -> ['Spp2-64:+']\n",
      "NDB_1246 -> ['Spp2-63:+']\n",
      "NDB_1249 -> ['Spp2-57:+']\n",
      "NDB_1252 -> ['Spp2-59:+']\n",
      "NDB_1253 -> ['Spp2-66:+']\n",
      "NDB_1255 -> ['Spp2-67:+']\n",
      "NDB_1258 -> ['Spp2-51:+']\n",
      "NDB_1259 -> ['Spp2-68:+']\n",
      "NDB_1261 -> ['Spp2-53:+']\n",
      "NDB_1264 -> ['Spp2-55:+']\n",
      "NDB_1265 -> ['Spp2-70:+']\n",
      "NDB_1267 -> ['Spp2-69:+']\n",
      "NDB_1270 -> ['Spp2-65:+']\n",
      "NDB_1271 -> ['Spp2-72:+']\n",
      "NDB_1273 -> ['Spp2-71:+']\n",
      "NDB_1276 -> ['Spp2-73:+']\n",
      "NDB_1277 -> ['Spp2-76:+']\n",
      "NDB_1279 -> ['Spp2-75:+']\n",
      "NDB_1282 -> ['Spp2-77:+']\n",
      "NDB_632 -> ['Camk1-2:-']\n",
      "NDB_640 -> ['Camk1-1:-']\n",
      "NDB_646 -> ['Camk1-3:-']\n",
      "NDB_655 -> ['Camk1-5:-']\n",
      "NDB_656 -> ['Camk1-4:-']\n",
      "NDB_658 -> ['Camk1-7:-']\n",
      "NDB_662 -> ['Camk1-6:-']\n",
      "NDB_664 -> ['Camk1-9:-']\n",
      "NDB_667 -> ['Camk1-11:-']\n",
      "NDB_671 -> ['Camk1-8:-']\n",
      "NDB_673 -> ['Camk1-13:-']\n",
      "NDB_674 -> ['Camk1-10:-']\n",
      "NDB_676 -> ['Camk1-15:-']\n",
      "NDB_677 -> ['Camk1-12:-']\n",
      "NDB_679 -> ['Camk1-17:-']\n",
      "NDB_682 -> ['Camk1-19:-']\n",
      "NDB_685 -> ['Camk1-21:-']\n",
      "NDB_686 -> ['Camk1-14:-']\n",
      "NDB_688 -> ['Camk1-23:-']\n",
      "NDB_692 -> ['Camk1-16:-']\n",
      "NDB_695 -> ['Camk1-18:-']\n",
      "NDB_697 -> ['Camk1-25:-']\n",
      "NDB_701 -> ['Camk1-20:-']\n",
      "NDB_704 -> ['Camk1-22:-']\n",
      "NDB_719 -> ['Camk1-24:-']\n",
      "NDB_722 -> ['Camk1-26:-']\n",
      "NDB_724 -> ['Camk1-27:-']\n",
      "NDB_731 -> ['Camk1-28:-']\n",
      "NDB_733 -> ['Camk1-29:-']\n",
      "NDB_745 -> ['Camk1-31:-']\n",
      "NDB_757 -> ['Camk1-33:-']\n",
      "NDB_763 -> ['Camk1-35:-']\n",
      "NDB_764 -> ['Camk1-30:-']\n",
      "NDB_766 -> ['Camk1-37:-']\n",
      "NDB_769 -> ['Camk1-39:-']\n",
      "NDB_770 -> ['Camk1-32:-']\n",
      "NDB_781 -> ['Camk1-41:-']\n",
      "NDB_782 -> ['Camk1-34:-']\n",
      "NDB_787 -> ['Camk1-43:-']\n",
      "NDB_794 -> ['Camk1-36:-']\n",
      "NDB_797 -> ['Camk1-38:-']\n",
      "NDB_800 -> ['Camk1-40:-']\n",
      "NDB_803 -> ['Camk1-42:-']\n",
      "NDB_808 -> ['Camk1-45:-']\n",
      "NDB_811 -> ['Camk1-47:-']\n",
      "NDB_820 -> ['Camk1-49:-']\n",
      "NDB_827 -> ['Camk1-44:-']\n",
      "NDB_830 -> ['Camk1-46:-']\n",
      "NDB_833 -> ['Camk1-48:-']\n",
      "NDB_839 -> ['Camk1-50:-']\n",
      "NDB_848 -> ['Camk1-52:-']\n",
      "NDB_850 -> ['Camk1-51:-']\n",
      "NDB_851 -> ['Camk1-54:-']\n",
      "NDB_854 -> ['Hoxd10-2:+']\n",
      "NDB_857 -> ['Hoxd10-4:+']\n",
      "NDB_874 -> ['Camk1-53:-']\n",
      "NDB_877 -> ['Hoxd10-1:+']\n",
      "NDB_884 -> ['Hoxd10-6:+']\n",
      "NDB_886 -> ['Hoxd10-3:+']\n",
      "NDB_890 -> ['Hoxd10-8:+']\n",
      "NDB_892 -> ['Hoxd10-5:+']\n",
      "NDB_893 -> ['Hoxd10-10:+']\n",
      "NDB_898 -> ['Hoxd10-7:+']\n",
      "NDB_913 -> ['Hoxd10-9:+']\n",
      "NDB_916 -> ['Hoxd10-11:+']\n",
      "NDB_919 -> ['Hoxd10-13:+']\n",
      "NDB_923 -> ['Hoxd10-12:+']\n",
      "NDB_932 -> ['Hoxd10-14:+']\n",
      "NDB_937 -> ['Hoxd10-15:+']\n",
      "NDB_938 -> ['Hoxd10-16:+']\n",
      "NDB_943 -> ['Hoxd10-17:+']\n",
      "NDB_947 -> ['Hoxd10-18:+']\n",
      "NDB_950 -> ['Hoxd10-20:+']\n",
      "NDB_952 -> ['Hoxd10-19:+']\n",
      "NDB_953 -> ['Hoxd10-22:+']\n",
      "NDB_956 -> ['Hoxd10-24:+']\n",
      "NDB_959 -> ['Hoxd10-26:+']\n",
      "NDB_962 -> ['Hoxd10-28:+']\n",
      "NDB_964 -> ['Hoxd10-21:+']\n",
      "NDB_967 -> ['Hoxd10-23:+']\n",
      "NDB_968 -> ['Hoxd10-30:+']\n",
      "NDB_970 -> ['Hoxd10-25:+']\n",
      "NDB_979 -> ['Hoxd10-27:+']\n",
      "NDB_980 -> ['Hoxd10-32:+']\n",
      "NDB_982 -> ['Hoxd10-29:+']\n",
      "NDB_985 -> ['Hoxd10-31:+']\n",
      "NDB_991 -> ['Hoxd10-33:+']\n",
      "NDB_992 -> ['Hoxd10-34:+']\n",
      "NDB_997 -> ['Hoxd10-35:+']\n",
      "NDB_998 -> ['Hoxd10-36:+']\n",
      "True\n"
     ]
    }
   ],
   "source": [
    "reload(check)\n",
    "readout_reg_dic, readout2reg_check = check._check_readout_to_region(reg_readout_dic, full_records, \n",
    "                                                                    check_readout_dict, primer_len=primer_len, \n",
    "                                                                    target_len=target_len,\n",
    "                                                                    probe_readout_num=num_readout_per_probe)\n",
    "\n",
    "print(readout2reg_check)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "graduate-scheduling",
   "metadata": {},
   "source": [
    "### 4.1.5 check bewteen probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 174,
   "id": "american-forestry",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- saving internal 17-mer map to file:\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\probe_table_17.npz\n"
     ]
    }
   ],
   "source": [
    "# construct internal map\n",
    "reload(check)\n",
    "int_map = check._construct_internal_map(full_records, library_folder) "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 177,
   "id": "aggressive-barrier",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "True\n"
     ]
    }
   ],
   "source": [
    "# internal check probes\n",
    "reload(check)\n",
    "readout_count_dic, readout_count_check = check._check_readout_in_probes(readout_reg_dic, reg_size_dic, int_map, readout_dict)\n",
    "print(readout_count_check)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 179,
   "id": "legendary-password",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- Sequence:loc_6:113284124-113294124_gene_Camk1-6:-_pb_53_pos_3540_strand_-_readouts_[NDB_662_u,NDB_662_u,NDB_662_u] got hits:53, dumped\n",
      "--- Sequence:loc_6:113284124-113294124_gene_Camk1-6:-_pb_54_pos_3581_strand_+_readouts_[NDB_662_u,NDB_662_u,NDB_662_u] got hits:52, dumped\n",
      "--- Sequence:loc_2:75741924-75751924_gene_Hoxd10-116:+_pb_65_pos_6013_strand_+_readouts_[NDB_1127_u,NDB_1127_u,NDB_1127_u] got hits:90, dumped\n",
      "--- Sequence:loc_2:75741924-75751924_gene_Hoxd10-116:+_pb_66_pos_6080_strand_+_readouts_[NDB_1127_u,NDB_1127_u,NDB_1127_u] got hits:91, dumped\n",
      "--- Sequence:loc_2:75741924-75751924_gene_Hoxd10-116:+_pb_67_pos_6518_strand_-_readouts_[NDB_1127_u,NDB_1127_u,NDB_1127_u] got hits:80, dumped\n",
      "--- Sequence:loc_2:75741924-75751924_gene_Hoxd10-116:+_pb_69_pos_6635_strand_+_readouts_[NDB_1127_u,NDB_1127_u,NDB_1127_u] got hits:71, dumped\n",
      "--- Sequence:loc_2:75741924-75751924_gene_Hoxd10-116:+_pb_70_pos_6754_strand_-_readouts_[NDB_1127_u,NDB_1127_u,NDB_1127_u] got hits:95, dumped\n",
      "--- Sequence:loc_2:74951924-74961924_gene_Hoxd10-37:+_pb_54_pos_4019_strand_+_readouts_[NDB_1000_u,NDB_1000_u,NDB_1000_u] got hits:55, dumped\n",
      "--- Sequence:loc_2:74951924-74961924_gene_Hoxd10-37:+_pb_55_pos_4060_strand_+_readouts_[NDB_1000_u,NDB_1000_u,NDB_1000_u] got hits:56, dumped\n",
      "--- Sequence:loc_1:88586961-88596961_gene_Spp2-29:+_pb_117_pos_8631_strand_+_readouts_[NDB_1210_u,NDB_1210_u,NDB_1210_u] got hits:125, dumped\n",
      "--- Sequence:loc_1:88586961-88596961_gene_Spp2-29:+_pb_118_pos_8711_strand_+_readouts_[NDB_1210_u,NDB_1210_u,NDB_1210_u] got hits:66, dumped\n",
      "--- Sequence:loc_1:88586961-88596961_gene_Spp2-29:+_pb_119_pos_8778_strand_+_readouts_[NDB_1210_u,NDB_1210_u,NDB_1210_u] got hits:54, dumped\n",
      "--- Sequence:loc_1:88586961-88596961_gene_Spp2-29:+_pb_121_pos_8901_strand_+_readouts_[NDB_1210_u,NDB_1210_u,NDB_1210_u] got hits:109, dumped\n",
      "--- Sequence:loc_1:88586961-88596961_gene_Spp2-29:+_pb_122_pos_8945_strand_+_readouts_[NDB_1210_u,NDB_1210_u,NDB_1210_u] got hits:129, dumped\n",
      "--- Sequence:loc_1:88586961-88596961_gene_Spp2-29:+_pb_124_pos_9059_strand_-_readouts_[NDB_1210_u,NDB_1210_u,NDB_1210_u] got hits:99, dumped\n",
      "--- Sequence:loc_1:88586961-88596961_gene_Spp2-29:+_pb_125_pos_9137_strand_+_readouts_[NDB_1210_u,NDB_1210_u,NDB_1210_u] got hits:56, dumped\n",
      "--- Sequence:loc_1:88586961-88596961_gene_Spp2-29:+_pb_126_pos_9178_strand_+_readouts_[NDB_1210_u,NDB_1210_u,NDB_1210_u] got hits:97, dumped\n",
      "--- Sequence:loc_1:88586961-88596961_gene_Spp2-29:+_pb_127_pos_9245_strand_+_readouts_[NDB_1210_u,NDB_1210_u,NDB_1210_u] got hits:103, dumped\n",
      "--- Sequence:loc_1:88326961-88336961_gene_Spp2-3:+_pb_59_pos_4026_strand_-_readouts_[NDB_1171_u,NDB_1171_u,NDB_1171_u] got hits:94, dumped\n",
      "--- Sequence:loc_1:88326961-88336961_gene_Spp2-3:+_pb_60_pos_4159_strand_+_readouts_[NDB_1171_u,NDB_1171_u,NDB_1171_u] got hits:59, dumped\n",
      "--- Sequence:loc_1:88326961-88336961_gene_Spp2-3:+_pb_61_pos_4353_strand_-_readouts_[NDB_1171_u,NDB_1171_u,NDB_1171_u] got hits:82, dumped\n",
      "--- Sequence:loc_1:88326961-88336961_gene_Spp2-3:+_pb_62_pos_4446_strand_-_readouts_[NDB_1171_u,NDB_1171_u,NDB_1171_u] got hits:63, dumped\n",
      "--- Sequence:loc_1:88326961-88336961_gene_Spp2-3:+_pb_63_pos_4701_strand_-_readouts_[NDB_1171_u,NDB_1171_u,NDB_1171_u] got hits:90, dumped\n",
      "--- Sequence:loc_1:88326961-88336961_gene_Spp2-3:+_pb_64_pos_4744_strand_-_readouts_[NDB_1171_u,NDB_1171_u,NDB_1171_u] got hits:96, dumped\n",
      "--- Sequence:loc_1:88376961-88386961_gene_Spp2-8:+_pb_129_pos_8824_strand_+_readouts_[NDB_1142_u,NDB_1142_u,NDB_1142_u] got hits:55, dumped\n",
      "--- Sequence:loc_1:88376961-88386961_gene_Spp2-8:+_pb_132_pos_8975_strand_+_readouts_[NDB_1142_u,NDB_1142_u,NDB_1142_u] got hits:53, dumped\n",
      "--- Sequence:loc_1:88376961-88386961_gene_Spp2-8:+_pb_133_pos_9033_strand_-_readouts_[NDB_1142_u,NDB_1142_u,NDB_1142_u] got hits:57, dumped\n",
      "--- Sequence:loc_1:88376961-88386961_gene_Spp2-8:+_pb_134_pos_9094_strand_+_readouts_[NDB_1142_u,NDB_1142_u,NDB_1142_u] got hits:60, dumped\n",
      "--- Sequence:loc_1:88376961-88386961_gene_Spp2-8:+_pb_135_pos_9145_strand_+_readouts_[NDB_1142_u,NDB_1142_u,NDB_1142_u] got hits:55, dumped\n",
      "-- total probes removed by internal screening: 29\n"
     ]
    }
   ],
   "source": [
    "# check between probes\n",
    "reload(check)\n",
    "kept_records, removed_count = check._check_between_probes(full_records, int_map, \n",
    "                                                          primer_len=primer_len, target_len=target_len)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 180,
   "id": "offensive-webster",
   "metadata": {},
   "outputs": [],
   "source": [
    "del(int_map)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "surface-calendar",
   "metadata": {},
   "source": [
    "### 4.1.6 finish checking, save"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 181,
   "id": "injured-grave",
   "metadata": {},
   "outputs": [],
   "source": [
    "# save kept records\n",
    "with open(os.path.join(library_folder, 'filtered_full_probes.fasta'), 'w') as output_handle:\n",
    "    SeqIO.write(kept_records, output_handle, \"fasta\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "extraordinary-speech",
   "metadata": {},
   "source": [
    "## 4.2 BLAST screening of probes\n",
    "\n",
    "create the local blast database by:\n",
    "```bash\n",
    "makeblastdb -in C:\\Users\\puzheng\\Documents\\Tools\\lowercase_masked\\Mus_musculus.GRCm38.dna_sm.merged.fa -dbtype nucl -parse_seqids -out GRCm38_ensembl -title \"Mouse chromosomes soft masked\" -blastdb_version 4![image.png](attachment:image.png)\n",
    "```\n",
    "\n",
    "The outputs are:\n",
    "```bash\n",
    "Building a new DB, current time: 03/30/2021 21:49:18\n",
    "New DB name:   C:\\Users\\puzheng\\Documents\\Tools\\GRCm38_ensembl\n",
    "New DB title:  Mouse chromosomes soft masked\n",
    "Sequence type: Nucleotide\n",
    "Keep MBits: T\n",
    "Maximum file size: 1000000000B\n",
    "Adding sequences from FASTA; added 66 sequences in 42.2668 seconds.\n",
    "```"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 182,
   "id": "biological-steering",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Blast probe against \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\blast_database\\GRCm38_ensembl\n",
      "-- number of probes loaded: 36608\n",
      "-- processing region: Camk1-10:- with 140 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-10:-.fasta\n",
      "--- *blasting region: Camk1-10:-\n",
      "--- total time for blast Camk1-10:-: 12.392135620117188\n",
      "-- processing region: Camk1-11:- with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-11:-.fasta\n",
      "--- *blasting region: Camk1-11:-\n",
      "--- total time for blast Camk1-11:-: 29.53890609741211\n",
      "-- processing region: Camk1-12:- with 109 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-12:-.fasta\n",
      "--- *blasting region: Camk1-12:-\n",
      "--- total time for blast Camk1-12:-: 7.975998640060425\n",
      "-- processing region: Camk1-13:- with 110 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-13:-.fasta\n",
      "--- *blasting region: Camk1-13:-\n",
      "--- total time for blast Camk1-13:-: 11.670998334884644\n",
      "-- processing region: Camk1-14:- with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-14:-.fasta\n",
      "--- *blasting region: Camk1-14:-\n",
      "--- total time for blast Camk1-14:-: 12.497000694274902\n",
      "-- processing region: Camk1-15:- with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-15:-.fasta\n",
      "--- *blasting region: Camk1-15:-\n",
      "--- total time for blast Camk1-15:-: 6.9410011768341064\n",
      "-- processing region: Camk1-16:- with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-16:-.fasta\n",
      "--- *blasting region: Camk1-16:-\n",
      "--- total time for blast Camk1-16:-: 29.2450008392334\n",
      "-- processing region: Camk1-17:- with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-17:-.fasta\n",
      "--- *blasting region: Camk1-17:-\n",
      "--- total time for blast Camk1-17:-: 11.676998376846313\n",
      "-- processing region: Camk1-18:- with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-18:-.fasta\n",
      "--- *blasting region: Camk1-18:-\n",
      "--- total time for blast Camk1-18:-: 10.179255723953247\n",
      "-- processing region: Camk1-19:- with 148 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-19:-.fasta\n",
      "--- *blasting region: Camk1-19:-\n",
      "--- total time for blast Camk1-19:-: 11.940005540847778\n",
      "-- processing region: Camk1-1:- with 149 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-1:-.fasta\n",
      "--- *blasting region: Camk1-1:-\n",
      "--- total time for blast Camk1-1:-: 8.855993747711182\n",
      "-- processing region: Camk1-20:- with 143 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-20:-.fasta\n",
      "--- *blasting region: Camk1-20:-\n",
      "--- total time for blast Camk1-20:-: 12.153999328613281\n",
      "-- processing region: Camk1-21:- with 136 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-21:-.fasta\n",
      "--- *blasting region: Camk1-21:-\n",
      "--- total time for blast Camk1-21:-: 12.913000106811523\n",
      "-- processing region: Camk1-22:- with 141 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-22:-.fasta\n",
      "--- *blasting region: Camk1-22:-\n",
      "--- total time for blast Camk1-22:-: 15.761998176574707\n",
      "-- processing region: Camk1-23:- with 136 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-23:-.fasta\n",
      "--- *blasting region: Camk1-23:-\n",
      "--- total time for blast Camk1-23:-: 14.096002340316772\n",
      "-- processing region: Camk1-24:- with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-24:-.fasta\n",
      "--- *blasting region: Camk1-24:-\n",
      "--- total time for blast Camk1-24:-: 10.727998971939087\n",
      "-- processing region: Camk1-25:- with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-25:-.fasta\n",
      "--- *blasting region: Camk1-25:-\n",
      "--- total time for blast Camk1-25:-: 11.875275611877441\n",
      "-- processing region: Camk1-26:- with 141 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-26:-.fasta\n",
      "--- *blasting region: Camk1-26:-\n",
      "--- total time for blast Camk1-26:-: 9.99999737739563\n",
      "-- processing region: Camk1-27:- with 147 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-27:-.fasta\n",
      "--- *blasting region: Camk1-27:-\n",
      "--- total time for blast Camk1-27:-: 10.522001028060913\n",
      "-- processing region: Camk1-28:- with 113 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-28:-.fasta\n",
      "--- *blasting region: Camk1-28:-\n",
      "--- total time for blast Camk1-28:-: 9.334000825881958\n",
      "-- processing region: Camk1-29:- with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-29:-.fasta\n",
      "--- *blasting region: Camk1-29:-\n",
      "--- total time for blast Camk1-29:-: 13.32665205001831\n",
      "-- processing region: Camk1-2:- with 140 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-2:-.fasta\n",
      "--- *blasting region: Camk1-2:-\n",
      "--- total time for blast Camk1-2:-: 7.8820013999938965\n",
      "-- processing region: Camk1-30:- with 138 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-30:-.fasta\n",
      "--- *blasting region: Camk1-30:-\n",
      "--- total time for blast Camk1-30:-: 14.65800142288208\n",
      "-- processing region: Camk1-31:- with 103 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-31:-.fasta\n",
      "--- *blasting region: Camk1-31:-\n",
      "--- total time for blast Camk1-31:-: 9.95199990272522\n",
      "-- processing region: Camk1-32:- with 132 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-32:-.fasta\n",
      "--- *blasting region: Camk1-32:-\n",
      "--- total time for blast Camk1-32:-: 9.71799921989441\n",
      "-- processing region: Camk1-33:- with 135 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-33:-.fasta\n",
      "--- *blasting region: Camk1-33:-\n",
      "--- total time for blast Camk1-33:-: 8.987000226974487\n",
      "-- processing region: Camk1-34:- with 126 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-34:-.fasta\n",
      "--- *blasting region: Camk1-34:-\n",
      "--- total time for blast Camk1-34:-: 5.71999979019165\n",
      "-- processing region: Camk1-35:- with 131 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-35:-.fasta\n",
      "--- *blasting region: Camk1-35:-\n",
      "--- total time for blast Camk1-35:-: 7.975001335144043\n",
      "-- processing region: Camk1-36:- with 133 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-36:-.fasta\n",
      "--- *blasting region: Camk1-36:-\n",
      "--- total time for blast Camk1-36:-: 13.399720191955566\n",
      "-- processing region: Camk1-37:- with 131 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-37:-.fasta\n",
      "--- *blasting region: Camk1-37:-\n",
      "--- total time for blast Camk1-37:-: 8.578001260757446\n",
      "-- processing region: Camk1-38:- with 142 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-38:-.fasta\n",
      "--- *blasting region: Camk1-38:-\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Camk1-38:-: 9.337525129318237\n",
      "-- processing region: Camk1-39:- with 142 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-39:-.fasta\n",
      "--- *blasting region: Camk1-39:-\n",
      "--- total time for blast Camk1-39:-: 24.34396266937256\n",
      "-- processing region: Camk1-3:- with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-3:-.fasta\n",
      "--- *blasting region: Camk1-3:-\n",
      "--- total time for blast Camk1-3:-: 11.654840469360352\n",
      "-- processing region: Camk1-40:- with 136 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-40:-.fasta\n",
      "--- *blasting region: Camk1-40:-\n",
      "--- total time for blast Camk1-40:-: 7.753001689910889\n",
      "-- processing region: Camk1-41:- with 121 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-41:-.fasta\n",
      "--- *blasting region: Camk1-41:-\n",
      "--- total time for blast Camk1-41:-: 37.60646677017212\n",
      "-- processing region: Camk1-42:- with 139 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-42:-.fasta\n",
      "--- *blasting region: Camk1-42:-\n",
      "--- total time for blast Camk1-42:-: 6.942000389099121\n",
      "-- processing region: Camk1-43:- with 132 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-43:-.fasta\n",
      "--- *blasting region: Camk1-43:-\n",
      "--- total time for blast Camk1-43:-: 23.18414044380188\n",
      "-- processing region: Camk1-44:- with 133 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-44:-.fasta\n",
      "--- *blasting region: Camk1-44:-\n",
      "--- total time for blast Camk1-44:-: 17.307178020477295\n",
      "-- processing region: Camk1-45:- with 144 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-45:-.fasta\n",
      "--- *blasting region: Camk1-45:-\n",
      "--- total time for blast Camk1-45:-: 18.293999433517456\n",
      "-- processing region: Camk1-46:- with 139 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-46:-.fasta\n",
      "--- *blasting region: Camk1-46:-\n",
      "--- total time for blast Camk1-46:-: 10.736000776290894\n",
      "-- processing region: Camk1-47:- with 141 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-47:-.fasta\n",
      "--- *blasting region: Camk1-47:-\n",
      "--- total time for blast Camk1-47:-: 7.552412033081055\n",
      "-- processing region: Camk1-48:- with 145 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-48:-.fasta\n",
      "--- *blasting region: Camk1-48:-\n",
      "--- total time for blast Camk1-48:-: 14.755001544952393\n",
      "-- processing region: Camk1-49:- with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-49:-.fasta\n",
      "--- *blasting region: Camk1-49:-\n",
      "--- total time for blast Camk1-49:-: 8.487998723983765\n",
      "-- processing region: Camk1-4:- with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-4:-.fasta\n",
      "--- *blasting region: Camk1-4:-\n",
      "--- total time for blast Camk1-4:-: 8.010001182556152\n",
      "-- processing region: Camk1-50:- with 149 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-50:-.fasta\n",
      "--- *blasting region: Camk1-50:-\n",
      "--- total time for blast Camk1-50:-: 26.618000268936157\n",
      "-- processing region: Camk1-51:- with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-51:-.fasta\n",
      "--- *blasting region: Camk1-51:-\n",
      "--- total time for blast Camk1-51:-: 25.696999549865723\n",
      "-- processing region: Camk1-52:- with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-52:-.fasta\n",
      "--- *blasting region: Camk1-52:-\n",
      "--- total time for blast Camk1-52:-: 6.4709999561309814\n",
      "-- processing region: Camk1-53:- with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-53:-.fasta\n",
      "--- *blasting region: Camk1-53:-\n",
      "--- total time for blast Camk1-53:-: 9.508999347686768\n",
      "-- processing region: Camk1-54:- with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-54:-.fasta\n",
      "--- *blasting region: Camk1-54:-\n",
      "--- total time for blast Camk1-54:-: 5.799999475479126\n",
      "-- processing region: Camk1-5:- with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-5:-.fasta\n",
      "--- *blasting region: Camk1-5:-\n",
      "--- total time for blast Camk1-5:-: 9.25300121307373\n",
      "-- processing region: Camk1-6:- with 148 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-6:-.fasta\n",
      "--- *blasting region: Camk1-6:-\n",
      "--- total time for blast Camk1-6:-: 11.697850465774536\n",
      "-- processing region: Camk1-7:- with 139 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-7:-.fasta\n",
      "--- *blasting region: Camk1-7:-\n",
      "--- total time for blast Camk1-7:-: 10.246997594833374\n",
      "-- processing region: Camk1-8:- with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-8:-.fasta\n",
      "--- *blasting region: Camk1-8:-\n",
      "--- total time for blast Camk1-8:-: 12.991000413894653\n",
      "-- processing region: Camk1-9:- with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Camk1-9:-.fasta\n",
      "--- *blasting region: Camk1-9:-\n",
      "--- total time for blast Camk1-9:-: 6.45799994468689\n",
      "-- processing region: Hoxd10-100:+ with 135 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-100:+.fasta\n",
      "--- *blasting region: Hoxd10-100:+\n",
      "--- total time for blast Hoxd10-100:+: 14.39400029182434\n",
      "-- processing region: Hoxd10-101:+ with 138 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-101:+.fasta\n",
      "--- *blasting region: Hoxd10-101:+\n",
      "--- total time for blast Hoxd10-101:+: 9.394001245498657\n",
      "-- processing region: Hoxd10-102:+ with 144 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-102:+.fasta\n",
      "--- *blasting region: Hoxd10-102:+\n",
      "--- total time for blast Hoxd10-102:+: 10.624001026153564\n",
      "-- processing region: Hoxd10-103:+ with 118 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-103:+.fasta\n",
      "--- *blasting region: Hoxd10-103:+\n",
      "--- total time for blast Hoxd10-103:+: 10.899000644683838\n",
      "-- processing region: Hoxd10-104:+ with 118 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-104:+.fasta\n",
      "--- *blasting region: Hoxd10-104:+\n",
      "--- total time for blast Hoxd10-104:+: 6.71900200843811\n",
      "-- processing region: Hoxd10-105:+ with 73 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-105:+.fasta\n",
      "--- *blasting region: Hoxd10-105:+\n",
      "--- total time for blast Hoxd10-105:+: 4.9250006675720215\n",
      "-- processing region: Hoxd10-106:+ with 108 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-106:+.fasta\n",
      "--- *blasting region: Hoxd10-106:+\n",
      "--- total time for blast Hoxd10-106:+: 9.146000623703003\n",
      "-- processing region: Hoxd10-107:+ with 126 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-107:+.fasta\n",
      "--- *blasting region: Hoxd10-107:+\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Hoxd10-107:+: 7.447998762130737\n",
      "-- processing region: Hoxd10-108:+ with 107 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-108:+.fasta\n",
      "--- *blasting region: Hoxd10-108:+\n",
      "--- total time for blast Hoxd10-108:+: 67.81745767593384\n",
      "-- processing region: Hoxd10-109:+ with 132 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-109:+.fasta\n",
      "--- *blasting region: Hoxd10-109:+\n",
      "--- total time for blast Hoxd10-109:+: 13.780049562454224\n",
      "-- processing region: Hoxd10-10:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-10:+.fasta\n",
      "--- *blasting region: Hoxd10-10:+\n",
      "--- total time for blast Hoxd10-10:+: 5.8699517250061035\n",
      "-- processing region: Hoxd10-110:+ with 142 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-110:+.fasta\n",
      "--- *blasting region: Hoxd10-110:+\n",
      "--- total time for blast Hoxd10-110:+: 10.519999742507935\n",
      "-- processing region: Hoxd10-111:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-111:+.fasta\n",
      "--- *blasting region: Hoxd10-111:+\n",
      "--- total time for blast Hoxd10-111:+: 9.78499984741211\n",
      "-- processing region: Hoxd10-112:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-112:+.fasta\n",
      "--- *blasting region: Hoxd10-112:+\n",
      "--- total time for blast Hoxd10-112:+: 8.427000284194946\n",
      "-- processing region: Hoxd10-113:+ with 129 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-113:+.fasta\n",
      "--- *blasting region: Hoxd10-113:+\n",
      "--- total time for blast Hoxd10-113:+: 19.06951117515564\n",
      "-- processing region: Hoxd10-114:+ with 129 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-114:+.fasta\n",
      "--- *blasting region: Hoxd10-114:+\n",
      "--- total time for blast Hoxd10-114:+: 8.267001152038574\n",
      "-- processing region: Hoxd10-115:+ with 141 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-115:+.fasta\n",
      "--- *blasting region: Hoxd10-115:+\n",
      "--- total time for blast Hoxd10-115:+: 9.793002605438232\n",
      "-- processing region: Hoxd10-116:+ with 108 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-116:+.fasta\n",
      "--- *blasting region: Hoxd10-116:+\n",
      "--- total time for blast Hoxd10-116:+: 7.104002237319946\n",
      "-- processing region: Hoxd10-117:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-117:+.fasta\n",
      "--- *blasting region: Hoxd10-117:+\n",
      "--- total time for blast Hoxd10-117:+: 18.472220420837402\n",
      "-- processing region: Hoxd10-118:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-118:+.fasta\n",
      "--- *blasting region: Hoxd10-118:+\n",
      "--- total time for blast Hoxd10-118:+: 16.50200057029724\n",
      "-- processing region: Hoxd10-11:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-11:+.fasta\n",
      "--- *blasting region: Hoxd10-11:+\n",
      "--- total time for blast Hoxd10-11:+: 6.155000925064087\n",
      "-- processing region: Hoxd10-12:+ with 147 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-12:+.fasta\n",
      "--- *blasting region: Hoxd10-12:+\n",
      "--- total time for blast Hoxd10-12:+: 6.672000885009766\n",
      "-- processing region: Hoxd10-13:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-13:+.fasta\n",
      "--- *blasting region: Hoxd10-13:+\n",
      "--- total time for blast Hoxd10-13:+: 10.160998821258545\n",
      "-- processing region: Hoxd10-14:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-14:+.fasta\n",
      "--- *blasting region: Hoxd10-14:+\n",
      "--- total time for blast Hoxd10-14:+: 7.454000473022461\n",
      "-- processing region: Hoxd10-15:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-15:+.fasta\n",
      "--- *blasting region: Hoxd10-15:+\n",
      "--- total time for blast Hoxd10-15:+: 5.063001394271851\n",
      "-- processing region: Hoxd10-16:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-16:+.fasta\n",
      "--- *blasting region: Hoxd10-16:+\n",
      "--- total time for blast Hoxd10-16:+: 7.1526501178741455\n",
      "-- processing region: Hoxd10-17:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-17:+.fasta\n",
      "--- *blasting region: Hoxd10-17:+\n",
      "--- total time for blast Hoxd10-17:+: 10.799195528030396\n",
      "-- processing region: Hoxd10-18:+ with 144 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-18:+.fasta\n",
      "--- *blasting region: Hoxd10-18:+\n",
      "--- total time for blast Hoxd10-18:+: 6.296999931335449\n",
      "-- processing region: Hoxd10-19:+ with 128 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-19:+.fasta\n",
      "--- *blasting region: Hoxd10-19:+\n",
      "--- total time for blast Hoxd10-19:+: 17.399999141693115\n",
      "-- processing region: Hoxd10-1:+ with 121 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-1:+.fasta\n",
      "--- *blasting region: Hoxd10-1:+\n",
      "--- total time for blast Hoxd10-1:+: 18.55800151824951\n",
      "-- processing region: Hoxd10-20:+ with 127 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-20:+.fasta\n",
      "--- *blasting region: Hoxd10-20:+\n",
      "--- total time for blast Hoxd10-20:+: 7.470998048782349\n",
      "-- processing region: Hoxd10-21:+ with 116 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-21:+.fasta\n",
      "--- *blasting region: Hoxd10-21:+\n",
      "--- total time for blast Hoxd10-21:+: 13.055201768875122\n",
      "-- processing region: Hoxd10-22:+ with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-22:+.fasta\n",
      "--- *blasting region: Hoxd10-22:+\n",
      "--- total time for blast Hoxd10-22:+: 8.804999828338623\n",
      "-- processing region: Hoxd10-23:+ with 119 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-23:+.fasta\n",
      "--- *blasting region: Hoxd10-23:+\n",
      "--- total time for blast Hoxd10-23:+: 8.423000812530518\n",
      "-- processing region: Hoxd10-24:+ with 123 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-24:+.fasta\n",
      "--- *blasting region: Hoxd10-24:+\n",
      "--- total time for blast Hoxd10-24:+: 6.550748348236084\n",
      "-- processing region: Hoxd10-25:+ with 118 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-25:+.fasta\n",
      "--- *blasting region: Hoxd10-25:+\n",
      "--- total time for blast Hoxd10-25:+: 8.186001062393188\n",
      "-- processing region: Hoxd10-26:+ with 118 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-26:+.fasta\n",
      "--- *blasting region: Hoxd10-26:+\n",
      "--- total time for blast Hoxd10-26:+: 9.769999265670776\n",
      "-- processing region: Hoxd10-27:+ with 105 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-27:+.fasta\n",
      "--- *blasting region: Hoxd10-27:+\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Hoxd10-27:+: 77.93261909484863\n",
      "-- processing region: Hoxd10-28:+ with 109 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-28:+.fasta\n",
      "--- *blasting region: Hoxd10-28:+\n",
      "--- total time for blast Hoxd10-28:+: 9.436001777648926\n",
      "-- processing region: Hoxd10-29:+ with 125 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-29:+.fasta\n",
      "--- *blasting region: Hoxd10-29:+\n",
      "--- total time for blast Hoxd10-29:+: 8.259000062942505\n",
      "-- processing region: Hoxd10-2:+ with 84 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-2:+.fasta\n",
      "--- *blasting region: Hoxd10-2:+\n",
      "--- total time for blast Hoxd10-2:+: 13.399704217910767\n",
      "-- processing region: Hoxd10-30:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-30:+.fasta\n",
      "--- *blasting region: Hoxd10-30:+\n",
      "--- total time for blast Hoxd10-30:+: 5.689000368118286\n",
      "-- processing region: Hoxd10-31:+ with 148 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-31:+.fasta\n",
      "--- *blasting region: Hoxd10-31:+\n",
      "--- total time for blast Hoxd10-31:+: 11.263998031616211\n",
      "-- processing region: Hoxd10-32:+ with 119 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-32:+.fasta\n",
      "--- *blasting region: Hoxd10-32:+\n",
      "--- total time for blast Hoxd10-32:+: 8.989001750946045\n",
      "-- processing region: Hoxd10-33:+ with 130 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-33:+.fasta\n",
      "--- *blasting region: Hoxd10-33:+\n",
      "--- total time for blast Hoxd10-33:+: 11.546999454498291\n",
      "-- processing region: Hoxd10-34:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-34:+.fasta\n",
      "--- *blasting region: Hoxd10-34:+\n",
      "--- total time for blast Hoxd10-34:+: 9.64099907875061\n",
      "-- processing region: Hoxd10-35:+ with 149 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-35:+.fasta\n",
      "--- *blasting region: Hoxd10-35:+\n",
      "--- total time for blast Hoxd10-35:+: 9.453002452850342\n",
      "-- processing region: Hoxd10-36:+ with 146 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-36:+.fasta\n",
      "--- *blasting region: Hoxd10-36:+\n",
      "--- total time for blast Hoxd10-36:+: 10.1750009059906\n",
      "-- processing region: Hoxd10-37:+ with 148 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-37:+.fasta\n",
      "--- *blasting region: Hoxd10-37:+\n",
      "--- total time for blast Hoxd10-37:+: 11.758004665374756\n",
      "-- processing region: Hoxd10-38:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-38:+.fasta\n",
      "--- *blasting region: Hoxd10-38:+\n",
      "--- total time for blast Hoxd10-38:+: 13.969995021820068\n",
      "-- processing region: Hoxd10-39:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-39:+.fasta\n",
      "--- *blasting region: Hoxd10-39:+\n",
      "--- total time for blast Hoxd10-39:+: 9.369002103805542\n",
      "-- processing region: Hoxd10-3:+ with 126 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-3:+.fasta\n",
      "--- *blasting region: Hoxd10-3:+\n",
      "--- total time for blast Hoxd10-3:+: 24.49614191055298\n",
      "-- processing region: Hoxd10-40:+ with 124 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-40:+.fasta\n",
      "--- *blasting region: Hoxd10-40:+\n",
      "--- total time for blast Hoxd10-40:+: 7.946000576019287\n",
      "-- processing region: Hoxd10-41:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-41:+.fasta\n",
      "--- *blasting region: Hoxd10-41:+\n",
      "--- total time for blast Hoxd10-41:+: 6.009000539779663\n",
      "-- processing region: Hoxd10-42:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-42:+.fasta\n",
      "--- *blasting region: Hoxd10-42:+\n",
      "--- total time for blast Hoxd10-42:+: 8.417001008987427\n",
      "-- processing region: Hoxd10-43:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-43:+.fasta\n",
      "--- *blasting region: Hoxd10-43:+\n",
      "--- total time for blast Hoxd10-43:+: 7.408000230789185\n",
      "-- processing region: Hoxd10-44:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-44:+.fasta\n",
      "--- *blasting region: Hoxd10-44:+\n",
      "--- total time for blast Hoxd10-44:+: 6.913002967834473\n",
      "-- processing region: Hoxd10-45:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-45:+.fasta\n",
      "--- *blasting region: Hoxd10-45:+\n",
      "--- total time for blast Hoxd10-45:+: 10.404997110366821\n",
      "-- processing region: Hoxd10-46:+ with 119 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-46:+.fasta\n",
      "--- *blasting region: Hoxd10-46:+\n",
      "--- total time for blast Hoxd10-46:+: 7.978999614715576\n",
      "-- processing region: Hoxd10-47:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-47:+.fasta\n",
      "--- *blasting region: Hoxd10-47:+\n",
      "--- total time for blast Hoxd10-47:+: 16.187825441360474\n",
      "-- processing region: Hoxd10-48:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-48:+.fasta\n",
      "--- *blasting region: Hoxd10-48:+\n",
      "--- total time for blast Hoxd10-48:+: 11.599001169204712\n",
      "-- processing region: Hoxd10-49:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-49:+.fasta\n",
      "--- *blasting region: Hoxd10-49:+\n",
      "--- total time for blast Hoxd10-49:+: 6.7539989948272705\n",
      "-- processing region: Hoxd10-4:+ with 118 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-4:+.fasta\n",
      "--- *blasting region: Hoxd10-4:+\n",
      "--- total time for blast Hoxd10-4:+: 14.496001720428467\n",
      "-- processing region: Hoxd10-50:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-50:+.fasta\n",
      "--- *blasting region: Hoxd10-50:+\n",
      "--- total time for blast Hoxd10-50:+: 9.369490623474121\n",
      "-- processing region: Hoxd10-51:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-51:+.fasta\n",
      "--- *blasting region: Hoxd10-51:+\n",
      "--- total time for blast Hoxd10-51:+: 7.245999813079834\n",
      "-- processing region: Hoxd10-52:+ with 143 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-52:+.fasta\n",
      "--- *blasting region: Hoxd10-52:+\n",
      "--- total time for blast Hoxd10-52:+: 10.113000869750977\n",
      "-- processing region: Hoxd10-53:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-53:+.fasta\n",
      "--- *blasting region: Hoxd10-53:+\n",
      "--- total time for blast Hoxd10-53:+: 11.876001596450806\n",
      "-- processing region: Hoxd10-54:+ with 134 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-54:+.fasta\n",
      "--- *blasting region: Hoxd10-54:+\n",
      "--- total time for blast Hoxd10-54:+: 14.371998310089111\n",
      "-- processing region: Hoxd10-55:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-55:+.fasta\n",
      "--- *blasting region: Hoxd10-55:+\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Hoxd10-55:+: 12.827172756195068\n",
      "-- processing region: Hoxd10-56:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-56:+.fasta\n",
      "--- *blasting region: Hoxd10-56:+\n",
      "--- total time for blast Hoxd10-56:+: 7.828001976013184\n",
      "-- processing region: Hoxd10-57:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-57:+.fasta\n",
      "--- *blasting region: Hoxd10-57:+\n",
      "--- total time for blast Hoxd10-57:+: 7.975000381469727\n",
      "-- processing region: Hoxd10-58:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-58:+.fasta\n",
      "--- *blasting region: Hoxd10-58:+\n",
      "--- total time for blast Hoxd10-58:+: 9.5749990940094\n",
      "-- processing region: Hoxd10-59:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-59:+.fasta\n",
      "--- *blasting region: Hoxd10-59:+\n",
      "--- total time for blast Hoxd10-59:+: 6.9629998207092285\n",
      "-- processing region: Hoxd10-5:+ with 127 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-5:+.fasta\n",
      "--- *blasting region: Hoxd10-5:+\n",
      "--- total time for blast Hoxd10-5:+: 10.36900019645691\n",
      "-- processing region: Hoxd10-60:+ with 126 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-60:+.fasta\n",
      "--- *blasting region: Hoxd10-60:+\n",
      "--- total time for blast Hoxd10-60:+: 5.065000534057617\n",
      "-- processing region: Hoxd10-61:+ with 144 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-61:+.fasta\n",
      "--- *blasting region: Hoxd10-61:+\n",
      "--- total time for blast Hoxd10-61:+: 7.085000991821289\n",
      "-- processing region: Hoxd10-62:+ with 134 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-62:+.fasta\n",
      "--- *blasting region: Hoxd10-62:+\n",
      "--- total time for blast Hoxd10-62:+: 10.84500002861023\n",
      "-- processing region: Hoxd10-63:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-63:+.fasta\n",
      "--- *blasting region: Hoxd10-63:+\n",
      "--- total time for blast Hoxd10-63:+: 10.203001022338867\n",
      "-- processing region: Hoxd10-64:+ with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-64:+.fasta\n",
      "--- *blasting region: Hoxd10-64:+\n",
      "--- total time for blast Hoxd10-64:+: 5.0166099071502686\n",
      "-- processing region: Hoxd10-65:+ with 133 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-65:+.fasta\n",
      "--- *blasting region: Hoxd10-65:+\n",
      "--- total time for blast Hoxd10-65:+: 10.214999914169312\n",
      "-- processing region: Hoxd10-66:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-66:+.fasta\n",
      "--- *blasting region: Hoxd10-66:+\n",
      "--- total time for blast Hoxd10-66:+: 36.00212049484253\n",
      "-- processing region: Hoxd10-67:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-67:+.fasta\n",
      "--- *blasting region: Hoxd10-67:+\n",
      "--- total time for blast Hoxd10-67:+: 6.592999696731567\n",
      "-- processing region: Hoxd10-68:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-68:+.fasta\n",
      "--- *blasting region: Hoxd10-68:+\n",
      "--- total time for blast Hoxd10-68:+: 9.400001525878906\n",
      "-- processing region: Hoxd10-69:+ with 141 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-69:+.fasta\n",
      "--- *blasting region: Hoxd10-69:+\n",
      "--- total time for blast Hoxd10-69:+: 9.647000074386597\n",
      "-- processing region: Hoxd10-6:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-6:+.fasta\n",
      "--- *blasting region: Hoxd10-6:+\n",
      "--- total time for blast Hoxd10-6:+: 11.26399850845337\n",
      "-- processing region: Hoxd10-70:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-70:+.fasta\n",
      "--- *blasting region: Hoxd10-70:+\n",
      "--- total time for blast Hoxd10-70:+: 11.950998783111572\n",
      "-- processing region: Hoxd10-71:+ with 143 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-71:+.fasta\n",
      "--- *blasting region: Hoxd10-71:+\n",
      "--- total time for blast Hoxd10-71:+: 15.429002046585083\n",
      "-- processing region: Hoxd10-72:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-72:+.fasta\n",
      "--- *blasting region: Hoxd10-72:+\n",
      "--- total time for blast Hoxd10-72:+: 25.105682849884033\n",
      "-- processing region: Hoxd10-73:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-73:+.fasta\n",
      "--- *blasting region: Hoxd10-73:+\n",
      "--- total time for blast Hoxd10-73:+: 8.519455432891846\n",
      "-- processing region: Hoxd10-74:+ with 145 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-74:+.fasta\n",
      "--- *blasting region: Hoxd10-74:+\n",
      "--- total time for blast Hoxd10-74:+: 38.61509895324707\n",
      "-- processing region: Hoxd10-75:+ with 117 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-75:+.fasta\n",
      "--- *blasting region: Hoxd10-75:+\n",
      "--- total time for blast Hoxd10-75:+: 10.384998559951782\n",
      "-- processing region: Hoxd10-76:+ with 119 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-76:+.fasta\n",
      "--- *blasting region: Hoxd10-76:+\n",
      "--- total time for blast Hoxd10-76:+: 21.49395513534546\n",
      "-- processing region: Hoxd10-77:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-77:+.fasta\n",
      "--- *blasting region: Hoxd10-77:+\n",
      "--- total time for blast Hoxd10-77:+: 7.334000825881958\n",
      "-- processing region: Hoxd10-78:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-78:+.fasta\n",
      "--- *blasting region: Hoxd10-78:+\n",
      "--- total time for blast Hoxd10-78:+: 37.590235233306885\n",
      "-- processing region: Hoxd10-79:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-79:+.fasta\n",
      "--- *blasting region: Hoxd10-79:+\n",
      "--- total time for blast Hoxd10-79:+: 9.718977928161621\n",
      "-- processing region: Hoxd10-7:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-7:+.fasta\n",
      "--- *blasting region: Hoxd10-7:+\n",
      "--- total time for blast Hoxd10-7:+: 5.982998609542847\n",
      "-- processing region: Hoxd10-80:+ with 143 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-80:+.fasta\n",
      "--- *blasting region: Hoxd10-80:+\n",
      "--- total time for blast Hoxd10-80:+: 10.088223457336426\n",
      "-- processing region: Hoxd10-81:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-81:+.fasta\n",
      "--- *blasting region: Hoxd10-81:+\n",
      "--- total time for blast Hoxd10-81:+: 9.000996828079224\n",
      "-- processing region: Hoxd10-82:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-82:+.fasta\n",
      "--- *blasting region: Hoxd10-82:+\n",
      "--- total time for blast Hoxd10-82:+: 8.288999557495117\n",
      "-- processing region: Hoxd10-83:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-83:+.fasta\n",
      "--- *blasting region: Hoxd10-83:+\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Hoxd10-83:+: 6.524001359939575\n",
      "-- processing region: Hoxd10-84:+ with 134 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-84:+.fasta\n",
      "--- *blasting region: Hoxd10-84:+\n",
      "--- total time for blast Hoxd10-84:+: 6.841999530792236\n",
      "-- processing region: Hoxd10-85:+ with 148 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-85:+.fasta\n",
      "--- *blasting region: Hoxd10-85:+\n",
      "--- total time for blast Hoxd10-85:+: 9.07400131225586\n",
      "-- processing region: Hoxd10-86:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-86:+.fasta\n",
      "--- *blasting region: Hoxd10-86:+\n",
      "--- total time for blast Hoxd10-86:+: 6.68600058555603\n",
      "-- processing region: Hoxd10-87:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-87:+.fasta\n",
      "--- *blasting region: Hoxd10-87:+\n",
      "--- total time for blast Hoxd10-87:+: 15.546999454498291\n",
      "-- processing region: Hoxd10-88:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-88:+.fasta\n",
      "--- *blasting region: Hoxd10-88:+\n",
      "--- total time for blast Hoxd10-88:+: 16.302000045776367\n",
      "-- processing region: Hoxd10-89:+ with 142 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-89:+.fasta\n",
      "--- *blasting region: Hoxd10-89:+\n",
      "--- total time for blast Hoxd10-89:+: 13.7330002784729\n",
      "-- processing region: Hoxd10-8:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-8:+.fasta\n",
      "--- *blasting region: Hoxd10-8:+\n",
      "--- total time for blast Hoxd10-8:+: 6.455999851226807\n",
      "-- processing region: Hoxd10-90:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-90:+.fasta\n",
      "--- *blasting region: Hoxd10-90:+\n",
      "--- total time for blast Hoxd10-90:+: 16.717998504638672\n",
      "-- processing region: Hoxd10-91:+ with 109 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-91:+.fasta\n",
      "--- *blasting region: Hoxd10-91:+\n",
      "--- total time for blast Hoxd10-91:+: 12.315400838851929\n",
      "-- processing region: Hoxd10-92:+ with 106 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-92:+.fasta\n",
      "--- *blasting region: Hoxd10-92:+\n",
      "--- total time for blast Hoxd10-92:+: 14.65414834022522\n",
      "-- processing region: Hoxd10-93:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-93:+.fasta\n",
      "--- *blasting region: Hoxd10-93:+\n",
      "--- total time for blast Hoxd10-93:+: 10.41890263557434\n",
      "-- processing region: Hoxd10-94:+ with 134 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-94:+.fasta\n",
      "--- *blasting region: Hoxd10-94:+\n",
      "--- total time for blast Hoxd10-94:+: 7.124999284744263\n",
      "-- processing region: Hoxd10-95:+ with 143 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-95:+.fasta\n",
      "--- *blasting region: Hoxd10-95:+\n",
      "--- total time for blast Hoxd10-95:+: 18.387000799179077\n",
      "-- processing region: Hoxd10-96:+ with 132 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-96:+.fasta\n",
      "--- *blasting region: Hoxd10-96:+\n",
      "--- total time for blast Hoxd10-96:+: 21.73972249031067\n",
      "-- processing region: Hoxd10-97:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-97:+.fasta\n",
      "--- *blasting region: Hoxd10-97:+\n",
      "--- total time for blast Hoxd10-97:+: 8.33899998664856\n",
      "-- processing region: Hoxd10-98:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-98:+.fasta\n",
      "--- *blasting region: Hoxd10-98:+\n",
      "--- total time for blast Hoxd10-98:+: 8.424000024795532\n",
      "-- processing region: Hoxd10-99:+ with 134 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-99:+.fasta\n",
      "--- *blasting region: Hoxd10-99:+\n",
      "--- total time for blast Hoxd10-99:+: 9.870001316070557\n",
      "-- processing region: Hoxd10-9:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Hoxd10-9:+.fasta\n",
      "--- *blasting region: Hoxd10-9:+\n",
      "--- total time for blast Hoxd10-9:+: 7.640998601913452\n",
      "-- processing region: Spp2-10:+ with 149 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-10:+.fasta\n",
      "--- *blasting region: Spp2-10:+\n",
      "--- total time for blast Spp2-10:+: 15.641000986099243\n",
      "-- processing region: Spp2-11:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-11:+.fasta\n",
      "--- *blasting region: Spp2-11:+\n",
      "--- total time for blast Spp2-11:+: 11.297000408172607\n",
      "-- processing region: Spp2-12:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-12:+.fasta\n",
      "--- *blasting region: Spp2-12:+\n",
      "--- total time for blast Spp2-12:+: 14.429997682571411\n",
      "-- processing region: Spp2-13:+ with 139 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-13:+.fasta\n",
      "--- *blasting region: Spp2-13:+\n",
      "--- total time for blast Spp2-13:+: 8.495000123977661\n",
      "-- processing region: Spp2-14:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-14:+.fasta\n",
      "--- *blasting region: Spp2-14:+\n",
      "--- total time for blast Spp2-14:+: 13.385777711868286\n",
      "-- processing region: Spp2-15:+ with 140 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-15:+.fasta\n",
      "--- *blasting region: Spp2-15:+\n",
      "--- total time for blast Spp2-15:+: 9.636999368667603\n",
      "-- processing region: Spp2-16:+ with 148 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-16:+.fasta\n",
      "--- *blasting region: Spp2-16:+\n",
      "--- total time for blast Spp2-16:+: 12.372999906539917\n",
      "-- processing region: Spp2-17:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-17:+.fasta\n",
      "--- *blasting region: Spp2-17:+\n",
      "--- total time for blast Spp2-17:+: 9.874000549316406\n",
      "-- processing region: Spp2-18:+ with 147 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-18:+.fasta\n",
      "--- *blasting region: Spp2-18:+\n",
      "--- total time for blast Spp2-18:+: 14.34799838066101\n",
      "-- processing region: Spp2-19:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-19:+.fasta\n",
      "--- *blasting region: Spp2-19:+\n",
      "--- total time for blast Spp2-19:+: 11.472001075744629\n",
      "-- processing region: Spp2-1:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-1:+.fasta\n",
      "--- *blasting region: Spp2-1:+\n",
      "--- total time for blast Spp2-1:+: 21.374000787734985\n",
      "-- processing region: Spp2-20:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-20:+.fasta\n",
      "--- *blasting region: Spp2-20:+\n",
      "--- total time for blast Spp2-20:+: 6.108999013900757\n",
      "-- processing region: Spp2-21:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-21:+.fasta\n",
      "--- *blasting region: Spp2-21:+\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Spp2-21:+: 16.80300235748291\n",
      "-- processing region: Spp2-22:+ with 127 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-22:+.fasta\n",
      "--- *blasting region: Spp2-22:+\n",
      "--- total time for blast Spp2-22:+: 12.573998928070068\n",
      "-- processing region: Spp2-23:+ with 135 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-23:+.fasta\n",
      "--- *blasting region: Spp2-23:+\n",
      "--- total time for blast Spp2-23:+: 8.64099907875061\n",
      "-- processing region: Spp2-24:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-24:+.fasta\n",
      "--- *blasting region: Spp2-24:+\n",
      "--- total time for blast Spp2-24:+: 9.747999429702759\n",
      "-- processing region: Spp2-25:+ with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-25:+.fasta\n",
      "--- *blasting region: Spp2-25:+\n",
      "--- total time for blast Spp2-25:+: 8.780000686645508\n",
      "-- processing region: Spp2-26:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-26:+.fasta\n",
      "--- *blasting region: Spp2-26:+\n",
      "--- total time for blast Spp2-26:+: 7.117000341415405\n",
      "-- processing region: Spp2-27:+ with 99 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-27:+.fasta\n",
      "--- *blasting region: Spp2-27:+\n",
      "--- total time for blast Spp2-27:+: 10.937002182006836\n",
      "-- processing region: Spp2-28:+ with 145 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-28:+.fasta\n",
      "--- *blasting region: Spp2-28:+\n",
      "--- total time for blast Spp2-28:+: 11.533997058868408\n",
      "-- processing region: Spp2-29:+ with 127 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-29:+.fasta\n",
      "--- *blasting region: Spp2-29:+\n",
      "--- total time for blast Spp2-29:+: 7.782000541687012\n",
      "-- processing region: Spp2-2:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-2:+.fasta\n",
      "--- *blasting region: Spp2-2:+\n",
      "--- total time for blast Spp2-2:+: 10.701001405715942\n",
      "-- processing region: Spp2-30:+ with 93 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-30:+.fasta\n",
      "--- *blasting region: Spp2-30:+\n",
      "--- total time for blast Spp2-30:+: 16.564269304275513\n",
      "-- processing region: Spp2-31:+ with 115 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-31:+.fasta\n",
      "--- *blasting region: Spp2-31:+\n",
      "--- total time for blast Spp2-31:+: 7.9489991664886475\n",
      "-- processing region: Spp2-32:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-32:+.fasta\n",
      "--- *blasting region: Spp2-32:+\n",
      "--- total time for blast Spp2-32:+: 18.900007009506226\n",
      "-- processing region: Spp2-33:+ with 139 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-33:+.fasta\n",
      "--- *blasting region: Spp2-33:+\n",
      "--- total time for blast Spp2-33:+: 9.802992343902588\n",
      "-- processing region: Spp2-34:+ with 144 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-34:+.fasta\n",
      "--- *blasting region: Spp2-34:+\n",
      "--- total time for blast Spp2-34:+: 11.364054441452026\n",
      "-- processing region: Spp2-35:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-35:+.fasta\n",
      "--- *blasting region: Spp2-35:+\n",
      "--- total time for blast Spp2-35:+: 10.142999649047852\n",
      "-- processing region: Spp2-36:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-36:+.fasta\n",
      "--- *blasting region: Spp2-36:+\n",
      "--- total time for blast Spp2-36:+: 8.17000150680542\n",
      "-- processing region: Spp2-37:+ with 134 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-37:+.fasta\n",
      "--- *blasting region: Spp2-37:+\n",
      "--- total time for blast Spp2-37:+: 22.593391180038452\n",
      "-- processing region: Spp2-38:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-38:+.fasta\n",
      "--- *blasting region: Spp2-38:+\n",
      "--- total time for blast Spp2-38:+: 6.799001216888428\n",
      "-- processing region: Spp2-39:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-39:+.fasta\n",
      "--- *blasting region: Spp2-39:+\n",
      "--- total time for blast Spp2-39:+: 13.62400197982788\n",
      "-- processing region: Spp2-3:+ with 139 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-3:+.fasta\n",
      "--- *blasting region: Spp2-3:+\n",
      "--- total time for blast Spp2-3:+: 6.166004657745361\n",
      "-- processing region: Spp2-40:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-40:+.fasta\n",
      "--- *blasting region: Spp2-40:+\n",
      "--- total time for blast Spp2-40:+: 7.1379945278167725\n",
      "-- processing region: Spp2-41:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-41:+.fasta\n",
      "--- *blasting region: Spp2-41:+\n",
      "--- total time for blast Spp2-41:+: 7.456001281738281\n",
      "-- processing region: Spp2-42:+ with 144 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-42:+.fasta\n",
      "--- *blasting region: Spp2-42:+\n",
      "--- total time for blast Spp2-42:+: 13.32099986076355\n",
      "-- processing region: Spp2-43:+ with 131 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-43:+.fasta\n",
      "--- *blasting region: Spp2-43:+\n",
      "--- total time for blast Spp2-43:+: 5.737999439239502\n",
      "-- processing region: Spp2-44:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-44:+.fasta\n",
      "--- *blasting region: Spp2-44:+\n",
      "--- total time for blast Spp2-44:+: 13.742001295089722\n",
      "-- processing region: Spp2-45:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-45:+.fasta\n",
      "--- *blasting region: Spp2-45:+\n",
      "--- total time for blast Spp2-45:+: 11.930005311965942\n",
      "-- processing region: Spp2-46:+ with 149 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-46:+.fasta\n",
      "--- *blasting region: Spp2-46:+\n",
      "--- total time for blast Spp2-46:+: 12.939993858337402\n",
      "-- processing region: Spp2-47:+ with 142 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-47:+.fasta\n",
      "--- *blasting region: Spp2-47:+\n",
      "--- total time for blast Spp2-47:+: 34.0561420917511\n",
      "-- processing region: Spp2-48:+ with 143 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-48:+.fasta\n",
      "--- *blasting region: Spp2-48:+\n",
      "--- total time for blast Spp2-48:+: 9.998998165130615\n",
      "-- processing region: Spp2-49:+ with 139 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-49:+.fasta\n",
      "--- *blasting region: Spp2-49:+\n",
      "--- total time for blast Spp2-49:+: 13.30500078201294\n",
      "-- processing region: Spp2-4:+ with 147 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-4:+.fasta\n",
      "--- *blasting region: Spp2-4:+\n",
      "--- total time for blast Spp2-4:+: 12.283000230789185\n",
      "-- processing region: Spp2-50:+ with 149 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-50:+.fasta\n",
      "--- *blasting region: Spp2-50:+\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Spp2-50:+: 16.80500102043152\n",
      "-- processing region: Spp2-51:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-51:+.fasta\n",
      "--- *blasting region: Spp2-51:+\n",
      "--- total time for blast Spp2-51:+: 7.694000005722046\n",
      "-- processing region: Spp2-52:+ with 144 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-52:+.fasta\n",
      "--- *blasting region: Spp2-52:+\n",
      "--- total time for blast Spp2-52:+: 13.240005016326904\n",
      "-- processing region: Spp2-53:+ with 53 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-53:+.fasta\n",
      "--- *blasting region: Spp2-53:+\n",
      "--- total time for blast Spp2-53:+: 4.402994871139526\n",
      "-- processing region: Spp2-54:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-54:+.fasta\n",
      "--- *blasting region: Spp2-54:+\n",
      "--- total time for blast Spp2-54:+: 13.919999599456787\n",
      "-- processing region: Spp2-55:+ with 87 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-55:+.fasta\n",
      "--- *blasting region: Spp2-55:+\n",
      "--- total time for blast Spp2-55:+: 118.47797656059265\n",
      "-- processing region: Spp2-56:+ with 126 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-56:+.fasta\n",
      "--- *blasting region: Spp2-56:+\n",
      "--- total time for blast Spp2-56:+: 8.037001132965088\n",
      "-- processing region: Spp2-57:+ with 149 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-57:+.fasta\n",
      "--- *blasting region: Spp2-57:+\n",
      "--- total time for blast Spp2-57:+: 8.189001083374023\n",
      "-- processing region: Spp2-58:+ with 146 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-58:+.fasta\n",
      "--- *blasting region: Spp2-58:+\n",
      "--- total time for blast Spp2-58:+: 16.67599892616272\n",
      "-- processing region: Spp2-59:+ with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-59:+.fasta\n",
      "--- *blasting region: Spp2-59:+\n",
      "--- total time for blast Spp2-59:+: 10.703001022338867\n",
      "-- processing region: Spp2-5:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-5:+.fasta\n",
      "--- *blasting region: Spp2-5:+\n",
      "--- total time for blast Spp2-5:+: 24.17099952697754\n",
      "-- processing region: Spp2-60:+ with 139 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-60:+.fasta\n",
      "--- *blasting region: Spp2-60:+\n",
      "--- total time for blast Spp2-60:+: 14.021000146865845\n",
      "-- processing region: Spp2-61:+ with 147 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-61:+.fasta\n",
      "--- *blasting region: Spp2-61:+\n",
      "--- total time for blast Spp2-61:+: 25.68300199508667\n",
      "-- processing region: Spp2-62:+ with 147 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-62:+.fasta\n",
      "--- *blasting region: Spp2-62:+\n",
      "--- total time for blast Spp2-62:+: 19.170997381210327\n",
      "-- processing region: Spp2-63:+ with 144 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-63:+.fasta\n",
      "--- *blasting region: Spp2-63:+\n",
      "--- total time for blast Spp2-63:+: 9.745004177093506\n",
      "-- processing region: Spp2-64:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-64:+.fasta\n",
      "--- *blasting region: Spp2-64:+\n",
      "--- total time for blast Spp2-64:+: 16.332000255584717\n",
      "-- processing region: Spp2-65:+ with 131 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-65:+.fasta\n",
      "--- *blasting region: Spp2-65:+\n",
      "--- total time for blast Spp2-65:+: 13.742000102996826\n",
      "-- processing region: Spp2-66:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-66:+.fasta\n",
      "--- *blasting region: Spp2-66:+\n",
      "--- total time for blast Spp2-66:+: 15.718001365661621\n",
      "-- processing region: Spp2-67:+ with 125 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-67:+.fasta\n",
      "--- *blasting region: Spp2-67:+\n",
      "--- total time for blast Spp2-67:+: 11.962001323699951\n",
      "-- processing region: Spp2-68:+ with 140 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-68:+.fasta\n",
      "--- *blasting region: Spp2-68:+\n",
      "--- total time for blast Spp2-68:+: 10.992001295089722\n",
      "-- processing region: Spp2-69:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-69:+.fasta\n",
      "--- *blasting region: Spp2-69:+\n",
      "--- total time for blast Spp2-69:+: 13.992001295089722\n",
      "-- processing region: Spp2-6:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-6:+.fasta\n",
      "--- *blasting region: Spp2-6:+\n",
      "--- total time for blast Spp2-6:+: 11.536998748779297\n",
      "-- processing region: Spp2-70:+ with 133 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-70:+.fasta\n",
      "--- *blasting region: Spp2-70:+\n",
      "--- total time for blast Spp2-70:+: 14.863001585006714\n",
      "-- processing region: Spp2-71:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-71:+.fasta\n",
      "--- *blasting region: Spp2-71:+\n",
      "--- total time for blast Spp2-71:+: 14.92499589920044\n",
      "-- processing region: Spp2-72:+ with 142 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-72:+.fasta\n",
      "--- *blasting region: Spp2-72:+\n",
      "--- total time for blast Spp2-72:+: 7.250002384185791\n",
      "-- processing region: Spp2-73:+ with 143 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-73:+.fasta\n",
      "--- *blasting region: Spp2-73:+\n",
      "--- total time for blast Spp2-73:+: 17.913000345230103\n",
      "-- processing region: Spp2-74:+ with 147 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-74:+.fasta\n",
      "--- *blasting region: Spp2-74:+\n",
      "--- total time for blast Spp2-74:+: 6.6870012283325195\n",
      "-- processing region: Spp2-75:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-75:+.fasta\n",
      "--- *blasting region: Spp2-75:+\n",
      "--- total time for blast Spp2-75:+: 9.608998775482178\n",
      "-- processing region: Spp2-76:+ with 132 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-76:+.fasta\n",
      "--- *blasting region: Spp2-76:+\n",
      "--- total time for blast Spp2-76:+: 16.00499987602234\n",
      "-- processing region: Spp2-77:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-77:+.fasta\n",
      "--- *blasting region: Spp2-77:+\n",
      "--- total time for blast Spp2-77:+: 9.160000085830688\n",
      "-- processing region: Spp2-78:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-78:+.fasta\n",
      "--- *blasting region: Spp2-78:+\n",
      "--- total time for blast Spp2-78:+: 4.674999952316284\n",
      "-- processing region: Spp2-79:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-79:+.fasta\n",
      "--- *blasting region: Spp2-79:+\n",
      "--- total time for blast Spp2-79:+: 30.376950979232788\n",
      "-- processing region: Spp2-7:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-7:+.fasta\n",
      "--- *blasting region: Spp2-7:+\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Spp2-7:+: 10.358999490737915\n",
      "-- processing region: Spp2-80:+ with 83 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-80:+.fasta\n",
      "--- *blasting region: Spp2-80:+\n",
      "--- total time for blast Spp2-80:+: 8.008002042770386\n",
      "-- processing region: Spp2-81:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-81:+.fasta\n",
      "--- *blasting region: Spp2-81:+\n",
      "--- total time for blast Spp2-81:+: 28.295573472976685\n",
      "-- processing region: Spp2-82:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-82:+.fasta\n",
      "--- *blasting region: Spp2-82:+\n",
      "--- total time for blast Spp2-82:+: 11.838996648788452\n",
      "-- processing region: Spp2-83:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-83:+.fasta\n",
      "--- *blasting region: Spp2-83:+\n",
      "--- total time for blast Spp2-83:+: 24.470999717712402\n",
      "-- processing region: Spp2-84:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-84:+.fasta\n",
      "--- *blasting region: Spp2-84:+\n",
      "--- total time for blast Spp2-84:+: 30.435144186019897\n",
      "-- processing region: Spp2-85:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-85:+.fasta\n",
      "--- *blasting region: Spp2-85:+\n",
      "--- total time for blast Spp2-85:+: 24.61200213432312\n",
      "-- processing region: Spp2-86:+ with 141 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-86:+.fasta\n",
      "--- *blasting region: Spp2-86:+\n",
      "--- total time for blast Spp2-86:+: 36.614049434661865\n",
      "-- processing region: Spp2-87:+ with 128 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-87:+.fasta\n",
      "--- *blasting region: Spp2-87:+\n",
      "--- total time for blast Spp2-87:+: 56.79060196876526\n",
      "-- processing region: Spp2-88:+ with 144 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-88:+.fasta\n",
      "--- *blasting region: Spp2-88:+\n",
      "--- total time for blast Spp2-88:+: 25.883540630340576\n",
      "-- processing region: Spp2-89:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-89:+.fasta\n",
      "--- *blasting region: Spp2-89:+\n",
      "--- total time for blast Spp2-89:+: 23.689005374908447\n",
      "-- processing region: Spp2-8:+ with 143 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-8:+.fasta\n",
      "--- *blasting region: Spp2-8:+\n",
      "--- total time for blast Spp2-8:+: 10.817994117736816\n",
      "-- processing region: Spp2-90:+ with 88 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-90:+.fasta\n",
      "--- *blasting region: Spp2-90:+\n",
      "--- total time for blast Spp2-90:+: 10.032999515533447\n",
      "-- processing region: Spp2-91:+ with 130 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-91:+.fasta\n",
      "--- *blasting region: Spp2-91:+\n",
      "--- total time for blast Spp2-91:+: 60.64404106140137\n",
      "-- processing region: Spp2-92:+ with 50 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-92:+.fasta\n",
      "--- *blasting region: Spp2-92:+\n",
      "--- total time for blast Spp2-92:+: 18.58689570426941\n",
      "-- processing region: Spp2-9:+ with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\blast\\probe_gene_Spp2-9:+.fasta\n",
      "--- *blasting region: Spp2-9:+\n",
      "--- total time for blast Spp2-9:+: 21.856810331344604\n"
     ]
    }
   ],
   "source": [
    "blast_database = os.path.join(reference_folder, 'blast_database', 'GRCm38_ensembl')\n",
    "\n",
    "check.Blast_probes(kept_records, library_folder, num_threads=32, \n",
    "                   blast_db=blast_database, blast_evalue=100, overwrite=False)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "operating-meter",
   "metadata": {},
   "source": [
    "### 4.2.1 screen probe by blast results\n",
    "\n",
    "This step has 2 different cutoffs:\n",
    "\n",
    "hard_cutoff: whether a probe has any ~40mer hit somewhere else, if so just dump the probe\n",
    "\n",
    "soft_cutoff: whether a probe has a lot of ~20mer off-target hits, if exceed certain amount then dump the probe"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 183,
   "id": "minimal-thursday",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Number of probes loaded: 36608\n",
      "- Number of regions in this library: 264\n",
      "-- checking probes in region: Camk1-1:-\n",
      "--- gene=Camk1-1:-, id=135 removed by soft count = 104\n",
      "--- number of probes: 149 , kept by blast: 148 , if remove dups: 148\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-10:-\n",
      "--- gene=Camk1-10:-, id=45 removed by soft count = 147\n",
      "--- gene=Camk1-10:-, id=99 removed by soft count = 175\n",
      "--- gene=Camk1-10:-, id=108 removed by soft count = 149\n",
      "--- number of probes: 140 , kept by blast: 137 , if remove dups: 137\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-11:-\n",
      "--- gene=Camk1-11:-, id=92 removed by soft count = 3137\n",
      "--- number of probes: 150 , kept by blast: 149 , if remove dups: 149\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-12:-\n",
      "--- gene=Camk1-12:-, id=17 removed by soft count = 57\n",
      "--- gene=Camk1-12:-, id=18 removed by soft count = 376\n",
      "--- number of probes: 109 , kept by blast: 107 , if remove dups: 107\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-13:-\n",
      "--- gene=Camk1-13:-, id=79 removed by soft count = 167\n",
      "--- gene=Camk1-13:-, id=88 removed by soft count = 53\n",
      "--- gene=Camk1-13:-, id=89 removed by soft count = 817\n",
      "--- gene=Camk1-13:-, id=90 removed by soft count = 138\n",
      "--- gene=Camk1-13:-, id=94 removed by soft count = 429\n",
      "--- gene=Camk1-13:-, id=95 removed by soft count = 174\n",
      "--- gene=Camk1-13:-, id=100 removed by soft count = 122\n",
      "--- number of probes: 110 , kept by blast: 103 , if remove dups: 103\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-14:-\n",
      "--- gene=Camk1-14:-, id=62 removed by soft count = 225\n",
      "--- number of probes: 150 , kept by blast: 149 , if remove dups: 148\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-15:-\n",
      "--- number of probes: 150 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-16:-\n",
      "--- gene=Camk1-16:-, id=81 removed by soft count = 58\n",
      "--- gene=Camk1-16:-, id=141 removed by soft count = 65\n",
      "--- gene=Camk1-16:-, id=146 removed by soft count = 515\n",
      "--- number of probes: 150 , kept by blast: 147 , if remove dups: 147\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-17:-\n",
      "--- gene=Camk1-17:-, id=58 removed by soft count = 101\n",
      "--- gene=Camk1-17:-, id=113 removed by soft count = 57\n",
      "--- number of probes: 150 , kept by blast: 148 , if remove dups: 148\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-18:-\n",
      "--- gene=Camk1-18:-, id=31 removed by soft count = 83\n",
      "--- gene=Camk1-18:-, id=49 removed by soft count = 236\n",
      "--- gene=Camk1-18:-, id=58 removed by soft count = 73\n",
      "--- gene=Camk1-18:-, id=109 removed by soft count = 60\n",
      "--- number of probes: 150 , kept by blast: 146 , if remove dups: 146\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-19:-\n",
      "--- gene=Camk1-19:-, id=11 removed by soft count = 147\n",
      "--- gene=Camk1-19:-, id=12 removed by soft count = 65\n",
      "--- gene=Camk1-19:-, id=13 removed by soft count = 203\n",
      "--- gene=Camk1-19:-, id=74 removed by soft count = 61\n",
      "--- number of probes: 148 , kept by blast: 144 , if remove dups: 144\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-2:-\n",
      "--- number of probes: 140 , kept by blast: 140 , if remove dups: 139\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-20:-\n",
      "--- gene=Camk1-20:-, id=74 removed by soft count = 84\n",
      "--- number of probes: 143 , kept by blast: 142 , if remove dups: 142\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-21:-\n",
      "--- gene=Camk1-21:-, id=39 removed by soft count = 971\n",
      "--- number of probes: 136 , kept by blast: 135 , if remove dups: 135\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-22:-\n",
      "--- gene=Camk1-22:-, id=0 removed by soft count = 151\n",
      "--- gene=Camk1-22:-, id=2 removed by soft count = 489\n",
      "--- gene=Camk1-22:-, id=8 removed by soft count = 236\n",
      "--- gene=Camk1-22:-, id=9 removed by soft count = 283\n",
      "--- gene=Camk1-22:-, id=78 removed by soft count = 93\n",
      "--- gene=Camk1-22:-, id=121 removed by soft count = 67\n",
      "--- gene=Camk1-22:-, id=123 removed by soft count = 64\n",
      "--- gene=Camk1-22:-, id=127 removed by soft count = 62\n",
      "--- gene=Camk1-22:-, id=129 removed by soft count = 103\n",
      "--- gene=Camk1-22:-, id=137 removed by soft count = 55\n",
      "--- number of probes: 141 , kept by blast: 131 , if remove dups: 131\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-23:-\n",
      "--- gene=Camk1-23:-, id=42 removed by soft count = 124\n",
      "--- gene=Camk1-23:-, id=112 removed by soft count = 144\n",
      "--- gene=Camk1-23:-, id=113 removed by soft count = 414\n",
      "--- gene=Camk1-23:-, id=132 removed by soft count = 76\n",
      "--- gene=Camk1-23:-, id=133 removed by soft count = 149\n",
      "--- gene=Camk1-23:-, id=134 removed by soft count = 61\n",
      "--- gene=Camk1-23:-, id=135 removed by soft count = 82\n",
      "--- number of probes: 136 , kept by blast: 129 , if remove dups: 129\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-24:-\n",
      "--- gene=Camk1-24:-, id=84 removed by soft count = 82\n",
      "--- number of probes: 150 , kept by blast: 149 , if remove dups: 149\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-25:-\n",
      "--- gene=Camk1-25:-, id=62 removed by soft count = 83\n",
      "--- number of probes: 150 , kept by blast: 149 , if remove dups: 148\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-26:-\n",
      "--- gene=Camk1-26:-, id=52 removed by soft count = 86\n",
      "--- number of probes: 141 , kept by blast: 140 , if remove dups: 140\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-27:-\n",
      "--- gene=Camk1-27:-, id=6 removed by soft count = 372\n",
      "--- gene=Camk1-27:-, id=68 removed by soft count = 53\n",
      "--- number of probes: 147 , kept by blast: 145 , if remove dups: 145\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-28:-\n",
      "--- gene=Camk1-28:-, id=4 removed by soft count = 63\n",
      "--- gene=Camk1-28:-, id=19 removed by soft count = 95\n",
      "--- gene=Camk1-28:-, id=27 removed by soft count = 54\n",
      "--- gene=Camk1-28:-, id=40 removed by soft count = 53\n",
      "--- gene=Camk1-28:-, id=41 removed by soft count = 64\n",
      "--- gene=Camk1-28:-, id=107 removed by soft count = 145\n",
      "--- number of probes: 113 , kept by blast: 107 , if remove dups: 107\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-29:-\n",
      "--- gene=Camk1-29:-, id=2 removed by soft count = 1347\n",
      "--- gene=Camk1-29:-, id=42 removed by soft count = 177\n",
      "--- gene=Camk1-29:-, id=49 removed by soft count = 78\n",
      "--- gene=Camk1-29:-, id=143 removed by soft count = 123\n",
      "--- number of probes: 150 , kept by blast: 146 , if remove dups: 146\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-3:-\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 150 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-30:-\n",
      "--- gene=Camk1-30:-, id=19 removed by soft count = 113\n",
      "--- gene=Camk1-30:-, id=20 removed by soft count = 156\n",
      "--- gene=Camk1-30:-, id=89 removed by soft count = 149\n",
      "--- gene=Camk1-30:-, id=98 removed by soft count = 190\n",
      "--- gene=Camk1-30:-, id=106 removed by soft count = 147\n",
      "--- gene=Camk1-30:-, id=108 removed by soft count = 80\n",
      "--- gene=Camk1-30:-, id=109 removed by soft count = 179\n",
      "--- gene=Camk1-30:-, id=110 removed by soft count = 353\n",
      "--- number of probes: 138 , kept by blast: 130 , if remove dups: 130\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-31:-\n",
      "--- gene=Camk1-31:-, id=19 removed by soft count = 83\n",
      "--- number of probes: 103 , kept by blast: 102 , if remove dups: 102\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-32:-\n",
      "--- gene=Camk1-32:-, id=22 removed by soft count = 132\n",
      "--- gene=Camk1-32:-, id=42 removed by soft count = 376\n",
      "--- gene=Camk1-32:-, id=131 removed by soft count = 104\n",
      "--- number of probes: 132 , kept by blast: 129 , if remove dups: 129\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-33:-\n",
      "--- gene=Camk1-33:-, id=98 removed by soft count = 182\n",
      "--- number of probes: 135 , kept by blast: 134 , if remove dups: 134\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-34:-\n",
      "--- gene=Camk1-34:-, id=32 removed by soft count = 343\n",
      "--- gene=Camk1-34:-, id=57 removed by soft count = 73\n",
      "--- gene=Camk1-34:-, id=114 removed by soft count = 53\n",
      "--- number of probes: 126 , kept by blast: 123 , if remove dups: 123\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-35:-\n",
      "--- gene=Camk1-35:-, id=58 removed by soft count = 234\n",
      "--- gene=Camk1-35:-, id=61 removed by soft count = 172\n",
      "--- number of probes: 131 , kept by blast: 129 , if remove dups: 129\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-36:-\n",
      "--- gene=Camk1-36:-, id=0 removed by soft count = 100\n",
      "--- gene=Camk1-36:-, id=1 removed by soft count = 77\n",
      "--- gene=Camk1-36:-, id=56 removed by soft count = 614\n",
      "--- number of probes: 133 , kept by blast: 130 , if remove dups: 130\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-37:-\n",
      "--- gene=Camk1-37:-, id=9 removed by soft count = 581\n",
      "--- number of probes: 131 , kept by blast: 130 , if remove dups: 130\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-38:-\n",
      "--- gene=Camk1-38:-, id=1 removed by soft count = 85\n",
      "--- gene=Camk1-38:-, id=38 removed by soft count = 61\n",
      "--- gene=Camk1-38:-, id=84 removed by soft count = 74\n",
      "--- gene=Camk1-38:-, id=105 removed by soft count = 80\n",
      "--- gene=Camk1-38:-, id=140 removed by soft count = 77\n",
      "--- number of probes: 142 , kept by blast: 137 , if remove dups: 137\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-39:-\n",
      "--- gene=Camk1-39:-, id=3 removed by soft count = 465\n",
      "--- gene=Camk1-39:-, id=5 removed by soft count = 100\n",
      "--- gene=Camk1-39:-, id=24 removed by soft count = 306\n",
      "--- gene=Camk1-39:-, id=25 removed by soft count = 78\n",
      "--- gene=Camk1-39:-, id=35 removed by soft count = 56\n",
      "--- gene=Camk1-39:-, id=78 removed by soft count = 1334\n",
      "--- gene=Camk1-39:-, id=81 removed by soft count = 306\n",
      "--- gene=Camk1-39:-, id=82 removed by soft count = 135\n",
      "--- number of probes: 142 , kept by blast: 134 , if remove dups: 133\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-4:-\n",
      "--- number of probes: 150 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-40:-\n",
      "--- gene=Camk1-40:-, id=8 removed by soft count = 77\n",
      "--- gene=Camk1-40:-, id=18 removed by soft count = 282\n",
      "--- gene=Camk1-40:-, id=19 removed by soft count = 125\n",
      "--- gene=Camk1-40:-, id=29 removed by soft count = 163\n",
      "--- number of probes: 136 , kept by blast: 132 , if remove dups: 132\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-41:-\n",
      "--- gene=Camk1-41:-, id=64 removed by soft count = 185\n",
      "--- gene=Camk1-41:-, id=65 removed by soft count = 588\n",
      "--- gene=Camk1-41:-, id=88 removed by soft count = 658\n",
      "--- gene=Camk1-41:-, id=97 removed by soft count = 51\n",
      "--- number of probes: 121 , kept by blast: 117 , if remove dups: 117\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-42:-\n",
      "--- number of probes: 139 , kept by blast: 139 , if remove dups: 139\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-43:-\n",
      "--- gene=Camk1-43:-, id=49 removed by soft count = 121\n",
      "--- gene=Camk1-43:-, id=98 removed by soft count = 897\n",
      "--- gene=Camk1-43:-, id=99 removed by soft count = 232\n",
      "--- number of probes: 132 , kept by blast: 129 , if remove dups: 129\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-44:-\n",
      "--- gene=Camk1-44:-, id=24 removed by soft count = 180\n",
      "--- gene=Camk1-44:-, id=32 removed by soft count = 102\n",
      "--- gene=Camk1-44:-, id=79 removed by soft count = 65\n",
      "--- gene=Camk1-44:-, id=99 removed by soft count = 56\n",
      "--- gene=Camk1-44:-, id=100 removed by soft count = 80\n",
      "--- gene=Camk1-44:-, id=103 removed by soft count = 160\n",
      "--- gene=Camk1-44:-, id=118 removed by soft count = 80\n",
      "--- number of probes: 133 , kept by blast: 126 , if remove dups: 126\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-45:-\n",
      "--- gene=Camk1-45:-, id=21 removed by soft count = 58\n",
      "--- gene=Camk1-45:-, id=84 removed by soft count = 56\n",
      "--- gene=Camk1-45:-, id=101 removed by soft count = 72\n",
      "--- gene=Camk1-45:-, id=104 removed by soft count = 873\n",
      "--- number of probes: 144 , kept by blast: 140 , if remove dups: 140\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-46:-\n",
      "--- gene=Camk1-46:-, id=81 removed by soft count = 185\n",
      "--- gene=Camk1-46:-, id=135 removed by soft count = 179\n",
      "--- number of probes: 139 , kept by blast: 137 , if remove dups: 135\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-47:-\n",
      "--- number of probes: 141 , kept by blast: 141 , if remove dups: 139\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-48:-\n",
      "--- gene=Camk1-48:-, id=20 removed by soft count = 66\n",
      "--- gene=Camk1-48:-, id=76 removed by soft count = 51\n",
      "--- gene=Camk1-48:-, id=77 removed by soft count = 526\n",
      "--- gene=Camk1-48:-, id=110 removed by soft count = 109\n",
      "--- number of probes: 145 , kept by blast: 141 , if remove dups: 141\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-49:-\n",
      "--- gene=Camk1-49:-, id=29 removed by soft count = 76\n",
      "--- number of probes: 150 , kept by blast: 149 , if remove dups: 149\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-5:-\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 150 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-50:-\n",
      "--- gene=Camk1-50:-, id=117 removed by soft count = 240\n",
      "--- number of probes: 149 , kept by blast: 148 , if remove dups: 148\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-51:-\n",
      "--- gene=Camk1-51:-, id=11 removed by soft count = 80\n",
      "--- gene=Camk1-51:-, id=12 removed by soft count = 52\n",
      "--- gene=Camk1-51:-, id=13 removed by soft count = 88\n",
      "--- gene=Camk1-51:-, id=41 removed by soft count = 789\n",
      "--- gene=Camk1-51:-, id=42 removed by soft count = 106\n",
      "--- gene=Camk1-51:-, id=66 removed by soft count = 427\n",
      "--- number of probes: 150 , kept by blast: 144 , if remove dups: 144\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-52:-\n",
      "--- gene=Camk1-52:-, id=143 removed by soft count = 99\n",
      "--- number of probes: 150 , kept by blast: 149 , if remove dups: 149\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-53:-\n",
      "--- number of probes: 150 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-54:-\n",
      "--- gene=Camk1-54:-, id=57 removed by hard count = 3\n",
      "--- number of probes: 150 , kept by blast: 149 , if remove dups: 148\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-6:-\n",
      "--- number of probes: 148 , kept by blast: 148 , if remove dups: 148\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-7:-\n",
      "--- number of probes: 139 , kept by blast: 139 , if remove dups: 139\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-8:-\n",
      "--- number of probes: 150 , kept by blast: 150 , if remove dups: 147\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Camk1-9:-\n",
      "--- gene=Camk1-9:-, id=115 removed by soft count = 82\n",
      "--- number of probes: 150 , kept by blast: 149 , if remove dups: 149\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-1:+\n",
      "--- gene=Hoxd10-1:+, id=84 removed by soft count = 460\n",
      "--- number of probes: 121 , kept by blast: 120 , if remove dups: 120\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-10:+\n",
      "--- number of probes: 150 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-100:+\n",
      "--- gene=Hoxd10-100:+, id=72 removed by soft count = 78\n",
      "--- number of probes: 135 , kept by blast: 134 , if remove dups: 134\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-101:+\n",
      "--- number of probes: 138 , kept by blast: 138 , if remove dups: 138\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-102:+\n",
      "--- gene=Hoxd10-102:+, id=140 removed by soft count = 194\n",
      "--- number of probes: 144 , kept by blast: 143 , if remove dups: 143\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-103:+\n",
      "--- gene=Hoxd10-103:+, id=5 removed by soft count = 366\n",
      "--- gene=Hoxd10-103:+, id=6 removed by soft count = 168\n",
      "--- gene=Hoxd10-103:+, id=33 removed by soft count = 62\n",
      "--- gene=Hoxd10-103:+, id=46 removed by soft count = 114\n",
      "--- gene=Hoxd10-103:+, id=87 removed by soft count = 191\n",
      "--- gene=Hoxd10-103:+, id=88 removed by soft count = 51\n",
      "--- number of probes: 118 , kept by blast: 112 , if remove dups: 112\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-104:+\n",
      "--- gene=Hoxd10-104:+, id=0 removed by soft count = 254\n",
      "--- gene=Hoxd10-104:+, id=11 removed by soft count = 57\n",
      "--- gene=Hoxd10-104:+, id=22 removed by soft count = 368\n",
      "--- gene=Hoxd10-104:+, id=25 removed by soft count = 140\n",
      "--- number of probes: 118 , kept by blast: 114 , if remove dups: 114\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-105:+\n",
      "--- gene=Hoxd10-105:+, id=26 removed by soft count = 63\n",
      "--- gene=Hoxd10-105:+, id=39 removed by soft count = 64\n",
      "--- gene=Hoxd10-105:+, id=40 removed by soft count = 77\n",
      "--- gene=Hoxd10-105:+, id=52 removed by hard count = 3\n",
      "--- gene=Hoxd10-105:+, id=55 removed by soft count = 54\n",
      "--- number of probes: 73 , kept by blast: 68 , if remove dups: 68\n",
      "-- number of probes kept for this region: 68\n",
      "-- checking probes in region: Hoxd10-106:+\n",
      "--- gene=Hoxd10-106:+, id=27 removed by soft count = 88\n",
      "--- gene=Hoxd10-106:+, id=31 removed by soft count = 130\n",
      "--- gene=Hoxd10-106:+, id=34 removed by soft count = 63\n",
      "--- gene=Hoxd10-106:+, id=52 removed by soft count = 81\n",
      "--- gene=Hoxd10-106:+, id=85 removed by soft count = 113\n",
      "--- gene=Hoxd10-106:+, id=87 removed by soft count = 60\n",
      "--- number of probes: 108 , kept by blast: 102 , if remove dups: 102\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-107:+\n",
      "--- gene=Hoxd10-107:+, id=10 removed by soft count = 55\n",
      "--- gene=Hoxd10-107:+, id=54 removed by soft count = 120\n",
      "--- gene=Hoxd10-107:+, id=55 removed by soft count = 74\n",
      "--- gene=Hoxd10-107:+, id=62 removed by soft count = 75\n",
      "--- gene=Hoxd10-107:+, id=71 removed by soft count = 54\n",
      "--- gene=Hoxd10-107:+, id=74 removed by soft count = 137\n",
      "--- gene=Hoxd10-107:+, id=75 removed by soft count = 85\n",
      "--- number of probes: 126 , kept by blast: 119 , if remove dups: 119\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-108:+\n",
      "--- gene=Hoxd10-108:+, id=0 removed by soft count = 1299\n",
      "--- gene=Hoxd10-108:+, id=20 removed by soft count = 3802\n",
      "--- gene=Hoxd10-108:+, id=21 removed by soft count = 901\n",
      "--- gene=Hoxd10-108:+, id=105 removed by soft count = 1337\n",
      "--- number of probes: 107 , kept by blast: 103 , if remove dups: 74\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-109:+\n",
      "--- gene=Hoxd10-109:+, id=27 removed by soft count = 105\n",
      "--- gene=Hoxd10-109:+, id=104 removed by soft count = 146\n",
      "--- gene=Hoxd10-109:+, id=106 removed by soft count = 379\n",
      "--- number of probes: 132 , kept by blast: 129 , if remove dups: 129\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-11:+\n",
      "--- number of probes: 150 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-110:+\n",
      "--- gene=Hoxd10-110:+, id=91 removed by soft count = 131\n",
      "--- number of probes: 142 , kept by blast: 141 , if remove dups: 141\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-111:+\n",
      "--- number of probes: 150 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-112:+\n",
      "--- gene=Hoxd10-112:+, id=62 removed by soft count = 266\n",
      "--- gene=Hoxd10-112:+, id=63 removed by soft count = 335\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 150 , kept by blast: 148 , if remove dups: 148\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-113:+\n",
      "--- gene=Hoxd10-113:+, id=11 removed by soft count = 284\n",
      "--- gene=Hoxd10-113:+, id=14 removed by soft count = 142\n",
      "--- gene=Hoxd10-113:+, id=27 removed by soft count = 64\n",
      "--- gene=Hoxd10-113:+, id=41 removed by soft count = 1212\n",
      "--- number of probes: 129 , kept by blast: 125 , if remove dups: 125\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-114:+\n",
      "--- gene=Hoxd10-114:+, id=46 removed by soft count = 98\n",
      "--- number of probes: 129 , kept by blast: 128 , if remove dups: 128\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-115:+\n",
      "--- gene=Hoxd10-115:+, id=85 removed by soft count = 101\n",
      "--- gene=Hoxd10-115:+, id=89 removed by soft count = 52\n",
      "--- gene=Hoxd10-115:+, id=130 removed by soft count = 64\n",
      "--- number of probes: 141 , kept by blast: 138 , if remove dups: 138\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-116:+\n",
      "--- number of probes: 108 , kept by blast: 108 , if remove dups: 108\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-117:+\n",
      "--- gene=Hoxd10-117:+, id=28 removed by soft count = 623\n",
      "--- gene=Hoxd10-117:+, id=111 removed by soft count = 91\n",
      "--- number of probes: 150 , kept by blast: 148 , if remove dups: 148\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-118:+\n",
      "--- gene=Hoxd10-118:+, id=10 removed by soft count = 157\n",
      "--- number of probes: 150 , kept by blast: 149 , if remove dups: 149\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-12:+\n",
      "--- number of probes: 147 , kept by blast: 147 , if remove dups: 147\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-13:+\n",
      "--- gene=Hoxd10-13:+, id=75 removed by soft count = 130\n",
      "--- gene=Hoxd10-13:+, id=82 removed by soft count = 59\n",
      "--- number of probes: 150 , kept by blast: 148 , if remove dups: 148\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-14:+\n",
      "--- number of probes: 150 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-15:+\n",
      "--- number of probes: 150 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-16:+\n",
      "--- number of probes: 150 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-17:+\n",
      "--- gene=Hoxd10-17:+, id=69 removed by soft count = 165\n",
      "--- number of probes: 150 , kept by blast: 149 , if remove dups: 149\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-18:+\n",
      "--- number of probes: 144 , kept by blast: 144 , if remove dups: 144\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-19:+\n",
      "--- gene=Hoxd10-19:+, id=9 removed by soft count = 53\n",
      "--- gene=Hoxd10-19:+, id=15 removed by soft count = 103\n",
      "--- gene=Hoxd10-19:+, id=17 removed by soft count = 59\n",
      "--- gene=Hoxd10-19:+, id=52 removed by soft count = 202\n",
      "--- gene=Hoxd10-19:+, id=98 removed by soft count = 631\n",
      "--- number of probes: 128 , kept by blast: 123 , if remove dups: 123\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-2:+\n",
      "--- gene=Hoxd10-2:+, id=66 removed by soft count = 80\n",
      "--- gene=Hoxd10-2:+, id=69 removed by soft count = 135\n",
      "--- gene=Hoxd10-2:+, id=71 removed by soft count = 331\n",
      "--- gene=Hoxd10-2:+, id=74 removed by soft count = 306\n",
      "--- gene=Hoxd10-2:+, id=78 removed by soft count = 627\n",
      "--- gene=Hoxd10-2:+, id=79 removed by soft count = 214\n",
      "--- gene=Hoxd10-2:+, id=81 removed by soft count = 112\n",
      "--- gene=Hoxd10-2:+, id=82 removed by soft count = 55\n",
      "--- gene=Hoxd10-2:+, id=83 removed by soft count = 425\n",
      "--- number of probes: 84 , kept by blast: 75 , if remove dups: 75\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 75\n",
      "-- checking probes in region: Hoxd10-20:+\n",
      "--- gene=Hoxd10-20:+, id=3 removed by soft count = 132\n",
      "--- gene=Hoxd10-20:+, id=9 removed by soft count = 79\n",
      "--- gene=Hoxd10-20:+, id=10 removed by soft count = 69\n",
      "--- gene=Hoxd10-20:+, id=120 removed by soft count = 93\n",
      "--- gene=Hoxd10-20:+, id=121 removed by soft count = 338\n",
      "--- gene=Hoxd10-20:+, id=122 removed by soft count = 106\n",
      "--- number of probes: 127 , kept by blast: 121 , if remove dups: 121\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-21:+\n",
      "--- gene=Hoxd10-21:+, id=3 removed by soft count = 62\n",
      "--- gene=Hoxd10-21:+, id=68 removed by soft count = 53\n",
      "--- gene=Hoxd10-21:+, id=69 removed by soft count = 53\n",
      "--- gene=Hoxd10-21:+, id=72 removed by soft count = 643\n",
      "--- number of probes: 116 , kept by blast: 112 , if remove dups: 111\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-22:+\n",
      "--- gene=Hoxd10-22:+, id=10 removed by soft count = 98\n",
      "--- gene=Hoxd10-22:+, id=46 removed by soft count = 68\n",
      "--- number of probes: 120 , kept by blast: 118 , if remove dups: 118\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-23:+\n",
      "--- gene=Hoxd10-23:+, id=24 removed by soft count = 66\n",
      "--- gene=Hoxd10-23:+, id=27 removed by soft count = 277\n",
      "--- number of probes: 119 , kept by blast: 117 , if remove dups: 117\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-24:+\n",
      "--- number of probes: 123 , kept by blast: 123 , if remove dups: 123\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-25:+\n",
      "--- gene=Hoxd10-25:+, id=10 removed by soft count = 230\n",
      "--- number of probes: 118 , kept by blast: 117 , if remove dups: 117\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-26:+\n",
      "--- gene=Hoxd10-26:+, id=9 removed by soft count = 151\n",
      "--- gene=Hoxd10-26:+, id=16 removed by soft count = 73\n",
      "--- gene=Hoxd10-26:+, id=36 removed by soft count = 252\n",
      "--- number of probes: 118 , kept by blast: 115 , if remove dups: 115\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-27:+\n",
      "--- gene=Hoxd10-27:+, id=0 removed by soft count = 566\n",
      "--- gene=Hoxd10-27:+, id=97 removed by soft count = 449\n",
      "--- gene=Hoxd10-27:+, id=99 removed by soft count = 63\n",
      "--- gene=Hoxd10-27:+, id=101 removed by soft count = 126\n",
      "--- gene=Hoxd10-27:+, id=102 removed by soft count = 790\n",
      "--- gene=Hoxd10-27:+, id=103 removed by soft count = 13827\n",
      "--- gene=Hoxd10-27:+, id=104 removed by soft count = 56\n",
      "--- number of probes: 105 , kept by blast: 98 , if remove dups: 98\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 98\n",
      "-- checking probes in region: Hoxd10-28:+\n",
      "--- gene=Hoxd10-28:+, id=0 removed by soft count = 177\n",
      "--- gene=Hoxd10-28:+, id=17 removed by soft count = 742\n",
      "--- gene=Hoxd10-28:+, id=31 removed by soft count = 57\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 109 , kept by blast: 106 , if remove dups: 106\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-29:+\n",
      "--- gene=Hoxd10-29:+, id=33 removed by soft count = 378\n",
      "--- gene=Hoxd10-29:+, id=74 removed by soft count = 125\n",
      "--- number of probes: 125 , kept by blast: 123 , if remove dups: 123\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-3:+\n",
      "--- gene=Hoxd10-3:+, id=0 removed by soft count = 241\n",
      "--- gene=Hoxd10-3:+, id=13 removed by soft count = 93\n",
      "--- gene=Hoxd10-3:+, id=14 removed by soft count = 9654\n",
      "--- gene=Hoxd10-3:+, id=56 removed by soft count = 60\n",
      "--- number of probes: 126 , kept by blast: 122 , if remove dups: 122\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-30:+\n",
      "--- number of probes: 150 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-31:+\n",
      "--- gene=Hoxd10-31:+, id=8 removed by soft count = 58\n",
      "--- gene=Hoxd10-31:+, id=85 removed by soft count = 82\n",
      "--- number of probes: 148 , kept by blast: 146 , if remove dups: 146\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-32:+\n",
      "--- gene=Hoxd10-32:+, id=12 removed by soft count = 85\n",
      "--- gene=Hoxd10-32:+, id=88 removed by soft count = 242\n",
      "--- gene=Hoxd10-32:+, id=93 removed by soft count = 223\n",
      "--- number of probes: 119 , kept by blast: 116 , if remove dups: 116\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-33:+\n",
      "--- number of probes: 130 , kept by blast: 130 , if remove dups: 130\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-34:+\n",
      "--- gene=Hoxd10-34:+, id=7 removed by soft count = 1049\n",
      "--- number of probes: 150 , kept by blast: 149 , if remove dups: 149\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-35:+\n",
      "--- gene=Hoxd10-35:+, id=98 removed by soft count = 148\n",
      "--- number of probes: 149 , kept by blast: 148 , if remove dups: 148\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-36:+\n",
      "--- gene=Hoxd10-36:+, id=37 removed by soft count = 83\n",
      "--- gene=Hoxd10-36:+, id=39 removed by soft count = 84\n",
      "--- number of probes: 146 , kept by blast: 144 , if remove dups: 144\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-37:+\n",
      "--- gene=Hoxd10-37:+, id=14 removed by soft count = 55\n",
      "--- gene=Hoxd10-37:+, id=54 removed by hard count = 6\n",
      "--- number of probes: 148 , kept by blast: 146 , if remove dups: 146\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-38:+\n",
      "--- gene=Hoxd10-38:+, id=30 removed by soft count = 747\n",
      "--- number of probes: 150 , kept by blast: 149 , if remove dups: 149\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-39:+\n",
      "--- gene=Hoxd10-39:+, id=3 removed by soft count = 490\n",
      "--- number of probes: 150 , kept by blast: 149 , if remove dups: 149\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-4:+\n",
      "--- gene=Hoxd10-4:+, id=45 removed by soft count = 274\n",
      "--- gene=Hoxd10-4:+, id=73 removed by soft count = 64\n",
      "--- number of probes: 118 , kept by blast: 116 , if remove dups: 116\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-40:+\n",
      "--- number of probes: 124 , kept by blast: 124 , if remove dups: 124\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-41:+\n",
      "--- gene=Hoxd10-41:+, id=98 removed by soft count = 63\n",
      "--- number of probes: 150 , kept by blast: 149 , if remove dups: 149\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-42:+\n",
      "--- gene=Hoxd10-42:+, id=57 removed by soft count = 108\n",
      "--- number of probes: 150 , kept by blast: 149 , if remove dups: 149\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-43:+\n",
      "--- number of probes: 150 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-44:+\n",
      "--- number of probes: 150 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-45:+\n",
      "--- gene=Hoxd10-45:+, id=24 removed by soft count = 324\n",
      "--- gene=Hoxd10-45:+, id=26 removed by soft count = 120\n",
      "--- number of probes: 150 , kept by blast: 148 , if remove dups: 148\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-46:+\n",
      "--- number of probes: 119 , kept by blast: 119 , if remove dups: 119\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-47:+\n",
      "--- gene=Hoxd10-47:+, id=27 removed by soft count = 233\n",
      "--- gene=Hoxd10-47:+, id=64 removed by soft count = 106\n",
      "--- number of probes: 150 , kept by blast: 148 , if remove dups: 148\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-48:+\n",
      "--- number of probes: 150 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-49:+\n",
      "--- number of probes: 150 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-5:+\n",
      "--- gene=Hoxd10-5:+, id=39 removed by soft count = 147\n",
      "--- gene=Hoxd10-5:+, id=57 removed by soft count = 148\n",
      "--- gene=Hoxd10-5:+, id=68 removed by soft count = 62\n",
      "--- gene=Hoxd10-5:+, id=98 removed by soft count = 98\n",
      "--- gene=Hoxd10-5:+, id=99 removed by soft count = 106\n",
      "--- gene=Hoxd10-5:+, id=103 removed by soft count = 57\n",
      "--- gene=Hoxd10-5:+, id=104 removed by soft count = 187\n",
      "--- gene=Hoxd10-5:+, id=123 removed by soft count = 119\n",
      "--- number of probes: 127 , kept by blast: 119 , if remove dups: 119\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-50:+\n",
      "--- number of probes: 150 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-51:+\n",
      "--- gene=Hoxd10-51:+, id=21 removed by soft count = 174\n",
      "--- gene=Hoxd10-51:+, id=96 removed by soft count = 65\n",
      "--- number of probes: 150 , kept by blast: 148 , if remove dups: 148\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-52:+\n",
      "--- gene=Hoxd10-52:+, id=45 removed by soft count = 56\n",
      "--- gene=Hoxd10-52:+, id=69 removed by soft count = 226\n",
      "--- gene=Hoxd10-52:+, id=70 removed by soft count = 111\n",
      "--- gene=Hoxd10-52:+, id=120 removed by soft count = 144\n",
      "--- gene=Hoxd10-52:+, id=125 removed by soft count = 117\n",
      "--- number of probes: 143 , kept by blast: 138 , if remove dups: 138\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-53:+\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 150 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-54:+\n",
      "--- gene=Hoxd10-54:+, id=41 removed by soft count = 760\n",
      "--- gene=Hoxd10-54:+, id=42 removed by soft count = 242\n",
      "--- gene=Hoxd10-54:+, id=43 removed by soft count = 153\n",
      "--- gene=Hoxd10-54:+, id=46 removed by soft count = 73\n",
      "--- gene=Hoxd10-54:+, id=47 removed by soft count = 195\n",
      "--- number of probes: 134 , kept by blast: 129 , if remove dups: 129\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-55:+\n",
      "--- number of probes: 150 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-56:+\n",
      "--- number of probes: 150 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-57:+\n",
      "--- gene=Hoxd10-57:+, id=125 removed by soft count = 87\n",
      "--- number of probes: 150 , kept by blast: 149 , if remove dups: 149\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-58:+\n",
      "--- number of probes: 150 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-59:+\n",
      "--- number of probes: 150 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-6:+\n",
      "--- gene=Hoxd10-6:+, id=20 removed by soft count = 255\n",
      "--- gene=Hoxd10-6:+, id=58 removed by soft count = 68\n",
      "--- number of probes: 150 , kept by blast: 148 , if remove dups: 148\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-60:+\n",
      "--- gene=Hoxd10-60:+, id=118 removed by soft count = 61\n",
      "--- number of probes: 126 , kept by blast: 125 , if remove dups: 125\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-61:+\n",
      "--- number of probes: 144 , kept by blast: 144 , if remove dups: 144\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-62:+\n",
      "--- gene=Hoxd10-62:+, id=27 removed by soft count = 67\n",
      "--- number of probes: 134 , kept by blast: 133 , if remove dups: 133\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-63:+\n",
      "--- gene=Hoxd10-63:+, id=40 removed by soft count = 55\n",
      "--- gene=Hoxd10-63:+, id=42 removed by soft count = 160\n",
      "--- gene=Hoxd10-63:+, id=69 removed by soft count = 127\n",
      "--- gene=Hoxd10-63:+, id=95 removed by soft count = 294\n",
      "--- number of probes: 150 , kept by blast: 146 , if remove dups: 146\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-64:+\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: Hoxd10-65:+\n",
      "--- gene=Hoxd10-65:+, id=96 removed by soft count = 194\n",
      "--- number of probes: 133 , kept by blast: 132 , if remove dups: 132\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-66:+\n",
      "--- gene=Hoxd10-66:+, id=81 removed by soft count = 5822\n",
      "--- number of probes: 150 , kept by blast: 149 , if remove dups: 149\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-67:+\n",
      "--- number of probes: 150 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-68:+\n",
      "--- number of probes: 150 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-69:+\n",
      "--- gene=Hoxd10-69:+, id=40 removed by soft count = 311\n",
      "--- gene=Hoxd10-69:+, id=50 removed by soft count = 364\n",
      "--- number of probes: 141 , kept by blast: 139 , if remove dups: 139\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-7:+\n",
      "--- number of probes: 150 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-70:+\n",
      "--- gene=Hoxd10-70:+, id=36 removed by soft count = 234\n",
      "--- number of probes: 150 , kept by blast: 149 , if remove dups: 149\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-71:+\n",
      "--- gene=Hoxd10-71:+, id=88 removed by soft count = 51\n",
      "--- gene=Hoxd10-71:+, id=105 removed by soft count = 71\n",
      "--- gene=Hoxd10-71:+, id=112 removed by soft count = 344\n",
      "--- number of probes: 143 , kept by blast: 140 , if remove dups: 140\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-72:+\n",
      "--- gene=Hoxd10-72:+, id=98 removed by soft count = 132\n",
      "--- gene=Hoxd10-72:+, id=99 removed by soft count = 5844\n",
      "--- number of probes: 150 , kept by blast: 148 , if remove dups: 148\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-73:+\n",
      "--- number of probes: 150 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-74:+\n",
      "--- gene=Hoxd10-74:+, id=4 removed by soft count = 1035\n",
      "--- gene=Hoxd10-74:+, id=5 removed by soft count = 136\n",
      "--- gene=Hoxd10-74:+, id=68 removed by soft count = 3751\n",
      "--- number of probes: 145 , kept by blast: 142 , if remove dups: 142\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-75:+\n",
      "--- gene=Hoxd10-75:+, id=54 removed by soft count = 189\n",
      "--- number of probes: 117 , kept by blast: 116 , if remove dups: 116\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-76:+\n",
      "--- gene=Hoxd10-76:+, id=20 removed by soft count = 68\n",
      "--- gene=Hoxd10-76:+, id=30 removed by soft count = 87\n",
      "--- gene=Hoxd10-76:+, id=31 removed by soft count = 118\n",
      "--- gene=Hoxd10-76:+, id=32 removed by soft count = 553\n",
      "--- gene=Hoxd10-76:+, id=33 removed by soft count = 7184\n",
      "--- number of probes: 119 , kept by blast: 114 , if remove dups: 114\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-77:+\n",
      "--- gene=Hoxd10-77:+, id=93 removed by soft count = 168\n",
      "--- gene=Hoxd10-77:+, id=115 removed by soft count = 122\n",
      "--- number of probes: 150 , kept by blast: 148 , if remove dups: 148\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-78:+\n",
      "--- gene=Hoxd10-78:+, id=0 removed by soft count = 1416\n",
      "--- gene=Hoxd10-78:+, id=35 removed by soft count = 451\n",
      "--- gene=Hoxd10-78:+, id=36 removed by soft count = 54\n",
      "--- gene=Hoxd10-78:+, id=43 removed by soft count = 293\n",
      "--- gene=Hoxd10-78:+, id=44 removed by soft count = 65\n",
      "--- number of probes: 150 , kept by blast: 145 , if remove dups: 145\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-79:+\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 150 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-8:+\n",
      "--- number of probes: 150 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-80:+\n",
      "--- gene=Hoxd10-80:+, id=82 removed by soft count = 128\n",
      "--- gene=Hoxd10-80:+, id=108 removed by soft count = 88\n",
      "--- gene=Hoxd10-80:+, id=110 removed by soft count = 141\n",
      "--- number of probes: 143 , kept by blast: 140 , if remove dups: 140\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-81:+\n",
      "--- number of probes: 150 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-82:+\n",
      "--- number of probes: 150 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-83:+\n",
      "--- gene=Hoxd10-83:+, id=89 removed by soft count = 91\n",
      "--- number of probes: 150 , kept by blast: 149 , if remove dups: 148\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-84:+\n",
      "--- gene=Hoxd10-84:+, id=133 removed by soft count = 192\n",
      "--- number of probes: 134 , kept by blast: 133 , if remove dups: 133\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-85:+\n",
      "--- gene=Hoxd10-85:+, id=0 removed by soft count = 406\n",
      "--- number of probes: 148 , kept by blast: 147 , if remove dups: 147\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-86:+\n",
      "--- number of probes: 150 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-87:+\n",
      "--- gene=Hoxd10-87:+, id=15 removed by soft count = 85\n",
      "--- number of probes: 150 , kept by blast: 149 , if remove dups: 149\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-88:+\n",
      "--- gene=Hoxd10-88:+, id=80 removed by soft count = 125\n",
      "--- gene=Hoxd10-88:+, id=142 removed by soft count = 95\n",
      "--- number of probes: 150 , kept by blast: 148 , if remove dups: 148\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-89:+\n",
      "--- number of probes: 142 , kept by blast: 142 , if remove dups: 142\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-9:+\n",
      "--- number of probes: 150 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-90:+\n",
      "--- gene=Hoxd10-90:+, id=76 removed by soft count = 71\n",
      "--- gene=Hoxd10-90:+, id=101 removed by soft count = 110\n",
      "--- number of probes: 150 , kept by blast: 148 , if remove dups: 148\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-91:+\n",
      "--- gene=Hoxd10-91:+, id=59 removed by soft count = 57\n",
      "--- gene=Hoxd10-91:+, id=85 removed by soft count = 1705\n",
      "--- gene=Hoxd10-91:+, id=108 removed by soft count = 2578\n",
      "--- number of probes: 109 , kept by blast: 106 , if remove dups: 106\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-92:+\n",
      "--- gene=Hoxd10-92:+, id=63 removed by soft count = 146\n",
      "--- number of probes: 106 , kept by blast: 105 , if remove dups: 105\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-93:+\n",
      "--- gene=Hoxd10-93:+, id=124 removed by soft count = 625\n",
      "--- gene=Hoxd10-93:+, id=125 removed by soft count = 57\n",
      "--- number of probes: 150 , kept by blast: 148 , if remove dups: 148\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-94:+\n",
      "--- gene=Hoxd10-94:+, id=89 removed by soft count = 115\n",
      "--- gene=Hoxd10-94:+, id=130 removed by soft count = 88\n",
      "--- gene=Hoxd10-94:+, id=132 removed by soft count = 60\n",
      "--- number of probes: 134 , kept by blast: 131 , if remove dups: 131\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-95:+\n",
      "--- gene=Hoxd10-95:+, id=122 removed by soft count = 60\n",
      "--- number of probes: 143 , kept by blast: 142 , if remove dups: 142\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-96:+\n",
      "--- gene=Hoxd10-96:+, id=37 removed by soft count = 745\n",
      "--- number of probes: 132 , kept by blast: 131 , if remove dups: 131\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-97:+\n",
      "--- gene=Hoxd10-97:+, id=102 removed by soft count = 60\n",
      "--- number of probes: 150 , kept by blast: 149 , if remove dups: 149\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-98:+\n",
      "--- gene=Hoxd10-98:+, id=51 removed by soft count = 125\n",
      "--- number of probes: 150 , kept by blast: 149 , if remove dups: 148\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Hoxd10-99:+\n",
      "--- gene=Hoxd10-99:+, id=26 removed by soft count = 175\n",
      "--- gene=Hoxd10-99:+, id=99 removed by soft count = 51\n",
      "--- number of probes: 134 , kept by blast: 132 , if remove dups: 132\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-1:+\n",
      "--- gene=Spp2-1:+, id=26 removed by soft count = 77\n",
      "--- gene=Spp2-1:+, id=28 removed by soft count = 122\n",
      "--- gene=Spp2-1:+, id=29 removed by soft count = 89\n",
      "--- gene=Spp2-1:+, id=41 removed by soft count = 213\n",
      "--- number of probes: 150 , kept by blast: 146 , if remove dups: 146\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-10:+\n",
      "--- gene=Spp2-10:+, id=1 removed by soft count = 151\n",
      "--- gene=Spp2-10:+, id=59 removed by soft count = 113\n",
      "--- gene=Spp2-10:+, id=113 removed by soft count = 73\n",
      "--- number of probes: 149 , kept by blast: 146 , if remove dups: 145\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-11:+\n",
      "--- number of probes: 150 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-12:+\n",
      "--- gene=Spp2-12:+, id=10 removed by soft count = 98\n",
      "--- gene=Spp2-12:+, id=79 removed by soft count = 215\n",
      "--- number of probes: 150 , kept by blast: 148 , if remove dups: 148\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-13:+\n",
      "--- gene=Spp2-13:+, id=125 removed by soft count = 99\n",
      "--- number of probes: 139 , kept by blast: 138 , if remove dups: 138\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-14:+\n",
      "--- gene=Spp2-14:+, id=39 removed by soft count = 75\n",
      "--- gene=Spp2-14:+, id=47 removed by soft count = 132\n",
      "--- gene=Spp2-14:+, id=52 removed by soft count = 81\n",
      "--- gene=Spp2-14:+, id=53 removed by soft count = 162\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Spp2-14:+, id=71 removed by soft count = 156\n",
      "--- gene=Spp2-14:+, id=114 removed by soft count = 64\n",
      "--- number of probes: 150 , kept by blast: 144 , if remove dups: 144\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-15:+\n",
      "--- gene=Spp2-15:+, id=12 removed by soft count = 115\n",
      "--- gene=Spp2-15:+, id=13 removed by soft count = 256\n",
      "--- gene=Spp2-15:+, id=14 removed by soft count = 430\n",
      "--- gene=Spp2-15:+, id=16 removed by soft count = 81\n",
      "--- gene=Spp2-15:+, id=29 removed by soft count = 100\n",
      "--- gene=Spp2-15:+, id=76 removed by soft count = 68\n",
      "--- number of probes: 140 , kept by blast: 134 , if remove dups: 134\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-16:+\n",
      "--- gene=Spp2-16:+, id=26 removed by soft count = 220\n",
      "--- gene=Spp2-16:+, id=115 removed by soft count = 1012\n",
      "--- number of probes: 148 , kept by blast: 146 , if remove dups: 146\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-17:+\n",
      "--- number of probes: 150 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-18:+\n",
      "--- gene=Spp2-18:+, id=7 removed by soft count = 53\n",
      "--- gene=Spp2-18:+, id=42 removed by soft count = 195\n",
      "--- gene=Spp2-18:+, id=134 removed by soft count = 287\n",
      "--- number of probes: 147 , kept by blast: 144 , if remove dups: 144\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-19:+\n",
      "--- gene=Spp2-19:+, id=47 removed by soft count = 129\n",
      "--- gene=Spp2-19:+, id=132 removed by soft count = 101\n",
      "--- number of probes: 150 , kept by blast: 148 , if remove dups: 148\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-2:+\n",
      "--- number of probes: 150 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-20:+\n",
      "--- gene=Spp2-20:+, id=85 removed by soft count = 68\n",
      "--- number of probes: 150 , kept by blast: 149 , if remove dups: 149\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-21:+\n",
      "--- gene=Spp2-21:+, id=57 removed by soft count = 161\n",
      "--- number of probes: 150 , kept by blast: 149 , if remove dups: 149\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-22:+\n",
      "--- gene=Spp2-22:+, id=79 removed by soft count = 1336\n",
      "--- number of probes: 127 , kept by blast: 126 , if remove dups: 126\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-23:+\n",
      "--- number of probes: 135 , kept by blast: 135 , if remove dups: 135\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-24:+\n",
      "--- gene=Spp2-24:+, id=70 removed by soft count = 54\n",
      "--- gene=Spp2-24:+, id=126 removed by soft count = 52\n",
      "--- gene=Spp2-24:+, id=127 removed by soft count = 87\n",
      "--- gene=Spp2-24:+, id=129 removed by soft count = 64\n",
      "--- gene=Spp2-24:+, id=130 removed by soft count = 135\n",
      "--- number of probes: 150 , kept by blast: 145 , if remove dups: 145\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-25:+\n",
      "--- gene=Spp2-25:+, id=61 removed by soft count = 85\n",
      "--- gene=Spp2-25:+, id=87 removed by soft count = 146\n",
      "--- number of probes: 120 , kept by blast: 118 , if remove dups: 118\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-26:+\n",
      "--- gene=Spp2-26:+, id=30 removed by soft count = 57\n",
      "--- gene=Spp2-26:+, id=50 removed by soft count = 78\n",
      "--- number of probes: 150 , kept by blast: 148 , if remove dups: 148\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-27:+\n",
      "--- gene=Spp2-27:+, id=3 removed by soft count = 135\n",
      "--- gene=Spp2-27:+, id=7 removed by soft count = 65\n",
      "--- gene=Spp2-27:+, id=16 removed by soft count = 60\n",
      "--- gene=Spp2-27:+, id=41 removed by soft count = 167\n",
      "--- gene=Spp2-27:+, id=43 removed by soft count = 60\n",
      "--- gene=Spp2-27:+, id=54 removed by soft count = 90\n",
      "--- gene=Spp2-27:+, id=55 removed by soft count = 114\n",
      "--- gene=Spp2-27:+, id=57 removed by soft count = 489\n",
      "--- gene=Spp2-27:+, id=59 removed by soft count = 433\n",
      "--- gene=Spp2-27:+, id=60 removed by soft count = 112\n",
      "--- gene=Spp2-27:+, id=62 removed by soft count = 98\n",
      "--- gene=Spp2-27:+, id=76 removed by soft count = 108\n",
      "--- number of probes: 99 , kept by blast: 87 , if remove dups: 87\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 87\n",
      "-- checking probes in region: Spp2-28:+\n",
      "--- gene=Spp2-28:+, id=31 removed by soft count = 528\n",
      "--- gene=Spp2-28:+, id=32 removed by soft count = 91\n",
      "--- gene=Spp2-28:+, id=35 removed by soft count = 103\n",
      "--- gene=Spp2-28:+, id=102 removed by soft count = 56\n",
      "--- gene=Spp2-28:+, id=103 removed by soft count = 52\n",
      "--- gene=Spp2-28:+, id=105 removed by soft count = 77\n",
      "--- number of probes: 145 , kept by blast: 139 , if remove dups: 139\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-29:+\n",
      "--- number of probes: 127 , kept by blast: 127 , if remove dups: 125\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-3:+\n",
      "--- number of probes: 139 , kept by blast: 139 , if remove dups: 139\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-30:+\n",
      "--- gene=Spp2-30:+, id=30 removed by soft count = 101\n",
      "--- gene=Spp2-30:+, id=32 removed by soft count = 984\n",
      "--- gene=Spp2-30:+, id=33 removed by soft count = 4211\n",
      "--- gene=Spp2-30:+, id=78 removed by soft count = 431\n",
      "--- number of probes: 93 , kept by blast: 89 , if remove dups: 89\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 89\n",
      "-- checking probes in region: Spp2-31:+\n",
      "--- gene=Spp2-31:+, id=31 removed by soft count = 405\n",
      "--- number of probes: 115 , kept by blast: 114 , if remove dups: 114\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-32:+\n",
      "--- gene=Spp2-32:+, id=18 removed by soft count = 293\n",
      "--- gene=Spp2-32:+, id=35 removed by soft count = 122\n",
      "--- gene=Spp2-32:+, id=36 removed by soft count = 173\n",
      "--- number of probes: 150 , kept by blast: 147 , if remove dups: 147\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-33:+\n",
      "--- gene=Spp2-33:+, id=37 removed by soft count = 70\n",
      "--- number of probes: 139 , kept by blast: 138 , if remove dups: 138\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-34:+\n",
      "--- gene=Spp2-34:+, id=26 removed by soft count = 184\n",
      "--- gene=Spp2-34:+, id=129 removed by soft count = 173\n",
      "--- gene=Spp2-34:+, id=141 removed by soft count = 100\n",
      "--- number of probes: 144 , kept by blast: 141 , if remove dups: 137\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-35:+\n",
      "--- gene=Spp2-35:+, id=16 removed by soft count = 84\n",
      "--- gene=Spp2-35:+, id=17 removed by soft count = 94\n",
      "--- gene=Spp2-35:+, id=18 removed by soft count = 179\n",
      "--- number of probes: 150 , kept by blast: 147 , if remove dups: 147\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-36:+\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Spp2-36:+, id=58 removed by soft count = 56\n",
      "--- gene=Spp2-36:+, id=116 removed by soft count = 86\n",
      "--- gene=Spp2-36:+, id=117 removed by soft count = 106\n",
      "--- number of probes: 150 , kept by blast: 147 , if remove dups: 147\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-37:+\n",
      "--- gene=Spp2-37:+, id=1 removed by soft count = 355\n",
      "--- gene=Spp2-37:+, id=43 removed by soft count = 236\n",
      "--- gene=Spp2-37:+, id=69 removed by soft count = 4048\n",
      "--- gene=Spp2-37:+, id=92 removed by soft count = 154\n",
      "--- gene=Spp2-37:+, id=102 removed by soft count = 170\n",
      "--- number of probes: 134 , kept by blast: 129 , if remove dups: 129\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-38:+\n",
      "--- gene=Spp2-38:+, id=87 removed by soft count = 52\n",
      "--- number of probes: 150 , kept by blast: 149 , if remove dups: 149\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-39:+\n",
      "--- gene=Spp2-39:+, id=98 removed by soft count = 91\n",
      "--- number of probes: 150 , kept by blast: 149 , if remove dups: 149\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-4:+\n",
      "--- gene=Spp2-4:+, id=54 removed by soft count = 156\n",
      "--- gene=Spp2-4:+, id=55 removed by soft count = 620\n",
      "--- number of probes: 147 , kept by blast: 145 , if remove dups: 145\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-40:+\n",
      "--- gene=Spp2-40:+, id=5 removed by soft count = 63\n",
      "--- number of probes: 150 , kept by blast: 149 , if remove dups: 148\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-41:+\n",
      "--- gene=Spp2-41:+, id=49 removed by soft count = 52\n",
      "--- number of probes: 150 , kept by blast: 149 , if remove dups: 149\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-42:+\n",
      "--- gene=Spp2-42:+, id=0 removed by soft count = 64\n",
      "--- gene=Spp2-42:+, id=82 removed by soft count = 221\n",
      "--- gene=Spp2-42:+, id=83 removed by soft count = 333\n",
      "--- gene=Spp2-42:+, id=84 removed by soft count = 56\n",
      "--- gene=Spp2-42:+, id=85 removed by soft count = 101\n",
      "--- gene=Spp2-42:+, id=86 removed by soft count = 101\n",
      "--- gene=Spp2-42:+, id=88 removed by soft count = 104\n",
      "--- gene=Spp2-42:+, id=89 removed by soft count = 100\n",
      "--- gene=Spp2-42:+, id=90 removed by soft count = 113\n",
      "--- gene=Spp2-42:+, id=102 removed by soft count = 57\n",
      "--- number of probes: 144 , kept by blast: 134 , if remove dups: 134\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-43:+\n",
      "--- gene=Spp2-43:+, id=106 removed by soft count = 58\n",
      "--- number of probes: 131 , kept by blast: 130 , if remove dups: 130\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-44:+\n",
      "--- gene=Spp2-44:+, id=41 removed by soft count = 75\n",
      "--- number of probes: 150 , kept by blast: 149 , if remove dups: 149\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-45:+\n",
      "--- gene=Spp2-45:+, id=54 removed by soft count = 61\n",
      "--- gene=Spp2-45:+, id=97 removed by soft count = 60\n",
      "--- gene=Spp2-45:+, id=113 removed by soft count = 180\n",
      "--- gene=Spp2-45:+, id=127 removed by soft count = 71\n",
      "--- gene=Spp2-45:+, id=130 removed by soft count = 126\n",
      "--- gene=Spp2-45:+, id=131 removed by soft count = 182\n",
      "--- number of probes: 150 , kept by blast: 144 , if remove dups: 144\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-46:+\n",
      "--- gene=Spp2-46:+, id=64 removed by soft count = 57\n",
      "--- gene=Spp2-46:+, id=68 removed by soft count = 52\n",
      "--- gene=Spp2-46:+, id=69 removed by soft count = 752\n",
      "--- gene=Spp2-46:+, id=70 removed by soft count = 251\n",
      "--- gene=Spp2-46:+, id=79 removed by soft count = 68\n",
      "--- gene=Spp2-46:+, id=98 removed by soft count = 68\n",
      "--- number of probes: 149 , kept by blast: 143 , if remove dups: 143\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-47:+\n",
      "--- gene=Spp2-47:+, id=72 removed by soft count = 3220\n",
      "--- gene=Spp2-47:+, id=74 removed by soft count = 147\n",
      "--- number of probes: 142 , kept by blast: 140 , if remove dups: 140\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-48:+\n",
      "--- gene=Spp2-48:+, id=73 removed by soft count = 91\n",
      "--- gene=Spp2-48:+, id=76 removed by soft count = 133\n",
      "--- gene=Spp2-48:+, id=77 removed by soft count = 345\n",
      "--- number of probes: 143 , kept by blast: 140 , if remove dups: 140\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-49:+\n",
      "--- gene=Spp2-49:+, id=8 removed by soft count = 57\n",
      "--- gene=Spp2-49:+, id=9 removed by soft count = 639\n",
      "--- gene=Spp2-49:+, id=103 removed by soft count = 105\n",
      "--- number of probes: 139 , kept by blast: 136 , if remove dups: 136\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-5:+\n",
      "--- gene=Spp2-5:+, id=36 removed by soft count = 389\n",
      "--- gene=Spp2-5:+, id=133 removed by soft count = 185\n",
      "--- number of probes: 150 , kept by blast: 148 , if remove dups: 148\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-50:+\n",
      "--- gene=Spp2-50:+, id=17 removed by soft count = 93\n",
      "--- gene=Spp2-50:+, id=67 removed by soft count = 74\n",
      "--- gene=Spp2-50:+, id=94 removed by soft count = 137\n",
      "--- gene=Spp2-50:+, id=97 removed by soft count = 243\n",
      "--- gene=Spp2-50:+, id=98 removed by soft count = 197\n",
      "--- gene=Spp2-50:+, id=99 removed by soft count = 171\n",
      "--- gene=Spp2-50:+, id=138 removed by soft count = 402\n",
      "--- number of probes: 149 , kept by blast: 142 , if remove dups: 142\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-51:+\n",
      "--- number of probes: 150 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-52:+\n",
      "--- gene=Spp2-52:+, id=20 removed by soft count = 61\n",
      "--- gene=Spp2-52:+, id=23 removed by soft count = 51\n",
      "--- gene=Spp2-52:+, id=40 removed by soft count = 238\n",
      "--- gene=Spp2-52:+, id=88 removed by soft count = 58\n",
      "--- gene=Spp2-52:+, id=89 removed by soft count = 135\n",
      "--- gene=Spp2-52:+, id=135 removed by soft count = 65\n",
      "--- gene=Spp2-52:+, id=137 removed by soft count = 226\n",
      "--- gene=Spp2-52:+, id=139 removed by soft count = 98\n",
      "--- number of probes: 144 , kept by blast: 136 , if remove dups: 136\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-53:+\n",
      "--- gene=Spp2-53:+, id=39 removed by soft count = 83\n",
      "--- number of probes: 53 , kept by blast: 52 , if remove dups: 52\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: Spp2-54:+\n",
      "--- gene=Spp2-54:+, id=106 removed by soft count = 86\n",
      "--- number of probes: 150 , kept by blast: 149 , if remove dups: 149\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-55:+\n",
      "--- gene=Spp2-55:+, id=29 removed by soft count = 249\n",
      "--- gene=Spp2-55:+, id=49 removed by soft count = 1262\n",
      "--- gene=Spp2-55:+, id=51 removed by soft count = 440\n",
      "--- gene=Spp2-55:+, id=86 removed by soft count = 27500\n",
      "--- number of probes: 87 , kept by blast: 83 , if remove dups: 83\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 83\n",
      "-- checking probes in region: Spp2-56:+\n",
      "--- gene=Spp2-56:+, id=0 removed by soft count = 601\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Spp2-56:+, id=51 removed by soft count = 176\n",
      "--- gene=Spp2-56:+, id=61 removed by soft count = 62\n",
      "--- number of probes: 126 , kept by blast: 123 , if remove dups: 123\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-57:+\n",
      "--- number of probes: 149 , kept by blast: 149 , if remove dups: 148\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-58:+\n",
      "--- gene=Spp2-58:+, id=124 removed by soft count = 510\n",
      "--- gene=Spp2-58:+, id=141 removed by soft count = 900\n",
      "--- number of probes: 146 , kept by blast: 144 , if remove dups: 143\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-59:+\n",
      "--- gene=Spp2-59:+, id=0 removed by soft count = 240\n",
      "--- gene=Spp2-59:+, id=1 removed by soft count = 86\n",
      "--- gene=Spp2-59:+, id=6 removed by soft count = 75\n",
      "--- gene=Spp2-59:+, id=27 removed by soft count = 89\n",
      "--- gene=Spp2-59:+, id=28 removed by soft count = 220\n",
      "--- gene=Spp2-59:+, id=40 removed by soft count = 155\n",
      "--- gene=Spp2-59:+, id=104 removed by soft count = 84\n",
      "--- gene=Spp2-59:+, id=105 removed by soft count = 162\n",
      "--- number of probes: 120 , kept by blast: 112 , if remove dups: 112\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-6:+\n",
      "--- gene=Spp2-6:+, id=132 removed by soft count = 78\n",
      "--- number of probes: 150 , kept by blast: 149 , if remove dups: 149\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-60:+\n",
      "--- gene=Spp2-60:+, id=60 removed by soft count = 125\n",
      "--- gene=Spp2-60:+, id=113 removed by soft count = 249\n",
      "--- gene=Spp2-60:+, id=116 removed by soft count = 68\n",
      "--- gene=Spp2-60:+, id=118 removed by soft count = 328\n",
      "--- gene=Spp2-60:+, id=120 removed by soft count = 137\n",
      "--- number of probes: 139 , kept by blast: 134 , if remove dups: 134\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-61:+\n",
      "--- gene=Spp2-61:+, id=4 removed by soft count = 173\n",
      "--- gene=Spp2-61:+, id=118 removed by soft count = 97\n",
      "--- gene=Spp2-61:+, id=131 removed by soft count = 129\n",
      "--- gene=Spp2-61:+, id=134 removed by soft count = 732\n",
      "--- number of probes: 147 , kept by blast: 143 , if remove dups: 143\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-62:+\n",
      "--- gene=Spp2-62:+, id=32 removed by soft count = 197\n",
      "--- gene=Spp2-62:+, id=34 removed by soft count = 58\n",
      "--- gene=Spp2-62:+, id=38 removed by soft count = 102\n",
      "--- gene=Spp2-62:+, id=39 removed by soft count = 181\n",
      "--- gene=Spp2-62:+, id=40 removed by soft count = 78\n",
      "--- gene=Spp2-62:+, id=41 removed by soft count = 119\n",
      "--- number of probes: 147 , kept by blast: 141 , if remove dups: 141\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-63:+\n",
      "--- gene=Spp2-63:+, id=22 removed by soft count = 112\n",
      "--- gene=Spp2-63:+, id=41 removed by soft count = 124\n",
      "--- number of probes: 144 , kept by blast: 142 , if remove dups: 142\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-64:+\n",
      "--- gene=Spp2-64:+, id=55 removed by soft count = 84\n",
      "--- number of probes: 150 , kept by blast: 149 , if remove dups: 149\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-65:+\n",
      "--- gene=Spp2-65:+, id=10 removed by soft count = 77\n",
      "--- gene=Spp2-65:+, id=56 removed by soft count = 100\n",
      "--- gene=Spp2-65:+, id=57 removed by soft count = 262\n",
      "--- gene=Spp2-65:+, id=63 removed by soft count = 115\n",
      "--- gene=Spp2-65:+, id=64 removed by soft count = 267\n",
      "--- gene=Spp2-65:+, id=66 removed by soft count = 68\n",
      "--- gene=Spp2-65:+, id=95 removed by soft count = 59\n",
      "--- gene=Spp2-65:+, id=108 removed by soft count = 211\n",
      "--- number of probes: 131 , kept by blast: 123 , if remove dups: 122\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-66:+\n",
      "--- gene=Spp2-66:+, id=13 removed by soft count = 51\n",
      "--- gene=Spp2-66:+, id=17 removed by soft count = 809\n",
      "--- gene=Spp2-66:+, id=19 removed by soft count = 791\n",
      "--- gene=Spp2-66:+, id=22 removed by soft count = 189\n",
      "--- number of probes: 150 , kept by blast: 146 , if remove dups: 146\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-67:+\n",
      "--- gene=Spp2-67:+, id=1 removed by soft count = 118\n",
      "--- gene=Spp2-67:+, id=8 removed by soft count = 166\n",
      "--- gene=Spp2-67:+, id=10 removed by soft count = 117\n",
      "--- gene=Spp2-67:+, id=21 removed by soft count = 83\n",
      "--- gene=Spp2-67:+, id=63 removed by soft count = 91\n",
      "--- gene=Spp2-67:+, id=65 removed by soft count = 126\n",
      "--- number of probes: 125 , kept by blast: 119 , if remove dups: 119\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-68:+\n",
      "--- gene=Spp2-68:+, id=9 removed by soft count = 440\n",
      "--- gene=Spp2-68:+, id=50 removed by soft count = 229\n",
      "--- gene=Spp2-68:+, id=129 removed by soft count = 51\n",
      "--- number of probes: 140 , kept by blast: 137 , if remove dups: 137\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-69:+\n",
      "--- gene=Spp2-69:+, id=28 removed by soft count = 68\n",
      "--- gene=Spp2-69:+, id=29 removed by soft count = 76\n",
      "--- gene=Spp2-69:+, id=103 removed by soft count = 55\n",
      "--- number of probes: 150 , kept by blast: 147 , if remove dups: 147\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-7:+\n",
      "--- gene=Spp2-7:+, id=23 removed by soft count = 79\n",
      "--- gene=Spp2-7:+, id=107 removed by soft count = 77\n",
      "--- number of probes: 150 , kept by blast: 148 , if remove dups: 148\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-70:+\n",
      "--- gene=Spp2-70:+, id=82 removed by soft count = 53\n",
      "--- gene=Spp2-70:+, id=105 removed by soft count = 89\n",
      "--- gene=Spp2-70:+, id=131 removed by soft count = 82\n",
      "--- gene=Spp2-70:+, id=132 removed by soft count = 184\n",
      "--- number of probes: 133 , kept by blast: 129 , if remove dups: 129\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-71:+\n",
      "--- gene=Spp2-71:+, id=3 removed by soft count = 51\n",
      "--- gene=Spp2-71:+, id=60 removed by soft count = 91\n",
      "--- gene=Spp2-71:+, id=64 removed by soft count = 185\n",
      "--- number of probes: 150 , kept by blast: 147 , if remove dups: 147\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-72:+\n",
      "--- number of probes: 142 , kept by blast: 142 , if remove dups: 142\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-73:+\n",
      "--- gene=Spp2-73:+, id=67 removed by soft count = 53\n",
      "--- gene=Spp2-73:+, id=68 removed by soft count = 74\n",
      "--- number of probes: 143 , kept by blast: 141 , if remove dups: 141\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-74:+\n",
      "--- gene=Spp2-74:+, id=111 removed by soft count = 93\n",
      "--- number of probes: 147 , kept by blast: 146 , if remove dups: 146\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-75:+\n",
      "--- gene=Spp2-75:+, id=79 removed by soft count = 308\n",
      "--- gene=Spp2-75:+, id=105 removed by soft count = 573\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Spp2-75:+, id=149 removed by soft count = 104\n",
      "--- number of probes: 150 , kept by blast: 147 , if remove dups: 147\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-76:+\n",
      "--- gene=Spp2-76:+, id=62 removed by soft count = 107\n",
      "--- gene=Spp2-76:+, id=65 removed by soft count = 89\n",
      "--- gene=Spp2-76:+, id=67 removed by soft count = 69\n",
      "--- number of probes: 132 , kept by blast: 129 , if remove dups: 129\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-77:+\n",
      "--- gene=Spp2-77:+, id=20 removed by soft count = 101\n",
      "--- gene=Spp2-77:+, id=103 removed by soft count = 69\n",
      "--- gene=Spp2-77:+, id=106 removed by soft count = 99\n",
      "--- number of probes: 150 , kept by blast: 147 , if remove dups: 147\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-78:+\n",
      "--- number of probes: 150 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-79:+\n",
      "--- gene=Spp2-79:+, id=54 removed by soft count = 212\n",
      "--- number of probes: 150 , kept by blast: 149 , if remove dups: 149\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-8:+\n",
      "--- number of probes: 143 , kept by blast: 143 , if remove dups: 143\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-80:+\n",
      "--- number of probes: 83 , kept by blast: 83 , if remove dups: 83\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 83\n",
      "-- checking probes in region: Spp2-81:+\n",
      "--- gene=Spp2-81:+, id=101 removed by soft count = 139\n",
      "--- number of probes: 150 , kept by blast: 149 , if remove dups: 149\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-82:+\n",
      "--- gene=Spp2-82:+, id=103 removed by hard count = 0\n",
      "--- gene=Spp2-82:+, id=133 removed by soft count = 872\n",
      "--- number of probes: 150 , kept by blast: 148 , if remove dups: 148\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-83:+\n",
      "--- gene=Spp2-83:+, id=5 removed by soft count = 244\n",
      "--- gene=Spp2-83:+, id=11 removed by soft count = 101\n",
      "--- gene=Spp2-83:+, id=45 removed by soft count = 72\n",
      "--- gene=Spp2-83:+, id=124 removed by soft count = 154\n",
      "--- number of probes: 150 , kept by blast: 146 , if remove dups: 146\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-84:+\n",
      "--- gene=Spp2-84:+, id=43 removed by soft count = 64\n",
      "--- gene=Spp2-84:+, id=138 removed by soft count = 55\n",
      "--- number of probes: 150 , kept by blast: 148 , if remove dups: 148\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-85:+\n",
      "--- gene=Spp2-85:+, id=48 removed by soft count = 67\n",
      "--- number of probes: 150 , kept by blast: 149 , if remove dups: 149\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-86:+\n",
      "--- gene=Spp2-86:+, id=60 removed by soft count = 88\n",
      "--- gene=Spp2-86:+, id=121 removed by soft count = 227\n",
      "--- number of probes: 141 , kept by blast: 139 , if remove dups: 139\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-87:+\n",
      "--- gene=Spp2-87:+, id=1 removed by soft count = 133\n",
      "--- gene=Spp2-87:+, id=18 removed by soft count = 59\n",
      "--- gene=Spp2-87:+, id=23 removed by soft count = 112\n",
      "--- gene=Spp2-87:+, id=24 removed by soft count = 63\n",
      "--- gene=Spp2-87:+, id=33 removed by soft count = 844\n",
      "--- gene=Spp2-87:+, id=68 removed by soft count = 118\n",
      "--- gene=Spp2-87:+, id=123 removed by soft count = 79\n",
      "--- gene=Spp2-87:+, id=126 removed by soft count = 161\n",
      "--- number of probes: 128 , kept by blast: 120 , if remove dups: 120\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-88:+\n",
      "--- gene=Spp2-88:+, id=61 removed by soft count = 410\n",
      "--- gene=Spp2-88:+, id=81 removed by soft count = 441\n",
      "--- gene=Spp2-88:+, id=82 removed by soft count = 159\n",
      "--- number of probes: 144 , kept by blast: 141 , if remove dups: 141\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-89:+\n",
      "--- gene=Spp2-89:+, id=25 removed by soft count = 188\n",
      "--- gene=Spp2-89:+, id=36 removed by soft count = 603\n",
      "--- gene=Spp2-89:+, id=37 removed by soft count = 1025\n",
      "--- gene=Spp2-89:+, id=68 removed by soft count = 64\n",
      "--- number of probes: 150 , kept by blast: 146 , if remove dups: 146\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-9:+\n",
      "--- gene=Spp2-9:+, id=36 removed by soft count = 247\n",
      "--- gene=Spp2-9:+, id=71 removed by soft count = 87\n",
      "--- gene=Spp2-9:+, id=75 removed by soft count = 76\n",
      "--- gene=Spp2-9:+, id=85 removed by soft count = 53\n",
      "--- gene=Spp2-9:+, id=93 removed by soft count = 185\n",
      "--- gene=Spp2-9:+, id=94 removed by soft count = 57\n",
      "--- gene=Spp2-9:+, id=95 removed by soft count = 59\n",
      "--- gene=Spp2-9:+, id=96 removed by soft count = 564\n",
      "--- gene=Spp2-9:+, id=142 removed by soft count = 129\n",
      "--- gene=Spp2-9:+, id=149 removed by soft count = 222\n",
      "--- number of probes: 150 , kept by blast: 140 , if remove dups: 140\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-90:+\n",
      "--- number of probes: 88 , kept by blast: 88 , if remove dups: 88\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 88\n",
      "-- checking probes in region: Spp2-91:+\n",
      "--- gene=Spp2-91:+, id=86 removed by soft count = 196\n",
      "--- gene=Spp2-91:+, id=102 removed by soft count = 606\n",
      "--- gene=Spp2-91:+, id=122 removed by soft count = 140\n",
      "--- number of probes: 130 , kept by blast: 127 , if remove dups: 127\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: Spp2-92:+\n",
      "--- gene=Spp2-92:+, id=16 removed by soft count = 52\n",
      "--- gene=Spp2-92:+, id=22 removed by soft count = 61\n",
      "--- gene=Spp2-92:+, id=34 removed by soft count = 107\n",
      "--- gene=Spp2-92:+, id=45 removed by soft count = 109\n",
      "--- gene=Spp2-92:+, id=46 removed by soft count = 150\n",
      "--- gene=Spp2-92:+, id=47 removed by soft count = 472\n",
      "--- number of probes: 50 , kept by blast: 44 , if remove dups: 44\n",
      "-- number of probes kept for this region: 44\n",
      "- summarize\n",
      "-- region: Camk1-1:- , number of probes: 100\n",
      "-- region: Camk1-10:- , number of probes: 100\n",
      "-- region: Camk1-11:- , number of probes: 100\n",
      "-- region: Camk1-12:- , number of probes: 100\n",
      "-- region: Camk1-13:- , number of probes: 100\n",
      "-- region: Camk1-14:- , number of probes: 100\n",
      "-- region: Camk1-15:- , number of probes: 100\n",
      "-- region: Camk1-16:- , number of probes: 100\n",
      "-- region: Camk1-17:- , number of probes: 100\n",
      "-- region: Camk1-18:- , number of probes: 100\n",
      "-- region: Camk1-19:- , number of probes: 100\n",
      "-- region: Camk1-2:- , number of probes: 100\n",
      "-- region: Camk1-20:- , number of probes: 100\n",
      "-- region: Camk1-21:- , number of probes: 100\n",
      "-- region: Camk1-22:- , number of probes: 100\n",
      "-- region: Camk1-23:- , number of probes: 100\n",
      "-- region: Camk1-24:- , number of probes: 100\n",
      "-- region: Camk1-25:- , number of probes: 100\n",
      "-- region: Camk1-26:- , number of probes: 100\n",
      "-- region: Camk1-27:- , number of probes: 100\n",
      "-- region: Camk1-28:- , number of probes: 100\n",
      "-- region: Camk1-29:- , number of probes: 100\n",
      "-- region: Camk1-3:- , number of probes: 100\n",
      "-- region: Camk1-30:- , number of probes: 100\n",
      "-- region: Camk1-31:- , number of probes: 100\n",
      "-- region: Camk1-32:- , number of probes: 100\n",
      "-- region: Camk1-33:- , number of probes: 100\n",
      "-- region: Camk1-34:- , number of probes: 100\n",
      "-- region: Camk1-35:- , number of probes: 100\n",
      "-- region: Camk1-36:- , number of probes: 100\n",
      "-- region: Camk1-37:- , number of probes: 100\n",
      "-- region: Camk1-38:- , number of probes: 100\n",
      "-- region: Camk1-39:- , number of probes: 100\n",
      "-- region: Camk1-4:- , number of probes: 100\n",
      "-- region: Camk1-40:- , number of probes: 100\n",
      "-- region: Camk1-41:- , number of probes: 100\n",
      "-- region: Camk1-42:- , number of probes: 100\n",
      "-- region: Camk1-43:- , number of probes: 100\n",
      "-- region: Camk1-44:- , number of probes: 100\n",
      "-- region: Camk1-45:- , number of probes: 100\n",
      "-- region: Camk1-46:- , number of probes: 100\n",
      "-- region: Camk1-47:- , number of probes: 100\n",
      "-- region: Camk1-48:- , number of probes: 100\n",
      "-- region: Camk1-49:- , number of probes: 100\n",
      "-- region: Camk1-5:- , number of probes: 100\n",
      "-- region: Camk1-50:- , number of probes: 100\n",
      "-- region: Camk1-51:- , number of probes: 100\n",
      "-- region: Camk1-52:- , number of probes: 100\n",
      "-- region: Camk1-53:- , number of probes: 100\n",
      "-- region: Camk1-54:- , number of probes: 100\n",
      "-- region: Camk1-6:- , number of probes: 100\n",
      "-- region: Camk1-7:- , number of probes: 100\n",
      "-- region: Camk1-8:- , number of probes: 100\n",
      "-- region: Camk1-9:- , number of probes: 100\n",
      "-- region: Hoxd10-1:+ , number of probes: 100\n",
      "-- region: Hoxd10-10:+ , number of probes: 100\n",
      "-- region: Hoxd10-100:+ , number of probes: 100\n",
      "-- region: Hoxd10-101:+ , number of probes: 100\n",
      "-- region: Hoxd10-102:+ , number of probes: 100\n",
      "-- region: Hoxd10-103:+ , number of probes: 100\n",
      "-- region: Hoxd10-104:+ , number of probes: 100\n",
      "-- region: Hoxd10-105:+ , number of probes: 68\n",
      "-- region: Hoxd10-106:+ , number of probes: 100\n",
      "-- region: Hoxd10-107:+ , number of probes: 100\n",
      "-- region: Hoxd10-108:+ , number of probes: 100\n",
      "-- region: Hoxd10-109:+ , number of probes: 100\n",
      "-- region: Hoxd10-11:+ , number of probes: 100\n",
      "-- region: Hoxd10-110:+ , number of probes: 100\n",
      "-- region: Hoxd10-111:+ , number of probes: 100\n",
      "-- region: Hoxd10-112:+ , number of probes: 100\n",
      "-- region: Hoxd10-113:+ , number of probes: 100\n",
      "-- region: Hoxd10-114:+ , number of probes: 100\n",
      "-- region: Hoxd10-115:+ , number of probes: 100\n",
      "-- region: Hoxd10-116:+ , number of probes: 100\n",
      "-- region: Hoxd10-117:+ , number of probes: 100\n",
      "-- region: Hoxd10-118:+ , number of probes: 100\n",
      "-- region: Hoxd10-12:+ , number of probes: 100\n",
      "-- region: Hoxd10-13:+ , number of probes: 100\n",
      "-- region: Hoxd10-14:+ , number of probes: 100\n",
      "-- region: Hoxd10-15:+ , number of probes: 100\n",
      "-- region: Hoxd10-16:+ , number of probes: 100\n",
      "-- region: Hoxd10-17:+ , number of probes: 100\n",
      "-- region: Hoxd10-18:+ , number of probes: 100\n",
      "-- region: Hoxd10-19:+ , number of probes: 100\n",
      "-- region: Hoxd10-2:+ , number of probes: 75\n",
      "-- region: Hoxd10-20:+ , number of probes: 100\n",
      "-- region: Hoxd10-21:+ , number of probes: 100\n",
      "-- region: Hoxd10-22:+ , number of probes: 100\n",
      "-- region: Hoxd10-23:+ , number of probes: 100\n",
      "-- region: Hoxd10-24:+ , number of probes: 100\n",
      "-- region: Hoxd10-25:+ , number of probes: 100\n",
      "-- region: Hoxd10-26:+ , number of probes: 100\n",
      "-- region: Hoxd10-27:+ , number of probes: 98\n",
      "-- region: Hoxd10-28:+ , number of probes: 100\n",
      "-- region: Hoxd10-29:+ , number of probes: 100\n",
      "-- region: Hoxd10-3:+ , number of probes: 100\n",
      "-- region: Hoxd10-30:+ , number of probes: 100\n",
      "-- region: Hoxd10-31:+ , number of probes: 100\n",
      "-- region: Hoxd10-32:+ , number of probes: 100\n",
      "-- region: Hoxd10-33:+ , number of probes: 100\n",
      "-- region: Hoxd10-34:+ , number of probes: 100\n",
      "-- region: Hoxd10-35:+ , number of probes: 100\n",
      "-- region: Hoxd10-36:+ , number of probes: 100\n",
      "-- region: Hoxd10-37:+ , number of probes: 100\n",
      "-- region: Hoxd10-38:+ , number of probes: 100\n",
      "-- region: Hoxd10-39:+ , number of probes: 100\n",
      "-- region: Hoxd10-4:+ , number of probes: 100\n",
      "-- region: Hoxd10-40:+ , number of probes: 100\n",
      "-- region: Hoxd10-41:+ , number of probes: 100\n",
      "-- region: Hoxd10-42:+ , number of probes: 100\n",
      "-- region: Hoxd10-43:+ , number of probes: 100\n",
      "-- region: Hoxd10-44:+ , number of probes: 100\n",
      "-- region: Hoxd10-45:+ , number of probes: 100\n",
      "-- region: Hoxd10-46:+ , number of probes: 100\n",
      "-- region: Hoxd10-47:+ , number of probes: 100\n",
      "-- region: Hoxd10-48:+ , number of probes: 100\n",
      "-- region: Hoxd10-49:+ , number of probes: 100\n",
      "-- region: Hoxd10-5:+ , number of probes: 100\n",
      "-- region: Hoxd10-50:+ , number of probes: 100\n",
      "-- region: Hoxd10-51:+ , number of probes: 100\n",
      "-- region: Hoxd10-52:+ , number of probes: 100\n",
      "-- region: Hoxd10-53:+ , number of probes: 100\n",
      "-- region: Hoxd10-54:+ , number of probes: 100\n",
      "-- region: Hoxd10-55:+ , number of probes: 100\n",
      "-- region: Hoxd10-56:+ , number of probes: 100\n",
      "-- region: Hoxd10-57:+ , number of probes: 100\n",
      "-- region: Hoxd10-58:+ , number of probes: 100\n",
      "-- region: Hoxd10-59:+ , number of probes: 100\n",
      "-- region: Hoxd10-6:+ , number of probes: 100\n",
      "-- region: Hoxd10-60:+ , number of probes: 100\n",
      "-- region: Hoxd10-61:+ , number of probes: 100\n",
      "-- region: Hoxd10-62:+ , number of probes: 100\n",
      "-- region: Hoxd10-63:+ , number of probes: 100\n",
      "-- region: Hoxd10-64:+ , number of probes: 80\n",
      "-- region: Hoxd10-65:+ , number of probes: 100\n",
      "-- region: Hoxd10-66:+ , number of probes: 100\n",
      "-- region: Hoxd10-67:+ , number of probes: 100\n",
      "-- region: Hoxd10-68:+ , number of probes: 100\n",
      "-- region: Hoxd10-69:+ , number of probes: 100\n",
      "-- region: Hoxd10-7:+ , number of probes: 100\n",
      "-- region: Hoxd10-70:+ , number of probes: 100\n",
      "-- region: Hoxd10-71:+ , number of probes: 100\n",
      "-- region: Hoxd10-72:+ , number of probes: 100\n",
      "-- region: Hoxd10-73:+ , number of probes: 100\n",
      "-- region: Hoxd10-74:+ , number of probes: 100\n",
      "-- region: Hoxd10-75:+ , number of probes: 100\n",
      "-- region: Hoxd10-76:+ , number of probes: 100\n",
      "-- region: Hoxd10-77:+ , number of probes: 100\n",
      "-- region: Hoxd10-78:+ , number of probes: 100\n",
      "-- region: Hoxd10-79:+ , number of probes: 100\n",
      "-- region: Hoxd10-8:+ , number of probes: 100\n",
      "-- region: Hoxd10-80:+ , number of probes: 100\n",
      "-- region: Hoxd10-81:+ , number of probes: 100\n",
      "-- region: Hoxd10-82:+ , number of probes: 100\n",
      "-- region: Hoxd10-83:+ , number of probes: 100\n",
      "-- region: Hoxd10-84:+ , number of probes: 100\n",
      "-- region: Hoxd10-85:+ , number of probes: 100\n",
      "-- region: Hoxd10-86:+ , number of probes: 100\n",
      "-- region: Hoxd10-87:+ , number of probes: 100\n",
      "-- region: Hoxd10-88:+ , number of probes: 100\n",
      "-- region: Hoxd10-89:+ , number of probes: 100\n",
      "-- region: Hoxd10-9:+ , number of probes: 100\n",
      "-- region: Hoxd10-90:+ , number of probes: 100\n",
      "-- region: Hoxd10-91:+ , number of probes: 100\n",
      "-- region: Hoxd10-92:+ , number of probes: 100\n",
      "-- region: Hoxd10-93:+ , number of probes: 100\n",
      "-- region: Hoxd10-94:+ , number of probes: 100\n",
      "-- region: Hoxd10-95:+ , number of probes: 100\n",
      "-- region: Hoxd10-96:+ , number of probes: 100\n",
      "-- region: Hoxd10-97:+ , number of probes: 100\n",
      "-- region: Hoxd10-98:+ , number of probes: 100\n",
      "-- region: Hoxd10-99:+ , number of probes: 100\n",
      "-- region: Spp2-1:+ , number of probes: 100\n",
      "-- region: Spp2-10:+ , number of probes: 100\n",
      "-- region: Spp2-11:+ , number of probes: 100\n",
      "-- region: Spp2-12:+ , number of probes: 100\n",
      "-- region: Spp2-13:+ , number of probes: 100\n",
      "-- region: Spp2-14:+ , number of probes: 100\n",
      "-- region: Spp2-15:+ , number of probes: 100\n",
      "-- region: Spp2-16:+ , number of probes: 100\n",
      "-- region: Spp2-17:+ , number of probes: 100\n",
      "-- region: Spp2-18:+ , number of probes: 100\n",
      "-- region: Spp2-19:+ , number of probes: 100\n",
      "-- region: Spp2-2:+ , number of probes: 100\n",
      "-- region: Spp2-20:+ , number of probes: 100\n",
      "-- region: Spp2-21:+ , number of probes: 100\n",
      "-- region: Spp2-22:+ , number of probes: 100\n",
      "-- region: Spp2-23:+ , number of probes: 100\n",
      "-- region: Spp2-24:+ , number of probes: 100\n",
      "-- region: Spp2-25:+ , number of probes: 100\n",
      "-- region: Spp2-26:+ , number of probes: 100\n",
      "-- region: Spp2-27:+ , number of probes: 87\n",
      "-- region: Spp2-28:+ , number of probes: 100\n",
      "-- region: Spp2-29:+ , number of probes: 100\n",
      "-- region: Spp2-3:+ , number of probes: 100\n",
      "-- region: Spp2-30:+ , number of probes: 89\n",
      "-- region: Spp2-31:+ , number of probes: 100\n",
      "-- region: Spp2-32:+ , number of probes: 100\n",
      "-- region: Spp2-33:+ , number of probes: 100\n",
      "-- region: Spp2-34:+ , number of probes: 100\n",
      "-- region: Spp2-35:+ , number of probes: 100\n",
      "-- region: Spp2-36:+ , number of probes: 100\n",
      "-- region: Spp2-37:+ , number of probes: 100\n",
      "-- region: Spp2-38:+ , number of probes: 100\n",
      "-- region: Spp2-39:+ , number of probes: 100\n",
      "-- region: Spp2-4:+ , number of probes: 100\n",
      "-- region: Spp2-40:+ , number of probes: 100\n",
      "-- region: Spp2-41:+ , number of probes: 100\n",
      "-- region: Spp2-42:+ , number of probes: 100\n",
      "-- region: Spp2-43:+ , number of probes: 100\n",
      "-- region: Spp2-44:+ , number of probes: 100\n",
      "-- region: Spp2-45:+ , number of probes: 100\n",
      "-- region: Spp2-46:+ , number of probes: 100\n",
      "-- region: Spp2-47:+ , number of probes: 100\n",
      "-- region: Spp2-48:+ , number of probes: 100\n",
      "-- region: Spp2-49:+ , number of probes: 100\n",
      "-- region: Spp2-5:+ , number of probes: 100\n",
      "-- region: Spp2-50:+ , number of probes: 100\n",
      "-- region: Spp2-51:+ , number of probes: 100\n",
      "-- region: Spp2-52:+ , number of probes: 100\n",
      "-- region: Spp2-53:+ , number of probes: 52\n",
      "-- region: Spp2-54:+ , number of probes: 100\n",
      "-- region: Spp2-55:+ , number of probes: 83\n",
      "-- region: Spp2-56:+ , number of probes: 100\n",
      "-- region: Spp2-57:+ , number of probes: 100\n",
      "-- region: Spp2-58:+ , number of probes: 100\n",
      "-- region: Spp2-59:+ , number of probes: 100\n",
      "-- region: Spp2-6:+ , number of probes: 100\n",
      "-- region: Spp2-60:+ , number of probes: 100\n",
      "-- region: Spp2-61:+ , number of probes: 100\n",
      "-- region: Spp2-62:+ , number of probes: 100\n",
      "-- region: Spp2-63:+ , number of probes: 100\n",
      "-- region: Spp2-64:+ , number of probes: 100\n",
      "-- region: Spp2-65:+ , number of probes: 100\n",
      "-- region: Spp2-66:+ , number of probes: 100\n",
      "-- region: Spp2-67:+ , number of probes: 100\n",
      "-- region: Spp2-68:+ , number of probes: 100\n",
      "-- region: Spp2-69:+ , number of probes: 100\n",
      "-- region: Spp2-7:+ , number of probes: 100\n",
      "-- region: Spp2-70:+ , number of probes: 100\n",
      "-- region: Spp2-71:+ , number of probes: 100\n",
      "-- region: Spp2-72:+ , number of probes: 100\n",
      "-- region: Spp2-73:+ , number of probes: 100\n",
      "-- region: Spp2-74:+ , number of probes: 100\n",
      "-- region: Spp2-75:+ , number of probes: 100\n",
      "-- region: Spp2-76:+ , number of probes: 100\n",
      "-- region: Spp2-77:+ , number of probes: 100\n",
      "-- region: Spp2-78:+ , number of probes: 100\n",
      "-- region: Spp2-79:+ , number of probes: 100\n",
      "-- region: Spp2-8:+ , number of probes: 100\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- region: Spp2-80:+ , number of probes: 83\n",
      "-- region: Spp2-81:+ , number of probes: 100\n",
      "-- region: Spp2-82:+ , number of probes: 100\n",
      "-- region: Spp2-83:+ , number of probes: 100\n",
      "-- region: Spp2-84:+ , number of probes: 100\n",
      "-- region: Spp2-85:+ , number of probes: 100\n",
      "-- region: Spp2-86:+ , number of probes: 100\n",
      "-- region: Spp2-87:+ , number of probes: 100\n",
      "-- region: Spp2-88:+ , number of probes: 100\n",
      "-- region: Spp2-89:+ , number of probes: 100\n",
      "-- region: Spp2-9:+ , number of probes: 100\n",
      "-- region: Spp2-90:+ , number of probes: 88\n",
      "-- region: Spp2-91:+ , number of probes: 100\n",
      "-- region: Spp2-92:+ , number of probes: 44\n",
      "- Number of probes kept: 26147\n",
      "- Saving to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Promoter_enhancer_pairs\\.\\blast_full_probes.fasta\n"
     ]
    }
   ],
   "source": [
    "# screen\n",
    "num_probes_per_region = 100\n",
    "kept_pbs, blast_keep_dic, hard_count_list, soft_count_list = check.Screening_Probes_by_Blast(library_folder, \n",
    "                                                                                             num_probes_per_region,\n",
    "                                                                                             hard_thres=target_len,\n",
    "                                                                                             soft_count_th=50,\n",
    "                                                                                             )"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "twenty-fellowship",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.9"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
