{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Library design for MOP mm10\n",
    "\n",
    "## Gene Scl17a7 with new probe_designer\n",
    "\n",
    "by Pu Zheng\n",
    "\n",
    "2020.11.01"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## load required packages"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "31340\n"
     ]
    }
   ],
   "source": [
    "%run \"..\\..\\Startup_py3.py\"\n",
    "sys.path.append(r\"..\\..\\..\\..\\Documents\")\n",
    "\n",
    "import ImageAnalysis3 as ia\n",
    "%matplotlib notebook\n",
    "\n",
    "from ImageAnalysis3 import *\n",
    "print(os.getpid())\n",
    "\n",
    "# library design specific tools\n",
    "from ImageAnalysis3.library_tools import LibraryDesigner as ld\n",
    "from ImageAnalysis3.library_tools import LibraryTools as lt\n",
    "# biopython imports\n",
    "from Bio import SeqIO\n",
    "from Bio.Seq import Seq\n",
    "from Bio.SeqRecord import SeqRecord\n",
    "from Bio.Blast.Applications import NcbiblastnCommandline\n",
    "from Bio.Blast import NCBIXML"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# 1 Extract region sequences"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "## Some folders\n",
    "# human genome\n",
    "reference_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl'\n",
    "genome_folder = os.path.join(reference_folder, 'Genome')\n",
    "# Library directories\n",
    "pool_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- library_folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\n",
      "-- sequence_folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\n",
      "-- report_folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\n"
     ]
    }
   ],
   "source": [
    "resolution = 10000\n",
    "flanking = 100000\n",
    "# folder for sub-pool\n",
    "library_folder = os.path.join(pool_folder, f'MOP_marker_genes_{int(resolution/1000)}kb')\n",
    "if not os.path.exists(library_folder):\n",
    "    print(f\"create library folder: {library_folder}\")\n",
    "    os.makedirs(library_folder)\n",
    "# folder for fasta sequences\n",
    "sequence_folder = os.path.join(library_folder, 'sequences')\n",
    "if not os.path.exists(sequence_folder):\n",
    "    print(f\"create sequence folder: {sequence_folder}\")\n",
    "    os.makedirs(sequence_folder)\n",
    "# folder to save result probes\n",
    "report_folder = os.path.join(library_folder, 'reports')\n",
    "if not os.path.exists(report_folder):\n",
    "    print(f\"create report folder: {report_folder}\")\n",
    "    os.makedirs(report_folder)\n",
    "    \n",
    "print(f\"-- library_folder: {library_folder}\")\n",
    "print(f\"-- sequence_folder: {sequence_folder}\")\n",
    "print(f\"-- report_folder: {report_folder}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Input region file is: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\Regions_10kb.txt\n",
      "-- searching among 23 references\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- saving sequences into folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_2.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_3.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_4.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_5.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_6.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_7.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_8.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_9.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_10.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_11.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_12.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_13.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_14.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_15.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_16.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_17.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_18.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_19.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_20.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_21.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_22.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_23.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_24.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_25.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_26.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_27.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_28.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_29.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_30.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_31.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_32.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_33.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_34.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_35.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_36.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_37.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_38.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_39.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_40.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_41.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_42.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_43.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_44.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_45.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_46.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_47.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_48.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_49.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_50.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_51.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_52.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_53.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_54.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_55.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_56.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_57.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_58.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_59.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_60.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc32a1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc32a1_reg_2.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc32a1_reg_3.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc32a1_reg_4.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc32a1_reg_5.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc32a1_reg_6.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc32a1_reg_7.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc32a1_reg_8.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc32a1_reg_9.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc32a1_reg_10.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc32a1_reg_11.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc32a1_reg_12.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc32a1_reg_13.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc32a1_reg_14.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc32a1_reg_15.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc32a1_reg_16.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc32a1_reg_17.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc32a1_reg_18.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc32a1_reg_19.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc32a1_reg_20.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc32a1_reg_21.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_2.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_3.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_4.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_5.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_6.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_7.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_8.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_9.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_10.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_11.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_12.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_13.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_14.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_15.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_16.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_17.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_18.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_19.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_20.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_21.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_22.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_23.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_24.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_25.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_26.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_27.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_28.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_29.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_30.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_31.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_32.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_33.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_34.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_35.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_36.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_37.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_38.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_39.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_40.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_41.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_42.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_43.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_44.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_45.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_46.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_47.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_48.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_49.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_50.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_51.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_52.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_53.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_54.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_55.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_56.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_57.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_58.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_59.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_60.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_1.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_2.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_3.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_4.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_5.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_6.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_7.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_8.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_9.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_10.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_11.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_12.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_13.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_14.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_15.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_16.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_17.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_18.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_19.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_20.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_21.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_22.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_23.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_24.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_25.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_26.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_27.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_28.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_29.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_30.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_31.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_32.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_33.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_34.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_35.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_36.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_37.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_38.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_39.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_40.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_41.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_42.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_43.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_44.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_45.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_46.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_47.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_48.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_49.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_50.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_51.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_52.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_53.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_54.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_55.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_56.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_57.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_58.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_59.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_60.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_2.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_3.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_4.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_5.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_6.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_7.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_8.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_9.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_10.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_11.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_12.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_13.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_14.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_15.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_16.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_17.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_18.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_19.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_20.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_21.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_22.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_23.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_24.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_25.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_26.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_27.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_28.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_29.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_30.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_31.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_32.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_33.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_34.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_35.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_36.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_37.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_38.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_39.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_40.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_41.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_42.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_43.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_44.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_45.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_46.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_47.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_48.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_49.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_50.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_51.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_52.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_53.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_54.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_55.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_56.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_57.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_58.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_59.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_60.fasta\n"
     ]
    }
   ],
   "source": [
    "reload(library_tools.sequences)\n",
    "region_file = os.path.join(library_folder, 'Regions_10kb.txt')\n",
    "region_dict = library_tools.sequences.read_region_file(region_file)\n",
    "records = library_tools.sequences.extract_sequence(region_dict, genome_folder, \n",
    "                                                   resolution=resolution,\n",
    "                                                   flanking=flanking,\n",
    "                                                   save_folder=sequence_folder)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# 2. Design probe targeting sequences by probe_designer\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 2.1 Construct count table with all the 17-mers in the genome\n",
    "\n",
    "Only do this if you don't have pre-built 17-mer\n",
    "\n",
    "However you can do almost the same thing for your own library during quality check\n",
    "\n",
    "This library requires mm10 genome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "overwrite_table = False"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### construct map for whole genome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "reload(library_tools.design)\n",
    " \n",
    "genome_table_file = os.path.join(reference_folder, 'GRCm38_genome_17w.npy')\n",
    "\n",
    "if not os.path.exists(genome_table_file) or overwrite_table:\n",
    "    # genome\n",
    "    _genome_filenames = [os.path.join(genome_folder, _fl) \n",
    "         for _fl in os.listdir(genome_folder) \n",
    "         if _fl.split(os.extsep)[-1]=='fasta' or _fl.split(os.extsep)[-1]=='fa']\n",
    "    print(len(_genome_filenames))\n",
    "\n",
    "    ct = library_tools.design.countTable(word=17,save_file=genome_table_file, \n",
    "                       sparse=False)\n",
    "    ct.verbose=True\n",
    "\n",
    "    ct.read(_genome_filenames) # read sequences from fasta files\n",
    "\n",
    "    ct.consume_loaded(num_threads=24) # convert sequences into integers\n",
    "\n",
    "    ct.complete(verbose=True)\n",
    "\n",
    "    ct.save()\n",
    "\n",
    "    # clear RAM if contructed countable \n",
    "    del(ct)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### construct map for transcriptome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "from tqdm import tqdm\n",
    "# transcriptome\n",
    "transcriptome_folder = os.path.join(reference_folder, 'Transcriptome')\n",
    "\n",
    "transcriptome_table_file = os.path.join(reference_folder, 'GRCm38_transcriptome_17w.npy')\n",
    "\n",
    "if not os.path.exists(transcriptome_table_file) or overwrite_table:\n",
    "    # transcriptome\n",
    "    _transcriptome_filenames = [os.path.join(transcriptome_folder, _fl) \n",
    "         for _fl in os.listdir(transcriptome_folder) \n",
    "         if _fl.split(os.extsep)[-1]=='fasta' or _fl.split(os.extsep)[-1]=='fa']\n",
    "    print(len(_transcriptome_filenames))\n",
    "    \n",
    "    ct = library_tools.design.countTable(word=17,save_file=transcriptome_table_file, \n",
    "                       sparse=False)\n",
    "    ct.verbose=True\n",
    "\n",
    "    ct.read(_transcriptome_filenames) # read sequences from fasta files\n",
    "\n",
    "    ct.consume_loaded(num_threads=24) # convert sequences into integers\n",
    "\n",
    "    ct.complete(verbose=True)\n",
    "\n",
    "    ct.save()\n",
    "    \n",
    "    # clear RAM if contructed countable \n",
    "    del(ct)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### construct map for repeats from RepBase"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "from tqdm import tqdm\n",
    "# repeat\n",
    "repeat_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\Repbase'\n",
    "\n",
    "repeat_table_file = os.path.join(reference_folder, 'Repbase_v2603_repeat_17w.npy')\n",
    "\n",
    "if not os.path.exists(repeat_table_file) or overwrite_table:\n",
    "    # repeat\n",
    "    _repeat_filenames = [os.path.join(repeat_folder, _fl) \n",
    "         for _fl in os.listdir(repeat_folder) \n",
    "         if _fl.split(os.extsep)[-1]=='fasta' or _fl.split(os.extsep)[-1]=='fa']\n",
    "    print(len(_repeat_filenames))\n",
    "    \n",
    "    ct = library_tools.design.countTable(word=17,save_file=repeat_table_file, \n",
    "                       sparse=False)\n",
    "    ct.verbose=True\n",
    "\n",
    "    ct.read(_repeat_filenames) # read sequences from fasta files\n",
    "\n",
    "    ct.consume_loaded(num_threads=24) # convert sequences into integers\n",
    "\n",
    "    ct.complete(verbose=True)\n",
    "\n",
    "    ct.save()\n",
    "    \n",
    "    # clear RAM if contructed countable \n",
    "    del(ct)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### construct map for meng's MOP MERFISH library"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "skipped because not necessary"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 2.2 Design probes"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### parameters"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "resolution = 10000\n",
    "\n",
    "## required folders\n",
    "# Library directories\n",
    "pool_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain'\n",
    "\n",
    "library_folder = os.path.join(pool_folder, f'MOP_marker_genes_{int(resolution/1000)}kb')\n",
    "sequence_folder = os.path.join(library_folder, 'sequences')\n",
    "report_folder = os.path.join(library_folder, 'reports')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "261 regions loaded to design probes.\n",
      "\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n"
     ]
    }
   ],
   "source": [
    "# requires pre_defined genome_folder and library_folder\n",
    "# Indices\n",
    "genome_index = os.path.join(reference_folder, 'GRCm38_genome_17w.npy')\n",
    "transcriptome_index = os.path.join(reference_folder, 'GRCm38_transcriptome_17w.npy') \n",
    "repeat_index = os.path.join(reference_folder, 'Repbase_v2603_repeat_17w.npy')\n",
    "#ref_merfish_index = os.path.join(reference_folder, 'M1_meng_MERFISH_17w.npy') # merfish designed by Meng\n",
    "# get input files \n",
    "input_files = glob.glob(os.path.join(sequence_folder, '*.fasta'))\n",
    "\n",
    "print(f\"{len(input_files)} regions loaded to design probes.\")\n",
    "\n",
    "if not os.path.exists(report_folder):\n",
    "    os.makedirs(report_folder)\n",
    "    \n",
    "# filename to save probe reports\n",
    "save_file = os.path.join(report_folder, f'merged_probes_{int(resolution/1000)}kb.pbr')\n",
    "print(save_file)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### create pb_designer class"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Probe designer derived from Bogdan Bintu:\n",
      "https://github.com/BogdanBintu/ChromatinImaging/blob/master/LibraryDesign/LibraryDesigner.py\n",
      "by Pu Zheng, 2020.11\n",
      "\n",
      "Major changes:\n",
      "    1. allow design of two strands\n",
      "    2. separate reverse_compliment (rev_com) and from two strands (two_stranded) as \n",
      "    two different inputs for map_dic and sequence_dic\n",
      "    3. replace 'local_genome' with 'self_sequences' to be more explicit, and only \n",
      "    exclude the counts for the corresponding self_sequence within each input. \n",
      "\n",
      "Key information:\n",
      "    - number of input_sequence(s): 261\n",
      "    - save_file location: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "\n",
      "- Fail to load from savefile: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr, file doesn't exist.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "False"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "reload(library_tools)\n",
    "reload(library_tools.design)\n",
    "\n",
    "pb_designer = library_tools.design.pb_reports_class(\n",
    "    sequence_dic={'file':input_files,\n",
    "                  'rev_com':True, # design two strands\n",
    "                  'two_stranded':True},\n",
    "    map_dic={'genome':{'file':genome_index,'rev_com':False,'two_stranded':True},\n",
    "             'transcriptome':{'file':transcriptome_index,'rev_com':True,'two_stranded':False},\n",
    "             'rep_genome':{'file':repeat_index,'rev_com':False,'two_stranded':True},\n",
    "             'self_sequences':{'file':input_files,'force_list':True,'rev_com':False,'two_stranded':True},\n",
    "             #'ref_merfish':{'file':ref_merfish_index,'rev_com':False,'two_stranded':True},\n",
    "             },\n",
    "        save_file=save_file,\n",
    "    params_dic={'word_size':17,'pb_len':42,'buffer_len':1,'max_count':2**16-1,\n",
    "                #'rev_com_score_ratio': 1.0, \n",
    "                'check_on_go': False, # whether automatically check probes\n",
    "                'auto': False, # whether automatically convert reference maps\n",
    "               },\n",
    "    check_dic={('genome','self_sequences'): 25,\n",
    "               'rep_genome': 0,\n",
    "               'transcriptome': 14,\n",
    "               #'ref_merfish': 14,\n",
    "               'gc':[0.25,0.85],'tm': 70,\n",
    "               }\n",
    "    )\n",
    "print(pb_designer)\n",
    "pb_designer.load_from_file(load_probes_only=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### calculate probe reports"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- setting attribute: map_genome\n",
      "--- finish map_genome in 88.690s.\n",
      "-- setting attribute: map_transcriptome\n",
      "--- finish map_transcriptome in 91.061s.\n",
      "-- setting attribute: map_rep_genome\n",
      "--- finish map_rep_genome in 83.359s.\n",
      "Time(s): 263.11810874938965\n",
      "- Designing targeting sequence for 261 regions\n",
      "-- designing region: 6:126980000-126990000_reg_Ccnd2-1 0, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.118s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 5.840s.\n",
      "-- designing region: 6:127070000-127080000_reg_Ccnd2-10 1, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_10.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.116s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 6.691s.\n",
      "-- designing region: 6:127080000-127090000_reg_Ccnd2-11 2, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_11.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.109s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 7.036s.\n",
      "-- designing region: 6:127090000-127100000_reg_Ccnd2-12 3, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_12.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.154s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 7.571s.\n",
      "-- designing region: 6:127100000-127110000_reg_Ccnd2-13 4, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_13.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.085s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 7.894s.\n",
      "-- designing region: 6:127110000-127120000_reg_Ccnd2-14 5, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_14.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.085s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 8.294s.\n",
      "-- designing region: 6:127120000-127130000_reg_Ccnd2-15 6, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_15.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.085s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 8.703s.\n",
      "-- designing region: 6:127130000-127140000_reg_Ccnd2-16 7, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_16.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.081s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 9.242s.\n",
      "-- designing region: 6:127140000-127150000_reg_Ccnd2-17 8, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_17.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.083s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 9.733s.\n",
      "-- designing region: 6:127150000-127160000_reg_Ccnd2-18 9, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_18.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.091s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 10.116s.\n",
      "-- designing region: 6:127160000-127170000_reg_Ccnd2-19 10, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_19.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.091s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 10.503s.\n",
      "-- designing region: 6:126990000-127000000_reg_Ccnd2-2 11, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_2.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.101s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 10.988s.\n",
      "-- designing region: 6:127170000-127180000_reg_Ccnd2-20 12, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_20.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.101s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 11.915s.\n",
      "-- designing region: 6:127180000-127190000_reg_Ccnd2-21 13, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_21.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.103s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 12.264s.\n",
      "-- designing region: 6:127190000-127200000_reg_Ccnd2-22 14, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_22.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 12.510s.\n",
      "-- designing region: 6:127200000-127210000_reg_Ccnd2-23 15, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_23.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.118s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 12.850s.\n",
      "-- designing region: 6:127210000-127220000_reg_Ccnd2-24 16, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_24.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.127s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 13.269s.\n",
      "-- designing region: 6:127220000-127230000_reg_Ccnd2-25 17, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_25.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.132s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 13.634s.\n",
      "-- designing region: 6:127230000-127240000_reg_Ccnd2-26 18, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_26.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.101s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 14.016s.\n",
      "-- designing region: 6:127240000-127250000_reg_Ccnd2-27 19, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_27.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.101s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 17.380s.\n",
      "-- designing region: 6:127250000-127260000_reg_Ccnd2-28 20, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_28.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.119s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 14.880s.\n",
      "-- designing region: 6:127260000-127270000_reg_Ccnd2-29 21, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_29.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.089s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 15.182s.\n",
      "-- designing region: 6:127000000-127010000_reg_Ccnd2-3 22, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_3.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.106s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 15.606s.\n",
      "-- designing region: 6:127270000-127280000_reg_Ccnd2-30 23, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_30.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.116s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 15.991s.\n",
      "-- designing region: 6:127280000-127290000_reg_Ccnd2-31 24, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_31.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.116s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 16.385s.\n",
      "-- designing region: 6:127290000-127300000_reg_Ccnd2-32 25, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_32.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.128s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 16.927s.\n",
      "-- designing region: 6:127300000-127310000_reg_Ccnd2-33 26, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_33.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.150s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 17.752s.\n",
      "-- designing region: 6:127310000-127320000_reg_Ccnd2-34 27, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_34.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.106s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 17.549s.\n",
      "-- designing region: 6:127320000-127330000_reg_Ccnd2-35 28, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_35.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.106s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 18.003s.\n",
      "-- designing region: 6:127330000-127340000_reg_Ccnd2-36 29, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_36.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.106s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 19.108s.\n",
      "-- designing region: 6:127340000-127350000_reg_Ccnd2-37 30, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_37.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.091s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 19.584s.\n",
      "-- designing region: 6:127350000-127360000_reg_Ccnd2-38 31, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_38.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.099s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 20.045s.\n",
      "-- designing region: 6:127360000-127370000_reg_Ccnd2-39 32, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_39.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.154s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 20.373s.\n",
      "-- designing region: 6:127010000-127020000_reg_Ccnd2-4 33, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_4.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.111s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 20.736s.\n",
      "-- designing region: 6:127370000-127380000_reg_Ccnd2-40 34, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_40.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.148s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 20.890s.\n",
      "-- designing region: 6:127380000-127390000_reg_Ccnd2-41 35, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_41.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 22.019s.\n",
      "-- designing region: 6:127390000-127400000_reg_Ccnd2-42 36, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_42.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.113s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 21.826s.\n",
      "-- designing region: 6:127400000-127410000_reg_Ccnd2-43 37, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_43.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.097s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "in 22.275s.\n",
      "-- designing region: 6:127410000-127420000_reg_Ccnd2-44 38, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_44.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.123s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 22.616s.\n",
      "-- designing region: 6:127420000-127430000_reg_Ccnd2-45 39, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_45.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.123s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 23.144s.\n",
      "-- designing region: 6:127430000-127440000_reg_Ccnd2-46 40, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_46.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.243s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 23.458s.\n",
      "-- designing region: 6:127440000-127450000_reg_Ccnd2-47 41, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_47.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.121s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 23.809s.\n",
      "-- designing region: 6:127450000-127460000_reg_Ccnd2-48 42, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_48.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.123s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 24.341s.\n",
      "-- designing region: 6:127460000-127470000_reg_Ccnd2-49 43, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_49.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.100s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 24.622s.\n",
      "-- designing region: 6:127020000-127030000_reg_Ccnd2-5 44, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_5.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.137s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 25.336s.\n",
      "-- designing region: 6:127470000-127480000_reg_Ccnd2-50 45, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_50.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.111s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 26.043s.\n",
      "-- designing region: 6:127480000-127490000_reg_Ccnd2-51 46, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_51.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.251s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 26.654s.\n",
      "-- designing region: 6:127490000-127500000_reg_Ccnd2-52 47, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_52.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.127s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 27.133s.\n",
      "-- designing region: 6:127500000-127510000_reg_Ccnd2-53 48, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_53.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.113s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 27.618s.\n",
      "-- designing region: 6:127510000-127520000_reg_Ccnd2-54 49, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_54.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.091s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 28.015s.\n",
      "-- designing region: 6:127520000-127530000_reg_Ccnd2-55 50, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_55.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.139s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 28.586s.\n",
      "-- designing region: 6:127530000-127540000_reg_Ccnd2-56 51, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_56.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.149s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 28.992s.\n",
      "-- designing region: 6:127540000-127550000_reg_Ccnd2-57 52, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_57.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.117s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 29.576s.\n",
      "-- designing region: 6:127550000-127560000_reg_Ccnd2-58 53, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_58.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.101s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 30.001s.\n",
      "-- designing region: 6:127560000-127570000_reg_Ccnd2-59 54, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_59.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.111s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 30.399s.\n",
      "-- designing region: 6:127030000-127040000_reg_Ccnd2-6 55, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_6.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.179s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 31.135s.\n",
      "-- designing region: 6:127570000-127580000_reg_Ccnd2-60 56, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_60.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.154s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 32.000s.\n",
      "-- designing region: 6:127040000-127050000_reg_Ccnd2-7 57, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_7.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.218s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 31.933s.\n",
      "-- designing region: 6:127050000-127060000_reg_Ccnd2-8 58, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_8.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.101s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 33.187s.\n",
      "-- designing region: 6:127060000-127070000_reg_Ccnd2-9 59, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Ccnd2_reg_9.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.093s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 34.191s.\n",
      "-- designing region: 12:56350000-56360000_reg_Nkx2-1-1 60, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.210s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 34.758s.\n",
      "-- designing region: 12:56440000-56450000_reg_Nkx2-1-10 61, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_10.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.216s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 35.108s.\n",
      "-- designing region: 12:56450000-56460000_reg_Nkx2-1-11 62, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_11.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.152s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 35.418s.\n",
      "-- designing region: 12:56460000-56470000_reg_Nkx2-1-12 63, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_12.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.162s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 35.964s.\n",
      "-- designing region: 12:56470000-56480000_reg_Nkx2-1-13 64, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_13.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 36.405s.\n",
      "-- designing region: 12:56480000-56490000_reg_Nkx2-1-14 65, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_14.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 36.852s.\n",
      "-- designing region: 12:56490000-56500000_reg_Nkx2-1-15 66, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_15.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.113s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 37.251s.\n",
      "-- designing region: 12:56500000-56510000_reg_Nkx2-1-16 67, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_16.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.142s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 37.764s.\n",
      "-- designing region: 12:56510000-56520000_reg_Nkx2-1-17 68, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_17.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.132s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 38.152s.\n",
      "-- designing region: 12:56520000-56530000_reg_Nkx2-1-18 69, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_18.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.101s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 38.550s.\n",
      "-- designing region: 12:56530000-56540000_reg_Nkx2-1-19 70, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_19.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.141s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 39.697s.\n",
      "-- designing region: 12:56360000-56370000_reg_Nkx2-1-2 71, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_2.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.142s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 39.862s.\n",
      "-- designing region: 12:56540000-56550000_reg_Nkx2-1-20 72, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_20.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.142s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 40.414s.\n",
      "-- designing region: 12:56550000-56560000_reg_Nkx2-1-21 73, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_21.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.152s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 41.774s.\n",
      "-- designing region: 12:56560000-56570000_reg_Nkx2-1-22 74, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_22.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 41.119s.\n",
      "-- designing region: 12:56570000-56580000_reg_Nkx2-1-23 75, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_23.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.162s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 41.589s.\n",
      "-- designing region: 12:56580000-56590000_reg_Nkx2-1-24 76, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_24.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.130s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 42.779s.\n",
      "-- designing region: 12:56590000-56600000_reg_Nkx2-1-25 77, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_25.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.111s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 42.796s.\n",
      "-- designing region: 12:56600000-56610000_reg_Nkx2-1-26 78, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_26.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.113s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 43.242s.\n",
      "-- designing region: 12:56610000-56620000_reg_Nkx2-1-27 79, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_27.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.160s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 43.610s.\n",
      "-- designing region: 12:56620000-56630000_reg_Nkx2-1-28 80, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_28.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.164s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 44.508s.\n",
      "-- designing region: 12:56630000-56640000_reg_Nkx2-1-29 81, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_29.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.162s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 44.711s.\n",
      "-- designing region: 12:56370000-56380000_reg_Nkx2-1-3 82, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_3.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.152s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 45.290s.\n",
      "-- designing region: 12:56640000-56650000_reg_Nkx2-1-30 83, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_30.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 45.364s.\n",
      "-- designing region: 12:56650000-56660000_reg_Nkx2-1-31 84, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_31.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.101s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 46.104s.\n",
      "-- designing region: 12:56660000-56670000_reg_Nkx2-1-32 85, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_32.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.119s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 46.575s.\n",
      "-- designing region: 12:56670000-56680000_reg_Nkx2-1-33 86, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_33.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.144s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 47.131s.\n",
      "-- designing region: 12:56680000-56690000_reg_Nkx2-1-34 87, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_34.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.200s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 47.421s.\n",
      "-- designing region: 12:56690000-56700000_reg_Nkx2-1-35 88, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_35.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.151s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 48.019s.\n",
      "-- designing region: 12:56700000-56710000_reg_Nkx2-1-36 89, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_36.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.121s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 48.296s.\n",
      "-- designing region: 12:56710000-56720000_reg_Nkx2-1-37 90, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_37.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.121s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 50.287s.\n",
      "-- designing region: 12:56720000-56730000_reg_Nkx2-1-38 91, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_38.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.111s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 49.490s.\n",
      "-- designing region: 12:56730000-56740000_reg_Nkx2-1-39 92, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_39.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.207s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 50.007s.\n",
      "-- designing region: 12:56380000-56390000_reg_Nkx2-1-4 93, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_4.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.205s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 50.639s.\n",
      "-- designing region: 12:56740000-56750000_reg_Nkx2-1-40 94, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_40.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.142s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 50.937s.\n",
      "-- designing region: 12:56750000-56760000_reg_Nkx2-1-41 95, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_41.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.111s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 51.685s.\n",
      "-- designing region: 12:56760000-56770000_reg_Nkx2-1-42 96, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_42.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.152s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 52.254s.\n",
      "-- designing region: 12:56770000-56780000_reg_Nkx2-1-43 97, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_43.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.121s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 52.645s.\n",
      "-- designing region: 12:56780000-56790000_reg_Nkx2-1-44 98, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_44.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 113.328s.\n",
      "-- designing region: 12:56790000-56800000_reg_Nkx2-1-45 99, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_45.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.157s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 54.039s.\n",
      "-- designing region: 12:56800000-56810000_reg_Nkx2-1-46 100, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_46.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.140s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 54.396s.\n",
      "-- designing region: 12:56810000-56820000_reg_Nkx2-1-47 101, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_47.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.101s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 54.584s.\n",
      "-- designing region: 12:56820000-56830000_reg_Nkx2-1-48 102, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_48.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.111s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 55.175s.\n",
      "-- designing region: 12:56830000-56840000_reg_Nkx2-1-49 103, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_49.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.173s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 55.832s.\n",
      "-- designing region: 12:56390000-56400000_reg_Nkx2-1-5 104, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_5.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.161s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 56.442s.\n",
      "-- designing region: 12:56840000-56850000_reg_Nkx2-1-50 105, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_50.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.147s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 57.363s.\n",
      "-- designing region: 12:56850000-56860000_reg_Nkx2-1-51 106, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_51.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 58.089s.\n",
      "-- designing region: 12:56860000-56870000_reg_Nkx2-1-52 107, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_52.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.141s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 58.718s.\n",
      "-- designing region: 12:56870000-56880000_reg_Nkx2-1-53 108, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_53.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.101s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 58.946s.\n",
      "-- designing region: 12:56880000-56890000_reg_Nkx2-1-54 109, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_54.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.147s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 59.580s.\n",
      "-- designing region: 12:56890000-56900000_reg_Nkx2-1-55 110, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_55.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.121s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 61.343s.\n",
      "-- designing region: 12:56900000-56910000_reg_Nkx2-1-56 111, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_56.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.109s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 60.945s.\n",
      "-- designing region: 12:56910000-56920000_reg_Nkx2-1-57 112, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_57.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.127s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 60.929s.\n",
      "-- designing region: 12:56920000-56930000_reg_Nkx2-1-58 113, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_58.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.127s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 61.680s.\n",
      "-- designing region: 12:56930000-56940000_reg_Nkx2-1-59 114, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_59.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.121s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 62.101s.\n",
      "-- designing region: 12:56400000-56410000_reg_Nkx2-1-6 115, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_6.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.160s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 63.001s.\n",
      "-- designing region: 12:56940000-56950000_reg_Nkx2-1-60 116, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_60.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.169s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 63.267s.\n",
      "-- designing region: 12:56410000-56420000_reg_Nkx2-1-7 117, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_7.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.121s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 63.878s.\n",
      "-- designing region: 12:56420000-56430000_reg_Nkx2-1-8 118, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_8.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.101s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 64.254s.\n",
      "-- designing region: 12:56430000-56440000_reg_Nkx2-1-9 119, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Nkx2-1_reg_9.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.125s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 67.162s.\n",
      "-- designing region: 7:44740000-44750000_reg_Slc17a7-1 120, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 67.875s.\n",
      "-- designing region: 7:44830000-44840000_reg_Slc17a7-10 121, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_10.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.111s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 68.569s.\n",
      "-- designing region: 7:44840000-44850000_reg_Slc17a7-11 122, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_11.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.111s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 68.489s.\n",
      "-- designing region: 7:44850000-44860000_reg_Slc17a7-12 123, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_12.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.112s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 69.225s.\n",
      "-- designing region: 7:44860000-44870000_reg_Slc17a7-13 124, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_13.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.121s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 70.156s.\n",
      "-- designing region: 7:44870000-44880000_reg_Slc17a7-14 125, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_14.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.223s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 70.746s.\n",
      "-- designing region: 7:44880000-44890000_reg_Slc17a7-15 126, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_15.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.124s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 71.455s.\n",
      "-- designing region: 7:44890000-44900000_reg_Slc17a7-16 127, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_16.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.111s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 71.819s.\n",
      "-- designing region: 7:44900000-44910000_reg_Slc17a7-17 128, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_17.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.130s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 70.344s.\n",
      "-- designing region: 7:44910000-44920000_reg_Slc17a7-18 129, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_18.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.162s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 72.855s.\n",
      "-- designing region: 7:44920000-44930000_reg_Slc17a7-19 130, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_19.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.160s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 73.117s.\n",
      "-- designing region: 7:44750000-44760000_reg_Slc17a7-2 131, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_2.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.113s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 73.862s.\n",
      "-- designing region: 7:44930000-44940000_reg_Slc17a7-20 132, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_20.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 74.402s.\n",
      "-- designing region: 7:44940000-44950000_reg_Slc17a7-21 133, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_21.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.101s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 74.892s.\n",
      "-- designing region: 7:44950000-44960000_reg_Slc17a7-22 134, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_22.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.152s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 75.860s.\n",
      "-- designing region: 7:44960000-44970000_reg_Slc17a7-23 135, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_23.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.112s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 74.420s.\n",
      "-- designing region: 7:44970000-44980000_reg_Slc17a7-24 136, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_24.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.113s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 76.448s.\n",
      "-- designing region: 7:44980000-44990000_reg_Slc17a7-25 137, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_25.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.111s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 78.324s.\n",
      "-- designing region: 7:44990000-45000000_reg_Slc17a7-26 138, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_26.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.111s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 78.049s.\n",
      "-- designing region: 7:45000000-45010000_reg_Slc17a7-27 139, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_27.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.139s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 78.365s.\n",
      "-- designing region: 7:45010000-45020000_reg_Slc17a7-28 140, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_28.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.152s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 78.839s.\n",
      "-- designing region: 7:45020000-45030000_reg_Slc17a7-29 141, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_29.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.142s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 79.285s.\n",
      "-- designing region: 7:44760000-44770000_reg_Slc17a7-3 142, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_3.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.111s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 79.855s.\n",
      "-- designing region: 7:45030000-45040000_reg_Slc17a7-30 143, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_30.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.121s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 80.447s.\n",
      "-- designing region: 7:45040000-45050000_reg_Slc17a7-31 144, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_31.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.129s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 81.135s.\n",
      "-- designing region: 7:45050000-45060000_reg_Slc17a7-32 145, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_32.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.121s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 81.624s.\n",
      "-- designing region: 7:45060000-45070000_reg_Slc17a7-33 146, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_33.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.132s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 82.334s.\n",
      "-- designing region: 7:45070000-45080000_reg_Slc17a7-34 147, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_34.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.142s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 82.962s.\n",
      "-- designing region: 7:45080000-45090000_reg_Slc17a7-35 148, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_35.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.152s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 83.228s.\n",
      "-- designing region: 7:45090000-45100000_reg_Slc17a7-36 149, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_36.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.111s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 84.397s.\n",
      "-- designing region: 7:45100000-45110000_reg_Slc17a7-37 150, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_37.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.152s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 83.599s.\n",
      "-- designing region: 7:45110000-45120000_reg_Slc17a7-38 151, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_38.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.113s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 85.957s.\n",
      "-- designing region: 7:45120000-45130000_reg_Slc17a7-39 152, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_39.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 84.765s.\n",
      "-- designing region: 7:44770000-44780000_reg_Slc17a7-4 153, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_4.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.172s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 86.188s.\n",
      "-- designing region: 7:45130000-45140000_reg_Slc17a7-40 154, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_40.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.121s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 86.902s.\n",
      "-- designing region: 7:45140000-45150000_reg_Slc17a7-41 155, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_41.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.142s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 87.487s.\n",
      "-- designing region: 7:45150000-45160000_reg_Slc17a7-42 156, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_42.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 88.306s.\n",
      "-- designing region: 7:45160000-45170000_reg_Slc17a7-43 157, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_43.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.152s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 90.558s.\n",
      "-- designing region: 7:45170000-45180000_reg_Slc17a7-44 158, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_44.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.124s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 90.615s.\n",
      "-- designing region: 7:45180000-45190000_reg_Slc17a7-45 159, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_45.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.162s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 90.752s.\n",
      "-- designing region: 7:45190000-45200000_reg_Slc17a7-46 160, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_46.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.109s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 91.804s.\n",
      "-- designing region: 7:45200000-45210000_reg_Slc17a7-47 161, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_47.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.132s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 91.763s.\n",
      "-- designing region: 7:45210000-45220000_reg_Slc17a7-48 162, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_48.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.121s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 92.604s.\n",
      "-- designing region: 7:45220000-45230000_reg_Slc17a7-49 163, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_49.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.121s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 92.948s.\n",
      "-- designing region: 7:44780000-44790000_reg_Slc17a7-5 164, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_5.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.152s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 94.071s.\n",
      "-- designing region: 7:45230000-45240000_reg_Slc17a7-50 165, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_50.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.162s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 96.042s.\n",
      "-- designing region: 7:45240000-45250000_reg_Slc17a7-51 166, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_51.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.142s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 94.868s.\n",
      "-- designing region: 7:45250000-45260000_reg_Slc17a7-52 167, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_52.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.142s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 95.329s.\n",
      "-- designing region: 7:45260000-45270000_reg_Slc17a7-53 168, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_53.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.121s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 96.310s.\n",
      "-- designing region: 7:45270000-45280000_reg_Slc17a7-54 169, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_54.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.144s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 97.838s.\n",
      "-- designing region: 7:45280000-45290000_reg_Slc17a7-55 170, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_55.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 97.476s.\n",
      "-- designing region: 7:45290000-45300000_reg_Slc17a7-56 171, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_56.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.150s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 97.974s.\n",
      "-- designing region: 7:45300000-45310000_reg_Slc17a7-57 172, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_57.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.132s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 98.120s.\n",
      "-- designing region: 7:45310000-45320000_reg_Slc17a7-58 173, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_58.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.121s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 99.070s.\n",
      "-- designing region: 7:45320000-45330000_reg_Slc17a7-59 174, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_59.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.144s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 99.429s.\n",
      "-- designing region: 7:44790000-44800000_reg_Slc17a7-6 175, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_6.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.152s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 100.821s.\n",
      "-- designing region: 7:45330000-45340000_reg_Slc17a7-60 176, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_60.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.152s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 100.883s.\n",
      "-- designing region: 7:44800000-44810000_reg_Slc17a7-7 177, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_7.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.142s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 102.258s.\n",
      "-- designing region: 7:44810000-44820000_reg_Slc17a7-8 178, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_8.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.152s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 102.358s.\n",
      "-- designing region: 7:44820000-44830000_reg_Slc17a7-9 179, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc17a7_reg_9.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.111s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 103.093s.\n",
      "-- designing region: 2:158510767-158520767_reg_Slc32a1-1 180, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc32a1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.152s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 103.515s.\n",
      "-- designing region: 2:158600767-158610767_reg_Slc32a1-10 181, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc32a1_reg_10.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.119s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 104.193s.\n",
      "-- designing region: 2:158610767-158620767_reg_Slc32a1-11 182, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc32a1_reg_11.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 104.529s.\n",
      "-- designing region: 2:158620767-158630767_reg_Slc32a1-12 183, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc32a1_reg_12.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.223s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 105.589s.\n",
      "-- designing region: 2:158630767-158640767_reg_Slc32a1-13 184, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc32a1_reg_13.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.093s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 105.715s.\n",
      "-- designing region: 2:158640767-158650767_reg_Slc32a1-14 185, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc32a1_reg_14.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.132s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 106.488s.\n",
      "-- designing region: 2:158650767-158660767_reg_Slc32a1-15 186, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc32a1_reg_15.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.111s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 109.203s.\n",
      "-- designing region: 2:158660767-158670767_reg_Slc32a1-16 187, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc32a1_reg_16.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.111s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 108.042s.\n",
      "-- designing region: 2:158670767-158680767_reg_Slc32a1-17 188, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc32a1_reg_17.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.121s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 108.562s.\n",
      "-- designing region: 2:158680767-158690767_reg_Slc32a1-18 189, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc32a1_reg_18.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.212s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 110.102s.\n",
      "-- designing region: 2:158690767-158700767_reg_Slc32a1-19 190, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc32a1_reg_19.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.132s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 109.395s.\n",
      "-- designing region: 2:158520767-158530767_reg_Slc32a1-2 191, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc32a1_reg_2.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.162s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 110.291s.\n",
      "-- designing region: 2:158700767-158710767_reg_Slc32a1-20 192, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc32a1_reg_20.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.211s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 111.530s.\n",
      "-- designing region: 2:158710767-158720767_reg_Slc32a1-21 193, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc32a1_reg_21.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.151s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 111.546s.\n",
      "-- designing region: 2:158530767-158540767_reg_Slc32a1-3 194, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc32a1_reg_3.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.142s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 112.371s.\n",
      "-- designing region: 2:158540767-158550767_reg_Slc32a1-4 195, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc32a1_reg_4.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.132s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 113.054s.\n",
      "-- designing region: 2:158550767-158560767_reg_Slc32a1-5 196, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc32a1_reg_5.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.121s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 114.015s.\n",
      "-- designing region: 2:158560767-158570767_reg_Slc32a1-6 197, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc32a1_reg_6.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.150s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 114.237s.\n",
      "-- designing region: 2:158570767-158580767_reg_Slc32a1-7 198, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc32a1_reg_7.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.121s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 116.603s.\n",
      "-- designing region: 2:158580767-158590767_reg_Slc32a1-8 199, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc32a1_reg_8.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.103s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 115.003s.\n",
      "-- designing region: 2:158590767-158600767_reg_Slc32a1-9 200, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\Slc32a1_reg_9.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.152s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 115.870s.\n",
      "-- designing region: 16:23730000-23740000_reg_SST-1 201, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.150s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 116.404s.\n",
      "-- designing region: 16:23820000-23830000_reg_SST-10 202, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_10.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.142s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 117.651s.\n",
      "-- designing region: 16:23830000-23840000_reg_SST-11 203, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_11.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.142s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 117.922s.\n",
      "-- designing region: 16:23840000-23850000_reg_SST-12 204, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_12.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.121s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 118.464s.\n",
      "-- designing region: 16:23850000-23860000_reg_SST-13 205, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_13.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 120.440s.\n",
      "-- designing region: 16:23860000-23870000_reg_SST-14 206, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_14.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.111s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 119.465s.\n",
      "-- designing region: 16:23870000-23880000_reg_SST-15 207, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_15.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.140s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 120.515s.\n",
      "-- designing region: 16:23880000-23890000_reg_SST-16 208, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_16.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.172s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 120.712s.\n",
      "-- designing region: 16:23890000-23900000_reg_SST-17 209, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_17.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.111s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 121.858s.\n",
      "-- designing region: 16:23900000-23910000_reg_SST-18 210, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_18.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.101s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 123.065s.\n",
      "-- designing region: 16:23910000-23920000_reg_SST-19 211, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_19.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.152s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 123.410s.\n",
      "-- designing region: 16:23740000-23750000_reg_SST-2 212, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_2.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.223s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 123.250s.\n",
      "-- designing region: 16:23920000-23930000_reg_SST-20 213, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_20.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 123.841s.\n",
      "-- designing region: 16:23930000-23940000_reg_SST-21 214, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_21.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 126.127s.\n",
      "-- designing region: 16:23940000-23950000_reg_SST-22 215, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_22.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.142s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 126.432s.\n",
      "-- designing region: 16:23950000-23960000_reg_SST-23 216, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_23.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.150s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 126.436s.\n",
      "-- designing region: 16:23960000-23970000_reg_SST-24 217, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_24.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.103s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 127.313s.\n",
      "-- designing region: 16:23970000-23980000_reg_SST-25 218, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_25.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.364s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 128.631s.\n",
      "-- designing region: 16:23980000-23990000_reg_SST-26 219, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_26.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 131.424s.\n",
      "-- designing region: 16:23990000-24000000_reg_SST-27 220, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_27.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.154s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 130.900s.\n",
      "-- designing region: 16:24000000-24010000_reg_SST-28 221, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_28.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.428s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 129.617s.\n",
      "-- designing region: 16:24010000-24020000_reg_SST-29 222, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_29.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.111s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 130.712s.\n",
      "-- designing region: 16:23750000-23760000_reg_SST-3 223, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_3.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 131.567s.\n",
      "-- designing region: 16:24020000-24030000_reg_SST-30 224, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_30.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.111s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 131.278s.\n",
      "-- designing region: 16:24030000-24040000_reg_SST-31 225, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_31.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.170s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 132.468s.\n",
      "-- designing region: 16:24040000-24050000_reg_SST-32 226, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_32.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.140s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 133.369s.\n",
      "-- designing region: 16:24050000-24060000_reg_SST-33 227, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_33.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.111s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 133.600s.\n",
      "-- designing region: 16:24060000-24070000_reg_SST-34 228, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_34.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.152s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "in 134.653s.\n",
      "-- designing region: 16:24070000-24080000_reg_SST-35 229, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_35.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.144s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 135.220s.\n",
      "-- designing region: 16:24080000-24090000_reg_SST-36 230, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_36.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.101s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 135.870s.\n",
      "-- designing region: 16:24090000-24100000_reg_SST-37 231, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_37.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.113s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 135.615s.\n",
      "-- designing region: 16:24100000-24110000_reg_SST-38 232, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_38.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.122s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 137.241s.\n",
      "-- designing region: 16:24110000-24120000_reg_SST-39 233, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_39.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.111s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 137.977s.\n",
      "-- designing region: 16:23760000-23770000_reg_SST-4 234, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_4.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.164s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 137.963s.\n",
      "-- designing region: 16:24120000-24130000_reg_SST-40 235, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_40.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.101s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 138.783s.\n",
      "-- designing region: 16:24130000-24140000_reg_SST-41 236, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_41.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.111s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 140.372s.\n",
      "-- designing region: 16:24140000-24150000_reg_SST-42 237, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_42.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.121s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 140.084s.\n",
      "-- designing region: 16:24150000-24160000_reg_SST-43 238, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_43.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.122s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 141.311s.\n",
      "-- designing region: 16:24160000-24170000_reg_SST-44 239, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_44.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.152s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 148.805s.\n",
      "-- designing region: 16:24170000-24180000_reg_SST-45 240, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_45.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.103s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 148.828s.\n",
      "-- designing region: 16:24180000-24190000_reg_SST-46 241, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_46.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.101s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 150.080s.\n",
      "-- designing region: 16:24190000-24200000_reg_SST-47 242, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_47.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.111s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 150.212s.\n",
      "-- designing region: 16:24200000-24210000_reg_SST-48 243, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_48.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 152.375s.\n",
      "-- designing region: 16:24210000-24220000_reg_SST-49 244, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_49.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.162s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 151.656s.\n",
      "-- designing region: 16:23770000-23780000_reg_SST-5 245, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_5.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.202s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 153.052s.\n",
      "-- designing region: 16:24220000-24230000_reg_SST-50 246, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_50.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.121s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 154.402s.\n",
      "-- designing region: 16:24230000-24240000_reg_SST-51 247, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_51.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.142s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 155.374s.\n",
      "-- designing region: 16:24240000-24250000_reg_SST-52 248, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_52.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.154s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 154.595s.\n",
      "-- designing region: 16:24250000-24260000_reg_SST-53 249, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_53.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.152s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 155.997s.\n",
      "-- designing region: 16:24260000-24270000_reg_SST-54 250, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_54.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.164s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 155.167s.\n",
      "-- designing region: 16:24270000-24280000_reg_SST-55 251, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_55.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.121s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 157.087s.\n",
      "-- designing region: 16:24280000-24290000_reg_SST-56 252, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_56.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.140s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 157.192s.\n",
      "-- designing region: 16:24290000-24300000_reg_SST-57 253, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_57.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.140s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 157.411s.\n",
      "-- designing region: 16:24300000-24310000_reg_SST-58 254, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_58.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.170s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 158.426s.\n",
      "-- designing region: 16:24310000-24320000_reg_SST-59 255, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_59.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.132s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 158.964s.\n",
      "-- designing region: 16:23780000-23790000_reg_SST-6 256, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_6.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.211s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 159.592s.\n",
      "-- designing region: 16:24320000-24330000_reg_SST-60 257, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_60.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.154s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 163.365s.\n",
      "-- designing region: 16:23790000-23800000_reg_SST-7 258, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_7.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.154s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 161.579s.\n",
      "-- designing region: 16:23800000-23810000_reg_SST-8 259, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_8.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.150s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 162.774s.\n",
      "-- designing region: 16:23810000-23820000_reg_SST-9 260, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\sequences\\SST_reg_9.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.123s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "in 163.087s.\n"
     ]
    }
   ],
   "source": [
    "reload(library_tools)\n",
    "reload(library_tools.design)\n",
    "pb_designer.computeOTmaps() # load the tables \n",
    "pb_designer.compute_pb_report() # design candidate probes"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### check probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:0 6:126980000-126990000_reg_Ccnd2-1, 19778 candidate probes\n",
      "--- 9363 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.902s, 135 probes kept.\n",
      "-- check region:1 6:127070000-127080000_reg_Ccnd2-10, 19732 candidate probes\n",
      "--- 10376 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.848s, 136 probes kept.\n",
      "-- check region:2 6:127080000-127090000_reg_Ccnd2-11, 19840 candidate probes\n",
      "--- 8162 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.810s, 139 probes kept.\n",
      "-- check region:3 6:127090000-127100000_reg_Ccnd2-12, 19416 candidate probes\n",
      "--- 3661 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.627s, 62 probes kept.\n",
      "-- check region:4 6:127100000-127110000_reg_Ccnd2-13, 19432 candidate probes\n",
      "--- 8445 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.803s, 127 probes kept.\n",
      "-- check region:5 6:127110000-127120000_reg_Ccnd2-14, 19616 candidate probes\n",
      "--- 7566 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.811s, 117 probes kept.\n",
      "-- check region:6 6:127120000-127130000_reg_Ccnd2-15, 19512 candidate probes\n",
      "--- 6915 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.768s, 125 probes kept.\n",
      "-- check region:7 6:127130000-127140000_reg_Ccnd2-16, 19796 candidate probes\n",
      "--- 9872 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.889s, 136 probes kept.\n",
      "-- check region:8 6:127140000-127150000_reg_Ccnd2-17, 19854 candidate probes\n",
      "--- 10372 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.928s, 147 probes kept.\n",
      "-- check region:9 6:127150000-127160000_reg_Ccnd2-18, 19918 candidate probes\n",
      "--- 11253 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.905s, 146 probes kept.\n",
      "-- check region:10 6:127160000-127170000_reg_Ccnd2-19, 19592 candidate probes\n",
      "--- 9793 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.869s, 143 probes kept.\n",
      "-- check region:11 6:126990000-127000000_reg_Ccnd2-2, 19856 candidate probes\n",
      "--- 6382 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.758s, 99 probes kept.\n",
      "-- check region:12 6:127170000-127180000_reg_Ccnd2-20, 19622 candidate probes\n",
      "--- 10091 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.849s, 142 probes kept.\n",
      "-- check region:13 6:127180000-127190000_reg_Ccnd2-21, 19824 candidate probes\n",
      "--- 9387 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.879s, 139 probes kept.\n",
      "-- check region:14 6:127190000-127200000_reg_Ccnd2-22, 19676 candidate probes\n",
      "--- 8498 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.820s, 126 probes kept.\n",
      "-- check region:15 6:127200000-127210000_reg_Ccnd2-23, 19862 candidate probes\n",
      "--- 10450 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.919s, 138 probes kept.\n",
      "-- check region:16 6:127210000-127220000_reg_Ccnd2-24, 19848 candidate probes\n",
      "--- 9326 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.788s, 136 probes kept.\n",
      "-- check region:17 6:127220000-127230000_reg_Ccnd2-25, 19794 candidate probes\n",
      "--- 8542 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.789s, 128 probes kept.\n",
      "-- check region:18 6:127230000-127240000_reg_Ccnd2-26, 19462 candidate probes\n",
      "--- 6498 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.674s, 101 probes kept.\n",
      "-- check region:19 6:127240000-127250000_reg_Ccnd2-27, 19824 candidate probes\n",
      "--- 10998 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.925s, 139 probes kept.\n",
      "-- check region:20 6:127250000-127260000_reg_Ccnd2-28, 19756 candidate probes\n",
      "--- 8500 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.768s, 128 probes kept.\n",
      "-- check region:21 6:127260000-127270000_reg_Ccnd2-29, 19688 candidate probes\n",
      "--- 11194 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.910s, 145 probes kept.\n",
      "-- check region:22 6:127000000-127010000_reg_Ccnd2-3, 19604 candidate probes\n",
      "--- 6672 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.746s, 104 probes kept.\n",
      "-- check region:23 6:127270000-127280000_reg_Ccnd2-30, 19918 candidate probes\n",
      "--- 8509 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.778s, 125 probes kept.\n",
      "-- check region:24 6:127280000-127290000_reg_Ccnd2-31, 19688 candidate probes\n",
      "--- 7936 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.778s, 124 probes kept.\n",
      "-- check region:25 6:127290000-127300000_reg_Ccnd2-32, 19906 candidate probes\n",
      "--- 9148 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.788s, 135 probes kept.\n",
      "-- check region:26 6:127300000-127310000_reg_Ccnd2-33, 19638 candidate probes\n",
      "--- 8696 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.819s, 127 probes kept.\n",
      "-- check region:27 6:127310000-127320000_reg_Ccnd2-34, 19884 candidate probes\n",
      "--- 10146 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.819s, 133 probes kept.\n",
      "-- check region:28 6:127320000-127330000_reg_Ccnd2-35, 19762 candidate probes\n",
      "--- 12211 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.953s, 149 probes kept.\n",
      "-- check region:29 6:127330000-127340000_reg_Ccnd2-36, 19778 candidate probes\n",
      "--- 10971 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.967s, 138 probes kept.\n",
      "-- check region:30 6:127340000-127350000_reg_Ccnd2-37, 18560 candidate probes\n",
      "--- 9152 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.821s, 118 probes kept.\n",
      "-- check region:31 6:127350000-127360000_reg_Ccnd2-38, 19838 candidate probes\n",
      "--- 10555 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.889s, 145 probes kept.\n",
      "-- check region:32 6:127360000-127370000_reg_Ccnd2-39, 19550 candidate probes\n",
      "--- 7940 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.788s, 119 probes kept.\n",
      "-- check region:33 6:127010000-127020000_reg_Ccnd2-4, 19850 candidate probes\n",
      "--- 12432 probes passed check_dic, GC and Tm selection.\n",
      "finish in 1.031s, 149 probes kept.\n",
      "-- check region:34 6:127370000-127380000_reg_Ccnd2-40, 19682 candidate probes\n",
      "--- 9855 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.867s, 135 probes kept.\n",
      "-- check region:35 6:127380000-127390000_reg_Ccnd2-41, 19396 candidate probes\n",
      "--- 9081 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.819s, 124 probes kept.\n",
      "-- check region:36 6:127390000-127400000_reg_Ccnd2-42, 19014 candidate probes\n",
      "--- 8347 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.826s, 117 probes kept.\n",
      "-- check region:37 6:127400000-127410000_reg_Ccnd2-43, 19690 candidate probes\n",
      "--- 8824 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.836s, 126 probes kept.\n",
      "-- check region:38 6:127410000-127420000_reg_Ccnd2-44, 19826 candidate probes\n",
      "--- 8857 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.841s, 128 probes kept.\n",
      "-- check region:39 6:127420000-127430000_reg_Ccnd2-45, 19766 candidate probes\n",
      "--- 8867 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.809s, 128 probes kept.\n",
      "-- check region:40 6:127430000-127440000_reg_Ccnd2-46, 19748 candidate probes\n",
      "--- 8973 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.789s, 130 probes kept.\n",
      "-- check region:41 6:127440000-127450000_reg_Ccnd2-47, 19706 candidate probes\n",
      "--- 10220 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.877s, 142 probes kept.\n",
      "-- check region:42 6:127450000-127460000_reg_Ccnd2-48, 19918 candidate probes\n",
      "--- 9416 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.879s, 135 probes kept.\n",
      "-- check region:43 6:127460000-127470000_reg_Ccnd2-49, 19140 candidate probes\n",
      "--- 9309 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.812s, 128 probes kept.\n",
      "-- check region:44 6:127020000-127030000_reg_Ccnd2-5, 19870 candidate probes\n",
      "--- 8980 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.816s, 152 probes kept.\n",
      "-- check region:45 6:127470000-127480000_reg_Ccnd2-50, 19918 candidate probes\n",
      "--- 7944 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.807s, 133 probes kept.\n",
      "-- check region:46 6:127480000-127490000_reg_Ccnd2-51, 19678 candidate probes\n",
      "--- 7159 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.800s, 99 probes kept.\n",
      "-- check region:47 6:127490000-127500000_reg_Ccnd2-52, 19866 candidate probes\n",
      "--- 8625 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.879s, 136 probes kept.\n",
      "-- check region:48 6:127500000-127510000_reg_Ccnd2-53, 19602 candidate probes\n",
      "--- 10232 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.856s, 132 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:49 6:127510000-127520000_reg_Ccnd2-54, 19324 candidate probes\n",
      "--- 2003 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.536s, 34 probes kept.\n",
      "-- check region:50 6:127520000-127530000_reg_Ccnd2-55, 19804 candidate probes\n",
      "--- 7027 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.799s, 101 probes kept.\n",
      "-- check region:51 6:127530000-127540000_reg_Ccnd2-56, 19464 candidate probes\n",
      "--- 7156 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.764s, 116 probes kept.\n",
      "-- check region:52 6:127540000-127550000_reg_Ccnd2-57, 19896 candidate probes\n",
      "--- 10335 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.908s, 134 probes kept.\n",
      "-- check region:53 6:127550000-127560000_reg_Ccnd2-58, 19900 candidate probes\n",
      "--- 10117 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.922s, 142 probes kept.\n",
      "-- check region:54 6:127560000-127570000_reg_Ccnd2-59, 19752 candidate probes\n",
      "--- 11473 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.860s, 147 probes kept.\n",
      "-- check region:55 6:127030000-127040000_reg_Ccnd2-6, 19520 candidate probes\n",
      "--- 8012 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.759s, 112 probes kept.\n",
      "-- check region:56 6:127570000-127580000_reg_Ccnd2-60, 19848 candidate probes\n",
      "--- 10010 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.934s, 143 probes kept.\n",
      "-- check region:57 6:127040000-127050000_reg_Ccnd2-7, 19732 candidate probes\n",
      "--- 10024 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.806s, 136 probes kept.\n",
      "-- check region:58 6:127050000-127060000_reg_Ccnd2-8, 19858 candidate probes\n",
      "--- 10232 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.860s, 140 probes kept.\n",
      "-- check region:59 6:127060000-127070000_reg_Ccnd2-9, 19440 candidate probes\n",
      "--- 8921 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.870s, 134 probes kept.\n",
      "-- check region:60 12:56350000-56360000_reg_Nkx2-1-1, 19226 candidate probes\n",
      "--- 4447 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.659s, 79 probes kept.\n",
      "-- check region:61 12:56440000-56450000_reg_Nkx2-1-10, 19756 candidate probes\n",
      "--- 9907 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.912s, 129 probes kept.\n",
      "-- check region:62 12:56450000-56460000_reg_Nkx2-1-11, 19902 candidate probes\n",
      "--- 8851 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.791s, 128 probes kept.\n",
      "-- check region:63 12:56460000-56470000_reg_Nkx2-1-12, 19586 candidate probes\n",
      "--- 7943 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.736s, 118 probes kept.\n",
      "-- check region:64 12:56470000-56480000_reg_Nkx2-1-13, 19864 candidate probes\n",
      "--- 9676 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.827s, 134 probes kept.\n",
      "-- check region:65 12:56480000-56490000_reg_Nkx2-1-14, 19918 candidate probes\n",
      "--- 8279 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.828s, 129 probes kept.\n",
      "-- check region:66 12:56490000-56500000_reg_Nkx2-1-15, 19736 candidate probes\n",
      "--- 6948 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.737s, 129 probes kept.\n",
      "-- check region:67 12:56500000-56510000_reg_Nkx2-1-16, 19706 candidate probes\n",
      "--- 8105 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.777s, 125 probes kept.\n",
      "-- check region:68 12:56510000-56520000_reg_Nkx2-1-17, 19562 candidate probes\n",
      "--- 9438 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.811s, 138 probes kept.\n",
      "-- check region:69 12:56520000-56530000_reg_Nkx2-1-18, 19798 candidate probes\n",
      "--- 10994 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.928s, 146 probes kept.\n",
      "-- check region:70 12:56530000-56540000_reg_Nkx2-1-19, 19680 candidate probes\n",
      "--- 10878 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.909s, 153 probes kept.\n",
      "-- check region:71 12:56360000-56370000_reg_Nkx2-1-2, 19184 candidate probes\n",
      "--- 3295 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.599s, 62 probes kept.\n",
      "-- check region:72 12:56540000-56550000_reg_Nkx2-1-20, 19686 candidate probes\n",
      "--- 8647 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.827s, 123 probes kept.\n",
      "-- check region:73 12:56550000-56560000_reg_Nkx2-1-21, 19514 candidate probes\n",
      "--- 5754 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.677s, 100 probes kept.\n",
      "-- check region:74 12:56560000-56570000_reg_Nkx2-1-22, 19522 candidate probes\n",
      "--- 3004 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.587s, 53 probes kept.\n",
      "-- check region:75 12:56570000-56580000_reg_Nkx2-1-23, 17818 candidate probes\n",
      "--- 5218 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.625s, 87 probes kept.\n",
      "-- check region:76 12:56580000-56590000_reg_Nkx2-1-24, 16960 candidate probes\n",
      "--- 5231 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.617s, 76 probes kept.\n",
      "-- check region:77 12:56590000-56600000_reg_Nkx2-1-25, 19698 candidate probes\n",
      "--- 6742 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.742s, 111 probes kept.\n",
      "-- check region:78 12:56600000-56610000_reg_Nkx2-1-26, 19842 candidate probes\n",
      "--- 7526 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.741s, 121 probes kept.\n",
      "-- check region:79 12:56610000-56620000_reg_Nkx2-1-27, 19828 candidate probes\n",
      "--- 10437 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.862s, 144 probes kept.\n",
      "-- check region:80 12:56620000-56630000_reg_Nkx2-1-28, 19598 candidate probes\n",
      "--- 7279 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.744s, 116 probes kept.\n",
      "-- check region:81 12:56630000-56640000_reg_Nkx2-1-29, 19834 candidate probes\n",
      "--- 9747 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.806s, 141 probes kept.\n",
      "-- check region:82 12:56370000-56380000_reg_Nkx2-1-3, 19164 candidate probes\n",
      "--- 5166 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.687s, 86 probes kept.\n",
      "-- check region:83 12:56640000-56650000_reg_Nkx2-1-30, 19866 candidate probes\n",
      "--- 7832 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.826s, 125 probes kept.\n",
      "-- check region:84 12:56650000-56660000_reg_Nkx2-1-31, 19878 candidate probes\n",
      "--- 5267 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.677s, 94 probes kept.\n",
      "-- check region:85 12:56660000-56670000_reg_Nkx2-1-32, 19570 candidate probes\n",
      "--- 6214 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.725s, 102 probes kept.\n",
      "-- check region:86 12:56670000-56680000_reg_Nkx2-1-33, 19856 candidate probes\n",
      "--- 9365 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.862s, 135 probes kept.\n",
      "-- check region:87 12:56680000-56690000_reg_Nkx2-1-34, 19822 candidate probes\n",
      "--- 11095 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.889s, 149 probes kept.\n",
      "-- check region:88 12:56690000-56700000_reg_Nkx2-1-35, 19808 candidate probes\n",
      "--- 12340 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.950s, 154 probes kept.\n",
      "-- check region:89 12:56700000-56710000_reg_Nkx2-1-36, 19544 candidate probes\n",
      "--- 8265 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.784s, 135 probes kept.\n",
      "-- check region:90 12:56710000-56720000_reg_Nkx2-1-37, 19764 candidate probes\n",
      "--- 7763 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.796s, 133 probes kept.\n",
      "-- check region:91 12:56720000-56730000_reg_Nkx2-1-38, 19654 candidate probes\n",
      "--- 8975 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.878s, 130 probes kept.\n",
      "-- check region:92 12:56730000-56740000_reg_Nkx2-1-39, 19786 candidate probes\n",
      "--- 8027 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.787s, 125 probes kept.\n",
      "-- check region:93 12:56380000-56390000_reg_Nkx2-1-4, 19770 candidate probes\n",
      "--- 10270 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.872s, 137 probes kept.\n",
      "-- check region:94 12:56740000-56750000_reg_Nkx2-1-40, 18968 candidate probes\n",
      "--- 3181 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.565s, 58 probes kept.\n",
      "-- check region:95 12:56750000-56760000_reg_Nkx2-1-41, 19794 candidate probes\n",
      "--- 7329 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.802s, 101 probes kept.\n",
      "-- check region:96 12:56760000-56770000_reg_Nkx2-1-42, 19680 candidate probes\n",
      "--- 7429 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.801s, 118 probes kept.\n",
      "-- check region:97 12:56770000-56780000_reg_Nkx2-1-43, 19812 candidate probes\n",
      "--- 8122 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.817s, 123 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:98 12:56780000-56790000_reg_Nkx2-1-44, 18880 candidate probes\n",
      "--- 7987 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.771s, 112 probes kept.\n",
      "-- check region:99 12:56790000-56800000_reg_Nkx2-1-45, 19602 candidate probes\n",
      "--- 9115 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.790s, 133 probes kept.\n",
      "-- check region:100 12:56800000-56810000_reg_Nkx2-1-46, 19644 candidate probes\n",
      "--- 10343 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.938s, 138 probes kept.\n",
      "-- check region:101 12:56810000-56820000_reg_Nkx2-1-47, 19774 candidate probes\n",
      "--- 8727 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.869s, 130 probes kept.\n",
      "-- check region:102 12:56820000-56830000_reg_Nkx2-1-48, 19654 candidate probes\n",
      "--- 9678 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.781s, 132 probes kept.\n",
      "-- check region:103 12:56830000-56840000_reg_Nkx2-1-49, 19566 candidate probes\n",
      "--- 7536 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.765s, 100 probes kept.\n",
      "-- check region:104 12:56390000-56400000_reg_Nkx2-1-5, 19858 candidate probes\n",
      "--- 8814 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.837s, 120 probes kept.\n",
      "-- check region:105 12:56840000-56850000_reg_Nkx2-1-50, 19834 candidate probes\n",
      "--- 6005 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.749s, 90 probes kept.\n",
      "-- check region:106 12:56850000-56860000_reg_Nkx2-1-51, 19418 candidate probes\n",
      "--- 8962 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.817s, 122 probes kept.\n",
      "-- check region:107 12:56860000-56870000_reg_Nkx2-1-52, 19892 candidate probes\n",
      "--- 9961 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.878s, 139 probes kept.\n",
      "-- check region:108 12:56870000-56880000_reg_Nkx2-1-53, 19626 candidate probes\n",
      "--- 8986 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.840s, 123 probes kept.\n",
      "-- check region:109 12:56880000-56890000_reg_Nkx2-1-54, 19910 candidate probes\n",
      "--- 9073 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.796s, 128 probes kept.\n",
      "-- check region:110 12:56890000-56900000_reg_Nkx2-1-55, 19820 candidate probes\n",
      "--- 10228 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.799s, 138 probes kept.\n",
      "-- check region:111 12:56900000-56910000_reg_Nkx2-1-56, 19914 candidate probes\n",
      "--- 9342 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.789s, 137 probes kept.\n",
      "-- check region:112 12:56910000-56920000_reg_Nkx2-1-57, 19104 candidate probes\n",
      "--- 6992 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.728s, 100 probes kept.\n",
      "-- check region:113 12:56920000-56930000_reg_Nkx2-1-58, 19794 candidate probes\n",
      "--- 8474 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.813s, 122 probes kept.\n",
      "-- check region:114 12:56930000-56940000_reg_Nkx2-1-59, 19540 candidate probes\n",
      "--- 8112 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.813s, 125 probes kept.\n",
      "-- check region:115 12:56400000-56410000_reg_Nkx2-1-6, 19626 candidate probes\n",
      "--- 11035 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.889s, 141 probes kept.\n",
      "-- check region:116 12:56940000-56950000_reg_Nkx2-1-60, 19106 candidate probes\n",
      "--- 4806 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.627s, 86 probes kept.\n",
      "-- check region:117 12:56410000-56420000_reg_Nkx2-1-7, 19680 candidate probes\n",
      "--- 9535 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.837s, 138 probes kept.\n",
      "-- check region:118 12:56420000-56430000_reg_Nkx2-1-8, 17550 candidate probes\n",
      "--- 8440 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.734s, 118 probes kept.\n",
      "-- check region:119 12:56430000-56440000_reg_Nkx2-1-9, 19770 candidate probes\n",
      "--- 9203 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.799s, 126 probes kept.\n",
      "-- check region:120 7:44740000-44750000_reg_Slc17a7-1, 19824 candidate probes\n",
      "--- 5674 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.684s, 104 probes kept.\n",
      "-- check region:121 7:44830000-44840000_reg_Slc17a7-10, 19552 candidate probes\n",
      "--- 5764 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.777s, 112 probes kept.\n",
      "-- check region:122 7:44840000-44850000_reg_Slc17a7-11, 19864 candidate probes\n",
      "--- 8514 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.809s, 137 probes kept.\n",
      "-- check region:123 7:44850000-44860000_reg_Slc17a7-12, 19790 candidate probes\n",
      "--- 9836 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.932s, 148 probes kept.\n",
      "-- check region:124 7:44860000-44870000_reg_Slc17a7-13, 19788 candidate probes\n",
      "--- 9716 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.858s, 164 probes kept.\n",
      "-- check region:125 7:44870000-44880000_reg_Slc17a7-14, 19642 candidate probes\n",
      "--- 7579 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.728s, 123 probes kept.\n",
      "-- check region:126 7:44880000-44890000_reg_Slc17a7-15, 19606 candidate probes\n",
      "--- 8978 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.837s, 140 probes kept.\n",
      "-- check region:127 7:44890000-44900000_reg_Slc17a7-16, 19668 candidate probes\n",
      "--- 7470 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.798s, 144 probes kept.\n",
      "-- check region:128 7:44900000-44910000_reg_Slc17a7-17, 19918 candidate probes\n",
      "--- 8690 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.879s, 148 probes kept.\n",
      "-- check region:129 7:44910000-44920000_reg_Slc17a7-18, 19488 candidate probes\n",
      "--- 7924 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.781s, 126 probes kept.\n",
      "-- check region:130 7:44920000-44930000_reg_Slc17a7-19, 19504 candidate probes\n",
      "--- 7619 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.777s, 114 probes kept.\n",
      "-- check region:131 7:44750000-44760000_reg_Slc17a7-2, 19852 candidate probes\n",
      "--- 5284 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.688s, 99 probes kept.\n",
      "-- check region:132 7:44930000-44940000_reg_Slc17a7-20, 19542 candidate probes\n",
      "--- 5406 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.726s, 94 probes kept.\n",
      "-- check region:133 7:44940000-44950000_reg_Slc17a7-21, 19370 candidate probes\n",
      "--- 5743 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.688s, 92 probes kept.\n",
      "-- check region:134 7:44950000-44960000_reg_Slc17a7-22, 19650 candidate probes\n",
      "--- 8835 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.849s, 130 probes kept.\n",
      "-- check region:135 7:44960000-44970000_reg_Slc17a7-23, 19094 candidate probes\n",
      "--- 6341 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.748s, 119 probes kept.\n",
      "-- check region:136 7:44970000-44980000_reg_Slc17a7-24, 19868 candidate probes\n",
      "--- 8736 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.819s, 127 probes kept.\n",
      "-- check region:137 7:44980000-44990000_reg_Slc17a7-25, 19500 candidate probes\n",
      "--- 8416 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.829s, 122 probes kept.\n",
      "-- check region:138 7:44990000-45000000_reg_Slc17a7-26, 19780 candidate probes\n",
      "--- 7324 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.799s, 134 probes kept.\n",
      "-- check region:139 7:45000000-45010000_reg_Slc17a7-27, 19808 candidate probes\n",
      "--- 8673 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.850s, 147 probes kept.\n",
      "-- check region:140 7:45010000-45020000_reg_Slc17a7-28, 19806 candidate probes\n",
      "--- 11016 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.860s, 153 probes kept.\n",
      "-- check region:141 7:45020000-45030000_reg_Slc17a7-29, 19734 candidate probes\n",
      "--- 7828 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.920s, 127 probes kept.\n",
      "-- check region:142 7:44760000-44770000_reg_Slc17a7-3, 19904 candidate probes\n",
      "--- 8474 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.798s, 137 probes kept.\n",
      "-- check region:143 7:45030000-45040000_reg_Slc17a7-30, 19770 candidate probes\n",
      "--- 8992 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.821s, 133 probes kept.\n",
      "-- check region:144 7:45040000-45050000_reg_Slc17a7-31, 19710 candidate probes\n",
      "--- 8122 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.829s, 140 probes kept.\n",
      "-- check region:145 7:45050000-45060000_reg_Slc17a7-32, 19736 candidate probes\n",
      "--- 9208 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.801s, 141 probes kept.\n",
      "-- check region:146 7:45060000-45070000_reg_Slc17a7-33, 19876 candidate probes\n",
      "--- 6431 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.787s, 112 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:147 7:45070000-45080000_reg_Slc17a7-34, 19788 candidate probes\n",
      "--- 7047 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.788s, 125 probes kept.\n",
      "-- check region:148 7:45080000-45090000_reg_Slc17a7-35, 19628 candidate probes\n",
      "--- 6192 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.760s, 102 probes kept.\n",
      "-- check region:149 7:45090000-45100000_reg_Slc17a7-36, 19580 candidate probes\n",
      "--- 6853 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.771s, 106 probes kept.\n",
      "-- check region:150 7:45100000-45110000_reg_Slc17a7-37, 19530 candidate probes\n",
      "--- 4934 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.657s, 99 probes kept.\n",
      "-- check region:151 7:45110000-45120000_reg_Slc17a7-38, 19340 candidate probes\n",
      "--- 5840 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.736s, 93 probes kept.\n",
      "-- check region:152 7:45120000-45130000_reg_Slc17a7-39, 19918 candidate probes\n",
      "--- 9946 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.849s, 145 probes kept.\n",
      "-- check region:153 7:44770000-44780000_reg_Slc17a7-4, 19778 candidate probes\n",
      "--- 6900 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.779s, 110 probes kept.\n",
      "-- check region:154 7:45130000-45140000_reg_Slc17a7-40, 19626 candidate probes\n",
      "--- 6912 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.769s, 121 probes kept.\n",
      "-- check region:155 7:45140000-45150000_reg_Slc17a7-41, 19894 candidate probes\n",
      "--- 7891 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.833s, 134 probes kept.\n",
      "-- check region:156 7:45150000-45160000_reg_Slc17a7-42, 19624 candidate probes\n",
      "--- 7555 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.689s, 130 probes kept.\n",
      "-- check region:157 7:45160000-45170000_reg_Slc17a7-43, 19868 candidate probes\n",
      "--- 8658 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.778s, 132 probes kept.\n",
      "-- check region:158 7:45170000-45180000_reg_Slc17a7-44, 19614 candidate probes\n",
      "--- 7971 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.821s, 141 probes kept.\n",
      "-- check region:159 7:45180000-45190000_reg_Slc17a7-45, 19810 candidate probes\n",
      "--- 10508 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.880s, 145 probes kept.\n",
      "-- check region:160 7:45190000-45200000_reg_Slc17a7-46, 19812 candidate probes\n",
      "--- 9395 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.908s, 132 probes kept.\n",
      "-- check region:161 7:45200000-45210000_reg_Slc17a7-47, 19626 candidate probes\n",
      "--- 6709 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.768s, 111 probes kept.\n",
      "-- check region:162 7:45210000-45220000_reg_Slc17a7-48, 19628 candidate probes\n",
      "--- 6927 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.696s, 129 probes kept.\n",
      "-- check region:163 7:45220000-45230000_reg_Slc17a7-49, 19740 candidate probes\n",
      "--- 8435 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.798s, 130 probes kept.\n",
      "-- check region:164 7:44780000-44790000_reg_Slc17a7-5, 19788 candidate probes\n",
      "--- 9345 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.868s, 124 probes kept.\n",
      "-- check region:165 7:45230000-45240000_reg_Slc17a7-50, 19558 candidate probes\n",
      "--- 7645 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.789s, 135 probes kept.\n",
      "-- check region:166 7:45240000-45250000_reg_Slc17a7-51, 19490 candidate probes\n",
      "--- 3367 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.631s, 65 probes kept.\n",
      "-- check region:167 7:45250000-45260000_reg_Slc17a7-52, 19332 candidate probes\n",
      "--- 4964 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.667s, 90 probes kept.\n",
      "-- check region:168 7:45260000-45270000_reg_Slc17a7-53, 19918 candidate probes\n",
      "--- 8480 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.864s, 125 probes kept.\n",
      "-- check region:169 7:45270000-45280000_reg_Slc17a7-54, 19828 candidate probes\n",
      "--- 8383 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.849s, 120 probes kept.\n",
      "-- check region:170 7:45280000-45290000_reg_Slc17a7-55, 19832 candidate probes\n",
      "--- 7619 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.756s, 132 probes kept.\n",
      "-- check region:171 7:45290000-45300000_reg_Slc17a7-56, 19912 candidate probes\n",
      "--- 6617 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.736s, 114 probes kept.\n",
      "-- check region:172 7:45300000-45310000_reg_Slc17a7-57, 19856 candidate probes\n",
      "--- 7480 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.738s, 130 probes kept.\n",
      "-- check region:173 7:45310000-45320000_reg_Slc17a7-58, 19646 candidate probes\n",
      "--- 7198 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.728s, 121 probes kept.\n",
      "-- check region:174 7:45320000-45330000_reg_Slc17a7-59, 19452 candidate probes\n",
      "--- 6379 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.678s, 102 probes kept.\n",
      "-- check region:175 7:44790000-44800000_reg_Slc17a7-6, 18430 candidate probes\n",
      "--- 2667 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.521s, 53 probes kept.\n",
      "-- check region:176 7:45330000-45340000_reg_Slc17a7-60, 19862 candidate probes\n",
      "--- 6928 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.765s, 142 probes kept.\n",
      "-- check region:177 7:44800000-44810000_reg_Slc17a7-7, 19624 candidate probes\n",
      "--- 3638 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.593s, 66 probes kept.\n",
      "-- check region:178 7:44810000-44820000_reg_Slc17a7-8, 19918 candidate probes\n",
      "--- 9043 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.850s, 141 probes kept.\n",
      "-- check region:179 7:44820000-44830000_reg_Slc17a7-9, 19834 candidate probes\n",
      "--- 10221 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.908s, 146 probes kept.\n",
      "-- check region:180 2:158510767-158520767_reg_Slc32a1-1, 19706 candidate probes\n",
      "--- 8853 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.819s, 129 probes kept.\n",
      "-- check region:181 2:158600767-158610767_reg_Slc32a1-10, 19750 candidate probes\n",
      "--- 8527 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.788s, 128 probes kept.\n",
      "-- check region:182 2:158610767-158620767_reg_Slc32a1-11, 19868 candidate probes\n",
      "--- 10433 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.931s, 144 probes kept.\n",
      "-- check region:183 2:158620767-158630767_reg_Slc32a1-12, 19770 candidate probes\n",
      "--- 7716 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.809s, 129 probes kept.\n",
      "-- check region:184 2:158630767-158640767_reg_Slc32a1-13, 19770 candidate probes\n",
      "--- 8996 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.894s, 144 probes kept.\n",
      "-- check region:185 2:158640767-158650767_reg_Slc32a1-14, 19740 candidate probes\n",
      "--- 8363 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.759s, 121 probes kept.\n",
      "-- check region:186 2:158650767-158660767_reg_Slc32a1-15, 19724 candidate probes\n",
      "--- 8314 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.847s, 127 probes kept.\n",
      "-- check region:187 2:158660767-158670767_reg_Slc32a1-16, 19870 candidate probes\n",
      "--- 8314 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.757s, 114 probes kept.\n",
      "-- check region:188 2:158670767-158680767_reg_Slc32a1-17, 19816 candidate probes\n",
      "--- 10377 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.849s, 141 probes kept.\n",
      "-- check region:189 2:158680767-158690767_reg_Slc32a1-18, 19896 candidate probes\n",
      "--- 10215 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.789s, 127 probes kept.\n",
      "-- check region:190 2:158690767-158700767_reg_Slc32a1-19, 19880 candidate probes\n",
      "--- 9847 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.835s, 136 probes kept.\n",
      "-- check region:191 2:158520767-158530767_reg_Slc32a1-2, 19818 candidate probes\n",
      "--- 10399 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.942s, 138 probes kept.\n",
      "-- check region:192 2:158700767-158710767_reg_Slc32a1-20, 19836 candidate probes\n",
      "--- 9877 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.922s, 135 probes kept.\n",
      "-- check region:193 2:158710767-158720767_reg_Slc32a1-21, 19896 candidate probes\n",
      "--- 11421 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.920s, 144 probes kept.\n",
      "-- check region:194 2:158530767-158540767_reg_Slc32a1-3, 19848 candidate probes\n",
      "--- 8517 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.750s, 129 probes kept.\n",
      "-- check region:195 2:158540767-158550767_reg_Slc32a1-4, 19684 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 8310 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.822s, 123 probes kept.\n",
      "-- check region:196 2:158550767-158560767_reg_Slc32a1-5, 19594 candidate probes\n",
      "--- 9716 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.818s, 134 probes kept.\n",
      "-- check region:197 2:158560767-158570767_reg_Slc32a1-6, 19726 candidate probes\n",
      "--- 9557 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.900s, 131 probes kept.\n",
      "-- check region:198 2:158570767-158580767_reg_Slc32a1-7, 19796 candidate probes\n",
      "--- 8722 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.816s, 123 probes kept.\n",
      "-- check region:199 2:158580767-158590767_reg_Slc32a1-8, 19858 candidate probes\n",
      "--- 7190 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.766s, 114 probes kept.\n",
      "-- check region:200 2:158590767-158600767_reg_Slc32a1-9, 19892 candidate probes\n",
      "--- 7209 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.825s, 123 probes kept.\n",
      "-- check region:201 16:23730000-23740000_reg_SST-1, 17852 candidate probes\n",
      "--- 6892 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.735s, 105 probes kept.\n",
      "-- check region:202 16:23820000-23830000_reg_SST-10, 19822 candidate probes\n",
      "--- 10983 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.927s, 141 probes kept.\n",
      "-- check region:203 16:23830000-23840000_reg_SST-11, 19630 candidate probes\n",
      "--- 11010 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.930s, 145 probes kept.\n",
      "-- check region:204 16:23840000-23850000_reg_SST-12, 19908 candidate probes\n",
      "--- 10531 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.952s, 141 probes kept.\n",
      "-- check region:205 16:23850000-23860000_reg_SST-13, 19074 candidate probes\n",
      "--- 4831 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.677s, 79 probes kept.\n",
      "-- check region:206 16:23860000-23870000_reg_SST-14, 19138 candidate probes\n",
      "--- 5303 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.707s, 95 probes kept.\n",
      "-- check region:207 16:23870000-23880000_reg_SST-15, 19856 candidate probes\n",
      "--- 5244 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.667s, 82 probes kept.\n",
      "-- check region:208 16:23880000-23890000_reg_SST-16, 19746 candidate probes\n",
      "--- 8736 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.749s, 131 probes kept.\n",
      "-- check region:209 16:23890000-23900000_reg_SST-17, 19896 candidate probes\n",
      "--- 11553 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.967s, 151 probes kept.\n",
      "-- check region:210 16:23900000-23910000_reg_SST-18, 19906 candidate probes\n",
      "--- 5851 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.748s, 96 probes kept.\n",
      "-- check region:211 16:23910000-23920000_reg_SST-19, 19910 candidate probes\n",
      "--- 6633 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.717s, 114 probes kept.\n",
      "-- check region:212 16:23740000-23750000_reg_SST-2, 19662 candidate probes\n",
      "--- 3097 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.595s, 49 probes kept.\n",
      "-- check region:213 16:23920000-23930000_reg_SST-20, 19874 candidate probes\n",
      "--- 6416 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.695s, 114 probes kept.\n",
      "-- check region:214 16:23930000-23940000_reg_SST-21, 18870 candidate probes\n",
      "--- 7223 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.760s, 103 probes kept.\n",
      "-- check region:215 16:23940000-23950000_reg_SST-22, 19280 candidate probes\n",
      "--- 3774 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.597s, 61 probes kept.\n",
      "-- check region:216 16:23950000-23960000_reg_SST-23, 19726 candidate probes\n",
      "--- 7518 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.773s, 112 probes kept.\n",
      "-- check region:217 16:23960000-23970000_reg_SST-24, 19918 candidate probes\n",
      "--- 9481 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.877s, 148 probes kept.\n",
      "-- check region:218 16:23970000-23980000_reg_SST-25, 19790 candidate probes\n",
      "--- 9540 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.821s, 140 probes kept.\n",
      "-- check region:219 16:23980000-23990000_reg_SST-26, 19908 candidate probes\n",
      "--- 11327 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.901s, 151 probes kept.\n",
      "-- check region:220 16:23990000-24000000_reg_SST-27, 19738 candidate probes\n",
      "--- 10502 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.859s, 141 probes kept.\n",
      "-- check region:221 16:24000000-24010000_reg_SST-28, 19918 candidate probes\n",
      "--- 9338 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.858s, 137 probes kept.\n",
      "-- check region:222 16:24010000-24020000_reg_SST-29, 19812 candidate probes\n",
      "--- 9276 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.738s, 124 probes kept.\n",
      "-- check region:223 16:23750000-23760000_reg_SST-3, 18802 candidate probes\n",
      "--- 3608 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.586s, 53 probes kept.\n",
      "-- check region:224 16:24020000-24030000_reg_SST-30, 19788 candidate probes\n",
      "--- 10192 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.899s, 141 probes kept.\n",
      "-- check region:225 16:24030000-24040000_reg_SST-31, 19824 candidate probes\n",
      "--- 6374 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.699s, 108 probes kept.\n",
      "-- check region:226 16:24040000-24050000_reg_SST-32, 19876 candidate probes\n",
      "--- 10003 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.850s, 139 probes kept.\n",
      "-- check region:227 16:24050000-24060000_reg_SST-33, 19888 candidate probes\n",
      "--- 11656 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.890s, 150 probes kept.\n",
      "-- check region:228 16:24060000-24070000_reg_SST-34, 19916 candidate probes\n",
      "--- 10431 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.950s, 148 probes kept.\n",
      "-- check region:229 16:24070000-24080000_reg_SST-35, 19818 candidate probes\n",
      "--- 9747 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.859s, 141 probes kept.\n",
      "-- check region:230 16:24080000-24090000_reg_SST-36, 19918 candidate probes\n",
      "--- 9165 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.878s, 144 probes kept.\n",
      "-- check region:231 16:24090000-24100000_reg_SST-37, 19896 candidate probes\n",
      "--- 10938 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.960s, 147 probes kept.\n",
      "-- check region:232 16:24100000-24110000_reg_SST-38, 19038 candidate probes\n",
      "--- 7990 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.738s, 114 probes kept.\n",
      "-- check region:233 16:24110000-24120000_reg_SST-39, 19590 candidate probes\n",
      "--- 7873 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.788s, 112 probes kept.\n",
      "-- check region:234 16:23760000-23770000_reg_SST-4, 19634 candidate probes\n",
      "--- 7452 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.771s, 118 probes kept.\n",
      "-- check region:235 16:24120000-24130000_reg_SST-40, 19884 candidate probes\n",
      "--- 7440 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.771s, 135 probes kept.\n",
      "-- check region:236 16:24130000-24140000_reg_SST-41, 19844 candidate probes\n",
      "--- 10352 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.896s, 147 probes kept.\n",
      "-- check region:237 16:24140000-24150000_reg_SST-42, 19816 candidate probes\n",
      "--- 11053 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.849s, 141 probes kept.\n",
      "-- check region:238 16:24150000-24160000_reg_SST-43, 19742 candidate probes\n",
      "--- 10587 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.898s, 139 probes kept.\n",
      "-- check region:239 16:24160000-24170000_reg_SST-44, 19846 candidate probes\n",
      "--- 11570 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.848s, 146 probes kept.\n",
      "-- check region:240 16:24170000-24180000_reg_SST-45, 19874 candidate probes\n",
      "--- 11189 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.968s, 139 probes kept.\n",
      "-- check region:241 16:24180000-24190000_reg_SST-46, 19850 candidate probes\n",
      "--- 9645 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.920s, 138 probes kept.\n",
      "-- check region:242 16:24190000-24200000_reg_SST-47, 19808 candidate probes\n",
      "--- 8431 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.819s, 130 probes kept.\n",
      "-- check region:243 16:24200000-24210000_reg_SST-48, 19918 candidate probes\n",
      "--- 12217 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.992s, 161 probes kept.\n",
      "-- check region:244 16:24210000-24220000_reg_SST-49, 19918 candidate probes\n",
      "--- 11764 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.960s, 151 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:245 16:23770000-23780000_reg_SST-5, 19862 candidate probes\n",
      "--- 7433 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.757s, 114 probes kept.\n",
      "-- check region:246 16:24220000-24230000_reg_SST-50, 19918 candidate probes\n",
      "--- 8319 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.825s, 135 probes kept.\n",
      "-- check region:247 16:24230000-24240000_reg_SST-51, 19890 candidate probes\n",
      "--- 10895 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.919s, 147 probes kept.\n",
      "-- check region:248 16:24240000-24250000_reg_SST-52, 19918 candidate probes\n",
      "--- 7795 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.789s, 116 probes kept.\n",
      "-- check region:249 16:24250000-24260000_reg_SST-53, 19860 candidate probes\n",
      "--- 8485 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.748s, 124 probes kept.\n",
      "-- check region:250 16:24260000-24270000_reg_SST-54, 19918 candidate probes\n",
      "--- 6066 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.728s, 101 probes kept.\n",
      "-- check region:251 16:24270000-24280000_reg_SST-55, 19726 candidate probes\n",
      "--- 10025 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.930s, 122 probes kept.\n",
      "-- check region:252 16:24280000-24290000_reg_SST-56, 19784 candidate probes\n",
      "--- 7483 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.767s, 116 probes kept.\n",
      "-- check region:253 16:24290000-24300000_reg_SST-57, 19878 candidate probes\n",
      "--- 7794 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.706s, 117 probes kept.\n",
      "-- check region:254 16:24300000-24310000_reg_SST-58, 19918 candidate probes\n",
      "--- 5985 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.708s, 108 probes kept.\n",
      "-- check region:255 16:24310000-24320000_reg_SST-59, 19912 candidate probes\n",
      "--- 10096 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.929s, 134 probes kept.\n",
      "-- check region:256 16:23780000-23790000_reg_SST-6, 19918 candidate probes\n",
      "--- 10542 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.818s, 149 probes kept.\n",
      "-- check region:257 16:24320000-24330000_reg_SST-60, 19918 candidate probes\n",
      "--- 8012 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.798s, 113 probes kept.\n",
      "-- check region:258 16:23790000-23800000_reg_SST-7, 19872 candidate probes\n",
      "--- 9314 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.890s, 143 probes kept.\n",
      "-- check region:259 16:23800000-23810000_reg_SST-8, 19798 candidate probes\n",
      "--- 10058 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.881s, 130 probes kept.\n",
      "-- check region:260 16:23810000-23820000_reg_SST-9, 19822 candidate probes\n",
      "--- 10366 probes passed check_dic, GC and Tm selection.\n",
      "finish in 0.930s, 145 probes kept.\n"
     ]
    }
   ],
   "source": [
    "pbs, pb_scores = pb_designer.check_probes()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### save probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'name': '6:126980000-126990000_reg_Ccnd2-1_pb_31',\n",
       " 'reg_index': 0,\n",
       " 'reg_name': '6:126980000-126990000_reg_Ccnd2-1',\n",
       " 'pb_index': 31,\n",
       " 'strand': '+',\n",
       " 'gc': 0.42857142857142855,\n",
       " 'tm': 77.04397805909713,\n",
       " 'map_genome': 35,\n",
       " 'map_transcriptome': 0,\n",
       " 'map_rep_genome': 0,\n",
       " 'map_self_sequences': 26}"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "list(pb_designer.kept_probes.values())[0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr\n",
      "-- number of probes kept: 32437\n"
     ]
    }
   ],
   "source": [
    "overwrite_savefile = True \n",
    "if not os.path.exists(pb_designer.save_file) or overwrite_savefile:\n",
    "    #pb_designer.plots()\n",
    "    #pb_designer.save_csv()\n",
    "    pb_designer.save_to_file()\n",
    "print(f\"-- number of probes kept: {len(pb_designer.kept_probes)}\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 2.3 summarize into a pb_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Loading from savefile: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\reports\\merged_probes_10kb.pbr.\n",
      "cand_probes\n",
      "kept_probes\n",
      "32437\n",
      "sequence_dic\n",
      "map_dic\n",
      "params_dic\n",
      "check_dic\n",
      "save_file\n",
      "-- appending region:Ccnd2-1 with 135 probes\n",
      "-- appending region:Ccnd2-10 with 136 probes\n",
      "-- appending region:Ccnd2-11 with 139 probes\n",
      "-- appending region:Ccnd2-12 with 62 probes\n",
      "-- appending region:Ccnd2-13 with 127 probes\n",
      "-- appending region:Ccnd2-14 with 117 probes\n",
      "-- appending region:Ccnd2-15 with 125 probes\n",
      "-- appending region:Ccnd2-16 with 136 probes\n",
      "-- appending region:Ccnd2-17 with 147 probes\n",
      "-- appending region:Ccnd2-18 with 146 probes\n",
      "-- appending region:Ccnd2-19 with 143 probes\n",
      "-- appending region:Ccnd2-2 with 99 probes\n",
      "-- appending region:Ccnd2-20 with 142 probes\n",
      "-- appending region:Ccnd2-21 with 139 probes\n",
      "-- appending region:Ccnd2-22 with 126 probes\n",
      "-- appending region:Ccnd2-23 with 138 probes\n",
      "-- appending region:Ccnd2-24 with 136 probes\n",
      "-- appending region:Ccnd2-25 with 128 probes\n",
      "-- appending region:Ccnd2-26 with 101 probes\n",
      "-- appending region:Ccnd2-27 with 139 probes\n",
      "-- appending region:Ccnd2-28 with 128 probes\n",
      "-- appending region:Ccnd2-29 with 145 probes\n",
      "-- appending region:Ccnd2-3 with 104 probes\n",
      "-- appending region:Ccnd2-30 with 125 probes\n",
      "-- appending region:Ccnd2-31 with 124 probes\n",
      "-- appending region:Ccnd2-32 with 135 probes\n",
      "-- appending region:Ccnd2-33 with 127 probes\n",
      "-- appending region:Ccnd2-34 with 133 probes\n",
      "-- appending region:Ccnd2-35 with 149 probes\n",
      "-- appending region:Ccnd2-36 with 138 probes\n",
      "-- appending region:Ccnd2-37 with 118 probes\n",
      "-- appending region:Ccnd2-38 with 145 probes\n",
      "-- appending region:Ccnd2-39 with 119 probes\n",
      "-- appending region:Ccnd2-4 with 149 probes\n",
      "-- appending region:Ccnd2-40 with 135 probes\n",
      "-- appending region:Ccnd2-41 with 124 probes\n",
      "-- appending region:Ccnd2-42 with 117 probes\n",
      "-- appending region:Ccnd2-43 with 126 probes\n",
      "-- appending region:Ccnd2-44 with 128 probes\n",
      "-- appending region:Ccnd2-45 with 128 probes\n",
      "-- appending region:Ccnd2-46 with 130 probes\n",
      "-- appending region:Ccnd2-47 with 142 probes\n",
      "-- appending region:Ccnd2-48 with 135 probes\n",
      "-- appending region:Ccnd2-49 with 128 probes\n",
      "-- appending region:Ccnd2-5 with 152 probes\n",
      "-- appending region:Ccnd2-50 with 133 probes\n",
      "-- appending region:Ccnd2-51 with 99 probes\n",
      "-- appending region:Ccnd2-52 with 136 probes\n",
      "-- appending region:Ccnd2-53 with 132 probes\n",
      "-- appending region:Ccnd2-54 with 34 probes\n",
      "-- appending region:Ccnd2-55 with 101 probes\n",
      "-- appending region:Ccnd2-56 with 116 probes\n",
      "-- appending region:Ccnd2-57 with 134 probes\n",
      "-- appending region:Ccnd2-58 with 142 probes\n",
      "-- appending region:Ccnd2-59 with 147 probes\n",
      "-- appending region:Ccnd2-6 with 112 probes\n",
      "-- appending region:Ccnd2-60 with 143 probes\n",
      "-- appending region:Ccnd2-7 with 136 probes\n",
      "-- appending region:Ccnd2-8 with 140 probes\n",
      "-- appending region:Ccnd2-9 with 134 probes\n",
      "-- appending region:Nkx2-1-1 with 79 probes\n",
      "-- appending region:Nkx2-1-10 with 129 probes\n",
      "-- appending region:Nkx2-1-11 with 128 probes\n",
      "-- appending region:Nkx2-1-12 with 118 probes\n",
      "-- appending region:Nkx2-1-13 with 134 probes\n",
      "-- appending region:Nkx2-1-14 with 129 probes\n",
      "-- appending region:Nkx2-1-15 with 129 probes\n",
      "-- appending region:Nkx2-1-16 with 125 probes\n",
      "-- appending region:Nkx2-1-17 with 138 probes\n",
      "-- appending region:Nkx2-1-18 with 146 probes\n",
      "-- appending region:Nkx2-1-19 with 153 probes\n",
      "-- appending region:Nkx2-1-2 with 62 probes\n",
      "-- appending region:Nkx2-1-20 with 123 probes\n",
      "-- appending region:Nkx2-1-21 with 100 probes\n",
      "-- appending region:Nkx2-1-22 with 53 probes\n",
      "-- appending region:Nkx2-1-23 with 87 probes\n",
      "-- appending region:Nkx2-1-24 with 76 probes\n",
      "-- appending region:Nkx2-1-25 with 111 probes\n",
      "-- appending region:Nkx2-1-26 with 121 probes\n",
      "-- appending region:Nkx2-1-27 with 144 probes\n",
      "-- appending region:Nkx2-1-28 with 116 probes\n",
      "-- appending region:Nkx2-1-29 with 141 probes\n",
      "-- appending region:Nkx2-1-3 with 86 probes\n",
      "-- appending region:Nkx2-1-30 with 125 probes\n",
      "-- appending region:Nkx2-1-31 with 94 probes\n",
      "-- appending region:Nkx2-1-32 with 102 probes\n",
      "-- appending region:Nkx2-1-33 with 135 probes\n",
      "-- appending region:Nkx2-1-34 with 149 probes\n",
      "-- appending region:Nkx2-1-35 with 154 probes\n",
      "-- appending region:Nkx2-1-36 with 135 probes\n",
      "-- appending region:Nkx2-1-37 with 133 probes\n",
      "-- appending region:Nkx2-1-38 with 130 probes\n",
      "-- appending region:Nkx2-1-39 with 125 probes\n",
      "-- appending region:Nkx2-1-4 with 137 probes\n",
      "-- appending region:Nkx2-1-40 with 58 probes\n",
      "-- appending region:Nkx2-1-41 with 101 probes\n",
      "-- appending region:Nkx2-1-42 with 118 probes\n",
      "-- appending region:Nkx2-1-43 with 123 probes\n",
      "-- appending region:Nkx2-1-44 with 112 probes\n",
      "-- appending region:Nkx2-1-45 with 133 probes\n",
      "-- appending region:Nkx2-1-46 with 138 probes\n",
      "-- appending region:Nkx2-1-47 with 130 probes\n",
      "-- appending region:Nkx2-1-48 with 132 probes\n",
      "-- appending region:Nkx2-1-49 with 100 probes\n",
      "-- appending region:Nkx2-1-5 with 120 probes\n",
      "-- appending region:Nkx2-1-50 with 90 probes\n",
      "-- appending region:Nkx2-1-51 with 122 probes\n",
      "-- appending region:Nkx2-1-52 with 139 probes\n",
      "-- appending region:Nkx2-1-53 with 123 probes\n",
      "-- appending region:Nkx2-1-54 with 128 probes\n",
      "-- appending region:Nkx2-1-55 with 138 probes\n",
      "-- appending region:Nkx2-1-56 with 137 probes\n",
      "-- appending region:Nkx2-1-57 with 100 probes\n",
      "-- appending region:Nkx2-1-58 with 122 probes\n",
      "-- appending region:Nkx2-1-59 with 125 probes\n",
      "-- appending region:Nkx2-1-6 with 141 probes\n",
      "-- appending region:Nkx2-1-60 with 86 probes\n",
      "-- appending region:Nkx2-1-7 with 138 probes\n",
      "-- appending region:Nkx2-1-8 with 118 probes\n",
      "-- appending region:Nkx2-1-9 with 126 probes\n",
      "-- appending region:Slc17a7-1 with 104 probes\n",
      "-- appending region:Slc17a7-10 with 112 probes\n",
      "-- appending region:Slc17a7-11 with 137 probes\n",
      "-- appending region:Slc17a7-12 with 148 probes\n",
      "-- appending region:Slc17a7-13 with 164 probes\n",
      "-- appending region:Slc17a7-14 with 123 probes\n",
      "-- appending region:Slc17a7-15 with 140 probes\n",
      "-- appending region:Slc17a7-16 with 144 probes\n",
      "-- appending region:Slc17a7-17 with 148 probes\n",
      "-- appending region:Slc17a7-18 with 126 probes\n",
      "-- appending region:Slc17a7-19 with 114 probes\n",
      "-- appending region:Slc17a7-2 with 99 probes\n",
      "-- appending region:Slc17a7-20 with 94 probes\n",
      "-- appending region:Slc17a7-21 with 92 probes\n",
      "-- appending region:Slc17a7-22 with 130 probes\n",
      "-- appending region:Slc17a7-23 with 119 probes\n",
      "-- appending region:Slc17a7-24 with 127 probes\n",
      "-- appending region:Slc17a7-25 with 122 probes\n",
      "-- appending region:Slc17a7-26 with 134 probes\n",
      "-- appending region:Slc17a7-27 with 147 probes\n",
      "-- appending region:Slc17a7-28 with 153 probes\n",
      "-- appending region:Slc17a7-29 with 127 probes\n",
      "-- appending region:Slc17a7-3 with 137 probes\n",
      "-- appending region:Slc17a7-30 with 133 probes\n",
      "-- appending region:Slc17a7-31 with 140 probes\n",
      "-- appending region:Slc17a7-32 with 141 probes\n",
      "-- appending region:Slc17a7-33 with 112 probes\n",
      "-- appending region:Slc17a7-34 with 125 probes\n",
      "-- appending region:Slc17a7-35 with 102 probes\n",
      "-- appending region:Slc17a7-36 with 106 probes\n",
      "-- appending region:Slc17a7-37 with 99 probes\n",
      "-- appending region:Slc17a7-38 with 93 probes\n",
      "-- appending region:Slc17a7-39 with 145 probes\n",
      "-- appending region:Slc17a7-4 with 110 probes\n",
      "-- appending region:Slc17a7-40 with 121 probes\n",
      "-- appending region:Slc17a7-41 with 134 probes\n",
      "-- appending region:Slc17a7-42 with 130 probes\n",
      "-- appending region:Slc17a7-43 with 132 probes\n",
      "-- appending region:Slc17a7-44 with 141 probes\n",
      "-- appending region:Slc17a7-45 with 145 probes\n",
      "-- appending region:Slc17a7-46 with 132 probes\n",
      "-- appending region:Slc17a7-47 with 111 probes\n",
      "-- appending region:Slc17a7-48 with 129 probes\n",
      "-- appending region:Slc17a7-49 with 130 probes\n",
      "-- appending region:Slc17a7-5 with 124 probes\n",
      "-- appending region:Slc17a7-50 with 135 probes\n",
      "-- appending region:Slc17a7-51 with 65 probes\n",
      "-- appending region:Slc17a7-52 with 90 probes\n",
      "-- appending region:Slc17a7-53 with 125 probes\n",
      "-- appending region:Slc17a7-54 with 120 probes\n",
      "-- appending region:Slc17a7-55 with 132 probes\n",
      "-- appending region:Slc17a7-56 with 114 probes\n",
      "-- appending region:Slc17a7-57 with 130 probes\n",
      "-- appending region:Slc17a7-58 with 121 probes\n",
      "-- appending region:Slc17a7-59 with 102 probes\n",
      "-- appending region:Slc17a7-6 with 53 probes\n",
      "-- appending region:Slc17a7-60 with 142 probes\n",
      "-- appending region:Slc17a7-7 with 66 probes\n",
      "-- appending region:Slc17a7-8 with 141 probes\n",
      "-- appending region:Slc17a7-9 with 146 probes\n",
      "-- appending region:Slc32a1-1 with 129 probes\n",
      "-- appending region:Slc32a1-10 with 128 probes\n",
      "-- appending region:Slc32a1-11 with 144 probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- appending region:Slc32a1-12 with 129 probes\n",
      "-- appending region:Slc32a1-13 with 144 probes\n",
      "-- appending region:Slc32a1-14 with 121 probes\n",
      "-- appending region:Slc32a1-15 with 127 probes\n",
      "-- appending region:Slc32a1-16 with 114 probes\n",
      "-- appending region:Slc32a1-17 with 141 probes\n",
      "-- appending region:Slc32a1-18 with 127 probes\n",
      "-- appending region:Slc32a1-19 with 136 probes\n",
      "-- appending region:Slc32a1-2 with 138 probes\n",
      "-- appending region:Slc32a1-20 with 135 probes\n",
      "-- appending region:Slc32a1-21 with 144 probes\n",
      "-- appending region:Slc32a1-3 with 129 probes\n",
      "-- appending region:Slc32a1-4 with 123 probes\n",
      "-- appending region:Slc32a1-5 with 134 probes\n",
      "-- appending region:Slc32a1-6 with 131 probes\n",
      "-- appending region:Slc32a1-7 with 123 probes\n",
      "-- appending region:Slc32a1-8 with 114 probes\n",
      "-- appending region:Slc32a1-9 with 123 probes\n",
      "-- appending region:SST-1 with 105 probes\n",
      "-- appending region:SST-10 with 141 probes\n",
      "-- appending region:SST-11 with 145 probes\n",
      "-- appending region:SST-12 with 141 probes\n",
      "-- appending region:SST-13 with 79 probes\n",
      "-- appending region:SST-14 with 95 probes\n",
      "-- appending region:SST-15 with 82 probes\n",
      "-- appending region:SST-16 with 131 probes\n",
      "-- appending region:SST-17 with 151 probes\n",
      "-- appending region:SST-18 with 96 probes\n",
      "-- appending region:SST-19 with 114 probes\n",
      "-- appending region:SST-2 with 49 probes\n",
      "-- appending region:SST-20 with 114 probes\n",
      "-- appending region:SST-21 with 103 probes\n",
      "-- appending region:SST-22 with 61 probes\n",
      "-- appending region:SST-23 with 112 probes\n",
      "-- appending region:SST-24 with 148 probes\n",
      "-- appending region:SST-25 with 140 probes\n",
      "-- appending region:SST-26 with 151 probes\n",
      "-- appending region:SST-27 with 141 probes\n",
      "-- appending region:SST-28 with 137 probes\n",
      "-- appending region:SST-29 with 124 probes\n",
      "-- appending region:SST-3 with 53 probes\n",
      "-- appending region:SST-30 with 141 probes\n",
      "-- appending region:SST-31 with 108 probes\n",
      "-- appending region:SST-32 with 139 probes\n",
      "-- appending region:SST-33 with 150 probes\n",
      "-- appending region:SST-34 with 148 probes\n",
      "-- appending region:SST-35 with 141 probes\n",
      "-- appending region:SST-36 with 144 probes\n",
      "-- appending region:SST-37 with 147 probes\n",
      "-- appending region:SST-38 with 114 probes\n",
      "-- appending region:SST-39 with 112 probes\n",
      "-- appending region:SST-4 with 118 probes\n",
      "-- appending region:SST-40 with 135 probes\n",
      "-- appending region:SST-41 with 147 probes\n",
      "-- appending region:SST-42 with 141 probes\n",
      "-- appending region:SST-43 with 139 probes\n",
      "-- appending region:SST-44 with 146 probes\n",
      "-- appending region:SST-45 with 139 probes\n",
      "-- appending region:SST-46 with 138 probes\n",
      "-- appending region:SST-47 with 130 probes\n",
      "-- appending region:SST-48 with 161 probes\n",
      "-- appending region:SST-49 with 151 probes\n",
      "-- appending region:SST-5 with 114 probes\n",
      "-- appending region:SST-50 with 135 probes\n",
      "-- appending region:SST-51 with 147 probes\n",
      "-- appending region:SST-52 with 116 probes\n",
      "-- appending region:SST-53 with 124 probes\n",
      "-- appending region:SST-54 with 101 probes\n",
      "-- appending region:SST-55 with 122 probes\n",
      "-- appending region:SST-56 with 116 probes\n",
      "-- appending region:SST-57 with 117 probes\n",
      "-- appending region:SST-58 with 108 probes\n",
      "-- appending region:SST-59 with 134 probes\n",
      "-- appending region:SST-6 with 149 probes\n",
      "-- appending region:SST-60 with 113 probes\n",
      "-- appending region:SST-7 with 143 probes\n",
      "-- appending region:SST-8 with 130 probes\n",
      "-- appending region:SST-9 with 145 probes\n"
     ]
    }
   ],
   "source": [
    "verbose=True\n",
    "cand_prob_nums = []\n",
    "pb_dic = {}\n",
    "# Parse segemented list\n",
    "_save_file = os.path.join(report_folder, f'merged_probes_{int(resolution/1000)}kb.pbr')\n",
    "_pbde = library_tools.design.pb_reports_class()\n",
    "_pbde.load_from_file(_save_file)\n",
    "\n",
    "reg_inds = np.sort(np.unique([_v['reg_index'] for _s, _v in _pbde.kept_probes.items()]))\n",
    "for _rid in reg_inds:\n",
    "    _sel_pbs = {_s:_v for _s, _v in _pbde.kept_probes.items() if _v['reg_index']==_rid}\n",
    "\n",
    "    if len(_sel_pbs) > 0:\n",
    "        _region_gid = list(_sel_pbs.values())[0]['reg_name'].split('_')[-1]\n",
    "        cand_prob_nums.append(len(_sel_pbs))\n",
    "        if verbose:\n",
    "            print(f\"-- appending region:{_region_gid} with {len(_sel_pbs)} probes\")\n",
    "        _splitted_pbde = ld.pb_reports_class()\n",
    "        _splitted_pbde.kept_probes = _sel_pbs\n",
    "        pb_dic[_region_gid] = _splitted_pbde"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Screen probes by genome counts.\n",
      "-- filtering region:Ccnd2-1, directly append 135 probes\n",
      "-- filtering region:Ccnd2-10, directly append 136 probes\n",
      "-- filtering region:Ccnd2-11, directly append 139 probes\n",
      "-- filtering region:Ccnd2-12, directly append 62 probes\n",
      "-- filtering region:Ccnd2-13, directly append 127 probes\n",
      "-- filtering region:Ccnd2-14, directly append 117 probes\n",
      "-- filtering region:Ccnd2-15, directly append 125 probes\n",
      "-- filtering region:Ccnd2-16, directly append 136 probes\n",
      "-- filtering region:Ccnd2-17, directly append 147 probes\n",
      "-- filtering region:Ccnd2-18, directly append 146 probes\n",
      "-- filtering region:Ccnd2-19, directly append 143 probes\n",
      "-- filtering region:Ccnd2-2, directly append 99 probes\n",
      "-- filtering region:Ccnd2-20, directly append 142 probes\n",
      "-- filtering region:Ccnd2-21, directly append 139 probes\n",
      "-- filtering region:Ccnd2-22, directly append 126 probes\n",
      "-- filtering region:Ccnd2-23, directly append 138 probes\n",
      "-- filtering region:Ccnd2-24, directly append 136 probes\n",
      "-- filtering region:Ccnd2-25, directly append 128 probes\n",
      "-- filtering region:Ccnd2-26, directly append 101 probes\n",
      "-- filtering region:Ccnd2-27, directly append 139 probes\n",
      "-- filtering region:Ccnd2-28, directly append 128 probes\n",
      "-- filtering region:Ccnd2-29, directly append 145 probes\n",
      "-- filtering region:Ccnd2-3, directly append 104 probes\n",
      "-- filtering region:Ccnd2-30, directly append 125 probes\n",
      "-- filtering region:Ccnd2-31, directly append 124 probes\n",
      "-- filtering region:Ccnd2-32, directly append 135 probes\n",
      "-- filtering region:Ccnd2-33, directly append 127 probes\n",
      "-- filtering region:Ccnd2-34, directly append 133 probes\n",
      "-- filtering region:Ccnd2-35, directly append 149 probes\n",
      "-- filtering region:Ccnd2-36, directly append 138 probes\n",
      "-- filtering region:Ccnd2-37, directly append 118 probes\n",
      "-- filtering region:Ccnd2-38, directly append 145 probes\n",
      "-- filtering region:Ccnd2-39, directly append 119 probes\n",
      "-- filtering region:Ccnd2-4, directly append 149 probes\n",
      "-- filtering region:Ccnd2-40, directly append 135 probes\n",
      "-- filtering region:Ccnd2-41, directly append 124 probes\n",
      "-- filtering region:Ccnd2-42, directly append 117 probes\n",
      "-- filtering region:Ccnd2-43, directly append 126 probes\n",
      "-- filtering region:Ccnd2-44, directly append 128 probes\n",
      "-- filtering region:Ccnd2-45, directly append 128 probes\n",
      "-- filtering region:Ccnd2-46, directly append 130 probes\n",
      "-- filtering region:Ccnd2-47, directly append 142 probes\n",
      "-- filtering region:Ccnd2-48, directly append 135 probes\n",
      "-- filtering region:Ccnd2-49, directly append 128 probes\n",
      "-- filtering region:Ccnd2-5, directly append 152 probes\n",
      "-- filtering region:Ccnd2-50, directly append 133 probes\n",
      "-- filtering region:Ccnd2-51, directly append 99 probes\n",
      "-- filtering region:Ccnd2-52, directly append 136 probes\n",
      "-- filtering region:Ccnd2-53, directly append 132 probes\n",
      "-- filtering region:Ccnd2-54, directly append 34 probes\n",
      "-- filtering region:Ccnd2-55, directly append 101 probes\n",
      "-- filtering region:Ccnd2-56, directly append 116 probes\n",
      "-- filtering region:Ccnd2-57, directly append 134 probes\n",
      "-- filtering region:Ccnd2-58, directly append 142 probes\n",
      "-- filtering region:Ccnd2-59, directly append 147 probes\n",
      "-- filtering region:Ccnd2-6, directly append 112 probes\n",
      "-- filtering region:Ccnd2-60, directly append 143 probes\n",
      "-- filtering region:Ccnd2-7, directly append 136 probes\n",
      "-- filtering region:Ccnd2-8, directly append 140 probes\n",
      "-- filtering region:Ccnd2-9, directly append 134 probes\n",
      "-- filtering region:Nkx2-1-1, directly append 79 probes\n",
      "-- filtering region:Nkx2-1-10, directly append 129 probes\n",
      "-- filtering region:Nkx2-1-11, directly append 128 probes\n",
      "-- filtering region:Nkx2-1-12, directly append 118 probes\n",
      "-- filtering region:Nkx2-1-13, directly append 134 probes\n",
      "-- filtering region:Nkx2-1-14, directly append 129 probes\n",
      "-- filtering region:Nkx2-1-15, directly append 129 probes\n",
      "-- filtering region:Nkx2-1-16, directly append 125 probes\n",
      "-- filtering region:Nkx2-1-17, directly append 138 probes\n",
      "-- filtering region:Nkx2-1-18, directly append 146 probes\n",
      "-- filtering region:Nkx2-1-19, directly append 153 probes\n",
      "-- filtering region:Nkx2-1-2, directly append 62 probes\n",
      "-- filtering region:Nkx2-1-20, directly append 123 probes\n",
      "-- filtering region:Nkx2-1-21, directly append 100 probes\n",
      "-- filtering region:Nkx2-1-22, directly append 53 probes\n",
      "-- filtering region:Nkx2-1-23, directly append 87 probes\n",
      "-- filtering region:Nkx2-1-24, directly append 76 probes\n",
      "-- filtering region:Nkx2-1-25, directly append 111 probes\n",
      "-- filtering region:Nkx2-1-26, directly append 121 probes\n",
      "-- filtering region:Nkx2-1-27, directly append 144 probes\n",
      "-- filtering region:Nkx2-1-28, directly append 116 probes\n",
      "-- filtering region:Nkx2-1-29, directly append 141 probes\n",
      "-- filtering region:Nkx2-1-3, directly append 86 probes\n",
      "-- filtering region:Nkx2-1-30, directly append 125 probes\n",
      "-- filtering region:Nkx2-1-31, directly append 94 probes\n",
      "-- filtering region:Nkx2-1-32, directly append 102 probes\n",
      "-- filtering region:Nkx2-1-33, directly append 135 probes\n",
      "-- filtering region:Nkx2-1-34, directly append 149 probes\n",
      "-- filtering region:Nkx2-1-35, directly append 154 probes\n",
      "-- filtering region:Nkx2-1-36, directly append 135 probes\n",
      "-- filtering region:Nkx2-1-37, directly append 133 probes\n",
      "-- filtering region:Nkx2-1-38, directly append 130 probes\n",
      "-- filtering region:Nkx2-1-39, directly append 125 probes\n",
      "-- filtering region:Nkx2-1-4, directly append 137 probes\n",
      "-- filtering region:Nkx2-1-40, directly append 58 probes\n",
      "-- filtering region:Nkx2-1-41, directly append 101 probes\n",
      "-- filtering region:Nkx2-1-42, directly append 118 probes\n",
      "-- filtering region:Nkx2-1-43, directly append 123 probes\n",
      "-- filtering region:Nkx2-1-44, directly append 112 probes\n",
      "-- filtering region:Nkx2-1-45, directly append 133 probes\n",
      "-- filtering region:Nkx2-1-46, directly append 138 probes\n",
      "-- filtering region:Nkx2-1-47, directly append 130 probes\n",
      "-- filtering region:Nkx2-1-48, directly append 132 probes\n",
      "-- filtering region:Nkx2-1-49, directly append 100 probes\n",
      "-- filtering region:Nkx2-1-5, directly append 120 probes\n",
      "-- filtering region:Nkx2-1-50, directly append 90 probes\n",
      "-- filtering region:Nkx2-1-51, directly append 122 probes\n",
      "-- filtering region:Nkx2-1-52, directly append 139 probes\n",
      "-- filtering region:Nkx2-1-53, directly append 123 probes\n",
      "-- filtering region:Nkx2-1-54, directly append 128 probes\n",
      "-- filtering region:Nkx2-1-55, directly append 138 probes\n",
      "-- filtering region:Nkx2-1-56, directly append 137 probes\n",
      "-- filtering region:Nkx2-1-57, directly append 100 probes\n",
      "-- filtering region:Nkx2-1-58, directly append 122 probes\n",
      "-- filtering region:Nkx2-1-59, directly append 125 probes\n",
      "-- filtering region:Nkx2-1-6, directly append 141 probes\n",
      "-- filtering region:Nkx2-1-60, directly append 86 probes\n",
      "-- filtering region:Nkx2-1-7, directly append 138 probes\n",
      "-- filtering region:Nkx2-1-8, directly append 118 probes\n",
      "-- filtering region:Nkx2-1-9, directly append 126 probes\n",
      "-- filtering region:Slc17a7-1, directly append 104 probes\n",
      "-- filtering region:Slc17a7-10, directly append 112 probes\n",
      "-- filtering region:Slc17a7-11, directly append 137 probes\n",
      "-- filtering region:Slc17a7-12, directly append 148 probes\n",
      "-- filtering region:Slc17a7-13, screen genome to keep 160 probes\n",
      "-- filtering region:Slc17a7-14, directly append 123 probes\n",
      "-- filtering region:Slc17a7-15, directly append 140 probes\n",
      "-- filtering region:Slc17a7-16, directly append 144 probes\n",
      "-- filtering region:Slc17a7-17, directly append 148 probes\n",
      "-- filtering region:Slc17a7-18, directly append 126 probes\n",
      "-- filtering region:Slc17a7-19, directly append 114 probes\n",
      "-- filtering region:Slc17a7-2, directly append 99 probes\n",
      "-- filtering region:Slc17a7-20, directly append 94 probes\n",
      "-- filtering region:Slc17a7-21, directly append 92 probes\n",
      "-- filtering region:Slc17a7-22, directly append 130 probes\n",
      "-- filtering region:Slc17a7-23, directly append 119 probes\n",
      "-- filtering region:Slc17a7-24, directly append 127 probes\n",
      "-- filtering region:Slc17a7-25, directly append 122 probes\n",
      "-- filtering region:Slc17a7-26, directly append 134 probes\n",
      "-- filtering region:Slc17a7-27, directly append 147 probes\n",
      "-- filtering region:Slc17a7-28, directly append 153 probes\n",
      "-- filtering region:Slc17a7-29, directly append 127 probes\n",
      "-- filtering region:Slc17a7-3, directly append 137 probes\n",
      "-- filtering region:Slc17a7-30, directly append 133 probes\n",
      "-- filtering region:Slc17a7-31, directly append 140 probes\n",
      "-- filtering region:Slc17a7-32, directly append 141 probes\n",
      "-- filtering region:Slc17a7-33, directly append 112 probes\n",
      "-- filtering region:Slc17a7-34, directly append 125 probes\n",
      "-- filtering region:Slc17a7-35, directly append 102 probes\n",
      "-- filtering region:Slc17a7-36, directly append 106 probes\n",
      "-- filtering region:Slc17a7-37, directly append 99 probes\n",
      "-- filtering region:Slc17a7-38, directly append 93 probes\n",
      "-- filtering region:Slc17a7-39, directly append 145 probes\n",
      "-- filtering region:Slc17a7-4, directly append 110 probes\n",
      "-- filtering region:Slc17a7-40, directly append 121 probes\n",
      "-- filtering region:Slc17a7-41, directly append 134 probes\n",
      "-- filtering region:Slc17a7-42, directly append 130 probes\n",
      "-- filtering region:Slc17a7-43, directly append 132 probes\n",
      "-- filtering region:Slc17a7-44, directly append 141 probes\n",
      "-- filtering region:Slc17a7-45, directly append 145 probes\n",
      "-- filtering region:Slc17a7-46, directly append 132 probes\n",
      "-- filtering region:Slc17a7-47, directly append 111 probes\n",
      "-- filtering region:Slc17a7-48, directly append 129 probes\n",
      "-- filtering region:Slc17a7-49, directly append 130 probes\n",
      "-- filtering region:Slc17a7-5, directly append 124 probes\n",
      "-- filtering region:Slc17a7-50, directly append 135 probes\n",
      "-- filtering region:Slc17a7-51, directly append 65 probes\n",
      "-- filtering region:Slc17a7-52, directly append 90 probes\n",
      "-- filtering region:Slc17a7-53, directly append 125 probes\n",
      "-- filtering region:Slc17a7-54, directly append 120 probes\n",
      "-- filtering region:Slc17a7-55, directly append 132 probes\n",
      "-- filtering region:Slc17a7-56, directly append 114 probes\n",
      "-- filtering region:Slc17a7-57, directly append 130 probes\n",
      "-- filtering region:Slc17a7-58, directly append 121 probes\n",
      "-- filtering region:Slc17a7-59, directly append 102 probes\n",
      "-- filtering region:Slc17a7-6, directly append 53 probes\n",
      "-- filtering region:Slc17a7-60, directly append 142 probes\n",
      "-- filtering region:Slc17a7-7, directly append 66 probes\n",
      "-- filtering region:Slc17a7-8, directly append 141 probes\n",
      "-- filtering region:Slc17a7-9, directly append 146 probes\n",
      "-- filtering region:Slc32a1-1, directly append 129 probes\n",
      "-- filtering region:Slc32a1-10, directly append 128 probes\n",
      "-- filtering region:Slc32a1-11, directly append 144 probes\n",
      "-- filtering region:Slc32a1-12, directly append 129 probes\n",
      "-- filtering region:Slc32a1-13, directly append 144 probes\n",
      "-- filtering region:Slc32a1-14, directly append 121 probes\n",
      "-- filtering region:Slc32a1-15, directly append 127 probes\n",
      "-- filtering region:Slc32a1-16, directly append 114 probes\n",
      "-- filtering region:Slc32a1-17, directly append 141 probes\n",
      "-- filtering region:Slc32a1-18, directly append 127 probes\n",
      "-- filtering region:Slc32a1-19, directly append 136 probes\n",
      "-- filtering region:Slc32a1-2, directly append 138 probes\n",
      "-- filtering region:Slc32a1-20, directly append 135 probes\n",
      "-- filtering region:Slc32a1-21, directly append 144 probes\n",
      "-- filtering region:Slc32a1-3, directly append 129 probes\n",
      "-- filtering region:Slc32a1-4, directly append 123 probes\n",
      "-- filtering region:Slc32a1-5, directly append 134 probes\n",
      "-- filtering region:Slc32a1-6, directly append 131 probes\n",
      "-- filtering region:Slc32a1-7, directly append 123 probes\n",
      "-- filtering region:Slc32a1-8, directly append 114 probes\n",
      "-- filtering region:Slc32a1-9, directly append 123 probes\n",
      "-- filtering region:SST-1, directly append 105 probes\n",
      "-- filtering region:SST-10, directly append 141 probes\n",
      "-- filtering region:SST-11, directly append 145 probes\n",
      "-- filtering region:SST-12, directly append 141 probes\n",
      "-- filtering region:SST-13, directly append 79 probes\n",
      "-- filtering region:SST-14, directly append 95 probes\n",
      "-- filtering region:SST-15, directly append 82 probes\n",
      "-- filtering region:SST-16, directly append 131 probes\n",
      "-- filtering region:SST-17, directly append 151 probes\n",
      "-- filtering region:SST-18, directly append 96 probes\n",
      "-- filtering region:SST-19, directly append 114 probes\n",
      "-- filtering region:SST-2, directly append 49 probes\n",
      "-- filtering region:SST-20, directly append 114 probes\n",
      "-- filtering region:SST-21, directly append 103 probes\n",
      "-- filtering region:SST-22, directly append 61 probes\n",
      "-- filtering region:SST-23, directly append 112 probes\n",
      "-- filtering region:SST-24, directly append 148 probes\n",
      "-- filtering region:SST-25, directly append 140 probes\n",
      "-- filtering region:SST-26, directly append 151 probes\n",
      "-- filtering region:SST-27, directly append 141 probes\n",
      "-- filtering region:SST-28, directly append 137 probes\n",
      "-- filtering region:SST-29, directly append 124 probes\n",
      "-- filtering region:SST-3, directly append 53 probes\n",
      "-- filtering region:SST-30, directly append 141 probes\n",
      "-- filtering region:SST-31, directly append 108 probes\n",
      "-- filtering region:SST-32, directly append 139 probes\n",
      "-- filtering region:SST-33, directly append 150 probes\n",
      "-- filtering region:SST-34, directly append 148 probes\n",
      "-- filtering region:SST-35, directly append 141 probes\n",
      "-- filtering region:SST-36, directly append 144 probes\n",
      "-- filtering region:SST-37, directly append 147 probes\n",
      "-- filtering region:SST-38, directly append 114 probes\n",
      "-- filtering region:SST-39, directly append 112 probes\n",
      "-- filtering region:SST-4, directly append 118 probes\n",
      "-- filtering region:SST-40, directly append 135 probes\n",
      "-- filtering region:SST-41, directly append 147 probes\n",
      "-- filtering region:SST-42, directly append 141 probes\n",
      "-- filtering region:SST-43, directly append 139 probes\n",
      "-- filtering region:SST-44, directly append 146 probes\n",
      "-- filtering region:SST-45, directly append 139 probes\n",
      "-- filtering region:SST-46, directly append 138 probes\n",
      "-- filtering region:SST-47, directly append 130 probes\n",
      "-- filtering region:SST-48, screen genome to keep 160 probes\n",
      "-- filtering region:SST-49, directly append 151 probes\n",
      "-- filtering region:SST-5, directly append 114 probes\n",
      "-- filtering region:SST-50, directly append 135 probes\n",
      "-- filtering region:SST-51, directly append 147 probes\n",
      "-- filtering region:SST-52, directly append 116 probes\n",
      "-- filtering region:SST-53, directly append 124 probes\n",
      "-- filtering region:SST-54, directly append 101 probes\n",
      "-- filtering region:SST-55, directly append 122 probes\n",
      "-- filtering region:SST-56, directly append 116 probes\n",
      "-- filtering region:SST-57, directly append 117 probes\n",
      "-- filtering region:SST-58, directly append 108 probes\n",
      "-- filtering region:SST-59, directly append 134 probes\n",
      "-- filtering region:SST-6, directly append 149 probes\n",
      "-- filtering region:SST-60, directly append 113 probes\n",
      "-- filtering region:SST-7, directly append 143 probes\n",
      "-- filtering region:SST-8, directly append 130 probes\n",
      "-- filtering region:SST-9, directly append 145 probes\n"
     ]
    }
   ],
   "source": [
    "# filter by genome counts\n",
    "reload(library_tools.quality_check)\n",
    "\n",
    "num_probes_kept = 160 # max num of probes kept for each region\n",
    "filtered_pb_dic = library_tools.quality_check.Screen_probe_by_hit(library_folder, pb_dic, num_probes_kept,\n",
    "                                                                  hit_type='genome')\n",
    "\n",
    "## filter by minimum number of probes in each region\n",
    "min_num_probes = 25 \n",
    "removed_regions = []\n",
    "for _rid, _pb_class in filtered_pb_dic.items():\n",
    "    if len(_pb_class.kept_probes) < min_num_probes:\n",
    "        print(f\"region: {_rid} removed with {len(_pb_class.kept_probes)} probes\")\n",
    "        removed_regions.append(_rid)\n",
    "for _rid in removed_regions:\n",
    "    del(filtered_pb_dic[_rid])"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 2.4 check candidate probe number per region"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "/* Put everything inside the global mpl namespace */\n",
       "/* global mpl */\n",
       "window.mpl = {};\n",
       "\n",
       "mpl.get_websocket_type = function () {\n",
       "    if (typeof WebSocket !== 'undefined') {\n",
       "        return WebSocket;\n",
       "    } else if (typeof MozWebSocket !== 'undefined') {\n",
       "        return MozWebSocket;\n",
       "    } else {\n",
       "        alert(\n",
       "            'Your browser does not have WebSocket support. ' +\n",
       "                'Please try Chrome, Safari or Firefox ≥ 6. ' +\n",
       "                'Firefox 4 and 5 are also supported but you ' +\n",
       "                'have to enable WebSockets in about:config.'\n",
       "        );\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure = function (figure_id, websocket, ondownload, parent_element) {\n",
       "    this.id = figure_id;\n",
       "\n",
       "    this.ws = websocket;\n",
       "\n",
       "    this.supports_binary = this.ws.binaryType !== undefined;\n",
       "\n",
       "    if (!this.supports_binary) {\n",
       "        var warnings = document.getElementById('mpl-warnings');\n",
       "        if (warnings) {\n",
       "            warnings.style.display = 'block';\n",
       "            warnings.textContent =\n",
       "                'This browser does not support binary websocket messages. ' +\n",
       "                'Performance may be slow.';\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.imageObj = new Image();\n",
       "\n",
       "    this.context = undefined;\n",
       "    this.message = undefined;\n",
       "    this.canvas = undefined;\n",
       "    this.rubberband_canvas = undefined;\n",
       "    this.rubberband_context = undefined;\n",
       "    this.format_dropdown = undefined;\n",
       "\n",
       "    this.image_mode = 'full';\n",
       "\n",
       "    this.root = document.createElement('div');\n",
       "    this.root.setAttribute('style', 'display: inline-block');\n",
       "    this._root_extra_style(this.root);\n",
       "\n",
       "    parent_element.appendChild(this.root);\n",
       "\n",
       "    this._init_header(this);\n",
       "    this._init_canvas(this);\n",
       "    this._init_toolbar(this);\n",
       "\n",
       "    var fig = this;\n",
       "\n",
       "    this.waiting = false;\n",
       "\n",
       "    this.ws.onopen = function () {\n",
       "        fig.send_message('supports_binary', { value: fig.supports_binary });\n",
       "        fig.send_message('send_image_mode', {});\n",
       "        if (fig.ratio !== 1) {\n",
       "            fig.send_message('set_dpi_ratio', { dpi_ratio: fig.ratio });\n",
       "        }\n",
       "        fig.send_message('refresh', {});\n",
       "    };\n",
       "\n",
       "    this.imageObj.onload = function () {\n",
       "        if (fig.image_mode === 'full') {\n",
       "            // Full images could contain transparency (where diff images\n",
       "            // almost always do), so we need to clear the canvas so that\n",
       "            // there is no ghosting.\n",
       "            fig.context.clearRect(0, 0, fig.canvas.width, fig.canvas.height);\n",
       "        }\n",
       "        fig.context.drawImage(fig.imageObj, 0, 0);\n",
       "    };\n",
       "\n",
       "    this.imageObj.onunload = function () {\n",
       "        fig.ws.close();\n",
       "    };\n",
       "\n",
       "    this.ws.onmessage = this._make_on_message_function(this);\n",
       "\n",
       "    this.ondownload = ondownload;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_header = function () {\n",
       "    var titlebar = document.createElement('div');\n",
       "    titlebar.classList =\n",
       "        'ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix';\n",
       "    var titletext = document.createElement('div');\n",
       "    titletext.classList = 'ui-dialog-title';\n",
       "    titletext.setAttribute(\n",
       "        'style',\n",
       "        'width: 100%; text-align: center; padding: 3px;'\n",
       "    );\n",
       "    titlebar.appendChild(titletext);\n",
       "    this.root.appendChild(titlebar);\n",
       "    this.header = titletext;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._init_canvas = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var canvas_div = (this.canvas_div = document.createElement('div'));\n",
       "    canvas_div.setAttribute(\n",
       "        'style',\n",
       "        'border: 1px solid #ddd;' +\n",
       "            'box-sizing: content-box;' +\n",
       "            'clear: both;' +\n",
       "            'min-height: 1px;' +\n",
       "            'min-width: 1px;' +\n",
       "            'outline: 0;' +\n",
       "            'overflow: hidden;' +\n",
       "            'position: relative;' +\n",
       "            'resize: both;'\n",
       "    );\n",
       "\n",
       "    function on_keyboard_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.key_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    canvas_div.addEventListener(\n",
       "        'keydown',\n",
       "        on_keyboard_event_closure('key_press')\n",
       "    );\n",
       "    canvas_div.addEventListener(\n",
       "        'keyup',\n",
       "        on_keyboard_event_closure('key_release')\n",
       "    );\n",
       "\n",
       "    this._canvas_extra_style(canvas_div);\n",
       "    this.root.appendChild(canvas_div);\n",
       "\n",
       "    var canvas = (this.canvas = document.createElement('canvas'));\n",
       "    canvas.classList.add('mpl-canvas');\n",
       "    canvas.setAttribute('style', 'box-sizing: content-box;');\n",
       "\n",
       "    this.context = canvas.getContext('2d');\n",
       "\n",
       "    var backingStore =\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        this.context.webkitBackingStorePixelRatio ||\n",
       "        this.context.mozBackingStorePixelRatio ||\n",
       "        this.context.msBackingStorePixelRatio ||\n",
       "        this.context.oBackingStorePixelRatio ||\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        1;\n",
       "\n",
       "    this.ratio = (window.devicePixelRatio || 1) / backingStore;\n",
       "\n",
       "    var rubberband_canvas = (this.rubberband_canvas = document.createElement(\n",
       "        'canvas'\n",
       "    ));\n",
       "    rubberband_canvas.setAttribute(\n",
       "        'style',\n",
       "        'box-sizing: content-box; position: absolute; left: 0; top: 0; z-index: 1;'\n",
       "    );\n",
       "\n",
       "    // Apply a ponyfill if ResizeObserver is not implemented by browser.\n",
       "    if (this.ResizeObserver === undefined) {\n",
       "        if (window.ResizeObserver !== undefined) {\n",
       "            this.ResizeObserver = window.ResizeObserver;\n",
       "        } else {\n",
       "            var obs = _JSXTOOLS_RESIZE_OBSERVER({});\n",
       "            this.ResizeObserver = obs.ResizeObserver;\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.resizeObserverInstance = new this.ResizeObserver(function (entries) {\n",
       "        var nentries = entries.length;\n",
       "        for (var i = 0; i < nentries; i++) {\n",
       "            var entry = entries[i];\n",
       "            var width, height;\n",
       "            if (entry.contentBoxSize) {\n",
       "                if (entry.contentBoxSize instanceof Array) {\n",
       "                    // Chrome 84 implements new version of spec.\n",
       "                    width = entry.contentBoxSize[0].inlineSize;\n",
       "                    height = entry.contentBoxSize[0].blockSize;\n",
       "                } else {\n",
       "                    // Firefox implements old version of spec.\n",
       "                    width = entry.contentBoxSize.inlineSize;\n",
       "                    height = entry.contentBoxSize.blockSize;\n",
       "                }\n",
       "            } else {\n",
       "                // Chrome <84 implements even older version of spec.\n",
       "                width = entry.contentRect.width;\n",
       "                height = entry.contentRect.height;\n",
       "            }\n",
       "\n",
       "            // Keep the size of the canvas and rubber band canvas in sync with\n",
       "            // the canvas container.\n",
       "            if (entry.devicePixelContentBoxSize) {\n",
       "                // Chrome 84 implements new version of spec.\n",
       "                canvas.setAttribute(\n",
       "                    'width',\n",
       "                    entry.devicePixelContentBoxSize[0].inlineSize\n",
       "                );\n",
       "                canvas.setAttribute(\n",
       "                    'height',\n",
       "                    entry.devicePixelContentBoxSize[0].blockSize\n",
       "                );\n",
       "            } else {\n",
       "                canvas.setAttribute('width', width * fig.ratio);\n",
       "                canvas.setAttribute('height', height * fig.ratio);\n",
       "            }\n",
       "            canvas.setAttribute(\n",
       "                'style',\n",
       "                'width: ' + width + 'px; height: ' + height + 'px;'\n",
       "            );\n",
       "\n",
       "            rubberband_canvas.setAttribute('width', width);\n",
       "            rubberband_canvas.setAttribute('height', height);\n",
       "\n",
       "            // And update the size in Python. We ignore the initial 0/0 size\n",
       "            // that occurs as the element is placed into the DOM, which should\n",
       "            // otherwise not happen due to the minimum size styling.\n",
       "            if (fig.ws.readyState == 1 && width != 0 && height != 0) {\n",
       "                fig.request_resize(width, height);\n",
       "            }\n",
       "        }\n",
       "    });\n",
       "    this.resizeObserverInstance.observe(canvas_div);\n",
       "\n",
       "    function on_mouse_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.mouse_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousedown',\n",
       "        on_mouse_event_closure('button_press')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseup',\n",
       "        on_mouse_event_closure('button_release')\n",
       "    );\n",
       "    // Throttle sequential mouse events to 1 every 20ms.\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousemove',\n",
       "        on_mouse_event_closure('motion_notify')\n",
       "    );\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseenter',\n",
       "        on_mouse_event_closure('figure_enter')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseleave',\n",
       "        on_mouse_event_closure('figure_leave')\n",
       "    );\n",
       "\n",
       "    canvas_div.addEventListener('wheel', function (event) {\n",
       "        if (event.deltaY < 0) {\n",
       "            event.step = 1;\n",
       "        } else {\n",
       "            event.step = -1;\n",
       "        }\n",
       "        on_mouse_event_closure('scroll')(event);\n",
       "    });\n",
       "\n",
       "    canvas_div.appendChild(canvas);\n",
       "    canvas_div.appendChild(rubberband_canvas);\n",
       "\n",
       "    this.rubberband_context = rubberband_canvas.getContext('2d');\n",
       "    this.rubberband_context.strokeStyle = '#000000';\n",
       "\n",
       "    this._resize_canvas = function (width, height, forward) {\n",
       "        if (forward) {\n",
       "            canvas_div.style.width = width + 'px';\n",
       "            canvas_div.style.height = height + 'px';\n",
       "        }\n",
       "    };\n",
       "\n",
       "    // Disable right mouse context menu.\n",
       "    this.rubberband_canvas.addEventListener('contextmenu', function (_e) {\n",
       "        event.preventDefault();\n",
       "        return false;\n",
       "    });\n",
       "\n",
       "    function set_focus() {\n",
       "        canvas.focus();\n",
       "        canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    window.setTimeout(set_focus, 100);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'mpl-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'mpl-button-group';\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'mpl-button-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        var button = (fig.buttons[name] = document.createElement('button'));\n",
       "        button.classList = 'mpl-widget';\n",
       "        button.setAttribute('role', 'button');\n",
       "        button.setAttribute('aria-disabled', 'false');\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "\n",
       "        var icon_img = document.createElement('img');\n",
       "        icon_img.src = '_images/' + image + '.png';\n",
       "        icon_img.srcset = '_images/' + image + '_large.png 2x';\n",
       "        icon_img.alt = tooltip;\n",
       "        button.appendChild(icon_img);\n",
       "\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    var fmt_picker = document.createElement('select');\n",
       "    fmt_picker.classList = 'mpl-widget';\n",
       "    toolbar.appendChild(fmt_picker);\n",
       "    this.format_dropdown = fmt_picker;\n",
       "\n",
       "    for (var ind in mpl.extensions) {\n",
       "        var fmt = mpl.extensions[ind];\n",
       "        var option = document.createElement('option');\n",
       "        option.selected = fmt === mpl.default_extension;\n",
       "        option.innerHTML = fmt;\n",
       "        fmt_picker.appendChild(option);\n",
       "    }\n",
       "\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.request_resize = function (x_pixels, y_pixels) {\n",
       "    // Request matplotlib to resize the figure. Matplotlib will then trigger a resize in the client,\n",
       "    // which will in turn request a refresh of the image.\n",
       "    this.send_message('resize', { width: x_pixels, height: y_pixels });\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_message = function (type, properties) {\n",
       "    properties['type'] = type;\n",
       "    properties['figure_id'] = this.id;\n",
       "    this.ws.send(JSON.stringify(properties));\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_draw_message = function () {\n",
       "    if (!this.waiting) {\n",
       "        this.waiting = true;\n",
       "        this.ws.send(JSON.stringify({ type: 'draw', figure_id: this.id }));\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    var format_dropdown = fig.format_dropdown;\n",
       "    var format = format_dropdown.options[format_dropdown.selectedIndex].value;\n",
       "    fig.ondownload(fig, format);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_resize = function (fig, msg) {\n",
       "    var size = msg['size'];\n",
       "    if (size[0] !== fig.canvas.width || size[1] !== fig.canvas.height) {\n",
       "        fig._resize_canvas(size[0], size[1], msg['forward']);\n",
       "        fig.send_message('refresh', {});\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_rubberband = function (fig, msg) {\n",
       "    var x0 = msg['x0'] / fig.ratio;\n",
       "    var y0 = (fig.canvas.height - msg['y0']) / fig.ratio;\n",
       "    var x1 = msg['x1'] / fig.ratio;\n",
       "    var y1 = (fig.canvas.height - msg['y1']) / fig.ratio;\n",
       "    x0 = Math.floor(x0) + 0.5;\n",
       "    y0 = Math.floor(y0) + 0.5;\n",
       "    x1 = Math.floor(x1) + 0.5;\n",
       "    y1 = Math.floor(y1) + 0.5;\n",
       "    var min_x = Math.min(x0, x1);\n",
       "    var min_y = Math.min(y0, y1);\n",
       "    var width = Math.abs(x1 - x0);\n",
       "    var height = Math.abs(y1 - y0);\n",
       "\n",
       "    fig.rubberband_context.clearRect(\n",
       "        0,\n",
       "        0,\n",
       "        fig.canvas.width / fig.ratio,\n",
       "        fig.canvas.height / fig.ratio\n",
       "    );\n",
       "\n",
       "    fig.rubberband_context.strokeRect(min_x, min_y, width, height);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_figure_label = function (fig, msg) {\n",
       "    // Updates the figure title.\n",
       "    fig.header.textContent = msg['label'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_cursor = function (fig, msg) {\n",
       "    var cursor = msg['cursor'];\n",
       "    switch (cursor) {\n",
       "        case 0:\n",
       "            cursor = 'pointer';\n",
       "            break;\n",
       "        case 1:\n",
       "            cursor = 'default';\n",
       "            break;\n",
       "        case 2:\n",
       "            cursor = 'crosshair';\n",
       "            break;\n",
       "        case 3:\n",
       "            cursor = 'move';\n",
       "            break;\n",
       "    }\n",
       "    fig.rubberband_canvas.style.cursor = cursor;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_message = function (fig, msg) {\n",
       "    fig.message.textContent = msg['message'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_draw = function (fig, _msg) {\n",
       "    // Request the server to send over a new figure.\n",
       "    fig.send_draw_message();\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_image_mode = function (fig, msg) {\n",
       "    fig.image_mode = msg['mode'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_history_buttons = function (fig, msg) {\n",
       "    for (var key in msg) {\n",
       "        if (!(key in fig.buttons)) {\n",
       "            continue;\n",
       "        }\n",
       "        fig.buttons[key].disabled = !msg[key];\n",
       "        fig.buttons[key].setAttribute('aria-disabled', !msg[key]);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_navigate_mode = function (fig, msg) {\n",
       "    if (msg['mode'] === 'PAN') {\n",
       "        fig.buttons['Pan'].classList.add('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    } else if (msg['mode'] === 'ZOOM') {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.add('active');\n",
       "    } else {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Called whenever the canvas gets updated.\n",
       "    this.send_message('ack', {});\n",
       "};\n",
       "\n",
       "// A function to construct a web socket function for onmessage handling.\n",
       "// Called in the figure constructor.\n",
       "mpl.figure.prototype._make_on_message_function = function (fig) {\n",
       "    return function socket_on_message(evt) {\n",
       "        if (evt.data instanceof Blob) {\n",
       "            /* FIXME: We get \"Resource interpreted as Image but\n",
       "             * transferred with MIME type text/plain:\" errors on\n",
       "             * Chrome.  But how to set the MIME type?  It doesn't seem\n",
       "             * to be part of the websocket stream */\n",
       "            evt.data.type = 'image/png';\n",
       "\n",
       "            /* Free the memory for the previous frames */\n",
       "            if (fig.imageObj.src) {\n",
       "                (window.URL || window.webkitURL).revokeObjectURL(\n",
       "                    fig.imageObj.src\n",
       "                );\n",
       "            }\n",
       "\n",
       "            fig.imageObj.src = (window.URL || window.webkitURL).createObjectURL(\n",
       "                evt.data\n",
       "            );\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        } else if (\n",
       "            typeof evt.data === 'string' &&\n",
       "            evt.data.slice(0, 21) === 'data:image/png;base64'\n",
       "        ) {\n",
       "            fig.imageObj.src = evt.data;\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        var msg = JSON.parse(evt.data);\n",
       "        var msg_type = msg['type'];\n",
       "\n",
       "        // Call the  \"handle_{type}\" callback, which takes\n",
       "        // the figure and JSON message as its only arguments.\n",
       "        try {\n",
       "            var callback = fig['handle_' + msg_type];\n",
       "        } catch (e) {\n",
       "            console.log(\n",
       "                \"No handler for the '\" + msg_type + \"' message type: \",\n",
       "                msg\n",
       "            );\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        if (callback) {\n",
       "            try {\n",
       "                // console.log(\"Handling '\" + msg_type + \"' message: \", msg);\n",
       "                callback(fig, msg);\n",
       "            } catch (e) {\n",
       "                console.log(\n",
       "                    \"Exception inside the 'handler_\" + msg_type + \"' callback:\",\n",
       "                    e,\n",
       "                    e.stack,\n",
       "                    msg\n",
       "                );\n",
       "            }\n",
       "        }\n",
       "    };\n",
       "};\n",
       "\n",
       "// from http://stackoverflow.com/questions/1114465/getting-mouse-location-in-canvas\n",
       "mpl.findpos = function (e) {\n",
       "    //this section is from http://www.quirksmode.org/js/events_properties.html\n",
       "    var targ;\n",
       "    if (!e) {\n",
       "        e = window.event;\n",
       "    }\n",
       "    if (e.target) {\n",
       "        targ = e.target;\n",
       "    } else if (e.srcElement) {\n",
       "        targ = e.srcElement;\n",
       "    }\n",
       "    if (targ.nodeType === 3) {\n",
       "        // defeat Safari bug\n",
       "        targ = targ.parentNode;\n",
       "    }\n",
       "\n",
       "    // pageX,Y are the mouse positions relative to the document\n",
       "    var boundingRect = targ.getBoundingClientRect();\n",
       "    var x = e.pageX - (boundingRect.left + document.body.scrollLeft);\n",
       "    var y = e.pageY - (boundingRect.top + document.body.scrollTop);\n",
       "\n",
       "    return { x: x, y: y };\n",
       "};\n",
       "\n",
       "/*\n",
       " * return a copy of an object with only non-object keys\n",
       " * we need this to avoid circular references\n",
       " * http://stackoverflow.com/a/24161582/3208463\n",
       " */\n",
       "function simpleKeys(original) {\n",
       "    return Object.keys(original).reduce(function (obj, key) {\n",
       "        if (typeof original[key] !== 'object') {\n",
       "            obj[key] = original[key];\n",
       "        }\n",
       "        return obj;\n",
       "    }, {});\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.mouse_event = function (event, name) {\n",
       "    var canvas_pos = mpl.findpos(event);\n",
       "\n",
       "    if (name === 'button_press') {\n",
       "        this.canvas.focus();\n",
       "        this.canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    var x = canvas_pos.x * this.ratio;\n",
       "    var y = canvas_pos.y * this.ratio;\n",
       "\n",
       "    this.send_message(name, {\n",
       "        x: x,\n",
       "        y: y,\n",
       "        button: event.button,\n",
       "        step: event.step,\n",
       "        guiEvent: simpleKeys(event),\n",
       "    });\n",
       "\n",
       "    /* This prevents the web browser from automatically changing to\n",
       "     * the text insertion cursor when the button is pressed.  We want\n",
       "     * to control all of the cursor setting manually through the\n",
       "     * 'cursor' event from matplotlib */\n",
       "    event.preventDefault();\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (_event, _name) {\n",
       "    // Handle any extra behaviour associated with a key event\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.key_event = function (event, name) {\n",
       "    // Prevent repeat events\n",
       "    if (name === 'key_press') {\n",
       "        if (event.which === this._key) {\n",
       "            return;\n",
       "        } else {\n",
       "            this._key = event.which;\n",
       "        }\n",
       "    }\n",
       "    if (name === 'key_release') {\n",
       "        this._key = null;\n",
       "    }\n",
       "\n",
       "    var value = '';\n",
       "    if (event.ctrlKey && event.which !== 17) {\n",
       "        value += 'ctrl+';\n",
       "    }\n",
       "    if (event.altKey && event.which !== 18) {\n",
       "        value += 'alt+';\n",
       "    }\n",
       "    if (event.shiftKey && event.which !== 16) {\n",
       "        value += 'shift+';\n",
       "    }\n",
       "\n",
       "    value += 'k';\n",
       "    value += event.which.toString();\n",
       "\n",
       "    this._key_event_extra(event, name);\n",
       "\n",
       "    this.send_message(name, { key: value, guiEvent: simpleKeys(event) });\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onclick = function (name) {\n",
       "    if (name === 'download') {\n",
       "        this.handle_save(this, null);\n",
       "    } else {\n",
       "        this.send_message('toolbar_button', { name: name });\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onmouseover = function (tooltip) {\n",
       "    this.message.textContent = tooltip;\n",
       "};\n",
       "\n",
       "///////////////// REMAINING CONTENT GENERATED BY embed_js.py /////////////////\n",
       "// prettier-ignore\n",
       "var _JSXTOOLS_RESIZE_OBSERVER=function(A){var t,i=new WeakMap,n=new WeakMap,a=new WeakMap,r=new WeakMap,o=new Set;function s(e){if(!(this instanceof s))throw new TypeError(\"Constructor requires 'new' operator\");i.set(this,e)}function h(){throw new TypeError(\"Function is not a constructor\")}function c(e,t,i,n){e=0 in arguments?Number(arguments[0]):0,t=1 in arguments?Number(arguments[1]):0,i=2 in arguments?Number(arguments[2]):0,n=3 in arguments?Number(arguments[3]):0,this.right=(this.x=this.left=e)+(this.width=i),this.bottom=(this.y=this.top=t)+(this.height=n),Object.freeze(this)}function d(){t=requestAnimationFrame(d);var s=new WeakMap,p=new Set;o.forEach((function(t){r.get(t).forEach((function(i){var r=t instanceof window.SVGElement,o=a.get(t),d=r?0:parseFloat(o.paddingTop),f=r?0:parseFloat(o.paddingRight),l=r?0:parseFloat(o.paddingBottom),u=r?0:parseFloat(o.paddingLeft),g=r?0:parseFloat(o.borderTopWidth),m=r?0:parseFloat(o.borderRightWidth),w=r?0:parseFloat(o.borderBottomWidth),b=u+f,F=d+l,v=(r?0:parseFloat(o.borderLeftWidth))+m,W=g+w,y=r?0:t.offsetHeight-W-t.clientHeight,E=r?0:t.offsetWidth-v-t.clientWidth,R=b+v,z=F+W,M=r?t.width:parseFloat(o.width)-R-E,O=r?t.height:parseFloat(o.height)-z-y;if(n.has(t)){var k=n.get(t);if(k[0]===M&&k[1]===O)return}n.set(t,[M,O]);var S=Object.create(h.prototype);S.target=t,S.contentRect=new c(u,d,M,O),s.has(i)||(s.set(i,[]),p.add(i)),s.get(i).push(S)}))})),p.forEach((function(e){i.get(e).call(e,s.get(e),e)}))}return s.prototype.observe=function(i){if(i instanceof window.Element){r.has(i)||(r.set(i,new Set),o.add(i),a.set(i,window.getComputedStyle(i)));var n=r.get(i);n.has(this)||n.add(this),cancelAnimationFrame(t),t=requestAnimationFrame(d)}},s.prototype.unobserve=function(i){if(i instanceof window.Element&&r.has(i)){var n=r.get(i);n.has(this)&&(n.delete(this),n.size||(r.delete(i),o.delete(i))),n.size||r.delete(i),o.size||cancelAnimationFrame(t)}},A.DOMRectReadOnly=c,A.ResizeObserver=s,A.ResizeObserverEntry=h,A}; // eslint-disable-line\n",
       "mpl.toolbar_items = [[\"Home\", \"Reset original view\", \"fa fa-home icon-home\", \"home\"], [\"Back\", \"Back to previous view\", \"fa fa-arrow-left icon-arrow-left\", \"back\"], [\"Forward\", \"Forward to next view\", \"fa fa-arrow-right icon-arrow-right\", \"forward\"], [\"\", \"\", \"\", \"\"], [\"Pan\", \"Left button pans, Right button zooms\\nx/y fixes axis, CTRL fixes aspect\", \"fa fa-arrows icon-move\", \"pan\"], [\"Zoom\", \"Zoom to rectangle\\nx/y fixes axis, CTRL fixes aspect\", \"fa fa-square-o icon-check-empty\", \"zoom\"], [\"\", \"\", \"\", \"\"], [\"Download\", \"Download plot\", \"fa fa-floppy-o icon-save\", \"download\"]];\n",
       "\n",
       "mpl.extensions = [\"eps\", \"jpeg\", \"pdf\", \"png\", \"ps\", \"raw\", \"svg\", \"tif\"];\n",
       "\n",
       "mpl.default_extension = \"png\";/* global mpl */\n",
       "\n",
       "var comm_websocket_adapter = function (comm) {\n",
       "    // Create a \"websocket\"-like object which calls the given IPython comm\n",
       "    // object with the appropriate methods. Currently this is a non binary\n",
       "    // socket, so there is still some room for performance tuning.\n",
       "    var ws = {};\n",
       "\n",
       "    ws.close = function () {\n",
       "        comm.close();\n",
       "    };\n",
       "    ws.send = function (m) {\n",
       "        //console.log('sending', m);\n",
       "        comm.send(m);\n",
       "    };\n",
       "    // Register the callback with on_msg.\n",
       "    comm.on_msg(function (msg) {\n",
       "        //console.log('receiving', msg['content']['data'], msg);\n",
       "        // Pass the mpl event to the overridden (by mpl) onmessage function.\n",
       "        ws.onmessage(msg['content']['data']);\n",
       "    });\n",
       "    return ws;\n",
       "};\n",
       "\n",
       "mpl.mpl_figure_comm = function (comm, msg) {\n",
       "    // This is the function which gets called when the mpl process\n",
       "    // starts-up an IPython Comm through the \"matplotlib\" channel.\n",
       "\n",
       "    var id = msg.content.data.id;\n",
       "    // Get hold of the div created by the display call when the Comm\n",
       "    // socket was opened in Python.\n",
       "    var element = document.getElementById(id);\n",
       "    var ws_proxy = comm_websocket_adapter(comm);\n",
       "\n",
       "    function ondownload(figure, _format) {\n",
       "        window.open(figure.canvas.toDataURL());\n",
       "    }\n",
       "\n",
       "    var fig = new mpl.figure(id, ws_proxy, ondownload, element);\n",
       "\n",
       "    // Call onopen now - mpl needs it, as it is assuming we've passed it a real\n",
       "    // web socket which is closed, not our websocket->open comm proxy.\n",
       "    ws_proxy.onopen();\n",
       "\n",
       "    fig.parent_element = element;\n",
       "    fig.cell_info = mpl.find_output_cell(\"<div id='\" + id + \"'></div>\");\n",
       "    if (!fig.cell_info) {\n",
       "        console.error('Failed to find cell for figure', id, fig);\n",
       "        return;\n",
       "    }\n",
       "    fig.cell_info[0].output_area.element.on(\n",
       "        'cleared',\n",
       "        { fig: fig },\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_close = function (fig, msg) {\n",
       "    var width = fig.canvas.width / fig.ratio;\n",
       "    fig.cell_info[0].output_area.element.off(\n",
       "        'cleared',\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "    fig.resizeObserverInstance.unobserve(fig.canvas_div);\n",
       "\n",
       "    // Update the output cell to use the data from the current canvas.\n",
       "    fig.push_to_output();\n",
       "    var dataURL = fig.canvas.toDataURL();\n",
       "    // Re-enable the keyboard manager in IPython - without this line, in FF,\n",
       "    // the notebook keyboard shortcuts fail.\n",
       "    IPython.keyboard_manager.enable();\n",
       "    fig.parent_element.innerHTML =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "    fig.close_ws(fig, msg);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.close_ws = function (fig, msg) {\n",
       "    fig.send_message('closing', msg);\n",
       "    // fig.ws.close()\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.push_to_output = function (_remove_interactive) {\n",
       "    // Turn the data on the canvas into data in the output cell.\n",
       "    var width = this.canvas.width / this.ratio;\n",
       "    var dataURL = this.canvas.toDataURL();\n",
       "    this.cell_info[1]['text/html'] =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Tell IPython that the notebook contents must change.\n",
       "    IPython.notebook.set_dirty(true);\n",
       "    this.send_message('ack', {});\n",
       "    var fig = this;\n",
       "    // Wait a second, then push the new image to the DOM so\n",
       "    // that it is saved nicely (might be nice to debounce this).\n",
       "    setTimeout(function () {\n",
       "        fig.push_to_output();\n",
       "    }, 1000);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'btn-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'btn-group';\n",
       "    var button;\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'btn-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        button = fig.buttons[name] = document.createElement('button');\n",
       "        button.classList = 'btn btn-default';\n",
       "        button.href = '#';\n",
       "        button.title = name;\n",
       "        button.innerHTML = '<i class=\"fa ' + image + ' fa-lg\"></i>';\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    // Add the status bar.\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message pull-right';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "\n",
       "    // Add the close button to the window.\n",
       "    var buttongrp = document.createElement('div');\n",
       "    buttongrp.classList = 'btn-group inline pull-right';\n",
       "    button = document.createElement('button');\n",
       "    button.classList = 'btn btn-mini btn-primary';\n",
       "    button.href = '#';\n",
       "    button.title = 'Stop Interaction';\n",
       "    button.innerHTML = '<i class=\"fa fa-power-off icon-remove icon-large\"></i>';\n",
       "    button.addEventListener('click', function (_evt) {\n",
       "        fig.handle_close(fig, {});\n",
       "    });\n",
       "    button.addEventListener(\n",
       "        'mouseover',\n",
       "        on_mouseover_closure('Stop Interaction')\n",
       "    );\n",
       "    buttongrp.appendChild(button);\n",
       "    var titlebar = this.root.querySelector('.ui-dialog-titlebar');\n",
       "    titlebar.insertBefore(buttongrp, titlebar.firstChild);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._remove_fig_handler = function (event) {\n",
       "    var fig = event.data.fig;\n",
       "    if (event.target !== this) {\n",
       "        // Ignore bubbled events from children.\n",
       "        return;\n",
       "    }\n",
       "    fig.close_ws(fig, {});\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (el) {\n",
       "    el.style.boxSizing = 'content-box'; // override notebook setting of border-box.\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (el) {\n",
       "    // this is important to make the div 'focusable\n",
       "    el.setAttribute('tabindex', 0);\n",
       "    // reach out to IPython and tell the keyboard manager to turn it's self\n",
       "    // off when our div gets focus\n",
       "\n",
       "    // location in version 3\n",
       "    if (IPython.notebook.keyboard_manager) {\n",
       "        IPython.notebook.keyboard_manager.register_events(el);\n",
       "    } else {\n",
       "        // location in version 2\n",
       "        IPython.keyboard_manager.register_events(el);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (event, _name) {\n",
       "    var manager = IPython.notebook.keyboard_manager;\n",
       "    if (!manager) {\n",
       "        manager = IPython.keyboard_manager;\n",
       "    }\n",
       "\n",
       "    // Check for shift+enter\n",
       "    if (event.shiftKey && event.which === 13) {\n",
       "        this.canvas_div.blur();\n",
       "        // select the cell after this one\n",
       "        var index = IPython.notebook.find_cell_index(this.cell_info[0]);\n",
       "        IPython.notebook.select(index + 1);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    fig.ondownload(fig, null);\n",
       "};\n",
       "\n",
       "mpl.find_output_cell = function (html_output) {\n",
       "    // Return the cell and output element which can be found *uniquely* in the notebook.\n",
       "    // Note - this is a bit hacky, but it is done because the \"notebook_saving.Notebook\"\n",
       "    // IPython event is triggered only after the cells have been serialised, which for\n",
       "    // our purposes (turning an active figure into a static one), is too late.\n",
       "    var cells = IPython.notebook.get_cells();\n",
       "    var ncells = cells.length;\n",
       "    for (var i = 0; i < ncells; i++) {\n",
       "        var cell = cells[i];\n",
       "        if (cell.cell_type === 'code') {\n",
       "            for (var j = 0; j < cell.output_area.outputs.length; j++) {\n",
       "                var data = cell.output_area.outputs[j];\n",
       "                if (data.data) {\n",
       "                    // IPython >= 3 moved mimebundle to data attribute of output\n",
       "                    data = data.data;\n",
       "                }\n",
       "                if (data['text/html'] === html_output) {\n",
       "                    return [cell, data, j];\n",
       "                }\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "};\n",
       "\n",
       "// Register the function which deals with the matplotlib target/channel.\n",
       "// The kernel may be null if the page has been refreshed.\n",
       "if (IPython.notebook.kernel !== null) {\n",
       "    IPython.notebook.kernel.comm_manager.register_target(\n",
       "        'matplotlib',\n",
       "        mpl.mpl_figure_comm\n",
       "    );\n",
       "}\n"
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<img src=\"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAApQAAAFKCAYAAACjE8/+AAAgAElEQVR4XuxdB5gUxda9IiqCikpWUATJSVBykCQg5oQ8I5if4amYRSRI8BkxPAyomMGsvwpIDkpGcs4gOeck+H+nZk/v3aJ7pmd2dnbFqu/zk52pru6+XdN1695zzznmr7/++ktccxZwFnAWcBZwFnAWcBZwFnAWSNACxziHMkHLucOcBZwFnAWcBZwFnAWcBZwFjAWcQ+kmgrOAs4CzgLOAs4CzgLOAs0CmLOAcykyZzx3sLOAs4CzgLOAs4CzgLOAs4BxKNwecBZwFnAWcBZwFnAWcBZwFMmUB51BmynzuYGcBZwFnAWcBZwFnAWcBZwHnULo54CzgLOAs4CzgLOAs4CzgLJApCziHMlPmcwc7CzgLOAs4CzgLOAs4CzgLOIfSzQFnAWcBZwFnAWcBZwFnAWeBTFnAOZSZMp872FnAWcBZwFnAWcBZwFnAWcA5lG4OOAs4CzgLOAs4CzgLOAs4C2TKAs6hzJT53MHOAs4CzgLOAs4CzgLOAs4CzqF0c8BZwFnAWcBZwFnAWcBZwFkgUxZwDmWmzOcOdhZwFnAWcBZwFnAWcBZwFnAOpZsDzgLOAs4CzgLOAs4CzgLOApmygHMoM2U+d7CzgLOAs4CzgLOAs4CzgLOAcyjdHHAWcBZwFnAWcBZwFnAWcBbIlAWcQ5kp87mDnQWcBZwFnAWcBZwFnAWcBZxD6eaAs4CzgLOAs4CzgLOAs4CzQKYs4BzKTJnPHews4CzgLOAs4CzgLOAs4CzgHEo3B5wFnAWcBZwFnAWcBZwFnAUyZQHnUGbKfO5gZwFnAWcBZwFnAWcBZwFnAedQujngLOAs4CzgLOAs4CzgLOAskCkLOIcyU+ZzBzsLOAs4CzgLOAs4CzgLOAs4h9LNAWcBZwFnAWcBZwFnAWcBZ4FMWcA5lJkynzvYWcBZwFnAWcBZwFnAWcBZwDmUbg44CzgLOAs4CzgLOAs4CzgLZMoCzqHMlPncwc4CzgLOAs4CzgLOAs4CzgLOoXRzwFnAWcBZwFnAWcBZwFnAWSBTFnAOZabM5w52FnAWcBZwFnAWcBZwFnAWcA6lmwPOAs4CzgLOAs4CzgLOAs4CmbKAcygzZT53sLOAs4CzgLOAs4CzgLOAs4BzKN0ccBZwFnAWcBZwFnAWcBZwFsiUBZxDmSnzuYOdBZwFnAWcBZwFnAWcBZwFnEPp5oCzgLOAs4CzgLOAs4CzgLNApizgHMpMmc8d7CzgLOAs4CzgLOAs4CzgLHBUOZTdu3eXYcOGyahRo8yTbdy4sYwePdr3KX/00Udyyy23yIoVK6RkyZJH9Onbt6/ccccdboY4CzgLOAs4CzgLOAs4CzgLxLDAUeNQ9u7dWzp06CCNGjXyHMotW7bIgQMHMpgATuKSJUtk0qRJcvLJJ8uPP/4obdq0kaVLl8oxxxzj9c2fP7+ceOKJbgI5CzgLOAs4CzgLOAs4CzgLHO0O5erVq00kcezYsVKiRAkpUqSI51Da996/f38Tlfz999+lSpUq5uuePXvKl19+KdOnT3eTxVnAWcBZwFnAWcBZwFnAWSABC/ztI5SIMMJRRLq7W7dusnz5cl+Hcvfu3VKuXDm54YYb5IUXXvBM1bZtW8mVK5d8/vnnCZgv/ZB9+/bJrFmzpFChQpI7d+5MjeUOdhZwFnAWcBZwFnAWyPkW+PPPP2Xjxo0mSJUnT56cf8FZeIV/e4dS26Zdu3aBDuXLL79sHM6VK1cK0tlslSpVkmLFisn+/ftl4cKFUqZMGenUqZO0bNkyqtl37Ngh+I8NEc7LLrssCx+VG9pZwFnAWcBZwFnAWSAnWgAwupo1a+bES0vZNf0jHMpDhw5JqVKlTHSyV69ennGBr8yXL59Ur15dXnzxRTnppJPk008/lddee02GDh0qzZo1C3wQXbp0ka5dux7xPSYVHFTXnAWcBZwFnAWcBZwFjm4LrF27VmrVqiXLli3zLfA9uu8+4939IxzK4cOHS/PmzU0xDhxL3ZAKR4r6hBNO8D5u1aqVKdAZNGhQ4FywI5ScVKtWrZLixYv/k+aQu1dnAWcBZwFnAWeBf6QF/vjjD1O/4dZ+kX+EQ/nAAw+Yqu6JEyeGmvCPPfaYDB482GAiwzY3qcJayvVzFnAWcBZwFnAWODos4Nb+9Of4j3AokdJG1FGnu2GCmTNnSr169Yzz2KBBA88qTZo0MdXiAwYMCD3j3aQKbSrX0VnAWcBZwFnAWeCosIBb+/9BDiXwk+CT/PDDDw2GUrfDhw8bhxJp77feeksKFiwo7777rvTp00cmT57sUQuFmfVuUoWxkuvjLOAs4CzgLOAscPRYwK39/yCHcsOGDSbaiCikX+U2yv2ffPJJGThwoGzbtk1q1Kghzz//vDRs2DCuGe8mVVzmcp2dBZwFnAWcBZwF/vYWcGv/UepQZufMdJMqO63vzu0s4CzgLOAs4CyQegu4td85lEmfdW5SJd2kbkBnAWcBZwFnAWeBHG0Bt/Y7hzLpE9RNqqSb1A3oLOAs4CzgLOAskKMt4NZ+51AmfYK6SZV0k7oBnQWcBZwFnAWcBXK0Bdza7xzKpE9QN6mSblI3oLOAs4CzgLOAs0COtoBb+51DmfQJ6iZV0k3qBnQWcBZwFnAWcBbI0RZwa79zKJM+Qd2kSrpJ3YDOAs4CzgLOAs4COdoCbu13DmXSJ6ibVEk3qRvQWcBZwFnAWcBZIEdbwK39zqFM+gR1kyrpJnUDOgs4CzgLOAs4C+RoC7i13zmUSZ+gblIl3aRuQGcBZwFnAWcBZ4EcbQG39juHMukT1E2qpJvUDegs4CzgLOAs4CyQoy3g1n7nUCZ9grpJlXSTugGdBZwFnAWcBZwFcrQF3NrvHMqkT1A3qZJuUjegs4CzgLOAs4CzQI62gFv7nUOZ9AnqJlXSTeoGdBZwFnAWcBZwFsjRFnBrv3Mokz5B3aRKukndgM4C2WaBHfsOysj5G6RxucKS/8Tjsu063ImdBY4WC6zbvk8mLd8iLSsVkRNyH3u03Ja4td85lEmfzG5SJd2kbkBngWyzQI+f50rfscukXb2S0uXyStl2He7EzgJHiwXa9ZskoxZslOeuqCQ31y15tNyWcyjVkzzmr7/++uuoebLZeCPOocxG47tTOwsk2QK3fDBJxizcKM0rFJH3br0gyaO74ZwF/nkWaP7KaFm8YZe0uaC4vHBttaPGAG7tdxHKpE9mN6mSblI3oLNAtlmgVe8xMn/dTmlYpqB8cnvtbLsOd2JngaPFAvWfHyGrt+2V2uecLl/cXfdouS0XoXQRyuTPZedQJt+mbkRngeyywHndhsi2PQelVsnT5ct7jp7FL7vs6c7rLFDjuaGyZfcBKZY/j4x/qtlRYxC39rsIZdIns5tUSTepG9BZIFsssO/gISnfabA5d9Xi+eX/7m+QLdfhTuoscDRZoEKnwbL34CFzS/OfayV5jjs6CnPc2u8cyqT/Tt2kSrpJ3YDOAtligZWb90ijF0eac5crcrL88nCjbLkOd1JngaPFAocP/yWlnh7o3c7QhxtJmSInHxW359Z+51AmfSK7SZV0k7oBnQWyxQKTl2+R694eb859doG8MvqxJtlyHe6kzgJHiwX2HPhTKj77i3c7791ygTSvWOSouD239juHMukT2U2qpJvUDegskC0W+HHGGnmg/zRz7qKn5JEJTx89eK9sMag76T/eApt37Zfzuw/z7NDp0opye4Nzjgq7uLXfOZRJn8huUiXdpG7AbLTAu2OWyPy1O6XXNVWOKhLiMCZ9b+xS6f7zPNP11LzHyfRnW4Q5zPVxFnAWCLDAqi17pOELERgJ2q11z5auV1Q+Kuzl1n7nUCZ9IrtJlXST/qMHXLh+p5x43LFS4vS8KbfDpl375YK0aMJrbc+TK847M+XXkJ0nJKk5rgHPYN5zrVJyOWu27TVVsJXPzJ+S87mTOAukygKL1u+Ui14d453uwrKF5KPbaqXq9Fl6Hrf2H6UOZffu3WXYsGEyatQo7w7bt28vH374YYYJdeaZZxruKLTDhw9L165d5b333pOtW7dKgwYNpE+fPnLuuefGNQndpIrLXK5zFAss2bhLmr08Wo479hiZ1aVlyqshB85aK/d+9ru5wh5XVZYba5/9j3pe/+k/Tf5vxhpzz8ccI7K0Z2s5Bv/Iwnbw0GEp+8wggczE0VSwkIUmc0P/jSwwY9U2ueJ/v3lXfE7BfDLy0cZ/ozsIvlS39h+FDmXv3r2lQ4cO0qhRowwO5fnnny+tWrWSBx54wLvrY489VgoVKmT+hjMJB7Jfv34CR/Pxxx+XpUuXypw5c+T4448PPeHdpAptqpR1HDp3vbw+fJF0ubyinH/26Sk7b2ZP1HPgPHl3zFIzzLgnm8oZp56Y2SHjOv7ZH2bLx+NXRH4fl1eSW+sdPTJpYQxx/TvjZeKyLV7XBd1bZXnaf+6aHdL69bHmnI+3Kif3No5vQxvmvlwfZ4HsssCEpZul7bsTvNNjszz/uYvl2FxZu1FLxf26tf8ocihXr14td9xxh4wdO1ZKlCghRYoU8RzKQ4cOyUknnST9+/eXK6+88oi5deDAASlYsKC88MILcs8995jvt23bJmeccYZ88MEH0rZt29Dz0U2q0KZKWcdbP5gkoxdulH/VKiG9rq6asvNm9kTt+02SkQs2ZptD2fLVMbJg/U5z/kcuKisPNCuT2Vv6Wx3f+MWRsnzzHu+aZ3ZpIafkOS5L72HApJXy5LezzDleuq6aXHt+8Sw9nxs8ey2wc99BGT5vgzQuV0hOzRs+cGFfNeApYxdtlBYVi0q+E3Jn701FOfvI+Ruk/YeTM/QY+3iTbIH0JNtIbu0/ihzKH3/80TiMSHd369ZNli9f7jmU8+fPlwoVKsiCBQukbNmyR8yjSZMmSe3atY/4HmnvqlWrmshl2OYmVVhLpa7f1X1+k99XbpOLKxeVt246P3UnzuSZGvx3hPyxda8ZJdUvXWD4oGjBdkeDc+SZSytm8o7+Pof/9ddfht6EBMy48kkdm0nhk/Nk6U089e0s6T9ppTlH9ysry011/lkwgyw1bg4c/IXB86XPqCVyY+2zpMdVVRK+wgcHTJMfpq+Rh5uXlQeb59yN388z18p9n/8uCEge/ityu5/fUVvqnVsw4XvPKQe6tf8ocij1pGrXrl0Gh/Krr76S66+/3kQfBw0aJLly5ZLWrVsb5zN//vzy7bffyjXXXCN79uyRE09MTyu2adPGfPbTTz8FztkdO3YI/mNbu3at1KpVS1atWiXFi7voQk74sTPSVq90Afn8zjo54ZJCXUPJJ3/2+o16tLGULJgv1HHJ6DR49lq559MIfhLtuvOLy4vXVUvG0H+LMbbvPSjVug7JcK2pcOovfWOszF4deZ883bq83NWo9N/CXu4iE7PAfZ/9Lj/PWivNyheW99vVTGwQEeE7rlWlovL2zeE3zcs27ZYb+k6QlpWKSpfLKyV8/rAHfj31D3n0qxlyWt7jBL8xOJXv3ny+tKhUNOwQObafcyj/IQ5l586dpUePHtKzZ0/jSC5atEgeffRRkxofMWKEfP7553LzzTcLUuNwNtluueUWWbNmjSnwCWpdunQx+Eu7OYfS32JQSvh4/HIpV/QUqVu6gG+nA38elk8mrJDqZ50qNc46LdMvkPrPj5DV2/ZK5TNPkZ8eaJjp8VIxwI59B6Vql3SHZvgjF0rpQiel4tTmHF3+b458OG65d76WlYrIOzdfkLLzZ/eJ7GpUXM+wDo3k3MJZp+oBqcfKnX+RP9NCN/9pVkY6XHRkRiW7bePOnzwL3Pz+RBm7aJM0LFNQPrm9dsIDV+nyi+zc96dUOTO//PhAeInQl4cskDdGLDbnTQXJ+Cfjl0unH+bImaeeKFv3HJA9Bw7J6/+qLpdXOyPhe9cHbttzQPpPWiWtKhcVFPyksjmH8h/iUCJ9tX37djn11FO9O54wYYLUrVtX8H9MhGuvvdY3Qrl//3754YcfAueli1DG95Mds3Cj3PLBJMPrN63TRb5Vs4yOnXV6XhnzeObVSRBpwm64xOknytjHm8Z3wdnUW6u04BKGPNxIyqZQouyKN3+VGX9s9+6+TqnTZcBddbPJGqk/LfBoN78/KcOJf3qgwRFUPlD+AKVQMqq/p63cKlf1Geed858GM0j9U87+M6LiGZXPify+sAE5IXcu2X0gshFBQ+RvWhx8qc98P0s+nRCBWMDJG9qhkeQ9PuswmOC17TlwvpQulM9QY23dc1BevLaqXHdBiaQ8jFeHLpTXhi+SFhWLyLu3pHYD7BzKf4hD6TdTd+/ebQp1vv76axOpBIZy8eLFUrp0eooJGMpq1arJ//73v9CT3U2q6Kb6fOJKefq7SNHB1GeaS4GTTjjiAEQwn/1hjsHZLOrROlMVgNhMlOk4yER9TsmTW2Z2aRn6WWZnR0RoO30/27uEQQ82lArFTknZJdXpOVzW7dgnhU8+QTbs3G/OjWv4pzSm5lB9eigtYvjNv+tmYAn4ddEmueWDiXJzneSQM380brl0/r85non/Vess6XV14ri6f8qz+jvfZ9OXRsnSTbvl/LNPk2/+XS/0razdvldavDpGKhY7xVB6NX8lndtxdteWclLIwpw7Ppoiw+at9877YLMy8nAWRsV7D1sovYctMpFUFBKt3b5PnruysvkNJaN1+GK6fDtttdQ461T59t76yRgy9Bhu7f+HOJQ33HCDbN68WX75JV1DFByVTZo0kblz50qpUqWkcOHC8sorr8jtt99urHI0VXnDOdm4c7/8+8LScuLxx4b+gWRFx/+NXCwv/rLADO0X8cHnb49eIs8Pmm/6TO7YXAqdfKTTGfbasIsv32mw6Q4KwSU9WkuuBCkqxi/ZLL/MWScPND3X1xEOe01h+unIQTRbhRkr3j5wwsGFePDQXyZyMmHpFhO9+O3Jv0d0N9779ev/5ohF8tKQhYIo+cotkUrvz+6oLfVV8cA9n0yVwXPWSalC+WTEI5nn0uvw5XT59vfV3uVccd4Z8lrb6sm4HTdGDrXABd2HyqZdB6Ra8fzyw/3hU9WvDF1oqNDQIDrw4IDp3h3+8lAjKVc0HDTjktfHypw16TUAF5x9mnyd5tj+eeiwifYVP+1Eub7mWb4WBDzppSELpGrx/HJp1dhp614D58k7Y5ZKrZKny8Zd+wUYzmcuqSB3NCxlxl+8YZdgY9WufsmEID5k9ICjPTDFG2DnUP5DHEoU1Vx++eXy3HPPGQqghQsXyn333WdS3p999pmxQseOHeWdd94xNEElS5aUxx57zBT2zJo162/NQwlHslbPYYYoGSnf/nfWkeKnpV51hVOt+09z5b1fl5k/g8DYTFugz+CHGkr5oolH5mzt2BmdW0j+ExOjfmGRDLRnoUGble26t8fJ5OVbvVP8cF99qVYiHbKRlefevuegVOsWwW+i+vSziSvl5BNyy6yuf4/objJsgyg6ounAtgHjhvZBuwukafki5t/YqFTvNtRUgWPDg41PZpu9uDevUETeuzW1abvM3kNWHr9+xz7Je/yxcnIWUzdl5T3osbFxK/fMYDlw6HDcGQAtC4qCua+mRgQ67Hka617Of26obN59wODVp63cZn7noMcChINRenBFIup5Qu4jgxFD5qyTuz6ZalLvOI599h44JFv2HDAbUd3IbQuFHDzP+et2ymMty8l9TSJ8q4wwtrmguLxwbfxFgCxqS9YmL5b99PfOoUy3xjF/YXYfJc2u8sZtffPNN6YwBxRCwFIiaokq7zx5IjQgKMh5+umnDbH53r17DTE6Ut1wLuNpOW1S2bgsRFxQ4HHcsenFR/HcX2b76ihMEFm2JvT+9Pba0qBM4pQSKzbvlgtfTFdMSrRSF3J49Z4fYW4fu+sv78k6PCF+ilW7DjEgezakw5AWS0VDlKD5K6PNqeA4P/fTXPPvJT0zBz9IxbUn6xzkAG1bs4R8N2217P/zsPS5sYa0rlLMnGL4vPVy+0dTzL+TJcsImUukAeE0oVihbqkC0v+uvw8rQbJs7zfO8k27TYq32Kl5ZOQjjRPOMmTlNcY7ts6enFv4JBnW4cLQQ2i+0mL585jUMVu3KyrJLXVjr1v6/Cj+QtQT7dcnmpigA/Gd+Gz6sxf58mR+8Osy6Zb2fvj+vvpyXtqm97I3fpXZa7bLj/dnxB2jwhuOKqrR1+7YZ/Cj/2l6rnRoUc6cmxvpiyoWkb4JYCDr9Roua7bvy5aMSk5b+0NPpizoeFQ5lFlgn9BD5rRJpeXzeBN6YQx9YwEdwSv244w10vWKSlLklNgcfZqs++4LS8lTF1cwIyMtP2nZFnn+6iom3Y2/0Xpff55cWT1xDek5a7bLJa//6l19UJo9lh2+m/aHPPzFDNPtlrpnS7crKsc6JOHvN+zYJ7V6Ds9w/Ff31JWaJVOj8jNx6Wa5Pk3Nol/7mtK+X4SIGEVUp+VLnHw5YYNkw4Gteo8x0RPw+r3/61LZse9PefX6anJV9QgV2JPfzJQBk1d5V7aox8WZ2qQBp1mm40BDo4J03dy1O+JOg0YzE2Q88bu6tW7J0Bs0pP2XbNwt/72mqhyfO3s2oLwnHZHL6fMQhVqdvp8jNc4+Napc6Yad+6RWj8jv/OwCeWX0Y+ELED/8bZl0+TGy0bPbXY1KydOtI+/VaA1OeuOXIpttZECu7PObyWS9f+sFhm/1sjfT35vjn2oqxfIfqdTVa9A8eWd0RM3ruSsqyc11S4rOCj15cXm558L0ugRwUGLNuLr6mfLHtr3mna+vt9ELIw3EJJGqd2zEAW/C5q/gScfLlGcuynD7I+avlxcGLzCKYx9kgqIpyKY5be2P9fyz8nvnUCbJujltUvFFjHR38VPzyvilm5Ma+Wj28iiz6CCShVRwrKZ3vRojhirFXfv/NFEgKEd883skhaPxNbHG9vseL6w274z3vkqURPeJr2fKF1MiDgQA5ACSZ1Ubt2ST3NB3YobhB9xVR+qU8qdZirVoQB+6TBwV4iQfPv7YXDLwwQYe4H/0Y43l7AKppeLIKhvHGpfMAC9cW1Ve+mWBKUxCgQwKZeD81e6JaOIBb5igCE6s8/B77Vy0rlJUBs5aJ/FGraKdiwt/g3MLyqd3xKan0ZH9j2+rJY3KRiRqs6vp4pERj1wopdIotFApvGzTLkMvhjQtIl6I2BUOsbn1u5d5a3eYgpYSpycOC3r/12Umqo+CLhQeBing6EzAGfnzyLinmoU2r8aZ2wdh/vS5MTYXpX7PzOrSQhBVhDIUUtArN+/x3ncYH3rbfjQ8Dw2YJt9Pj+jdQ9UJ6k7jFm+SG96LvL+uqn6mvHr9ed4l3vbhZBkxf4PcUPssI9oA1o9b60aK2gwEoNNgAS5TYznDGmX3/j+lUlq1O54h0vS6kbII82N8HLYOe/6ctvaHve6s6OccyiRZNadNKrzY8IKrdc7p0q5eSbn3swhZdbI49bjwImUC3rxY7cIXR8qKNDk7po6xQJd+eqA5FBHKMYs2mgUVDbtb7HITbbbU11s31pCL09KW8YypZfiymuSbFd54KcLJRrMLQsJcOyIltXsMl31/HjIv0II+FfV+47DaGIvc9/fX96Io/3d/falaPDU4zjD3l1V9YDeo5KB9cnst6fjdbBM16XJZRWlX/xyZumKLXPNW+iYF/RKFUvAeZq/eLpe+EYkI3du4tFFPidfJiGYPMAZgXgGHi2hUrKYjgsmkdYl1Xr/vwV1b/bmhhvoLTcM/qIIFp/e0vMebqBocHzhA8bZVW/aYiB3YIPB7yXNcYgWMWgP+lTbV5Ooa/gIXv6/cKlen0UTFi8N9bdgieXVYJEXNBhwjonNhC3y+mfqHPPLVDA8fffcnU+SXOesF+MaJyzbLvoOHvbEH/qehVDzjSCz7v96dYIIUaGWLnCRDHr5QdBrcZodgf1BirdiyR4bOXS/XX1BC/nttVQGH5HndIupclc44RX7+T3ysEnh+DV8YaY7PnesYWdyzdQb7sMI8kbHDzKWctvaHueas6uMcyiRZNqdNqns/m2qcM0QDsXsEyTeiLXAuM6uMAEfw3I4DTZrk/ibnyqMtIziYaI0EvOiD6sFfn2gq0LOtkkbijUjnr4s2ehrW3PXGGjfo+/+bsUb+03+a9/V/r6kSWLEYNMa67fukTq/0FPRl1c4w3GmImpxVIK/0uLJyUngIef6uP86Rfr8t94Dy+DyRKNHSjbuk6csRLOR399aT6iFJ4l8ZskBeH7HYUHsg1c4qeThXDctkb6Qq0XkQz3E6cgRc278/nSqLNuwyGxtscBjt0w5/0IIb9rwjF2zwoAWYo098M8sUj6GILBmNKfryRU+WwQ81ijkkovqI7qPpoomYB2ZBBxu2ooujKj472OBNn7q4vOG2hd3QFve4WHLHiRNnZB7Hj3msifltx9u27j4g53cf6skKRpN7HbVgg7RLg5Pg2qfHwR/538Hz5a1RSzJcHoIGeGYF8h0vUztlTPf63QcZN8oUPkmGdrhQdDEk+mt5xG/vrecrMkHaI/af1aWldPtxrhfdREHP3G6tPDgIM1TATS7bvMfApZipmr9uh7TqPdZcaqmC+WREnJuC6au2yZX/+827VcwB4ELB6fverTVNpgGbKkTb8T5Ndstpa3+y7y+e8ZxDGY+1ovTNaZMKPzD80BjpY5U1IxUL1+8U/HdJlWJxO0V4eSJygHZnw3Ok4yXRK59N6rXjIM96eNkseO5iQx9ROw0ziEjnb4s3ycS0xaxJuULSr334Hz92/Tv2HpTG5Qqb80AXGfrIbInI2f0wfXUGWg4Axi+tWsz7LNmpYBC/IxWEnTvT7P3a1ZQm5SP3FLbN/GObXP5m5AUbT4STetK0fblnBpnIx/9uqCGXVI0UpRzNTZOaz+3WUq5/Z4LMWr1dHmpeRh5qXla4iKIS9cspEWjGF3fVkdoJQBJox2B091UAACAASURBVC8nr5LHv5lpiKl7XV1V7vl0qomyAJuZDNJ0FsOVLJBXRsXA6iGNDDobai0nY/MZz3yBStSIeRukcblCJl2sI14Y5+Xrqsk15xc3lfbc7NzdqJTkz3ucwcihzXi2hfk7ntZn1GLv+Hg2YPocjPrxMxRY/d7pIt9op97s5jv+WJnTrVXoy+WmUx+ATf2bIyOqN5i3sQjKO343yzA40MEaNGut/Dstg4Ux4AwPmh3JFAVBhSo9O9gQq7Phd4CiSi2KoEUZWrw6Whau3yWPtyonSzfu9gp0IBc5GunvDyJiAomkpYGRvO3DSKEcGlLe53UdYjiIocbzy+x1RubSTsOHNnqMjjlt7U/WfSUyjnMoE7GazzE5bVKRoJqVf+TXY6SCqdxEsIo6AhamUEXjxGi6SU83M2ldRtJQqDNhyWbvhRSPlBhSlaiURcSCLzGdusM572tSWh5rGV8K3eaEBGAcyg6MfOqISTKmEaUiO7auID0HzTMR4ERk0TSWCRWTcISBU4rloNz58RSTimJ0uFaPYSaq3fOqKgb79HdrYe5Z39OXU1bJ41/P9CKE1741Tqas2Cr/blxarqlR3KuA//zO2h7WFYUMzSpEKIUSaYwWIW2IKD1VehZ0b+VL14JzxHNfD/SfZqJBRU/JIxOejo7VI10M7wObCGwmUtUYAaYje9fHU2TI3HTybb6rQO5dt1eEeQFz9dQTj/MoyViprK85lr00TjrR50luUrxfUdSFFrQZ/HTCCnkmTbwAeOWFPS4ObWJu+vQBH7av6UU8gyBN2gbEM4LJ4Plrqop+n2NcEPkT2uFnD51Z4nUgUgzictBpsWlpxYYvjJBVW/Ya+MjijbuMSg82Dh+2ryX83eG4eCO2OOarKavksa9neucd92RTj5kDcwYE7uDUzSrat5y29oeeTFnQ0TmUSTJqTppUIKYFQTUiDXQo+o5ZKj0GzvPImMmtiNtf/vwlcVlBY4CIg4k2wIJ1O6Vl73RFB/QF1QQiMcSPoeBlwtLNJsWIFs9OdfGGnV4BCVPbxM3wum6qc5Z0vzI+9RGNicI4wH7eUu9suf/zSCr9iVbljbORjAb+toqdBxsnEo7qXR9PNTvst2863+jTxtPIEYdjQH4MZwJVluB8a18/uICKuDRGtS96ZbR5Hsm8z3juIzN9wSUJveLebc8Lna4nPo2brpvemyi/Lt4k7euXNNWvSDdiwZvSsbnBfGFDBPtecV7ibATUTkfRDJRKrnkrIsHoV9GM6tyHvpguKEL44f76MSNRGIeOTpiFmlg62j2rabLs58tCD1DLoEivRvehsm1PBD+JBmGBR1qUE50Kb1q+sHEooZKCZpN7IzsCvGKe43IZCVEUzNit7bvjjcOBhmKsNnHKAWpuUlQ8gwUApOFBikc6IgrRhaU9W8fc7PGaydeo7wGFNYQO+Tmx2FyjmOedm883ik+ayeDB5mVMsVmlzoMNdhJMAz//p4FUeDbyt192QkNDUECGv4FPJFE66a/0Jp5E7ng/L1q/y2wASI/1xvBF8nIadRHwoAu6h3ewYQe7UAnXT4YPBCqAp0d0NKsgHDlp7c/MOzMZxzqHMhlWFDG64JByXLVqlRQv7g/GTtKpYg6zetteg5lEI10OCy5AOItd/DlPRYph0MAZVqV4/pjjsoNOMYRJI8BRbJtGR4MXKJymt2+qIafnO8GrxL66xpkGB4QKQDTs3BGliRVVQ19I4d30fqS6EIt/58sqSY+f50rfsREidbTLq51h0h/xNPIDnp7veKM/C1UIOL7cDQMD9MwlFU36BmlQPznJsOebu2aHtH49giNCKh3cezYHYtixvv39D+nwZYTqCBXKSzZEXuCxZN5I3cFIEJybqWkROjiVf6dGLGCYCDrvi3hDOCmgF7n9w8kyPK0yFVXAIIDGPH2lzXlSt9fwpMjHEeuM3xEceW687EgbKJ0QvQQZNlrYVDvvAQ7V/OeiL9Tk8ityygmyfsf+hItcEp0njKaC2P2V66tJ1TR8NdLCSK9yU6h/7yhEQXocaVM0WyZT/66GPtzIl/WA943jbbqbMPeiaXjg0A6YvNJgoWufc7p8cfeRvLU2DjIe3Od9n/1u0rdspMnhbxeFXY9bv9Ur3vzVZH6w+cXvWDMZ0HnmZuKNf1UXYMXP6zbEOPPA3yMKrJvOgMBJowIa+sBhxwYYuNRm5QvL+2k0PUyR4x28YN0O+d/IJQYr/t299cXOBMXLe6v5i3ENgPncmFZtDpoiFHuCmQGFn21rJT/TkpPW/jDzNSv7OIcySdZN1aRC6gKp3XxRNFunLN8i174dqUalbjYxhagqREpAYxrDRBkxFlLLIHOGTByqBNHCUFUQowPsJCI9cHifvbSinFMon1eQgKgEQNRQb2ALq26j8Uv1SheQz++sI099O1P6T0rnC0QF40dxALJRWYoXLxqiR4hUIS15U52zjd44GiJZIIxHWu5ftUoYDFyiDWlJLKhwpOc910pQxITnrNNGYcemJjr6I42KCC5sAYf4/6LIvPGlTw5QTfWBtHd2NERPEGWKt/KW0dV4ngsxrEjv4365eGOOA1OGjRC5XJMVvSWhM7CAmFusVtX4M9gdhPOIBLEhzUm8cLTncvP7Ez3Fn2W9giNhiLIhKoV7hEOB+Rgvvi+z84PONTC8cNqJ0wb+E7Q2TMHzt4LzocAPVd7AuqLZdtHvQqiF1S2dkYJr/58RPCblPcJyOep71UUl2AgAn4jCmaANHDGMHGNet1ahpXG5QeCxIBRHtoeRbhba6OujEhM223AAyWSgxSPwbkcRImmZom2YuGGFus7kZ5qbTRAZPBCxBBtGr0HzPZJxrFmlno4UcSJjNn/tDhORpEyirSs+p2vLqOubPc8e+XKGRzeH7+AU412KVv/cAgLpXGTrghTaMjtvU7X2Z/Y6U3G8cyiTZOVUTarHvpphFDxQrVZP6Qvr2yDoG+mD+c9Fonx0ulBBCoeSvF04DtELkMGiejWoodADTiqKeCqfmd9TUdG70KBjP5u4wlCwFD75BEPkC2lBFPOcV+I0k4pFAz5xyvKtGTA4UPYpncY7F+0xaZ1wRBPhRP9nwHSzKNov3rCPWysNYbGHDi2cR0gS4mWJBgc51zHHmEhi6UL5ZHgmdJ2ZbiUFB6viEyF41/Z4tEVZWbB+l7GFTeWhbYGUOxwKNC40D38x3cy1VGPpeF1YiK7sM06WbdxlbBuPtjtTbPGwBdBJhM3ub1pGWNCCQjbwHKINerChsSPhAX4RobBzDP1Ip4WoMGhmaqQVu+nqWqTWwdeqW1gaLJ3OxbsgyDHXsBHgrrlpindxj+fe7b6MkuFd8HKbah5tFbgJgWUll6beMCG9CocSm1S0N2+onkFbWhd8MPqmz6vTt/gcWFmcO56mq4ynPNNcPh633LAlBNH4MBLLc4TdOKP/je9NkN8WbzYbeRTf4HrhJOuooc0dyRQ3NlfQzm6WxgAR7f3a5KVRR+ht83rhLCPKSr5UXcyGokXgzFlkAycekUgWUSF6CMhCz4HzPfjV5W/+KjP/iGwI0GDDsFRn6K9FM/A3YAed0jb9WHOAA0ezo9fxPONofVO19ifrerNyHOdQJsm6qZhUmm9Lk4Pbt/DumCXmB6srO7mrR4QRDiV3/zwWqXE4Y3hZXF+zhHEadWPF5cl5chvqoTdGRKoKwxAm64KgskVOFji8cFIaly3kpY+pKavP+eXddQ2PZqzW+YfZ8tH4iMIO2qSOzeTJb2YZIl22eDVeWaCAKM3Tl1TwHGJErwA+txtS+TM7t0hYbxiFPrALIrWofIyWcoplD51Sg8ODIgHYIhphtp5b1FFn1CMR9Qq/a4SMJeYN5hel2qLdi+an08VJYCf4cNxyUw3vp3MO/sIyzwwy2DD+ToAhxjPCghrUqnT+RXbu/9OrJmYBBDCo63ZEJO4mPNVMiubPI4xmxpNS9zsv6W+AxWxZqai38OrokcYscwxWPMeaC4QtoF+0CmhKSsJGgMAQ25xsJoNo13vHR5Nl2LwNBluHlDcLb1pWKmJ4EskjaOOjsbE7eCiiIGzTg+kKZvKJmue4dLPZZGEe6oIOForEsqv+XkN6UGGMdyVoaxiBs8eCs8UUPb5jFinMOYNgKIjiQ58byk42owU3SoBVIApOnG40Uv6LXxsrgHk8clFZecDiGeZ7AdG/z+6ISIQy6gqHHtXjTV4cZbJNwMlDvYcyjdgogXsVGxbAr357sqkRCwDEgi1eblem9Hm8lpPUNh31aGMpWTD5Ag2pWPvDzI2c0Mc5lEl6CqmYVNpRIB7G7/L5g9eawCzUAMbltyeaZuBXxBgoBMHOF2ToaAB6n5wnnX6Du1J8hxcTIldoYYD74Cf74LcICBtYzXfHLDWKCEitdf6/SPoYkUumTXhPYaUiWXjA48CbCMcFmExEZJG+9pPkivboXxg835BMo9r8tgYljfwiiI//VfssT3LMPh7Vv/VKJ6Y/buOcGGF74Zqq0qZmibhm6bM/zJaP0xxsOP9YGEDHFE3mTUdkJ3dsbqKB5KUMirSA5gWpUsAYwjQuOnVKnW4KJGK1ZZt2CyIlaLpYgjJu5nMf+2zfc1CqdYvAFRDJAc6Veux+VcDopytX+RxJ0YLfDJxTNEb5eA1hMMRB96kVPnBO/D7O7RhxhFFAAUgFZE3xW4Nzi/kHqAv0m6HYBDxvrKajPxOfbhYok8oNI0jVQSzNDWcqpT/b9ZskoxZsNO8URAmZ/kdWAGlkOiD2BlLbwGat0JXr5MwF6wSidDv3/WnS+pr+JhYsxM/emlcSeMh3xy41NETMNtjHXNXnN4PHZeMmJdazxPdMX5PKSh/DjELNkqfJV/fU874i3RU2q3AoiTePxiTACLwfOwb4WQEBAT7xlTQ1HGQTduz906NsQsT7zo+nmiinbojwI9sF3lC8k3HvLCBlvyCsa5B9Gvx3hIe9Rx+k9oFhtdvMLi3kFLWmhbF3mD6pWPvDXEdO6OMcyiQ9hayeVMD6YMeO4hA0Vjz6XT5TRywgQB/90oOEGel6eDx29nAmUQ2HZnPQafJbTY0RRp2BlYlIl0N1ASBuvGyh1U3+OE0WzWsi5VGsR0TOTfbDooJFGFWHSE/CoUIUY2H38Nx+tCEiXHgRg6cNusag2qCzZl9XIoB+jkHMEu85M5Q9uhIUxULQh569ekdUBRbQBYE2CEWwi3q0NuB6Ui/5cRhiPqJoCQ5lWDWepi+PMhx0YQmYdWQOtCR3p2kDU/aTtvv6nrpygdI7144oCjyAG2OxC/BmftHRRet3ykWvRpgIGMmABjYqSNl0YQsLeDD+e7deEGuK+n6vr5N0L1W7/GKiTFeed4aRtgP2F/ZHBB6O1qbd+40Nw/KqMt2JC4gWbdSbUDi3WOQR9Qu7qUvIANZBxHvWOOtUg6Gk3vSDzcrIa8MXeZhOO2Wsh7EdLcruoQ9pcpgN8LtmOq3x3M/g2es8/lCotPB3E6TcY8/feCJyxNL6vWsGzlprFNEQZf79mYvktHzHm9sgrAJzCQ4lf+cofgkqeryh7wQZt2Sz3Fb/HHn2sow8w3Q2owU1cF5b/pZzEBCBBwdMNxArpN3JRUybQ9HJL/MQ9EwY5ef3/O3o/vG+/+N5/lm99sdzLdnd1zmUSXoCWT2pKMvHy432Y2ZUQqvYaP1WSOmR+JoUD0htDJu/wcOKwbFAAQdT33pxRRoDlDZoYRQ4GHlAlWa5oqcI5OBACwT8Dwl59WOA4wZdV0g6In0Rq+kqTfQFbg4FPoh4AuOJSl00gN//PHxYnvx2lomQRqPQ4Ysb50eUsv2Hk80YuGbqjfO66GCHKVDyuxekaLGAR2iCakirysWE9xQ2EqXHJZ8kPgOmCY41nJdoMm8s2tKRXPK7gXR7mqXmMeuP7UbuDo1a19Gek81FGkYDW8tnailOVqnyfCjwuk3pyaMynWk9pDAfbVHOS+EGKQ/pxY+YNju1qqmsKBQQNtrqZ5sM50wj5ObGgpF1HAe4BpxIRCSnrdpqNgd+ESq/c9CJx3c2pY7uz98onS7Ov66XV5Jb65WM9RNMyvd0YrBJRYSy+SsRBx+/Abwz0LApvP2jyV6hkX1iSPs9c2m6A6QpZeD8I3rFCmB9LDe0gAShKC6eRgEE6kh/+Nsy6fLjXE8RzB6Lm0V+HhYrjv6Mxun0PcfZsGOf1EoTitBOGfltUXV+Y52zDY9urIIrFv/4UR9xPL9r0PeK4Acxwfwc/MO/r9xmHHA4eV/fU0+goqPbgLvqSJ2QYgEa+80x4DhrSAE+B3PBxKebx/NYQ/fN6rU/9IXkgI7OoUzSQ8iqSYVUwpsjFssrwxZ6lYi45GjViKS76X5lZbMjRdM6xFA/uCGNVgGRQkQlgQUbMme9hxWzz+Gn0IA+flJZ+JEjunRBydMMOTPTuZDdKl34JLM7xQvt+ppnmVS43VDgsmTjbkOmHau6WDtjuJalm3ZL5TNPMRWLoIrAIgxnHA3pFZDcklQ4yKkBjycKVBChAQ/bafmO84ism1cobHBebIiAXlatmIm0JhLdwDj6xUvgeLSFI9aU1Tq7oMBBBezGnfujkgYT51quyMnyy8MRib5f5qyTuz+ZaqKVSOXpaIZ29sJUT+rKXIytdZmD7kfTH5GJwI9UGeob9zY+1xuGeEB8AJwXuAupnWwXbfAgLYFIGhcN80A/XdTEIqrM6AP/NHON4TTFBmpBWvGcHb3StulxVWX5YdoambR8i6BQ7KnWFWJNBSGhNDpGi/wwLUpbUiovliAAipWQlgeuNLONvK/AHgJDSTk+bLLu+TRSvAeHBJs7ch7a57Sr+gnbQD9EpoEjRhocm0D8+6eZEQoeMjng3/FUXaM/1Y4YeSdxeRCZfPlOgzLoZUdz9O3747vdjwIHUIkyHQeaimaNOeYmBY76jbXPNspMsSBAZDjQaW1cC9635TpFotfc/EZ77tW7DZGtiksUUCoUWLVPk57ks9Vyj/3a15QmaYpnsebUH1v3SIP/RnS82QBb0EU+9m831pjxfp9Va3+815ET+juHMklPIasmFVNR/FHsP3jIOE1+qQj0QSqy3DORal2t5KIjSiDLxg4RDQ4HCjZaVCxiInl4KYEEGRxkuoLbT6EBx/s5UcQ4IVUFsmYuajZVECKJeLnbDVElYKmg8AKaiWgNjlLNHsNMF6TpUayB6nbETxHlRGoI0VU0vLhB//PcT3PN36+0qWYqa+2mU5HA/ED9gQ4JqEAQAYODU7vU6VKz5OkmEkpcUjwAe55XE79TS5hKRqD9gcJDPE3j5hBBw7MHTixaVILzjLRLOB+4D69P4w+1Ae1anUIXkARdp801F4ZAGhAMPivOBU3RwnnKecZz62tDmviRFmW9+whS/aFzp9Pa/X5bJl1/jMwVNG0bXls0XGqsZ8bx9W/IrnjVY2Cz8frwxSb6go0SInexGhWz0C+IuxK/eTg5OsVNKhdAJl641r/qmfMDDjEih5ltVCbCpgYRShYGAdJAGjTQKbX7YJKs2R4pkrIbIvJvKnUfRpLRDxRDiDKDZQK/KUgMclxAiFhoGISzDbo/Vp3zOX4xeaWHEQR7hm76/czPyRUcxn4sHAtif6DDqec53o94TyJ4cEOts0z0tMTpJ8rYx5sGnpJUPLYm+eZd++X87pH3bZhNoS4KwzHYrMH+/+o7wYwB6AYKSFGNvW3vQfPODstggOO1zCze+2DcwL1BlUe3MMWjYezv1yer1v5Eryc7j3MOZZKsn1WTChVx1749TpCuwSKMnTr0nm+te7Z0veLIBUVX65LiBLeIyliQZaO9eG1Vr7IRkUlgAkGJs3LLHvM9tVw1do4UKra5/Ha6xNjwRwzKE1Cf4CVY4vS8XjoS1cNjF2064gnQMQwDkId9uPBAJYKpaQ6K6llERNFQYAAONFJK2C9LHqMrXhGtQLqRpONMP2qHXnNWRttdo/Jz9IIN0vfWCzIUsmjaDUZHmKoMi5XTRmS0CZ8hXU+Ovmgyb6Cj+mrqH4KK2ndujjjxeGYgTEYEFdyAIPtmlFJH78DviTRTtGbzKELB4qmLo0fYdHQJEIWv/11Phs1dL3ekYT2rFj/V6NXbY5HlANeDqBQcSkoaBuFc/ZwAqO08/V26HryGNFAuDswI0GxOpLHwiwTPGEPT/NhjIroDaUgURISlt0HlL7ld+ZwQgb/9oykCxobX2lYXSBkyykPnBveN+49W9ayVdaD2kstHhSYeu7BYBRkKYCiZCsVmhnhKOMW39puUIcKnz2Ffr94II52NewaNDLDKcMo7fj/bEP9DgpAFYIAEYW7BUfxh+hqz8Ty7QHB1MOcbmSQ0RRvgE7pt2rXfYI910xRRsexVtuMgQ24f5HSxOvvh5mUFKjhoSDvjN4x3PFLYKO4MKhji+bkBxO++X/ta3mVprHEYBoDHv57h6d7z/aOx0XDssTlDxBhMCghkhGUwwEXpzAL5SsGPCbYG3RIRtoj1LPh9Vq39Yc+fk/o5hzJJTyMrJxWcQRDWYjEn2TQqH3v4kE3rqJKmCdFqDp0vq+hFXrhD1GYgjxdSnXO7tTRpa11Zq/vixzura8sMVmSKBRWjox5rYvCBaFjQzjw1j4eNwj1RatHv/NEwf+yvnb+5XSOE4MR3og8WoLZ9J3i62Ms375buP88zhwM/CmfA5uZjxIGYOa11S9UcOxVoF9X4TSuCx3WBCfppomDassWrow0UIRHZQ0YkMDaeAaM50WTe7v/8d5P+A6C9d9t0RSFdJasLNBA5JCNALE1zvYiSFy5M9Bm4OcIVuFgTn4Zng5QlNiR2AZnG+wL+ABzsbR9OMY8kKIXLKmcsumMeb2L66pQ7/tYQDNLRxKvFrOcFo0DIDrybFokndY49fxj94sYuLF6XfKYYD5XjoCbSFf34fSBCSQgM2R1YhBctpa+rlRGhRKQyM43RWTgGL7c5LwO9DaJiuE5sSiE/ieZXyMeNB6/jwQHTjFNoN5sAHZExvqe4KSSdTaxK/teHL8pAE0RMpV9GwNbN5juqdgjMIO6/9NMRhbMgnXBSEmmpWWKO8T6FQ4nrjVVMGYQR9sMaR3vm74xe4vH2AhcMB1urF8HRA10aKtOhkgYGAw3VijWfvp+22swHzIWSBfMafLFfo4JarPES+T4r1/5Eric7j3EOZZKsn6pJxYKLIHUb/sDgLIGUmBElcACSOoU8XQgowHkASJsNnw1+qJEXzSSFQ9BCZ6e7tI44xhzW4UKj8oEGfjuAowkc99tJYjzgauAAwAFa8Fz0hYpRJBDhghC35atjZMH6nd79IEoLbBYqZ7HzRTTmpSELve/9nCEuEMTMadsR60Pyaw4UjQgYfTR4nNJ+PJYvXY1HZXWufZ4w07VCp8EeQbzm6MOxQTJvTHHaGDRgeCGbCdohpJJGPdrEYCp1pW0sDCUpq2A7VGojugkHcXiHC6NKa9LJxXWzMIiSmliAsEABz2pf8xNfz5QvpkRUkpA+BeyCEI+gVDH1hDVOklWztLl2RrX8XzTC8GjPi1XNmsvSrkDGvIZDDuzue7fW9GTqwvIlarwedceJjcW1oYgF6iiISOpoK3GA2ABM6uhfzKDpWqLhDjGHwkiotn5trGEkQGoav1XCLcDteOELI02kFYV6+H2iwSmCpKBudpGgrcLCvjb5Nz5nhT3kBuHoACsIdRdsGiY83czYx68x0ozq9G/vrS+cN35QAE2CzrFA9l0/QKRCnw9qNlS5CaIoe/SrGQZGpDcpzBAhOguHErRtQbKQPN/LQxYYCACqrYG9ZdP0czau2s82ZI/Ad5xfSzbu8sjVQZU1fulmA68C1AhwLpv6KdpviKIZWFdKnJbX4DP9WiLv0TDvWvRJ1dof9nqys59zKJNk/VRNKnKABaW8+oxabApEbCJvHSVCQQ9eKkgBIY1JPAtMgQUEZLNwSnTVsZZws02mJd2084V+qEYFETicw1ldWsqxxxzjKbL4mR64OKgrIHWDFotSA2lkLDCIRP30QEOxIxI4/rq3x5t0Cna+63fs87BSGN9Pmk/zZva/q06Gohles41tjBVR1HaBIz29cwvjmKEx2qdf8pe+MdbstnXqKsxUhUMPLsOgFrTw3/TeRIMvtaN9GMeGFTQpX1javD3eFIegxcI8DZi00lTWwznqfmUlA9vAvSO9juIK6Pn6LdZ6zmH+LO7RWuBkIuWLqBFSf9AMtiNId308xchhosFJh0NJKbYgQYBeg+YZflHN4aeLezBWx9YV5M5GEWJ0Wx0lHmUPPhvOGdAa3dckUlRkS4bCccB1YROIxZ26xSD8B/F/rIaIFjk0SfpNZxHHIiKLaDGizTr1rjcBpJGyz4WIHiJ7aEEcfyiauPat8dK0QuGYBXbcRCH6DIeSUVM47OBfRKEeZSFxTtCiQQYWDfMDzh+c0V+fSMcGUllGXzt+dtA1tyOq3BQia3Nx5WIeDyaOjUYJxt8veX+17Zb2uiSD2QBXAim+bmGLUHTxXlCa3A9GwQ0mNpdta55lov52Ktt+tlTbsh10FiCFhXrYEdnlz19iVI1QKY6GLAPUipAZQUYGGwo/MvWgeQ7Bgh4D55nf+RmnnmjeYX4tCDsd6/cT5vtUrf1hriW7+ziHMklPIFWTipEhOzXJ22Ca0AYhg4S6apcI2TOcKGg7I8IDYDUjiPiOuEVi+LjYaQfCNpmO0GjKFvQjroUas4hWkOPOz/RYTBBVPK/bUPN1UCEBjyUfIAuI6FDze6S0AQzH7hc7XziUfcemV5b70b7YgHQdGeC49guK0RUs/Iii2E07ZfgO0VqQvKMxKqULCnRlfIcW5ULPUo3n9DsoSOYtSIGDYzC9STuT2w7f+0na6XPrlz7SruR7ZB9GzuzrpVPNz6d1usgsxsCEoohizbZ9hsLJTv9SHxvHIaoKp7zDlxHt+aCFlL8brfn+2+JNPCxwqwAAIABJREFUGShmgD2GrByaluxLVIFDR8RQoIamoQTYXE236Jq4gQI29scHGkSdFzpFio6kAOIYPJg0XRqXrVkhYHdyGuoTlnzyZ+/PoD50XnWxU9BFE2cL5xy4RTpei3pcLGAuQPQJKXhWeKMinTy2hHbYNrM5ajkn/ApS+BsAJheYdWxE2TCPRj/axBcnSsJ+zi3NgACuR24cMRY2QIAPwR4ogsIzihXh5zXoTenA/zQ0nL528yv0Iu4SfcnRGAsyEVR05mVTCuWTESGkZqHgU0ZtcOFQ6sIeFtYhcwC+YDzjWPyW+p5J7YWAAqrqNQOH7odC1FaVi4Z+j8bTMVVrfzzXlF19nUOZJMunalJRDcGuZuRtMDV93fnF5cXr0qszdXUhZA/xYgOmBhxodDQxBomamSoiOXq06lPtpLBa1jarjqjqQgG7H1OhSO2gujqWljX588iXNmL+eg8vh7GhBnHFm78Z+UGkPQDI18Tkfko/tCE5+UCVUSoNu8TrffX6anJV9fQKcdKsgBrJzwHUhTcYQ0e7SPOjC32iKVVEm7J69+/XL6gKnQocQRFRFhogEoRKdCz+qKhEi/WMtOLOl/fUNdHvNBpTczxSjHSo9DXbChiYq6gEBg0JaFNmrt5uCkdsCIGm3sEi8/BFZUzVLRqq9LGJspsm3//fjTXM11OWb/GqgPG3pmLRnH96c+BncyjiQOFGNw2BgLJTwzKRoiZdiMSoe4aFcfQSw1oQTUaT/UE6Tw1lfEbsrh9jA9KhsC9Vj3QRH5kHMAY2hGQMOOepCJ4PDXKnfopJ+n648fSzB8ZgMRngDSjKYYEdMiCA+sBZIG8u8Indr6psFKzQsDGcsHSLkfqDA8oUO6PAGebVuQXl0ztqH/GoCCfCu7NBmYKmmI+RT3T+7t56Uv2s0444zt6ARoND6Gj9rv1QmjpsqMnwTo7VNPtEEHelHxVVqad+9n5v5GjUohd+5yWUyOZvpFpb0O/Ibyy98YBD6adND4lYbBT9MNHR7MKIPbI72Ij8PCtCA4WmMbZZqfiUqrU/1vzICd87hzJJTyFVk4oYmaAKZToGNik4FgIuAHypID2ElDAWHToHLPZhCpAYGhubqM1GqT58xmiUbVatesPUkp/puYgyQhqtKAX3BAk14G7oCOldPFI8SNcxSgGidxBsfzklnaqIuCd9LaQv0dXDepePvjYHG6NimoBbj0mgPj8jJg5/0wHS98rxwlRD6/NoCiI/+wbJvMWqKodzUqfXcFOFCcD/pxNWesMHOYTsQEoiRs1JicTvg8jbif1iPxR1oegADVRFoLsCj6nWFMZ3rGrFv8EN+NBFZT1ibEbKbdswTa5pcuyoslbkCYNnwzlQ3PLGiEXGabi4SrrToIvkQIUDjXs0XcRAXXd9rTZFTbTXl83Zyd+IHx4ahXg3100nMAePK543Giu/kb6GEguKKsBUQC5B9Bn/VFMplv/EIy5H41nBIfner8uMkgzowJpVKJKhPxgFwDQB5xYOJRw8RPcQ5eN7jwfg3YUsAaOYwJQTN6szJiTh1icK4rcl3ABR+JrnnG4cd6jdbNq531QNB1Fd2TKctrY3HBtvDo9bbuRmcf3IJkD+MdaGjMcigkcoEGBJKNSym100A7lO7fiTozGoqJPjsSANx8/skl50qW30frua0aaf953tUPrBcvDuQ/U3MJfcyIcZnGwE2FQi2kkIBI7V0ex4yOPDnFf3SdXaH+91ZUd/51AmyeqpmlRM8QZVyXIx9SO+pVMEJwpqBcRZavJj4ldIi4KX+8zOLQylxvLNEVohu2netiACdC15x3Su31jEsBHTpwsW7P66CpdVm3Ayq3YdYl7ULORgBBAFCIhQoqqQzQad4/OLXhltqs81YbaulkUfmyqHKh+gwQDeCvKSJU470RAJg06F1ck8L/CriIJced6ZBjeHoiHtmJE+5s6G50jHSzJKn0WbsjpV6dcvCJPqKfNYjoUeg/yidjEVsXlB18XKZG6Ceg2cJ33HLvWiJn4gfF11y3EBv4Bd0ZBmhgOBAh8dLUE0+dw0cmf0w/yFI9UtjXs0iGyaWDtdDaopUjAWCswQGUTDPNO626ie9mvcGNh8jhnYGDq3MPATNK2IZau+4Pto6kX2+XVqEd8xlcjfHxx8OD/gU/34ttoZUrNaZxyYZizWUJih9KutRhI0r1h1jPPDcX7g82mmaM7PoaHzhygkMJSUO0UFOYuxeI9I+UOhiZRhem6gOI+YVr254LE2ywI/f/GX+fK/kUsM1RQKfiB3Ce5RRGRB4B60uaODTjy2/g3a4gka/gHeRdhTQymi/bZ1dX5QpsGW8yxZIF8GTDUcWVRTx3qvBLEY3PPJVBk8JzxtFe6Ha4kujNHYXvQBvh3OMN7NQThnP9s8NGCakShFxu6UE48zGQs2rB3IxOU94VgZ3qFxplkIgp5Nqtb+aHMjp3znHMokPYlUTSridew0H25Dp7j8uAEZ8aE6DgHXTK9ijBeuqSptapbIoKyDdNblb/yWQUUH/JNQokHTuz++cIARQjoHDWkoVGqSnidagU+jsoUE8njkRfTTSga59eZdBwz2EJWfWNzgUDLNRbwnAfrt+02SkQs2mkUMxUm/zIkUbKD5YdFIu+NHDszjUBCBwgg2Lpx4icHZJ+8hokwvtalmIrfQI8aOH84jG8l48beWBaSDE0RgHzRtQXrNKJ5fn6CdOhfeaITjujpYjw0FFzjOQY2pRDpVcMYQ8Wn7zoRAEL4mrOe4KOLBs0MaElGot0ctlVeHLfQKstBv254DHv4Wf8NxBx8fye3hrMztdqS0no5iP9oyglnVnK74217ESccSjTePUBFwrn5ye3qaFQsn5i9+JyiU4tzVmyTQe9nyoMTgYd4siEEmrqOMuH46y9w8IGsAYv88uXNJ7mMzUv5ohxkYP1QuY+Fms9kD/Kqm0VdriSPtiAgnnq1tD/QlCTswnbAp8OJ8XvaGDM4wsLf4rQI+8f6tFxhuTTSNaS33zCCTfdHvoyCFF54DhYlwKlHYBagObPHttNUeHMie53ZBG5xPcmjq7A2O08UuW/cckPU79ptIK6Kmsdr4JZu9AkoweNgwChyvNwIo6MJmi0IX+B5QATjIQdAcXkMQDpQbXb/NTrTr377noOTPG9k0oVV6drC5DjY8y3GLN5tNoq5Qj2UTzXoCe2jlNQRH7mhYSnLlEkN9l1UtVWt/Vl1/Msc9qhzK7t27y7Bhw2TUqFGejX788Ufp1q2bzJs3TwoWLCjXXXed+fvEEyPpghUrVkjJkkdq1fbt21fuuOOO0LZO1aRilIiOl75AvTvVaTT2odOAwhfwfTE6pwmKGenbuvuAVH8uUhiDCMW9n03NIKGl0wkaIM7oB64PFY1oNhaMsl5+xiWxNrFXNg+eVkHB8VhYhz58oZxVIK83HAss6DDryng4lFrnFTJvAx9s6B2LxaNcp8GmelXzLuooLjrbuDkdpQAeDgsnGzBZJx5/rMFuwi6XVilm1HaYomM/TUSPVF40AvugiUmHI+j7IJk3VoJGK7DRc0KPr+EMfuflIoToLarj2VgEAfzU463KZzhURwdZNMJ5S1wXdZo1ntCuKsVmBko6Lw9Np4oCxu44y4FiVFrDDmz9cZtyKYw8JjF8NuaRkSotHgADDJ69zqM4QloYmxPdohV82La3HWLgjHteVdkriotVnU+HGZFzYGhB7+JH9YXzkl7MvgYtvQfHFL8LsEdovk8ew40cqrCR8ga/ICLMYIcA7g4bJfxu0FDtjbkKKMnu/YekWYXCUjtNx5opep1a5SYax+rfmb5enapGJBvMECj+wkYYkXGbOcP7fb89zqi/MII5Z812ueT1iM69DQVgEQk2snAoES0Mq5euSbztYh99H+S7hZOG+UOqId3Hliu1n1uQ85oZOjN9DjtyDF5NvJehdOa32Qh6n+nMAjaPfUYt8brGQz8UNH6Yz1O19oe5luzuc9Q4lL1795YOHTpIo0aNPIdy7Nix0qRJE4Gjee2118rixYvlrrvukmbNmkm/fv2M7eFwtmnTRpYuXZqBKy1//vye0xnmIaVqUpHSBrixj9rXMrqsSGsAMzlu8aYjCIr1tZN8m2BlFqRoAunBDzWU8kUj1YOkBcHC88hXM2TPgUOG6uW3xZvNIg26BjSdzq7VY5hJKyOt1CtN8tDGLNnUKHqRIgVM/0krBcUDmp4CziAwk8AeoWHhgZxju/oZpQknL98it/WbbHaniE7pgovNu/cb8D6b1q7GZ7pYQvPD0eHgccM6NJJzC0dwb2h0WlFcgugJyZfxHRZPYJds4nC7AlVHwBhV1QTFYeYhFV/ohNnH+Mm8wYlG0RFoV3Thid/5dMSJ3+MZ3BZFHpK4XlBIPdS8rDcsIzt+pMMaC4ZnpLlFueAwoqSpYvxS/sDOvjlysXdeVP7bNEV+KX9dMU/HRtskzOLKanhEhhClhzILNkG5jjnGUPXY9D+6eMvP8YmGz7Ofl+b7w3fQZYZDz40ideOD5hUdZkRKgTs2GL7KRQ1tk938rtWWGdSCCsBGIsqsHXvtZCBSDlUg/fvHJg9V8N9NW23SxBqTqiNzwDEvWLdDHmhWxpNMBWaZFcBB0T04rYC24HfAhughoDNIv2NzMs+6ZvS77I1fTUEJJUB1QZMNBSCtDyKH2KDFw7vIzQax4UHPjXMODhWyTbroksf4Rb/1eEG0WCR7j4d83O86+Xvjd5iLeD6AsNjk9NHee3yHwvHPnSuXyVjoZxcm8hvmvRqtT6rW/sxeZyqO/9s7lKtXrzaRRDiPJUqUkCJFingO5U033SQbNmyQIUMidDlon376qbRv31527dolJ5xwgvTs2VO+/PJLmT49or6QaEvVpNI8dEiv3v95hJR84tPNTIUcwOt+6jXoo6le8DcXZhJ54zNN/8HoAl7eT3wz06SWDJi+fGHZc/CQIIWORmofvPBJBowUFBxOyBbatDC8B9oawHdEV9FYrT1qwQZp12+y+Ywge2LxkAaDYwTeMVvlhmMCS0cpOFa14rqRIsfLEmTF4DG0I0cgPq/bK8KRpp0vLhoc39b7JfUP8D8gKcZiyIaF6IKSpxlHVjtPTH+hHxbYRd0v9q45iGg81vxEwQOUgIiVsvv78ddpqEQsXW6tLc+xdcW63/Wx2MJ2PKPdoyZQBrwDYH02UlmxYlYrKmkiZfYHbECnwhBNQ6qxxtmnSt7jIwUTfulr7RD5RdTCFE5pLW1sUIBDRCOdli0Jxwgroi3AAtppTa1dHFRZzfvWuuf4DFQxcOgpwaqrt/2eG2X8QIX1xeRVhj8QzxCVvizi43F+le52hBS0RMAlstnSfaRRwvdwWJ75frZhokDaWDf92+bnelPEz8hmgb/h7AFyYkeE7fu2N47giETxC22mcbQ81o5u6yj5iEcuNMTxbMSCoiod+Ml41LBY2IeAADYnQc2bl41KGdwsKdh0/1i4Zz139LuOEII3b6gul1Y9I9brKPB7uzATGTU4zKC0iqbOFPS8kFlAgIFBDPSzmTgSvtgYB6Zq7c+q60/muH97hxIRxv79+5soJFLZy5cv9xzKadOmSa5cuaRatXT6HDiUN998s2zcuNGkwNu2bWv6fP7555mya6omlaZtuL5mCc9xwQ4PaQqowATptNovS3IKcmFGVGvBc+l4LoLkEdVAZACNOD/NL8bP9AKCNHjuY48xzhtwSJqL7c0RizKo1SBSSpJspkX1Lh+YqMN//SVNX44o7sRyYOwHyUIhRHW37D5oIi3kP9PqNDhOv0h1dEFzG6KfHeUidQgWbTiUHb+bneEyEOUA3Y3mqdRpXZuegzAEu5gj1iTVvGyo+AbXnW5+vJ4adxgrakWwvh4zljyk3piQxxHHR+NU1QTKwFQNmBxRvkHjNX437Q9DG6OjhzxOXx95V/nZJVWKGXoRqk3BGQFXXoTIP52vTjspfrrylEHlJsjv2ZzXbYipjEdDShQk5br5FUfAmQbvY82S6RhdHrN4w05PujSWlvKsP7bLZW9GUq9ocMzhXN/0fsSpjaZug+/bvDPeFErgGlHsAMwbNocfjVtuivp086PUsWmXsIHFppfNxnnrqn5E17AxovxprHmP7+3COage4beOBkcQznilM/JLuaLpmQV7XP6O+TkgIpD0I9WVH2ck4TCM+un3oA090pAlFE2BUzOIu9a+Nj+9eT+7AJ40cFaE/L/jJRWO0A7HMUHcrxxvxebdcuGLEegYszFB2Zswz8buY2c6AA34acbaCEl5SI5LjMk1CrAbRJZRQc+WldyT+n5StfYnYudUH/O3dyi1wdq1a5fBobSNeeDAAaldu7ZxIKdOnWq+rlSpkhQrVkz2798vCxculDJlykinTp2kZcvgHSCO27Fjh/mPbe3atVKrVi1ZtWqVFC+ezk+Y7AfqcfqVONUAqwlEx45x3JLN5sWvyZn1+Zl65GesugVWDBQgkHN7rGU6lo0OqE4ZshgFiy0q9RC1BJYJ1al4CSHyg2ZXN+rrIPUJP+MCj7+RunikRTnRlCcAl2/dfdAQAiOyOKdbyyMwcNHsrKXRsLgjzYSoAyIuZxfIK6Mfi2g3o+mUInWN8bldSGQvxqy+RxERHNeuP8711RrWFDmwIavn7eIg4kxj8cXZ960jIPPW7vTgAeyHKBnuEYoSwC2dmvd48aOICbKnXTmMftFkzYKcNRz3+NczTCqVuFl9Tk2gjApqpMLQUFwxs3NLU7HpVaLmziWoBEbTtDscj3rB/JuFCUytRYvQUrrQD7PMwrFouuRaBhNYYltrOF6cl+YZhVPw9uilgshf21pHFnXYIgOoWL7uguIRJ/yE3ELd+KBnTVwwNoQgkEcDxnrUggjWTTfN84d31KiFG02KvcuPkY0omh01t6mKtEwkq7Zt5Ztov3M7jUrOShyDDE6RU/JEO9x8B2dZOyVUAGJ2x2/zROwnqs6xuYj2e2K2BCl4FDViw813XqyL47VR1z2ov6bpAkE8pW51/1hk6hCAsDGpmpLt5/80MM55os2G+yDiCtlgRKXj2URwwwac76HDhz2+WfMsb6tl1sKsbs6hTLfwP8ah/PPPP+XGG2+U77//XsaMGWMcSziY+fLlk+rVq8uLL74oJ510kkmJv/baazJ06FCDtQz80XbpIl27dj3i66x2KF8btsirbH2wWVnD1YaGqN2YRRtNBCAoYkK1E150kNoOvydZN3AopGOAriuKedC4AIA3TEeQ8J2WY7SNRL1xfq5TYVqGjhELVHwCIB8NGB/tpcGUPgpwgIvDogw+QlAD2S9nVjIjoqq1am1N4KU9W2dQzSAOFU45Fu6eA+ebnTYWF2BP2WwSY6qWAHuJlyJbLEWkoPslPhWOLYoDUHylG1J49376uyGNf/1f1Y1mcRjCZD2GPY9sbKTuq6Man99RW+opzeJoBWY6Eo9KeeJ1tQKUJrHn82AxE/gnAW9A0/g5fW0s2oomacdUuB+VyUu/LDDYTFDM/HD/kao1etOF82qSbF5HLJUh+znrwiiSeduFZTxGb47wGejCoBZi5mbBfDLi0cZR11qKKIA0GlruaMBLL9u0yyMU5wAo3KtbuoD5U/MO6hPY929XCmuuV2yWXx+x2PBAooI8TPMjMedxenMYbSxNzaNTy4xG24IRGIuRUThvqJrXMrcaX46+OpOxaecBk5kJyzXLzVLpQvlkeBSVGkJpkKn6sH0tqZcmc6jvW5Pp+9lD44e5WQD/aOvXI5K4QTyYYZ4T+lz/znhvTiFVjSIjcEgCq4/s0fu31pTtew9I0/IZi9Ls8TlngPMHhAlk9GxZSWaur8M5lOnW+Ec4lDt37jSFNyNHjpSvvvpKLrvsMs8Cu3fvlty5cxs8JVurVpG076BBwZrI2RWh1LQTwAUBLI4GDWbgx+AsBUU9WG3L+4yVTiW3ogbi66IdLrZIpYFkmi3WC087AogiAFcI/jc0DRbnAoGU0Oqte01VtB9dUqyX2LtjlngOHugr4GjAKQa9h70bDtKqhYY0imrQ/GTkdLFUvdIFjfMLhwWpeuCk2BBtxfdswOmBLgMYy5PzpNNq2IpIcE5ILRPtfnmdSHfN+GObwbDqBunDuz+JROeJo9KEyeOebGqwqdEaaF+QTgdv34w/IjKIiCrbDdeMvoyQ2Dg7kuf7qRVpAmUUXwAbjKZ1fnURGnC2cN4bvxRJ0wE7xqgmNJZRoWw3biaipShZhKBlCTkOOSNtuAK/9+PStK/Bpp+KNZd1NJVV75AF9JMStNWZ4HhiswNyca0bH3TOZ3+YbZgJGM1HPzh3mNMojtNNOyhBDqV9HjuyqxVd7m5USt4ZszSUIhDHtTc6+nzRqqJ1P9gXG1nAHzQJPiP/WgiBv0kWL5IVAu+Yat0iuH1Nho+/NdYavw1klWz2g6DnwY1xLIyhfochANDwhZFHDBkL2qLnLiFN+vc2t1tLD38ca876fa/5SUmeDmoqsmMAQ4xNr41B1WPZ1wixgXs+TWfX8CtATORaYx3jHMp0Cx31DiVS0RdffLEsW7bMRCdR9R2rPfbYYzJ48GCZNSsi1xampWpSkSoFLztUMeIFhYY0K6qv0ZDKbFK+8BGXzegNv4hVQcyonI5QaI43VnRTfQHjIjWIaBsiX0FNY6vAZwmuPRJWa0J2clqCtBa7fhS1wHHucnmlMI/E6/PJ+OXS6Yc5Ar1f7LyBBcPCipc5OOcmKdC/53xaERyt1EHCdH0R2jmqU7qA4OUPu4BgWTvb2iGPdhM6knFZ1TPkqe9mSZfLKsmV1c+Meu+UooQ2LtJpqD7VDUB1ytV1uSxSIQ+Fiqv7jDPd/Cqgg05IRSE/7V04hEPnbjBOqx8/IMbUFCq2LrUmUAYulWNoB0yndAGxAGXIu2OWGnUccD4ymoIIIhxfu3Ehi6ZAwlQnK3j1GCwcQoQFikwaJ4x+tlKNnx1jFcbYx2hcJyN+fvMRx9kypIiYA1rxQxoR9Js3RCQmgxqJvnF/lMpE0R4iSCisg0NEbkpEvpuUKyy2dnO08TXWWyt54RhmLRhFDvODD+K3DaMjrsenhrymryHrBAjoMdf6T1plipOQVWCW6IN2F5iImk0Kz8gtzsF5jUgn2DBAlRNNvEFfl4bufHtv/UCTkOMU0I6f/9PQ22TpA2KlrHV0nelxUpLZWPswz8buQ2UqfM6NnabEYv9okVQb+w1+X63elJXqOPp+UrX2J2LnVB9zVDuUW7dulXr16hmsIxzEKlWqZLDvzJkzzff4rkGD9JQVnE5Uiw8YMCD080jVpGIVL1JW90M3+stI5EbTxAQtUnbaNtbOmLtppodxHs2rRjA6HLU12/dJrMgkjakLblA9e1v9kh7WSoPFIVuHykycH5QeSN/Gorvwe2BUagAFyY69B030ASnhYfPWCxzaKc9c5B2m1TKQrmJ75vtZntygH4bp5SEL5I0Riw0hMhYQ0l+gMACRLLawWC5K1qEgBak3ECtHw+pxfC0BCUcRRRW6AbcG5xqNXHQ68hCrUEOPxbTVXY1KydOtK3hfwWlHIQ6cEKSK4cCg2cTgTOH56VJr7krMU1SKo+J31GONPZJiLY2IAhVANICRBa9l25pnSaMXI5EZjL94Q3qUmBfKVBsc02vfHm8+nvFsiwwEzCyOAmuBLRWoCaz9nqsfOTvOAdzuijTVKS0TGPZlY5NCB9HIaE5LjI15i98borV+VE32+bl51Z/bkb5znvrZFEOQM9PPiQbLAX5zdtOk7rYcH2E0saJxekx7w8zvNPVQGBsThqI3EVrZCE71HR9PMZyY2Owyc0FIh2YHsB2i2z+cLMPnbzBE5ht27DfvICrsxLo2ZkGwGf78zjqB3RnpAz0VHMrmr2SMJuPAaJE/Dsx5xnfyZxNXmGJDexMe67r9vufagu+4adDclzwGqez6ZQrKR78tl+YViwjep2waTwzaKvz2/9V3gvd9mGxLItduH5OqtT8Z15rVYxzVDiWKdFABDoexQoX0BQ9GLVSokEkhwqFE2vutt94yVd/vvvuu9OnTRyZPnnyEAxrtYaRqUvX7bZkp+MDCgGIZ8FDqBscSToEdLUEfm1DcL7Kkx2LaUavi6AgWNahJwVP9rFPluyg7Z46tQet4mYDDkDQ7Giw+ePZak8LAvSDNhoWLUYB4fhjcsWunm4UaiLZMf7aFNxyViICFBP6ITUu/+TnOxLZiAUSKFWlF/B/jaCqLBd1bhVJt0OB9UCNh0Yq1kOBaWT0JLCoiwVAI0g1O5AuDI/KFVMvwwyKGsW+Qms/weeu9iCKe7/x1O81wKJzBM2CLpktNyhqmuKEjDYofzR8JJ5GLpXaUUZ0PhgHSP3HD43dP4CScsmKrpy5kk57DOQD5dGlF/cJxdOHC/91fX6oWj2CL2XDNDf57ZLoR0pxQ7rE3M2Fsjj4XdB9mIva6+c2rH2esMZX0bIjconocdotVmY9jgJuGVjKbHxcnMWxUn/FzojWpOMbSDj4dcZ3KRx9ANsA3GYRP9bMV6ans7+Ip7MGxhw7/ZWyEjSypx/Q7CxtNbM7xDDRDxTf/rmeUaUBrBF5XNDtbxCgqMi0ohkQ1NoqeXm6Tjp8Omgdat/qDKDramnILjhYpj/S4Qdrrus/5zw018CCqpxFuZXP3hp23uh8L2vAZIS96g8a+oKzDBhUV//Zc0CwZCKKAY/iqtEwLjo9WGJrINQcdk6q1P5nXnFVjHbUO5eHDh03Bzb59GYsSaEikwKGQA/qgJ598UgYOHCjbtm2TGjVqyPPPPy8NG6arp4QxfqomFdO3iDbc26T0EfQ00X7sxOXxfvzSePpeQRUE8mU4dHjJomnsTOvXxhrpPDa/Slg/2+mUEF7AoDJBBTea5kHUxSIcx48HLtbz8eMmJJWMvUgSg2gXLDECiXPZyj/4rM+oxcZRg/2NNnKaKg6oaXhvsfizBo1jAAAgAElEQVTj9H0wIgqHNE/uY41+rh91jX3vJKOG+gaik6DH0Q3RO6pJkLImHjk/PVaQmo/mGeVmw08qkFgvODpTO6VHiXGOxi+ONNrxoKxChNKvaewjMZOsXtaFEUhTkgzfHgcODSJNwG7B2WW1eKw5he8RVQN+jvystqqNTSyOY/C7HdqhkVkgUWDUukqxMKfK0MfPcfLTdoa6DYoc2JAChcQibKF144MuwHZI/bCatsKSzT2JsXUUD38DEvP11EjVOCEMwL9pRRfAHOBsaZ32WIZitTX6cd7h38lwgOAkVnh2sOHfRDU3NnxoOtqs08jEg9rV1DqqDyc1Hu1q8vDCNn1uPD/QHHqDiGuiao8+IIyzRUoeKvlwU22T8cd6Ln7fazENFM29d2tN0ZkrHoOMFPDz2KBjHZrdpaVRHUOzydexmcFGlC3s5j2R69fHpGrtz+x1puL4HONQ7t27V4499lg5/vjjjUzizz//bKKH+O/v0FI1qRg1QNrhvibnZqC4gJ2ivWxIbUN7xpLf0k4Uj9EpLxsEDyJhVDHHalonGDglOAwkMddgcTixAMgDnI1GDed4dVnt4gSMRbJrLLJzlLYzIwh2EYYmIa9Z8jT56p6M81JDEeBQAmOFtDpgCaDIQAsqnvCzF6k/YB8sjkiTxarMhV3Ld4oseqjghnQjF26eA3gtOLtoxNCiDzCidrQ21nNk5Ss00ntclQ4nofymPh5YUhB168bIMQqzgGN9cfB8g8FFapnFMNF0jnU0DJi0r6b+YXCy455qJrowQjsX9j2BYw+cirYqS6x75/e8Tk0Hxe+0BB8/i0dWLuga/NSK/DgpyS/LcXSVdawqXxyjxQXwt99GihXOJJHWPJk8L6LMWvoS0XNAWVBUQdaDHfsOZlB0YaFfPM4LUvz//myq+W3j30iJooXNnMR65iwSpLOL/lojXG927UIdjs13JrJLgPCAjimWg8hjsTHF5g90TK9cf17g5eJ3j80eGvhBddSOB4VxthD9R6QWEXWoDxFH7kfzFct29vc640N1NL/NCOYOMgSUqWUUGOPZRXmgNSJXMeEsYQoZ4712u3+q1v7MXmcqjs8RDiVofK688kr5+uuvpWLFilK2bFnDFQk1G6Ssob+d01uqJhXxgIjqICqDSIdu0TjNWLXJ/rE48IADBPCczcZqaeoH9AmLBTIv+W5DDNE3Xk6QSLwuDcOmdcHRTzsnsfjXguYIpBg5PvvATsA82oB9KuJAyhLV5Wx0GPG3n1NAjjik12qfU8BbKLpdUdkjFo4nfafxUogqYZHwUw3R96wdLCji/DBtdQZlEvQFryXoOdCYavt0wgrD/wb94glPB1Nl2fZFMQKiv8C7PX9NVfM1MHSo/rchc37OMCPHcHT+e01Eao9ON1VToqldaCekSblCJr3PQg874hU0N7DgItUGvsR4U6MY8/I3f5WZAZXuutiJ50+kqMy+dr9qZr+KVmYz/O49TKTf5rHUlE0ck3KJSE2CtF7jWtkHBTu6WAKFd3AwMV+5YdAFFjiORXNhYB76/hB9RbHVlX3GmeeKhoLFz+4IxhyGXVuIQcf4KACxm1aUsfGH7MuCn4ebl5V1O/Z6G8/3br0g5mWQFzTWe1Y7WgPuqiNt303HFeIkYZ0tXitpwXh+/XuPedEBHUi5ha9ZlOQHl0C1/6qte0y0Gk1j6HVRHAIdcNBJkRRPNijRe+BxqVr7M3udqTg+RziUKIiBEwn+xw8++MBwQi5atMj8+7333hMo3uT0lqpJ9e3vf5hCHKTx4FACi6VbtAW4+09zTeqAzSYWtm1sE5DbP1IbBG8XZ0R7ZqyexW779obneGkZ0JKAe46N5Nf4G5jE/nfFvzD4LXLEsSF6tbBHhBQbLUgmkE4X+vjt0AlYB4VMrXMKCNKFKEjpff15JlW27+DhQMJ5PzsxbYwKe6R6UJEOKg1oCQc1zaEHWb4Pfl0uKKzQDUU+Q9JkDBkZobMcD+cfxtT65eTQHLlgQwbngecGTRMoTHT7ddEmT7UFi1bvYYs81Rsbm+d3z5o2BNXLqGgnrYtftTE2RLZyECJ1cAhJ8zT4oUZxvWroZPjxE/oVGfhFMuM6oYgQu6qPg+NQp1SEB5INcBUqXNnnCNKz1v00Rg2f+2UgbFJvzeCAxR/SkY3LFspArg3qKthbR780RAHnQqobDm2iEV1udjBWmGK2MM+AEKCgvojAIxKPpgm3keJnY5QTGFbIvCJbECREYZ/npvcmGkGCWJsSXUAEcm9Q9OhmZ2WC7seWFiWrQ1jezGg21Wpp9zUpbQQ1NBSKxyL7gQI23Dcao5n4N/mMCXPRfLJ+GZEwzziRPqla+xO5tlQfkyMcyrx588rs2bOlVKlS0rp1a6PJ/c4778iKFSukfPnygnR4Tm+pmlRazxUOJel2aB+/4gB+R9oJ7+9rqkqbmiUCTctUKDvYWDf90kYfzREY63kxEoi067OXVjIVusCwffvvehkKij74dZl0S5N9THRn7JeGg1TXsz/MMbv1pb0u8S6XkTGQqV+jFgJtCz9CeB05hmQeMI90Mli8pF+GseyDjQKcQai55DrmGE+aMhqfHjFvwCuiehgRWFSsojE1x8gPPkNUr1/7WsKXOyooAeIP2zTn5atpKThet51m9nMMpq7YIte8FamuBhk/qFlQEYxrP7djhAMWyiyNyx1JgYXvNLUJORm5ONs0NOiPCCwI8tGY/gV3IBxRROPjwevRRsS6+uGHtXONIjBsCn64v77BUWam2WwNGMuvCt2vSht9SZcU6xp0IQr6ojIZEUXdqE5DR1lvEphWtQtuQDiNeQKHkYWBG3bsy+B0UlUoEd5ZXB8LWPDvWAIOsezA7+0Ntn0c1F6w6UZj4ZSmQTO/uZdGGSEBZIcQUYPTHzYKS4cO6WdsiIOajiwDw3lXGu8s+4ctBtMFRKBq43uMKfCwdvPr13fMUk+sgOMB4gT1Nd2wKV++abdH+6VlGfUmfuLTzUVnJVC0Oubx2BSBmbkHHpuqtT8Z15rVY+QIhxIV12PHjpVzzjlHChQoIP369TNp7okTJ8oVV1wh69ZFwt05uaVqUrGAAg4CXiyI6uimX2q2vcj7x88RPYvGa6iJZnGMnXImpofjkdcwzHNixTlxnHAA0GzMi07fxMJ8Bp3XD5uDiBrJsqnqgxfauR0Hmmpye4H+aeYauf/zSKTcL+Xk7Zbz5DZVi6QGwQJMxyuaoox97UwJAf+F6wEAHW1G5xYmOu3XCFFgelmn6elwac5QKK0MuKuuISgHsXy8WLOHBkwzPISXVTvDyG+ikbDYVqfxw4lpjKHuj4KB87oNNeP5aY/re2dRCKOPOopWpuPADBFJOtOYx3ByWMGK6/ho/IqEomFvDF9k0rd+fImk7YGTvLhna1P9y6rhML+RoD6acoV9/LSZeW12ZDYst6MdMUJB1+OtMjoyto71sLnrDaUOHHaoF/H3rGUVgVvt8fM8A1Ggk6pl/XBPpCpLNLqo33V+jnAi9gdnpB3t0+NAWQvwFLS6vYYbh9GOSLPQBdmhP7btNdrufsT+ftfHTXis94iulsbvUlf6Y9ywWG4GDFBY2PPqKoLNNjh8wxR0xbKvhmP0uKqy3Fj7bHOITYrfrHxh44BDLpcN8weyldBm16pP2iFNRiFWrHvg96la+8NeT3b2yxEOJVRsEIU8/fTT5ZtvvhGQkS9ZskTuvvtuE7UEjjKnt1RNKi5SWCTublTaSL+xBSl28HsbEwmOL6iQBDUblG/j4Fh1yOPtqF60ZwbKldELNpo0GtJiQU1rR9uyhWHnhJ1Ow3G4d6oMMeqncVy2wgUXShzrx+GnHX1EKCGBydQUoi+IVqGiV6vhRLt+cuHBAQRlEjWgo0meMVrGaKCmfSF9D54hX85MQ4flt7OvV5Ovs+pUYwoRIWOKGYvSf6+N4CzZlm7c5YHombLGdyAkphJLtIg7+hKLyzG1s1/x2cEZZC9RMY7oTLmiJxu1IKTSECmC5jmKI6htH3ZeoR8j034FTcwmhE0xhj2vXVyH47pfWVluqhNZlNmgqQ35QlAtIR3Ixsh0rPPp4jn0feri8nL3haUzHNb0pVFmPhGPzY2XDc+o03O4Fx1GRTrSx9iMoAIcBOv2pg9Oz6otexN6JrhARq/wb7IZxLrfWN8jUkYlJrsvYCl4j7DZjjY/J0QAkUsUDZG7VnPeBl3HRa+MNnKxsSKEeqOmN84cVxPKR7tnblwQXQfm/KJXx5juyGJoPshYdvP7ngwP+I4SsPi37VCiKAtqX5omi0T7hEXg3UFhBG4iwQccxqaJXLt9TKrW/mRca1aPkSMcSlD33HPPPbJ06VLp3LmzKdB55JFHTIQShTpFixbNajtkevxUTSodAQBmETtcpKJRsAFQfBDFCm7QxlT5pcm0IexiFjuyQa5EHmNTZGTaqGkDgBsS9ESQAAvrkOlzgxQd1eK6IZXKynLyI67YvFsufDEi3Ydd8LmFT/YO0ak8Yn70eJr7DWlqaB/Hgym1baVl1rDzJpfjLw81Mg6RX6NKDrXctWY61Y4wTwB+R+PCEpbfzj6nJl9/95ZIUQEX0mcvrSj9xi0zTgGarduMz4AhI1ckNkPrd0SuCylRFlENfbiRlCnif7/oqx0V/K2dB0qD8rp1JEQXHMxfu9NAFGw99TDzV0etbJLyaLRIYcYO6tP1xznS77flGb72czIIP9DUNjgonoidtiGlOvWJ6eSQ1/KrKavksa9niq3e0/LVMbJg/U4TuVzU/WLjUOqosO2scZ7S4YzXXvw94ji7wC7esdgfuFywKJBCTY9jbxroaHdsXUHubFTK60p7YvMNhxIbx7Dk7fxtxcoEQRa1Ze+I84c5DzJy3cIWBxK/Dtx1y0pFDQUVMmOg7mEkNlFbcrOF46myhH/bDiVsA6wtmCvsRtiKlhGt0vkX2bn/z4Tx9oncT6rW/kSuLdXH5AiH0u+mDxw4YCiE/i4tVZNK47IQKcPCErZYhdrDtGks6hCb9sTe9ZHahuP1v7OOUYnJac1W4UB096P2teSG9yaaS6U6jC5qmdyxuXHS2XSxwaMtysr9TctkuE0tGwY7IUXt53iGtY3WbMdCtiRNkzuaBi9pPkDLAkoppIQuf/M3KVkgr5QsmE9GLdhoCntIw8TUV4cvphslnkuqFJP/3Rib9on3QEcUaan304iWNY1M/4mrPOwnqlofbJ7RZjoirOX9tN44SMpLnJ430Gzkq2QHfR6bAPyFa6tKmwsimGEq8cDRhaOjI8phnxH6zV+3Q1r1jnDf2deqJT9BZZSsZmOhMa4fuwOLSIhH5Pn95m/QtZHXFN+/fdP50qpyxs09Ceg5Jt8xgFiMV/fc5p3xhheVkVxSktG50ST1OBcrqRPFP2qOQr/IaqLPgk6dfbyNL2fxjQ3TYdQcqWjQ4YBFI2zE0C6ACroHzX+KjR0x6OwfRscdfTv/MNtz+pHZwAYAm+Wv/515Kr9f5qwzWQI0TQVEbmOIR+Cdp0UJtKa8vneNs+VvXr+TEn3WYY9L1dof9nqys1+OcSiR5u7bt6/Mnz9fevfuLaNHjzZKNSjK+Tu0VE0qzanIQoawlZA6zQCbal1kPxvbxOL2i6jXwHnyzpil3qGx9GGz8zlqPB2A86DpIJ0Gcada4UXjoXDdulLcj25JRzCB34GTEgvrFM0eWrP9wKHDnlQfqjZReGI3pCdZTa7xdCh8KZb/RFOABJk33VgJyWrtsIodHIM0VCyE0Y47FEK+n77ak13EwgZFJN3sYg1+p4mjbcfevm+bk1GfhwUjPEZjhlnYglT8og07DRelH0Yw1pzVTjEiq4A7sCVaPR/rnLpCln39Kn/5fHQhFvrHg4Gjw4jj/DaMxPVRJCHononHs/G92OyMeqzJEaTWKCxDVCqRqDGuVcvyJaOynnbWeuHgT91zIMKRa+PL6RiBegwRUja+h+CcQ0kJtG+0CX7DxJzqf/NYm/MzaJ7oTAsix5r6DceEhTzw/Q7eXUBX4KSD4/PZyyrGmqIxv9dwKp2FADRj3todBgLx5LezMohqQPTiz8OHDVXZZxNXeufQUWxugMJyIse80BAdUrX2h7iUbO+SIxzKxYsXS+3atSV//vyChwOn8oknnpBBgwbJkCFD/hbk5qmaVJqKBJQ7iCyFrYTUKVDMPGBMEE0LanblpV3JSowWj48VTcrO2c5UCK4BODq80D395rRCl2h4OF0prlOnvCdN1cGddKJFRBiTVZBYbOB4QSsdLQhHqrGmflHMez+b6nG58ZpJA9W+3yRTIGETlMd6Xky9kp9QY1XB7/jLnPUebRF5CvWYWDQhUZdWj+V9hQgr2QuiFZnhAJtEXUchbUUZbTsdlUV0DBsARnZj3bd9D+U6DTYk3W/eUF0urXqG9zUdv7BFMGHPq6ErTPv5OV6UTgXB/vD56z07U3M6zPkYWURfP+ycJup+tGU5jzHAvmdK7bGSnr816mzPXbNDWr+ernLCa0uU2UHrab/SpppcXSOduifMfQf1oTQrIBqI9kPNCU1XH+t5SXlTfGZLMq7csscIU+B9UaFYZBP60wMNjTMFSjbggUGnw0ay9FjYd+1Mw9EHibxuYYnUXx260ByLAqkVW/aYOe5X/JWIPScs3ext6Cc81UyK5s+TYRi7IBRfkjvVrrbXc4RZGj8ar0SuM8wxqVr7w1xLdvfJEQ4lKrlR6Y0I5SmnnCIzZsyQs88+W6DFvXLlShOtzOktVZNK4xpRRDBo9rrQPGuDZq31ClFgT5tE3LaxjT0EloZYOfTV6jH4O4ycV3Y9R53+xAv8jRuqy9Vpuq/TOl1kNI4ZFfTT6tZFA34LlCaxBn4MpO2xiOOj2YJ0ScC/IQpC3KMfjg3j6MpO6iPr8R8cMM2LFvJzEhz/q+8EmbB0iy/OMdo1Uu2C1eLa6R71aGMZs2ijiYyi+aVL8TmrtPV5gAOG04RmR4rt69EOT+Q8NaRV5UihGWlOeIzG+DJ6h+g+QP9YhCkxF+8cZVREky5jDC+t68PBGe85dH/QK1H6j5uXVpWKyv+zdx1QUhRb9EnOWXJmyUkyS85JBQQ+IoqKIiAiSFRyDqICKiAKAkpQiSI55yA55xwlJ8nBf24NNVvbzOx0z/T09Oy8OuefLzsVXt2umXpd9d69Y5tFluOTSXMIZVh56LIz1EHL9RqVLSpF0cYvqlB6DeWRlqtQshNow2Pk9b9MaJPJhUhmwTPef+E2vfb9+hdMMfqSo3YgT089/c4ZeRZS3QnMBmeuRdDZ5EuXhBYqlFsNf9gYiRYJY6hOLk7b4FAibETNwu9eJ49Q+cGJOcqpoQ5Ks6j0wbX2qxrzYALRctFC3GB4Y/dKO7I/Nc5R/s3I2okKV+iYlxu6SiSMrfu8MsV+nh0v26hqP/JvW7pXpdRJ4r2g4KSemr763TqxlrRKZ0aesdG6Vu39Ru0KRH1bOJRwJuE0QiUnceLEwqFEdjdOKnFyeevWrUBgY2hMqxaV6rhIdRC9b5zqlS4mh2zaHC8ncjtPLS+YSg+DRiqXGP7tafM3BKjJldXYJ5wmjGj8iuC+RJHXqpJM3BWNh6rioDot0sy9527R66McGyL4NPE2762Dgj5U5R3Qt8BBRXHnpMosc/A/IjlES08jZdO0sIIr8M0fN4vrrKhUllw9Dpn0IaUo1TjT3b1r0N8nrzk58NydikmlFbV/efKuJZ13ZYN6BYnPVS147XW4GvyvUiUhIQT4ujpF1bMM5SmpFj+VnP6PVuF6utJVR93o4RAj/tOVig3oYsBNijhEJA/JNaRNHopqUHmSizoH+tekBHEiMzK89dNm2nTimvNlRIonyJcM2TdOoxHbly1VInGNqd604BQap8RSolS1xxfHALyE1/59FGUMri7AlUqOedwl8ByCHgmOD4qWckuN0e35muOKGCpSBfsuFf+NcCNcTSOBSS1wKPFd3n3OsefJ39T7j56KkBaUaR+VojI5Urk1Xb2tkBKzamW9TjpCWPAbCQcNJXG8WLSnT40XqN2MYijrg+cUp7xJE7xIg6ZVaUIbuW61oVjq906S/puViKVnblbt/XpsCXQdWziUyZMnFzyUBQoUiORQIsu7du3adP369UDj5HF8qxaVTLSAQTI2CrQOoF7wVLSa1nquqFX+OO21mnr1gHiiA4omtidbrP5cZqNiXJwm4GpUnojIEz1Jg+PqtEfdDFyRbavJGXJuiAVEnKs3RSrzILnh3wdPROYiiru4zB/XHKchiw6JBBzEpGmLvP7U/h2O35s/bRJZ5EavfFWuzDltyorYJsTK4eTz2KA6IjNfYjyvbTkqmDHpC3ZJXj71A8RkwgHSQ8CtJddXwzgkhZHse2qLUlQ2zLERSwqtsNSJ6My1e4Q4VU9Xie6eowwZkNn1sp48BXVFeu7NmpBt1AxmOFxIlnBFk9J68nYnwT6URsCLKK+Y9Y4vEzPg3OPlQ8sTqyW/ltRVlXK/TJOal3Q7jHrFjZNPZPxLknu1EU6rQfdkxyIddtimTYzU4oI6qpILQkJAW/XZH7siTQ30T4gvXH7wsvi7PBW+cfcRFRng4GZVk1hc4aLq2COjH/RhajFCo4QXTYQ1ICzFLAlLPc/yyKU7VOM5TRHq4xQXbBxYf3hZx74k5V0lwwDqIWZ85vbz1K5qGCF23Ipi1d5vxVx8HcMWDmWTJk3EPKZOnUrJkiWjPXv2iKvvBg0aUOrUqWnGjBm+ztPv7a1aVGrmNWTmcDWi9wpDjfMDIPIKISpwig1YJgigUaBqM7B+hFLGH1vP0Oez9orPPHFg+v0BeBhAXn+hGk4TQDiOhAP1R1vS7riiVcHJBN7WT165Sys7V4qUAY4+1MxKaYqR5Aet+fJaM3XiuATNakg3orjbDHr9uY+QYevqpArttDrucjzEL8GhxOaGjTsq2imtjTLGClyZf7Ut5+RklJQxiP1ETBOy1HFVpj3dQn/aa2n8TZKv61lTkjRe2qaeujcYs8F5dYjP1SQ06bBLBSF8rk2q0btm5emvVpJT0ip5S87tbnyc7ADXHKkTCZ5GnBTDMUaMmVo+nLTVSbCPE0Gc7OilqHGu4SWHBdetOyk76UzL3waJhauXMtU2Nc4PsZl3HjwhhC9oC7Scu9XJq/dRWFpPfucwqDbRxVVcsqo8hATGU1fvEcQh1ILEMZzs/7blrPizjIdWr7Fd6barfaihSki0A8eqWtS4Tj2AyRdHd+E2evowWkdLdK99EVJvnKB69m54VqNDmFbfqr3fNIP92JEtHMoLFy5QpUqV6Nq1a+J6G5ndkF0E0fnatWtFPKXdi1WLSuUYw0kbToFckUa7wkulxcHnOJ1ydd2gtlW/uNrTAjXJx9WGZqdn1njsJieFDU4T+tXL73wDlie1Kil3pxq5XzAfV0A4yXLlGLlS4/ElgF2Vcrx5/7GT+86VSg8MlSd17rJi3ekQr+xUkRBDCQ5IV8lGUT1DqcQi48fURCI43Sg4TQAxe7zYMV12Jfkg1Q9lXKC701a1rvYqX31JevPHTYIPVBacChXJnFz801V82I5e1cUJntGiZsPOaB1BqeJKScho3+7qIwwCmOLFo+ef+4Ss5ObukamJ1FMyJEHgFBqKROPfK6HbDOl4y5cGbUMZYykTI6SD74nuRz3xhw451kjTcQ4aL7V4k3mve3I+VpQxsuhGS4qvqsxIQn/1NwKZzXDwtbKIuGnae+4mjVvniCEGvRBCjbRto+JmVdkTkAE9f8/FSDP1JlkQTqqUlfQRNl3N8RJd6Hl4ABoglnyNcvMidc3xmRFBDV2DG6xk1d5v0KyAVLeFQ4mZ37t3Tyji7Ny5k549eyauv9955x1xUhkMxapFpfK1IRYQCQV6Y2K0mZSSfzEqfNU4NO2Pu5qJZ1S2z+pnqsba4Tqu56v5xCkPChJIwNMor1+1yRV6bNXqHqONq1hLPX2hzqzt5wSRMGQWb913xE+iuCN6dnXFpo4l4x214+OkBHFwtx88IaPZsGNWH6Nhiw+TlDmT/IhG1oJM6lDtklyZerKj5RWrbK+uaW18pUprtfLQJfpg0jbnsK6UbvQ+KxluoM30lVfO3lLf6BlfOsYyY19tI+P4cKq97fQN2nnmJjUrnUXIAeotcFwnbTxFFXK+7DJkQVJOyTlqHUx346gqPEiWwkvaOz+/6FAaPU3TOy8z6qnSpm8UyUBSzx59S1YF9fZI/e1e06WSUBhqPnFrJFPwHYQq1oQNDocSMZUtK+QgVVVK/l65m4NK34XTcYSiqMWb3zcz8DLShzZ+H1yqyICXRdVq9+V31ohN7upatfebYau/+7CNQ4mJPnjwgI4cOUJx48YVSTmxY7vWLPY3KN70b9WiUhUl5EmOKw46V3PQkgdLycGo5itpQVBHy6mmJvmYHSfmzTOIqo0aa4fruM9r56HKXztUceQ1qcw4VqXA9NrhSt7RkxJRVH1LR0G9kkV9d7Fp2mxbbd/qaYr6Ga7U3vrpb69iCH9ae1xo6cqseKmcpJfGCnZonT7VNj2OqXryKjOGZZzfB5O2iuxmWVS+O5Bsq1esks5G7/NW60l1GK1T6i0dkxEb1O8gtLPVZCyZaYwXQZy8QuYPcaJlnseRGhnHXV3tiaSnF5tIz7f/UmcyFEQEpHKVWscVIb4ZdpvRh3zu6EtLb6TKFsr4dvArqmE2OAzQOtHDGhaifRdu0a+bTgsT5W+7GqMdlfwq2sBZz9ZtoWiP32UkDqmcmVZeXfuCsyqdqo1RlS9x6B8KaphnoIpVe3+g5mdkXFs4lE+fPqXOnTvT2LFjCQo5KPHjxxdclD179jQtq8wIMEbrWrWo1KsPqDMgvtGVrJ0r+9W2sUDXoWjPupuverWgVZxQyby9lUgzirO39eUPPNrjOq5j9dxU4atVojs4GhmTJ3BmUarJG3rHwykiZNXU4tSc9jgAACAASURBVMsPHbJzEfSvLTKjWvt3LcG09nMp5aj9O4jS35uwRfwZZOSV86TWO2WnlKe8mnZ1KuOps5a/bqOlmhMU2QZJZ9M+Kh1lFzJbG5W0Dl2rydsEF6Ys6smO9rS+cfGMNKxRYU/muvxcPe1UmQ5kBrQ/E0vUuGgtZ6cM4ZCk467Isr2asNKowx+7aM7O886Tc/liA7oayEFGVSp+tcqpp47TXfXEWLYzmijm63yMtFeTo7Qv9ZKySWXgUKm9tvWsRscv/0tv/rQ50pAIO8EJJUIZUGQsqsoisaVHVUqdODJvo9busO4L6cmz/8iV5KovoThG8PG1bslBy+nyc5lYLWWdpJ3CGFGph/lqg572Vu39emwJdB1bOJQ9evSgCRMm0IABAwRNEBxMZH337duX2rZtS/369Qs0Th7Ht2pRqVer0I+9++gptaqYnbrV9hy4rgZ2683KVjd8rYasesqjzXD1CJjFFaQuLYZFzGm7ajnFFTfK4s/Ki/igcl86HExXBM6ezFVpPWRdxIaVzu6dFKWWM1T26e4aWKuprLVXZjVr/47Npf3vjkxTo9KZKrXR+s+riKtzUMgYUdNwxY8pbdQjn6Y6yhmTxyfYIQsSHkDBIot6sqONeXWlhe3pmcvPVZk/VdlHJgX5Mw5QVXDSEkTLcBV/OmWSsFw6PvLFRo9KlHR4QfFSIH2SF+IJga+Zsol6n6feeurvn/Y32FVCViRarT41RCKf5MKVY4JqDM90xnZHIo2UplRpdPTw/Up2DikDmy1VQjp17a7I1h73bnHBXWz3UuWb1SKkC0Wr4qWe9uL3O0/awIXGWbX32/15wT5bOJRp0qShiRMnUp06dSJhNmvWLOFQQpbR7sWqRaXyIUr947aVwwgqFZ6KKhMnM3E9tZEnEKg3tEFBaqLQ4Khv3HbOxoTtapYzqFY+rhRGpYesENNH1iRidiQvpZ7sdy1uWr1wfO6J3iMq7FWtW7We1mmSn8nkKXfxUWq8l9of6vebd0D8yZNyktZembAhE0KkA9Opei76VJGbi2qeX8zaQ79vdWS0aosebXF57Y62WmdbXbv4XD3ZUb8L+MyXTVZ1TtXNTZIs+/PaVpXZW96xAoWlTuyEUWbQ96iTlz6qkN3TV92rz1U1HsiZyhcbPQ66ej2Ok7SPp0bOeIZBvogDeDUhA41Uaht5CiybS0UdNURl47Gr1HS8I04UnIoIQdKSuWO+4H3EqS8KQgHwkqIqy7jiA9WaXaDPEkIiTd50SYSUIfTCEQe78+xNAu1Z4nj2DydTVbC0J8AgescLI+Q58d1FuEugilV7f6DmZ2RcWziUoAratGkT5c0b+ZTtwIED4sTyzp07RuYUkLpWLSqVj0xOVM9pAOqC6Ddf7yWimR5KFtRTg59HvFmY3igSIWGmxvV0rpGL2laJ0KwNyEOIYlBVd7xlhezUonw2KjnI4VD+1basIEBuPskRIH90UO0XlBs8zUeNW5J13XEveuoLn2tJ6GUbd8kjUgnIHfelPE3Ujo2kh+9WHhN/Nvqm//uWM0JvV1LKhA9ZIbgOjWgn9/1rv0j6cFX0aIurXKhaQnr1VBr9qyc72hcAZLtnj4LkP6pnpn6vVAJ36dD585RNJbFGtmv8ODFFxjHiSKX0pPZmQc/601tHUudIx0m+2OgZ85OpO2jB3ouEhJaqeVNT22kvhnjo6UevrWbXU298VC5EjCPXtUrjpepXI34dGvK1RkaWm4QjjhNKmZkNac3DA2oLKqFmPztCU/T8Pr3SfyndvPdYSELilE9PgpvZ+PjanyQpRz92Ts6yau/3FU8r2tvCoWzfvj1dvnyZfvnlF4oTJ4K244MPPqBYsWLRTz/9ZAUWPo1h1aLS0inAaL1XWuommilFfFrXNeJ60N3kpbwePh/zdlGqU9Aha4eiKhb4ogrjE/A6Gw9fdoRwPSp/nN4rk5WKDVwu/g06GahfgIIGWdW7+9TQ2Wvkarl6LBLJLbIs+awC5U4bcWJkpNNVhy+/kAGK9oh9xYaiJZiWWuVap1+OCXJjvBxoC/gDp2x2xGsh8zRLyoS6zZRJCfK0W16zSaoTPR19ufiQIBl3VbS8p67qqFyo2mQgeUok2+3vV5MSxo1QepFJWMD04IBahl8iVHtczV1KMvrTKVLl/KQ9UmJSOvhG6aD0PDdZR+s4yRcb7W2Gqz7VJK56r6R3hl6odUH0/U5pe9LGqfQ82psB+buJk1epkiRjLnGaBofSFXctfssRL7l4/z9OGEBrhtPQD39xsBKcHFLHY15B8YHL6Oq/j4ROODg/tVnSRp5xoOqqMdB2Pqm2au8P1HMwMm7AHMoqVSKcmSdPntD69espXbp0VKJECYoZM6agDzp79ixB53vmzJlG5hSQulYtKvU0RE7UyAlI9m4LhMKAXt5ISV6NsSa8X5yq5ImIvVHJiY1Szlj9kCTFDcbFj/Y7pbJQ4f6OJBoEdSNGCRnL2VMlFMTl3pT8vReLmFZZfDn10qoaqfa4ks7L2WMhPX76n1u1FzUjVe3r1ULpnHGGrjTAo8JB8pBCkg3Xcnl6OaThVPlDTzh+u/wojVh+xGU1PYoeKp8knJJvm0QoRmlPP6G0AVlMWWTQvyvtdk92az+XlFPqi1WJQcuFBrsvikl67NC+yOAEvnudvCRFCaAK1bh4Jj1dGa6j1XOXLzYg6K5fJEOU/Ukqq+JZkhPEBDpO3/1CfT2OqWGjTWyQu+cice0KoQTMQRb5oqSyB0h5VDA3HBpQOxK3pGyH2yacUEqlHPz9j5al6ca9R9R6yg7SI0eKNqUHr6B/bj8QtwdgoJCxmCZO3e9dycQmDGTnzHSr9n6/A27CAAFzKJs3b67bfMRX2r1YtahcnUgYUTiRJymSjNoTrmqMmjb7WaXK8SUGzZMNZnw+ccNJZ6wg3nbfLJHJqauLH+yVhy/Tj2tOEDa3mR9HkFMbGVteM8k2eqQt3fWvxltp6yBDFBuFLCpnm7tMbVdE3mgv5Tvx3ziZxQmt3iIz0ZHgtbJTpUgxqQUyvCiz6KpfdX1pP9dzzbV430Wx0aJo+VjV03V8rj3ZkUH/2gxSvfNX68lkFFVDuGDfJUIBxt2psTfjuGpT5Dn9jvxMUnjJ8fU4d97aojqF+N7IFxs93IAyUSxn6kQixrOrRtcaNnmrr+7tfIy2A/UUknNU0nz0MXzpYRFKojpyc3aeow5/7HZKirrirkU8/N7zt4T0qCxInMONBJgqkIi5X4fErXzBwcse1iBU1Wa3KWt0egGtL2U/YYT2diyghmkGt2rvt9Oc3dkSMIcyGMAxYqNVi0pL+AobjUhPgdoGFDeutH9dzRdyfoiTQpnZOpyKZ03hrIbM5gJ9l4iEFu0PqhHsrKgrpQwxFq7RQDgs40mnfVSK5uw4LzIrfXEu5ImUnI83yT2yrVYmU8VIS2wM8un8fRyxsZhLmRwOvWq1yNMR7d8RWwUFFRTtCZ6n5yIz0XHqN/eTsk6OvfWfVxY0THrK5E2nqNfc/aIquBKhdyyLHkUPNTRAm2kridbRH7SAjw6KnPQn1ZP0JrVFNR9QL8EJUK/pc/VcJJSC/L0Zyqt1aZ+Mj5Yvj6ObFiWcRPujSH5T/J4gCS1Hdwf/4cTmJahy7qgpqKb+fZp6zNlHkBcFxy3icbXF3864r5jg2hun0JlSRF7vkn1AfXGfvvUsdZ21h1IlikPbelYXJ4cIEVAL1vD+87cJ2uuyID4TGEHoQG8yZaWvVtGpa/fEiTzWoDa+2Nd5W9FepQSb/GFJKp8zcFyTUc3Xqr3fCsx9HcM2DiUeyujRo4WONwjN8+fPT61ataLMmSOuETxNduDAgbR8+XJavdpBWI2ya9cuQozmtm3bKGXKlNSuXTvBeSkLVHlASzR+/Hi6ceMGlStXjsaMGUNhYWGehov0uVWLylXyh/a6JSrDpdOjxvZEVX/2jnPOqyhXGrJw1M7duEeda+T2GNdjCFCTK8vTAXSLUw/ImalXtFCmAAm2O2lDPeaUGbKCLtx64Ky6s1d1Su6FlB86UClGtGNrn4OamOHOsVc589T+4HxAdhFJkojr0sZmRjVvNSYMP/hSOk8bqxhVH+pVvOrcoo0eRY+Nx686x9XGEqvhGq5Odraeuk6zd5ynDtVyUuokUfP6eXr+Hf/YRbN3nifJO6h+T30huPc0Lj7XXnnjb1h7xQctFy97MqZST19G64xcfoRGLj8qYvT+aBnufLHRQ0E1f88FkYiDK2AoV0FCUlu8ERkwOgd/1JchNsiuXtrBobEuWRHSJ41HG7tVFS/2Wu5acJbiyluVDH03PIvI1kbMKRzLLT2qeTRZJoTJiqWzp6DfW4Z7bGenCirVmVEGCivnYdXeb+WcvB3LFg7lvn37qEKFCpQgQQIqWbKk4KGEA3j37l3asGGDcC49lZEjR1LHjh1FP9KhhDY4dMHr168vPtu8eTO1adNGOIzyyh3OJP6Na/UMGTJQ165d6cSJE7R///5ICUKexrdyUclrJWmTkRgpeRVSPmcqmvxhKU/TIpVAVlUa8djQZhXUq1EkjSATNqzHImElyL2RtAMapE8q56AuNaMmZHY3NUnULD834lhp+9TqrqufazdrNZbVHYemNskHiSggPkZMFhKJ9F6lqXaohN44hWsz1RHjdXhgLd2OqXQq0C8yhVcfjrjq0xM/t+PMDSeXnzYxbNTKo/T1Ukd8pi/SinqWsrxeL5ktBU1vFU5qwoY3RPl6xpR1sn6x4IXqyDaXFDV6TguNjKfWHb3qGH215LDIIsY81UQ3qZvurm81ThjURoMWHnyhKpR9aiuJgN7aaXU7V7r2E9afpP7zDzh1qV1x14LSbN+F2yKmWxYIMRTNkpx6z90vkmzAp+qp1Byxlg5fimBHUbPNPbW1y+fq7ZhUM7OLbaodVu79dpy/apMtHMratWtTwoQJaerUqUJ2EQUyjNDyxv/Pnz/fLY7nz5+nFi1aCCL0TJkyETgtpUM5ZMgQcep56tQpkS2O0r17d5o9ezYdOnRIqPKkSpWKhg0bRq1btxaf37x5k9KnTy+I1ps0aaL7+Vm5qORVljTOSEJMla9XCw3ZannTEHjjPBX1R39tl8qUOaW+q0xP/Vr9uepQ4cQGxL5SngzJRn3+2k9nr9+n3q/low/KZfPKPGiDg1tOFqNXyOqge87dpLqjNri0Y/y7xamaQkysZouu6lyJQGKsLRuOXaW3n3Pg4TMZrC/rSb47IxPHFa9U2RlQL7+4ujbaj0qPBPk6lZNSj6KHqnij/R6o8mxG7TKCA+rK0xSZ7Hbr3mNn0pc2VMRo357qS83u/OmTCGURXMEiUW/IokOiqT8d2rFrjhPiKDFvvJhJsQA94gAqITxe5EavejHb35+nq55w9eVzGbOtsmnI9SjXiCvuWiT27D9/i3afu+UcHi+/SO4ZuOCg+G7jO+6pSA5UWc/u0riu5qPeKulRB/KEib8+t3Lv99cczOrXFg5lokSJxOlhgQIFIs0L1984cYST567MmzePfvvtN8J1d//+/YXzKB1KEKWD43LatGnO5rgSr169Ol26dEnUBc/l4cOHKVeuXM46uPYuVKiQOLnUW6xcVJK0Vtpm5FpIkk/rIY1G/6pCgy8xgXpx9Fe9Tcev0VvjHDJnUhIxW7cFTuWIz37fKTK09Tgx7myETi9IhGXRQ+/hrq/9F27Rq9+tj/SxjIfSJlmodTd1q0LpksZ/oVttTCY2NdX59SYZSXVSQez87YqjgkBZXvHpeZZq8hESWiS1E9r+2KwY1cyfNspuVGda63zIEyF0oPdkR4/NrurI+DjEge7oVZ0u335AJQc7eE594SPVYw/IzUGEDXodkLmvO3pVxExKlSCcmOLk1B9FEubD0cELatVv1ohh3L3YqDaoaidQV0LYibZomSX8MQd/9Kkl/ccY3684St8sO0IyrtJV+BJUvJCUc0D5HYFiFGLXkTmeO01iWtKhgkeTVVJwVNZSannswAYVlh24RB/96qBKcsVsYQMThQlW7v12mbM7O2zhUIIuCKeQxYoVi2Tn1q1bhfMXlUOpNnj//fcjOZRwCnH6+eWXXzqrgSwdV+jo+8yZM9SwYUO6d++e0A6XpXHjxuJvUZ2M3r59m/A/WaDmg+t6UB1lzBhB/u2PBSATa2TfRq6F5A8NyIRHvPmKR/NwdQeJtJQJ44qEDyMxdh47t7CCehqCrO5S2VOS1Lsd+04xgg416JR8yVaXUnKYll56D3cQqKTxso6Md9QSh6tOv7u4TfVqGP3BgdymXKthI/uyUSFDT0RV72heNitN3HBKd7KXHEi1C3yJSNKQRY8Wunrdr01IUq/MpN64oQkaqLz8wCVq8es2AhH1sUF16PyN+5G04nOm8Y6P1IAJourA+Qdo/PqT4lr19LV74m/+jD+btOEk9Z13gKDghBcA+RLk7sVGnQ9eaHCqjwLpVqlfrdbBqWfFXPZMxojq2Uh+VJmAg7rOzO9MyUQSG4o2/hXJgoihPHIp4qYD19V4IUBYjl4+yYY/bIx0bY4bGfy2BVOBmtXro9ZTwQxJaczbkX0DO82DHcqIp2ELhxJX2xcuXBB8kylSON6kr169SnDscMKIK2o9RetQIrGmadOm4uRSFsRH5siRQ1yR44SyWbNmImYzRowIfrp3331X2IPTTHcFOuOuNMatcCglv5y0zYgT9L+xG2nrqRuE68WhDfU5EHiTRglWZxK2qw4afswLZ0pGMgt3VNMiTpUOX+LN1B9xb2ISI2+2d6ja8LXOPyFpBkou2IS1vKPqKd/B/rWEWoq2qJrP+Kxa3tSRuO6618lDLSvk0PM1c9ZRE4fkdbXeZC/ZycVb96nM0JXCAQflkYz7w+czWodTCYVVwJVx4GUtOmCZ4AJESIaabauSnoOaZllHR3KEP4rqGG/vWU1kq1cf4Xh+VoaKzNx+ThD0q2VBu3KUP70+Giej2MiTuHRJ49GopkUJ3wEUPQlpqmQlXnCl3KBqg6o8ZNS2QNaftf2cyMpWhRKGLDooqMnUjGstd23dwumFQ4mwJCmti/pwKEetOkZFMiejOTrofySdkcRAaq0HEhNvxsbeY/d9hx3KiCdrC4cSD6RMmTJ0/fp1cfWMBYRr6OTJk9OaNWsoe3Z9OrRah7JgwYJCH9zVCeWOHTtE8k2jRo1cnlA+fPiQ5s6d6/Y7EMgTSknILI0z4gSp+rl963pOdvLmR8CObc5cu+c8MVraoQLlSpOY8vVeTPcePSXE3klSZV/oKWQsG+bvaxLIiSv/UpXn14foD9fdYS8nEldhWq1sNTnmxOA6FMOFru3hf+5QzZERDmqjYhkJzocs2rhMPc9QTRzCRvjX7gukN9lL7X/tkSvCCQYliupE670qhmML3eJKGpoaualjLL28q3rm7aqOur4QP4jsaqnTbJQw3lsb0E774oC/+TOZTpXf/K7JK84XAncvNurcVB5GODyqOoysJ28TfMEkEG0l76v6Ytl/3gFxrV82LCVNbVFamKXlrgVLAK68Ec8tOSTBZVk6e0r6ce0J4VgihMFTUWULURchEKCP4mI+AuxQ2syhxFsIMronT55MyPjGv+EM4nQxaVL9b9ZahxLOJKiC0K8srmIojx07Jk4tZUEMZeHChUVCj95i5aLS0tMYUSZZuPcifb/yGOF6sWjm5HqnF/T1sKba/raTHj5+JmhU4HRJVQ8omYCSA0UP3Yk7MKSzjs/10nu460t1UFAHG1NYmsQiE/3TKmHUqUZuZ1M8U5FhHSuG4JJ0VbQyb1ChGbcuImZNT8ybtl9IxOFKCkWeeOL/x79Xwqv1ol5fo4PlHStQWGrvr4rh4IIMGgUn0vKa0SvjPDRCggVCURCHCxovxJI2GrtJtNrduwYlTaCfMN4X+yB8gBhrqCbJouUt9aV/bVt5IoqXgW8aF6YPJjli3ty92KjtcYqL02UUJI3gxUJboGJVLIt/4j/NxEHbl6vvZM8/9wqZ08q5X6aJzUuKJlruWlxN46Xg4q0HlDZJPKF2gwz68BwpRUiJ3mztdydsiYQnXvgQa8/FfASs3PvNt97cHm1xQlm8eHGRVY2YR1+K1qEcOnQojR07lo4fPy7kHFG6detGc+bMEVneOIVMnTo1DR8+nD788EPxeTBkeZcftlK8wcrye8vS4g2WizEEZCwqtJYRB4biS0buB5O2Ci5LFF+TQMDtWe7LVc4J4cQzV+rEtOXUdWpVITt1q5PX+ZnkCk0SLxbt6VvTJQjq9SIqgLMRdC8oIP3GiVKsmBFhH3qQVDOspeKO3mQvV/2rDgY+N0KQ7qo/lSqqRNbkNKO1dwpIerBAHUluDglISB3KrHqrEwqkao+0GzQzWI/+KPIkLnHcWCIGFy82cWOBOsr1i41qw50Hj51qVbjWxdrWFn/Gf/oDD9mn5GjFZcGJIa+KP38+cw/9se1sJPEE7eEAnE3QBiFTH5Kgx6/cFXKwZcJSCmdUb3LNh5O20ornv0UYu0GRDDRcR8y8PzGJrn2zQxnxZG3hUIK65++//450SujN4tM6lJcvXxY8lHXr1hX8klu2bKGPP/5YOJnvvfeeGKJHjx70448/Coc2a9as1KVLFxFbuXfvXtvyUErqH4lRsL7Fe/OMzWyD0xE4MSrFii+KP60mb6Ml+y8JE/XSe7ibD2ILw4esdH4Mmh+cVEBBAwkwfV6PCFeY9vcZ6j4natJjrcwb1IIkkbReXXetrUcu3aEaz+MEcQKI01O9yV6u5q1yN+JzxCKmVCQmjT57lZIIDu+0jxzXjP4qkswayVM4pfzwF/2ndWba1GPOXpr69xlnl/6kXJE8ovFjxxS3HggdUeMGo5qXKiOLxAtc9WqLKzEFM7HyV1+rDl2m5pO2iu7laS0y8BEn+lqhdCLeFEXLXYuQEZxQ3rj3mAplTEp7zt0SLwM4mYQzWjN/Gvqxmefkmpa/bqOlBxy/RSj/K5aRvvpfYX9NN6T7ZYcy4vHbwqH86quvaMGCBcKZQyKNmnENU/Wq5WgdSrRFNjfUcXbu3EnIJu/UqRO1bdvWiQAScsBNCWLz+/fvC5oiXHXDuTRSrFxU1YevoaMK32GwvsUbwdcfdeV1E2TfkEGJ4ssG1nbaDpq/56LoRy+9h7t5Xb7zgEoOctDOoCDpAWoZOAHValZLepzMKRLQ2q6VXXapqumggpqI5K3cpHqNjiteZKYaSfbSGoqwhOzdFwoqJ5QD/WtSgjgO/lhvisqTCdL0Sc+vGb3pS08bNTEHspHDFh+OMgxBT5/e1JEURrLtrt7VKVmCON505bGNFD7AKXe/ugXEi42UfvTU+Nkzx/NG0dJYybaLPytPedIm8dSV7T5X+XtB9B83Vkz6ZNoOQeWknhZquWvDs6cUDuWdh08I/73pxDXBGQtHU+uMRjXpT6buoAV7Hb9FKL58L20Hrs0MsnLvt9nUXzDHFg6lmmGtZnTJDC84fXYvVi4qySUpMfHFCbI7rv60r/TgFSJGSSVV9mUDk/J7sFkvvYe7+WkdQBAkI7EEJ6Da0wZXMm/afrUyb0g+avbzFlGtRbls1PO1fIahBv9hxa8cMqegjTl34z5BJq5/vch8skY6lolSaKMnDi+qvlVZRitoUx4/j6NEohey3SGfh8SKvW7CEIzgYqSuyu+Idr4oNnkaV6VL6vVqvkhKMJ7a4nOp+oVTOMTQaouvcbR6bPBHHZX3FrRu0NXefvq6YFZQnbtPf9tJ83ZfcDrUCM3ASe2Dx8+E+MTyg5fEGkKMqdYZjcpuxA4jhlgW7UuoP+Ycqn1auffbHWNbOJTI5I6qVKzoP7oPsx6QlYvqte/X0b7zERyYSz6rQLnTep+8YBYGwdaPlKGEQwXuPpTlHSuKH3dvioyRQlu99B7uxlGVVlAHcVT5MyQVmw9i9L5tEhFgj9NVEILjiuyvtuVcdqnKvCHGDeopMmmk12v5CBrCRosa5wlCb4QPeOucyrGLD1xGV/99pDsOLyqbVVojX2I7jeCiJmahHU6XtvX0rL1sZAxPdbUKLL4oNnkaa/Xhy/T+RMfVbucauYTUpZHTefkCkTJhHLp299ELw3mTLObJZis+h/PY8AdHUpYs4CjF6Xuz0lkIXLIoeAm5cPO+YFxAsuQrmZKJE0rIouJ7PnfXBfFdABcnrrD1njR2nL5L6NTL4uuLnhWYBesYVu79dsfIFg6lChKog5BAYyS72w4gW7mo6o3eIOLVZLGzzqkdno07G2RyE37gQYKN4ktGrBq7ZpSPUWujmrCAz7BJF8iQlGbtOEegFlGJfocsPOigFMmagqa3dk0pojoZSN754Z1izqQRELvXKhC1Io0rDNW4zHixY4hTFV+00DFGuS9XipNOX2mX0Be+I/iuoEAPeaTihPtrXUpta9k/Tm7Xf+5Ze9lse1R9b18UmzzZtf7oVXrn579FtVYVswueRdDczHXzYqPtT9LmJIgTU1B4acu6rpG5RT3ZY5fP1bWntQkvb3iJU8u3y4/SiOUO4nJ5WKCSvUude9UZjWquXWfupunbImjBtHHXdsEpOthh5d5vd7xs41AijvLbb78lKM6gZMuWjT7//HP66KOP7I6hsM/KRaVVQbCSODkoHoZOIyt/vZpOXr1LjYtndP74+pJZ3Pev/TRp4ykxujd8jKrZ6oki/g6d5kIZkwk1ES01T5+5++iXTafFtRjUZVwVNT4RutZYM1W+WU2xYr5EyzpUpHixXyRD9wQjMlERh6qWDtVyUftqOT01dft5jRFrRCwmYkY3davqdT9oqGahW5WUAAJ9SHDKOFBk6q7o5Fl72aeJumgsT9/x0amhjixjfxRVLemd0plFJrJerkTYI+OY5emd1kY9ijv+mJevfbqSTpV9tq6Yg76onSfSEPJFRI0lVW9OZBY8JCp7v+45PAU0aKryEGjCerzquZ2v8w7FSpV45AAAIABJREFU9lbu/XbH1xYOJYjHoWaD5Jnw8HB69uwZbdiwQWhpjxgxglq2bGl3HC11KLUqCBu/qELp/UQLYnvgfTCw6jerBS2HvFpCV76QUA9eeJB+WntCWKSX3sOd+WoGLOogi/qVjEmF46h1Vl3RkbjqV8q8IR5zXdcqhKzqmDFeotgG6YJk3zfuPqIiz3kE5d8+r5WHPq5kTHFHtVWevuOKf2Vn3xyxo5fuONVqmpbKLDKvrSiSbxBj+aqY5K29P645TkMWHRLN/elQqmEFOAX+c9eFKF9stPNRHV9Xc/Vnhrq32OpppzIgaOtDsx6JgGqRz0uNJW1fNSd9u+KoqIaEPMTG4hS4W+0IyjB3tvSeu49+3eS4dUHR207P3LhOZATYoYzAwxYOZaZMmWjw4MFCBlEtoPIZMmQIHT3q+FLZuVi5qJqO20wbj19zwhGsP7qBfp7yNExV6fCFquarJYdo9KrjYlq+Sp1BaSXH8wxY9AftbcRlgoxce53e/vedItbKE3mxlHnzliZI+7xuP3hMhfoujfTn3q/low+8iMeUnUi1ITOUbU5dvUuVvnYkDb1fJitZpQylJkCBTufggFqWL3Wsn4kbTgrJRZBi+6vsOnuT6j8PK5BJJEZYA+QtgTv7dvSqTojPDbaiVbpS7UesadsqkU/xf15/kgbMP0BqLCnkUAcvdLwU4CUQ3MNaUQN3uPSbt18QocviayhKsOFvpb1W7v1WzsubsWzhUCZKlIgghQjZRbXAkYRizb1797yZm6VtrFxU2sB/Pbq5loIRJIPJbHmQCa867FDp8EXVZMSyI84ThdcLp6fvfVCmwBV1tm4OShWU0tlTULEsyYXDWjRzMpqt6PlK/ktc3Q9r5J5rTsar4fp8QbvyPj8l6Gjn670kUj/gt3yndBav+5bk8JjrrI99IyJXlXesvvJbdfgydZ25h9pWDqP3yhijIPMavAA0VKUesUY3n7ju8cVGNVO+1LkzfXefGoLXMtiKVkhAtR+OYssKkU/xf910inrP3U9qLCl4PXvM2SeayqQ3reyqO1zU2xLUaVcljDoq6lrBhqed7bVy77czDrDNFg4lSMYTJ05Mo0aNioRXv379hBTjjBkz7I6jpVfeqiILgNnTtwYliRd8P7qBfqgyW16qvMAeXyhW1ISMBkUz0PDGr/g0RZxQ4qQJBdfcxbOkcAbuz/80wiGUCi2eMjllvJrWIfXWSO21PPoZ1qiQUInxtkj+PFXv2Nu+VC7PNpVyUNdakePWvO1XbztJe6a3fjDWQ8xorZHrhOkyocTTi406Ty1jhRYDX7lIA4WpVphAtaPP6/moednIrApSnADKOs+/8jTyzVfosz92iaZxYsagR0+fidhLxGB6Kl8uPkQ/rHbclqD4GtvsabxQ/pwdyoinbwuHsmPHjiJeMm/evASKoNixYwtC8nXr1lG9evUoWbJkTotxDW7HYuWi+ujXbbRMUUGAbF78OMaTKuyIo5U21Ru1nnafuyVO/rafviGGliTE3tjx09rjziuqt0pmoiENfJMSlTGPsAUxmcWzJhdk2SARX9ohgkrrzR83Cc7DlhWyU3dFklE7BynzBsLk31r6rhqjvZbHeN82eYXqvZLBG/hEG5mdiuvT8e95VgSJaiA1xhPxaB00cWteG8kNnQgcu3yHqg1fK/4NdSgkuXl6sVHhe2PMBtp5JoKxQgutL9/HQD4mVwlr0h6cPL5dKvIp/vRtZ8WJtlrGvVuc8FuvFr0UX8OXHqbvVh5zNoXU6ieVwwIJSbQd28q93+4g2sKhrFzZtbqHK/BWrYrQN7YTuFYuqo+nbKdF+/5xTv/ooNpeJ1bYCUOrbZGbmUrVcXxwHZGo4k1BzFq/55rgRjZVd2Pl7bWY7j92UKkgLg3ZswMXHKSsKRPQ6i4R3xmZyOLpWkvKvJmlGqO9loedY98pSrUKpPMGPtHm7xPXBIadauSiqnnTeN0PGv778AkV6OO4kucN1Sco3TZW41TltayrLGZ3HciXIXef+/J99M+M9fV6894jeqX/MpeVXZ3iz9l5jjr8sTtS/WkflaKm4xyUTLIMqJefmoV7DqGQNESyna/JcvpmHZq1rNz77Y6wLRxKu4Okxz4rF5Uq8Qfb/Mkzp2fuwVqn0Q8badvpG+LED1Q1oC45OcR7ipUpm0879bFdcc0ZxQnOEJwiFBBzI0at19z9Qtt3wxcR3IYyFtST0yRl3vTqAeuxN6z7QkHCLMvE90tQ5Typ9TT1ex31Sr5Hnbz0UYXsfh8z1AZQYwXxIoZTa1dJJ+5w0caDy6tdWT9Yf9vuPnxC+Z+/zGjn7uoUH4IFUM1Ry9xPyjp5VOXfv2xYkN4skdnjMtPyofL69wiZ1xWs3Pu9NtKihuxQmgS0lYvqs993CnoOlFgxXqJjg+uYNIvQ6kbSL0ED+8z1eyJO6cig2l6D8MfWM/T5rL2iPahzcCrgSyncbykhYxgFlCzI1kX/Un0FWsgxYrxElb5aJaTdPF2HgR8R1COessGN2Jy75yJ6+OSZswkUeMqGpTLShd/qqlrRfV/PR+9r4tb8NnAIdayS28tpG8n0/3DSVlpx6LITscRxYwkd62D/bXMVXywnOebtolSnYORT/EV7L9LHU3dEWjmQnZThBPKD4Y0LU4OiGT2uMJU2CpWNPBOPnXOFSAhYuffbHXp2KE16QlYuqk7TdwvFFBQolBwa4L0TZNL0g7IbSb+UJklcunT7ociwPNDfe4qX2TvOUcfpjmsrV1xzRkEqOmCZkDNEaVQsIyF5CP1D6WZt18pU59t1FJYmMR3+57aw31VsljqmjBk1kjThyWZJRSTrzWgdTiWypvDUzLLPZWKTJ2wsMyiaDXT134dUfGBkcvthDQtR4xL6ErNaT95Oi/dHhO+kShRHSG8G+2+b+jKjfeSIjYS2vFoQE6+Nl8QtBHg61QLmCDBIeCqShkjW61c3f7RmG/CEhz8/t3Lv9+c8zOibHUozULRYKUfVjMYb/d5+NU2aRWh1I6/bkieITTfuPabE8WLR3r7eY6leWxm59nOHuszKxudI8sHJX9tpO8VLBOQSpYayzAz95n+FqWEx96cXUmFJr3ybntVQqO8Suv3AcaKE8lfbskLRxy4lT69FQhLS1+xzu8zHbna4ihUc3bQovVpIXxxtu9920l+7HbctKOmTxqMLtx6I/04UNxbtC+LftuzdFjgzttXn9ssHJYU2t1pAM9X8uSa6/PvWHtWozNAV9PhpREiJXpnUXzaeoj5/7XcO4Sudl93WnZ3sYYcy4mkEzKH866+/qEaNGhQvXjw7rQ2vbbFyUXWfs5dAM4ECZ2hn7xpe2x3KDSXdjuR+Q1IBiJS9LYv3/UOtp2wXzV1xzRntt/TgFfTPbcfmCicQ1EEtJ28XSUPYtOVYsl9PG3nziVsE3ya4ETvXzG3UHJf11VNUVFj8WXnKkzaJKX2b0UnJQcvp8p2HPicLmWFLdOxDTXyS85vYvARVzq0vjla9bUF7mSmO/wb/JHgog7WoLA3qHJBsUyZH5LAQVRNd1t3VuzqV+3KVM44af5/wfnGqksdzstrUv087OSzRbkiDggRtcC7mI2Dl3m++9eb2GDCHEmTmhw4doowZM1L27NkFTVDKlP5TdDAXthd7s3JRSe1mWAFdZrzJcjGOgOTzlCd8qRPHpS0+YLni4CX68BcHzYcrrjmjFqqydM3LZhWSdvIUA6eRnWZEzgr9+b3iUWZG7zl3k2ZuPyfiO9MljW/UHJf11VNUVFjZqSJlfzmRKX2b0cnS/f/QlpPXhQPtjV65GTZE5z4g35mn1+JIUzQS9qDVnM6TNjEd+ueO6A+qMdt9eMELNO5Zv1jgNAGKSZKxYdbH4VQsS+SwEFUTXTbC6WyFYaucYS/4+5QPS1G5nJ5jlNV4brTjE3r/rQYr937/zcKcngPmUKZOnZpatWpF1apVI9AGzZkzh5InT+5yVhUqVDBntn7sxcpF1X/eAZqw4aSYTbqk8WhTt6p+nFn07VrL54nrto0+YLn2yBV6d8IWAZgZV0zYTJAshNKqQnZxTdZ0vINGBEkmfZ9TFMknFIiEmPAhK+ji8ytK2IG4L2ShcwkNBJ48fUZhPRZFmuzCduUpX3p9p9TqyzE6KZwxqeCGRUFs89/dg/dlWXUoISV64OJtMS9XYSHbT1+nhj9sioTjoQG1CNKU6vfrj5alqVR2zwcveHHsrLxw6k3mCY1Va+4srdz7zbXc/N4C5lB++eWX1L17d5JqEvh/V+Wll16ip08dXHx2LlYuKlVWCxqv67pGUMjYGSO72aZNCEC2N5JdvC2bjl+jt8ZtFs3NOBGo8vVqOnH1rugPWry4Rmw01rHpdKyei4YvOxLJVEgVgqTdylLuy5V07sZ955A4LcepOZfQQMAVF+maLpUoS8qEugAYtOCA0KeXpWTWFLTl1HXxTy09lq4ObVRJdSgRU7pgz0VhnauwkN1nb75AEQQOzqrfrBYMDrLMblOGimb2/B2fu+s8tf/dobKD4qvggI1gtZ0pVu79tpu8xqCAOZSw49atW3Tz5k3Kli0bbdmyhV5+OXKgsrQ1SxbvtYGtegBWLipVVit7qoS0snMlq6YZrcaRMn9yUtlfTkgrO3mPpXrKANm0+kW8V4yBTZI3Ev8NpZeqeVNT3VEbhLktymWj8esjNmL8bUG7cpQ/fVJLn5GkLJKDBqv2sqWgRbPBtFyk23pWE9RWesqwxYdojCIRWC4sFa0/dlU09fUFT8/4/qyjOpR4IRy9yiGFuKJTRcqhCQvZf+EWvfrdeqc5khNXq3U+/9NyVCCD5+/4/D0XRAKfLKOaFqHXCnnODvcnHtG1byv3frtjGFCHUoKzZs0aKlu2LMWKFcvueLm1z8pF9c3Sw/T9c1ktrQxf0AIYAMNBJIzMbFlyp0lMSzp4H16BGEXp8LnimjM6RUlYjnbIGodyDLgkUUD9M32bgzpKFlcbldExjdbHCcrxK45TVBRc03GsolEUg7u+lovUyBoYufwIjVx+1AlA1TypnbyUwf6yrDqU4IgdMP+AmCf4JcNSJ4700A//c4dqjnRIWKLEjvkSHR1Uh17/fj3tPe8IAUBZ2qEC5UoTua2r1aMmCOLzH94uSrU13JfBversY72Ve799Zu3aEls4lDBt/vz5NGjQINqzZ4/Q8s6XLx916dKF3njjDbtjKOyzclGpP8KIzVnYvnxQYGQ3Izv8sYvm7DzvNMtXLA9cuE11vnM4fK645ozOHzyTMu7qi9p5BHdd1W/WiG5q5U8bib8PfwtE/GLNEWvp8CVHEgXKicF1BNk6l9BBQFV0wqPHVS1ClfQUraJL7QJpnbKyOVMnomUdIzTr9fRnpzqqQwlHsMYIh8OI7O1kCeJEMvX4lX+d3218gCSegwNqkaT6kpVXda4kMuE9leUHLlELRQf8p2bFqEb+tJ6a8edeIGDl3u+FeZY2sYVDOXfuXGrQoIFwHpGAg5jJtWvX0rx582j27NlUt25dS0HxZjArF9WolUfp66WO+DkEsc9tW84bk0O+jZayxFcsj12+41S2mNS8BFXSSZ3i7kHUHbWe9jxPUMAJB/S8yw9zaNmHZ09Jm05ci9R0e89qlFLnVaNZD191en1VGjLLJu7HWgRURSejXK7j150Q+vSyQBFKqoDlTZeEFgXxy7LqUJ4a+iptPHaV4saO8UKGN+Z+5to9qvCV47uNInGU4gvy7+s/r0wZkyfw+IBXH77s5KlFZb10Qx475govIGDl3m93+G3hUBYtWlQ4k7169YqEV//+/cXJJeIr7V6sXFQ/rD5OiKNEQRIGkjG4GEeg68zdka6NfcXy9LW7VPGr1cIQV1xzRi2sP3oD7Tp7UzTrXy+/OJUsOXiF+Hf+9Elo/wVH1qgs+/vVpIRxrQ0bUZ3eYCeiNvp8uL4DgeIDlznVbYyyTmgJuN8snon+2HZW9FswQ1Ka92nwvixrHcqo1suFm/epjKKKIzlx35+4hVYfvuJsuqV7VUqdxDN3s5bX0owXXF7vrhGwcu+3+zOwhUMZP3582rt3L4WFhUXC6+jRo1S4cGG6dy8iy82ugFq5qNS3+lLZUtAfrcLtCout7eo2ew/9tsWxeaH4iqW6KcxsHU7FfZQgbPTDRtp2+oawbfAbBQnXgUUGLBP/Rnb/2esR2dX4G64aQXpuZXljzAbaecbh9PpKDG+l3TyWeQiUGrxcSH+i5Hg5Ia0wkNj225YzBC5KWd4Nz0K/bjot/vlKpmT05ydlzTPU4p6MOJSX7zygkoMcL4sokhO31eRttGT/Jeffd/aqTskTRr4udzUtLa+lXv5KiyGKFsNZuffbHTBbOJQ5cuSg77//nurUqRMJrwULFtBHH31EFy5EJE7YFVArF9XEDSep33MOwrJhKWlqi9J2hcXWdvX8cy9N2exQHELxFcsbdx85HT4zMq4b/7hJkHKjgIbo1YLpKH+fJeLfyRLEppv3Hjttl0H8VgP+v7Ebaesph9Nr9HTKalt5PP8goBLwGw0b0fIlflQ+m5NGqHiW5DQziG9fpEOJmEfEPkZV1N8O1JOUSVppSr23ENtOXXdSjKE/M25M/LN6gr9XK/d+u6NlC4cSV92TJ0+mH374QWR7o6xfv57atGlD9erVo2+//dZrHFevXi2I010V0BWdOHGCmjdvTpMmTYpUJUOGDCLRRm+xclFN3nSKes116LSC7BrasFyMI6AlVTYDS3BDXvv3oSA215uY4M5yNX4KNETgssv5nERaqvvItoHSdG/y0ybafMLh9GZJmYDWdPGex9P4E+QWdkCg4ler6PRzrsQyOVLStI/0v+BCxxtOkywqvU7p7Cno95bBe/uy5sgVmrL5NPWok5eyekikufPgMRXsu9SJg/wugZwcTrcsRwbWpjixYnh87AiVQciMLNNbhVPJbJHVeTx2whV0IWDl3q/LoABWsoVD+eDBA2rSpAlB31tuwiDMxYnl9OnTKUECz0HI7jB89OgRXb/u2PBkwfU6+oYD26JFCypWrBjVqlWLPv30U2edmDFjuuXFdDWWlYsKOt7Q80apljc1jX+vRACXUPAO3W/efpq44ZRzAnbDstnPf9O6ow5OPvDI4YQye/eF5EoDALx/4P+zurwz/m8nb2CwZ+VajV10GU+ljgITARgO9JbF+y5S6yk7nNVVwn5wUk5pUUpvV0FdTythKUMHeszZS1P/jrhFOTlEXwb9vvO36LXvI3gtAyF6ENQPxIDxVu79BswKSFVbOJRy5tD2hrMHZ7JgwYKUN29e00F5/PgxFSlShAoUKEC///67yCiHrvhvv/1G9evX93o8KxfV9K1nqeusPcJWJGqMbVbMa7tDueHA+QcikYPXzJ+GfmymfzP0N3ZqQP7Yd4pRrQJpKVfPRfToybMXhs6YPD6t/9x6xaT3JmwhnMSgFMiQhOZ/yhRW/l4XdutfpY5ClvbIJkV0m7ji4CX68Jdtzvqgxxq6yJFwaMaNgW5DAlxRK2EJTfPFn1Ug9aXXSFjLwYu3nZy1mBpiURGTysV8BKzc+8233twebeVQmjs1172NHDmS+vTpQ3Be06VLJ/4fjuvhw4cpV65cXptg5aKatf0cdXqu0/paoXQ0qmlRr+0O5YZDFh6kH9eecEKAK+XRNsLyw0lbnSTPP79XXBCbF+yzhO48fOK0GUk4T5/9R2GpE9HyAHD2qTYWzZyMZrcJ3iSKUP4u+DL3V79b52QceLtUZhr0RkHd3a07eoWa/RzB4tHn9XzO+HCQnP/8fmjcvmglLMHisKBdeeFcj13jUNhJGCcm7e9fSxe2KoUZGuhV2NHVOVeKhICVe7/doQ8phxJX65kyZRKxmf369RPPZsaMGfTmm29S69atadGiRRQjRgxxHT5w4EBKmtS9xNXt27cJ/5Pl4sWLVLJkSTp79ixlzJjRr89d1Wl9o0gGGvHmK34dL7p2rpV9M3q64m9cWv66jZYecGR4Ik4WJzbFBiyja3cfOYcGN+Wqw5fp7VJZqG/d/P426YX+VRuDPebNcvCiyYD1Rm8gaFGjtKqQnbrV0X+zpM1GBpuBDOex242Bvx+XKmFZOFMymvtJWUJM9ncrHEpCSMTb1buGLjNOXr1Llb92UJihLGxXnvKlT6KrLVcyhgA7lBF4hZRDOW7cOOrQoQOdPn2aUqZMKVDAaSUUegYPHiwcSVAVde7cWTieK1euFA6mq9K3b1+nU6p+boVDqeq0/q9YRvrqf4WNfQO4tkBAlbDEvxsVy0hf2wjLj6dsd6qGyCzN8CEr6OKtB84nOKRBQar/SgaKHydmQJ5qm6nbaeHef8TYoXRFGRCwbTqoqubSoVoual8tp25Ld5y5QQ3GbHTW/+Z/hZ23L4gZHv126Ny+5Om1iB48doSzyAx3VUno5cRxaWsPfXHSZ6/fc4ogoD+9ko26HxxXdCLADmWIOpTly5en7Nmz0y+//OJEAFcNt27domTJIuJLNm/eTOHh4YT/L1XKdVB4IE8o1UD2t0pmJjgVXIwjMGLZEfr2+ds/WjcpkYmGNixkvCM/tWg7bQfN33NR9D6jdTiVyJqCKgxbRWeuR/CyjnizML1RxL8n4lFNT9VDN5qQ4SfYuFuLEXjzx03093N6q56v5qUW5bPrtkCbPILks7bTHFnfdQunp+/e0h+PqXtQm1ZUw1nkab/KOSyphPSYf/HWfQofstJZdUWnipTj5UR6mnIdgwiwQxmCDuWVK1cobdq0Qnmndu3aUS6Zu3fvikSdmTNnUsOGDXUtLysXlarTCiLg/vUK6LKRK0VGAFdJuFKS5Z3SmWlgffs45+1/30lzdzk4WOe0KUNFMienasPX0LHL/zpt/uHtolS7YLqAPVpVD53jeQP2GAI68NvjN9OGYw4Z0KENClKTkpl123Pk0h2nxjUaQXO65eTton2DIhloeAiF8xTpv5RuPOeWLZ8zFU3+sBSpFHFZUyag1Tppua7ceUglBi13PofVnSt5pC7S/dC4YiQErNz77Q69La68T548SZ9//jnt27ePEOeoLeCK9LVALxzOIU4WVRqipk2b0rVr12jJEgdhNIrkrjxw4IDuTHMrFxVi5ppP3CpsbV42K/V53frYOV+fhx3aq9dJsOf9MlkDEofoDouO03fR7B3nxccyqF7VzsbfJzYvQZV91Az35VmoPHkNimag4Y05ntcXPIOxrZrpjxNFnCzqLdpYP0gEvv/8ty3UwnngAMIRRKmc+2Wa2LwkqYweudIkoqUdKuqCVkuUvq5rZcqUwnv6PV2DhmglK/d+u0NsC4eyXLlyhKSWxo0bU7x4L+qUIs7R1wJd8ClTptCRIxEnUmKjnj+f6tatSwMGDBBcmPj8k08+EVfeU6dO1T2slYtKzYxsWSE7dTcQBK97QiFQEdmTkqIE04VKR49X89lm5l1m7KYZz0mNF39WnvKkTUJqAgQM/e2j0hSewxEPHIjyxaw99PtWh3zlWyUz0ZAG9gkZCAQeoTimmuk/4f3iVCVPGt0wnL95n6C0I8vUFqUI/KvP/sN6Cq1wnjJDVtCF5/HRMnzkz53n6bM/dgl4ZOa3HnC1ROkbv6hC6ZPF19OU6xhEwMq936Bplle3hUOZMGFCWrduHRUt6r8AbGR279y5kzZt2vQCyLNmzRKJOaAQQiwlTi2R5e3KuXX3hKxcVBuPX6Wm4/4WprSplIO61spj+cKJDgOOW3uCBi086JxK64o5CDx4dimqsyZjoBqP3URbTkUQ9cur8EDZrBIv2+2EN1CYhNq4qt70Hy1LU6ns+l9wtBrWv7csTe/+vIUePX1GzUpnoQH1QyecR42PrlMwLY15uxgt2nuRPp7qIH43om1+/9FTytt7sXMpbulelVInefGwJtTWqj/ma+Xe7w/7zezTFg4lHEnwQ1aoUMHMuVnal5WLCvrO0HlGaVc1J0FdgotxBH5ef5IGzD/gbPhplTDqVCO38Y781EJ11tZ2qUyZUyYgVZkGwy5qX57ypgscHYgqX8mn5X5aCDbv9pOpO2jBXkfymFG+w1v3HlPh/hGSgzNbh9O7E7bQvUdPbReC4u/HUOXr1XTi6l0xzOuF09P3bxUhlfi9ZNYUNL21PinKx0+fOWVa0d/2ntUoZaK4/p5CSPZv5d5vd4Bt4VAim/rjjz+mdu3aEfS1tVQ9weBoWrmotp++QaDqQOlUPRd9WlU/TYfdF6SV9v2y8RT1+cuhiY5ilPLE37b2nruPft10Wgwjr6w+mLSVVh667Bw60MH2/ecdoAkbTgp72lYOo8417eOQ+/v5cP8OBNTksVWdK1E2D7rVKm73Hj2hfL0j4tdntylDiMm88+AJtSiXjXq+Zp8QFH8/7xoj1tCRS46EO5mQtP7oVXrnZ8dtlBEpSi1R+q7e1SlZgjj+nkJI9m/l3m93gG3hUP7888/CoXzyJEIBRAIHbW/II9q9WLmoQCKMWDqUz2vloY8r5bA7PLa0b/Lm09Trz31O27rUzE2fVA6zja2q7NqWHlUpdeJ4pF4vwtDN3apS2qSBu8oavPAg/fRcbYhfbmyzdCw1RE0ek+tUrwFaycG/2pYVDiWyne0WgqJ3Tt7WUxPuGhfPSMMaFSb1Nkom6ujtP0f3hUJFC2Vv3xqUOF5svU25ngEErNz7DZgVkKq2cChBIv7aa6+JE0o1A1sikiVLloCAY2RQKxeVyt1mlPfNyJyie91pf59xqnJgrt1q56FWFe3jnKta4zt7VafkCeOQyvsImwN98vDl4kP0w2qHNJzd8Ivu69cu8/t85h76Y5sjMWt/v5qUMG4s3abhJC1794X0n8PvEVfmyPK++u/DkDvxrjdqPe0+d0vg0LRUZoJqkHp4AFWsn94trhvbXD0X0aMnDqL0A/1rUoI4+p+L7kG4Ilm599sdbls4lEjK2b9/P2XNmtXueLm1z8pPmXuBAAAgAElEQVRFdfifO1Rz5FphS9/X89H7ZbMFLW6BNPyPrWfo81l7nSb0ei0ffVjOPlgOWXSQflzjoMza168mJYobizpN302zdpxz2nxoQC2KFzswKjkwQlUbgg5zc16LgVzSARkbUol4OXvpJaITg+sQbpWMlNw9F9HD544P2Azen7CV/rn9gNpXzUkdQig+XFUcei88C/WrV4AOXrxNtb9dJ+B8tVA6Gt1Uf+Jqvt6LRSwqyuGBtShurMD9ThhZD8FW18q93+7Y2MKhbNSokZA9/OCDD+yOly0cShBbg+AaBVmQyIbkYhyBGdvOUpeZe5wN+9fLT++G2+elRtUal45jt9l76bctZ4TN3m7gxpFy32Lk8iM0crlDaxgnKjhZ4RJaCMjELLzw4MXHaFEVYpZ1qCBOKEEnFGohFKriEF5s8YJ74sq/VOUbx2/9G0Uy0AgDRO+F+i6h2w8cYWTHB9ehmDGMOfpGn2Oo1meHMuLJ28KhHDVqFHXr1k0o2OTOnZtix44c69G7d2/br1UrF9Wpq3ep0terBSZGlSlsD6SFBs7ZeY46/LHbOeKgNwrQ26Xs45wPX3qYvlt5LNKGoGZVx4sdgw4NiFr1yd9wjlp5lL5e6uB2hQ469NC5hBYCMjErTZK49Hd3fVrTKkLFBiyja3cfiT+t7FSRkHh26to96lorN7WpZJ+YZn8/VZXBoVXF7NStdl7hWEuezjeLZ6IvG+nneS06YBldf47rySHGT479Pd/o0r+Ve7/dMbOFQ4nMbncF1ydmKOX4+0FYuajOXr9H5Yet4k3cx4c6d9d5av+7gzQYZVjDQtS4RCYfezWvuTz9w0nkySGvio4HLThA49Y5sqqTJ4hNO3vXMG9AL3pC/CTiKFFAcwK6Ey6hhcCQhQfpx7UnKHuqhLSycyXDky89eIW44kYBPdZnf+ykHWdu0pcNC9KbJULnxPv9iVto9eErAgfJmKBKKBqVhi05aDldvvNQnEzihJKLfxCwcu/3zwzM69UWDqV50wlcT1Yuqou37lP4EIe6xLdNXqF6r2QI3MSDeOT5ey5Q22k7nTMY3rgwNShqnxM2efoXJ1YMOjLQcRL51ZJDNHqVIwkmXdJ4tKlb1YA+AZUc/sdmxahm/rQBtYcHtx6Br5ccplGrjlHBDElp3qflDBugEnqv/7wy/fvwCYEuB7cF8eOETtxfi1+20vKDDkowGT96+8FjKtTXwdNpVGZXKu/EiRmDjgwK7E2G4UURRA2s3PvtDgs7lCY9ISsXlaougSBtBGtzMY6AqkKB1kZ1iI2PaKzFmNXHaNjiw5QwTkza37+W4wVi+VEasdxxxQy+P/D+BbJMWH+S+j8nhw+0rnggcQjlscGL22bqdmpVIQd94EVSW9VvVtPxKw5C70DTYAXyObaevJ0W7/9HmNC5Ri5qWyUnPXj8lPL0cijetKqQnboZkNmVjnr82DHp4ADH7wcX8xGwcu8333pze7SFQwki86gyA5mHMvJDR1wM4mNQ+FTI+y/E0v3/UMvJ250djHm7KNUpaB/n/Ke1x2nwwkOUNH5s2t3HcbUtnUz8NxRyoJQTyPLrplPUe66DHH7aR6WoTI5UgTSHxw5CBJDFjGxmFKM8lkE4Xbcmt522g+bvcSgOQQIWPJwqQblRJS+pvJM4biza60WyVHTC1p9zYYcyAl1bOJSTJk2K5FA+fvyYjh49Svj78OHD6e233/bnejClbysX1a37j6lwP8c1yIT3i1OVPGlMmUOodaLKmmHuPzUrRjVsdGU7ft0JGrjgIKVKFIe29awuHo/8G/67SOZkNKdN2YA+tql/n6Yecxzk8LM+DqdiWVIE1B4ePPgQUPkXd/SqTikShqaiS4c/dtGcnefFA1T5hXP1WCS0zSGxC6ldvUUq76gvpHrbcj39CFi59+u3KjA1beFQupv6tGnTaPLkybRo0aLAoGNgVCsXlSpX9usHJalCrpcNWMpVJQKrDl+m5hO3OgGZ+H4JqpwntW0AmrThJPWdd4DSJolHm7s7YiUnbzpFvZ6fCIZnT0m/tSwdUHtVLs95bctRwYxJA2oPDx58CDT6YSNtO31DGL67dw1KmiA0FV26zNhNM7Y7OGb71c1P75VxUJhJWiWjqmjy5Ddlwji0vZfjhZSL+QhYufebb725PdraoTx9+jTly5eP7t51xNfYuVi5qCCnBSqJS3ceELScs6RMaGdobGvb2iNX6N0JW5z22c05X334suDkUx1H1YGrkic1TXi/REDxnbn9HHWe4aBeWtqhAuVKkzig9vDgwYdA03GbaePxa8JwSeAffLPw3WKVY1alMJO0SkZV0eqOWk97zt2i1Inj0pYexumcfJ9RaPRg5d5vd0Rt7VAOHTqURo8eTWfPOmS97FysXlSXbj+gm/ceU+60vIF7uy42HrtKTcf/7WxutxhAxE9hQ8iaKqGIo0SZveMcdZzucODqFExLY94u5u30TWmnUi/h5Qa2cmEEjCAA7e41Rxx0OYFWfjJit9l1e8/dR79uOi26VSnMKn61ik5fu0dDGhSkt0rqp1F6Y8wG2nnmpi3YIMzGyk79Wb3322nuWlts4VCCh1JNysFGevv2bbp58yYNGjSIvvjiCztjKGzjRWX7R/SCgZtPXKMmP212/n1G63AqkdXeMYAq1VGDIhlouAHlDH88IdWejV9UofTJ4vtjGO4zGiPw0a/baNmBS2KGoMcCTVYoFkkQj7mrFGZ/7b5ASCDsX6+AofjSxmM30ZZT1ylj8vi0/vMqoQipJXPmvT8CZls4lH379n0hyztOnDhUtmxZqlChgiWLwtdBeFH5iqD17beeuk7/G7vJOfCcNmWoSObk1htiYEQ1Mx0yh5A7DGRZvO8itZ6yQ5iwvWc1SpkobiDN4bGDEIFPpu6gBXsd2c3QAo8RohKBkiAeOJhBYfbWT5tp04lrlCVlAlrTpXIQrozgMJn3fps5lMGxbKK2khdV8D1F8Oc1/GGj0/BgSCpRE4k+KJuNer+eL6DA42QJJ0woe/vWoMTxQjOhIqAPIcgHV7ObTw11KEKFYhm2+BCNWe0QLTCDwqzZz3/TuqNXKfvLCWllp8Dy1Ubn58l7vw0dyg0bNtD69evp0aNHgntLLazlHZ2/joGb2+6zN6ne6A1OA8DpCG5HOxc17rNNpRzUtVaegJq76tBlaj7JkSl/eGAtihsrdJRNAgp8NBq868zdNH3bOYoV4yU6FsISgcOXHaHvVhwVT9YMCrPmE7fQqsNXKFeaRLS0Q8VotGLsNRV2KG3mUCL5pnv37pQ8eXJKkiTyhs5a3vb68kQna/adv0Wvfb/eOaXlHStQWGp7JzltO3WdGj2/pu9UPRd9aoCXzh/PDlrD1Uesobxpkwhi86gECvwxPvcZ/Aj0/HMvTdl8RsROSonR4J+V8Rl8v+IofbPMoYJlBoVZi1+20fKDlyhP2sS0+LPgCB0zjlrgW7BDaTOHMmPGjPTBBx9Q//79A786vLSAF5WXwAWw2YELt6nOd+ucFgRDlvKeczep7ijHqWqPOnnpowrZA4igY+iHT54S9ILZmQz4owhKA/rN208TN5yiBHFi0oHnEqNBOREfjf5h9XH6cvEh0cvkD0tS+Zy+8Qt/PGU7Ldr3DxXIkITmfxpYRS0fobF1c977beZQJkiQgPbs2UNhYWG2XjhRGceLKvge3eF/7lDNkWudhq/rWpkypUhg64lAog6ExSgD6uWnZuEO8mMujECwIjBk0UH6cc0JShwvFu3tWzNYp+Gz3aoK1m8flabwHCl96lNKORbOmJTmti3nU1/c2D0CvPfbzKGsUaMGNW/enN56662gXbe8qILv0R27fIeqDY9wKDd3q0ppk8az9USOX/mXqn6zRtg4rFEhalw8k63tZeMYAU8IfLP0MH2/8hglSxCbdvV2aNaHYpHKWJj7zNbhVNxHCjOZ7FQ0czKaHWCJ1uj8PHnvt5lDOX78eOratatwKvPkyUNx40amHnn33Xdtvx55Udn+Eb1g4Mmrd6ny16udf9/Wsxqlsjntzdnr96j8sFXC5u/fKkKvF04ffMCzxYyAgoCMHVQ160MRoCmbT1PPP/eJqf/5SVl6JVMyn2CAghWUrEpmTUHTW4f71Bc3do8A7/02cyhjxHBPZIu4rKdPn9p+PfOisv0jesHAM9fuUYWvHM4Zyq7e1SlZgji2nsjl2w+o5OAVwsZx7xan6vnS2NpeNo4R8ITA2DXHaeiiQyEvEajKqs7/tBwVyJDUE3RRft5t9h76bctZKp09Bf3ekh1Kn8CMojHv/TZzKP31oNEv9MCzZn0xzmzcuHHUokUL2rVrF7Vv3562bdtGKVOmpHbt2lHnzp0Nm8SLyjBkAW9w7sY9KvdlhEMZDDrCN+4+oiIDlgnszAjcD/hDYANCHoEJ609S//kHKH3SeLSxW9WQxQOniThVRFn8WXnKk9Y3CjOZPV8uLBVNaVEqZHH198R57w8hh3LevHnUuHFjOnHiRKQs1KRJk9K9e/fEFXv9+vWpY8eOtHnzZmrTpg2NGTNGXL8bKbyojKBlj7oXb92n8CErncYEg47wk6fPKHzoSrr270Na27UyZUxu7yQiezxptsLOCKw+fJnen7iVwrOnpN9alrazqX61be6u89T+911ijOUdK1JY6kQ+jffz+pM0YP4Beqd0ZhpYP7CKWj5NxOaNee8PIYdy8ODBNH36dHESqS1Dhgyh0aNH06lTpyhWrFjiY/Bhzp49mw4dctA36C28qPQiZZ96l+88oJKDHNfHKMcH16GYQSD79s+tB3TnwWPKmcbenJn2edJsiZ0RgJDF3vO3KGuqhJQkhJWWFuy5SJ9Mc8iYrulSibKkTOjTY3v67D/adfYm5U+fhOLFZsEBn8CMojHv/SHkUDZp0oQQozlt2rQXlkSdOnUoWbJkkT5bvnw5Va9enS5dukSpU6fWvQZ5UemGyjYVccpXbOBypz0nh9RhLkXbPB02hBEILQSW7P+HWk3eLia94YsqlCFZ/NACIEhny3t/CDmU+fPnp3Tp0tHDhw/pyJEjlDNnTurVqxfVrFmTChUqRLVr16Yvv/zSiciBAwcIbbZu3UrFixfXvcR5UemGyjYVb957RK/0d8Qjxo75Eh0dVMc2trEhjAAjEFoIrDx0iT6YtE1Mekv3qpQ6ib0pzELr6bifLe/9IeJQQhc8YcKEVKRIEfrqq68oUaJENGXKFPr2229p2bJl1KpVK2ratGkkhR7EWubIkYPWrVtH5cq5J4O9ffs24X+yXLx4kUqWLElnz54lKP9wsT8Ctx88pkJ9lwpD48eOSQcH1LK/0WwhI8AIREsE1h65Qu9O2CLmtqNXdUqR0N6ME9HyIXgxKXYoQ8ShxDTv3r0r4iNVbstatWqJq00sBFx7uzqh3LFjh3BE3ZW+fftSv379XviYHUovvpEBanLv0RPK13uJGD3UVToC9Ah4WEaAEXiOwMbjV6npuL/Fv/b0rRHS8aTBtCjYoQwhh9LVwuzSpQstXryYMmXKJKiCJk+e7KymN4aSTyiD6Svv2tYHj59Snl6LxYfJE8SmnSGs0hH8T5NnwAgENwK37j2mqsPXUOYU8WnWx2U4njtIHic7lCHiUEIfvEyZMsJ5VK+vK1euTGnSpKFXXnmFxo4dS8ePH6eYMR1ZcN26daM5c+ZwlneQfJl9MfPx02eUs8ci0cXLiePS1h7VfOmO2zICjAAj4BMCj548o1gxXqIYQcA24dNEo1FjdihDxKF89uyZcChx7f3DDz9QqlSp6KeffhI8k0i6gVMJHsq6desK6cctW7bQxx9/LJzM9957z9CS50VlCC5bVH727D/K3n2hsCXUSZVt8UDYCEaAEWAEggwB3vtDxKHENK9cuUJffPEFLVy4kG7evElFixaloUOHUvny5QUKcCyhjrNz506RDd6pUydq27at4SXNi8owZLZokPWLBcKOTCni07quVWxhExvBCDACjAAjEBwI8N4fQg6lVUuSF5VVSJs7Tlj3hfQEJ5WpEtLKzpXM7Zx7YwQYAUaAEYjWCPDezw6l6QucF5XpkFrSYa6eiwhxS7nSJKKlHSpaMiYPwggwAowAIxA9EOC9nx1K01cyLyrTIbWkw7y9FtP9x08pX7oktLC9IwyCCyPACDACjAAjoAcB3vvZodSzTgzV4UVlCC7bVC7QZwn9+/AJFcqYlP5q657I3jYGsyGMACPACDACtkGA9352KE1fjLyoTIfUkg4L91tKt+4/pqKZk9HsNmUtGZMHYQQYAUaAEYgeCPDezw6l6SuZF5XpkFrSYdEBy+j63UdUMlsKmt4q3JIxeRBGgBFgBBiB6IEA7/3sUJq+knlRmQ6pJR0WH7icrv77kMqGpaSpLUpbMiYPwggwAowAIxA9EOC9nx1K01cyLyrTIbWkw1KDl9Ol2w+pQq6X6dcPSloyJg/CCDACjAAjED0Q4L2fHUrTVzIvKtMhtaTDskNX0vmb96lqntT08/slLBmTB2EEGAFGgBGIHgjw3s8OpekrmReV6ZBa0mH5YSvp7PX7VDN/GvqxWXFLxuRBGAFGgBFgBKIHArz3s0Np+krmRWU6pJZ0WOmrVXTq2j16tWA6Gv12UUvG5EEYAUaAEWAEogcCvPezQ2n6SuZFZTqklnRY5ZvVdOLKXar3Snr6tkkRS8bkQRgBRoARYASiBwK897NDafpK5kVlOqSWdFhjxBo6culfalg0I33TuLAlY/IgjAAjwAgwAtEDAd772aE0fSXzojIdUks6rDVyLR365w41KZGJhjYsZMmYPAgjwAgwAoxA9ECA9352KE1fybyoTIfUkg5f/W4d7b9wm94ulZkGvVHQkjF5EEaAEWAEGIHogQDv/exQmr6SeVGZDqklHdYdtZ72nLtF75fJSn3r5rdkTB6EEWAEGAFGIHogwHs/O5Smr2ReVKZDakmH9UdvoF1nb1KLctmo52v5LBmTB2EEGAFGgBGIHgjw3s8OpekrmReV6ZBa0mHTcZtp4/Fr1K5KGHWskduSMXkQRoARYAQYgeiBAO/97FCavpJ5UZkOqSUdbjx2lX7bepa61sxNmVIksGRMHoQRYAQYAUYgeiDAez87lKavZF5UpkPKHTICjAAjwAgwArZGgPd+dihNX6C8qEyHlDtkBBgBRoARYARsjQDv/exQmr5AeVGZDil3yAgwAowAI8AI2BoB3vvZoTR9gfKiMh1S7pARYAQYAUaAEbA1Arz3s0Np+gLlRWU6pNwhI8AIMAKMACNgawR472eH0vQFyovKdEi5Q0aAEWAEGAFGwNYI8N4fQg7l9evXqXv37jR//ny6ffs2FSpUiIYOHUrlypUTKDRv3pwmTZoUacFmyJCBsEiMFF5URtDiuowAI8AIMAKMQPAjwHt/CDmUNWrUoMuXL9OoUaPo5ZdfptGjR9P48eNpx44dlCdPHipWrBjVqlWLPv30UycqMWPGFHWNFF5URtDiuowAI8AIMAKMQPAjwHt/iDiUx44do5w5c9KGDRuoTJkyYtb//fcf5cqVi9566y3q06cPJUqUiH777TeqX7++TyubF5VP8HFjRoARYAQYAUYg6BDgvT9EHMqbN2/Sxo0bqWrVqhQ3blznrMPCwsSpZNu2bSlv3rx0+PBh4WT6UnhR+YIet2UEGAFGgBFgBIIPAd77Q8ShdLU0Z8yYQY0bN6a//vqLHjx4QG+++Sa1bt2aFi1aRDFixKA6derQwIEDKWnSpIZWNi8qQ3BxZUaAEWAEGAFGIOgR4L0/RB1KXH3jZBInln/++ae48h40aBANHjxYOJJHjx6lzp07U6ZMmWjlypXCwXRXkOCD/8ly8eJFKlmyJJ09e5YyZswY9F8SngAjwAgwAowAI8AIRI0AO5Qh6FDOnTuXmjZtSqVLlxYZ3/HjxxfxlLdu3aJkyZI5Edm8eTOFh4cT/r9UqVJuV1Lfvn2pX79+L3zODiX//DACjAAjwAgwAqGBADuUIeZQIsO7ffv21KBBA5oyZUqkeErtkr97965I1Jk5cyY1bNiQTyhD4zeBZ8kIMAKMACPACBhGgB3KEHIof/jhB2rTpg21a9eORowYEekaGyeW165doyVLljgRWb16NVWuXJkOHDggEnb0Fl5UepHieowAI8AIMAKMQPRAgPf+EHEojxw5QgUKFKDXXnuNxowZE2n14sp73bp1VLduXRowYAA1adKEUP+TTz4RV95Tp041tNp5URmCiyszAowAI8AIMAJBjwDv/SHiUCLZpkePHi4X7HvvvScUcmbNmiUScw4dOiRiKXFqiSzvePHiGVrovKgMwcWVGQFGgBFgBBiBoEeA9/4QcSitXKm8qKxEm8diBBgBRoARYAQCjwDv/exQmr4KeVGZDil3yAgwAowAI8AI2BoB3vvZoTR9gfKiMh1S7pARYAQYAUaAEbA1Arz3s0Np+gLlRWU6pNwhI8AIMAKMACNgawR472eH0vQFyovKdEi5Q0aAEWAEGAFGwNYI8N7PDqXpC5QXlemQcoeMACPACDACjICtEeC9nx1K0xcoLyrTIeUOGQFGgBFgBBgBWyPAez87lKYvUF5UpkPKHTICjAAjwAgwArZGgPd+dihNX6C8qEyHlDtkBBgBRoARYARsjQDv/exQmr5AeVGZDil3yAgwAowAI8AI2BoB3vvZoTR9gfKiMh1S7pARYAQYAUaAEbA1Arz3s0Np+gLlRWU6pNwhI8AIMAKMACNgawR472eH0vQFyovKdEi5Q0aAEWAEGAFGwNYI8N7PDqXpC5QXlemQcoeMACPACDACjICtEeC9nx1K0xcoLyrTIeUOGQFGgBFgBBgBWyPAez87lKYvUF5UpkPKHTICjAAjwAgwArZGgPd+dihNX6C8qEyHlDtkBBgBRoARYARsjQDv/exQmr5AeVGZDil3yAgwAowAI8AI2BoB3vvZoTR9gfKiMh1S7pARYAQYAUaAEbA1Arz3s0Np+gLlRWU6pNwhI8AIMAKMACNgawR472eH0vQFyovKdEi5Q0aAEWAEGAFGwNYI8N7PDqXpC5QXlemQcoeMACPACDACjICtEeC9nx1K0xcoLyrTIeUOGQFGgBFgBBgBWyPAez87lKYvUF5UpkPKHTICjAAjwAgwArZGgPd+dihNX6C8qEyHlDtkBBgBRoARYARsjQDv/exQRlqgz549o379+tH48ePpxo0bVK5cORozZgyFhYXpXsi8qHRDxRUZAUaAEWAEGIFogQDv/exQRlrIcCbhQE6cOJEyZMhAXbt2pRMnTtD+/fspTpw4uhY9LypdMHElRoARYAQYAUYg2iDAez87lE4EHj16RKlSpaJhw4ZR69atxd9v3rxJ6dOnpwkTJlCTJk10LXxeVLpg4kqMACPACDACjEC0QYD3fnYonQhs2bKFSpUqRYcPH6ZcuXI5/45r70KFComTSz2FF5UelLgOI8AIMAKMACMQfRDgvZ8dSicCs2fPpoYNG9K9e/cofvz4zr83btxY/G3+/Pm6Vj4vKl0wcSVGgBFgBBgBRiDaIMB7PzuUTgSmTJlCzZo1o6dPn1KMGDGcf3/33XfpwoULtHz5cpcL//bt24T/yXL27FkqU6YM4cQzXbp00ebLwhNhBBgBRoARYAQYAdcIXLx4kUqWLEknT56krFmzhjRML/3333//hTICs2bNokaNGrk8oXz48CHNnTvXJTx9+/YVmeFcGAFGgBFgBBgBRiC0EcBhUokSJUIahJB3KGUM5bFjxyhHjhzOxYAYysKFC9Po0aN1nVA+ePCAcEqZLVs2ihUrlqmLSr4B8emnqbBG6owx9h+2as+Ms/9xZowZY/8jYM0IwbCWnzx5QleuXKGCBQtSvHjxrAHGpqOEvEOJU8jUqVPT8OHD6cMPPxSPyZssb38+X47R8Ce6jr4ZY/9jzDgzxtYg4P9R+PfC/xjz74U1GJs5Ssg7lACzR48e9OOPPwqaIMRAdOnShU6dOkV79+7VzUNp5kPR9sU/Xv5Elx1K/6MbMQKvZf+jzRgzxv5HwJoReC1bg7NZo7BDSSQScrp37y6Ize/fv08VKlQQV912CbDlL5VZy919P4yx/zHmEwfG2BoE/D8K/174H2P+vbAGYzNHYYfSTDT91BeyyXEl37FjR0qSJImfRgntbhlja54/4+x/nBljxtj/CFgzAq9la3A2axR2KM1CkvthBBgBRoARYAQYAUYgRBFghzJEHzxPmxFgBBgBRoARYAQYAbMQYIfSLCS5H0aAEWAEGAFGgBFgBEIUAXYoQ/TB87QZAUaAEWAEGAFGgBEwCwF2KM1C0g/9PHv2TKjxjB8/nm7cuEEgWx8zZgyFhYX5YbTQ6PL06dMus/fHjRtHLVq0oF27dlH79u1p27ZtlDJlSmrXrh117tw5NMAxaZYDBw4UkqWrV6929ugJV17rxsB3hXHz5s1p0qRJkTrKkCGD4FhFYYw9Y3z9+nXB+DF//nwhrVuoUCEaOnSo+O1F4XXsGUM9NTzhzGtZD4r2q8MOpf2eidMiOJNwIEFnhI2ha9eudOLECdq/f78t+DFtDJ1b0+bNm0eNGzcWOL700kvOekmTJhXym3ny5KH69euLjPrNmzdTmzZtxDPADxwXzwiMHDlSYAfqLelQXrt2zSOuvNY9YytruMIYnxUrVoxq1apFn376qbOzmDFj0v/bO9OQqN4ojJ/sQ4ttVFJRFi0GVqKGpQURISqtlFZq0A5pFFSUUZBYRjtYkZUW1Yd2NdM22xDag8KiDcuiEgosqPwQRQv9ec6/O+g0zdxRp7mOz/ulHN9773t/7/HOc89533MCAgL0ZzJ2zTg2NlbevXsn2dnZyg3p4/BCX1ZWpj+7ej6QsWvG6OGMMxjTls1xtFovCkqrzcjv8Xz79k06d+4smzdvltTUVP3UahV8LIrO6bDWr18veXl56mmwbwXY/HAAAAljSURBVBs2bNAvECS1N8pnwltRWFgo5eXljfF2/9mY37x5ox7ea9euSWBgoHTp0sUmKF1xpa2bmyZnjJFLt02bNnL06FF9IbJvZOyaMcrvBgUFyY0bN2T48OF6wK9fv6R///6SnJwsrVq1cvp8IGPXjNHDFeeMjAzasjmUlutFQWm5Kfl/QEaN8adPn+oDzWgIvSAMA68Zm/sEkpKSxM/PT44cOfLHwWPGjJEOHTrU+h1CtzExMVJVVaUlOtkcE4DnF2IGodjMzEwV5YaH0hVX9I2MjBTaunPrcsYYLzzBwcF/MDTOyOeJ679cvLDfvHlToqOjpUWLFrYDsMQInl9ENZw9H2jHrhmjhyvOCxcupC2bQ2m5XhSUlpuS/wcEr1hCQoKGYfFmbDSEa/EZ1viwuU9g4MCB0q1bN0EN92fPnqlHIj09XeLi4lSojx49WjZt2mQ78ZMnTwTH3LlzRyIiIty/YBM8YtasWbUEpSuulZWVtHU37cSecX5+viQmJmo0o6SkRF+aIOQh8LGcg88TNwH/7g6ueOaeOnVKS/Q6ez7QjuvGGEfV5Pz161fact1RevVICkqv4v/7xQ8dOiTTp0/XspD4cjDajBkz5O3bt7rpgc09AghJ+fv7S3h4uGzZskXDKuC8fft2uXTpkqSkpMi0adPUw2Y0eCX69u2roVxjYb57V216ve3FDjw8zrjCs0Nbd89O7BkjTLhu3TrBkg4IyYqKCt1MhuUHpaWl6nUnY/cYI/QNzyQ8lkVFRboZknbsHkMzve0505bNULNmHwpKa86LnDhxQiZPnuzQQwnvWnFxsUVHbu1hff78WddH1gxp4UsDG3SwGxZfxo48lFiUDyHK5pqAvdgJCQlxyhWinbbummvNHvaMsdavurpaQ7JGw6ayYcOG6eYy2DYZm2eM5yvEY1RUlEaDECWiHZvnZ7anI860ZbP0rNePgtJ6c6IjMtY8YQEzPGRGg5csNDRUF4ezNQyBtLQ0OX/+vHpzkCro4MGDthNzDaX7jO3FDkS6M67G2jPaunnW9owdHYmXJ3jhCwoK1LaxTpWMXTPGDm+kDouPj9cIhvHySTt2zc6dHn/jTFt2h6K1+lJQWms+bKOBFxKbQLKysmTu3Ln6OXd512+yHjx4oLs3IR5rhq9HjRqlu5LDwsIkJydHXrx4IUi3grZy5Uo5efIkd3m7gd5e7CCPnzOutHU34P7uas8Y3jSkZ7pw4YLtZNgUBdvGOuA+ffrweWIC8+7duzVVGPLPbt26tdZyI9qxCYAmuzjjTFs2CdGC3SgoLTgpxpCwCDw3N1f279+vybjhSYM35+HDh8xDWYd5Q2JnCEp4bvBAQ1qmPXv26I55bLqBqEQOtAkTJmjOT3iJ58+fr2Jo5syZdbhi0zzEXuwgr58rrrR192zFnjHCsrDbtWvXCjIZYMPZggULNOR9+PBhPTkZO2cMZoMGDZJx48b9kUUDIW+8+NCO3bNTR71dccZ6ddpy/Tl74wwUlN6gbvKa2JCDPIhIbP7lyxdNFo1QN8QlW90IvH//XlasWCHnzp1Tj+/gwYO1EsaIESP0hBCW8E7cu3dPd4MvXbpUkMaCzTwBR+FYV1xp6+b5oqcjxlh3jY05SCGEtZTw9GCXd8uWLfXkZOycMTY0QXQ7anihRBUi2rF7duqotxnOtOX6c/bGGSgovUGd1yQBEiABEiABEiABHyJAQelDk8lbIQESIAESIAESIAFvEKCg9AZ1XpMESIAESIAESIAEfIgABaUPTSZvhQRIgARIgARIgAS8QYCC0hvUeU0SIAESIAESIAES8CECFJQ+NJm8FRIgAc8SQIYF7LBevXq1Ry5k5I58+fLlX7M5eHoMHrkxnpQESMDnCVBQ+vwU8wZJgAQaigDSTiEnISrQeKKh3vyHDx8kICDAllzf/joUlJ4gz3OSAAnUlwAFZX0J8ngSIAES+IcEKCj/IWxeigRIwDQBCkrTqNiRBEigsRBo1qyZJqlGXXZUOLly5Yr07t1b0tPTtT5zdXW1VkXJzMyU2NhY221dvHhRy20a5QqR2B6lT40QtL2YO3v2rFanefTokbRr106Sk5M1ubiRTBzjQDWmvLw8uX79unTs2FET5eMajpp9yBvjRKL94uJirY6FQgfbtm3zaNi9scwxx0kCJGAtAhSU1poPjoYESKABCEDIobRmSUmJ/PjxQ6KiorRyzOPHj1WQ9ejRQ06fPq1Vk1CrfezYsXL//n0ZMmSILFmyREUkfkbpzY8fPzoUlEVFRZKQkKDrKadOnSoVFRXaf+jQoYJKH2gYR/v27SU7O1s/h8BF9ZqrV6/aqjPVvF17QRkXFyeVlZUqStu2bauVm0pLSyUjI8Nj6zgbAD9PQQIk0AQJUFA2wUnnLZOArxOAkIMwzMrK0lt9/vy5BAUFaem8iIgI2+2jpN7r168FQg7/R53hW7du2X6/Y8cO9RA68lBGRkZKYGCgFBQU2PpDpKIOMTycwcHBKigXLVqkItZo8FKmpaU59FLWFJRG7ejLly9LdHS0Hl5VVSW9evVSIeypjUG+bhu8PxIgAc8QoKD0DFeelQRIwIsEIOR27dqlHkO0/Px89SL6+/vXGtX379+17jWEWkhIiMTExNhEKDrCSxkeHu5QULZu3VpQl3jx4sW2cyJEjfMdO3ZMEhMTVVDm5ORISkqKrQ/C5rNnz1Yvo32rKSjv3r0rU6ZM0fA8wulGGzBggN4LBaUXDYyXJgES+IMABSWNggRIwOcIQMgdOHBA1xqiHT9+XJKSkqSsrExDxzVb8+bNdX0lhOPIkSNreROdCUrs9t64caN6II2GHdqdOnXSkHd8fLwKyprjQD9nm2ocCUqE3CFSjRYaGiqTJk2ioPQ5q+UNkUDjJkBB2bjnj6MnARJwQMBeyGHtJDbhnDlzRtdLGg0bd/z8/HRjDcRneXm53L592/b73NxcSU1NdeihxJrInj171gp5Y/PMxIkTa4W86yoof/78Kf369as15k+fPkn37t01ZE4PJU2fBEjASgQoKK00GxwLCZBAgxBw5BkcP368hrB37typ4rKwsFCWL18u+/bt0xA0dmqHhYXJsmXLZM6cOSoK582bJ8g9+erVK127WNO7iDA6wtpr1qzRf7H+EiF2nANrKdHq46HEtRDaxrrPvXv3SteuXXXdJTYarVq1ioKyQSyFJyEBEmgoAv8BnLTIG4vP/ScAAAAASUVORK5CYII=\" width=\"599.9999869953505\">"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "pb_num_dict = {_k:len(_v.kept_probes) for _k,_v in filtered_pb_dic.items()}\n",
    "\n",
    "pb_nums = np.array(list(pb_num_dict.values()))\n",
    "plt.figure(figsize=(6,3), dpi=100)\n",
    "plt.plot(pb_nums)\n",
    "#plt.plot(np.ones(len(pb_nums))*20)\n",
    "plt.ylabel(f\"num of probes\", labelpad=1)\n",
    "plt.xlabel(f\"region id\", labelpad=1)\n",
    "plt.ylim([0, num_probes_kept*1.1])\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# 3. Load and asssemble probes"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 3.1 design readout mapping dict (region_to_readout)\n",
    "\n",
    "idea for this library:\n",
    "\n",
    "1. each gene has bit 1-60 as common readouts to allow parallel sequential imaging\n",
    "\n",
    "2. each gene has unique readouts (the same encoding probe has one site for part 1, one site for part 2)\n",
    "\n",
    "3. eventually 5% of all probes have their probes converted into gene-specific readouts"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'Ccnd2-1': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff88fbc8>,\n",
       " 'Ccnd2-10': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff88fd88>,\n",
       " 'Ccnd2-11': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff88fec8>,\n",
       " 'Ccnd2-12': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251aa96d3c8>,\n",
       " 'Ccnd2-13': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251aa96e4c8>,\n",
       " 'Ccnd2-14': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff8cf448>,\n",
       " 'Ccnd2-15': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff8cf608>,\n",
       " 'Ccnd2-16': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251aeb7bd88>,\n",
       " 'Ccnd2-17': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff8cfb08>,\n",
       " 'Ccnd2-18': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff8cfc88>,\n",
       " 'Ccnd2-19': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff8cff08>,\n",
       " 'Ccnd2-2': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2512a979288>,\n",
       " 'Ccnd2-20': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2512a979f08>,\n",
       " 'Ccnd2-21': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2512a97c6c8>,\n",
       " 'Ccnd2-22': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2512a97f3c8>,\n",
       " 'Ccnd2-23': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff929d48>,\n",
       " 'Ccnd2-24': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff929ec8>,\n",
       " 'Ccnd2-25': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff92a1c8>,\n",
       " 'Ccnd2-26': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff92a288>,\n",
       " 'Ccnd2-27': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff92a3c8>,\n",
       " 'Ccnd2-28': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff92a5c8>,\n",
       " 'Ccnd2-29': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff92a788>,\n",
       " 'Ccnd2-3': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff92a488>,\n",
       " 'Ccnd2-30': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff92aa48>,\n",
       " 'Ccnd2-31': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff92ac08>,\n",
       " 'Ccnd2-32': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff92ad88>,\n",
       " 'Ccnd2-33': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff92af48>,\n",
       " 'Ccnd2-34': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2517f92b0c8>,\n",
       " 'Ccnd2-35': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2517f92b248>,\n",
       " 'Ccnd2-36': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2517f92b448>,\n",
       " 'Ccnd2-37': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2517f92b708>,\n",
       " 'Ccnd2-38': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2517f92b808>,\n",
       " 'Ccnd2-39': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff8cfa48>,\n",
       " 'Ccnd2-4': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2517f92bc08>,\n",
       " 'Ccnd2-40': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2517f92bdc8>,\n",
       " 'Ccnd2-41': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2517f92bbc8>,\n",
       " 'Ccnd2-42': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff92c0c8>,\n",
       " 'Ccnd2-43': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff92c288>,\n",
       " 'Ccnd2-44': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff92c408>,\n",
       " 'Ccnd2-45': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff92c588>,\n",
       " 'Ccnd2-46': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff92c748>,\n",
       " 'Ccnd2-47': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff92c948>,\n",
       " 'Ccnd2-48': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff92cac8>,\n",
       " 'Ccnd2-49': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff92cdc8>,\n",
       " 'Ccnd2-5': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff92ce88>,\n",
       " 'Ccnd2-50': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff92cf88>,\n",
       " 'Ccnd2-51': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2517f92b908>,\n",
       " 'Ccnd2-52': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251bfeddc48>,\n",
       " 'Ccnd2-53': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2517f933c48>,\n",
       " 'Ccnd2-54': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff933148>,\n",
       " 'Ccnd2-55': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff933348>,\n",
       " 'Ccnd2-56': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff9334c8>,\n",
       " 'Ccnd2-57': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff933608>,\n",
       " 'Ccnd2-58': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff9337c8>,\n",
       " 'Ccnd2-59': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff933948>,\n",
       " 'Ccnd2-6': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff933ac8>,\n",
       " 'Ccnd2-60': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff933e08>,\n",
       " 'Ccnd2-7': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff933f08>,\n",
       " 'Ccnd2-8': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff930048>,\n",
       " 'Ccnd2-9': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff930248>,\n",
       " 'Nkx2-1-1': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff9303c8>,\n",
       " 'Nkx2-1-10': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff9304c8>,\n",
       " 'Nkx2-1-11': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff9306c8>,\n",
       " 'Nkx2-1-12': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff930848>,\n",
       " 'Nkx2-1-13': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff9309c8>,\n",
       " 'Nkx2-1-14': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff930b48>,\n",
       " 'Nkx2-1-15': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff930d88>,\n",
       " 'Nkx2-1-16': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff930ec8>,\n",
       " 'Nkx2-1-17': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff938048>,\n",
       " 'Nkx2-1-18': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff938108>,\n",
       " 'Nkx2-1-19': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff938388>,\n",
       " 'Nkx2-1-2': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff938508>,\n",
       " 'Nkx2-1-20': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff938348>,\n",
       " 'Nkx2-1-21': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff9387c8>,\n",
       " 'Nkx2-1-22': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff938948>,\n",
       " 'Nkx2-1-23': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff938ac8>,\n",
       " 'Nkx2-1-24': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff938c48>,\n",
       " 'Nkx2-1-25': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff938dc8>,\n",
       " 'Nkx2-1-26': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff9380c8>,\n",
       " 'Nkx2-1-27': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2517f93b0c8>,\n",
       " 'Nkx2-1-28': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2517f93b2c8>,\n",
       " 'Nkx2-1-29': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2517f93b488>,\n",
       " 'Nkx2-1-3': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2517f93b688>,\n",
       " 'Nkx2-1-30': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2517f93b448>,\n",
       " 'Nkx2-1-31': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2517f93b948>,\n",
       " 'Nkx2-1-32': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2517f93bac8>,\n",
       " 'Nkx2-1-33': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2517f93bc08>,\n",
       " 'Nkx2-1-34': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2517f93bd08>,\n",
       " 'Nkx2-1-35': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2517f93bf08>,\n",
       " 'Nkx2-1-36': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff9371c8>,\n",
       " 'Nkx2-1-37': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff937548>,\n",
       " 'Nkx2-1-38': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff937188>,\n",
       " 'Nkx2-1-39': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff937908>,\n",
       " 'Nkx2-1-4': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff937a88>,\n",
       " 'Nkx2-1-40': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff937c08>,\n",
       " 'Nkx2-1-41': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff937d88>,\n",
       " 'Nkx2-1-42': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff937388>,\n",
       " 'Nkx2-1-43': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff937488>,\n",
       " 'Nkx2-1-44': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff937f88>,\n",
       " 'Nkx2-1-45': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff93c308>,\n",
       " 'Nkx2-1-46': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff93c488>,\n",
       " 'Nkx2-1-47': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff93c148>,\n",
       " 'Nkx2-1-48': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff93c788>,\n",
       " 'Nkx2-1-49': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff93c988>,\n",
       " 'Nkx2-1-5': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff93cb08>,\n",
       " 'Nkx2-1-50': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff93cc88>,\n",
       " 'Nkx2-1-51': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff93cdc8>,\n",
       " 'Nkx2-1-52': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff93cf88>,\n",
       " 'Nkx2-1-53': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2517f93d108>,\n",
       " 'Nkx2-1-54': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff93cf48>,\n",
       " 'Nkx2-1-55': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2517f93d4c8>,\n",
       " 'Nkx2-1-56': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2517f93d648>,\n",
       " 'Nkx2-1-57': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2517f93d308>,\n",
       " 'Nkx2-1-58': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2517f93d908>,\n",
       " 'Nkx2-1-59': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2517f93da88>,\n",
       " 'Nkx2-1-6': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2517f93dc08>,\n",
       " 'Nkx2-1-60': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2517f93ddc8>,\n",
       " 'Nkx2-1-7': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2517f93d248>,\n",
       " 'Nkx2-1-8': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff9410c8>,\n",
       " 'Nkx2-1-9': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff941288>,\n",
       " 'Slc17a7-1': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff941448>,\n",
       " 'Slc17a7-10': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff9415c8>,\n",
       " 'Slc17a7-11': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff941308>,\n",
       " 'Slc17a7-12': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff9418c8>,\n",
       " 'Slc17a7-13': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x237cf88cac8>,\n",
       " 'Slc17a7-14': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff941c08>,\n",
       " 'Slc17a7-15': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff941d88>,\n",
       " 'Slc17a7-16': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff9411c8>,\n",
       " 'Slc17a7-17': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff941f08>,\n",
       " 'Slc17a7-18': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff943208>,\n",
       " 'Slc17a7-19': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff9433c8>,\n",
       " 'Slc17a7-2': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff943548>,\n",
       " 'Slc17a7-20': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff943388>,\n",
       " 'Slc17a7-21': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff943808>,\n",
       " 'Slc17a7-22': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff943988>,\n",
       " 'Slc17a7-23': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff943b48>,\n",
       " 'Slc17a7-24': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff943cc8>,\n",
       " 'Slc17a7-25': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff943148>,\n",
       " 'Slc17a7-26': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff948048>,\n",
       " 'Slc17a7-27': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff948208>,\n",
       " 'Slc17a7-28': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff9483c8>,\n",
       " 'Slc17a7-29': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff948548>,\n",
       " 'Slc17a7-3': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff948688>,\n",
       " 'Slc17a7-30': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff948808>,\n",
       " 'Slc17a7-31': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff948988>,\n",
       " 'Slc17a7-32': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff948b48>,\n",
       " 'Slc17a7-33': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff948c88>,\n",
       " 'Slc17a7-34': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff948148>,\n",
       " 'Slc17a7-35': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2513fee11c8>,\n",
       " 'Slc17a7-36': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff94d088>,\n",
       " 'Slc17a7-37': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff94d208>,\n",
       " 'Slc17a7-38': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff94d348>,\n",
       " 'Slc17a7-39': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff94d4c8>,\n",
       " 'Slc17a7-4': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff94d648>,\n",
       " 'Slc17a7-40': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff94d908>,\n",
       " 'Slc17a7-41': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff94da08>,\n",
       " 'Slc17a7-42': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff94db48>,\n",
       " 'Slc17a7-43': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff94dd08>,\n",
       " 'Slc17a7-44': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff94de88>,\n",
       " 'Slc17a7-45': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff94dcc8>,\n",
       " 'Slc17a7-46': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff94f188>,\n",
       " 'Slc17a7-47': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff94f348>,\n",
       " 'Slc17a7-48': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff94f4c8>,\n",
       " 'Slc17a7-49': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff94f688>,\n",
       " 'Slc17a7-5': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff94d848>,\n",
       " 'Slc17a7-50': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff94fa88>,\n",
       " 'Slc17a7-51': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff94f808>,\n",
       " 'Slc17a7-52': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff94fd08>,\n",
       " 'Slc17a7-53': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff94fec8>,\n",
       " 'Slc17a7-54': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ff94fcc8>,\n",
       " 'Slc17a7-55': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2505543e648>,\n",
       " 'Slc17a7-56': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2513feec6c8>,\n",
       " 'Slc17a7-57': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2513feee4c8>,\n",
       " 'Slc17a7-58': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2513feef988>,\n",
       " 'Slc17a7-59': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251c66a5e88>,\n",
       " 'Slc17a7-6': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2511544b1c8>,\n",
       " 'Slc17a7-60': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250ffa4c088>,\n",
       " 'Slc17a7-7': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250b48cf108>,\n",
       " 'Slc17a7-8': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251c8990c48>,\n",
       " 'Slc17a7-9': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251a1d42dc8>,\n",
       " 'Slc32a1-1': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251cebdbfc8>,\n",
       " 'Slc32a1-10': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251cd3d77c8>,\n",
       " 'Slc32a1-11': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251cd3d7348>,\n",
       " 'Slc32a1-12': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251cd3d7488>,\n",
       " 'Slc32a1-13': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251cd3d7688>,\n",
       " 'Slc32a1-14': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251cd3d7988>,\n",
       " 'Slc32a1-15': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251cd3d7e48>,\n",
       " 'Slc32a1-16': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x238538a33c8>,\n",
       " 'Slc32a1-17': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2505543a648>,\n",
       " 'Slc32a1-18': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2505543ac48>,\n",
       " 'Slc32a1-19': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251cebdc088>,\n",
       " 'Slc32a1-2': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251cebdc2c8>,\n",
       " 'Slc32a1-20': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251cebdc688>,\n",
       " 'Slc32a1-21': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251cebdc508>,\n",
       " 'Slc32a1-3': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2513fef31c8>,\n",
       " 'Slc32a1-4': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251cebdce08>,\n",
       " 'Slc32a1-5': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251cebdc588>,\n",
       " 'Slc32a1-6': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251cf07c088>,\n",
       " 'Slc32a1-7': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251cf07c308>,\n",
       " 'Slc32a1-8': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251cf07c548>,\n",
       " 'Slc32a1-9': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251cf07c788>,\n",
       " 'SST-1': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251cf07c9c8>,\n",
       " 'SST-10': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251cf07cc48>,\n",
       " 'SST-11': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251cf07ce08>,\n",
       " 'SST-12': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251bd72a248>,\n",
       " 'SST-13': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251bd72a188>,\n",
       " 'SST-14': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251bd72a788>,\n",
       " 'SST-15': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251bd72acc8>,\n",
       " 'SST-16': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251bd72a4c8>,\n",
       " 'SST-17': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251bd72ac08>,\n",
       " 'SST-18': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251bd72a688>,\n",
       " 'SST-19': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250b9eae2c8>,\n",
       " 'SST-2': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250b9eae3c8>,\n",
       " 'SST-20': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250b9eae688>,\n",
       " 'SST-21': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250b9eae948>,\n",
       " 'SST-22': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250b9eaed48>,\n",
       " 'SST-23': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250b9eae888>,\n",
       " 'SST-24': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251cebdcb48>,\n",
       " 'SST-25': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2516d5c0888>,\n",
       " 'SST-26': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251cf9752c8>,\n",
       " 'SST-27': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251cf975608>,\n",
       " 'SST-28': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251cf975248>,\n",
       " 'SST-29': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251cf975b08>,\n",
       " 'SST-3': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251cf975e48>,\n",
       " 'SST-30': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251cf975348>,\n",
       " 'SST-31': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251cf975d48>,\n",
       " 'SST-32': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251cf977208>,\n",
       " 'SST-33': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251cf977548>,\n",
       " 'SST-34': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x250b9eaea88>,\n",
       " 'SST-35': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251cf977a08>,\n",
       " 'SST-36': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251cf977d08>,\n",
       " 'SST-37': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251cf977f48>,\n",
       " 'SST-38': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251cf977c88>,\n",
       " 'SST-39': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251c6ed4208>,\n",
       " 'SST-4': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251c6ed4488>,\n",
       " 'SST-40': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251c6ed4788>,\n",
       " 'SST-41': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251c6ed49c8>,\n",
       " 'SST-42': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251c6ed4c48>,\n",
       " 'SST-43': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251c6ed4ec8>,\n",
       " 'SST-44': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251cf977508>,\n",
       " 'SST-45': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2513362f308>,\n",
       " 'SST-46': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2513362f348>,\n",
       " 'SST-47': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2513362f088>,\n",
       " 'SST-48': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x237826c19c8>,\n",
       " 'SST-49': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2513362ff08>,\n",
       " 'SST-5': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2513362fb88>,\n",
       " 'SST-50': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2513362ff48>,\n",
       " 'SST-51': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251d0194188>,\n",
       " 'SST-52': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251d0194388>,\n",
       " 'SST-53': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251d0194648>,\n",
       " 'SST-54': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251d01948c8>,\n",
       " 'SST-55': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251d0194b48>,\n",
       " 'SST-56': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251d0194d08>,\n",
       " 'SST-57': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251d0194f48>,\n",
       " 'SST-58': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251d061ee08>,\n",
       " 'SST-59': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251d061e188>,\n",
       " 'SST-6': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251d061e448>,\n",
       " 'SST-60': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251d061e688>,\n",
       " 'SST-7': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251d061e408>,\n",
       " 'SST-8': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251d061ed48>,\n",
       " 'SST-9': <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x251d061e908>}"
      ]
     },
     "execution_count": 36,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "filtered_pb_dic"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "metadata": {},
   "outputs": [],
   "source": [
    "# split regions based on genes\n",
    "gene_2_region_2_pbs_dict = {}\n",
    "for _reg, _pbs in filtered_pb_dic.items():\n",
    "    \n",
    "    _ind = int(_reg.split('-')[-1])\n",
    "    _gene = _reg[:-len(str(_ind))-1]\n",
    "    if _gene not in gene_2_region_2_pbs_dict:\n",
    "        gene_2_region_2_pbs_dict[_gene] = {}\n",
    "    gene_2_region_2_pbs_dict[_gene][_ind] = _pbs\n",
    "max_gene_length = max([len(_dict) for _dict in gene_2_region_2_pbs_dict.values()])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 62,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of used readouts: 321\n"
     ]
    }
   ],
   "source": [
    "# generate region_2_readout_dict\n",
    "region_2_readout_dict = {}\n",
    "num_readout_per_probe = 2\n",
    "\n",
    "total_readout_usage = max_gene_length\n",
    "\n",
    "for _gene, _region_2_pbs in gene_2_region_2_pbs_dict.items():\n",
    "    for _i, _ind in enumerate(sorted(_region_2_pbs)):\n",
    "        # for reach region, generate readout usage\n",
    "        #print(_gene, _ind, total_readout_usage)\n",
    "        _readout_usage = [f'u{_i}']\n",
    "        for _j in range(num_readout_per_probe-1):\n",
    "            _readout_usage += [f'u{total_readout_usage}']\n",
    "            total_readout_usage += 1\n",
    "        # append\n",
    "        region_2_readout_dict[f\"{_gene}-{_ind}\"] = _readout_usage\n",
    "\n",
    "print(f\"Number of used readouts: {total_readout_usage}\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 3.2 loading components"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### primers\n",
    "pick primer set 2, 9 for 5kb (consistent with CTP-06)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "120\n"
     ]
    }
   ],
   "source": [
    "num_readout_per_probe = 2\n",
    "total_readout_usage = 0\n",
    "region_2_readout_dict = {}\n",
    "for _i, (_k,_v) in enumerate(sorted(filtered_pb_dic.items())):\n",
    "    region_2_readout_dict[_k] = []\n",
    "    for _j in range(num_readout_per_probe):\n",
    "        region_2_readout_dict[_k].append(f\"u{int(_i+_j*len(filtered_pb_dic))}\")\n",
    "        total_readout_usage += 1\n",
    "print(total_readout_usage)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Picked primer: ID: W1A03_primer_2\n",
      "Name: W1A03_primer_2\n",
      "Description: W1A03_primer_2\n",
      "Number of features: 0\n",
      "Seq('CCCGCAATGGCTGACAACCG')\n",
      "- Picked primer: ID: W1A10_primer_9\n",
      "Name: W1A10_primer_9\n",
      "Description: W1A10_primer_9\n",
      "Number of features: 0\n",
      "Seq('TAATACGACTCACTATAGGGATTGCCGCATGGTTTCCG')\n"
     ]
    }
   ],
   "source": [
    "primer_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Primers'\n",
    "primers = library_tools.assemble.load_primers([2,9], _primer_folder=primer_folder)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 55,
   "metadata": {},
   "outputs": [],
   "source": [
    "reload(library_tools.assemble)\n",
    "readout_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Readouts'\n",
    "readout_start = 0\n",
    "num_color = 2\n",
    "readouts = library_tools.assemble.load_readouts(total_readout_usage, \n",
    "                                               _type='designed_readouts', \n",
    "                                               _readout_folder=readout_folder, \n",
    "                                               _num_colors=2, # 750, 647\n",
    "                                               )\n",
    "\n",
    "readout_dict = {'u': readouts}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 57,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[SeqRecord(seq=Seq('ACAAATCCGACCAGATCGGACGATCATGGG'), id='Stv_3', name='Stv_3', description='Stv_3 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AGACGACGCACGTTCGTACCGCGTACTTCG'), id='Stv_32', name='Stv_32', description='Stv_32 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CAAGTATGCAGCGCGATTGACCGTCTCGTT'), id='Stv_4', name='Stv_4', description='Stv_4 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TTTGCTCGCAAGTGCGCACGAGTTGAACTG'), id='Stv_33', name='Stv_33', description='Stv_33 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AAGTCGTACGCCGATGCGCAGCAATTCACT'), id='Stv_5', name='Stv_5', description='Stv_5 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AGTTTCCATACGCGGCGTGAATATCGCGGC'), id='Stv_35', name='Stv_35', description='Stv_35 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CGAAACATCGGCCACGGTCCCGTTGAACTT'), id='Stv_6', name='Stv_6', description='Stv_6 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GCTACAGCGTTACGGGCCCAGACGTTTCAT'), id='Stv_36', name='Stv_36', description='Stv_36 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACGAATCCACCGTCCAGCGCGTCAAACAGA'), id='Stv_7', name='Stv_7', description='Stv_7 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TTTATCGTGCGTGTCCGCCAGTACCGTGAG'), id='Stv_37', name='Stv_37', description='Stv_37 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CGCGAAATCCCCGTAACGAGCGTCCCTTGC'), id='Stv_8', name='Stv_8', description='Stv_8 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CGTGCGGGAACCGTATCCCTGGCGCGGACT'), id='Stv_39', name='Stv_39', description='Stv_39 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GCATGAGTTGCCTGGCGTTGCGACGACTAA'), id='Stv_9', name='Stv_9', description='Stv_9 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AATCCGGCGTACGTAAGGCAGCTTGCGTTA'), id='Stv_40', name='Stv_40', description='Stv_40 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CCGTCGTCTCCGGTCCACCGTTGCGCTTAC'), id='Stv_10', name='Stv_10', description='Stv_10 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GCGGGATTCGTACGAGCCCTCTTGGACGGG'), id='Stv_42', name='Stv_42', description='Stv_42 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GGCCAATGGCCCAGGTCCGTCACGCAATTT'), id='Stv_11', name='Stv_11', description='Stv_11 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CTTTGTTCGGCTCGGAGCGTTACTGCGGGC'), id='Stv_44', name='Stv_44', description='Stv_44 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TTGATCGAATCGGAGCGTAGCGGAATCTGC'), id='Stv_12', name='Stv_12', description='Stv_12 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TAATTCCGGTATCGGGCCCTTTTGTCTGAC'), id='Stv_45', name='Stv_45', description='Stv_45 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CGCGCGGATCCGCTTGTCGGGAACGGATAC'), id='Stv_13', name='Stv_13', description='Stv_13 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CGTGAGAATGCGTCGCGATACTGGTGTAAG'), id='Stv_46', name='Stv_46', description='Stv_46 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GCCTCGATTACGACGGATGTAATTCGGCCG'), id='Stv_14', name='Stv_14', description='Stv_14 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TCTCATGGTCGCCGGACCGTGATGCCGTGT'), id='Stv_48', name='Stv_48', description='Stv_48 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GCGGGAAGCACGTGGATTAGGGCATCGACC'), id='Stv_16', name='Stv_16', description='Stv_16 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GCCATGGTACGACGGCAAGAGAGCGTGCGT'), id='Stv_50', name='Stv_50', description='Stv_50 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TTGTGTATCCCGCCGGATCTTTGAATCAAC'), id='Stv_19', name='Stv_19', description='Stv_19 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACACCCGATACGTCGTGGGACGTGGACCGT'), id='Stv_53', name='Stv_53', description='Stv_53 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TGCTAGTTCGATCGCGCAATTGCGTAGTTC'), id='Stv_20', name='Stv_20', description='Stv_20 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CGACGATGATCGCAATCGGAGAACTACACC'), id='Stv_54', name='Stv_54', description='Stv_54 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GGACACGGGATCGACGGTAGTGGATCGATT'), id='Stv_21', name='Stv_21', description='Stv_21 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GGGCGACGGGACTCTATCGGCATGGAGGTA'), id='Stv_59', name='Stv_59', description='Stv_59 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CGACCTTAGGTTCCGCGTCATGTACCGGTT'), id='Stv_22', name='Stv_22', description='Stv_22 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AGTCTGGGTCGTCCGTAGGCAAAGGGTCCG'), id='Stv_60', name='Stv_60', description='Stv_60 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TGTTTACGACGTGGGCTGCTGCGATTTCAC'), id='Stv_23', name='Stv_23', description='Stv_23 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GCCATGCGTCGACAATGGTGCGAAAGACCC'), id='Stv_61', name='Stv_61', description='Stv_61 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GATGGTCGACTGGCGGTCTTAATATGCCCA'), id='Stv_25', name='Stv_25', description='Stv_25 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TTGAGCGTTGCGCGAGGAATTGGCGCATCG'), id='Stv_63', name='Stv_63', description='Stv_63 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ATCCATATGACCGGCGGCCTTTTCTCGGAC'), id='Stv_26', name='Stv_26', description='Stv_26 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GCGCGGTCCCAAGATGCCGTGAGCCTTTCA'), id='Stv_64', name='Stv_64', description='Stv_64 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ATGATCCACGACCGAGCAGGTTAGTTGACG'), id='Stv_27', name='Stv_27', description='Stv_27 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACCGAATCGGCTTCCAATCCCTAAGGCCAC'), id='Stv_65', name='Stv_65', description='Stv_65 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TAGACGCACCACGCTCGACAAGAACAGGAC'), id='Stv_28', name='Stv_28', description='Stv_28 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CGGTGCCAAGCTTCACCGCGTTTGTTCAAC'), id='Stv_86', name='Stv_86', description='Stv_86 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACTCGTTTAACGGCCGGTTGGTCTACGGAT'), id='Stv_29', name='Stv_29', description='Stv_29 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TGCCGATTGCCCAGAGTCGGACATCTCCCA'), id='Stv_87', name='Stv_87', description='Stv_87 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GGGTACTCGCGTCTGGAACGACGGATTCAG'), id='Stv_30', name='Stv_30', description='Stv_30 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TGGCCGACGCGACACGTCCCTCAATCGAAC'), id='Stv_88', name='Stv_88', description='Stv_88 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AACATCGGATCGGTGCGGTGGGATGGATAA'), id='Stv_31', name='Stv_31', description='Stv_31 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GGGCGGGAGTTGGACTTCCGGTCGATATCC'), id='Stv_90', name='Stv_90', description='Stv_90 old_barcode', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACTAACCGATGTTGACGGACTAATCCTGAC'), id='NDB_1', name='NDB_1', description='NDB_1', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GCATATAGTAGTTCAGACGCCGTTAAGCGC'), id='NDB_2', name='NDB_2', description='NDB_2', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ATCTTGGTATGGCACGCCTAATCTGGACAC'), id='NDB_4', name='NDB_4', description='NDB_4', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AGCTTGCTATTACGAGCGCTTGGATCCCGT'), id='NDB_5', name='NDB_5', description='NDB_5', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GCTGTTCGGTTTCAAGGATGATCCGCGCTT'), id='NDB_7', name='NDB_7', description='NDB_7', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TATATAGAGACTGCCCGACACATCTTAGTG'), id='NDB_8', name='NDB_8', description='NDB_8', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CAAGATGCACCGTTCATATGTTATCGGGAC'), id='NDB_10', name='NDB_10', description='NDB_10', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TAGCTAGAGACTTACACCCGTAGACGTCGG'), id='NDB_11', name='NDB_11', description='NDB_11', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACACTTCGGGCGGTTCAACTTACTGGTAGA'), id='NDB_13', name='NDB_13', description='NDB_13', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CCTGTCTAAGTGGCGCCGATTGTTTGTCCA'), id='NDB_14', name='NDB_14', description='NDB_14', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GGAGATTTAGCTCTATTAGCCGCGATGCGC'), id='NDB_16', name='NDB_16', description='NDB_16', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GCCCTGTTGCGATGTTCTACCGGATACAAG'), id='NDB_17', name='NDB_17', description='NDB_17', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACTGTGGATCCAAATCGCGGTTCCTTATCA'), id='NDB_19', name='NDB_19', description='NDB_19', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AGATACGTTCGCACCGTGTCCATCGTATTT'), id='NDB_20', name='NDB_20', description='NDB_20', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GCCCTTGGGATGGCGTCGTCTTAGCGTTAT'), id='NDB_22', name='NDB_22', description='NDB_22', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CTAAAGCTAACTGGAACGTCCCGGAATTGC'), id='NDB_23', name='NDB_23', description='NDB_23', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CAGGATCTCACGTTATATTCGTCTCAGTAC'), id='NDB_25', name='NDB_25', description='NDB_25', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACCGAAGTACGAGCACTAGAGGAGTTCGCG'), id='NDB_26', name='NDB_26', description='NDB_26', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACGTTGAGCGACAATCGACTTTCGTGGATC'), id='NDB_28', name='NDB_28', description='NDB_28', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AAGGTGATTCGTGCGCGTCCATGCCTAAGT'), id='NDB_29', name='NDB_29', description='NDB_29', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CCTGTTTGCGTCGGGACAACTATCTACAAT'), id='NDB_31', name='NDB_31', description='NDB_31', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CTCTATAGATCATACGATCCGGACCTGGAG'), id='NDB_32', name='NDB_32', description='NDB_32', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TACTAATCGTTCAGGAATGCCGTTCTCGAC'), id='NDB_34', name='NDB_34', description='NDB_34', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CGATATCATACGTAATGGAGGCTCGCCTTG'), id='NDB_35', name='NDB_35', description='NDB_35', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GACGATAGCCGTTATCGATTAATCGTCCTG'), id='NDB_37', name='NDB_37', description='NDB_37', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TATCACGGGTATGCTTGCCTACTAACTGTT'), id='NDB_38', name='NDB_38', description='NDB_38', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TTCTTGACGATGACAACGCGCAGTTCTTAC'), id='NDB_40', name='NDB_40', description='NDB_40', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CTCATGTCTAAAGTATGCGTCGCGGCATGA'), id='NDB_41', name='NDB_41', description='NDB_41', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ATCATCGAATTGATCCGAAGAGCGTGCACA'), id='NDB_43', name='NDB_43', description='NDB_43', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TTCCATGGATAAACCGGCTAGGTCGCAGAG'), id='NDB_44', name='NDB_44', description='NDB_44', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ATTGTGGGATCGTTTCACCCTTTCGAGTCG'), id='NDB_46', name='NDB_46', description='NDB_46', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TTGATCTGATACTCGACTCGCCATACTTGG'), id='NDB_47', name='NDB_47', description='NDB_47', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TCCGAATACGGCAGTTGTGGCTCCCGTTTA'), id='NDB_49', name='NDB_49', description='NDB_49', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CGTCAAGACGTACAGTAATATTCGCTTCGT'), id='NDB_50', name='NDB_50', description='NDB_50', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CCACAGTTAGCTCGTGCGTTATCGGTTCTT'), id='NDB_52', name='NDB_52', description='NDB_52', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACACTGACATGATTGCCGTTCCGCAAGTTT'), id='NDB_53', name='NDB_53', description='NDB_53', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TTCCTTCTCTAGTCGGCGTGATACATAGAG'), id='NDB_55', name='NDB_55', description='NDB_55', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CACCAGTTAAGGATCGCCGAGCTAGTACAG'), id='NDB_56', name='NDB_56', description='NDB_56', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACCGTAGTGTAAAGGCACGCTCCGATGTGA'), id='NDB_58', name='NDB_58', description='NDB_58', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CAACTATCTTGTTAGCCGTTGTAGCGCCTT'), id='NDB_59', name='NDB_59', description='NDB_59', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TCAAAGAGAGGGTTTACCGGCCTACTCCGA'), id='NDB_61', name='NDB_61', description='NDB_61', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACATTCAAAGTCGGATGCGCCGTCTTAGTG'), id='NDB_62', name='NDB_62', description='NDB_62', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AGGTTCCTTAATCTGACCGCACGGAATATG'), id='NDB_64', name='NDB_64', description='NDB_64', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GGCCAGGTATGTCGCGTCCGTTACAGTAAG'), id='NDB_65', name='NDB_65', description='NDB_65', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GGCGAACGCGGGTTAGACCACCGTCTTAAT'), id='NDB_67', name='NDB_67', description='NDB_67', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACGTATCATCGGTAGACAGCTGCGAATGTC'), id='NDB_68', name='NDB_68', description='NDB_68', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GCATATTAGTAAGCTACTTGCGGACTCGCA'), id='NDB_70', name='NDB_70', description='NDB_70', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CGTATGGGACTATCGCTCACCGGTGCTAAT'), id='NDB_71', name='NDB_71', description='NDB_71', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GAACTACGTGCCGGCTCTCGCTTATTTGAA'), id='NDB_73', name='NDB_73', description='NDB_73', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ATCTATTGCGCTGGTCATCGTCCATACTAA'), id='NDB_74', name='NDB_74', description='NDB_74', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ATGGTAATTTCCCTACGAACCTTGCGTTAC'), id='NDB_76', name='NDB_76', description='NDB_76', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CGGTTGTATCGGCGGCCCGTCATTGCTATT'), id='NDB_77', name='NDB_77', description='NDB_77', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CTCTAGAGTCGAGCTAGCATAGGAGTTCAC'), id='NDB_79', name='NDB_79', description='NDB_79', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GGTCTGATTTCGCTACACGAGCAGGTCTAC'), id='NDB_80', name='NDB_80', description='NDB_80', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TCCGTTATTGCGCTTTCGCAGTATTGTTTC'), id='NDB_82', name='NDB_82', description='NDB_82', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TCCCTATGCGCGAGTTATTTACGCTCGTAT'), id='NDB_83', name='NDB_83', description='NDB_83', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TTCCATTGCATTTACGGACGGCTACAGTCT'), id='NDB_85', name='NDB_85', description='NDB_85', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CATTAGGAGCGATGACCGTAACCTATCTGA'), id='NDB_86', name='NDB_86', description='NDB_86', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACGCAGGTCGGTTCATAACGTCCTTAGCTA'), id='NDB_88', name='NDB_88', description='NDB_88', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACTCTGTATATGTTCGCGCACGGTGTAGAC'), id='NDB_89', name='NDB_89', description='NDB_89', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GAGTAATAGCGTACACTTTGCGATCGGCCG'), id='NDB_91', name='NDB_91', description='NDB_91', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GTGTAAACAAGTACACGCCGTCATTCTCAT'), id='NDB_92', name='NDB_92', description='NDB_92', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GAGTAGTACCACATTCGGCATTGCCTGTTA'), id='NDB_94', name='NDB_94', description='NDB_94', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACTATATGAGGGTGCTCTACCGGGTTCACA'), id='NDB_95', name='NDB_95', description='NDB_95', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AAAGTTTGTTCGTTCCCGTCGCTGAACTTC'), id='NDB_97', name='NDB_97', description='NDB_97', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GCTCAAGCTAAGTGCGCCGTACGGATAAGT'), id='NDB_98', name='NDB_98', description='NDB_98', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CCCTAGGCGTATATTGAGTTAGACACGTCC'), id='NDB_100', name='NDB_100', description='NDB_100', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TGCCTAATAGGTCGGTCCGCGGGTCATTAT'), id='NDB_101', name='NDB_101', description='NDB_101', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AGTCAAAGATGTTCGACCTACGCTCGTAGA'), id='NDB_103', name='NDB_103', description='NDB_103', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CACGATACGGGTATAGTCGCGTCGAGTATC'), id='NDB_104', name='NDB_104', description='NDB_104', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACTCAAGGATCGGGCCGTATGTCTAGTTGC'), id='NDB_106', name='NDB_106', description='NDB_106', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CCCATGGACAACTGTCGGGTAGGTATCCTT'), id='NDB_107', name='NDB_107', description='NDB_107', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AGGCTCTATGACCGTTTACGTTCTTATCGA'), id='NDB_109', name='NDB_109', description='NDB_109', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CGTGAATATATCGTCGGGCTCGTCCATCTT'), id='NDB_110', name='NDB_110', description='NDB_110', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GGTTAGCGGAATCCTCCACCGAGATCGATG'), id='NDB_112', name='NDB_112', description='NDB_112', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TCACTAGGTCGAAGTGTCGCGTAAACTGGC'), id='NDB_113', name='NDB_113', description='NDB_113', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TATGACCGACTATCGTAATGCCTTACGCGG'), id='NDB_115', name='NDB_115', description='NDB_115', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GTTCTCATCGTAGGTGCATATTCGGTGCCG'), id='NDB_116', name='NDB_116', description='NDB_116', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TTAGATAATGCTTTACCGCCGGACTCGTTG'), id='NDB_118', name='NDB_118', description='NDB_118', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CCTCAAACGTTCGGGCGTGGCATCTTATAG'), id='NDB_119', name='NDB_119', description='NDB_119', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CCGAACGGTTATGCGGTCTTTCATTGGTAC'), id='NDB_121', name='NDB_121', description='NDB_121', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GCGCTGTCTTGCGTGCGTAACAACTAAATT'), id='NDB_122', name='NDB_122', description='NDB_122', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TCTCTATGTAGGCGAGCGGAATCTTGCACC'), id='NDB_124', name='NDB_124', description='NDB_124', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TCCAAGCTTGATCGGCACGTATCTCGATGG'), id='NDB_125', name='NDB_125', description='NDB_125', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CGGGTTCGGATAATCGGTTCCTTCACGATA'), id='NDB_127', name='NDB_127', description='NDB_127', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TTGATGTCCCGGCAACCGTTTACGCTAAAG'), id='NDB_128', name='NDB_128', description='NDB_128', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TAACTTGCGTAGAGCCCTTTAGCGTTACAA'), id='NDB_130', name='NDB_130', description='NDB_130', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CTGGTCATGTCTAGGTCGTACTGACCATAA'), id='NDB_131', name='NDB_131', description='NDB_131', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TCACAGCGTTGGCACGTGTTAGTATACAGC'), id='NDB_133', name='NDB_133', description='NDB_133', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CATGTAGACGCTTGTAGTCGGTCACACAAT'), id='NDB_134', name='NDB_134', description='NDB_134', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TGCTTAATTTGCTATCGCCTCTTTGCGGTA'), id='NDB_136', name='NDB_136', description='NDB_136', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AAGCAAGGGATGGTACACCTAGTTCCCTAA'), id='NDB_137', name='NDB_137', description='NDB_137', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TTCTACCACGAATTGTCGCATGACGACGGT'), id='NDB_139', name='NDB_139', description='NDB_139', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CGCTAGTTGTTTACGAATACCGTCCGATGA'), id='NDB_140', name='NDB_140', description='NDB_140', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TACGTGGTACGCATCGTCTACCGGTTAAGA'), id='NDB_142', name='NDB_142', description='NDB_142', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CTAGTATAGCACTATCCATCGTGGTCTGCT'), id='NDB_143', name='NDB_143', description='NDB_143', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GGTGTGGCAGCATTCGAGACGCCTACTTGA'), id='NDB_145', name='NDB_145', description='NDB_145', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CCTTAGGTATGGTCGGAGTCACCGTGACTT'), id='NDB_146', name='NDB_146', description='NDB_146', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TCAATAACTACCGATACGCAAGGGCGAAGG'), id='NDB_148', name='NDB_148', description='NDB_148', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GTGCTATCTGTTGTCCGTTGCGGTTCCTCT'), id='NDB_149', name='NDB_149', description='NDB_149', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CCCTAAGCGGTAATGTCTTATTCGCCGTGG'), id='NDB_151', name='NDB_151', description='NDB_151', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACCGTCACGTTGGTTCGCGTAGTAGGAAAC'), id='NDB_152', name='NDB_152', description='NDB_152', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CCATTGCAAGCACGGGTCGTTAAGTATAGC'), id='NDB_154', name='NDB_154', description='NDB_154', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CACAAGACGTGGTGGACGTAGTTACGGACC'), id='NDB_155', name='NDB_155', description='NDB_155', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AAACTACGTGTTTCTACATCGCACGTCAGT'), id='NDB_157', name='NDB_157', description='NDB_157', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CAGCTGGATACGAGACTGGCGCGAAAGCAA'), id='NDB_158', name='NDB_158', description='NDB_158', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACGTACATCAGATATGGCACGCAAGGGTTC'), id='NDB_160', name='NDB_160', description='NDB_160', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AGTAACGATTGGCGTATCGTACTCCAAGCA'), id='NDB_161', name='NDB_161', description='NDB_161', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CTTCTGCGAGTTATTCATCGATAGCACGTG'), id='NDB_163', name='NDB_163', description='NDB_163', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TTCATCATATGGCAGTCGGCTCAACTGACG'), id='NDB_164', name='NDB_164', description='NDB_164', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TGACATTTGGCGCATCAAGCTCGTCGAACA'), id='NDB_166', name='NDB_166', description='NDB_166', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TCACATGTCGTTAGCTTAGAGGACGTCGAC'), id='NDB_167', name='NDB_167', description='NDB_167', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GGCATGTGAGTATAAACGCGCCGACTCTTA'), id='NDB_169', name='NDB_169', description='NDB_169', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TGAGAGAGCAACTATCGCGATACTAGCCTG'), id='NDB_170', name='NDB_170', description='NDB_170', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CGCATTAGTCAAGCACCGCGCGGTTTGTTA'), id='NDB_172', name='NDB_172', description='NDB_172', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACCCTGTTTCATGTTCGCGGCAAGGCTAAG'), id='NDB_173', name='NDB_173', description='NDB_173', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CGCTTATGAATCGATGGACTGCCTACCGGA'), id='NDB_175', name='NDB_175', description='NDB_175', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TTTAAGTCAATCCTCGGATAGACGCCGCTG'), id='NDB_176', name='NDB_176', description='NDB_176', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CACGACACGGTCAAATTGCGGGCTATAGGG'), id='NDB_178', name='NDB_178', description='NDB_178', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CGTTAATCGTTATGCCCGATCAAGCGCTGT'), id='NDB_179', name='NDB_179', description='NDB_179', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TAGATACCCTGAATCGTGTTGTATCGCACT'), id='NDB_181', name='NDB_181', description='NDB_181', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GTCGTACAAACCGGCAGCGGGATTCTATCT'), id='NDB_182', name='NDB_182', description='NDB_182', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TTCGTCAGGTTATGGACAACGTATACCGCG'), id='NDB_184', name='NDB_184', description='NDB_184', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CCACTTAGAGCACGGTTGTACGATTGACAC'), id='NDB_185', name='NDB_185', description='NDB_185', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ATGGTTAGTTGAAGCCGTCAACCCGCTTTG'), id='NDB_187', name='NDB_187', description='NDB_187', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ATGAAGTTAGCTGACGGTCTCGCCTAAGCT'), id='NDB_188', name='NDB_188', description='NDB_188', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CTCGATCGGACGTAGTCTATATGTCGTCGT'), id='NDB_190', name='NDB_190', description='NDB_190', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CTTGTCTGTTTACGGCGCATGATTCCAATT'), id='NDB_191', name='NDB_191', description='NDB_191', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CCAAATATACGGGTGAGCGTCCCTGATACA'), id='NDB_193', name='NDB_193', description='NDB_193', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACTCTTCTTTCGTCTGATGGTTGCGACAAG'), id='NDB_194', name='NDB_194', description='NDB_194', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CATGAATCAGTAGTCCGACATTTCTCGAAG'), id='NDB_196', name='NDB_196', description='NDB_196', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ATCCTGTATCGTGAACGTTCCGTCATGCAA'), id='NDB_197', name='NDB_197', description='NDB_197', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACTGTACGTGACGGACTCTATAGGACCGCA'), id='NDB_199', name='NDB_199', description='NDB_199', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CCTCTATCGTATACGGACGGATCAGGGTAC'), id='NDB_200', name='NDB_200', description='NDB_200', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CATCATGCGGCTTGTACGGCTCTTAATCAA'), id='NDB_202', name='NDB_202', description='NDB_202', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACACAGGAGAGTCGTACGTCATGGTGCAGC'), id='NDB_203', name='NDB_203', description='NDB_203', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CAGCTCGTCGATGTTACTTGTCGGTCGCGA'), id='NDB_205', name='NDB_205', description='NDB_205', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TGTCAACGTAGGATAACGACCCTTGCACGT'), id='NDB_206', name='NDB_206', description='NDB_206', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AAGTTATCTATTACGTCGGGCCCGTGCTAG'), id='NDB_208', name='NDB_208', description='NDB_208', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AATGTGAATAGACGCCCGCTGGCTATAAGC'), id='NDB_209', name='NDB_209', description='NDB_209', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CAGTTCCGTTATGAGCGACTGCTCGTGCAT'), id='NDB_211', name='NDB_211', description='NDB_211', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AACCTCGTACGGCACCTGGATTGTTCATGG'), id='NDB_212', name='NDB_212', description='NDB_212', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AACCTGAGAGACCGGCGTATTAACACACAG'), id='NDB_214', name='NDB_214', description='NDB_214', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GGCAAAGACCCATCGATGCGAATGCTTGAC'), id='NDB_215', name='NDB_215', description='NDB_215', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TCGGAAATCGCGCTAATATGCCAAGAGACG'), id='NDB_217', name='NDB_217', description='NDB_217', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TAGTTGTTTAGATCGCCTCATGTCCGACGC'), id='NDB_218', name='NDB_218', description='NDB_218', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GCTGAACCAGACTGACGTTCGGAGTGCCTA'), id='NDB_220', name='NDB_220', description='NDB_220', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CGCTTGTAGCGTGGGTATAACCCGATTCGG'), id='NDB_221', name='NDB_221', description='NDB_221', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GCCCTGGGTTTACCGTACGATTGGTGCTAT'), id='NDB_223', name='NDB_223', description='NDB_223', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GCCTAAAGTCGGTGAGACGTCCGTCCAAAG'), id='NDB_224', name='NDB_224', description='NDB_224', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TAGCAGCCACACGATTAGACCAACGGATGG'), id='NDB_226', name='NDB_226', description='NDB_226', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TTGGTCTTCGTACGACCTCTATCATCAAGG'), id='NDB_227', name='NDB_227', description='NDB_227', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TCCAACGAAGGGTTGCCCGGATGTTACTTA'), id='NDB_229', name='NDB_229', description='NDB_229', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TCAGTTAGAGTCGTACTTATCTCGCTTCGT'), id='NDB_230', name='NDB_230', description='NDB_230', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GGATAATCGATCTACATACGTTACCCTGTC'), id='NDB_232', name='NDB_232', description='NDB_232', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TCCTATGTAATAACGGCTTCGAGACTGGAC'), id='NDB_233', name='NDB_233', description='NDB_233', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GGCGTAACTTTAAAGCGGCGTGTCCTCTCT'), id='NDB_235', name='NDB_235', description='NDB_235', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ATCCATATTGTCTATCGACTTGGGTCGACC'), id='NDB_236', name='NDB_236', description='NDB_236', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CGCATGGATTTCTCGCAGTCATCTAGTGAT'), id='NDB_238', name='NDB_238', description='NDB_238', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AGGCAACGCTGTATTCAACGTCTGAGAACC'), id='NDB_239', name='NDB_239', description='NDB_239', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TGGATCTGCGGCCTCTATTGCATGTCGATG'), id='NDB_241', name='NDB_241', description='NDB_241', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ATGATCACTACTTCGAGACTTCATGCGTTC'), id='NDB_242', name='NDB_242', description='NDB_242', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CCTGTAGGACGTTGATCGATTCTCCTGTTA'), id='NDB_244', name='NDB_244', description='NDB_244', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GCCGTCGAACGTTGGCTATAATAAGATTCC'), id='NDB_245', name='NDB_245', description='NDB_245', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AAAGTTGCACGCTATAACCGGCTTCAATCA'), id='NDB_247', name='NDB_247', description='NDB_247', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TACCTGTGTCTCGGTAGAGGACCGCCTTAG'), id='NDB_248', name='NDB_248', description='NDB_248', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACCGTATTTGCAGTCGCGTATCCGTTTAAA'), id='NDB_250', name='NDB_250', description='NDB_250', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TCGCAACTTTCTGATTCGCCTAGCGGAGAG'), id='NDB_251', name='NDB_251', description='NDB_251', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CTGCATACGTAAGTCGGGATATCGCCATTA'), id='NDB_253', name='NDB_253', description='NDB_253', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TACCTACAAAGCGGATCTTAGTACTGGTCA'), id='NDB_254', name='NDB_254', description='NDB_254', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TACTTAGCGGAGTAAATCGCCATTGACTTC'), id='NDB_256', name='NDB_256', description='NDB_256', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CCGCTATTATAGCGTGGGCGGCTTAAGTCG'), id='NDB_257', name='NDB_257', description='NDB_257', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GTCCAGTTATTATCGGTGTCCCGACTAGTT'), id='NDB_259', name='NDB_259', description='NDB_259', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CCCGTAAGAATAAGATTGCGTGCGCCTTAA'), id='NDB_260', name='NDB_260', description='NDB_260', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GGTCAGTCTTTGGTCGTAACGAATCTCCGT'), id='NDB_262', name='NDB_262', description='NDB_262', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AGCCTAGGTATCATGCGGGCCGAATCTTGG'), id='NDB_263', name='NDB_263', description='NDB_263', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CCCTATTTCCTTGTTCGGATCGGTCGAGAA'), id='NDB_265', name='NDB_265', description='NDB_265', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GCGCAGCTGTCAGTAGCGCCAGTACAATTT'), id='NDB_266', name='NDB_266', description='NDB_266', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CACTTGTGCTACCATCGAGCGGAGGTATTT'), id='NDB_268', name='NDB_268', description='NDB_268', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GCAGAGATATATCGGGTTGCACCGACCACT'), id='NDB_269', name='NDB_269', description='NDB_269', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACCGTGCTGAGGCGTTCCACTTGGATCGTT'), id='NDB_271', name='NDB_271', description='NDB_271', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GTGCATACATGTTCGTTCTGCGTTACTCAC'), id='NDB_272', name='NDB_272', description='NDB_272', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACACTGATACTGTGCGGTTACTAGTCGACA'), id='NDB_274', name='NDB_274', description='NDB_274', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CCTTTATGAAGTTCGACGGGTCTATCGGAC'), id='NDB_275', name='NDB_275', description='NDB_275', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TATCTGAAGGTAGACTCGTACTCTCCGCTG'), id='NDB_277', name='NDB_277', description='NDB_277', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GGAGAATAGGACCGCTAACACCGCAAATGC'), id='NDB_278', name='NDB_278', description='NDB_278', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AACGTCAGTAAGTCGCTCCGGGTGGACTTG'), id='NDB_280', name='NDB_280', description='NDB_280', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GCCGTCGATTAGAAGAGTCTCCGTTCACGT'), id='NDB_281', name='NDB_281', description='NDB_281', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TCACTTCAGCTTCGATCAGCGGCGTATGTA'), id='NDB_283', name='NDB_283', description='NDB_283', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CGTATACACCGGATTCGAGTAGTCTGCTGA'), id='NDB_284', name='NDB_284', description='NDB_284', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GAATAGGACATACTTGGCTCGCCTCTTAAG'), id='NDB_286', name='NDB_286', description='NDB_286', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACCGAGTCATGCGCGTGTGGCACATATTGA'), id='NDB_287', name='NDB_287', description='NDB_287', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACATTTCACGGATAGCGGTCTTGCTACGTA'), id='NDB_289', name='NDB_289', description='NDB_289', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TCGGAGTGCTATTGCCTCATCCGCGGAAGT'), id='NDB_290', name='NDB_290', description='NDB_290', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TGCTAGATGTAACGTCGTCGCTATAGCCGG'), id='NDB_292', name='NDB_292', description='NDB_292', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GTCCATTGTGGGTCGGTAGCACCACGTATG'), id='NDB_293', name='NDB_293', description='NDB_293', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TTTCTGGTTCACAAGCCGGAGTCCGCTAAA'), id='NDB_295', name='NDB_295', description='NDB_295', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GACCAGAAAGGCTAGCGCGTCTGACGTTTA'), id='NDB_296', name='NDB_296', description='NDB_296', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GATCAATGCATTGGCCGGCTTGAATCGATC'), id='NDB_298', name='NDB_298', description='NDB_298', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CCTTTGGTTCCGATTATTCAAACGGAGGAC'), id='NDB_299', name='NDB_299', description='NDB_299', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GTACTTCGACATTTGCGTATTGGTGCCTGC'), id='NDB_301', name='NDB_301', description='NDB_301', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CGGGAATAGTCACAGGCCGCTAACCTATAT'), id='NDB_302', name='NDB_302', description='NDB_302', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GCCTAGCGTGTTAACGTTGTGTGCCTCGTG'), id='NDB_304', name='NDB_304', description='NDB_304', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CAAATAAGTGGTACGCGACCCTCTATACGG'), id='NDB_305', name='NDB_305', description='NDB_305', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AACTTGTAAATACGCGGCGCCATACTGTCG'), id='NDB_307', name='NDB_307', description='NDB_307', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CGAGAATTATTGGTCGACGCCGCGACTAAA'), id='NDB_308', name='NDB_308', description='NDB_308', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GGGCATCGTTAAGAAGCGTAGGACTGCCAC'), id='NDB_310', name='NDB_310', description='NDB_310', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TTCGAATGACGATCGGTTTCAACCTCCGAG'), id='NDB_311', name='NDB_311', description='NDB_311', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CAAATCTTATCGTAAACGTCAGCCACTGGG'), id='NDB_313', name='NDB_313', description='NDB_313', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AACTAGGTCGTCAGTTTATGCACTTCGCAT'), id='NDB_314', name='NDB_314', description='NDB_314', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TTACTAAGCTGTTGTTCGGGTTTACCCGTC'), id='NDB_316', name='NDB_316', description='NDB_316', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CCACTGTGATGAGTTCCGTCGTATTCTATC'), id='NDB_317', name='NDB_317', description='NDB_317', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GACCTAGAGGGCCGGAGTTAGCCTTTGTAC'), id='NDB_319', name='NDB_319', description='NDB_319', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GCCGAGCTCACAATCTTGATCGATGCGATT'), id='NDB_320', name='NDB_320', description='NDB_320', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TGCCTAGAGCGGTCGGGAATACCTATCGTG'), id='NDB_322', name='NDB_322', description='NDB_322', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GCCTTAGCATGTTACATCGGTGCGCGAAGT'), id='NDB_323', name='NDB_323', description='NDB_323', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CCCAAGACCTTAGGGTTACGAGTTGCGATC'), id='NDB_325', name='NDB_325', description='NDB_325', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TCCTTGTGTGATCGTCTCGCTACAGCGTAT'), id='NDB_326', name='NDB_326', description='NDB_326', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AGCCTGAGTATCGTATCTTAGCGTCGCAGT'), id='NDB_328', name='NDB_328', description='NDB_328', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AGATACGATCATGTTAGTCGCCGTCGGCAT'), id='NDB_329', name='NDB_329', description='NDB_329', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GACCTAGTCGGTATGAGTATTCGTTCCGCC'), id='NDB_331', name='NDB_331', description='NDB_331', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AGCGTTTATACGATGGTCGTCCTCGTTTCG'), id='NDB_332', name='NDB_332', description='NDB_332', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TACTATCAGAAGTCGCGGCCAACCTGATTA'), id='NDB_334', name='NDB_334', description='NDB_334', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACCGATTATTTGGCACGTATCGACCGGTTT'), id='NDB_335', name='NDB_335', description='NDB_335', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TGTTAGTACGATCTTACAGACCGGCACCAA'), id='NDB_337', name='NDB_337', description='NDB_337', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TGGAACTTTGAGTCGCAGCGTCCCGATATC'), id='NDB_338', name='NDB_338', description='NDB_338', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GAGTTTATTTCGCCGCGAACCGGAGAACTC'), id='NDB_340', name='NDB_340', description='NDB_340', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CAGATACCGTTTAATCTCGGCTGAGCGGAG'), id='NDB_341', name='NDB_341', description='NDB_341', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CCACTAGCGTCTGGACTAAGTAGCGGTTGA'), id='NDB_343', name='NDB_343', description='NDB_343', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GAGCAGATTCAATCTATGCGCGTTCACATC'), id='NDB_344', name='NDB_344', description='NDB_344', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GCCATGAACGGTAATCGACCGAGGGTCTAG'), id='NDB_346', name='NDB_346', description='NDB_346', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACAGTTGAGCTAGTACGTCCCAGTAAGGTC'), id='NDB_347', name='NDB_347', description='NDB_347', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACAGAGCTCTTATCCGGCATATCTGTCGTA'), id='NDB_349', name='NDB_349', description='NDB_349', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GCGTTGGACTTAAGTATTACGCCTGACGAC'), id='NDB_350', name='NDB_350', description='NDB_350', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TCGCTCTTGAGTCGATCGCAGACTATGGTA'), id='NDB_352', name='NDB_352', description='NDB_352', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CTGCATAAGGCATAGATCGCGGTCATCTTA'), id='NDB_353', name='NDB_353', description='NDB_353', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AGCCATCGGGTGGAGCCGCAATCTAATACG'), id='NDB_355', name='NDB_355', description='NDB_355', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AGGTTCGTCATGGCATCCGCAAGTCGTTAT'), id='NDB_356', name='NDB_356', description='NDB_356', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CGGAAATCGTACGTATCGCTTCTAGATTCG'), id='NDB_358', name='NDB_358', description='NDB_358', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CGACATGCGCAGATGGTATGGATTGCCCTA'), id='NDB_359', name='NDB_359', description='NDB_359', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GTGGTTTACGTTTGTTCCGTCGCGACCGTC'), id='NDB_361', name='NDB_361', description='NDB_361', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CGTATGAATTATAGACCCGGCACTTCGCAG'), id='NDB_362', name='NDB_362', description='NDB_362', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TACCTTTCCCGACGGCGGTATGATGGAAAT'), id='NDB_364', name='NDB_364', description='NDB_364', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GGCGTCATTGAACAACCGGTCTTGCCAGAG'), id='NDB_365', name='NDB_365', description='NDB_365', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CGCTAGGTCCATGTGATGCGTGCCGAAGAT'), id='NDB_367', name='NDB_367', description='NDB_367', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TTGGACGCATCTGACGCTGAAGTGCCTAGT'), id='NDB_368', name='NDB_368', description='NDB_368', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CCGCTGCATGCCAGCGTAATGGTTTATATG'), id='NDB_370', name='NDB_370', description='NDB_370', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TAACTGAGCTTATCGCTTCGCACGAAATGG'), id='NDB_371', name='NDB_371', description='NDB_371', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CGAAATAGCGGACTTATGCGTACATCACTG'), id='NDB_373', name='NDB_373', description='NDB_373', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AACCTTTAGGCGATGTTCGTGCCAATCTTT'), id='NDB_374', name='NDB_374', description='NDB_374', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CCCTAATCCCGACTTTGTGGGTACGTTGAT'), id='NDB_376', name='NDB_376', description='NDB_376', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AAGCACCGTAGTCGTTGCCGTACTTTATTT'), id='NDB_377', name='NDB_377', description='NDB_377', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ATCCAACAGTAGCGGCCGAGATTGTTCATA'), id='NDB_379', name='NDB_379', description='NDB_379', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GCCGATACGATAATAAGTTGGCCCGCCAGG'), id='NDB_380', name='NDB_380', description='NDB_380', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GCGTTGCAAGATATCTCGGCCCGTGTAATA'), id='NDB_382', name='NDB_382', description='NDB_382', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CGTCTGACCGTTAAGTTGTTTCTCGCCGGG'), id='NDB_383', name='NDB_383', description='NDB_383', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACCCTTTAGGGACTACGCTAGTGTTGTGCC'), id='NDB_385', name='NDB_385', description='NDB_385', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACTATCGTATCAACTCCGAGCCTGATGAGG'), id='NDB_386', name='NDB_386', description='NDB_386', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ATGGAGTGAAAGTCGCCCTGCATAGGTACC'), id='NDB_388', name='NDB_388', description='NDB_388', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CCTTTGAGCTTTGCGGCGATCGGTCGAATA'), id='NDB_389', name='NDB_389', description='NDB_389', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AGGAACGTAGATACCGCGGCGTAAGTACCC'), id='NDB_391', name='NDB_391', description='NDB_391', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CCTTAGGCATAGATAAGCTCGCTGGTCACA'), id='NDB_392', name='NDB_392', description='NDB_392', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GGGAATCGATCAGCATTCCGCTTTAACACA'), id='NDB_394', name='NDB_394', description='NDB_394', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CGCATTGGCTTCGTACGCGCTTTATAGGCG'), id='NDB_395', name='NDB_395', description='NDB_395', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ATACACGAAGGCACGCCGGATTTACTAGAG'), id='NDB_397', name='NDB_397', description='NDB_397', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TCCATACTTCGGCACTAGGATAACTTTAGG'), id='NDB_398', name='NDB_398', description='NDB_398', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AGCCAGCTCGTAGCGCCTGAATTAGCTATA'), id='NDB_400', name='NDB_400', description='NDB_400', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CCCTTTGTAGAACGTAAGCGGCAATACTTT'), id='NDB_401', name='NDB_401', description='NDB_401', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CGCTAATCGGATGCTATACGGTGTCGAGCA'), id='NDB_403', name='NDB_403', description='NDB_403', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CCACTATGGATGACGTGCGCTACAGCTTAA'), id='NDB_404', name='NDB_404', description='NDB_404', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TGCTTGTCTAGAATATGCGATCGTCGACGC'), id='NDB_406', name='NDB_406', description='NDB_406', dbxrefs=[])]"
      ]
     },
     "execution_count": 57,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "readout_dict['u']"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 3.3 assemble probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 199,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Assemble probes by given target sequences, readouts and primers.\n",
      "-- included readout types: ['u']\n",
      "--- assemblying 135 probes in region: Ccnd2-1\n",
      "--- assemblying 136 probes in region: Ccnd2-10\n",
      "--- assemblying 139 probes in region: Ccnd2-11\n",
      "--- assemblying 62 probes in region: Ccnd2-12\n",
      "--- assemblying 127 probes in region: Ccnd2-13\n",
      "--- assemblying 117 probes in region: Ccnd2-14\n",
      "--- assemblying 125 probes in region: Ccnd2-15\n",
      "--- assemblying 136 probes in region: Ccnd2-16\n",
      "--- assemblying 147 probes in region: Ccnd2-17\n",
      "--- assemblying 146 probes in region: Ccnd2-18\n",
      "--- assemblying 143 probes in region: Ccnd2-19\n",
      "--- assemblying 99 probes in region: Ccnd2-2\n",
      "--- assemblying 142 probes in region: Ccnd2-20\n",
      "--- assemblying 139 probes in region: Ccnd2-21\n",
      "--- assemblying 126 probes in region: Ccnd2-22\n",
      "--- assemblying 138 probes in region: Ccnd2-23\n",
      "--- assemblying 136 probes in region: Ccnd2-24\n",
      "--- assemblying 128 probes in region: Ccnd2-25\n",
      "--- assemblying 101 probes in region: Ccnd2-26\n",
      "--- assemblying 139 probes in region: Ccnd2-27\n",
      "--- assemblying 128 probes in region: Ccnd2-28\n",
      "--- assemblying 145 probes in region: Ccnd2-29\n",
      "--- assemblying 104 probes in region: Ccnd2-3\n",
      "--- assemblying 125 probes in region: Ccnd2-30\n",
      "--- assemblying 124 probes in region: Ccnd2-31\n",
      "--- assemblying 135 probes in region: Ccnd2-32\n",
      "--- assemblying 127 probes in region: Ccnd2-33\n",
      "--- assemblying 133 probes in region: Ccnd2-34\n",
      "--- assemblying 149 probes in region: Ccnd2-35\n",
      "--- assemblying 138 probes in region: Ccnd2-36\n",
      "--- assemblying 118 probes in region: Ccnd2-37\n",
      "--- assemblying 145 probes in region: Ccnd2-38\n",
      "--- assemblying 119 probes in region: Ccnd2-39\n",
      "--- assemblying 149 probes in region: Ccnd2-4\n",
      "--- assemblying 135 probes in region: Ccnd2-40\n",
      "--- assemblying 124 probes in region: Ccnd2-41\n",
      "--- assemblying 117 probes in region: Ccnd2-42\n",
      "--- assemblying 126 probes in region: Ccnd2-43\n",
      "--- assemblying 128 probes in region: Ccnd2-44\n",
      "--- assemblying 128 probes in region: Ccnd2-45\n",
      "--- assemblying 130 probes in region: Ccnd2-46\n",
      "--- assemblying 142 probes in region: Ccnd2-47\n",
      "--- assemblying 135 probes in region: Ccnd2-48\n",
      "--- assemblying 128 probes in region: Ccnd2-49\n",
      "--- assemblying 152 probes in region: Ccnd2-5\n",
      "--- assemblying 133 probes in region: Ccnd2-50\n",
      "--- assemblying 99 probes in region: Ccnd2-51\n",
      "--- assemblying 136 probes in region: Ccnd2-52\n",
      "--- assemblying 132 probes in region: Ccnd2-53\n",
      "--- assemblying 34 probes in region: Ccnd2-54\n",
      "--- assemblying 101 probes in region: Ccnd2-55\n",
      "--- assemblying 116 probes in region: Ccnd2-56\n",
      "--- assemblying 134 probes in region: Ccnd2-57\n",
      "--- assemblying 142 probes in region: Ccnd2-58\n",
      "--- assemblying 147 probes in region: Ccnd2-59\n",
      "--- assemblying 112 probes in region: Ccnd2-6\n",
      "--- assemblying 143 probes in region: Ccnd2-60\n",
      "--- assemblying 136 probes in region: Ccnd2-7\n",
      "--- assemblying 140 probes in region: Ccnd2-8\n",
      "--- assemblying 134 probes in region: Ccnd2-9\n",
      "--- assemblying 79 probes in region: Nkx2-1-1\n",
      "--- assemblying 129 probes in region: Nkx2-1-10\n",
      "--- assemblying 128 probes in region: Nkx2-1-11\n",
      "--- assemblying 118 probes in region: Nkx2-1-12\n",
      "--- assemblying 134 probes in region: Nkx2-1-13\n",
      "--- assemblying 129 probes in region: Nkx2-1-14\n",
      "--- assemblying 129 probes in region: Nkx2-1-15\n",
      "--- assemblying 125 probes in region: Nkx2-1-16\n",
      "--- assemblying 138 probes in region: Nkx2-1-17\n",
      "--- assemblying 146 probes in region: Nkx2-1-18\n",
      "--- assemblying 153 probes in region: Nkx2-1-19\n",
      "--- assemblying 62 probes in region: Nkx2-1-2\n",
      "--- assemblying 123 probes in region: Nkx2-1-20\n",
      "--- assemblying 100 probes in region: Nkx2-1-21\n",
      "--- assemblying 53 probes in region: Nkx2-1-22\n",
      "--- assemblying 87 probes in region: Nkx2-1-23\n",
      "--- assemblying 76 probes in region: Nkx2-1-24\n",
      "--- assemblying 111 probes in region: Nkx2-1-25\n",
      "--- assemblying 121 probes in region: Nkx2-1-26\n",
      "--- assemblying 144 probes in region: Nkx2-1-27\n",
      "--- assemblying 116 probes in region: Nkx2-1-28\n",
      "--- assemblying 141 probes in region: Nkx2-1-29\n",
      "--- assemblying 86 probes in region: Nkx2-1-3\n",
      "--- assemblying 125 probes in region: Nkx2-1-30\n",
      "--- assemblying 94 probes in region: Nkx2-1-31\n",
      "--- assemblying 102 probes in region: Nkx2-1-32\n",
      "--- assemblying 135 probes in region: Nkx2-1-33\n",
      "--- assemblying 149 probes in region: Nkx2-1-34\n",
      "--- assemblying 154 probes in region: Nkx2-1-35\n",
      "--- assemblying 135 probes in region: Nkx2-1-36\n",
      "--- assemblying 133 probes in region: Nkx2-1-37\n",
      "--- assemblying 130 probes in region: Nkx2-1-38\n",
      "--- assemblying 125 probes in region: Nkx2-1-39\n",
      "--- assemblying 137 probes in region: Nkx2-1-4\n",
      "--- assemblying 58 probes in region: Nkx2-1-40\n",
      "--- assemblying 101 probes in region: Nkx2-1-41\n",
      "--- assemblying 118 probes in region: Nkx2-1-42\n",
      "--- assemblying 123 probes in region: Nkx2-1-43\n",
      "--- assemblying 112 probes in region: Nkx2-1-44\n",
      "--- assemblying 133 probes in region: Nkx2-1-45\n",
      "--- assemblying 138 probes in region: Nkx2-1-46\n",
      "--- assemblying 130 probes in region: Nkx2-1-47\n",
      "--- assemblying 132 probes in region: Nkx2-1-48\n",
      "--- assemblying 100 probes in region: Nkx2-1-49\n",
      "--- assemblying 120 probes in region: Nkx2-1-5\n",
      "--- assemblying 90 probes in region: Nkx2-1-50\n",
      "--- assemblying 122 probes in region: Nkx2-1-51\n",
      "--- assemblying 139 probes in region: Nkx2-1-52\n",
      "--- assemblying 123 probes in region: Nkx2-1-53\n",
      "--- assemblying 128 probes in region: Nkx2-1-54\n",
      "--- assemblying 138 probes in region: Nkx2-1-55\n",
      "--- assemblying 137 probes in region: Nkx2-1-56\n",
      "--- assemblying 100 probes in region: Nkx2-1-57\n",
      "--- assemblying 122 probes in region: Nkx2-1-58\n",
      "--- assemblying 125 probes in region: Nkx2-1-59\n",
      "--- assemblying 141 probes in region: Nkx2-1-6\n",
      "--- assemblying 86 probes in region: Nkx2-1-60\n",
      "--- assemblying 138 probes in region: Nkx2-1-7\n",
      "--- assemblying 118 probes in region: Nkx2-1-8\n",
      "--- assemblying 126 probes in region: Nkx2-1-9\n",
      "--- assemblying 104 probes in region: Slc17a7-1\n",
      "--- assemblying 112 probes in region: Slc17a7-10\n",
      "--- assemblying 137 probes in region: Slc17a7-11\n",
      "--- assemblying 148 probes in region: Slc17a7-12\n",
      "--- assemblying 160 probes in region: Slc17a7-13\n",
      "--- assemblying 123 probes in region: Slc17a7-14\n",
      "--- assemblying 140 probes in region: Slc17a7-15\n",
      "--- assemblying 144 probes in region: Slc17a7-16\n",
      "--- assemblying 148 probes in region: Slc17a7-17\n",
      "--- assemblying 126 probes in region: Slc17a7-18\n",
      "--- assemblying 114 probes in region: Slc17a7-19\n",
      "--- assemblying 99 probes in region: Slc17a7-2\n",
      "--- assemblying 94 probes in region: Slc17a7-20\n",
      "--- assemblying 92 probes in region: Slc17a7-21\n",
      "--- assemblying 130 probes in region: Slc17a7-22\n",
      "--- assemblying 119 probes in region: Slc17a7-23\n",
      "--- assemblying 127 probes in region: Slc17a7-24\n",
      "--- assemblying 122 probes in region: Slc17a7-25\n",
      "--- assemblying 134 probes in region: Slc17a7-26\n",
      "--- assemblying 147 probes in region: Slc17a7-27\n",
      "--- assemblying 153 probes in region: Slc17a7-28\n",
      "--- assemblying 127 probes in region: Slc17a7-29\n",
      "--- assemblying 137 probes in region: Slc17a7-3\n",
      "--- assemblying 133 probes in region: Slc17a7-30\n",
      "--- assemblying 140 probes in region: Slc17a7-31\n",
      "--- assemblying 141 probes in region: Slc17a7-32\n",
      "--- assemblying 112 probes in region: Slc17a7-33\n",
      "--- assemblying 125 probes in region: Slc17a7-34\n",
      "--- assemblying 102 probes in region: Slc17a7-35\n",
      "--- assemblying 106 probes in region: Slc17a7-36\n",
      "--- assemblying 99 probes in region: Slc17a7-37\n",
      "--- assemblying 93 probes in region: Slc17a7-38\n",
      "--- assemblying 145 probes in region: Slc17a7-39\n",
      "--- assemblying 110 probes in region: Slc17a7-4\n",
      "--- assemblying 121 probes in region: Slc17a7-40\n",
      "--- assemblying 134 probes in region: Slc17a7-41\n",
      "--- assemblying 130 probes in region: Slc17a7-42\n",
      "--- assemblying 132 probes in region: Slc17a7-43\n",
      "--- assemblying 141 probes in region: Slc17a7-44\n",
      "--- assemblying 145 probes in region: Slc17a7-45\n",
      "--- assemblying 132 probes in region: Slc17a7-46\n",
      "--- assemblying 111 probes in region: Slc17a7-47\n",
      "--- assemblying 129 probes in region: Slc17a7-48\n",
      "--- assemblying 130 probes in region: Slc17a7-49\n",
      "--- assemblying 124 probes in region: Slc17a7-5\n",
      "--- assemblying 135 probes in region: Slc17a7-50\n",
      "--- assemblying 65 probes in region: Slc17a7-51\n",
      "--- assemblying 90 probes in region: Slc17a7-52\n",
      "--- assemblying 125 probes in region: Slc17a7-53\n",
      "--- assemblying 120 probes in region: Slc17a7-54\n",
      "--- assemblying 132 probes in region: Slc17a7-55\n",
      "--- assemblying 114 probes in region: Slc17a7-56\n",
      "--- assemblying 130 probes in region: Slc17a7-57\n",
      "--- assemblying 121 probes in region: Slc17a7-58\n",
      "--- assemblying 102 probes in region: Slc17a7-59\n",
      "--- assemblying 53 probes in region: Slc17a7-6\n",
      "--- assemblying 142 probes in region: Slc17a7-60\n",
      "--- assemblying 66 probes in region: Slc17a7-7\n",
      "--- assemblying 141 probes in region: Slc17a7-8\n",
      "--- assemblying 146 probes in region: Slc17a7-9\n",
      "--- assemblying 129 probes in region: Slc32a1-1\n",
      "--- assemblying 128 probes in region: Slc32a1-10\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- assemblying 144 probes in region: Slc32a1-11\n",
      "--- assemblying 129 probes in region: Slc32a1-12\n",
      "--- assemblying 144 probes in region: Slc32a1-13\n",
      "--- assemblying 121 probes in region: Slc32a1-14\n",
      "--- assemblying 127 probes in region: Slc32a1-15\n",
      "--- assemblying 114 probes in region: Slc32a1-16\n",
      "--- assemblying 141 probes in region: Slc32a1-17\n",
      "--- assemblying 127 probes in region: Slc32a1-18\n",
      "--- assemblying 136 probes in region: Slc32a1-19\n",
      "--- assemblying 138 probes in region: Slc32a1-2\n",
      "--- assemblying 135 probes in region: Slc32a1-20\n",
      "--- assemblying 144 probes in region: Slc32a1-21\n",
      "--- assemblying 129 probes in region: Slc32a1-3\n",
      "--- assemblying 123 probes in region: Slc32a1-4\n",
      "--- assemblying 134 probes in region: Slc32a1-5\n",
      "--- assemblying 131 probes in region: Slc32a1-6\n",
      "--- assemblying 123 probes in region: Slc32a1-7\n",
      "--- assemblying 114 probes in region: Slc32a1-8\n",
      "--- assemblying 123 probes in region: Slc32a1-9\n",
      "--- assemblying 105 probes in region: SST-1\n",
      "--- assemblying 141 probes in region: SST-10\n",
      "--- assemblying 145 probes in region: SST-11\n",
      "--- assemblying 141 probes in region: SST-12\n",
      "--- assemblying 79 probes in region: SST-13\n",
      "--- assemblying 95 probes in region: SST-14\n",
      "--- assemblying 82 probes in region: SST-15\n",
      "--- assemblying 131 probes in region: SST-16\n",
      "--- assemblying 151 probes in region: SST-17\n",
      "--- assemblying 96 probes in region: SST-18\n",
      "--- assemblying 114 probes in region: SST-19\n",
      "--- assemblying 49 probes in region: SST-2\n",
      "--- assemblying 114 probes in region: SST-20\n",
      "--- assemblying 103 probes in region: SST-21\n",
      "--- assemblying 61 probes in region: SST-22\n",
      "--- assemblying 112 probes in region: SST-23\n",
      "--- assemblying 148 probes in region: SST-24\n",
      "--- assemblying 140 probes in region: SST-25\n",
      "--- assemblying 151 probes in region: SST-26\n",
      "--- assemblying 141 probes in region: SST-27\n",
      "--- assemblying 137 probes in region: SST-28\n",
      "--- assemblying 124 probes in region: SST-29\n",
      "--- assemblying 53 probes in region: SST-3\n",
      "--- assemblying 141 probes in region: SST-30\n",
      "--- assemblying 108 probes in region: SST-31\n",
      "--- assemblying 139 probes in region: SST-32\n",
      "--- assemblying 150 probes in region: SST-33\n",
      "--- assemblying 148 probes in region: SST-34\n",
      "--- assemblying 141 probes in region: SST-35\n",
      "--- assemblying 144 probes in region: SST-36\n",
      "--- assemblying 147 probes in region: SST-37\n",
      "--- assemblying 114 probes in region: SST-38\n",
      "--- assemblying 112 probes in region: SST-39\n",
      "--- assemblying 118 probes in region: SST-4\n",
      "--- assemblying 135 probes in region: SST-40\n",
      "--- assemblying 147 probes in region: SST-41\n",
      "--- assemblying 141 probes in region: SST-42\n",
      "--- assemblying 139 probes in region: SST-43\n",
      "--- assemblying 146 probes in region: SST-44\n",
      "--- assemblying 139 probes in region: SST-45\n",
      "--- assemblying 138 probes in region: SST-46\n",
      "--- assemblying 130 probes in region: SST-47\n",
      "--- assemblying 160 probes in region: SST-48\n",
      "--- assemblying 151 probes in region: SST-49\n",
      "--- assemblying 114 probes in region: SST-5\n",
      "--- assemblying 135 probes in region: SST-50\n",
      "--- assemblying 147 probes in region: SST-51\n",
      "--- assemblying 116 probes in region: SST-52\n",
      "--- assemblying 124 probes in region: SST-53\n",
      "--- assemblying 101 probes in region: SST-54\n",
      "--- assemblying 122 probes in region: SST-55\n",
      "--- assemblying 116 probes in region: SST-56\n",
      "--- assemblying 117 probes in region: SST-57\n",
      "--- assemblying 108 probes in region: SST-58\n",
      "--- assemblying 134 probes in region: SST-59\n",
      "--- assemblying 149 probes in region: SST-6\n",
      "--- assemblying 113 probes in region: SST-60\n",
      "--- assemblying 143 probes in region: SST-7\n",
      "--- assemblying 130 probes in region: SST-8\n",
      "--- assemblying 145 probes in region: SST-9\n",
      "-- 32432 probes assembled in total.\n",
      "-- saving 32432 probes into file:\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\candidate_probes.fasta\n",
      "-- saving readout_summary into file:\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\readout_summary.pkl\n"
     ]
    }
   ],
   "source": [
    "reload(library_tools.assemble)\n",
    "from ImageAnalysis3.library_tools.assemble import Assemble_probes\n",
    "\n",
    "cand_probes, readout_summary = Assemble_probes(library_folder, filtered_pb_dic, \n",
    "                                               region_2_readout_dict, readout_dict, primers, \n",
    "                                               primer_len=19,\n",
    "                                               rc_targets=False, overwrite=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### specific to this library: swap out ~5% of the probes from this library to convert into gene_specific readouts"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 200,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "6:126980000-126990000_gene_Ccnd2-1_pb_10_pos_943_strand_+_readouts_[Stv_3_u,NDB_16_u]\n",
      "CCGCAATGGCTGACAACCGCCCATGATCGTCCGATCTGGCATAGACAGGTGTGACCCACGAGGAGAGCATATGGACAGATCGCGCATCGCGGCTAATAGAGCGGAAACCATGCGGCAATC\n"
     ]
    }
   ],
   "source": [
    "pb_id = 10\n",
    "print(cand_probes[pb_id].id)\n",
    "print(cand_probes[pb_id].seq)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 206,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[['Ccnd2', 'Stv_91'], ['Nkx2-1', 'Stv_92'], ['Slc17a7', 'Stv_94'], ['Slc32a1', 'Stv_95'], ['SST', 'Stv_99']]\n"
     ]
    }
   ],
   "source": [
    "gene_2_readout = {}\n",
    "gene_readouts = library_tools.assemble.load_readouts(len(gene_2_region_2_pbs_dict), \n",
    "                                                     _type='designed_readouts', \n",
    "                                                     _readout_folder=readout_folder, \n",
    "                                                     _num_colors=1, _start_channel=2,# 561\n",
    "                                                     )\n",
    "_gene_readout_usage = 0\n",
    "for _reg, _pbs in filtered_pb_dic.items():\n",
    "    _ind = int(_reg.split('-')[-1])\n",
    "    _gene = _reg[:-len(str(_ind))-1]\n",
    "    if _gene not in gene_2_readout:\n",
    "        gene_2_readout[_gene] = gene_readouts[_gene_readout_usage]\n",
    "        _gene_readout_usage += 1\n",
    "print([[_gene, _r.id] for _gene, _r in gene_2_readout.items()])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 239,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- replace 0.05 of Ccnd2 probes with readout: Stv_91\n",
      "-- 7 probes in gene:Ccnd2, reg:1\n",
      "-- 5 probes in gene:Ccnd2, reg:2\n",
      "-- 6 probes in gene:Ccnd2, reg:3\n",
      "-- 8 probes in gene:Ccnd2, reg:4\n",
      "-- 8 probes in gene:Ccnd2, reg:5\n",
      "-- 6 probes in gene:Ccnd2, reg:6\n",
      "-- 7 probes in gene:Ccnd2, reg:7\n",
      "-- 7 probes in gene:Ccnd2, reg:8\n",
      "-- 7 probes in gene:Ccnd2, reg:9\n",
      "-- 7 probes in gene:Ccnd2, reg:10\n",
      "-- 7 probes in gene:Ccnd2, reg:11\n",
      "-- skip gene:Ccnd2, reg:12 with 62 probes\n",
      "-- 7 probes in gene:Ccnd2, reg:13\n",
      "-- 6 probes in gene:Ccnd2, reg:14\n",
      "-- 7 probes in gene:Ccnd2, reg:15\n",
      "-- 7 probes in gene:Ccnd2, reg:16\n",
      "-- 7 probes in gene:Ccnd2, reg:17\n",
      "-- 8 probes in gene:Ccnd2, reg:18\n",
      "-- 8 probes in gene:Ccnd2, reg:19\n",
      "-- 8 probes in gene:Ccnd2, reg:20\n",
      "-- 7 probes in gene:Ccnd2, reg:21\n",
      "-- 6 probes in gene:Ccnd2, reg:22\n",
      "-- 6 probes in gene:Ccnd2, reg:23\n",
      "-- 7 probes in gene:Ccnd2, reg:24\n",
      "-- 7 probes in gene:Ccnd2, reg:25\n",
      "-- 6 probes in gene:Ccnd2, reg:26\n",
      "-- 7 probes in gene:Ccnd2, reg:27\n",
      "-- 7 probes in gene:Ccnd2, reg:28\n",
      "-- 8 probes in gene:Ccnd2, reg:29\n",
      "-- 7 probes in gene:Ccnd2, reg:30\n",
      "-- 7 probes in gene:Ccnd2, reg:31\n",
      "-- 7 probes in gene:Ccnd2, reg:32\n",
      "-- 7 probes in gene:Ccnd2, reg:33\n",
      "-- 7 probes in gene:Ccnd2, reg:34\n",
      "-- 8 probes in gene:Ccnd2, reg:35\n",
      "-- 6 probes in gene:Ccnd2, reg:36\n",
      "-- 6 probes in gene:Ccnd2, reg:37\n",
      "-- 8 probes in gene:Ccnd2, reg:38\n",
      "-- 6 probes in gene:Ccnd2, reg:39\n",
      "-- 7 probes in gene:Ccnd2, reg:40\n",
      "-- 7 probes in gene:Ccnd2, reg:41\n",
      "-- 6 probes in gene:Ccnd2, reg:42\n",
      "-- 6 probes in gene:Ccnd2, reg:43\n",
      "-- 7 probes in gene:Ccnd2, reg:44\n",
      "-- 7 probes in gene:Ccnd2, reg:45\n",
      "-- 7 probes in gene:Ccnd2, reg:46\n",
      "-- 8 probes in gene:Ccnd2, reg:47\n",
      "-- 7 probes in gene:Ccnd2, reg:48\n",
      "-- 7 probes in gene:Ccnd2, reg:49\n",
      "-- 7 probes in gene:Ccnd2, reg:50\n",
      "-- 5 probes in gene:Ccnd2, reg:51\n",
      "-- 7 probes in gene:Ccnd2, reg:52\n",
      "-- 6 probes in gene:Ccnd2, reg:53\n",
      "-- skip gene:Ccnd2, reg:54 with 34 probes\n",
      "-- 6 probes in gene:Ccnd2, reg:55\n",
      "-- 6 probes in gene:Ccnd2, reg:56\n",
      "-- 7 probes in gene:Ccnd2, reg:57\n",
      "-- 8 probes in gene:Ccnd2, reg:58\n",
      "-- 7 probes in gene:Ccnd2, reg:59\n",
      "-- 8 probes in gene:Ccnd2, reg:60\n",
      "- 349 probes for Ccnd2 replaced.\n",
      "- replace 0.05 of Nkx2-1 probes with readout: Stv_92\n",
      "-- skip gene:Nkx2-1, reg:1 with 79 probes\n",
      "-- skip gene:Nkx2-1, reg:2 with 62 probes\n",
      "-- 5 probes in gene:Nkx2-1, reg:3\n",
      "-- 7 probes in gene:Nkx2-1, reg:4\n",
      "-- 6 probes in gene:Nkx2-1, reg:5\n",
      "-- 8 probes in gene:Nkx2-1, reg:6\n",
      "-- 6 probes in gene:Nkx2-1, reg:7\n",
      "-- 6 probes in gene:Nkx2-1, reg:8\n",
      "-- 6 probes in gene:Nkx2-1, reg:9\n",
      "-- 7 probes in gene:Nkx2-1, reg:10\n",
      "-- 7 probes in gene:Nkx2-1, reg:11\n",
      "-- 6 probes in gene:Nkx2-1, reg:12\n",
      "-- 7 probes in gene:Nkx2-1, reg:13\n",
      "-- 7 probes in gene:Nkx2-1, reg:14\n",
      "-- 7 probes in gene:Nkx2-1, reg:15\n",
      "-- 7 probes in gene:Nkx2-1, reg:16\n",
      "-- 6 probes in gene:Nkx2-1, reg:17\n",
      "-- 8 probes in gene:Nkx2-1, reg:18\n",
      "-- 8 probes in gene:Nkx2-1, reg:19\n",
      "-- 7 probes in gene:Nkx2-1, reg:20\n",
      "-- 5 probes in gene:Nkx2-1, reg:21\n",
      "-- skip gene:Nkx2-1, reg:22 with 53 probes\n",
      "-- 5 probes in gene:Nkx2-1, reg:23\n",
      "-- skip gene:Nkx2-1, reg:24 with 76 probes\n",
      "-- 6 probes in gene:Nkx2-1, reg:25\n",
      "-- 7 probes in gene:Nkx2-1, reg:26\n",
      "-- 8 probes in gene:Nkx2-1, reg:27\n",
      "-- 6 probes in gene:Nkx2-1, reg:28\n",
      "-- 8 probes in gene:Nkx2-1, reg:29\n",
      "-- 7 probes in gene:Nkx2-1, reg:30\n",
      "-- 5 probes in gene:Nkx2-1, reg:31\n",
      "-- 6 probes in gene:Nkx2-1, reg:32\n",
      "-- 7 probes in gene:Nkx2-1, reg:33\n",
      "-- 8 probes in gene:Nkx2-1, reg:34\n",
      "-- 7 probes in gene:Nkx2-1, reg:35\n",
      "-- 7 probes in gene:Nkx2-1, reg:36\n",
      "-- 7 probes in gene:Nkx2-1, reg:37\n",
      "-- 7 probes in gene:Nkx2-1, reg:38\n",
      "-- 7 probes in gene:Nkx2-1, reg:39\n",
      "-- skip gene:Nkx2-1, reg:40 with 58 probes\n",
      "-- 6 probes in gene:Nkx2-1, reg:41\n",
      "-- 6 probes in gene:Nkx2-1, reg:42\n",
      "-- 7 probes in gene:Nkx2-1, reg:43\n",
      "-- 6 probes in gene:Nkx2-1, reg:44\n",
      "-- 7 probes in gene:Nkx2-1, reg:45\n",
      "-- 6 probes in gene:Nkx2-1, reg:46\n",
      "-- 7 probes in gene:Nkx2-1, reg:47\n",
      "-- 6 probes in gene:Nkx2-1, reg:48\n",
      "-- 5 probes in gene:Nkx2-1, reg:49\n",
      "-- 5 probes in gene:Nkx2-1, reg:50\n",
      "-- 7 probes in gene:Nkx2-1, reg:51\n",
      "-- 7 probes in gene:Nkx2-1, reg:52\n",
      "-- 7 probes in gene:Nkx2-1, reg:53\n",
      "-- 7 probes in gene:Nkx2-1, reg:54\n",
      "-- 6 probes in gene:Nkx2-1, reg:55\n",
      "-- 7 probes in gene:Nkx2-1, reg:56\n",
      "-- 5 probes in gene:Nkx2-1, reg:57\n",
      "-- 7 probes in gene:Nkx2-1, reg:58\n",
      "-- 7 probes in gene:Nkx2-1, reg:59\n",
      "-- 5 probes in gene:Nkx2-1, reg:60\n",
      "- 316 probes for Nkx2-1 replaced.\n",
      "- replace 0.05 of Slc17a7 probes with readout: Stv_94\n",
      "-- 6 probes in gene:Slc17a7, reg:1\n",
      "-- 5 probes in gene:Slc17a7, reg:2\n",
      "-- 7 probes in gene:Slc17a7, reg:3\n",
      "-- 5 probes in gene:Slc17a7, reg:4\n",
      "-- 7 probes in gene:Slc17a7, reg:5\n",
      "-- skip gene:Slc17a7, reg:6 with 53 probes\n",
      "-- skip gene:Slc17a7, reg:7 with 66 probes\n",
      "-- 8 probes in gene:Slc17a7, reg:8\n",
      "-- 8 probes in gene:Slc17a7, reg:9\n",
      "-- 6 probes in gene:Slc17a7, reg:10\n",
      "-- 7 probes in gene:Slc17a7, reg:11\n",
      "-- 8 probes in gene:Slc17a7, reg:12\n",
      "-- 8 probes in gene:Slc17a7, reg:13\n",
      "-- 7 probes in gene:Slc17a7, reg:14\n",
      "-- 7 probes in gene:Slc17a7, reg:15\n",
      "-- 8 probes in gene:Slc17a7, reg:16\n",
      "-- 8 probes in gene:Slc17a7, reg:17\n",
      "-- 6 probes in gene:Slc17a7, reg:18\n",
      "-- 6 probes in gene:Slc17a7, reg:19\n",
      "-- 5 probes in gene:Slc17a7, reg:20\n",
      "-- 4 probes in gene:Slc17a7, reg:21\n",
      "-- 7 probes in gene:Slc17a7, reg:22\n",
      "-- 6 probes in gene:Slc17a7, reg:23\n",
      "-- 7 probes in gene:Slc17a7, reg:24\n",
      "-- 7 probes in gene:Slc17a7, reg:25\n",
      "-- 7 probes in gene:Slc17a7, reg:26\n",
      "-- 7 probes in gene:Slc17a7, reg:27\n",
      "-- 8 probes in gene:Slc17a7, reg:28\n",
      "-- 7 probes in gene:Slc17a7, reg:29\n",
      "-- 7 probes in gene:Slc17a7, reg:30\n",
      "-- 7 probes in gene:Slc17a7, reg:31\n",
      "-- 8 probes in gene:Slc17a7, reg:32\n",
      "-- 6 probes in gene:Slc17a7, reg:33\n",
      "-- 7 probes in gene:Slc17a7, reg:34\n",
      "-- 6 probes in gene:Slc17a7, reg:35\n",
      "-- 6 probes in gene:Slc17a7, reg:36\n",
      "-- 5 probes in gene:Slc17a7, reg:37\n",
      "-- 5 probes in gene:Slc17a7, reg:38\n",
      "-- 8 probes in gene:Slc17a7, reg:39\n",
      "-- 7 probes in gene:Slc17a7, reg:40\n",
      "-- 7 probes in gene:Slc17a7, reg:41\n",
      "-- 7 probes in gene:Slc17a7, reg:42\n",
      "-- 6 probes in gene:Slc17a7, reg:43\n",
      "-- 8 probes in gene:Slc17a7, reg:44\n",
      "-- 8 probes in gene:Slc17a7, reg:45\n",
      "-- 6 probes in gene:Slc17a7, reg:46\n",
      "-- 6 probes in gene:Slc17a7, reg:47\n",
      "-- 7 probes in gene:Slc17a7, reg:48\n",
      "-- 7 probes in gene:Slc17a7, reg:49\n",
      "-- 7 probes in gene:Slc17a7, reg:50\n",
      "-- skip gene:Slc17a7, reg:51 with 65 probes\n",
      "-- 5 probes in gene:Slc17a7, reg:52\n",
      "-- 7 probes in gene:Slc17a7, reg:53\n",
      "-- 6 probes in gene:Slc17a7, reg:54\n",
      "-- 6 probes in gene:Slc17a7, reg:55\n",
      "-- 6 probes in gene:Slc17a7, reg:56\n",
      "-- 7 probes in gene:Slc17a7, reg:57\n",
      "-- 7 probes in gene:Slc17a7, reg:58\n",
      "-- 6 probes in gene:Slc17a7, reg:59\n",
      "-- 8 probes in gene:Slc17a7, reg:60\n",
      "- 335 probes for Slc17a7 replaced.\n",
      "- replace 0.05 of Slc32a1 probes with readout: Stv_95\n",
      "-- 7 probes in gene:Slc32a1, reg:1\n",
      "-- 6 probes in gene:Slc32a1, reg:2\n",
      "-- 7 probes in gene:Slc32a1, reg:3\n",
      "-- 7 probes in gene:Slc32a1, reg:4\n",
      "-- 7 probes in gene:Slc32a1, reg:5\n",
      "-- 7 probes in gene:Slc32a1, reg:6\n",
      "-- 7 probes in gene:Slc32a1, reg:7\n",
      "-- 6 probes in gene:Slc32a1, reg:8\n",
      "-- 7 probes in gene:Slc32a1, reg:9\n",
      "-- 7 probes in gene:Slc32a1, reg:10\n",
      "-- 8 probes in gene:Slc32a1, reg:11\n",
      "-- 7 probes in gene:Slc32a1, reg:12\n",
      "-- 8 probes in gene:Slc32a1, reg:13\n",
      "-- 7 probes in gene:Slc32a1, reg:14\n",
      "-- 7 probes in gene:Slc32a1, reg:15\n",
      "-- 6 probes in gene:Slc32a1, reg:16\n",
      "-- 8 probes in gene:Slc32a1, reg:17\n",
      "-- 7 probes in gene:Slc32a1, reg:18\n",
      "-- 7 probes in gene:Slc32a1, reg:19\n",
      "-- 7 probes in gene:Slc32a1, reg:20\n",
      "-- 8 probes in gene:Slc32a1, reg:21\n",
      "- 128 probes for Slc32a1 replaced.\n",
      "- replace 0.05 of SST probes with readout: Stv_99\n",
      "-- 5 probes in gene:SST, reg:1\n",
      "-- skip gene:SST, reg:2 with 49 probes\n",
      "-- skip gene:SST, reg:3 with 53 probes\n",
      "-- 6 probes in gene:SST, reg:4\n",
      "-- 6 probes in gene:SST, reg:5\n",
      "-- 8 probes in gene:SST, reg:6\n",
      "-- 8 probes in gene:SST, reg:7\n",
      "-- 7 probes in gene:SST, reg:8\n",
      "-- 8 probes in gene:SST, reg:9\n",
      "-- 8 probes in gene:SST, reg:10\n",
      "-- 8 probes in gene:SST, reg:11\n",
      "-- 8 probes in gene:SST, reg:12\n",
      "-- skip gene:SST, reg:13 with 79 probes\n",
      "-- 5 probes in gene:SST, reg:14\n",
      "-- 5 probes in gene:SST, reg:15\n",
      "-- 7 probes in gene:SST, reg:16\n",
      "-- 8 probes in gene:SST, reg:17\n",
      "-- 4 probes in gene:SST, reg:18\n",
      "-- 6 probes in gene:SST, reg:19\n",
      "-- 6 probes in gene:SST, reg:20\n",
      "-- 6 probes in gene:SST, reg:21\n",
      "-- skip gene:SST, reg:22 with 61 probes\n",
      "-- 6 probes in gene:SST, reg:23\n",
      "-- 8 probes in gene:SST, reg:24\n",
      "-- 7 probes in gene:SST, reg:25\n",
      "-- 8 probes in gene:SST, reg:26\n",
      "-- 8 probes in gene:SST, reg:27\n",
      "-- 7 probes in gene:SST, reg:28\n",
      "-- 7 probes in gene:SST, reg:29\n",
      "-- 8 probes in gene:SST, reg:30\n",
      "-- 6 probes in gene:SST, reg:31\n",
      "-- 7 probes in gene:SST, reg:32\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- 8 probes in gene:SST, reg:33\n",
      "-- 8 probes in gene:SST, reg:34\n",
      "-- 8 probes in gene:SST, reg:35\n",
      "-- 8 probes in gene:SST, reg:36\n",
      "-- 7 probes in gene:SST, reg:37\n",
      "-- 6 probes in gene:SST, reg:38\n",
      "-- 6 probes in gene:SST, reg:39\n",
      "-- 7 probes in gene:SST, reg:40\n",
      "-- 7 probes in gene:SST, reg:41\n",
      "-- 8 probes in gene:SST, reg:42\n",
      "-- 7 probes in gene:SST, reg:43\n",
      "-- 8 probes in gene:SST, reg:44\n",
      "-- 7 probes in gene:SST, reg:45\n",
      "-- 6 probes in gene:SST, reg:46\n",
      "-- 7 probes in gene:SST, reg:47\n",
      "-- 8 probes in gene:SST, reg:48\n",
      "-- 8 probes in gene:SST, reg:49\n",
      "-- 7 probes in gene:SST, reg:50\n",
      "-- 7 probes in gene:SST, reg:51\n",
      "-- 6 probes in gene:SST, reg:52\n",
      "-- 7 probes in gene:SST, reg:53\n",
      "-- 6 probes in gene:SST, reg:54\n",
      "-- 7 probes in gene:SST, reg:55\n",
      "-- 6 probes in gene:SST, reg:56\n",
      "-- 6 probes in gene:SST, reg:57\n",
      "-- 6 probes in gene:SST, reg:58\n",
      "-- 7 probes in gene:SST, reg:59\n",
      "-- 6 probes in gene:SST, reg:60\n",
      "- 338 probes for SST replaced.\n",
      "32432\n"
     ]
    }
   ],
   "source": [
    "reload(library_tools.assemble)\n",
    "reload(library_tools.quality_check)\n",
    "swap_ratio = 0.05\n",
    "new_cand_probes = library_tools.assemble.replace_gene_specifc_readouts(cand_probes, gene_2_readout, swap_ratio,\n",
    "                                                                       region_pb_lower_limit=80,\n",
    "                                                                       primer_len=19)\n",
    "print(len(new_cand_probes))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 240,
   "metadata": {},
   "outputs": [],
   "source": [
    "# save replaced \n",
    "with open(os.path.join(library_folder, 'gene_replaced_candidate_probes.fasta'), 'w') as _output_handle:\n",
    "    SeqIO.write(new_cand_probes, _output_handle, \"fasta\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# 4.Check quality"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 4.1 Load probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 241,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Total probe loaded: 32432\n"
     ]
    }
   ],
   "source": [
    "# folder for this library\n",
    "# candidate full-length probe filename\n",
    "#candidate_full_name = 'candidate_probes.fasta' # old version\n",
    "candidate_full_name = 'gene_replaced_candidate_probes.fasta' # gene readout replaced version\n",
    "# load full probes\n",
    "full_records = []\n",
    "with open(os.path.join(library_folder, candidate_full_name), 'r') as handle:\n",
    "    for record in SeqIO.parse(handle, \"fasta\"):\n",
    "        full_records.append(record)\n",
    "\n",
    "print(f\"Total probe loaded: {len(full_records)}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 242,
   "metadata": {},
   "outputs": [],
   "source": [
    "num_readout_per_probe = 2\n",
    "primer_len=19\n",
    "min_num_probes = 25 "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 243,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- Checking primer usage, total probes: 32432\n"
     ]
    }
   ],
   "source": [
    "import ImageAnalysis3.library_tools.quality_check as check\n",
    "reload(check)\n",
    "primer_check = check._check_primer_usage(full_records, primers[0], primers[1], primer_len=primer_len)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 244,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "gene: Ccnd2-1 -> 135\n",
      "gene: Ccnd2-2 -> 99\n",
      "gene: Ccnd2-3 -> 104\n",
      "gene: Ccnd2-4 -> 149\n",
      "gene: Ccnd2-5 -> 152\n",
      "gene: Ccnd2-6 -> 112\n",
      "gene: Ccnd2-7 -> 136\n",
      "gene: Ccnd2-8 -> 140\n",
      "gene: Ccnd2-9 -> 134\n",
      "gene: Ccnd2-10 -> 136\n",
      "gene: Ccnd2-11 -> 139\n",
      "gene: Ccnd2-12 -> 62\n",
      "gene: Ccnd2-13 -> 127\n",
      "gene: Ccnd2-14 -> 117\n",
      "gene: Ccnd2-15 -> 125\n",
      "gene: Ccnd2-16 -> 136\n",
      "gene: Ccnd2-17 -> 147\n",
      "gene: Ccnd2-18 -> 146\n",
      "gene: Ccnd2-19 -> 143\n",
      "gene: Ccnd2-20 -> 142\n",
      "gene: Ccnd2-21 -> 139\n",
      "gene: Ccnd2-22 -> 126\n",
      "gene: Ccnd2-23 -> 138\n",
      "gene: Ccnd2-24 -> 136\n",
      "gene: Ccnd2-25 -> 128\n",
      "gene: Ccnd2-26 -> 101\n",
      "gene: Ccnd2-27 -> 139\n",
      "gene: Ccnd2-28 -> 128\n",
      "gene: Ccnd2-29 -> 145\n",
      "gene: Ccnd2-30 -> 125\n",
      "gene: Ccnd2-31 -> 124\n",
      "gene: Ccnd2-32 -> 135\n",
      "gene: Ccnd2-33 -> 127\n",
      "gene: Ccnd2-34 -> 133\n",
      "gene: Ccnd2-35 -> 149\n",
      "gene: Ccnd2-36 -> 138\n",
      "gene: Ccnd2-37 -> 118\n",
      "gene: Ccnd2-38 -> 145\n",
      "gene: Ccnd2-39 -> 119\n",
      "gene: Ccnd2-40 -> 135\n",
      "gene: Ccnd2-41 -> 124\n",
      "gene: Ccnd2-42 -> 117\n",
      "gene: Ccnd2-43 -> 126\n",
      "gene: Ccnd2-44 -> 128\n",
      "gene: Ccnd2-45 -> 128\n",
      "gene: Ccnd2-46 -> 130\n",
      "gene: Ccnd2-47 -> 142\n",
      "gene: Ccnd2-48 -> 135\n",
      "gene: Ccnd2-49 -> 128\n",
      "gene: Ccnd2-50 -> 133\n",
      "gene: Ccnd2-51 -> 99\n",
      "gene: Ccnd2-52 -> 136\n",
      "gene: Ccnd2-53 -> 132\n",
      "gene: Ccnd2-54 -> 34\n",
      "gene: Ccnd2-55 -> 101\n",
      "gene: Ccnd2-56 -> 116\n",
      "gene: Ccnd2-57 -> 134\n",
      "gene: Ccnd2-58 -> 142\n",
      "gene: Ccnd2-59 -> 147\n",
      "gene: Ccnd2-60 -> 143\n",
      "gene: Nkx2-1-1 -> 79\n",
      "gene: Nkx2-1-2 -> 62\n",
      "gene: Nkx2-1-3 -> 86\n",
      "gene: Nkx2-1-4 -> 137\n",
      "gene: Nkx2-1-5 -> 120\n",
      "gene: Nkx2-1-6 -> 141\n",
      "gene: Nkx2-1-7 -> 138\n",
      "gene: Nkx2-1-8 -> 118\n",
      "gene: Nkx2-1-9 -> 126\n",
      "gene: Nkx2-1-10 -> 129\n",
      "gene: Nkx2-1-11 -> 128\n",
      "gene: Nkx2-1-12 -> 118\n",
      "gene: Nkx2-1-13 -> 134\n",
      "gene: Nkx2-1-14 -> 129\n",
      "gene: Nkx2-1-15 -> 129\n",
      "gene: Nkx2-1-16 -> 125\n",
      "gene: Nkx2-1-17 -> 138\n",
      "gene: Nkx2-1-18 -> 146\n",
      "gene: Nkx2-1-19 -> 153\n",
      "gene: Nkx2-1-20 -> 123\n",
      "gene: Nkx2-1-21 -> 100\n",
      "gene: Nkx2-1-22 -> 53\n",
      "gene: Nkx2-1-23 -> 87\n",
      "gene: Nkx2-1-24 -> 76\n",
      "gene: Nkx2-1-25 -> 111\n",
      "gene: Nkx2-1-26 -> 121\n",
      "gene: Nkx2-1-27 -> 144\n",
      "gene: Nkx2-1-28 -> 116\n",
      "gene: Nkx2-1-29 -> 141\n",
      "gene: Nkx2-1-30 -> 125\n",
      "gene: Nkx2-1-31 -> 94\n",
      "gene: Nkx2-1-32 -> 102\n",
      "gene: Nkx2-1-33 -> 135\n",
      "gene: Nkx2-1-34 -> 149\n",
      "gene: Nkx2-1-35 -> 154\n",
      "gene: Nkx2-1-36 -> 135\n",
      "gene: Nkx2-1-37 -> 133\n",
      "gene: Nkx2-1-38 -> 130\n",
      "gene: Nkx2-1-39 -> 125\n",
      "gene: Nkx2-1-40 -> 58\n",
      "gene: Nkx2-1-41 -> 101\n",
      "gene: Nkx2-1-42 -> 118\n",
      "gene: Nkx2-1-43 -> 123\n",
      "gene: Nkx2-1-44 -> 112\n",
      "gene: Nkx2-1-45 -> 133\n",
      "gene: Nkx2-1-46 -> 138\n",
      "gene: Nkx2-1-47 -> 130\n",
      "gene: Nkx2-1-48 -> 132\n",
      "gene: Nkx2-1-49 -> 100\n",
      "gene: Nkx2-1-50 -> 90\n",
      "gene: Nkx2-1-51 -> 122\n",
      "gene: Nkx2-1-52 -> 139\n",
      "gene: Nkx2-1-53 -> 123\n",
      "gene: Nkx2-1-54 -> 128\n",
      "gene: Nkx2-1-55 -> 138\n",
      "gene: Nkx2-1-56 -> 137\n",
      "gene: Nkx2-1-57 -> 100\n",
      "gene: Nkx2-1-58 -> 122\n",
      "gene: Nkx2-1-59 -> 125\n",
      "gene: Nkx2-1-60 -> 86\n",
      "gene: Slc17a7-1 -> 104\n",
      "gene: Slc17a7-2 -> 99\n",
      "gene: Slc17a7-3 -> 137\n",
      "gene: Slc17a7-4 -> 110\n",
      "gene: Slc17a7-5 -> 124\n",
      "gene: Slc17a7-6 -> 53\n",
      "gene: Slc17a7-7 -> 66\n",
      "gene: Slc17a7-8 -> 141\n",
      "gene: Slc17a7-9 -> 146\n",
      "gene: Slc17a7-10 -> 112\n",
      "gene: Slc17a7-11 -> 137\n",
      "gene: Slc17a7-12 -> 148\n",
      "gene: Slc17a7-13 -> 160\n",
      "gene: Slc17a7-14 -> 123\n",
      "gene: Slc17a7-15 -> 140\n",
      "gene: Slc17a7-16 -> 144\n",
      "gene: Slc17a7-17 -> 148\n",
      "gene: Slc17a7-18 -> 126\n",
      "gene: Slc17a7-19 -> 114\n",
      "gene: Slc17a7-20 -> 94\n",
      "gene: Slc17a7-21 -> 92\n",
      "gene: Slc17a7-22 -> 130\n",
      "gene: Slc17a7-23 -> 119\n",
      "gene: Slc17a7-24 -> 127\n",
      "gene: Slc17a7-25 -> 122\n",
      "gene: Slc17a7-26 -> 134\n",
      "gene: Slc17a7-27 -> 147\n",
      "gene: Slc17a7-28 -> 153\n",
      "gene: Slc17a7-29 -> 127\n",
      "gene: Slc17a7-30 -> 133\n",
      "gene: Slc17a7-31 -> 140\n",
      "gene: Slc17a7-32 -> 141\n",
      "gene: Slc17a7-33 -> 112\n",
      "gene: Slc17a7-34 -> 125\n",
      "gene: Slc17a7-35 -> 102\n",
      "gene: Slc17a7-36 -> 106\n",
      "gene: Slc17a7-37 -> 99\n",
      "gene: Slc17a7-38 -> 93\n",
      "gene: Slc17a7-39 -> 145\n",
      "gene: Slc17a7-40 -> 121\n",
      "gene: Slc17a7-41 -> 134\n",
      "gene: Slc17a7-42 -> 130\n",
      "gene: Slc17a7-43 -> 132\n",
      "gene: Slc17a7-44 -> 141\n",
      "gene: Slc17a7-45 -> 145\n",
      "gene: Slc17a7-46 -> 132\n",
      "gene: Slc17a7-47 -> 111\n",
      "gene: Slc17a7-48 -> 129\n",
      "gene: Slc17a7-49 -> 130\n",
      "gene: Slc17a7-50 -> 135\n",
      "gene: Slc17a7-51 -> 65\n",
      "gene: Slc17a7-52 -> 90\n",
      "gene: Slc17a7-53 -> 125\n",
      "gene: Slc17a7-54 -> 120\n",
      "gene: Slc17a7-55 -> 132\n",
      "gene: Slc17a7-56 -> 114\n",
      "gene: Slc17a7-57 -> 130\n",
      "gene: Slc17a7-58 -> 121\n",
      "gene: Slc17a7-59 -> 102\n",
      "gene: Slc17a7-60 -> 142\n",
      "gene: Slc32a1-1 -> 129\n",
      "gene: Slc32a1-2 -> 138\n",
      "gene: Slc32a1-3 -> 129\n",
      "gene: Slc32a1-4 -> 123\n",
      "gene: Slc32a1-5 -> 134\n",
      "gene: Slc32a1-6 -> 131\n",
      "gene: Slc32a1-7 -> 123\n",
      "gene: Slc32a1-8 -> 114\n",
      "gene: Slc32a1-9 -> 123\n",
      "gene: Slc32a1-10 -> 128\n",
      "gene: Slc32a1-11 -> 144\n",
      "gene: Slc32a1-12 -> 129\n",
      "gene: Slc32a1-13 -> 144\n",
      "gene: Slc32a1-14 -> 121\n",
      "gene: Slc32a1-15 -> 127\n",
      "gene: Slc32a1-16 -> 114\n",
      "gene: Slc32a1-17 -> 141\n",
      "gene: Slc32a1-18 -> 127\n",
      "gene: Slc32a1-19 -> 136\n",
      "gene: Slc32a1-20 -> 135\n",
      "gene: Slc32a1-21 -> 144\n",
      "gene: SST-1 -> 105\n",
      "gene: SST-2 -> 49\n",
      "gene: SST-3 -> 53\n",
      "gene: SST-4 -> 118\n",
      "gene: SST-5 -> 114\n",
      "gene: SST-6 -> 149\n",
      "gene: SST-7 -> 143\n",
      "gene: SST-8 -> 130\n",
      "gene: SST-9 -> 145\n",
      "gene: SST-10 -> 141\n",
      "gene: SST-11 -> 145\n",
      "gene: SST-12 -> 141\n",
      "gene: SST-13 -> 79\n",
      "gene: SST-14 -> 95\n",
      "gene: SST-15 -> 82\n",
      "gene: SST-16 -> 131\n",
      "gene: SST-17 -> 151\n",
      "gene: SST-18 -> 96\n",
      "gene: SST-19 -> 114\n",
      "gene: SST-20 -> 114\n",
      "gene: SST-21 -> 103\n",
      "gene: SST-22 -> 61\n",
      "gene: SST-23 -> 112\n",
      "gene: SST-24 -> 148\n",
      "gene: SST-25 -> 140\n",
      "gene: SST-26 -> 151\n",
      "gene: SST-27 -> 141\n",
      "gene: SST-28 -> 137\n",
      "gene: SST-29 -> 124\n",
      "gene: SST-30 -> 141\n",
      "gene: SST-31 -> 108\n",
      "gene: SST-32 -> 139\n",
      "gene: SST-33 -> 150\n",
      "gene: SST-34 -> 148\n",
      "gene: SST-35 -> 141\n",
      "gene: SST-36 -> 144\n",
      "gene: SST-37 -> 147\n",
      "gene: SST-38 -> 114\n",
      "gene: SST-39 -> 112\n",
      "gene: SST-40 -> 135\n",
      "gene: SST-41 -> 147\n",
      "gene: SST-42 -> 141\n",
      "gene: SST-43 -> 139\n",
      "gene: SST-44 -> 146\n",
      "gene: SST-45 -> 139\n",
      "gene: SST-46 -> 138\n",
      "gene: SST-47 -> 130\n",
      "gene: SST-48 -> 160\n",
      "gene: SST-49 -> 151\n",
      "gene: SST-50 -> 135\n",
      "gene: SST-51 -> 147\n",
      "gene: SST-52 -> 116\n",
      "gene: SST-53 -> 124\n",
      "gene: SST-54 -> 101\n",
      "gene: SST-55 -> 122\n",
      "gene: SST-56 -> 116\n",
      "gene: SST-57 -> 117\n",
      "gene: SST-58 -> 108\n",
      "gene: SST-59 -> 134\n",
      "gene: SST-60 -> 113\n"
     ]
    }
   ],
   "source": [
    "reg_size_dic, len_check = check._check_region_size(full_records, \n",
    "                                                   min_size=min_num_probes)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 245,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Ccnd2-1 -> ['Stv_3', 'Stv_91', 'NDB_16']\n",
      "Ccnd2-2 -> ['Stv_32', 'Stv_91', 'NDB_17']\n",
      "Ccnd2-3 -> ['Stv_4', 'Stv_91', 'NDB_19']\n",
      "Ccnd2-4 -> ['Stv_33', 'Stv_91', 'NDB_20']\n",
      "Ccnd2-5 -> ['Stv_5', 'Stv_91', 'NDB_22']\n",
      "Ccnd2-6 -> ['Stv_35', 'Stv_91', 'NDB_23']\n",
      "Ccnd2-7 -> ['Stv_6', 'Stv_91', 'NDB_25']\n",
      "Ccnd2-8 -> ['Stv_36', 'Stv_91', 'NDB_26']\n",
      "Ccnd2-9 -> ['Stv_7', 'Stv_91', 'NDB_28']\n",
      "Ccnd2-10 -> ['Stv_37', 'Stv_91', 'NDB_29']\n",
      "Ccnd2-11 -> ['Stv_8', 'Stv_91', 'NDB_31']\n",
      "-- Error in barcode number for region: Ccnd2-12\n",
      "Ccnd2-12 -> ['Stv_39', 'NDB_32']\n",
      "Ccnd2-13 -> ['Stv_9', 'Stv_91', 'NDB_34']\n",
      "Ccnd2-14 -> ['Stv_40', 'Stv_91', 'NDB_35']\n",
      "Ccnd2-15 -> ['Stv_10', 'Stv_91', 'NDB_37']\n",
      "Ccnd2-16 -> ['Stv_42', 'Stv_91', 'NDB_38']\n",
      "Ccnd2-17 -> ['Stv_11', 'Stv_91', 'NDB_40']\n",
      "Ccnd2-18 -> ['Stv_44', 'Stv_91', 'NDB_41']\n",
      "Ccnd2-19 -> ['Stv_12', 'Stv_91', 'NDB_43']\n",
      "Ccnd2-20 -> ['Stv_45', 'Stv_91', 'NDB_44']\n",
      "Ccnd2-21 -> ['Stv_13', 'Stv_91', 'NDB_46']\n",
      "Ccnd2-22 -> ['Stv_46', 'Stv_91', 'NDB_47']\n",
      "Ccnd2-23 -> ['Stv_14', 'Stv_91', 'NDB_49']\n",
      "Ccnd2-24 -> ['Stv_48', 'Stv_91', 'NDB_50']\n",
      "Ccnd2-25 -> ['Stv_16', 'Stv_91', 'NDB_52']\n",
      "Ccnd2-26 -> ['Stv_50', 'Stv_91', 'NDB_53']\n",
      "Ccnd2-27 -> ['Stv_19', 'Stv_91', 'NDB_55']\n",
      "Ccnd2-28 -> ['Stv_53', 'Stv_91', 'NDB_56']\n",
      "Ccnd2-29 -> ['Stv_20', 'Stv_91', 'NDB_58']\n",
      "Ccnd2-30 -> ['Stv_54', 'Stv_91', 'NDB_59']\n",
      "Ccnd2-31 -> ['Stv_21', 'Stv_91', 'NDB_61']\n",
      "Ccnd2-32 -> ['Stv_59', 'Stv_91', 'NDB_62']\n",
      "Ccnd2-33 -> ['Stv_22', 'Stv_91', 'NDB_64']\n",
      "Ccnd2-34 -> ['Stv_60', 'Stv_91', 'NDB_65']\n",
      "Ccnd2-35 -> ['Stv_23', 'Stv_91', 'NDB_67']\n",
      "Ccnd2-36 -> ['Stv_61', 'Stv_91', 'NDB_68']\n",
      "Ccnd2-37 -> ['Stv_25', 'Stv_91', 'NDB_70']\n",
      "Ccnd2-38 -> ['Stv_63', 'Stv_91', 'NDB_71']\n",
      "Ccnd2-39 -> ['Stv_26', 'Stv_91', 'NDB_73']\n",
      "Ccnd2-40 -> ['Stv_64', 'Stv_91', 'NDB_74']\n",
      "Ccnd2-41 -> ['Stv_27', 'Stv_91', 'NDB_76']\n",
      "Ccnd2-42 -> ['Stv_65', 'Stv_91', 'NDB_77']\n",
      "Ccnd2-43 -> ['Stv_28', 'Stv_91', 'NDB_79']\n",
      "Ccnd2-44 -> ['Stv_86', 'Stv_91', 'NDB_80']\n",
      "Ccnd2-45 -> ['Stv_29', 'Stv_91', 'NDB_82']\n",
      "Ccnd2-46 -> ['Stv_87', 'Stv_91', 'NDB_83']\n",
      "Ccnd2-47 -> ['Stv_30', 'Stv_91', 'NDB_85']\n",
      "Ccnd2-48 -> ['Stv_88', 'Stv_91', 'NDB_86']\n",
      "Ccnd2-49 -> ['Stv_31', 'Stv_91', 'NDB_88']\n",
      "Ccnd2-50 -> ['Stv_90', 'Stv_91', 'NDB_89']\n",
      "Ccnd2-51 -> ['Stv_91', 'NDB_1', 'NDB_91']\n",
      "Ccnd2-52 -> ['Stv_91', 'NDB_2', 'NDB_92']\n",
      "Ccnd2-53 -> ['Stv_91', 'NDB_4', 'NDB_94']\n",
      "-- Error in barcode number for region: Ccnd2-54\n",
      "Ccnd2-54 -> ['NDB_5', 'NDB_95']\n",
      "Ccnd2-55 -> ['Stv_91', 'NDB_7', 'NDB_97']\n",
      "Ccnd2-56 -> ['Stv_91', 'NDB_8', 'NDB_98']\n",
      "Ccnd2-57 -> ['Stv_91', 'NDB_10', 'NDB_100']\n",
      "Ccnd2-58 -> ['Stv_91', 'NDB_11', 'NDB_101']\n",
      "Ccnd2-59 -> ['Stv_91', 'NDB_13', 'NDB_103']\n",
      "Ccnd2-60 -> ['Stv_91', 'NDB_14', 'NDB_104']\n",
      "-- Error in barcode number for region: Nkx2-1-1\n",
      "Nkx2-1-1 -> ['Stv_3', 'NDB_106']\n",
      "-- Error in barcode number for region: Nkx2-1-2\n",
      "Nkx2-1-2 -> ['Stv_32', 'NDB_107']\n",
      "Nkx2-1-3 -> ['Stv_4', 'Stv_92', 'NDB_109']\n",
      "Nkx2-1-4 -> ['Stv_33', 'Stv_92', 'NDB_110']\n",
      "Nkx2-1-5 -> ['Stv_5', 'Stv_92', 'NDB_112']\n",
      "Nkx2-1-6 -> ['Stv_35', 'Stv_92', 'NDB_113']\n",
      "Nkx2-1-7 -> ['Stv_6', 'Stv_92', 'NDB_115']\n",
      "Nkx2-1-8 -> ['Stv_36', 'Stv_92', 'NDB_116']\n",
      "Nkx2-1-9 -> ['Stv_7', 'Stv_92', 'NDB_118']\n",
      "Nkx2-1-10 -> ['Stv_37', 'Stv_92', 'NDB_119']\n",
      "Nkx2-1-11 -> ['Stv_8', 'Stv_92', 'NDB_121']\n",
      "Nkx2-1-12 -> ['Stv_39', 'Stv_92', 'NDB_122']\n",
      "Nkx2-1-13 -> ['Stv_9', 'Stv_92', 'NDB_124']\n",
      "Nkx2-1-14 -> ['Stv_40', 'Stv_92', 'NDB_125']\n",
      "Nkx2-1-15 -> ['Stv_10', 'Stv_92', 'NDB_127']\n",
      "Nkx2-1-16 -> ['Stv_42', 'Stv_92', 'NDB_128']\n",
      "Nkx2-1-17 -> ['Stv_11', 'Stv_92', 'NDB_130']\n",
      "Nkx2-1-18 -> ['Stv_44', 'Stv_92', 'NDB_131']\n",
      "Nkx2-1-19 -> ['Stv_12', 'Stv_92', 'NDB_133']\n",
      "Nkx2-1-20 -> ['Stv_45', 'Stv_92', 'NDB_134']\n",
      "Nkx2-1-21 -> ['Stv_13', 'Stv_92', 'NDB_136']\n",
      "-- Error in barcode number for region: Nkx2-1-22\n",
      "Nkx2-1-22 -> ['Stv_46', 'NDB_137']\n",
      "Nkx2-1-23 -> ['Stv_14', 'Stv_92', 'NDB_139']\n",
      "-- Error in barcode number for region: Nkx2-1-24\n",
      "Nkx2-1-24 -> ['Stv_48', 'NDB_140']\n",
      "Nkx2-1-25 -> ['Stv_16', 'Stv_92', 'NDB_142']\n",
      "Nkx2-1-26 -> ['Stv_50', 'Stv_92', 'NDB_143']\n",
      "Nkx2-1-27 -> ['Stv_19', 'Stv_92', 'NDB_145']\n",
      "Nkx2-1-28 -> ['Stv_53', 'Stv_92', 'NDB_146']\n",
      "Nkx2-1-29 -> ['Stv_20', 'Stv_92', 'NDB_148']\n",
      "Nkx2-1-30 -> ['Stv_54', 'Stv_92', 'NDB_149']\n",
      "Nkx2-1-31 -> ['Stv_21', 'Stv_92', 'NDB_151']\n",
      "Nkx2-1-32 -> ['Stv_59', 'Stv_92', 'NDB_152']\n",
      "Nkx2-1-33 -> ['Stv_22', 'Stv_92', 'NDB_154']\n",
      "Nkx2-1-34 -> ['Stv_60', 'Stv_92', 'NDB_155']\n",
      "Nkx2-1-35 -> ['Stv_23', 'Stv_92', 'NDB_157']\n",
      "Nkx2-1-36 -> ['Stv_61', 'Stv_92', 'NDB_158']\n",
      "Nkx2-1-37 -> ['Stv_25', 'Stv_92', 'NDB_160']\n",
      "Nkx2-1-38 -> ['Stv_63', 'Stv_92', 'NDB_161']\n",
      "Nkx2-1-39 -> ['Stv_26', 'Stv_92', 'NDB_163']\n",
      "-- Error in barcode number for region: Nkx2-1-40\n",
      "Nkx2-1-40 -> ['Stv_64', 'NDB_164']\n",
      "Nkx2-1-41 -> ['Stv_27', 'Stv_92', 'NDB_166']\n",
      "Nkx2-1-42 -> ['Stv_65', 'Stv_92', 'NDB_167']\n",
      "Nkx2-1-43 -> ['Stv_28', 'Stv_92', 'NDB_169']\n",
      "Nkx2-1-44 -> ['Stv_86', 'Stv_92', 'NDB_170']\n",
      "Nkx2-1-45 -> ['Stv_29', 'Stv_92', 'NDB_172']\n",
      "Nkx2-1-46 -> ['Stv_87', 'Stv_92', 'NDB_173']\n",
      "Nkx2-1-47 -> ['Stv_30', 'Stv_92', 'NDB_175']\n",
      "Nkx2-1-48 -> ['Stv_88', 'Stv_92', 'NDB_176']\n",
      "Nkx2-1-49 -> ['Stv_31', 'Stv_92', 'NDB_178']\n",
      "Nkx2-1-50 -> ['Stv_90', 'Stv_92', 'NDB_179']\n",
      "Nkx2-1-51 -> ['Stv_92', 'NDB_1', 'NDB_181']\n",
      "Nkx2-1-52 -> ['Stv_92', 'NDB_2', 'NDB_182']\n",
      "Nkx2-1-53 -> ['Stv_92', 'NDB_4', 'NDB_184']\n",
      "Nkx2-1-54 -> ['Stv_92', 'NDB_5', 'NDB_185']\n",
      "Nkx2-1-55 -> ['Stv_92', 'NDB_7', 'NDB_187']\n",
      "Nkx2-1-56 -> ['Stv_92', 'NDB_8', 'NDB_188']\n",
      "Nkx2-1-57 -> ['Stv_92', 'NDB_10', 'NDB_190']\n",
      "Nkx2-1-58 -> ['Stv_92', 'NDB_11', 'NDB_191']\n",
      "Nkx2-1-59 -> ['Stv_92', 'NDB_13', 'NDB_193']\n",
      "Nkx2-1-60 -> ['Stv_92', 'NDB_14', 'NDB_194']\n",
      "Slc17a7-1 -> ['Stv_3', 'Stv_94', 'NDB_196']\n",
      "Slc17a7-2 -> ['Stv_32', 'Stv_94', 'NDB_197']\n",
      "Slc17a7-3 -> ['Stv_4', 'Stv_94', 'NDB_199']\n",
      "Slc17a7-4 -> ['Stv_33', 'Stv_94', 'NDB_200']\n",
      "Slc17a7-5 -> ['Stv_5', 'Stv_94', 'NDB_202']\n",
      "-- Error in barcode number for region: Slc17a7-6\n",
      "Slc17a7-6 -> ['Stv_35', 'NDB_203']\n",
      "-- Error in barcode number for region: Slc17a7-7\n",
      "Slc17a7-7 -> ['Stv_6', 'NDB_205']\n",
      "Slc17a7-8 -> ['Stv_36', 'Stv_94', 'NDB_206']\n",
      "Slc17a7-9 -> ['Stv_7', 'Stv_94', 'NDB_208']\n",
      "Slc17a7-10 -> ['Stv_37', 'Stv_94', 'NDB_209']\n",
      "Slc17a7-11 -> ['Stv_8', 'Stv_94', 'NDB_211']\n",
      "Slc17a7-12 -> ['Stv_39', 'Stv_94', 'NDB_212']\n",
      "Slc17a7-13 -> ['Stv_9', 'Stv_94', 'NDB_214']\n",
      "Slc17a7-14 -> ['Stv_40', 'Stv_94', 'NDB_215']\n",
      "Slc17a7-15 -> ['Stv_10', 'Stv_94', 'NDB_217']\n",
      "Slc17a7-16 -> ['Stv_42', 'Stv_94', 'NDB_218']\n",
      "Slc17a7-17 -> ['Stv_11', 'Stv_94', 'NDB_220']\n",
      "Slc17a7-18 -> ['Stv_44', 'Stv_94', 'NDB_221']\n",
      "Slc17a7-19 -> ['Stv_12', 'Stv_94', 'NDB_223']\n",
      "Slc17a7-20 -> ['Stv_45', 'Stv_94', 'NDB_224']\n",
      "Slc17a7-21 -> ['Stv_13', 'Stv_94', 'NDB_226']\n",
      "Slc17a7-22 -> ['Stv_46', 'Stv_94', 'NDB_227']\n",
      "Slc17a7-23 -> ['Stv_14', 'Stv_94', 'NDB_229']\n",
      "Slc17a7-24 -> ['Stv_48', 'Stv_94', 'NDB_230']\n",
      "Slc17a7-25 -> ['Stv_16', 'Stv_94', 'NDB_232']\n",
      "Slc17a7-26 -> ['Stv_50', 'Stv_94', 'NDB_233']\n",
      "Slc17a7-27 -> ['Stv_19', 'Stv_94', 'NDB_235']\n",
      "Slc17a7-28 -> ['Stv_53', 'Stv_94', 'NDB_236']\n",
      "Slc17a7-29 -> ['Stv_20', 'Stv_94', 'NDB_238']\n",
      "Slc17a7-30 -> ['Stv_54', 'Stv_94', 'NDB_239']\n",
      "Slc17a7-31 -> ['Stv_21', 'Stv_94', 'NDB_241']\n",
      "Slc17a7-32 -> ['Stv_59', 'Stv_94', 'NDB_242']\n",
      "Slc17a7-33 -> ['Stv_22', 'Stv_94', 'NDB_244']\n",
      "Slc17a7-34 -> ['Stv_60', 'Stv_94', 'NDB_245']\n",
      "Slc17a7-35 -> ['Stv_23', 'Stv_94', 'NDB_247']\n",
      "Slc17a7-36 -> ['Stv_61', 'Stv_94', 'NDB_248']\n",
      "Slc17a7-37 -> ['Stv_25', 'Stv_94', 'NDB_250']\n",
      "Slc17a7-38 -> ['Stv_63', 'Stv_94', 'NDB_251']\n",
      "Slc17a7-39 -> ['Stv_26', 'Stv_94', 'NDB_253']\n",
      "Slc17a7-40 -> ['Stv_64', 'Stv_94', 'NDB_254']\n",
      "Slc17a7-41 -> ['Stv_27', 'Stv_94', 'NDB_256']\n",
      "Slc17a7-42 -> ['Stv_65', 'Stv_94', 'NDB_257']\n",
      "Slc17a7-43 -> ['Stv_28', 'Stv_94', 'NDB_259']\n",
      "Slc17a7-44 -> ['Stv_86', 'Stv_94', 'NDB_260']\n",
      "Slc17a7-45 -> ['Stv_29', 'Stv_94', 'NDB_262']\n",
      "Slc17a7-46 -> ['Stv_87', 'Stv_94', 'NDB_263']\n",
      "Slc17a7-47 -> ['Stv_30', 'Stv_94', 'NDB_265']\n",
      "Slc17a7-48 -> ['Stv_88', 'Stv_94', 'NDB_266']\n",
      "Slc17a7-49 -> ['Stv_31', 'Stv_94', 'NDB_268']\n",
      "Slc17a7-50 -> ['Stv_90', 'Stv_94', 'NDB_269']\n",
      "-- Error in barcode number for region: Slc17a7-51\n",
      "Slc17a7-51 -> ['NDB_1', 'NDB_271']\n",
      "Slc17a7-52 -> ['Stv_94', 'NDB_2', 'NDB_272']\n",
      "Slc17a7-53 -> ['Stv_94', 'NDB_4', 'NDB_274']\n",
      "Slc17a7-54 -> ['Stv_94', 'NDB_5', 'NDB_275']\n",
      "Slc17a7-55 -> ['Stv_94', 'NDB_7', 'NDB_277']\n",
      "Slc17a7-56 -> ['Stv_94', 'NDB_8', 'NDB_278']\n",
      "Slc17a7-57 -> ['Stv_94', 'NDB_10', 'NDB_280']\n",
      "Slc17a7-58 -> ['Stv_94', 'NDB_11', 'NDB_281']\n",
      "Slc17a7-59 -> ['Stv_94', 'NDB_13', 'NDB_283']\n",
      "Slc17a7-60 -> ['Stv_94', 'NDB_14', 'NDB_284']\n",
      "Slc32a1-1 -> ['Stv_3', 'Stv_95', 'NDB_286']\n",
      "Slc32a1-2 -> ['Stv_32', 'Stv_95', 'NDB_287']\n",
      "Slc32a1-3 -> ['Stv_4', 'Stv_95', 'NDB_289']\n",
      "Slc32a1-4 -> ['Stv_33', 'Stv_95', 'NDB_290']\n",
      "Slc32a1-5 -> ['Stv_5', 'Stv_95', 'NDB_292']\n",
      "Slc32a1-6 -> ['Stv_35', 'Stv_95', 'NDB_293']\n",
      "Slc32a1-7 -> ['Stv_6', 'Stv_95', 'NDB_295']\n",
      "Slc32a1-8 -> ['Stv_36', 'Stv_95', 'NDB_296']\n",
      "Slc32a1-9 -> ['Stv_7', 'Stv_95', 'NDB_298']\n",
      "Slc32a1-10 -> ['Stv_37', 'Stv_95', 'NDB_299']\n",
      "Slc32a1-11 -> ['Stv_8', 'Stv_95', 'NDB_301']\n",
      "Slc32a1-12 -> ['Stv_39', 'Stv_95', 'NDB_302']\n",
      "Slc32a1-13 -> ['Stv_9', 'Stv_95', 'NDB_304']\n",
      "Slc32a1-14 -> ['Stv_40', 'Stv_95', 'NDB_305']\n",
      "Slc32a1-15 -> ['Stv_10', 'Stv_95', 'NDB_307']\n",
      "Slc32a1-16 -> ['Stv_42', 'Stv_95', 'NDB_308']\n",
      "Slc32a1-17 -> ['Stv_11', 'Stv_95', 'NDB_310']\n",
      "Slc32a1-18 -> ['Stv_44', 'Stv_95', 'NDB_311']\n",
      "Slc32a1-19 -> ['Stv_12', 'Stv_95', 'NDB_313']\n",
      "Slc32a1-20 -> ['Stv_45', 'Stv_95', 'NDB_314']\n",
      "Slc32a1-21 -> ['Stv_13', 'Stv_95', 'NDB_316']\n",
      "SST-1 -> ['Stv_3', 'Stv_99', 'NDB_317']\n",
      "-- Error in barcode number for region: SST-2\n",
      "SST-2 -> ['Stv_32', 'NDB_319']\n",
      "-- Error in barcode number for region: SST-3\n",
      "SST-3 -> ['Stv_4', 'NDB_320']\n",
      "SST-4 -> ['Stv_33', 'Stv_99', 'NDB_322']\n",
      "SST-5 -> ['Stv_5', 'Stv_99', 'NDB_323']\n",
      "SST-6 -> ['Stv_35', 'Stv_99', 'NDB_325']\n",
      "SST-7 -> ['Stv_6', 'Stv_99', 'NDB_326']\n",
      "SST-8 -> ['Stv_36', 'Stv_99', 'NDB_328']\n",
      "SST-9 -> ['Stv_7', 'Stv_99', 'NDB_329']\n",
      "SST-10 -> ['Stv_37', 'Stv_99', 'NDB_331']\n",
      "SST-11 -> ['Stv_8', 'Stv_99', 'NDB_332']\n",
      "SST-12 -> ['Stv_39', 'Stv_99', 'NDB_334']\n",
      "-- Error in barcode number for region: SST-13\n",
      "SST-13 -> ['Stv_9', 'NDB_335']\n",
      "SST-14 -> ['Stv_40', 'Stv_99', 'NDB_337']\n",
      "SST-15 -> ['Stv_10', 'Stv_99', 'NDB_338']\n",
      "SST-16 -> ['Stv_42', 'Stv_99', 'NDB_340']\n",
      "SST-17 -> ['Stv_11', 'Stv_99', 'NDB_341']\n",
      "SST-18 -> ['Stv_44', 'Stv_99', 'NDB_343']\n",
      "SST-19 -> ['Stv_12', 'Stv_99', 'NDB_344']\n",
      "SST-20 -> ['Stv_45', 'Stv_99', 'NDB_346']\n",
      "SST-21 -> ['Stv_13', 'Stv_99', 'NDB_347']\n",
      "-- Error in barcode number for region: SST-22\n",
      "SST-22 -> ['Stv_46', 'NDB_349']\n",
      "SST-23 -> ['Stv_14', 'Stv_99', 'NDB_350']\n",
      "SST-24 -> ['Stv_48', 'Stv_99', 'NDB_352']\n",
      "SST-25 -> ['Stv_16', 'Stv_99', 'NDB_353']\n",
      "SST-26 -> ['Stv_50', 'Stv_99', 'NDB_355']\n",
      "SST-27 -> ['Stv_19', 'Stv_99', 'NDB_356']\n",
      "SST-28 -> ['Stv_53', 'Stv_99', 'NDB_358']\n",
      "SST-29 -> ['Stv_20', 'Stv_99', 'NDB_359']\n",
      "SST-30 -> ['Stv_54', 'Stv_99', 'NDB_361']\n",
      "SST-31 -> ['Stv_21', 'Stv_99', 'NDB_362']\n",
      "SST-32 -> ['Stv_59', 'Stv_99', 'NDB_364']\n",
      "SST-33 -> ['Stv_22', 'Stv_99', 'NDB_365']\n",
      "SST-34 -> ['Stv_60', 'Stv_99', 'NDB_367']\n",
      "SST-35 -> ['Stv_23', 'Stv_99', 'NDB_368']\n",
      "SST-36 -> ['Stv_61', 'Stv_99', 'NDB_370']\n",
      "SST-37 -> ['Stv_25', 'Stv_99', 'NDB_371']\n",
      "SST-38 -> ['Stv_63', 'Stv_99', 'NDB_373']\n",
      "SST-39 -> ['Stv_26', 'Stv_99', 'NDB_374']\n",
      "SST-40 -> ['Stv_64', 'Stv_99', 'NDB_376']\n",
      "SST-41 -> ['Stv_27', 'Stv_99', 'NDB_377']\n",
      "SST-42 -> ['Stv_65', 'Stv_99', 'NDB_379']\n",
      "SST-43 -> ['Stv_28', 'Stv_99', 'NDB_380']\n",
      "SST-44 -> ['Stv_86', 'Stv_99', 'NDB_382']\n",
      "SST-45 -> ['Stv_29', 'Stv_99', 'NDB_383']\n",
      "SST-46 -> ['Stv_87', 'Stv_99', 'NDB_385']\n",
      "SST-47 -> ['Stv_30', 'Stv_99', 'NDB_386']\n",
      "SST-48 -> ['Stv_88', 'Stv_99', 'NDB_388']\n",
      "SST-49 -> ['Stv_31', 'Stv_99', 'NDB_389']\n",
      "SST-50 -> ['Stv_90', 'Stv_99', 'NDB_391']\n",
      "SST-51 -> ['Stv_99', 'NDB_1', 'NDB_392']\n",
      "SST-52 -> ['Stv_99', 'NDB_2', 'NDB_394']\n",
      "SST-53 -> ['Stv_99', 'NDB_4', 'NDB_395']\n",
      "SST-54 -> ['Stv_99', 'NDB_5', 'NDB_397']\n",
      "SST-55 -> ['Stv_99', 'NDB_7', 'NDB_398']\n",
      "SST-56 -> ['Stv_99', 'NDB_8', 'NDB_400']\n",
      "SST-57 -> ['Stv_99', 'NDB_10', 'NDB_401']\n",
      "SST-58 -> ['Stv_99', 'NDB_11', 'NDB_403']\n",
      "SST-59 -> ['Stv_99', 'NDB_13', 'NDB_404']\n",
      "SST-60 -> ['Stv_99', 'NDB_14', 'NDB_406']\n"
     ]
    }
   ],
   "source": [
    "# update readout_dict\n",
    "updated_readout_dict = {_k:_v for _k,_v in readout_dict.items()}\n",
    "updated_readout_dict['u'].extend(gene_readouts)\n",
    "\n",
    "reg_readout_dic, reg2readout_check = check._check_region_to_readouts(full_records, updated_readout_dict, \n",
    "                                                                     total_readout_num=num_readout_per_probe+1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 247,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- extracting probes from >region:Ccnd2-1, region:Ccnd2-2, region:Ccnd2-3, region:Ccnd2-4, region:Ccnd2-5, region:Ccnd2-6, region:Ccnd2-7, region:Ccnd2-8, region:Ccnd2-9, region:Ccnd2-10, region:Ccnd2-11, region:Ccnd2-12, region:Ccnd2-13, region:Ccnd2-14, region:Ccnd2-15, region:Ccnd2-16, region:Ccnd2-17, region:Ccnd2-18, region:Ccnd2-19, region:Ccnd2-20, region:Ccnd2-21, region:Ccnd2-22, region:Ccnd2-23, region:Ccnd2-24, region:Ccnd2-25, region:Ccnd2-26, region:Ccnd2-27, region:Ccnd2-28, region:Ccnd2-29, region:Ccnd2-30, region:Ccnd2-31, region:Ccnd2-32, region:Ccnd2-33, region:Ccnd2-34, region:Ccnd2-35, region:Ccnd2-36, region:Ccnd2-37, region:Ccnd2-38, region:Ccnd2-39, region:Ccnd2-40, region:Ccnd2-41, region:Ccnd2-42, region:Ccnd2-43, region:Ccnd2-44, region:Ccnd2-45, region:Ccnd2-46, region:Ccnd2-47, region:Ccnd2-48, region:Ccnd2-49, region:Ccnd2-50, region:Ccnd2-51, region:Ccnd2-52, region:Ccnd2-53, region:Ccnd2-54, region:Ccnd2-55, region:Ccnd2-56, region:Ccnd2-57, region:Ccnd2-58, region:Ccnd2-59, region:Ccnd2-60, region:Nkx2-1-1, region:Nkx2-1-2, region:Nkx2-1-3, region:Nkx2-1-4, region:Nkx2-1-5, region:Nkx2-1-6, region:Nkx2-1-7, region:Nkx2-1-8, region:Nkx2-1-9, region:Nkx2-1-10, region:Nkx2-1-11, region:Nkx2-1-12, region:Nkx2-1-13, region:Nkx2-1-14, region:Nkx2-1-15, region:Nkx2-1-16, region:Nkx2-1-17, region:Nkx2-1-18, region:Nkx2-1-19, region:Nkx2-1-20, region:Nkx2-1-21, region:Nkx2-1-22, region:Nkx2-1-23, region:Nkx2-1-24, region:Nkx2-1-25, region:Nkx2-1-26, region:Nkx2-1-27, region:Nkx2-1-28, region:Nkx2-1-29, region:Nkx2-1-30, region:Nkx2-1-31, region:Nkx2-1-32, region:Nkx2-1-33, region:Nkx2-1-34, region:Nkx2-1-35, region:Nkx2-1-36, region:Nkx2-1-37, region:Nkx2-1-38, region:Nkx2-1-39, region:Nkx2-1-40, region:Nkx2-1-41, region:Nkx2-1-42, region:Nkx2-1-43, region:Nkx2-1-44, region:Nkx2-1-45, region:Nkx2-1-46, region:Nkx2-1-47, region:Nkx2-1-48, region:Nkx2-1-49, region:Nkx2-1-50, region:Nkx2-1-51, region:Nkx2-1-52, region:Nkx2-1-53, region:Nkx2-1-54, region:Nkx2-1-55, region:Nkx2-1-56, region:Nkx2-1-57, region:Nkx2-1-58, region:Nkx2-1-59, region:Nkx2-1-60, region:Slc17a7-1, region:Slc17a7-2, region:Slc17a7-3, region:Slc17a7-4, region:Slc17a7-5, region:Slc17a7-6, region:Slc17a7-7, region:Slc17a7-8, region:Slc17a7-9, region:Slc17a7-10, region:Slc17a7-11, region:Slc17a7-12, region:Slc17a7-13, region:Slc17a7-14, region:Slc17a7-15, region:Slc17a7-16, region:Slc17a7-17, region:Slc17a7-18, region:Slc17a7-19, region:Slc17a7-20, region:Slc17a7-21, region:Slc17a7-22, region:Slc17a7-23, region:Slc17a7-24, region:Slc17a7-25, region:Slc17a7-26, region:Slc17a7-27, region:Slc17a7-28, region:Slc17a7-29, region:Slc17a7-30, region:Slc17a7-31, region:Slc17a7-32, region:Slc17a7-33, region:Slc17a7-34, region:Slc17a7-35, region:Slc17a7-36, region:Slc17a7-37, region:Slc17a7-38, region:Slc17a7-39, region:Slc17a7-40, region:Slc17a7-41, region:Slc17a7-42, region:Slc17a7-43, region:Slc17a7-44, region:Slc17a7-45, region:Slc17a7-46, region:Slc17a7-47, region:Slc17a7-48, region:Slc17a7-49, region:Slc17a7-50, region:Slc17a7-51, region:Slc17a7-52, region:Slc17a7-53, region:Slc17a7-54, region:Slc17a7-55, region:Slc17a7-56, region:Slc17a7-57, region:Slc17a7-58, region:Slc17a7-59, region:Slc17a7-60, region:Slc32a1-1, region:Slc32a1-2, region:Slc32a1-3, region:Slc32a1-4, region:Slc32a1-5, region:Slc32a1-6, region:Slc32a1-7, region:Slc32a1-8, region:Slc32a1-9, region:Slc32a1-10, region:Slc32a1-11, region:Slc32a1-12, region:Slc32a1-13, region:Slc32a1-14, region:Slc32a1-15, region:Slc32a1-16, region:Slc32a1-17, region:Slc32a1-18, region:Slc32a1-19, region:Slc32a1-20, region:Slc32a1-21, region:SST-1, region:SST-2, region:SST-3, region:SST-4, region:SST-5, region:SST-6, region:SST-7, region:SST-8, region:SST-9, region:SST-10, region:SST-11, region:SST-12, region:SST-13, region:SST-14, region:SST-15, region:SST-16, region:SST-17, region:SST-18, region:SST-19, region:SST-20, region:SST-21, region:SST-22, region:SST-23, region:SST-24, region:SST-25, region:SST-26, region:SST-27, region:SST-28, region:SST-29, region:SST-30, region:SST-31, region:SST-32, region:SST-33, region:SST-34, region:SST-35, region:SST-36, region:SST-37, region:SST-38, region:SST-39, region:SST-40, region:SST-41, region:SST-42, region:SST-43, region:SST-44, region:SST-45, region:SST-46, region:SST-47, region:SST-48, region:SST-49, region:SST-50, region:SST-51, region:SST-52, region:SST-53, region:SST-54, region:SST-55, region:SST-56, region:SST-57, region:SST-58, region:SST-59, region:SST-60,  > Done.\n",
      "-- inverting region_to_readout dict\n",
      "NDB_1 -> ['Ccnd2-51', 'Nkx2-1-51', 'SST-51', 'Slc17a7-51']\n",
      "NDB_10 -> ['Ccnd2-57', 'Nkx2-1-57', 'SST-57', 'Slc17a7-57']\n",
      "NDB_100 -> ['Ccnd2-57']\n",
      "NDB_101 -> ['Ccnd2-58']\n",
      "NDB_103 -> ['Ccnd2-59']\n",
      "NDB_104 -> ['Ccnd2-60']\n",
      "NDB_106 -> ['Nkx2-1-1']\n",
      "NDB_107 -> ['Nkx2-1-2']\n",
      "NDB_109 -> ['Nkx2-1-3']\n",
      "NDB_11 -> ['Ccnd2-58', 'Nkx2-1-58', 'SST-58', 'Slc17a7-58']\n",
      "NDB_110 -> ['Nkx2-1-4']\n",
      "NDB_112 -> ['Nkx2-1-5']\n",
      "NDB_113 -> ['Nkx2-1-6']\n",
      "NDB_115 -> ['Nkx2-1-7']\n",
      "NDB_116 -> ['Nkx2-1-8']\n",
      "NDB_118 -> ['Nkx2-1-9']\n",
      "NDB_119 -> ['Nkx2-1-10']\n",
      "NDB_121 -> ['Nkx2-1-11']\n",
      "NDB_122 -> ['Nkx2-1-12']\n",
      "NDB_124 -> ['Nkx2-1-13']\n",
      "NDB_125 -> ['Nkx2-1-14']\n",
      "NDB_127 -> ['Nkx2-1-15']\n",
      "NDB_128 -> ['Nkx2-1-16']\n",
      "NDB_13 -> ['Ccnd2-59', 'Nkx2-1-59', 'SST-59', 'Slc17a7-59']\n",
      "NDB_130 -> ['Nkx2-1-17']\n",
      "NDB_131 -> ['Nkx2-1-18']\n",
      "NDB_133 -> ['Nkx2-1-19']\n",
      "NDB_134 -> ['Nkx2-1-20']\n",
      "NDB_136 -> ['Nkx2-1-21']\n",
      "NDB_137 -> ['Nkx2-1-22']\n",
      "NDB_139 -> ['Nkx2-1-23']\n",
      "NDB_14 -> ['Ccnd2-60', 'Nkx2-1-60', 'SST-60', 'Slc17a7-60']\n",
      "NDB_140 -> ['Nkx2-1-24']\n",
      "NDB_142 -> ['Nkx2-1-25']\n",
      "NDB_143 -> ['Nkx2-1-26']\n",
      "NDB_145 -> ['Nkx2-1-27']\n",
      "NDB_146 -> ['Nkx2-1-28']\n",
      "NDB_148 -> ['Nkx2-1-29']\n",
      "NDB_149 -> ['Nkx2-1-30']\n",
      "NDB_151 -> ['Nkx2-1-31']\n",
      "NDB_152 -> ['Nkx2-1-32']\n",
      "NDB_154 -> ['Nkx2-1-33']\n",
      "NDB_155 -> ['Nkx2-1-34']\n",
      "NDB_157 -> ['Nkx2-1-35']\n",
      "NDB_158 -> ['Nkx2-1-36']\n",
      "NDB_16 -> ['Ccnd2-1']\n",
      "NDB_160 -> ['Nkx2-1-37']\n",
      "NDB_161 -> ['Nkx2-1-38']\n",
      "NDB_163 -> ['Nkx2-1-39']\n",
      "NDB_164 -> ['Nkx2-1-40']\n",
      "NDB_166 -> ['Nkx2-1-41']\n",
      "NDB_167 -> ['Nkx2-1-42']\n",
      "NDB_169 -> ['Nkx2-1-43']\n",
      "NDB_17 -> ['Ccnd2-2']\n",
      "NDB_170 -> ['Nkx2-1-44']\n",
      "NDB_172 -> ['Nkx2-1-45']\n",
      "NDB_173 -> ['Nkx2-1-46']\n",
      "NDB_175 -> ['Nkx2-1-47']\n",
      "NDB_176 -> ['Nkx2-1-48']\n",
      "NDB_178 -> ['Nkx2-1-49']\n",
      "NDB_179 -> ['Nkx2-1-50']\n",
      "NDB_181 -> ['Nkx2-1-51']\n",
      "NDB_182 -> ['Nkx2-1-52']\n",
      "NDB_184 -> ['Nkx2-1-53']\n",
      "NDB_185 -> ['Nkx2-1-54']\n",
      "NDB_187 -> ['Nkx2-1-55']\n",
      "NDB_188 -> ['Nkx2-1-56']\n",
      "NDB_19 -> ['Ccnd2-3']\n",
      "NDB_190 -> ['Nkx2-1-57']\n",
      "NDB_191 -> ['Nkx2-1-58']\n",
      "NDB_193 -> ['Nkx2-1-59']\n",
      "NDB_194 -> ['Nkx2-1-60']\n",
      "NDB_196 -> ['Slc17a7-1']\n",
      "NDB_197 -> ['Slc17a7-2']\n",
      "NDB_199 -> ['Slc17a7-3']\n",
      "NDB_2 -> ['Ccnd2-52', 'Nkx2-1-52', 'SST-52', 'Slc17a7-52']\n",
      "NDB_20 -> ['Ccnd2-4']\n",
      "NDB_200 -> ['Slc17a7-4']\n",
      "NDB_202 -> ['Slc17a7-5']\n",
      "NDB_203 -> ['Slc17a7-6']\n",
      "NDB_205 -> ['Slc17a7-7']\n",
      "NDB_206 -> ['Slc17a7-8']\n",
      "NDB_208 -> ['Slc17a7-9']\n",
      "NDB_209 -> ['Slc17a7-10']\n",
      "NDB_211 -> ['Slc17a7-11']\n",
      "NDB_212 -> ['Slc17a7-12']\n",
      "NDB_214 -> ['Slc17a7-13']\n",
      "NDB_215 -> ['Slc17a7-14']\n",
      "NDB_217 -> ['Slc17a7-15']\n",
      "NDB_218 -> ['Slc17a7-16']\n",
      "NDB_22 -> ['Ccnd2-5']\n",
      "NDB_220 -> ['Slc17a7-17']\n",
      "NDB_221 -> ['Slc17a7-18']\n",
      "NDB_223 -> ['Slc17a7-19']\n",
      "NDB_224 -> ['Slc17a7-20']\n",
      "NDB_226 -> ['Slc17a7-21']\n",
      "NDB_227 -> ['Slc17a7-22']\n",
      "NDB_229 -> ['Slc17a7-23']\n",
      "NDB_23 -> ['Ccnd2-6']\n",
      "NDB_230 -> ['Slc17a7-24']\n",
      "NDB_232 -> ['Slc17a7-25']\n",
      "NDB_233 -> ['Slc17a7-26']\n",
      "NDB_235 -> ['Slc17a7-27']\n",
      "NDB_236 -> ['Slc17a7-28']\n",
      "NDB_238 -> ['Slc17a7-29']\n",
      "NDB_239 -> ['Slc17a7-30']\n",
      "NDB_241 -> ['Slc17a7-31']\n",
      "NDB_242 -> ['Slc17a7-32']\n",
      "NDB_244 -> ['Slc17a7-33']\n",
      "NDB_245 -> ['Slc17a7-34']\n",
      "NDB_247 -> ['Slc17a7-35']\n",
      "NDB_248 -> ['Slc17a7-36']\n",
      "NDB_25 -> ['Ccnd2-7']\n",
      "NDB_250 -> ['Slc17a7-37']\n",
      "NDB_251 -> ['Slc17a7-38']\n",
      "NDB_253 -> ['Slc17a7-39']\n",
      "NDB_254 -> ['Slc17a7-40']\n",
      "NDB_256 -> ['Slc17a7-41']\n",
      "NDB_257 -> ['Slc17a7-42']\n",
      "NDB_259 -> ['Slc17a7-43']\n",
      "NDB_26 -> ['Ccnd2-8']\n",
      "NDB_260 -> ['Slc17a7-44']\n",
      "NDB_262 -> ['Slc17a7-45']\n",
      "NDB_263 -> ['Slc17a7-46']\n",
      "NDB_265 -> ['Slc17a7-47']\n",
      "NDB_266 -> ['Slc17a7-48']\n",
      "NDB_268 -> ['Slc17a7-49']\n",
      "NDB_269 -> ['Slc17a7-50']\n",
      "NDB_271 -> ['Slc17a7-51']\n",
      "NDB_272 -> ['Slc17a7-52']\n",
      "NDB_274 -> ['Slc17a7-53']\n",
      "NDB_275 -> ['Slc17a7-54']\n",
      "NDB_277 -> ['Slc17a7-55']\n",
      "NDB_278 -> ['Slc17a7-56']\n",
      "NDB_28 -> ['Ccnd2-9']\n",
      "NDB_280 -> ['Slc17a7-57']\n",
      "NDB_281 -> ['Slc17a7-58']\n",
      "NDB_283 -> ['Slc17a7-59']\n",
      "NDB_284 -> ['Slc17a7-60']\n",
      "NDB_286 -> ['Slc32a1-1']\n",
      "NDB_287 -> ['Slc32a1-2']\n",
      "NDB_289 -> ['Slc32a1-3']\n",
      "NDB_29 -> ['Ccnd2-10']\n",
      "NDB_290 -> ['Slc32a1-4']\n",
      "NDB_292 -> ['Slc32a1-5']\n",
      "NDB_293 -> ['Slc32a1-6']\n",
      "NDB_295 -> ['Slc32a1-7']\n",
      "NDB_296 -> ['Slc32a1-8']\n",
      "NDB_298 -> ['Slc32a1-9']\n",
      "NDB_299 -> ['Slc32a1-10']\n",
      "NDB_301 -> ['Slc32a1-11']\n",
      "NDB_302 -> ['Slc32a1-12']\n",
      "NDB_304 -> ['Slc32a1-13']\n",
      "NDB_305 -> ['Slc32a1-14']\n",
      "NDB_307 -> ['Slc32a1-15']\n",
      "NDB_308 -> ['Slc32a1-16']\n",
      "NDB_31 -> ['Ccnd2-11']\n",
      "NDB_310 -> ['Slc32a1-17']\n",
      "NDB_311 -> ['Slc32a1-18']\n",
      "NDB_313 -> ['Slc32a1-19']\n",
      "NDB_314 -> ['Slc32a1-20']\n",
      "NDB_316 -> ['Slc32a1-21']\n",
      "NDB_317 -> ['SST-1']\n",
      "NDB_319 -> ['SST-2']\n",
      "NDB_32 -> ['Ccnd2-12']\n",
      "NDB_320 -> ['SST-3']\n",
      "NDB_322 -> ['SST-4']\n",
      "NDB_323 -> ['SST-5']\n",
      "NDB_325 -> ['SST-6']\n",
      "NDB_326 -> ['SST-7']\n",
      "NDB_328 -> ['SST-8']\n",
      "NDB_329 -> ['SST-9']\n",
      "NDB_331 -> ['SST-10']\n",
      "NDB_332 -> ['SST-11']\n",
      "NDB_334 -> ['SST-12']\n",
      "NDB_335 -> ['SST-13']\n",
      "NDB_337 -> ['SST-14']\n",
      "NDB_338 -> ['SST-15']\n",
      "NDB_34 -> ['Ccnd2-13']\n",
      "NDB_340 -> ['SST-16']\n",
      "NDB_341 -> ['SST-17']\n",
      "NDB_343 -> ['SST-18']\n",
      "NDB_344 -> ['SST-19']\n",
      "NDB_346 -> ['SST-20']\n",
      "NDB_347 -> ['SST-21']\n",
      "NDB_349 -> ['SST-22']\n",
      "NDB_35 -> ['Ccnd2-14']\n",
      "NDB_350 -> ['SST-23']\n",
      "NDB_352 -> ['SST-24']\n",
      "NDB_353 -> ['SST-25']\n",
      "NDB_355 -> ['SST-26']\n",
      "NDB_356 -> ['SST-27']\n",
      "NDB_358 -> ['SST-28']\n",
      "NDB_359 -> ['SST-29']\n",
      "NDB_361 -> ['SST-30']\n",
      "NDB_362 -> ['SST-31']\n",
      "NDB_364 -> ['SST-32']\n",
      "NDB_365 -> ['SST-33']\n",
      "NDB_367 -> ['SST-34']\n",
      "NDB_368 -> ['SST-35']\n",
      "NDB_37 -> ['Ccnd2-15']\n",
      "NDB_370 -> ['SST-36']\n",
      "NDB_371 -> ['SST-37']\n",
      "NDB_373 -> ['SST-38']\n",
      "NDB_374 -> ['SST-39']\n",
      "NDB_376 -> ['SST-40']\n",
      "NDB_377 -> ['SST-41']\n",
      "NDB_379 -> ['SST-42']\n",
      "NDB_38 -> ['Ccnd2-16']\n",
      "NDB_380 -> ['SST-43']\n",
      "NDB_382 -> ['SST-44']\n",
      "NDB_383 -> ['SST-45']\n",
      "NDB_385 -> ['SST-46']\n",
      "NDB_386 -> ['SST-47']\n",
      "NDB_388 -> ['SST-48']\n",
      "NDB_389 -> ['SST-49']\n",
      "NDB_391 -> ['SST-50']\n",
      "NDB_392 -> ['SST-51']\n",
      "NDB_394 -> ['SST-52']\n",
      "NDB_395 -> ['SST-53']\n",
      "NDB_397 -> ['SST-54']\n",
      "NDB_398 -> ['SST-55']\n",
      "NDB_4 -> ['Ccnd2-53', 'Nkx2-1-53', 'SST-53', 'Slc17a7-53']\n",
      "NDB_40 -> ['Ccnd2-17']\n",
      "NDB_400 -> ['SST-56']\n",
      "NDB_401 -> ['SST-57']\n",
      "NDB_403 -> ['SST-58']\n",
      "NDB_404 -> ['SST-59']\n",
      "NDB_406 -> ['SST-60']\n",
      "NDB_41 -> ['Ccnd2-18']\n",
      "NDB_43 -> ['Ccnd2-19']\n",
      "NDB_44 -> ['Ccnd2-20']\n",
      "NDB_46 -> ['Ccnd2-21']\n",
      "NDB_47 -> ['Ccnd2-22']\n",
      "NDB_49 -> ['Ccnd2-23']\n",
      "NDB_5 -> ['Ccnd2-54', 'Nkx2-1-54', 'SST-54', 'Slc17a7-54']\n",
      "NDB_50 -> ['Ccnd2-24']\n",
      "NDB_52 -> ['Ccnd2-25']\n",
      "NDB_53 -> ['Ccnd2-26']\n",
      "NDB_55 -> ['Ccnd2-27']\n",
      "NDB_56 -> ['Ccnd2-28']\n",
      "NDB_58 -> ['Ccnd2-29']\n",
      "NDB_59 -> ['Ccnd2-30']\n",
      "NDB_61 -> ['Ccnd2-31']\n",
      "NDB_62 -> ['Ccnd2-32']\n",
      "NDB_64 -> ['Ccnd2-33']\n",
      "NDB_65 -> ['Ccnd2-34']\n",
      "NDB_67 -> ['Ccnd2-35']\n",
      "NDB_68 -> ['Ccnd2-36']\n",
      "NDB_7 -> ['Ccnd2-55', 'Nkx2-1-55', 'SST-55', 'Slc17a7-55']\n",
      "NDB_70 -> ['Ccnd2-37']\n",
      "NDB_71 -> ['Ccnd2-38']\n",
      "NDB_73 -> ['Ccnd2-39']\n",
      "NDB_74 -> ['Ccnd2-40']\n",
      "NDB_76 -> ['Ccnd2-41']\n",
      "NDB_77 -> ['Ccnd2-42']\n",
      "NDB_79 -> ['Ccnd2-43']\n",
      "NDB_8 -> ['Ccnd2-56', 'Nkx2-1-56', 'SST-56', 'Slc17a7-56']\n",
      "NDB_80 -> ['Ccnd2-44']\n",
      "NDB_82 -> ['Ccnd2-45']\n",
      "NDB_83 -> ['Ccnd2-46']\n",
      "NDB_85 -> ['Ccnd2-47']\n",
      "NDB_86 -> ['Ccnd2-48']\n",
      "NDB_88 -> ['Ccnd2-49']\n",
      "NDB_89 -> ['Ccnd2-50']\n",
      "NDB_91 -> ['Ccnd2-51']\n",
      "NDB_92 -> ['Ccnd2-52']\n",
      "NDB_94 -> ['Ccnd2-53']\n",
      "NDB_95 -> ['Ccnd2-54']\n",
      "NDB_97 -> ['Ccnd2-55']\n",
      "NDB_98 -> ['Ccnd2-56']\n",
      "Stv_10 -> ['Ccnd2-15', 'Nkx2-1-15', 'SST-15', 'Slc17a7-15', 'Slc32a1-15']\n",
      "Stv_11 -> ['Ccnd2-17', 'Nkx2-1-17', 'SST-17', 'Slc17a7-17', 'Slc32a1-17']\n",
      "Stv_12 -> ['Ccnd2-19', 'Nkx2-1-19', 'SST-19', 'Slc17a7-19', 'Slc32a1-19']\n",
      "Stv_13 -> ['Ccnd2-21', 'Nkx2-1-21', 'SST-21', 'Slc17a7-21', 'Slc32a1-21']\n",
      "Stv_14 -> ['Ccnd2-23', 'Nkx2-1-23', 'SST-23', 'Slc17a7-23']\n",
      "Stv_16 -> ['Ccnd2-25', 'Nkx2-1-25', 'SST-25', 'Slc17a7-25']\n",
      "Stv_19 -> ['Ccnd2-27', 'Nkx2-1-27', 'SST-27', 'Slc17a7-27']\n",
      "Stv_20 -> ['Ccnd2-29', 'Nkx2-1-29', 'SST-29', 'Slc17a7-29']\n",
      "Stv_21 -> ['Ccnd2-31', 'Nkx2-1-31', 'SST-31', 'Slc17a7-31']\n",
      "Stv_22 -> ['Ccnd2-33', 'Nkx2-1-33', 'SST-33', 'Slc17a7-33']\n",
      "Stv_23 -> ['Ccnd2-35', 'Nkx2-1-35', 'SST-35', 'Slc17a7-35']\n",
      "Stv_25 -> ['Ccnd2-37', 'Nkx2-1-37', 'SST-37', 'Slc17a7-37']\n",
      "Stv_26 -> ['Ccnd2-39', 'Nkx2-1-39', 'SST-39', 'Slc17a7-39']\n",
      "Stv_27 -> ['Ccnd2-41', 'Nkx2-1-41', 'SST-41', 'Slc17a7-41']\n",
      "Stv_28 -> ['Ccnd2-43', 'Nkx2-1-43', 'SST-43', 'Slc17a7-43']\n",
      "Stv_29 -> ['Ccnd2-45', 'Nkx2-1-45', 'SST-45', 'Slc17a7-45']\n",
      "Stv_3 -> ['Ccnd2-1', 'Nkx2-1-1', 'SST-1', 'Slc17a7-1', 'Slc32a1-1']\n",
      "Stv_30 -> ['Ccnd2-47', 'Nkx2-1-47', 'SST-47', 'Slc17a7-47']\n",
      "Stv_31 -> ['Ccnd2-49', 'Nkx2-1-49', 'SST-49', 'Slc17a7-49']\n",
      "Stv_32 -> ['Ccnd2-2', 'Nkx2-1-2', 'SST-2', 'Slc17a7-2', 'Slc32a1-2']\n",
      "Stv_33 -> ['Ccnd2-4', 'Nkx2-1-4', 'SST-4', 'Slc17a7-4', 'Slc32a1-4']\n",
      "Stv_35 -> ['Ccnd2-6', 'Nkx2-1-6', 'SST-6', 'Slc17a7-6', 'Slc32a1-6']\n",
      "Stv_36 -> ['Ccnd2-8', 'Nkx2-1-8', 'SST-8', 'Slc17a7-8', 'Slc32a1-8']\n",
      "Stv_37 -> ['Ccnd2-10', 'Nkx2-1-10', 'SST-10', 'Slc17a7-10', 'Slc32a1-10']\n",
      "Stv_39 -> ['Ccnd2-12', 'Nkx2-1-12', 'SST-12', 'Slc17a7-12', 'Slc32a1-12']\n",
      "Stv_4 -> ['Ccnd2-3', 'Nkx2-1-3', 'SST-3', 'Slc17a7-3', 'Slc32a1-3']\n",
      "Stv_40 -> ['Ccnd2-14', 'Nkx2-1-14', 'SST-14', 'Slc17a7-14', 'Slc32a1-14']\n",
      "Stv_42 -> ['Ccnd2-16', 'Nkx2-1-16', 'SST-16', 'Slc17a7-16', 'Slc32a1-16']\n",
      "Stv_44 -> ['Ccnd2-18', 'Nkx2-1-18', 'SST-18', 'Slc17a7-18', 'Slc32a1-18']\n",
      "Stv_45 -> ['Ccnd2-20', 'Nkx2-1-20', 'SST-20', 'Slc17a7-20', 'Slc32a1-20']\n",
      "Stv_46 -> ['Ccnd2-22', 'Nkx2-1-22', 'SST-22', 'Slc17a7-22']\n",
      "Stv_48 -> ['Ccnd2-24', 'Nkx2-1-24', 'SST-24', 'Slc17a7-24']\n",
      "Stv_5 -> ['Ccnd2-5', 'Nkx2-1-5', 'SST-5', 'Slc17a7-5', 'Slc32a1-5']\n",
      "Stv_50 -> ['Ccnd2-26', 'Nkx2-1-26', 'SST-26', 'Slc17a7-26']\n",
      "Stv_53 -> ['Ccnd2-28', 'Nkx2-1-28', 'SST-28', 'Slc17a7-28']\n",
      "Stv_54 -> ['Ccnd2-30', 'Nkx2-1-30', 'SST-30', 'Slc17a7-30']\n",
      "Stv_59 -> ['Ccnd2-32', 'Nkx2-1-32', 'SST-32', 'Slc17a7-32']\n",
      "Stv_6 -> ['Ccnd2-7', 'Nkx2-1-7', 'SST-7', 'Slc17a7-7', 'Slc32a1-7']\n",
      "Stv_60 -> ['Ccnd2-34', 'Nkx2-1-34', 'SST-34', 'Slc17a7-34']\n",
      "Stv_61 -> ['Ccnd2-36', 'Nkx2-1-36', 'SST-36', 'Slc17a7-36']\n",
      "Stv_63 -> ['Ccnd2-38', 'Nkx2-1-38', 'SST-38', 'Slc17a7-38']\n",
      "Stv_64 -> ['Ccnd2-40', 'Nkx2-1-40', 'SST-40', 'Slc17a7-40']\n",
      "Stv_65 -> ['Ccnd2-42', 'Nkx2-1-42', 'SST-42', 'Slc17a7-42']\n",
      "Stv_7 -> ['Ccnd2-9', 'Nkx2-1-9', 'SST-9', 'Slc17a7-9', 'Slc32a1-9']\n",
      "Stv_8 -> ['Ccnd2-11', 'Nkx2-1-11', 'SST-11', 'Slc17a7-11', 'Slc32a1-11']\n",
      "Stv_86 -> ['Ccnd2-44', 'Nkx2-1-44', 'SST-44', 'Slc17a7-44']\n",
      "Stv_87 -> ['Ccnd2-46', 'Nkx2-1-46', 'SST-46', 'Slc17a7-46']\n",
      "Stv_88 -> ['Ccnd2-48', 'Nkx2-1-48', 'SST-48', 'Slc17a7-48']\n",
      "Stv_9 -> ['Ccnd2-13', 'Nkx2-1-13', 'SST-13', 'Slc17a7-13', 'Slc32a1-13']\n",
      "Stv_90 -> ['Ccnd2-50', 'Nkx2-1-50', 'SST-50', 'Slc17a7-50']\n",
      "Stv_91 -> ['Ccnd2-1', 'Ccnd2-10', 'Ccnd2-11', 'Ccnd2-13', 'Ccnd2-14', 'Ccnd2-15', 'Ccnd2-16', 'Ccnd2-17', 'Ccnd2-18', 'Ccnd2-19', 'Ccnd2-2', 'Ccnd2-20', 'Ccnd2-21', 'Ccnd2-22', 'Ccnd2-23', 'Ccnd2-24', 'Ccnd2-25', 'Ccnd2-26', 'Ccnd2-27', 'Ccnd2-28', 'Ccnd2-29', 'Ccnd2-3', 'Ccnd2-30', 'Ccnd2-31', 'Ccnd2-32', 'Ccnd2-33', 'Ccnd2-34', 'Ccnd2-35', 'Ccnd2-36', 'Ccnd2-37', 'Ccnd2-38', 'Ccnd2-39', 'Ccnd2-4', 'Ccnd2-40', 'Ccnd2-41', 'Ccnd2-42', 'Ccnd2-43', 'Ccnd2-44', 'Ccnd2-45', 'Ccnd2-46', 'Ccnd2-47', 'Ccnd2-48', 'Ccnd2-49', 'Ccnd2-5', 'Ccnd2-50', 'Ccnd2-51', 'Ccnd2-52', 'Ccnd2-53', 'Ccnd2-55', 'Ccnd2-56', 'Ccnd2-57', 'Ccnd2-58', 'Ccnd2-59', 'Ccnd2-6', 'Ccnd2-60', 'Ccnd2-7', 'Ccnd2-8', 'Ccnd2-9']\n",
      "Stv_92 -> ['Nkx2-1-10', 'Nkx2-1-11', 'Nkx2-1-12', 'Nkx2-1-13', 'Nkx2-1-14', 'Nkx2-1-15', 'Nkx2-1-16', 'Nkx2-1-17', 'Nkx2-1-18', 'Nkx2-1-19', 'Nkx2-1-20', 'Nkx2-1-21', 'Nkx2-1-23', 'Nkx2-1-25', 'Nkx2-1-26', 'Nkx2-1-27', 'Nkx2-1-28', 'Nkx2-1-29', 'Nkx2-1-3', 'Nkx2-1-30', 'Nkx2-1-31', 'Nkx2-1-32', 'Nkx2-1-33', 'Nkx2-1-34', 'Nkx2-1-35', 'Nkx2-1-36', 'Nkx2-1-37', 'Nkx2-1-38', 'Nkx2-1-39', 'Nkx2-1-4', 'Nkx2-1-41', 'Nkx2-1-42', 'Nkx2-1-43', 'Nkx2-1-44', 'Nkx2-1-45', 'Nkx2-1-46', 'Nkx2-1-47', 'Nkx2-1-48', 'Nkx2-1-49', 'Nkx2-1-5', 'Nkx2-1-50', 'Nkx2-1-51', 'Nkx2-1-52', 'Nkx2-1-53', 'Nkx2-1-54', 'Nkx2-1-55', 'Nkx2-1-56', 'Nkx2-1-57', 'Nkx2-1-58', 'Nkx2-1-59', 'Nkx2-1-6', 'Nkx2-1-60', 'Nkx2-1-7', 'Nkx2-1-8', 'Nkx2-1-9']\n",
      "Stv_94 -> ['Slc17a7-1', 'Slc17a7-10', 'Slc17a7-11', 'Slc17a7-12', 'Slc17a7-13', 'Slc17a7-14', 'Slc17a7-15', 'Slc17a7-16', 'Slc17a7-17', 'Slc17a7-18', 'Slc17a7-19', 'Slc17a7-2', 'Slc17a7-20', 'Slc17a7-21', 'Slc17a7-22', 'Slc17a7-23', 'Slc17a7-24', 'Slc17a7-25', 'Slc17a7-26', 'Slc17a7-27', 'Slc17a7-28', 'Slc17a7-29', 'Slc17a7-3', 'Slc17a7-30', 'Slc17a7-31', 'Slc17a7-32', 'Slc17a7-33', 'Slc17a7-34', 'Slc17a7-35', 'Slc17a7-36', 'Slc17a7-37', 'Slc17a7-38', 'Slc17a7-39', 'Slc17a7-4', 'Slc17a7-40', 'Slc17a7-41', 'Slc17a7-42', 'Slc17a7-43', 'Slc17a7-44', 'Slc17a7-45', 'Slc17a7-46', 'Slc17a7-47', 'Slc17a7-48', 'Slc17a7-49', 'Slc17a7-5', 'Slc17a7-50', 'Slc17a7-52', 'Slc17a7-53', 'Slc17a7-54', 'Slc17a7-55', 'Slc17a7-56', 'Slc17a7-57', 'Slc17a7-58', 'Slc17a7-59', 'Slc17a7-60', 'Slc17a7-8', 'Slc17a7-9']\n",
      "Stv_95 -> ['Slc32a1-1', 'Slc32a1-10', 'Slc32a1-11', 'Slc32a1-12', 'Slc32a1-13', 'Slc32a1-14', 'Slc32a1-15', 'Slc32a1-16', 'Slc32a1-17', 'Slc32a1-18', 'Slc32a1-19', 'Slc32a1-2', 'Slc32a1-20', 'Slc32a1-21', 'Slc32a1-3', 'Slc32a1-4', 'Slc32a1-5', 'Slc32a1-6', 'Slc32a1-7', 'Slc32a1-8', 'Slc32a1-9']\n",
      "Stv_99 -> ['SST-1', 'SST-10', 'SST-11', 'SST-12', 'SST-14', 'SST-15', 'SST-16', 'SST-17', 'SST-18', 'SST-19', 'SST-20', 'SST-21', 'SST-23', 'SST-24', 'SST-25', 'SST-26', 'SST-27', 'SST-28', 'SST-29', 'SST-30', 'SST-31', 'SST-32', 'SST-33', 'SST-34', 'SST-35', 'SST-36', 'SST-37', 'SST-38', 'SST-39', 'SST-4', 'SST-40', 'SST-41', 'SST-42', 'SST-43', 'SST-44', 'SST-45', 'SST-46', 'SST-47', 'SST-48', 'SST-49', 'SST-5', 'SST-50', 'SST-51', 'SST-52', 'SST-53', 'SST-54', 'SST-55', 'SST-56', 'SST-57', 'SST-58', 'SST-59', 'SST-6', 'SST-60', 'SST-7', 'SST-8', 'SST-9']\n"
     ]
    }
   ],
   "source": [
    "readout_reg_dic, readout2reg_check = check._check_readout_to_region(reg_readout_dic, full_records, \n",
    "                                                                    updated_readout_dict, primer_len=primer_len, \n",
    "                                                                    probe_readout_num=num_readout_per_probe)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 249,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "True True False True\n"
     ]
    }
   ],
   "source": [
    "print(primer_check, len_check, reg2readout_check, readout2reg_check)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 254,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- saving internal 17-mer map to file:\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\probe_table_17.npz\n",
      "True\n"
     ]
    }
   ],
   "source": [
    "# construct internal map\n",
    "int_map = check._construct_internal_map(full_records, library_folder) \n",
    "# internal check probes\n",
    "readout_count_dic, readout_count_check = check._check_readout_in_probes(readout_reg_dic, reg_size_dic, int_map, readout_dict)\n",
    "print(readout_count_check)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 257,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- total probes removed by internal screening: 0\n"
     ]
    }
   ],
   "source": [
    "# check between probes\n",
    "kept_records, removed_count = check._check_between_probes(full_records, int_map, primer_len=primer_len)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 258,
   "metadata": {},
   "outputs": [],
   "source": [
    "# save kept records\n",
    "with open(os.path.join(library_folder, 'filtered_full_probes.fasta'), 'w') as output_handle:\n",
    "    SeqIO.write(kept_records, output_handle, \"fasta\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 4.2 BLAST screening of probes\n",
    "\n",
    "create the local blast database by:\n",
    "```bash\n",
    "makeblastdb -in C:\\Users\\puzheng\\Documents\\Tools\\lowercase_masked\\Mus_musculus.GRCm38.dna_sm.merged.fa -dbtype nucl -parse_seqids -out GRCm38_ensembl -title \"Mouse chromosomes soft masked\" -blastdb_version 4![image.png](attachment:image.png)\n",
    "```\n",
    "\n",
    "The outputs are:\n",
    "```bash\n",
    "Building a new DB, current time: 03/30/2021 21:49:18\n",
    "New DB name:   C:\\Users\\puzheng\\Documents\\Tools\\GRCm38_ensembl\n",
    "New DB title:  Mouse chromosomes soft masked\n",
    "Sequence type: Nucleotide\n",
    "Keep MBits: T\n",
    "Maximum file size: 1000000000B\n",
    "Adding sequences from FASTA; added 66 sequences in 42.2668 seconds.\n",
    "```"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 259,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Blast probe against \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\blast_database\\GRCm38_ensembl\n",
      "-- number of probes loaded: 32432\n",
      "--- processing region: Ccnd2-1 with 135 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-1.fasta\n",
      "-- *blasting region: Ccnd2-1\n",
      "--- total time for blast Ccnd2-1: 9.974999189376831\n",
      "--- processing region: Ccnd2-10 with 136 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-10.fasta\n",
      "-- *blasting region: Ccnd2-10\n",
      "--- total time for blast Ccnd2-10: 9.420455694198608\n",
      "--- processing region: Ccnd2-11 with 139 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-11.fasta\n",
      "-- *blasting region: Ccnd2-11\n",
      "--- total time for blast Ccnd2-11: 9.010926723480225\n",
      "--- processing region: Ccnd2-12 with 62 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-12.fasta\n",
      "-- *blasting region: Ccnd2-12\n",
      "--- total time for blast Ccnd2-12: 5.5778586864471436\n",
      "--- processing region: Ccnd2-13 with 127 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-13.fasta\n",
      "-- *blasting region: Ccnd2-13\n",
      "--- total time for blast Ccnd2-13: 7.946595907211304\n",
      "--- processing region: Ccnd2-14 with 117 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-14.fasta\n",
      "-- *blasting region: Ccnd2-14\n",
      "--- total time for blast Ccnd2-14: 11.151198148727417\n",
      "--- processing region: Ccnd2-15 with 125 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-15.fasta\n",
      "-- *blasting region: Ccnd2-15\n",
      "--- total time for blast Ccnd2-15: 8.57679033279419\n",
      "--- processing region: Ccnd2-16 with 136 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-16.fasta\n",
      "-- *blasting region: Ccnd2-16\n",
      "--- total time for blast Ccnd2-16: 9.557434797286987\n",
      "--- processing region: Ccnd2-17 with 147 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-17.fasta\n",
      "-- *blasting region: Ccnd2-17\n",
      "--- total time for blast Ccnd2-17: 9.406280040740967\n",
      "--- processing region: Ccnd2-18 with 146 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-18.fasta\n",
      "-- *blasting region: Ccnd2-18\n",
      "--- total time for blast Ccnd2-18: 7.063315391540527\n",
      "--- processing region: Ccnd2-19 with 143 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-19.fasta\n",
      "-- *blasting region: Ccnd2-19\n",
      "--- total time for blast Ccnd2-19: 11.173682689666748\n",
      "--- processing region: Ccnd2-2 with 99 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-2.fasta\n",
      "-- *blasting region: Ccnd2-2\n",
      "--- total time for blast Ccnd2-2: 9.133249044418335\n",
      "--- processing region: Ccnd2-20 with 142 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-20.fasta\n",
      "-- *blasting region: Ccnd2-20\n",
      "--- total time for blast Ccnd2-20: 9.517035007476807\n",
      "--- processing region: Ccnd2-21 with 139 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-21.fasta\n",
      "-- *blasting region: Ccnd2-21\n",
      "--- total time for blast Ccnd2-21: 7.388776540756226\n",
      "--- processing region: Ccnd2-22 with 126 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-22.fasta\n",
      "-- *blasting region: Ccnd2-22\n",
      "--- total time for blast Ccnd2-22: 8.544465780258179\n",
      "--- processing region: Ccnd2-23 with 138 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-23.fasta\n",
      "-- *blasting region: Ccnd2-23\n",
      "--- total time for blast Ccnd2-23: 8.730704545974731\n",
      "--- processing region: Ccnd2-24 with 136 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-24.fasta\n",
      "-- *blasting region: Ccnd2-24\n",
      "--- total time for blast Ccnd2-24: 8.964593887329102\n",
      "--- processing region: Ccnd2-25 with 128 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-25.fasta\n",
      "-- *blasting region: Ccnd2-25\n",
      "--- total time for blast Ccnd2-25: 9.651612758636475\n",
      "--- processing region: Ccnd2-26 with 101 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-26.fasta\n",
      "-- *blasting region: Ccnd2-26\n",
      "--- total time for blast Ccnd2-26: 8.951801776885986\n",
      "--- processing region: Ccnd2-27 with 139 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-27.fasta\n",
      "-- *blasting region: Ccnd2-27\n",
      "--- total time for blast Ccnd2-27: 9.368527889251709\n",
      "--- processing region: Ccnd2-28 with 128 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-28.fasta\n",
      "-- *blasting region: Ccnd2-28\n",
      "--- total time for blast Ccnd2-28: 8.204753875732422\n",
      "--- processing region: Ccnd2-29 with 145 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-29.fasta\n",
      "-- *blasting region: Ccnd2-29\n",
      "--- total time for blast Ccnd2-29: 8.89605975151062\n",
      "--- processing region: Ccnd2-3 with 104 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-3.fasta\n",
      "-- *blasting region: Ccnd2-3\n",
      "--- total time for blast Ccnd2-3: 8.175504446029663\n",
      "--- processing region: Ccnd2-30 with 125 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-30.fasta\n",
      "-- *blasting region: Ccnd2-30\n",
      "--- total time for blast Ccnd2-30: 8.05381989479065\n",
      "--- processing region: Ccnd2-31 with 124 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-31.fasta\n",
      "-- *blasting region: Ccnd2-31\n",
      "--- total time for blast Ccnd2-31: 7.696232318878174\n",
      "--- processing region: Ccnd2-32 with 135 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-32.fasta\n",
      "-- *blasting region: Ccnd2-32\n",
      "--- total time for blast Ccnd2-32: 10.143283367156982\n",
      "--- processing region: Ccnd2-33 with 127 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-33.fasta\n",
      "-- *blasting region: Ccnd2-33\n",
      "--- total time for blast Ccnd2-33: 8.668092489242554\n",
      "--- processing region: Ccnd2-34 with 133 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-34.fasta\n",
      "-- *blasting region: Ccnd2-34\n",
      "--- total time for blast Ccnd2-34: 9.594848155975342\n",
      "--- processing region: Ccnd2-35 with 149 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-35.fasta\n",
      "-- *blasting region: Ccnd2-35\n",
      "--- total time for blast Ccnd2-35: 10.574043273925781\n",
      "--- processing region: Ccnd2-36 with 138 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-36.fasta\n",
      "-- *blasting region: Ccnd2-36\n",
      "--- total time for blast Ccnd2-36: 9.266704082489014\n",
      "--- processing region: Ccnd2-37 with 118 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-37.fasta\n",
      "-- *blasting region: Ccnd2-37\n",
      "--- total time for blast Ccnd2-37: 7.349963426589966\n",
      "--- processing region: Ccnd2-38 with 145 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-38.fasta\n",
      "-- *blasting region: Ccnd2-38\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Ccnd2-38: 8.778576135635376\n",
      "--- processing region: Ccnd2-39 with 119 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-39.fasta\n",
      "-- *blasting region: Ccnd2-39\n",
      "--- total time for blast Ccnd2-39: 9.420511245727539\n",
      "--- processing region: Ccnd2-4 with 149 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-4.fasta\n",
      "-- *blasting region: Ccnd2-4\n",
      "--- total time for blast Ccnd2-4: 8.989366292953491\n",
      "--- processing region: Ccnd2-40 with 135 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-40.fasta\n",
      "-- *blasting region: Ccnd2-40\n",
      "--- total time for blast Ccnd2-40: 9.973755836486816\n",
      "--- processing region: Ccnd2-41 with 124 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-41.fasta\n",
      "-- *blasting region: Ccnd2-41\n",
      "--- total time for blast Ccnd2-41: 8.335584878921509\n",
      "--- processing region: Ccnd2-42 with 117 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-42.fasta\n",
      "-- *blasting region: Ccnd2-42\n",
      "--- total time for blast Ccnd2-42: 10.34399127960205\n",
      "--- processing region: Ccnd2-43 with 126 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-43.fasta\n",
      "-- *blasting region: Ccnd2-43\n",
      "--- total time for blast Ccnd2-43: 9.087300300598145\n",
      "--- processing region: Ccnd2-44 with 128 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-44.fasta\n",
      "-- *blasting region: Ccnd2-44\n",
      "--- total time for blast Ccnd2-44: 9.258533477783203\n",
      "--- processing region: Ccnd2-45 with 128 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-45.fasta\n",
      "-- *blasting region: Ccnd2-45\n",
      "--- total time for blast Ccnd2-45: 9.295217275619507\n",
      "--- processing region: Ccnd2-46 with 130 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-46.fasta\n",
      "-- *blasting region: Ccnd2-46\n",
      "--- total time for blast Ccnd2-46: 9.177222967147827\n",
      "--- processing region: Ccnd2-47 with 142 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-47.fasta\n",
      "-- *blasting region: Ccnd2-47\n",
      "--- total time for blast Ccnd2-47: 10.833421468734741\n",
      "--- processing region: Ccnd2-48 with 135 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-48.fasta\n",
      "-- *blasting region: Ccnd2-48\n",
      "--- total time for blast Ccnd2-48: 11.462846279144287\n",
      "--- processing region: Ccnd2-49 with 128 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-49.fasta\n",
      "-- *blasting region: Ccnd2-49\n",
      "--- total time for blast Ccnd2-49: 9.508288621902466\n",
      "--- processing region: Ccnd2-5 with 152 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-5.fasta\n",
      "-- *blasting region: Ccnd2-5\n",
      "--- total time for blast Ccnd2-5: 8.763738870620728\n",
      "--- processing region: Ccnd2-50 with 133 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-50.fasta\n",
      "-- *blasting region: Ccnd2-50\n",
      "--- total time for blast Ccnd2-50: 9.69798493385315\n",
      "--- processing region: Ccnd2-51 with 99 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-51.fasta\n",
      "-- *blasting region: Ccnd2-51\n",
      "--- total time for blast Ccnd2-51: 7.558862924575806\n",
      "--- processing region: Ccnd2-52 with 136 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-52.fasta\n",
      "-- *blasting region: Ccnd2-52\n",
      "--- total time for blast Ccnd2-52: 9.661570310592651\n",
      "--- processing region: Ccnd2-53 with 132 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-53.fasta\n",
      "-- *blasting region: Ccnd2-53\n",
      "--- total time for blast Ccnd2-53: 9.25652027130127\n",
      "--- processing region: Ccnd2-54 with 34 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-54.fasta\n",
      "-- *blasting region: Ccnd2-54\n",
      "--- total time for blast Ccnd2-54: 4.1911845207214355\n",
      "--- processing region: Ccnd2-55 with 101 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-55.fasta\n",
      "-- *blasting region: Ccnd2-55\n",
      "--- total time for blast Ccnd2-55: 9.2459876537323\n",
      "--- processing region: Ccnd2-56 with 116 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-56.fasta\n",
      "-- *blasting region: Ccnd2-56\n",
      "--- total time for blast Ccnd2-56: 9.043564796447754\n",
      "--- processing region: Ccnd2-57 with 134 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-57.fasta\n",
      "-- *blasting region: Ccnd2-57\n",
      "--- total time for blast Ccnd2-57: 7.844364166259766\n",
      "--- processing region: Ccnd2-58 with 142 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-58.fasta\n",
      "-- *blasting region: Ccnd2-58\n",
      "--- total time for blast Ccnd2-58: 10.046679019927979\n",
      "--- processing region: Ccnd2-59 with 147 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-59.fasta\n",
      "-- *blasting region: Ccnd2-59\n",
      "--- total time for blast Ccnd2-59: 9.398969650268555\n",
      "--- processing region: Ccnd2-6 with 112 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-6.fasta\n",
      "-- *blasting region: Ccnd2-6\n",
      "--- total time for blast Ccnd2-6: 7.232837438583374\n",
      "--- processing region: Ccnd2-60 with 143 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-60.fasta\n",
      "-- *blasting region: Ccnd2-60\n",
      "--- total time for blast Ccnd2-60: 8.870467185974121\n",
      "--- processing region: Ccnd2-7 with 136 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-7.fasta\n",
      "-- *blasting region: Ccnd2-7\n",
      "--- total time for blast Ccnd2-7: 9.06948709487915\n",
      "--- processing region: Ccnd2-8 with 140 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-8.fasta\n",
      "-- *blasting region: Ccnd2-8\n",
      "--- total time for blast Ccnd2-8: 8.073832035064697\n",
      "--- processing region: Ccnd2-9 with 134 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Ccnd2-9.fasta\n",
      "-- *blasting region: Ccnd2-9\n",
      "--- total time for blast Ccnd2-9: 9.571076393127441\n",
      "--- processing region: Nkx2-1-1 with 79 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-1.fasta\n",
      "-- *blasting region: Nkx2-1-1\n",
      "--- total time for blast Nkx2-1-1: 7.617678880691528\n",
      "--- processing region: Nkx2-1-10 with 129 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-10.fasta\n",
      "-- *blasting region: Nkx2-1-10\n",
      "--- total time for blast Nkx2-1-10: 9.23909330368042\n",
      "--- processing region: Nkx2-1-11 with 128 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-11.fasta\n",
      "-- *blasting region: Nkx2-1-11\n",
      "--- total time for blast Nkx2-1-11: 8.987948894500732\n",
      "--- processing region: Nkx2-1-12 with 118 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-12.fasta\n",
      "-- *blasting region: Nkx2-1-12\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Nkx2-1-12: 9.640823602676392\n",
      "--- processing region: Nkx2-1-13 with 134 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-13.fasta\n",
      "-- *blasting region: Nkx2-1-13\n",
      "--- total time for blast Nkx2-1-13: 9.032560586929321\n",
      "--- processing region: Nkx2-1-14 with 129 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-14.fasta\n",
      "-- *blasting region: Nkx2-1-14\n",
      "--- total time for blast Nkx2-1-14: 7.930529356002808\n",
      "--- processing region: Nkx2-1-15 with 129 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-15.fasta\n",
      "-- *blasting region: Nkx2-1-15\n",
      "--- total time for blast Nkx2-1-15: 8.215431213378906\n",
      "--- processing region: Nkx2-1-16 with 125 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-16.fasta\n",
      "-- *blasting region: Nkx2-1-16\n",
      "--- total time for blast Nkx2-1-16: 7.903391361236572\n",
      "--- processing region: Nkx2-1-17 with 138 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-17.fasta\n",
      "-- *blasting region: Nkx2-1-17\n",
      "--- total time for blast Nkx2-1-17: 10.239591598510742\n",
      "--- processing region: Nkx2-1-18 with 146 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-18.fasta\n",
      "-- *blasting region: Nkx2-1-18\n",
      "--- total time for blast Nkx2-1-18: 8.945417404174805\n",
      "--- processing region: Nkx2-1-19 with 153 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-19.fasta\n",
      "-- *blasting region: Nkx2-1-19\n",
      "--- total time for blast Nkx2-1-19: 8.174898386001587\n",
      "--- processing region: Nkx2-1-2 with 62 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-2.fasta\n",
      "-- *blasting region: Nkx2-1-2\n",
      "--- total time for blast Nkx2-1-2: 5.800440073013306\n",
      "--- processing region: Nkx2-1-20 with 123 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-20.fasta\n",
      "-- *blasting region: Nkx2-1-20\n",
      "--- total time for blast Nkx2-1-20: 13.128427743911743\n",
      "--- processing region: Nkx2-1-21 with 100 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-21.fasta\n",
      "-- *blasting region: Nkx2-1-21\n",
      "--- total time for blast Nkx2-1-21: 16.175859928131104\n",
      "--- processing region: Nkx2-1-22 with 53 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-22.fasta\n",
      "-- *blasting region: Nkx2-1-22\n",
      "--- total time for blast Nkx2-1-22: 6.313684463500977\n",
      "--- processing region: Nkx2-1-23 with 87 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-23.fasta\n",
      "-- *blasting region: Nkx2-1-23\n",
      "--- total time for blast Nkx2-1-23: 6.408446788787842\n",
      "--- processing region: Nkx2-1-24 with 76 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-24.fasta\n",
      "-- *blasting region: Nkx2-1-24\n",
      "--- total time for blast Nkx2-1-24: 5.750730991363525\n",
      "--- processing region: Nkx2-1-25 with 111 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-25.fasta\n",
      "-- *blasting region: Nkx2-1-25\n",
      "--- total time for blast Nkx2-1-25: 8.270111322402954\n",
      "--- processing region: Nkx2-1-26 with 121 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-26.fasta\n",
      "-- *blasting region: Nkx2-1-26\n",
      "--- total time for blast Nkx2-1-26: 9.886419296264648\n",
      "--- processing region: Nkx2-1-27 with 144 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-27.fasta\n",
      "-- *blasting region: Nkx2-1-27\n",
      "--- total time for blast Nkx2-1-27: 8.304961442947388\n",
      "--- processing region: Nkx2-1-28 with 116 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-28.fasta\n",
      "-- *blasting region: Nkx2-1-28\n",
      "--- total time for blast Nkx2-1-28: 8.337033987045288\n",
      "--- processing region: Nkx2-1-29 with 141 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-29.fasta\n",
      "-- *blasting region: Nkx2-1-29\n",
      "--- total time for blast Nkx2-1-29: 10.601664543151855\n",
      "--- processing region: Nkx2-1-3 with 86 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-3.fasta\n",
      "-- *blasting region: Nkx2-1-3\n",
      "--- total time for blast Nkx2-1-3: 5.837124586105347\n",
      "--- processing region: Nkx2-1-30 with 125 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-30.fasta\n",
      "-- *blasting region: Nkx2-1-30\n",
      "--- total time for blast Nkx2-1-30: 9.937474727630615\n",
      "--- processing region: Nkx2-1-31 with 94 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-31.fasta\n",
      "-- *blasting region: Nkx2-1-31\n",
      "--- total time for blast Nkx2-1-31: 10.430240154266357\n",
      "--- processing region: Nkx2-1-32 with 102 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-32.fasta\n",
      "-- *blasting region: Nkx2-1-32\n",
      "--- total time for blast Nkx2-1-32: 8.777730464935303\n",
      "--- processing region: Nkx2-1-33 with 135 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-33.fasta\n",
      "-- *blasting region: Nkx2-1-33\n",
      "--- total time for blast Nkx2-1-33: 9.39578652381897\n",
      "--- processing region: Nkx2-1-34 with 149 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-34.fasta\n",
      "-- *blasting region: Nkx2-1-34\n",
      "--- total time for blast Nkx2-1-34: 8.269207239151001\n",
      "--- processing region: Nkx2-1-35 with 154 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-35.fasta\n",
      "-- *blasting region: Nkx2-1-35\n",
      "--- total time for blast Nkx2-1-35: 8.136078596115112\n",
      "--- processing region: Nkx2-1-36 with 135 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-36.fasta\n",
      "-- *blasting region: Nkx2-1-36\n",
      "--- total time for blast Nkx2-1-36: 10.50989294052124\n",
      "--- processing region: Nkx2-1-37 with 133 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-37.fasta\n",
      "-- *blasting region: Nkx2-1-37\n",
      "--- total time for blast Nkx2-1-37: 9.114640474319458\n",
      "--- processing region: Nkx2-1-38 with 130 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-38.fasta\n",
      "-- *blasting region: Nkx2-1-38\n",
      "--- total time for blast Nkx2-1-38: 8.436733961105347\n",
      "--- processing region: Nkx2-1-39 with 125 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-39.fasta\n",
      "-- *blasting region: Nkx2-1-39\n",
      "--- total time for blast Nkx2-1-39: 7.94460654258728\n",
      "--- processing region: Nkx2-1-4 with 137 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-4.fasta\n",
      "-- *blasting region: Nkx2-1-4\n",
      "--- total time for blast Nkx2-1-4: 9.917839765548706\n",
      "--- processing region: Nkx2-1-40 with 58 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-40.fasta\n",
      "-- *blasting region: Nkx2-1-40\n",
      "--- total time for blast Nkx2-1-40: 8.99744701385498\n",
      "--- processing region: Nkx2-1-41 with 101 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-41.fasta\n",
      "-- *blasting region: Nkx2-1-41\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Nkx2-1-41: 7.263504505157471\n",
      "--- processing region: Nkx2-1-42 with 118 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-42.fasta\n",
      "-- *blasting region: Nkx2-1-42\n",
      "--- total time for blast Nkx2-1-42: 12.518338918685913\n",
      "--- processing region: Nkx2-1-43 with 123 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-43.fasta\n",
      "-- *blasting region: Nkx2-1-43\n",
      "--- total time for blast Nkx2-1-43: 8.83612060546875\n",
      "--- processing region: Nkx2-1-44 with 112 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-44.fasta\n",
      "-- *blasting region: Nkx2-1-44\n",
      "--- total time for blast Nkx2-1-44: 9.259239912033081\n",
      "--- processing region: Nkx2-1-45 with 133 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-45.fasta\n",
      "-- *blasting region: Nkx2-1-45\n",
      "--- total time for blast Nkx2-1-45: 9.070887804031372\n",
      "--- processing region: Nkx2-1-46 with 138 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-46.fasta\n",
      "-- *blasting region: Nkx2-1-46\n",
      "--- total time for blast Nkx2-1-46: 10.245638370513916\n",
      "--- processing region: Nkx2-1-47 with 130 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-47.fasta\n",
      "-- *blasting region: Nkx2-1-47\n",
      "--- total time for blast Nkx2-1-47: 9.192774057388306\n",
      "--- processing region: Nkx2-1-48 with 132 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-48.fasta\n",
      "-- *blasting region: Nkx2-1-48\n",
      "--- total time for blast Nkx2-1-48: 9.758150100708008\n",
      "--- processing region: Nkx2-1-49 with 100 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-49.fasta\n",
      "-- *blasting region: Nkx2-1-49\n",
      "--- total time for blast Nkx2-1-49: 14.354262590408325\n",
      "--- processing region: Nkx2-1-5 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-5.fasta\n",
      "-- *blasting region: Nkx2-1-5\n",
      "--- total time for blast Nkx2-1-5: 7.830253601074219\n",
      "--- processing region: Nkx2-1-50 with 90 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-50.fasta\n",
      "-- *blasting region: Nkx2-1-50\n",
      "--- total time for blast Nkx2-1-50: 7.606117010116577\n",
      "--- processing region: Nkx2-1-51 with 122 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-51.fasta\n",
      "-- *blasting region: Nkx2-1-51\n",
      "--- total time for blast Nkx2-1-51: 7.942934274673462\n",
      "--- processing region: Nkx2-1-52 with 139 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-52.fasta\n",
      "-- *blasting region: Nkx2-1-52\n",
      "--- total time for blast Nkx2-1-52: 14.067299842834473\n",
      "--- processing region: Nkx2-1-53 with 123 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-53.fasta\n",
      "-- *blasting region: Nkx2-1-53\n",
      "--- total time for blast Nkx2-1-53: 7.570263147354126\n",
      "--- processing region: Nkx2-1-54 with 128 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-54.fasta\n",
      "-- *blasting region: Nkx2-1-54\n",
      "--- total time for blast Nkx2-1-54: 8.946087121963501\n",
      "--- processing region: Nkx2-1-55 with 138 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-55.fasta\n",
      "-- *blasting region: Nkx2-1-55\n",
      "--- total time for blast Nkx2-1-55: 10.623014450073242\n",
      "--- processing region: Nkx2-1-56 with 137 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-56.fasta\n",
      "-- *blasting region: Nkx2-1-56\n",
      "--- total time for blast Nkx2-1-56: 9.460401773452759\n",
      "--- processing region: Nkx2-1-57 with 100 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-57.fasta\n",
      "-- *blasting region: Nkx2-1-57\n",
      "--- total time for blast Nkx2-1-57: 6.89188551902771\n",
      "--- processing region: Nkx2-1-58 with 122 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-58.fasta\n",
      "-- *blasting region: Nkx2-1-58\n",
      "--- total time for blast Nkx2-1-58: 9.914610147476196\n",
      "--- processing region: Nkx2-1-59 with 125 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-59.fasta\n",
      "-- *blasting region: Nkx2-1-59\n",
      "--- total time for blast Nkx2-1-59: 9.111568212509155\n",
      "--- processing region: Nkx2-1-6 with 141 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-6.fasta\n",
      "-- *blasting region: Nkx2-1-6\n",
      "--- total time for blast Nkx2-1-6: 7.7545342445373535\n",
      "--- processing region: Nkx2-1-60 with 86 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-60.fasta\n",
      "-- *blasting region: Nkx2-1-60\n",
      "--- total time for blast Nkx2-1-60: 8.818785429000854\n",
      "--- processing region: Nkx2-1-7 with 138 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-7.fasta\n",
      "-- *blasting region: Nkx2-1-7\n",
      "--- total time for blast Nkx2-1-7: 8.665974140167236\n",
      "--- processing region: Nkx2-1-8 with 118 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-8.fasta\n",
      "-- *blasting region: Nkx2-1-8\n",
      "--- total time for blast Nkx2-1-8: 7.276771306991577\n",
      "--- processing region: Nkx2-1-9 with 126 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Nkx2-1-9.fasta\n",
      "-- *blasting region: Nkx2-1-9\n",
      "--- total time for blast Nkx2-1-9: 8.782130002975464\n",
      "--- processing region: SST-1 with 105 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-1.fasta\n",
      "-- *blasting region: SST-1\n",
      "--- total time for blast SST-1: 8.831598997116089\n",
      "--- processing region: SST-10 with 141 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-10.fasta\n",
      "-- *blasting region: SST-10\n",
      "--- total time for blast SST-10: 9.597867488861084\n",
      "--- processing region: SST-11 with 145 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-11.fasta\n",
      "-- *blasting region: SST-11\n",
      "--- total time for blast SST-11: 7.665231227874756\n",
      "--- processing region: SST-12 with 141 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-12.fasta\n",
      "-- *blasting region: SST-12\n",
      "--- total time for blast SST-12: 11.090912818908691\n",
      "--- processing region: SST-13 with 79 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-13.fasta\n",
      "-- *blasting region: SST-13\n",
      "--- total time for blast SST-13: 7.446819543838501\n",
      "--- processing region: SST-14 with 95 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-14.fasta\n",
      "-- *blasting region: SST-14\n",
      "--- total time for blast SST-14: 11.390736103057861\n",
      "--- processing region: SST-15 with 82 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-15.fasta\n",
      "-- *blasting region: SST-15\n",
      "--- total time for blast SST-15: 9.675102472305298\n",
      "--- processing region: SST-16 with 131 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-16.fasta\n",
      "-- *blasting region: SST-16\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast SST-16: 8.865233898162842\n",
      "--- processing region: SST-17 with 151 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-17.fasta\n",
      "-- *blasting region: SST-17\n",
      "--- total time for blast SST-17: 10.462542533874512\n",
      "--- processing region: SST-18 with 96 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-18.fasta\n",
      "-- *blasting region: SST-18\n",
      "--- total time for blast SST-18: 17.58848810195923\n",
      "--- processing region: SST-19 with 114 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-19.fasta\n",
      "-- *blasting region: SST-19\n",
      "--- total time for blast SST-19: 10.00341510772705\n",
      "--- processing region: SST-2 with 49 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-2.fasta\n",
      "-- *blasting region: SST-2\n",
      "--- total time for blast SST-2: 5.596716403961182\n",
      "--- processing region: SST-20 with 114 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-20.fasta\n",
      "-- *blasting region: SST-20\n",
      "--- total time for blast SST-20: 9.412886381149292\n",
      "--- processing region: SST-21 with 103 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-21.fasta\n",
      "-- *blasting region: SST-21\n",
      "--- total time for blast SST-21: 6.996670722961426\n",
      "--- processing region: SST-22 with 61 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-22.fasta\n",
      "-- *blasting region: SST-22\n",
      "--- total time for blast SST-22: 5.860093116760254\n",
      "--- processing region: SST-23 with 112 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-23.fasta\n",
      "-- *blasting region: SST-23\n",
      "--- total time for blast SST-23: 7.843860149383545\n",
      "--- processing region: SST-24 with 148 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-24.fasta\n",
      "-- *blasting region: SST-24\n",
      "--- total time for blast SST-24: 8.83679485321045\n",
      "--- processing region: SST-25 with 140 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-25.fasta\n",
      "-- *blasting region: SST-25\n",
      "--- total time for blast SST-25: 8.782823324203491\n",
      "--- processing region: SST-26 with 151 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-26.fasta\n",
      "-- *blasting region: SST-26\n",
      "--- total time for blast SST-26: 7.256294250488281\n",
      "--- processing region: SST-27 with 141 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-27.fasta\n",
      "-- *blasting region: SST-27\n",
      "--- total time for blast SST-27: 7.961184978485107\n",
      "--- processing region: SST-28 with 137 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-28.fasta\n",
      "-- *blasting region: SST-28\n",
      "--- total time for blast SST-28: 8.642737627029419\n",
      "--- processing region: SST-29 with 124 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-29.fasta\n",
      "-- *blasting region: SST-29\n",
      "--- total time for blast SST-29: 7.8501200675964355\n",
      "--- processing region: SST-3 with 53 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-3.fasta\n",
      "-- *blasting region: SST-3\n",
      "--- total time for blast SST-3: 6.551663160324097\n",
      "--- processing region: SST-30 with 141 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-30.fasta\n",
      "-- *blasting region: SST-30\n",
      "--- total time for blast SST-30: 7.748406410217285\n",
      "--- processing region: SST-31 with 108 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-31.fasta\n",
      "-- *blasting region: SST-31\n",
      "--- total time for blast SST-31: 8.59980058670044\n",
      "--- processing region: SST-32 with 139 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-32.fasta\n",
      "-- *blasting region: SST-32\n",
      "--- total time for blast SST-32: 16.756877899169922\n",
      "--- processing region: SST-33 with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-33.fasta\n",
      "-- *blasting region: SST-33\n",
      "--- total time for blast SST-33: 10.776965379714966\n",
      "--- processing region: SST-34 with 148 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-34.fasta\n",
      "-- *blasting region: SST-34\n",
      "--- total time for blast SST-34: 9.220540285110474\n",
      "--- processing region: SST-35 with 141 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-35.fasta\n",
      "-- *blasting region: SST-35\n",
      "--- total time for blast SST-35: 12.067551851272583\n",
      "--- processing region: SST-36 with 144 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-36.fasta\n",
      "-- *blasting region: SST-36\n",
      "--- total time for blast SST-36: 12.363521814346313\n",
      "--- processing region: SST-37 with 147 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-37.fasta\n",
      "-- *blasting region: SST-37\n",
      "--- total time for blast SST-37: 10.356110095977783\n",
      "--- processing region: SST-38 with 114 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-38.fasta\n",
      "-- *blasting region: SST-38\n",
      "--- total time for blast SST-38: 8.421722412109375\n",
      "--- processing region: SST-39 with 112 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-39.fasta\n",
      "-- *blasting region: SST-39\n",
      "--- total time for blast SST-39: 8.790127992630005\n",
      "--- processing region: SST-4 with 118 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-4.fasta\n",
      "-- *blasting region: SST-4\n",
      "--- total time for blast SST-4: 8.642899990081787\n",
      "--- processing region: SST-40 with 135 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-40.fasta\n",
      "-- *blasting region: SST-40\n",
      "--- total time for blast SST-40: 11.857863664627075\n",
      "--- processing region: SST-41 with 147 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-41.fasta\n",
      "-- *blasting region: SST-41\n",
      "--- total time for blast SST-41: 15.655989408493042\n",
      "--- processing region: SST-42 with 141 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-42.fasta\n",
      "-- *blasting region: SST-42\n",
      "--- total time for blast SST-42: 13.208593368530273\n",
      "--- processing region: SST-43 with 139 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-43.fasta\n",
      "-- *blasting region: SST-43\n",
      "--- total time for blast SST-43: 9.569838762283325\n",
      "--- processing region: SST-44 with 146 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-44.fasta\n",
      "-- *blasting region: SST-44\n",
      "--- total time for blast SST-44: 9.615945100784302\n",
      "--- processing region: SST-45 with 139 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-45.fasta\n",
      "-- *blasting region: SST-45\n",
      "--- total time for blast SST-45: 7.508138656616211\n",
      "--- processing region: SST-46 with 138 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-46.fasta\n",
      "-- *blasting region: SST-46\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast SST-46: 10.263249158859253\n",
      "--- processing region: SST-47 with 130 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-47.fasta\n",
      "-- *blasting region: SST-47\n",
      "--- total time for blast SST-47: 9.985980033874512\n",
      "--- processing region: SST-48 with 160 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-48.fasta\n",
      "-- *blasting region: SST-48\n",
      "--- total time for blast SST-48: 12.421011686325073\n",
      "--- processing region: SST-49 with 151 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-49.fasta\n",
      "-- *blasting region: SST-49\n",
      "--- total time for blast SST-49: 8.286566019058228\n",
      "--- processing region: SST-5 with 114 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-5.fasta\n",
      "-- *blasting region: SST-5\n",
      "--- total time for blast SST-5: 9.73773717880249\n",
      "--- processing region: SST-50 with 135 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-50.fasta\n",
      "-- *blasting region: SST-50\n",
      "--- total time for blast SST-50: 10.642311096191406\n",
      "--- processing region: SST-51 with 147 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-51.fasta\n",
      "-- *blasting region: SST-51\n",
      "--- total time for blast SST-51: 9.989386320114136\n",
      "--- processing region: SST-52 with 116 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-52.fasta\n",
      "-- *blasting region: SST-52\n",
      "--- total time for blast SST-52: 11.083113431930542\n",
      "--- processing region: SST-53 with 124 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-53.fasta\n",
      "-- *blasting region: SST-53\n",
      "--- total time for blast SST-53: 8.197234392166138\n",
      "--- processing region: SST-54 with 101 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-54.fasta\n",
      "-- *blasting region: SST-54\n",
      "--- total time for blast SST-54: 8.429495573043823\n",
      "--- processing region: SST-55 with 122 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-55.fasta\n",
      "-- *blasting region: SST-55\n",
      "--- total time for blast SST-55: 10.186382055282593\n",
      "--- processing region: SST-56 with 116 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-56.fasta\n",
      "-- *blasting region: SST-56\n",
      "--- total time for blast SST-56: 10.357298135757446\n",
      "--- processing region: SST-57 with 117 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-57.fasta\n",
      "-- *blasting region: SST-57\n",
      "--- total time for blast SST-57: 9.210790634155273\n",
      "--- processing region: SST-58 with 108 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-58.fasta\n",
      "-- *blasting region: SST-58\n",
      "--- total time for blast SST-58: 14.853931427001953\n",
      "--- processing region: SST-59 with 134 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-59.fasta\n",
      "-- *blasting region: SST-59\n",
      "--- total time for blast SST-59: 9.459715604782104\n",
      "--- processing region: SST-6 with 149 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-6.fasta\n",
      "-- *blasting region: SST-6\n",
      "--- total time for blast SST-6: 8.077598333358765\n",
      "--- processing region: SST-60 with 113 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-60.fasta\n",
      "-- *blasting region: SST-60\n",
      "--- total time for blast SST-60: 7.25792121887207\n",
      "--- processing region: SST-7 with 143 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-7.fasta\n",
      "-- *blasting region: SST-7\n",
      "--- total time for blast SST-7: 9.388401508331299\n",
      "--- processing region: SST-8 with 130 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-8.fasta\n",
      "-- *blasting region: SST-8\n",
      "--- total time for blast SST-8: 7.907023668289185\n",
      "--- processing region: SST-9 with 145 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_SST-9.fasta\n",
      "-- *blasting region: SST-9\n",
      "--- total time for blast SST-9: 9.484328985214233\n",
      "--- processing region: Slc17a7-1 with 104 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-1.fasta\n",
      "-- *blasting region: Slc17a7-1\n",
      "--- total time for blast Slc17a7-1: 8.058455228805542\n",
      "--- processing region: Slc17a7-10 with 112 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-10.fasta\n",
      "-- *blasting region: Slc17a7-10\n",
      "--- total time for blast Slc17a7-10: 8.174474477767944\n",
      "--- processing region: Slc17a7-11 with 137 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-11.fasta\n",
      "-- *blasting region: Slc17a7-11\n",
      "--- total time for blast Slc17a7-11: 7.808016777038574\n",
      "--- processing region: Slc17a7-12 with 148 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-12.fasta\n",
      "-- *blasting region: Slc17a7-12\n",
      "--- total time for blast Slc17a7-12: 8.966336488723755\n",
      "--- processing region: Slc17a7-13 with 160 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-13.fasta\n",
      "-- *blasting region: Slc17a7-13\n",
      "--- total time for blast Slc17a7-13: 11.012213468551636\n",
      "--- processing region: Slc17a7-14 with 123 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-14.fasta\n",
      "-- *blasting region: Slc17a7-14\n",
      "--- total time for blast Slc17a7-14: 8.805633068084717\n",
      "--- processing region: Slc17a7-15 with 140 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-15.fasta\n",
      "-- *blasting region: Slc17a7-15\n",
      "--- total time for blast Slc17a7-15: 7.53840708732605\n",
      "--- processing region: Slc17a7-16 with 144 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-16.fasta\n",
      "-- *blasting region: Slc17a7-16\n",
      "--- total time for blast Slc17a7-16: 7.27437424659729\n",
      "--- processing region: Slc17a7-17 with 148 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-17.fasta\n",
      "-- *blasting region: Slc17a7-17\n",
      "--- total time for blast Slc17a7-17: 9.098026275634766\n",
      "--- processing region: Slc17a7-18 with 126 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-18.fasta\n",
      "-- *blasting region: Slc17a7-18\n",
      "--- total time for blast Slc17a7-18: 7.661040544509888\n",
      "--- processing region: Slc17a7-19 with 114 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-19.fasta\n",
      "-- *blasting region: Slc17a7-19\n",
      "--- total time for blast Slc17a7-19: 9.062481164932251\n",
      "--- processing region: Slc17a7-2 with 99 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-2.fasta\n",
      "-- *blasting region: Slc17a7-2\n",
      "--- total time for blast Slc17a7-2: 7.249603509902954\n",
      "--- processing region: Slc17a7-20 with 94 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-20.fasta\n",
      "-- *blasting region: Slc17a7-20\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Slc17a7-20: 8.244128704071045\n",
      "--- processing region: Slc17a7-21 with 92 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-21.fasta\n",
      "-- *blasting region: Slc17a7-21\n",
      "--- total time for blast Slc17a7-21: 9.282278776168823\n",
      "--- processing region: Slc17a7-22 with 130 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-22.fasta\n",
      "-- *blasting region: Slc17a7-22\n",
      "--- total time for blast Slc17a7-22: 14.174342393875122\n",
      "--- processing region: Slc17a7-23 with 119 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-23.fasta\n",
      "-- *blasting region: Slc17a7-23\n",
      "--- total time for blast Slc17a7-23: 9.150672435760498\n",
      "--- processing region: Slc17a7-24 with 127 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-24.fasta\n",
      "-- *blasting region: Slc17a7-24\n",
      "--- total time for blast Slc17a7-24: 6.825181007385254\n",
      "--- processing region: Slc17a7-25 with 122 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-25.fasta\n",
      "-- *blasting region: Slc17a7-25\n",
      "--- total time for blast Slc17a7-25: 7.287864923477173\n",
      "--- processing region: Slc17a7-26 with 134 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-26.fasta\n",
      "-- *blasting region: Slc17a7-26\n",
      "--- total time for blast Slc17a7-26: 10.655402660369873\n",
      "--- processing region: Slc17a7-27 with 147 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-27.fasta\n",
      "-- *blasting region: Slc17a7-27\n",
      "--- total time for blast Slc17a7-27: 10.88587474822998\n",
      "--- processing region: Slc17a7-28 with 153 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-28.fasta\n",
      "-- *blasting region: Slc17a7-28\n",
      "--- total time for blast Slc17a7-28: 7.345606565475464\n",
      "--- processing region: Slc17a7-29 with 127 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-29.fasta\n",
      "-- *blasting region: Slc17a7-29\n",
      "--- total time for blast Slc17a7-29: 10.180800676345825\n",
      "--- processing region: Slc17a7-3 with 137 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-3.fasta\n",
      "-- *blasting region: Slc17a7-3\n",
      "--- total time for blast Slc17a7-3: 8.083963632583618\n",
      "--- processing region: Slc17a7-30 with 133 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-30.fasta\n",
      "-- *blasting region: Slc17a7-30\n",
      "--- total time for blast Slc17a7-30: 8.355713605880737\n",
      "--- processing region: Slc17a7-31 with 140 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-31.fasta\n",
      "-- *blasting region: Slc17a7-31\n",
      "--- total time for blast Slc17a7-31: 7.623572587966919\n",
      "--- processing region: Slc17a7-32 with 141 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-32.fasta\n",
      "-- *blasting region: Slc17a7-32\n",
      "--- total time for blast Slc17a7-32: 9.718257665634155\n",
      "--- processing region: Slc17a7-33 with 112 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-33.fasta\n",
      "-- *blasting region: Slc17a7-33\n",
      "--- total time for blast Slc17a7-33: 10.7243173122406\n",
      "--- processing region: Slc17a7-34 with 125 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-34.fasta\n",
      "-- *blasting region: Slc17a7-34\n",
      "--- total time for blast Slc17a7-34: 9.804908037185669\n",
      "--- processing region: Slc17a7-35 with 102 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-35.fasta\n",
      "-- *blasting region: Slc17a7-35\n",
      "--- total time for blast Slc17a7-35: 10.391136169433594\n",
      "--- processing region: Slc17a7-36 with 106 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-36.fasta\n",
      "-- *blasting region: Slc17a7-36\n",
      "--- total time for blast Slc17a7-36: 8.236426830291748\n",
      "--- processing region: Slc17a7-37 with 99 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-37.fasta\n",
      "-- *blasting region: Slc17a7-37\n",
      "--- total time for blast Slc17a7-37: 9.893271684646606\n",
      "--- processing region: Slc17a7-38 with 93 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-38.fasta\n",
      "-- *blasting region: Slc17a7-38\n",
      "--- total time for blast Slc17a7-38: 7.558013916015625\n",
      "--- processing region: Slc17a7-39 with 145 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-39.fasta\n",
      "-- *blasting region: Slc17a7-39\n",
      "--- total time for blast Slc17a7-39: 7.123534440994263\n",
      "--- processing region: Slc17a7-4 with 110 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-4.fasta\n",
      "-- *blasting region: Slc17a7-4\n",
      "--- total time for blast Slc17a7-4: 8.811824083328247\n",
      "--- processing region: Slc17a7-40 with 121 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-40.fasta\n",
      "-- *blasting region: Slc17a7-40\n",
      "--- total time for blast Slc17a7-40: 9.742437601089478\n",
      "--- processing region: Slc17a7-41 with 134 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-41.fasta\n",
      "-- *blasting region: Slc17a7-41\n",
      "--- total time for blast Slc17a7-41: 9.679171323776245\n",
      "--- processing region: Slc17a7-42 with 130 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-42.fasta\n",
      "-- *blasting region: Slc17a7-42\n",
      "--- total time for blast Slc17a7-42: 9.574732065200806\n",
      "--- processing region: Slc17a7-43 with 132 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-43.fasta\n",
      "-- *blasting region: Slc17a7-43\n",
      "--- total time for blast Slc17a7-43: 9.100081443786621\n",
      "--- processing region: Slc17a7-44 with 141 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-44.fasta\n",
      "-- *blasting region: Slc17a7-44\n",
      "--- total time for blast Slc17a7-44: 8.186474561691284\n",
      "--- processing region: Slc17a7-45 with 145 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-45.fasta\n",
      "-- *blasting region: Slc17a7-45\n",
      "--- total time for blast Slc17a7-45: 6.944546699523926\n",
      "--- processing region: Slc17a7-46 with 132 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-46.fasta\n",
      "-- *blasting region: Slc17a7-46\n",
      "--- total time for blast Slc17a7-46: 11.321743249893188\n",
      "--- processing region: Slc17a7-47 with 111 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-47.fasta\n",
      "-- *blasting region: Slc17a7-47\n",
      "--- total time for blast Slc17a7-47: 8.476691246032715\n",
      "--- processing region: Slc17a7-48 with 129 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-48.fasta\n",
      "-- *blasting region: Slc17a7-48\n",
      "--- total time for blast Slc17a7-48: 11.838241338729858\n",
      "--- processing region: Slc17a7-49 with 130 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-49.fasta\n",
      "-- *blasting region: Slc17a7-49\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Slc17a7-49: 11.242235898971558\n",
      "--- processing region: Slc17a7-5 with 124 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-5.fasta\n",
      "-- *blasting region: Slc17a7-5\n",
      "--- total time for blast Slc17a7-5: 10.305974245071411\n",
      "--- processing region: Slc17a7-50 with 135 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-50.fasta\n",
      "-- *blasting region: Slc17a7-50\n",
      "--- total time for blast Slc17a7-50: 9.893279075622559\n",
      "--- processing region: Slc17a7-51 with 65 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-51.fasta\n",
      "-- *blasting region: Slc17a7-51\n",
      "--- total time for blast Slc17a7-51: 6.742665529251099\n",
      "--- processing region: Slc17a7-52 with 90 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-52.fasta\n",
      "-- *blasting region: Slc17a7-52\n",
      "--- total time for blast Slc17a7-52: 8.150315284729004\n",
      "--- processing region: Slc17a7-53 with 125 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-53.fasta\n",
      "-- *blasting region: Slc17a7-53\n",
      "--- total time for blast Slc17a7-53: 8.290000915527344\n",
      "--- processing region: Slc17a7-54 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-54.fasta\n",
      "-- *blasting region: Slc17a7-54\n",
      "--- total time for blast Slc17a7-54: 6.299970626831055\n",
      "--- processing region: Slc17a7-55 with 132 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-55.fasta\n",
      "-- *blasting region: Slc17a7-55\n",
      "--- total time for blast Slc17a7-55: 12.749056577682495\n",
      "--- processing region: Slc17a7-56 with 114 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-56.fasta\n",
      "-- *blasting region: Slc17a7-56\n",
      "--- total time for blast Slc17a7-56: 10.260374069213867\n",
      "--- processing region: Slc17a7-57 with 130 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-57.fasta\n",
      "-- *blasting region: Slc17a7-57\n",
      "--- total time for blast Slc17a7-57: 7.822099685668945\n",
      "--- processing region: Slc17a7-58 with 121 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-58.fasta\n",
      "-- *blasting region: Slc17a7-58\n",
      "--- total time for blast Slc17a7-58: 8.019070148468018\n",
      "--- processing region: Slc17a7-59 with 102 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-59.fasta\n",
      "-- *blasting region: Slc17a7-59\n",
      "--- total time for blast Slc17a7-59: 7.1992082595825195\n",
      "--- processing region: Slc17a7-6 with 53 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-6.fasta\n",
      "-- *blasting region: Slc17a7-6\n",
      "--- total time for blast Slc17a7-6: 4.625462293624878\n",
      "--- processing region: Slc17a7-60 with 142 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-60.fasta\n",
      "-- *blasting region: Slc17a7-60\n",
      "--- total time for blast Slc17a7-60: 10.228617668151855\n",
      "--- processing region: Slc17a7-7 with 66 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-7.fasta\n",
      "-- *blasting region: Slc17a7-7\n",
      "--- total time for blast Slc17a7-7: 5.299976110458374\n",
      "--- processing region: Slc17a7-8 with 141 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-8.fasta\n",
      "-- *blasting region: Slc17a7-8\n",
      "--- total time for blast Slc17a7-8: 7.694511413574219\n",
      "--- processing region: Slc17a7-9 with 146 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc17a7-9.fasta\n",
      "-- *blasting region: Slc17a7-9\n",
      "--- total time for blast Slc17a7-9: 9.436544418334961\n",
      "--- processing region: Slc32a1-1 with 129 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc32a1-1.fasta\n",
      "-- *blasting region: Slc32a1-1\n",
      "--- total time for blast Slc32a1-1: 13.772072076797485\n",
      "--- processing region: Slc32a1-10 with 128 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc32a1-10.fasta\n",
      "-- *blasting region: Slc32a1-10\n",
      "--- total time for blast Slc32a1-10: 8.929011106491089\n",
      "--- processing region: Slc32a1-11 with 144 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc32a1-11.fasta\n",
      "-- *blasting region: Slc32a1-11\n",
      "--- total time for blast Slc32a1-11: 8.73991322517395\n",
      "--- processing region: Slc32a1-12 with 129 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc32a1-12.fasta\n",
      "-- *blasting region: Slc32a1-12\n",
      "--- total time for blast Slc32a1-12: 9.247138738632202\n",
      "--- processing region: Slc32a1-13 with 144 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc32a1-13.fasta\n",
      "-- *blasting region: Slc32a1-13\n",
      "--- total time for blast Slc32a1-13: 9.22034215927124\n",
      "--- processing region: Slc32a1-14 with 121 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc32a1-14.fasta\n",
      "-- *blasting region: Slc32a1-14\n",
      "--- total time for blast Slc32a1-14: 8.716408967971802\n",
      "--- processing region: Slc32a1-15 with 127 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc32a1-15.fasta\n",
      "-- *blasting region: Slc32a1-15\n",
      "--- total time for blast Slc32a1-15: 7.428279161453247\n",
      "--- processing region: Slc32a1-16 with 114 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc32a1-16.fasta\n",
      "-- *blasting region: Slc32a1-16\n",
      "--- total time for blast Slc32a1-16: 8.776275634765625\n",
      "--- processing region: Slc32a1-17 with 141 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc32a1-17.fasta\n",
      "-- *blasting region: Slc32a1-17\n",
      "--- total time for blast Slc32a1-17: 9.871868371963501\n",
      "--- processing region: Slc32a1-18 with 127 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc32a1-18.fasta\n",
      "-- *blasting region: Slc32a1-18\n",
      "--- total time for blast Slc32a1-18: 7.668562650680542\n",
      "--- processing region: Slc32a1-19 with 136 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc32a1-19.fasta\n",
      "-- *blasting region: Slc32a1-19\n",
      "--- total time for blast Slc32a1-19: 8.910733461380005\n",
      "--- processing region: Slc32a1-2 with 138 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc32a1-2.fasta\n",
      "-- *blasting region: Slc32a1-2\n",
      "--- total time for blast Slc32a1-2: 8.829658269882202\n",
      "--- processing region: Slc32a1-20 with 135 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc32a1-20.fasta\n",
      "-- *blasting region: Slc32a1-20\n",
      "--- total time for blast Slc32a1-20: 9.641202449798584\n",
      "--- processing region: Slc32a1-21 with 144 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc32a1-21.fasta\n",
      "-- *blasting region: Slc32a1-21\n",
      "--- total time for blast Slc32a1-21: 7.615466117858887\n",
      "--- processing region: Slc32a1-3 with 129 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc32a1-3.fasta\n",
      "-- *blasting region: Slc32a1-3\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Slc32a1-3: 7.974017143249512\n",
      "--- processing region: Slc32a1-4 with 123 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc32a1-4.fasta\n",
      "-- *blasting region: Slc32a1-4\n",
      "--- total time for blast Slc32a1-4: 10.112616777420044\n",
      "--- processing region: Slc32a1-5 with 134 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc32a1-5.fasta\n",
      "-- *blasting region: Slc32a1-5\n",
      "--- total time for blast Slc32a1-5: 7.092902898788452\n",
      "--- processing region: Slc32a1-6 with 131 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc32a1-6.fasta\n",
      "-- *blasting region: Slc32a1-6\n",
      "--- total time for blast Slc32a1-6: 7.5186285972595215\n",
      "--- processing region: Slc32a1-7 with 123 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc32a1-7.fasta\n",
      "-- *blasting region: Slc32a1-7\n",
      "--- total time for blast Slc32a1-7: 8.930176019668579\n",
      "--- processing region: Slc32a1-8 with 114 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc32a1-8.fasta\n",
      "-- *blasting region: Slc32a1-8\n",
      "--- total time for blast Slc32a1-8: 9.75313663482666\n",
      "--- processing region: Slc32a1-9 with 123 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast\\probe_gene_Slc32a1-9.fasta\n",
      "-- *blasting region: Slc32a1-9\n",
      "--- total time for blast Slc32a1-9: 8.269656419754028\n"
     ]
    }
   ],
   "source": [
    "blast_database = os.path.join(reference_folder, 'blast_database', 'GRCm38_ensembl')\n",
    "\n",
    "check.Blast_probes(kept_records, library_folder, num_threads=20, \n",
    "                   blast_db=blast_database)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 4.2.1 screen probe by blast results\n",
    "\n",
    "This step has 2 different cutoffs:\n",
    "\n",
    "hard_cutoff: whether a probe has any ~40mer hit somewhere else, if so just dump the probe\n",
    "\n",
    "soft_cutoff: whether a probe has a lot of ~20mer off-target hits, if exceed certain amount then dump the probe"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 260,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Number of probes loaded: 32432\n",
      "- Number of regions in this library: 261\n",
      "-- checking probes in region: Ccnd2-1\n",
      "--- gene=Ccnd2-1, id=64 removed by soft count = 110\n",
      "--- gene=Ccnd2-1, id=65 removed by soft count = 61\n",
      "--- gene=Ccnd2-1, id=66 removed by soft count = 86\n",
      "--- gene=Ccnd2-1, id=68 removed by soft count = 83\n",
      "--- gene=Ccnd2-1, id=69 removed by soft count = 47\n",
      "--- gene=Ccnd2-1, id=70 removed by soft count = 72\n",
      "--- gene=Ccnd2-1, id=71 removed by soft count = 76\n",
      "--- gene=Ccnd2-1, id=113 removed by hard count = 3\n",
      "--- gene=Ccnd2-1, id=115 removed by hard count = 3\n",
      "--- gene=Ccnd2-1, id=117 removed by hard count = 4\n",
      "--- number of probes: 135 , kept by blast: 125 , if remove dups: 124\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 124\n",
      "-- checking probes in region: Ccnd2-2\n",
      "--- gene=Ccnd2-2, id=0 removed by soft count = 35\n",
      "--- gene=Ccnd2-2, id=1 removed by soft count = 35\n",
      "--- gene=Ccnd2-2, id=2 removed by soft count = 693\n",
      "--- gene=Ccnd2-2, id=16 removed by soft count = 69\n",
      "--- gene=Ccnd2-2, id=26 removed by soft count = 337\n",
      "--- number of probes: 99 , kept by blast: 94 , if remove dups: 94\n",
      "-- number of probes kept for this region: 94\n",
      "-- checking probes in region: Ccnd2-3\n",
      "--- gene=Ccnd2-3, id=24 removed by soft count = 87\n",
      "--- gene=Ccnd2-3, id=42 removed by soft count = 180\n",
      "--- gene=Ccnd2-3, id=43 removed by soft count = 81\n",
      "--- gene=Ccnd2-3, id=49 removed by soft count = 31\n",
      "--- gene=Ccnd2-3, id=57 removed by soft count = 33\n",
      "--- gene=Ccnd2-3, id=58 removed by soft count = 226\n",
      "--- number of probes: 104 , kept by blast: 98 , if remove dups: 98\n",
      "-- number of probes kept for this region: 98\n",
      "-- checking probes in region: Ccnd2-4\n",
      "--- number of probes: 149 , kept by blast: 149 , if remove dups: 149\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 149\n",
      "-- checking probes in region: Ccnd2-5\n",
      "--- gene=Ccnd2-5, id=33 removed by soft count = 133\n",
      "--- gene=Ccnd2-5, id=58 removed by soft count = 204\n",
      "--- number of probes: 152 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 150\n",
      "-- checking probes in region: Ccnd2-6\n",
      "--- gene=Ccnd2-6, id=0 removed by soft count = 65\n",
      "--- gene=Ccnd2-6, id=12 removed by soft count = 34\n",
      "--- gene=Ccnd2-6, id=76 removed by soft count = 54\n",
      "--- gene=Ccnd2-6, id=102 removed by soft count = 46\n",
      "--- gene=Ccnd2-6, id=105 removed by soft count = 37\n",
      "--- gene=Ccnd2-6, id=106 removed by soft count = 69\n",
      "--- number of probes: 112 , kept by blast: 106 , if remove dups: 106\n",
      "-- number of probes kept for this region: 106\n",
      "-- checking probes in region: Ccnd2-7\n",
      "--- gene=Ccnd2-7, id=90 removed by soft count = 44\n",
      "--- number of probes: 136 , kept by blast: 135 , if remove dups: 135\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 135\n",
      "-- checking probes in region: Ccnd2-8\n",
      "--- gene=Ccnd2-8, id=44 removed by soft count = 38\n",
      "--- gene=Ccnd2-8, id=45 removed by soft count = 41\n",
      "--- gene=Ccnd2-8, id=66 removed by soft count = 32\n",
      "--- number of probes: 140 , kept by blast: 137 , if remove dups: 137\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 137\n",
      "-- checking probes in region: Ccnd2-9\n",
      "--- gene=Ccnd2-9, id=14 removed by soft count = 286\n",
      "--- gene=Ccnd2-9, id=26 removed by soft count = 48\n",
      "--- gene=Ccnd2-9, id=29 removed by soft count = 101\n",
      "--- gene=Ccnd2-9, id=128 removed by soft count = 38\n",
      "--- number of probes: 134 , kept by blast: 130 , if remove dups: 127\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 127\n",
      "-- checking probes in region: Ccnd2-10\n",
      "--- gene=Ccnd2-10, id=3 removed by soft count = 49\n",
      "--- number of probes: 136 , kept by blast: 135 , if remove dups: 135\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 135\n",
      "-- checking probes in region: Ccnd2-11\n",
      "--- gene=Ccnd2-11, id=46 removed by soft count = 35\n",
      "--- gene=Ccnd2-11, id=60 removed by soft count = 38\n",
      "--- number of probes: 139 , kept by blast: 137 , if remove dups: 137\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 137\n",
      "-- checking probes in region: Ccnd2-12\n",
      "--- number of probes: 62 , kept by blast: 62 , if remove dups: 62\n",
      "-- number of probes kept for this region: 62\n",
      "-- checking probes in region: Ccnd2-13\n",
      "--- gene=Ccnd2-13, id=39 removed by soft count = 243\n",
      "--- gene=Ccnd2-13, id=40 removed by soft count = 231\n",
      "--- gene=Ccnd2-13, id=44 removed by soft count = 187\n",
      "--- gene=Ccnd2-13, id=67 removed by soft count = 31\n",
      "--- number of probes: 127 , kept by blast: 123 , if remove dups: 123\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 123\n",
      "-- checking probes in region: Ccnd2-14\n",
      "--- gene=Ccnd2-14, id=14 removed by soft count = 45\n",
      "--- gene=Ccnd2-14, id=16 removed by soft count = 116\n",
      "--- gene=Ccnd2-14, id=18 removed by soft count = 269\n",
      "--- gene=Ccnd2-14, id=19 removed by soft count = 1871\n",
      "--- gene=Ccnd2-14, id=22 removed by soft count = 140\n",
      "--- gene=Ccnd2-14, id=33 removed by soft count = 83\n",
      "--- gene=Ccnd2-14, id=34 removed by soft count = 33\n",
      "--- gene=Ccnd2-14, id=35 removed by soft count = 200\n",
      "--- gene=Ccnd2-14, id=60 removed by soft count = 93\n",
      "--- number of probes: 117 , kept by blast: 108 , if remove dups: 108\n",
      "-- number of probes kept for this region: 108\n",
      "-- checking probes in region: Ccnd2-15\n",
      "--- gene=Ccnd2-15, id=2 removed by soft count = 70\n",
      "--- gene=Ccnd2-15, id=14 removed by soft count = 100\n",
      "--- gene=Ccnd2-15, id=15 removed by soft count = 223\n",
      "--- gene=Ccnd2-15, id=16 removed by soft count = 44\n",
      "--- number of probes: 125 , kept by blast: 121 , if remove dups: 121\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 121\n",
      "-- checking probes in region: Ccnd2-16\n",
      "--- gene=Ccnd2-16, id=131 removed by soft count = 36\n",
      "--- number of probes: 136 , kept by blast: 135 , if remove dups: 135\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 135\n",
      "-- checking probes in region: Ccnd2-17\n",
      "--- gene=Ccnd2-17, id=80 removed by soft count = 34\n",
      "--- number of probes: 147 , kept by blast: 146 , if remove dups: 146\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 146\n",
      "-- checking probes in region: Ccnd2-18\n",
      "--- number of probes: 146 , kept by blast: 146 , if remove dups: 146\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 146\n",
      "-- checking probes in region: Ccnd2-19\n",
      "--- gene=Ccnd2-19, id=29 removed by soft count = 37\n",
      "--- gene=Ccnd2-19, id=59 removed by soft count = 245\n",
      "--- gene=Ccnd2-19, id=90 removed by soft count = 32\n",
      "--- gene=Ccnd2-19, id=104 removed by soft count = 94\n",
      "--- gene=Ccnd2-19, id=139 removed by soft count = 129\n",
      "--- number of probes: 143 , kept by blast: 138 , if remove dups: 138\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 138\n",
      "-- checking probes in region: Ccnd2-20\n",
      "--- gene=Ccnd2-20, id=25 removed by soft count = 34\n",
      "--- gene=Ccnd2-20, id=73 removed by soft count = 81\n",
      "--- gene=Ccnd2-20, id=123 removed by soft count = 96\n",
      "--- gene=Ccnd2-20, id=138 removed by soft count = 41\n",
      "--- number of probes: 142 , kept by blast: 138 , if remove dups: 138\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 138\n",
      "-- checking probes in region: Ccnd2-21\n",
      "--- gene=Ccnd2-21, id=90 removed by soft count = 31\n",
      "--- gene=Ccnd2-21, id=108 removed by soft count = 36\n",
      "--- gene=Ccnd2-21, id=133 removed by soft count = 89\n",
      "--- number of probes: 139 , kept by blast: 136 , if remove dups: 135\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 135\n",
      "-- checking probes in region: Ccnd2-22\n",
      "--- gene=Ccnd2-22, id=97 removed by soft count = 101\n",
      "--- number of probes: 126 , kept by blast: 125 , if remove dups: 125\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 125\n",
      "-- checking probes in region: Ccnd2-23\n",
      "--- gene=Ccnd2-23, id=32 removed by soft count = 99\n",
      "--- gene=Ccnd2-23, id=75 removed by soft count = 173\n",
      "--- number of probes: 138 , kept by blast: 136 , if remove dups: 136\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 136\n",
      "-- checking probes in region: Ccnd2-24\n",
      "--- gene=Ccnd2-24, id=4 removed by soft count = 55\n",
      "--- gene=Ccnd2-24, id=102 removed by soft count = 46\n",
      "--- gene=Ccnd2-24, id=135 removed by soft count = 244\n",
      "--- number of probes: 136 , kept by blast: 133 , if remove dups: 133\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 133\n",
      "-- checking probes in region: Ccnd2-25\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Ccnd2-25, id=8 removed by soft count = 43\n",
      "--- gene=Ccnd2-25, id=16 removed by soft count = 395\n",
      "--- number of probes: 128 , kept by blast: 126 , if remove dups: 126\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 126\n",
      "-- checking probes in region: Ccnd2-26\n",
      "--- gene=Ccnd2-26, id=8 removed by soft count = 55\n",
      "--- gene=Ccnd2-26, id=47 removed by soft count = 51\n",
      "--- gene=Ccnd2-26, id=48 removed by soft count = 94\n",
      "--- gene=Ccnd2-26, id=55 removed by soft count = 310\n",
      "--- gene=Ccnd2-26, id=89 removed by soft count = 31\n",
      "--- number of probes: 101 , kept by blast: 96 , if remove dups: 96\n",
      "-- number of probes kept for this region: 96\n",
      "-- checking probes in region: Ccnd2-27\n",
      "--- gene=Ccnd2-27, id=74 removed by soft count = 42\n",
      "--- number of probes: 139 , kept by blast: 138 , if remove dups: 138\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 138\n",
      "-- checking probes in region: Ccnd2-28\n",
      "--- gene=Ccnd2-28, id=80 removed by soft count = 32\n",
      "--- number of probes: 128 , kept by blast: 127 , if remove dups: 127\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 127\n",
      "-- checking probes in region: Ccnd2-29\n",
      "--- number of probes: 145 , kept by blast: 145 , if remove dups: 145\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 145\n",
      "-- checking probes in region: Ccnd2-30\n",
      "--- gene=Ccnd2-30, id=120 removed by soft count = 56\n",
      "--- gene=Ccnd2-30, id=121 removed by soft count = 64\n",
      "--- gene=Ccnd2-30, id=122 removed by soft count = 46\n",
      "--- number of probes: 125 , kept by blast: 122 , if remove dups: 122\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 122\n",
      "-- checking probes in region: Ccnd2-31\n",
      "--- gene=Ccnd2-31, id=123 removed by soft count = 152\n",
      "--- number of probes: 124 , kept by blast: 123 , if remove dups: 123\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 123\n",
      "-- checking probes in region: Ccnd2-32\n",
      "--- gene=Ccnd2-32, id=81 removed by soft count = 406\n",
      "--- gene=Ccnd2-32, id=122 removed by soft count = 41\n",
      "--- number of probes: 135 , kept by blast: 133 , if remove dups: 133\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 133\n",
      "-- checking probes in region: Ccnd2-33\n",
      "--- gene=Ccnd2-33, id=8 removed by soft count = 57\n",
      "--- gene=Ccnd2-33, id=10 removed by soft count = 42\n",
      "--- number of probes: 127 , kept by blast: 125 , if remove dups: 125\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 125\n",
      "-- checking probes in region: Ccnd2-34\n",
      "--- gene=Ccnd2-34, id=55 removed by soft count = 259\n",
      "--- number of probes: 133 , kept by blast: 132 , if remove dups: 132\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 132\n",
      "-- checking probes in region: Ccnd2-35\n",
      "--- gene=Ccnd2-35, id=55 removed by soft count = 33\n",
      "--- gene=Ccnd2-35, id=131 removed by soft count = 114\n",
      "--- number of probes: 149 , kept by blast: 147 , if remove dups: 147\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 147\n",
      "-- checking probes in region: Ccnd2-36\n",
      "--- gene=Ccnd2-36, id=135 removed by soft count = 230\n",
      "--- number of probes: 138 , kept by blast: 137 , if remove dups: 137\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 137\n",
      "-- checking probes in region: Ccnd2-37\n",
      "--- number of probes: 118 , kept by blast: 118 , if remove dups: 118\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 118\n",
      "-- checking probes in region: Ccnd2-38\n",
      "--- gene=Ccnd2-38, id=53 removed by soft count = 689\n",
      "--- gene=Ccnd2-38, id=54 removed by soft count = 36\n",
      "--- number of probes: 145 , kept by blast: 143 , if remove dups: 143\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 143\n",
      "-- checking probes in region: Ccnd2-39\n",
      "--- gene=Ccnd2-39, id=16 removed by soft count = 67\n",
      "--- gene=Ccnd2-39, id=19 removed by soft count = 71\n",
      "--- gene=Ccnd2-39, id=115 removed by soft count = 86\n",
      "--- number of probes: 119 , kept by blast: 116 , if remove dups: 116\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 116\n",
      "-- checking probes in region: Ccnd2-40\n",
      "--- number of probes: 135 , kept by blast: 135 , if remove dups: 135\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 135\n",
      "-- checking probes in region: Ccnd2-41\n",
      "--- gene=Ccnd2-41, id=69 removed by soft count = 33\n",
      "--- gene=Ccnd2-41, id=70 removed by soft count = 33\n",
      "--- gene=Ccnd2-41, id=71 removed by hard count = 4\n",
      "--- gene=Ccnd2-41, id=75 removed by hard count = 3\n",
      "--- gene=Ccnd2-41, id=77 removed by hard count = 3\n",
      "--- number of probes: 124 , kept by blast: 119 , if remove dups: 115\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 115\n",
      "-- checking probes in region: Ccnd2-42\n",
      "--- number of probes: 117 , kept by blast: 117 , if remove dups: 117\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 117\n",
      "-- checking probes in region: Ccnd2-43\n",
      "--- gene=Ccnd2-43, id=13 removed by soft count = 265\n",
      "--- gene=Ccnd2-43, id=97 removed by soft count = 242\n",
      "--- number of probes: 126 , kept by blast: 124 , if remove dups: 124\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 124\n",
      "-- checking probes in region: Ccnd2-44\n",
      "--- number of probes: 128 , kept by blast: 128 , if remove dups: 128\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 128\n",
      "-- checking probes in region: Ccnd2-45\n",
      "--- gene=Ccnd2-45, id=7 removed by soft count = 92\n",
      "--- gene=Ccnd2-45, id=11 removed by soft count = 108\n",
      "--- number of probes: 128 , kept by blast: 126 , if remove dups: 126\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 126\n",
      "-- checking probes in region: Ccnd2-46\n",
      "--- gene=Ccnd2-46, id=28 removed by soft count = 54\n",
      "--- gene=Ccnd2-46, id=93 removed by soft count = 53\n",
      "--- gene=Ccnd2-46, id=94 removed by soft count = 59\n",
      "--- gene=Ccnd2-46, id=120 removed by soft count = 60\n",
      "--- number of probes: 130 , kept by blast: 126 , if remove dups: 126\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 126\n",
      "-- checking probes in region: Ccnd2-47\n",
      "--- gene=Ccnd2-47, id=49 removed by soft count = 132\n",
      "--- number of probes: 142 , kept by blast: 141 , if remove dups: 141\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 141\n",
      "-- checking probes in region: Ccnd2-48\n",
      "--- gene=Ccnd2-48, id=96 removed by soft count = 94\n",
      "--- gene=Ccnd2-48, id=98 removed by soft count = 46\n",
      "--- gene=Ccnd2-48, id=102 removed by soft count = 32\n",
      "--- gene=Ccnd2-48, id=105 removed by soft count = 65\n",
      "--- number of probes: 135 , kept by blast: 131 , if remove dups: 131\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 131\n",
      "-- checking probes in region: Ccnd2-49\n",
      "--- gene=Ccnd2-49, id=4 removed by soft count = 80\n",
      "--- gene=Ccnd2-49, id=30 removed by soft count = 35\n",
      "--- number of probes: 128 , kept by blast: 126 , if remove dups: 126\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 126\n",
      "-- checking probes in region: Ccnd2-50\n",
      "--- gene=Ccnd2-50, id=76 removed by soft count = 39\n",
      "--- number of probes: 133 , kept by blast: 132 , if remove dups: 132\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 132\n",
      "-- checking probes in region: Ccnd2-51\n",
      "--- gene=Ccnd2-51, id=1 removed by soft count = 107\n",
      "--- gene=Ccnd2-51, id=56 removed by soft count = 39\n",
      "--- gene=Ccnd2-51, id=63 removed by soft count = 91\n",
      "--- gene=Ccnd2-51, id=64 removed by soft count = 175\n",
      "--- gene=Ccnd2-51, id=65 removed by soft count = 827\n",
      "--- number of probes: 99 , kept by blast: 94 , if remove dups: 94\n",
      "-- number of probes kept for this region: 94\n",
      "-- checking probes in region: Ccnd2-52\n",
      "--- gene=Ccnd2-52, id=81 removed by soft count = 34\n",
      "--- gene=Ccnd2-52, id=123 removed by soft count = 35\n",
      "--- number of probes: 136 , kept by blast: 134 , if remove dups: 134\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 134\n",
      "-- checking probes in region: Ccnd2-53\n",
      "--- gene=Ccnd2-53, id=31 removed by soft count = 122\n",
      "--- gene=Ccnd2-53, id=96 removed by soft count = 52\n",
      "--- gene=Ccnd2-53, id=97 removed by soft count = 63\n",
      "--- gene=Ccnd2-53, id=99 removed by soft count = 776\n",
      "--- gene=Ccnd2-53, id=101 removed by soft count = 110\n",
      "--- gene=Ccnd2-53, id=127 removed by soft count = 539\n",
      "--- gene=Ccnd2-53, id=128 removed by soft count = 223\n",
      "--- number of probes: 132 , kept by blast: 125 , if remove dups: 125\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 125\n",
      "-- checking probes in region: Ccnd2-54\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Ccnd2-54, id=11 removed by soft count = 406\n",
      "--- gene=Ccnd2-54, id=13 removed by soft count = 66\n",
      "--- gene=Ccnd2-54, id=14 removed by soft count = 61\n",
      "--- gene=Ccnd2-54, id=16 removed by soft count = 108\n",
      "--- gene=Ccnd2-54, id=21 removed by soft count = 127\n",
      "--- gene=Ccnd2-54, id=29 removed by soft count = 53\n",
      "--- number of probes: 34 , kept by blast: 28 , if remove dups: 28\n",
      "-- number of probes kept for this region: 28\n",
      "-- checking probes in region: Ccnd2-55\n",
      "--- gene=Ccnd2-55, id=88 removed by soft count = 57\n",
      "--- gene=Ccnd2-55, id=90 removed by soft count = 121\n",
      "--- number of probes: 101 , kept by blast: 99 , if remove dups: 99\n",
      "-- number of probes kept for this region: 99\n",
      "-- checking probes in region: Ccnd2-56\n",
      "--- gene=Ccnd2-56, id=29 removed by soft count = 245\n",
      "--- number of probes: 116 , kept by blast: 115 , if remove dups: 115\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 115\n",
      "-- checking probes in region: Ccnd2-57\n",
      "--- number of probes: 134 , kept by blast: 134 , if remove dups: 134\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 134\n",
      "-- checking probes in region: Ccnd2-58\n",
      "--- number of probes: 142 , kept by blast: 142 , if remove dups: 142\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 142\n",
      "-- checking probes in region: Ccnd2-59\n",
      "--- gene=Ccnd2-59, id=76 removed by hard count = 3\n",
      "--- gene=Ccnd2-59, id=80 removed by hard count = 3\n",
      "--- number of probes: 147 , kept by blast: 145 , if remove dups: 144\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 144\n",
      "-- checking probes in region: Ccnd2-60\n",
      "--- number of probes: 143 , kept by blast: 143 , if remove dups: 143\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 143\n",
      "-- checking probes in region: Nkx2-1-1\n",
      "--- gene=Nkx2-1-1, id=1 removed by soft count = 73\n",
      "--- gene=Nkx2-1-1, id=2 removed by soft count = 81\n",
      "--- gene=Nkx2-1-1, id=4 removed by soft count = 93\n",
      "--- gene=Nkx2-1-1, id=15 removed by soft count = 91\n",
      "--- gene=Nkx2-1-1, id=66 removed by soft count = 68\n",
      "--- number of probes: 79 , kept by blast: 74 , if remove dups: 74\n",
      "-- number of probes kept for this region: 74\n",
      "-- checking probes in region: Nkx2-1-2\n",
      "--- gene=Nkx2-1-2, id=22 removed by soft count = 179\n",
      "--- gene=Nkx2-1-2, id=49 removed by soft count = 61\n",
      "--- gene=Nkx2-1-2, id=50 removed by soft count = 77\n",
      "--- gene=Nkx2-1-2, id=52 removed by soft count = 70\n",
      "--- gene=Nkx2-1-2, id=61 removed by soft count = 470\n",
      "--- number of probes: 62 , kept by blast: 57 , if remove dups: 57\n",
      "-- number of probes kept for this region: 57\n",
      "-- checking probes in region: Nkx2-1-3\n",
      "--- gene=Nkx2-1-3, id=7 removed by soft count = 277\n",
      "--- gene=Nkx2-1-3, id=48 removed by soft count = 74\n",
      "--- number of probes: 86 , kept by blast: 84 , if remove dups: 84\n",
      "-- number of probes kept for this region: 84\n",
      "-- checking probes in region: Nkx2-1-4\n",
      "--- number of probes: 137 , kept by blast: 137 , if remove dups: 137\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 137\n",
      "-- checking probes in region: Nkx2-1-5\n",
      "--- number of probes: 120 , kept by blast: 120 , if remove dups: 120\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: Nkx2-1-6\n",
      "--- number of probes: 141 , kept by blast: 141 , if remove dups: 141\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 141\n",
      "-- checking probes in region: Nkx2-1-7\n",
      "--- gene=Nkx2-1-7, id=0 removed by soft count = 43\n",
      "--- gene=Nkx2-1-7, id=26 removed by soft count = 56\n",
      "--- gene=Nkx2-1-7, id=32 removed by soft count = 60\n",
      "--- gene=Nkx2-1-7, id=36 removed by soft count = 98\n",
      "--- gene=Nkx2-1-7, id=48 removed by soft count = 124\n",
      "--- number of probes: 138 , kept by blast: 133 , if remove dups: 133\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 133\n",
      "-- checking probes in region: Nkx2-1-8\n",
      "--- gene=Nkx2-1-8, id=32 removed by hard count = 9\n",
      "--- gene=Nkx2-1-8, id=33 removed by soft count = 54\n",
      "--- gene=Nkx2-1-8, id=34 removed by soft count = 53\n",
      "--- gene=Nkx2-1-8, id=35 removed by soft count = 50\n",
      "--- gene=Nkx2-1-8, id=36 removed by soft count = 45\n",
      "--- gene=Nkx2-1-8, id=37 removed by hard count = 8\n",
      "--- gene=Nkx2-1-8, id=38 removed by hard count = 11\n",
      "--- gene=Nkx2-1-8, id=92 removed by soft count = 75\n",
      "--- number of probes: 118 , kept by blast: 110 , if remove dups: 110\n",
      "-- number of probes kept for this region: 110\n",
      "-- checking probes in region: Nkx2-1-9\n",
      "--- gene=Nkx2-1-9, id=0 removed by soft count = 339\n",
      "--- gene=Nkx2-1-9, id=1 removed by soft count = 852\n",
      "--- gene=Nkx2-1-9, id=2 removed by soft count = 40\n",
      "--- gene=Nkx2-1-9, id=55 removed by soft count = 145\n",
      "--- number of probes: 126 , kept by blast: 122 , if remove dups: 122\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 122\n",
      "-- checking probes in region: Nkx2-1-10\n",
      "--- number of probes: 129 , kept by blast: 129 , if remove dups: 129\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 129\n",
      "-- checking probes in region: Nkx2-1-11\n",
      "--- gene=Nkx2-1-11, id=8 removed by soft count = 55\n",
      "--- gene=Nkx2-1-11, id=37 removed by soft count = 88\n",
      "--- number of probes: 128 , kept by blast: 126 , if remove dups: 126\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 126\n",
      "-- checking probes in region: Nkx2-1-12\n",
      "--- gene=Nkx2-1-12, id=42 removed by soft count = 76\n",
      "--- gene=Nkx2-1-12, id=77 removed by soft count = 204\n",
      "--- gene=Nkx2-1-12, id=78 removed by soft count = 32\n",
      "--- number of probes: 118 , kept by blast: 115 , if remove dups: 115\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 115\n",
      "-- checking probes in region: Nkx2-1-13\n",
      "--- gene=Nkx2-1-13, id=68 removed by soft count = 51\n",
      "--- number of probes: 134 , kept by blast: 133 , if remove dups: 133\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 133\n",
      "-- checking probes in region: Nkx2-1-14\n",
      "--- gene=Nkx2-1-14, id=1 removed by soft count = 48\n",
      "--- gene=Nkx2-1-14, id=62 removed by soft count = 150\n",
      "--- number of probes: 129 , kept by blast: 127 , if remove dups: 127\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 127\n",
      "-- checking probes in region: Nkx2-1-15\n",
      "--- gene=Nkx2-1-15, id=111 removed by soft count = 105\n",
      "--- gene=Nkx2-1-15, id=124 removed by soft count = 33\n",
      "--- gene=Nkx2-1-15, id=127 removed by soft count = 103\n",
      "--- number of probes: 129 , kept by blast: 126 , if remove dups: 126\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 126\n",
      "-- checking probes in region: Nkx2-1-16\n",
      "--- number of probes: 125 , kept by blast: 125 , if remove dups: 125\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 125\n",
      "-- checking probes in region: Nkx2-1-17\n",
      "--- number of probes: 138 , kept by blast: 138 , if remove dups: 137\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 137\n",
      "-- checking probes in region: Nkx2-1-18\n",
      "--- gene=Nkx2-1-18, id=55 removed by soft count = 59\n",
      "--- number of probes: 146 , kept by blast: 145 , if remove dups: 145\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 145\n",
      "-- checking probes in region: Nkx2-1-19\n",
      "--- number of probes: 153 , kept by blast: 153 , if remove dups: 153\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 150\n",
      "-- checking probes in region: Nkx2-1-20\n",
      "--- gene=Nkx2-1-20, id=103 removed by soft count = 43\n",
      "--- gene=Nkx2-1-20, id=122 removed by soft count = 66\n",
      "--- number of probes: 123 , kept by blast: 121 , if remove dups: 121\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 121\n",
      "-- checking probes in region: Nkx2-1-21\n",
      "--- gene=Nkx2-1-21, id=11 removed by soft count = 45\n",
      "--- gene=Nkx2-1-21, id=22 removed by soft count = 216\n",
      "--- gene=Nkx2-1-21, id=23 removed by soft count = 576\n",
      "--- gene=Nkx2-1-21, id=52 removed by soft count = 11203\n",
      "--- gene=Nkx2-1-21, id=99 removed by soft count = 415\n",
      "--- number of probes: 100 , kept by blast: 95 , if remove dups: 95\n",
      "-- number of probes kept for this region: 95\n",
      "-- checking probes in region: Nkx2-1-22\n",
      "--- gene=Nkx2-1-22, id=0 removed by soft count = 276\n",
      "--- gene=Nkx2-1-22, id=1 removed by soft count = 145\n",
      "--- gene=Nkx2-1-22, id=9 removed by soft count = 89\n",
      "--- gene=Nkx2-1-22, id=10 removed by soft count = 190\n",
      "--- gene=Nkx2-1-22, id=13 removed by soft count = 52\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Nkx2-1-22, id=14 removed by soft count = 1384\n",
      "--- gene=Nkx2-1-22, id=51 removed by soft count = 51\n",
      "--- number of probes: 53 , kept by blast: 46 , if remove dups: 46\n",
      "-- number of probes kept for this region: 46\n",
      "-- checking probes in region: Nkx2-1-23\n",
      "--- gene=Nkx2-1-23, id=2 removed by soft count = 38\n",
      "--- gene=Nkx2-1-23, id=23 removed by soft count = 242\n",
      "--- gene=Nkx2-1-23, id=48 removed by soft count = 74\n",
      "--- gene=Nkx2-1-23, id=69 removed by soft count = 50\n",
      "--- gene=Nkx2-1-23, id=82 removed by soft count = 61\n",
      "--- gene=Nkx2-1-23, id=83 removed by soft count = 949\n",
      "--- number of probes: 87 , kept by blast: 81 , if remove dups: 81\n",
      "-- number of probes kept for this region: 81\n",
      "-- checking probes in region: Nkx2-1-24\n",
      "--- gene=Nkx2-1-24, id=5 removed by soft count = 330\n",
      "--- gene=Nkx2-1-24, id=60 removed by soft count = 620\n",
      "--- gene=Nkx2-1-24, id=61 removed by soft count = 85\n",
      "--- number of probes: 76 , kept by blast: 73 , if remove dups: 73\n",
      "-- number of probes kept for this region: 73\n",
      "-- checking probes in region: Nkx2-1-25\n",
      "--- gene=Nkx2-1-25, id=97 removed by soft count = 58\n",
      "--- gene=Nkx2-1-25, id=98 removed by soft count = 270\n",
      "--- gene=Nkx2-1-25, id=100 removed by soft count = 59\n",
      "--- gene=Nkx2-1-25, id=103 removed by soft count = 82\n",
      "--- gene=Nkx2-1-25, id=106 removed by soft count = 67\n",
      "--- number of probes: 111 , kept by blast: 106 , if remove dups: 106\n",
      "-- number of probes kept for this region: 106\n",
      "-- checking probes in region: Nkx2-1-26\n",
      "--- gene=Nkx2-1-26, id=0 removed by soft count = 33\n",
      "--- gene=Nkx2-1-26, id=7 removed by soft count = 74\n",
      "--- gene=Nkx2-1-26, id=12 removed by soft count = 40\n",
      "--- gene=Nkx2-1-26, id=13 removed by soft count = 58\n",
      "--- gene=Nkx2-1-26, id=25 removed by soft count = 189\n",
      "--- number of probes: 121 , kept by blast: 116 , if remove dups: 116\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 116\n",
      "-- checking probes in region: Nkx2-1-27\n",
      "--- number of probes: 144 , kept by blast: 144 , if remove dups: 143\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 143\n",
      "-- checking probes in region: Nkx2-1-28\n",
      "--- gene=Nkx2-1-28, id=47 removed by soft count = 123\n",
      "--- gene=Nkx2-1-28, id=48 removed by soft count = 332\n",
      "--- gene=Nkx2-1-28, id=49 removed by soft count = 33\n",
      "--- gene=Nkx2-1-28, id=50 removed by soft count = 85\n",
      "--- gene=Nkx2-1-28, id=52 removed by soft count = 76\n",
      "--- number of probes: 116 , kept by blast: 111 , if remove dups: 111\n",
      "-- number of probes kept for this region: 111\n",
      "-- checking probes in region: Nkx2-1-29\n",
      "--- gene=Nkx2-1-29, id=124 removed by soft count = 84\n",
      "--- gene=Nkx2-1-29, id=128 removed by soft count = 236\n",
      "--- number of probes: 141 , kept by blast: 139 , if remove dups: 139\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 139\n",
      "-- checking probes in region: Nkx2-1-30\n",
      "--- gene=Nkx2-1-30, id=39 removed by soft count = 75\n",
      "--- gene=Nkx2-1-30, id=67 removed by soft count = 40\n",
      "--- gene=Nkx2-1-30, id=74 removed by soft count = 87\n",
      "--- gene=Nkx2-1-30, id=79 removed by soft count = 45\n",
      "--- gene=Nkx2-1-30, id=80 removed by soft count = 147\n",
      "--- gene=Nkx2-1-30, id=120 removed by soft count = 133\n",
      "--- number of probes: 125 , kept by blast: 119 , if remove dups: 119\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 119\n",
      "-- checking probes in region: Nkx2-1-31\n",
      "--- gene=Nkx2-1-31, id=7 removed by soft count = 82\n",
      "--- gene=Nkx2-1-31, id=17 removed by soft count = 134\n",
      "--- gene=Nkx2-1-31, id=38 removed by soft count = 320\n",
      "--- gene=Nkx2-1-31, id=42 removed by soft count = 107\n",
      "--- gene=Nkx2-1-31, id=43 removed by soft count = 33\n",
      "--- gene=Nkx2-1-31, id=50 removed by soft count = 172\n",
      "--- gene=Nkx2-1-31, id=51 removed by soft count = 335\n",
      "--- gene=Nkx2-1-31, id=54 removed by soft count = 838\n",
      "--- gene=Nkx2-1-31, id=64 removed by soft count = 393\n",
      "--- gene=Nkx2-1-31, id=67 removed by soft count = 49\n",
      "--- gene=Nkx2-1-31, id=73 removed by soft count = 3339\n",
      "--- gene=Nkx2-1-31, id=76 removed by soft count = 33\n",
      "--- gene=Nkx2-1-31, id=77 removed by soft count = 86\n",
      "--- gene=Nkx2-1-31, id=92 removed by soft count = 169\n",
      "--- number of probes: 94 , kept by blast: 80 , if remove dups: 80\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: Nkx2-1-32\n",
      "--- gene=Nkx2-1-32, id=0 removed by soft count = 60\n",
      "--- gene=Nkx2-1-32, id=4 removed by soft count = 31\n",
      "--- gene=Nkx2-1-32, id=38 removed by soft count = 146\n",
      "--- gene=Nkx2-1-32, id=41 removed by soft count = 101\n",
      "--- gene=Nkx2-1-32, id=54 removed by soft count = 97\n",
      "--- number of probes: 102 , kept by blast: 97 , if remove dups: 97\n",
      "-- number of probes kept for this region: 97\n",
      "-- checking probes in region: Nkx2-1-33\n",
      "--- gene=Nkx2-1-33, id=102 removed by soft count = 36\n",
      "--- gene=Nkx2-1-33, id=104 removed by soft count = 149\n",
      "--- gene=Nkx2-1-33, id=105 removed by soft count = 32\n",
      "--- gene=Nkx2-1-33, id=119 removed by soft count = 38\n",
      "--- number of probes: 135 , kept by blast: 131 , if remove dups: 131\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 131\n",
      "-- checking probes in region: Nkx2-1-34\n",
      "--- gene=Nkx2-1-34, id=0 removed by soft count = 130\n",
      "--- number of probes: 149 , kept by blast: 148 , if remove dups: 148\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 148\n",
      "-- checking probes in region: Nkx2-1-35\n",
      "--- number of probes: 154 , kept by blast: 154 , if remove dups: 154\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 150\n",
      "-- checking probes in region: Nkx2-1-36\n",
      "--- gene=Nkx2-1-36, id=70 removed by hard count = 3\n",
      "--- number of probes: 135 , kept by blast: 134 , if remove dups: 133\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 133\n",
      "-- checking probes in region: Nkx2-1-37\n",
      "--- gene=Nkx2-1-37, id=112 removed by hard count = 0\n",
      "--- number of probes: 133 , kept by blast: 132 , if remove dups: 132\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 132\n",
      "-- checking probes in region: Nkx2-1-38\n",
      "--- gene=Nkx2-1-38, id=74 removed by soft count = 221\n",
      "--- number of probes: 130 , kept by blast: 129 , if remove dups: 129\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 129\n",
      "-- checking probes in region: Nkx2-1-39\n",
      "--- gene=Nkx2-1-39, id=13 removed by soft count = 188\n",
      "--- number of probes: 125 , kept by blast: 124 , if remove dups: 124\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 124\n",
      "-- checking probes in region: Nkx2-1-40\n",
      "--- gene=Nkx2-1-40, id=4 removed by soft count = 39\n",
      "--- gene=Nkx2-1-40, id=6 removed by soft count = 157\n",
      "--- gene=Nkx2-1-40, id=7 removed by soft count = 68\n",
      "--- gene=Nkx2-1-40, id=8 removed by soft count = 47\n",
      "--- gene=Nkx2-1-40, id=10 removed by soft count = 99\n",
      "--- gene=Nkx2-1-40, id=26 removed by soft count = 124\n",
      "--- gene=Nkx2-1-40, id=27 removed by soft count = 1211\n",
      "--- gene=Nkx2-1-40, id=28 removed by soft count = 219\n",
      "--- gene=Nkx2-1-40, id=29 removed by soft count = 127\n",
      "--- gene=Nkx2-1-40, id=32 removed by soft count = 576\n",
      "--- gene=Nkx2-1-40, id=33 removed by soft count = 139\n",
      "--- number of probes: 58 , kept by blast: 47 , if remove dups: 47\n",
      "-- number of probes kept for this region: 47\n",
      "-- checking probes in region: Nkx2-1-41\n",
      "--- number of probes: 101 , kept by blast: 101 , if remove dups: 101\n",
      "-- number of probes kept for this region: 101\n",
      "-- checking probes in region: Nkx2-1-42\n",
      "--- gene=Nkx2-1-42, id=7 removed by soft count = 99\n",
      "--- gene=Nkx2-1-42, id=15 removed by soft count = 82\n",
      "--- gene=Nkx2-1-42, id=17 removed by soft count = 39\n",
      "--- gene=Nkx2-1-42, id=96 removed by soft count = 426\n",
      "--- gene=Nkx2-1-42, id=97 removed by soft count = 111\n",
      "--- number of probes: 118 , kept by blast: 113 , if remove dups: 113\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 113\n",
      "-- checking probes in region: Nkx2-1-43\n",
      "--- gene=Nkx2-1-43, id=20 removed by soft count = 120\n",
      "--- gene=Nkx2-1-43, id=106 removed by soft count = 42\n",
      "--- number of probes: 123 , kept by blast: 121 , if remove dups: 121\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 121\n",
      "-- checking probes in region: Nkx2-1-44\n",
      "--- gene=Nkx2-1-44, id=79 removed by hard count = 3\n",
      "--- number of probes: 112 , kept by blast: 111 , if remove dups: 110\n",
      "-- number of probes kept for this region: 111\n",
      "-- checking probes in region: Nkx2-1-45\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Nkx2-1-45, id=89 removed by soft count = 160\n",
      "--- number of probes: 133 , kept by blast: 132 , if remove dups: 132\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 132\n",
      "-- checking probes in region: Nkx2-1-46\n",
      "--- number of probes: 138 , kept by blast: 138 , if remove dups: 138\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 138\n",
      "-- checking probes in region: Nkx2-1-47\n",
      "--- number of probes: 130 , kept by blast: 130 , if remove dups: 130\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 130\n",
      "-- checking probes in region: Nkx2-1-48\n",
      "--- gene=Nkx2-1-48, id=1 removed by soft count = 58\n",
      "--- gene=Nkx2-1-48, id=4 removed by soft count = 205\n",
      "--- number of probes: 132 , kept by blast: 130 , if remove dups: 130\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 130\n",
      "-- checking probes in region: Nkx2-1-49\n",
      "--- gene=Nkx2-1-49, id=7 removed by soft count = 112\n",
      "--- gene=Nkx2-1-49, id=95 removed by soft count = 6544\n",
      "--- number of probes: 100 , kept by blast: 98 , if remove dups: 98\n",
      "-- number of probes kept for this region: 98\n",
      "-- checking probes in region: Nkx2-1-50\n",
      "--- gene=Nkx2-1-50, id=3 removed by soft count = 95\n",
      "--- gene=Nkx2-1-50, id=83 removed by soft count = 81\n",
      "--- gene=Nkx2-1-50, id=84 removed by soft count = 32\n",
      "--- gene=Nkx2-1-50, id=85 removed by soft count = 31\n",
      "--- gene=Nkx2-1-50, id=88 removed by soft count = 54\n",
      "--- gene=Nkx2-1-50, id=89 removed by soft count = 58\n",
      "--- number of probes: 90 , kept by blast: 84 , if remove dups: 84\n",
      "-- number of probes kept for this region: 84\n",
      "-- checking probes in region: Nkx2-1-51\n",
      "--- number of probes: 122 , kept by blast: 122 , if remove dups: 122\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 122\n",
      "-- checking probes in region: Nkx2-1-52\n",
      "--- gene=Nkx2-1-52, id=72 removed by soft count = 14674\n",
      "--- gene=Nkx2-1-52, id=109 removed by soft count = 45\n",
      "--- gene=Nkx2-1-52, id=110 removed by soft count = 96\n",
      "--- gene=Nkx2-1-52, id=138 removed by soft count = 43\n",
      "--- number of probes: 139 , kept by blast: 135 , if remove dups: 135\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 135\n",
      "-- checking probes in region: Nkx2-1-53\n",
      "--- gene=Nkx2-1-53, id=46 removed by hard count = 3\n",
      "--- number of probes: 123 , kept by blast: 122 , if remove dups: 122\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 122\n",
      "-- checking probes in region: Nkx2-1-54\n",
      "--- gene=Nkx2-1-54, id=36 removed by soft count = 49\n",
      "--- gene=Nkx2-1-54, id=41 removed by soft count = 76\n",
      "--- gene=Nkx2-1-54, id=55 removed by soft count = 46\n",
      "--- number of probes: 128 , kept by blast: 125 , if remove dups: 125\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 125\n",
      "-- checking probes in region: Nkx2-1-55\n",
      "--- gene=Nkx2-1-55, id=54 removed by soft count = 127\n",
      "--- number of probes: 138 , kept by blast: 137 , if remove dups: 137\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 137\n",
      "-- checking probes in region: Nkx2-1-56\n",
      "--- gene=Nkx2-1-56, id=68 removed by soft count = 49\n",
      "--- gene=Nkx2-1-56, id=69 removed by soft count = 68\n",
      "--- number of probes: 137 , kept by blast: 135 , if remove dups: 135\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 135\n",
      "-- checking probes in region: Nkx2-1-57\n",
      "--- gene=Nkx2-1-57, id=31 removed by soft count = 68\n",
      "--- number of probes: 100 , kept by blast: 99 , if remove dups: 99\n",
      "-- number of probes kept for this region: 99\n",
      "-- checking probes in region: Nkx2-1-58\n",
      "--- gene=Nkx2-1-58, id=88 removed by soft count = 298\n",
      "--- number of probes: 122 , kept by blast: 121 , if remove dups: 121\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 121\n",
      "-- checking probes in region: Nkx2-1-59\n",
      "--- number of probes: 125 , kept by blast: 125 , if remove dups: 125\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 125\n",
      "-- checking probes in region: Nkx2-1-60\n",
      "--- gene=Nkx2-1-60, id=18 removed by soft count = 60\n",
      "--- gene=Nkx2-1-60, id=19 removed by soft count = 192\n",
      "--- gene=Nkx2-1-60, id=20 removed by soft count = 316\n",
      "--- gene=Nkx2-1-60, id=21 removed by soft count = 1847\n",
      "--- gene=Nkx2-1-60, id=48 removed by soft count = 44\n",
      "--- gene=Nkx2-1-60, id=67 removed by soft count = 95\n",
      "--- gene=Nkx2-1-60, id=68 removed by soft count = 34\n",
      "--- gene=Nkx2-1-60, id=69 removed by soft count = 66\n",
      "--- number of probes: 86 , kept by blast: 78 , if remove dups: 78\n",
      "-- number of probes kept for this region: 78\n",
      "-- checking probes in region: Slc17a7-1\n",
      "--- gene=Slc17a7-1, id=24 removed by soft count = 47\n",
      "--- gene=Slc17a7-1, id=28 removed by soft count = 31\n",
      "--- gene=Slc17a7-1, id=40 removed by soft count = 596\n",
      "--- gene=Slc17a7-1, id=64 removed by soft count = 49\n",
      "--- gene=Slc17a7-1, id=65 removed by soft count = 36\n",
      "--- gene=Slc17a7-1, id=66 removed by soft count = 49\n",
      "--- gene=Slc17a7-1, id=68 removed by soft count = 77\n",
      "--- gene=Slc17a7-1, id=70 removed by soft count = 61\n",
      "--- gene=Slc17a7-1, id=71 removed by soft count = 65\n",
      "--- gene=Slc17a7-1, id=72 removed by soft count = 66\n",
      "--- gene=Slc17a7-1, id=73 removed by soft count = 46\n",
      "--- gene=Slc17a7-1, id=100 removed by soft count = 50\n",
      "--- gene=Slc17a7-1, id=102 removed by soft count = 100\n",
      "--- number of probes: 104 , kept by blast: 91 , if remove dups: 91\n",
      "-- number of probes kept for this region: 91\n",
      "-- checking probes in region: Slc17a7-2\n",
      "--- gene=Slc17a7-2, id=40 removed by soft count = 263\n",
      "--- gene=Slc17a7-2, id=50 removed by soft count = 40\n",
      "--- gene=Slc17a7-2, id=51 removed by soft count = 306\n",
      "--- gene=Slc17a7-2, id=52 removed by soft count = 31\n",
      "--- gene=Slc17a7-2, id=53 removed by soft count = 60\n",
      "--- gene=Slc17a7-2, id=55 removed by soft count = 62\n",
      "--- gene=Slc17a7-2, id=61 removed by soft count = 153\n",
      "--- gene=Slc17a7-2, id=62 removed by soft count = 114\n",
      "--- number of probes: 99 , kept by blast: 91 , if remove dups: 91\n",
      "-- number of probes kept for this region: 91\n",
      "-- checking probes in region: Slc17a7-3\n",
      "--- gene=Slc17a7-3, id=30 removed by soft count = 40\n",
      "--- gene=Slc17a7-3, id=69 removed by soft count = 39\n",
      "--- gene=Slc17a7-3, id=71 removed by soft count = 48\n",
      "--- gene=Slc17a7-3, id=96 removed by soft count = 57\n",
      "--- number of probes: 137 , kept by blast: 133 , if remove dups: 133\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 133\n",
      "-- checking probes in region: Slc17a7-4\n",
      "--- gene=Slc17a7-4, id=32 removed by soft count = 67\n",
      "--- gene=Slc17a7-4, id=47 removed by soft count = 49\n",
      "--- gene=Slc17a7-4, id=51 removed by soft count = 47\n",
      "--- gene=Slc17a7-4, id=52 removed by soft count = 37\n",
      "--- gene=Slc17a7-4, id=54 removed by soft count = 39\n",
      "--- gene=Slc17a7-4, id=94 removed by soft count = 151\n",
      "--- gene=Slc17a7-4, id=95 removed by soft count = 270\n",
      "--- gene=Slc17a7-4, id=96 removed by soft count = 125\n",
      "--- number of probes: 110 , kept by blast: 102 , if remove dups: 102\n",
      "-- number of probes kept for this region: 102\n",
      "-- checking probes in region: Slc17a7-5\n",
      "--- gene=Slc17a7-5, id=13 removed by soft count = 46\n",
      "--- gene=Slc17a7-5, id=60 removed by soft count = 112\n",
      "--- gene=Slc17a7-5, id=64 removed by soft count = 78\n",
      "--- gene=Slc17a7-5, id=69 removed by soft count = 617\n",
      "--- gene=Slc17a7-5, id=72 removed by soft count = 64\n",
      "--- gene=Slc17a7-5, id=95 removed by soft count = 48\n",
      "--- gene=Slc17a7-5, id=97 removed by soft count = 66\n",
      "--- gene=Slc17a7-5, id=98 removed by soft count = 99\n",
      "--- gene=Slc17a7-5, id=99 removed by soft count = 107\n",
      "--- number of probes: 124 , kept by blast: 115 , if remove dups: 115\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 115\n",
      "-- checking probes in region: Slc17a7-6\n",
      "--- gene=Slc17a7-6, id=32 removed by soft count = 71\n",
      "--- gene=Slc17a7-6, id=33 removed by soft count = 65\n",
      "--- gene=Slc17a7-6, id=35 removed by soft count = 65\n",
      "--- gene=Slc17a7-6, id=46 removed by soft count = 34\n",
      "--- gene=Slc17a7-6, id=47 removed by soft count = 96\n",
      "--- gene=Slc17a7-6, id=48 removed by soft count = 102\n",
      "--- gene=Slc17a7-6, id=49 removed by soft count = 125\n",
      "--- gene=Slc17a7-6, id=50 removed by soft count = 131\n",
      "--- gene=Slc17a7-6, id=51 removed by soft count = 196\n",
      "--- gene=Slc17a7-6, id=52 removed by soft count = 49\n",
      "--- number of probes: 53 , kept by blast: 43 , if remove dups: 42\n",
      "-- number of probes kept for this region: 43\n",
      "-- checking probes in region: Slc17a7-7\n",
      "--- gene=Slc17a7-7, id=0 removed by soft count = 207\n",
      "--- gene=Slc17a7-7, id=1 removed by soft count = 52\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Slc17a7-7, id=49 removed by soft count = 38\n",
      "--- gene=Slc17a7-7, id=54 removed by soft count = 33\n",
      "--- gene=Slc17a7-7, id=56 removed by soft count = 39\n",
      "--- gene=Slc17a7-7, id=58 removed by soft count = 41\n",
      "--- number of probes: 66 , kept by blast: 60 , if remove dups: 60\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: Slc17a7-8\n",
      "--- gene=Slc17a7-8, id=0 removed by soft count = 187\n",
      "--- number of probes: 141 , kept by blast: 140 , if remove dups: 140\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 140\n",
      "-- checking probes in region: Slc17a7-9\n",
      "--- gene=Slc17a7-9, id=109 removed by soft count = 42\n",
      "--- number of probes: 146 , kept by blast: 145 , if remove dups: 145\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 145\n",
      "-- checking probes in region: Slc17a7-10\n",
      "--- gene=Slc17a7-10, id=21 removed by soft count = 95\n",
      "--- gene=Slc17a7-10, id=54 removed by hard count = 5\n",
      "--- number of probes: 112 , kept by blast: 110 , if remove dups: 110\n",
      "-- number of probes kept for this region: 110\n",
      "-- checking probes in region: Slc17a7-11\n",
      "--- gene=Slc17a7-11, id=59 removed by soft count = 147\n",
      "--- gene=Slc17a7-11, id=102 removed by soft count = 35\n",
      "--- number of probes: 137 , kept by blast: 135 , if remove dups: 135\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 135\n",
      "-- checking probes in region: Slc17a7-12\n",
      "--- gene=Slc17a7-12, id=94 removed by soft count = 48\n",
      "--- number of probes: 148 , kept by blast: 147 , if remove dups: 147\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 147\n",
      "-- checking probes in region: Slc17a7-13\n",
      "--- gene=Slc17a7-13, id=32 removed by hard count = 4\n",
      "--- gene=Slc17a7-13, id=33 removed by hard count = 3\n",
      "--- gene=Slc17a7-13, id=34 removed by hard count = 3\n",
      "--- number of probes: 160 , kept by blast: 157 , if remove dups: 156\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 150\n",
      "-- checking probes in region: Slc17a7-14\n",
      "--- gene=Slc17a7-14, id=21 removed by soft count = 85\n",
      "--- gene=Slc17a7-14, id=22 removed by soft count = 42\n",
      "--- gene=Slc17a7-14, id=24 removed by soft count = 129\n",
      "--- gene=Slc17a7-14, id=26 removed by soft count = 96\n",
      "--- number of probes: 123 , kept by blast: 119 , if remove dups: 119\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 119\n",
      "-- checking probes in region: Slc17a7-15\n",
      "--- number of probes: 140 , kept by blast: 140 , if remove dups: 140\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 140\n",
      "-- checking probes in region: Slc17a7-16\n",
      "--- gene=Slc17a7-16, id=54 removed by soft count = 85\n",
      "--- gene=Slc17a7-16, id=55 removed by soft count = 36\n",
      "--- number of probes: 144 , kept by blast: 142 , if remove dups: 142\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 142\n",
      "-- checking probes in region: Slc17a7-17\n",
      "--- number of probes: 148 , kept by blast: 148 , if remove dups: 148\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 148\n",
      "-- checking probes in region: Slc17a7-18\n",
      "--- gene=Slc17a7-18, id=113 removed by hard count = 5\n",
      "--- number of probes: 126 , kept by blast: 125 , if remove dups: 125\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 125\n",
      "-- checking probes in region: Slc17a7-19\n",
      "--- gene=Slc17a7-19, id=3 removed by soft count = 1642\n",
      "--- gene=Slc17a7-19, id=60 removed by soft count = 122\n",
      "--- number of probes: 114 , kept by blast: 112 , if remove dups: 112\n",
      "-- number of probes kept for this region: 112\n",
      "-- checking probes in region: Slc17a7-20\n",
      "--- gene=Slc17a7-20, id=23 removed by soft count = 161\n",
      "--- gene=Slc17a7-20, id=28 removed by soft count = 48\n",
      "--- gene=Slc17a7-20, id=30 removed by soft count = 52\n",
      "--- gene=Slc17a7-20, id=42 removed by soft count = 79\n",
      "--- gene=Slc17a7-20, id=43 removed by soft count = 79\n",
      "--- gene=Slc17a7-20, id=59 removed by soft count = 32\n",
      "--- gene=Slc17a7-20, id=70 removed by soft count = 51\n",
      "--- number of probes: 94 , kept by blast: 87 , if remove dups: 87\n",
      "-- number of probes kept for this region: 87\n",
      "-- checking probes in region: Slc17a7-21\n",
      "--- gene=Slc17a7-21, id=0 removed by soft count = 222\n",
      "--- gene=Slc17a7-21, id=1 removed by soft count = 166\n",
      "--- gene=Slc17a7-21, id=10 removed by soft count = 50\n",
      "--- gene=Slc17a7-21, id=38 removed by soft count = 135\n",
      "--- gene=Slc17a7-21, id=57 removed by soft count = 419\n",
      "--- gene=Slc17a7-21, id=88 removed by soft count = 2429\n",
      "--- number of probes: 92 , kept by blast: 86 , if remove dups: 86\n",
      "-- number of probes kept for this region: 86\n",
      "-- checking probes in region: Slc17a7-22\n",
      "--- gene=Slc17a7-22, id=6 removed by soft count = 3216\n",
      "--- gene=Slc17a7-22, id=44 removed by soft count = 203\n",
      "--- gene=Slc17a7-22, id=69 removed by soft count = 38\n",
      "--- gene=Slc17a7-22, id=81 removed by soft count = 53\n",
      "--- gene=Slc17a7-22, id=85 removed by soft count = 168\n",
      "--- gene=Slc17a7-22, id=107 removed by soft count = 33\n",
      "--- number of probes: 130 , kept by blast: 124 , if remove dups: 124\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 124\n",
      "-- checking probes in region: Slc17a7-23\n",
      "--- gene=Slc17a7-23, id=82 removed by soft count = 105\n",
      "--- number of probes: 119 , kept by blast: 118 , if remove dups: 118\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 118\n",
      "-- checking probes in region: Slc17a7-24\n",
      "--- number of probes: 127 , kept by blast: 127 , if remove dups: 127\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 127\n",
      "-- checking probes in region: Slc17a7-25\n",
      "--- gene=Slc17a7-25, id=105 removed by soft count = 41\n",
      "--- number of probes: 122 , kept by blast: 121 , if remove dups: 121\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 121\n",
      "-- checking probes in region: Slc17a7-26\n",
      "--- gene=Slc17a7-26, id=5 removed by soft count = 373\n",
      "--- gene=Slc17a7-26, id=7 removed by soft count = 131\n",
      "--- gene=Slc17a7-26, id=41 removed by soft count = 437\n",
      "--- gene=Slc17a7-26, id=53 removed by soft count = 267\n",
      "--- gene=Slc17a7-26, id=55 removed by soft count = 31\n",
      "--- gene=Slc17a7-26, id=75 removed by soft count = 32\n",
      "--- gene=Slc17a7-26, id=122 removed by soft count = 43\n",
      "--- number of probes: 134 , kept by blast: 127 , if remove dups: 127\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 127\n",
      "-- checking probes in region: Slc17a7-27\n",
      "--- gene=Slc17a7-27, id=58 removed by soft count = 34\n",
      "--- number of probes: 147 , kept by blast: 146 , if remove dups: 144\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 144\n",
      "-- checking probes in region: Slc17a7-28\n",
      "--- number of probes: 153 , kept by blast: 153 , if remove dups: 153\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 150\n",
      "-- checking probes in region: Slc17a7-29\n",
      "--- gene=Slc17a7-29, id=31 removed by soft count = 71\n",
      "--- gene=Slc17a7-29, id=34 removed by soft count = 72\n",
      "--- gene=Slc17a7-29, id=38 removed by soft count = 31\n",
      "--- gene=Slc17a7-29, id=51 removed by soft count = 57\n",
      "--- number of probes: 127 , kept by blast: 123 , if remove dups: 123\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 123\n",
      "-- checking probes in region: Slc17a7-30\n",
      "--- number of probes: 133 , kept by blast: 133 , if remove dups: 133\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 133\n",
      "-- checking probes in region: Slc17a7-31\n",
      "--- gene=Slc17a7-31, id=28 removed by soft count = 48\n",
      "--- number of probes: 140 , kept by blast: 139 , if remove dups: 139\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 139\n",
      "-- checking probes in region: Slc17a7-32\n",
      "--- gene=Slc17a7-32, id=109 removed by soft count = 76\n",
      "--- number of probes: 141 , kept by blast: 140 , if remove dups: 140\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 140\n",
      "-- checking probes in region: Slc17a7-33\n",
      "--- gene=Slc17a7-33, id=63 removed by soft count = 42\n",
      "--- gene=Slc17a7-33, id=70 removed by soft count = 246\n",
      "--- gene=Slc17a7-33, id=72 removed by soft count = 77\n",
      "--- gene=Slc17a7-33, id=77 removed by soft count = 34\n",
      "--- gene=Slc17a7-33, id=109 removed by soft count = 3079\n",
      "--- number of probes: 112 , kept by blast: 107 , if remove dups: 107\n",
      "-- number of probes kept for this region: 107\n",
      "-- checking probes in region: Slc17a7-34\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Slc17a7-34, id=15 removed by soft count = 294\n",
      "--- gene=Slc17a7-34, id=41 removed by soft count = 34\n",
      "--- number of probes: 125 , kept by blast: 123 , if remove dups: 123\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 123\n",
      "-- checking probes in region: Slc17a7-35\n",
      "--- gene=Slc17a7-35, id=4 removed by soft count = 1144\n",
      "--- gene=Slc17a7-35, id=5 removed by soft count = 343\n",
      "--- gene=Slc17a7-35, id=9 removed by soft count = 98\n",
      "--- gene=Slc17a7-35, id=23 removed by soft count = 34\n",
      "--- gene=Slc17a7-35, id=85 removed by soft count = 53\n",
      "--- gene=Slc17a7-35, id=96 removed by soft count = 40\n",
      "--- number of probes: 102 , kept by blast: 96 , if remove dups: 96\n",
      "-- number of probes kept for this region: 96\n",
      "-- checking probes in region: Slc17a7-36\n",
      "--- gene=Slc17a7-36, id=32 removed by soft count = 46\n",
      "--- gene=Slc17a7-36, id=54 removed by soft count = 117\n",
      "--- gene=Slc17a7-36, id=56 removed by soft count = 31\n",
      "--- gene=Slc17a7-36, id=70 removed by soft count = 119\n",
      "--- number of probes: 106 , kept by blast: 102 , if remove dups: 102\n",
      "-- number of probes kept for this region: 102\n",
      "-- checking probes in region: Slc17a7-37\n",
      "--- gene=Slc17a7-37, id=1 removed by soft count = 173\n",
      "--- gene=Slc17a7-37, id=5 removed by soft count = 160\n",
      "--- gene=Slc17a7-37, id=11 removed by soft count = 162\n",
      "--- gene=Slc17a7-37, id=13 removed by soft count = 56\n",
      "--- gene=Slc17a7-37, id=56 removed by soft count = 60\n",
      "--- gene=Slc17a7-37, id=97 removed by soft count = 176\n",
      "--- number of probes: 99 , kept by blast: 93 , if remove dups: 93\n",
      "-- number of probes kept for this region: 93\n",
      "-- checking probes in region: Slc17a7-38\n",
      "--- gene=Slc17a7-38, id=0 removed by soft count = 57\n",
      "--- gene=Slc17a7-38, id=3 removed by soft count = 170\n",
      "--- gene=Slc17a7-38, id=58 removed by soft count = 113\n",
      "--- gene=Slc17a7-38, id=80 removed by soft count = 50\n",
      "--- number of probes: 93 , kept by blast: 89 , if remove dups: 89\n",
      "-- number of probes kept for this region: 89\n",
      "-- checking probes in region: Slc17a7-39\n",
      "--- number of probes: 145 , kept by blast: 145 , if remove dups: 145\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 145\n",
      "-- checking probes in region: Slc17a7-40\n",
      "--- gene=Slc17a7-40, id=7 removed by soft count = 31\n",
      "--- gene=Slc17a7-40, id=13 removed by soft count = 54\n",
      "--- gene=Slc17a7-40, id=119 removed by soft count = 31\n",
      "--- number of probes: 121 , kept by blast: 118 , if remove dups: 118\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 118\n",
      "-- checking probes in region: Slc17a7-41\n",
      "--- number of probes: 134 , kept by blast: 134 , if remove dups: 134\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 134\n",
      "-- checking probes in region: Slc17a7-42\n",
      "--- gene=Slc17a7-42, id=65 removed by soft count = 58\n",
      "--- number of probes: 130 , kept by blast: 129 , if remove dups: 129\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 129\n",
      "-- checking probes in region: Slc17a7-43\n",
      "--- gene=Slc17a7-43, id=17 removed by soft count = 2021\n",
      "--- number of probes: 132 , kept by blast: 131 , if remove dups: 131\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 131\n",
      "-- checking probes in region: Slc17a7-44\n",
      "--- gene=Slc17a7-44, id=101 removed by soft count = 67\n",
      "--- number of probes: 141 , kept by blast: 140 , if remove dups: 140\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 140\n",
      "-- checking probes in region: Slc17a7-45\n",
      "--- number of probes: 145 , kept by blast: 145 , if remove dups: 145\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 145\n",
      "-- checking probes in region: Slc17a7-46\n",
      "--- gene=Slc17a7-46, id=20 removed by soft count = 55\n",
      "--- gene=Slc17a7-46, id=88 removed by soft count = 32\n",
      "--- number of probes: 132 , kept by blast: 130 , if remove dups: 130\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 130\n",
      "-- checking probes in region: Slc17a7-47\n",
      "--- gene=Slc17a7-47, id=27 removed by soft count = 33\n",
      "--- gene=Slc17a7-47, id=72 removed by soft count = 60\n",
      "--- gene=Slc17a7-47, id=73 removed by soft count = 88\n",
      "--- gene=Slc17a7-47, id=75 removed by soft count = 64\n",
      "--- gene=Slc17a7-47, id=77 removed by soft count = 31\n",
      "--- gene=Slc17a7-47, id=78 removed by soft count = 36\n",
      "--- gene=Slc17a7-47, id=80 removed by soft count = 70\n",
      "--- number of probes: 111 , kept by blast: 104 , if remove dups: 104\n",
      "-- number of probes kept for this region: 104\n",
      "-- checking probes in region: Slc17a7-48\n",
      "--- gene=Slc17a7-48, id=14 removed by soft count = 233\n",
      "--- gene=Slc17a7-48, id=50 removed by soft count = 983\n",
      "--- number of probes: 129 , kept by blast: 127 , if remove dups: 127\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 127\n",
      "-- checking probes in region: Slc17a7-49\n",
      "--- gene=Slc17a7-49, id=41 removed by soft count = 51\n",
      "--- gene=Slc17a7-49, id=43 removed by soft count = 258\n",
      "--- number of probes: 130 , kept by blast: 128 , if remove dups: 128\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 128\n",
      "-- checking probes in region: Slc17a7-50\n",
      "--- gene=Slc17a7-50, id=126 removed by soft count = 70\n",
      "--- number of probes: 135 , kept by blast: 134 , if remove dups: 134\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 134\n",
      "-- checking probes in region: Slc17a7-51\n",
      "--- gene=Slc17a7-51, id=29 removed by soft count = 52\n",
      "--- gene=Slc17a7-51, id=34 removed by soft count = 217\n",
      "--- gene=Slc17a7-51, id=42 removed by soft count = 82\n",
      "--- gene=Slc17a7-51, id=44 removed by soft count = 121\n",
      "--- gene=Slc17a7-51, id=46 removed by soft count = 38\n",
      "--- gene=Slc17a7-51, id=47 removed by soft count = 34\n",
      "--- gene=Slc17a7-51, id=52 removed by soft count = 134\n",
      "--- gene=Slc17a7-51, id=53 removed by soft count = 48\n",
      "--- gene=Slc17a7-51, id=56 removed by soft count = 265\n",
      "--- gene=Slc17a7-51, id=58 removed by soft count = 128\n",
      "--- number of probes: 65 , kept by blast: 55 , if remove dups: 55\n",
      "-- number of probes kept for this region: 55\n",
      "-- checking probes in region: Slc17a7-52\n",
      "--- gene=Slc17a7-52, id=0 removed by soft count = 69\n",
      "--- gene=Slc17a7-52, id=3 removed by soft count = 36\n",
      "--- gene=Slc17a7-52, id=4 removed by soft count = 46\n",
      "--- gene=Slc17a7-52, id=6 removed by soft count = 66\n",
      "--- gene=Slc17a7-52, id=26 removed by soft count = 41\n",
      "--- gene=Slc17a7-52, id=27 removed by soft count = 326\n",
      "--- number of probes: 90 , kept by blast: 84 , if remove dups: 84\n",
      "-- number of probes kept for this region: 84\n",
      "-- checking probes in region: Slc17a7-53\n",
      "--- gene=Slc17a7-53, id=33 removed by soft count = 42\n",
      "--- gene=Slc17a7-53, id=69 removed by soft count = 364\n",
      "--- number of probes: 125 , kept by blast: 123 , if remove dups: 123\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 123\n",
      "-- checking probes in region: Slc17a7-54\n",
      "--- number of probes: 120 , kept by blast: 120 , if remove dups: 118\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 118\n",
      "-- checking probes in region: Slc17a7-55\n",
      "--- gene=Slc17a7-55, id=50 removed by soft count = 39\n",
      "--- gene=Slc17a7-55, id=52 removed by soft count = 10385\n",
      "--- gene=Slc17a7-55, id=67 removed by soft count = 186\n",
      "--- number of probes: 132 , kept by blast: 129 , if remove dups: 129\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 129\n",
      "-- checking probes in region: Slc17a7-56\n",
      "--- gene=Slc17a7-56, id=22 removed by soft count = 382\n",
      "--- gene=Slc17a7-56, id=45 removed by soft count = 44\n",
      "--- gene=Slc17a7-56, id=57 removed by soft count = 80\n",
      "--- gene=Slc17a7-56, id=58 removed by soft count = 33\n",
      "--- gene=Slc17a7-56, id=66 removed by soft count = 104\n",
      "--- gene=Slc17a7-56, id=80 removed by soft count = 106\n",
      "--- gene=Slc17a7-56, id=84 removed by soft count = 78\n",
      "--- gene=Slc17a7-56, id=86 removed by soft count = 73\n",
      "--- gene=Slc17a7-56, id=88 removed by soft count = 49\n",
      "--- gene=Slc17a7-56, id=89 removed by soft count = 46\n",
      "--- gene=Slc17a7-56, id=91 removed by soft count = 43\n",
      "--- gene=Slc17a7-56, id=107 removed by soft count = 1363\n",
      "--- number of probes: 114 , kept by blast: 102 , if remove dups: 102\n",
      "-- number of probes kept for this region: 102\n",
      "-- checking probes in region: Slc17a7-57\n",
      "--- gene=Slc17a7-57, id=84 removed by soft count = 55\n",
      "--- gene=Slc17a7-57, id=94 removed by soft count = 52\n",
      "--- gene=Slc17a7-57, id=98 removed by soft count = 38\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 130 , kept by blast: 127 , if remove dups: 127\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 127\n",
      "-- checking probes in region: Slc17a7-58\n",
      "--- number of probes: 121 , kept by blast: 121 , if remove dups: 121\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 121\n",
      "-- checking probes in region: Slc17a7-59\n",
      "--- gene=Slc17a7-59, id=11 removed by soft count = 45\n",
      "--- gene=Slc17a7-59, id=12 removed by soft count = 38\n",
      "--- gene=Slc17a7-59, id=33 removed by soft count = 45\n",
      "--- gene=Slc17a7-59, id=50 removed by soft count = 34\n",
      "--- number of probes: 102 , kept by blast: 98 , if remove dups: 98\n",
      "-- number of probes kept for this region: 98\n",
      "-- checking probes in region: Slc17a7-60\n",
      "--- gene=Slc17a7-60, id=2 removed by soft count = 71\n",
      "--- number of probes: 142 , kept by blast: 141 , if remove dups: 141\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 141\n",
      "-- checking probes in region: Slc32a1-1\n",
      "--- gene=Slc32a1-1, id=21 removed by soft count = 33\n",
      "--- gene=Slc32a1-1, id=79 removed by soft count = 2510\n",
      "--- gene=Slc32a1-1, id=93 removed by soft count = 38\n",
      "--- gene=Slc32a1-1, id=127 removed by soft count = 95\n",
      "--- number of probes: 129 , kept by blast: 125 , if remove dups: 125\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 125\n",
      "-- checking probes in region: Slc32a1-2\n",
      "--- gene=Slc32a1-2, id=121 removed by soft count = 131\n",
      "--- gene=Slc32a1-2, id=123 removed by soft count = 90\n",
      "--- gene=Slc32a1-2, id=131 removed by soft count = 67\n",
      "--- number of probes: 138 , kept by blast: 135 , if remove dups: 135\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 135\n",
      "-- checking probes in region: Slc32a1-3\n",
      "--- gene=Slc32a1-3, id=34 removed by soft count = 52\n",
      "--- gene=Slc32a1-3, id=45 removed by soft count = 41\n",
      "--- number of probes: 129 , kept by blast: 127 , if remove dups: 127\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 127\n",
      "-- checking probes in region: Slc32a1-4\n",
      "--- gene=Slc32a1-4, id=46 removed by soft count = 377\n",
      "--- gene=Slc32a1-4, id=55 removed by soft count = 2617\n",
      "--- number of probes: 123 , kept by blast: 121 , if remove dups: 121\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 121\n",
      "-- checking probes in region: Slc32a1-5\n",
      "--- number of probes: 134 , kept by blast: 134 , if remove dups: 134\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 134\n",
      "-- checking probes in region: Slc32a1-6\n",
      "--- gene=Slc32a1-6, id=49 removed by soft count = 93\n",
      "--- gene=Slc32a1-6, id=51 removed by soft count = 31\n",
      "--- gene=Slc32a1-6, id=102 removed by soft count = 37\n",
      "--- number of probes: 131 , kept by blast: 128 , if remove dups: 128\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 128\n",
      "-- checking probes in region: Slc32a1-7\n",
      "--- gene=Slc32a1-7, id=91 removed by soft count = 115\n",
      "--- number of probes: 123 , kept by blast: 122 , if remove dups: 122\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 122\n",
      "-- checking probes in region: Slc32a1-8\n",
      "--- gene=Slc32a1-8, id=26 removed by soft count = 37\n",
      "--- gene=Slc32a1-8, id=27 removed by soft count = 1561\n",
      "--- gene=Slc32a1-8, id=60 removed by soft count = 135\n",
      "--- gene=Slc32a1-8, id=61 removed by soft count = 689\n",
      "--- gene=Slc32a1-8, id=65 removed by soft count = 63\n",
      "--- gene=Slc32a1-8, id=69 removed by soft count = 221\n",
      "--- gene=Slc32a1-8, id=70 removed by soft count = 110\n",
      "--- gene=Slc32a1-8, id=93 removed by soft count = 31\n",
      "--- gene=Slc32a1-8, id=94 removed by soft count = 52\n",
      "--- gene=Slc32a1-8, id=96 removed by soft count = 121\n",
      "--- number of probes: 114 , kept by blast: 104 , if remove dups: 104\n",
      "-- number of probes kept for this region: 104\n",
      "-- checking probes in region: Slc32a1-9\n",
      "--- number of probes: 123 , kept by blast: 123 , if remove dups: 123\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 123\n",
      "-- checking probes in region: Slc32a1-10\n",
      "--- gene=Slc32a1-10, id=13 removed by soft count = 116\n",
      "--- gene=Slc32a1-10, id=14 removed by soft count = 151\n",
      "--- gene=Slc32a1-10, id=92 removed by hard count = 0\n",
      "--- number of probes: 128 , kept by blast: 125 , if remove dups: 125\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 125\n",
      "-- checking probes in region: Slc32a1-11\n",
      "--- number of probes: 144 , kept by blast: 144 , if remove dups: 144\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 144\n",
      "-- checking probes in region: Slc32a1-12\n",
      "--- gene=Slc32a1-12, id=22 removed by soft count = 82\n",
      "--- gene=Slc32a1-12, id=103 removed by soft count = 89\n",
      "--- number of probes: 129 , kept by blast: 127 , if remove dups: 127\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 127\n",
      "-- checking probes in region: Slc32a1-13\n",
      "--- gene=Slc32a1-13, id=1 removed by soft count = 56\n",
      "--- number of probes: 144 , kept by blast: 143 , if remove dups: 143\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 143\n",
      "-- checking probes in region: Slc32a1-14\n",
      "--- gene=Slc32a1-14, id=3 removed by soft count = 36\n",
      "--- gene=Slc32a1-14, id=63 removed by soft count = 97\n",
      "--- gene=Slc32a1-14, id=64 removed by soft count = 86\n",
      "--- gene=Slc32a1-14, id=67 removed by soft count = 73\n",
      "--- gene=Slc32a1-14, id=70 removed by soft count = 53\n",
      "--- gene=Slc32a1-14, id=72 removed by soft count = 46\n",
      "--- number of probes: 121 , kept by blast: 115 , if remove dups: 115\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 115\n",
      "-- checking probes in region: Slc32a1-15\n",
      "--- gene=Slc32a1-15, id=78 removed by soft count = 87\n",
      "--- gene=Slc32a1-15, id=89 removed by soft count = 304\n",
      "--- number of probes: 127 , kept by blast: 125 , if remove dups: 125\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 125\n",
      "-- checking probes in region: Slc32a1-16\n",
      "--- gene=Slc32a1-16, id=2 removed by soft count = 47\n",
      "--- gene=Slc32a1-16, id=5 removed by soft count = 31\n",
      "--- gene=Slc32a1-16, id=6 removed by soft count = 817\n",
      "--- gene=Slc32a1-16, id=7 removed by soft count = 285\n",
      "--- gene=Slc32a1-16, id=10 removed by soft count = 63\n",
      "--- gene=Slc32a1-16, id=11 removed by soft count = 1683\n",
      "--- gene=Slc32a1-16, id=12 removed by soft count = 62\n",
      "--- gene=Slc32a1-16, id=17 removed by soft count = 46\n",
      "--- number of probes: 114 , kept by blast: 106 , if remove dups: 106\n",
      "-- number of probes kept for this region: 106\n",
      "-- checking probes in region: Slc32a1-17\n",
      "--- gene=Slc32a1-17, id=112 removed by soft count = 150\n",
      "--- number of probes: 141 , kept by blast: 140 , if remove dups: 140\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 140\n",
      "-- checking probes in region: Slc32a1-18\n",
      "--- gene=Slc32a1-18, id=7 removed by soft count = 81\n",
      "--- number of probes: 127 , kept by blast: 126 , if remove dups: 126\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 126\n",
      "-- checking probes in region: Slc32a1-19\n",
      "--- gene=Slc32a1-19, id=44 removed by soft count = 35\n",
      "--- gene=Slc32a1-19, id=53 removed by soft count = 63\n",
      "--- gene=Slc32a1-19, id=60 removed by soft count = 91\n",
      "--- number of probes: 136 , kept by blast: 133 , if remove dups: 133\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 133\n",
      "-- checking probes in region: Slc32a1-20\n",
      "--- gene=Slc32a1-20, id=48 removed by soft count = 57\n",
      "--- gene=Slc32a1-20, id=51 removed by soft count = 307\n",
      "--- gene=Slc32a1-20, id=61 removed by soft count = 32\n",
      "--- number of probes: 135 , kept by blast: 132 , if remove dups: 132\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 132\n",
      "-- checking probes in region: Slc32a1-21\n",
      "--- gene=Slc32a1-21, id=116 removed by soft count = 48\n",
      "--- number of probes: 144 , kept by blast: 143 , if remove dups: 143\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 143\n",
      "-- checking probes in region: SST-1\n",
      "--- gene=SST-1, id=1 removed by soft count = 735\n",
      "--- gene=SST-1, id=38 removed by soft count = 63\n",
      "--- number of probes: 105 , kept by blast: 103 , if remove dups: 103\n",
      "-- number of probes kept for this region: 103\n",
      "-- checking probes in region: SST-2\n",
      "--- gene=SST-2, id=48 removed by soft count = 497\n",
      "--- number of probes: 49 , kept by blast: 48 , if remove dups: 48\n",
      "-- number of probes kept for this region: 48\n",
      "-- checking probes in region: SST-3\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=SST-3, id=0 removed by soft count = 3751\n",
      "--- gene=SST-3, id=1 removed by soft count = 68\n",
      "--- number of probes: 53 , kept by blast: 51 , if remove dups: 51\n",
      "-- number of probes kept for this region: 51\n",
      "-- checking probes in region: SST-4\n",
      "--- gene=SST-4, id=44 removed by soft count = 40\n",
      "--- number of probes: 118 , kept by blast: 117 , if remove dups: 116\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 116\n",
      "-- checking probes in region: SST-5\n",
      "--- gene=SST-5, id=1 removed by soft count = 80\n",
      "--- gene=SST-5, id=7 removed by soft count = 64\n",
      "--- gene=SST-5, id=32 removed by soft count = 88\n",
      "--- gene=SST-5, id=37 removed by soft count = 1035\n",
      "--- gene=SST-5, id=38 removed by soft count = 40\n",
      "--- gene=SST-5, id=46 removed by soft count = 64\n",
      "--- number of probes: 114 , kept by blast: 108 , if remove dups: 108\n",
      "-- number of probes kept for this region: 108\n",
      "-- checking probes in region: SST-6\n",
      "--- number of probes: 149 , kept by blast: 149 , if remove dups: 149\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 149\n",
      "-- checking probes in region: SST-7\n",
      "--- gene=SST-7, id=102 removed by soft count = 117\n",
      "--- gene=SST-7, id=103 removed by soft count = 44\n",
      "--- gene=SST-7, id=104 removed by soft count = 33\n",
      "--- gene=SST-7, id=105 removed by soft count = 42\n",
      "--- number of probes: 143 , kept by blast: 139 , if remove dups: 139\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 139\n",
      "-- checking probes in region: SST-8\n",
      "--- gene=SST-8, id=8 removed by soft count = 56\n",
      "--- gene=SST-8, id=26 removed by soft count = 34\n",
      "--- number of probes: 130 , kept by blast: 128 , if remove dups: 128\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 128\n",
      "-- checking probes in region: SST-9\n",
      "--- number of probes: 145 , kept by blast: 145 , if remove dups: 145\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 145\n",
      "-- checking probes in region: SST-10\n",
      "--- gene=SST-10, id=10 removed by soft count = 40\n",
      "--- number of probes: 141 , kept by blast: 140 , if remove dups: 140\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 140\n",
      "-- checking probes in region: SST-11\n",
      "--- gene=SST-11, id=33 removed by soft count = 58\n",
      "--- gene=SST-11, id=34 removed by soft count = 50\n",
      "--- number of probes: 145 , kept by blast: 143 , if remove dups: 143\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 143\n",
      "-- checking probes in region: SST-12\n",
      "--- gene=SST-12, id=63 removed by soft count = 2244\n",
      "--- gene=SST-12, id=111 removed by soft count = 673\n",
      "--- number of probes: 141 , kept by blast: 139 , if remove dups: 139\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 139\n",
      "-- checking probes in region: SST-13\n",
      "--- gene=SST-13, id=78 removed by soft count = 2979\n",
      "--- number of probes: 79 , kept by blast: 78 , if remove dups: 78\n",
      "-- number of probes kept for this region: 78\n",
      "-- checking probes in region: SST-14\n",
      "--- gene=SST-14, id=8 removed by soft count = 61\n",
      "--- gene=SST-14, id=18 removed by soft count = 50\n",
      "--- gene=SST-14, id=20 removed by soft count = 69\n",
      "--- gene=SST-14, id=32 removed by soft count = 124\n",
      "--- gene=SST-14, id=88 removed by soft count = 5759\n",
      "--- gene=SST-14, id=89 removed by soft count = 1254\n",
      "--- number of probes: 95 , kept by blast: 89 , if remove dups: 89\n",
      "-- number of probes kept for this region: 89\n",
      "-- checking probes in region: SST-15\n",
      "--- gene=SST-15, id=5 removed by soft count = 932\n",
      "--- gene=SST-15, id=7 removed by soft count = 188\n",
      "--- gene=SST-15, id=8 removed by soft count = 646\n",
      "--- number of probes: 82 , kept by blast: 79 , if remove dups: 79\n",
      "-- number of probes kept for this region: 79\n",
      "-- checking probes in region: SST-16\n",
      "--- number of probes: 131 , kept by blast: 131 , if remove dups: 131\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 131\n",
      "-- checking probes in region: SST-17\n",
      "--- gene=SST-17, id=58 removed by soft count = 47\n",
      "--- gene=SST-17, id=64 removed by soft count = 824\n",
      "--- gene=SST-17, id=66 removed by soft count = 795\n",
      "--- gene=SST-17, id=68 removed by soft count = 193\n",
      "--- number of probes: 151 , kept by blast: 147 , if remove dups: 147\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 147\n",
      "-- checking probes in region: SST-18\n",
      "--- gene=SST-18, id=24 removed by soft count = 122\n",
      "--- gene=SST-18, id=25 removed by soft count = 336\n",
      "--- gene=SST-18, id=37 removed by soft count = 60\n",
      "--- gene=SST-18, id=64 removed by soft count = 35\n",
      "--- gene=SST-18, id=65 removed by soft count = 84\n",
      "--- gene=SST-18, id=68 removed by soft count = 142\n",
      "--- gene=SST-18, id=87 removed by soft count = 74\n",
      "--- gene=SST-18, id=88 removed by soft count = 9830\n",
      "--- number of probes: 96 , kept by blast: 88 , if remove dups: 88\n",
      "-- number of probes kept for this region: 88\n",
      "-- checking probes in region: SST-19\n",
      "--- gene=SST-19, id=35 removed by hard count = 0\n",
      "--- gene=SST-19, id=39 removed by soft count = 79\n",
      "--- gene=SST-19, id=40 removed by soft count = 52\n",
      "--- gene=SST-19, id=52 removed by soft count = 37\n",
      "--- gene=SST-19, id=77 removed by soft count = 31\n",
      "--- gene=SST-19, id=78 removed by soft count = 82\n",
      "--- gene=SST-19, id=79 removed by soft count = 93\n",
      "--- gene=SST-19, id=82 removed by soft count = 51\n",
      "--- gene=SST-19, id=101 removed by soft count = 72\n",
      "--- gene=SST-19, id=102 removed by soft count = 84\n",
      "--- gene=SST-19, id=103 removed by soft count = 2367\n",
      "--- gene=SST-19, id=104 removed by soft count = 184\n",
      "--- gene=SST-19, id=105 removed by soft count = 70\n",
      "--- gene=SST-19, id=112 removed by soft count = 192\n",
      "--- number of probes: 114 , kept by blast: 100 , if remove dups: 100\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: SST-20\n",
      "--- gene=SST-20, id=1 removed by soft count = 35\n",
      "--- gene=SST-20, id=2 removed by soft count = 61\n",
      "--- gene=SST-20, id=5 removed by soft count = 36\n",
      "--- gene=SST-20, id=6 removed by soft count = 124\n",
      "--- gene=SST-20, id=7 removed by soft count = 947\n",
      "--- gene=SST-20, id=10 removed by soft count = 101\n",
      "--- gene=SST-20, id=11 removed by soft count = 340\n",
      "--- gene=SST-20, id=12 removed by soft count = 43\n",
      "--- gene=SST-20, id=16 removed by soft count = 252\n",
      "--- gene=SST-20, id=19 removed by soft count = 71\n",
      "--- gene=SST-20, id=33 removed by soft count = 85\n",
      "--- gene=SST-20, id=34 removed by soft count = 87\n",
      "--- gene=SST-20, id=35 removed by soft count = 43\n",
      "--- number of probes: 114 , kept by blast: 101 , if remove dups: 101\n",
      "-- number of probes kept for this region: 101\n",
      "-- checking probes in region: SST-21\n",
      "--- gene=SST-21, id=33 removed by soft count = 32\n",
      "--- gene=SST-21, id=88 removed by soft count = 137\n",
      "--- gene=SST-21, id=89 removed by soft count = 43\n",
      "--- number of probes: 103 , kept by blast: 100 , if remove dups: 100\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: SST-22\n",
      "--- gene=SST-22, id=0 removed by soft count = 133\n",
      "--- gene=SST-22, id=8 removed by soft count = 35\n",
      "--- gene=SST-22, id=26 removed by soft count = 164\n",
      "--- gene=SST-22, id=27 removed by soft count = 149\n",
      "--- gene=SST-22, id=29 removed by soft count = 225\n",
      "--- gene=SST-22, id=30 removed by soft count = 98\n",
      "--- gene=SST-22, id=31 removed by soft count = 596\n",
      "--- gene=SST-22, id=45 removed by soft count = 34\n",
      "--- gene=SST-22, id=46 removed by soft count = 32\n",
      "--- number of probes: 61 , kept by blast: 52 , if remove dups: 52\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: SST-23\n",
      "--- gene=SST-23, id=18 removed by soft count = 54\n",
      "--- gene=SST-23, id=42 removed by soft count = 407\n",
      "--- gene=SST-23, id=58 removed by soft count = 225\n",
      "--- gene=SST-23, id=77 removed by soft count = 33\n",
      "--- gene=SST-23, id=85 removed by soft count = 87\n",
      "--- number of probes: 112 , kept by blast: 107 , if remove dups: 107\n",
      "-- number of probes kept for this region: 107\n",
      "-- checking probes in region: SST-24\n",
      "--- gene=SST-24, id=143 removed by soft count = 43\n",
      "--- number of probes: 148 , kept by blast: 147 , if remove dups: 147\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 147\n",
      "-- checking probes in region: SST-25\n",
      "--- number of probes: 140 , kept by blast: 140 , if remove dups: 140\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 140\n",
      "-- checking probes in region: SST-26\n",
      "--- number of probes: 151 , kept by blast: 151 , if remove dups: 151\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 150\n",
      "-- checking probes in region: SST-27\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 141 , kept by blast: 141 , if remove dups: 141\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 141\n",
      "-- checking probes in region: SST-28\n",
      "--- gene=SST-28, id=56 removed by soft count = 415\n",
      "--- gene=SST-28, id=94 removed by soft count = 52\n",
      "--- gene=SST-28, id=132 removed by soft count = 189\n",
      "--- number of probes: 137 , kept by blast: 134 , if remove dups: 134\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 134\n",
      "-- checking probes in region: SST-29\n",
      "--- gene=SST-29, id=90 removed by soft count = 79\n",
      "--- number of probes: 124 , kept by blast: 123 , if remove dups: 123\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 123\n",
      "-- checking probes in region: SST-30\n",
      "--- gene=SST-30, id=67 removed by soft count = 41\n",
      "--- number of probes: 141 , kept by blast: 140 , if remove dups: 140\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 140\n",
      "-- checking probes in region: SST-31\n",
      "--- gene=SST-31, id=11 removed by soft count = 63\n",
      "--- gene=SST-31, id=12 removed by soft count = 66\n",
      "--- gene=SST-31, id=13 removed by soft count = 41\n",
      "--- gene=SST-31, id=15 removed by soft count = 35\n",
      "--- gene=SST-31, id=16 removed by soft count = 145\n",
      "--- gene=SST-31, id=18 removed by soft count = 59\n",
      "--- gene=SST-31, id=19 removed by soft count = 89\n",
      "--- gene=SST-31, id=20 removed by soft count = 250\n",
      "--- gene=SST-31, id=24 removed by soft count = 32\n",
      "--- gene=SST-31, id=27 removed by soft count = 55\n",
      "--- gene=SST-31, id=37 removed by soft count = 219\n",
      "--- gene=SST-31, id=82 removed by soft count = 266\n",
      "--- number of probes: 108 , kept by blast: 96 , if remove dups: 96\n",
      "-- number of probes kept for this region: 96\n",
      "-- checking probes in region: SST-32\n",
      "--- gene=SST-32, id=2 removed by soft count = 13331\n",
      "--- gene=SST-32, id=6 removed by soft count = 223\n",
      "--- gene=SST-32, id=9 removed by soft count = 43\n",
      "--- number of probes: 139 , kept by blast: 136 , if remove dups: 136\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 136\n",
      "-- checking probes in region: SST-33\n",
      "--- number of probes: 150 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 150\n",
      "-- checking probes in region: SST-34\n",
      "--- number of probes: 148 , kept by blast: 148 , if remove dups: 148\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 148\n",
      "-- checking probes in region: SST-35\n",
      "--- gene=SST-35, id=71 removed by soft count = 228\n",
      "--- gene=SST-35, id=72 removed by soft count = 50\n",
      "--- number of probes: 141 , kept by blast: 139 , if remove dups: 139\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 139\n",
      "-- checking probes in region: SST-36\n",
      "--- number of probes: 144 , kept by blast: 144 , if remove dups: 144\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 144\n",
      "-- checking probes in region: SST-37\n",
      "--- gene=SST-37, id=139 removed by soft count = 113\n",
      "--- number of probes: 147 , kept by blast: 146 , if remove dups: 146\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 146\n",
      "-- checking probes in region: SST-38\n",
      "--- gene=SST-38, id=12 removed by soft count = 60\n",
      "--- number of probes: 114 , kept by blast: 113 , if remove dups: 113\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 113\n",
      "-- checking probes in region: SST-39\n",
      "--- number of probes: 112 , kept by blast: 112 , if remove dups: 112\n",
      "-- number of probes kept for this region: 112\n",
      "-- checking probes in region: SST-40\n",
      "--- gene=SST-40, id=17 removed by soft count = 114\n",
      "--- gene=SST-40, id=33 removed by soft count = 33\n",
      "--- gene=SST-40, id=73 removed by soft count = 31\n",
      "--- gene=SST-40, id=85 removed by soft count = 36\n",
      "--- gene=SST-40, id=89 removed by soft count = 53\n",
      "--- gene=SST-40, id=93 removed by soft count = 45\n",
      "--- number of probes: 135 , kept by blast: 129 , if remove dups: 129\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 129\n",
      "-- checking probes in region: SST-41\n",
      "--- gene=SST-41, id=127 removed by soft count = 969\n",
      "--- number of probes: 147 , kept by blast: 146 , if remove dups: 146\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 146\n",
      "-- checking probes in region: SST-42\n",
      "--- gene=SST-42, id=20 removed by soft count = 107\n",
      "--- gene=SST-42, id=30 removed by soft count = 89\n",
      "--- gene=SST-42, id=32 removed by soft count = 369\n",
      "--- number of probes: 141 , kept by blast: 138 , if remove dups: 138\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 138\n",
      "-- checking probes in region: SST-43\n",
      "--- gene=SST-43, id=59 removed by soft count = 40\n",
      "--- gene=SST-43, id=82 removed by soft count = 34\n",
      "--- gene=SST-43, id=91 removed by soft count = 37\n",
      "--- number of probes: 139 , kept by blast: 136 , if remove dups: 136\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 136\n",
      "-- checking probes in region: SST-44\n",
      "--- number of probes: 146 , kept by blast: 146 , if remove dups: 146\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 146\n",
      "-- checking probes in region: SST-45\n",
      "--- gene=SST-45, id=17 removed by soft count = 42\n",
      "--- number of probes: 139 , kept by blast: 138 , if remove dups: 138\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 138\n",
      "-- checking probes in region: SST-46\n",
      "--- gene=SST-46, id=119 removed by soft count = 47\n",
      "--- number of probes: 138 , kept by blast: 137 , if remove dups: 137\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 137\n",
      "-- checking probes in region: SST-47\n",
      "--- gene=SST-47, id=0 removed by soft count = 66\n",
      "--- gene=SST-47, id=2 removed by soft count = 37\n",
      "--- gene=SST-47, id=9 removed by soft count = 83\n",
      "--- gene=SST-47, id=62 removed by soft count = 1374\n",
      "--- number of probes: 130 , kept by blast: 126 , if remove dups: 124\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 124\n",
      "-- checking probes in region: SST-48\n",
      "--- gene=SST-48, id=41 removed by soft count = 46\n",
      "--- number of probes: 160 , kept by blast: 159 , if remove dups: 159\n",
      "--- remove duplicated probes\n",
      "--- keep probes from beginning\n",
      "-- number of probes kept for this region: 150\n",
      "-- checking probes in region: SST-49\n",
      "--- gene=SST-49, id=0 removed by soft count = 53\n",
      "--- number of probes: 151 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 150\n",
      "-- checking probes in region: SST-50\n",
      "--- gene=SST-50, id=66 removed by soft count = 52\n",
      "--- gene=SST-50, id=76 removed by soft count = 230\n",
      "--- gene=SST-50, id=82 removed by soft count = 62\n",
      "--- number of probes: 135 , kept by blast: 132 , if remove dups: 132\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 132\n",
      "-- checking probes in region: SST-51\n",
      "--- gene=SST-51, id=90 removed by soft count = 79\n",
      "--- number of probes: 147 , kept by blast: 146 , if remove dups: 146\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 146\n",
      "-- checking probes in region: SST-52\n",
      "--- gene=SST-52, id=16 removed by soft count = 2795\n",
      "--- gene=SST-52, id=17 removed by soft count = 111\n",
      "--- gene=SST-52, id=32 removed by soft count = 31\n",
      "--- gene=SST-52, id=49 removed by soft count = 31\n",
      "--- number of probes: 116 , kept by blast: 112 , if remove dups: 112\n",
      "-- number of probes kept for this region: 112\n",
      "-- checking probes in region: SST-53\n",
      "--- gene=SST-53, id=10 removed by soft count = 80\n",
      "--- gene=SST-53, id=18 removed by soft count = 37\n",
      "--- gene=SST-53, id=23 removed by soft count = 65\n",
      "--- gene=SST-53, id=85 removed by soft count = 42\n",
      "--- number of probes: 124 , kept by blast: 120 , if remove dups: 120\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: SST-54\n",
      "--- gene=SST-54, id=10 removed by soft count = 86\n",
      "--- gene=SST-54, id=21 removed by soft count = 45\n",
      "--- gene=SST-54, id=24 removed by soft count = 51\n",
      "--- gene=SST-54, id=28 removed by soft count = 31\n",
      "--- gene=SST-54, id=32 removed by soft count = 33\n",
      "--- gene=SST-54, id=80 removed by soft count = 85\n",
      "--- gene=SST-54, id=83 removed by soft count = 135\n",
      "--- gene=SST-54, id=84 removed by soft count = 197\n",
      "--- number of probes: 101 , kept by blast: 93 , if remove dups: 93\n",
      "-- number of probes kept for this region: 93\n",
      "-- checking probes in region: SST-55\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=SST-55, id=63 removed by soft count = 210\n",
      "--- gene=SST-55, id=90 removed by soft count = 188\n",
      "--- number of probes: 122 , kept by blast: 120 , if remove dups: 120\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: SST-56\n",
      "--- gene=SST-56, id=16 removed by soft count = 37\n",
      "--- gene=SST-56, id=92 removed by soft count = 34\n",
      "--- gene=SST-56, id=94 removed by soft count = 221\n",
      "--- gene=SST-56, id=95 removed by soft count = 65\n",
      "--- gene=SST-56, id=96 removed by soft count = 99\n",
      "--- gene=SST-56, id=99 removed by soft count = 108\n",
      "--- gene=SST-56, id=100 removed by soft count = 409\n",
      "--- gene=SST-56, id=101 removed by soft count = 54\n",
      "--- gene=SST-56, id=104 removed by soft count = 35\n",
      "--- gene=SST-56, id=108 removed by soft count = 209\n",
      "--- number of probes: 116 , kept by blast: 106 , if remove dups: 106\n",
      "-- number of probes kept for this region: 106\n",
      "-- checking probes in region: SST-57\n",
      "--- gene=SST-57, id=13 removed by soft count = 122\n",
      "--- gene=SST-57, id=16 removed by soft count = 36\n",
      "--- number of probes: 117 , kept by blast: 115 , if remove dups: 115\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 115\n",
      "-- checking probes in region: SST-58\n",
      "--- gene=SST-58, id=40 removed by soft count = 142\n",
      "--- gene=SST-58, id=62 removed by soft count = 33\n",
      "--- gene=SST-58, id=75 removed by soft count = 47\n",
      "--- gene=SST-58, id=76 removed by soft count = 47\n",
      "--- gene=SST-58, id=82 removed by soft count = 45\n",
      "--- gene=SST-58, id=84 removed by soft count = 112\n",
      "--- gene=SST-58, id=85 removed by soft count = 182\n",
      "--- gene=SST-58, id=87 removed by soft count = 643\n",
      "--- gene=SST-58, id=88 removed by soft count = 122\n",
      "--- gene=SST-58, id=89 removed by soft count = 31\n",
      "--- gene=SST-58, id=90 removed by soft count = 519\n",
      "--- gene=SST-58, id=91 removed by soft count = 46\n",
      "--- gene=SST-58, id=96 removed by soft count = 1846\n",
      "--- gene=SST-58, id=100 removed by soft count = 126\n",
      "--- gene=SST-58, id=101 removed by soft count = 51\n",
      "--- number of probes: 108 , kept by blast: 93 , if remove dups: 93\n",
      "-- number of probes kept for this region: 93\n",
      "-- checking probes in region: SST-59\n",
      "--- gene=SST-59, id=81 removed by soft count = 85\n",
      "--- gene=SST-59, id=85 removed by soft count = 61\n",
      "--- number of probes: 134 , kept by blast: 132 , if remove dups: 132\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 132\n",
      "-- checking probes in region: SST-60\n",
      "--- gene=SST-60, id=80 removed by soft count = 42\n",
      "--- gene=SST-60, id=83 removed by soft count = 648\n",
      "--- number of probes: 113 , kept by blast: 111 , if remove dups: 111\n",
      "-- number of probes kept for this region: 111\n",
      "- summarize\n",
      "-- region: Ccnd2-1 , number of probes: 124\n",
      "-- region: Ccnd2-2 , number of probes: 94\n",
      "-- region: Ccnd2-3 , number of probes: 98\n",
      "-- region: Ccnd2-4 , number of probes: 149\n",
      "-- region: Ccnd2-5 , number of probes: 150\n",
      "-- region: Ccnd2-6 , number of probes: 106\n",
      "-- region: Ccnd2-7 , number of probes: 135\n",
      "-- region: Ccnd2-8 , number of probes: 137\n",
      "-- region: Ccnd2-9 , number of probes: 127\n",
      "-- region: Ccnd2-10 , number of probes: 135\n",
      "-- region: Ccnd2-11 , number of probes: 137\n",
      "-- region: Ccnd2-12 , number of probes: 62\n",
      "-- region: Ccnd2-13 , number of probes: 123\n",
      "-- region: Ccnd2-14 , number of probes: 108\n",
      "-- region: Ccnd2-15 , number of probes: 121\n",
      "-- region: Ccnd2-16 , number of probes: 135\n",
      "-- region: Ccnd2-17 , number of probes: 146\n",
      "-- region: Ccnd2-18 , number of probes: 146\n",
      "-- region: Ccnd2-19 , number of probes: 138\n",
      "-- region: Ccnd2-20 , number of probes: 138\n",
      "-- region: Ccnd2-21 , number of probes: 135\n",
      "-- region: Ccnd2-22 , number of probes: 125\n",
      "-- region: Ccnd2-23 , number of probes: 136\n",
      "-- region: Ccnd2-24 , number of probes: 133\n",
      "-- region: Ccnd2-25 , number of probes: 126\n",
      "-- region: Ccnd2-26 , number of probes: 96\n",
      "-- region: Ccnd2-27 , number of probes: 138\n",
      "-- region: Ccnd2-28 , number of probes: 127\n",
      "-- region: Ccnd2-29 , number of probes: 145\n",
      "-- region: Ccnd2-30 , number of probes: 122\n",
      "-- region: Ccnd2-31 , number of probes: 123\n",
      "-- region: Ccnd2-32 , number of probes: 133\n",
      "-- region: Ccnd2-33 , number of probes: 125\n",
      "-- region: Ccnd2-34 , number of probes: 132\n",
      "-- region: Ccnd2-35 , number of probes: 147\n",
      "-- region: Ccnd2-36 , number of probes: 137\n",
      "-- region: Ccnd2-37 , number of probes: 118\n",
      "-- region: Ccnd2-38 , number of probes: 143\n",
      "-- region: Ccnd2-39 , number of probes: 116\n",
      "-- region: Ccnd2-40 , number of probes: 135\n",
      "-- region: Ccnd2-41 , number of probes: 115\n",
      "-- region: Ccnd2-42 , number of probes: 117\n",
      "-- region: Ccnd2-43 , number of probes: 124\n",
      "-- region: Ccnd2-44 , number of probes: 128\n",
      "-- region: Ccnd2-45 , number of probes: 126\n",
      "-- region: Ccnd2-46 , number of probes: 126\n",
      "-- region: Ccnd2-47 , number of probes: 141\n",
      "-- region: Ccnd2-48 , number of probes: 131\n",
      "-- region: Ccnd2-49 , number of probes: 126\n",
      "-- region: Ccnd2-50 , number of probes: 132\n",
      "-- region: Ccnd2-51 , number of probes: 94\n",
      "-- region: Ccnd2-52 , number of probes: 134\n",
      "-- region: Ccnd2-53 , number of probes: 125\n",
      "-- region: Ccnd2-54 , number of probes: 28\n",
      "-- region: Ccnd2-55 , number of probes: 99\n",
      "-- region: Ccnd2-56 , number of probes: 115\n",
      "-- region: Ccnd2-57 , number of probes: 134\n",
      "-- region: Ccnd2-58 , number of probes: 142\n",
      "-- region: Ccnd2-59 , number of probes: 144\n",
      "-- region: Ccnd2-60 , number of probes: 143\n",
      "-- region: Nkx2-1-1 , number of probes: 74\n",
      "-- region: Nkx2-1-2 , number of probes: 57\n",
      "-- region: Nkx2-1-3 , number of probes: 84\n",
      "-- region: Nkx2-1-4 , number of probes: 137\n",
      "-- region: Nkx2-1-5 , number of probes: 120\n",
      "-- region: Nkx2-1-6 , number of probes: 141\n",
      "-- region: Nkx2-1-7 , number of probes: 133\n",
      "-- region: Nkx2-1-8 , number of probes: 110\n",
      "-- region: Nkx2-1-9 , number of probes: 122\n",
      "-- region: Nkx2-1-10 , number of probes: 129\n",
      "-- region: Nkx2-1-11 , number of probes: 126\n",
      "-- region: Nkx2-1-12 , number of probes: 115\n",
      "-- region: Nkx2-1-13 , number of probes: 133\n",
      "-- region: Nkx2-1-14 , number of probes: 127\n",
      "-- region: Nkx2-1-15 , number of probes: 126\n",
      "-- region: Nkx2-1-16 , number of probes: 125\n",
      "-- region: Nkx2-1-17 , number of probes: 137\n",
      "-- region: Nkx2-1-18 , number of probes: 145\n",
      "-- region: Nkx2-1-19 , number of probes: 150\n",
      "-- region: Nkx2-1-20 , number of probes: 121\n",
      "-- region: Nkx2-1-21 , number of probes: 95\n",
      "-- region: Nkx2-1-22 , number of probes: 46\n",
      "-- region: Nkx2-1-23 , number of probes: 81\n",
      "-- region: Nkx2-1-24 , number of probes: 73\n",
      "-- region: Nkx2-1-25 , number of probes: 106\n",
      "-- region: Nkx2-1-26 , number of probes: 116\n",
      "-- region: Nkx2-1-27 , number of probes: 143\n",
      "-- region: Nkx2-1-28 , number of probes: 111\n",
      "-- region: Nkx2-1-29 , number of probes: 139\n",
      "-- region: Nkx2-1-30 , number of probes: 119\n",
      "-- region: Nkx2-1-31 , number of probes: 80\n",
      "-- region: Nkx2-1-32 , number of probes: 97\n",
      "-- region: Nkx2-1-33 , number of probes: 131\n",
      "-- region: Nkx2-1-34 , number of probes: 148\n",
      "-- region: Nkx2-1-35 , number of probes: 150\n",
      "-- region: Nkx2-1-36 , number of probes: 133\n",
      "-- region: Nkx2-1-37 , number of probes: 132\n",
      "-- region: Nkx2-1-38 , number of probes: 129\n",
      "-- region: Nkx2-1-39 , number of probes: 124\n",
      "-- region: Nkx2-1-40 , number of probes: 47\n",
      "-- region: Nkx2-1-41 , number of probes: 101\n",
      "-- region: Nkx2-1-42 , number of probes: 113\n",
      "-- region: Nkx2-1-43 , number of probes: 121\n",
      "-- region: Nkx2-1-44 , number of probes: 111\n",
      "-- region: Nkx2-1-45 , number of probes: 132\n",
      "-- region: Nkx2-1-46 , number of probes: 138\n",
      "-- region: Nkx2-1-47 , number of probes: 130\n",
      "-- region: Nkx2-1-48 , number of probes: 130\n",
      "-- region: Nkx2-1-49 , number of probes: 98\n",
      "-- region: Nkx2-1-50 , number of probes: 84\n",
      "-- region: Nkx2-1-51 , number of probes: 122\n",
      "-- region: Nkx2-1-52 , number of probes: 135\n",
      "-- region: Nkx2-1-53 , number of probes: 122\n",
      "-- region: Nkx2-1-54 , number of probes: 125\n",
      "-- region: Nkx2-1-55 , number of probes: 137\n",
      "-- region: Nkx2-1-56 , number of probes: 135\n",
      "-- region: Nkx2-1-57 , number of probes: 99\n",
      "-- region: Nkx2-1-58 , number of probes: 121\n",
      "-- region: Nkx2-1-59 , number of probes: 125\n",
      "-- region: Nkx2-1-60 , number of probes: 78\n",
      "-- region: Slc17a7-1 , number of probes: 91\n",
      "-- region: Slc17a7-2 , number of probes: 91\n",
      "-- region: Slc17a7-3 , number of probes: 133\n",
      "-- region: Slc17a7-4 , number of probes: 102\n",
      "-- region: Slc17a7-5 , number of probes: 115\n",
      "-- region: Slc17a7-6 , number of probes: 43\n",
      "-- region: Slc17a7-7 , number of probes: 60\n",
      "-- region: Slc17a7-8 , number of probes: 140\n",
      "-- region: Slc17a7-9 , number of probes: 145\n",
      "-- region: Slc17a7-10 , number of probes: 110\n",
      "-- region: Slc17a7-11 , number of probes: 135\n",
      "-- region: Slc17a7-12 , number of probes: 147\n",
      "-- region: Slc17a7-13 , number of probes: 150\n",
      "-- region: Slc17a7-14 , number of probes: 119\n",
      "-- region: Slc17a7-15 , number of probes: 140\n",
      "-- region: Slc17a7-16 , number of probes: 142\n",
      "-- region: Slc17a7-17 , number of probes: 148\n",
      "-- region: Slc17a7-18 , number of probes: 125\n",
      "-- region: Slc17a7-19 , number of probes: 112\n",
      "-- region: Slc17a7-20 , number of probes: 87\n",
      "-- region: Slc17a7-21 , number of probes: 86\n",
      "-- region: Slc17a7-22 , number of probes: 124\n",
      "-- region: Slc17a7-23 , number of probes: 118\n",
      "-- region: Slc17a7-24 , number of probes: 127\n",
      "-- region: Slc17a7-25 , number of probes: 121\n",
      "-- region: Slc17a7-26 , number of probes: 127\n",
      "-- region: Slc17a7-27 , number of probes: 144\n",
      "-- region: Slc17a7-28 , number of probes: 150\n",
      "-- region: Slc17a7-29 , number of probes: 123\n",
      "-- region: Slc17a7-30 , number of probes: 133\n",
      "-- region: Slc17a7-31 , number of probes: 139\n",
      "-- region: Slc17a7-32 , number of probes: 140\n",
      "-- region: Slc17a7-33 , number of probes: 107\n",
      "-- region: Slc17a7-34 , number of probes: 123\n",
      "-- region: Slc17a7-35 , number of probes: 96\n",
      "-- region: Slc17a7-36 , number of probes: 102\n",
      "-- region: Slc17a7-37 , number of probes: 93\n",
      "-- region: Slc17a7-38 , number of probes: 89\n",
      "-- region: Slc17a7-39 , number of probes: 145\n",
      "-- region: Slc17a7-40 , number of probes: 118\n",
      "-- region: Slc17a7-41 , number of probes: 134\n",
      "-- region: Slc17a7-42 , number of probes: 129\n",
      "-- region: Slc17a7-43 , number of probes: 131\n",
      "-- region: Slc17a7-44 , number of probes: 140\n",
      "-- region: Slc17a7-45 , number of probes: 145\n",
      "-- region: Slc17a7-46 , number of probes: 130\n",
      "-- region: Slc17a7-47 , number of probes: 104\n",
      "-- region: Slc17a7-48 , number of probes: 127\n",
      "-- region: Slc17a7-49 , number of probes: 128\n",
      "-- region: Slc17a7-50 , number of probes: 134\n",
      "-- region: Slc17a7-51 , number of probes: 55\n",
      "-- region: Slc17a7-52 , number of probes: 84\n",
      "-- region: Slc17a7-53 , number of probes: 123\n",
      "-- region: Slc17a7-54 , number of probes: 118\n",
      "-- region: Slc17a7-55 , number of probes: 129\n",
      "-- region: Slc17a7-56 , number of probes: 102\n",
      "-- region: Slc17a7-57 , number of probes: 127\n",
      "-- region: Slc17a7-58 , number of probes: 121\n",
      "-- region: Slc17a7-59 , number of probes: 98\n",
      "-- region: Slc17a7-60 , number of probes: 141\n",
      "-- region: Slc32a1-1 , number of probes: 125\n",
      "-- region: Slc32a1-2 , number of probes: 135\n",
      "-- region: Slc32a1-3 , number of probes: 127\n",
      "-- region: Slc32a1-4 , number of probes: 121\n",
      "-- region: Slc32a1-5 , number of probes: 134\n",
      "-- region: Slc32a1-6 , number of probes: 128\n",
      "-- region: Slc32a1-7 , number of probes: 122\n",
      "-- region: Slc32a1-8 , number of probes: 104\n",
      "-- region: Slc32a1-9 , number of probes: 123\n",
      "-- region: Slc32a1-10 , number of probes: 125\n",
      "-- region: Slc32a1-11 , number of probes: 144\n",
      "-- region: Slc32a1-12 , number of probes: 127\n",
      "-- region: Slc32a1-13 , number of probes: 143\n",
      "-- region: Slc32a1-14 , number of probes: 115\n",
      "-- region: Slc32a1-15 , number of probes: 125\n",
      "-- region: Slc32a1-16 , number of probes: 106\n",
      "-- region: Slc32a1-17 , number of probes: 140\n",
      "-- region: Slc32a1-18 , number of probes: 126\n",
      "-- region: Slc32a1-19 , number of probes: 133\n",
      "-- region: Slc32a1-20 , number of probes: 132\n",
      "-- region: Slc32a1-21 , number of probes: 143\n",
      "-- region: SST-1 , number of probes: 103\n",
      "-- region: SST-2 , number of probes: 48\n",
      "-- region: SST-3 , number of probes: 51\n",
      "-- region: SST-4 , number of probes: 116\n",
      "-- region: SST-5 , number of probes: 108\n",
      "-- region: SST-6 , number of probes: 149\n",
      "-- region: SST-7 , number of probes: 139\n",
      "-- region: SST-8 , number of probes: 128\n",
      "-- region: SST-9 , number of probes: 145\n",
      "-- region: SST-10 , number of probes: 140\n",
      "-- region: SST-11 , number of probes: 143\n",
      "-- region: SST-12 , number of probes: 139\n",
      "-- region: SST-13 , number of probes: 78\n",
      "-- region: SST-14 , number of probes: 89\n",
      "-- region: SST-15 , number of probes: 79\n",
      "-- region: SST-16 , number of probes: 131\n",
      "-- region: SST-17 , number of probes: 147\n",
      "-- region: SST-18 , number of probes: 88\n",
      "-- region: SST-19 , number of probes: 100\n",
      "-- region: SST-20 , number of probes: 101\n",
      "-- region: SST-21 , number of probes: 100\n",
      "-- region: SST-22 , number of probes: 52\n",
      "-- region: SST-23 , number of probes: 107\n",
      "-- region: SST-24 , number of probes: 147\n",
      "-- region: SST-25 , number of probes: 140\n",
      "-- region: SST-26 , number of probes: 150\n",
      "-- region: SST-27 , number of probes: 141\n",
      "-- region: SST-28 , number of probes: 134\n",
      "-- region: SST-29 , number of probes: 123\n",
      "-- region: SST-30 , number of probes: 140\n",
      "-- region: SST-31 , number of probes: 96\n",
      "-- region: SST-32 , number of probes: 136\n",
      "-- region: SST-33 , number of probes: 150\n",
      "-- region: SST-34 , number of probes: 148\n",
      "-- region: SST-35 , number of probes: 139\n",
      "-- region: SST-36 , number of probes: 144\n",
      "-- region: SST-37 , number of probes: 146\n",
      "-- region: SST-38 , number of probes: 113\n",
      "-- region: SST-39 , number of probes: 112\n",
      "-- region: SST-40 , number of probes: 129\n",
      "-- region: SST-41 , number of probes: 146\n",
      "-- region: SST-42 , number of probes: 138\n",
      "-- region: SST-43 , number of probes: 136\n",
      "-- region: SST-44 , number of probes: 146\n",
      "-- region: SST-45 , number of probes: 138\n",
      "-- region: SST-46 , number of probes: 137\n",
      "-- region: SST-47 , number of probes: 124\n",
      "-- region: SST-48 , number of probes: 150\n",
      "-- region: SST-49 , number of probes: 150\n",
      "-- region: SST-50 , number of probes: 132\n",
      "-- region: SST-51 , number of probes: 146\n",
      "-- region: SST-52 , number of probes: 112\n",
      "-- region: SST-53 , number of probes: 120\n",
      "-- region: SST-54 , number of probes: 93\n",
      "-- region: SST-55 , number of probes: 120\n",
      "-- region: SST-56 , number of probes: 106\n",
      "-- region: SST-57 , number of probes: 115\n",
      "-- region: SST-58 , number of probes: 93\n",
      "-- region: SST-59 , number of probes: 132\n",
      "-- region: SST-60 , number of probes: 111\n",
      "- Number of probes kept: 31626\n",
      "- Saving to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\.\\blast_full_probes.fasta\n"
     ]
    }
   ],
   "source": [
    "# screen\n",
    "num_probes_per_region = 150\n",
    "kept_pbs, blast_keep_dic, hard_count_list, soft_count_list = check.Screening_Probes_by_Blast(library_folder, \n",
    "                                                                                             num_probes_per_region,\n",
    "                                                                                             )"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 268,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "6:126980000-126990000_gene_Ccnd2-1_pb_1_pos_93_strand_-_readouts_[Stv_3_u,NDB_16_u]\n",
      "CCGCAATGGCTGACAACCGCCCATGATCGTCCGATCTGGTTCAGCAGTTTGCTTTTGGACCATTTGAAATAGCAAAGTAATGCGCATCGCGGCTAATAGAGCGGAAACCATGCGGCAATC\n"
     ]
    }
   ],
   "source": [
    "_pid = 1\n",
    "print(kept_pbs[_pid].id)\n",
    "print(kept_pbs[_pid].seq)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# 5. Generate color_usage"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 271,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "load from: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\blast_full_probes.fasta\n"
     ]
    }
   ],
   "source": [
    "probe_filename = os.path.join(library_folder, 'blast_full_probes.fasta')\n",
    "if os.path.isfile(probe_filename):\n",
    "    print(f\"load from: {probe_filename}\")\n",
    "    pb_records = []\n",
    "    with open(probe_filename, 'r') as handle:\n",
    "        for record in SeqIO.parse(handle, \"fasta\"):\n",
    "            pb_records.append(record)\n",
    "else:\n",
    "    raise IOError(f\"input probe file: {probe_filename} doesn't exist.\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 342,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "create folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_brain\\MOP_marker_genes_10kb\\Summary_table\n"
     ]
    }
   ],
   "source": [
    "table_folder = os.path.join(library_folder, 'Summary_table')\n",
    "if not os.path.exists(table_folder):\n",
    "    print(f\"create folder: {table_folder}\")\n",
    "    os.makedirs(table_folder)\n",
    "else:\n",
    "    print(f\"{table_folder}\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## load readouts as reference"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 402,
   "metadata": {},
   "outputs": [],
   "source": [
    "readout_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Readouts'\n",
    "ref_files = [_fl for _fl in os.listdir(readout_folder) if 'designed_readouts' in _fl]\n",
    "\n",
    "ref_readout_name_dict = {}\n",
    "for _fl in ref_files:\n",
    "    _channel = int(_fl.split('designed_readouts_')[1].split('.fasta')[0])\n",
    "    _ref_readout_names = []\n",
    "    with open(os.path.join(readout_folder, _fl), 'r') as _rd_handle:\n",
    "        for _readout in SeqIO.parse(_rd_handle, \"fasta\"):\n",
    "            _ref_readout_names.append(_readout.id)\n",
    "    ref_readout_name_dict[_channel] = _ref_readout_names"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 273,
   "metadata": {},
   "outputs": [],
   "source": [
    "# split by region\n",
    "pb_dict = library_tools.quality_check.split_probe_by_gene(pb_records)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 287,
   "metadata": {},
   "outputs": [],
   "source": [
    "readout_names = {}\n",
    "for _i, _r in enumerate(pb_records):\n",
    "    _rname = _r.id.split('gene_')[1].split('_')[0]\n",
    "    _rd_names = _r.id.split('[')[1].split(']')[0].split(',')\n",
    "    # initialize if necessary\n",
    "    if _rname not in readout_names:\n",
    "        readout_names[_rname] = []\n",
    "    for _rd in _rd_names:\n",
    "        _clean_rd = _rd.split(_rd.split('_')[-1])[0][:-1]\n",
    "        if _clean_rd not in readout_names[_rname]:\n",
    "            readout_names[_rname].append(_clean_rd)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 304,
   "metadata": {},
   "outputs": [],
   "source": [
    "# generate a readout_2_region dict\n",
    "readout_2_region = {}\n",
    "for _rname, _rds in readout_names.items():\n",
    "    for _rd in _rds:\n",
    "        # initialize\n",
    "        if _rd not in readout_2_region:\n",
    "            readout_2_region[_rd] = []\n",
    "        readout_2_region[_rd].append(_rname)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 403,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{1: 'Stv_3', 2: 'Stv_32', 3: 'Stv_4', 4: 'Stv_33', 5: 'Stv_5', 6: 'Stv_35', 7: 'Stv_6', 8: 'Stv_36', 9: 'Stv_7', 10: 'Stv_37', 11: 'Stv_8', 12: 'Stv_39', 13: 'Stv_9', 14: 'Stv_40', 15: 'Stv_10', 16: 'Stv_42', 17: 'Stv_11', 18: 'Stv_44', 19: 'Stv_12', 20: 'Stv_45', 21: 'Stv_13', 22: 'Stv_46', 23: 'Stv_14', 24: 'Stv_48', 25: 'Stv_16', 26: 'Stv_50', 27: 'Stv_19', 28: 'Stv_53', 29: 'Stv_20', 30: 'Stv_54', 31: 'Stv_21', 32: 'Stv_59', 33: 'Stv_22', 34: 'Stv_60', 35: 'Stv_23', 36: 'Stv_61', 37: 'Stv_25', 38: 'Stv_63', 39: 'Stv_26', 40: 'Stv_64', 41: 'Stv_27', 42: 'Stv_65', 43: 'Stv_28', 44: 'Stv_86', 45: 'Stv_29', 46: 'Stv_87', 47: 'Stv_30', 48: 'Stv_88', 49: 'Stv_31', 50: 'Stv_90', 51: 'NDB_1', 52: 'NDB_2', 53: 'NDB_4', 54: 'NDB_5', 55: 'NDB_7', 56: 'NDB_8', 57: 'NDB_10', 58: 'NDB_11', 59: 'NDB_13', 60: 'NDB_14'}\n"
     ]
    }
   ],
   "source": [
    "# step 1: find parallel tracing\n",
    "parallel_readouts = {}\n",
    "for _rd, _rnames in readout_2_region.items():\n",
    "    _reg_ids = [int(_rn.split('-')[-1]) for _rn in _rnames]\n",
    "    _related_gnames = [_rn.split('-'+_rn.split('-')[-1])[0] for _rn in _rnames]\n",
    "    if len(np.unique(_related_gnames)) > 1 and len(np.unique(_reg_ids)) == 1:\n",
    "        parallel_readouts[np.unique(_reg_ids)[0]] = _rd\n",
    "print(parallel_readouts)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 404,
   "metadata": {},
   "outputs": [],
   "source": [
    "# sort regions with readout types\n",
    "readout_by_channel = {_c:{} for _c in ref_readout_name_dict}\n",
    "for _rid, _rname in parallel_readouts.items():\n",
    "    for _c in readout_by_channel:\n",
    "        if _rname in ref_readout_name_dict[_c] and _rid not in readout_by_channel[_c]:\n",
    "            readout_by_channel[_c][_rid] = _rname\n",
    "readout_by_channel = {_k: _v for _k, _v in readout_by_channel.items() if len(_v) > 0}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 405,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{647: {2: 'Stv_32',\n",
       "  4: 'Stv_33',\n",
       "  6: 'Stv_35',\n",
       "  8: 'Stv_36',\n",
       "  10: 'Stv_37',\n",
       "  12: 'Stv_39',\n",
       "  14: 'Stv_40',\n",
       "  16: 'Stv_42',\n",
       "  18: 'Stv_44',\n",
       "  20: 'Stv_45',\n",
       "  22: 'Stv_46',\n",
       "  24: 'Stv_48',\n",
       "  26: 'Stv_50',\n",
       "  28: 'Stv_53',\n",
       "  30: 'Stv_54',\n",
       "  32: 'Stv_59',\n",
       "  34: 'Stv_60',\n",
       "  36: 'Stv_61',\n",
       "  38: 'Stv_63',\n",
       "  40: 'Stv_64',\n",
       "  42: 'Stv_65',\n",
       "  44: 'Stv_86',\n",
       "  46: 'Stv_87',\n",
       "  48: 'Stv_88',\n",
       "  50: 'Stv_90',\n",
       "  52: 'NDB_2',\n",
       "  54: 'NDB_5',\n",
       "  56: 'NDB_8',\n",
       "  58: 'NDB_11',\n",
       "  60: 'NDB_14'},\n",
       " 750: {1: 'Stv_3',\n",
       "  3: 'Stv_4',\n",
       "  5: 'Stv_5',\n",
       "  7: 'Stv_6',\n",
       "  9: 'Stv_7',\n",
       "  11: 'Stv_8',\n",
       "  13: 'Stv_9',\n",
       "  15: 'Stv_10',\n",
       "  17: 'Stv_11',\n",
       "  19: 'Stv_12',\n",
       "  21: 'Stv_13',\n",
       "  23: 'Stv_14',\n",
       "  25: 'Stv_16',\n",
       "  27: 'Stv_19',\n",
       "  29: 'Stv_20',\n",
       "  31: 'Stv_21',\n",
       "  33: 'Stv_22',\n",
       "  35: 'Stv_23',\n",
       "  37: 'Stv_25',\n",
       "  39: 'Stv_26',\n",
       "  41: 'Stv_27',\n",
       "  43: 'Stv_28',\n",
       "  45: 'Stv_29',\n",
       "  47: 'Stv_30',\n",
       "  49: 'Stv_31',\n",
       "  51: 'NDB_1',\n",
       "  53: 'NDB_4',\n",
       "  55: 'NDB_7',\n",
       "  57: 'NDB_10',\n",
       "  59: 'NDB_13'}}"
      ]
     },
     "execution_count": 405,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "readout_by_channel"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 359,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['Hyb', '750', '647', '488', '405']\n",
      "['H0R0', 'forward_chrom', 'reverse_chrom', 'beads', 'DAPI']\n",
      "['H1R1', 'm1', 'm2', 'beads']\n",
      "['H2R2', 'm3', 'm4', 'beads']\n",
      "['H3R3', 'm5', 'm6', 'beads']\n",
      "['H4R4', 'm7', 'm8', 'beads']\n",
      "['H5R5', 'm9', 'm10', 'beads']\n",
      "['H6R6', 'm11', 'm12', 'beads']\n",
      "['H7R7', 'm13', 'm14', 'beads']\n",
      "['H8R8', 'm15', 'm16', 'beads']\n",
      "['H9R9', 'm17', 'm18', 'beads']\n",
      "['H10R10', 'm19', 'm20', 'beads']\n",
      "['H11R11', 'm21', 'm22', 'beads']\n",
      "['H12R12', 'm23', 'm24', 'beads']\n",
      "['H13R13', 'm25', 'm26', 'beads']\n",
      "['H14R14', 'm27', 'm28', 'beads']\n",
      "['H15R15', 'm29', 'm30', 'beads']\n",
      "['H16R16', 'm31', 'm32', 'beads']\n",
      "['H17R17', 'm33', 'm34', 'beads']\n",
      "['H18R18', 'm35', 'm36', 'beads']\n",
      "['H19R19', 'm37', 'm38', 'beads']\n",
      "['H20R20', 'm39', 'm40', 'beads']\n",
      "['H21R21', 'm41', 'm42', 'beads']\n",
      "['H22R22', 'm43', 'm44', 'beads']\n",
      "['H23R23', 'm45', 'm46', 'beads']\n",
      "['H24R24', 'm47', 'm48', 'beads']\n",
      "['H25R25', 'm49', 'm50', 'beads']\n",
      "['H26R26', 'm51', 'm52', 'beads']\n",
      "['H27R27', 'm53', 'm54', 'beads']\n",
      "['H28R28', 'm55', 'm56', 'beads']\n",
      "['H29R29', 'm57', 'm58', 'beads']\n",
      "['H30R30', 'm59', 'm60', 'beads']\n"
     ]
    }
   ],
   "source": [
    "# save color usage for parallel\n",
    "import csv\n",
    "\n",
    "drift_channel = '488'\n",
    "dapi_channel = '405'\n",
    "chrom_labels = {'750': 'forward',\n",
    "                '647': 'reverse',}\n",
    "dict_sizes = [len(_v) for _k,_v in readout_by_channel.items()]\n",
    "\n",
    "with open(os.path.join(table_folder, 'Color_Usage_parallel.csv'), 'w', newline='') as csvfile:\n",
    "    csvwriter = csv.writer(csvfile, delimiter=',', \n",
    "                           quotechar='|', quoting=csv.QUOTE_MINIMAL)\n",
    "    \n",
    "    # write header\n",
    "    _header = ['Hyb']\n",
    "    for _ch in sorted(readout_by_channel)[::-1]:\n",
    "        _header .append(str(_ch))\n",
    "    _header.append(drift_channel)\n",
    "    _header.append(dapi_channel)\n",
    "    print(_header)\n",
    "    csvwriter.writerow(_header)\n",
    "    \n",
    "    # write reference frame\n",
    "    _ref_row = ['H0R0']\n",
    "    for _ch in sorted(readout_by_channel)[::-1]:\n",
    "        if str(_ch) in chrom_labels:\n",
    "            _ref_row.append(chrom_labels[str(_ch)]+'_chrom')\n",
    "        else:\n",
    "            _ref_row.append(\"\")\n",
    "    _ref_row.append('beads')\n",
    "    _ref_row.append('DAPI')\n",
    "    print(_ref_row)\n",
    "    csvwriter.writerow(_ref_row)\n",
    "    \n",
    "    \n",
    "    for _i in range(max(dict_sizes)):\n",
    "        _row = [f\"H{int(_i)+1}R{int(_i)+1}\",] \n",
    "        for _ch in sorted(readout_by_channel)[::-1]:\n",
    "            _dict = readout_by_channel[_ch]\n",
    "            if _i >= len(_dict):\n",
    "                _row += ['']\n",
    "            else:\n",
    "                _regs = sorted(_dict) \n",
    "                _row += [f\"m{_regs[_i]}\"]\n",
    "        _row.append(\"beads\")\n",
    "        print(_row)\n",
    "        csvwriter.writerow(_row)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 364,
   "metadata": {},
   "outputs": [],
   "source": [
    "# save adaptor sequences for parallel\n",
    "with open(os.path.join(table_folder, 'adaptor_sequences_parallel.csv'), 'w', newline='') as csvfile:\n",
    "    csvwriter = csv.writer(csvfile, delimiter=',', \n",
    "                           quotechar='|', quoting=csv.QUOTE_MINIMAL)\n",
    "    \n",
    "    _header = ['group', 'hyb', ]\n",
    "    for _ch in sorted(readout_by_channel)[::-1]:\n",
    "        _header += [f\"{_ch}_region\", f\"{_ch}_readout\"]\n",
    "        \n",
    "    csvwriter.writerow(_header)\n",
    "    \n",
    "    for _i in range(max(dict_sizes)):\n",
    "        _row = [int((_i)/32)+1, _i%32+1,] \n",
    "        for _ch, _dict in sorted(readout_by_channel.items())[::-1]:\n",
    "            if _i >= len(_dict):\n",
    "                _row += ['', '']\n",
    "            else:\n",
    "                _regs = sorted(_dict) \n",
    "                _row += [_regs[_i], _dict[_regs[_i]]]      \n",
    "        csvwriter.writerow(_row)\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## unique"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 378,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'Ccnd2-1': 'NDB_16', 'Ccnd2-2': 'NDB_17', 'Ccnd2-3': 'NDB_19', 'Ccnd2-4': 'NDB_20', 'Ccnd2-5': 'NDB_22', 'Ccnd2-6': 'NDB_23', 'Ccnd2-7': 'NDB_25', 'Ccnd2-8': 'NDB_26', 'Ccnd2-9': 'NDB_28', 'Ccnd2-10': 'NDB_29', 'Ccnd2-11': 'NDB_31', 'Ccnd2-12': 'NDB_32', 'Ccnd2-13': 'NDB_34', 'Ccnd2-14': 'NDB_35', 'Ccnd2-15': 'NDB_37', 'Ccnd2-16': 'NDB_38', 'Ccnd2-17': 'NDB_40', 'Ccnd2-18': 'NDB_41', 'Ccnd2-19': 'NDB_43', 'Ccnd2-20': 'NDB_44', 'Ccnd2-21': 'NDB_46', 'Ccnd2-22': 'NDB_47', 'Ccnd2-23': 'NDB_49', 'Ccnd2-24': 'NDB_50', 'Ccnd2-25': 'NDB_52', 'Ccnd2-26': 'NDB_53', 'Ccnd2-27': 'NDB_55', 'Ccnd2-28': 'NDB_56', 'Ccnd2-29': 'NDB_58', 'Ccnd2-30': 'NDB_59', 'Ccnd2-31': 'NDB_61', 'Ccnd2-32': 'NDB_62', 'Ccnd2-33': 'NDB_64', 'Ccnd2-34': 'NDB_65', 'Ccnd2-35': 'NDB_67', 'Ccnd2-36': 'NDB_68', 'Ccnd2-37': 'NDB_70', 'Ccnd2-38': 'NDB_71', 'Ccnd2-39': 'NDB_73', 'Ccnd2-40': 'NDB_74', 'Ccnd2-41': 'NDB_76', 'Ccnd2-42': 'NDB_77', 'Ccnd2-43': 'NDB_79', 'Ccnd2-44': 'NDB_80', 'Ccnd2-45': 'NDB_82', 'Ccnd2-46': 'NDB_83', 'Ccnd2-47': 'NDB_85', 'Ccnd2-48': 'NDB_86', 'Ccnd2-49': 'NDB_88', 'Ccnd2-50': 'NDB_89', 'Ccnd2-51': 'NDB_91', 'Ccnd2-52': 'NDB_92', 'Ccnd2-53': 'NDB_94', 'Ccnd2-54': 'NDB_95', 'Ccnd2-55': 'NDB_97', 'Ccnd2-56': 'NDB_98', 'Ccnd2-57': 'NDB_100', 'Ccnd2-58': 'NDB_101', 'Ccnd2-59': 'NDB_103', 'Ccnd2-60': 'NDB_104'}\n"
     ]
    }
   ],
   "source": [
    "# step 1: find unique tracing\n",
    "unique_readouts_by_gene = {}\n",
    "for _rd, _rnames in readout_2_region.items():\n",
    "    #_reg_ids = [int(_rn.split('-')[-1]) for _rn in _rnames]\n",
    "    _related_gnames = [_rn.split('-'+_rn.split('-')[-1])[0] for _rn in _rnames]\n",
    "    if len(np.unique(_related_gnames)) == 1 and len(np.unique(_reg_ids)) == 1:\n",
    "        _gname = np.unique(_related_gnames)[0]\n",
    "        if _gname not in unique_readouts_by_gene:\n",
    "            unique_readouts_by_gene[_gname] = {}\n",
    "        unique_readouts_by_gene[_gname][_rnames[0]] = _rd\n",
    "print(unique_readouts_by_gene['Ccnd2'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 406,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['Hyb', '750', '647', '488', '405']\n",
      "['H0R0', 'forward_chrom', 'reverse_chrom', 'beads', 'DAPI']\n",
      "['H1R1', 'u1', 'u2', 'beads']\n",
      "['H2R2', 'u3', 'u4', 'beads']\n",
      "['H3R3', 'u5', 'u6', 'beads']\n",
      "['H4R4', 'u7', 'u8', 'beads']\n",
      "['H5R5', 'u9', 'u10', 'beads']\n",
      "['H6R6', 'u11', 'u12', 'beads']\n",
      "['H7R7', 'u13', 'u14', 'beads']\n",
      "['H8R8', 'u15', 'u16', 'beads']\n",
      "['H9R9', 'u17', 'u18', 'beads']\n",
      "['H10R10', 'u19', 'u20', 'beads']\n",
      "['H11R11', 'u21', 'u22', 'beads']\n",
      "['H12R12', 'u23', 'u24', 'beads']\n",
      "['H13R13', 'u25', 'u26', 'beads']\n",
      "['H14R14', 'u27', 'u28', 'beads']\n",
      "['H15R15', 'u29', 'u30', 'beads']\n",
      "['H16R16', 'u31', 'u32', 'beads']\n",
      "['H17R17', 'u33', 'u34', 'beads']\n",
      "['H18R18', 'u35', 'u36', 'beads']\n",
      "['H19R19', 'u37', 'u38', 'beads']\n",
      "['H20R20', 'u39', 'u40', 'beads']\n",
      "['H21R21', 'u41', 'u42', 'beads']\n",
      "['H22R22', 'u43', 'u44', 'beads']\n",
      "['H23R23', 'u45', 'u46', 'beads']\n",
      "['H24R24', 'u47', 'u48', 'beads']\n",
      "['H25R25', 'u49', 'u50', 'beads']\n",
      "['H26R26', 'u51', 'u52', 'beads']\n",
      "['H27R27', 'u53', 'u54', 'beads']\n",
      "['H28R28', 'u55', 'u56', 'beads']\n",
      "['H29R29', 'u57', 'u58', 'beads']\n",
      "['H30R30', 'u59', 'u60', 'beads']\n",
      "['Hyb', '750', '488', '405']\n",
      "['H0R0', 'forward_chrom', 'beads', 'DAPI']\n",
      "['H1R1', 'u61', 'beads']\n",
      "['Hyb', '750', '647', '488', '405']\n",
      "['H0R0', 'forward_chrom', 'reverse_chrom', 'beads', 'DAPI']\n",
      "['H1R1', 'u63', 'u62', 'beads']\n",
      "['H2R2', 'u65', 'u64', 'beads']\n",
      "['H3R3', 'u67', 'u66', 'beads']\n",
      "['H4R4', 'u69', 'u68', 'beads']\n",
      "['H5R5', 'u71', 'u70', 'beads']\n",
      "['H6R6', 'u73', 'u72', 'beads']\n",
      "['H7R7', 'u75', 'u74', 'beads']\n",
      "['H8R8', 'u77', 'u76', 'beads']\n",
      "['H9R9', 'u79', 'u78', 'beads']\n",
      "['H10R10', 'u81', 'u80', 'beads']\n",
      "['H11R11', 'u83', 'u82', 'beads']\n",
      "['H12R12', 'u85', 'u84', 'beads']\n",
      "['H13R13', 'u87', 'u86', 'beads']\n",
      "['H14R14', 'u89', 'u88', 'beads']\n",
      "['H15R15', 'u91', 'u90', 'beads']\n",
      "['H16R16', 'u93', 'u92', 'beads']\n",
      "['H17R17', 'u95', 'u94', 'beads']\n",
      "['H18R18', 'u97', 'u96', 'beads']\n",
      "['H19R19', 'u99', 'u98', 'beads']\n",
      "['H20R20', 'u101', 'u100', 'beads']\n",
      "['H21R21', 'u103', 'u102', 'beads']\n",
      "['H22R22', 'u105', 'u104', 'beads']\n",
      "['H23R23', 'u107', 'u106', 'beads']\n",
      "['H24R24', 'u109', 'u108', 'beads']\n",
      "['H25R25', 'u111', 'u110', 'beads']\n",
      "['H26R26', 'u113', 'u112', 'beads']\n",
      "['H27R27', 'u115', 'u114', 'beads']\n",
      "['H28R28', 'u117', 'u116', 'beads']\n",
      "['H29R29', 'u119', 'u118', 'beads']\n",
      "['H30R30', '', 'u120', 'beads']\n",
      "['Hyb', '750', '647', '488', '405']\n",
      "['H0R0', 'forward_chrom', 'reverse_chrom', 'beads', 'DAPI']\n",
      "['H1R1', 'u121', 'u122', 'beads']\n",
      "['H2R2', 'u123', 'u124', 'beads']\n",
      "['H3R3', 'u125', 'u126', 'beads']\n",
      "['H4R4', 'u127', 'u128', 'beads']\n",
      "['H5R5', 'u129', 'u130', 'beads']\n",
      "['H6R6', 'u131', 'u132', 'beads']\n",
      "['H7R7', 'u133', 'u134', 'beads']\n",
      "['H8R8', 'u135', 'u136', 'beads']\n",
      "['H9R9', 'u137', 'u138', 'beads']\n",
      "['H10R10', 'u139', 'u140', 'beads']\n",
      "['H11R11', 'u141', 'u142', 'beads']\n",
      "['H12R12', 'u143', 'u144', 'beads']\n",
      "['H13R13', 'u145', 'u146', 'beads']\n",
      "['H14R14', 'u147', 'u148', 'beads']\n",
      "['H15R15', 'u149', 'u150', 'beads']\n",
      "['H16R16', 'u151', 'u152', 'beads']\n",
      "['H17R17', 'u153', 'u154', 'beads']\n",
      "['H18R18', 'u155', 'u156', 'beads']\n",
      "['H19R19', 'u157', 'u158', 'beads']\n",
      "['H20R20', 'u159', 'u160', 'beads']\n",
      "['H21R21', 'u161', 'u162', 'beads']\n",
      "['H22R22', 'u163', 'u164', 'beads']\n",
      "['H23R23', 'u165', 'u166', 'beads']\n",
      "['H24R24', 'u167', 'u168', 'beads']\n",
      "['H25R25', 'u169', 'u170', 'beads']\n",
      "['H26R26', 'u171', 'u172', 'beads']\n",
      "['H27R27', 'u173', 'u174', 'beads']\n",
      "['H28R28', 'u175', 'u176', 'beads']\n",
      "['H29R29', 'u177', 'u178', 'beads']\n",
      "['H30R30', 'u179', 'u180', 'beads']\n",
      "['Hyb', '750', '647', '488', '405']\n",
      "['H0R0', 'forward_chrom', 'reverse_chrom', 'beads', 'DAPI']\n",
      "['H1R1', 'u181', 'u182', 'beads']\n",
      "['H2R2', 'u183', 'u184', 'beads']\n",
      "['H3R3', 'u185', 'u186', 'beads']\n",
      "['H4R4', 'u187', 'u188', 'beads']\n",
      "['H5R5', 'u189', 'u190', 'beads']\n",
      "['H6R6', 'u191', 'u192', 'beads']\n",
      "['H7R7', 'u193', 'u194', 'beads']\n",
      "['H8R8', 'u195', 'u196', 'beads']\n",
      "['H9R9', 'u197', 'u198', 'beads']\n",
      "['H10R10', 'u199', 'u200', 'beads']\n",
      "['H11R11', 'u201', '', 'beads']\n",
      "['Hyb', '750', '647', '488', '405']\n",
      "['H0R0', 'forward_chrom', 'reverse_chrom', 'beads', 'DAPI']\n",
      "['H1R1', 'u203', 'u202', 'beads']\n",
      "['H2R2', 'u205', 'u204', 'beads']\n",
      "['H3R3', 'u207', 'u206', 'beads']\n",
      "['H4R4', 'u209', 'u208', 'beads']\n",
      "['H5R5', 'u211', 'u210', 'beads']\n",
      "['H6R6', 'u213', 'u212', 'beads']\n",
      "['H7R7', 'u215', 'u214', 'beads']\n",
      "['H8R8', 'u217', 'u216', 'beads']\n",
      "['H9R9', 'u219', 'u218', 'beads']\n",
      "['H10R10', 'u221', 'u220', 'beads']\n",
      "['H11R11', 'u223', 'u222', 'beads']\n",
      "['H12R12', 'u225', 'u224', 'beads']\n",
      "['H13R13', 'u227', 'u226', 'beads']\n",
      "['H14R14', 'u229', 'u228', 'beads']\n",
      "['H15R15', 'u231', 'u230', 'beads']\n",
      "['H16R16', 'u233', 'u232', 'beads']\n",
      "['H17R17', 'u235', 'u234', 'beads']\n",
      "['H18R18', 'u237', 'u236', 'beads']\n",
      "['H19R19', 'u239', 'u238', 'beads']\n",
      "['H20R20', 'u241', 'u240', 'beads']\n",
      "['H21R21', 'u243', 'u242', 'beads']\n",
      "['H22R22', 'u245', 'u244', 'beads']\n",
      "['H23R23', 'u247', 'u246', 'beads']\n",
      "['H24R24', 'u249', 'u248', 'beads']\n",
      "['H25R25', 'u251', 'u250', 'beads']\n",
      "['H26R26', 'u253', 'u252', 'beads']\n",
      "['H27R27', 'u255', 'u254', 'beads']\n",
      "['H28R28', 'u257', 'u256', 'beads']\n",
      "['H29R29', 'u259', 'u258', 'beads']\n",
      "['H30R30', 'u261', 'u260', 'beads']\n"
     ]
    }
   ],
   "source": [
    "unique_id = 1\n",
    "for _gname, _gene_unique_rds in unique_readouts_by_gene.items():\n",
    "\n",
    "    # sort regions with readout types\n",
    "    readout_by_channel = {_c:{} for _c in ref_readout_name_dict}\n",
    "    for _rid, (_reg_name, _rname) in enumerate(_gene_unique_rds.items()):\n",
    "        for _c in readout_by_channel:\n",
    "            if _rname in ref_readout_name_dict[_c] and _rid not in readout_by_channel[_c]:\n",
    "                readout_by_channel[_c][unique_id] = _rname\n",
    "                unique_id += 1\n",
    "    readout_by_channel = {_k: _v for _k, _v in readout_by_channel.items() if len(_v) > 0}\n",
    "    \n",
    "    # save color usage for parallel\n",
    "    import csv\n",
    "    drift_channel = '488'\n",
    "    dapi_channel = '405'\n",
    "    chrom_labels = {'750': 'forward',\n",
    "                    '647': 'reverse',}\n",
    "    dict_sizes = [len(_v) for _k,_v in readout_by_channel.items()]\n",
    "\n",
    "    with open(os.path.join(table_folder, f'Color_Usage_{_gname}.csv'), 'w', newline='') as csvfile:\n",
    "        csvwriter = csv.writer(csvfile, delimiter=',', \n",
    "                               quotechar='|', quoting=csv.QUOTE_MINIMAL)\n",
    "\n",
    "        # write header\n",
    "        _header = ['Hyb']\n",
    "        for _ch in sorted(readout_by_channel)[::-1]:\n",
    "            _header .append(str(_ch))\n",
    "        _header.append(drift_channel)\n",
    "        _header.append(dapi_channel)\n",
    "        print(_header)\n",
    "        csvwriter.writerow(_header)\n",
    "\n",
    "        # write reference frame\n",
    "        _ref_row = ['H0R0']\n",
    "        for _ch in sorted(readout_by_channel)[::-1]:\n",
    "            if str(_ch) in chrom_labels:\n",
    "                _ref_row.append(chrom_labels[str(_ch)]+'_chrom')\n",
    "            else:\n",
    "                _ref_row.append(\"\")\n",
    "        _ref_row.append('beads')\n",
    "        _ref_row.append('DAPI')\n",
    "        print(_ref_row)\n",
    "        csvwriter.writerow(_ref_row)\n",
    "\n",
    "\n",
    "        for _i in range(max(dict_sizes)):\n",
    "            _row = [f\"H{int(_i)+1}R{int(_i)+1}\",] \n",
    "            for _ch in sorted(readout_by_channel)[::-1]:\n",
    "                _dict = readout_by_channel[_ch]\n",
    "                if _i >= len(_dict):\n",
    "                    _row += ['']\n",
    "                else:\n",
    "                    _regs = sorted(_dict) \n",
    "                    _row += [f\"u{_regs[_i]}\"]\n",
    "            _row.append(\"beads\")\n",
    "            print(_row)\n",
    "            csvwriter.writerow(_row)\n",
    "\n",
    "            \n",
    "    # save adaptor sequences for parallel\n",
    "    with open(os.path.join(table_folder, f'adaptor_sequences_{_gname}.csv'), 'w', newline='') as csvfile:\n",
    "        csvwriter = csv.writer(csvfile, delimiter=',', \n",
    "                               quotechar='|', quoting=csv.QUOTE_MINIMAL)\n",
    "\n",
    "        _header = ['group', 'hyb', ]\n",
    "        for _ch in sorted(readout_by_channel)[::-1]:\n",
    "            _header += [f\"{_ch}_region\", f\"{_ch}_readout\"]\n",
    "\n",
    "        csvwriter.writerow(_header)\n",
    "\n",
    "        for _i in range(max(dict_sizes)):\n",
    "            _row = [int((_i)/32)+1, _i%32+1,] \n",
    "            for _ch, _dict in sorted(readout_by_channel.items())[::-1]:\n",
    "                if _i >= len(_dict):\n",
    "                    _row += ['', '']\n",
    "                else:\n",
    "                    _regs = sorted(_dict) \n",
    "                    _row += [_regs[_i], _dict[_regs[_i]]]      \n",
    "            csvwriter.writerow(_row)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Gene specific"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 392,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'Ccnd2': ['Stv_91'], 'Nkx2-1': ['Stv_92'], 'Slc17a7': ['Stv_94'], 'Slc32a1': ['Stv_95'], 'SST': ['Stv_99']}\n"
     ]
    }
   ],
   "source": [
    "# step 1: find unique tracing\n",
    "gene_readouts = {}\n",
    "for _rd, _rnames in readout_2_region.items():\n",
    "    #print(_rnames)\n",
    "    _reg_ids = [int(_rn.split('-')[-1]) for _rn in _rnames]\n",
    "    _related_gnames = [_rn.split('-'+_rn.split('-')[-1])[0] for _rn in _rnames]\n",
    "    if len(np.unique(_related_gnames)) == 1 and len(np.unique(_reg_ids)) > 1:\n",
    "        _gname = np.unique(_related_gnames)[0]\n",
    "        if _gname not in gene_readouts:\n",
    "            gene_readouts[_gname] = []\n",
    "        gene_readouts[_gname].append(_rd)\n",
    "print(gene_readouts)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### design color-swap adaptors"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 408,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Load readouts\n",
    "readout_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Readouts'\n",
    "ref_files = [_fl for _fl in os.listdir(readout_folder) if 'designed_readouts' in _fl]\n",
    "\n",
    "ref_readout_dict = {}\n",
    "for _fl in ref_files:\n",
    "    _channel = int(_fl.split('designed_readouts_')[1].split('.fasta')[0])\n",
    "    with open(os.path.join(readout_folder, _fl), 'r') as _rd_handle:\n",
    "        for _readout in SeqIO.parse(_rd_handle, \"fasta\"):\n",
    "            ref_readout_dict[_readout.id] = _readout"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 394,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[SeqRecord(seq=Seq('TTTGCACTGCCGTCCTTGAC'), id='Stv_82', name='Stv_82', description='Stv_82 cy7 rev-com_last20', dbxrefs=[]), SeqRecord(seq=Seq('GATCCGATTGGAACCGTCCC'), id='Stv_1', name='Stv_1', description='Stv_1 cy5 rev-com_last20', dbxrefs=[]), SeqRecord(seq=Seq('TGCGAACTGTCCGGCTTTCA'), id='Stv_79', name='Stv_79', description='Stv_79 cy3 rev-com_last20', dbxrefs=[])]\n"
     ]
    }
   ],
   "source": [
    "# load readout sites\n",
    "adaptor_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Adaptors'\n",
    "readout_site_file = os.path.join(adaptor_folder, 'Readout_sites.fasta')\n",
    "readout_sites = []\n",
    "with open(readout_site_file, 'r') as _rd_handle:\n",
    "    for _readout in SeqIO.parse(_rd_handle, \"fasta\"):\n",
    "        readout_sites.append(_readout)\n",
    "print(readout_sites)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 411,
   "metadata": {},
   "outputs": [],
   "source": [
    "colorswap_adaptors = []\n",
    "for _i, (_gene, _rds) in enumerate(gene_readouts.items()):\n",
    "    _rd_record = [_v for _k,_v in ref_readout_dict.items() if _k in _rds]\n",
    "    colorswap_adaptors.extend(library_tools.readouts.Generate_adaptors(_rd_record, [readout_sites[_i%2]]))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 414,
   "metadata": {},
   "outputs": [],
   "source": [
    "# generate csv\n",
    "import csv\n",
    "\n",
    "with open(os.path.join(table_folder, 'colorswap_gene_adaptors.tsv'), 'w') as csvfile:\n",
    "    csvwriter = csv.writer(csvfile, lineterminator=\"\\n\", delimiter='\\t', \n",
    "                       quotechar='|', quoting=csv.QUOTE_MINIMAL)\n",
    "    # write header\n",
    "    _header = ['Name', 'Sequence', 'Scale', 'Purification']\n",
    "    csvwriter.writerow(_header)\n",
    "    # write sequence\n",
    "    for _adaptor in colorswap_adaptors:\n",
    "        _info = [_adaptor.id, str(_adaptor.seq), '25nm', 'STD']\n",
    "        csvwriter.writerow(_info)\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 5.3 save region information"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 291,
   "metadata": {},
   "outputs": [],
   "source": [
    "region_info = {}\n",
    "for _r in pb_records:\n",
    "    _rname = _r.id.split('gene_')[1].split('_')[0]\n",
    "    \n",
    "    if _rname not in region_info:\n",
    "        # extract region info\n",
    "        _reg_info = _r.id.split('_gene')[0]\n",
    "        if 'chr' in _reg_info:\n",
    "            _reg_info = _reg_info.split('chr')[1]\n",
    "        #print(_reg_info)\n",
    "        chr_name = 'chr'+_reg_info.split(':')[0]\n",
    "        start = int(_reg_info.split(':')[1].split('-')[0])\n",
    "        end = int(_reg_info.split(':')[1].split('-')[1])\n",
    "        mid = int((start+end)/2)\n",
    "        region_info[_rname] = {'chr': chr_name,\n",
    "                             'start': start,\n",
    "                             'end': end,\n",
    "                             'mid': mid,\n",
    "                            }"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 292,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'Ccnd2-1': {'chr': 'chr6',\n",
       "  'start': 126980000,\n",
       "  'end': 126990000,\n",
       "  'mid': 126985000},\n",
       " 'Ccnd2-2': {'chr': 'chr6',\n",
       "  'start': 126990000,\n",
       "  'end': 127000000,\n",
       "  'mid': 126995000},\n",
       " 'Ccnd2-3': {'chr': 'chr6',\n",
       "  'start': 127000000,\n",
       "  'end': 127010000,\n",
       "  'mid': 127005000},\n",
       " 'Ccnd2-4': {'chr': 'chr6',\n",
       "  'start': 127010000,\n",
       "  'end': 127020000,\n",
       "  'mid': 127015000},\n",
       " 'Ccnd2-5': {'chr': 'chr6',\n",
       "  'start': 127020000,\n",
       "  'end': 127030000,\n",
       "  'mid': 127025000},\n",
       " 'Ccnd2-6': {'chr': 'chr6',\n",
       "  'start': 127030000,\n",
       "  'end': 127040000,\n",
       "  'mid': 127035000},\n",
       " 'Ccnd2-7': {'chr': 'chr6',\n",
       "  'start': 127040000,\n",
       "  'end': 127050000,\n",
       "  'mid': 127045000},\n",
       " 'Ccnd2-8': {'chr': 'chr6',\n",
       "  'start': 127050000,\n",
       "  'end': 127060000,\n",
       "  'mid': 127055000},\n",
       " 'Ccnd2-9': {'chr': 'chr6',\n",
       "  'start': 127060000,\n",
       "  'end': 127070000,\n",
       "  'mid': 127065000},\n",
       " 'Ccnd2-10': {'chr': 'chr6',\n",
       "  'start': 127070000,\n",
       "  'end': 127080000,\n",
       "  'mid': 127075000},\n",
       " 'Ccnd2-11': {'chr': 'chr6',\n",
       "  'start': 127080000,\n",
       "  'end': 127090000,\n",
       "  'mid': 127085000},\n",
       " 'Ccnd2-12': {'chr': 'chr6',\n",
       "  'start': 127090000,\n",
       "  'end': 127100000,\n",
       "  'mid': 127095000},\n",
       " 'Ccnd2-13': {'chr': 'chr6',\n",
       "  'start': 127100000,\n",
       "  'end': 127110000,\n",
       "  'mid': 127105000},\n",
       " 'Ccnd2-14': {'chr': 'chr6',\n",
       "  'start': 127110000,\n",
       "  'end': 127120000,\n",
       "  'mid': 127115000},\n",
       " 'Ccnd2-15': {'chr': 'chr6',\n",
       "  'start': 127120000,\n",
       "  'end': 127130000,\n",
       "  'mid': 127125000},\n",
       " 'Ccnd2-16': {'chr': 'chr6',\n",
       "  'start': 127130000,\n",
       "  'end': 127140000,\n",
       "  'mid': 127135000},\n",
       " 'Ccnd2-17': {'chr': 'chr6',\n",
       "  'start': 127140000,\n",
       "  'end': 127150000,\n",
       "  'mid': 127145000},\n",
       " 'Ccnd2-18': {'chr': 'chr6',\n",
       "  'start': 127150000,\n",
       "  'end': 127160000,\n",
       "  'mid': 127155000},\n",
       " 'Ccnd2-19': {'chr': 'chr6',\n",
       "  'start': 127160000,\n",
       "  'end': 127170000,\n",
       "  'mid': 127165000},\n",
       " 'Ccnd2-20': {'chr': 'chr6',\n",
       "  'start': 127170000,\n",
       "  'end': 127180000,\n",
       "  'mid': 127175000},\n",
       " 'Ccnd2-21': {'chr': 'chr6',\n",
       "  'start': 127180000,\n",
       "  'end': 127190000,\n",
       "  'mid': 127185000},\n",
       " 'Ccnd2-22': {'chr': 'chr6',\n",
       "  'start': 127190000,\n",
       "  'end': 127200000,\n",
       "  'mid': 127195000},\n",
       " 'Ccnd2-23': {'chr': 'chr6',\n",
       "  'start': 127200000,\n",
       "  'end': 127210000,\n",
       "  'mid': 127205000},\n",
       " 'Ccnd2-24': {'chr': 'chr6',\n",
       "  'start': 127210000,\n",
       "  'end': 127220000,\n",
       "  'mid': 127215000},\n",
       " 'Ccnd2-25': {'chr': 'chr6',\n",
       "  'start': 127220000,\n",
       "  'end': 127230000,\n",
       "  'mid': 127225000},\n",
       " 'Ccnd2-26': {'chr': 'chr6',\n",
       "  'start': 127230000,\n",
       "  'end': 127240000,\n",
       "  'mid': 127235000},\n",
       " 'Ccnd2-27': {'chr': 'chr6',\n",
       "  'start': 127240000,\n",
       "  'end': 127250000,\n",
       "  'mid': 127245000},\n",
       " 'Ccnd2-28': {'chr': 'chr6',\n",
       "  'start': 127250000,\n",
       "  'end': 127260000,\n",
       "  'mid': 127255000},\n",
       " 'Ccnd2-29': {'chr': 'chr6',\n",
       "  'start': 127260000,\n",
       "  'end': 127270000,\n",
       "  'mid': 127265000},\n",
       " 'Ccnd2-30': {'chr': 'chr6',\n",
       "  'start': 127270000,\n",
       "  'end': 127280000,\n",
       "  'mid': 127275000},\n",
       " 'Ccnd2-31': {'chr': 'chr6',\n",
       "  'start': 127280000,\n",
       "  'end': 127290000,\n",
       "  'mid': 127285000},\n",
       " 'Ccnd2-32': {'chr': 'chr6',\n",
       "  'start': 127290000,\n",
       "  'end': 127300000,\n",
       "  'mid': 127295000},\n",
       " 'Ccnd2-33': {'chr': 'chr6',\n",
       "  'start': 127300000,\n",
       "  'end': 127310000,\n",
       "  'mid': 127305000},\n",
       " 'Ccnd2-34': {'chr': 'chr6',\n",
       "  'start': 127310000,\n",
       "  'end': 127320000,\n",
       "  'mid': 127315000},\n",
       " 'Ccnd2-35': {'chr': 'chr6',\n",
       "  'start': 127320000,\n",
       "  'end': 127330000,\n",
       "  'mid': 127325000},\n",
       " 'Ccnd2-36': {'chr': 'chr6',\n",
       "  'start': 127330000,\n",
       "  'end': 127340000,\n",
       "  'mid': 127335000},\n",
       " 'Ccnd2-37': {'chr': 'chr6',\n",
       "  'start': 127340000,\n",
       "  'end': 127350000,\n",
       "  'mid': 127345000},\n",
       " 'Ccnd2-38': {'chr': 'chr6',\n",
       "  'start': 127350000,\n",
       "  'end': 127360000,\n",
       "  'mid': 127355000},\n",
       " 'Ccnd2-39': {'chr': 'chr6',\n",
       "  'start': 127360000,\n",
       "  'end': 127370000,\n",
       "  'mid': 127365000},\n",
       " 'Ccnd2-40': {'chr': 'chr6',\n",
       "  'start': 127370000,\n",
       "  'end': 127380000,\n",
       "  'mid': 127375000},\n",
       " 'Ccnd2-41': {'chr': 'chr6',\n",
       "  'start': 127380000,\n",
       "  'end': 127390000,\n",
       "  'mid': 127385000},\n",
       " 'Ccnd2-42': {'chr': 'chr6',\n",
       "  'start': 127390000,\n",
       "  'end': 127400000,\n",
       "  'mid': 127395000},\n",
       " 'Ccnd2-43': {'chr': 'chr6',\n",
       "  'start': 127400000,\n",
       "  'end': 127410000,\n",
       "  'mid': 127405000},\n",
       " 'Ccnd2-44': {'chr': 'chr6',\n",
       "  'start': 127410000,\n",
       "  'end': 127420000,\n",
       "  'mid': 127415000},\n",
       " 'Ccnd2-45': {'chr': 'chr6',\n",
       "  'start': 127420000,\n",
       "  'end': 127430000,\n",
       "  'mid': 127425000},\n",
       " 'Ccnd2-46': {'chr': 'chr6',\n",
       "  'start': 127430000,\n",
       "  'end': 127440000,\n",
       "  'mid': 127435000},\n",
       " 'Ccnd2-47': {'chr': 'chr6',\n",
       "  'start': 127440000,\n",
       "  'end': 127450000,\n",
       "  'mid': 127445000},\n",
       " 'Ccnd2-48': {'chr': 'chr6',\n",
       "  'start': 127450000,\n",
       "  'end': 127460000,\n",
       "  'mid': 127455000},\n",
       " 'Ccnd2-49': {'chr': 'chr6',\n",
       "  'start': 127460000,\n",
       "  'end': 127470000,\n",
       "  'mid': 127465000},\n",
       " 'Ccnd2-50': {'chr': 'chr6',\n",
       "  'start': 127470000,\n",
       "  'end': 127480000,\n",
       "  'mid': 127475000},\n",
       " 'Ccnd2-51': {'chr': 'chr6',\n",
       "  'start': 127480000,\n",
       "  'end': 127490000,\n",
       "  'mid': 127485000},\n",
       " 'Ccnd2-52': {'chr': 'chr6',\n",
       "  'start': 127490000,\n",
       "  'end': 127500000,\n",
       "  'mid': 127495000},\n",
       " 'Ccnd2-53': {'chr': 'chr6',\n",
       "  'start': 127500000,\n",
       "  'end': 127510000,\n",
       "  'mid': 127505000},\n",
       " 'Ccnd2-54': {'chr': 'chr6',\n",
       "  'start': 127510000,\n",
       "  'end': 127520000,\n",
       "  'mid': 127515000},\n",
       " 'Ccnd2-55': {'chr': 'chr6',\n",
       "  'start': 127520000,\n",
       "  'end': 127530000,\n",
       "  'mid': 127525000},\n",
       " 'Ccnd2-56': {'chr': 'chr6',\n",
       "  'start': 127530000,\n",
       "  'end': 127540000,\n",
       "  'mid': 127535000},\n",
       " 'Ccnd2-57': {'chr': 'chr6',\n",
       "  'start': 127540000,\n",
       "  'end': 127550000,\n",
       "  'mid': 127545000},\n",
       " 'Ccnd2-58': {'chr': 'chr6',\n",
       "  'start': 127550000,\n",
       "  'end': 127560000,\n",
       "  'mid': 127555000},\n",
       " 'Ccnd2-59': {'chr': 'chr6',\n",
       "  'start': 127560000,\n",
       "  'end': 127570000,\n",
       "  'mid': 127565000},\n",
       " 'Ccnd2-60': {'chr': 'chr6',\n",
       "  'start': 127570000,\n",
       "  'end': 127580000,\n",
       "  'mid': 127575000},\n",
       " 'Nkx2-1-1': {'chr': 'chr12',\n",
       "  'start': 56350000,\n",
       "  'end': 56360000,\n",
       "  'mid': 56355000},\n",
       " 'Nkx2-1-2': {'chr': 'chr12',\n",
       "  'start': 56360000,\n",
       "  'end': 56370000,\n",
       "  'mid': 56365000},\n",
       " 'Nkx2-1-3': {'chr': 'chr12',\n",
       "  'start': 56370000,\n",
       "  'end': 56380000,\n",
       "  'mid': 56375000},\n",
       " 'Nkx2-1-4': {'chr': 'chr12',\n",
       "  'start': 56380000,\n",
       "  'end': 56390000,\n",
       "  'mid': 56385000},\n",
       " 'Nkx2-1-5': {'chr': 'chr12',\n",
       "  'start': 56390000,\n",
       "  'end': 56400000,\n",
       "  'mid': 56395000},\n",
       " 'Nkx2-1-6': {'chr': 'chr12',\n",
       "  'start': 56400000,\n",
       "  'end': 56410000,\n",
       "  'mid': 56405000},\n",
       " 'Nkx2-1-7': {'chr': 'chr12',\n",
       "  'start': 56410000,\n",
       "  'end': 56420000,\n",
       "  'mid': 56415000},\n",
       " 'Nkx2-1-8': {'chr': 'chr12',\n",
       "  'start': 56420000,\n",
       "  'end': 56430000,\n",
       "  'mid': 56425000},\n",
       " 'Nkx2-1-9': {'chr': 'chr12',\n",
       "  'start': 56430000,\n",
       "  'end': 56440000,\n",
       "  'mid': 56435000},\n",
       " 'Nkx2-1-10': {'chr': 'chr12',\n",
       "  'start': 56440000,\n",
       "  'end': 56450000,\n",
       "  'mid': 56445000},\n",
       " 'Nkx2-1-11': {'chr': 'chr12',\n",
       "  'start': 56450000,\n",
       "  'end': 56460000,\n",
       "  'mid': 56455000},\n",
       " 'Nkx2-1-12': {'chr': 'chr12',\n",
       "  'start': 56460000,\n",
       "  'end': 56470000,\n",
       "  'mid': 56465000},\n",
       " 'Nkx2-1-13': {'chr': 'chr12',\n",
       "  'start': 56470000,\n",
       "  'end': 56480000,\n",
       "  'mid': 56475000},\n",
       " 'Nkx2-1-14': {'chr': 'chr12',\n",
       "  'start': 56480000,\n",
       "  'end': 56490000,\n",
       "  'mid': 56485000},\n",
       " 'Nkx2-1-15': {'chr': 'chr12',\n",
       "  'start': 56490000,\n",
       "  'end': 56500000,\n",
       "  'mid': 56495000},\n",
       " 'Nkx2-1-16': {'chr': 'chr12',\n",
       "  'start': 56500000,\n",
       "  'end': 56510000,\n",
       "  'mid': 56505000},\n",
       " 'Nkx2-1-17': {'chr': 'chr12',\n",
       "  'start': 56510000,\n",
       "  'end': 56520000,\n",
       "  'mid': 56515000},\n",
       " 'Nkx2-1-18': {'chr': 'chr12',\n",
       "  'start': 56520000,\n",
       "  'end': 56530000,\n",
       "  'mid': 56525000},\n",
       " 'Nkx2-1-19': {'chr': 'chr12',\n",
       "  'start': 56530000,\n",
       "  'end': 56540000,\n",
       "  'mid': 56535000},\n",
       " 'Nkx2-1-20': {'chr': 'chr12',\n",
       "  'start': 56540000,\n",
       "  'end': 56550000,\n",
       "  'mid': 56545000},\n",
       " 'Nkx2-1-21': {'chr': 'chr12',\n",
       "  'start': 56550000,\n",
       "  'end': 56560000,\n",
       "  'mid': 56555000},\n",
       " 'Nkx2-1-22': {'chr': 'chr12',\n",
       "  'start': 56560000,\n",
       "  'end': 56570000,\n",
       "  'mid': 56565000},\n",
       " 'Nkx2-1-23': {'chr': 'chr12',\n",
       "  'start': 56570000,\n",
       "  'end': 56580000,\n",
       "  'mid': 56575000},\n",
       " 'Nkx2-1-24': {'chr': 'chr12',\n",
       "  'start': 56580000,\n",
       "  'end': 56590000,\n",
       "  'mid': 56585000},\n",
       " 'Nkx2-1-25': {'chr': 'chr12',\n",
       "  'start': 56590000,\n",
       "  'end': 56600000,\n",
       "  'mid': 56595000},\n",
       " 'Nkx2-1-26': {'chr': 'chr12',\n",
       "  'start': 56600000,\n",
       "  'end': 56610000,\n",
       "  'mid': 56605000},\n",
       " 'Nkx2-1-27': {'chr': 'chr12',\n",
       "  'start': 56610000,\n",
       "  'end': 56620000,\n",
       "  'mid': 56615000},\n",
       " 'Nkx2-1-28': {'chr': 'chr12',\n",
       "  'start': 56620000,\n",
       "  'end': 56630000,\n",
       "  'mid': 56625000},\n",
       " 'Nkx2-1-29': {'chr': 'chr12',\n",
       "  'start': 56630000,\n",
       "  'end': 56640000,\n",
       "  'mid': 56635000},\n",
       " 'Nkx2-1-30': {'chr': 'chr12',\n",
       "  'start': 56640000,\n",
       "  'end': 56650000,\n",
       "  'mid': 56645000},\n",
       " 'Nkx2-1-31': {'chr': 'chr12',\n",
       "  'start': 56650000,\n",
       "  'end': 56660000,\n",
       "  'mid': 56655000},\n",
       " 'Nkx2-1-32': {'chr': 'chr12',\n",
       "  'start': 56660000,\n",
       "  'end': 56670000,\n",
       "  'mid': 56665000},\n",
       " 'Nkx2-1-33': {'chr': 'chr12',\n",
       "  'start': 56670000,\n",
       "  'end': 56680000,\n",
       "  'mid': 56675000},\n",
       " 'Nkx2-1-34': {'chr': 'chr12',\n",
       "  'start': 56680000,\n",
       "  'end': 56690000,\n",
       "  'mid': 56685000},\n",
       " 'Nkx2-1-35': {'chr': 'chr12',\n",
       "  'start': 56690000,\n",
       "  'end': 56700000,\n",
       "  'mid': 56695000},\n",
       " 'Nkx2-1-36': {'chr': 'chr12',\n",
       "  'start': 56700000,\n",
       "  'end': 56710000,\n",
       "  'mid': 56705000},\n",
       " 'Nkx2-1-37': {'chr': 'chr12',\n",
       "  'start': 56710000,\n",
       "  'end': 56720000,\n",
       "  'mid': 56715000},\n",
       " 'Nkx2-1-38': {'chr': 'chr12',\n",
       "  'start': 56720000,\n",
       "  'end': 56730000,\n",
       "  'mid': 56725000},\n",
       " 'Nkx2-1-39': {'chr': 'chr12',\n",
       "  'start': 56730000,\n",
       "  'end': 56740000,\n",
       "  'mid': 56735000},\n",
       " 'Nkx2-1-40': {'chr': 'chr12',\n",
       "  'start': 56740000,\n",
       "  'end': 56750000,\n",
       "  'mid': 56745000},\n",
       " 'Nkx2-1-41': {'chr': 'chr12',\n",
       "  'start': 56750000,\n",
       "  'end': 56760000,\n",
       "  'mid': 56755000},\n",
       " 'Nkx2-1-42': {'chr': 'chr12',\n",
       "  'start': 56760000,\n",
       "  'end': 56770000,\n",
       "  'mid': 56765000},\n",
       " 'Nkx2-1-43': {'chr': 'chr12',\n",
       "  'start': 56770000,\n",
       "  'end': 56780000,\n",
       "  'mid': 56775000},\n",
       " 'Nkx2-1-44': {'chr': 'chr12',\n",
       "  'start': 56780000,\n",
       "  'end': 56790000,\n",
       "  'mid': 56785000},\n",
       " 'Nkx2-1-45': {'chr': 'chr12',\n",
       "  'start': 56790000,\n",
       "  'end': 56800000,\n",
       "  'mid': 56795000},\n",
       " 'Nkx2-1-46': {'chr': 'chr12',\n",
       "  'start': 56800000,\n",
       "  'end': 56810000,\n",
       "  'mid': 56805000},\n",
       " 'Nkx2-1-47': {'chr': 'chr12',\n",
       "  'start': 56810000,\n",
       "  'end': 56820000,\n",
       "  'mid': 56815000},\n",
       " 'Nkx2-1-48': {'chr': 'chr12',\n",
       "  'start': 56820000,\n",
       "  'end': 56830000,\n",
       "  'mid': 56825000},\n",
       " 'Nkx2-1-49': {'chr': 'chr12',\n",
       "  'start': 56830000,\n",
       "  'end': 56840000,\n",
       "  'mid': 56835000},\n",
       " 'Nkx2-1-50': {'chr': 'chr12',\n",
       "  'start': 56840000,\n",
       "  'end': 56850000,\n",
       "  'mid': 56845000},\n",
       " 'Nkx2-1-51': {'chr': 'chr12',\n",
       "  'start': 56850000,\n",
       "  'end': 56860000,\n",
       "  'mid': 56855000},\n",
       " 'Nkx2-1-52': {'chr': 'chr12',\n",
       "  'start': 56860000,\n",
       "  'end': 56870000,\n",
       "  'mid': 56865000},\n",
       " 'Nkx2-1-53': {'chr': 'chr12',\n",
       "  'start': 56870000,\n",
       "  'end': 56880000,\n",
       "  'mid': 56875000},\n",
       " 'Nkx2-1-54': {'chr': 'chr12',\n",
       "  'start': 56880000,\n",
       "  'end': 56890000,\n",
       "  'mid': 56885000},\n",
       " 'Nkx2-1-55': {'chr': 'chr12',\n",
       "  'start': 56890000,\n",
       "  'end': 56900000,\n",
       "  'mid': 56895000},\n",
       " 'Nkx2-1-56': {'chr': 'chr12',\n",
       "  'start': 56900000,\n",
       "  'end': 56910000,\n",
       "  'mid': 56905000},\n",
       " 'Nkx2-1-57': {'chr': 'chr12',\n",
       "  'start': 56910000,\n",
       "  'end': 56920000,\n",
       "  'mid': 56915000},\n",
       " 'Nkx2-1-58': {'chr': 'chr12',\n",
       "  'start': 56920000,\n",
       "  'end': 56930000,\n",
       "  'mid': 56925000},\n",
       " 'Nkx2-1-59': {'chr': 'chr12',\n",
       "  'start': 56930000,\n",
       "  'end': 56940000,\n",
       "  'mid': 56935000},\n",
       " 'Nkx2-1-60': {'chr': 'chr12',\n",
       "  'start': 56940000,\n",
       "  'end': 56950000,\n",
       "  'mid': 56945000},\n",
       " 'Slc17a7-1': {'chr': 'chr7',\n",
       "  'start': 44740000,\n",
       "  'end': 44750000,\n",
       "  'mid': 44745000},\n",
       " 'Slc17a7-2': {'chr': 'chr7',\n",
       "  'start': 44750000,\n",
       "  'end': 44760000,\n",
       "  'mid': 44755000},\n",
       " 'Slc17a7-3': {'chr': 'chr7',\n",
       "  'start': 44760000,\n",
       "  'end': 44770000,\n",
       "  'mid': 44765000},\n",
       " 'Slc17a7-4': {'chr': 'chr7',\n",
       "  'start': 44770000,\n",
       "  'end': 44780000,\n",
       "  'mid': 44775000},\n",
       " 'Slc17a7-5': {'chr': 'chr7',\n",
       "  'start': 44780000,\n",
       "  'end': 44790000,\n",
       "  'mid': 44785000},\n",
       " 'Slc17a7-6': {'chr': 'chr7',\n",
       "  'start': 44790000,\n",
       "  'end': 44800000,\n",
       "  'mid': 44795000},\n",
       " 'Slc17a7-7': {'chr': 'chr7',\n",
       "  'start': 44800000,\n",
       "  'end': 44810000,\n",
       "  'mid': 44805000},\n",
       " 'Slc17a7-8': {'chr': 'chr7',\n",
       "  'start': 44810000,\n",
       "  'end': 44820000,\n",
       "  'mid': 44815000},\n",
       " 'Slc17a7-9': {'chr': 'chr7',\n",
       "  'start': 44820000,\n",
       "  'end': 44830000,\n",
       "  'mid': 44825000},\n",
       " 'Slc17a7-10': {'chr': 'chr7',\n",
       "  'start': 44830000,\n",
       "  'end': 44840000,\n",
       "  'mid': 44835000},\n",
       " 'Slc17a7-11': {'chr': 'chr7',\n",
       "  'start': 44840000,\n",
       "  'end': 44850000,\n",
       "  'mid': 44845000},\n",
       " 'Slc17a7-12': {'chr': 'chr7',\n",
       "  'start': 44850000,\n",
       "  'end': 44860000,\n",
       "  'mid': 44855000},\n",
       " 'Slc17a7-13': {'chr': 'chr7',\n",
       "  'start': 44860000,\n",
       "  'end': 44870000,\n",
       "  'mid': 44865000},\n",
       " 'Slc17a7-14': {'chr': 'chr7',\n",
       "  'start': 44870000,\n",
       "  'end': 44880000,\n",
       "  'mid': 44875000},\n",
       " 'Slc17a7-15': {'chr': 'chr7',\n",
       "  'start': 44880000,\n",
       "  'end': 44890000,\n",
       "  'mid': 44885000},\n",
       " 'Slc17a7-16': {'chr': 'chr7',\n",
       "  'start': 44890000,\n",
       "  'end': 44900000,\n",
       "  'mid': 44895000},\n",
       " 'Slc17a7-17': {'chr': 'chr7',\n",
       "  'start': 44900000,\n",
       "  'end': 44910000,\n",
       "  'mid': 44905000},\n",
       " 'Slc17a7-18': {'chr': 'chr7',\n",
       "  'start': 44910000,\n",
       "  'end': 44920000,\n",
       "  'mid': 44915000},\n",
       " 'Slc17a7-19': {'chr': 'chr7',\n",
       "  'start': 44920000,\n",
       "  'end': 44930000,\n",
       "  'mid': 44925000},\n",
       " 'Slc17a7-20': {'chr': 'chr7',\n",
       "  'start': 44930000,\n",
       "  'end': 44940000,\n",
       "  'mid': 44935000},\n",
       " 'Slc17a7-21': {'chr': 'chr7',\n",
       "  'start': 44940000,\n",
       "  'end': 44950000,\n",
       "  'mid': 44945000},\n",
       " 'Slc17a7-22': {'chr': 'chr7',\n",
       "  'start': 44950000,\n",
       "  'end': 44960000,\n",
       "  'mid': 44955000},\n",
       " 'Slc17a7-23': {'chr': 'chr7',\n",
       "  'start': 44960000,\n",
       "  'end': 44970000,\n",
       "  'mid': 44965000},\n",
       " 'Slc17a7-24': {'chr': 'chr7',\n",
       "  'start': 44970000,\n",
       "  'end': 44980000,\n",
       "  'mid': 44975000},\n",
       " 'Slc17a7-25': {'chr': 'chr7',\n",
       "  'start': 44980000,\n",
       "  'end': 44990000,\n",
       "  'mid': 44985000},\n",
       " 'Slc17a7-26': {'chr': 'chr7',\n",
       "  'start': 44990000,\n",
       "  'end': 45000000,\n",
       "  'mid': 44995000},\n",
       " 'Slc17a7-27': {'chr': 'chr7',\n",
       "  'start': 45000000,\n",
       "  'end': 45010000,\n",
       "  'mid': 45005000},\n",
       " 'Slc17a7-28': {'chr': 'chr7',\n",
       "  'start': 45010000,\n",
       "  'end': 45020000,\n",
       "  'mid': 45015000},\n",
       " 'Slc17a7-29': {'chr': 'chr7',\n",
       "  'start': 45020000,\n",
       "  'end': 45030000,\n",
       "  'mid': 45025000},\n",
       " 'Slc17a7-30': {'chr': 'chr7',\n",
       "  'start': 45030000,\n",
       "  'end': 45040000,\n",
       "  'mid': 45035000},\n",
       " 'Slc17a7-31': {'chr': 'chr7',\n",
       "  'start': 45040000,\n",
       "  'end': 45050000,\n",
       "  'mid': 45045000},\n",
       " 'Slc17a7-32': {'chr': 'chr7',\n",
       "  'start': 45050000,\n",
       "  'end': 45060000,\n",
       "  'mid': 45055000},\n",
       " 'Slc17a7-33': {'chr': 'chr7',\n",
       "  'start': 45060000,\n",
       "  'end': 45070000,\n",
       "  'mid': 45065000},\n",
       " 'Slc17a7-34': {'chr': 'chr7',\n",
       "  'start': 45070000,\n",
       "  'end': 45080000,\n",
       "  'mid': 45075000},\n",
       " 'Slc17a7-35': {'chr': 'chr7',\n",
       "  'start': 45080000,\n",
       "  'end': 45090000,\n",
       "  'mid': 45085000},\n",
       " 'Slc17a7-36': {'chr': 'chr7',\n",
       "  'start': 45090000,\n",
       "  'end': 45100000,\n",
       "  'mid': 45095000},\n",
       " 'Slc17a7-37': {'chr': 'chr7',\n",
       "  'start': 45100000,\n",
       "  'end': 45110000,\n",
       "  'mid': 45105000},\n",
       " 'Slc17a7-38': {'chr': 'chr7',\n",
       "  'start': 45110000,\n",
       "  'end': 45120000,\n",
       "  'mid': 45115000},\n",
       " 'Slc17a7-39': {'chr': 'chr7',\n",
       "  'start': 45120000,\n",
       "  'end': 45130000,\n",
       "  'mid': 45125000},\n",
       " 'Slc17a7-40': {'chr': 'chr7',\n",
       "  'start': 45130000,\n",
       "  'end': 45140000,\n",
       "  'mid': 45135000},\n",
       " 'Slc17a7-41': {'chr': 'chr7',\n",
       "  'start': 45140000,\n",
       "  'end': 45150000,\n",
       "  'mid': 45145000},\n",
       " 'Slc17a7-42': {'chr': 'chr7',\n",
       "  'start': 45150000,\n",
       "  'end': 45160000,\n",
       "  'mid': 45155000},\n",
       " 'Slc17a7-43': {'chr': 'chr7',\n",
       "  'start': 45160000,\n",
       "  'end': 45170000,\n",
       "  'mid': 45165000},\n",
       " 'Slc17a7-44': {'chr': 'chr7',\n",
       "  'start': 45170000,\n",
       "  'end': 45180000,\n",
       "  'mid': 45175000},\n",
       " 'Slc17a7-45': {'chr': 'chr7',\n",
       "  'start': 45180000,\n",
       "  'end': 45190000,\n",
       "  'mid': 45185000},\n",
       " 'Slc17a7-46': {'chr': 'chr7',\n",
       "  'start': 45190000,\n",
       "  'end': 45200000,\n",
       "  'mid': 45195000},\n",
       " 'Slc17a7-47': {'chr': 'chr7',\n",
       "  'start': 45200000,\n",
       "  'end': 45210000,\n",
       "  'mid': 45205000},\n",
       " 'Slc17a7-48': {'chr': 'chr7',\n",
       "  'start': 45210000,\n",
       "  'end': 45220000,\n",
       "  'mid': 45215000},\n",
       " 'Slc17a7-49': {'chr': 'chr7',\n",
       "  'start': 45220000,\n",
       "  'end': 45230000,\n",
       "  'mid': 45225000},\n",
       " 'Slc17a7-50': {'chr': 'chr7',\n",
       "  'start': 45230000,\n",
       "  'end': 45240000,\n",
       "  'mid': 45235000},\n",
       " 'Slc17a7-51': {'chr': 'chr7',\n",
       "  'start': 45240000,\n",
       "  'end': 45250000,\n",
       "  'mid': 45245000},\n",
       " 'Slc17a7-52': {'chr': 'chr7',\n",
       "  'start': 45250000,\n",
       "  'end': 45260000,\n",
       "  'mid': 45255000},\n",
       " 'Slc17a7-53': {'chr': 'chr7',\n",
       "  'start': 45260000,\n",
       "  'end': 45270000,\n",
       "  'mid': 45265000},\n",
       " 'Slc17a7-54': {'chr': 'chr7',\n",
       "  'start': 45270000,\n",
       "  'end': 45280000,\n",
       "  'mid': 45275000},\n",
       " 'Slc17a7-55': {'chr': 'chr7',\n",
       "  'start': 45280000,\n",
       "  'end': 45290000,\n",
       "  'mid': 45285000},\n",
       " 'Slc17a7-56': {'chr': 'chr7',\n",
       "  'start': 45290000,\n",
       "  'end': 45300000,\n",
       "  'mid': 45295000},\n",
       " 'Slc17a7-57': {'chr': 'chr7',\n",
       "  'start': 45300000,\n",
       "  'end': 45310000,\n",
       "  'mid': 45305000},\n",
       " 'Slc17a7-58': {'chr': 'chr7',\n",
       "  'start': 45310000,\n",
       "  'end': 45320000,\n",
       "  'mid': 45315000},\n",
       " 'Slc17a7-59': {'chr': 'chr7',\n",
       "  'start': 45320000,\n",
       "  'end': 45330000,\n",
       "  'mid': 45325000},\n",
       " 'Slc17a7-60': {'chr': 'chr7',\n",
       "  'start': 45330000,\n",
       "  'end': 45340000,\n",
       "  'mid': 45335000},\n",
       " 'Slc32a1-1': {'chr': 'chr2',\n",
       "  'start': 158510767,\n",
       "  'end': 158520767,\n",
       "  'mid': 158515767},\n",
       " 'Slc32a1-2': {'chr': 'chr2',\n",
       "  'start': 158520767,\n",
       "  'end': 158530767,\n",
       "  'mid': 158525767},\n",
       " 'Slc32a1-3': {'chr': 'chr2',\n",
       "  'start': 158530767,\n",
       "  'end': 158540767,\n",
       "  'mid': 158535767},\n",
       " 'Slc32a1-4': {'chr': 'chr2',\n",
       "  'start': 158540767,\n",
       "  'end': 158550767,\n",
       "  'mid': 158545767},\n",
       " 'Slc32a1-5': {'chr': 'chr2',\n",
       "  'start': 158550767,\n",
       "  'end': 158560767,\n",
       "  'mid': 158555767},\n",
       " 'Slc32a1-6': {'chr': 'chr2',\n",
       "  'start': 158560767,\n",
       "  'end': 158570767,\n",
       "  'mid': 158565767},\n",
       " 'Slc32a1-7': {'chr': 'chr2',\n",
       "  'start': 158570767,\n",
       "  'end': 158580767,\n",
       "  'mid': 158575767},\n",
       " 'Slc32a1-8': {'chr': 'chr2',\n",
       "  'start': 158580767,\n",
       "  'end': 158590767,\n",
       "  'mid': 158585767},\n",
       " 'Slc32a1-9': {'chr': 'chr2',\n",
       "  'start': 158590767,\n",
       "  'end': 158600767,\n",
       "  'mid': 158595767},\n",
       " 'Slc32a1-10': {'chr': 'chr2',\n",
       "  'start': 158600767,\n",
       "  'end': 158610767,\n",
       "  'mid': 158605767},\n",
       " 'Slc32a1-11': {'chr': 'chr2',\n",
       "  'start': 158610767,\n",
       "  'end': 158620767,\n",
       "  'mid': 158615767},\n",
       " 'Slc32a1-12': {'chr': 'chr2',\n",
       "  'start': 158620767,\n",
       "  'end': 158630767,\n",
       "  'mid': 158625767},\n",
       " 'Slc32a1-13': {'chr': 'chr2',\n",
       "  'start': 158630767,\n",
       "  'end': 158640767,\n",
       "  'mid': 158635767},\n",
       " 'Slc32a1-14': {'chr': 'chr2',\n",
       "  'start': 158640767,\n",
       "  'end': 158650767,\n",
       "  'mid': 158645767},\n",
       " 'Slc32a1-15': {'chr': 'chr2',\n",
       "  'start': 158650767,\n",
       "  'end': 158660767,\n",
       "  'mid': 158655767},\n",
       " 'Slc32a1-16': {'chr': 'chr2',\n",
       "  'start': 158660767,\n",
       "  'end': 158670767,\n",
       "  'mid': 158665767},\n",
       " 'Slc32a1-17': {'chr': 'chr2',\n",
       "  'start': 158670767,\n",
       "  'end': 158680767,\n",
       "  'mid': 158675767},\n",
       " 'Slc32a1-18': {'chr': 'chr2',\n",
       "  'start': 158680767,\n",
       "  'end': 158690767,\n",
       "  'mid': 158685767},\n",
       " 'Slc32a1-19': {'chr': 'chr2',\n",
       "  'start': 158690767,\n",
       "  'end': 158700767,\n",
       "  'mid': 158695767},\n",
       " 'Slc32a1-20': {'chr': 'chr2',\n",
       "  'start': 158700767,\n",
       "  'end': 158710767,\n",
       "  'mid': 158705767},\n",
       " 'Slc32a1-21': {'chr': 'chr2',\n",
       "  'start': 158710767,\n",
       "  'end': 158720767,\n",
       "  'mid': 158715767},\n",
       " 'SST-1': {'chr': 'chr16',\n",
       "  'start': 23730000,\n",
       "  'end': 23740000,\n",
       "  'mid': 23735000},\n",
       " 'SST-2': {'chr': 'chr16',\n",
       "  'start': 23740000,\n",
       "  'end': 23750000,\n",
       "  'mid': 23745000},\n",
       " 'SST-3': {'chr': 'chr16',\n",
       "  'start': 23750000,\n",
       "  'end': 23760000,\n",
       "  'mid': 23755000},\n",
       " 'SST-4': {'chr': 'chr16',\n",
       "  'start': 23760000,\n",
       "  'end': 23770000,\n",
       "  'mid': 23765000},\n",
       " 'SST-5': {'chr': 'chr16',\n",
       "  'start': 23770000,\n",
       "  'end': 23780000,\n",
       "  'mid': 23775000},\n",
       " 'SST-6': {'chr': 'chr16',\n",
       "  'start': 23780000,\n",
       "  'end': 23790000,\n",
       "  'mid': 23785000},\n",
       " 'SST-7': {'chr': 'chr16',\n",
       "  'start': 23790000,\n",
       "  'end': 23800000,\n",
       "  'mid': 23795000},\n",
       " 'SST-8': {'chr': 'chr16',\n",
       "  'start': 23800000,\n",
       "  'end': 23810000,\n",
       "  'mid': 23805000},\n",
       " 'SST-9': {'chr': 'chr16',\n",
       "  'start': 23810000,\n",
       "  'end': 23820000,\n",
       "  'mid': 23815000},\n",
       " 'SST-10': {'chr': 'chr16',\n",
       "  'start': 23820000,\n",
       "  'end': 23830000,\n",
       "  'mid': 23825000},\n",
       " 'SST-11': {'chr': 'chr16',\n",
       "  'start': 23830000,\n",
       "  'end': 23840000,\n",
       "  'mid': 23835000},\n",
       " 'SST-12': {'chr': 'chr16',\n",
       "  'start': 23840000,\n",
       "  'end': 23850000,\n",
       "  'mid': 23845000},\n",
       " 'SST-13': {'chr': 'chr16',\n",
       "  'start': 23850000,\n",
       "  'end': 23860000,\n",
       "  'mid': 23855000},\n",
       " 'SST-14': {'chr': 'chr16',\n",
       "  'start': 23860000,\n",
       "  'end': 23870000,\n",
       "  'mid': 23865000},\n",
       " 'SST-15': {'chr': 'chr16',\n",
       "  'start': 23870000,\n",
       "  'end': 23880000,\n",
       "  'mid': 23875000},\n",
       " 'SST-16': {'chr': 'chr16',\n",
       "  'start': 23880000,\n",
       "  'end': 23890000,\n",
       "  'mid': 23885000},\n",
       " 'SST-17': {'chr': 'chr16',\n",
       "  'start': 23890000,\n",
       "  'end': 23900000,\n",
       "  'mid': 23895000},\n",
       " 'SST-18': {'chr': 'chr16',\n",
       "  'start': 23900000,\n",
       "  'end': 23910000,\n",
       "  'mid': 23905000},\n",
       " 'SST-19': {'chr': 'chr16',\n",
       "  'start': 23910000,\n",
       "  'end': 23920000,\n",
       "  'mid': 23915000},\n",
       " 'SST-20': {'chr': 'chr16',\n",
       "  'start': 23920000,\n",
       "  'end': 23930000,\n",
       "  'mid': 23925000},\n",
       " 'SST-21': {'chr': 'chr16',\n",
       "  'start': 23930000,\n",
       "  'end': 23940000,\n",
       "  'mid': 23935000},\n",
       " 'SST-22': {'chr': 'chr16',\n",
       "  'start': 23940000,\n",
       "  'end': 23950000,\n",
       "  'mid': 23945000},\n",
       " 'SST-23': {'chr': 'chr16',\n",
       "  'start': 23950000,\n",
       "  'end': 23960000,\n",
       "  'mid': 23955000},\n",
       " 'SST-24': {'chr': 'chr16',\n",
       "  'start': 23960000,\n",
       "  'end': 23970000,\n",
       "  'mid': 23965000},\n",
       " 'SST-25': {'chr': 'chr16',\n",
       "  'start': 23970000,\n",
       "  'end': 23980000,\n",
       "  'mid': 23975000},\n",
       " 'SST-26': {'chr': 'chr16',\n",
       "  'start': 23980000,\n",
       "  'end': 23990000,\n",
       "  'mid': 23985000},\n",
       " 'SST-27': {'chr': 'chr16',\n",
       "  'start': 23990000,\n",
       "  'end': 24000000,\n",
       "  'mid': 23995000},\n",
       " 'SST-28': {'chr': 'chr16',\n",
       "  'start': 24000000,\n",
       "  'end': 24010000,\n",
       "  'mid': 24005000},\n",
       " 'SST-29': {'chr': 'chr16',\n",
       "  'start': 24010000,\n",
       "  'end': 24020000,\n",
       "  'mid': 24015000},\n",
       " 'SST-30': {'chr': 'chr16',\n",
       "  'start': 24020000,\n",
       "  'end': 24030000,\n",
       "  'mid': 24025000},\n",
       " 'SST-31': {'chr': 'chr16',\n",
       "  'start': 24030000,\n",
       "  'end': 24040000,\n",
       "  'mid': 24035000},\n",
       " 'SST-32': {'chr': 'chr16',\n",
       "  'start': 24040000,\n",
       "  'end': 24050000,\n",
       "  'mid': 24045000},\n",
       " 'SST-33': {'chr': 'chr16',\n",
       "  'start': 24050000,\n",
       "  'end': 24060000,\n",
       "  'mid': 24055000},\n",
       " 'SST-34': {'chr': 'chr16',\n",
       "  'start': 24060000,\n",
       "  'end': 24070000,\n",
       "  'mid': 24065000},\n",
       " 'SST-35': {'chr': 'chr16',\n",
       "  'start': 24070000,\n",
       "  'end': 24080000,\n",
       "  'mid': 24075000},\n",
       " 'SST-36': {'chr': 'chr16',\n",
       "  'start': 24080000,\n",
       "  'end': 24090000,\n",
       "  'mid': 24085000},\n",
       " 'SST-37': {'chr': 'chr16',\n",
       "  'start': 24090000,\n",
       "  'end': 24100000,\n",
       "  'mid': 24095000},\n",
       " 'SST-38': {'chr': 'chr16',\n",
       "  'start': 24100000,\n",
       "  'end': 24110000,\n",
       "  'mid': 24105000},\n",
       " 'SST-39': {'chr': 'chr16',\n",
       "  'start': 24110000,\n",
       "  'end': 24120000,\n",
       "  'mid': 24115000},\n",
       " 'SST-40': {'chr': 'chr16',\n",
       "  'start': 24120000,\n",
       "  'end': 24130000,\n",
       "  'mid': 24125000},\n",
       " 'SST-41': {'chr': 'chr16',\n",
       "  'start': 24130000,\n",
       "  'end': 24140000,\n",
       "  'mid': 24135000},\n",
       " 'SST-42': {'chr': 'chr16',\n",
       "  'start': 24140000,\n",
       "  'end': 24150000,\n",
       "  'mid': 24145000},\n",
       " 'SST-43': {'chr': 'chr16',\n",
       "  'start': 24150000,\n",
       "  'end': 24160000,\n",
       "  'mid': 24155000},\n",
       " 'SST-44': {'chr': 'chr16',\n",
       "  'start': 24160000,\n",
       "  'end': 24170000,\n",
       "  'mid': 24165000},\n",
       " 'SST-45': {'chr': 'chr16',\n",
       "  'start': 24170000,\n",
       "  'end': 24180000,\n",
       "  'mid': 24175000},\n",
       " 'SST-46': {'chr': 'chr16',\n",
       "  'start': 24180000,\n",
       "  'end': 24190000,\n",
       "  'mid': 24185000},\n",
       " 'SST-47': {'chr': 'chr16',\n",
       "  'start': 24190000,\n",
       "  'end': 24200000,\n",
       "  'mid': 24195000},\n",
       " 'SST-48': {'chr': 'chr16',\n",
       "  'start': 24200000,\n",
       "  'end': 24210000,\n",
       "  'mid': 24205000},\n",
       " 'SST-49': {'chr': 'chr16',\n",
       "  'start': 24210000,\n",
       "  'end': 24220000,\n",
       "  'mid': 24215000},\n",
       " 'SST-50': {'chr': 'chr16',\n",
       "  'start': 24220000,\n",
       "  'end': 24230000,\n",
       "  'mid': 24225000},\n",
       " 'SST-51': {'chr': 'chr16',\n",
       "  'start': 24230000,\n",
       "  'end': 24240000,\n",
       "  'mid': 24235000},\n",
       " 'SST-52': {'chr': 'chr16',\n",
       "  'start': 24240000,\n",
       "  'end': 24250000,\n",
       "  'mid': 24245000},\n",
       " 'SST-53': {'chr': 'chr16',\n",
       "  'start': 24250000,\n",
       "  'end': 24260000,\n",
       "  'mid': 24255000},\n",
       " 'SST-54': {'chr': 'chr16',\n",
       "  'start': 24260000,\n",
       "  'end': 24270000,\n",
       "  'mid': 24265000},\n",
       " 'SST-55': {'chr': 'chr16',\n",
       "  'start': 24270000,\n",
       "  'end': 24280000,\n",
       "  'mid': 24275000},\n",
       " 'SST-56': {'chr': 'chr16',\n",
       "  'start': 24280000,\n",
       "  'end': 24290000,\n",
       "  'mid': 24285000},\n",
       " 'SST-57': {'chr': 'chr16',\n",
       "  'start': 24290000,\n",
       "  'end': 24300000,\n",
       "  'mid': 24295000},\n",
       " 'SST-58': {'chr': 'chr16',\n",
       "  'start': 24300000,\n",
       "  'end': 24310000,\n",
       "  'mid': 24305000},\n",
       " 'SST-59': {'chr': 'chr16',\n",
       "  'start': 24310000,\n",
       "  'end': 24320000,\n",
       "  'mid': 24315000},\n",
       " 'SST-60': {'chr': 'chr16',\n",
       "  'start': 24320000,\n",
       "  'end': 24330000,\n",
       "  'mid': 24325000}}"
      ]
     },
     "execution_count": 292,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "region_info"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 352,
   "metadata": {},
   "outputs": [],
   "source": [
    "import csv\n",
    "\n",
    "with open(os.path.join(table_folder, 'Region_Positions.csv'), 'w', newline='') as csvfile:\n",
    "    csvwriter = csv.writer(csvfile, delimiter=',', \n",
    "                           quotechar='|', quoting=csv.QUOTE_MINIMAL)\n",
    "    # write header\n",
    "    csvwriter.writerow(['region', 'chr', 'start', 'end', 'midpoint',])\n",
    "    \n",
    "    for _i,_info in region_info.items():\n",
    "        csvwriter.writerow([_i, \n",
    "                            _info['chr'], \n",
    "                            _info['start'],\n",
    "                            _info['end'],\n",
    "                            _info['mid'],\n",
    "                           ])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.9"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
