{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<a id='0.1'></a>\n",
    "## 0.1 load required packages"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Table of contents\n",
    "\n",
    "\n",
    "> 0. [Minimum required packages and settings](#0)\n",
    ">>\n",
    ">> 0.1: [import required packages](#0.1)\n",
    ">\n",
    "> 1. [Extract region sequences](#1)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<a id='0.1'></a>\n",
    "## 0.1 load required packages"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "creating new log file\n",
      "2022-01-26 17:04:37,588 [INFO] WRITING LOG OUTPUT TO C:\\Users\\shiwei\\.cellpose\\run.log\n",
      "5408\n"
     ]
    }
   ],
   "source": [
    "%run \"C:\\\\Users\\\\shiwei\\\\Documents\\\\ImageAnalysis3\\\\required_files\\\\Startup_py3.py\"\n",
    "sys.path.append(r\"C:\\\\Users\\\\shiwei\\\\Documents\")\n",
    "\n",
    "import ImageAnalysis3 as ia\n",
    "%matplotlib notebook\n",
    "\n",
    "from ImageAnalysis3 import *\n",
    "print(os.getpid())\n",
    "\n",
    "# library design specific tools\n",
    "from ImageAnalysis3.library_tools import LibraryDesigner as ld\n",
    "from ImageAnalysis3.library_tools import LibraryTools as lt\n",
    "# biopython imports\n",
    "from Bio import SeqIO\n",
    "from Bio.Seq import Seq\n",
    "from Bio.SeqRecord import SeqRecord\n",
    "from Bio.Blast.Applications import NcbiblastnCommandline\n",
    "from Bio.Blast import NCBIXML"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<a id='1'></a>\n",
    "# 1 Set folders"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Note: First copy releveant files to the new library folder\n",
    "icluding: common_pb_dict.pkl; blast_full_probes.fasta; chr_2_region_num.pkl"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "## Some folders\n",
    "# human genome\n",
    "reference_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl'\n",
    "genome_folder = os.path.join(reference_folder, 'Genome')\n",
    "# Library directories\n",
    "pool_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers'\n",
    "\n",
    "library_folder =r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_20_sequential'\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "from ImageAnalysis3 import library_tools"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# 5. Add sequential probes from subset of regions for control experiment (sequential)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 5.1 Generate new filtered pb dict for seleted 10 region"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### 5.11 Load all common probes to filter by genome count; subset 10 regions for further step"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "975\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "{b'CCAATGGCTCTAGCTAATTATTATTTTACCGTGGTATTCTCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_52',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 52,\n",
       "  'strand': '-',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.5139860484968,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCTACACAGTTTCACACCATAAGACTGATGGAGTGTAATGAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_177',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 177,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 74.26324583685204,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTGCACACACCAAGGGACCCGTTTATAGGATAGTTATATTTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_403',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 403,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 76.35069927644616,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTCTTTGTACAACCCTACAGGTATGCATAGTCACGCAACAGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_447',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 447,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.6440029684519,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATCCACTTTGACAACAAGGCTTACATTTCAGGAGGTGAGAGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_496',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 496,\n",
       "  'strand': '-',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.50706167489221,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCTTCATTTGTCATATGTTAACTCCGCTCCCTTCAAAGTTTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_582',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 582,\n",
       "  'strand': '-',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.71657668554138,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATAGACGAAACGTCAGTTTTGCTATCAGTGTGCCCGGGTCTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_669',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 669,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.23779389603277,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACTCAGTTAGCGCACAGCTTGGCTGGAGGATTTCACAGAAGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_714',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 714,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.6169753342977,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 12,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCATAAGATGCCACAATAGGGGACGCCAAGAGCGGTTGTTAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_1193',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 1193,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 80.95701030038435,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTACCAAGTACTAGTGTACCTTAGTCCGACAGCACCGGGTCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_1241',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 1241,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 80.73795693419203,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AATGACATCGACACGAAGAAGCCAGCAGACCTCGGGACCCGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_1285',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 1285,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 83.18259143006128,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTGTTTCCCTATGATTGATGTTTATCCTTGTGCCCCTCTTCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_1343',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 1343,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 76.49802864689451,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTGAGCCTGCTGGGTTGTCCGTTTCGAATTAAGTGAAGCAGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_1419',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 1419,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 80.03217178471095,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTGTTTCCATCGTTGTCTCTTTAACAATCACTTATTGTCCCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_1465',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 1465,\n",
       "  'strand': '-',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.54391655134776,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 1,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTTCCTTTACCGGAGACAATAAGAAAGCTTATTAGGCTGCCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_1515',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 1515,\n",
       "  'strand': '-',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 77.3936204620644,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGTTGGCTTTGGAACGGCTGATTCTCTCTTCAGAACCTGCCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_1563',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 1563,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 80.81478599204331,\n",
       "  'map_genome': 41,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCTGGGTGTTTGATTGCCTTTCTGGAGTCTTTTCCCTCCCAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_1614',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 1614,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 80.1039757475902,\n",
       "  'map_genome': 48,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCCTTTGTGGTCCTATGGATAAGTCAGCCTGGGGTCAGCTTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_1664',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 1664,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 81.98219527803002,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCAGGACCCAGAGGTTATAACGTCAAGATTCCCTTTTGGTAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_1708',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 1708,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.44534335510161,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCTGACTGGATCATCGGCCCTTCCTCAAATTTGGACTCACAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_1777',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 1777,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.65348542195011,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACAGCAGGACAAGGGGCTACAGGGTCAGAACAATGGCAAGGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_1822',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 1822,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 83.71420365018815,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATTTTAGCCAAATATCACAGTCAAGCCAGGCGTAAGCCTGTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_1866',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 1866,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 78.13578254293878,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTGAGTCTCCACCCCTGGTAACCCCCCTCAGTATTGGACAGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_1962',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 1962,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 83.79668633434068,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCCTCAGTAGTAGATGTTGAGAGCTCAATTTATGTTTAGCTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_2020',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 2020,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 73.35588044671749,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTCAGAATCTCCAGAGCGAATTTCAACTCCTACACATGGGCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_2090',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 2090,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.55926008127074,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGAAAGGCCACCTCTATACACACAGATCTAATTCATTCCTCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_2147',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 2147,\n",
       "  'strand': '-',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 75.73576120318518,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATGGACAACTGGTTCACCACTAGCAATGCAACCCCACCCTAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_2234',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 2234,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 82.10933589013803,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 1,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTATAGGTCTTATGTGGTAGATCAGCGGTATAGCTCTTGCTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_2283',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 2283,\n",
       "  'strand': '-',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 75.58151141236624,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 27},\n",
       " b'AGTTCCAAGTATGCTACTAAGCATAGATTGGTTTTCAGCTTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_2351',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 2351,\n",
       "  'strand': '-',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.82072448145374,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GATTTTATAACACATGGGTCCTTCTAGTCTTGTTTGCATGTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_2438',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 2438,\n",
       "  'strand': '-',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.28250581319537,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTTCCAAAAATTAACAGTGAGATTAGACCCCTTCTCACCTTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_2523',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 2523,\n",
       "  'strand': '-',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 74.03471645972581,\n",
       "  'map_genome': 48,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATAGATATGAGCCACTCTTCTCAGTGGAAGCAGTCATAATCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_2569',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 2569,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 74.14973744972463,\n",
       "  'map_genome': 44,\n",
       "  'map_transcriptome': 6,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTATGCCCCAGTTGGCCTTGGACCTGACAATTCTCAGCTTTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_2618',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 2618,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 80.64159784270453,\n",
       "  'map_genome': 49,\n",
       "  'map_transcriptome': 3,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGTGGGAACTCTTGCCCGGAGAGATTATTTAGCTGGGCCAAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_2769',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 2769,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 81.68089954498981,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTATCAGTCTATGTATTCCCCATGAGAATCCTGGAAGCTAAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_2833',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 2833,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 74.77128470950322,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTTAATGGTGTGTGATTGTACCAACTCCATTACAACTGTTCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_2877',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 2877,\n",
       "  'strand': '-',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.73559825882921,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATTCTTCCTAGAAAGTAACTCCCCGGCAGTGAAAACTTAGTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_3030',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 3030,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 76.08294000933063,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGGGTGAAGCAAGGCATGGAAAGAGATGTGCTTCACCCACTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_3096',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 3096,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 80.99163743387658,\n",
       "  'map_genome': 50,\n",
       "  'map_transcriptome': 9,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGAGAGCATCATTCTGTCCGTGGGAGCTCTCTGCTCAAAGGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_3152',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 3152,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 80.18998880442734,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGAAAGCAACAGATAGCCCTTATTACAGATCACCCAAAGTAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_3212',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 3212,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.28723479808309,\n",
       "  'map_genome': 26,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAACACAAGTTGGCTAGAGTTTGTCTAACAAGGTCTGAGCCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_3258',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 3258,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.35175813979606,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGAGGCTATGTATGAGCCATAGGCACCAATGAATGATCACT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_3318',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 3318,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.03181567407523,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGCAGTGTAGCCTGGATATCCTAGTGGTTAGAACTGAAAGCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_3385',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 3385,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.126267902498,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGGGGTACCAGCACTTGGTAAGTTACGGTGGGAGAATCATGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_3453',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 3453,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 81.37798435779314,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGCCCTATCTCATAAAGAATGTGGGGCTTTGGCTCAGTAAGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_3524',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 3524,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.9542817264113,\n",
       "  'map_genome': 41,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAAACTGATGAGCTCTCAGTCGATGAGGAGTTCAGTGAGAGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_3711',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 3711,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 75.45177723286872,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGGGTCACAACATTCACTCAGTTCTTGGAAGAATGGCGGCCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_3956',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 3956,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 80.70050607529993,\n",
       "  'map_genome': 47,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGCAGTGCTGTGGGATACGTGCGGCATGGCAACTCAGGGTGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_4005',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 4005,\n",
       "  'strand': '+',\n",
       "  'gc': 0.6190476190476191,\n",
       "  'tm': 86.23421388311732,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGATATGCTAATGGTTGAGCCTGATGGTGATGGCTAGTCAGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_4054',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 4054,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.19578878037134,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCAGGTTGATCCCAACACAGGTACACTGGAAAGAGAAGTGGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_4100',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 4100,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.99903351873513,\n",
       "  'map_genome': 48,\n",
       "  'map_transcriptome': 2,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCTACACTGTTTAACAGACCTTGTAACATTGGCAGTAGAAAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_4144',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 4144,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 74.1920890564067,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAACCATCAGCCAAGTCTAAAGCTAATGTGTGTCCTCTTCAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_4220',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 4220,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.57113285498127,\n",
       "  'map_genome': 43,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTTGTACCAGTACTTAATTCTAACAAGTTCCAAGAGAAGGGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_4350',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 4350,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 74.47700461911364,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGGAGGGAATAATGAGATGGGCAGGCAGCAAAAAGTTCCTA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_4410',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 4410,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 79.40424949982372,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGTCACAGCCTTCCATCGGTAGACATTACACCTGGTTCTTCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_4463',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 4463,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.71378412615275,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TACTATTATTTCCTGGGTACACACTCTTTGCCAGGTTCTGAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_4532',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 4532,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.24319367839121,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTTGACATAGCCCTCTAATGTGAGTATCACTATCATCCCTGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_4606',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 4606,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.17544234576775,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGAGAGAGTGAGAAAACTGCCTAGCCTGTCAAGTTCAGATTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_4672',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 4672,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 75.56512694791769,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATCCAACTGGAGGAAGCGGGAGTCGCTAACTCCAAGCTTCAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_4716',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 4716,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 81.69806778880411,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACATCAGGGTGTGGTTCCCATTATCCTCTGCAAAGCCTTGTA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_4761',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 4761,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 80.15336308925237,\n",
       "  'map_genome': 47,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCTGTGATCTGTGTTAAAAAGCCAATTCAGACCAGGCGGTGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_4807',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 4807,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.51188940791502,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTTTCTAATCAGGATGGAGACATGGTAGGATTTTGATTCCAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_5384',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 5384,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.62927494070846,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCACAGCAATATGGAAGGTATATGACTGGGGTTAGGGTGAGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_5430',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 5430,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 78.68979498796307,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTATATGCTCCCCTGATGTGAGGAACTTCGAGTAGTCGACCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_5474',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 5474,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 78.57421062667447,\n",
       "  'map_genome': 26,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTTGGCTGGGCCGAGGTGATCTTACTATAGTAGTAGTCACAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_5559',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 5559,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 78.31902218633081,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGCAGGCAGACACTCTCACAACTAGCTATATCTTTTCCTACT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_5827',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 5827,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.45634218932275,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTGGTCTGTATACCAGTGTGGTTAAATAGCAACATTTACAGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_5940',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 5940,\n",
       "  'strand': '-',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 74.13443465331687,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTTCAGGGCAATGAAAAAGTCAGATCGATGGTGGCAGTGGTA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_6001',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 6001,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.66466809328915,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGCCCCCCAGCCCCGGGAATGACCAGTCTACTTTGTGTCTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_6067',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 6067,\n",
       "  'strand': '+',\n",
       "  'gc': 0.6190476190476191,\n",
       "  'tm': 85.83044980355265,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGATACTTCATATAGGTAAACTTGAGAGCATGGACGAGATCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_6127',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 6127,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.10469455481234,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAGGCTGAGCAATGTTCCAGTGTGTGGGCATACCATATTTGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_6234',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 6234,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 79.02896917245727,\n",
       "  'map_genome': 44,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCTGTTGATGGGGGCTAGATTCTTGGCAACCTTTAGCTATTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_6287',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 6287,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 78.1521179687204,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATTTTGTTGTAGTCAAGCCAAGTGACATCTTAAGTTCCAGTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_6346',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 6346,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.58471163178194,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGCCCTCAGGTCTGTACTATAGCCTTACAGGCATTTTCTAAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_6439',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 6439,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.68233699813868,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGTCAAGAGCTCTCACCACCTCTTGGCCAAGAATGTAAGGAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_6525',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 6525,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.7218014009436,\n",
       "  'map_genome': 44,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTCCCTCTCCCCTGATCCCAAGGATGAAGGGGCAAGGGGTGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_6604',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 6604,\n",
       "  'strand': '+',\n",
       "  'gc': 0.6190476190476191,\n",
       "  'tm': 87.18595883861985,\n",
       "  'map_genome': 48,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAAATAACTGACAAATATCAAGCCTATGCCCAATAGTGCCCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_6682',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 6682,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 76.72657078970616,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAATGTCCTCACTATGCCATTTGCCCCGGGACAGCAAAGACT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_6726',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 6726,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 81.53214581101435,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATATGTAATGTTACAGATGCCTAGAGCAGGATGGTACATCCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_6797',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 6797,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.69569893664641,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATCTGTTTATTCTGCTTGGAGCCCTATGTGACAAGAAATTGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_6890',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 6890,\n",
       "  'strand': '-',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 74.61199576376339,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTTGCCTGCAGCCTCATCAACATGGGCTTTCAGTTTTAAGTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_7008',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 7008,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 77.95045474947051,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGAATGTGTAGGCAACATAAAGGTAGTGCATTAAAAATACCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_8187',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 8187,\n",
       "  'strand': '-',\n",
       "  'gc': 0.33333333333333337,\n",
       "  'tm': 73.33723203974296,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAGGCTAACGGTGCTTTGCATCTTTCACTTTACCTGCACATT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_8231',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 8231,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.77545779687773,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CATCTGCACCATGTCTCCGAAGGAAGGTGATAATCTGCTGTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_8293',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 8293,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 77.87282935636375,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AATACATGTACATTGATCATCCTGATCAGGGCGGAGTAGCCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_8483',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 8483,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 78.01901807454408,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGAGATTTTTGTAATGGATACATGCTGATTTGGGAACTGGAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_8553',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 8553,\n",
       "  'strand': '-',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.92238125809934,\n",
       "  'map_genome': 41,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AACAAGTAAACAAAGTCATGTATCCAGGTTCCAGTCACATGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_8696',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 8696,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.00008379389868,\n",
       "  'map_genome': 44,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTAAAGCATGCTGTGAAGCAATATATTAGGGTCCCGTGTCTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_8812',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 8812,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 76.54525772097566,\n",
       "  'map_genome': 41,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGGTGTATTTCTCCTTGCATCCATCCGGACTTATCAGGACAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_8869',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 8869,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.0719084736254,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTGTGTAAGATACTGGAGATCATCCAAGCTTCCACCCACAGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_8932',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 8932,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.34644237788001,\n",
       "  'map_genome': 51,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGAAAACGTTTTCCATGTACAGGAAGTACAGTTCTACTGATC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_8988',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 8988,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.04635264404271,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAATGCTGCAGGCAGAATCTCATGATGGACTCAGGCTTGCAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_9032',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 9032,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.97456794169108,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCCTGCTGCCACCCAACCCAAGTCCCATGCGCCCCCCACCCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_9092',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 9092,\n",
       "  'strand': '+',\n",
       "  'gc': 0.7380952380952381,\n",
       "  'tm': 94.40988071442985,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGAAGATGTATAGAGTTATTGACCACTCAACAGTTTTCCCTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_9279',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 9279,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.07977881202851,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGTCCTTATATAAAGCTACACAACTCACATAGAAGCAAGCAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_9342',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 9342,\n",
       "  'strand': '-',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.10026625978225,\n",
       "  'map_genome': 48,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGAAGGTGTGTTGGTCGTGAGGTCCTGACCTCACTCTGATCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_9406',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 9406,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 80.61403919603163,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTTGTTTGCAGGACAGTGGGTGTGTGCGAAAGAGGGGAGTGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_9492',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 9492,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 82.76325328186459,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCCAGATGCCCACCCAAGCATGCGCCGTCCTCCCGCATCTGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_9576',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 9576,\n",
       "  'strand': '-',\n",
       "  'gc': 0.6904761904761905,\n",
       "  'tm': 89.30679592676375,\n",
       "  'map_genome': 41,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATCGTTCAAGCACAGCTAGCCTTCATGACCAGAGACAGTCTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_9906',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 9906,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 78.01205592882218,\n",
       "  'map_genome': 48,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAACCCCGGTGGGATATGGTGGGTGTGAATTTCCACCTACAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_10314',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 10314,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 82.78465364962938,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAGTTAGAAAACTTTACTGCAGGGGTTTAGGCCTTATCCAGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_10430',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 10430,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 77.08293937297634,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGTTTTGTGGTTAAATCCAGGACTTTCTAAATGGAGGAGCTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_10478',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 10478,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.51330522898417,\n",
       "  'map_genome': 44,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTGGGCAGGGCAGCTGAAGTCTGATCCAGCCTGGTGGTACAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_10531',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 10531,\n",
       "  'strand': '+',\n",
       "  'gc': 0.6190476190476191,\n",
       "  'tm': 84.92768969158703,\n",
       "  'map_genome': 50,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGAAGCCAGTGGGCTAGAAGTTCGAAGATATCCTCAGTTACA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_10599',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 10599,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.00261514799479,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CACACATATATCCTTATGTAGTCTGTGTAGCCTCCAGCTCCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_10643',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 10643,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.7544104716697,\n",
       "  'map_genome': 48,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTGTGTGTATACGGAGCTTGGACTCTGGCTTGACCCAAAATG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_10718',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 10718,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.01882910528553,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGCATTTAGTCACTAACAGTAAGCATATGTTATGGGTTTTGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_10818',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 10818,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.4998568312908,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTGTTCAACCACAGCTATGACCTGGCTTATCACTGAAACCCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_10987',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 10987,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 78.51729354225188,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTTTAAAAGCCCATTTACTCTTCTTCGGCATCCCCCTAAACC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_11114',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 11114,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 78.23079706108723,\n",
       "  'map_genome': 44,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTTTGGCAGGCCCTCTTCAATTCACCACCCAGAACTATAGTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_11158',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 11158,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 79.4366661027941,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGGAGCTGACACAGAATGGGCCCTTGCAGAGCAGCTGTGTGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_11254',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 11254,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5952380952380952,\n",
       "  'tm': 84.04378031667403,\n",
       "  'map_genome': 45,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGGGGAGGATGGGGCATTATGGAATACTGGCGCTTCTAATGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_11326',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 11326,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 82.02067504353988,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATGAATGACTCTGTGTCCGGAAATCGATAAAATATGCAGAGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_11397',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 11397,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.80544832411903,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCCTGGCTGCCAAGATTCCCTTCATAACTCCCCGTATTAAGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_11444',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 11444,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 80.96864048169567,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCCCTGAGGGTACTGAGGGGCTGCTGTACTTAAATCATTGAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_11532',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 11532,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 79.79725982443608,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCTGAGCATGTGAATGTTGGTATCTGTGGACATCTTGGAACC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_11576',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 11576,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.48464913384572,\n",
       "  'map_genome': 48,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGATATTGGCATGTTATGAAAACCATGGCATGCCATCTGCTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_11705',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 11705,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 77.06821853712171,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCAGTGGAGATTCAGGGTTAGGAATGTCATACAATACTTTGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_11775',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 11775,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 74.81695505657154,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGATTTGTGTTGGCTGACCGACTCCTGGGGGAGGCCCACTGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_11927',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 11927,\n",
       "  'strand': '+',\n",
       "  'gc': 0.6190476190476191,\n",
       "  'tm': 86.6371095413308,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTCCTCATGGTTGTCAATTCTGGGTGGCCTCTGGGCTAGTGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_12003',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 12003,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 82.33529087586447,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTCTGGACTAGAGCTTGTGCATGCCGCACAGTCTCTATAGAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_12080',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 12080,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 78.18736594180854,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CATATTCACCACTACTCCCATTCCTAACTCATCCAGTTTCAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_12241',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 12241,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 74.80854092212161,\n",
       "  'map_genome': 48,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACCCATAGCACGAAGGCAGTACAAAGCTGACTTAGTAGGTTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_12298',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 12298,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.57640107317695,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTGAAGCATGGGTGATAGCATTGAAGAAAACACGCTCTTCCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_12353',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 12353,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.95073792562044,\n",
       "  'map_genome': 51,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAAGCCATCAACTGTCGATGGCACTTCGGCAATGGGGGAGGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_12401',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 12401,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 83.73449585492517,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGTCCTATCCAGAAAGGGCACTGCTTACTCTCGTCCTCCCTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_12512',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 12512,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 82.12757423883647,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCCAGGATCCCAGTTACTGATGTACTGTTTATGGTTGAGCTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_12605',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 12605,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.7200364983745,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCAGATGATGGTAATGCAACAACCACCTTAGCCTTTAGCACC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_12649',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 12649,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.7410005572309,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACCCTGGTCCAGGCTCACCTCGCTGTGGCCATTTATTCATTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_12709',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 12709,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 82.86413147204473,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTCCCTAGTGGACATTAGGTGGGTGCGGTCCCCAGGCGCTGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_12758',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 12758,\n",
       "  'strand': '+',\n",
       "  'gc': 0.6428571428571429,\n",
       "  'tm': 88.5446047996366,\n",
       "  'map_genome': 26,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCAGGTGGTGATCGAGTCCGACCTGTACCCGCCACCAAGGCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_12837',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 12837,\n",
       "  'strand': '+',\n",
       "  'gc': 0.6666666666666667,\n",
       "  'tm': 87.73465612665115,\n",
       "  'map_genome': 26,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGGAGCTACTGCCTCAACGCTGTGAGCGCAGGGACACAGGTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_12881',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 12881,\n",
       "  'strand': '+',\n",
       "  'gc': 0.6190476190476191,\n",
       "  'tm': 84.87418553133108,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGGCCACCCGGGGCGCATCCCACCAAAACGCCCTCCAGACCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_12961',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 12961,\n",
       "  'strand': '+',\n",
       "  'gc': 0.6904761904761905,\n",
       "  'tm': 92.57359817835442,\n",
       "  'map_genome': 26,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TAGGACACCTATAAGCTGTTACTGCGCTTTTGCTACATCACT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_13048',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 13048,\n",
       "  'strand': '-',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.81345269100012,\n",
       "  'map_genome': 26,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCTGGCAGACTTATTATTGAACATCCGACTTTGGGGGGTGGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_13092',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 13092,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 81.70515573169712,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAAGGTACCTGCTATTGGATCTGACCTACAGCAACAGGAGCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_13138',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 13138,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 79.28507853419643,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTTCACCTACACTGTTGCCACCTCCTTAGGTGCAGTGTGAGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_13187',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 13187,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 80.43949894160284,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AACAGCGACAGCTACCTTTGCAGCACGTGGCACACAATCATC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_13232',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 13232,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 80.52437834882619,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTGCTTCTTCCTAGTCCCATGGCTCTTGCTCAGGGCACAATG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_13288',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 13288,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 81.27562961286503,\n",
       "  'map_genome': 51,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TACACCTCCATCAGCCCTACACACGTGCGCACATACGCGCGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_13397',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 13397,\n",
       "  'strand': '-',\n",
       "  'gc': 0.6190476190476191,\n",
       "  'tm': 85.42315277552638,\n",
       "  'map_genome': 41,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACCACAGTTATAACAACCTAGCTGTACCTTTGGGGTAGGGTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_13477',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 13477,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 79.93741192246222,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCAGTCCATGGGAAAGCAGGTGCAGGTCTATCTCTGTAAGTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_13585',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 13585,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.53887340555457,\n",
       "  'map_genome': 43,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTGGAAACAGTGATGTTACCTTGGATACTAGTTGTTTGGATT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_13737',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 13737,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.98727928489444,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGATTCAGCATCTAGCTTACTTGACCTTTCTCGTAGTATTTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_13818',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 13818,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.23466551186306,\n",
       "  'map_genome': 51,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCATTTTTTTCTGTTGTGCCAACAACGTAAAGTTACTCATGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_13880',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 13880,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.38834425036373,\n",
       "  'map_genome': 47,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTTTAAGGTCAGCTTCGGGAAGATAGCCTACCTCAAGAACAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_14005',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 14005,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.35268815262089,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGTCTGGAGATTGGACTTAGGCCCTGATATTTGTATCACAAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_14091',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 14091,\n",
       "  'strand': '-',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.51086390265942,\n",
       "  'map_genome': 41,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACATAGAAAGCTAGGCGTGATTCCAACCCAGTACTGGAGCTA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_14135',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 14135,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 79.31150277489081,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACTCCCTGGCCAGCCTAACATGCAAGCTTCAGATCTCAGTGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_14209',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 14209,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 81.86291689604025,\n",
       "  'map_genome': 43,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATTGTTGTTTGTTCTAATTGTTGCCTTTCCTAGGCATCATAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_14728',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 14728,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.48982186383932,\n",
       "  'map_genome': 44,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTGTACCTTCGATCCTAAGTGCAGGGTGTGACCAAAGAAACT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_14772',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 14772,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 78.17260693738757,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGATAAAACTAAGCGACATTTTCAGCCAATGTCGATTGCATA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_14816',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 14816,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.56189419084325,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAAAGCCACAGTCTGACTTGCAAATTAAGAGGAGATAGATTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_14883',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 14883,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.04280121781147,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGCATCAGGAAAGCCCCCTAGCTGACTCTTGTATCCACAATA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_15071',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 15071,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 79.58763968174259,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTTTAATTCAGAAGAACCCTGGTCCACTAATAGATCCCAGGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_15115',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 15115,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.99795977535473,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCATTTCTGATATGGCACTGCTCTTGCAGCTACAGGGTCAAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_15159',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 15159,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.05326775574008,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCAGGGGGTTAGAAAGAAGTTAATGTAGTGGTTTCCAAAATG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_15211',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 15211,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.55354572542495,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCAGGAGCAGCACCTGGAACTATGTTGGATGTTGGAAAAAAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_15258',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 15258,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 78.33741257696835,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGATCGATTGACATGCCTTAGTTCTGGAATTCCTAGTCACAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_15310',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 15310,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 74.92810303803327,\n",
       "  'map_genome': 26,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CACACACCCCTCAACCGTGCTGCAAGGCAGATCGTGCTTTTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_15368',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 15368,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 82.48785342493409,\n",
       "  'map_genome': 26,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCTTCCATTTGACCTACTGCCATCACCCTCTGCCAGATCACA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_15414',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 15414,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 80.1700248385066,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAACCAATGCTGTAGAAATTGTTTGACTCATGCAAGGTGACA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_15464',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 15464,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 74.77430483434671,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AACTAGACAGCCAGGAGATACAGTGCTGACCATTACATCAGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_15545',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 15545,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.65465513870726,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCCAACCATAGTAATGCACTCATGGTCCTAGCTCTTGGGATG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_15669',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 15669,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.24702068418122,\n",
       "  'map_genome': 41,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTTTCACAACTTTGCAGGTAGTTCGTGTCCTCTCATACTGCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_15713',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 15713,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.47830588662703,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAACCTGAGTAGAACATCCCTGCATGAGCTGACAGTCACATG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_15814',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 15814,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.3996432534774,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAATAGATTCAGCTGGGCATTGATTCAATTCCCTCAGAACTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_15868',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 15868,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 74.72899450706285,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATCTGATAGATTCCAGGATGGGTTGGAGGAGTGTTGCTGAGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_15913',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 15913,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.1481950453636,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACCTTGATGCTCTCCAAGGAAAGGCTAGGCACCAGTCCATGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_15964',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 15964,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 82.53336948625576,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAGCAATCCCCTAGATCTGATTGATGTCCCTAGAGACATGCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_16038',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 16038,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.53590779462871,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGGCTCTTGGATGCTACAGACATGTCAGGTAGGAGAGAGGTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_16100',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 16100,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 79.50215590658428,\n",
       "  'map_genome': 47,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAGATCACAGGGATACTATAGGTAGAACCAACTAAGACATTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_16152',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 16152,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 73.29402991277857,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGGCCTGGAAAACTAACCATTTATAGCCCTTTTTCCCAGAGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_16235',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 16235,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 79.31601675526366,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTTTAGCTTATTTTGATAAGCTGGTTCTGAAGTGCCCTAGTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_16279',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 16279,\n",
       "  'strand': '-',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 74.07217768822483,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGAAAGTCTCCAGCTTCGTCCCTAATGCTTCTGTATCCTTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_16358',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 16358,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 76.76611980181457,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 1,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGGGAAGCTTGAGGTAGGTTACAAAGTGGTAAGGCAAGAGCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_16442',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 16442,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 79.91794159066694,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATATGTGCACAATGACAGCGAAGGGCCTCAGCCTCTGGGGCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_16507',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 16507,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5952380952380952,\n",
       "  'tm': 85.21057164096493,\n",
       "  'map_genome': 41,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCTGCCCTTCAAGCCATGATGGCTAACCTCCTGGGACACCCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_16647',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 16647,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5952380952380952,\n",
       "  'tm': 85.85552459859366,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCTAACCCTGCAGCTCGGGGTTATTCTACCTCGCGTAACCCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_16700',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 16700,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 84.34912331164242,\n",
       "  'map_genome': 26,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGAAGTAACACTAGTGAAGGCTCAGGGCAACATAATGGACT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_16750',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 16750,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.00524835956799,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACCTTTCAAAGACCCGAGGACACTTGATTAAGGGACAGCATA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_16839',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 16839,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 78.35876048393459,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTTACTAGCTAGTAAAGGTTTCTAGGGCTAGTAACCATAAAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_17019',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 17019,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.62144543660878,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 27},\n",
       " b'TCGTTTGATATATGAGTTCTCAGCCACATCTGATGTTCTAGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_17092',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 17092,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 73.5968745215269,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAAGGCTACGCCTCCTCAGCCTACCCAAACAGATGTACCAAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_17202',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 17202,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 81.8004357640803,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGAACCAAATATTCAACTGCCTGAAGATATAGGAAACATCTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_17246',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 17246,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.04258484335202,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CACATCTGCCTTCAGATTAATGGCTATGTATCACCAATGCCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_17299',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 17299,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.4025880437361,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCTCTCCCTTTATGGTGACAGTAGGGACTAAATGAGGGTGTA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_17381',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 17381,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.81867419939891,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGACCCATTTAAGATGTTAGACTCTACAATGCTCAACCCCTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_17454',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 17454,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.88498217514945,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACTGCATTGTTTAAGCTAGGGCAACAAGGAAATTAAATATGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_17619',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 17619,\n",
       "  'strand': '+',\n",
       "  'gc': 0.33333333333333337,\n",
       "  'tm': 73.48236140816573,\n",
       "  'map_genome': 45,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGAGAGTCTCATGATCTGTGGTATGCCTTGGACTCACTTGCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_17711',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 17711,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.57594125352585,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CATGGATGTGTGCTACCTTGCTTCATGCAAGTATTACCATGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_17801',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 17801,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.99206733661856,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTCCTGTCCTGAGCAAGTTGCCTACTACTGGACCAAACACCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_18000',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 18000,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 81.26338092201416,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGTACTTTTTAACTATGAATGTGGGGCCTACAGGTAATGAGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_18066',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 18066,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.96779071266081,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGTGCCAGGCGTTGAGCTGGGCATAAACTGCACAATGATGGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_18111',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 18111,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 82.50757933907511,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAAAATTCCCCTTTAACCTTGCCAAGATCAAACCACAATAAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_18195',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 18195,\n",
       "  'strand': '-',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 74.5269928719627,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATGGTTTTACCCAAAGACAGATGCTACTAAATTCAGTTGGTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_18239',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 18239,\n",
       "  'strand': '-',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 74.29991737028269,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AACTCCTGGAGCAGGTTCTTCACTTAACCAACACCACTGGCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_18311',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 18311,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 80.74342651842784,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACCAATGAATCAACTGGTTCTCACTTAACTAGTAAGTTGCTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_18391',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 18391,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.13352217821739,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AACCCAGCCAGCCGGCAAGCCACTGGATATGTGTAACCATTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_18435',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 18435,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 83.56993255743379,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGGTTGGGTTTGGGCATACCCAGTCCACTCTAAAGTTGCCTA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_18506',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 18506,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 82.88388358772409,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGATTTGCATTGCAGGTGCTTTTAGCAACCAAGTCATCTTCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_18596',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 18596,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.64967771499033,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTCCCTCCCTCACTTAGATCTCATCCCTGTCTCATATTCTCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_18685',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 18685,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 76.80185008855466,\n",
       "  'map_genome': 45,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAAGGGGGAGACCAGCAGGAAGAGCGAGGACTAGGGTTGGAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_18782',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 18782,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5952380952380952,\n",
       "  'tm': 84.62264399568954,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAGGGGTAGGGTGGGTTTGAGCAAAGTACACAAGGAAATGCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_18827',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 18827,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 81.68783504040181,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACCCATTCTTTTAACGAATTTCTTGTGAATTGAGGTCTAAGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_18880',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 18880,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.04269272928389,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGACTTAATTTTTTCCGGCAGTTTGAGGCTCCCAGCAAAACT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_18998',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 18998,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 78.12924107875938,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGCTCAACAGCTATGAGAAGTATGCCGCTGTGGGATGTTCAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_19104',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 19104,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 78.9197314001575,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGAGAGTAAGGGGTGACCCCACTTGGCAGTTAGCATTGCCTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_19174',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 19174,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 82.8101127990476,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGAAATTCCTTTCTCGGCTGGATAGCTAGCAAGATAATTGAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_19257',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 19257,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 74.35002010529041,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAACAGAAAAGACATTACTCAGTTGTACTCAGAGTTCTGCCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_19313',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 19313,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 73.4124709664199,\n",
       "  'map_genome': 49,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAAATCTGTGTGGGAATCGAGCATATGGGAAGTCGTTGGTAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_19389',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 19389,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.53821466840111,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TAAGCCAGATTAATTGTATGCACTGTGCTCAAAACTTCTTGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_19437',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 19437,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.38629564587387,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTCCTTTGCCCTGAGCTGAGGATTGTTAAAAAAGCAACACTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_19509',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 19509,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.02456892530944,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTTCTTCGCTCTTGGAAATAGCCACTCTGTATACTCAGGATA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_19553',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 19553,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 75.25533135304357,\n",
       "  'map_genome': 46,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCTGTGACTTGCTAGCATCTCATGCATGGGGGAAGACACAAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_19649',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 19649,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.52412029579222,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATCTGGCCAGATAGCAATGCCCAATTAATTCCCAGGTGCAGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_19693',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 19693,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 81.20780709059352,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTTGCAATACAGCCTCTCTTAGTGCTAGAGAACGTTACATCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_19747',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 19747,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 75.63450476252245,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAAAACACATGGCACCGCAACATTTGCTTATGGCAGACTCGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_19791',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 19791,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.78480128791102,\n",
       "  'map_genome': 26,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTGGAGAGGGGTCTCCCCTTGCCTTGCATTTGGATTCATTCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_19836',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 19836,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 81.61800229186991,\n",
       "  'map_genome': 44,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTTCTTGTGGGACACTAATATCATCATAGATGCTCCACCTAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_19926',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 19926,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 74.50924307006761,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TAAATACCATCACACTACAGGATAGGTAGGGCCTTAACAAAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_19997',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 19997,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 75.67546976775361,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTTTAAGTCAGGCACAAACATTCACTCCATAACAACCAGTCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_20041',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 20041,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.09190112915547,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGGGTACTTCTGGGAGCACCCTTTATGGGTCAGGTTCACTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_20091',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 20091,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 81.89896843320639,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCCTCAAGACTGGTATTGATCTCTAATATACATCTTACCCTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_20147',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 20147,\n",
       "  'strand': '-',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.48208183207237,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCTGGATCTTAGTTCAGTTACTCCCTCCACCTAAGCTACCTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_20193',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 20193,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 78.52447253921594,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACTCTTTCATGTGTAGGCTCTAGGGTTTAAGTTGCCCACACA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_20239',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 20239,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 78.43699032008493,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCTTCCTCACTCCATTAGGGTGTGTGAACTATCCCAGAAAGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_20322',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 20322,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 78.94154183151278,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGACTCTGGGCAAGGTGGCCTGCGGCAGAAAGACACTTTCCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_20367',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 20367,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 84.3259632873768,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCTCTGCCCATCCTGAAGTTGAACAGTTTGTCCTGTCTTAAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_20411',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 20411,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.45904547051123,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGAAAAAGTGGAGGTAAATATGGGGACATGCCCCCTAATCCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_20455',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 20455,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 80.80542262221502,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACTCTCTTGGGTGAGAGGAAGCGCCACACTCATCCCTAGAAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_20504',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 20504,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 80.94283034822553,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTGCTAGAGCCTGGTATGTCTGAGATGATTCCCACTCATCAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_20548',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 20548,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 77.44759920431443,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 3,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACAAGGATGAGGGGCTGGCCCTGAATGCTTCTGTCATCTCTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_20616',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 20616,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 81.65650995602277,\n",
       "  'map_genome': 49,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACAAATTAGACAGATATGGGTATTCATGCTCTGGACAGGAGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_20661',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 20661,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.1281112860346,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTGGGATCAGGAATCCCAGGCTACTAGATTCCTTTTGTATAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_20744',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 20744,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.81282146586148,\n",
       "  'map_genome': 49,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCCAGAGGAGGTTTAGATTGTGCCCTTGGCAGATAAGCAGCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_20788',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 20788,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 81.47663075297612,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CGCTCTTTCCAGGCCCATGGAAGTGCGGATTGTGCCTCTTTA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_20879',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 20879,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 82.80609066357727,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCTATGCCCCATCACTCATCCGATGTTCGTTGTGCAGAGGCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_20923',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 20923,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 81.85641436298619,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCTTAGTCCTATCTCATCCATGTCAGCACTGAAACATATTTA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_20991',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 20991,\n",
       "  'strand': '-',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.22400986884634,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATAGCTCCTTAGCTCTCGAAGGGAATGCCTGGTTCCTGCCCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_21035',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 21035,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 83.92979648369868,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCTGTTAGCTGAACCATTGGCATGATAAGGAATAGCTAGGGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_21079',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 21079,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.86850913049369,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCCAGGTCAAGGACATCATTATTTACCCCTGTCCTGGGGTGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_21162',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 21162,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 82.30743213643404,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCTGTTCCAAGCCTCATCAAAATGTTAAGTCACCACATTTCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_21222',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 21222,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.21000037430451,\n",
       "  'map_genome': 47,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGCAACTGCTGGACTTAGCCCTGATTAGGCCGTCAGTTAAGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_21295',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 21295,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 80.69558311291956,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGAGGAAGGATCAGCCCAGATGCTCTCACTTGTCATGCAGTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_21355',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 21355,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.3647526723467,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAACGTGAGAAGGCATGATTTATCACGTAGTCTATATAGTGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_21445',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 21445,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.19570925760962,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGAGACACTGGGTCAGACAAAGGTGAACATGGTCACCTTTAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_21490',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 21490,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.42681495834438,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGAAAGAGCAGAGTCGGCCTGAGTCTTCCATGCTGAGGTGTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_21569',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 21569,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 80.30880272209168,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAACAAGGTCTGAGTGGACCATGATTTGGAAGGGGCAAGAAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_21670',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 21670,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 78.950434845433,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCTTCAGCCCTGCTGCCCACACCAGCATGAGCCTTTCCATTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_21714',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 21714,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 83.73139933742829,\n",
       "  'map_genome': 49,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGAAGGGAAAGGGTGACAGCTCTCTTGGAAGGTTGAATGGCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_21792',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 21792,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 81.26054107344271,\n",
       "  'map_genome': 47,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGTCCTGGGACTCTAAGAGTCACTAGCCTACAGACATGTGTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_21838',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 21838,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 78.76860640675403,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCAGCAATCCCAGGTAATGTCCGACCTGCTAAATCAGACACT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_21905',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 21905,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 79.0363970915704,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCCTCCCCCAGTCAGTGTCAGACATGATGGAGGCCTTCTGTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_21962',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 21962,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 82.57584546842577,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCATTTATTGGAGCCATTTATAAGAGGGCTCTCTAAAGGTAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_22026',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 22026,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.0318986297234,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAATGGGAATGTGGTGCCATCTCAATTCTTTCCCCGTTTCCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_22070',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 22070,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 79.5825051435034,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATTCCTCCAGTGGCTAGGATCTCGGTGAAAGCTCCAGCAGGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_22122',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 22122,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 83.1274594280573,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 1,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAGGTGGCAGCCAAGGGCTCAGAACATGGAGCTGTTCTGGGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_22167',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 22167,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5952380952380952,\n",
       "  'tm': 84.60043330356046,\n",
       "  'map_genome': 50,\n",
       "  'map_transcriptome': 1,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATGGATGGGCAGAGAGCCACGTAGGCAGCACAGCAGGAAAGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_22225',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 22225,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5952380952380952,\n",
       "  'tm': 84.50946177180344,\n",
       "  'map_genome': 43,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGAGGAACAGGTGGCCAATTCTTCACTCACTTATTATGCACA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_22281',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 22281,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.8827709564087,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGCAAGCAGTTGATGTGAGATAGCTGAACAGAGGATTTCAGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_22362',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 22362,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 76.79278844874625,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATGGTTTCCACCTCATTAACACTGCTCTTGGGGCTTTGTTTA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_22427',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 22427,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 78.18986356513915,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATCAGGAGTCATATCCAGAACCTCTGTAGATGGTGTCAGGAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_22483',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 22483,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.80195419825884,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGAATTCTTTTAGGGACCCTCACCCACAGGTAAAAAAGTGTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_22532',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 22532,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 77.2556651964075,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTTTCAGGCCAGGTATGCAGCTCGGAGGTCCAGTACCTGGCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_22582',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 22582,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 84.65820175949636,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGATCTTATGGAAGCTTATTCTCAGCTAAGGTTCCCCCCTTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_22912',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 22912,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 77.66608073193697,\n",
       "  'map_genome': 50,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAAACTCACCAGCATATAAAGCTTAACTCAAACTGCTGACAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_22985',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 22985,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.40740723178646,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGAGAAACATCTAATTCAGTGACTGGGAAGCGGCAAGCTCAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_23031',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 23031,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.66104228679228,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAAGAAGAGAGGTTAATTTGGGTGTAATCCAGTGCCAATGAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_23145',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 23145,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 75.31294963325445,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TATGAGAGATATATGCAAGCTGGCTTTTATGACAGACCCATT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_23266',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 23266,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 74.54626554376966,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGTAGAACCTTTTAGATTAAGGTTTTGCCTCATGAAAGGATT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_23340',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 23340,\n",
       "  'strand': '-',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.53850071008964,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTTTGGATGCTGGGACTGGAATCAAGGCTTGCATGTGTGAGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_23401',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 23401,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 80.24741904239153,\n",
       "  'map_genome': 47,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AATCAAACAAGGGCTGCAGATACAACTCAGTGGAAGAGTATT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_23445',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 23445,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.4788035797439,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTCCAAGCTAGCTTGGATGATGCTATAAGAAAACCAAACATC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_23497',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 23497,\n",
       "  'strand': '-',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 74.17150267780988,\n",
       "  'map_genome': 51,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGCTGAGAATACAAGTGTGGTCCCTGATCTTTTCAGATCTCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_23568',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 23568,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.21625621731556,\n",
       "  'map_genome': 48,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGTGTCTGCTTTTCAACAACTCATCTTGATTCCTAATCTCCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_23612',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 23612,\n",
       "  'strand': '-',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.6791596795253,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCCTGAAAGTTCCCACTTTAACTCCCAGGGTTTTACTTCCTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_23679',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 23679,\n",
       "  'strand': '-',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 78.0889896690087,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGTCAGAGGGGCGGTGGCTGTCGTGGACTGGTCAGGTAGACC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_23734',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 23734,\n",
       "  'strand': '+',\n",
       "  'gc': 0.6428571428571429,\n",
       "  'tm': 86.54317443716275,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTTGATAAAAGTTAAGCTTTCTAGGCTACAGTGCTGCATGGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_23794',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 23794,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.12514440831467,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCCCCCACCTCACCTGCCTATCTCTATACTAGAGATCAAATC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_23857',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 23857,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.99612008479505,\n",
       "  'map_genome': 46,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TAGAAATCACTGAGTTATGAGGTGGTAGATCAGCGGTAGAGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_23924',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 23924,\n",
       "  'strand': '-',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 75.60847379758962,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 27},\n",
       " b'ATGGTCATCTCTTTGTACCCCATCTTAGATGAACCCCAGAGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_24031',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 24031,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.88627477774804,\n",
       "  'map_genome': 43,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGTGACCCTCCAGCTATGCAGGACATGGACATAGGATGGAGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_24088',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 24088,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 82.0414463648138,\n",
       "  'map_genome': 43,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCACTCTCCTCAAAGTTACAACTTAAGTCTGGCTCAGTGCAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_24152',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 24152,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 76.71897906469815,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCTGGTTTTCATTTAGATCCAGACACCTCAAGAAGATAGCAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_24226',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 24226,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 73.9756949334095,\n",
       "  'map_genome': 44,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTCCTGTGGCGTCCCTGGCATCCTACGAAAAATGTGGACCAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_24326',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 24326,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 82.84751181727637,\n",
       "  'map_genome': 26,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGCCACCAACAGCGCCTTTGAAGCCGACTAAACTCGACATTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_24395',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 24395,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 80.83912132932488,\n",
       "  'map_genome': 26,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGGGCACCTTGCATTGGTCAAGGTTCGGAGGAAGATAGAAGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_24439',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 24439,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 80.79648002471106,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCCTCCCCTCTGCTGGTTGCTGGTCCAAACACATCATCATTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_24503',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 24503,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 81.70499308719707,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TATACTCTGACATGGGGCATGGAAAGTAACATCCTCAGAAGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_24547',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 24547,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.28150409262622,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTGCTAAAGCCATTGGTCTTAAAGTCGTATTGGTCAATTACA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_24610',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 24610,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 74.38639858909232,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAAAAGCAAACTTTCCTGGCCTGTTTAAAAGCCTCCAAGGAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_24686',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 24686,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 77.85930832210596,\n",
       "  'map_genome': 47,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGAGTTTTCCCAAAACGTATGGTTTCTGGTGTAAATGTAAAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_24757',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 24757,\n",
       "  'strand': '+',\n",
       "  'gc': 0.33333333333333337,\n",
       "  'tm': 73.57986730719693,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTACTCAGAACACAACAGTTCTTACTTTTCCCATCCCATGTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_24812',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 24812,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 75.01812763923652,\n",
       "  'map_genome': 51,\n",
       "  'map_transcriptome': 1,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCAATCTGCTTTGCTAATCATGGGCATGACTGCATGAGCTCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_24926',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 24926,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.2585433090847,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGGTGCATTCTTGTCTATAGAAAAGCACTTAAAATCCCAATG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_24979',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 24979,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.1085642164598,\n",
       "  'map_genome': 50,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTGTGATGTGGTGCAATTGACAAGCTTTGTATAGTGACTTTA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_25042',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 25042,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.45458927458685,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGCATTCTCCCATCATTGTCTCTTCTCACCATTACAAACCCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_25092',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 25092,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.70487488305258,\n",
       "  'map_genome': 51,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTGGATGGCAGTATGTGACTTAACAGCAGCTTGTGGGACTGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_25186',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 25186,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.38682625392084,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATCATTGTGTATTAGATAATTAAGACGGGTGCAGAGAACAGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_25330',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 25330,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.61631185185513,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGGATCACATGTTTGGGACATGCCTGGACAGCCTAGCAATAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_25402',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 25402,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 79.85420508837404,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGAGAGAGATTATGCATAGATGAAACCAGAAGAAATCGCCAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_25496',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 25496,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 74.23325017497632,\n",
       "  'map_genome': 48,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TAATGGTAATCAAAGCTATCATTTGTTAAATGCCTACCTGCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_25622',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 25622,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 74.04681370570194,\n",
       "  'map_genome': 43,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTGCTGAGCACAGTGTTGAACACCACGCATGAATTAAGCTAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_25666',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 25666,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.99612820157256,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAGTGATTAGATTGAAGTCGCCTAACTCTAGGTGTCTAACTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_25773',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 25773,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.54971705517107,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGGAAATCAGCAAAACGTCTTCCTCTGCAATGGCCTGGGAAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_25867',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 25867,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 80.20272264924591,\n",
       "  'map_genome': 41,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCTACCTGGAGCACAGCGTGGAGATGAGGTATCAGAGTGCAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_25911',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 25911,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 81.07248040673647,\n",
       "  'map_genome': 44,\n",
       "  'map_transcriptome': 4,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTGCCTGTGAAGTGGTCGGTAACATAGACCTTATATATTTCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_25970',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 25970,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 74.91925715175267,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTGTGGCTTAAATTCTGCCCCCCGGAAGTCTTGTTCCCTAAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_26015',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 26015,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 81.28372441574135,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGAAACTCCGTGGTAAACTCTACCCTAAGGTTATGTTGAGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_26094',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 26094,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.0146609476808,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGGTGTTGGCACAAGCACGAGTTTGAACTGTTTGGGTATCAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_26142',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 26142,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.72719534104209,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCTCTGGATAAACTGAAACATGGCGGTAAGTCAAACCCAGAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_26228',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 26228,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.02503376582922,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGCATCTGACCCTCATGTCCAAACATAGTCTGGACACTTGGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_26324',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 26324,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.3132452089049,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAGTGCTTTGTCCCAGTGAACCATCTAATGTCATATACAATA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_26370',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 26370,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.50249782999816,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TAGGGATAAAACATGTCACAGAGTCTCCAACGATTGAATTTA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_26436',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 26436,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.09637861702493,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCAAACCTTCTCTCATTGATCGAGGGAGCCTCAGACACAATC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_26666',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 26666,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 77.67414129742792,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCCTCACTGCTCCCTGAAATGAGAGGGAATCCAGCATGCTCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_26734',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 26734,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 81.45761777865357,\n",
       "  'map_genome': 41,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCCTGAGGCAATGTCTAGGTCTGATATCACAGGCCCTGCCTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_26781',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 26781,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 81.6567238883739,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTCTGCTGAAGGGACTGACTGAATGAGGTATAATCCCAGGGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_26839',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 26839,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 79.78459277583175,\n",
       "  'map_genome': 44,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCCTCCCGCCTGCTGCTAAGGCGTTTCCGTAGCACCCATAAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_26883',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 26883,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5952380952380952,\n",
       "  'tm': 86.1070669725159,\n",
       "  'map_genome': 26,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCAGAGAAGCCAGCAGGCCGAGGAGGGTCACTGGGACCCTGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_26927',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 26927,\n",
       "  'strand': '-',\n",
       "  'gc': 0.6904761904761905,\n",
       "  'tm': 88.71636955595613,\n",
       "  'map_genome': 43,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGTGCCCACCTCATGCCTCCTTGCTGGAGACTGCCCCCCCCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_26995',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 26995,\n",
       "  'strand': '+',\n",
       "  'gc': 0.6904761904761905,\n",
       "  'tm': 91.28049243741214,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGATGCCAAGGCTGACTGCCGACGCCCATCTGGCAATAACCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_27069',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 27069,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5952380952380952,\n",
       "  'tm': 85.49556775594306,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGGCAGGGTTTCCAGGACATGTAGGGTAATAAAACTGCCTCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_27134',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 27134,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 80.50934188757356,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGACAGAGGCCAGTCACATATTTCCGTTCTCACTCTGGAGAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_27180',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 27180,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 78.49727988641382,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCCCATCTGCCAGCTCTAGGCAACCTTGGTGCTGGGTGGGGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_27227',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 27227,\n",
       "  'strand': '-',\n",
       "  'gc': 0.6428571428571429,\n",
       "  'tm': 88.54001723273723,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTAAAAAGGAATCATGTGACAGGTCGTTTCTGACACAAAGTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_27304',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 27304,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.22634270233732,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGCACTCCATCTCTAATGTCACGGGAATGTCATCCTCTACCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_27411',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 27411,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.47611936801246,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAGCCCAAGAAGTTCCCAACAAATGCATCCCTGTCTCTCTCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_27455',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 27455,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 79.36006813046401,\n",
       "  'map_genome': 45,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGTAAAAATCAAATGACCAGGTGTGTGGGCTAGTTTTCAGAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_27517',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 27517,\n",
       "  'strand': '-',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 75.14517034838138,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGGGACCGGGGACACAATTCGTTATCTACCAATATACCCTCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_27567',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 27567,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 80.43645588763741,\n",
       "  'map_genome': 26,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAACAGAGAAACCTAACACGCAGAATCATTATGGAGAGGGTA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_27640',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 27640,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 75.80203507934982,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGCTTGGATACCTCTACAGACTCCAACAAGGGCCAGGAGTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_27704',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 27704,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 80.99381106062856,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGGATGTGGTATCCCAGAAAGCGAGCTTGTGTTTGGCCTGAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_27788',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 27788,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 81.01379351099723,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGTACTAGGATTTAATTCAAGGGTCCTCTGGCACTAGGCAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_27854',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 27854,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.73509594729506,\n",
       "  'map_genome': 41,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGTGGCCCTGGCTCATGACCCCAGCTATTTGGGAGCCCTGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_27997',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 27997,\n",
       "  'strand': '-',\n",
       "  'gc': 0.6190476190476191,\n",
       "  'tm': 87.1582223329633,\n",
       "  'map_genome': 41,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGCCATAAGGCAATTTCCCCTACCAACAGGTGGCTGAGGCCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_28065',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 28065,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 84.98560369658765,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGGGTAGGCCTCCAAATATGGGTCAGCATTTGCTCAGCACTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_28149',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 28149,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 81.70852453536077,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCTGGAGATCAGGTGGTTGAATAGAACGCTTACTTTGTTTCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_28194',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 28194,\n",
       "  'strand': '-',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 75.87133353281467,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGCTGGTGGAAGGCTTAGCTCAATGGAGTGTCACACAGGACA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_28271',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 28271,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 81.86234407945693,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCAGCAGCTCTCACTAAATAGGACGCAGCTATTTGTAACATA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_28360',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 28360,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 75.83219656959585,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGATTCTATAGGTAATGCTCCAAGAGCACACATGTTCCCCAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_28404',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 28404,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.20449225324364,\n",
       "  'map_genome': 41,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTGCATAGTTTGTGCTGCCTCTTGTCTTTATCTAAAGGTTGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_28449',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 28449,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.39875063702164,\n",
       "  'map_genome': 50,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTAGATTTGGTTTGAGCTAGTCCTGGGTGGTACCAAGAAACA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_28493',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 28493,\n",
       "  'strand': '-',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 77.63059884433437,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTTCCCCACACCTCTCACCCTTTGACCTCAGTAGGGCCTCAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_28627',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 28627,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 83.72057980619326,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCAAGACTTTGGGTGGAGGAGATGACAGTCCTGGCTGGTCAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_28680',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 28680,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 81.99022159027675,\n",
       "  'map_genome': 44,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCCCGTGCTCTGCACTTCAAACGTGACAACGAGAACTTGAGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_28725',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 28725,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.1922591839458,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AATGATTAAACAAAATTACCTGGGCCAATATGCAGCTGATTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_28852',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 28852,\n",
       "  'strand': '+',\n",
       "  'gc': 0.33333333333333337,\n",
       "  'tm': 74.01965256325036,\n",
       "  'map_genome': 50,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATTAGTGTGGCTACTCAAGTGAGACTATTGGAGCACTTCCAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_28896',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 28896,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.46838022704844,\n",
       "  'map_genome': 26,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTCCCTGGTTACTGAGCCCCTCCCTTAGGGTTCTCACCAACT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_28983',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 28983,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 84.23251867418736,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGAAGGCTCCATTAGTATTATGACCAGTATTCATCCTTGCCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_29072',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 29072,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.77504825266328,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCCAGTTCTTAAAAGCTGGACTGTCCTAGAGACAATGCATTA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_29132',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 29132,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 75.89343635552171,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATGTACCTGTCTTATAGGCTAAAGTATAGGCCACACAAGCAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_29176',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 29176,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 76.28873525974228,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAGGTTTATATGACCCACAACAACGTAAAGGGCAACCCCTTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_29235',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 29235,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 79.30974758931217,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACATTTGCTTCAGAAGGGTGTTAGGCTGTGGGAGTGCAAAGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_29335',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 29335,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 79.65873235158546,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGAAACTGTTAGCAGGGTCCCCTGCCATTCAGCCCAAGAGCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_29394',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 29394,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 84.37518317192627,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAACTGCTGCCAAGTGAGAAGCCGGGTGCCAGACCCCACCTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_29456',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 29456,\n",
       "  'strand': '+',\n",
       "  'gc': 0.6190476190476191,\n",
       "  'tm': 86.44837650440718,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATTAAATTACCTACCCGATCTACCTCACCGGCTGATGAAAAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_29554',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 29554,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.92016214754443,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATTAACATATGGTGTGATAATCCGCCCCATTCAGAAGAAGCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_29641',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 29641,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 76.50105706044536,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GACGGGTGGCGAGCAGTGAGTGATTCAGCCAGCCAGGCAGGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_29685',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 29685,\n",
       "  'strand': '+',\n",
       "  'gc': 0.6428571428571429,\n",
       "  'tm': 86.4253219017121,\n",
       "  'map_genome': 47,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTCCTGAGCCAGGGTCCAGGGTTCTCTCTCAGAAGAAGGTGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_29744',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 29744,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 82.74068620683016,\n",
       "  'map_genome': 51,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGGAAGGGCTCCAGGGCGGCTGAACACAAGGCTCTCCATTTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_29790',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 29790,\n",
       "  'strand': '+',\n",
       "  'gc': 0.6190476190476191,\n",
       "  'tm': 85.82769146108552,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CACCCAGTGTGCCAGGTCAGTCCGGGCCAGTGACCCTTGCTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_29872',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 29872,\n",
       "  'strand': '+',\n",
       "  'gc': 0.6428571428571429,\n",
       "  'tm': 87.68083260952295,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCAGAAGCAAGATGTGGGTGGTGACGACTCTTTTGAATTCAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_29917',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 29917,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.267522682086,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 27},\n",
       " b'CTCCTGCCTGCATTCCCTACCAGCAAAATTAGTCCTCAACTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_29993',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 29993,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.29056682476778,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGGGCTGGGCCTTCTTCCTCCGAGTTGGAGCAGACAGACAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_30074',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 30074,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5952380952380952,\n",
       "  'tm': 84.2421624013648,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGACTTCAAACCAGACCCTAGGACTAAAGTGCTCCTGAGGAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_30132',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 30132,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 78.80951944440784,\n",
       "  'map_genome': 45,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGGTCTGATGGGAAGCTTGCTTGGTAAGGTATAACCAGGCAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_30176',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 30176,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 80.45552637065924,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTTGTTATTCATGCAGGCCAAGTAAAGCAAGACAATAACTCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_30257',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 30257,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 74.14727072406362,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACAAGCCTCACGGCCATAAATCCAGGAGACTATTCCATGAAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_30344',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 30344,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 78.59425368327976,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGCATAAAGGGAGATCTCTGTAGATAAGAGGGCTCTCTCCTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_30429',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 30429,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 77.66156958652556,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTATGTCTACAAAGTGACAAGGGTACAGGTCCACCTGTGTGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_30548',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 30548,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.25044777209695,\n",
       "  'map_genome': 43,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGCCTGGACAGGCCCTAGAGAACGAAGAGGACGAAGATGACA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_30611',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 30611,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 81.71534064741161,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GACACAGATGCGGAAGCATAGTTTACAATAGCATCCAGCTGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_30672',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 30672,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.47409675545521,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGCTCTCCCAGCCCAAGGAGGAGTGTAAAAGGCTCTATTCCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_30800',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 30800,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 82.02479932327037,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAATCCCAGAAGATCTGTAGCTTTAACCTGAGGATTCAGAGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_30844',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 30844,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 75.63093402831151,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GACAGATAGACAGTTAACAGGCACTGGTGTATGTAGTTTACA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_30918',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 30918,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 74.33145778437449,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TAAGTGCTTAAAGTACCTCAGCTCATGTGATCCTCCATCCAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_30977',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 30977,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.53866672269118,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAGGCCTTCATGTTCCTGATGAGAGCCGAAATTTCCTTTTAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_31060',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 31060,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.43900135694611,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGAAGGAGGCCAAAAAGGGCCCTCAGCCAGCCCATTACTCAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_31143',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 31143,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5952380952380952,\n",
       "  'tm': 85.41371840469185,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGTAAGCAAGCTAGTTGTGAAAGACAGTGGCTCAGTTGTCAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_31187',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 31187,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.68735869153636,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACAAACAGTGTACATAACACAGGTCAATGCCCCAGCCTTTCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_31250',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 31250,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 78.79513529822464,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGGTACAGGGAAAGCAGGTAGGCTTCCTGATGACATGAGAAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_31339',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 31339,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 80.12322287485085,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTTCCAGCCTCTTTGTTGTGCTTTGGCGGTGCTTGTGCAGAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_31400',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 31400,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 81.29022658752581,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGAGACCACCTCCACACTCAAAGAATAACTGAAGTGGGAACT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_31459',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 31459,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.0681696442017,\n",
       "  'map_genome': 49,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTGTATTTGTGACAAAGGTTTCAGAAACATGCCATCCCCCCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_31530',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 31530,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 79.19313360576177,\n",
       "  'map_genome': 44,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCACTTCACAATTTAATGCTACACTTTGCAATGCAGATCCTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_31574',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 31574,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 74.49032405547536,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TAAATGGAGAAAGAGATGTGGAAGTTTGAGACAGCAGAAAGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_31712',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 31712,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.39690892847932,\n",
       "  'map_genome': 48,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGGCTATGGGGACAATTCAGCATTGGGGAAAACATGCTGCTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_31797',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 31797,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 81.65007941588686,\n",
       "  'map_genome': 46,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATTTTGGGGATTCCACACCCTTTTCCGACTTCATAGGCACTA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_31869',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 31869,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 79.47166272127686,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATTTATTTTAAAAACGCAGCTCAAGAGTGGGTAAGCAAGATC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_31983',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 31983,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.26244585752397,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTCACTTTTCAACCTCCAAGACTGGACACCAGGTGGGCAGAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_32050',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 32050,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 80.92092235965708,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTGCCATAGTTCATGAAAGGTACCAACATACGAAAATTAGAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_32109',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 32109,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.20198432272161,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGAGCAAGAACCAAAGGACGTCGTCCCAAAGTCAATTAGAAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_32274',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 32274,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.81868194771317,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGCCTGGCTGGCAGTTAATGGGTCAGTAAAAACTGTGGGAAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_32351',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 32351,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 80.91273345245116,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GATGTCACACAGCCCAAAACATTGGTCACATCAATCAAGGAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_32405',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 32405,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.4142887687376,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TAGTCAGTTGCTGCAGCCGTTTTTGCTGGTGTTGGCCATTGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_32463',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 32463,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 82.01567881953014,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAAGTGCTATTAGACAGGTGGGTCAACTGATTTCTGGCCATG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_32507',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 32507,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.8352557367794,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATGCCCCTGGTCAGGAACAACATGAGGGTATTGGCTGATAAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_32551',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 32551,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 80.67235780485385,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCTCCCAGCTGCACTCAGATGGAACCAGAGGTGAAGAGAGAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_32600',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 32600,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 80.04293958640392,\n",
       "  'map_genome': 48,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCCACCACACTGGTGTCACACCCTGCTCAACAGGATAGGCAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_32647',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 32647,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5952380952380952,\n",
       "  'tm': 83.67581427918071,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAGCAGAGCCCACCAAGTACATCTATGGTTACCTCAAGAACA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_32693',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 32693,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 78.20735969938681,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTCTCTGGTATCCTGTTTGGATTAACATTTGCCCTAGATTTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_32758',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 32758,\n",
       "  'strand': '-',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 74.22327750335393,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGGATGGCCCAAAGCATTCCATACACTCTAATAACTGAAAGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_32817',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 32817,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.61335664538007,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGTGAGTGTGAGTTAAGTTCTCTTGTTGCCAAACTCAATTA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_32921',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 32921,\n",
       "  'strand': '-',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.03808518205358,\n",
       "  'map_genome': 41,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGACTGAACTGCTGGGTGATTTGCAATCTGCTACTCACAGAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_33053',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 33053,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.98317831081187,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAACCTCTCCTGGCAACTACCCAGTCAACAACTATCCTGAGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_33116',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 33116,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.05320834496786,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTAAGCTTCTGGTCCCTAGCAATCCTGGGTTCTCTATCTCCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_33258',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 33258,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.26799110769485,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGCTCCTTTCCCGACCTCATAATATAACTAAATTGTCACTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_33322',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 33322,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 74.08103808437147,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCCTTCCTGTAGGTGTACCCACTTGATAGGGCTGGAGGCAAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_33380',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 33380,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 82.72415001034773,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AACTGACTGGCCTTATTGTTGGCTCTTGTCTGAACTCTCTTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_33666',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 33666,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.35191033016775,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCATGTGTGAGATGTCCAGCCATCTCAGCTCAACTCCACTCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_33745',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 33745,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 78.51127417161138,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGTGTCCTGAGGAACATAGGCTGTCTTGCTTTGTAGGGTTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_33854',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 33854,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 78.53333699831654,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGCCTCCTGAGATGATTATTCTATGGCATATGCTTGCATAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_33903',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 33903,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 75.09664110967162,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTCAGTGTTTTAACCAAAATCCCTCACCCTTCCCCAACATTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_33955',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 33955,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 77.69114371217779,\n",
       "  'map_genome': 45,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCTGTTTATACAGTCCTCGTCTACTTTCTATCATCCCAATAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_34017',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 34017,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.43949745785136,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATACTAGGGAGCAAAGGGATGCTGACGGCCCTGTGTATGCTA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_34135',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 34135,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 82.37817097540324,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAGGGATGGTTGTTGTGTAGTTAACTATTACTCCATTGTAAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_34187',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 34187,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.41163765336199,\n",
       "  'map_genome': 26,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGGGATAGGGATAGAGTGTAGGACCCTGAATACCAGTTTAAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_34243',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 34243,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.74670269595202,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAAATAACACCTCAACCTGACCTCAACTTATCAATCCCAAAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_34287',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 34287,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 74.55213513621442,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTTGGAATTTCTTGAACCTGGGGAGACCATTATTATGGGATT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_34469',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 34469,\n",
       "  'strand': '-',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 76.13050458444269,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGCATTGACTTCCTCATTCCCTTCAATGGCTTGTCCACATTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_34582',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 34582,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.53706822885562,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAAGAAAAGGTGAAGCGCGATGTCATTGGGAATGAAGACAGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_34658',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 34658,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.31773157325676,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAGGATGTGGTGTTCACGGCACTGGAGTTACTTCCCAAAAGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_34706',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 34706,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 80.45001645745253,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGAAACTTGAAGCAAGTTGGTTCAGGACTCCAGGTAGGGCAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_34781',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 34781,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.80344436186749,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCTGAGCTCTCACCTACATATTTATGGATGTGCCTTTGATCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_35416',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 35416,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.41909391981085,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCCACTCCACTCAAAACCTAAGTGCCACAAAGTCAGTATCTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_35571',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 35571,\n",
       "  'strand': '-',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.6466675617279,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TAAAGGTCCACAAACTAGCAATAAGTATGCTGTATGTAGTCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_35629',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 35629,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.46389504210958,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTGAAGACACCAGACGTCTACTGTCTAGTTTGATAGTTCTAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_35704',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 35704,\n",
       "  'strand': '-',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 73.17112111237435,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATGCACAGAAACCAGTTGAATACACGTTCCCCATATCACATG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_35827',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 35827,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.43819060633541,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTGCTTGTATACATTTACCAGTGCAGCCCACAAACCTCATTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_35879',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 35879,\n",
       "  'strand': '-',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 77.08975574405196,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGTGCAAGGAAACCAACTGTCCAGCAGACCAAAGAGCAATTA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_35949',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 35949,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 78.36593476808491,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCTGTGACATCATTTCCCAGTCACCTCTGGAATGATTCCTTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_35993',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 35993,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.98403130335197,\n",
       "  'map_genome': 50,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CACTGGGTGCTGGGCAACACTGGTGTCACCCATGTACACCCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_36042',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 36042,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5952380952380952,\n",
       "  'tm': 85.43708295410056,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCTCAGTCACTACCACTGGGAAAGGGGCCTACCTCTCCATTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_36098',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 36098,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 82.810470968397,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TATATTTCAAGGCCCCCATAAAAGACACACACTAGGAATGAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_36152',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 36152,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.92176115663352,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTTCAAGTAGCTGGTGAGTCTCTTTTTGGAGAGAAGATCAAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_36229',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 36229,\n",
       "  'strand': '-',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 73.73213263101098,\n",
       "  'map_genome': 44,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCTTCAAGGGGATGCCAAGAGTATCACGAACAGCTTCAACTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_36273',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 36273,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.4225792573032,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTCACTGAAGCTCAAGCAATCATCATCTGACACAAAAGCTGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_36351',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 36351,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 74.61821722414572,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTCACACTTCCTGTGATTGTCTTCCCGTTCCAGTCACACTTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_36470',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 36470,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 77.7440152501419,\n",
       "  'map_genome': 73,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 65},\n",
       " b'CAGTCACAGGAAGTATGTCTGGGAACGGGAAGACAGTCACAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_36776',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 36776,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 78.83232745796505,\n",
       "  'map_genome': 173,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 160},\n",
       " b'ACTGTCTTCCCGTTCCAGTCACACTTCCTGCGACTGTCTTCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_36847',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 36847,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 80.82148706265616,\n",
       "  'map_genome': 60,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 54},\n",
       " b'ACAGGAAGTGTGTCTGAGAACGGGAAGACAATCACAGGAAGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_36901',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 36901,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.98879000285729,\n",
       "  'map_genome': 56,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 47},\n",
       " b'AATTTGGCTGTTTAGTTCACCCCTACTTTTGCCAGTCCCTTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_37015',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 37015,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 78.73005242126357,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGACATGACATGATAACCCATGTCAAAGGTGCATCTGGGCAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_37095',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 37095,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.61738801414964,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAGTATTAGAGTCCTTGTAATGTCGGGAGGCTCTGTGGTCCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_37147',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 37147,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.65824886684953,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCAAAGATTGTAAGACTAACAGTCCTGCTCAGGTCTGTGGTA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_37203',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 37203,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.65323796843097,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TAAACACATGTATAACGTTCATTTTGGTATGGGATGATAGGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_37248',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 37248,\n",
       "  'strand': '+',\n",
       "  'gc': 0.33333333333333337,\n",
       "  'tm': 73.13005965657845,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTACTTGCACACAATATCTCTGGGATGGGACCTTGAGATTGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_37310',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 37310,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.94148829057826,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGGGGTGAGGGGCAGTTCATACCTCATGCCTTTTGAATTCTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_37362',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 37362,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 80.53185130674353,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCCCAATTCCAACCTCACTTTAACTTTGTTACAATTGATTCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_37440',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 37440,\n",
       "  'strand': '+',\n",
       "  'gc': 0.33333333333333337,\n",
       "  'tm': 73.04302002296106,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CACACACCTGCCAAAGGTGAATTACTAGAAGGGAGCACTAAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_37488',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 37488,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.84296769885674,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTGGAATGTCAGAATAGTCTCACTACCCTAAACATCCTTCTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_37551',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 37551,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 74.08859703622204,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGACCCTGTCAACAACTAGGTCTTTTTCCTAGTGTGTCCTTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_37701',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 37701,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.72630396884796,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGAGAGGCTGAGTTAAATTGGACTTAAGGTTTTAGATGTACT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_38084',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 38084,\n",
       "  'strand': '-',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.16786709925412,\n",
       "  'map_genome': 44,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCAGTGATGTGTCAAGTGTCTAAAACTTACTTCAGCTGATAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_38225',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 38225,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 73.02161981992475,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTGGACAGGGGCCAGGGAAACAGTTCAGGGGGTTACAGCACT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_38278',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 38278,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 85.82777641754979,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCATTTTAACCTTTATCCTTGATCCCTACCTTAGCCCCTTCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_38345',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 38345,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 77.03265809491256,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AATGAAACAAGAGGGTCATTTATCTGAGTCCTGCTCAGCTCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_38431',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 38431,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.5465581154366,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGACAAAATCAGTGTGCCCTGAATGAGGTGCCAGTCCACCTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_38576',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 38576,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 80.77066694360565,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGGCACAAGTAGAAAAGTGATTGACTCCAGCAAGCCCGGGCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_38621',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 38621,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 82.33596313353797,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGAGAAGCACTCTCATCCAAGCGTGCTATGCCCCACCCCCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_38665',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 38665,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5952380952380952,\n",
       "  'tm': 84.75748085828718,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 1,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGATGTTTTTCACAGACCGGTTTTCCAGGATGAAATGCTTTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_38711',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 38711,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 75.31810558329016,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATTTGGACAGTTCAGAAGTTCTGGTAGTGTCTAGTGAGAGCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_38776',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 38776,\n",
       "  'strand': '-',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 75.33590330756346,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GACTTCATGGTGTTGGGGGAGTAATGGCAGCTTAACTAAATA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_38854',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 38854,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 77.23824654054715,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCTAGTCTTCTTTGTCCCTTAATCACCTTCTGGCCTAAGTTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_38898',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 38898,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.69828797251859,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AACAGGGAGCATGACAAGTTCCAAGTATAGACCCGGGGTTGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_38953',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 38953,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 81.36029382694488,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCATAACAGGGTATCAGTAAAAGTCACTATCAAGAAAGTGCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_39050',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 39050,\n",
       "  'strand': '-',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.49697662791698,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGGTCTCTGAATCCATTTCCGTCAGTTGCTGCATAGAGCCTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_39126',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 39126,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 79.42605309266474,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGGATTTCATAGTCAGGTGACAGCTTTAAGGACAACCTCTGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_39414',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 39414,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.93574748076469,\n",
       "  'map_genome': 50,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCTTCACAGTTAGGCACATCCTTGTCTAAAGCAAGATAAGGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_39471',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 39471,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.22363892606887,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CATCTCAGATGCTGCTCCACCTCCCGGTCACCTCCCCACCCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_39610',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 39610,\n",
       "  'strand': '-',\n",
       "  'gc': 0.6666666666666667,\n",
       "  'tm': 87.72383103778981,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGGTGACTTTAGGAGAAGATTAGGGTTTTATGTAGACACAGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_39678',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 39678,\n",
       "  'strand': '-',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 75.44445688066133,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 1,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCTCTTGGTGAGAAGCCTAGGCATATGCCTTTGTACAGACTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_39723',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 39723,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.80415890638744,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGCCAATTGGCCTACGGTCATACAGATGCCCTAAGGTTTTAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_39809',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 39809,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 80.42401265286861,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTCCAGTTTCTCTGCCGGAAAGCCGAGGCAATGGGAACATGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_39901',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 39901,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 82.86144560263193,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTCTTGAAACAGATCGGGTGTAAGTGACAAGGAAATACCATG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_39979',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 39979,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 74.93627885180456,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TATTCTCCAAATTCACCTCCCATCTCAGACCTGCTCAGGCAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_40037',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 40037,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.12263368171466,\n",
       "  'map_genome': 47,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAAAAAGATGAGCCGCTGGTAAATGTTTGCACTCTGTGAGGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_40089',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 40089,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.42331599550067,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGTTATCCTCCTGGCAGCTATACTTCGTTTGAGAAGCCAGAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_40146',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 40146,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.91748975515662,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGCTTCCACTCTGATCCTGGATGGCAGGTTACCTACCTAACA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_40240',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 40240,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 80.6822537687429,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGTGTGAAACCCTTTGTGTTTAGATCTAGTAGGCTAAGAAAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_40313',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 40313,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.40450327106066,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCTTTCACTTGTTGGCTACATGAGCCAACTGGAAATAACCAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_40376',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 40376,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.90904286299559,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AATAATTTGGCTACTTTTGGTTTAGGAATCCTGCCCCACCAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_40430',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 40430,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 78.11569386315784,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAGCAGAGGGCACAGTACGCTAGGAGGTCACTGGGAAGTATG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_40478',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 40478,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 81.72211620392551,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGGAGAAAGGGGGATTCCATTGAGTCTAAAATCTTAGTTCAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_40544',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 40544,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 74.86259075270499,\n",
       "  'map_genome': 44,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGATGACTCAAGGGCCATTGGCAATGTATCTTCCCAACCTGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_40590',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 40590,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 81.2135496425758,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCCTTCTGTTGTGTGCTGACCGTGTGTGTCTTCTTAGTGGGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_40635',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 40635,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 80.76743163484383,\n",
       "  'map_genome': 45,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTGATAATTAGTGAACCTAGACTCACCTCTTTCCTTGTCTGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_40764',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 40764,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.57008881807138,\n",
       "  'map_genome': 50,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGCATTAGCTGATATAAGAAGACTCATCCAGACTATGAGTGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_40822',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 40822,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 73.65757750226658,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGGTGGGAGATGTCAAAGGAGCAGATGCTTGAAGCAGCTGTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_40952',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 40952,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 80.0761090235863,\n",
       "  'map_genome': 49,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CATGGCTGCCCTGACTTACTTATGCTATTCTAGCTTCTTTCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_41079',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 41079,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.65407402260973,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTGACTCCTTCCTGGGCTGTCAAAAGAAGTGGGAAACATTGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_41150',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 41150,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.5948754489134,\n",
       "  'map_genome': 44,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TAAATAGAGTATTCAGCCAAAGTGAATCCCTGTTGACCTCAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_41194',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 41194,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 74.46574792535984,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTGGTGACCGTGGGTATGATGTATGCTGGGATACAGGAGGAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_41306',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 41306,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 81.23485157339525,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGAGGAGGACTGACTTGCTCATTCAACCCAGGGGAAGCTGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_41362',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 41362,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 82.1865065524301,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGATGAAGAACCAGGAGGTACTTGGTAACAAAATAGAAAGAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_41406',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 41406,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.5641848256721,\n",
       "  'map_genome': 49,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCCTGTCTCCTAAGCATGTTCTCAGGTCCTGATGGTTTTGTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_41470',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 41470,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.41597326668847,\n",
       "  'map_genome': 48,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCTGAGCCCTGTGTTTTACTGTGATTGACCATGGGGAGCCAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_41621',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 41621,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 81.59835570860747,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGAGTAGAACAAAACCGAATCTGGGCATTTCACTTACCCTCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_41678',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 41678,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.6805542798819,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCATCCCTTGAGTAAGGATAAGGGATACAGTGAGACATTTGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_41722',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 41722,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 75.85069081415912,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTAGTCACACTTTCGAATGTCCCACCACATACAACACTTGAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_41817',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 41817,\n",
       "  'strand': '-',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 75.76647320153495,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACGCACAGCATACAAAATGGCACTTGGTCCCATCAAGTTGTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_41861',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 41861,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 79.01327490658514,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAAGGTCAGCACCTGCGGAGGGGGCATCTCCCACAGATTTCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_41909',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 41909,\n",
       "  'strand': '+',\n",
       "  'gc': 0.6190476190476191,\n",
       "  'tm': 85.41078039457994,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAACCTGTAAGATGCTCATCATGTGTCCCCAGAGAAGCCCTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_41961',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 41961,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.50782785043589,\n",
       "  'map_genome': 46,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGACAGTTGTCAATATCTCTTGCGTTCTAGCAATCTAAAGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_42030',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 42030,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 73.17882892735878,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAGAGTTTCTAGGGTGGGGCATAGCTTCGACAGTATGAAATC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_42115',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 42115,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.28329623680139,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGAATACAATCCAAACGCTGCCTTAAGTCTTCTTGTTTTAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_42229',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 42229,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.046352644043,\n",
       "  'map_genome': 43,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCTCTGGGCAGATACTACATTTTTAACAGGTGCTGGATCCAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_42276',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 42276,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.76505126038666,\n",
       "  'map_genome': 44,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTGTATTTTCCTAAGACCCCTCTGGTGGTGTGTGTTCAAGCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_42332',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 42332,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.414604611127,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAGAAGAGAAAAATTGGCCCTCACCGGGAAGCAGACCAGGCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_42611',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 42611,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 82.53992488382278,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGTAGATTCAGTGCTAGAATACGACTCCTGTAAGCTTTGAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_42663',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 42663,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 73.82023202292578,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGTCAGCGGGCATGGGAGCTTAGATCTCTGTTTCACCTGTCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_42707',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 42707,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 81.50651961736838,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCTGCCCTGTTGCCCTGCCATATCAACACTGCCATACAAATT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_42753',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 42753,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 81.24603801940117,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGTGCACATGTCCACTAAACAATATCTAGGTATCAGCTAAAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_42808',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 42808,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.22882954123912,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATGATACTTTAAGGTACCTAGCGGTGTCATCAAATGATTGCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_42865',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 42865,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 74.71925624696058,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGCTTAGGAAATCCTGGCAGGCTTCATGCGGGAAGCAGAAGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_42990',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 42990,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 82.80624054903456,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGCCTTGAATGGATTGTACAAATTGGAAGTGGCCAAGTCTAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_43037',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 43037,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.75420571392249,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TACCATTCCTTATGTTGTGGGGGGGGGCCCAACCATAAAATT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_43119',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 43119,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 83.16663748670095,\n",
       "  'map_genome': 49,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTGCGCGCACGCGCTATGTATATACAGGTACAGATGGTACAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_43331',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 43331,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 79.95903202466411,\n",
       "  'map_genome': 47,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCAATCCCTCAGCACATTTTTTATGATAGGTAGGGATGTAAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_43478',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 43478,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.15847822639608,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGCATTTATACCAAAACATTGAGGATTGGGAAGATGACTAAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_43603',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 43603,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.61185573897518,\n",
       "  'map_genome': 44,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTGAAGAACTGGAATCTGGTTGTGGATCTGGCAAAATGGCTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_43649',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 43649,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.60689699258796,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTCTTGGCATACAAATATGAGAGTGGAAGCTCAAGTCTTCAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_43702',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 43702,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 74.0308768325641,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAAATTCCAAGTGAGCATAGAGAACCACTTATTAGCCCTGTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_43752',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 43752,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 75.48070686044946,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GACAAATCAGCTAGTCCAGACGCTTGCTCCACGGATTCCATG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_43809',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 43809,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 79.36258404315748,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCTGAGATCAACTGAAAGGTCCCGACTTGATATATAAAATGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_43855',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 43855,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.37251143617812,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TATGCACATGCAAAAGTAGATCTAGTTGGTATGAAGGAACAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_43994',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 43994,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.39690892847938,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGGGAAGGCTTAGGGGTTTGCCCCAGATATCCAACTGTGGGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_44041',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 44041,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 85.70214913289635,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTTGAATAGTTGAGTGACATGCTTGTATGTTGGCTGTGGAGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_44104',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 44104,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 75.67100018173636,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCTGTTTGGACTGGAAGAAAGATCTCCTAATCCTGGTCACTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_44181',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 44181,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.1526385928509,\n",
       "  'map_genome': 44,\n",
       "  'map_transcriptome': 5,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AACTCCATGCCAACTGTGCAGTCAATAGATGCTTAACACTCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_44273',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 44273,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.51661694434978,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTCAAAGGTTAGAGTTGCATCTAGACCCCACAAAGGCTTTGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_44319',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 44319,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.59424375759158,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACCACTGGCTTGCCAAAACTGGCTTTTCACGATTGTCACTGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_44383',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 44383,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 79.79088376369941,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGGGTTAGGAGTCTGTCATCACCATATCGAATAAACATGACC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_44504',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 44504,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 75.80757578255657,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGCAGGAACCAATGGGAGTTCACAGTCTGAACTCCCAGCAGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_44561',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 44561,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 82.40682544110365,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGAAGGCTTTGACAGTAGTTACCTTTCACTGGAGCTGCTTCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_44799',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 44799,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.21014869581836,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTCATTCTACAGTTCTATCTAGATACCAAGTTCCTTGCCTGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_44918',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 44918,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 73.81000656142459,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAATACAATGAACTACTGATTGCAGTAGGCCAATCTTAGGGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_44985',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 44985,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.19462176541265,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAAACCCCCACTTTCAAGGGTACGAGGGTATCAGACATTTTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_45065',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 45065,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.62659849235837,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGAGCTCTGGCCTCATTGTGACCTGTAGGCCAGCTGGATGTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_45151',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 45151,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 82.72524060909188,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TAGGAAGGGGGCAGGATGTCCTGATTGCATCACAGATGGGCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_45227',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 45227,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 83.68957874882841,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGTTAAGACATTTCCACATGTTCTACTTGGCCTAGACCAAAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_45354',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 45354,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.04533933614312,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTTCACTCCCGCTCTGCAAGTGATTTCACACCAATGTCTTTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_45398',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 45398,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.1424054613247,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGAAGGACTAACCTTGCACTTACGTGATTCTCCACAGGGATT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_45496',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 45496,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.55947938148012,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATGAGGCATGGAGATCTTACCTCTGCTCAGAGAAATAAGAAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_45540',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 45540,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 74.70525822328563,\n",
       "  'map_genome': 51,\n",
       "  'map_transcriptome': 1,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCCACAGGGACTCAGGTCTCCCAAGTCAAATCACTGTAAGGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_45591',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 45591,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 80.05814511617172,\n",
       "  'map_genome': 43,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAAGCCAGTGAAGTAAGGCAATAACGTGAGCACATCCAAATT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_45673',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 45673,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.49219744292327,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGATCAAAGGGTGAATCTGATGTTCATTGTTCTTGTGAGCTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_45759',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 45759,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 73.91341210845627,\n",
       "  'map_genome': 45,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAGCTCAGGGGAACCAAATCTGTTCTGAAATGGAAGATCTGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_45838',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 45838,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.3252196495705,\n",
       "  'map_genome': 45,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCAAGGAGAAGGCAATGACAGATCAGAGCATTCTGTCACTGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_45928',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 45928,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 76.87161574986197,\n",
       "  'map_genome': 41,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGTTCACTGTCCAGCACTCATGCCCCCTCAGAAAGCTGGCTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_46004',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 46004,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 82.99429097463354,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCAGAAAATATACCATACCATGCCAGATCTATAGGATTTAGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_46309',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 46309,\n",
       "  'strand': '-',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.05135978134689,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATTCCGATTCAAACCAGCCTCCGTTTCAAGTGATGAGTGGCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_46362',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 46362,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 80.00831508504785,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGTTTGGTTACCTTTACCTTCATGTTTACTTCTCTGACTCAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_46433',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 46433,\n",
       "  'strand': '-',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.28981779469416,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CATAAGTACAGAGAAGGCCATGGTTACCATCGTCACAGTCAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_46561',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 46561,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.83998646450812,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTCCATCTTTAGCCTGACCCTGTGAATTGGGTATGTCTCTCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_46639',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 46639,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.89680726463149,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGCAGATGAAGGACTTATCAGCCAAGGTCATACCATGTATCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_46752',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 46752,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.46870948911993,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGCCCTCCACAGATTTAACTCTTTAGCTTGTTTGGGATTTTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_46825',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 46825,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.84563891446658,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGTCTATTTGAGTAGAGTCTCACTCTCTGGCCCAAAGTAGCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_46869',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 46869,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.34169019430817,\n",
       "  'map_genome': 46,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGTCCTGCTTCCTTACTAACTGATGTTGAGACTATTGCAGAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_47030',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 47030,\n",
       "  'strand': '-',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 74.93564395742521,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTCAGGTCCCCTAGTGGGAGATCTGTCTTAGCTAGCTTCATA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_47091',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 47091,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.67168659149547,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGGAACCCATTGCTGGGCACACAAAATCTCCTGGGTCTCACA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_47135',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 47135,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 82.46673773479341,\n",
       "  'map_genome': 44,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGAGTGCCATCACATGTATAACCTGAACTGGCTGTTACTCTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_47197',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 47197,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.60175185721039,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGGAAAGGGGCATTCAATAGTGACCAGATCATCTTTGAATCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_47279',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 47279,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.32240489123996,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTAAAATGACCTCCATCCTATTGACTTGCCCTCTTCAGGTGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_47346',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 47346,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.87731515329017,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCTCTAACTCCTGCATCAGGGACCCTACTCTCAGTCTAATGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_47457',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 47457,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 78.51029180270018,\n",
       "  'map_genome': 43,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCAGGGTAGGAGTGTTTTACAGCTGACCTCTCCTGTTCGAAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_48082',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 48082,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.30099009279286,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGCCTTAGCTAGCTACAGGCCTGGATAAGCTGTGTTCACCAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_48144',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 48144,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 81.0049195545102,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACTACAGGCAGAGCACGGAGTTCTGCCGGCCCCTACCCAACC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_48199',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 48199,\n",
       "  'strand': '+',\n",
       "  'gc': 0.6428571428571429,\n",
       "  'tm': 87.81666938640097,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCTGTCAAACTATAAGCCCACTTTGGAGATGCTCATCTCCTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_48243',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 48243,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.20690710965772,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAAACTCTGGTCCAGATGTACTTGGGAGGAAACTTGCAGATC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_48295',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 48295,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.98971173375253,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CACGTGCAGACAGGAGGACTCTGGCGAGAAGGCAGAGACACT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_48339',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 48339,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5952380952380952,\n",
       "  'tm': 82.3767147571885,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTGGGGCAGCCCTGGACTCAGCATGAATCCATAATCCATTAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_48399',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 48399,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 80.66070579032396,\n",
       "  'map_genome': 47,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTTAACAATGACCCATGGGACCAGCCGGTGCTCGGTGCTCGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_48469',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 48469,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5952380952380952,\n",
       "  'tm': 85.4730438345876,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAACTCTCCCGCAGCATTCGTTGGCCCAGATCGCACAGGCAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_48557',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 48557,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 84.50029853485972,\n",
       "  'map_genome': 26,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGAAGGTAAGAGACTGGTTCTGAGCTACAACTCCTCCCCACA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_48613',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 48613,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.61281326812434,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TAAAAGAGCCGTCTTCTCTGACAGGCCCTTTGAACAAATGTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_48675',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 48675,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.09948354002586,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTAATTTTCTTTTCCCGCCAAAATGCCTATAGCAGCAGCAGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_48732',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 48732,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.95996427329237,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGGGACACCTGCCATCTGACCCGTGGGTGCCATCCCTAATT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_48790',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 48790,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5952380952380952,\n",
       "  'tm': 85.85806781536962,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCTAGCAAATACAAAAGCTGGCAGCCGCTGTGGGTCAGGCTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_48847',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 48847,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 82.6403229518732,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCACAGCCGCCTCGCGTCAAGACGACGCTACACCCTTCTGGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_48908',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 48908,\n",
       "  'strand': '+',\n",
       "  'gc': 0.6666666666666667,\n",
       "  'tm': 86.59078806438134,\n",
       "  'map_genome': 26,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCCTGGGTCCCGCCCCGCTCCTTGGCCGCACTGAGGACAGCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_48982',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 48982,\n",
       "  'strand': '+',\n",
       "  'gc': 0.7380952380952381,\n",
       "  'tm': 92.97116955107413,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGGCCTTCAGCAAGAGGCAGGCGATTTAGGGACCCTGGGACC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_49062',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 49062,\n",
       "  'strand': '+',\n",
       "  'gc': 0.6190476190476191,\n",
       "  'tm': 87.06253421801705,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TAGTGTGCATTCCTCAGTCTGACTCAGTGCTTCAATCTTGAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_49106',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 49106,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 75.22041655211223,\n",
       "  'map_genome': 50,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGCACTGGCTGTAGATCGTTGTGTGTTGTTAAATGCCAGGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_49151',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 49151,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.44206751151341,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACCTTTTCTGTGTACTAGCAAAACTTTGGACACACAATTTCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_49211',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 49211,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.69887836287398,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTAAGTTGCAGAGGAGAACGTGCTTCGAAAACCAAAAGCATT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_49342',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 49342,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 76.02373633285072,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCAGTTCTAGCCCACCTAGGTATGAGTCATGCCCTGTATTCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_49441',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 49441,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.80990233183763,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGAGTGAATAGCTCAGGTTACATAAATTTTGGGGCAATCATC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_49527',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 49527,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 74.2808128699674,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTGATCCCATCACTCCTCCGGCATTATTTCTGGGTACTTAGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_49581',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 49581,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 78.4770023149199,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GATGTTCAAGGTGATTATTTCTAGGTAGTGAACTTGGGTCTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_49734',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 49734,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 73.82944418858028,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAGCCATGGATGGAATGACATTATCTGTGGGTGCAGATCATT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_49884',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 49884,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 77.21623745851895,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGCCTGGGGACTCCAATATTTACAAGTTAGACACAGAAGTAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_49992',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 49992,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.86666880343978,\n",
       "  'map_genome': 49,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGGCTACACAGAGTTGGCTGTGGGCCCCTATGAGGAGGCAGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_50115',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 50115,\n",
       "  'strand': '+',\n",
       "  'gc': 0.6190476190476191,\n",
       "  'tm': 86.61093074967391,\n",
       "  'map_genome': 49,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAGACTGAAGGAACTCCACCCTGCCTGAGTAGCTTCCCACCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_50159',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 50159,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 83.48287364817992,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCACATATGTACCTTGAGAATGCCCTACTCACAAACAAGTTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_50204',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 50204,\n",
       "  'strand': '-',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 74.56077009957932,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGAGACAGGGAATTGGGGCATAATAAAATGGCCAGTGGAGAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_50274',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 50274,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 79.54237560050365,\n",
       "  'map_genome': 43,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGGCTTACATGGAACTTGAACCATGACATCCATTCTTGCCAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_50379',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 50379,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 77.62784171249842,\n",
       "  'map_genome': 41,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGCAAAAGTGACCATGATGCCAGGGAACTGGCTGGGTATGTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_50434',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 50434,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 81.7291169570426,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAATCCATATTGTAATCACGCAACAAACAAAGTCCTCAGTCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_50483',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 50483,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 74.36235578982007,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCTCTCCTGGCTCAGGGGGCTCGTTCTCATGTTCTTTACTCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_50527',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 50527,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 81.3703574933761,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AATGACACCACAGAACCAAGCAATCCCTTGTGTGGTTGCCTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_50575',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 50575,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 80.41488279493745,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGGGACCTCCTCCAAACTTGGACAGACTGGTGGAGGTAAGGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_50662',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 50662,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 83.8291018729227,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTGCCCTTTGCCTCCTTATTAGGAATAGCTTCCTACCCCTGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_50745',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 50745,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 81.71913491890064,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGAAAGCTGGCAAGGACTACCACCGAGGACAGGAAGTGTCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_50833',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 50833,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 81.51020587287775,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTTGGGGGGTCTCAAGAAACTCTGGGTCTAGTTTTAGAAACA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_50890',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 50890,\n",
       "  'strand': '-',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 77.52666464138287,\n",
       "  'map_genome': 43,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATTGAACTCATGTTTCTATGTTTCACACCAAGAAGCCAAGAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_50953',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 50953,\n",
       "  'strand': '-',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.30751791427116,\n",
       "  'map_genome': 49,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGATCTTGGCACTACAAAACCCTAGAAGACCTGCTTGACTTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_51057',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 51057,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.91451413760387,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGGCAAGTGCCCTGCCTTGGAACTGCATCTTGAATCCTAATT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_51122',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 51122,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 80.20874689174036,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TAATAAAACCACGTTTTGAGTTCCAAAACTGTCCCAGGTTAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_51275',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 51275,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 74.95650436121554,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTCCCAAAATCACAAGTCCCCCATAGGTACTTCAGCACCCAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_51375',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 51375,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 81.15463320689264,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACTGAACAAGGTTGGGGTAGTTACATTGGTCCAAACATTCAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_51440',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 51440,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 76.91006416248445,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTAAGATATACAATTCGTAGTCAGCCATGGTTGTTCACATCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_51544',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 51544,\n",
       "  'strand': '-',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.10827351436751,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATTCTAACCAGGACTTAAATCCTTTCTACGTTAGGCAAGAGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_51711',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 51711,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.13805652232668,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGCTACTGTGCTAATGAGCTAGTGCTCCCAGGCAATTATTTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_51755',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 51755,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 77.28971873154421,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACACCACAGGGGTCATATTAATTTTAGTCCTGTAATTAAAGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_51885',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 51885,\n",
       "  'strand': '+',\n",
       "  'gc': 0.33333333333333337,\n",
       "  'tm': 73.35824287171596,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAAAAGGACAAAAGAGTTAGGACATACATGCTTGGAATGGAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_51982',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 51982,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 74.22876703708027,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATGGAATCACACAATAACCATGTTCTTGAGAGTGGTGCTGTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_52092',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 52092,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.51695532503135,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGAGTCTCTCCTTACACTACCGGAGAAAGTTGATAGCTGACA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_52220',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 52220,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.13239353587875,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAAGGCTGGTAGAGGGTTGTATTGCTGGTGTTAGGGTAGGAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_52307',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 52307,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 80.89529840181473,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCTTTTACTTAGACACCCTGCCTTGTCATTTACTCCCATTTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_52351',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 52351,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.22784598243646,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGTTTTTCCTTGACTCTGTGTACTTTCGAGAATCCTAGATCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_52445',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 52445,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.12900684720933,\n",
       "  'map_genome': 45,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCCAGTAACTTTAATTGAAAGATAGCATCGCCATGTTCTTAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_52549',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 52549,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.09946820216288,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTCATGAAACTGACTGAGATGAGCTTGTGCTTCCTTGGCACC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_52643',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 52643,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 77.94743635740474,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATGAAGTTCCTTACCTAAGAAGCTCAAGTGGTATCCAGAATA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_52777',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 52777,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 74.22898042949612,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTTACCCTGAGAGCCATCCTCCCGACCTGTTTTTGAAACCAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_52833',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 52833,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 80.56032199048656,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAAGTATCTATCATAGTAGTGTTCCTGCAGGGAAATTGGAGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_52947',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 52947,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.02492644720331,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TATGTGCTTTTGCCATGGAGCACGTGTGGAGGGAGGCTGAAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_53101',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 53101,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 82.35254464477157,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGTAAGAGGGAGGCCTAGGGCTGCTAGGGGAGAATTATCATC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_53258',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 53258,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 81.50856655014633,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCCTGTCTAGTTGCTAGTGTAGACTCAGGGTTCACAGCAAAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_53310',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 53310,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.20297811223003,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGATAGTGGCAAGTCAATTAATGAGTGTTAGTAACCAACTGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_53370',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 53370,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.87037445814286,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCTGATTAGACTGGAGTCCAAATTCATAGCTGGCACTGTAAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_53414',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 53414,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.24649456108233,\n",
       "  'map_genome': 48,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCCATGGTGGAGAAGCCGTTGCTATTGTTTCACTAAATGGAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_53489',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 53489,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.47996350475955,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTACATTTATGCACCACCAGCTTTGGTCAGAGTAGCTCAGTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_53562',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 53562,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.94665912207017,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCTTTGACTAGTTATGTCACTGCAGTTATTGTCCCTCACTAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_53606',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 53606,\n",
       "  'strand': '-',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 74.70476420283694,\n",
       "  'map_genome': 51,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATGCCCTGAGGCATGTATATGCCCTCACACATGAATTTGTGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_53722',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 53722,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 78.8208637926549,\n",
       "  'map_genome': 47,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTGTCACAAAATAACCTTGGTGTCAGGTCCTGCCTTCCATTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_53766',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 53766,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.21299834856734,\n",
       "  'map_genome': 43,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGTGTTCCAGTCTAGCTGCCATCTAATTCCCAGCTCATTGTA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_53848',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 53848,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.86850913049363,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTAGGGTTACAAACACTTGCCGCCACATCTAGCTTTTGTTGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_53896',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 53896,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 78.1501363381393,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAGGCTCAACCACACTTAAGGCTTCTACACCACCACCTCCAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_54212',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 54212,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 81.26338092201416,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGAGGAAGTAGATTATGGACAAACACAATCAGGACATGCCAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_54301',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 54301,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.47783872193781,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTCTTACATTCTTGATCCTATACAGTTGTAACTAACAGCCCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_54345',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 54345,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.35853081598248,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AACCACACAAGATTGGAACTCTCAGCATTCCCTTGCATGAAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_54389',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 54389,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 77.03760565911688,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAGATAATGGCAGATAGAGGGGAGAGGTTCATGAGGCCCGTA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_54472',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 54472,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 80.76915286630162,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTCCCTGGGTATCGGTGGAGAGTTAATTGTAACTGGGGGAGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_54517',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 54517,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 82.36838800677577,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTTATGCTCTGATGAAGAACCCCATAAGAAACCCCAATGAAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_54598',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 54598,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 75.81036663362562,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAAAGGCAAGGTGGTTGTGGGGAGTGAAAAATAGAGTACCCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_54655',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 54655,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 79.75913511415024,\n",
       "  'map_genome': 43,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACCGTAATGAATCCCAATAAGCATAATCAATATAGGCTAACG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_54780',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 54780,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.69647526447852,\n",
       "  'map_genome': 41,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGAAAGCGGGTTACATCCCAGCGAATGGGTGGGTCTCCACAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_55073',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 55073,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 84.24908279612151,\n",
       "  'map_genome': 26,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GACAAGAGATGTCCGGTCGGAGCTTCAGCTTCCCTGGTGTTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_55136',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 55136,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 81.45315425950486,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATGGCCCTTCATTTTAACTCTCGCTCCCATAGCCTCTCCCTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_55259',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 55259,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 81.28511596947033,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCTGGCTGATGCTCCTCCATGATGTATAATCAATACATGTTA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_55323',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 55323,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.00327593525526,\n",
       "  'map_genome': 41,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAATTCATGACCCAGGTACTCATGTCATATCTAAGATCACAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_55456',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 55456,\n",
       "  'strand': '-',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.01794690941603,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCTTGAATAAGATTATGTCCTCACTCAAAGTCCTTATCCCTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_55542',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 55542,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.62666580647323,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGGAGGGATAGGGTCTTAAACCACACTTCTTGCATGAAATTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_55605',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 55605,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 76.50258605664425,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACCACTCCCCCTTTCAGATCTGCCCAACTTCTTCTATGTCTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_55660',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 55660,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 79.27946857109725,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCCAGTCAGAGAATGTAGGTGGTTAGGTGGAATATGACTGAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_55704',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 55704,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.40493055004742,\n",
       "  'map_genome': 43,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TAGAATCTGTGATGGCTCTCCATGATCTGCACACCACAGATA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_55836',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 55836,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.75991782142529,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACTCTTTAGTTGTATTTGAAACCGTGGGTCCTGAAGCATGTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_55920',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 55920,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.24069656340947,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26}}"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# load the saved common pb dict\n",
    "pb_dict_from_load = pickle.load(open(os.path.join(library_folder, 'common_pb_dict.pkl'), 'rb'))\n",
    "\n",
    "\n",
    "print(len(pb_dict_from_load))\n",
    "pb_dict_from_load[1].kept_probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Screen probes by genome counts.\n",
      "-- filtering region:251, screen genome to keep 400 probes\n",
      "-- filtering region:252, screen genome to keep 400 probes\n",
      "-- filtering region:253, screen genome to keep 400 probes\n",
      "-- filtering region:255, screen genome to keep 400 probes\n",
      "-- filtering region:256, screen genome to keep 400 probes\n",
      "-- filtering region:257, screen genome to keep 400 probes\n",
      "-- filtering region:258, screen genome to keep 400 probes\n",
      "-- filtering region:259, screen genome to keep 400 probes\n",
      "-- filtering region:260, screen genome to keep 400 probes\n",
      "-- filtering region:261, screen genome to keep 400 probes\n",
      "-- filtering region:262, screen genome to keep 400 probes\n",
      "-- filtering region:263, screen genome to keep 400 probes\n",
      "-- filtering region:264, screen genome to keep 400 probes\n",
      "-- filtering region:265, screen genome to keep 400 probes\n",
      "-- filtering region:266, screen genome to keep 400 probes\n",
      "-- filtering region:267, screen genome to keep 400 probes\n",
      "-- filtering region:268, screen genome to keep 400 probes\n",
      "-- filtering region:269, screen genome to keep 400 probes\n",
      "-- filtering region:270, screen genome to keep 400 probes\n",
      "-- filtering region:271, screen genome to keep 400 probes\n",
      "-- filtering region:272, screen genome to keep 400 probes\n",
      "-- filtering region:273, screen genome to keep 400 probes\n",
      "-- filtering region:274, screen genome to keep 400 probes\n",
      "-- filtering region:275, screen genome to keep 400 probes\n",
      "-- filtering region:277, screen genome to keep 400 probes\n",
      "-- filtering region:278, screen genome to keep 400 probes\n",
      "-- filtering region:279, screen genome to keep 400 probes\n",
      "-- filtering region:280, screen genome to keep 400 probes\n",
      "-- filtering region:281, screen genome to keep 400 probes\n",
      "-- filtering region:282, screen genome to keep 400 probes\n",
      "-- filtering region:283, screen genome to keep 400 probes\n",
      "-- filtering region:284, screen genome to keep 400 probes\n",
      "-- filtering region:285, screen genome to keep 400 probes\n",
      "-- filtering region:286, screen genome to keep 400 probes\n",
      "-- filtering region:287, screen genome to keep 400 probes\n",
      "-- filtering region:288, screen genome to keep 400 probes\n",
      "-- filtering region:289, screen genome to keep 400 probes\n",
      "-- filtering region:290, screen genome to keep 400 probes\n",
      "-- filtering region:291, screen genome to keep 400 probes\n",
      "-- filtering region:292, screen genome to keep 400 probes\n",
      "-- filtering region:293, screen genome to keep 400 probes\n",
      "-- filtering region:295, screen genome to keep 400 probes\n",
      "-- filtering region:296, screen genome to keep 400 probes\n",
      "-- filtering region:297, screen genome to keep 400 probes\n",
      "-- filtering region:298, screen genome to keep 400 probes\n",
      "-- filtering region:299, screen genome to keep 400 probes\n",
      "-- filtering region:300, screen genome to keep 400 probes\n",
      "-- filtering region:301, screen genome to keep 400 probes\n",
      "-- filtering region:302, screen genome to keep 400 probes\n",
      "-- filtering region:303, screen genome to keep 400 probes\n",
      "-- filtering region:304, screen genome to keep 400 probes\n",
      "-- filtering region:305, screen genome to keep 400 probes\n",
      "-- filtering region:306, screen genome to keep 400 probes\n",
      "-- filtering region:307, screen genome to keep 400 probes\n",
      "-- filtering region:308, screen genome to keep 400 probes\n",
      "-- filtering region:309, screen genome to keep 400 probes\n",
      "-- filtering region:310, screen genome to keep 400 probes\n",
      "-- filtering region:311, screen genome to keep 400 probes\n",
      "-- filtering region:312, screen genome to keep 400 probes\n",
      "-- filtering region:313, screen genome to keep 400 probes\n",
      "-- filtering region:314, screen genome to keep 400 probes\n",
      "-- filtering region:315, screen genome to keep 400 probes\n",
      "-- filtering region:316, screen genome to keep 400 probes\n",
      "-- filtering region:317, screen genome to keep 400 probes\n",
      "-- filtering region:318, screen genome to keep 400 probes\n",
      "-- filtering region:319, screen genome to keep 400 probes\n",
      "-- filtering region:320, screen genome to keep 400 probes\n",
      "-- filtering region:321, screen genome to keep 400 probes\n",
      "-- filtering region:322, screen genome to keep 400 probes\n",
      "-- filtering region:323, directly append 38 probes\n",
      "-- filtering region:324, screen genome to keep 400 probes\n",
      "-- filtering region:325, screen genome to keep 400 probes\n",
      "-- filtering region:326, screen genome to keep 400 probes\n",
      "-- filtering region:327, screen genome to keep 400 probes\n",
      "-- filtering region:328, screen genome to keep 400 probes\n",
      "-- filtering region:329, screen genome to keep 400 probes\n",
      "-- filtering region:330, screen genome to keep 400 probes\n",
      "-- filtering region:332, screen genome to keep 400 probes\n",
      "-- filtering region:333, screen genome to keep 400 probes\n",
      "-- filtering region:334, screen genome to keep 400 probes\n",
      "-- filtering region:335, screen genome to keep 400 probes\n",
      "-- filtering region:336, screen genome to keep 400 probes\n",
      "-- filtering region:337, screen genome to keep 400 probes\n",
      "-- filtering region:338, screen genome to keep 400 probes\n",
      "-- filtering region:339, screen genome to keep 400 probes\n",
      "-- filtering region:340, screen genome to keep 400 probes\n",
      "-- filtering region:341, screen genome to keep 400 probes\n",
      "-- filtering region:342, screen genome to keep 400 probes\n",
      "-- filtering region:343, screen genome to keep 400 probes\n",
      "-- filtering region:344, screen genome to keep 400 probes\n",
      "-- filtering region:345, screen genome to keep 400 probes\n",
      "-- filtering region:347, screen genome to keep 400 probes\n",
      "-- filtering region:348, screen genome to keep 400 probes\n",
      "-- filtering region:349, screen genome to keep 400 probes\n",
      "-- filtering region:350, screen genome to keep 400 probes\n",
      "-- filtering region:351, screen genome to keep 400 probes\n",
      "-- filtering region:352, screen genome to keep 400 probes\n",
      "-- filtering region:353, screen genome to keep 400 probes\n",
      "-- filtering region:354, screen genome to keep 400 probes\n",
      "-- filtering region:355, screen genome to keep 400 probes\n",
      "-- filtering region:356, screen genome to keep 400 probes\n",
      "-- filtering region:357, screen genome to keep 400 probes\n",
      "-- filtering region:358, screen genome to keep 400 probes\n",
      "-- filtering region:359, screen genome to keep 400 probes\n",
      "-- filtering region:360, screen genome to keep 400 probes\n",
      "-- filtering region:361, screen genome to keep 400 probes\n",
      "-- filtering region:362, directly append 320 probes\n",
      "-- filtering region:363, screen genome to keep 400 probes\n",
      "-- filtering region:364, screen genome to keep 400 probes\n",
      "-- filtering region:365, screen genome to keep 400 probes\n",
      "-- filtering region:366, screen genome to keep 400 probes\n",
      "-- filtering region:367, screen genome to keep 400 probes\n",
      "-- filtering region:368, screen genome to keep 400 probes\n",
      "-- filtering region:369, screen genome to keep 400 probes\n",
      "-- filtering region:370, screen genome to keep 400 probes\n",
      "-- filtering region:371, screen genome to keep 400 probes\n",
      "-- filtering region:372, screen genome to keep 400 probes\n",
      "-- filtering region:373, screen genome to keep 400 probes\n",
      "-- filtering region:374, screen genome to keep 400 probes\n",
      "-- filtering region:375, screen genome to keep 400 probes\n",
      "-- filtering region:376, screen genome to keep 400 probes\n",
      "-- filtering region:377, screen genome to keep 400 probes\n",
      "-- filtering region:378, screen genome to keep 400 probes\n",
      "-- filtering region:379, screen genome to keep 400 probes\n",
      "-- filtering region:380, screen genome to keep 400 probes\n",
      "-- filtering region:381, screen genome to keep 400 probes\n",
      "-- filtering region:382, screen genome to keep 400 probes\n",
      "-- filtering region:383, screen genome to keep 400 probes\n",
      "-- filtering region:384, screen genome to keep 400 probes\n",
      "-- filtering region:385, screen genome to keep 400 probes\n",
      "-- filtering region:386, screen genome to keep 400 probes\n",
      "-- filtering region:387, screen genome to keep 400 probes\n",
      "-- filtering region:388, screen genome to keep 400 probes\n",
      "-- filtering region:389, screen genome to keep 400 probes\n",
      "-- filtering region:390, screen genome to keep 400 probes\n",
      "-- filtering region:391, screen genome to keep 400 probes\n",
      "-- filtering region:392, screen genome to keep 400 probes\n",
      "-- filtering region:393, screen genome to keep 400 probes\n",
      "-- filtering region:394, screen genome to keep 400 probes\n",
      "-- filtering region:395, screen genome to keep 400 probes\n",
      "-- filtering region:396, screen genome to keep 400 probes\n",
      "-- filtering region:397, screen genome to keep 400 probes\n",
      "-- filtering region:398, screen genome to keep 400 probes\n",
      "-- filtering region:399, screen genome to keep 400 probes\n",
      "-- filtering region:400, screen genome to keep 400 probes\n",
      "-- filtering region:401, screen genome to keep 400 probes\n",
      "-- filtering region:402, directly append 73 probes\n",
      "-- filtering region:403, screen genome to keep 400 probes\n",
      "-- filtering region:404, screen genome to keep 400 probes\n",
      "-- filtering region:405, screen genome to keep 400 probes\n",
      "-- filtering region:406, screen genome to keep 400 probes\n",
      "-- filtering region:407, screen genome to keep 400 probes\n",
      "-- filtering region:408, screen genome to keep 400 probes\n",
      "-- filtering region:409, screen genome to keep 400 probes\n",
      "-- filtering region:410, screen genome to keep 400 probes\n",
      "-- filtering region:411, screen genome to keep 400 probes\n",
      "-- filtering region:412, screen genome to keep 400 probes\n",
      "-- filtering region:413, screen genome to keep 400 probes\n",
      "-- filtering region:414, screen genome to keep 400 probes\n",
      "-- filtering region:415, screen genome to keep 400 probes\n",
      "-- filtering region:416, screen genome to keep 400 probes\n",
      "-- filtering region:417, screen genome to keep 400 probes\n",
      "-- filtering region:418, directly append 378 probes\n",
      "-- filtering region:419, screen genome to keep 400 probes\n",
      "-- filtering region:420, screen genome to keep 400 probes\n",
      "-- filtering region:421, screen genome to keep 400 probes\n",
      "-- filtering region:422, screen genome to keep 400 probes\n",
      "-- filtering region:423, screen genome to keep 400 probes\n",
      "-- filtering region:424, screen genome to keep 400 probes\n",
      "-- filtering region:425, screen genome to keep 400 probes\n",
      "-- filtering region:426, directly append 313 probes\n",
      "-- filtering region:427, screen genome to keep 400 probes\n",
      "-- filtering region:428, screen genome to keep 400 probes\n",
      "-- filtering region:429, screen genome to keep 400 probes\n",
      "-- filtering region:430, screen genome to keep 400 probes\n",
      "-- filtering region:431, screen genome to keep 400 probes\n",
      "-- filtering region:432, screen genome to keep 400 probes\n",
      "-- filtering region:433, screen genome to keep 400 probes\n",
      "-- filtering region:434, screen genome to keep 400 probes\n",
      "-- filtering region:435, screen genome to keep 400 probes\n",
      "-- filtering region:436, screen genome to keep 400 probes\n",
      "-- filtering region:437, screen genome to keep 400 probes\n",
      "-- filtering region:438, screen genome to keep 400 probes\n",
      "-- filtering region:439, screen genome to keep 400 probes\n",
      "-- filtering region:440, screen genome to keep 400 probes\n",
      "-- filtering region:441, screen genome to keep 400 probes\n",
      "-- filtering region:442, screen genome to keep 400 probes\n",
      "-- filtering region:443, directly append 3 probes\n",
      "-- filtering region:444, screen genome to keep 400 probes\n",
      "-- filtering region:445, screen genome to keep 400 probes\n",
      "-- filtering region:446, screen genome to keep 400 probes\n",
      "-- filtering region:447, screen genome to keep 400 probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- filtering region:448, screen genome to keep 400 probes\n",
      "-- filtering region:449, screen genome to keep 400 probes\n",
      "-- filtering region:450, screen genome to keep 400 probes\n",
      "-- filtering region:451, screen genome to keep 400 probes\n",
      "-- filtering region:452, screen genome to keep 400 probes\n",
      "-- filtering region:453, screen genome to keep 400 probes\n",
      "-- filtering region:454, screen genome to keep 400 probes\n",
      "-- filtering region:455, screen genome to keep 400 probes\n",
      "-- filtering region:456, screen genome to keep 400 probes\n",
      "-- filtering region:457, screen genome to keep 400 probes\n",
      "-- filtering region:458, screen genome to keep 400 probes\n",
      "-- filtering region:459, screen genome to keep 400 probes\n",
      "-- filtering region:460, screen genome to keep 400 probes\n",
      "-- filtering region:461, screen genome to keep 400 probes\n",
      "-- filtering region:462, screen genome to keep 400 probes\n",
      "-- filtering region:463, screen genome to keep 400 probes\n",
      "-- filtering region:464, screen genome to keep 400 probes\n",
      "-- filtering region:465, screen genome to keep 400 probes\n",
      "-- filtering region:466, screen genome to keep 400 probes\n",
      "-- filtering region:467, screen genome to keep 400 probes\n",
      "-- filtering region:468, screen genome to keep 400 probes\n",
      "-- filtering region:469, screen genome to keep 400 probes\n",
      "-- filtering region:470, screen genome to keep 400 probes\n",
      "-- filtering region:471, screen genome to keep 400 probes\n",
      "-- filtering region:472, screen genome to keep 400 probes\n",
      "-- filtering region:473, screen genome to keep 400 probes\n",
      "-- filtering region:474, screen genome to keep 400 probes\n",
      "-- filtering region:475, screen genome to keep 400 probes\n",
      "-- filtering region:476, screen genome to keep 400 probes\n",
      "-- filtering region:477, screen genome to keep 400 probes\n",
      "-- filtering region:478, screen genome to keep 400 probes\n",
      "-- filtering region:479, screen genome to keep 400 probes\n",
      "-- filtering region:480, screen genome to keep 400 probes\n",
      "-- filtering region:481, screen genome to keep 400 probes\n",
      "-- filtering region:482, screen genome to keep 400 probes\n",
      "-- filtering region:483, screen genome to keep 400 probes\n",
      "-- filtering region:484, screen genome to keep 400 probes\n",
      "-- filtering region:485, screen genome to keep 400 probes\n",
      "-- filtering region:486, screen genome to keep 400 probes\n",
      "-- filtering region:487, screen genome to keep 400 probes\n",
      "-- filtering region:488, screen genome to keep 400 probes\n",
      "-- filtering region:489, screen genome to keep 400 probes\n",
      "-- filtering region:490, directly append 19 probes\n",
      "-- filtering region:491, screen genome to keep 400 probes\n",
      "-- filtering region:492, screen genome to keep 400 probes\n",
      "-- filtering region:493, screen genome to keep 400 probes\n",
      "-- filtering region:494, screen genome to keep 400 probes\n",
      "-- filtering region:495, screen genome to keep 400 probes\n",
      "-- filtering region:496, screen genome to keep 400 probes\n",
      "-- filtering region:497, screen genome to keep 400 probes\n",
      "-- filtering region:498, screen genome to keep 400 probes\n",
      "-- filtering region:499, screen genome to keep 400 probes\n",
      "-- filtering region:500, screen genome to keep 400 probes\n",
      "-- filtering region:501, screen genome to keep 400 probes\n",
      "-- filtering region:502, screen genome to keep 400 probes\n",
      "-- filtering region:503, screen genome to keep 400 probes\n",
      "-- filtering region:504, screen genome to keep 400 probes\n",
      "-- filtering region:505, screen genome to keep 400 probes\n",
      "-- filtering region:506, screen genome to keep 400 probes\n",
      "-- filtering region:507, screen genome to keep 400 probes\n",
      "-- filtering region:508, screen genome to keep 400 probes\n",
      "-- filtering region:509, screen genome to keep 400 probes\n",
      "-- filtering region:510, screen genome to keep 400 probes\n",
      "-- filtering region:511, screen genome to keep 400 probes\n",
      "-- filtering region:512, screen genome to keep 400 probes\n",
      "-- filtering region:513, screen genome to keep 400 probes\n",
      "-- filtering region:514, screen genome to keep 400 probes\n",
      "-- filtering region:515, screen genome to keep 400 probes\n",
      "-- filtering region:516, screen genome to keep 400 probes\n",
      "-- filtering region:517, screen genome to keep 400 probes\n",
      "-- filtering region:518, screen genome to keep 400 probes\n",
      "-- filtering region:519, screen genome to keep 400 probes\n",
      "-- filtering region:520, screen genome to keep 400 probes\n",
      "-- filtering region:521, screen genome to keep 400 probes\n",
      "-- filtering region:523, screen genome to keep 400 probes\n",
      "-- filtering region:524, screen genome to keep 400 probes\n",
      "-- filtering region:525, screen genome to keep 400 probes\n",
      "-- filtering region:526, screen genome to keep 400 probes\n",
      "-- filtering region:527, screen genome to keep 400 probes\n",
      "-- filtering region:528, screen genome to keep 400 probes\n",
      "-- filtering region:529, screen genome to keep 400 probes\n",
      "-- filtering region:530, screen genome to keep 400 probes\n",
      "-- filtering region:531, screen genome to keep 400 probes\n",
      "-- filtering region:532, screen genome to keep 400 probes\n",
      "-- filtering region:533, screen genome to keep 400 probes\n",
      "-- filtering region:534, screen genome to keep 400 probes\n",
      "-- filtering region:535, screen genome to keep 400 probes\n",
      "-- filtering region:536, screen genome to keep 400 probes\n",
      "-- filtering region:537, screen genome to keep 400 probes\n",
      "-- filtering region:538, screen genome to keep 400 probes\n",
      "-- filtering region:539, directly append 268 probes\n",
      "-- filtering region:540, screen genome to keep 400 probes\n",
      "-- filtering region:541, screen genome to keep 400 probes\n",
      "-- filtering region:542, screen genome to keep 400 probes\n",
      "-- filtering region:543, screen genome to keep 400 probes\n",
      "-- filtering region:544, screen genome to keep 400 probes\n",
      "-- filtering region:545, screen genome to keep 400 probes\n",
      "-- filtering region:546, screen genome to keep 400 probes\n",
      "-- filtering region:547, screen genome to keep 400 probes\n",
      "-- filtering region:548, screen genome to keep 400 probes\n",
      "-- filtering region:549, screen genome to keep 400 probes\n",
      "-- filtering region:550, screen genome to keep 400 probes\n",
      "-- filtering region:551, screen genome to keep 400 probes\n",
      "-- filtering region:552, screen genome to keep 400 probes\n",
      "-- filtering region:553, screen genome to keep 400 probes\n",
      "-- filtering region:554, screen genome to keep 400 probes\n",
      "-- filtering region:555, screen genome to keep 400 probes\n",
      "-- filtering region:556, screen genome to keep 400 probes\n",
      "-- filtering region:557, screen genome to keep 400 probes\n",
      "-- filtering region:558, screen genome to keep 400 probes\n",
      "-- filtering region:559, screen genome to keep 400 probes\n",
      "-- filtering region:560, screen genome to keep 400 probes\n",
      "-- filtering region:561, screen genome to keep 400 probes\n",
      "-- filtering region:562, screen genome to keep 400 probes\n",
      "-- filtering region:563, screen genome to keep 400 probes\n",
      "-- filtering region:564, screen genome to keep 400 probes\n",
      "-- filtering region:565, screen genome to keep 400 probes\n",
      "-- filtering region:566, screen genome to keep 400 probes\n",
      "-- filtering region:567, screen genome to keep 400 probes\n",
      "-- filtering region:568, screen genome to keep 400 probes\n",
      "-- filtering region:569, screen genome to keep 400 probes\n",
      "-- filtering region:570, screen genome to keep 400 probes\n",
      "-- filtering region:571, screen genome to keep 400 probes\n",
      "-- filtering region:572, screen genome to keep 400 probes\n",
      "-- filtering region:573, screen genome to keep 400 probes\n",
      "-- filtering region:574, screen genome to keep 400 probes\n",
      "-- filtering region:575, screen genome to keep 400 probes\n",
      "-- filtering region:576, screen genome to keep 400 probes\n",
      "-- filtering region:577, screen genome to keep 400 probes\n",
      "-- filtering region:579, screen genome to keep 400 probes\n",
      "-- filtering region:580, screen genome to keep 400 probes\n",
      "-- filtering region:581, screen genome to keep 400 probes\n",
      "-- filtering region:582, screen genome to keep 400 probes\n",
      "-- filtering region:583, screen genome to keep 400 probes\n",
      "-- filtering region:584, screen genome to keep 400 probes\n",
      "-- filtering region:585, screen genome to keep 400 probes\n",
      "-- filtering region:586, screen genome to keep 400 probes\n",
      "-- filtering region:587, screen genome to keep 400 probes\n",
      "-- filtering region:588, screen genome to keep 400 probes\n",
      "-- filtering region:589, screen genome to keep 400 probes\n",
      "-- filtering region:590, screen genome to keep 400 probes\n",
      "-- filtering region:591, screen genome to keep 400 probes\n",
      "-- filtering region:592, screen genome to keep 400 probes\n",
      "-- filtering region:593, screen genome to keep 400 probes\n",
      "-- filtering region:594, screen genome to keep 400 probes\n",
      "-- filtering region:595, screen genome to keep 400 probes\n",
      "-- filtering region:596, screen genome to keep 400 probes\n",
      "-- filtering region:597, screen genome to keep 400 probes\n",
      "-- filtering region:598, screen genome to keep 400 probes\n",
      "-- filtering region:599, screen genome to keep 400 probes\n",
      "-- filtering region:600, screen genome to keep 400 probes\n",
      "-- filtering region:601, screen genome to keep 400 probes\n",
      "-- filtering region:602, screen genome to keep 400 probes\n",
      "-- filtering region:603, screen genome to keep 400 probes\n",
      "-- filtering region:604, screen genome to keep 400 probes\n",
      "-- filtering region:605, screen genome to keep 400 probes\n",
      "-- filtering region:606, screen genome to keep 400 probes\n",
      "-- filtering region:607, screen genome to keep 400 probes\n",
      "-- filtering region:608, screen genome to keep 400 probes\n",
      "-- filtering region:609, screen genome to keep 400 probes\n",
      "-- filtering region:610, screen genome to keep 400 probes\n",
      "-- filtering region:611, screen genome to keep 400 probes\n",
      "-- filtering region:612, screen genome to keep 400 probes\n",
      "-- filtering region:613, screen genome to keep 400 probes\n",
      "-- filtering region:614, screen genome to keep 400 probes\n",
      "-- filtering region:615, screen genome to keep 400 probes\n",
      "-- filtering region:616, screen genome to keep 400 probes\n",
      "-- filtering region:617, screen genome to keep 400 probes\n",
      "-- filtering region:618, screen genome to keep 400 probes\n",
      "-- filtering region:619, screen genome to keep 400 probes\n",
      "-- filtering region:620, screen genome to keep 400 probes\n",
      "-- filtering region:621, screen genome to keep 400 probes\n",
      "-- filtering region:622, screen genome to keep 400 probes\n",
      "-- filtering region:623, screen genome to keep 400 probes\n",
      "-- filtering region:624, screen genome to keep 400 probes\n",
      "-- filtering region:625, screen genome to keep 400 probes\n",
      "-- filtering region:626, screen genome to keep 400 probes\n",
      "-- filtering region:627, screen genome to keep 400 probes\n",
      "-- filtering region:628, screen genome to keep 400 probes\n",
      "-- filtering region:629, screen genome to keep 400 probes\n",
      "-- filtering region:630, screen genome to keep 400 probes\n",
      "-- filtering region:631, screen genome to keep 400 probes\n",
      "-- filtering region:632, screen genome to keep 400 probes\n",
      "-- filtering region:633, screen genome to keep 400 probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- filtering region:634, screen genome to keep 400 probes\n",
      "-- filtering region:635, screen genome to keep 400 probes\n",
      "-- filtering region:636, screen genome to keep 400 probes\n",
      "-- filtering region:637, screen genome to keep 400 probes\n",
      "-- filtering region:638, screen genome to keep 400 probes\n",
      "-- filtering region:639, screen genome to keep 400 probes\n",
      "-- filtering region:640, screen genome to keep 400 probes\n",
      "-- filtering region:641, screen genome to keep 400 probes\n",
      "-- filtering region:642, screen genome to keep 400 probes\n",
      "-- filtering region:643, screen genome to keep 400 probes\n",
      "-- filtering region:644, screen genome to keep 400 probes\n",
      "-- filtering region:645, screen genome to keep 400 probes\n",
      "-- filtering region:646, screen genome to keep 400 probes\n",
      "-- filtering region:647, screen genome to keep 400 probes\n",
      "-- filtering region:648, screen genome to keep 400 probes\n",
      "-- filtering region:649, screen genome to keep 400 probes\n",
      "-- filtering region:650, screen genome to keep 400 probes\n",
      "-- filtering region:651, screen genome to keep 400 probes\n",
      "-- filtering region:652, screen genome to keep 400 probes\n",
      "-- filtering region:653, screen genome to keep 400 probes\n",
      "-- filtering region:654, screen genome to keep 400 probes\n",
      "-- filtering region:655, screen genome to keep 400 probes\n",
      "-- filtering region:656, screen genome to keep 400 probes\n",
      "-- filtering region:657, screen genome to keep 400 probes\n",
      "-- filtering region:658, screen genome to keep 400 probes\n",
      "-- filtering region:659, screen genome to keep 400 probes\n",
      "-- filtering region:660, screen genome to keep 400 probes\n",
      "-- filtering region:661, screen genome to keep 400 probes\n",
      "-- filtering region:662, screen genome to keep 400 probes\n",
      "-- filtering region:663, screen genome to keep 400 probes\n",
      "-- filtering region:664, screen genome to keep 400 probes\n",
      "-- filtering region:665, screen genome to keep 400 probes\n",
      "-- filtering region:666, screen genome to keep 400 probes\n",
      "-- filtering region:667, screen genome to keep 400 probes\n",
      "-- filtering region:668, screen genome to keep 400 probes\n",
      "-- filtering region:669, screen genome to keep 400 probes\n",
      "-- filtering region:670, screen genome to keep 400 probes\n",
      "-- filtering region:671, screen genome to keep 400 probes\n",
      "-- filtering region:672, screen genome to keep 400 probes\n",
      "-- filtering region:673, screen genome to keep 400 probes\n",
      "-- filtering region:674, screen genome to keep 400 probes\n",
      "-- filtering region:675, screen genome to keep 400 probes\n",
      "-- filtering region:676, screen genome to keep 400 probes\n",
      "-- filtering region:677, screen genome to keep 400 probes\n",
      "-- filtering region:678, screen genome to keep 400 probes\n",
      "-- filtering region:679, screen genome to keep 400 probes\n",
      "-- filtering region:680, screen genome to keep 400 probes\n",
      "-- filtering region:681, screen genome to keep 400 probes\n",
      "-- filtering region:682, screen genome to keep 400 probes\n",
      "-- filtering region:683, screen genome to keep 400 probes\n",
      "-- filtering region:684, screen genome to keep 400 probes\n",
      "-- filtering region:685, screen genome to keep 400 probes\n",
      "-- filtering region:687, screen genome to keep 400 probes\n",
      "-- filtering region:688, screen genome to keep 400 probes\n",
      "-- filtering region:689, screen genome to keep 400 probes\n",
      "-- filtering region:690, screen genome to keep 400 probes\n",
      "-- filtering region:691, screen genome to keep 400 probes\n",
      "-- filtering region:692, screen genome to keep 400 probes\n",
      "-- filtering region:693, screen genome to keep 400 probes\n",
      "-- filtering region:694, screen genome to keep 400 probes\n",
      "-- filtering region:695, screen genome to keep 400 probes\n",
      "-- filtering region:696, screen genome to keep 400 probes\n",
      "-- filtering region:697, screen genome to keep 400 probes\n",
      "-- filtering region:698, screen genome to keep 400 probes\n",
      "-- filtering region:699, screen genome to keep 400 probes\n",
      "-- filtering region:700, screen genome to keep 400 probes\n",
      "-- filtering region:701, screen genome to keep 400 probes\n",
      "-- filtering region:702, screen genome to keep 400 probes\n",
      "-- filtering region:703, screen genome to keep 400 probes\n",
      "-- filtering region:704, screen genome to keep 400 probes\n",
      "-- filtering region:705, screen genome to keep 400 probes\n",
      "-- filtering region:706, screen genome to keep 400 probes\n",
      "-- filtering region:707, screen genome to keep 400 probes\n",
      "-- filtering region:708, screen genome to keep 400 probes\n",
      "-- filtering region:709, screen genome to keep 400 probes\n",
      "-- filtering region:710, screen genome to keep 400 probes\n",
      "-- filtering region:711, screen genome to keep 400 probes\n",
      "-- filtering region:712, screen genome to keep 400 probes\n",
      "-- filtering region:713, screen genome to keep 400 probes\n",
      "-- filtering region:714, screen genome to keep 400 probes\n",
      "-- filtering region:715, screen genome to keep 400 probes\n",
      "-- filtering region:716, screen genome to keep 400 probes\n",
      "-- filtering region:717, screen genome to keep 400 probes\n",
      "-- filtering region:718, screen genome to keep 400 probes\n",
      "-- filtering region:719, screen genome to keep 400 probes\n",
      "-- filtering region:720, screen genome to keep 400 probes\n",
      "-- filtering region:721, screen genome to keep 400 probes\n",
      "-- filtering region:722, screen genome to keep 400 probes\n",
      "-- filtering region:723, screen genome to keep 400 probes\n",
      "-- filtering region:724, screen genome to keep 400 probes\n",
      "-- filtering region:725, screen genome to keep 400 probes\n",
      "-- filtering region:726, screen genome to keep 400 probes\n",
      "-- filtering region:727, screen genome to keep 400 probes\n",
      "-- filtering region:728, screen genome to keep 400 probes\n",
      "-- filtering region:729, screen genome to keep 400 probes\n",
      "-- filtering region:730, screen genome to keep 400 probes\n",
      "-- filtering region:731, screen genome to keep 400 probes\n",
      "-- filtering region:732, screen genome to keep 400 probes\n",
      "-- filtering region:733, screen genome to keep 400 probes\n",
      "-- filtering region:734, screen genome to keep 400 probes\n",
      "-- filtering region:735, screen genome to keep 400 probes\n",
      "-- filtering region:736, screen genome to keep 400 probes\n",
      "-- filtering region:737, screen genome to keep 400 probes\n",
      "-- filtering region:738, screen genome to keep 400 probes\n",
      "-- filtering region:739, screen genome to keep 400 probes\n",
      "-- filtering region:740, screen genome to keep 400 probes\n",
      "-- filtering region:741, screen genome to keep 400 probes\n",
      "-- filtering region:742, screen genome to keep 400 probes\n",
      "-- filtering region:743, screen genome to keep 400 probes\n",
      "-- filtering region:744, screen genome to keep 400 probes\n",
      "-- filtering region:745, screen genome to keep 400 probes\n",
      "-- filtering region:746, screen genome to keep 400 probes\n",
      "-- filtering region:747, screen genome to keep 400 probes\n",
      "-- filtering region:748, screen genome to keep 400 probes\n",
      "-- filtering region:749, screen genome to keep 400 probes\n",
      "-- filtering region:750, screen genome to keep 400 probes\n",
      "-- filtering region:1, screen genome to keep 400 probes\n",
      "-- filtering region:2, screen genome to keep 400 probes\n",
      "-- filtering region:3, screen genome to keep 400 probes\n",
      "-- filtering region:4, screen genome to keep 400 probes\n",
      "-- filtering region:5, screen genome to keep 400 probes\n",
      "-- filtering region:6, screen genome to keep 400 probes\n",
      "-- filtering region:7, screen genome to keep 400 probes\n",
      "-- filtering region:8, screen genome to keep 400 probes\n",
      "-- filtering region:9, screen genome to keep 400 probes\n",
      "-- filtering region:10, screen genome to keep 400 probes\n",
      "-- filtering region:11, screen genome to keep 400 probes\n",
      "-- filtering region:12, screen genome to keep 400 probes\n",
      "-- filtering region:13, screen genome to keep 400 probes\n",
      "-- filtering region:14, screen genome to keep 400 probes\n",
      "-- filtering region:15, screen genome to keep 400 probes\n",
      "-- filtering region:16, screen genome to keep 400 probes\n",
      "-- filtering region:17, screen genome to keep 400 probes\n",
      "-- filtering region:18, screen genome to keep 400 probes\n",
      "-- filtering region:19, screen genome to keep 400 probes\n",
      "-- filtering region:20, screen genome to keep 400 probes\n",
      "-- filtering region:21, screen genome to keep 400 probes\n",
      "-- filtering region:22, screen genome to keep 400 probes\n",
      "-- filtering region:23, screen genome to keep 400 probes\n",
      "-- filtering region:24, screen genome to keep 400 probes\n",
      "-- filtering region:25, screen genome to keep 400 probes\n",
      "-- filtering region:26, screen genome to keep 400 probes\n",
      "-- filtering region:27, screen genome to keep 400 probes\n",
      "-- filtering region:28, screen genome to keep 400 probes\n",
      "-- filtering region:29, screen genome to keep 400 probes\n",
      "-- filtering region:30, screen genome to keep 400 probes\n",
      "-- filtering region:31, screen genome to keep 400 probes\n",
      "-- filtering region:32, screen genome to keep 400 probes\n",
      "-- filtering region:33, screen genome to keep 400 probes\n",
      "-- filtering region:34, screen genome to keep 400 probes\n",
      "-- filtering region:35, screen genome to keep 400 probes\n",
      "-- filtering region:36, screen genome to keep 400 probes\n",
      "-- filtering region:37, screen genome to keep 400 probes\n",
      "-- filtering region:38, screen genome to keep 400 probes\n",
      "-- filtering region:39, screen genome to keep 400 probes\n",
      "-- filtering region:40, screen genome to keep 400 probes\n",
      "-- filtering region:41, screen genome to keep 400 probes\n",
      "-- filtering region:42, screen genome to keep 400 probes\n",
      "-- filtering region:43, screen genome to keep 400 probes\n",
      "-- filtering region:44, screen genome to keep 400 probes\n",
      "-- filtering region:45, screen genome to keep 400 probes\n",
      "-- filtering region:46, screen genome to keep 400 probes\n",
      "-- filtering region:47, screen genome to keep 400 probes\n",
      "-- filtering region:48, screen genome to keep 400 probes\n",
      "-- filtering region:49, screen genome to keep 400 probes\n",
      "-- filtering region:50, screen genome to keep 400 probes\n",
      "-- filtering region:51, screen genome to keep 400 probes\n",
      "-- filtering region:52, screen genome to keep 400 probes\n",
      "-- filtering region:53, screen genome to keep 400 probes\n",
      "-- filtering region:54, screen genome to keep 400 probes\n",
      "-- filtering region:55, screen genome to keep 400 probes\n",
      "-- filtering region:56, screen genome to keep 400 probes\n",
      "-- filtering region:57, screen genome to keep 400 probes\n",
      "-- filtering region:58, screen genome to keep 400 probes\n",
      "-- filtering region:59, screen genome to keep 400 probes\n",
      "-- filtering region:60, screen genome to keep 400 probes\n",
      "-- filtering region:61, screen genome to keep 400 probes\n",
      "-- filtering region:62, screen genome to keep 400 probes\n",
      "-- filtering region:63, screen genome to keep 400 probes\n",
      "-- filtering region:64, screen genome to keep 400 probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- filtering region:65, screen genome to keep 400 probes\n",
      "-- filtering region:66, screen genome to keep 400 probes\n",
      "-- filtering region:67, screen genome to keep 400 probes\n",
      "-- filtering region:68, screen genome to keep 400 probes\n",
      "-- filtering region:69, screen genome to keep 400 probes\n",
      "-- filtering region:70, screen genome to keep 400 probes\n",
      "-- filtering region:71, screen genome to keep 400 probes\n",
      "-- filtering region:72, screen genome to keep 400 probes\n",
      "-- filtering region:73, screen genome to keep 400 probes\n",
      "-- filtering region:74, screen genome to keep 400 probes\n",
      "-- filtering region:75, screen genome to keep 400 probes\n",
      "-- filtering region:76, screen genome to keep 400 probes\n",
      "-- filtering region:77, screen genome to keep 400 probes\n",
      "-- filtering region:78, screen genome to keep 400 probes\n",
      "-- filtering region:79, screen genome to keep 400 probes\n",
      "-- filtering region:80, screen genome to keep 400 probes\n",
      "-- filtering region:81, screen genome to keep 400 probes\n",
      "-- filtering region:82, screen genome to keep 400 probes\n",
      "-- filtering region:83, screen genome to keep 400 probes\n",
      "-- filtering region:84, screen genome to keep 400 probes\n",
      "-- filtering region:85, screen genome to keep 400 probes\n",
      "-- filtering region:86, screen genome to keep 400 probes\n",
      "-- filtering region:87, screen genome to keep 400 probes\n",
      "-- filtering region:88, screen genome to keep 400 probes\n",
      "-- filtering region:89, screen genome to keep 400 probes\n",
      "-- filtering region:90, screen genome to keep 400 probes\n",
      "-- filtering region:91, screen genome to keep 400 probes\n",
      "-- filtering region:92, screen genome to keep 400 probes\n",
      "-- filtering region:93, screen genome to keep 400 probes\n",
      "-- filtering region:94, screen genome to keep 400 probes\n",
      "-- filtering region:95, screen genome to keep 400 probes\n",
      "-- filtering region:96, screen genome to keep 400 probes\n",
      "-- filtering region:97, screen genome to keep 400 probes\n",
      "-- filtering region:98, screen genome to keep 400 probes\n",
      "-- filtering region:99, screen genome to keep 400 probes\n",
      "-- filtering region:100, screen genome to keep 400 probes\n",
      "-- filtering region:101, screen genome to keep 400 probes\n",
      "-- filtering region:102, screen genome to keep 400 probes\n",
      "-- filtering region:103, screen genome to keep 400 probes\n",
      "-- filtering region:104, screen genome to keep 400 probes\n",
      "-- filtering region:105, screen genome to keep 400 probes\n",
      "-- filtering region:106, screen genome to keep 400 probes\n",
      "-- filtering region:107, screen genome to keep 400 probes\n",
      "-- filtering region:108, screen genome to keep 400 probes\n",
      "-- filtering region:109, screen genome to keep 400 probes\n",
      "-- filtering region:111, screen genome to keep 400 probes\n",
      "-- filtering region:112, screen genome to keep 400 probes\n",
      "-- filtering region:113, screen genome to keep 400 probes\n",
      "-- filtering region:114, screen genome to keep 400 probes\n",
      "-- filtering region:115, screen genome to keep 400 probes\n",
      "-- filtering region:116, screen genome to keep 400 probes\n",
      "-- filtering region:117, screen genome to keep 400 probes\n",
      "-- filtering region:118, screen genome to keep 400 probes\n",
      "-- filtering region:119, screen genome to keep 400 probes\n",
      "-- filtering region:120, screen genome to keep 400 probes\n",
      "-- filtering region:121, screen genome to keep 400 probes\n",
      "-- filtering region:122, screen genome to keep 400 probes\n",
      "-- filtering region:123, screen genome to keep 400 probes\n",
      "-- filtering region:124, screen genome to keep 400 probes\n",
      "-- filtering region:125, screen genome to keep 400 probes\n",
      "-- filtering region:126, screen genome to keep 400 probes\n",
      "-- filtering region:127, screen genome to keep 400 probes\n",
      "-- filtering region:129, screen genome to keep 400 probes\n",
      "-- filtering region:130, screen genome to keep 400 probes\n",
      "-- filtering region:131, screen genome to keep 400 probes\n",
      "-- filtering region:132, screen genome to keep 400 probes\n",
      "-- filtering region:133, screen genome to keep 400 probes\n",
      "-- filtering region:134, screen genome to keep 400 probes\n",
      "-- filtering region:135, screen genome to keep 400 probes\n",
      "-- filtering region:136, screen genome to keep 400 probes\n",
      "-- filtering region:137, screen genome to keep 400 probes\n",
      "-- filtering region:138, screen genome to keep 400 probes\n",
      "-- filtering region:139, screen genome to keep 400 probes\n",
      "-- filtering region:140, screen genome to keep 400 probes\n",
      "-- filtering region:141, screen genome to keep 400 probes\n",
      "-- filtering region:142, screen genome to keep 400 probes\n",
      "-- filtering region:143, screen genome to keep 400 probes\n",
      "-- filtering region:144, screen genome to keep 400 probes\n",
      "-- filtering region:145, screen genome to keep 400 probes\n",
      "-- filtering region:146, screen genome to keep 400 probes\n",
      "-- filtering region:147, screen genome to keep 400 probes\n",
      "-- filtering region:148, screen genome to keep 400 probes\n",
      "-- filtering region:149, screen genome to keep 400 probes\n",
      "-- filtering region:150, screen genome to keep 400 probes\n",
      "-- filtering region:151, screen genome to keep 400 probes\n",
      "-- filtering region:152, screen genome to keep 400 probes\n",
      "-- filtering region:153, screen genome to keep 400 probes\n",
      "-- filtering region:154, screen genome to keep 400 probes\n",
      "-- filtering region:155, screen genome to keep 400 probes\n",
      "-- filtering region:156, screen genome to keep 400 probes\n",
      "-- filtering region:157, screen genome to keep 400 probes\n",
      "-- filtering region:158, screen genome to keep 400 probes\n",
      "-- filtering region:159, screen genome to keep 400 probes\n",
      "-- filtering region:160, screen genome to keep 400 probes\n",
      "-- filtering region:161, screen genome to keep 400 probes\n",
      "-- filtering region:162, screen genome to keep 400 probes\n",
      "-- filtering region:163, screen genome to keep 400 probes\n",
      "-- filtering region:164, screen genome to keep 400 probes\n",
      "-- filtering region:165, screen genome to keep 400 probes\n",
      "-- filtering region:166, screen genome to keep 400 probes\n",
      "-- filtering region:168, screen genome to keep 400 probes\n",
      "-- filtering region:169, screen genome to keep 400 probes\n",
      "-- filtering region:170, screen genome to keep 400 probes\n",
      "-- filtering region:171, screen genome to keep 400 probes\n",
      "-- filtering region:172, screen genome to keep 400 probes\n",
      "-- filtering region:173, screen genome to keep 400 probes\n",
      "-- filtering region:174, screen genome to keep 400 probes\n",
      "-- filtering region:175, screen genome to keep 400 probes\n",
      "-- filtering region:176, screen genome to keep 400 probes\n",
      "-- filtering region:177, screen genome to keep 400 probes\n",
      "-- filtering region:178, screen genome to keep 400 probes\n",
      "-- filtering region:179, screen genome to keep 400 probes\n",
      "-- filtering region:180, screen genome to keep 400 probes\n",
      "-- filtering region:181, screen genome to keep 400 probes\n",
      "-- filtering region:182, screen genome to keep 400 probes\n",
      "-- filtering region:183, screen genome to keep 400 probes\n",
      "-- filtering region:184, screen genome to keep 400 probes\n",
      "-- filtering region:185, directly append 115 probes\n",
      "-- filtering region:186, screen genome to keep 400 probes\n",
      "-- filtering region:187, screen genome to keep 400 probes\n",
      "-- filtering region:188, screen genome to keep 400 probes\n",
      "-- filtering region:189, screen genome to keep 400 probes\n",
      "-- filtering region:190, screen genome to keep 400 probes\n",
      "-- filtering region:191, screen genome to keep 400 probes\n",
      "-- filtering region:192, screen genome to keep 400 probes\n",
      "-- filtering region:193, screen genome to keep 400 probes\n",
      "-- filtering region:194, screen genome to keep 400 probes\n",
      "-- filtering region:195, screen genome to keep 400 probes\n",
      "-- filtering region:196, screen genome to keep 400 probes\n",
      "-- filtering region:197, screen genome to keep 400 probes\n",
      "-- filtering region:198, screen genome to keep 400 probes\n",
      "-- filtering region:199, screen genome to keep 400 probes\n",
      "-- filtering region:200, screen genome to keep 400 probes\n",
      "-- filtering region:201, screen genome to keep 400 probes\n",
      "-- filtering region:202, screen genome to keep 400 probes\n",
      "-- filtering region:203, screen genome to keep 400 probes\n",
      "-- filtering region:204, screen genome to keep 400 probes\n",
      "-- filtering region:205, screen genome to keep 400 probes\n",
      "-- filtering region:206, screen genome to keep 400 probes\n",
      "-- filtering region:207, screen genome to keep 400 probes\n",
      "-- filtering region:208, screen genome to keep 400 probes\n",
      "-- filtering region:209, screen genome to keep 400 probes\n",
      "-- filtering region:210, screen genome to keep 400 probes\n",
      "-- filtering region:211, screen genome to keep 400 probes\n",
      "-- filtering region:212, directly append 209 probes\n",
      "-- filtering region:213, directly append 283 probes\n",
      "-- filtering region:214, screen genome to keep 400 probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- filtering region:215, screen genome to keep 400 probes\n",
      "-- filtering region:216, screen genome to keep 400 probes\n",
      "-- filtering region:217, screen genome to keep 400 probes\n",
      "-- filtering region:218, screen genome to keep 400 probes\n",
      "-- filtering region:219, screen genome to keep 400 probes\n",
      "-- filtering region:220, screen genome to keep 400 probes\n",
      "-- filtering region:221, screen genome to keep 400 probes\n",
      "-- filtering region:222, screen genome to keep 400 probes\n",
      "-- filtering region:223, screen genome to keep 400 probes\n",
      "-- filtering region:224, screen genome to keep 400 probes\n",
      "-- filtering region:225, screen genome to keep 400 probes\n",
      "-- filtering region:226, screen genome to keep 400 probes\n",
      "-- filtering region:227, directly append 6 probes\n",
      "-- filtering region:228, screen genome to keep 400 probes\n",
      "-- filtering region:229, screen genome to keep 400 probes\n",
      "-- filtering region:230, screen genome to keep 400 probes\n",
      "-- filtering region:231, screen genome to keep 400 probes\n",
      "-- filtering region:232, screen genome to keep 400 probes\n",
      "-- filtering region:233, screen genome to keep 400 probes\n",
      "-- filtering region:234, screen genome to keep 400 probes\n",
      "-- filtering region:235, screen genome to keep 400 probes\n",
      "-- filtering region:236, screen genome to keep 400 probes\n",
      "-- filtering region:237, screen genome to keep 400 probes\n",
      "-- filtering region:238, screen genome to keep 400 probes\n",
      "-- filtering region:239, screen genome to keep 400 probes\n",
      "-- filtering region:240, screen genome to keep 400 probes\n",
      "-- filtering region:241, screen genome to keep 400 probes\n",
      "-- filtering region:242, screen genome to keep 400 probes\n",
      "-- filtering region:243, screen genome to keep 400 probes\n",
      "-- filtering region:244, screen genome to keep 400 probes\n",
      "-- filtering region:245, screen genome to keep 400 probes\n",
      "-- filtering region:246, screen genome to keep 400 probes\n",
      "-- filtering region:247, screen genome to keep 400 probes\n",
      "-- filtering region:248, screen genome to keep 400 probes\n",
      "-- filtering region:249, screen genome to keep 400 probes\n",
      "-- filtering region:250, screen genome to keep 400 probes\n",
      "-- filtering region:751, screen genome to keep 400 probes\n",
      "-- filtering region:752, directly append 176 probes\n",
      "-- filtering region:756, directly append 8 probes\n",
      "-- filtering region:757, directly append 8 probes\n",
      "-- filtering region:758, screen genome to keep 400 probes\n",
      "-- filtering region:759, screen genome to keep 400 probes\n",
      "-- filtering region:760, screen genome to keep 400 probes\n",
      "-- filtering region:761, screen genome to keep 400 probes\n",
      "-- filtering region:762, screen genome to keep 400 probes\n",
      "-- filtering region:763, screen genome to keep 400 probes\n",
      "-- filtering region:764, screen genome to keep 400 probes\n",
      "-- filtering region:765, screen genome to keep 400 probes\n",
      "-- filtering region:766, screen genome to keep 400 probes\n",
      "-- filtering region:767, screen genome to keep 400 probes\n",
      "-- filtering region:768, screen genome to keep 400 probes\n",
      "-- filtering region:769, screen genome to keep 400 probes\n",
      "-- filtering region:770, screen genome to keep 400 probes\n",
      "-- filtering region:771, directly append 36 probes\n",
      "-- filtering region:772, screen genome to keep 400 probes\n",
      "-- filtering region:773, screen genome to keep 400 probes\n",
      "-- filtering region:774, screen genome to keep 400 probes\n",
      "-- filtering region:775, screen genome to keep 400 probes\n",
      "-- filtering region:776, directly append 62 probes\n",
      "-- filtering region:777, screen genome to keep 400 probes\n",
      "-- filtering region:778, screen genome to keep 400 probes\n",
      "-- filtering region:779, directly append 393 probes\n",
      "-- filtering region:780, screen genome to keep 400 probes\n",
      "-- filtering region:781, screen genome to keep 400 probes\n",
      "-- filtering region:782, screen genome to keep 400 probes\n",
      "-- filtering region:783, screen genome to keep 400 probes\n",
      "-- filtering region:784, screen genome to keep 400 probes\n",
      "-- filtering region:785, screen genome to keep 400 probes\n",
      "-- filtering region:786, screen genome to keep 400 probes\n",
      "-- filtering region:787, screen genome to keep 400 probes\n",
      "-- filtering region:788, screen genome to keep 400 probes\n",
      "-- filtering region:789, screen genome to keep 400 probes\n",
      "-- filtering region:790, screen genome to keep 400 probes\n",
      "-- filtering region:791, screen genome to keep 400 probes\n",
      "-- filtering region:792, screen genome to keep 400 probes\n",
      "-- filtering region:793, screen genome to keep 400 probes\n",
      "-- filtering region:794, screen genome to keep 400 probes\n",
      "-- filtering region:795, screen genome to keep 400 probes\n",
      "-- filtering region:796, screen genome to keep 400 probes\n",
      "-- filtering region:797, screen genome to keep 400 probes\n",
      "-- filtering region:798, screen genome to keep 400 probes\n",
      "-- filtering region:799, screen genome to keep 400 probes\n",
      "-- filtering region:800, screen genome to keep 400 probes\n",
      "-- filtering region:801, screen genome to keep 400 probes\n",
      "-- filtering region:802, screen genome to keep 400 probes\n",
      "-- filtering region:803, screen genome to keep 400 probes\n",
      "-- filtering region:804, screen genome to keep 400 probes\n",
      "-- filtering region:805, screen genome to keep 400 probes\n",
      "-- filtering region:806, screen genome to keep 400 probes\n",
      "-- filtering region:807, screen genome to keep 400 probes\n",
      "-- filtering region:808, screen genome to keep 400 probes\n",
      "-- filtering region:809, screen genome to keep 400 probes\n",
      "-- filtering region:810, screen genome to keep 400 probes\n",
      "-- filtering region:811, screen genome to keep 400 probes\n",
      "-- filtering region:812, screen genome to keep 400 probes\n",
      "-- filtering region:813, screen genome to keep 400 probes\n",
      "-- filtering region:814, screen genome to keep 400 probes\n",
      "-- filtering region:815, screen genome to keep 400 probes\n",
      "-- filtering region:816, screen genome to keep 400 probes\n",
      "-- filtering region:817, screen genome to keep 400 probes\n",
      "-- filtering region:818, screen genome to keep 400 probes\n",
      "-- filtering region:819, screen genome to keep 400 probes\n",
      "-- filtering region:820, screen genome to keep 400 probes\n",
      "-- filtering region:821, screen genome to keep 400 probes\n",
      "-- filtering region:822, screen genome to keep 400 probes\n",
      "-- filtering region:823, screen genome to keep 400 probes\n",
      "-- filtering region:824, screen genome to keep 400 probes\n",
      "-- filtering region:825, screen genome to keep 400 probes\n",
      "-- filtering region:826, screen genome to keep 400 probes\n",
      "-- filtering region:827, screen genome to keep 400 probes\n",
      "-- filtering region:828, screen genome to keep 400 probes\n",
      "-- filtering region:829, screen genome to keep 400 probes\n",
      "-- filtering region:830, screen genome to keep 400 probes\n",
      "-- filtering region:831, screen genome to keep 400 probes\n",
      "-- filtering region:832, screen genome to keep 400 probes\n",
      "-- filtering region:833, screen genome to keep 400 probes\n",
      "-- filtering region:834, screen genome to keep 400 probes\n",
      "-- filtering region:835, screen genome to keep 400 probes\n",
      "-- filtering region:836, screen genome to keep 400 probes\n",
      "-- filtering region:837, screen genome to keep 400 probes\n",
      "-- filtering region:838, screen genome to keep 400 probes\n",
      "-- filtering region:839, screen genome to keep 400 probes\n",
      "-- filtering region:840, screen genome to keep 400 probes\n",
      "-- filtering region:841, screen genome to keep 400 probes\n",
      "-- filtering region:842, screen genome to keep 400 probes\n",
      "-- filtering region:843, screen genome to keep 400 probes\n",
      "-- filtering region:844, screen genome to keep 400 probes\n",
      "-- filtering region:845, screen genome to keep 400 probes\n",
      "-- filtering region:846, screen genome to keep 400 probes\n",
      "-- filtering region:847, screen genome to keep 400 probes\n",
      "-- filtering region:848, screen genome to keep 400 probes\n",
      "-- filtering region:849, screen genome to keep 400 probes\n",
      "-- filtering region:850, screen genome to keep 400 probes\n",
      "-- filtering region:851, screen genome to keep 400 probes\n",
      "-- filtering region:852, screen genome to keep 400 probes\n",
      "-- filtering region:853, screen genome to keep 400 probes\n",
      "-- filtering region:854, screen genome to keep 400 probes\n",
      "-- filtering region:855, screen genome to keep 400 probes\n",
      "-- filtering region:856, screen genome to keep 400 probes\n",
      "-- filtering region:857, screen genome to keep 400 probes\n",
      "-- filtering region:858, screen genome to keep 400 probes\n",
      "-- filtering region:859, screen genome to keep 400 probes\n",
      "-- filtering region:860, screen genome to keep 400 probes\n",
      "-- filtering region:861, screen genome to keep 400 probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- filtering region:862, screen genome to keep 400 probes\n",
      "-- filtering region:863, screen genome to keep 400 probes\n",
      "-- filtering region:864, screen genome to keep 400 probes\n",
      "-- filtering region:866, screen genome to keep 400 probes\n",
      "-- filtering region:867, screen genome to keep 400 probes\n",
      "-- filtering region:868, screen genome to keep 400 probes\n",
      "-- filtering region:869, screen genome to keep 400 probes\n",
      "-- filtering region:870, screen genome to keep 400 probes\n",
      "-- filtering region:871, screen genome to keep 400 probes\n",
      "-- filtering region:872, screen genome to keep 400 probes\n",
      "-- filtering region:873, screen genome to keep 400 probes\n",
      "-- filtering region:874, screen genome to keep 400 probes\n",
      "-- filtering region:875, screen genome to keep 400 probes\n",
      "-- filtering region:876, screen genome to keep 400 probes\n",
      "-- filtering region:877, screen genome to keep 400 probes\n",
      "-- filtering region:878, screen genome to keep 400 probes\n",
      "-- filtering region:879, screen genome to keep 400 probes\n",
      "-- filtering region:880, screen genome to keep 400 probes\n",
      "-- filtering region:881, screen genome to keep 400 probes\n",
      "-- filtering region:882, screen genome to keep 400 probes\n",
      "-- filtering region:883, screen genome to keep 400 probes\n",
      "-- filtering region:884, screen genome to keep 400 probes\n",
      "-- filtering region:885, screen genome to keep 400 probes\n",
      "-- filtering region:886, screen genome to keep 400 probes\n",
      "-- filtering region:887, directly append 400 probes\n",
      "-- filtering region:888, screen genome to keep 400 probes\n",
      "-- filtering region:889, screen genome to keep 400 probes\n",
      "-- filtering region:890, screen genome to keep 400 probes\n",
      "-- filtering region:891, screen genome to keep 400 probes\n",
      "-- filtering region:892, screen genome to keep 400 probes\n",
      "-- filtering region:893, screen genome to keep 400 probes\n",
      "-- filtering region:894, screen genome to keep 400 probes\n",
      "-- filtering region:895, screen genome to keep 400 probes\n",
      "-- filtering region:896, screen genome to keep 400 probes\n",
      "-- filtering region:897, screen genome to keep 400 probes\n",
      "-- filtering region:898, screen genome to keep 400 probes\n",
      "-- filtering region:899, screen genome to keep 400 probes\n",
      "-- filtering region:900, screen genome to keep 400 probes\n",
      "-- filtering region:901, screen genome to keep 400 probes\n",
      "-- filtering region:902, directly append 381 probes\n",
      "-- filtering region:903, screen genome to keep 400 probes\n",
      "-- filtering region:904, screen genome to keep 400 probes\n",
      "-- filtering region:905, screen genome to keep 400 probes\n",
      "-- filtering region:906, screen genome to keep 400 probes\n",
      "-- filtering region:907, screen genome to keep 400 probes\n",
      "-- filtering region:908, screen genome to keep 400 probes\n",
      "-- filtering region:909, screen genome to keep 400 probes\n",
      "-- filtering region:910, screen genome to keep 400 probes\n",
      "-- filtering region:911, screen genome to keep 400 probes\n",
      "-- filtering region:912, screen genome to keep 400 probes\n",
      "-- filtering region:913, screen genome to keep 400 probes\n",
      "-- filtering region:914, screen genome to keep 400 probes\n",
      "-- filtering region:915, screen genome to keep 400 probes\n",
      "-- filtering region:916, screen genome to keep 400 probes\n",
      "-- filtering region:917, screen genome to keep 400 probes\n",
      "-- filtering region:918, screen genome to keep 400 probes\n",
      "-- filtering region:919, screen genome to keep 400 probes\n",
      "-- filtering region:920, screen genome to keep 400 probes\n",
      "-- filtering region:921, screen genome to keep 400 probes\n",
      "-- filtering region:922, screen genome to keep 400 probes\n",
      "-- filtering region:923, screen genome to keep 400 probes\n",
      "-- filtering region:925, screen genome to keep 400 probes\n",
      "-- filtering region:926, screen genome to keep 400 probes\n",
      "-- filtering region:927, screen genome to keep 400 probes\n",
      "-- filtering region:928, screen genome to keep 400 probes\n",
      "-- filtering region:929, screen genome to keep 400 probes\n",
      "-- filtering region:930, screen genome to keep 400 probes\n",
      "-- filtering region:931, screen genome to keep 400 probes\n",
      "-- filtering region:932, screen genome to keep 400 probes\n",
      "-- filtering region:933, screen genome to keep 400 probes\n",
      "-- filtering region:934, screen genome to keep 400 probes\n",
      "-- filtering region:935, screen genome to keep 400 probes\n",
      "-- filtering region:936, screen genome to keep 400 probes\n",
      "-- filtering region:937, screen genome to keep 400 probes\n",
      "-- filtering region:938, screen genome to keep 400 probes\n",
      "-- filtering region:939, screen genome to keep 400 probes\n",
      "-- filtering region:940, screen genome to keep 400 probes\n",
      "-- filtering region:941, screen genome to keep 400 probes\n",
      "-- filtering region:942, screen genome to keep 400 probes\n",
      "-- filtering region:943, screen genome to keep 400 probes\n",
      "-- filtering region:944, screen genome to keep 400 probes\n",
      "-- filtering region:945, screen genome to keep 400 probes\n",
      "-- filtering region:946, screen genome to keep 400 probes\n",
      "-- filtering region:947, screen genome to keep 400 probes\n",
      "-- filtering region:948, screen genome to keep 400 probes\n",
      "-- filtering region:949, screen genome to keep 400 probes\n",
      "-- filtering region:950, screen genome to keep 400 probes\n",
      "-- filtering region:951, screen genome to keep 400 probes\n",
      "-- filtering region:952, screen genome to keep 400 probes\n",
      "-- filtering region:953, screen genome to keep 400 probes\n",
      "-- filtering region:954, screen genome to keep 400 probes\n",
      "-- filtering region:955, screen genome to keep 400 probes\n",
      "-- filtering region:956, screen genome to keep 400 probes\n",
      "-- filtering region:957, screen genome to keep 400 probes\n",
      "-- filtering region:958, screen genome to keep 400 probes\n",
      "-- filtering region:959, screen genome to keep 400 probes\n",
      "-- filtering region:960, screen genome to keep 400 probes\n",
      "-- filtering region:961, screen genome to keep 400 probes\n",
      "-- filtering region:962, screen genome to keep 400 probes\n",
      "-- filtering region:963, screen genome to keep 400 probes\n",
      "-- filtering region:964, screen genome to keep 400 probes\n",
      "-- filtering region:965, screen genome to keep 400 probes\n",
      "-- filtering region:966, screen genome to keep 400 probes\n",
      "-- filtering region:967, screen genome to keep 400 probes\n",
      "-- filtering region:968, screen genome to keep 400 probes\n",
      "-- filtering region:969, screen genome to keep 400 probes\n",
      "-- filtering region:970, screen genome to keep 400 probes\n",
      "-- filtering region:971, screen genome to keep 400 probes\n",
      "-- filtering region:972, screen genome to keep 400 probes\n",
      "-- filtering region:973, screen genome to keep 400 probes\n",
      "-- filtering region:974, screen genome to keep 400 probes\n",
      "-- filtering region:975, screen genome to keep 400 probes\n",
      "-- filtering region:976, screen genome to keep 400 probes\n",
      "-- filtering region:977, screen genome to keep 400 probes\n",
      "-- filtering region:978, screen genome to keep 400 probes\n",
      "-- filtering region:979, screen genome to keep 400 probes\n",
      "-- filtering region:980, screen genome to keep 400 probes\n",
      "-- filtering region:981, screen genome to keep 400 probes\n",
      "-- filtering region:982, screen genome to keep 400 probes\n",
      "-- filtering region:983, screen genome to keep 400 probes\n",
      "-- filtering region:984, screen genome to keep 400 probes\n",
      "-- filtering region:985, screen genome to keep 400 probes\n",
      "-- filtering region:986, screen genome to keep 400 probes\n",
      "-- filtering region:987, screen genome to keep 400 probes\n",
      "-- filtering region:988, screen genome to keep 400 probes\n",
      "-- filtering region:989, screen genome to keep 400 probes\n",
      "-- filtering region:990, screen genome to keep 400 probes\n",
      "-- filtering region:991, directly append 127 probes\n",
      "region: 323 removed with 38 probes\n",
      "region: 362 removed with 320 probes\n",
      "region: 402 removed with 73 probes\n",
      "region: 418 removed with 378 probes\n",
      "region: 426 removed with 313 probes\n",
      "region: 443 removed with 3 probes\n",
      "region: 490 removed with 19 probes\n",
      "region: 539 removed with 268 probes\n",
      "region: 185 removed with 115 probes\n",
      "region: 212 removed with 209 probes\n",
      "region: 213 removed with 283 probes\n",
      "region: 227 removed with 6 probes\n",
      "region: 752 removed with 176 probes\n",
      "region: 756 removed with 8 probes\n",
      "region: 757 removed with 8 probes\n",
      "region: 771 removed with 36 probes\n",
      "region: 776 removed with 62 probes\n",
      "region: 779 removed with 393 probes\n",
      "region: 902 removed with 381 probes\n",
      "region: 991 removed with 127 probes\n"
     ]
    }
   ],
   "source": [
    "# filter by genome counts to only keep regions that have 400 pbs\n",
    "reload(library_tools.quality_check)\n",
    "\n",
    "num_probes_kept = 400 # max num of probes kept for each region\n",
    "filtered_pb_dict_seq = library_tools.quality_check.Screen_probe_by_hit(library_folder, pb_dict_from_load, num_probes_kept,\n",
    "                                                                  hit_type='genome')\n",
    "\n",
    "## filter by minimum number of probes in each region\n",
    "min_num_probes = 400\n",
    "removed_regions = []\n",
    "for _rid, _pb_class in filtered_pb_dict_seq.items():\n",
    "    if len(_pb_class.kept_probes) < min_num_probes:\n",
    "        print(f\"region: {_rid} removed with {len(_pb_class.kept_probes)} probes\")\n",
    "        removed_regions.append(_rid)\n",
    "for _rid in removed_regions:\n",
    "    del(filtered_pb_dict_seq[_rid])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "/* Put everything inside the global mpl namespace */\n",
       "/* global mpl */\n",
       "window.mpl = {};\n",
       "\n",
       "mpl.get_websocket_type = function () {\n",
       "    if (typeof WebSocket !== 'undefined') {\n",
       "        return WebSocket;\n",
       "    } else if (typeof MozWebSocket !== 'undefined') {\n",
       "        return MozWebSocket;\n",
       "    } else {\n",
       "        alert(\n",
       "            'Your browser does not have WebSocket support. ' +\n",
       "                'Please try Chrome, Safari or Firefox  6. ' +\n",
       "                'Firefox 4 and 5 are also supported but you ' +\n",
       "                'have to enable WebSockets in about:config.'\n",
       "        );\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure = function (figure_id, websocket, ondownload, parent_element) {\n",
       "    this.id = figure_id;\n",
       "\n",
       "    this.ws = websocket;\n",
       "\n",
       "    this.supports_binary = this.ws.binaryType !== undefined;\n",
       "\n",
       "    if (!this.supports_binary) {\n",
       "        var warnings = document.getElementById('mpl-warnings');\n",
       "        if (warnings) {\n",
       "            warnings.style.display = 'block';\n",
       "            warnings.textContent =\n",
       "                'This browser does not support binary websocket messages. ' +\n",
       "                'Performance may be slow.';\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.imageObj = new Image();\n",
       "\n",
       "    this.context = undefined;\n",
       "    this.message = undefined;\n",
       "    this.canvas = undefined;\n",
       "    this.rubberband_canvas = undefined;\n",
       "    this.rubberband_context = undefined;\n",
       "    this.format_dropdown = undefined;\n",
       "\n",
       "    this.image_mode = 'full';\n",
       "\n",
       "    this.root = document.createElement('div');\n",
       "    this.root.setAttribute('style', 'display: inline-block');\n",
       "    this._root_extra_style(this.root);\n",
       "\n",
       "    parent_element.appendChild(this.root);\n",
       "\n",
       "    this._init_header(this);\n",
       "    this._init_canvas(this);\n",
       "    this._init_toolbar(this);\n",
       "\n",
       "    var fig = this;\n",
       "\n",
       "    this.waiting = false;\n",
       "\n",
       "    this.ws.onopen = function () {\n",
       "        fig.send_message('supports_binary', { value: fig.supports_binary });\n",
       "        fig.send_message('send_image_mode', {});\n",
       "        if (fig.ratio !== 1) {\n",
       "            fig.send_message('set_dpi_ratio', { dpi_ratio: fig.ratio });\n",
       "        }\n",
       "        fig.send_message('refresh', {});\n",
       "    };\n",
       "\n",
       "    this.imageObj.onload = function () {\n",
       "        if (fig.image_mode === 'full') {\n",
       "            // Full images could contain transparency (where diff images\n",
       "            // almost always do), so we need to clear the canvas so that\n",
       "            // there is no ghosting.\n",
       "            fig.context.clearRect(0, 0, fig.canvas.width, fig.canvas.height);\n",
       "        }\n",
       "        fig.context.drawImage(fig.imageObj, 0, 0);\n",
       "    };\n",
       "\n",
       "    this.imageObj.onunload = function () {\n",
       "        fig.ws.close();\n",
       "    };\n",
       "\n",
       "    this.ws.onmessage = this._make_on_message_function(this);\n",
       "\n",
       "    this.ondownload = ondownload;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_header = function () {\n",
       "    var titlebar = document.createElement('div');\n",
       "    titlebar.classList =\n",
       "        'ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix';\n",
       "    var titletext = document.createElement('div');\n",
       "    titletext.classList = 'ui-dialog-title';\n",
       "    titletext.setAttribute(\n",
       "        'style',\n",
       "        'width: 100%; text-align: center; padding: 3px;'\n",
       "    );\n",
       "    titlebar.appendChild(titletext);\n",
       "    this.root.appendChild(titlebar);\n",
       "    this.header = titletext;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._init_canvas = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var canvas_div = (this.canvas_div = document.createElement('div'));\n",
       "    canvas_div.setAttribute(\n",
       "        'style',\n",
       "        'border: 1px solid #ddd;' +\n",
       "            'box-sizing: content-box;' +\n",
       "            'clear: both;' +\n",
       "            'min-height: 1px;' +\n",
       "            'min-width: 1px;' +\n",
       "            'outline: 0;' +\n",
       "            'overflow: hidden;' +\n",
       "            'position: relative;' +\n",
       "            'resize: both;'\n",
       "    );\n",
       "\n",
       "    function on_keyboard_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.key_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    canvas_div.addEventListener(\n",
       "        'keydown',\n",
       "        on_keyboard_event_closure('key_press')\n",
       "    );\n",
       "    canvas_div.addEventListener(\n",
       "        'keyup',\n",
       "        on_keyboard_event_closure('key_release')\n",
       "    );\n",
       "\n",
       "    this._canvas_extra_style(canvas_div);\n",
       "    this.root.appendChild(canvas_div);\n",
       "\n",
       "    var canvas = (this.canvas = document.createElement('canvas'));\n",
       "    canvas.classList.add('mpl-canvas');\n",
       "    canvas.setAttribute('style', 'box-sizing: content-box;');\n",
       "\n",
       "    this.context = canvas.getContext('2d');\n",
       "\n",
       "    var backingStore =\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        this.context.webkitBackingStorePixelRatio ||\n",
       "        this.context.mozBackingStorePixelRatio ||\n",
       "        this.context.msBackingStorePixelRatio ||\n",
       "        this.context.oBackingStorePixelRatio ||\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        1;\n",
       "\n",
       "    this.ratio = (window.devicePixelRatio || 1) / backingStore;\n",
       "    if (this.ratio !== 1) {\n",
       "        fig.send_message('set_dpi_ratio', { dpi_ratio: this.ratio });\n",
       "    }\n",
       "\n",
       "    var rubberband_canvas = (this.rubberband_canvas = document.createElement(\n",
       "        'canvas'\n",
       "    ));\n",
       "    rubberband_canvas.setAttribute(\n",
       "        'style',\n",
       "        'box-sizing: content-box; position: absolute; left: 0; top: 0; z-index: 1;'\n",
       "    );\n",
       "\n",
       "    var resizeObserver = new ResizeObserver(function (entries) {\n",
       "        var nentries = entries.length;\n",
       "        for (var i = 0; i < nentries; i++) {\n",
       "            var entry = entries[i];\n",
       "            var width, height;\n",
       "            if (entry.contentBoxSize) {\n",
       "                if (entry.contentBoxSize instanceof Array) {\n",
       "                    // Chrome 84 implements new version of spec.\n",
       "                    width = entry.contentBoxSize[0].inlineSize;\n",
       "                    height = entry.contentBoxSize[0].blockSize;\n",
       "                } else {\n",
       "                    // Firefox implements old version of spec.\n",
       "                    width = entry.contentBoxSize.inlineSize;\n",
       "                    height = entry.contentBoxSize.blockSize;\n",
       "                }\n",
       "            } else {\n",
       "                // Chrome <84 implements even older version of spec.\n",
       "                width = entry.contentRect.width;\n",
       "                height = entry.contentRect.height;\n",
       "            }\n",
       "\n",
       "            // Keep the size of the canvas and rubber band canvas in sync with\n",
       "            // the canvas container.\n",
       "            if (entry.devicePixelContentBoxSize) {\n",
       "                // Chrome 84 implements new version of spec.\n",
       "                canvas.setAttribute(\n",
       "                    'width',\n",
       "                    entry.devicePixelContentBoxSize[0].inlineSize\n",
       "                );\n",
       "                canvas.setAttribute(\n",
       "                    'height',\n",
       "                    entry.devicePixelContentBoxSize[0].blockSize\n",
       "                );\n",
       "            } else {\n",
       "                canvas.setAttribute('width', width * fig.ratio);\n",
       "                canvas.setAttribute('height', height * fig.ratio);\n",
       "            }\n",
       "            canvas.setAttribute(\n",
       "                'style',\n",
       "                'width: ' + width + 'px; height: ' + height + 'px;'\n",
       "            );\n",
       "\n",
       "            rubberband_canvas.setAttribute('width', width);\n",
       "            rubberband_canvas.setAttribute('height', height);\n",
       "\n",
       "            // And update the size in Python. We ignore the initial 0/0 size\n",
       "            // that occurs as the element is placed into the DOM, which should\n",
       "            // otherwise not happen due to the minimum size styling.\n",
       "            if (width != 0 && height != 0) {\n",
       "                fig.request_resize(width, height);\n",
       "            }\n",
       "        }\n",
       "    });\n",
       "    resizeObserver.observe(canvas_div);\n",
       "\n",
       "    function on_mouse_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.mouse_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousedown',\n",
       "        on_mouse_event_closure('button_press')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseup',\n",
       "        on_mouse_event_closure('button_release')\n",
       "    );\n",
       "    // Throttle sequential mouse events to 1 every 20ms.\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousemove',\n",
       "        on_mouse_event_closure('motion_notify')\n",
       "    );\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseenter',\n",
       "        on_mouse_event_closure('figure_enter')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseleave',\n",
       "        on_mouse_event_closure('figure_leave')\n",
       "    );\n",
       "\n",
       "    canvas_div.addEventListener('wheel', function (event) {\n",
       "        if (event.deltaY < 0) {\n",
       "            event.step = 1;\n",
       "        } else {\n",
       "            event.step = -1;\n",
       "        }\n",
       "        on_mouse_event_closure('scroll')(event);\n",
       "    });\n",
       "\n",
       "    canvas_div.appendChild(canvas);\n",
       "    canvas_div.appendChild(rubberband_canvas);\n",
       "\n",
       "    this.rubberband_context = rubberband_canvas.getContext('2d');\n",
       "    this.rubberband_context.strokeStyle = '#000000';\n",
       "\n",
       "    this._resize_canvas = function (width, height, forward) {\n",
       "        if (forward) {\n",
       "            canvas_div.style.width = width + 'px';\n",
       "            canvas_div.style.height = height + 'px';\n",
       "        }\n",
       "    };\n",
       "\n",
       "    // Disable right mouse context menu.\n",
       "    this.rubberband_canvas.addEventListener('contextmenu', function (_e) {\n",
       "        event.preventDefault();\n",
       "        return false;\n",
       "    });\n",
       "\n",
       "    function set_focus() {\n",
       "        canvas.focus();\n",
       "        canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    window.setTimeout(set_focus, 100);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'mpl-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'mpl-button-group';\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'mpl-button-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        var button = (fig.buttons[name] = document.createElement('button'));\n",
       "        button.classList = 'mpl-widget';\n",
       "        button.setAttribute('role', 'button');\n",
       "        button.setAttribute('aria-disabled', 'false');\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "\n",
       "        var icon_img = document.createElement('img');\n",
       "        icon_img.src = '_images/' + image + '.png';\n",
       "        icon_img.srcset = '_images/' + image + '_large.png 2x';\n",
       "        icon_img.alt = tooltip;\n",
       "        button.appendChild(icon_img);\n",
       "\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    var fmt_picker = document.createElement('select');\n",
       "    fmt_picker.classList = 'mpl-widget';\n",
       "    toolbar.appendChild(fmt_picker);\n",
       "    this.format_dropdown = fmt_picker;\n",
       "\n",
       "    for (var ind in mpl.extensions) {\n",
       "        var fmt = mpl.extensions[ind];\n",
       "        var option = document.createElement('option');\n",
       "        option.selected = fmt === mpl.default_extension;\n",
       "        option.innerHTML = fmt;\n",
       "        fmt_picker.appendChild(option);\n",
       "    }\n",
       "\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.request_resize = function (x_pixels, y_pixels) {\n",
       "    // Request matplotlib to resize the figure. Matplotlib will then trigger a resize in the client,\n",
       "    // which will in turn request a refresh of the image.\n",
       "    this.send_message('resize', { width: x_pixels, height: y_pixels });\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_message = function (type, properties) {\n",
       "    properties['type'] = type;\n",
       "    properties['figure_id'] = this.id;\n",
       "    this.ws.send(JSON.stringify(properties));\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_draw_message = function () {\n",
       "    if (!this.waiting) {\n",
       "        this.waiting = true;\n",
       "        this.ws.send(JSON.stringify({ type: 'draw', figure_id: this.id }));\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    var format_dropdown = fig.format_dropdown;\n",
       "    var format = format_dropdown.options[format_dropdown.selectedIndex].value;\n",
       "    fig.ondownload(fig, format);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_resize = function (fig, msg) {\n",
       "    var size = msg['size'];\n",
       "    if (size[0] !== fig.canvas.width || size[1] !== fig.canvas.height) {\n",
       "        fig._resize_canvas(size[0], size[1], msg['forward']);\n",
       "        fig.send_message('refresh', {});\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_rubberband = function (fig, msg) {\n",
       "    var x0 = msg['x0'] / fig.ratio;\n",
       "    var y0 = (fig.canvas.height - msg['y0']) / fig.ratio;\n",
       "    var x1 = msg['x1'] / fig.ratio;\n",
       "    var y1 = (fig.canvas.height - msg['y1']) / fig.ratio;\n",
       "    x0 = Math.floor(x0) + 0.5;\n",
       "    y0 = Math.floor(y0) + 0.5;\n",
       "    x1 = Math.floor(x1) + 0.5;\n",
       "    y1 = Math.floor(y1) + 0.5;\n",
       "    var min_x = Math.min(x0, x1);\n",
       "    var min_y = Math.min(y0, y1);\n",
       "    var width = Math.abs(x1 - x0);\n",
       "    var height = Math.abs(y1 - y0);\n",
       "\n",
       "    fig.rubberband_context.clearRect(\n",
       "        0,\n",
       "        0,\n",
       "        fig.canvas.width / fig.ratio,\n",
       "        fig.canvas.height / fig.ratio\n",
       "    );\n",
       "\n",
       "    fig.rubberband_context.strokeRect(min_x, min_y, width, height);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_figure_label = function (fig, msg) {\n",
       "    // Updates the figure title.\n",
       "    fig.header.textContent = msg['label'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_cursor = function (fig, msg) {\n",
       "    var cursor = msg['cursor'];\n",
       "    switch (cursor) {\n",
       "        case 0:\n",
       "            cursor = 'pointer';\n",
       "            break;\n",
       "        case 1:\n",
       "            cursor = 'default';\n",
       "            break;\n",
       "        case 2:\n",
       "            cursor = 'crosshair';\n",
       "            break;\n",
       "        case 3:\n",
       "            cursor = 'move';\n",
       "            break;\n",
       "    }\n",
       "    fig.rubberband_canvas.style.cursor = cursor;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_message = function (fig, msg) {\n",
       "    fig.message.textContent = msg['message'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_draw = function (fig, _msg) {\n",
       "    // Request the server to send over a new figure.\n",
       "    fig.send_draw_message();\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_image_mode = function (fig, msg) {\n",
       "    fig.image_mode = msg['mode'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_history_buttons = function (fig, msg) {\n",
       "    for (var key in msg) {\n",
       "        if (!(key in fig.buttons)) {\n",
       "            continue;\n",
       "        }\n",
       "        fig.buttons[key].disabled = !msg[key];\n",
       "        fig.buttons[key].setAttribute('aria-disabled', !msg[key]);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_navigate_mode = function (fig, msg) {\n",
       "    if (msg['mode'] === 'PAN') {\n",
       "        fig.buttons['Pan'].classList.add('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    } else if (msg['mode'] === 'ZOOM') {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.add('active');\n",
       "    } else {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Called whenever the canvas gets updated.\n",
       "    this.send_message('ack', {});\n",
       "};\n",
       "\n",
       "// A function to construct a web socket function for onmessage handling.\n",
       "// Called in the figure constructor.\n",
       "mpl.figure.prototype._make_on_message_function = function (fig) {\n",
       "    return function socket_on_message(evt) {\n",
       "        if (evt.data instanceof Blob) {\n",
       "            /* FIXME: We get \"Resource interpreted as Image but\n",
       "             * transferred with MIME type text/plain:\" errors on\n",
       "             * Chrome.  But how to set the MIME type?  It doesn't seem\n",
       "             * to be part of the websocket stream */\n",
       "            evt.data.type = 'image/png';\n",
       "\n",
       "            /* Free the memory for the previous frames */\n",
       "            if (fig.imageObj.src) {\n",
       "                (window.URL || window.webkitURL).revokeObjectURL(\n",
       "                    fig.imageObj.src\n",
       "                );\n",
       "            }\n",
       "\n",
       "            fig.imageObj.src = (window.URL || window.webkitURL).createObjectURL(\n",
       "                evt.data\n",
       "            );\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        } else if (\n",
       "            typeof evt.data === 'string' &&\n",
       "            evt.data.slice(0, 21) === 'data:image/png;base64'\n",
       "        ) {\n",
       "            fig.imageObj.src = evt.data;\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        var msg = JSON.parse(evt.data);\n",
       "        var msg_type = msg['type'];\n",
       "\n",
       "        // Call the  \"handle_{type}\" callback, which takes\n",
       "        // the figure and JSON message as its only arguments.\n",
       "        try {\n",
       "            var callback = fig['handle_' + msg_type];\n",
       "        } catch (e) {\n",
       "            console.log(\n",
       "                \"No handler for the '\" + msg_type + \"' message type: \",\n",
       "                msg\n",
       "            );\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        if (callback) {\n",
       "            try {\n",
       "                // console.log(\"Handling '\" + msg_type + \"' message: \", msg);\n",
       "                callback(fig, msg);\n",
       "            } catch (e) {\n",
       "                console.log(\n",
       "                    \"Exception inside the 'handler_\" + msg_type + \"' callback:\",\n",
       "                    e,\n",
       "                    e.stack,\n",
       "                    msg\n",
       "                );\n",
       "            }\n",
       "        }\n",
       "    };\n",
       "};\n",
       "\n",
       "// from http://stackoverflow.com/questions/1114465/getting-mouse-location-in-canvas\n",
       "mpl.findpos = function (e) {\n",
       "    //this section is from http://www.quirksmode.org/js/events_properties.html\n",
       "    var targ;\n",
       "    if (!e) {\n",
       "        e = window.event;\n",
       "    }\n",
       "    if (e.target) {\n",
       "        targ = e.target;\n",
       "    } else if (e.srcElement) {\n",
       "        targ = e.srcElement;\n",
       "    }\n",
       "    if (targ.nodeType === 3) {\n",
       "        // defeat Safari bug\n",
       "        targ = targ.parentNode;\n",
       "    }\n",
       "\n",
       "    // pageX,Y are the mouse positions relative to the document\n",
       "    var boundingRect = targ.getBoundingClientRect();\n",
       "    var x = e.pageX - (boundingRect.left + document.body.scrollLeft);\n",
       "    var y = e.pageY - (boundingRect.top + document.body.scrollTop);\n",
       "\n",
       "    return { x: x, y: y };\n",
       "};\n",
       "\n",
       "/*\n",
       " * return a copy of an object with only non-object keys\n",
       " * we need this to avoid circular references\n",
       " * http://stackoverflow.com/a/24161582/3208463\n",
       " */\n",
       "function simpleKeys(original) {\n",
       "    return Object.keys(original).reduce(function (obj, key) {\n",
       "        if (typeof original[key] !== 'object') {\n",
       "            obj[key] = original[key];\n",
       "        }\n",
       "        return obj;\n",
       "    }, {});\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.mouse_event = function (event, name) {\n",
       "    var canvas_pos = mpl.findpos(event);\n",
       "\n",
       "    if (name === 'button_press') {\n",
       "        this.canvas.focus();\n",
       "        this.canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    var x = canvas_pos.x * this.ratio;\n",
       "    var y = canvas_pos.y * this.ratio;\n",
       "\n",
       "    this.send_message(name, {\n",
       "        x: x,\n",
       "        y: y,\n",
       "        button: event.button,\n",
       "        step: event.step,\n",
       "        guiEvent: simpleKeys(event),\n",
       "    });\n",
       "\n",
       "    /* This prevents the web browser from automatically changing to\n",
       "     * the text insertion cursor when the button is pressed.  We want\n",
       "     * to control all of the cursor setting manually through the\n",
       "     * 'cursor' event from matplotlib */\n",
       "    event.preventDefault();\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (_event, _name) {\n",
       "    // Handle any extra behaviour associated with a key event\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.key_event = function (event, name) {\n",
       "    // Prevent repeat events\n",
       "    if (name === 'key_press') {\n",
       "        if (event.which === this._key) {\n",
       "            return;\n",
       "        } else {\n",
       "            this._key = event.which;\n",
       "        }\n",
       "    }\n",
       "    if (name === 'key_release') {\n",
       "        this._key = null;\n",
       "    }\n",
       "\n",
       "    var value = '';\n",
       "    if (event.ctrlKey && event.which !== 17) {\n",
       "        value += 'ctrl+';\n",
       "    }\n",
       "    if (event.altKey && event.which !== 18) {\n",
       "        value += 'alt+';\n",
       "    }\n",
       "    if (event.shiftKey && event.which !== 16) {\n",
       "        value += 'shift+';\n",
       "    }\n",
       "\n",
       "    value += 'k';\n",
       "    value += event.which.toString();\n",
       "\n",
       "    this._key_event_extra(event, name);\n",
       "\n",
       "    this.send_message(name, { key: value, guiEvent: simpleKeys(event) });\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onclick = function (name) {\n",
       "    if (name === 'download') {\n",
       "        this.handle_save(this, null);\n",
       "    } else {\n",
       "        this.send_message('toolbar_button', { name: name });\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onmouseover = function (tooltip) {\n",
       "    this.message.textContent = tooltip;\n",
       "};\n",
       "mpl.toolbar_items = [[\"Home\", \"Reset original view\", \"fa fa-home icon-home\", \"home\"], [\"Back\", \"Back to previous view\", \"fa fa-arrow-left icon-arrow-left\", \"back\"], [\"Forward\", \"Forward to next view\", \"fa fa-arrow-right icon-arrow-right\", \"forward\"], [\"\", \"\", \"\", \"\"], [\"Pan\", \"Left button pans, Right button zooms\\nx/y fixes axis, CTRL fixes aspect\", \"fa fa-arrows icon-move\", \"pan\"], [\"Zoom\", \"Zoom to rectangle\\nx/y fixes axis, CTRL fixes aspect\", \"fa fa-square-o icon-check-empty\", \"zoom\"], [\"\", \"\", \"\", \"\"], [\"Download\", \"Download plot\", \"fa fa-floppy-o icon-save\", \"download\"]];\n",
       "\n",
       "mpl.extensions = [\"eps\", \"jpeg\", \"pdf\", \"png\", \"ps\", \"raw\", \"svg\", \"tif\"];\n",
       "\n",
       "mpl.default_extension = \"png\";/* global mpl */\n",
       "\n",
       "var comm_websocket_adapter = function (comm) {\n",
       "    // Create a \"websocket\"-like object which calls the given IPython comm\n",
       "    // object with the appropriate methods. Currently this is a non binary\n",
       "    // socket, so there is still some room for performance tuning.\n",
       "    var ws = {};\n",
       "\n",
       "    ws.close = function () {\n",
       "        comm.close();\n",
       "    };\n",
       "    ws.send = function (m) {\n",
       "        //console.log('sending', m);\n",
       "        comm.send(m);\n",
       "    };\n",
       "    // Register the callback with on_msg.\n",
       "    comm.on_msg(function (msg) {\n",
       "        //console.log('receiving', msg['content']['data'], msg);\n",
       "        // Pass the mpl event to the overridden (by mpl) onmessage function.\n",
       "        ws.onmessage(msg['content']['data']);\n",
       "    });\n",
       "    return ws;\n",
       "};\n",
       "\n",
       "mpl.mpl_figure_comm = function (comm, msg) {\n",
       "    // This is the function which gets called when the mpl process\n",
       "    // starts-up an IPython Comm through the \"matplotlib\" channel.\n",
       "\n",
       "    var id = msg.content.data.id;\n",
       "    // Get hold of the div created by the display call when the Comm\n",
       "    // socket was opened in Python.\n",
       "    var element = document.getElementById(id);\n",
       "    var ws_proxy = comm_websocket_adapter(comm);\n",
       "\n",
       "    function ondownload(figure, _format) {\n",
       "        window.open(figure.canvas.toDataURL());\n",
       "    }\n",
       "\n",
       "    var fig = new mpl.figure(id, ws_proxy, ondownload, element);\n",
       "\n",
       "    // Call onopen now - mpl needs it, as it is assuming we've passed it a real\n",
       "    // web socket which is closed, not our websocket->open comm proxy.\n",
       "    ws_proxy.onopen();\n",
       "\n",
       "    fig.parent_element = element;\n",
       "    fig.cell_info = mpl.find_output_cell(\"<div id='\" + id + \"'></div>\");\n",
       "    if (!fig.cell_info) {\n",
       "        console.error('Failed to find cell for figure', id, fig);\n",
       "        return;\n",
       "    }\n",
       "    fig.cell_info[0].output_area.element.one(\n",
       "        'cleared',\n",
       "        { fig: fig },\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_close = function (fig, msg) {\n",
       "    var width = fig.canvas.width / fig.ratio;\n",
       "    fig.cell_info[0].output_area.element.off(\n",
       "        'cleared',\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "\n",
       "    // Update the output cell to use the data from the current canvas.\n",
       "    fig.push_to_output();\n",
       "    var dataURL = fig.canvas.toDataURL();\n",
       "    // Re-enable the keyboard manager in IPython - without this line, in FF,\n",
       "    // the notebook keyboard shortcuts fail.\n",
       "    IPython.keyboard_manager.enable();\n",
       "    fig.parent_element.innerHTML =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "    fig.close_ws(fig, msg);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.close_ws = function (fig, msg) {\n",
       "    fig.send_message('closing', msg);\n",
       "    // fig.ws.close()\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.push_to_output = function (_remove_interactive) {\n",
       "    // Turn the data on the canvas into data in the output cell.\n",
       "    var width = this.canvas.width / this.ratio;\n",
       "    var dataURL = this.canvas.toDataURL();\n",
       "    this.cell_info[1]['text/html'] =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Tell IPython that the notebook contents must change.\n",
       "    IPython.notebook.set_dirty(true);\n",
       "    this.send_message('ack', {});\n",
       "    var fig = this;\n",
       "    // Wait a second, then push the new image to the DOM so\n",
       "    // that it is saved nicely (might be nice to debounce this).\n",
       "    setTimeout(function () {\n",
       "        fig.push_to_output();\n",
       "    }, 1000);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'btn-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'btn-group';\n",
       "    var button;\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'btn-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        button = fig.buttons[name] = document.createElement('button');\n",
       "        button.classList = 'btn btn-default';\n",
       "        button.href = '#';\n",
       "        button.title = name;\n",
       "        button.innerHTML = '<i class=\"fa ' + image + ' fa-lg\"></i>';\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    // Add the status bar.\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message pull-right';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "\n",
       "    // Add the close button to the window.\n",
       "    var buttongrp = document.createElement('div');\n",
       "    buttongrp.classList = 'btn-group inline pull-right';\n",
       "    button = document.createElement('button');\n",
       "    button.classList = 'btn btn-mini btn-primary';\n",
       "    button.href = '#';\n",
       "    button.title = 'Stop Interaction';\n",
       "    button.innerHTML = '<i class=\"fa fa-power-off icon-remove icon-large\"></i>';\n",
       "    button.addEventListener('click', function (_evt) {\n",
       "        fig.handle_close(fig, {});\n",
       "    });\n",
       "    button.addEventListener(\n",
       "        'mouseover',\n",
       "        on_mouseover_closure('Stop Interaction')\n",
       "    );\n",
       "    buttongrp.appendChild(button);\n",
       "    var titlebar = this.root.querySelector('.ui-dialog-titlebar');\n",
       "    titlebar.insertBefore(buttongrp, titlebar.firstChild);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._remove_fig_handler = function (event) {\n",
       "    var fig = event.data.fig;\n",
       "    fig.close_ws(fig, {});\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (el) {\n",
       "    el.style.boxSizing = 'content-box'; // override notebook setting of border-box.\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (el) {\n",
       "    // this is important to make the div 'focusable\n",
       "    el.setAttribute('tabindex', 0);\n",
       "    // reach out to IPython and tell the keyboard manager to turn it's self\n",
       "    // off when our div gets focus\n",
       "\n",
       "    // location in version 3\n",
       "    if (IPython.notebook.keyboard_manager) {\n",
       "        IPython.notebook.keyboard_manager.register_events(el);\n",
       "    } else {\n",
       "        // location in version 2\n",
       "        IPython.keyboard_manager.register_events(el);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (event, _name) {\n",
       "    var manager = IPython.notebook.keyboard_manager;\n",
       "    if (!manager) {\n",
       "        manager = IPython.keyboard_manager;\n",
       "    }\n",
       "\n",
       "    // Check for shift+enter\n",
       "    if (event.shiftKey && event.which === 13) {\n",
       "        this.canvas_div.blur();\n",
       "        // select the cell after this one\n",
       "        var index = IPython.notebook.find_cell_index(this.cell_info[0]);\n",
       "        IPython.notebook.select(index + 1);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    fig.ondownload(fig, null);\n",
       "};\n",
       "\n",
       "mpl.find_output_cell = function (html_output) {\n",
       "    // Return the cell and output element which can be found *uniquely* in the notebook.\n",
       "    // Note - this is a bit hacky, but it is done because the \"notebook_saving.Notebook\"\n",
       "    // IPython event is triggered only after the cells have been serialised, which for\n",
       "    // our purposes (turning an active figure into a static one), is too late.\n",
       "    var cells = IPython.notebook.get_cells();\n",
       "    var ncells = cells.length;\n",
       "    for (var i = 0; i < ncells; i++) {\n",
       "        var cell = cells[i];\n",
       "        if (cell.cell_type === 'code') {\n",
       "            for (var j = 0; j < cell.output_area.outputs.length; j++) {\n",
       "                var data = cell.output_area.outputs[j];\n",
       "                if (data.data) {\n",
       "                    // IPython >= 3 moved mimebundle to data attribute of output\n",
       "                    data = data.data;\n",
       "                }\n",
       "                if (data['text/html'] === html_output) {\n",
       "                    return [cell, data, j];\n",
       "                }\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "};\n",
       "\n",
       "// Register the function which deals with the matplotlib target/channel.\n",
       "// The kernel may be null if the page has been refreshed.\n",
       "if (IPython.notebook.kernel !== null) {\n",
       "    IPython.notebook.kernel.comm_manager.register_target(\n",
       "        'matplotlib',\n",
       "        mpl.mpl_figure_comm\n",
       "    );\n",
       "}\n"
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<img src=\"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAApQAAAFKCAYAAACjE8/+AAAAAXNSR0IArs4c6QAAIABJREFUeF7t3QuYznX+//G3wTAUI4OcDcM6KxtKohNF5dyEXefK4bLsOpVDGUoO/VdsiBK1igqDsG2tNseacGFzyPk0G0uLyf6cBuN/vT+7951hNN+Z+3PPfOb2/F5X1y7zvd/393583nN/Xr7HXFevXr0qLAgggAACCCCAAAIIZFIgF4Eyk3K8DAEEEEAAAQQQQMAIEChpBAQQQAABBBBAAIGABAiUAfHxYgQQQAABBBBAAAECJT2AAAIIIIAAAgggEJAAgTIgPl6MAAIIIIAAAgggQKCkBxBAAAEEEEAAAQQCEiBQBsTHixFAAAEEEEAAAQQIlPQAAggggAACCCCAQEACBMqA+HgxAggggAACCCCAAIGSHkAAAQQQQAABBBAISIBAGRAfL0YAAQQQQAABBBAgUNIDCCCAAAIIIIAAAgEJECgD4uPFCCCAAAIIIIAAAgRKegABBBBAAAEEEEAgIAECZUB8vBgBBBBAAAEEEECAQEkPIIAAAggggAACCAQkQKAMiI8XI4AAAggggAACCBAo6QEEEEAAAQQQQACBgAQIlAHx8WIEEEAAAQQQQAABAiU9gAACCCCAAAIIIBCQAIEyID5ejAACCCCAAAIIIECgpAcQQAABBBBAAAEEAhIgUAbEx4sRQAABBBBAAAEECJT0AAIIIIAAAggggEBAAgTKgPh4MQIIIIAAAggggACBkh5AAAEEEEAAAQQQCEiAQBkQHy9GAAEEEEAAAQQQIFDSAwgggAACCCCAAAIBCRAoA+LjxQgggAACCCCAAAIESnoAAQQQQAABBBBAICABAmVAfLwYAQQQQAABBBBAgEBJDyCAAAIIIIAAAggEJECgDIiPFyOAAAIIIIAAAggQKOkBBBBAAAEEEEAAgYAECJQB8fFiBBBAAAEEEEAAAQIlPYAAAggggAACCCAQkACBMiA+XowAAggggAACCCBAoKQHEEAAAQQQQAABBAISIFAGxPfziy9cuCDbtm2TYsWKSZ48eSxVpQwCCCCAAAIIuCpw+fJl+fHHH6VWrVqSP39+VzczS7aLQGmJeePGjVK/fn1L1SiDAAIIIIAAAjlFYMOGDVKvXr2csrlB2U4CpSXWQ4cOSXR0tGhTlSxZ0lJVyiCAAAIIIICAqwLHjh0zO5MOHjwoFSpUcHUzs2S7CJSWmP/5z39K2bJlJTExUcqUKWOpKmUQQAABBBBAwFUB5v6fR4ZAaalLaSpLkJRBAAEEEEAghwgw9xMorbcqTWWdlIIIIIAAAgg4LcDcT6C03qA0lXVSCiKAAAIIIOC0AHM/gdJ6g9JU1kkpiAACCCCAgNMCzP0ESusNSlNZJ6UgAggggAACTgsw9xMorTcoTWWdlIIIIIAAAgg4LcDcT6C03qA0lXVSCiKAAAIIIOC0AHM/gdJ6g9JU1kkpiAACCCCAgNMCzP0ESusNSlNZJ6UgAggggAACTgsw9xMorTcoTWWdlIIIIIAAAgg4LcDcT6C03qA0lXVSCiKAAAIIIOC0AHM/gdJ6g9JU1kkpiAACCCCAgNMCzP0ESusNSlNZJ6UgAggggAACTgsw9xMorTcoTWWdlIIIIIAAAgg4LcDcH6KBcs+ePVK3bl2ZOnWqdOvWzXzKrVu3yoABA2TTpk1StGhR6d+/vwwePNgvkJKSIqNHj5ZZs2bJ6dOnpVGjRjJ9+nSJiYnJUBPTVBniYmUEEEAAAQRyvABzfwgGykuXLknDhg1NcJwzZ44JlCdPnpSqVatK69atZeDAgZKQkCB9+/Y1gbF79+5GQcOk/llfU7p0aRk6dKgcOHBAduzYIeHh4Z6bnabyTMWKCCCAAAIIhIQAc38IBsrhw4ebwPjVV1/5A+W4ceNk2rRpcujQIcmTJ4/51LpefHy87Nq1S5KTkyUqKkomTpwovXv3Nj9PSkqSUqVKyezZs6VDhw6eG56m8kzFiggggAACCISEAHN/iAXKNWvWSJs2bczh7XLlyvkDZYsWLSQyMlLmzZvn/8QrV66Upk2byvHjx03QbNCggezevVuqVKniX0cPe9euXdvsufS60FRepVgPAQQQQACB0BBg7g+hQKl7FOvUqSO6N7JTp06SK1cuf6DUUNi8eXOZMGGC/xPv3LlTatSoIRs3bpQjR45Iu3bt5Ny5cxIREeFfJzY21vzd8uXLPXd8sJoq+XKK/JB03vN2sCICCCCAAAII3ChQOjJCwvOEWaUJ1txvdSOzqFiuq1evXs2i9wrK23Ts2FH0I3z00Uem/rWBUi+s0ZA5ZswY/3vr+ZGVKlWStWvXmj2UnTt3litXrkhY2M9N1qVLFzl69Kjo3sybLWfOnBH9z7ccO3ZM6tevL4mJiVKmTBlrn/Xgv8/KQ/9vlbV6FEIAAQQQQOBWFPhq8IMSHVXQ6kcnUP7MmaMD5dy5c2XYsGGybds2KVKkyA2BslatWqKHvdPaQ7l582Zz8U379u3T3EN58eJFWbp06U0bLy4uzlzQc/1CoLT6u0oxBBBAAAEErAgQKK0w3rRIjg6UDz30kKxbt07y5cvn/4Bnz541f9a9kOXLlze3CtLg6VvSOody3759Zn3foudQ6mF0vaDnZktW7aHkkHdwfwGojgACCCBwawhwyDu445yjA+UPP/wg58+nPr+wcuXKZo/kM888I/Pnz5cZM2bI/v37JXfu3EZS92guXrzYXOWteyGLFy8ukyZNkp49e5qfc5V3cBuO6ggggAACCISKAIe8fx7JHB0o02rIa8+hPHHihLkPZcuWLc39JTds2CB9+vQxIbNr167m5SNGjJCZM2ea2wRVqFBBhgwZYs6t1MPo3IcyVH7l+RwIIIAAAgjYFyBQ3iKBUj+mXs2tT8fZsmWLlCxZUgYNGiT9+vXzC+gFOXpvSr2xue7tbNy4sTnUreEyIwtNlREt1kUAAQQQQCDnCzD3h3CgzK72pKmyS573RQABBBBAIHsEmPsJlNY7j6ayTkpBBBBAAAEEnBZg7idQWm9Qmso6KQURQAABBBBwWoC5n0BpvUFpKuukFEQAAQQQQMBpAeZ+AqX1BqWprJNSEAEEEEAAAacFmPsJlNYblKayTkpBBBBAAAEEnBZg7idQWm9Qmso6KQURQAABBBBwWoC5n0BpvUFpKuukFEQAAQQQQMBpAeZ+AqX1BqWprJNSEAEEEEAAAacFmPsJlNYblKayTkpBBBBAAAEEnBZg7idQWm9Qmso6KQURQAABBBBwWoC5n0BpvUFpKuukFEQAAQQQQMBpAeZ+AqX1BqWprJNSEAEEEEAAAacFmPsJlNYblKayTkpBBBBAAAEEnBZg7idQWm9Qmso6KQURQAABBBBwWoC5n0BpvUFpKuukFEQAAQQQQMBpAeZ+AqX1BqWprJNSEAEEEEAAAacFmPsJlNYblKayTkpBBBBAAAEEnBZg7idQWm9Qmso6KQURQAABBBBwWoC5n0BpvUFpKuukFEQAAQQQQMBpAeZ+AqX1BqWprJNSEAEEEEAAAacFmPsJlNYblKayTkpBBBBAAAEEnBZg7idQWm9Qmso6KQURQAABBBBwWoC5n0BpvUFpKuukFEQAAQQQQMBpAeZ+AqX1BqWprJNSEAEEEEAAAacFmPsJlNYblKayTkpBBBBAAAEEnBZg7idQWm9Qmso6KQURQAABBBBwWoC5n0BpvUFpKuukFEQAAQQQQMBpAeZ+AqX1BqWprJNSEAEEEEAAAacFmPsJlNYblKayTkpBBBBAAAEEnBZg7idQWm9Qmso6KQURQAABBBBwWoC5n0BpvUFpKuukFEQAAQQQQMBpAeZ+AqX1BqWprJNSEAEEEEAAAacFmPsJlNYblKayTkpBBBBAAAEEnBZg7idQWm9Qmso6KQURQAABBBBwWoC5n0BpvUFpKuukFEQAAQQQQMBpAeZ+AqX1BqWprJNSEAEEEEAAAacFmPsJlNYblKayTkpBBBBAAAEEnBZg7idQWm9Qmso6KQURQAABBBBwWoC5n0BpvUFpKuukFEQAAQQQQMBpAeZ+AqX1BqWprJNSEAEEEEAAAacFmPsJlNYblKayTkpBBBBAAAEEnBZg7idQWm9Qmso6KQURQAABBBBwWoC5n0BpvUFpKuukFEQAAQQQQMBpAeZ+AqX1BqWprJNSEAEEEEAAAacFmPsJlNYblKayTkpBBBBAAAEEnBZg7idQWm9Qmso6KQURQAABBBBwWoC5P4QC5YkTJ2TQoEHy17/+Vc6fPy9NmjSR119/XapXr24+5datW2XAgAGyadMmKVq0qPTv318GDx7sF0hJSZHRo0fLrFmz5PTp09KoUSOZPn26xMTEZKiJaaoMcbEyAggggAACOV6AuT+EAuW9994rYWFh8qc//UkKFiwoL730knz99deyb98+EzCrVq0qrVu3loEDB0pCQoL07dvXBMbu3bsbBQ2T+uc5c+ZI6dKlZejQoXLgwAHZsWOHhIeHe252msozFSsigAACCCAQEgLM/SESKE+ePCn9+vWTkSNHSo0aNcyn+sc//iF33XWXbNiwQVauXCnTpk2TQ4cOSZ48eczPhw8fLvHx8bJr1y5JTk6WqKgomThxovTu3dv8PCkpSUqVKiWzZ8+WDh06eG54msozFSsigAACCCAQEgLM/SESKK/vxuPHj5vD2atWrTKB8emnn5bIyEiZN2+ef1UNmU2bNhVdV4NmgwYNZPfu3VKlShX/OnrYu3bt2mbPpdeFpvIqxXoIIIAAAgiEhgBzv4OBUg9P586d2xxm/v7772XFihXSsGFD85+X5fnnn5d33nlH8uXLJ59++qk0a9bMhMLmzZvLhAkT/CV27txp9mZu3LhRjhw5Iu3atZNz585JRESEf53Y2Fjzd8uXL7/pW585c0b0P99y7NgxqV+/viQmJkqZMmW8bDLrIIAAAggggEAOFiBQOhYo16xZY85zXLhwobmYRvcW6nmR//d//yfz5883exrTWzQoaih96623zB7JdevWiQbDTp06yZgxY/wv1/MjK1WqJGvXrjV7KDt37ixXrlwx7+dbunTpIkePHjWHzG+2xMXFmfMvr18IlOmNFD9HAAEEEEAgNAQIlI4FSj3ErCFyypQp5txFvUp779695v/r1ddbtmzx3Hl61XatWrXMoWzdC9miRYs091Bu3rzZXHzTvn37NPdQXrx4UZYuXcoeSs/yrIgAAggggMCtJUCgdCxQFihQQLZv3y4VK1Y0AbBs2bIyc+ZMOXz4sLlKW/c8prXoLYO+/PJLsydSD5f7Ft2jqYej9e/0VkFz5871/yytcyj1inDda+lbNODWqVPHXNDjdaGpvEqxHgIIIIAAAqEhwNzvWKAsVqyYOQQdHR1tAqDewkdD4bfffiutWrWSf/3rX2l2nt5j8u677zah8uGHHzbrXLp0yYTQli1bSokSJWTGjBmyf/9+f+AcNmyYLF682Fy0o3shixcvLpMmTZKePXua13OVd2j8kvMpEEAAAQQQCLYAgdKxQKl7GHUv5B133CGLFi0SvcBFQ2CvXr3MXks9jzKt5erVq+aiGz0XUi/IKVKkiIwdO1Y+//xzc0Pz/Pnz+8Ol3l9SbyXUp08fEzK7du1qSo4YMcLsDdXD6xUqVJAhQ4aYetu2beM+lMH+TaQ+AggggAACOViAQOlYoPzxxx/NfSD1nMZRo0aZC3T06Te6h1Iv1Lnzzjtv2m4//fST6F7HJUuWmL2LDzzwgNnj6LsvpZ5HqU/H0fMwS5YsaerqvSt9i16Qo/em1L2iGmobN25sDnVruMzIQlNlRIt1EUAAAQQQyPkCzP2OBcq0WkpvOp6RJ9Vkd1vSVNk9Arw/AggggAACWSvA3O9goNTD3HrYWs9tnDx5sqxevdpcra3nQ+aEhabKCaPENiKAAAIIIGBPgLnfsUCpV1nrbX4KFy4sOjgaKl944QX57LPP5IsvvvB8c3N7LZLxSjRVxs14BQIIIIAAAjlZgLnfsUCpV3Lrld66h7JQoULmedzly5eXbt26mafZ6N5K1xeayvURYvsQQAABBBCwK8Dc71ig1DCpoVGfknP77bebQKlXd+ueSt1zqRfeuL7QVK6PENuHAAIIIICAXQHmfscCpd7uR+9DWbNmzVSBUq/y1tsCnTp1ym4HBKEaTRUEVEoigAACCCDgsABzv2OBskOHDmaLPvzwQ4mMjJTvvvvOHPpu27atufH4ggULHG6n/24aTeX8ELGBCCCAAAIIWBVg7ncsUB49elQefPBBOXnypDm8rVd262MX9Ubna9asMedTur7QVK6PENuHAAIIIICAXQHmfscCpW7OuXPnzBNx9AbkKSkp5vD3b3/7W7OnMicsNFVOGCW2EQEEEEAAAXsCzP0OBkrdpAsXLsiePXskX7585qKcvHnz2hv1IFeiqYIMTHkEEEAAAQQcE2DudyxQ6uMPBw8ebJ6xrU/I0SUiIsLci3LkyJGSK1cux1roxs2hqZwfIjYQAQQQQAABqwLM/Y4FyhEjRsjs2bPllVdeMbcJ0oCpV33HxcWZ526PHj3aagMEoxhNFQxVaiKAAAIIIOCuAHO/Y4GyRIkSMmfOHGnRokWqrlm0aJEJlPpYRtcXmsr1EWL7EEAAAQQQsCvA3O9YoNRbBX3zzTdSrVq1VCO9c+dOs8fyP//5j90OCEI1mioIqJREAAEEEEDAYQHmfscC5YABA+TEiRPy/vvvS3h4uH/revToIXny5JG3337b4Xb676bRVM4PERuIAAIIIICAVQHmfgcC5cMPP+zfisuXL8u6deukZMmSUq9ePcmdO7e5fVBiYqLoc74XLlxotQGCUYymCoYqNRFAAAEEEHBXgLnfgUDZvXt3zx2i51e6vtBUro8Q24cAAggggIBdAeZ+BwKl3SHN/mo0VfaPAVuAAAIIIIBAVgow9zsYKHVQpk2bZp7jrTc0r1GjhvTq1UvKlSuXlb2R6feiqTJNxwsRQAABBBDIkQLM/Y4Fyu3bt0vjxo2lQIECUr9+fXMfyk2bNsnZs2dl/fr1Jly6vtBUro8Q24cAAggggIBdAeZ+xwJl8+bNpWDBgvLhhx+axy7qoo9h1Gd56/8uX77cbgcEoRpNFQRUSiKAAAIIIOCwAHO/Y4Hytttuk4SEBKlZs2aqttHD37rnMikpyeF2+u+m0VTODxEbiAACCCCAgFUB5n7HAqXeLkj3Qv76179ONdAbN26Upk2bEiittj/FEEAAAQQQQMCGAIHSsUCph7aPHj1q7jd5xx13mK3797//LbGxsaJP0YmPj7cx7kGtQVMFlZfiCCCAAAIIOCfA3O9YoNQBadiwoZw6dUqqVKkiuXLlkt27d0uRIkVk9erVUrFiReea6PoNoqmcHyI2EAEEEEAAAasCzP2OBcqrV6+aK7rnzp0resW3/rlWrVrSqVMnKVy4sNXBD1YxmipYstRFAAEEEEDATQHmfscC5T333COzZ8+W2rVru9kxHraKpvKAxCoIIIAAAgiEkABzv2OBMioqSr799lupVKlSjm0zmirHDh0bjgACCCCAQKYEmPsdC5Svv/66rFixQoYMGSIxMTESERGRamBzwtNyaKpM/S7yIgQQQAABBHKsAHO/Y4EyLCzMv0V6QY5v0XMp9c/65BzXF5rK9RFi+xBAAAEEELArwNzvWKDUK7l/aWnSpIndDghCNZoqCKiURAABBBBAwGEB5n7HAuW1vaK3DsqdO3eOubrbt+00lcO/8WwaAggggAACQRBg7ncwUOp5lFOmTJFjx46ZrYuOjpYXXnhBnnvuuSC0gP2SNJV9UyoigAACCCDgsgBzv2OBcsKECTJmzBjp37+/3HfffZKSkiLr16+X6dOnyxtvvCHPP/+8y/1kto2mcn6I2EAEEEAAAQSsCjD3OxYoy5YtK6+99pp07tw51UDrvSnHjRsne/futdoAwShGUwVDlZoIIIAAAgi4K8Dc71igvO2222Tz5s3msYvXLhok69SpI+fOnXO3m/63ZTSV80PEBiKAAAIIIGBVgLnfsUDZtWtXuf3222Xq1KmpBnr06NHmUYwLFiyw2gDBKEZTBUOVmggggAACCLgrwNzvWKAcOHCgOV+yWrVqorcIyps3r2zcuFHWrl0rrVq1ksjISP8W62FwFxeaysVRYZsQQAABBBAIngBzv2OB8qGHHvI82l999ZXndbNyRZoqK7V5LwQQQAABBLJfgLnfsUCZ/S0R+BbQVIEbUgEBBBBAAIGcJMDcT6C03q80lXVSCiKAAAIIIOC0AHM/gdJ6g9JU1kkpiAACCCCAgNMCzP0ESusNSlNZJ6UgAggggAACTgsw9zsQKD/99FNp1qyZ5M+f3+lm8bpxNJVXKdZDAAEEEEAgNASY+x0IlHoz8127dkmZMmWkYsWK5jZBRYsWzbEdRlPl2KFjwxFAAAEEEMiUAHO/A4GyePHi0qtXL3n00UdFbxu0ePFiKVKkSJoD2rhx40wNdFa+iKbKSm3eCwEEEEAAgewXYO53IFBOmDBBhg8fLlevXpVcuXKZ/01r0Z9duXIl+7smnS2gqZwfIjYQAQQQQAABqwLM/Q4ESt2En376SZKSkiQ6Olo2bNggxYoVS3Ogy5cvf9MGOHXqlAmmy5cvlzNnzkjt2rVl/Pjx0qhRI/OarVu3yoABA2TTpk3mkHr//v1l8ODB/nopKSmij3icNWuWnD592rxOn9oTExOToaajqTLExcoIIIAAAgjkeAHmfkcCpW8zVq9eLffff7/kyZMnw82lF/acOHHCPAdcA+m0adNMONy8ebP5c9WqVaV169aij3dMSEiQvn37msDYvXt3814aJvXPc+bMkdKlS8vQoUPlwIEDsmPHDgkPD/e8PTSVZypWRAABBBBAICQEmPsdC5S6ObqHcezYsfLdd9+ZZ3lXr15dhgwZIm3atLlp0+3bt08qV64s69evl4YNG5r19NB5lSpVpGPHjhIREWEC5qFDh/xhVfdmxsfHmwuCkpOTJSoqSiZOnCi9e/c2r9c9pqVKlRJ9ZniHDh08NzxN5ZmKFRFAAAEEEAgJAeZ+xwLl0qVLpW3btiY86gU4es7kmjVrZNmyZSb8tWzZMs3G0/D39ddfyyOPPCL58uXzr6OHqx9//HGzpzEyMlLmzZvn/9nKlSuladOmcvz4cRM0GzRoILt37zYh1LfoYW89dK57Lr0uNJVXKdZDAAEEEEAgNASY+x0LlHXr1jVh8qWXXkrVYWPGjDF7LvX8Sq/LggULJDY2VvQ+lyNGjJDmzZuLXgDkW3bu3Ck1atQwtyk6cuSItGvXTs6dO2f2ZvoWfb3+nb73zRY9X1P/8y3Hjh2T+vXrS2JiorkVEgsCCCCAAAIIhLYAgdKxQKlhbtu2bTdcCLN3716pU6eOCXdeFj30rXsmdY/lkiVLTL1OnTqJBlPfonstK1WqJGvXrjV7KDt37mz2iIaFhfnX6dKlixw9elR0b+bNlri4OHP+5fULgdLLSLEOAggggAACOV+AQOlYoNSA9+abb0qLFi1SddeKFSvkueeeM+EuvUUPm2t4vPfee82eRQ2ptWrVMjXT2kOpF+1ouGzfvn2aeygvXrwoWvNmC3so0xsRfo4AAggggEBoCxAoHQuUeqh77ty58tZbb5mrvXVZt26duSK7VatWMmXKlF/sSL3CW28NpOdhfvDBB/7zKTVM6q2CtLZvSescSr24R0Otb9FzKHXPqF7Q43WhqbxKsR4CCCCAAAKhIcDc71igvHDhgrmiWs971BuZ66JXa2sg/OSTT6RAgQI37TwNoRo89f6Sb7zxRqpD13o/yhkzZsj+/fsld+7cpsawYcPMU3n0Km/dC6lP7Jk0aZL07NnT/JyrvEPjl5xPgQACCCCAQLAFCJSOBUrf5mjI03MpNUzq4epq1ar9Yi/s2bNHatasKU8++eQNV2TrIW8NjHofSr1KXO8vqRf39OnTx4TMrl27mtp64c7MmTPNbYIqVKhgblWk51bqdnAfymD/KlIfAQQQQACBnCtAoHQ0UGa0pV577TUTCNNaNDC+99575mpu3Xu5ZcsWKVmypAwaNEj69evnf4lekKP3ptQbm58/f97ctkgPdWu4zMhCU2VEi3URQAABBBDI+QLM/SESKF1qRZrKpdFgWxBAAAEEEAi+AHM/gdJ6l9FU1kkpiAACCCCAgNMCzP0ESusNSlNZJ6UgAggggAACTgsw9xMorTcoTWWdlIIIIIAAAgg4LcDc71igPHjwoLzwwguyfft20VsIXb/oDchdX2gq10eI7UMAAQQQQMCuAHO/Y4FSbySuz8LWZ2jnz5//htEeNWqU3Q4IQjWaKgiolEQAAQQQQMBhAeZ+xwJlwYIFzbO169at63Db/PKm0VQ5dujYcAQQQAABBDIlwNzvWKDUIDl58mRzD8icutBUOXXk2G4EEEAAAQQyJ8Dc71igTEhIME+w0RuQR0dHp3p8om5qTgiaNFXmfhl5FQIIIIAAAjlVgLnfsUD57rvvmkB5+fLlG3pKn+2tT7NxfaGpXB8htg8BBBBAAAG7Asz9jgXKsmXLmudx6x7KAgUK3DDa5cuXt9sBQahGUwUBlZIIIIAAAgg4LMDc71ig1ItyduzYkeHnZ7vUYzSVS6PBtiCAAAIIIBB8AeZ+xwJl+/btpUWLFtKjR4/gj36Q3oGmChIsZRFAAAEEEHBUgLnfsUA5depUGTZsmDRv3lx+9atfSd68eVO1zssvv+xoK/28WTSV80PEBiKAAAIIIGBVgLnfsUCpV3bfbNGLcnhSjtX+pxgCCCCAAAIIWBAgUDoWKC2MabaXoKmyfQjYAAQQQAABBLJUgLmfQGm94Wgq66QURAABBBBAwGkB5n7HAmVYWJjooe2bLdyH0unfJzYOAQQQQACBW1KAQOlYoHzvvfdSBcpLly7J3r17Rf9+0qRJ8pvf/Mb5RqWpnB8iNhABBBBAAAGrAsz9jgVUkWS+AAAZYUlEQVTKm43uvHnzZO7cufLZZ59ZbYBgFKOpgqFKTQQQQAABBNwVYO7PIYHy8OHDUr16dTl79qy73fS/LaOpnB8iNhABBBBAAAGrAsz9OSRQjh8/XqZNmyaJiYlWGyAYxWiqYKhSEwEEEEAAAXcFmPsdC5R6H8prL8q5evWqnDlzRpKSkmTs2LHy4osvuttN7KF0fmzYQAQQQAABBIIhQKB0LFDGxcXdcJV3eHi43H///dK4ceNg9ID1mjSVdVIKIoAAAggg4LQAc79jgdLpbvG4cTSVRyhWQwABBBBAIEQEmPsdDJTr16+XdevWSXJysugh72sXnuUdIr95fAwEEEAAAQRCSIBA6Vig1Itvhg8fLkWKFJFChQqlajWe5R1Cv3l8FAQQQAABBEJIgEDpWKAsU6aM9OjRQ8aMGZNj24ymyrFDx4YjgAACCCCQKQHmfscCZYECBeS7776TmJiYTA2oCy+iqVwYBbYBAQQQQACBrBNg7ncsUDZr1ky6d+8uHTt2zLousPxONJVlUMohgAACCCDguABzv2OBctasWTJ06FATKqtWrSr58uVL1UJdunRxvKVEaCrnh4gNRAABBBBAwKoAc79jgTIsLOymA6wX5Vy5csVqAwSjGE0VDFVqIoAAAggg4K4Ac79jgdLdVvG+ZTSVdyvWRAABBBBAIBQEmPsJlNb7mKayTkpBBBBAAAEEnBZg7idQWm9Qmso6KQURQAABBBBwWoC5n0BpvUFpKuukFEQAAQQQQMBpAeZ+AqX1BqWprJNSEAEEEEAAAacFmPsJlNYblKayTkpBBBBAAAEEnBZg7idQWm9Qmso6KQURQAABBBBwWoC5n0BpvUFpKuukFEQAAQQQQMBpAeZ+AqX1BqWprJNSEAEEEEAAAacFmPsJlNYblKayTkpBBBBAAAEEnBZg7idQWm9Qmso6KQURQAABBBBwWoC5n0BpvUFpKuukFEQAAQQQQMBpAeZ+AqX1BqWprJNSEAEEEEAAAacFmPsJlNYblKayTkpBBBBAAAEEnBZg7g/RQPnqq6/KypUrZdWqVf5PuHXrVhkwYIBs2rRJihYtKv3795fBgwf7f56SkiKjR4+WWbNmyenTp6VRo0Yyffp0iYmJyVAT01QZ4mJlBBBAAAEEcrwAc38IBsrJkyfLwIEDpXHjxv5AefLkSalataq0bt3a/CwhIUH69u1rAmP37t2NgoZJ/fOcOXOkdOnSMnToUDlw4IDs2LFDwsPDPTc7TeWZihURQAABBBAICQHm/hAKlD/88IM8++yzsnbtWilbtqyUKFHCHyjHjRsn06ZNk0OHDkmePHnMpx4+fLjEx8fLrl27JDk5WaKiomTixInSu3dv8/OkpCQpVaqUzJ49Wzp06OC54Wkqz1SsiAACCCCAQEgIMPeHUKBctmyZzJ8/X/Rw95gxY0x49B3ybtGihURGRsq8efP8n1gPiTdt2lSOHz9u1m3QoIHs3r1bqlSp4l9HD3vXrl3b7Ln0utBUXqVYDwEEEEAAgdAQYO4PoUB5bUt269YtVaDUUNi8eXOZMGGCf7WdO3dKjRo1ZOPGjXLkyBFp166dnDt3TiIiIvzrxMbGmr9bvny5546nqTxTsSICCCCAAAIhIcDcf4sESr2wplOnTmbPpW/R8yMrVapkDpHrHsrOnTvLlStXJCwszL9Oly5d5OjRo+YCn5stZ86cEf3Ptxw7dkzq168viYmJUqZMmZD4ReFDIIAAAggggMDNBQiUt0igrFWrluhh77T2UG7evNlcfNO+ffs091BevHhRli5detMuiouLMxf0XL8QKPnqQQABBBBA4NYQIFDeIoFSw6TeKmju3Ln+T5zWOZT79u0zey19i55DWadOHXNBD3sob40vBT4lAggggAACGRUgUN4igXL8+PEyY8YM2b9/v+TOndt86mHDhsnixYvNVd66F7J48eIyadIk6dmzp/k5V3ln9NeJ9RFAAAEEELg1BQiUt0igPHHihLkPZcuWLc39JTds2CB9+vQxIbNr165GYcSIETJz5kxzm6AKFSrIkCFDzLmV27Zt4z6Ut+b3A58aAQQQQAABTwIEylskUOrH1Ku59ek4W7ZskZIlS8qgQYOkX79+fgG9IEfvTak3Nj9//ry5Mboe6tZwmZGFpsqIFusigAACCCCQ8wWY+0M0UGZna9JU2anPeyOAAAIIIJD1Asz9BErrXUdTWSelIAIIIIAAAk4LMPcTKK03KE1lnZSCCCCAAAIIOC3A3E+gtN6gNJV1UgoigAACCCDgtABzP4HSeoPSVNZJKYgAAggggIDTAsz9BErrDUpTWSelIAIIIIAAAk4LMPcTKK03KE1lnZSCCCCAAAIIOC3A3E+gtN6gNJV1UgoigAACCCDgtABzP4HSeoPSVNZJKYgAAggggIDTAsz9BErrDUpTWSelIAIIIIAAAk4LMPcTKK03KE1lnZSCCCCAAAIIOC3A3E+gtN6gNJV1UgoigAACCCDgtABzP4HSeoPSVNZJKYgAAggggIDTAsz9BErrDUpTWSelIAIIIIAAAk4LMPcTKK03KE1lnZSCCCCAAAIIOC3A3E+gtN6gNJV1UgoigAACCCDgtABzP4HSeoPSVNZJKYgAAggggIDTAsz9BErrDUpTWSelIAIIIIAAAk4LMPcTKK03KE1lnZSCCCCAAAIIOC3A3E+gtN6gNJV1UgoigAACCCDgtABzP4HSeoPSVNZJKYgAAggggIDTAsz9BErrDUpTWSelIAIIIIAAAk4LMPcTKK03KE1lnZSCCCCAAAIIOC3A3E+gtN6gNJV1UgoigAACCCDgtABzP4HSeoPSVNZJKYgAAggggIDTAsz9BErrDUpTWSelIAIIIIAAAk4LMPcTKK03KE1lnZSCCCCAAAIIOC3A3E+gtN6gNJV1UgoigAACCCDgtABzP4HSeoPSVNZJKYgAAggggIDTAsz9BErrDUpTWSelIAIIIIAAAk4LMPcTKK03KE1lnZSCCCCAAAIIOC3A3E+gtN6gNJV1UgoigAACCCDgtABzP4HSeoPSVNZJKYgAAggggIDTAsz9BErrDUpTWSelIAIIIIAAAk4LMPcTKK03KE1lnZSCCCCAAAIIOC3A3E+gtN6gNJV1UgoigAACCCDgtABzP4HSeoPSVNZJKYgAAggggIDTAsz9BErrDUpTWSelIAIIIIAAAk4LMPcTKK03KE1lnZSCCCCAAAIIOC3A3E+gtN6gNJV1UgoigAACCCDgtABzP4HSeoPSVNZJKYgAAggggIDTAsz9BErrDUpTWSelIAIIIIAAAk4LMPcTKK03KE1lnZSCCCCAAAIIOC3A3E+gtN6gNJV1UgoigAACCCDgtABzP4HSeoPSVNZJKYgAAggggIDTAsz9BErrDUpTWSelIAIIIIAAAk4LMPcTKK03KE1lnZSCCCCAAAIIOC3A3E+gTNWgKSkpMnr0aJk1a5acPn1aGjVqJNOnT5eYmBjPjUxTeaZiRQQQQAABBEJCgLmfQJmqkTVMaoCcM2eOlC5dWoYOHSoHDhyQHTt2SHh4uKemp6k8MbESAggggAACISPA3E+g9AskJydLVFSUTJw4UXr37m3+PikpSUqVKiWzZ8+WDh06eGp8msoTEyshgAACCCAQMgLM/QRKv8CGDRukQYMGsnv3bqlSpYr/7/Wwd+3atc2eSy8LTeVFiXUQQAABBBAIHQHmfgKlXyA+Pl7atWsn586dk4iICP/fx8bGmr9bvny5p86nqTwxsRICCCCAAAIhI8DcT6D0C3zwwQfSuXNnuXLlioSFhfn/vkuXLnL06FFZuXJlmo1/5swZ0f98S2JiojRs2FB0j2fJkiVD5peFD4IAAggggAACaQscO3ZM6tevLwcPHpQKFSrc0ky5rl69evVWFli0aJG0b98+zT2UFy9elKVLl6bJExcXZ64MZ0EAAQQQQACBW1tAdybVq1fvlka45QOl7xzKffv2SaVKlfzNoOdQ1qlTR6ZNm+ZpD+WFCxdE91JGR0dLnjx5rDaV719A7P20ypqhYoxBhriCtjLjEDRaz4UZA89UQV2RcQgqr6fivjFYtmyZPProo5I/f35PrwvVlW75QKl7IYsXLy6TJk2Snj17mnHOzFXewWwQztEIpq632oyBN6dgr8U4BFs4/fqMQfpGWbEG45AVyr/8HoxBap9bPlAqx4gRI2TmzJnmNkF6DsSQIUPk0KFDsm3bNs/3oQxma9O0wdT1Vpsx8OYU7LUYh2ALp1+fMUjfKCvWYByyQplAmRFlAqWIuSBn+PDh5sbm58+fl8aNG5tD3a6cYMsXR0ZaOjjrMgbBcc1oVcYho2L212cM7JtmpiLjkBk1u69hDNhDabejsqCaXk2uh+QHDhwohQoVyoJ35C2uF2AM3OgJxiH7x4ExyP4x0C1gHLJ/HBgDAmX2dyFbgAACCCCAAAIIhJAAh7xDaDD5KAgggAACCCCAQHYIECizQ533RAABBBBAAAEEQkiAQBlCg8lHQQABBBBAAAEEskOAQJkd6h7fMyUlxTyNZ9asWXL69GnRm61Pnz5dYmJiPFZgtfQETp06Za7w12e26wnWtWvXlvHjxxtrXbZu3SoDBgyQTZs2SdGiRaV///4yePBgf1nGKD3hjP18z549UrduXZk6dap069aNMcgYX8Br//nPfzb9f+DAAfOgB30i2NNPP804BCzrvcClS5eM+9y5c833/l133SUTJkwwj/blO8m7Y2bXfPXVV80jl1etWuUvYWMeSK9GZrfXpdcRKF0ajeu2RcOkBki9nVHp0qVl6NCh5ot+x44dTtwf02E6z5vWrFkzOXHihAkwxYoVM7eL0gC/efNm8+eqVatK69atzRX2CQkJ0rdvXzMm3bt3N+/BGHmmTndFnUh10tTwrj2vgfLkyZOMQbpydlb44IMPjPkf//hHeeKJJ2T+/Pkm2Kxbt06qVKnCONhhTrfKqFGj5O2335b3339fKlasaMLkJ598It9//73ky5ePcUhXMPMrTJ482XzX660DfYHSxneQlxqZ32p3XkmgdGcsUm1JcnKyREVFycSJE6V3797mZ649wcdROs+bpY/brFy5sqxfv97/r399tL1Onh07dpSIiAgTMPUm977HaerezPj4eNm1a5cwRp6pPa2othrav/rqK3+gHDduHGPgSS+wlbTvNby0b99eXn/9dX+xxx57TB588EHzZ34XAjP2+mrdI/nII4+YYK+LHjkpXLiwLFy4UHQPPuPgVdL7ej/88IM8++yzsnbtWilbtqyUKFHCHyhtfAelV8P7lrq9JoHS0fHxPWN89+7dJuD4Fj0Uq4dldS8ZS2ACGtC//vpr8+Wt//L3LXpKweOPP272BkdGRsq8efP8P9NDIU2bNpXjx4+boNmgQQNhjAIbB331mjVrpE2bNuYUg3LlyvkDZYsWLRiDwHnTraD/QKpWrZrZM3/33XffsD7jkC6htRX0+0hvmP3555+bcPPGG2/IyJEjzZGp3/3ud/w+WJP+uZA+i1v3yOvh7jFjxpjvdt8eShu9n14NffxzKCwESkdHUfeCtWvXTs6dO2f2lPmW2NhY83d6zh+LfYEFCxaIGn/66afmkZzNmzc3h5x8y86dO6VGjRqyceNGOXLkCGNkYQg02NepU0f0X/GdOnWSXLly+QOl/uOJMbCAnE4J7fdWrVqZPTSvvPKKbNmyRaKjo02Qeeqpp8w/YhmH4I+DvsP27dvlmWeeEf2uyZ07t4SFhZm9ky1btmQcsmAI9LSPawOljd5Pr8Y999yTBZ8s+G9BoAy+cabeQc9n6ty5s3kspH6h+JYuXbrI0aNHzUnDLHYF9NC37pnUPQRLliwxFz9pwNF/sfoW38UKOvHqlw5jFPgY6OkFesj1o48+MsWuDZSMQeC+Xir4vm80ROo5fBrwFy1aJGPHjpW//e1v0qtXL34XvEBaWEf/UTtlyhQZMmSIOXf+nXfekY8//tjsxW/bti3jYMH4l0pcHyhtfAelV8N3EWiQP1rQyxMog06cuTfQL3M9nymtPZQXL16UpUuXZq4wr0pTQD01PN57771m76/uFa5Vq5booYq09lDqoUENl4xRYA2lV7IOGzZMtm3bJkWKFLkhUDIGgfl6fbVe9KF7xfRUmj59+vhfphfn6KJ74/ld8KqZ+fUOHz5szuv+8ssv5YEHHvAX0v+v59Tred+MQ+Z9vbzy+kBp4zsovRppnWbiZVtdW4dA6dqI/G97fOdQ6heI3r7Dt+i/ZHTvgZ6YzWJHQK/w1lsD6b/+dU+N73xK/eLWWwVp6PEtaZ1DyRhlfhweeughcxXxteewnj171vxZ+758+fKMQeZ5Pb9SzyW+//77zSHva/eW6J0l9B9YFSpUYBw8a2Z+Rd8pN+fPn5f8+fP7C+neyr/85S/8PmSe1vMrrw+UNuaB9GpwDqXn4WHFzAjoXkhtskmTJknPnj1NCa7yzozkL7/mrbfeMrcC0vtL6snv155eoPfjmzFjhuzfv9+cy6SL7k1bvHixucqbMQp8PPTqSp08r110D43uFdY9ZnqiPGMQuHN6FXQM9MrWl19+OdV9VvW8ybx585q7IDAO6SkG/nO9y8F9990n3377rdSvX99fUE/F0YCpR1AYh8Cdf6nC9YHSxjyQXo3gfqKsq84eyqyzzvA76UUhM2fOlNmzZ5s9BPqvVD1vTw8PhoeHZ7geL0gtoLfgqFmzpjz55JM3XDWvh7w1MOp9KPVkeN1To3uN9XCgfqF37drVFGOM7HfVtedQ6j1CGQP7xmlV1Ctc9TZl+p2jYUbPadWAqYdfq1evzjhkwTDogxL0Nk0//vij+U4qU6aM6M3m9YI13ZOvt3bi9yG4A3F9oLTxHeSlRnA/VdZUJ1BmjXOm3kUvyNF78+lNnnUPgt5sVQ91a7hkCVzgtddeM4EwrUUD43vvvWeu5ta9l3rVa8mSJWXQoEHSr18//0sYo8DH4foK1wZK/RljYN/4ZhX1iIieAqJ7jvU2Qnrjfr36m3HIujHQp+Po1fV6qoH+fz3/Tr+rmjRpwjhkwTBcHyht9X5632NZ8NGC/hYEyqAT8wYIIIAAAggggEBoCxAoQ3t8+XQIIIAAAggggEDQBQiUQSfmDRBAAAEEEEAAgdAWIFCG9vjy6RBAAAEEEEAAgaALECiDTswbIIAAAggggAACoS1AoAzt8eXTIYCARQG9w4JeBRoXF2ex6s+lVq1aJXqz94MHD970bg7B3oagfDCKIoBAyAsQKEN+iPmACCBgS0DvD6j3KL3ttttslUxVJzk5WU6dOiXFihXz30z/+jciUAaFnqIIIBCgAIEyQEBejgACCGSlAIEyK7V5LwQQ8CpAoPQqxXoIIJBjBPTm6HrTen0Ouz7xaPXq1RIdHS0vvfSSeV77Tz/9ZJ6SNGbMGGnWrJn/c33xxRfm8Zo7d+40TyXRG9nro099h6CvD3MrVqyQV155RbZv3y6FChWSjh07ytixY/3PYdbtePvtt+WTTz4xTzq54447zI3x9T3SWq4/5K3bqTfWX7p0qXk6lj7oYPLkyUE97J5jBpkNRQABpwQIlE4NBxuDAAI2BDTIRUVFyWeffSaXL182z0Du1KmT7NixwwQyfaTdsmXL5MUXXzTPZn/iiSdk69atUq9ePfnDH/5gQqT+WR+1qU8rSStQLlmyRNq1a2fOp4yNjZW9e/ea9fWxhYsWLTIfQ7ejcOHC5ukz+vcacPURh2vWrJEHHnjgho96faB87LHH5MiRIyaU3n777Sbg/v3vf5dRo0YF7TxOG/7UQACBW0+AQHnrjTmfGIGQF9Agp8FQHyWoy759+6Ry5crmMY733HOP//PrIzYPHz4sGuT0/+vz3b/55hv/z998802zhzCtQNmgQQMpW7asLFy40L++hlR99rvu4dRHF+p2DBgwwIRY36J7KYcMGZLmXsprA6XvWfIrV66URx55xLz8+PHjUr58eROEg3VhUMg3Bx8QAQSCIkCgDAorRRFAIDsFNMhNnz7d7DHUZcGCBWYvYsGCBVNt1qVLlyQyMtIENX1mctOmTf0hVFfUvZR33313moGyQIEC5hnLv//97/019RC11vvoo4/kmWeeMYFyxowZ0qtXL/86eti8e/fuZi/j9cu1gXLTpk3y9NNPm8Pzejjdt1SvXt18FgJldnYY740AAtcLECjpCQQQCDkBDXJz5swx5xrq8vHHH0uHDh1k8+bN5tDxtUvu3LnN+ZUaHJs0aZJqb+IvBUq92nv8+PFmD6Rv0Su0ixYtag55t23b1gTKa7dD1/uli2rSCpR6yF1Dqm+pU6eOtGnThkAZcl3LB0IgZwsQKHP2+LH1CCCQhsD1QU7PndSLcJYvX27Ol/QteuFOWFiYubBGw+euXbskISHB//OZM2dK796909xDqedElitXLtUhb714pnXr1qkOeWc2UF65ckViYmJSbXNSUpKULl3aHDJnDyWtjwACLgkQKF0aDbYFAQSsCKS1Z/Cpp54yh7CnTZtmwmV8fLwMHTpU3n33XXMIWq/Uvuuuu2Tw4MHSo0cPEwqff/550XtPHjp0yJy7eO3eRT2Mroe1R48ebf5Xz7/UQ+xaQ8+l1CWQPZT6XnpoW8/7fOedd+TOO+80513qhUYjR44kUFrpFIoggIAtgf8PDjieMGOrMRcAAAAASUVORK5CYII=\" width=\"599.9999869953505\">"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# plot the probe number against all regins\n",
    "pb_num_dict = {_k:len(_v.kept_probes) for _k,_v in filtered_pb_dict_seq.items()}\n",
    "\n",
    "pb_nums = np.array(list(pb_num_dict.values()))\n",
    "plt.figure(figsize=(6,3), dpi=100)\n",
    "plt.plot(pb_nums)\n",
    "#plt.plot(np.ones(len(pb_nums))*20)\n",
    "plt.ylabel(f\"num of probes\", labelpad=1)\n",
    "plt.xlabel(f\"region id\", labelpad=1)\n",
    "plt.ylim([0, num_probes_kept*1.1])\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'4': 61,\n",
       " '5': 56,\n",
       " '6': 56,\n",
       " '7': 45,\n",
       " '8': 52,\n",
       " '9': 54,\n",
       " '10': 64,\n",
       " '11': 57,\n",
       " '12': 35,\n",
       " '13': 49,\n",
       " '1': 76,\n",
       " '2': 73,\n",
       " '3': 59,\n",
       " '14': 51,\n",
       " '15': 35,\n",
       " '16': 37,\n",
       " '17': 34,\n",
       " '18': 32,\n",
       " '19': 35,\n",
       " 'X': 7}"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# load num per chr to select \n",
    "chr_2_region_num = pickle.load(open(os.path.join(library_folder, 'chr_2_region_num.pkl'), 'rb'))\n",
    "\n",
    "chr_2_region_num"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[36, 56, 109, 129, 168, 188, 229, 249, 285]"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Select 2 regions each from the first 5 chromosome to be used as controls\n",
    "sel_regions = []\n",
    "cum_reg_end = 0\n",
    "for _chr in range(5):\n",
    "    reg_end = chr_2_region_num[str(_chr+1)]\n",
    "    cum_reg_end+=reg_end\n",
    "    sel_regions.append(cum_reg_end-40)\n",
    "    sel_regions.append(cum_reg_end-20)\n",
    "    \n",
    "for _reg in sel_regions:\n",
    "    if _reg not in list(filtered_pb_dict_seq.keys()):\n",
    "        print('Region not found.')\n",
    "\n",
    "# remove one to keep 9 regions\n",
    "sel_regions = sel_regions[:9]\n",
    "\n",
    "sel_regions"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{36: <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2284c9c27f0>,\n",
       " 56: <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2284c9b71c0>,\n",
       " 109: <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2284c9b7cd0>,\n",
       " 129: <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2284c9b7a30>,\n",
       " 168: <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2284c9a3580>,\n",
       " 188: <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2284c9a3c10>,\n",
       " 229: <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2284da1b940>,\n",
       " 249: <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2284da4a310>,\n",
       " 285: <ImageAnalysis3.library_tools.LibraryDesigner.pb_reports_class at 0x2284dd51610>}"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Generate subset filtered_pb_dict_seq_sel\n",
    "filtered_pb_dict_seq_sel = {}\n",
    "for _reg in sel_regions:\n",
    "    filtered_pb_dict_seq_sel[_reg] = filtered_pb_dict_seq[_reg]\n",
    "\n",
    "filtered_pb_dict_seq_sel"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### 5.13 Assemble probes "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Picked primer: ID: W1A03_primer_2\n",
      "Name: W1A03_primer_2\n",
      "Description: W1A03_primer_2\n",
      "Number of features: 0\n",
      "Seq('CCCGCAATGGCTGACAACCG')\n",
      "- Picked primer: ID: W1A10_primer_9\n",
      "Name: W1A10_primer_9\n",
      "Description: W1A10_primer_9\n",
      "Number of features: 0\n",
      "Seq('TAATACGACTCACTATAGGGATTGCCGCATGGTTTCCG')\n"
     ]
    }
   ],
   "source": [
    "# Use same primer set\n",
    "from ImageAnalysis3 import library_tools\n",
    "primer_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Primers'\n",
    "primers = library_tools.assemble.load_primers([2,9], _primer_folder=primer_folder)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Use different readout \n",
    "# Make region_2_readout_dict for sequential lib\n",
    "\n",
    "region_2_readout_dict_seq = {}\n",
    "# start from u0\n",
    "for _i, _reg in enumerate(filtered_pb_dict_seq_sel.keys()):\n",
    "    region_2_readout_dict_seq[_reg] = []\n",
    "    region_2_readout_dict_seq[_reg].append(f'u{_i*2}')\n",
    "    region_2_readout_dict_seq[_reg].append(f'u{_i*2+1}')\n",
    "\n",
    "    \n",
    "\n",
    "# save\n",
    "pickle.dump(region_2_readout_dict_seq, open(os.path.join(library_folder,'region_2_readout_dict_seq.pkl'), 'wb'))\n",
    "region_2_readout_dict_seq    \n",
    "\n",
    "# collect types\n",
    "# one type for this library which is 'u'\n",
    "\n",
    "readout_usage_by_type = {}\n",
    "for _region, _infos in region_2_readout_dict_seq.items():\n",
    "    # for each bit \n",
    "    for _info in _infos:\n",
    "        if _info[0] not in readout_usage_by_type:\n",
    "            readout_usage_by_type[_info[0]] = []\n",
    "        if int(_info[1:]) not in readout_usage_by_type[_info[0]]:\n",
    "            readout_usage_by_type[_info[0]].append(int(_info[1:]))\n",
    "            \n",
    "readout_usage_by_type\n",
    "readout_usage_dict = {_type:len(_v) for _type,_v in readout_usage_by_type.items()}\n",
    "readout_dict ={}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{36: ['u0', 'u1'],\n",
       " 56: ['u2', 'u3'],\n",
       " 109: ['u4', 'u5'],\n",
       " 129: ['u6', 'u7'],\n",
       " 168: ['u8', 'u9'],\n",
       " 188: ['u10', 'u11'],\n",
       " 229: ['u12', 'u13'],\n",
       " 249: ['u14', 'u15'],\n",
       " 285: ['u16', 'u17']}"
      ]
     },
     "execution_count": 15,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "region_2_readout_dict_seq"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [],
   "source": [
    "reload(library_tools.assemble)\n",
    "overwrite_readout_usage = True\n",
    "readout_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Readouts'\n",
    "readout_start = int(1 + 95/2)\n",
    "num_color = 2\n",
    "\n",
    "# generate the 'snr_before_630_750.fasta' and 'snr_before_630_647.fasta' (see above) first before proceeding\n",
    "\n",
    "seq_readouts = library_tools.assemble.load_readouts(readout_usage_dict['u'], \n",
    "                                                       _type='snr_before_630', \n",
    "                                                       _readout_folder=readout_folder, \n",
    "                                                       _num_colors=2, # 750, 647\n",
    "                                                       _start_id=readout_start,\n",
    "                                                       )\n",
    "readout_dict['u'] = seq_readouts"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'u': [SeqRecord(seq=Seq('GTAGATAACACACGCCTCGGTGCTGGATTA'), id='NDB_496', name='NDB_496', description='NDB_496', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGATACGATCATGTTAGTCGCCGTCGGCAT'), id='NDB_329', name='NDB_329', description='NDB_329', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCTCACTAGGTGTTACGTGACCGCGGGAAT'), id='NDB_553', name='NDB_553', description='NDB_553', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TACGTGGGCTCGATTCGGTACTTAGCCTGG'), id='NDB_488', name='NDB_488', description='NDB_488', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCGCTGTGAGCTAACGCGCTTTGAAGCACT'), id='NDB_586', name='NDB_586', description='NDB_586', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGTAAACTAGTTGTCACCCGTCGTGTTCCG'), id='NDB_416', name='NDB_416', description='NDB_416', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTAAAGTTGCGATCTAGGGTCCCTAACTAA'), id='NDB_595', name='NDB_595', description='NDB_595', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATCGATGGAGTTCGTCTTACCGCGAGTCTT'), id='NDB_503', name='NDB_503', description='NDB_503', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGGATGCCGCGTCCCTTATTAACGAAAGCG'), id='NDB_484', name='NDB_484', description='NDB_484', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGAGATACATTAGCGTCCCAACATCTAGTT'), id='NDB_413', name='NDB_413', description='NDB_413', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGATAAGATCAACTCACTCGCGGGACGACT'), id='NDB_442', name='NDB_442', description='NDB_442', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGTCTGACCGTTAAGTTGTTTCTCGCCGGG'), id='NDB_383', name='NDB_383', description='NDB_383', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GACTAGTTAGGGAGTCGCTCGCCGAGAATC'), id='NDB_475', name='NDB_475', description='NDB_475', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAGAATTACTAGCTACGCTCTGACACGGGA'), id='NDB_407', name='NDB_407', description='NDB_407', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTGCACGAAGTTTACGCTTATCAGATCGTT'), id='NDB_607', name='NDB_607', description='NDB_607', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CATTAATACGTAGAGTTCACGACGTCCGTA'), id='NDB_500', name='NDB_500', description='NDB_500', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAGATTTGGTATACCCGGACCCGTATATCT'), id='NDB_610', name='NDB_610', description='NDB_610', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGCGTGTTAGCGCTAGACTACCTCATTGAG'), id='NDB_593', name='NDB_593', description='NDB_593', dbxrefs=[])]}"
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "readout_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "save\n"
     ]
    }
   ],
   "source": [
    "# save\n",
    "readout_usage_file = os.path.join(library_folder, 'readout_usage_seq.pkl')\n",
    "if not os.path.isfile(readout_usage_file) or overwrite_readout_usage:\n",
    "    print('save')\n",
    "    pickle.dump(readout_dict, open(readout_usage_file, 'wb'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Assemble probes by given target sequences, readouts and primers.\n",
      "-- included readout types: ['u']\n",
      "--- assemblying 400 probes in region: 36\n",
      "--- assemblying 400 probes in region: 56\n",
      "--- assemblying 400 probes in region: 109\n",
      "--- assemblying 400 probes in region: 129\n",
      "--- assemblying 400 probes in region: 168\n",
      "--- assemblying 400 probes in region: 188\n",
      "--- assemblying 400 probes in region: 229\n",
      "--- assemblying 400 probes in region: 249\n",
      "--- assemblying 400 probes in region: 285\n",
      "-- 3600 probes assembled in total.\n",
      "-- saving 3600 probes into file:\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_20_sequential\\candidate_probes.fasta\n",
      "-- saving readout_summary into file:\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_20_sequential\\readout_summary.pkl\n"
     ]
    }
   ],
   "source": [
    "# assemble full-length probes using primer, readouts, and targeting regions (from filtered_pb_dict)\n",
    "\n",
    "reload(library_tools.assemble)\n",
    "from ImageAnalysis3.library_tools.assemble import Assemble_probes\n",
    "\n",
    "cand_probes, readout_summary = Assemble_probes(library_folder, filtered_pb_dict_seq_sel, \n",
    "                                               region_2_readout_dict_seq, readout_dict, primers, \n",
    "                                               primer_len=20, \n",
    "                                               num_readout_per_probe=3, \n",
    "                                               unique_readout_per_probe=True,\n",
    "                                               rc_targets=False, overwrite=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### 5.14 Check probes "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Total probe loaded: 3600\n"
     ]
    }
   ],
   "source": [
    "# folder for this library\n",
    "# candidate full-length probe filename\n",
    "candidate_basename = 'candidate_probes.fasta' # old version\n",
    "\n",
    "\n",
    "\n",
    "# load assembled full-length probes from the \"candidate_probes.fasta\" file\n",
    "full_records = []\n",
    "\n",
    "\n",
    "with open(os.path.join(library_folder, candidate_basename), 'r') as handle:\n",
    "    for record in SeqIO.parse(handle, \"fasta\"):\n",
    "        full_records.append(record)\n",
    "\n",
    "print(f\"Total probe loaded: {len(full_records)}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [],
   "source": [
    "num_readout_per_probe = 3\n",
    "primer_len=20\n",
    "target_len=42\n",
    "min_num_probes = 40"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- Checking primer usage, total probes: 3600\n",
      "True\n"
     ]
    }
   ],
   "source": [
    "# check\n",
    "\n",
    "import ImageAnalysis3.library_tools.quality_check as check\n",
    "reload(check)\n",
    "primer_check = check._check_primer_usage(full_records, primers[0], primers[1], primer_len=primer_len)\n",
    "print(primer_check)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "gene: 36 -> 400\n",
      "gene: 56 -> 400\n",
      "gene: 109 -> 400\n",
      "gene: 129 -> 400\n",
      "gene: 168 -> 400\n",
      "gene: 188 -> 400\n",
      "gene: 229 -> 400\n",
      "gene: 249 -> 400\n",
      "gene: 285 -> 400\n",
      "True\n"
     ]
    }
   ],
   "source": [
    "# get num of pbs for each region for the full records before the check steps below\n",
    "\n",
    "reload(check)\n",
    "reg_size_dic, len_check = check._check_region_size(full_records, \n",
    "                                                   min_size=min_num_probes)\n",
    "print(len_check)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [],
   "source": [
    "# construct internal map\n",
    "reload(check)\n",
    "#int_map = check._construct_internal_map(full_records, library_folder, save=True) # file is big; not necessary to save\n",
    "\n",
    "int_map = check._construct_internal_map(full_records, library_folder, save=False) "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- Sequence:loc_4:40762947-40854743_gene_229_pb_330_pos_87082_strand_-_readouts_[NDB_475_u,NDB_475_u,NDB_475_u]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:80, dumped\n",
      "--- Sequence:loc_4:40762947-40854743_gene_229_pb_331_pos_87157_strand_-_readouts_[NDB_407_u,NDB_407_u,NDB_407_u]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:55, dumped\n",
      "--- Sequence:loc_4:40762947-40854743_gene_229_pb_332_pos_87244_strand_-_readouts_[NDB_475_u,NDB_475_u,NDB_475_u]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:64, dumped\n",
      "--- Sequence:loc_4:40762947-40854743_gene_229_pb_334_pos_87423_strand_-_readouts_[NDB_475_u,NDB_475_u,NDB_475_u]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:72, dumped\n",
      "-- total probes removed by internal screening: 4\n"
     ]
    }
   ],
   "source": [
    "# check between probes\n",
    "kept_records, removed_count = check._check_between_probes(full_records, int_map, \n",
    "                                                          primer_len=primer_len, \n",
    "                                                          target_len=target_len)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [],
   "source": [
    "# save kept records as filtered probes after internal check\n",
    "with open(os.path.join(library_folder, 'filtered_full_probes.fasta'), 'w') as output_handle:\n",
    "    SeqIO.write(kept_records, output_handle, \"fasta\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 5.2 Subtract the used probes from combinatory lib"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "115776\n"
     ]
    }
   ],
   "source": [
    "# get all final used probes \n",
    "# load final probes\n",
    "\n",
    "library_folder =r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_20_sequential'\n",
    "final_pb_filename = os.path.join(library_folder, 'blast_full_probes_from_comb.fasta')\n",
    "# SeqIO\n",
    "final_pb_records = []\n",
    "with open(final_pb_filename, 'r') as handle:\n",
    "    for record in SeqIO.parse(handle, \"fasta\"):\n",
    "        final_pb_records.append(record)\n",
    "\n",
    "print(len(final_pb_records))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "100%|| 3596/3596 [00:01<00:00, 1854.56it/s]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- The number of used probe for each region equal 120.0.\n"
     ]
    }
   ],
   "source": [
    "#get unused probes by subtracting the final_pb_records from the # load all the \n",
    "from tqdm import tqdm\n",
    "\n",
    "unused_pb_records = []\n",
    "# final pb sequence for the sel regions above\n",
    "final_pb_records_sel = [_pb for _pb in final_pb_records if int(_pb.id.split('gene_')[1].split('_pb')[0]) in sel_regions]\n",
    "\n",
    "final_pb_records_sel_tar = [_pb.seq[40:82] for _pb in final_pb_records_sel]\n",
    "\n",
    "for _pb in tqdm(kept_records):\n",
    "    \n",
    "    if _pb.seq[40:82] not in final_pb_records_sel_tar:\n",
    "        unused_pb_records.append(_pb)\n",
    "        \n",
    "print (f'-- The number of used probe for each region equal {(len(kept_records) -len(unused_pb_records))/9}.')  # use pb for each should be 120 if no issue\n",
    "\n",
    "\n",
    "# save kept records as filtered probes after removing used \n",
    "with open(os.path.join(library_folder, 'filtered_full_probes.fasta'), 'w') as output_handle:\n",
    "    SeqIO.write(unused_pb_records, output_handle, \"fasta\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 5.3 Check against the ctp13-comb internal map"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 56,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "115776\n"
     ]
    }
   ],
   "source": [
    "# load probes for ctp-13\n",
    "\n",
    "ctp13_filename = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_20_sequential\\blast_full_probes_from_comb.fasta'\n",
    "\n",
    "# convert probes into SeqIO list as full_records_ctp11\n",
    "full_records_ctp13 = []\n",
    "\n",
    "with open(ctp13_filename, 'r') as _handle:\n",
    "    for _record in SeqIO.parse(_handle, \"fasta\"):\n",
    "        full_records_ctp13.append(_record)\n",
    "        \n",
    "print(len(full_records_ctp13))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 57,
   "metadata": {},
   "outputs": [],
   "source": [
    "# construct internal map using loaded ctp-11 (as full_records_ctp11)\n",
    "reload(check)\n",
    "#int_map = check._construct_internal_map(full_records, library_folder, save=True) # file is big; not necessary to save\n",
    "\n",
    "int_map_ctp13 = check._construct_internal_map(full_records_ctp13, library_folder, save=False) "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 61,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- Sequence:loc_1:79813369-79876804_gene_36_pb_186_pos_46971_strand_+_readouts_[NDB_496_u,NDB_496_u,NDB_496_u]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:2, dumped\n",
      "--- Sequence:loc_2:84845394-84900174_gene_109_pb_222_pos_39323_strand_-_readouts_[NDB_586_u,NDB_586_u,NDB_586_u]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:2, dumped\n",
      "--- Sequence:loc_2:84845394-84900174_gene_109_pb_227_pos_39667_strand_+_readouts_[NDB_416_u,NDB_416_u,NDB_416_u]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:2, dumped\n",
      "--- Sequence:loc_2:132260572-132326514_gene_129_pb_172_pos_44876_strand_+_readouts_[NDB_595_u,NDB_595_u,NDB_595_u]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:2, dumped\n",
      "--- Sequence:loc_2:132260572-132326514_gene_129_pb_241_pos_49497_strand_+_readouts_[NDB_503_u,NDB_503_u,NDB_503_u]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:2, dumped\n",
      "--- Sequence:loc_3:50892858-50950176_gene_168_pb_22_pos_9462_strand_-_readouts_[NDB_484_u,NDB_484_u,NDB_484_u]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:3, dumped\n",
      "--- Sequence:loc_4:40762947-40854743_gene_229_pb_259_pos_70867_strand_+_readouts_[NDB_407_u,NDB_407_u,NDB_407_u]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:2, dumped\n",
      "-- total probes removed by internal screening: 7\n",
      "2509\n"
     ]
    }
   ],
   "source": [
    "# check filtered probes (kept_records, excluding primer regions by primer_len) against probes from ctp11 (int_map_ctp11)\n",
    "\n",
    "# try use stringent hit th to avoid overlapping/similar targeting regions between libraries\n",
    "# try _max_internal_hits=1 for 17-mer\n",
    "# both strands considered by default using the function\n",
    "\n",
    "unused_pb_records_new, removed_count_ctp13 = check._check_between_probes(unused_pb_records, int_map_ctp13, \n",
    "                                                          primer_len=primer_len, \n",
    "                                                          target_len=target_len,\n",
    "                                                                     _max_internal_hits=1)\n",
    "\n",
    "print(len(unused_pb_records_new))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 5.4 Check against the ctp11-comb internal map"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 62,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "114719\n"
     ]
    }
   ],
   "source": [
    "# load probes for ctp-11\n",
    "\n",
    "ctp11_filename = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-11_brain\\mouse_genome_1000\\blast_full_probes.fasta'\n",
    "\n",
    "# convert probes into SeqIO list as full_records_ctp11\n",
    "full_records_ctp11 = []\n",
    "\n",
    "with open(ctp11_filename, 'r') as _handle:\n",
    "    for _record in SeqIO.parse(_handle, \"fasta\"):\n",
    "        full_records_ctp11.append(_record)\n",
    "        \n",
    "print(len(full_records_ctp11))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 63,
   "metadata": {},
   "outputs": [],
   "source": [
    "# construct internal map using loaded ctp-11 (as full_records_ctp11)\n",
    "reload(check)\n",
    "#int_map = check._construct_internal_map(full_records, library_folder, save=True) # file is big; not necessary to save\n",
    "\n",
    "int_map_ctp11 = check._construct_internal_map(full_records_ctp11, library_folder, save=False) "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 64,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- Sequence:loc_1:79813369-79876804_gene_36_pb_13_pos_34877_strand_+_readouts_[NDB_329_u,NDB_329_u,NDB_329_u]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:2, dumped\n",
      "--- Sequence:loc_1:79813369-79876804_gene_36_pb_57_pos_37953_strand_+_readouts_[NDB_329_u,NDB_329_u,NDB_329_u]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:2, dumped\n",
      "--- Sequence:loc_1:152397309-152486632_gene_56_pb_172_pos_54869_strand_-_readouts_[NDB_553_u,NDB_553_u,NDB_553_u]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:3, dumped\n",
      "--- Sequence:loc_1:152397309-152486632_gene_56_pb_269_pos_62995_strand_+_readouts_[NDB_488_u,NDB_488_u,NDB_488_u]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:2, dumped\n",
      "--- Sequence:loc_1:152397309-152486632_gene_56_pb_284_pos_72645_strand_+_readouts_[NDB_553_u,NDB_553_u,NDB_553_u]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:2, dumped\n",
      "--- Sequence:loc_2:132260572-132326514_gene_129_pb_234_pos_49066_strand_+_readouts_[NDB_595_u,NDB_595_u,NDB_595_u]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:2, dumped\n",
      "--- Sequence:loc_3:97659551-97729267_gene_188_pb_184_pos_47459_strand_-_readouts_[NDB_442_u,NDB_442_u,NDB_442_u]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:2, dumped\n",
      "--- Sequence:loc_4:100963562-101018943_gene_249_pb_89_pos_11831_strand_-_readouts_[NDB_500_u,NDB_500_u,NDB_500_u]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:2, dumped\n",
      "--- Sequence:loc_5:30381241-30452747_gene_285_pb_50_pos_37171_strand_+_readouts_[NDB_610_u,NDB_610_u,NDB_610_u]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:2, dumped\n",
      "--- Sequence:loc_5:30381241-30452747_gene_285_pb_213_pos_48984_strand_+_readouts_[NDB_593_u,NDB_593_u,NDB_593_u]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:2, dumped\n",
      "--- Sequence:loc_5:30381241-30452747_gene_285_pb_287_pos_61433_strand_+_readouts_[NDB_593_u,NDB_593_u,NDB_593_u]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:2, dumped\n",
      "-- total probes removed by internal screening: 11\n",
      "2498\n"
     ]
    }
   ],
   "source": [
    "# check filtered probes (kept_records, excluding primer regions by primer_len) against probes from ctp11 (int_map_ctp11)\n",
    "\n",
    "# try use stringent hit th to avoid overlapping/similar targeting regions between libraries\n",
    "# try _max_internal_hits=1 for 17-mer\n",
    "# both strands considered by default using the function\n",
    "\n",
    "unused_pb_records_new_new, removed_count_ctp11 = check._check_between_probes(unused_pb_records_new, int_map_ctp11, \n",
    "                                                          primer_len=primer_len, \n",
    "                                                          target_len=target_len,\n",
    "                                                                     _max_internal_hits=1)\n",
    "\n",
    "print(len(unused_pb_records_new_new))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 65,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "gene: 36 -> 277\n",
      "gene: 56 -> 277\n",
      "gene: 109 -> 278\n",
      "gene: 129 -> 277\n",
      "gene: 168 -> 279\n",
      "gene: 188 -> 279\n",
      "gene: 229 -> 275\n",
      "gene: 249 -> 279\n",
      "gene: 285 -> 277\n"
     ]
    }
   ],
   "source": [
    "reload(check)\n",
    "\n",
    "# get num of pbs for each region after check against ctp11 library\n",
    "reg_size_dic_temp, len_check_temp = check._check_region_size(unused_pb_records_new_new, \n",
    "                                                   min_size=min_num_probes)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 67,
   "metadata": {},
   "outputs": [],
   "source": [
    "# save kept records after check against ctp11  #use default filename or specify the filename when using blastn screen probes\n",
    "with open(os.path.join(library_folder, 'filtered_full_probes.fasta'), 'w') as output_handle:\n",
    "    SeqIO.write(unused_pb_records_new_new, output_handle, \"fasta\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 5.5 BLAST screening of probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 68,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "USAGE\n",
      "  blastn [-h] [-help] [-import_search_strategy filename]\n",
      "    [-export_search_strategy filename] [-task task_name] [-db database_name]\n",
      "    [-dbsize num_letters] [-gilist filename] [-seqidlist filename]\n",
      "    [-negative_gilist filename] [-negative_seqidlist filename]\n",
      "    [-taxids taxids] [-negative_taxids taxids] [-taxidlist filename]\n",
      "    [-negative_taxidlist filename] [-entrez_query entrez_query]\n",
      "    [-db_soft_mask filtering_algorithm] [-db_hard_mask filtering_algorithm]\n",
      "    [-subject subject_input_file] [-subject_loc range] [-query input_file]\n",
      "    [-out output_file] [-evalue evalue] [-word_size int_value]\n",
      "    [-gapopen open_penalty] [-gapextend extend_penalty]\n",
      "    [-perc_identity float_value] [-qcov_hsp_perc float_value]\n",
      "    [-max_hsps int_value] [-xdrop_ungap float_value] [-xdrop_gap float_value]\n",
      "    [-xdrop_gap_final float_value] [-searchsp int_value]\n",
      "    [-sum_stats bool_value] [-penalty penalty] [-reward reward] [-no_greedy]\n",
      "    [-min_raw_gapped_score int_value] [-template_type type]\n",
      "    [-template_length int_value] [-dust DUST_options]\n",
      "    [-filtering_db filtering_database]\n",
      "    [-window_masker_taxid window_masker_taxid]\n",
      "    [-window_masker_db window_masker_db] [-soft_masking soft_masking]\n",
      "    [-ungapped] [-culling_limit int_value] [-best_hit_overhang float_value]\n",
      "    [-best_hit_score_edge float_value] [-subject_besthit]\n",
      "    [-window_size int_value] [-off_diagonal_range int_value]\n",
      "    [-use_index boolean] [-index_name string] [-lcase_masking]\n",
      "    [-query_loc range] [-strand strand] [-parse_deflines] [-outfmt format]\n",
      "    [-show_gis] [-num_descriptions int_value] [-num_alignments int_value]\n",
      "    [-line_length line_length] [-html] [-sorthits sort_hits]\n",
      "    [-sorthsps sort_hsps] [-max_target_seqs num_sequences]\n",
      "    [-num_threads int_value] [-mt_mode int_value] [-remote] [-version]\n",
      "\n",
      "DESCRIPTION\n",
      "   Nucleotide-Nucleotide BLAST 2.12.0+\n",
      "\n",
      "Use '-help' to print detailed descriptions of command line arguments\n"
     ]
    }
   ],
   "source": [
    "# if blastn is installed and can be found by jupyter, the below command will show the help page for blastn\n",
    "!blastn -h"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 69,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Blast probe against \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\blast_database\\GRCm38_ensembl\n",
      "-- number of probes loaded: 2498\n",
      "-- processing region: 109 with 278 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_20_sequential\\blast\\probe_gene_109.fasta\n",
      "--- *blasting region: 109\n",
      "--- total time for blast 109: 13.299001932144165\n",
      "-- processing region: 129 with 277 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_20_sequential\\blast\\probe_gene_129.fasta\n",
      "--- *blasting region: 129\n",
      "--- total time for blast 129: 10.940000772476196\n",
      "-- processing region: 168 with 279 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_20_sequential\\blast\\probe_gene_168.fasta\n",
      "--- *blasting region: 168\n",
      "--- total time for blast 168: 9.980000734329224\n",
      "-- processing region: 188 with 279 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_20_sequential\\blast\\probe_gene_188.fasta\n",
      "--- *blasting region: 188\n",
      "--- total time for blast 188: 11.33600115776062\n",
      "-- processing region: 229 with 275 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_20_sequential\\blast\\probe_gene_229.fasta\n",
      "--- *blasting region: 229\n",
      "--- total time for blast 229: 8.381000280380249\n",
      "-- processing region: 249 with 279 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_20_sequential\\blast\\probe_gene_249.fasta\n",
      "--- *blasting region: 249\n",
      "--- total time for blast 249: 10.396857023239136\n",
      "-- processing region: 285 with 277 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_20_sequential\\blast\\probe_gene_285.fasta\n",
      "--- *blasting region: 285\n",
      "--- total time for blast 285: 10.144004344940186\n",
      "-- processing region: 36 with 277 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_20_sequential\\blast\\probe_gene_36.fasta\n",
      "--- *blasting region: 36\n",
      "--- total time for blast 36: 9.093994379043579\n",
      "-- processing region: 56 with 277 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_20_sequential\\blast\\probe_gene_56.fasta\n",
      "--- *blasting region: 56\n",
      "--- total time for blast 56: 9.301007986068726\n"
     ]
    }
   ],
   "source": [
    "import ImageAnalysis3.library_tools.quality_check as check\n",
    "reload(check)\n",
    "\n",
    "blast_database = os.path.join(reference_folder, 'blast_database', 'GRCm38_ensembl')\n",
    "\n",
    "# run blast using the pre-generated blastn database (on NAS2)\n",
    "# use kept_records_ctp11 after check against ctp11 library\n",
    "\n",
    "# output as blast files saved in the library folder; \n",
    "# make sure to delete any old blast files if re-run this step (with different input)\n",
    "\n",
    "check.Blast_probes(unused_pb_records_new_new, library_folder, num_threads=32, \n",
    "                   blast_db=blast_database, blast_evalue=100, overwrite=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 70,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Number of probes loaded: 2498\n",
      "- Number of regions in this library: 9\n",
      "-- checking probes in region: 36\n",
      "--- gene=36, id=89 removed by soft count = 51\n",
      "--- gene=36, id=200 removed by soft count = 53\n",
      "--- number of probes: 277 , kept by blast: 275 , if remove dups: 275\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 56\n",
      "--- gene=56, id=123 removed by soft count = 77\n",
      "--- gene=56, id=206 removed by soft count = 138\n",
      "--- gene=56, id=235 removed by soft count = 76\n",
      "--- number of probes: 277 , kept by blast: 274 , if remove dups: 273\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 109\n",
      "--- gene=109, id=1 removed by soft count = 87\n",
      "--- gene=109, id=261 removed by soft count = 76\n",
      "--- number of probes: 278 , kept by blast: 276 , if remove dups: 276\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 129\n",
      "--- gene=129, id=185 removed by soft count = 60\n",
      "--- gene=129, id=268 removed by soft count = 130\n",
      "--- number of probes: 277 , kept by blast: 275 , if remove dups: 274\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 168\n",
      "--- gene=168, id=53 removed by soft count = 112\n",
      "--- gene=168, id=205 removed by soft count = 65\n",
      "--- gene=168, id=243 removed by soft count = 136\n",
      "--- gene=168, id=245 removed by soft count = 126\n",
      "--- gene=168, id=247 removed by soft count = 65\n",
      "--- number of probes: 279 , kept by blast: 274 , if remove dups: 274\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 188\n",
      "--- gene=188, id=1 removed by soft count = 268\n",
      "--- gene=188, id=19 removed by soft count = 100\n",
      "--- gene=188, id=207 removed by soft count = 78\n",
      "--- gene=188, id=267 removed by soft count = 134\n",
      "--- number of probes: 279 , kept by blast: 275 , if remove dups: 275\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 229\n",
      "--- gene=229, id=83 removed by soft count = 136\n",
      "--- gene=229, id=85 removed by soft count = 206\n",
      "--- gene=229, id=165 removed by soft count = 55\n",
      "--- number of probes: 275 , kept by blast: 272 , if remove dups: 271\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 249\n",
      "--- gene=249, id=6 removed by soft count = 85\n",
      "--- gene=249, id=95 removed by soft count = 1093\n",
      "--- gene=249, id=106 removed by soft count = 60\n",
      "--- gene=249, id=188 removed by soft count = 290\n",
      "--- gene=249, id=240 removed by soft count = 682\n",
      "--- gene=249, id=270 removed by soft count = 97\n",
      "--- number of probes: 279 , kept by blast: 273 , if remove dups: 273\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 285\n",
      "--- gene=285, id=1 removed by soft count = 129\n",
      "--- gene=285, id=2 removed by soft count = 62\n",
      "--- gene=285, id=217 removed by soft count = 85\n",
      "--- number of probes: 277 , kept by blast: 274 , if remove dups: 274\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 120\n",
      "- summarize\n",
      "-- region: 36 , number of probes: 120\n",
      "-- region: 56 , number of probes: 120\n",
      "-- region: 109 , number of probes: 120\n",
      "-- region: 129 , number of probes: 120\n",
      "-- region: 168 , number of probes: 120\n",
      "-- region: 188 , number of probes: 120\n",
      "-- region: 229 , number of probes: 120\n",
      "-- region: 249 , number of probes: 120\n",
      "-- region: 285 , number of probes: 120\n",
      "- Number of probes kept: 1080\n",
      "- Saving to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_20_sequential\\.\\blast_full_probes_seq.fasta\n"
     ]
    }
   ],
   "source": [
    "# screen\n",
    "import ImageAnalysis3.library_tools.quality_check as check\n",
    "reload(check)\n",
    "\n",
    "# the max number to keep for each region\n",
    "num_probes_per_region = 120\n",
    "\n",
    "\n",
    "# read the blast files saved from above to screen probes;\n",
    "# the screen step also keeps probes close to the center of each region;\n",
    "kept_pbs, blast_keep_dic, hard_count_list, soft_count_list = check.Screening_Probes_by_Blast(library_folder, \n",
    "                                                                                             num_probes_per_region,\n",
    "                                                                                             hard_thres=target_len,\n",
    "                                                                                             soft_count_th=50,\n",
    "                                                                                             keep_mode='front',\n",
    "                                                                                             save_filename='blast_full_probes_seq.fasta',\n",
    "                                                                                             )\n",
    "\n",
    "# 'blast_full_probes_seq.fasta' saved automatically from the function above"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 5.6 Reload to check"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 71,
   "metadata": {},
   "outputs": [],
   "source": [
    "# load final probes\n",
    "final_pb_filename = os.path.join(library_folder, 'blast_full_probes_seq.fasta')\n",
    "\n",
    "# SeqIO\n",
    "final_pb_records = []\n",
    "with open(final_pb_filename, 'r') as handle:\n",
    "    for record in SeqIO.parse(handle, \"fasta\"):\n",
    "        final_pb_records.append(record)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 72,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "gene: 36 -> 120\n",
      "gene: 56 -> 120\n",
      "gene: 109 -> 120\n",
      "gene: 129 -> 120\n",
      "gene: 168 -> 120\n",
      "gene: 188 -> 120\n",
      "gene: 229 -> 120\n",
      "gene: 249 -> 120\n",
      "gene: 285 -> 120\n"
     ]
    }
   ],
   "source": [
    "reload(check)\n",
    "\n",
    "# get final num of pbs for each region after blastn\n",
    "reg_size_dic_final, len_check_final = check._check_region_size(final_pb_records, \n",
    "                                                   min_size=min_num_probes)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 73,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "/* Put everything inside the global mpl namespace */\n",
       "/* global mpl */\n",
       "window.mpl = {};\n",
       "\n",
       "mpl.get_websocket_type = function () {\n",
       "    if (typeof WebSocket !== 'undefined') {\n",
       "        return WebSocket;\n",
       "    } else if (typeof MozWebSocket !== 'undefined') {\n",
       "        return MozWebSocket;\n",
       "    } else {\n",
       "        alert(\n",
       "            'Your browser does not have WebSocket support. ' +\n",
       "                'Please try Chrome, Safari or Firefox  6. ' +\n",
       "                'Firefox 4 and 5 are also supported but you ' +\n",
       "                'have to enable WebSockets in about:config.'\n",
       "        );\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure = function (figure_id, websocket, ondownload, parent_element) {\n",
       "    this.id = figure_id;\n",
       "\n",
       "    this.ws = websocket;\n",
       "\n",
       "    this.supports_binary = this.ws.binaryType !== undefined;\n",
       "\n",
       "    if (!this.supports_binary) {\n",
       "        var warnings = document.getElementById('mpl-warnings');\n",
       "        if (warnings) {\n",
       "            warnings.style.display = 'block';\n",
       "            warnings.textContent =\n",
       "                'This browser does not support binary websocket messages. ' +\n",
       "                'Performance may be slow.';\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.imageObj = new Image();\n",
       "\n",
       "    this.context = undefined;\n",
       "    this.message = undefined;\n",
       "    this.canvas = undefined;\n",
       "    this.rubberband_canvas = undefined;\n",
       "    this.rubberband_context = undefined;\n",
       "    this.format_dropdown = undefined;\n",
       "\n",
       "    this.image_mode = 'full';\n",
       "\n",
       "    this.root = document.createElement('div');\n",
       "    this.root.setAttribute('style', 'display: inline-block');\n",
       "    this._root_extra_style(this.root);\n",
       "\n",
       "    parent_element.appendChild(this.root);\n",
       "\n",
       "    this._init_header(this);\n",
       "    this._init_canvas(this);\n",
       "    this._init_toolbar(this);\n",
       "\n",
       "    var fig = this;\n",
       "\n",
       "    this.waiting = false;\n",
       "\n",
       "    this.ws.onopen = function () {\n",
       "        fig.send_message('supports_binary', { value: fig.supports_binary });\n",
       "        fig.send_message('send_image_mode', {});\n",
       "        if (fig.ratio !== 1) {\n",
       "            fig.send_message('set_dpi_ratio', { dpi_ratio: fig.ratio });\n",
       "        }\n",
       "        fig.send_message('refresh', {});\n",
       "    };\n",
       "\n",
       "    this.imageObj.onload = function () {\n",
       "        if (fig.image_mode === 'full') {\n",
       "            // Full images could contain transparency (where diff images\n",
       "            // almost always do), so we need to clear the canvas so that\n",
       "            // there is no ghosting.\n",
       "            fig.context.clearRect(0, 0, fig.canvas.width, fig.canvas.height);\n",
       "        }\n",
       "        fig.context.drawImage(fig.imageObj, 0, 0);\n",
       "    };\n",
       "\n",
       "    this.imageObj.onunload = function () {\n",
       "        fig.ws.close();\n",
       "    };\n",
       "\n",
       "    this.ws.onmessage = this._make_on_message_function(this);\n",
       "\n",
       "    this.ondownload = ondownload;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_header = function () {\n",
       "    var titlebar = document.createElement('div');\n",
       "    titlebar.classList =\n",
       "        'ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix';\n",
       "    var titletext = document.createElement('div');\n",
       "    titletext.classList = 'ui-dialog-title';\n",
       "    titletext.setAttribute(\n",
       "        'style',\n",
       "        'width: 100%; text-align: center; padding: 3px;'\n",
       "    );\n",
       "    titlebar.appendChild(titletext);\n",
       "    this.root.appendChild(titlebar);\n",
       "    this.header = titletext;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._init_canvas = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var canvas_div = (this.canvas_div = document.createElement('div'));\n",
       "    canvas_div.setAttribute(\n",
       "        'style',\n",
       "        'border: 1px solid #ddd;' +\n",
       "            'box-sizing: content-box;' +\n",
       "            'clear: both;' +\n",
       "            'min-height: 1px;' +\n",
       "            'min-width: 1px;' +\n",
       "            'outline: 0;' +\n",
       "            'overflow: hidden;' +\n",
       "            'position: relative;' +\n",
       "            'resize: both;'\n",
       "    );\n",
       "\n",
       "    function on_keyboard_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.key_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    canvas_div.addEventListener(\n",
       "        'keydown',\n",
       "        on_keyboard_event_closure('key_press')\n",
       "    );\n",
       "    canvas_div.addEventListener(\n",
       "        'keyup',\n",
       "        on_keyboard_event_closure('key_release')\n",
       "    );\n",
       "\n",
       "    this._canvas_extra_style(canvas_div);\n",
       "    this.root.appendChild(canvas_div);\n",
       "\n",
       "    var canvas = (this.canvas = document.createElement('canvas'));\n",
       "    canvas.classList.add('mpl-canvas');\n",
       "    canvas.setAttribute('style', 'box-sizing: content-box;');\n",
       "\n",
       "    this.context = canvas.getContext('2d');\n",
       "\n",
       "    var backingStore =\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        this.context.webkitBackingStorePixelRatio ||\n",
       "        this.context.mozBackingStorePixelRatio ||\n",
       "        this.context.msBackingStorePixelRatio ||\n",
       "        this.context.oBackingStorePixelRatio ||\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        1;\n",
       "\n",
       "    this.ratio = (window.devicePixelRatio || 1) / backingStore;\n",
       "    if (this.ratio !== 1) {\n",
       "        fig.send_message('set_dpi_ratio', { dpi_ratio: this.ratio });\n",
       "    }\n",
       "\n",
       "    var rubberband_canvas = (this.rubberband_canvas = document.createElement(\n",
       "        'canvas'\n",
       "    ));\n",
       "    rubberband_canvas.setAttribute(\n",
       "        'style',\n",
       "        'box-sizing: content-box; position: absolute; left: 0; top: 0; z-index: 1;'\n",
       "    );\n",
       "\n",
       "    var resizeObserver = new ResizeObserver(function (entries) {\n",
       "        var nentries = entries.length;\n",
       "        for (var i = 0; i < nentries; i++) {\n",
       "            var entry = entries[i];\n",
       "            var width, height;\n",
       "            if (entry.contentBoxSize) {\n",
       "                if (entry.contentBoxSize instanceof Array) {\n",
       "                    // Chrome 84 implements new version of spec.\n",
       "                    width = entry.contentBoxSize[0].inlineSize;\n",
       "                    height = entry.contentBoxSize[0].blockSize;\n",
       "                } else {\n",
       "                    // Firefox implements old version of spec.\n",
       "                    width = entry.contentBoxSize.inlineSize;\n",
       "                    height = entry.contentBoxSize.blockSize;\n",
       "                }\n",
       "            } else {\n",
       "                // Chrome <84 implements even older version of spec.\n",
       "                width = entry.contentRect.width;\n",
       "                height = entry.contentRect.height;\n",
       "            }\n",
       "\n",
       "            // Keep the size of the canvas and rubber band canvas in sync with\n",
       "            // the canvas container.\n",
       "            if (entry.devicePixelContentBoxSize) {\n",
       "                // Chrome 84 implements new version of spec.\n",
       "                canvas.setAttribute(\n",
       "                    'width',\n",
       "                    entry.devicePixelContentBoxSize[0].inlineSize\n",
       "                );\n",
       "                canvas.setAttribute(\n",
       "                    'height',\n",
       "                    entry.devicePixelContentBoxSize[0].blockSize\n",
       "                );\n",
       "            } else {\n",
       "                canvas.setAttribute('width', width * fig.ratio);\n",
       "                canvas.setAttribute('height', height * fig.ratio);\n",
       "            }\n",
       "            canvas.setAttribute(\n",
       "                'style',\n",
       "                'width: ' + width + 'px; height: ' + height + 'px;'\n",
       "            );\n",
       "\n",
       "            rubberband_canvas.setAttribute('width', width);\n",
       "            rubberband_canvas.setAttribute('height', height);\n",
       "\n",
       "            // And update the size in Python. We ignore the initial 0/0 size\n",
       "            // that occurs as the element is placed into the DOM, which should\n",
       "            // otherwise not happen due to the minimum size styling.\n",
       "            if (width != 0 && height != 0) {\n",
       "                fig.request_resize(width, height);\n",
       "            }\n",
       "        }\n",
       "    });\n",
       "    resizeObserver.observe(canvas_div);\n",
       "\n",
       "    function on_mouse_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.mouse_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousedown',\n",
       "        on_mouse_event_closure('button_press')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseup',\n",
       "        on_mouse_event_closure('button_release')\n",
       "    );\n",
       "    // Throttle sequential mouse events to 1 every 20ms.\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousemove',\n",
       "        on_mouse_event_closure('motion_notify')\n",
       "    );\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseenter',\n",
       "        on_mouse_event_closure('figure_enter')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseleave',\n",
       "        on_mouse_event_closure('figure_leave')\n",
       "    );\n",
       "\n",
       "    canvas_div.addEventListener('wheel', function (event) {\n",
       "        if (event.deltaY < 0) {\n",
       "            event.step = 1;\n",
       "        } else {\n",
       "            event.step = -1;\n",
       "        }\n",
       "        on_mouse_event_closure('scroll')(event);\n",
       "    });\n",
       "\n",
       "    canvas_div.appendChild(canvas);\n",
       "    canvas_div.appendChild(rubberband_canvas);\n",
       "\n",
       "    this.rubberband_context = rubberband_canvas.getContext('2d');\n",
       "    this.rubberband_context.strokeStyle = '#000000';\n",
       "\n",
       "    this._resize_canvas = function (width, height, forward) {\n",
       "        if (forward) {\n",
       "            canvas_div.style.width = width + 'px';\n",
       "            canvas_div.style.height = height + 'px';\n",
       "        }\n",
       "    };\n",
       "\n",
       "    // Disable right mouse context menu.\n",
       "    this.rubberband_canvas.addEventListener('contextmenu', function (_e) {\n",
       "        event.preventDefault();\n",
       "        return false;\n",
       "    });\n",
       "\n",
       "    function set_focus() {\n",
       "        canvas.focus();\n",
       "        canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    window.setTimeout(set_focus, 100);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'mpl-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'mpl-button-group';\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'mpl-button-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        var button = (fig.buttons[name] = document.createElement('button'));\n",
       "        button.classList = 'mpl-widget';\n",
       "        button.setAttribute('role', 'button');\n",
       "        button.setAttribute('aria-disabled', 'false');\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "\n",
       "        var icon_img = document.createElement('img');\n",
       "        icon_img.src = '_images/' + image + '.png';\n",
       "        icon_img.srcset = '_images/' + image + '_large.png 2x';\n",
       "        icon_img.alt = tooltip;\n",
       "        button.appendChild(icon_img);\n",
       "\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    var fmt_picker = document.createElement('select');\n",
       "    fmt_picker.classList = 'mpl-widget';\n",
       "    toolbar.appendChild(fmt_picker);\n",
       "    this.format_dropdown = fmt_picker;\n",
       "\n",
       "    for (var ind in mpl.extensions) {\n",
       "        var fmt = mpl.extensions[ind];\n",
       "        var option = document.createElement('option');\n",
       "        option.selected = fmt === mpl.default_extension;\n",
       "        option.innerHTML = fmt;\n",
       "        fmt_picker.appendChild(option);\n",
       "    }\n",
       "\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.request_resize = function (x_pixels, y_pixels) {\n",
       "    // Request matplotlib to resize the figure. Matplotlib will then trigger a resize in the client,\n",
       "    // which will in turn request a refresh of the image.\n",
       "    this.send_message('resize', { width: x_pixels, height: y_pixels });\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_message = function (type, properties) {\n",
       "    properties['type'] = type;\n",
       "    properties['figure_id'] = this.id;\n",
       "    this.ws.send(JSON.stringify(properties));\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_draw_message = function () {\n",
       "    if (!this.waiting) {\n",
       "        this.waiting = true;\n",
       "        this.ws.send(JSON.stringify({ type: 'draw', figure_id: this.id }));\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    var format_dropdown = fig.format_dropdown;\n",
       "    var format = format_dropdown.options[format_dropdown.selectedIndex].value;\n",
       "    fig.ondownload(fig, format);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_resize = function (fig, msg) {\n",
       "    var size = msg['size'];\n",
       "    if (size[0] !== fig.canvas.width || size[1] !== fig.canvas.height) {\n",
       "        fig._resize_canvas(size[0], size[1], msg['forward']);\n",
       "        fig.send_message('refresh', {});\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_rubberband = function (fig, msg) {\n",
       "    var x0 = msg['x0'] / fig.ratio;\n",
       "    var y0 = (fig.canvas.height - msg['y0']) / fig.ratio;\n",
       "    var x1 = msg['x1'] / fig.ratio;\n",
       "    var y1 = (fig.canvas.height - msg['y1']) / fig.ratio;\n",
       "    x0 = Math.floor(x0) + 0.5;\n",
       "    y0 = Math.floor(y0) + 0.5;\n",
       "    x1 = Math.floor(x1) + 0.5;\n",
       "    y1 = Math.floor(y1) + 0.5;\n",
       "    var min_x = Math.min(x0, x1);\n",
       "    var min_y = Math.min(y0, y1);\n",
       "    var width = Math.abs(x1 - x0);\n",
       "    var height = Math.abs(y1 - y0);\n",
       "\n",
       "    fig.rubberband_context.clearRect(\n",
       "        0,\n",
       "        0,\n",
       "        fig.canvas.width / fig.ratio,\n",
       "        fig.canvas.height / fig.ratio\n",
       "    );\n",
       "\n",
       "    fig.rubberband_context.strokeRect(min_x, min_y, width, height);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_figure_label = function (fig, msg) {\n",
       "    // Updates the figure title.\n",
       "    fig.header.textContent = msg['label'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_cursor = function (fig, msg) {\n",
       "    var cursor = msg['cursor'];\n",
       "    switch (cursor) {\n",
       "        case 0:\n",
       "            cursor = 'pointer';\n",
       "            break;\n",
       "        case 1:\n",
       "            cursor = 'default';\n",
       "            break;\n",
       "        case 2:\n",
       "            cursor = 'crosshair';\n",
       "            break;\n",
       "        case 3:\n",
       "            cursor = 'move';\n",
       "            break;\n",
       "    }\n",
       "    fig.rubberband_canvas.style.cursor = cursor;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_message = function (fig, msg) {\n",
       "    fig.message.textContent = msg['message'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_draw = function (fig, _msg) {\n",
       "    // Request the server to send over a new figure.\n",
       "    fig.send_draw_message();\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_image_mode = function (fig, msg) {\n",
       "    fig.image_mode = msg['mode'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_history_buttons = function (fig, msg) {\n",
       "    for (var key in msg) {\n",
       "        if (!(key in fig.buttons)) {\n",
       "            continue;\n",
       "        }\n",
       "        fig.buttons[key].disabled = !msg[key];\n",
       "        fig.buttons[key].setAttribute('aria-disabled', !msg[key]);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_navigate_mode = function (fig, msg) {\n",
       "    if (msg['mode'] === 'PAN') {\n",
       "        fig.buttons['Pan'].classList.add('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    } else if (msg['mode'] === 'ZOOM') {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.add('active');\n",
       "    } else {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Called whenever the canvas gets updated.\n",
       "    this.send_message('ack', {});\n",
       "};\n",
       "\n",
       "// A function to construct a web socket function for onmessage handling.\n",
       "// Called in the figure constructor.\n",
       "mpl.figure.prototype._make_on_message_function = function (fig) {\n",
       "    return function socket_on_message(evt) {\n",
       "        if (evt.data instanceof Blob) {\n",
       "            /* FIXME: We get \"Resource interpreted as Image but\n",
       "             * transferred with MIME type text/plain:\" errors on\n",
       "             * Chrome.  But how to set the MIME type?  It doesn't seem\n",
       "             * to be part of the websocket stream */\n",
       "            evt.data.type = 'image/png';\n",
       "\n",
       "            /* Free the memory for the previous frames */\n",
       "            if (fig.imageObj.src) {\n",
       "                (window.URL || window.webkitURL).revokeObjectURL(\n",
       "                    fig.imageObj.src\n",
       "                );\n",
       "            }\n",
       "\n",
       "            fig.imageObj.src = (window.URL || window.webkitURL).createObjectURL(\n",
       "                evt.data\n",
       "            );\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        } else if (\n",
       "            typeof evt.data === 'string' &&\n",
       "            evt.data.slice(0, 21) === 'data:image/png;base64'\n",
       "        ) {\n",
       "            fig.imageObj.src = evt.data;\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        var msg = JSON.parse(evt.data);\n",
       "        var msg_type = msg['type'];\n",
       "\n",
       "        // Call the  \"handle_{type}\" callback, which takes\n",
       "        // the figure and JSON message as its only arguments.\n",
       "        try {\n",
       "            var callback = fig['handle_' + msg_type];\n",
       "        } catch (e) {\n",
       "            console.log(\n",
       "                \"No handler for the '\" + msg_type + \"' message type: \",\n",
       "                msg\n",
       "            );\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        if (callback) {\n",
       "            try {\n",
       "                // console.log(\"Handling '\" + msg_type + \"' message: \", msg);\n",
       "                callback(fig, msg);\n",
       "            } catch (e) {\n",
       "                console.log(\n",
       "                    \"Exception inside the 'handler_\" + msg_type + \"' callback:\",\n",
       "                    e,\n",
       "                    e.stack,\n",
       "                    msg\n",
       "                );\n",
       "            }\n",
       "        }\n",
       "    };\n",
       "};\n",
       "\n",
       "// from http://stackoverflow.com/questions/1114465/getting-mouse-location-in-canvas\n",
       "mpl.findpos = function (e) {\n",
       "    //this section is from http://www.quirksmode.org/js/events_properties.html\n",
       "    var targ;\n",
       "    if (!e) {\n",
       "        e = window.event;\n",
       "    }\n",
       "    if (e.target) {\n",
       "        targ = e.target;\n",
       "    } else if (e.srcElement) {\n",
       "        targ = e.srcElement;\n",
       "    }\n",
       "    if (targ.nodeType === 3) {\n",
       "        // defeat Safari bug\n",
       "        targ = targ.parentNode;\n",
       "    }\n",
       "\n",
       "    // pageX,Y are the mouse positions relative to the document\n",
       "    var boundingRect = targ.getBoundingClientRect();\n",
       "    var x = e.pageX - (boundingRect.left + document.body.scrollLeft);\n",
       "    var y = e.pageY - (boundingRect.top + document.body.scrollTop);\n",
       "\n",
       "    return { x: x, y: y };\n",
       "};\n",
       "\n",
       "/*\n",
       " * return a copy of an object with only non-object keys\n",
       " * we need this to avoid circular references\n",
       " * http://stackoverflow.com/a/24161582/3208463\n",
       " */\n",
       "function simpleKeys(original) {\n",
       "    return Object.keys(original).reduce(function (obj, key) {\n",
       "        if (typeof original[key] !== 'object') {\n",
       "            obj[key] = original[key];\n",
       "        }\n",
       "        return obj;\n",
       "    }, {});\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.mouse_event = function (event, name) {\n",
       "    var canvas_pos = mpl.findpos(event);\n",
       "\n",
       "    if (name === 'button_press') {\n",
       "        this.canvas.focus();\n",
       "        this.canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    var x = canvas_pos.x * this.ratio;\n",
       "    var y = canvas_pos.y * this.ratio;\n",
       "\n",
       "    this.send_message(name, {\n",
       "        x: x,\n",
       "        y: y,\n",
       "        button: event.button,\n",
       "        step: event.step,\n",
       "        guiEvent: simpleKeys(event),\n",
       "    });\n",
       "\n",
       "    /* This prevents the web browser from automatically changing to\n",
       "     * the text insertion cursor when the button is pressed.  We want\n",
       "     * to control all of the cursor setting manually through the\n",
       "     * 'cursor' event from matplotlib */\n",
       "    event.preventDefault();\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (_event, _name) {\n",
       "    // Handle any extra behaviour associated with a key event\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.key_event = function (event, name) {\n",
       "    // Prevent repeat events\n",
       "    if (name === 'key_press') {\n",
       "        if (event.which === this._key) {\n",
       "            return;\n",
       "        } else {\n",
       "            this._key = event.which;\n",
       "        }\n",
       "    }\n",
       "    if (name === 'key_release') {\n",
       "        this._key = null;\n",
       "    }\n",
       "\n",
       "    var value = '';\n",
       "    if (event.ctrlKey && event.which !== 17) {\n",
       "        value += 'ctrl+';\n",
       "    }\n",
       "    if (event.altKey && event.which !== 18) {\n",
       "        value += 'alt+';\n",
       "    }\n",
       "    if (event.shiftKey && event.which !== 16) {\n",
       "        value += 'shift+';\n",
       "    }\n",
       "\n",
       "    value += 'k';\n",
       "    value += event.which.toString();\n",
       "\n",
       "    this._key_event_extra(event, name);\n",
       "\n",
       "    this.send_message(name, { key: value, guiEvent: simpleKeys(event) });\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onclick = function (name) {\n",
       "    if (name === 'download') {\n",
       "        this.handle_save(this, null);\n",
       "    } else {\n",
       "        this.send_message('toolbar_button', { name: name });\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onmouseover = function (tooltip) {\n",
       "    this.message.textContent = tooltip;\n",
       "};\n",
       "mpl.toolbar_items = [[\"Home\", \"Reset original view\", \"fa fa-home icon-home\", \"home\"], [\"Back\", \"Back to previous view\", \"fa fa-arrow-left icon-arrow-left\", \"back\"], [\"Forward\", \"Forward to next view\", \"fa fa-arrow-right icon-arrow-right\", \"forward\"], [\"\", \"\", \"\", \"\"], [\"Pan\", \"Left button pans, Right button zooms\\nx/y fixes axis, CTRL fixes aspect\", \"fa fa-arrows icon-move\", \"pan\"], [\"Zoom\", \"Zoom to rectangle\\nx/y fixes axis, CTRL fixes aspect\", \"fa fa-square-o icon-check-empty\", \"zoom\"], [\"\", \"\", \"\", \"\"], [\"Download\", \"Download plot\", \"fa fa-floppy-o icon-save\", \"download\"]];\n",
       "\n",
       "mpl.extensions = [\"eps\", \"jpeg\", \"pdf\", \"png\", \"ps\", \"raw\", \"svg\", \"tif\"];\n",
       "\n",
       "mpl.default_extension = \"png\";/* global mpl */\n",
       "\n",
       "var comm_websocket_adapter = function (comm) {\n",
       "    // Create a \"websocket\"-like object which calls the given IPython comm\n",
       "    // object with the appropriate methods. Currently this is a non binary\n",
       "    // socket, so there is still some room for performance tuning.\n",
       "    var ws = {};\n",
       "\n",
       "    ws.close = function () {\n",
       "        comm.close();\n",
       "    };\n",
       "    ws.send = function (m) {\n",
       "        //console.log('sending', m);\n",
       "        comm.send(m);\n",
       "    };\n",
       "    // Register the callback with on_msg.\n",
       "    comm.on_msg(function (msg) {\n",
       "        //console.log('receiving', msg['content']['data'], msg);\n",
       "        // Pass the mpl event to the overridden (by mpl) onmessage function.\n",
       "        ws.onmessage(msg['content']['data']);\n",
       "    });\n",
       "    return ws;\n",
       "};\n",
       "\n",
       "mpl.mpl_figure_comm = function (comm, msg) {\n",
       "    // This is the function which gets called when the mpl process\n",
       "    // starts-up an IPython Comm through the \"matplotlib\" channel.\n",
       "\n",
       "    var id = msg.content.data.id;\n",
       "    // Get hold of the div created by the display call when the Comm\n",
       "    // socket was opened in Python.\n",
       "    var element = document.getElementById(id);\n",
       "    var ws_proxy = comm_websocket_adapter(comm);\n",
       "\n",
       "    function ondownload(figure, _format) {\n",
       "        window.open(figure.canvas.toDataURL());\n",
       "    }\n",
       "\n",
       "    var fig = new mpl.figure(id, ws_proxy, ondownload, element);\n",
       "\n",
       "    // Call onopen now - mpl needs it, as it is assuming we've passed it a real\n",
       "    // web socket which is closed, not our websocket->open comm proxy.\n",
       "    ws_proxy.onopen();\n",
       "\n",
       "    fig.parent_element = element;\n",
       "    fig.cell_info = mpl.find_output_cell(\"<div id='\" + id + \"'></div>\");\n",
       "    if (!fig.cell_info) {\n",
       "        console.error('Failed to find cell for figure', id, fig);\n",
       "        return;\n",
       "    }\n",
       "    fig.cell_info[0].output_area.element.one(\n",
       "        'cleared',\n",
       "        { fig: fig },\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_close = function (fig, msg) {\n",
       "    var width = fig.canvas.width / fig.ratio;\n",
       "    fig.cell_info[0].output_area.element.off(\n",
       "        'cleared',\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "\n",
       "    // Update the output cell to use the data from the current canvas.\n",
       "    fig.push_to_output();\n",
       "    var dataURL = fig.canvas.toDataURL();\n",
       "    // Re-enable the keyboard manager in IPython - without this line, in FF,\n",
       "    // the notebook keyboard shortcuts fail.\n",
       "    IPython.keyboard_manager.enable();\n",
       "    fig.parent_element.innerHTML =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "    fig.close_ws(fig, msg);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.close_ws = function (fig, msg) {\n",
       "    fig.send_message('closing', msg);\n",
       "    // fig.ws.close()\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.push_to_output = function (_remove_interactive) {\n",
       "    // Turn the data on the canvas into data in the output cell.\n",
       "    var width = this.canvas.width / this.ratio;\n",
       "    var dataURL = this.canvas.toDataURL();\n",
       "    this.cell_info[1]['text/html'] =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Tell IPython that the notebook contents must change.\n",
       "    IPython.notebook.set_dirty(true);\n",
       "    this.send_message('ack', {});\n",
       "    var fig = this;\n",
       "    // Wait a second, then push the new image to the DOM so\n",
       "    // that it is saved nicely (might be nice to debounce this).\n",
       "    setTimeout(function () {\n",
       "        fig.push_to_output();\n",
       "    }, 1000);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'btn-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'btn-group';\n",
       "    var button;\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'btn-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        button = fig.buttons[name] = document.createElement('button');\n",
       "        button.classList = 'btn btn-default';\n",
       "        button.href = '#';\n",
       "        button.title = name;\n",
       "        button.innerHTML = '<i class=\"fa ' + image + ' fa-lg\"></i>';\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    // Add the status bar.\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message pull-right';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "\n",
       "    // Add the close button to the window.\n",
       "    var buttongrp = document.createElement('div');\n",
       "    buttongrp.classList = 'btn-group inline pull-right';\n",
       "    button = document.createElement('button');\n",
       "    button.classList = 'btn btn-mini btn-primary';\n",
       "    button.href = '#';\n",
       "    button.title = 'Stop Interaction';\n",
       "    button.innerHTML = '<i class=\"fa fa-power-off icon-remove icon-large\"></i>';\n",
       "    button.addEventListener('click', function (_evt) {\n",
       "        fig.handle_close(fig, {});\n",
       "    });\n",
       "    button.addEventListener(\n",
       "        'mouseover',\n",
       "        on_mouseover_closure('Stop Interaction')\n",
       "    );\n",
       "    buttongrp.appendChild(button);\n",
       "    var titlebar = this.root.querySelector('.ui-dialog-titlebar');\n",
       "    titlebar.insertBefore(buttongrp, titlebar.firstChild);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._remove_fig_handler = function (event) {\n",
       "    var fig = event.data.fig;\n",
       "    fig.close_ws(fig, {});\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (el) {\n",
       "    el.style.boxSizing = 'content-box'; // override notebook setting of border-box.\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (el) {\n",
       "    // this is important to make the div 'focusable\n",
       "    el.setAttribute('tabindex', 0);\n",
       "    // reach out to IPython and tell the keyboard manager to turn it's self\n",
       "    // off when our div gets focus\n",
       "\n",
       "    // location in version 3\n",
       "    if (IPython.notebook.keyboard_manager) {\n",
       "        IPython.notebook.keyboard_manager.register_events(el);\n",
       "    } else {\n",
       "        // location in version 2\n",
       "        IPython.keyboard_manager.register_events(el);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (event, _name) {\n",
       "    var manager = IPython.notebook.keyboard_manager;\n",
       "    if (!manager) {\n",
       "        manager = IPython.keyboard_manager;\n",
       "    }\n",
       "\n",
       "    // Check for shift+enter\n",
       "    if (event.shiftKey && event.which === 13) {\n",
       "        this.canvas_div.blur();\n",
       "        // select the cell after this one\n",
       "        var index = IPython.notebook.find_cell_index(this.cell_info[0]);\n",
       "        IPython.notebook.select(index + 1);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    fig.ondownload(fig, null);\n",
       "};\n",
       "\n",
       "mpl.find_output_cell = function (html_output) {\n",
       "    // Return the cell and output element which can be found *uniquely* in the notebook.\n",
       "    // Note - this is a bit hacky, but it is done because the \"notebook_saving.Notebook\"\n",
       "    // IPython event is triggered only after the cells have been serialised, which for\n",
       "    // our purposes (turning an active figure into a static one), is too late.\n",
       "    var cells = IPython.notebook.get_cells();\n",
       "    var ncells = cells.length;\n",
       "    for (var i = 0; i < ncells; i++) {\n",
       "        var cell = cells[i];\n",
       "        if (cell.cell_type === 'code') {\n",
       "            for (var j = 0; j < cell.output_area.outputs.length; j++) {\n",
       "                var data = cell.output_area.outputs[j];\n",
       "                if (data.data) {\n",
       "                    // IPython >= 3 moved mimebundle to data attribute of output\n",
       "                    data = data.data;\n",
       "                }\n",
       "                if (data['text/html'] === html_output) {\n",
       "                    return [cell, data, j];\n",
       "                }\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "};\n",
       "\n",
       "// Register the function which deals with the matplotlib target/channel.\n",
       "// The kernel may be null if the page has been refreshed.\n",
       "if (IPython.notebook.kernel !== null) {\n",
       "    IPython.notebook.kernel.comm_manager.register_target(\n",
       "        'matplotlib',\n",
       "        mpl.mpl_figure_comm\n",
       "    );\n",
       "}\n"
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<img src=\"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAsAAAAIQCAYAAACPEdjAAAAAAXNSR0IArs4c6QAAIABJREFUeF7t3Q3MnfP9P/Bvy6gsmck8BEWpim3aIlF0DGvmxzxsntqReFgsC7KQbIinTWseimBbqmYEoxljHlfLZMRCiSFYuhpidOtoRoY1oZ7vf66T/32no+o6p73u9zk9rzuR/PZzzvU59+v6fOvd9316OmJgYGCg+CJAgAABAgQIECDQJwIjBOA+udO+TQIECBAgQIAAgZaAAGwRCBAgQIAAAQIE+kpAAO6r2+2bJUCAAAECBAgQEIDtAAECBAgQIECAQF8JCMB9dbt9swQIECBAgAABAgKwHSBAgAABAgQIEOgrAQG4r263b5YAAQIECBAgQEAAtgMECBAgQIAAAQJ9JSAA99Xt9s0SIECAAAECBAgIwHaAAAECBAgQIECgrwQE4L663b5ZAgQIECBAgAABAdgOECBAgAABAgQI9JWAANxXt9s3S4AAAQIECBAgIADbAQIECBAgQIAAgb4SEID76nb7ZgkQIECAAAECBARgO0CAAAECBAgQINBXAgJwX91u3ywBAgQIECBAgIAAbAcIECBAgAABAgT6SkAA7qvb7ZslQIAAAQIECBAQgO0AAQIECBAgQIBAXwkIwH11u32zBAgQIECAAAECArAdIECAAAECBAgQ6CsBAbivbrdvlgABAgQIECBAQAC2AwQIECBAgAABAn0lIAD31e32zRIgQIAAAQIECAjAdoAAAQIECBAgQKCvBATgvrrdvlkCBAgQIECAAAEB2A4QIECAAAECBAj0lYAA3Fe32zdLgAABAgQIECAgANsBAgQIECBAgACBvhIQgPvqdvtmCRAgQIAAAQIEBGA7QIAAAQIECBAg0FcCAnBf3W7fLAECBAgQIECAgABsBwgQIECAAAECBPpKQADuq9vtmyVAgAABAgQIEBCA7QABAgQIECBAgEBfCQjAfXW7fbMECBAgQIAAAQICsB0gQIAAAQIECBDoKwEBuK9ut2+WAAECBAgQIEBAALYDBAgQIECAAAECfSUgAPfV7fbNEiBAgAABAgQICMB2gAABAgQIECBAoK8EBOC+ut2+WQIECBAgQIAAAQHYDhAgQIAAAQIECPSVgADcV7fbN0uAAAECBAgQICAA2wECBAgQIECAAIG+EhCA++p2+2YJECBAgAABAgQEYDtAgAABAgQIECDQVwICcF/dbt8sAQIECBAgQICAAGwHCBAgQIAAAQIE+kpAAO6r2+2bJUCAAAECBAgQEIDtAAECBAgQIECAQF8JCMB9dbt9swQIECBAgAABAgKwHSBAgAABAgQIEOgrAQG4r263b5YAAQIECBAgQEAAtgMECBAgQIAAAQJ9JSAA99Xt9s0SIECAAAECBAgIwHaAAAECBAgQIECgrwQE4L663b5ZAgQIECBAgAABAdgOECBAgAABAgQI9JWAANxXt9s3S4AAAQIECBAgIAA3uANvv/12mT9/ftlggw3Kmmuu2eAklyZAgAABAgQI/K/A+++/X1599dUyfvz4MmrUKDzLCAjADa7DY489ViZNmtTgBJcmQIAAAQIECKxY4NFHHy077bQTJgF4eHZg4cKFZcsttyzV4m288cbDM9QUAgQIECBAgEApZfHixa0i7sUXXyxjxoxhIgAPzw7861//KptttllZtGhRGT169PAMNYUAAQIECBAgUEqRQz55DbwFosEjYvEaxHVpAgQIECBAYIUCcogAHDkiFi/CbigBAgQIECCgAV7hDmiAGzwiAnCDuC5NgAABAgQIaIA73AEBuEO4Ok8TgOsoeQwBAgQIECDQhIAc8smqAnATG/f/r2nxGsR1aQIECBAgQEAD3OEOCMAdwtV5mgBcR8ljCBAgQIAAgSYE5BANcBN79anXtHifSuQBBAgQIECAQEMCcogA3NBqrfiyFi/CbigBAgQIECDgUyBWuAPeAtHgERGAG8R1aQIECBAgQGCFAnKIBjhyRCxehN1QAgQIECBAQAOsAU6dAgE4JW8uAQIECBAgIIdogCOnwOJF2A0lQIAAAQIENMAa4NQpEIBT8uYSIECAAAECcogGOHIKLF6E3VACBAgQIEBAA6wBTp0CATglby4BAgQIECAgh2iAI6fA4kXYDSVAgAABAgQ0wBrg1CkQgFPy5hIgQIAAAQJyiAY4cgosXoTd0GUExpx2d22PhTP3q/1YDyRAgACB7heQQwTgyJZavAi7oQKwHSBAgAABb4FY4Q74q5AbPCICcIO4Ll1LQANci8mDCBAgsFoKyCEa4MhiW7wIu6EaYDtAgAABAhpgDXDqFAjAKXlzBwU0wHaBAAEC/Ssgh2iAI9tv8SLshmqA7QABAgQIaIA1wKlTIACn5M3VANsBAgQIEJBDNMCRU2DxIuyGaoDtAAECBAhogDXAqVMgAKfkzdUA2wECBAgQkEM0wJFTYPEi7IZqgO0AAQIECGiANcCpUyAAp+TN1QDbAQIECBCQQzTAkVNg8SLshmqA7QABAgQIaIA1wKlTIACn5M3VANsBAgQIEJBDNMCRU2DxIuyGaoDtAAECBAhogDXAqVMgAKfkzdUA2wECBAgQkEM0wJFTYPEi7IZqgO0AAQIECGiANcCpUyAAp+TN1QDbAQIECBCQQzTAkVNg8SLshmqA7QABAgQIaIA1wKlTIACn5M3VANsBAgQIEJBDNMCRU2DxIuyGaoDtAAECBAhogDXAqVMgAKfkzdUA2wECBAgQkEM0wJFTYPEi7IZqgO0AAQIECGiANcCpUyAAp+TN1QDbAQIECBCQQzTAkVNg8SLshmqA7QABAgQIaIA1wKlTIACn5M3VANsBAgQIEJBDNMCRU2DxIuyGaoDtAAECBAhogDXAqVMgAKfkzdUA2wECBAgQkEM0wJFTYPEi7IZqgO0AAQIECGiANcCpUyAAp+TN1QDbAQIECBCQQzTAkVNg8SLshmqA7QABAgQIaIA1wKlTIACn5M3VANsBAgQIEJBDNMCRU2DxIuyGaoDtAAECBAhogDXAqVMgAKfkzdUA2wECBAgQkEM0wJFTYPEi7IZqgO0AAQIECGiANcCpUyAAp+TN1QDbAQIECBCQQzTAkVNg8SLshmqA7QABAgQIaIA1wKlTIACn5M3VANsBAgQIEJBDNMBtn4L33nuvTJ8+vdxwww3l9ddfL9tvv3258MILy+TJk2tfy+LVpvLAhgTGnHZ37SsvnLlf7cd6IAECBAh0v4AcIgC3vaVnn312+eUvf1l+9atfla222qoVfm+++ebyt7/9rWyyySa1rmfxajF5UIMCAnCDuC5NgACBLheQQwTgtle0anynTJlSLrnkktZzlyxZUtZdd93y29/+thxyyCG1rmfxajF5UIMCAnCDuC5NgACBLheQQwTgtle0Cr/V4txzzz1ls802K5dddlk566yzyoIFC8rYsWNrXc/i1WLyoAYFBOAGcV2aAAECXS4ghwjAba/oX//61zJt2rTy9NNPlzXWWKOMHDmy1f4eeOCBta9l8WpTeWBDAgJwQ7AuS4AAgR4QkEME4LbX9JZbbik/+9nPyimnnFI23XTTctVVV5Xf/OY35YEHHigTJkxY7vWqt0lU/wx+LV68uEyaNKksWrSojB49uu3X4AkEVlZAAF5ZQc8nQIBA7woIwAJwW9v7j3/8o4wbN67cd999Zffddx96bvV/r7/++uX2229f7vWqT42YMWPGx/6dANwWvwevQgEBeBViuhQBAgR6TEAAFoDbWtmq/Z06dWpZunRpGTVq1NBzqzb497//fet9wMv70gC3xezBwyAgAA8DshEECBDoUgEBWABuazUfeeSRsuuuu5Y///nPrbcwDH7ts88+rUB8xx131LqexavF5EENCgjADeK6NAECBLpcQA4RgNta0Q8//LDsueee5dVXXy2zZ89uvX/3+uuvLxdccEGZN29e2WWXXWpdz+LVYvKgBgUE4AZxXZoAAQJdLiCHCMBtr2j1t79VH3s2d+7c1t8EN378+HL++eeXPfbYo/a1LF5tKg9sSEAAbgjWZQkQINADAnKIABxZU4sXYTd0GQEB2DoQIECgfwXkEAE4sv0WL8JuqABsBwgQIECglNZf6FX9ZV4+jerj6zBiYGBgwJY0I2DxmnF11foCGuD6Vh5JgACB1U1ADtEAR3ba4kXYDdUA2wECBAgQ0ACvcAc0wA0eEQG4QVyXriWgAa7F5EEECBBYLQXkEA1wZLEtXoTdUA2wHSBAgAABDbAGOHUKBOCUvLmDAhpgu0CAAIH+FZBDNMCR7bd4EXZDNcB2gAABAgQ0wBrg1CkQgFPy5mqA7QABAgQIyCEa4MgpsHgRdkM1wHaAAAECBDTAGuDUKRCAU/LmaoDtAAECBAjIIRrgyCmweBF2QzXAdoAAAQIENMAa4NQpEIBT8uZqgO0AAQIECMghGuDIKbB4EXZDNcB2gAABAgQ0wBrg1CkQgFPy5mqA7QABAgQIyCEa4MgpsHgRdkM1wHaAAAECBDTAGuDUKRCAU/LmaoDtAAECBAjIIRrgyCmweBF2QzXAdoAAAQIENMAa4NQpEIBT8uZqgO0AAQIECMghGuDIKbB4EXZDNcB2gAABAgQ0wBrg1CkQgFPy5mqA7QABAgQIyCEa4MgpsHgRdkM1wHaAAAECBDTAGuDUKRCAU/LmaoDtAAECBAjIIRrgyCmweBF2QzXAdoAAAQIENMAa4NQpEIBT8uZqgO0AAQIECMghGuDIKbB4EXZDNcB2gAABAgQ0wBrg1CkQgFPy5mqA7QABAgQIyCEa4MgpsHgRdkM1wHaAAAECBDTAGuDUKRCAU/LmaoDtAAECBAjIIRrgyCmweBF2QzXAdoAAAQIENMAa4NQpEIBT8uZqgO0AAQIECMghGuDIKbB4EXZDNcB2gAABAgQ0wBrg1CkQgFPy5mqA7QABAgQIyCEa4MgpsHgRdkM1wHaAAAECBDTAGuDUKRCAU/LmaoDtAAECBAjIIRrgyCmweBF2QzXAdoAAAQIENMAa4NQpEIBT8uZqgO0AAQIECMghGuDIKbB4EXZDNcB2gAABAgQ0wBrg1CkQgFPy5mqA7QABAgQIyCEa4MgpsHgRdkM1wHaAAAECBDTAGuDUKRCAU/LmaoDtAAECBAjIIRrgyCmweBF2QzXAdoAAAQIENMAa4NQpEIBT8uZqgO0AAQIECMghGuDIKbB4EXZDNcB2gAABAgQ0wBrg1CkQgFPy5mqA7QABAgQIyCEa4MgpsHgRdkM1wHaAAAECBDTAGuDUKRCAU/LmaoDtAAECBAjIIRrgyCmweBF2QzXAdoAAAQIENMAa4NQpEIBT8uZqgO0AAQIECMghGuDIKbB4EXZDNcB2gAABAgQ0wBrg1CkQgFPy5mqA7QABAgQIyCEa4MgpsHgRdkM1wHaAAAECBDTAGuDUKRCAU/LmaoDtAAECBAjIIRrgyCmweBF2QzXAdoAAAQIENMAa4NQpEIBT8uZqgO0AAQIECMghGuDIKbB4EXZDNcB2gAABAgQ0wBrg1CkQgFPy5mqA7QABAgQIyCEa4MgpsHgRdkM1wHaAAAECBDTAGuDUKRCAU/LmaoDtAAECBAjIIRrgyCmweBF2QzXAdoAAAQIENMAa4NQpEIBT8uZqgO0AAQIECMghGuDIKbB4EXZDNcB2gAABAgQ0wBrg1CkQgFPy5mqA7QABAgQIyCEa4MgpsHgRdkM1wHaAAAECBDTAGuDUKRCAU/LmaoDtAAECBAjIIRrgyCmweBF2QzXAdoAAAQIENMAa4NQpEIBT8uZqgO0AAQIECMghGuDIKbB4EXZDNcB2gAABAgQ0wBrg1CkQgFPy5mqA7QABAgQIyCEa4MgpsHgRdkM1wHaAAAECBDTAGuDUKRCAU/LmaoDtAAECBAjIIRrgyCmweBF2QzXAdoAAAQIENMAa4NQpEIBT8uZqgO0AAQIECMghGuDIKbB4EXZDNcB2gAABAgQ0wBrg1CkQgFPy5mqA7QABAgQIyCEa4I5OwfXXX19mzpxZXnjhhTJ27Ngyffr0cthhh9W+lsWrTeWBDQmMOe3u2ldeOHO/2o/1QAIECBDofgE5RABue0vnzJlTjjnmmHLJJZeU/fbbr9x4442tADxv3ryy66671rqexavF5EENCgjADeK6NAECBLpcQA4RgNta0YGBgbLVVluVQw89tFx88cVDz/2///u/sueee5bTTz+91vUsXi0mD2pQQABuENelCRAg0OUCcogA3NaKPvPMM+WLX/xieeKJJ8oOO+zQ1nOXfbDF65jOE1eRgAC8iiBdhgABAj0oIIcIwG2t7V133VW++c1vlgcffLD85Cc/KU8++WTZcssty1lnnVUOOOCA2teyeLWpPLAhAQG4IViXJUCAQA8IyCECcFtrWr3/98gjj2yF3rPPPrtMnDix3HrrreW8884rf/zjH8uUKVOWe70lS5aU6p/Br8WLF5dJkyaVRYsWldGjR7f1GjyYwKoQEIBXhaJrECBAoDcFBGABuK3Nvfnmm8u0adPK7Nmzy/HHHz/03OoPw1Vfd9+9/D9ZX/0huRkzZnxslgDcFr8Hr0IBAXgVYroUAQIEekxAABaA21rZhx9+uHzlK19pvQVit912G3ruqaeeWubOnVuefvppDXBboh6cEhCAU/LmEiBAIC8gAAvAbW3h0qVLy0YbbVR+/OMfl5NPPnnoufvuu2/5zGc+U6r3CNf5snh1lDymSQEBuEld1yZAgEB3C8ghAnDbG3ruueeWiy66qFx55ZWt9/HedNNNrUB83333tT4Krc6Xxauj5DFNCgjATeq6NgECBLpbQA4RgDva0EsvvbTMmjWrvPTSS62PRave31t9OkTdL4tXV8rjmhIQgJuSdV0CBAh0v4AcIgBHttTiRdgNXUZAALYOBAgQ6F8BOUQAjmy/xYuwGyoA2wECBAgQKKXIIQJw5CBYvAi7oQKwHSBAgAABAXiFOzBiYGBgwJY0IyAAN+PqqvUFvAWivpVHEiBAYHUTkEM0wJGdtngRdkM1wHaAAAECBDTAGuDUKRCAU/LmDgpogO0CAQIE+ldADtEAR7bf4kXYDdUA2wECBAgQ0ABrgFOnQABOyZurAbYDBAgQICCHaIAjp8DiRdgN1QDbAQIECBDQAGuAU6dAAE7Jm6sBtgMECBAgIIdogCOnwOJF2A3VANsBAgQIENAAa4BTp0AATsmbqwG2AwQIECAgh2iAI6fA4kXYDdUA2wECBAgQ0ABrgFOnQABOyZurAbYDBAgQICCHaIAjp8DiRdgN1QDbAQIECBDQAGuAU6dAAE7Jm6sBtgMECBAgIIdogCOnwOJF2A3VANsBAgQIENAAa4BTp0AATsmbqwG2AwQIECAgh2iAI6fA4kXYDdUA2wECBAgQ0ABrgFOnQABOyZurAbYDBAgQICCHaIAjp8DiRdgN1QDbAQIECBDQAGuAU6dAAE7Jm6sBtgMECBAgIIdogCOnwOJF2A3VANsBAgQIENAAa4BTp0AATsmbqwG2AwQIECAgh2iAI6fA4kXYDdUA2wECBAgQ0ABrgFOnQABOyZurAbYDBAgQICCHaIAjp8DiRdgN1QDbAQIECBDQAGuAU6dAAE7Jm6sBtgMECBAgIIdogCOnwOJF2A3VANsBAgQIENAAa4BTp0AATsmbqwG2AwQIECAgh2iAI6fA4kXYDdUA2wECBAgQ0ABrgFOnQABOyZurAbYDBAgQICCHaIAjp8DiRdgN1QDbAQIECBDQAGuAU6dAAE7Jm6sBtgMECBAgIIdogCOnwOJF2A3VANsBAgQIENAAa4BTp0AATsmbqwG2AwQIECAgh2iAI6fA4kXYDdUA2wECBAgQ0ABrgFOnQABOyZurAbYDBAgQICCHaIAjp8DiRdgN1QDbAQIECBDQAGuAU6dAAE7Jm6sBtgMECBAgIIdogCOnwOJF2A3VANsBAgQIENAAa4BTp0AATsmbqwG2AwQIECAgh2iAI6fA4kXYDdUA2wECBAgQ0ABrgFOnQABOyZurAbYDBAgQICCHaIAjp8DiRdgN1QDbAQIECBDQAGuAU6dAAE7Jm6sBtgMECBAgIIdogCOnwOJF2A3VANsBAgQIENAAa4BTp0AATsmbqwG2AwQIECAgh2iAI6fA4kXYDdUA2wECBAgQ0ABrgFOnQABOyZurAbYDBAgQICCHaIAjp8DiRdgN1QDbAQIECBDQAGuAU6dAAE7Jm6sBtgMECBAgIIdogCOnwOJF2A3VANsBAgQIENAAa4BTp0AATsmbqwG2AwQIECAgh2iAI6fA4kXYDdUA2wECBAgQ0ABrgFOnQABOyZurAbYDBAgQICCHaIAjp8DiRdgN1QDbAQIECBDQAGuAU6dAAE7Jm6sBtgMECBAgIIdogCOnwOJF2A3VANsBAgQIENAAa4BTp0AATsmbqwG2AwQIECAgh2iAI6fA4kXYDdUA2wECBAgQ0ABrgFOnQABOyZurAbYDBAgQICCHaIAjp8DiRdgN1QDbAQIECBDQAGuAU6dAAE7Jm6sBtgMECBAgIIdogCOnwOJF2A3VANsBAgQIENAAa4BTp0AATsmbqwG2AwQIECAgh2iAI6fA4kXYDdUA2wECBAgQ0ABrgFOnQABOyZurAbYDBAgQICCHaIAjp8DiRdgN1QDbAQIECBDQAGuAU6dAAE7Jm6sBtgMECBAgIIdogCOnwOJF2A3VANsBAgQIENAAa4BTp0AATsmbqwG2AwQIECAgh2iAI6fA4kXYDdUA2wECBAgQ0ABrgFOnQABOyZurAbYDBAgQICCHaIAjp8DiRdgN1QDbAQIECBDQAGuAV/YUPPfcc2XHHXcss2bNKsccc0ztywnAtak8sCGBMafdXfvKC2fuV/uxHkiAAAEC3S8gh2iAO97S9957r0yePLk8/vjj5dprrxWAO5b0xISAAJxQN5MAAQLdISAAC8Adb+IZZ5xRHnnkkXL//fcLwB0remJKQABOyZtLgACBvIAALAB3tIUPPPBAOeigg8pTTz1VNt98cwG4I0VPSgoIwEl9swkQIJAVEIAF4LY38I033igTJ04sF1xwQTniiCPKiBEjBOC2FT0hLSAAp++A+QQIEMgJCMACcNvbd/jhh5eBgYFy0003tZ5bJwAvWbKkVP8Mfi1evLhMmjSpLFq0qIwePbrt1+AJBFZWQABeWUHPJ0CAQO8KCMACcFvbe8MNN5TTTz+9zJ8/v6y33nq1A/D06dPLjBkzPjZLAG6L34NXoYAAvAoxXYoAAQI9JiAAC8Btrexee+1V5s2bV9Zee+2h57355put/z127NiyYMGC5V5PA9wWswcPg4AAPAzIRhAgQKBLBQRgAbit1XzppZfK0qVL/+c548aNKxdeeGGZNm1a2WKLLWpdz+LVYvKgBgUE4AZxXZoAAQJdLiCHCMArvaJ13gP80SEWb6XZXWAlBQTglQT0dAIECPSwgBwiAK/0+grAK03oAgEBATiAbiQBAgS6REAAFoAjq2jxIuyGLiMgAFsHAgQI9K+AHCIAR7bf4kXYDRWA7QABAgQIlFLkEAE4chAsXoTdUAHYDhAgQICAALzCHRgxUP1tD74aERCAG2F10TYEvAWiDSwPJUCAwGomIIdogCMrbfEi7IZqgO0AAQIECGiANcCpUyAAp+TNHRTQANsFAgQI9K+AHKIBjmy/xYuwG6oBtgMECBAgoAHWAKdOgQCckjdXA2wHCBAgQEAO0QBHToHFi7AbqgG2AwQIECCgAdYAp06BAJySN1cDbAcIECBAQA7RAEdOgcWLsBuqAbYDBAgQIKAB1gCnToEAnJI3VwNsBwgQIEBADtEAR06BxYuwG6oBtgMECBAgoAHWAKdOgQCckjdXA2wHCBAgQEAO0QBHToHFi7AbqgG2AwQIECCgAdYAp06BAJySN1cDbAcIECBAQA7RAEdOgcWLsBuqAbYDBAgQIKAB1gCnToEAnJI3VwNsBwgQIEBADtEAR06BxYuwG6oBtgMECBAgoAHWAKdOgQCckjdXA2wHCBAgQEAO0QBHToHFi7AbqgG2AwQIECCgAdYAp06BAJySN1cDbAcIECBAQA7RAEdOgcWLsBuqAbYDBAgQIKAB1gCnToEAnJI3VwNsBwgQIEBADtEAR06BxYuwG6oBtgMECBAgoAHWAKdOgQCckjdXA2wHCBAgQEAO0QBHToHFi7AbqgG2AwQIECCgAdYAp06BAJySN1cDbAcIECBAQA7RAEdOgcWLsBuqAbYDBAgQIKAB1gCnToEAnJI3VwNsBwgQIEBADtEAR06BxYuwG6oBtgMECBAgoAHWAKdOgQCckjdXA2wHCBAgQEAO0QBHToHFi7AbqgG2AwQIECCgAdYAp06BAJySN1cDbAcIECBAQA7RAEdOgcWLsBuqAbYDBAgQIKAB1gCnToEAnJI3VwNsBwgQIEBADtEAR06BxYuwG6oBtgMECBAgoAHWAKdOgQCckjdXA2wHCBAgQEAO0QBHToHFi7AbqgG2AwQIECCgAdYAp06BAJySN1cDbAcIECBAQA7RAEdOgcWLsBuqAbYDBAgQIKAB1gCnToEAnJI3VwNsBwgQIEBADtEAR06BxYuwG6oBtgMECBAgoAHWAKdOgQCckjdXA2wHCBAgQEAO0QBHToHFi7AbqgG2AwQIECCgAdYAp06BAJySN1cDbAcIECBAQA7RAEdOgcWLsBuqAbYDBAgQIKAB1gCnToEAnJI3VwNsBwgQIEBADtEAR06BxYuwG6oBtgMECBAgoAHWAKdOgQCckjdXA2wHCBAgQEAO0QBHToHFi7AbqgG2AwQIECCgAdYAp06BAJySN1cDbAcIECBAQA7RAEdOgcWLsBuqAbYDBAgQIKAB1gCnToEAnJI3VwNsBwgQIEBADtEAR06BxYuwG6oBtgMECBAgoAHWAKdOgQCckjdXA2wHCBAgQEAO0QBHToHFi7AbqgG2AwQIECCgAdYAp06BAJx2LpY3AAAWK0lEQVSSN1cDbAcIECBAQA7RAEdOgcWLsBuqAbYDBAgQIKAB1gCnToEAnJI3VwNsBwgQIEBADtEAR06BxYuwG6oBtgMECBAgoAHWAKdOgQCckjdXA2wHCBAgQEAO0QBHToHFi7AbqgG2AwQIECCgAdYAp06BAJySN1cDbAcIECBAQA7RAEdOgcWLsBuqAbYDBAgQIKAB1gCnToEAnJI3VwNsBwgQIEBADtEAR06BxYuwG6oBtgMECBAgoAHWAKdOgQCckjdXA2wHCBAgQEAO0QBHToHFi7AbqgG2AwQIECCgAdYAp06BAJySN1cDbAcIECBAQA7RAEdOgcWLsBuqAbYDBAgQIKAB1gCnToEAnJI3VwNsBwgQIEBADtEAR06BxYuwG6oBtgMECBAgoAHWAKdOgQCckjdXA2wHCBAgQEAO0QBHToHFi7AbqgG2AwQIECCgAdYAd3IKXnvttXLGGWeUuXPnliVLlpQJEyaUmTNnlt1226325QTg2lQe2JDAmNPurn3lhTP3q/1YDyRAgACB7heQQzTAbW/p3nvvXV555ZUya9asssEGG5TLL7+8XH311eWJJ54o2267ba3rWbxaTB7UoIAA3CCuSxMgQKDLBeQQAbitFX3++efLuHHjykMPPVQmT57ceu7AwEDZZpttyuGHH17OOeecWtezeLWYPKhBAQG4QVyXJkCAQJcLyCECcFsr+sYbb5SHH364TJkypay99tpDz916663LPvvs02qF63xZvDpKHtOkgADcpK5rEyBAoLsF5BABeKU39JZbbilTp04td911VznggANqXc/i1WLyoAYFBOAGcV2aAAECXS4ghwjAK7Wi1Vshqua3aoTvuOOOT7xW9Yflqn8GvxYvXlwmTZpUFi1aVEaPHr1Sr8GTCXQiIAB3ouY5BAgQWD0EBGABuONNvvPOO8sRRxxRdtlll9YnQqyzzjqfeK3p06eXGTNmfOzfC8Ad83viSgoIwCsJ6OkECBDoYQEBWADuaH2r9/qedNJJ5eCDDy5z5sz5n/cDL++CGuCOmD2pQQEBuEFclyZAgECXCwjAAnDbK3rFFVeUE044oZx44onlsssuKyNHjmz7GhavbTJPWMUCAvAqBnU5AgQI9JCAHCIAt7Wuzz33XNluu+3K/vvvX2bPnv0/z63eArHuuuvWup7Fq8XkQQ0KCMAN4ro0AQIEulxADhGA21rR888/v5x55pnLfc7RRx9drrvuulrXs3i1mDyoQQEBuEFclyZAgECXC8ghAnBkRS1ehN3QZQQEYOtAgACB/hWQQwTgyPZbvAi7oQKwHSBAgACBUoocIgBHDoLFi7AbKgDbAQIECBAQgFe4AyMGBgYGbEkzAgJwM66uWl/AWyDqW3kkAQIEVjcBOUQDHNlpixdhN1QDbAcIECBAQAOsAU6dAgE4JW/uoIAG2C4QIECgfwXkEA1wZPstXoTdUA2wHSBAgAABDbAGOHUKBOCUvLkaYDtAgAABAnKIBjhyCixehN1QDbAdIECAAAENsAY4dQoE4JS8uRpgO0CAAAECcogGOHIKLF6E3VANsB0gQIAAAQ2wBjh1CgTglLy5GmA7QIAAAQJyiAY4cgosXoTdUA2wHSBAgAABDbAGOHUKBOCUvLkaYDtAgAABAnKIBjhyCixehN1QDbAdIECAAAENsAY4dQoE4JS8uRpgO0CAAAECcogGOHIKLF6E3VANsB0gQIAAAQ2wBjh1CgTglLy5GmA7QIAAAQJyiAY4cgosXoTdUA2wHSBAgAABDbAGOHUKBOCUvLkaYDtAgAABAnKIBjhyCixehN1QDbAdIECAAAENsAY4dQoE4JS8uRpgO0CAAAECcogGOHIKLF6E3VANsB0gQIAAAQ2wBjh1CgTglLy5GmA7QIAAAQJyiAY4cgosXoTdUA2wHSBAgAABDbAGOHUKBOCUvLkaYDtAgAABAnKIBjhyCixehN1QDbAdIECAAAENsAY4dQoE4JS8uRpgO0CAAAECcogGOHIKLF6E3VANsB0gQIAAAQ2wBjh1CgTglLy5GmA7QIAAAQJyiAY4cgosXoTdUA2wHSBAgAABDbAGOHUKBOCUvLkaYDtAgAABAnKIBjhyCixehN1QDbAdIECAAAENsAY4dQoE4JS8uRpgO0CAAAECcogGOHIKLF6E3VANsB0gQIAAAQ2wBjh1CgTglLy5GmA7QIAAAQJyiAY4cgosXoTdUA2wHSBAgAABDbAGOHUKBOCUvLkaYDtAgAABAnKIBjhyCixehN1QDbAdIECAAAENsAY4dQoE4JS8uRpgO0CAAAECcogGOHIKLF6E3VANsB0gQIAAAQ2wBjh1CgTglLy5GmA7QIAAAQJyiAY4cgosXoTdUA2wHSBAgAABDbAGOHUKBOCUvLkaYDtAgAABAnKIBjhyCixehN1QDbAdIECAAAENsAY4dQoE4JS8uRpgO0CAAAECcogGOHIKLF6E3VANsB0gQIAAAQ2wBjh1CgTglLy5GmA7QIAAAQJyiAY4cgosXoTdUA2wHSBAgAABDbAGOHUKBOCUvLkaYDtAgAABAnKIBjhyCixehN1QDbAdIECAAAENsAY4dQoE4JS8uRpgO0CAAAECcogGOHIKLF6E3VANsB0gQIAAAQ2wBjh1CgTglLy5GmA7QIAAAQJyiAY4cgosXoTdUA2wHSBAgAABDbAGOHUKBOCUvLkaYDtAgAABAnKIBjhyCixehN1QDbAdIECAAAENsAY4dQoE4JS8uRpgO0CAAAECcogGOHIKLF6E3VANsB0gQIAAAQ2wBjh1CgTglLy5GmA7QIAAAQJyiAY4cgosXoTdUA2wHSBAgAABDbAGOHUKBOCUvLkaYDtAgAABAnKIBjhyCixehN1QDbAdIECAAAENsAY4dQoE4JS8uRpgO0CAAAECcogGOHIKLF6E3VANsB0gQIAAAQ2wBjh1CgTglLy5GmA7QIAAAQJyiAY4cgosXoTdUA2wHSBAgAABDbAGOHUKBOCUvLkaYDtAgAABAnKIBjhyCixehN1QDbAdIECAAAENsAY4dQoE4JS8uRpgO0CAAAECcogGOHIKLF6E3VANsB0gQIAAAQ2wBjh1CgTglLy5GmA7QIAAAQJyiAY4cgosXoTdUA2wHSBAgAABDbAGOHUKBOCUvLkaYDtAgAABAnKIBjhyCixehN1QDbAdIECAAAENsAa4k1Pw4YcflhkzZpSrr766vP7662W33XYrs2fPLltvvXXtywnAtak8sCGBMafdXfvKC2fuV/uxHkiAAAEC3S8gh2iA297SKvxWgffaa68tm266aTn11FPLCy+8UBYsWFDWWmutWtezeLWYPKhBAQG4QVyXJkCAQJcLyCECcFsr+u6775b111+/XHTRReW4445rPfeNN94om2yySbnmmmvKt7/97VrXs3i1mDyoQQEBuEFclyZAgECXC8ghAnBbK/roo4+WnXfeuTz77LNlm222GXpu9TaICRMmtJrhOl8Wr46SxzQpIAA3qevaBAgQ6G4BOUQAbmtDb7vttnLIIYeUt956q6yzzjpDz506dWrr/zd37txa17N4tZg8qEEBAbhBXJcmQIBAlwvIIQJwWys6Z86ccuSRR5YPPvigjBw5cui5Rx11VHn55ZfLvffeu9zrLVmypFT/DH4tWrSoTJ48uVSN8sYbb9zWa/BgAqtCYJfz76t9mUfOmFL7sR5IgAABAt0vsHjx4jJp0qTy4osvljFjxnT/Cx7GVzhiYGBgYBjn9cSoW2+9tRx66KHLbYDfeeedcueddy73+5g+fXrrkyN8ESBAgAABAgS6RaAq4nbaaadueTld8ToE4OXchsH3AD///PNl7NixQ4+o3gM8ceLEcvnlly/35n20AX777bdL1QJvueWWZc0112zkhg/+7k7L3AjvsFzUPRwW5saGuH+N0Q7bhd3DYaNubJB7uHza999/v7z66qtl/PjxZdSoUY359+KFBeDl3LWq5d1www3LpZdeWo499tjWIzr5FIjhWAjv7xkO5WZnuIfN+jZ9dfevaeHmr+8eNm/c9AT3sGnh1e/6AvAn3NMzzzyzXHnlla2PPaveN3PKKaeUhQsXlvnz59f+HODhWBeHfjiUm53hHjbr2/TV3b+mhZu/vnvYvHHTE9zDpoVXv+sLwJ9wT6s/AHfGGWe0/iKMpUuXlq9+9auttz5025vIHfreP5TuYW/fQ/evt+9f9erdQ/ew9wV8B+0KCMDtinXZ46v3HVdv1fjBD35QPve5z3XZq/Ny6gi4h3WUuvcx7l/33pu6r8w9rCvVvY9zD7v33nTrKxOAu/XOeF0ECBAgQIAAAQKNCAjAjbC6KAECBAgQIECAQLcKCMDdeme8LgIECBAgQIAAgUYEBOBGWF2UAAECBAgQIECgWwUE4G69M14XAQIECBAgQIBAIwICcCOszV/0ww8/bP21y1dffXV5/fXXS/W31M2ePbtsvfXWzQ83oS2Bc889t9x7773lT3/609DznnrqqXLSSSeVxx9/vHzhC18oJ554Yjn55JOH/r372xZxIw9+7bXXWh+FOHfu3FL9CfMJEyaUmTNnts5a9eUeNsK+Si/6yiuvlB/+8IflD3/4Q+vjLPfYY49y8cUXly996Uvu4SqVbv5izz33XNlxxx3LrFmzyjHHHOP+NU++2k8QgHv0Flfhtwq81ecUb7rppuXUU08tL7zwQlmwYEFX/UUdPcq7yl72T3/609ZH1FWfIz0YgP/zn/+UbbfdtnzrW99q/btHHnmknHDCCa37+Z3vfKc12/1dZbeg4wvtvffepQpQ1X9wN9hgg9bngFe/4XziiSda/9s97Jh22J64yy67lJEjR5af//zn5bOf/Wz50Y9+VB5++OFS/TX3VSB2D4ftVqzUoPfee69Mnjy5VRhU/82rArBfR1eK1JNLKQJwD67Bu+++W9Zff/1y0UUXleOOO671HXTrX9Xcg7yr5CW/9NJL5bvf/W558MEHy2abbVY22mijoQB8wQUXtMJU9TcLrrnmmq15VdN42223lWeeeaa4v6vkFqzURaqANG7cuPLQQw+1/sNbfQ0MDJRtttmmHH744WWdddZxD1dKuPknVwHp+9//fjnrrLPKl7/85dbAv/zlL2X77bcvjz76aOunMs5h8/dhVUyofn2sioL7779/KAD7dXRVyPb3NQTgHrz/1S/eO++8c3n22Wdb/0Ee/Kp+NFv9mLZqEn1lBX73u9+VG2+8sVRvfzjnnHNaYXewAf7GN75RPv/5z5df//rXQy+y+o/x17/+9fLvf/+79Vj3N3v/qt9QVk3hlClTytprrz30Yqq3GO2zzz6tn7a4h9l71O706mxVbzOqzmH1G83DDjvMPWwXMfD4Bx54oBx00EGttxxtvvnmQwHYr6OBm7GajRSAe/CGVk3hIYccUt56661WEzX4NXXq1Nb/r3rPoq/uEah+XLdsAK5+k7LvvvuWCy+8cOhFPv30062W6rHHHiv//Oc/3d/uuX1Dr+SWW24p1Rm76667yplnnukeduE9+qSX9L3vfa9cddVVrd/MVPevenuLc9j9N7D6jejEiRNL1fYeccQRZcSIEUMB2P3r/vvX7a9QAO72O7Sc1zdnzpxy5JFHlg8++KD1/rbBr6OOOqq8/PLLrR/t+eoegY8G4KpFrH4xr5rhwa+qURw7dmzrLRNVWHZ/u+f+Va+keitE1fxWjfAdd9zR+sOm7mF33aMVvZrqN5jVe36vuOKK1k9e5s2b1/rNjHvY3fewertR9dajm266qfVClw3AzmB337teeHUCcC/cpY+8xltvvbUceuihy22A33nnnXLnnXf24He1+r7kjwbg8ePHl+rHd8trgKs/YFWFYfe3e/ahOk9VUKr+QFX105Xqpy7uYffcn3ZeSfXpKtW9q95iVP20xTlsR294H3vDDTeU008/vcyfP7+st956HwvAzuDw3o/VcZoA3IN3dfA9wNUf1Klaw8Gv6j3A1Y+Lqj/Y4at7BD4agKv/6FYffVb9Aj/4tbz3ALu/+XtYfQJE9XF1Bx98cKl+8jL4fmD3MH9vPu0VVJ/gcd9997Wa3jXWWGPo4dV7f6uPtav+f87hpynm/v1ee+3VauqXfQ/+m2++2frf1X/3tthiC/cvd3tWi8kCcA/exqrl3XDDDcull15ajj322NZ34FMguvdGfjQAV58l+4tf/KL8/e9/H/oPc9V03H777a0/nOP+dse9rH5cXn08XfUZzZdddtn/vN3IPeyOe7SiV1H9oakddtihFYK/9rWvtR5afZxW9dFnBx54YOuTWZzD7r2P1SfpVG9bWfar+mSW6idn06ZNa/0hY/eve+9fL7wyAbgX7tJyXmP1h3CuvPLKcs0115QxY8aUU045pfXe0erHRWuttVaPfler58v+aACumqnB/whXn99cNfrHH3986xfzo48+uoXg/mZ3ofrQ/e22267sv//+H/tUleotENVvUtzD7D36tOnVe0erP2xa/bpY/QG46sfo5513XrnnnntanygwatQo9/DTELvs3y/7HmC/jnbZzenBlyMA9+BNq15y9Qfgqs9GrD4UvPpdcvUXLVRvfajCsK/uEvhoAK5eXfX+w6pZfPLJJ8vGG2/c+tuqqs8sHfxyf7P38Pzzz2/9JmR5X9VvUq677jr3MHuLak3/73//23ofafUHF6ufku2+++6tn5wNfi6wc1iLsWsetGwA9uto19yWnn0hAnDP3jovnAABAgQIECBAoBMBAbgTNc8hQIAAAQIECBDoWQEBuGdvnRdOgAABAgQIECDQiYAA3Ima5xAgQIAAAQIECPSsgADcs7fOCydAgAABAgQIEOhEQADuRM1zCBAgQIAAAQIEelZAAO7ZW+eFEyBAgAABAgQIdCIgAHei5jkECBAgQIAAAQI9KyAA9+yt88IJECBAgAABAgQ6ERCAO1HzHAIECBAgQIAAgZ4VEIB79tZ54QQIECBAgAABAp0ICMCdqHkOAQIECBAgQIBAzwoIwD1767xwAgQIECBAgACBTgQE4E7UPIcAAQIECBAgQKBnBQTgnr11XjgBAgQIECBAgEAnAgJwJ2qeQ4AAAQIECBAg0LMCAnDP3jovnAABAgQIECBAoBMBAbgTNc8hQIAAAQIECBDoWQEBuGdvnRdOgAABAgQIECDQiYAA3Ima5xAgQIAAAQIECPSsgADcs7fOCydAgAABAgQIEOhEQADuRM1zCBAgQIAAAQIEelZAAO7ZW+eFEyBAgAABAgQIdCIgAHei5jkECBAgQIAAAQI9KyAA9+yt88IJECBAgAABAgQ6ERCAO1HzHAIECBAgQIAAgZ4VEIB79tZ54QQIECBAgAABAp0ICMCdqHkOAQIECBAgQIBAzwoIwD1767xwAgQIECBAgACBTgQE4E7UPIcAAQIECBAgQKBnBQTgnr11XjgBAgQIECBAgEAnAgJwJ2qeQ4AAAQIECBAg0LMCAnDP3jovnAABAgQIECBAoBMBAbgTNc8hQIAAAQIECBDoWQEBuGdvnRdOgAABAgQIECDQiYAA3Ima5xAgQIAAAQIECPSswP8DLLNhKVNbxPEAAAAASUVORK5CYII=\" width=\"639.9999861283738\">"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "there are 9 regions with more than 100 probes before blastn.\n"
     ]
    }
   ],
   "source": [
    "plt.figure()\n",
    "plt.hist(list(reg_size_dic_final.values()), bins=np.arange(0,450,10))\n",
    "plt.show()\n",
    "\n",
    "\n",
    "pb_num_goal = 100\n",
    "\n",
    "print(f'there are {np.sum(np.array(list(reg_size_dic_final.values()))>pb_num_goal)} regions with more than {pb_num_goal} probes before blastn.')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
