{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<a id='0.1'></a>\n",
    "## 0.1 load required packages"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Table of contents\n",
    "\n",
    "\n",
    "> 0. [Minimum required packages and settings](#0)\n",
    ">>\n",
    ">> 0.1: [import required packages](#0.1)\n",
    ">\n",
    "> 1. [Extract region sequences](#1)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<a id='0.1'></a>\n",
    "## 0.1 load required packages"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2022-01-19 18:16:15,577 [INFO] WRITING LOG OUTPUT TO C:\\Users\\shiwei\\.cellpose\\run.log\n",
      "19764\n"
     ]
    }
   ],
   "source": [
    "%run \"C:\\\\Users\\\\shiwei\\\\Documents\\\\ImageAnalysis3\\\\required_files\\\\Startup_py3.py\"\n",
    "sys.path.append(r\"C:\\\\Users\\\\shiwei\\\\Documents\")\n",
    "\n",
    "import ImageAnalysis3 as ia\n",
    "%matplotlib notebook\n",
    "\n",
    "from ImageAnalysis3 import *\n",
    "print(os.getpid())\n",
    "\n",
    "# library design specific tools\n",
    "from ImageAnalysis3.library_tools import LibraryDesigner as ld\n",
    "from ImageAnalysis3.library_tools import LibraryTools as lt\n",
    "# biopython imports\n",
    "from Bio import SeqIO\n",
    "from Bio.Seq import Seq\n",
    "from Bio.SeqRecord import SeqRecord\n",
    "from Bio.Blast.Applications import NcbiblastnCommandline\n",
    "from Bio.Blast import NCBIXML"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<a id='1'></a>\n",
    "# 1 Extract region sequences"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "## Some folders\n",
    "# human genome\n",
    "reference_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl'\n",
    "genome_folder = os.path.join(reference_folder, 'Genome')\n",
    "# Library directories\n",
    "pool_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- library_folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\n",
      "-- sequence_folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\n",
      "-- report_folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\reports\n"
     ]
    }
   ],
   "source": [
    "resolution = 0\n",
    "flanking = 5000 #\n",
    "\n",
    "\n",
    "# generate folders for sub-pool\n",
    "library_folder = os.path.join(pool_folder, \n",
    "                              f'mop_SE_1000')\n",
    "if not os.path.exists(library_folder):\n",
    "    print(f\"create library folder: {library_folder}\")\n",
    "    os.makedirs(library_folder)\n",
    "# folder for fasta sequences\n",
    "sequence_folder = os.path.join(library_folder, 'sequences')\n",
    "if not os.path.exists(sequence_folder):\n",
    "    print(f\"create sequence folder: {sequence_folder}\")\n",
    "    os.makedirs(sequence_folder)\n",
    "# folder to save result probes\n",
    "report_folder = os.path.join(library_folder, 'reports')\n",
    "if not os.path.exists(report_folder):\n",
    "    print(f\"create report folder: {report_folder}\")\n",
    "    os.makedirs(report_folder)\n",
    "    \n",
    "print(f\"-- library_folder: {library_folder}\")\n",
    "print(f\"-- sequence_folder: {sequence_folder}\")\n",
    "print(f\"-- report_folder: {report_folder}\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 1.1 Convert SE peak dataframe into bed-like format"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "\n",
    "sel_SE_df = pd.read_excel(library_folder+os.sep+'merged_selected_SE_all_cell_types.xlsx', index_col=None)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "991\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>CHROM</th>\n",
       "      <th>merged_START</th>\n",
       "      <th>merged_STOP</th>\n",
       "      <th>merged_CELL_TYPE</th>\n",
       "      <th>merged_initial_index</th>\n",
       "      <th>score</th>\n",
       "      <th>ave_score</th>\n",
       "      <th>merged_SIZE</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>chr1</td>\n",
       "      <td>9610471</td>\n",
       "      <td>9646458</td>\n",
       "      <td>['L6_CT']</td>\n",
       "      <td>[10392]</td>\n",
       "      <td>[1.]</td>\n",
       "      <td>1.0</td>\n",
       "      <td>35987</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>chr1</td>\n",
       "      <td>9759317</td>\n",
       "      <td>9835402</td>\n",
       "      <td>['L2_3_IT']</td>\n",
       "      <td>[2107]</td>\n",
       "      <td>[1.]</td>\n",
       "      <td>1.0</td>\n",
       "      <td>76085</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>chr1</td>\n",
       "      <td>20898605</td>\n",
       "      <td>20943251</td>\n",
       "      <td>['Astro']</td>\n",
       "      <td>[194]</td>\n",
       "      <td>[1.]</td>\n",
       "      <td>1.0</td>\n",
       "      <td>44646</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>chr1</td>\n",
       "      <td>21485446</td>\n",
       "      <td>21554433</td>\n",
       "      <td>['L2_3_IT']</td>\n",
       "      <td>[2210]</td>\n",
       "      <td>[1.]</td>\n",
       "      <td>1.0</td>\n",
       "      <td>68987</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>chr1</td>\n",
       "      <td>21684557</td>\n",
       "      <td>21783163</td>\n",
       "      <td>['Sncg']</td>\n",
       "      <td>[17985]</td>\n",
       "      <td>[1.]</td>\n",
       "      <td>1.0</td>\n",
       "      <td>98606</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  CHROM  merged_START  merged_STOP merged_CELL_TYPE merged_initial_index  \\\n",
       "0  chr1       9610471      9646458        ['L6_CT']              [10392]   \n",
       "1  chr1       9759317      9835402      ['L2_3_IT']               [2107]   \n",
       "2  chr1      20898605     20943251        ['Astro']                [194]   \n",
       "3  chr1      21485446     21554433      ['L2_3_IT']               [2210]   \n",
       "4  chr1      21684557     21783163         ['Sncg']              [17985]   \n",
       "\n",
       "  score  ave_score  merged_SIZE  \n",
       "0  [1.]        1.0        35987  \n",
       "1  [1.]        1.0        76085  \n",
       "2  [1.]        1.0        44646  \n",
       "3  [1.]        1.0        68987  \n",
       "4  [1.]        1.0        98606  "
      ]
     },
     "execution_count": 33,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "print(len(sel_SE_df))\n",
    "sel_SE_df.head()\n",
    "\n",
    "# merged_initial_index shows the initial index before merging, which can be used to locate detailed info (e.g., loci number)\n",
    "# using the combined_cell_type_SE.xlsx"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-36-6bd7711621a2>:7: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  sel_SE_df_chrN['sort_CHROM'] = [int(i.split('chr')[-1]) for i in sel_SE_df_chrN['CHROM']]\n",
      "<ipython-input-36-6bd7711621a2>:9: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  sel_SE_df_chrX['sort_CHROM'] = 20\n"
     ]
    }
   ],
   "source": [
    "# split chrX from other chromosomes to sort chromosomes (not required for probe design later)\n",
    "sel_SE_df_chrX = sel_SE_df[984:]\n",
    "sel_SE_df_chrN = sel_SE_df[:984]\n",
    "\n",
    "\n",
    "# sort other chromosomes\n",
    "sel_SE_df_chrN['sort_CHROM'] = [int(i.split('chr')[-1]) for i in sel_SE_df_chrN['CHROM']]\n",
    "# set chrX as chr20\n",
    "sel_SE_df_chrX['sort_CHROM'] = 20\n",
    "\n",
    "# sort\n",
    "sel_SE_df_sort_chrN = sel_SE_df_chrN.sort_values(by =['sort_CHROM', 'merged_START'])\n",
    "sel_SE_df_sort_chrX = sel_SE_df_chrX.sort_values(by =['sort_CHROM', 'merged_START'])\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>index</th>\n",
       "      <th>CHROM</th>\n",
       "      <th>merged_START</th>\n",
       "      <th>merged_STOP</th>\n",
       "      <th>merged_CELL_TYPE</th>\n",
       "      <th>merged_initial_index</th>\n",
       "      <th>score</th>\n",
       "      <th>ave_score</th>\n",
       "      <th>merged_SIZE</th>\n",
       "      <th>sort_CHROM</th>\n",
       "      <th>region_name</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0</td>\n",
       "      <td>chr1</td>\n",
       "      <td>9610471</td>\n",
       "      <td>9646458</td>\n",
       "      <td>['L6_CT']</td>\n",
       "      <td>[10392]</td>\n",
       "      <td>[1.]</td>\n",
       "      <td>1.00000</td>\n",
       "      <td>35987</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1</td>\n",
       "      <td>chr1</td>\n",
       "      <td>9759317</td>\n",
       "      <td>9835402</td>\n",
       "      <td>['L2_3_IT']</td>\n",
       "      <td>[2107]</td>\n",
       "      <td>[1.]</td>\n",
       "      <td>1.00000</td>\n",
       "      <td>76085</td>\n",
       "      <td>1</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>2</td>\n",
       "      <td>chr1</td>\n",
       "      <td>20898605</td>\n",
       "      <td>20943251</td>\n",
       "      <td>['Astro']</td>\n",
       "      <td>[194]</td>\n",
       "      <td>[1.]</td>\n",
       "      <td>1.00000</td>\n",
       "      <td>44646</td>\n",
       "      <td>1</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>3</td>\n",
       "      <td>chr1</td>\n",
       "      <td>21485446</td>\n",
       "      <td>21554433</td>\n",
       "      <td>['L2_3_IT']</td>\n",
       "      <td>[2210]</td>\n",
       "      <td>[1.]</td>\n",
       "      <td>1.00000</td>\n",
       "      <td>68987</td>\n",
       "      <td>1</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>4</td>\n",
       "      <td>chr1</td>\n",
       "      <td>21684557</td>\n",
       "      <td>21783163</td>\n",
       "      <td>['Sncg']</td>\n",
       "      <td>[17985]</td>\n",
       "      <td>[1.]</td>\n",
       "      <td>1.00000</td>\n",
       "      <td>98606</td>\n",
       "      <td>1</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>95</th>\n",
       "      <td>534</td>\n",
       "      <td>chr2</td>\n",
       "      <td>49908807</td>\n",
       "      <td>49960037</td>\n",
       "      <td>['Sst']</td>\n",
       "      <td>[19796]</td>\n",
       "      <td>[1.]</td>\n",
       "      <td>1.00000</td>\n",
       "      <td>51230</td>\n",
       "      <td>2</td>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>96</th>\n",
       "      <td>535</td>\n",
       "      <td>chr2</td>\n",
       "      <td>58015861</td>\n",
       "      <td>58055893</td>\n",
       "      <td>['Oligo']</td>\n",
       "      <td>[15779]</td>\n",
       "      <td>[1.]</td>\n",
       "      <td>1.00000</td>\n",
       "      <td>40032</td>\n",
       "      <td>2</td>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>97</th>\n",
       "      <td>536</td>\n",
       "      <td>chr2</td>\n",
       "      <td>59644429</td>\n",
       "      <td>59675853</td>\n",
       "      <td>['Oligo']</td>\n",
       "      <td>[16120]</td>\n",
       "      <td>[1.]</td>\n",
       "      <td>1.00000</td>\n",
       "      <td>31424</td>\n",
       "      <td>2</td>\n",
       "      <td>98</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>98</th>\n",
       "      <td>537</td>\n",
       "      <td>chr2</td>\n",
       "      <td>62477453</td>\n",
       "      <td>62533544</td>\n",
       "      <td>['L6_CT', 'L5_IT', 'L4_5_IT', 'L2_3_IT', 'L6_IT']</td>\n",
       "      <td>[10667, 6494, 3413, 2156, 12347]</td>\n",
       "      <td>[0.88267 0.99257 0.82405 0.78742 0.78774]</td>\n",
       "      <td>0.85489</td>\n",
       "      <td>56091</td>\n",
       "      <td>2</td>\n",
       "      <td>99</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>99</th>\n",
       "      <td>538</td>\n",
       "      <td>chr2</td>\n",
       "      <td>62680661</td>\n",
       "      <td>62728745</td>\n",
       "      <td>['L4_5_IT']</td>\n",
       "      <td>[4165]</td>\n",
       "      <td>[1.]</td>\n",
       "      <td>1.00000</td>\n",
       "      <td>48084</td>\n",
       "      <td>2</td>\n",
       "      <td>100</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>100 rows × 11 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "    index CHROM  merged_START  merged_STOP  \\\n",
       "0       0  chr1       9610471      9646458   \n",
       "1       1  chr1       9759317      9835402   \n",
       "2       2  chr1      20898605     20943251   \n",
       "3       3  chr1      21485446     21554433   \n",
       "4       4  chr1      21684557     21783163   \n",
       "..    ...   ...           ...          ...   \n",
       "95    534  chr2      49908807     49960037   \n",
       "96    535  chr2      58015861     58055893   \n",
       "97    536  chr2      59644429     59675853   \n",
       "98    537  chr2      62477453     62533544   \n",
       "99    538  chr2      62680661     62728745   \n",
       "\n",
       "                                     merged_CELL_TYPE  \\\n",
       "0                                           ['L6_CT']   \n",
       "1                                         ['L2_3_IT']   \n",
       "2                                           ['Astro']   \n",
       "3                                         ['L2_3_IT']   \n",
       "4                                            ['Sncg']   \n",
       "..                                                ...   \n",
       "95                                            ['Sst']   \n",
       "96                                          ['Oligo']   \n",
       "97                                          ['Oligo']   \n",
       "98  ['L6_CT', 'L5_IT', 'L4_5_IT', 'L2_3_IT', 'L6_IT']   \n",
       "99                                        ['L4_5_IT']   \n",
       "\n",
       "                merged_initial_index  \\\n",
       "0                            [10392]   \n",
       "1                             [2107]   \n",
       "2                              [194]   \n",
       "3                             [2210]   \n",
       "4                            [17985]   \n",
       "..                               ...   \n",
       "95                           [19796]   \n",
       "96                           [15779]   \n",
       "97                           [16120]   \n",
       "98  [10667, 6494, 3413, 2156, 12347]   \n",
       "99                            [4165]   \n",
       "\n",
       "                                        score  ave_score  merged_SIZE  \\\n",
       "0                                        [1.]    1.00000        35987   \n",
       "1                                        [1.]    1.00000        76085   \n",
       "2                                        [1.]    1.00000        44646   \n",
       "3                                        [1.]    1.00000        68987   \n",
       "4                                        [1.]    1.00000        98606   \n",
       "..                                        ...        ...          ...   \n",
       "95                                       [1.]    1.00000        51230   \n",
       "96                                       [1.]    1.00000        40032   \n",
       "97                                       [1.]    1.00000        31424   \n",
       "98  [0.88267 0.99257 0.82405 0.78742 0.78774]    0.85489        56091   \n",
       "99                                       [1.]    1.00000        48084   \n",
       "\n",
       "    sort_CHROM  region_name  \n",
       "0            1            1  \n",
       "1            1            2  \n",
       "2            1            3  \n",
       "3            1            4  \n",
       "4            1            5  \n",
       "..         ...          ...  \n",
       "95           2           96  \n",
       "96           2           97  \n",
       "97           2           98  \n",
       "98           2           99  \n",
       "99           2          100  \n",
       "\n",
       "[100 rows x 11 columns]"
      ]
     },
     "execution_count": 37,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# combine chrX and other chromosome after sorting\n",
    "sel_SE_df_sort_all = pd.concat([sel_SE_df_sort_chrN, sel_SE_df_sort_chrX])\n",
    "# reset index\n",
    "sel_SE_df_new = sel_SE_df_sort_all.reset_index()\n",
    "\n",
    "# add new region name;\n",
    "sel_SE_df_new['region_name'] = sel_SE_df_new.index +1\n",
    "\n",
    "# index col shows the original index(region) name which can be used to locate the SE before merging \n",
    "# original SE de: merged_selected_SE_all_cell_types.xlsx or the sel_SE_df\n",
    "# merged_selected_SE_all_cell_types.xlsx contains the top 991 SE after filtering using the merged_SE_all_cell_types.xlsx\n",
    "# see merging and filtering details from below: \n",
    "# http://localhost:8889/notebooks/Documents/Chromatin_Analysis_Scripts/Library_design/CTP13-mop-merged-1000-SE/super_enhancer_MOP_combined_all_cell_types.ipynb\n",
    "\n",
    "sel_SE_df_new[:100]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-38-10a9c88213e7>:3: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  sel_SE_df_for_bed['merged_START_str']=[str(i) for i in sel_SE_df_for_bed['merged_START']]\n",
      "<ipython-input-38-10a9c88213e7>:4: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  sel_SE_df_for_bed['merged_STOP_str']=[str(i) for i in sel_SE_df_for_bed['merged_STOP']]\n",
      "<ipython-input-38-10a9c88213e7>:5: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  sel_SE_df_for_bed['region_name_str']=[str(i) for i in sel_SE_df_for_bed['region_name']]\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>CHROM</th>\n",
       "      <th>merged_START_str</th>\n",
       "      <th>merged_STOP_str</th>\n",
       "      <th>region_name_str</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>chr1</td>\n",
       "      <td>9610471</td>\n",
       "      <td>9646458</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>chr1</td>\n",
       "      <td>9759317</td>\n",
       "      <td>9835402</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>chr1</td>\n",
       "      <td>20898605</td>\n",
       "      <td>20943251</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>chr1</td>\n",
       "      <td>21485446</td>\n",
       "      <td>21554433</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>chr1</td>\n",
       "      <td>21684557</td>\n",
       "      <td>21783163</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  CHROM merged_START_str merged_STOP_str region_name_str\n",
       "0  chr1          9610471         9646458               1\n",
       "1  chr1          9759317         9835402               2\n",
       "2  chr1         20898605        20943251               3\n",
       "3  chr1         21485446        21554433               4\n",
       "4  chr1         21684557        21783163               5"
      ]
     },
     "execution_count": 38,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Use relevant columns to make a new dataframe for bed\n",
    "sel_SE_df_for_bed =sel_SE_df_new[['CHROM','merged_START','merged_STOP','region_name']]\n",
    "sel_SE_df_for_bed['merged_START_str']=[str(i) for i in sel_SE_df_for_bed['merged_START']]\n",
    "sel_SE_df_for_bed['merged_STOP_str']=[str(i) for i in sel_SE_df_for_bed['merged_STOP']]\n",
    "sel_SE_df_for_bed['region_name_str']=[str(i) for i in sel_SE_df_for_bed['region_name']]\n",
    "\n",
    "sel_SE_df_for_bed = sel_SE_df_for_bed[['CHROM','merged_START_str','merged_STOP_str','region_name_str']]\n",
    "sel_SE_df_for_bed.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#generate a 'bed' like file from the processed dataframe above\n",
    "\n",
    "region_file = os.path.join(library_folder, 'Seleted_merged_super_enhancer_list.bed')\n",
    "import csv\n",
    "\n",
    "\n",
    "with open(region_file, 'w') as _reg_file: # bed-file\n",
    "\n",
    "    #tsv_writer = csv.writer(_reg_file, delimiter='\\\\t')  \n",
    "    #new_line_writer = csv.writer(_reg_file, delimiter='\\\\n')\n",
    "    \n",
    "    # get all columns\n",
    "    df_columns = list(sel_SE_df_for_bed.columns)\n",
    "    \n",
    "    # for each SE, then for each column\n",
    "    for _peak in sel_SE_df_for_bed.iloc():\n",
    "        for  _col in df_columns:\n",
    "            _reg_file.write(str(_peak[_col])) # append info\n",
    "            _reg_file.write('\\\\t') # tab each column\n",
    "        \n",
    "        # newline for each SE\n",
    "        _reg_file.write('\\\\n')\n",
    "        \n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 1.2 Extract seqeuence from the bed file"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Input region file is: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\Super_enhancer_list_mTEC.bed\n",
      "- 966 regions loaded from file: Super_enhancer_list_mTEC.bed\n",
      "-- searching among 23 references\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.X.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- saving sequences into folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\20-chr1-6997-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\19-chr14-16350-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\17-chr6-21729-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\18-chr16-1616-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\19-chr6-17740-lociStitched_reg_1.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\15-chr5-17565-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\19-chr8-15670-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\16-chr7-17933-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\13-chr11-636-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\14-chr11-5021-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\11-chr16-472-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\18-chr15-14399-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\14-chr2-23423-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\14-chr3-450-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\15-chr7-15564-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\13-chr12-22409-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\14-chr2-23217-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\12-chr11-14010-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\14-chr16-25870-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\chr17-1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\10-chr15-5429-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\13-chr17-16574-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\12-chr17-14525-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\13-chr6-9380-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\13-chr18-19508-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\12-chr9-24618-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\12-chr8-9480-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\12-chr3-9253-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\10-chr14-25472-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\10-chr14-25596-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\11-chr11-458-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\13-chr14-9704-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\14-chr14-12796-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\8-chr11-25008-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\11-chr10-22208-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\9-chr4-6530-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\9-chr4-21424-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\11-chr2-23252-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\9-chr2-17022-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\10-chr2-218-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\11-chr14-25531-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\11-chr6-8550-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\10-chr14-1234-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\15-chr8-4113-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\11-chr12-2380-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\12-chr2-8319-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\10-chr5-7148-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\10-chr16-14422-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr2-11986-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\9-chr11-3900-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\9-chr1-853-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\13-chr11-18648-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\10-chr18-14559-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\13-chr18-11736-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\11-chr16-20780-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr11-13993-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr8-4115-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\8-chr5-3452-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\9-chr2-10840-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\8-chr7-12325-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr8-1278-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\9-chr7-5265-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\10-chr19-7541-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\8-chr10-2220-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\10-chr1-23037-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\8-chr15-14407-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\10-chr3-4019-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\8-chr15-10639-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\9-chr7-17939-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\11-chr8-10294-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\12-chr2-21275-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\9-chr7-3856-lociStitched_reg_1.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr16-32-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\9-chr18-7526-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr1-198-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\8-chr1-23055-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\9-chr5-17543-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\10-chr2-16995-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr14-9697-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr8-20297-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\8-chr17-4791-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr9-9523-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\8-chr14-96-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr11-484-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\8-chr1-7565-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr17-25888-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\8-chr10-5331-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\8-chr11-2359-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\9-chr2-10008-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\10-chr6-3467-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr5-386-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\9-chr12-763-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\10-chr8-7886-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\8-chr10-5001-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\8-chr5-15341-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\11-chr11-6804-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\8-chr11-5380-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr1-9150-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\10-chr17-8179-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\8-chr13-20594-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr10-5783-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\8-chr2-9219-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr7-11206-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr10-52-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\8-chr7-3274-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\8-chr11-14077-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr15-806-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr15-118-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\8-chr12-2377-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr12-11492-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr1-21042-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\8-chr6-4641-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\10-chr14-25565-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr5-17605-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\9-chr2-14947-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\9-chr6-17801-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\10-chr11-16021-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\9-chr1-713-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\11-chr12-8049-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr19-3572-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr16-11680-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr2-19803-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr14-3341-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\9-chr4-17411-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr1-11812-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\10-chr12-20567-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\9-chr6-15435-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr7-13702-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr7-7835-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\8-chr4-23729-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\9-chr4-11002-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\8-chr5-9352-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr11-1396-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\8-chr1-3219-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr16-22765-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\8-chr5-13478-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\8-chr8-11222-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr11-20426-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\8-chr7-17910-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr15-16415-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr7-10252-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr2-6027-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr2-11952-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr15-22664-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr9-13867-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\8-chr1-11885-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr7-10220-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr4-10073-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\8-chr4-7109-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\8-chr14-12813-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr2-10843-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr11-5813-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr5-7140-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr7-3480-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr1-13115-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr9-24743-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr3-6524-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\chr10-2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\8-chr1-6435-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr11-799-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr5-276-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr11-88-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr2-1308-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr12-1778-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr9-2503-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr4-3617-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr1-74-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr7-21780-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\8-chr4-15189-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr11-3313-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr2-9954-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr17-6910-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr11-16102-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr2-13194-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr18-2685-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr15-19102-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr1-6452-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr6-63-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr19-7547-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr14-19055-lociStitched_reg_1.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr11-5024-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr9-4391-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr9-18171-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr9-2500-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr6-21733-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\8-chr11-1765-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr16-19253-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr17-3959-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr8-8652-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr7-5731-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr15-25633-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr2-4854-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr5-7736-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\8-chr1-5537-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr3-10048-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr14-5417-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr3-10954-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr3-15009-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr12-3321-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr6-24136-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr13-548-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr8-15619-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr11-2222-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr9-7303-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\8-chr11-7372-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr18-2055-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr7-4091-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr15-10603-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr11-5822-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr7-12304-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr15-6369-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr5-9342-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr6-7167-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr1-1010-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr4-9268-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr11-2952-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\9-chr8-1925-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr2-240-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr7-17814-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr17-4517-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr14-16336-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr12-14145-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr9-24652-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr1-8254-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr15-14391-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr17-849-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr11-12673-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr15-4767-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr11-10491-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr5-8510-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr4-13415-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\10-chr3-4298-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr4-12131-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\10-chr11-10420-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr5-2304-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr6-15420-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr12-4170-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\9-chr10-24837-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr6-21739-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr1-9157-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr3-493-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr8-18011-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\8-chr2-2169-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr19-20948-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr3-397-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr7-340-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr5-1819-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr6-20132-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr1-16925-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr4-17388-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr13-2237-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr18-580-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr1-9117-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr15-14382-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr6-1825-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr2-332-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr2-11921-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr10-389-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr4-15200-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr9-7910-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr6-13626-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr7-20193-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr11-2373-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr6-20085-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr8-13729-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr6-11137-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\8-chr1-10787-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr15-2997-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr7-8579-lociStitched_reg_1.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr4-19941-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr10-1529-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr11-1855-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr12-14160-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr2-13153-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr16-2829-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr13-14190-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr13-22531-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr2-9197-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr3-17206-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr4-15215-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr11-22255-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr9-2108-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr9-1672-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr7-5257-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr9-10352-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr5-17603-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr9-15729-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr15-11617-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr9-7916-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr2-13192-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr4-13377-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr5-688-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr5-77-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr18-19509-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr15-2825-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr1-23034-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr16-1958-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr14-19004-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr11-14002-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr10-1077-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr19-4807-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr8-9465-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr5-149-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr10-7946-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr15-22686-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr4-9291-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr8-6199-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr9-697-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr13-25470-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr7-20173-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr11-1400-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr6-7166-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr4-7722-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr14-16318-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr13-18884-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr2-17043-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\8-chr11-25078-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr6-15444-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr13-8864-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr18-16666-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr2-23413-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr11-22378-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr8-10314-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr17-4509-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr2-2440-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr3-28-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr7-20185-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr2-23394-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr8-8659-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr5-7153-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr1-14674-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr6-8565-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr9-24615-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr13-11523-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr1-35-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr13-1474-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr19-1883-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr4-6534-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr11-985-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr10-22105-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr6-15407-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr11-11434-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr3-21382-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr1-4826-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr2-23171-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr19-7540-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr11-1767-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr7-17961-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr14-22535-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr18-19523-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr12-18787-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr14-12791-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr8-13786-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr13-95-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr14-8941-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr13-25-lociStitched_reg_1.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr18-9829-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr8-20289-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr2-431-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr6-2314-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr11-20455-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr4-9275-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr11-22310-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr11-24985-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr2-3048-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr9-13855-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr11-547-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr4-15223-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr1-23077-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr16-9018-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\8-chr4-17351-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr10-1163-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr17-173-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr9-8699-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr15-19202-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr2-331-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr1-1636-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr12-25322-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr19-19612-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr7-11214-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr6-7797-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr3-106-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr4-11027-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr18-104-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr13-5059-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr5-10149-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr3-9257-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr4-325-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr14-25589-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr5-17657-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr6-288-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr7-24392-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr19-26197-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr1-19666-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr11-22262-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr1-10783-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr3-10929-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr11-3147-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\8-chr17-9031-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr10-6254-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr5-211-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr14-16365-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr13-6321-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr14-202-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr1-17-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr7-24355-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr4-2092-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\8-chr7-24264-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr2-23364-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr8-20261-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr2-10898-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr13-1406-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr1-21048-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr3-23586-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr12-18817-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr10-1221-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr8-1840-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr6-15390-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr12-2033-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr15-11593-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr9-3669-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr7-12356-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr5-9328-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr16-7495-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr15-6358-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr3-9258-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr9-7274-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr13-16212-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr9-24634-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr10-8754-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr1-9158-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr1-16785-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr3-17212-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr5-2302-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr14-1411-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr11-16072-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr18-6414-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr16-11654-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr16-22742-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr2-1362-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr11-732-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr12-7401-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr1-23095-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr2-8339-lociStitched_reg_1.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr11-24971-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr15-7473-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr11-4722-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr4-158-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr9-15748-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr9-12484-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr5-8521-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr6-8548-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr2-21203-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr12-25252-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr4-10119-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr7-7812-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr7-9449-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr4-6104-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr19-14630-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr6-13620-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr10-12529-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr1-23009-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr14-3714-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr15-2403-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr12-22393-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr6-13625-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr2-2078-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr15-8124-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr10-22187-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr6-17737-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr2-11924-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr4-19959-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr7-13652-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr5-7768-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr10-3885-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr7-24216-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr11-3697-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr2-2168-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr11-2520-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr5-23989-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr7-600-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr6-24116-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr18-1004-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr14-19083-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr10-2019-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr10-3136-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr6-17742-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr11-25016-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr5-23999-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr4-19989-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr12-1083-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr8-24420-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr9-22010-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr4-15158-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr1-10777-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr8-2490-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr14-25501-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr9-1676-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr8-10315-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr14-19048-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr11-6800-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr4-346-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr17-25980-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr4-13405-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr9-2779-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr9-4395-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr6-1067-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr13-18864-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr10-9561-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr11-25105-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr12-12719-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr4-19984-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr11-8821-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr3-4582-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr15-25661-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr12-25325-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr5-11057-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr19-16728-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr10-12544-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr14-1410-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr3-2454-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr4-8432-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr17-1136-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr8-9456-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr2-1015-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr17-5459-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr2-23369-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr2-9968-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr11-5809-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr8-2333-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr11-4437-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr10-18423-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr13-9682-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr14-19067-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr2-21172-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr2-16967-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr10-3301-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr8-9468-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr2-1148-lociStitched_reg_1.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr6-9365-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr11-18702-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr1-14711-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr9-3289-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr15-8115-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr1-19646-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr2-17071-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr15-7467-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr9-8711-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr2-11896-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr17-19329-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr12-5037-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr7-1831-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr10-632-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr7-150-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr11-8791-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr8-24575-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr4-12128-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr19-2147-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr8-11252-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr11-5017-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr1-19652-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr9-11323-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr12-9666-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr12-18801-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr10-20377-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr11-8799-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr13-3542-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr2-4842-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr8-13788-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr2-10015-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr6-17747-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr18-1554-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr10-18419-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr5-10157-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\1-chr4-4-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr4-10988-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr2-448-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr13-3917-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr5-21584-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr1-10801-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr14-22593-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr14-3725-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr18-19527-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr11-7368-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr7-11185-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr5-15348-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr15-6894-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr1-3221-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr13-11520-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr7-17837-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr10-24839-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr2-6047-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr7-15566-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr6-11138-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr15-22689-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr16-25873-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr12-8849-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr5-17505-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr17-5479-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr7-194-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr7-15532-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr8-24406-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr4-19979-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr16-6907-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr8-6681-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr9-3500-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr4-12121-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr2-23318-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr1-13045-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr12-6823-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr2-2276-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr6-17697-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\7-chr10-3682-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr17-25964-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr11-6806-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr16-4780-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr11-5806-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr10-2643-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr9-8701-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr11-8007-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr2-10883-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr19-238-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr12-16143-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr15-11599-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr4-23688-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr11-15990-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr14-12829-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr3-19870-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr15-6354-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr2-14836-lociStitched_reg_1.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr11-10471-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr9-20331-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr10-12541-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr10-6756-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr13-8871-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr8-6677-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr7-4935-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr8-24485-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr2-16991-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr17-9801-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr2-6058-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr5-23878-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr11-12645-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr4-21504-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr1-4821-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr2-13209-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr2-184-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr2-17116-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr7-204-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr19-58-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr2-4558-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr1-16842-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr11-25074-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr1-13114-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr11-12642-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr7-2485-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr11-20450-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr14-14267-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr13-12774-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr13-20587-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr5-9349-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr2-2712-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr1-21081-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr19-11776-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr6-3638-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr7-289-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr3-3605-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr16-11649-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr5-3628-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr11-12670-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr15-3342-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr3-17180-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr5-23847-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr19-9088-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr7-3853-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr9-6226-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr6-11145-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr3-9255-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr17-531-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr7-13679-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr14-14309-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr7-13646-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr1-8272-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr4-17375-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr1-3396-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr8-15614-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr10-4989-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr12-8044-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr2-7052-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr9-2934-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr13-16211-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr15-25746-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr1-13105-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr16-25798-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr17-14489-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr5-20058-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr15-64-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr1-16877-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr2-21247-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr17-36-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr7-17925-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr4-8484-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr2-14872-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr11-16101-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr14-9701-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr5-192-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr3-13313-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr4-560-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr17-1628-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr10-24785-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr15-19112-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr5-20040-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr2-10850-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr11-25214-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr2-11932-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr8-24528-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr11-6271-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr9-2013-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr11-5369-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr2-4858-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr11-4709-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr9-2939-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr16-19298-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr11-12664-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr3-4296-lociStitched_reg_1.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr16-16503-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr17-9784-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr5-15322-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr17-14482-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr8-20284-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr16-22743-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr7-67-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr15-25687-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr11-22329-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr11-12617-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr4-11011-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr8-12395-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr11-10475-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr2-4845-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr10-24882-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr18-1793-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr8-11242-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr9-24724-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr8-18014-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr8-7862-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr9-56-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr4-21462-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr2-7054-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr14-14333-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr4-19983-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr4-1196-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr6-17717-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr2-9997-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr9-15779-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr4-1650-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr19-2148-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr2-3423-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr1-1252-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr8-18141-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr2-895-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr3-23596-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr1-11827-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr15-22675-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr6-24129-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr10-7955-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr12-3702-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr3-5603-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr12-2381-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr13-5866-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr6-7164-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr17-1183-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr1-13129-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr1-9132-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\6-chr2-8335-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr11-25086-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr11-13999-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr5-20016-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr11-6798-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr11-25102-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr2-75-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr12-10494-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr4-6099-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr13-6837-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr13-18890-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr1-6979-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr5-13537-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr16-25783-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr10-5333-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr2-641-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr13-20598-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr1-1723-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr17-16567-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr4-1509-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr6-7195-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr7-21812-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\1-chr10-16-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr4-19975-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr16-16521-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr7-2620-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr10-5332-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr2-3044-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr3-1566-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr19-22945-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr7-12322-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr3-1503-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr6-4351-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr8-9484-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr10-4994-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr2-14906-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr10-24867-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr9-1593-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chrX-11793-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr8-4667-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr15-8973-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr2-13169-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr1-10802-lociStitched_reg_1.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr16-19309-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr11-14052-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr4-17451-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr13-2815-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr11-18662-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr11-7990-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr4-21432-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr17-1627-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr1-16810-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr19-26186-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr6-7775-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr10-22182-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr7-15607-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr15-14394-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr7-1923-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr11-20435-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr15-2545-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr4-1372-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr1-16815-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr13-22529-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr12-18788-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr13-25377-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr4-6549-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr1-11806-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr11-5830-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr6-1063-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr13-22492-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr11-20488-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr4-335-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr16-510-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr10-9543-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr9-675-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr19-10730-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr13-18847-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr9-22006-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr10-11327-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr3-8410-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr5-23927-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr4-17469-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr4-8423-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr6-6165-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr4-21425-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr9-24731-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr14-25477-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr6-11124-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr3-15074-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr5-5654-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr5-15271-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr5-17568-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr17-4508-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr16-5917-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr4-87-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr1-1887-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr1-7557-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr13-8068-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr16-3949-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr19-1795-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr12-16111-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr9-7294-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr13-16200-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr6-21706-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr2-11898-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr17-4512-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr2-9171-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr19-10745-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr8-8637-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr11-8012-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr11-16077-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr19-100-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr6-7173-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr9-24644-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr4-6105-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr15-12864-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr15-4494-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr10-13957-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr11-12594-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr8-833-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr11-25180-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr9-7905-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr5-11089-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr11-18677-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr11-16082-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr17-3567-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr11-24947-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr7-872-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr17-9823-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr11-15984-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr6-11131-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr17-215-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr15-14354-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr2-14873-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr15-1415-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr1-16934-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr11-6274-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr9-11324-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr1-14666-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr4-17349-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr11-14110-lociStitched_reg_1.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr14-7426-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr8-18052-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr16-19238-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr5-15261-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr12-5851-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr5-15326-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr3-12045-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr16-19252-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr17-5937-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr10-3683-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr15-3942-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr11-11419-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr13-11517-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr2-11913-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr10-22142-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr4-12077-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr11-3695-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr1-9165-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr7-20211-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr2-21198-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr17-16562-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr5-1514-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr16-11666-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr18-1426-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr14-6864-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\1-chr4-30-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr7-3471-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr10-11366-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\2-chr8-10310-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr17-5463-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr15-22637-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr9-20355-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr6-13545-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr1-7554-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr4-21499-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr5-17530-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr10-7327-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr6-1577-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\5-chr5-12167-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr1-9897-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr8-6696-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr15-22640-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr9-15792-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr17-9802-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr9-4399-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr9-18326-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr2-1057-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr4-417-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr2-23182-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\4-chr13-5398-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr9-8684-lociStitched_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\SuperEnhancers\\sequences\\3-chr1-23126-lociStitched_reg_1.fasta\n"
     ]
    }
   ],
   "source": [
    "\n",
    "\n",
    "reload(library_tools.sequences)\n",
    "region_file = os.path.join(library_folder, 'Seleted_merged_super_enhancer_list.bed')\n",
    "region_dict = library_tools.sequences.read_region_file(region_file)\n",
    "# add strand information\n",
    "for _i in range(len(region_dict)):\n",
    "    region_dict[_i]['Strand'] = '+'\n",
    "# extract and save the corresponding sequence as individual fasta    \n",
    "records = library_tools.sequences.extract_sequence(region_dict, genome_folder, \n",
    "                                                   resolution=resolution,\n",
    "                                                   flanking=flanking,\n",
    "                                                   save_folder=sequence_folder)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "metadata": {},
   "outputs": [],
   "source": [
    "# calculate num of regions in each chro\n",
    "chr_2_region_num = {}\n",
    "for _reg_info in region_dict:\n",
    "    _chr_name = _reg_info['Chr']\n",
    "    if 'chr' in _chr_name:\n",
    "        _chr = _chr_name.split('chr')[1]\n",
    "    else:\n",
    "        _chr = _chr_name\n",
    "    if _chr not in chr_2_region_num:\n",
    "        chr_2_region_num[_chr] = 0\n",
    "    chr_2_region_num[_chr] += 1\n",
    "# save\n",
    "pickle.dump(chr_2_region_num, open(os.path.join(library_folder,'chr_2_region_num.pkl'), 'wb'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[{'Chr': 'chr1',\n",
       "  'Start': '189749530',\n",
       "  'End': '189847686',\n",
       "  'Name': '20-chr1-6997-lociStitched',\n",
       "  'Gene': '20-chr1-6997-lociStitched',\n",
       "  'Region': '1:189749530-189847686',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '74848823',\n",
       "  'End': '74943405',\n",
       "  'Name': '19-chr14-16350-lociStitched',\n",
       "  'Gene': '19-chr14-16350-lociStitched',\n",
       "  'Region': '14:74848823-74943405',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '134116442',\n",
       "  'End': '134191730',\n",
       "  'Name': '17-chr6-21729-lociStitched',\n",
       "  'Gene': '17-chr6-21729-lociStitched',\n",
       "  'Region': '6:134116442-134191730',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '10434893',\n",
       "  'End': '10507397',\n",
       "  'Name': '18-chr16-1616-lociStitched',\n",
       "  'Gene': '18-chr16-1616-lociStitched',\n",
       "  'Region': '16:10434893-10507397',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '91969691',\n",
       "  'End': '92044798',\n",
       "  'Name': '19-chr6-17740-lociStitched',\n",
       "  'Gene': '19-chr6-17740-lociStitched',\n",
       "  'Region': '6:91969691-92044798',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '96855351',\n",
       "  'End': '96923237',\n",
       "  'Name': '15-chr5-17565-lociStitched',\n",
       "  'Gene': '15-chr5-17565-lociStitched',\n",
       "  'Region': '5:96855351-96923237',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '79339832',\n",
       "  'End': '79392770',\n",
       "  'Name': '19-chr8-15670-lociStitched',\n",
       "  'Gene': '19-chr8-15670-lociStitched',\n",
       "  'Region': '8:79339832-79392770',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '110233169',\n",
       "  'End': '110307805',\n",
       "  'Name': '16-chr7-17933-lociStitched',\n",
       "  'Gene': '16-chr7-17933-lociStitched',\n",
       "  'Region': '7:110233169-110307805',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '118179682',\n",
       "  'End': '118246330',\n",
       "  'Name': '13-chr11-636-lociStitched',\n",
       "  'Gene': '13-chr11-636-lociStitched',\n",
       "  'Region': '11:118179682-118246330',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '79076877',\n",
       "  'End': '79142316',\n",
       "  'Name': '14-chr11-5021-lociStitched',\n",
       "  'Gene': '14-chr11-5021-lociStitched',\n",
       "  'Region': '11:79076877-79142316',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '10728247',\n",
       "  'End': '10783712',\n",
       "  'Name': '11-chr16-472-lociStitched',\n",
       "  'Gene': '11-chr16-472-lociStitched',\n",
       "  'Region': '16:10728247-10783712',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '86062708',\n",
       "  'End': '86134699',\n",
       "  'Name': '18-chr15-14399-lociStitched',\n",
       "  'Gene': '18-chr15-14399-lociStitched',\n",
       "  'Region': '15:86062708-86134699',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '167886812',\n",
       "  'End': '167973354',\n",
       "  'Name': '14-chr2-23423-lociStitched',\n",
       "  'Gene': '14-chr2-23423-lociStitched',\n",
       "  'Region': '2:167886812-167973354',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '27987647',\n",
       "  'End': '28062308',\n",
       "  'Name': '14-chr3-450-lociStitched',\n",
       "  'Gene': '14-chr3-450-lociStitched',\n",
       "  'Region': '3:27987647-28062308',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '111689090',\n",
       "  'End': '111777268',\n",
       "  'Name': '15-chr7-15564-lociStitched',\n",
       "  'Gene': '15-chr7-15564-lociStitched',\n",
       "  'Region': '7:111689090-111777268',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '55435977',\n",
       "  'End': '55499173',\n",
       "  'Name': '13-chr12-22409-lociStitched',\n",
       "  'Gene': '13-chr12-22409-lociStitched',\n",
       "  'Region': '12:55435977-55499173',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '31024671',\n",
       "  'End': '31103011',\n",
       "  'Name': '14-chr2-23217-lociStitched',\n",
       "  'Gene': '14-chr2-23217-lociStitched',\n",
       "  'Region': '2:31024671-31103011',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '67603478',\n",
       "  'End': '67658181',\n",
       "  'Name': '12-chr11-14010-lociStitched',\n",
       "  'Gene': '12-chr11-14010-lociStitched',\n",
       "  'Region': '11:67603478-67658181',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '95194411',\n",
       "  'End': '95282448',\n",
       "  'Name': '14-chr16-25870-lociStitched',\n",
       "  'Gene': '14-chr16-25870-lociStitched',\n",
       "  'Region': '16:95194411-95282448',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '34288790',\n",
       "  'End': '34305579',\n",
       "  'Name': 'chr17-1',\n",
       "  'Gene': 'chr17-1',\n",
       "  'Region': '17:34288790-34305579',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '59471275',\n",
       "  'End': '59526119',\n",
       "  'Name': '10-chr15-5429-lociStitched',\n",
       "  'Gene': '10-chr15-5429-lociStitched',\n",
       "  'Region': '15:59471275-59526119',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '31819903',\n",
       "  'End': '31874263',\n",
       "  'Name': '13-chr17-16574-lociStitched',\n",
       "  'Gene': '13-chr17-16574-lociStitched',\n",
       "  'Region': '17:31819903-31874263',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '49989719',\n",
       "  'End': '50031213',\n",
       "  'Name': '12-chr17-14525-lociStitched',\n",
       "  'Gene': '12-chr17-14525-lociStitched',\n",
       "  'Region': '17:49989719-50031213',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '90711053',\n",
       "  'End': '90783591',\n",
       "  'Name': '13-chr6-9380-lociStitched',\n",
       "  'Gene': '13-chr6-9380-lociStitched',\n",
       "  'Region': '6:90711053-90783591',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '61940239',\n",
       "  'End': '62013012',\n",
       "  'Name': '13-chr18-19508-lociStitched',\n",
       "  'Gene': '13-chr18-19508-lociStitched',\n",
       "  'Region': '18:61940239-62013012',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '43748376',\n",
       "  'End': '43790105',\n",
       "  'Name': '12-chr9-24618-lociStitched',\n",
       "  'Gene': '12-chr9-24618-lociStitched',\n",
       "  'Region': '9:43748376-43790105',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '117304480',\n",
       "  'End': '117344911',\n",
       "  'Name': '12-chr8-9480-lociStitched',\n",
       "  'Gene': '12-chr8-9480-lociStitched',\n",
       "  'Region': '8:117304480-117344911',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '135580643',\n",
       "  'End': '135641672',\n",
       "  'Name': '12-chr3-9253-lociStitched',\n",
       "  'Gene': '12-chr3-9253-lociStitched',\n",
       "  'Region': '3:135580643-135641672',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '7943314',\n",
       "  'End': '8000992',\n",
       "  'Name': '10-chr14-25472-lociStitched',\n",
       "  'Gene': '10-chr14-25472-lociStitched',\n",
       "  'Region': '14:7943314-8000992',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '120277538',\n",
       "  'End': '120334594',\n",
       "  'Name': '10-chr14-25596-lociStitched',\n",
       "  'Gene': '10-chr14-25596-lociStitched',\n",
       "  'Region': '14:120277538-120334594',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '67543680',\n",
       "  'End': '67581237',\n",
       "  'Name': '11-chr11-458-lociStitched',\n",
       "  'Gene': '11-chr11-458-lociStitched',\n",
       "  'Region': '11:67543680-67581237',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '12334476',\n",
       "  'End': '12391401',\n",
       "  'Name': '13-chr14-9704-lociStitched',\n",
       "  'Gene': '13-chr14-9704-lociStitched',\n",
       "  'Region': '14:12334476-12391401',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '23872035',\n",
       "  'End': '23932575',\n",
       "  'Name': '14-chr14-12796-lociStitched',\n",
       "  'Gene': '14-chr14-12796-lociStitched',\n",
       "  'Region': '14:23872035-23932575',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '53626723',\n",
       "  'End': '53673943',\n",
       "  'Name': '8-chr11-25008-lociStitched',\n",
       "  'Gene': '8-chr11-25008-lociStitched',\n",
       "  'Region': '11:53626723-53673943',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '121427339',\n",
       "  'End': '121474138',\n",
       "  'Name': '11-chr10-22208-lociStitched',\n",
       "  'Gene': '11-chr10-22208-lociStitched',\n",
       "  'Region': '10:121427339-121474138',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '45473668',\n",
       "  'End': '45524340',\n",
       "  'Name': '9-chr4-6530-lociStitched',\n",
       "  'Gene': '9-chr4-6530-lociStitched',\n",
       "  'Region': '4:45473668-45524340',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '57771804',\n",
       "  'End': '57817667',\n",
       "  'Name': '9-chr4-21424-lociStitched',\n",
       "  'Gene': '9-chr4-21424-lociStitched',\n",
       "  'Region': '4:57771804-57817667',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '52898995',\n",
       "  'End': '52953537',\n",
       "  'Name': '11-chr2-23252-lociStitched',\n",
       "  'Gene': '11-chr2-23252-lociStitched',\n",
       "  'Region': '2:52898995-52953537',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '70079812',\n",
       "  'End': '70123801',\n",
       "  'Name': '9-chr2-17022-lociStitched',\n",
       "  'Gene': '9-chr2-17022-lociStitched',\n",
       "  'Region': '2:70079812-70123801',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '157371808',\n",
       "  'End': '157419059',\n",
       "  'Name': '10-chr2-218-lociStitched',\n",
       "  'Gene': '10-chr2-218-lociStitched',\n",
       "  'Region': '2:157371808-157419059',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '57318323',\n",
       "  'End': '57366738',\n",
       "  'Name': '11-chr14-25531-lociStitched',\n",
       "  'Gene': '11-chr14-25531-lociStitched',\n",
       "  'Region': '14:57318323-57366738',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '86567081',\n",
       "  'End': '86618209',\n",
       "  'Name': '11-chr6-8550-lociStitched',\n",
       "  'Gene': '11-chr6-8550-lociStitched',\n",
       "  'Region': '6:86567081-86618209',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '31532835',\n",
       "  'End': '31576744',\n",
       "  'Name': '10-chr14-1234-lociStitched',\n",
       "  'Gene': '10-chr14-1234-lociStitched',\n",
       "  'Region': '14:31532835-31576744',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '120701024',\n",
       "  'End': '120754738',\n",
       "  'Name': '15-chr8-4113-lociStitched',\n",
       "  'Gene': '15-chr8-4113-lociStitched',\n",
       "  'Region': '8:120701024-120754738',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '99319934',\n",
       "  'End': '99369423',\n",
       "  'Name': '11-chr12-2380-lociStitched',\n",
       "  'Gene': '11-chr12-2380-lociStitched',\n",
       "  'Region': '12:99319934-99369423',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '60791099',\n",
       "  'End': '60840297',\n",
       "  'Name': '12-chr2-8319-lociStitched',\n",
       "  'Gene': '12-chr2-8319-lociStitched',\n",
       "  'Region': '2:60791099-60840297',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '96980046',\n",
       "  'End': '97035658',\n",
       "  'Name': '10-chr5-7148-lociStitched',\n",
       "  'Gene': '10-chr5-7148-lociStitched',\n",
       "  'Region': '5:96980046-97035658',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '11360811',\n",
       "  'End': '11404642',\n",
       "  'Name': '10-chr16-14422-lociStitched',\n",
       "  'Gene': '10-chr16-14422-lociStitched',\n",
       "  'Region': '16:11360811-11404642',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '165852652',\n",
       "  'End': '165887750',\n",
       "  'Name': '6-chr2-11986-lociStitched',\n",
       "  'Gene': '6-chr2-11986-lociStitched',\n",
       "  'Region': '2:165852652-165887750',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '54904225',\n",
       "  'End': '54941098',\n",
       "  'Name': '9-chr11-3900-lociStitched',\n",
       "  'Gene': '9-chr11-3900-lociStitched',\n",
       "  'Region': '11:54904225-54941098',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '152023297',\n",
       "  'End': '152065572',\n",
       "  'Name': '9-chr1-853-lociStitched',\n",
       "  'Gene': '9-chr1-853-lociStitched',\n",
       "  'Region': '1:152023297-152065572',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '100892913',\n",
       "  'End': '100936165',\n",
       "  'Name': '13-chr11-18648-lociStitched',\n",
       "  'Gene': '13-chr11-18648-lociStitched',\n",
       "  'Region': '11:100892913-100936165',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '20541576',\n",
       "  'End': '20589819',\n",
       "  'Name': '10-chr18-14559-lociStitched',\n",
       "  'Gene': '10-chr18-14559-lociStitched',\n",
       "  'Region': '18:20541576-20589819',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '49981863',\n",
       "  'End': '50039388',\n",
       "  'Name': '13-chr18-11736-lociStitched',\n",
       "  'Gene': '13-chr18-11736-lociStitched',\n",
       "  'Region': '18:49981863-50039388',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '11417687',\n",
       "  'End': '11478512',\n",
       "  'Name': '11-chr16-20780-lociStitched',\n",
       "  'Gene': '11-chr16-20780-lociStitched',\n",
       "  'Region': '16:11417687-11478512',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '54954781',\n",
       "  'End': '54981323',\n",
       "  'Name': '6-chr11-13993-lociStitched',\n",
       "  'Gene': '6-chr11-13993-lociStitched',\n",
       "  'Region': '11:54954781-54981323',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '122689495',\n",
       "  'End': '122737263',\n",
       "  'Name': '7-chr8-4115-lociStitched',\n",
       "  'Gene': '7-chr8-4115-lociStitched',\n",
       "  'Region': '8:122689495-122737263',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '113110236',\n",
       "  'End': '113154726',\n",
       "  'Name': '8-chr5-3452-lociStitched',\n",
       "  'Gene': '8-chr5-3452-lociStitched',\n",
       "  'Region': '5:113110236-113154726',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '29671165',\n",
       "  'End': '29719475',\n",
       "  'Name': '9-chr2-10840-lociStitched',\n",
       "  'Gene': '9-chr2-10840-lociStitched',\n",
       "  'Region': '2:29671165-29719475',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '66340017',\n",
       "  'End': '66385269',\n",
       "  'Name': '8-chr7-12325-lociStitched',\n",
       "  'Gene': '8-chr7-12325-lociStitched',\n",
       "  'Region': '7:66340017-66385269',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '94353273',\n",
       "  'End': '94392932',\n",
       "  'Name': '7-chr8-1278-lociStitched',\n",
       "  'Gene': '7-chr8-1278-lociStitched',\n",
       "  'Region': '8:94353273-94392932',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '73516455',\n",
       "  'End': '73552871',\n",
       "  'Name': '9-chr7-5265-lociStitched',\n",
       "  'Gene': '9-chr7-5265-lociStitched',\n",
       "  'Region': '7:73516455-73552871',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '29356203',\n",
       "  'End': '29405241',\n",
       "  'Name': '10-chr19-7541-lociStitched',\n",
       "  'Gene': '10-chr19-7541-lociStitched',\n",
       "  'Region': '19:29356203-29405241',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '94519831',\n",
       "  'End': '94570848',\n",
       "  'Name': '8-chr10-2220-lociStitched',\n",
       "  'Gene': '8-chr10-2220-lociStitched',\n",
       "  'Region': '10:94519831-94570848',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '72182232',\n",
       "  'End': '72208823',\n",
       "  'Name': '10-chr1-23037-lociStitched',\n",
       "  'Gene': '10-chr1-23037-lociStitched',\n",
       "  'Region': '1:72182232-72208823',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '99834365',\n",
       "  'End': '99868283',\n",
       "  'Name': '8-chr15-14407-lociStitched',\n",
       "  'Gene': '8-chr15-14407-lociStitched',\n",
       "  'Region': '15:99834365-99868283',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '123169914',\n",
       "  'End': '123191832',\n",
       "  'Name': '10-chr3-4019-lociStitched',\n",
       "  'Gene': '10-chr3-4019-lociStitched',\n",
       "  'Region': '3:123169914-123191832',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '102013725',\n",
       "  'End': '102066093',\n",
       "  'Name': '8-chr15-10639-lociStitched',\n",
       "  'Gene': '8-chr15-10639-lociStitched',\n",
       "  'Region': '15:102013725-102066093',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '111582530',\n",
       "  'End': '111626687',\n",
       "  'Name': '9-chr7-17939-lociStitched',\n",
       "  'Gene': '9-chr7-17939-lociStitched',\n",
       "  'Region': '7:111582530-111626687',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '33861864',\n",
       "  'End': '33904507',\n",
       "  'Name': '11-chr8-10294-lociStitched',\n",
       "  'Gene': '11-chr8-10294-lociStitched',\n",
       "  'Region': '8:33861864-33904507',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '168507674',\n",
       "  'End': '168558506',\n",
       "  'Name': '12-chr2-21275-lociStitched',\n",
       "  'Gene': '12-chr2-21275-lociStitched',\n",
       "  'Region': '2:168507674-168558506',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '133934465',\n",
       "  'End': '133993699',\n",
       "  'Name': '9-chr7-3856-lociStitched',\n",
       "  'Gene': '9-chr7-3856-lociStitched',\n",
       "  'Region': '7:133934465-133993699',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '49936424',\n",
       "  'End': '49971545',\n",
       "  'Name': '6-chr16-32-lociStitched',\n",
       "  'Gene': '6-chr16-32-lociStitched',\n",
       "  'Region': '16:49936424-49971545',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '35835536',\n",
       "  'End': '35876304',\n",
       "  'Name': '9-chr18-7526-lociStitched',\n",
       "  'Gene': '9-chr18-7526-lociStitched',\n",
       "  'Region': '18:35835536-35876304',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '86469811',\n",
       "  'End': '86505433',\n",
       "  'Name': '7-chr1-198-lociStitched',\n",
       "  'Gene': '7-chr1-198-lociStitched',\n",
       "  'Region': '1:86469811-86505433',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '92674254',\n",
       "  'End': '92710015',\n",
       "  'Name': '8-chr1-23055-lociStitched',\n",
       "  'Gene': '8-chr1-23055-lociStitched',\n",
       "  'Region': '1:92674254-92710015',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '72760886',\n",
       "  'End': '72828765',\n",
       "  'Name': '9-chr5-17543-lociStitched',\n",
       "  'Gene': '9-chr5-17543-lociStitched',\n",
       "  'Region': '5:72760886-72828765',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '33228692',\n",
       "  'End': '33266102',\n",
       "  'Name': '10-chr2-16995-lociStitched',\n",
       "  'Gene': '10-chr2-16995-lociStitched',\n",
       "  'Region': '2:33228692-33266102',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '7888104',\n",
       "  'End': '7929529',\n",
       "  'Name': '7-chr14-9697-lociStitched',\n",
       "  'Gene': '7-chr14-9697-lociStitched',\n",
       "  'Region': '14:7888104-7929529',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '126821810',\n",
       "  'End': '126852356',\n",
       "  'Name': '7-chr8-20297-lociStitched',\n",
       "  'Gene': '7-chr8-20297-lociStitched',\n",
       "  'Region': '8:126821810-126852356',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '50145615',\n",
       "  'End': '50187414',\n",
       "  'Name': '8-chr17-4791-lociStitched',\n",
       "  'Gene': '8-chr17-4791-lociStitched',\n",
       "  'Region': '17:50145615-50187414',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '96969013',\n",
       "  'End': '97012514',\n",
       "  'Name': '7-chr9-9523-lociStitched',\n",
       "  'Gene': '7-chr9-9523-lociStitched',\n",
       "  'Region': '9:96969013-97012514',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '63382513',\n",
       "  'End': '63423786',\n",
       "  'Name': '8-chr14-96-lociStitched',\n",
       "  'Gene': '8-chr14-96-lociStitched',\n",
       "  'Region': '14:63382513-63423786',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '97106465',\n",
       "  'End': '97146814',\n",
       "  'Name': '6-chr11-484-lociStitched',\n",
       "  'Gene': '6-chr11-484-lociStitched',\n",
       "  'Region': '11:97106465-97146814',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '58720625',\n",
       "  'End': '58752740',\n",
       "  'Name': '8-chr1-7565-lociStitched',\n",
       "  'Gene': '8-chr1-7565-lociStitched',\n",
       "  'Region': '1:58720625-58752740',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '5977038',\n",
       "  'End': '6017163',\n",
       "  'Name': '6-chr17-25888-lociStitched',\n",
       "  'Gene': '6-chr17-25888-lociStitched',\n",
       "  'Region': '17:5977038-6017163',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '37314861',\n",
       "  'End': '37344183',\n",
       "  'Name': '8-chr10-5331-lociStitched',\n",
       "  'Gene': '8-chr10-5331-lociStitched',\n",
       "  'Region': '10:37314861-37344183',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '54050908',\n",
       "  'End': '54092698',\n",
       "  'Name': '8-chr11-2359-lociStitched',\n",
       "  'Gene': '8-chr11-2359-lociStitched',\n",
       "  'Region': '11:54050908-54092698',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '157438062',\n",
       "  'End': '157484181',\n",
       "  'Name': '9-chr2-10008-lociStitched',\n",
       "  'Gene': '9-chr2-10008-lociStitched',\n",
       "  'Region': '2:157438062-157484181',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '134693544',\n",
       "  'End': '134733163',\n",
       "  'Name': '10-chr6-3467-lociStitched',\n",
       "  'Gene': '10-chr6-3467-lociStitched',\n",
       "  'Region': '6:134693544-134733163',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '125123914',\n",
       "  'End': '125159971',\n",
       "  'Name': '7-chr5-386-lociStitched',\n",
       "  'Gene': '7-chr5-386-lociStitched',\n",
       "  'Region': '5:125123914-125159971',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '75710453',\n",
       "  'End': '75743019',\n",
       "  'Name': '9-chr12-763-lociStitched',\n",
       "  'Gene': '9-chr12-763-lociStitched',\n",
       "  'Region': '12:75710453-75743019',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '122979978',\n",
       "  'End': '123023466',\n",
       "  'Name': '10-chr8-7886-lociStitched',\n",
       "  'Gene': '10-chr8-7886-lociStitched',\n",
       "  'Region': '8:122979978-123023466',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '80665569',\n",
       "  'End': '80698684',\n",
       "  'Name': '8-chr10-5001-lociStitched',\n",
       "  'Gene': '8-chr10-5001-lociStitched',\n",
       "  'Region': '10:80665569-80698684',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '140352596',\n",
       "  'End': '140382885',\n",
       "  'Name': '8-chr5-15341-lociStitched',\n",
       "  'Gene': '8-chr5-15341-lociStitched',\n",
       "  'Region': '5:140352596-140382885',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '113456558',\n",
       "  'End': '113522948',\n",
       "  'Name': '11-chr11-6804-lociStitched',\n",
       "  'Gene': '11-chr11-6804-lociStitched',\n",
       "  'Region': '11:113456558-113522948',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '119125956',\n",
       "  'End': '119161799',\n",
       "  'Name': '8-chr11-5380-lociStitched',\n",
       "  'Gene': '8-chr11-5380-lociStitched',\n",
       "  'Region': '11:119125956-119161799',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '160037490',\n",
       "  'End': '160066999',\n",
       "  'Name': '7-chr1-9150-lociStitched',\n",
       "  'Gene': '7-chr1-9150-lociStitched',\n",
       "  'Region': '1:160037490-160066999',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '27752695',\n",
       "  'End': '27800076',\n",
       "  'Name': '10-chr17-8179-lociStitched',\n",
       "  'Gene': '10-chr17-8179-lociStitched',\n",
       "  'Region': '17:27752695-27800076',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '44503635',\n",
       "  'End': '44547709',\n",
       "  'Name': '8-chr13-20594-lociStitched',\n",
       "  'Gene': '8-chr13-20594-lociStitched',\n",
       "  'Region': '13:44503635-44547709',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '58895418',\n",
       "  'End': '58933533',\n",
       "  'Name': '7-chr10-5783-lociStitched',\n",
       "  'Gene': '7-chr10-5783-lociStitched',\n",
       "  'Region': '10:58895418-58933533',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '167803760',\n",
       "  'End': '167850587',\n",
       "  'Name': '8-chr2-9219-lociStitched',\n",
       "  'Gene': '8-chr2-9219-lociStitched',\n",
       "  'Region': '2:167803760-167850587',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '117541353',\n",
       "  'End': '117587065',\n",
       "  'Name': '7-chr7-11206-lociStitched',\n",
       "  'Gene': '7-chr7-11206-lociStitched',\n",
       "  'Region': '7:117541353-117587065',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '78126402',\n",
       "  'End': '78142264',\n",
       "  'Name': '4-chr10-52-lociStitched',\n",
       "  'Gene': '4-chr10-52-lociStitched',\n",
       "  'Region': '10:78126402-78142264',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '109582271',\n",
       "  'End': '109615122',\n",
       "  'Name': '8-chr7-3274-lociStitched',\n",
       "  'Gene': '8-chr7-3274-lociStitched',\n",
       "  'Region': '7:109582271-109615122',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '107323355',\n",
       "  'End': '107363732',\n",
       "  'Name': '8-chr11-14077-lociStitched',\n",
       "  'Gene': '8-chr11-14077-lociStitched',\n",
       "  'Region': '11:107323355-107363732',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '85631788',\n",
       "  'End': '85668144',\n",
       "  'Name': '7-chr15-806-lociStitched',\n",
       "  'Gene': '7-chr15-806-lociStitched',\n",
       "  'Region': '15:85631788-85668144',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '78564657',\n",
       "  'End': '78602131',\n",
       "  'Name': '5-chr15-118-lociStitched',\n",
       "  'Gene': '5-chr15-118-lociStitched',\n",
       "  'Region': '15:78564657-78602131',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '12263762',\n",
       "  'End': '12301838',\n",
       "  'Name': '8-chr12-2377-lociStitched',\n",
       "  'Gene': '8-chr12-2377-lociStitched',\n",
       "  'Region': '12:12263762-12301838',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '91644104',\n",
       "  'End': '91685107',\n",
       "  'Name': '6-chr12-11492-lociStitched',\n",
       "  'Gene': '6-chr12-11492-lociStitched',\n",
       "  'Region': '12:91644104-91685107',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '131734470',\n",
       "  'End': '131765538',\n",
       "  'Name': '5-chr1-21042-lociStitched',\n",
       "  'Gene': '5-chr1-21042-lociStitched',\n",
       "  'Region': '1:131734470-131765538',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '134042582',\n",
       "  'End': '134065828',\n",
       "  'Name': '8-chr6-4641-lociStitched',\n",
       "  'Gene': '8-chr6-4641-lociStitched',\n",
       "  'Region': '6:134042582-134065828',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '75170508',\n",
       "  'End': '75193968',\n",
       "  'Name': '10-chr14-25565-lociStitched',\n",
       "  'Gene': '10-chr14-25565-lociStitched',\n",
       "  'Region': '14:75170508-75193968',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '115000867',\n",
       "  'End': '115040963',\n",
       "  'Name': '7-chr5-17605-lociStitched',\n",
       "  'Gene': '7-chr5-17605-lociStitched',\n",
       "  'Region': '5:115000867-115040963',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '161048632',\n",
       "  'End': '161089339',\n",
       "  'Name': '9-chr2-14947-lociStitched',\n",
       "  'Gene': '9-chr2-14947-lociStitched',\n",
       "  'Region': '2:161048632-161089339',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '145831096',\n",
       "  'End': '145868204',\n",
       "  'Name': '9-chr6-17801-lociStitched',\n",
       "  'Gene': '9-chr6-17801-lociStitched',\n",
       "  'Region': '6:145831096-145868204',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '78845142',\n",
       "  'End': '78887124',\n",
       "  'Name': '10-chr11-16021-lociStitched',\n",
       "  'Gene': '10-chr11-16021-lociStitched',\n",
       "  'Region': '11:78845142-78887124',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '37478663',\n",
       "  'End': '37524425',\n",
       "  'Name': '9-chr1-713-lociStitched',\n",
       "  'Gene': '9-chr1-713-lociStitched',\n",
       "  'Region': '1:37478663-37524425',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '102296541',\n",
       "  'End': '102337814',\n",
       "  'Name': '11-chr12-8049-lociStitched',\n",
       "  'Gene': '11-chr12-8049-lociStitched',\n",
       "  'Region': '12:102296541-102337814',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '5804996',\n",
       "  'End': '5848256',\n",
       "  'Name': '7-chr19-3572-lociStitched',\n",
       "  'Gene': '7-chr19-3572-lociStitched',\n",
       "  'Region': '19:5804996-5848256',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '89944358',\n",
       "  'End': '89990806',\n",
       "  'Name': '7-chr16-11680-lociStitched',\n",
       "  'Gene': '7-chr16-11680-lociStitched',\n",
       "  'Region': '16:89944358-89990806',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '146253987',\n",
       "  'End': '146286340',\n",
       "  'Name': '7-chr2-19803-lociStitched',\n",
       "  'Gene': '7-chr2-19803-lociStitched',\n",
       "  'Region': '2:146253987-146286340',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '86228132',\n",
       "  'End': '86256912',\n",
       "  'Name': '7-chr14-3341-lociStitched',\n",
       "  'Gene': '7-chr14-3341-lociStitched',\n",
       "  'Region': '14:86228132-86256912',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '120037144',\n",
       "  'End': '120075517',\n",
       "  'Name': '9-chr4-17411-lociStitched',\n",
       "  'Gene': '9-chr4-17411-lociStitched',\n",
       "  'Region': '4:120037144-120075517',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '39430905',\n",
       "  'End': '39453076',\n",
       "  'Name': '5-chr1-11812-lociStitched',\n",
       "  'Gene': '5-chr1-11812-lociStitched',\n",
       "  'Region': '1:39430905-39453076',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '111177379',\n",
       "  'End': '111211880',\n",
       "  'Name': '10-chr12-20567-lociStitched',\n",
       "  'Gene': '10-chr12-20567-lociStitched',\n",
       "  'Region': '12:111177379-111211880',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '114893822',\n",
       "  'End': '114937554',\n",
       "  'Name': '9-chr6-15435-lociStitched',\n",
       "  'Gene': '9-chr6-15435-lociStitched',\n",
       "  'Region': '6:114893822-114937554',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '125555084',\n",
       "  'End': '125582395',\n",
       "  'Name': '4-chr7-13702-lociStitched',\n",
       "  'Gene': '4-chr7-13702-lociStitched',\n",
       "  'Region': '7:125555084-125582395',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '97328334',\n",
       "  'End': '97357125',\n",
       "  'Name': '6-chr7-7835-lociStitched',\n",
       "  'Gene': '6-chr7-7835-lociStitched',\n",
       "  'Region': '7:97328334-97357125',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '123790631',\n",
       "  'End': '123822302',\n",
       "  'Name': '8-chr4-23729-lociStitched',\n",
       "  'Gene': '8-chr4-23729-lociStitched',\n",
       "  'Region': '4:123790631-123822302',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '128978110',\n",
       "  'End': '129017596',\n",
       "  'Name': '9-chr4-11002-lociStitched',\n",
       "  'Gene': '9-chr4-11002-lociStitched',\n",
       "  'Region': '4:128978110-129017596',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '134950641',\n",
       "  'End': '134982422',\n",
       "  'Name': '8-chr5-9352-lociStitched',\n",
       "  'Gene': '8-chr5-9352-lociStitched',\n",
       "  'Region': '5:134950641-134982422',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '8557098',\n",
       "  'End': '8585223',\n",
       "  'Name': '7-chr11-1396-lociStitched',\n",
       "  'Gene': '7-chr11-1396-lociStitched',\n",
       "  'Region': '11:8557098-8585223',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '171424880',\n",
       "  'End': '171467060',\n",
       "  'Name': '8-chr1-3219-lociStitched',\n",
       "  'Gene': '8-chr1-3219-lociStitched',\n",
       "  'Region': '1:171424880-171467060',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '91521576',\n",
       "  'End': '91563080',\n",
       "  'Name': '6-chr16-22765-lociStitched',\n",
       "  'Gene': '6-chr16-22765-lociStitched',\n",
       "  'Region': '16:91521576-91563080',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '66057060',\n",
       "  'End': '66086950',\n",
       "  'Name': '8-chr5-13478-lociStitched',\n",
       "  'Gene': '8-chr5-13478-lociStitched',\n",
       "  'Region': '5:66057060-66086950',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '3212865',\n",
       "  'End': '3253490',\n",
       "  'Name': '8-chr8-11222-lociStitched',\n",
       "  'Gene': '8-chr8-11222-lociStitched',\n",
       "  'Region': '8:3212865-3253490',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '4262302',\n",
       "  'End': '4308419',\n",
       "  'Name': '6-chr11-20426-lociStitched',\n",
       "  'Gene': '6-chr11-20426-lociStitched',\n",
       "  'Region': '11:4262302-4308419',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '92357018',\n",
       "  'End': '92396513',\n",
       "  'Name': '8-chr7-17910-lociStitched',\n",
       "  'Gene': '8-chr7-17910-lociStitched',\n",
       "  'Region': '7:92357018-92396513',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '67114399',\n",
       "  'End': '67152571',\n",
       "  'Name': '7-chr15-16415-lociStitched',\n",
       "  'Gene': '7-chr15-16415-lociStitched',\n",
       "  'Region': '15:67114399-67152571',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '80763652',\n",
       "  'End': '80789812',\n",
       "  'Name': '7-chr7-10252-lociStitched',\n",
       "  'Gene': '7-chr7-10252-lociStitched',\n",
       "  'Region': '7:80763652-80789812',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '35412468',\n",
       "  'End': '35449082',\n",
       "  'Name': '6-chr2-6027-lociStitched',\n",
       "  'Gene': '6-chr2-6027-lociStitched',\n",
       "  'Region': '2:35412468-35449082',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '122137663',\n",
       "  'End': '122165640',\n",
       "  'Name': '4-chr2-11952-lociStitched',\n",
       "  'Gene': '4-chr2-11952-lociStitched',\n",
       "  'Region': '2:122137663-122165640',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '79889566',\n",
       "  'End': '79916931',\n",
       "  'Name': '6-chr15-22664-lociStitched',\n",
       "  'Gene': '6-chr15-22664-lociStitched',\n",
       "  'Region': '15:79889566-79916931',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '114733925',\n",
       "  'End': '114762064',\n",
       "  'Name': '6-chr9-13867-lociStitched',\n",
       "  'Gene': '6-chr9-13867-lociStitched',\n",
       "  'Region': '9:114733925-114762064',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '191176621',\n",
       "  'End': '191225757',\n",
       "  'Name': '8-chr1-11885-lociStitched',\n",
       "  'Gene': '8-chr1-11885-lociStitched',\n",
       "  'Region': '1:191176621-191225757',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '19623151',\n",
       "  'End': '19641953',\n",
       "  'Name': '4-chr7-10220-lociStitched',\n",
       "  'Gene': '4-chr7-10220-lociStitched',\n",
       "  'Region': '7:19623151-19641953',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '43953219',\n",
       "  'End': '43978713',\n",
       "  'Name': '7-chr4-10073-lociStitched',\n",
       "  'Gene': '7-chr4-10073-lociStitched',\n",
       "  'Region': '4:43953219-43978713',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '97899500',\n",
       "  'End': '97954742',\n",
       "  'Name': '8-chr4-7109-lociStitched',\n",
       "  'Gene': '8-chr4-7109-lociStitched',\n",
       "  'Region': '4:97899500-97954742',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '64797651',\n",
       "  'End': '64847102',\n",
       "  'Name': '8-chr14-12813-lociStitched',\n",
       "  'Gene': '8-chr14-12813-lociStitched',\n",
       "  'Region': '14:64797651-64847102',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '30600074',\n",
       "  'End': '30626921',\n",
       "  'Name': '6-chr2-10843-lociStitched',\n",
       "  'Gene': '6-chr2-10843-lociStitched',\n",
       "  'Region': '2:30600074-30626921',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '61675519',\n",
       "  'End': '61710065',\n",
       "  'Name': '6-chr11-5813-lociStitched',\n",
       "  'Gene': '6-chr11-5813-lociStitched',\n",
       "  'Region': '11:61675519-61710065',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '53377774',\n",
       "  'End': '53414843',\n",
       "  'Name': '6-chr5-7140-lociStitched',\n",
       "  'Gene': '6-chr5-7140-lociStitched',\n",
       "  'Region': '5:53377774-53414843',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '79292713',\n",
       "  'End': '79325732',\n",
       "  'Name': '7-chr7-3480-lociStitched',\n",
       "  'Gene': '7-chr7-3480-lociStitched',\n",
       "  'Region': '7:79292713-79325732',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '165822296',\n",
       "  'End': '165857610',\n",
       "  'Name': '6-chr1-13115-lociStitched',\n",
       "  'Gene': '6-chr1-13115-lociStitched',\n",
       "  'Region': '1:165822296-165857610',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '110651579',\n",
       "  'End': '110684138',\n",
       "  'Name': '7-chr9-24743-lociStitched',\n",
       "  'Gene': '7-chr9-24743-lociStitched',\n",
       "  'Region': '9:110651579-110684138',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '152452546',\n",
       "  'End': '152490900',\n",
       "  'Name': '5-chr3-6524-lociStitched',\n",
       "  'Gene': '5-chr3-6524-lociStitched',\n",
       "  'Region': '3:152452546-152490900',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '78043061',\n",
       "  'End': '78047799',\n",
       "  'Name': 'chr10-2',\n",
       "  'Gene': 'chr10-2',\n",
       "  'Region': '10:78043061-78047799',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '36060548',\n",
       "  'End': '36101254',\n",
       "  'Name': '8-chr1-6435-lociStitched',\n",
       "  'Gene': '8-chr1-6435-lociStitched',\n",
       "  'Region': '1:36060548-36101254',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '108272009',\n",
       "  'End': '108309699',\n",
       "  'Name': '6-chr11-799-lociStitched',\n",
       "  'Gene': '6-chr11-799-lociStitched',\n",
       "  'Region': '11:108272009-108309699',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '136002351',\n",
       "  'End': '136027863',\n",
       "  'Name': '6-chr5-276-lociStitched',\n",
       "  'Gene': '6-chr5-276-lociStitched',\n",
       "  'Region': '5:136002351-136027863',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '61644382',\n",
       "  'End': '61662450',\n",
       "  'Name': '4-chr11-88-lociStitched',\n",
       "  'Gene': '4-chr11-88-lociStitched',\n",
       "  'Region': '11:61644382-61662450',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '25532393',\n",
       "  'End': '25553488',\n",
       "  'Name': '5-chr2-1308-lociStitched',\n",
       "  'Gene': '5-chr2-1308-lociStitched',\n",
       "  'Region': '2:25532393-25553488',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '105558336',\n",
       "  'End': '105585079',\n",
       "  'Name': '5-chr12-1778-lociStitched',\n",
       "  'Gene': '5-chr12-1778-lociStitched',\n",
       "  'Region': '12:105558336-105585079',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '107612041',\n",
       "  'End': '107646209',\n",
       "  'Name': '7-chr9-2503-lociStitched',\n",
       "  'Gene': '7-chr9-2503-lociStitched',\n",
       "  'Region': '9:107612041-107646209',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '134361413',\n",
       "  'End': '134377571',\n",
       "  'Name': '3-chr4-3617-lociStitched',\n",
       "  'Gene': '3-chr4-3617-lociStitched',\n",
       "  'Region': '4:134361413-134377571',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '39204439',\n",
       "  'End': '39241658',\n",
       "  'Name': '6-chr1-74-lociStitched',\n",
       "  'Gene': '6-chr1-74-lociStitched',\n",
       "  'Region': '1:39204439-39241658',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '44818538',\n",
       "  'End': '44839607',\n",
       "  'Name': '4-chr7-21780-lociStitched',\n",
       "  'Gene': '4-chr7-21780-lociStitched',\n",
       "  'Region': '7:44818538-44839607',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '130895750',\n",
       "  'End': '130922375',\n",
       "  'Name': '8-chr4-15189-lociStitched',\n",
       "  'Gene': '8-chr4-15189-lociStitched',\n",
       "  'Region': '4:130895750-130922375',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '70583615',\n",
       "  'End': '70603965',\n",
       "  'Name': '6-chr11-3313-lociStitched',\n",
       "  'Gene': '6-chr11-3313-lociStitched',\n",
       "  'Region': '11:70583615-70603965',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '27395577',\n",
       "  'End': '27420769',\n",
       "  'Name': '5-chr2-9954-lociStitched',\n",
       "  'Gene': '5-chr2-9954-lociStitched',\n",
       "  'Region': '2:27395577-27420769',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '6795253',\n",
       "  'End': '6829474',\n",
       "  'Name': '6-chr17-6910-lociStitched',\n",
       "  'Gene': '6-chr17-6910-lociStitched',\n",
       "  'Region': '17:6795253-6829474',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '119612655',\n",
       "  'End': '119648270',\n",
       "  'Name': '6-chr11-16102-lociStitched',\n",
       "  'Gene': '6-chr11-16102-lociStitched',\n",
       "  'Region': '11:119612655-119648270',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '119227359',\n",
       "  'End': '119249750',\n",
       "  'Name': '4-chr2-13194-lociStitched',\n",
       "  'Gene': '4-chr2-13194-lociStitched',\n",
       "  'Region': '2:119227359-119249750',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '60620124',\n",
       "  'End': '60649194',\n",
       "  'Name': '6-chr18-2685-lociStitched',\n",
       "  'Gene': '6-chr18-2685-lociStitched',\n",
       "  'Region': '18:60620124-60649194',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '12210868',\n",
       "  'End': '12243054',\n",
       "  'Name': '6-chr15-19102-lociStitched',\n",
       "  'Gene': '6-chr15-19102-lociStitched',\n",
       "  'Region': '15:12210868-12243054',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '159233181',\n",
       "  'End': '159269512',\n",
       "  'Name': '6-chr1-6452-lociStitched',\n",
       "  'Gene': '6-chr1-6452-lociStitched',\n",
       "  'Region': '1:159233181-159269512',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '113641468',\n",
       "  'End': '113671161',\n",
       "  'Name': '5-chr6-63-lociStitched',\n",
       "  'Gene': '5-chr6-63-lociStitched',\n",
       "  'Region': '6:113641468-113671161',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '47391106',\n",
       "  'End': '47417852',\n",
       "  'Name': '5-chr19-7547-lociStitched',\n",
       "  'Gene': '5-chr19-7547-lociStitched',\n",
       "  'Region': '19:47391106-47417852',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '73368424',\n",
       "  'End': '73389215',\n",
       "  'Name': '6-chr14-19055-lociStitched',\n",
       "  'Gene': '6-chr14-19055-lociStitched',\n",
       "  'Region': '14:73368424-73389215',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '94151189',\n",
       "  'End': '94194648',\n",
       "  'Name': '7-chr11-5024-lociStitched',\n",
       "  'Gene': '7-chr11-5024-lociStitched',\n",
       "  'Region': '11:94151189-94194648',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '40321974',\n",
       "  'End': '40358107',\n",
       "  'Name': '7-chr9-4391-lociStitched',\n",
       "  'Gene': '7-chr9-4391-lociStitched',\n",
       "  'Region': '9:40321974-40358107',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '21430901',\n",
       "  'End': '21464130',\n",
       "  'Name': '5-chr9-18171-lociStitched',\n",
       "  'Gene': '5-chr9-18171-lociStitched',\n",
       "  'Region': '9:21430901-21464130',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '66246312',\n",
       "  'End': '66281238',\n",
       "  'Name': '7-chr9-2500-lociStitched',\n",
       "  'Gene': '7-chr9-2500-lociStitched',\n",
       "  'Region': '9:66246312-66281238',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '134799931',\n",
       "  'End': '134818219',\n",
       "  'Name': '7-chr6-21733-lociStitched',\n",
       "  'Gene': '7-chr6-21733-lociStitched',\n",
       "  'Region': '6:134799931-134818219',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '16769402',\n",
       "  'End': '16805231',\n",
       "  'Name': '8-chr11-1765-lociStitched',\n",
       "  'Gene': '8-chr11-1765-lociStitched',\n",
       "  'Region': '11:16769402-16805231',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '31910812',\n",
       "  'End': '31941921',\n",
       "  'Name': '5-chr16-19253-lociStitched',\n",
       "  'Gene': '5-chr16-19253-lociStitched',\n",
       "  'Region': '16:31910812-31941921',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '49009235',\n",
       "  'End': '49029091',\n",
       "  'Name': '5-chr17-3959-lociStitched',\n",
       "  'Gene': '5-chr17-3959-lociStitched',\n",
       "  'Region': '17:49009235-49029091',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '94536153',\n",
       "  'End': '94571058',\n",
       "  'Name': '6-chr8-8652-lociStitched',\n",
       "  'Gene': '6-chr8-8652-lociStitched',\n",
       "  'Region': '8:94536153-94571058',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '139534550',\n",
       "  'End': '139562758',\n",
       "  'Name': '6-chr7-5731-lociStitched',\n",
       "  'Gene': '6-chr7-5731-lociStitched',\n",
       "  'Region': '7:139534550-139562758',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '31326167',\n",
       "  'End': '31361008',\n",
       "  'Name': '5-chr15-25633-lociStitched',\n",
       "  'Gene': '5-chr15-25633-lociStitched',\n",
       "  'Region': '15:31326167-31361008',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '145822127',\n",
       "  'End': '145848090',\n",
       "  'Name': '6-chr2-4854-lociStitched',\n",
       "  'Gene': '6-chr2-4854-lociStitched',\n",
       "  'Region': '2:145822127-145848090',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '23417203',\n",
       "  'End': '23440124',\n",
       "  'Name': '5-chr5-7736-lociStitched',\n",
       "  'Gene': '5-chr5-7736-lociStitched',\n",
       "  'Region': '5:23417203-23440124',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '184040747',\n",
       "  'End': '184067263',\n",
       "  'Name': '8-chr1-5537-lociStitched',\n",
       "  'Gene': '8-chr1-5537-lociStitched',\n",
       "  'Region': '1:184040747-184067263',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '105035500',\n",
       "  'End': '105059340',\n",
       "  'Name': '6-chr3-10048-lociStitched',\n",
       "  'Gene': '6-chr3-10048-lociStitched',\n",
       "  'Region': '3:105035500-105059340',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '63437319',\n",
       "  'End': '63454951',\n",
       "  'Name': '7-chr14-5417-lociStitched',\n",
       "  'Gene': '7-chr14-5417-lociStitched',\n",
       "  'Region': '14:63437319-63454951',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '137965746',\n",
       "  'End': '137978090',\n",
       "  'Name': '5-chr3-10954-lociStitched',\n",
       "  'Gene': '5-chr3-10954-lociStitched',\n",
       "  'Region': '3:137965746-137978090',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '52017644',\n",
       "  'End': '52046966',\n",
       "  'Name': '7-chr3-15009-lociStitched',\n",
       "  'Gene': '7-chr3-15009-lociStitched',\n",
       "  'Region': '3:52017644-52046966',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '99239398',\n",
       "  'End': '99270359',\n",
       "  'Name': '5-chr12-3321-lociStitched',\n",
       "  'Gene': '5-chr12-3321-lociStitched',\n",
       "  'Region': '12:99239398-99270359',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '114771155',\n",
       "  'End': '114792263',\n",
       "  'Name': '6-chr6-24136-lociStitched',\n",
       "  'Gene': '6-chr6-24136-lociStitched',\n",
       "  'Region': '6:114771155-114792263',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '29618862',\n",
       "  'End': '29654078',\n",
       "  'Name': '6-chr13-548-lociStitched',\n",
       "  'Gene': '6-chr13-548-lociStitched',\n",
       "  'Region': '13:29618862-29654078',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '11037879',\n",
       "  'End': '11072858',\n",
       "  'Name': '5-chr8-15619-lociStitched',\n",
       "  'Gene': '5-chr8-15619-lociStitched',\n",
       "  'Region': '8:11037879-11072858',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '32650306',\n",
       "  'End': '32680523',\n",
       "  'Name': '5-chr11-2222-lociStitched',\n",
       "  'Gene': '5-chr11-2222-lociStitched',\n",
       "  'Region': '11:32650306-32680523',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '108021949',\n",
       "  'End': '108041386',\n",
       "  'Name': '6-chr9-7303-lociStitched',\n",
       "  'Gene': '6-chr9-7303-lociStitched',\n",
       "  'Region': '9:108021949-108041386',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '100277741',\n",
       "  'End': '100306313',\n",
       "  'Name': '8-chr11-7372-lociStitched',\n",
       "  'Gene': '8-chr11-7372-lociStitched',\n",
       "  'Region': '11:100277741-100306313',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '76063038',\n",
       "  'End': '76090164',\n",
       "  'Name': '6-chr18-2055-lociStitched',\n",
       "  'Gene': '6-chr18-2055-lociStitched',\n",
       "  'Region': '18:76063038-76090164',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '73596715',\n",
       "  'End': '73625602',\n",
       "  'Name': '6-chr7-4091-lociStitched',\n",
       "  'Gene': '6-chr7-4091-lociStitched',\n",
       "  'Region': '7:73596715-73625602',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '25626059',\n",
       "  'End': '25653928',\n",
       "  'Name': '5-chr15-10603-lociStitched',\n",
       "  'Gene': '5-chr15-10603-lociStitched',\n",
       "  'Region': '15:25626059-25653928',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '87801701',\n",
       "  'End': '87829800',\n",
       "  'Name': '5-chr11-5822-lociStitched',\n",
       "  'Gene': '5-chr11-5822-lociStitched',\n",
       "  'Region': '11:87801701-87829800',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '28566800',\n",
       "  'End': '28591508',\n",
       "  'Name': '5-chr7-12304-lociStitched',\n",
       "  'Gene': '5-chr7-12304-lociStitched',\n",
       "  'Region': '7:28566800-28591508',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '100045607',\n",
       "  'End': '100067950',\n",
       "  'Name': '5-chr15-6369-lociStitched',\n",
       "  'Gene': '5-chr15-6369-lociStitched',\n",
       "  'Region': '15:100045607-100067950',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '118448017',\n",
       "  'End': '118479047',\n",
       "  'Name': '6-chr5-9342-lociStitched',\n",
       "  'Gene': '6-chr5-9342-lociStitched',\n",
       "  'Region': '5:118448017-118479047',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '31075812',\n",
       "  'End': '31093514',\n",
       "  'Name': '6-chr6-7167-lociStitched',\n",
       "  'Gene': '6-chr6-7167-lociStitched',\n",
       "  'Region': '6:31075812-31093514',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '86434911',\n",
       "  'End': '86453635',\n",
       "  'Name': '6-chr1-1010-lociStitched',\n",
       "  'Gene': '6-chr1-1010-lociStitched',\n",
       "  'Region': '1:86434911-86453635',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '45403164',\n",
       "  'End': '45431634',\n",
       "  'Name': '6-chr4-9268-lociStitched',\n",
       "  'Gene': '6-chr4-9268-lociStitched',\n",
       "  'Region': '4:45403164-45431634',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '20635472',\n",
       "  'End': '20669430',\n",
       "  'Name': '7-chr11-2952-lociStitched',\n",
       "  'Gene': '7-chr11-2952-lociStitched',\n",
       "  'Region': '11:20635472-20669430',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '35592447',\n",
       "  'End': '35630969',\n",
       "  'Name': '9-chr8-1925-lociStitched',\n",
       "  'Gene': '9-chr8-1925-lociStitched',\n",
       "  'Region': '8:35592447-35630969',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '28517659',\n",
       "  'End': '28540705',\n",
       "  'Name': '4-chr2-240-lociStitched',\n",
       "  'Gene': '4-chr2-240-lociStitched',\n",
       "  'Region': '2:28517659-28540705',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '16030645',\n",
       "  'End': '16053879',\n",
       "  'Name': '6-chr7-17814-lociStitched',\n",
       "  'Gene': '6-chr7-17814-lociStitched',\n",
       "  'Region': '7:16030645-16053879',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '56327599',\n",
       "  'End': '56363775',\n",
       "  'Name': '6-chr17-4517-lociStitched',\n",
       "  'Gene': '6-chr17-4517-lociStitched',\n",
       "  'Region': '17:56327599-56363775',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '62272882',\n",
       "  'End': '62301729',\n",
       "  'Name': '7-chr14-16336-lociStitched',\n",
       "  'Gene': '7-chr14-16336-lociStitched',\n",
       "  'Region': '14:62272882-62301729',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '71842848',\n",
       "  'End': '71871286',\n",
       "  'Name': '5-chr12-14145-lociStitched',\n",
       "  'Gene': '5-chr12-14145-lociStitched',\n",
       "  'Region': '12:71842848-71871286',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '58098461',\n",
       "  'End': '58114667',\n",
       "  'Name': '3-chr9-24652-lociStitched',\n",
       "  'Gene': '3-chr9-24652-lociStitched',\n",
       "  'Region': '9:58098461-58114667',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '36554814',\n",
       "  'End': '36573690',\n",
       "  'Name': '4-chr1-8254-lociStitched',\n",
       "  'Gene': '4-chr1-8254-lociStitched',\n",
       "  'Region': '1:36554814-36573690',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '83133854',\n",
       "  'End': '83167292',\n",
       "  'Name': '7-chr15-14391-lociStitched',\n",
       "  'Gene': '7-chr15-14391-lociStitched',\n",
       "  'Region': '15:83133854-83167292',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '26686655',\n",
       "  'End': '26710258',\n",
       "  'Name': '5-chr17-849-lociStitched',\n",
       "  'Gene': '5-chr17-849-lociStitched',\n",
       "  'Region': '17:26686655-26710258',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '116427073',\n",
       "  'End': '116443709',\n",
       "  'Name': '5-chr11-12673-lociStitched',\n",
       "  'Gene': '5-chr11-12673-lociStitched',\n",
       "  'Region': '11:116427073-116443709',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '79367685',\n",
       "  'End': '79391643',\n",
       "  'Name': '4-chr15-4767-lociStitched',\n",
       "  'Gene': '4-chr15-4767-lociStitched',\n",
       "  'Region': '15:79367685-79391643',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '121539286',\n",
       "  'End': '121558117',\n",
       "  'Name': '7-chr11-10491-lociStitched',\n",
       "  'Gene': '7-chr11-10491-lociStitched',\n",
       "  'Region': '11:121539286-121558117',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '115516155',\n",
       "  'End': '115545083',\n",
       "  'Name': '4-chr5-8510-lociStitched',\n",
       "  'Gene': '4-chr5-8510-lociStitched',\n",
       "  'Region': '5:115516155-115545083',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '132940703',\n",
       "  'End': '132967897',\n",
       "  'Name': '5-chr4-13415-lociStitched',\n",
       "  'Gene': '5-chr4-13415-lociStitched',\n",
       "  'Region': '4:132940703-132967897',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '122542172',\n",
       "  'End': '122575620',\n",
       "  'Name': '10-chr3-4298-lociStitched',\n",
       "  'Gene': '10-chr3-4298-lociStitched',\n",
       "  'Region': '3:122542172-122575620',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '142143262',\n",
       "  'End': '142172088',\n",
       "  'Name': '6-chr4-12131-lociStitched',\n",
       "  'Gene': '6-chr4-12131-lociStitched',\n",
       "  'Region': '4:142143262-142172088',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '8609192',\n",
       "  'End': '8640117',\n",
       "  'Name': '10-chr11-10420-lociStitched',\n",
       "  'Gene': '10-chr11-10420-lociStitched',\n",
       "  'Region': '11:8609192-8640117',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '147049421',\n",
       "  'End': '147073968',\n",
       "  'Name': '6-chr5-2304-lociStitched',\n",
       "  'Gene': '6-chr5-2304-lociStitched',\n",
       "  'Region': '5:147049421-147073968',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '94241075',\n",
       "  'End': '94279075',\n",
       "  'Name': '5-chr6-15420-lociStitched',\n",
       "  'Gene': '5-chr6-15420-lociStitched',\n",
       "  'Region': '6:94241075-94279075',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '55352401',\n",
       "  'End': '55367025',\n",
       "  'Name': '5-chr12-4170-lociStitched',\n",
       "  'Gene': '5-chr12-4170-lociStitched',\n",
       "  'Region': '12:55352401-55367025',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '67032520',\n",
       "  'End': '67066601',\n",
       "  'Name': '9-chr10-24837-lociStitched',\n",
       "  'Gene': '9-chr10-24837-lociStitched',\n",
       "  'Region': '10:67032520-67066601',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '142337431',\n",
       "  'End': '142349730',\n",
       "  'Name': '2-chr6-21739-lociStitched',\n",
       "  'Gene': '2-chr6-21739-lociStitched',\n",
       "  'Region': '6:142337431-142349730',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '180383095',\n",
       "  'End': '180409035',\n",
       "  'Name': '7-chr1-9157-lociStitched',\n",
       "  'Gene': '7-chr1-9157-lociStitched',\n",
       "  'Region': '1:180383095-180409035',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '32685653',\n",
       "  'End': '32696318',\n",
       "  'Name': '3-chr3-493-lociStitched',\n",
       "  'Gene': '3-chr3-493-lociStitched',\n",
       "  'Region': '3:32685653-32696318',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '11371566',\n",
       "  'End': '11401869',\n",
       "  'Name': '6-chr8-18011-lociStitched',\n",
       "  'Gene': '6-chr8-18011-lociStitched',\n",
       "  'Region': '8:11371566-11401869',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '172275569',\n",
       "  'End': '172304115',\n",
       "  'Name': '8-chr2-2169-lociStitched',\n",
       "  'Gene': '8-chr2-2169-lociStitched',\n",
       "  'Region': '2:172275569-172304115',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '46465981',\n",
       "  'End': '46486328',\n",
       "  'Name': '5-chr19-20948-lociStitched',\n",
       "  'Gene': '5-chr19-20948-lociStitched',\n",
       "  'Region': '19:46465981-46486328',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '87507828',\n",
       "  'End': '87530192',\n",
       "  'Name': '4-chr3-397-lociStitched',\n",
       "  'Gene': '4-chr3-397-lociStitched',\n",
       "  'Region': '3:87507828-87530192',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '79264107',\n",
       "  'End': '79279527',\n",
       "  'Name': '4-chr7-340-lociStitched',\n",
       "  'Gene': '4-chr7-340-lociStitched',\n",
       "  'Region': '7:79264107-79279527',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '103445952',\n",
       "  'End': '103464837',\n",
       "  'Name': '6-chr5-1819-lociStitched',\n",
       "  'Gene': '6-chr5-1819-lociStitched',\n",
       "  'Region': '5:103445952-103464837',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '125446995',\n",
       "  'End': '125478372',\n",
       "  'Name': '4-chr6-20132-lociStitched',\n",
       "  'Gene': '4-chr6-20132-lociStitched',\n",
       "  'Region': '6:125446995-125478372',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '191065202',\n",
       "  'End': '191094188',\n",
       "  'Name': '4-chr1-16925-lociStitched',\n",
       "  'Gene': '4-chr1-16925-lociStitched',\n",
       "  'Region': '1:191065202-191094188',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '102506859',\n",
       "  'End': '102528693',\n",
       "  'Name': '5-chr4-17388-lociStitched',\n",
       "  'Gene': '5-chr4-17388-lociStitched',\n",
       "  'Region': '4:102506859-102528693',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '52805699',\n",
       "  'End': '52841971',\n",
       "  'Name': '7-chr13-2237-lociStitched',\n",
       "  'Gene': '7-chr13-2237-lociStitched',\n",
       "  'Region': '13:52805699-52841971',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '82478760',\n",
       "  'End': '82496995',\n",
       "  'Name': '7-chr18-580-lociStitched',\n",
       "  'Gene': '7-chr18-580-lociStitched',\n",
       "  'Region': '18:82478760-82496995',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '43519886',\n",
       "  'End': '43546390',\n",
       "  'Name': '6-chr1-9117-lociStitched',\n",
       "  'Gene': '6-chr1-9117-lociStitched',\n",
       "  'Region': '1:43519886-43546390',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '77816554',\n",
       "  'End': '77840181',\n",
       "  'Name': '4-chr15-14382-lociStitched',\n",
       "  'Gene': '4-chr15-14382-lociStitched',\n",
       "  'Region': '15:77816554-77840181',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '94429600',\n",
       "  'End': '94459314',\n",
       "  'Name': '5-chr6-1825-lociStitched',\n",
       "  'Gene': '5-chr6-1825-lociStitched',\n",
       "  'Region': '6:94429600-94459314',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '170116697',\n",
       "  'End': '170140014',\n",
       "  'Name': '7-chr2-332-lociStitched',\n",
       "  'Gene': '7-chr2-332-lociStitched',\n",
       "  'Region': '2:170116697-170140014',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '60855223',\n",
       "  'End': '60878176',\n",
       "  'Name': '6-chr2-11921-lociStitched',\n",
       "  'Gene': '6-chr2-11921-lociStitched',\n",
       "  'Region': '2:60855223-60878176',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '68122625',\n",
       "  'End': '68145738',\n",
       "  'Name': '4-chr10-389-lociStitched',\n",
       "  'Gene': '4-chr10-389-lociStitched',\n",
       "  'Region': '10:68122625-68145738',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '135260438',\n",
       "  'End': '135292900',\n",
       "  'Name': '6-chr4-15200-lociStitched',\n",
       "  'Gene': '6-chr4-15200-lociStitched',\n",
       "  'Region': '4:135260438-135292900',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '70237801',\n",
       "  'End': '70276917',\n",
       "  'Name': '7-chr9-7910-lociStitched',\n",
       "  'Gene': '7-chr9-7910-lociStitched',\n",
       "  'Region': '9:70237801-70276917',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '134342342',\n",
       "  'End': '134377414',\n",
       "  'Name': '5-chr6-13626-lociStitched',\n",
       "  'Gene': '5-chr6-13626-lociStitched',\n",
       "  'Region': '6:134342342-134377414',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '98796591',\n",
       "  'End': '98817385',\n",
       "  'Name': '7-chr7-20193-lociStitched',\n",
       "  'Gene': '7-chr7-20193-lociStitched',\n",
       "  'Region': '7:98796591-98817385',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '117996968',\n",
       "  'End': '118017005',\n",
       "  'Name': '4-chr11-2373-lociStitched',\n",
       "  'Gene': '4-chr11-2373-lociStitched',\n",
       "  'Region': '11:117996968-118017005',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '31115751',\n",
       "  'End': '31140683',\n",
       "  'Name': '7-chr6-20085-lociStitched',\n",
       "  'Gene': '7-chr6-20085-lociStitched',\n",
       "  'Region': '6:31115751-31140683',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '13345931',\n",
       "  'End': '13369802',\n",
       "  'Name': '4-chr8-13729-lociStitched',\n",
       "  'Gene': '4-chr8-13729-lociStitched',\n",
       "  'Region': '8:13345931-13369802',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '121050599',\n",
       "  'End': '121067991',\n",
       "  'Name': '4-chr6-11137-lociStitched',\n",
       "  'Gene': '4-chr6-11137-lociStitched',\n",
       "  'Region': '6:121050599-121067991',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '105790839',\n",
       "  'End': '105820620',\n",
       "  'Name': '8-chr1-10787-lociStitched',\n",
       "  'Gene': '8-chr1-10787-lociStitched',\n",
       "  'Region': '1:105790839-105820620',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '27625551',\n",
       "  'End': '27650644',\n",
       "  'Name': '5-chr15-2997-lociStitched',\n",
       "  'Gene': '5-chr15-2997-lociStitched',\n",
       "  'Region': '15:27625551-27650644',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '15992448',\n",
       "  'End': '16015116',\n",
       "  'Name': '7-chr7-8579-lociStitched',\n",
       "  'Gene': '7-chr7-8579-lociStitched',\n",
       "  'Region': '7:15992448-16015116',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '105245167',\n",
       "  'End': '105266792',\n",
       "  'Name': '5-chr4-19941-lociStitched',\n",
       "  'Gene': '5-chr4-19941-lociStitched',\n",
       "  'Region': '4:105245167-105266792',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '75304452',\n",
       "  'End': '75330990',\n",
       "  'Name': '5-chr10-1529-lociStitched',\n",
       "  'Gene': '5-chr10-1529-lociStitched',\n",
       "  'Region': '10:75304452-75330990',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '75917459',\n",
       "  'End': '75937700',\n",
       "  'Name': '4-chr11-1855-lociStitched',\n",
       "  'Gene': '4-chr11-1855-lociStitched',\n",
       "  'Region': '11:75917459-75937700',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '84865320',\n",
       "  'End': '84891515',\n",
       "  'Name': '7-chr12-14160-lociStitched',\n",
       "  'Gene': '7-chr12-14160-lociStitched',\n",
       "  'Region': '12:84865320-84891515',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '29622146',\n",
       "  'End': '29638586',\n",
       "  'Name': '6-chr2-13153-lociStitched',\n",
       "  'Gene': '6-chr2-13153-lociStitched',\n",
       "  'Region': '2:29622146-29638586',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '85117282',\n",
       "  'End': '85142981',\n",
       "  'Name': '7-chr16-2829-lociStitched',\n",
       "  'Gene': '7-chr16-2829-lociStitched',\n",
       "  'Region': '16:85117282-85142981',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '12542753',\n",
       "  'End': '12574015',\n",
       "  'Name': '6-chr13-14190-lociStitched',\n",
       "  'Gene': '6-chr13-14190-lociStitched',\n",
       "  'Region': '13:12542753-12574015',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '116942930',\n",
       "  'End': '116968663',\n",
       "  'Name': '4-chr13-22531-lociStitched',\n",
       "  'Gene': '4-chr13-22531-lociStitched',\n",
       "  'Region': '13:116942930-116968663',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '116042921',\n",
       "  'End': '116071830',\n",
       "  'Name': '6-chr2-9197-lociStitched',\n",
       "  'Gene': '6-chr2-9197-lociStitched',\n",
       "  'Region': '2:116042921-116071830',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '68675706',\n",
       "  'End': '68694563',\n",
       "  'Name': '3-chr3-17206-lociStitched',\n",
       "  'Gene': '3-chr3-17206-lociStitched',\n",
       "  'Region': '3:68675706-68694563',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '142212126',\n",
       "  'End': '142234912',\n",
       "  'Name': '4-chr4-15215-lociStitched',\n",
       "  'Gene': '4-chr4-15215-lociStitched',\n",
       "  'Region': '4:142212126-142234912',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '53763032',\n",
       "  'End': '53782415',\n",
       "  'Name': '3-chr11-22255-lociStitched',\n",
       "  'Gene': '3-chr11-22255-lociStitched',\n",
       "  'Region': '11:53763032-53782415',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '31116203',\n",
       "  'End': '31129539',\n",
       "  'Name': '4-chr9-2108-lociStitched',\n",
       "  'Gene': '4-chr9-2108-lociStitched',\n",
       "  'Region': '9:31116203-31129539',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '14714507',\n",
       "  'End': '14731116',\n",
       "  'Name': '6-chr9-1672-lociStitched',\n",
       "  'Gene': '6-chr9-1672-lociStitched',\n",
       "  'Region': '9:14714507-14731116',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '19314646',\n",
       "  'End': '19327478',\n",
       "  'Name': '5-chr7-5257-lociStitched',\n",
       "  'Gene': '5-chr7-5257-lociStitched',\n",
       "  'Region': '7:19314646-19327478',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '90247166',\n",
       "  'End': '90267743',\n",
       "  'Name': '5-chr9-10352-lociStitched',\n",
       "  'Gene': '5-chr9-10352-lociStitched',\n",
       "  'Region': '9:90247166-90267743',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '114949961',\n",
       "  'End': '114980111',\n",
       "  'Name': '4-chr5-17603-lociStitched',\n",
       "  'Gene': '4-chr5-17603-lociStitched',\n",
       "  'Region': '5:114949961-114980111',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '21713636',\n",
       "  'End': '21730629',\n",
       "  'Name': '3-chr9-15729-lociStitched',\n",
       "  'Gene': '3-chr9-15729-lociStitched',\n",
       "  'Region': '9:21713636-21730629',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '79089397',\n",
       "  'End': '79120907',\n",
       "  'Name': '6-chr15-11617-lociStitched',\n",
       "  'Gene': '6-chr15-11617-lociStitched',\n",
       "  'Region': '15:79089397-79120907',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '86616192',\n",
       "  'End': '86632326',\n",
       "  'Name': '6-chr9-7916-lociStitched',\n",
       "  'Gene': '6-chr9-7916-lociStitched',\n",
       "  'Region': '9:86616192-86632326',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '118520320',\n",
       "  'End': '118545785',\n",
       "  'Name': '6-chr2-13192-lociStitched',\n",
       "  'Gene': '6-chr2-13192-lociStitched',\n",
       "  'Region': '2:118520320-118545785',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '97779517',\n",
       "  'End': '97802042',\n",
       "  'Name': '7-chr4-13377-lociStitched',\n",
       "  'Gene': '7-chr4-13377-lociStitched',\n",
       "  'Region': '4:97779517-97802042',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '99214647',\n",
       "  'End': '99235580',\n",
       "  'Name': '5-chr5-688-lociStitched',\n",
       "  'Gene': '5-chr5-688-lociStitched',\n",
       "  'Region': '5:99214647-99235580',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '150599571',\n",
       "  'End': '150621043',\n",
       "  'Name': '3-chr5-77-lociStitched',\n",
       "  'Gene': '3-chr5-77-lociStitched',\n",
       "  'Region': '5:150599571-150621043',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '62158009',\n",
       "  'End': '62189951',\n",
       "  'Name': '4-chr18-19509-lociStitched',\n",
       "  'Gene': '4-chr18-19509-lociStitched',\n",
       "  'Region': '18:62158009-62189951',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '101965479',\n",
       "  'End': '101998729',\n",
       "  'Name': '7-chr15-2825-lociStitched',\n",
       "  'Gene': '7-chr15-2825-lociStitched',\n",
       "  'Region': '15:101965479-101998729',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '63105663',\n",
       "  'End': '63133889',\n",
       "  'Name': '5-chr1-23034-lociStitched',\n",
       "  'Gene': '5-chr1-23034-lociStitched',\n",
       "  'Region': '1:63105663-63133889',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '11576274',\n",
       "  'End': '11605724',\n",
       "  'Name': '7-chr16-1958-lociStitched',\n",
       "  'Gene': '7-chr16-1958-lociStitched',\n",
       "  'Region': '16:11576274-11605724',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '36946182',\n",
       "  'End': '36966383',\n",
       "  'Name': '5-chr14-19004-lociStitched',\n",
       "  'Gene': '5-chr14-19004-lociStitched',\n",
       "  'Region': '14:36946182-36966383',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '59884223',\n",
       "  'End': '59902586',\n",
       "  'Name': '5-chr11-14002-lociStitched',\n",
       "  'Gene': '5-chr11-14002-lociStitched',\n",
       "  'Region': '11:59884223-59902586',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '61159459',\n",
       "  'End': '61178388',\n",
       "  'Name': '5-chr10-1077-lociStitched',\n",
       "  'Gene': '5-chr10-1077-lociStitched',\n",
       "  'Region': '10:61159459-61178388',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '26751697',\n",
       "  'End': '26771541',\n",
       "  'Name': '7-chr19-4807-lociStitched',\n",
       "  'Gene': '7-chr19-4807-lociStitched',\n",
       "  'Region': '19:26751697-26771541',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '84187128',\n",
       "  'End': '84202964',\n",
       "  'Name': '5-chr8-9465-lociStitched',\n",
       "  'Gene': '5-chr8-9465-lociStitched',\n",
       "  'Region': '8:84187128-84202964',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '142956332',\n",
       "  'End': '142975618',\n",
       "  'Name': '3-chr5-149-lociStitched',\n",
       "  'Gene': '3-chr5-149-lociStitched',\n",
       "  'Region': '5:142956332-142975618',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '93249200',\n",
       "  'End': '93275092',\n",
       "  'Name': '5-chr10-7946-lociStitched',\n",
       "  'Gene': '5-chr10-7946-lociStitched',\n",
       "  'Region': '10:93249200-93275092',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '95802963',\n",
       "  'End': '95833520',\n",
       "  'Name': '6-chr15-22686-lociStitched',\n",
       "  'Gene': '6-chr15-22686-lociStitched',\n",
       "  'Region': '15:95802963-95833520',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '128785011',\n",
       "  'End': '128803666',\n",
       "  'Name': '5-chr4-9291-lociStitched',\n",
       "  'Gene': '5-chr4-9291-lociStitched',\n",
       "  'Region': '4:128785011-128803666',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '12873926',\n",
       "  'End': '12906094',\n",
       "  'Name': '4-chr8-6199-lociStitched',\n",
       "  'Gene': '4-chr8-6199-lociStitched',\n",
       "  'Region': '8:12873926-12906094',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '43712965',\n",
       "  'End': '43728719',\n",
       "  'Name': '4-chr9-697-lociStitched',\n",
       "  'Gene': '4-chr9-697-lociStitched',\n",
       "  'Region': '9:43712965-43728719',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '116988079',\n",
       "  'End': '117032962',\n",
       "  'Name': '6-chr13-25470-lociStitched',\n",
       "  'Gene': '6-chr13-25470-lociStitched',\n",
       "  'Region': '13:116988079-117032962',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '46112011',\n",
       "  'End': '46131236',\n",
       "  'Name': '5-chr7-20173-lociStitched',\n",
       "  'Gene': '5-chr7-20173-lociStitched',\n",
       "  'Region': '7:46112011-46131236',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '77783795',\n",
       "  'End': '77806045',\n",
       "  'Name': '6-chr11-1400-lociStitched',\n",
       "  'Gene': '6-chr11-1400-lociStitched',\n",
       "  'Region': '11:77783795-77806045',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '29833130',\n",
       "  'End': '29850904',\n",
       "  'Name': '5-chr6-7166-lociStitched',\n",
       "  'Gene': '5-chr6-7166-lociStitched',\n",
       "  'Region': '6:29833130-29850904',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '139924082',\n",
       "  'End': '139938790',\n",
       "  'Name': '3-chr4-7722-lociStitched',\n",
       "  'Gene': '3-chr4-7722-lociStitched',\n",
       "  'Region': '4:139924082-139938790',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '40921006',\n",
       "  'End': '40934221',\n",
       "  'Name': '6-chr14-16318-lociStitched',\n",
       "  'Gene': '6-chr14-16318-lociStitched',\n",
       "  'Region': '14:40921006-40934221',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '51732900',\n",
       "  'End': '51756618',\n",
       "  'Name': '5-chr13-18884-lociStitched',\n",
       "  'Gene': '5-chr13-18884-lociStitched',\n",
       "  'Region': '13:51732900-51756618',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '91452771',\n",
       "  'End': '91478002',\n",
       "  'Name': '7-chr2-17043-lociStitched',\n",
       "  'Gene': '7-chr2-17043-lociStitched',\n",
       "  'Region': '2:91452771-91478002',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '76466513',\n",
       "  'End': '76506297',\n",
       "  'Name': '8-chr11-25078-lociStitched',\n",
       "  'Gene': '8-chr11-25078-lociStitched',\n",
       "  'Region': '11:76466513-76506297',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '122575987',\n",
       "  'End': '122600279',\n",
       "  'Name': '4-chr6-15444-lociStitched',\n",
       "  'Gene': '4-chr6-15444-lociStitched',\n",
       "  'Region': '6:122575987-122600279',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '29756151',\n",
       "  'End': '29781365',\n",
       "  'Name': '4-chr13-8864-lociStitched',\n",
       "  'Gene': '4-chr13-8864-lociStitched',\n",
       "  'Region': '13:29756151-29781365',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '75422499',\n",
       "  'End': '75456900',\n",
       "  'Name': '7-chr18-16666-lociStitched',\n",
       "  'Gene': '7-chr18-16666-lociStitched',\n",
       "  'Region': '18:75422499-75456900',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '166696773',\n",
       "  'End': '166711170',\n",
       "  'Name': '5-chr2-23413-lociStitched',\n",
       "  'Gene': '5-chr2-23413-lociStitched',\n",
       "  'Region': '2:166696773-166711170',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '121740908',\n",
       "  'End': '121777232',\n",
       "  'Name': '5-chr11-22378-lociStitched',\n",
       "  'Gene': '5-chr11-22378-lociStitched',\n",
       "  'Region': '11:121740908-121777232',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '111545743',\n",
       "  'End': '111571233',\n",
       "  'Name': '5-chr8-10314-lociStitched',\n",
       "  'Gene': '5-chr8-10314-lociStitched',\n",
       "  'Region': '8:111545743-111571233',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '29459515',\n",
       "  'End': '29478629',\n",
       "  'Name': '3-chr17-4509-lociStitched',\n",
       "  'Gene': '3-chr17-4509-lociStitched',\n",
       "  'Region': '17:29459515-29478629',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '33325600',\n",
       "  'End': '33349018',\n",
       "  'Name': '5-chr2-2440-lociStitched',\n",
       "  'Gene': '5-chr2-2440-lociStitched',\n",
       "  'Region': '2:33325600-33349018',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '95667391',\n",
       "  'End': '95676194',\n",
       "  'Name': '3-chr3-28-lociStitched',\n",
       "  'Gene': '3-chr3-28-lociStitched',\n",
       "  'Region': '3:95667391-95676194',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '84215521',\n",
       "  'End': '84255609',\n",
       "  'Name': '5-chr7-20185-lociStitched',\n",
       "  'Gene': '5-chr7-20185-lociStitched',\n",
       "  'Region': '7:84215521-84255609',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '152782209',\n",
       "  'End': '152817365',\n",
       "  'Name': '6-chr2-23394-lociStitched',\n",
       "  'Gene': '6-chr2-23394-lociStitched',\n",
       "  'Region': '2:152782209-152817365',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '117361386',\n",
       "  'End': '117382493',\n",
       "  'Name': '7-chr8-8659-lociStitched',\n",
       "  'Gene': '7-chr8-8659-lociStitched',\n",
       "  'Region': '8:117361386-117382493',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '112264708',\n",
       "  'End': '112301471',\n",
       "  'Name': '7-chr5-7153-lociStitched',\n",
       "  'Gene': '7-chr5-7153-lociStitched',\n",
       "  'Region': '5:112264708-112301471',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '39922774',\n",
       "  'End': '39947938',\n",
       "  'Name': '4-chr1-14674-lociStitched',\n",
       "  'Gene': '4-chr1-14674-lociStitched',\n",
       "  'Region': '1:39922774-39947938',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '119383277',\n",
       "  'End': '119415124',\n",
       "  'Name': '4-chr6-8565-lociStitched',\n",
       "  'Gene': '4-chr6-8565-lociStitched',\n",
       "  'Region': '6:119383277-119415124',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '43548246',\n",
       "  'End': '43572814',\n",
       "  'Name': '6-chr9-24615-lociStitched',\n",
       "  'Gene': '6-chr9-24615-lociStitched',\n",
       "  'Region': '9:43548246-43572814',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '48840291',\n",
       "  'End': '48858405',\n",
       "  'Name': '5-chr13-11523-lociStitched',\n",
       "  'Gene': '5-chr13-11523-lociStitched',\n",
       "  'Region': '13:48840291-48858405',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '75343846',\n",
       "  'End': '75357521',\n",
       "  'Name': '4-chr1-35-lociStitched',\n",
       "  'Gene': '4-chr1-35-lociStitched',\n",
       "  'Region': '1:75343846-75357521',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '30686032',\n",
       "  'End': '30714800',\n",
       "  'Name': '5-chr13-1474-lociStitched',\n",
       "  'Gene': '5-chr13-1474-lociStitched',\n",
       "  'Region': '13:30686032-30714800',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '37332954',\n",
       "  'End': '37350780',\n",
       "  'Name': '5-chr19-1883-lociStitched',\n",
       "  'Gene': '5-chr19-1883-lociStitched',\n",
       "  'Region': '19:37332954-37350780',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '62729690',\n",
       "  'End': '62747142',\n",
       "  'Name': '5-chr4-6534-lociStitched',\n",
       "  'Gene': '5-chr4-6534-lociStitched',\n",
       "  'Region': '4:62729690-62747142',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '65138591',\n",
       "  'End': '65152409',\n",
       "  'Name': '4-chr11-985-lociStitched',\n",
       "  'Gene': '4-chr11-985-lociStitched',\n",
       "  'Region': '11:65138591-65152409',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '7811290',\n",
       "  'End': '7838884',\n",
       "  'Name': '6-chr10-22105-lociStitched',\n",
       "  'Gene': '6-chr10-22105-lociStitched',\n",
       "  'Region': '10:7811290-7838884',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '86635105',\n",
       "  'End': '86654458',\n",
       "  'Name': '5-chr6-15407-lociStitched',\n",
       "  'Gene': '5-chr6-15407-lociStitched',\n",
       "  'Region': '6:86635105-86654458',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '95402399',\n",
       "  'End': '95424332',\n",
       "  'Name': '4-chr11-11434-lociStitched',\n",
       "  'Gene': '4-chr11-11434-lociStitched',\n",
       "  'Region': '11:95402399-95424332',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '144456535',\n",
       "  'End': '144470870',\n",
       "  'Name': '5-chr3-21382-lociStitched',\n",
       "  'Gene': '5-chr3-21382-lociStitched',\n",
       "  'Region': '3:144456535-144470870',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '135925625',\n",
       "  'End': '135944168',\n",
       "  'Name': '4-chr1-4826-lociStitched',\n",
       "  'Gene': '4-chr1-4826-lociStitched',\n",
       "  'Region': '1:135925625-135944168',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '3738556',\n",
       "  'End': '3756358',\n",
       "  'Name': '6-chr2-23171-lociStitched',\n",
       "  'Gene': '6-chr2-23171-lociStitched',\n",
       "  'Region': '2:3738556-3756358',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '24417433',\n",
       "  'End': '24444390',\n",
       "  'Name': '6-chr19-7540-lociStitched',\n",
       "  'Gene': '6-chr19-7540-lociStitched',\n",
       "  'Region': '19:24417433-24444390',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '82835669',\n",
       "  'End': '82862295',\n",
       "  'Name': '5-chr11-1767-lociStitched',\n",
       "  'Gene': '5-chr11-1767-lociStitched',\n",
       "  'Region': '11:82835669-82862295',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '125477522',\n",
       "  'End': '125502660',\n",
       "  'Name': '5-chr7-17961-lociStitched',\n",
       "  'Gene': '5-chr7-17961-lociStitched',\n",
       "  'Region': '7:125477522-125502660',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '7846961',\n",
       "  'End': '7875171',\n",
       "  'Name': '6-chr14-22535-lociStitched',\n",
       "  'Gene': '6-chr14-22535-lociStitched',\n",
       "  'Region': '14:7846961-7875171',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '75494633',\n",
       "  'End': '75517811',\n",
       "  'Name': '5-chr18-19523-lociStitched',\n",
       "  'Gene': '5-chr18-19523-lociStitched',\n",
       "  'Region': '18:75494633-75517811',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '87174581',\n",
       "  'End': '87198025',\n",
       "  'Name': '6-chr12-18787-lociStitched',\n",
       "  'Gene': '6-chr12-18787-lociStitched',\n",
       "  'Region': '12:87174581-87198025',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '11632920',\n",
       "  'End': '11650858',\n",
       "  'Name': '5-chr14-12791-lociStitched',\n",
       "  'Gene': '5-chr14-12791-lociStitched',\n",
       "  'Region': '14:11632920-11650858',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '122137152',\n",
       "  'End': '122164848',\n",
       "  'Name': '4-chr8-13786-lociStitched',\n",
       "  'Gene': '4-chr8-13786-lociStitched',\n",
       "  'Region': '8:122137152-122164848',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '28929819',\n",
       "  'End': '28950655',\n",
       "  'Name': '3-chr13-95-lociStitched',\n",
       "  'Gene': '3-chr13-95-lociStitched',\n",
       "  'Region': '13:28929819-28950655',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '118671791',\n",
       "  'End': '118692744',\n",
       "  'Name': '3-chr14-8941-lociStitched',\n",
       "  'Gene': '3-chr14-8941-lociStitched',\n",
       "  'Region': '14:118671791-118692744',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '108336441',\n",
       "  'End': '108351930',\n",
       "  'Name': '3-chr13-25-lociStitched',\n",
       "  'Gene': '3-chr13-25-lociStitched',\n",
       "  'Region': '13:108336441-108351930',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '32391290',\n",
       "  'End': '32413924',\n",
       "  'Name': '4-chr18-9829-lociStitched',\n",
       "  'Gene': '4-chr18-9829-lociStitched',\n",
       "  'Region': '18:32391290-32413924',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '120364261',\n",
       "  'End': '120383898',\n",
       "  'Name': '4-chr8-20289-lociStitched',\n",
       "  'Gene': '4-chr8-20289-lociStitched',\n",
       "  'Region': '8:120364261-120383898',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '91662764',\n",
       "  'End': '91678098',\n",
       "  'Name': '4-chr2-431-lociStitched',\n",
       "  'Gene': '4-chr2-431-lociStitched',\n",
       "  'Region': '2:91662764-91678098',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '115941689',\n",
       "  'End': '115959284',\n",
       "  'Name': '4-chr6-2314-lociStitched',\n",
       "  'Gene': '4-chr6-2314-lociStitched',\n",
       "  'Region': '6:115941689-115959284',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '60301040',\n",
       "  'End': '60322687',\n",
       "  'Name': '5-chr11-20455-lociStitched',\n",
       "  'Gene': '5-chr11-20455-lociStitched',\n",
       "  'Region': '11:60301040-60322687',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '57725204',\n",
       "  'End': '57736455',\n",
       "  'Name': '5-chr4-9275-lociStitched',\n",
       "  'Gene': '5-chr4-9275-lociStitched',\n",
       "  'Region': '4:57725204-57736455',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '88507561',\n",
       "  'End': '88528569',\n",
       "  'Name': '5-chr11-22310-lociStitched',\n",
       "  'Gene': '5-chr11-22310-lociStitched',\n",
       "  'Region': '11:88507561-88528569',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '44513915',\n",
       "  'End': '44541052',\n",
       "  'Name': '5-chr11-24985-lociStitched',\n",
       "  'Gene': '5-chr11-24985-lociStitched',\n",
       "  'Region': '11:44513915-44541052',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '124617497',\n",
       "  'End': '124642915',\n",
       "  'Name': '4-chr2-3048-lociStitched',\n",
       "  'Gene': '4-chr2-3048-lociStitched',\n",
       "  'Region': '2:124617497-124642915',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '103032805',\n",
       "  'End': '103053640',\n",
       "  'Name': '3-chr9-13855-lociStitched',\n",
       "  'Gene': '3-chr9-13855-lociStitched',\n",
       "  'Region': '9:103032805-103053640',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '121477927',\n",
       "  'End': '121489551',\n",
       "  'Name': '3-chr11-547-lociStitched',\n",
       "  'Gene': '3-chr11-547-lociStitched',\n",
       "  'Region': '11:121477927-121489551',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '150445591',\n",
       "  'End': '150474902',\n",
       "  'Name': '5-chr4-15223-lociStitched',\n",
       "  'Gene': '5-chr4-15223-lociStitched',\n",
       "  'Region': '4:150445591-150474902',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '133201896',\n",
       "  'End': '133218599',\n",
       "  'Name': '7-chr1-23077-lociStitched',\n",
       "  'Gene': '7-chr1-23077-lociStitched',\n",
       "  'Region': '1:133201896-133218599',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '94577774',\n",
       "  'End': '94596499',\n",
       "  'Name': '5-chr16-9018-lociStitched',\n",
       "  'Gene': '5-chr16-9018-lociStitched',\n",
       "  'Region': '16:94577774-94596499',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '58480016',\n",
       "  'End': '58536120',\n",
       "  'Name': '8-chr4-17351-lociStitched',\n",
       "  'Gene': '8-chr4-17351-lociStitched',\n",
       "  'Region': '4:58480016-58536120',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '70108647',\n",
       "  'End': '70121563',\n",
       "  'Name': '3-chr10-1163-lociStitched',\n",
       "  'Gene': '3-chr10-1163-lociStitched',\n",
       "  'Region': '10:70108647-70121563',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '56797831',\n",
       "  'End': '56815533',\n",
       "  'Name': '3-chr17-173-lociStitched',\n",
       "  'Gene': '3-chr17-173-lociStitched',\n",
       "  'Region': '17:56797831-56815533',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '84090148',\n",
       "  'End': '84111025',\n",
       "  'Name': '5-chr9-8699-lociStitched',\n",
       "  'Gene': '5-chr9-8699-lociStitched',\n",
       "  'Region': '9:84090148-84111025',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '97878596',\n",
       "  'End': '97898071',\n",
       "  'Name': '5-chr15-19202-lociStitched',\n",
       "  'Gene': '5-chr15-19202-lociStitched',\n",
       "  'Region': '15:97878596-97898071',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '24632558',\n",
       "  'End': '24645521',\n",
       "  'Name': '4-chr2-331-lociStitched',\n",
       "  'Gene': '4-chr2-331-lociStitched',\n",
       "  'Region': '2:24632558-24645521',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '155047070',\n",
       "  'End': '155064749',\n",
       "  'Name': '4-chr1-1636-lociStitched',\n",
       "  'Gene': '4-chr1-1636-lociStitched',\n",
       "  'Region': '1:155047070-155064749',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '108667845',\n",
       "  'End': '108695703',\n",
       "  'Name': '4-chr12-25322-lociStitched',\n",
       "  'Gene': '4-chr12-25322-lociStitched',\n",
       "  'Region': '12:108667845-108695703',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '55598659',\n",
       "  'End': '55628514',\n",
       "  'Name': '4-chr19-19612-lociStitched',\n",
       "  'Gene': '4-chr19-19612-lociStitched',\n",
       "  'Region': '19:55598659-55628514',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '127906198',\n",
       "  'End': '127927720',\n",
       "  'Name': '4-chr7-11214-lociStitched',\n",
       "  'Gene': '4-chr7-11214-lociStitched',\n",
       "  'Region': '7:127906198-127927720',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '145882290',\n",
       "  'End': '145905907',\n",
       "  'Name': '5-chr6-7797-lociStitched',\n",
       "  'Gene': '5-chr6-7797-lociStitched',\n",
       "  'Region': '6:145882290-145905907',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '88509115',\n",
       "  'End': '88523152',\n",
       "  'Name': '4-chr3-106-lociStitched',\n",
       "  'Gene': '4-chr3-106-lociStitched',\n",
       "  'Region': '3:88509115-88523152',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '150048553',\n",
       "  'End': '150066435',\n",
       "  'Name': '6-chr4-11027-lociStitched',\n",
       "  'Gene': '6-chr4-11027-lociStitched',\n",
       "  'Region': '4:150048553-150066435',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '61019921',\n",
       "  'End': '61033124',\n",
       "  'Name': '3-chr18-104-lociStitched',\n",
       "  'Gene': '3-chr18-104-lociStitched',\n",
       "  'Region': '18:61019921-61033124',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '63242882',\n",
       "  'End': '63263215',\n",
       "  'Name': '5-chr13-5059-lociStitched',\n",
       "  'Gene': '5-chr13-5059-lociStitched',\n",
       "  'Region': '13:63242882-63263215',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '113973820',\n",
       "  'End': '113990793',\n",
       "  'Name': '6-chr5-10149-lociStitched',\n",
       "  'Gene': '6-chr5-10149-lociStitched',\n",
       "  'Region': '5:113973820-113990793',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '145819305',\n",
       "  'End': '145843016',\n",
       "  'Name': '5-chr3-9257-lociStitched',\n",
       "  'Gene': '5-chr3-9257-lociStitched',\n",
       "  'Region': '3:145819305-145843016',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '126238333',\n",
       "  'End': '126251449',\n",
       "  'Name': '2-chr4-325-lociStitched',\n",
       "  'Gene': '2-chr4-325-lociStitched',\n",
       "  'Region': '4:126238333-126251449',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '103077996',\n",
       "  'End': '103104632',\n",
       "  'Name': '4-chr14-25589-lociStitched',\n",
       "  'Gene': '4-chr14-25589-lociStitched',\n",
       "  'Region': '14:103077996-103104632',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '140198782',\n",
       "  'End': '140219596',\n",
       "  'Name': '5-chr5-17657-lociStitched',\n",
       "  'Gene': '5-chr5-17657-lociStitched',\n",
       "  'Region': '5:140198782-140219596',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '134833306',\n",
       "  'End': '134848229',\n",
       "  'Name': '6-chr6-288-lociStitched',\n",
       "  'Gene': '6-chr6-288-lociStitched',\n",
       "  'Region': '6:134833306-134848229',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '139903678',\n",
       "  'End': '139923142',\n",
       "  'Name': '4-chr7-24392-lociStitched',\n",
       "  'Gene': '4-chr7-24392-lociStitched',\n",
       "  'Region': '7:139903678-139923142',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '60876796',\n",
       "  'End': '60897811',\n",
       "  'Name': '4-chr19-26197-lociStitched',\n",
       "  'Gene': '4-chr19-26197-lociStitched',\n",
       "  'Region': '19:60876796-60897811',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '80649126',\n",
       "  'End': '80675061',\n",
       "  'Name': '4-chr1-19666-lociStitched',\n",
       "  'Gene': '4-chr1-19666-lociStitched',\n",
       "  'Region': '1:80649126-80675061',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '57944959',\n",
       "  'End': '57984571',\n",
       "  'Name': '6-chr11-22262-lociStitched',\n",
       "  'Gene': '6-chr11-22262-lociStitched',\n",
       "  'Region': '11:57944959-57984571',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '89462659',\n",
       "  'End': '89475111',\n",
       "  'Name': '4-chr1-10783-lociStitched',\n",
       "  'Gene': '4-chr1-10783-lociStitched',\n",
       "  'Region': '1:89462659-89475111',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '88133639',\n",
       "  'End': '88151043',\n",
       "  'Name': '4-chr3-10929-lociStitched',\n",
       "  'Gene': '4-chr3-10929-lociStitched',\n",
       "  'Region': '3:88133639-88151043',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '116071703',\n",
       "  'End': '116091105',\n",
       "  'Name': '3-chr11-3147-lociStitched',\n",
       "  'Gene': '3-chr11-3147-lociStitched',\n",
       "  'Region': '11:116071703-116091105',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '31071099',\n",
       "  'End': '31090736',\n",
       "  'Name': '8-chr17-9031-lociStitched',\n",
       "  'Gene': '8-chr17-9031-lociStitched',\n",
       "  'Region': '17:31071099-31090736',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '121492367',\n",
       "  'End': '121516482',\n",
       "  'Name': '6-chr10-6254-lociStitched',\n",
       "  'Gene': '6-chr10-6254-lociStitched',\n",
       "  'Region': '10:121492367-121516482',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '135931837',\n",
       "  'End': '135962382',\n",
       "  'Name': '5-chr5-211-lociStitched',\n",
       "  'Gene': '5-chr5-211-lociStitched',\n",
       "  'Region': '5:135931837-135962382',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '101834153',\n",
       "  'End': '101853699',\n",
       "  'Name': '4-chr14-16365-lociStitched',\n",
       "  'Gene': '4-chr14-16365-lociStitched',\n",
       "  'Region': '14:101834153-101853699',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '48954583',\n",
       "  'End': '48979337',\n",
       "  'Name': '4-chr13-6321-lociStitched',\n",
       "  'Gene': '4-chr13-6321-lociStitched',\n",
       "  'Region': '13:48954583-48979337',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '40894850',\n",
       "  'End': '40905790',\n",
       "  'Name': '3-chr14-202-lociStitched',\n",
       "  'Gene': '3-chr14-202-lociStitched',\n",
       "  'Region': '14:40894850-40905790',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '177641720',\n",
       "  'End': '177652368',\n",
       "  'Name': '2-chr1-17-lociStitched',\n",
       "  'Gene': '2-chr1-17-lociStitched',\n",
       "  'Region': '1:177641720-177652368',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '126111205',\n",
       "  'End': '126129515',\n",
       "  'Name': '6-chr7-24355-lociStitched',\n",
       "  'Gene': '6-chr7-24355-lociStitched',\n",
       "  'Region': '7:126111205-126129515',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '149394482',\n",
       "  'End': '149408262',\n",
       "  'Name': '4-chr4-2092-lociStitched',\n",
       "  'Gene': '4-chr4-2092-lociStitched',\n",
       "  'Region': '4:149394482-149408262',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '68044482',\n",
       "  'End': '68073698',\n",
       "  'Name': '8-chr7-24264-lociStitched',\n",
       "  'Gene': '8-chr7-24264-lociStitched',\n",
       "  'Region': '7:68044482-68073698',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '131825039',\n",
       "  'End': '131843985',\n",
       "  'Name': '3-chr2-23364-lociStitched',\n",
       "  'Gene': '3-chr2-23364-lociStitched',\n",
       "  'Region': '2:131825039-131843985',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '61553186',\n",
       "  'End': '61574571',\n",
       "  'Name': '4-chr8-20261-lociStitched',\n",
       "  'Gene': '4-chr8-20261-lociStitched',\n",
       "  'Region': '8:61553186-61574571',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '166808941',\n",
       "  'End': '166827430',\n",
       "  'Name': '6-chr2-10898-lociStitched',\n",
       "  'Gene': '6-chr2-10898-lociStitched',\n",
       "  'Region': '2:166808941-166827430',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '43210124',\n",
       "  'End': '43228034',\n",
       "  'Name': '4-chr13-1406-lociStitched',\n",
       "  'Gene': '4-chr13-1406-lociStitched',\n",
       "  'Region': '13:43210124-43228034',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '132365787',\n",
       "  'End': '132389067',\n",
       "  'Name': '4-chr1-21048-lociStitched',\n",
       "  'Gene': '4-chr1-21048-lociStitched',\n",
       "  'Region': '1:132365787-132389067',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '136675780',\n",
       "  'End': '136707594',\n",
       "  'Name': '6-chr3-23586-lociStitched',\n",
       "  'Gene': '6-chr3-23586-lociStitched',\n",
       "  'Region': '3:136675780-136707594',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '112328861',\n",
       "  'End': '112349868',\n",
       "  'Name': '5-chr12-18817-lociStitched',\n",
       "  'Gene': '5-chr12-18817-lociStitched',\n",
       "  'Region': '12:112328861-112349868',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '43990207',\n",
       "  'End': '44003635',\n",
       "  'Name': '4-chr10-1221-lociStitched',\n",
       "  'Gene': '4-chr10-1221-lociStitched',\n",
       "  'Region': '10:43990207-44003635',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '106606179',\n",
       "  'End': '106620337',\n",
       "  'Name': '4-chr8-1840-lociStitched',\n",
       "  'Gene': '4-chr8-1840-lociStitched',\n",
       "  'Region': '8:106606179-106620337',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '51533583',\n",
       "  'End': '51553857',\n",
       "  'Name': '3-chr6-15390-lociStitched',\n",
       "  'Gene': '3-chr6-15390-lociStitched',\n",
       "  'Region': '6:51533583-51553857',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '69488357',\n",
       "  'End': '69516904',\n",
       "  'Name': '4-chr12-2033-lociStitched',\n",
       "  'Gene': '4-chr12-2033-lociStitched',\n",
       "  'Region': '12:69488357-69516904',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '8639928',\n",
       "  'End': '8664425',\n",
       "  'Name': '5-chr15-11593-lociStitched',\n",
       "  'Gene': '5-chr15-11593-lociStitched',\n",
       "  'Region': '15:8639928-8664425',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '66183580',\n",
       "  'End': '66198911',\n",
       "  'Name': '5-chr9-3669-lociStitched',\n",
       "  'Gene': '5-chr9-3669-lociStitched',\n",
       "  'Region': '9:66183580-66198911',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '114711245',\n",
       "  'End': '114737856',\n",
       "  'Name': '5-chr7-12356-lociStitched',\n",
       "  'Gene': '5-chr7-12356-lociStitched',\n",
       "  'Region': '7:114711245-114737856',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '91604749',\n",
       "  'End': '91619221',\n",
       "  'Name': '5-chr5-9328-lociStitched',\n",
       "  'Gene': '5-chr5-9328-lociStitched',\n",
       "  'Region': '5:91604749-91619221',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '90408627',\n",
       "  'End': '90425384',\n",
       "  'Name': '5-chr16-7495-lociStitched',\n",
       "  'Gene': '5-chr16-7495-lociStitched',\n",
       "  'Region': '16:90408627-90425384',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '73152472',\n",
       "  'End': '73168325',\n",
       "  'Name': '5-chr15-6358-lociStitched',\n",
       "  'Gene': '5-chr15-6358-lociStitched',\n",
       "  'Region': '15:73152472-73168325',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '152105298',\n",
       "  'End': '152123239',\n",
       "  'Name': '4-chr3-9258-lociStitched',\n",
       "  'Gene': '4-chr3-9258-lociStitched',\n",
       "  'Region': '3:152105298-152123239',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '31265487',\n",
       "  'End': '31280871',\n",
       "  'Name': '4-chr9-7274-lociStitched',\n",
       "  'Gene': '4-chr9-7274-lociStitched',\n",
       "  'Region': '9:31265487-31280871',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '36444684',\n",
       "  'End': '36464787',\n",
       "  'Name': '3-chr13-16212-lociStitched',\n",
       "  'Gene': '3-chr13-16212-lociStitched',\n",
       "  'Region': '13:36444684-36464787',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '51159303',\n",
       "  'End': '51183939',\n",
       "  'Name': '6-chr9-24634-lociStitched',\n",
       "  'Gene': '6-chr9-24634-lociStitched',\n",
       "  'Region': '9:51159303-51183939',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '86310747',\n",
       "  'End': '86321839',\n",
       "  'Name': '3-chr10-8754-lociStitched',\n",
       "  'Gene': '3-chr10-8754-lociStitched',\n",
       "  'Region': '10:86310747-86321839',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '182461110',\n",
       "  'End': '182474965',\n",
       "  'Name': '3-chr1-9158-lociStitched',\n",
       "  'Gene': '3-chr1-9158-lociStitched',\n",
       "  'Region': '1:182461110-182474965',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '39502206',\n",
       "  'End': '39520407',\n",
       "  'Name': '4-chr1-16785-lociStitched',\n",
       "  'Gene': '4-chr1-16785-lociStitched',\n",
       "  'Region': '1:39502206-39520407',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '79158512',\n",
       "  'End': '79188069',\n",
       "  'Name': '6-chr3-17212-lociStitched',\n",
       "  'Gene': '6-chr3-17212-lociStitched',\n",
       "  'Region': '3:79158512-79188069',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '131284726',\n",
       "  'End': '131301746',\n",
       "  'Name': '3-chr5-2302-lociStitched',\n",
       "  'Gene': '3-chr5-2302-lociStitched',\n",
       "  'Region': '5:131284726-131301746',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '74810223',\n",
       "  'End': '74822523',\n",
       "  'Name': '3-chr14-1411-lociStitched',\n",
       "  'Gene': '3-chr14-1411-lociStitched',\n",
       "  'Region': '14:74810223-74822523',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '106754857',\n",
       "  'End': '106778515',\n",
       "  'Name': '3-chr11-16072-lociStitched',\n",
       "  'Gene': '3-chr11-16072-lociStitched',\n",
       "  'Region': '11:106754857-106778515',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '37870703',\n",
       "  'End': '37885543',\n",
       "  'Name': '5-chr18-6414-lociStitched',\n",
       "  'Gene': '5-chr18-6414-lociStitched',\n",
       "  'Region': '18:37870703-37885543',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '22131036',\n",
       "  'End': '22153740',\n",
       "  'Name': '6-chr16-11654-lociStitched',\n",
       "  'Gene': '6-chr16-11654-lociStitched',\n",
       "  'Region': '16:22131036-22153740',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '36691017',\n",
       "  'End': '36709501',\n",
       "  'Name': '4-chr16-22742-lociStitched',\n",
       "  'Gene': '4-chr16-22742-lociStitched',\n",
       "  'Region': '16:36691017-36709501',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '59670000',\n",
       "  'End': '59686035',\n",
       "  'Name': '5-chr2-1362-lociStitched',\n",
       "  'Gene': '5-chr2-1362-lociStitched',\n",
       "  'Region': '2:59670000-59686035',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '106664038',\n",
       "  'End': '106677672',\n",
       "  'Name': '5-chr11-732-lociStitched',\n",
       "  'Gene': '5-chr11-732-lociStitched',\n",
       "  'Region': '11:106664038-106677672',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '104835670',\n",
       "  'End': '104854377',\n",
       "  'Name': '3-chr12-7401-lociStitched',\n",
       "  'Gene': '3-chr12-7401-lociStitched',\n",
       "  'Region': '12:104835670-104854377',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '151939433',\n",
       "  'End': '151965502',\n",
       "  'Name': '5-chr1-23095-lociStitched',\n",
       "  'Gene': '5-chr1-23095-lociStitched',\n",
       "  'Region': '1:151939433-151965502',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '128302284',\n",
       "  'End': '128339842',\n",
       "  'Name': '5-chr2-8339-lociStitched',\n",
       "  'Gene': '5-chr2-8339-lociStitched',\n",
       "  'Region': '2:128302284-128339842',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '31881364',\n",
       "  'End': '31898113',\n",
       "  'Name': '6-chr11-24971-lociStitched',\n",
       "  'Gene': '6-chr11-24971-lociStitched',\n",
       "  'Region': '11:31881364-31898113',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '81740968',\n",
       "  'End': '81761143',\n",
       "  'Name': '3-chr15-7473-lociStitched',\n",
       "  'Gene': '3-chr15-7473-lociStitched',\n",
       "  'Region': '15:81740968-81761143',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '108232545',\n",
       "  'End': '108248654',\n",
       "  'Name': '5-chr11-4722-lociStitched',\n",
       "  'Gene': '5-chr11-4722-lociStitched',\n",
       "  'Region': '11:108232545-108248654',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '57940408',\n",
       "  'End': '57954261',\n",
       "  'Name': '4-chr4-158-lociStitched',\n",
       "  'Gene': '4-chr4-158-lociStitched',\n",
       "  'Region': '4:57940408-57954261',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '45241609',\n",
       "  'End': '45263419',\n",
       "  'Name': '5-chr9-15748-lociStitched',\n",
       "  'Gene': '5-chr9-15748-lociStitched',\n",
       "  'Region': '9:45241609-45263419',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '102984273',\n",
       "  'End': '102996147',\n",
       "  'Name': '4-chr9-12484-lociStitched',\n",
       "  'Gene': '4-chr9-12484-lociStitched',\n",
       "  'Region': '9:102984273-102996147',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '134267119',\n",
       "  'End': '134288009',\n",
       "  'Name': '6-chr5-8521-lociStitched',\n",
       "  'Gene': '6-chr5-8521-lociStitched',\n",
       "  'Region': '5:134267119-134288009',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '86208964',\n",
       "  'End': '86234142',\n",
       "  'Name': '4-chr6-8548-lociStitched',\n",
       "  'Gene': '4-chr6-8548-lociStitched',\n",
       "  'Region': '6:86208964-86234142',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '92967247',\n",
       "  'End': '92999173',\n",
       "  'Name': '6-chr2-21203-lociStitched',\n",
       "  'Gene': '6-chr2-21203-lociStitched',\n",
       "  'Region': '2:92967247-92999173',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '33157956',\n",
       "  'End': '33179851',\n",
       "  'Name': '4-chr12-25252-lociStitched',\n",
       "  'Gene': '4-chr12-25252-lociStitched',\n",
       "  'Region': '12:33157956-33179851',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '154219613',\n",
       "  'End': '154248189',\n",
       "  'Name': '4-chr4-10119-lociStitched',\n",
       "  'Gene': '4-chr4-10119-lociStitched',\n",
       "  'Region': '4:154219613-154248189',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '28424237',\n",
       "  'End': '28445456',\n",
       "  'Name': '5-chr7-7812-lociStitched',\n",
       "  'Gene': '5-chr7-7812-lociStitched',\n",
       "  'Region': '7:28424237-28445456',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '138603554',\n",
       "  'End': '138618977',\n",
       "  'Name': '4-chr7-9449-lociStitched',\n",
       "  'Gene': '4-chr7-9449-lociStitched',\n",
       "  'Region': '7:138603554-138618977',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '133042197',\n",
       "  'End': '133058543',\n",
       "  'Name': '6-chr4-6104-lociStitched',\n",
       "  'Gene': '6-chr4-6104-lociStitched',\n",
       "  'Region': '4:133042197-133058543',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '46412774',\n",
       "  'End': '46425401',\n",
       "  'Name': '3-chr19-14630-lociStitched',\n",
       "  'Gene': '3-chr19-14630-lociStitched',\n",
       "  'Region': '19:46412774-46425401',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '134081344',\n",
       "  'End': '134103257',\n",
       "  'Name': '5-chr6-13620-lociStitched',\n",
       "  'Gene': '5-chr6-13620-lociStitched',\n",
       "  'Region': '6:134081344-134103257',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '62529436',\n",
       "  'End': '62547265',\n",
       "  'Name': '4-chr10-12529-lociStitched',\n",
       "  'Gene': '4-chr10-12529-lociStitched',\n",
       "  'Region': '10:62529436-62547265',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '43463240',\n",
       "  'End': '43476441',\n",
       "  'Name': '3-chr1-23009-lociStitched',\n",
       "  'Gene': '3-chr1-23009-lociStitched',\n",
       "  'Region': '1:43463240-43476441',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '31430796',\n",
       "  'End': '31443253',\n",
       "  'Name': '3-chr14-3714-lociStitched',\n",
       "  'Gene': '3-chr14-3714-lociStitched',\n",
       "  'Region': '14:31430796-31443253',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '83396465',\n",
       "  'End': '83411026',\n",
       "  'Name': '3-chr15-2403-lociStitched',\n",
       "  'Gene': '3-chr15-2403-lociStitched',\n",
       "  'Region': '15:83396465-83411026',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '31286546',\n",
       "  'End': '31304183',\n",
       "  'Name': '4-chr12-22393-lociStitched',\n",
       "  'Gene': '4-chr12-22393-lociStitched',\n",
       "  'Region': '12:31286546-31304183',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '134254475',\n",
       "  'End': '134275028',\n",
       "  'Name': '6-chr6-13625-lociStitched',\n",
       "  'Gene': '6-chr6-13625-lociStitched',\n",
       "  'Region': '6:134254475-134275028',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '143890017',\n",
       "  'End': '143910920',\n",
       "  'Name': '5-chr2-2078-lociStitched',\n",
       "  'Gene': '5-chr2-2078-lociStitched',\n",
       "  'Region': '2:143890017-143910920',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '83435249',\n",
       "  'End': '83453855',\n",
       "  'Name': '3-chr15-8124-lociStitched',\n",
       "  'Gene': '3-chr15-8124-lociStitched',\n",
       "  'Region': '15:83435249-83453855',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '108436000',\n",
       "  'End': '108454142',\n",
       "  'Name': '4-chr10-22187-lociStitched',\n",
       "  'Gene': '4-chr10-22187-lociStitched',\n",
       "  'Region': '10:108436000-108454142',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '89585451',\n",
       "  'End': '89610314',\n",
       "  'Name': '4-chr6-17737-lociStitched',\n",
       "  'Gene': '4-chr6-17737-lociStitched',\n",
       "  'Region': '6:89585451-89610314',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '71744275',\n",
       "  'End': '71756689',\n",
       "  'Name': '4-chr2-11924-lociStitched',\n",
       "  'Gene': '4-chr2-11924-lociStitched',\n",
       "  'Region': '2:71744275-71756689',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '132179243',\n",
       "  'End': '132199987',\n",
       "  'Name': '5-chr4-19959-lociStitched',\n",
       "  'Gene': '5-chr4-19959-lociStitched',\n",
       "  'Region': '4:132179243-132199987',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '35128971',\n",
       "  'End': '35156379',\n",
       "  'Name': '4-chr7-13652-lociStitched',\n",
       "  'Gene': '4-chr7-13652-lociStitched',\n",
       "  'Region': '7:35128971-35156379',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '150207629',\n",
       "  'End': '150224389',\n",
       "  'Name': '4-chr5-7768-lociStitched',\n",
       "  'Gene': '4-chr5-7768-lociStitched',\n",
       "  'Region': '5:150207629-150224389',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '69915502',\n",
       "  'End': '69928463',\n",
       "  'Name': '3-chr10-3885-lociStitched',\n",
       "  'Gene': '3-chr10-3885-lociStitched',\n",
       "  'Region': '10:69915502-69928463',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '28379457',\n",
       "  'End': '28386807',\n",
       "  'Name': '2-chr7-24216-lociStitched',\n",
       "  'Gene': '2-chr7-24216-lociStitched',\n",
       "  'Region': '7:28379457-28386807',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '117305671',\n",
       "  'End': '117322749',\n",
       "  'Name': '6-chr11-3697-lociStitched',\n",
       "  'Gene': '6-chr11-3697-lociStitched',\n",
       "  'Region': '11:117305671-117322749',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '170152608',\n",
       "  'End': '170177557',\n",
       "  'Name': '4-chr2-2168-lociStitched',\n",
       "  'Gene': '4-chr2-2168-lociStitched',\n",
       "  'Region': '2:170152608-170177557',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '78381847',\n",
       "  'End': '78411991',\n",
       "  'Name': '5-chr11-2520-lociStitched',\n",
       "  'Gene': '5-chr11-2520-lociStitched',\n",
       "  'Region': '11:78381847-78411991',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '136167257',\n",
       "  'End': '136180756',\n",
       "  'Name': '3-chr5-23989-lociStitched',\n",
       "  'Gene': '3-chr5-23989-lociStitched',\n",
       "  'Region': '5:136167257-136180756',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '136479225',\n",
       "  'End': '136488226',\n",
       "  'Name': '2-chr7-600-lociStitched',\n",
       "  'Gene': '2-chr7-600-lociStitched',\n",
       "  'Region': '7:136479225-136488226',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '94126952',\n",
       "  'End': '94152540',\n",
       "  'Name': '5-chr6-24116-lociStitched',\n",
       "  'Gene': '5-chr6-24116-lociStitched',\n",
       "  'Region': '6:94126952-94152540',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '20956780',\n",
       "  'End': '20971325',\n",
       "  'Name': '4-chr18-1004-lociStitched',\n",
       "  'Gene': '4-chr18-1004-lociStitched',\n",
       "  'Region': '18:20956780-20971325',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '120355157',\n",
       "  'End': '120373716',\n",
       "  'Name': '3-chr14-19083-lociStitched',\n",
       "  'Gene': '3-chr14-19083-lociStitched',\n",
       "  'Region': '14:120355157-120373716',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '24159108',\n",
       "  'End': '24173221',\n",
       "  'Name': '3-chr10-2019-lociStitched',\n",
       "  'Gene': '3-chr10-2019-lociStitched',\n",
       "  'Region': '10:24159108-24173221',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '96688146',\n",
       "  'End': '96705560',\n",
       "  'Name': '4-chr10-3136-lociStitched',\n",
       "  'Gene': '4-chr10-3136-lociStitched',\n",
       "  'Region': '10:96688146-96705560',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '92062521',\n",
       "  'End': '92083843',\n",
       "  'Name': '4-chr6-17742-lociStitched',\n",
       "  'Gene': '4-chr6-17742-lociStitched',\n",
       "  'Region': '6:92062521-92083843',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '54802772',\n",
       "  'End': '54847167',\n",
       "  'Name': '5-chr11-25016-lociStitched',\n",
       "  'Gene': '5-chr11-25016-lociStitched',\n",
       "  'Region': '11:54802772-54847167',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '139658114',\n",
       "  'End': '139679858',\n",
       "  'Name': '4-chr5-23999-lociStitched',\n",
       "  'Gene': '4-chr5-23999-lociStitched',\n",
       "  'Region': '5:139658114-139679858',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '154895665',\n",
       "  'End': '154913375',\n",
       "  'Name': '3-chr4-19989-lociStitched',\n",
       "  'Gene': '3-chr4-19989-lociStitched',\n",
       "  'Region': '4:154895665-154913375',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '102052558',\n",
       "  'End': '102069509',\n",
       "  'Name': '3-chr12-1083-lociStitched',\n",
       "  'Gene': '3-chr12-1083-lociStitched',\n",
       "  'Region': '12:102052558-102069509',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '13594364',\n",
       "  'End': '13609929',\n",
       "  'Name': '3-chr8-24420-lociStitched',\n",
       "  'Gene': '3-chr8-24420-lociStitched',\n",
       "  'Region': '8:13594364-13609929',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '40757818',\n",
       "  'End': '40784889',\n",
       "  'Name': '4-chr9-22010-lociStitched',\n",
       "  'Gene': '4-chr9-22010-lociStitched',\n",
       "  'Region': '9:40757818-40784889',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '106981364',\n",
       "  'End': '107005461',\n",
       "  'Name': '5-chr4-15158-lociStitched',\n",
       "  'Gene': '5-chr4-15158-lociStitched',\n",
       "  'Region': '4:106981364-107005461',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '59428596',\n",
       "  'End': '59449904',\n",
       "  'Name': '6-chr1-10777-lociStitched',\n",
       "  'Gene': '6-chr1-10777-lociStitched',\n",
       "  'Region': '1:59428596-59449904',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '80833859',\n",
       "  'End': '80848439',\n",
       "  'Name': '4-chr8-2490-lociStitched',\n",
       "  'Gene': '4-chr8-2490-lociStitched',\n",
       "  'Region': '8:80833859-80848439',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '24607316',\n",
       "  'End': '24624317',\n",
       "  'Name': '4-chr14-25501-lociStitched',\n",
       "  'Gene': '4-chr14-25501-lociStitched',\n",
       "  'Region': '14:24607316-24624317',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '77909188',\n",
       "  'End': '77924810',\n",
       "  'Name': '3-chr9-1676-lociStitched',\n",
       "  'Gene': '3-chr9-1676-lociStitched',\n",
       "  'Region': '9:77909188-77924810',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '117280645',\n",
       "  'End': '117288312',\n",
       "  'Name': '4-chr8-10315-lociStitched',\n",
       "  'Gene': '4-chr8-10315-lociStitched',\n",
       "  'Region': '8:117280645-117288312',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '69174174',\n",
       "  'End': '69184926',\n",
       "  'Name': '5-chr14-19048-lociStitched',\n",
       "  'Gene': '5-chr14-19048-lociStitched',\n",
       "  'Region': '14:69174174-69184926',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '107413482',\n",
       "  'End': '107439545',\n",
       "  'Name': '4-chr11-6800-lociStitched',\n",
       "  'Gene': '4-chr11-6800-lociStitched',\n",
       "  'Region': '11:107413482-107439545',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '154933381',\n",
       "  'End': '154944503',\n",
       "  'Name': '3-chr4-346-lociStitched',\n",
       "  'Gene': '3-chr4-346-lociStitched',\n",
       "  'Region': '4:154933381-154944503',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '69167020',\n",
       "  'End': '69193605',\n",
       "  'Name': '4-chr17-25980-lociStitched',\n",
       "  'Gene': '4-chr17-25980-lociStitched',\n",
       "  'Region': '17:69167020-69193605',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '129232118',\n",
       "  'End': '129245020',\n",
       "  'Name': '4-chr4-13405-lociStitched',\n",
       "  'Gene': '4-chr4-13405-lociStitched',\n",
       "  'Region': '4:129232118-129245020',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '63990661',\n",
       "  'End': '64008286',\n",
       "  'Name': '6-chr9-2779-lociStitched',\n",
       "  'Gene': '6-chr9-2779-lociStitched',\n",
       "  'Region': '9:63990661-64008286',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '44479554',\n",
       "  'End': '44496835',\n",
       "  'Name': '4-chr9-4395-lociStitched',\n",
       "  'Gene': '4-chr9-4395-lociStitched',\n",
       "  'Region': '9:44479554-44496835',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '92919395',\n",
       "  'End': '92937702',\n",
       "  'Name': '4-chr6-1067-lociStitched',\n",
       "  'Gene': '4-chr6-1067-lociStitched',\n",
       "  'Region': '6:92919395-92937702',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '37541838',\n",
       "  'End': '37562393',\n",
       "  'Name': '3-chr13-18864-lociStitched',\n",
       "  'Gene': '3-chr13-18864-lociStitched',\n",
       "  'Region': '13:37541838-37562393',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '116557787',\n",
       "  'End': '116577347',\n",
       "  'Name': '4-chr10-9561-lociStitched',\n",
       "  'Gene': '4-chr10-9561-lociStitched',\n",
       "  'Region': '10:116557787-116577347',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '80483374',\n",
       "  'End': '80514994',\n",
       "  'Name': '6-chr11-25105-lociStitched',\n",
       "  'Gene': '6-chr11-25105-lociStitched',\n",
       "  'Region': '11:80483374-80514994',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '102183441',\n",
       "  'End': '102203734',\n",
       "  'Name': '5-chr12-12719-lociStitched',\n",
       "  'Gene': '5-chr12-12719-lociStitched',\n",
       "  'Region': '12:102183441-102203734',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '150545271',\n",
       "  'End': '150570506',\n",
       "  'Name': '4-chr4-19984-lociStitched',\n",
       "  'Gene': '4-chr4-19984-lociStitched',\n",
       "  'Region': '4:150545271-150570506',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '108134441',\n",
       "  'End': '108155734',\n",
       "  'Name': '5-chr11-8821-lociStitched',\n",
       "  'Gene': '5-chr11-8821-lociStitched',\n",
       "  'Region': '11:108134441-108155734',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '36194016',\n",
       "  'End': '36205088',\n",
       "  'Name': '3-chr3-4582-lociStitched',\n",
       "  'Gene': '3-chr3-4582-lociStitched',\n",
       "  'Region': '3:36194016-36205088',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '59442403',\n",
       "  'End': '59457933',\n",
       "  'Name': '6-chr15-25661-lociStitched',\n",
       "  'Gene': '6-chr15-25661-lociStitched',\n",
       "  'Region': '15:59442403-59457933',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '111749307',\n",
       "  'End': '111767900',\n",
       "  'Name': '4-chr12-25325-lociStitched',\n",
       "  'Gene': '4-chr12-25325-lociStitched',\n",
       "  'Region': '12:111749307-111767900',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '99161172',\n",
       "  'End': '99169861',\n",
       "  'Name': '3-chr5-11057-lociStitched',\n",
       "  'Gene': '3-chr5-11057-lociStitched',\n",
       "  'Region': '5:99161172-99169861',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '47301211',\n",
       "  'End': '47326997',\n",
       "  'Name': '4-chr19-16728-lociStitched',\n",
       "  'Gene': '4-chr19-16728-lociStitched',\n",
       "  'Region': '19:47301211-47326997',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '84386849',\n",
       "  'End': '84401999',\n",
       "  'Name': '4-chr10-12544-lociStitched',\n",
       "  'Gene': '4-chr10-12544-lociStitched',\n",
       "  'Region': '10:84386849-84401999',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '61399339',\n",
       "  'End': '61415948',\n",
       "  'Name': '4-chr14-1410-lociStitched',\n",
       "  'Gene': '4-chr14-1410-lociStitched',\n",
       "  'Region': '14:61399339-61415948',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '108844101',\n",
       "  'End': '108856400',\n",
       "  'Name': '4-chr3-2454-lociStitched',\n",
       "  'Gene': '4-chr3-2454-lociStitched',\n",
       "  'Region': '3:108844101-108856400',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '57125589',\n",
       "  'End': '57140551',\n",
       "  'Name': '5-chr4-8432-lociStitched',\n",
       "  'Gene': '5-chr4-8432-lociStitched',\n",
       "  'Region': '4:57125589-57140551',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '44238513',\n",
       "  'End': '44251482',\n",
       "  'Name': '4-chr17-1136-lociStitched',\n",
       "  'Gene': '4-chr17-1136-lociStitched',\n",
       "  'Region': '17:44238513-44251482',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '14917448',\n",
       "  'End': '14933263',\n",
       "  'Name': '3-chr8-9456-lociStitched',\n",
       "  'Gene': '3-chr8-9456-lociStitched',\n",
       "  'Region': '8:14917448-14933263',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '11687008',\n",
       "  'End': '11693363',\n",
       "  'Name': '3-chr2-1015-lociStitched',\n",
       "  'Gene': '3-chr2-1015-lociStitched',\n",
       "  'Region': '2:11687008-11693363',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '28506067',\n",
       "  'End': '28519588',\n",
       "  'Name': '4-chr17-5459-lociStitched',\n",
       "  'Gene': '4-chr17-5459-lociStitched',\n",
       "  'Region': '17:28506067-28519588',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '132260271',\n",
       "  'End': '132293419',\n",
       "  'Name': '5-chr2-23369-lociStitched',\n",
       "  'Gene': '5-chr2-23369-lociStitched',\n",
       "  'Region': '2:132260271-132293419',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '60078963',\n",
       "  'End': '60090549',\n",
       "  'Name': '2-chr2-9968-lociStitched',\n",
       "  'Gene': '2-chr2-9968-lociStitched',\n",
       "  'Region': '2:60078963-60090549',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '52314872',\n",
       "  'End': '52333263',\n",
       "  'Name': '4-chr11-5809-lociStitched',\n",
       "  'Gene': '4-chr11-5809-lociStitched',\n",
       "  'Region': '11:52314872-52333263',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '84975605',\n",
       "  'End': '84985136',\n",
       "  'Name': '2-chr8-2333-lociStitched',\n",
       "  'Gene': '2-chr8-2333-lociStitched',\n",
       "  'Region': '8:84975605-84985136',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '117814123',\n",
       "  'End': '117831592',\n",
       "  'Name': '4-chr11-4437-lociStitched',\n",
       "  'Gene': '4-chr11-4437-lociStitched',\n",
       "  'Region': '11:117814123-117831592',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '80070138',\n",
       "  'End': '80079881',\n",
       "  'Name': '3-chr10-18423-lociStitched',\n",
       "  'Gene': '3-chr10-18423-lociStitched',\n",
       "  'Region': '10:80070138-80079881',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '52024553',\n",
       "  'End': '52049679',\n",
       "  'Name': '4-chr13-9682-lociStitched',\n",
       "  'Gene': '4-chr13-9682-lociStitched',\n",
       "  'Region': '13:52024553-52049679',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '79872000',\n",
       "  'End': '79890439',\n",
       "  'Name': '3-chr14-19067-lociStitched',\n",
       "  'Gene': '3-chr14-19067-lociStitched',\n",
       "  'Region': '14:79872000-79890439',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '68375870',\n",
       "  'End': '68397946',\n",
       "  'Name': '4-chr2-21172-lociStitched',\n",
       "  'Gene': '4-chr2-21172-lociStitched',\n",
       "  'Region': '2:68375870-68397946',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '26474161',\n",
       "  'End': '26494007',\n",
       "  'Name': '4-chr2-16967-lociStitched',\n",
       "  'Gene': '4-chr2-16967-lociStitched',\n",
       "  'Region': '2:26474161-26494007',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '58627248',\n",
       "  'End': '58641828',\n",
       "  'Name': '4-chr10-3301-lociStitched',\n",
       "  'Gene': '4-chr10-3301-lociStitched',\n",
       "  'Region': '10:58627248-58641828',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '91392351',\n",
       "  'End': '91412305',\n",
       "  'Name': '3-chr8-9468-lociStitched',\n",
       "  'Gene': '3-chr8-9468-lociStitched',\n",
       "  'Region': '8:91392351-91412305',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '61583364',\n",
       "  'End': '61596232',\n",
       "  'Name': '4-chr2-1148-lociStitched',\n",
       "  'Gene': '4-chr2-1148-lociStitched',\n",
       "  'Region': '2:61583364-61596232',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '31186352',\n",
       "  'End': '31200862',\n",
       "  'Name': '5-chr6-9365-lociStitched',\n",
       "  'Gene': '5-chr6-9365-lociStitched',\n",
       "  'Region': '6:31186352-31200862',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '119671134',\n",
       "  'End': '119694814',\n",
       "  'Name': '5-chr11-18702-lociStitched',\n",
       "  'Gene': '5-chr11-18702-lociStitched',\n",
       "  'Region': '11:119671134-119694814',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '89103325',\n",
       "  'End': '89126294',\n",
       "  'Name': '3-chr1-14711-lociStitched',\n",
       "  'Gene': '3-chr1-14711-lociStitched',\n",
       "  'Region': '1:89103325-89126294',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '40380744',\n",
       "  'End': '40398675',\n",
       "  'Name': '4-chr9-3289-lociStitched',\n",
       "  'Gene': '4-chr9-3289-lociStitched',\n",
       "  'Region': '9:40380744-40398675',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '66259206',\n",
       "  'End': '66282502',\n",
       "  'Name': '4-chr15-8115-lociStitched',\n",
       "  'Gene': '4-chr15-8115-lociStitched',\n",
       "  'Region': '15:66259206-66282502',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '39381538',\n",
       "  'End': '39400966',\n",
       "  'Name': '4-chr1-19646-lociStitched',\n",
       "  'Gene': '4-chr1-19646-lociStitched',\n",
       "  'Region': '1:39381538-39400966',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '117351450',\n",
       "  'End': '117361726',\n",
       "  'Name': '3-chr2-17071-lociStitched',\n",
       "  'Gene': '3-chr2-17071-lociStitched',\n",
       "  'Region': '2:117351450-117361726',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '78512072',\n",
       "  'End': '78526821',\n",
       "  'Name': '3-chr15-7467-lociStitched',\n",
       "  'Gene': '3-chr15-7467-lociStitched',\n",
       "  'Region': '15:78512072-78526821',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '107721276',\n",
       "  'End': '107736223',\n",
       "  'Name': '3-chr9-8711-lociStitched',\n",
       "  'Gene': '3-chr9-8711-lociStitched',\n",
       "  'Region': '9:107721276-107736223',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '26928802',\n",
       "  'End': '26955542',\n",
       "  'Name': '4-chr2-11896-lociStitched',\n",
       "  'Gene': '4-chr2-11896-lociStitched',\n",
       "  'Region': '2:26928802-26955542',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '14126254',\n",
       "  'End': '14146585',\n",
       "  'Name': '4-chr17-19329-lociStitched',\n",
       "  'Gene': '4-chr17-19329-lociStitched',\n",
       "  'Region': '17:14126254-14146585',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '43829820',\n",
       "  'End': '43848564',\n",
       "  'Name': '4-chr12-5037-lociStitched',\n",
       "  'Gene': '4-chr12-5037-lociStitched',\n",
       "  'Region': '12:43829820-43848564',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '64436392',\n",
       "  'End': '64451352',\n",
       "  'Name': '5-chr7-1831-lociStitched',\n",
       "  'Gene': '5-chr7-1831-lociStitched',\n",
       "  'Region': '7:64436392-64451352',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '127135714',\n",
       "  'End': '127145847',\n",
       "  'Name': '3-chr10-632-lociStitched',\n",
       "  'Gene': '3-chr10-632-lociStitched',\n",
       "  'Region': '10:127135714-127145847',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '27478961',\n",
       "  'End': '27493987',\n",
       "  'Name': '3-chr7-150-lociStitched',\n",
       "  'Gene': '3-chr7-150-lociStitched',\n",
       "  'Region': '7:27478961-27493987',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '69360334',\n",
       "  'End': '69383025',\n",
       "  'Name': '6-chr11-8791-lociStitched',\n",
       "  'Gene': '6-chr11-8791-lociStitched',\n",
       "  'Region': '11:69360334-69383025',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '126787175',\n",
       "  'End': '126808516',\n",
       "  'Name': '5-chr8-24575-lociStitched',\n",
       "  'Gene': '5-chr8-24575-lociStitched',\n",
       "  'Region': '8:126787175-126808516',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '140623292',\n",
       "  'End': '140638690',\n",
       "  'Name': '3-chr4-12128-lociStitched',\n",
       "  'Gene': '3-chr4-12128-lociStitched',\n",
       "  'Region': '4:140623292-140638690',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '10806503',\n",
       "  'End': '10818686',\n",
       "  'Name': '5-chr19-2147-lociStitched',\n",
       "  'Gene': '5-chr19-2147-lociStitched',\n",
       "  'Region': '19:10806503-10818686',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '84779872',\n",
       "  'End': '84794872',\n",
       "  'Name': '4-chr8-11252-lociStitched',\n",
       "  'Gene': '4-chr8-11252-lociStitched',\n",
       "  'Region': '8:84779872-84794872',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '69408792',\n",
       "  'End': '69418808',\n",
       "  'Name': '3-chr11-5017-lociStitched',\n",
       "  'Gene': '3-chr11-5017-lociStitched',\n",
       "  'Region': '11:69408792-69418808',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '57842330',\n",
       "  'End': '57855408',\n",
       "  'Name': '5-chr1-19652-lociStitched',\n",
       "  'Gene': '5-chr1-19652-lociStitched',\n",
       "  'Region': '1:57842330-57855408',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '123200444',\n",
       "  'End': '123210823',\n",
       "  'Name': '4-chr9-11323-lociStitched',\n",
       "  'Gene': '4-chr9-11323-lociStitched',\n",
       "  'Region': '9:123200444-123210823',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '111227180',\n",
       "  'End': '111244938',\n",
       "  'Name': '4-chr12-9666-lociStitched',\n",
       "  'Gene': '4-chr12-9666-lociStitched',\n",
       "  'Region': '12:111227180-111244938',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '105457036',\n",
       "  'End': '105477144',\n",
       "  'Name': '4-chr12-18801-lociStitched',\n",
       "  'Gene': '4-chr12-18801-lociStitched',\n",
       "  'Region': '12:105457036-105477144',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '5847074',\n",
       "  'End': '5864648',\n",
       "  'Name': '4-chr10-20377-lociStitched',\n",
       "  'Gene': '4-chr10-20377-lociStitched',\n",
       "  'Region': '10:5847074-5864648',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '76318924',\n",
       "  'End': '76340151',\n",
       "  'Name': '4-chr11-8799-lociStitched',\n",
       "  'Gene': '4-chr11-8799-lociStitched',\n",
       "  'Region': '11:76318924-76340151',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '102896927',\n",
       "  'End': '102914644',\n",
       "  'Name': '5-chr13-3542-lociStitched',\n",
       "  'Gene': '5-chr13-3542-lociStitched',\n",
       "  'Region': '13:102896927-102914644',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '84511257',\n",
       "  'End': '84529125',\n",
       "  'Name': '5-chr2-4842-lociStitched',\n",
       "  'Gene': '5-chr2-4842-lociStitched',\n",
       "  'Region': '2:84511257-84529125',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '122759308',\n",
       "  'End': '122773073',\n",
       "  'Name': '4-chr8-13788-lociStitched',\n",
       "  'Gene': '4-chr8-13788-lociStitched',\n",
       "  'Region': '8:122759308-122773073',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '164941279',\n",
       "  'End': '164962791',\n",
       "  'Name': '4-chr2-10015-lociStitched',\n",
       "  'Gene': '4-chr2-10015-lociStitched',\n",
       "  'Region': '2:164941279-164962791',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '94166141',\n",
       "  'End': '94186543',\n",
       "  'Name': '5-chr6-17747-lociStitched',\n",
       "  'Gene': '5-chr6-17747-lociStitched',\n",
       "  'Region': '6:94166141-94186543',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '75996395',\n",
       "  'End': '76014279',\n",
       "  'Name': '3-chr18-1554-lociStitched',\n",
       "  'Gene': '3-chr18-1554-lociStitched',\n",
       "  'Region': '18:75996395-76014279',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '78298664',\n",
       "  'End': '78321084',\n",
       "  'Name': '4-chr10-18419-lociStitched',\n",
       "  'Gene': '4-chr10-18419-lociStitched',\n",
       "  'Region': '10:78298664-78321084',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '134245000',\n",
       "  'End': '134253051',\n",
       "  'Name': '3-chr5-10157-lociStitched',\n",
       "  'Gene': '3-chr5-10157-lociStitched',\n",
       "  'Region': '5:134245000-134253051',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '62473078',\n",
       "  'End': '62478171',\n",
       "  'Name': '1-chr4-4-lociStitched',\n",
       "  'Gene': '1-chr4-4-lociStitched',\n",
       "  'Region': '4:62473078-62478171',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '105174137',\n",
       "  'End': '105190363',\n",
       "  'Name': '4-chr4-10988-lociStitched',\n",
       "  'Gene': '4-chr4-10988-lociStitched',\n",
       "  'Region': '4:105174137-105190363',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '167776581',\n",
       "  'End': '167790398',\n",
       "  'Name': '3-chr2-448-lociStitched',\n",
       "  'Gene': '3-chr2-448-lociStitched',\n",
       "  'Region': '2:167776581-167790398',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '12471364',\n",
       "  'End': '12487437',\n",
       "  'Name': '3-chr13-3917-lociStitched',\n",
       "  'Gene': '3-chr13-3917-lociStitched',\n",
       "  'Region': '13:12471364-12487437',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '123138939',\n",
       "  'End': '123156467',\n",
       "  'Name': '3-chr5-21584-lociStitched',\n",
       "  'Gene': '3-chr5-21584-lociStitched',\n",
       "  'Region': '5:123138939-123156467',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '135725488',\n",
       "  'End': '135742885',\n",
       "  'Name': '3-chr1-10801-lociStitched',\n",
       "  'Gene': '3-chr1-10801-lociStitched',\n",
       "  'Region': '1:135725488-135742885',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '76722423',\n",
       "  'End': '76745390',\n",
       "  'Name': '3-chr14-22593-lociStitched',\n",
       "  'Gene': '3-chr14-22593-lociStitched',\n",
       "  'Region': '14:76722423-76745390',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '79507620',\n",
       "  'End': '79522192',\n",
       "  'Name': '5-chr14-3725-lociStitched',\n",
       "  'Gene': '5-chr14-3725-lociStitched',\n",
       "  'Region': '14:79507620-79522192',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '76109149',\n",
       "  'End': '76125045',\n",
       "  'Name': '5-chr18-19527-lociStitched',\n",
       "  'Gene': '5-chr18-19527-lociStitched',\n",
       "  'Region': '18:76109149-76125045',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '87744156',\n",
       "  'End': '87753699',\n",
       "  'Name': '4-chr11-7368-lociStitched',\n",
       "  'Gene': '4-chr11-7368-lociStitched',\n",
       "  'Region': '11:87744156-87753699',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '73480638',\n",
       "  'End': '73498590',\n",
       "  'Name': '3-chr7-11185-lociStitched',\n",
       "  'Gene': '3-chr7-11185-lociStitched',\n",
       "  'Region': '7:73480638-73498590',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '142898052',\n",
       "  'End': '142912742',\n",
       "  'Name': '4-chr5-15348-lociStitched',\n",
       "  'Gene': '4-chr5-15348-lociStitched',\n",
       "  'Region': '5:142898052-142912742',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '101319146',\n",
       "  'End': '101338541',\n",
       "  'Name': '3-chr15-6894-lociStitched',\n",
       "  'Gene': '3-chr15-6894-lociStitched',\n",
       "  'Region': '15:101319146-101338541',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '189866084',\n",
       "  'End': '189878264',\n",
       "  'Name': '4-chr1-3221-lociStitched',\n",
       "  'Gene': '4-chr1-3221-lociStitched',\n",
       "  'Region': '1:189866084-189878264',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '43241277',\n",
       "  'End': '43254793',\n",
       "  'Name': '4-chr13-11520-lociStitched',\n",
       "  'Gene': '4-chr13-11520-lociStitched',\n",
       "  'Region': '13:43241277-43254793',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '35076811',\n",
       "  'End': '35091695',\n",
       "  'Name': '5-chr7-17837-lociStitched',\n",
       "  'Gene': '5-chr7-17837-lociStitched',\n",
       "  'Region': '7:35076811-35091695',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '68239106',\n",
       "  'End': '68257411',\n",
       "  'Name': '4-chr10-24839-lociStitched',\n",
       "  'Gene': '4-chr10-24839-lociStitched',\n",
       "  'Region': '10:68239106-68257411',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '132929060',\n",
       "  'End': '132943531',\n",
       "  'Name': '4-chr2-6047-lociStitched',\n",
       "  'Gene': '4-chr2-6047-lociStitched',\n",
       "  'Region': '2:132929060-132943531',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '112808891',\n",
       "  'End': '112828441',\n",
       "  'Name': '5-chr7-15566-lociStitched',\n",
       "  'Gene': '5-chr7-15566-lociStitched',\n",
       "  'Region': '7:112808891-112828441',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '121087890',\n",
       "  'End': '121100600',\n",
       "  'Name': '3-chr6-11138-lociStitched',\n",
       "  'Gene': '3-chr6-11138-lociStitched',\n",
       "  'Region': '6:121087890-121100600',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '96619140',\n",
       "  'End': '96639719',\n",
       "  'Name': '3-chr15-22689-lociStitched',\n",
       "  'Gene': '3-chr15-22689-lociStitched',\n",
       "  'Region': '15:96619140-96639719',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '95340152',\n",
       "  'End': '95353016',\n",
       "  'Name': '4-chr16-25873-lociStitched',\n",
       "  'Gene': '4-chr16-25873-lociStitched',\n",
       "  'Region': '16:95340152-95353016',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '101008228',\n",
       "  'End': '101017265',\n",
       "  'Name': '4-chr12-8849-lociStitched',\n",
       "  'Gene': '4-chr12-8849-lociStitched',\n",
       "  'Region': '12:101008228-101017265',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '32321991',\n",
       "  'End': '32343646',\n",
       "  'Name': '4-chr5-17505-lociStitched',\n",
       "  'Gene': '4-chr5-17505-lociStitched',\n",
       "  'Region': '5:32321991-32343646',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '84405767',\n",
       "  'End': '84421008',\n",
       "  'Name': '4-chr17-5479-lociStitched',\n",
       "  'Gene': '4-chr17-5479-lociStitched',\n",
       "  'Region': '17:84405767-84421008',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '16398485',\n",
       "  'End': '16406882',\n",
       "  'Name': '2-chr7-194-lociStitched',\n",
       "  'Gene': '2-chr7-194-lociStitched',\n",
       "  'Region': '7:16398485-16406882',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '84181922',\n",
       "  'End': '84202018',\n",
       "  'Name': '4-chr7-15532-lociStitched',\n",
       "  'Gene': '4-chr7-15532-lociStitched',\n",
       "  'Region': '7:84181922-84202018',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '3396065',\n",
       "  'End': '3417600',\n",
       "  'Name': '4-chr8-24406-lociStitched',\n",
       "  'Gene': '4-chr8-24406-lociStitched',\n",
       "  'Region': '8:3396065-3417600',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '143141273',\n",
       "  'End': '143162959',\n",
       "  'Name': '4-chr4-19979-lociStitched',\n",
       "  'Gene': '4-chr4-19979-lociStitched',\n",
       "  'Region': '4:143141273-143162959',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '95729133',\n",
       "  'End': '95743431',\n",
       "  'Name': '4-chr16-6907-lociStitched',\n",
       "  'Gene': '4-chr16-6907-lociStitched',\n",
       "  'Region': '16:95729133-95743431',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '88638121',\n",
       "  'End': '88652675',\n",
       "  'Name': '4-chr8-6681-lociStitched',\n",
       "  'Gene': '4-chr8-6681-lociStitched',\n",
       "  'Region': '8:88638121-88652675',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '121331350',\n",
       "  'End': '121349607',\n",
       "  'Name': '3-chr9-3500-lociStitched',\n",
       "  'Gene': '3-chr9-3500-lociStitched',\n",
       "  'Region': '9:121331350-121349607',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '136027397',\n",
       "  'End': '136045470',\n",
       "  'Name': '4-chr4-12121-lociStitched',\n",
       "  'Gene': '4-chr4-12121-lociStitched',\n",
       "  'Region': '4:136027397-136045470',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '102116956',\n",
       "  'End': '102137537',\n",
       "  'Name': '5-chr2-23318-lociStitched',\n",
       "  'Gene': '5-chr2-23318-lociStitched',\n",
       "  'Region': '2:102116956-102137537',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '36331611',\n",
       "  'End': '36344705',\n",
       "  'Name': '2-chr1-13045-lociStitched',\n",
       "  'Gene': '2-chr1-13045-lociStitched',\n",
       "  'Region': '1:36331611-36344705',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '112576707',\n",
       "  'End': '112587539',\n",
       "  'Name': '2-chr12-6823-lociStitched',\n",
       "  'Gene': '2-chr12-6823-lociStitched',\n",
       "  'Region': '12:112576707-112587539',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '75655075',\n",
       "  'End': '75665333',\n",
       "  'Name': '2-chr2-2276-lociStitched',\n",
       "  'Gene': '2-chr2-2276-lociStitched',\n",
       "  'Region': '2:75655075-75665333',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '48825125',\n",
       "  'End': '48839098',\n",
       "  'Name': '5-chr6-17697-lociStitched',\n",
       "  'Gene': '5-chr6-17697-lociStitched',\n",
       "  'Region': '6:48825125-48839098',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '108833258',\n",
       "  'End': '108856638',\n",
       "  'Name': '7-chr10-3682-lociStitched',\n",
       "  'Gene': '7-chr10-3682-lociStitched',\n",
       "  'Region': '10:108833258-108856638',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '48953848',\n",
       "  'End': '48974456',\n",
       "  'Name': '4-chr17-25964-lociStitched',\n",
       "  'Gene': '4-chr17-25964-lociStitched',\n",
       "  'Region': '17:48953848-48974456',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '117268619',\n",
       "  'End': '117281186',\n",
       "  'Name': '6-chr11-6806-lociStitched',\n",
       "  'Gene': '6-chr11-6806-lociStitched',\n",
       "  'Region': '11:117268619-117281186',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '90071940',\n",
       "  'End': '90087523',\n",
       "  'Name': '3-chr16-4780-lociStitched',\n",
       "  'Gene': '3-chr16-4780-lociStitched',\n",
       "  'Region': '16:90071940-90087523',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '50244288',\n",
       "  'End': '50259325',\n",
       "  'Name': '4-chr11-5806-lociStitched',\n",
       "  'Gene': '4-chr11-5806-lociStitched',\n",
       "  'Region': '11:50244288-50259325',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '115358801',\n",
       "  'End': '115370385',\n",
       "  'Name': '2-chr10-2643-lociStitched',\n",
       "  'Gene': '2-chr10-2643-lociStitched',\n",
       "  'Region': '10:115358801-115370385',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '96706090',\n",
       "  'End': '96729438',\n",
       "  'Name': '5-chr9-8701-lociStitched',\n",
       "  'Gene': '5-chr9-8701-lociStitched',\n",
       "  'Region': '9:96706090-96729438',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '103225459',\n",
       "  'End': '103238707',\n",
       "  'Name': '3-chr11-8007-lociStitched',\n",
       "  'Gene': '3-chr11-8007-lociStitched',\n",
       "  'Region': '11:103225459-103238707',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '146438831',\n",
       "  'End': '146457735',\n",
       "  'Name': '4-chr2-10883-lociStitched',\n",
       "  'Gene': '4-chr2-10883-lociStitched',\n",
       "  'Region': '2:146438831-146457735',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '3846323',\n",
       "  'End': '3856699',\n",
       "  'Name': '2-chr19-238-lociStitched',\n",
       "  'Gene': '2-chr19-238-lociStitched',\n",
       "  'Region': '19:3846323-3856699',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '69448124',\n",
       "  'End': '69468779',\n",
       "  'Name': '3-chr12-16143-lociStitched',\n",
       "  'Gene': '3-chr12-16143-lociStitched',\n",
       "  'Region': '12:69448124-69468779',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '31428874',\n",
       "  'End': '31447411',\n",
       "  'Name': '5-chr15-11599-lociStitched',\n",
       "  'Gene': '5-chr15-11599-lociStitched',\n",
       "  'Region': '15:31428874-31447411',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '105203095',\n",
       "  'End': '105222710',\n",
       "  'Name': '3-chr4-23688-lociStitched',\n",
       "  'Gene': '3-chr4-23688-lociStitched',\n",
       "  'Region': '4:105203095-105222710',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '62865459',\n",
       "  'End': '62878110',\n",
       "  'Name': '4-chr11-15990-lociStitched',\n",
       "  'Gene': '4-chr11-15990-lociStitched',\n",
       "  'Region': '11:62865459-62878110',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '122048314',\n",
       "  'End': '122057169',\n",
       "  'Name': '4-chr14-12829-lociStitched',\n",
       "  'Gene': '4-chr14-12829-lociStitched',\n",
       "  'Region': '14:122048314-122057169',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '98018016',\n",
       "  'End': '98038905',\n",
       "  'Name': '4-chr3-19870-lociStitched',\n",
       "  'Gene': '4-chr3-19870-lociStitched',\n",
       "  'Region': '3:98018016-98038905',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '59548442',\n",
       "  'End': '59581597',\n",
       "  'Name': '5-chr15-6354-lociStitched',\n",
       "  'Gene': '5-chr15-6354-lociStitched',\n",
       "  'Region': '15:59548442-59581597',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '30688466',\n",
       "  'End': '30707843',\n",
       "  'Name': '4-chr2-14836-lociStitched',\n",
       "  'Gene': '4-chr2-14836-lociStitched',\n",
       "  'Region': '2:30688466-30707843',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '100833850',\n",
       "  'End': '100849097',\n",
       "  'Name': '5-chr11-10471-lociStitched',\n",
       "  'Gene': '5-chr11-10471-lociStitched',\n",
       "  'Region': '11:100833850-100849097',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '57957134',\n",
       "  'End': '57974872',\n",
       "  'Name': '3-chr9-20331-lociStitched',\n",
       "  'Gene': '3-chr9-20331-lociStitched',\n",
       "  'Region': '9:57957134-57974872',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '81285948',\n",
       "  'End': '81302291',\n",
       "  'Name': '5-chr10-12541-lociStitched',\n",
       "  'Gene': '5-chr10-12541-lociStitched',\n",
       "  'Region': '10:81285948-81302291',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '108920014',\n",
       "  'End': '108949472',\n",
       "  'Name': '4-chr10-6756-lociStitched',\n",
       "  'Gene': '4-chr10-6756-lociStitched',\n",
       "  'Region': '10:108920014-108949472',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '38081482',\n",
       "  'End': '38098974',\n",
       "  'Name': '3-chr13-8871-lociStitched',\n",
       "  'Gene': '3-chr13-8871-lociStitched',\n",
       "  'Region': '13:38081482-38098974',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '72532847',\n",
       "  'End': '72552626',\n",
       "  'Name': '4-chr8-6677-lociStitched',\n",
       "  'Gene': '4-chr8-6677-lociStitched',\n",
       "  'Region': '8:72532847-72552626',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '26769257',\n",
       "  'End': '26787783',\n",
       "  'Name': '4-chr7-4935-lociStitched',\n",
       "  'Gene': '4-chr7-4935-lociStitched',\n",
       "  'Region': '7:26769257-26787783',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '80802316',\n",
       "  'End': '80820423',\n",
       "  'Name': '3-chr8-24485-lociStitched',\n",
       "  'Gene': '3-chr8-24485-lociStitched',\n",
       "  'Region': '8:80802316-80820423',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '32167488',\n",
       "  'End': '32182464',\n",
       "  'Name': '3-chr2-16991-lociStitched',\n",
       "  'Gene': '3-chr2-16991-lociStitched',\n",
       "  'Region': '2:32167488-32182464',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '47756852',\n",
       "  'End': '47781913',\n",
       "  'Name': '4-chr17-9801-lociStitched',\n",
       "  'Gene': '4-chr17-9801-lociStitched',\n",
       "  'Region': '17:47756852-47781913',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '167621737',\n",
       "  'End': '167635155',\n",
       "  'Name': '2-chr2-6058-lociStitched',\n",
       "  'Gene': '2-chr2-6058-lociStitched',\n",
       "  'Region': '2:167621737-167635155',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '72854719',\n",
       "  'End': '72876599',\n",
       "  'Name': '4-chr5-23878-lociStitched',\n",
       "  'Gene': '4-chr5-23878-lociStitched',\n",
       "  'Region': '5:72854719-72876599',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '88612766',\n",
       "  'End': '88626654',\n",
       "  'Name': '4-chr11-12645-lociStitched',\n",
       "  'Gene': '4-chr11-12645-lociStitched',\n",
       "  'Region': '11:88612766-88626654',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '141924214',\n",
       "  'End': '141937926',\n",
       "  'Name': '5-chr4-21504-lociStitched',\n",
       "  'Gene': '5-chr4-21504-lociStitched',\n",
       "  'Region': '4:141924214-141937926',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '93704101',\n",
       "  'End': '93712685',\n",
       "  'Name': '5-chr1-4821-lociStitched',\n",
       "  'Gene': '5-chr1-4821-lociStitched',\n",
       "  'Region': '1:93704101-93712685',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '132012966',\n",
       "  'End': '132023147',\n",
       "  'Name': '5-chr2-13209-lociStitched',\n",
       "  'Gene': '5-chr2-13209-lociStitched',\n",
       "  'Region': '2:132012966-132023147',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '93443805',\n",
       "  'End': '93449854',\n",
       "  'Name': '3-chr2-184-lociStitched',\n",
       "  'Gene': '3-chr2-184-lociStitched',\n",
       "  'Region': '2:93443805-93449854',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '145862988',\n",
       "  'End': '145872379',\n",
       "  'Name': '4-chr2-17116-lociStitched',\n",
       "  'Gene': '4-chr2-17116-lociStitched',\n",
       "  'Region': '2:145862988-145872379',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '111652038',\n",
       "  'End': '111658726',\n",
       "  'Name': '3-chr7-204-lociStitched',\n",
       "  'Gene': '3-chr7-204-lociStitched',\n",
       "  'Region': '7:111652038-111658726',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '44245692',\n",
       "  'End': '44254358',\n",
       "  'Name': '3-chr19-58-lociStitched',\n",
       "  'Gene': '3-chr19-58-lociStitched',\n",
       "  'Region': '19:44245692-44254358',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '32418408',\n",
       "  'End': '32428605',\n",
       "  'Name': '3-chr2-4558-lociStitched',\n",
       "  'Gene': '3-chr2-4558-lociStitched',\n",
       "  'Region': '2:32418408-32428605',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '119496335',\n",
       "  'End': '119518670',\n",
       "  'Name': '3-chr1-16842-lociStitched',\n",
       "  'Gene': '3-chr1-16842-lociStitched',\n",
       "  'Region': '1:119496335-119518670',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '75741344',\n",
       "  'End': '75767022',\n",
       "  'Name': '5-chr11-25074-lociStitched',\n",
       "  'Gene': '5-chr11-25074-lociStitched',\n",
       "  'Region': '11:75741344-75767022',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '165590310',\n",
       "  'End': '165610592',\n",
       "  'Name': '3-chr1-13114-lociStitched',\n",
       "  'Gene': '3-chr1-13114-lociStitched',\n",
       "  'Region': '1:165590310-165610592',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '87437938',\n",
       "  'End': '87461475',\n",
       "  'Name': '5-chr11-12642-lociStitched',\n",
       "  'Gene': '5-chr11-12642-lociStitched',\n",
       "  'Region': '11:87437938-87461475',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '80060536',\n",
       "  'End': '80069636',\n",
       "  'Name': '4-chr7-2485-lociStitched',\n",
       "  'Gene': '4-chr7-2485-lociStitched',\n",
       "  'Region': '7:80060536-80069636',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '54217808',\n",
       "  'End': '54239392',\n",
       "  'Name': '3-chr11-20450-lociStitched',\n",
       "  'Gene': '3-chr11-20450-lociStitched',\n",
       "  'Region': '11:54217808-54239392',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '31314571',\n",
       "  'End': '31327916',\n",
       "  'Name': '3-chr14-14267-lociStitched',\n",
       "  'Gene': '3-chr14-14267-lociStitched',\n",
       "  'Region': '14:31314571-31327916',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '98121538',\n",
       "  'End': '98148270',\n",
       "  'Name': '4-chr13-12774-lociStitched',\n",
       "  'Gene': '4-chr13-12774-lociStitched',\n",
       "  'Region': '13:98121538-98148270',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '34180784',\n",
       "  'End': '34201383',\n",
       "  'Name': '5-chr13-20587-lociStitched',\n",
       "  'Gene': '5-chr13-20587-lociStitched',\n",
       "  'Region': '13:34180784-34201383',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '131222229',\n",
       "  'End': '131243882',\n",
       "  'Name': '3-chr5-9349-lociStitched',\n",
       "  'Gene': '3-chr5-9349-lociStitched',\n",
       "  'Region': '5:131222229-131243882',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '84373379',\n",
       "  'End': '84388347',\n",
       "  'Name': '5-chr2-2712-lociStitched',\n",
       "  'Gene': '5-chr2-2712-lociStitched',\n",
       "  'Region': '2:84373379-84388347',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '172511905',\n",
       "  'End': '172524306',\n",
       "  'Name': '3-chr1-21081-lociStitched',\n",
       "  'Gene': '3-chr1-21081-lociStitched',\n",
       "  'Region': '1:172511905-172524306',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '40246544',\n",
       "  'End': '40260600',\n",
       "  'Name': '4-chr19-11776-lociStitched',\n",
       "  'Gene': '4-chr19-11776-lociStitched',\n",
       "  'Region': '19:40246544-40260600',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '55218347',\n",
       "  'End': '55234808',\n",
       "  'Name': '3-chr6-3638-lociStitched',\n",
       "  'Gene': '3-chr6-3638-lociStitched',\n",
       "  'Region': '6:55218347-55234808',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '44473445',\n",
       "  'End': '44484895',\n",
       "  'Name': '2-chr7-289-lociStitched',\n",
       "  'Gene': '2-chr7-289-lociStitched',\n",
       "  'Region': '7:44473445-44484895',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '90576371',\n",
       "  'End': '90587971',\n",
       "  'Name': '3-chr3-3605-lociStitched',\n",
       "  'Gene': '3-chr3-3605-lociStitched',\n",
       "  'Region': '3:90576371-90587971',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '11620996',\n",
       "  'End': '11649313',\n",
       "  'Name': '4-chr16-11649-lociStitched',\n",
       "  'Gene': '4-chr16-11649-lociStitched',\n",
       "  'Region': '16:11620996-11649313',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '105131470',\n",
       "  'End': '105143537',\n",
       "  'Name': '2-chr5-3628-lociStitched',\n",
       "  'Gene': '2-chr5-3628-lociStitched',\n",
       "  'Region': '5:105131470-105143537',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '114678617',\n",
       "  'End': '114694370',\n",
       "  'Name': '5-chr11-12670-lociStitched',\n",
       "  'Gene': '5-chr11-12670-lociStitched',\n",
       "  'Region': '11:114678617-114694370',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '27926350',\n",
       "  'End': '27938689',\n",
       "  'Name': '2-chr15-3342-lociStitched',\n",
       "  'Gene': '2-chr15-3342-lociStitched',\n",
       "  'Region': '15:27926350-27938689',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '32381602',\n",
       "  'End': '32398712',\n",
       "  'Name': '3-chr3-17180-lociStitched',\n",
       "  'Gene': '3-chr3-17180-lociStitched',\n",
       "  'Region': '3:32381602-32398712',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '35877019',\n",
       "  'End': '35897765',\n",
       "  'Name': '3-chr5-23847-lociStitched',\n",
       "  'Gene': '3-chr5-23847-lociStitched',\n",
       "  'Region': '5:35877019-35897765',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '16311139',\n",
       "  'End': '16328197',\n",
       "  'Name': '4-chr19-9088-lociStitched',\n",
       "  'Gene': '4-chr19-9088-lociStitched',\n",
       "  'Region': '19:16311139-16328197',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '110162315',\n",
       "  'End': '110166111',\n",
       "  'Name': '2-chr7-3853-lociStitched',\n",
       "  'Gene': '2-chr7-3853-lociStitched',\n",
       "  'Region': '7:110162315-110166111',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '40630442',\n",
       "  'End': '40650479',\n",
       "  'Name': '4-chr9-6226-lociStitched',\n",
       "  'Gene': '4-chr9-6226-lociStitched',\n",
       "  'Region': '9:40630442-40650479',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '134210345',\n",
       "  'End': '134221383',\n",
       "  'Name': '4-chr6-11145-lociStitched',\n",
       "  'Gene': '4-chr6-11145-lociStitched',\n",
       "  'Region': '6:134210345-134221383',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '135673650',\n",
       "  'End': '135688695',\n",
       "  'Name': '3-chr3-9255-lociStitched',\n",
       "  'Gene': '3-chr3-9255-lociStitched',\n",
       "  'Region': '3:135673650-135688695',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '47852449',\n",
       "  'End': '47867017',\n",
       "  'Name': '5-chr17-531-lociStitched',\n",
       "  'Gene': '5-chr17-531-lociStitched',\n",
       "  'Region': '17:47852449-47867017',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '99126035',\n",
       "  'End': '99144808',\n",
       "  'Name': '4-chr7-13679-lociStitched',\n",
       "  'Gene': '4-chr7-13679-lociStitched',\n",
       "  'Region': '7:99126035-99144808',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '73433345',\n",
       "  'End': '73453040',\n",
       "  'Name': '5-chr14-14309-lociStitched',\n",
       "  'Gene': '5-chr14-14309-lociStitched',\n",
       "  'Region': '14:73433345-73453040',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '28801823',\n",
       "  'End': '28814299',\n",
       "  'Name': '5-chr7-13646-lociStitched',\n",
       "  'Gene': '5-chr7-13646-lociStitched',\n",
       "  'Region': '7:28801823-28814299',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '131075126',\n",
       "  'End': '131087798',\n",
       "  'Name': '3-chr1-8272-lociStitched',\n",
       "  'Gene': '3-chr1-8272-lociStitched',\n",
       "  'Region': '1:131075126-131087798',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '97859803',\n",
       "  'End': '97876584',\n",
       "  'Name': '4-chr4-17375-lociStitched',\n",
       "  'Gene': '4-chr4-17375-lociStitched',\n",
       "  'Region': '4:97859803-97876584',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '134808910',\n",
       "  'End': '134823519',\n",
       "  'Name': '3-chr1-3396-lociStitched',\n",
       "  'Gene': '3-chr1-3396-lociStitched',\n",
       "  'Region': '1:134808910-134823519',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '8637808',\n",
       "  'End': '8658860',\n",
       "  'Name': '5-chr8-15614-lociStitched',\n",
       "  'Gene': '5-chr8-15614-lociStitched',\n",
       "  'Region': '8:8637808-8658860',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '12328764',\n",
       "  'End': '12352782',\n",
       "  'Name': '4-chr10-4989-lociStitched',\n",
       "  'Gene': '4-chr10-4989-lociStitched',\n",
       "  'Region': '10:12328764-12352782',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '85454587',\n",
       "  'End': '85463449',\n",
       "  'Name': '2-chr12-8044-lociStitched',\n",
       "  'Gene': '2-chr12-8044-lociStitched',\n",
       "  'Region': '12:85454587-85463449',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '167377317',\n",
       "  'End': '167388806',\n",
       "  'Name': '3-chr2-7052-lociStitched',\n",
       "  'Gene': '3-chr2-7052-lociStitched',\n",
       "  'Region': '2:167377317-167388806',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '45495827',\n",
       "  'End': '45504930',\n",
       "  'Name': '3-chr9-2934-lociStitched',\n",
       "  'Gene': '3-chr9-2934-lociStitched',\n",
       "  'Region': '9:45495827-45504930',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '34313317',\n",
       "  'End': '34325356',\n",
       "  'Name': '2-chr13-16211-lociStitched',\n",
       "  'Gene': '2-chr13-16211-lociStitched',\n",
       "  'Region': '13:34313317-34325356',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '98965274',\n",
       "  'End': '98978990',\n",
       "  'Name': '3-chr15-25746-lociStitched',\n",
       "  'Gene': '3-chr15-25746-lociStitched',\n",
       "  'Region': '15:98965274-98978990',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '154815303',\n",
       "  'End': '154834175',\n",
       "  'Name': '3-chr1-13105-lociStitched',\n",
       "  'Gene': '3-chr1-13105-lociStitched',\n",
       "  'Region': '1:154815303-154834175',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '21980750',\n",
       "  'End': '21998537',\n",
       "  'Name': '3-chr16-25798-lociStitched',\n",
       "  'Gene': '3-chr16-25798-lociStitched',\n",
       "  'Region': '16:21980750-21998537',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '25219938',\n",
       "  'End': '25231772',\n",
       "  'Name': '3-chr17-14489-lociStitched',\n",
       "  'Gene': '3-chr17-14489-lociStitched',\n",
       "  'Region': '17:25219938-25231772',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '123859504',\n",
       "  'End': '123885507',\n",
       "  'Name': '4-chr5-20058-lociStitched',\n",
       "  'Gene': '4-chr5-20058-lociStitched',\n",
       "  'Region': '5:123859504-123885507',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '83534955',\n",
       "  'End': '83543469',\n",
       "  'Name': '2-chr15-64-lociStitched',\n",
       "  'Gene': '2-chr15-64-lociStitched',\n",
       "  'Region': '15:83534955-83543469',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '155220179',\n",
       "  'End': '155238830',\n",
       "  'Name': '6-chr1-16877-lociStitched',\n",
       "  'Gene': '6-chr1-16877-lociStitched',\n",
       "  'Region': '1:155220179-155238830',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '145771876',\n",
       "  'End': '145782013',\n",
       "  'Name': '3-chr2-21247-lociStitched',\n",
       "  'Gene': '3-chr2-21247-lociStitched',\n",
       "  'Region': '2:145771876-145782013',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '83195870',\n",
       "  'End': '83203961',\n",
       "  'Name': '3-chr17-36-lociStitched',\n",
       "  'Gene': '3-chr17-36-lociStitched',\n",
       "  'Region': '17:83195870-83203961',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '102226744',\n",
       "  'End': '102248125',\n",
       "  'Name': '4-chr7-17925-lociStitched',\n",
       "  'Gene': '4-chr7-17925-lociStitched',\n",
       "  'Region': '7:102226744-102248125',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '155178248',\n",
       "  'End': '155202899',\n",
       "  'Name': '4-chr4-8484-lociStitched',\n",
       "  'Gene': '4-chr4-8484-lociStitched',\n",
       "  'Region': '4:155178248-155202899',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '71790083',\n",
       "  'End': '71814645',\n",
       "  'Name': '5-chr2-14872-lociStitched',\n",
       "  'Gene': '5-chr2-14872-lociStitched',\n",
       "  'Region': '2:71790083-71814645',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '119182516',\n",
       "  'End': '119224276',\n",
       "  'Name': '5-chr11-16101-lociStitched',\n",
       "  'Gene': '5-chr11-16101-lociStitched',\n",
       "  'Region': '11:119182516-119224276',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '12037160',\n",
       "  'End': '12060605',\n",
       "  'Name': '5-chr14-9701-lociStitched',\n",
       "  'Gene': '5-chr14-9701-lociStitched',\n",
       "  'Region': '14:12037160-12060605',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '139249750',\n",
       "  'End': '139257782',\n",
       "  'Name': '2-chr5-192-lociStitched',\n",
       "  'Gene': '2-chr5-192-lociStitched',\n",
       "  'Region': '5:139249750-139257782',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '108892794',\n",
       "  'End': '108905274',\n",
       "  'Name': '3-chr3-13313-lociStitched',\n",
       "  'Gene': '3-chr3-13313-lociStitched',\n",
       "  'Region': '3:108892794-108905274',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '11142519',\n",
       "  'End': '11149511',\n",
       "  'Name': '3-chr4-560-lociStitched',\n",
       "  'Gene': '3-chr4-560-lociStitched',\n",
       "  'Region': '4:11142519-11149511',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '84097073',\n",
       "  'End': '84116909',\n",
       "  'Name': '4-chr17-1628-lociStitched',\n",
       "  'Gene': '4-chr17-1628-lociStitched',\n",
       "  'Region': '17:84097073-84116909',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '10544812',\n",
       "  'End': '10555586',\n",
       "  'Name': '3-chr10-24785-lociStitched',\n",
       "  'Gene': '3-chr10-24785-lociStitched',\n",
       "  'Region': '10:10544812-10555586',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '27525723',\n",
       "  'End': '27548330',\n",
       "  'Name': '4-chr15-19112-lociStitched',\n",
       "  'Gene': '4-chr15-19112-lociStitched',\n",
       "  'Region': '15:27525723-27548330',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '111428584',\n",
       "  'End': '111438078',\n",
       "  'Name': '3-chr5-20040-lociStitched',\n",
       "  'Gene': '3-chr5-20040-lociStitched',\n",
       "  'Region': '5:111428584-111438078',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '32537327',\n",
       "  'End': '32548168',\n",
       "  'Name': '3-chr2-10850-lociStitched',\n",
       "  'Gene': '3-chr2-10850-lociStitched',\n",
       "  'Region': '2:32537327-32548168',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '121705183',\n",
       "  'End': '121719112',\n",
       "  'Name': '5-chr11-25214-lociStitched',\n",
       "  'Gene': '5-chr11-25214-lociStitched',\n",
       "  'Region': '11:121705183-121719112',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '84624084',\n",
       "  'End': '84644450',\n",
       "  'Name': '3-chr2-11932-lociStitched',\n",
       "  'Gene': '3-chr2-11932-lociStitched',\n",
       "  'Region': '2:84624084-84644450',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '106226669',\n",
       "  'End': '106241561',\n",
       "  'Name': '3-chr8-24528-lociStitched',\n",
       "  'Gene': '3-chr8-24528-lociStitched',\n",
       "  'Region': '8:106226669-106241561',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '86575283',\n",
       "  'End': '86588164',\n",
       "  'Name': '4-chr11-6271-lociStitched',\n",
       "  'Gene': '4-chr11-6271-lociStitched',\n",
       "  'Region': '11:86575283-86588164',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '50471292',\n",
       "  'End': '50479941',\n",
       "  'Name': '4-chr9-2013-lociStitched',\n",
       "  'Gene': '4-chr9-2013-lociStitched',\n",
       "  'Region': '9:50471292-50479941',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '96774506',\n",
       "  'End': '96794541',\n",
       "  'Name': '3-chr11-5369-lociStitched',\n",
       "  'Gene': '3-chr11-5369-lociStitched',\n",
       "  'Region': '11:96774506-96794541',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '172498258',\n",
       "  'End': '172521238',\n",
       "  'Name': '5-chr2-4858-lociStitched',\n",
       "  'Gene': '5-chr2-4858-lociStitched',\n",
       "  'Region': '2:172498258-172521238',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '60000155',\n",
       "  'End': '60022567',\n",
       "  'Name': '4-chr11-4709-lociStitched',\n",
       "  'Gene': '4-chr11-4709-lociStitched',\n",
       "  'Region': '11:60000155-60022567',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '83471422',\n",
       "  'End': '83487160',\n",
       "  'Name': '5-chr9-2939-lociStitched',\n",
       "  'Gene': '5-chr9-2939-lociStitched',\n",
       "  'Region': '9:83471422-83487160',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '91428135',\n",
       "  'End': '91453080',\n",
       "  'Name': '5-chr16-19298-lociStitched',\n",
       "  'Gene': '5-chr16-19298-lociStitched',\n",
       "  'Region': '16:91428135-91453080',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '109349509',\n",
       "  'End': '109362577',\n",
       "  'Name': '3-chr11-12664-lociStitched',\n",
       "  'Gene': '3-chr11-12664-lociStitched',\n",
       "  'Region': '11:109349509-109362577',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '122462284',\n",
       "  'End': '122483615',\n",
       "  'Name': '5-chr3-4296-lociStitched',\n",
       "  'Gene': '5-chr3-4296-lociStitched',\n",
       "  'Region': '3:122462284-122483615',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '31491886',\n",
       "  'End': '31501765',\n",
       "  'Name': '3-chr16-16503-lociStitched',\n",
       "  'Gene': '3-chr16-16503-lociStitched',\n",
       "  'Region': '16:31491886-31501765',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '6768243',\n",
       "  'End': '6780621',\n",
       "  'Name': '5-chr17-9784-lociStitched',\n",
       "  'Gene': '5-chr17-9784-lociStitched',\n",
       "  'Region': '17:6768243-6780621',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '125091968',\n",
       "  'End': '125108640',\n",
       "  'Name': '4-chr5-15322-lociStitched',\n",
       "  'Gene': '4-chr5-15322-lociStitched',\n",
       "  'Region': '5:125091968-125108640',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '5050085',\n",
       "  'End': '5063587',\n",
       "  'Name': '3-chr17-14482-lociStitched',\n",
       "  'Gene': '3-chr17-14482-lociStitched',\n",
       "  'Region': '17:5050085-5063587',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '117244719',\n",
       "  'End': '117263063',\n",
       "  'Name': '4-chr8-20284-lociStitched',\n",
       "  'Gene': '4-chr8-20284-lociStitched',\n",
       "  'Region': '8:117244719-117263063',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '38129233',\n",
       "  'End': '38139086',\n",
       "  'Name': '4-chr16-22743-lociStitched',\n",
       "  'Gene': '4-chr16-22743-lociStitched',\n",
       "  'Region': '16:38129233-38139086',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '126627197',\n",
       "  'End': '126636445',\n",
       "  'Name': '2-chr7-67-lociStitched',\n",
       "  'Gene': '2-chr7-67-lociStitched',\n",
       "  'Region': '7:126627197-126636445',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '74914998',\n",
       "  'End': '74931474',\n",
       "  'Name': '4-chr15-25687-lociStitched',\n",
       "  'Gene': '4-chr15-25687-lociStitched',\n",
       "  'Region': '15:74914998-74931474',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '100706778',\n",
       "  'End': '100722223',\n",
       "  'Name': '5-chr11-22329-lociStitched',\n",
       "  'Gene': '5-chr11-22329-lociStitched',\n",
       "  'Region': '11:100706778-100722223',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '69116887',\n",
       "  'End': '69130232',\n",
       "  'Name': '2-chr11-12617-lociStitched',\n",
       "  'Gene': '2-chr11-12617-lociStitched',\n",
       "  'Region': '11:69116887-69130232',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '133612485',\n",
       "  'End': '133623375',\n",
       "  'Name': '3-chr4-11011-lociStitched',\n",
       "  'Gene': '3-chr4-11011-lociStitched',\n",
       "  'Region': '4:133612485-133623375',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '25606110',\n",
       "  'End': '25624347',\n",
       "  'Name': '4-chr8-12395-lociStitched',\n",
       "  'Gene': '4-chr8-12395-lociStitched',\n",
       "  'Region': '8:25606110-25624347',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '113083101',\n",
       "  'End': '113101146',\n",
       "  'Name': '4-chr11-10475-lociStitched',\n",
       "  'Gene': '4-chr11-10475-lociStitched',\n",
       "  'Region': '11:113083101-113101146',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '113645146',\n",
       "  'End': '113664208',\n",
       "  'Name': '3-chr2-4845-lociStitched',\n",
       "  'Gene': '3-chr2-4845-lociStitched',\n",
       "  'Region': '2:113645146-113664208',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '91146447',\n",
       "  'End': '91162791',\n",
       "  'Name': '3-chr10-24882-lociStitched',\n",
       "  'Gene': '3-chr10-24882-lociStitched',\n",
       "  'Region': '10:91146447-91162791',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '82922438',\n",
       "  'End': '82933531',\n",
       "  'Name': '3-chr18-1793-lociStitched',\n",
       "  'Gene': '3-chr18-1793-lociStitched',\n",
       "  'Region': '18:82922438-82933531',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '46142129',\n",
       "  'End': '46164567',\n",
       "  'Name': '3-chr8-11242-lociStitched',\n",
       "  'Gene': '3-chr8-11242-lociStitched',\n",
       "  'Region': '8:46142129-46164567',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '100456874',\n",
       "  'End': '100483592',\n",
       "  'Name': '4-chr9-24724-lociStitched',\n",
       "  'Gene': '4-chr9-24724-lociStitched',\n",
       "  'Region': '9:100456874-100483592',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '12925876',\n",
       "  'End': '12946147',\n",
       "  'Name': '4-chr8-18014-lociStitched',\n",
       "  'Gene': '4-chr8-18014-lociStitched',\n",
       "  'Region': '8:12925876-12946147',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '17558103',\n",
       "  'End': '17573738',\n",
       "  'Name': '3-chr8-7862-lociStitched',\n",
       "  'Gene': '3-chr8-7862-lociStitched',\n",
       "  'Region': '8:17558103-17573738',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '74897944',\n",
       "  'End': '74909460',\n",
       "  'Name': '3-chr9-56-lociStitched',\n",
       "  'Gene': '3-chr9-56-lociStitched',\n",
       "  'Region': '9:74897944-74909460',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '109411414',\n",
       "  'End': '109430655',\n",
       "  'Name': '3-chr4-21462-lociStitched',\n",
       "  'Gene': '3-chr4-21462-lociStitched',\n",
       "  'Region': '4:109411414-109430655',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '170204295',\n",
       "  'End': '170219680',\n",
       "  'Name': '5-chr2-7054-lociStitched',\n",
       "  'Gene': '5-chr2-7054-lociStitched',\n",
       "  'Region': '2:170204295-170219680',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '121356967',\n",
       "  'End': '121366158',\n",
       "  'Name': '4-chr14-14333-lociStitched',\n",
       "  'Gene': '4-chr14-14333-lociStitched',\n",
       "  'Region': '14:121356967-121366158',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '149562455',\n",
       "  'End': '149573352',\n",
       "  'Name': '3-chr4-19983-lociStitched',\n",
       "  'Gene': '3-chr4-19983-lociStitched',\n",
       "  'Region': '4:149562455-149573352',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '129034044',\n",
       "  'End': '129050233',\n",
       "  'Name': '4-chr4-1196-lociStitched',\n",
       "  'Gene': '4-chr4-1196-lociStitched',\n",
       "  'Region': '4:129034044-129050233',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '72387652',\n",
       "  'End': '72396121',\n",
       "  'Name': '2-chr6-17717-lociStitched',\n",
       "  'Gene': '2-chr6-17717-lociStitched',\n",
       "  'Region': '6:72387652-72396121',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '130733165',\n",
       "  'End': '130752144',\n",
       "  'Name': '5-chr2-9997-lociStitched',\n",
       "  'Gene': '5-chr2-9997-lociStitched',\n",
       "  'Region': '2:130733165-130752144',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '70772356',\n",
       "  'End': '70788904',\n",
       "  'Name': '3-chr9-15779-lociStitched',\n",
       "  'Gene': '3-chr9-15779-lociStitched',\n",
       "  'Region': '9:70772356-70788904',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '89374083',\n",
       "  'End': '89394970',\n",
       "  'Name': '3-chr4-1650-lociStitched',\n",
       "  'Gene': '3-chr4-1650-lociStitched',\n",
       "  'Region': '4:89374083-89394970',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '32290080',\n",
       "  'End': '32305180',\n",
       "  'Name': '3-chr19-2148-lociStitched',\n",
       "  'Gene': '3-chr19-2148-lociStitched',\n",
       "  'Region': '19:32290080-32305180',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '166771220',\n",
       "  'End': '166784959',\n",
       "  'Name': '4-chr2-3423-lociStitched',\n",
       "  'Gene': '4-chr2-3423-lociStitched',\n",
       "  'Region': '2:166771220-166784959',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '12956410',\n",
       "  'End': '12978578',\n",
       "  'Name': '3-chr1-1252-lociStitched',\n",
       "  'Gene': '3-chr1-1252-lociStitched',\n",
       "  'Region': '1:12956410-12978578',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '120795874',\n",
       "  'End': '120808933',\n",
       "  'Name': '6-chr8-18141-lociStitched',\n",
       "  'Gene': '6-chr8-18141-lociStitched',\n",
       "  'Region': '8:120795874-120808933',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '92379045',\n",
       "  'End': '92390187',\n",
       "  'Name': '3-chr2-895-lociStitched',\n",
       "  'Gene': '3-chr2-895-lociStitched',\n",
       "  'Region': '2:92379045-92390187',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '144548502',\n",
       "  'End': '144561427',\n",
       "  'Name': '3-chr3-23596-lociStitched',\n",
       "  'Gene': '3-chr3-23596-lociStitched',\n",
       "  'Region': '3:144548502-144561427',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '88580121',\n",
       "  'End': '88593653',\n",
       "  'Name': '4-chr1-11827-lociStitched',\n",
       "  'Gene': '4-chr1-11827-lociStitched',\n",
       "  'Region': '1:88580121-88593653',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '85936148',\n",
       "  'End': '85962015',\n",
       "  'Name': '4-chr15-22675-lociStitched',\n",
       "  'Gene': '4-chr15-22675-lociStitched',\n",
       "  'Region': '15:85936148-85962015',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '100825157',\n",
       "  'End': '100846166',\n",
       "  'Name': '4-chr6-24129-lociStitched',\n",
       "  'Gene': '4-chr6-24129-lociStitched',\n",
       "  'Region': '6:100825157-100846166',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '123182856',\n",
       "  'End': '123202904',\n",
       "  'Name': '3-chr10-7955-lociStitched',\n",
       "  'Gene': '3-chr10-7955-lociStitched',\n",
       "  'Region': '10:123182856-123202904',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '84171446',\n",
       "  'End': '84179371',\n",
       "  'Name': '3-chr12-3702-lociStitched',\n",
       "  'Gene': '3-chr12-3702-lociStitched',\n",
       "  'Region': '12:84171446-84179371',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '138800481',\n",
       "  'End': '138822674',\n",
       "  'Name': '4-chr3-5603-lociStitched',\n",
       "  'Gene': '4-chr3-5603-lociStitched',\n",
       "  'Region': '3:138800481-138822674',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '102150498',\n",
       "  'End': '102162635',\n",
       "  'Name': '3-chr12-2381-lociStitched',\n",
       "  'Gene': '3-chr12-2381-lociStitched',\n",
       "  'Region': '12:102150498-102162635',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '38145451',\n",
       "  'End': '38158233',\n",
       "  'Name': '4-chr13-5866-lociStitched',\n",
       "  'Gene': '4-chr13-5866-lociStitched',\n",
       "  'Region': '13:38145451-38158233',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '17723245',\n",
       "  'End': '17734193',\n",
       "  'Name': '3-chr6-7164-lociStitched',\n",
       "  'Gene': '3-chr6-7164-lociStitched',\n",
       "  'Region': '6:17723245-17734193',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '45529971',\n",
       "  'End': '45544284',\n",
       "  'Name': '2-chr17-1183-lociStitched',\n",
       "  'Gene': '2-chr17-1183-lociStitched',\n",
       "  'Region': '17:45529971-45544284',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '182002623',\n",
       "  'End': '182017955',\n",
       "  'Name': '3-chr1-13129-lociStitched',\n",
       "  'Gene': '3-chr1-13129-lociStitched',\n",
       "  'Region': '1:182002623-182017955',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '105848743',\n",
       "  'End': '105868210',\n",
       "  'Name': '3-chr1-9132-lociStitched',\n",
       "  'Gene': '3-chr1-9132-lociStitched',\n",
       "  'Region': '1:105848743-105868210',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '121209606',\n",
       "  'End': '121244451',\n",
       "  'Name': '6-chr2-8335-lociStitched',\n",
       "  'Gene': '6-chr2-8335-lociStitched',\n",
       "  'Region': '2:121209606-121244451',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '78348613',\n",
       "  'End': '78363566',\n",
       "  'Name': '3-chr11-25086-lociStitched',\n",
       "  'Gene': '3-chr11-25086-lociStitched',\n",
       "  'Region': '11:78348613-78363566',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '58095991',\n",
       "  'End': '58107853',\n",
       "  'Name': '2-chr11-13999-lociStitched',\n",
       "  'Gene': '2-chr11-13999-lociStitched',\n",
       "  'Region': '11:58095991-58107853',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '53437058',\n",
       "  'End': '53453355',\n",
       "  'Name': '5-chr5-20016-lociStitched',\n",
       "  'Gene': '5-chr5-20016-lociStitched',\n",
       "  'Region': '5:53437058-53453355',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '103030649',\n",
       "  'End': '103038342',\n",
       "  'Name': '3-chr11-6798-lociStitched',\n",
       "  'Gene': '3-chr11-6798-lociStitched',\n",
       "  'Region': '11:103030649-103038342',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '79644319',\n",
       "  'End': '79660325',\n",
       "  'Name': '3-chr11-25102-lociStitched',\n",
       "  'Gene': '3-chr11-25102-lociStitched',\n",
       "  'Region': '11:79644319-79660325',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '84959977',\n",
       "  'End': '84965159',\n",
       "  'Name': '2-chr2-75-lociStitched',\n",
       "  'Gene': '2-chr2-75-lociStitched',\n",
       "  'Region': '2:84959977-84965159',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '5337249',\n",
       "  'End': '5351379',\n",
       "  'Name': '3-chr12-10494-lociStitched',\n",
       "  'Gene': '3-chr12-10494-lociStitched',\n",
       "  'Region': '12:5337249-5351379',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '126812398',\n",
       "  'End': '126823972',\n",
       "  'Name': '2-chr4-6099-lociStitched',\n",
       "  'Gene': '2-chr4-6099-lociStitched',\n",
       "  'Region': '4:126812398-126823972',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '69807057',\n",
       "  'End': '69822387',\n",
       "  'Name': '3-chr13-6837-lociStitched',\n",
       "  'Gene': '3-chr13-6837-lociStitched',\n",
       "  'Region': '13:69807057-69822387',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '52652244',\n",
       "  'End': '52665619',\n",
       "  'Name': '4-chr13-18890-lociStitched',\n",
       "  'Gene': '4-chr13-18890-lociStitched',\n",
       "  'Region': '13:52652244-52665619',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '134189270',\n",
       "  'End': '134201372',\n",
       "  'Name': '3-chr1-6979-lociStitched',\n",
       "  'Gene': '3-chr1-6979-lociStitched',\n",
       "  'Region': '1:134189270-134201372',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '150245292',\n",
       "  'End': '150263258',\n",
       "  'Name': '3-chr5-13537-lociStitched',\n",
       "  'Gene': '3-chr5-13537-lociStitched',\n",
       "  'Region': '5:150245292-150263258',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '13350297',\n",
       "  'End': '13373935',\n",
       "  'Name': '3-chr16-25783-lociStitched',\n",
       "  'Gene': '3-chr16-25783-lociStitched',\n",
       "  'Region': '16:13350297-13373935',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '63336247',\n",
       "  'End': '63350546',\n",
       "  'Name': '3-chr10-5333-lociStitched',\n",
       "  'Gene': '3-chr10-5333-lociStitched',\n",
       "  'Region': '10:63336247-63350546',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '165969212',\n",
       "  'End': '165983374',\n",
       "  'Name': '2-chr2-641-lociStitched',\n",
       "  'Gene': '2-chr2-641-lociStitched',\n",
       "  'Region': '2:165969212-165983374',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '45674444',\n",
       "  'End': '45691095',\n",
       "  'Name': '4-chr13-20598-lociStitched',\n",
       "  'Gene': '4-chr13-20598-lociStitched',\n",
       "  'Region': '13:45674444-45691095',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '134082957',\n",
       "  'End': '134093186',\n",
       "  'Name': '2-chr1-1723-lociStitched',\n",
       "  'Gene': '2-chr1-1723-lociStitched',\n",
       "  'Region': '1:134082957-134093186',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '28923001',\n",
       "  'End': '28931091',\n",
       "  'Name': '3-chr17-16567-lociStitched',\n",
       "  'Gene': '3-chr17-16567-lociStitched',\n",
       "  'Region': '17:28923001-28931091',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '11401677',\n",
       "  'End': '11419475',\n",
       "  'Name': '4-chr4-1509-lociStitched',\n",
       "  'Gene': '4-chr4-1509-lociStitched',\n",
       "  'Region': '4:11401677-11419475',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '146207024',\n",
       "  'End': '146222958',\n",
       "  'Name': '3-chr6-7195-lociStitched',\n",
       "  'Gene': '3-chr6-7195-lociStitched',\n",
       "  'Region': '6:146207024-146222958',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '92482650',\n",
       "  'End': '92499377',\n",
       "  'Name': '4-chr7-21812-lociStitched',\n",
       "  'Gene': '4-chr7-21812-lociStitched',\n",
       "  'Region': '7:92482650-92499377',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '108536575',\n",
       "  'End': '108539742',\n",
       "  'Name': '1-chr10-16-lociStitched',\n",
       "  'Gene': '1-chr10-16-lociStitched',\n",
       "  'Region': '10:108536575-108539742',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '142039185',\n",
       "  'End': '142050196',\n",
       "  'Name': '3-chr4-19975-lociStitched',\n",
       "  'Gene': '3-chr4-19975-lociStitched',\n",
       "  'Region': '4:142039185-142050196',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '57194359',\n",
       "  'End': '57200415',\n",
       "  'Name': '3-chr16-16521-lociStitched',\n",
       "  'Gene': '3-chr16-16521-lociStitched',\n",
       "  'Region': '16:57194359-57200415',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '24808676',\n",
       "  'End': '24822024',\n",
       "  'Name': '3-chr7-2620-lociStitched',\n",
       "  'Gene': '3-chr7-2620-lociStitched',\n",
       "  'Region': '7:24808676-24822024',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '62495998',\n",
       "  'End': '62511469',\n",
       "  'Name': '3-chr10-5332-lociStitched',\n",
       "  'Gene': '3-chr10-5332-lociStitched',\n",
       "  'Region': '10:62495998-62511469',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '83718415',\n",
       "  'End': '83735918',\n",
       "  'Name': '3-chr2-3044-lociStitched',\n",
       "  'Gene': '3-chr2-3044-lociStitched',\n",
       "  'Region': '2:83718415-83735918',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '32599517',\n",
       "  'End': '32611009',\n",
       "  'Name': '3-chr3-1566-lociStitched',\n",
       "  'Gene': '3-chr3-1566-lociStitched',\n",
       "  'Region': '3:32599517-32611009',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '43876193',\n",
       "  'End': '43891099',\n",
       "  'Name': '3-chr19-22945-lociStitched',\n",
       "  'Gene': '3-chr19-22945-lociStitched',\n",
       "  'Region': '19:43876193-43891099',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '64747927',\n",
       "  'End': '64757523',\n",
       "  'Name': '3-chr7-12322-lociStitched',\n",
       "  'Gene': '3-chr7-12322-lociStitched',\n",
       "  'Region': '7:64747927-64757523',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '79362240',\n",
       "  'End': '79372206',\n",
       "  'Name': '3-chr3-1503-lociStitched',\n",
       "  'Gene': '3-chr3-1503-lociStitched',\n",
       "  'Region': '3:79362240-79372206',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '128837935',\n",
       "  'End': '128856586',\n",
       "  'Name': '3-chr6-4351-lociStitched',\n",
       "  'Gene': '3-chr6-4351-lociStitched',\n",
       "  'Region': '6:128837935-128856586',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '122912402',\n",
       "  'End': '122936706',\n",
       "  'Name': '5-chr8-9484-lociStitched',\n",
       "  'Gene': '5-chr8-9484-lociStitched',\n",
       "  'Region': '8:122912402-122936706',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '58357926',\n",
       "  'End': '58369360',\n",
       "  'Name': '2-chr10-4994-lociStitched',\n",
       "  'Gene': '2-chr10-4994-lociStitched',\n",
       "  'Region': '10:58357926-58369360',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '126670701',\n",
       "  'End': '126683861',\n",
       "  'Name': '2-chr2-14906-lociStitched',\n",
       "  'Gene': '2-chr2-14906-lociStitched',\n",
       "  'Region': '2:126670701-126683861',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '80424063',\n",
       "  'End': '80447363',\n",
       "  'Name': '3-chr10-24867-lociStitched',\n",
       "  'Gene': '3-chr10-24867-lociStitched',\n",
       "  'Region': '10:80424063-80447363',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '45188488',\n",
       "  'End': '45201693',\n",
       "  'Name': '3-chr9-1593-lociStitched',\n",
       "  'Gene': '3-chr9-1593-lociStitched',\n",
       "  'Region': '9:45188488-45201693',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chrX',\n",
       "  'Start': '169926803',\n",
       "  'End': '169942140',\n",
       "  'Name': '3-chrX-11793-lociStitched',\n",
       "  'Gene': '3-chrX-11793-lociStitched',\n",
       "  'Region': 'X:169926803-169942140',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '70635506',\n",
       "  'End': '70652977',\n",
       "  'Name': '4-chr8-4667-lociStitched',\n",
       "  'Gene': '4-chr8-4667-lociStitched',\n",
       "  'Region': '8:70635506-70652977',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '85896643',\n",
       "  'End': '85921376',\n",
       "  'Name': '4-chr15-8973-lociStitched',\n",
       "  'Gene': '4-chr15-8973-lociStitched',\n",
       "  'Region': '15:85896643-85921376',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '68415294',\n",
       "  'End': '68431318',\n",
       "  'Name': '5-chr2-13169-lociStitched',\n",
       "  'Gene': '5-chr2-13169-lociStitched',\n",
       "  'Region': '2:68415294-68431318',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '135810341',\n",
       "  'End': '135822582',\n",
       "  'Name': '4-chr1-10802-lociStitched',\n",
       "  'Gene': '4-chr1-10802-lociStitched',\n",
       "  'Region': '1:135810341-135822582',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '97471112',\n",
       "  'End': '97486277',\n",
       "  'Name': '4-chr16-19309-lociStitched',\n",
       "  'Gene': '4-chr16-19309-lociStitched',\n",
       "  'Region': '16:97471112-97486277',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '95548611',\n",
       "  'End': '95576411',\n",
       "  'Name': '4-chr11-14052-lociStitched',\n",
       "  'Gene': '4-chr11-14052-lociStitched',\n",
       "  'Region': '11:95548611-95576411',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '137485710',\n",
       "  'End': '137498162',\n",
       "  'Name': '3-chr4-17451-lociStitched',\n",
       "  'Gene': '3-chr4-17451-lociStitched',\n",
       "  'Region': '4:137485710-137498162',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '30754740',\n",
       "  'End': '30764328',\n",
       "  'Name': '3-chr13-2815-lociStitched',\n",
       "  'Gene': '3-chr13-2815-lociStitched',\n",
       "  'Region': '13:30754740-30764328',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '107376880',\n",
       "  'End': '107390910',\n",
       "  'Name': '4-chr11-18662-lociStitched',\n",
       "  'Gene': '4-chr11-18662-lociStitched',\n",
       "  'Region': '11:107376880-107390910',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '74826915',\n",
       "  'End': '74839292',\n",
       "  'Name': '2-chr11-7990-lociStitched',\n",
       "  'Gene': '2-chr11-7990-lociStitched',\n",
       "  'Region': '11:74826915-74839292',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '62496371',\n",
       "  'End': '62509403',\n",
       "  'Name': '3-chr4-21432-lociStitched',\n",
       "  'Gene': '3-chr4-21432-lociStitched',\n",
       "  'Region': '4:62496371-62509403',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '74419918',\n",
       "  'End': '74440992',\n",
       "  'Name': '4-chr17-1627-lociStitched',\n",
       "  'Gene': '4-chr17-1627-lociStitched',\n",
       "  'Region': '17:74419918-74440992',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '78646866',\n",
       "  'End': '78661048',\n",
       "  'Name': '3-chr1-16810-lociStitched',\n",
       "  'Gene': '3-chr1-16810-lociStitched',\n",
       "  'Region': '1:78646866-78661048',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '57173516',\n",
       "  'End': '57189818',\n",
       "  'Name': '5-chr19-26186-lociStitched',\n",
       "  'Gene': '5-chr19-26186-lociStitched',\n",
       "  'Region': '19:57173516-57189818',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '30352783',\n",
       "  'End': '30366603',\n",
       "  'Name': '3-chr6-7775-lociStitched',\n",
       "  'Gene': '3-chr6-7775-lociStitched',\n",
       "  'Region': '6:30352783-30366603',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '95296997',\n",
       "  'End': '95316983',\n",
       "  'Name': '3-chr10-22182-lociStitched',\n",
       "  'Gene': '3-chr10-22182-lociStitched',\n",
       "  'Region': '10:95296997-95316983',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '144985905',\n",
       "  'End': '145000464',\n",
       "  'Name': '4-chr7-15607-lociStitched',\n",
       "  'Gene': '4-chr7-15607-lociStitched',\n",
       "  'Region': '7:144985905-145000464',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '84942950',\n",
       "  'End': '84954708',\n",
       "  'Name': '3-chr15-14394-lociStitched',\n",
       "  'Gene': '3-chr15-14394-lociStitched',\n",
       "  'Region': '15:84942950-84954708',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '113536283',\n",
       "  'End': '113550132',\n",
       "  'Name': '2-chr7-1923-lociStitched',\n",
       "  'Gene': '2-chr7-1923-lociStitched',\n",
       "  'Region': '7:113536283-113550132',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '8674425',\n",
       "  'End': '8686797',\n",
       "  'Name': '4-chr11-20435-lociStitched',\n",
       "  'Gene': '4-chr11-20435-lociStitched',\n",
       "  'Region': '11:8674425-8686797',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '58365264',\n",
       "  'End': '58382618',\n",
       "  'Name': '3-chr15-2545-lociStitched',\n",
       "  'Gene': '3-chr15-2545-lociStitched',\n",
       "  'Region': '15:58365264-58382618',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '149592784',\n",
       "  'End': '149609683',\n",
       "  'Name': '3-chr4-1372-lociStitched',\n",
       "  'Gene': '3-chr4-1372-lociStitched',\n",
       "  'Region': '4:149592784-149609683',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '86522897',\n",
       "  'End': '86530219',\n",
       "  'Name': '2-chr1-16815-lociStitched',\n",
       "  'Gene': '2-chr1-16815-lociStitched',\n",
       "  'Region': '1:86522897-86530219',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '110080171',\n",
       "  'End': '110098719',\n",
       "  'Name': '4-chr13-22529-lociStitched',\n",
       "  'Gene': '4-chr13-22529-lociStitched',\n",
       "  'Region': '13:110080171-110098719',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '91611891',\n",
       "  'End': '91628948',\n",
       "  'Name': '3-chr12-18788-lociStitched',\n",
       "  'Gene': '3-chr12-18788-lociStitched',\n",
       "  'Region': '12:91611891-91628948',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '41746965',\n",
       "  'End': '41761377',\n",
       "  'Name': '3-chr13-25377-lociStitched',\n",
       "  'Gene': '3-chr13-25377-lociStitched',\n",
       "  'Region': '13:41746965-41761377',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '133483812',\n",
       "  'End': '133495220',\n",
       "  'Name': '2-chr4-6549-lociStitched',\n",
       "  'Gene': '2-chr4-6549-lociStitched',\n",
       "  'Region': '4:133483812-133495220',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '36588810',\n",
       "  'End': '36604762',\n",
       "  'Name': '4-chr1-11806-lociStitched',\n",
       "  'Gene': '4-chr1-11806-lociStitched',\n",
       "  'Region': '1:36588810-36604762',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '101012128',\n",
       "  'End': '101020915',\n",
       "  'Name': '4-chr11-5830-lociStitched',\n",
       "  'Gene': '4-chr11-5830-lociStitched',\n",
       "  'Region': '11:101012128-101020915',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '70806503',\n",
       "  'End': '70815040',\n",
       "  'Name': '2-chr6-1063-lociStitched',\n",
       "  'Gene': '2-chr6-1063-lociStitched',\n",
       "  'Region': '6:70806503-70815040',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '51108876',\n",
       "  'End': '51121030',\n",
       "  'Name': '2-chr13-22492-lociStitched',\n",
       "  'Gene': '2-chr13-22492-lociStitched',\n",
       "  'Region': '13:51108876-51121030',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '94126865',\n",
       "  'End': '94138007',\n",
       "  'Name': '2-chr11-20488-lociStitched',\n",
       "  'Gene': '2-chr11-20488-lociStitched',\n",
       "  'Region': '11:94126865-94138007',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '89163042',\n",
       "  'End': '89172873',\n",
       "  'Name': '3-chr4-335-lociStitched',\n",
       "  'Gene': '3-chr4-335-lociStitched',\n",
       "  'Region': '4:89163042-89172873',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '14215309',\n",
       "  'End': '14237389',\n",
       "  'Name': '3-chr16-510-lociStitched',\n",
       "  'Gene': '3-chr16-510-lociStitched',\n",
       "  'Region': '16:14215309-14237389',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '58817558',\n",
       "  'End': '58834411',\n",
       "  'Name': '3-chr10-9543-lociStitched',\n",
       "  'Gene': '3-chr10-9543-lociStitched',\n",
       "  'Region': '10:58817558-58834411',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '114464440',\n",
       "  'End': '114468815',\n",
       "  'Name': '2-chr9-675-lociStitched',\n",
       "  'Gene': '2-chr9-675-lociStitched',\n",
       "  'Region': '9:114464440-114468815',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '10276468',\n",
       "  'End': '10283125',\n",
       "  'Name': '3-chr19-10730-lociStitched',\n",
       "  'Gene': '3-chr19-10730-lociStitched',\n",
       "  'Region': '19:10276468-10283125',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '28479121',\n",
       "  'End': '28493089',\n",
       "  'Name': '4-chr13-18847-lociStitched',\n",
       "  'Gene': '4-chr13-18847-lociStitched',\n",
       "  'Region': '13:28479121-28493089',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '40439740',\n",
       "  'End': '40460073',\n",
       "  'Name': '4-chr9-22006-lociStitched',\n",
       "  'Gene': '4-chr9-22006-lociStitched',\n",
       "  'Region': '9:40439740-40460073',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '13044097',\n",
       "  'End': '13060967',\n",
       "  'Name': '5-chr10-11327-lociStitched',\n",
       "  'Gene': '5-chr10-11327-lociStitched',\n",
       "  'Region': '10:13044097-13060967',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '144498411',\n",
       "  'End': '144513903',\n",
       "  'Name': '3-chr3-8410-lociStitched',\n",
       "  'Gene': '3-chr3-8410-lociStitched',\n",
       "  'Region': '3:144498411-144513903',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '117353053',\n",
       "  'End': '117362724',\n",
       "  'Name': '2-chr5-23927-lociStitched',\n",
       "  'Gene': '2-chr5-23927-lociStitched',\n",
       "  'Region': '5:117353053-117362724',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '149255506',\n",
       "  'End': '149282840',\n",
       "  'Name': '4-chr4-17469-lociStitched',\n",
       "  'Gene': '4-chr4-17469-lociStitched',\n",
       "  'Region': '4:149255506-149282840',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '41185389',\n",
       "  'End': '41192967',\n",
       "  'Name': '4-chr4-8423-lociStitched',\n",
       "  'Gene': '4-chr4-8423-lociStitched',\n",
       "  'Region': '4:41185389-41192967',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '122290600',\n",
       "  'End': '122302924',\n",
       "  'Name': '3-chr6-6165-lociStitched',\n",
       "  'Gene': '3-chr6-6165-lociStitched',\n",
       "  'Region': '6:122290600-122302924',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '57857787',\n",
       "  'End': '57873071',\n",
       "  'Name': '4-chr4-21425-lociStitched',\n",
       "  'Gene': '4-chr4-21425-lociStitched',\n",
       "  'Region': '4:57857787-57873071',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '104864758',\n",
       "  'End': '104886842',\n",
       "  'Name': '4-chr9-24731-lociStitched',\n",
       "  'Gene': '4-chr9-24731-lociStitched',\n",
       "  'Region': '9:104864758-104886842',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '11933525',\n",
       "  'End': '11959106',\n",
       "  'Name': '4-chr14-25477-lociStitched',\n",
       "  'Gene': '4-chr14-25477-lociStitched',\n",
       "  'Region': '14:11933525-11959106',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '94651965',\n",
       "  'End': '94667209',\n",
       "  'Name': '4-chr6-11124-lociStitched',\n",
       "  'Gene': '4-chr6-11124-lociStitched',\n",
       "  'Region': '6:94651965-94667209',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '138059802',\n",
       "  'End': '138075123',\n",
       "  'Name': '3-chr3-15074-lociStitched',\n",
       "  'Gene': '3-chr3-15074-lociStitched',\n",
       "  'Region': '3:138059802-138075123',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '128551227',\n",
       "  'End': '128570629',\n",
       "  'Name': '5-chr5-5654-lociStitched',\n",
       "  'Gene': '5-chr5-5654-lociStitched',\n",
       "  'Region': '5:128551227-128570629',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '67425289',\n",
       "  'End': '67447887',\n",
       "  'Name': '4-chr5-15271-lociStitched',\n",
       "  'Gene': '4-chr5-15271-lociStitched',\n",
       "  'Region': '5:67425289-67447887',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '97077410',\n",
       "  'End': '97087836',\n",
       "  'Name': '3-chr5-17568-lociStitched',\n",
       "  'Gene': '3-chr5-17568-lociStitched',\n",
       "  'Region': '5:97077410-97087836',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '28329933',\n",
       "  'End': '28344121',\n",
       "  'Name': '2-chr17-4508-lociStitched',\n",
       "  'Gene': '2-chr17-4508-lociStitched',\n",
       "  'Region': '17:28329933-28344121',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '10655508',\n",
       "  'End': '10667438',\n",
       "  'Name': '3-chr16-5917-lociStitched',\n",
       "  'Gene': '3-chr16-5917-lociStitched',\n",
       "  'Region': '16:10655508-10667438',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '133345713',\n",
       "  'End': '133348926',\n",
       "  'Name': '2-chr4-87-lociStitched',\n",
       "  'Gene': '2-chr4-87-lociStitched',\n",
       "  'Region': '4:133345713-133348926',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '98171921',\n",
       "  'End': '98181714',\n",
       "  'Name': '4-chr1-1887-lociStitched',\n",
       "  'Gene': '4-chr1-1887-lociStitched',\n",
       "  'Region': '1:98171921-98181714',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '34061335',\n",
       "  'End': '34075491',\n",
       "  'Name': '3-chr1-7557-lociStitched',\n",
       "  'Gene': '3-chr1-7557-lociStitched',\n",
       "  'Region': '1:34061335-34075491',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '49087749',\n",
       "  'End': '49100695',\n",
       "  'Name': '2-chr13-8068-lociStitched',\n",
       "  'Gene': '2-chr13-8068-lociStitched',\n",
       "  'Region': '13:49087749-49100695',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '30934686',\n",
       "  'End': '30945237',\n",
       "  'Name': '4-chr16-3949-lociStitched',\n",
       "  'Gene': '4-chr16-3949-lociStitched',\n",
       "  'Region': '16:30934686-30945237',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '6291454',\n",
       "  'End': '6299788',\n",
       "  'Name': '3-chr19-1795-lociStitched',\n",
       "  'Gene': '3-chr19-1795-lociStitched',\n",
       "  'Region': '19:6291454-6299788',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '5211923',\n",
       "  'End': '5235122',\n",
       "  'Name': '3-chr12-16111-lociStitched',\n",
       "  'Gene': '3-chr12-16111-lociStitched',\n",
       "  'Region': '12:5211923-5235122',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '70834195',\n",
       "  'End': '70846610',\n",
       "  'Name': '3-chr9-7294-lociStitched',\n",
       "  'Gene': '3-chr9-7294-lociStitched',\n",
       "  'Region': '9:70834195-70846610',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '23435099',\n",
       "  'End': '23446862',\n",
       "  'Name': '2-chr13-16200-lociStitched',\n",
       "  'Gene': '2-chr13-16200-lociStitched',\n",
       "  'Region': '13:23435099-23446862',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '111328286',\n",
       "  'End': '111347349',\n",
       "  'Name': '4-chr6-21706-lociStitched',\n",
       "  'Gene': '4-chr6-21706-lociStitched',\n",
       "  'Region': '6:111328286-111347349',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '27730267',\n",
       "  'End': '27741891',\n",
       "  'Name': '3-chr2-11898-lociStitched',\n",
       "  'Gene': '3-chr2-11898-lociStitched',\n",
       "  'Region': '2:27730267-27741891',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '34442237',\n",
       "  'End': '34456334',\n",
       "  'Name': '3-chr17-4512-lociStitched',\n",
       "  'Gene': '3-chr17-4512-lociStitched',\n",
       "  'Region': '17:34442237-34456334',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '29468181',\n",
       "  'End': '29484324',\n",
       "  'Name': '4-chr2-9171-lociStitched',\n",
       "  'Gene': '4-chr2-9171-lociStitched',\n",
       "  'Region': '2:29468181-29484324',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '55813870',\n",
       "  'End': '55828829',\n",
       "  'Name': '4-chr19-10745-lociStitched',\n",
       "  'Gene': '4-chr19-10745-lociStitched',\n",
       "  'Region': '19:55813870-55828829',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '47212344',\n",
       "  'End': '47242996',\n",
       "  'Name': '4-chr8-8637-lociStitched',\n",
       "  'Gene': '4-chr8-8637-lociStitched',\n",
       "  'Region': '8:47212344-47242996',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '107452912',\n",
       "  'End': '107466407',\n",
       "  'Name': '3-chr11-8012-lociStitched',\n",
       "  'Gene': '3-chr11-8012-lociStitched',\n",
       "  'Region': '11:107452912-107466407',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '109573555',\n",
       "  'End': '109585006',\n",
       "  'Name': '3-chr11-16077-lociStitched',\n",
       "  'Gene': '3-chr11-16077-lociStitched',\n",
       "  'Region': '11:109573555-109585006',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '10689041',\n",
       "  'End': '10693959',\n",
       "  'Name': '2-chr19-100-lociStitched',\n",
       "  'Gene': '2-chr19-100-lociStitched',\n",
       "  'Region': '19:10689041-10693959',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '51332990',\n",
       "  'End': '51347930',\n",
       "  'Name': '3-chr6-7173-lociStitched',\n",
       "  'Gene': '3-chr6-7173-lociStitched',\n",
       "  'Region': '6:51332990-51347930',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '56894421',\n",
       "  'End': '56908023',\n",
       "  'Name': '3-chr9-24644-lociStitched',\n",
       "  'Gene': '3-chr9-24644-lociStitched',\n",
       "  'Region': '9:56894421-56908023',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '133408755',\n",
       "  'End': '133425126',\n",
       "  'Name': '3-chr4-6105-lociStitched',\n",
       "  'Gene': '3-chr4-6105-lociStitched',\n",
       "  'Region': '4:133408755-133425126',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '83293350',\n",
       "  'End': '83307759',\n",
       "  'Name': '4-chr15-12864-lociStitched',\n",
       "  'Gene': '4-chr15-12864-lociStitched',\n",
       "  'Region': '15:83293350-83307759',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '101477540',\n",
       "  'End': '101496046',\n",
       "  'Name': '3-chr15-4494-lociStitched',\n",
       "  'Gene': '3-chr15-4494-lociStitched',\n",
       "  'Region': '15:101477540-101496046',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '128489694',\n",
       "  'End': '128504862',\n",
       "  'Name': '3-chr10-13957-lociStitched',\n",
       "  'Gene': '3-chr10-13957-lociStitched',\n",
       "  'Region': '10:128489694-128504862',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '53803012',\n",
       "  'End': '53820635',\n",
       "  'Name': '4-chr11-12594-lociStitched',\n",
       "  'Gene': '4-chr11-12594-lociStitched',\n",
       "  'Region': '11:53803012-53820635',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '72502711',\n",
       "  'End': '72512577',\n",
       "  'Name': '3-chr8-833-lociStitched',\n",
       "  'Gene': '3-chr8-833-lociStitched',\n",
       "  'Region': '8:72502711-72512577',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '115580906',\n",
       "  'End': '115585054',\n",
       "  'Name': '2-chr11-25180-lociStitched',\n",
       "  'Gene': '2-chr11-25180-lociStitched',\n",
       "  'Region': '11:115580906-115585054',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '56776820',\n",
       "  'End': '56789403',\n",
       "  'Name': '3-chr9-7905-lociStitched',\n",
       "  'Gene': '3-chr9-7905-lociStitched',\n",
       "  'Region': '9:56776820-56789403',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '142363778',\n",
       "  'End': '142375804',\n",
       "  'Name': '3-chr5-11089-lociStitched',\n",
       "  'Gene': '3-chr5-11089-lociStitched',\n",
       "  'Region': '5:142363778-142375804',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '113658418',\n",
       "  'End': '113673667',\n",
       "  'Name': '3-chr11-18677-lociStitched',\n",
       "  'Gene': '3-chr11-18677-lociStitched',\n",
       "  'Region': '11:113658418-113673667',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '113268217',\n",
       "  'End': '113289587',\n",
       "  'Name': '4-chr11-16082-lociStitched',\n",
       "  'Gene': '4-chr11-16082-lociStitched',\n",
       "  'Region': '11:113268217-113289587',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '47946478',\n",
       "  'End': '47958836',\n",
       "  'Name': '3-chr17-3567-lociStitched',\n",
       "  'Gene': '3-chr17-3567-lociStitched',\n",
       "  'Region': '17:47946478-47958836',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '19956417',\n",
       "  'End': '19971703',\n",
       "  'Name': '3-chr11-24947-lociStitched',\n",
       "  'Gene': '3-chr11-24947-lociStitched',\n",
       "  'Region': '11:19956417-19971703',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '112466458',\n",
       "  'End': '112479893',\n",
       "  'Name': '3-chr7-872-lociStitched',\n",
       "  'Gene': '3-chr7-872-lociStitched',\n",
       "  'Region': '7:112466458-112479893',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '87609371',\n",
       "  'End': '87626511',\n",
       "  'Name': '4-chr17-9823-lociStitched',\n",
       "  'Gene': '4-chr17-9823-lociStitched',\n",
       "  'Region': '17:87609371-87626511',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '59732894',\n",
       "  'End': '59745514',\n",
       "  'Name': '2-chr11-15984-lociStitched',\n",
       "  'Gene': '2-chr11-15984-lociStitched',\n",
       "  'Region': '11:59732894-59745514',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '114981709',\n",
       "  'End': '114989972',\n",
       "  'Name': '2-chr6-11131-lociStitched',\n",
       "  'Gene': '2-chr6-11131-lociStitched',\n",
       "  'Region': '6:114981709-114989972',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '80513560',\n",
       "  'End': '80521917',\n",
       "  'Name': '2-chr17-215-lociStitched',\n",
       "  'Gene': '2-chr17-215-lociStitched',\n",
       "  'Region': '17:80513560-80521917',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '36924471',\n",
       "  'End': '36938625',\n",
       "  'Name': '2-chr15-14354-lociStitched',\n",
       "  'Gene': '2-chr15-14354-lociStitched',\n",
       "  'Region': '15:36924471-36938625',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '72833126',\n",
       "  'End': '72856409',\n",
       "  'Name': '4-chr2-14873-lociStitched',\n",
       "  'Gene': '4-chr2-14873-lociStitched',\n",
       "  'Region': '2:72833126-72856409',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '82152157',\n",
       "  'End': '82165631',\n",
       "  'Name': '3-chr15-1415-lociStitched',\n",
       "  'Gene': '3-chr15-1415-lociStitched',\n",
       "  'Region': '15:82152157-82165631',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '192747521',\n",
       "  'End': '192755990',\n",
       "  'Name': '4-chr1-16934-lociStitched',\n",
       "  'Gene': '4-chr1-16934-lociStitched',\n",
       "  'Region': '1:192747521-192755990',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '97808472',\n",
       "  'End': '97820839',\n",
       "  'Name': '4-chr11-6274-lociStitched',\n",
       "  'Gene': '4-chr11-6274-lociStitched',\n",
       "  'Region': '11:97808472-97820839',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '123925646',\n",
       "  'End': '123940507',\n",
       "  'Name': '3-chr9-11324-lociStitched',\n",
       "  'Gene': '3-chr9-11324-lociStitched',\n",
       "  'Region': '9:123925646-123940507',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '34017248',\n",
       "  'End': '34033735',\n",
       "  'Name': '3-chr1-14666-lociStitched',\n",
       "  'Gene': '3-chr1-14666-lociStitched',\n",
       "  'Region': '1:34017248-34033735',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '57082151',\n",
       "  'End': '57092688',\n",
       "  'Name': '3-chr4-17349-lociStitched',\n",
       "  'Gene': '3-chr4-17349-lociStitched',\n",
       "  'Region': '4:57082151-57092688',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '117676866',\n",
       "  'End': '117688158',\n",
       "  'Name': '3-chr11-14110-lociStitched',\n",
       "  'Gene': '3-chr11-14110-lociStitched',\n",
       "  'Region': '11:117676866-117688158',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '11604060',\n",
       "  'End': '11618064',\n",
       "  'Name': '4-chr14-7426-lociStitched',\n",
       "  'Gene': '4-chr14-7426-lociStitched',\n",
       "  'Region': '14:11604060-11618064',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '46323217',\n",
       "  'End': '46341084',\n",
       "  'Name': '4-chr8-18052-lociStitched',\n",
       "  'Gene': '4-chr8-18052-lociStitched',\n",
       "  'Region': '8:46323217-46341084',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '14365909',\n",
       "  'End': '14378550',\n",
       "  'Name': '2-chr16-19238-lociStitched',\n",
       "  'Gene': '2-chr16-19238-lociStitched',\n",
       "  'Region': '16:14365909-14378550',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '64232461',\n",
       "  'End': '64253831',\n",
       "  'Name': '4-chr5-15261-lociStitched',\n",
       "  'Gene': '4-chr5-15261-lociStitched',\n",
       "  'Region': '5:64232461-64253831',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '73034598',\n",
       "  'End': '73052840',\n",
       "  'Name': '4-chr12-5851-lociStitched',\n",
       "  'Gene': '4-chr12-5851-lociStitched',\n",
       "  'Region': '12:73034598-73052840',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '132467018',\n",
       "  'End': '132485720',\n",
       "  'Name': '3-chr5-15326-lociStitched',\n",
       "  'Gene': '3-chr5-15326-lociStitched',\n",
       "  'Region': '5:132467018-132485720',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '133073058',\n",
       "  'End': '133082092',\n",
       "  'Name': '4-chr3-12045-lociStitched',\n",
       "  'Gene': '4-chr3-12045-lociStitched',\n",
       "  'Region': '3:133073058-133082092',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '31520115',\n",
       "  'End': '31532708',\n",
       "  'Name': '4-chr16-19252-lociStitched',\n",
       "  'Gene': '4-chr16-19252-lociStitched',\n",
       "  'Region': '16:31520115-31532708',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '31965681',\n",
       "  'End': '31977486',\n",
       "  'Name': '4-chr17-5937-lociStitched',\n",
       "  'Gene': '4-chr17-5937-lociStitched',\n",
       "  'Region': '17:31965681-31977486',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '115302917',\n",
       "  'End': '115313180',\n",
       "  'Name': '2-chr10-3683-lociStitched',\n",
       "  'Gene': '2-chr10-3683-lociStitched',\n",
       "  'Region': '10:115302917-115313180',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '80250318',\n",
       "  'End': '80258194',\n",
       "  'Name': '2-chr15-3942-lociStitched',\n",
       "  'Gene': '2-chr15-3942-lociStitched',\n",
       "  'Region': '15:80250318-80258194',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '69062335',\n",
       "  'End': '69074506',\n",
       "  'Name': '2-chr11-11419-lociStitched',\n",
       "  'Gene': '2-chr11-11419-lociStitched',\n",
       "  'Region': '11:69062335-69074506',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '30585801',\n",
       "  'End': '30601933',\n",
       "  'Name': '5-chr13-11517-lociStitched',\n",
       "  'Gene': '5-chr13-11517-lociStitched',\n",
       "  'Region': '13:30585801-30601933',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '35634453',\n",
       "  'End': '35656331',\n",
       "  'Name': '4-chr2-11913-lociStitched',\n",
       "  'Gene': '4-chr2-11913-lociStitched',\n",
       "  'Region': '2:35634453-35656331',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '59943205',\n",
       "  'End': '59959082',\n",
       "  'Name': '3-chr10-22142-lociStitched',\n",
       "  'Gene': '3-chr10-22142-lociStitched',\n",
       "  'Region': '10:59943205-59959082',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '55411848',\n",
       "  'End': '55426398',\n",
       "  'Name': '4-chr4-12077-lociStitched',\n",
       "  'Gene': '4-chr4-12077-lociStitched',\n",
       "  'Region': '4:55411848-55426398',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '110444016',\n",
       "  'End': '110450207',\n",
       "  'Name': '3-chr11-3695-lociStitched',\n",
       "  'Gene': '3-chr11-3695-lociStitched',\n",
       "  'Region': '11:110444016-110450207',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '191980215',\n",
       "  'End': '191989936',\n",
       "  'Name': '2-chr1-9165-lociStitched',\n",
       "  'Gene': '2-chr1-9165-lociStitched',\n",
       "  'Region': '1:191980215-191989936',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '123183906',\n",
       "  'End': '123205711',\n",
       "  'Name': '4-chr7-20211-lociStitched',\n",
       "  'Gene': '4-chr7-20211-lociStitched',\n",
       "  'Region': '7:123183906-123205711',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '91707490',\n",
       "  'End': '91717765',\n",
       "  'Name': '2-chr2-21198-lociStitched',\n",
       "  'Gene': '2-chr2-21198-lociStitched',\n",
       "  'Region': '2:91707490-91717765',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '27172201',\n",
       "  'End': '27180858',\n",
       "  'Name': '3-chr17-16562-lociStitched',\n",
       "  'Gene': '3-chr17-16562-lociStitched',\n",
       "  'Region': '17:27172201-27180858',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '36372906',\n",
       "  'End': '36381626',\n",
       "  'Name': '4-chr5-1514-lociStitched',\n",
       "  'Gene': '4-chr5-1514-lociStitched',\n",
       "  'Region': '5:36372906-36381626',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '35236606',\n",
       "  'End': '35249041',\n",
       "  'Name': '4-chr16-11666-lociStitched',\n",
       "  'Gene': '4-chr16-11666-lociStitched',\n",
       "  'Region': '16:35236606-35249041',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '84037430',\n",
       "  'End': '84051502',\n",
       "  'Name': '2-chr18-1426-lociStitched',\n",
       "  'Gene': '2-chr18-1426-lociStitched',\n",
       "  'Region': '18:84037430-84051502',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '66228556',\n",
       "  'End': '66242422',\n",
       "  'Name': '3-chr14-6864-lociStitched',\n",
       "  'Gene': '3-chr14-6864-lociStitched',\n",
       "  'Region': '14:66228556-66242422',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '156057647',\n",
       "  'End': '156061610',\n",
       "  'Name': '1-chr4-30-lociStitched',\n",
       "  'Gene': '1-chr4-30-lociStitched',\n",
       "  'Region': '4:156057647-156061610',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '25266973',\n",
       "  'End': '25277244',\n",
       "  'Name': '3-chr7-3471-lociStitched',\n",
       "  'Gene': '3-chr7-3471-lociStitched',\n",
       "  'Region': '7:25266973-25277244',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '85495111',\n",
       "  'End': '85504135',\n",
       "  'Name': '3-chr10-11366-lociStitched',\n",
       "  'Gene': '3-chr10-11366-lociStitched',\n",
       "  'Region': '10:85495111-85504135',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '106395808',\n",
       "  'End': '106405902',\n",
       "  'Name': '2-chr8-10310-lociStitched',\n",
       "  'Gene': '2-chr8-10310-lociStitched',\n",
       "  'Region': '8:106395808-106405902',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '34366780',\n",
       "  'End': '34382033',\n",
       "  'Name': '3-chr17-5463-lociStitched',\n",
       "  'Gene': '3-chr17-5463-lociStitched',\n",
       "  'Region': '17:34366780-34382033',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '57842933',\n",
       "  'End': '57854394',\n",
       "  'Name': '4-chr15-22637-lociStitched',\n",
       "  'Gene': '4-chr15-22637-lociStitched',\n",
       "  'Region': '15:57842933-57854394',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '96858418',\n",
       "  'End': '96875214',\n",
       "  'Name': '4-chr9-20355-lociStitched',\n",
       "  'Gene': '4-chr9-20355-lociStitched',\n",
       "  'Region': '9:96858418-96875214',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '30666371',\n",
       "  'End': '30688455',\n",
       "  'Name': '3-chr6-13545-lociStitched',\n",
       "  'Gene': '3-chr6-13545-lociStitched',\n",
       "  'Region': '6:30666371-30688455',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '17049111',\n",
       "  'End': '17060713',\n",
       "  'Name': '3-chr1-7554-lociStitched',\n",
       "  'Gene': '3-chr1-7554-lociStitched',\n",
       "  'Region': '1:17049111-17060713',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '140077110',\n",
       "  'End': '140092027',\n",
       "  'Name': '4-chr4-21499-lociStitched',\n",
       "  'Gene': '4-chr4-21499-lociStitched',\n",
       "  'Region': '4:140077110-140092027',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '63918558',\n",
       "  'End': '63939051',\n",
       "  'Name': '4-chr5-17530-lociStitched',\n",
       "  'Gene': '4-chr5-17530-lociStitched',\n",
       "  'Region': '5:63918558-63939051',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '96090892',\n",
       "  'End': '96106878',\n",
       "  'Name': '4-chr10-7327-lociStitched',\n",
       "  'Gene': '4-chr10-7327-lociStitched',\n",
       "  'Region': '10:96090892-96106878',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '38857967',\n",
       "  'End': '38874927',\n",
       "  'Name': '3-chr6-1577-lociStitched',\n",
       "  'Gene': '3-chr6-1577-lociStitched',\n",
       "  'Region': '6:38857967-38874927',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '66105336',\n",
       "  'End': '66120665',\n",
       "  'Name': '5-chr5-12167-lociStitched',\n",
       "  'Gene': '5-chr5-12167-lociStitched',\n",
       "  'Region': '5:66105336-66120665',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '92168741',\n",
       "  'End': '92183394',\n",
       "  'Name': '3-chr1-9897-lociStitched',\n",
       "  'Gene': '3-chr1-9897-lociStitched',\n",
       "  'Region': '1:92168741-92183394',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '126908038',\n",
       "  'End': '126919398',\n",
       "  'Name': '4-chr8-6696-lociStitched',\n",
       "  'Gene': '4-chr8-6696-lociStitched',\n",
       "  'Region': '8:126908038-126919398',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '59416074',\n",
       "  'End': '59427126',\n",
       "  'Name': '3-chr15-22640-lociStitched',\n",
       "  'Gene': '3-chr15-22640-lociStitched',\n",
       "  'Region': '15:59416074-59427126',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '83568056',\n",
       "  'End': '83580584',\n",
       "  'Name': '3-chr9-15792-lociStitched',\n",
       "  'Gene': '3-chr9-15792-lociStitched',\n",
       "  'Region': '9:83568056-83580584',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '47906272',\n",
       "  'End': '47922220',\n",
       "  'Name': '3-chr17-9802-lociStitched',\n",
       "  'Gene': '3-chr17-9802-lociStitched',\n",
       "  'Region': '17:47906272-47922220',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '65454163',\n",
       "  'End': '65468168',\n",
       "  'Name': '3-chr9-4399-lociStitched',\n",
       "  'Gene': '3-chr9-4399-lociStitched',\n",
       "  'Region': '9:65454163-65468168',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '121078125',\n",
       "  'End': '121087874',\n",
       "  'Name': '3-chr9-18326-lociStitched',\n",
       "  'Gene': '3-chr9-18326-lociStitched',\n",
       "  'Region': '9:121078125-121087874',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '165601313',\n",
       "  'End': '165606901',\n",
       "  'Name': '3-chr2-1057-lociStitched',\n",
       "  'Gene': '3-chr2-1057-lociStitched',\n",
       "  'Region': '2:165601313-165606901',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '94857235',\n",
       "  'End': '94865092',\n",
       "  'Name': '3-chr4-417-lociStitched',\n",
       "  'Gene': '3-chr4-417-lociStitched',\n",
       "  'Region': '4:94857235-94865092',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '18897636',\n",
       "  'End': '18919429',\n",
       "  'Name': '4-chr2-23182-lociStitched',\n",
       "  'Gene': '4-chr2-23182-lociStitched',\n",
       "  'Region': '2:18897636-18919429',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '24467677',\n",
       "  'End': '24488323',\n",
       "  'Name': '4-chr13-5398-lociStitched',\n",
       "  'Gene': '4-chr13-5398-lociStitched',\n",
       "  'Region': '13:24467677-24488323',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '44597102',\n",
       "  'End': '44609629',\n",
       "  'Name': '3-chr9-8684-lociStitched',\n",
       "  'Gene': '3-chr9-8684-lociStitched',\n",
       "  'Region': '9:44597102-44609629',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '168282818',\n",
       "  'End': '168302295',\n",
       "  'Name': '3-chr1-23126-lociStitched',\n",
       "  'Gene': '3-chr1-23126-lociStitched',\n",
       "  'Region': '1:168282818-168302295',\n",
       "  'Strand': '+'}]"
      ]
     },
     "execution_count": 47,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "region_dict"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<a id='2'></a>\n",
    "# 2. Design probe targeting sequences by probe_designer\n",
    "\n",
    "run probe_designer, remeber to clear the memory usage because each table should take ~32GB"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<a id='2.1'></a>\n",
    "## 2.1 Construct count table with all the 17-mers in the genome\n",
    "\n",
    "Only do this if you don't have pre-built 17-mer\n",
    "\n",
    "However you can do almost the same thing for your own library during quality check\n",
    "\n",
    "This library requires mm10 genome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [],
   "source": [
    "overwrite_table = False"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [],
   "source": [
    "from ImageAnalysis3 import library_tools"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### construct map for whole genome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "reference table: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\GRCm38_genome_17w.npy already exist, skip.\n"
     ]
    }
   ],
   "source": [
    "reload(library_tools.design)\n",
    " \n",
    "genome_table_file = os.path.join(reference_folder, 'GRCm38_genome_17w.npy')\n",
    "\n",
    "if not os.path.exists(genome_table_file) or overwrite_table:\n",
    "    # genome\n",
    "    _genome_filenames = [os.path.join(genome_folder, _fl) \n",
    "         for _fl in os.listdir(genome_folder) \n",
    "         if _fl.split(os.extsep)[-1]=='fasta' or _fl.split(os.extsep)[-1]=='fa']\n",
    "    print(len(_genome_filenames))\n",
    "\n",
    "    ct = library_tools.design.countTable(word=17,save_file=genome_table_file, \n",
    "                       sparse=False)\n",
    "    ct.verbose=True\n",
    "\n",
    "    ct.read(_genome_filenames) # read sequences from fasta files\n",
    "\n",
    "    ct.consume_loaded(num_threads=24) # convert sequences into integers\n",
    "\n",
    "    ct.complete(verbose=True)\n",
    "\n",
    "    ct.save()\n",
    "\n",
    "    # clear RAM if contructed countable \n",
    "    del(ct)\n",
    "else:\n",
    "    print(f\"reference table: {genome_table_file} already exist, skip.\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### construct map for transcriptome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "reference table: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\GRCm38_transcriptome_17w.npy already exist, skip.\n"
     ]
    }
   ],
   "source": [
    "from tqdm import tqdm\n",
    "# transcriptome\n",
    "transcriptome_folder = os.path.join(reference_folder, 'Transcriptome')\n",
    "\n",
    "transcriptome_table_file = os.path.join(reference_folder, 'GRCm38_transcriptome_17w.npy')\n",
    "\n",
    "if not os.path.exists(transcriptome_table_file) or overwrite_table:\n",
    "    # transcriptome\n",
    "    _transcriptome_filenames = [os.path.join(transcriptome_folder, _fl) \n",
    "         for _fl in os.listdir(transcriptome_folder) \n",
    "         if _fl.split(os.extsep)[-1]=='fasta' or _fl.split(os.extsep)[-1]=='fa']\n",
    "    print(len(_transcriptome_filenames))\n",
    "    \n",
    "    ct = library_tools.design.countTable(word=17,save_file=transcriptome_table_file, \n",
    "                       sparse=False)\n",
    "    ct.verbose=True\n",
    "\n",
    "    ct.read(_transcriptome_filenames) # read sequences from fasta files\n",
    "\n",
    "    ct.consume_loaded(num_threads=24) # convert sequences into integers\n",
    "\n",
    "    ct.complete(verbose=True)\n",
    "\n",
    "    ct.save()\n",
    "    \n",
    "    # clear RAM if contructed countable \n",
    "    del(ct)\n",
    "else:\n",
    "    print(f\"reference table: {transcriptome_table_file} already exist, skip.\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### construct map for repeats from RepBase"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "reference table: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Repbase_v2603_repeat_17w.npy already exist, skip.\n"
     ]
    }
   ],
   "source": [
    "from tqdm import tqdm\n",
    "# repeat\n",
    "repeat_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\Repbase'\n",
    "\n",
    "repeat_table_file = os.path.join(reference_folder, 'Repbase_v2603_repeat_17w.npy')\n",
    "\n",
    "if not os.path.exists(repeat_table_file) or overwrite_table:\n",
    "    # repeat\n",
    "    _repeat_filenames = [os.path.join(repeat_folder, _fl) \n",
    "         for _fl in os.listdir(repeat_folder) \n",
    "         if _fl.split(os.extsep)[-1]=='fasta' or _fl.split(os.extsep)[-1]=='fa']\n",
    "    print(len(_repeat_filenames))\n",
    "    \n",
    "    ct = library_tools.design.countTable(word=17,save_file=repeat_table_file, \n",
    "                       sparse=False)\n",
    "    ct.verbose=True\n",
    "\n",
    "    ct.read(_repeat_filenames) # read sequences from fasta files\n",
    "\n",
    "    ct.consume_loaded(num_threads=24) # convert sequences into integers\n",
    "\n",
    "    ct.complete(verbose=True)\n",
    "\n",
    "    ct.save()\n",
    "    \n",
    "    # clear RAM if contructed countable \n",
    "    del(ct)\n",
    "else:\n",
    "    print(f\"reference table: {repeat_table_file} already exist, skip.\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<a id='2.2'></a>\n",
    "## 2.2 Design probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 64,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "991 regions loaded to design probes.\n",
      "\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\reports\\merged_probes.pbr\n"
     ]
    }
   ],
   "source": [
    "# requires pre_defined genome_folder and library_folder\n",
    "# Indices\n",
    "genome_index = os.path.join(reference_folder, 'GRCm38_genome_17w.npy')\n",
    "transcriptome_index = os.path.join(reference_folder, 'GRCm38_transcriptome_17w.npy') \n",
    "repeat_index = os.path.join(reference_folder, 'Repbase_v2603_repeat_17w.npy')\n",
    "#ref_merfish_index = os.path.join(reference_folder, 'M1_meng_MERFISH_17w.npy') # merfish designed by Meng\n",
    "\n",
    "\n",
    "# get input files (which are fasta for the sequences corresponding to the targetting regions)\n",
    "input_files = glob.glob(os.path.join(sequence_folder, '*.fasta'))\n",
    "\n",
    "print(f\"{len(input_files)} regions loaded to design probes.\")\n",
    "\n",
    "if not os.path.exists(report_folder):\n",
    "    os.makedirs(report_folder)\n",
    "    \n",
    "# filename to save probe reports\n",
    "save_file = os.path.join(report_folder, f'merged_probes.pbr')\n",
    "print(save_file)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Split region fasta files into subsets\n",
    "As the number of probes/regions are too large, split them into subsets by providing input fasta as subsets.\n",
    "\n",
    "#### 1. region 0-249: as merged_probes_1; done\n",
    "#### 2. region 250-499: as merged_probes_2; done\n",
    "#### 3. region 500-749: as merged_probes_3; done\n",
    "#### 4. region 750-991: as merged_probes_4; done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 65,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "241\n",
      "['\\\\\\\\10.245.74.212\\\\Chromatin_NAS_2\\\\Chromatin_Libraries\\\\CTP-13_brain-super-enhancers\\\\mop_SE_1000\\\\sequences\\\\751_reg_1.fasta', '\\\\\\\\10.245.74.212\\\\Chromatin_NAS_2\\\\Chromatin_Libraries\\\\CTP-13_brain-super-enhancers\\\\mop_SE_1000\\\\sequences\\\\752_reg_1.fasta', '\\\\\\\\10.245.74.212\\\\Chromatin_NAS_2\\\\Chromatin_Libraries\\\\CTP-13_brain-super-enhancers\\\\mop_SE_1000\\\\sequences\\\\753_reg_1.fasta', '\\\\\\\\10.245.74.212\\\\Chromatin_NAS_2\\\\Chromatin_Libraries\\\\CTP-13_brain-super-enhancers\\\\mop_SE_1000\\\\sequences\\\\754_reg_1.fasta', '\\\\\\\\10.245.74.212\\\\Chromatin_NAS_2\\\\Chromatin_Libraries\\\\CTP-13_brain-super-enhancers\\\\mop_SE_1000\\\\sequences\\\\755_reg_1.fasta', '\\\\\\\\10.245.74.212\\\\Chromatin_NAS_2\\\\Chromatin_Libraries\\\\CTP-13_brain-super-enhancers\\\\mop_SE_1000\\\\sequences\\\\756_reg_1.fasta', '\\\\\\\\10.245.74.212\\\\Chromatin_NAS_2\\\\Chromatin_Libraries\\\\CTP-13_brain-super-enhancers\\\\mop_SE_1000\\\\sequences\\\\757_reg_1.fasta', '\\\\\\\\10.245.74.212\\\\Chromatin_NAS_2\\\\Chromatin_Libraries\\\\CTP-13_brain-super-enhancers\\\\mop_SE_1000\\\\sequences\\\\758_reg_1.fasta', '\\\\\\\\10.245.74.212\\\\Chromatin_NAS_2\\\\Chromatin_Libraries\\\\CTP-13_brain-super-enhancers\\\\mop_SE_1000\\\\sequences\\\\759_reg_1.fasta', '\\\\\\\\10.245.74.212\\\\Chromatin_NAS_2\\\\Chromatin_Libraries\\\\CTP-13_brain-super-enhancers\\\\mop_SE_1000\\\\sequences\\\\760_reg_1.fasta']\n"
     ]
    }
   ],
   "source": [
    "# re-define the input files and the designer file savename\n",
    "# after this, run the boxes through until each designer has been saved\n",
    "\n",
    "#save_file = r'L:\\Shiwei\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\merged_probes_1.pbr'\n",
    "#input_files = input_files[0:250]\n",
    "\n",
    "#save_file = r'L:\\Shiwei\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\merged_probes_2.pbr'\n",
    "#input_files = input_files[250:500]\n",
    "\n",
    "#save_file = r'L:\\Shiwei\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\merged_probes_3.pbr'\n",
    "#input_files = input_files[500:750]\n",
    "\n",
    "\n",
    "save_file = r'L:\\Shiwei\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\merged_probes_4.pbr'\n",
    "input_files = input_files[750:]\n",
    "\n",
    "# test only\n",
    "#save_file = r'L:\\Shiwei\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\merged_probes_test.pbr'\n",
    "#input_files = input_files[:20]\n",
    "\n",
    "print(len(input_files))\n",
    "\n",
    "print((input_files[:10]))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### create pb_designer class (for each subset)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 66,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Probe designer derived from Bogdan Bintu:\n",
      "https://github.com/BogdanBintu/ChromatinImaging/blob/master/LibraryDesign/LibraryDesigner.py\n",
      "by Pu Zheng, 2020.11\n",
      "\n",
      "Major changes:\n",
      "    1. allow design of two strands\n",
      "    2. separate reverse_complement (rev_com) and from two strands (two_stranded) as \n",
      "    two different inputs for map_dic and sequence_dic\n",
      "    3. replace 'local_genome' with 'self_sequences' to be more explicit, and only \n",
      "    exclude the counts for the corresponding self_sequence within each input. \n",
      "\n",
      "Key information:\n",
      "    - number of input_sequence(s): 241\n",
      "    - save_file location: L:\\Shiwei\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\merged_probes_4.pbr\n",
      "\n",
      "- Fail to load from savefile: L:\\Shiwei\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\merged_probes_4.pbr, file doesn't exist.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "False"
      ]
     },
     "execution_count": 66,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# define dict parameters for each subset pb designer\n",
    "reload(library_tools)\n",
    "reload(library_tools.design)\n",
    "\n",
    "pb_designer = library_tools.design.pb_reports_class(\n",
    "    sequence_dic={'file':input_files,\n",
    "                  'rev_com':True, # design two strands\n",
    "                  'two_stranded':True},\n",
    "    map_dic={'genome':{'file':genome_index,'rev_com':False,'two_stranded':True},\n",
    "             'transcriptome':{'file':transcriptome_index,'rev_com':True,'two_stranded':False},\n",
    "             'rep_genome':{'file':repeat_index,'rev_com':False,'two_stranded':True},\n",
    "             'self_sequences':{'file':input_files,'force_list':True,'rev_com':False,'two_stranded':True},\n",
    "             #'ref_merfish':{'file':ref_merfish_index,'rev_com':False,'two_stranded':True},\n",
    "             },\n",
    "        save_file=save_file,\n",
    "    params_dic={'word_size':17,'pb_len':42,'buffer_len':2,'max_count':2**16-1,\n",
    "                'check_on_go': False, # whether automatically check probes\n",
    "                'auto': False, # whether automatically convert reference maps\n",
    "               },\n",
    "    check_dic={('genome','self_sequences'): 25,\n",
    "               'rep_genome': 0,\n",
    "               'transcriptome': 14,\n",
    "               #'ref_merfish': 14,\n",
    "               'gc':[0.25,0.85],'tm': 37+0.62*50+5,\n",
    "               }\n",
    "    )\n",
    "print(pb_designer)\n",
    "pb_designer.load_from_file(load_probes_only=True) # if has not been generated, will return False"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### calculate probe reports (for each subset)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 67,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- setting attribute: map_genome\n",
      "--- finish map_genome in 122.468s.\n",
      "-- setting attribute: map_transcriptome\n",
      "--- finish map_transcriptome in 122.021s.\n",
      "-- setting attribute: map_rep_genome\n",
      "--- finish map_rep_genome in 114.726s.\n",
      "Time(s): 359.21700525283813\n",
      "- Designing targeting sequence for 241 regions\n",
      "-- designing region: 13:114065922-114149495_:+_gene_751 region: 0, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\751_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.642s.\n",
      "in 56.505s.\n",
      "-- designing region: 13:119586445-119624951_:+_gene_752 region: 1, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\752_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "in 19.934s.\n",
      "-- designing region: 14:3298996-3368904_:+_gene_753 region: 2, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\753_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.388s.\n",
      "in 47.871s.\n",
      "-- designing region: 14:3528167-3583135_:+_gene_754 region: 3, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\754_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.340s.\n",
      "in 37.811s.\n",
      "-- designing region: 14:5449405-5512217_:+_gene_755 region: 4, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\755_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.347s.\n",
      "in 42.882s.\n",
      "-- designing region: 14:5950658-6033064_:+_gene_756 region: 5, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\756_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.448s.\n",
      "in 56.978s.\n",
      "-- designing region: 14:7284561-7335660_:+_gene_757 region: 6, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\757_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.331s.\n",
      "in 35.192s.\n",
      "-- designing region: 14:13323083-13378913_:+_gene_758 region: 7, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\758_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.376s.\n",
      "in 38.978s.\n",
      "-- designing region: 14:14043804-14108999_:+_gene_759 region: 8, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\759_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.406s.\n",
      "in 46.407s.\n",
      "-- designing region: 14:15345670-15436164_:+_gene_760 region: 9, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\760_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.401s.\n",
      "in 63.020s.\n",
      "-- designing region: 14:21333204-21412833_:+_gene_761 region: 10, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\761_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.436s.\n",
      "in 54.536s.\n",
      "-- designing region: 14:21959106-22004415_:+_gene_762 region: 11, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\762_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "in 31.531s.\n",
      "-- designing region: 14:26419598-26536458_:+_gene_763 region: 12, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\763_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.498s.\n",
      "in 82.291s.\n",
      "-- designing region: 14:29018427-29124966_:+_gene_764 region: 13, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\764_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.485s.\n",
      "in 74.416s.\n",
      "-- designing region: 14:29313773-29399124_:+_gene_765 region: 14, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\765_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.433s.\n",
      "in 68.871s.\n",
      "-- designing region: 14:31127141-31206382_:+_gene_766 region: 15, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\766_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.438s.\n",
      "in 55.029s.\n",
      "-- designing region: 14:31992648-32046083_:+_gene_767 region: 16, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\767_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.352s.\n",
      "in 36.635s.\n",
      "-- designing region: 14:32517175-32620238_:+_gene_768 region: 17, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\768_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.568s.\n",
      "in 75.247s.\n",
      "-- designing region: 14:34178487-34245598_:+_gene_769 region: 18, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\769_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.421s.\n",
      "in 46.537s.\n",
      "-- designing region: 14:40891564-40969316_:+_gene_770 region: 19, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\770_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.476s.\n",
      "in 51.884s.\n",
      "-- designing region: 14:43230240-43361198_:+_gene_771 region: 20, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\771_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.536s.\n",
      "in 87.534s.\n",
      "-- designing region: 14:45529453-45635426_:+_gene_772 region: 21, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\772_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.537s.\n",
      "in 77.676s.\n",
      "-- designing region: 14:46891206-47000060_:+_gene_773 region: 22, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\773_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.531s.\n",
      "in 73.608s.\n",
      "-- designing region: 14:48145216-48234948_:+_gene_774 region: 23, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\774_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.484s.\n",
      "in 61.677s.\n",
      "-- designing region: 14:55487236-55538412_:+_gene_775 region: 24, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\775_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.325s.\n",
      "in 35.628s.\n",
      "-- designing region: 14:57498975-57575644_:+_gene_776 region: 25, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\776_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.401s.\n",
      "in 51.726s.\n",
      "-- designing region: 14:57498975-57569853_:+_gene_777 region: 26, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\777_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.411s.\n",
      "in 48.409s.\n",
      "-- designing region: 14:57669680-57747633_:+_gene_778 region: 27, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\778_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.395s.\n",
      "in 58.640s.\n",
      "-- designing region: 14:57891541-57932428_:+_gene_779 region: 28, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\779_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.315s.\n",
      "in 27.063s.\n",
      "-- designing region: 14:58064999-58123602_:+_gene_780 region: 29, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\780_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.386s.\n",
      "in 40.216s.\n",
      "-- designing region: 14:59196105-59261023_:+_gene_781 region: 30, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\781_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.362s.\n",
      "in 44.722s.\n",
      "-- designing region: 14:59691594-59748067_:+_gene_782 region: 31, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\782_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.323s.\n",
      "in 41.332s.\n",
      "-- designing region: 14:61628986-61683707_:+_gene_783 region: 32, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\783_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.372s.\n",
      "in 42.242s.\n",
      "-- designing region: 14:61793570-61846789_:+_gene_784 region: 33, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\784_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.296s.\n",
      "in 36.163s.\n",
      "-- designing region: 14:62163770-62246386_:+_gene_785 region: 34, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\785_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.417s.\n",
      "in 57.797s.\n",
      "-- designing region: 14:62834246-62950869_:+_gene_786 region: 35, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\786_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.613s.\n",
      "in 87.465s.\n",
      "-- designing region: 14:63118357-63175661_:+_gene_787 region: 36, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\787_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.345s.\n",
      "in 39.498s.\n",
      "-- designing region: 14:65504499-65594282_:+_gene_788 region: 37, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\788_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.419s.\n",
      "in 61.055s.\n",
      "-- designing region: 14:65960189-65998537_:+_gene_789 region: 38, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\789_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "in 26.914s.\n",
      "-- designing region: 14:67878001-67932629_:+_gene_790 region: 39, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\790_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.362s.\n",
      "in 38.829s.\n",
      "-- designing region: 14:69791321-69871987_:+_gene_791 region: 40, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\791_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.445s.\n",
      "in 56.155s.\n",
      "-- designing region: 14:70292991-70355449_:+_gene_792 region: 41, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\792_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.342s.\n",
      "in 42.263s.\n",
      "-- designing region: 14:75463070-75553422_:+_gene_793 region: 42, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\793_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.532s.\n",
      "in 62.276s.\n",
      "-- designing region: 14:75860410-75925995_:+_gene_794 region: 43, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\794_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.419s.\n",
      "in 55.928s.\n",
      "-- designing region: 14:76415881-76490537_:+_gene_795 region: 44, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\795_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.487s.\n",
      "in 50.749s.\n",
      "-- designing region: 14:79478685-79573260_:+_gene_796 region: 45, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\796_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.547s.\n",
      "in 67.365s.\n",
      "-- designing region: 14:79899686-79984133_:+_gene_797 region: 46, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\797_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.404s.\n",
      "in 59.967s.\n",
      "-- designing region: 14:80227149-80288631_:+_gene_798 region: 47, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\798_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.342s.\n",
      "in 42.583s.\n",
      "-- designing region: 14:99526543-99602180_:+_gene_799 region: 48, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\799_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.447s.\n",
      "in 52.441s.\n",
      "-- designing region: 14:102774665-102854185_:+_gene_800 region: 49, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\800_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.410s.\n",
      "in 55.182s.\n",
      "-- designing region: 14:103099997-103173301_:+_gene_801 region: 50, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\801_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.431s.\n",
      "in 52.125s.\n",
      "-- designing region: 14:103414208-103482720_:+_gene_802 region: 51, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\802_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.427s.\n",
      "in 46.890s.\n",
      "-- designing region: 14:105264551-105344049_:+_gene_803 region: 52, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\803_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.450s.\n",
      "in 55.093s.\n",
      "-- designing region: 14:108781216-108854003_:+_gene_804 region: 53, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\804_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.472s.\n",
      "in 48.183s.\n",
      "-- designing region: 14:114844433-114923616_:+_gene_805 region: 54, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\805_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.490s.\n",
      "in 65.800s.\n",
      "-- designing region: 14:119193071-119288918_:+_gene_806 region: 55, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\806_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.442s.\n",
      "in 64.213s.\n",
      "-- designing region: 14:119621024-119697288_:+_gene_807 region: 56, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\807_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.476s.\n",
      "in 53.297s.\n",
      "-- designing region: 14:120779255-120908010_:+_gene_808 region: 57, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\808_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.659s.\n",
      "in 88.666s.\n",
      "-- designing region: 14:121918265-122090024_:+_gene_809 region: 58, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\809_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.845s.\n",
      "in 116.868s.\n",
      "-- designing region: 15:6509514-6581190_:+_gene_810 region: 59, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\810_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.365s.\n",
      "in 48.437s.\n",
      "-- designing region: 15:7170479-7236680_:+_gene_811 region: 60, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\811_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.362s.\n",
      "in 44.263s.\n",
      "-- designing region: 15:8375489-8461753_:+_gene_812 region: 61, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\812_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.504s.\n",
      "in 58.329s.\n",
      "-- designing region: 15:12417846-12507837_:+_gene_813 region: 62, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\813_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.543s.\n",
      "in 61.160s.\n",
      "-- designing region: 15:13071050-13175931_:+_gene_814 region: 63, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\814_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.529s.\n",
      "in 70.977s.\n",
      "-- designing region: 15:16742412-16833900_:+_gene_815 region: 64, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\815_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.430s.\n",
      "in 76.875s.\n",
      "-- designing region: 15:18847236-18992658_:+_gene_816 region: 65, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\816_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.651s.\n",
      "in 101.603s.\n",
      "-- designing region: 15:24958652-25046793_:+_gene_817 region: 66, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\817_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.399s.\n",
      "in 60.114s.\n",
      "-- designing region: 15:28171581-28259520_:+_gene_818 region: 67, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\818_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.497s.\n",
      "in 59.041s.\n",
      "-- designing region: 15:30249955-30317077_:+_gene_819 region: 68, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\819_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.418s.\n",
      "in 46.473s.\n",
      "-- designing region: 15:36478751-36528020_:+_gene_820 region: 69, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\820_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.402s.\n",
      "in 35.372s.\n",
      "-- designing region: 15:40814516-40989464_:+_gene_821 region: 70, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\821_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.909s.\n",
      "in 117.883s.\n",
      "-- designing region: 15:44711034-44779894_:+_gene_822 region: 71, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\822_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.361s.\n",
      "in 49.637s.\n",
      "-- designing region: 15:45075447-45142386_:+_gene_823 region: 72, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\823_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.445s.\n",
      "in 47.674s.\n",
      "-- designing region: 15:50654938-50758251_:+_gene_824 region: 73, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\824_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.552s.\n",
      "in 70.229s.\n",
      "-- designing region: 15:52730443-52820439_:+_gene_825 region: 74, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\825_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.501s.\n",
      "in 63.569s.\n",
      "-- designing region: 15:58158195-58230642_:+_gene_826 region: 75, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\826_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.420s.\n",
      "in 47.400s.\n",
      "-- designing region: 15:67091131-67172296_:+_gene_827 region: 76, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\827_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.420s.\n",
      "in 56.772s.\n",
      "-- designing region: 15:68684491-68749615_:+_gene_828 region: 77, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\828_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.412s.\n",
      "in 43.505s.\n",
      "-- designing region: 15:68923567-69002893_:+_gene_829 region: 78, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\829_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.596s.\n",
      "in 72.830s.\n",
      "-- designing region: 15:71879586-71959467_:+_gene_830 region: 79, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\830_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.450s.\n",
      "in 52.185s.\n",
      "-- designing region: 15:76273099-76323683_:+_gene_831 region: 80, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\831_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.308s.\n",
      "in 33.363s.\n",
      "-- designing region: 15:76434091-76478590_:+_gene_832 region: 81, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\832_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "in 30.420s.\n",
      "-- designing region: 15:80684138-80732788_:+_gene_833 region: 82, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\833_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.458s.\n",
      "in 35.770s.\n",
      "-- designing region: 15:81504063-81579541_:+_gene_834 region: 83, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\834_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.489s.\n",
      "in 54.704s.\n",
      "-- designing region: 15:81672506-81729092_:+_gene_835 region: 84, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\835_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.330s.\n",
      "in 38.971s.\n",
      "-- designing region: 15:83364966-83464933_:+_gene_836 region: 85, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\836_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.559s.\n",
      "in 71.968s.\n",
      "-- designing region: 15:88479865-88547648_:+_gene_837 region: 86, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\837_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.475s.\n",
      "in 47.835s.\n",
      "-- designing region: 15:88818050-88864719_:+_gene_838 region: 87, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\838_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.388s.\n",
      "in 30.759s.\n",
      "-- designing region: 15:92180253-92287287_:+_gene_839 region: 88, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\839_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.469s.\n",
      "in 75.282s.\n",
      "-- designing region: 15:93482810-93529982_:+_gene_840 region: 89, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\840_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "in 31.965s.\n",
      "-- designing region: 15:95034078-95096462_:+_gene_841 region: 90, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\841_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.397s.\n",
      "in 44.448s.\n",
      "-- designing region: 15:96274908-96319877_:+_gene_842 region: 91, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\842_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.356s.\n",
      "in 32.237s.\n",
      "-- designing region: 15:101815997-101884700_:+_gene_843 region: 92, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\843_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.422s.\n",
      "in 47.478s.\n",
      "-- designing region: 15:103501670-103562538_:+_gene_844 region: 93, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\844_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.393s.\n",
      "in 42.674s.\n",
      "-- designing region: 16:5140368-5205722_:+_gene_845 region: 94, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\845_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.353s.\n",
      "in 43.951s.\n",
      "-- designing region: 16:10331430-10395351_:+_gene_846 region: 95, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\846_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.448s.\n",
      "in 45.977s.\n",
      "-- designing region: 16:21147080-21202863_:+_gene_847 region: 96, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\847_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.328s.\n",
      "in 37.262s.\n",
      "-- designing region: 16:22756585-22854730_:+_gene_848 region: 97, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\848_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.449s.\n",
      "in 67.744s.\n",
      "-- designing region: 16:23642588-23704832_:+_gene_849 region: 98, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\849_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.407s.\n",
      "in 43.613s.\n",
      "-- designing region: 16:23942714-24001471_:+_gene_850 region: 99, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\850_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.572s.\n",
      "in 41.064s.\n",
      "-- designing region: 16:25973196-26088540_:+_gene_851 region: 100, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\851_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.553s.\n",
      "in 100.859s.\n",
      "-- designing region: 16:28583146-28642529_:+_gene_852 region: 101, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\852_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.419s.\n",
      "in 56.088s.\n",
      "-- designing region: 16:28813123-28919744_:+_gene_853 region: 102, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\853_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.599s.\n",
      "in 76.123s.\n",
      "-- designing region: 16:29118866-29189467_:+_gene_854 region: 103, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\854_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.464s.\n",
      "in 49.631s.\n",
      "-- designing region: 16:32431344-32553252_:+_gene_855 region: 104, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\855_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.621s.\n",
      "in 82.955s.\n",
      "-- designing region: 16:32643912-32726340_:+_gene_856 region: 105, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\856_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.456s.\n",
      "in 58.319s.\n",
      "-- designing region: 16:34957154-35017777_:+_gene_857 region: 106, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\857_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.404s.\n",
      "in 42.905s.\n",
      "-- designing region: 16:35723966-35794035_:+_gene_858 region: 107, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\858_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.463s.\n",
      "in 47.583s.\n",
      "-- designing region: 16:36610671-36680286_:+_gene_859 region: 108, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\859_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.391s.\n",
      "in 48.345s.\n",
      "-- designing region: 16:38150332-38259251_:+_gene_860 region: 109, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\860_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.495s.\n",
      "in 74.504s.\n",
      "-- designing region: 16:41514064-41562563_:+_gene_861 region: 110, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\861_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.346s.\n",
      "in 32.314s.\n",
      "-- designing region: 16:42245627-42341328_:+_gene_862 region: 111, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\862_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.457s.\n",
      "in 65.441s.\n",
      "-- designing region: 16:42875154-42969540_:+_gene_863 region: 112, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\863_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.499s.\n",
      "in 63.616s.\n",
      "-- designing region: 16:43912104-43983953_:+_gene_864 region: 113, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\864_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.418s.\n",
      "in 50.162s.\n",
      "-- designing region: 16:44200258-44301077_:+_gene_865 region: 114, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\865_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.639s.\n",
      "in 69.458s.\n",
      "-- designing region: 16:44200258-44301077_:+_gene_866 region: 115, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\866_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.590s.\n",
      "in 65.758s.\n",
      "-- designing region: 16:50679262-50747219_:+_gene_867 region: 116, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\867_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.365s.\n",
      "in 48.667s.\n",
      "-- designing region: 16:52068029-52157877_:+_gene_868 region: 117, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\868_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.559s.\n",
      "in 61.942s.\n",
      "-- designing region: 16:58434511-58529914_:+_gene_869 region: 118, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\869_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.537s.\n",
      "in 65.011s.\n",
      "-- designing region: 16:67559729-67628201_:+_gene_870 region: 119, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\870_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.464s.\n",
      "in 49.361s.\n",
      "-- designing region: 16:74079872-74201979_:+_gene_871 region: 120, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\871_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.655s.\n",
      "in 82.770s.\n",
      "-- designing region: 16:77374249-77459818_:+_gene_872 region: 121, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\872_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.485s.\n",
      "in 56.857s.\n",
      "-- designing region: 16:77620781-77677713_:+_gene_873 region: 122, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\873_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.341s.\n",
      "in 38.290s.\n",
      "-- designing region: 16:78553141-78609787_:+_gene_874 region: 123, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\874_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.445s.\n",
      "in 63.462s.\n",
      "-- designing region: 16:88117436-88198446_:+_gene_875 region: 124, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\875_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.486s.\n",
      "in 54.508s.\n",
      "-- designing region: 16:89507846-89608464_:+_gene_876 region: 125, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\876_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.466s.\n",
      "in 68.052s.\n",
      "-- designing region: 16:90915327-91010712_:+_gene_877 region: 126, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\877_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.561s.\n",
      "in 66.155s.\n",
      "-- designing region: 16:93967016-94031875_:+_gene_878 region: 127, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\878_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.392s.\n",
      "in 44.720s.\n",
      "-- designing region: 16:94279006-94324592_:+_gene_879 region: 128, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\879_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.308s.\n",
      "in 31.238s.\n",
      "-- designing region: 16:94419141-94483370_:+_gene_880 region: 129, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\880_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.451s.\n",
      "in 44.659s.\n",
      "-- designing region: 16:95361862-95460042_:+_gene_881 region: 130, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\881_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.457s.\n",
      "in 64.990s.\n",
      "-- designing region: 16:97687705-97762534_:+_gene_882 region: 131, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\882_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.462s.\n",
      "in 52.029s.\n",
      "-- designing region: 17:6758025-6822752_:+_gene_883 region: 132, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\883_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.439s.\n",
      "in 47.067s.\n",
      "-- designing region: 17:11654178-11797180_:+_gene_884 region: 133, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\884_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.560s.\n",
      "in 92.868s.\n",
      "-- designing region: 17:13808723-13929026_:+_gene_885 region: 134, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\885_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.516s.\n",
      "in 83.568s.\n",
      "-- designing region: 17:15361839-15397147_:+_gene_886 region: 135, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\886_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.361s.\n",
      "in 25.080s.\n",
      "-- designing region: 17:23516967-23561137_:+_gene_887 region: 136, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\887_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.382s.\n",
      "in 33.692s.\n",
      "-- designing region: 17:24038288-24101203_:+_gene_888 region: 137, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\888_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.419s.\n",
      "in 41.369s.\n",
      "-- designing region: 17:31799132-31853772_:+_gene_889 region: 138, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\889_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.397s.\n",
      "in 36.890s.\n",
      "-- designing region: 17:34090951-34145951_:+_gene_890 region: 139, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\890_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.568s.\n",
      "in 37.748s.\n",
      "-- designing region: 17:35516761-35560061_:+_gene_891 region: 140, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\891_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.355s.\n",
      "in 29.274s.\n",
      "-- designing region: 17:43011066-43103508_:+_gene_892 region: 141, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\892_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.551s.\n",
      "in 63.334s.\n",
      "-- designing region: 17:44188990-44272216_:+_gene_893 region: 142, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\893_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.704s.\n",
      "in 60.080s.\n",
      "-- designing region: 17:46828396-46877205_:+_gene_894 region: 143, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\894_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.353s.\n",
      "in 34.168s.\n",
      "-- designing region: 17:48353042-48410987_:+_gene_895 region: 144, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\895_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.355s.\n",
      "in 40.554s.\n",
      "-- designing region: 17:48532940-48612895_:+_gene_896 region: 145, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\896_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.491s.\n",
      "in 53.913s.\n",
      "-- designing region: 17:48761285-48835829_:+_gene_897 region: 146, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\897_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.474s.\n",
      "in 52.266s.\n",
      "-- designing region: 17:49458027-49562568_:+_gene_898 region: 147, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\898_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.476s.\n",
      "in 73.233s.\n",
      "-- designing region: 17:50115429-50204110_:+_gene_899 region: 148, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\899_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.541s.\n",
      "in 63.509s.\n",
      "-- designing region: 17:51727923-51816643_:+_gene_900 region: 149, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\900_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.544s.\n",
      "in 57.427s.\n",
      "-- designing region: 17:53567112-53638475_:+_gene_901 region: 150, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\901_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.468s.\n",
      "in 47.593s.\n",
      "-- designing region: 17:55266212-55319734_:+_gene_902 region: 151, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\902_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.419s.\n",
      "in 35.974s.\n",
      "-- designing region: 17:56872991-56939799_:+_gene_903 region: 152, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\903_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.452s.\n",
      "in 48.948s.\n",
      "-- designing region: 17:57254102-57357459_:+_gene_904 region: 153, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\904_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.469s.\n",
      "in 68.104s.\n",
      "-- designing region: 17:62630369-62740980_:+_gene_905 region: 154, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\905_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.574s.\n",
      "in 106.306s.\n",
      "-- designing region: 17:67430764-67483458_:+_gene_906 region: 155, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\906_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.340s.\n",
      "in 39.053s.\n",
      "-- designing region: 17:73209957-73282539_:+_gene_907 region: 156, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\907_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.464s.\n",
      "in 50.799s.\n",
      "-- designing region: 17:73871483-73952612_:+_gene_908 region: 157, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\908_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.572s.\n",
      "in 56.381s.\n",
      "-- designing region: 17:75432566-75547174_:+_gene_909 region: 158, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\909_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.491s.\n",
      "in 78.094s.\n",
      "-- designing region: 17:80137911-80209379_:+_gene_910 region: 159, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\910_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.480s.\n",
      "in 48.628s.\n",
      "-- designing region: 17:83530997-83606275_:+_gene_911 region: 160, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\911_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.488s.\n",
      "in 50.588s.\n",
      "-- designing region: 17:84464454-84530370_:+_gene_912 region: 161, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\912_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.436s.\n",
      "in 44.442s.\n",
      "-- designing region: 17:84850274-84953300_:+_gene_913 region: 162, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\913_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.582s.\n",
      "in 69.800s.\n",
      "-- designing region: 17:89322309-89377903_:+_gene_914 region: 163, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\914_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.315s.\n",
      "in 37.980s.\n",
      "-- designing region: 17:90044561-90126527_:+_gene_915 region: 164, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\915_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.547s.\n",
      "in 55.773s.\n",
      "-- designing region: 17:90276015-90421156_:+_gene_916 region: 165, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\916_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.614s.\n",
      "in 99.229s.\n",
      "-- designing region: 18:6929796-7011984_:+_gene_917 region: 166, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\917_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.413s.\n",
      "in 55.656s.\n",
      "-- designing region: 18:15391109-15452687_:+_gene_918 region: 167, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\918_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.432s.\n",
      "in 42.976s.\n",
      "-- designing region: 18:21151931-21203166_:+_gene_919 region: 168, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\919_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.384s.\n",
      "in 35.176s.\n",
      "-- designing region: 18:24089649-24145055_:+_gene_920 region: 169, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\920_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.367s.\n",
      "in 39.246s.\n",
      "-- designing region: 18:24408559-24500784_:+_gene_921 region: 170, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\921_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.466s.\n",
      "in 61.924s.\n",
      "-- designing region: 18:24712942-24888570_:+_gene_922 region: 171, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\922_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.864s.\n",
      "in 120.548s.\n",
      "-- designing region: 18:31929025-31995900_:+_gene_923 region: 172, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\923_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.423s.\n",
      "in 48.347s.\n",
      "-- designing region: 18:32936362-33016449_:+_gene_924 region: 173, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\924_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.392s.\n",
      "in 53.894s.\n",
      "-- designing region: 18:32936362-33016449_:+_gene_925 region: 174, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\925_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.423s.\n",
      "in 54.688s.\n",
      "-- designing region: 18:34069958-34139601_:+_gene_926 region: 175, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\926_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.427s.\n",
      "in 48.168s.\n",
      "-- designing region: 18:35472165-35550427_:+_gene_927 region: 176, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\927_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.453s.\n",
      "in 54.634s.\n",
      "-- designing region: 18:36681158-36730587_:+_gene_928 region: 177, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\928_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.386s.\n",
      "in 34.491s.\n",
      "-- designing region: 18:37139186-37201875_:+_gene_929 region: 178, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\929_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.580s.\n",
      "in 43.850s.\n",
      "-- designing region: 18:39886744-39952821_:+_gene_930 region: 179, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\930_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.385s.\n",
      "in 44.031s.\n",
      "-- designing region: 18:41736971-41828086_:+_gene_931 region: 180, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\931_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.539s.\n",
      "in 61.104s.\n",
      "-- designing region: 18:42055872-42140040_:+_gene_932 region: 181, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\932_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.425s.\n",
      "in 56.263s.\n",
      "-- designing region: 18:45568608-45662084_:+_gene_933 region: 182, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\933_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.583s.\n",
      "in 64.321s.\n",
      "-- designing region: 18:46169765-46214913_:+_gene_934 region: 183, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\934_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.428s.\n",
      "in 32.331s.\n",
      "-- designing region: 18:49582614-49668484_:+_gene_935 region: 184, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\935_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.575s.\n",
      "in 59.279s.\n",
      "-- designing region: 18:50001602-50079289_:+_gene_936 region: 185, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\936_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.544s.\n",
      "in 54.200s.\n",
      "-- designing region: 18:53219139-53335927_:+_gene_937 region: 186, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\937_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.644s.\n",
      "in 79.639s.\n",
      "-- designing region: 18:55993891-56082519_:+_gene_938 region: 187, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\938_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.471s.\n",
      "in 59.584s.\n",
      "-- designing region: 18:57239664-57307027_:+_gene_939 region: 188, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\939_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.399s.\n",
      "in 45.996s.\n",
      "-- designing region: 18:59180021-59350310_:+_gene_940 region: 189, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\940_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.773s.\n",
      "in 161.211s.\n",
      "-- designing region: 18:63719174-63787284_:+_gene_941 region: 190, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\941_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.408s.\n",
      "in 47.538s.\n",
      "-- designing region: 18:64575603-64648451_:+_gene_942 region: 191, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\942_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.442s.\n",
      "in 52.265s.\n",
      "-- designing region: 18:65843454-65945591_:+_gene_943 region: 192, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\943_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.530s.\n",
      "in 68.134s.\n",
      "-- designing region: 18:68933328-69000228_:+_gene_944 region: 193, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\944_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.372s.\n",
      "in 45.294s.\n",
      "-- designing region: 18:74461676-74530383_:+_gene_945 region: 194, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\945_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.486s.\n",
      "in 47.856s.\n",
      "-- designing region: 18:74669677-74751625_:+_gene_946 region: 195, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\946_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.397s.\n",
      "in 55.447s.\n",
      "-- designing region: 18:77958807-78042826_:+_gene_947 region: 196, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\947_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.478s.\n",
      "in 59.166s.\n",
      "-- designing region: 18:80950121-81002541_:+_gene_948 region: 197, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\948_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.418s.\n",
      "in 35.156s.\n",
      "-- designing region: 18:85278941-85379839_:+_gene_949 region: 198, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\949_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.598s.\n",
      "in 68.909s.\n",
      "-- designing region: 19:3625409-3709978_:+_gene_950 region: 199, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\950_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.509s.\n",
      "in 57.871s.\n",
      "-- designing region: 19:3822930-3906324_:+_gene_951 region: 200, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\951_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.518s.\n",
      "in 57.635s.\n",
      "-- designing region: 19:5638442-5685296_:+_gene_952 region: 201, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\952_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.336s.\n",
      "in 35.607s.\n",
      "-- designing region: 19:6276522-6315238_:+_gene_953 region: 202, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\953_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.378s.\n",
      "in 26.332s.\n",
      "-- designing region: 19:7054548-7207140_:+_gene_954 region: 203, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\954_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.762s.\n",
      "in 103.709s.\n",
      "-- designing region: 19:8544284-8597137_:+_gene_955 region: 204, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\955_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.337s.\n",
      "in 36.090s.\n",
      "-- designing region: 19:10685617-10740864_:+_gene_956 region: 205, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\956_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.317s.\n",
      "in 37.501s.\n",
      "-- designing region: 19:10930147-10970163_:+_gene_957 region: 206, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\957_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.340s.\n",
      "in 29.311s.\n",
      "-- designing region: 19:12424826-12504327_:+_gene_958 region: 207, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\958_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.410s.\n",
      "in 57.260s.\n",
      "-- designing region: 19:15780510-15825819_:+_gene_959 region: 208, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\959_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.298s.\n",
      "in 32.055s.\n",
      "-- designing region: 19:17296782-17355629_:+_gene_960 region: 209, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\960_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.423s.\n",
      "in 39.031s.\n",
      "-- designing region: 19:20518919-20570378_:+_gene_961 region: 210, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\961_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.384s.\n",
      "in 34.893s.\n",
      "-- designing region: 19:21713365-21757827_:+_gene_962 region: 211, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\962_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.489s.\n",
      "in 31.034s.\n",
      "-- designing region: 19:23553461-23620199_:+_gene_963 region: 212, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\963_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.400s.\n",
      "in 46.357s.\n",
      "-- designing region: 19:24600840-24687705_:+_gene_964 region: 213, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\964_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.531s.\n",
      "in 56.388s.\n",
      "-- designing region: 19:24944141-25013094_:+_gene_965 region: 214, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\965_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.379s.\n",
      "in 46.490s.\n",
      "-- designing region: 19:27524144-27591990_:+_gene_966 region: 215, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\966_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.472s.\n",
      "in 46.452s.\n",
      "-- designing region: 19:28355825-28409866_:+_gene_967 region: 216, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\967_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.467s.\n",
      "in 37.600s.\n",
      "-- designing region: 19:28563435-28646484_:+_gene_968 region: 217, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\968_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.501s.\n",
      "in 57.662s.\n",
      "-- designing region: 19:29856927-29928435_:+_gene_969 region: 218, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\969_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.427s.\n",
      "in 51.068s.\n",
      "-- designing region: 19:32895563-32971276_:+_gene_970 region: 219, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\970_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.504s.\n",
      "in 52.976s.\n",
      "-- designing region: 19:33213942-33295336_:+_gene_971 region: 220, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\971_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.411s.\n",
      "in 55.264s.\n",
      "-- designing region: 19:36172931-36275082_:+_gene_972 region: 221, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\972_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.578s.\n",
      "in 70.887s.\n",
      "-- designing region: 19:36718674-36759528_:+_gene_973 region: 222, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\973_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.324s.\n",
      "in 30.647s.\n",
      "-- designing region: 19:37472495-37520209_:+_gene_974 region: 223, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\974_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.397s.\n",
      "in 34.290s.\n",
      "-- designing region: 19:37965494-38041498_:+_gene_975 region: 224, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\975_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.449s.\n",
      "in 52.319s.\n",
      "-- designing region: 19:41050775-41097097_:+_gene_976 region: 225, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\976_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.308s.\n",
      "in 32.418s.\n",
      "-- designing region: 19:41281381-41355899_:+_gene_977 region: 226, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\977_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.379s.\n",
      "in 51.817s.\n",
      "-- designing region: 19:41963511-42036554_:+_gene_978 region: 227, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\978_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.498s.\n",
      "in 53.698s.\n",
      "-- designing region: 19:45861417-45967203_:+_gene_979 region: 228, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\979_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.581s.\n",
      "in 77.137s.\n",
      "-- designing region: 19:47634250-47669784_:+_gene_980 region: 229, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\980_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.337s.\n",
      "in 25.262s.\n",
      "-- designing region: 19:48639897-48747838_:+_gene_981 region: 230, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\981_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.607s.\n",
      "in 73.491s.\n",
      "-- designing region: 19:58794802-58866457_:+_gene_982 region: 231, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\982_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.412s.\n",
      "in 50.921s.\n",
      "-- designing region: 19:59015717-59063288_:+_gene_983 region: 232, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\983_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.318s.\n",
      "in 33.727s.\n",
      "-- designing region: 19:59369976-59445429_:+_gene_984 region: 233, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\984_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.429s.\n",
      "in 52.821s.\n",
      "-- designing region: X:93271165-93309819_:+_gene_985 region: 234, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\985_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.344s.\n",
      "in 26.651s.\n",
      "-- designing region: X:98979231-99088826_:+_gene_986 region: 235, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\986_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.593s.\n",
      "in 73.304s.\n",
      "-- designing region: X:101910335-102015609_:+_gene_987 region: 236, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\987_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.572s.\n",
      "in 72.592s.\n",
      "-- designing region: X:136772307-136851006_:+_gene_988 region: 237, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\988_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.476s.\n",
      "in 53.787s.\n",
      "-- designing region: X:140481839-140547985_:+_gene_989 region: 238, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\989_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.407s.\n",
      "in 96.867s.\n",
      "-- designing region: X:167122110-167191320_:+_gene_990 region: 239, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\990_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.499s.\n",
      "in 49.015s.\n",
      "-- designing region: X:169962881-170032477_:+_gene_991 region: 240, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\991_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.342s.\n",
      "in 44.458s.\n",
      "- Save reports into file: L:\\Shiwei\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\merged_probes_4.pbr\n"
     ]
    }
   ],
   "source": [
    "# OTmaps for each subset pb designer\n",
    "\n",
    "reload(library_tools)\n",
    "reload(library_tools.design)\n",
    "pb_designer.computeOTmaps() # load the tables \n",
    "pb_designer.compute_pb_report() # design candidate probes"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### check probes (for each subset)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 68,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:0 13:114065922-114149495_:+_gene_751, 165016 candidate probes\n",
      "--- 75728 probes passed check_dic selection.\n",
      "finish in 7.743s, 1082 probes kept.\n",
      "-- check region:1 13:119586445-119624951_:+_gene_752, 49610 candidate probes\n",
      "--- 10792 probes passed check_dic selection.\n",
      "finish in 1.761s, 176 probes kept.\n",
      "-- check region:2 14:3298996-3368904_:+_gene_753, 29622 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.867s, 0 probes kept.\n",
      "-- check region:3 14:3528167-3583135_:+_gene_754, 55234 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 1.378s, 0 probes kept.\n",
      "-- check region:4 14:5449405-5512217_:+_gene_755, 35116 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.874s, 0 probes kept.\n",
      "-- check region:5 14:5950658-6033064_:+_gene_756, 121030 candidate probes\n",
      "--- 114 probes passed check_dic selection.\n",
      "finish in 2.734s, 8 probes kept.\n",
      "-- check region:6 14:7284561-7335660_:+_gene_757, 99018 candidate probes\n",
      "--- 98 probes passed check_dic selection.\n",
      "finish in 2.770s, 8 probes kept.\n",
      "-- check region:7 14:13323083-13378913_:+_gene_758, 107162 candidate probes\n",
      "--- 32469 probes passed check_dic selection.\n",
      "finish in 4.542s, 511 probes kept.\n",
      "-- check region:8 14:14043804-14108999_:+_gene_759, 128152 candidate probes\n",
      "--- 51282 probes passed check_dic selection.\n",
      "finish in 5.411s, 769 probes kept.\n",
      "-- check region:9 14:15345670-15436164_:+_gene_760, 178306 candidate probes\n",
      "--- 75578 probes passed check_dic selection.\n",
      "finish in 7.712s, 1094 probes kept.\n",
      "-- check region:10 14:21333204-21412833_:+_gene_761, 148672 candidate probes\n",
      "--- 47351 probes passed check_dic selection.\n",
      "finish in 5.319s, 789 probes kept.\n",
      "-- check region:11 14:21959106-22004415_:+_gene_762, 88198 candidate probes\n",
      "--- 40131 probes passed check_dic selection.\n",
      "finish in 3.988s, 558 probes kept.\n",
      "-- check region:12 14:26419598-26536458_:+_gene_763, 230778 candidate probes\n",
      "--- 64465 probes passed check_dic selection.\n",
      "finish in 8.030s, 1161 probes kept.\n",
      "-- check region:13 14:29018427-29124966_:+_gene_764, 209896 candidate probes\n",
      "--- 95182 probes passed check_dic selection.\n",
      "finish in 9.627s, 1417 probes kept.\n",
      "-- check region:14 14:29313773-29399124_:+_gene_765, 167146 candidate probes\n",
      "--- 72117 probes passed check_dic selection.\n",
      "finish in 7.324s, 1052 probes kept.\n",
      "-- check region:15 14:31127141-31206382_:+_gene_766, 156948 candidate probes\n",
      "--- 74051 probes passed check_dic selection.\n",
      "finish in 7.460s, 1119 probes kept.\n",
      "-- check region:16 14:31992648-32046083_:+_gene_767, 104716 candidate probes\n",
      "--- 47411 probes passed check_dic selection.\n",
      "finish in 4.836s, 682 probes kept.\n",
      "-- check region:17 14:32517175-32620238_:+_gene_768, 205600 candidate probes\n",
      "--- 95588 probes passed check_dic selection.\n",
      "finish in 8.974s, 1375 probes kept.\n",
      "-- check region:18 14:34178487-34245598_:+_gene_769, 132742 candidate probes\n",
      "--- 55237 probes passed check_dic selection.\n",
      "finish in 6.328s, 874 probes kept.\n",
      "-- check region:19 14:40891564-40969316_:+_gene_770, 151836 candidate probes\n",
      "--- 76519 probes passed check_dic selection.\n",
      "finish in 7.825s, 1032 probes kept.\n",
      "-- check region:20 14:43230240-43361198_:+_gene_771, 239288 candidate probes\n",
      "--- 492 probes passed check_dic selection.\n",
      "finish in 5.186s, 36 probes kept.\n",
      "-- check region:21 14:45529453-45635426_:+_gene_772, 209036 candidate probes\n",
      "--- 72492 probes passed check_dic selection.\n",
      "finish in 9.211s, 1193 probes kept.\n",
      "-- check region:22 14:46891206-47000060_:+_gene_773, 213146 candidate probes\n",
      "--- 100791 probes passed check_dic selection.\n",
      "finish in 10.079s, 1405 probes kept.\n",
      "-- check region:23 14:48145216-48234948_:+_gene_774, 175780 candidate probes\n",
      "--- 81189 probes passed check_dic selection.\n",
      "finish in 8.465s, 1141 probes kept.\n",
      "-- check region:24 14:55487236-55538412_:+_gene_775, 100248 candidate probes\n",
      "--- 40238 probes passed check_dic selection.\n",
      "finish in 4.447s, 679 probes kept.\n",
      "-- check region:25 14:57498975-57575644_:+_gene_776, 11286 candidate probes\n",
      "--- 3459 probes passed check_dic selection.\n",
      "finish in 0.481s, 62 probes kept.\n",
      "-- check region:26 14:57498975-57569853_:+_gene_777, 136478 candidate probes\n",
      "--- 55231 probes passed check_dic selection.\n",
      "finish in 5.802s, 806 probes kept.\n",
      "-- check region:27 14:57669680-57747633_:+_gene_778, 148022 candidate probes\n",
      "--- 57235 probes passed check_dic selection.\n",
      "finish in 6.112s, 869 probes kept.\n",
      "-- check region:28 14:57891541-57932428_:+_gene_779, 78836 candidate probes\n",
      "--- 21950 probes passed check_dic selection.\n",
      "finish in 3.030s, 393 probes kept.\n",
      "-- check region:29 14:58064999-58123602_:+_gene_780, 116376 candidate probes\n",
      "--- 48694 probes passed check_dic selection.\n",
      "finish in 4.980s, 727 probes kept.\n",
      "-- check region:30 14:59196105-59261023_:+_gene_781, 126414 candidate probes\n",
      "--- 31359 probes passed check_dic selection.\n",
      "finish in 4.445s, 556 probes kept.\n",
      "-- check region:31 14:59691594-59748067_:+_gene_782, 111648 candidate probes\n",
      "--- 50289 probes passed check_dic selection.\n",
      "finish in 6.513s, 739 probes kept.\n",
      "-- check region:32 14:61628986-61683707_:+_gene_783, 108486 candidate probes\n",
      "--- 29766 probes passed check_dic selection.\n",
      "finish in 4.247s, 527 probes kept.\n",
      "-- check region:33 14:61793570-61846789_:+_gene_784, 103368 candidate probes\n",
      "--- 38091 probes passed check_dic selection.\n",
      "finish in 4.104s, 581 probes kept.\n",
      "-- check region:34 14:62163770-62246386_:+_gene_785, 161832 candidate probes\n",
      "--- 66173 probes passed check_dic selection.\n",
      "finish in 7.951s, 970 probes kept.\n",
      "-- check region:35 14:62834246-62950869_:+_gene_786, 230488 candidate probes\n",
      "--- 86381 probes passed check_dic selection.\n",
      "finish in 9.099s, 1369 probes kept.\n",
      "-- check region:36 14:63118357-63175661_:+_gene_787, 113290 candidate probes\n",
      "--- 47036 probes passed check_dic selection.\n",
      "finish in 5.559s, 712 probes kept.\n",
      "-- check region:37 14:65504499-65594282_:+_gene_788, 177746 candidate probes\n",
      "--- 81892 probes passed check_dic selection.\n",
      "finish in 8.387s, 1117 probes kept.\n",
      "-- check region:38 14:65960189-65998537_:+_gene_789, 75410 candidate probes\n",
      "--- 34060 probes passed check_dic selection.\n",
      "finish in 3.361s, 503 probes kept.\n",
      "-- check region:39 14:67878001-67932629_:+_gene_790, 108054 candidate probes\n",
      "--- 46872 probes passed check_dic selection.\n",
      "finish in 5.023s, 672 probes kept.\n",
      "-- check region:40 14:69791321-69871987_:+_gene_791, 158132 candidate probes\n",
      "--- 70322 probes passed check_dic selection.\n",
      "finish in 6.970s, 1007 probes kept.\n",
      "-- check region:41 14:70292991-70355449_:+_gene_792, 123434 candidate probes\n",
      "--- 46758 probes passed check_dic selection.\n",
      "finish in 5.919s, 763 probes kept.\n",
      "-- check region:42 14:75463070-75553422_:+_gene_793, 177882 candidate probes\n",
      "--- 79117 probes passed check_dic selection.\n",
      "finish in 8.751s, 1148 probes kept.\n",
      "-- check region:43 14:75860410-75925995_:+_gene_794, 127978 candidate probes\n",
      "--- 45067 probes passed check_dic selection.\n",
      "finish in 5.603s, 710 probes kept.\n",
      "-- check region:44 14:76415881-76490537_:+_gene_795, 147714 candidate probes\n",
      "--- 64474 probes passed check_dic selection.\n",
      "finish in 6.502s, 950 probes kept.\n",
      "-- check region:45 14:79478685-79573260_:+_gene_796, 185462 candidate probes\n",
      "--- 59089 probes passed check_dic selection.\n",
      "finish in 7.556s, 1048 probes kept.\n",
      "-- check region:46 14:79899686-79984133_:+_gene_797, 150674 candidate probes\n",
      "--- 54928 probes passed check_dic selection.\n",
      "finish in 7.116s, 869 probes kept.\n",
      "-- check region:47 14:80227149-80288631_:+_gene_798, 120220 candidate probes\n",
      "--- 49664 probes passed check_dic selection.\n",
      "finish in 5.331s, 743 probes kept.\n",
      "-- check region:48 14:99526543-99602180_:+_gene_799, 148356 candidate probes\n",
      "--- 53230 probes passed check_dic selection.\n",
      "finish in 6.104s, 823 probes kept.\n",
      "-- check region:49 14:102774665-102854185_:+_gene_800, 141308 candidate probes\n",
      "--- 46447 probes passed check_dic selection.\n",
      "finish in 6.069s, 746 probes kept.\n",
      "-- check region:50 14:103099997-103173301_:+_gene_801, 142025 candidate probes\n",
      "--- 39654 probes passed check_dic selection.\n",
      "finish in 5.464s, 735 probes kept.\n",
      "-- check region:51 14:103414208-103482720_:+_gene_802, 132720 candidate probes\n",
      "--- 46463 probes passed check_dic selection.\n",
      "finish in 5.662s, 733 probes kept.\n",
      "-- check region:52 14:105264551-105344049_:+_gene_803, 154392 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 51864 probes passed check_dic selection.\n",
      "finish in 6.835s, 886 probes kept.\n",
      "-- check region:53 14:108781216-108854003_:+_gene_804, 143200 candidate probes\n",
      "--- 27198 probes passed check_dic selection.\n",
      "finish in 4.093s, 564 probes kept.\n",
      "-- check region:54 14:114844433-114923616_:+_gene_805, 143914 candidate probes\n",
      "--- 50189 probes passed check_dic selection.\n",
      "finish in 5.564s, 815 probes kept.\n",
      "-- check region:55 14:119193071-119288918_:+_gene_806, 185422 candidate probes\n",
      "--- 64047 probes passed check_dic selection.\n",
      "finish in 7.099s, 971 probes kept.\n",
      "-- check region:56 14:119621024-119697288_:+_gene_807, 138314 candidate probes\n",
      "--- 46079 probes passed check_dic selection.\n",
      "finish in 5.777s, 734 probes kept.\n",
      "-- check region:57 14:120779255-120908010_:+_gene_808, 234166 candidate probes\n",
      "--- 74111 probes passed check_dic selection.\n",
      "finish in 9.254s, 1163 probes kept.\n",
      "-- check region:58 14:121918265-122090024_:+_gene_809, 339288 candidate probes\n",
      "--- 136789 probes passed check_dic selection.\n",
      "finish in 15.290s, 2018 probes kept.\n",
      "-- check region:59 15:6509514-6581190_:+_gene_810, 127988 candidate probes\n",
      "--- 46438 probes passed check_dic selection.\n",
      "finish in 5.491s, 724 probes kept.\n",
      "-- check region:60 15:7170479-7236680_:+_gene_811, 130090 candidate probes\n",
      "--- 57747 probes passed check_dic selection.\n",
      "finish in 6.561s, 887 probes kept.\n",
      "-- check region:61 15:8375489-8461753_:+_gene_812, 168824 candidate probes\n",
      "--- 38967 probes passed check_dic selection.\n",
      "finish in 5.521s, 767 probes kept.\n",
      "-- check region:62 15:12417846-12507837_:+_gene_813, 178606 candidate probes\n",
      "--- 84646 probes passed check_dic selection.\n",
      "finish in 8.409s, 1191 probes kept.\n",
      "-- check region:63 15:13071050-13175931_:+_gene_814, 198410 candidate probes\n",
      "--- 82098 probes passed check_dic selection.\n",
      "finish in 8.419s, 1207 probes kept.\n",
      "-- check region:64 15:16742412-16833900_:+_gene_815, 178694 candidate probes\n",
      "--- 32764 probes passed check_dic selection.\n",
      "finish in 6.160s, 674 probes kept.\n",
      "-- check region:65 15:18847236-18992658_:+_gene_816, 284288 candidate probes\n",
      "--- 64555 probes passed check_dic selection.\n",
      "finish in 9.327s, 1242 probes kept.\n",
      "-- check region:66 15:24958652-25046793_:+_gene_817, 171996 candidate probes\n",
      "--- 41151 probes passed check_dic selection.\n",
      "finish in 6.253s, 755 probes kept.\n",
      "-- check region:67 15:28171581-28259520_:+_gene_818, 160824 candidate probes\n",
      "--- 61146 probes passed check_dic selection.\n",
      "finish in 7.006s, 959 probes kept.\n",
      "-- check region:68 15:30249955-30317077_:+_gene_819, 123546 candidate probes\n",
      "--- 42464 probes passed check_dic selection.\n",
      "finish in 5.862s, 670 probes kept.\n",
      "-- check region:69 15:36478751-36528020_:+_gene_820, 96526 candidate probes\n",
      "--- 37029 probes passed check_dic selection.\n",
      "finish in 4.768s, 599 probes kept.\n",
      "-- check region:70 15:40814516-40989464_:+_gene_821, 327161 candidate probes\n",
      "--- 100045 probes passed check_dic selection.\n",
      "finish in 11.694s, 1713 probes kept.\n",
      "-- check region:71 15:44711034-44779894_:+_gene_822, 126988 candidate probes\n",
      "--- 37072 probes passed check_dic selection.\n",
      "finish in 4.627s, 644 probes kept.\n",
      "-- check region:72 15:45075447-45142386_:+_gene_823, 132012 candidate probes\n",
      "--- 22676 probes passed check_dic selection.\n",
      "finish in 4.066s, 492 probes kept.\n",
      "-- check region:73 15:50654938-50758251_:+_gene_824, 204720 candidate probes\n",
      "--- 63424 probes passed check_dic selection.\n",
      "finish in 7.487s, 1105 probes kept.\n",
      "-- check region:74 15:52730443-52820439_:+_gene_825, 152788 candidate probes\n",
      "--- 47649 probes passed check_dic selection.\n",
      "finish in 5.926s, 744 probes kept.\n",
      "-- check region:75 15:58158195-58230642_:+_gene_826, 142738 candidate probes\n",
      "--- 56727 probes passed check_dic selection.\n",
      "finish in 5.949s, 875 probes kept.\n",
      "-- check region:76 15:67091131-67172296_:+_gene_827, 158486 candidate probes\n",
      "--- 79367 probes passed check_dic selection.\n",
      "finish in 7.782s, 1099 probes kept.\n",
      "-- check region:77 15:68684491-68749615_:+_gene_828, 124686 candidate probes\n",
      "--- 48506 probes passed check_dic selection.\n",
      "finish in 6.076s, 724 probes kept.\n",
      "-- check region:78 15:68923567-69002893_:+_gene_829, 142846 candidate probes\n",
      "--- 67178 probes passed check_dic selection.\n",
      "finish in 6.422s, 951 probes kept.\n",
      "-- check region:79 15:71879586-71959467_:+_gene_830, 157948 candidate probes\n",
      "--- 73877 probes passed check_dic selection.\n",
      "finish in 6.973s, 1044 probes kept.\n",
      "-- check region:80 15:76273099-76323683_:+_gene_831, 99084 candidate probes\n",
      "--- 39054 probes passed check_dic selection.\n",
      "finish in 4.400s, 617 probes kept.\n",
      "-- check region:81 15:76434091-76478590_:+_gene_832, 88356 candidate probes\n",
      "--- 43596 probes passed check_dic selection.\n",
      "finish in 4.526s, 652 probes kept.\n",
      "-- check region:82 15:80684138-80732788_:+_gene_833, 95732 candidate probes\n",
      "--- 36484 probes passed check_dic selection.\n",
      "finish in 5.131s, 567 probes kept.\n",
      "-- check region:83 15:81504063-81579541_:+_gene_834, 145788 candidate probes\n",
      "--- 38227 probes passed check_dic selection.\n",
      "finish in 5.258s, 652 probes kept.\n",
      "-- check region:84 15:81672506-81729092_:+_gene_835, 110126 candidate probes\n",
      "--- 42538 probes passed check_dic selection.\n",
      "finish in 5.131s, 673 probes kept.\n",
      "-- check region:85 15:83364966-83464933_:+_gene_836, 196066 candidate probes\n",
      "--- 86624 probes passed check_dic selection.\n",
      "finish in 8.570s, 1230 probes kept.\n",
      "-- check region:86 15:88479865-88547648_:+_gene_837, 133094 candidate probes\n",
      "--- 57831 probes passed check_dic selection.\n",
      "finish in 5.947s, 814 probes kept.\n",
      "-- check region:87 15:88818050-88864719_:+_gene_838, 91738 candidate probes\n",
      "--- 51225 probes passed check_dic selection.\n",
      "finish in 4.839s, 648 probes kept.\n",
      "-- check region:88 15:92180253-92287287_:+_gene_839, 209906 candidate probes\n",
      "--- 70945 probes passed check_dic selection.\n",
      "finish in 10.002s, 1154 probes kept.\n",
      "-- check region:89 15:93482810-93529982_:+_gene_840, 91854 candidate probes\n",
      "--- 39949 probes passed check_dic selection.\n",
      "finish in 4.507s, 572 probes kept.\n",
      "-- check region:90 15:95034078-95096462_:+_gene_841, 118548 candidate probes\n",
      "--- 46727 probes passed check_dic selection.\n",
      "finish in 6.308s, 716 probes kept.\n",
      "-- check region:91 15:96274908-96319877_:+_gene_842, 88570 candidate probes\n",
      "--- 31479 probes passed check_dic selection.\n",
      "finish in 3.633s, 495 probes kept.\n",
      "-- check region:92 15:101815997-101884700_:+_gene_843, 135970 candidate probes\n",
      "--- 64262 probes passed check_dic selection.\n",
      "finish in 6.868s, 924 probes kept.\n",
      "-- check region:93 15:103501670-103562538_:+_gene_844, 120550 candidate probes\n",
      "--- 50702 probes passed check_dic selection.\n",
      "finish in 5.618s, 780 probes kept.\n",
      "-- check region:94 16:5140368-5205722_:+_gene_845, 128514 candidate probes\n",
      "--- 47170 probes passed check_dic selection.\n",
      "finish in 6.401s, 760 probes kept.\n",
      "-- check region:95 16:10331430-10395351_:+_gene_846, 126098 candidate probes\n",
      "--- 54385 probes passed check_dic selection.\n",
      "finish in 5.616s, 784 probes kept.\n",
      "-- check region:96 16:21147080-21202863_:+_gene_847, 110194 candidate probes\n",
      "--- 25985 probes passed check_dic selection.\n",
      "finish in 3.535s, 456 probes kept.\n",
      "-- check region:97 16:22756585-22854730_:+_gene_848, 192870 candidate probes\n",
      "--- 81725 probes passed check_dic selection.\n",
      "finish in 8.975s, 1187 probes kept.\n",
      "-- check region:98 16:23642588-23704832_:+_gene_849, 122534 candidate probes\n",
      "--- 47082 probes passed check_dic selection.\n",
      "finish in 5.825s, 715 probes kept.\n",
      "-- check region:99 16:23942714-24001471_:+_gene_850, 116106 candidate probes\n",
      "--- 48465 probes passed check_dic selection.\n",
      "finish in 5.844s, 730 probes kept.\n",
      "-- check region:100 16:25973196-26088540_:+_gene_851, 226416 candidate probes\n",
      "--- 75484 probes passed check_dic selection.\n",
      "finish in 8.880s, 1231 probes kept.\n",
      "-- check region:101 16:28583146-28642529_:+_gene_852, 117174 candidate probes\n",
      "--- 44537 probes passed check_dic selection.\n",
      "finish in 4.629s, 655 probes kept.\n",
      "-- check region:102 16:28813123-28919744_:+_gene_853, 208134 candidate probes\n",
      "--- 101637 probes passed check_dic selection.\n",
      "finish in 10.548s, 1370 probes kept.\n",
      "-- check region:103 16:29118866-29189467_:+_gene_854, 138580 candidate probes\n",
      "--- 46314 probes passed check_dic selection.\n",
      "finish in 5.656s, 724 probes kept.\n",
      "-- check region:104 16:32431344-32553252_:+_gene_855, 237958 candidate probes\n",
      "--- 79120 probes passed check_dic selection.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "finish in 9.523s, 1295 probes kept.\n",
      "-- check region:105 16:32643912-32726340_:+_gene_856, 161623 candidate probes\n",
      "--- 68754 probes passed check_dic selection.\n",
      "finish in 7.216s, 1008 probes kept.\n",
      "-- check region:106 16:34957154-35017777_:+_gene_857, 120240 candidate probes\n",
      "--- 54370 probes passed check_dic selection.\n",
      "finish in 5.232s, 771 probes kept.\n",
      "-- check region:107 16:35723966-35794035_:+_gene_858, 137756 candidate probes\n",
      "--- 48755 probes passed check_dic selection.\n",
      "finish in 5.777s, 782 probes kept.\n",
      "-- check region:108 16:36610671-36680286_:+_gene_859, 137440 candidate probes\n",
      "--- 52929 probes passed check_dic selection.\n",
      "finish in 5.940s, 821 probes kept.\n",
      "-- check region:109 16:38150332-38259251_:+_gene_860, 214936 candidate probes\n",
      "--- 69735 probes passed check_dic selection.\n",
      "finish in 8.932s, 1199 probes kept.\n",
      "-- check region:110 16:41514064-41562563_:+_gene_861, 96158 candidate probes\n",
      "--- 33992 probes passed check_dic selection.\n",
      "finish in 3.935s, 542 probes kept.\n",
      "-- check region:111 16:42245627-42341328_:+_gene_862, 185395 candidate probes\n",
      "--- 71433 probes passed check_dic selection.\n",
      "finish in 8.330s, 1123 probes kept.\n",
      "-- check region:112 16:42875154-42969540_:+_gene_863, 184858 candidate probes\n",
      "--- 45270 probes passed check_dic selection.\n",
      "finish in 6.334s, 812 probes kept.\n",
      "-- check region:113 16:43912104-43983953_:+_gene_864, 138832 candidate probes\n",
      "--- 52079 probes passed check_dic selection.\n",
      "finish in 6.028s, 859 probes kept.\n",
      "-- check region:114 16:44200258-44301077_:+_gene_865, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.010s, 0 probes kept.\n",
      "-- check region:115 16:44200258-44301077_:+_gene_866, 198300 candidate probes\n",
      "--- 72412 probes passed check_dic selection.\n",
      "finish in 8.240s, 1189 probes kept.\n",
      "-- check region:116 16:50679262-50747219_:+_gene_867, 131978 candidate probes\n",
      "--- 48305 probes passed check_dic selection.\n",
      "finish in 5.144s, 754 probes kept.\n",
      "-- check region:117 16:52068029-52157877_:+_gene_868, 175038 candidate probes\n",
      "--- 67949 probes passed check_dic selection.\n",
      "finish in 7.016s, 1040 probes kept.\n",
      "-- check region:118 16:58434511-58529914_:+_gene_869, 188860 candidate probes\n",
      "--- 67564 probes passed check_dic selection.\n",
      "finish in 7.997s, 1095 probes kept.\n",
      "-- check region:119 16:67559729-67628201_:+_gene_870, 128082 candidate probes\n",
      "--- 33001 probes passed check_dic selection.\n",
      "finish in 4.743s, 574 probes kept.\n",
      "-- check region:120 16:74079872-74201979_:+_gene_871, 235434 candidate probes\n",
      "--- 72475 probes passed check_dic selection.\n",
      "finish in 9.820s, 1237 probes kept.\n",
      "-- check region:121 16:77374249-77459818_:+_gene_872, 168928 candidate probes\n",
      "--- 47047 probes passed check_dic selection.\n",
      "finish in 6.708s, 832 probes kept.\n",
      "-- check region:122 16:77620781-77677713_:+_gene_873, 112566 candidate probes\n",
      "--- 32808 probes passed check_dic selection.\n",
      "finish in 4.010s, 570 probes kept.\n",
      "-- check region:123 16:78553141-78609787_:+_gene_874, 102246 candidate probes\n",
      "--- 34660 probes passed check_dic selection.\n",
      "finish in 3.994s, 562 probes kept.\n",
      "-- check region:124 16:88117436-88198446_:+_gene_875, 160832 candidate probes\n",
      "--- 58667 probes passed check_dic selection.\n",
      "finish in 6.589s, 921 probes kept.\n",
      "-- check region:125 16:89507846-89608464_:+_gene_876, 195386 candidate probes\n",
      "--- 75723 probes passed check_dic selection.\n",
      "finish in 8.060s, 1125 probes kept.\n",
      "-- check region:126 16:90915327-91010712_:+_gene_877, 187792 candidate probes\n",
      "--- 72295 probes passed check_dic selection.\n",
      "finish in 8.057s, 1126 probes kept.\n",
      "-- check region:127 16:93967016-94031875_:+_gene_878, 124710 candidate probes\n",
      "--- 55871 probes passed check_dic selection.\n",
      "finish in 5.426s, 775 probes kept.\n",
      "-- check region:128 16:94279006-94324592_:+_gene_879, 84578 candidate probes\n",
      "--- 29256 probes passed check_dic selection.\n",
      "finish in 3.475s, 454 probes kept.\n",
      "-- check region:129 16:94419141-94483370_:+_gene_880, 125258 candidate probes\n",
      "--- 43345 probes passed check_dic selection.\n",
      "finish in 5.916s, 730 probes kept.\n",
      "-- check region:130 16:95361862-95460042_:+_gene_881, 194460 candidate probes\n",
      "--- 92253 probes passed check_dic selection.\n",
      "finish in 8.998s, 1256 probes kept.\n",
      "-- check region:131 16:97687705-97762534_:+_gene_882, 147356 candidate probes\n",
      "--- 72448 probes passed check_dic selection.\n",
      "finish in 7.083s, 1000 probes kept.\n",
      "-- check region:132 17:6758025-6822752_:+_gene_883, 128272 candidate probes\n",
      "--- 58393 probes passed check_dic selection.\n",
      "finish in 6.109s, 817 probes kept.\n",
      "-- check region:133 17:11654178-11797180_:+_gene_884, 275389 candidate probes\n",
      "--- 121493 probes passed check_dic selection.\n",
      "finish in 12.424s, 1710 probes kept.\n",
      "-- check region:134 17:13808723-13929026_:+_gene_885, 225386 candidate probes\n",
      "--- 72749 probes passed check_dic selection.\n",
      "finish in 8.700s, 1200 probes kept.\n",
      "-- check region:135 17:15361839-15397147_:+_gene_886, 69886 candidate probes\n",
      "--- 29920 probes passed check_dic selection.\n",
      "finish in 3.849s, 423 probes kept.\n",
      "-- check region:136 17:23516967-23561137_:+_gene_887, 82768 candidate probes\n",
      "--- 25787 probes passed check_dic selection.\n",
      "finish in 4.313s, 400 probes kept.\n",
      "-- check region:137 17:24038288-24101203_:+_gene_888, 122354 candidate probes\n",
      "--- 53165 probes passed check_dic selection.\n",
      "finish in 5.566s, 786 probes kept.\n",
      "-- check region:138 17:31799132-31853772_:+_gene_889, 108230 candidate probes\n",
      "--- 57354 probes passed check_dic selection.\n",
      "finish in 6.659s, 784 probes kept.\n",
      "-- check region:139 17:34090951-34145951_:+_gene_890, 108042 candidate probes\n",
      "--- 45909 probes passed check_dic selection.\n",
      "finish in 4.922s, 729 probes kept.\n",
      "-- check region:140 17:35516761-35560061_:+_gene_891, 84700 candidate probes\n",
      "--- 33055 probes passed check_dic selection.\n",
      "finish in 3.994s, 502 probes kept.\n",
      "-- check region:141 17:43011066-43103508_:+_gene_892, 181472 candidate probes\n",
      "--- 75593 probes passed check_dic selection.\n",
      "finish in 8.032s, 1126 probes kept.\n",
      "-- check region:142 17:44188990-44272216_:+_gene_893, 164168 candidate probes\n",
      "--- 75739 probes passed check_dic selection.\n",
      "finish in 7.436s, 1083 probes kept.\n",
      "-- check region:143 17:46828396-46877205_:+_gene_894, 94920 candidate probes\n",
      "--- 34353 probes passed check_dic selection.\n",
      "finish in 3.766s, 522 probes kept.\n",
      "-- check region:144 17:48353042-48410987_:+_gene_895, 113432 candidate probes\n",
      "--- 38202 probes passed check_dic selection.\n",
      "finish in 4.891s, 662 probes kept.\n",
      "-- check region:145 17:48532940-48612895_:+_gene_896, 158478 candidate probes\n",
      "--- 65785 probes passed check_dic selection.\n",
      "finish in 7.044s, 977 probes kept.\n",
      "-- check region:146 17:48761285-48835829_:+_gene_897, 147676 candidate probes\n",
      "--- 66998 probes passed check_dic selection.\n",
      "finish in 6.568s, 928 probes kept.\n",
      "-- check region:147 17:49458027-49562568_:+_gene_898, 207220 candidate probes\n",
      "--- 95039 probes passed check_dic selection.\n",
      "finish in 9.680s, 1375 probes kept.\n",
      "-- check region:148 17:50115429-50204110_:+_gene_899, 174624 candidate probes\n",
      "--- 83885 probes passed check_dic selection.\n",
      "finish in 8.292s, 1184 probes kept.\n",
      "-- check region:149 17:51727923-51816643_:+_gene_900, 176536 candidate probes\n",
      "--- 72216 probes passed check_dic selection.\n",
      "finish in 7.412s, 1145 probes kept.\n",
      "-- check region:150 17:53567112-53638475_:+_gene_901, 127226 candidate probes\n",
      "--- 41456 probes passed check_dic selection.\n",
      "finish in 4.987s, 666 probes kept.\n",
      "-- check region:151 17:55266212-55319734_:+_gene_902, 95544 candidate probes\n",
      "--- 21904 probes passed check_dic selection.\n",
      "finish in 3.184s, 381 probes kept.\n",
      "-- check region:152 17:56872991-56939799_:+_gene_903, 132182 candidate probes\n",
      "--- 59829 probes passed check_dic selection.\n",
      "finish in 5.784s, 855 probes kept.\n",
      "-- check region:153 17:57254102-57357459_:+_gene_904, 203192 candidate probes\n",
      "--- 75617 probes passed check_dic selection.\n",
      "finish in 8.764s, 1167 probes kept.\n",
      "-- check region:154 17:62630369-62740980_:+_gene_905, 219050 candidate probes\n",
      "--- 98354 probes passed check_dic selection.\n",
      "finish in 9.337s, 1396 probes kept.\n",
      "-- check region:155 17:67430764-67483458_:+_gene_906, 104476 candidate probes\n",
      "--- 50902 probes passed check_dic selection.\n",
      "finish in 5.086s, 696 probes kept.\n",
      "-- check region:156 17:73209957-73282539_:+_gene_907, 140756 candidate probes\n",
      "--- 60388 probes passed check_dic selection.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "finish in 6.593s, 888 probes kept.\n",
      "-- check region:157 17:73871483-73952612_:+_gene_908, 157348 candidate probes\n",
      "--- 71897 probes passed check_dic selection.\n",
      "finish in 7.623s, 1039 probes kept.\n",
      "-- check region:158 17:75432566-75547174_:+_gene_909, 225966 candidate probes\n",
      "--- 85252 probes passed check_dic selection.\n",
      "finish in 10.178s, 1331 probes kept.\n",
      "-- check region:159 17:80137911-80209379_:+_gene_910, 139668 candidate probes\n",
      "--- 47501 probes passed check_dic selection.\n",
      "finish in 5.894s, 748 probes kept.\n",
      "-- check region:160 17:83530997-83606275_:+_gene_911, 147590 candidate probes\n",
      "--- 56773 probes passed check_dic selection.\n",
      "finish in 6.347s, 841 probes kept.\n",
      "-- check region:161 17:84464454-84530370_:+_gene_912, 128478 candidate probes\n",
      "--- 40638 probes passed check_dic selection.\n",
      "finish in 5.288s, 642 probes kept.\n",
      "-- check region:162 17:84850274-84953300_:+_gene_913, 193946 candidate probes\n",
      "--- 69327 probes passed check_dic selection.\n",
      "finish in 8.201s, 1054 probes kept.\n",
      "-- check region:163 17:89322309-89377903_:+_gene_914, 110378 candidate probes\n",
      "--- 33984 probes passed check_dic selection.\n",
      "finish in 5.105s, 577 probes kept.\n",
      "-- check region:164 17:90044561-90126527_:+_gene_915, 156222 candidate probes\n",
      "--- 46596 probes passed check_dic selection.\n",
      "finish in 5.613s, 786 probes kept.\n",
      "-- check region:165 17:90276015-90421156_:+_gene_916, 287021 candidate probes\n",
      "--- 73573 probes passed check_dic selection.\n",
      "finish in 9.893s, 1351 probes kept.\n",
      "-- check region:166 18:6929796-7011984_:+_gene_917, 162792 candidate probes\n",
      "--- 60630 probes passed check_dic selection.\n",
      "finish in 6.948s, 949 probes kept.\n",
      "-- check region:167 18:15391109-15452687_:+_gene_918, 121380 candidate probes\n",
      "--- 41555 probes passed check_dic selection.\n",
      "finish in 5.263s, 705 probes kept.\n",
      "-- check region:168 18:21151931-21203166_:+_gene_919, 100548 candidate probes\n",
      "--- 41742 probes passed check_dic selection.\n",
      "finish in 4.406s, 627 probes kept.\n",
      "-- check region:169 18:24089649-24145055_:+_gene_920, 109366 candidate probes\n",
      "--- 41587 probes passed check_dic selection.\n",
      "finish in 4.483s, 648 probes kept.\n",
      "-- check region:170 18:24408559-24500784_:+_gene_921, 180828 candidate probes\n",
      "--- 66267 probes passed check_dic selection.\n",
      "finish in 9.076s, 999 probes kept.\n",
      "-- check region:171 18:24712942-24888570_:+_gene_922, 346324 candidate probes\n",
      "--- 153150 probes passed check_dic selection.\n",
      "finish in 15.956s, 2217 probes kept.\n",
      "-- check region:172 18:31929025-31995900_:+_gene_923, 133202 candidate probes\n",
      "--- 60058 probes passed check_dic selection.\n",
      "finish in 7.007s, 886 probes kept.\n",
      "-- check region:173 18:32936362-33016449_:+_gene_924, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.010s, 0 probes kept.\n",
      "-- check region:174 18:32936362-33016449_:+_gene_925, 146198 candidate probes\n",
      "--- 48386 probes passed check_dic selection.\n",
      "finish in 8.131s, 786 probes kept.\n",
      "-- check region:175 18:34069958-34139601_:+_gene_926, 133400 candidate probes\n",
      "--- 46514 probes passed check_dic selection.\n",
      "finish in 5.538s, 703 probes kept.\n",
      "-- check region:176 18:35472165-35550427_:+_gene_927, 153422 candidate probes\n",
      "--- 48045 probes passed check_dic selection.\n",
      "finish in 5.764s, 765 probes kept.\n",
      "-- check region:177 18:36681158-36730587_:+_gene_928, 96388 candidate probes\n",
      "--- 26876 probes passed check_dic selection.\n",
      "finish in 3.435s, 481 probes kept.\n",
      "-- check region:178 18:37139186-37201875_:+_gene_929, 124588 candidate probes\n",
      "--- 41144 probes passed check_dic selection.\n",
      "finish in 6.279s, 724 probes kept.\n",
      "-- check region:179 18:39886744-39952821_:+_gene_930, 125732 candidate probes\n",
      "--- 44794 probes passed check_dic selection.\n",
      "finish in 5.601s, 699 probes kept.\n",
      "-- check region:180 18:41736971-41828086_:+_gene_931, 172826 candidate probes\n",
      "--- 49317 probes passed check_dic selection.\n",
      "finish in 6.498s, 801 probes kept.\n",
      "-- check region:181 18:42055872-42140040_:+_gene_932, 165654 candidate probes\n",
      "--- 67942 probes passed check_dic selection.\n",
      "finish in 8.506s, 973 probes kept.\n",
      "-- check region:182 18:45568608-45662084_:+_gene_933, 186014 candidate probes\n",
      "--- 70346 probes passed check_dic selection.\n",
      "finish in 7.270s, 1106 probes kept.\n",
      "-- check region:183 18:46169765-46214913_:+_gene_934, 88332 candidate probes\n",
      "--- 33604 probes passed check_dic selection.\n",
      "finish in 4.056s, 525 probes kept.\n",
      "-- check region:184 18:49582614-49668484_:+_gene_935, 167234 candidate probes\n",
      "--- 57990 probes passed check_dic selection.\n",
      "finish in 7.160s, 903 probes kept.\n",
      "-- check region:185 18:50001602-50079289_:+_gene_936, 153934 candidate probes\n",
      "--- 68785 probes passed check_dic selection.\n",
      "finish in 6.772s, 1000 probes kept.\n",
      "-- check region:186 18:53219139-53335927_:+_gene_937, 227208 candidate probes\n",
      "--- 86645 probes passed check_dic selection.\n",
      "finish in 10.427s, 1295 probes kept.\n",
      "-- check region:187 18:55993891-56082519_:+_gene_938, 173202 candidate probes\n",
      "--- 57726 probes passed check_dic selection.\n",
      "finish in 7.241s, 935 probes kept.\n",
      "-- check region:188 18:57239664-57307027_:+_gene_939, 131504 candidate probes\n",
      "--- 52697 probes passed check_dic selection.\n",
      "finish in 6.350s, 796 probes kept.\n",
      "-- check region:189 18:59180021-59350310_:+_gene_940, 331026 candidate probes\n",
      "--- 97531 probes passed check_dic selection.\n",
      "finish in 11.498s, 1657 probes kept.\n",
      "-- check region:190 18:63719174-63787284_:+_gene_941, 135330 candidate probes\n",
      "--- 39824 probes passed check_dic selection.\n",
      "finish in 5.214s, 688 probes kept.\n",
      "-- check region:191 18:64575603-64648451_:+_gene_942, 143296 candidate probes\n",
      "--- 56496 probes passed check_dic selection.\n",
      "finish in 6.255s, 810 probes kept.\n",
      "-- check region:192 18:65843454-65945591_:+_gene_943, 199556 candidate probes\n",
      "--- 80100 probes passed check_dic selection.\n",
      "finish in 8.407s, 1206 probes kept.\n",
      "-- check region:193 18:68933328-69000228_:+_gene_944, 128380 candidate probes\n",
      "--- 51010 probes passed check_dic selection.\n",
      "finish in 6.110s, 782 probes kept.\n",
      "-- check region:194 18:74461676-74530383_:+_gene_945, 136324 candidate probes\n",
      "--- 69174 probes passed check_dic selection.\n",
      "finish in 7.686s, 922 probes kept.\n",
      "-- check region:195 18:74669677-74751625_:+_gene_946, 159078 candidate probes\n",
      "--- 80576 probes passed check_dic selection.\n",
      "finish in 7.361s, 1055 probes kept.\n",
      "-- check region:196 18:77958807-78042826_:+_gene_947, 161284 candidate probes\n",
      "--- 64689 probes passed check_dic selection.\n",
      "finish in 6.896s, 996 probes kept.\n",
      "-- check region:197 18:80950121-81002541_:+_gene_948, 103958 candidate probes\n",
      "--- 48418 probes passed check_dic selection.\n",
      "finish in 5.589s, 704 probes kept.\n",
      "-- check region:198 18:85278941-85379839_:+_gene_949, 186437 candidate probes\n",
      "--- 60194 probes passed check_dic selection.\n",
      "finish in 8.010s, 952 probes kept.\n",
      "-- check region:199 19:3625409-3709978_:+_gene_950, 166186 candidate probes\n",
      "--- 72599 probes passed check_dic selection.\n",
      "finish in 7.677s, 994 probes kept.\n",
      "-- check region:200 19:3822930-3906324_:+_gene_951, 161652 candidate probes\n",
      "--- 59290 probes passed check_dic selection.\n",
      "finish in 6.553s, 945 probes kept.\n",
      "-- check region:201 19:5638442-5685296_:+_gene_952, 91892 candidate probes\n",
      "--- 43937 probes passed check_dic selection.\n",
      "finish in 4.286s, 662 probes kept.\n",
      "-- check region:202 19:6276522-6315238_:+_gene_953, 76590 candidate probes\n",
      "--- 40200 probes passed check_dic selection.\n",
      "finish in 4.545s, 550 probes kept.\n",
      "-- check region:203 19:7054548-7207140_:+_gene_954, 302382 candidate probes\n",
      "--- 159830 probes passed check_dic selection.\n",
      "finish in 14.842s, 2153 probes kept.\n",
      "-- check region:204 19:8544284-8597137_:+_gene_955, 105176 candidate probes\n",
      "--- 30683 probes passed check_dic selection.\n",
      "finish in 3.725s, 501 probes kept.\n",
      "-- check region:205 19:10685617-10740864_:+_gene_956, 109142 candidate probes\n",
      "--- 53734 probes passed check_dic selection.\n",
      "finish in 5.075s, 766 probes kept.\n",
      "-- check region:206 19:10930147-10970163_:+_gene_957, 77164 candidate probes\n",
      "--- 33979 probes passed check_dic selection.\n",
      "finish in 3.518s, 489 probes kept.\n",
      "-- check region:207 19:12424826-12504327_:+_gene_958, 155798 candidate probes\n",
      "--- 56276 probes passed check_dic selection.\n",
      "finish in 6.593s, 905 probes kept.\n",
      "-- check region:208 19:15780510-15825819_:+_gene_959, 88668 candidate probes\n",
      "--- 32963 probes passed check_dic selection.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "finish in 3.886s, 519 probes kept.\n",
      "-- check region:209 19:17296782-17355629_:+_gene_960, 112328 candidate probes\n",
      "--- 36758 probes passed check_dic selection.\n",
      "finish in 4.346s, 590 probes kept.\n",
      "-- check region:210 19:20518919-20570378_:+_gene_961, 93020 candidate probes\n",
      "--- 21335 probes passed check_dic selection.\n",
      "finish in 3.458s, 404 probes kept.\n",
      "-- check region:211 19:21713365-21757827_:+_gene_962, 87522 candidate probes\n",
      "--- 27386 probes passed check_dic selection.\n",
      "finish in 3.878s, 447 probes kept.\n",
      "-- check region:212 19:23553461-23620199_:+_gene_963, 131520 candidate probes\n",
      "--- 43540 probes passed check_dic selection.\n",
      "finish in 5.015s, 698 probes kept.\n",
      "-- check region:213 19:24600840-24687705_:+_gene_964, 170872 candidate probes\n",
      "--- 74472 probes passed check_dic selection.\n",
      "finish in 7.698s, 1086 probes kept.\n",
      "-- check region:214 19:24944141-25013094_:+_gene_965, 137442 candidate probes\n",
      "--- 38361 probes passed check_dic selection.\n",
      "finish in 6.264s, 686 probes kept.\n",
      "-- check region:215 19:27524144-27591990_:+_gene_966, 134150 candidate probes\n",
      "--- 52866 probes passed check_dic selection.\n",
      "finish in 6.347s, 786 probes kept.\n",
      "-- check region:216 19:28355825-28409866_:+_gene_967, 107908 candidate probes\n",
      "--- 51524 probes passed check_dic selection.\n",
      "finish in 6.157s, 744 probes kept.\n",
      "-- check region:217 19:28563435-28646484_:+_gene_968, 164778 candidate probes\n",
      "--- 73317 probes passed check_dic selection.\n",
      "finish in 7.809s, 1046 probes kept.\n",
      "-- check region:218 19:29856927-29928435_:+_gene_969, 142054 candidate probes\n",
      "--- 48238 probes passed check_dic selection.\n",
      "finish in 5.907s, 758 probes kept.\n",
      "-- check region:219 19:32895563-32971276_:+_gene_970, 149922 candidate probes\n",
      "--- 63212 probes passed check_dic selection.\n",
      "finish in 7.120s, 922 probes kept.\n",
      "-- check region:220 19:33213942-33295336_:+_gene_971, 161202 candidate probes\n",
      "--- 51576 probes passed check_dic selection.\n",
      "finish in 6.069s, 812 probes kept.\n",
      "-- check region:221 19:36172931-36275082_:+_gene_972, 202098 candidate probes\n",
      "--- 77573 probes passed check_dic selection.\n",
      "finish in 8.058s, 1197 probes kept.\n",
      "-- check region:222 19:36718674-36759528_:+_gene_973, 80686 candidate probes\n",
      "--- 24163 probes passed check_dic selection.\n",
      "finish in 3.289s, 408 probes kept.\n",
      "-- check region:223 19:37472495-37520209_:+_gene_974, 93710 candidate probes\n",
      "--- 32364 probes passed check_dic selection.\n",
      "finish in 3.900s, 510 probes kept.\n",
      "-- check region:224 19:37965494-38041498_:+_gene_975, 150412 candidate probes\n",
      "--- 63017 probes passed check_dic selection.\n",
      "finish in 6.758s, 931 probes kept.\n",
      "-- check region:225 19:41050775-41097097_:+_gene_976, 91600 candidate probes\n",
      "--- 43482 probes passed check_dic selection.\n",
      "finish in 4.922s, 609 probes kept.\n",
      "-- check region:226 19:41281381-41355899_:+_gene_977, 147874 candidate probes\n",
      "--- 72286 probes passed check_dic selection.\n",
      "finish in 8.187s, 969 probes kept.\n",
      "-- check region:227 19:41963511-42036554_:+_gene_978, 144910 candidate probes\n",
      "--- 66220 probes passed check_dic selection.\n",
      "finish in 6.767s, 907 probes kept.\n",
      "-- check region:228 19:45861417-45967203_:+_gene_979, 208864 candidate probes\n",
      "--- 69804 probes passed check_dic selection.\n",
      "finish in 8.393s, 1129 probes kept.\n",
      "-- check region:229 19:47634250-47669784_:+_gene_980, 70112 candidate probes\n",
      "--- 32485 probes passed check_dic selection.\n",
      "finish in 3.216s, 485 probes kept.\n",
      "-- check region:230 19:48639897-48747838_:+_gene_981, 214012 candidate probes\n",
      "--- 87733 probes passed check_dic selection.\n",
      "finish in 9.185s, 1329 probes kept.\n",
      "-- check region:231 19:58794802-58866457_:+_gene_982, 142650 candidate probes\n",
      "--- 71778 probes passed check_dic selection.\n",
      "finish in 7.063s, 987 probes kept.\n",
      "-- check region:232 19:59015717-59063288_:+_gene_983, 94604 candidate probes\n",
      "--- 34643 probes passed check_dic selection.\n",
      "finish in 3.830s, 521 probes kept.\n",
      "-- check region:233 19:59369976-59445429_:+_gene_984, 149608 candidate probes\n",
      "--- 66530 probes passed check_dic selection.\n",
      "finish in 8.071s, 970 probes kept.\n",
      "-- check region:234 X:93271165-93309819_:+_gene_985, 76912 candidate probes\n",
      "--- 33745 probes passed check_dic selection.\n",
      "finish in 3.525s, 517 probes kept.\n",
      "-- check region:235 X:98979231-99088826_:+_gene_986, 217776 candidate probes\n",
      "--- 100514 probes passed check_dic selection.\n",
      "finish in 10.637s, 1406 probes kept.\n",
      "-- check region:236 X:101910335-102015609_:+_gene_987, 209108 candidate probes\n",
      "--- 87774 probes passed check_dic selection.\n",
      "finish in 9.874s, 1296 probes kept.\n",
      "-- check region:237 X:136772307-136851006_:+_gene_988, 156918 candidate probes\n",
      "--- 66111 probes passed check_dic selection.\n",
      "finish in 7.519s, 983 probes kept.\n",
      "-- check region:238 X:140481839-140547985_:+_gene_989, 131729 candidate probes\n",
      "--- 51350 probes passed check_dic selection.\n",
      "finish in 5.906s, 785 probes kept.\n",
      "-- check region:239 X:167122110-167191320_:+_gene_990, 138103 candidate probes\n",
      "--- 46077 probes passed check_dic selection.\n",
      "finish in 5.461s, 799 probes kept.\n",
      "-- check region:240 X:169962881-170032477_:+_gene_991, 96216 candidate probes\n",
      "--- 5815 probes passed check_dic selection.\n",
      "finish in 2.630s, 127 probes kept.\n"
     ]
    }
   ],
   "source": [
    "# design probes for each subset pb designer\n",
    "pbs, pb_scores = pb_designer.check_probes()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# quick check probe info for the steps above\n",
    "list(_pbde.kept_probes.items())[0]"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### modify region information for probes (for each subset)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# modify the probe info keys for each subset pb designer\n",
    "# eg., change old reg_name  \":+...gene_\" to new reg_name \"_strand_+_region_X\" \n",
    "\n",
    "\n",
    "# for cand_probes\n",
    "for _pb, _info in _pbde.cand_probes.items():\n",
    "    \n",
    "    if _info['reg_name'].split('gene')[-1][0] == '_':\n",
    "        old_name = str(_info['reg_name'])\n",
    "        new_name = _info['reg_name'].replace(':+', 'strand_+')\n",
    "        \n",
    "        new_name = new_name.replace('gene','reg')\n",
    "        \n",
    "        _info['name'] = _info['name'].replace(old_name, new_name)\n",
    "        _info['reg_name'] = new_name\n",
    "        \n",
    "        new_index = int(new_name.split('reg_')[-1])\n",
    "        _info['reg_index'] = new_index\n",
    "        \n",
    "# for kept_probes        \n",
    "for _pb, _info in _pbde.kept_probes.items():\n",
    "    \n",
    "    if _info['reg_name'].split('gene')[-1][0] == '_':\n",
    "        old_name = str(_info['reg_name'])\n",
    "        new_name = _info['reg_name'].replace(':+', 'strand_+')\n",
    "        \n",
    "        new_name = new_name.replace('gene','reg')\n",
    "        \n",
    "        _info['name'] = _info['name'].replace(old_name, new_name)\n",
    "        _info['reg_name'] = new_name\n",
    "        \n",
    "        new_index = int(new_name.split('reg_')[-1])\n",
    "        _info['reg_index'] = new_index  "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# quick check probe info for the modify above\n",
    "list(_pbde.kept_probes.items())[0]"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### save probes (for each subset)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 69,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Save reports into file: L:\\Shiwei\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\merged_probes_4.pbr\n",
      "-- number of probes kept: 203208\n"
     ]
    }
   ],
   "source": [
    "# save probes for each subset pb designer\n",
    "overwrite_savefile = True \n",
    "if not os.path.exists(pb_designer.save_file) or overwrite_savefile:\n",
    "    #pb_designer.plots()\n",
    "    #pb_designer.save_csv()\n",
    "    pb_designer.save_to_file()\n",
    "print(f\"-- number of probes kept: {len(pb_designer.kept_probes)}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 70,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Time to release OTmaps: 14.727s. \n"
     ]
    }
   ],
   "source": [
    "# after finish designing, remove all loaded OT maps to freeup memory usage\n",
    "pb_designer.release_OTmaps()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Define the common pb_dict before loading each pbde to combine all results"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Inititate the common pb_dict to store all probes from pbde\n",
    "# do not reset when appending\n",
    "cand_prob_nums = []\n",
    "pb_dict = {}"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### load probes from each pbde (and modify if necessary) and add them to the common pb_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [],
   "source": [
    "from ImageAnalysis3 import library_tools"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 55,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Loading from savefile: L:\\Shiwei\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\merged_probes_4.pbr.\n",
      "- loading cand_probes from file\n",
      "- loading kept_probes from file\n",
      "- 203208 filtered probes loaded\n",
      "- loading sequence_dic from file\n",
      "- loading map_dic from file\n",
      "- loading params_dic from file\n",
      "- loading check_dic from file\n",
      "- loading save_file from file\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "True"
      ]
     },
     "execution_count": 55,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# load each pb desginer saved file, process and append to the common pb_dict sequentially\n",
    "\n",
    "#save_file = r'L:\\Shiwei\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\merged_probes_1.pbr'\n",
    "#save_file = r'L:\\Shiwei\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\merged_probes_2.pbr'\n",
    "#save_file = r'L:\\Shiwei\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\merged_probes_3.pbr'\n",
    "save_file = r'L:\\Shiwei\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\merged_probes_4.pbr'\n",
    "\n",
    "reload(library_tools)\n",
    "reload(library_tools.design)\n",
    "\n",
    "verbose=True\n",
    "\n",
    "# load\n",
    "_pbde = library_tools.design.pb_reports_class()\n",
    "_pbde.load_from_file(save_file)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# append probe info into the common pd_dict with relevant info\n",
    "\n",
    "gene_names = np.sort(np.unique([int(_v['reg_name'].split('reg_')[1]) for _s, _v in _pbde.kept_probes.items()]))\n",
    "for _gene_name in gene_names:\n",
    "    _sel_pbs = {_s:_v for _s, _v in _pbde.kept_probes.items() if int(_v['reg_name'].split('reg_')[1])==_gene_name}\n",
    "    \n",
    "    if len(_sel_pbs) > 0:\n",
    "        cand_prob_nums.append(len(_sel_pbs))\n",
    "        if verbose:\n",
    "            print(f\"-- appending region:{_gene_name} with {len(_sel_pbs)} probes\")\n",
    "        _splitted_pbde = ld.pb_reports_class()\n",
    "        _splitted_pbde.kept_probes = _sel_pbs\n",
    "        pb_dict[_gene_name] = _splitted_pbde"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 62,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "975\n"
     ]
    }
   ],
   "source": [
    "# check pb_dict\n",
    "\n",
    "print(len(pb_dict))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 69,
   "metadata": {},
   "outputs": [],
   "source": [
    "# save the common pb_dict\n",
    "pickle.dump(pb_dict, open(os.path.join(library_folder, 'common_pb_dict.pkl'), 'wb'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 72,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "975\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "{b'CCAATGGCTCTAGCTAATTATTATTTTACCGTGGTATTCTCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_52',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 52,\n",
       "  'strand': '-',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.5139860484968,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCTACACAGTTTCACACCATAAGACTGATGGAGTGTAATGAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_177',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 177,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 74.26324583685204,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTGCACACACCAAGGGACCCGTTTATAGGATAGTTATATTTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_403',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 403,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 76.35069927644616,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTCTTTGTACAACCCTACAGGTATGCATAGTCACGCAACAGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_447',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 447,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.6440029684519,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATCCACTTTGACAACAAGGCTTACATTTCAGGAGGTGAGAGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_496',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 496,\n",
       "  'strand': '-',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.50706167489221,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCTTCATTTGTCATATGTTAACTCCGCTCCCTTCAAAGTTTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_582',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 582,\n",
       "  'strand': '-',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.71657668554138,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATAGACGAAACGTCAGTTTTGCTATCAGTGTGCCCGGGTCTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_669',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 669,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.23779389603277,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACTCAGTTAGCGCACAGCTTGGCTGGAGGATTTCACAGAAGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_714',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 714,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.6169753342977,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 12,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCATAAGATGCCACAATAGGGGACGCCAAGAGCGGTTGTTAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_1193',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 1193,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 80.95701030038435,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTACCAAGTACTAGTGTACCTTAGTCCGACAGCACCGGGTCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_1241',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 1241,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 80.73795693419203,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AATGACATCGACACGAAGAAGCCAGCAGACCTCGGGACCCGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_1285',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 1285,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 83.18259143006128,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTGTTTCCCTATGATTGATGTTTATCCTTGTGCCCCTCTTCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_1343',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 1343,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 76.49802864689451,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTGAGCCTGCTGGGTTGTCCGTTTCGAATTAAGTGAAGCAGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_1419',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 1419,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 80.03217178471095,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTGTTTCCATCGTTGTCTCTTTAACAATCACTTATTGTCCCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_1465',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 1465,\n",
       "  'strand': '-',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.54391655134776,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 1,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTTCCTTTACCGGAGACAATAAGAAAGCTTATTAGGCTGCCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_1515',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 1515,\n",
       "  'strand': '-',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 77.3936204620644,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGTTGGCTTTGGAACGGCTGATTCTCTCTTCAGAACCTGCCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_1563',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 1563,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 80.81478599204331,\n",
       "  'map_genome': 41,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCTGGGTGTTTGATTGCCTTTCTGGAGTCTTTTCCCTCCCAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_1614',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 1614,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 80.1039757475902,\n",
       "  'map_genome': 48,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCCTTTGTGGTCCTATGGATAAGTCAGCCTGGGGTCAGCTTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_1664',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 1664,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 81.98219527803002,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCAGGACCCAGAGGTTATAACGTCAAGATTCCCTTTTGGTAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_1708',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 1708,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.44534335510161,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCTGACTGGATCATCGGCCCTTCCTCAAATTTGGACTCACAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_1777',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 1777,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.65348542195011,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACAGCAGGACAAGGGGCTACAGGGTCAGAACAATGGCAAGGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_1822',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 1822,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 83.71420365018815,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATTTTAGCCAAATATCACAGTCAAGCCAGGCGTAAGCCTGTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_1866',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 1866,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 78.13578254293878,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTGAGTCTCCACCCCTGGTAACCCCCCTCAGTATTGGACAGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_1962',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 1962,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 83.79668633434068,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCCTCAGTAGTAGATGTTGAGAGCTCAATTTATGTTTAGCTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_2020',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 2020,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 73.35588044671749,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTCAGAATCTCCAGAGCGAATTTCAACTCCTACACATGGGCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_2090',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 2090,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.55926008127074,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGAAAGGCCACCTCTATACACACAGATCTAATTCATTCCTCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_2147',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 2147,\n",
       "  'strand': '-',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 75.73576120318518,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATGGACAACTGGTTCACCACTAGCAATGCAACCCCACCCTAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_2234',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 2234,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 82.10933589013803,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 1,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTATAGGTCTTATGTGGTAGATCAGCGGTATAGCTCTTGCTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_2283',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 2283,\n",
       "  'strand': '-',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 75.58151141236624,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 27},\n",
       " b'AGTTCCAAGTATGCTACTAAGCATAGATTGGTTTTCAGCTTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_2351',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 2351,\n",
       "  'strand': '-',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.82072448145374,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GATTTTATAACACATGGGTCCTTCTAGTCTTGTTTGCATGTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_2438',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 2438,\n",
       "  'strand': '-',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.28250581319537,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTTCCAAAAATTAACAGTGAGATTAGACCCCTTCTCACCTTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_2523',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 2523,\n",
       "  'strand': '-',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 74.03471645972581,\n",
       "  'map_genome': 48,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATAGATATGAGCCACTCTTCTCAGTGGAAGCAGTCATAATCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_2569',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 2569,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 74.14973744972463,\n",
       "  'map_genome': 44,\n",
       "  'map_transcriptome': 6,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTATGCCCCAGTTGGCCTTGGACCTGACAATTCTCAGCTTTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_2618',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 2618,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 80.64159784270453,\n",
       "  'map_genome': 49,\n",
       "  'map_transcriptome': 3,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGTGGGAACTCTTGCCCGGAGAGATTATTTAGCTGGGCCAAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_2769',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 2769,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 81.68089954498981,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTATCAGTCTATGTATTCCCCATGAGAATCCTGGAAGCTAAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_2833',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 2833,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 74.77128470950322,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTTAATGGTGTGTGATTGTACCAACTCCATTACAACTGTTCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_2877',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 2877,\n",
       "  'strand': '-',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.73559825882921,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATTCTTCCTAGAAAGTAACTCCCCGGCAGTGAAAACTTAGTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_3030',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 3030,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 76.08294000933063,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGGGTGAAGCAAGGCATGGAAAGAGATGTGCTTCACCCACTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_3096',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 3096,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 80.99163743387658,\n",
       "  'map_genome': 50,\n",
       "  'map_transcriptome': 9,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGAGAGCATCATTCTGTCCGTGGGAGCTCTCTGCTCAAAGGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_3152',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 3152,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 80.18998880442734,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGAAAGCAACAGATAGCCCTTATTACAGATCACCCAAAGTAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_3212',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 3212,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.28723479808309,\n",
       "  'map_genome': 26,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAACACAAGTTGGCTAGAGTTTGTCTAACAAGGTCTGAGCCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_3258',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 3258,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.35175813979606,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGAGGCTATGTATGAGCCATAGGCACCAATGAATGATCACT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_3318',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 3318,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.03181567407523,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGCAGTGTAGCCTGGATATCCTAGTGGTTAGAACTGAAAGCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_3385',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 3385,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.126267902498,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGGGGTACCAGCACTTGGTAAGTTACGGTGGGAGAATCATGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_3453',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 3453,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 81.37798435779314,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGCCCTATCTCATAAAGAATGTGGGGCTTTGGCTCAGTAAGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_3524',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 3524,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.9542817264113,\n",
       "  'map_genome': 41,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAAACTGATGAGCTCTCAGTCGATGAGGAGTTCAGTGAGAGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_3711',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 3711,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 75.45177723286872,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGGGTCACAACATTCACTCAGTTCTTGGAAGAATGGCGGCCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_3956',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 3956,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 80.70050607529993,\n",
       "  'map_genome': 47,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGCAGTGCTGTGGGATACGTGCGGCATGGCAACTCAGGGTGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_4005',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 4005,\n",
       "  'strand': '+',\n",
       "  'gc': 0.6190476190476191,\n",
       "  'tm': 86.23421388311732,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGATATGCTAATGGTTGAGCCTGATGGTGATGGCTAGTCAGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_4054',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 4054,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.19578878037134,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCAGGTTGATCCCAACACAGGTACACTGGAAAGAGAAGTGGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_4100',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 4100,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.99903351873513,\n",
       "  'map_genome': 48,\n",
       "  'map_transcriptome': 2,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCTACACTGTTTAACAGACCTTGTAACATTGGCAGTAGAAAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_4144',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 4144,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 74.1920890564067,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAACCATCAGCCAAGTCTAAAGCTAATGTGTGTCCTCTTCAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_4220',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 4220,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.57113285498127,\n",
       "  'map_genome': 43,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTTGTACCAGTACTTAATTCTAACAAGTTCCAAGAGAAGGGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_4350',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 4350,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 74.47700461911364,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGGAGGGAATAATGAGATGGGCAGGCAGCAAAAAGTTCCTA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_4410',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 4410,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 79.40424949982372,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGTCACAGCCTTCCATCGGTAGACATTACACCTGGTTCTTCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_4463',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 4463,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.71378412615275,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TACTATTATTTCCTGGGTACACACTCTTTGCCAGGTTCTGAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_4532',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 4532,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.24319367839121,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTTGACATAGCCCTCTAATGTGAGTATCACTATCATCCCTGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_4606',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 4606,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.17544234576775,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGAGAGAGTGAGAAAACTGCCTAGCCTGTCAAGTTCAGATTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_4672',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 4672,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 75.56512694791769,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATCCAACTGGAGGAAGCGGGAGTCGCTAACTCCAAGCTTCAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_4716',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 4716,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 81.69806778880411,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACATCAGGGTGTGGTTCCCATTATCCTCTGCAAAGCCTTGTA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_4761',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 4761,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 80.15336308925237,\n",
       "  'map_genome': 47,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCTGTGATCTGTGTTAAAAAGCCAATTCAGACCAGGCGGTGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_4807',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 4807,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.51188940791502,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTTTCTAATCAGGATGGAGACATGGTAGGATTTTGATTCCAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_5384',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 5384,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.62927494070846,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCACAGCAATATGGAAGGTATATGACTGGGGTTAGGGTGAGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_5430',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 5430,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 78.68979498796307,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTATATGCTCCCCTGATGTGAGGAACTTCGAGTAGTCGACCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_5474',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 5474,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 78.57421062667447,\n",
       "  'map_genome': 26,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTTGGCTGGGCCGAGGTGATCTTACTATAGTAGTAGTCACAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_5559',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 5559,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 78.31902218633081,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGCAGGCAGACACTCTCACAACTAGCTATATCTTTTCCTACT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_5827',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 5827,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.45634218932275,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTGGTCTGTATACCAGTGTGGTTAAATAGCAACATTTACAGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_5940',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 5940,\n",
       "  'strand': '-',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 74.13443465331687,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTTCAGGGCAATGAAAAAGTCAGATCGATGGTGGCAGTGGTA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_6001',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 6001,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.66466809328915,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGCCCCCCAGCCCCGGGAATGACCAGTCTACTTTGTGTCTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_6067',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 6067,\n",
       "  'strand': '+',\n",
       "  'gc': 0.6190476190476191,\n",
       "  'tm': 85.83044980355265,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGATACTTCATATAGGTAAACTTGAGAGCATGGACGAGATCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_6127',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 6127,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.10469455481234,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAGGCTGAGCAATGTTCCAGTGTGTGGGCATACCATATTTGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_6234',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 6234,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 79.02896917245727,\n",
       "  'map_genome': 44,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCTGTTGATGGGGGCTAGATTCTTGGCAACCTTTAGCTATTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_6287',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 6287,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 78.1521179687204,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATTTTGTTGTAGTCAAGCCAAGTGACATCTTAAGTTCCAGTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_6346',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 6346,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.58471163178194,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGCCCTCAGGTCTGTACTATAGCCTTACAGGCATTTTCTAAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_6439',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 6439,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.68233699813868,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGTCAAGAGCTCTCACCACCTCTTGGCCAAGAATGTAAGGAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_6525',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 6525,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.7218014009436,\n",
       "  'map_genome': 44,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTCCCTCTCCCCTGATCCCAAGGATGAAGGGGCAAGGGGTGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_6604',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 6604,\n",
       "  'strand': '+',\n",
       "  'gc': 0.6190476190476191,\n",
       "  'tm': 87.18595883861985,\n",
       "  'map_genome': 48,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAAATAACTGACAAATATCAAGCCTATGCCCAATAGTGCCCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_6682',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 6682,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 76.72657078970616,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAATGTCCTCACTATGCCATTTGCCCCGGGACAGCAAAGACT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_6726',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 6726,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 81.53214581101435,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATATGTAATGTTACAGATGCCTAGAGCAGGATGGTACATCCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_6797',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 6797,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.69569893664641,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATCTGTTTATTCTGCTTGGAGCCCTATGTGACAAGAAATTGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_6890',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 6890,\n",
       "  'strand': '-',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 74.61199576376339,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTTGCCTGCAGCCTCATCAACATGGGCTTTCAGTTTTAAGTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_7008',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 7008,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 77.95045474947051,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGAATGTGTAGGCAACATAAAGGTAGTGCATTAAAAATACCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_8187',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 8187,\n",
       "  'strand': '-',\n",
       "  'gc': 0.33333333333333337,\n",
       "  'tm': 73.33723203974296,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAGGCTAACGGTGCTTTGCATCTTTCACTTTACCTGCACATT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_8231',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 8231,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.77545779687773,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CATCTGCACCATGTCTCCGAAGGAAGGTGATAATCTGCTGTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_8293',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 8293,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 77.87282935636375,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AATACATGTACATTGATCATCCTGATCAGGGCGGAGTAGCCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_8483',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 8483,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 78.01901807454408,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGAGATTTTTGTAATGGATACATGCTGATTTGGGAACTGGAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_8553',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 8553,\n",
       "  'strand': '-',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.92238125809934,\n",
       "  'map_genome': 41,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AACAAGTAAACAAAGTCATGTATCCAGGTTCCAGTCACATGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_8696',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 8696,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.00008379389868,\n",
       "  'map_genome': 44,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTAAAGCATGCTGTGAAGCAATATATTAGGGTCCCGTGTCTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_8812',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 8812,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 76.54525772097566,\n",
       "  'map_genome': 41,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGGTGTATTTCTCCTTGCATCCATCCGGACTTATCAGGACAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_8869',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 8869,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.0719084736254,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTGTGTAAGATACTGGAGATCATCCAAGCTTCCACCCACAGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_8932',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 8932,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.34644237788001,\n",
       "  'map_genome': 51,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGAAAACGTTTTCCATGTACAGGAAGTACAGTTCTACTGATC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_8988',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 8988,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.04635264404271,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAATGCTGCAGGCAGAATCTCATGATGGACTCAGGCTTGCAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_9032',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 9032,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.97456794169108,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCCTGCTGCCACCCAACCCAAGTCCCATGCGCCCCCCACCCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_9092',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 9092,\n",
       "  'strand': '+',\n",
       "  'gc': 0.7380952380952381,\n",
       "  'tm': 94.40988071442985,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGAAGATGTATAGAGTTATTGACCACTCAACAGTTTTCCCTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_9279',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 9279,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.07977881202851,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGTCCTTATATAAAGCTACACAACTCACATAGAAGCAAGCAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_9342',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 9342,\n",
       "  'strand': '-',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.10026625978225,\n",
       "  'map_genome': 48,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGAAGGTGTGTTGGTCGTGAGGTCCTGACCTCACTCTGATCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_9406',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 9406,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 80.61403919603163,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTTGTTTGCAGGACAGTGGGTGTGTGCGAAAGAGGGGAGTGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_9492',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 9492,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 82.76325328186459,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCCAGATGCCCACCCAAGCATGCGCCGTCCTCCCGCATCTGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_9576',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 9576,\n",
       "  'strand': '-',\n",
       "  'gc': 0.6904761904761905,\n",
       "  'tm': 89.30679592676375,\n",
       "  'map_genome': 41,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATCGTTCAAGCACAGCTAGCCTTCATGACCAGAGACAGTCTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_9906',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 9906,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 78.01205592882218,\n",
       "  'map_genome': 48,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAACCCCGGTGGGATATGGTGGGTGTGAATTTCCACCTACAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_10314',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 10314,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 82.78465364962938,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAGTTAGAAAACTTTACTGCAGGGGTTTAGGCCTTATCCAGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_10430',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 10430,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 77.08293937297634,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGTTTTGTGGTTAAATCCAGGACTTTCTAAATGGAGGAGCTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_10478',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 10478,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.51330522898417,\n",
       "  'map_genome': 44,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTGGGCAGGGCAGCTGAAGTCTGATCCAGCCTGGTGGTACAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_10531',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 10531,\n",
       "  'strand': '+',\n",
       "  'gc': 0.6190476190476191,\n",
       "  'tm': 84.92768969158703,\n",
       "  'map_genome': 50,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGAAGCCAGTGGGCTAGAAGTTCGAAGATATCCTCAGTTACA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_10599',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 10599,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.00261514799479,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CACACATATATCCTTATGTAGTCTGTGTAGCCTCCAGCTCCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_10643',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 10643,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.7544104716697,\n",
       "  'map_genome': 48,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTGTGTGTATACGGAGCTTGGACTCTGGCTTGACCCAAAATG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_10718',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 10718,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.01882910528553,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGCATTTAGTCACTAACAGTAAGCATATGTTATGGGTTTTGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_10818',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 10818,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.4998568312908,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTGTTCAACCACAGCTATGACCTGGCTTATCACTGAAACCCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_10987',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 10987,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 78.51729354225188,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTTTAAAAGCCCATTTACTCTTCTTCGGCATCCCCCTAAACC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_11114',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 11114,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 78.23079706108723,\n",
       "  'map_genome': 44,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTTTGGCAGGCCCTCTTCAATTCACCACCCAGAACTATAGTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_11158',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 11158,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 79.4366661027941,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGGAGCTGACACAGAATGGGCCCTTGCAGAGCAGCTGTGTGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_11254',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 11254,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5952380952380952,\n",
       "  'tm': 84.04378031667403,\n",
       "  'map_genome': 45,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGGGGAGGATGGGGCATTATGGAATACTGGCGCTTCTAATGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_11326',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 11326,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 82.02067504353988,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATGAATGACTCTGTGTCCGGAAATCGATAAAATATGCAGAGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_11397',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 11397,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.80544832411903,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCCTGGCTGCCAAGATTCCCTTCATAACTCCCCGTATTAAGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_11444',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 11444,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 80.96864048169567,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCCCTGAGGGTACTGAGGGGCTGCTGTACTTAAATCATTGAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_11532',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 11532,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 79.79725982443608,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCTGAGCATGTGAATGTTGGTATCTGTGGACATCTTGGAACC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_11576',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 11576,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.48464913384572,\n",
       "  'map_genome': 48,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGATATTGGCATGTTATGAAAACCATGGCATGCCATCTGCTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_11705',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 11705,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 77.06821853712171,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCAGTGGAGATTCAGGGTTAGGAATGTCATACAATACTTTGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_11775',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 11775,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 74.81695505657154,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGATTTGTGTTGGCTGACCGACTCCTGGGGGAGGCCCACTGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_11927',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 11927,\n",
       "  'strand': '+',\n",
       "  'gc': 0.6190476190476191,\n",
       "  'tm': 86.6371095413308,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTCCTCATGGTTGTCAATTCTGGGTGGCCTCTGGGCTAGTGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_12003',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 12003,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 82.33529087586447,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTCTGGACTAGAGCTTGTGCATGCCGCACAGTCTCTATAGAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_12080',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 12080,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 78.18736594180854,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CATATTCACCACTACTCCCATTCCTAACTCATCCAGTTTCAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_12241',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 12241,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 74.80854092212161,\n",
       "  'map_genome': 48,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACCCATAGCACGAAGGCAGTACAAAGCTGACTTAGTAGGTTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_12298',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 12298,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.57640107317695,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTGAAGCATGGGTGATAGCATTGAAGAAAACACGCTCTTCCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_12353',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 12353,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.95073792562044,\n",
       "  'map_genome': 51,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAAGCCATCAACTGTCGATGGCACTTCGGCAATGGGGGAGGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_12401',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 12401,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 83.73449585492517,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGTCCTATCCAGAAAGGGCACTGCTTACTCTCGTCCTCCCTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_12512',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 12512,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 82.12757423883647,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCCAGGATCCCAGTTACTGATGTACTGTTTATGGTTGAGCTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_12605',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 12605,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.7200364983745,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCAGATGATGGTAATGCAACAACCACCTTAGCCTTTAGCACC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_12649',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 12649,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.7410005572309,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACCCTGGTCCAGGCTCACCTCGCTGTGGCCATTTATTCATTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_12709',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 12709,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 82.86413147204473,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTCCCTAGTGGACATTAGGTGGGTGCGGTCCCCAGGCGCTGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_12758',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 12758,\n",
       "  'strand': '+',\n",
       "  'gc': 0.6428571428571429,\n",
       "  'tm': 88.5446047996366,\n",
       "  'map_genome': 26,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCAGGTGGTGATCGAGTCCGACCTGTACCCGCCACCAAGGCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_12837',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 12837,\n",
       "  'strand': '+',\n",
       "  'gc': 0.6666666666666667,\n",
       "  'tm': 87.73465612665115,\n",
       "  'map_genome': 26,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGGAGCTACTGCCTCAACGCTGTGAGCGCAGGGACACAGGTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_12881',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 12881,\n",
       "  'strand': '+',\n",
       "  'gc': 0.6190476190476191,\n",
       "  'tm': 84.87418553133108,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGGCCACCCGGGGCGCATCCCACCAAAACGCCCTCCAGACCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_12961',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 12961,\n",
       "  'strand': '+',\n",
       "  'gc': 0.6904761904761905,\n",
       "  'tm': 92.57359817835442,\n",
       "  'map_genome': 26,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TAGGACACCTATAAGCTGTTACTGCGCTTTTGCTACATCACT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_13048',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 13048,\n",
       "  'strand': '-',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.81345269100012,\n",
       "  'map_genome': 26,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCTGGCAGACTTATTATTGAACATCCGACTTTGGGGGGTGGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_13092',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 13092,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 81.70515573169712,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAAGGTACCTGCTATTGGATCTGACCTACAGCAACAGGAGCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_13138',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 13138,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 79.28507853419643,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTTCACCTACACTGTTGCCACCTCCTTAGGTGCAGTGTGAGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_13187',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 13187,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 80.43949894160284,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AACAGCGACAGCTACCTTTGCAGCACGTGGCACACAATCATC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_13232',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 13232,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 80.52437834882619,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTGCTTCTTCCTAGTCCCATGGCTCTTGCTCAGGGCACAATG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_13288',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 13288,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 81.27562961286503,\n",
       "  'map_genome': 51,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TACACCTCCATCAGCCCTACACACGTGCGCACATACGCGCGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_13397',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 13397,\n",
       "  'strand': '-',\n",
       "  'gc': 0.6190476190476191,\n",
       "  'tm': 85.42315277552638,\n",
       "  'map_genome': 41,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACCACAGTTATAACAACCTAGCTGTACCTTTGGGGTAGGGTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_13477',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 13477,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 79.93741192246222,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCAGTCCATGGGAAAGCAGGTGCAGGTCTATCTCTGTAAGTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_13585',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 13585,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.53887340555457,\n",
       "  'map_genome': 43,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTGGAAACAGTGATGTTACCTTGGATACTAGTTGTTTGGATT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_13737',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 13737,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.98727928489444,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGATTCAGCATCTAGCTTACTTGACCTTTCTCGTAGTATTTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_13818',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 13818,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.23466551186306,\n",
       "  'map_genome': 51,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCATTTTTTTCTGTTGTGCCAACAACGTAAAGTTACTCATGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_13880',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 13880,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.38834425036373,\n",
       "  'map_genome': 47,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTTTAAGGTCAGCTTCGGGAAGATAGCCTACCTCAAGAACAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_14005',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 14005,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.35268815262089,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGTCTGGAGATTGGACTTAGGCCCTGATATTTGTATCACAAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_14091',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 14091,\n",
       "  'strand': '-',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.51086390265942,\n",
       "  'map_genome': 41,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACATAGAAAGCTAGGCGTGATTCCAACCCAGTACTGGAGCTA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_14135',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 14135,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 79.31150277489081,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACTCCCTGGCCAGCCTAACATGCAAGCTTCAGATCTCAGTGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_14209',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 14209,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 81.86291689604025,\n",
       "  'map_genome': 43,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATTGTTGTTTGTTCTAATTGTTGCCTTTCCTAGGCATCATAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_14728',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 14728,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.48982186383932,\n",
       "  'map_genome': 44,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTGTACCTTCGATCCTAAGTGCAGGGTGTGACCAAAGAAACT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_14772',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 14772,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 78.17260693738757,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGATAAAACTAAGCGACATTTTCAGCCAATGTCGATTGCATA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_14816',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 14816,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.56189419084325,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAAAGCCACAGTCTGACTTGCAAATTAAGAGGAGATAGATTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_14883',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 14883,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.04280121781147,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGCATCAGGAAAGCCCCCTAGCTGACTCTTGTATCCACAATA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_15071',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 15071,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 79.58763968174259,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTTTAATTCAGAAGAACCCTGGTCCACTAATAGATCCCAGGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_15115',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 15115,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.99795977535473,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCATTTCTGATATGGCACTGCTCTTGCAGCTACAGGGTCAAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_15159',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 15159,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.05326775574008,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCAGGGGGTTAGAAAGAAGTTAATGTAGTGGTTTCCAAAATG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_15211',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 15211,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.55354572542495,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCAGGAGCAGCACCTGGAACTATGTTGGATGTTGGAAAAAAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_15258',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 15258,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 78.33741257696835,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGATCGATTGACATGCCTTAGTTCTGGAATTCCTAGTCACAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_15310',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 15310,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 74.92810303803327,\n",
       "  'map_genome': 26,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CACACACCCCTCAACCGTGCTGCAAGGCAGATCGTGCTTTTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_15368',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 15368,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 82.48785342493409,\n",
       "  'map_genome': 26,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCTTCCATTTGACCTACTGCCATCACCCTCTGCCAGATCACA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_15414',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 15414,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 80.1700248385066,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAACCAATGCTGTAGAAATTGTTTGACTCATGCAAGGTGACA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_15464',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 15464,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 74.77430483434671,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AACTAGACAGCCAGGAGATACAGTGCTGACCATTACATCAGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_15545',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 15545,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.65465513870726,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCCAACCATAGTAATGCACTCATGGTCCTAGCTCTTGGGATG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_15669',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 15669,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.24702068418122,\n",
       "  'map_genome': 41,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTTTCACAACTTTGCAGGTAGTTCGTGTCCTCTCATACTGCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_15713',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 15713,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.47830588662703,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAACCTGAGTAGAACATCCCTGCATGAGCTGACAGTCACATG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_15814',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 15814,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.3996432534774,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAATAGATTCAGCTGGGCATTGATTCAATTCCCTCAGAACTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_15868',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 15868,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 74.72899450706285,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATCTGATAGATTCCAGGATGGGTTGGAGGAGTGTTGCTGAGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_15913',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 15913,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.1481950453636,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACCTTGATGCTCTCCAAGGAAAGGCTAGGCACCAGTCCATGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_15964',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 15964,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 82.53336948625576,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAGCAATCCCCTAGATCTGATTGATGTCCCTAGAGACATGCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_16038',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 16038,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.53590779462871,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGGCTCTTGGATGCTACAGACATGTCAGGTAGGAGAGAGGTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_16100',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 16100,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 79.50215590658428,\n",
       "  'map_genome': 47,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAGATCACAGGGATACTATAGGTAGAACCAACTAAGACATTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_16152',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 16152,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 73.29402991277857,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGGCCTGGAAAACTAACCATTTATAGCCCTTTTTCCCAGAGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_16235',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 16235,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 79.31601675526366,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTTTAGCTTATTTTGATAAGCTGGTTCTGAAGTGCCCTAGTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_16279',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 16279,\n",
       "  'strand': '-',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 74.07217768822483,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGAAAGTCTCCAGCTTCGTCCCTAATGCTTCTGTATCCTTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_16358',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 16358,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 76.76611980181457,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 1,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGGGAAGCTTGAGGTAGGTTACAAAGTGGTAAGGCAAGAGCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_16442',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 16442,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 79.91794159066694,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATATGTGCACAATGACAGCGAAGGGCCTCAGCCTCTGGGGCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_16507',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 16507,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5952380952380952,\n",
       "  'tm': 85.21057164096493,\n",
       "  'map_genome': 41,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCTGCCCTTCAAGCCATGATGGCTAACCTCCTGGGACACCCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_16647',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 16647,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5952380952380952,\n",
       "  'tm': 85.85552459859366,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCTAACCCTGCAGCTCGGGGTTATTCTACCTCGCGTAACCCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_16700',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 16700,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 84.34912331164242,\n",
       "  'map_genome': 26,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGAAGTAACACTAGTGAAGGCTCAGGGCAACATAATGGACT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_16750',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 16750,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.00524835956799,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACCTTTCAAAGACCCGAGGACACTTGATTAAGGGACAGCATA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_16839',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 16839,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 78.35876048393459,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTTACTAGCTAGTAAAGGTTTCTAGGGCTAGTAACCATAAAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_17019',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 17019,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.62144543660878,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 27},\n",
       " b'TCGTTTGATATATGAGTTCTCAGCCACATCTGATGTTCTAGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_17092',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 17092,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 73.5968745215269,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAAGGCTACGCCTCCTCAGCCTACCCAAACAGATGTACCAAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_17202',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 17202,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 81.8004357640803,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGAACCAAATATTCAACTGCCTGAAGATATAGGAAACATCTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_17246',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 17246,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.04258484335202,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CACATCTGCCTTCAGATTAATGGCTATGTATCACCAATGCCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_17299',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 17299,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.4025880437361,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCTCTCCCTTTATGGTGACAGTAGGGACTAAATGAGGGTGTA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_17381',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 17381,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.81867419939891,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGACCCATTTAAGATGTTAGACTCTACAATGCTCAACCCCTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_17454',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 17454,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.88498217514945,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACTGCATTGTTTAAGCTAGGGCAACAAGGAAATTAAATATGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_17619',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 17619,\n",
       "  'strand': '+',\n",
       "  'gc': 0.33333333333333337,\n",
       "  'tm': 73.48236140816573,\n",
       "  'map_genome': 45,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGAGAGTCTCATGATCTGTGGTATGCCTTGGACTCACTTGCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_17711',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 17711,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.57594125352585,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CATGGATGTGTGCTACCTTGCTTCATGCAAGTATTACCATGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_17801',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 17801,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.99206733661856,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTCCTGTCCTGAGCAAGTTGCCTACTACTGGACCAAACACCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_18000',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 18000,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 81.26338092201416,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGTACTTTTTAACTATGAATGTGGGGCCTACAGGTAATGAGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_18066',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 18066,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.96779071266081,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGTGCCAGGCGTTGAGCTGGGCATAAACTGCACAATGATGGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_18111',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 18111,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 82.50757933907511,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAAAATTCCCCTTTAACCTTGCCAAGATCAAACCACAATAAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_18195',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 18195,\n",
       "  'strand': '-',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 74.5269928719627,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATGGTTTTACCCAAAGACAGATGCTACTAAATTCAGTTGGTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_18239',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 18239,\n",
       "  'strand': '-',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 74.29991737028269,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AACTCCTGGAGCAGGTTCTTCACTTAACCAACACCACTGGCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_18311',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 18311,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 80.74342651842784,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACCAATGAATCAACTGGTTCTCACTTAACTAGTAAGTTGCTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_18391',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 18391,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.13352217821739,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AACCCAGCCAGCCGGCAAGCCACTGGATATGTGTAACCATTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_18435',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 18435,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 83.56993255743379,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGGTTGGGTTTGGGCATACCCAGTCCACTCTAAAGTTGCCTA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_18506',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 18506,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 82.88388358772409,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGATTTGCATTGCAGGTGCTTTTAGCAACCAAGTCATCTTCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_18596',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 18596,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.64967771499033,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTCCCTCCCTCACTTAGATCTCATCCCTGTCTCATATTCTCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_18685',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 18685,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 76.80185008855466,\n",
       "  'map_genome': 45,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAAGGGGGAGACCAGCAGGAAGAGCGAGGACTAGGGTTGGAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_18782',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 18782,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5952380952380952,\n",
       "  'tm': 84.62264399568954,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAGGGGTAGGGTGGGTTTGAGCAAAGTACACAAGGAAATGCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_18827',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 18827,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 81.68783504040181,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACCCATTCTTTTAACGAATTTCTTGTGAATTGAGGTCTAAGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_18880',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 18880,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.04269272928389,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGACTTAATTTTTTCCGGCAGTTTGAGGCTCCCAGCAAAACT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_18998',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 18998,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 78.12924107875938,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGCTCAACAGCTATGAGAAGTATGCCGCTGTGGGATGTTCAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_19104',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 19104,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 78.9197314001575,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGAGAGTAAGGGGTGACCCCACTTGGCAGTTAGCATTGCCTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_19174',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 19174,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 82.8101127990476,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGAAATTCCTTTCTCGGCTGGATAGCTAGCAAGATAATTGAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_19257',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 19257,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 74.35002010529041,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAACAGAAAAGACATTACTCAGTTGTACTCAGAGTTCTGCCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_19313',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 19313,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 73.4124709664199,\n",
       "  'map_genome': 49,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAAATCTGTGTGGGAATCGAGCATATGGGAAGTCGTTGGTAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_19389',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 19389,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.53821466840111,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TAAGCCAGATTAATTGTATGCACTGTGCTCAAAACTTCTTGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_19437',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 19437,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.38629564587387,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTCCTTTGCCCTGAGCTGAGGATTGTTAAAAAAGCAACACTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_19509',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 19509,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.02456892530944,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTTCTTCGCTCTTGGAAATAGCCACTCTGTATACTCAGGATA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_19553',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 19553,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 75.25533135304357,\n",
       "  'map_genome': 46,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCTGTGACTTGCTAGCATCTCATGCATGGGGGAAGACACAAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_19649',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 19649,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.52412029579222,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATCTGGCCAGATAGCAATGCCCAATTAATTCCCAGGTGCAGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_19693',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 19693,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 81.20780709059352,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTTGCAATACAGCCTCTCTTAGTGCTAGAGAACGTTACATCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_19747',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 19747,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 75.63450476252245,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAAAACACATGGCACCGCAACATTTGCTTATGGCAGACTCGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_19791',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 19791,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.78480128791102,\n",
       "  'map_genome': 26,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTGGAGAGGGGTCTCCCCTTGCCTTGCATTTGGATTCATTCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_19836',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 19836,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 81.61800229186991,\n",
       "  'map_genome': 44,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTTCTTGTGGGACACTAATATCATCATAGATGCTCCACCTAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_19926',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 19926,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 74.50924307006761,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TAAATACCATCACACTACAGGATAGGTAGGGCCTTAACAAAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_19997',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 19997,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 75.67546976775361,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTTTAAGTCAGGCACAAACATTCACTCCATAACAACCAGTCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_20041',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 20041,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.09190112915547,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGGGTACTTCTGGGAGCACCCTTTATGGGTCAGGTTCACTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_20091',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 20091,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 81.89896843320639,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCCTCAAGACTGGTATTGATCTCTAATATACATCTTACCCTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_20147',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 20147,\n",
       "  'strand': '-',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.48208183207237,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCTGGATCTTAGTTCAGTTACTCCCTCCACCTAAGCTACCTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_20193',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 20193,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 78.52447253921594,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACTCTTTCATGTGTAGGCTCTAGGGTTTAAGTTGCCCACACA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_20239',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 20239,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 78.43699032008493,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCTTCCTCACTCCATTAGGGTGTGTGAACTATCCCAGAAAGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_20322',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 20322,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 78.94154183151278,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGACTCTGGGCAAGGTGGCCTGCGGCAGAAAGACACTTTCCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_20367',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 20367,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 84.3259632873768,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCTCTGCCCATCCTGAAGTTGAACAGTTTGTCCTGTCTTAAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_20411',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 20411,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.45904547051123,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGAAAAAGTGGAGGTAAATATGGGGACATGCCCCCTAATCCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_20455',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 20455,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 80.80542262221502,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACTCTCTTGGGTGAGAGGAAGCGCCACACTCATCCCTAGAAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_20504',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 20504,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 80.94283034822553,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTGCTAGAGCCTGGTATGTCTGAGATGATTCCCACTCATCAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_20548',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 20548,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 77.44759920431443,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 3,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACAAGGATGAGGGGCTGGCCCTGAATGCTTCTGTCATCTCTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_20616',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 20616,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 81.65650995602277,\n",
       "  'map_genome': 49,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACAAATTAGACAGATATGGGTATTCATGCTCTGGACAGGAGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_20661',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 20661,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.1281112860346,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTGGGATCAGGAATCCCAGGCTACTAGATTCCTTTTGTATAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_20744',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 20744,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.81282146586148,\n",
       "  'map_genome': 49,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCCAGAGGAGGTTTAGATTGTGCCCTTGGCAGATAAGCAGCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_20788',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 20788,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 81.47663075297612,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CGCTCTTTCCAGGCCCATGGAAGTGCGGATTGTGCCTCTTTA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_20879',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 20879,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 82.80609066357727,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCTATGCCCCATCACTCATCCGATGTTCGTTGTGCAGAGGCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_20923',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 20923,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 81.85641436298619,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCTTAGTCCTATCTCATCCATGTCAGCACTGAAACATATTTA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_20991',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 20991,\n",
       "  'strand': '-',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.22400986884634,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATAGCTCCTTAGCTCTCGAAGGGAATGCCTGGTTCCTGCCCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_21035',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 21035,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 83.92979648369868,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCTGTTAGCTGAACCATTGGCATGATAAGGAATAGCTAGGGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_21079',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 21079,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.86850913049369,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCCAGGTCAAGGACATCATTATTTACCCCTGTCCTGGGGTGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_21162',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 21162,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 82.30743213643404,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCTGTTCCAAGCCTCATCAAAATGTTAAGTCACCACATTTCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_21222',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 21222,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.21000037430451,\n",
       "  'map_genome': 47,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGCAACTGCTGGACTTAGCCCTGATTAGGCCGTCAGTTAAGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_21295',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 21295,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 80.69558311291956,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGAGGAAGGATCAGCCCAGATGCTCTCACTTGTCATGCAGTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_21355',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 21355,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.3647526723467,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAACGTGAGAAGGCATGATTTATCACGTAGTCTATATAGTGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_21445',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 21445,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.19570925760962,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGAGACACTGGGTCAGACAAAGGTGAACATGGTCACCTTTAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_21490',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 21490,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.42681495834438,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGAAAGAGCAGAGTCGGCCTGAGTCTTCCATGCTGAGGTGTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_21569',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 21569,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 80.30880272209168,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAACAAGGTCTGAGTGGACCATGATTTGGAAGGGGCAAGAAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_21670',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 21670,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 78.950434845433,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCTTCAGCCCTGCTGCCCACACCAGCATGAGCCTTTCCATTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_21714',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 21714,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 83.73139933742829,\n",
       "  'map_genome': 49,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGAAGGGAAAGGGTGACAGCTCTCTTGGAAGGTTGAATGGCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_21792',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 21792,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 81.26054107344271,\n",
       "  'map_genome': 47,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGTCCTGGGACTCTAAGAGTCACTAGCCTACAGACATGTGTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_21838',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 21838,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 78.76860640675403,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCAGCAATCCCAGGTAATGTCCGACCTGCTAAATCAGACACT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_21905',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 21905,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 79.0363970915704,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCCTCCCCCAGTCAGTGTCAGACATGATGGAGGCCTTCTGTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_21962',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 21962,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 82.57584546842577,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCATTTATTGGAGCCATTTATAAGAGGGCTCTCTAAAGGTAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_22026',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 22026,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.0318986297234,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAATGGGAATGTGGTGCCATCTCAATTCTTTCCCCGTTTCCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_22070',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 22070,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 79.5825051435034,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATTCCTCCAGTGGCTAGGATCTCGGTGAAAGCTCCAGCAGGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_22122',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 22122,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 83.1274594280573,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 1,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAGGTGGCAGCCAAGGGCTCAGAACATGGAGCTGTTCTGGGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_22167',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 22167,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5952380952380952,\n",
       "  'tm': 84.60043330356046,\n",
       "  'map_genome': 50,\n",
       "  'map_transcriptome': 1,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATGGATGGGCAGAGAGCCACGTAGGCAGCACAGCAGGAAAGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_22225',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 22225,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5952380952380952,\n",
       "  'tm': 84.50946177180344,\n",
       "  'map_genome': 43,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGAGGAACAGGTGGCCAATTCTTCACTCACTTATTATGCACA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_22281',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 22281,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.8827709564087,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGCAAGCAGTTGATGTGAGATAGCTGAACAGAGGATTTCAGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_22362',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 22362,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 76.79278844874625,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATGGTTTCCACCTCATTAACACTGCTCTTGGGGCTTTGTTTA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_22427',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 22427,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 78.18986356513915,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATCAGGAGTCATATCCAGAACCTCTGTAGATGGTGTCAGGAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_22483',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 22483,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.80195419825884,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGAATTCTTTTAGGGACCCTCACCCACAGGTAAAAAAGTGTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_22532',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 22532,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 77.2556651964075,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTTTCAGGCCAGGTATGCAGCTCGGAGGTCCAGTACCTGGCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_22582',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 22582,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 84.65820175949636,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGATCTTATGGAAGCTTATTCTCAGCTAAGGTTCCCCCCTTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_22912',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 22912,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 77.66608073193697,\n",
       "  'map_genome': 50,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAAACTCACCAGCATATAAAGCTTAACTCAAACTGCTGACAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_22985',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 22985,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.40740723178646,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGAGAAACATCTAATTCAGTGACTGGGAAGCGGCAAGCTCAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_23031',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 23031,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.66104228679228,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAAGAAGAGAGGTTAATTTGGGTGTAATCCAGTGCCAATGAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_23145',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 23145,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 75.31294963325445,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TATGAGAGATATATGCAAGCTGGCTTTTATGACAGACCCATT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_23266',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 23266,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 74.54626554376966,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGTAGAACCTTTTAGATTAAGGTTTTGCCTCATGAAAGGATT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_23340',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 23340,\n",
       "  'strand': '-',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.53850071008964,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTTTGGATGCTGGGACTGGAATCAAGGCTTGCATGTGTGAGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_23401',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 23401,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 80.24741904239153,\n",
       "  'map_genome': 47,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AATCAAACAAGGGCTGCAGATACAACTCAGTGGAAGAGTATT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_23445',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 23445,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.4788035797439,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTCCAAGCTAGCTTGGATGATGCTATAAGAAAACCAAACATC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_23497',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 23497,\n",
       "  'strand': '-',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 74.17150267780988,\n",
       "  'map_genome': 51,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGCTGAGAATACAAGTGTGGTCCCTGATCTTTTCAGATCTCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_23568',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 23568,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.21625621731556,\n",
       "  'map_genome': 48,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGTGTCTGCTTTTCAACAACTCATCTTGATTCCTAATCTCCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_23612',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 23612,\n",
       "  'strand': '-',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.6791596795253,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCCTGAAAGTTCCCACTTTAACTCCCAGGGTTTTACTTCCTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_23679',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 23679,\n",
       "  'strand': '-',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 78.0889896690087,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGTCAGAGGGGCGGTGGCTGTCGTGGACTGGTCAGGTAGACC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_23734',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 23734,\n",
       "  'strand': '+',\n",
       "  'gc': 0.6428571428571429,\n",
       "  'tm': 86.54317443716275,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTTGATAAAAGTTAAGCTTTCTAGGCTACAGTGCTGCATGGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_23794',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 23794,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.12514440831467,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCCCCCACCTCACCTGCCTATCTCTATACTAGAGATCAAATC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_23857',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 23857,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.99612008479505,\n",
       "  'map_genome': 46,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TAGAAATCACTGAGTTATGAGGTGGTAGATCAGCGGTAGAGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_23924',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 23924,\n",
       "  'strand': '-',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 75.60847379758962,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 27},\n",
       " b'ATGGTCATCTCTTTGTACCCCATCTTAGATGAACCCCAGAGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_24031',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 24031,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.88627477774804,\n",
       "  'map_genome': 43,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGTGACCCTCCAGCTATGCAGGACATGGACATAGGATGGAGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_24088',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 24088,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 82.0414463648138,\n",
       "  'map_genome': 43,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCACTCTCCTCAAAGTTACAACTTAAGTCTGGCTCAGTGCAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_24152',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 24152,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 76.71897906469815,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCTGGTTTTCATTTAGATCCAGACACCTCAAGAAGATAGCAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_24226',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 24226,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 73.9756949334095,\n",
       "  'map_genome': 44,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTCCTGTGGCGTCCCTGGCATCCTACGAAAAATGTGGACCAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_24326',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 24326,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 82.84751181727637,\n",
       "  'map_genome': 26,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGCCACCAACAGCGCCTTTGAAGCCGACTAAACTCGACATTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_24395',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 24395,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 80.83912132932488,\n",
       "  'map_genome': 26,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGGGCACCTTGCATTGGTCAAGGTTCGGAGGAAGATAGAAGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_24439',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 24439,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 80.79648002471106,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCCTCCCCTCTGCTGGTTGCTGGTCCAAACACATCATCATTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_24503',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 24503,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 81.70499308719707,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TATACTCTGACATGGGGCATGGAAAGTAACATCCTCAGAAGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_24547',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 24547,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.28150409262622,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTGCTAAAGCCATTGGTCTTAAAGTCGTATTGGTCAATTACA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_24610',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 24610,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 74.38639858909232,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAAAAGCAAACTTTCCTGGCCTGTTTAAAAGCCTCCAAGGAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_24686',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 24686,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 77.85930832210596,\n",
       "  'map_genome': 47,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGAGTTTTCCCAAAACGTATGGTTTCTGGTGTAAATGTAAAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_24757',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 24757,\n",
       "  'strand': '+',\n",
       "  'gc': 0.33333333333333337,\n",
       "  'tm': 73.57986730719693,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTACTCAGAACACAACAGTTCTTACTTTTCCCATCCCATGTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_24812',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 24812,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 75.01812763923652,\n",
       "  'map_genome': 51,\n",
       "  'map_transcriptome': 1,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCAATCTGCTTTGCTAATCATGGGCATGACTGCATGAGCTCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_24926',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 24926,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.2585433090847,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGGTGCATTCTTGTCTATAGAAAAGCACTTAAAATCCCAATG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_24979',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 24979,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.1085642164598,\n",
       "  'map_genome': 50,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTGTGATGTGGTGCAATTGACAAGCTTTGTATAGTGACTTTA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_25042',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 25042,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.45458927458685,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGCATTCTCCCATCATTGTCTCTTCTCACCATTACAAACCCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_25092',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 25092,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.70487488305258,\n",
       "  'map_genome': 51,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTGGATGGCAGTATGTGACTTAACAGCAGCTTGTGGGACTGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_25186',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 25186,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.38682625392084,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATCATTGTGTATTAGATAATTAAGACGGGTGCAGAGAACAGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_25330',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 25330,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.61631185185513,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGGATCACATGTTTGGGACATGCCTGGACAGCCTAGCAATAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_25402',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 25402,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 79.85420508837404,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGAGAGAGATTATGCATAGATGAAACCAGAAGAAATCGCCAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_25496',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 25496,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 74.23325017497632,\n",
       "  'map_genome': 48,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TAATGGTAATCAAAGCTATCATTTGTTAAATGCCTACCTGCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_25622',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 25622,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 74.04681370570194,\n",
       "  'map_genome': 43,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTGCTGAGCACAGTGTTGAACACCACGCATGAATTAAGCTAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_25666',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 25666,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.99612820157256,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAGTGATTAGATTGAAGTCGCCTAACTCTAGGTGTCTAACTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_25773',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 25773,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.54971705517107,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGGAAATCAGCAAAACGTCTTCCTCTGCAATGGCCTGGGAAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_25867',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 25867,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 80.20272264924591,\n",
       "  'map_genome': 41,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCTACCTGGAGCACAGCGTGGAGATGAGGTATCAGAGTGCAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_25911',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 25911,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 81.07248040673647,\n",
       "  'map_genome': 44,\n",
       "  'map_transcriptome': 4,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTGCCTGTGAAGTGGTCGGTAACATAGACCTTATATATTTCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_25970',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 25970,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 74.91925715175267,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTGTGGCTTAAATTCTGCCCCCCGGAAGTCTTGTTCCCTAAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_26015',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 26015,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 81.28372441574135,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGAAACTCCGTGGTAAACTCTACCCTAAGGTTATGTTGAGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_26094',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 26094,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.0146609476808,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGGTGTTGGCACAAGCACGAGTTTGAACTGTTTGGGTATCAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_26142',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 26142,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.72719534104209,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCTCTGGATAAACTGAAACATGGCGGTAAGTCAAACCCAGAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_26228',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 26228,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.02503376582922,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGCATCTGACCCTCATGTCCAAACATAGTCTGGACACTTGGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_26324',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 26324,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.3132452089049,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAGTGCTTTGTCCCAGTGAACCATCTAATGTCATATACAATA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_26370',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 26370,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.50249782999816,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TAGGGATAAAACATGTCACAGAGTCTCCAACGATTGAATTTA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_26436',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 26436,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.09637861702493,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCAAACCTTCTCTCATTGATCGAGGGAGCCTCAGACACAATC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_26666',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 26666,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 77.67414129742792,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCCTCACTGCTCCCTGAAATGAGAGGGAATCCAGCATGCTCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_26734',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 26734,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 81.45761777865357,\n",
       "  'map_genome': 41,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCCTGAGGCAATGTCTAGGTCTGATATCACAGGCCCTGCCTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_26781',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 26781,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 81.6567238883739,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTCTGCTGAAGGGACTGACTGAATGAGGTATAATCCCAGGGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_26839',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 26839,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 79.78459277583175,\n",
       "  'map_genome': 44,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCCTCCCGCCTGCTGCTAAGGCGTTTCCGTAGCACCCATAAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_26883',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 26883,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5952380952380952,\n",
       "  'tm': 86.1070669725159,\n",
       "  'map_genome': 26,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCAGAGAAGCCAGCAGGCCGAGGAGGGTCACTGGGACCCTGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_26927',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 26927,\n",
       "  'strand': '-',\n",
       "  'gc': 0.6904761904761905,\n",
       "  'tm': 88.71636955595613,\n",
       "  'map_genome': 43,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGTGCCCACCTCATGCCTCCTTGCTGGAGACTGCCCCCCCCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_26995',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 26995,\n",
       "  'strand': '+',\n",
       "  'gc': 0.6904761904761905,\n",
       "  'tm': 91.28049243741214,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGATGCCAAGGCTGACTGCCGACGCCCATCTGGCAATAACCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_27069',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 27069,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5952380952380952,\n",
       "  'tm': 85.49556775594306,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGGCAGGGTTTCCAGGACATGTAGGGTAATAAAACTGCCTCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_27134',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 27134,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 80.50934188757356,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGACAGAGGCCAGTCACATATTTCCGTTCTCACTCTGGAGAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_27180',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 27180,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 78.49727988641382,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCCCATCTGCCAGCTCTAGGCAACCTTGGTGCTGGGTGGGGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_27227',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 27227,\n",
       "  'strand': '-',\n",
       "  'gc': 0.6428571428571429,\n",
       "  'tm': 88.54001723273723,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTAAAAAGGAATCATGTGACAGGTCGTTTCTGACACAAAGTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_27304',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 27304,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.22634270233732,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGCACTCCATCTCTAATGTCACGGGAATGTCATCCTCTACCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_27411',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 27411,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.47611936801246,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAGCCCAAGAAGTTCCCAACAAATGCATCCCTGTCTCTCTCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_27455',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 27455,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 79.36006813046401,\n",
       "  'map_genome': 45,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGTAAAAATCAAATGACCAGGTGTGTGGGCTAGTTTTCAGAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_27517',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 27517,\n",
       "  'strand': '-',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 75.14517034838138,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGGGACCGGGGACACAATTCGTTATCTACCAATATACCCTCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_27567',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 27567,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 80.43645588763741,\n",
       "  'map_genome': 26,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAACAGAGAAACCTAACACGCAGAATCATTATGGAGAGGGTA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_27640',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 27640,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 75.80203507934982,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGCTTGGATACCTCTACAGACTCCAACAAGGGCCAGGAGTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_27704',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 27704,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 80.99381106062856,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGGATGTGGTATCCCAGAAAGCGAGCTTGTGTTTGGCCTGAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_27788',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 27788,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 81.01379351099723,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGTACTAGGATTTAATTCAAGGGTCCTCTGGCACTAGGCAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_27854',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 27854,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.73509594729506,\n",
       "  'map_genome': 41,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGTGGCCCTGGCTCATGACCCCAGCTATTTGGGAGCCCTGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_27997',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 27997,\n",
       "  'strand': '-',\n",
       "  'gc': 0.6190476190476191,\n",
       "  'tm': 87.1582223329633,\n",
       "  'map_genome': 41,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGCCATAAGGCAATTTCCCCTACCAACAGGTGGCTGAGGCCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_28065',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 28065,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 84.98560369658765,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGGGTAGGCCTCCAAATATGGGTCAGCATTTGCTCAGCACTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_28149',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 28149,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 81.70852453536077,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCTGGAGATCAGGTGGTTGAATAGAACGCTTACTTTGTTTCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_28194',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 28194,\n",
       "  'strand': '-',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 75.87133353281467,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGCTGGTGGAAGGCTTAGCTCAATGGAGTGTCACACAGGACA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_28271',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 28271,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 81.86234407945693,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCAGCAGCTCTCACTAAATAGGACGCAGCTATTTGTAACATA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_28360',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 28360,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 75.83219656959585,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGATTCTATAGGTAATGCTCCAAGAGCACACATGTTCCCCAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_28404',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 28404,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.20449225324364,\n",
       "  'map_genome': 41,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTGCATAGTTTGTGCTGCCTCTTGTCTTTATCTAAAGGTTGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_28449',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 28449,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.39875063702164,\n",
       "  'map_genome': 50,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTAGATTTGGTTTGAGCTAGTCCTGGGTGGTACCAAGAAACA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_28493',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 28493,\n",
       "  'strand': '-',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 77.63059884433437,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTTCCCCACACCTCTCACCCTTTGACCTCAGTAGGGCCTCAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_28627',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 28627,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 83.72057980619326,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCAAGACTTTGGGTGGAGGAGATGACAGTCCTGGCTGGTCAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_28680',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 28680,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 81.99022159027675,\n",
       "  'map_genome': 44,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCCCGTGCTCTGCACTTCAAACGTGACAACGAGAACTTGAGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_28725',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 28725,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.1922591839458,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AATGATTAAACAAAATTACCTGGGCCAATATGCAGCTGATTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_28852',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 28852,\n",
       "  'strand': '+',\n",
       "  'gc': 0.33333333333333337,\n",
       "  'tm': 74.01965256325036,\n",
       "  'map_genome': 50,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATTAGTGTGGCTACTCAAGTGAGACTATTGGAGCACTTCCAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_28896',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 28896,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.46838022704844,\n",
       "  'map_genome': 26,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTCCCTGGTTACTGAGCCCCTCCCTTAGGGTTCTCACCAACT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_28983',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 28983,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 84.23251867418736,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGAAGGCTCCATTAGTATTATGACCAGTATTCATCCTTGCCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_29072',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 29072,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.77504825266328,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCCAGTTCTTAAAAGCTGGACTGTCCTAGAGACAATGCATTA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_29132',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 29132,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 75.89343635552171,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATGTACCTGTCTTATAGGCTAAAGTATAGGCCACACAAGCAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_29176',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 29176,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 76.28873525974228,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAGGTTTATATGACCCACAACAACGTAAAGGGCAACCCCTTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_29235',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 29235,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 79.30974758931217,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACATTTGCTTCAGAAGGGTGTTAGGCTGTGGGAGTGCAAAGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_29335',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 29335,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 79.65873235158546,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGAAACTGTTAGCAGGGTCCCCTGCCATTCAGCCCAAGAGCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_29394',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 29394,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 84.37518317192627,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAACTGCTGCCAAGTGAGAAGCCGGGTGCCAGACCCCACCTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_29456',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 29456,\n",
       "  'strand': '+',\n",
       "  'gc': 0.6190476190476191,\n",
       "  'tm': 86.44837650440718,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATTAAATTACCTACCCGATCTACCTCACCGGCTGATGAAAAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_29554',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 29554,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.92016214754443,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATTAACATATGGTGTGATAATCCGCCCCATTCAGAAGAAGCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_29641',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 29641,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 76.50105706044536,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GACGGGTGGCGAGCAGTGAGTGATTCAGCCAGCCAGGCAGGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_29685',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 29685,\n",
       "  'strand': '+',\n",
       "  'gc': 0.6428571428571429,\n",
       "  'tm': 86.4253219017121,\n",
       "  'map_genome': 47,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTCCTGAGCCAGGGTCCAGGGTTCTCTCTCAGAAGAAGGTGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_29744',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 29744,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 82.74068620683016,\n",
       "  'map_genome': 51,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGGAAGGGCTCCAGGGCGGCTGAACACAAGGCTCTCCATTTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_29790',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 29790,\n",
       "  'strand': '+',\n",
       "  'gc': 0.6190476190476191,\n",
       "  'tm': 85.82769146108552,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CACCCAGTGTGCCAGGTCAGTCCGGGCCAGTGACCCTTGCTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_29872',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 29872,\n",
       "  'strand': '+',\n",
       "  'gc': 0.6428571428571429,\n",
       "  'tm': 87.68083260952295,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCAGAAGCAAGATGTGGGTGGTGACGACTCTTTTGAATTCAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_29917',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 29917,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.267522682086,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 27},\n",
       " b'CTCCTGCCTGCATTCCCTACCAGCAAAATTAGTCCTCAACTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_29993',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 29993,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.29056682476778,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGGGCTGGGCCTTCTTCCTCCGAGTTGGAGCAGACAGACAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_30074',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 30074,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5952380952380952,\n",
       "  'tm': 84.2421624013648,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGACTTCAAACCAGACCCTAGGACTAAAGTGCTCCTGAGGAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_30132',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 30132,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 78.80951944440784,\n",
       "  'map_genome': 45,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGGTCTGATGGGAAGCTTGCTTGGTAAGGTATAACCAGGCAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_30176',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 30176,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 80.45552637065924,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTTGTTATTCATGCAGGCCAAGTAAAGCAAGACAATAACTCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_30257',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 30257,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 74.14727072406362,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACAAGCCTCACGGCCATAAATCCAGGAGACTATTCCATGAAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_30344',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 30344,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 78.59425368327976,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGCATAAAGGGAGATCTCTGTAGATAAGAGGGCTCTCTCCTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_30429',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 30429,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 77.66156958652556,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTATGTCTACAAAGTGACAAGGGTACAGGTCCACCTGTGTGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_30548',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 30548,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.25044777209695,\n",
       "  'map_genome': 43,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGCCTGGACAGGCCCTAGAGAACGAAGAGGACGAAGATGACA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_30611',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 30611,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 81.71534064741161,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GACACAGATGCGGAAGCATAGTTTACAATAGCATCCAGCTGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_30672',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 30672,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.47409675545521,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGCTCTCCCAGCCCAAGGAGGAGTGTAAAAGGCTCTATTCCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_30800',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 30800,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 82.02479932327037,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAATCCCAGAAGATCTGTAGCTTTAACCTGAGGATTCAGAGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_30844',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 30844,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 75.63093402831151,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GACAGATAGACAGTTAACAGGCACTGGTGTATGTAGTTTACA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_30918',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 30918,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 74.33145778437449,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TAAGTGCTTAAAGTACCTCAGCTCATGTGATCCTCCATCCAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_30977',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 30977,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.53866672269118,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAGGCCTTCATGTTCCTGATGAGAGCCGAAATTTCCTTTTAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_31060',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 31060,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.43900135694611,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGAAGGAGGCCAAAAAGGGCCCTCAGCCAGCCCATTACTCAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_31143',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 31143,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5952380952380952,\n",
       "  'tm': 85.41371840469185,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGTAAGCAAGCTAGTTGTGAAAGACAGTGGCTCAGTTGTCAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_31187',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 31187,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.68735869153636,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACAAACAGTGTACATAACACAGGTCAATGCCCCAGCCTTTCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_31250',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 31250,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 78.79513529822464,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGGTACAGGGAAAGCAGGTAGGCTTCCTGATGACATGAGAAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_31339',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 31339,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 80.12322287485085,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTTCCAGCCTCTTTGTTGTGCTTTGGCGGTGCTTGTGCAGAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_31400',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 31400,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 81.29022658752581,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGAGACCACCTCCACACTCAAAGAATAACTGAAGTGGGAACT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_31459',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 31459,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.0681696442017,\n",
       "  'map_genome': 49,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTGTATTTGTGACAAAGGTTTCAGAAACATGCCATCCCCCCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_31530',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 31530,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 79.19313360576177,\n",
       "  'map_genome': 44,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCACTTCACAATTTAATGCTACACTTTGCAATGCAGATCCTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_31574',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 31574,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 74.49032405547536,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TAAATGGAGAAAGAGATGTGGAAGTTTGAGACAGCAGAAAGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_31712',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 31712,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.39690892847932,\n",
       "  'map_genome': 48,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGGCTATGGGGACAATTCAGCATTGGGGAAAACATGCTGCTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_31797',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 31797,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 81.65007941588686,\n",
       "  'map_genome': 46,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATTTTGGGGATTCCACACCCTTTTCCGACTTCATAGGCACTA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_31869',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 31869,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 79.47166272127686,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATTTATTTTAAAAACGCAGCTCAAGAGTGGGTAAGCAAGATC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_31983',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 31983,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.26244585752397,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTCACTTTTCAACCTCCAAGACTGGACACCAGGTGGGCAGAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_32050',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 32050,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 80.92092235965708,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTGCCATAGTTCATGAAAGGTACCAACATACGAAAATTAGAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_32109',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 32109,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.20198432272161,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGAGCAAGAACCAAAGGACGTCGTCCCAAAGTCAATTAGAAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_32274',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 32274,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.81868194771317,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGCCTGGCTGGCAGTTAATGGGTCAGTAAAAACTGTGGGAAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_32351',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 32351,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 80.91273345245116,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GATGTCACACAGCCCAAAACATTGGTCACATCAATCAAGGAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_32405',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 32405,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.4142887687376,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TAGTCAGTTGCTGCAGCCGTTTTTGCTGGTGTTGGCCATTGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_32463',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 32463,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 82.01567881953014,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAAGTGCTATTAGACAGGTGGGTCAACTGATTTCTGGCCATG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_32507',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 32507,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.8352557367794,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATGCCCCTGGTCAGGAACAACATGAGGGTATTGGCTGATAAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_32551',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 32551,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 80.67235780485385,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCTCCCAGCTGCACTCAGATGGAACCAGAGGTGAAGAGAGAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_32600',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 32600,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 80.04293958640392,\n",
       "  'map_genome': 48,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCCACCACACTGGTGTCACACCCTGCTCAACAGGATAGGCAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_32647',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 32647,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5952380952380952,\n",
       "  'tm': 83.67581427918071,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAGCAGAGCCCACCAAGTACATCTATGGTTACCTCAAGAACA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_32693',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 32693,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 78.20735969938681,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTCTCTGGTATCCTGTTTGGATTAACATTTGCCCTAGATTTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_32758',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 32758,\n",
       "  'strand': '-',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 74.22327750335393,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGGATGGCCCAAAGCATTCCATACACTCTAATAACTGAAAGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_32817',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 32817,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.61335664538007,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGTGAGTGTGAGTTAAGTTCTCTTGTTGCCAAACTCAATTA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_32921',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 32921,\n",
       "  'strand': '-',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.03808518205358,\n",
       "  'map_genome': 41,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGACTGAACTGCTGGGTGATTTGCAATCTGCTACTCACAGAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_33053',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 33053,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.98317831081187,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAACCTCTCCTGGCAACTACCCAGTCAACAACTATCCTGAGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_33116',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 33116,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.05320834496786,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTAAGCTTCTGGTCCCTAGCAATCCTGGGTTCTCTATCTCCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_33258',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 33258,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.26799110769485,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGCTCCTTTCCCGACCTCATAATATAACTAAATTGTCACTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_33322',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 33322,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 74.08103808437147,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCCTTCCTGTAGGTGTACCCACTTGATAGGGCTGGAGGCAAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_33380',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 33380,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 82.72415001034773,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AACTGACTGGCCTTATTGTTGGCTCTTGTCTGAACTCTCTTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_33666',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 33666,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.35191033016775,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCATGTGTGAGATGTCCAGCCATCTCAGCTCAACTCCACTCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_33745',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 33745,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 78.51127417161138,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGTGTCCTGAGGAACATAGGCTGTCTTGCTTTGTAGGGTTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_33854',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 33854,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 78.53333699831654,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGCCTCCTGAGATGATTATTCTATGGCATATGCTTGCATAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_33903',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 33903,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 75.09664110967162,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTCAGTGTTTTAACCAAAATCCCTCACCCTTCCCCAACATTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_33955',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 33955,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 77.69114371217779,\n",
       "  'map_genome': 45,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCTGTTTATACAGTCCTCGTCTACTTTCTATCATCCCAATAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_34017',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 34017,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.43949745785136,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATACTAGGGAGCAAAGGGATGCTGACGGCCCTGTGTATGCTA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_34135',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 34135,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 82.37817097540324,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAGGGATGGTTGTTGTGTAGTTAACTATTACTCCATTGTAAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_34187',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 34187,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.41163765336199,\n",
       "  'map_genome': 26,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGGGATAGGGATAGAGTGTAGGACCCTGAATACCAGTTTAAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_34243',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 34243,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.74670269595202,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAAATAACACCTCAACCTGACCTCAACTTATCAATCCCAAAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_34287',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 34287,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 74.55213513621442,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTTGGAATTTCTTGAACCTGGGGAGACCATTATTATGGGATT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_34469',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 34469,\n",
       "  'strand': '-',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 76.13050458444269,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGCATTGACTTCCTCATTCCCTTCAATGGCTTGTCCACATTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_34582',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 34582,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.53706822885562,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAAGAAAAGGTGAAGCGCGATGTCATTGGGAATGAAGACAGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_34658',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 34658,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.31773157325676,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAGGATGTGGTGTTCACGGCACTGGAGTTACTTCCCAAAAGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_34706',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 34706,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 80.45001645745253,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGAAACTTGAAGCAAGTTGGTTCAGGACTCCAGGTAGGGCAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_34781',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 34781,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.80344436186749,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCTGAGCTCTCACCTACATATTTATGGATGTGCCTTTGATCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_35416',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 35416,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.41909391981085,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCCACTCCACTCAAAACCTAAGTGCCACAAAGTCAGTATCTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_35571',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 35571,\n",
       "  'strand': '-',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.6466675617279,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TAAAGGTCCACAAACTAGCAATAAGTATGCTGTATGTAGTCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_35629',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 35629,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.46389504210958,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTGAAGACACCAGACGTCTACTGTCTAGTTTGATAGTTCTAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_35704',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 35704,\n",
       "  'strand': '-',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 73.17112111237435,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATGCACAGAAACCAGTTGAATACACGTTCCCCATATCACATG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_35827',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 35827,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.43819060633541,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTGCTTGTATACATTTACCAGTGCAGCCCACAAACCTCATTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_35879',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 35879,\n",
       "  'strand': '-',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 77.08975574405196,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGTGCAAGGAAACCAACTGTCCAGCAGACCAAAGAGCAATTA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_35949',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 35949,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 78.36593476808491,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCTGTGACATCATTTCCCAGTCACCTCTGGAATGATTCCTTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_35993',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 35993,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.98403130335197,\n",
       "  'map_genome': 50,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CACTGGGTGCTGGGCAACACTGGTGTCACCCATGTACACCCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_36042',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 36042,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5952380952380952,\n",
       "  'tm': 85.43708295410056,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCTCAGTCACTACCACTGGGAAAGGGGCCTACCTCTCCATTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_36098',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 36098,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 82.810470968397,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TATATTTCAAGGCCCCCATAAAAGACACACACTAGGAATGAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_36152',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 36152,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.92176115663352,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTTCAAGTAGCTGGTGAGTCTCTTTTTGGAGAGAAGATCAAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_36229',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 36229,\n",
       "  'strand': '-',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 73.73213263101098,\n",
       "  'map_genome': 44,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCTTCAAGGGGATGCCAAGAGTATCACGAACAGCTTCAACTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_36273',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 36273,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.4225792573032,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTCACTGAAGCTCAAGCAATCATCATCTGACACAAAAGCTGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_36351',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 36351,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 74.61821722414572,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTCACACTTCCTGTGATTGTCTTCCCGTTCCAGTCACACTTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_36470',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 36470,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 77.7440152501419,\n",
       "  'map_genome': 73,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 65},\n",
       " b'CAGTCACAGGAAGTATGTCTGGGAACGGGAAGACAGTCACAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_36776',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 36776,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 78.83232745796505,\n",
       "  'map_genome': 173,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 160},\n",
       " b'ACTGTCTTCCCGTTCCAGTCACACTTCCTGCGACTGTCTTCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_36847',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 36847,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 80.82148706265616,\n",
       "  'map_genome': 60,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 54},\n",
       " b'ACAGGAAGTGTGTCTGAGAACGGGAAGACAATCACAGGAAGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_36901',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 36901,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.98879000285729,\n",
       "  'map_genome': 56,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 47},\n",
       " b'AATTTGGCTGTTTAGTTCACCCCTACTTTTGCCAGTCCCTTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_37015',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 37015,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 78.73005242126357,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGACATGACATGATAACCCATGTCAAAGGTGCATCTGGGCAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_37095',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 37095,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.61738801414964,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAGTATTAGAGTCCTTGTAATGTCGGGAGGCTCTGTGGTCCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_37147',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 37147,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.65824886684953,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCAAAGATTGTAAGACTAACAGTCCTGCTCAGGTCTGTGGTA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_37203',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 37203,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.65323796843097,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TAAACACATGTATAACGTTCATTTTGGTATGGGATGATAGGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_37248',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 37248,\n",
       "  'strand': '+',\n",
       "  'gc': 0.33333333333333337,\n",
       "  'tm': 73.13005965657845,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTACTTGCACACAATATCTCTGGGATGGGACCTTGAGATTGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_37310',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 37310,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.94148829057826,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGGGGTGAGGGGCAGTTCATACCTCATGCCTTTTGAATTCTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_37362',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 37362,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 80.53185130674353,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCCCAATTCCAACCTCACTTTAACTTTGTTACAATTGATTCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_37440',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 37440,\n",
       "  'strand': '+',\n",
       "  'gc': 0.33333333333333337,\n",
       "  'tm': 73.04302002296106,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CACACACCTGCCAAAGGTGAATTACTAGAAGGGAGCACTAAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_37488',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 37488,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.84296769885674,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTGGAATGTCAGAATAGTCTCACTACCCTAAACATCCTTCTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_37551',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 37551,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 74.08859703622204,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGACCCTGTCAACAACTAGGTCTTTTTCCTAGTGTGTCCTTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_37701',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 37701,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.72630396884796,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGAGAGGCTGAGTTAAATTGGACTTAAGGTTTTAGATGTACT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_38084',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 38084,\n",
       "  'strand': '-',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.16786709925412,\n",
       "  'map_genome': 44,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCAGTGATGTGTCAAGTGTCTAAAACTTACTTCAGCTGATAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_38225',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 38225,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 73.02161981992475,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTGGACAGGGGCCAGGGAAACAGTTCAGGGGGTTACAGCACT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_38278',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 38278,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 85.82777641754979,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCATTTTAACCTTTATCCTTGATCCCTACCTTAGCCCCTTCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_38345',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 38345,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 77.03265809491256,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AATGAAACAAGAGGGTCATTTATCTGAGTCCTGCTCAGCTCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_38431',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 38431,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.5465581154366,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGACAAAATCAGTGTGCCCTGAATGAGGTGCCAGTCCACCTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_38576',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 38576,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 80.77066694360565,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGGCACAAGTAGAAAAGTGATTGACTCCAGCAAGCCCGGGCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_38621',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 38621,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 82.33596313353797,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGAGAAGCACTCTCATCCAAGCGTGCTATGCCCCACCCCCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_38665',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 38665,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5952380952380952,\n",
       "  'tm': 84.75748085828718,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 1,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGATGTTTTTCACAGACCGGTTTTCCAGGATGAAATGCTTTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_38711',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 38711,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 75.31810558329016,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATTTGGACAGTTCAGAAGTTCTGGTAGTGTCTAGTGAGAGCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_38776',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 38776,\n",
       "  'strand': '-',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 75.33590330756346,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GACTTCATGGTGTTGGGGGAGTAATGGCAGCTTAACTAAATA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_38854',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 38854,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 77.23824654054715,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCTAGTCTTCTTTGTCCCTTAATCACCTTCTGGCCTAAGTTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_38898',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 38898,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.69828797251859,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AACAGGGAGCATGACAAGTTCCAAGTATAGACCCGGGGTTGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_38953',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 38953,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 81.36029382694488,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCATAACAGGGTATCAGTAAAAGTCACTATCAAGAAAGTGCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_39050',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 39050,\n",
       "  'strand': '-',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.49697662791698,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGGTCTCTGAATCCATTTCCGTCAGTTGCTGCATAGAGCCTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_39126',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 39126,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 79.42605309266474,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGGATTTCATAGTCAGGTGACAGCTTTAAGGACAACCTCTGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_39414',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 39414,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.93574748076469,\n",
       "  'map_genome': 50,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCTTCACAGTTAGGCACATCCTTGTCTAAAGCAAGATAAGGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_39471',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 39471,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.22363892606887,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CATCTCAGATGCTGCTCCACCTCCCGGTCACCTCCCCACCCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_39610',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 39610,\n",
       "  'strand': '-',\n",
       "  'gc': 0.6666666666666667,\n",
       "  'tm': 87.72383103778981,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGGTGACTTTAGGAGAAGATTAGGGTTTTATGTAGACACAGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_39678',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 39678,\n",
       "  'strand': '-',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 75.44445688066133,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 1,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCTCTTGGTGAGAAGCCTAGGCATATGCCTTTGTACAGACTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_39723',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 39723,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.80415890638744,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGCCAATTGGCCTACGGTCATACAGATGCCCTAAGGTTTTAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_39809',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 39809,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 80.42401265286861,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTCCAGTTTCTCTGCCGGAAAGCCGAGGCAATGGGAACATGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_39901',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 39901,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 82.86144560263193,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTCTTGAAACAGATCGGGTGTAAGTGACAAGGAAATACCATG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_39979',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 39979,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 74.93627885180456,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TATTCTCCAAATTCACCTCCCATCTCAGACCTGCTCAGGCAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_40037',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 40037,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.12263368171466,\n",
       "  'map_genome': 47,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAAAAAGATGAGCCGCTGGTAAATGTTTGCACTCTGTGAGGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_40089',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 40089,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.42331599550067,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGTTATCCTCCTGGCAGCTATACTTCGTTTGAGAAGCCAGAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_40146',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 40146,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.91748975515662,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGCTTCCACTCTGATCCTGGATGGCAGGTTACCTACCTAACA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_40240',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 40240,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 80.6822537687429,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGTGTGAAACCCTTTGTGTTTAGATCTAGTAGGCTAAGAAAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_40313',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 40313,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.40450327106066,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCTTTCACTTGTTGGCTACATGAGCCAACTGGAAATAACCAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_40376',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 40376,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.90904286299559,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AATAATTTGGCTACTTTTGGTTTAGGAATCCTGCCCCACCAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_40430',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 40430,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 78.11569386315784,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAGCAGAGGGCACAGTACGCTAGGAGGTCACTGGGAAGTATG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_40478',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 40478,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 81.72211620392551,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGGAGAAAGGGGGATTCCATTGAGTCTAAAATCTTAGTTCAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_40544',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 40544,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 74.86259075270499,\n",
       "  'map_genome': 44,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGATGACTCAAGGGCCATTGGCAATGTATCTTCCCAACCTGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_40590',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 40590,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 81.2135496425758,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCCTTCTGTTGTGTGCTGACCGTGTGTGTCTTCTTAGTGGGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_40635',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 40635,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 80.76743163484383,\n",
       "  'map_genome': 45,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTGATAATTAGTGAACCTAGACTCACCTCTTTCCTTGTCTGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_40764',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 40764,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.57008881807138,\n",
       "  'map_genome': 50,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGCATTAGCTGATATAAGAAGACTCATCCAGACTATGAGTGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_40822',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 40822,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 73.65757750226658,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGGTGGGAGATGTCAAAGGAGCAGATGCTTGAAGCAGCTGTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_40952',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 40952,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 80.0761090235863,\n",
       "  'map_genome': 49,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CATGGCTGCCCTGACTTACTTATGCTATTCTAGCTTCTTTCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_41079',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 41079,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.65407402260973,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTGACTCCTTCCTGGGCTGTCAAAAGAAGTGGGAAACATTGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_41150',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 41150,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.5948754489134,\n",
       "  'map_genome': 44,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TAAATAGAGTATTCAGCCAAAGTGAATCCCTGTTGACCTCAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_41194',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 41194,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 74.46574792535984,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTGGTGACCGTGGGTATGATGTATGCTGGGATACAGGAGGAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_41306',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 41306,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 81.23485157339525,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGAGGAGGACTGACTTGCTCATTCAACCCAGGGGAAGCTGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_41362',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 41362,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 82.1865065524301,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGATGAAGAACCAGGAGGTACTTGGTAACAAAATAGAAAGAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_41406',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 41406,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.5641848256721,\n",
       "  'map_genome': 49,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCCTGTCTCCTAAGCATGTTCTCAGGTCCTGATGGTTTTGTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_41470',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 41470,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.41597326668847,\n",
       "  'map_genome': 48,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCTGAGCCCTGTGTTTTACTGTGATTGACCATGGGGAGCCAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_41621',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 41621,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 81.59835570860747,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGAGTAGAACAAAACCGAATCTGGGCATTTCACTTACCCTCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_41678',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 41678,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.6805542798819,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCATCCCTTGAGTAAGGATAAGGGATACAGTGAGACATTTGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_41722',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 41722,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 75.85069081415912,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTAGTCACACTTTCGAATGTCCCACCACATACAACACTTGAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_41817',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 41817,\n",
       "  'strand': '-',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 75.76647320153495,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACGCACAGCATACAAAATGGCACTTGGTCCCATCAAGTTGTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_41861',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 41861,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 79.01327490658514,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAAGGTCAGCACCTGCGGAGGGGGCATCTCCCACAGATTTCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_41909',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 41909,\n",
       "  'strand': '+',\n",
       "  'gc': 0.6190476190476191,\n",
       "  'tm': 85.41078039457994,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAACCTGTAAGATGCTCATCATGTGTCCCCAGAGAAGCCCTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_41961',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 41961,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.50782785043589,\n",
       "  'map_genome': 46,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGACAGTTGTCAATATCTCTTGCGTTCTAGCAATCTAAAGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_42030',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 42030,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 73.17882892735878,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAGAGTTTCTAGGGTGGGGCATAGCTTCGACAGTATGAAATC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_42115',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 42115,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.28329623680139,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGAATACAATCCAAACGCTGCCTTAAGTCTTCTTGTTTTAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_42229',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 42229,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.046352644043,\n",
       "  'map_genome': 43,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCTCTGGGCAGATACTACATTTTTAACAGGTGCTGGATCCAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_42276',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 42276,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.76505126038666,\n",
       "  'map_genome': 44,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTGTATTTTCCTAAGACCCCTCTGGTGGTGTGTGTTCAAGCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_42332',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 42332,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.414604611127,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAGAAGAGAAAAATTGGCCCTCACCGGGAAGCAGACCAGGCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_42611',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 42611,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 82.53992488382278,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGTAGATTCAGTGCTAGAATACGACTCCTGTAAGCTTTGAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_42663',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 42663,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 73.82023202292578,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGTCAGCGGGCATGGGAGCTTAGATCTCTGTTTCACCTGTCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_42707',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 42707,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 81.50651961736838,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCTGCCCTGTTGCCCTGCCATATCAACACTGCCATACAAATT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_42753',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 42753,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 81.24603801940117,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGTGCACATGTCCACTAAACAATATCTAGGTATCAGCTAAAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_42808',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 42808,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.22882954123912,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATGATACTTTAAGGTACCTAGCGGTGTCATCAAATGATTGCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_42865',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 42865,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 74.71925624696058,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGCTTAGGAAATCCTGGCAGGCTTCATGCGGGAAGCAGAAGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_42990',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 42990,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 82.80624054903456,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGCCTTGAATGGATTGTACAAATTGGAAGTGGCCAAGTCTAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_43037',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 43037,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.75420571392249,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TACCATTCCTTATGTTGTGGGGGGGGGCCCAACCATAAAATT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_43119',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 43119,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 83.16663748670095,\n",
       "  'map_genome': 49,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTGCGCGCACGCGCTATGTATATACAGGTACAGATGGTACAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_43331',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 43331,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 79.95903202466411,\n",
       "  'map_genome': 47,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCAATCCCTCAGCACATTTTTTATGATAGGTAGGGATGTAAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_43478',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 43478,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.15847822639608,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGCATTTATACCAAAACATTGAGGATTGGGAAGATGACTAAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_43603',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 43603,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.61185573897518,\n",
       "  'map_genome': 44,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTGAAGAACTGGAATCTGGTTGTGGATCTGGCAAAATGGCTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_43649',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 43649,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.60689699258796,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTCTTGGCATACAAATATGAGAGTGGAAGCTCAAGTCTTCAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_43702',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 43702,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 74.0308768325641,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAAATTCCAAGTGAGCATAGAGAACCACTTATTAGCCCTGTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_43752',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 43752,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 75.48070686044946,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GACAAATCAGCTAGTCCAGACGCTTGCTCCACGGATTCCATG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_43809',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 43809,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 79.36258404315748,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCTGAGATCAACTGAAAGGTCCCGACTTGATATATAAAATGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_43855',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 43855,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.37251143617812,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TATGCACATGCAAAAGTAGATCTAGTTGGTATGAAGGAACAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_43994',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 43994,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.39690892847938,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGGGAAGGCTTAGGGGTTTGCCCCAGATATCCAACTGTGGGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_44041',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 44041,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 85.70214913289635,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTTGAATAGTTGAGTGACATGCTTGTATGTTGGCTGTGGAGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_44104',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 44104,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 75.67100018173636,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCTGTTTGGACTGGAAGAAAGATCTCCTAATCCTGGTCACTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_44181',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 44181,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.1526385928509,\n",
       "  'map_genome': 44,\n",
       "  'map_transcriptome': 5,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AACTCCATGCCAACTGTGCAGTCAATAGATGCTTAACACTCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_44273',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 44273,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.51661694434978,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTCAAAGGTTAGAGTTGCATCTAGACCCCACAAAGGCTTTGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_44319',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 44319,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.59424375759158,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACCACTGGCTTGCCAAAACTGGCTTTTCACGATTGTCACTGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_44383',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 44383,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 79.79088376369941,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGGGTTAGGAGTCTGTCATCACCATATCGAATAAACATGACC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_44504',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 44504,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 75.80757578255657,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGCAGGAACCAATGGGAGTTCACAGTCTGAACTCCCAGCAGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_44561',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 44561,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 82.40682544110365,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGAAGGCTTTGACAGTAGTTACCTTTCACTGGAGCTGCTTCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_44799',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 44799,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.21014869581836,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTCATTCTACAGTTCTATCTAGATACCAAGTTCCTTGCCTGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_44918',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 44918,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 73.81000656142459,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAATACAATGAACTACTGATTGCAGTAGGCCAATCTTAGGGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_44985',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 44985,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.19462176541265,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAAACCCCCACTTTCAAGGGTACGAGGGTATCAGACATTTTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_45065',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 45065,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.62659849235837,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGAGCTCTGGCCTCATTGTGACCTGTAGGCCAGCTGGATGTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_45151',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 45151,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 82.72524060909188,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TAGGAAGGGGGCAGGATGTCCTGATTGCATCACAGATGGGCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_45227',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 45227,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 83.68957874882841,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGTTAAGACATTTCCACATGTTCTACTTGGCCTAGACCAAAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_45354',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 45354,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.04533933614312,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTTCACTCCCGCTCTGCAAGTGATTTCACACCAATGTCTTTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_45398',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 45398,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.1424054613247,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGAAGGACTAACCTTGCACTTACGTGATTCTCCACAGGGATT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_45496',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 45496,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.55947938148012,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATGAGGCATGGAGATCTTACCTCTGCTCAGAGAAATAAGAAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_45540',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 45540,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 74.70525822328563,\n",
       "  'map_genome': 51,\n",
       "  'map_transcriptome': 1,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCCACAGGGACTCAGGTCTCCCAAGTCAAATCACTGTAAGGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_45591',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 45591,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 80.05814511617172,\n",
       "  'map_genome': 43,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAAGCCAGTGAAGTAAGGCAATAACGTGAGCACATCCAAATT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_45673',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 45673,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.49219744292327,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGATCAAAGGGTGAATCTGATGTTCATTGTTCTTGTGAGCTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_45759',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 45759,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 73.91341210845627,\n",
       "  'map_genome': 45,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAGCTCAGGGGAACCAAATCTGTTCTGAAATGGAAGATCTGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_45838',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 45838,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.3252196495705,\n",
       "  'map_genome': 45,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCAAGGAGAAGGCAATGACAGATCAGAGCATTCTGTCACTGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_45928',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 45928,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 76.87161574986197,\n",
       "  'map_genome': 41,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGTTCACTGTCCAGCACTCATGCCCCCTCAGAAAGCTGGCTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_46004',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 46004,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 82.99429097463354,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCAGAAAATATACCATACCATGCCAGATCTATAGGATTTAGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_46309',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 46309,\n",
       "  'strand': '-',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.05135978134689,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATTCCGATTCAAACCAGCCTCCGTTTCAAGTGATGAGTGGCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_46362',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 46362,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 80.00831508504785,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGTTTGGTTACCTTTACCTTCATGTTTACTTCTCTGACTCAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_46433',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 46433,\n",
       "  'strand': '-',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.28981779469416,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CATAAGTACAGAGAAGGCCATGGTTACCATCGTCACAGTCAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_46561',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 46561,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.83998646450812,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTCCATCTTTAGCCTGACCCTGTGAATTGGGTATGTCTCTCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_46639',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 46639,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.89680726463149,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGCAGATGAAGGACTTATCAGCCAAGGTCATACCATGTATCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_46752',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 46752,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.46870948911993,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGCCCTCCACAGATTTAACTCTTTAGCTTGTTTGGGATTTTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_46825',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 46825,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.84563891446658,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGTCTATTTGAGTAGAGTCTCACTCTCTGGCCCAAAGTAGCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_46869',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 46869,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.34169019430817,\n",
       "  'map_genome': 46,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGTCCTGCTTCCTTACTAACTGATGTTGAGACTATTGCAGAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_47030',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 47030,\n",
       "  'strand': '-',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 74.93564395742521,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTCAGGTCCCCTAGTGGGAGATCTGTCTTAGCTAGCTTCATA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_47091',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 47091,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.67168659149547,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGGAACCCATTGCTGGGCACACAAAATCTCCTGGGTCTCACA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_47135',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 47135,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 82.46673773479341,\n",
       "  'map_genome': 44,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGAGTGCCATCACATGTATAACCTGAACTGGCTGTTACTCTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_47197',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 47197,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.60175185721039,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGGAAAGGGGCATTCAATAGTGACCAGATCATCTTTGAATCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_47279',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 47279,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.32240489123996,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTAAAATGACCTCCATCCTATTGACTTGCCCTCTTCAGGTGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_47346',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 47346,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.87731515329017,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCTCTAACTCCTGCATCAGGGACCCTACTCTCAGTCTAATGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_47457',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 47457,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 78.51029180270018,\n",
       "  'map_genome': 43,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCAGGGTAGGAGTGTTTTACAGCTGACCTCTCCTGTTCGAAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_48082',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 48082,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.30099009279286,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGCCTTAGCTAGCTACAGGCCTGGATAAGCTGTGTTCACCAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_48144',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 48144,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 81.0049195545102,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACTACAGGCAGAGCACGGAGTTCTGCCGGCCCCTACCCAACC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_48199',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 48199,\n",
       "  'strand': '+',\n",
       "  'gc': 0.6428571428571429,\n",
       "  'tm': 87.81666938640097,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCTGTCAAACTATAAGCCCACTTTGGAGATGCTCATCTCCTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_48243',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 48243,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.20690710965772,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAAACTCTGGTCCAGATGTACTTGGGAGGAAACTTGCAGATC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_48295',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 48295,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.98971173375253,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CACGTGCAGACAGGAGGACTCTGGCGAGAAGGCAGAGACACT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_48339',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 48339,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5952380952380952,\n",
       "  'tm': 82.3767147571885,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTGGGGCAGCCCTGGACTCAGCATGAATCCATAATCCATTAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_48399',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 48399,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 80.66070579032396,\n",
       "  'map_genome': 47,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTTAACAATGACCCATGGGACCAGCCGGTGCTCGGTGCTCGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_48469',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 48469,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5952380952380952,\n",
       "  'tm': 85.4730438345876,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAACTCTCCCGCAGCATTCGTTGGCCCAGATCGCACAGGCAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_48557',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 48557,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 84.50029853485972,\n",
       "  'map_genome': 26,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGAAGGTAAGAGACTGGTTCTGAGCTACAACTCCTCCCCACA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_48613',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 48613,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.61281326812434,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TAAAAGAGCCGTCTTCTCTGACAGGCCCTTTGAACAAATGTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_48675',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 48675,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.09948354002586,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTAATTTTCTTTTCCCGCCAAAATGCCTATAGCAGCAGCAGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_48732',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 48732,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.95996427329237,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGGGACACCTGCCATCTGACCCGTGGGTGCCATCCCTAATT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_48790',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 48790,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5952380952380952,\n",
       "  'tm': 85.85806781536962,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCTAGCAAATACAAAAGCTGGCAGCCGCTGTGGGTCAGGCTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_48847',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 48847,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 82.6403229518732,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCACAGCCGCCTCGCGTCAAGACGACGCTACACCCTTCTGGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_48908',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 48908,\n",
       "  'strand': '+',\n",
       "  'gc': 0.6666666666666667,\n",
       "  'tm': 86.59078806438134,\n",
       "  'map_genome': 26,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCCTGGGTCCCGCCCCGCTCCTTGGCCGCACTGAGGACAGCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_48982',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 48982,\n",
       "  'strand': '+',\n",
       "  'gc': 0.7380952380952381,\n",
       "  'tm': 92.97116955107413,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGGCCTTCAGCAAGAGGCAGGCGATTTAGGGACCCTGGGACC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_49062',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 49062,\n",
       "  'strand': '+',\n",
       "  'gc': 0.6190476190476191,\n",
       "  'tm': 87.06253421801705,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TAGTGTGCATTCCTCAGTCTGACTCAGTGCTTCAATCTTGAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_49106',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 49106,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 75.22041655211223,\n",
       "  'map_genome': 50,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGCACTGGCTGTAGATCGTTGTGTGTTGTTAAATGCCAGGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_49151',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 49151,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.44206751151341,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACCTTTTCTGTGTACTAGCAAAACTTTGGACACACAATTTCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_49211',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 49211,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.69887836287398,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTAAGTTGCAGAGGAGAACGTGCTTCGAAAACCAAAAGCATT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_49342',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 49342,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 76.02373633285072,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCAGTTCTAGCCCACCTAGGTATGAGTCATGCCCTGTATTCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_49441',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 49441,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 79.80990233183763,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGAGTGAATAGCTCAGGTTACATAAATTTTGGGGCAATCATC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_49527',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 49527,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 74.2808128699674,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTGATCCCATCACTCCTCCGGCATTATTTCTGGGTACTTAGA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_49581',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 49581,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 78.4770023149199,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GATGTTCAAGGTGATTATTTCTAGGTAGTGAACTTGGGTCTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_49734',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 49734,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 73.82944418858028,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAGCCATGGATGGAATGACATTATCTGTGGGTGCAGATCATT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_49884',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 49884,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 77.21623745851895,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGCCTGGGGACTCCAATATTTACAAGTTAGACACAGAAGTAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_49992',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 49992,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.86666880343978,\n",
       "  'map_genome': 49,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGGCTACACAGAGTTGGCTGTGGGCCCCTATGAGGAGGCAGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_50115',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 50115,\n",
       "  'strand': '+',\n",
       "  'gc': 0.6190476190476191,\n",
       "  'tm': 86.61093074967391,\n",
       "  'map_genome': 49,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAGACTGAAGGAACTCCACCCTGCCTGAGTAGCTTCCCACCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_50159',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 50159,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 83.48287364817992,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCACATATGTACCTTGAGAATGCCCTACTCACAAACAAGTTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_50204',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 50204,\n",
       "  'strand': '-',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 74.56077009957932,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGAGACAGGGAATTGGGGCATAATAAAATGGCCAGTGGAGAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_50274',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 50274,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 79.54237560050365,\n",
       "  'map_genome': 43,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGGCTTACATGGAACTTGAACCATGACATCCATTCTTGCCAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_50379',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 50379,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 77.62784171249842,\n",
       "  'map_genome': 41,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGCAAAAGTGACCATGATGCCAGGGAACTGGCTGGGTATGTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_50434',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 50434,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 81.7291169570426,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAATCCATATTGTAATCACGCAACAAACAAAGTCCTCAGTCC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_50483',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 50483,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 74.36235578982007,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCTCTCCTGGCTCAGGGGGCTCGTTCTCATGTTCTTTACTCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_50527',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 50527,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 81.3703574933761,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AATGACACCACAGAACCAAGCAATCCCTTGTGTGGTTGCCTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_50575',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 50575,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 80.41488279493745,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGGGACCTCCTCCAAACTTGGACAGACTGGTGGAGGTAAGGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_50662',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 50662,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 83.8291018729227,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTGCCCTTTGCCTCCTTATTAGGAATAGCTTCCTACCCCTGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_50745',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 50745,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 81.71913491890064,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAGAAAGCTGGCAAGGACTACCACCGAGGACAGGAAGTGTCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_50833',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 50833,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 81.51020587287775,\n",
       "  'map_genome': 27,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTTGGGGGGTCTCAAGAAACTCTGGGTCTAGTTTTAGAAACA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_50890',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 50890,\n",
       "  'strand': '-',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 77.52666464138287,\n",
       "  'map_genome': 43,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATTGAACTCATGTTTCTATGTTTCACACCAAGAAGCCAAGAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_50953',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 50953,\n",
       "  'strand': '-',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.30751791427116,\n",
       "  'map_genome': 49,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGATCTTGGCACTACAAAACCCTAGAAGACCTGCTTGACTTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_51057',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 51057,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.91451413760387,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGGCAAGTGCCCTGCCTTGGAACTGCATCTTGAATCCTAATT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_51122',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 51122,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 80.20874689174036,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TAATAAAACCACGTTTTGAGTTCCAAAACTGTCCCAGGTTAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_51275',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 51275,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 74.95650436121554,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTCCCAAAATCACAAGTCCCCCATAGGTACTTCAGCACCCAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_51375',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 51375,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 81.15463320689264,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACTGAACAAGGTTGGGGTAGTTACATTGGTCCAAACATTCAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_51440',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 51440,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 76.91006416248445,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTAAGATATACAATTCGTAGTCAGCCATGGTTGTTCACATCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_51544',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 51544,\n",
       "  'strand': '-',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.10827351436751,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATTCTAACCAGGACTTAAATCCTTTCTACGTTAGGCAAGAGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_51711',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 51711,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.13805652232668,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGCTACTGTGCTAATGAGCTAGTGCTCCCAGGCAATTATTTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_51755',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 51755,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 77.28971873154421,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACACCACAGGGGTCATATTAATTTTAGTCCTGTAATTAAAGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_51885',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 51885,\n",
       "  'strand': '+',\n",
       "  'gc': 0.33333333333333337,\n",
       "  'tm': 73.35824287171596,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAAAAGGACAAAAGAGTTAGGACATACATGCTTGGAATGGAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_51982',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 51982,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 74.22876703708027,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATGGAATCACACAATAACCATGTTCTTGAGAGTGGTGCTGTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_52092',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 52092,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.51695532503135,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GGAGTCTCTCCTTACACTACCGGAGAAAGTTGATAGCTGACA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_52220',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 52220,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 77.13239353587875,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CAAGGCTGGTAGAGGGTTGTATTGCTGGTGTTAGGGTAGGAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_52307',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 52307,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 80.89529840181473,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCTTTTACTTAGACACCCTGCCTTGTCATTTACTCCCATTTG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_52351',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 52351,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.22784598243646,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGTTTTTCCTTGACTCTGTGTACTTTCGAGAATCCTAGATCT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_52445',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 52445,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.12900684720933,\n",
       "  'map_genome': 45,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCCAGTAACTTTAATTGAAAGATAGCATCGCCATGTTCTTAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_52549',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 52549,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.09946820216288,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTCATGAAACTGACTGAGATGAGCTTGTGCTTCCTTGGCACC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_52643',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 52643,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 77.94743635740474,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATGAAGTTCCTTACCTAAGAAGCTCAAGTGGTATCCAGAATA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_52777',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 52777,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 74.22898042949612,\n",
       "  'map_genome': 29,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTTACCCTGAGAGCCATCCTCCCGACCTGTTTTTGAAACCAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_52833',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 52833,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5,\n",
       "  'tm': 80.56032199048656,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAAGTATCTATCATAGTAGTGTTCCTGCAGGGAAATTGGAGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_52947',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 52947,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.02492644720331,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TATGTGCTTTTGCCATGGAGCACGTGTGGAGGGAGGCTGAAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_53101',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 53101,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 82.35254464477157,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGTAAGAGGGAGGCCTAGGGCTGCTAGGGGAGAATTATCATC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_53258',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 53258,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 81.50856655014633,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCCTGTCTAGTTGCTAGTGTAGACTCAGGGTTCACAGCAAAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_53310',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 53310,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.20297811223003,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGATAGTGGCAAGTCAATTAATGAGTGTTAGTAACCAACTGC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_53370',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 53370,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.87037445814286,\n",
       "  'map_genome': 31,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCTGATTAGACTGGAGTCCAAATTCATAGCTGGCACTGTAAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_53414',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 53414,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.24649456108233,\n",
       "  'map_genome': 48,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCCATGGTGGAGAAGCCGTTGCTATTGTTTCACTAAATGGAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_53489',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 53489,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.47996350475955,\n",
       "  'map_genome': 33,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTACATTTATGCACCACCAGCTTTGGTCAGAGTAGCTCAGTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_53562',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 53562,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.94665912207017,\n",
       "  'map_genome': 36,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GCTTTGACTAGTTATGTCACTGCAGTTATTGTCCCTCACTAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_53606',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 53606,\n",
       "  'strand': '-',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 74.70476420283694,\n",
       "  'map_genome': 51,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATGCCCTGAGGCATGTATATGCCCTCACACATGAATTTGTGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_53722',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 53722,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 78.8208637926549,\n",
       "  'map_genome': 47,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GTGTCACAAAATAACCTTGGTGTCAGGTCCTGCCTTCCATTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_53766',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 53766,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 78.21299834856734,\n",
       "  'map_genome': 43,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGTGTTCCAGTCTAGCTGCCATCTAATTCCCAGCTCATTGTA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_53848',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 53848,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 77.86850913049363,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTAGGGTTACAAACACTTGCCGCCACATCTAGCTTTTGTTGT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_53896',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 53896,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 78.1501363381393,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAGGCTCAACCACACTTAAGGCTTCTACACCACCACCTCCAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_54212',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 54212,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5238095238095238,\n",
       "  'tm': 81.26338092201416,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGAGGAAGTAGATTATGGACAAACACAATCAGGACATGCCAT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_54301',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 54301,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.47783872193781,\n",
       "  'map_genome': 32,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TTCTTACATTCTTGATCCTATACAGTTGTAACTAACAGCCCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_54345',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 54345,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.35853081598248,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AACCACACAAGATTGGAACTCTCAGCATTCCCTTGCATGAAA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_54389',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 54389,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 77.03760565911688,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAGATAATGGCAGATAGAGGGGAGAGGTTCATGAGGCCCGTA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_54472',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 54472,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 80.76915286630162,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTCCCTGGGTATCGGTGGAGAGTTAATTGTAACTGGGGGAGG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_54517',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 54517,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 82.36838800677577,\n",
       "  'map_genome': 28,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CTTATGCTCTGATGAAGAACCCCATAAGAAACCCCAATGAAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_54598',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 54598,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 75.81036663362562,\n",
       "  'map_genome': 30,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GAAAGGCAAGGTGGTTGTGGGGAGTGAAAAATAGAGTACCCA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_54655',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 54655,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 79.75913511415024,\n",
       "  'map_genome': 43,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACCGTAATGAATCCCAATAAGCATAATCAATATAGGCTAACG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_54780',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 54780,\n",
       "  'strand': '+',\n",
       "  'gc': 0.35714285714285715,\n",
       "  'tm': 73.69647526447852,\n",
       "  'map_genome': 41,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TGAAAGCGGGTTACATCCCAGCGAATGGGTGGGTCTCCACAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_55073',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 55073,\n",
       "  'strand': '-',\n",
       "  'gc': 0.5714285714285715,\n",
       "  'tm': 84.24908279612151,\n",
       "  'map_genome': 26,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'GACAAGAGATGTCCGGTCGGAGCTTCAGCTTCCCTGGTGTTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_55136',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 55136,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5476190476190476,\n",
       "  'tm': 81.45315425950486,\n",
       "  'map_genome': 42,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ATGGCCCTTCATTTTAACTCTCGCTCCCATAGCCTCTCCCTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_55259',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 55259,\n",
       "  'strand': '+',\n",
       "  'gc': 0.5,\n",
       "  'tm': 81.28511596947033,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCTGGCTGATGCTCCTCCATGATGTATAATCAATACATGTTA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_55323',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 55323,\n",
       "  'strand': '+',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 75.00327593525526,\n",
       "  'map_genome': 41,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AAATTCATGACCCAGGTACTCATGTCATATCTAAGATCACAC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_55456',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 55456,\n",
       "  'strand': '-',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.01794690941603,\n",
       "  'map_genome': 40,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'CCTTGAATAAGATTATGTCCTCACTCAAAGTCCTTATCCCTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_55542',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 55542,\n",
       "  'strand': '+',\n",
       "  'gc': 0.38095238095238093,\n",
       "  'tm': 73.62666580647323,\n",
       "  'map_genome': 37,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'AGGAGGGATAGGGTCTTAAACCACACTTCTTGCATGAAATTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_55605',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 55605,\n",
       "  'strand': '-',\n",
       "  'gc': 0.40476190476190477,\n",
       "  'tm': 76.50258605664425,\n",
       "  'map_genome': 39,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACCACTCCCCCTTTCAGATCTGCCCAACTTCTTCTATGTCTT': {'name': '1:9600471-9656458_strand_+_reg_1_pb_55660',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 55660,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4761904761904762,\n",
       "  'tm': 79.27946857109725,\n",
       "  'map_genome': 38,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TCCAGTCAGAGAATGTAGGTGGTTAGGTGGAATATGACTGAG': {'name': '1:9600471-9656458_strand_+_reg_1_pb_55704',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 55704,\n",
       "  'strand': '-',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.40493055004742,\n",
       "  'map_genome': 43,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'TAGAATCTGTGATGGCTCTCCATGATCTGCACACCACAGATA': {'name': '1:9600471-9656458_strand_+_reg_1_pb_55836',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 55836,\n",
       "  'strand': '+',\n",
       "  'gc': 0.4523809523809524,\n",
       "  'tm': 76.75991782142529,\n",
       "  'map_genome': 35,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26},\n",
       " b'ACTCTTTAGTTGTATTTGAAACCGTGGGTCCTGAAGCATGTC': {'name': '1:9600471-9656458_strand_+_reg_1_pb_55920',\n",
       "  'reg_index': 1,\n",
       "  'reg_name': '1:9600471-9656458_strand_+_reg_1',\n",
       "  'pb_index': 55920,\n",
       "  'strand': '+',\n",
       "  'gc': 0.42857142857142855,\n",
       "  'tm': 76.24069656340947,\n",
       "  'map_genome': 34,\n",
       "  'map_transcriptome': 0,\n",
       "  'map_rep_genome': 0,\n",
       "  'map_self_sequences': 26}}"
      ]
     },
     "execution_count": 72,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# load to check if necessary\n",
    "pb_dict_from_load = pickle.load(open(os.path.join(library_folder, 'common_pb_dict.pkl'), 'rb'))\n",
    "\n",
    "\n",
    "print(len(pb_dict_from_load))\n",
    "pb_dict_from_load[1].kept_probes"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### filter the combined pb_dict by the genome count"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 63,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Screen probes by genome counts.\n",
      "-- filtering region:251, screen genome to keep 400 probes\n",
      "-- filtering region:252, screen genome to keep 400 probes\n",
      "-- filtering region:253, screen genome to keep 400 probes\n",
      "-- filtering region:255, screen genome to keep 400 probes\n",
      "-- filtering region:256, screen genome to keep 400 probes\n",
      "-- filtering region:257, screen genome to keep 400 probes\n",
      "-- filtering region:258, screen genome to keep 400 probes\n",
      "-- filtering region:259, screen genome to keep 400 probes\n",
      "-- filtering region:260, screen genome to keep 400 probes\n",
      "-- filtering region:261, screen genome to keep 400 probes\n",
      "-- filtering region:262, screen genome to keep 400 probes\n",
      "-- filtering region:263, screen genome to keep 400 probes\n",
      "-- filtering region:264, screen genome to keep 400 probes\n",
      "-- filtering region:265, screen genome to keep 400 probes\n",
      "-- filtering region:266, screen genome to keep 400 probes\n",
      "-- filtering region:267, screen genome to keep 400 probes\n",
      "-- filtering region:268, screen genome to keep 400 probes\n",
      "-- filtering region:269, screen genome to keep 400 probes\n",
      "-- filtering region:270, screen genome to keep 400 probes\n",
      "-- filtering region:271, screen genome to keep 400 probes\n",
      "-- filtering region:272, screen genome to keep 400 probes\n",
      "-- filtering region:273, screen genome to keep 400 probes\n",
      "-- filtering region:274, screen genome to keep 400 probes\n",
      "-- filtering region:275, screen genome to keep 400 probes\n",
      "-- filtering region:277, screen genome to keep 400 probes\n",
      "-- filtering region:278, screen genome to keep 400 probes\n",
      "-- filtering region:279, screen genome to keep 400 probes\n",
      "-- filtering region:280, screen genome to keep 400 probes\n",
      "-- filtering region:281, screen genome to keep 400 probes\n",
      "-- filtering region:282, screen genome to keep 400 probes\n",
      "-- filtering region:283, screen genome to keep 400 probes\n",
      "-- filtering region:284, screen genome to keep 400 probes\n",
      "-- filtering region:285, screen genome to keep 400 probes\n",
      "-- filtering region:286, screen genome to keep 400 probes\n",
      "-- filtering region:287, screen genome to keep 400 probes\n",
      "-- filtering region:288, screen genome to keep 400 probes\n",
      "-- filtering region:289, screen genome to keep 400 probes\n",
      "-- filtering region:290, screen genome to keep 400 probes\n",
      "-- filtering region:291, screen genome to keep 400 probes\n",
      "-- filtering region:292, screen genome to keep 400 probes\n",
      "-- filtering region:293, screen genome to keep 400 probes\n",
      "-- filtering region:295, screen genome to keep 400 probes\n",
      "-- filtering region:296, screen genome to keep 400 probes\n",
      "-- filtering region:297, screen genome to keep 400 probes\n",
      "-- filtering region:298, screen genome to keep 400 probes\n",
      "-- filtering region:299, screen genome to keep 400 probes\n",
      "-- filtering region:300, screen genome to keep 400 probes\n",
      "-- filtering region:301, screen genome to keep 400 probes\n",
      "-- filtering region:302, screen genome to keep 400 probes\n",
      "-- filtering region:303, screen genome to keep 400 probes\n",
      "-- filtering region:304, screen genome to keep 400 probes\n",
      "-- filtering region:305, screen genome to keep 400 probes\n",
      "-- filtering region:306, screen genome to keep 400 probes\n",
      "-- filtering region:307, screen genome to keep 400 probes\n",
      "-- filtering region:308, screen genome to keep 400 probes\n",
      "-- filtering region:309, screen genome to keep 400 probes\n",
      "-- filtering region:310, screen genome to keep 400 probes\n",
      "-- filtering region:311, screen genome to keep 400 probes\n",
      "-- filtering region:312, screen genome to keep 400 probes\n",
      "-- filtering region:313, screen genome to keep 400 probes\n",
      "-- filtering region:314, screen genome to keep 400 probes\n",
      "-- filtering region:315, screen genome to keep 400 probes\n",
      "-- filtering region:316, screen genome to keep 400 probes\n",
      "-- filtering region:317, screen genome to keep 400 probes\n",
      "-- filtering region:318, screen genome to keep 400 probes\n",
      "-- filtering region:319, screen genome to keep 400 probes\n",
      "-- filtering region:320, screen genome to keep 400 probes\n",
      "-- filtering region:321, screen genome to keep 400 probes\n",
      "-- filtering region:322, screen genome to keep 400 probes\n",
      "-- filtering region:323, directly append 38 probes\n",
      "-- filtering region:324, screen genome to keep 400 probes\n",
      "-- filtering region:325, screen genome to keep 400 probes\n",
      "-- filtering region:326, screen genome to keep 400 probes\n",
      "-- filtering region:327, screen genome to keep 400 probes\n",
      "-- filtering region:328, screen genome to keep 400 probes\n",
      "-- filtering region:329, screen genome to keep 400 probes\n",
      "-- filtering region:330, screen genome to keep 400 probes\n",
      "-- filtering region:332, screen genome to keep 400 probes\n",
      "-- filtering region:333, screen genome to keep 400 probes\n",
      "-- filtering region:334, screen genome to keep 400 probes\n",
      "-- filtering region:335, screen genome to keep 400 probes\n",
      "-- filtering region:336, screen genome to keep 400 probes\n",
      "-- filtering region:337, screen genome to keep 400 probes\n",
      "-- filtering region:338, screen genome to keep 400 probes\n",
      "-- filtering region:339, screen genome to keep 400 probes\n",
      "-- filtering region:340, screen genome to keep 400 probes\n",
      "-- filtering region:341, screen genome to keep 400 probes\n",
      "-- filtering region:342, screen genome to keep 400 probes\n",
      "-- filtering region:343, screen genome to keep 400 probes\n",
      "-- filtering region:344, screen genome to keep 400 probes\n",
      "-- filtering region:345, screen genome to keep 400 probes\n",
      "-- filtering region:347, screen genome to keep 400 probes\n",
      "-- filtering region:348, screen genome to keep 400 probes\n",
      "-- filtering region:349, screen genome to keep 400 probes\n",
      "-- filtering region:350, screen genome to keep 400 probes\n",
      "-- filtering region:351, screen genome to keep 400 probes\n",
      "-- filtering region:352, screen genome to keep 400 probes\n",
      "-- filtering region:353, screen genome to keep 400 probes\n",
      "-- filtering region:354, screen genome to keep 400 probes\n",
      "-- filtering region:355, screen genome to keep 400 probes\n",
      "-- filtering region:356, screen genome to keep 400 probes\n",
      "-- filtering region:357, screen genome to keep 400 probes\n",
      "-- filtering region:358, screen genome to keep 400 probes\n",
      "-- filtering region:359, screen genome to keep 400 probes\n",
      "-- filtering region:360, screen genome to keep 400 probes\n",
      "-- filtering region:361, screen genome to keep 400 probes\n",
      "-- filtering region:362, directly append 320 probes\n",
      "-- filtering region:363, screen genome to keep 400 probes\n",
      "-- filtering region:364, screen genome to keep 400 probes\n",
      "-- filtering region:365, screen genome to keep 400 probes\n",
      "-- filtering region:366, screen genome to keep 400 probes\n",
      "-- filtering region:367, screen genome to keep 400 probes\n",
      "-- filtering region:368, screen genome to keep 400 probes\n",
      "-- filtering region:369, screen genome to keep 400 probes\n",
      "-- filtering region:370, screen genome to keep 400 probes\n",
      "-- filtering region:371, screen genome to keep 400 probes\n",
      "-- filtering region:372, screen genome to keep 400 probes\n",
      "-- filtering region:373, screen genome to keep 400 probes\n",
      "-- filtering region:374, screen genome to keep 400 probes\n",
      "-- filtering region:375, screen genome to keep 400 probes\n",
      "-- filtering region:376, screen genome to keep 400 probes\n",
      "-- filtering region:377, screen genome to keep 400 probes\n",
      "-- filtering region:378, screen genome to keep 400 probes\n",
      "-- filtering region:379, screen genome to keep 400 probes\n",
      "-- filtering region:380, screen genome to keep 400 probes\n",
      "-- filtering region:381, screen genome to keep 400 probes\n",
      "-- filtering region:382, screen genome to keep 400 probes\n",
      "-- filtering region:383, screen genome to keep 400 probes\n",
      "-- filtering region:384, screen genome to keep 400 probes\n",
      "-- filtering region:385, screen genome to keep 400 probes\n",
      "-- filtering region:386, screen genome to keep 400 probes\n",
      "-- filtering region:387, screen genome to keep 400 probes\n",
      "-- filtering region:388, screen genome to keep 400 probes\n",
      "-- filtering region:389, screen genome to keep 400 probes\n",
      "-- filtering region:390, screen genome to keep 400 probes\n",
      "-- filtering region:391, screen genome to keep 400 probes\n",
      "-- filtering region:392, screen genome to keep 400 probes\n",
      "-- filtering region:393, screen genome to keep 400 probes\n",
      "-- filtering region:394, screen genome to keep 400 probes\n",
      "-- filtering region:395, screen genome to keep 400 probes\n",
      "-- filtering region:396, screen genome to keep 400 probes\n",
      "-- filtering region:397, screen genome to keep 400 probes\n",
      "-- filtering region:398, screen genome to keep 400 probes\n",
      "-- filtering region:399, screen genome to keep 400 probes\n",
      "-- filtering region:400, screen genome to keep 400 probes\n",
      "-- filtering region:401, screen genome to keep 400 probes\n",
      "-- filtering region:402, directly append 73 probes\n",
      "-- filtering region:403, screen genome to keep 400 probes\n",
      "-- filtering region:404, screen genome to keep 400 probes\n",
      "-- filtering region:405, screen genome to keep 400 probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- filtering region:406, screen genome to keep 400 probes\n",
      "-- filtering region:407, screen genome to keep 400 probes\n",
      "-- filtering region:408, screen genome to keep 400 probes\n",
      "-- filtering region:409, screen genome to keep 400 probes\n",
      "-- filtering region:410, screen genome to keep 400 probes\n",
      "-- filtering region:411, screen genome to keep 400 probes\n",
      "-- filtering region:412, screen genome to keep 400 probes\n",
      "-- filtering region:413, screen genome to keep 400 probes\n",
      "-- filtering region:414, screen genome to keep 400 probes\n",
      "-- filtering region:415, screen genome to keep 400 probes\n",
      "-- filtering region:416, screen genome to keep 400 probes\n",
      "-- filtering region:417, screen genome to keep 400 probes\n",
      "-- filtering region:418, directly append 378 probes\n",
      "-- filtering region:419, screen genome to keep 400 probes\n",
      "-- filtering region:420, screen genome to keep 400 probes\n",
      "-- filtering region:421, screen genome to keep 400 probes\n",
      "-- filtering region:422, screen genome to keep 400 probes\n",
      "-- filtering region:423, screen genome to keep 400 probes\n",
      "-- filtering region:424, screen genome to keep 400 probes\n",
      "-- filtering region:425, screen genome to keep 400 probes\n",
      "-- filtering region:426, directly append 313 probes\n",
      "-- filtering region:427, screen genome to keep 400 probes\n",
      "-- filtering region:428, screen genome to keep 400 probes\n",
      "-- filtering region:429, screen genome to keep 400 probes\n",
      "-- filtering region:430, screen genome to keep 400 probes\n",
      "-- filtering region:431, screen genome to keep 400 probes\n",
      "-- filtering region:432, screen genome to keep 400 probes\n",
      "-- filtering region:433, screen genome to keep 400 probes\n",
      "-- filtering region:434, screen genome to keep 400 probes\n",
      "-- filtering region:435, screen genome to keep 400 probes\n",
      "-- filtering region:436, screen genome to keep 400 probes\n",
      "-- filtering region:437, screen genome to keep 400 probes\n",
      "-- filtering region:438, screen genome to keep 400 probes\n",
      "-- filtering region:439, screen genome to keep 400 probes\n",
      "-- filtering region:440, screen genome to keep 400 probes\n",
      "-- filtering region:441, screen genome to keep 400 probes\n",
      "-- filtering region:442, screen genome to keep 400 probes\n",
      "-- filtering region:443, directly append 3 probes\n",
      "-- filtering region:444, screen genome to keep 400 probes\n",
      "-- filtering region:445, screen genome to keep 400 probes\n",
      "-- filtering region:446, screen genome to keep 400 probes\n",
      "-- filtering region:447, screen genome to keep 400 probes\n",
      "-- filtering region:448, screen genome to keep 400 probes\n",
      "-- filtering region:449, screen genome to keep 400 probes\n",
      "-- filtering region:450, screen genome to keep 400 probes\n",
      "-- filtering region:451, screen genome to keep 400 probes\n",
      "-- filtering region:452, screen genome to keep 400 probes\n",
      "-- filtering region:453, screen genome to keep 400 probes\n",
      "-- filtering region:454, screen genome to keep 400 probes\n",
      "-- filtering region:455, screen genome to keep 400 probes\n",
      "-- filtering region:456, screen genome to keep 400 probes\n",
      "-- filtering region:457, screen genome to keep 400 probes\n",
      "-- filtering region:458, screen genome to keep 400 probes\n",
      "-- filtering region:459, screen genome to keep 400 probes\n",
      "-- filtering region:460, screen genome to keep 400 probes\n",
      "-- filtering region:461, screen genome to keep 400 probes\n",
      "-- filtering region:462, screen genome to keep 400 probes\n",
      "-- filtering region:463, screen genome to keep 400 probes\n",
      "-- filtering region:464, screen genome to keep 400 probes\n",
      "-- filtering region:465, screen genome to keep 400 probes\n",
      "-- filtering region:466, screen genome to keep 400 probes\n",
      "-- filtering region:467, screen genome to keep 400 probes\n",
      "-- filtering region:468, screen genome to keep 400 probes\n",
      "-- filtering region:469, screen genome to keep 400 probes\n",
      "-- filtering region:470, screen genome to keep 400 probes\n",
      "-- filtering region:471, screen genome to keep 400 probes\n",
      "-- filtering region:472, screen genome to keep 400 probes\n",
      "-- filtering region:473, screen genome to keep 400 probes\n",
      "-- filtering region:474, screen genome to keep 400 probes\n",
      "-- filtering region:475, screen genome to keep 400 probes\n",
      "-- filtering region:476, screen genome to keep 400 probes\n",
      "-- filtering region:477, screen genome to keep 400 probes\n",
      "-- filtering region:478, screen genome to keep 400 probes\n",
      "-- filtering region:479, screen genome to keep 400 probes\n",
      "-- filtering region:480, screen genome to keep 400 probes\n",
      "-- filtering region:481, screen genome to keep 400 probes\n",
      "-- filtering region:482, screen genome to keep 400 probes\n",
      "-- filtering region:483, screen genome to keep 400 probes\n",
      "-- filtering region:484, screen genome to keep 400 probes\n",
      "-- filtering region:485, screen genome to keep 400 probes\n",
      "-- filtering region:486, screen genome to keep 400 probes\n",
      "-- filtering region:487, screen genome to keep 400 probes\n",
      "-- filtering region:488, screen genome to keep 400 probes\n",
      "-- filtering region:489, screen genome to keep 400 probes\n",
      "-- filtering region:490, directly append 19 probes\n",
      "-- filtering region:491, screen genome to keep 400 probes\n",
      "-- filtering region:492, screen genome to keep 400 probes\n",
      "-- filtering region:493, screen genome to keep 400 probes\n",
      "-- filtering region:494, screen genome to keep 400 probes\n",
      "-- filtering region:495, screen genome to keep 400 probes\n",
      "-- filtering region:496, screen genome to keep 400 probes\n",
      "-- filtering region:497, screen genome to keep 400 probes\n",
      "-- filtering region:498, screen genome to keep 400 probes\n",
      "-- filtering region:499, screen genome to keep 400 probes\n",
      "-- filtering region:500, screen genome to keep 400 probes\n",
      "-- filtering region:501, screen genome to keep 400 probes\n",
      "-- filtering region:502, screen genome to keep 400 probes\n",
      "-- filtering region:503, screen genome to keep 400 probes\n",
      "-- filtering region:504, screen genome to keep 400 probes\n",
      "-- filtering region:505, screen genome to keep 400 probes\n",
      "-- filtering region:506, screen genome to keep 400 probes\n",
      "-- filtering region:507, screen genome to keep 400 probes\n",
      "-- filtering region:508, screen genome to keep 400 probes\n",
      "-- filtering region:509, screen genome to keep 400 probes\n",
      "-- filtering region:510, screen genome to keep 400 probes\n",
      "-- filtering region:511, screen genome to keep 400 probes\n",
      "-- filtering region:512, screen genome to keep 400 probes\n",
      "-- filtering region:513, screen genome to keep 400 probes\n",
      "-- filtering region:514, screen genome to keep 400 probes\n",
      "-- filtering region:515, screen genome to keep 400 probes\n",
      "-- filtering region:516, screen genome to keep 400 probes\n",
      "-- filtering region:517, screen genome to keep 400 probes\n",
      "-- filtering region:518, screen genome to keep 400 probes\n",
      "-- filtering region:519, screen genome to keep 400 probes\n",
      "-- filtering region:520, screen genome to keep 400 probes\n",
      "-- filtering region:521, screen genome to keep 400 probes\n",
      "-- filtering region:523, screen genome to keep 400 probes\n",
      "-- filtering region:524, screen genome to keep 400 probes\n",
      "-- filtering region:525, screen genome to keep 400 probes\n",
      "-- filtering region:526, screen genome to keep 400 probes\n",
      "-- filtering region:527, screen genome to keep 400 probes\n",
      "-- filtering region:528, screen genome to keep 400 probes\n",
      "-- filtering region:529, screen genome to keep 400 probes\n",
      "-- filtering region:530, screen genome to keep 400 probes\n",
      "-- filtering region:531, screen genome to keep 400 probes\n",
      "-- filtering region:532, screen genome to keep 400 probes\n",
      "-- filtering region:533, screen genome to keep 400 probes\n",
      "-- filtering region:534, screen genome to keep 400 probes\n",
      "-- filtering region:535, screen genome to keep 400 probes\n",
      "-- filtering region:536, screen genome to keep 400 probes\n",
      "-- filtering region:537, screen genome to keep 400 probes\n",
      "-- filtering region:538, screen genome to keep 400 probes\n",
      "-- filtering region:539, directly append 268 probes\n",
      "-- filtering region:540, screen genome to keep 400 probes\n",
      "-- filtering region:541, screen genome to keep 400 probes\n",
      "-- filtering region:542, screen genome to keep 400 probes\n",
      "-- filtering region:543, screen genome to keep 400 probes\n",
      "-- filtering region:544, screen genome to keep 400 probes\n",
      "-- filtering region:545, screen genome to keep 400 probes\n",
      "-- filtering region:546, screen genome to keep 400 probes\n",
      "-- filtering region:547, screen genome to keep 400 probes\n",
      "-- filtering region:548, screen genome to keep 400 probes\n",
      "-- filtering region:549, screen genome to keep 400 probes\n",
      "-- filtering region:550, screen genome to keep 400 probes\n",
      "-- filtering region:551, screen genome to keep 400 probes\n",
      "-- filtering region:552, screen genome to keep 400 probes\n",
      "-- filtering region:553, screen genome to keep 400 probes\n",
      "-- filtering region:554, screen genome to keep 400 probes\n",
      "-- filtering region:555, screen genome to keep 400 probes\n",
      "-- filtering region:556, screen genome to keep 400 probes\n",
      "-- filtering region:557, screen genome to keep 400 probes\n",
      "-- filtering region:558, screen genome to keep 400 probes\n",
      "-- filtering region:559, screen genome to keep 400 probes\n",
      "-- filtering region:560, screen genome to keep 400 probes\n",
      "-- filtering region:561, screen genome to keep 400 probes\n",
      "-- filtering region:562, screen genome to keep 400 probes\n",
      "-- filtering region:563, screen genome to keep 400 probes\n",
      "-- filtering region:564, screen genome to keep 400 probes\n",
      "-- filtering region:565, screen genome to keep 400 probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- filtering region:566, screen genome to keep 400 probes\n",
      "-- filtering region:567, screen genome to keep 400 probes\n",
      "-- filtering region:568, screen genome to keep 400 probes\n",
      "-- filtering region:569, screen genome to keep 400 probes\n",
      "-- filtering region:570, screen genome to keep 400 probes\n",
      "-- filtering region:571, screen genome to keep 400 probes\n",
      "-- filtering region:572, screen genome to keep 400 probes\n",
      "-- filtering region:573, screen genome to keep 400 probes\n",
      "-- filtering region:574, screen genome to keep 400 probes\n",
      "-- filtering region:575, screen genome to keep 400 probes\n",
      "-- filtering region:576, screen genome to keep 400 probes\n",
      "-- filtering region:577, screen genome to keep 400 probes\n",
      "-- filtering region:579, screen genome to keep 400 probes\n",
      "-- filtering region:580, screen genome to keep 400 probes\n",
      "-- filtering region:581, screen genome to keep 400 probes\n",
      "-- filtering region:582, screen genome to keep 400 probes\n",
      "-- filtering region:583, screen genome to keep 400 probes\n",
      "-- filtering region:584, screen genome to keep 400 probes\n",
      "-- filtering region:585, screen genome to keep 400 probes\n",
      "-- filtering region:586, screen genome to keep 400 probes\n",
      "-- filtering region:587, screen genome to keep 400 probes\n",
      "-- filtering region:588, screen genome to keep 400 probes\n",
      "-- filtering region:589, screen genome to keep 400 probes\n",
      "-- filtering region:590, screen genome to keep 400 probes\n",
      "-- filtering region:591, screen genome to keep 400 probes\n",
      "-- filtering region:592, screen genome to keep 400 probes\n",
      "-- filtering region:593, screen genome to keep 400 probes\n",
      "-- filtering region:594, screen genome to keep 400 probes\n",
      "-- filtering region:595, screen genome to keep 400 probes\n",
      "-- filtering region:596, screen genome to keep 400 probes\n",
      "-- filtering region:597, screen genome to keep 400 probes\n",
      "-- filtering region:598, screen genome to keep 400 probes\n",
      "-- filtering region:599, screen genome to keep 400 probes\n",
      "-- filtering region:600, screen genome to keep 400 probes\n",
      "-- filtering region:601, screen genome to keep 400 probes\n",
      "-- filtering region:602, screen genome to keep 400 probes\n",
      "-- filtering region:603, screen genome to keep 400 probes\n",
      "-- filtering region:604, screen genome to keep 400 probes\n",
      "-- filtering region:605, screen genome to keep 400 probes\n",
      "-- filtering region:606, screen genome to keep 400 probes\n",
      "-- filtering region:607, screen genome to keep 400 probes\n",
      "-- filtering region:608, screen genome to keep 400 probes\n",
      "-- filtering region:609, screen genome to keep 400 probes\n",
      "-- filtering region:610, screen genome to keep 400 probes\n",
      "-- filtering region:611, screen genome to keep 400 probes\n",
      "-- filtering region:612, screen genome to keep 400 probes\n",
      "-- filtering region:613, screen genome to keep 400 probes\n",
      "-- filtering region:614, screen genome to keep 400 probes\n",
      "-- filtering region:615, screen genome to keep 400 probes\n",
      "-- filtering region:616, screen genome to keep 400 probes\n",
      "-- filtering region:617, screen genome to keep 400 probes\n",
      "-- filtering region:618, screen genome to keep 400 probes\n",
      "-- filtering region:619, screen genome to keep 400 probes\n",
      "-- filtering region:620, screen genome to keep 400 probes\n",
      "-- filtering region:621, screen genome to keep 400 probes\n",
      "-- filtering region:622, screen genome to keep 400 probes\n",
      "-- filtering region:623, screen genome to keep 400 probes\n",
      "-- filtering region:624, screen genome to keep 400 probes\n",
      "-- filtering region:625, screen genome to keep 400 probes\n",
      "-- filtering region:626, screen genome to keep 400 probes\n",
      "-- filtering region:627, screen genome to keep 400 probes\n",
      "-- filtering region:628, screen genome to keep 400 probes\n",
      "-- filtering region:629, screen genome to keep 400 probes\n",
      "-- filtering region:630, screen genome to keep 400 probes\n",
      "-- filtering region:631, screen genome to keep 400 probes\n",
      "-- filtering region:632, screen genome to keep 400 probes\n",
      "-- filtering region:633, screen genome to keep 400 probes\n",
      "-- filtering region:634, screen genome to keep 400 probes\n",
      "-- filtering region:635, screen genome to keep 400 probes\n",
      "-- filtering region:636, screen genome to keep 400 probes\n",
      "-- filtering region:637, screen genome to keep 400 probes\n",
      "-- filtering region:638, screen genome to keep 400 probes\n",
      "-- filtering region:639, screen genome to keep 400 probes\n",
      "-- filtering region:640, screen genome to keep 400 probes\n",
      "-- filtering region:641, screen genome to keep 400 probes\n",
      "-- filtering region:642, screen genome to keep 400 probes\n",
      "-- filtering region:643, screen genome to keep 400 probes\n",
      "-- filtering region:644, screen genome to keep 400 probes\n",
      "-- filtering region:645, screen genome to keep 400 probes\n",
      "-- filtering region:646, screen genome to keep 400 probes\n",
      "-- filtering region:647, screen genome to keep 400 probes\n",
      "-- filtering region:648, screen genome to keep 400 probes\n",
      "-- filtering region:649, screen genome to keep 400 probes\n",
      "-- filtering region:650, screen genome to keep 400 probes\n",
      "-- filtering region:651, screen genome to keep 400 probes\n",
      "-- filtering region:652, screen genome to keep 400 probes\n",
      "-- filtering region:653, screen genome to keep 400 probes\n",
      "-- filtering region:654, screen genome to keep 400 probes\n",
      "-- filtering region:655, screen genome to keep 400 probes\n",
      "-- filtering region:656, screen genome to keep 400 probes\n",
      "-- filtering region:657, screen genome to keep 400 probes\n",
      "-- filtering region:658, screen genome to keep 400 probes\n",
      "-- filtering region:659, screen genome to keep 400 probes\n",
      "-- filtering region:660, screen genome to keep 400 probes\n",
      "-- filtering region:661, screen genome to keep 400 probes\n",
      "-- filtering region:662, screen genome to keep 400 probes\n",
      "-- filtering region:663, screen genome to keep 400 probes\n",
      "-- filtering region:664, screen genome to keep 400 probes\n",
      "-- filtering region:665, screen genome to keep 400 probes\n",
      "-- filtering region:666, screen genome to keep 400 probes\n",
      "-- filtering region:667, screen genome to keep 400 probes\n",
      "-- filtering region:668, screen genome to keep 400 probes\n",
      "-- filtering region:669, screen genome to keep 400 probes\n",
      "-- filtering region:670, screen genome to keep 400 probes\n",
      "-- filtering region:671, screen genome to keep 400 probes\n",
      "-- filtering region:672, screen genome to keep 400 probes\n",
      "-- filtering region:673, screen genome to keep 400 probes\n",
      "-- filtering region:674, screen genome to keep 400 probes\n",
      "-- filtering region:675, screen genome to keep 400 probes\n",
      "-- filtering region:676, screen genome to keep 400 probes\n",
      "-- filtering region:677, screen genome to keep 400 probes\n",
      "-- filtering region:678, screen genome to keep 400 probes\n",
      "-- filtering region:679, screen genome to keep 400 probes\n",
      "-- filtering region:680, screen genome to keep 400 probes\n",
      "-- filtering region:681, screen genome to keep 400 probes\n",
      "-- filtering region:682, screen genome to keep 400 probes\n",
      "-- filtering region:683, screen genome to keep 400 probes\n",
      "-- filtering region:684, screen genome to keep 400 probes\n",
      "-- filtering region:685, screen genome to keep 400 probes\n",
      "-- filtering region:687, screen genome to keep 400 probes\n",
      "-- filtering region:688, screen genome to keep 400 probes\n",
      "-- filtering region:689, screen genome to keep 400 probes\n",
      "-- filtering region:690, screen genome to keep 400 probes\n",
      "-- filtering region:691, screen genome to keep 400 probes\n",
      "-- filtering region:692, screen genome to keep 400 probes\n",
      "-- filtering region:693, screen genome to keep 400 probes\n",
      "-- filtering region:694, screen genome to keep 400 probes\n",
      "-- filtering region:695, screen genome to keep 400 probes\n",
      "-- filtering region:696, screen genome to keep 400 probes\n",
      "-- filtering region:697, screen genome to keep 400 probes\n",
      "-- filtering region:698, screen genome to keep 400 probes\n",
      "-- filtering region:699, screen genome to keep 400 probes\n",
      "-- filtering region:700, screen genome to keep 400 probes\n",
      "-- filtering region:701, screen genome to keep 400 probes\n",
      "-- filtering region:702, screen genome to keep 400 probes\n",
      "-- filtering region:703, screen genome to keep 400 probes\n",
      "-- filtering region:704, screen genome to keep 400 probes\n",
      "-- filtering region:705, screen genome to keep 400 probes\n",
      "-- filtering region:706, screen genome to keep 400 probes\n",
      "-- filtering region:707, screen genome to keep 400 probes\n",
      "-- filtering region:708, screen genome to keep 400 probes\n",
      "-- filtering region:709, screen genome to keep 400 probes\n",
      "-- filtering region:710, screen genome to keep 400 probes\n",
      "-- filtering region:711, screen genome to keep 400 probes\n",
      "-- filtering region:712, screen genome to keep 400 probes\n",
      "-- filtering region:713, screen genome to keep 400 probes\n",
      "-- filtering region:714, screen genome to keep 400 probes\n",
      "-- filtering region:715, screen genome to keep 400 probes\n",
      "-- filtering region:716, screen genome to keep 400 probes\n",
      "-- filtering region:717, screen genome to keep 400 probes\n",
      "-- filtering region:718, screen genome to keep 400 probes\n",
      "-- filtering region:719, screen genome to keep 400 probes\n",
      "-- filtering region:720, screen genome to keep 400 probes\n",
      "-- filtering region:721, screen genome to keep 400 probes\n",
      "-- filtering region:722, screen genome to keep 400 probes\n",
      "-- filtering region:723, screen genome to keep 400 probes\n",
      "-- filtering region:724, screen genome to keep 400 probes\n",
      "-- filtering region:725, screen genome to keep 400 probes\n",
      "-- filtering region:726, screen genome to keep 400 probes\n",
      "-- filtering region:727, screen genome to keep 400 probes\n",
      "-- filtering region:728, screen genome to keep 400 probes\n",
      "-- filtering region:729, screen genome to keep 400 probes\n",
      "-- filtering region:730, screen genome to keep 400 probes\n",
      "-- filtering region:731, screen genome to keep 400 probes\n",
      "-- filtering region:732, screen genome to keep 400 probes\n",
      "-- filtering region:733, screen genome to keep 400 probes\n",
      "-- filtering region:734, screen genome to keep 400 probes\n",
      "-- filtering region:735, screen genome to keep 400 probes\n",
      "-- filtering region:736, screen genome to keep 400 probes\n",
      "-- filtering region:737, screen genome to keep 400 probes\n",
      "-- filtering region:738, screen genome to keep 400 probes\n",
      "-- filtering region:739, screen genome to keep 400 probes\n",
      "-- filtering region:740, screen genome to keep 400 probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- filtering region:741, screen genome to keep 400 probes\n",
      "-- filtering region:742, screen genome to keep 400 probes\n",
      "-- filtering region:743, screen genome to keep 400 probes\n",
      "-- filtering region:744, screen genome to keep 400 probes\n",
      "-- filtering region:745, screen genome to keep 400 probes\n",
      "-- filtering region:746, screen genome to keep 400 probes\n",
      "-- filtering region:747, screen genome to keep 400 probes\n",
      "-- filtering region:748, screen genome to keep 400 probes\n",
      "-- filtering region:749, screen genome to keep 400 probes\n",
      "-- filtering region:750, screen genome to keep 400 probes\n",
      "-- filtering region:1, screen genome to keep 400 probes\n",
      "-- filtering region:2, screen genome to keep 400 probes\n",
      "-- filtering region:3, screen genome to keep 400 probes\n",
      "-- filtering region:4, screen genome to keep 400 probes\n",
      "-- filtering region:5, screen genome to keep 400 probes\n",
      "-- filtering region:6, screen genome to keep 400 probes\n",
      "-- filtering region:7, screen genome to keep 400 probes\n",
      "-- filtering region:8, screen genome to keep 400 probes\n",
      "-- filtering region:9, screen genome to keep 400 probes\n",
      "-- filtering region:10, screen genome to keep 400 probes\n",
      "-- filtering region:11, screen genome to keep 400 probes\n",
      "-- filtering region:12, screen genome to keep 400 probes\n",
      "-- filtering region:13, screen genome to keep 400 probes\n",
      "-- filtering region:14, screen genome to keep 400 probes\n",
      "-- filtering region:15, screen genome to keep 400 probes\n",
      "-- filtering region:16, screen genome to keep 400 probes\n",
      "-- filtering region:17, screen genome to keep 400 probes\n",
      "-- filtering region:18, screen genome to keep 400 probes\n",
      "-- filtering region:19, screen genome to keep 400 probes\n",
      "-- filtering region:20, screen genome to keep 400 probes\n",
      "-- filtering region:21, screen genome to keep 400 probes\n",
      "-- filtering region:22, screen genome to keep 400 probes\n",
      "-- filtering region:23, screen genome to keep 400 probes\n",
      "-- filtering region:24, screen genome to keep 400 probes\n",
      "-- filtering region:25, screen genome to keep 400 probes\n",
      "-- filtering region:26, screen genome to keep 400 probes\n",
      "-- filtering region:27, screen genome to keep 400 probes\n",
      "-- filtering region:28, screen genome to keep 400 probes\n",
      "-- filtering region:29, screen genome to keep 400 probes\n",
      "-- filtering region:30, screen genome to keep 400 probes\n",
      "-- filtering region:31, screen genome to keep 400 probes\n",
      "-- filtering region:32, screen genome to keep 400 probes\n",
      "-- filtering region:33, screen genome to keep 400 probes\n",
      "-- filtering region:34, screen genome to keep 400 probes\n",
      "-- filtering region:35, screen genome to keep 400 probes\n",
      "-- filtering region:36, screen genome to keep 400 probes\n",
      "-- filtering region:37, screen genome to keep 400 probes\n",
      "-- filtering region:38, screen genome to keep 400 probes\n",
      "-- filtering region:39, screen genome to keep 400 probes\n",
      "-- filtering region:40, screen genome to keep 400 probes\n",
      "-- filtering region:41, screen genome to keep 400 probes\n",
      "-- filtering region:42, screen genome to keep 400 probes\n",
      "-- filtering region:43, screen genome to keep 400 probes\n",
      "-- filtering region:44, screen genome to keep 400 probes\n",
      "-- filtering region:45, screen genome to keep 400 probes\n",
      "-- filtering region:46, screen genome to keep 400 probes\n",
      "-- filtering region:47, screen genome to keep 400 probes\n",
      "-- filtering region:48, screen genome to keep 400 probes\n",
      "-- filtering region:49, screen genome to keep 400 probes\n",
      "-- filtering region:50, screen genome to keep 400 probes\n",
      "-- filtering region:51, screen genome to keep 400 probes\n",
      "-- filtering region:52, screen genome to keep 400 probes\n",
      "-- filtering region:53, screen genome to keep 400 probes\n",
      "-- filtering region:54, screen genome to keep 400 probes\n",
      "-- filtering region:55, screen genome to keep 400 probes\n",
      "-- filtering region:56, screen genome to keep 400 probes\n",
      "-- filtering region:57, screen genome to keep 400 probes\n",
      "-- filtering region:58, screen genome to keep 400 probes\n",
      "-- filtering region:59, screen genome to keep 400 probes\n",
      "-- filtering region:60, screen genome to keep 400 probes\n",
      "-- filtering region:61, screen genome to keep 400 probes\n",
      "-- filtering region:62, screen genome to keep 400 probes\n",
      "-- filtering region:63, screen genome to keep 400 probes\n",
      "-- filtering region:64, screen genome to keep 400 probes\n",
      "-- filtering region:65, screen genome to keep 400 probes\n",
      "-- filtering region:66, screen genome to keep 400 probes\n",
      "-- filtering region:67, screen genome to keep 400 probes\n",
      "-- filtering region:68, screen genome to keep 400 probes\n",
      "-- filtering region:69, screen genome to keep 400 probes\n",
      "-- filtering region:70, screen genome to keep 400 probes\n",
      "-- filtering region:71, screen genome to keep 400 probes\n",
      "-- filtering region:72, screen genome to keep 400 probes\n",
      "-- filtering region:73, screen genome to keep 400 probes\n",
      "-- filtering region:74, screen genome to keep 400 probes\n",
      "-- filtering region:75, screen genome to keep 400 probes\n",
      "-- filtering region:76, screen genome to keep 400 probes\n",
      "-- filtering region:77, screen genome to keep 400 probes\n",
      "-- filtering region:78, screen genome to keep 400 probes\n",
      "-- filtering region:79, screen genome to keep 400 probes\n",
      "-- filtering region:80, screen genome to keep 400 probes\n",
      "-- filtering region:81, screen genome to keep 400 probes\n",
      "-- filtering region:82, screen genome to keep 400 probes\n",
      "-- filtering region:83, screen genome to keep 400 probes\n",
      "-- filtering region:84, screen genome to keep 400 probes\n",
      "-- filtering region:85, screen genome to keep 400 probes\n",
      "-- filtering region:86, screen genome to keep 400 probes\n",
      "-- filtering region:87, screen genome to keep 400 probes\n",
      "-- filtering region:88, screen genome to keep 400 probes\n",
      "-- filtering region:89, screen genome to keep 400 probes\n",
      "-- filtering region:90, screen genome to keep 400 probes\n",
      "-- filtering region:91, screen genome to keep 400 probes\n",
      "-- filtering region:92, screen genome to keep 400 probes\n",
      "-- filtering region:93, screen genome to keep 400 probes\n",
      "-- filtering region:94, screen genome to keep 400 probes\n",
      "-- filtering region:95, screen genome to keep 400 probes\n",
      "-- filtering region:96, screen genome to keep 400 probes\n",
      "-- filtering region:97, screen genome to keep 400 probes\n",
      "-- filtering region:98, screen genome to keep 400 probes\n",
      "-- filtering region:99, screen genome to keep 400 probes\n",
      "-- filtering region:100, screen genome to keep 400 probes\n",
      "-- filtering region:101, screen genome to keep 400 probes\n",
      "-- filtering region:102, screen genome to keep 400 probes\n",
      "-- filtering region:103, screen genome to keep 400 probes\n",
      "-- filtering region:104, screen genome to keep 400 probes\n",
      "-- filtering region:105, screen genome to keep 400 probes\n",
      "-- filtering region:106, screen genome to keep 400 probes\n",
      "-- filtering region:107, screen genome to keep 400 probes\n",
      "-- filtering region:108, screen genome to keep 400 probes\n",
      "-- filtering region:109, screen genome to keep 400 probes\n",
      "-- filtering region:111, screen genome to keep 400 probes\n",
      "-- filtering region:112, screen genome to keep 400 probes\n",
      "-- filtering region:113, screen genome to keep 400 probes\n",
      "-- filtering region:114, screen genome to keep 400 probes\n",
      "-- filtering region:115, screen genome to keep 400 probes\n",
      "-- filtering region:116, screen genome to keep 400 probes\n",
      "-- filtering region:117, screen genome to keep 400 probes\n",
      "-- filtering region:118, screen genome to keep 400 probes\n",
      "-- filtering region:119, screen genome to keep 400 probes\n",
      "-- filtering region:120, screen genome to keep 400 probes\n",
      "-- filtering region:121, screen genome to keep 400 probes\n",
      "-- filtering region:122, screen genome to keep 400 probes\n",
      "-- filtering region:123, screen genome to keep 400 probes\n",
      "-- filtering region:124, screen genome to keep 400 probes\n",
      "-- filtering region:125, screen genome to keep 400 probes\n",
      "-- filtering region:126, screen genome to keep 400 probes\n",
      "-- filtering region:127, screen genome to keep 400 probes\n",
      "-- filtering region:129, screen genome to keep 400 probes\n",
      "-- filtering region:130, screen genome to keep 400 probes\n",
      "-- filtering region:131, screen genome to keep 400 probes\n",
      "-- filtering region:132, screen genome to keep 400 probes\n",
      "-- filtering region:133, screen genome to keep 400 probes\n",
      "-- filtering region:134, screen genome to keep 400 probes\n",
      "-- filtering region:135, screen genome to keep 400 probes\n",
      "-- filtering region:136, screen genome to keep 400 probes\n",
      "-- filtering region:137, screen genome to keep 400 probes\n",
      "-- filtering region:138, screen genome to keep 400 probes\n",
      "-- filtering region:139, screen genome to keep 400 probes\n",
      "-- filtering region:140, screen genome to keep 400 probes\n",
      "-- filtering region:141, screen genome to keep 400 probes\n",
      "-- filtering region:142, screen genome to keep 400 probes\n",
      "-- filtering region:143, screen genome to keep 400 probes\n",
      "-- filtering region:144, screen genome to keep 400 probes\n",
      "-- filtering region:145, screen genome to keep 400 probes\n",
      "-- filtering region:146, screen genome to keep 400 probes\n",
      "-- filtering region:147, screen genome to keep 400 probes\n",
      "-- filtering region:148, screen genome to keep 400 probes\n",
      "-- filtering region:149, screen genome to keep 400 probes\n",
      "-- filtering region:150, screen genome to keep 400 probes\n",
      "-- filtering region:151, screen genome to keep 400 probes\n",
      "-- filtering region:152, screen genome to keep 400 probes\n",
      "-- filtering region:153, screen genome to keep 400 probes\n",
      "-- filtering region:154, screen genome to keep 400 probes\n",
      "-- filtering region:155, screen genome to keep 400 probes\n",
      "-- filtering region:156, screen genome to keep 400 probes\n",
      "-- filtering region:157, screen genome to keep 400 probes\n",
      "-- filtering region:158, screen genome to keep 400 probes\n",
      "-- filtering region:159, screen genome to keep 400 probes\n",
      "-- filtering region:160, screen genome to keep 400 probes\n",
      "-- filtering region:161, screen genome to keep 400 probes\n",
      "-- filtering region:162, screen genome to keep 400 probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- filtering region:163, screen genome to keep 400 probes\n",
      "-- filtering region:164, screen genome to keep 400 probes\n",
      "-- filtering region:165, screen genome to keep 400 probes\n",
      "-- filtering region:166, screen genome to keep 400 probes\n",
      "-- filtering region:168, screen genome to keep 400 probes\n",
      "-- filtering region:169, screen genome to keep 400 probes\n",
      "-- filtering region:170, screen genome to keep 400 probes\n",
      "-- filtering region:171, screen genome to keep 400 probes\n",
      "-- filtering region:172, screen genome to keep 400 probes\n",
      "-- filtering region:173, screen genome to keep 400 probes\n",
      "-- filtering region:174, screen genome to keep 400 probes\n",
      "-- filtering region:175, screen genome to keep 400 probes\n",
      "-- filtering region:176, screen genome to keep 400 probes\n",
      "-- filtering region:177, screen genome to keep 400 probes\n",
      "-- filtering region:178, screen genome to keep 400 probes\n",
      "-- filtering region:179, screen genome to keep 400 probes\n",
      "-- filtering region:180, screen genome to keep 400 probes\n",
      "-- filtering region:181, screen genome to keep 400 probes\n",
      "-- filtering region:182, screen genome to keep 400 probes\n",
      "-- filtering region:183, screen genome to keep 400 probes\n",
      "-- filtering region:184, screen genome to keep 400 probes\n",
      "-- filtering region:185, directly append 115 probes\n",
      "-- filtering region:186, screen genome to keep 400 probes\n",
      "-- filtering region:187, screen genome to keep 400 probes\n",
      "-- filtering region:188, screen genome to keep 400 probes\n",
      "-- filtering region:189, screen genome to keep 400 probes\n",
      "-- filtering region:190, screen genome to keep 400 probes\n",
      "-- filtering region:191, screen genome to keep 400 probes\n",
      "-- filtering region:192, screen genome to keep 400 probes\n",
      "-- filtering region:193, screen genome to keep 400 probes\n",
      "-- filtering region:194, screen genome to keep 400 probes\n",
      "-- filtering region:195, screen genome to keep 400 probes\n",
      "-- filtering region:196, screen genome to keep 400 probes\n",
      "-- filtering region:197, screen genome to keep 400 probes\n",
      "-- filtering region:198, screen genome to keep 400 probes\n",
      "-- filtering region:199, screen genome to keep 400 probes\n",
      "-- filtering region:200, screen genome to keep 400 probes\n",
      "-- filtering region:201, screen genome to keep 400 probes\n",
      "-- filtering region:202, screen genome to keep 400 probes\n",
      "-- filtering region:203, screen genome to keep 400 probes\n",
      "-- filtering region:204, screen genome to keep 400 probes\n",
      "-- filtering region:205, screen genome to keep 400 probes\n",
      "-- filtering region:206, screen genome to keep 400 probes\n",
      "-- filtering region:207, screen genome to keep 400 probes\n",
      "-- filtering region:208, screen genome to keep 400 probes\n",
      "-- filtering region:209, screen genome to keep 400 probes\n",
      "-- filtering region:210, screen genome to keep 400 probes\n",
      "-- filtering region:211, screen genome to keep 400 probes\n",
      "-- filtering region:212, directly append 209 probes\n",
      "-- filtering region:213, directly append 283 probes\n",
      "-- filtering region:214, screen genome to keep 400 probes\n",
      "-- filtering region:215, screen genome to keep 400 probes\n",
      "-- filtering region:216, screen genome to keep 400 probes\n",
      "-- filtering region:217, screen genome to keep 400 probes\n",
      "-- filtering region:218, screen genome to keep 400 probes\n",
      "-- filtering region:219, screen genome to keep 400 probes\n",
      "-- filtering region:220, screen genome to keep 400 probes\n",
      "-- filtering region:221, screen genome to keep 400 probes\n",
      "-- filtering region:222, screen genome to keep 400 probes\n",
      "-- filtering region:223, screen genome to keep 400 probes\n",
      "-- filtering region:224, screen genome to keep 400 probes\n",
      "-- filtering region:225, screen genome to keep 400 probes\n",
      "-- filtering region:226, screen genome to keep 400 probes\n",
      "-- filtering region:227, directly append 6 probes\n",
      "-- filtering region:228, screen genome to keep 400 probes\n",
      "-- filtering region:229, screen genome to keep 400 probes\n",
      "-- filtering region:230, screen genome to keep 400 probes\n",
      "-- filtering region:231, screen genome to keep 400 probes\n",
      "-- filtering region:232, screen genome to keep 400 probes\n",
      "-- filtering region:233, screen genome to keep 400 probes\n",
      "-- filtering region:234, screen genome to keep 400 probes\n",
      "-- filtering region:235, screen genome to keep 400 probes\n",
      "-- filtering region:236, screen genome to keep 400 probes\n",
      "-- filtering region:237, screen genome to keep 400 probes\n",
      "-- filtering region:238, screen genome to keep 400 probes\n",
      "-- filtering region:239, screen genome to keep 400 probes\n",
      "-- filtering region:240, screen genome to keep 400 probes\n",
      "-- filtering region:241, screen genome to keep 400 probes\n",
      "-- filtering region:242, screen genome to keep 400 probes\n",
      "-- filtering region:243, screen genome to keep 400 probes\n",
      "-- filtering region:244, screen genome to keep 400 probes\n",
      "-- filtering region:245, screen genome to keep 400 probes\n",
      "-- filtering region:246, screen genome to keep 400 probes\n",
      "-- filtering region:247, screen genome to keep 400 probes\n",
      "-- filtering region:248, screen genome to keep 400 probes\n",
      "-- filtering region:249, screen genome to keep 400 probes\n",
      "-- filtering region:250, screen genome to keep 400 probes\n",
      "-- filtering region:751, screen genome to keep 400 probes\n",
      "-- filtering region:752, directly append 176 probes\n",
      "-- filtering region:756, directly append 8 probes\n",
      "-- filtering region:757, directly append 8 probes\n",
      "-- filtering region:758, screen genome to keep 400 probes\n",
      "-- filtering region:759, screen genome to keep 400 probes\n",
      "-- filtering region:760, screen genome to keep 400 probes\n",
      "-- filtering region:761, screen genome to keep 400 probes\n",
      "-- filtering region:762, screen genome to keep 400 probes\n",
      "-- filtering region:763, screen genome to keep 400 probes\n",
      "-- filtering region:764, screen genome to keep 400 probes\n",
      "-- filtering region:765, screen genome to keep 400 probes\n",
      "-- filtering region:766, screen genome to keep 400 probes\n",
      "-- filtering region:767, screen genome to keep 400 probes\n",
      "-- filtering region:768, screen genome to keep 400 probes\n",
      "-- filtering region:769, screen genome to keep 400 probes\n",
      "-- filtering region:770, screen genome to keep 400 probes\n",
      "-- filtering region:771, directly append 36 probes\n",
      "-- filtering region:772, screen genome to keep 400 probes\n",
      "-- filtering region:773, screen genome to keep 400 probes\n",
      "-- filtering region:774, screen genome to keep 400 probes\n",
      "-- filtering region:775, screen genome to keep 400 probes\n",
      "-- filtering region:776, directly append 62 probes\n",
      "-- filtering region:777, screen genome to keep 400 probes\n",
      "-- filtering region:778, screen genome to keep 400 probes\n",
      "-- filtering region:779, directly append 393 probes\n",
      "-- filtering region:780, screen genome to keep 400 probes\n",
      "-- filtering region:781, screen genome to keep 400 probes\n",
      "-- filtering region:782, screen genome to keep 400 probes\n",
      "-- filtering region:783, screen genome to keep 400 probes\n",
      "-- filtering region:784, screen genome to keep 400 probes\n",
      "-- filtering region:785, screen genome to keep 400 probes\n",
      "-- filtering region:786, screen genome to keep 400 probes\n",
      "-- filtering region:787, screen genome to keep 400 probes\n",
      "-- filtering region:788, screen genome to keep 400 probes\n",
      "-- filtering region:789, screen genome to keep 400 probes\n",
      "-- filtering region:790, screen genome to keep 400 probes\n",
      "-- filtering region:791, screen genome to keep 400 probes\n",
      "-- filtering region:792, screen genome to keep 400 probes\n",
      "-- filtering region:793, screen genome to keep 400 probes\n",
      "-- filtering region:794, screen genome to keep 400 probes\n",
      "-- filtering region:795, screen genome to keep 400 probes\n",
      "-- filtering region:796, screen genome to keep 400 probes\n",
      "-- filtering region:797, screen genome to keep 400 probes\n",
      "-- filtering region:798, screen genome to keep 400 probes\n",
      "-- filtering region:799, screen genome to keep 400 probes\n",
      "-- filtering region:800, screen genome to keep 400 probes\n",
      "-- filtering region:801, screen genome to keep 400 probes\n",
      "-- filtering region:802, screen genome to keep 400 probes\n",
      "-- filtering region:803, screen genome to keep 400 probes\n",
      "-- filtering region:804, screen genome to keep 400 probes\n",
      "-- filtering region:805, screen genome to keep 400 probes\n",
      "-- filtering region:806, screen genome to keep 400 probes\n",
      "-- filtering region:807, screen genome to keep 400 probes\n",
      "-- filtering region:808, screen genome to keep 400 probes\n",
      "-- filtering region:809, screen genome to keep 400 probes\n",
      "-- filtering region:810, screen genome to keep 400 probes\n",
      "-- filtering region:811, screen genome to keep 400 probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- filtering region:812, screen genome to keep 400 probes\n",
      "-- filtering region:813, screen genome to keep 400 probes\n",
      "-- filtering region:814, screen genome to keep 400 probes\n",
      "-- filtering region:815, screen genome to keep 400 probes\n",
      "-- filtering region:816, screen genome to keep 400 probes\n",
      "-- filtering region:817, screen genome to keep 400 probes\n",
      "-- filtering region:818, screen genome to keep 400 probes\n",
      "-- filtering region:819, screen genome to keep 400 probes\n",
      "-- filtering region:820, screen genome to keep 400 probes\n",
      "-- filtering region:821, screen genome to keep 400 probes\n",
      "-- filtering region:822, screen genome to keep 400 probes\n",
      "-- filtering region:823, screen genome to keep 400 probes\n",
      "-- filtering region:824, screen genome to keep 400 probes\n",
      "-- filtering region:825, screen genome to keep 400 probes\n",
      "-- filtering region:826, screen genome to keep 400 probes\n",
      "-- filtering region:827, screen genome to keep 400 probes\n",
      "-- filtering region:828, screen genome to keep 400 probes\n",
      "-- filtering region:829, screen genome to keep 400 probes\n",
      "-- filtering region:830, screen genome to keep 400 probes\n",
      "-- filtering region:831, screen genome to keep 400 probes\n",
      "-- filtering region:832, screen genome to keep 400 probes\n",
      "-- filtering region:833, screen genome to keep 400 probes\n",
      "-- filtering region:834, screen genome to keep 400 probes\n",
      "-- filtering region:835, screen genome to keep 400 probes\n",
      "-- filtering region:836, screen genome to keep 400 probes\n",
      "-- filtering region:837, screen genome to keep 400 probes\n",
      "-- filtering region:838, screen genome to keep 400 probes\n",
      "-- filtering region:839, screen genome to keep 400 probes\n",
      "-- filtering region:840, screen genome to keep 400 probes\n",
      "-- filtering region:841, screen genome to keep 400 probes\n",
      "-- filtering region:842, screen genome to keep 400 probes\n",
      "-- filtering region:843, screen genome to keep 400 probes\n",
      "-- filtering region:844, screen genome to keep 400 probes\n",
      "-- filtering region:845, screen genome to keep 400 probes\n",
      "-- filtering region:846, screen genome to keep 400 probes\n",
      "-- filtering region:847, screen genome to keep 400 probes\n",
      "-- filtering region:848, screen genome to keep 400 probes\n",
      "-- filtering region:849, screen genome to keep 400 probes\n",
      "-- filtering region:850, screen genome to keep 400 probes\n",
      "-- filtering region:851, screen genome to keep 400 probes\n",
      "-- filtering region:852, screen genome to keep 400 probes\n",
      "-- filtering region:853, screen genome to keep 400 probes\n",
      "-- filtering region:854, screen genome to keep 400 probes\n",
      "-- filtering region:855, screen genome to keep 400 probes\n",
      "-- filtering region:856, screen genome to keep 400 probes\n",
      "-- filtering region:857, screen genome to keep 400 probes\n",
      "-- filtering region:858, screen genome to keep 400 probes\n",
      "-- filtering region:859, screen genome to keep 400 probes\n",
      "-- filtering region:860, screen genome to keep 400 probes\n",
      "-- filtering region:861, screen genome to keep 400 probes\n",
      "-- filtering region:862, screen genome to keep 400 probes\n",
      "-- filtering region:863, screen genome to keep 400 probes\n",
      "-- filtering region:864, screen genome to keep 400 probes\n",
      "-- filtering region:866, screen genome to keep 400 probes\n",
      "-- filtering region:867, screen genome to keep 400 probes\n",
      "-- filtering region:868, screen genome to keep 400 probes\n",
      "-- filtering region:869, screen genome to keep 400 probes\n",
      "-- filtering region:870, screen genome to keep 400 probes\n",
      "-- filtering region:871, screen genome to keep 400 probes\n",
      "-- filtering region:872, screen genome to keep 400 probes\n",
      "-- filtering region:873, screen genome to keep 400 probes\n",
      "-- filtering region:874, screen genome to keep 400 probes\n",
      "-- filtering region:875, screen genome to keep 400 probes\n",
      "-- filtering region:876, screen genome to keep 400 probes\n",
      "-- filtering region:877, screen genome to keep 400 probes\n",
      "-- filtering region:878, screen genome to keep 400 probes\n",
      "-- filtering region:879, screen genome to keep 400 probes\n",
      "-- filtering region:880, screen genome to keep 400 probes\n",
      "-- filtering region:881, screen genome to keep 400 probes\n",
      "-- filtering region:882, screen genome to keep 400 probes\n",
      "-- filtering region:883, screen genome to keep 400 probes\n",
      "-- filtering region:884, screen genome to keep 400 probes\n",
      "-- filtering region:885, screen genome to keep 400 probes\n",
      "-- filtering region:886, screen genome to keep 400 probes\n",
      "-- filtering region:887, directly append 400 probes\n",
      "-- filtering region:888, screen genome to keep 400 probes\n",
      "-- filtering region:889, screen genome to keep 400 probes\n",
      "-- filtering region:890, screen genome to keep 400 probes\n",
      "-- filtering region:891, screen genome to keep 400 probes\n",
      "-- filtering region:892, screen genome to keep 400 probes\n",
      "-- filtering region:893, screen genome to keep 400 probes\n",
      "-- filtering region:894, screen genome to keep 400 probes\n",
      "-- filtering region:895, screen genome to keep 400 probes\n",
      "-- filtering region:896, screen genome to keep 400 probes\n",
      "-- filtering region:897, screen genome to keep 400 probes\n",
      "-- filtering region:898, screen genome to keep 400 probes\n",
      "-- filtering region:899, screen genome to keep 400 probes\n",
      "-- filtering region:900, screen genome to keep 400 probes\n",
      "-- filtering region:901, screen genome to keep 400 probes\n",
      "-- filtering region:902, directly append 381 probes\n",
      "-- filtering region:903, screen genome to keep 400 probes\n",
      "-- filtering region:904, screen genome to keep 400 probes\n",
      "-- filtering region:905, screen genome to keep 400 probes\n",
      "-- filtering region:906, screen genome to keep 400 probes\n",
      "-- filtering region:907, screen genome to keep 400 probes\n",
      "-- filtering region:908, screen genome to keep 400 probes\n",
      "-- filtering region:909, screen genome to keep 400 probes\n",
      "-- filtering region:910, screen genome to keep 400 probes\n",
      "-- filtering region:911, screen genome to keep 400 probes\n",
      "-- filtering region:912, screen genome to keep 400 probes\n",
      "-- filtering region:913, screen genome to keep 400 probes\n",
      "-- filtering region:914, screen genome to keep 400 probes\n",
      "-- filtering region:915, screen genome to keep 400 probes\n",
      "-- filtering region:916, screen genome to keep 400 probes\n",
      "-- filtering region:917, screen genome to keep 400 probes\n",
      "-- filtering region:918, screen genome to keep 400 probes\n",
      "-- filtering region:919, screen genome to keep 400 probes\n",
      "-- filtering region:920, screen genome to keep 400 probes\n",
      "-- filtering region:921, screen genome to keep 400 probes\n",
      "-- filtering region:922, screen genome to keep 400 probes\n",
      "-- filtering region:923, screen genome to keep 400 probes\n",
      "-- filtering region:925, screen genome to keep 400 probes\n",
      "-- filtering region:926, screen genome to keep 400 probes\n",
      "-- filtering region:927, screen genome to keep 400 probes\n",
      "-- filtering region:928, screen genome to keep 400 probes\n",
      "-- filtering region:929, screen genome to keep 400 probes\n",
      "-- filtering region:930, screen genome to keep 400 probes\n",
      "-- filtering region:931, screen genome to keep 400 probes\n",
      "-- filtering region:932, screen genome to keep 400 probes\n",
      "-- filtering region:933, screen genome to keep 400 probes\n",
      "-- filtering region:934, screen genome to keep 400 probes\n",
      "-- filtering region:935, screen genome to keep 400 probes\n",
      "-- filtering region:936, screen genome to keep 400 probes\n",
      "-- filtering region:937, screen genome to keep 400 probes\n",
      "-- filtering region:938, screen genome to keep 400 probes\n",
      "-- filtering region:939, screen genome to keep 400 probes\n",
      "-- filtering region:940, screen genome to keep 400 probes\n",
      "-- filtering region:941, screen genome to keep 400 probes\n",
      "-- filtering region:942, screen genome to keep 400 probes\n",
      "-- filtering region:943, screen genome to keep 400 probes\n",
      "-- filtering region:944, screen genome to keep 400 probes\n",
      "-- filtering region:945, screen genome to keep 400 probes\n",
      "-- filtering region:946, screen genome to keep 400 probes\n",
      "-- filtering region:947, screen genome to keep 400 probes\n",
      "-- filtering region:948, screen genome to keep 400 probes\n",
      "-- filtering region:949, screen genome to keep 400 probes\n",
      "-- filtering region:950, screen genome to keep 400 probes\n",
      "-- filtering region:951, screen genome to keep 400 probes\n",
      "-- filtering region:952, screen genome to keep 400 probes\n",
      "-- filtering region:953, screen genome to keep 400 probes\n",
      "-- filtering region:954, screen genome to keep 400 probes\n",
      "-- filtering region:955, screen genome to keep 400 probes\n",
      "-- filtering region:956, screen genome to keep 400 probes\n",
      "-- filtering region:957, screen genome to keep 400 probes\n",
      "-- filtering region:958, screen genome to keep 400 probes\n",
      "-- filtering region:959, screen genome to keep 400 probes\n",
      "-- filtering region:960, screen genome to keep 400 probes\n",
      "-- filtering region:961, screen genome to keep 400 probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- filtering region:962, screen genome to keep 400 probes\n",
      "-- filtering region:963, screen genome to keep 400 probes\n",
      "-- filtering region:964, screen genome to keep 400 probes\n",
      "-- filtering region:965, screen genome to keep 400 probes\n",
      "-- filtering region:966, screen genome to keep 400 probes\n",
      "-- filtering region:967, screen genome to keep 400 probes\n",
      "-- filtering region:968, screen genome to keep 400 probes\n",
      "-- filtering region:969, screen genome to keep 400 probes\n",
      "-- filtering region:970, screen genome to keep 400 probes\n",
      "-- filtering region:971, screen genome to keep 400 probes\n",
      "-- filtering region:972, screen genome to keep 400 probes\n",
      "-- filtering region:973, screen genome to keep 400 probes\n",
      "-- filtering region:974, screen genome to keep 400 probes\n",
      "-- filtering region:975, screen genome to keep 400 probes\n",
      "-- filtering region:976, screen genome to keep 400 probes\n",
      "-- filtering region:977, screen genome to keep 400 probes\n",
      "-- filtering region:978, screen genome to keep 400 probes\n",
      "-- filtering region:979, screen genome to keep 400 probes\n",
      "-- filtering region:980, screen genome to keep 400 probes\n",
      "-- filtering region:981, screen genome to keep 400 probes\n",
      "-- filtering region:982, screen genome to keep 400 probes\n",
      "-- filtering region:983, screen genome to keep 400 probes\n",
      "-- filtering region:984, screen genome to keep 400 probes\n",
      "-- filtering region:985, screen genome to keep 400 probes\n",
      "-- filtering region:986, screen genome to keep 400 probes\n",
      "-- filtering region:987, screen genome to keep 400 probes\n",
      "-- filtering region:988, screen genome to keep 400 probes\n",
      "-- filtering region:989, screen genome to keep 400 probes\n",
      "-- filtering region:990, screen genome to keep 400 probes\n",
      "-- filtering region:991, directly append 127 probes\n",
      "region: 323 removed with 38 probes\n",
      "region: 443 removed with 3 probes\n",
      "region: 490 removed with 19 probes\n",
      "region: 227 removed with 6 probes\n",
      "region: 756 removed with 8 probes\n",
      "region: 757 removed with 8 probes\n",
      "region: 771 removed with 36 probes\n"
     ]
    }
   ],
   "source": [
    "# filter by genome counts\n",
    "reload(library_tools.quality_check)\n",
    "\n",
    "num_probes_kept = 400 # max num of probes kept for each region\n",
    "filtered_pb_dict = library_tools.quality_check.Screen_probe_by_hit(library_folder, pb_dict, num_probes_kept,\n",
    "                                                                  hit_type='genome')\n",
    "\n",
    "## filter by minimum number of probes in each region\n",
    "min_num_probes = 50\n",
    "removed_regions = []\n",
    "for _rid, _pb_class in filtered_pb_dict.items():\n",
    "    if len(_pb_class.kept_probes) < min_num_probes:\n",
    "        print(f\"region: {_rid} removed with {len(_pb_class.kept_probes)} probes\")\n",
    "        removed_regions.append(_rid)\n",
    "for _rid in removed_regions:\n",
    "    del(filtered_pb_dict[_rid])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 64,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "/* Put everything inside the global mpl namespace */\n",
       "/* global mpl */\n",
       "window.mpl = {};\n",
       "\n",
       "mpl.get_websocket_type = function () {\n",
       "    if (typeof WebSocket !== 'undefined') {\n",
       "        return WebSocket;\n",
       "    } else if (typeof MozWebSocket !== 'undefined') {\n",
       "        return MozWebSocket;\n",
       "    } else {\n",
       "        alert(\n",
       "            'Your browser does not have WebSocket support. ' +\n",
       "                'Please try Chrome, Safari or Firefox ≥ 6. ' +\n",
       "                'Firefox 4 and 5 are also supported but you ' +\n",
       "                'have to enable WebSockets in about:config.'\n",
       "        );\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure = function (figure_id, websocket, ondownload, parent_element) {\n",
       "    this.id = figure_id;\n",
       "\n",
       "    this.ws = websocket;\n",
       "\n",
       "    this.supports_binary = this.ws.binaryType !== undefined;\n",
       "\n",
       "    if (!this.supports_binary) {\n",
       "        var warnings = document.getElementById('mpl-warnings');\n",
       "        if (warnings) {\n",
       "            warnings.style.display = 'block';\n",
       "            warnings.textContent =\n",
       "                'This browser does not support binary websocket messages. ' +\n",
       "                'Performance may be slow.';\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.imageObj = new Image();\n",
       "\n",
       "    this.context = undefined;\n",
       "    this.message = undefined;\n",
       "    this.canvas = undefined;\n",
       "    this.rubberband_canvas = undefined;\n",
       "    this.rubberband_context = undefined;\n",
       "    this.format_dropdown = undefined;\n",
       "\n",
       "    this.image_mode = 'full';\n",
       "\n",
       "    this.root = document.createElement('div');\n",
       "    this.root.setAttribute('style', 'display: inline-block');\n",
       "    this._root_extra_style(this.root);\n",
       "\n",
       "    parent_element.appendChild(this.root);\n",
       "\n",
       "    this._init_header(this);\n",
       "    this._init_canvas(this);\n",
       "    this._init_toolbar(this);\n",
       "\n",
       "    var fig = this;\n",
       "\n",
       "    this.waiting = false;\n",
       "\n",
       "    this.ws.onopen = function () {\n",
       "        fig.send_message('supports_binary', { value: fig.supports_binary });\n",
       "        fig.send_message('send_image_mode', {});\n",
       "        if (fig.ratio !== 1) {\n",
       "            fig.send_message('set_dpi_ratio', { dpi_ratio: fig.ratio });\n",
       "        }\n",
       "        fig.send_message('refresh', {});\n",
       "    };\n",
       "\n",
       "    this.imageObj.onload = function () {\n",
       "        if (fig.image_mode === 'full') {\n",
       "            // Full images could contain transparency (where diff images\n",
       "            // almost always do), so we need to clear the canvas so that\n",
       "            // there is no ghosting.\n",
       "            fig.context.clearRect(0, 0, fig.canvas.width, fig.canvas.height);\n",
       "        }\n",
       "        fig.context.drawImage(fig.imageObj, 0, 0);\n",
       "    };\n",
       "\n",
       "    this.imageObj.onunload = function () {\n",
       "        fig.ws.close();\n",
       "    };\n",
       "\n",
       "    this.ws.onmessage = this._make_on_message_function(this);\n",
       "\n",
       "    this.ondownload = ondownload;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_header = function () {\n",
       "    var titlebar = document.createElement('div');\n",
       "    titlebar.classList =\n",
       "        'ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix';\n",
       "    var titletext = document.createElement('div');\n",
       "    titletext.classList = 'ui-dialog-title';\n",
       "    titletext.setAttribute(\n",
       "        'style',\n",
       "        'width: 100%; text-align: center; padding: 3px;'\n",
       "    );\n",
       "    titlebar.appendChild(titletext);\n",
       "    this.root.appendChild(titlebar);\n",
       "    this.header = titletext;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._init_canvas = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var canvas_div = (this.canvas_div = document.createElement('div'));\n",
       "    canvas_div.setAttribute(\n",
       "        'style',\n",
       "        'border: 1px solid #ddd;' +\n",
       "            'box-sizing: content-box;' +\n",
       "            'clear: both;' +\n",
       "            'min-height: 1px;' +\n",
       "            'min-width: 1px;' +\n",
       "            'outline: 0;' +\n",
       "            'overflow: hidden;' +\n",
       "            'position: relative;' +\n",
       "            'resize: both;'\n",
       "    );\n",
       "\n",
       "    function on_keyboard_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.key_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    canvas_div.addEventListener(\n",
       "        'keydown',\n",
       "        on_keyboard_event_closure('key_press')\n",
       "    );\n",
       "    canvas_div.addEventListener(\n",
       "        'keyup',\n",
       "        on_keyboard_event_closure('key_release')\n",
       "    );\n",
       "\n",
       "    this._canvas_extra_style(canvas_div);\n",
       "    this.root.appendChild(canvas_div);\n",
       "\n",
       "    var canvas = (this.canvas = document.createElement('canvas'));\n",
       "    canvas.classList.add('mpl-canvas');\n",
       "    canvas.setAttribute('style', 'box-sizing: content-box;');\n",
       "\n",
       "    this.context = canvas.getContext('2d');\n",
       "\n",
       "    var backingStore =\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        this.context.webkitBackingStorePixelRatio ||\n",
       "        this.context.mozBackingStorePixelRatio ||\n",
       "        this.context.msBackingStorePixelRatio ||\n",
       "        this.context.oBackingStorePixelRatio ||\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        1;\n",
       "\n",
       "    this.ratio = (window.devicePixelRatio || 1) / backingStore;\n",
       "    if (this.ratio !== 1) {\n",
       "        fig.send_message('set_dpi_ratio', { dpi_ratio: this.ratio });\n",
       "    }\n",
       "\n",
       "    var rubberband_canvas = (this.rubberband_canvas = document.createElement(\n",
       "        'canvas'\n",
       "    ));\n",
       "    rubberband_canvas.setAttribute(\n",
       "        'style',\n",
       "        'box-sizing: content-box; position: absolute; left: 0; top: 0; z-index: 1;'\n",
       "    );\n",
       "\n",
       "    var resizeObserver = new ResizeObserver(function (entries) {\n",
       "        var nentries = entries.length;\n",
       "        for (var i = 0; i < nentries; i++) {\n",
       "            var entry = entries[i];\n",
       "            var width, height;\n",
       "            if (entry.contentBoxSize) {\n",
       "                if (entry.contentBoxSize instanceof Array) {\n",
       "                    // Chrome 84 implements new version of spec.\n",
       "                    width = entry.contentBoxSize[0].inlineSize;\n",
       "                    height = entry.contentBoxSize[0].blockSize;\n",
       "                } else {\n",
       "                    // Firefox implements old version of spec.\n",
       "                    width = entry.contentBoxSize.inlineSize;\n",
       "                    height = entry.contentBoxSize.blockSize;\n",
       "                }\n",
       "            } else {\n",
       "                // Chrome <84 implements even older version of spec.\n",
       "                width = entry.contentRect.width;\n",
       "                height = entry.contentRect.height;\n",
       "            }\n",
       "\n",
       "            // Keep the size of the canvas and rubber band canvas in sync with\n",
       "            // the canvas container.\n",
       "            if (entry.devicePixelContentBoxSize) {\n",
       "                // Chrome 84 implements new version of spec.\n",
       "                canvas.setAttribute(\n",
       "                    'width',\n",
       "                    entry.devicePixelContentBoxSize[0].inlineSize\n",
       "                );\n",
       "                canvas.setAttribute(\n",
       "                    'height',\n",
       "                    entry.devicePixelContentBoxSize[0].blockSize\n",
       "                );\n",
       "            } else {\n",
       "                canvas.setAttribute('width', width * fig.ratio);\n",
       "                canvas.setAttribute('height', height * fig.ratio);\n",
       "            }\n",
       "            canvas.setAttribute(\n",
       "                'style',\n",
       "                'width: ' + width + 'px; height: ' + height + 'px;'\n",
       "            );\n",
       "\n",
       "            rubberband_canvas.setAttribute('width', width);\n",
       "            rubberband_canvas.setAttribute('height', height);\n",
       "\n",
       "            // And update the size in Python. We ignore the initial 0/0 size\n",
       "            // that occurs as the element is placed into the DOM, which should\n",
       "            // otherwise not happen due to the minimum size styling.\n",
       "            if (width != 0 && height != 0) {\n",
       "                fig.request_resize(width, height);\n",
       "            }\n",
       "        }\n",
       "    });\n",
       "    resizeObserver.observe(canvas_div);\n",
       "\n",
       "    function on_mouse_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.mouse_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousedown',\n",
       "        on_mouse_event_closure('button_press')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseup',\n",
       "        on_mouse_event_closure('button_release')\n",
       "    );\n",
       "    // Throttle sequential mouse events to 1 every 20ms.\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousemove',\n",
       "        on_mouse_event_closure('motion_notify')\n",
       "    );\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseenter',\n",
       "        on_mouse_event_closure('figure_enter')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseleave',\n",
       "        on_mouse_event_closure('figure_leave')\n",
       "    );\n",
       "\n",
       "    canvas_div.addEventListener('wheel', function (event) {\n",
       "        if (event.deltaY < 0) {\n",
       "            event.step = 1;\n",
       "        } else {\n",
       "            event.step = -1;\n",
       "        }\n",
       "        on_mouse_event_closure('scroll')(event);\n",
       "    });\n",
       "\n",
       "    canvas_div.appendChild(canvas);\n",
       "    canvas_div.appendChild(rubberband_canvas);\n",
       "\n",
       "    this.rubberband_context = rubberband_canvas.getContext('2d');\n",
       "    this.rubberband_context.strokeStyle = '#000000';\n",
       "\n",
       "    this._resize_canvas = function (width, height, forward) {\n",
       "        if (forward) {\n",
       "            canvas_div.style.width = width + 'px';\n",
       "            canvas_div.style.height = height + 'px';\n",
       "        }\n",
       "    };\n",
       "\n",
       "    // Disable right mouse context menu.\n",
       "    this.rubberband_canvas.addEventListener('contextmenu', function (_e) {\n",
       "        event.preventDefault();\n",
       "        return false;\n",
       "    });\n",
       "\n",
       "    function set_focus() {\n",
       "        canvas.focus();\n",
       "        canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    window.setTimeout(set_focus, 100);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'mpl-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'mpl-button-group';\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'mpl-button-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        var button = (fig.buttons[name] = document.createElement('button'));\n",
       "        button.classList = 'mpl-widget';\n",
       "        button.setAttribute('role', 'button');\n",
       "        button.setAttribute('aria-disabled', 'false');\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "\n",
       "        var icon_img = document.createElement('img');\n",
       "        icon_img.src = '_images/' + image + '.png';\n",
       "        icon_img.srcset = '_images/' + image + '_large.png 2x';\n",
       "        icon_img.alt = tooltip;\n",
       "        button.appendChild(icon_img);\n",
       "\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    var fmt_picker = document.createElement('select');\n",
       "    fmt_picker.classList = 'mpl-widget';\n",
       "    toolbar.appendChild(fmt_picker);\n",
       "    this.format_dropdown = fmt_picker;\n",
       "\n",
       "    for (var ind in mpl.extensions) {\n",
       "        var fmt = mpl.extensions[ind];\n",
       "        var option = document.createElement('option');\n",
       "        option.selected = fmt === mpl.default_extension;\n",
       "        option.innerHTML = fmt;\n",
       "        fmt_picker.appendChild(option);\n",
       "    }\n",
       "\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.request_resize = function (x_pixels, y_pixels) {\n",
       "    // Request matplotlib to resize the figure. Matplotlib will then trigger a resize in the client,\n",
       "    // which will in turn request a refresh of the image.\n",
       "    this.send_message('resize', { width: x_pixels, height: y_pixels });\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_message = function (type, properties) {\n",
       "    properties['type'] = type;\n",
       "    properties['figure_id'] = this.id;\n",
       "    this.ws.send(JSON.stringify(properties));\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_draw_message = function () {\n",
       "    if (!this.waiting) {\n",
       "        this.waiting = true;\n",
       "        this.ws.send(JSON.stringify({ type: 'draw', figure_id: this.id }));\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    var format_dropdown = fig.format_dropdown;\n",
       "    var format = format_dropdown.options[format_dropdown.selectedIndex].value;\n",
       "    fig.ondownload(fig, format);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_resize = function (fig, msg) {\n",
       "    var size = msg['size'];\n",
       "    if (size[0] !== fig.canvas.width || size[1] !== fig.canvas.height) {\n",
       "        fig._resize_canvas(size[0], size[1], msg['forward']);\n",
       "        fig.send_message('refresh', {});\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_rubberband = function (fig, msg) {\n",
       "    var x0 = msg['x0'] / fig.ratio;\n",
       "    var y0 = (fig.canvas.height - msg['y0']) / fig.ratio;\n",
       "    var x1 = msg['x1'] / fig.ratio;\n",
       "    var y1 = (fig.canvas.height - msg['y1']) / fig.ratio;\n",
       "    x0 = Math.floor(x0) + 0.5;\n",
       "    y0 = Math.floor(y0) + 0.5;\n",
       "    x1 = Math.floor(x1) + 0.5;\n",
       "    y1 = Math.floor(y1) + 0.5;\n",
       "    var min_x = Math.min(x0, x1);\n",
       "    var min_y = Math.min(y0, y1);\n",
       "    var width = Math.abs(x1 - x0);\n",
       "    var height = Math.abs(y1 - y0);\n",
       "\n",
       "    fig.rubberband_context.clearRect(\n",
       "        0,\n",
       "        0,\n",
       "        fig.canvas.width / fig.ratio,\n",
       "        fig.canvas.height / fig.ratio\n",
       "    );\n",
       "\n",
       "    fig.rubberband_context.strokeRect(min_x, min_y, width, height);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_figure_label = function (fig, msg) {\n",
       "    // Updates the figure title.\n",
       "    fig.header.textContent = msg['label'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_cursor = function (fig, msg) {\n",
       "    var cursor = msg['cursor'];\n",
       "    switch (cursor) {\n",
       "        case 0:\n",
       "            cursor = 'pointer';\n",
       "            break;\n",
       "        case 1:\n",
       "            cursor = 'default';\n",
       "            break;\n",
       "        case 2:\n",
       "            cursor = 'crosshair';\n",
       "            break;\n",
       "        case 3:\n",
       "            cursor = 'move';\n",
       "            break;\n",
       "    }\n",
       "    fig.rubberband_canvas.style.cursor = cursor;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_message = function (fig, msg) {\n",
       "    fig.message.textContent = msg['message'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_draw = function (fig, _msg) {\n",
       "    // Request the server to send over a new figure.\n",
       "    fig.send_draw_message();\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_image_mode = function (fig, msg) {\n",
       "    fig.image_mode = msg['mode'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_history_buttons = function (fig, msg) {\n",
       "    for (var key in msg) {\n",
       "        if (!(key in fig.buttons)) {\n",
       "            continue;\n",
       "        }\n",
       "        fig.buttons[key].disabled = !msg[key];\n",
       "        fig.buttons[key].setAttribute('aria-disabled', !msg[key]);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_navigate_mode = function (fig, msg) {\n",
       "    if (msg['mode'] === 'PAN') {\n",
       "        fig.buttons['Pan'].classList.add('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    } else if (msg['mode'] === 'ZOOM') {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.add('active');\n",
       "    } else {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Called whenever the canvas gets updated.\n",
       "    this.send_message('ack', {});\n",
       "};\n",
       "\n",
       "// A function to construct a web socket function for onmessage handling.\n",
       "// Called in the figure constructor.\n",
       "mpl.figure.prototype._make_on_message_function = function (fig) {\n",
       "    return function socket_on_message(evt) {\n",
       "        if (evt.data instanceof Blob) {\n",
       "            /* FIXME: We get \"Resource interpreted as Image but\n",
       "             * transferred with MIME type text/plain:\" errors on\n",
       "             * Chrome.  But how to set the MIME type?  It doesn't seem\n",
       "             * to be part of the websocket stream */\n",
       "            evt.data.type = 'image/png';\n",
       "\n",
       "            /* Free the memory for the previous frames */\n",
       "            if (fig.imageObj.src) {\n",
       "                (window.URL || window.webkitURL).revokeObjectURL(\n",
       "                    fig.imageObj.src\n",
       "                );\n",
       "            }\n",
       "\n",
       "            fig.imageObj.src = (window.URL || window.webkitURL).createObjectURL(\n",
       "                evt.data\n",
       "            );\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        } else if (\n",
       "            typeof evt.data === 'string' &&\n",
       "            evt.data.slice(0, 21) === 'data:image/png;base64'\n",
       "        ) {\n",
       "            fig.imageObj.src = evt.data;\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        var msg = JSON.parse(evt.data);\n",
       "        var msg_type = msg['type'];\n",
       "\n",
       "        // Call the  \"handle_{type}\" callback, which takes\n",
       "        // the figure and JSON message as its only arguments.\n",
       "        try {\n",
       "            var callback = fig['handle_' + msg_type];\n",
       "        } catch (e) {\n",
       "            console.log(\n",
       "                \"No handler for the '\" + msg_type + \"' message type: \",\n",
       "                msg\n",
       "            );\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        if (callback) {\n",
       "            try {\n",
       "                // console.log(\"Handling '\" + msg_type + \"' message: \", msg);\n",
       "                callback(fig, msg);\n",
       "            } catch (e) {\n",
       "                console.log(\n",
       "                    \"Exception inside the 'handler_\" + msg_type + \"' callback:\",\n",
       "                    e,\n",
       "                    e.stack,\n",
       "                    msg\n",
       "                );\n",
       "            }\n",
       "        }\n",
       "    };\n",
       "};\n",
       "\n",
       "// from http://stackoverflow.com/questions/1114465/getting-mouse-location-in-canvas\n",
       "mpl.findpos = function (e) {\n",
       "    //this section is from http://www.quirksmode.org/js/events_properties.html\n",
       "    var targ;\n",
       "    if (!e) {\n",
       "        e = window.event;\n",
       "    }\n",
       "    if (e.target) {\n",
       "        targ = e.target;\n",
       "    } else if (e.srcElement) {\n",
       "        targ = e.srcElement;\n",
       "    }\n",
       "    if (targ.nodeType === 3) {\n",
       "        // defeat Safari bug\n",
       "        targ = targ.parentNode;\n",
       "    }\n",
       "\n",
       "    // pageX,Y are the mouse positions relative to the document\n",
       "    var boundingRect = targ.getBoundingClientRect();\n",
       "    var x = e.pageX - (boundingRect.left + document.body.scrollLeft);\n",
       "    var y = e.pageY - (boundingRect.top + document.body.scrollTop);\n",
       "\n",
       "    return { x: x, y: y };\n",
       "};\n",
       "\n",
       "/*\n",
       " * return a copy of an object with only non-object keys\n",
       " * we need this to avoid circular references\n",
       " * http://stackoverflow.com/a/24161582/3208463\n",
       " */\n",
       "function simpleKeys(original) {\n",
       "    return Object.keys(original).reduce(function (obj, key) {\n",
       "        if (typeof original[key] !== 'object') {\n",
       "            obj[key] = original[key];\n",
       "        }\n",
       "        return obj;\n",
       "    }, {});\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.mouse_event = function (event, name) {\n",
       "    var canvas_pos = mpl.findpos(event);\n",
       "\n",
       "    if (name === 'button_press') {\n",
       "        this.canvas.focus();\n",
       "        this.canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    var x = canvas_pos.x * this.ratio;\n",
       "    var y = canvas_pos.y * this.ratio;\n",
       "\n",
       "    this.send_message(name, {\n",
       "        x: x,\n",
       "        y: y,\n",
       "        button: event.button,\n",
       "        step: event.step,\n",
       "        guiEvent: simpleKeys(event),\n",
       "    });\n",
       "\n",
       "    /* This prevents the web browser from automatically changing to\n",
       "     * the text insertion cursor when the button is pressed.  We want\n",
       "     * to control all of the cursor setting manually through the\n",
       "     * 'cursor' event from matplotlib */\n",
       "    event.preventDefault();\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (_event, _name) {\n",
       "    // Handle any extra behaviour associated with a key event\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.key_event = function (event, name) {\n",
       "    // Prevent repeat events\n",
       "    if (name === 'key_press') {\n",
       "        if (event.which === this._key) {\n",
       "            return;\n",
       "        } else {\n",
       "            this._key = event.which;\n",
       "        }\n",
       "    }\n",
       "    if (name === 'key_release') {\n",
       "        this._key = null;\n",
       "    }\n",
       "\n",
       "    var value = '';\n",
       "    if (event.ctrlKey && event.which !== 17) {\n",
       "        value += 'ctrl+';\n",
       "    }\n",
       "    if (event.altKey && event.which !== 18) {\n",
       "        value += 'alt+';\n",
       "    }\n",
       "    if (event.shiftKey && event.which !== 16) {\n",
       "        value += 'shift+';\n",
       "    }\n",
       "\n",
       "    value += 'k';\n",
       "    value += event.which.toString();\n",
       "\n",
       "    this._key_event_extra(event, name);\n",
       "\n",
       "    this.send_message(name, { key: value, guiEvent: simpleKeys(event) });\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onclick = function (name) {\n",
       "    if (name === 'download') {\n",
       "        this.handle_save(this, null);\n",
       "    } else {\n",
       "        this.send_message('toolbar_button', { name: name });\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onmouseover = function (tooltip) {\n",
       "    this.message.textContent = tooltip;\n",
       "};\n",
       "mpl.toolbar_items = [[\"Home\", \"Reset original view\", \"fa fa-home icon-home\", \"home\"], [\"Back\", \"Back to previous view\", \"fa fa-arrow-left icon-arrow-left\", \"back\"], [\"Forward\", \"Forward to next view\", \"fa fa-arrow-right icon-arrow-right\", \"forward\"], [\"\", \"\", \"\", \"\"], [\"Pan\", \"Left button pans, Right button zooms\\nx/y fixes axis, CTRL fixes aspect\", \"fa fa-arrows icon-move\", \"pan\"], [\"Zoom\", \"Zoom to rectangle\\nx/y fixes axis, CTRL fixes aspect\", \"fa fa-square-o icon-check-empty\", \"zoom\"], [\"\", \"\", \"\", \"\"], [\"Download\", \"Download plot\", \"fa fa-floppy-o icon-save\", \"download\"]];\n",
       "\n",
       "mpl.extensions = [\"eps\", \"jpeg\", \"pdf\", \"png\", \"ps\", \"raw\", \"svg\", \"tif\"];\n",
       "\n",
       "mpl.default_extension = \"png\";/* global mpl */\n",
       "\n",
       "var comm_websocket_adapter = function (comm) {\n",
       "    // Create a \"websocket\"-like object which calls the given IPython comm\n",
       "    // object with the appropriate methods. Currently this is a non binary\n",
       "    // socket, so there is still some room for performance tuning.\n",
       "    var ws = {};\n",
       "\n",
       "    ws.close = function () {\n",
       "        comm.close();\n",
       "    };\n",
       "    ws.send = function (m) {\n",
       "        //console.log('sending', m);\n",
       "        comm.send(m);\n",
       "    };\n",
       "    // Register the callback with on_msg.\n",
       "    comm.on_msg(function (msg) {\n",
       "        //console.log('receiving', msg['content']['data'], msg);\n",
       "        // Pass the mpl event to the overridden (by mpl) onmessage function.\n",
       "        ws.onmessage(msg['content']['data']);\n",
       "    });\n",
       "    return ws;\n",
       "};\n",
       "\n",
       "mpl.mpl_figure_comm = function (comm, msg) {\n",
       "    // This is the function which gets called when the mpl process\n",
       "    // starts-up an IPython Comm through the \"matplotlib\" channel.\n",
       "\n",
       "    var id = msg.content.data.id;\n",
       "    // Get hold of the div created by the display call when the Comm\n",
       "    // socket was opened in Python.\n",
       "    var element = document.getElementById(id);\n",
       "    var ws_proxy = comm_websocket_adapter(comm);\n",
       "\n",
       "    function ondownload(figure, _format) {\n",
       "        window.open(figure.canvas.toDataURL());\n",
       "    }\n",
       "\n",
       "    var fig = new mpl.figure(id, ws_proxy, ondownload, element);\n",
       "\n",
       "    // Call onopen now - mpl needs it, as it is assuming we've passed it a real\n",
       "    // web socket which is closed, not our websocket->open comm proxy.\n",
       "    ws_proxy.onopen();\n",
       "\n",
       "    fig.parent_element = element;\n",
       "    fig.cell_info = mpl.find_output_cell(\"<div id='\" + id + \"'></div>\");\n",
       "    if (!fig.cell_info) {\n",
       "        console.error('Failed to find cell for figure', id, fig);\n",
       "        return;\n",
       "    }\n",
       "    fig.cell_info[0].output_area.element.one(\n",
       "        'cleared',\n",
       "        { fig: fig },\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_close = function (fig, msg) {\n",
       "    var width = fig.canvas.width / fig.ratio;\n",
       "    fig.cell_info[0].output_area.element.off(\n",
       "        'cleared',\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "\n",
       "    // Update the output cell to use the data from the current canvas.\n",
       "    fig.push_to_output();\n",
       "    var dataURL = fig.canvas.toDataURL();\n",
       "    // Re-enable the keyboard manager in IPython - without this line, in FF,\n",
       "    // the notebook keyboard shortcuts fail.\n",
       "    IPython.keyboard_manager.enable();\n",
       "    fig.parent_element.innerHTML =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "    fig.close_ws(fig, msg);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.close_ws = function (fig, msg) {\n",
       "    fig.send_message('closing', msg);\n",
       "    // fig.ws.close()\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.push_to_output = function (_remove_interactive) {\n",
       "    // Turn the data on the canvas into data in the output cell.\n",
       "    var width = this.canvas.width / this.ratio;\n",
       "    var dataURL = this.canvas.toDataURL();\n",
       "    this.cell_info[1]['text/html'] =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Tell IPython that the notebook contents must change.\n",
       "    IPython.notebook.set_dirty(true);\n",
       "    this.send_message('ack', {});\n",
       "    var fig = this;\n",
       "    // Wait a second, then push the new image to the DOM so\n",
       "    // that it is saved nicely (might be nice to debounce this).\n",
       "    setTimeout(function () {\n",
       "        fig.push_to_output();\n",
       "    }, 1000);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'btn-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'btn-group';\n",
       "    var button;\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'btn-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        button = fig.buttons[name] = document.createElement('button');\n",
       "        button.classList = 'btn btn-default';\n",
       "        button.href = '#';\n",
       "        button.title = name;\n",
       "        button.innerHTML = '<i class=\"fa ' + image + ' fa-lg\"></i>';\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    // Add the status bar.\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message pull-right';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "\n",
       "    // Add the close button to the window.\n",
       "    var buttongrp = document.createElement('div');\n",
       "    buttongrp.classList = 'btn-group inline pull-right';\n",
       "    button = document.createElement('button');\n",
       "    button.classList = 'btn btn-mini btn-primary';\n",
       "    button.href = '#';\n",
       "    button.title = 'Stop Interaction';\n",
       "    button.innerHTML = '<i class=\"fa fa-power-off icon-remove icon-large\"></i>';\n",
       "    button.addEventListener('click', function (_evt) {\n",
       "        fig.handle_close(fig, {});\n",
       "    });\n",
       "    button.addEventListener(\n",
       "        'mouseover',\n",
       "        on_mouseover_closure('Stop Interaction')\n",
       "    );\n",
       "    buttongrp.appendChild(button);\n",
       "    var titlebar = this.root.querySelector('.ui-dialog-titlebar');\n",
       "    titlebar.insertBefore(buttongrp, titlebar.firstChild);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._remove_fig_handler = function (event) {\n",
       "    var fig = event.data.fig;\n",
       "    fig.close_ws(fig, {});\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (el) {\n",
       "    el.style.boxSizing = 'content-box'; // override notebook setting of border-box.\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (el) {\n",
       "    // this is important to make the div 'focusable\n",
       "    el.setAttribute('tabindex', 0);\n",
       "    // reach out to IPython and tell the keyboard manager to turn it's self\n",
       "    // off when our div gets focus\n",
       "\n",
       "    // location in version 3\n",
       "    if (IPython.notebook.keyboard_manager) {\n",
       "        IPython.notebook.keyboard_manager.register_events(el);\n",
       "    } else {\n",
       "        // location in version 2\n",
       "        IPython.keyboard_manager.register_events(el);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (event, _name) {\n",
       "    var manager = IPython.notebook.keyboard_manager;\n",
       "    if (!manager) {\n",
       "        manager = IPython.keyboard_manager;\n",
       "    }\n",
       "\n",
       "    // Check for shift+enter\n",
       "    if (event.shiftKey && event.which === 13) {\n",
       "        this.canvas_div.blur();\n",
       "        // select the cell after this one\n",
       "        var index = IPython.notebook.find_cell_index(this.cell_info[0]);\n",
       "        IPython.notebook.select(index + 1);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    fig.ondownload(fig, null);\n",
       "};\n",
       "\n",
       "mpl.find_output_cell = function (html_output) {\n",
       "    // Return the cell and output element which can be found *uniquely* in the notebook.\n",
       "    // Note - this is a bit hacky, but it is done because the \"notebook_saving.Notebook\"\n",
       "    // IPython event is triggered only after the cells have been serialised, which for\n",
       "    // our purposes (turning an active figure into a static one), is too late.\n",
       "    var cells = IPython.notebook.get_cells();\n",
       "    var ncells = cells.length;\n",
       "    for (var i = 0; i < ncells; i++) {\n",
       "        var cell = cells[i];\n",
       "        if (cell.cell_type === 'code') {\n",
       "            for (var j = 0; j < cell.output_area.outputs.length; j++) {\n",
       "                var data = cell.output_area.outputs[j];\n",
       "                if (data.data) {\n",
       "                    // IPython >= 3 moved mimebundle to data attribute of output\n",
       "                    data = data.data;\n",
       "                }\n",
       "                if (data['text/html'] === html_output) {\n",
       "                    return [cell, data, j];\n",
       "                }\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "};\n",
       "\n",
       "// Register the function which deals with the matplotlib target/channel.\n",
       "// The kernel may be null if the page has been refreshed.\n",
       "if (IPython.notebook.kernel !== null) {\n",
       "    IPython.notebook.kernel.comm_manager.register_target(\n",
       "        'matplotlib',\n",
       "        mpl.mpl_figure_comm\n",
       "    );\n",
       "}\n"
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<img src=\"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAtAAAAGrCAYAAADpWqADAAAAAXNSR0IArs4c6QAAIABJREFUeF7snQd4FcX6h3+EXkMNJSAdaaF3QuhIUAEFEVBARJGO0pRepKOIShUEvDQbvSO9htAJgTRCqIHQQ+/3mT0pG5LDnpM92TPn5DfPc//3D5nZmX2/ufry8e1MilevXr0CGwmQAAmQAAmQAAmQAAmQgEUEUlCgLeLETiRAAiRAAiRAAiRAAiSgEKBAcyOQAAmQAAmQAAmQAAmQgBUEKNBWwGJXEiABEiABEiABEiABEqBAcw+QAAmQAAmQAAmQAAmQgBUEKNBWwGJXEiABEiABEiABEiABEqBAcw+QAAmQAAmQAAmQAAmQgBUEKNBWwGJXEiABEiABEiABEiABEqBAcw+QAAmQAAmQAAmQAAmQgBUEKNBWwGJXEiABEiABEiABEiABEqBAcw+QAAmQAAmQAAmQAAmQgBUEKNBWwGJXEiABEiABEiABEiABEqBAcw+QAAmQAAmQAAmQAAmQgBUEKNBWwGJXEiABEiABEiABEiABEqBAcw+QAAmQAAmQAAmQAAmQgBUEKNBWwGJXEiABEiABEiABEiABEqBAcw+QAAmQAAmQAAmQAAmQgBUEKNBWwGJXEiABEiABEiABEiABEqBAcw+QAAmQAAmQAAmQAAmQgBUEKNBWwGJXEiABEiABEiABEiABEqBAcw+QAAmQAAmQAAmQAAmQgBUEKNBWwGJXEiABEiABEiABEiABEqBAcw+QAAmQAAmQAAmQAAmQgBUEKNBWwGJXEiABEiABEiABEiABEqBAcw+QAAmQAAmQAAmQAAmQgBUEKNBWwGJXEiABEiABEiABEiABEqBAcw+QAAmQAAmQAAmQAAmQgBUEKNBWwGJXEiABEiABEiABEiABEqBAcw+QAAmQAAmQAAmQAAmQgBUEKNBWwGJXEiABEiABEiABEiABEqBAcw+QAAmQAAmQAAmQAAmQgBUEKNBWwGJXEiABEiABEiABEiABEqBAcw+QAAmQAAmQAAmQAAmQgBUEKNBWwGJXEiABEiABEiABEiABEqBAcw+QAAmQAAmQAAmQAAmQgBUEKNBWwGJXEiABEiABEiABEiABEqBAcw+QAAmQAAmQAAmQAAmQgBUEKNBWwGJXEiABEiABEiABEiABEqBAcw+QAAmQAAmQAAmQAAmQgBUEKNBWwGJXEiABEiABEiABEiABEqBAcw+QAAmQAAmQAAmQAAmQgBUEKNBWwGJXEiABEiABEiABEiABEqBAcw+QAAmQAAmQAAmQAAmQgBUEKNBWwGJXEiABEiABEiABEiABEqBAcw+QAAmQAAmQAAmQAAmQgBUEKNBWwGJXEiABEiABEiABEiABEqBAcw+QAAmQAAmQAAmQAAmQgBUEKNBWwGJXEiABEiABEiABEiABEqBAcw+QAAmQAAmQAAmQAAmQgBUEKNBWwEps18ePH8PPzw+5cuVCqlSpEvsYjiMBEiABEiABEiCBJCfw/PlzXL9+HR4eHkiXLl2Sz+eIE1CgDYjaoUOHUK1aNQNm4hQkQAIkQAIkQAIkYBsCvr6+qFq1qm0e5mRPoUAbENCwsDAULlwYYiPmzZvXgBk5BQmQAAmQAAmQAAkkjkB4eLiS+Dt37hwKFSqUuIc4+SgKtAEBvnTpEgoUKICLFy8if/78BszIKUiABEiABEiABEggcQToLdrcKNDajHT34EbUjZAPIAESIAESIAESMIgAvUUbNAVam5HuHtyIuhHyASRAAiRAAiRAAgYRoLdog6ZAazPS3YMbUTdCPoAESIAESIAESMAgAvQWbdAUaG1GuntwI+pGyAeQAAmQAAmQAAkYRIDeog2aAq3NSHcPbkTdCPkAEiABEiABEiABgwjQW7RBU6C1GenuwY2oGyEfQAIkQAIkQAIkYBABeos2aAq0NiPdPbgRdSPkA0iABEiABEiABAwiQG/RBk2B1makuwc3om6EfAAJkAAJkAAJkIBBBOgt2qAp0NqMdPfgRtSNkA8gARIgARIgARIwiAC9RRs0BVqbke4e3Ii6EfIBJEACJEACJEACBhGgt2iDpkBrM9LdgxtRN0I+gARIgARIgARIwCAC9BZt0BRobUa6e3Aj6kbIB5AACZAACZAACRhEgN6iDZoCrc1Idw9uRN0I+QASIAESIAESIAGDCNBbtEFToLUZ6e7BjagbIR9AAiRAAiRAAiRgEAF6izZoCrQ2I909uBF1I+QDSIAESIAESIAEDCJAb9EGTYHWZqS7BzeiboR8AAmQAAmQAAmQgEEE6C3aoCnQ2ox09+BG1I2QDyABEiABEiABEjCIAL1FGzQFWpuR7h7ciLoR8gEkQAIkQAIkQAIGEaC3aIOmQGsz0t2DG1E3Qj6ABEiABEiABEjAIAL0Fm3QFGhtRrp7cCPqRsgHkAAJkAAJkAAJGESA3qINmgKtzUh3D25E3Qj5ABIgARIgARIgAYMI0Fu0QVOgtRnp7sGNqBshH0ACJEACJEACJGAQAXqLNuhkI9BBQUGoVKkSpk+fjs8++0whc/z4cfTt2xeHDx9Gjhw50KdPHwwYMCCG2suXLzF69GjMmzcPt2/fhqenJ2bOnIlixYppk1X14Ea0Chc7kwAJkAAJkAAJ2JEAvUUbfrIQ6GfPnqFWrVqKKC9YsEAR6Js3b6JkyZJo2bIl+vXrBx8fH/To0UMR5M6dOyvkhDyLX4sx7u7uGDRoEEJDQ+Hv7480adJo043qwY1oMSp2JAESIAESIAESsDMBeot2AJKFQA8ZMkQR5B07dsQI9IQJEzBjxgyEhYUhVapUCinRb8WKFQgICMDTp0+RM2dOTJ48Gd26dVN+fufOHeTLlw/z589H27ZttelSoC1mxI4kQAIkQAIkQAJyEKBAa8fB6QV69+7d+OCDD5RyjbfeeitGoJs1a4asWbNi6dKlMZS2bt2Kxo0b49q1a4pYV69eHYGBgShRokRMH1HGUa5cOSUzbWnjRrSUFPuRAAmQAAmQAAnYmwC9RTsCTi3QImNcvnx5iGxz+/btkSJFihiBFhLs7e2NSZMmxVA6ffo0ypQpg0OHDuHChQto1aoVHj58iPTp08f0adOmjfJ769atM0s3MjIS4j/RLTw8HNWqVcPFixeRP39+7agkoke3RUewMygiESMdY8jjZy+ROmUKpHRJYbMFaz1T/Fy0dKldlP9+/dc2W4hkD0ou7ykZdi7HgQkk9n8zKZACqVKK/ws8fWH6543eZslaovtEzxX9zzi9czvaeEtYyfhO6VOnxNiWHni3XN4kWx4FWhutUwt0u3bt8OrVK/z5558KCbVAiw8BhVSPGTMmhpKoby5atCj27NmjZKA7dOiAFy9ewMXFJFCidezYEVeuXIHIVptro0aNUuqnX29JKdCfLzyE7QHOK9DaW5k9SIAESIAESCB5EGhcOjfmdqySZC9LgdZG67QCvWjRIgwePBh+fn7Ili1bPIH28PCAKONIKAN99OhR5WPB1q1bJ5iBfvLkCVavXm2Wrj0y0IfCbuH6vSfaEXewHncfPcPgFX4xqx74ztsonDOjrre4/fAphq48FfOM/o1LoKhbpjjPnLsnFMcu3FF+r07xnMiSLjXW+4Urv3bPmh5D3y2law2yDv5hSyBCrz9Qllfv7VxoU6WArEvlukhACgJrT1zBxlNXlbXkdU2H4e+VtmhdR87fxu97z8X0fadMbrSo4G7RWHOdRqw+hRv3nyo/blEhH94pkyde112B1/HX4Ytxfv+zWoVQrXB2XXM72uBpW4MQdO2+smyRyX3XI+myubZks/TgBewNuYFGpdwwr1NVWz46zrMo0NponVag69evj7179yJt2rQxFB48eKD8WmSZCxYsqBxdJ0Q7uiVUAx0SEqL0j26iBlqUhYgPEC1t3IiWkorfT/yhoOq42Gz/qp61UaFA1sQ/EMDVu49RY8K2mGcs714TlQvG/ZfH138ew6rjV5Q+XTwLKwL909Yg5del82bBhr51dK1B1sFtZh+Ab9gtZXmdaxfCyPfLyLpUrosEpCAwdUsgftkeoqzl7dyZsfkbL4vWtf5kOHouPRrTt1f9YhjwztsWjTXXyWvyDly49VD58aCmb6NHvfhHri46EIbhq/3jPGJRl2qoUzyXrrkdbXCn+b7YFXRdWfZ33iXRrW7sv+dlfheRUFrme4ECLUGQnFagL1++jEePHsVBXLx4cSXj/PHHH2PZsmWYPXs2zp49i5QpUyr9RMZ65cqVyikcIsvs5uaGqVOnokuXLsrPeQqH8Tv25v0nqDw2VqDX9KqNcvmTXqC//fdkTJamR72iEDVnP/5nEuiy7lmwrreTCvScA/A9ZxLoz2sXxoj3LcumGb8zOCMJyEFg6n9B+GVbsLKYknkyY9PXlgn0Br9w9FgSK9B9GhRDvyb6BLrulB04f9Mk0N82LYnu9eJLYUICvflrL7ydJ7McQA1ahfhuaJO/6W8ORjcvg061Chk0s75phqz0g8hCNyzpht8/YwZaH019o51WoBPCoq6BjoiIUM6Bbt68uXK+s6+vL7p3765IdadOnZThQ4cOxZw5c5Rj6woVKoSBAwcqtdGiLITnQOvbeJaOvvPwKSqM+S+m+7renijr7mrp8AT7XYt8jOrj1RnoWqhc0FTmE91Grj6FPw6cV375TaMSyoc+UzYHKr8ul98Va3p56lqDrIPb/nYAPqEmgf7CszCGWfjX0bK+D9dFAklNILECvdEvHN3VAt2wOPo1jj3xKTHrrjdlB8KiBNpcVnWRz3kMXxVbwibmOT6iMbJmsPxug8SsTbYx6r9lnNTKAx9XfUu2JSa4nmiBblDSDfMp0HaNWbIVaEFdnLYhbh88duwY8ubNi/79+6NXr14xAREfEIqzocVFKiKb7eXlpZRuCJm2prGEwxpacfuKGujyo7fE/OaGPnVQOl+WxD8QQETkY1RTCfSKHrVQ6a24Aj1+wxn8tjtUmWewd0m8ePUKkzeZBLp8gaxY3bO2rjXIOrjdbz44EHpTWd6XdQpj6LvMQMsaK65LDgI//ReEnxORgd50KhzdFsdmoL9uVBxfN7KPQJ+b0Ez5yD45tcErTmKZr6kW/Oe2FXTXnxvFbuhKPyw5eAEUaKOIm58nWQm0vXBToBNP/t7jZ/AYFSvQm76ug5J5kl6gf9wSiF+j6hpHvV8aj5+/xMSNAcqLVHwrK1b2cE6Bbj/XB/vPmgS6q1cRDGnmnB9LJn5HciQJxCWQeIG+im6Lj8Q8TPxNV99GxXXhrf/DTpy7YfoIWPzB/6sE6noX+5zHMFUG+p9uNVG1UPL6gFDwGbXGHwv3hyms5nSonOAHl7qCkUSDh63yw2KfC6j/di4s6FwtiWYB6C3aaCnQ2ox09+BGTDzCB0+eo8zIzTEP2PKNF0rk1lerF3HvMaqNiy3hWNmjFiq+loGevj0YP2wx1TxP+NADQuTHbzAJtCj3WN69VuJfSuKRn847qHzhLdpXdYtgsDcFWuJwcWkSEBCnOUzbaqqBLpU3CzZa+IHxZv+r+GpRrECL04B6N9Qn0A1+2InQKIEe0qwkunrFr4FecvB8zClEOTOlweFhjSWgaPwSxq47jXlRp6D87/Nq8CrhGB9RRgu0OCVpIQXa+I2jmpECbQB+CnTiIT9+9gIlh2+KecDWfl4o5pb0Aj1vTyjGrj+jzDu1TXncevA05tdVC2XDP92cU6A7/H4Qe4JNAi2+Shd1lGwkQALmCfy8NThRJ/Rs8b+KriqBHtCkBHo10CnQP+6MOYZyaLNS+NKrSLyFxxXotDg8rFGyDK9aoP/+qqbDHOMn6tdFHTsF2v7blgJtQAwo0ImH/OT5C7w9LFagt/eviyK54p7ZbO3TLTkaT/2l+oz2lXA18jG+X3damUqclyr+geuMTS3Q4gt+8SU/GwmQgO0F+r/T1/Dl/w7HPFiccd+zfvxj56xh3/DHnTgbdY77sHdL4Ys68QVanOAgPkQTLWem5CvQ4p/n0edwr+3lCY/8+j5OtyZOevqKs77/d+A86pbIhT8+ZwmHHpZ6x1Kg9RK0YDwF2gJIZro8e/ESxYdujPnpzgH1UEjnRSqWCPRfhy7g2+Wmf8nM61gFF28/xOi1JoGuUSQ7/uzqnAKtPhu1Z/2iGPgOBTrxu5cjkwMBcYSdOIlDtDL5smB9H8uOuNx6+hq+UAm0uXObrWHYaOouhESYLgexRKBzZU6LQ0OTZwZaLdD/feOF4jpLA62Jk56+0QItSk5E6UlSNXqLNlkKtDYj3T24EROP8MXLVyg6ZEPMA3YPrI+3cmRI/AMB3Lj/BFVUZ0uLEzXEyRrq9s/hixj470nltxZ3qY7QG/cxIurygdrFcmDJFzV0rUHWwZ8t8MXOQNPlAra42EHW9+S6SMBWBH7dFpyoM+K3nbmGLn/EZqBtcZmHJQItLuGIvt2VAm26CXLPoPookF3fv1dstZ+0nhN9xCoFWotU0v+cAp30jPk1qw7Gr169QuHBsQJti3/QWSvQ4iv1gKv3Ys5OFVd7L+pSXcdbyTu08wJf7IgS6N4NiqG/zosd5H1TrowEbEMgsQK9PeAaPl8YK9DmPvqzZpWNp+5CcFQGWlwpLm5Rfb2pBdotc1r4MgOtZOHFHyYcoUWfHpLU/x5i4k97N1CgtRnp7sGNqA9hoe/Wxzxg/3cNkC9rel0PtOR2Q3UGWpzScTo8MubL9aT+k7+ul9M5uMvCQ9gWEKE8pY8NLnbQuRwOJwHpCahP7PFwd8Xa3pZdsvS6QJv76M8aAE1+2oWga6YSjhHvlcbnCQj0n74X8N0KU3kaBdqUgfYb1QSZ06W2BrXd+lKg7YY+3sQUaANiQYHWB1kt0D6DGyKPazpdD3xdoBP6gEQt0Kt61saZ8MiYv/ZM6q+fdb2czsFf/HEIW8+YBLpvw+L4RufNaDqXw+EkID0BtUBbc0vpjoAIdF54KOb9zNUsWwPgnZ92I/DaPWXIyPdLo3Pt+BlotUDnzpIWB4ckzxro3UHX0XG+r8IqZJw3UqV0sQa13fpSoO2GngJtD/QUaH3U1QLtO6Qh3LLoE2hxJF2l72OvB7dEoAOvRsZ8VJjUN0Dpo6VvtDgVQJwOIJotLnbQtxqOJgH5CczYEYIpm023lFol0IER6LwgVqDNlVxYQ6DptN1KudmbBFr9gXRyFmjBSFynLv5G8/VvYKxhbnTf0Wv9sWBfGDyL5cTiL5KulJDeoh1ZZqC1GenuwY2oD6FaoG1Rq3b7wVNUVAn0ut6eKOse9wij/WdvoP3cg8rCxYeLPuduYlDUR4WNSrlhXqeq+l5K0tFd/3cYW6IEul/jEkoZBxsJkIB5AmqBLp/fFat7WVbCsTMwAp+pBNpcxtga9mqBFjeofpZABlot0HmypIPPkIbWTMG+diYQLdBJ/TE7vUU70BRobUa6e3Aj6kOoFuijwxsje8Y0uh5oiUCLjxfFFb2iLk5cRvDvkUsY8M8JZd7GpXNjbscqutYg6+CvFh3GZn9TBtoWN6PJ+p5cFwnYikBiBXpX0HWIYyOjmznhtWadaoEe3bwMOtUqFG/434cuYtBy0wlDFGhr6MrRd8za05i/7xwo0PaPBwXagBhQoPVBVgv08RGNkTWDPoG+8/ApKoyJLeFIKAP9+opXHL2Efn+bBPqdMrkxp4NzCnT3xUew8dRV5T1tcbGDvshzNAnIT2DmzhBM3mQq4RClAOJYTEuaugZX9B/Togw61owvvJY8K7qP9897lO813vQ8tUDndU2HA4OZgbaGsb37Rgt0raI5sPTLpDtOld6iHWkKtDYj3T24EfUhVAv0iZFN4Jpe39fSiRHoVccu4+u/jisv4l02D2Z9WlnfS0k6useSI9jgR4GWNDxcloQEZu08i0mbApSVVSiQFeKjY0vanuDr6PB7bAb6+xZl0EGnQDf7eY9yYpBo5p739+GLMeVo+VzTYT8F2pJwSdMn+gKYmkVyYFlXCrQ9A0OBNoA+BVofZLVA2+K4obsPn6H8mC0xi1rfxxNl8r35GtfVxy+j758mgX7XIy9mfFJJ30tJOrrnkqNY7xeurM4WN6NJ+ppcFgnYjEBiBXpv8A18+rvpOwvRxrYsi09rFNS1rnd/2QP/KxRoXRAlHzx23WnM23sOFGj7B4oCbUAMKND6IKsF+vSYd5AhTSpdD0yMQK89cQW9lx1T5n2vXF5Mb++cAt1r6VGsO2kSaFvcjKYrUBxMAg5AYPaus5i40ZSBrvhWVqzsYVkGel/IDXwyL1agx31QFp9Ut6FAtyyLDgkIufqITves6bHvuwYOQJlLjPmDVpRA1yiSHX92rZlkYOgt2mgp0NqMdPfgRtSHUC3QAd83RbrUKXU98O6jZyg/OjYDvaFPHZTOl+WNz1x/Mhw9lx5V+jQvnw+/tKuoaw2yDhZ/SBB/WBBtsHdJfFW3qKxL5bpIQAoCc3adxYQoga70VlassFCg94fcQHuVQE/40APtqr2l653e+3UPTl02ZaDNZbTVH0RToHXhtsvgcetPY+6ec6heODv++ooCbZcgRE1KgTaAPgVaH2S1QAeObYq0qYwXaHFeaPclJoFuWSEfprV1ToHu++cxrD5uEmhb3IymL/IcTQLyE0i0QKuOyhRvOamVBz6uqk+g3/91L/wu31WgmctoU6Dl31NvWuH4DWfw2+5QCrQEYaRAGxAECrQ+yGqBDh7njdQ6b4yKfPwM5UbFZqA39q2DUnnfnIHedOoqui0+orzIhxXdMfXjCvpeStLRX/95DKuiBNoWN6NJ+ppcFgnYjMBvu89i/AZTCUflgtmwvHsti5594OxNtJvrE9N3cqtyaFO1gEVjzXVqPn0vTl4yCfT4DzzQvnp8IV9+5BL6Rx3JyQy0Ltx2GRwt0NUKZ8ffzEDbJQbRk1KgDcBPgdYHWS3QoeObwcUlha4Hvi7Qm76ug5J53izQ4nY+cUufaK0r58cPH5XXtQZZB3/z13GsPHZZWR4FWtYocV0yEVALdJWC2fCvhQLtE3oTbX+LFegprcvhoyr6BLrF9L04YYVA58+WHnu/ZQ20TPtJay0TNpzBnN2hqFYoO/7uxhIOLV5J+XMKdFLSjXo2BVofZLVAn5vQDClS6BPoe4+fwUOVgbZEoLeduYYuf5gEuk2V/Jjc2jkFut/fx7HiqEmgR7xXGp97FtYXPI4mAScnMHd3KMZtOKO8pTUCfTD0Jj5WCbT4Q7n4w7mephZoczXV6jPtKdB6aNtn7ISNZzBnFwXaPvTjzkqBNiAKFGh9kNUCHTbxXX0PA3D/yXOUHbk55jmbv/bC23kyv/G5OwIi0HnhIaXPx1UKYFLrcrrXIeMD+v99AsuPXlKWZourhWV8R66JBGxJYN6eUIxdbxLoqoWy4Z9ulpVw+J67hTZzDsQsZWqb8viwkk6BnrEPJy7eUZ458UMPtE3go0S1QBfInh57BjEDbcv9kNTPihZoa/ZaYtZEb9GmRoHWZqS7BzeiPoQyCPTOwAh8tsAk0G2rFsDEVs4p0OK6cvGRkWi2uFpYX+Q5mgTkJ5BYgT4UdgsfzY4V6J8+Lo8PKuoT6JYz9uF4lECb+yhx5bFL+OYv062qFGj599frKxRHJoqjE635247EvCW9RZsaBVqbke4e3Ij6ENpaoB88eY4yqgz0lm+8UCL3mzPQ6mt3nVmgB/17An8fNgm0La4W1hd5jiYB+QmoBdqautTDYbfQWiXQP7etgBYV3HW98Acz9+HYBVMG2txHiWqBfit7BuweVF/XnBxsLAFx66W4vIcCbSz3hGajQBsQAwq0PsgyCLT61jBnFuhv/z2Jvw5fVAJmi6uF9UWeo0lAfgK/7z0Hcb2yaNacjHDk/C20mhWbgbaFQH84cx+ORgt063Jok8BHiauOXcbXf5luVaVAy7+/Xl9htEBbc+JLYt6S3qJNjQKtzUh3D25EfQhtLdAPnz5H6RGxNdD/feOF4hoZaPWlB84s0N8tP4k/D0UJtJmbzPRFk6NJwLkI2Eqgf21XEe+Xz6cLjrUCXTBHBuwayAy0LugGD568KQAzd56FNZf2JGaJ9BZtak4t0BEREejfvz82bdqER48eoW7dupgyZQpKly6tkOncuTMWLlwYh5K7uzvExhHt5cuXGD16NObNm4fbt2/D09MTM2fORLFixbTJqnpwI1qFK15nGQRafWarMwv04BV+WOZ7QYmBLa4W1hd5jiYB+QnM33sOY6Iy0NbcDnfk/G20mrU/5gWnt6+I98rpE2jxPPFc0cwdi7f6+GX0/dOUgaZAy7+/Xl/hlM0BmLGDAi1D5JxaoGvUqAEXFxf88ssvyJgxI4YPH479+/cjJCQEGTJkQOXKldG0aVP07t07JhYpU6ZErly5lF8LeRbCvGDBAgixHjRoEEJDQ+Hv7480adJYHD8KtMWoEuxoa4F+9PQFSo3YFDPX1n5eKOb25hpo9ZFTzizQQ1b6YelBk0Cbu4hBXzQ5mgSci4BaoGsUyY4/u1p2Nu/RC7fx4cxYgZ75SSU088irC07rWftxOEqgzR2LpxboQjkyYCcz0LqYGz04WqArvpUVKy28Nj4xa6S3aFNzWoG+efMmevXqhWHDhqFMmTIKiRMnTqBChQrw9fVFpUqVkClTJixbtgwtW7aMR+rp06fImTMnJk+ejG7duik/v3PnDvLly4f58+ejbdu22nSjenAjWozKTgJdF8XcMr1xkeov5p35GLthq/yw2Mck0OaOwdIXTY4mAecisGDfOYxea6qBtkagj124jQ9UAj3700poWlafQH80ez8OhZky0D9+VB6tEjhXmgLt2Pvvh82BmL4jBBRo+8fRaQX6dbTXrl3DgAEDsHPnTgQEBODixYsoVaoUAgMDUaJEiXiREJJdvXr1eD8XZRzlypVTMtOWNgq0paQS7mfrDPTjZy9Qcrg6A60t0OoPfpxZoIevOoVFPueVQJg7BktfNDmaBJyLwMJ95zAqSqBrFskU14nJAAAgAElEQVSBZV1rWPSC4rg5cexcdJv9aWU0LZvHorHmOrWZfQC+YbeUH5s7V3rNiSvos+yY0qdwzozYMaCerjk52FgCP24JxK/bQ1ChQFas6lk7ySant2ijTRYC3bVrV8ydOxdp06bFmjVr0KRJE/zzzz/4+OOPlezyxo0blVKPZs2aYezYsXB1dcWKFSvQqlUrPHz4EOnTp48h2aZNG+X31q1bZ5ZuZGQkxH+iW3h4OKpVq6ZIe/78+s751A6p8/VIaoHe1r8uiuZ6cwZa/detzizQI1afwv8OmATa3DFYzrfD+EYkkHgCaoGuVTQHln5pmUCLC09aqAT6tw6V0aSMToGecwDighZLBbpIzozYToFOfPDtMDJaoMsXyIrVFGg7RCB2ymQh0KdPn1Y+Ipw1axaWLl2KvXv3YvXq1Rg3bhzGjx+viHNwcLCSoS5QoAC2b9+u9OvQoQNevHihyHV069ixI65cuYKtW7eaDdyoUaOU+unXGwU6cXtdBoFWZ4ucWaBHrfHHwv1hSqDMfYSUuChyFAk4J4E/9odh5Bp/5eWsEeiTl+6g+fTYDPTcjlXQuHRuXZDEzYbRAm3uYpa1J66gd1QGmgKtC7ddBk/dEohftoeAAm0X/HEmTRYCHf3G4lQNDw8PpTTj999/x927d5E1a9YYID4+PqhZsybEf4u/vmjdunWCGegnT54oAm6uMQNt241ta4F+8vwF3h4WW8KxvX9dFNHIQPtduov3p+9VXsyZPyJUC7S5j5BsG10+jQQcm8D/DoRhxGqTQNculgNLvrAsA63+Z4oY+3unKmhYSp9AfzznAA5GZaCnfVwBLSvGv5gljkDnyojt/VnC4Ug7cOp/QfhlWzDK53fF6l6eSbZ0lnBoo3VagRZH2G3btg2i5EKcrBHdPvroI6W8YvPm2HOAo3/24MED5cPCf//9V8lEC9EWJ3YULVo0ZryogS5fvjxmzJihTTeqBzeixagS7CiDQJ+6fBfv/er8Aj16rT8W7DNloM19hKQvmhxNAs5FILECrf5niiAy/7MqaFBSn0C3/e0AfEJNJRzmLmZZd/IKei011UAXoUA73GaMFuhy+V2xhgJt1/g5rUAfP34cFStWVCS6QYMGCuRnz56hZMmSaN68OcRHheKkDrVIiw8M69evD1HyUaRIEbi5uWHq1Kno0qWLMp6ncNhnr9paoJ8+f4kSwzbGvIz4iEZ8TPOmdvpKJJr9skfp4swZaHGjmrgYQjRzfwVsn13AWUlATgKLDoRheFQG2rNYTiz+orpFC31doBd0ror6b7tZNNZcp3a/+eBA6E2LBbporozYxgy0LuZGD/7pvyD8vC0YFGijycefz2kF+tWrV/D29kZYWJjyAWG2bNmUmmchzEKuT548qYj0999/rxxJFxQUhJ49eyolHEuWLFFIDR06FHPmzFGOrStUqBAGDhyoPM/Pz4/nQBu4d2UQ6ICrkWg6zfkFeuy605gXJdDm/grYwNBzKhKQnoBaoOsUz4lFXSwTaP8rd/HuL6a/1RJtYeeqqKdToNvP9cH+s28W6PUnw9Fz6VFlTgq09Nsr3gKnbQ3CtK3B8HB3xdreLOGwZwSdVqAFVFHjPHjwYKxatUrJHtepU0fJKEefC718+XJFqsWxdqIWun379sopHOnSpVNiIj4gHDJkiHKRivgI0cvLSyndEDJtTWMJhzW04ve1tUA/e/ESxYfGZqB3DqiHQhoZ6KBr99Dkp93K4pw5Az1+wxn8tjv0jRksfdHkaBJwLgLi2Edx/KNo1gi0+m+1xNj/fV4NXiVMl3gltqkF+pd2FdE8gavB1QItzr/f2q9uYqfjODsQiBbosu5ZsK53nSRbAb1FG61TC7T26xvTgxtRH+ekFuhdA+uhYI43l3CERNxDo6nOL9ATNpzBnCiBNvcvYH3R5GgScC4Ci33OY1giBPpMeCS8fzb9rZZoi7pUQ53i+gT6k3k+2BdiykD/2q4i3k9AoDf4haPHElMGmgLteHvx563B+GlrECjQ9o8dBdqAGFCg9UG2tUA/f/ESxVQZaEsE+uz1+2j44y7lRZw5Az1h4xnM2WXKQJv7F7C+aHI0CTgXAbVAiwyyyCRb0tRlYaL/ki+qo3axnJYMNdvn03kHsTfkhvLz6e0r4r1y+eL1VQt0cbdM+I8ZaF3MjR4sTuAQHxKWyZcF6/swA200f/V8FGgD6FOg9UFOaoHePbA+3sqR4Y2LPHfjAer/sNPpBXrSpgDM2nlWec8Z7Svh3XL6rhbWF3mOJgH5CSw5eB5DV5pKOKwR6MCr9/DONNPfaom29MvqqFVUn0B3+P0g9gSbBNrc/343+oWje1QGmgIt//56fYXRAl06bxZs6EuBtmcEKdAG0KdA64Nsa4F+8fIVig7ZELMoSwT6/M0HqDvF+QV68qYAzIwS6JmfVEIzDwq0vt3L0c5OYOnBCxiy0k95zbolcuEPCzPQ6u8qxNhlX9ZAzaI5dOGyVqBL5M6ELd+wBloXdIMH/7otGD/+FwQKtMHgE5iOAm1ADCjQ+iDbWqBfvnyFIiqB3jOoPgpkf3MG+uKth6gzeYfyIs5cwjFlcwBm7DBloGd9UgneFGh9m5ejnZ5AYgU6+No9NI76MFlA+qtrDVQvYjuBNvcH4E2nwtFtsakGmgLteNtz+vZg/LAlCKXyZsFGZqDtGkAKtAH4KdD6IMsg0JduP4TnJOcX6B+3BOLX7SFKwGZ/WhlNy+bRFzyOJgEnJ7DM9wIGrzBloOu9nQsLO1tWA/26QP/9VU1UK5xdF62O832xO+j6G/8ArBbot3NnxuZvvHTNycHGEogW6JJ5MmPT10kXO3qLdlwp0NqMdPfgRtSH0NYCLc4ILzw4toTDkgz0lTuPUGviduVFPq5SAJNal9P3UpKOnrolEL9ECfScDpXxThkKtKSh4rIkIfCn7wV8FyXQ9d/OhQUWCrT6ZB/xKv92q4kqhfQJdKf5vtgVJdCzP62EpmXjl2BtOnUV3RYfUehRoCXZRFYsY8aOEEzZHAgKtBXQkqgrBTqJwKofS4HWB1kGgb569zFqTNjm/AL9XxDERyqi/dahMppQoPVtXo52egKJF+j7aDTVdLKPaMu710TlgvoE+rMFvtgZaMpAm/sbpM3+V/HVIpNAJ7WEOX3w7fCCFGg7QDczJQXagFhQoPVBtrVAi9Won7n32/rIn+3NNdARkY9RbbzzC3T0If2C0byOVdCodG59weNoEnByAn8duoBvl5tKOBqUdMP8z6pa9MbqozHFgBU9aqHSW9ksGmuuU+cFvthBgdbFUPbB0QKd1H97QG/R3gkUaG1GuntwI+pDmNQCve+7BnDPmv6Ni7xx/wmqjN2q9HHmjwijD+kX7/l7pypoWIoCrW/3crSzE1ALdMOSbvjdQoEOvX4fDaLOlheMVvaohYo2FGhzJVhb/K+iKzPQDrstZ+4MweRNgUlefkNv0d4iFGhtRrp7cCPqQyiDQN968BSVvv/P6QU6+oxR8aILPquK+iXd9AWPo0nAyQn8fegiBi0/qbylNQKtPltejF3dszbKF8iqi9bnCw9he0CE8gxzJVhqgU7qkxx0vQwHJ0hAnNMvzutP6hNU6C3aG5ACrc1Idw9uRH0Ik1qg93/XAPk0MtB3Hj5FhTHJTKA7V0X9tynQ+nYvRzs7gb8PX8Sgf00C3aiUG+Z1sqyEI+zGA9SLupxJjF3TqzbK5dcn0F0WHsK2KIGe27EKGidQgvXf6Wv48n+HlfVSoB1vd0YLdFJfgkNv0d4bFGhtRrp7cCPqQyiDQN999AzlR29JVhnohZ2roh4FWt/m5WinJxBXoHNjXqcqFr2z+nImMWBdb0+UdXe1aKy5Tl/8cQhbz5gy0Oa+YVALdFJfxqHrZTg4QQKzd53FxI0BoEDbf4NQoA2IAQVaH+SkFugDgxsgr+uba6DvPX4Gj1HJS6D/93k15WpiNhIgAfME/jl8EQNjMtCWC/SFmw/hNcV0trxo6/t4okw+vQJ9GFvPXHujQG89fQ1fRGWgKdCOt7Pn7DqLCRsDUMwtE7b2S7pbJOkt2nuDAq3NSHcPbkR9CGUQ6AdPnqPMyM3KizjzR4TqGuhFXaqhTnEKtL7dy9HOTuDfI5cw4J8TymuKkglROmFJU99uKvpv6FMHpfNlsWSo2T6iNENkmEUz9xGwWqDL5MuC9X3q6JqTg40lEC3QRXNlxLb+9ZJscnqLNloKtDYj3T24EfUhTGqB9hncEHlc071xkY+evkCpEZuSlUAv7lIdnsVz6gseR5OAkxOwlUBv+roOSuaxnUDP/6wKGpSMf4rOtjPX0OUPUw00BdrxNudvu89i/IYAUKDtHzsKtAExoEDrgyyDQD9+9gIlhycvgV76RXXUKkaB1rd7OdrZCSw/cgn9ozLQTUrnxm8WZqAv3X4Iz0mxJRybv/bC23ky68KlzkCbO0VHLdBl3bNgXW9moHVBN3hwtEAXyZUR25mBNph+3Oko0Abgp0Drg3wo7BaGrTyFrxsVh7dH/KtpE/N0tZQfHNIQubO8OQP99PlLlBi2MVlloJd+WR21ilKgE7O/OCb5EFAL9DtlcmNOB8tKOC7feYTaE7fHgPrvGy8Uz21DgTZzis72gGv4fKEpA02Bdrx9Ond3KMZtOIMiOTNi+wCWcNgzghRoA+hToA2AbOUU1gr08xcvUWxo8hLoP7vWQI0iOawky+4kkLwIrDh6Cf3+NtVAWyPQV+48Qi2VQG/t54VibvoEuuv/DmNLVA30Agq0U27EeXtCMXY9BVqG4FKgDYgCBdoAyFZOoRZo3yEN4aaRgX758hWKDNmQrDLQf3WtgeoUaCt3FrsnNwIrj13CN3+ZBLppmTyY3aGyRQjC7z5CzQmxGeht/euiaK5MFo0110ldwmHuGModARHovPAQM9C6SNtvcLRAF86ZETuYgbZfIABQoA3AT4E2ALKVU8QR6KEN4Zb5zSUcr169QuHByUug//6qJqoVzm4lWXYngeRFQC3Q3mXzYNanlgn01buPUWPCthhY2/vXRREbCvQfn1dD3QSOoaRAO/b+jBboQjkyYOfA+kn2MvQWbbQUaG1GuntwI+pGaPMHWCvQYgHRY5LLMXb/dquJKoUo0DbffHygUxFYdewyvv7ruPJO1gj0tcjHqD4+VqB3DqiHQjkz6mKjzkCbO8d9R2AEOi9gBloXaDsO/n3vOXy/7jQo0HYMQtTUFGgDYkCBNgCylVOoBfrQ0EbIlTmt5hPa/eaDk5fu4L9+dTWv/tZ8mKQd1OdAL+9eE5ULUqAlDRWXJQmB1ccvo++fJoFu5pEHMz+xLAP9ukDvGlgPBXPYTqDNneO+MzACn0UJtIe7K9b29pSEJJdhCYFogS6YIwN2MQNtCbIk60OBTjK0sQ+mQBsA2copEiPQog768fMXyJAmlZWzOU73uAJdC5ULZnOcxXOlJGAHAmqBftcjL2Z8UsmiVUREPkY1VQZ6z6D6KJA9g0VjzXVSf0RIgdaFUtrB8/eew5h1p/FW9gzYPYglHPYMFAXaAPoUaAMgWzmFWqAPD2uEnJm0M9BWTuGQ3dUCvbJHLVR8iwLtkIHkog0jkGiBvvcY1cbFlnDs/bY+8mfTJ9DqEg5zFyHtCrqOTvN9FT7MQBu2TWw20YJ95zB6LQXaZkB1PIgCrQOepUMp0JaSMq4fBTph1mqBXtWzNioUyGpcUDgTCTgggTgCXS4vZrS3LAN9/d4TVB23NeaN933XAO5Z0+sioBboJV9UR+0ELkKiQOtCbPfB0QJdIHt67BnUIMnWQ2/RRuvUAh0REYH+/ftj06ZNePToEerWrYspU6agdOnSCpnjx4+jb9++OHz4MHLkyIE+ffpgwIABMdRevnyJ0aNHY968ebh9+zY8PT0xc+ZMFCtWTJusqgc3olW4DOmsFugjwxohBzPQCne1QK/uWRvlKdCG7EdO4rgE1py4gj7Ljikv8K4VAn3j/hNUGRsr0AcGN0BeV9sJtLmbRHcHXUdHZqAddsMt3HcOo9aeRv5s6bH3Wwq0PQPp1AJdo0YNuLi44JdffkHGjBkxfPhw7N+/HyEhIYpQlyxZEi1btkS/fv3g4+ODHj16KILcuXNnJSZCnsWvFyxYAHd3dwwaNAihoaHw9/dHmjRpLI4bBdpiVIZ1pEAnjFot0Gt7ecIjv6thMeFEJOCIBNaeuILeUQL9Xrm8mG5hBvrm/SeorBJon8ENkcf1zcdpavFRZ6DN3SRKgdaiKPfP/9gfhpFr/CnQEoTJaQX65s2b6NWrF4YNG4YyZcooqE+cOIEKFSrA19cXW7duxYwZMxAWFoZUqUwfhQ0ZMgQrVqxAQEAAnj59ipw5c2Ly5Mno1q2b8vM7d+4gX758mD9/Ptq2bWtx+CjQFqMyrKNaoI8Ob4zsGS3/A5Fhi7TDRGqBXtfbE2XdKdB2CAOndCACaoF+v3w+/NquokWrv/XgKSp9/19MX0sudNJ6sPojQnMCvSf4Ojr8zhpoLZay/jxaoEW5jyj7SapGb9Em67QC/fqrX7t2TSnP2LlzpyLIH330EbJmzYqlS5fGdBVS3bhxY4i+QqyrV6+OwMBAlChRIqaPKOMoV66ckpm2tHEjWkrKuH5qgT42vDGyUaAV+GqBXt/HE2XyUaCN25WcyREJrDt5Bb2Wmko4rBHo2w+eoqJaoC240EmLjzoDvezLGqhZNEe8IWqBLuueBet619F6LH8uEYH/HQjDiNX+Sr08Bdq+gUkWAt21a1fMnTsXadOmxZo1a9CkSRNFgr29vTFp0qSYCJw+fVrJVh86dAgXLlxAq1at8PDhQ6RPH1uX1qZNG+X31q1bZ3HkKNAWozKsIwU6YdRqgd7Qpw5K58tiWEw4EQk4IoH1J8PRc+lRZenNy+fDLxZmoO88fIoKY2Iz0LY4DUgt0H92rYEaReIL9N7gG/j094PKeinQjrfjFh0Iw3AKtBSBSxYCLcRY1DzPmjVLyTjv3bsXQoTbt2+PMWPGxARC1DcXLVoUe/bsUTLQHTp0wIsXL5Q66ujWsWNHXLlyRSkBMdciIyMh/hPdwsPDUa1aNVy8eBH58+eXIvDJfRFqgT4+ojGyZmAJh9gTaoHe2LcOSuWlQCf3/63w/d9MQC3QLSrkw89tLSvhuPvwGcqP2RLzcFt8zKwW6L+61kB1CrTTbd9ogc7nmg77BzdMsvdj4k8bbbIQ6GgM4lQNDw8PpTRDZJmbNWuWYAb66NGjyseCrVu3TjAD/eTJE6xevdos3VGjRikfIL7eKNDaG9KoHhTohEmrBXrz1154O09mo0LCeUjAIQls8AtHjyWmDLRVAv3oGcqPjhVoW5SSqQX6769qolrh+DeJ7gu5gU/mMQPtkJsNwCKf8xi+6hTyuqbDAQq0XcPotAItjrDbtm2bkmlOmTJlDGRR+yyyw+L3xNF1ixYtivlZQjXQ4sQOkZWObqIGunz58soHiOYaM9B23dMWTa4W6BMjmsA1Q2qLxjl7J7VAb/nGCyVyU6CdPeZ8P30ENvqFo3uUQLeskA/TLMxARz5+hnKjYgXaFn8Tpv6IkAKtL66yjl7scx7DKNBShMdpBVqc8VyxYkVFohs0MH2p+uzZM+XouubNmyN37tyYPXs2zp49GyPYgwcPxsqVK5WPDEWW2c3NDVOnTkWXLl2U8TyFQ4o9a5NFUKC1M9D/feOF4hRom+w3PsR5CagF+oOK7vjp4woWvey9x8/goRLoEyObwDW9vj/IqzPQ/3SriaqF4meg94fcQHtmoC2KkYydogU6T5Z08BnCEg57xshpBfrVq1fKR4Killl8QJgtWzaMGzcOmzdvVi5QSZcuXYxMi/OdxdF23bt3V6S6U6dOSkyGDh2KOXPmKMfWFSpUCAMHDlSe5+fnx3Og7blrbTB3HIG2wb+4bLAkKR6hzkBv7VcXxdwySbEuLoIEZCWw6VQ4ui02lXBYI9D3nzxH2ZGbY17Lb1QTZE5nO4H+t1tNVKFAy7ptEr2uJQfPY+jKU8idJS0ODmmU6OdoDWQNtBYhwGkFWrz63bt3IbLKq1atUrLHderUUTLK0edCizpocfvgsWPHkDdvXuXWQnF2dHQTHxCKs6HFRSriI0QvLy+ldEPItDWNG9EaWsb0pUAnzFkt0Nv610XRXBRoY3YkZ3FUAmqB/rCiO6ZamIF+8OQ5yqgE+tTod5AprelOgsQ2dQZ6efeaqFwwgQz02RtoP5c10IllbO9xSw9ewJCVfhRoewcCTi7QEvBVlkCBliUSsetQC/TJUU2QRWfmR743TNyK1AK9Y0A9FM6ZMXEP4igSSCYENp26im6Ljyhva41AP3z6HKVHxGagT495BxnS2FKga6FywWzxonDg7E20m+uj/L6HuyvW9vZMJpFyjteMFmi3zGnhO5QZaHtG1akz0PYEq56bAi1LJBIWaFv81al8b5i4FakFeueAeihEgU4cSI5KNgQ2+1/FV4uiBLqSO6a2sawG+tHTFyg1YlMMpzNjmiJ9mtgP3hMDUP0R4fLuFOjEMJR9zDLfCxi8wg+5MqfFIQq0XcNFgTYAPwXaAMhWTqHOQFOgY+GpBXrXwHoomIMZaCu3FrsnMwJqgW5VKT9+bFPeIgKPn71AyeGxAh3wfVOkS207gV7RoxYqvRU/A+0TehNtf2MG2qIgSdjpT98L+I4CLUVkKNAGhIECbQBkK6dQC7Qtag+tnF7a7mqB3jOoPgpkzyDtWrkwEpCBwBb/q+galYHWI9CBY5sibSp9Aq2ugV7ZoxYqUqBl2CI2XUO0QOfMlBbi9sqkavQWbbIUaG1GuntwI+pGaPMHUKATRkqBtvlW4wOdnMB/p69BiKtorSvnxw8fJS4DHTzOG6lTxt56mxhsaoFe1bM2KhTIGu8xB0Nv4mNmoBODV4oxfx26gG+X+4ECbf9wUKANiAEF2gDIVk6hFmj/0e8go86v362cXtruaoHe+2195M/GDLS0weLCpCCgFuiPKufHFAsF+snzF3h7WGwJR8g4b6SyoUCv7lkb5TUEuqx7FqzrXUcKjlyEZQT+PnQRg5afRM5MaXB4WGPLBiWiF71FG5pDCLQ4Qk7cHJgmTRqcOXMG69evR61atZT/OELjRpQvShRo7Qz0/u8aIF/W9PIFjysiAYkIbD19DV9EZaCtEeinz1+ixLCNMW8SOr4ZXFxS6Hoz9UeE5gTa99wttJlzQJmHAq0Lt10GRwt0joxpcGQ4BdouQYiaVHqB3r17N1q2bIl///0XpUuXRokSJeDi4oL79+9j2bJlEFdzy94o0PJFSC3Qtjg+Sr43TNyK1BnoA4MbIK8rBTpxJDkquRBQC3SbKvkxubVlJRzPXrxE8aGxAn1uQjOkSGE7gV7TqzbK5Y9fwkGBduyd+ffhixj070lQoO0fR+kF2tPTU5Hmn3/+WbkRcMqUKQgODlb+/3nz5imXoMjeKNDyRYgCrZ2B9hncEHlc08kXPK6IBCQisO3MNXT5w1QDbY1AP3/xEsVUAh028V3db6WugV7byxMe+V3jPfNQ2C18NJsZaN2w7fSAfw5fxMB/TyJ7xjQ4ygy0naJgmlZ6gc6QIQNOnTqFIkWKoFmzZihQoIByvfb58+eVq7hFeYfsjQItX4TUAm2L81fle8PErUidgT44pCFyZ6FAJ44kRyUXAtsDruHzhSaB/rhKAUxqXc6iV3/x8hWKDtkQ09fWAr2utyfKulOgLQqGA3WKFuhsGVLj2IgmSbZyeos2WukFOleuXNizZw8KFy6MHDlyKNdqi7KNgwcPokWLFrh69ar2W9q5BzeinQOQwPRqgbbF+avyvWHiVqQWaN8hDeFGgU4cSI5KNgQSK9AvX75CETsI9OGwW2jNDLTD7s9/j1zCgH9OgAJt/xBKL9Bt2rRRsszZs2fH8uXLER4ejrNnz+Krr75SstKiDlr2RoGWL0IU6IRjQoGWb69yRXIT2BEQgc4LD1mdgX716hUKD7ZtBlr9EaG5DDQFWu79pLW65Ucuof8/J5A1Q2ocZwZaC1eS/lx6gb5+/Tq6deuG0NBQjBw5UvmgsH///koGWnxYmCdPniQFZIuHU6BtQdG2z6BAU6Btu6P4tORKYEdgBDovMAl026oFMLGVZSUcor/6n0O2KOFQC/T6Pp4oky9+CceR87fQahZroB11v0YLtGv61DgxkiUc9oyj9AKdEJynT58qR9o5SqNAyxcpCjQFWr5dyRU5IgG1QFtTA50UAq3+iHBDnzoonS9LPKRHzt9Gq1n7ld/nMXaOt+NWHL2Efn+fAAXa/rFzCIEWZRtz585FQEAApk2bhl27dsHDw0P5iNARGgVaviipBdoWV+jK94aJWxFLOBLHjaOSL4GdgRH4LCoDLZNAb+xbB6XyUqCdbWeuPHYJ3/x1AlnSpcLJUe8k2evRW7TRSi/QISEhqF69OlxdXSECKiT622+/xcaNG7FlyxaHuEyFG1F7IxrdgwLNDLTRe47zOScBtUDbu4RDnYHe9HUdlMwTX6CPXriND2cyA+2ouzFaoDOnSwU/CrRdwyi9QIuTNsRJHCIDnSVLFpw4cQIFCxbEZ599hgsXLijZaNkbBVq+CKkFOmisN9KkcpFvkXZYETPQdoDOKR2awK6g6+g031d5B3tnoNU10BRoh95WZhe/6thlfP3XcVCg7R9f6QVayLOQZHELYebMmRWBFqdviEy0yEzfvXvX/hQ1VkCBli9EFGhmoOXblVyRIxLYHXQdHSUU6M1fe+HtPJnjIT124TY+iMpAe7i7Ym1vT0fEnmzXvPr4ZfT98zgyp00Fv9Es4bDnRpBeoLNly6acA122bNk4Ai1O4fD29satW7fsyc+iuSnQFmEytJNaoIPHeSN1SmagRQA2nQpHt8VHlVicHNUEWdKlNjQunIwEHI2ATAKtLuHY8o0XSuSmQDvaftJab7RAZ9fZ4EcAACAASURBVEqbCqco0Fq4kvTn0gt027ZtFQBLlixB1qxZcfLkSaWU48MPP4Sbmxv++eefJAVki4dToG1B0bbPoEAnzFOcTTt9ewjcs6XHh5Xy2xY6n0YCTkhgT/B1dPhdjhIOtUD/940Xiicg0Mcv3kHLGfuUSDAD7XgbkgItT8ykF+grV66gXr16uHnzplKuIU7eENd4i4tVdu/erdRDy94o0PJFSC3QIeO8kYoZaPmCxBWRgAMQ2Bt8A5/+flBZqUwfEW7t54VibvEz0GqB5jF2DrDBXlvimhNX0GfZMWRMkxL+Y5om2QvQW7TRSi/Q4hUePnyo3Dh47NgxvHz5Uinn+PTTT5VMtCM0bkT5oqQW6LPjmyGlSwr5FskVkQAJSE9ALdAyfURoTqBPXLyDFlEZaAq09Nsr3gKjBTpDmpQ4TYG2awAdQqAFocePHyMoKAhp06ZVPiJMndpxajMp0Hbd4wlOToGWLyZcEQk4IoF9ITfwyTxTBlouga6LYm6Z4iGlQDviLotd89oTV9B72TFQoO0fR+kF+sWLFxgwYABmz54NcQOhaOnTp1fOgh42bBhSpJA/c0iBtv9Gf30FaoEOHd8MLsxAyxckrogEHIDA/pAbaB8l0PYu4VAfY7etf10UzRVfoE9euoPm00010MxAO8AGe22J0QKdPnVKnPmeJRz2jKD0Aj106FDMnz8f33//vXJsnRBqcSrHqFGj0KtXL4wePdqe/CyamwJtESZDO1GgDcXNyUjAaQmoBdreGWj1R4Tb+9dFEQq00+27dSevoNfSY0iX2gUB33sn2fvRW7TRSi/QuXPnxoIFC9CsWbM4b7N8+XJFoMU137I3bkT5IqQW6HMTmjnE32TIR5ErIgES2H/2BtrPlaOEQy3QOwbUQ+GcGeMFyO/SXbw/fS8z0A66ddefDEfPpUcp0BLET3qBFkfXHThwAKVKlYqD6/Tp00pG+t69e2YxijOihwwZgnXr1iEyMhLlypXDxIkT4elpOji+c+fOWLhwYZzx7u7uypXhookPFkWGe968ebh9+7YybubMmShWrJhVoaNAW4XLkM4UaEMwcxIScHoCaoGWqYSDAu2cWy9aoNOmckHgWGag7Rll6QW6b9++iIiIwB9//IE0adLEsPr888+RKlUq/Pbbb2b5NWnSRBk7ffp05TrwGTNmKDJ89OhR5Ti8ypUro2nTpujdu3fMM1KmTKn0FU3IsxBmkQEXYj1o0CCEhobC398/zlq0AkiB1iJk/M/VAh028V3jF8AZSYAEnILAgbM30W6uj/Iu9i7hUNdA7xxQD4USyECfunwX7/3KDLSjbr4NfuHoseQo0qRyQRAF2q5hlFKgGzRoEAPl+fPn2Lt3L/LmzYuqVatCCK44zu7ixYto0aIF/v333wQBhoSEoHjx4ti3bx9q1aql9BGXRJQoUQLt2rXDyJEjkSlTJuV4vJYtW8Z7hvhgMWfOnJg8eTK6deum/PzOnTvIly+fUpMdfcGLJdGjQFtCydg+FGhjeXM2EnBWAj6hN9H2N/kEetfAeiiYI34JBwXasXfiRr9wdKdASxFEKQValFZY2kR2OKEmZHf//v1o2LChcvRddBPlFyLrLOqnRVlIYGCgItWvN19fX6VE5PWfizIOUQoiMtOWNgq0paSM60eBNo41ZyIBZyYgk0Cra6B3D6yPt3JkiIeeAu3YuzFGoFO6IGgcSzjsGU0pBTqpgIhrv9u0aYM1a9Yo50p//PHHSnZ548aNcHFxUT5UHDt2LFxdXbFixQq0atVKucRFHJsX3cR48XuirtrSRoG2lJRx/SjQxrHmTCTgzAQOht7Ex5JkoNUCvWdQfRTIHl+g/a/cxbu/sITDUffkplPh6Lb4KFKnTIHgcXEPV7DlO9FbtGk6hECLQIr65ZMnTyoXqJQpUwZfffUV3nrrLe03jOohSjlE5llkpFetWqWUcIwbNw7jx49XxDk4OFg5b7pAgQLYvn07li5dig4dOijH5gm5jm4dO3aEuF5869atZucWHyyK/0Q3cVJItWrVlLKT/PnzW7xmdkw6AhTopGPLJ5NAciLge+4W2sw5oLyyTB8RUqCdcxduOnUV3RYfoUBLEF7pBfrUqVPw8vJChgwZFAkVQnv48GE8ePBAqW8WMq3VVq9ejfbt26NGjRpK5lhklEU99N27dyFO+YhuPj4+qFmzJsR/C2lv3bp1ghnoJ0+eQDzTXBNnVCd0PjUFWitSxv2cAm0ca85EAs5MQC3QMn1EaE6gT1+JRLNf9igh8XB3xdreplOp2ByDQLRAp3JJgZDxzEDbM2rSC7S3tzcyZsyIJUuWxNQyi/KLTz/9VCnD0CqlECdwiJM8PvzwQyxevDhOPfTr4IWUiw8LxYeJIhMtaqDFx4hFixaN6SpqoMuXL69kxM01ZqDtuaUtm5sCbRkn9iIBEngzgUNht/DRbFMGWiaB3vttfeTPFr+EQy3QvInQ8Xb3Zv+r+GrREaR0SYGzFGi7BlB6gRZCKzLCZcuWjQNKlHOIzLT4WNBcmzVrFnr06IE+ffrgp59+ilOKITLSN2/exObNm2OG79y5E/Xr14c4Y7pIkSJwc3PD1KlT0aVLF6UPT+Gw61616eQUaJvi5MNIINkSUAu0vUs41DXQ+75rAPessd/vRAfoTHgkvH82ZaAp0I63bbf4X0VXCrQUgZNeoMXxdSLLLM5sVrdDhw6hcePGZgU6KChIke733nsv3okZooRDXAfevHlz5YpwcSSd6N+zZ0+lhENku0UT14jPmTNHObauUKFCGDhwIMLCwuDn58dzoKXYvolfBAU68ew4kgRIIJbA4bBbaC1JBlot0Pu/a4B8FGin26rRAu2SAgidkHR3GPAjQu2tI71Ai1IN8dGeKKvInj278kY3btxQTtMQ9cvitIyEmvg4UAhwQq1Tp07KDYTiOnDxIWFAQIDyLJGVFqdwpEuXThkm6q3FTYbiqLxHjx4pGW9RuiFk2prGjWgNLWP6UqCN4cxZSMDZCRw5fwutZslRwqEW6AODGyCva/wMdMDVSDSdxgy0o+7L/05fg4hzihTAOQq0XcMovUAL+RQXoYhrucV5zSlSpFDOZs6WLRt27dqllFrI3ijQ8kWIAi1fTLgiEnBEAmqBtncJh/omQgq0I+4m7TVvPX0NX1CgtUEZ0EN6gRanZYiP+xYtWgRxIof4tYeHh5ItFuc1O0KjQMsXJQq0fDHhikjAEQkcOX8brWbtV5Yuk0D7DG6IPK6mv01Vt8Cr9/DOtN3Kb7EG2vF2XLRAi5WHTWQJhz0jKL1AV6lSRalBFrf/OWqjQMsXOQq0fDHhikjAEQkcvXAbH86UQ6DVJRwHhzRE7iwUaEfcU29a87Yz19Dlj8NKFwq0faMrvUDnzJkTBw8ejHOUnH2RWT87Bdp6Zkk9ggKd1IT5fBJIHgTUAm3vY+zUAu07pCHcEhDooGv30OQnZqAddXduD7iGzxdSoGWIn/QCPWXKFKxfv145AaNYsWJxrtUWAK25jdBewCnQ9iJvfl4KtHwx4YpIwBEJHLtwGx9EZaClEuihDeGWOX4GmgLtiLssds1qgT43oZnyXVhSNHqLNlXpBVp9jbZ6o4haaPFrcVKG7I0bUb4IUaDliwlXRAKOSOD4xTtoOWOfsnSZaqB9zQh08LV7aMwMtCNuNWXNOwIi0HnhIeX/Dx3fDC7iPLskaPQWbajSC7Q4aeNNrW7dutpvaece3Ih2DkAC01Og5YsJV0QCjkhALdD2zkCrT+E4NLQRcmVOGw9pSMQ9NJrKEg5H3GuKQAdGoPMCCrQM8ZNeoNWQxFF2KVOmdJjTN6LXToGWYavHXQMFWr6YcEUk4IgETly8gxZRGWh7C7S6BvrwsEbImYkC7Yh76k1rVgu0uMpbXOmdFI3eok3VIQRa1EH//PPPCA8PV96ocOHC+Pbbb/Hll19qv6EEPbgRJQjCa0ugQMsXE66IBByRgFqg7V3CoRboI8MaIUeCAn0fjaaa/maXx9g53o7bGRiBz6Iy0CHjvJEqpUuSvAS9RRur9AI9adIkjBkzBn369FGu2X758iX27dunXM/9008/oWvXrtpvaece3Ih2DkAC01Og5YsJV0QCjkjg5KU7aD7dVAMtUwb66PDGyJ4xTTykIREUaEfcZ9Fr3hV0HZ3m+yq/pEDbN5LSC3SBAgUgruXu0KFDHFLibOgJEyYgODjYvgQtmJ0CbQEkg7tQoA0GzulIwEkJ+F26i/en75VCoNU10OYE+uz1+2j4oykD7eHuirW9PZ00Ms75WmqBDh7njdTMQNst0NILdKZMmXD06FHlGm91E+Jcvnx5PHz40G7wLJ2YAm0pKeP6UaCNY82ZSMCZCagF2t4lHGqBPja8MbIlkIFWCzRLOBxvZ+4Ouo6OURnooLHeSJOKJRz2iqL0At2pUydkzpwZ06dPj8No9OjRytXe//zzj73YWTwvBdpiVIZ1pEAbhpoTkYBTEzh1+S7e+9WUgZZJoI+PaIysGeKXcIRev48GURloCrTjbc09wdfR4XdTCQcF2r7xk16g+/Xrp9Q7lypVCuLIutSpU+PQoUPYs2cPWrRogaxZs8YQFGUdMjYKtHxRoUDLFxOuiAQckYBMAq3+iPDEiCZwzZA6HlIKtCPustg1qwU6cGxTpE2VMkleiN6ijVV6ga5fv772W0T12LFjh8V9jezIjWgkbcvmokBbxom9SIAE3kxALdAyfUR4YmQTuKaPL9DnbjxA/R92Ki/FDLTj7e69wTfw6e8HlYUHfN8U6VJToO0VRekF2l5gbDkvBdqWNG3zLAq0bTjyKSSQ3An4X7mLd3+R7yNCCrRz7sx9ITfwyTwKtAzRpUAbEAUKtAGQrZyCAm0lMHYnARJIkIBaoGWqgT45qgmypIufgQ678QD1mIF22N1MgZYndBRoA2JBgTYAspVTUKCtBMbuJEACCRI4fSUSzX7Zo/zM3iUc6lM4/EY1QWYKtNPt2v0hN9CeGWgp4kqBNiAMFGgDIFs5BQXaSmDsTgIkkCCBM+GR8P5Zn0CnSAGcm/CubsLqjwhPjX4HmdKmivfM8zcfoO4U1kDrhm2nB+w/ewPt55pKOM6MaYr0aVgDbadQQEqBXrNmDZo0aYJ06dLZi4tN56VA2xSnTR5GgbYJRj6EBJI9AbVAJ7aEwyUFEGpjgfYf/Q4yUqCdbn+qBfr0mHeQIU38PyTZ4qXpLdoUpRRocXlKQEAA8ufPjyJFiijH1uXIkUP7bSTtwY0oX2Ao0PLFhCsiAUckEHA1Ek2nmTLQiRXoVC4pEDK+me7XV5dwmBPoCzcfwmuK6cQqnsKhG7nhDzhw9ibazfVR5qVAG44/zoRSCrSbmxu++uorNGrUCOIYu5UrVyJbtmwJkvLy8rIvQQtmp0BbAMngLhRog4FzOhJwUgKBV+/hnWm7dQl06pQpEDzOtgJtTq4u3nqIOpMp0I66HX1Cb6LtbyaBNveHJFu8G71Fm6KUAj1p0iQMGTIEr169QooUKZT/TqiJn7148UL7Le3cgxvRzgFIYHoKtHwx4YpIwBEJqAU6sR8RpknpgqBx3rpfX52BNlcfS4HWjdmuD1ALtLk6d1sskN6iTVFKgRbLvnv3Lu7cuYPChQvD19cXuXLlSvBtChYsqP2Wdu7BjWjnAFCg5QsAV0QCTkIg6No9NPnJlIFOrECnTeWCwLH6BVr9EaG5SzYo0I698Q6G3sTHURloCrR9YymtQEdj2bVrF2rXro1UqZKmUN4I/BRoIyhbNwcz0NbxYm8SIIGECagFOrE10OlSuyDgewo095g2Ad9zt9BmzgGlo7mjCrWfot2D3qLNSHqBFq+wbt06jBs3DidPnkTq1KlRunRpDBw4EB988IH2G0rQgxtRgiC8tgQKtHwx4YpIwBEJBF+7h8Y6M9AtKuTDz20r6n59dQmHuQz0pdsP4TmJNdC6YdvpAWqBNndZji2WRm/Rpii9QK9evRoffvihIsvig0FR87x7926sXbsWK1asQPPmzbXf0s49uBHtHIAEpqdAyxcTrogEHJFASMQ9NJqauBKOk5fuYOuZCHTxLAzX9PFvDbSWh1qgA8c2RdpU8c8IpkBbS1Wu/ofCbuGj2aYMNAXavrGRXqArVaqkyPPw4cPjkBozZoySmRb10ebarVu3lI8RRb/IyEiUK1cOEydOhKenpzLk+PHj6Nu3Lw4fPqwck9enTx8MGDAg5nEvX77E6NGjMW/ePNy+fVsZN3PmTBQrVsyqqFGgrcJlSGcKtCGYOQkJOD0BtUBbW8JhazhqgQ4a6400qVziTXH5ziPUnrhd+X0eY2frCCT98w6H3ULrKIE+MbKJTf7gldCq6S3asZReoNOnTw8/P7940hocHIzy5cvj4cOHZt9SXMYSERGB6dOnKx8hzpgxQ5Hho0ePKr8uWbIkWrZsiX79+sHHxwc9evRQBLlz587KM4U8i18vWLAA7u7uGDRoEEJDQ+Hv7480adJo043qwY1oMSrDOlKgDUPNiUjAqQmERNxHo6m7lHe09iNCW4NRf0QYPM4bqVNSoG3N2N7PiyPQI5rANYP+v7mgQCcuqtILdNGiRfHrr7+iWbO4Z2SuX78eX375Ja5cuZLgm4eEhKB48eLYt28fatWqpfQRx+GVKFEC7dq1gxBzIdRhYWExHyiKbLUoCxGXuDx9+hQ5c+bE5MmT0a1bN2W8OBUkX758mD9/Ptq2bWsxcQq0xagM60iBNgw1JyIBpyZw9vp9NPxRPoEOGeeNVAkI9JU7j1CLGWiH3ZNHzt9Cq1mmEo4TFGi7xlF6gRalG4sWLcKsWbOU0zhE27t3r5ItbtGiBX7++ecEAQrZ3b9/Pxo2bIi0adPG9BHlF02bNlUyyVmzZsXSpUtjfrZ161Y0btwY165dU8S6evXqCAwMVKQ7uokyDlEKIjLTljYKtKWkjOtHgTaONWciAWcmoBZomUo4KNDOuevUAn18RGNkzWD534ZbQ4Teok1LeoF+/Pixku1ds2aNcqmKaCKTLDLSf//9NzJkyKD9llE9/vnnH7Rp00Z51tChQ+Ht7Q1xaUt0O336NMqUKaNcHX7hwgW0atVKKRER2eroJsaL3xN11eaaqLcW/4lu4eHhqFatGi5evKhcT85mfwIUaPvHgCsgAWcgEHr9PhpEZaBlEuiz45shpYvp35nqFn73EWpOYA20o+69I+dvo9Ws/cryjw1vjGwZKdD2iqX0Ah0NRpRViFpoIc8eHh4oVaqUVcxEKYfIPIuM9KpVq5Sa6vbt20N8jBjdRFZalIzs2bNHyUB36NBBOfXDxSW2jqxjx45K2YjIVptro0aNUuqnX28UaKtClqSdKdBJipcPJ4FkQ+DcjQeo/8NO5X1lEujQ8c3gQoF2un149MJtfDiTAi1DYB1GoPXAEkfhCVmuUaOGkjkWGWUh4SKLnVAGWnxkKGS6devWCWagnzx5AvFMc40ZaD3RMmYsBdoYzpyFBJydgFqg7f0RofoUjnMTmsX8ra06BlfvPkaNCduU3+IpHI63O9UCfXR4Y2RnBtpuQXR6gRYncIij6sRZ0osXL46phxbyLI6uE/XV0S2hGmjxMaLISkc3UQMtTv8QHyBa2lhLZCkp4/pRoI1jzZlIwJkJhN14gHpRGWiZBDps4rsJYr8W+RjVx1OgHXVPHrtwGx9EZaCPDGuEHJliv/Gy5TvRW7RpOrVAiw8PxceG4nznn376KU4phjgPevbs2Th79ixSpjQdNj948GCsXLlSOYVDZJnd3NwwdepUdOnSRfk5T+HQ3lCO0oMC7SiR4jpJQG4CaoGWqYSDAi33vkns6o5fvIOWM/YpwynQiaVom3FOK9BBQUEoW7Ys3nvvvXgnZogSDiHI4hxocZOhON9ZXMjSvXt3Rao7deqk0BUfGs6ZM0c5tq5QoULK9eGiNlrUYvMcaNtsQHs9hQJtL/KclwSci8D5mw9Qd4qpBtoRMtARkY9RjRloh92EaoE+PKwRcjIDbbdYOq1Ajx8/XhHghJoQ5IULFyqnbYjs9LFjx5A3b170798fvXr1ihkiPiAUZ0OLi1QePXqkXCUuSjeETFvT+Fch1tAypi8F2hjOnIUEnJ3AhZsP4TVlBwXa2QMtyfuduHgHLaIy0IeGNkKuzCzhsFdopBfoc+fO4dtvv8WpU6cgjrR7vYmP/WRvFGj5IkSBli8mXBEJOCIBtUA7QglHxL3HqDaONdCOuNfEmk9euoPm000lHBRo+0ZReoEWH+2Jc5TF+cvp0qWLR2vkyJH2JWjB7BRoCyAZ3IUCbTBwTkcCTkrg4q2HqDPZlIGmQDtpkCV6LbVA+w5tCLfM8b3IFsult2hTlF6gM2bMqJzLXKlSJe23kbQHN6J8gaFAyxcTrogEHJGAown09XtPUHWc6R4DHmPneDvO79JdvD99r7Jw3yEN4ZaFAm2vKEov0EKcp02bptQfO2qjQMsXOQq0fDHhikjAEQlQoB0xao675lOX7+K9XynQMkRQeoH28fFRTscQH/sVLlw4zlF0AqAjiDUFWoatHncNFGj5YsIVkYAjErh0+yE8JzlOCceN+09QZSwz0I6418Sa1QJ9cEhD5GYG2m6hlF6gf//9d0Wgnz9/Hg9SihQplKu2ZW8UaPkiRIGWLyZcEQk4IgEKtCNGzXHXrBZon8ENkceVJRz2iqb0Al2gQAHlLGeRgc6QIUM8TgULFrQXO4vnpUBbjMqwjhRow1BzIhJwagKX7zxC7YnblXd0hHOgb95/gsrMQDvsnvS/chfv/mIq4aBA2zeM0gu0+IjQ39/f6rOX7Ys17uwUaJmiYVoLBVq+mHBFJOCIBK7ceYRaFGhHDJ1Drlkt0AcGN0Be1/RJ8h70Fm2s0gt069at0axZM3z++efabyNpD25E+QJDgZYvJlwRCTgiAbVAO8IxdrcePEWl7/9TUPMUDsfbcaevRKLZL3uUhe//rgHyZaVA2yuK0gv09OnTMXjwYHh7e+Ptt99G6tSp47AaMWKEvdhZPC8F2mJUhnWkQBuGmhORgFMTCL/7CDUnOE4JBwXasbfjmfBIeP9MgZYhitILtDh5w1wTHxHyJkIZtpHjrYEC7Xgx44pJQEYCV+8+Ro0Jppv9HKEG+vaDp6jIDLSMW8miNakFet93DeDODLRF3JKik/QCnRQvbfQzmYE2mrj2fBRobUbsQQIkoE1ALdCOUMJx5+FTVBjDEg7tyMrZI+BqJJpOM2Wg935bH/mzxT9cwRYrp7doU6RAazPS3YMbUTdCmz+AAm1zpHwgCSRLAtciH6P6eFMG2t4CHZ2dLJMvC9b3qZNgPCjQjr1NA6/ewzvTdlOgJQij9ALt4uICUaphrvEcaAl2kQMugQLtgEHjkklAQgIyCbTAI46py5I+NVKndEmQ1t2Hz1B+zBblZ/yIUMINpbEktUDvGVQfBbIzA22vKEov0AsXLowj0M+ePUNwcDDE70+dOhWffPKJvdhZPC8z0BajMqwjBdow1JyIBJyaQETkY1STJANtCWgKtCWU5O0TdO0emvxkykBToO0bJ+kF2hyepUuXYtGiRdi4caN9CVowOwXaAkgGd6FAGwyc05GAkxKIuPcY1cbJUcJhCeK7j56h/GhmoC1hJWOf4Gv30JgCLUVoHFagz58/j9KlS+PBgwdSgHzTIijQ8oWIAi1fTLgiEnBEAhRoR4ya465ZLdC7B9bHWzlYwmGvaDqsQE+cOBEzZszAxYsX7cXO4nkp0BajMqwjBdow1JyIBJyawPV7T1B13FblHe19jJ0loCMfP0O5UcxAW8JKxj4hEffQaKqphGPXwHoomCNjkiyT3qKNVXqBFudAqz8ifPXqFSIjI3Hnzh2MGzcO3333nfZb2rkHN6KdA5DA9BRo+WLCFZGAIxK4cf8JqoylQDti7BxxzSER99Fo6i4KtATBk16gR40aFe8UjjRp0qB27drw8vKSAKH2EijQ2oyM7kGBNpo45yMB5ySgFmh7H2NnCeF7j5/BgxloS1BJ2Uct0DsH1EOhnMxA2ytQ0gu0vcDYcl4KtC1p2uZZFGjbcORTSCC5ExDHxlWOykBToJP7bkj69z97/T4a/mjKQO8YUA+FKdBJD93MDA4h0Pv27cPevXvx9OlTiBIOdRsxYoTd4Fk6MQXaUlLG9aNAG8eaM5GAMxO49eApKkVdje0IAn3/yXOUHblZCQnPgXa8nRl6/T4aUKClCJz0Ai0+FhwyZAiyZcuGLFmyxIEmaqNDQ0OlAPmmRVCg5QsRBVq+mHBFJOCIBCjQjhg1x12zWqC396+LIrkyJcnL0Fu0sUov0Pnz58fnn3+OMWPGaL+NpD24EeULDAVavphwRSTgiARuP3iKig6UgX7w5DnKMAPtiFtNWfO5Gw9Q/4edyv+/rX9dFKVA2y2W0gt0hgwZcPLkSRQrVsxukPROTIHWS9D24ynQtmfKJ5JAciRAgU6OUbffO4fdeIB6FGj7BUA1s/QC3aRJE3Tu3Bnt2rWTAlhiFkGBTgy1pB1DgU5avnw6CSQXAncePkWFMf8pr+sINdAPnz5H6RGsgXbU/akW6K396qKYG0s47BVL6QV63rx5GDRokCLRJUuWRNq0aeOw6tixo8Xsxo4di61bt2LnTtNff4gmnrtw4cI4z3B3d4eQXtFevnyJ0aNHQ6zj9u3b8PT0xMyZM63KiFOgLQ6RYR0p0Iah5kQk4NQE7j58hvJjTBeTOIJAP3r6AqVGbFLWy48IHW9rnr/5AHWnmBxmaz8vFHPLnCQvQW/Rxiq9QLu4uJh9C/ER4YsXL7TfEsC0adPQr18/5exotUBXrlwZTZs2Re/evWOekzJlSuTKlUv5tZBnIcwLFiyAEGsh8+LDRX9/f4jzqC1p3IiWUDK2DwXaWN6cjQSclQAF2lkjK+d7Xbj5EF5TdlCgJQiP9AKtl9Hly5fxLaBR7gAAIABJREFUxRdfYM+ePShQoABy584dI9BCvjNlyoRly5ahZcuW8ab6f3t3AmRVcf59/BEIQjRAAg6yKcgiimz+EZRFw2shi5GgICL1ghCosIigbCWgYZPVCsFKQLaAEQKKAhKIxkgM0SwEKKCCEEBAlj8Q5EWWyAzDMrz1NN5bV7jSd+xzz3Lv91RZKtP0Oefz9Mz8pqdPH902r1y5cjJ16lTp27ev+bi+AbFixYoyf/586dKlS0qXR4BOicnXRgRoX7k5GQIZK3Aq77zUH3t5BjoKr/I+e/6i1H6RGeioDsjEAP3Bc/dLzfLMQAdVy4wP0KtWrTIBWZdv6E4e+/btiwfoHTt2yB133CE7d+6UWrVqXVWD9evXS5MmTa76uC7jqFevnpmZTuUgQKei5G8bArS/3pwNgUwVOH32vNT76s1+BOhMrXJ47uvgF7nSYurlGWgCdLB1yfgAncjbo0ePrwXot956S5544gkzu/zee++JLhdp166dCdulS5eW5cuXS8eOHSU3N1dKliwZ76pz587mz1avXp20eqdPnxb9J3YcOXJEGjduLAcPHhTdlo8jeAECdPA14AoQyASBxAAdhTXQzEBHe9QlBug/Pne/1GIGOrCCZnWAHj16tEyYMEEmTpxogvOnn34qQ4cONUs9PvzwQ1m8eLF069bNrLNOXIutDy4ePnzYPJCY7BgzZoxZO33lQYAObJxfdWICdHhqwZUgEGWB/549L3W/moEmQEe5ktG49sQA/f6z98vtN7OEI6jKZXWA1teCnzp1SsqUKRP3X7dundx3332i/9alF506dUo6A52fny8rV65kBjqoket4XgK0IyB/HQEEjEDUAnT+hYty+wusgY7q8P3fE7nSfMrlJRwE6GCrmNUBOhn9mTNnzIOFb7/9tpmJ1jXQu3fvlurVq8eb6xro+vXry4wZM1KqHmugU2LytREB2lduToZAxgp8mX9B7vrqzX5RmIEmQEd7KCYG6D8820Jq31wqLTdEbrGzZnWA7tq1qxw/flzef//ypvJ66BZ3LVu2lO3bt8ttt90mOTk5Mm3aNOnVq5f5OLtw2AdVFFoQoKNQJa4RgfALJL4aOwoB+tyFAqn1wnsGln2gwz++rrzCQyfzpNnkD80fvzeohdxRgQAdVBWzOkDrQ4Dt27eX8ePHmy3pdu3aJU8//bRZwvHb3/7W1GTUqFEye/Zss21d1apVZdiwYeZBxK1bt7IPdFCj1oPzEqA9QKQLBBAQAjSDwE+BwyfzpCkB2k/ybzxXVgdoVVm2bJl5kFC3tNO10DorrbtwlChRwqDpA4QjR440L1LJy8szL2LRpRsaplM9+FVIqlL+tSNA+2fNmRDIZIHEV2NHYRu78xcLpOYoZqCjOiYTA/S7A1vInRWZgQ6qllkVoINCJkAHJf/N5yVAh68mXBECURRIfDU2ATqKFYzWNR85lSf3Tbq8hOP3A5tLnYql03ID5BY7KwHabuTcgoHoTOh5BwRoz0npEIGsFEgM0FFYA33hYoHUYAY6smP1P6fOyr2T/kSADkEFCdA+FIEA7QNyIU9BgC4kGM0RQCCpQOKLSaIQoC8WXJLqI98198JDhNEb1IkBevUzzeWuSsxAB1VFArQP8gRoH5ALeQoCdCHBaI4AAgRoxkDgAkdPn5UmEy/PQBOggy0HAdoHfwK0D8iFPAUBupBgNEcAgYwI0AUFl+Q2ZqAjO5o/P31WGhOgQ1E/ArQPZSBA+4BcyFMQoAsJRnMEEEgqkPhikigs4SBAR3sgJwboVQOaS93KLOEIqqIEaB/kCdA+IBfyFAToQoLRHAEEMiJAX7p0SaqNYA10VIfz5/89K40nXF7C8bsBzaRe5TJpuRVyi52VAG03cm7BQHQm9LwDArTnpHSIQFYKJL7ZLwoz0AToaA/TY//Nl3smrCFAh6CMBGgfikCA9gG5kKcgQBcSjOYIIJBUIPHFJFEI0HoTsa9/7MIRvUGdGKBXPt1M6ldhBjqoKhKgfZAnQPuAXMhTEKALCUZzBBAgQDMGAhf4f1/mS6OXLs9Av/N0M2lAgA6sJgRoH+gJ0D4gF/IUBOhCgtEcAQSSCiS+mCQKbyJkBjraA/n4l/nyPwToUBSRAO1DGQjQPiAX8hQE6EKC0RwBBJIKJL6YhADNIEm3QGKAXtG/qTS85ftpOSW5xc5KgLYbObdgIDoTet4BAdpzUjpEICsFEgM0a6Czcgj4etP/PXteBi7ZbM456uE7pUbOjWk5P7nFzkqAths5t2AgOhN63gEB2nNSOkQgKwUS91UmQGflEMjImya32MtKgLYbObdgIDoTet4BAdpzUjpEICsFCNBZWfaMv2lyi73EBGi7kXMLBqIzoecdEKA9J6VDBLJSIHFfZWags3IIZORNk1vsZSVA242cWzAQnQk974AA7TkpHSKQtQKxrycE6KwdAhl34+QWe0kJ0HYj5xYMRGdCzzsgQHtOSocIZK0AATprS5+xN05usZeWAG03cm7BQHQm9LwDArTnpHSIQNYKEKCztvQZe+PkFntpCdB2I+cWDERnQs87IEB7TkqHCGStAAE6a0ufsTdObrGXlgBtN3JuwUB0JvS8AwK056R0iEDWCsS+nrzU4S75v/feGnqH2PXeVamUrH6mReivlwv0X4DcYjcnQNuNnFswEJ0JPe+AAO05KR0ikLUC72/7j/z7yGl55v/UlKJFrgu9AwE69CUK/ALJLfYSEKDtRs4tGIjOhJ53QID2nJQOEUAgIgIE6IgUKsDLJLfY8QnQdiPnFgxEZ0LPOyBAe05KhwggEBEBAnREChXgZZJb7PgEaLuRcwsGojOh5x0QoD0npUMEEIiIAAE6IoUK8DLJLXZ8ArTdyLkFA9GZ0PMOCNCek9IhAghERIAAHZFCBXiZ5BY7flYF6JdeeknWrFkja9eujcts2bJFBg0aJBs3bpSyZcvKwIEDZejQofGPFxQUyNixY2XevHly4sQJad68ucycOVNq1Khh1/2qBQMxZSrfGhKgfaPmRAggEDIBAnTIChLCyyG32IuSNQF6+vTpMnjwYLn//vvjAfr48eNSu3Zt6dChg/nYunXrpH///iYg9+zZ0+hpeNb/X7BggVSqVEmGDx8ue/fulW3btknx4sXtwiLCQEyJyddGBGhfuTkZAgiESIAAHaJihPRSyC32wmR8gD506JD07t1bPv74Y6lSpYqUL18+HqAnTZokM2bMkH379kmxYsWM1siRI2X58uWyY8cOOXfunJQrV06mTp0qffv2NR8/efKkVKxYUebPny9dunSxCxOgUzLyuxEB2m9xzocAAmERIECHpRLhvQ4CtL02GR+gV61aJUuWLBFdvjFu3DgTlmNLONq1aydlypSRxYsXx6V0iUerVq3k6NGjpm2TJk1k586dUqtWrXgbXcZRr149MzOdysFATEXJ3zYEaH+9ORsCCIRHgAAdnlqE9UrILfbKZHyATiTo0aPH1wK0huC2bdvKlClT4s22b98uderUkQ0bNsiBAwekY8eOkpubKyVLloy36dy5s/mz1atX24WZgU7JyO9GBGi/xTkfAgiERYAAHZZKhPc6CND22mR1gNYHAbt27WpmpmOHrm+uXr26WfKhM9DdunWTixcvSpEiReJtunfvLocPHzYPJCY7Tp8+LfpP7Dhy5Ig0btxYDh48KJUrV7ZXhRZpFyBAp52YEyCAQEgFCNAhLUyILosAbS9GVgfounXrii7jSDYDvWnTJvOwYKdOnZLOQOfn58vKlSuTCo8ZM8Y8fHjlQYC2D0i/WhCg/ZLmPAggEDYBAnTYKhK+6yFA22uS1QFaw7NuXbdw4cK4VLI10Lt37zaz0rFD10DXr1/fPIDIDLR9kIWxBQE6jFXhmhBAwA8BArQfytE+BwHaXr+sDtCTJ0+WWbNmyZ49e6Ro0aJGa8SIEbJixQqzC4fOMufk5Mi0adOkV69e5uPswmEfVFFoQYCOQpW4RgQQSIcAATodqpnVJwHaXs+sDtCff/652Qe6ffv2Zn/n9evXS79+/Uyofuqpp4zeqFGjZPbs2WbbuqpVq8qwYcPM2uitW7eyD7R9fIW2BQE6tKXhwhBAIM0CBOg0A2dA9wRoexGzOkArj+62oW8f3Lx5s1SoUEGGDBkiAwYMiMvpA4S6N7S+SCUvL8+8iEWXbmiYTvVgIKYq5V87ArR/1pwJAQTCJUCADlc9wng15BZ7VbIqQNs50tOCgZgeV5deCdAuevxdBBCIsgABOsrV8+fayS12ZwK03ci5BQPRmdDzDgjQnpPSIQIIRESAAB2RQgV4meQWOz4B2m7k3IKB6EzoeQcEaM9J6RABBCIiQICOSKECvExyix2fAG03cm7BQHQm9LwDArTnpHSIAAIRESBAR6RQAV4mucWOT4C2Gzm3YCA6E3reAQHac1I6RACBiAgQoCNSqAAvk9xixydA242cWzAQnQk974AA7TkpHSKAQEQECNARKVSAl0luseMToO1Gzi0YiM6EnndAgPaclA4RQCAiAgToiBQqwMskt9jxCdB2I+cWDERnQs87IEB7TkqHCCAQEQECdEQKFeBlklvs+ARou5FzCwaiM6HnHRCgPSelQwQQiIgAAToihQrwMsktdnwCtN3IuQUD0ZnQ8w4I0J6T0iECCEREgAAdkUIFeJnkFjs+Adpu5NyCgehM6HkHBGjPSekQAQQiIkCAjkihArxMcosdnwBtN3JuwUB0JvS8AwK056R0iAACEREgQEekUAFeJrnFjk+Aths5t2AgOhN63gEB2nNSOkQAgYgIEKAjUqgAL5PcYscnQNuNnFswEJ0JPe+AAO05KR0igEBEBAjQESlUgJdJbrHjE6DtRs4tGIjOhJ53QID2nJQOEUAgIgIE6IgUKsDLJLfY8QnQdiPnFgxEZ0LPO1j/2RcyasVWea5VLWlXt4Ln/dMhAgggEFYBAnRYKxOe6yK32GtBgLYbObdgIDoT0gECCCCAgEcCBGiPIDO4G3KLvbgEaLuRcwsGojMhHSCAAAIIeCRAgPYIMoO7IbfYi0uAths5t2AgOhPSAQIIIICARwIEaI8gM7gbcou9uARou5FzCwaiMyEdIIAAAgh4JECA9ggyg7sht9iLS4C2Gzm3YCA6E9IBAggggIBHAgRojyAzuBtyi724BGi7kXMLBqIzIR0ggAACCHgkQID2CDKDuyG32ItLgLYbObdgIDoT0gECCCCAgEcCsQBdt1JpWfVMc496pZtMEiC32KtJgLYbObdgIDoT0gECCCCAgEcCz76xWd775D+yon8zubNiKY96pZtMEiC32KtJgLYbObdgIDoT0gECCCCAgEcCly5dkrzzF+W7xYt51CPdZJoAucVeUQK03ci5BQPRmZAOEEAAAQQQQMAnAXKLHTrrA/T+/fulatWqV0nNnTtXevfuLVu2bJFBgwbJxo0bpWzZsjJw4EAZOnSoXTahBQOxUFw0RgABBBBAAIEABcgtdvysD9CrVq2Szp07y969e+W6666Li5UuXVpyc3Oldu3a0qFDBxk8eLCsW7dO+vfvLzNnzpSePXvadb9qwUBMmYqGCCCAAAIIIBCwALnFXoCsD9ATJ06UpUuXmpnmK49JkybJjBkzZN++fVKs2OW1YiNHjpTly5fLjh077LoE6JSNaIgAAggggAAC4RAgQNvrkPUBukuXLlKkSBFZvHjxVVrt2rWTMmXKfO1ja9askVatWsnRo0clJyfHLiwiDMSUmGiEAAIIIIAAAiEQILfYi5D1AbpOnTpSoUIFyc/Pl127dknNmjXlxRdflNatW0u9evWkbdu2MmXKlLjk9u3bRf/Ohg0bpFGjRnZhAnRKRjRCAAEEEEAAgXAIEKDtdcjqAH3u3Dm54YYbpGHDhvLyyy/LjTfeKIsWLZJXXnlFPvjgA+nTp4907dpVxo0bF5fUtdLVq1eXjz/+WJo3T74B/enTp0X/iR1HjhyRxo0by8GDB6Vy5cr2qtACAQQQQAABBBAISIAAbYfP6gCtPGfOnDHrm6+//vq4Vps2bcwDhTqAdBlHshnoTZs2meCd7BgzZoyMHTv2qg8RoO0DkhYIIIAAAgggEKwAAdrun/UBOhnRsGHD5A9/+INUqVLFbF23cOHCeLNU1kAzA20feLRAAAEEEEAAgXAKEKDtdcnqAP2vf/1LmjZtasJy4nKMli1bSvny5aVBgwYya9Ys2bNnjxQtWtRojhgxQlasWMEuHPaxRQsEEEAAAQQQiKAAAdpetKwO0AUFBSZA6zKOV199VcqVKydz5swx+zzrQ4IaonUf6Pbt28vw4cNl/fr10q9fPxOqn3rqKbvuVy0YiClT0RABBBBAAAEEAhYgt9gLkNUBWnmOHTsmzz//vLz77rty8uRJufvuu2Xy5MnSokULo6dBWt8+uHnzZrNbx5AhQ2TAgAF22YQWDMRCcdEYAQQQQAABBAIUILfY8bM+QNuJ3FswEN0N6QEBBBBAAAEE/BEgt9idCdB2I+cWDERnQjpAAAEEEEAAAZ8EyC12aAK03ci5BQPRmZAOEEAAAQQQQMAnAXKLHZoAbTdybsFAdCakAwQQQAABBBDwSYDcYocmQNuNnFswEJ0J6QABBBBAAAEEfBIgt9ihCdB2I+cWDERnQjpAAAEEEEAAAZ8EyC12aAK03ci5BQPRmZAOEEAAAQQQQMAnAXKLHZoAbTdybsFAdCakAwQQQAABBBDwSYDcYocmQNuNnFswEJ0J6QABBBBAAAEEfBIgt9ihCdB2I+cWDERnQjpAAAEEEEAAAZ8EyC12aAK03ci5BQPRmZAOEEAAAQQQQMAnAXKLHZoAbTdybsFAdCakAwQQQAABBBDwSYDcYocmQNuNnFswEJ0J6QABBBBAAAEEfBIgt9ihCdB2I+cWDERnQjpAAAEEEEAAAZ8EyC12aAK03ci5BQPRmZAOEEAAAQQQQMAnAXKLHZoAbTdybsFAdCakAwQQQAABBBDwSYDcYocmQNuNnFswEJ0J6QABBBBAAAEEfBIgt9ihCdB2I+cWDERnQjpAAAEEEEAAAZ8EyC12aAK03ci5BQPRmZAOEEAAAQQQQMAnAXKLHZoAbTdybsFAdCakAwQQQAABBBDwSYDcYocmQNuNnFswEJ0J6QABBBBAAAEEfBIgt9ihCdB2I+cWDERnQjpAAAEEEEAAAZ8EyC12aAK03ci5BQPRmZAOEEAAAQQQQMAnAXKLHZoAbTdybsFAdCakAwQQQAABBBDwSYDcYocmQNuNnFswEJ0J6QABBBBAAAEEfBIgt9ihCdB2I+cWDERnQjpAAAEEEEAAAZ8EyC12aAK03ci5BQPRmZAOEEAAAQQQQMAnAXKLHZoAbTEqKCiQsWPHyrx58+TEiRPSvHlzmTlzptSoUcOu+1ULBmLKVDREAAEEEEAAgYAFyC32AhCgLUYanjUwL1iwQCpVqiTDhw+XvXv3yrZt26R48eJ2YRFhIKbERCMEEEAAAQQQCIEAucVeBAL0NYzOnTsn5cqVk6lTp0rfvn1Ny5MnT0rFihVl/vz50qVLF7swATolIxohgAACCCCAQDgECND2OhCgr2G0fv16adKkiezcuVNq1aoVb6nLOOrVq2dmplM5GIipKNEGAQQQQAABBMIgQG6xV4EAfQ2j5cuXS8eOHSU3N1dKliwZb9m5c2fzZ6tXr076t0+fPi36T+w4ePCgNG3aVDSQV6hQwV4VWiCAAAIIIIAAAgEJHDlyRBo3biyfffaZVK1aNaCrCPdpCdDXqM+iRYukW7ducvHiRSlSpEi8Zffu3eXw4cOyZs2apH97zJgx5sFDDgQQQAABBBBAIKoCOvF3zz33RPXy03rdBOhr8C5btkw6deqUdAY6Pz9fVq5cmdIM9NmzZ0VnoatVqybFihVLS0FjPy0yy50WXk86pUaeMKa1E2qUVl5POqdGnjCmtRNqlFZeTzq31ejChQty7NgxqVu3rpQoUcKTc2ZaJwToa1Q0tgZ69+7dUr169XhLXQNdv359mTFjRmjGA+uVQlOKb7wQakSNwi8Q/ivk84gahV8g/FfI55F7jQjQ1zDUWeacnByZNm2a9OrVy7T8NrtwuJfJ3gOfDHajoFtQo6ArYD8/NbIbBd2CGgVdAfv5qZHdKOgW1Mi9AgRoi+GoUaNk9uzZZts6XUg/bNgw2bdvn2zdujXlfaDdy2TvgU8Gu1HQLahR0BWwn58a2Y2CbkGNgq6A/fzUyG4UdAtq5F4BArTFUB8gHDlypHmRSl5entx///1m6UbYnkrVXT90pnzw4MFSqlQp95FBD54LUCPPST3vkBp5Tup5h9TIc1LPO6RGnpN63iE1ciclQLsb0gMCCCCAAAIIIIBAFgkQoLOo2NwqAggggAACCCCAgLsAAdrdkB4QQAABBBBAAAEEskiAAJ1FxeZWEUAAAQQQQAABBNwFCNDuhvSAAAIIIIAAAgggkEUCBOgMKHZBQYF5dfi8efPkxIkToi96mTlzptSoUSMD7i78t/DFF1+YnVpWr14t+mRzvXr1ZPLkyaYOemzZskUGDRokGzdulLJly8rAgQNl6NCh8Rujfv7WeNeuXXL33XfLr371K+nRowc18pf/mmd7/fXXzefO3r17zcurxowZI48//jg1CkmNzp8/b2qycOFC872mQYMGMmXKFGnatCk1CkGNXnrpJVmzZo2sXbs2fjVefP+x9RGCWw/kEgjQgbB7e1INzxqYdau9SpUqyfDhw803oG3btoVqr2pv7zo8vT300EPy+eefm0B20003mW0O9YeZTZs2mf+vXbu2dOjQwWwxuG7dOunfv7+pV8+ePc1NUD//aqkBQL/Z6w8z+vmiAfr48ePUyL8SfOOZFi1aZOrx85//XB5++GFZsmSJCWt//etfpVatWtQoBDUaPXq0zJkzR37zm9/IbbfdZsLz0qVL5d///rdcf/311CjAGk2fPt18j9GtdmMB2ouvban0EeBtB3pqAnSg/O4nP3funJQrV06mTp0qffv2NR2G9W2J7ncbvh70Ne81a9aUv/3tb/FZmEuXLplv+E8++aSULFnSBGp9+U6xYsXMDehs9fLly2XHjh1C/fytqdrrDzF//vOf4wF60qRJ1MjfMlx1Nv2c0UDWqVMnefnll+Mfb926tfzwhz80/8/nUcBFEjEzzg8++KD5IUcP/Y1b6dKl5e233xb9zQ418r9Ghw4dkt69e8vHH38sVapUkfLly8cDtBdf22x9+H/H4TkjATo8tfhWV7J+/Xpp0qSJ7Ny504S22KHLB3Qpgc50cqRPQH9Y+fvf/26+qegMTOzQ5TNt2rQxvwkoU6aMLF68OP4x/RVbq1at5OjRoyZYU7/01Sex548++kgeffRRs6TmlltuiQfodu3aUSN/SvCNZ9EfJu+44w7zW5uGDRte1Y4aBVygr06vX+f0DXbvv/++CWu/+MUv5IUXXjC/7XzmmWf4PAqgTKtWrTK/rdHlG+PGjTPfU2Iz0F583tj6yMnJCeCuw3FKAnQ46vCtr0JnMjt27Ci5ublmtjN2dO7c2fyZrsvl8FfgrbfeEvX/3e9+J/oq+LZt25pfdcaO7du3S506dWTDhg1y4MAB6udDefQHnfr164vOpnTt2lWuu+66eIDWHzSpkQ9FuMYp9HPlxz/+sZlFGz9+vGzevFmqVatmwtkjjzxiJgOoUbA10rN/8skn8sQTT4h+DStatKgUKVLEzD63b9+eGgVfHrMEKjFAe/F5Y+ujUaNGIbjzYC6BAB2Mu2dn1XWD3bp1E33luH4xix3du3eXw4cPmwcKOPwT0KUcOvOsMzXvvPOOeZBTA5vODMSO2ANSGhb0ix31S399dDmNLhN44403zMkSAzQ1Sr+/7Qyxr2MamnWdrf6ws2zZMpkwYYJ88MEH0qdPHz6PbIg+fFwnB1555RUZNmyYed5m7ty58uabb4r+duexxx6jRj7U4FqnuDJAe/G1zdZH7GH5gG89kNMToANh9+6k+k1G1w0mm4HOz8+XlStXencyerqmgFprWL733nvNzL/+RqBu3bqivwJLNgOtv67WME390juwdMeAESNGyNatW+X73//+VQGaGqXXP5Xe9UE0ndnUJWf9+vWL/xV9mFAP/U0Nn0epSKavzf79+83zHn/605+kRYsW8RPpf+tzOPo8CDVKn38qPV8ZoL342mbrI9mSq1SuNRPaEKAjXsXYGmj94qXbPsUO/alQZ3H0oQ6O9AvoDhy6VZ3OwuhsWmw9tH5D0a3rNMTFjmRroKlf+mrUsmVLs5ND4hr1M2fOmP/Xz5lbb72VGqWPP6We9TmCZs2amSUciTNauqOQ/jBatWpVapSSZPoaxZam5eXlSYkSJeIn0tnod999l8+j9NGn3POVAdqL7z+2PlgDnXJ5aBg2AZ1l1gE8bdo06dWrl7k8duHwt0qvvvqq2ZpO93fWh2oSl9LonrazZs2SPXv2mDWDeuhs6IoVK8wuHNQv/bXSp9T1m37ioTNp+lsBnfXUB3CoUfrrcK0zaH1094Cf/exnX9sjXdc9f+c73zE73FCjYGuku9fcd9998s9//lMaN24cvxhdsqaBWn/zRo2CrdGVAdqL7z+2PoK942DPzgx0sP6enF0fVJs9e7bMnz/fzNTojICurdVfWRcvXtyTc9BJcgHduumuu+6SH/3oR1fteKJLODQg6z7Q+pCNzqbpbwz0V9T6jeapp54ynVI//0dX4hpo3cObGvlfgyvPqLsI6Hac+rVMA5quV9dArUsG7rzzTmoUcIn0hU+6peCxY8fM17rKlSuLvvhGH8zV3/DoNoR8HgVbpCsDtBdf21LpI9i7Du7sBOjg7D07sz5AqPvb6oshdCZHN1LXpRsapjnSKzBx4kQTgJMdGpBfe+01s9uGzk7rzgIVKlSQIUOGyIABA+J/hfqlt0bJek8M0PpxauRNF7naAAAGkUlEQVR/DZKdUX+Tpsuh9LcGuq2dvmRId+egRuGoj759UHdG0WU1+t+6Pla/Bj7wwAPUKAQlujJAe/V5Y/v6GIJbD+QSCNCBsHNSBBBAAAEEEEAAgagKEKCjWjmuGwEEEEAAAQQQQCAQAQJ0IOycFAEEEEAAAQQQQCCqAgToqFaO60YAAQQQQAABBBAIRIAAHQg7J0UAAQQKJ6APBetDQmPGjCncX0yx9dq1a0X3zP7ss8++8QHkdF9DipdKMwQQQCBwAQJ04CXgAhBAAAG7gG4fplsj3njjjfbG36LFuXPn5IsvvpCbbropvmf5ld0QoL8FLH8FAQQyUoAAnZFl5aYQQAAB7wUI0N6b0iMCCERTgAAdzbpx1QggEFIB3WNa9wbX17fri3T+8pe/SLVq1eTFF180r3k/deqUefnOuHHj5KGHHorfxR//+Efzlsrt27ebl1LofuH6dtHYkoorw+vvf/97GT9+vHzyySdSqlQpefLJJ2XChAnx1yzrdcyZM0eWLl1qXnTxgx/8wOw/rudIdly5hEOvU/cvX7lypXkhk+41P3369LQuIwlpSbksBBBA4CoBAjSDAgEEEPBQQINruXLl5L333pMLFy6YVxx37dpVtm3bZgKovsFt1apV8vzzz5tXuj/88MOyZcsWueeee+S5554zoVn/X99YqS+rSBag33nnHenYsaNZD925c2f59NNPTXt9g9+yZcvM3eh1lC5d2ryYRP9cA72+7e+jjz6SFi1aXHXHVwbo1q1by4EDB0wI/973vmcC/YcffiijR49O2zpsD8tAVwgggEBaBQjQaeWlcwQQyDYBDa4ahPWtenrs3r1batasad522KhRoziHvqly//79osFV/1tfC/+Pf/wj/vFf/vKXZgY4WYBu0qSJVKlSRd5+++14ew3l+sp4ncHWt/jpdQwaNMiE9tihs9DDhg1LOgudGKBjr6Bfs2aNPPjgg+avHz16VG699VYT/NP1IGO2jRXuFwEEoitAgI5u7bhyBBAIoYAG15kzZ5oZYT3eeustM0t8ww03fO1qz58/L2XKlDHBVF+J3KpVq3jo1oY6C92wYcOkAfq73/2ueYXys88+G+9Tl1xof2+88YY88cQTJkDPmjVL+vTpE2+jy0B69uxpZpGvPBID9MaNG+Xxxx83y010eUjsuPPOO829EKBDOPC4JAQQ8FWAAO0rNydDAIFMF9DgumDBArNWWI8333xTunTpIps2bTJLIRKPokWLmvXRGpQfeOCBr80WXytA624ckydPNjPMsUN30ChbtqxZwvHYY4+ZAJ14HdruWg8BJgvQuoREQ3nsqF+/vjz66KME6EwfxNwfAghYBQjQViIaIIAAAqkLXBlcde2zPjS4evVqs945duiDhkWKFDEPAmrY3rFjh6xbty7+8dmzZ0vfvn2TzkDrmuZbbrnla0s49GG/Dh06fG0Jx7cN0BcvXpQaNWp87ZpPnjwplSpVMktAmIFOfTzQEgEEMlOAAJ2ZdeWuEEAgIIFkM7+PPPKIWZIxY8YME6aXL18uw4cPl1//+tdmSYXupNGgQQMZOnSo/OQnPzEh+Kc//ano3s/79u0za48TZ491WYgu0xg7dqz5t66f1iUj2oeuhdbDZQZaz6VLNXTd9ty5c+Xmm28266b1wcgXXniBAB3Q2OK0CCAQHgECdHhqwZUggEAGCCQLrrm5uWZrO13OoUstdJu6wYMHS+/eveN3rMFXQ6ruqHH77bfHd9k4fPiwVKhQ4arlF0uWLJFJkybJzp07zctPdKcPDdS6vMOLAJ2Xl2dmm/U8BQUFZi21LvNo06YNAToDxim3gAACbgIEaDc//jYCCCDgLKAzvcWKFTNroWPH4sWLzWz0l19+aT7GgQACCCAQHgECdHhqwZUggECWCugyCZ3tff31180yDN36Tmd8mzVrJq+99lqWqnDbCCCAQHgFCNDhrQ1XhgACWSJw6dIl8zChhuVDhw5JTk6OebNg4pKMLKHgNhFAAIFICBCgI1EmLhIBBBBAAAEEEEAgLAIE6LBUgutAAAEEEEAAAQQQiIQAAToSZeIiEUAAAQQQQAABBMIiQIAOSyW4DgQQQAABBBBAAIFICBCgI1EmLhIBBBBAAAEEEEAgLAIE6LBUgutAAAEEEEAAAQQQiIQAAToSZeIiEUAAAQQQQAABBMIicF1YLoTrQAABBBBAAAEEEEAgCgIE6ChUiWtEAAEEEEAAAQQQCI3A/wdr7SF8zdicIgAAAABJRU5ErkJggg==\" width=\"654.5454403585642\">"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# plot the probe number against all regins\n",
    "pb_num_dict = {_k:len(_v.kept_probes) for _k,_v in filtered_pb_dict.items()}\n",
    "\n",
    "pb_nums = np.array(list(pb_num_dict.values()))\n",
    "plt.figure(figsize=(6,3), dpi=100)\n",
    "plt.plot(pb_nums)\n",
    "#plt.plot(np.ones(len(pb_nums))*20)\n",
    "plt.ylabel(f\"num of probes\", labelpad=1)\n",
    "plt.xlabel(f\"region id\", labelpad=1)\n",
    "plt.ylim([0, num_probes_kept*1.1])\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 65,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "968"
      ]
     },
     "execution_count": 65,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(pb_num_dict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 66,
   "metadata": {},
   "outputs": [],
   "source": [
    "# keep regions that have probes\n",
    "\n",
    "filtered_reg_names = {}\n",
    "for _reg, _pbde in filtered_pb_dict.items():\n",
    "    if len(_pbde.kept_probes) > 0:\n",
    "        filtered_reg_names[_reg] = list(_pbde.kept_probes.values())[0]['reg_name']\n",
    "    \n",
    "        "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 68,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{251: '4:105899094-105952080_strand_+_reg_251',\n",
       " 252: '4:114589240-114676242_strand_+_reg_252',\n",
       " 253: '4:115037700-115112192_strand_+_reg_253',\n",
       " 255: '4:115837369-115913327_strand_+_reg_255',\n",
       " 256: '4:116628906-116683403_strand_+_reg_256',\n",
       " 257: '4:117618081-117687368_strand_+_reg_257',\n",
       " 258: '4:123199825-123264995_strand_+_reg_258',\n",
       " 259: '4:123961238-124063077_strand_+_reg_259',\n",
       " 260: '4:124824301-124866678_strand_+_reg_260',\n",
       " 261: '4:126207621-126263017_strand_+_reg_261',\n",
       " 262: '4:126995025-127049608_strand_+_reg_262',\n",
       " 263: '4:127493306-127563000_strand_+_reg_263',\n",
       " 264: '4:127666510-127718932_strand_+_reg_264',\n",
       " 265: '4:127800701-127872488_strand_+_reg_265',\n",
       " 266: '4:129485757-129532804_strand_+_reg_266',\n",
       " 267: '4:130236140-130307752_strand_+_reg_267',\n",
       " 268: '4:130509810-130579025_strand_+_reg_268',\n",
       " 269: '4:131733908-131844723_strand_+_reg_269',\n",
       " 270: '4:132792079-132834284_strand_+_reg_270',\n",
       " 271: '4:135807186-135873467_strand_+_reg_271',\n",
       " 272: '4:137277244-137324456_strand_+_reg_272',\n",
       " 273: '4:148835951-148933937_strand_+_reg_273',\n",
       " 274: '4:155759707-155802098_strand_+_reg_274',\n",
       " 275: '5:3994952-4090244_strand_+_reg_275',\n",
       " 277: '5:4937946-5030393_strand_+_reg_277',\n",
       " 278: '5:5216972-5313712_strand_+_reg_278',\n",
       " 279: '5:15947115-16026399_strand_+_reg_279',\n",
       " 280: '5:19894500-19961425_strand_+_reg_280',\n",
       " 281: '5:21457453-21528145_strand_+_reg_281',\n",
       " 282: '5:24770084-24831106_strand_+_reg_282',\n",
       " 283: '5:25594458-25634378_strand_+_reg_283',\n",
       " 284: '5:27049822-27109916_strand_+_reg_284',\n",
       " 285: '5:30381241-30452747_strand_+_reg_285',\n",
       " 286: '5:33193371-33256343_strand_+_reg_286',\n",
       " 287: '5:38342152-38414629_strand_+_reg_287',\n",
       " 288: '5:38506393-38564776_strand_+_reg_288',\n",
       " 289: '5:38850470-38910029_strand_+_reg_289',\n",
       " 290: '5:39106286-39168083_strand_+_reg_290',\n",
       " 291: '5:43318583-43370939_strand_+_reg_291',\n",
       " 292: '5:44189271-44278791_strand_+_reg_292',\n",
       " 293: '5:44636554-44752497_strand_+_reg_293',\n",
       " 295: '5:44853523-44919113_strand_+_reg_295',\n",
       " 296: '5:49951907-50059873_strand_+_reg_296',\n",
       " 297: '5:52779669-52833675_strand_+_reg_297',\n",
       " 298: '5:58013151-58138957_strand_+_reg_298',\n",
       " 299: '5:62628998-62700814_strand_+_reg_299',\n",
       " 300: '5:63805226-63916930_strand_+_reg_300',\n",
       " 301: '5:64740899-64823898_strand_+_reg_301',\n",
       " 302: '5:66052841-66098951_strand_+_reg_302',\n",
       " 303: '5:67330974-67411928_strand_+_reg_303',\n",
       " 304: '5:71592818-71661574_strand_+_reg_304',\n",
       " 305: '5:72807519-72901992_strand_+_reg_305',\n",
       " 306: '5:74601244-74658537_strand_+_reg_306',\n",
       " 307: '5:75625002-75694903_strand_+_reg_307',\n",
       " 308: '5:77344237-77404169_strand_+_reg_308',\n",
       " 309: '5:88554694-88611033_strand_+_reg_309',\n",
       " 310: '5:91538047-91648373_strand_+_reg_310',\n",
       " 311: '5:92331917-92408829_strand_+_reg_311',\n",
       " 312: '5:104291330-104355424_strand_+_reg_312',\n",
       " 313: '5:106763128-106836267_strand_+_reg_313',\n",
       " 314: '5:108667189-108745317_strand_+_reg_314',\n",
       " 315: '5:116975881-117047536_strand_+_reg_315',\n",
       " 316: '5:119037555-119110105_strand_+_reg_316',\n",
       " 317: '5:126228032-126292172_strand_+_reg_317',\n",
       " 318: '5:128181775-128254686_strand_+_reg_318',\n",
       " 319: '5:129741425-129789547_strand_+_reg_319',\n",
       " 320: '5:129925623-129977156_strand_+_reg_320',\n",
       " 321: '5:132237594-132324711_strand_+_reg_321',\n",
       " 322: '5:133953020-134030324_strand_+_reg_322',\n",
       " 324: '5:135005402-135064026_strand_+_reg_324',\n",
       " 325: '5:138757956-138807859_strand_+_reg_325',\n",
       " 326: '5:141028058-141104622_strand_+_reg_326',\n",
       " 327: '5:141735482-141804869_strand_+_reg_327',\n",
       " 328: '5:142013394-142093792_strand_+_reg_328',\n",
       " 329: '5:143460529-143531173_strand_+_reg_329',\n",
       " 330: '5:144532912-144614769_strand_+_reg_330',\n",
       " 332: '5:147010488-147081113_strand_+_reg_332',\n",
       " 333: '5:148476071-148552936_strand_+_reg_333',\n",
       " 334: '5:150087399-150179014_strand_+_reg_334',\n",
       " 335: '6:4913611-5062849_strand_+_reg_335',\n",
       " 336: '6:5816895-5945849_strand_+_reg_336',\n",
       " 337: '6:6142617-6218463_strand_+_reg_337',\n",
       " 338: '6:17092402-17181916_strand_+_reg_338',\n",
       " 339: '6:17574012-17640341_strand_+_reg_339',\n",
       " 340: '6:18398404-18461976_strand_+_reg_340',\n",
       " 341: '6:23436147-23521005_strand_+_reg_341',\n",
       " 342: '6:24510973-24581391_strand_+_reg_342',\n",
       " 343: '6:27875819-27995930_strand_+_reg_343',\n",
       " 344: '6:29188389-29226427_strand_+_reg_344',\n",
       " 345: '6:30517033-30590432_strand_+_reg_345',\n",
       " 347: '6:34471419-34525893_strand_+_reg_347',\n",
       " 348: '6:34856828-34912427_strand_+_reg_348',\n",
       " 349: '6:41067178-41147415_strand_+_reg_349',\n",
       " 350: '6:41494636-41570746_strand_+_reg_350',\n",
       " 351: '6:42329795-42373704_strand_+_reg_351',\n",
       " 352: '6:48379712-48444906_strand_+_reg_352',\n",
       " 353: '6:49807253-49847041_strand_+_reg_353',\n",
       " 354: '6:51158260-51195983_strand_+_reg_354',\n",
       " 355: '6:53570205-53695638_strand_+_reg_355',\n",
       " 356: '6:54688853-54746443_strand_+_reg_356',\n",
       " 357: '6:55781236-55825137_strand_+_reg_357',\n",
       " 358: '6:57733581-57826248_strand_+_reg_358',\n",
       " 359: '6:72814718-72899657_strand_+_reg_359',\n",
       " 360: '6:77403708-77486471_strand_+_reg_360',\n",
       " 361: '6:81772875-81833647_strand_+_reg_361',\n",
       " 362: '6:81965780-82001014_strand_+_reg_362',\n",
       " 363: '6:82986732-83034917_strand_+_reg_363',\n",
       " 364: '6:83174995-83226237_strand_+_reg_364',\n",
       " 365: '6:83847140-83910764_strand_+_reg_365',\n",
       " 366: '6:84109647-84145755_strand_+_reg_366',\n",
       " 367: '6:84552937-84600580_strand_+_reg_367',\n",
       " 368: '6:90393398-90445394_strand_+_reg_368',\n",
       " 369: '6:97827648-97913932_strand_+_reg_369',\n",
       " 370: '6:98198965-98298774_strand_+_reg_370',\n",
       " 371: '6:98920069-99030726_strand_+_reg_371',\n",
       " 372: '6:99319695-99393737_strand_+_reg_372',\n",
       " 373: '6:100901891-100949915_strand_+_reg_373',\n",
       " 374: '6:106370677-106451259_strand_+_reg_374',\n",
       " 375: '6:111471365-111546119_strand_+_reg_375',\n",
       " 376: '6:113251068-113297839_strand_+_reg_376',\n",
       " 377: '6:115452622-115533851_strand_+_reg_377',\n",
       " 378: '6:116336040-116434204_strand_+_reg_378',\n",
       " 379: '6:118583580-118659770_strand_+_reg_379',\n",
       " 380: '6:121356679-121478570_strand_+_reg_380',\n",
       " 381: '6:127069741-127152836_strand_+_reg_381',\n",
       " 382: '6:128391401-128443169_strand_+_reg_382',\n",
       " 383: '6:131356549-131444281_strand_+_reg_383',\n",
       " 384: '6:134686900-134773941_strand_+_reg_384',\n",
       " 385: '6:137078345-137173906_strand_+_reg_385',\n",
       " 386: '6:142477481-142535279_strand_+_reg_386',\n",
       " 387: '6:144585170-144652901_strand_+_reg_387',\n",
       " 388: '6:144980272-145047109_strand_+_reg_388',\n",
       " 389: '6:145265832-145317520_strand_+_reg_389',\n",
       " 390: '6:145723149-145813383_strand_+_reg_390',\n",
       " 391: '6:146320927-146368669_strand_+_reg_391',\n",
       " 392: '7:4468044-4523679_strand_+_reg_392',\n",
       " 393: '7:25614559-25661968_strand_+_reg_393',\n",
       " 394: '7:28165434-28224615_strand_+_reg_394',\n",
       " 395: '7:28549913-28611972_strand_+_reg_395',\n",
       " 396: '7:30246410-30311465_strand_+_reg_396',\n",
       " 397: '7:30885416-30938012_strand_+_reg_397',\n",
       " 398: '7:35676730-35786268_strand_+_reg_398',\n",
       " 399: '7:36035635-36095581_strand_+_reg_399',\n",
       " 400: '7:36903879-36964287_strand_+_reg_400',\n",
       " 401: '7:43533744-43582104_strand_+_reg_401',\n",
       " 402: '7:47046107-47127036_strand_+_reg_402',\n",
       " 403: '7:47046107-47121005_strand_+_reg_403',\n",
       " 404: '7:50493534-50595678_strand_+_reg_404',\n",
       " 405: '7:64285872-64371201_strand_+_reg_405',\n",
       " 406: '7:65860622-65945471_strand_+_reg_406',\n",
       " 407: '7:67813905-67893031_strand_+_reg_407',\n",
       " 408: '7:70316208-70366368_strand_+_reg_408',\n",
       " 409: '7:81251500-81332036_strand_+_reg_409',\n",
       " 410: '7:82791317-82863443_strand_+_reg_410',\n",
       " 411: '7:83034578-83105834_strand_+_reg_411',\n",
       " 412: '7:83629439-83686053_strand_+_reg_412',\n",
       " 413: '7:83790534-83849178_strand_+_reg_413',\n",
       " 414: '7:84235734-84311961_strand_+_reg_414',\n",
       " 415: '7:89259710-89342531_strand_+_reg_415',\n",
       " 416: '7:97735161-97815491_strand_+_reg_416',\n",
       " 417: '7:98125903-98181642_strand_+_reg_417',\n",
       " 418: '7:101890453-101928530_strand_+_reg_418',\n",
       " 419: '7:107651264-107719048_strand_+_reg_419',\n",
       " 420: '7:109914369-109963604_strand_+_reg_420',\n",
       " 421: '7:112752685-112894620_strand_+_reg_421',\n",
       " 422: '7:114281439-114333757_strand_+_reg_422',\n",
       " 423: '7:115477765-115548476_strand_+_reg_423',\n",
       " 424: '7:115636108-115717089_strand_+_reg_424',\n",
       " 425: '7:118113230-118170731_strand_+_reg_425',\n",
       " 426: '7:119716960-119755414_strand_+_reg_426',\n",
       " 427: '7:121203744-121312825_strand_+_reg_427',\n",
       " 428: '7:126238527-126290938_strand_+_reg_428',\n",
       " 429: '7:127071384-127140312_strand_+_reg_429',\n",
       " 430: '7:127626855-127710283_strand_+_reg_430',\n",
       " 431: '7:130205243-130269178_strand_+_reg_431',\n",
       " 432: '7:132216605-132295072_strand_+_reg_432',\n",
       " 433: '7:134653339-134690281_strand_+_reg_433',\n",
       " 434: '7:136036278-136105849_strand_+_reg_434',\n",
       " 435: '7:138166035-138259068_strand_+_reg_435',\n",
       " 436: '7:142323852-142438691_strand_+_reg_436',\n",
       " 437: '8:8590019-8660327_strand_+_reg_437',\n",
       " 438: '8:13909259-13976263_strand_+_reg_438',\n",
       " 439: '8:15539842-15619200_strand_+_reg_439',\n",
       " 440: '8:15984403-16043579_strand_+_reg_440',\n",
       " 441: '8:17444670-17529064_strand_+_reg_441',\n",
       " 442: '8:18906539-18964569_strand_+_reg_442',\n",
       " 444: '8:25338122-25379634_strand_+_reg_444',\n",
       " 445: '8:26676886-26724279_strand_+_reg_445',\n",
       " 446: '8:34805877-34903394_strand_+_reg_446',\n",
       " 447: '8:35149287-35254368_strand_+_reg_447',\n",
       " 448: '8:35358926-35444058_strand_+_reg_448',\n",
       " 449: '8:36400955-36469588_strand_+_reg_449',\n",
       " 450: '8:36619739-36725384_strand_+_reg_450',\n",
       " 451: '8:40562713-40638441_strand_+_reg_451',\n",
       " 452: '8:41328066-41394762_strand_+_reg_452',\n",
       " 453: '8:47670301-47717341_strand_+_reg_453',\n",
       " 454: '8:47821006-47913730_strand_+_reg_454',\n",
       " 455: '8:48176246-48232764_strand_+_reg_455',\n",
       " 456: '8:48470073-48578850_strand_+_reg_456',\n",
       " 457: '8:61239539-61344852_strand_+_reg_457',\n",
       " 458: '8:67928931-68005568_strand_+_reg_458',\n",
       " 459: '8:68161685-68224740_strand_+_reg_459',\n",
       " 460: '8:68504342-68569036_strand_+_reg_460',\n",
       " 461: '8:71286208-71369935_strand_+_reg_461',\n",
       " 462: '8:72632943-72708838_strand_+_reg_462',\n",
       " 463: '8:75033814-75099992_strand_+_reg_463',\n",
       " 464: '8:76990149-77104743_strand_+_reg_464',\n",
       " 465: '8:78008249-78074137_strand_+_reg_465',\n",
       " 466: '8:79321964-79402981_strand_+_reg_466',\n",
       " 467: '8:81331631-81410269_strand_+_reg_467',\n",
       " 468: '8:83392102-83461711_strand_+_reg_468',\n",
       " 469: '8:84173157-84214478_strand_+_reg_469',\n",
       " 470: '8:88269593-88342980_strand_+_reg_470',\n",
       " 471: '8:90263317-90334986_strand_+_reg_471',\n",
       " 472: '8:90481007-90553012_strand_+_reg_472',\n",
       " 473: '8:90827081-90904757_strand_+_reg_473',\n",
       " 474: '8:92109780-92144791_strand_+_reg_474',\n",
       " 475: '8:95146834-95186362_strand_+_reg_475',\n",
       " 476: '8:103822726-103892126_strand_+_reg_476',\n",
       " 477: '8:104393397-104447891_strand_+_reg_477',\n",
       " 478: '8:105422888-105494815_strand_+_reg_478',\n",
       " 479: '8:107434145-107533024_strand_+_reg_479',\n",
       " 480: '8:109893221-109994503_strand_+_reg_480',\n",
       " 481: '8:110915709-110996339_strand_+_reg_481',\n",
       " 482: '8:112606029-112692110_strand_+_reg_482',\n",
       " 483: '8:114440631-114516127_strand_+_reg_483',\n",
       " 484: '8:114888227-115019038_strand_+_reg_484',\n",
       " 485: '8:115121077-115190969_strand_+_reg_485',\n",
       " 486: '8:116572552-116632019_strand_+_reg_486',\n",
       " 487: '8:118085022-118172697_strand_+_reg_487',\n",
       " 488: '8:119110924-119171282_strand_+_reg_488',\n",
       " 489: '8:119775828-119840110_strand_+_reg_489',\n",
       " 491: '9:8889764-8938793_strand_+_reg_491',\n",
       " 492: '9:9083125-9155126_strand_+_reg_492',\n",
       " 493: '9:14502860-14595101_strand_+_reg_493',\n",
       " 494: '9:15483272-15563906_strand_+_reg_494',\n",
       " 495: '9:22547286-22665336_strand_+_reg_495',\n",
       " 496: '9:22783499-22864592_strand_+_reg_496',\n",
       " 497: '9:24398454-24493239_strand_+_reg_497',\n",
       " 498: '9:32479082-32541627_strand_+_reg_498',\n",
       " 499: '9:32691049-32769587_strand_+_reg_499',\n",
       " 500: '9:37396529-37448756_strand_+_reg_500',\n",
       " 501: '9:46291898-46364450_strand_+_reg_501',\n",
       " 502: '9:48007145-48083481_strand_+_reg_502',\n",
       " 503: '9:48176289-48277700_strand_+_reg_503',\n",
       " 504: '9:50054571-50160541_strand_+_reg_504',\n",
       " 505: '9:52357436-52446749_strand_+_reg_505',\n",
       " 506: '9:53893431-53958595_strand_+_reg_506',\n",
       " 507: '9:54298886-54354997_strand_+_reg_507',\n",
       " 508: '9:54508565-54565386_strand_+_reg_508',\n",
       " 509: '9:54645501-54702064_strand_+_reg_509',\n",
       " 510: '9:57240601-57299305_strand_+_reg_510',\n",
       " 511: '9:63600882-63666718_strand_+_reg_511',\n",
       " 512: '9:64174332-64252851_strand_+_reg_512',\n",
       " 513: '9:67385710-67451674_strand_+_reg_513',\n",
       " 514: '9:67673420-67763632_strand_+_reg_514',\n",
       " 515: '9:69074838-69145422_strand_+_reg_515',\n",
       " 516: '9:71524978-71592718_strand_+_reg_516',\n",
       " 517: '9:71901483-71970383_strand_+_reg_517',\n",
       " 518: '9:73144502-73202707_strand_+_reg_518',\n",
       " 519: '9:77698777-77761284_strand_+_reg_519',\n",
       " 520: '9:78111826-78179455_strand_+_reg_520',\n",
       " 521: '9:80159682-80266480_strand_+_reg_521',\n",
       " 523: '9:83294219-83369953_strand_+_reg_523',\n",
       " 524: '9:86573601-86647179_strand_+_reg_524',\n",
       " 525: '9:89246295-89303992_strand_+_reg_525',\n",
       " 526: '9:89831577-89911977_strand_+_reg_526',\n",
       " 527: '9:90004552-90087294_strand_+_reg_527',\n",
       " 528: '9:90175735-90277232_strand_+_reg_528',\n",
       " 529: '9:96414547-96482062_strand_+_reg_529',\n",
       " 530: '9:97889864-97985879_strand_+_reg_530',\n",
       " 531: '9:99046362-99131260_strand_+_reg_531',\n",
       " 532: '9:99753284-99818353_strand_+_reg_532',\n",
       " 533: '9:102482333-102523822_strand_+_reg_533',\n",
       " 534: '9:104888780-104988138_strand_+_reg_534',\n",
       " 535: '9:105265008-105329057_strand_+_reg_535',\n",
       " 536: '9:105834542-105920743_strand_+_reg_536',\n",
       " 537: '9:106914617-106985928_strand_+_reg_537',\n",
       " 538: '9:108390917-108463922_strand_+_reg_538',\n",
       " 539: '9:109517022-109573625_strand_+_reg_539',\n",
       " 540: '9:110403831-110459627_strand_+_reg_540',\n",
       " 541: '9:110879347-110944746_strand_+_reg_541',\n",
       " 542: '9:112421698-112487152_strand_+_reg_542',\n",
       " 543: '9:113760660-113813213_strand_+_reg_543',\n",
       " 544: '9:119399249-119449068_strand_+_reg_544',\n",
       " 545: '9:123421255-123491622_strand_+_reg_545',\n",
       " 546: '10:8166017-8250797_strand_+_reg_546',\n",
       " 547: '10:9791406-9903185_strand_+_reg_547',\n",
       " 548: '10:12830738-12885969_strand_+_reg_548',\n",
       " 549: '10:13554012-13611581_strand_+_reg_549',\n",
       " 550: '10:19536426-19607009_strand_+_reg_550',\n",
       " 551: '10:21881312-21991778_strand_+_reg_551',\n",
       " 552: '10:23181367-23277580_strand_+_reg_552',\n",
       " 553: '10:24951345-25011868_strand_+_reg_553',\n",
       " 554: '10:25400587-25476543_strand_+_reg_554',\n",
       " 555: '10:26257092-26325268_strand_+_reg_555',\n",
       " 556: '10:28073265-28199418_strand_+_reg_556',\n",
       " 557: '10:34121442-34208522_strand_+_reg_557',\n",
       " 558: '10:38962790-39040119_strand_+_reg_558',\n",
       " 559: '10:40058995-40139830_strand_+_reg_559',\n",
       " 560: '10:41176005-41259634_strand_+_reg_560',\n",
       " 561: '10:42547711-42602574_strand_+_reg_561',\n",
       " 562: '10:43266076-43360048_strand_+_reg_562',\n",
       " 563: '10:44282271-44365306_strand_+_reg_563',\n",
       " 564: '10:44615388-44713060_strand_+_reg_564',\n",
       " 565: '10:49750898-49817819_strand_+_reg_565',\n",
       " 566: '10:52692507-52771120_strand_+_reg_566',\n",
       " 567: '10:52891887-52963599_strand_+_reg_567',\n",
       " 568: '10:58590997-58683875_strand_+_reg_568',\n",
       " 569: '10:61307332-61385454_strand_+_reg_569',\n",
       " 570: '10:61524277-61616208_strand_+_reg_570',\n",
       " 571: '10:62836832-62897538_strand_+_reg_571',\n",
       " 572: '10:63133174-63200019_strand_+_reg_572',\n",
       " 573: '10:64029707-64098108_strand_+_reg_573',\n",
       " 574: '10:66869118-66941818_strand_+_reg_574',\n",
       " 575: '10:67841028-67914443_strand_+_reg_575',\n",
       " 576: '10:69229620-69270356_strand_+_reg_576',\n",
       " 577: '10:71281190-71349781_strand_+_reg_577',\n",
       " 579: '10:75862641-75913051_strand_+_reg_579',\n",
       " 580: '10:79702586-79748477_strand_+_reg_580',\n",
       " 581: '10:82628910-82692199_strand_+_reg_581',\n",
       " 582: '10:83607495-83670925_strand_+_reg_582',\n",
       " 583: '10:83837002-83953108_strand_+_reg_583',\n",
       " 584: '10:84077311-84175022_strand_+_reg_584',\n",
       " 585: '10:84921289-85033749_strand_+_reg_585',\n",
       " 586: '10:86022255-86086646_strand_+_reg_586',\n",
       " 587: '10:87475669-87514242_strand_+_reg_587',\n",
       " 588: '10:87862034-87951886_strand_+_reg_588',\n",
       " 589: '10:91171367-91217531_strand_+_reg_589',\n",
       " 590: '10:92215234-92259181_strand_+_reg_590',\n",
       " 591: '10:93638004-93714413_strand_+_reg_591',\n",
       " 592: '10:94036937-94108212_strand_+_reg_592',\n",
       " 593: '10:94220759-94276806_strand_+_reg_593',\n",
       " 594: '10:96115549-96224510_strand_+_reg_594',\n",
       " 595: '10:98552540-98672759_strand_+_reg_595',\n",
       " 596: '10:98911845-98969798_strand_+_reg_596',\n",
       " 597: '10:99141730-99215126_strand_+_reg_597',\n",
       " 598: '10:105474481-105550782_strand_+_reg_598',\n",
       " 599: '10:106575266-106742470_strand_+_reg_599',\n",
       " 600: '10:110917563-111009521_strand_+_reg_600',\n",
       " 601: '10:111525577-111633055_strand_+_reg_601',\n",
       " 602: '10:114478817-114628882_strand_+_reg_602',\n",
       " 603: '10:115360947-115403646_strand_+_reg_603',\n",
       " 604: '10:116990815-117077224_strand_+_reg_604',\n",
       " 605: '10:117659609-117705189_strand_+_reg_605',\n",
       " 606: '10:118733724-118815013_strand_+_reg_606',\n",
       " 607: '10:123325185-123388689_strand_+_reg_607',\n",
       " 608: '10:125248103-125399970_strand_+_reg_608',\n",
       " 609: '10:126447909-126551128_strand_+_reg_609',\n",
       " 610: '10:128407759-128476427_strand_+_reg_610',\n",
       " 611: '11:4944021-5006201_strand_+_reg_611',\n",
       " 612: '11:5336596-5447936_strand_+_reg_612',\n",
       " 613: '11:6558052-6610060_strand_+_reg_613',\n",
       " 614: '11:11160943-11253518_strand_+_reg_614',\n",
       " 615: '11:20425884-20510944_strand_+_reg_615',\n",
       " 616: '11:21983178-22041941_strand_+_reg_616',\n",
       " 617: '11:22621290-22687776_strand_+_reg_617',\n",
       " 618: '11:25838906-25913888_strand_+_reg_618',\n",
       " 619: '11:26050799-26139369_strand_+_reg_619',\n",
       " 620: '11:26946039-27007404_strand_+_reg_620',\n",
       " 621: '11:28645032-28711785_strand_+_reg_621',\n",
       " 622: '11:34044208-34109026_strand_+_reg_622',\n",
       " 623: '11:35828462-35937732_strand_+_reg_623',\n",
       " 624: '11:36130151-36214786_strand_+_reg_624',\n",
       " 625: '11:38049705-38102695_strand_+_reg_625',\n",
       " 626: '11:40754297-40797367_strand_+_reg_626',\n",
       " 627: '11:43338561-43423182_strand_+_reg_627',\n",
       " 628: '11:43756737-43838613_strand_+_reg_628',\n",
       " 629: '11:44493963-44596487_strand_+_reg_629',\n",
       " 630: '11:46420972-46472278_strand_+_reg_630',\n",
       " 631: '11:50659370-50717733_strand_+_reg_631',\n",
       " 632: '11:52624034-52707755_strand_+_reg_632',\n",
       " 633: '11:54301823-54370331_strand_+_reg_633',\n",
       " 634: '11:55130329-55205754_strand_+_reg_634',\n",
       " 635: '11:55376303-55463852_strand_+_reg_635',\n",
       " 636: '11:57657256-57770935_strand_+_reg_636',\n",
       " 637: '11:58007673-58081281_strand_+_reg_637',\n",
       " 638: '11:58291211-58327455_strand_+_reg_638',\n",
       " 639: '11:62284006-62352698_strand_+_reg_639',\n",
       " 640: '11:63881715-63935692_strand_+_reg_640',\n",
       " 641: '11:67980926-68054725_strand_+_reg_641',\n",
       " 642: '11:69581649-69624548_strand_+_reg_642',\n",
       " 643: '11:70010422-70060516_strand_+_reg_643',\n",
       " 644: '11:70556947-70618824_strand_+_reg_644',\n",
       " 645: '11:70860197-70955859_strand_+_reg_645',\n",
       " 646: '11:71748129-71802987_strand_+_reg_646',\n",
       " 647: '11:72436391-72493071_strand_+_reg_647',\n",
       " 648: '11:73271570-73325274_strand_+_reg_648',\n",
       " 649: '11:78098903-78180841_strand_+_reg_649',\n",
       " 650: '11:78261740-78324241_strand_+_reg_650',\n",
       " 651: '11:84022044-84084532_strand_+_reg_651',\n",
       " 652: '11:88810839-88868107_strand_+_reg_652',\n",
       " 653: '11:89586560-89655737_strand_+_reg_653',\n",
       " 654: '11:95665026-95715325_strand_+_reg_654',\n",
       " 655: '11:97109256-97171020_strand_+_reg_655',\n",
       " 656: '11:100709943-100764691_strand_+_reg_656',\n",
       " 657: '11:101999867-102041606_strand_+_reg_657',\n",
       " 658: '11:106785701-106842613_strand_+_reg_658',\n",
       " 659: '11:107015657-107057289_strand_+_reg_659',\n",
       " 660: '11:109457924-109531070_strand_+_reg_660',\n",
       " 661: '11:109676821-109738725_strand_+_reg_661',\n",
       " 662: '11:111501725-111548093_strand_+_reg_662',\n",
       " 663: '11:112788498-112850927_strand_+_reg_663',\n",
       " 664: '11:113651313-113714386_strand_+_reg_664',\n",
       " 665: '11:115381470-115424234_strand_+_reg_665',\n",
       " 666: '11:120969342-121034517_strand_+_reg_666',\n",
       " 667: '11:121465660-121560309_strand_+_reg_667',\n",
       " 668: '12:4669994-4767519_strand_+_reg_668',\n",
       " 669: '12:8649279-8709465_strand_+_reg_669',\n",
       " 670: '12:8906293-8946352_strand_+_reg_670',\n",
       " 671: '12:11312932-11414073_strand_+_reg_671',\n",
       " 672: '12:12755678-12838753_strand_+_reg_672',\n",
       " 673: '12:13069761-13155339_strand_+_reg_673',\n",
       " 674: '12:21414404-21467592_strand_+_reg_674',\n",
       " 675: '12:29527911-29582105_strand_+_reg_675',\n",
       " 676: '12:30217718-30280914_strand_+_reg_676',\n",
       " 677: '12:34420416-34527401_strand_+_reg_677',\n",
       " 678: '12:34993090-35050003_strand_+_reg_678',\n",
       " 679: '12:45004756-45093297_strand_+_reg_679',\n",
       " 680: '12:49359077-49421839_strand_+_reg_680',\n",
       " 681: '12:51689811-51761995_strand_+_reg_681',\n",
       " 682: '12:52731783-52806084_strand_+_reg_682',\n",
       " 683: '12:67084035-67208980_strand_+_reg_683',\n",
       " 684: '12:69789098-69891374_strand_+_reg_684',\n",
       " 685: '12:70857906-70924672_strand_+_reg_685',\n",
       " 687: '12:72169666-72249828_strand_+_reg_687',\n",
       " 688: '12:73583311-73644095_strand_+_reg_688',\n",
       " 689: '12:75487290-75582550_strand_+_reg_689',\n",
       " 690: '12:83524044-83589479_strand_+_reg_690',\n",
       " 691: '12:84151919-84198789_strand_+_reg_691',\n",
       " 692: '12:84501230-84556432_strand_+_reg_692',\n",
       " 693: '12:85147576-85218677_strand_+_reg_693',\n",
       " 694: '12:86672397-86725863_strand_+_reg_694',\n",
       " 695: '12:88398166-88453232_strand_+_reg_695',\n",
       " 696: '12:89835982-89898938_strand_+_reg_696',\n",
       " 697: '12:90064487-90184183_strand_+_reg_697',\n",
       " 698: '12:103952927-104010067_strand_+_reg_698',\n",
       " 699: '12:110193166-110281004_strand_+_reg_699',\n",
       " 700: '12:110488383-110596477_strand_+_reg_700',\n",
       " 701: '12:112273397-112371328_strand_+_reg_701',\n",
       " 702: '12:113265834-113332430_strand_+_reg_702',\n",
       " 703: '12:118826228-118866588_strand_+_reg_703',\n",
       " 704: '13:6145921-6202393_strand_+_reg_704',\n",
       " 705: '13:9643963-9718881_strand_+_reg_705',\n",
       " 706: '13:13784267-13837202_strand_+_reg_706',\n",
       " 707: '13:15511949-15613228_strand_+_reg_707',\n",
       " 708: '13:15969446-16034031_strand_+_reg_708',\n",
       " 709: '13:18296121-18382753_strand_+_reg_709',\n",
       " 710: '13:18790540-18886160_strand_+_reg_710',\n",
       " 711: '13:23690967-23739842_strand_+_reg_711',\n",
       " 712: '13:28582966-28640622_strand_+_reg_712',\n",
       " 713: '13:32778496-32844925_strand_+_reg_713',\n",
       " 714: '13:36247139-36325532_strand_+_reg_714',\n",
       " 715: '13:37810544-37982052_strand_+_reg_715',\n",
       " 716: '13:40911149-40989375_strand_+_reg_716',\n",
       " 717: '13:41292633-41362801_strand_+_reg_717',\n",
       " 718: '13:43729520-43816764_strand_+_reg_718',\n",
       " 719: '13:44008093-44111293_strand_+_reg_719',\n",
       " 720: '13:44198135-44248787_strand_+_reg_720',\n",
       " 721: '13:44459166-44548478_strand_+_reg_721',\n",
       " 722: '13:45064772-45169265_strand_+_reg_722',\n",
       " 723: '13:46801717-46906339_strand_+_reg_723',\n",
       " 724: '13:52004806-52065454_strand_+_reg_724',\n",
       " 725: '13:57236124-57302330_strand_+_reg_725',\n",
       " 726: '13:59788227-59853221_strand_+_reg_726',\n",
       " 727: '13:60034633-60143990_strand_+_reg_727',\n",
       " 728: '13:60434888-60508636_strand_+_reg_728',\n",
       " 729: '13:62887747-62937194_strand_+_reg_729',\n",
       " 730: '13:64427959-64496783_strand_+_reg_730',\n",
       " 731: '13:69312990-69368200_strand_+_reg_731',\n",
       " 732: '13:70852179-70914310_strand_+_reg_732',\n",
       " 733: '13:75713653-75820309_strand_+_reg_733',\n",
       " 734: '13:81123389-81170001_strand_+_reg_734',\n",
       " 735: '13:83089701-83146173_strand_+_reg_735',\n",
       " 736: '13:84011657-84075757_strand_+_reg_736',\n",
       " 737: '13:88889893-89025343_strand_+_reg_737',\n",
       " 738: '13:89573484-89618121_strand_+_reg_738',\n",
       " 739: '13:97151046-97230203_strand_+_reg_739',\n",
       " 740: '13:99342904-99407764_strand_+_reg_740',\n",
       " 741: '13:101198612-101249406_strand_+_reg_741',\n",
       " 742: '13:101440340-101523506_strand_+_reg_742',\n",
       " 743: '13:103252792-103337542_strand_+_reg_743',\n",
       " 744: '13:106678768-106757437_strand_+_reg_744',\n",
       " 745: '13:108046902-108146316_strand_+_reg_745',\n",
       " 746: '13:108254869-108312090_strand_+_reg_746',\n",
       " 747: '13:111373959-111432442_strand_+_reg_747',\n",
       " 748: '13:111731847-111807966_strand_+_reg_748',\n",
       " 749: '13:111919531-111990923_strand_+_reg_749',\n",
       " 750: '13:114065922-114149764_strand_+_reg_750',\n",
       " 1: '1:9600471-9656458_strand_+_reg_1',\n",
       " 2: '1:9749317-9845402_strand_+_reg_2',\n",
       " 3: '1:20888605-20953251_strand_+_reg_3',\n",
       " 4: '1:21475446-21564433_strand_+_reg_4',\n",
       " 5: '1:21674557-21793163_strand_+_reg_5',\n",
       " 6: '1:23245924-23297215_strand_+_reg_6',\n",
       " 7: '1:23918055-23999346_strand_+_reg_7',\n",
       " 8: '1:24208993-24275055_strand_+_reg_8',\n",
       " 9: '1:35485994-35596498_strand_+_reg_9',\n",
       " 10: '1:36042487-36110247_strand_+_reg_10',\n",
       " 11: '1:37043543-37132697_strand_+_reg_11',\n",
       " 12: '1:39053266-39116581_strand_+_reg_12',\n",
       " 13: '1:39720514-39784704_strand_+_reg_13',\n",
       " 14: '1:42219499-42273271_strand_+_reg_14',\n",
       " 15: '1:45847775-45927301_strand_+_reg_15',\n",
       " 16: '1:51081069-51175247_strand_+_reg_16',\n",
       " 17: '1:51570703-51660839_strand_+_reg_17',\n",
       " 18: '1:52630368-52700586_strand_+_reg_18',\n",
       " 19: '1:54779256-54924777_strand_+_reg_19',\n",
       " 20: '1:55183046-55231757_strand_+_reg_20',\n",
       " 21: '1:55419953-55484150_strand_+_reg_21',\n",
       " 22: '1:55599660-55771983_strand_+_reg_22',\n",
       " 23: '1:57195401-57235355_strand_+_reg_23',\n",
       " 24: '1:57774408-57850178_strand_+_reg_24',\n",
       " 25: '1:58710248-58755177_strand_+_reg_25',\n",
       " 26: '1:60406810-60510189_strand_+_reg_26',\n",
       " 27: '1:60828959-60878677_strand_+_reg_27',\n",
       " 28: '1:62890963-62958089_strand_+_reg_28',\n",
       " 29: '1:68028967-68171164_strand_+_reg_29',\n",
       " 30: '1:68682949-68800249_strand_+_reg_30',\n",
       " 31: '1:72404007-72475858_strand_+_reg_31',\n",
       " 32: '1:72583027-72630300_strand_+_reg_32',\n",
       " 33: '1:73522522-73603287_strand_+_reg_33',\n",
       " 34: '1:75423170-75478047_strand_+_reg_34',\n",
       " 35: '1:75854682-75920607_strand_+_reg_35',\n",
       " 36: '1:79813369-79876804_strand_+_reg_36',\n",
       " 37: '1:81222704-81365836_strand_+_reg_37',\n",
       " 38: '1:82422928-82493138_strand_+_reg_38',\n",
       " 39: '1:83313920-83400588_strand_+_reg_39',\n",
       " 40: '1:84168846-84261608_strand_+_reg_40',\n",
       " 41: '1:86959177-87062202_strand_+_reg_41',\n",
       " 42: '1:89121460-89206528_strand_+_reg_42',\n",
       " 43: '1:91412081-91466772_strand_+_reg_43',\n",
       " 44: '1:93189428-93238116_strand_+_reg_44',\n",
       " 45: '1:97604468-97662595_strand_+_reg_45',\n",
       " 46: '1:105961567-106009318_strand_+_reg_46',\n",
       " 47: '1:115727054-115804837_strand_+_reg_47',\n",
       " 48: '1:119437659-119505173_strand_+_reg_48',\n",
       " 49: '1:120161583-120218334_strand_+_reg_49',\n",
       " 50: '1:121430601-121489275_strand_+_reg_50',\n",
       " 51: '1:124674853-124801140_strand_+_reg_51',\n",
       " 52: '1:137888029-137965735_strand_+_reg_52',\n",
       " 53: '1:140354959-140541768_strand_+_reg_53',\n",
       " 54: '1:143861617-143934455_strand_+_reg_54',\n",
       " 55: '1:149846612-149908891_strand_+_reg_55',\n",
       " 56: '1:152397309-152486632_strand_+_reg_56',\n",
       " 57: '1:155665230-155748254_strand_+_reg_57',\n",
       " 58: '1:161214625-161297711_strand_+_reg_58',\n",
       " 59: '1:162012959-162104685_strand_+_reg_59',\n",
       " 60: '1:162655058-162776885_strand_+_reg_60',\n",
       " 61: '1:167034296-167100658_strand_+_reg_61',\n",
       " 62: '1:167197268-167284083_strand_+_reg_62',\n",
       " 63: '1:169792098-169928411_strand_+_reg_63',\n",
       " 64: '1:171429944-171499765_strand_+_reg_64',\n",
       " 65: '1:171805726-171877501_strand_+_reg_65',\n",
       " 66: '1:173307246-173374219_strand_+_reg_66',\n",
       " 67: '1:178341436-178410984_strand_+_reg_67',\n",
       " 68: '1:179031899-179125551_strand_+_reg_68',\n",
       " 69: '1:182429520-182519678_strand_+_reg_69',\n",
       " 70: '1:186617359-186706395_strand_+_reg_70',\n",
       " 71: '1:187635279-187704850_strand_+_reg_71',\n",
       " 72: '1:189007529-189076052_strand_+_reg_72',\n",
       " 73: '1:189175060-189239230_strand_+_reg_73',\n",
       " 74: '1:189480922-189543419_strand_+_reg_74',\n",
       " 75: '1:189866129-189938237_strand_+_reg_75',\n",
       " 76: '1:193075619-193133645_strand_+_reg_76',\n",
       " 77: '2:3129373-3183216_strand_+_reg_77',\n",
       " 78: '2:4026085-4083325_strand_+_reg_78',\n",
       " 79: '2:5151601-5210127_strand_+_reg_79',\n",
       " 80: '2:11309886-11385319_strand_+_reg_80',\n",
       " 81: '2:14897328-14968889_strand_+_reg_81',\n",
       " 82: '2:16513406-16660259_strand_+_reg_82',\n",
       " 83: '2:17628682-17734416_strand_+_reg_83',\n",
       " 84: '2:19888794-19999195_strand_+_reg_84',\n",
       " 85: '2:21251409-21298244_strand_+_reg_85',\n",
       " 86: '2:21622583-21703613_strand_+_reg_86',\n",
       " 87: '2:25420535-25461500_strand_+_reg_87',\n",
       " 88: '2:27071925-27131679_strand_+_reg_88',\n",
       " 89: '2:30236602-30280512_strand_+_reg_89',\n",
       " 90: '2:32093351-32148238_strand_+_reg_90',\n",
       " 91: '2:37953869-38034097_strand_+_reg_91',\n",
       " 92: '2:41928242-42023962_strand_+_reg_92',\n",
       " 93: '2:43753833-43799050_strand_+_reg_93',\n",
       " 94: '2:44084305-44153466_strand_+_reg_94',\n",
       " 95: '2:44324153-44432010_strand_+_reg_95',\n",
       " 96: '2:49898807-49970037_strand_+_reg_96',\n",
       " 97: '2:58005861-58065893_strand_+_reg_97',\n",
       " 98: '2:59634429-59685853_strand_+_reg_98',\n",
       " 99: '2:62467453-62543544_strand_+_reg_99',\n",
       " 100: '2:62670661-62738745_strand_+_reg_100',\n",
       " 101: '2:70541303-70603453_strand_+_reg_101',\n",
       " 102: '2:73253634-73311570_strand_+_reg_102',\n",
       " 103: '2:74102486-74171621_strand_+_reg_103',\n",
       " 104: '2:75747159-75820879_strand_+_reg_104',\n",
       " 105: '2:77013944-77091549_strand_+_reg_105',\n",
       " 106: '2:77277948-77367263_strand_+_reg_106',\n",
       " 107: '2:83704396-83781664_strand_+_reg_107',\n",
       " 108: '2:84515475-84567580_strand_+_reg_108',\n",
       " 109: '2:84845394-84900174_strand_+_reg_109',\n",
       " 111: '2:85037645-85079130_strand_+_reg_111',\n",
       " 112: '2:85198480-85295653_strand_+_reg_112',\n",
       " 113: '2:91302426-91340781_strand_+_reg_113',\n",
       " 114: '2:97564926-97650442_strand_+_reg_114',\n",
       " 115: '2:110162327-110221668_strand_+_reg_115',\n",
       " 116: '2:118252753-118346305_strand_+_reg_116',\n",
       " 117: '2:120618697-120674306_strand_+_reg_117',\n",
       " 118: '2:122327422-122368386_strand_+_reg_118',\n",
       " 119: '2:122548219-122636386_strand_+_reg_119',\n",
       " 120: '2:123300821-123374509_strand_+_reg_120',\n",
       " 121: '2:124108515-124225589_strand_+_reg_121',\n",
       " 122: '2:127069492-127147676_strand_+_reg_122',\n",
       " 123: '2:128289201-128390141_strand_+_reg_123',\n",
       " 124: '2:128694272-128817233_strand_+_reg_124',\n",
       " 125: '2:129854913-129936172_strand_+_reg_125',\n",
       " 126: '2:130444939-130507012_strand_+_reg_126',\n",
       " 127: '2:131363155-131441109_strand_+_reg_127',\n",
       " 129: '2:132260572-132326514_strand_+_reg_129',\n",
       " 130: '2:135268536-135340636_strand_+_reg_130',\n",
       " 131: '2:135648540-135694141_strand_+_reg_131',\n",
       " 132: '2:136179077-136262638_strand_+_reg_132',\n",
       " 133: '2:139256790-139346099_strand_+_reg_133',\n",
       " 134: '2:143172015-143254256_strand_+_reg_134',\n",
       " 135: '2:143965229-144015150_strand_+_reg_135',\n",
       " 136: '2:147168730-147215713_strand_+_reg_136',\n",
       " 137: '2:148128049-148221409_strand_+_reg_137',\n",
       " 138: '2:148667886-148731885_strand_+_reg_138',\n",
       " 139: '2:152668254-152718287_strand_+_reg_139',\n",
       " 140: '2:153903330-153973662_strand_+_reg_140',\n",
       " 141: '2:160197345-160260687_strand_+_reg_141',\n",
       " 142: '2:161118134-161195413_strand_+_reg_142',\n",
       " 143: '2:161757252-161819816_strand_+_reg_143',\n",
       " 144: '2:162746567-162835217_strand_+_reg_144',\n",
       " 145: '2:172117465-172170675_strand_+_reg_145',\n",
       " 146: '2:172390054-172457136_strand_+_reg_146',\n",
       " 147: '2:172797494-172886579_strand_+_reg_147',\n",
       " 148: '2:178118671-178177048_strand_+_reg_148',\n",
       " 149: '2:178484185-178533313_strand_+_reg_149',\n",
       " 150: '2:178654771-178739514_strand_+_reg_150',\n",
       " 151: '2:180410644-180466893_strand_+_reg_151',\n",
       " 152: '3:10116470-10161129_strand_+_reg_152',\n",
       " 153: '3:15080099-15131257_strand_+_reg_153',\n",
       " 154: '3:19284209-19338497_strand_+_reg_154',\n",
       " 155: '3:19992968-20039109_strand_+_reg_155',\n",
       " 156: '3:20975804-21056576_strand_+_reg_156',\n",
       " 157: '3:26804177-26864088_strand_+_reg_157',\n",
       " 158: '3:27183615-27247800_strand_+_reg_158',\n",
       " 159: '3:27430699-27527522_strand_+_reg_159',\n",
       " 160: '3:27639981-27714878_strand_+_reg_160',\n",
       " 161: '3:27985084-28070214_strand_+_reg_161',\n",
       " 162: '3:33340421-33427551_strand_+_reg_162',\n",
       " 163: '3:35074184-35113818_strand_+_reg_163',\n",
       " 164: '3:35313296-35364803_strand_+_reg_164',\n",
       " 165: '3:37638345-37722903_strand_+_reg_165',\n",
       " 166: '3:50411173-50448666_strand_+_reg_166',\n",
       " 168: '3:50892858-50950176_strand_+_reg_168',\n",
       " 169: '3:53719899-53805193_strand_+_reg_169',\n",
       " 170: '3:54216201-54302564_strand_+_reg_170',\n",
       " 171: '3:55041185-55109341_strand_+_reg_171',\n",
       " 172: '3:57736286-57847155_strand_+_reg_172',\n",
       " 173: '3:58079664-58133121_strand_+_reg_173',\n",
       " 174: '3:59191631-59297167_strand_+_reg_174',\n",
       " 175: '3:62385566-62475321_strand_+_reg_175',\n",
       " 176: '3:78938383-79045067_strand_+_reg_176',\n",
       " 177: '3:82013253-82074011_strand_+_reg_177',\n",
       " 178: '3:83926234-84003443_strand_+_reg_178',\n",
       " 179: '3:85960586-86088957_strand_+_reg_179',\n",
       " 180: '3:87079865-87164939_strand_+_reg_180',\n",
       " 181: '3:87262909-87334252_strand_+_reg_181',\n",
       " 182: '3:88457083-88500649_strand_+_reg_182',\n",
       " 183: '3:90073915-90138447_strand_+_reg_183',\n",
       " 184: '3:90262479-90347057_strand_+_reg_184',\n",
       " 185: '3:92217911-92290171_strand_+_reg_185',\n",
       " 186: '3:95498508-95551908_strand_+_reg_186',\n",
       " 187: '3:97271448-97349071_strand_+_reg_187',\n",
       " 188: '3:97659551-97729267_strand_+_reg_188',\n",
       " 189: '3:100383410-100488704_strand_+_reg_189',\n",
       " 190: '3:102135537-102204731_strand_+_reg_190',\n",
       " 191: '3:102681535-102725439_strand_+_reg_191',\n",
       " 192: '3:103092173-103150905_strand_+_reg_192',\n",
       " 193: '3:103363744-103456515_strand_+_reg_193',\n",
       " 194: '3:104633899-104693362_strand_+_reg_194',\n",
       " 195: '3:105881717-105941481_strand_+_reg_195',\n",
       " 196: '3:109002413-109073430_strand_+_reg_196',\n",
       " 197: '3:117207508-117290078_strand_+_reg_197',\n",
       " 198: '3:117778923-117864154_strand_+_reg_198',\n",
       " 199: '3:118415335-118490496_strand_+_reg_199',\n",
       " 200: '3:125833531-125935817_strand_+_reg_200',\n",
       " 201: '3:126599799-126711558_strand_+_reg_201',\n",
       " 202: '3:127684895-127739338_strand_+_reg_202',\n",
       " 203: '3:129542117-129668198_strand_+_reg_203',\n",
       " 204: '3:130232712-130315166_strand_+_reg_204',\n",
       " 205: '3:133451837-133520394_strand_+_reg_205',\n",
       " 206: '3:135569252-135673396_strand_+_reg_206',\n",
       " 207: '3:137598529-137658281_strand_+_reg_207',\n",
       " 208: '3:145795719-145916046_strand_+_reg_208',\n",
       " 209: '3:146300985-146364892_strand_+_reg_209',\n",
       " 210: '3:152275792-152343485_strand_+_reg_210',\n",
       " 211: '3:157139588-157226143_strand_+_reg_211',\n",
       " 212: '4:3040005-3100987_strand_+_reg_212',\n",
       " 213: '4:3471494-3540953_strand_+_reg_213',\n",
       " 214: '4:6346853-6400453_strand_+_reg_214',\n",
       " 215: '4:8546598-8621298_strand_+_reg_215',\n",
       " 216: '4:9621515-9668472_strand_+_reg_216',\n",
       " 217: '4:10962894-11009200_strand_+_reg_217',\n",
       " 218: '4:11962627-12051893_strand_+_reg_218',\n",
       " 219: '4:22467081-22509068_strand_+_reg_219',\n",
       " 220: '4:24745226-24835054_strand_+_reg_220',\n",
       " 221: '4:24955708-25039170_strand_+_reg_221',\n",
       " 222: '4:25657294-25764933_strand_+_reg_222',\n",
       " 223: '4:28804761-28881872_strand_+_reg_223',\n",
       " 224: '4:32845518-32900800_strand_+_reg_224',\n",
       " 225: '4:33921425-33962072_strand_+_reg_225',\n",
       " 226: '4:39175238-39234393_strand_+_reg_226',\n",
       " 228: '4:40468049-40529504_strand_+_reg_228',\n",
       " 229: '4:40762947-40854743_strand_+_reg_229',\n",
       " 230: '4:41137781-41221678_strand_+_reg_230',\n",
       " 231: '4:43195060-43271596_strand_+_reg_231',\n",
       " 232: '4:43536822-43581834_strand_+_reg_232',\n",
       " 233: '4:47556560-47631977_strand_+_reg_233',\n",
       " 234: '4:49711396-49823368_strand_+_reg_234',\n",
       " 235: '4:56810348-56866569_strand_+_reg_235',\n",
       " 236: '4:56953595-57059748_strand_+_reg_236',\n",
       " 237: '4:57402930-57483013_strand_+_reg_237',\n",
       " 238: '4:58488835-58557154_strand_+_reg_238',\n",
       " 239: '4:62288487-62379491_strand_+_reg_239',\n",
       " 240: '4:65706234-65796043_strand_+_reg_240',\n",
       " 241: '4:66351813-66425032_strand_+_reg_241',\n",
       " 242: '4:70432043-70539439_strand_+_reg_242',\n",
       " 243: '4:86148471-86220141_strand_+_reg_243',\n",
       " 244: '4:86991120-87073412_strand_+_reg_244',\n",
       " 245: '4:88142498-88275144_strand_+_reg_245',\n",
       " 246: '4:98072752-98129663_strand_+_reg_246',\n",
       " 247: '4:98531272-98638773_strand_+_reg_247',\n",
       " 248: '4:99330446-99395864_strand_+_reg_248',\n",
       " 249: '4:100963562-101018943_strand_+_reg_249',\n",
       " 250: '4:104983954-105060480_strand_+_reg_250',\n",
       " 751: '13:114065922-114149495_strand_+_reg_751',\n",
       " 752: '13:119586445-119624951_strand_+_reg_752',\n",
       " 758: '14:13323083-13378913_strand_+_reg_758',\n",
       " 759: '14:14043804-14108999_strand_+_reg_759',\n",
       " 760: '14:15345670-15436164_strand_+_reg_760',\n",
       " 761: '14:21333204-21412833_strand_+_reg_761',\n",
       " 762: '14:21959106-22004415_strand_+_reg_762',\n",
       " 763: '14:26419598-26536458_strand_+_reg_763',\n",
       " 764: '14:29018427-29124966_strand_+_reg_764',\n",
       " 765: '14:29313773-29399124_strand_+_reg_765',\n",
       " 766: '14:31127141-31206382_strand_+_reg_766',\n",
       " 767: '14:31992648-32046083_strand_+_reg_767',\n",
       " 768: '14:32517175-32620238_strand_+_reg_768',\n",
       " 769: '14:34178487-34245598_strand_+_reg_769',\n",
       " 770: '14:40891564-40969316_strand_+_reg_770',\n",
       " 772: '14:45529453-45635426_strand_+_reg_772',\n",
       " 773: '14:46891206-47000060_strand_+_reg_773',\n",
       " 774: '14:48145216-48234948_strand_+_reg_774',\n",
       " 775: '14:55487236-55538412_strand_+_reg_775',\n",
       " 776: '14:57498975-57575644_strand_+_reg_776',\n",
       " 777: '14:57498975-57569853_strand_+_reg_777',\n",
       " 778: '14:57669680-57747633_strand_+_reg_778',\n",
       " 779: '14:57891541-57932428_strand_+_reg_779',\n",
       " 780: '14:58064999-58123602_strand_+_reg_780',\n",
       " 781: '14:59196105-59261023_strand_+_reg_781',\n",
       " 782: '14:59691594-59748067_strand_+_reg_782',\n",
       " 783: '14:61628986-61683707_strand_+_reg_783',\n",
       " 784: '14:61793570-61846789_strand_+_reg_784',\n",
       " 785: '14:62163770-62246386_strand_+_reg_785',\n",
       " 786: '14:62834246-62950869_strand_+_reg_786',\n",
       " 787: '14:63118357-63175661_strand_+_reg_787',\n",
       " 788: '14:65504499-65594282_strand_+_reg_788',\n",
       " 789: '14:65960189-65998537_strand_+_reg_789',\n",
       " 790: '14:67878001-67932629_strand_+_reg_790',\n",
       " 791: '14:69791321-69871987_strand_+_reg_791',\n",
       " 792: '14:70292991-70355449_strand_+_reg_792',\n",
       " 793: '14:75463070-75553422_strand_+_reg_793',\n",
       " 794: '14:75860410-75925995_strand_+_reg_794',\n",
       " 795: '14:76415881-76490537_strand_+_reg_795',\n",
       " 796: '14:79478685-79573260_strand_+_reg_796',\n",
       " 797: '14:79899686-79984133_strand_+_reg_797',\n",
       " 798: '14:80227149-80288631_strand_+_reg_798',\n",
       " 799: '14:99526543-99602180_strand_+_reg_799',\n",
       " 800: '14:102774665-102854185_strand_+_reg_800',\n",
       " 801: '14:103099997-103173301_strand_+_reg_801',\n",
       " 802: '14:103414208-103482720_strand_+_reg_802',\n",
       " 803: '14:105264551-105344049_strand_+_reg_803',\n",
       " 804: '14:108781216-108854003_strand_+_reg_804',\n",
       " 805: '14:114844433-114923616_strand_+_reg_805',\n",
       " 806: '14:119193071-119288918_strand_+_reg_806',\n",
       " 807: '14:119621024-119697288_strand_+_reg_807',\n",
       " 808: '14:120779255-120908010_strand_+_reg_808',\n",
       " 809: '14:121918265-122090024_strand_+_reg_809',\n",
       " 810: '15:6509514-6581190_strand_+_reg_810',\n",
       " 811: '15:7170479-7236680_strand_+_reg_811',\n",
       " 812: '15:8375489-8461753_strand_+_reg_812',\n",
       " 813: '15:12417846-12507837_strand_+_reg_813',\n",
       " 814: '15:13071050-13175931_strand_+_reg_814',\n",
       " 815: '15:16742412-16833900_strand_+_reg_815',\n",
       " 816: '15:18847236-18992658_strand_+_reg_816',\n",
       " 817: '15:24958652-25046793_strand_+_reg_817',\n",
       " 818: '15:28171581-28259520_strand_+_reg_818',\n",
       " 819: '15:30249955-30317077_strand_+_reg_819',\n",
       " 820: '15:36478751-36528020_strand_+_reg_820',\n",
       " 821: '15:40814516-40989464_strand_+_reg_821',\n",
       " 822: '15:44711034-44779894_strand_+_reg_822',\n",
       " 823: '15:45075447-45142386_strand_+_reg_823',\n",
       " 824: '15:50654938-50758251_strand_+_reg_824',\n",
       " 825: '15:52730443-52820439_strand_+_reg_825',\n",
       " 826: '15:58158195-58230642_strand_+_reg_826',\n",
       " 827: '15:67091131-67172296_strand_+_reg_827',\n",
       " 828: '15:68684491-68749615_strand_+_reg_828',\n",
       " 829: '15:68923567-69002893_strand_+_reg_829',\n",
       " 830: '15:71879586-71959467_strand_+_reg_830',\n",
       " 831: '15:76273099-76323683_strand_+_reg_831',\n",
       " 832: '15:76434091-76478590_strand_+_reg_832',\n",
       " 833: '15:80684138-80732788_strand_+_reg_833',\n",
       " 834: '15:81504063-81579541_strand_+_reg_834',\n",
       " 835: '15:81672506-81729092_strand_+_reg_835',\n",
       " 836: '15:83364966-83464933_strand_+_reg_836',\n",
       " 837: '15:88479865-88547648_strand_+_reg_837',\n",
       " 838: '15:88818050-88864719_strand_+_reg_838',\n",
       " 839: '15:92180253-92287287_strand_+_reg_839',\n",
       " 840: '15:93482810-93529982_strand_+_reg_840',\n",
       " 841: '15:95034078-95096462_strand_+_reg_841',\n",
       " 842: '15:96274908-96319877_strand_+_reg_842',\n",
       " 843: '15:101815997-101884700_strand_+_reg_843',\n",
       " 844: '15:103501670-103562538_strand_+_reg_844',\n",
       " 845: '16:5140368-5205722_strand_+_reg_845',\n",
       " 846: '16:10331430-10395351_strand_+_reg_846',\n",
       " 847: '16:21147080-21202863_strand_+_reg_847',\n",
       " 848: '16:22756585-22854730_strand_+_reg_848',\n",
       " 849: '16:23642588-23704832_strand_+_reg_849',\n",
       " 850: '16:23942714-24001471_strand_+_reg_850',\n",
       " 851: '16:25973196-26088540_strand_+_reg_851',\n",
       " 852: '16:28583146-28642529_strand_+_reg_852',\n",
       " 853: '16:28813123-28919744_strand_+_reg_853',\n",
       " 854: '16:29118866-29189467_strand_+_reg_854',\n",
       " 855: '16:32431344-32553252_strand_+_reg_855',\n",
       " 856: '16:32643912-32726340_strand_+_reg_856',\n",
       " 857: '16:34957154-35017777_strand_+_reg_857',\n",
       " 858: '16:35723966-35794035_strand_+_reg_858',\n",
       " 859: '16:36610671-36680286_strand_+_reg_859',\n",
       " 860: '16:38150332-38259251_strand_+_reg_860',\n",
       " 861: '16:41514064-41562563_strand_+_reg_861',\n",
       " 862: '16:42245627-42341328_strand_+_reg_862',\n",
       " 863: '16:42875154-42969540_strand_+_reg_863',\n",
       " 864: '16:43912104-43983953_strand_+_reg_864',\n",
       " 866: '16:44200258-44301077_strand_+_reg_866',\n",
       " 867: '16:50679262-50747219_strand_+_reg_867',\n",
       " 868: '16:52068029-52157877_strand_+_reg_868',\n",
       " 869: '16:58434511-58529914_strand_+_reg_869',\n",
       " 870: '16:67559729-67628201_strand_+_reg_870',\n",
       " 871: '16:74079872-74201979_strand_+_reg_871',\n",
       " 872: '16:77374249-77459818_strand_+_reg_872',\n",
       " 873: '16:77620781-77677713_strand_+_reg_873',\n",
       " 874: '16:78553141-78609787_strand_+_reg_874',\n",
       " 875: '16:88117436-88198446_strand_+_reg_875',\n",
       " 876: '16:89507846-89608464_strand_+_reg_876',\n",
       " 877: '16:90915327-91010712_strand_+_reg_877',\n",
       " 878: '16:93967016-94031875_strand_+_reg_878',\n",
       " 879: '16:94279006-94324592_strand_+_reg_879',\n",
       " 880: '16:94419141-94483370_strand_+_reg_880',\n",
       " 881: '16:95361862-95460042_strand_+_reg_881',\n",
       " 882: '16:97687705-97762534_strand_+_reg_882',\n",
       " 883: '17:6758025-6822752_strand_+_reg_883',\n",
       " 884: '17:11654178-11797180_strand_+_reg_884',\n",
       " 885: '17:13808723-13929026_strand_+_reg_885',\n",
       " 886: '17:15361839-15397147_strand_+_reg_886',\n",
       " 887: '17:23516967-23561137_strand_+_reg_887',\n",
       " 888: '17:24038288-24101203_strand_+_reg_888',\n",
       " 889: '17:31799132-31853772_strand_+_reg_889',\n",
       " 890: '17:34090951-34145951_strand_+_reg_890',\n",
       " 891: '17:35516761-35560061_strand_+_reg_891',\n",
       " 892: '17:43011066-43103508_strand_+_reg_892',\n",
       " 893: '17:44188990-44272216_strand_+_reg_893',\n",
       " 894: '17:46828396-46877205_strand_+_reg_894',\n",
       " 895: '17:48353042-48410987_strand_+_reg_895',\n",
       " 896: '17:48532940-48612895_strand_+_reg_896',\n",
       " 897: '17:48761285-48835829_strand_+_reg_897',\n",
       " 898: '17:49458027-49562568_strand_+_reg_898',\n",
       " 899: '17:50115429-50204110_strand_+_reg_899',\n",
       " 900: '17:51727923-51816643_strand_+_reg_900',\n",
       " 901: '17:53567112-53638475_strand_+_reg_901',\n",
       " 902: '17:55266212-55319734_strand_+_reg_902',\n",
       " 903: '17:56872991-56939799_strand_+_reg_903',\n",
       " 904: '17:57254102-57357459_strand_+_reg_904',\n",
       " 905: '17:62630369-62740980_strand_+_reg_905',\n",
       " 906: '17:67430764-67483458_strand_+_reg_906',\n",
       " 907: '17:73209957-73282539_strand_+_reg_907',\n",
       " 908: '17:73871483-73952612_strand_+_reg_908',\n",
       " 909: '17:75432566-75547174_strand_+_reg_909',\n",
       " 910: '17:80137911-80209379_strand_+_reg_910',\n",
       " 911: '17:83530997-83606275_strand_+_reg_911',\n",
       " 912: '17:84464454-84530370_strand_+_reg_912',\n",
       " 913: '17:84850274-84953300_strand_+_reg_913',\n",
       " 914: '17:89322309-89377903_strand_+_reg_914',\n",
       " 915: '17:90044561-90126527_strand_+_reg_915',\n",
       " 916: '17:90276015-90421156_strand_+_reg_916',\n",
       " 917: '18:6929796-7011984_strand_+_reg_917',\n",
       " 918: '18:15391109-15452687_strand_+_reg_918',\n",
       " 919: '18:21151931-21203166_strand_+_reg_919',\n",
       " 920: '18:24089649-24145055_strand_+_reg_920',\n",
       " 921: '18:24408559-24500784_strand_+_reg_921',\n",
       " 922: '18:24712942-24888570_strand_+_reg_922',\n",
       " 923: '18:31929025-31995900_strand_+_reg_923',\n",
       " 925: '18:32936362-33016449_strand_+_reg_925',\n",
       " 926: '18:34069958-34139601_strand_+_reg_926',\n",
       " 927: '18:35472165-35550427_strand_+_reg_927',\n",
       " 928: '18:36681158-36730587_strand_+_reg_928',\n",
       " 929: '18:37139186-37201875_strand_+_reg_929',\n",
       " 930: '18:39886744-39952821_strand_+_reg_930',\n",
       " 931: '18:41736971-41828086_strand_+_reg_931',\n",
       " 932: '18:42055872-42140040_strand_+_reg_932',\n",
       " 933: '18:45568608-45662084_strand_+_reg_933',\n",
       " 934: '18:46169765-46214913_strand_+_reg_934',\n",
       " 935: '18:49582614-49668484_strand_+_reg_935',\n",
       " 936: '18:50001602-50079289_strand_+_reg_936',\n",
       " 937: '18:53219139-53335927_strand_+_reg_937',\n",
       " 938: '18:55993891-56082519_strand_+_reg_938',\n",
       " 939: '18:57239664-57307027_strand_+_reg_939',\n",
       " 940: '18:59180021-59350310_strand_+_reg_940',\n",
       " 941: '18:63719174-63787284_strand_+_reg_941',\n",
       " 942: '18:64575603-64648451_strand_+_reg_942',\n",
       " 943: '18:65843454-65945591_strand_+_reg_943',\n",
       " 944: '18:68933328-69000228_strand_+_reg_944',\n",
       " 945: '18:74461676-74530383_strand_+_reg_945',\n",
       " 946: '18:74669677-74751625_strand_+_reg_946',\n",
       " 947: '18:77958807-78042826_strand_+_reg_947',\n",
       " 948: '18:80950121-81002541_strand_+_reg_948',\n",
       " 949: '18:85278941-85379839_strand_+_reg_949',\n",
       " 950: '19:3625409-3709978_strand_+_reg_950',\n",
       " 951: '19:3822930-3906324_strand_+_reg_951',\n",
       " 952: '19:5638442-5685296_strand_+_reg_952',\n",
       " 953: '19:6276522-6315238_strand_+_reg_953',\n",
       " 954: '19:7054548-7207140_strand_+_reg_954',\n",
       " 955: '19:8544284-8597137_strand_+_reg_955',\n",
       " 956: '19:10685617-10740864_strand_+_reg_956',\n",
       " 957: '19:10930147-10970163_strand_+_reg_957',\n",
       " 958: '19:12424826-12504327_strand_+_reg_958',\n",
       " 959: '19:15780510-15825819_strand_+_reg_959',\n",
       " 960: '19:17296782-17355629_strand_+_reg_960',\n",
       " 961: '19:20518919-20570378_strand_+_reg_961',\n",
       " 962: '19:21713365-21757827_strand_+_reg_962',\n",
       " 963: '19:23553461-23620199_strand_+_reg_963',\n",
       " 964: '19:24600840-24687705_strand_+_reg_964',\n",
       " 965: '19:24944141-25013094_strand_+_reg_965',\n",
       " 966: '19:27524144-27591990_strand_+_reg_966',\n",
       " 967: '19:28355825-28409866_strand_+_reg_967',\n",
       " 968: '19:28563435-28646484_strand_+_reg_968',\n",
       " 969: '19:29856927-29928435_strand_+_reg_969',\n",
       " 970: '19:32895563-32971276_strand_+_reg_970',\n",
       " 971: '19:33213942-33295336_strand_+_reg_971',\n",
       " 972: '19:36172931-36275082_strand_+_reg_972',\n",
       " 973: '19:36718674-36759528_strand_+_reg_973',\n",
       " 974: '19:37472495-37520209_strand_+_reg_974',\n",
       " 975: '19:37965494-38041498_strand_+_reg_975',\n",
       " 976: '19:41050775-41097097_strand_+_reg_976',\n",
       " 977: '19:41281381-41355899_strand_+_reg_977',\n",
       " 978: '19:41963511-42036554_strand_+_reg_978',\n",
       " 979: '19:45861417-45967203_strand_+_reg_979',\n",
       " 980: '19:47634250-47669784_strand_+_reg_980',\n",
       " 981: '19:48639897-48747838_strand_+_reg_981',\n",
       " 982: '19:58794802-58866457_strand_+_reg_982',\n",
       " 983: '19:59015717-59063288_strand_+_reg_983',\n",
       " 984: '19:59369976-59445429_strand_+_reg_984',\n",
       " 985: 'X:93271165-93309819_strand_+_reg_985',\n",
       " 986: 'X:98979231-99088826_strand_+_reg_986',\n",
       " 987: 'X:101910335-102015609_strand_+_reg_987',\n",
       " 988: 'X:136772307-136851006_strand_+_reg_988',\n",
       " 989: 'X:140481839-140547985_strand_+_reg_989',\n",
       " 990: 'X:167122110-167191320_strand_+_reg_990',\n",
       " 991: 'X:169962881-170032477_strand_+_reg_991'}"
      ]
     },
     "execution_count": 68,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(filtered_reg_names)\n",
    "\n",
    "filtered_reg_names"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 73,
   "metadata": {},
   "outputs": [],
   "source": [
    "# count the region number for each chr\n",
    "chr_2_region_num = {}\n",
    "for _reg, _info in filtered_reg_names.items():\n",
    "    _chr = _info.split('_')[0].split(':')[0]\n",
    "    if _chr not in chr_2_region_num:\n",
    "        chr_2_region_num[_chr] = 1\n",
    "    else:\n",
    "        chr_2_region_num[_chr] += 1\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 75,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'4': 61,\n",
       " '5': 56,\n",
       " '6': 56,\n",
       " '7': 45,\n",
       " '8': 52,\n",
       " '9': 54,\n",
       " '10': 64,\n",
       " '11': 57,\n",
       " '12': 35,\n",
       " '13': 49,\n",
       " '1': 76,\n",
       " '2': 73,\n",
       " '3': 59,\n",
       " '14': 51,\n",
       " '15': 35,\n",
       " '16': 37,\n",
       " '17': 34,\n",
       " '18': 32,\n",
       " '19': 35,\n",
       " 'X': 7}"
      ]
     },
     "execution_count": 75,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "chr_2_region_num"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 74,
   "metadata": {},
   "outputs": [],
   "source": [
    "# save region number info for chr\n",
    "pickle.dump(chr_2_region_num, open(os.path.join(library_folder, 'chr_2_region_num.pkl'), 'wb'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 76,
   "metadata": {},
   "outputs": [],
   "source": [
    "# save region name info for chr\n",
    "chr_2_region_names = {}\n",
    "for _reg, _info in filtered_reg_names.items():\n",
    "    _chr = _info.split('_')[0].split(':')[0]\n",
    "    if _chr not in chr_2_region_names:\n",
    "        chr_2_region_names[_chr] = [_reg]\n",
    "    else:\n",
    "        chr_2_region_names[_chr].append(_reg)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 77,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'4': [251,\n",
       "  252,\n",
       "  253,\n",
       "  255,\n",
       "  256,\n",
       "  257,\n",
       "  258,\n",
       "  259,\n",
       "  260,\n",
       "  261,\n",
       "  262,\n",
       "  263,\n",
       "  264,\n",
       "  265,\n",
       "  266,\n",
       "  267,\n",
       "  268,\n",
       "  269,\n",
       "  270,\n",
       "  271,\n",
       "  272,\n",
       "  273,\n",
       "  274,\n",
       "  212,\n",
       "  213,\n",
       "  214,\n",
       "  215,\n",
       "  216,\n",
       "  217,\n",
       "  218,\n",
       "  219,\n",
       "  220,\n",
       "  221,\n",
       "  222,\n",
       "  223,\n",
       "  224,\n",
       "  225,\n",
       "  226,\n",
       "  228,\n",
       "  229,\n",
       "  230,\n",
       "  231,\n",
       "  232,\n",
       "  233,\n",
       "  234,\n",
       "  235,\n",
       "  236,\n",
       "  237,\n",
       "  238,\n",
       "  239,\n",
       "  240,\n",
       "  241,\n",
       "  242,\n",
       "  243,\n",
       "  244,\n",
       "  245,\n",
       "  246,\n",
       "  247,\n",
       "  248,\n",
       "  249,\n",
       "  250],\n",
       " '5': [275,\n",
       "  277,\n",
       "  278,\n",
       "  279,\n",
       "  280,\n",
       "  281,\n",
       "  282,\n",
       "  283,\n",
       "  284,\n",
       "  285,\n",
       "  286,\n",
       "  287,\n",
       "  288,\n",
       "  289,\n",
       "  290,\n",
       "  291,\n",
       "  292,\n",
       "  293,\n",
       "  295,\n",
       "  296,\n",
       "  297,\n",
       "  298,\n",
       "  299,\n",
       "  300,\n",
       "  301,\n",
       "  302,\n",
       "  303,\n",
       "  304,\n",
       "  305,\n",
       "  306,\n",
       "  307,\n",
       "  308,\n",
       "  309,\n",
       "  310,\n",
       "  311,\n",
       "  312,\n",
       "  313,\n",
       "  314,\n",
       "  315,\n",
       "  316,\n",
       "  317,\n",
       "  318,\n",
       "  319,\n",
       "  320,\n",
       "  321,\n",
       "  322,\n",
       "  324,\n",
       "  325,\n",
       "  326,\n",
       "  327,\n",
       "  328,\n",
       "  329,\n",
       "  330,\n",
       "  332,\n",
       "  333,\n",
       "  334],\n",
       " '6': [335,\n",
       "  336,\n",
       "  337,\n",
       "  338,\n",
       "  339,\n",
       "  340,\n",
       "  341,\n",
       "  342,\n",
       "  343,\n",
       "  344,\n",
       "  345,\n",
       "  347,\n",
       "  348,\n",
       "  349,\n",
       "  350,\n",
       "  351,\n",
       "  352,\n",
       "  353,\n",
       "  354,\n",
       "  355,\n",
       "  356,\n",
       "  357,\n",
       "  358,\n",
       "  359,\n",
       "  360,\n",
       "  361,\n",
       "  362,\n",
       "  363,\n",
       "  364,\n",
       "  365,\n",
       "  366,\n",
       "  367,\n",
       "  368,\n",
       "  369,\n",
       "  370,\n",
       "  371,\n",
       "  372,\n",
       "  373,\n",
       "  374,\n",
       "  375,\n",
       "  376,\n",
       "  377,\n",
       "  378,\n",
       "  379,\n",
       "  380,\n",
       "  381,\n",
       "  382,\n",
       "  383,\n",
       "  384,\n",
       "  385,\n",
       "  386,\n",
       "  387,\n",
       "  388,\n",
       "  389,\n",
       "  390,\n",
       "  391],\n",
       " '7': [392,\n",
       "  393,\n",
       "  394,\n",
       "  395,\n",
       "  396,\n",
       "  397,\n",
       "  398,\n",
       "  399,\n",
       "  400,\n",
       "  401,\n",
       "  402,\n",
       "  403,\n",
       "  404,\n",
       "  405,\n",
       "  406,\n",
       "  407,\n",
       "  408,\n",
       "  409,\n",
       "  410,\n",
       "  411,\n",
       "  412,\n",
       "  413,\n",
       "  414,\n",
       "  415,\n",
       "  416,\n",
       "  417,\n",
       "  418,\n",
       "  419,\n",
       "  420,\n",
       "  421,\n",
       "  422,\n",
       "  423,\n",
       "  424,\n",
       "  425,\n",
       "  426,\n",
       "  427,\n",
       "  428,\n",
       "  429,\n",
       "  430,\n",
       "  431,\n",
       "  432,\n",
       "  433,\n",
       "  434,\n",
       "  435,\n",
       "  436],\n",
       " '8': [437,\n",
       "  438,\n",
       "  439,\n",
       "  440,\n",
       "  441,\n",
       "  442,\n",
       "  444,\n",
       "  445,\n",
       "  446,\n",
       "  447,\n",
       "  448,\n",
       "  449,\n",
       "  450,\n",
       "  451,\n",
       "  452,\n",
       "  453,\n",
       "  454,\n",
       "  455,\n",
       "  456,\n",
       "  457,\n",
       "  458,\n",
       "  459,\n",
       "  460,\n",
       "  461,\n",
       "  462,\n",
       "  463,\n",
       "  464,\n",
       "  465,\n",
       "  466,\n",
       "  467,\n",
       "  468,\n",
       "  469,\n",
       "  470,\n",
       "  471,\n",
       "  472,\n",
       "  473,\n",
       "  474,\n",
       "  475,\n",
       "  476,\n",
       "  477,\n",
       "  478,\n",
       "  479,\n",
       "  480,\n",
       "  481,\n",
       "  482,\n",
       "  483,\n",
       "  484,\n",
       "  485,\n",
       "  486,\n",
       "  487,\n",
       "  488,\n",
       "  489],\n",
       " '9': [491,\n",
       "  492,\n",
       "  493,\n",
       "  494,\n",
       "  495,\n",
       "  496,\n",
       "  497,\n",
       "  498,\n",
       "  499,\n",
       "  500,\n",
       "  501,\n",
       "  502,\n",
       "  503,\n",
       "  504,\n",
       "  505,\n",
       "  506,\n",
       "  507,\n",
       "  508,\n",
       "  509,\n",
       "  510,\n",
       "  511,\n",
       "  512,\n",
       "  513,\n",
       "  514,\n",
       "  515,\n",
       "  516,\n",
       "  517,\n",
       "  518,\n",
       "  519,\n",
       "  520,\n",
       "  521,\n",
       "  523,\n",
       "  524,\n",
       "  525,\n",
       "  526,\n",
       "  527,\n",
       "  528,\n",
       "  529,\n",
       "  530,\n",
       "  531,\n",
       "  532,\n",
       "  533,\n",
       "  534,\n",
       "  535,\n",
       "  536,\n",
       "  537,\n",
       "  538,\n",
       "  539,\n",
       "  540,\n",
       "  541,\n",
       "  542,\n",
       "  543,\n",
       "  544,\n",
       "  545],\n",
       " '10': [546,\n",
       "  547,\n",
       "  548,\n",
       "  549,\n",
       "  550,\n",
       "  551,\n",
       "  552,\n",
       "  553,\n",
       "  554,\n",
       "  555,\n",
       "  556,\n",
       "  557,\n",
       "  558,\n",
       "  559,\n",
       "  560,\n",
       "  561,\n",
       "  562,\n",
       "  563,\n",
       "  564,\n",
       "  565,\n",
       "  566,\n",
       "  567,\n",
       "  568,\n",
       "  569,\n",
       "  570,\n",
       "  571,\n",
       "  572,\n",
       "  573,\n",
       "  574,\n",
       "  575,\n",
       "  576,\n",
       "  577,\n",
       "  579,\n",
       "  580,\n",
       "  581,\n",
       "  582,\n",
       "  583,\n",
       "  584,\n",
       "  585,\n",
       "  586,\n",
       "  587,\n",
       "  588,\n",
       "  589,\n",
       "  590,\n",
       "  591,\n",
       "  592,\n",
       "  593,\n",
       "  594,\n",
       "  595,\n",
       "  596,\n",
       "  597,\n",
       "  598,\n",
       "  599,\n",
       "  600,\n",
       "  601,\n",
       "  602,\n",
       "  603,\n",
       "  604,\n",
       "  605,\n",
       "  606,\n",
       "  607,\n",
       "  608,\n",
       "  609,\n",
       "  610],\n",
       " '11': [611,\n",
       "  612,\n",
       "  613,\n",
       "  614,\n",
       "  615,\n",
       "  616,\n",
       "  617,\n",
       "  618,\n",
       "  619,\n",
       "  620,\n",
       "  621,\n",
       "  622,\n",
       "  623,\n",
       "  624,\n",
       "  625,\n",
       "  626,\n",
       "  627,\n",
       "  628,\n",
       "  629,\n",
       "  630,\n",
       "  631,\n",
       "  632,\n",
       "  633,\n",
       "  634,\n",
       "  635,\n",
       "  636,\n",
       "  637,\n",
       "  638,\n",
       "  639,\n",
       "  640,\n",
       "  641,\n",
       "  642,\n",
       "  643,\n",
       "  644,\n",
       "  645,\n",
       "  646,\n",
       "  647,\n",
       "  648,\n",
       "  649,\n",
       "  650,\n",
       "  651,\n",
       "  652,\n",
       "  653,\n",
       "  654,\n",
       "  655,\n",
       "  656,\n",
       "  657,\n",
       "  658,\n",
       "  659,\n",
       "  660,\n",
       "  661,\n",
       "  662,\n",
       "  663,\n",
       "  664,\n",
       "  665,\n",
       "  666,\n",
       "  667],\n",
       " '12': [668,\n",
       "  669,\n",
       "  670,\n",
       "  671,\n",
       "  672,\n",
       "  673,\n",
       "  674,\n",
       "  675,\n",
       "  676,\n",
       "  677,\n",
       "  678,\n",
       "  679,\n",
       "  680,\n",
       "  681,\n",
       "  682,\n",
       "  683,\n",
       "  684,\n",
       "  685,\n",
       "  687,\n",
       "  688,\n",
       "  689,\n",
       "  690,\n",
       "  691,\n",
       "  692,\n",
       "  693,\n",
       "  694,\n",
       "  695,\n",
       "  696,\n",
       "  697,\n",
       "  698,\n",
       "  699,\n",
       "  700,\n",
       "  701,\n",
       "  702,\n",
       "  703],\n",
       " '13': [704,\n",
       "  705,\n",
       "  706,\n",
       "  707,\n",
       "  708,\n",
       "  709,\n",
       "  710,\n",
       "  711,\n",
       "  712,\n",
       "  713,\n",
       "  714,\n",
       "  715,\n",
       "  716,\n",
       "  717,\n",
       "  718,\n",
       "  719,\n",
       "  720,\n",
       "  721,\n",
       "  722,\n",
       "  723,\n",
       "  724,\n",
       "  725,\n",
       "  726,\n",
       "  727,\n",
       "  728,\n",
       "  729,\n",
       "  730,\n",
       "  731,\n",
       "  732,\n",
       "  733,\n",
       "  734,\n",
       "  735,\n",
       "  736,\n",
       "  737,\n",
       "  738,\n",
       "  739,\n",
       "  740,\n",
       "  741,\n",
       "  742,\n",
       "  743,\n",
       "  744,\n",
       "  745,\n",
       "  746,\n",
       "  747,\n",
       "  748,\n",
       "  749,\n",
       "  750,\n",
       "  751,\n",
       "  752],\n",
       " '1': [1,\n",
       "  2,\n",
       "  3,\n",
       "  4,\n",
       "  5,\n",
       "  6,\n",
       "  7,\n",
       "  8,\n",
       "  9,\n",
       "  10,\n",
       "  11,\n",
       "  12,\n",
       "  13,\n",
       "  14,\n",
       "  15,\n",
       "  16,\n",
       "  17,\n",
       "  18,\n",
       "  19,\n",
       "  20,\n",
       "  21,\n",
       "  22,\n",
       "  23,\n",
       "  24,\n",
       "  25,\n",
       "  26,\n",
       "  27,\n",
       "  28,\n",
       "  29,\n",
       "  30,\n",
       "  31,\n",
       "  32,\n",
       "  33,\n",
       "  34,\n",
       "  35,\n",
       "  36,\n",
       "  37,\n",
       "  38,\n",
       "  39,\n",
       "  40,\n",
       "  41,\n",
       "  42,\n",
       "  43,\n",
       "  44,\n",
       "  45,\n",
       "  46,\n",
       "  47,\n",
       "  48,\n",
       "  49,\n",
       "  50,\n",
       "  51,\n",
       "  52,\n",
       "  53,\n",
       "  54,\n",
       "  55,\n",
       "  56,\n",
       "  57,\n",
       "  58,\n",
       "  59,\n",
       "  60,\n",
       "  61,\n",
       "  62,\n",
       "  63,\n",
       "  64,\n",
       "  65,\n",
       "  66,\n",
       "  67,\n",
       "  68,\n",
       "  69,\n",
       "  70,\n",
       "  71,\n",
       "  72,\n",
       "  73,\n",
       "  74,\n",
       "  75,\n",
       "  76],\n",
       " '2': [77,\n",
       "  78,\n",
       "  79,\n",
       "  80,\n",
       "  81,\n",
       "  82,\n",
       "  83,\n",
       "  84,\n",
       "  85,\n",
       "  86,\n",
       "  87,\n",
       "  88,\n",
       "  89,\n",
       "  90,\n",
       "  91,\n",
       "  92,\n",
       "  93,\n",
       "  94,\n",
       "  95,\n",
       "  96,\n",
       "  97,\n",
       "  98,\n",
       "  99,\n",
       "  100,\n",
       "  101,\n",
       "  102,\n",
       "  103,\n",
       "  104,\n",
       "  105,\n",
       "  106,\n",
       "  107,\n",
       "  108,\n",
       "  109,\n",
       "  111,\n",
       "  112,\n",
       "  113,\n",
       "  114,\n",
       "  115,\n",
       "  116,\n",
       "  117,\n",
       "  118,\n",
       "  119,\n",
       "  120,\n",
       "  121,\n",
       "  122,\n",
       "  123,\n",
       "  124,\n",
       "  125,\n",
       "  126,\n",
       "  127,\n",
       "  129,\n",
       "  130,\n",
       "  131,\n",
       "  132,\n",
       "  133,\n",
       "  134,\n",
       "  135,\n",
       "  136,\n",
       "  137,\n",
       "  138,\n",
       "  139,\n",
       "  140,\n",
       "  141,\n",
       "  142,\n",
       "  143,\n",
       "  144,\n",
       "  145,\n",
       "  146,\n",
       "  147,\n",
       "  148,\n",
       "  149,\n",
       "  150,\n",
       "  151],\n",
       " '3': [152,\n",
       "  153,\n",
       "  154,\n",
       "  155,\n",
       "  156,\n",
       "  157,\n",
       "  158,\n",
       "  159,\n",
       "  160,\n",
       "  161,\n",
       "  162,\n",
       "  163,\n",
       "  164,\n",
       "  165,\n",
       "  166,\n",
       "  168,\n",
       "  169,\n",
       "  170,\n",
       "  171,\n",
       "  172,\n",
       "  173,\n",
       "  174,\n",
       "  175,\n",
       "  176,\n",
       "  177,\n",
       "  178,\n",
       "  179,\n",
       "  180,\n",
       "  181,\n",
       "  182,\n",
       "  183,\n",
       "  184,\n",
       "  185,\n",
       "  186,\n",
       "  187,\n",
       "  188,\n",
       "  189,\n",
       "  190,\n",
       "  191,\n",
       "  192,\n",
       "  193,\n",
       "  194,\n",
       "  195,\n",
       "  196,\n",
       "  197,\n",
       "  198,\n",
       "  199,\n",
       "  200,\n",
       "  201,\n",
       "  202,\n",
       "  203,\n",
       "  204,\n",
       "  205,\n",
       "  206,\n",
       "  207,\n",
       "  208,\n",
       "  209,\n",
       "  210,\n",
       "  211],\n",
       " '14': [758,\n",
       "  759,\n",
       "  760,\n",
       "  761,\n",
       "  762,\n",
       "  763,\n",
       "  764,\n",
       "  765,\n",
       "  766,\n",
       "  767,\n",
       "  768,\n",
       "  769,\n",
       "  770,\n",
       "  772,\n",
       "  773,\n",
       "  774,\n",
       "  775,\n",
       "  776,\n",
       "  777,\n",
       "  778,\n",
       "  779,\n",
       "  780,\n",
       "  781,\n",
       "  782,\n",
       "  783,\n",
       "  784,\n",
       "  785,\n",
       "  786,\n",
       "  787,\n",
       "  788,\n",
       "  789,\n",
       "  790,\n",
       "  791,\n",
       "  792,\n",
       "  793,\n",
       "  794,\n",
       "  795,\n",
       "  796,\n",
       "  797,\n",
       "  798,\n",
       "  799,\n",
       "  800,\n",
       "  801,\n",
       "  802,\n",
       "  803,\n",
       "  804,\n",
       "  805,\n",
       "  806,\n",
       "  807,\n",
       "  808,\n",
       "  809],\n",
       " '15': [810,\n",
       "  811,\n",
       "  812,\n",
       "  813,\n",
       "  814,\n",
       "  815,\n",
       "  816,\n",
       "  817,\n",
       "  818,\n",
       "  819,\n",
       "  820,\n",
       "  821,\n",
       "  822,\n",
       "  823,\n",
       "  824,\n",
       "  825,\n",
       "  826,\n",
       "  827,\n",
       "  828,\n",
       "  829,\n",
       "  830,\n",
       "  831,\n",
       "  832,\n",
       "  833,\n",
       "  834,\n",
       "  835,\n",
       "  836,\n",
       "  837,\n",
       "  838,\n",
       "  839,\n",
       "  840,\n",
       "  841,\n",
       "  842,\n",
       "  843,\n",
       "  844],\n",
       " '16': [845,\n",
       "  846,\n",
       "  847,\n",
       "  848,\n",
       "  849,\n",
       "  850,\n",
       "  851,\n",
       "  852,\n",
       "  853,\n",
       "  854,\n",
       "  855,\n",
       "  856,\n",
       "  857,\n",
       "  858,\n",
       "  859,\n",
       "  860,\n",
       "  861,\n",
       "  862,\n",
       "  863,\n",
       "  864,\n",
       "  866,\n",
       "  867,\n",
       "  868,\n",
       "  869,\n",
       "  870,\n",
       "  871,\n",
       "  872,\n",
       "  873,\n",
       "  874,\n",
       "  875,\n",
       "  876,\n",
       "  877,\n",
       "  878,\n",
       "  879,\n",
       "  880,\n",
       "  881,\n",
       "  882],\n",
       " '17': [883,\n",
       "  884,\n",
       "  885,\n",
       "  886,\n",
       "  887,\n",
       "  888,\n",
       "  889,\n",
       "  890,\n",
       "  891,\n",
       "  892,\n",
       "  893,\n",
       "  894,\n",
       "  895,\n",
       "  896,\n",
       "  897,\n",
       "  898,\n",
       "  899,\n",
       "  900,\n",
       "  901,\n",
       "  902,\n",
       "  903,\n",
       "  904,\n",
       "  905,\n",
       "  906,\n",
       "  907,\n",
       "  908,\n",
       "  909,\n",
       "  910,\n",
       "  911,\n",
       "  912,\n",
       "  913,\n",
       "  914,\n",
       "  915,\n",
       "  916],\n",
       " '18': [917,\n",
       "  918,\n",
       "  919,\n",
       "  920,\n",
       "  921,\n",
       "  922,\n",
       "  923,\n",
       "  925,\n",
       "  926,\n",
       "  927,\n",
       "  928,\n",
       "  929,\n",
       "  930,\n",
       "  931,\n",
       "  932,\n",
       "  933,\n",
       "  934,\n",
       "  935,\n",
       "  936,\n",
       "  937,\n",
       "  938,\n",
       "  939,\n",
       "  940,\n",
       "  941,\n",
       "  942,\n",
       "  943,\n",
       "  944,\n",
       "  945,\n",
       "  946,\n",
       "  947,\n",
       "  948,\n",
       "  949],\n",
       " '19': [950,\n",
       "  951,\n",
       "  952,\n",
       "  953,\n",
       "  954,\n",
       "  955,\n",
       "  956,\n",
       "  957,\n",
       "  958,\n",
       "  959,\n",
       "  960,\n",
       "  961,\n",
       "  962,\n",
       "  963,\n",
       "  964,\n",
       "  965,\n",
       "  966,\n",
       "  967,\n",
       "  968,\n",
       "  969,\n",
       "  970,\n",
       "  971,\n",
       "  972,\n",
       "  973,\n",
       "  974,\n",
       "  975,\n",
       "  976,\n",
       "  977,\n",
       "  978,\n",
       "  979,\n",
       "  980,\n",
       "  981,\n",
       "  982,\n",
       "  983,\n",
       "  984],\n",
       " 'X': [985, 986, 987, 988, 989, 990, 991]}"
      ]
     },
     "execution_count": 77,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "chr_2_region_names"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 78,
   "metadata": {
    "scrolled": false
   },
   "outputs": [],
   "source": [
    "# save\n",
    "pickle.dump(chr_2_region_names, open(os.path.join(library_folder, 'chr_2_region_names.pkl'), 'wb'))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# 3. Load and asssemble probes"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 3.1 design readout mapping dict (region_to_readout)\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "Generate 'region_to_readout dict' with the above 'chr_2_region_num.pkl' and 'chr_2_region_names.pkl' as inputs by using the jupyter notebook from http://localhost:8888/notebooks/Documents/Chromatin_Analysis_Scripts/Library_design/CTP13-mop-merged-1000-SE/Design_codebook_CTP13-mop-merged-1000-SE.ipynb;\n",
    "\n",
    "\n",
    "Next, load the 'region_to_readout dict' in the step below and proceed."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 3.2 loading components"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### primers\n",
    "pick primer set 2,9"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Picked primer: ID: W1A03_primer_2\n",
      "Name: W1A03_primer_2\n",
      "Description: W1A03_primer_2\n",
      "Number of features: 0\n",
      "Seq('CCCGCAATGGCTGACAACCG')\n",
      "- Picked primer: ID: W1A10_primer_9\n",
      "Name: W1A10_primer_9\n",
      "Description: W1A10_primer_9\n",
      "Number of features: 0\n",
      "Seq('TAATACGACTCACTATAGGGATTGCCGCATGGTTTCCG')\n"
     ]
    }
   ],
   "source": [
    "from ImageAnalysis3 import library_tools\n",
    "primer_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Primers'\n",
    "primers = library_tools.assemble.load_primers([2,9], _primer_folder=primer_folder)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### readouts"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "# summarize total readout usage\n",
    "region_2_readout_dict = pickle.load(open(os.path.join(library_folder, 'region_2_readout.pkl'), 'rb'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "968\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "{1: ['c3', 'c47', 'c76'],\n",
       " 2: ['c8', 'c54', 'c90'],\n",
       " 3: ['c29', 'c69', 'c87'],\n",
       " 4: ['c51', 'c65', 'c73'],\n",
       " 5: ['c35', 'c53', 'c60'],\n",
       " 6: ['c11', 'c18', 'c19'],\n",
       " 7: ['c52', 'c62', 'c89'],\n",
       " 8: ['c14', 'c30', 'c85'],\n",
       " 9: ['c15', 'c63', 'c84'],\n",
       " 10: ['c43', 'c68', 'c70'],\n",
       " 11: ['c45', 'c57', 'c81'],\n",
       " 12: ['c12', 'c79', 'c92'],\n",
       " 13: ['c42', 'c47', 'c66'],\n",
       " 14: ['c20', 'c21', 'c39'],\n",
       " 15: ['c9', 'c71', 'c72'],\n",
       " 16: ['c17', 'c31', 'c61'],\n",
       " 17: ['c24', 'c74', 'c94'],\n",
       " 18: ['c4', 'c28', 'c83'],\n",
       " 19: ['c18', 'c41', 'c60'],\n",
       " 20: ['c0', 'c27', 'c52'],\n",
       " 21: ['c33', 'c59', 'c93'],\n",
       " 22: ['c30', 'c75', 'c82'],\n",
       " 23: ['c10', 'c34', 'c77'],\n",
       " 24: ['c40', 'c55', 'c67'],\n",
       " 25: ['c23', 'c32', 'c56'],\n",
       " 26: ['c9', 'c22', 'c54'],\n",
       " 27: ['c16', 'c44', 'c88'],\n",
       " 28: ['c49', 'c63', 'c81'],\n",
       " 29: ['c37', 'c39', 'c87'],\n",
       " 30: ['c69', 'c74', 'c91'],\n",
       " 31: ['c11', 'c12', 'c26'],\n",
       " 32: ['c6', 'c50', 'c78'],\n",
       " 33: ['c1', 'c13', 'c38'],\n",
       " 34: ['c41', 'c84', 'c90'],\n",
       " 35: ['c35', 'c36', 'c66'],\n",
       " 36: ['c7', 'c30', 'c76'],\n",
       " 37: ['c58', 'c70', 'c75'],\n",
       " 38: ['c3', 'c18', 'c22'],\n",
       " 39: ['c0', 'c19', 'c20'],\n",
       " 40: ['c29', 'c89', 'c91'],\n",
       " 41: ['c24', 'c37', 'c62'],\n",
       " 42: ['c72', 'c83', 'c86'],\n",
       " 43: ['c5', 'c28', 'c65'],\n",
       " 44: ['c26', 'c53', 'c81'],\n",
       " 45: ['c27', 'c59', 'c71'],\n",
       " 46: ['c9', 'c33', 'c46'],\n",
       " 47: ['c8', 'c61', 'c82'],\n",
       " 48: ['c43', 'c64', 'c92'],\n",
       " 49: ['c45', 'c78', 'c88'],\n",
       " 50: ['c25', 'c42', 'c80'],\n",
       " 51: ['c2', 'c48', 'c85'],\n",
       " 52: ['c6', 'c23', 'c57'],\n",
       " 53: ['c15', 'c55', 'c68'],\n",
       " 54: ['c1', 'c44', 'c47'],\n",
       " 55: ['c12', 'c51', 'c56'],\n",
       " 56: ['c39', 'c73', 'c91'],\n",
       " 57: ['c11', 'c67', 'c86'],\n",
       " 58: ['c31', 'c53', 'c70'],\n",
       " 59: ['c34', 'c62', 'c83'],\n",
       " 60: ['c14', 'c37', 'c40'],\n",
       " 61: ['c5', 'c32', 'c49'],\n",
       " 62: ['c21', 'c27', 'c50'],\n",
       " 63: ['c13', 'c16', 'c41'],\n",
       " 64: ['c2', 'c74', 'c77'],\n",
       " 65: ['c7', 'c33', 'c58'],\n",
       " 66: ['c36', 'c87', 'c94'],\n",
       " 67: ['c64', 'c75', 'c93'],\n",
       " 68: ['c4', 'c17', 'c22'],\n",
       " 69: ['c29', 'c38', 'c82'],\n",
       " 70: ['c46', 'c60', 'c63'],\n",
       " 71: ['c54', 'c65', 'c79'],\n",
       " 72: ['c10', 'c69', 'c76'],\n",
       " 73: ['c26', 'c31', 'c80'],\n",
       " 74: ['c3', 'c48', 'c52'],\n",
       " 75: ['c8', 'c25', 'c73'],\n",
       " 76: ['c34', 'c49', 'c90'],\n",
       " 77: ['c5', 'c38', 'c72'],\n",
       " 78: ['c29', 'c32', 'c59'],\n",
       " 79: ['c17', 'c24', 'c45'],\n",
       " 80: ['c7', 'c13', 'c20'],\n",
       " 81: ['c33', 'c34', 'c36'],\n",
       " 82: ['c54', 'c76', 'c84'],\n",
       " 83: ['c22', 'c37', 'c71'],\n",
       " 84: ['c39', 'c56', 'c82'],\n",
       " 85: ['c46', 'c79', 'c89'],\n",
       " 86: ['c43', 'c58', 'c88'],\n",
       " 87: ['c66', 'c67', 'c74'],\n",
       " 88: ['c21', 'c68', 'c92'],\n",
       " 89: ['c15', 'c44', 'c50'],\n",
       " 90: ['c47', 'c65', 'c83'],\n",
       " 91: ['c5', 'c25', 'c85'],\n",
       " 92: ['c45', 'c63', 'c86'],\n",
       " 93: ['c11', 'c60', 'c70'],\n",
       " 94: ['c9', 'c18', 'c26'],\n",
       " 95: ['c3', 'c62', 'c69'],\n",
       " 96: ['c32', 'c38', 'c93'],\n",
       " 97: ['c13', 'c59', 'c72'],\n",
       " 98: ['c2', 'c80', 'c89'],\n",
       " 99: ['c8', 'c27', 'c53'],\n",
       " 100: ['c6', 'c28', 'c66'],\n",
       " 101: ['c49', 'c78', 'c87'],\n",
       " 102: ['c1', 'c12', 'c73'],\n",
       " 103: ['c15', 'c17', 'c77'],\n",
       " 104: ['c0', 'c10', 'c75'],\n",
       " 105: ['c56', 'c90', 'c91'],\n",
       " 106: ['c16', 'c52', 'c64'],\n",
       " 107: ['c4', 'c19', 'c48'],\n",
       " 108: ['c30', 'c31', 'c39'],\n",
       " 109: ['c14', 'c57', 'c92'],\n",
       " 111: ['c7', 'c25', 'c86'],\n",
       " 112: ['c21', 'c23', 'c71'],\n",
       " 113: ['c28', 'c80', 'c84'],\n",
       " 114: ['c40', 'c47', 'c70'],\n",
       " 115: ['c6', 'c13', 'c94'],\n",
       " 116: ['c3', 'c41', 'c42'],\n",
       " 117: ['c15', 'c35', 'c51'],\n",
       " 118: ['c36', 'c67', 'c93'],\n",
       " 119: ['c18', 'c53', 'c55'],\n",
       " 120: ['c2', 'c44', 'c68'],\n",
       " 121: ['c20', 'c66', 'c85'],\n",
       " 122: ['c11', 'c63', 'c76'],\n",
       " 123: ['c50', 'c61', 'c77'],\n",
       " 124: ['c8', 'c26', 'c29'],\n",
       " 125: ['c49', 'c54', 'c69'],\n",
       " 126: ['c33', 'c37', 'c75'],\n",
       " 127: ['c12', 'c62', 'c90'],\n",
       " 129: ['c52', 'c60', 'c83'],\n",
       " 130: ['c21', 'c43', 'c57'],\n",
       " 131: ['c22', 'c27', 'c91'],\n",
       " 132: ['c65', 'c81', 'c87'],\n",
       " 133: ['c42', 'c46', 'c82'],\n",
       " 134: ['c9', 'c23', 'c40'],\n",
       " 135: ['c14', 'c31', 'c55'],\n",
       " 136: ['c30', 'c41', 'c74'],\n",
       " 137: ['c0', 'c35', 'c71'],\n",
       " 138: ['c56', 'c61', 'c89'],\n",
       " 139: ['c24', 'c78', 'c80'],\n",
       " 140: ['c5', 'c48', 'c58'],\n",
       " 141: ['c10', 'c16', 'c25'],\n",
       " 142: ['c2', 'c34', 'c73'],\n",
       " 143: ['c72', 'c79', 'c93'],\n",
       " 144: ['c4', 'c45', 'c59'],\n",
       " 145: ['c17', 'c28', 'c64'],\n",
       " 146: ['c1', 'c6', 'c32'],\n",
       " 147: ['c57', 'c86', 'c94'],\n",
       " 148: ['c36', 'c43', 'c51'],\n",
       " 149: ['c84', 'c85', 'c88'],\n",
       " 150: ['c19', 'c38', 'c77'],\n",
       " 151: ['c20', 'c61', 'c81'],\n",
       " 546: ['c36', 'c68', 'c83'],\n",
       " 547: ['c1', 'c26', 'c34'],\n",
       " 548: ['c31', 'c32', 'c88'],\n",
       " 549: ['c4', 'c51', 'c79'],\n",
       " 550: ['c3', 'c82', 'c90'],\n",
       " 551: ['c22', 'c61', 'c84'],\n",
       " 552: ['c38', 'c41', 'c91'],\n",
       " 553: ['c20', 'c62', 'c77'],\n",
       " 554: ['c27', 'c29', 'c45'],\n",
       " 555: ['c37', 'c53', 'c72'],\n",
       " 556: ['c6', 'c43', 'c86'],\n",
       " 557: ['c8', 'c71', 'c89'],\n",
       " 558: ['c25', 'c39', 'c49'],\n",
       " 559: ['c9', 'c80', 'c85'],\n",
       " 560: ['c2', 'c28', 'c57'],\n",
       " 561: ['c59', 'c63', 'c65'],\n",
       " 562: ['c30', 'c70', 'c81'],\n",
       " 563: ['c73', 'c75', 'c76'],\n",
       " 564: ['c5', 'c69', 'c79'],\n",
       " 565: ['c15', 'c54', 'c87'],\n",
       " 566: ['c11', 'c47', 'c74'],\n",
       " 567: ['c19', 'c33', 'c66'],\n",
       " 568: ['c13', 'c14', 'c48'],\n",
       " 569: ['c0', 'c21', 'c64'],\n",
       " 570: ['c17', 'c35', 'c55'],\n",
       " 571: ['c23', 'c44', 'c52'],\n",
       " 572: ['c60', 'c68', 'c88'],\n",
       " 573: ['c50', 'c56', 'c94'],\n",
       " 574: ['c12', 'c16', 'c24'],\n",
       " 575: ['c10', 'c67', 'c78'],\n",
       " 576: ['c46', 'c49', 'c92'],\n",
       " 577: ['c36', 'c47', 'c58'],\n",
       " 579: ['c2', 'c42', 'c81'],\n",
       " 580: ['c7', 'c45', 'c65'],\n",
       " 581: ['c40', 'c54', 'c72'],\n",
       " 582: ['c5', 'c31', 'c37'],\n",
       " 583: ['c21', 'c22', 'c25'],\n",
       " 584: ['c13', 'c66', 'c76'],\n",
       " 585: ['c6', 'c27', 'c75'],\n",
       " 586: ['c32', 'c86', 'c89'],\n",
       " 587: ['c1', 'c57', 'c63'],\n",
       " 588: ['c17', 'c20', 'c90'],\n",
       " 589: ['c26', 'c52', 'c77'],\n",
       " 590: ['c8', 'c12', 'c38'],\n",
       " 591: ['c3', 'c87', 'c91'],\n",
       " 592: ['c59', 'c73', 'c82'],\n",
       " 593: ['c28', 'c33', 'c85'],\n",
       " 594: ['c9', 'c41', 'c53'],\n",
       " 595: ['c15', 'c29', 'c61'],\n",
       " 596: ['c71', 'c83', 'c84'],\n",
       " 597: ['c7', 'c39', 'c74'],\n",
       " 598: ['c51', 'c60', 'c64'],\n",
       " 599: ['c0', 'c48', 'c69'],\n",
       " 600: ['c10', 'c14', 'c56'],\n",
       " 601: ['c44', 'c55', 'c94'],\n",
       " 602: ['c16', 'c23', 'c42'],\n",
       " 603: ['c50', 'c67', 'c92'],\n",
       " 604: ['c4', 'c24', 'c58'],\n",
       " 605: ['c11', 'c30', 'c93'],\n",
       " 606: ['c18', 'c43', 'c80'],\n",
       " 607: ['c40', 'c78', 'c79'],\n",
       " 608: ['c19', 'c46', 'c68'],\n",
       " 609: ['c34', 'c35', 'c88'],\n",
       " 610: ['c1', 'c62', 'c70'],\n",
       " 251: ['c10', 'c48', 'c93'],\n",
       " 252: ['c11', 'c28', 'c73'],\n",
       " 253: ['c12', 'c29', 'c86'],\n",
       " 255: ['c13', 'c26', 'c74'],\n",
       " 256: ['c37', 'c55', 'c84'],\n",
       " 257: ['c32', 'c40', 'c44'],\n",
       " 258: ['c4', 'c64', 'c67'],\n",
       " 259: ['c24', 'c46', 'c51'],\n",
       " 260: ['c0', 'c36', 'c92'],\n",
       " 261: ['c2', 'c56', 'c78'],\n",
       " 262: ['c16', 'c27', 'c47'],\n",
       " 263: ['c42', 'c53', 'c89'],\n",
       " 264: ['c18', 'c21', 'c82'],\n",
       " 265: ['c8', 'c37', 'c68'],\n",
       " 266: ['c17', 'c43', 'c62'],\n",
       " 267: ['c49', 'c61', 'c71'],\n",
       " 268: ['c26', 'c63', 'c87'],\n",
       " 269: ['c38', 'c73', 'c90'],\n",
       " 270: ['c30', 'c66', 'c91'],\n",
       " 271: ['c13', 'c65', 'c75'],\n",
       " 272: ['c5', 'c59', 'c80'],\n",
       " 273: ['c28', 'c45', 'c60'],\n",
       " 274: ['c3', 'c79', 'c84'],\n",
       " 212: ['c31', 'c34', 'c86'],\n",
       " 213: ['c15', 'c39', 'c52'],\n",
       " 214: ['c9', 'c77', 'c83'],\n",
       " 215: ['c0', 'c57', 'c58'],\n",
       " 216: ['c4', 'c29', 'c44'],\n",
       " 217: ['c7', 'c51', 'c69'],\n",
       " 218: ['c10', 'c23', 'c46'],\n",
       " 219: ['c16', 'c18', 'c92'],\n",
       " 220: ['c19', 'c93', 'c94'],\n",
       " 221: ['c40', 'c50', 'c64'],\n",
       " 222: ['c14', 'c81', 'c88'],\n",
       " 223: ['c22', 'c41', 'c76'],\n",
       " 224: ['c20', 'c25', 'c32'],\n",
       " 225: ['c54', 'c70', 'c74'],\n",
       " 226: ['c6', 'c12', 'c72'],\n",
       " 228: ['c55', 'c78', 'c85'],\n",
       " 229: ['c1', 'c11', 'c33'],\n",
       " 230: ['c14', 'c24', 'c67'],\n",
       " 231: ['c35', 'c49', 'c58'],\n",
       " 232: ['c39', 'c43', 'c48'],\n",
       " 233: ['c7', 'c19', 'c47'],\n",
       " 234: ['c23', 'c50', 'c53'],\n",
       " 235: ['c21', 'c69', 'c94'],\n",
       " 236: ['c57', 'c66', 'c90'],\n",
       " 237: ['c60', 'c71', 'c80'],\n",
       " 238: ['c9', 'c52', 'c65'],\n",
       " 239: ['c3', 'c15', 'c36'],\n",
       " 240: ['c31', 'c76', 'c89'],\n",
       " 241: ['c5', 'c22', 'c82'],\n",
       " 242: ['c8', 'c63', 'c72'],\n",
       " 243: ['c2', 'c54', 'c62'],\n",
       " 244: ['c27', 'c61', 'c87'],\n",
       " 245: ['c34', 'c38', 'c68'],\n",
       " 246: ['c56', 'c83', 'c88'],\n",
       " 247: ['c59', 'c77', 'c79'],\n",
       " 248: ['c17', 'c75', 'c91'],\n",
       " 249: ['c20', 'c30', 'c45'],\n",
       " 250: ['c35', 'c42', 'c48'],\n",
       " 152: ['c9', 'c61', 'c73'],\n",
       " 153: ['c33', 'c38', 'c67'],\n",
       " 154: ['c19', 'c25', 'c51'],\n",
       " 155: ['c0', 'c4', 'c85'],\n",
       " 156: ['c53', 'c63', 'c69'],\n",
       " 157: ['c26', 'c27', 'c60'],\n",
       " 158: ['c48', 'c59', 'c75'],\n",
       " 159: ['c31', 'c45', 'c56'],\n",
       " 160: ['c2', 'c20', 'c88'],\n",
       " 161: ['c22', 'c43', 'c89'],\n",
       " 162: ['c23', 'c74', 'c82'],\n",
       " 163: ['c58', 'c66', 'c83'],\n",
       " 164: ['c16', 'c72', 'c87'],\n",
       " 165: ['c8', 'c35', 'c64'],\n",
       " 166: ['c10', 'c24', 'c29'],\n",
       " 168: ['c18', 'c67', 'c85'],\n",
       " 169: ['c1', 'c14', 'c19'],\n",
       " 170: ['c6', 'c46', 'c70'],\n",
       " 171: ['c7', 'c55', 'c93'],\n",
       " 172: ['c4', 'c42', 'c50'],\n",
       " 173: ['c25', 'c40', 'c81'],\n",
       " 174: ['c41', 'c78', 'c94'],\n",
       " 175: ['c33', 'c44', 'c92'],\n",
       " 176: ['c23', 'c30', 'c63'],\n",
       " 177: ['c12', 'c54', 'c77'],\n",
       " 178: ['c17', 'c52', 'c69'],\n",
       " 179: ['c3', 'c21', 'c37'],\n",
       " 180: ['c49', 'c62', 'c86'],\n",
       " 181: ['c65', 'c84', 'c91'],\n",
       " 182: ['c28', 'c76', 'c79'],\n",
       " 183: ['c32', 'c71', 'c90'],\n",
       " 184: ['c5', 'c27', 'c68'],\n",
       " 185: ['c39', 'c61', 'c80'],\n",
       " 186: ['c9', 'c15', 'c47'],\n",
       " 187: ['c0', 'c13', 'c51'],\n",
       " 188: ['c36', 'c57', 'c73'],\n",
       " 189: ['c16', 'c30', 'c38'],\n",
       " 190: ['c11', 'c34', 'c46'],\n",
       " 191: ['c18', 'c35', 'c44'],\n",
       " 192: ['c14', 'c70', 'c93'],\n",
       " 193: ['c6', 'c24', 'c41'],\n",
       " 194: ['c10', 'c55', 'c92'],\n",
       " 195: ['c1', 'c42', 'c58'],\n",
       " 196: ['c26', 'c64', 'c78'],\n",
       " 197: ['c31', 'c81', 'c94'],\n",
       " 198: ['c7', 'c48', 'c50'],\n",
       " 199: ['c40', 'c45', 'c84'],\n",
       " 200: ['c71', 'c74', 'c75'],\n",
       " 201: ['c37', 'c82', 'c83'],\n",
       " 202: ['c49', 'c53', 'c65'],\n",
       " 203: ['c17', 'c56', 'c60'],\n",
       " 204: ['c8', 'c79', 'c88'],\n",
       " 205: ['c34', 'c54', 'c89'],\n",
       " 206: ['c5', 'c15', 'c66'],\n",
       " 207: ['c29', 'c43', 'c76'],\n",
       " 208: ['c11', 'c21', 'c72'],\n",
       " 209: ['c12', 'c20', 'c59'],\n",
       " 210: ['c22', 'c39', 'c86'],\n",
       " 211: ['c2', 'c32', 'c87'],\n",
       " 611: ['c1', 'c7', 'c56'],\n",
       " 612: ['c37', 'c38', 'c66'],\n",
       " 613: ['c17', 'c63', 'c88'],\n",
       " 614: ['c30', 'c65', 'c92'],\n",
       " 615: ['c5', 'c43', 'c67'],\n",
       " 616: ['c55', 'c70', 'c71'],\n",
       " 617: ['c13', 'c39', 'c85'],\n",
       " 618: ['c24', 'c33', 'c54'],\n",
       " 619: ['c12', 'c47', 'c80'],\n",
       " 620: ['c0', 'c3', 'c81'],\n",
       " 621: ['c18', 'c28', 'c62'],\n",
       " 622: ['c25', 'c68', 'c94'],\n",
       " 623: ['c41', 'c51', 'c52'],\n",
       " 624: ['c16', 'c58', 'c78'],\n",
       " 625: ['c14', 'c64', 'c77'],\n",
       " 626: ['c4', 'c23', 'c93'],\n",
       " 627: ['c6', 'c19', 'c35'],\n",
       " 628: ['c10', 'c57', 'c91'],\n",
       " 629: ['c29', 'c46', 'c48'],\n",
       " 630: ['c15', 'c59', 'c90'],\n",
       " 631: ['c9', 'c20', 'c87'],\n",
       " 632: ['c8', 'c22', 'c45'],\n",
       " 633: ['c2', 'c75', 'c84'],\n",
       " 634: ['c21', 'c32', 'c79'],\n",
       " 635: ['c56', 'c69', 'c73'],\n",
       " 636: ['c53', 'c74', 'c83'],\n",
       " 637: ['c1', 'c31', 'c50'],\n",
       " 638: ['c7', 'c26', 'c44'],\n",
       " 639: ['c36', 'c42', 'c72'],\n",
       " 640: ['c11', 'c27', 'c55'],\n",
       " 641: ['c61', 'c76', 'c86'],\n",
       " 642: ['c49', 'c82', 'c89'],\n",
       " 643: ['c40', 'c46', 'c90'],\n",
       " 644: ['c3', 'c34', 'c60'],\n",
       " 645: ['c47', 'c81', 'c93'],\n",
       " 646: ['c16', 'c51', 'c77'],\n",
       " 647: ['c4', 'c68', 'c78'],\n",
       " 648: ['c67', 'c80', 'c91'],\n",
       " 649: ['c0', 'c28', 'c94'],\n",
       " 650: ['c52', 'c58', 'c92'],\n",
       " 651: ['c10', 'c35', 'c70'],\n",
       " 652: ['c23', 'c24', 'c36'],\n",
       " 653: ['c13', 'c42', 'c87'],\n",
       " 654: ['c38', 'c44', 'c85'],\n",
       " 655: ['c15', 'c19', 'c57'],\n",
       " 656: ['c6', 'c33', 'c73'],\n",
       " 657: ['c21', 'c40', 'c62'],\n",
       " 658: ['c25', 'c41', 'c63'],\n",
       " 659: ['c18', 'c50', 'c54'],\n",
       " 660: ['c14', 'c20', 'c74'],\n",
       " 661: ['c34', 'c64', 'c66'],\n",
       " 662: ['c8', 'c30', 'c43'],\n",
       " 663: ['c17', 'c48', 'c79'],\n",
       " 664: ['c2', 'c39', 'c45'],\n",
       " 665: ['c53', 'c71', 'c88'],\n",
       " 666: ['c5', 'c60', 'c75'],\n",
       " 667: ['c12', 'c37', 'c65'],\n",
       " 275: ['c0', 'c46', 'c80'],\n",
       " 277: ['c23', 'c55', 'c69'],\n",
       " 278: ['c1', 'c51', 'c53'],\n",
       " 279: ['c4', 'c10', 'c26'],\n",
       " 280: ['c24', 'c27', 'c64'],\n",
       " 281: ['c47', 'c72', 'c78'],\n",
       " 282: ['c9', 'c62', 'c76'],\n",
       " 283: ['c11', 'c36', 'c52'],\n",
       " 284: ['c29', 'c42', 'c84'],\n",
       " 285: ['c16', 'c57', 'c89'],\n",
       " 286: ['c7', 'c41', 'c68'],\n",
       " 287: ['c35', 'c80', 'c81'],\n",
       " 288: ['c33', 'c61', 'c94'],\n",
       " 289: ['c6', 'c58', 'c59'],\n",
       " 290: ['c3', 'c40', 'c49'],\n",
       " 291: ['c13', 'c32', 'c70'],\n",
       " 292: ['c86', 'c91', 'c92'],\n",
       " 293: ['c48', 'c66', 'c82'],\n",
       " 295: ['c21', 'c77', 'c88'],\n",
       " 296: ['c8', 'c28', 'c74'],\n",
       " 297: ['c18', 'c37', 'c45'],\n",
       " 298: ['c10', 'c81', 'c83'],\n",
       " 299: ['c19', 'c31', 'c65'],\n",
       " 300: ['c25', 'c60', 'c90'],\n",
       " 301: ['c14', 'c59', 'c68'],\n",
       " 302: ['c2', 'c72', 'c94'],\n",
       " 303: ['c0', 'c55', 'c56'],\n",
       " 304: ['c22', 'c69', 'c93'],\n",
       " 305: ['c15', 'c34', 'c76'],\n",
       " 306: ['c3', 'c38', 'c39'],\n",
       " 307: ['c30', 'c79', 'c87'],\n",
       " 308: ['c5', 'c20', 'c54'],\n",
       " 309: ['c17', 'c71', 'c73'],\n",
       " 310: ['c43', 'c75', 'c83'],\n",
       " 311: ['c12', 'c50', 'c63'],\n",
       " 312: ['c23', 'c77', 'c85'],\n",
       " 313: ['c14', 'c44', 'c82'],\n",
       " 314: ['c28', 'c46', 'c67'],\n",
       " 315: ['c2', 'c26', 'c92'],\n",
       " 316: ['c25', 'c52', 'c93'],\n",
       " 317: ['c11', 'c24', 'c42'],\n",
       " 318: ['c33', 'c64', 'c91'],\n",
       " 319: ['c4', 'c84', 'c86'],\n",
       " 320: ['c41', 'c47', 'c89'],\n",
       " 321: ['c1', 'c18', 'c49'],\n",
       " 322: ['c7', 'c57', 'c70'],\n",
       " 324: ['c32', 'c61', 'c78'],\n",
       " 325: ['c31', 'c44', 'c62'],\n",
       " 326: ['c19', 'c29', 'c67'],\n",
       " 327: ['c16', 'c36', 'c40'],\n",
       " 328: ['c50', 'c58', 'c90'],\n",
       " 329: ['c6', 'c53', 'c85'],\n",
       " 330: ['c9', 'c51', 'c74'],\n",
       " 332: ['c20', 'c27', 'c48'],\n",
       " 333: ['c13', 'c45', 'c73'],\n",
       " 334: ['c21', 'c30', 'c35'],\n",
       " 335: ['c39', 'c41', 'c92'],\n",
       " 336: ['c27', 'c43', 'c72'],\n",
       " 337: ['c4', 'c34', 'c37'],\n",
       " 338: ['c3', 'c55', 'c88'],\n",
       " 339: ['c14', 'c16', 'c29'],\n",
       " 340: ['c19', 'c23', 'c90'],\n",
       " 341: ['c7', 'c12', 'c81'],\n",
       " 342: ['c0', 'c24', 'c65'],\n",
       " 343: ['c9', 'c70', 'c78'],\n",
       " 344: ['c8', 'c80', 'c87'],\n",
       " 345: ['c36', 'c54', 'c75'],\n",
       " 347: ['c32', 'c50', 'c84'],\n",
       " 348: ['c39', 'c47', 'c68'],\n",
       " 349: ['c6', 'c61', 'c67'],\n",
       " 350: ['c64', 'c89', 'c90'],\n",
       " 351: ['c59', 'c76', 'c91'],\n",
       " 352: ['c38', 'c92', 'c94'],\n",
       " 353: ['c17', 'c21', 'c58'],\n",
       " 354: ['c22', 'c49', 'c73'],\n",
       " 355: ['c45', 'c46', 'c69'],\n",
       " 356: ['c3', 'c74', 'c93'],\n",
       " 357: ['c20', 'c42', 'c79'],\n",
       " 358: ['c5', 'c8', 'c56'],\n",
       " 359: ['c24', 'c70', 'c77'],\n",
       " 360: ['c26', 'c40', 'c43'],\n",
       " 361: ['c12', 'c32', 'c83'],\n",
       " 362: ['c28', 'c51', 'c89'],\n",
       " 363: ['c52', 'c63', 'c82'],\n",
       " 364: ['c18', 'c31', 'c66'],\n",
       " 365: ['c57', 'c72', 'c85'],\n",
       " 366: ['c1', 'c37', 'c86'],\n",
       " 367: ['c22', 'c35', 'c62'],\n",
       " 368: ['c30', 'c48', 'c78'],\n",
       " 369: ['c2', 'c52', 'c53'],\n",
       " 370: ['c10', 'c11', 'c71'],\n",
       " 371: ['c13', 'c25', 'c27'],\n",
       " 372: ['c15', 'c33', 'c60'],\n",
       " 373: ['c41', 'c56', 'c58'],\n",
       " 374: ['c44', 'c46', 'c77'],\n",
       " 375: ['c5', 'c17', 'c83'],\n",
       " 376: ['c42', 'c49', 'c64'],\n",
       " 377: ['c47', 'c67', 'c79'],\n",
       " 378: ['c69', 'c81', 'c84'],\n",
       " 379: ['c51', 'c61', 'c85'],\n",
       " 380: ['c0', 'c40', 'c59'],\n",
       " 381: ['c16', 'c66', 'c86'],\n",
       " 382: ['c35', 'c68', 'c76'],\n",
       " 383: ['c38', 'c55', 'c80'],\n",
       " 384: ['c6', 'c88', 'c93'],\n",
       " 385: ['c7', 'c29', 'c54'],\n",
       " 386: ['c4', 'c18', 'c87'],\n",
       " 387: ['c11', 'c15', 'c65'],\n",
       " 388: ['c9', 'c14', 'c50'],\n",
       " 389: ['c44', 'c63', 'c75'],\n",
       " 390: ['c19', 'c36', 'c91'],\n",
       " 391: ['c23', 'c34', 'c94'],\n",
       " 491: ['c43', 'c61', 'c79'],\n",
       " 492: ['c40', 'c65', 'c94'],\n",
       " 493: ['c16', 'c19', 'c34'],\n",
       " 494: ['c7', 'c21', 'c36'],\n",
       " 495: ['c15', 'c64', 'c80'],\n",
       " 496: ['c12', 'c45', 'c89'],\n",
       " 497: ['c18', 'c86', 'c90'],\n",
       " 498: ['c51', 'c55', 'c72'],\n",
       " 499: ['c10', 'c52', 'c73'],\n",
       " 500: ['c13', 'c28', 'c69'],\n",
       " 501: ['c24', 'c25', 'c31'],\n",
       " 502: ['c60', 'c62', 'c82'],\n",
       " 503: ['c14', 'c26', 'c38'],\n",
       " 504: ['c71', 'c85', 'c92'],\n",
       " 505: ['c0', 'c37', 'c88'],\n",
       " 506: ['c9', 'c27', 'c81'],\n",
       " 507: ['c42', 'c61', 'c83'],\n",
       " 508: ['c3', 'c65', 'c70'],\n",
       " 509: ['c53', 'c58', 'c94'],\n",
       " 510: ['c43', 'c66', 'c93'],\n",
       " 511: ['c10', 'c41', 'c82'],\n",
       " 512: ['c46', 'c76', 'c78'],\n",
       " 513: ['c4', 'c56', 'c77'],\n",
       " 514: ['c44', 'c64', 'c73'],\n",
       " 515: ['c0', 'c29', 'c34'],\n",
       " 516: ['c7', 'c22', 'c60'],\n",
       " 517: ['c13', 'c31', 'c33'],\n",
       " 518: ['c6', 'c30', 'c47'],\n",
       " 519: ['c35', 'c39', 'c75'],\n",
       " 520: ['c49', 'c57', 'c84'],\n",
       " 521: ['c1', 'c8', 'c48'],\n",
       " 523: ['c2', 'c40', 'c91'],\n",
       " 524: ['c54', 'c63', 'c67'],\n",
       " 525: ['c41', 'c50', 'c59'],\n",
       " 526: ['c5', 'c23', 'c29'],\n",
       " 527: ['c32', 'c68', 'c74'],\n",
       " 528: ['c11', 'c17', 'c87'],\n",
       " 529: ['c1', 'c20', 'c22'],\n",
       " 530: ['c26', 'c33', 'c71'],\n",
       " 531: ['c24', 'c44', 'c79'],\n",
       " 532: ['c25', 'c28', 'c70'],\n",
       " 533: ['c46', 'c57', 'c62'],\n",
       " 534: ['c51', 'c58', 'c84'],\n",
       " 535: ['c6', 'c21', 'c45'],\n",
       " 536: ['c35', 'c47', 'c86'],\n",
       " 537: ['c83', 'c89', 'c93'],\n",
       " 538: ['c12', 'c36', 'c39'],\n",
       " 539: ['c18', 'c69', 'c77'],\n",
       " 540: ['c74', 'c88', 'c90'],\n",
       " 541: ['c19', 'c85', 'c87'],\n",
       " 542: ['c38', 'c52', 'c78'],\n",
       " 543: ['c4', 'c27', 'c80'],\n",
       " 544: ['c15', 'c72', 'c81'],\n",
       " 545: ['c16', 'c54', 'c55'],\n",
       " 437: ['c15', 'c18', 'c74'],\n",
       " 438: ['c5', 'c35', 'c92'],\n",
       " 439: ['c17', 'c33', 'c47'],\n",
       " 440: ['c13', 'c58', 'c82'],\n",
       " 441: ['c8', 'c50', 'c62'],\n",
       " 442: ['c28', 'c48', 'c71'],\n",
       " 444: ['c14', 'c32', 'c42'],\n",
       " 445: ['c3', 'c9', 'c30'],\n",
       " 446: ['c37', 'c56', 'c63'],\n",
       " 447: ['c26', 'c66', 'c75'],\n",
       " 448: ['c53', 'c57', 'c67'],\n",
       " 449: ['c40', 'c60', 'c69'],\n",
       " 450: ['c11', 'c39', 'c79'],\n",
       " 451: ['c10', 'c38', 'c87'],\n",
       " 452: ['c52', 'c90', 'c94'],\n",
       " 453: ['c8', 'c20', 'c76'],\n",
       " 454: ['c21', 'c59', 'c83'],\n",
       " 455: ['c22', 'c68', 'c77'],\n",
       " 456: ['c1', 'c25', 'c54'],\n",
       " 457: ['c16', 'c61', 'c93'],\n",
       " 458: ['c31', 'c48', 'c74'],\n",
       " 459: ['c36', 'c45', 'c64'],\n",
       " 460: ['c72', 'c73', 'c88'],\n",
       " 461: ['c23', 'c35', 'c67'],\n",
       " 462: ['c7', 'c15', 'c85'],\n",
       " 463: ['c12', 'c19', 'c27'],\n",
       " 464: ['c70', 'c80', 'c86'],\n",
       " 465: ['c9', 'c68', 'c91'],\n",
       " 466: ['c29', 'c51', 'c78'],\n",
       " 467: ['c0', 'c44', 'c76'],\n",
       " 468: ['c24', 'c34', 'c81'],\n",
       " 469: ['c41', 'c46', 'c55'],\n",
       " 470: ['c4', 'c6', 'c89'],\n",
       " 471: ['c43', 'c59', 'c84'],\n",
       " 472: ['c18', 'c25', 'c65'],\n",
       " 473: ['c2', 'c10', 'c49'],\n",
       " 474: ['c23', 'c60', 'c91'],\n",
       " 475: ['c11', 'c20', 'c31'],\n",
       " 476: ['c62', 'c66', 'c92'],\n",
       " 477: ['c22', 'c42', 'c75'],\n",
       " 478: ['c3', 'c13', 'c57'],\n",
       " 479: ['c17', 'c53', 'c82'],\n",
       " 480: ['c28', 'c30', 'c37'],\n",
       " 481: ['c1', 'c71', 'c78'],\n",
       " 482: ['c26', 'c32', 'c47'],\n",
       " 483: ['c14', 'c33', 'c52'],\n",
       " 484: ['c49', 'c50', 'c93'],\n",
       " 485: ['c2', 'c5', 'c36'],\n",
       " 486: ['c39', 'c63', 'c83'],\n",
       " 487: ['c40', 'c56', 'c86'],\n",
       " 488: ['c43', 'c73', 'c94'],\n",
       " 489: ['c27', 'c38', 'c58'],\n",
       " 758: ['c20', 'c51', 'c71'],\n",
       " 759: ['c22', 'c53', 'c90'],\n",
       " 760: ['c0', 'c12', 'c33'],\n",
       " 761: ['c21', 'c76', 'c85'],\n",
       " 762: ['c32', 'c48', 'c77'],\n",
       " 763: ['c55', 'c59', 'c61'],\n",
       " 764: ['c5', 'c7', 'c46'],\n",
       " 765: ['c3', 'c6', 'c92'],\n",
       " 766: ['c9', 'c39', 'c64'],\n",
       " 767: ['c41', 'c67', 'c73'],\n",
       " 768: ['c35', 'c45', 'c79'],\n",
       " 769: ['c14', 'c47', 'c54'],\n",
       " 770: ['c11', 'c44', 'c69'],\n",
       " 772: ['c17', 'c37', 'c81'],\n",
       " 773: ['c23', 'c62', 'c65'],\n",
       " 774: ['c10', 'c19', 'c30'],\n",
       " 775: ['c4', 'c72', 'c75'],\n",
       " 776: ['c1', 'c2', 'c60'],\n",
       " 777: ['c25', 'c50', 'c87'],\n",
       " 778: ['c16', 'c71', 'c82'],\n",
       " 779: ['c29', 'c56', 'c68'],\n",
       " 780: ['c13', 'c43', 'c81'],\n",
       " 781: ['c34', 'c42', 'c70'],\n",
       " 782: ['c51', 'c66', 'c80'],\n",
       " 783: ['c8', 'c31', 'c57'],\n",
       " 784: ['c5', 'c52', 'c86'],\n",
       " 785: ['c46', 'c83', 'c94'],\n",
       " 786: ['c15', 'c58', 'c91'],\n",
       " 787: ['c18', 'c20', 'c93'],\n",
       " 788: ['c28', 'c36', 'c38'],\n",
       " 789: ['c13', 'c84', 'c89'],\n",
       " 790: ['c26', 'c49', 'c88'],\n",
       " 791: ['c27', 'c74', 'c78'],\n",
       " 792: ['c24', 'c40', 'c63'],\n",
       " 793: ['c59', 'c69', 'c92'],\n",
       " 794: ['c2', 'c33', 'c70'],\n",
       " 795: ['c32', 'c37', 'c85'],\n",
       " 796: ['c42', 'c54', 'c68'],\n",
       " 797: ['c19', 'c45', 'c55'],\n",
       " 798: ['c6', 'c10', 'c60'],\n",
       " 799: ['c21', 'c65', 'c80'],\n",
       " 800: ['c3', 'c12', 'c53'],\n",
       " 801: ['c0', 'c23', 'c25'],\n",
       " 802: ['c7', 'c64', 'c72'],\n",
       " 803: ['c9', 'c16', 'c28'],\n",
       " 804: ['c48', 'c87', 'c88'],\n",
       " 805: ['c41', 'c57', 'c79'],\n",
       " 806: ['c14', 'c61', 'c75'],\n",
       " 807: ['c8', 'c11', 'c91'],\n",
       " 808: ['c76', 'c77', 'c90'],\n",
       " 809: ['c22', 'c56', 'c67'],\n",
       " 704: ['c40', 'c48', 'c53'],\n",
       " 705: ['c15', 'c71', 'c79'],\n",
       " 706: ['c45', 'c77', 'c92'],\n",
       " 707: ['c22', 'c32', 'c51'],\n",
       " 708: ['c12', 'c21', 'c44'],\n",
       " 709: ['c17', 'c34', 'c74'],\n",
       " 710: ['c1', 'c24', 'c66'],\n",
       " 711: ['c30', 'c50', 'c89'],\n",
       " 712: ['c4', 'c36', 'c82'],\n",
       " 713: ['c5', 'c73', 'c87'],\n",
       " 714: ['c28', 'c42', 'c55'],\n",
       " 715: ['c11', 'c54', 'c58'],\n",
       " 716: ['c48', 'c67', 'c72'],\n",
       " 717: ['c20', 'c52', 'c68'],\n",
       " 718: ['c9', 'c35', 'c59'],\n",
       " 719: ['c46', 'c53', 'c56'],\n",
       " 720: ['c3', 'c25', 'c29'],\n",
       " 721: ['c26', 'c61', 'c62'],\n",
       " 722: ['c31', 'c49', 'c83'],\n",
       " 723: ['c37', 'c47', 'c91'],\n",
       " 724: ['c19', 'c22', 'c78'],\n",
       " 725: ['c21', 'c66', 'c84'],\n",
       " 726: ['c0', 'c1', 'c41'],\n",
       " 727: ['c15', 'c27', 'c69'],\n",
       " 728: ['c10', 'c65', 'c85'],\n",
       " 729: ['c2', 'c8', 'c13'],\n",
       " 730: ['c59', 'c70', 'c87'],\n",
       " 731: ['c6', 'c17', 'c26'],\n",
       " 732: ['c23', 'c75', 'c86'],\n",
       " 733: ['c31', 'c63', 'c92'],\n",
       " 734: ['c14', 'c79', 'c94'],\n",
       " 735: ['c16', 'c37', 'c90'],\n",
       " 736: ['c71', 'c76', 'c93'],\n",
       " 737: ['c33', 'c39', 'c57'],\n",
       " 738: ['c18', 'c64', 'c81'],\n",
       " 739: ['c38', 'c43', 'c60'],\n",
       " 740: ['c7', 'c80', 'c88'],\n",
       " 741: ['c6', 'c82', 'c84'],\n",
       " 742: ['c4', 'c63', 'c70'],\n",
       " 743: ['c13', 'c34', 'c40'],\n",
       " 744: ['c30', 'c51', 'c62'],\n",
       " 745: ['c11', 'c29', 'c49'],\n",
       " 746: ['c24', 'c50', 'c72'],\n",
       " 747: ['c44', 'c74', 'c89'],\n",
       " 748: ['c3', 'c73', 'c77'],\n",
       " 749: ['c9', 'c12', 'c42'],\n",
       " 750: ['c52', 'c56', 'c85'],\n",
       " 751: ['c32', 'c45', 'c58'],\n",
       " 752: ['c20', 'c23', 'c28'],\n",
       " 392: ['c55', 'c83', 'c91'],\n",
       " 393: ['c8', 'c24', 'c69'],\n",
       " 394: ['c0', 'c31', 'c60'],\n",
       " 395: ['c6', 'c34', 'c44'],\n",
       " 396: ['c29', 'c47', 'c90'],\n",
       " 397: ['c33', 'c81', 'c82'],\n",
       " 398: ['c4', 'c46', 'c65'],\n",
       " 399: ['c1', 'c27', 'c30'],\n",
       " 400: ['c17', 'c25', 'c67'],\n",
       " 401: ['c38', 'c61', 'c88'],\n",
       " 402: ['c50', 'c55', 'c57'],\n",
       " 403: ['c28', 'c52', 'c54'],\n",
       " 404: ['c59', 'c85', 'c89'],\n",
       " 405: ['c15', 'c45', 'c49'],\n",
       " 406: ['c14', 'c34', 'c43'],\n",
       " 407: ['c40', 'c76', 'c87'],\n",
       " 408: ['c13', 'c37', 'c78'],\n",
       " 409: ['c9', 'c58', 'c63'],\n",
       " 410: ['c11', 'c23', 'c92'],\n",
       " 411: ['c39', 'c42', 'c51'],\n",
       " 412: ['c8', 'c47', 'c75'],\n",
       " 413: ['c19', 'c62', 'c81'],\n",
       " 414: ['c4', 'c66', 'c69'],\n",
       " 415: ['c36', 'c41', 'c71'],\n",
       " 416: ['c16', 'c68', 'c73'],\n",
       " 417: ['c7', 'c10', 'c89'],\n",
       " 418: ['c54', 'c80', 'c93'],\n",
       " 419: ['c18', 'c91', 'c94'],\n",
       " 420: ['c14', 'c35', 'c84'],\n",
       " 421: ['c3', 'c20', 'c56'],\n",
       " 422: ['c12', 'c22', 'c74'],\n",
       " 423: ['c26', 'c48', 'c86'],\n",
       " 424: ['c5', 'c64', 'c70'],\n",
       " 425: ['c16', 'c53', 'c79'],\n",
       " 426: ['c32', 'c72', 'c92'],\n",
       " 427: ['c2', 'c43', 'c63'],\n",
       " 428: ['c49', 'c66', 'c77'],\n",
       " 429: ['c21', 'c24', 'c61'],\n",
       " 430: ['c62', 'c75', 'c78'],\n",
       " 431: ['c19', 'c76', 'c83'],\n",
       " 432: ['c27', 'c31', 'c41'],\n",
       " 433: ['c30', 'c67', 'c90'],\n",
       " 434: ['c17', 'c38', 'c46'],\n",
       " 435: ['c39', 'c44', 'c60'],\n",
       " 436: ['c0', 'c50', 'c82'],\n",
       " 845: ['c1', 'c84', 'c94'],\n",
       " 846: ['c29', 'c30', 'c57'],\n",
       " 847: ['c31', 'c38', 'c79'],\n",
       " 848: ['c3', 'c44', 'c80'],\n",
       " 849: ['c34', 'c39', 'c65'],\n",
       " 850: ['c27', 'c76', 'c82'],\n",
       " 851: ['c6', 'c20', 'c40'],\n",
       " 852: ['c14', 'c66', 'c73'],\n",
       " 853: ['c4', 'c15', 'c53'],\n",
       " 854: ['c45', 'c68', 'c93'],\n",
       " 855: ['c9', 'c56', 'c92'],\n",
       " 856: ['c28', 'c47', 'c61'],\n",
       " 857: ['c42', 'c52', 'c59'],\n",
       " 858: ['c37', 'c41', 'c54'],\n",
       " 859: ['c11', 'c50', 'c51'],\n",
       " 860: ['c58', 'c87', 'c89'],\n",
       " 861: ['c1', 'c29', 'c36'],\n",
       " 862: ['c16', 'c69', 'c83'],\n",
       " 863: ['c13', 'c22', 'c46'],\n",
       " 864: ['c57', 'c76', 'c88'],\n",
       " 866: ['c8', 'c17', 'c19'],\n",
       " 867: ['c7', 'c73', 'c84'],\n",
       " 868: ['c21', 'c67', 'c70'],\n",
       " 869: ['c60', 'c72', 'c77'],\n",
       " 870: ['c23', 'c48', 'c49'],\n",
       " 871: ['c32', 'c55', 'c62'],\n",
       " 872: ['c81', 'c85', 'c91'],\n",
       " 873: ['c5', 'c63', 'c78'],\n",
       " 874: ['c2', 'c12', 'c25'],\n",
       " 875: ['c24', 'c26', 'c35'],\n",
       " 876: ['c64', 'c71', 'c86'],\n",
       " 877: ['c0', 'c90', 'c93'],\n",
       " 878: ['c4', 'c43', 'c74'],\n",
       " 879: ['c10', 'c36', 'c80'],\n",
       " 880: ['c33', 'c65', 'c88'],\n",
       " 881: ['c7', 'c18', 'c79'],\n",
       " 882: ['c20', 'c75', 'c94'],\n",
       " 668: ['c20', 'c53', 'c92'],\n",
       " 669: ['c10', 'c12', 'c43'],\n",
       " 670: ['c33', 'c55', 'c77'],\n",
       " 671: ['c35', 'c52', 'c72'],\n",
       " 672: ['c15', 'c24', 'c48'],\n",
       " 673: ['c60', 'c65', 'c66'],\n",
       " 674: ['c1', 'c17', 'c93'],\n",
       " 675: ['c2', 'c3', 'c67'],\n",
       " 676: ['c64', 'c68', 'c85'],\n",
       " 677: ['c21', 'c73', 'c78'],\n",
       " 678: ['c16', 'c59', 'c74'],\n",
       " 679: ['c0', 'c8', 'c39'],\n",
       " 680: ['c13', 'c83', 'c90'],\n",
       " 681: ['c14', 'c22', 'c23'],\n",
       " 682: ['c6', 'c42', 'c91'],\n",
       " 683: ['c11', 'c32', 'c82'],\n",
       " 684: ['c27', 'c37', 'c57'],\n",
       " 685: ['c30', 'c36', 'c49'],\n",
       " 687: ['c7', 'c38', 'c62'],\n",
       " 688: ['c44', 'c70', 'c84'],\n",
       " 689: ['c51', 'c75', 'c81'],\n",
       " 690: ['c17', 'c68', 'c86'],\n",
       " 691: ['c29', 'c58', 'c71'],\n",
       " 692: ['c34', 'c50', 'c80'],\n",
       " 693: ['c19', 'c40', 'c61'],\n",
       " 694: ['c54', 'c88', 'c94'],\n",
       " 695: ['c31', 'c47', 'c69'],\n",
       " 696: ['c41', 'c45', 'c87'],\n",
       " 697: ['c25', 'c26', 'c46'],\n",
       " 698: ['c5', 'c18', 'c89'],\n",
       " 699: ['c15', 'c28', 'c56'],\n",
       " 700: ['c9', 'c60', 'c79'],\n",
       " 701: ['c2', 'c4', 'c76'],\n",
       " 702: ['c33', 'c35', 'c63'],\n",
       " 703: ['c10', 'c53', 'c86'],\n",
       " 810: ['c0', 'c15', 'c26'],\n",
       " 811: ['c30', 'c56', 'c64'],\n",
       " 812: ['c13', 'c36', 'c53'],\n",
       " 813: ['c19', 'c42', 'c71'],\n",
       " 814: ['c2', 'c6', 'c51'],\n",
       " 815: ['c28', 'c49', 'c72'],\n",
       " 816: ['c14', 'c80', 'c90'],\n",
       " 817: ['c18', 'c57', 'c83'],\n",
       " 818: ['c17', 'c78', 'c92'],\n",
       " 819: ['c54', 'c60', 'c73'],\n",
       " 820: ['c22', 'c29', 'c50'],\n",
       " 821: ['c3', 'c8', 'c16'],\n",
       " 822: ['c12', 'c35', 'c46'],\n",
       " 823: ['c31', 'c58', 'c68'],\n",
       " 824: ['c27', 'c39', 'c93'],\n",
       " 825: ['c7', 'c23', 'c37'],\n",
       " 826: ['c77', 'c84', 'c87'],\n",
       " 827: ['c11', 'c59', 'c88'],\n",
       " 828: ['c25', 'c72', 'c76'],\n",
       " 829: ['c4', 'c5', 'c14'],\n",
       " 830: ['c21', 'c55', 'c81'],\n",
       " 831: ['c1', 'c65', 'c89'],\n",
       " 832: ['c10', 'c66', 'c94'],\n",
       " 833: ['c33', 'c74', 'c86'],\n",
       " 834: ['c40', 'c41', 'c75'],\n",
       " 835: ['c9', 'c34', 'c67'],\n",
       " 836: ['c24', 'c32', 'c43'],\n",
       " 837: ['c45', 'c47', 'c48'],\n",
       " 838: ['c62', 'c63', 'c85'],\n",
       " 839: ['c52', 'c61', 'c70'],\n",
       " 840: ['c38', 'c42', 'c69'],\n",
       " 841: ['c20', 'c36', 'c44'],\n",
       " 842: ['c5', 'c26', 'c91'],\n",
       " 843: ['c25', 'c64', 'c82'],\n",
       " 844: ['c6', 'c79', 'c83'],\n",
       " 950: ['c27', 'c49', 'c79'],\n",
       " 951: ['c13', 'c29', 'c60'],\n",
       " 952: ['c22', 'c28', 'c34'],\n",
       " 953: ['c1', 'c10', 'c68'],\n",
       " 954: ['c43', 'c47', 'c71'],\n",
       " 955: ['c46', 'c74', 'c84'],\n",
       " 956: ['c45', 'c50', 'c70'],\n",
       " 957: ['c58', 'c86', 'c93'],\n",
       " 958: ['c18', 'c56', 'c75'],\n",
       " 959: ['c42', 'c76', 'c92'],\n",
       " 960: ['c6', 'c69', 'c80'],\n",
       " 961: ['c2', 'c15', 'c82'],\n",
       " 962: ['c30', 'c40', 'c83'],\n",
       " 963: ['c33', 'c51', 'c87'],\n",
       " 964: ['c11', 'c16', 'c62'],\n",
       " 965: ['c31', 'c35', 'c90'],\n",
       " 966: ['c37', 'c64', 'c94'],\n",
       " 967: ['c0', 'c7', 'c9'],\n",
       " 968: ['c4', 'c47', 'c52'],\n",
       " 969: ['c23', 'c38', 'c59'],\n",
       " 970: ['c8', 'c65', 'c78'],\n",
       " 971: ['c32', 'c54', 'c81'],\n",
       " 972: ['c12', 'c41', 'c85'],\n",
       " 973: ['c5', 'c57', 'c77'],\n",
       " 974: ['c21', 'c26', 'c89'],\n",
       " 975: ['c55', 'c63', 'c66'],\n",
       " 976: ['c44', 'c61', 'c91'],\n",
       " 977: ['c39', 'c67', 'c88'],\n",
       " 978: ['c25', 'c48', 'c84'],\n",
       " 979: ['c3', 'c19', 'c58'],\n",
       " 980: ['c20', 'c29', 'c73'],\n",
       " 981: ['c14', 'c28', 'c53'],\n",
       " 982: ['c17', 'c36', 'c50'],\n",
       " 983: ['c24', 'c49', 'c52'],\n",
       " 984: ['c18', 'c68', 'c72'],\n",
       " 883: ['c8', 'c15', 'c46'],\n",
       " 884: ['c45', 'c71', 'c94'],\n",
       " 885: ['c51', 'c91', 'c93'],\n",
       " 886: ['c18', 'c23', 'c33'],\n",
       " 887: ['c27', 'c67', 'c77'],\n",
       " 888: ['c75', 'c87', 'c90'],\n",
       " 889: ['c25', 'c37', 'c74'],\n",
       " 890: ['c0', 'c11', 'c43'],\n",
       " 891: ['c2', 'c61', 'c65'],\n",
       " 892: ['c50', 'c60', 'c76'],\n",
       " 893: ['c17', 'c29', 'c72'],\n",
       " 894: ['c6', 'c14', 'c63'],\n",
       " 895: ['c53', 'c73', 'c80'],\n",
       " 896: ['c1', 'c52', 'c55'],\n",
       " 897: ['c39', 'c62', 'c84'],\n",
       " 898: ['c7', 'c59', 'c66'],\n",
       " 899: ['c41', 'c44', 'c81'],\n",
       " 900: ['c22', 'c26', 'c30'],\n",
       " 901: ['c3', 'c32', 'c64'],\n",
       " 902: ['c16', 'c56', 'c70'],\n",
       " 903: ['c40', 'c82', 'c85'],\n",
       " 904: ['c35', 'c57', 'c78'],\n",
       " 905: ['c24', 'c38', 'c89'],\n",
       " 906: ['c9', 'c86', 'c88'],\n",
       " 907: ['c5', 'c10', 'c13'],\n",
       " 908: ['c19', 'c21', 'c54'],\n",
       " 909: ['c12', 'c34', 'c69'],\n",
       " 910: ['c4', 'c31', 'c71'],\n",
       " 911: ['c48', 'c51', 'c92'],\n",
       " 912: ['c20', 'c43', 'c49'],\n",
       " 913: ['c36', 'c37', 'c79'],\n",
       " 914: ['c28', 'c68', 'c87'],\n",
       " 915: ['c58', 'c65', 'c67'],\n",
       " 916: ['c8', 'c33', 'c42'],\n",
       " 917: ['c26', 'c57', 'c69'],\n",
       " 918: ['c5', 'c12', 'c55'],\n",
       " 919: ['c9', 'c13', 'c19'],\n",
       " 920: ['c16', 'c63', 'c94'],\n",
       " 921: ['c7', 'c78', 'c83'],\n",
       " 922: ['c10', 'c61', 'c64'],\n",
       " 923: ['c14', 'c58', 'c72'],\n",
       " 925: ['c43', 'c65', 'c90'],\n",
       " 926: ['c67', 'c68', 'c75'],\n",
       " 927: ['c6', 'c11', 'c37'],\n",
       " 928: ['c32', 'c36', 'c60'],\n",
       " 929: ['c40', 'c52', 'c74'],\n",
       " 930: ['c23', 'c31', 'c51'],\n",
       " 931: ['c27', 'c62', 'c73'],\n",
       " 932: ['c39', 'c76', 'c81'],\n",
       " 933: ['c28', 'c44', 'c93'],\n",
       " 934: ['c18', 'c46', 'c47'],\n",
       " 935: ['c3', 'c59', 'c86'],\n",
       " 936: ['c25', 'c34', 'c56'],\n",
       " 937: ['c45', 'c54', 'c66'],\n",
       " 938: ['c2', 'c35', 'c38'],\n",
       " 939: ['c0', 'c70', 'c89'],\n",
       " 940: ['c22', 'c24', 'c85'],\n",
       " 941: ['c21', 'c41', 'c48'],\n",
       " 942: ['c30', 'c77', 'c80'],\n",
       " 943: ['c1', 'c88', 'c91'],\n",
       " 944: ['c57', 'c82', 'c87'],\n",
       " 945: ['c4', 'c16', 'c20'],\n",
       " 946: ['c8', 'c49', 'c55'],\n",
       " 947: ['c33', 'c53', 'c78'],\n",
       " 948: ['c13', 'c50', 'c79'],\n",
       " 949: ['c10', 'c17', 'c84'],\n",
       " 985: ['c21', 'c46', 'c91'],\n",
       " 986: ['c23', 'c41', 'c61'],\n",
       " 987: ['c29', 'c62', 'c88'],\n",
       " 988: ['c15', 'c25', 'c83'],\n",
       " 989: ['c9', 'c69', 'c75'],\n",
       " 990: ['c19', 'c44', 'c56'],\n",
       " 991: ['c3', 'c24', 'c71']}"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "print(len(region_2_readout_dict))\n",
    "\n",
    "region_2_readout_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "# collect types\n",
    "# only one type for this library which is 'c'\n",
    "\n",
    "readout_usage_by_type = {}\n",
    "for _region, _infos in region_2_readout_dict.items():\n",
    "    # for each bit \n",
    "    for _info in _infos:\n",
    "        if _info[0] not in readout_usage_by_type:\n",
    "            readout_usage_by_type[_info[0]] = []\n",
    "        if int(_info[1:]) not in readout_usage_by_type[_info[0]]:\n",
    "            readout_usage_by_type[_info[0]].append(int(_info[1:]))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [],
   "source": [
    "#readout_usage_by_type   "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "readout_usage_dict = {_type:len(_v) for _type,_v in readout_usage_by_type.items()}\n",
    "readout_dict ={}"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### readout: generate readout files if necessary\n",
    "\n",
    "\n",
    "only required if relevant files have not been generated before."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 72,
   "metadata": {},
   "outputs": [],
   "source": [
    "# load source readout fasta \n",
    "readout_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Readouts'\n",
    "\n",
    "# good adaptor list\n",
    "snr_file_647 = 'highest_snr_647.fasta'\n",
    "snr_file_750 = 'highest_snr_750.fasta'\n",
    "\n",
    "\n",
    "from ImageAnalysis3.library_tools import LibraryTools\n",
    "from ImageAnalysis3.library_tools.LibraryTools import fastaread, fastawrite"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 78,
   "metadata": {},
   "outputs": [],
   "source": [
    "# extract adaptor names and seqs\n",
    "readout_647_name, readout_647_seq = fastaread(readout_folder+os.sep+snr_file_647)\n",
    "readout_750_name, readout_750_seq = fastaread(readout_folder+os.sep+snr_file_750)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 79,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "149\n",
      "149\n",
      "150\n",
      "150\n"
     ]
    }
   ],
   "source": [
    "print(len(readout_647_name))\n",
    "print(len(readout_647_seq))\n",
    "print(len(readout_750_name))\n",
    "print(len(readout_750_seq))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 93,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "NDB_755\n",
      "NDB_784\n"
     ]
    }
   ],
   "source": [
    "print(readout_647_name[0])\n",
    "print(readout_750_name[0])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 99,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "NDB_578\n",
      "TGCCACATCTAGGCGATACATGAGAGGCGT\n",
      "NDB_376\n",
      "CCCTAATCCCGACTTTGTGGGTACGTTGAT\n"
     ]
    }
   ],
   "source": [
    "# filter for channel 647: use NDB index <= 630 (as NDB >630 has been used for the compartment library design) \n",
    "readout_before_630_647_name = []\n",
    "readout_before_630_647_seq = []\n",
    "\n",
    "for _name, _seq in zip(readout_647_name, readout_647_seq):\n",
    "    if int(_name.split('_')[-1])<=630:\n",
    "        readout_before_630_647_name.append(_name)\n",
    "        readout_before_630_647_seq.append(_seq)\n",
    "        \n",
    "print(readout_before_630_647_name[0])\n",
    "print(readout_before_630_647_seq[0])\n",
    "\n",
    "# filter for channel 750: use NDB index <= 630 similarly \n",
    "readout_before_630_750_name = []\n",
    "readout_before_630_750_seq = []\n",
    "\n",
    "for _name, _seq in zip(readout_750_name, readout_750_seq):\n",
    "    if int(_name.split('_')[-1])<=630:\n",
    "        readout_before_630_750_name.append(_name)\n",
    "        readout_before_630_750_seq.append(_seq)\n",
    "        \n",
    "print(readout_before_630_750_name[0])\n",
    "print(readout_before_630_750_seq[0])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 100,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "57\n",
      "57\n",
      "68\n",
      "68\n"
     ]
    }
   ],
   "source": [
    "print(len(readout_before_630_750_name))\n",
    "print(len(readout_before_630_750_seq))\n",
    "print(len(readout_before_630_647_name))\n",
    "print(len(readout_before_630_647_seq))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 101,
   "metadata": {},
   "outputs": [],
   "source": [
    "#save the filtered readout files\n",
    "\n",
    "# save for 750\n",
    "fastawrite(readout_folder+os.sep+'snr_before_630_750.fasta',readout_before_630_750_name,readout_before_630_750_seq,append=False)\n",
    "# save for 647\n",
    "fastawrite(readout_folder+os.sep+'snr_before_630_647.fasta',readout_before_630_647_name,readout_before_630_647_seq,append=False)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### after generating the relevant readout files, proceed"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "reload(library_tools.assemble)\n",
    "overwrite_readout_usage = True\n",
    "readout_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Readouts'\n",
    "readout_start = 0\n",
    "num_color = 2\n",
    "\n",
    "# generate the 'snr_before_630_750.fasta' and 'snr_before_630_647.fasta' (see above) first before proceeding\n",
    "\n",
    "comb_readouts = library_tools.assemble.load_readouts(readout_usage_dict['c'], \n",
    "                                                       _type='snr_before_630', \n",
    "                                                       _readout_folder=readout_folder, \n",
    "                                                       _num_colors=2, # 750, 647\n",
    "                                                       _start_id=readout_start,\n",
    "                                                       )\n",
    "readout_dict['c'] = comb_readouts"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[SeqRecord(seq=Seq('CCCTAATCCCGACTTTGTGGGTACGTTGAT'), id='NDB_376', name='NDB_376', description='NDB_376', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TGCCACATCTAGGCGATACATGAGAGGCGT'), id='NDB_578', name='NDB_578', description='NDB_578', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AGCCAGCTCGTAGCGCCTGAATTAGCTATA'), id='NDB_400', name='NDB_400', description='NDB_400', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TCCGTTAAACGTCGGCTGCGTACTGGTACA'), id='NDB_533', name='NDB_533', description='NDB_533', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TCTCAAGGTTACCGGTTGTCTATATCGTCT'), id='NDB_568', name='NDB_568', description='NDB_568', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AGAGAGCGACTTGGGAACGCCTTCATATCC'), id='NDB_464', name='NDB_464', description='NDB_464', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GCCGTATCAGCGTGTACGGTCAAATACGAG'), id='NDB_466', name='NDB_466', description='NDB_466', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GGACAGGACTGTTCACCGGAATACTTCTGA'), id='NDB_431', name='NDB_431', description='NDB_431', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GCCGATCTTTAGCGGATTAATCGTACGACA'), id='NDB_511', name='NDB_511', description='NDB_511', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GTACATCGTAATCAATCCTCGTTGGCGTTC'), id='NDB_554', name='NDB_554', description='NDB_554', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GGTATCAGCTTTCACGAACGTCTCTATCTA'), id='NDB_487', name='NDB_487', description='NDB_487', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TTCTACGTCTAAATTCCGGAAGCTTGGGCA'), id='NDB_458', name='NDB_458', description='NDB_458', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACGTTAACCCGTAAATTTAGTCTGCGGACC'), id='NDB_409', name='NDB_409', description='NDB_409', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GCGTTGCAAATACGCTGTTGGCGCTATACT'), id='NDB_443', name='NDB_443', description='NDB_443', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TCTTACTTCTCGCGGTTCACGGTATAAGCA'), id='NDB_571', name='NDB_571', description='NDB_571', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GTACACAAGATGTCCGATAGCTTGCGCATA'), id='NDB_455', name='NDB_455', description='NDB_455', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CCATTGTTTACGGGCGACTGCGGTATATCT'), id='NDB_448', name='NDB_448', description='NDB_448', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TCGGTGTATCACTCCGGACGCGAGGTATAA'), id='NDB_470', name='NDB_470', description='NDB_470', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GCCCAGGTAGCGTTTCCGACTGGGATAATC'), id='NDB_529', name='NDB_529', description='NDB_529', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AGCGTTTATACGATGGTCGTCCTCGTTTCG'), id='NDB_332', name='NDB_332', description='NDB_332', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACGCTCGGGCATGTATGCGGGCTCATAATA'), id='NDB_508', name='NDB_508', description='NDB_508', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CACGATGCAATATCGAGTGAGGTGCACCGT'), id='NDB_536', name='NDB_536', description='NDB_536', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CGCTAATCGGATGCTATACGGTGTCGAGCA'), id='NDB_403', name='NDB_403', description='NDB_403', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TTGGACTAACGACCAACGCCTAACATGTAG'), id='NDB_461', name='NDB_461', description='NDB_461', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AGGAACGTAGATACCGCGGCGTAAGTACCC'), id='NDB_391', name='NDB_391', description='NDB_391', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TCCATACTTCGGCACTAGGATAACTTTAGG'), id='NDB_398', name='NDB_398', description='NDB_398', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CTTCTACCGTGTTGGTGCGCTCTGGATATT'), id='NDB_532', name='NDB_532', description='NDB_532', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AATGAGGCGGCGGTTCTACTTGTCCTAATC'), id='NDB_467', name='NDB_467', description='NDB_467', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TTAGACTAATGCACGTATGTCCCGTCCATT'), id='NDB_616', name='NDB_616', description='NDB_616', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACCGATTATTTGGCACGTATCGACCGGTTT'), id='NDB_335', name='NDB_335', description='NDB_335', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ATGGAGTGAAAGTCGCCCTGCATAGGTACC'), id='NDB_388', name='NDB_388', description='NDB_388', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CAGCTGAGAGGAGTGCGCGTCATACCTTTG'), id='NDB_506', name='NDB_506', description='NDB_506', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CGCGTAAGCGAGTTTACCCTTGGCGATGAA'), id='NDB_427', name='NDB_427', description='NDB_427', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACAGTTGAGCTAGTACGTCCCAGTAAGGTC'), id='NDB_347', name='NDB_347', description='NDB_347', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TGTTTCAAACTCGGCCGCGTTCAACATGTG'), id='NDB_619', name='NDB_619', description='NDB_619', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GTCGTTTGCCCATCGCCTTAAACGGTTAAT'), id='NDB_482', name='NDB_482', description='NDB_482', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CCCTAGTACTGTGCGTTATTAATTGCCGGA'), id='NDB_433', name='NDB_433', description='NDB_433', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GGGCTTTACGTGCCGTTCCATAGTTTACCG'), id='NDB_446', name='NDB_446', description='NDB_446', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ATACACGAAGGCACGCCGGATTTACTAGAG'), id='NDB_397', name='NDB_397', description='NDB_397', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CGACACGTTAATTTCCGATGATCCGTTTAC'), id='NDB_473', name='NDB_473', description='NDB_473', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CCGCAAGCGGATTGTCCGAGATGAGTATAC'), id='NDB_505', name='NDB_505', description='NDB_505', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GTACAAGCCCTGTCGCTTGATTAGACGTGC'), id='NDB_539', name='NDB_539', description='NDB_539', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACCCTTTAGGGACTACGCTAGTGTTGTGCC'), id='NDB_385', name='NDB_385', description='NDB_385', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TGTTATTCCCAGCCGATCGCAGGAGTTAAC'), id='NDB_452', name='NDB_452', description='NDB_452', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GCGGTCACGACTTAATCTCGATGGAGCATG'), id='NDB_514', name='NDB_514', description='NDB_514', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AGGTTCGTCATGGCATCCGCAAGTCGTTAT'), id='NDB_356', name='NDB_356', description='NDB_356', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CTACTAATTATCGATTGAGCCGGCAAGTCG'), id='NDB_445', name='NDB_445', description='NDB_445', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CGGGTTGACCCATCGCAAGCAATATATTGT'), id='NDB_587', name='NDB_587', description='NDB_587', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GCCGAGACGTATCCCGGTCATAGATAAATC'), id='NDB_628', name='NDB_628', description='NDB_628', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CCTAACCGATCCGGGTGGAATAGATGTAAC'), id='NDB_551', name='NDB_551', description='NDB_551', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TGCGACTTAATTATCCCGGACACATGGATT'), id='NDB_421', name='NDB_421', description='NDB_421', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CGGCAGAATCGCACGGTTTACTTTAATCCA'), id='NDB_557', name='NDB_557', description='NDB_557', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ATCCAACAGTAGCGGCCGAGATTGTTCATA'), id='NDB_379', name='NDB_379', description='NDB_379', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CGGCAGTCTGTCGTATTCGTAGGATCCGTA'), id='NDB_491', name='NDB_491', description='NDB_491', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GTAAACTTTGTACGGTACCCTGCGCTTCAT'), id='NDB_418', name='NDB_418', description='NDB_418', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CCCTTTGTAGAACGTAAGCGGCAATACTTT'), id='NDB_401', name='NDB_401', description='NDB_401', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CGCATAAGCGACTTGCTGGTCGTACAGTTT'), id='NDB_526', name='NDB_526', description='NDB_526', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AGGTTGGGTGCGACATAGACGACCAGACTC'), id='NDB_611', name='NDB_611', description='NDB_611', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CTAGTTGTATATCGGACAGTCCCTCTTGAA'), id='NDB_472', name='NDB_472', description='NDB_472', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CAGATACCGTTTAATCTCGGCTGAGCGGAG'), id='NDB_341', name='NDB_341', description='NDB_341', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACGCAATCGGCGGTTAATCGAAACGGGTCG'), id='NDB_622', name='NDB_622', description='NDB_622', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACTATCGTATCAACTCCGAGCCTGATGAGG'), id='NDB_386', name='NDB_386', description='NDB_386', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AGCAACGCAGACCTGGCGCTATAGGAAGTC'), id='NDB_517', name='NDB_517', description='NDB_517', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CCTTAGGCATAGATAAGCTCGCTGGTCACA'), id='NDB_392', name='NDB_392', description='NDB_392', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TCCTTGATGTAACCGATAGGCGTCGTGACT'), id='NDB_544', name='NDB_544', description='NDB_544', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CATGACTACGCATCCAAGTTAAGCGCTTAT'), id='NDB_614', name='NDB_614', description='NDB_614', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CTTCATGCTAACGTTCGGGTCTAGGTGTCA'), id='NDB_625', name='NDB_625', description='NDB_625', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CGCATTGGCTTCGTACGCGCTTTATAGGCG'), id='NDB_395', name='NDB_395', description='NDB_395', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CCAATTATCGAAGTCGGTCGTTCTGACATA'), id='NDB_577', name='NDB_577', description='NDB_577', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CGACATGCGCAGATGGTATGGATTGCCCTA'), id='NDB_359', name='NDB_359', description='NDB_359', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TAACACGAAGATCCATAGGCGACCTTTACT'), id='NDB_556', name='NDB_556', description='NDB_556', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACCGAGTAACGTGTGATGACGGTGTACCCG'), id='NDB_581', name='NDB_581', description='NDB_581', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CGGCTACGTTAGGTGCCTCGCTGCGTATAT'), id='NDB_481', name='NDB_481', description='NDB_481', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GCGTTGGACTTAAGTATTACGCCTGACGAC'), id='NDB_350', name='NDB_350', description='NDB_350', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GCCAATAGCTCGGTGAGGATCTCTACGAGA'), id='NDB_439', name='NDB_439', description='NDB_439', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TAACTGAGCTTATCGCTTCGCACGAAATGG'), id='NDB_371', name='NDB_371', description='NDB_371', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACCGAAGATCGTAGTTAAGTACCGCCATTT'), id='NDB_493', name='NDB_493', description='NDB_493', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CTACAGGATACGAGCCGCCATGCGAATAGT'), id='NDB_476', name='NDB_476', description='NDB_476', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ATGTTCCAATGTACTGCCGGCTACATGCTG'), id='NDB_547', name='NDB_547', description='NDB_547', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CCACATAAGGTGCGTACTAGCTGACTATCT'), id='NDB_428', name='NDB_428', description='NDB_428', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TCCATTTAGCGTGCGATAGATCCTCTCGGT'), id='NDB_538', name='NDB_538', description='NDB_538', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GGCGTCATTGAACAACCGGTCTTGCCAGAG'), id='NDB_365', name='NDB_365', description='NDB_365', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TAGTTCGTTCGGTGTCCTTATGCTCGGACT'), id='NDB_589', name='NDB_589', description='NDB_589', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CACTTGTGTCCAATCCGTACGTTTGTGACT'), id='NDB_494', name='NDB_494', description='NDB_494', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ATTCAGGGTGATCGCTTGCATACTTACTTC'), id='NDB_601', name='NDB_601', description='NDB_601', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CGGGTTCAATTACGAATGGAGCTTCCTATC'), id='NDB_479', name='NDB_479', description='NDB_479', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('CACCAAGTATAATGTACGTGGACTCGCATG'), id='NDB_562', name='NDB_562', description='NDB_562', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TAATTATTCCAAACGACTCGTGGCTAGCCA'), id='NDB_608', name='NDB_608', description='NDB_608', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACCAAGCTAAATGCACGTCATCGTGTGTGG'), id='NDB_460', name='NDB_460', description='NDB_460', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('AACCTGACTAGATAGGTCCGCGAGATGCGT'), id='NDB_563', name='NDB_563', description='NDB_563', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('GTACAGCTTAGTTACGTTATCGGATTCTCC'), id='NDB_592', name='NDB_592', description='NDB_592', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TTGGACGCATCTGACGCTGAAGTGCCTAGT'), id='NDB_368', name='NDB_368', description='NDB_368', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ACACATCTTCTCGTGAGGCGGTATTTACAA'), id='NDB_436', name='NDB_436', description='NDB_436', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('ATGCAGACTGGGTAAGCGGTACGTCCTGTC'), id='NDB_629', name='NDB_629', description='NDB_629', dbxrefs=[]),\n",
       " SeqRecord(seq=Seq('TCTCATGTATAGGGCCGAGCGACTATCCTA'), id='NDB_604', name='NDB_604', description='NDB_604', dbxrefs=[])]"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "readout_dict['c']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 87,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "save\n"
     ]
    }
   ],
   "source": [
    "# save\n",
    "readout_usage_file = os.path.join(library_folder, 'readout_usage.pkl')\n",
    "if not os.path.isfile(readout_usage_file) or overwrite_readout_usage:\n",
    "    print('save')\n",
    "    pickle.dump(readout_dict, open(readout_usage_file, 'wb'))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 3.3 assemble probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "ename": "NameError",
     "evalue": "name 'filtered_pb_dict' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[1;31mNameError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[1;32m<ipython-input-23-305b5a736351>\u001b[0m in \u001b[0;36m<module>\u001b[1;34m\u001b[0m\n\u001b[0;32m      2\u001b[0m \u001b[1;32mfrom\u001b[0m \u001b[0mImageAnalysis3\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mlibrary_tools\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0massemble\u001b[0m \u001b[1;32mimport\u001b[0m \u001b[0mAssemble_probes\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m      3\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m----> 4\u001b[1;33m cand_probes, readout_summary = Assemble_probes(library_folder, filtered_pb_dict, \n\u001b[0m\u001b[0;32m      5\u001b[0m                                                \u001b[0mregion_2_readout_dict\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mreadout_dict\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mprimers\u001b[0m\u001b[1;33m,\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m      6\u001b[0m                                                \u001b[0mprimer_len\u001b[0m\u001b[1;33m=\u001b[0m\u001b[1;36m20\u001b[0m\u001b[1;33m,\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;31mNameError\u001b[0m: name 'filtered_pb_dict' is not defined"
     ]
    }
   ],
   "source": [
    "# assemble full-length probes using primer, readouts, and targeting regions (from filtered_pb_dict)\n",
    "\n",
    "reload(library_tools.assemble)\n",
    "from ImageAnalysis3.library_tools.assemble import Assemble_probes\n",
    "\n",
    "cand_probes, readout_summary = Assemble_probes(library_folder, filtered_pb_dict, \n",
    "                                               region_2_readout_dict, readout_dict, primers, \n",
    "                                               primer_len=20, \n",
    "                                               num_readout_per_probe=3, \n",
    "                                               unique_readout_per_probe=False,\n",
    "                                               rc_targets=False, overwrite=True)\n",
    "\n",
    "\n",
    "# candidate_probes.fasta and readout_summary.pkl are generated after this step"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# 4.Check quality"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 4.1 Load probes\n",
    "\n",
    "Checking procedures could restart from here"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Total probe loaded: 385098\n"
     ]
    }
   ],
   "source": [
    "# folder for this library\n",
    "# candidate full-length probe filename\n",
    "candidate_basename = 'candidate_probes.fasta' # old version\n",
    "\n",
    "\n",
    "\n",
    "# load assembled full-length probes from the \"candidate_probes.fasta\" file\n",
    "full_records = []\n",
    "\n",
    "\n",
    "with open(os.path.join(library_folder, candidate_basename), 'r') as handle:\n",
    "    for record in SeqIO.parse(handle, \"fasta\"):\n",
    "        full_records.append(record)\n",
    "\n",
    "print(f\"Total probe loaded: {len(full_records)}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [],
   "source": [
    "num_readout_per_probe = 3\n",
    "primer_len=20\n",
    "target_len=42\n",
    "min_num_probes = 40"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 4.1.1 check primer usage"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- Checking primer usage, total probes: 385098\n",
      "True\n"
     ]
    }
   ],
   "source": [
    "# check\n",
    "\n",
    "import ImageAnalysis3.library_tools.quality_check as check\n",
    "reload(check)\n",
    "primer_check = check._check_primer_usage(full_records, primers[0], primers[1], primer_len=primer_len)\n",
    "print(primer_check)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 4.1.2 check region probe number"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "gene: 251 -> 400\n",
      "gene: 252 -> 400\n",
      "gene: 253 -> 400\n",
      "gene: 255 -> 400\n",
      "gene: 256 -> 400\n",
      "gene: 257 -> 400\n",
      "gene: 258 -> 400\n",
      "gene: 259 -> 400\n",
      "gene: 260 -> 400\n",
      "gene: 261 -> 400\n",
      "gene: 262 -> 400\n",
      "gene: 263 -> 400\n",
      "gene: 264 -> 400\n",
      "gene: 265 -> 400\n",
      "gene: 266 -> 400\n",
      "gene: 267 -> 400\n",
      "gene: 268 -> 400\n",
      "gene: 269 -> 400\n",
      "gene: 270 -> 400\n",
      "gene: 271 -> 400\n",
      "gene: 272 -> 400\n",
      "gene: 273 -> 400\n",
      "gene: 274 -> 400\n",
      "gene: 275 -> 400\n",
      "gene: 277 -> 400\n",
      "gene: 278 -> 400\n",
      "gene: 279 -> 400\n",
      "gene: 280 -> 400\n",
      "gene: 281 -> 400\n",
      "gene: 282 -> 400\n",
      "gene: 283 -> 400\n",
      "gene: 284 -> 400\n",
      "gene: 285 -> 400\n",
      "gene: 286 -> 400\n",
      "gene: 287 -> 400\n",
      "gene: 288 -> 400\n",
      "gene: 289 -> 400\n",
      "gene: 290 -> 400\n",
      "gene: 291 -> 400\n",
      "gene: 292 -> 400\n",
      "gene: 293 -> 400\n",
      "gene: 295 -> 400\n",
      "gene: 296 -> 400\n",
      "gene: 297 -> 400\n",
      "gene: 298 -> 400\n",
      "gene: 299 -> 400\n",
      "gene: 300 -> 400\n",
      "gene: 301 -> 400\n",
      "gene: 302 -> 400\n",
      "gene: 303 -> 400\n",
      "gene: 304 -> 400\n",
      "gene: 305 -> 400\n",
      "gene: 306 -> 400\n",
      "gene: 307 -> 400\n",
      "gene: 308 -> 400\n",
      "gene: 309 -> 400\n",
      "gene: 310 -> 400\n",
      "gene: 311 -> 400\n",
      "gene: 312 -> 400\n",
      "gene: 313 -> 400\n",
      "gene: 314 -> 400\n",
      "gene: 315 -> 400\n",
      "gene: 316 -> 400\n",
      "gene: 317 -> 400\n",
      "gene: 318 -> 400\n",
      "gene: 319 -> 400\n",
      "gene: 320 -> 400\n",
      "gene: 321 -> 400\n",
      "gene: 322 -> 400\n",
      "gene: 324 -> 400\n",
      "gene: 325 -> 400\n",
      "gene: 326 -> 400\n",
      "gene: 327 -> 400\n",
      "gene: 328 -> 400\n",
      "gene: 329 -> 400\n",
      "gene: 330 -> 400\n",
      "gene: 332 -> 400\n",
      "gene: 333 -> 400\n",
      "gene: 334 -> 400\n",
      "gene: 335 -> 400\n",
      "gene: 336 -> 400\n",
      "gene: 337 -> 400\n",
      "gene: 338 -> 400\n",
      "gene: 339 -> 400\n",
      "gene: 340 -> 400\n",
      "gene: 341 -> 400\n",
      "gene: 342 -> 400\n",
      "gene: 343 -> 400\n",
      "gene: 344 -> 400\n",
      "gene: 345 -> 400\n",
      "gene: 347 -> 400\n",
      "gene: 348 -> 400\n",
      "gene: 349 -> 400\n",
      "gene: 350 -> 400\n",
      "gene: 351 -> 400\n",
      "gene: 352 -> 400\n",
      "gene: 353 -> 400\n",
      "gene: 354 -> 400\n",
      "gene: 355 -> 400\n",
      "gene: 356 -> 400\n",
      "gene: 357 -> 400\n",
      "gene: 358 -> 400\n",
      "gene: 359 -> 400\n",
      "gene: 360 -> 400\n",
      "gene: 361 -> 400\n",
      "gene: 362 -> 320\n",
      "gene: 363 -> 400\n",
      "gene: 364 -> 400\n",
      "gene: 365 -> 400\n",
      "gene: 366 -> 400\n",
      "gene: 367 -> 400\n",
      "gene: 368 -> 400\n",
      "gene: 369 -> 400\n",
      "gene: 370 -> 400\n",
      "gene: 371 -> 400\n",
      "gene: 372 -> 400\n",
      "gene: 373 -> 400\n",
      "gene: 374 -> 400\n",
      "gene: 375 -> 400\n",
      "gene: 376 -> 400\n",
      "gene: 377 -> 400\n",
      "gene: 378 -> 400\n",
      "gene: 379 -> 400\n",
      "gene: 380 -> 400\n",
      "gene: 381 -> 400\n",
      "gene: 382 -> 400\n",
      "gene: 383 -> 400\n",
      "gene: 384 -> 400\n",
      "gene: 385 -> 400\n",
      "gene: 386 -> 400\n",
      "gene: 387 -> 400\n",
      "gene: 388 -> 400\n",
      "gene: 389 -> 400\n",
      "gene: 390 -> 400\n",
      "gene: 391 -> 400\n",
      "gene: 392 -> 400\n",
      "gene: 393 -> 400\n",
      "gene: 394 -> 400\n",
      "gene: 395 -> 400\n",
      "gene: 396 -> 400\n",
      "gene: 397 -> 400\n",
      "gene: 398 -> 400\n",
      "gene: 399 -> 400\n",
      "gene: 400 -> 400\n",
      "gene: 401 -> 400\n",
      "gene: 402 -> 73\n",
      "gene: 403 -> 400\n",
      "gene: 404 -> 400\n",
      "gene: 405 -> 400\n",
      "gene: 406 -> 400\n",
      "gene: 407 -> 400\n",
      "gene: 408 -> 400\n",
      "gene: 409 -> 400\n",
      "gene: 410 -> 400\n",
      "gene: 411 -> 400\n",
      "gene: 412 -> 400\n",
      "gene: 413 -> 400\n",
      "gene: 414 -> 400\n",
      "gene: 415 -> 400\n",
      "gene: 416 -> 400\n",
      "gene: 417 -> 400\n",
      "gene: 418 -> 378\n",
      "gene: 419 -> 400\n",
      "gene: 420 -> 400\n",
      "gene: 421 -> 400\n",
      "gene: 422 -> 400\n",
      "gene: 423 -> 400\n",
      "gene: 424 -> 400\n",
      "gene: 425 -> 400\n",
      "gene: 426 -> 313\n",
      "gene: 427 -> 400\n",
      "gene: 428 -> 400\n",
      "gene: 429 -> 400\n",
      "gene: 430 -> 400\n",
      "gene: 431 -> 400\n",
      "gene: 432 -> 400\n",
      "gene: 433 -> 400\n",
      "gene: 434 -> 400\n",
      "gene: 435 -> 400\n",
      "gene: 436 -> 400\n",
      "gene: 437 -> 400\n",
      "gene: 438 -> 400\n",
      "gene: 439 -> 400\n",
      "gene: 440 -> 400\n",
      "gene: 441 -> 400\n",
      "gene: 442 -> 400\n",
      "gene: 444 -> 400\n",
      "gene: 445 -> 400\n",
      "gene: 446 -> 400\n",
      "gene: 447 -> 400\n",
      "gene: 448 -> 400\n",
      "gene: 449 -> 400\n",
      "gene: 450 -> 400\n",
      "gene: 451 -> 400\n",
      "gene: 452 -> 400\n",
      "gene: 453 -> 400\n",
      "gene: 454 -> 400\n",
      "gene: 455 -> 400\n",
      "gene: 456 -> 400\n",
      "gene: 457 -> 400\n",
      "gene: 458 -> 400\n",
      "gene: 459 -> 400\n",
      "gene: 460 -> 400\n",
      "gene: 461 -> 400\n",
      "gene: 462 -> 400\n",
      "gene: 463 -> 400\n",
      "gene: 464 -> 400\n",
      "gene: 465 -> 400\n",
      "gene: 466 -> 400\n",
      "gene: 467 -> 400\n",
      "gene: 468 -> 400\n",
      "gene: 469 -> 400\n",
      "gene: 470 -> 400\n",
      "gene: 471 -> 400\n",
      "gene: 472 -> 400\n",
      "gene: 473 -> 400\n",
      "gene: 474 -> 400\n",
      "gene: 475 -> 400\n",
      "gene: 476 -> 400\n",
      "gene: 477 -> 400\n",
      "gene: 478 -> 400\n",
      "gene: 479 -> 400\n",
      "gene: 480 -> 400\n",
      "gene: 481 -> 400\n",
      "gene: 482 -> 400\n",
      "gene: 483 -> 400\n",
      "gene: 484 -> 400\n",
      "gene: 485 -> 400\n",
      "gene: 486 -> 400\n",
      "gene: 487 -> 400\n",
      "gene: 488 -> 400\n",
      "gene: 489 -> 400\n",
      "gene: 491 -> 400\n",
      "gene: 492 -> 400\n",
      "gene: 493 -> 400\n",
      "gene: 494 -> 400\n",
      "gene: 495 -> 400\n",
      "gene: 496 -> 400\n",
      "gene: 497 -> 400\n",
      "gene: 498 -> 400\n",
      "gene: 499 -> 400\n",
      "gene: 500 -> 400\n",
      "gene: 501 -> 400\n",
      "gene: 502 -> 400\n",
      "gene: 503 -> 400\n",
      "gene: 504 -> 400\n",
      "gene: 505 -> 400\n",
      "gene: 506 -> 400\n",
      "gene: 507 -> 400\n",
      "gene: 508 -> 400\n",
      "gene: 509 -> 400\n",
      "gene: 510 -> 400\n",
      "gene: 511 -> 400\n",
      "gene: 512 -> 400\n",
      "gene: 513 -> 400\n",
      "gene: 514 -> 400\n",
      "gene: 515 -> 400\n",
      "gene: 516 -> 400\n",
      "gene: 517 -> 400\n",
      "gene: 518 -> 400\n",
      "gene: 519 -> 400\n",
      "gene: 520 -> 400\n",
      "gene: 521 -> 400\n",
      "gene: 523 -> 400\n",
      "gene: 524 -> 400\n",
      "gene: 525 -> 400\n",
      "gene: 526 -> 400\n",
      "gene: 527 -> 400\n",
      "gene: 528 -> 400\n",
      "gene: 529 -> 400\n",
      "gene: 530 -> 400\n",
      "gene: 531 -> 400\n",
      "gene: 532 -> 400\n",
      "gene: 533 -> 400\n",
      "gene: 534 -> 400\n",
      "gene: 535 -> 400\n",
      "gene: 536 -> 400\n",
      "gene: 537 -> 400\n",
      "gene: 538 -> 400\n",
      "gene: 539 -> 268\n",
      "gene: 540 -> 400\n",
      "gene: 541 -> 400\n",
      "gene: 542 -> 400\n",
      "gene: 543 -> 400\n",
      "gene: 544 -> 400\n",
      "gene: 545 -> 400\n",
      "gene: 546 -> 400\n",
      "gene: 547 -> 400\n",
      "gene: 548 -> 400\n",
      "gene: 549 -> 400\n",
      "gene: 550 -> 400\n",
      "gene: 551 -> 400\n",
      "gene: 552 -> 400\n",
      "gene: 553 -> 400\n",
      "gene: 554 -> 400\n",
      "gene: 555 -> 400\n",
      "gene: 556 -> 400\n",
      "gene: 557 -> 400\n",
      "gene: 558 -> 400\n",
      "gene: 559 -> 400\n",
      "gene: 560 -> 400\n",
      "gene: 561 -> 400\n",
      "gene: 562 -> 400\n",
      "gene: 563 -> 400\n",
      "gene: 564 -> 400\n",
      "gene: 565 -> 400\n",
      "gene: 566 -> 400\n",
      "gene: 567 -> 400\n",
      "gene: 568 -> 400\n",
      "gene: 569 -> 400\n",
      "gene: 570 -> 400\n",
      "gene: 571 -> 400\n",
      "gene: 572 -> 400\n",
      "gene: 573 -> 400\n",
      "gene: 574 -> 400\n",
      "gene: 575 -> 400\n",
      "gene: 576 -> 400\n",
      "gene: 577 -> 400\n",
      "gene: 579 -> 400\n",
      "gene: 580 -> 400\n",
      "gene: 581 -> 400\n",
      "gene: 582 -> 400\n",
      "gene: 583 -> 400\n",
      "gene: 584 -> 400\n",
      "gene: 585 -> 400\n",
      "gene: 586 -> 400\n",
      "gene: 587 -> 400\n",
      "gene: 588 -> 400\n",
      "gene: 589 -> 400\n",
      "gene: 590 -> 400\n",
      "gene: 591 -> 400\n",
      "gene: 592 -> 400\n",
      "gene: 593 -> 400\n",
      "gene: 594 -> 400\n",
      "gene: 595 -> 400\n",
      "gene: 596 -> 400\n",
      "gene: 597 -> 400\n",
      "gene: 598 -> 400\n",
      "gene: 599 -> 400\n",
      "gene: 600 -> 400\n",
      "gene: 601 -> 400\n",
      "gene: 602 -> 400\n",
      "gene: 603 -> 400\n",
      "gene: 604 -> 400\n",
      "gene: 605 -> 400\n",
      "gene: 606 -> 400\n",
      "gene: 607 -> 400\n",
      "gene: 608 -> 400\n",
      "gene: 609 -> 400\n",
      "gene: 610 -> 400\n",
      "gene: 611 -> 400\n",
      "gene: 612 -> 400\n",
      "gene: 613 -> 400\n",
      "gene: 614 -> 400\n",
      "gene: 615 -> 400\n",
      "gene: 616 -> 400\n",
      "gene: 617 -> 400\n",
      "gene: 618 -> 400\n",
      "gene: 619 -> 400\n",
      "gene: 620 -> 400\n",
      "gene: 621 -> 400\n",
      "gene: 622 -> 400\n",
      "gene: 623 -> 400\n",
      "gene: 624 -> 400\n",
      "gene: 625 -> 400\n",
      "gene: 626 -> 400\n",
      "gene: 627 -> 400\n",
      "gene: 628 -> 400\n",
      "gene: 629 -> 400\n",
      "gene: 630 -> 400\n",
      "gene: 631 -> 400\n",
      "gene: 632 -> 400\n",
      "gene: 633 -> 400\n",
      "gene: 634 -> 400\n",
      "gene: 635 -> 400\n",
      "gene: 636 -> 400\n",
      "gene: 637 -> 400\n",
      "gene: 638 -> 400\n",
      "gene: 639 -> 400\n",
      "gene: 640 -> 400\n",
      "gene: 641 -> 400\n",
      "gene: 642 -> 400\n",
      "gene: 643 -> 400\n",
      "gene: 644 -> 400\n",
      "gene: 645 -> 400\n",
      "gene: 646 -> 400\n",
      "gene: 647 -> 400\n",
      "gene: 648 -> 400\n",
      "gene: 649 -> 400\n",
      "gene: 650 -> 400\n",
      "gene: 651 -> 400\n",
      "gene: 652 -> 400\n",
      "gene: 653 -> 400\n",
      "gene: 654 -> 400\n",
      "gene: 655 -> 400\n",
      "gene: 656 -> 400\n",
      "gene: 657 -> 400\n",
      "gene: 658 -> 400\n",
      "gene: 659 -> 400\n",
      "gene: 660 -> 400\n",
      "gene: 661 -> 400\n",
      "gene: 662 -> 400\n",
      "gene: 663 -> 400\n",
      "gene: 664 -> 400\n",
      "gene: 665 -> 400\n",
      "gene: 666 -> 400\n",
      "gene: 667 -> 400\n",
      "gene: 668 -> 400\n",
      "gene: 669 -> 400\n",
      "gene: 670 -> 400\n",
      "gene: 671 -> 400\n",
      "gene: 672 -> 400\n",
      "gene: 673 -> 400\n",
      "gene: 674 -> 400\n",
      "gene: 675 -> 400\n",
      "gene: 676 -> 400\n",
      "gene: 677 -> 400\n",
      "gene: 678 -> 400\n",
      "gene: 679 -> 400\n",
      "gene: 680 -> 400\n",
      "gene: 681 -> 400\n",
      "gene: 682 -> 400\n",
      "gene: 683 -> 400\n",
      "gene: 684 -> 400\n",
      "gene: 685 -> 400\n",
      "gene: 687 -> 400\n",
      "gene: 688 -> 400\n",
      "gene: 689 -> 400\n",
      "gene: 690 -> 400\n",
      "gene: 691 -> 400\n",
      "gene: 692 -> 400\n",
      "gene: 693 -> 400\n",
      "gene: 694 -> 400\n",
      "gene: 695 -> 400\n",
      "gene: 696 -> 400\n",
      "gene: 697 -> 400\n",
      "gene: 698 -> 400\n",
      "gene: 699 -> 400\n",
      "gene: 700 -> 400\n",
      "gene: 701 -> 400\n",
      "gene: 702 -> 400\n",
      "gene: 703 -> 400\n",
      "gene: 704 -> 400\n",
      "gene: 705 -> 400\n",
      "gene: 706 -> 400\n",
      "gene: 707 -> 400\n",
      "gene: 708 -> 400\n",
      "gene: 709 -> 400\n",
      "gene: 710 -> 400\n",
      "gene: 711 -> 400\n",
      "gene: 712 -> 400\n",
      "gene: 713 -> 400\n",
      "gene: 714 -> 400\n",
      "gene: 715 -> 400\n",
      "gene: 716 -> 400\n",
      "gene: 717 -> 400\n",
      "gene: 718 -> 400\n",
      "gene: 719 -> 400\n",
      "gene: 720 -> 400\n",
      "gene: 721 -> 400\n",
      "gene: 722 -> 400\n",
      "gene: 723 -> 400\n",
      "gene: 724 -> 400\n",
      "gene: 725 -> 400\n",
      "gene: 726 -> 400\n",
      "gene: 727 -> 400\n",
      "gene: 728 -> 400\n",
      "gene: 729 -> 400\n",
      "gene: 730 -> 400\n",
      "gene: 731 -> 400\n",
      "gene: 732 -> 400\n",
      "gene: 733 -> 400\n",
      "gene: 734 -> 400\n",
      "gene: 735 -> 400\n",
      "gene: 736 -> 400\n",
      "gene: 737 -> 400\n",
      "gene: 738 -> 400\n",
      "gene: 739 -> 400\n",
      "gene: 740 -> 400\n",
      "gene: 741 -> 400\n",
      "gene: 742 -> 400\n",
      "gene: 743 -> 400\n",
      "gene: 744 -> 400\n",
      "gene: 745 -> 400\n",
      "gene: 746 -> 400\n",
      "gene: 747 -> 400\n",
      "gene: 748 -> 400\n",
      "gene: 749 -> 400\n",
      "gene: 750 -> 400\n",
      "gene: 1 -> 400\n",
      "gene: 2 -> 400\n",
      "gene: 3 -> 400\n",
      "gene: 4 -> 400\n",
      "gene: 5 -> 400\n",
      "gene: 6 -> 400\n",
      "gene: 7 -> 400\n",
      "gene: 8 -> 400\n",
      "gene: 9 -> 400\n",
      "gene: 10 -> 400\n",
      "gene: 11 -> 400\n",
      "gene: 12 -> 400\n",
      "gene: 13 -> 400\n",
      "gene: 14 -> 400\n",
      "gene: 15 -> 400\n",
      "gene: 16 -> 400\n",
      "gene: 17 -> 400\n",
      "gene: 18 -> 400\n",
      "gene: 19 -> 400\n",
      "gene: 20 -> 400\n",
      "gene: 21 -> 400\n",
      "gene: 22 -> 400\n",
      "gene: 23 -> 400\n",
      "gene: 24 -> 400\n",
      "gene: 25 -> 400\n",
      "gene: 26 -> 400\n",
      "gene: 27 -> 400\n",
      "gene: 28 -> 400\n",
      "gene: 29 -> 400\n",
      "gene: 30 -> 400\n",
      "gene: 31 -> 400\n",
      "gene: 32 -> 400\n",
      "gene: 33 -> 400\n",
      "gene: 34 -> 400\n",
      "gene: 35 -> 400\n",
      "gene: 36 -> 400\n",
      "gene: 37 -> 400\n",
      "gene: 38 -> 400\n",
      "gene: 39 -> 400\n",
      "gene: 40 -> 400\n",
      "gene: 41 -> 400\n",
      "gene: 42 -> 400\n",
      "gene: 43 -> 400\n",
      "gene: 44 -> 400\n",
      "gene: 45 -> 400\n",
      "gene: 46 -> 400\n",
      "gene: 47 -> 400\n",
      "gene: 48 -> 400\n",
      "gene: 49 -> 400\n",
      "gene: 50 -> 400\n",
      "gene: 51 -> 400\n",
      "gene: 52 -> 400\n",
      "gene: 53 -> 400\n",
      "gene: 54 -> 400\n",
      "gene: 55 -> 400\n",
      "gene: 56 -> 400\n",
      "gene: 57 -> 400\n",
      "gene: 58 -> 400\n",
      "gene: 59 -> 400\n",
      "gene: 60 -> 400\n",
      "gene: 61 -> 400\n",
      "gene: 62 -> 400\n",
      "gene: 63 -> 400\n",
      "gene: 64 -> 400\n",
      "gene: 65 -> 400\n",
      "gene: 66 -> 400\n",
      "gene: 67 -> 400\n",
      "gene: 68 -> 400\n",
      "gene: 69 -> 400\n",
      "gene: 70 -> 400\n",
      "gene: 71 -> 400\n",
      "gene: 72 -> 400\n",
      "gene: 73 -> 400\n",
      "gene: 74 -> 400\n",
      "gene: 75 -> 400\n",
      "gene: 76 -> 400\n",
      "gene: 77 -> 400\n",
      "gene: 78 -> 400\n",
      "gene: 79 -> 400\n",
      "gene: 80 -> 400\n",
      "gene: 81 -> 400\n",
      "gene: 82 -> 400\n",
      "gene: 83 -> 400\n",
      "gene: 84 -> 400\n",
      "gene: 85 -> 400\n",
      "gene: 86 -> 400\n",
      "gene: 87 -> 400\n",
      "gene: 88 -> 400\n",
      "gene: 89 -> 400\n",
      "gene: 90 -> 400\n",
      "gene: 91 -> 400\n",
      "gene: 92 -> 400\n",
      "gene: 93 -> 400\n",
      "gene: 94 -> 400\n",
      "gene: 95 -> 400\n",
      "gene: 96 -> 400\n",
      "gene: 97 -> 400\n",
      "gene: 98 -> 400\n",
      "gene: 99 -> 400\n",
      "gene: 100 -> 400\n",
      "gene: 101 -> 400\n",
      "gene: 102 -> 400\n",
      "gene: 103 -> 400\n",
      "gene: 104 -> 400\n",
      "gene: 105 -> 400\n",
      "gene: 106 -> 400\n",
      "gene: 107 -> 400\n",
      "gene: 108 -> 400\n",
      "gene: 109 -> 400\n",
      "gene: 111 -> 400\n",
      "gene: 112 -> 400\n",
      "gene: 113 -> 400\n",
      "gene: 114 -> 400\n",
      "gene: 115 -> 400\n",
      "gene: 116 -> 400\n",
      "gene: 117 -> 400\n",
      "gene: 118 -> 400\n",
      "gene: 119 -> 400\n",
      "gene: 120 -> 400\n",
      "gene: 121 -> 400\n",
      "gene: 122 -> 400\n",
      "gene: 123 -> 400\n",
      "gene: 124 -> 400\n",
      "gene: 125 -> 400\n",
      "gene: 126 -> 400\n",
      "gene: 127 -> 400\n",
      "gene: 129 -> 400\n",
      "gene: 130 -> 400\n",
      "gene: 131 -> 400\n",
      "gene: 132 -> 400\n",
      "gene: 133 -> 400\n",
      "gene: 134 -> 400\n",
      "gene: 135 -> 400\n",
      "gene: 136 -> 400\n",
      "gene: 137 -> 400\n",
      "gene: 138 -> 400\n",
      "gene: 139 -> 400\n",
      "gene: 140 -> 400\n",
      "gene: 141 -> 400\n",
      "gene: 142 -> 400\n",
      "gene: 143 -> 400\n",
      "gene: 144 -> 400\n",
      "gene: 145 -> 400\n",
      "gene: 146 -> 400\n",
      "gene: 147 -> 400\n",
      "gene: 148 -> 400\n",
      "gene: 149 -> 400\n",
      "gene: 150 -> 400\n",
      "gene: 151 -> 400\n",
      "gene: 152 -> 400\n",
      "gene: 153 -> 400\n",
      "gene: 154 -> 400\n",
      "gene: 155 -> 400\n",
      "gene: 156 -> 400\n",
      "gene: 157 -> 400\n",
      "gene: 158 -> 400\n",
      "gene: 159 -> 400\n",
      "gene: 160 -> 400\n",
      "gene: 161 -> 400\n",
      "gene: 162 -> 400\n",
      "gene: 163 -> 400\n",
      "gene: 164 -> 400\n",
      "gene: 165 -> 400\n",
      "gene: 166 -> 400\n",
      "gene: 168 -> 400\n",
      "gene: 169 -> 400\n",
      "gene: 170 -> 400\n",
      "gene: 171 -> 400\n",
      "gene: 172 -> 400\n",
      "gene: 173 -> 400\n",
      "gene: 174 -> 400\n",
      "gene: 175 -> 400\n",
      "gene: 176 -> 400\n",
      "gene: 177 -> 400\n",
      "gene: 178 -> 400\n",
      "gene: 179 -> 400\n",
      "gene: 180 -> 400\n",
      "gene: 181 -> 400\n",
      "gene: 182 -> 400\n",
      "gene: 183 -> 400\n",
      "gene: 184 -> 400\n",
      "gene: 185 -> 115\n",
      "gene: 186 -> 400\n",
      "gene: 187 -> 400\n",
      "gene: 188 -> 400\n",
      "gene: 189 -> 400\n",
      "gene: 190 -> 400\n",
      "gene: 191 -> 400\n",
      "gene: 192 -> 400\n",
      "gene: 193 -> 400\n",
      "gene: 194 -> 400\n",
      "gene: 195 -> 400\n",
      "gene: 196 -> 400\n",
      "gene: 197 -> 400\n",
      "gene: 198 -> 400\n",
      "gene: 199 -> 400\n",
      "gene: 200 -> 400\n",
      "gene: 201 -> 400\n",
      "gene: 202 -> 400\n",
      "gene: 203 -> 400\n",
      "gene: 204 -> 400\n",
      "gene: 205 -> 400\n",
      "gene: 206 -> 400\n",
      "gene: 207 -> 400\n",
      "gene: 208 -> 400\n",
      "gene: 209 -> 400\n",
      "gene: 210 -> 400\n",
      "gene: 211 -> 400\n",
      "gene: 212 -> 209\n",
      "gene: 213 -> 283\n",
      "gene: 214 -> 400\n",
      "gene: 215 -> 400\n",
      "gene: 216 -> 400\n",
      "gene: 217 -> 400\n",
      "gene: 218 -> 400\n",
      "gene: 219 -> 400\n",
      "gene: 220 -> 400\n",
      "gene: 221 -> 400\n",
      "gene: 222 -> 400\n",
      "gene: 223 -> 400\n",
      "gene: 224 -> 400\n",
      "gene: 225 -> 400\n",
      "gene: 226 -> 400\n",
      "gene: 228 -> 400\n",
      "gene: 229 -> 400\n",
      "gene: 230 -> 400\n",
      "gene: 231 -> 400\n",
      "gene: 232 -> 400\n",
      "gene: 233 -> 400\n",
      "gene: 234 -> 400\n",
      "gene: 235 -> 400\n",
      "gene: 236 -> 400\n",
      "gene: 237 -> 400\n",
      "gene: 238 -> 400\n",
      "gene: 239 -> 400\n",
      "gene: 240 -> 400\n",
      "gene: 241 -> 400\n",
      "gene: 242 -> 400\n",
      "gene: 243 -> 400\n",
      "gene: 244 -> 400\n",
      "gene: 245 -> 400\n",
      "gene: 246 -> 400\n",
      "gene: 247 -> 400\n",
      "gene: 248 -> 400\n",
      "gene: 249 -> 400\n",
      "gene: 250 -> 400\n",
      "gene: 751 -> 400\n",
      "gene: 752 -> 176\n",
      "gene: 758 -> 400\n",
      "gene: 759 -> 400\n",
      "gene: 760 -> 400\n",
      "gene: 761 -> 400\n",
      "gene: 762 -> 400\n",
      "gene: 763 -> 400\n",
      "gene: 764 -> 400\n",
      "gene: 765 -> 400\n",
      "gene: 766 -> 400\n",
      "gene: 767 -> 400\n",
      "gene: 768 -> 400\n",
      "gene: 769 -> 400\n",
      "gene: 770 -> 400\n",
      "gene: 772 -> 400\n",
      "gene: 773 -> 400\n",
      "gene: 774 -> 400\n",
      "gene: 775 -> 400\n",
      "gene: 776 -> 62\n",
      "gene: 777 -> 400\n",
      "gene: 778 -> 400\n",
      "gene: 779 -> 393\n",
      "gene: 780 -> 400\n",
      "gene: 781 -> 400\n",
      "gene: 782 -> 400\n",
      "gene: 783 -> 400\n",
      "gene: 784 -> 400\n",
      "gene: 785 -> 400\n",
      "gene: 786 -> 400\n",
      "gene: 787 -> 400\n",
      "gene: 788 -> 400\n",
      "gene: 789 -> 400\n",
      "gene: 790 -> 400\n",
      "gene: 791 -> 400\n",
      "gene: 792 -> 400\n",
      "gene: 793 -> 400\n",
      "gene: 794 -> 400\n",
      "gene: 795 -> 400\n",
      "gene: 796 -> 400\n",
      "gene: 797 -> 400\n",
      "gene: 798 -> 400\n",
      "gene: 799 -> 400\n",
      "gene: 800 -> 400\n",
      "gene: 801 -> 400\n",
      "gene: 802 -> 400\n",
      "gene: 803 -> 400\n",
      "gene: 804 -> 400\n",
      "gene: 805 -> 400\n",
      "gene: 806 -> 400\n",
      "gene: 807 -> 400\n",
      "gene: 808 -> 400\n",
      "gene: 809 -> 400\n",
      "gene: 810 -> 400\n",
      "gene: 811 -> 400\n",
      "gene: 812 -> 400\n",
      "gene: 813 -> 400\n",
      "gene: 814 -> 400\n",
      "gene: 815 -> 400\n",
      "gene: 816 -> 400\n",
      "gene: 817 -> 400\n",
      "gene: 818 -> 400\n",
      "gene: 819 -> 400\n",
      "gene: 820 -> 400\n",
      "gene: 821 -> 400\n",
      "gene: 822 -> 400\n",
      "gene: 823 -> 400\n",
      "gene: 824 -> 400\n",
      "gene: 825 -> 400\n",
      "gene: 826 -> 400\n",
      "gene: 827 -> 400\n",
      "gene: 828 -> 400\n",
      "gene: 829 -> 400\n",
      "gene: 830 -> 400\n",
      "gene: 831 -> 400\n",
      "gene: 832 -> 400\n",
      "gene: 833 -> 400\n",
      "gene: 834 -> 400\n",
      "gene: 835 -> 400\n",
      "gene: 836 -> 400\n",
      "gene: 837 -> 400\n",
      "gene: 838 -> 400\n",
      "gene: 839 -> 400\n",
      "gene: 840 -> 400\n",
      "gene: 841 -> 400\n",
      "gene: 842 -> 400\n",
      "gene: 843 -> 400\n",
      "gene: 844 -> 400\n",
      "gene: 845 -> 400\n",
      "gene: 846 -> 400\n",
      "gene: 847 -> 400\n",
      "gene: 848 -> 400\n",
      "gene: 849 -> 400\n",
      "gene: 850 -> 400\n",
      "gene: 851 -> 400\n",
      "gene: 852 -> 400\n",
      "gene: 853 -> 400\n",
      "gene: 854 -> 400\n",
      "gene: 855 -> 400\n",
      "gene: 856 -> 400\n",
      "gene: 857 -> 400\n",
      "gene: 858 -> 400\n",
      "gene: 859 -> 400\n",
      "gene: 860 -> 400\n",
      "gene: 861 -> 400\n",
      "gene: 862 -> 400\n",
      "gene: 863 -> 400\n",
      "gene: 864 -> 400\n",
      "gene: 866 -> 400\n",
      "gene: 867 -> 400\n",
      "gene: 868 -> 400\n",
      "gene: 869 -> 400\n",
      "gene: 870 -> 400\n",
      "gene: 871 -> 400\n",
      "gene: 872 -> 400\n",
      "gene: 873 -> 400\n",
      "gene: 874 -> 400\n",
      "gene: 875 -> 400\n",
      "gene: 876 -> 400\n",
      "gene: 877 -> 400\n",
      "gene: 878 -> 400\n",
      "gene: 879 -> 400\n",
      "gene: 880 -> 400\n",
      "gene: 881 -> 400\n",
      "gene: 882 -> 400\n",
      "gene: 883 -> 400\n",
      "gene: 884 -> 400\n",
      "gene: 885 -> 400\n",
      "gene: 886 -> 400\n",
      "gene: 887 -> 400\n",
      "gene: 888 -> 400\n",
      "gene: 889 -> 400\n",
      "gene: 890 -> 400\n",
      "gene: 891 -> 400\n",
      "gene: 892 -> 400\n",
      "gene: 893 -> 400\n",
      "gene: 894 -> 400\n",
      "gene: 895 -> 400\n",
      "gene: 896 -> 400\n",
      "gene: 897 -> 400\n",
      "gene: 898 -> 400\n",
      "gene: 899 -> 400\n",
      "gene: 900 -> 400\n",
      "gene: 901 -> 400\n",
      "gene: 902 -> 381\n",
      "gene: 903 -> 400\n",
      "gene: 904 -> 400\n",
      "gene: 905 -> 400\n",
      "gene: 906 -> 400\n",
      "gene: 907 -> 400\n",
      "gene: 908 -> 400\n",
      "gene: 909 -> 400\n",
      "gene: 910 -> 400\n",
      "gene: 911 -> 400\n",
      "gene: 912 -> 400\n",
      "gene: 913 -> 400\n",
      "gene: 914 -> 400\n",
      "gene: 915 -> 400\n",
      "gene: 916 -> 400\n",
      "gene: 917 -> 400\n",
      "gene: 918 -> 400\n",
      "gene: 919 -> 400\n",
      "gene: 920 -> 400\n",
      "gene: 921 -> 400\n",
      "gene: 922 -> 400\n",
      "gene: 923 -> 400\n",
      "gene: 925 -> 400\n",
      "gene: 926 -> 400\n",
      "gene: 927 -> 400\n",
      "gene: 928 -> 400\n",
      "gene: 929 -> 400\n",
      "gene: 930 -> 400\n",
      "gene: 931 -> 400\n",
      "gene: 932 -> 400\n",
      "gene: 933 -> 400\n",
      "gene: 934 -> 400\n",
      "gene: 935 -> 400\n",
      "gene: 936 -> 400\n",
      "gene: 937 -> 400\n",
      "gene: 938 -> 400\n",
      "gene: 939 -> 400\n",
      "gene: 940 -> 400\n",
      "gene: 941 -> 400\n",
      "gene: 942 -> 400\n",
      "gene: 943 -> 400\n",
      "gene: 944 -> 400\n",
      "gene: 945 -> 400\n",
      "gene: 946 -> 400\n",
      "gene: 947 -> 400\n",
      "gene: 948 -> 400\n",
      "gene: 949 -> 400\n",
      "gene: 950 -> 400\n",
      "gene: 951 -> 400\n",
      "gene: 952 -> 400\n",
      "gene: 953 -> 400\n",
      "gene: 954 -> 400\n",
      "gene: 955 -> 400\n",
      "gene: 956 -> 400\n",
      "gene: 957 -> 400\n",
      "gene: 958 -> 400\n",
      "gene: 959 -> 400\n",
      "gene: 960 -> 400\n",
      "gene: 961 -> 400\n",
      "gene: 962 -> 400\n",
      "gene: 963 -> 400\n",
      "gene: 964 -> 400\n",
      "gene: 965 -> 400\n",
      "gene: 966 -> 400\n",
      "gene: 967 -> 400\n",
      "gene: 968 -> 400\n",
      "gene: 969 -> 400\n",
      "gene: 970 -> 400\n",
      "gene: 971 -> 400\n",
      "gene: 972 -> 400\n",
      "gene: 973 -> 400\n",
      "gene: 974 -> 400\n",
      "gene: 975 -> 400\n",
      "gene: 976 -> 400\n",
      "gene: 977 -> 400\n",
      "gene: 978 -> 400\n",
      "gene: 979 -> 400\n",
      "gene: 980 -> 400\n",
      "gene: 981 -> 400\n",
      "gene: 982 -> 400\n",
      "gene: 983 -> 400\n",
      "gene: 984 -> 400\n",
      "gene: 985 -> 400\n",
      "gene: 986 -> 400\n",
      "gene: 987 -> 400\n",
      "gene: 988 -> 400\n",
      "gene: 989 -> 400\n",
      "gene: 990 -> 400\n",
      "gene: 991 -> 127\n",
      "True\n"
     ]
    }
   ],
   "source": [
    "reload(check)\n",
    "reg_size_dic, len_check = check._check_region_size(full_records, \n",
    "                                                   min_size=min_num_probes)\n",
    "print(len_check)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "/* Put everything inside the global mpl namespace */\n",
       "/* global mpl */\n",
       "window.mpl = {};\n",
       "\n",
       "mpl.get_websocket_type = function () {\n",
       "    if (typeof WebSocket !== 'undefined') {\n",
       "        return WebSocket;\n",
       "    } else if (typeof MozWebSocket !== 'undefined') {\n",
       "        return MozWebSocket;\n",
       "    } else {\n",
       "        alert(\n",
       "            'Your browser does not have WebSocket support. ' +\n",
       "                'Please try Chrome, Safari or Firefox ≥ 6. ' +\n",
       "                'Firefox 4 and 5 are also supported but you ' +\n",
       "                'have to enable WebSockets in about:config.'\n",
       "        );\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure = function (figure_id, websocket, ondownload, parent_element) {\n",
       "    this.id = figure_id;\n",
       "\n",
       "    this.ws = websocket;\n",
       "\n",
       "    this.supports_binary = this.ws.binaryType !== undefined;\n",
       "\n",
       "    if (!this.supports_binary) {\n",
       "        var warnings = document.getElementById('mpl-warnings');\n",
       "        if (warnings) {\n",
       "            warnings.style.display = 'block';\n",
       "            warnings.textContent =\n",
       "                'This browser does not support binary websocket messages. ' +\n",
       "                'Performance may be slow.';\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.imageObj = new Image();\n",
       "\n",
       "    this.context = undefined;\n",
       "    this.message = undefined;\n",
       "    this.canvas = undefined;\n",
       "    this.rubberband_canvas = undefined;\n",
       "    this.rubberband_context = undefined;\n",
       "    this.format_dropdown = undefined;\n",
       "\n",
       "    this.image_mode = 'full';\n",
       "\n",
       "    this.root = document.createElement('div');\n",
       "    this.root.setAttribute('style', 'display: inline-block');\n",
       "    this._root_extra_style(this.root);\n",
       "\n",
       "    parent_element.appendChild(this.root);\n",
       "\n",
       "    this._init_header(this);\n",
       "    this._init_canvas(this);\n",
       "    this._init_toolbar(this);\n",
       "\n",
       "    var fig = this;\n",
       "\n",
       "    this.waiting = false;\n",
       "\n",
       "    this.ws.onopen = function () {\n",
       "        fig.send_message('supports_binary', { value: fig.supports_binary });\n",
       "        fig.send_message('send_image_mode', {});\n",
       "        if (fig.ratio !== 1) {\n",
       "            fig.send_message('set_dpi_ratio', { dpi_ratio: fig.ratio });\n",
       "        }\n",
       "        fig.send_message('refresh', {});\n",
       "    };\n",
       "\n",
       "    this.imageObj.onload = function () {\n",
       "        if (fig.image_mode === 'full') {\n",
       "            // Full images could contain transparency (where diff images\n",
       "            // almost always do), so we need to clear the canvas so that\n",
       "            // there is no ghosting.\n",
       "            fig.context.clearRect(0, 0, fig.canvas.width, fig.canvas.height);\n",
       "        }\n",
       "        fig.context.drawImage(fig.imageObj, 0, 0);\n",
       "    };\n",
       "\n",
       "    this.imageObj.onunload = function () {\n",
       "        fig.ws.close();\n",
       "    };\n",
       "\n",
       "    this.ws.onmessage = this._make_on_message_function(this);\n",
       "\n",
       "    this.ondownload = ondownload;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_header = function () {\n",
       "    var titlebar = document.createElement('div');\n",
       "    titlebar.classList =\n",
       "        'ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix';\n",
       "    var titletext = document.createElement('div');\n",
       "    titletext.classList = 'ui-dialog-title';\n",
       "    titletext.setAttribute(\n",
       "        'style',\n",
       "        'width: 100%; text-align: center; padding: 3px;'\n",
       "    );\n",
       "    titlebar.appendChild(titletext);\n",
       "    this.root.appendChild(titlebar);\n",
       "    this.header = titletext;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._init_canvas = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var canvas_div = (this.canvas_div = document.createElement('div'));\n",
       "    canvas_div.setAttribute(\n",
       "        'style',\n",
       "        'border: 1px solid #ddd;' +\n",
       "            'box-sizing: content-box;' +\n",
       "            'clear: both;' +\n",
       "            'min-height: 1px;' +\n",
       "            'min-width: 1px;' +\n",
       "            'outline: 0;' +\n",
       "            'overflow: hidden;' +\n",
       "            'position: relative;' +\n",
       "            'resize: both;'\n",
       "    );\n",
       "\n",
       "    function on_keyboard_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.key_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    canvas_div.addEventListener(\n",
       "        'keydown',\n",
       "        on_keyboard_event_closure('key_press')\n",
       "    );\n",
       "    canvas_div.addEventListener(\n",
       "        'keyup',\n",
       "        on_keyboard_event_closure('key_release')\n",
       "    );\n",
       "\n",
       "    this._canvas_extra_style(canvas_div);\n",
       "    this.root.appendChild(canvas_div);\n",
       "\n",
       "    var canvas = (this.canvas = document.createElement('canvas'));\n",
       "    canvas.classList.add('mpl-canvas');\n",
       "    canvas.setAttribute('style', 'box-sizing: content-box;');\n",
       "\n",
       "    this.context = canvas.getContext('2d');\n",
       "\n",
       "    var backingStore =\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        this.context.webkitBackingStorePixelRatio ||\n",
       "        this.context.mozBackingStorePixelRatio ||\n",
       "        this.context.msBackingStorePixelRatio ||\n",
       "        this.context.oBackingStorePixelRatio ||\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        1;\n",
       "\n",
       "    this.ratio = (window.devicePixelRatio || 1) / backingStore;\n",
       "    if (this.ratio !== 1) {\n",
       "        fig.send_message('set_dpi_ratio', { dpi_ratio: this.ratio });\n",
       "    }\n",
       "\n",
       "    var rubberband_canvas = (this.rubberband_canvas = document.createElement(\n",
       "        'canvas'\n",
       "    ));\n",
       "    rubberband_canvas.setAttribute(\n",
       "        'style',\n",
       "        'box-sizing: content-box; position: absolute; left: 0; top: 0; z-index: 1;'\n",
       "    );\n",
       "\n",
       "    var resizeObserver = new ResizeObserver(function (entries) {\n",
       "        var nentries = entries.length;\n",
       "        for (var i = 0; i < nentries; i++) {\n",
       "            var entry = entries[i];\n",
       "            var width, height;\n",
       "            if (entry.contentBoxSize) {\n",
       "                if (entry.contentBoxSize instanceof Array) {\n",
       "                    // Chrome 84 implements new version of spec.\n",
       "                    width = entry.contentBoxSize[0].inlineSize;\n",
       "                    height = entry.contentBoxSize[0].blockSize;\n",
       "                } else {\n",
       "                    // Firefox implements old version of spec.\n",
       "                    width = entry.contentBoxSize.inlineSize;\n",
       "                    height = entry.contentBoxSize.blockSize;\n",
       "                }\n",
       "            } else {\n",
       "                // Chrome <84 implements even older version of spec.\n",
       "                width = entry.contentRect.width;\n",
       "                height = entry.contentRect.height;\n",
       "            }\n",
       "\n",
       "            // Keep the size of the canvas and rubber band canvas in sync with\n",
       "            // the canvas container.\n",
       "            if (entry.devicePixelContentBoxSize) {\n",
       "                // Chrome 84 implements new version of spec.\n",
       "                canvas.setAttribute(\n",
       "                    'width',\n",
       "                    entry.devicePixelContentBoxSize[0].inlineSize\n",
       "                );\n",
       "                canvas.setAttribute(\n",
       "                    'height',\n",
       "                    entry.devicePixelContentBoxSize[0].blockSize\n",
       "                );\n",
       "            } else {\n",
       "                canvas.setAttribute('width', width * fig.ratio);\n",
       "                canvas.setAttribute('height', height * fig.ratio);\n",
       "            }\n",
       "            canvas.setAttribute(\n",
       "                'style',\n",
       "                'width: ' + width + 'px; height: ' + height + 'px;'\n",
       "            );\n",
       "\n",
       "            rubberband_canvas.setAttribute('width', width);\n",
       "            rubberband_canvas.setAttribute('height', height);\n",
       "\n",
       "            // And update the size in Python. We ignore the initial 0/0 size\n",
       "            // that occurs as the element is placed into the DOM, which should\n",
       "            // otherwise not happen due to the minimum size styling.\n",
       "            if (width != 0 && height != 0) {\n",
       "                fig.request_resize(width, height);\n",
       "            }\n",
       "        }\n",
       "    });\n",
       "    resizeObserver.observe(canvas_div);\n",
       "\n",
       "    function on_mouse_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.mouse_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousedown',\n",
       "        on_mouse_event_closure('button_press')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseup',\n",
       "        on_mouse_event_closure('button_release')\n",
       "    );\n",
       "    // Throttle sequential mouse events to 1 every 20ms.\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousemove',\n",
       "        on_mouse_event_closure('motion_notify')\n",
       "    );\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseenter',\n",
       "        on_mouse_event_closure('figure_enter')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseleave',\n",
       "        on_mouse_event_closure('figure_leave')\n",
       "    );\n",
       "\n",
       "    canvas_div.addEventListener('wheel', function (event) {\n",
       "        if (event.deltaY < 0) {\n",
       "            event.step = 1;\n",
       "        } else {\n",
       "            event.step = -1;\n",
       "        }\n",
       "        on_mouse_event_closure('scroll')(event);\n",
       "    });\n",
       "\n",
       "    canvas_div.appendChild(canvas);\n",
       "    canvas_div.appendChild(rubberband_canvas);\n",
       "\n",
       "    this.rubberband_context = rubberband_canvas.getContext('2d');\n",
       "    this.rubberband_context.strokeStyle = '#000000';\n",
       "\n",
       "    this._resize_canvas = function (width, height, forward) {\n",
       "        if (forward) {\n",
       "            canvas_div.style.width = width + 'px';\n",
       "            canvas_div.style.height = height + 'px';\n",
       "        }\n",
       "    };\n",
       "\n",
       "    // Disable right mouse context menu.\n",
       "    this.rubberband_canvas.addEventListener('contextmenu', function (_e) {\n",
       "        event.preventDefault();\n",
       "        return false;\n",
       "    });\n",
       "\n",
       "    function set_focus() {\n",
       "        canvas.focus();\n",
       "        canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    window.setTimeout(set_focus, 100);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'mpl-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'mpl-button-group';\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'mpl-button-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        var button = (fig.buttons[name] = document.createElement('button'));\n",
       "        button.classList = 'mpl-widget';\n",
       "        button.setAttribute('role', 'button');\n",
       "        button.setAttribute('aria-disabled', 'false');\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "\n",
       "        var icon_img = document.createElement('img');\n",
       "        icon_img.src = '_images/' + image + '.png';\n",
       "        icon_img.srcset = '_images/' + image + '_large.png 2x';\n",
       "        icon_img.alt = tooltip;\n",
       "        button.appendChild(icon_img);\n",
       "\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    var fmt_picker = document.createElement('select');\n",
       "    fmt_picker.classList = 'mpl-widget';\n",
       "    toolbar.appendChild(fmt_picker);\n",
       "    this.format_dropdown = fmt_picker;\n",
       "\n",
       "    for (var ind in mpl.extensions) {\n",
       "        var fmt = mpl.extensions[ind];\n",
       "        var option = document.createElement('option');\n",
       "        option.selected = fmt === mpl.default_extension;\n",
       "        option.innerHTML = fmt;\n",
       "        fmt_picker.appendChild(option);\n",
       "    }\n",
       "\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.request_resize = function (x_pixels, y_pixels) {\n",
       "    // Request matplotlib to resize the figure. Matplotlib will then trigger a resize in the client,\n",
       "    // which will in turn request a refresh of the image.\n",
       "    this.send_message('resize', { width: x_pixels, height: y_pixels });\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_message = function (type, properties) {\n",
       "    properties['type'] = type;\n",
       "    properties['figure_id'] = this.id;\n",
       "    this.ws.send(JSON.stringify(properties));\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_draw_message = function () {\n",
       "    if (!this.waiting) {\n",
       "        this.waiting = true;\n",
       "        this.ws.send(JSON.stringify({ type: 'draw', figure_id: this.id }));\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    var format_dropdown = fig.format_dropdown;\n",
       "    var format = format_dropdown.options[format_dropdown.selectedIndex].value;\n",
       "    fig.ondownload(fig, format);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_resize = function (fig, msg) {\n",
       "    var size = msg['size'];\n",
       "    if (size[0] !== fig.canvas.width || size[1] !== fig.canvas.height) {\n",
       "        fig._resize_canvas(size[0], size[1], msg['forward']);\n",
       "        fig.send_message('refresh', {});\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_rubberband = function (fig, msg) {\n",
       "    var x0 = msg['x0'] / fig.ratio;\n",
       "    var y0 = (fig.canvas.height - msg['y0']) / fig.ratio;\n",
       "    var x1 = msg['x1'] / fig.ratio;\n",
       "    var y1 = (fig.canvas.height - msg['y1']) / fig.ratio;\n",
       "    x0 = Math.floor(x0) + 0.5;\n",
       "    y0 = Math.floor(y0) + 0.5;\n",
       "    x1 = Math.floor(x1) + 0.5;\n",
       "    y1 = Math.floor(y1) + 0.5;\n",
       "    var min_x = Math.min(x0, x1);\n",
       "    var min_y = Math.min(y0, y1);\n",
       "    var width = Math.abs(x1 - x0);\n",
       "    var height = Math.abs(y1 - y0);\n",
       "\n",
       "    fig.rubberband_context.clearRect(\n",
       "        0,\n",
       "        0,\n",
       "        fig.canvas.width / fig.ratio,\n",
       "        fig.canvas.height / fig.ratio\n",
       "    );\n",
       "\n",
       "    fig.rubberband_context.strokeRect(min_x, min_y, width, height);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_figure_label = function (fig, msg) {\n",
       "    // Updates the figure title.\n",
       "    fig.header.textContent = msg['label'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_cursor = function (fig, msg) {\n",
       "    var cursor = msg['cursor'];\n",
       "    switch (cursor) {\n",
       "        case 0:\n",
       "            cursor = 'pointer';\n",
       "            break;\n",
       "        case 1:\n",
       "            cursor = 'default';\n",
       "            break;\n",
       "        case 2:\n",
       "            cursor = 'crosshair';\n",
       "            break;\n",
       "        case 3:\n",
       "            cursor = 'move';\n",
       "            break;\n",
       "    }\n",
       "    fig.rubberband_canvas.style.cursor = cursor;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_message = function (fig, msg) {\n",
       "    fig.message.textContent = msg['message'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_draw = function (fig, _msg) {\n",
       "    // Request the server to send over a new figure.\n",
       "    fig.send_draw_message();\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_image_mode = function (fig, msg) {\n",
       "    fig.image_mode = msg['mode'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_history_buttons = function (fig, msg) {\n",
       "    for (var key in msg) {\n",
       "        if (!(key in fig.buttons)) {\n",
       "            continue;\n",
       "        }\n",
       "        fig.buttons[key].disabled = !msg[key];\n",
       "        fig.buttons[key].setAttribute('aria-disabled', !msg[key]);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_navigate_mode = function (fig, msg) {\n",
       "    if (msg['mode'] === 'PAN') {\n",
       "        fig.buttons['Pan'].classList.add('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    } else if (msg['mode'] === 'ZOOM') {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.add('active');\n",
       "    } else {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Called whenever the canvas gets updated.\n",
       "    this.send_message('ack', {});\n",
       "};\n",
       "\n",
       "// A function to construct a web socket function for onmessage handling.\n",
       "// Called in the figure constructor.\n",
       "mpl.figure.prototype._make_on_message_function = function (fig) {\n",
       "    return function socket_on_message(evt) {\n",
       "        if (evt.data instanceof Blob) {\n",
       "            /* FIXME: We get \"Resource interpreted as Image but\n",
       "             * transferred with MIME type text/plain:\" errors on\n",
       "             * Chrome.  But how to set the MIME type?  It doesn't seem\n",
       "             * to be part of the websocket stream */\n",
       "            evt.data.type = 'image/png';\n",
       "\n",
       "            /* Free the memory for the previous frames */\n",
       "            if (fig.imageObj.src) {\n",
       "                (window.URL || window.webkitURL).revokeObjectURL(\n",
       "                    fig.imageObj.src\n",
       "                );\n",
       "            }\n",
       "\n",
       "            fig.imageObj.src = (window.URL || window.webkitURL).createObjectURL(\n",
       "                evt.data\n",
       "            );\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        } else if (\n",
       "            typeof evt.data === 'string' &&\n",
       "            evt.data.slice(0, 21) === 'data:image/png;base64'\n",
       "        ) {\n",
       "            fig.imageObj.src = evt.data;\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        var msg = JSON.parse(evt.data);\n",
       "        var msg_type = msg['type'];\n",
       "\n",
       "        // Call the  \"handle_{type}\" callback, which takes\n",
       "        // the figure and JSON message as its only arguments.\n",
       "        try {\n",
       "            var callback = fig['handle_' + msg_type];\n",
       "        } catch (e) {\n",
       "            console.log(\n",
       "                \"No handler for the '\" + msg_type + \"' message type: \",\n",
       "                msg\n",
       "            );\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        if (callback) {\n",
       "            try {\n",
       "                // console.log(\"Handling '\" + msg_type + \"' message: \", msg);\n",
       "                callback(fig, msg);\n",
       "            } catch (e) {\n",
       "                console.log(\n",
       "                    \"Exception inside the 'handler_\" + msg_type + \"' callback:\",\n",
       "                    e,\n",
       "                    e.stack,\n",
       "                    msg\n",
       "                );\n",
       "            }\n",
       "        }\n",
       "    };\n",
       "};\n",
       "\n",
       "// from http://stackoverflow.com/questions/1114465/getting-mouse-location-in-canvas\n",
       "mpl.findpos = function (e) {\n",
       "    //this section is from http://www.quirksmode.org/js/events_properties.html\n",
       "    var targ;\n",
       "    if (!e) {\n",
       "        e = window.event;\n",
       "    }\n",
       "    if (e.target) {\n",
       "        targ = e.target;\n",
       "    } else if (e.srcElement) {\n",
       "        targ = e.srcElement;\n",
       "    }\n",
       "    if (targ.nodeType === 3) {\n",
       "        // defeat Safari bug\n",
       "        targ = targ.parentNode;\n",
       "    }\n",
       "\n",
       "    // pageX,Y are the mouse positions relative to the document\n",
       "    var boundingRect = targ.getBoundingClientRect();\n",
       "    var x = e.pageX - (boundingRect.left + document.body.scrollLeft);\n",
       "    var y = e.pageY - (boundingRect.top + document.body.scrollTop);\n",
       "\n",
       "    return { x: x, y: y };\n",
       "};\n",
       "\n",
       "/*\n",
       " * return a copy of an object with only non-object keys\n",
       " * we need this to avoid circular references\n",
       " * http://stackoverflow.com/a/24161582/3208463\n",
       " */\n",
       "function simpleKeys(original) {\n",
       "    return Object.keys(original).reduce(function (obj, key) {\n",
       "        if (typeof original[key] !== 'object') {\n",
       "            obj[key] = original[key];\n",
       "        }\n",
       "        return obj;\n",
       "    }, {});\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.mouse_event = function (event, name) {\n",
       "    var canvas_pos = mpl.findpos(event);\n",
       "\n",
       "    if (name === 'button_press') {\n",
       "        this.canvas.focus();\n",
       "        this.canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    var x = canvas_pos.x * this.ratio;\n",
       "    var y = canvas_pos.y * this.ratio;\n",
       "\n",
       "    this.send_message(name, {\n",
       "        x: x,\n",
       "        y: y,\n",
       "        button: event.button,\n",
       "        step: event.step,\n",
       "        guiEvent: simpleKeys(event),\n",
       "    });\n",
       "\n",
       "    /* This prevents the web browser from automatically changing to\n",
       "     * the text insertion cursor when the button is pressed.  We want\n",
       "     * to control all of the cursor setting manually through the\n",
       "     * 'cursor' event from matplotlib */\n",
       "    event.preventDefault();\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (_event, _name) {\n",
       "    // Handle any extra behaviour associated with a key event\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.key_event = function (event, name) {\n",
       "    // Prevent repeat events\n",
       "    if (name === 'key_press') {\n",
       "        if (event.which === this._key) {\n",
       "            return;\n",
       "        } else {\n",
       "            this._key = event.which;\n",
       "        }\n",
       "    }\n",
       "    if (name === 'key_release') {\n",
       "        this._key = null;\n",
       "    }\n",
       "\n",
       "    var value = '';\n",
       "    if (event.ctrlKey && event.which !== 17) {\n",
       "        value += 'ctrl+';\n",
       "    }\n",
       "    if (event.altKey && event.which !== 18) {\n",
       "        value += 'alt+';\n",
       "    }\n",
       "    if (event.shiftKey && event.which !== 16) {\n",
       "        value += 'shift+';\n",
       "    }\n",
       "\n",
       "    value += 'k';\n",
       "    value += event.which.toString();\n",
       "\n",
       "    this._key_event_extra(event, name);\n",
       "\n",
       "    this.send_message(name, { key: value, guiEvent: simpleKeys(event) });\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onclick = function (name) {\n",
       "    if (name === 'download') {\n",
       "        this.handle_save(this, null);\n",
       "    } else {\n",
       "        this.send_message('toolbar_button', { name: name });\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onmouseover = function (tooltip) {\n",
       "    this.message.textContent = tooltip;\n",
       "};\n",
       "mpl.toolbar_items = [[\"Home\", \"Reset original view\", \"fa fa-home icon-home\", \"home\"], [\"Back\", \"Back to previous view\", \"fa fa-arrow-left icon-arrow-left\", \"back\"], [\"Forward\", \"Forward to next view\", \"fa fa-arrow-right icon-arrow-right\", \"forward\"], [\"\", \"\", \"\", \"\"], [\"Pan\", \"Left button pans, Right button zooms\\nx/y fixes axis, CTRL fixes aspect\", \"fa fa-arrows icon-move\", \"pan\"], [\"Zoom\", \"Zoom to rectangle\\nx/y fixes axis, CTRL fixes aspect\", \"fa fa-square-o icon-check-empty\", \"zoom\"], [\"\", \"\", \"\", \"\"], [\"Download\", \"Download plot\", \"fa fa-floppy-o icon-save\", \"download\"]];\n",
       "\n",
       "mpl.extensions = [\"eps\", \"jpeg\", \"pdf\", \"png\", \"ps\", \"raw\", \"svg\", \"tif\"];\n",
       "\n",
       "mpl.default_extension = \"png\";/* global mpl */\n",
       "\n",
       "var comm_websocket_adapter = function (comm) {\n",
       "    // Create a \"websocket\"-like object which calls the given IPython comm\n",
       "    // object with the appropriate methods. Currently this is a non binary\n",
       "    // socket, so there is still some room for performance tuning.\n",
       "    var ws = {};\n",
       "\n",
       "    ws.close = function () {\n",
       "        comm.close();\n",
       "    };\n",
       "    ws.send = function (m) {\n",
       "        //console.log('sending', m);\n",
       "        comm.send(m);\n",
       "    };\n",
       "    // Register the callback with on_msg.\n",
       "    comm.on_msg(function (msg) {\n",
       "        //console.log('receiving', msg['content']['data'], msg);\n",
       "        // Pass the mpl event to the overridden (by mpl) onmessage function.\n",
       "        ws.onmessage(msg['content']['data']);\n",
       "    });\n",
       "    return ws;\n",
       "};\n",
       "\n",
       "mpl.mpl_figure_comm = function (comm, msg) {\n",
       "    // This is the function which gets called when the mpl process\n",
       "    // starts-up an IPython Comm through the \"matplotlib\" channel.\n",
       "\n",
       "    var id = msg.content.data.id;\n",
       "    // Get hold of the div created by the display call when the Comm\n",
       "    // socket was opened in Python.\n",
       "    var element = document.getElementById(id);\n",
       "    var ws_proxy = comm_websocket_adapter(comm);\n",
       "\n",
       "    function ondownload(figure, _format) {\n",
       "        window.open(figure.canvas.toDataURL());\n",
       "    }\n",
       "\n",
       "    var fig = new mpl.figure(id, ws_proxy, ondownload, element);\n",
       "\n",
       "    // Call onopen now - mpl needs it, as it is assuming we've passed it a real\n",
       "    // web socket which is closed, not our websocket->open comm proxy.\n",
       "    ws_proxy.onopen();\n",
       "\n",
       "    fig.parent_element = element;\n",
       "    fig.cell_info = mpl.find_output_cell(\"<div id='\" + id + \"'></div>\");\n",
       "    if (!fig.cell_info) {\n",
       "        console.error('Failed to find cell for figure', id, fig);\n",
       "        return;\n",
       "    }\n",
       "    fig.cell_info[0].output_area.element.one(\n",
       "        'cleared',\n",
       "        { fig: fig },\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_close = function (fig, msg) {\n",
       "    var width = fig.canvas.width / fig.ratio;\n",
       "    fig.cell_info[0].output_area.element.off(\n",
       "        'cleared',\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "\n",
       "    // Update the output cell to use the data from the current canvas.\n",
       "    fig.push_to_output();\n",
       "    var dataURL = fig.canvas.toDataURL();\n",
       "    // Re-enable the keyboard manager in IPython - without this line, in FF,\n",
       "    // the notebook keyboard shortcuts fail.\n",
       "    IPython.keyboard_manager.enable();\n",
       "    fig.parent_element.innerHTML =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "    fig.close_ws(fig, msg);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.close_ws = function (fig, msg) {\n",
       "    fig.send_message('closing', msg);\n",
       "    // fig.ws.close()\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.push_to_output = function (_remove_interactive) {\n",
       "    // Turn the data on the canvas into data in the output cell.\n",
       "    var width = this.canvas.width / this.ratio;\n",
       "    var dataURL = this.canvas.toDataURL();\n",
       "    this.cell_info[1]['text/html'] =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Tell IPython that the notebook contents must change.\n",
       "    IPython.notebook.set_dirty(true);\n",
       "    this.send_message('ack', {});\n",
       "    var fig = this;\n",
       "    // Wait a second, then push the new image to the DOM so\n",
       "    // that it is saved nicely (might be nice to debounce this).\n",
       "    setTimeout(function () {\n",
       "        fig.push_to_output();\n",
       "    }, 1000);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'btn-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'btn-group';\n",
       "    var button;\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'btn-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        button = fig.buttons[name] = document.createElement('button');\n",
       "        button.classList = 'btn btn-default';\n",
       "        button.href = '#';\n",
       "        button.title = name;\n",
       "        button.innerHTML = '<i class=\"fa ' + image + ' fa-lg\"></i>';\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    // Add the status bar.\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message pull-right';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "\n",
       "    // Add the close button to the window.\n",
       "    var buttongrp = document.createElement('div');\n",
       "    buttongrp.classList = 'btn-group inline pull-right';\n",
       "    button = document.createElement('button');\n",
       "    button.classList = 'btn btn-mini btn-primary';\n",
       "    button.href = '#';\n",
       "    button.title = 'Stop Interaction';\n",
       "    button.innerHTML = '<i class=\"fa fa-power-off icon-remove icon-large\"></i>';\n",
       "    button.addEventListener('click', function (_evt) {\n",
       "        fig.handle_close(fig, {});\n",
       "    });\n",
       "    button.addEventListener(\n",
       "        'mouseover',\n",
       "        on_mouseover_closure('Stop Interaction')\n",
       "    );\n",
       "    buttongrp.appendChild(button);\n",
       "    var titlebar = this.root.querySelector('.ui-dialog-titlebar');\n",
       "    titlebar.insertBefore(buttongrp, titlebar.firstChild);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._remove_fig_handler = function (event) {\n",
       "    var fig = event.data.fig;\n",
       "    fig.close_ws(fig, {});\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (el) {\n",
       "    el.style.boxSizing = 'content-box'; // override notebook setting of border-box.\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (el) {\n",
       "    // this is important to make the div 'focusable\n",
       "    el.setAttribute('tabindex', 0);\n",
       "    // reach out to IPython and tell the keyboard manager to turn it's self\n",
       "    // off when our div gets focus\n",
       "\n",
       "    // location in version 3\n",
       "    if (IPython.notebook.keyboard_manager) {\n",
       "        IPython.notebook.keyboard_manager.register_events(el);\n",
       "    } else {\n",
       "        // location in version 2\n",
       "        IPython.keyboard_manager.register_events(el);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (event, _name) {\n",
       "    var manager = IPython.notebook.keyboard_manager;\n",
       "    if (!manager) {\n",
       "        manager = IPython.keyboard_manager;\n",
       "    }\n",
       "\n",
       "    // Check for shift+enter\n",
       "    if (event.shiftKey && event.which === 13) {\n",
       "        this.canvas_div.blur();\n",
       "        // select the cell after this one\n",
       "        var index = IPython.notebook.find_cell_index(this.cell_info[0]);\n",
       "        IPython.notebook.select(index + 1);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    fig.ondownload(fig, null);\n",
       "};\n",
       "\n",
       "mpl.find_output_cell = function (html_output) {\n",
       "    // Return the cell and output element which can be found *uniquely* in the notebook.\n",
       "    // Note - this is a bit hacky, but it is done because the \"notebook_saving.Notebook\"\n",
       "    // IPython event is triggered only after the cells have been serialised, which for\n",
       "    // our purposes (turning an active figure into a static one), is too late.\n",
       "    var cells = IPython.notebook.get_cells();\n",
       "    var ncells = cells.length;\n",
       "    for (var i = 0; i < ncells; i++) {\n",
       "        var cell = cells[i];\n",
       "        if (cell.cell_type === 'code') {\n",
       "            for (var j = 0; j < cell.output_area.outputs.length; j++) {\n",
       "                var data = cell.output_area.outputs[j];\n",
       "                if (data.data) {\n",
       "                    // IPython >= 3 moved mimebundle to data attribute of output\n",
       "                    data = data.data;\n",
       "                }\n",
       "                if (data['text/html'] === html_output) {\n",
       "                    return [cell, data, j];\n",
       "                }\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "};\n",
       "\n",
       "// Register the function which deals with the matplotlib target/channel.\n",
       "// The kernel may be null if the page has been refreshed.\n",
       "if (IPython.notebook.kernel !== null) {\n",
       "    IPython.notebook.kernel.comm_manager.register_target(\n",
       "        'matplotlib',\n",
       "        mpl.mpl_figure_comm\n",
       "    );\n",
       "}\n"
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<img src=\"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAsAAAAIQCAYAAACPEdjAAAAAAXNSR0IArs4c6QAAIABJREFUeF7t3Q+snnV9//+PpSs0SzYMpaYUtdrSMJFWzaSMIeoITlAZ2lqQpAVSsxXCIAHafGndBKb8qbG6BSqdTXFCwMksVvtdZlaiETRdIcDSlVECtYNRAkbLSCy22vabz/XLOT/Ew7hfn9NTzp/HnZgIfZ/73Pfjerc8e/W6r77hwIEDB4oHAQIECBAgQIAAgTEi8AYBPEaOtLdJgAABAgQIECDQCQhgi0CAAAECBAgQIDCmBATwmDrc3iwBAgQIECBAgIAAtgMECBAgQIAAAQJjSkAAj6nD7c0SIECAAAECBAgIYDtAgAABAgQIECAwpgQE8Jg63N4sAQIECBAgQICAALYDBAgQIECAAAECY0pAAI+pw+3NEiBAgAABAgQICGA7QIAAAQIECBAgMKYEBPCYOtzeLAECBAgQIECAgAC2AwQIECBAgAABAmNKQACPqcPtzRIgQIAAAQIECAhgO0CAAAECBAgQIDCmBATwmDrc3iwBAgQIECBAgIAAtgMECBAgQIAAAQJjSkAAj6nD7c0SIECAAAECBAgIYDtAgAABAgQIECAwpgQE8Jg63N4sAQIECBAgQICAALYDBAgQIECAAAECY0pAAI+pw+3NEiBAgAABAgQICGA7QIAAAQIECBAgMKYEBPCYOtzeLAECBAgQIECAgAC2AwQIECBAgAABAmNKQACPqcPtzRIgQIAAAQIECAhgO0CAAAECBAgQIDCmBATwmDrc3iwBAgQIECBAgIAAtgMECBAgQIAAAQJjSkAAj6nD7c0SIECAAAECBAgIYDtAgAABAgQIECAwpgQE8Jg63N4sAQIECBAgQICAALYDBAgQIECAAAECY0pAAI+pw+3NEiBAgAABAgQICGA7QIAAAQIECBAgMKYEBPCYOtzeLAECBAgQIECAgAC2AwQIECBAgAABAmNKQACPqcPtzRIgQIAAAQIECAhgO0CAAAECBAgQIDCmBATwmDrc3iwBAgQIECBAgIAAtgMECBAgQIAAAQJjSkAAj6nD7c0SIECAAAECBAgIYDtAgAABAgQIECAwpgQE8Jg63N4sAQIECBAgQICAALYDBAgQIECAAAECY0pAAI+pw+3NEiBAgAABAgQICGA7QIAAAQIECBAgMKYEBPCYOtzeLAECBAgQIECAgAC2AwQIECBAgAABAmNKQACPqcPtzRIgQIAAAQIECAhgO0CAAAECBAgQIDCmBATwmDrc3iwBAgQIECBAgIAAtgMECBAgQIAAAQJjSkAAj6nD7c0SIECAAAECBAgIYDtAgAABAgQIECAwpgQE8Jg63N4sAQIECBAgQICAALYDBAgQIECAAAECY0pAAI+pw+3NEiBAgAABAgQICGA7QIAAAQIECBAgMKYERnwAf+5znysbN24sP/jBD/oP3COPPFIuv/zy8uCDD5ajjjqqXHbZZeWqq67q//H9+/eXa6+9tqxZs6bs2rWrnHrqqWXVqlVlxowZPT/HmNoSb5YAAQIECBAgMIoERnQAf/nLXy5XXHFFOe200/oD+Gc/+1k5/vjjyznnnNP92KZNm8oll1zSBe5FF13UHboav/Wfb7vttjJ16tSydOnSsn379rJ169YyYcKE0stzjKId8FYIECBAgAABAmNKYEQG8DPPPFM+/elPl/vuu6+8+c1vLm9605v6A/iGG24ot9xyS9mxY0cZP358dzCXLVtW1q1bVx577LGyd+/eMmnSpLJixYqyePHi7sdfeOGFcswxx5S1a9eW8847r7zWc4ypDfFmCRAgQIAAAQKjTGBEBvB3v/vdctddd5V6+cN1113XxW7fJRBnnXVWOfLII8udd97Zf6jqJRJnnHFGee6557rZOXPmlG3btpWZM2f2z9TLIGbNmtWdGX6t55g8eXJPa/DLX/6ybNmypRx99NH9Md7TFxoiQIAAAQIECAxS4Ne//nX56U9/Wk488cRyxBFHDPLZRteXj8gAfvkhuPDCC38jgGvEnnnmmeWmm27qH3v00UfLCSecUB544IHy1FNPlblz55bdu3eXiRMn9s/Mnz+/+3cbNmzoQvh/e44//MM/HHALXnzxxVL/1/eo1yJ/7GMfG10b490QIECAAAECI0pg8+bN5b3vfe+Ies1D/WJHXQDXD7Kdf/753Znhvke9vnf69OndJRP1DPCCBQvKvn37yrhx4/pnFi5cWHbu3Nl9oO61nqOeLR7occ0113TXF7/yURdvypQpQ30sPT8BAgQIECBAoF/g2WefLSeddFL5yU9+UqZNm0bmZQKjLoDraf56CcNAZ4Afeuih7sNu8+bNG/AM8J49e8r69eu7Pyr4357j3e9+94BL9MozwH2L9/TTT5djjz3W4hEgQIAAAQIEDpnAf//3f3efldIhv00+6gK4hmu99dntt9/e/24Hugb4iSee6M4K9z3qWd3Zs2d3H6B7refo9Rpgi3fIfo77RgQIECBAgMArBHTIq6/EqAvgG2+8sdx6663lySefLIcddlj3zq+++upyzz33dHeBqGd5a8CuXLmyLFq0qPvxV94F4rWeo9efYRavVylzBAgQIECAwMEW0CFjKICff/757j7AZ599dnd/33r97cUXX9xF8QUXXNBJLF++vKxevbq77Vm9JmbJkiXdtcH1jg31PsC9PEcvS2rxelEyQ4AAAQIECAyFgA4ZQwFc32q920P9298efvjh7sNnV155Zbn00kv7FeoH4Oq9getfhPHSSy91f5FGvfTh5ReIv9Zz9LKoFq8XJTMECBAgQIDAUAjokFEcwEOxMAfrOS3ewZL0PAQIECBAgEAqoEMEcLozB2Xe4h0URk9CgAABAgQINAjoEAHcsDaD/xKLN3hDz0CAAAECBAi0CegQAdy2OYP8Kos3SEBfToAAAQIECDQL6BAB3Lw8g/lCizcYPV9LgAABAgQIDEZAhwjgwexP89davGY6X0iAAAECBAgMUkCHCOBBrlDbl1u8NjdfRYAAAQIECAxeQIcI4MFvUcMzWLwGNF9CgAABAgQIHBQBHSKAD8oipU9i8VIx8wQIECBAgMDBEtAhAvhg7VL0PBYv4jJMgAABAgQIHEQBHSKAD+I69f5UFq93K5MECBAgQIDAwRXQIQL44G5Uj89m8XqEMkaAAAECBAgcdAEdIoAP+lL18oQWrxclMwQIECBAgMBQCOgQATwUe/Waz2nxXpPIAAECBAgQOCgC0/7P/+35eXbc+JGeZ0fyoA4RwK/L/lq814XdNyVAgACBMSgggH/7oOsQAfy6/FJg8V4Xdt+UAAECBMaggAAWwMnav+HAgQMHki8w27uAAO7dyiQBAgQIEBiMgAAWwMn+COBEK5wVwCGYcQIECBAg0CgggAVwsjoCONEKZwVwCGacAAECBAg0CghgAZysjgBOtMJZARyCGSdAgAABAo0CAlgAJ6sjgBOtcFYAh2DGCRAgQIBAo4AAFsDJ6gjgRCucFcAhmHECBAgQINAoIIAFcLI6AjjRCmcFcAhmnAABAgQINAoIYAGcrI4ATrTCWQEcghknQIAAAQKNAgJYACerI4ATrXBWAIdgxgkQIECAQKOAABbAyeoI4EQrnBXAIZhxAgQIECDQKCCABXCyOgI40QpnBXAIZpwAAQIECDQKCGABnKyOAE60wlkBHIIZJ0CAAAECjQICWAAnqyOAE61wVgCHYMYJECBAgECjgAAWwMnqCOBEK5wVwCGYcQIECBAg0CgggAVwsjoCONEKZwVwCGacAAECBAg0CghgAZysjgBOtMJZARyCGSdAgAABAo0CAlgAJ6sjgBOtcFYAh2DGCRAgQIBAo4AAFsDJ6gjgRCucFcAhmHECBAgQINAoIIAFcLI6AjjRCmcFcAhmnAABAgQINAoIYAGcrI4ATrTCWQEcghknQIAAAQKNAgJYACerI4ATrXBWAIdgxgkQIECAQKOAABbAyeoI4EQrnBXAIZhxAgQIECDQKCCABXCyOgI40QpnBXAIZpwAAQIECDQKCGABnKyOAE60wlkBHIIZJ0CAAAECjQICWAAnqyOAE61wVgCHYMYJECBAgECjgAAWwMnqCOBEK5wVwCGYcQIECBAg0CgggAVwsjoCONEKZwVwCGacAAECBAg0CghgAZysjgBOtMJZARyCGSdAgAABAo0CAlgAJ6sjgBOtcFYAh2DGCRAgQIBAo4AAFsDJ6gjgRCucFcAhmHECBAgQINAoIIAFcLI6AjjRCmcFcAhmnAABAgQINAoIYAGcrI4ATrTCWQEcghknQIAAAQKNAgJYACerI4ATrXBWAIdgxgkQIECAQKOAABbAyeoI4EQrnBXAIZhxAgQIECDQKCCABXCyOgI40QpnBXAIZpwAAQIECDQKCGABnKyOAE60wlkBHIIZJ0CAAAECjQICWAAnqyOAE61wVgCHYMYJECBAgECjgAAWwMnqCOBEK5wVwCGYcQIECBAg0CgggAVwsjoCONEKZwVwCGacAAECBAg0CghgAZysjgBOtMJZARyCGSdAgAABAo0CAlgAJ6sjgBOtcFYAh2DGCRAgQIBAo4AAFsDJ6gjgRCucFcAhmHECBAgQINAoIIAFcLI6AjjRCmcFcAhmnAABAgQINAoIYAGcrI4ATrTCWQEcghknQIAAAQKNAgJYACerI4ATrXBWAIdgxgkQIECAQKOAABbAyeoI4EQrnBXAIZhxAgQIECDQKCCABXCyOgI40QpnBXAIZpwAAQIECDQKCGABnKyOAE60wlkBHIIZJ0CAAAECjQICWAAnqyOAE61wVgCHYMYJECBAgECjgAAWwMnqCOBEK5wVwCGYcQIECBAg0CgggAVwsjoCONEKZwVwCGacAAECBAg0CghgAZysjgBOtMJZARyCGSdAgAABAo0CAlgAJ6sjgBOtcFYAh2DGCRAgQIBAo4AAFsDJ6gjgRCucFcAhmHECBAgQINAoIIAFcLI6ozKAf/WrX5Vrrrmm3H777WXXrl3lXe96V7npppvKKaec0tk88sgj5fLLLy8PPvhgOeqoo8pll11Wrrrqqn63/fv3l2uvvbasWbOm+/pTTz21rFq1qsyYMSOxLQI44jJMgAABAgSaBQSwAE6WZ1QG8Gc/+9ny93//9+Uf/uEfytvf/vYufr/5zW+W//zP/yyHH354Of7448s555xTrrjiirJp06ZyySWXdIF70UUXdXY1fus/33bbbWXq1Kll6dKlZfv27WXr1q1lwoQJPfsK4J6pDBIgQIAAgUEJCGABnCzQqAzgesb39NNPL1/84hc7ixdffLH8/u//fvmnf/qn8vjjj5dbbrml7Nixo4wfP7778WXLlpV169aVxx57rOzdu7dMmjSprFixoixevLj78RdeeKEcc8wxZe3ateW8887r2VcA90xlkAABAgQIDEpAAAvgZIFGZQDX+K3x+b3vfa+8+c1vLl/60pfKZz7zme4M7l/+5V+WI488stx55539Ths3bixnnHFGee6557ownjNnTtm2bVuZOXNm/0y9DGLWrFndmeFeHwK4VylzBAgQIEBgcAICWAAnGzQqA/g//uM/yrnnnlseffTRcthhh5Vx48Z1Z3/PPvvsLmLPPPPM7rKIvkedO+GEE8oDDzxQnnrqqTJ37tyye/fuMnHixP6Z+fPnd/9uw4YNPfsK4J6pDBIgQIAAgUEJCGABnCzQqAzgu+++u/zt3/5tWbJkSXcN71e/+tXyj//4j+WHP/xh+cQnPlHOP//8ct111/U71et7p0+fXu67777uDPCCBQvKvn37unDueyxcuLDs3Lmz1LPFr/aol1rU//U9nn322XLSSSeVp59+uhx77LHJcTFLgAABAgQIBAICWAAH61JGXQD/13/9VznuuOPKvffeW973vvf1W9T/X6/tfeKJJ8pZZ5014Bnghx56qPuw27x58wY8A7xnz56yfv36V/Wtd56oH6B75UMAJytplgABAgQI5AICWAAnWzPqArie/a2XK7z00kvliCOO6LeoZ4P/+Z//ubz1rW/tbn1Wb5HW9xjoGuAayvWscN+jXgM8e/bs7gN0r/ZwBjhZPbMECBAgQODgCQhgAZxs06gL4Hpbsz/6oz8q//Zv/9ZdftD3+PCHP9wF8cknn1xuvfXW8uSTT3bXB9fH1VdfXe65557uLhD1LO/kyZPLypUry6JFi7ofdxeIZKXMEiBAgACBQy8ggAVwsnWjLoDrX2LxgQ98oPz0pz/t7thQr739+te/Xm644YZy//33d/cFrvcBrh+Iq/f33bx5c7n44ou7KL7gggs6u+XLl5fVq1d3tz2bNm1ady1xvTZ4y5Yt7gOcbJdZAgQIECBwiAQEsABOVm3UBXB98/Vvb6u3Pat3bKj//8QTTyzXX399ef/739/Z1Ls91L/97eGHHy5TpkwpV155Zbn00kv73eoH4Oq9getfhFEvpTjttNO6Sx9qDCcPd4FItMwSIECAAIF2AQEsgJPtGZUBnAAM5awAHkpdz02AAAECBP5/AQEsgJOfDwI40QpnBXAIZpwAAQIECDQKCGABnKyOAE60wlkBHIIZJ0CAAAECjQICWAAnqyOAE61wVgCHYMYJECBAgECjgAAWwMnqCOBEK5wVwCGYcQIECBAg0CgggAVwsjoCONEKZwVwCGacAAECBAg0CghgAZysjgBOtMJZARyCGSdAgAABAo0CAlgAJ6sjgBOtcFYAh2DGCRAgQIBAo4AAFsDJ6gjgRCucFcAhmHECBAgQINAoIIAFcLI6AjjRCmcFcAhmnAABAgQINAoIYAGcrI4ATrTCWQEcghknQIAAAQKNAgJYACerI4ATrXBWAIdgxgkQIECAQKOAABbAyeoI4EQrnBXAIZhxAgQIECDQKCCABXCyOgI40QpnBXAIZpwAAQIECDQKCGABnKyOAE60wlkBHIIZJ0CAAAECjQICWAAnqyOAE61wVgCHYMYJECBAgECjgAAWwMnqCOBEK5wVwCGYcQIECBAg0CgggAVwsjoCONEKZwVwCGacAAECBAg0CghgAZysjgBOtMJZARyCGSdAgAABAo0CAlgAJ6sjgBOtcFYAh2DGCRAgQIBAo4AAFsDJ6gjgRCucFcAhmHECBAgQINAoIIAFcLI6AjjRCmcFcAhmnAABAgQINAoIYAGcrI4ATrTCWQEcghknQIAAAQKNAgJYACerI4ATrXBWAIdgxgkQIECAQKOAABbAyeoI4EQrnBXAIZhxAgQIECDQKCCABXCyOgI40QpnBXAIZpwAAQIECDQKCGABnKyOAE60wlkBHIIZJ0CAAAECjQICWAAnqyOAE61wVgCHYMYJECBAgECjgAAWwMnqCOBEK5wVwCGYcQIECBAg0CgggAVwsjoCONEKZwVwCGacAAECBAg0CghgAZysjgBOtMJZARyCGSdAgAABAo0CAlgAJ6sjgBOtcFYAh2DGCRAgQIBAo4AAFsDJ6gjgRCucFcAhmHECBAgQINAoIIAFcLI6AjjRCmcFcAhmnAABAgQINAoIYAGcrI4ATrTCWQEcghknQIAAAQKNAgJYACerI4ATrXBWAIdgxgkQIECAQKOAABbAyeoI4EQrnBXAIZhxAgQIECDQKCCABXCyOgI40QpnBXAIZpwAAQIECDQKCGABnKyOAE60wlkBHIIZJ0CAAAECjQICWAAnqyOAE61wVgCHYMYJECBAgECjgAAWwMnqCOBEK5wVwCGYcQIECBAg0CgggAVwsjoCONEKZwVwCGacAAECBAg0CghgAZysjgBOtMJZARyCGSdAgAABAo0CAlgAJ6sjgBOtcFYAh2DGCRAgQIBAo4AAFsDJ6gjgRCucFcAhmHECBAgQINAoIIAFcLI6AjjRCmcFcAhmnAABAgQINAoIYAGcrI4ATrTCWQEcghknQIAAAQKNAgJYACerI4ATrXBWAIdgxgkQIECAQKOAABbAyeoI4EQrnBXAIZhxAgQIECDQKCCABXCyOgI40QpnBXAIZpwAAQIECDQKCGABnKyOAE60wlkBHIIZJ0CAAAECjQICWAAnqyOAE61wVgCHYMYJECBAgECjgAAWwMnqCOBEK5wVwCGYcQIECBAg0CgggAVwsjoCONEKZwVwCGacAAECBAg0CghgAZysjgBOtMJZARyCGSdAgAABAo0CAlgAJ6sjgBOtcFYAh2DGCRAgQIBAo4AAFsDJ6gjgRCucFcAhmHECBAgQINAoIIAFcLI6AjjRCmcFcAhmnAABAgQINAoIYAGcrI4ATrTCWQEcghknQIAAAQKNAgJYACerI4ATrXBWAIdgxgkQIECAQKOAABbAyeoI4EQrnBXAIZhxAgQIECDQKCCABXCyOgI40QpnBXAIZpwAAQIECDQKCGABnKyOAE60wlkBHIIZJ0CAAAECjQICWAAnqyOAE61wVgCHYMYJECBAgECjgAAWwMnqCOBEK5wVwCGYcQIECBAg0CgggAVwsjoCONEKZwVwCGacAAECBAg0CghgAZysjgBOtMJZARyCGSdAgAABAo0CAlgAJ6sjgBOtcFYAh2DGCRAgQIBAo4AAFsDJ6gjgRCucFcAhmHECBAgQINAoIIAFcLI6AjjRCmcFcAhmnAABAgQINAoIYAGcrM6oDeCvf/3r5cYbbyzbt28v06dPL9dcc0355Cc/2dk88sgj5fLLLy8PPvhgOeqoo8pll11Wrrrqqn63/fv3l2uvvbasWbOm7Nq1q5x66qll1apVZcaMGYltEcARl2ECBAgQINAsIIAFcLI8ozKA77jjjnLhhReWL37xi+UjH/lIueuuu7oAvv/++8vMmTPL8ccfX84555xyxRVXlE2bNpVLLrmkC9yLLrqos6vxW//5tttuK1OnTi1Lly7tQnrr1q1lwoQJPfsK4J6pDBIgQIAAgUEJCGABnCzQqAvgAwcOlLe//e1l3rx55Qtf+EK/xZ/+6Z+WD3zgA90/33LLLWXHjh1l/Pjx3T8vW7asrFu3rjz22GNl7969ZdKkSWXFihVl8eLF3Y+/8MIL5Zhjjilr164t5513Xs++ArhnKoMECBAgQGBQAgJYACcLNOoCuEbsH/zBH5SHHnqovPvd7/4ti7POOqsceeSR5c477+z/sY0bN5YzzjijPPfcc10Yz5kzp2zbtq07W9z3qJdBzJo1qzsz3OtDAPcqZY4AAQIECAxOQAAL4GSDRl0Af+c73yl/9md/Vu67777yN3/zN+Xhhx8ub3vb28pnPvOZ8rGPfayL2DPPPLPcdNNN/U6PPvpoOeGEE8oDDzxQnnrqqTJ37tyye/fuMnHixP6Z+fPnd/9uw4YNPfsK4J6pDBIgQIAAgUEJCGABnCzQqAvgev3vggULuuj97Gc/W2bPnl2+9a1vlc9//vPlX//1X8tf/MVflPPPP79cd911/U59H5Sr0VzPANev37dvXxk3blz/zMKFC8vOnTtLPVv8ao8XX3yx1P/1PZ599tly0kknlaeffroce+yxyXExS4AAAQIECAQCAlgAB+tSRl0Af/Ob3yznnntud6nCxRdf3G9RPwxXH/UMb70MYqAzwPWyiRrD9frhgc4A79mzp6xfv/5VfesH7eoH6F75EMDJSpolQIAAAQK5gAAWwMnWjLoA/vGPf1z++I//uLsEol632/eod3Koly9Mmzatu/XZ7bff3v9jA10D/MQTT3S3T+t71OeqZ5PrB+he7eEMcLJ6ZgkQIECAwMETEMACONmmURfAL730UnnTm95U/vqv//o37u1br/v9nd/5nXLKKaeUW2+9tTz55JPlsMMO66yuvvrqcs8993R3gahneSdPnlxWrlxZFi1a1P24u0AkK2WWAAECBAgcegEBLICTrRt1AVzf/Oc+97nuNmarV6/ursH9xje+0QXxvffeW97xjnd09wE+++yzu/v7bt68ubtUokbxBRdc0NktX768+9p627N6xnjJkiXdtcFbtmxxH+Bku8wSIECAAIFDJCCABXCyaqMygCtAPYN78803l2eeeaa7LVq9NrfeHaI+6t0e6t/+Vu8QMWXKlHLllVeWSy+9tN+tfgCu3hu4/kUY9Yzyaaed1l36UGM4ebgLRKJllgABAgQItAsIYAGcbM+oDeAEYahmBfBQyXpeAgQIECDwmwICWAAnPycEcKIVzgrgEMw4AQIECBBoFBDAAjhZHQGcaIWzAjgEM06AAAECBBoFBLAATlZHACda4awADsGMEyBAgACBRgEBLICT1RHAiVY4K4BDMOMECBAgQKBRQAAL4GR1BHCiFc4K4BDMOAECBAgQaBQQwAI4WR0BnGiFswI4BDNOgAABAgQaBQSwAE5WRwAnWuGsAA7BjBMgQIAAgUYBASyAk9URwIlWOCuAQzDjBAgQIECgUUAAC+BkdQRwohXOCuAQzDgBAgQIEGgUEMACOFkdAZxohbMCOAQzToAAAQIEGgUEsABOVkcAJ1rhrAAOwYwTIECAAIFGAQEsgJPVEcCJVjgrgEMw4wQIECBAoFFAAAvgZHUEcKIVzgrgEMw4AQIECBBoFBDAAjhZHQGcaIWzAjgEM06AAAECBBoFBLAATlZHACda4awADsGMEyBAgACBRgEBLICT1RHAiVY4K4BDMOMECBAgQKBRQAAL4GR1BHCiFc4K4BDMOAECBAgQaBQQwAI4WR0BnGiFswI4BDNOgAABAgQaBQSwAE5WRwAnWuGsAA7BjBMgQIAAgUYBASyAk9URwIlWOCuAQzDjBAgQIECgUUAAC+BkdQRwohXOCuAQzDgBAgQIEGgUEMACOFkdAZxohbMCOAQzToAAAQIEGgUEsABOVkcAJ1rhrAAOwYwTIECAAIFGAQEsgJPVEcCJVjgrgEMw4wQIECBAoFFAAAvgZHUEcKIVzgrgEMw4AQIECBBoFBDAAjhZHQGcaIWzAjgEM06AAAECBBoFBLAATlZHACda4awADsGMEyBAgACBRgEBLICT1RHAiVY4K4BDMOM+6hfJAAAgAElEQVQECBAgQKBRQAAL4GR1BHCiFc4K4BDMOAECBAgQaBQQwAI4WR0BnGiFswI4BDNOgAABAgQaBQSwAE5WRwAnWuGsAA7BjBMgQIAAgUYBASyAk9URwIlWOCuAQzDjBAgQIECgUUAAC+BkdQRwohXOCuAQzDgBAgQIEGgUEMACOFkdAZxohbMCOAQzToAAAQIEGgUEsABOVkcAJ1rhrAAOwYwTIECAAIFGAQEsgJPVEcCJVjgrgEMw4wQIECBAoFFAAAvgZHUEcKIVzgrgEMw4AQIECBBoFBDAAjhZHQGcaIWzAjgEM06AAAECBBoFBLAATlZHACda4awADsGMEyBAgACBRgEBLICT1RHAiVY4K4BDMOMECBAgQKBRQAAL4GR1BHCiFc4K4BDMOAECBAgQaBQQwAI4WR0BnGiFswI4BDNOgAABAgQaBQSwAE5WRwAnWuGsAA7BjBMgQIAAgUYBASyAk9URwIlWOCuAQzDjBAgQIECgUUAAC+BkdQRwohXOCuAQzDgBAgQIEGgUEMACOFkdAZxohbMCOAQzToAAAQIEGgUEsABOVkcAJ1rhrAAOwYwTIECAAIFGAQEsgJPVEcCJVjgrgEMw4wQIECBAoFFAAAvgZHUEcKIVzgrgEMw4AQIECBBoFBDAAjhZHQGcaIWzAjgEM06AAAECBBoFBLAATlZHACda4awADsGMEyBAgACBRgEBLICT1RHAiVY4K4BDMOMECBAgQKBRQAAL4GR1BHCiFc4K4BDMOAECBAgQaBQQwAI4WR0BnGiFswI4BDNOgAABAgQaBQSwAE5WRwAnWuGsAA7BjBMgQIAAgUYBASyAk9URwIlWOCuAQzDjBAgQIECgUUAAC+BkdQRwohXOCuAQzDgBAgQIEGgUEMACOFkdAZxohbMCOAQzToAAAQIEGgUEsABOVkcAJ1rhrAAOwYwTIECAAIFGAQEsgJPVEcCJVjgrgEMw4wQIECBAoFFAAAvgZHUEcKIVzgrgEMw4AQIECBBoFBDAAjhZHQGcaIWzAjgEM06AAAECBBoFBLAATlZHACda4awADsGMEyBAgACBRgEBLICT1RHAiVY4K4BDMOMECBAgQKBRQAAL4GR1BHCiFc4K4BDMOAECBAgQaBQQwAI4WR0BnGiFswI4BDNOgAABAgQaBQSwAE5WRwAnWuGsAA7BjBMgQIAAgUYBASyAk9URwIlWOCuAQzDjBAgQIECgUUAAC+BkdQRwohXOCuAQzDgBAgQIEGgUEMACOFmdUR/Ajz/+eHnPe95Tbr755nLhhRd2No888ki5/PLLy4MPPliOOuqoctlll5Wrrrqq323//v3l2muvLWvWrCm7du0qp556alm1alWZMWNGYlsEcMRlmAABAgQINAsIYAGcLM+oDuBf/epX5ZRTTulC97bbbusC+Gc/+1k5/vjjyznnnFOuuOKKsmnTpnLJJZd0gXvRRRd1djV+6z/Xr5k6dWpZunRp2b59e9m6dWuZMGFCz74CuGcqgwQIECBAYFACAlgAJws0qgN42bJlXeB+//vf7w/gG264odxyyy1lx44dZfz48Z1VnVu3bl157LHHyt69e8ukSZPKihUryuLFi7sff+GFF8oxxxxT1q5dW84777yefQVwz1QGCRAgQIDAoAQEsABOFmjUBvAPf/jD8vGPf7y73OEtb3lLfwCfddZZ5cgjjyx33nlnv9PGjRvLGWecUZ577rkujOfMmVO2bdtWZs6c2T9TL4OYNWtWd2a414cA7lXKHAECBAgQGJyAABbAyQaNygCuZ2xnz55d6tne888/v7zhDW/oD+AasWeeeWa56aab+p0effTRcsIJJ5QHHnigPPXUU2Xu3Lll9+7dZeLEif0z8+fP7/7dhg0bevYVwD1TGSRAgAABAoMSEMACOFmgURnAn/rUp8qBAwfKN77xjc7i5QFcP8hWo/i6667rd6rX906fPr3cd9993RngBQsWlH379pVx48b1zyxcuLDs3Lmz1LPFr/Z48cUXS/1f3+PZZ58tJ510Unn66afLsccemxwXswQIECBAgEAgIIAFcLAuZdQF8O23316uvvrqsmXLlvLGN77xtwL4xBNPLPUyiIHOAD/00EPdh93mzZs34BngPXv2lPXr17+q7zXXXNN9gO6VDwGcrKRZAgQIECCQCwhgAZxszagL4A9+8IPl/vvvL4cffni/wy9+8Yvun+tZ3re+9a3drc9qKPc9BroG+Iknnujm+x71GuB6WUX9AN2rPZwBTlbPLAECBAgQOHgCAlgAJ9s06gL4mWeeKS+99NJvGBx33HHdGd9zzz233HXXXeXWW28tTz75ZDnssMO6uXrG+J577unuAlHP8k6ePLmsXLmyLFq0qPtxd4FIVsosAQIECBA49AICWAAnWzfqAnigN//ya4Cff/757j7AZ599dnd/382bN5eLL764i+ILLrig+/Lly5eX1atXd7c9mzZtWlmyZEl3bXC9rMJ9gJP1MkuAAAECBA6NgAAWwMmmjbkArjj1bg/1b397+OGHy5QpU8qVV15ZLr300n63+gG4em/g+hdh1LPJp512WnfpQ43h5OEuEImWWQIECBAg0C4ggAVwsj1jIoATkIM5K4APpqbnIkCAAAECry4ggAVw8vNDACda4awADsGMEyBAgACBRgEBLICT1RHAiVY4K4BDMOMECBAgQKBRQAAL4GR1BHCiFc4K4BDMOAECBAgQaBQQwAI4WR0BnGiFswI4BDNOgAABAgQaBQSwAE5WRwAnWuGsAA7BjBMgQIAAgUYBASyAk9URwIlWOCuAQzDjBAgQIECgUUAAC+BkdQRwohXOCuAQzDgBAgQIEGgUEMACOFkdAZxohbMCOAQzToAAAQIEGgUEsABOVkcAJ1rhrAAOwYwTIECAAIFGAQEsgJPVEcCJVjgrgEMw4wQIECBAoFFAAAvgZHUEcKIVzgrgEMw4AQIECBBoFBDAAjhZHQGcaIWzAjgEM06AAAECBBoFBLAATlZHACda4awADsGMEyBAgACBRgEBLICT1RHAiVY4K4BDMOMECBAgQKBRQAAL4GR1BHCiFc4K4BDMOAECBAgQaBQQwAI4WR0BnGiFswI4BDNOgAABAgQaBQSwAE5WRwAnWuGsAA7BjBMgQIAAgUYBASyAk9URwIlWOCuAQzDjBAgQIECgUUAAC+BkdQRwohXOCuAQzDgBAgQIEGgUEMACOFkdAZxohbMCOAQzToAAAQIEGgUEsABOVkcAJ1rhrAAOwYwTIECAAIFGAQEsgJPVEcCJVjgrgEMw4wQIECBAoFFAAAvgZHUEcKIVzgrgEMw4AQIECBBoFBDAAjhZHQGcaIWzAjgEM06AAAECBBoFBLAATlZHACda4awADsGMEyBAgACBRgEBLICT1RHAiVY4K4BDMOMECBAgQKBRQAAL4GR1BHCiFc4K4BDMOAECBAgQaBQQwAI4WR0BnGiFswI4BDNOgAABAgQaBQSwAE5WRwAnWuGsAA7BjBMgQIAAgUYBASyAk9URwIlWOCuAQzDjBAgQIECgUUAAC+BkdQRwohXOCuAQzDgBAgQIEGgUEMACOFkdAZxohbMCOAQzToAAAQIEGgUEsABOVkcAJ1rhrAAOwYwTIECAAIFGAQEsgJPVEcCJVjgrgEMw4wQIECBAoFFAAAvgZHUEcKIVzgrgEMw4AQIECBBoFBDAAjhZHQGcaIWzAjgEM06AAAECBBoFBLAATlZHACda4awADsGMEyBAgACBRgEBLICT1RHAiVY4K4BDMOMECBAgQKBRQAAL4GR1BHCiFc4K4BDMOAECBAgQaBQQwAI4WR0BnGiFswI4BDNOgAABAgQaBQSwAE5WRwAnWuGsAA7BjBMgQIAAgUYBASyAk9URwIlWOCuAQzDjBAgQIECgUUAAC+BkdQRwohXOCuAQzDgBAgQIEGgUEMACOFkdAZxohbMCOAQzToAAAQIEGgUEsABOVkcAJ1rhrAAOwYwTIECAAIFGAQEsgJPVEcCJVjgrgEMw4wQIECBAoFFAAAvgZHUEcKIVzgrgEMw4AQIECBBoFBDAAjhZHQGcaIWzAjgEM06AAAECBBoFBLAATlZHACda4awADsGMEyBAgACBRgEBLICT1RHAiVY4K4BDMOMECBAgQKBRQAAL4GR1BHCiFc4K4BDMOAECBAgQaBQQwAI4WR0BnGiFswI4BDNOgAABAgQaBQSwAE5WRwAnWuGsAA7BjBMgQIAAgUYBASyAk9URwIlWOCuAQzDjBAgQIECgUUAAC+BkdQRwohXOCuAQzDgBAgQIEGgUEMACOFkdAZxohbMCOAQzToAAAQIEGgUEsABOVkcAJ1rhrAAOwYwTIECAAIFGAQEsgJPVEcCJVjgrgEMw4wQIECBAoFFAAAvgZHUEcKIVzgrgEMw4AQIECBBoFBDAAjhZHQGcaIWzAjgEM06AAAECBBoFBLAATlZHACda4awADsGMEyBAgACBRgEBLICT1RHAiVY4K4BDMOMECBAgQKBRQAAL4GR1BHCiFc4K4BDMOAECBAgQaBQQwAI4WR0BnGiFswI4BDNOgAABAgQaBQSwAE5WRwAnWuGsAA7BjBMgQIAAgUYBASyAk9URwIlWOCuAQzDjBAgQIECgUUAAC+BkdQRwohXOCuAQzDgBAgQIEGgUEMACOFkdAZxohbMCOAQzToAAAQIEGgUEsABOVkcAJ1rhrAAOwYwTIECAAIFGAQEsgJPVGZUB/POf/7wsW7asbNiwobz44otl1qxZ5cYbbyynnnpqZ/PII4+Uyy+/vDz44IPlqKOOKpdddlm56qqr+t32799frr322rJmzZqya9eu7utWrVpVZsyYkdgWARxxGSZAgAABAs0CAlgAJ8szKgP4Qx/6UHn++efLzTffXI4++uhyyy23dDH70EMPdf98/PHHl3POOadcccUVZdOmTeWSSy7pAveiiy7q7Gr81n++7bbbytSpU8vSpUvL9u3by9atW8uECRN69hXAPVMZJECAAAECgxIQwAI4WaBRF8BPPPFEOe6448qPfvSjcsopp3QWBw4cKDNnziyf+tSnysSJE7sg3rFjRxk/fnz34/Vs8bp168pjjz1W9u7dWyZNmlRWrFhRFi9e3P34Cy+8UI455piydu3act555/XsK4B7pjJIgAABAgQGJSCABXCyQKMugGus/vjHPy6nn356Ofzww/st6uULH/7wh7szuUceeWS58847+39s48aN5YwzzijPPfdcF8Zz5swp27Zt66K571Evg6iXUtQzw70+BHCvUuYIECBAgMDgBASwAE42aNQF8EBv/u677y7z588v3/nOd8ry5cvLmWeeWW666ab+0UcffbSccMIJ5YEHHihPPfVUmTt3btm9e3d3trjvUb++/rt6XXGvDwHcq5Q5AgQIECAwOAEBLICTDRr1AVwvhahnfusZ4W9/+9vdB9nOP//8ct111/U71bPC06dPL/fdd193BnjBggVl3759Zdy4cf0zCxcuLDt37iz1bPGrPeoH7ur/+h7PPvtsOemkk8rTTz9djj322OS4mCVAgAABAgQCAQEsgIN1KaM6gNevX9/F7sknn9ydua1ndE888cRy1llnDXgGuH5IrsbwvHnzBjwDvGfPnlKf89Ue11xzTfcBulc+BHCykmYJECBAgEAuIIAFcLI1ozaA6x0g6q3OPvGJT5Q77rij/3rgGr/11me33357v9NA1wDXD9PVs8J9j3oN8OzZs7sP0DkDnKyYWQIECBAgMPQCAlgAJ1s2KgP4K1/5Sndrs3p/3y996Uu/cSlDvR/wrbfeWp588sly2GGHdVZXX311ueeee7q7QNSzvJMnTy4rV64sixYt6n7cXSCSlTJLgAABAgQOvYAAFsDJ1o26AH788cfLO9/5zvLRj370t+7YUC+BqIFb7wN89tlnd/f33bx5c7n44ou7KL7gggs6u/pBudWrV3e3PZs2bVpZsmRJd23wli1b3Ac42S6zBAgQIEDgEAkIYAGcrNqoC+Drr7++C9iBHjVwv/a1r3V3e6hnhx9++OEyZcqUcuWVV5ZLL720/0vqB+DqvYHrX4Tx0ksvldNOO6279KHGcPJwF4hEyywBAgQIEGgXEMACONmeURfAyZsf6lkBPNTCnp8AAQIECPx/AgJYACc/FwRwohXOCuAQzDgBAgQIEGgUEMACOFkdAZxohbMCOAQzToAAAQIEGgUEsABOVkcAJ1rhrAAOwYwTIECAAIFGAQEsgJPVEcCJVjgrgEMw4wQIECBAoFFAAAvgZHUEcKIVzgrgEMw4AQIECBBoFBDAAjhZHQGcaIWzAjgEM06AAAECBBoFBLAATlZHACda4awADsGMEyBAgACBRgEBLICT1RHAiVY4K4BDMOMECBAgQKBRQAAL4GR1BHCiFc4K4BDMOAECBAgQaBQQwAI4WR0BnGiFswI4BDNOgAABAgQaBQSwAE5WRwAnWuGsAA7BjBMgQIAAgUYBASyAk9URwIlWOCuAQzDjBAgQIECgUUAAC+BkdQRwohXOCuAQzDgBAgQIEGgUEMACOFkdAZxohbMCOAQzToAAAQIEGgUEsABOVkcAJ1rhrAAOwYwTIECAAIFGAQEsgJPVEcCJVjgrgEMw4wQIECBAoFFAAAvgZHUEcKIVzgrgEMw4AQIECBBoFBDAAjhZHQGcaIWzAjgEM06AAAECBBoFBLAATlZHACda4awADsGMEyBAgACBRgEBLICT1RHAiVY4K4BDMOMECBAgQKBRQAAL4GR1BHCiFc4K4BDMOAECBAgQaBQQwAI4WR0BnGiFswI4BDNOgAABAgQaBQSwAE5WRwAnWuGsAA7BjBMgQIAAgUYBASyAk9URwIlWOCuAQzDjBAgQIECgUUAAC+BkdQRwohXOCuAQzDgBAgQIEGgUEMACOFkdAZxohbMCOAQzToAAAQIEGgUEsABOVkcAJ1rhrAAOwYwTIECAAIFGAQEsgJPVEcCJVjgrgEMw4wQIECBAoFFAAAvgZHUEcKIVzgrgEMw4AQIECBBoFBDAAjhZHQGcaIWzAjgEM06AAAECBBoFBLAATlZHACda4awADsGMEyBAgACBRgEBLICT1RHAiVY4K4BDMOMECBAgQKBRQAAL4GR1BHCiFc4K4BDMOAECBAgQaBQQwAI4WR0BnGiFswI4BDNOgAABAgQaBQSwAE5WRwAnWuGsAA7BjBMgQIAAgUYBASyAk9URwIlWOCuAQzDjBAgQIECgUUAAC+BkdQRwohXOCuAQzDgBAgQIEGgUEMACOFkdAZxohbMCOAQzToAAAQIEGgUEsABOVkcAJ1rhrAAOwYwTIECAAIFGAQEsgJPVEcCJVjgrgEMw4wQIECBAoFFAAAvgZHUEcKIVzgrgEMw4AQIECBBoFBDAAjhZHQGcaIWzAjgEM06AAAECBBoFBLAATlZHACda4awADsGMEyBAgACBRgEBLICT1RHAiVY4K4BDMOMECBAgQKBRQAAL4GR1BHCiFc4K4BDMOAECBAgQaBQQwAI4WR0BnGiFswI4BDNOgAABAgQaBQSwAE5WRwAnWuGsAA7BjBMgQIAAgUYBASyAk9URwIlWOCuAQzDjBAgQIECgUUAAC+BkdQRwohXOCuAQzDgBAgQIEGgUEMACOFkdAZxohbMCOAQzToAAAQIEGgUEsABOVkcAJ1rhrAAOwYwTIECAAIFGAQEsgJPVEcCJVjgrgEMw4wQIECBAoFFAAAvgZHUEcKIVzgrgEMw4AQIECBBoFBDAAjhZHQGcaIWzAjgEM06AAAECBBoFBLAATlZHACda4awADsGMEyBAgACBRgEBLICT1RHAiVY4K4BDMOMECBAgQKBRQAAL4GR1BHCiFc4K4BDMOAECBAgQaBQQwAI4WR0BnGiFswI4BDNOgAABAgQaBQSwAE5WRwAnWuGsAA7BjBMgQIAAgUYBASyAk9URwIlWOCuAQzDjBAgQIECgUUAAC+BkdQRwohXOCuAQzDgBAgQIEGgUEMACOFkdAZxohbMCOAQzToAAAQIEGgUEsABOVkcAJ1rhrAAOwYwTIECAAIFGAQEsgJPVEcCJVjgrgEMw4wQIECBAoFFAAAvgZHUEcKIVzgrgEMw4AQIECBBoFBDAAjhZHQGcaIWzAjgEM06AAAECBBoFBLAATlZHACda4awADsGMEyBAgACBRgEBLICT1RHAiVY4K4BDMOMECBAgQKBRQAAL4GR1BHCiFc4K4BDMOAECBAgQaBQQwAI4WR0B/Cpa+/fvL9dee21Zs2ZN2bVrVzn11FPLqlWryowZM3r2FcA9UxkkQIAAAQKDEhDAAjhZIAH8Klo1fmvw3nbbbWXq1Kll6dKlZfv27WXr1q1lwoQJPRkL4J6YDBEgQIAAgUELCGABnCyRAB5Aa+/evWXSpEllxYoVZfHixd3ECy+8UI455piydu3act555/VkLIB7YjJEgAABAgQGLSCABXCyRAJ4AK3NmzeXOXPmlG3btpWZM2f2T9TLIGbNmtWdGe7lIYB7UTJDgAABAgQGLyCABXCyRQJ4AK1169aVuXPnlt27d5eJEyf2T8yfP7/7dxs2bOjJWAD3xGSIAAECBAgMWkAAC+BkiQTwAFp33HFHWbBgQdm3b18ZN25c/8TChQvLzp07y8aNGwc0fvHFF0v9X9/j6aefLqecckqpZ5SnTJmSHBezBAgQIECAQCBw8vX39jy9adnpPc+O5MFnn322nHTSSeUnP/lJmTZt2kh+Kwf9tQvgAUi/9a1vlXnz5g14BnjPnj1l/fr1Ax6Ia665prtzhAcBAgQIECBAYLgI1BNx733ve4fLyxkWr0MAD3AY+q4BfuKJJ8r06dP7J+o1wLNnzy633HLLgAfvlWeAf/nLX5Z6Fvhtb3tbGT9+/JAc8L7f3TnLPCS8h+RJHcNDwjxk38TxGzLaQ/bEjuEhox6yb+QYDkz761//uvz0pz8tJ554YjniiCOGzH8kPrEAHuCo1bO8kydPLitXriyLFi3qJlruAnEoFsJ1xodCeWi/h2M4tL5D/eyO31ALD/3zO4ZDbzzU38ExHGrh0ff8AvhVjuny5cvL6tWru9ue1etmlixZUnbs2FG2bNnS832AD8W6+El/KJSH9ns4hkPrO9TP7vgNtfDQP79jOPTGQ/0dHMOhFh59zy+AX+WY1g/ALVu2rPuLMF566aVy2mmndZc+DLeLyP2kH/k/KR3DkX0MHb+Rffzqq3cMHcORL+AdpAICOBUbZvP1uuN6qcYVV1xRfu/3fm+YvTovpxcBx7AXpeE74/gN32PT6ytzDHuVGr5zjuHwPTbD9ZUJ4OF6ZLwuAgQIECBAgACBIREQwEPC6kkJECBAgAABAgSGq4AAHq5HxusiQIAAAQIECBAYEgEBPCSsnpQAAQIECBAgQGC4Cgjg4XpkvC4CBAgQIECAAIEhERDAQ8I69E+6f//+7q9dXrNmTdm1a1epf0vdqlWryowZM4b+m/sOkcDnPve5snHjxvKDH/yg/+seeeSRcvnll5cHH3ywHHXUUeWyyy4rV111Vf+PO74R8ZAM//znP+9uhbhhw4ZSP2E+a9ascuONN3Y/1+rDMRwS9oP6pM8//3y58sory7/8y790t7N8//vfX77whS+Ud7zjHY7hQZUe+id7/PHHy3ve855y8803lwsvvNDxG3ryUf8dBPAIPcQ1fmvw1vsUT506tSxdurRs3769bN26dVj9RR0jlPegvewvf/nL3S3q6n2k+wL4Zz/7WTn++OPLOeec0/3Ypk2byiWXXNIdz4suuqj73o7vQTsEzU/0oQ99qNSAqv/BPfroo7v7gNffcD700EPdPzuGzbSH7AtPPvnkMm7cuPJ3f/d35Xd/93fLX/3VX5Uf//jHpf419zWIHcNDdigG9Y1+9atflVNOOaU7YVD/m1cD2K+jgyL1xaUUATwC12Dv3r1l0qRJZcWKFWXx4sXdOxiuf1XzCOQ9KC/5mWeeKZ/+9KfLfffdV9785jeXN73pTf0BfMMNN3QxVf9mwfHjx3ffr55pXLduXXnssceK43tQDsGgnqQG0nHHHVd+9KMfdf/hrY8DBw6UmTNnlk996lNl4sSJjuGghIf+i2sgXXrppeUzn/lMOeGEE7pv+O///u/lXe96V9m8eXP3pzJ+Hg79cTgY36H++lhPFHz/+9/vD2C/jh4M2bH9HAJ4BB7/+ov3nDlzyrZt27r/IPc96h/N1j+mrWcSPV5fge9+97vlrrvuKvXyh+uuu66L3b4zwGeddVY58sgjy5133tn/Iut/jM8444zy3HPPdbOO7+t7/OpvKOuZwtNPP70cfvjh/S+mXmL04Q9/uPvTFsfw9T1G6XevP7fqZUb152H9jeYnP/lJxzBFfB3mf/jDH5aPf/zj3SVHb3nLW/oD2K+jr8PBGGXfUgCPwANazxTOnTu37N69uzsT1feYP39+9+/qNYsew0eg/nHdywO4/iblzDPPLDfddFP/i3z00Ue7s1QPPPBAeeqppxzf4XP4+l/J3XffXerPse985ztl+fLljuEwPEav9pL+/M//vHz1q1/tfjNTj1+9vMXPw+F/AOtvRGfPnl3q2d7zzz+/vOENb+gPYMdv+B+/4f4KBfBwP0IDvL477rijLFiwoOzbt6+7vq3vsXDhwrJz587uj/Y8ho/AKwO4nkWsv5jXM8N9j3pGcfr06d0lEzWWHd/hc/zqK6mXQtQzv/WM8Le//e3uw6aO4fA6Rv/bq6m/wazX/H7lK1/p/uTl/vvv734z4xgO72NYLzeqlx594xvf6F7oywPYz8HhfexGwqsTwCPhKL3iNX7rW98q8+bNG/AM8J49e8r69etH4LsavS/5lQF84oknlvrHdwOdAa4fsKox7PgOn32oP59qKNUPVNU/Xal/6uIYDp/jk7ySeneVeoiHrioAAATaSURBVOzqJUb1T1v8PEz0Du3s7bffXq6++uqyZcuW8sY3vvG3AtjPwUN7PEbjdxPAI/Co9l0DXD+oU88a9j3qNcD1j4vqBzs8ho/AKwO4/ke33vqs/gLf9xjoGmDH9/U/hvUOEPV2dZ/4xCdK/ZOXvuuBHcPX/9i81iuod/C49957uzO9hx12WP94vfa33tau/js/D19L8fX78Q9+8IPdmfqXX4P/i1/8ovvn+t+9t771rY7f63d4RsV3FsAj8DDWs7yTJ08uK1euLIsWLeregbtADN8D+coArveSvfXWW8uTTz7Z/x/meqbjnnvu6T6c4/gOj2NZ/7i83p6u3qP5S1/60m9cbuQYDo9j9L+9ivqhqXe/+91dBP/Jn/xJN1pvp1VvfXb22Wd3d2bx83D4Hsd6J5162crLH/XOLPVPzs4999zuQ8aO3/A9fiPhlQngkXCUBniN9UM4q1evLmvXri3Tpk0rS5Ys6a4drX9cNGHChBH6rkbny35lANczU33/Ea73b65n9C+++OLuF/MLLrigQ3B8X99dqDfdf+c731k++tGP/tZdVeolEPU3KY7h63uMXuu712tH64dN66+L9QNw9Y/RP//5z5fvfe973R0FjjjiCMfwtRCH2Y+//Bpgv44Os4MzAl+OAB6BB62+5PoBuHpvxHpT8Pq75PoXLdRLH2oMewwvgVcGcH119frDembx4YcfLlOmTOn+tqp6z9K+h+P7+h7D66+/vvtNyECP+puUr33ta47h63uIevru//M//9NdR1o/uFj/lOx973tf9ydnffcF9vOwJ8ZhM/TyAPbr6LA5LCP2hQjgEXvovHACBAgQIECAAIEWAQHcouZrCBAgQIAAAQIERqyAAB6xh84LJ0CAAAECBAgQaBEQwC1qvoYAAQIECBAgQGDECgjgEXvovHACBAgQIECAAIEWAQHcouZrCBAgQIAAAQIERqyAAB6xh84LJ0CAAAECBAgQaBEQwC1qvoYAAQIECBAgQGDECgjgEXvovHACBAgQIECAAIEWAQHcouZrCBAgQIAAAQIERqyAAB6xh84LJ0CAAAECBAgQaBEQwC1qvoYAAQIECBAgQGDECgjgEXvovHACBAgQIECAAIEWAQHcouZrCBAgQIAAAQIERqyAAB6xh84LJ0CAAAECBAgQaBEQwC1qvoYAAQIECBAgQGDECgjgEXvovHACBAgQIECAAIEWAQHcouZrCBAgQIAAAQIERqyAAB6xh84LJ0CAAAECBAgQaBEQwC1qvoYAAQIECBAgQGDECgjgEXvovHACBAgQIECAAIEWAQHcouZrCBAgQIAAAQIERqyAAB6xh84LJ0CAAAECBAgQaBEQwC1qvoYAAQIECBAgQGDECgjgEXvovHACBAgQIECAAIEWAQHcouZrCBAgQIAAAQIERqyAAB6xh84LJ0CAAAECBAgQaBEQwC1qvoYAAQIECBAgQGDECgjgEXvovHACBAgQIECAAIEWAQHcouZrCBAgQIAAAQIERqyAAB6xh84LJ0CAAAECBAgQaBEQwC1qvoYAAQIECBAgQGDECgjgEXvovHACBAgQIECAAIEWAQHcouZrCBAgQIAAAQIERqyAAB6xh84LJ0CAAAECBAgQaBEQwC1qvoYAAQIECBAgQGDECvw/zPqkdHHAAa4AAAAASUVORK5CYII=\" width=\"639.9999861283738\">"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "plt.figure()\n",
    "plt.hist(list(reg_size_dic.values()), bins=np.arange(0,450,10))\n",
    "plt.show()\n",
    "\n",
    "# majority (>900) of regions have 400 probes"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 4.1.3 check readout usage for each gene"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "3"
      ]
     },
     "execution_count": 16,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "num_readout_per_probe"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "251 -> ['NDB_487', 'NDB_628', 'NDB_629']\n",
      "252 -> ['NDB_350', 'NDB_458', 'NDB_616']\n",
      "253 -> ['NDB_335', 'NDB_409', 'NDB_562']\n",
      "255 -> ['NDB_439', 'NDB_443', 'NDB_532']\n",
      "256 -> ['NDB_401', 'NDB_446', 'NDB_601']\n",
      "257 -> ['NDB_427', 'NDB_505', 'NDB_514']\n",
      "258 -> ['NDB_395', 'NDB_544', 'NDB_568']\n",
      "259 -> ['NDB_391', 'NDB_445', 'NDB_557']\n",
      "260 -> ['NDB_376', 'NDB_433', 'NDB_436']\n",
      "261 -> ['NDB_400', 'NDB_526', 'NDB_547']\n",
      "262 -> ['NDB_448', 'NDB_467', 'NDB_587']\n",
      "263 -> ['NDB_385', 'NDB_491', 'NDB_563']\n",
      "264 -> ['NDB_529', 'NDB_536', 'NDB_589']\n",
      "265 -> ['NDB_446', 'NDB_511', 'NDB_577']\n",
      "266 -> ['NDB_452', 'NDB_470', 'NDB_517']\n",
      "267 -> ['NDB_386', 'NDB_551', 'NDB_581']\n",
      "268 -> ['NDB_392', 'NDB_532', 'NDB_608']\n",
      "269 -> ['NDB_350', 'NDB_397', 'NDB_592']\n",
      "270 -> ['NDB_368', 'NDB_388', 'NDB_625']\n",
      "271 -> ['NDB_371', 'NDB_443', 'NDB_614']\n",
      "272 -> ['NDB_341', 'NDB_464', 'NDB_538']\n",
      "273 -> ['NDB_356', 'NDB_616', 'NDB_622']\n",
      "274 -> ['NDB_428', 'NDB_533', 'NDB_601']\n",
      "275 -> ['NDB_376', 'NDB_445', 'NDB_538']\n",
      "277 -> ['NDB_359', 'NDB_401', 'NDB_461']\n",
      "278 -> ['NDB_491', 'NDB_557', 'NDB_578']\n",
      "279 -> ['NDB_487', 'NDB_532', 'NDB_568']\n",
      "280 -> ['NDB_391', 'NDB_467', 'NDB_544']\n",
      "281 -> ['NDB_481', 'NDB_547', 'NDB_587']\n",
      "282 -> ['NDB_493', 'NDB_517', 'NDB_554']\n",
      "283 -> ['NDB_379', 'NDB_433', 'NDB_458']\n",
      "284 -> ['NDB_335', 'NDB_385', 'NDB_601']\n",
      "285 -> ['NDB_448', 'NDB_563', 'NDB_611']\n",
      "286 -> ['NDB_431', 'NDB_539', 'NDB_577']\n",
      "287 -> ['NDB_365', 'NDB_482', 'NDB_538']\n",
      "288 -> ['NDB_347', 'NDB_386', 'NDB_604']\n",
      "289 -> ['NDB_341', 'NDB_466', 'NDB_472']\n",
      "290 -> ['NDB_505', 'NDB_533', 'NDB_551']\n",
      "291 -> ['NDB_427', 'NDB_443', 'NDB_556']\n",
      "292 -> ['NDB_368', 'NDB_436', 'NDB_562']\n",
      "293 -> ['NDB_589', 'NDB_625', 'NDB_628']\n",
      "295 -> ['NDB_460', 'NDB_476', 'NDB_536']\n",
      "296 -> ['NDB_439', 'NDB_511', 'NDB_616']\n",
      "297 -> ['NDB_356', 'NDB_446', 'NDB_529']\n",
      "298 -> ['NDB_365', 'NDB_487', 'NDB_494']\n",
      "299 -> ['NDB_332', 'NDB_506', 'NDB_614']\n",
      "300 -> ['NDB_398', 'NDB_592', 'NDB_622']\n",
      "301 -> ['NDB_341', 'NDB_571', 'NDB_577']\n",
      "302 -> ['NDB_400', 'NDB_481', 'NDB_604']\n",
      "303 -> ['NDB_376', 'NDB_401', 'NDB_526']\n",
      "304 -> ['NDB_359', 'NDB_403', 'NDB_629']\n",
      "305 -> ['NDB_455', 'NDB_493', 'NDB_619']\n",
      "306 -> ['NDB_397', 'NDB_473', 'NDB_533']\n",
      "307 -> ['NDB_388', 'NDB_428', 'NDB_608']\n",
      "308 -> ['NDB_418', 'NDB_464', 'NDB_508']\n",
      "309 -> ['NDB_350', 'NDB_470', 'NDB_581']\n",
      "310 -> ['NDB_371', 'NDB_452', 'NDB_494']\n",
      "311 -> ['NDB_392', 'NDB_409', 'NDB_421']\n",
      "312 -> ['NDB_461', 'NDB_476', 'NDB_479']\n",
      "313 -> ['NDB_514', 'NDB_571', 'NDB_589']\n",
      "314 -> ['NDB_395', 'NDB_445', 'NDB_616']\n",
      "315 -> ['NDB_400', 'NDB_436', 'NDB_532']\n",
      "316 -> ['NDB_379', 'NDB_398', 'NDB_629']\n",
      "317 -> ['NDB_385', 'NDB_391', 'NDB_458']\n",
      "318 -> ['NDB_347', 'NDB_368', 'NDB_544']\n",
      "319 -> ['NDB_562', 'NDB_568', 'NDB_601']\n",
      "320 -> ['NDB_539', 'NDB_563', 'NDB_587']\n",
      "321 -> ['NDB_529', 'NDB_551', 'NDB_578']\n",
      "322 -> ['NDB_431', 'NDB_556', 'NDB_611']\n",
      "324 -> ['NDB_386', 'NDB_427', 'NDB_547']\n",
      "325 -> ['NDB_506', 'NDB_514', 'NDB_517']\n",
      "326 -> ['NDB_332', 'NDB_335', 'NDB_395']\n",
      "327 -> ['NDB_433', 'NDB_448', 'NDB_505']\n",
      "328 -> ['NDB_421', 'NDB_472', 'NDB_592']\n",
      "329 -> ['NDB_466', 'NDB_479', 'NDB_491']\n",
      "330 -> ['NDB_439', 'NDB_554', 'NDB_557']\n",
      "332 -> ['NDB_467', 'NDB_508', 'NDB_628']\n",
      "333 -> ['NDB_350', 'NDB_356', 'NDB_443']\n",
      "334 -> ['NDB_388', 'NDB_482', 'NDB_536']\n",
      "335 -> ['NDB_436', 'NDB_473', 'NDB_539']\n",
      "336 -> ['NDB_452', 'NDB_467', 'NDB_481']\n",
      "337 -> ['NDB_446', 'NDB_568', 'NDB_619']\n",
      "338 -> ['NDB_401', 'NDB_460', 'NDB_533']\n",
      "339 -> ['NDB_335', 'NDB_448', 'NDB_571']\n",
      "340 -> ['NDB_332', 'NDB_461', 'NDB_592']\n",
      "341 -> ['NDB_365', 'NDB_409', 'NDB_431']\n",
      "342 -> ['NDB_376', 'NDB_391', 'NDB_614']\n",
      "343 -> ['NDB_547', 'NDB_554', 'NDB_556']\n",
      "344 -> ['NDB_511', 'NDB_538', 'NDB_608']\n",
      "345 -> ['NDB_371', 'NDB_418', 'NDB_433']\n",
      "347 -> ['NDB_421', 'NDB_427', 'NDB_601']\n",
      "348 -> ['NDB_473', 'NDB_577', 'NDB_587']\n",
      "349 -> ['NDB_386', 'NDB_395', 'NDB_466']\n",
      "350 -> ['NDB_544', 'NDB_563', 'NDB_592']\n",
      "351 -> ['NDB_341', 'NDB_368', 'NDB_493']\n",
      "352 -> ['NDB_397', 'NDB_436', 'NDB_604']\n",
      "353 -> ['NDB_470', 'NDB_472', 'NDB_536']\n",
      "354 -> ['NDB_350', 'NDB_403', 'NDB_551']\n",
      "355 -> ['NDB_356', 'NDB_359', 'NDB_445']\n",
      "356 -> ['NDB_439', 'NDB_533', 'NDB_629']\n",
      "357 -> ['NDB_385', 'NDB_428', 'NDB_508']\n",
      "358 -> ['NDB_464', 'NDB_511', 'NDB_526']\n",
      "359 -> ['NDB_391', 'NDB_476', 'NDB_556']\n",
      "360 -> ['NDB_452', 'NDB_505', 'NDB_532']\n",
      "361 -> ['NDB_409', 'NDB_427', 'NDB_494']\n",
      "362 -> ['NDB_557', 'NDB_563', 'NDB_616']\n",
      "363 -> ['NDB_379', 'NDB_392', 'NDB_589']\n",
      "364 -> ['NDB_506', 'NDB_529', 'NDB_625']\n",
      "365 -> ['NDB_479', 'NDB_481', 'NDB_611']\n",
      "366 -> ['NDB_446', 'NDB_562', 'NDB_578']\n",
      "367 -> ['NDB_403', 'NDB_482', 'NDB_517']\n",
      "368 -> ['NDB_388', 'NDB_547', 'NDB_628']\n",
      "369 -> ['NDB_379', 'NDB_400', 'NDB_491']\n",
      "370 -> ['NDB_458', 'NDB_487', 'NDB_581']\n",
      "371 -> ['NDB_398', 'NDB_443', 'NDB_467']\n",
      "372 -> ['NDB_347', 'NDB_455', 'NDB_622']\n",
      "373 -> ['NDB_472', 'NDB_526', 'NDB_539']\n",
      "374 -> ['NDB_445', 'NDB_476', 'NDB_514']\n",
      "375 -> ['NDB_464', 'NDB_470', 'NDB_494']\n",
      "376 -> ['NDB_385', 'NDB_544', 'NDB_551']\n",
      "377 -> ['NDB_395', 'NDB_428', 'NDB_587']\n",
      "378 -> ['NDB_359', 'NDB_365', 'NDB_601']\n",
      "379 -> ['NDB_386', 'NDB_479', 'NDB_557']\n",
      "380 -> ['NDB_341', 'NDB_376', 'NDB_505']\n",
      "381 -> ['NDB_448', 'NDB_562', 'NDB_625']\n",
      "382 -> ['NDB_482', 'NDB_493', 'NDB_577']\n",
      "383 -> ['NDB_397', 'NDB_401', 'NDB_538']\n",
      "384 -> ['NDB_460', 'NDB_466', 'NDB_629']\n",
      "385 -> ['NDB_335', 'NDB_418', 'NDB_431']\n",
      "386 -> ['NDB_529', 'NDB_568', 'NDB_608']\n",
      "387 -> ['NDB_455', 'NDB_458', 'NDB_614']\n",
      "388 -> ['NDB_421', 'NDB_554', 'NDB_571']\n",
      "389 -> ['NDB_371', 'NDB_392', 'NDB_514']\n",
      "390 -> ['NDB_332', 'NDB_368', 'NDB_433']\n",
      "391 -> ['NDB_461', 'NDB_604', 'NDB_619']\n",
      "392 -> ['NDB_368', 'NDB_401', 'NDB_494']\n",
      "393 -> ['NDB_359', 'NDB_391', 'NDB_511']\n",
      "394 -> ['NDB_376', 'NDB_506', 'NDB_622']\n",
      "395 -> ['NDB_466', 'NDB_514', 'NDB_619']\n",
      "396 -> ['NDB_335', 'NDB_587', 'NDB_592']\n",
      "397 -> ['NDB_347', 'NDB_365', 'NDB_589']\n",
      "398 -> ['NDB_445', 'NDB_568', 'NDB_614']\n",
      "399 -> ['NDB_388', 'NDB_467', 'NDB_578']\n",
      "400 -> ['NDB_395', 'NDB_398', 'NDB_470']\n",
      "401 -> ['NDB_386', 'NDB_397', 'NDB_460']\n",
      "402 -> ['NDB_401', 'NDB_421', 'NDB_611']\n",
      "403 -> ['NDB_379', 'NDB_418', 'NDB_616']\n",
      "404 -> ['NDB_341', 'NDB_479', 'NDB_563']\n",
      "405 -> ['NDB_356', 'NDB_455', 'NDB_551']\n",
      "406 -> ['NDB_452', 'NDB_571', 'NDB_619']\n",
      "407 -> ['NDB_493', 'NDB_505', 'NDB_608']\n",
      "408 -> ['NDB_443', 'NDB_446', 'NDB_547']\n",
      "409 -> ['NDB_392', 'NDB_472', 'NDB_554']\n",
      "410 -> ['NDB_436', 'NDB_458', 'NDB_461']\n",
      "411 -> ['NDB_385', 'NDB_473', 'NDB_557']\n",
      "412 -> ['NDB_371', 'NDB_511', 'NDB_587']\n",
      "413 -> ['NDB_332', 'NDB_365', 'NDB_517']\n",
      "414 -> ['NDB_359', 'NDB_568', 'NDB_625']\n",
      "415 -> ['NDB_433', 'NDB_539', 'NDB_581']\n",
      "416 -> ['NDB_350', 'NDB_448', 'NDB_577']\n",
      "417 -> ['NDB_431', 'NDB_487', 'NDB_563']\n",
      "418 -> ['NDB_418', 'NDB_538', 'NDB_629']\n",
      "419 -> ['NDB_368', 'NDB_529', 'NDB_604']\n",
      "420 -> ['NDB_482', 'NDB_571', 'NDB_601']\n",
      "421 -> ['NDB_508', 'NDB_526', 'NDB_533']\n",
      "422 -> ['NDB_403', 'NDB_409', 'NDB_439']\n",
      "423 -> ['NDB_532', 'NDB_562', 'NDB_628']\n",
      "424 -> ['NDB_464', 'NDB_544', 'NDB_556']\n",
      "425 -> ['NDB_428', 'NDB_448', 'NDB_491']\n",
      "426 -> ['NDB_427', 'NDB_436', 'NDB_481']\n",
      "427 -> ['NDB_392', 'NDB_400', 'NDB_452']\n",
      "428 -> ['NDB_476', 'NDB_551', 'NDB_625']\n",
      "429 -> ['NDB_386', 'NDB_391', 'NDB_536']\n",
      "430 -> ['NDB_371', 'NDB_517', 'NDB_547']\n",
      "431 -> ['NDB_332', 'NDB_493', 'NDB_494']\n",
      "432 -> ['NDB_467', 'NDB_506', 'NDB_539']\n",
      "433 -> ['NDB_388', 'NDB_395', 'NDB_592']\n",
      "434 -> ['NDB_397', 'NDB_445', 'NDB_470']\n",
      "435 -> ['NDB_473', 'NDB_514', 'NDB_622']\n",
      "436 -> ['NDB_376', 'NDB_421', 'NDB_589']\n",
      "437 -> ['NDB_439', 'NDB_455', 'NDB_529']\n",
      "438 -> ['NDB_436', 'NDB_464', 'NDB_482']\n",
      "439 -> ['NDB_347', 'NDB_470', 'NDB_587']\n",
      "440 -> ['NDB_443', 'NDB_472', 'NDB_589']\n",
      "441 -> ['NDB_421', 'NDB_511', 'NDB_517']\n",
      "442 -> ['NDB_581', 'NDB_616', 'NDB_628']\n",
      "444 -> ['NDB_385', 'NDB_427', 'NDB_571']\n",
      "445 -> ['NDB_388', 'NDB_533', 'NDB_554']\n",
      "446 -> ['NDB_392', 'NDB_446', 'NDB_526']\n",
      "447 -> ['NDB_371', 'NDB_532', 'NDB_625']\n",
      "448 -> ['NDB_395', 'NDB_491', 'NDB_611']\n",
      "449 -> ['NDB_359', 'NDB_505', 'NDB_622']\n",
      "450 -> ['NDB_428', 'NDB_458', 'NDB_473']\n",
      "451 -> ['NDB_397', 'NDB_487', 'NDB_608']\n",
      "452 -> ['NDB_379', 'NDB_592', 'NDB_604']\n",
      "453 -> ['NDB_493', 'NDB_508', 'NDB_511']\n",
      "454 -> ['NDB_341', 'NDB_494', 'NDB_536']\n",
      "455 -> ['NDB_403', 'NDB_476', 'NDB_577']\n",
      "456 -> ['NDB_398', 'NDB_418', 'NDB_578']\n",
      "457 -> ['NDB_386', 'NDB_448', 'NDB_629']\n",
      "458 -> ['NDB_439', 'NDB_506', 'NDB_628']\n",
      "459 -> ['NDB_356', 'NDB_433', 'NDB_544']\n",
      "460 -> ['NDB_350', 'NDB_460', 'NDB_481']\n",
      "461 -> ['NDB_395', 'NDB_461', 'NDB_482']\n",
      "462 -> ['NDB_431', 'NDB_455', 'NDB_479']\n",
      "463 -> ['NDB_332', 'NDB_409', 'NDB_467']\n",
      "464 -> ['NDB_538', 'NDB_556', 'NDB_562']\n",
      "465 -> ['NDB_368', 'NDB_554', 'NDB_577']\n",
      "466 -> ['NDB_335', 'NDB_547', 'NDB_557']\n",
      "467 -> ['NDB_376', 'NDB_493', 'NDB_514']\n",
      "468 -> ['NDB_365', 'NDB_391', 'NDB_619']\n",
      "469 -> ['NDB_401', 'NDB_445', 'NDB_539']\n",
      "470 -> ['NDB_466', 'NDB_563', 'NDB_568']\n",
      "471 -> ['NDB_341', 'NDB_452', 'NDB_601']\n",
      "472 -> ['NDB_398', 'NDB_529', 'NDB_614']\n",
      "473 -> ['NDB_400', 'NDB_487', 'NDB_551']\n",
      "474 -> ['NDB_368', 'NDB_461', 'NDB_622']\n",
      "475 -> ['NDB_458', 'NDB_506', 'NDB_508']\n",
      "476 -> ['NDB_436', 'NDB_517', 'NDB_625']\n",
      "477 -> ['NDB_371', 'NDB_385', 'NDB_403']\n",
      "478 -> ['NDB_443', 'NDB_533', 'NDB_611']\n",
      "479 -> ['NDB_470', 'NDB_491', 'NDB_589']\n",
      "480 -> ['NDB_388', 'NDB_446', 'NDB_616']\n",
      "481 -> ['NDB_547', 'NDB_578', 'NDB_581']\n",
      "482 -> ['NDB_427', 'NDB_532', 'NDB_587']\n",
      "483 -> ['NDB_347', 'NDB_379', 'NDB_571']\n",
      "484 -> ['NDB_421', 'NDB_551', 'NDB_629']\n",
      "485 -> ['NDB_400', 'NDB_433', 'NDB_464']\n",
      "486 -> ['NDB_392', 'NDB_473', 'NDB_494']\n",
      "487 -> ['NDB_505', 'NDB_526', 'NDB_562']\n",
      "488 -> ['NDB_350', 'NDB_452', 'NDB_604']\n",
      "489 -> ['NDB_397', 'NDB_467', 'NDB_472']\n",
      "491 -> ['NDB_386', 'NDB_428', 'NDB_452']\n",
      "492 -> ['NDB_505', 'NDB_604', 'NDB_614']\n",
      "493 -> ['NDB_332', 'NDB_448', 'NDB_619']\n",
      "494 -> ['NDB_431', 'NDB_433', 'NDB_536']\n",
      "495 -> ['NDB_455', 'NDB_538', 'NDB_544']\n",
      "496 -> ['NDB_356', 'NDB_409', 'NDB_563']\n",
      "497 -> ['NDB_529', 'NDB_562', 'NDB_592']\n",
      "498 -> ['NDB_401', 'NDB_481', 'NDB_557']\n",
      "499 -> ['NDB_350', 'NDB_379', 'NDB_487']\n",
      "500 -> ['NDB_359', 'NDB_443', 'NDB_616']\n",
      "501 -> ['NDB_391', 'NDB_398', 'NDB_506']\n",
      "502 -> ['NDB_517', 'NDB_589', 'NDB_622']\n",
      "503 -> ['NDB_397', 'NDB_532', 'NDB_571']\n",
      "504 -> ['NDB_436', 'NDB_479', 'NDB_581']\n",
      "505 -> ['NDB_376', 'NDB_446', 'NDB_460']\n",
      "506 -> ['NDB_365', 'NDB_467', 'NDB_554']\n",
      "507 -> ['NDB_385', 'NDB_386', 'NDB_494']\n",
      "508 -> ['NDB_533', 'NDB_556', 'NDB_614']\n",
      "509 -> ['NDB_472', 'NDB_491', 'NDB_604']\n",
      "510 -> ['NDB_452', 'NDB_625', 'NDB_629']\n",
      "511 -> ['NDB_487', 'NDB_539', 'NDB_589']\n",
      "512 -> ['NDB_445', 'NDB_493', 'NDB_547']\n",
      "513 -> ['NDB_476', 'NDB_526', 'NDB_568']\n",
      "514 -> ['NDB_350', 'NDB_514', 'NDB_544']\n",
      "515 -> ['NDB_335', 'NDB_376', 'NDB_619']\n",
      "516 -> ['NDB_403', 'NDB_431', 'NDB_622']\n",
      "517 -> ['NDB_347', 'NDB_443', 'NDB_506']\n",
      "518 -> ['NDB_388', 'NDB_466', 'NDB_587']\n",
      "519 -> ['NDB_371', 'NDB_473', 'NDB_482']\n",
      "520 -> ['NDB_551', 'NDB_601', 'NDB_611']\n",
      "521 -> ['NDB_511', 'NDB_578', 'NDB_628']\n",
      "523 -> ['NDB_368', 'NDB_400', 'NDB_505']\n",
      "524 -> ['NDB_392', 'NDB_395', 'NDB_418']\n",
      "525 -> ['NDB_341', 'NDB_421', 'NDB_539']\n",
      "526 -> ['NDB_335', 'NDB_461', 'NDB_464']\n",
      "527 -> ['NDB_427', 'NDB_439', 'NDB_577']\n",
      "528 -> ['NDB_458', 'NDB_470', 'NDB_608']\n",
      "529 -> ['NDB_403', 'NDB_508', 'NDB_578']\n",
      "530 -> ['NDB_347', 'NDB_532', 'NDB_581']\n",
      "531 -> ['NDB_391', 'NDB_428', 'NDB_514']\n",
      "532 -> ['NDB_398', 'NDB_556', 'NDB_616']\n",
      "533 -> ['NDB_445', 'NDB_517', 'NDB_611']\n",
      "534 -> ['NDB_472', 'NDB_557', 'NDB_601']\n",
      "535 -> ['NDB_356', 'NDB_466', 'NDB_536']\n",
      "536 -> ['NDB_482', 'NDB_562', 'NDB_587']\n",
      "537 -> ['NDB_494', 'NDB_563', 'NDB_629']\n",
      "538 -> ['NDB_409', 'NDB_433', 'NDB_473']\n",
      "539 -> ['NDB_359', 'NDB_476', 'NDB_529']\n",
      "540 -> ['NDB_439', 'NDB_460', 'NDB_592']\n",
      "541 -> ['NDB_332', 'NDB_479', 'NDB_608']\n",
      "542 -> ['NDB_379', 'NDB_397', 'NDB_547']\n",
      "543 -> ['NDB_467', 'NDB_538', 'NDB_568']\n",
      "544 -> ['NDB_365', 'NDB_455', 'NDB_481']\n",
      "545 -> ['NDB_401', 'NDB_418', 'NDB_448']\n",
      "546 -> ['NDB_433', 'NDB_494', 'NDB_577']\n",
      "547 -> ['NDB_532', 'NDB_578', 'NDB_619']\n",
      "548 -> ['NDB_427', 'NDB_460', 'NDB_506']\n",
      "549 -> ['NDB_428', 'NDB_557', 'NDB_568']\n",
      "550 -> ['NDB_533', 'NDB_589', 'NDB_592']\n",
      "551 -> ['NDB_386', 'NDB_403', 'NDB_601']\n",
      "552 -> ['NDB_368', 'NDB_397', 'NDB_539']\n",
      "553 -> ['NDB_476', 'NDB_508', 'NDB_517']\n",
      "554 -> ['NDB_335', 'NDB_356', 'NDB_467']\n",
      "555 -> ['NDB_446', 'NDB_481', 'NDB_491']\n",
      "556 -> ['NDB_452', 'NDB_466', 'NDB_562']\n",
      "557 -> ['NDB_511', 'NDB_563', 'NDB_581']\n",
      "558 -> ['NDB_398', 'NDB_473', 'NDB_551']\n",
      "559 -> ['NDB_479', 'NDB_538', 'NDB_554']\n",
      "560 -> ['NDB_400', 'NDB_611', 'NDB_616']\n",
      "561 -> ['NDB_341', 'NDB_392', 'NDB_614']\n",
      "562 -> ['NDB_365', 'NDB_388', 'NDB_556']\n",
      "563 -> ['NDB_350', 'NDB_371', 'NDB_493']\n",
      "564 -> ['NDB_359', 'NDB_428', 'NDB_464']\n",
      "565 -> ['NDB_418', 'NDB_455', 'NDB_608']\n",
      "566 -> ['NDB_439', 'NDB_458', 'NDB_587']\n",
      "567 -> ['NDB_332', 'NDB_347', 'NDB_625']\n",
      "568 -> ['NDB_443', 'NDB_571', 'NDB_628']\n",
      "569 -> ['NDB_376', 'NDB_536', 'NDB_544']\n",
      "570 -> ['NDB_401', 'NDB_470', 'NDB_482']\n",
      "571 -> ['NDB_379', 'NDB_461', 'NDB_514']\n",
      "572 -> ['NDB_460', 'NDB_577', 'NDB_622']\n",
      "573 -> ['NDB_421', 'NDB_526', 'NDB_604']\n",
      "574 -> ['NDB_391', 'NDB_409', 'NDB_448']\n",
      "575 -> ['NDB_395', 'NDB_487', 'NDB_547']\n",
      "576 -> ['NDB_436', 'NDB_445', 'NDB_551']\n",
      "577 -> ['NDB_433', 'NDB_472', 'NDB_587']\n",
      "579 -> ['NDB_365', 'NDB_385', 'NDB_400']\n",
      "580 -> ['NDB_356', 'NDB_431', 'NDB_614']\n",
      "581 -> ['NDB_418', 'NDB_481', 'NDB_505']\n",
      "582 -> ['NDB_446', 'NDB_464', 'NDB_506']\n",
      "583 -> ['NDB_398', 'NDB_403', 'NDB_536']\n",
      "584 -> ['NDB_443', 'NDB_493', 'NDB_625']\n",
      "585 -> ['NDB_371', 'NDB_466', 'NDB_467']\n",
      "586 -> ['NDB_427', 'NDB_562', 'NDB_563']\n",
      "587 -> ['NDB_392', 'NDB_578', 'NDB_611']\n",
      "588 -> ['NDB_470', 'NDB_508', 'NDB_592']\n",
      "589 -> ['NDB_379', 'NDB_476', 'NDB_532']\n",
      "590 -> ['NDB_397', 'NDB_409', 'NDB_511']\n",
      "591 -> ['NDB_368', 'NDB_533', 'NDB_608']\n",
      "592 -> ['NDB_341', 'NDB_350', 'NDB_589']\n",
      "593 -> ['NDB_347', 'NDB_479', 'NDB_616']\n",
      "594 -> ['NDB_491', 'NDB_539', 'NDB_554']\n",
      "595 -> ['NDB_335', 'NDB_386', 'NDB_455']\n",
      "596 -> ['NDB_494', 'NDB_581', 'NDB_601']\n",
      "597 -> ['NDB_431', 'NDB_439', 'NDB_473']\n",
      "598 -> ['NDB_544', 'NDB_557', 'NDB_622']\n",
      "599 -> ['NDB_359', 'NDB_376', 'NDB_628']\n",
      "600 -> ['NDB_487', 'NDB_526', 'NDB_571']\n",
      "601 -> ['NDB_401', 'NDB_514', 'NDB_604']\n",
      "602 -> ['NDB_385', 'NDB_448', 'NDB_461']\n",
      "603 -> ['NDB_395', 'NDB_421', 'NDB_436']\n",
      "604 -> ['NDB_391', 'NDB_472', 'NDB_568']\n",
      "605 -> ['NDB_388', 'NDB_458', 'NDB_629']\n",
      "606 -> ['NDB_452', 'NDB_529', 'NDB_538']\n",
      "607 -> ['NDB_428', 'NDB_505', 'NDB_547']\n",
      "608 -> ['NDB_332', 'NDB_445', 'NDB_577']\n",
      "609 -> ['NDB_460', 'NDB_482', 'NDB_619']\n",
      "610 -> ['NDB_517', 'NDB_556', 'NDB_578']\n",
      "611 -> ['NDB_431', 'NDB_526', 'NDB_578']\n",
      "612 -> ['NDB_397', 'NDB_446', 'NDB_625']\n",
      "613 -> ['NDB_392', 'NDB_460', 'NDB_470']\n",
      "614 -> ['NDB_388', 'NDB_436', 'NDB_614']\n",
      "615 -> ['NDB_395', 'NDB_452', 'NDB_464']\n",
      "616 -> ['NDB_401', 'NDB_556', 'NDB_581']\n",
      "617 -> ['NDB_443', 'NDB_473', 'NDB_479']\n",
      "618 -> ['NDB_347', 'NDB_391', 'NDB_418']\n",
      "619 -> ['NDB_409', 'NDB_538', 'NDB_587']\n",
      "620 -> ['NDB_365', 'NDB_376', 'NDB_533']\n",
      "621 -> ['NDB_517', 'NDB_529', 'NDB_616']\n",
      "622 -> ['NDB_398', 'NDB_577', 'NDB_604']\n",
      "623 -> ['NDB_379', 'NDB_539', 'NDB_557']\n",
      "624 -> ['NDB_448', 'NDB_472', 'NDB_547']\n",
      "625 -> ['NDB_476', 'NDB_544', 'NDB_571']\n",
      "626 -> ['NDB_461', 'NDB_568', 'NDB_629']\n",
      "627 -> ['NDB_332', 'NDB_466', 'NDB_482']\n",
      "628 -> ['NDB_368', 'NDB_487', 'NDB_611']\n",
      "629 -> ['NDB_335', 'NDB_445', 'NDB_628']\n",
      "630 -> ['NDB_341', 'NDB_455', 'NDB_592']\n",
      "631 -> ['NDB_508', 'NDB_554', 'NDB_608']\n",
      "632 -> ['NDB_356', 'NDB_403', 'NDB_511']\n",
      "633 -> ['NDB_371', 'NDB_400', 'NDB_601']\n",
      "634 -> ['NDB_427', 'NDB_428', 'NDB_536']\n",
      "635 -> ['NDB_350', 'NDB_359', 'NDB_526']\n",
      "636 -> ['NDB_439', 'NDB_491', 'NDB_494']\n",
      "637 -> ['NDB_421', 'NDB_506', 'NDB_578']\n",
      "638 -> ['NDB_431', 'NDB_514', 'NDB_532']\n",
      "639 -> ['NDB_385', 'NDB_433', 'NDB_481']\n",
      "640 -> ['NDB_401', 'NDB_458', 'NDB_467']\n",
      "641 -> ['NDB_386', 'NDB_493', 'NDB_562']\n",
      "642 -> ['NDB_551', 'NDB_563', 'NDB_589']\n",
      "643 -> ['NDB_445', 'NDB_505', 'NDB_592']\n",
      "644 -> ['NDB_533', 'NDB_619', 'NDB_622']\n",
      "645 -> ['NDB_365', 'NDB_587', 'NDB_629']\n",
      "646 -> ['NDB_448', 'NDB_476', 'NDB_557']\n",
      "647 -> ['NDB_547', 'NDB_568', 'NDB_577']\n",
      "648 -> ['NDB_368', 'NDB_395', 'NDB_538']\n",
      "649 -> ['NDB_376', 'NDB_604', 'NDB_616']\n",
      "650 -> ['NDB_379', 'NDB_436', 'NDB_472']\n",
      "651 -> ['NDB_482', 'NDB_487', 'NDB_556']\n",
      "652 -> ['NDB_391', 'NDB_433', 'NDB_461']\n",
      "653 -> ['NDB_385', 'NDB_443', 'NDB_608']\n",
      "654 -> ['NDB_397', 'NDB_479', 'NDB_514']\n",
      "655 -> ['NDB_332', 'NDB_455', 'NDB_611']\n",
      "656 -> ['NDB_347', 'NDB_350', 'NDB_466']\n",
      "657 -> ['NDB_505', 'NDB_517', 'NDB_536']\n",
      "658 -> ['NDB_392', 'NDB_398', 'NDB_539']\n",
      "659 -> ['NDB_418', 'NDB_421', 'NDB_529']\n",
      "660 -> ['NDB_439', 'NDB_508', 'NDB_571']\n",
      "661 -> ['NDB_544', 'NDB_619', 'NDB_625']\n",
      "662 -> ['NDB_388', 'NDB_452', 'NDB_511']\n",
      "663 -> ['NDB_428', 'NDB_470', 'NDB_628']\n",
      "664 -> ['NDB_356', 'NDB_400', 'NDB_473']\n",
      "665 -> ['NDB_460', 'NDB_491', 'NDB_581']\n",
      "666 -> ['NDB_371', 'NDB_464', 'NDB_622']\n",
      "667 -> ['NDB_409', 'NDB_446', 'NDB_614']\n",
      "668 -> ['NDB_436', 'NDB_491', 'NDB_508']\n",
      "669 -> ['NDB_409', 'NDB_452', 'NDB_487']\n",
      "670 -> ['NDB_347', 'NDB_401', 'NDB_476']\n",
      "671 -> ['NDB_379', 'NDB_481', 'NDB_482']\n",
      "672 -> ['NDB_391', 'NDB_455', 'NDB_628']\n",
      "673 -> ['NDB_614', 'NDB_622', 'NDB_625']\n",
      "674 -> ['NDB_470', 'NDB_578', 'NDB_629']\n",
      "675 -> ['NDB_395', 'NDB_400', 'NDB_533']\n",
      "676 -> ['NDB_479', 'NDB_544', 'NDB_577']\n",
      "677 -> ['NDB_350', 'NDB_536', 'NDB_547']\n",
      "678 -> ['NDB_341', 'NDB_439', 'NDB_448']\n",
      "679 -> ['NDB_376', 'NDB_473', 'NDB_511']\n",
      "680 -> ['NDB_443', 'NDB_494', 'NDB_592']\n",
      "681 -> ['NDB_403', 'NDB_461', 'NDB_571']\n",
      "682 -> ['NDB_368', 'NDB_385', 'NDB_466']\n",
      "683 -> ['NDB_427', 'NDB_458', 'NDB_589']\n",
      "684 -> ['NDB_446', 'NDB_467', 'NDB_611']\n",
      "685 -> ['NDB_388', 'NDB_433', 'NDB_551']\n",
      "687 -> ['NDB_397', 'NDB_431', 'NDB_517']\n",
      "688 -> ['NDB_514', 'NDB_556', 'NDB_601']\n",
      "689 -> ['NDB_365', 'NDB_371', 'NDB_557']\n",
      "690 -> ['NDB_470', 'NDB_562', 'NDB_577']\n",
      "691 -> ['NDB_335', 'NDB_472', 'NDB_581']\n",
      "692 -> ['NDB_421', 'NDB_538', 'NDB_619']\n",
      "693 -> ['NDB_332', 'NDB_386', 'NDB_505']\n",
      "694 -> ['NDB_418', 'NDB_460', 'NDB_604']\n",
      "695 -> ['NDB_359', 'NDB_506', 'NDB_587']\n",
      "696 -> ['NDB_356', 'NDB_539', 'NDB_608']\n",
      "697 -> ['NDB_398', 'NDB_445', 'NDB_532']\n",
      "698 -> ['NDB_464', 'NDB_529', 'NDB_563']\n",
      "699 -> ['NDB_455', 'NDB_526', 'NDB_616']\n",
      "700 -> ['NDB_428', 'NDB_554', 'NDB_622']\n",
      "701 -> ['NDB_400', 'NDB_493', 'NDB_568']\n",
      "702 -> ['NDB_347', 'NDB_392', 'NDB_482']\n",
      "703 -> ['NDB_487', 'NDB_491', 'NDB_562']\n",
      "704 -> ['NDB_491', 'NDB_505', 'NDB_628']\n",
      "705 -> ['NDB_428', 'NDB_455', 'NDB_581']\n",
      "706 -> ['NDB_356', 'NDB_436', 'NDB_476']\n",
      "707 -> ['NDB_403', 'NDB_427', 'NDB_557']\n",
      "708 -> ['NDB_409', 'NDB_514', 'NDB_536']\n",
      "709 -> ['NDB_439', 'NDB_470', 'NDB_619']\n",
      "710 -> ['NDB_391', 'NDB_578', 'NDB_625']\n",
      "711 -> ['NDB_388', 'NDB_421', 'NDB_563']\n",
      "712 -> ['NDB_433', 'NDB_568', 'NDB_589']\n",
      "713 -> ['NDB_350', 'NDB_464', 'NDB_608']\n",
      "714 -> ['NDB_385', 'NDB_401', 'NDB_616']\n",
      "715 -> ['NDB_418', 'NDB_458', 'NDB_472']\n",
      "716 -> ['NDB_395', 'NDB_481', 'NDB_628']\n",
      "717 -> ['NDB_379', 'NDB_508', 'NDB_577']\n",
      "718 -> ['NDB_341', 'NDB_482', 'NDB_554']\n",
      "719 -> ['NDB_445', 'NDB_491', 'NDB_526']\n",
      "720 -> ['NDB_335', 'NDB_398', 'NDB_533']\n",
      "721 -> ['NDB_386', 'NDB_517', 'NDB_532']\n",
      "722 -> ['NDB_494', 'NDB_506', 'NDB_551']\n",
      "723 -> ['NDB_368', 'NDB_446', 'NDB_587']\n",
      "724 -> ['NDB_332', 'NDB_403', 'NDB_547']\n",
      "725 -> ['NDB_536', 'NDB_601', 'NDB_625']\n",
      "726 -> ['NDB_376', 'NDB_539', 'NDB_578']\n",
      "727 -> ['NDB_359', 'NDB_455', 'NDB_467']\n",
      "728 -> ['NDB_479', 'NDB_487', 'NDB_614']\n",
      "729 -> ['NDB_400', 'NDB_443', 'NDB_511']\n",
      "730 -> ['NDB_341', 'NDB_556', 'NDB_608']\n",
      "731 -> ['NDB_466', 'NDB_470', 'NDB_532']\n",
      "732 -> ['NDB_371', 'NDB_461', 'NDB_562']\n",
      "733 -> ['NDB_392', 'NDB_436', 'NDB_506']\n",
      "734 -> ['NDB_428', 'NDB_571', 'NDB_604']\n",
      "735 -> ['NDB_446', 'NDB_448', 'NDB_592']\n",
      "736 -> ['NDB_493', 'NDB_581', 'NDB_629']\n",
      "737 -> ['NDB_347', 'NDB_473', 'NDB_611']\n",
      "738 -> ['NDB_365', 'NDB_529', 'NDB_544']\n",
      "739 -> ['NDB_397', 'NDB_452', 'NDB_622']\n",
      "740 -> ['NDB_431', 'NDB_460', 'NDB_538']\n",
      "741 -> ['NDB_466', 'NDB_589', 'NDB_601']\n",
      "742 -> ['NDB_392', 'NDB_556', 'NDB_568']\n",
      "743 -> ['NDB_443', 'NDB_505', 'NDB_619']\n",
      "744 -> ['NDB_388', 'NDB_517', 'NDB_557']\n",
      "745 -> ['NDB_335', 'NDB_458', 'NDB_551']\n",
      "746 -> ['NDB_391', 'NDB_421', 'NDB_481']\n",
      "747 -> ['NDB_439', 'NDB_514', 'NDB_563']\n",
      "748 -> ['NDB_350', 'NDB_476', 'NDB_533']\n",
      "749 -> ['NDB_385', 'NDB_409', 'NDB_554']\n",
      "750 -> ['NDB_379', 'NDB_479', 'NDB_526']\n",
      "1 -> ['NDB_493', 'NDB_533', 'NDB_587']\n",
      "2 -> ['NDB_418', 'NDB_511', 'NDB_592']\n",
      "3 -> ['NDB_335', 'NDB_359', 'NDB_608']\n",
      "4 -> ['NDB_350', 'NDB_557', 'NDB_614']\n",
      "5 -> ['NDB_482', 'NDB_491', 'NDB_622']\n",
      "6 -> ['NDB_332', 'NDB_458', 'NDB_529']\n",
      "7 -> ['NDB_379', 'NDB_517', 'NDB_563']\n",
      "8 -> ['NDB_388', 'NDB_479', 'NDB_571']\n",
      "9 -> ['NDB_392', 'NDB_455', 'NDB_601']\n",
      "10 -> ['NDB_452', 'NDB_556', 'NDB_577']\n",
      "11 -> ['NDB_356', 'NDB_365', 'NDB_611']\n",
      "12 -> ['NDB_409', 'NDB_428', 'NDB_436']\n",
      "13 -> ['NDB_385', 'NDB_587', 'NDB_625']\n",
      "14 -> ['NDB_473', 'NDB_508', 'NDB_536']\n",
      "15 -> ['NDB_481', 'NDB_554', 'NDB_581']\n",
      "16 -> ['NDB_386', 'NDB_470', 'NDB_506']\n",
      "17 -> ['NDB_391', 'NDB_439', 'NDB_604']\n",
      "18 -> ['NDB_494', 'NDB_568', 'NDB_616']\n",
      "19 -> ['NDB_529', 'NDB_539', 'NDB_622']\n",
      "20 -> ['NDB_376', 'NDB_379', 'NDB_467']\n",
      "21 -> ['NDB_341', 'NDB_347', 'NDB_629']\n",
      "22 -> ['NDB_371', 'NDB_388', 'NDB_589']\n",
      "23 -> ['NDB_476', 'NDB_487', 'NDB_619']\n",
      "24 -> ['NDB_395', 'NDB_401', 'NDB_505']\n",
      "25 -> ['NDB_427', 'NDB_461', 'NDB_526']\n",
      "26 -> ['NDB_403', 'NDB_418', 'NDB_554']\n",
      "27 -> ['NDB_448', 'NDB_460', 'NDB_514']\n",
      "28 -> ['NDB_365', 'NDB_392', 'NDB_551']\n",
      "29 -> ['NDB_446', 'NDB_473', 'NDB_608']\n",
      "30 -> ['NDB_359', 'NDB_368', 'NDB_439']\n",
      "31 -> ['NDB_409', 'NDB_458', 'NDB_532']\n",
      "32 -> ['NDB_421', 'NDB_466', 'NDB_547']\n",
      "33 -> ['NDB_397', 'NDB_443', 'NDB_578']\n",
      "34 -> ['NDB_539', 'NDB_592', 'NDB_601']\n",
      "35 -> ['NDB_433', 'NDB_482', 'NDB_625']\n",
      "36 -> ['NDB_388', 'NDB_431', 'NDB_493']\n",
      "37 -> ['NDB_371', 'NDB_472', 'NDB_556']\n",
      "38 -> ['NDB_403', 'NDB_529', 'NDB_533']\n",
      "39 -> ['NDB_332', 'NDB_376', 'NDB_508']\n",
      "40 -> ['NDB_335', 'NDB_368', 'NDB_563']\n",
      "41 -> ['NDB_391', 'NDB_446', 'NDB_517']\n",
      "42 -> ['NDB_481', 'NDB_494', 'NDB_562']\n",
      "43 -> ['NDB_464', 'NDB_614', 'NDB_616']\n",
      "44 -> ['NDB_365', 'NDB_491', 'NDB_532']\n",
      "45 -> ['NDB_341', 'NDB_467', 'NDB_581']\n",
      "46 -> ['NDB_347', 'NDB_445', 'NDB_554']\n",
      "47 -> ['NDB_386', 'NDB_511', 'NDB_589']\n",
      "48 -> ['NDB_436', 'NDB_452', 'NDB_544']\n",
      "49 -> ['NDB_356', 'NDB_460', 'NDB_547']\n",
      "50 -> ['NDB_385', 'NDB_398', 'NDB_538']\n",
      "51 -> ['NDB_400', 'NDB_479', 'NDB_628']\n",
      "52 -> ['NDB_461', 'NDB_466', 'NDB_611']\n",
      "53 -> ['NDB_401', 'NDB_455', 'NDB_577']\n",
      "54 -> ['NDB_514', 'NDB_578', 'NDB_587']\n",
      "55 -> ['NDB_409', 'NDB_526', 'NDB_557']\n",
      "56 -> ['NDB_350', 'NDB_368', 'NDB_473']\n",
      "57 -> ['NDB_395', 'NDB_458', 'NDB_562']\n",
      "58 -> ['NDB_491', 'NDB_506', 'NDB_556']\n",
      "59 -> ['NDB_494', 'NDB_517', 'NDB_619']\n",
      "60 -> ['NDB_446', 'NDB_505', 'NDB_571']\n",
      "61 -> ['NDB_427', 'NDB_464', 'NDB_551']\n",
      "62 -> ['NDB_421', 'NDB_467', 'NDB_536']\n",
      "63 -> ['NDB_443', 'NDB_448', 'NDB_539']\n",
      "64 -> ['NDB_400', 'NDB_439', 'NDB_476']\n",
      "65 -> ['NDB_347', 'NDB_431', 'NDB_472']\n",
      "66 -> ['NDB_433', 'NDB_604', 'NDB_608']\n",
      "67 -> ['NDB_371', 'NDB_544', 'NDB_629']\n",
      "68 -> ['NDB_403', 'NDB_470', 'NDB_568']\n",
      "69 -> ['NDB_335', 'NDB_397', 'NDB_589']\n",
      "70 -> ['NDB_392', 'NDB_445', 'NDB_622']\n",
      "71 -> ['NDB_418', 'NDB_428', 'NDB_614']\n",
      "72 -> ['NDB_359', 'NDB_487', 'NDB_493']\n",
      "73 -> ['NDB_506', 'NDB_532', 'NDB_538']\n",
      "74 -> ['NDB_379', 'NDB_533', 'NDB_628']\n",
      "75 -> ['NDB_350', 'NDB_398', 'NDB_511']\n",
      "76 -> ['NDB_551', 'NDB_592', 'NDB_619']\n",
      "77 -> ['NDB_397', 'NDB_464', 'NDB_481']\n",
      "78 -> ['NDB_335', 'NDB_341', 'NDB_427']\n",
      "79 -> ['NDB_356', 'NDB_391', 'NDB_470']\n",
      "80 -> ['NDB_431', 'NDB_443', 'NDB_508']\n",
      "81 -> ['NDB_347', 'NDB_433', 'NDB_619']\n",
      "82 -> ['NDB_418', 'NDB_493', 'NDB_601']\n",
      "83 -> ['NDB_403', 'NDB_446', 'NDB_581']\n",
      "84 -> ['NDB_473', 'NDB_526', 'NDB_589']\n",
      "85 -> ['NDB_428', 'NDB_445', 'NDB_563']\n",
      "86 -> ['NDB_452', 'NDB_460', 'NDB_472']\n",
      "87 -> ['NDB_395', 'NDB_439', 'NDB_625']\n",
      "88 -> ['NDB_436', 'NDB_536', 'NDB_577']\n",
      "89 -> ['NDB_421', 'NDB_455', 'NDB_514']\n",
      "90 -> ['NDB_494', 'NDB_587', 'NDB_614']\n",
      "91 -> ['NDB_398', 'NDB_464', 'NDB_479']\n",
      "92 -> ['NDB_356', 'NDB_392', 'NDB_562']\n",
      "93 -> ['NDB_458', 'NDB_556', 'NDB_622']\n",
      "94 -> ['NDB_529', 'NDB_532', 'NDB_554']\n",
      "95 -> ['NDB_359', 'NDB_517', 'NDB_533']\n",
      "96 -> ['NDB_397', 'NDB_427', 'NDB_629']\n",
      "97 -> ['NDB_341', 'NDB_443', 'NDB_481']\n",
      "98 -> ['NDB_400', 'NDB_538', 'NDB_563']\n",
      "99 -> ['NDB_467', 'NDB_491', 'NDB_511']\n",
      "100 -> ['NDB_466', 'NDB_616', 'NDB_625']\n",
      "101 -> ['NDB_547', 'NDB_551', 'NDB_608']\n",
      "102 -> ['NDB_350', 'NDB_409', 'NDB_578']\n",
      "103 -> ['NDB_455', 'NDB_470', 'NDB_476']\n",
      "104 -> ['NDB_371', 'NDB_376', 'NDB_487']\n",
      "105 -> ['NDB_368', 'NDB_526', 'NDB_592']\n",
      "106 -> ['NDB_379', 'NDB_448', 'NDB_544']\n",
      "107 -> ['NDB_332', 'NDB_568', 'NDB_628']\n",
      "108 -> ['NDB_388', 'NDB_473', 'NDB_506']\n",
      "109 -> ['NDB_436', 'NDB_571', 'NDB_611']\n",
      "111 -> ['NDB_398', 'NDB_431', 'NDB_562']\n",
      "112 -> ['NDB_461', 'NDB_536', 'NDB_581']\n",
      "113 -> ['NDB_538', 'NDB_601', 'NDB_616']\n",
      "114 -> ['NDB_505', 'NDB_556', 'NDB_587']\n",
      "115 -> ['NDB_443', 'NDB_466', 'NDB_604']\n",
      "116 -> ['NDB_385', 'NDB_533', 'NDB_539']\n",
      "117 -> ['NDB_455', 'NDB_482', 'NDB_557']\n",
      "118 -> ['NDB_395', 'NDB_433', 'NDB_629']\n",
      "119 -> ['NDB_401', 'NDB_491', 'NDB_529']\n",
      "120 -> ['NDB_400', 'NDB_514', 'NDB_577']\n",
      "121 -> ['NDB_479', 'NDB_508', 'NDB_625']\n",
      "122 -> ['NDB_392', 'NDB_458', 'NDB_493']\n",
      "123 -> ['NDB_386', 'NDB_421', 'NDB_476']\n",
      "124 -> ['NDB_335', 'NDB_511', 'NDB_532']\n",
      "125 -> ['NDB_359', 'NDB_418', 'NDB_551']\n",
      "126 -> ['NDB_347', 'NDB_371', 'NDB_446']\n",
      "127 -> ['NDB_409', 'NDB_517', 'NDB_592']\n",
      "129 -> ['NDB_379', 'NDB_494', 'NDB_622']\n",
      "130 -> ['NDB_452', 'NDB_536', 'NDB_611']\n",
      "131 -> ['NDB_368', 'NDB_403', 'NDB_467']\n",
      "132 -> ['NDB_365', 'NDB_608', 'NDB_614']\n",
      "133 -> ['NDB_385', 'NDB_445', 'NDB_589']\n",
      "134 -> ['NDB_461', 'NDB_505', 'NDB_554']\n",
      "135 -> ['NDB_401', 'NDB_506', 'NDB_571']\n",
      "136 -> ['NDB_388', 'NDB_439', 'NDB_539']\n",
      "137 -> ['NDB_376', 'NDB_482', 'NDB_581']\n",
      "138 -> ['NDB_386', 'NDB_526', 'NDB_563']\n",
      "139 -> ['NDB_391', 'NDB_538', 'NDB_547']\n",
      "140 -> ['NDB_464', 'NDB_472', 'NDB_628']\n",
      "141 -> ['NDB_398', 'NDB_448', 'NDB_487']\n",
      "142 -> ['NDB_350', 'NDB_400', 'NDB_619']\n",
      "143 -> ['NDB_428', 'NDB_481', 'NDB_629']\n",
      "144 -> ['NDB_341', 'NDB_356', 'NDB_568']\n",
      "145 -> ['NDB_470', 'NDB_544', 'NDB_616']\n",
      "146 -> ['NDB_427', 'NDB_466', 'NDB_578']\n",
      "147 -> ['NDB_562', 'NDB_604', 'NDB_611']\n",
      "148 -> ['NDB_433', 'NDB_452', 'NDB_557']\n",
      "149 -> ['NDB_460', 'NDB_479', 'NDB_601']\n",
      "150 -> ['NDB_332', 'NDB_397', 'NDB_476']\n",
      "151 -> ['NDB_365', 'NDB_386', 'NDB_508']\n",
      "152 -> ['NDB_350', 'NDB_386', 'NDB_554']\n",
      "153 -> ['NDB_347', 'NDB_395', 'NDB_397']\n",
      "154 -> ['NDB_332', 'NDB_398', 'NDB_557']\n",
      "155 -> ['NDB_376', 'NDB_479', 'NDB_568']\n",
      "156 -> ['NDB_359', 'NDB_392', 'NDB_491']\n",
      "157 -> ['NDB_467', 'NDB_532', 'NDB_622']\n",
      "158 -> ['NDB_341', 'NDB_371', 'NDB_628']\n",
      "159 -> ['NDB_356', 'NDB_506', 'NDB_526']\n",
      "160 -> ['NDB_400', 'NDB_460', 'NDB_508']\n",
      "161 -> ['NDB_403', 'NDB_452', 'NDB_563']\n",
      "162 -> ['NDB_439', 'NDB_461', 'NDB_589']\n",
      "163 -> ['NDB_472', 'NDB_494', 'NDB_625']\n",
      "164 -> ['NDB_448', 'NDB_481', 'NDB_608']\n",
      "165 -> ['NDB_482', 'NDB_511', 'NDB_544']\n",
      "166 -> ['NDB_335', 'NDB_391', 'NDB_487']\n",
      "168 -> ['NDB_395', 'NDB_479', 'NDB_529']\n",
      "169 -> ['NDB_332', 'NDB_571', 'NDB_578']\n",
      "170 -> ['NDB_445', 'NDB_466', 'NDB_556']\n",
      "171 -> ['NDB_401', 'NDB_431', 'NDB_629']\n",
      "172 -> ['NDB_385', 'NDB_421', 'NDB_568']\n",
      "173 -> ['NDB_365', 'NDB_398', 'NDB_505']\n",
      "174 -> ['NDB_539', 'NDB_547', 'NDB_604']\n",
      "175 -> ['NDB_347', 'NDB_436', 'NDB_514']\n",
      "176 -> ['NDB_388', 'NDB_392', 'NDB_461']\n",
      "177 -> ['NDB_409', 'NDB_418', 'NDB_476']\n",
      "178 -> ['NDB_359', 'NDB_379', 'NDB_470']\n",
      "179 -> ['NDB_446', 'NDB_533', 'NDB_536']\n",
      "180 -> ['NDB_517', 'NDB_551', 'NDB_562']\n",
      "181 -> ['NDB_368', 'NDB_601', 'NDB_614']\n",
      "182 -> ['NDB_428', 'NDB_493', 'NDB_616']\n",
      "183 -> ['NDB_427', 'NDB_581', 'NDB_592']\n",
      "184 -> ['NDB_464', 'NDB_467', 'NDB_577']\n",
      "185 -> ['NDB_386', 'NDB_473', 'NDB_538']\n",
      "186 -> ['NDB_455', 'NDB_554', 'NDB_587']\n",
      "187 -> ['NDB_376', 'NDB_443', 'NDB_557']\n",
      "188 -> ['NDB_350', 'NDB_433', 'NDB_611']\n",
      "189 -> ['NDB_388', 'NDB_397', 'NDB_448']\n",
      "190 -> ['NDB_445', 'NDB_458', 'NDB_619']\n",
      "191 -> ['NDB_482', 'NDB_514', 'NDB_529']\n",
      "192 -> ['NDB_556', 'NDB_571', 'NDB_629']\n",
      "193 -> ['NDB_391', 'NDB_466', 'NDB_539']\n",
      "194 -> ['NDB_401', 'NDB_436', 'NDB_487']\n",
      "195 -> ['NDB_385', 'NDB_472', 'NDB_578']\n",
      "196 -> ['NDB_532', 'NDB_544', 'NDB_547']\n",
      "197 -> ['NDB_365', 'NDB_506', 'NDB_604']\n",
      "198 -> ['NDB_421', 'NDB_431', 'NDB_628']\n",
      "199 -> ['NDB_356', 'NDB_505', 'NDB_601']\n",
      "200 -> ['NDB_371', 'NDB_439', 'NDB_581']\n",
      "201 -> ['NDB_446', 'NDB_494', 'NDB_589']\n",
      "202 -> ['NDB_491', 'NDB_551', 'NDB_614']\n",
      "203 -> ['NDB_470', 'NDB_526', 'NDB_622']\n",
      "204 -> ['NDB_428', 'NDB_460', 'NDB_511']\n",
      "205 -> ['NDB_418', 'NDB_563', 'NDB_619']\n",
      "206 -> ['NDB_455', 'NDB_464', 'NDB_625']\n",
      "207 -> ['NDB_335', 'NDB_452', 'NDB_493']\n",
      "208 -> ['NDB_458', 'NDB_481', 'NDB_536']\n",
      "209 -> ['NDB_341', 'NDB_409', 'NDB_508']\n",
      "210 -> ['NDB_403', 'NDB_473', 'NDB_562']\n",
      "211 -> ['NDB_400', 'NDB_427', 'NDB_608']\n",
      "212 -> ['NDB_506', 'NDB_562', 'NDB_619']\n",
      "213 -> ['NDB_379', 'NDB_455', 'NDB_473']\n",
      "214 -> ['NDB_476', 'NDB_494', 'NDB_554']\n",
      "215 -> ['NDB_376', 'NDB_472', 'NDB_611']\n",
      "216 -> ['NDB_335', 'NDB_514', 'NDB_568']\n",
      "217 -> ['NDB_359', 'NDB_431', 'NDB_557']\n",
      "218 -> ['NDB_445', 'NDB_461', 'NDB_487']\n",
      "219 -> ['NDB_436', 'NDB_448', 'NDB_529']\n",
      "220 -> ['NDB_332', 'NDB_604', 'NDB_629']\n",
      "221 -> ['NDB_421', 'NDB_505', 'NDB_544']\n",
      "222 -> ['NDB_365', 'NDB_460', 'NDB_571']\n",
      "223 -> ['NDB_403', 'NDB_493', 'NDB_539']\n",
      "224 -> ['NDB_398', 'NDB_427', 'NDB_508']\n",
      "225 -> ['NDB_418', 'NDB_439', 'NDB_556']\n",
      "226 -> ['NDB_409', 'NDB_466', 'NDB_481']\n",
      "228 -> ['NDB_401', 'NDB_479', 'NDB_547']\n",
      "229 -> ['NDB_347', 'NDB_458', 'NDB_578']\n",
      "230 -> ['NDB_391', 'NDB_395', 'NDB_571']\n",
      "231 -> ['NDB_472', 'NDB_482', 'NDB_551']\n",
      "232 -> ['NDB_452', 'NDB_473', 'NDB_628']\n",
      "233 -> ['NDB_332', 'NDB_431', 'NDB_587']\n",
      "234 -> ['NDB_421', 'NDB_461', 'NDB_491']\n",
      "235 -> ['NDB_359', 'NDB_536', 'NDB_604']\n",
      "236 -> ['NDB_592', 'NDB_611', 'NDB_625']\n",
      "237 -> ['NDB_538', 'NDB_581', 'NDB_622']\n",
      "238 -> ['NDB_379', 'NDB_554', 'NDB_614']\n",
      "239 -> ['NDB_433', 'NDB_455', 'NDB_533']\n",
      "240 -> ['NDB_493', 'NDB_506', 'NDB_563']\n",
      "241 -> ['NDB_403', 'NDB_464', 'NDB_589']\n",
      "242 -> ['NDB_392', 'NDB_481', 'NDB_511']\n",
      "243 -> ['NDB_400', 'NDB_418', 'NDB_517']\n",
      "244 -> ['NDB_386', 'NDB_467', 'NDB_608']\n",
      "245 -> ['NDB_397', 'NDB_577', 'NDB_619']\n",
      "246 -> ['NDB_460', 'NDB_494', 'NDB_526']\n",
      "247 -> ['NDB_341', 'NDB_428', 'NDB_476']\n",
      "248 -> ['NDB_368', 'NDB_371', 'NDB_470']\n",
      "249 -> ['NDB_356', 'NDB_388', 'NDB_508']\n",
      "250 -> ['NDB_385', 'NDB_482', 'NDB_628']\n",
      "751 -> ['NDB_356', 'NDB_427', 'NDB_472']\n",
      "752 -> ['NDB_461', 'NDB_508', 'NDB_616']\n",
      "758 -> ['NDB_508', 'NDB_557', 'NDB_581']\n",
      "759 -> ['NDB_403', 'NDB_491', 'NDB_592']\n",
      "760 -> ['NDB_347', 'NDB_376', 'NDB_409']\n",
      "761 -> ['NDB_479', 'NDB_493', 'NDB_536']\n",
      "762 -> ['NDB_427', 'NDB_476', 'NDB_628']\n",
      "763 -> ['NDB_341', 'NDB_386', 'NDB_401']\n",
      "764 -> ['NDB_431', 'NDB_445', 'NDB_464']\n",
      "765 -> ['NDB_436', 'NDB_466', 'NDB_533']\n",
      "766 -> ['NDB_473', 'NDB_544', 'NDB_554']\n",
      "767 -> ['NDB_350', 'NDB_395', 'NDB_539']\n",
      "768 -> ['NDB_356', 'NDB_428', 'NDB_482']\n",
      "769 -> ['NDB_418', 'NDB_571', 'NDB_587']\n",
      "770 -> ['NDB_359', 'NDB_458', 'NDB_514']\n",
      "772 -> ['NDB_365', 'NDB_446', 'NDB_470']\n",
      "773 -> ['NDB_461', 'NDB_517', 'NDB_614']\n",
      "774 -> ['NDB_332', 'NDB_388', 'NDB_487']\n",
      "775 -> ['NDB_371', 'NDB_481', 'NDB_568']\n",
      "776 -> ['NDB_400', 'NDB_578', 'NDB_622']\n",
      "777 -> ['NDB_398', 'NDB_421', 'NDB_608']\n",
      "778 -> ['NDB_448', 'NDB_581', 'NDB_589']\n",
      "779 -> ['NDB_335', 'NDB_526', 'NDB_577']\n",
      "780 -> ['NDB_365', 'NDB_443', 'NDB_452']\n",
      "781 -> ['NDB_385', 'NDB_556', 'NDB_619']\n",
      "782 -> ['NDB_538', 'NDB_557', 'NDB_625']\n",
      "783 -> ['NDB_506', 'NDB_511', 'NDB_611']\n",
      "784 -> ['NDB_379', 'NDB_464', 'NDB_562']\n",
      "785 -> ['NDB_445', 'NDB_494', 'NDB_604']\n",
      "786 -> ['NDB_368', 'NDB_455', 'NDB_472']\n",
      "787 -> ['NDB_508', 'NDB_529', 'NDB_629']\n",
      "788 -> ['NDB_397', 'NDB_433', 'NDB_616']\n",
      "789 -> ['NDB_443', 'NDB_563', 'NDB_601']\n",
      "790 -> ['NDB_460', 'NDB_532', 'NDB_551']\n",
      "791 -> ['NDB_439', 'NDB_467', 'NDB_547']\n",
      "792 -> ['NDB_391', 'NDB_392', 'NDB_505']\n",
      "793 -> ['NDB_341', 'NDB_359', 'NDB_436']\n",
      "794 -> ['NDB_347', 'NDB_400', 'NDB_556']\n",
      "795 -> ['NDB_427', 'NDB_446', 'NDB_479']\n",
      "796 -> ['NDB_385', 'NDB_418', 'NDB_577']\n",
      "797 -> ['NDB_332', 'NDB_356', 'NDB_401']\n",
      "798 -> ['NDB_466', 'NDB_487', 'NDB_622']\n",
      "799 -> ['NDB_536', 'NDB_538', 'NDB_614']\n",
      "800 -> ['NDB_409', 'NDB_491', 'NDB_533']\n",
      "801 -> ['NDB_376', 'NDB_398', 'NDB_461']\n",
      "802 -> ['NDB_431', 'NDB_481', 'NDB_544']\n",
      "803 -> ['NDB_448', 'NDB_554', 'NDB_616']\n",
      "804 -> ['NDB_460', 'NDB_608', 'NDB_628']\n",
      "805 -> ['NDB_428', 'NDB_539', 'NDB_611']\n",
      "806 -> ['NDB_371', 'NDB_386', 'NDB_571']\n",
      "807 -> ['NDB_368', 'NDB_458', 'NDB_511']\n",
      "808 -> ['NDB_476', 'NDB_493', 'NDB_592']\n",
      "809 -> ['NDB_395', 'NDB_403', 'NDB_526']\n",
      "810 -> ['NDB_376', 'NDB_455', 'NDB_532']\n",
      "811 -> ['NDB_388', 'NDB_526', 'NDB_544']\n",
      "812 -> ['NDB_433', 'NDB_443', 'NDB_491']\n",
      "813 -> ['NDB_332', 'NDB_385', 'NDB_581']\n",
      "814 -> ['NDB_400', 'NDB_466', 'NDB_557']\n",
      "815 -> ['NDB_481', 'NDB_551', 'NDB_616']\n",
      "816 -> ['NDB_538', 'NDB_571', 'NDB_592']\n",
      "817 -> ['NDB_494', 'NDB_529', 'NDB_611']\n",
      "818 -> ['NDB_436', 'NDB_470', 'NDB_547']\n",
      "819 -> ['NDB_350', 'NDB_418', 'NDB_622']\n",
      "820 -> ['NDB_335', 'NDB_403', 'NDB_421']\n",
      "821 -> ['NDB_448', 'NDB_511', 'NDB_533']\n",
      "822 -> ['NDB_409', 'NDB_445', 'NDB_482']\n",
      "823 -> ['NDB_472', 'NDB_506', 'NDB_577']\n",
      "824 -> ['NDB_467', 'NDB_473', 'NDB_629']\n",
      "825 -> ['NDB_431', 'NDB_446', 'NDB_461']\n",
      "826 -> ['NDB_476', 'NDB_601', 'NDB_608']\n",
      "827 -> ['NDB_341', 'NDB_458', 'NDB_460']\n",
      "828 -> ['NDB_398', 'NDB_481', 'NDB_493']\n",
      "829 -> ['NDB_464', 'NDB_568', 'NDB_571']\n",
      "830 -> ['NDB_365', 'NDB_401', 'NDB_536']\n",
      "831 -> ['NDB_563', 'NDB_578', 'NDB_614']\n",
      "832 -> ['NDB_487', 'NDB_604', 'NDB_625']\n",
      "833 -> ['NDB_347', 'NDB_439', 'NDB_562']\n",
      "834 -> ['NDB_371', 'NDB_505', 'NDB_539']\n",
      "835 -> ['NDB_395', 'NDB_554', 'NDB_619']\n",
      "836 -> ['NDB_391', 'NDB_427', 'NDB_452']\n",
      "837 -> ['NDB_356', 'NDB_587', 'NDB_628']\n",
      "838 -> ['NDB_392', 'NDB_479', 'NDB_517']\n",
      "839 -> ['NDB_379', 'NDB_386', 'NDB_556']\n",
      "840 -> ['NDB_359', 'NDB_385', 'NDB_397']\n",
      "841 -> ['NDB_433', 'NDB_508', 'NDB_514']\n",
      "842 -> ['NDB_368', 'NDB_464', 'NDB_532']\n",
      "843 -> ['NDB_398', 'NDB_544', 'NDB_589']\n",
      "844 -> ['NDB_428', 'NDB_466', 'NDB_494']\n",
      "845 -> ['NDB_578', 'NDB_601', 'NDB_604']\n",
      "846 -> ['NDB_335', 'NDB_388', 'NDB_611']\n",
      "847 -> ['NDB_397', 'NDB_428', 'NDB_506']\n",
      "848 -> ['NDB_514', 'NDB_533', 'NDB_538']\n",
      "849 -> ['NDB_473', 'NDB_614', 'NDB_619']\n",
      "850 -> ['NDB_467', 'NDB_493', 'NDB_589']\n",
      "851 -> ['NDB_466', 'NDB_505', 'NDB_508']\n",
      "852 -> ['NDB_350', 'NDB_571', 'NDB_625']\n",
      "853 -> ['NDB_455', 'NDB_491', 'NDB_568']\n",
      "854 -> ['NDB_356', 'NDB_577', 'NDB_629']\n",
      "855 -> ['NDB_436', 'NDB_526', 'NDB_554']\n",
      "856 -> ['NDB_386', 'NDB_587', 'NDB_616']\n",
      "857 -> ['NDB_341', 'NDB_379', 'NDB_385']\n",
      "858 -> ['NDB_418', 'NDB_446', 'NDB_539']\n",
      "859 -> ['NDB_421', 'NDB_458', 'NDB_557']\n",
      "860 -> ['NDB_472', 'NDB_563', 'NDB_608']\n",
      "861 -> ['NDB_335', 'NDB_433', 'NDB_578']\n",
      "862 -> ['NDB_359', 'NDB_448', 'NDB_494']\n",
      "863 -> ['NDB_403', 'NDB_443', 'NDB_445']\n",
      "864 -> ['NDB_460', 'NDB_493', 'NDB_611']\n",
      "866 -> ['NDB_332', 'NDB_470', 'NDB_511']\n",
      "867 -> ['NDB_350', 'NDB_431', 'NDB_601']\n",
      "868 -> ['NDB_395', 'NDB_536', 'NDB_556']\n",
      "869 -> ['NDB_476', 'NDB_481', 'NDB_622']\n",
      "870 -> ['NDB_461', 'NDB_551', 'NDB_628']\n",
      "871 -> ['NDB_401', 'NDB_427', 'NDB_517']\n",
      "872 -> ['NDB_365', 'NDB_368', 'NDB_479']\n",
      "873 -> ['NDB_392', 'NDB_464', 'NDB_547']\n",
      "874 -> ['NDB_398', 'NDB_400', 'NDB_409']\n",
      "875 -> ['NDB_391', 'NDB_482', 'NDB_532']\n",
      "876 -> ['NDB_544', 'NDB_562', 'NDB_581']\n",
      "877 -> ['NDB_376', 'NDB_592', 'NDB_629']\n",
      "878 -> ['NDB_439', 'NDB_452', 'NDB_568']\n",
      "879 -> ['NDB_433', 'NDB_487', 'NDB_538']\n",
      "880 -> ['NDB_347', 'NDB_460', 'NDB_614']\n",
      "881 -> ['NDB_428', 'NDB_431', 'NDB_529']\n",
      "882 -> ['NDB_371', 'NDB_508', 'NDB_604']\n",
      "883 -> ['NDB_445', 'NDB_455', 'NDB_511']\n",
      "884 -> ['NDB_356', 'NDB_581', 'NDB_604']\n",
      "885 -> ['NDB_368', 'NDB_557', 'NDB_629']\n",
      "886 -> ['NDB_347', 'NDB_461', 'NDB_529']\n",
      "887 -> ['NDB_395', 'NDB_467', 'NDB_476']\n",
      "888 -> ['NDB_371', 'NDB_592', 'NDB_608']\n",
      "889 -> ['NDB_398', 'NDB_439', 'NDB_446']\n",
      "890 -> ['NDB_376', 'NDB_452', 'NDB_458']\n",
      "891 -> ['NDB_386', 'NDB_400', 'NDB_614']\n",
      "892 -> ['NDB_421', 'NDB_493', 'NDB_622']\n",
      "893 -> ['NDB_335', 'NDB_470', 'NDB_481']\n",
      "894 -> ['NDB_392', 'NDB_466', 'NDB_571']\n",
      "895 -> ['NDB_350', 'NDB_491', 'NDB_538']\n",
      "896 -> ['NDB_379', 'NDB_401', 'NDB_578']\n",
      "897 -> ['NDB_473', 'NDB_517', 'NDB_601']\n",
      "898 -> ['NDB_341', 'NDB_431', 'NDB_625']\n",
      "899 -> ['NDB_365', 'NDB_514', 'NDB_539']\n",
      "900 -> ['NDB_388', 'NDB_403', 'NDB_532']\n",
      "901 -> ['NDB_427', 'NDB_533', 'NDB_544']\n",
      "902 -> ['NDB_448', 'NDB_526', 'NDB_556']\n",
      "903 -> ['NDB_479', 'NDB_505', 'NDB_589']\n",
      "904 -> ['NDB_482', 'NDB_547', 'NDB_611']\n",
      "905 -> ['NDB_391', 'NDB_397', 'NDB_563']\n",
      "906 -> ['NDB_460', 'NDB_554', 'NDB_562']\n",
      "907 -> ['NDB_443', 'NDB_464', 'NDB_487']\n",
      "908 -> ['NDB_332', 'NDB_418', 'NDB_536']\n",
      "909 -> ['NDB_359', 'NDB_409', 'NDB_619']\n",
      "910 -> ['NDB_506', 'NDB_568', 'NDB_581']\n",
      "911 -> ['NDB_436', 'NDB_557', 'NDB_628']\n",
      "912 -> ['NDB_452', 'NDB_508', 'NDB_551']\n",
      "913 -> ['NDB_428', 'NDB_433', 'NDB_446']\n",
      "914 -> ['NDB_577', 'NDB_608', 'NDB_616']\n",
      "915 -> ['NDB_395', 'NDB_472', 'NDB_614']\n",
      "916 -> ['NDB_347', 'NDB_385', 'NDB_511']\n",
      "917 -> ['NDB_359', 'NDB_532', 'NDB_611']\n",
      "918 -> ['NDB_401', 'NDB_409', 'NDB_464']\n",
      "919 -> ['NDB_332', 'NDB_443', 'NDB_554']\n",
      "920 -> ['NDB_392', 'NDB_448', 'NDB_604']\n",
      "921 -> ['NDB_431', 'NDB_494', 'NDB_547']\n",
      "922 -> ['NDB_386', 'NDB_487', 'NDB_544']\n",
      "923 -> ['NDB_472', 'NDB_481', 'NDB_571']\n",
      "925 -> ['NDB_452', 'NDB_592', 'NDB_614']\n",
      "926 -> ['NDB_371', 'NDB_395', 'NDB_577']\n",
      "927 -> ['NDB_446', 'NDB_458', 'NDB_466']\n",
      "928 -> ['NDB_427', 'NDB_433', 'NDB_622']\n",
      "929 -> ['NDB_379', 'NDB_439', 'NDB_505']\n",
      "930 -> ['NDB_461', 'NDB_506', 'NDB_557']\n",
      "931 -> ['NDB_350', 'NDB_467', 'NDB_517']\n",
      "932 -> ['NDB_365', 'NDB_473', 'NDB_493']\n",
      "933 -> ['NDB_514', 'NDB_616', 'NDB_629']\n",
      "934 -> ['NDB_445', 'NDB_529', 'NDB_587']\n",
      "935 -> ['NDB_341', 'NDB_533', 'NDB_562']\n",
      "936 -> ['NDB_398', 'NDB_526', 'NDB_619']\n",
      "937 -> ['NDB_356', 'NDB_418', 'NDB_625']\n",
      "938 -> ['NDB_397', 'NDB_400', 'NDB_482']\n",
      "939 -> ['NDB_376', 'NDB_556', 'NDB_563']\n",
      "940 -> ['NDB_391', 'NDB_403', 'NDB_479']\n",
      "941 -> ['NDB_536', 'NDB_539', 'NDB_628']\n",
      "942 -> ['NDB_388', 'NDB_476', 'NDB_538']\n",
      "943 -> ['NDB_368', 'NDB_460', 'NDB_578']\n",
      "944 -> ['NDB_589', 'NDB_608', 'NDB_611']\n",
      "945 -> ['NDB_448', 'NDB_508', 'NDB_568']\n",
      "946 -> ['NDB_401', 'NDB_511', 'NDB_551']\n",
      "947 -> ['NDB_347', 'NDB_491', 'NDB_547']\n",
      "948 -> ['NDB_421', 'NDB_428', 'NDB_443']\n",
      "949 -> ['NDB_470', 'NDB_487', 'NDB_601']\n",
      "950 -> ['NDB_428', 'NDB_467', 'NDB_551']\n",
      "951 -> ['NDB_335', 'NDB_443', 'NDB_622']\n",
      "952 -> ['NDB_403', 'NDB_616', 'NDB_619']\n",
      "953 -> ['NDB_487', 'NDB_577', 'NDB_578']\n",
      "954 -> ['NDB_452', 'NDB_581', 'NDB_587']\n",
      "955 -> ['NDB_439', 'NDB_445', 'NDB_601']\n",
      "956 -> ['NDB_356', 'NDB_421', 'NDB_556']\n",
      "957 -> ['NDB_472', 'NDB_562', 'NDB_629']\n",
      "958 -> ['NDB_371', 'NDB_526', 'NDB_529']\n",
      "959 -> ['NDB_385', 'NDB_436', 'NDB_493']\n",
      "960 -> ['NDB_359', 'NDB_466', 'NDB_538']\n",
      "961 -> ['NDB_400', 'NDB_455', 'NDB_589']\n",
      "962 -> ['NDB_388', 'NDB_494', 'NDB_505']\n",
      "963 -> ['NDB_347', 'NDB_557', 'NDB_608']\n",
      "964 -> ['NDB_448', 'NDB_458', 'NDB_517']\n",
      "965 -> ['NDB_482', 'NDB_506', 'NDB_592']\n",
      "966 -> ['NDB_446', 'NDB_544', 'NDB_604']\n",
      "967 -> ['NDB_376', 'NDB_431', 'NDB_554']\n",
      "968 -> ['NDB_379', 'NDB_568', 'NDB_587']\n",
      "969 -> ['NDB_341', 'NDB_397', 'NDB_461']\n",
      "970 -> ['NDB_511', 'NDB_547', 'NDB_614']\n",
      "971 -> ['NDB_365', 'NDB_418', 'NDB_427']\n",
      "972 -> ['NDB_409', 'NDB_479', 'NDB_539']\n",
      "973 -> ['NDB_464', 'NDB_476', 'NDB_611']\n",
      "974 -> ['NDB_532', 'NDB_536', 'NDB_563']\n",
      "975 -> ['NDB_392', 'NDB_401', 'NDB_625']\n",
      "976 -> ['NDB_368', 'NDB_386', 'NDB_514']\n",
      "977 -> ['NDB_395', 'NDB_460', 'NDB_473']\n",
      "978 -> ['NDB_398', 'NDB_601', 'NDB_628']\n",
      "979 -> ['NDB_332', 'NDB_472', 'NDB_533']\n",
      "980 -> ['NDB_335', 'NDB_350', 'NDB_508']\n",
      "981 -> ['NDB_491', 'NDB_571', 'NDB_616']\n",
      "982 -> ['NDB_421', 'NDB_433', 'NDB_470']\n",
      "983 -> ['NDB_379', 'NDB_391', 'NDB_551']\n",
      "984 -> ['NDB_481', 'NDB_529', 'NDB_577']\n",
      "985 -> ['NDB_368', 'NDB_445', 'NDB_536']\n",
      "986 -> ['NDB_386', 'NDB_461', 'NDB_539']\n",
      "987 -> ['NDB_335', 'NDB_460', 'NDB_517']\n",
      "988 -> ['NDB_398', 'NDB_455', 'NDB_494']\n",
      "989 -> ['NDB_359', 'NDB_371', 'NDB_554']\n",
      "990 -> ['NDB_332', 'NDB_514', 'NDB_526']\n",
      "991 -> ['NDB_391', 'NDB_533', 'NDB_581']\n",
      "True\n"
     ]
    }
   ],
   "source": [
    "reload(check)\n",
    "check_readout_dict = pickle.load(open(os.path.join(library_folder, 'readout_usage.pkl'), 'rb'))\n",
    "\n",
    "reg_readout_dic, reg2readout_check = check._check_region_to_readouts(full_records, check_readout_dict, \n",
    "                                                                     total_readout_num=num_readout_per_probe)\n",
    "print(reg2readout_check)\n",
    "# bit-to-adaptor usage"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 4.1.4 check readout usage in the library"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- extracting probes from >region:251, region:252, region:253, region:255, region:256, region:257, region:258, region:259, region:260, region:261, region:262, region:263, region:264, region:265, region:266, region:267, region:268, region:269, region:270, region:271, region:272, region:273, region:274, region:275, region:277, region:278, region:279, region:280, region:281, region:282, region:283, region:284, region:285, region:286, region:287, region:288, region:289, region:290, region:291, region:292, region:293, region:295, region:296, region:297, region:298, region:299, region:300, region:301, region:302, region:303, region:304, region:305, region:306, region:307, region:308, region:309, region:310, region:311, region:312, region:313, region:314, region:315, region:316, region:317, region:318, region:319, region:320, region:321, region:322, region:324, region:325, region:326, region:327, region:328, region:329, region:330, region:332, region:333, region:334, region:335, region:336, region:337, region:338, region:339, region:340, region:341, region:342, region:343, region:344, region:345, region:347, region:348, region:349, region:350, region:351, region:352, region:353, region:354, region:355, region:356, region:357, region:358, region:359, region:360, region:361, region:362, region:363, region:364, region:365, region:366, region:367, region:368, region:369, region:370, region:371, region:372, region:373, region:374, region:375, region:376, region:377, region:378, region:379, region:380, region:381, region:382, region:383, region:384, region:385, region:386, region:387, region:388, region:389, region:390, region:391, region:392, region:393, region:394, region:395, region:396, region:397, region:398, region:399, region:400, region:401, region:402, region:403, region:404, region:405, region:406, region:407, region:408, region:409, region:410, region:411, region:412, region:413, region:414, region:415, region:416, region:417, region:418, region:419, region:420, region:421, region:422, region:423, region:424, region:425, region:426, region:427, region:428, region:429, region:430, region:431, region:432, region:433, region:434, region:435, region:436, region:437, region:438, region:439, region:440, region:441, region:442, region:444, region:445, region:446, region:447, region:448, region:449, region:450, region:451, region:452, region:453, region:454, region:455, region:456, region:457, region:458, region:459, region:460, region:461, region:462, region:463, region:464, region:465, region:466, region:467, region:468, region:469, region:470, region:471, region:472, region:473, region:474, region:475, region:476, region:477, region:478, region:479, region:480, region:481, region:482, region:483, region:484, region:485, region:486, region:487, region:488, region:489, region:491, region:492, region:493, region:494, region:495, region:496, region:497, region:498, region:499, region:500, region:501, region:502, region:503, region:504, region:505, region:506, region:507, region:508, region:509, region:510, region:511, region:512, region:513, region:514, region:515, region:516, region:517, region:518, region:519, region:520, region:521, region:523, region:524, region:525, region:526, region:527, region:528, region:529, region:530, region:531, region:532, region:533, region:534, region:535, region:536, region:537, region:538, region:539, region:540, region:541, region:542, region:543, region:544, region:545, region:546, region:547, region:548, region:549, region:550, region:551, region:552, region:553, region:554, region:555, region:556, region:557, region:558, region:559, region:560, region:561, region:562, region:563, region:564, region:565, region:566, region:567, region:568, region:569, region:570, region:571, region:572, region:573, region:574, region:575, region:576, region:577, region:579, region:580, region:581, region:582, region:583, region:584, region:585, region:586, region:587, region:588, region:589, region:590, region:591, region:592, region:593, region:594, region:595, region:596, region:597, region:598, region:599, region:600, region:601, region:602, region:603, region:604, region:605, region:606, region:607, region:608, region:609, region:610, region:611, region:612, region:613, region:614, region:615, region:616, region:617, region:618, region:619, region:620, region:621, region:622, region:623, region:624, region:625, region:626, region:627, region:628, region:629, region:630, region:631, region:632, region:633, region:634, region:635, region:636, region:637, region:638, region:639, region:640, region:641, region:642, region:643, region:644, region:645, region:646, region:647, region:648, region:649, region:650, region:651, region:652, region:653, region:654, region:655, region:656, region:657, region:658, region:659, region:660, region:661, region:662, region:663, region:664, region:665, region:666, region:667, region:668, region:669, region:670, region:671, region:672, region:673, region:674, region:675, region:676, region:677, region:678, region:679, region:680, region:681, region:682, region:683, region:684, region:685, region:687, region:688, region:689, region:690, region:691, region:692, region:693, region:694, region:695, region:696, region:697, region:698, region:699, region:700, region:701, region:702, region:703, region:704, region:705, region:706, region:707, region:708, region:709, region:710, region:711, region:712, region:713, region:714, region:715, region:716, region:717, region:718, region:719, region:720, region:721, region:722, region:723, region:724, region:725, region:726, region:727, region:728, region:729, region:730, region:731, region:732, region:733, region:734, region:735, region:736, region:737, region:738, region:739, region:740, region:741, region:742, region:743, region:744, region:745, region:746, region:747, region:748, region:749, region:750, region:1, region:2, region:3, region:4, region:5, region:6, region:7, region:8, region:9, region:10, region:11, region:12, region:13, region:14, region:15, region:16, region:17, region:18, region:19, region:20, region:21, region:22, region:23, region:24, region:25, region:26, region:27, region:28, region:29, region:30, region:31, region:32, region:33, region:34, region:35, region:36, region:37, region:38, region:39, region:40, region:41, region:42, region:43, region:44, region:45, region:46, region:47, region:48, region:49, region:50, region:51, region:52, region:53, region:54, region:55, region:56, region:57, region:58, region:59, region:60, region:61, region:62, region:63, region:64, region:65, region:66, region:67, region:68, region:69, region:70, region:71, region:72, region:73, region:74, region:75, region:76, region:77, region:78, region:79, region:80, region:81, region:82, region:83, region:84, region:85, region:86, region:87, region:88, region:89, region:90, region:91, region:92, region:93, region:94, region:95, region:96, region:97, region:98, region:99, region:100, region:101, region:102, region:103, region:104, region:105, region:106, region:107, region:108, region:109, region:111, region:112, region:113, region:114, region:115, region:116, region:117, region:118, region:119, region:120, region:121, region:122, region:123, region:124, region:125, region:126, region:127, region:129, region:130, region:131, region:132, region:133, region:134, region:135, region:136, region:137, region:138, region:139, region:140, region:141, region:142, region:143, region:144, region:145, region:146, region:147, region:148, region:149, region:150, region:151, region:152, region:153, region:154, region:155, region:156, region:157, region:158, region:159, region:160, region:161, region:162, region:163, region:164, region:165, region:166, region:168, region:169, region:170, region:171, region:172, region:173, region:174, region:175, region:176, region:177, region:178, region:179, region:180, region:181, region:182, region:183, region:184, region:185, region:186, region:187, region:188, region:189, region:190, region:191, region:192, region:193, region:194, region:195, region:196, region:197, region:198, region:199, region:200, region:201, region:202, region:203, region:204, region:205"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ", region:206, region:207, region:208, region:209, region:210, region:211, region:212, region:213, region:214, region:215, region:216, region:217, region:218, region:219, region:220, region:221, region:222, region:223, region:224, region:225, region:226, region:228, region:229, region:230, region:231, region:232, region:233, region:234, region:235, region:236, region:237, region:238, region:239, region:240, region:241, region:242, region:243, region:244, region:245, region:246, region:247, region:248, region:249, region:250, region:751, region:752, region:758, region:759, region:760, region:761, region:762, region:763, region:764, region:765, region:766, region:767, region:768, region:769, region:770, region:772, region:773, region:774, region:775, region:776, region:777, region:778, region:779, region:780, region:781, region:782, region:783, region:784, region:785, region:786, region:787, region:788, region:789, region:790, region:791, region:792, region:793, region:794, region:795, region:796, region:797, region:798, region:799, region:800, region:801, region:802, region:803, region:804, region:805, region:806, region:807, region:808, region:809, region:810, region:811, region:812, region:813, region:814, region:815, region:816, region:817, region:818, region:819, region:820, region:821, region:822, region:823, region:824, region:825, region:826, region:827, region:828, region:829, region:830, region:831, region:832, region:833, region:834, region:835, region:836, region:837, region:838, region:839, region:840, region:841, region:842, region:843, region:844, region:845, region:846, region:847, region:848, region:849, region:850, region:851, region:852, region:853, region:854, region:855, region:856, region:857, region:858, region:859, region:860, region:861, region:862, region:863, region:864, region:866, region:867, region:868, region:869, region:870, region:871, region:872, region:873, region:874, region:875, region:876, region:877, region:878, region:879, region:880, region:881, region:882, region:883, region:884, region:885, region:886, region:887, region:888, region:889, region:890, region:891, region:892, region:893, region:894, region:895, region:896, region:897, region:898, region:899, region:900, region:901, region:902, region:903, region:904, region:905, region:906, region:907, region:908, region:909, region:910, region:911, region:912, region:913, region:914, region:915, region:916, region:917, region:918, region:919, region:920, region:921, region:922, region:923, region:925, region:926, region:927, region:928, region:929, region:930, region:931, region:932, region:933, region:934, region:935, region:936, region:937, region:938, region:939, region:940, region:941, region:942, region:943, region:944, region:945, region:946, region:947, region:948, region:949, region:950, region:951, region:952, region:953, region:954, region:955, region:956, region:957, region:958, region:959, region:960, region:961, region:962, region:963, region:964, region:965, region:966, region:967, region:968, region:969, region:970, region:971, region:972, region:973, region:974, region:975, region:976, region:977, region:978, region:979, region:980, region:981, region:982, region:983, region:984, region:985, region:986, region:987, region:988, region:989, region:990, region:991, {'NDB_487': ['251', '279', '298', '370', '417', '451', '473', '499', '511', '575', '600', '628', '651', '669', '703', '728', '23', '72', '104', '141', '166', '194', '218', '774', '798', '832', '879', '907', '922', '949', '953'], 'NDB_628': ['251', '293', '332', '368', '423', '442', '458', '521', '568', '599', '629', '663', '672', '704', '716', '51', '74', '107', '140', '158', '198', '232', '250', '762', '804', '837', '870', '911', '941', '978'], 'NDB_629': ['251', '304', '316', '356', '384', '418', '457', '484', '510', '537', '605', '626', '645', '674', '736', '21', '67', '96', '118', '143', '171', '192', '220', '787', '824', '854', '877', '885', '933', '957'], 'NDB_350': ['252', '269', '309', '333', '354', '416', '460', '488', '499', '514', '563', '592', '635', '656', '677', '713', '748', '4', '56', '75', '102', '142', '152', '188', '767', '819', '852', '867', '895', '931', '980'], 'NDB_458': ['252', '283', '317', '370', '387', '410', '450', '475', '528', '566', '605', '640', '683', '715', '745', '6', '31', '57', '93', '122', '190', '208', '229', '770', '807', '827', '859', '890', '927', '964'], 'NDB_616': ['252', '273', '296', '314', '362', '403', '442', '480', '500', '532', '560', '593', '621', '649', '699', '714', '18', '43', '100', '113', '145', '182', '752', '788', '803', '815', '856', '914', '933', '952', '981'], 'NDB_335': ['253', '284', '326', '339', '385', '396', '466', '515', '526', '554', '595', '629', '691', '720', '745', '3', '40', '69', '78', '124', '166', '207', '216', '779', '820', '846', '861', '893', '951', '980', '987'], 'NDB_409': ['253', '311', '341', '361', '422', '463', '496', '538', '574', '590', '619', '667', '669', '708', '749', '12', '31', '55', '102', '127', '177', '209', '226', '760', '800', '822', '874', '909', '918', '972'], 'NDB_562': ['253', '292', '319', '366', '381', '423', '464', '487', '497', '536', '556', '586', '641', '690', '703', '732', '42', '57', '92', '111', '147', '180', '210', '212', '784', '833', '876', '906', '935', '957'], 'NDB_439': ['255', '296', '330', '356', '422', '437', '458', '527', '540', '566', '597', '636', '660', '678', '709', '747', '17', '30', '64', '87', '136', '162', '200', '225', '791', '833', '878', '889', '929', '955'], 'NDB_443': ['255', '271', '291', '333', '371', '408', '440', '478', '500', '517', '568', '584', '617', '653', '680', '729', '743', '33', '63', '80', '97', '115', '187', '780', '789', '812', '863', '907', '919', '948', '951'], 'NDB_532': ['255', '268', '279', '315', '360', '423', '447', '482', '503', '530', '547', '589', '638', '697', '721', '731', '31', '44', '73', '94', '124', '157', '196', '790', '810', '842', '875', '900', '917', '974'], 'NDB_401': ['256', '277', '303', '338', '383', '392', '402', '469', '498', '545', '570', '601', '616', '640', '670', '714', '24', '53', '119', '135', '171', '194', '228', '763', '797', '830', '871', '896', '918', '946', '975'], 'NDB_446': ['256', '265', '297', '337', '366', '408', '446', '480', '505', '555', '582', '612', '667', '684', '723', '735', '29', '41', '60', '83', '126', '179', '201', '772', '795', '825', '858', '889', '913', '927', '966'], 'NDB_601': ['256', '274', '284', '319', '347', '378', '420', '471', '520', '534', '551', '596', '633', '688', '725', '741', '9', '34', '82', '113', '149', '181', '199', '789', '826', '845', '867', '897', '949', '955', '978'], 'NDB_427': ['257', '291', '324', '347', '361', '426', '444', '482', '527', '548', '586', '634', '683', '707', '25', '61', '78', '96', '146', '183', '211', '224', '751', '762', '795', '836', '871', '901', '928', '971'], 'NDB_505': ['257', '290', '327', '360', '380', '407', '449', '487', '492', '523', '581', '607', '643', '657', '693', '704', '743', '24', '60', '114', '134', '173', '199', '221', '792', '834', '851', '903', '929', '962'], 'NDB_514': ['257', '313', '325', '374', '389', '395', '435', '467', '514', '531', '571', '601', '638', '654', '688', '708', '747', '27', '54', '89', '120', '175', '191', '216', '770', '841', '848', '899', '933', '976', '990'], 'NDB_395': ['258', '314', '326', '349', '377', '400', '433', '448', '461', '524', '575', '603', '615', '648', '675', '716', '24', '57', '87', '118', '153', '168', '230', '767', '809', '835', '868', '887', '915', '926', '977'], 'NDB_544': ['258', '280', '318', '350', '376', '424', '459', '495', '514', '569', '598', '625', '661', '676', '738', '48', '67', '106', '145', '165', '196', '221', '766', '802', '811', '843', '876', '901', '922', '966'], 'NDB_568': ['258', '279', '319', '337', '386', '398', '414', '470', '513', '543', '549', '604', '626', '647', '701', '712', '742', '18', '68', '107', '144', '155', '172', '216', '775', '829', '853', '878', '910', '945', '968'], 'NDB_391': ['259', '280', '317', '342', '359', '393', '429', '468', '501', '531', '574', '604', '618', '652', '672', '710', '746', '17', '41', '79', '139', '166', '193', '230', '792', '836', '875', '905', '940', '983', '991'], 'NDB_445': ['259', '275', '314', '355', '374', '398', '434', '469', '512', '533', '576', '608', '629', '643', '697', '719', '46', '70', '85', '133', '170', '190', '218', '764', '785', '822', '863', '883', '934', '955', '985'], 'NDB_557': ['259', '278', '330', '362', '379', '411', '466', '498', '534', '549', '598', '623', '646', '689', '707', '744', '4', '55', '117', '148', '154', '187', '217', '758', '782', '814', '859', '885', '911', '930', '963'], 'NDB_376': ['260', '275', '303', '342', '380', '394', '436', '467', '505', '515', '569', '599', '620', '649', '679', '726', '20', '39', '104', '137', '155', '187', '215', '760', '801', '810', '877', '890', '939', '967'], 'NDB_433': ['260', '283', '327', '345', '390', '415', '459', '485', '494', '538', '546', '577', '639', '652', '685', '712', '35', '66', '81', '118', '148', '188', '239', '788', '812', '841', '861', '879', '913', '928', '982'], 'NDB_436': ['260', '292', '315', '335', '352', '410', '426', '438', '476', '504', '576', '603', '614', '650', '668', '706', '733', '12', '48', '88', '109', '175', '194', '219', '765', '793', '818', '855', '911', '959'], 'NDB_400': ['261', '302', '315', '369', '427', '473', '485', '523', '560', '579', '633', '664', '675', '701', '729', '51', '64', '98', '120', '142', '160', '211', '243', '776', '794', '814', '874', '891', '938', '961'], 'NDB_526': ['261', '303', '358', '373', '421', '446', '487', '513', '573', '600', '611', '635', '699', '719', '750', '25', '55', '84', '105', '138', '159', '203', '246', '779', '809', '811', '855', '902', '936', '958', '990'], 'NDB_547': ['261', '281', '324', '343', '368', '408', '430', '466', '481', '512', '542', '575', '607', '624', '647', '677', '724', '32', '49', '101', '139', '174', '196', '228', '791', '818', '873', '904', '921', '947', '970'], 'NDB_448': ['262', '285', '327', '339', '381', '416', '425', '457', '493', '545', '574', '602', '624', '646', '678', '735', '27', '63', '106', '141', '164', '189', '219', '778', '803', '821', '862', '902', '920', '945', '964'], 'NDB_467': ['262', '280', '332', '336', '371', '399', '432', '463', '489', '506', '543', '554', '585', '640', '684', '727', '20', '45', '62', '99', '131', '157', '184', '244', '791', '824', '850', '887', '931', '950'], 'NDB_587': ['262', '281', '320', '348', '377', '396', '412', '439', '482', '518', '536', '566', '577', '619', '645', '695', '723', '1', '13', '54', '90', '114', '186', '233', '769', '837', '856', '934', '954', '968'], 'NDB_385': ['263', '284', '317', '357', '376', '411', '444', '477', '507', '579', '602', '639', '653', '682', '714', '749', '13', '50', '116', '133', '172', '195', '250', '781', '796', '813', '840', '857', '916', '959'], 'NDB_491': ['263', '278', '329', '369', '425', '448', '479', '509', '555', '594', '636', '665', '668', '703', '704', '719', '5', '44', '58', '99', '119', '156', '202', '234', '759', '800', '812', '853', '895', '947', '981'], 'NDB_563': ['263', '285', '320', '350', '362', '404', '417', '470', '496', '537', '557', '586', '642', '698', '711', '747', '7', '40', '85', '98', '138', '161', '205', '240', '789', '831', '860', '905', '939', '974'], 'NDB_529': ['264', '297', '321', '364', '386', '419', '437', '472', '497', '539', '606', '621', '659', '698', '738', '6', '19', '38', '94', '119', '168', '191', '219', '787', '817', '881', '886', '934', '958', '984'], 'NDB_536': ['264', '295', '334', '353', '429', '454', '494', '535', '569', '583', '634', '657', '677', '708', '725', '14', '62', '88', '112', '130', '179', '208', '235', '761', '799', '830', '868', '908', '941', '974', '985'], 'NDB_589': ['264', '293', '313', '363', '397', '436', '440', '479', '502', '511', '550', '592', '642', '683', '712', '741', '22', '47', '69', '84', '133', '162', '201', '241', '778', '843', '850', '903', '944', '961'], 'NDB_511': ['265', '296', '344', '358', '393', '412', '441', '453', '521', '557', '590', '632', '662', '679', '729', '2', '47', '75', '99', '124', '165', '204', '242', '783', '807', '821', '866', '883', '916', '946', '970'], 'NDB_577': ['265', '286', '301', '348', '382', '416', '455', '465', '527', '546', '572', '608', '622', '647', '676', '690', '717', '10', '53', '88', '120', '184', '245', '779', '796', '823', '854', '914', '926', '953', '984'], 'NDB_452': ['266', '310', '336', '360', '406', '427', '471', '488', '491', '510', '556', '606', '615', '662', '669', '739', '10', '48', '86', '130', '148', '161', '207', '232', '780', '836', '878', '890', '912', '925', '954'], 'NDB_470': ['266', '309', '353', '375', '400', '434', '439', '479', '528', '570', '588', '613', '663', '674', '690', '709', '731', '16', '68', '79', '103', '145', '178', '203', '248', '772', '818', '866', '893', '949', '982'], 'NDB_517': ['266', '282', '325', '367', '413', '430', '441', '476', '502', '533', '553', '610', '621', '657', '687', '721', '744', '7', '41', '59', '95', '127', '180', '243', '773', '838', '871', '897', '931', '964', '987'], 'NDB_386': ['267', '288', '324', '349', '379', '401', '429', '457', '491', '507', '551', '595', '641', '693', '721', '16', '47', '123', '138', '151', '152', '185', '244', '763', '806', '839', '856', '891', '922', '976', '986'], 'NDB_551': ['267', '290', '321', '354', '376', '405', '428', '473', '484', '520', '558', '576', '642', '685', '722', '745', '28', '61', '76', '101', '125', '180', '202', '231', '790', '815', '870', '912', '946', '950', '983'], 'NDB_581': ['267', '309', '370', '415', '442', '481', '504', '530', '557', '596', '616', '665', '691', '705', '736', '15', '45', '83', '112', '137', '183', '200', '237', '758', '778', '813', '876', '884', '910', '954', '991'], 'NDB_392': ['268', '311', '363', '389', '409', '427', '446', '486', '524', '561', '587', '613', '658', '702', '733', '742', '9', '28', '70', '92', '122', '156', '176', '242', '792', '838', '873', '894', '920', '975'], 'NDB_608': ['268', '307', '344', '386', '407', '451', '528', '541', '565', '591', '631', '653', '696', '713', '730', '3', '29', '66', '101', '132', '164', '211', '244', '777', '804', '826', '860', '888', '914', '944', '963'], 'NDB_397': ['269', '306', '352', '383', '401', '434', '451', '489', '503', '542', '552', '590', '612', '654', '687', '739', '33', '69', '77', '96', '150', '153', '189', '245', '788', '840', '847', '905', '938', '969'], 'NDB_592': ['269', '300', '328', '340', '350', '396', '433', '452', '497', '540', '550', '588', '630', '643', '680', '735', '2', '34', '76', '105', '127', '183', '236', '759', '808', '816', '877', '888', '925', '965'], 'NDB_368': ['270', '292', '318', '351', '390', '392', '419', '465', '474', '523', '552', '591', '628', '648', '682', '723', '30', '40', '56', '105', '131', '181', '248', '786', '807', '842', '872', '885', '943', '976', '985'], 'NDB_388': ['270', '307', '334', '368', '399', '433', '445', '480', '518', '562', '605', '614', '662', '685', '711', '744', '8', '22', '36', '108', '136', '176', '189', '249', '774', '811', '846', '900', '942', '962'], 'NDB_625': ['270', '293', '364', '381', '414', '428', '447', '476', '510', '567', '584', '612', '661', '673', '710', '725', '13', '35', '87', '100', '121', '163', '206', '236', '782', '832', '852', '898', '937', '975'], 'NDB_371': ['271', '310', '345', '389', '412', '430', '447', '477', '519', '563', '585', '633', '666', '689', '732', '22', '37', '67', '104', '126', '158', '200', '248', '775', '806', '834', '882', '888', '926', '958', '989'], 'NDB_614': ['271', '299', '342', '387', '398', '472', '492', '508', '561', '580', '614', '667', '673', '728', '4', '43', '71', '90', '132', '181', '202', '238', '773', '799', '831', '849', '880', '891', '915', '925', '970'], 'NDB_341': ['272', '289', '301', '351', '380', '404', '454', '471', '525', '561', '592', '630', '678', '718', '730', '21', '45', '78', '97', '144', '158', '209', '247', '763', '793', '827', '857', '898', '935', '969'], 'NDB_464': ['272', '308', '358', '375', '424', '438', '485', '526', '564', '582', '615', '666', '698', '713', '43', '61', '77', '91', '140', '184', '206', '241', '764', '784', '829', '842', '873', '907', '918', '973'], 'NDB_538': ['272', '275', '287', '344', '383', '418', '464', '495', '543', '559', '606', '619', '648', '692', '740', '50', '73', '98', '113', '139', '185', '237', '782', '799', '816', '848', '879', '895', '942', '960'], 'NDB_356': ['273', '297', '333', '355', '405', '459', '496', '535', '554', '580', '632', '664', '696', '706', '11', '49', '79', '92', '144', '159', '199', '249', '751', '768', '797', '837', '854', '884', '937', '956'], 'NDB_622': ['273', '300', '372', '394', '435', '449', '474', '502', '516', '572', '598', '644', '666', '673', '700', '739', '5', '19', '70', '93', '129', '157', '203', '237', '776', '798', '819', '869', '892', '928', '951'], 'NDB_428': ['274', '307', '357', '377', '425', '450', '491', '531', '549', '564', '607', '634', '663', '700', '705', '734', '12', '71', '85', '143', '182', '204', '247', '768', '805', '844', '847', '881', '913', '948', '950'], 'NDB_533': ['274', '290', '306', '338', '356', '421', '445', '478', '508', '550', '591', '620', '644', '675', '720', '748', '1', '38', '74', '95', '116', '179', '239', '765', '800', '821', '848', '901', '935', '979', '991'], 'NDB_359': ['277', '304', '355', '378', '393', '414', '449', '500', '539', '564', '599', '635', '695', '727', '3', '30', '72', '95', '125', '156', '178', '217', '235', '770', '793', '840', '862', '909', '917', '960', '989'], 'NDB_461': ['277', '312', '340', '391', '410', '461', '474', '526', '571', '602', '626', '652', '681', '732', '25', '52', '112', '134', '162', '176', '218', '234', '752', '773', '801', '825', '870', '886', '930', '969', '986'], 'NDB_578': ['278', '321', '366', '399', '456', '481', '521', '529', '547', '587', '610', '611', '637', '674', '710', '726', '33', '54', '102', '146', '169', '195', '229', '776', '831', '845', '861', '896', '943', '953'], 'NDB_481': ['281', '302', '336', '365', '426', '460', '498', '544', '555', '581', '639', '671', '716', '746', '15', '42', '77', '97', '143', '164', '208', '226', '242', '775', '802', '815', '828', '869', '893', '923', '984'], 'NDB_493': ['282', '305', '351', '382', '407', '431', '453', '467', '512', '563', '584', '641', '701', '736', '1', '36', '72', '82', '122', '182', '207', '223', '240', '761', '808', '828', '850', '864', '892', '932', '959'], 'NDB_554': ['282', '330', '343', '388', '409', '445', '465', '506', '559', '594', '631', '700', '718', '749', '15', '26', '46', '94', '134', '152', '186', '214', '238', '766', '803', '835', '855', '906', '919', '967', '989'], 'NDB_379': ['283', '316', '363', '369', '403', '452', '483', '499', '542', '571', '589', '623', '650', '671', '717', '750', '7', '20', '74', '106', '129', '178', '213', '238', '784', '839', '857', '896', '929', '968', '983'], 'NDB_611': ['285', '322', '365', '402', '448', '478', '520', '533', '560', '587', '628', '655', '684', '737', '11', '52', '109', '130', '147', '188', '215', '236', '783', '805', '817', '846', '864', '904', '917', '944', '973'], 'NDB_431': ['286', '322', '341', '385', '417', '462', '494', '516', '580', '597', '611', '638', '687', '740', '36', '65', '80', '111', '171', '198', '217', '233', '764', '802', '825', '867', '881', '898', '921', '967'], 'NDB_539': ['286', '320', '335', '373', '415', '432', '469', '511', '525', '552', '594', '623', '658', '696', '726', '19', '34', '63', '116', '136', '174', '193', '223', '767', '805', '834', '858', '899', '941', '972', '986'], 'NDB_365': ['287', '298', '341', '378', '397', '413', '468', '506', '544', '562', '579', '620', '645', '689', '738', '11', '28', '44', '132', '151', '173', '197', '222', '772', '780', '830', '872', '899', '932', '971'], 'NDB_482': ['287', '334', '367', '382', '420', '438', '461', '519', '536', '570', '609', '627', '651', '671', '702', '718', '5', '35', '117', '137', '165', '191', '231', '250', '768', '822', '875', '904', '938', '965'], 'NDB_347': ['288', '318', '372', '397', '439', '483', '517', '530', '567', '593', '618', '656', '670', '702', '737', '21', '46', '65', '81', '126', '153', '175', '229', '760', '794', '833', '880', '886', '916', '947', '963'], 'NDB_604': ['288', '302', '352', '391', '419', '452', '488', '492', '509', '573', '601', '622', '649', '694', '734', '17', '66', '115', '147', '174', '197', '220', '235', '785', '832', '845', '882', '884', '920', '966'], 'NDB_466': ['289', '329', '349', '384', '395', '470', '518', '535', '556', '585', '627', '656', '682', '731', '741', '32', '52', '100', '115', '146', '170', '193', '226', '765', '798', '814', '844', '851', '894', '927', '960'], 'NDB_472': ['289', '328', '353', '373', '409', '440', '489', '509', '534', '577', '604', '624', '650', '691', '715', '37', '65', '86', '140', '163', '195', '215', '231', '751', '786', '823', '860', '915', '923', '957', '979'], 'NDB_556': ['291', '322', '343', '359', '424', '464', '508', '532', '562', '610', '616', '651', '688', '730', '742', '10', '37', '58', '93', '114', '170', '192', '225', '781', '794', '839', '868', '902', '939', '956'], 'NDB_460': ['295', '338', '384', '401', '460', '505', '540', '548', '572', '609', '613', '665', '694', '740', '27', '49', '86', '149', '160', '204', '222', '246', '790', '804', '827', '864', '880', '906', '943', '977', '987'], 'NDB_476': ['295', '312', '359', '374', '428', '455', '513', '539', '553', '589', '625', '646', '670', '706', '748', '23', '64', '103', '123', '150', '177', '214', '247', '762', '808', '826', '869', '887', '942', '973'], 'NDB_494': ['298', '310', '361', '375', '392', '431', '454', '486', '507', '537', '546', '596', '636', '680', '722', '18', '42', '59', '90', '129', '163', '201', '214', '246', '785', '817', '844', '862', '921', '962', '988'], 'NDB_332': ['299', '326', '340', '390', '413', '431', '463', '493', '541', '567', '608', '627', '655', '693', '724', '6', '39', '107', '150', '154', '169', '220', '233', '774', '797', '813', '866', '908', '919', '979', '990'], 'NDB_506': ['299', '325', '364', '394', '432', '458', '475', '501', '517', '548', '582', '637', '695', '722', '733', '16', '58', '73', '108', '135', '159', '197', '212', '240', '783', '823', '847', '910', '930', '965'], 'NDB_398': ['300', '316', '371', '400', '456', '472', '501', '532', '558', '583', '622', '658', '697', '720', '50', '75', '91', '111', '141', '154', '173', '224', '777', '801', '828', '843', '874', '889', '936', '978', '988'], 'NDB_571': ['301', '313', '339', '388', '406', '420', '444', '483', '503', '568', '600', '625', '660', '681', '734', '8', '60', '109', '135', '169', '192', '222', '230', '769', '806', '816', '829', '852', '894', '923', '981'], 'NDB_403': ['304', '354', '367', '422', '455', '477', '516', '529', '551', '583', '632', '681', '707', '724', '26', '38', '68', '83', '131', '161', '210', '223', '241', '759', '809', '820', '863', '900', '940', '952'], 'NDB_455': ['305', '372', '387', '405', '437', '462', '495', '544', '565', '595', '630', '655', '672', '699', '705', '727', '9', '53', '89', '103', '117', '186', '206', '213', '239', '786', '810', '853', '883', '961', '988'], 'NDB_619': ['305', '337', '391', '395', '406', '468', '493', '515', '547', '609', '644', '661', '692', '709', '743', '23', '59', '76', '81', '142', '190', '205', '212', '245', '781', '835', '849', '909', '936', '952'], 'NDB_473': ['306', '335', '348', '411', '435', '450', '486', '519', '538', '558', '597', '617', '664', '679', '737', '14', '29', '56', '84', '108', '185', '210', '213', '232', '766', '824', '849', '897', '932', '977'], 'NDB_418': ['308', '345', '385', '403', '418', '456', '524', '545', '565', '581', '618', '659', '694', '715', '2', '26', '71', '82', '125', '177', '205', '225', '243', '769', '796', '819', '858', '908', '937', '971'], 'NDB_508': ['308', '332', '357', '421', '453', '475', '529', '553', '588', '631', '660', '668', '717', '14', '39', '80', '121', '151', '160', '209', '224', '249', '752', '758', '787', '841', '851', '882', '912', '945', '980'], 'NDB_421': ['311', '328', '347', '388', '402', '436', '441', '484', '525', '573', '603', '637', '659', '692', '711', '746', '32', '62', '89', '123', '172', '198', '221', '234', '777', '820', '859', '892', '948', '956', '982'], 'NDB_479': ['312', '329', '365', '379', '404', '462', '504', '541', '559', '593', '617', '654', '676', '728', '750', '8', '51', '91', '121', '149', '155', '168', '228', '761', '795', '838', '872', '903', '940', '972']}\n",
      " > Done.\n",
      "-- inverting region_to_readout dict\n",
      "NDB_332 -> ['107', '150', '154', '169', '220', '233', '299', '326', '340', '39', '390', '413', '431', '463', '493', '541', '567', '6', '608', '627', '655', '693', '724', '774', '797', '813', '866', '908', '919', '979', '990']\n",
      "NDB_335 -> ['124', '166', '207', '216', '253', '284', '3', '326', '339', '385', '396', '40', '466', '515', '526', '554', '595', '629', '69', '691', '720', '745', '779', '78', '820', '846', '861', '893', '951', '980', '987']\n",
      "NDB_341 -> ['144', '158', '209', '21', '247', '272', '289', '301', '351', '380', '404', '45', '454', '471', '525', '561', '592', '630', '678', '718', '730', '763', '78', '793', '827', '857', '898', '935', '969', '97']\n",
      "NDB_347 -> ['126', '153', '175', '21', '229', '288', '318', '372', '397', '439', '46', '483', '517', '530', '567', '593', '618', '65', '656', '670', '702', '737', '760', '794', '81', '833', '880', '886', '916', '947', '963']\n",
      "NDB_350 -> ['102', '142', '152', '188', '252', '269', '309', '333', '354', '4', '416', '460', '488', '499', '514', '56', '563', '592', '635', '656', '677', '713', '748', '75', '767', '819', '852', '867', '895', '931', '980']\n",
      "NDB_356 -> ['11', '144', '159', '199', '249', '273', '297', '333', '355', '405', '459', '49', '496', '535', '554', '580', '632', '664', '696', '706', '751', '768', '79', '797', '837', '854', '884', '92', '937', '956']\n",
      "NDB_359 -> ['125', '156', '178', '217', '235', '277', '3', '30', '304', '355', '378', '393', '414', '449', '500', '539', '564', '599', '635', '695', '72', '727', '770', '793', '840', '862', '909', '917', '95', '960', '989']\n",
      "NDB_365 -> ['11', '132', '151', '173', '197', '222', '28', '287', '298', '341', '378', '397', '413', '44', '468', '506', '544', '562', '579', '620', '645', '689', '738', '772', '780', '830', '872', '899', '932', '971']\n",
      "NDB_368 -> ['105', '131', '181', '248', '270', '292', '30', '318', '351', '390', '392', '40', '419', '465', '474', '523', '552', '56', '591', '628', '648', '682', '723', '786', '807', '842', '872', '885', '943', '976', '985']\n",
      "NDB_371 -> ['104', '126', '158', '200', '22', '248', '271', '310', '345', '37', '389', '412', '430', '447', '477', '519', '563', '585', '633', '666', '67', '689', '732', '775', '806', '834', '882', '888', '926', '958', '989']\n",
      "NDB_376 -> ['104', '137', '155', '187', '20', '215', '260', '275', '303', '342', '380', '39', '394', '436', '467', '505', '515', '569', '599', '620', '649', '679', '726', '760', '801', '810', '877', '890', '939', '967']\n",
      "NDB_379 -> ['106', '129', '178', '20', '213', '238', '283', '316', '363', '369', '403', '452', '483', '499', '542', '571', '589', '623', '650', '671', '7', '717', '74', '750', '784', '839', '857', '896', '929', '968', '983']\n",
      "NDB_385 -> ['116', '13', '133', '172', '195', '250', '263', '284', '317', '357', '376', '411', '444', '477', '50', '507', '579', '602', '639', '653', '682', '714', '749', '781', '796', '813', '840', '857', '916', '959']\n",
      "NDB_386 -> ['123', '138', '151', '152', '16', '185', '244', '267', '288', '324', '349', '379', '401', '429', '457', '47', '491', '507', '551', '595', '641', '693', '721', '763', '806', '839', '856', '891', '922', '976', '986']\n",
      "NDB_388 -> ['108', '136', '176', '189', '22', '249', '270', '307', '334', '36', '368', '399', '433', '445', '480', '518', '562', '605', '614', '662', '685', '711', '744', '774', '8', '811', '846', '900', '942', '962']\n",
      "NDB_391 -> ['139', '166', '17', '193', '230', '259', '280', '317', '342', '359', '393', '41', '429', '468', '501', '531', '574', '604', '618', '652', '672', '710', '746', '79', '792', '836', '875', '905', '940', '983', '991']\n",
      "NDB_392 -> ['122', '156', '176', '242', '268', '28', '311', '363', '389', '409', '427', '446', '486', '524', '561', '587', '613', '658', '70', '702', '733', '742', '792', '838', '873', '894', '9', '92', '920', '975']\n",
      "NDB_395 -> ['118', '153', '168', '230', '24', '258', '314', '326', '349', '377', '400', '433', '448', '461', '524', '57', '575', '603', '615', '648', '675', '716', '767', '809', '835', '868', '87', '887', '915', '926', '977']\n",
      "NDB_397 -> ['150', '153', '189', '245', '269', '306', '33', '352', '383', '401', '434', '451', '489', '503', '542', '552', '590', '612', '654', '687', '69', '739', '77', '788', '840', '847', '905', '938', '96', '969']\n",
      "NDB_398 -> ['111', '141', '154', '173', '224', '300', '316', '371', '400', '456', '472', '50', '501', '532', '558', '583', '622', '658', '697', '720', '75', '777', '801', '828', '843', '874', '889', '91', '936', '978', '988']\n",
      "NDB_400 -> ['120', '142', '160', '211', '243', '261', '302', '315', '369', '427', '473', '485', '51', '523', '560', '579', '633', '64', '664', '675', '701', '729', '776', '794', '814', '874', '891', '938', '961', '98']\n",
      "NDB_401 -> ['119', '135', '171', '194', '228', '24', '256', '277', '303', '338', '383', '392', '402', '469', '498', '53', '545', '570', '601', '616', '640', '670', '714', '763', '797', '830', '871', '896', '918', '946', '975']\n",
      "NDB_403 -> ['131', '161', '210', '223', '241', '26', '304', '354', '367', '38', '422', '455', '477', '516', '529', '551', '583', '632', '68', '681', '707', '724', '759', '809', '820', '83', '863', '900', '940', '952']\n",
      "NDB_409 -> ['102', '12', '127', '177', '209', '226', '253', '31', '311', '341', '361', '422', '463', '496', '538', '55', '574', '590', '619', '667', '669', '708', '749', '760', '800', '822', '874', '909', '918', '972']\n",
      "NDB_418 -> ['125', '177', '2', '205', '225', '243', '26', '308', '345', '385', '403', '418', '456', '524', '545', '565', '581', '618', '659', '694', '71', '715', '769', '796', '819', '82', '858', '908', '937', '971']\n",
      "NDB_421 -> ['123', '172', '198', '221', '234', '311', '32', '328', '347', '388', '402', '436', '441', '484', '525', '573', '603', '62', '637', '659', '692', '711', '746', '777', '820', '859', '89', '892', '948', '956', '982']\n",
      "NDB_427 -> ['146', '183', '211', '224', '25', '257', '291', '324', '347', '361', '426', '444', '482', '527', '548', '586', '61', '634', '683', '707', '751', '762', '78', '795', '836', '871', '901', '928', '96', '971']\n",
      "NDB_428 -> ['12', '143', '182', '204', '247', '274', '307', '357', '377', '425', '450', '491', '531', '549', '564', '607', '634', '663', '700', '705', '71', '734', '768', '805', '844', '847', '85', '881', '913', '948', '950']\n",
      "NDB_431 -> ['111', '171', '198', '217', '233', '286', '322', '341', '36', '385', '417', '462', '494', '516', '580', '597', '611', '638', '65', '687', '740', '764', '80', '802', '825', '867', '881', '898', '921', '967']\n",
      "NDB_433 -> ['118', '148', '188', '239', '260', '283', '327', '345', '35', '390', '415', '459', '485', '494', '538', '546', '577', '639', '652', '66', '685', '712', '788', '81', '812', '841', '861', '879', '913', '928', '982']\n",
      "NDB_436 -> ['109', '12', '175', '194', '219', '260', '292', '315', '335', '352', '410', '426', '438', '476', '48', '504', '576', '603', '614', '650', '668', '706', '733', '765', '793', '818', '855', '88', '911', '959']\n",
      "NDB_439 -> ['136', '162', '17', '200', '225', '255', '296', '30', '330', '356', '422', '437', '458', '527', '540', '566', '597', '636', '64', '660', '678', '709', '747', '791', '833', '87', '878', '889', '929', '955']\n",
      "NDB_443 -> ['115', '187', '255', '271', '291', '33', '333', '371', '408', '440', '478', '500', '517', '568', '584', '617', '63', '653', '680', '729', '743', '780', '789', '80', '812', '863', '907', '919', '948', '951', '97']\n",
      "NDB_445 -> ['133', '170', '190', '218', '259', '275', '314', '355', '374', '398', '434', '46', '469', '512', '533', '576', '608', '629', '643', '697', '70', '719', '764', '785', '822', '85', '863', '883', '934', '955', '985']\n",
      "NDB_446 -> ['126', '179', '201', '256', '265', '29', '297', '337', '366', '408', '41', '446', '480', '505', '555', '582', '60', '612', '667', '684', '723', '735', '772', '795', '825', '83', '858', '889', '913', '927', '966']\n",
      "NDB_448 -> ['106', '141', '164', '189', '219', '262', '27', '285', '327', '339', '381', '416', '425', '457', '493', '545', '574', '602', '624', '63', '646', '678', '735', '778', '803', '821', '862', '902', '920', '945', '964']\n",
      "NDB_452 -> ['10', '130', '148', '161', '207', '232', '266', '310', '336', '360', '406', '427', '471', '48', '488', '491', '510', '556', '606', '615', '662', '669', '739', '780', '836', '86', '878', '890', '912', '925', '954']\n",
      "NDB_455 -> ['103', '117', '186', '206', '213', '239', '305', '372', '387', '405', '437', '462', '495', '53', '544', '565', '595', '630', '655', '672', '699', '705', '727', '786', '810', '853', '883', '89', '9', '961', '988']\n",
      "NDB_458 -> ['122', '190', '208', '229', '252', '283', '31', '317', '370', '387', '410', '450', '475', '528', '566', '57', '6', '605', '640', '683', '715', '745', '770', '807', '827', '859', '890', '927', '93', '964']\n",
      "NDB_460 -> ['149', '160', '204', '222', '246', '27', '295', '338', '384', '401', '460', '49', '505', '540', '548', '572', '609', '613', '665', '694', '740', '790', '804', '827', '86', '864', '880', '906', '943', '977', '987']\n",
      "NDB_461 -> ['112', '134', '162', '176', '218', '234', '25', '277', '312', '340', '391', '410', '461', '474', '52', '526', '571', '602', '626', '652', '681', '732', '752', '773', '801', '825', '870', '886', '930', '969', '986']\n",
      "NDB_464 -> ['140', '184', '206', '241', '272', '308', '358', '375', '424', '43', '438', '485', '526', '564', '582', '61', '615', '666', '698', '713', '764', '77', '784', '829', '842', '873', '907', '91', '918', '973']\n",
      "NDB_466 -> ['100', '115', '146', '170', '193', '226', '289', '32', '329', '349', '384', '395', '470', '518', '52', '535', '556', '585', '627', '656', '682', '731', '741', '765', '798', '814', '844', '851', '894', '927', '960']\n",
      "NDB_467 -> ['131', '157', '184', '20', '244', '262', '280', '332', '336', '371', '399', '432', '45', '463', '489', '506', '543', '554', '585', '62', '640', '684', '727', '791', '824', '850', '887', '931', '950', '99']\n",
      "NDB_470 -> ['103', '145', '16', '178', '203', '248', '266', '309', '353', '375', '400', '434', '439', '479', '528', '570', '588', '613', '663', '674', '68', '690', '709', '731', '772', '79', '818', '866', '893', '949', '982']\n",
      "NDB_472 -> ['140', '163', '195', '215', '231', '289', '328', '353', '37', '373', '409', '440', '489', '509', '534', '577', '604', '624', '65', '650', '691', '715', '751', '786', '823', '86', '860', '915', '923', '957', '979']\n",
      "NDB_473 -> ['108', '14', '185', '210', '213', '232', '29', '306', '335', '348', '411', '435', '450', '486', '519', '538', '558', '56', '597', '617', '664', '679', '737', '766', '824', '84', '849', '897', '932', '977']\n",
      "NDB_476 -> ['103', '123', '150', '177', '214', '23', '247', '295', '312', '359', '374', '428', '455', '513', '539', '553', '589', '625', '64', '646', '670', '706', '748', '762', '808', '826', '869', '887', '942', '973']\n",
      "NDB_479 -> ['121', '149', '155', '168', '228', '312', '329', '365', '379', '404', '462', '504', '51', '541', '559', '593', '617', '654', '676', '728', '750', '761', '795', '8', '838', '872', '903', '91', '940', '972']\n",
      "NDB_481 -> ['143', '15', '164', '208', '226', '242', '281', '302', '336', '365', '42', '426', '460', '498', '544', '555', '581', '639', '671', '716', '746', '77', '775', '802', '815', '828', '869', '893', '923', '97', '984']\n",
      "NDB_482 -> ['117', '137', '165', '191', '231', '250', '287', '334', '35', '367', '382', '420', '438', '461', '5', '519', '536', '570', '609', '627', '651', '671', '702', '718', '768', '822', '875', '904', '938', '965']\n",
      "NDB_487 -> ['104', '141', '166', '194', '218', '23', '251', '279', '298', '370', '417', '451', '473', '499', '511', '575', '600', '628', '651', '669', '703', '72', '728', '774', '798', '832', '879', '907', '922', '949', '953']\n",
      "NDB_491 -> ['119', '156', '202', '234', '263', '278', '329', '369', '425', '44', '448', '479', '5', '509', '555', '58', '594', '636', '665', '668', '703', '704', '719', '759', '800', '812', '853', '895', '947', '981', '99']\n",
      "NDB_493 -> ['1', '122', '182', '207', '223', '240', '282', '305', '351', '36', '382', '407', '431', '453', '467', '512', '563', '584', '641', '701', '72', '736', '761', '808', '82', '828', '850', '864', '892', '932', '959']\n",
      "NDB_494 -> ['129', '163', '18', '201', '214', '246', '298', '310', '361', '375', '392', '42', '431', '454', '486', '507', '537', '546', '59', '596', '636', '680', '722', '785', '817', '844', '862', '90', '921', '962', '988']\n",
      "NDB_505 -> ['114', '134', '173', '199', '221', '24', '257', '290', '327', '360', '380', '407', '449', '487', '492', '523', '581', '60', '607', '643', '657', '693', '704', '743', '792', '834', '851', '903', '929', '962']\n",
      "NDB_506 -> ['108', '135', '159', '16', '197', '212', '240', '299', '325', '364', '394', '432', '458', '475', '501', '517', '548', '58', '582', '637', '695', '722', '73', '733', '783', '823', '847', '910', '930', '965']\n",
      "NDB_508 -> ['121', '14', '151', '160', '209', '224', '249', '308', '332', '357', '39', '421', '453', '475', '529', '553', '588', '631', '660', '668', '717', '752', '758', '787', '80', '841', '851', '882', '912', '945', '980']\n",
      "NDB_511 -> ['124', '165', '2', '204', '242', '265', '296', '344', '358', '393', '412', '441', '453', '47', '521', '557', '590', '632', '662', '679', '729', '75', '783', '807', '821', '866', '883', '916', '946', '970', '99']\n",
      "NDB_514 -> ['120', '175', '191', '216', '257', '27', '313', '325', '374', '389', '395', '435', '467', '514', '531', '54', '571', '601', '638', '654', '688', '708', '747', '770', '841', '848', '89', '899', '933', '976', '990']\n",
      "NDB_517 -> ['127', '180', '243', '266', '282', '325', '367', '41', '413', '430', '441', '476', '502', '533', '553', '59', '610', '621', '657', '687', '7', '721', '744', '773', '838', '871', '897', '931', '95', '964', '987']\n",
      "NDB_526 -> ['105', '138', '159', '203', '246', '25', '261', '303', '358', '373', '421', '446', '487', '513', '55', '573', '600', '611', '635', '699', '719', '750', '779', '809', '811', '84', '855', '902', '936', '958', '990']\n",
      "NDB_529 -> ['119', '168', '19', '191', '219', '264', '297', '321', '364', '38', '386', '419', '437', '472', '497', '539', '6', '606', '621', '659', '698', '738', '787', '817', '881', '886', '934', '94', '958', '984']\n",
      "NDB_532 -> ['124', '157', '196', '255', '268', '279', '31', '315', '360', '423', '44', '447', '482', '503', '530', '547', '589', '638', '697', '721', '73', '731', '790', '810', '842', '875', '900', '917', '94', '974']\n",
      "NDB_533 -> ['1', '116', '179', '239', '274', '290', '306', '338', '356', '38', '421', '445', '478', '508', '550', '591', '620', '644', '675', '720', '74', '748', '765', '800', '821', '848', '901', '935', '95', '979', '991']\n",
      "NDB_536 -> ['112', '130', '14', '179', '208', '235', '264', '295', '334', '353', '429', '454', '494', '535', '569', '583', '62', '634', '657', '677', '708', '725', '761', '799', '830', '868', '88', '908', '941', '974', '985']\n",
      "NDB_538 -> ['113', '139', '185', '237', '272', '275', '287', '344', '383', '418', '464', '495', '50', '543', '559', '606', '619', '648', '692', '73', '740', '782', '799', '816', '848', '879', '895', '942', '960', '98']\n",
      "NDB_539 -> ['116', '136', '174', '19', '193', '223', '286', '320', '335', '34', '373', '415', '432', '469', '511', '525', '552', '594', '623', '63', '658', '696', '726', '767', '805', '834', '858', '899', '941', '972', '986']\n",
      "NDB_544 -> ['106', '145', '165', '196', '221', '258', '280', '318', '350', '376', '424', '459', '48', '495', '514', '569', '598', '625', '661', '67', '676', '738', '766', '802', '811', '843', '876', '901', '922', '966']\n",
      "NDB_547 -> ['101', '139', '174', '196', '228', '261', '281', '32', '324', '343', '368', '408', '430', '466', '481', '49', '512', '542', '575', '607', '624', '647', '677', '724', '791', '818', '873', '904', '921', '947', '970']\n",
      "NDB_551 -> ['101', '125', '180', '202', '231', '267', '28', '290', '321', '354', '376', '405', '428', '473', '484', '520', '558', '576', '61', '642', '685', '722', '745', '76', '790', '815', '870', '912', '946', '950', '983']\n",
      "NDB_554 -> ['134', '15', '152', '186', '214', '238', '26', '282', '330', '343', '388', '409', '445', '46', '465', '506', '559', '594', '631', '700', '718', '749', '766', '803', '835', '855', '906', '919', '94', '967', '989']\n",
      "NDB_556 -> ['10', '114', '170', '192', '225', '291', '322', '343', '359', '37', '424', '464', '508', '532', '562', '58', '610', '616', '651', '688', '730', '742', '781', '794', '839', '868', '902', '93', '939', '956']\n",
      "NDB_557 -> ['117', '148', '154', '187', '217', '259', '278', '330', '362', '379', '4', '411', '466', '498', '534', '549', '55', '598', '623', '646', '689', '707', '744', '758', '782', '814', '859', '885', '911', '930', '963']\n",
      "NDB_562 -> ['111', '147', '180', '210', '212', '253', '292', '319', '366', '381', '42', '423', '464', '487', '497', '536', '556', '57', '586', '641', '690', '703', '732', '784', '833', '876', '906', '92', '935', '957']\n",
      "NDB_563 -> ['138', '161', '205', '240', '263', '285', '320', '350', '362', '40', '404', '417', '470', '496', '537', '557', '586', '642', '698', '7', '711', '747', '789', '831', '85', '860', '905', '939', '974', '98']\n",
      "NDB_568 -> ['107', '144', '155', '172', '18', '216', '258', '279', '319', '337', '386', '398', '414', '470', '513', '543', '549', '604', '626', '647', '68', '701', '712', '742', '775', '829', '853', '878', '910', '945', '968']\n",
      "NDB_571 -> ['109', '135', '169', '192', '222', '230', '301', '313', '339', '388', '406', '420', '444', '483', '503', '568', '60', '600', '625', '660', '681', '734', '769', '8', '806', '816', '829', '852', '894', '923', '981']\n",
      "NDB_577 -> ['10', '120', '184', '245', '265', '286', '301', '348', '382', '416', '455', '465', '527', '53', '546', '572', '608', '622', '647', '676', '690', '717', '779', '796', '823', '854', '88', '914', '926', '953', '984']\n",
      "NDB_578 -> ['102', '146', '169', '195', '229', '278', '321', '33', '366', '399', '456', '481', '521', '529', '54', '547', '587', '610', '611', '637', '674', '710', '726', '776', '831', '845', '861', '896', '943', '953']\n",
      "NDB_581 -> ['112', '137', '15', '183', '200', '237', '267', '309', '370', '415', '442', '45', '481', '504', '530', '557', '596', '616', '665', '691', '705', '736', '758', '778', '813', '83', '876', '884', '910', '954', '991']\n",
      "NDB_587 -> ['1', '114', '13', '186', '233', '262', '281', '320', '348', '377', '396', '412', '439', '482', '518', '536', '54', '566', '577', '619', '645', '695', '723', '769', '837', '856', '90', '934', '954', '968']\n",
      "NDB_589 -> ['133', '162', '201', '22', '241', '264', '293', '313', '363', '397', '436', '440', '47', '479', '502', '511', '550', '592', '642', '683', '69', '712', '741', '778', '84', '843', '850', '903', '944', '961']\n",
      "NDB_592 -> ['105', '127', '183', '2', '236', '269', '300', '328', '34', '340', '350', '396', '433', '452', '497', '540', '550', '588', '630', '643', '680', '735', '759', '76', '808', '816', '877', '888', '925', '965']\n",
      "NDB_601 -> ['113', '149', '181', '199', '256', '274', '284', '319', '34', '347', '378', '420', '471', '520', '534', '551', '596', '633', '688', '725', '741', '789', '82', '826', '845', '867', '897', '9', '949', '955', '978']\n",
      "NDB_604 -> ['115', '147', '17', '174', '197', '220', '235', '288', '302', '352', '391', '419', '452', '488', '492', '509', '573', '601', '622', '649', '66', '694', '734', '785', '832', '845', '882', '884', '920', '966']\n",
      "NDB_608 -> ['101', '132', '164', '211', '244', '268', '29', '3', '307', '344', '386', '407', '451', '528', '541', '565', '591', '631', '653', '66', '696', '713', '730', '777', '804', '826', '860', '888', '914', '944', '963']\n",
      "NDB_611 -> ['109', '11', '130', '147', '188', '215', '236', '285', '322', '365', '402', '448', '478', '52', '520', '533', '560', '587', '628', '655', '684', '737', '783', '805', '817', '846', '864', '904', '917', '944', '973']\n",
      "NDB_614 -> ['132', '181', '202', '238', '271', '299', '342', '387', '398', '4', '43', '472', '492', '508', '561', '580', '614', '667', '673', '71', '728', '773', '799', '831', '849', '880', '891', '90', '915', '925', '970']\n",
      "NDB_616 -> ['100', '113', '145', '18', '182', '252', '273', '296', '314', '362', '403', '43', '442', '480', '500', '532', '560', '593', '621', '649', '699', '714', '752', '788', '803', '815', '856', '914', '933', '952', '981']\n",
      "NDB_619 -> ['142', '190', '205', '212', '23', '245', '305', '337', '391', '395', '406', '468', '493', '515', '547', '59', '609', '644', '661', '692', '709', '743', '76', '781', '81', '835', '849', '909', '936', '952']\n",
      "NDB_622 -> ['129', '157', '19', '203', '237', '273', '300', '372', '394', '435', '449', '474', '5', '502', '516', '572', '598', '644', '666', '673', '70', '700', '739', '776', '798', '819', '869', '892', '928', '93', '951']\n",
      "NDB_625 -> ['100', '121', '13', '163', '206', '236', '270', '293', '35', '364', '381', '414', '428', '447', '476', '510', '567', '584', '612', '661', '673', '710', '725', '782', '832', '852', '87', '898', '937', '975']\n",
      "NDB_628 -> ['107', '140', '158', '198', '232', '250', '251', '293', '332', '368', '423', '442', '458', '51', '521', '568', '599', '629', '663', '672', '704', '716', '74', '762', '804', '837', '870', '911', '941', '978']\n",
      "NDB_629 -> ['118', '143', '171', '192', '21', '220', '251', '304', '316', '356', '384', '418', '457', '484', '510', '537', '605', '626', '645', '67', '674', '736', '787', '824', '854', '877', '885', '933', '957', '96']\n",
      "True\n"
     ]
    }
   ],
   "source": [
    "reload(check)\n",
    "readout_reg_dic, readout2reg_check = check._check_readout_to_region(reg_readout_dic, full_records, \n",
    "                                                                    check_readout_dict, primer_len=primer_len, \n",
    "                                                                    target_len=target_len,\n",
    "                                                                    probe_readout_num=num_readout_per_probe)\n",
    "\n",
    "print(readout2reg_check)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 4.1.5 check bewteen probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [],
   "source": [
    "# construct internal map\n",
    "reload(check)\n",
    "#int_map = check._construct_internal_map(full_records, library_folder, save=True) # file is big; not necessary to save\n",
    "\n",
    "int_map = check._construct_internal_map(full_records, library_folder, save=False) "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "True\n"
     ]
    }
   ],
   "source": [
    "# internal check probes\n",
    "readout_count_dic, readout_count_check = check._check_readout_in_probes(readout_reg_dic, reg_size_dic, int_map, readout_dict)\n",
    "print(readout_count_check)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- Sequence:loc_4:105899094-105952080_gene_251_pb_112_pos_30682_strand_+_readouts_[NDB_487_c,NDB_628_c,NDB_629_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n",
      "--- Sequence:loc_4:105899094-105952080_gene_251_pb_113_pos_30738_strand_-_readouts_[NDB_487_c,NDB_628_c,NDB_629_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:56, dumped\n",
      "--- Sequence:loc_4:114589240-114676242_gene_252_pb_154_pos_54592_strand_+_readouts_[NDB_458_c,NDB_616_c,NDB_350_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:59, dumped\n",
      "--- Sequence:loc_5:3994952-4090244_gene_275_pb_58_pos_57879_strand_+_readouts_[NDB_376_c,NDB_445_c,NDB_538_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:77, dumped\n",
      "--- Sequence:loc_5:3994952-4090244_gene_275_pb_59_pos_58013_strand_-_readouts_[NDB_376_c,NDB_445_c,NDB_538_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:97, dumped\n",
      "--- Sequence:loc_5:3994952-4090244_gene_275_pb_60_pos_58151_strand_-_readouts_[NDB_376_c,NDB_445_c,NDB_538_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:81, dumped\n",
      "--- Sequence:loc_5:3994952-4090244_gene_275_pb_61_pos_58633_strand_-_readouts_[NDB_376_c,NDB_445_c,NDB_538_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:93, dumped\n",
      "--- Sequence:loc_5:3994952-4090244_gene_275_pb_63_pos_58899_strand_-_readouts_[NDB_376_c,NDB_445_c,NDB_538_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:90, dumped\n",
      "--- Sequence:loc_5:3994952-4090244_gene_275_pb_64_pos_59071_strand_-_readouts_[NDB_376_c,NDB_445_c,NDB_538_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:83, dumped\n",
      "--- Sequence:loc_5:3994952-4090244_gene_275_pb_65_pos_59119_strand_-_readouts_[NDB_376_c,NDB_445_c,NDB_538_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:92, dumped\n",
      "--- Sequence:loc_5:3994952-4090244_gene_275_pb_66_pos_59236_strand_+_readouts_[NDB_376_c,NDB_445_c,NDB_538_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:82, dumped\n",
      "--- Sequence:loc_5:3994952-4090244_gene_275_pb_67_pos_59338_strand_-_readouts_[NDB_376_c,NDB_445_c,NDB_538_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:110, dumped\n",
      "--- Sequence:loc_5:3994952-4090244_gene_275_pb_69_pos_59451_strand_-_readouts_[NDB_376_c,NDB_445_c,NDB_538_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:103, dumped\n",
      "--- Sequence:loc_5:3994952-4090244_gene_275_pb_70_pos_59536_strand_-_readouts_[NDB_376_c,NDB_445_c,NDB_538_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:94, dumped\n",
      "--- Sequence:loc_5:3994952-4090244_gene_275_pb_105_pos_62510_strand_-_readouts_[NDB_376_c,NDB_445_c,NDB_538_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:85, dumped\n",
      "--- Sequence:loc_5:3994952-4090244_gene_275_pb_106_pos_62579_strand_-_readouts_[NDB_376_c,NDB_445_c,NDB_538_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:92, dumped\n",
      "--- Sequence:loc_5:3994952-4090244_gene_275_pb_214_pos_71922_strand_-_readouts_[NDB_376_c,NDB_445_c,NDB_538_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_5:4937946-5030393_gene_277_pb_28_pos_49892_strand_+_readouts_[NDB_461_c,NDB_401_c,NDB_359_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_5:21457453-21528145_gene_281_pb_249_pos_57918_strand_-_readouts_[NDB_587_c,NDB_481_c,NDB_547_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:66, dumped\n",
      "--- Sequence:loc_5:21457453-21528145_gene_281_pb_250_pos_58001_strand_-_readouts_[NDB_587_c,NDB_481_c,NDB_547_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n",
      "--- Sequence:loc_5:21457453-21528145_gene_281_pb_251_pos_58064_strand_-_readouts_[NDB_587_c,NDB_481_c,NDB_547_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:62, dumped\n",
      "--- Sequence:loc_5:21457453-21528145_gene_281_pb_253_pos_58230_strand_-_readouts_[NDB_587_c,NDB_481_c,NDB_547_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:62, dumped\n",
      "--- Sequence:loc_5:72807519-72901992_gene_305_pb_245_pos_66677_strand_+_readouts_[NDB_455_c,NDB_619_c,NDB_493_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:427, dumped\n",
      "--- Sequence:loc_5:104291330-104355424_gene_312_pb_318_pos_57197_strand_+_readouts_[NDB_461_c,NDB_476_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:121, dumped\n",
      "--- Sequence:loc_5:104291330-104355424_gene_312_pb_319_pos_57264_strand_+_readouts_[NDB_461_c,NDB_476_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:111, dumped\n",
      "--- Sequence:loc_5:104291330-104355424_gene_312_pb_320_pos_57313_strand_+_readouts_[NDB_461_c,NDB_476_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:136, dumped\n",
      "--- Sequence:loc_5:104291330-104355424_gene_312_pb_321_pos_57363_strand_+_readouts_[NDB_461_c,NDB_476_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:74, dumped\n",
      "--- Sequence:loc_5:104291330-104355424_gene_312_pb_322_pos_57454_strand_+_readouts_[NDB_461_c,NDB_476_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:158, dumped\n",
      "--- Sequence:loc_5:104291330-104355424_gene_312_pb_323_pos_57517_strand_+_readouts_[NDB_461_c,NDB_476_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:168, dumped\n",
      "--- Sequence:loc_5:104291330-104355424_gene_312_pb_324_pos_57561_strand_+_readouts_[NDB_461_c,NDB_476_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:80, dumped\n",
      "--- Sequence:loc_5:104291330-104355424_gene_312_pb_325_pos_57637_strand_-_readouts_[NDB_461_c,NDB_476_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:122, dumped\n",
      "--- Sequence:loc_5:104291330-104355424_gene_312_pb_326_pos_57684_strand_-_readouts_[NDB_461_c,NDB_476_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:122, dumped\n",
      "--- Sequence:loc_5:104291330-104355424_gene_312_pb_327_pos_57730_strand_-_readouts_[NDB_461_c,NDB_476_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:79, dumped\n",
      "--- Sequence:loc_5:106763128-106836267_gene_313_pb_132_pos_49024_strand_+_readouts_[NDB_571_c,NDB_514_c,NDB_589_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:63, dumped\n",
      "--- Sequence:loc_5:106763128-106836267_gene_313_pb_133_pos_49078_strand_-_readouts_[NDB_571_c,NDB_514_c,NDB_589_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:71, dumped\n",
      "--- Sequence:loc_5:106763128-106836267_gene_313_pb_134_pos_49122_strand_-_readouts_[NDB_571_c,NDB_514_c,NDB_589_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:64, dumped\n",
      "--- Sequence:loc_5:106763128-106836267_gene_313_pb_135_pos_49172_strand_+_readouts_[NDB_571_c,NDB_514_c,NDB_589_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:55, dumped\n",
      "--- Sequence:loc_6:27875819-27995930_gene_343_pb_193_pos_83178_strand_+_readouts_[NDB_554_c,NDB_556_c,NDB_547_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_6:34471419-34525893_gene_347_pb_80_pos_30263_strand_+_readouts_[NDB_427_c,NDB_421_c,NDB_601_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:62, dumped\n",
      "--- Sequence:loc_6:41067178-41147415_gene_349_pb_120_pos_51888_strand_+_readouts_[NDB_466_c,NDB_386_c,NDB_395_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:59, dumped\n",
      "--- Sequence:loc_6:41067178-41147415_gene_349_pb_153_pos_54457_strand_+_readouts_[NDB_466_c,NDB_386_c,NDB_395_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:55, dumped\n",
      "--- Sequence:loc_6:41067178-41147415_gene_349_pb_211_pos_60597_strand_+_readouts_[NDB_466_c,NDB_386_c,NDB_395_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:58, dumped\n",
      "--- Sequence:loc_6:41067178-41147415_gene_349_pb_246_pos_63209_strand_+_readouts_[NDB_466_c,NDB_386_c,NDB_395_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_6:83174995-83226237_gene_364_pb_287_pos_40901_strand_+_readouts_[NDB_529_c,NDB_506_c,NDB_625_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:224, dumped\n",
      "--- Sequence:loc_6:83174995-83226237_gene_364_pb_288_pos_40946_strand_-_readouts_[NDB_529_c,NDB_506_c,NDB_625_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:190, dumped\n",
      "--- Sequence:loc_6:83174995-83226237_gene_364_pb_289_pos_41013_strand_+_readouts_[NDB_529_c,NDB_506_c,NDB_625_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:134, dumped\n",
      "--- Sequence:loc_6:83174995-83226237_gene_364_pb_290_pos_41374_strand_-_readouts_[NDB_529_c,NDB_506_c,NDB_625_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:168, dumped\n",
      "--- Sequence:loc_6:83174995-83226237_gene_364_pb_291_pos_41445_strand_+_readouts_[NDB_529_c,NDB_506_c,NDB_625_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:179, dumped\n",
      "--- Sequence:loc_6:83174995-83226237_gene_364_pb_292_pos_41749_strand_+_readouts_[NDB_529_c,NDB_506_c,NDB_625_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:129, dumped\n",
      "--- Sequence:loc_6:83174995-83226237_gene_364_pb_293_pos_41909_strand_-_readouts_[NDB_529_c,NDB_506_c,NDB_625_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:123, dumped\n",
      "--- Sequence:loc_6:83174995-83226237_gene_364_pb_294_pos_42000_strand_+_readouts_[NDB_529_c,NDB_506_c,NDB_625_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:245, dumped\n",
      "--- Sequence:loc_6:83174995-83226237_gene_364_pb_295_pos_42114_strand_-_readouts_[NDB_529_c,NDB_506_c,NDB_625_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:195, dumped\n",
      "--- Sequence:loc_6:83174995-83226237_gene_364_pb_296_pos_42556_strand_-_readouts_[NDB_529_c,NDB_506_c,NDB_625_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:226, dumped\n",
      "--- Sequence:loc_6:83174995-83226237_gene_364_pb_297_pos_42640_strand_+_readouts_[NDB_529_c,NDB_506_c,NDB_625_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:110, dumped\n",
      "--- Sequence:loc_6:83174995-83226237_gene_364_pb_298_pos_42773_strand_-_readouts_[NDB_529_c,NDB_506_c,NDB_625_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:243, dumped\n",
      "--- Sequence:loc_6:83174995-83226237_gene_364_pb_299_pos_42899_strand_+_readouts_[NDB_529_c,NDB_506_c,NDB_625_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:204, dumped\n",
      "--- Sequence:loc_6:83174995-83226237_gene_364_pb_300_pos_43127_strand_-_readouts_[NDB_529_c,NDB_506_c,NDB_625_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:194, dumped\n",
      "--- Sequence:loc_6:83174995-83226237_gene_364_pb_301_pos_43182_strand_+_readouts_[NDB_529_c,NDB_506_c,NDB_625_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:198, dumped\n",
      "--- Sequence:loc_6:83174995-83226237_gene_364_pb_302_pos_43249_strand_+_readouts_[NDB_529_c,NDB_506_c,NDB_625_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:82, dumped\n",
      "--- Sequence:loc_6:83174995-83226237_gene_364_pb_303_pos_43293_strand_+_readouts_[NDB_529_c,NDB_506_c,NDB_625_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:108, dumped\n",
      "--- Sequence:loc_6:83174995-83226237_gene_364_pb_304_pos_43359_strand_+_readouts_[NDB_529_c,NDB_506_c,NDB_625_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:223, dumped\n",
      "--- Sequence:loc_6:83174995-83226237_gene_364_pb_305_pos_43580_strand_+_readouts_[NDB_529_c,NDB_506_c,NDB_625_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:251, dumped\n",
      "--- Sequence:loc_6:83174995-83226237_gene_364_pb_306_pos_43642_strand_+_readouts_[NDB_529_c,NDB_506_c,NDB_625_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:191, dumped\n",
      "--- Sequence:loc_6:83174995-83226237_gene_364_pb_307_pos_43728_strand_+_readouts_[NDB_529_c,NDB_506_c,NDB_625_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:206, dumped\n",
      "--- Sequence:loc_6:83174995-83226237_gene_364_pb_308_pos_43799_strand_-_readouts_[NDB_529_c,NDB_506_c,NDB_625_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:121, dumped\n",
      "--- Sequence:loc_6:83174995-83226237_gene_364_pb_309_pos_43865_strand_-_readouts_[NDB_529_c,NDB_506_c,NDB_625_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:244, dumped\n",
      "--- Sequence:loc_6:83174995-83226237_gene_364_pb_310_pos_43924_strand_-_readouts_[NDB_529_c,NDB_506_c,NDB_625_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:62, dumped\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- Sequence:loc_6:128391401-128443169_gene_382_pb_377_pos_49338_strand_-_readouts_[NDB_482_c,NDB_577_c,NDB_493_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:81, dumped\n",
      "--- Sequence:loc_6:128391401-128443169_gene_382_pb_378_pos_49443_strand_+_readouts_[NDB_482_c,NDB_577_c,NDB_493_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:72, dumped\n",
      "--- Sequence:loc_6:128391401-128443169_gene_382_pb_379_pos_49583_strand_-_readouts_[NDB_482_c,NDB_577_c,NDB_493_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:108, dumped\n",
      "--- Sequence:loc_6:128391401-128443169_gene_382_pb_380_pos_49632_strand_+_readouts_[NDB_482_c,NDB_577_c,NDB_493_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:86, dumped\n",
      "--- Sequence:loc_6:128391401-128443169_gene_382_pb_381_pos_49842_strand_+_readouts_[NDB_482_c,NDB_577_c,NDB_493_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:76, dumped\n",
      "--- Sequence:loc_6:128391401-128443169_gene_382_pb_382_pos_49887_strand_+_readouts_[NDB_482_c,NDB_577_c,NDB_493_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:97, dumped\n",
      "--- Sequence:loc_6:128391401-128443169_gene_382_pb_383_pos_49931_strand_-_readouts_[NDB_482_c,NDB_577_c,NDB_493_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:85, dumped\n",
      "--- Sequence:loc_6:128391401-128443169_gene_382_pb_384_pos_49989_strand_+_readouts_[NDB_482_c,NDB_577_c,NDB_493_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:68, dumped\n",
      "--- Sequence:loc_6:128391401-128443169_gene_382_pb_385_pos_50054_strand_+_readouts_[NDB_482_c,NDB_577_c,NDB_493_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:121, dumped\n",
      "--- Sequence:loc_6:128391401-128443169_gene_382_pb_386_pos_50113_strand_-_readouts_[NDB_482_c,NDB_577_c,NDB_493_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:103, dumped\n",
      "--- Sequence:loc_6:144585170-144652901_gene_387_pb_321_pos_61719_strand_+_readouts_[NDB_458_c,NDB_455_c,NDB_614_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:56, dumped\n",
      "--- Sequence:loc_6:144585170-144652901_gene_387_pb_324_pos_61883_strand_-_readouts_[NDB_458_c,NDB_455_c,NDB_614_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_6:145723149-145813383_gene_390_pb_36_pos_46295_strand_+_readouts_[NDB_332_c,NDB_433_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:104, dumped\n",
      "--- Sequence:loc_6:145723149-145813383_gene_390_pb_37_pos_46380_strand_+_readouts_[NDB_332_c,NDB_433_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:106, dumped\n",
      "--- Sequence:loc_6:145723149-145813383_gene_390_pb_38_pos_46442_strand_-_readouts_[NDB_332_c,NDB_433_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:108, dumped\n",
      "--- Sequence:loc_6:145723149-145813383_gene_390_pb_39_pos_46589_strand_+_readouts_[NDB_332_c,NDB_433_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:70, dumped\n",
      "--- Sequence:loc_6:145723149-145813383_gene_390_pb_40_pos_46652_strand_+_readouts_[NDB_332_c,NDB_433_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:70, dumped\n",
      "--- Sequence:loc_6:145723149-145813383_gene_390_pb_41_pos_46710_strand_+_readouts_[NDB_332_c,NDB_433_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:74, dumped\n",
      "--- Sequence:loc_6:145723149-145813383_gene_390_pb_193_pos_60517_strand_+_readouts_[NDB_332_c,NDB_433_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:94, dumped\n",
      "--- Sequence:loc_6:145723149-145813383_gene_390_pb_194_pos_60587_strand_+_readouts_[NDB_332_c,NDB_433_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:214, dumped\n",
      "--- Sequence:loc_6:145723149-145813383_gene_390_pb_195_pos_60656_strand_+_readouts_[NDB_332_c,NDB_433_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:108, dumped\n",
      "--- Sequence:loc_6:145723149-145813383_gene_390_pb_196_pos_60700_strand_-_readouts_[NDB_332_c,NDB_433_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:82, dumped\n",
      "--- Sequence:loc_6:145723149-145813383_gene_390_pb_197_pos_61103_strand_+_readouts_[NDB_332_c,NDB_433_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:150, dumped\n",
      "--- Sequence:loc_6:145723149-145813383_gene_390_pb_198_pos_61515_strand_+_readouts_[NDB_332_c,NDB_433_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:77, dumped\n",
      "--- Sequence:loc_6:145723149-145813383_gene_390_pb_199_pos_61586_strand_-_readouts_[NDB_332_c,NDB_433_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:289, dumped\n",
      "--- Sequence:loc_6:145723149-145813383_gene_390_pb_200_pos_61719_strand_+_readouts_[NDB_332_c,NDB_433_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:299, dumped\n",
      "--- Sequence:loc_6:145723149-145813383_gene_390_pb_201_pos_62101_strand_-_readouts_[NDB_332_c,NDB_433_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:184, dumped\n",
      "--- Sequence:loc_6:145723149-145813383_gene_390_pb_202_pos_62423_strand_-_readouts_[NDB_332_c,NDB_433_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:101, dumped\n",
      "--- Sequence:loc_6:145723149-145813383_gene_390_pb_203_pos_62498_strand_-_readouts_[NDB_332_c,NDB_433_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:275, dumped\n",
      "--- Sequence:loc_6:145723149-145813383_gene_390_pb_204_pos_62623_strand_+_readouts_[NDB_332_c,NDB_433_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:113, dumped\n",
      "--- Sequence:loc_6:145723149-145813383_gene_390_pb_205_pos_62740_strand_-_readouts_[NDB_332_c,NDB_433_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:88, dumped\n",
      "--- Sequence:loc_6:145723149-145813383_gene_390_pb_206_pos_62864_strand_-_readouts_[NDB_332_c,NDB_433_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:283, dumped\n",
      "--- Sequence:loc_6:145723149-145813383_gene_390_pb_207_pos_62964_strand_-_readouts_[NDB_332_c,NDB_433_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:233, dumped\n",
      "--- Sequence:loc_6:145723149-145813383_gene_390_pb_208_pos_63008_strand_-_readouts_[NDB_332_c,NDB_433_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:223, dumped\n",
      "--- Sequence:loc_6:145723149-145813383_gene_390_pb_209_pos_63085_strand_-_readouts_[NDB_332_c,NDB_433_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:72, dumped\n",
      "--- Sequence:loc_6:145723149-145813383_gene_390_pb_210_pos_63160_strand_+_readouts_[NDB_332_c,NDB_433_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:111, dumped\n",
      "--- Sequence:loc_6:145723149-145813383_gene_390_pb_211_pos_63232_strand_-_readouts_[NDB_332_c,NDB_433_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:136, dumped\n",
      "--- Sequence:loc_6:145723149-145813383_gene_390_pb_212_pos_63307_strand_-_readouts_[NDB_332_c,NDB_433_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:220, dumped\n",
      "--- Sequence:loc_6:145723149-145813383_gene_390_pb_213_pos_63411_strand_-_readouts_[NDB_332_c,NDB_433_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:214, dumped\n",
      "--- Sequence:loc_6:145723149-145813383_gene_390_pb_214_pos_63510_strand_+_readouts_[NDB_332_c,NDB_433_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:177, dumped\n",
      "--- Sequence:loc_6:145723149-145813383_gene_390_pb_215_pos_63560_strand_+_readouts_[NDB_332_c,NDB_433_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:298, dumped\n",
      "--- Sequence:loc_6:145723149-145813383_gene_390_pb_216_pos_63624_strand_-_readouts_[NDB_332_c,NDB_433_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:308, dumped\n",
      "--- Sequence:loc_6:145723149-145813383_gene_390_pb_217_pos_63673_strand_+_readouts_[NDB_332_c,NDB_433_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:300, dumped\n",
      "--- Sequence:loc_6:145723149-145813383_gene_390_pb_342_pos_84443_strand_+_readouts_[NDB_332_c,NDB_433_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:108, dumped\n",
      "--- Sequence:loc_6:145723149-145813383_gene_390_pb_343_pos_84542_strand_+_readouts_[NDB_332_c,NDB_433_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:114, dumped\n",
      "--- Sequence:loc_6:145723149-145813383_gene_390_pb_344_pos_84651_strand_+_readouts_[NDB_332_c,NDB_433_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:94, dumped\n",
      "--- Sequence:loc_6:145723149-145813383_gene_390_pb_345_pos_84750_strand_+_readouts_[NDB_332_c,NDB_433_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:105, dumped\n",
      "--- Sequence:loc_6:145723149-145813383_gene_390_pb_346_pos_84794_strand_+_readouts_[NDB_332_c,NDB_433_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:114, dumped\n",
      "--- Sequence:loc_6:145723149-145813383_gene_390_pb_347_pos_84842_strand_-_readouts_[NDB_332_c,NDB_433_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:114, dumped\n",
      "--- Sequence:loc_6:145723149-145813383_gene_390_pb_393_pos_89179_strand_+_readouts_[NDB_332_c,NDB_433_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:56, dumped\n",
      "--- Sequence:loc_6:146320927-146368669_gene_391_pb_59_pos_7557_strand_+_readouts_[NDB_461_c,NDB_619_c,NDB_604_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:60, dumped\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- Sequence:loc_7:64285872-64371201_gene_405_pb_343_pos_80521_strand_+_readouts_[NDB_455_c,NDB_356_c,NDB_551_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:63, dumped\n",
      "--- Sequence:loc_7:64285872-64371201_gene_405_pb_344_pos_80576_strand_+_readouts_[NDB_455_c,NDB_356_c,NDB_551_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:65, dumped\n",
      "--- Sequence:loc_7:83790534-83849178_gene_413_pb_114_pos_35047_strand_+_readouts_[NDB_332_c,NDB_517_c,NDB_365_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:149, dumped\n",
      "--- Sequence:loc_7:83790534-83849178_gene_413_pb_115_pos_35153_strand_-_readouts_[NDB_332_c,NDB_517_c,NDB_365_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:168, dumped\n",
      "--- Sequence:loc_7:83790534-83849178_gene_413_pb_116_pos_35207_strand_-_readouts_[NDB_332_c,NDB_517_c,NDB_365_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:146, dumped\n",
      "--- Sequence:loc_7:83790534-83849178_gene_413_pb_117_pos_35466_strand_-_readouts_[NDB_332_c,NDB_517_c,NDB_365_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:162, dumped\n",
      "--- Sequence:loc_7:83790534-83849178_gene_413_pb_118_pos_35543_strand_-_readouts_[NDB_332_c,NDB_517_c,NDB_365_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:156, dumped\n",
      "--- Sequence:loc_7:83790534-83849178_gene_413_pb_119_pos_35647_strand_-_readouts_[NDB_332_c,NDB_517_c,NDB_365_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:250, dumped\n",
      "--- Sequence:loc_7:83790534-83849178_gene_413_pb_120_pos_35711_strand_+_readouts_[NDB_332_c,NDB_517_c,NDB_365_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:104, dumped\n",
      "--- Sequence:loc_7:83790534-83849178_gene_413_pb_121_pos_35791_strand_+_readouts_[NDB_332_c,NDB_517_c,NDB_365_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:250, dumped\n",
      "--- Sequence:loc_7:83790534-83849178_gene_413_pb_122_pos_35835_strand_-_readouts_[NDB_332_c,NDB_517_c,NDB_365_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:172, dumped\n",
      "--- Sequence:loc_7:98125903-98181642_gene_417_pb_27_pos_28046_strand_+_readouts_[NDB_431_c,NDB_487_c,NDB_563_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:67, dumped\n",
      "--- Sequence:loc_7:109914369-109963604_gene_420_pb_272_pos_38478_strand_+_readouts_[NDB_571_c,NDB_482_c,NDB_601_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_7:109914369-109963604_gene_420_pb_275_pos_38729_strand_+_readouts_[NDB_571_c,NDB_482_c,NDB_601_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:85, dumped\n",
      "--- Sequence:loc_7:109914369-109963604_gene_420_pb_276_pos_38832_strand_+_readouts_[NDB_571_c,NDB_482_c,NDB_601_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:84, dumped\n",
      "--- Sequence:loc_7:109914369-109963604_gene_420_pb_277_pos_38913_strand_+_readouts_[NDB_571_c,NDB_482_c,NDB_601_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:65, dumped\n",
      "--- Sequence:loc_7:109914369-109963604_gene_420_pb_278_pos_39006_strand_-_readouts_[NDB_571_c,NDB_482_c,NDB_601_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:60, dumped\n",
      "--- Sequence:loc_7:109914369-109963604_gene_420_pb_279_pos_39052_strand_-_readouts_[NDB_571_c,NDB_482_c,NDB_601_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_7:109914369-109963604_gene_420_pb_280_pos_39128_strand_+_readouts_[NDB_571_c,NDB_482_c,NDB_601_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:59, dumped\n",
      "--- Sequence:loc_7:109914369-109963604_gene_420_pb_281_pos_39173_strand_+_readouts_[NDB_571_c,NDB_482_c,NDB_601_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_7:109914369-109963604_gene_420_pb_282_pos_39238_strand_+_readouts_[NDB_571_c,NDB_482_c,NDB_601_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_7:142323852-142438691_gene_436_pb_324_pos_81309_strand_+_readouts_[NDB_376_c,NDB_421_c,NDB_589_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:59, dumped\n",
      "--- Sequence:loc_7:142323852-142438691_gene_436_pb_325_pos_81396_strand_+_readouts_[NDB_376_c,NDB_421_c,NDB_589_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:80, dumped\n",
      "--- Sequence:loc_8:13909259-13976263_gene_438_pb_284_pos_57847_strand_+_readouts_[NDB_464_c,NDB_482_c,NDB_436_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:226, dumped\n",
      "--- Sequence:loc_8:13909259-13976263_gene_438_pb_285_pos_57906_strand_+_readouts_[NDB_464_c,NDB_482_c,NDB_436_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:216, dumped\n",
      "--- Sequence:loc_8:13909259-13976263_gene_438_pb_286_pos_57990_strand_+_readouts_[NDB_464_c,NDB_482_c,NDB_436_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:216, dumped\n",
      "--- Sequence:loc_8:13909259-13976263_gene_438_pb_287_pos_58057_strand_-_readouts_[NDB_464_c,NDB_482_c,NDB_436_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:145, dumped\n",
      "--- Sequence:loc_8:13909259-13976263_gene_438_pb_288_pos_58108_strand_-_readouts_[NDB_464_c,NDB_482_c,NDB_436_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:228, dumped\n",
      "--- Sequence:loc_8:13909259-13976263_gene_438_pb_289_pos_58156_strand_-_readouts_[NDB_464_c,NDB_482_c,NDB_436_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_8:13909259-13976263_gene_438_pb_290_pos_58213_strand_-_readouts_[NDB_464_c,NDB_482_c,NDB_436_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:150, dumped\n",
      "--- Sequence:loc_8:13909259-13976263_gene_438_pb_291_pos_58290_strand_+_readouts_[NDB_464_c,NDB_482_c,NDB_436_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:203, dumped\n",
      "--- Sequence:loc_8:13909259-13976263_gene_438_pb_292_pos_58350_strand_+_readouts_[NDB_464_c,NDB_482_c,NDB_436_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:227, dumped\n",
      "--- Sequence:loc_8:13909259-13976263_gene_438_pb_293_pos_58398_strand_+_readouts_[NDB_464_c,NDB_482_c,NDB_436_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:230, dumped\n",
      "--- Sequence:loc_8:13909259-13976263_gene_438_pb_294_pos_58482_strand_+_readouts_[NDB_464_c,NDB_482_c,NDB_436_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:179, dumped\n",
      "--- Sequence:loc_8:13909259-13976263_gene_438_pb_295_pos_58552_strand_+_readouts_[NDB_464_c,NDB_482_c,NDB_436_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:85, dumped\n",
      "--- Sequence:loc_8:13909259-13976263_gene_438_pb_296_pos_58621_strand_+_readouts_[NDB_464_c,NDB_482_c,NDB_436_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:164, dumped\n",
      "--- Sequence:loc_8:13909259-13976263_gene_438_pb_297_pos_58666_strand_+_readouts_[NDB_464_c,NDB_482_c,NDB_436_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:128, dumped\n",
      "--- Sequence:loc_8:13909259-13976263_gene_438_pb_298_pos_58715_strand_-_readouts_[NDB_464_c,NDB_482_c,NDB_436_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:105, dumped\n",
      "--- Sequence:loc_8:15984403-16043579_gene_440_pb_90_pos_33566_strand_-_readouts_[NDB_443_c,NDB_472_c,NDB_589_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_8:15984403-16043579_gene_440_pb_91_pos_33644_strand_-_readouts_[NDB_443_c,NDB_472_c,NDB_589_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:73, dumped\n",
      "--- Sequence:loc_8:25338122-25379634_gene_444_pb_228_pos_26551_strand_-_readouts_[NDB_571_c,NDB_427_c,NDB_385_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_8:25338122-25379634_gene_444_pb_229_pos_26708_strand_-_readouts_[NDB_571_c,NDB_427_c,NDB_385_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:112, dumped\n",
      "--- Sequence:loc_8:25338122-25379634_gene_444_pb_230_pos_26766_strand_-_readouts_[NDB_571_c,NDB_427_c,NDB_385_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:112, dumped\n",
      "--- Sequence:loc_8:25338122-25379634_gene_444_pb_231_pos_26833_strand_-_readouts_[NDB_571_c,NDB_427_c,NDB_385_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:135, dumped\n",
      "--- Sequence:loc_8:25338122-25379634_gene_444_pb_232_pos_26928_strand_-_readouts_[NDB_571_c,NDB_427_c,NDB_385_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:96, dumped\n",
      "--- Sequence:loc_8:25338122-25379634_gene_444_pb_233_pos_26991_strand_+_readouts_[NDB_571_c,NDB_427_c,NDB_385_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:112, dumped\n",
      "--- Sequence:loc_8:25338122-25379634_gene_444_pb_234_pos_27044_strand_-_readouts_[NDB_571_c,NDB_427_c,NDB_385_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:61, dumped\n",
      "--- Sequence:loc_8:68504342-68569036_gene_460_pb_87_pos_38982_strand_-_readouts_[NDB_481_c,NDB_350_c,NDB_460_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- Sequence:loc_8:78008249-78074137_gene_465_pb_211_pos_48574_strand_+_readouts_[NDB_554_c,NDB_577_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:68, dumped\n",
      "--- Sequence:loc_8:78008249-78074137_gene_465_pb_212_pos_48618_strand_+_readouts_[NDB_554_c,NDB_577_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_8:79321964-79402981_gene_466_pb_205_pos_57384_strand_+_readouts_[NDB_335_c,NDB_557_c,NDB_547_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:75, dumped\n",
      "--- Sequence:loc_8:79321964-79402981_gene_466_pb_206_pos_57533_strand_+_readouts_[NDB_335_c,NDB_557_c,NDB_547_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:63, dumped\n",
      "--- Sequence:loc_8:79321964-79402981_gene_466_pb_208_pos_57665_strand_+_readouts_[NDB_335_c,NDB_557_c,NDB_547_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:73, dumped\n",
      "--- Sequence:loc_8:90827081-90904757_gene_473_pb_85_pos_42216_strand_+_readouts_[NDB_400_c,NDB_487_c,NDB_551_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:427, dumped\n",
      "--- Sequence:loc_9:46291898-46364450_gene_501_pb_80_pos_39222_strand_-_readouts_[NDB_391_c,NDB_398_c,NDB_506_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:117, dumped\n",
      "--- Sequence:loc_9:46291898-46364450_gene_501_pb_82_pos_39378_strand_+_readouts_[NDB_391_c,NDB_398_c,NDB_506_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:98, dumped\n",
      "--- Sequence:loc_9:46291898-46364450_gene_501_pb_83_pos_39434_strand_+_readouts_[NDB_391_c,NDB_398_c,NDB_506_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:113, dumped\n",
      "--- Sequence:loc_9:54645501-54702064_gene_509_pb_242_pos_43698_strand_-_readouts_[NDB_491_c,NDB_472_c,NDB_604_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:56, dumped\n",
      "--- Sequence:loc_9:54645501-54702064_gene_509_pb_243_pos_43744_strand_+_readouts_[NDB_491_c,NDB_472_c,NDB_604_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:56, dumped\n",
      "--- Sequence:loc_9:54645501-54702064_gene_509_pb_244_pos_43790_strand_+_readouts_[NDB_491_c,NDB_472_c,NDB_604_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:55, dumped\n",
      "--- Sequence:loc_9:54645501-54702064_gene_509_pb_246_pos_43883_strand_-_readouts_[NDB_491_c,NDB_472_c,NDB_604_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:55, dumped\n",
      "--- Sequence:loc_9:64174332-64252851_gene_512_pb_364_pos_75240_strand_+_readouts_[NDB_445_c,NDB_493_c,NDB_547_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_9:64174332-64252851_gene_512_pb_365_pos_75382_strand_-_readouts_[NDB_445_c,NDB_493_c,NDB_547_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:68, dumped\n",
      "--- Sequence:loc_9:80159682-80266480_gene_521_pb_126_pos_62128_strand_-_readouts_[NDB_578_c,NDB_511_c,NDB_628_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:62, dumped\n",
      "--- Sequence:loc_9:80159682-80266480_gene_521_pb_129_pos_62408_strand_+_readouts_[NDB_578_c,NDB_511_c,NDB_628_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:64, dumped\n",
      "--- Sequence:loc_9:86573601-86647179_gene_524_pb_275_pos_60333_strand_-_readouts_[NDB_418_c,NDB_392_c,NDB_395_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:431, dumped\n",
      "--- Sequence:loc_9:89831577-89911977_gene_526_pb_138_pos_48334_strand_+_readouts_[NDB_464_c,NDB_461_c,NDB_335_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:68, dumped\n",
      "--- Sequence:loc_9:89831577-89911977_gene_526_pb_139_pos_48477_strand_-_readouts_[NDB_464_c,NDB_461_c,NDB_335_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:75, dumped\n",
      "--- Sequence:loc_9:89831577-89911977_gene_526_pb_141_pos_48679_strand_-_readouts_[NDB_464_c,NDB_461_c,NDB_335_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:78, dumped\n",
      "--- Sequence:loc_9:89831577-89911977_gene_526_pb_144_pos_49102_strand_-_readouts_[NDB_464_c,NDB_461_c,NDB_335_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:71, dumped\n",
      "--- Sequence:loc_9:89831577-89911977_gene_526_pb_145_pos_49192_strand_+_readouts_[NDB_464_c,NDB_461_c,NDB_335_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:63, dumped\n",
      "--- Sequence:loc_9:89831577-89911977_gene_526_pb_146_pos_49311_strand_+_readouts_[NDB_464_c,NDB_461_c,NDB_335_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:62, dumped\n",
      "--- Sequence:loc_9:89831577-89911977_gene_526_pb_147_pos_49475_strand_+_readouts_[NDB_464_c,NDB_461_c,NDB_335_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:61, dumped\n",
      "--- Sequence:loc_9:89831577-89911977_gene_526_pb_148_pos_49702_strand_+_readouts_[NDB_464_c,NDB_461_c,NDB_335_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:83, dumped\n",
      "--- Sequence:loc_9:89831577-89911977_gene_526_pb_151_pos_49937_strand_+_readouts_[NDB_464_c,NDB_461_c,NDB_335_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:61, dumped\n",
      "--- Sequence:loc_9:99753284-99818353_gene_532_pb_271_pos_54835_strand_+_readouts_[NDB_398_c,NDB_616_c,NDB_556_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:69, dumped\n",
      "--- Sequence:loc_9:99753284-99818353_gene_532_pb_272_pos_54894_strand_+_readouts_[NDB_398_c,NDB_616_c,NDB_556_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:59, dumped\n",
      "--- Sequence:loc_9:99753284-99818353_gene_532_pb_273_pos_54949_strand_-_readouts_[NDB_398_c,NDB_616_c,NDB_556_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:73, dumped\n",
      "--- Sequence:loc_9:99753284-99818353_gene_532_pb_274_pos_55003_strand_+_readouts_[NDB_398_c,NDB_616_c,NDB_556_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:62, dumped\n",
      "--- Sequence:loc_9:99753284-99818353_gene_532_pb_275_pos_55077_strand_+_readouts_[NDB_398_c,NDB_616_c,NDB_556_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:66, dumped\n",
      "--- Sequence:loc_10:9791406-9903185_gene_547_pb_19_pos_62038_strand_-_readouts_[NDB_578_c,NDB_532_c,NDB_619_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:163, dumped\n",
      "--- Sequence:loc_10:9791406-9903185_gene_547_pb_20_pos_62098_strand_-_readouts_[NDB_578_c,NDB_532_c,NDB_619_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:208, dumped\n",
      "--- Sequence:loc_10:9791406-9903185_gene_547_pb_21_pos_62171_strand_+_readouts_[NDB_578_c,NDB_532_c,NDB_619_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:201, dumped\n",
      "--- Sequence:loc_10:9791406-9903185_gene_547_pb_22_pos_62217_strand_+_readouts_[NDB_578_c,NDB_532_c,NDB_619_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:168, dumped\n",
      "--- Sequence:loc_10:9791406-9903185_gene_547_pb_24_pos_62370_strand_+_readouts_[NDB_578_c,NDB_532_c,NDB_619_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:86, dumped\n",
      "--- Sequence:loc_10:9791406-9903185_gene_547_pb_25_pos_62417_strand_-_readouts_[NDB_578_c,NDB_532_c,NDB_619_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:148, dumped\n",
      "--- Sequence:loc_10:9791406-9903185_gene_547_pb_26_pos_62463_strand_-_readouts_[NDB_578_c,NDB_532_c,NDB_619_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:69, dumped\n",
      "--- Sequence:loc_10:9791406-9903185_gene_547_pb_27_pos_62545_strand_-_readouts_[NDB_578_c,NDB_532_c,NDB_619_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:129, dumped\n",
      "--- Sequence:loc_10:9791406-9903185_gene_547_pb_29_pos_62683_strand_-_readouts_[NDB_578_c,NDB_532_c,NDB_619_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:172, dumped\n",
      "--- Sequence:loc_10:9791406-9903185_gene_547_pb_30_pos_62727_strand_-_readouts_[NDB_578_c,NDB_532_c,NDB_619_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:112, dumped\n",
      "--- Sequence:loc_10:9791406-9903185_gene_547_pb_32_pos_62833_strand_+_readouts_[NDB_578_c,NDB_532_c,NDB_619_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:129, dumped\n",
      "--- Sequence:loc_10:9791406-9903185_gene_547_pb_33_pos_62912_strand_-_readouts_[NDB_578_c,NDB_532_c,NDB_619_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:110, dumped\n",
      "--- Sequence:loc_10:9791406-9903185_gene_547_pb_34_pos_62973_strand_+_readouts_[NDB_578_c,NDB_532_c,NDB_619_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:90, dumped\n",
      "--- Sequence:loc_10:9791406-9903185_gene_547_pb_35_pos_63116_strand_-_readouts_[NDB_578_c,NDB_532_c,NDB_619_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:201, dumped\n",
      "--- Sequence:loc_10:9791406-9903185_gene_547_pb_36_pos_63162_strand_-_readouts_[NDB_578_c,NDB_532_c,NDB_619_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:120, dumped\n",
      "--- Sequence:loc_10:9791406-9903185_gene_547_pb_37_pos_63347_strand_+_readouts_[NDB_578_c,NDB_532_c,NDB_619_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:131, dumped\n",
      "--- Sequence:loc_10:9791406-9903185_gene_547_pb_38_pos_63406_strand_+_readouts_[NDB_578_c,NDB_532_c,NDB_619_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:144, dumped\n",
      "--- Sequence:loc_10:9791406-9903185_gene_547_pb_39_pos_63450_strand_-_readouts_[NDB_578_c,NDB_532_c,NDB_619_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:110, dumped\n",
      "--- Sequence:loc_10:9791406-9903185_gene_547_pb_40_pos_63524_strand_+_readouts_[NDB_578_c,NDB_532_c,NDB_619_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:86, dumped\n",
      "--- Sequence:loc_10:9791406-9903185_gene_547_pb_41_pos_63583_strand_-_readouts_[NDB_578_c,NDB_532_c,NDB_619_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:161, dumped\n",
      "--- Sequence:loc_10:9791406-9903185_gene_547_pb_42_pos_63630_strand_+_readouts_[NDB_578_c,NDB_532_c,NDB_619_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:87, dumped\n",
      "--- Sequence:loc_10:9791406-9903185_gene_547_pb_43_pos_63702_strand_+_readouts_[NDB_578_c,NDB_532_c,NDB_619_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:147, dumped\n",
      "--- Sequence:loc_10:9791406-9903185_gene_547_pb_44_pos_63763_strand_+_readouts_[NDB_578_c,NDB_532_c,NDB_619_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:206, dumped\n",
      "--- Sequence:loc_10:9791406-9903185_gene_547_pb_45_pos_63834_strand_-_readouts_[NDB_578_c,NDB_532_c,NDB_619_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:133, dumped\n",
      "--- Sequence:loc_10:9791406-9903185_gene_547_pb_46_pos_63878_strand_+_readouts_[NDB_578_c,NDB_532_c,NDB_619_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:89, dumped\n",
      "--- Sequence:loc_10:9791406-9903185_gene_547_pb_74_pos_65705_strand_-_readouts_[NDB_578_c,NDB_532_c,NDB_619_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_10:9791406-9903185_gene_547_pb_75_pos_65776_strand_-_readouts_[NDB_578_c,NDB_532_c,NDB_619_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:56, dumped\n",
      "--- Sequence:loc_10:21881312-21991778_gene_551_pb_192_pos_72257_strand_-_readouts_[NDB_403_c,NDB_386_c,NDB_601_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:62, dumped\n",
      "--- Sequence:loc_10:21881312-21991778_gene_551_pb_193_pos_72332_strand_+_readouts_[NDB_403_c,NDB_386_c,NDB_601_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:68, dumped\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- Sequence:loc_10:23181367-23277580_gene_552_pb_354_pos_92946_strand_+_readouts_[NDB_397_c,NDB_539_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:56, dumped\n",
      "--- Sequence:loc_10:23181367-23277580_gene_552_pb_358_pos_93152_strand_+_readouts_[NDB_397_c,NDB_539_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:76, dumped\n",
      "--- Sequence:loc_10:23181367-23277580_gene_552_pb_359_pos_93196_strand_+_readouts_[NDB_397_c,NDB_539_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:118, dumped\n",
      "--- Sequence:loc_10:23181367-23277580_gene_552_pb_360_pos_93240_strand_+_readouts_[NDB_397_c,NDB_539_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:117, dumped\n",
      "--- Sequence:loc_10:23181367-23277580_gene_552_pb_361_pos_93353_strand_+_readouts_[NDB_397_c,NDB_539_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:140, dumped\n",
      "--- Sequence:loc_10:23181367-23277580_gene_552_pb_362_pos_93433_strand_+_readouts_[NDB_397_c,NDB_539_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:138, dumped\n",
      "--- Sequence:loc_10:23181367-23277580_gene_552_pb_363_pos_93477_strand_+_readouts_[NDB_397_c,NDB_539_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:103, dumped\n",
      "--- Sequence:loc_10:23181367-23277580_gene_552_pb_364_pos_93521_strand_+_readouts_[NDB_397_c,NDB_539_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:119, dumped\n",
      "--- Sequence:loc_10:23181367-23277580_gene_552_pb_365_pos_93585_strand_+_readouts_[NDB_397_c,NDB_539_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:89, dumped\n",
      "--- Sequence:loc_10:23181367-23277580_gene_552_pb_366_pos_93663_strand_+_readouts_[NDB_397_c,NDB_539_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:121, dumped\n",
      "--- Sequence:loc_10:23181367-23277580_gene_552_pb_367_pos_93707_strand_+_readouts_[NDB_397_c,NDB_539_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:142, dumped\n",
      "--- Sequence:loc_10:23181367-23277580_gene_552_pb_368_pos_93751_strand_+_readouts_[NDB_397_c,NDB_539_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_10:25400587-25476543_gene_554_pb_217_pos_57876_strand_+_readouts_[NDB_467_c,NDB_335_c,NDB_356_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_10:25400587-25476543_gene_554_pb_218_pos_57936_strand_+_readouts_[NDB_467_c,NDB_335_c,NDB_356_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:97, dumped\n",
      "--- Sequence:loc_10:25400587-25476543_gene_554_pb_219_pos_57983_strand_+_readouts_[NDB_467_c,NDB_335_c,NDB_356_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:93, dumped\n",
      "--- Sequence:loc_10:25400587-25476543_gene_554_pb_220_pos_58471_strand_+_readouts_[NDB_467_c,NDB_335_c,NDB_356_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:80, dumped\n",
      "--- Sequence:loc_10:25400587-25476543_gene_554_pb_221_pos_58517_strand_+_readouts_[NDB_467_c,NDB_335_c,NDB_356_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:86, dumped\n",
      "--- Sequence:loc_10:25400587-25476543_gene_554_pb_222_pos_58564_strand_+_readouts_[NDB_467_c,NDB_335_c,NDB_356_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:74, dumped\n",
      "--- Sequence:loc_10:25400587-25476543_gene_554_pb_223_pos_58609_strand_+_readouts_[NDB_467_c,NDB_335_c,NDB_356_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:76, dumped\n",
      "--- Sequence:loc_10:25400587-25476543_gene_554_pb_382_pos_73299_strand_-_readouts_[NDB_467_c,NDB_335_c,NDB_356_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:192, dumped\n",
      "--- Sequence:loc_10:25400587-25476543_gene_554_pb_383_pos_73568_strand_+_readouts_[NDB_467_c,NDB_335_c,NDB_356_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:149, dumped\n",
      "--- Sequence:loc_10:25400587-25476543_gene_554_pb_384_pos_73615_strand_-_readouts_[NDB_467_c,NDB_335_c,NDB_356_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:104, dumped\n",
      "--- Sequence:loc_10:25400587-25476543_gene_554_pb_385_pos_73659_strand_+_readouts_[NDB_467_c,NDB_335_c,NDB_356_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:167, dumped\n",
      "--- Sequence:loc_10:25400587-25476543_gene_554_pb_386_pos_73830_strand_+_readouts_[NDB_467_c,NDB_335_c,NDB_356_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:108, dumped\n",
      "--- Sequence:loc_10:25400587-25476543_gene_554_pb_387_pos_74440_strand_+_readouts_[NDB_467_c,NDB_335_c,NDB_356_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:192, dumped\n",
      "--- Sequence:loc_10:25400587-25476543_gene_554_pb_388_pos_74510_strand_+_readouts_[NDB_467_c,NDB_335_c,NDB_356_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:167, dumped\n",
      "--- Sequence:loc_10:25400587-25476543_gene_554_pb_389_pos_74767_strand_+_readouts_[NDB_467_c,NDB_335_c,NDB_356_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:175, dumped\n",
      "--- Sequence:loc_10:25400587-25476543_gene_554_pb_390_pos_74908_strand_-_readouts_[NDB_467_c,NDB_335_c,NDB_356_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:129, dumped\n",
      "--- Sequence:loc_10:25400587-25476543_gene_554_pb_391_pos_74979_strand_+_readouts_[NDB_467_c,NDB_335_c,NDB_356_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:176, dumped\n",
      "--- Sequence:loc_10:25400587-25476543_gene_554_pb_392_pos_75092_strand_+_readouts_[NDB_467_c,NDB_335_c,NDB_356_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:94, dumped\n",
      "--- Sequence:loc_10:25400587-25476543_gene_554_pb_393_pos_75144_strand_+_readouts_[NDB_467_c,NDB_335_c,NDB_356_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:169, dumped\n",
      "--- Sequence:loc_10:40058995-40139830_gene_559_pb_187_pos_56022_strand_+_readouts_[NDB_554_c,NDB_538_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:62, dumped\n",
      "--- Sequence:loc_10:40058995-40139830_gene_559_pb_188_pos_56128_strand_-_readouts_[NDB_554_c,NDB_538_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:60, dumped\n",
      "--- Sequence:loc_10:82628910-82692199_gene_581_pb_20_pos_8459_strand_+_readouts_[NDB_505_c,NDB_418_c,NDB_481_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_10:82628910-82692199_gene_581_pb_22_pos_8564_strand_+_readouts_[NDB_505_c,NDB_418_c,NDB_481_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_10:84077311-84175022_gene_584_pb_16_pos_46035_strand_+_readouts_[NDB_443_c,NDB_625_c,NDB_493_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_10:84921289-85033749_gene_585_pb_73_pos_64959_strand_+_readouts_[NDB_466_c,NDB_467_c,NDB_371_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_10:110917563-111009521_gene_600_pb_84_pos_49510_strand_-_readouts_[NDB_487_c,NDB_571_c,NDB_526_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_11:22621290-22687776_gene_617_pb_57_pos_10547_strand_+_readouts_[NDB_443_c,NDB_473_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:427, dumped\n",
      "--- Sequence:loc_11:34044208-34109026_gene_622_pb_147_pos_42598_strand_+_readouts_[NDB_398_c,NDB_577_c,NDB_604_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:66, dumped\n",
      "--- Sequence:loc_11:34044208-34109026_gene_622_pb_148_pos_42659_strand_+_readouts_[NDB_398_c,NDB_577_c,NDB_604_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_11:34044208-34109026_gene_622_pb_149_pos_42703_strand_-_readouts_[NDB_398_c,NDB_577_c,NDB_604_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:64, dumped\n",
      "--- Sequence:loc_11:46420972-46472278_gene_630_pb_37_pos_3813_strand_+_readouts_[NDB_455_c,NDB_341_c,NDB_592_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_11:46420972-46472278_gene_630_pb_53_pos_5330_strand_-_readouts_[NDB_455_c,NDB_341_c,NDB_592_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_11:46420972-46472278_gene_630_pb_54_pos_5374_strand_-_readouts_[NDB_455_c,NDB_341_c,NDB_592_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_11:52624034-52707755_gene_632_pb_293_pos_76326_strand_-_readouts_[NDB_511_c,NDB_403_c,NDB_356_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:127, dumped\n",
      "--- Sequence:loc_11:52624034-52707755_gene_632_pb_294_pos_76409_strand_-_readouts_[NDB_511_c,NDB_403_c,NDB_356_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:88, dumped\n",
      "--- Sequence:loc_11:52624034-52707755_gene_632_pb_296_pos_76583_strand_-_readouts_[NDB_511_c,NDB_403_c,NDB_356_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:75, dumped\n",
      "--- Sequence:loc_11:52624034-52707755_gene_632_pb_297_pos_76633_strand_+_readouts_[NDB_511_c,NDB_403_c,NDB_356_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:119, dumped\n",
      "--- Sequence:loc_11:52624034-52707755_gene_632_pb_298_pos_76677_strand_+_readouts_[NDB_511_c,NDB_403_c,NDB_356_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:105, dumped\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- Sequence:loc_11:63881715-63935692_gene_640_pb_84_pos_31149_strand_+_readouts_[NDB_458_c,NDB_467_c,NDB_401_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n",
      "--- Sequence:loc_11:63881715-63935692_gene_640_pb_85_pos_31194_strand_+_readouts_[NDB_458_c,NDB_467_c,NDB_401_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:56, dumped\n",
      "--- Sequence:loc_11:63881715-63935692_gene_640_pb_87_pos_31372_strand_+_readouts_[NDB_458_c,NDB_467_c,NDB_401_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:71, dumped\n",
      "--- Sequence:loc_12:12755678-12838753_gene_672_pb_237_pos_60206_strand_+_readouts_[NDB_455_c,NDB_391_c,NDB_628_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_12:12755678-12838753_gene_672_pb_238_pos_60252_strand_+_readouts_[NDB_455_c,NDB_391_c,NDB_628_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:58, dumped\n",
      "--- Sequence:loc_12:45004756-45093297_gene_679_pb_261_pos_65287_strand_+_readouts_[NDB_376_c,NDB_511_c,NDB_473_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:56, dumped\n",
      "--- Sequence:loc_12:70857906-70924672_gene_685_pb_224_pos_45696_strand_+_readouts_[NDB_388_c,NDB_433_c,NDB_551_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_12:72169666-72249828_gene_687_pb_306_pos_71410_strand_-_readouts_[NDB_431_c,NDB_397_c,NDB_517_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:66, dumped\n",
      "--- Sequence:loc_12:72169666-72249828_gene_687_pb_307_pos_71461_strand_+_readouts_[NDB_431_c,NDB_397_c,NDB_517_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:61, dumped\n",
      "--- Sequence:loc_12:72169666-72249828_gene_687_pb_308_pos_71528_strand_+_readouts_[NDB_431_c,NDB_397_c,NDB_517_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:66, dumped\n",
      "--- Sequence:loc_12:85147576-85218677_gene_693_pb_381_pos_69303_strand_+_readouts_[NDB_332_c,NDB_505_c,NDB_386_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:97, dumped\n",
      "--- Sequence:loc_12:85147576-85218677_gene_693_pb_382_pos_69386_strand_-_readouts_[NDB_332_c,NDB_505_c,NDB_386_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:96, dumped\n",
      "--- Sequence:loc_12:85147576-85218677_gene_693_pb_383_pos_69458_strand_-_readouts_[NDB_332_c,NDB_505_c,NDB_386_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:101, dumped\n",
      "--- Sequence:loc_12:85147576-85218677_gene_693_pb_384_pos_69519_strand_+_readouts_[NDB_332_c,NDB_505_c,NDB_386_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:63, dumped\n",
      "--- Sequence:loc_12:85147576-85218677_gene_693_pb_385_pos_69589_strand_+_readouts_[NDB_332_c,NDB_505_c,NDB_386_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:97, dumped\n",
      "--- Sequence:loc_12:113265834-113332430_gene_702_pb_163_pos_44494_strand_-_readouts_[NDB_347_c,NDB_482_c,NDB_392_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:61, dumped\n",
      "--- Sequence:loc_12:113265834-113332430_gene_702_pb_166_pos_45735_strand_-_readouts_[NDB_347_c,NDB_482_c,NDB_392_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_12:113265834-113332430_gene_702_pb_167_pos_45919_strand_+_readouts_[NDB_347_c,NDB_482_c,NDB_392_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:59, dumped\n",
      "--- Sequence:loc_12:113265834-113332430_gene_702_pb_168_pos_45964_strand_-_readouts_[NDB_347_c,NDB_482_c,NDB_392_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n",
      "--- Sequence:loc_12:113265834-113332430_gene_702_pb_169_pos_46069_strand_-_readouts_[NDB_347_c,NDB_482_c,NDB_392_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_12:113265834-113332430_gene_702_pb_171_pos_46158_strand_-_readouts_[NDB_347_c,NDB_482_c,NDB_392_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:66, dumped\n",
      "--- Sequence:loc_12:113265834-113332430_gene_702_pb_174_pos_46363_strand_-_readouts_[NDB_347_c,NDB_482_c,NDB_392_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n",
      "--- Sequence:loc_12:113265834-113332430_gene_702_pb_176_pos_46496_strand_+_readouts_[NDB_347_c,NDB_482_c,NDB_392_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:65, dumped\n",
      "--- Sequence:loc_12:113265834-113332430_gene_702_pb_181_pos_46791_strand_-_readouts_[NDB_347_c,NDB_482_c,NDB_392_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:72, dumped\n",
      "--- Sequence:loc_12:113265834-113332430_gene_702_pb_182_pos_46842_strand_-_readouts_[NDB_347_c,NDB_482_c,NDB_392_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:69, dumped\n",
      "--- Sequence:loc_12:113265834-113332430_gene_702_pb_184_pos_46943_strand_+_readouts_[NDB_347_c,NDB_482_c,NDB_392_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:61, dumped\n",
      "--- Sequence:loc_12:113265834-113332430_gene_702_pb_187_pos_47089_strand_-_readouts_[NDB_347_c,NDB_482_c,NDB_392_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:67, dumped\n",
      "--- Sequence:loc_12:113265834-113332430_gene_702_pb_189_pos_47188_strand_+_readouts_[NDB_347_c,NDB_482_c,NDB_392_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:73, dumped\n",
      "--- Sequence:loc_12:113265834-113332430_gene_702_pb_190_pos_47232_strand_+_readouts_[NDB_347_c,NDB_482_c,NDB_392_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:63, dumped\n",
      "--- Sequence:loc_12:113265834-113332430_gene_702_pb_196_pos_47578_strand_+_readouts_[NDB_347_c,NDB_482_c,NDB_392_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:61, dumped\n",
      "--- Sequence:loc_13:15511949-15613228_gene_707_pb_377_pos_92936_strand_+_readouts_[NDB_403_c,NDB_427_c,NDB_557_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:59, dumped\n",
      "--- Sequence:loc_13:60034633-60143990_gene_727_pb_42_pos_55246_strand_+_readouts_[NDB_455_c,NDB_467_c,NDB_359_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:64, dumped\n",
      "--- Sequence:loc_13:60034633-60143990_gene_727_pb_44_pos_55337_strand_+_readouts_[NDB_455_c,NDB_467_c,NDB_359_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:62, dumped\n",
      "--- Sequence:loc_13:60034633-60143990_gene_727_pb_45_pos_55382_strand_+_readouts_[NDB_455_c,NDB_467_c,NDB_359_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:102, dumped\n",
      "--- Sequence:loc_13:60034633-60143990_gene_727_pb_46_pos_55441_strand_-_readouts_[NDB_455_c,NDB_467_c,NDB_359_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_13:60034633-60143990_gene_727_pb_47_pos_55499_strand_+_readouts_[NDB_455_c,NDB_467_c,NDB_359_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:116, dumped\n",
      "--- Sequence:loc_13:60034633-60143990_gene_727_pb_48_pos_55572_strand_+_readouts_[NDB_455_c,NDB_467_c,NDB_359_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:123, dumped\n",
      "--- Sequence:loc_13:60034633-60143990_gene_727_pb_49_pos_55623_strand_+_readouts_[NDB_455_c,NDB_467_c,NDB_359_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:127, dumped\n",
      "--- Sequence:loc_13:60034633-60143990_gene_727_pb_50_pos_55677_strand_+_readouts_[NDB_455_c,NDB_467_c,NDB_359_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:102, dumped\n",
      "--- Sequence:loc_13:60034633-60143990_gene_727_pb_51_pos_55751_strand_-_readouts_[NDB_455_c,NDB_467_c,NDB_359_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:60034633-60143990_gene_727_pb_52_pos_55810_strand_+_readouts_[NDB_455_c,NDB_467_c,NDB_359_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:77, dumped\n",
      "--- Sequence:loc_13:60034633-60143990_gene_727_pb_53_pos_55854_strand_-_readouts_[NDB_455_c,NDB_467_c,NDB_359_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:77, dumped\n",
      "--- Sequence:loc_13:64427959-64496783_gene_730_pb_171_pos_49031_strand_-_readouts_[NDB_341_c,NDB_556_c,NDB_608_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:75, dumped\n",
      "--- Sequence:loc_13:64427959-64496783_gene_730_pb_172_pos_49131_strand_-_readouts_[NDB_341_c,NDB_556_c,NDB_608_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:71, dumped\n",
      "--- Sequence:loc_13:64427959-64496783_gene_730_pb_173_pos_49236_strand_+_readouts_[NDB_341_c,NDB_556_c,NDB_608_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:69, dumped\n",
      "--- Sequence:loc_13:70852179-70914310_gene_732_pb_151_pos_42073_strand_-_readouts_[NDB_461_c,NDB_371_c,NDB_562_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:97, dumped\n",
      "--- Sequence:loc_13:70852179-70914310_gene_732_pb_152_pos_42117_strand_+_readouts_[NDB_461_c,NDB_371_c,NDB_562_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:66, dumped\n",
      "--- Sequence:loc_13:70852179-70914310_gene_732_pb_154_pos_42322_strand_+_readouts_[NDB_461_c,NDB_371_c,NDB_562_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:74, dumped\n",
      "--- Sequence:loc_13:70852179-70914310_gene_732_pb_155_pos_42366_strand_+_readouts_[NDB_461_c,NDB_371_c,NDB_562_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:66, dumped\n",
      "--- Sequence:loc_13:70852179-70914310_gene_732_pb_156_pos_42417_strand_-_readouts_[NDB_461_c,NDB_371_c,NDB_562_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:89, dumped\n",
      "--- Sequence:loc_13:70852179-70914310_gene_732_pb_157_pos_42507_strand_+_readouts_[NDB_461_c,NDB_371_c,NDB_562_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:55, dumped\n",
      "--- Sequence:loc_13:70852179-70914310_gene_732_pb_158_pos_42556_strand_+_readouts_[NDB_461_c,NDB_371_c,NDB_562_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:76, dumped\n",
      "--- Sequence:loc_13:70852179-70914310_gene_732_pb_159_pos_42600_strand_-_readouts_[NDB_461_c,NDB_371_c,NDB_562_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:65, dumped\n",
      "--- Sequence:loc_13:70852179-70914310_gene_732_pb_160_pos_42644_strand_+_readouts_[NDB_461_c,NDB_371_c,NDB_562_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:68, dumped\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- Sequence:loc_13:99342904-99407764_gene_740_pb_165_pos_45941_strand_+_readouts_[NDB_431_c,NDB_538_c,NDB_460_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:140, dumped\n",
      "--- Sequence:loc_13:99342904-99407764_gene_740_pb_166_pos_45992_strand_+_readouts_[NDB_431_c,NDB_538_c,NDB_460_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:100, dumped\n",
      "--- Sequence:loc_13:99342904-99407764_gene_740_pb_167_pos_46041_strand_+_readouts_[NDB_431_c,NDB_538_c,NDB_460_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:90, dumped\n",
      "--- Sequence:loc_13:99342904-99407764_gene_740_pb_168_pos_46103_strand_+_readouts_[NDB_431_c,NDB_538_c,NDB_460_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:80, dumped\n",
      "--- Sequence:loc_13:99342904-99407764_gene_740_pb_169_pos_46174_strand_-_readouts_[NDB_431_c,NDB_538_c,NDB_460_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:94, dumped\n",
      "--- Sequence:loc_13:99342904-99407764_gene_740_pb_170_pos_46223_strand_+_readouts_[NDB_431_c,NDB_538_c,NDB_460_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:132, dumped\n",
      "--- Sequence:loc_13:99342904-99407764_gene_740_pb_171_pos_46466_strand_+_readouts_[NDB_431_c,NDB_538_c,NDB_460_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:144, dumped\n",
      "--- Sequence:loc_13:99342904-99407764_gene_740_pb_172_pos_46668_strand_-_readouts_[NDB_431_c,NDB_538_c,NDB_460_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:94, dumped\n",
      "--- Sequence:loc_13:99342904-99407764_gene_740_pb_173_pos_46723_strand_-_readouts_[NDB_431_c,NDB_538_c,NDB_460_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:94, dumped\n",
      "--- Sequence:loc_13:99342904-99407764_gene_740_pb_174_pos_46767_strand_-_readouts_[NDB_431_c,NDB_538_c,NDB_460_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:137, dumped\n",
      "--- Sequence:loc_13:99342904-99407764_gene_740_pb_175_pos_46837_strand_+_readouts_[NDB_431_c,NDB_538_c,NDB_460_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:163, dumped\n",
      "--- Sequence:loc_13:99342904-99407764_gene_740_pb_176_pos_46881_strand_+_readouts_[NDB_431_c,NDB_538_c,NDB_460_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:137, dumped\n",
      "--- Sequence:loc_13:108046902-108146316_gene_745_pb_192_pos_65952_strand_+_readouts_[NDB_458_c,NDB_335_c,NDB_551_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:64, dumped\n",
      "--- Sequence:loc_13:108046902-108146316_gene_745_pb_193_pos_66124_strand_+_readouts_[NDB_458_c,NDB_335_c,NDB_551_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:93, dumped\n",
      "--- Sequence:loc_13:108046902-108146316_gene_745_pb_194_pos_66190_strand_+_readouts_[NDB_458_c,NDB_335_c,NDB_551_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:81, dumped\n",
      "--- Sequence:loc_13:108046902-108146316_gene_745_pb_195_pos_66240_strand_+_readouts_[NDB_458_c,NDB_335_c,NDB_551_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:85, dumped\n",
      "--- Sequence:loc_13:108254869-108312090_gene_746_pb_55_pos_11855_strand_+_readouts_[NDB_391_c,NDB_421_c,NDB_481_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_0_pos_176_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_1_pos_44169_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_2_pos_44214_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_3_pos_44391_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_4_pos_44469_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_5_pos_44548_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_6_pos_44656_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_7_pos_44733_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_8_pos_44798_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_9_pos_44858_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_10_pos_44905_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_11_pos_45008_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_12_pos_45193_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_13_pos_45260_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_14_pos_45334_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_15_pos_45383_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_16_pos_45513_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_17_pos_45566_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_18_pos_45947_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_19_pos_45993_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_20_pos_46044_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_21_pos_46126_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_22_pos_46170_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_23_pos_46219_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_24_pos_46281_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_25_pos_46345_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_26_pos_46409_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_27_pos_46485_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_28_pos_46529_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_29_pos_46615_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_30_pos_46722_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_31_pos_46804_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_32_pos_46861_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_33_pos_47010_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_34_pos_47054_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_35_pos_47154_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_36_pos_47211_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_37_pos_47276_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_38_pos_47323_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_39_pos_47389_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_40_pos_47460_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_41_pos_47504_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_42_pos_47637_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_43_pos_47681_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_44_pos_47783_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_45_pos_48024_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_46_pos_48070_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_47_pos_48114_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_48_pos_48222_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_49_pos_48318_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_50_pos_48398_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_51_pos_48458_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_52_pos_48556_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_53_pos_48633_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_54_pos_48717_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_55_pos_48783_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_56_pos_48827_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_57_pos_48875_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_58_pos_48947_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_59_pos_49006_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_60_pos_49061_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_61_pos_49105_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_62_pos_49149_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_63_pos_49221_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_64_pos_49292_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_65_pos_49367_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_66_pos_49414_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_67_pos_49468_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_68_pos_49538_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_69_pos_49628_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_70_pos_49684_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_71_pos_49728_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_72_pos_49818_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_73_pos_49871_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_74_pos_49941_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_75_pos_50003_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_76_pos_50075_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_77_pos_50129_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_78_pos_50174_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_79_pos_50218_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_80_pos_50281_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_81_pos_50352_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_82_pos_50396_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_83_pos_50480_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_84_pos_50572_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_85_pos_50616_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_86_pos_50700_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_87_pos_50746_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_88_pos_50812_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_89_pos_50932_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_90_pos_50983_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_91_pos_51086_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_92_pos_51153_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_93_pos_51225_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_94_pos_51326_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_95_pos_51429_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_96_pos_52110_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_97_pos_52154_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_98_pos_52198_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_99_pos_52244_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_100_pos_52321_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_101_pos_52365_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_102_pos_52430_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_103_pos_52493_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_104_pos_52548_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_105_pos_52632_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_106_pos_52685_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_107_pos_52760_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_108_pos_52804_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_109_pos_52848_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_110_pos_52906_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_111_pos_52954_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_112_pos_53051_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_113_pos_53119_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_114_pos_53175_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_115_pos_53225_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_116_pos_53269_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_117_pos_53313_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_118_pos_53371_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_119_pos_53418_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_120_pos_53463_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_121_pos_53599_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_122_pos_53681_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_123_pos_53726_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_124_pos_53827_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_125_pos_53892_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_126_pos_54006_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_127_pos_54080_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_128_pos_54124_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_129_pos_54170_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_130_pos_54214_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_131_pos_54271_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_132_pos_54317_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_133_pos_54377_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_134_pos_54430_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_135_pos_54474_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_136_pos_54537_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_137_pos_54583_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_138_pos_54805_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_139_pos_54849_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_140_pos_54899_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_141_pos_54958_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_142_pos_55002_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_143_pos_55050_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_144_pos_55104_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_145_pos_55167_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_146_pos_55314_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_147_pos_55394_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_148_pos_55447_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_149_pos_55514_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_150_pos_55610_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_151_pos_55654_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_152_pos_55721_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_153_pos_55819_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_154_pos_55863_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_155_pos_55908_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_156_pos_55956_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_157_pos_56000_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_158_pos_56064_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_159_pos_56108_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_160_pos_56154_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_161_pos_56201_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_162_pos_56340_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_163_pos_56384_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_164_pos_56432_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_165_pos_56491_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_166_pos_56596_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_167_pos_56646_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_168_pos_56720_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_169_pos_56764_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_170_pos_56819_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_171_pos_56872_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_172_pos_56956_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_173_pos_57000_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_174_pos_57052_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_175_pos_57129_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_176_pos_57177_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_177_pos_57268_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_178_pos_57321_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_179_pos_57365_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_180_pos_57419_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_181_pos_57531_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_182_pos_57580_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_183_pos_57643_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_184_pos_57778_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_185_pos_57850_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_186_pos_58021_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_187_pos_58124_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_188_pos_58196_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_189_pos_58240_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_190_pos_58284_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_191_pos_58379_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_192_pos_58491_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_193_pos_58562_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_194_pos_58625_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_195_pos_58689_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_196_pos_58754_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_197_pos_58803_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_198_pos_58848_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_199_pos_58896_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_200_pos_59028_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_201_pos_59164_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_202_pos_59213_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_203_pos_59257_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_204_pos_59326_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_205_pos_59394_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_206_pos_59451_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_207_pos_59511_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_208_pos_59636_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_209_pos_59682_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_210_pos_59763_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_211_pos_60217_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_212_pos_60288_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_213_pos_60357_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_214_pos_60435_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_215_pos_60513_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_216_pos_60580_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_217_pos_60626_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_218_pos_60674_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_219_pos_60756_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_220_pos_60840_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_221_pos_60898_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_222_pos_60942_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_223_pos_60992_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_224_pos_61037_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_225_pos_61082_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_226_pos_61162_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_227_pos_61209_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_228_pos_61255_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_229_pos_61300_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_230_pos_61365_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_231_pos_61412_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_232_pos_61469_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_233_pos_61525_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_234_pos_61569_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_235_pos_61656_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_236_pos_61710_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_237_pos_61770_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_238_pos_61864_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_239_pos_61951_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_240_pos_62006_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_241_pos_62050_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_242_pos_62117_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_243_pos_62166_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_244_pos_62230_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_245_pos_62303_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_246_pos_62347_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_247_pos_62402_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_248_pos_62475_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_249_pos_62519_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_250_pos_62645_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_251_pos_62821_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_252_pos_62869_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_253_pos_62950_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_254_pos_62994_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_255_pos_63038_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_256_pos_63238_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_257_pos_63282_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_258_pos_63534_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_259_pos_63594_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_260_pos_63648_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_261_pos_63693_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_262_pos_63738_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_263_pos_63810_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_264_pos_63989_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_265_pos_64065_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_266_pos_64140_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_267_pos_64184_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_268_pos_64229_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_272_pos_74395_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_273_pos_74468_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_274_pos_74515_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_275_pos_74790_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_276_pos_74880_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_277_pos_74925_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_278_pos_74976_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_279_pos_75033_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_280_pos_75092_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_281_pos_75136_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_282_pos_75298_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_283_pos_75342_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_284_pos_75408_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_285_pos_75498_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_286_pos_75624_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_287_pos_75670_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_288_pos_75749_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_289_pos_75793_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_290_pos_75886_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_291_pos_75963_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_292_pos_76039_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_293_pos_76098_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_294_pos_76175_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_295_pos_76264_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_296_pos_76310_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_297_pos_76373_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_298_pos_76419_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_299_pos_76486_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_300_pos_76537_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_301_pos_76585_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_302_pos_76629_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_303_pos_76692_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_304_pos_76736_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_305_pos_76782_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_306_pos_76833_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_307_pos_76923_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_308_pos_76999_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_309_pos_77082_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_310_pos_77137_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_311_pos_77181_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_312_pos_77229_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_313_pos_77274_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_314_pos_77318_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_315_pos_77433_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_316_pos_77512_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_317_pos_77575_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_318_pos_77619_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_319_pos_77663_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_320_pos_77716_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_321_pos_77793_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_322_pos_77862_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_323_pos_77912_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_324_pos_77992_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_325_pos_78086_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_326_pos_78200_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_327_pos_78244_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_328_pos_78298_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_329_pos_78359_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_330_pos_78443_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_331_pos_78563_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_332_pos_78607_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_333_pos_78672_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_334_pos_78716_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_335_pos_78771_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_336_pos_78815_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_337_pos_78884_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_338_pos_78937_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_339_pos_78991_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_340_pos_79056_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_341_pos_79126_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_342_pos_79198_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_343_pos_79278_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_344_pos_79342_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_345_pos_79402_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_346_pos_79446_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_347_pos_79503_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_348_pos_79576_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_349_pos_79622_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_350_pos_79671_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_351_pos_79723_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_353_pos_79859_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_354_pos_79972_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_355_pos_80087_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_356_pos_80312_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_357_pos_80379_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_358_pos_80423_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_359_pos_80468_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_360_pos_80557_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_361_pos_80615_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_362_pos_80659_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_363_pos_80719_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_366_pos_81330_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_367_pos_81456_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_368_pos_81572_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_369_pos_81667_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_370_pos_81743_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_371_pos_81802_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_372_pos_81846_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_373_pos_81900_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_374_pos_81959_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_375_pos_82034_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_376_pos_82130_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_377_pos_82219_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_378_pos_82280_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_379_pos_82324_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_380_pos_82428_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_381_pos_82563_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_382_pos_82652_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_383_pos_82700_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_384_pos_82759_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_385_pos_82822_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_386_pos_82888_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_387_pos_82961_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_388_pos_83016_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_389_pos_83089_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_390_pos_83133_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_391_pos_83181_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_392_pos_83228_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_393_pos_83272_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_394_pos_83317_strand_+_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149764_gene_750_pb_395_pos_83382_strand_-_readouts_[NDB_379_c,NDB_526_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- Sequence:loc_1:9749317-9845402_gene_2_pb_260_pos_76489_strand_+_readouts_[NDB_511_c,NDB_418_c,NDB_592_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:55, dumped\n",
      "--- Sequence:loc_1:9749317-9845402_gene_2_pb_261_pos_76536_strand_+_readouts_[NDB_511_c,NDB_418_c,NDB_592_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:60, dumped\n",
      "--- Sequence:loc_1:42219499-42273271_gene_14_pb_67_pos_26949_strand_-_readouts_[NDB_508_c,NDB_536_c,NDB_473_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:66, dumped\n",
      "--- Sequence:loc_1:42219499-42273271_gene_14_pb_68_pos_27068_strand_-_readouts_[NDB_508_c,NDB_536_c,NDB_473_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:67, dumped\n",
      "--- Sequence:loc_1:54779256-54924777_gene_19_pb_357_pos_103071_strand_-_readouts_[NDB_529_c,NDB_539_c,NDB_622_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:63, dumped\n",
      "--- Sequence:loc_1:54779256-54924777_gene_19_pb_359_pos_103275_strand_-_readouts_[NDB_529_c,NDB_539_c,NDB_622_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:85, dumped\n",
      "--- Sequence:loc_1:54779256-54924777_gene_19_pb_362_pos_103447_strand_-_readouts_[NDB_529_c,NDB_539_c,NDB_622_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_1:54779256-54924777_gene_19_pb_363_pos_103492_strand_+_readouts_[NDB_529_c,NDB_539_c,NDB_622_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:73, dumped\n",
      "--- Sequence:loc_1:54779256-54924777_gene_19_pb_364_pos_103556_strand_-_readouts_[NDB_529_c,NDB_539_c,NDB_622_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:73, dumped\n",
      "--- Sequence:loc_1:54779256-54924777_gene_19_pb_365_pos_103614_strand_+_readouts_[NDB_529_c,NDB_539_c,NDB_622_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:67, dumped\n",
      "--- Sequence:loc_1:54779256-54924777_gene_19_pb_366_pos_103689_strand_-_readouts_[NDB_529_c,NDB_539_c,NDB_622_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:65, dumped\n",
      "--- Sequence:loc_1:62890963-62958089_gene_28_pb_123_pos_45169_strand_+_readouts_[NDB_551_c,NDB_392_c,NDB_365_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:12831, dumped\n",
      "--- Sequence:loc_1:91412081-91466772_gene_43_pb_63_pos_32369_strand_+_readouts_[NDB_464_c,NDB_616_c,NDB_614_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:986, dumped\n",
      "--- Sequence:loc_1:91412081-91466772_gene_43_pb_337_pos_50629_strand_+_readouts_[NDB_464_c,NDB_616_c,NDB_614_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_1:93189428-93238116_gene_44_pb_286_pos_41193_strand_+_readouts_[NDB_532_c,NDB_491_c,NDB_365_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:99, dumped\n",
      "--- Sequence:loc_1:93189428-93238116_gene_44_pb_287_pos_41257_strand_+_readouts_[NDB_532_c,NDB_491_c,NDB_365_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:89, dumped\n",
      "--- Sequence:loc_1:93189428-93238116_gene_44_pb_288_pos_41307_strand_+_readouts_[NDB_532_c,NDB_491_c,NDB_365_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:102, dumped\n",
      "--- Sequence:loc_1:93189428-93238116_gene_44_pb_289_pos_41351_strand_+_readouts_[NDB_532_c,NDB_491_c,NDB_365_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:89, dumped\n",
      "--- Sequence:loc_1:93189428-93238116_gene_44_pb_290_pos_41403_strand_+_readouts_[NDB_532_c,NDB_491_c,NDB_365_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:91, dumped\n",
      "--- Sequence:loc_1:97604468-97662595_gene_45_pb_167_pos_37305_strand_-_readouts_[NDB_467_c,NDB_341_c,NDB_581_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:78, dumped\n",
      "--- Sequence:loc_1:121430601-121489275_gene_50_pb_334_pos_52917_strand_-_readouts_[NDB_398_c,NDB_385_c,NDB_538_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:137, dumped\n",
      "--- Sequence:loc_1:121430601-121489275_gene_50_pb_335_pos_53009_strand_-_readouts_[NDB_398_c,NDB_385_c,NDB_538_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_1:121430601-121489275_gene_50_pb_336_pos_53053_strand_+_readouts_[NDB_398_c,NDB_385_c,NDB_538_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:117, dumped\n",
      "--- Sequence:loc_1:121430601-121489275_gene_50_pb_337_pos_53203_strand_+_readouts_[NDB_398_c,NDB_385_c,NDB_538_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:161, dumped\n",
      "--- Sequence:loc_1:121430601-121489275_gene_50_pb_338_pos_53431_strand_+_readouts_[NDB_398_c,NDB_385_c,NDB_538_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:145, dumped\n",
      "--- Sequence:loc_1:121430601-121489275_gene_50_pb_339_pos_53504_strand_+_readouts_[NDB_398_c,NDB_385_c,NDB_538_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:161, dumped\n",
      "--- Sequence:loc_1:121430601-121489275_gene_50_pb_340_pos_53584_strand_+_readouts_[NDB_398_c,NDB_385_c,NDB_538_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:161, dumped\n",
      "--- Sequence:loc_1:121430601-121489275_gene_50_pb_341_pos_53737_strand_-_readouts_[NDB_398_c,NDB_385_c,NDB_538_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:155, dumped\n",
      "--- Sequence:loc_1:121430601-121489275_gene_50_pb_342_pos_54258_strand_-_readouts_[NDB_398_c,NDB_385_c,NDB_538_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:143, dumped\n",
      "--- Sequence:loc_1:121430601-121489275_gene_50_pb_343_pos_54323_strand_+_readouts_[NDB_398_c,NDB_385_c,NDB_538_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:123, dumped\n",
      "--- Sequence:loc_1:121430601-121489275_gene_50_pb_344_pos_54426_strand_-_readouts_[NDB_398_c,NDB_385_c,NDB_538_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:129, dumped\n",
      "--- Sequence:loc_1:186617359-186706395_gene_70_pb_338_pos_80986_strand_+_readouts_[NDB_445_c,NDB_622_c,NDB_392_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:79, dumped\n",
      "--- Sequence:loc_1:186617359-186706395_gene_70_pb_339_pos_81042_strand_+_readouts_[NDB_445_c,NDB_622_c,NDB_392_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:82, dumped\n",
      "--- Sequence:loc_1:189866129-189938237_gene_75_pb_64_pos_39621_strand_+_readouts_[NDB_511_c,NDB_398_c,NDB_350_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:61, dumped\n",
      "--- Sequence:loc_1:189866129-189938237_gene_75_pb_65_pos_39665_strand_+_readouts_[NDB_511_c,NDB_398_c,NDB_350_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:69, dumped\n",
      "--- Sequence:loc_1:189866129-189938237_gene_75_pb_66_pos_39995_strand_+_readouts_[NDB_511_c,NDB_398_c,NDB_350_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:77, dumped\n",
      "--- Sequence:loc_1:189866129-189938237_gene_75_pb_67_pos_40079_strand_+_readouts_[NDB_511_c,NDB_398_c,NDB_350_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:73, dumped\n",
      "--- Sequence:loc_2:27071925-27131679_gene_88_pb_378_pos_57379_strand_+_readouts_[NDB_536_c,NDB_577_c,NDB_436_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:62, dumped\n",
      "--- Sequence:loc_2:27071925-27131679_gene_88_pb_383_pos_57689_strand_-_readouts_[NDB_536_c,NDB_577_c,NDB_436_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:68, dumped\n",
      "--- Sequence:loc_2:27071925-27131679_gene_88_pb_384_pos_58120_strand_+_readouts_[NDB_536_c,NDB_577_c,NDB_436_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:62, dumped\n",
      "--- Sequence:loc_2:27071925-27131679_gene_88_pb_385_pos_58258_strand_-_readouts_[NDB_536_c,NDB_577_c,NDB_436_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:71, dumped\n",
      "--- Sequence:loc_2:30236602-30280512_gene_89_pb_86_pos_7350_strand_+_readouts_[NDB_455_c,NDB_514_c,NDB_421_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:75, dumped\n",
      "--- Sequence:loc_2:30236602-30280512_gene_89_pb_87_pos_7418_strand_+_readouts_[NDB_455_c,NDB_514_c,NDB_421_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_2:30236602-30280512_gene_89_pb_343_pos_39166_strand_+_readouts_[NDB_455_c,NDB_514_c,NDB_421_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_2:30236602-30280512_gene_89_pb_344_pos_39243_strand_+_readouts_[NDB_455_c,NDB_514_c,NDB_421_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:61, dumped\n",
      "--- Sequence:loc_2:73253634-73311570_gene_102_pb_34_pos_10527_strand_+_readouts_[NDB_578_c,NDB_409_c,NDB_350_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:284, dumped\n",
      "--- Sequence:loc_2:127069492-127147676_gene_122_pb_26_pos_45985_strand_+_readouts_[NDB_458_c,NDB_392_c,NDB_493_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:426, dumped\n",
      "--- Sequence:loc_2:131363155-131441109_gene_127_pb_41_pos_45080_strand_+_readouts_[NDB_409_c,NDB_517_c,NDB_592_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_2:131363155-131441109_gene_127_pb_42_pos_45175_strand_+_readouts_[NDB_409_c,NDB_517_c,NDB_592_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:66, dumped\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- Sequence:loc_2:135648540-135694141_gene_131_pb_225_pos_29446_strand_+_readouts_[NDB_403_c,NDB_467_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:71, dumped\n",
      "--- Sequence:loc_2:135648540-135694141_gene_131_pb_227_pos_29641_strand_-_readouts_[NDB_403_c,NDB_467_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:64, dumped\n",
      "--- Sequence:loc_2:135648540-135694141_gene_131_pb_228_pos_30331_strand_+_readouts_[NDB_403_c,NDB_467_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:67, dumped\n",
      "--- Sequence:loc_2:135648540-135694141_gene_131_pb_229_pos_30495_strand_+_readouts_[NDB_403_c,NDB_467_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:56, dumped\n",
      "--- Sequence:loc_2:143965229-144015150_gene_135_pb_68_pos_24140_strand_-_readouts_[NDB_571_c,NDB_506_c,NDB_401_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n",
      "--- Sequence:loc_2:148667886-148731885_gene_138_pb_50_pos_33407_strand_+_readouts_[NDB_526_c,NDB_386_c,NDB_563_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_2:153903330-153973662_gene_140_pb_43_pos_36929_strand_+_readouts_[NDB_464_c,NDB_628_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:60, dumped\n",
      "--- Sequence:loc_2:153903330-153973662_gene_140_pb_45_pos_37321_strand_+_readouts_[NDB_464_c,NDB_628_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:60, dumped\n",
      "--- Sequence:loc_2:178484185-178533313_gene_149_pb_328_pos_44029_strand_+_readouts_[NDB_601_c,NDB_479_c,NDB_460_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:121, dumped\n",
      "--- Sequence:loc_2:178484185-178533313_gene_149_pb_329_pos_44099_strand_+_readouts_[NDB_601_c,NDB_479_c,NDB_460_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:72, dumped\n",
      "--- Sequence:loc_2:178484185-178533313_gene_149_pb_330_pos_44145_strand_+_readouts_[NDB_601_c,NDB_479_c,NDB_460_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:127, dumped\n",
      "--- Sequence:loc_2:178484185-178533313_gene_149_pb_331_pos_44232_strand_+_readouts_[NDB_601_c,NDB_479_c,NDB_460_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:151, dumped\n",
      "--- Sequence:loc_2:178484185-178533313_gene_149_pb_332_pos_44285_strand_-_readouts_[NDB_601_c,NDB_479_c,NDB_460_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:59, dumped\n",
      "--- Sequence:loc_2:178484185-178533313_gene_149_pb_333_pos_44367_strand_+_readouts_[NDB_601_c,NDB_479_c,NDB_460_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:147, dumped\n",
      "--- Sequence:loc_2:178484185-178533313_gene_149_pb_334_pos_44435_strand_+_readouts_[NDB_601_c,NDB_479_c,NDB_460_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:140, dumped\n",
      "--- Sequence:loc_2:178484185-178533313_gene_149_pb_335_pos_44483_strand_+_readouts_[NDB_601_c,NDB_479_c,NDB_460_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:146, dumped\n",
      "--- Sequence:loc_2:178484185-178533313_gene_149_pb_336_pos_44531_strand_+_readouts_[NDB_601_c,NDB_479_c,NDB_460_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:135, dumped\n",
      "--- Sequence:loc_2:178484185-178533313_gene_149_pb_337_pos_44652_strand_-_readouts_[NDB_601_c,NDB_479_c,NDB_460_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:71, dumped\n",
      "--- Sequence:loc_2:178484185-178533313_gene_149_pb_338_pos_44696_strand_+_readouts_[NDB_601_c,NDB_479_c,NDB_460_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:146, dumped\n",
      "--- Sequence:loc_2:178484185-178533313_gene_149_pb_339_pos_44797_strand_-_readouts_[NDB_601_c,NDB_479_c,NDB_460_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:103, dumped\n",
      "--- Sequence:loc_2:178484185-178533313_gene_149_pb_340_pos_44860_strand_+_readouts_[NDB_601_c,NDB_479_c,NDB_460_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:163, dumped\n",
      "--- Sequence:loc_2:178484185-178533313_gene_149_pb_341_pos_44908_strand_+_readouts_[NDB_601_c,NDB_479_c,NDB_460_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:178, dumped\n",
      "--- Sequence:loc_2:178484185-178533313_gene_149_pb_342_pos_44957_strand_+_readouts_[NDB_601_c,NDB_479_c,NDB_460_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:67, dumped\n",
      "--- Sequence:loc_2:178484185-178533313_gene_149_pb_343_pos_45005_strand_+_readouts_[NDB_601_c,NDB_479_c,NDB_460_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:89, dumped\n",
      "--- Sequence:loc_3:19284209-19338497_gene_154_pb_363_pos_51383_strand_-_readouts_[NDB_332_c,NDB_398_c,NDB_557_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:67, dumped\n",
      "--- Sequence:loc_3:19284209-19338497_gene_154_pb_364_pos_51446_strand_+_readouts_[NDB_332_c,NDB_398_c,NDB_557_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:60, dumped\n",
      "--- Sequence:loc_3:19284209-19338497_gene_154_pb_365_pos_51492_strand_-_readouts_[NDB_332_c,NDB_398_c,NDB_557_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:95, dumped\n",
      "--- Sequence:loc_3:19284209-19338497_gene_154_pb_366_pos_51540_strand_-_readouts_[NDB_332_c,NDB_398_c,NDB_557_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:63, dumped\n",
      "--- Sequence:loc_3:19284209-19338497_gene_154_pb_368_pos_51670_strand_-_readouts_[NDB_332_c,NDB_398_c,NDB_557_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:90, dumped\n",
      "--- Sequence:loc_3:19284209-19338497_gene_154_pb_370_pos_51810_strand_-_readouts_[NDB_332_c,NDB_398_c,NDB_557_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:60, dumped\n",
      "--- Sequence:loc_3:27183615-27247800_gene_158_pb_280_pos_54317_strand_+_readouts_[NDB_628_c,NDB_341_c,NDB_371_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_3:27183615-27247800_gene_158_pb_284_pos_54546_strand_+_readouts_[NDB_628_c,NDB_341_c,NDB_371_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_3:50411173-50448666_gene_166_pb_261_pos_25475_strand_-_readouts_[NDB_487_c,NDB_391_c,NDB_335_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_3:50411173-50448666_gene_166_pb_278_pos_27045_strand_+_readouts_[NDB_487_c,NDB_391_c,NDB_335_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_3:87262909-87334252_gene_181_pb_139_pos_40531_strand_-_readouts_[NDB_614_c,NDB_601_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:101, dumped\n",
      "--- Sequence:loc_3:87262909-87334252_gene_181_pb_140_pos_40910_strand_-_readouts_[NDB_614_c,NDB_601_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:88, dumped\n",
      "--- Sequence:loc_3:87262909-87334252_gene_181_pb_141_pos_41043_strand_+_readouts_[NDB_614_c,NDB_601_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:64, dumped\n",
      "--- Sequence:loc_3:87262909-87334252_gene_181_pb_142_pos_41211_strand_+_readouts_[NDB_614_c,NDB_601_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:115, dumped\n",
      "--- Sequence:loc_3:87262909-87334252_gene_181_pb_143_pos_41362_strand_+_readouts_[NDB_614_c,NDB_601_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:61, dumped\n",
      "--- Sequence:loc_3:87262909-87334252_gene_181_pb_144_pos_41413_strand_-_readouts_[NDB_614_c,NDB_601_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:99, dumped\n",
      "--- Sequence:loc_3:87262909-87334252_gene_181_pb_145_pos_41537_strand_+_readouts_[NDB_614_c,NDB_601_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:114, dumped\n",
      "--- Sequence:loc_3:87262909-87334252_gene_181_pb_160_pos_42570_strand_-_readouts_[NDB_614_c,NDB_601_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:76, dumped\n",
      "--- Sequence:loc_3:87262909-87334252_gene_181_pb_161_pos_42618_strand_+_readouts_[NDB_614_c,NDB_601_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:98, dumped\n",
      "--- Sequence:loc_3:87262909-87334252_gene_181_pb_162_pos_42696_strand_+_readouts_[NDB_614_c,NDB_601_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:91, dumped\n",
      "--- Sequence:loc_3:87262909-87334252_gene_181_pb_164_pos_42838_strand_+_readouts_[NDB_614_c,NDB_601_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:76, dumped\n",
      "--- Sequence:loc_3:87262909-87334252_gene_181_pb_165_pos_42887_strand_+_readouts_[NDB_614_c,NDB_601_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:62, dumped\n",
      "--- Sequence:loc_3:87262909-87334252_gene_181_pb_167_pos_43004_strand_+_readouts_[NDB_614_c,NDB_601_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:70, dumped\n",
      "--- Sequence:loc_3:87262909-87334252_gene_181_pb_168_pos_43098_strand_-_readouts_[NDB_614_c,NDB_601_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:117, dumped\n",
      "--- Sequence:loc_3:87262909-87334252_gene_181_pb_169_pos_43182_strand_+_readouts_[NDB_614_c,NDB_601_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:82, dumped\n",
      "--- Sequence:loc_3:92217911-92290171_gene_185_pb_100_pos_70868_strand_+_readouts_[NDB_473_c,NDB_386_c,NDB_538_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- Sequence:loc_3:103092173-103150905_gene_192_pb_229_pos_41699_strand_+_readouts_[NDB_571_c,NDB_556_c,NDB_629_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:56, dumped\n",
      "--- Sequence:loc_3:103092173-103150905_gene_192_pb_230_pos_41746_strand_-_readouts_[NDB_571_c,NDB_556_c,NDB_629_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n",
      "--- Sequence:loc_3:103092173-103150905_gene_192_pb_236_pos_42537_strand_-_readouts_[NDB_571_c,NDB_556_c,NDB_629_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:68, dumped\n",
      "--- Sequence:loc_3:103092173-103150905_gene_192_pb_237_pos_43285_strand_+_readouts_[NDB_571_c,NDB_556_c,NDB_629_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:63, dumped\n",
      "--- Sequence:loc_3:103092173-103150905_gene_192_pb_238_pos_43669_strand_-_readouts_[NDB_571_c,NDB_556_c,NDB_629_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:67, dumped\n",
      "--- Sequence:loc_3:103092173-103150905_gene_192_pb_239_pos_43787_strand_-_readouts_[NDB_571_c,NDB_556_c,NDB_629_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:75, dumped\n",
      "--- Sequence:loc_3:127684895-127739338_gene_202_pb_62_pos_13065_strand_-_readouts_[NDB_551_c,NDB_491_c,NDB_614_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:59, dumped\n",
      "--- Sequence:loc_3:127684895-127739338_gene_202_pb_64_pos_13167_strand_+_readouts_[NDB_551_c,NDB_491_c,NDB_614_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_3:127684895-127739338_gene_202_pb_65_pos_13253_strand_-_readouts_[NDB_551_c,NDB_491_c,NDB_614_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:83, dumped\n",
      "--- Sequence:loc_3:127684895-127739338_gene_202_pb_66_pos_13307_strand_+_readouts_[NDB_551_c,NDB_491_c,NDB_614_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:75, dumped\n",
      "--- Sequence:loc_3:127684895-127739338_gene_202_pb_67_pos_13370_strand_-_readouts_[NDB_551_c,NDB_491_c,NDB_614_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:110, dumped\n",
      "--- Sequence:loc_3:127684895-127739338_gene_202_pb_68_pos_13418_strand_-_readouts_[NDB_551_c,NDB_491_c,NDB_614_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:72, dumped\n",
      "--- Sequence:loc_3:127684895-127739338_gene_202_pb_70_pos_13609_strand_+_readouts_[NDB_551_c,NDB_491_c,NDB_614_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:82, dumped\n",
      "--- Sequence:loc_3:127684895-127739338_gene_202_pb_72_pos_13708_strand_+_readouts_[NDB_551_c,NDB_491_c,NDB_614_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:56, dumped\n",
      "--- Sequence:loc_3:130232712-130315166_gene_204_pb_133_pos_53636_strand_+_readouts_[NDB_511_c,NDB_428_c,NDB_460_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:426, dumped\n",
      "--- Sequence:loc_3:135569252-135673396_gene_206_pb_225_pos_68651_strand_+_readouts_[NDB_464_c,NDB_455_c,NDB_625_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_3:145795719-145916046_gene_208_pb_80_pos_64775_strand_+_readouts_[NDB_458_c,NDB_536_c,NDB_481_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:55, dumped\n",
      "--- Sequence:loc_3:145795719-145916046_gene_208_pb_82_pos_64892_strand_+_readouts_[NDB_458_c,NDB_536_c,NDB_481_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:62, dumped\n",
      "--- Sequence:loc_3:152275792-152343485_gene_210_pb_21_pos_9664_strand_-_readouts_[NDB_403_c,NDB_473_c,NDB_562_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:63, dumped\n",
      "--- Sequence:loc_3:152275792-152343485_gene_210_pb_22_pos_9736_strand_-_readouts_[NDB_403_c,NDB_473_c,NDB_562_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:55, dumped\n",
      "--- Sequence:loc_3:152275792-152343485_gene_210_pb_23_pos_9780_strand_-_readouts_[NDB_403_c,NDB_473_c,NDB_562_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:61, dumped\n",
      "--- Sequence:loc_4:3040005-3100987_gene_212_pb_72_pos_21565_strand_+_readouts_[NDB_506_c,NDB_619_c,NDB_562_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_4:3040005-3100987_gene_212_pb_102_pos_26486_strand_+_readouts_[NDB_506_c,NDB_619_c,NDB_562_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:56, dumped\n",
      "--- Sequence:loc_4:3040005-3100987_gene_212_pb_118_pos_28272_strand_+_readouts_[NDB_506_c,NDB_619_c,NDB_562_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:56, dumped\n",
      "--- Sequence:loc_4:11962627-12051893_gene_218_pb_317_pos_79374_strand_-_readouts_[NDB_487_c,NDB_461_c,NDB_445_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:63, dumped\n",
      "--- Sequence:loc_4:24955708-25039170_gene_221_pb_163_pos_53714_strand_-_readouts_[NDB_505_c,NDB_421_c,NDB_544_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:61, dumped\n",
      "--- Sequence:loc_4:24955708-25039170_gene_221_pb_164_pos_53810_strand_+_readouts_[NDB_505_c,NDB_421_c,NDB_544_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:67, dumped\n",
      "--- Sequence:loc_4:24955708-25039170_gene_221_pb_165_pos_53855_strand_+_readouts_[NDB_505_c,NDB_421_c,NDB_544_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:62, dumped\n",
      "--- Sequence:loc_4:40762947-40854743_gene_229_pb_330_pos_87082_strand_-_readouts_[NDB_578_c,NDB_458_c,NDB_347_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:84, dumped\n",
      "--- Sequence:loc_4:40762947-40854743_gene_229_pb_331_pos_87157_strand_-_readouts_[NDB_578_c,NDB_458_c,NDB_347_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:58, dumped\n",
      "--- Sequence:loc_4:40762947-40854743_gene_229_pb_332_pos_87244_strand_-_readouts_[NDB_578_c,NDB_458_c,NDB_347_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:65, dumped\n",
      "--- Sequence:loc_4:40762947-40854743_gene_229_pb_334_pos_87423_strand_-_readouts_[NDB_578_c,NDB_458_c,NDB_347_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:75, dumped\n",
      "--- Sequence:loc_4:56810348-56866569_gene_235_pb_54_pos_11103_strand_-_readouts_[NDB_536_c,NDB_359_c,NDB_604_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:67, dumped\n",
      "--- Sequence:loc_4:56810348-56866569_gene_235_pb_58_pos_11381_strand_+_readouts_[NDB_536_c,NDB_359_c,NDB_604_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:67, dumped\n",
      "--- Sequence:loc_4:86148471-86220141_gene_243_pb_165_pos_50116_strand_+_readouts_[NDB_400_c,NDB_418_c,NDB_517_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:76, dumped\n",
      "--- Sequence:loc_4:86148471-86220141_gene_243_pb_166_pos_50160_strand_+_readouts_[NDB_400_c,NDB_418_c,NDB_517_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:83, dumped\n",
      "--- Sequence:loc_4:86148471-86220141_gene_243_pb_167_pos_50208_strand_-_readouts_[NDB_400_c,NDB_418_c,NDB_517_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:138, dumped\n",
      "--- Sequence:loc_4:86148471-86220141_gene_243_pb_168_pos_50332_strand_+_readouts_[NDB_400_c,NDB_418_c,NDB_517_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:144, dumped\n",
      "--- Sequence:loc_4:86148471-86220141_gene_243_pb_169_pos_50384_strand_+_readouts_[NDB_400_c,NDB_418_c,NDB_517_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:101, dumped\n",
      "--- Sequence:loc_4:104983954-105060480_gene_250_pb_43_pos_38890_strand_+_readouts_[NDB_482_c,NDB_385_c,NDB_628_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_4:104983954-105060480_gene_250_pb_184_pos_53203_strand_+_readouts_[NDB_482_c,NDB_385_c,NDB_628_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:77, dumped\n",
      "--- Sequence:loc_4:104983954-105060480_gene_250_pb_185_pos_53307_strand_+_readouts_[NDB_482_c,NDB_385_c,NDB_628_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:82, dumped\n",
      "--- Sequence:loc_4:104983954-105060480_gene_250_pb_186_pos_53447_strand_+_readouts_[NDB_482_c,NDB_385_c,NDB_628_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:81, dumped\n",
      "--- Sequence:loc_4:104983954-105060480_gene_250_pb_187_pos_53507_strand_-_readouts_[NDB_482_c,NDB_385_c,NDB_628_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:72, dumped\n",
      "--- Sequence:loc_4:104983954-105060480_gene_250_pb_188_pos_53587_strand_-_readouts_[NDB_482_c,NDB_385_c,NDB_628_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:71, dumped\n",
      "--- Sequence:loc_4:104983954-105060480_gene_250_pb_189_pos_53657_strand_+_readouts_[NDB_482_c,NDB_385_c,NDB_628_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:83, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_0_pos_176_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_3_pos_44169_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_4_pos_44214_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_5_pos_44391_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_6_pos_44469_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_7_pos_44548_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_8_pos_44656_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_9_pos_44733_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_10_pos_44798_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_11_pos_44858_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_12_pos_44905_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_13_pos_45008_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_14_pos_45193_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_15_pos_45260_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_16_pos_45334_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_17_pos_45383_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_18_pos_45513_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_19_pos_45566_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_20_pos_45947_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_21_pos_45993_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_22_pos_46044_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_23_pos_46126_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_24_pos_46170_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_25_pos_46219_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_26_pos_46281_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_27_pos_46345_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_28_pos_46409_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_29_pos_46485_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_30_pos_46529_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_31_pos_46615_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_32_pos_46722_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_33_pos_46804_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_34_pos_46861_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_35_pos_47010_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_36_pos_47054_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_37_pos_47154_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_38_pos_47211_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_39_pos_47276_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_40_pos_47323_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_41_pos_47389_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_42_pos_47460_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_43_pos_47504_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_44_pos_47637_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_45_pos_47681_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_46_pos_47783_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_47_pos_48024_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_48_pos_48070_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_49_pos_48114_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_50_pos_48222_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_51_pos_48318_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_52_pos_48398_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_53_pos_48458_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_54_pos_48556_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_55_pos_48633_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_56_pos_48717_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_57_pos_48783_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_58_pos_48827_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_59_pos_48875_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_60_pos_48947_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_61_pos_49006_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_62_pos_49061_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_63_pos_49105_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_64_pos_49149_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_65_pos_49221_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_66_pos_49292_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_67_pos_49367_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_68_pos_49414_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_69_pos_49468_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_70_pos_49538_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_71_pos_49628_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_72_pos_49684_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_73_pos_49728_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_74_pos_49818_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_75_pos_49871_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_76_pos_49941_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_77_pos_50003_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_78_pos_50075_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_79_pos_50129_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_80_pos_50174_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_81_pos_50218_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_82_pos_50281_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_83_pos_50352_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_84_pos_50396_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_85_pos_50480_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_86_pos_50572_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_87_pos_50616_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_88_pos_50700_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_89_pos_50746_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_90_pos_50812_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_91_pos_50932_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_92_pos_50983_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_93_pos_51086_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_94_pos_51153_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_95_pos_51225_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_96_pos_51326_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_97_pos_51429_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_98_pos_52110_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_99_pos_52154_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_100_pos_52198_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_101_pos_52244_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_102_pos_52321_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_103_pos_52365_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_104_pos_52430_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_105_pos_52493_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_106_pos_52548_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_107_pos_52632_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_108_pos_52685_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_109_pos_52760_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_110_pos_52804_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_111_pos_52848_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_112_pos_52906_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_113_pos_52954_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_114_pos_53051_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_115_pos_53119_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_116_pos_53175_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_117_pos_53225_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_118_pos_53269_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_119_pos_53313_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_120_pos_53371_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_121_pos_53418_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_122_pos_53463_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_123_pos_53599_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_124_pos_53681_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_125_pos_53726_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_126_pos_53827_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_127_pos_53892_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_128_pos_54006_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_129_pos_54080_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_130_pos_54124_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_131_pos_54170_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_132_pos_54214_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_133_pos_54271_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_134_pos_54317_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_135_pos_54377_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_136_pos_54430_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_137_pos_54474_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_138_pos_54537_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_139_pos_54583_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_140_pos_54805_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_141_pos_54849_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_142_pos_54899_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_143_pos_54958_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_144_pos_55002_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_145_pos_55050_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_146_pos_55104_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_147_pos_55167_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_148_pos_55314_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_149_pos_55394_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_150_pos_55447_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_151_pos_55514_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_152_pos_55610_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_153_pos_55654_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_154_pos_55721_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_155_pos_55819_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_156_pos_55863_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_157_pos_55908_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_158_pos_55956_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_159_pos_56000_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_160_pos_56064_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_161_pos_56108_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_162_pos_56154_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_163_pos_56201_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_164_pos_56340_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_165_pos_56384_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_166_pos_56432_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_167_pos_56491_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_168_pos_56596_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_169_pos_56646_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_170_pos_56720_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_171_pos_56764_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_172_pos_56819_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_173_pos_56872_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_174_pos_56956_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_175_pos_57000_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_176_pos_57052_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_177_pos_57129_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_178_pos_57177_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_179_pos_57268_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_180_pos_57321_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_181_pos_57365_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_182_pos_57419_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_183_pos_57531_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_184_pos_57580_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_185_pos_57643_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_186_pos_57778_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_187_pos_57850_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_188_pos_58021_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_189_pos_58124_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_190_pos_58196_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_191_pos_58240_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_192_pos_58284_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_193_pos_58379_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_194_pos_58491_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_195_pos_58562_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_196_pos_58625_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_197_pos_58689_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_198_pos_58754_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_199_pos_58803_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_200_pos_58848_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_201_pos_58896_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_202_pos_59028_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_203_pos_59164_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_204_pos_59213_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_205_pos_59257_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_206_pos_59326_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_207_pos_59394_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_208_pos_59451_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_209_pos_59511_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_210_pos_59636_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_211_pos_59682_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_212_pos_59763_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_213_pos_60217_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_214_pos_60288_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_215_pos_60357_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_216_pos_60435_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_217_pos_60513_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_218_pos_60580_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_219_pos_60626_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_220_pos_60674_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_221_pos_60756_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_222_pos_60840_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_223_pos_60898_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_224_pos_60942_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_225_pos_60992_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_226_pos_61037_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_227_pos_61082_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_228_pos_61162_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_229_pos_61209_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_230_pos_61255_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_231_pos_61300_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_232_pos_61365_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_233_pos_61412_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_234_pos_61469_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_235_pos_61525_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_236_pos_61569_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_237_pos_61656_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_238_pos_61710_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_239_pos_61770_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_240_pos_61864_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_241_pos_61951_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_242_pos_62006_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_243_pos_62050_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_244_pos_62117_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_245_pos_62166_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_246_pos_62230_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_247_pos_62303_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_248_pos_62347_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_249_pos_62402_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_250_pos_62475_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_251_pos_62519_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_252_pos_62645_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_253_pos_62821_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_254_pos_62869_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_255_pos_62950_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_256_pos_62994_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_257_pos_63038_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_258_pos_63238_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_259_pos_63282_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_260_pos_63534_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_261_pos_63594_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_262_pos_63648_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_263_pos_63693_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_264_pos_63738_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_265_pos_63810_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_266_pos_63989_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_267_pos_64065_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_268_pos_64140_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_269_pos_64184_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_270_pos_64229_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_274_pos_74395_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_275_pos_74468_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_276_pos_74515_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_277_pos_74790_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_278_pos_74880_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_279_pos_74925_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_280_pos_74976_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_281_pos_75033_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_282_pos_75092_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_283_pos_75136_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_284_pos_75298_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_285_pos_75342_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_286_pos_75408_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_287_pos_75498_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_288_pos_75624_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_289_pos_75670_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_290_pos_75749_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_291_pos_75793_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_292_pos_75886_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_293_pos_75963_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_294_pos_76039_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_295_pos_76098_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_296_pos_76175_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_297_pos_76264_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_298_pos_76310_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_299_pos_76373_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_300_pos_76419_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_301_pos_76486_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_302_pos_76537_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_303_pos_76585_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_304_pos_76629_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_305_pos_76692_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_306_pos_76736_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_307_pos_76782_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_308_pos_76833_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_309_pos_76923_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_310_pos_76999_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_311_pos_77082_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_312_pos_77137_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_313_pos_77181_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_314_pos_77229_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_315_pos_77274_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_316_pos_77318_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_317_pos_77433_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_318_pos_77512_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_319_pos_77575_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_320_pos_77619_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_321_pos_77663_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_322_pos_77716_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_323_pos_77793_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_324_pos_77862_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_325_pos_77912_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_326_pos_77992_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_327_pos_78086_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_328_pos_78200_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_329_pos_78244_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_330_pos_78298_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_331_pos_78359_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_332_pos_78443_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_333_pos_78563_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_334_pos_78607_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_335_pos_78672_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_336_pos_78716_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_337_pos_78771_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_338_pos_78815_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_339_pos_78884_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_340_pos_78937_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_341_pos_78991_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_342_pos_79056_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_343_pos_79126_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_344_pos_79198_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_345_pos_79278_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_346_pos_79342_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_347_pos_79402_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_348_pos_79446_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_349_pos_79503_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_350_pos_79576_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_351_pos_79622_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_352_pos_79671_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_353_pos_79723_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_354_pos_79859_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_355_pos_79972_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_356_pos_80087_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_358_pos_80312_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_359_pos_80379_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_360_pos_80423_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_361_pos_80468_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_362_pos_80557_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_363_pos_80615_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_364_pos_80659_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_365_pos_80719_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_370_pos_81330_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_371_pos_81456_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_372_pos_81572_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_373_pos_81667_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_374_pos_81743_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_375_pos_81802_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_376_pos_81846_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_377_pos_81900_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_378_pos_81959_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_379_pos_82034_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_380_pos_82130_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_381_pos_82219_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_382_pos_82280_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_383_pos_82324_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_384_pos_82428_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_385_pos_82563_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_386_pos_82652_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_387_pos_82700_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_388_pos_82759_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_389_pos_82822_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_390_pos_82888_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_391_pos_82961_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_392_pos_83016_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_393_pos_83089_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_394_pos_83133_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_395_pos_83181_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_396_pos_83228_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_397_pos_83272_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_398_pos_83317_strand_+_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:114065922-114149495_gene_751_pb_399_pos_83382_strand_-_readouts_[NDB_427_c,NDB_356_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- Sequence:loc_14:31127141-31206382_gene_766_pb_127_pos_47424_strand_-_readouts_[NDB_554_c,NDB_473_c,NDB_544_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:58, dumped\n",
      "--- Sequence:loc_14:31127141-31206382_gene_766_pb_128_pos_47468_strand_-_readouts_[NDB_554_c,NDB_473_c,NDB_544_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:61, dumped\n",
      "--- Sequence:loc_14:31127141-31206382_gene_766_pb_129_pos_47550_strand_-_readouts_[NDB_554_c,NDB_473_c,NDB_544_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:85, dumped\n",
      "--- Sequence:loc_14:40891564-40969316_gene_770_pb_280_pos_69786_strand_-_readouts_[NDB_458_c,NDB_514_c,NDB_359_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:78, dumped\n",
      "--- Sequence:loc_14:40891564-40969316_gene_770_pb_281_pos_69943_strand_+_readouts_[NDB_458_c,NDB_514_c,NDB_359_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:93, dumped\n",
      "--- Sequence:loc_14:40891564-40969316_gene_770_pb_282_pos_69991_strand_+_readouts_[NDB_458_c,NDB_514_c,NDB_359_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:94, dumped\n",
      "--- Sequence:loc_14:40891564-40969316_gene_770_pb_283_pos_70035_strand_+_readouts_[NDB_458_c,NDB_514_c,NDB_359_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:79, dumped\n",
      "--- Sequence:loc_14:40891564-40969316_gene_770_pb_284_pos_70084_strand_+_readouts_[NDB_458_c,NDB_514_c,NDB_359_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_14:46891206-47000060_gene_773_pb_225_pos_72483_strand_+_readouts_[NDB_461_c,NDB_517_c,NDB_614_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:101, dumped\n",
      "--- Sequence:loc_14:46891206-47000060_gene_773_pb_226_pos_72527_strand_+_readouts_[NDB_461_c,NDB_517_c,NDB_614_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:56, dumped\n",
      "--- Sequence:loc_14:46891206-47000060_gene_773_pb_227_pos_72571_strand_+_readouts_[NDB_461_c,NDB_517_c,NDB_614_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:78, dumped\n",
      "--- Sequence:loc_14:46891206-47000060_gene_773_pb_228_pos_72692_strand_-_readouts_[NDB_461_c,NDB_517_c,NDB_614_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:61, dumped\n",
      "--- Sequence:loc_14:46891206-47000060_gene_773_pb_229_pos_72845_strand_+_readouts_[NDB_461_c,NDB_517_c,NDB_614_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:88, dumped\n",
      "--- Sequence:loc_14:46891206-47000060_gene_773_pb_231_pos_73036_strand_+_readouts_[NDB_461_c,NDB_517_c,NDB_614_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:61, dumped\n",
      "--- Sequence:loc_14:46891206-47000060_gene_773_pb_233_pos_73139_strand_+_readouts_[NDB_461_c,NDB_517_c,NDB_614_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_14:57498975-57569853_gene_777_pb_260_pos_55750_strand_+_readouts_[NDB_398_c,NDB_421_c,NDB_608_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n",
      "--- Sequence:loc_14:57498975-57569853_gene_777_pb_261_pos_55886_strand_+_readouts_[NDB_398_c,NDB_421_c,NDB_608_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:56, dumped\n",
      "--- Sequence:loc_14:57498975-57569853_gene_777_pb_307_pos_60307_strand_-_readouts_[NDB_398_c,NDB_421_c,NDB_608_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:60, dumped\n",
      "--- Sequence:loc_14:57498975-57569853_gene_777_pb_308_pos_60837_strand_-_readouts_[NDB_398_c,NDB_421_c,NDB_608_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:64, dumped\n",
      "--- Sequence:loc_14:57498975-57569853_gene_777_pb_309_pos_60912_strand_+_readouts_[NDB_398_c,NDB_421_c,NDB_608_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:63, dumped\n",
      "--- Sequence:loc_14:65960189-65998537_gene_789_pb_371_pos_36121_strand_+_readouts_[NDB_443_c,NDB_601_c,NDB_563_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_14:65960189-65998537_gene_789_pb_374_pos_36277_strand_+_readouts_[NDB_443_c,NDB_601_c,NDB_563_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_14:76415881-76490537_gene_795_pb_382_pos_73110_strand_-_readouts_[NDB_427_c,NDB_446_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_14:76415881-76490537_gene_795_pb_383_pos_73200_strand_-_readouts_[NDB_427_c,NDB_446_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_14:99526543-99602180_gene_799_pb_174_pos_56975_strand_-_readouts_[NDB_536_c,NDB_614_c,NDB_538_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:77, dumped\n",
      "--- Sequence:loc_14:99526543-99602180_gene_799_pb_175_pos_57019_strand_+_readouts_[NDB_536_c,NDB_614_c,NDB_538_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:104, dumped\n",
      "--- Sequence:loc_14:99526543-99602180_gene_799_pb_176_pos_57065_strand_+_readouts_[NDB_536_c,NDB_614_c,NDB_538_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_14:99526543-99602180_gene_799_pb_177_pos_57109_strand_+_readouts_[NDB_536_c,NDB_614_c,NDB_538_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:74, dumped\n",
      "--- Sequence:loc_14:99526543-99602180_gene_799_pb_178_pos_57153_strand_+_readouts_[NDB_536_c,NDB_614_c,NDB_538_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:83, dumped\n",
      "--- Sequence:loc_14:99526543-99602180_gene_799_pb_179_pos_57225_strand_-_readouts_[NDB_536_c,NDB_614_c,NDB_538_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:89, dumped\n",
      "--- Sequence:loc_14:119621024-119697288_gene_807_pb_216_pos_53202_strand_-_readouts_[NDB_511_c,NDB_458_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:64, dumped\n",
      "--- Sequence:loc_14:119621024-119697288_gene_807_pb_371_pos_66858_strand_+_readouts_[NDB_511_c,NDB_458_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:56, dumped\n",
      "--- Sequence:loc_14:119621024-119697288_gene_807_pb_378_pos_67247_strand_+_readouts_[NDB_511_c,NDB_458_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:56, dumped\n",
      "--- Sequence:loc_14:121918265-122090024_gene_809_pb_15_pos_89888_strand_+_readouts_[NDB_403_c,NDB_526_c,NDB_395_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:71, dumped\n",
      "--- Sequence:loc_14:121918265-122090024_gene_809_pb_17_pos_90006_strand_+_readouts_[NDB_403_c,NDB_526_c,NDB_395_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:64, dumped\n",
      "--- Sequence:loc_14:121918265-122090024_gene_809_pb_18_pos_90053_strand_+_readouts_[NDB_403_c,NDB_526_c,NDB_395_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:61, dumped\n",
      "--- Sequence:loc_15:7170479-7236680_gene_811_pb_145_pos_45053_strand_-_readouts_[NDB_388_c,NDB_526_c,NDB_544_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:61, dumped\n",
      "--- Sequence:loc_15:7170479-7236680_gene_811_pb_146_pos_45128_strand_-_readouts_[NDB_388_c,NDB_526_c,NDB_544_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:61, dumped\n",
      "--- Sequence:loc_15:7170479-7236680_gene_811_pb_147_pos_45205_strand_+_readouts_[NDB_388_c,NDB_526_c,NDB_544_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_15:12417846-12507837_gene_813_pb_156_pos_55163_strand_-_readouts_[NDB_332_c,NDB_385_c,NDB_581_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:67, dumped\n",
      "--- Sequence:loc_15:12417846-12507837_gene_813_pb_157_pos_55224_strand_+_readouts_[NDB_332_c,NDB_385_c,NDB_581_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:70, dumped\n",
      "--- Sequence:loc_15:12417846-12507837_gene_813_pb_158_pos_55310_strand_+_readouts_[NDB_332_c,NDB_385_c,NDB_581_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:81, dumped\n",
      "--- Sequence:loc_15:12417846-12507837_gene_813_pb_160_pos_55402_strand_+_readouts_[NDB_332_c,NDB_385_c,NDB_581_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:83, dumped\n",
      "--- Sequence:loc_15:30249955-30317077_gene_819_pb_196_pos_49159_strand_-_readouts_[NDB_418_c,NDB_622_c,NDB_350_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:60, dumped\n",
      "--- Sequence:loc_15:30249955-30317077_gene_819_pb_197_pos_49203_strand_-_readouts_[NDB_418_c,NDB_622_c,NDB_350_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_15:30249955-30317077_gene_819_pb_201_pos_49400_strand_+_readouts_[NDB_418_c,NDB_622_c,NDB_350_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_15:30249955-30317077_gene_819_pb_202_pos_49477_strand_-_readouts_[NDB_418_c,NDB_622_c,NDB_350_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:65, dumped\n",
      "--- Sequence:loc_15:30249955-30317077_gene_819_pb_204_pos_49576_strand_+_readouts_[NDB_418_c,NDB_622_c,NDB_350_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:61, dumped\n",
      "--- Sequence:loc_15:36478751-36528020_gene_820_pb_32_pos_6367_strand_-_readouts_[NDB_403_c,NDB_335_c,NDB_421_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_15:36478751-36528020_gene_820_pb_33_pos_6428_strand_-_readouts_[NDB_403_c,NDB_335_c,NDB_421_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:55, dumped\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- Sequence:loc_15:58158195-58230642_gene_826_pb_368_pos_68569_strand_+_readouts_[NDB_476_c,NDB_601_c,NDB_608_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_15:58158195-58230642_gene_826_pb_369_pos_68617_strand_+_readouts_[NDB_476_c,NDB_601_c,NDB_608_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_15:81672506-81729092_gene_835_pb_229_pos_42048_strand_+_readouts_[NDB_554_c,NDB_619_c,NDB_395_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:67, dumped\n",
      "--- Sequence:loc_15:81672506-81729092_gene_835_pb_230_pos_42136_strand_+_readouts_[NDB_554_c,NDB_619_c,NDB_395_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:66, dumped\n",
      "--- Sequence:loc_15:81672506-81729092_gene_835_pb_232_pos_42438_strand_+_readouts_[NDB_554_c,NDB_619_c,NDB_395_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:64, dumped\n",
      "--- Sequence:loc_15:92180253-92287287_gene_839_pb_13_pos_54258_strand_-_readouts_[NDB_379_c,NDB_386_c,NDB_556_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:142, dumped\n",
      "--- Sequence:loc_16:22756585-22854730_gene_848_pb_154_pos_59895_strand_+_readouts_[NDB_533_c,NDB_514_c,NDB_538_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:88, dumped\n",
      "--- Sequence:loc_16:22756585-22854730_gene_848_pb_155_pos_60004_strand_+_readouts_[NDB_533_c,NDB_514_c,NDB_538_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:84, dumped\n",
      "--- Sequence:loc_16:22756585-22854730_gene_848_pb_156_pos_60074_strand_-_readouts_[NDB_533_c,NDB_514_c,NDB_538_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:120, dumped\n",
      "--- Sequence:loc_16:22756585-22854730_gene_848_pb_157_pos_60269_strand_+_readouts_[NDB_533_c,NDB_514_c,NDB_538_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:107, dumped\n",
      "--- Sequence:loc_16:22756585-22854730_gene_848_pb_158_pos_60356_strand_+_readouts_[NDB_533_c,NDB_514_c,NDB_538_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:62, dumped\n",
      "--- Sequence:loc_16:22756585-22854730_gene_848_pb_159_pos_60430_strand_+_readouts_[NDB_533_c,NDB_514_c,NDB_538_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:76, dumped\n",
      "--- Sequence:loc_16:22756585-22854730_gene_848_pb_161_pos_60597_strand_+_readouts_[NDB_533_c,NDB_514_c,NDB_538_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:121, dumped\n",
      "--- Sequence:loc_16:22756585-22854730_gene_848_pb_162_pos_60672_strand_+_readouts_[NDB_533_c,NDB_514_c,NDB_538_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:110, dumped\n",
      "--- Sequence:loc_16:38150332-38259251_gene_860_pb_175_pos_75793_strand_+_readouts_[NDB_472_c,NDB_608_c,NDB_563_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:64, dumped\n",
      "--- Sequence:loc_16:77374249-77459818_gene_872_pb_318_pos_77482_strand_+_readouts_[NDB_365_c,NDB_479_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_16:77374249-77459818_gene_872_pb_319_pos_77564_strand_+_readouts_[NDB_365_c,NDB_479_c,NDB_368_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_16:89507846-89608464_gene_876_pb_71_pos_58612_strand_-_readouts_[NDB_544_c,NDB_581_c,NDB_562_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_16:90915327-91010712_gene_877_pb_257_pos_70244_strand_-_readouts_[NDB_376_c,NDB_592_c,NDB_629_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:72, dumped\n",
      "--- Sequence:loc_16:90915327-91010712_gene_877_pb_258_pos_70331_strand_-_readouts_[NDB_376_c,NDB_592_c,NDB_629_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:80, dumped\n",
      "--- Sequence:loc_16:90915327-91010712_gene_877_pb_260_pos_70449_strand_-_readouts_[NDB_376_c,NDB_592_c,NDB_629_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:79, dumped\n",
      "--- Sequence:loc_16:90915327-91010712_gene_877_pb_261_pos_70493_strand_+_readouts_[NDB_376_c,NDB_592_c,NDB_629_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_17:6758025-6822752_gene_883_pb_248_pos_51293_strand_+_readouts_[NDB_511_c,NDB_455_c,NDB_445_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:426, dumped\n",
      "--- Sequence:loc_17:35516761-35560061_gene_891_pb_340_pos_38187_strand_+_readouts_[NDB_400_c,NDB_386_c,NDB_614_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:58, dumped\n",
      "--- Sequence:loc_17:35516761-35560061_gene_891_pb_341_pos_38246_strand_+_readouts_[NDB_400_c,NDB_386_c,NDB_614_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_17:48532940-48612895_gene_896_pb_381_pos_78476_strand_+_readouts_[NDB_578_c,NDB_379_c,NDB_401_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:69, dumped\n",
      "--- Sequence:loc_17:48532940-48612895_gene_896_pb_382_pos_78527_strand_+_readouts_[NDB_578_c,NDB_379_c,NDB_401_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:89, dumped\n",
      "--- Sequence:loc_17:56872991-56939799_gene_903_pb_238_pos_52638_strand_+_readouts_[NDB_505_c,NDB_589_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:87, dumped\n",
      "--- Sequence:loc_17:56872991-56939799_gene_903_pb_239_pos_52736_strand_+_readouts_[NDB_505_c,NDB_589_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:79, dumped\n",
      "--- Sequence:loc_17:56872991-56939799_gene_903_pb_240_pos_52844_strand_-_readouts_[NDB_505_c,NDB_589_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:92, dumped\n",
      "--- Sequence:loc_17:56872991-56939799_gene_903_pb_241_pos_52890_strand_-_readouts_[NDB_505_c,NDB_589_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:58, dumped\n",
      "--- Sequence:loc_17:56872991-56939799_gene_903_pb_242_pos_52948_strand_-_readouts_[NDB_505_c,NDB_589_c,NDB_479_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:87, dumped\n",
      "--- Sequence:loc_17:62630369-62740980_gene_905_pb_173_pos_69908_strand_+_readouts_[NDB_391_c,NDB_397_c,NDB_563_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:82, dumped\n",
      "--- Sequence:loc_17:73209957-73282539_gene_907_pb_204_pos_52741_strand_+_readouts_[NDB_464_c,NDB_487_c,NDB_443_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:65, dumped\n",
      "--- Sequence:loc_17:84464454-84530370_gene_912_pb_198_pos_42297_strand_+_readouts_[NDB_508_c,NDB_452_c,NDB_551_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:58, dumped\n",
      "--- Sequence:loc_17:84464454-84530370_gene_912_pb_200_pos_42490_strand_+_readouts_[NDB_508_c,NDB_452_c,NDB_551_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:55, dumped\n",
      "--- Sequence:loc_17:84850274-84953300_gene_913_pb_338_pos_96829_strand_-_readouts_[NDB_433_c,NDB_446_c,NDB_428_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_17:84850274-84953300_gene_913_pb_340_pos_96975_strand_-_readouts_[NDB_433_c,NDB_446_c,NDB_428_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:66, dumped\n",
      "--- Sequence:loc_17:90044561-90126527_gene_915_pb_33_pos_39048_strand_+_readouts_[NDB_472_c,NDB_614_c,NDB_395_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:100, dumped\n",
      "--- Sequence:loc_17:90044561-90126527_gene_915_pb_35_pos_39154_strand_-_readouts_[NDB_472_c,NDB_614_c,NDB_395_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:110, dumped\n",
      "--- Sequence:loc_17:90044561-90126527_gene_915_pb_36_pos_39208_strand_-_readouts_[NDB_472_c,NDB_614_c,NDB_395_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:114, dumped\n",
      "--- Sequence:loc_17:90044561-90126527_gene_915_pb_37_pos_39275_strand_+_readouts_[NDB_472_c,NDB_614_c,NDB_395_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:107, dumped\n",
      "--- Sequence:loc_18:15391109-15452687_gene_918_pb_205_pos_43146_strand_+_readouts_[NDB_464_c,NDB_409_c,NDB_401_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:85, dumped\n",
      "--- Sequence:loc_18:15391109-15452687_gene_918_pb_206_pos_43430_strand_+_readouts_[NDB_464_c,NDB_409_c,NDB_401_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:84, dumped\n",
      "--- Sequence:loc_18:15391109-15452687_gene_918_pb_207_pos_43474_strand_-_readouts_[NDB_464_c,NDB_409_c,NDB_401_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:81, dumped\n",
      "--- Sequence:loc_18:15391109-15452687_gene_918_pb_208_pos_43526_strand_+_readouts_[NDB_464_c,NDB_409_c,NDB_401_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:58, dumped\n",
      "--- Sequence:loc_18:24408559-24500784_gene_921_pb_40_pos_46295_strand_+_readouts_[NDB_431_c,NDB_547_c,NDB_494_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:65, dumped\n",
      "--- Sequence:loc_18:24408559-24500784_gene_921_pb_41_pos_46340_strand_+_readouts_[NDB_431_c,NDB_547_c,NDB_494_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:66, dumped\n",
      "--- Sequence:loc_18:24408559-24500784_gene_921_pb_42_pos_46458_strand_+_readouts_[NDB_431_c,NDB_547_c,NDB_494_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:72, dumped\n",
      "--- Sequence:loc_18:24408559-24500784_gene_921_pb_43_pos_46502_strand_+_readouts_[NDB_431_c,NDB_547_c,NDB_494_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:59, dumped\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- Sequence:loc_18:53219139-53335927_gene_937_pb_265_pos_84933_strand_-_readouts_[NDB_356_c,NDB_418_c,NDB_625_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:55, dumped\n",
      "--- Sequence:loc_18:53219139-53335927_gene_937_pb_268_pos_85385_strand_-_readouts_[NDB_356_c,NDB_418_c,NDB_625_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:58, dumped\n",
      "--- Sequence:loc_18:53219139-53335927_gene_937_pb_269_pos_85468_strand_-_readouts_[NDB_356_c,NDB_418_c,NDB_625_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:58, dumped\n",
      "--- Sequence:loc_18:63719174-63787284_gene_941_pb_296_pos_57281_strand_+_readouts_[NDB_536_c,NDB_539_c,NDB_628_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n",
      "--- Sequence:loc_18:63719174-63787284_gene_941_pb_298_pos_57390_strand_-_readouts_[NDB_536_c,NDB_539_c,NDB_628_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:56, dumped\n",
      "--- Sequence:loc_18:77958807-78042826_gene_947_pb_5_pos_43354_strand_-_readouts_[NDB_347_c,NDB_491_c,NDB_547_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:107, dumped\n",
      "--- Sequence:loc_18:77958807-78042826_gene_947_pb_6_pos_43417_strand_+_readouts_[NDB_347_c,NDB_491_c,NDB_547_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:77, dumped\n",
      "--- Sequence:loc_18:77958807-78042826_gene_947_pb_7_pos_43477_strand_-_readouts_[NDB_347_c,NDB_491_c,NDB_547_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:60, dumped\n",
      "--- Sequence:loc_18:77958807-78042826_gene_947_pb_8_pos_43567_strand_-_readouts_[NDB_347_c,NDB_491_c,NDB_547_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:77, dumped\n",
      "--- Sequence:loc_18:77958807-78042826_gene_947_pb_9_pos_43650_strand_-_readouts_[NDB_347_c,NDB_491_c,NDB_547_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:63, dumped\n",
      "--- Sequence:loc_18:77958807-78042826_gene_947_pb_10_pos_43713_strand_-_readouts_[NDB_347_c,NDB_491_c,NDB_547_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:63, dumped\n",
      "--- Sequence:loc_18:77958807-78042826_gene_947_pb_11_pos_43797_strand_-_readouts_[NDB_347_c,NDB_491_c,NDB_547_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:130, dumped\n",
      "--- Sequence:loc_18:77958807-78042826_gene_947_pb_12_pos_43866_strand_-_readouts_[NDB_347_c,NDB_491_c,NDB_547_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:104, dumped\n",
      "--- Sequence:loc_18:77958807-78042826_gene_947_pb_187_pos_57505_strand_+_readouts_[NDB_347_c,NDB_491_c,NDB_547_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:64, dumped\n",
      "--- Sequence:loc_18:77958807-78042826_gene_947_pb_188_pos_57920_strand_-_readouts_[NDB_347_c,NDB_491_c,NDB_547_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:61, dumped\n",
      "--- Sequence:loc_18:77958807-78042826_gene_947_pb_189_pos_58074_strand_+_readouts_[NDB_347_c,NDB_491_c,NDB_547_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:68, dumped\n",
      "--- Sequence:loc_18:77958807-78042826_gene_947_pb_190_pos_58631_strand_-_readouts_[NDB_347_c,NDB_491_c,NDB_547_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:55, dumped\n",
      "--- Sequence:loc_18:77958807-78042826_gene_947_pb_191_pos_58695_strand_+_readouts_[NDB_347_c,NDB_491_c,NDB_547_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_18:77958807-78042826_gene_947_pb_192_pos_58748_strand_-_readouts_[NDB_347_c,NDB_491_c,NDB_547_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:76, dumped\n",
      "--- Sequence:loc_19:5638442-5685296_gene_952_pb_204_pos_32518_strand_+_readouts_[NDB_403_c,NDB_616_c,NDB_619_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_19:5638442-5685296_gene_952_pb_302_pos_39727_strand_-_readouts_[NDB_403_c,NDB_616_c,NDB_619_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:74, dumped\n",
      "--- Sequence:loc_19:5638442-5685296_gene_952_pb_303_pos_39982_strand_+_readouts_[NDB_403_c,NDB_616_c,NDB_619_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:60, dumped\n",
      "--- Sequence:loc_19:5638442-5685296_gene_952_pb_304_pos_40066_strand_+_readouts_[NDB_403_c,NDB_616_c,NDB_619_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:62, dumped\n",
      "--- Sequence:loc_19:5638442-5685296_gene_952_pb_305_pos_40112_strand_-_readouts_[NDB_403_c,NDB_616_c,NDB_619_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:72, dumped\n",
      "--- Sequence:loc_19:5638442-5685296_gene_952_pb_314_pos_40827_strand_+_readouts_[NDB_403_c,NDB_616_c,NDB_619_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_19:7054548-7207140_gene_954_pb_146_pos_96260_strand_-_readouts_[NDB_452_c,NDB_587_c,NDB_581_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:93, dumped\n",
      "--- Sequence:loc_19:7054548-7207140_gene_954_pb_147_pos_96310_strand_-_readouts_[NDB_452_c,NDB_587_c,NDB_581_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:72, dumped\n",
      "--- Sequence:loc_19:7054548-7207140_gene_954_pb_150_pos_96500_strand_-_readouts_[NDB_452_c,NDB_587_c,NDB_581_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:101, dumped\n",
      "--- Sequence:loc_19:10685617-10740864_gene_956_pb_83_pos_31912_strand_-_readouts_[NDB_356_c,NDB_421_c,NDB_556_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:70, dumped\n",
      "--- Sequence:loc_19:10685617-10740864_gene_956_pb_84_pos_32022_strand_-_readouts_[NDB_356_c,NDB_421_c,NDB_556_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:100, dumped\n",
      "--- Sequence:loc_19:10685617-10740864_gene_956_pb_85_pos_32098_strand_-_readouts_[NDB_356_c,NDB_421_c,NDB_556_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:95, dumped\n",
      "--- Sequence:loc_19:10685617-10740864_gene_956_pb_86_pos_32164_strand_-_readouts_[NDB_356_c,NDB_421_c,NDB_556_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:97, dumped\n",
      "--- Sequence:loc_19:10685617-10740864_gene_956_pb_87_pos_32208_strand_-_readouts_[NDB_356_c,NDB_421_c,NDB_556_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:96, dumped\n",
      "--- Sequence:loc_19:17296782-17355629_gene_960_pb_198_pos_40510_strand_-_readouts_[NDB_466_c,NDB_359_c,NDB_538_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:118, dumped\n",
      "--- Sequence:loc_19:41963511-42036554_gene_978_pb_142_pos_48677_strand_+_readouts_[NDB_398_c,NDB_628_c,NDB_601_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:66, dumped\n",
      "--- Sequence:loc_19:41963511-42036554_gene_978_pb_144_pos_48784_strand_-_readouts_[NDB_398_c,NDB_628_c,NDB_601_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:58, dumped\n",
      "--- Sequence:loc_19:41963511-42036554_gene_978_pb_146_pos_48930_strand_+_readouts_[NDB_398_c,NDB_628_c,NDB_601_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_19:41963511-42036554_gene_978_pb_148_pos_49084_strand_-_readouts_[NDB_398_c,NDB_628_c,NDB_601_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:75, dumped\n",
      "--- Sequence:loc_19:41963511-42036554_gene_978_pb_149_pos_49163_strand_-_readouts_[NDB_398_c,NDB_628_c,NDB_601_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:71, dumped\n",
      "--- Sequence:loc_19:41963511-42036554_gene_978_pb_150_pos_49222_strand_+_readouts_[NDB_398_c,NDB_628_c,NDB_601_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:94, dumped\n",
      "--- Sequence:loc_19:45861417-45967203_gene_979_pb_102_pos_62314_strand_+_readouts_[NDB_533_c,NDB_332_c,NDB_472_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:65, dumped\n",
      "--- Sequence:loc_19:59369976-59445429_gene_984_pb_190_pos_51956_strand_+_readouts_[NDB_529_c,NDB_577_c,NDB_481_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_19:59369976-59445429_gene_984_pb_191_pos_52059_strand_-_readouts_[NDB_529_c,NDB_577_c,NDB_481_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "-- total probes removed by internal screening: 1416\n"
     ]
    }
   ],
   "source": [
    "# check between probes\n",
    "kept_records, removed_count = check._check_between_probes(full_records, int_map, \n",
    "                                                          primer_len=primer_len, \n",
    "                                                          target_len=target_len)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [],
   "source": [
    "del(int_map)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 4.1.6 finish checking, save"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [],
   "source": [
    "# save kept records\n",
    "with open(os.path.join(library_folder, 'filtered_full_probes.fasta'), 'w') as output_handle:\n",
    "    SeqIO.write(kept_records, output_handle, \"fasta\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 4.2 BLAST screening of probes\n",
    "\n",
    "the local database is created as follows (with first downloading the Mus_musculus.GRCm38.dna_sm.merged.fa in the path as indicated)\n",
    "\n",
    "example for creating the local blast database by:\n",
    "```bash\n",
    "makeblastdb -in C:\\Users\\puzheng\\Documents\\Tools\\lowercase_masked\\Mus_musculus.GRCm38.dna_sm.merged.fa -dbtype nucl -parse_seqids -out GRCm38_ensembl -title \"Mouse chromosomes soft masked\" -blastdb_version 4![image.png](attachment:image.png)\n",
    "```\n",
    "\n",
    "The outputs are:\n",
    "```bash\n",
    "Building a new DB, current time: 03/30/2021 21:49:18\n",
    "New DB name:   C:\\Users\\puzheng\\Documents\\Tools\\GRCm38_ensembl\n",
    "New DB title:  Mouse chromosomes soft masked\n",
    "Sequence type: Nucleotide\n",
    "Keep MBits: T\n",
    "Maximum file size: 1000000000B\n",
    "Adding sequences from FASTA; added 66 sequences in 42.2668 seconds.\n",
    "```\n",
    "\n",
    "\n",
    "these pre-generated blast files are now located in the NAS2.\n",
    "\n",
    "blast_database = os.path.join(reference_folder, 'blast_database', 'GRCm38_ensembl')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "\n",
    "\n",
    "\n",
    "Other tips for running and installing the blastn\n",
    "\n",
    "Installation instrucntion:\n",
    "see instruction: https://www.ncbi.nlm.nih.gov/books/NBK52637/\n",
    "\n",
    "in cmd using the below command (refer to instrcution for how to use)\n",
    "\n",
    "\n",
    "use cmd to open jupyter so that blastn -h can be found by jupyter (need to use cmd to restart jupyter after first-time installing)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "USAGE\n",
      "  blastn [-h] [-help] [-import_search_strategy filename]\n",
      "    [-export_search_strategy filename] [-task task_name] [-db database_name]\n",
      "    [-dbsize num_letters] [-gilist filename] [-seqidlist filename]\n",
      "    [-negative_gilist filename] [-negative_seqidlist filename]\n",
      "    [-taxids taxids] [-negative_taxids taxids] [-taxidlist filename]\n",
      "    [-negative_taxidlist filename] [-entrez_query entrez_query]\n",
      "    [-db_soft_mask filtering_algorithm] [-db_hard_mask filtering_algorithm]\n",
      "    [-subject subject_input_file] [-subject_loc range] [-query input_file]\n",
      "    [-out output_file] [-evalue evalue] [-word_size int_value]\n",
      "    [-gapopen open_penalty] [-gapextend extend_penalty]\n",
      "    [-perc_identity float_value] [-qcov_hsp_perc float_value]\n",
      "    [-max_hsps int_value] [-xdrop_ungap float_value] [-xdrop_gap float_value]\n",
      "    [-xdrop_gap_final float_value] [-searchsp int_value]\n",
      "    [-sum_stats bool_value] [-penalty penalty] [-reward reward] [-no_greedy]\n",
      "    [-min_raw_gapped_score int_value] [-template_type type]\n",
      "    [-template_length int_value] [-dust DUST_options]\n",
      "    [-filtering_db filtering_database]\n",
      "    [-window_masker_taxid window_masker_taxid]\n",
      "    [-window_masker_db window_masker_db] [-soft_masking soft_masking]\n",
      "    [-ungapped] [-culling_limit int_value] [-best_hit_overhang float_value]\n",
      "    [-best_hit_score_edge float_value] [-subject_besthit]\n",
      "    [-window_size int_value] [-off_diagonal_range int_value]\n",
      "    [-use_index boolean] [-index_name string] [-lcase_masking]\n",
      "    [-query_loc range] [-strand strand] [-parse_deflines] [-outfmt format]\n",
      "    [-show_gis] [-num_descriptions int_value] [-num_alignments int_value]\n",
      "    [-line_length line_length] [-html] [-sorthits sort_hits]\n",
      "    [-sorthsps sort_hsps] [-max_target_seqs num_sequences]\n",
      "    [-num_threads int_value] [-mt_mode int_value] [-remote] [-version]\n",
      "\n",
      "DESCRIPTION\n",
      "   Nucleotide-Nucleotide BLAST 2.12.0+\n",
      "\n",
      "Use '-help' to print detailed descriptions of command line arguments\n"
     ]
    }
   ],
   "source": [
    "# if blastn is installed and can be found by jupyter, the below command will show the help page for blastn\n",
    "!blastn -h"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Blast probe against \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\blast_database\\GRCm38_ensembl\n",
      "-- number of probes loaded: 383682\n",
      "-- processing region: 1 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_1.fasta\n",
      "--- *blasting region: 1\n",
      "--- total time for blast 1: 24.73100185394287\n",
      "-- processing region: 10 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_10.fasta\n",
      "--- *blasting region: 10\n",
      "--- total time for blast 10: 24.191994190216064\n",
      "-- processing region: 100 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_100.fasta\n",
      "--- *blasting region: 100\n",
      "--- total time for blast 100: 26.69500231742859\n",
      "-- processing region: 101 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_101.fasta\n",
      "--- *blasting region: 101\n",
      "--- total time for blast 101: 20.381998538970947\n",
      "-- processing region: 102 with 399 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_102.fasta\n",
      "--- *blasting region: 102\n",
      "--- total time for blast 102: 18.812002182006836\n",
      "-- processing region: 103 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_103.fasta\n",
      "--- *blasting region: 103\n",
      "--- total time for blast 103: 18.85996150970459\n",
      "-- processing region: 104 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_104.fasta\n",
      "--- *blasting region: 104\n",
      "--- total time for blast 104: 23.59799885749817\n",
      "-- processing region: 105 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_105.fasta\n",
      "--- *blasting region: 105\n",
      "--- total time for blast 105: 21.662986278533936\n",
      "-- processing region: 106 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_106.fasta\n",
      "--- *blasting region: 106\n",
      "--- total time for blast 106: 20.769998788833618\n",
      "-- processing region: 107 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_107.fasta\n",
      "--- *blasting region: 107\n",
      "--- total time for blast 107: 19.082000970840454\n",
      "-- processing region: 108 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_108.fasta\n",
      "--- *blasting region: 108\n",
      "--- total time for blast 108: 21.34899926185608\n",
      "-- processing region: 109 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_109.fasta\n",
      "--- *blasting region: 109\n",
      "--- total time for blast 109: 15.687004327774048\n",
      "-- processing region: 11 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_11.fasta\n",
      "--- *blasting region: 11\n",
      "--- total time for blast 11: 20.545993328094482\n",
      "-- processing region: 111 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_111.fasta\n",
      "--- *blasting region: 111\n",
      "--- total time for blast 111: 21.85300302505493\n",
      "-- processing region: 112 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_112.fasta\n",
      "--- *blasting region: 112\n",
      "--- total time for blast 112: 22.244996547698975\n",
      "-- processing region: 113 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_113.fasta\n",
      "--- *blasting region: 113\n",
      "--- total time for blast 113: 22.192999362945557\n",
      "-- processing region: 114 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_114.fasta\n",
      "--- *blasting region: 114\n",
      "--- total time for blast 114: 25.02200484275818\n",
      "-- processing region: 115 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_115.fasta\n",
      "--- *blasting region: 115\n",
      "--- total time for blast 115: 17.872995376586914\n",
      "-- processing region: 116 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_116.fasta\n",
      "--- *blasting region: 116\n",
      "--- total time for blast 116: 17.97700023651123\n",
      "-- processing region: 117 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_117.fasta\n",
      "--- *blasting region: 117\n",
      "--- total time for blast 117: 22.984999656677246\n",
      "-- processing region: 118 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_118.fasta\n",
      "--- *blasting region: 118\n",
      "--- total time for blast 118: 16.114001989364624\n",
      "-- processing region: 119 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_119.fasta\n",
      "--- *blasting region: 119\n",
      "--- total time for blast 119: 16.347001791000366\n",
      "-- processing region: 12 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_12.fasta\n",
      "--- *blasting region: 12\n",
      "--- total time for blast 12: 25.249999523162842\n",
      "-- processing region: 120 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_120.fasta\n",
      "--- *blasting region: 120\n",
      "--- total time for blast 120: 24.72699737548828\n",
      "-- processing region: 121 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_121.fasta\n",
      "--- *blasting region: 121\n",
      "--- total time for blast 121: 27.25500226020813\n",
      "-- processing region: 122 with 399 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_122.fasta\n",
      "--- *blasting region: 122\n",
      "--- total time for blast 122: 23.81899666786194\n",
      "-- processing region: 123 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_123.fasta\n",
      "--- *blasting region: 123\n",
      "--- total time for blast 123: 22.050007820129395\n",
      "-- processing region: 124 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_124.fasta\n",
      "--- *blasting region: 124\n",
      "--- total time for blast 124: 20.49799680709839\n",
      "-- processing region: 125 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_125.fasta\n",
      "--- *blasting region: 125\n",
      "--- total time for blast 125: 32.09299945831299\n",
      "-- processing region: 126 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_126.fasta\n",
      "--- *blasting region: 126\n",
      "--- total time for blast 126: 20.741997003555298\n",
      "-- processing region: 127 with 398 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_127.fasta\n",
      "--- *blasting region: 127\n",
      "--- total time for blast 127: 22.810001850128174\n",
      "-- processing region: 129 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_129.fasta\n",
      "--- *blasting region: 129\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 129: 16.280003309249878\n",
      "-- processing region: 13 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_13.fasta\n",
      "--- *blasting region: 13\n",
      "--- total time for blast 13: 21.118999242782593\n",
      "-- processing region: 130 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_130.fasta\n",
      "--- *blasting region: 130\n",
      "--- total time for blast 130: 19.21800422668457\n",
      "-- processing region: 131 with 396 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_131.fasta\n",
      "--- *blasting region: 131\n",
      "--- total time for blast 131: 22.153993844985962\n",
      "-- processing region: 132 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_132.fasta\n",
      "--- *blasting region: 132\n",
      "--- total time for blast 132: 27.76200795173645\n",
      "-- processing region: 133 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_133.fasta\n",
      "--- *blasting region: 133\n",
      "--- total time for blast 133: 19.452815294265747\n",
      "-- processing region: 134 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_134.fasta\n",
      "--- *blasting region: 134\n",
      "--- total time for blast 134: 22.55499839782715\n",
      "-- processing region: 135 with 399 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_135.fasta\n",
      "--- *blasting region: 135\n",
      "--- total time for blast 135: 18.174999475479126\n",
      "-- processing region: 136 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_136.fasta\n",
      "--- *blasting region: 136\n",
      "--- total time for blast 136: 17.42700219154358\n",
      "-- processing region: 137 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_137.fasta\n",
      "--- *blasting region: 137\n",
      "--- total time for blast 137: 26.408997535705566\n",
      "-- processing region: 138 with 399 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_138.fasta\n",
      "--- *blasting region: 138\n",
      "--- total time for blast 138: 21.74599599838257\n",
      "-- processing region: 139 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_139.fasta\n",
      "--- *blasting region: 139\n",
      "--- total time for blast 139: 21.747998237609863\n",
      "-- processing region: 14 with 398 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_14.fasta\n",
      "--- *blasting region: 14\n",
      "--- total time for blast 14: 19.985011100769043\n",
      "-- processing region: 140 with 398 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_140.fasta\n",
      "--- *blasting region: 140\n",
      "--- total time for blast 140: 23.728004932403564\n",
      "-- processing region: 141 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_141.fasta\n",
      "--- *blasting region: 141\n",
      "--- total time for blast 141: 25.39500403404236\n",
      "-- processing region: 142 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_142.fasta\n",
      "--- *blasting region: 142\n",
      "--- total time for blast 142: 25.184998989105225\n",
      "-- processing region: 143 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_143.fasta\n",
      "--- *blasting region: 143\n",
      "--- total time for blast 143: 24.79899501800537\n",
      "-- processing region: 144 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_144.fasta\n",
      "--- *blasting region: 144\n",
      "--- total time for blast 144: 18.485998392105103\n",
      "-- processing region: 145 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_145.fasta\n",
      "--- *blasting region: 145\n",
      "--- total time for blast 145: 23.943997144699097\n",
      "-- processing region: 146 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_146.fasta\n",
      "--- *blasting region: 146\n",
      "--- total time for blast 146: 20.077006816864014\n",
      "-- processing region: 147 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_147.fasta\n",
      "--- *blasting region: 147\n",
      "--- total time for blast 147: 18.949000597000122\n",
      "-- processing region: 148 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_148.fasta\n",
      "--- *blasting region: 148\n",
      "--- total time for blast 148: 20.195002794265747\n",
      "-- processing region: 149 with 384 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_149.fasta\n",
      "--- *blasting region: 149\n",
      "--- total time for blast 149: 21.294997453689575\n",
      "-- processing region: 15 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_15.fasta\n",
      "--- *blasting region: 15\n",
      "--- total time for blast 15: 19.727004289627075\n",
      "-- processing region: 150 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_150.fasta\n",
      "--- *blasting region: 150\n",
      "--- total time for blast 150: 17.27599310874939\n",
      "-- processing region: 151 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_151.fasta\n",
      "--- *blasting region: 151\n",
      "--- total time for blast 151: 22.81200408935547\n",
      "-- processing region: 152 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_152.fasta\n",
      "--- *blasting region: 152\n",
      "--- total time for blast 152: 21.429997444152832\n",
      "-- processing region: 153 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_153.fasta\n",
      "--- *blasting region: 153\n",
      "--- total time for blast 153: 21.29800009727478\n",
      "-- processing region: 154 with 394 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_154.fasta\n",
      "--- *blasting region: 154\n",
      "--- total time for blast 154: 27.757997512817383\n",
      "-- processing region: 155 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_155.fasta\n",
      "--- *blasting region: 155\n",
      "--- total time for blast 155: 23.628004789352417\n",
      "-- processing region: 156 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_156.fasta\n",
      "--- *blasting region: 156\n",
      "--- total time for blast 156: 27.607999563217163\n",
      "-- processing region: 157 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_157.fasta\n",
      "--- *blasting region: 157\n",
      "--- total time for blast 157: 20.161999225616455\n",
      "-- processing region: 158 with 398 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_158.fasta\n",
      "--- *blasting region: 158\n",
      "--- total time for blast 158: 20.89600682258606\n",
      "-- processing region: 159 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_159.fasta\n",
      "--- *blasting region: 159\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 159: 18.79399585723877\n",
      "-- processing region: 16 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_16.fasta\n",
      "--- *blasting region: 16\n",
      "--- total time for blast 16: 21.981003999710083\n",
      "-- processing region: 160 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_160.fasta\n",
      "--- *blasting region: 160\n",
      "--- total time for blast 160: 20.894002676010132\n",
      "-- processing region: 161 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_161.fasta\n",
      "--- *blasting region: 161\n",
      "--- total time for blast 161: 20.9749972820282\n",
      "-- processing region: 162 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_162.fasta\n",
      "--- *blasting region: 162\n",
      "--- total time for blast 162: 22.99500274658203\n",
      "-- processing region: 163 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_163.fasta\n",
      "--- *blasting region: 163\n",
      "--- total time for blast 163: 20.379003286361694\n",
      "-- processing region: 164 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_164.fasta\n",
      "--- *blasting region: 164\n",
      "--- total time for blast 164: 17.20699977874756\n",
      "-- processing region: 165 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_165.fasta\n",
      "--- *blasting region: 165\n",
      "--- total time for blast 165: 18.969998598098755\n",
      "-- processing region: 166 with 398 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_166.fasta\n",
      "--- *blasting region: 166\n",
      "--- total time for blast 166: 18.830997943878174\n",
      "-- processing region: 168 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_168.fasta\n",
      "--- *blasting region: 168\n",
      "--- total time for blast 168: 16.837998390197754\n",
      "-- processing region: 169 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_169.fasta\n",
      "--- *blasting region: 169\n",
      "--- total time for blast 169: 23.3729989528656\n",
      "-- processing region: 17 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_17.fasta\n",
      "--- *blasting region: 17\n",
      "--- total time for blast 17: 18.987001657485962\n",
      "-- processing region: 170 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_170.fasta\n",
      "--- *blasting region: 170\n",
      "--- total time for blast 170: 16.25200128555298\n",
      "-- processing region: 171 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_171.fasta\n",
      "--- *blasting region: 171\n",
      "--- total time for blast 171: 22.811996459960938\n",
      "-- processing region: 172 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_172.fasta\n",
      "--- *blasting region: 172\n",
      "--- total time for blast 172: 26.10500407218933\n",
      "-- processing region: 173 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_173.fasta\n",
      "--- *blasting region: 173\n",
      "--- total time for blast 173: 22.832995891571045\n",
      "-- processing region: 174 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_174.fasta\n",
      "--- *blasting region: 174\n",
      "--- total time for blast 174: 20.75800657272339\n",
      "-- processing region: 175 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_175.fasta\n",
      "--- *blasting region: 175\n",
      "--- total time for blast 175: 23.101004600524902\n",
      "-- processing region: 176 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_176.fasta\n",
      "--- *blasting region: 176\n",
      "--- total time for blast 176: 23.163992881774902\n",
      "-- processing region: 177 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_177.fasta\n",
      "--- *blasting region: 177\n",
      "--- total time for blast 177: 22.362005710601807\n",
      "-- processing region: 178 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_178.fasta\n",
      "--- *blasting region: 178\n",
      "--- total time for blast 178: 21.08999752998352\n",
      "-- processing region: 179 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_179.fasta\n",
      "--- *blasting region: 179\n",
      "--- total time for blast 179: 19.010000467300415\n",
      "-- processing region: 18 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_18.fasta\n",
      "--- *blasting region: 18\n",
      "--- total time for blast 18: 21.531997680664062\n",
      "-- processing region: 180 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_180.fasta\n",
      "--- *blasting region: 180\n",
      "--- total time for blast 180: 19.564000368118286\n",
      "-- processing region: 181 with 385 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_181.fasta\n",
      "--- *blasting region: 181\n",
      "--- total time for blast 181: 25.423001766204834\n",
      "-- processing region: 182 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_182.fasta\n",
      "--- *blasting region: 182\n",
      "--- total time for blast 182: 21.300000429153442\n",
      "-- processing region: 183 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_183.fasta\n",
      "--- *blasting region: 183\n",
      "--- total time for blast 183: 23.161003589630127\n",
      "-- processing region: 184 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_184.fasta\n",
      "--- *blasting region: 184\n",
      "--- total time for blast 184: 34.251993894577026\n",
      "-- processing region: 185 with 114 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_185.fasta\n",
      "--- *blasting region: 185\n",
      "--- total time for blast 185: 9.148005485534668\n",
      "-- processing region: 186 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_186.fasta\n",
      "--- *blasting region: 186\n",
      "--- total time for blast 186: 21.14499545097351\n",
      "-- processing region: 187 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_187.fasta\n",
      "--- *blasting region: 187\n",
      "--- total time for blast 187: 24.010002851486206\n",
      "-- processing region: 188 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_188.fasta\n",
      "--- *blasting region: 188\n",
      "--- total time for blast 188: 19.33800220489502\n",
      "-- processing region: 189 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_189.fasta\n",
      "--- *blasting region: 189\n",
      "--- total time for blast 189: 17.785995483398438\n",
      "-- processing region: 19 with 393 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_19.fasta\n",
      "--- *blasting region: 19\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 19: 16.72299861907959\n",
      "-- processing region: 190 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_190.fasta\n",
      "--- *blasting region: 190\n",
      "--- total time for blast 190: 26.29800033569336\n",
      "-- processing region: 191 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_191.fasta\n",
      "--- *blasting region: 191\n",
      "--- total time for blast 191: 18.600998878479004\n",
      "-- processing region: 192 with 394 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_192.fasta\n",
      "--- *blasting region: 192\n",
      "--- total time for blast 192: 19.421009302139282\n",
      "-- processing region: 193 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_193.fasta\n",
      "--- *blasting region: 193\n",
      "--- total time for blast 193: 20.618999481201172\n",
      "-- processing region: 194 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_194.fasta\n",
      "--- *blasting region: 194\n",
      "--- total time for blast 194: 23.334002017974854\n",
      "-- processing region: 195 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_195.fasta\n",
      "--- *blasting region: 195\n",
      "--- total time for blast 195: 22.954001903533936\n",
      "-- processing region: 196 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_196.fasta\n",
      "--- *blasting region: 196\n",
      "--- total time for blast 196: 20.595003604888916\n",
      "-- processing region: 197 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_197.fasta\n",
      "--- *blasting region: 197\n",
      "--- total time for blast 197: 27.813995361328125\n",
      "-- processing region: 198 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_198.fasta\n",
      "--- *blasting region: 198\n",
      "--- total time for blast 198: 22.975998640060425\n",
      "-- processing region: 199 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_199.fasta\n",
      "--- *blasting region: 199\n",
      "--- total time for blast 199: 21.825001001358032\n",
      "-- processing region: 2 with 398 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_2.fasta\n",
      "--- *blasting region: 2\n",
      "--- total time for blast 2: 20.691999912261963\n",
      "-- processing region: 20 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_20.fasta\n",
      "--- *blasting region: 20\n",
      "--- total time for blast 20: 23.23000168800354\n",
      "-- processing region: 200 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_200.fasta\n",
      "--- *blasting region: 200\n",
      "--- total time for blast 200: 21.60099506378174\n",
      "-- processing region: 201 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_201.fasta\n",
      "--- *blasting region: 201\n",
      "--- total time for blast 201: 22.27201199531555\n",
      "-- processing region: 202 with 392 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_202.fasta\n",
      "--- *blasting region: 202\n",
      "--- total time for blast 202: 17.533992052078247\n",
      "-- processing region: 203 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_203.fasta\n",
      "--- *blasting region: 203\n",
      "--- total time for blast 203: 20.629001140594482\n",
      "-- processing region: 204 with 399 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_204.fasta\n",
      "--- *blasting region: 204\n",
      "--- total time for blast 204: 19.59200143814087\n",
      "-- processing region: 205 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_205.fasta\n",
      "--- *blasting region: 205\n",
      "--- total time for blast 205: 20.77000141143799\n",
      "-- processing region: 206 with 399 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_206.fasta\n",
      "--- *blasting region: 206\n",
      "--- total time for blast 206: 20.074997425079346\n",
      "-- processing region: 207 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_207.fasta\n",
      "--- *blasting region: 207\n",
      "--- total time for blast 207: 20.7700035572052\n",
      "-- processing region: 208 with 398 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_208.fasta\n",
      "--- *blasting region: 208\n",
      "--- total time for blast 208: 29.931001663208008\n",
      "-- processing region: 209 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_209.fasta\n",
      "--- *blasting region: 209\n",
      "--- total time for blast 209: 23.309988737106323\n",
      "-- processing region: 21 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_21.fasta\n",
      "--- *blasting region: 21\n",
      "--- total time for blast 21: 25.537003755569458\n",
      "-- processing region: 210 with 397 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_210.fasta\n",
      "--- *blasting region: 210\n",
      "--- total time for blast 210: 21.35600209236145\n",
      "-- processing region: 211 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_211.fasta\n",
      "--- *blasting region: 211\n",
      "--- total time for blast 211: 22.279995679855347\n",
      "-- processing region: 212 with 206 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_212.fasta\n",
      "--- *blasting region: 212\n",
      "--- total time for blast 212: 24.77400541305542\n",
      "-- processing region: 213 with 283 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_213.fasta\n",
      "--- *blasting region: 213\n",
      "--- total time for blast 213: 24.07000184059143\n",
      "-- processing region: 214 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_214.fasta\n",
      "--- *blasting region: 214\n",
      "--- total time for blast 214: 24.46799874305725\n",
      "-- processing region: 215 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_215.fasta\n",
      "--- *blasting region: 215\n",
      "--- total time for blast 215: 22.77100133895874\n",
      "-- processing region: 216 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_216.fasta\n",
      "--- *blasting region: 216\n",
      "--- total time for blast 216: 18.8209969997406\n",
      "-- processing region: 217 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_217.fasta\n",
      "--- *blasting region: 217\n",
      "--- total time for blast 217: 24.965999126434326\n",
      "-- processing region: 218 with 399 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_218.fasta\n",
      "--- *blasting region: 218\n",
      "--- total time for blast 218: 20.45299768447876\n",
      "-- processing region: 219 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_219.fasta\n",
      "--- *blasting region: 219\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 219: 17.724000692367554\n",
      "-- processing region: 22 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_22.fasta\n",
      "--- *blasting region: 22\n",
      "--- total time for blast 22: 20.75999927520752\n",
      "-- processing region: 220 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_220.fasta\n",
      "--- *blasting region: 220\n",
      "--- total time for blast 220: 19.317882776260376\n",
      "-- processing region: 221 with 397 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_221.fasta\n",
      "--- *blasting region: 221\n",
      "--- total time for blast 221: 23.335996627807617\n",
      "-- processing region: 222 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_222.fasta\n",
      "--- *blasting region: 222\n",
      "--- total time for blast 222: 30.638001203536987\n",
      "-- processing region: 223 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_223.fasta\n",
      "--- *blasting region: 223\n",
      "--- total time for blast 223: 19.198991537094116\n",
      "-- processing region: 224 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_224.fasta\n",
      "--- *blasting region: 224\n",
      "--- total time for blast 224: 25.58199977874756\n",
      "-- processing region: 225 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_225.fasta\n",
      "--- *blasting region: 225\n",
      "--- total time for blast 225: 19.870002508163452\n",
      "-- processing region: 226 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_226.fasta\n",
      "--- *blasting region: 226\n",
      "--- total time for blast 226: 19.925997495651245\n",
      "-- processing region: 228 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_228.fasta\n",
      "--- *blasting region: 228\n",
      "--- total time for blast 228: 21.079007148742676\n",
      "-- processing region: 229 with 396 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_229.fasta\n",
      "--- *blasting region: 229\n",
      "--- total time for blast 229: 20.883999824523926\n",
      "-- processing region: 23 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_23.fasta\n",
      "--- *blasting region: 23\n",
      "--- total time for blast 23: 24.905999422073364\n",
      "-- processing region: 230 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_230.fasta\n",
      "--- *blasting region: 230\n",
      "--- total time for blast 230: 16.219000816345215\n",
      "-- processing region: 231 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_231.fasta\n",
      "--- *blasting region: 231\n",
      "--- total time for blast 231: 20.676997900009155\n",
      "-- processing region: 232 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_232.fasta\n",
      "--- *blasting region: 232\n",
      "--- total time for blast 232: 17.63300371170044\n",
      "-- processing region: 233 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_233.fasta\n",
      "--- *blasting region: 233\n",
      "--- total time for blast 233: 23.131999969482422\n",
      "-- processing region: 234 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_234.fasta\n",
      "--- *blasting region: 234\n",
      "--- total time for blast 234: 25.434000253677368\n",
      "-- processing region: 235 with 398 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_235.fasta\n",
      "--- *blasting region: 235\n",
      "--- total time for blast 235: 23.87099528312683\n",
      "-- processing region: 236 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_236.fasta\n",
      "--- *blasting region: 236\n",
      "--- total time for blast 236: 19.383002519607544\n",
      "-- processing region: 237 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_237.fasta\n",
      "--- *blasting region: 237\n",
      "--- total time for blast 237: 20.8850040435791\n",
      "-- processing region: 238 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_238.fasta\n",
      "--- *blasting region: 238\n",
      "--- total time for blast 238: 20.882999897003174\n",
      "-- processing region: 239 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_239.fasta\n",
      "--- *blasting region: 239\n",
      "--- total time for blast 239: 18.14899969100952\n",
      "-- processing region: 24 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_24.fasta\n",
      "--- *blasting region: 24\n",
      "--- total time for blast 24: 21.89299988746643\n",
      "-- processing region: 240 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_240.fasta\n",
      "--- *blasting region: 240\n",
      "--- total time for blast 240: 22.672998905181885\n",
      "-- processing region: 241 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_241.fasta\n",
      "--- *blasting region: 241\n",
      "--- total time for blast 241: 21.179993629455566\n",
      "-- processing region: 242 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_242.fasta\n",
      "--- *blasting region: 242\n",
      "--- total time for blast 242: 22.265007495880127\n",
      "-- processing region: 243 with 395 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_243.fasta\n",
      "--- *blasting region: 243\n",
      "--- total time for blast 243: 20.760998249053955\n",
      "-- processing region: 244 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_244.fasta\n",
      "--- *blasting region: 244\n",
      "--- total time for blast 244: 19.593000411987305\n",
      "-- processing region: 245 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_245.fasta\n",
      "--- *blasting region: 245\n",
      "--- total time for blast 245: 21.72100305557251\n",
      "-- processing region: 246 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_246.fasta\n",
      "--- *blasting region: 246\n",
      "--- total time for blast 246: 24.081995964050293\n",
      "-- processing region: 247 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_247.fasta\n",
      "--- *blasting region: 247\n",
      "--- total time for blast 247: 21.2529935836792\n",
      "-- processing region: 248 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_248.fasta\n",
      "--- *blasting region: 248\n",
      "--- total time for blast 248: 18.670957326889038\n",
      "-- processing region: 249 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_249.fasta\n",
      "--- *blasting region: 249\n",
      "--- total time for blast 249: 22.02599334716797\n",
      "-- processing region: 25 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_25.fasta\n",
      "--- *blasting region: 25\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 25: 20.009997367858887\n",
      "-- processing region: 250 with 393 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_250.fasta\n",
      "--- *blasting region: 250\n",
      "--- total time for blast 250: 20.47700333595276\n",
      "-- processing region: 251 with 398 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_251.fasta\n",
      "--- *blasting region: 251\n",
      "--- total time for blast 251: 21.841999530792236\n",
      "-- processing region: 252 with 399 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_252.fasta\n",
      "--- *blasting region: 252\n",
      "--- total time for blast 252: 20.899006128311157\n",
      "-- processing region: 253 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_253.fasta\n",
      "--- *blasting region: 253\n",
      "--- total time for blast 253: 20.413997173309326\n",
      "-- processing region: 255 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_255.fasta\n",
      "--- *blasting region: 255\n",
      "--- total time for blast 255: 20.707995414733887\n",
      "-- processing region: 256 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_256.fasta\n",
      "--- *blasting region: 256\n",
      "--- total time for blast 256: 23.55400037765503\n",
      "-- processing region: 257 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_257.fasta\n",
      "--- *blasting region: 257\n",
      "--- total time for blast 257: 24.640056610107422\n",
      "-- processing region: 258 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_258.fasta\n",
      "--- *blasting region: 258\n",
      "--- total time for blast 258: 22.283995389938354\n",
      "-- processing region: 259 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_259.fasta\n",
      "--- *blasting region: 259\n",
      "--- total time for blast 259: 21.85999870300293\n",
      "-- processing region: 26 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_26.fasta\n",
      "--- *blasting region: 26\n",
      "--- total time for blast 26: 21.361995935440063\n",
      "-- processing region: 260 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_260.fasta\n",
      "--- *blasting region: 260\n",
      "--- total time for blast 260: 20.88699436187744\n",
      "-- processing region: 261 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_261.fasta\n",
      "--- *blasting region: 261\n",
      "--- total time for blast 261: 21.19200110435486\n",
      "-- processing region: 262 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_262.fasta\n",
      "--- *blasting region: 262\n",
      "--- total time for blast 262: 20.651996612548828\n",
      "-- processing region: 263 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_263.fasta\n",
      "--- *blasting region: 263\n",
      "--- total time for blast 263: 18.332000017166138\n",
      "-- processing region: 264 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_264.fasta\n",
      "--- *blasting region: 264\n",
      "--- total time for blast 264: 17.132943391799927\n",
      "-- processing region: 265 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_265.fasta\n",
      "--- *blasting region: 265\n",
      "--- total time for blast 265: 20.366004705429077\n",
      "-- processing region: 266 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_266.fasta\n",
      "--- *blasting region: 266\n",
      "--- total time for blast 266: 22.087002992630005\n",
      "-- processing region: 267 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_267.fasta\n",
      "--- *blasting region: 267\n",
      "--- total time for blast 267: 27.77191472053528\n",
      "-- processing region: 268 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_268.fasta\n",
      "--- *blasting region: 268\n",
      "--- total time for blast 268: 21.95600199699402\n",
      "-- processing region: 269 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_269.fasta\n",
      "--- *blasting region: 269\n",
      "--- total time for blast 269: 20.096997261047363\n",
      "-- processing region: 27 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_27.fasta\n",
      "--- *blasting region: 27\n",
      "--- total time for blast 27: 25.371007680892944\n",
      "-- processing region: 270 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_270.fasta\n",
      "--- *blasting region: 270\n",
      "--- total time for blast 270: 21.432997941970825\n",
      "-- processing region: 271 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_271.fasta\n",
      "--- *blasting region: 271\n",
      "--- total time for blast 271: 18.753995656967163\n",
      "-- processing region: 272 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_272.fasta\n",
      "--- *blasting region: 272\n",
      "--- total time for blast 272: 19.226995944976807\n",
      "-- processing region: 273 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_273.fasta\n",
      "--- *blasting region: 273\n",
      "--- total time for blast 273: 15.933999300003052\n",
      "-- processing region: 274 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_274.fasta\n",
      "--- *blasting region: 274\n",
      "--- total time for blast 274: 24.196000576019287\n",
      "-- processing region: 275 with 386 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_275.fasta\n",
      "--- *blasting region: 275\n",
      "--- total time for blast 275: 21.30900478363037\n",
      "-- processing region: 277 with 399 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_277.fasta\n",
      "--- *blasting region: 277\n",
      "--- total time for blast 277: 24.066000938415527\n",
      "-- processing region: 278 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_278.fasta\n",
      "--- *blasting region: 278\n",
      "--- total time for blast 278: 21.970995903015137\n",
      "-- processing region: 279 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_279.fasta\n",
      "--- *blasting region: 279\n",
      "--- total time for blast 279: 26.814997673034668\n",
      "-- processing region: 28 with 399 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_28.fasta\n",
      "--- *blasting region: 28\n",
      "--- total time for blast 28: 22.832009077072144\n",
      "-- processing region: 280 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_280.fasta\n",
      "--- *blasting region: 280\n",
      "--- total time for blast 280: 20.122992277145386\n",
      "-- processing region: 281 with 396 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_281.fasta\n",
      "--- *blasting region: 281\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 281: 21.03499937057495\n",
      "-- processing region: 282 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_282.fasta\n",
      "--- *blasting region: 282\n",
      "--- total time for blast 282: 22.525002241134644\n",
      "-- processing region: 283 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_283.fasta\n",
      "--- *blasting region: 283\n",
      "--- total time for blast 283: 25.907999753952026\n",
      "-- processing region: 284 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_284.fasta\n",
      "--- *blasting region: 284\n",
      "--- total time for blast 284: 24.386038780212402\n",
      "-- processing region: 285 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_285.fasta\n",
      "--- *blasting region: 285\n",
      "--- total time for blast 285: 18.601958990097046\n",
      "-- processing region: 286 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_286.fasta\n",
      "--- *blasting region: 286\n",
      "--- total time for blast 286: 21.889995098114014\n",
      "-- processing region: 287 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_287.fasta\n",
      "--- *blasting region: 287\n",
      "--- total time for blast 287: 17.818999528884888\n",
      "-- processing region: 288 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_288.fasta\n",
      "--- *blasting region: 288\n",
      "--- total time for blast 288: 20.16599941253662\n",
      "-- processing region: 289 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_289.fasta\n",
      "--- *blasting region: 289\n",
      "--- total time for blast 289: 22.463998317718506\n",
      "-- processing region: 29 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_29.fasta\n",
      "--- *blasting region: 29\n",
      "--- total time for blast 29: 19.93299627304077\n",
      "-- processing region: 290 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_290.fasta\n",
      "--- *blasting region: 290\n",
      "--- total time for blast 290: 20.660001039505005\n",
      "-- processing region: 291 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_291.fasta\n",
      "--- *blasting region: 291\n",
      "--- total time for blast 291: 20.070998668670654\n",
      "-- processing region: 292 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_292.fasta\n",
      "--- *blasting region: 292\n",
      "--- total time for blast 292: 23.70800232887268\n",
      "-- processing region: 293 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_293.fasta\n",
      "--- *blasting region: 293\n",
      "--- total time for blast 293: 21.338998079299927\n",
      "-- processing region: 295 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_295.fasta\n",
      "--- *blasting region: 295\n",
      "--- total time for blast 295: 19.000993490219116\n",
      "-- processing region: 296 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_296.fasta\n",
      "--- *blasting region: 296\n",
      "--- total time for blast 296: 22.426006078720093\n",
      "-- processing region: 297 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_297.fasta\n",
      "--- *blasting region: 297\n",
      "--- total time for blast 297: 18.391996383666992\n",
      "-- processing region: 298 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_298.fasta\n",
      "--- *blasting region: 298\n",
      "--- total time for blast 298: 24.541006326675415\n",
      "-- processing region: 299 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_299.fasta\n",
      "--- *blasting region: 299\n",
      "--- total time for blast 299: 24.138999223709106\n",
      "-- processing region: 3 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_3.fasta\n",
      "--- *blasting region: 3\n",
      "--- total time for blast 3: 20.57300043106079\n",
      "-- processing region: 30 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_30.fasta\n",
      "--- *blasting region: 30\n",
      "--- total time for blast 30: 23.547993659973145\n",
      "-- processing region: 300 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_300.fasta\n",
      "--- *blasting region: 300\n",
      "--- total time for blast 300: 17.191995859146118\n",
      "-- processing region: 301 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_301.fasta\n",
      "--- *blasting region: 301\n",
      "--- total time for blast 301: 20.455002069473267\n",
      "-- processing region: 302 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_302.fasta\n",
      "--- *blasting region: 302\n",
      "--- total time for blast 302: 22.558003664016724\n",
      "-- processing region: 303 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_303.fasta\n",
      "--- *blasting region: 303\n",
      "--- total time for blast 303: 24.00599718093872\n",
      "-- processing region: 304 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_304.fasta\n",
      "--- *blasting region: 304\n",
      "--- total time for blast 304: 22.40999412536621\n",
      "-- processing region: 305 with 399 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_305.fasta\n",
      "--- *blasting region: 305\n",
      "--- total time for blast 305: 20.266000986099243\n",
      "-- processing region: 306 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_306.fasta\n",
      "--- *blasting region: 306\n",
      "--- total time for blast 306: 21.7719943523407\n",
      "-- processing region: 307 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_307.fasta\n",
      "--- *blasting region: 307\n",
      "--- total time for blast 307: 22.77699375152588\n",
      "-- processing region: 308 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_308.fasta\n",
      "--- *blasting region: 308\n",
      "--- total time for blast 308: 22.6780047416687\n",
      "-- processing region: 309 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_309.fasta\n",
      "--- *blasting region: 309\n",
      "--- total time for blast 309: 16.63099503517151\n",
      "-- processing region: 31 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_31.fasta\n",
      "--- *blasting region: 31\n",
      "--- total time for blast 31: 24.66900086402893\n",
      "-- processing region: 310 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_310.fasta\n",
      "--- *blasting region: 310\n",
      "--- total time for blast 310: 19.52199673652649\n",
      "-- processing region: 311 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_311.fasta\n",
      "--- *blasting region: 311\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 311: 24.802997589111328\n",
      "-- processing region: 312 with 390 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_312.fasta\n",
      "--- *blasting region: 312\n",
      "--- total time for blast 312: 20.8849995136261\n",
      "-- processing region: 313 with 396 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_313.fasta\n",
      "--- *blasting region: 313\n",
      "--- total time for blast 313: 22.261006593704224\n",
      "-- processing region: 314 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_314.fasta\n",
      "--- *blasting region: 314\n",
      "--- total time for blast 314: 21.9129958152771\n",
      "-- processing region: 315 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_315.fasta\n",
      "--- *blasting region: 315\n",
      "--- total time for blast 315: 23.879004955291748\n",
      "-- processing region: 316 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_316.fasta\n",
      "--- *blasting region: 316\n",
      "--- total time for blast 316: 22.063000440597534\n",
      "-- processing region: 317 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_317.fasta\n",
      "--- *blasting region: 317\n",
      "--- total time for blast 317: 22.0629940032959\n",
      "-- processing region: 318 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_318.fasta\n",
      "--- *blasting region: 318\n",
      "--- total time for blast 318: 20.810996294021606\n",
      "-- processing region: 319 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_319.fasta\n",
      "--- *blasting region: 319\n",
      "--- total time for blast 319: 22.669008255004883\n",
      "-- processing region: 32 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_32.fasta\n",
      "--- *blasting region: 32\n",
      "--- total time for blast 32: 21.067999124526978\n",
      "-- processing region: 320 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_320.fasta\n",
      "--- *blasting region: 320\n",
      "--- total time for blast 320: 27.446996927261353\n",
      "-- processing region: 321 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_321.fasta\n",
      "--- *blasting region: 321\n",
      "--- total time for blast 321: 20.846006870269775\n",
      "-- processing region: 322 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_322.fasta\n",
      "--- *blasting region: 322\n",
      "--- total time for blast 322: 21.769993782043457\n",
      "-- processing region: 324 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_324.fasta\n",
      "--- *blasting region: 324\n",
      "--- total time for blast 324: 23.294004440307617\n",
      "-- processing region: 325 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_325.fasta\n",
      "--- *blasting region: 325\n",
      "--- total time for blast 325: 21.081998586654663\n",
      "-- processing region: 326 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_326.fasta\n",
      "--- *blasting region: 326\n",
      "--- total time for blast 326: 18.08999490737915\n",
      "-- processing region: 327 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_327.fasta\n",
      "--- *blasting region: 327\n",
      "--- total time for blast 327: 16.033002614974976\n",
      "-- processing region: 328 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_328.fasta\n",
      "--- *blasting region: 328\n",
      "--- total time for blast 328: 23.277994871139526\n",
      "-- processing region: 329 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_329.fasta\n",
      "--- *blasting region: 329\n",
      "--- total time for blast 329: 17.617998123168945\n",
      "-- processing region: 33 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_33.fasta\n",
      "--- *blasting region: 33\n",
      "--- total time for blast 33: 21.002999305725098\n",
      "-- processing region: 330 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_330.fasta\n",
      "--- *blasting region: 330\n",
      "--- total time for blast 330: 25.674999475479126\n",
      "-- processing region: 332 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_332.fasta\n",
      "--- *blasting region: 332\n",
      "--- total time for blast 332: 21.016003370285034\n",
      "-- processing region: 333 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_333.fasta\n",
      "--- *blasting region: 333\n",
      "--- total time for blast 333: 18.148993968963623\n",
      "-- processing region: 334 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_334.fasta\n",
      "--- *blasting region: 334\n",
      "--- total time for blast 334: 18.237005472183228\n",
      "-- processing region: 335 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_335.fasta\n",
      "--- *blasting region: 335\n",
      "--- total time for blast 335: 20.79299759864807\n",
      "-- processing region: 336 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_336.fasta\n",
      "--- *blasting region: 336\n",
      "--- total time for blast 336: 17.639001607894897\n",
      "-- processing region: 337 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_337.fasta\n",
      "--- *blasting region: 337\n",
      "--- total time for blast 337: 20.682002067565918\n",
      "-- processing region: 338 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_338.fasta\n",
      "--- *blasting region: 338\n",
      "--- total time for blast 338: 21.949994564056396\n",
      "-- processing region: 339 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_339.fasta\n",
      "--- *blasting region: 339\n",
      "--- total time for blast 339: 18.625998973846436\n",
      "-- processing region: 34 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_34.fasta\n",
      "--- *blasting region: 34\n",
      "--- total time for blast 34: 21.33200192451477\n",
      "-- processing region: 340 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_340.fasta\n",
      "--- *blasting region: 340\n",
      "--- total time for blast 340: 18.54299831390381\n",
      "-- processing region: 341 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_341.fasta\n",
      "--- *blasting region: 341\n",
      "--- total time for blast 341: 25.233009099960327\n",
      "-- processing region: 342 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_342.fasta\n",
      "--- *blasting region: 342\n",
      "--- total time for blast 342: 33.54799795150757\n",
      "-- processing region: 343 with 399 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_343.fasta\n",
      "--- *blasting region: 343\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 343: 23.857993602752686\n",
      "-- processing region: 344 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_344.fasta\n",
      "--- *blasting region: 344\n",
      "--- total time for blast 344: 17.863003730773926\n",
      "-- processing region: 345 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_345.fasta\n",
      "--- *blasting region: 345\n",
      "--- total time for blast 345: 19.348004817962646\n",
      "-- processing region: 347 with 399 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_347.fasta\n",
      "--- *blasting region: 347\n",
      "--- total time for blast 347: 24.83999514579773\n",
      "-- processing region: 348 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_348.fasta\n",
      "--- *blasting region: 348\n",
      "--- total time for blast 348: 24.44000244140625\n",
      "-- processing region: 349 with 396 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_349.fasta\n",
      "--- *blasting region: 349\n",
      "--- total time for blast 349: 22.096001863479614\n",
      "-- processing region: 35 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_35.fasta\n",
      "--- *blasting region: 35\n",
      "--- total time for blast 35: 18.000998497009277\n",
      "-- processing region: 350 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_350.fasta\n",
      "--- *blasting region: 350\n",
      "--- total time for blast 350: 17.571001052856445\n",
      "-- processing region: 351 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_351.fasta\n",
      "--- *blasting region: 351\n",
      "--- total time for blast 351: 22.26400375366211\n",
      "-- processing region: 352 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_352.fasta\n",
      "--- *blasting region: 352\n",
      "--- total time for blast 352: 19.216999292373657\n",
      "-- processing region: 353 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_353.fasta\n",
      "--- *blasting region: 353\n",
      "--- total time for blast 353: 19.953004121780396\n",
      "-- processing region: 354 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_354.fasta\n",
      "--- *blasting region: 354\n",
      "--- total time for blast 354: 20.6589937210083\n",
      "-- processing region: 355 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_355.fasta\n",
      "--- *blasting region: 355\n",
      "--- total time for blast 355: 17.902002096176147\n",
      "-- processing region: 356 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_356.fasta\n",
      "--- *blasting region: 356\n",
      "--- total time for blast 356: 19.703996658325195\n",
      "-- processing region: 357 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_357.fasta\n",
      "--- *blasting region: 357\n",
      "--- total time for blast 357: 23.004998922348022\n",
      "-- processing region: 358 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_358.fasta\n",
      "--- *blasting region: 358\n",
      "--- total time for blast 358: 25.726001024246216\n",
      "-- processing region: 359 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_359.fasta\n",
      "--- *blasting region: 359\n",
      "--- total time for blast 359: 23.425002336502075\n",
      "-- processing region: 36 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_36.fasta\n",
      "--- *blasting region: 36\n",
      "--- total time for blast 36: 18.959999799728394\n",
      "-- processing region: 360 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_360.fasta\n",
      "--- *blasting region: 360\n",
      "--- total time for blast 360: 22.963993549346924\n",
      "-- processing region: 361 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_361.fasta\n",
      "--- *blasting region: 361\n",
      "--- total time for blast 361: 25.28100323677063\n",
      "-- processing region: 362 with 320 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_362.fasta\n",
      "--- *blasting region: 362\n",
      "--- total time for blast 362: 22.250999212265015\n",
      "-- processing region: 363 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_363.fasta\n",
      "--- *blasting region: 363\n",
      "--- total time for blast 363: 21.539003372192383\n",
      "-- processing region: 364 with 376 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_364.fasta\n",
      "--- *blasting region: 364\n",
      "--- total time for blast 364: 21.272005081176758\n",
      "-- processing region: 365 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_365.fasta\n",
      "--- *blasting region: 365\n",
      "--- total time for blast 365: 25.125999212265015\n",
      "-- processing region: 366 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_366.fasta\n",
      "--- *blasting region: 366\n",
      "--- total time for blast 366: 18.81899905204773\n",
      "-- processing region: 367 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_367.fasta\n",
      "--- *blasting region: 367\n",
      "--- total time for blast 367: 18.777002334594727\n",
      "-- processing region: 368 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_368.fasta\n",
      "--- *blasting region: 368\n",
      "--- total time for blast 368: 21.22600030899048\n",
      "-- processing region: 369 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_369.fasta\n",
      "--- *blasting region: 369\n",
      "--- total time for blast 369: 21.477001190185547\n",
      "-- processing region: 37 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_37.fasta\n",
      "--- *blasting region: 37\n",
      "--- total time for blast 37: 19.59899616241455\n",
      "-- processing region: 370 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_370.fasta\n",
      "--- *blasting region: 370\n",
      "--- total time for blast 370: 22.44500231742859\n",
      "-- processing region: 371 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_371.fasta\n",
      "--- *blasting region: 371\n",
      "--- total time for blast 371: 19.92500376701355\n",
      "-- processing region: 372 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_372.fasta\n",
      "--- *blasting region: 372\n",
      "--- total time for blast 372: 19.89400362968445\n",
      "-- processing region: 373 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_373.fasta\n",
      "--- *blasting region: 373\n",
      "--- total time for blast 373: 22.41900372505188\n",
      "-- processing region: 374 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_374.fasta\n",
      "--- *blasting region: 374\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 374: 20.622997283935547\n",
      "-- processing region: 375 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_375.fasta\n",
      "--- *blasting region: 375\n",
      "--- total time for blast 375: 22.125991106033325\n",
      "-- processing region: 376 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_376.fasta\n",
      "--- *blasting region: 376\n",
      "--- total time for blast 376: 20.61100745201111\n",
      "-- processing region: 377 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_377.fasta\n",
      "--- *blasting region: 377\n",
      "--- total time for blast 377: 16.78899908065796\n",
      "-- processing region: 378 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_378.fasta\n",
      "--- *blasting region: 378\n",
      "--- total time for blast 378: 25.989996433258057\n",
      "-- processing region: 379 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_379.fasta\n",
      "--- *blasting region: 379\n",
      "--- total time for blast 379: 22.077003002166748\n",
      "-- processing region: 38 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_38.fasta\n",
      "--- *blasting region: 38\n",
      "--- total time for blast 38: 18.775912046432495\n",
      "-- processing region: 380 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_380.fasta\n",
      "--- *blasting region: 380\n",
      "--- total time for blast 380: 22.83699607849121\n",
      "-- processing region: 381 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_381.fasta\n",
      "--- *blasting region: 381\n",
      "--- total time for blast 381: 17.290000677108765\n",
      "-- processing region: 382 with 390 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_382.fasta\n",
      "--- *blasting region: 382\n",
      "--- total time for blast 382: 21.326998949050903\n",
      "-- processing region: 383 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_383.fasta\n",
      "--- *blasting region: 383\n",
      "--- total time for blast 383: 24.098998308181763\n",
      "-- processing region: 384 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_384.fasta\n",
      "--- *blasting region: 384\n",
      "--- total time for blast 384: 20.07999324798584\n",
      "-- processing region: 385 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_385.fasta\n",
      "--- *blasting region: 385\n",
      "--- total time for blast 385: 19.2100031375885\n",
      "-- processing region: 386 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_386.fasta\n",
      "--- *blasting region: 386\n",
      "--- total time for blast 386: 21.598997116088867\n",
      "-- processing region: 387 with 398 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_387.fasta\n",
      "--- *blasting region: 387\n",
      "--- total time for blast 387: 20.672996997833252\n",
      "-- processing region: 388 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_388.fasta\n",
      "--- *blasting region: 388\n",
      "--- total time for blast 388: 22.270005464553833\n",
      "-- processing region: 389 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_389.fasta\n",
      "--- *blasting region: 389\n",
      "--- total time for blast 389: 22.385001182556152\n",
      "-- processing region: 39 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_39.fasta\n",
      "--- *blasting region: 39\n",
      "--- total time for blast 39: 21.13800287246704\n",
      "-- processing region: 390 with 362 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_390.fasta\n",
      "--- *blasting region: 390\n",
      "--- total time for blast 390: 18.189995050430298\n",
      "-- processing region: 391 with 399 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_391.fasta\n",
      "--- *blasting region: 391\n",
      "--- total time for blast 391: 21.618000745773315\n",
      "-- processing region: 392 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_392.fasta\n",
      "--- *blasting region: 392\n",
      "--- total time for blast 392: 22.271044492721558\n",
      "-- processing region: 393 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_393.fasta\n",
      "--- *blasting region: 393\n",
      "--- total time for blast 393: 21.82699680328369\n",
      "-- processing region: 394 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_394.fasta\n",
      "--- *blasting region: 394\n",
      "--- total time for blast 394: 24.440003633499146\n",
      "-- processing region: 395 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_395.fasta\n",
      "--- *blasting region: 395\n",
      "--- total time for blast 395: 20.930991888046265\n",
      "-- processing region: 396 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_396.fasta\n",
      "--- *blasting region: 396\n",
      "--- total time for blast 396: 22.129998683929443\n",
      "-- processing region: 397 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_397.fasta\n",
      "--- *blasting region: 397\n",
      "--- total time for blast 397: 23.242997407913208\n",
      "-- processing region: 398 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_398.fasta\n",
      "--- *blasting region: 398\n",
      "--- total time for blast 398: 21.7330002784729\n",
      "-- processing region: 399 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_399.fasta\n",
      "--- *blasting region: 399\n",
      "--- total time for blast 399: 19.60999608039856\n",
      "-- processing region: 4 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_4.fasta\n",
      "--- *blasting region: 4\n",
      "--- total time for blast 4: 27.22499680519104\n",
      "-- processing region: 40 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_40.fasta\n",
      "--- *blasting region: 40\n",
      "--- total time for blast 40: 17.457996606826782\n",
      "-- processing region: 400 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_400.fasta\n",
      "--- *blasting region: 400\n",
      "--- total time for blast 400: 20.950000524520874\n",
      "-- processing region: 401 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_401.fasta\n",
      "--- *blasting region: 401\n",
      "--- total time for blast 401: 27.876999855041504\n",
      "-- processing region: 402 with 73 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_402.fasta\n",
      "--- *blasting region: 402\n",
      "--- total time for blast 402: 7.7569966316223145\n",
      "-- processing region: 403 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_403.fasta\n",
      "--- *blasting region: 403\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 403: 19.363012552261353\n",
      "-- processing region: 404 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_404.fasta\n",
      "--- *blasting region: 404\n",
      "--- total time for blast 404: 45.67299246788025\n",
      "-- processing region: 405 with 398 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_405.fasta\n",
      "--- *blasting region: 405\n",
      "--- total time for blast 405: 87.3990044593811\n",
      "-- processing region: 406 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_406.fasta\n",
      "--- *blasting region: 406\n",
      "--- total time for blast 406: 19.069997310638428\n",
      "-- processing region: 407 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_407.fasta\n",
      "--- *blasting region: 407\n",
      "--- total time for blast 407: 87.44900369644165\n",
      "-- processing region: 408 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_408.fasta\n",
      "--- *blasting region: 408\n",
      "--- total time for blast 408: 44.40499949455261\n",
      "-- processing region: 409 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_409.fasta\n",
      "--- *blasting region: 409\n",
      "--- total time for blast 409: 24.025994539260864\n",
      "-- processing region: 41 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_41.fasta\n",
      "--- *blasting region: 41\n",
      "--- total time for blast 41: 32.75700259208679\n",
      "-- processing region: 410 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_410.fasta\n",
      "--- *blasting region: 410\n",
      "--- total time for blast 410: 25.65699577331543\n",
      "-- processing region: 411 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_411.fasta\n",
      "--- *blasting region: 411\n",
      "--- total time for blast 411: 33.67200493812561\n",
      "-- processing region: 412 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_412.fasta\n",
      "--- *blasting region: 412\n",
      "--- total time for blast 412: 52.659000635147095\n",
      "-- processing region: 413 with 391 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_413.fasta\n",
      "--- *blasting region: 413\n",
      "--- total time for blast 413: 36.16600155830383\n",
      "-- processing region: 414 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_414.fasta\n",
      "--- *blasting region: 414\n",
      "--- total time for blast 414: 45.422995805740356\n",
      "-- processing region: 415 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_415.fasta\n",
      "--- *blasting region: 415\n",
      "--- total time for blast 415: 31.9730007648468\n",
      "-- processing region: 416 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_416.fasta\n",
      "--- *blasting region: 416\n",
      "--- total time for blast 416: 59.19699764251709\n",
      "-- processing region: 417 with 399 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_417.fasta\n",
      "--- *blasting region: 417\n",
      "--- total time for blast 417: 34.18099927902222\n",
      "-- processing region: 418 with 378 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_418.fasta\n",
      "--- *blasting region: 418\n",
      "--- total time for blast 418: 37.254000663757324\n",
      "-- processing region: 419 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_419.fasta\n",
      "--- *blasting region: 419\n",
      "--- total time for blast 419: 32.829005002975464\n",
      "-- processing region: 42 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_42.fasta\n",
      "--- *blasting region: 42\n",
      "--- total time for blast 42: 44.93199634552002\n",
      "-- processing region: 420 with 391 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_420.fasta\n",
      "--- *blasting region: 420\n",
      "--- total time for blast 420: 22.32000184059143\n",
      "-- processing region: 421 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_421.fasta\n",
      "--- *blasting region: 421\n",
      "--- total time for blast 421: 50.55699324607849\n",
      "-- processing region: 422 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_422.fasta\n",
      "--- *blasting region: 422\n",
      "--- total time for blast 422: 43.23999571800232\n",
      "-- processing region: 423 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_423.fasta\n",
      "--- *blasting region: 423\n",
      "--- total time for blast 423: 66.45300340652466\n",
      "-- processing region: 424 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_424.fasta\n",
      "--- *blasting region: 424\n",
      "--- total time for blast 424: 52.46000528335571\n",
      "-- processing region: 425 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_425.fasta\n",
      "--- *blasting region: 425\n",
      "--- total time for blast 425: 59.65899419784546\n",
      "-- processing region: 426 with 313 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_426.fasta\n",
      "--- *blasting region: 426\n",
      "--- total time for blast 426: 25.370004892349243\n",
      "-- processing region: 427 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_427.fasta\n",
      "--- *blasting region: 427\n",
      "--- total time for blast 427: 19.539997577667236\n",
      "-- processing region: 428 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_428.fasta\n",
      "--- *blasting region: 428\n",
      "--- total time for blast 428: 43.5400025844574\n",
      "-- processing region: 429 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_429.fasta\n",
      "--- *blasting region: 429\n",
      "--- total time for blast 429: 21.201996564865112\n",
      "-- processing region: 43 with 398 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_43.fasta\n",
      "--- *blasting region: 43\n",
      "--- total time for blast 43: 19.77400302886963\n",
      "-- processing region: 430 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_430.fasta\n",
      "--- *blasting region: 430\n",
      "--- total time for blast 430: 46.853997468948364\n",
      "-- processing region: 431 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_431.fasta\n",
      "--- *blasting region: 431\n",
      "--- total time for blast 431: 20.2020046710968\n",
      "-- processing region: 432 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_432.fasta\n",
      "--- *blasting region: 432\n",
      "--- total time for blast 432: 19.352998971939087\n",
      "-- processing region: 433 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_433.fasta\n",
      "--- *blasting region: 433\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 433: 19.697998523712158\n",
      "-- processing region: 434 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_434.fasta\n",
      "--- *blasting region: 434\n",
      "--- total time for blast 434: 19.153003454208374\n",
      "-- processing region: 435 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_435.fasta\n",
      "--- *blasting region: 435\n",
      "--- total time for blast 435: 20.640995502471924\n",
      "-- processing region: 436 with 398 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_436.fasta\n",
      "--- *blasting region: 436\n",
      "--- total time for blast 436: 23.175003051757812\n",
      "-- processing region: 437 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_437.fasta\n",
      "--- *blasting region: 437\n",
      "--- total time for blast 437: 18.03299880027771\n",
      "-- processing region: 438 with 385 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_438.fasta\n",
      "--- *blasting region: 438\n",
      "--- total time for blast 438: 20.42299747467041\n",
      "-- processing region: 439 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_439.fasta\n",
      "--- *blasting region: 439\n",
      "--- total time for blast 439: 20.948001861572266\n",
      "-- processing region: 44 with 395 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_44.fasta\n",
      "--- *blasting region: 44\n",
      "--- total time for blast 44: 23.634004592895508\n",
      "-- processing region: 440 with 398 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_440.fasta\n",
      "--- *blasting region: 440\n",
      "--- total time for blast 440: 26.585001230239868\n",
      "-- processing region: 441 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_441.fasta\n",
      "--- *blasting region: 441\n",
      "--- total time for blast 441: 23.613998651504517\n",
      "-- processing region: 442 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_442.fasta\n",
      "--- *blasting region: 442\n",
      "--- total time for blast 442: 21.443001747131348\n",
      "-- processing region: 444 with 393 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_444.fasta\n",
      "--- *blasting region: 444\n",
      "--- total time for blast 444: 27.8700008392334\n",
      "-- processing region: 445 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_445.fasta\n",
      "--- *blasting region: 445\n",
      "--- total time for blast 445: 20.67999243736267\n",
      "-- processing region: 446 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_446.fasta\n",
      "--- *blasting region: 446\n",
      "--- total time for blast 446: 21.164003610610962\n",
      "-- processing region: 447 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_447.fasta\n",
      "--- *blasting region: 447\n",
      "--- total time for blast 447: 18.575002670288086\n",
      "-- processing region: 448 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_448.fasta\n",
      "--- *blasting region: 448\n",
      "--- total time for blast 448: 17.96600031852722\n",
      "-- processing region: 449 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_449.fasta\n",
      "--- *blasting region: 449\n",
      "--- total time for blast 449: 18.352004289627075\n",
      "-- processing region: 45 with 399 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_45.fasta\n",
      "--- *blasting region: 45\n",
      "--- total time for blast 45: 18.02299475669861\n",
      "-- processing region: 450 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_450.fasta\n",
      "--- *blasting region: 450\n",
      "--- total time for blast 450: 25.023003339767456\n",
      "-- processing region: 451 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_451.fasta\n",
      "--- *blasting region: 451\n",
      "--- total time for blast 451: 20.473992347717285\n",
      "-- processing region: 452 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_452.fasta\n",
      "--- *blasting region: 452\n",
      "--- total time for blast 452: 19.497000694274902\n",
      "-- processing region: 453 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_453.fasta\n",
      "--- *blasting region: 453\n",
      "--- total time for blast 453: 22.313997507095337\n",
      "-- processing region: 454 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_454.fasta\n",
      "--- *blasting region: 454\n",
      "--- total time for blast 454: 22.17200469970703\n",
      "-- processing region: 455 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_455.fasta\n",
      "--- *blasting region: 455\n",
      "--- total time for blast 455: 21.8300039768219\n",
      "-- processing region: 456 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_456.fasta\n",
      "--- *blasting region: 456\n",
      "--- total time for blast 456: 21.017993927001953\n",
      "-- processing region: 457 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_457.fasta\n",
      "--- *blasting region: 457\n",
      "--- total time for blast 457: 23.346004486083984\n",
      "-- processing region: 458 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_458.fasta\n",
      "--- *blasting region: 458\n",
      "--- total time for blast 458: 20.71299934387207\n",
      "-- processing region: 459 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_459.fasta\n",
      "--- *blasting region: 459\n",
      "--- total time for blast 459: 20.87599754333496\n",
      "-- processing region: 46 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_46.fasta\n",
      "--- *blasting region: 46\n",
      "--- total time for blast 46: 20.621010780334473\n",
      "-- processing region: 460 with 399 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_460.fasta\n",
      "--- *blasting region: 460\n",
      "--- total time for blast 460: 19.491994857788086\n",
      "-- processing region: 461 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_461.fasta\n",
      "--- *blasting region: 461\n",
      "--- total time for blast 461: 20.344999074935913\n",
      "-- processing region: 462 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_462.fasta\n",
      "--- *blasting region: 462\n",
      "--- total time for blast 462: 24.190003871917725\n",
      "-- processing region: 463 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_463.fasta\n",
      "--- *blasting region: 463\n",
      "--- total time for blast 463: 20.136000394821167\n",
      "-- processing region: 464 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_464.fasta\n",
      "--- *blasting region: 464\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 464: 21.1289963722229\n",
      "-- processing region: 465 with 398 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_465.fasta\n",
      "--- *blasting region: 465\n",
      "--- total time for blast 465: 21.710994958877563\n",
      "-- processing region: 466 with 397 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_466.fasta\n",
      "--- *blasting region: 466\n",
      "--- total time for blast 466: 21.301008462905884\n",
      "-- processing region: 467 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_467.fasta\n",
      "--- *blasting region: 467\n",
      "--- total time for blast 467: 27.955992937088013\n",
      "-- processing region: 468 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_468.fasta\n",
      "--- *blasting region: 468\n",
      "--- total time for blast 468: 23.591002941131592\n",
      "-- processing region: 469 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_469.fasta\n",
      "--- *blasting region: 469\n",
      "--- total time for blast 469: 19.528034687042236\n",
      "-- processing region: 47 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_47.fasta\n",
      "--- *blasting region: 47\n",
      "--- total time for blast 47: 25.629000425338745\n",
      "-- processing region: 470 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_470.fasta\n",
      "--- *blasting region: 470\n",
      "--- total time for blast 470: 17.64499521255493\n",
      "-- processing region: 471 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_471.fasta\n",
      "--- *blasting region: 471\n",
      "--- total time for blast 471: 19.17600703239441\n",
      "-- processing region: 472 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_472.fasta\n",
      "--- *blasting region: 472\n",
      "--- total time for blast 472: 21.16099452972412\n",
      "-- processing region: 473 with 399 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_473.fasta\n",
      "--- *blasting region: 473\n",
      "--- total time for blast 473: 27.59300470352173\n",
      "-- processing region: 474 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_474.fasta\n",
      "--- *blasting region: 474\n",
      "--- total time for blast 474: 19.31299352645874\n",
      "-- processing region: 475 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_475.fasta\n",
      "--- *blasting region: 475\n",
      "--- total time for blast 475: 26.558002471923828\n",
      "-- processing region: 476 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_476.fasta\n",
      "--- *blasting region: 476\n",
      "--- total time for blast 476: 26.494003772735596\n",
      "-- processing region: 477 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_477.fasta\n",
      "--- *blasting region: 477\n",
      "--- total time for blast 477: 20.266002893447876\n",
      "-- processing region: 478 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_478.fasta\n",
      "--- *blasting region: 478\n",
      "--- total time for blast 478: 20.21300172805786\n",
      "-- processing region: 479 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_479.fasta\n",
      "--- *blasting region: 479\n",
      "--- total time for blast 479: 19.638002157211304\n",
      "-- processing region: 48 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_48.fasta\n",
      "--- *blasting region: 48\n",
      "--- total time for blast 48: 21.659997940063477\n",
      "-- processing region: 480 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_480.fasta\n",
      "--- *blasting region: 480\n",
      "--- total time for blast 480: 18.880995750427246\n",
      "-- processing region: 481 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_481.fasta\n",
      "--- *blasting region: 481\n",
      "--- total time for blast 481: 19.815999269485474\n",
      "-- processing region: 482 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_482.fasta\n",
      "--- *blasting region: 482\n",
      "--- total time for blast 482: 25.5860013961792\n",
      "-- processing region: 483 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_483.fasta\n",
      "--- *blasting region: 483\n",
      "--- total time for blast 483: 17.85400152206421\n",
      "-- processing region: 484 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_484.fasta\n",
      "--- *blasting region: 484\n",
      "--- total time for blast 484: 21.325998306274414\n",
      "-- processing region: 485 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_485.fasta\n",
      "--- *blasting region: 485\n",
      "--- total time for blast 485: 19.90799903869629\n",
      "-- processing region: 486 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_486.fasta\n",
      "--- *blasting region: 486\n",
      "--- total time for blast 486: 22.639004230499268\n",
      "-- processing region: 487 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_487.fasta\n",
      "--- *blasting region: 487\n",
      "--- total time for blast 487: 21.594003438949585\n",
      "-- processing region: 488 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_488.fasta\n",
      "--- *blasting region: 488\n",
      "--- total time for blast 488: 18.07300329208374\n",
      "-- processing region: 489 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_489.fasta\n",
      "--- *blasting region: 489\n",
      "--- total time for blast 489: 20.52900004386902\n",
      "-- processing region: 49 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_49.fasta\n",
      "--- *blasting region: 49\n",
      "--- total time for blast 49: 24.70699977874756\n",
      "-- processing region: 491 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_491.fasta\n",
      "--- *blasting region: 491\n",
      "--- total time for blast 491: 24.814006090164185\n",
      "-- processing region: 492 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_492.fasta\n",
      "--- *blasting region: 492\n",
      "--- total time for blast 492: 27.90899968147278\n",
      "-- processing region: 493 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_493.fasta\n",
      "--- *blasting region: 493\n",
      "--- total time for blast 493: 15.748003721237183\n",
      "-- processing region: 494 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_494.fasta\n",
      "--- *blasting region: 494\n",
      "--- total time for blast 494: 18.342002630233765\n",
      "-- processing region: 495 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_495.fasta\n",
      "--- *blasting region: 495\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 495: 19.737996816635132\n",
      "-- processing region: 496 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_496.fasta\n",
      "--- *blasting region: 496\n",
      "--- total time for blast 496: 27.204996824264526\n",
      "-- processing region: 497 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_497.fasta\n",
      "--- *blasting region: 497\n",
      "--- total time for blast 497: 19.64200234413147\n",
      "-- processing region: 498 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_498.fasta\n",
      "--- *blasting region: 498\n",
      "--- total time for blast 498: 19.937997817993164\n",
      "-- processing region: 499 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_499.fasta\n",
      "--- *blasting region: 499\n",
      "--- total time for blast 499: 21.298999547958374\n",
      "-- processing region: 5 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_5.fasta\n",
      "--- *blasting region: 5\n",
      "--- total time for blast 5: 21.910996198654175\n",
      "-- processing region: 50 with 389 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_50.fasta\n",
      "--- *blasting region: 50\n",
      "--- total time for blast 50: 20.267991065979004\n",
      "-- processing region: 500 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_500.fasta\n",
      "--- *blasting region: 500\n",
      "--- total time for blast 500: 18.914007425308228\n",
      "-- processing region: 501 with 397 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_501.fasta\n",
      "--- *blasting region: 501\n",
      "--- total time for blast 501: 29.626993417739868\n",
      "-- processing region: 502 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_502.fasta\n",
      "--- *blasting region: 502\n",
      "--- total time for blast 502: 21.327993631362915\n",
      "-- processing region: 503 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_503.fasta\n",
      "--- *blasting region: 503\n",
      "--- total time for blast 503: 20.462002754211426\n",
      "-- processing region: 504 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_504.fasta\n",
      "--- *blasting region: 504\n",
      "--- total time for blast 504: 22.62800121307373\n",
      "-- processing region: 505 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_505.fasta\n",
      "--- *blasting region: 505\n",
      "--- total time for blast 505: 20.849000692367554\n",
      "-- processing region: 506 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_506.fasta\n",
      "--- *blasting region: 506\n",
      "--- total time for blast 506: 21.906998872756958\n",
      "-- processing region: 507 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_507.fasta\n",
      "--- *blasting region: 507\n",
      "--- total time for blast 507: 25.750985860824585\n",
      "-- processing region: 508 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_508.fasta\n",
      "--- *blasting region: 508\n",
      "--- total time for blast 508: 19.91699719429016\n",
      "-- processing region: 509 with 396 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_509.fasta\n",
      "--- *blasting region: 509\n",
      "--- total time for blast 509: 21.433000564575195\n",
      "-- processing region: 51 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_51.fasta\n",
      "--- *blasting region: 51\n",
      "--- total time for blast 51: 24.174998998641968\n",
      "-- processing region: 510 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_510.fasta\n",
      "--- *blasting region: 510\n",
      "--- total time for blast 510: 20.52099847793579\n",
      "-- processing region: 511 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_511.fasta\n",
      "--- *blasting region: 511\n",
      "--- total time for blast 511: 18.041996479034424\n",
      "-- processing region: 512 with 398 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_512.fasta\n",
      "--- *blasting region: 512\n",
      "--- total time for blast 512: 25.853000164031982\n",
      "-- processing region: 513 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_513.fasta\n",
      "--- *blasting region: 513\n",
      "--- total time for blast 513: 22.66899275779724\n",
      "-- processing region: 514 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_514.fasta\n",
      "--- *blasting region: 514\n",
      "--- total time for blast 514: 20.984995365142822\n",
      "-- processing region: 515 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_515.fasta\n",
      "--- *blasting region: 515\n",
      "--- total time for blast 515: 23.15000557899475\n",
      "-- processing region: 516 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_516.fasta\n",
      "--- *blasting region: 516\n",
      "--- total time for blast 516: 25.63899803161621\n",
      "-- processing region: 517 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_517.fasta\n",
      "--- *blasting region: 517\n",
      "--- total time for blast 517: 22.25800371170044\n",
      "-- processing region: 518 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_518.fasta\n",
      "--- *blasting region: 518\n",
      "--- total time for blast 518: 20.54099702835083\n",
      "-- processing region: 519 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_519.fasta\n",
      "--- *blasting region: 519\n",
      "--- total time for blast 519: 19.214001417160034\n",
      "-- processing region: 52 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_52.fasta\n",
      "--- *blasting region: 52\n",
      "--- total time for blast 52: 17.779992818832397\n",
      "-- processing region: 520 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_520.fasta\n",
      "--- *blasting region: 520\n",
      "--- total time for blast 520: 18.88301110267639\n",
      "-- processing region: 521 with 398 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_521.fasta\n",
      "--- *blasting region: 521\n",
      "--- total time for blast 521: 18.8439998626709\n",
      "-- processing region: 523 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_523.fasta\n",
      "--- *blasting region: 523\n",
      "--- total time for blast 523: 22.034996509552002\n",
      "-- processing region: 524 with 399 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_524.fasta\n",
      "--- *blasting region: 524\n",
      "--- total time for blast 524: 20.16599726676941\n",
      "-- processing region: 525 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_525.fasta\n",
      "--- *blasting region: 525\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 525: 26.47499179840088\n",
      "-- processing region: 526 with 391 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_526.fasta\n",
      "--- *blasting region: 526\n",
      "--- total time for blast 526: 22.21499752998352\n",
      "-- processing region: 527 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_527.fasta\n",
      "--- *blasting region: 527\n",
      "--- total time for blast 527: 24.63500213623047\n",
      "-- processing region: 528 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_528.fasta\n",
      "--- *blasting region: 528\n",
      "--- total time for blast 528: 22.26599669456482\n",
      "-- processing region: 529 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_529.fasta\n",
      "--- *blasting region: 529\n",
      "--- total time for blast 529: 27.371995210647583\n",
      "-- processing region: 53 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_53.fasta\n",
      "--- *blasting region: 53\n",
      "--- total time for blast 53: 26.668006896972656\n",
      "-- processing region: 530 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_530.fasta\n",
      "--- *blasting region: 530\n",
      "--- total time for blast 530: 20.350999116897583\n",
      "-- processing region: 531 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_531.fasta\n",
      "--- *blasting region: 531\n",
      "--- total time for blast 531: 33.54299879074097\n",
      "-- processing region: 532 with 395 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_532.fasta\n",
      "--- *blasting region: 532\n",
      "--- total time for blast 532: 19.92899775505066\n",
      "-- processing region: 533 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_533.fasta\n",
      "--- *blasting region: 533\n",
      "--- total time for blast 533: 17.06000256538391\n",
      "-- processing region: 534 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_534.fasta\n",
      "--- *blasting region: 534\n",
      "--- total time for blast 534: 24.867998600006104\n",
      "-- processing region: 535 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_535.fasta\n",
      "--- *blasting region: 535\n",
      "--- total time for blast 535: 25.54199743270874\n",
      "-- processing region: 536 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_536.fasta\n",
      "--- *blasting region: 536\n",
      "--- total time for blast 536: 22.305999755859375\n",
      "-- processing region: 537 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_537.fasta\n",
      "--- *blasting region: 537\n",
      "--- total time for blast 537: 20.740999460220337\n",
      "-- processing region: 538 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_538.fasta\n",
      "--- *blasting region: 538\n",
      "--- total time for blast 538: 18.705005407333374\n",
      "-- processing region: 539 with 268 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_539.fasta\n",
      "--- *blasting region: 539\n",
      "--- total time for blast 539: 18.78399682044983\n",
      "-- processing region: 54 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_54.fasta\n",
      "--- *blasting region: 54\n",
      "--- total time for blast 54: 27.603001832962036\n",
      "-- processing region: 540 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_540.fasta\n",
      "--- *blasting region: 540\n",
      "--- total time for blast 540: 21.27600359916687\n",
      "-- processing region: 541 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_541.fasta\n",
      "--- *blasting region: 541\n",
      "--- total time for blast 541: 18.973998069763184\n",
      "-- processing region: 542 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_542.fasta\n",
      "--- *blasting region: 542\n",
      "--- total time for blast 542: 25.61599373817444\n",
      "-- processing region: 543 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_543.fasta\n",
      "--- *blasting region: 543\n",
      "--- total time for blast 543: 17.828000783920288\n",
      "-- processing region: 544 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_544.fasta\n",
      "--- *blasting region: 544\n",
      "--- total time for blast 544: 19.802001953125\n",
      "-- processing region: 545 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_545.fasta\n",
      "--- *blasting region: 545\n",
      "--- total time for blast 545: 19.500000476837158\n",
      "-- processing region: 546 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_546.fasta\n",
      "--- *blasting region: 546\n",
      "--- total time for blast 546: 21.055994987487793\n",
      "-- processing region: 547 with 373 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_547.fasta\n",
      "--- *blasting region: 547\n",
      "--- total time for blast 547: 22.335005283355713\n",
      "-- processing region: 548 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_548.fasta\n",
      "--- *blasting region: 548\n",
      "--- total time for blast 548: 22.267002820968628\n",
      "-- processing region: 549 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_549.fasta\n",
      "--- *blasting region: 549\n",
      "--- total time for blast 549: 24.471993446350098\n",
      "-- processing region: 55 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_55.fasta\n",
      "--- *blasting region: 55\n",
      "--- total time for blast 55: 25.575000762939453\n",
      "-- processing region: 550 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_550.fasta\n",
      "--- *blasting region: 550\n",
      "--- total time for blast 550: 22.808008193969727\n",
      "-- processing region: 551 with 398 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_551.fasta\n",
      "--- *blasting region: 551\n",
      "--- total time for blast 551: 25.704992055892944\n",
      "-- processing region: 552 with 388 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_552.fasta\n",
      "--- *blasting region: 552\n",
      "--- total time for blast 552: 16.64001178741455\n",
      "-- processing region: 553 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_553.fasta\n",
      "--- *blasting region: 553\n",
      "--- total time for blast 553: 24.39999794960022\n",
      "-- processing region: 554 with 381 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_554.fasta\n",
      "--- *blasting region: 554\n",
      "--- total time for blast 554: 17.26699662208557\n",
      "-- processing region: 555 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_555.fasta\n",
      "--- *blasting region: 555\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 555: 17.152002096176147\n",
      "-- processing region: 556 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_556.fasta\n",
      "--- *blasting region: 556\n",
      "--- total time for blast 556: 18.621997356414795\n",
      "-- processing region: 557 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_557.fasta\n",
      "--- *blasting region: 557\n",
      "--- total time for blast 557: 17.12200665473938\n",
      "-- processing region: 558 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_558.fasta\n",
      "--- *blasting region: 558\n",
      "--- total time for blast 558: 30.1439950466156\n",
      "-- processing region: 559 with 398 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_559.fasta\n",
      "--- *blasting region: 559\n",
      "--- total time for blast 559: 21.77700686454773\n",
      "-- processing region: 56 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_56.fasta\n",
      "--- *blasting region: 56\n",
      "--- total time for blast 56: 19.425002813339233\n",
      "-- processing region: 560 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_560.fasta\n",
      "--- *blasting region: 560\n",
      "--- total time for blast 560: 19.754998445510864\n",
      "-- processing region: 561 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_561.fasta\n",
      "--- *blasting region: 561\n",
      "--- total time for blast 561: 22.234002113342285\n",
      "-- processing region: 562 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_562.fasta\n",
      "--- *blasting region: 562\n",
      "--- total time for blast 562: 24.221997261047363\n",
      "-- processing region: 563 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_563.fasta\n",
      "--- *blasting region: 563\n",
      "--- total time for blast 563: 22.200993061065674\n",
      "-- processing region: 564 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_564.fasta\n",
      "--- *blasting region: 564\n",
      "--- total time for blast 564: 24.805003881454468\n",
      "-- processing region: 565 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_565.fasta\n",
      "--- *blasting region: 565\n",
      "--- total time for blast 565: 22.60999298095703\n",
      "-- processing region: 566 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_566.fasta\n",
      "--- *blasting region: 566\n",
      "--- total time for blast 566: 27.18400764465332\n",
      "-- processing region: 567 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_567.fasta\n",
      "--- *blasting region: 567\n",
      "--- total time for blast 567: 20.278997898101807\n",
      "-- processing region: 568 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_568.fasta\n",
      "--- *blasting region: 568\n",
      "--- total time for blast 568: 18.978999853134155\n",
      "-- processing region: 569 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_569.fasta\n",
      "--- *blasting region: 569\n",
      "--- total time for blast 569: 20.111995935440063\n",
      "-- processing region: 57 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_57.fasta\n",
      "--- *blasting region: 57\n",
      "--- total time for blast 57: 18.18800687789917\n",
      "-- processing region: 570 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_570.fasta\n",
      "--- *blasting region: 570\n",
      "--- total time for blast 570: 21.03899908065796\n",
      "-- processing region: 571 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_571.fasta\n",
      "--- *blasting region: 571\n",
      "--- total time for blast 571: 24.637996912002563\n",
      "-- processing region: 572 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_572.fasta\n",
      "--- *blasting region: 572\n",
      "--- total time for blast 572: 21.25099539756775\n",
      "-- processing region: 573 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_573.fasta\n",
      "--- *blasting region: 573\n",
      "--- total time for blast 573: 22.942996740341187\n",
      "-- processing region: 574 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_574.fasta\n",
      "--- *blasting region: 574\n",
      "--- total time for blast 574: 32.181994676589966\n",
      "-- processing region: 575 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_575.fasta\n",
      "--- *blasting region: 575\n",
      "--- total time for blast 575: 18.368996620178223\n",
      "-- processing region: 576 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_576.fasta\n",
      "--- *blasting region: 576\n",
      "--- total time for blast 576: 19.64100170135498\n",
      "-- processing region: 577 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_577.fasta\n",
      "--- *blasting region: 577\n",
      "--- total time for blast 577: 25.719000339508057\n",
      "-- processing region: 579 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_579.fasta\n",
      "--- *blasting region: 579\n",
      "--- total time for blast 579: 20.660998106002808\n",
      "-- processing region: 58 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_58.fasta\n",
      "--- *blasting region: 58\n",
      "--- total time for blast 58: 23.351994514465332\n",
      "-- processing region: 580 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_580.fasta\n",
      "--- *blasting region: 580\n",
      "--- total time for blast 580: 19.407000064849854\n",
      "-- processing region: 581 with 398 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_581.fasta\n",
      "--- *blasting region: 581\n",
      "--- total time for blast 581: 21.223005533218384\n",
      "-- processing region: 582 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_582.fasta\n",
      "--- *blasting region: 582\n",
      "--- total time for blast 582: 23.652000188827515\n",
      "-- processing region: 583 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_583.fasta\n",
      "--- *blasting region: 583\n",
      "--- total time for blast 583: 21.969998836517334\n",
      "-- processing region: 584 with 399 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_584.fasta\n",
      "--- *blasting region: 584\n",
      "--- total time for blast 584: 28.48800277709961\n",
      "-- processing region: 585 with 399 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_585.fasta\n",
      "--- *blasting region: 585\n",
      "--- total time for blast 585: 18.151000022888184\n",
      "-- processing region: 586 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_586.fasta\n",
      "--- *blasting region: 586\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 586: 19.39200472831726\n",
      "-- processing region: 587 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_587.fasta\n",
      "--- *blasting region: 587\n",
      "--- total time for blast 587: 21.81799864768982\n",
      "-- processing region: 588 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_588.fasta\n",
      "--- *blasting region: 588\n",
      "--- total time for blast 588: 21.7630033493042\n",
      "-- processing region: 589 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_589.fasta\n",
      "--- *blasting region: 589\n",
      "--- total time for blast 589: 22.757996320724487\n",
      "-- processing region: 59 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_59.fasta\n",
      "--- *blasting region: 59\n",
      "--- total time for blast 59: 23.67700171470642\n",
      "-- processing region: 590 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_590.fasta\n",
      "--- *blasting region: 590\n",
      "--- total time for blast 590: 20.323997735977173\n",
      "-- processing region: 591 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_591.fasta\n",
      "--- *blasting region: 591\n",
      "--- total time for blast 591: 19.164002656936646\n",
      "-- processing region: 592 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_592.fasta\n",
      "--- *blasting region: 592\n",
      "--- total time for blast 592: 23.39600396156311\n",
      "-- processing region: 593 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_593.fasta\n",
      "--- *blasting region: 593\n",
      "--- total time for blast 593: 19.967998504638672\n",
      "-- processing region: 594 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_594.fasta\n",
      "--- *blasting region: 594\n",
      "--- total time for blast 594: 32.482994556427\n",
      "-- processing region: 595 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_595.fasta\n",
      "--- *blasting region: 595\n",
      "--- total time for blast 595: 19.46500253677368\n",
      "-- processing region: 596 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_596.fasta\n",
      "--- *blasting region: 596\n",
      "--- total time for blast 596: 23.98099994659424\n",
      "-- processing region: 597 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_597.fasta\n",
      "--- *blasting region: 597\n",
      "--- total time for blast 597: 19.664997577667236\n",
      "-- processing region: 598 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_598.fasta\n",
      "--- *blasting region: 598\n",
      "--- total time for blast 598: 19.700998067855835\n",
      "-- processing region: 599 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_599.fasta\n",
      "--- *blasting region: 599\n",
      "--- total time for blast 599: 26.720000505447388\n",
      "-- processing region: 6 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_6.fasta\n",
      "--- *blasting region: 6\n",
      "--- total time for blast 6: 28.437999486923218\n",
      "-- processing region: 60 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_60.fasta\n",
      "--- *blasting region: 60\n",
      "--- total time for blast 60: 20.595003366470337\n",
      "-- processing region: 600 with 399 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_600.fasta\n",
      "--- *blasting region: 600\n",
      "--- total time for blast 600: 21.56399965286255\n",
      "-- processing region: 601 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_601.fasta\n",
      "--- *blasting region: 601\n",
      "--- total time for blast 601: 22.763001918792725\n",
      "-- processing region: 602 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_602.fasta\n",
      "--- *blasting region: 602\n",
      "--- total time for blast 602: 19.4319965839386\n",
      "-- processing region: 603 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_603.fasta\n",
      "--- *blasting region: 603\n",
      "--- total time for blast 603: 20.10999822616577\n",
      "-- processing region: 604 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_604.fasta\n",
      "--- *blasting region: 604\n",
      "--- total time for blast 604: 17.50405478477478\n",
      "-- processing region: 605 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_605.fasta\n",
      "--- *blasting region: 605\n",
      "--- total time for blast 605: 21.321992874145508\n",
      "-- processing region: 606 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_606.fasta\n",
      "--- *blasting region: 606\n",
      "--- total time for blast 606: 18.24199914932251\n",
      "-- processing region: 607 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_607.fasta\n",
      "--- *blasting region: 607\n",
      "--- total time for blast 607: 24.476999521255493\n",
      "-- processing region: 608 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_608.fasta\n",
      "--- *blasting region: 608\n",
      "--- total time for blast 608: 16.9950008392334\n",
      "-- processing region: 609 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_609.fasta\n",
      "--- *blasting region: 609\n",
      "--- total time for blast 609: 21.41900062561035\n",
      "-- processing region: 61 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_61.fasta\n",
      "--- *blasting region: 61\n",
      "--- total time for blast 61: 23.37199854850769\n",
      "-- processing region: 610 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_610.fasta\n",
      "--- *blasting region: 610\n",
      "--- total time for blast 610: 23.452000856399536\n",
      "-- processing region: 611 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_611.fasta\n",
      "--- *blasting region: 611\n",
      "--- total time for blast 611: 24.504006147384644\n",
      "-- processing region: 612 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_612.fasta\n",
      "--- *blasting region: 612\n",
      "--- total time for blast 612: 24.030007362365723\n",
      "-- processing region: 613 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_613.fasta\n",
      "--- *blasting region: 613\n",
      "--- total time for blast 613: 21.978993892669678\n",
      "-- processing region: 614 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_614.fasta\n",
      "--- *blasting region: 614\n",
      "--- total time for blast 614: 21.860998392105103\n",
      "-- processing region: 615 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_615.fasta\n",
      "--- *blasting region: 615\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 615: 21.493998289108276\n",
      "-- processing region: 616 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_616.fasta\n",
      "--- *blasting region: 616\n",
      "--- total time for blast 616: 21.225000858306885\n",
      "-- processing region: 617 with 399 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_617.fasta\n",
      "--- *blasting region: 617\n",
      "--- total time for blast 617: 21.38000226020813\n",
      "-- processing region: 618 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_618.fasta\n",
      "--- *blasting region: 618\n",
      "--- total time for blast 618: 28.243000984191895\n",
      "-- processing region: 619 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_619.fasta\n",
      "--- *blasting region: 619\n",
      "--- total time for blast 619: 26.768000602722168\n",
      "-- processing region: 62 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_62.fasta\n",
      "--- *blasting region: 62\n",
      "--- total time for blast 62: 22.498001098632812\n",
      "-- processing region: 620 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_620.fasta\n",
      "--- *blasting region: 620\n",
      "--- total time for blast 620: 22.747997522354126\n",
      "-- processing region: 621 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_621.fasta\n",
      "--- *blasting region: 621\n",
      "--- total time for blast 621: 22.380003929138184\n",
      "-- processing region: 622 with 397 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_622.fasta\n",
      "--- *blasting region: 622\n",
      "--- total time for blast 622: 23.351000547409058\n",
      "-- processing region: 623 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_623.fasta\n",
      "--- *blasting region: 623\n",
      "--- total time for blast 623: 23.815991163253784\n",
      "-- processing region: 624 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_624.fasta\n",
      "--- *blasting region: 624\n",
      "--- total time for blast 624: 20.914003610610962\n",
      "-- processing region: 625 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_625.fasta\n",
      "--- *blasting region: 625\n",
      "--- total time for blast 625: 25.928000688552856\n",
      "-- processing region: 626 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_626.fasta\n",
      "--- *blasting region: 626\n",
      "--- total time for blast 626: 22.59999918937683\n",
      "-- processing region: 627 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_627.fasta\n",
      "--- *blasting region: 627\n",
      "--- total time for blast 627: 17.35799741744995\n",
      "-- processing region: 628 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_628.fasta\n",
      "--- *blasting region: 628\n",
      "--- total time for blast 628: 20.920000314712524\n",
      "-- processing region: 629 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_629.fasta\n",
      "--- *blasting region: 629\n",
      "--- total time for blast 629: 19.208998441696167\n",
      "-- processing region: 63 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_63.fasta\n",
      "--- *blasting region: 63\n",
      "--- total time for blast 63: 26.005005598068237\n",
      "-- processing region: 630 with 397 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_630.fasta\n",
      "--- *blasting region: 630\n",
      "--- total time for blast 630: 20.60899519920349\n",
      "-- processing region: 631 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_631.fasta\n",
      "--- *blasting region: 631\n",
      "--- total time for blast 631: 22.220995664596558\n",
      "-- processing region: 632 with 395 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_632.fasta\n",
      "--- *blasting region: 632\n",
      "--- total time for blast 632: 18.783002376556396\n",
      "-- processing region: 633 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_633.fasta\n",
      "--- *blasting region: 633\n",
      "--- total time for blast 633: 19.85099983215332\n",
      "-- processing region: 634 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_634.fasta\n",
      "--- *blasting region: 634\n",
      "--- total time for blast 634: 21.317999124526978\n",
      "-- processing region: 635 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_635.fasta\n",
      "--- *blasting region: 635\n",
      "--- total time for blast 635: 22.715999364852905\n",
      "-- processing region: 636 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_636.fasta\n",
      "--- *blasting region: 636\n",
      "--- total time for blast 636: 22.451995372772217\n",
      "-- processing region: 637 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_637.fasta\n",
      "--- *blasting region: 637\n",
      "--- total time for blast 637: 23.240005254745483\n",
      "-- processing region: 638 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_638.fasta\n",
      "--- *blasting region: 638\n",
      "--- total time for blast 638: 22.987990379333496\n",
      "-- processing region: 639 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_639.fasta\n",
      "--- *blasting region: 639\n",
      "--- total time for blast 639: 20.186000108718872\n",
      "-- processing region: 64 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_64.fasta\n",
      "--- *blasting region: 64\n",
      "--- total time for blast 64: 30.525992155075073\n",
      "-- processing region: 640 with 397 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_640.fasta\n",
      "--- *blasting region: 640\n",
      "--- total time for blast 640: 21.83199977874756\n",
      "-- processing region: 641 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_641.fasta\n",
      "--- *blasting region: 641\n",
      "--- total time for blast 641: 21.09600329399109\n",
      "-- processing region: 642 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_642.fasta\n",
      "--- *blasting region: 642\n",
      "--- total time for blast 642: 19.38900327682495\n",
      "-- processing region: 643 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_643.fasta\n",
      "--- *blasting region: 643\n",
      "--- total time for blast 643: 16.673001527786255\n",
      "-- processing region: 644 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_644.fasta\n",
      "--- *blasting region: 644\n",
      "--- total time for blast 644: 19.0330011844635\n",
      "-- processing region: 645 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_645.fasta\n",
      "--- *blasting region: 645\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 645: 25.634000778198242\n",
      "-- processing region: 646 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_646.fasta\n",
      "--- *blasting region: 646\n",
      "--- total time for blast 646: 21.54800033569336\n",
      "-- processing region: 647 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_647.fasta\n",
      "--- *blasting region: 647\n",
      "--- total time for blast 647: 18.902992963790894\n",
      "-- processing region: 648 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_648.fasta\n",
      "--- *blasting region: 648\n",
      "--- total time for blast 648: 18.980002403259277\n",
      "-- processing region: 649 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_649.fasta\n",
      "--- *blasting region: 649\n",
      "--- total time for blast 649: 19.51399874687195\n",
      "-- processing region: 65 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_65.fasta\n",
      "--- *blasting region: 65\n",
      "--- total time for blast 65: 27.332003116607666\n",
      "-- processing region: 650 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_650.fasta\n",
      "--- *blasting region: 650\n",
      "--- total time for blast 650: 27.32999014854431\n",
      "-- processing region: 651 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_651.fasta\n",
      "--- *blasting region: 651\n",
      "--- total time for blast 651: 18.832005977630615\n",
      "-- processing region: 652 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_652.fasta\n",
      "--- *blasting region: 652\n",
      "--- total time for blast 652: 18.44099712371826\n",
      "-- processing region: 653 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_653.fasta\n",
      "--- *blasting region: 653\n",
      "--- total time for blast 653: 18.965999841690063\n",
      "-- processing region: 654 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_654.fasta\n",
      "--- *blasting region: 654\n",
      "--- total time for blast 654: 21.180996894836426\n",
      "-- processing region: 655 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_655.fasta\n",
      "--- *blasting region: 655\n",
      "--- total time for blast 655: 16.36999750137329\n",
      "-- processing region: 656 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_656.fasta\n",
      "--- *blasting region: 656\n",
      "--- total time for blast 656: 16.480002403259277\n",
      "-- processing region: 657 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_657.fasta\n",
      "--- *blasting region: 657\n",
      "--- total time for blast 657: 28.12100124359131\n",
      "-- processing region: 658 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_658.fasta\n",
      "--- *blasting region: 658\n",
      "--- total time for blast 658: 21.629996299743652\n",
      "-- processing region: 659 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_659.fasta\n",
      "--- *blasting region: 659\n",
      "--- total time for blast 659: 21.757999897003174\n",
      "-- processing region: 66 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_66.fasta\n",
      "--- *blasting region: 66\n",
      "--- total time for blast 66: 20.08000135421753\n",
      "-- processing region: 660 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_660.fasta\n",
      "--- *blasting region: 660\n",
      "--- total time for blast 660: 21.677001476287842\n",
      "-- processing region: 661 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_661.fasta\n",
      "--- *blasting region: 661\n",
      "--- total time for blast 661: 17.415998220443726\n",
      "-- processing region: 662 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_662.fasta\n",
      "--- *blasting region: 662\n",
      "--- total time for blast 662: 18.965001583099365\n",
      "-- processing region: 663 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_663.fasta\n",
      "--- *blasting region: 663\n",
      "--- total time for blast 663: 20.493001222610474\n",
      "-- processing region: 664 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_664.fasta\n",
      "--- *blasting region: 664\n",
      "--- total time for blast 664: 18.772994995117188\n",
      "-- processing region: 665 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_665.fasta\n",
      "--- *blasting region: 665\n",
      "--- total time for blast 665: 19.246004581451416\n",
      "-- processing region: 666 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_666.fasta\n",
      "--- *blasting region: 666\n",
      "--- total time for blast 666: 20.238832235336304\n",
      "-- processing region: 667 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_667.fasta\n",
      "--- *blasting region: 667\n",
      "--- total time for blast 667: 19.212007761001587\n",
      "-- processing region: 668 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_668.fasta\n",
      "--- *blasting region: 668\n",
      "--- total time for blast 668: 19.328998565673828\n",
      "-- processing region: 669 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_669.fasta\n",
      "--- *blasting region: 669\n",
      "--- total time for blast 669: 21.490992307662964\n",
      "-- processing region: 67 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_67.fasta\n",
      "--- *blasting region: 67\n",
      "--- total time for blast 67: 24.80500555038452\n",
      "-- processing region: 670 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_670.fasta\n",
      "--- *blasting region: 670\n",
      "--- total time for blast 670: 24.173999071121216\n",
      "-- processing region: 671 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_671.fasta\n",
      "--- *blasting region: 671\n",
      "--- total time for blast 671: 22.003995656967163\n",
      "-- processing region: 672 with 398 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_672.fasta\n",
      "--- *blasting region: 672\n",
      "--- total time for blast 672: 22.310003757476807\n",
      "-- processing region: 673 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_673.fasta\n",
      "--- *blasting region: 673\n",
      "--- total time for blast 673: 20.468998908996582\n",
      "-- processing region: 674 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_674.fasta\n",
      "--- *blasting region: 674\n",
      "--- total time for blast 674: 22.880000114440918\n",
      "-- processing region: 675 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_675.fasta\n",
      "--- *blasting region: 675\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 675: 19.80799627304077\n",
      "-- processing region: 676 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_676.fasta\n",
      "--- *blasting region: 676\n",
      "--- total time for blast 676: 24.856002807617188\n",
      "-- processing region: 677 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_677.fasta\n",
      "--- *blasting region: 677\n",
      "--- total time for blast 677: 18.97099995613098\n",
      "-- processing region: 678 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_678.fasta\n",
      "--- *blasting region: 678\n",
      "--- total time for blast 678: 23.386000871658325\n",
      "-- processing region: 679 with 399 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_679.fasta\n",
      "--- *blasting region: 679\n",
      "--- total time for blast 679: 23.36399817466736\n",
      "-- processing region: 68 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_68.fasta\n",
      "--- *blasting region: 68\n",
      "--- total time for blast 68: 18.410996437072754\n",
      "-- processing region: 680 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_680.fasta\n",
      "--- *blasting region: 680\n",
      "--- total time for blast 680: 20.355000495910645\n",
      "-- processing region: 681 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_681.fasta\n",
      "--- *blasting region: 681\n",
      "--- total time for blast 681: 18.518003940582275\n",
      "-- processing region: 682 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_682.fasta\n",
      "--- *blasting region: 682\n",
      "--- total time for blast 682: 19.900001764297485\n",
      "-- processing region: 683 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_683.fasta\n",
      "--- *blasting region: 683\n",
      "--- total time for blast 683: 26.2729971408844\n",
      "-- processing region: 684 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_684.fasta\n",
      "--- *blasting region: 684\n",
      "--- total time for blast 684: 17.55899715423584\n",
      "-- processing region: 685 with 399 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_685.fasta\n",
      "--- *blasting region: 685\n",
      "--- total time for blast 685: 17.166999340057373\n",
      "-- processing region: 687 with 397 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_687.fasta\n",
      "--- *blasting region: 687\n",
      "--- total time for blast 687: 21.805000066757202\n",
      "-- processing region: 688 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_688.fasta\n",
      "--- *blasting region: 688\n",
      "--- total time for blast 688: 23.442999601364136\n",
      "-- processing region: 689 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_689.fasta\n",
      "--- *blasting region: 689\n",
      "--- total time for blast 689: 23.071998596191406\n",
      "-- processing region: 69 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_69.fasta\n",
      "--- *blasting region: 69\n",
      "--- total time for blast 69: 19.358001470565796\n",
      "-- processing region: 690 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_690.fasta\n",
      "--- *blasting region: 690\n",
      "--- total time for blast 690: 19.79000210762024\n",
      "-- processing region: 691 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_691.fasta\n",
      "--- *blasting region: 691\n",
      "--- total time for blast 691: 19.66300058364868\n",
      "-- processing region: 692 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_692.fasta\n",
      "--- *blasting region: 692\n",
      "--- total time for blast 692: 21.96100616455078\n",
      "-- processing region: 693 with 395 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_693.fasta\n",
      "--- *blasting region: 693\n",
      "--- total time for blast 693: 22.48499894142151\n",
      "-- processing region: 694 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_694.fasta\n",
      "--- *blasting region: 694\n",
      "--- total time for blast 694: 19.077993392944336\n",
      "-- processing region: 695 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_695.fasta\n",
      "--- *blasting region: 695\n",
      "--- total time for blast 695: 24.089991807937622\n",
      "-- processing region: 696 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_696.fasta\n",
      "--- *blasting region: 696\n",
      "--- total time for blast 696: 20.960008144378662\n",
      "-- processing region: 697 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_697.fasta\n",
      "--- *blasting region: 697\n",
      "--- total time for blast 697: 23.64699673652649\n",
      "-- processing region: 698 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_698.fasta\n",
      "--- *blasting region: 698\n",
      "--- total time for blast 698: 21.32800054550171\n",
      "-- processing region: 699 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_699.fasta\n",
      "--- *blasting region: 699\n",
      "--- total time for blast 699: 19.0479998588562\n",
      "-- processing region: 7 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_7.fasta\n",
      "--- *blasting region: 7\n",
      "--- total time for blast 7: 22.703994274139404\n",
      "-- processing region: 70 with 398 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_70.fasta\n",
      "--- *blasting region: 70\n",
      "--- total time for blast 70: 15.081998348236084\n",
      "-- processing region: 700 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_700.fasta\n",
      "--- *blasting region: 700\n",
      "--- total time for blast 700: 18.059003591537476\n",
      "-- processing region: 701 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_701.fasta\n",
      "--- *blasting region: 701\n",
      "--- total time for blast 701: 19.272999048233032\n",
      "-- processing region: 702 with 385 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_702.fasta\n",
      "--- *blasting region: 702\n",
      "--- total time for blast 702: 19.278002738952637\n",
      "-- processing region: 703 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_703.fasta\n",
      "--- *blasting region: 703\n",
      "--- total time for blast 703: 16.496000051498413\n",
      "-- processing region: 704 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_704.fasta\n",
      "--- *blasting region: 704\n",
      "--- total time for blast 704: 24.47999405860901\n",
      "-- processing region: 705 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_705.fasta\n",
      "--- *blasting region: 705\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 705: 20.59500551223755\n",
      "-- processing region: 706 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_706.fasta\n",
      "--- *blasting region: 706\n",
      "--- total time for blast 706: 20.11000156402588\n",
      "-- processing region: 707 with 399 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_707.fasta\n",
      "--- *blasting region: 707\n",
      "--- total time for blast 707: 25.738994359970093\n",
      "-- processing region: 708 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_708.fasta\n",
      "--- *blasting region: 708\n",
      "--- total time for blast 708: 21.432005405426025\n",
      "-- processing region: 709 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_709.fasta\n",
      "--- *blasting region: 709\n",
      "--- total time for blast 709: 23.731001615524292\n",
      "-- processing region: 71 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_71.fasta\n",
      "--- *blasting region: 71\n",
      "--- total time for blast 71: 20.159000635147095\n",
      "-- processing region: 710 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_710.fasta\n",
      "--- *blasting region: 710\n",
      "--- total time for blast 710: 20.531999349594116\n",
      "-- processing region: 711 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_711.fasta\n",
      "--- *blasting region: 711\n",
      "--- total time for blast 711: 21.91499161720276\n",
      "-- processing region: 712 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_712.fasta\n",
      "--- *blasting region: 712\n",
      "--- total time for blast 712: 18.022006273269653\n",
      "-- processing region: 713 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_713.fasta\n",
      "--- *blasting region: 713\n",
      "--- total time for blast 713: 20.118998765945435\n",
      "-- processing region: 714 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_714.fasta\n",
      "--- *blasting region: 714\n",
      "--- total time for blast 714: 18.12399959564209\n",
      "-- processing region: 715 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_715.fasta\n",
      "--- *blasting region: 715\n",
      "--- total time for blast 715: 21.835004806518555\n",
      "-- processing region: 716 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_716.fasta\n",
      "--- *blasting region: 716\n",
      "--- total time for blast 716: 19.934998035430908\n",
      "-- processing region: 717 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_717.fasta\n",
      "--- *blasting region: 717\n",
      "--- total time for blast 717: 22.55099391937256\n",
      "-- processing region: 718 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_718.fasta\n",
      "--- *blasting region: 718\n",
      "--- total time for blast 718: 19.25999665260315\n",
      "-- processing region: 719 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_719.fasta\n",
      "--- *blasting region: 719\n",
      "--- total time for blast 719: 17.52999472618103\n",
      "-- processing region: 72 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_72.fasta\n",
      "--- *blasting region: 72\n",
      "--- total time for blast 72: 22.279000282287598\n",
      "-- processing region: 720 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_720.fasta\n",
      "--- *blasting region: 720\n",
      "--- total time for blast 720: 20.189003229141235\n",
      "-- processing region: 721 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_721.fasta\n",
      "--- *blasting region: 721\n",
      "--- total time for blast 721: 23.3759868144989\n",
      "-- processing region: 722 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_722.fasta\n",
      "--- *blasting region: 722\n",
      "--- total time for blast 722: 24.35800337791443\n",
      "-- processing region: 723 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_723.fasta\n",
      "--- *blasting region: 723\n",
      "--- total time for blast 723: 22.4209942817688\n",
      "-- processing region: 724 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_724.fasta\n",
      "--- *blasting region: 724\n",
      "--- total time for blast 724: 20.08699941635132\n",
      "-- processing region: 725 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_725.fasta\n",
      "--- *blasting region: 725\n",
      "--- total time for blast 725: 22.263002157211304\n",
      "-- processing region: 726 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_726.fasta\n",
      "--- *blasting region: 726\n",
      "--- total time for blast 726: 25.471003532409668\n",
      "-- processing region: 727 with 389 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_727.fasta\n",
      "--- *blasting region: 727\n",
      "--- total time for blast 727: 21.719997882843018\n",
      "-- processing region: 728 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_728.fasta\n",
      "--- *blasting region: 728\n",
      "--- total time for blast 728: 25.45899987220764\n",
      "-- processing region: 729 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_729.fasta\n",
      "--- *blasting region: 729\n",
      "--- total time for blast 729: 18.450008153915405\n",
      "-- processing region: 73 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_73.fasta\n",
      "--- *blasting region: 73\n",
      "--- total time for blast 73: 21.450990915298462\n",
      "-- processing region: 730 with 397 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_730.fasta\n",
      "--- *blasting region: 730\n",
      "--- total time for blast 730: 25.246002197265625\n",
      "-- processing region: 731 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_731.fasta\n",
      "--- *blasting region: 731\n",
      "--- total time for blast 731: 23.165995597839355\n",
      "-- processing region: 732 with 391 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_732.fasta\n",
      "--- *blasting region: 732\n",
      "--- total time for blast 732: 21.048006057739258\n",
      "-- processing region: 733 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_733.fasta\n",
      "--- *blasting region: 733\n",
      "--- total time for blast 733: 24.62899899482727\n",
      "-- processing region: 734 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_734.fasta\n",
      "--- *blasting region: 734\n",
      "--- total time for blast 734: 21.20087695121765\n",
      "-- processing region: 735 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_735.fasta\n",
      "--- *blasting region: 735\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 735: 19.290997982025146\n",
      "-- processing region: 736 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_736.fasta\n",
      "--- *blasting region: 736\n",
      "--- total time for blast 736: 23.29500436782837\n",
      "-- processing region: 737 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_737.fasta\n",
      "--- *blasting region: 737\n",
      "--- total time for blast 737: 24.623006343841553\n",
      "-- processing region: 738 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_738.fasta\n",
      "--- *blasting region: 738\n",
      "--- total time for blast 738: 18.361997842788696\n",
      "-- processing region: 739 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_739.fasta\n",
      "--- *blasting region: 739\n",
      "--- total time for blast 739: 18.187999486923218\n",
      "-- processing region: 74 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_74.fasta\n",
      "--- *blasting region: 74\n",
      "--- total time for blast 74: 21.93700122833252\n",
      "-- processing region: 740 with 388 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_740.fasta\n",
      "--- *blasting region: 740\n",
      "--- total time for blast 740: 20.251996517181396\n",
      "-- processing region: 741 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_741.fasta\n",
      "--- *blasting region: 741\n",
      "--- total time for blast 741: 18.307005405426025\n",
      "-- processing region: 742 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_742.fasta\n",
      "--- *blasting region: 742\n",
      "--- total time for blast 742: 23.5809965133667\n",
      "-- processing region: 743 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_743.fasta\n",
      "--- *blasting region: 743\n",
      "--- total time for blast 743: 21.255996227264404\n",
      "-- processing region: 744 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_744.fasta\n",
      "--- *blasting region: 744\n",
      "--- total time for blast 744: 28.413000106811523\n",
      "-- processing region: 745 with 396 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_745.fasta\n",
      "--- *blasting region: 745\n",
      "--- total time for blast 745: 22.9399995803833\n",
      "-- processing region: 746 with 399 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_746.fasta\n",
      "--- *blasting region: 746\n",
      "--- total time for blast 746: 22.091997861862183\n",
      "-- processing region: 747 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_747.fasta\n",
      "--- *blasting region: 747\n",
      "--- total time for blast 747: 21.5999972820282\n",
      "-- processing region: 748 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_748.fasta\n",
      "--- *blasting region: 748\n",
      "--- total time for blast 748: 15.333002805709839\n",
      "-- processing region: 749 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_749.fasta\n",
      "--- *blasting region: 749\n",
      "--- total time for blast 749: 19.09300184249878\n",
      "-- processing region: 75 with 396 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_75.fasta\n",
      "--- *blasting region: 75\n",
      "--- total time for blast 75: 19.878995895385742\n",
      "-- processing region: 750 with 10 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_750.fasta\n",
      "--- *blasting region: 750\n",
      "--- total time for blast 750: 3.992992401123047\n",
      "-- processing region: 751 with 10 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_751.fasta\n",
      "--- *blasting region: 751\n",
      "--- total time for blast 751: 1.6830050945281982\n",
      "-- processing region: 752 with 176 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_752.fasta\n",
      "--- *blasting region: 752\n",
      "--- total time for blast 752: 11.334996461868286\n",
      "-- processing region: 758 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_758.fasta\n",
      "--- *blasting region: 758\n",
      "--- total time for blast 758: 24.020002841949463\n",
      "-- processing region: 759 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_759.fasta\n",
      "--- *blasting region: 759\n",
      "--- total time for blast 759: 19.694000720977783\n",
      "-- processing region: 76 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_76.fasta\n",
      "--- *blasting region: 76\n",
      "--- total time for blast 76: 19.563995599746704\n",
      "-- processing region: 760 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_760.fasta\n",
      "--- *blasting region: 760\n",
      "--- total time for blast 760: 22.274996757507324\n",
      "-- processing region: 761 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_761.fasta\n",
      "--- *blasting region: 761\n",
      "--- total time for blast 761: 23.103999853134155\n",
      "-- processing region: 762 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_762.fasta\n",
      "--- *blasting region: 762\n",
      "--- total time for blast 762: 22.720995903015137\n",
      "-- processing region: 763 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_763.fasta\n",
      "--- *blasting region: 763\n",
      "--- total time for blast 763: 20.892995834350586\n",
      "-- processing region: 764 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_764.fasta\n",
      "--- *blasting region: 764\n",
      "--- total time for blast 764: 22.28299856185913\n",
      "-- processing region: 765 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_765.fasta\n",
      "--- *blasting region: 765\n",
      "--- total time for blast 765: 18.994998931884766\n",
      "-- processing region: 766 with 397 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_766.fasta\n",
      "--- *blasting region: 766\n",
      "--- total time for blast 766: 21.555005073547363\n",
      "-- processing region: 767 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_767.fasta\n",
      "--- *blasting region: 767\n",
      "--- total time for blast 767: 16.919997453689575\n",
      "-- processing region: 768 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_768.fasta\n",
      "--- *blasting region: 768\n",
      "--- total time for blast 768: 18.484999656677246\n",
      "-- processing region: 769 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_769.fasta\n",
      "--- *blasting region: 769\n",
      "--- total time for blast 769: 19.20499897003174\n",
      "-- processing region: 77 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_77.fasta\n",
      "--- *blasting region: 77\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 77: 22.16200351715088\n",
      "-- processing region: 770 with 395 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_770.fasta\n",
      "--- *blasting region: 770\n",
      "--- total time for blast 770: 28.144004821777344\n",
      "-- processing region: 772 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_772.fasta\n",
      "--- *blasting region: 772\n",
      "--- total time for blast 772: 18.067042589187622\n",
      "-- processing region: 773 with 393 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_773.fasta\n",
      "--- *blasting region: 773\n",
      "--- total time for blast 773: 20.515958070755005\n",
      "-- processing region: 774 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_774.fasta\n",
      "--- *blasting region: 774\n",
      "--- total time for blast 774: 17.021000146865845\n",
      "-- processing region: 775 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_775.fasta\n",
      "--- *blasting region: 775\n",
      "--- total time for blast 775: 18.751001358032227\n",
      "-- processing region: 776 with 62 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_776.fasta\n",
      "--- *blasting region: 776\n",
      "--- total time for blast 776: 5.5310022830963135\n",
      "-- processing region: 777 with 395 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_777.fasta\n",
      "--- *blasting region: 777\n",
      "--- total time for blast 777: 21.98799467086792\n",
      "-- processing region: 778 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_778.fasta\n",
      "--- *blasting region: 778\n",
      "--- total time for blast 778: 18.404998302459717\n",
      "-- processing region: 779 with 393 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_779.fasta\n",
      "--- *blasting region: 779\n",
      "--- total time for blast 779: 25.566006898880005\n",
      "-- processing region: 78 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_78.fasta\n",
      "--- *blasting region: 78\n",
      "--- total time for blast 78: 21.194992542266846\n",
      "-- processing region: 780 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_780.fasta\n",
      "--- *blasting region: 780\n",
      "--- total time for blast 780: 20.057003498077393\n",
      "-- processing region: 781 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_781.fasta\n",
      "--- *blasting region: 781\n",
      "--- total time for blast 781: 18.133999586105347\n",
      "-- processing region: 782 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_782.fasta\n",
      "--- *blasting region: 782\n",
      "--- total time for blast 782: 19.996002674102783\n",
      "-- processing region: 783 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_783.fasta\n",
      "--- *blasting region: 783\n",
      "--- total time for blast 783: 20.983999967575073\n",
      "-- processing region: 784 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_784.fasta\n",
      "--- *blasting region: 784\n",
      "--- total time for blast 784: 27.18899703025818\n",
      "-- processing region: 785 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_785.fasta\n",
      "--- *blasting region: 785\n",
      "--- total time for blast 785: 22.528998851776123\n",
      "-- processing region: 786 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_786.fasta\n",
      "--- *blasting region: 786\n",
      "--- total time for blast 786: 23.619003772735596\n",
      "-- processing region: 787 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_787.fasta\n",
      "--- *blasting region: 787\n",
      "--- total time for blast 787: 22.253003358840942\n",
      "-- processing region: 788 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_788.fasta\n",
      "--- *blasting region: 788\n",
      "--- total time for blast 788: 18.60500192642212\n",
      "-- processing region: 789 with 398 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_789.fasta\n",
      "--- *blasting region: 789\n",
      "--- total time for blast 789: 20.99199342727661\n",
      "-- processing region: 79 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_79.fasta\n",
      "--- *blasting region: 79\n",
      "--- total time for blast 79: 17.247989416122437\n",
      "-- processing region: 790 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_790.fasta\n",
      "--- *blasting region: 790\n",
      "--- total time for blast 790: 23.628002166748047\n",
      "-- processing region: 791 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_791.fasta\n",
      "--- *blasting region: 791\n",
      "--- total time for blast 791: 18.211005926132202\n",
      "-- processing region: 792 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_792.fasta\n",
      "--- *blasting region: 792\n",
      "--- total time for blast 792: 22.05799388885498\n",
      "-- processing region: 793 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_793.fasta\n",
      "--- *blasting region: 793\n",
      "--- total time for blast 793: 24.098002433776855\n",
      "-- processing region: 794 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_794.fasta\n",
      "--- *blasting region: 794\n",
      "--- total time for blast 794: 19.880000114440918\n",
      "-- processing region: 795 with 398 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_795.fasta\n",
      "--- *blasting region: 795\n",
      "--- total time for blast 795: 18.969000339508057\n",
      "-- processing region: 796 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_796.fasta\n",
      "--- *blasting region: 796\n",
      "--- total time for blast 796: 23.47799515724182\n",
      "-- processing region: 797 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_797.fasta\n",
      "--- *blasting region: 797\n",
      "--- total time for blast 797: 18.17500352859497\n",
      "-- processing region: 798 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_798.fasta\n",
      "--- *blasting region: 798\n",
      "--- total time for blast 798: 19.427000999450684\n",
      "-- processing region: 799 with 394 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_799.fasta\n",
      "--- *blasting region: 799\n",
      "--- total time for blast 799: 22.264997720718384\n",
      "-- processing region: 8 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_8.fasta\n",
      "--- *blasting region: 8\n",
      "--- total time for blast 8: 17.82200598716736\n",
      "-- processing region: 80 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_80.fasta\n",
      "--- *blasting region: 80\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 80: 23.686004161834717\n",
      "-- processing region: 800 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_800.fasta\n",
      "--- *blasting region: 800\n",
      "--- total time for blast 800: 17.485002040863037\n",
      "-- processing region: 801 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_801.fasta\n",
      "--- *blasting region: 801\n",
      "--- total time for blast 801: 23.689998149871826\n",
      "-- processing region: 802 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_802.fasta\n",
      "--- *blasting region: 802\n",
      "--- total time for blast 802: 19.53100275993347\n",
      "-- processing region: 803 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_803.fasta\n",
      "--- *blasting region: 803\n",
      "--- total time for blast 803: 21.944002151489258\n",
      "-- processing region: 804 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_804.fasta\n",
      "--- *blasting region: 804\n",
      "--- total time for blast 804: 26.735002279281616\n",
      "-- processing region: 805 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_805.fasta\n",
      "--- *blasting region: 805\n",
      "--- total time for blast 805: 20.824996948242188\n",
      "-- processing region: 806 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_806.fasta\n",
      "--- *blasting region: 806\n",
      "--- total time for blast 806: 20.939998865127563\n",
      "-- processing region: 807 with 397 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_807.fasta\n",
      "--- *blasting region: 807\n",
      "--- total time for blast 807: 21.597997665405273\n",
      "-- processing region: 808 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_808.fasta\n",
      "--- *blasting region: 808\n",
      "--- total time for blast 808: 20.20600938796997\n",
      "-- processing region: 809 with 397 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_809.fasta\n",
      "--- *blasting region: 809\n",
      "--- total time for blast 809: 23.03699278831482\n",
      "-- processing region: 81 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_81.fasta\n",
      "--- *blasting region: 81\n",
      "--- total time for blast 81: 33.30795335769653\n",
      "-- processing region: 810 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_810.fasta\n",
      "--- *blasting region: 810\n",
      "--- total time for blast 810: 23.64799976348877\n",
      "-- processing region: 811 with 397 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_811.fasta\n",
      "--- *blasting region: 811\n",
      "--- total time for blast 811: 21.0870041847229\n",
      "-- processing region: 812 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_812.fasta\n",
      "--- *blasting region: 812\n",
      "--- total time for blast 812: 17.371994256973267\n",
      "-- processing region: 813 with 396 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_813.fasta\n",
      "--- *blasting region: 813\n",
      "--- total time for blast 813: 17.394001483917236\n",
      "-- processing region: 814 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_814.fasta\n",
      "--- *blasting region: 814\n",
      "--- total time for blast 814: 21.10499858856201\n",
      "-- processing region: 815 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_815.fasta\n",
      "--- *blasting region: 815\n",
      "--- total time for blast 815: 26.566997528076172\n",
      "-- processing region: 816 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_816.fasta\n",
      "--- *blasting region: 816\n",
      "--- total time for blast 816: 17.949005603790283\n",
      "-- processing region: 817 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_817.fasta\n",
      "--- *blasting region: 817\n",
      "--- total time for blast 817: 22.84500479698181\n",
      "-- processing region: 818 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_818.fasta\n",
      "--- *blasting region: 818\n",
      "--- total time for blast 818: 21.350993156433105\n",
      "-- processing region: 819 with 395 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_819.fasta\n",
      "--- *blasting region: 819\n",
      "--- total time for blast 819: 16.25700283050537\n",
      "-- processing region: 82 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_82.fasta\n",
      "--- *blasting region: 82\n",
      "--- total time for blast 82: 20.398998260498047\n",
      "-- processing region: 820 with 398 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_820.fasta\n",
      "--- *blasting region: 820\n",
      "--- total time for blast 820: 25.285001754760742\n",
      "-- processing region: 821 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_821.fasta\n",
      "--- *blasting region: 821\n",
      "--- total time for blast 821: 17.243001699447632\n",
      "-- processing region: 822 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_822.fasta\n",
      "--- *blasting region: 822\n",
      "--- total time for blast 822: 24.20199680328369\n",
      "-- processing region: 823 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_823.fasta\n",
      "--- *blasting region: 823\n",
      "--- total time for blast 823: 24.58699655532837\n",
      "-- processing region: 824 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_824.fasta\n",
      "--- *blasting region: 824\n",
      "--- total time for blast 824: 19.740998029708862\n",
      "-- processing region: 825 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_825.fasta\n",
      "--- *blasting region: 825\n",
      "--- total time for blast 825: 23.015997648239136\n",
      "-- processing region: 826 with 398 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_826.fasta\n",
      "--- *blasting region: 826\n",
      "--- total time for blast 826: 18.48800230026245\n",
      "-- processing region: 827 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_827.fasta\n",
      "--- *blasting region: 827\n",
      "--- total time for blast 827: 22.89300537109375\n",
      "-- processing region: 828 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_828.fasta\n",
      "--- *blasting region: 828\n",
      "--- total time for blast 828: 32.94499611854553\n",
      "-- processing region: 829 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_829.fasta\n",
      "--- *blasting region: 829\n",
      "--- total time for blast 829: 22.846001148223877\n",
      "-- processing region: 83 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_83.fasta\n",
      "--- *blasting region: 83\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 83: 18.2479989528656\n",
      "-- processing region: 830 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_830.fasta\n",
      "--- *blasting region: 830\n",
      "--- total time for blast 830: 21.253000736236572\n",
      "-- processing region: 831 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_831.fasta\n",
      "--- *blasting region: 831\n",
      "--- total time for blast 831: 19.77999997138977\n",
      "-- processing region: 832 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_832.fasta\n",
      "--- *blasting region: 832\n",
      "--- total time for blast 832: 19.68900227546692\n",
      "-- processing region: 833 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_833.fasta\n",
      "--- *blasting region: 833\n",
      "--- total time for blast 833: 20.21300196647644\n",
      "-- processing region: 834 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_834.fasta\n",
      "--- *blasting region: 834\n",
      "--- total time for blast 834: 28.164994716644287\n",
      "-- processing region: 835 with 397 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_835.fasta\n",
      "--- *blasting region: 835\n",
      "--- total time for blast 835: 24.004006147384644\n",
      "-- processing region: 836 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_836.fasta\n",
      "--- *blasting region: 836\n",
      "--- total time for blast 836: 22.28800058364868\n",
      "-- processing region: 837 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_837.fasta\n",
      "--- *blasting region: 837\n",
      "--- total time for blast 837: 28.959999799728394\n",
      "-- processing region: 838 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_838.fasta\n",
      "--- *blasting region: 838\n",
      "--- total time for blast 838: 22.72099280357361\n",
      "-- processing region: 839 with 399 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_839.fasta\n",
      "--- *blasting region: 839\n",
      "--- total time for blast 839: 20.43599820137024\n",
      "-- processing region: 84 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_84.fasta\n",
      "--- *blasting region: 84\n",
      "--- total time for blast 84: 25.024006843566895\n",
      "-- processing region: 840 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_840.fasta\n",
      "--- *blasting region: 840\n",
      "--- total time for blast 840: 18.061001539230347\n",
      "-- processing region: 841 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_841.fasta\n",
      "--- *blasting region: 841\n",
      "--- total time for blast 841: 21.662991523742676\n",
      "-- processing region: 842 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_842.fasta\n",
      "--- *blasting region: 842\n",
      "--- total time for blast 842: 29.44100260734558\n",
      "-- processing region: 843 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_843.fasta\n",
      "--- *blasting region: 843\n",
      "--- total time for blast 843: 20.385995864868164\n",
      "-- processing region: 844 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_844.fasta\n",
      "--- *blasting region: 844\n",
      "--- total time for blast 844: 20.11100697517395\n",
      "-- processing region: 845 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_845.fasta\n",
      "--- *blasting region: 845\n",
      "--- total time for blast 845: 22.453996419906616\n",
      "-- processing region: 846 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_846.fasta\n",
      "--- *blasting region: 846\n",
      "--- total time for blast 846: 22.331000566482544\n",
      "-- processing region: 847 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_847.fasta\n",
      "--- *blasting region: 847\n",
      "--- total time for blast 847: 31.796000003814697\n",
      "-- processing region: 848 with 392 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_848.fasta\n",
      "--- *blasting region: 848\n",
      "--- total time for blast 848: 24.12400221824646\n",
      "-- processing region: 849 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_849.fasta\n",
      "--- *blasting region: 849\n",
      "--- total time for blast 849: 22.31000590324402\n",
      "-- processing region: 85 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_85.fasta\n",
      "--- *blasting region: 85\n",
      "--- total time for blast 85: 20.062995672225952\n",
      "-- processing region: 850 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_850.fasta\n",
      "--- *blasting region: 850\n",
      "--- total time for blast 850: 19.659995794296265\n",
      "-- processing region: 851 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_851.fasta\n",
      "--- *blasting region: 851\n",
      "--- total time for blast 851: 23.17199993133545\n",
      "-- processing region: 852 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_852.fasta\n",
      "--- *blasting region: 852\n",
      "--- total time for blast 852: 18.30800437927246\n",
      "-- processing region: 853 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_853.fasta\n",
      "--- *blasting region: 853\n",
      "--- total time for blast 853: 19.941997528076172\n",
      "-- processing region: 854 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_854.fasta\n",
      "--- *blasting region: 854\n",
      "--- total time for blast 854: 32.391910552978516\n",
      "-- processing region: 855 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_855.fasta\n",
      "--- *blasting region: 855\n",
      "--- total time for blast 855: 23.942999124526978\n",
      "-- processing region: 856 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_856.fasta\n",
      "--- *blasting region: 856\n",
      "--- total time for blast 856: 22.93500328063965\n",
      "-- processing region: 857 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_857.fasta\n",
      "--- *blasting region: 857\n",
      "--- total time for blast 857: 20.52199649810791\n",
      "-- processing region: 858 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_858.fasta\n",
      "--- *blasting region: 858\n",
      "--- total time for blast 858: 22.030994176864624\n",
      "-- processing region: 859 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_859.fasta\n",
      "--- *blasting region: 859\n",
      "--- total time for blast 859: 27.858006954193115\n",
      "-- processing region: 86 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_86.fasta\n",
      "--- *blasting region: 86\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 86: 24.695997714996338\n",
      "-- processing region: 860 with 399 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_860.fasta\n",
      "--- *blasting region: 860\n",
      "--- total time for blast 860: 19.443997859954834\n",
      "-- processing region: 861 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_861.fasta\n",
      "--- *blasting region: 861\n",
      "--- total time for blast 861: 19.163002252578735\n",
      "-- processing region: 862 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_862.fasta\n",
      "--- *blasting region: 862\n",
      "--- total time for blast 862: 19.663000345230103\n",
      "-- processing region: 863 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_863.fasta\n",
      "--- *blasting region: 863\n",
      "--- total time for blast 863: 18.94500207901001\n",
      "-- processing region: 864 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_864.fasta\n",
      "--- *blasting region: 864\n",
      "--- total time for blast 864: 20.567996978759766\n",
      "-- processing region: 866 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_866.fasta\n",
      "--- *blasting region: 866\n",
      "--- total time for blast 866: 18.16099500656128\n",
      "-- processing region: 867 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_867.fasta\n",
      "--- *blasting region: 867\n",
      "--- total time for blast 867: 21.75400733947754\n",
      "-- processing region: 868 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_868.fasta\n",
      "--- *blasting region: 868\n",
      "--- total time for blast 868: 18.01900029182434\n",
      "-- processing region: 869 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_869.fasta\n",
      "--- *blasting region: 869\n",
      "--- total time for blast 869: 18.265998363494873\n",
      "-- processing region: 87 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_87.fasta\n",
      "--- *blasting region: 87\n",
      "--- total time for blast 87: 15.627997875213623\n",
      "-- processing region: 870 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_870.fasta\n",
      "--- *blasting region: 870\n",
      "--- total time for blast 870: 22.804000854492188\n",
      "-- processing region: 871 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_871.fasta\n",
      "--- *blasting region: 871\n",
      "--- total time for blast 871: 26.286004543304443\n",
      "-- processing region: 872 with 398 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_872.fasta\n",
      "--- *blasting region: 872\n",
      "--- total time for blast 872: 23.32699489593506\n",
      "-- processing region: 873 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_873.fasta\n",
      "--- *blasting region: 873\n",
      "--- total time for blast 873: 24.600001573562622\n",
      "-- processing region: 874 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_874.fasta\n",
      "--- *blasting region: 874\n",
      "--- total time for blast 874: 22.228001832962036\n",
      "-- processing region: 875 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_875.fasta\n",
      "--- *blasting region: 875\n",
      "--- total time for blast 875: 22.299004554748535\n",
      "-- processing region: 876 with 399 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_876.fasta\n",
      "--- *blasting region: 876\n",
      "--- total time for blast 876: 17.14299464225769\n",
      "-- processing region: 877 with 396 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_877.fasta\n",
      "--- *blasting region: 877\n",
      "--- total time for blast 877: 22.187001943588257\n",
      "-- processing region: 878 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_878.fasta\n",
      "--- *blasting region: 878\n",
      "--- total time for blast 878: 17.531008005142212\n",
      "-- processing region: 879 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_879.fasta\n",
      "--- *blasting region: 879\n",
      "--- total time for blast 879: 21.751992225646973\n",
      "-- processing region: 88 with 396 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_88.fasta\n",
      "--- *blasting region: 88\n",
      "--- total time for blast 88: 20.9710054397583\n",
      "-- processing region: 880 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_880.fasta\n",
      "--- *blasting region: 880\n",
      "--- total time for blast 880: 20.721996307373047\n",
      "-- processing region: 881 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_881.fasta\n",
      "--- *blasting region: 881\n",
      "--- total time for blast 881: 21.341997146606445\n",
      "-- processing region: 882 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_882.fasta\n",
      "--- *blasting region: 882\n",
      "--- total time for blast 882: 21.022997856140137\n",
      "-- processing region: 883 with 399 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_883.fasta\n",
      "--- *blasting region: 883\n",
      "--- total time for blast 883: 19.723008394241333\n",
      "-- processing region: 884 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_884.fasta\n",
      "--- *blasting region: 884\n",
      "--- total time for blast 884: 19.132993936538696\n",
      "-- processing region: 885 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_885.fasta\n",
      "--- *blasting region: 885\n",
      "--- total time for blast 885: 22.67200994491577\n",
      "-- processing region: 886 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_886.fasta\n",
      "--- *blasting region: 886\n",
      "--- total time for blast 886: 17.8499915599823\n",
      "-- processing region: 887 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_887.fasta\n",
      "--- *blasting region: 887\n",
      "--- total time for blast 887: 16.999998331069946\n",
      "-- processing region: 888 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_888.fasta\n",
      "--- *blasting region: 888\n",
      "--- total time for blast 888: 20.606001377105713\n",
      "-- processing region: 889 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_889.fasta\n",
      "--- *blasting region: 889\n",
      "--- total time for blast 889: 20.38199758529663\n",
      "-- processing region: 89 with 396 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_89.fasta\n",
      "--- *blasting region: 89\n",
      "--- total time for blast 89: 21.683001279830933\n",
      "-- processing region: 890 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_890.fasta\n",
      "--- *blasting region: 890\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 890: 22.603004455566406\n",
      "-- processing region: 891 with 398 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_891.fasta\n",
      "--- *blasting region: 891\n",
      "--- total time for blast 891: 18.892990350723267\n",
      "-- processing region: 892 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_892.fasta\n",
      "--- *blasting region: 892\n",
      "--- total time for blast 892: 15.838002681732178\n",
      "-- processing region: 893 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_893.fasta\n",
      "--- *blasting region: 893\n",
      "--- total time for blast 893: 18.037002325057983\n",
      "-- processing region: 894 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_894.fasta\n",
      "--- *blasting region: 894\n",
      "--- total time for blast 894: 21.868002891540527\n",
      "-- processing region: 895 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_895.fasta\n",
      "--- *blasting region: 895\n",
      "--- total time for blast 895: 16.22100305557251\n",
      "-- processing region: 896 with 398 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_896.fasta\n",
      "--- *blasting region: 896\n",
      "--- total time for blast 896: 26.279993295669556\n",
      "-- processing region: 897 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_897.fasta\n",
      "--- *blasting region: 897\n",
      "--- total time for blast 897: 23.632999658584595\n",
      "-- processing region: 898 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_898.fasta\n",
      "--- *blasting region: 898\n",
      "--- total time for blast 898: 20.1789972782135\n",
      "-- processing region: 899 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_899.fasta\n",
      "--- *blasting region: 899\n",
      "--- total time for blast 899: 22.494000911712646\n",
      "-- processing region: 9 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_9.fasta\n",
      "--- *blasting region: 9\n",
      "--- total time for blast 9: 26.765002012252808\n",
      "-- processing region: 90 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_90.fasta\n",
      "--- *blasting region: 90\n",
      "--- total time for blast 90: 22.221004724502563\n",
      "-- processing region: 900 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_900.fasta\n",
      "--- *blasting region: 900\n",
      "--- total time for blast 900: 23.180001497268677\n",
      "-- processing region: 901 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_901.fasta\n",
      "--- *blasting region: 901\n",
      "--- total time for blast 901: 23.558995962142944\n",
      "-- processing region: 902 with 381 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_902.fasta\n",
      "--- *blasting region: 902\n",
      "--- total time for blast 902: 21.43799877166748\n",
      "-- processing region: 903 with 395 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_903.fasta\n",
      "--- *blasting region: 903\n",
      "--- total time for blast 903: 21.679007053375244\n",
      "-- processing region: 904 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_904.fasta\n",
      "--- *blasting region: 904\n",
      "--- total time for blast 904: 23.97599506378174\n",
      "-- processing region: 905 with 399 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_905.fasta\n",
      "--- *blasting region: 905\n",
      "--- total time for blast 905: 19.417004108428955\n",
      "-- processing region: 906 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_906.fasta\n",
      "--- *blasting region: 906\n",
      "--- total time for blast 906: 20.956997394561768\n",
      "-- processing region: 907 with 399 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_907.fasta\n",
      "--- *blasting region: 907\n",
      "--- total time for blast 907: 22.61099886894226\n",
      "-- processing region: 908 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_908.fasta\n",
      "--- *blasting region: 908\n",
      "--- total time for blast 908: 18.73500347137451\n",
      "-- processing region: 909 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_909.fasta\n",
      "--- *blasting region: 909\n",
      "--- total time for blast 909: 30.85399889945984\n",
      "-- processing region: 91 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_91.fasta\n",
      "--- *blasting region: 91\n",
      "--- total time for blast 91: 24.25500226020813\n",
      "-- processing region: 910 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_910.fasta\n",
      "--- *blasting region: 910\n",
      "--- total time for blast 910: 19.92999839782715\n",
      "-- processing region: 911 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_911.fasta\n",
      "--- *blasting region: 911\n",
      "--- total time for blast 911: 25.010995864868164\n",
      "-- processing region: 912 with 398 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_912.fasta\n",
      "--- *blasting region: 912\n",
      "--- total time for blast 912: 26.59599995613098\n",
      "-- processing region: 913 with 398 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_913.fasta\n",
      "--- *blasting region: 913\n",
      "--- total time for blast 913: 19.050999641418457\n",
      "-- processing region: 914 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_914.fasta\n",
      "--- *blasting region: 914\n",
      "--- total time for blast 914: 21.43399977684021\n",
      "-- processing region: 915 with 396 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_915.fasta\n",
      "--- *blasting region: 915\n",
      "--- total time for blast 915: 19.941998720169067\n",
      "-- processing region: 916 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_916.fasta\n",
      "--- *blasting region: 916\n",
      "--- total time for blast 916: 20.81299877166748\n",
      "-- processing region: 917 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_917.fasta\n",
      "--- *blasting region: 917\n",
      "--- total time for blast 917: 22.918999910354614\n",
      "-- processing region: 918 with 396 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_918.fasta\n",
      "--- *blasting region: 918\n",
      "--- total time for blast 918: 29.439531326293945\n",
      "-- processing region: 919 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_919.fasta\n",
      "--- *blasting region: 919\n",
      "--- total time for blast 919: 19.91800594329834\n",
      "-- processing region: 92 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_92.fasta\n",
      "--- *blasting region: 92\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 92: 24.04099202156067\n",
      "-- processing region: 920 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_920.fasta\n",
      "--- *blasting region: 920\n",
      "--- total time for blast 920: 28.77600622177124\n",
      "-- processing region: 921 with 396 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_921.fasta\n",
      "--- *blasting region: 921\n",
      "--- total time for blast 921: 22.741003036499023\n",
      "-- processing region: 922 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_922.fasta\n",
      "--- *blasting region: 922\n",
      "--- total time for blast 922: 21.16900134086609\n",
      "-- processing region: 923 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_923.fasta\n",
      "--- *blasting region: 923\n",
      "--- total time for blast 923: 23.067997694015503\n",
      "-- processing region: 925 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_925.fasta\n",
      "--- *blasting region: 925\n",
      "--- total time for blast 925: 21.4610013961792\n",
      "-- processing region: 926 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_926.fasta\n",
      "--- *blasting region: 926\n",
      "--- total time for blast 926: 19.208001375198364\n",
      "-- processing region: 927 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_927.fasta\n",
      "--- *blasting region: 927\n",
      "--- total time for blast 927: 21.532999277114868\n",
      "-- processing region: 928 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_928.fasta\n",
      "--- *blasting region: 928\n",
      "--- total time for blast 928: 24.459999561309814\n",
      "-- processing region: 929 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_929.fasta\n",
      "--- *blasting region: 929\n",
      "--- total time for blast 929: 23.045995473861694\n",
      "-- processing region: 93 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_93.fasta\n",
      "--- *blasting region: 93\n",
      "--- total time for blast 93: 18.35200333595276\n",
      "-- processing region: 930 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_930.fasta\n",
      "--- *blasting region: 930\n",
      "--- total time for blast 930: 23.91199803352356\n",
      "-- processing region: 931 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_931.fasta\n",
      "--- *blasting region: 931\n",
      "--- total time for blast 931: 21.154000759124756\n",
      "-- processing region: 932 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_932.fasta\n",
      "--- *blasting region: 932\n",
      "--- total time for blast 932: 23.733001947402954\n",
      "-- processing region: 933 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_933.fasta\n",
      "--- *blasting region: 933\n",
      "--- total time for blast 933: 23.03400206565857\n",
      "-- processing region: 934 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_934.fasta\n",
      "--- *blasting region: 934\n",
      "--- total time for blast 934: 17.557998180389404\n",
      "-- processing region: 935 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_935.fasta\n",
      "--- *blasting region: 935\n",
      "--- total time for blast 935: 19.90999698638916\n",
      "-- processing region: 936 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_936.fasta\n",
      "--- *blasting region: 936\n",
      "--- total time for blast 936: 23.5809965133667\n",
      "-- processing region: 937 with 397 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_937.fasta\n",
      "--- *blasting region: 937\n",
      "--- total time for blast 937: 20.628000497817993\n",
      "-- processing region: 938 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_938.fasta\n",
      "--- *blasting region: 938\n",
      "--- total time for blast 938: 21.85602641105652\n",
      "-- processing region: 939 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_939.fasta\n",
      "--- *blasting region: 939\n",
      "--- total time for blast 939: 21.974966764450073\n",
      "-- processing region: 94 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_94.fasta\n",
      "--- *blasting region: 94\n",
      "--- total time for blast 94: 20.464007139205933\n",
      "-- processing region: 940 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_940.fasta\n",
      "--- *blasting region: 940\n",
      "--- total time for blast 940: 31.39099621772766\n",
      "-- processing region: 941 with 398 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_941.fasta\n",
      "--- *blasting region: 941\n",
      "--- total time for blast 941: 21.0190007686615\n",
      "-- processing region: 942 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_942.fasta\n",
      "--- *blasting region: 942\n",
      "--- total time for blast 942: 20.90500044822693\n",
      "-- processing region: 943 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_943.fasta\n",
      "--- *blasting region: 943\n",
      "--- total time for blast 943: 23.341006994247437\n",
      "-- processing region: 944 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_944.fasta\n",
      "--- *blasting region: 944\n",
      "--- total time for blast 944: 19.95499038696289\n",
      "-- processing region: 945 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_945.fasta\n",
      "--- *blasting region: 945\n",
      "--- total time for blast 945: 19.97599959373474\n",
      "-- processing region: 946 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_946.fasta\n",
      "--- *blasting region: 946\n",
      "--- total time for blast 946: 18.3310067653656\n",
      "-- processing region: 947 with 386 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_947.fasta\n",
      "--- *blasting region: 947\n",
      "--- total time for blast 947: 17.61199951171875\n",
      "-- processing region: 948 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_948.fasta\n",
      "--- *blasting region: 948\n",
      "--- total time for blast 948: 19.95999526977539\n",
      "-- processing region: 949 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_949.fasta\n",
      "--- *blasting region: 949\n",
      "--- total time for blast 949: 35.663001537323\n",
      "-- processing region: 95 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_95.fasta\n",
      "--- *blasting region: 95\n",
      "--- total time for blast 95: 20.934992790222168\n",
      "-- processing region: 950 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_950.fasta\n",
      "--- *blasting region: 950\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 950: 21.19000482559204\n",
      "-- processing region: 951 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_951.fasta\n",
      "--- *blasting region: 951\n",
      "--- total time for blast 951: 17.663996934890747\n",
      "-- processing region: 952 with 394 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_952.fasta\n",
      "--- *blasting region: 952\n",
      "--- total time for blast 952: 19.011997938156128\n",
      "-- processing region: 953 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_953.fasta\n",
      "--- *blasting region: 953\n",
      "--- total time for blast 953: 21.160009145736694\n",
      "-- processing region: 954 with 397 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_954.fasta\n",
      "--- *blasting region: 954\n",
      "--- total time for blast 954: 20.443997144699097\n",
      "-- processing region: 955 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_955.fasta\n",
      "--- *blasting region: 955\n",
      "--- total time for blast 955: 21.44099760055542\n",
      "-- processing region: 956 with 395 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_956.fasta\n",
      "--- *blasting region: 956\n",
      "--- total time for blast 956: 23.005001306533813\n",
      "-- processing region: 957 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_957.fasta\n",
      "--- *blasting region: 957\n",
      "--- total time for blast 957: 18.417999505996704\n",
      "-- processing region: 958 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_958.fasta\n",
      "--- *blasting region: 958\n",
      "--- total time for blast 958: 20.989004850387573\n",
      "-- processing region: 959 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_959.fasta\n",
      "--- *blasting region: 959\n",
      "--- total time for blast 959: 21.699002027511597\n",
      "-- processing region: 96 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_96.fasta\n",
      "--- *blasting region: 96\n",
      "--- total time for blast 96: 22.50199556350708\n",
      "-- processing region: 960 with 399 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_960.fasta\n",
      "--- *blasting region: 960\n",
      "--- total time for blast 960: 20.168004274368286\n",
      "-- processing region: 961 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_961.fasta\n",
      "--- *blasting region: 961\n",
      "--- total time for blast 961: 19.16100001335144\n",
      "-- processing region: 962 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_962.fasta\n",
      "--- *blasting region: 962\n",
      "--- total time for blast 962: 30.396004915237427\n",
      "-- processing region: 963 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_963.fasta\n",
      "--- *blasting region: 963\n",
      "--- total time for blast 963: 22.151992559432983\n",
      "-- processing region: 964 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_964.fasta\n",
      "--- *blasting region: 964\n",
      "--- total time for blast 964: 21.374999046325684\n",
      "-- processing region: 965 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_965.fasta\n",
      "--- *blasting region: 965\n",
      "--- total time for blast 965: 25.810002088546753\n",
      "-- processing region: 966 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_966.fasta\n",
      "--- *blasting region: 966\n",
      "--- total time for blast 966: 18.71299695968628\n",
      "-- processing region: 967 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_967.fasta\n",
      "--- *blasting region: 967\n",
      "--- total time for blast 967: 24.19199562072754\n",
      "-- processing region: 968 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_968.fasta\n",
      "--- *blasting region: 968\n",
      "--- total time for blast 968: 22.342008352279663\n",
      "-- processing region: 969 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_969.fasta\n",
      "--- *blasting region: 969\n",
      "--- total time for blast 969: 22.374995708465576\n",
      "-- processing region: 97 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_97.fasta\n",
      "--- *blasting region: 97\n",
      "--- total time for blast 97: 20.5770001411438\n",
      "-- processing region: 970 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_970.fasta\n",
      "--- *blasting region: 970\n",
      "--- total time for blast 970: 21.76599669456482\n",
      "-- processing region: 971 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_971.fasta\n",
      "--- *blasting region: 971\n",
      "--- total time for blast 971: 22.99700665473938\n",
      "-- processing region: 972 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_972.fasta\n",
      "--- *blasting region: 972\n",
      "--- total time for blast 972: 22.576000213623047\n",
      "-- processing region: 973 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_973.fasta\n",
      "--- *blasting region: 973\n",
      "--- total time for blast 973: 22.63099718093872\n",
      "-- processing region: 974 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_974.fasta\n",
      "--- *blasting region: 974\n",
      "--- total time for blast 974: 27.610007524490356\n",
      "-- processing region: 975 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_975.fasta\n",
      "--- *blasting region: 975\n",
      "--- total time for blast 975: 21.800999641418457\n",
      "-- processing region: 976 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_976.fasta\n",
      "--- *blasting region: 976\n",
      "--- total time for blast 976: 23.521003246307373\n",
      "-- processing region: 977 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_977.fasta\n",
      "--- *blasting region: 977\n",
      "--- total time for blast 977: 17.34798812866211\n",
      "-- processing region: 978 with 394 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_978.fasta\n",
      "--- *blasting region: 978\n",
      "--- total time for blast 978: 23.86400532722473\n",
      "-- processing region: 979 with 399 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_979.fasta\n",
      "--- *blasting region: 979\n",
      "--- total time for blast 979: 18.030999660491943\n",
      "-- processing region: 98 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_98.fasta\n",
      "--- *blasting region: 98\n",
      "--- total time for blast 98: 17.22400450706482\n",
      "-- processing region: 980 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_980.fasta\n",
      "--- *blasting region: 980\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 980: 19.98399519920349\n",
      "-- processing region: 981 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_981.fasta\n",
      "--- *blasting region: 981\n",
      "--- total time for blast 981: 19.20900845527649\n",
      "-- processing region: 982 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_982.fasta\n",
      "--- *blasting region: 982\n",
      "--- total time for blast 982: 18.042001485824585\n",
      "-- processing region: 983 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_983.fasta\n",
      "--- *blasting region: 983\n",
      "--- total time for blast 983: 21.304999828338623\n",
      "-- processing region: 984 with 398 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_984.fasta\n",
      "--- *blasting region: 984\n",
      "--- total time for blast 984: 20.734005451202393\n",
      "-- processing region: 985 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_985.fasta\n",
      "--- *blasting region: 985\n",
      "--- total time for blast 985: 17.227997541427612\n",
      "-- processing region: 986 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_986.fasta\n",
      "--- *blasting region: 986\n",
      "--- total time for blast 986: 16.402994632720947\n",
      "-- processing region: 987 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_987.fasta\n",
      "--- *blasting region: 987\n",
      "--- total time for blast 987: 21.762000799179077\n",
      "-- processing region: 988 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_988.fasta\n",
      "--- *blasting region: 988\n",
      "--- total time for blast 988: 21.074003219604492\n",
      "-- processing region: 989 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_989.fasta\n",
      "--- *blasting region: 989\n",
      "--- total time for blast 989: 20.538997888565063\n",
      "-- processing region: 99 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_99.fasta\n",
      "--- *blasting region: 99\n",
      "--- total time for blast 99: 19.02900195121765\n",
      "-- processing region: 990 with 400 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_990.fasta\n",
      "--- *blasting region: 990\n",
      "--- total time for blast 990: 32.660999059677124\n",
      "-- processing region: 991 with 127 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast\\probe_gene_991.fasta\n",
      "--- *blasting region: 991\n",
      "--- total time for blast 991: 8.099997758865356\n"
     ]
    }
   ],
   "source": [
    "import ImageAnalysis3.library_tools.quality_check as check\n",
    "reload(check)\n",
    "\n",
    "blast_database = os.path.join(reference_folder, 'blast_database', 'GRCm38_ensembl')\n",
    "\n",
    "# run blast using the pre-generated blastn database (on NAS2)\n",
    "check.Blast_probes(kept_records, library_folder, num_threads=32, \n",
    "                   blast_db=blast_database, blast_evalue=100, overwrite=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 4.2.1 screen probe by blast results\n",
    "\n",
    "This step has 2 different cutoffs:\n",
    "\n",
    "hard_cutoff: whether a probe has any ~40mer (default) hit somewhere else, if so just dump the probe\n",
    "\n",
    "soft_cutoff: whether a probe has a lot of ~20mer (default) off-target hits, if exceed certain amount then dump the probe"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Number of probes loaded: 383682\n",
      "- Number of regions in this library: 968\n",
      "-- checking probes in region: 251\n",
      "--- gene=251, id=162 removed by soft count = 61\n",
      "--- gene=251, id=281 removed by soft count = 186\n",
      "--- number of probes: 398 , kept by blast: 396 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 252\n",
      "--- number of probes: 399 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 253\n",
      "--- gene=253, id=36 removed by soft count = 74\n",
      "--- gene=253, id=38 removed by soft count = 74\n",
      "--- gene=253, id=346 removed by soft count = 83\n",
      "--- gene=253, id=347 removed by soft count = 78\n",
      "--- gene=253, id=360 removed by soft count = 51\n",
      "--- gene=253, id=361 removed by soft count = 70\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 255\n",
      "--- gene=255, id=304 removed by soft count = 97\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 256\n",
      "--- gene=256, id=10 removed by soft count = 52\n",
      "--- gene=256, id=16 removed by soft count = 92\n",
      "--- gene=256, id=19 removed by soft count = 68\n",
      "--- gene=256, id=182 removed by soft count = 169\n",
      "--- gene=256, id=183 removed by soft count = 306\n",
      "--- gene=256, id=273 removed by soft count = 187\n",
      "--- gene=256, id=298 removed by soft count = 95\n",
      "--- gene=256, id=300 removed by soft count = 146\n",
      "--- gene=256, id=324 removed by soft count = 65\n",
      "--- gene=256, id=356 removed by soft count = 73\n",
      "--- number of probes: 400 , kept by blast: 390 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 257\n",
      "--- gene=257, id=81 removed by soft count = 248\n",
      "--- gene=257, id=135 removed by soft count = 368\n",
      "--- gene=257, id=145 removed by soft count = 204\n",
      "--- gene=257, id=247 removed by soft count = 51\n",
      "--- gene=257, id=361 removed by soft count = 710\n",
      "--- gene=257, id=371 removed by soft count = 98\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 258\n",
      "--- gene=258, id=68 removed by soft count = 1063\n",
      "--- gene=258, id=96 removed by soft count = 253\n",
      "--- gene=258, id=295 removed by soft count = 75\n",
      "--- gene=258, id=346 removed by soft count = 268\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 259\n",
      "--- gene=259, id=13 removed by soft count = 136\n",
      "--- gene=259, id=14 removed by soft count = 362\n",
      "--- gene=259, id=16 removed by soft count = 117\n",
      "--- gene=259, id=20 removed by soft count = 87\n",
      "--- gene=259, id=27 removed by soft count = 608\n",
      "--- gene=259, id=28 removed by soft count = 625\n",
      "--- gene=259, id=29 removed by soft count = 63\n",
      "--- gene=259, id=40 removed by soft count = 51\n",
      "--- gene=259, id=44 removed by soft count = 213\n",
      "--- gene=259, id=71 removed by soft count = 127\n",
      "--- gene=259, id=72 removed by soft count = 59\n",
      "--- gene=259, id=82 removed by soft count = 84\n",
      "--- gene=259, id=195 removed by soft count = 814\n",
      "--- gene=259, id=233 removed by soft count = 759\n",
      "--- gene=259, id=317 removed by soft count = 228\n",
      "--- gene=259, id=319 removed by soft count = 67\n",
      "--- gene=259, id=321 removed by soft count = 99\n",
      "--- gene=259, id=363 removed by soft count = 344\n",
      "--- number of probes: 400 , kept by blast: 382 , if remove dups: 382\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 260\n",
      "--- gene=260, id=22 removed by soft count = 194\n",
      "--- gene=260, id=48 removed by soft count = 63\n",
      "--- gene=260, id=49 removed by soft count = 164\n",
      "--- gene=260, id=391 removed by soft count = 72\n",
      "--- gene=260, id=392 removed by soft count = 134\n",
      "--- gene=260, id=394 removed by soft count = 225\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 261\n",
      "--- gene=261, id=40 removed by soft count = 371\n",
      "--- gene=261, id=53 removed by soft count = 128\n",
      "--- gene=261, id=56 removed by soft count = 128\n",
      "--- gene=261, id=185 removed by soft count = 81\n",
      "--- gene=261, id=191 removed by soft count = 196\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 262\n",
      "--- gene=262, id=11 removed by soft count = 79\n",
      "--- gene=262, id=31 removed by soft count = 87\n",
      "--- gene=262, id=33 removed by soft count = 572\n",
      "--- gene=262, id=66 removed by soft count = 184\n",
      "--- gene=262, id=68 removed by soft count = 92\n",
      "--- gene=262, id=303 removed by soft count = 55\n",
      "--- gene=262, id=336 removed by soft count = 2361\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 263\n",
      "--- gene=263, id=70 removed by soft count = 2228\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 264\n",
      "--- gene=264, id=24 removed by soft count = 99\n",
      "--- gene=264, id=218 removed by soft count = 136\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 265\n",
      "--- gene=265, id=0 removed by soft count = 102\n",
      "--- gene=265, id=25 removed by soft count = 8164\n",
      "--- gene=265, id=137 removed by soft count = 215\n",
      "--- gene=265, id=138 removed by soft count = 144\n",
      "--- gene=265, id=139 removed by soft count = 103\n",
      "--- gene=265, id=140 removed by soft count = 122\n",
      "--- gene=265, id=141 removed by soft count = 79\n",
      "--- gene=265, id=211 removed by soft count = 58\n",
      "--- gene=265, id=335 removed by soft count = 117\n",
      "--- gene=265, id=336 removed by soft count = 87\n",
      "--- number of probes: 400 , kept by blast: 390 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 266\n",
      "--- gene=266, id=83 removed by soft count = 251\n",
      "--- gene=266, id=93 removed by soft count = 1121\n",
      "--- gene=266, id=262 removed by soft count = 91\n",
      "--- gene=266, id=306 removed by soft count = 224\n",
      "--- gene=266, id=350 removed by soft count = 579\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 267\n",
      "--- gene=267, id=143 removed by hard count = 0\n",
      "--- gene=267, id=196 removed by soft count = 93\n",
      "--- gene=267, id=243 removed by soft count = 128\n",
      "--- gene=267, id=255 removed by soft count = 2702\n",
      "--- gene=267, id=313 removed by soft count = 142\n",
      "--- gene=267, id=374 removed by soft count = 57\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 268\n",
      "--- gene=268, id=20 removed by soft count = 94\n",
      "--- gene=268, id=55 removed by soft count = 250\n",
      "--- gene=268, id=111 removed by soft count = 173\n",
      "--- gene=268, id=112 removed by soft count = 127\n",
      "--- gene=268, id=370 removed by soft count = 51\n",
      "--- gene=268, id=391 removed by soft count = 55\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 269\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=269, id=183 removed by soft count = 142\n",
      "--- gene=269, id=191 removed by soft count = 108\n",
      "--- gene=269, id=273 removed by soft count = 93\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 270\n",
      "--- gene=270, id=81 removed by soft count = 428\n",
      "--- gene=270, id=82 removed by soft count = 341\n",
      "--- gene=270, id=139 removed by soft count = 181\n",
      "--- gene=270, id=170 removed by soft count = 51\n",
      "--- gene=270, id=244 removed by soft count = 160\n",
      "--- gene=270, id=254 removed by soft count = 160\n",
      "--- gene=270, id=256 removed by soft count = 88\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 271\n",
      "--- gene=271, id=57 removed by soft count = 59\n",
      "--- gene=271, id=140 removed by soft count = 235\n",
      "--- gene=271, id=142 removed by soft count = 75\n",
      "--- gene=271, id=159 removed by soft count = 146\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 272\n",
      "--- gene=272, id=158 removed by soft count = 599\n",
      "--- gene=272, id=207 removed by soft count = 146\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 273\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 274\n",
      "--- gene=274, id=0 removed by soft count = 94\n",
      "--- gene=274, id=34 removed by soft count = 157\n",
      "--- gene=274, id=35 removed by soft count = 166\n",
      "--- gene=274, id=37 removed by soft count = 129\n",
      "--- gene=274, id=51 removed by soft count = 53\n",
      "--- gene=274, id=52 removed by soft count = 57\n",
      "--- gene=274, id=266 removed by soft count = 73\n",
      "--- gene=274, id=268 removed by soft count = 78\n",
      "--- gene=274, id=283 removed by soft count = 57\n",
      "--- gene=274, id=284 removed by soft count = 96\n",
      "--- gene=274, id=289 removed by soft count = 410\n",
      "--- gene=274, id=352 removed by soft count = 67\n",
      "--- gene=274, id=355 removed by soft count = 55\n",
      "--- gene=274, id=383 removed by soft count = 124\n",
      "--- gene=274, id=389 removed by soft count = 234\n",
      "--- gene=274, id=391 removed by soft count = 66\n",
      "--- gene=274, id=399 removed by soft count = 329\n",
      "--- number of probes: 400 , kept by blast: 383 , if remove dups: 383\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 275\n",
      "--- gene=275, id=0 removed by soft count = 157\n",
      "--- gene=275, id=28 removed by soft count = 66\n",
      "--- gene=275, id=58 removed by soft count = 55\n",
      "--- gene=275, id=59 removed by soft count = 56\n",
      "--- gene=275, id=156 removed by soft count = 138\n",
      "--- gene=275, id=159 removed by soft count = 210\n",
      "--- gene=275, id=160 removed by soft count = 110\n",
      "--- gene=275, id=194 removed by soft count = 89\n",
      "--- gene=275, id=200 removed by soft count = 174\n",
      "--- gene=275, id=201 removed by hard count = 0\n",
      "--- gene=275, id=256 removed by soft count = 74\n",
      "--- gene=275, id=332 removed by soft count = 81\n",
      "--- gene=275, id=354 removed by soft count = 105\n",
      "--- gene=275, id=355 removed by soft count = 329\n",
      "--- number of probes: 386 , kept by blast: 372 , if remove dups: 372\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 277\n",
      "--- gene=277, id=21 removed by soft count = 256\n",
      "--- gene=277, id=22 removed by soft count = 128\n",
      "--- gene=277, id=44 removed by soft count = 75\n",
      "--- gene=277, id=46 removed by soft count = 59\n",
      "--- gene=277, id=73 removed by soft count = 110\n",
      "--- gene=277, id=74 removed by soft count = 117\n",
      "--- gene=277, id=181 removed by soft count = 103\n",
      "--- gene=277, id=219 removed by soft count = 55\n",
      "--- gene=277, id=223 removed by soft count = 57\n",
      "--- gene=277, id=304 removed by soft count = 299\n",
      "--- gene=277, id=306 removed by soft count = 540\n",
      "--- number of probes: 399 , kept by blast: 388 , if remove dups: 388\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 278\n",
      "--- gene=278, id=23 removed by soft count = 55\n",
      "--- gene=278, id=258 removed by soft count = 123\n",
      "--- gene=278, id=311 removed by soft count = 161\n",
      "--- gene=278, id=312 removed by soft count = 126\n",
      "--- gene=278, id=366 removed by soft count = 275\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 279\n",
      "--- gene=279, id=127 removed by soft count = 119\n",
      "--- gene=279, id=339 removed by soft count = 196\n",
      "--- gene=279, id=362 removed by soft count = 75\n",
      "--- gene=279, id=363 removed by soft count = 92\n",
      "--- gene=279, id=364 removed by soft count = 742\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 280\n",
      "--- gene=280, id=41 removed by soft count = 83\n",
      "--- gene=280, id=234 removed by soft count = 72\n",
      "--- gene=280, id=276 removed by soft count = 744\n",
      "--- gene=280, id=280 removed by soft count = 188\n",
      "--- gene=280, id=281 removed by soft count = 67\n",
      "--- gene=280, id=317 removed by soft count = 54\n",
      "--- gene=280, id=319 removed by soft count = 292\n",
      "--- gene=280, id=381 removed by soft count = 1161\n",
      "--- gene=280, id=385 removed by soft count = 59\n",
      "--- number of probes: 400 , kept by blast: 391 , if remove dups: 391\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 281\n",
      "--- gene=281, id=140 removed by soft count = 66\n",
      "--- gene=281, id=259 removed by soft count = 374\n",
      "--- number of probes: 396 , kept by blast: 394 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 282\n",
      "--- gene=282, id=0 removed by soft count = 74\n",
      "--- gene=282, id=1 removed by soft count = 83\n",
      "--- gene=282, id=2 removed by soft count = 73\n",
      "--- gene=282, id=3 removed by soft count = 65\n",
      "--- gene=282, id=25 removed by soft count = 60\n",
      "--- gene=282, id=57 removed by soft count = 317\n",
      "--- gene=282, id=58 removed by soft count = 55\n",
      "--- gene=282, id=59 removed by soft count = 624\n",
      "--- gene=282, id=278 removed by soft count = 950\n",
      "--- number of probes: 400 , kept by blast: 391 , if remove dups: 391\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 283\n",
      "--- gene=283, id=1 removed by soft count = 532\n",
      "--- gene=283, id=4 removed by soft count = 100\n",
      "--- gene=283, id=13 removed by soft count = 116\n",
      "--- gene=283, id=15 removed by soft count = 262\n",
      "--- gene=283, id=74 removed by soft count = 924\n",
      "--- gene=283, id=120 removed by soft count = 62\n",
      "--- gene=283, id=137 removed by soft count = 279\n",
      "--- gene=283, id=192 removed by soft count = 79\n",
      "--- gene=283, id=193 removed by soft count = 68\n",
      "--- gene=283, id=194 removed by soft count = 75\n",
      "--- number of probes: 400 , kept by blast: 390 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 284\n",
      "--- gene=284, id=14 removed by soft count = 308\n",
      "--- gene=284, id=98 removed by soft count = 63\n",
      "--- gene=284, id=107 removed by hard count = 0\n",
      "--- gene=284, id=130 removed by soft count = 53\n",
      "--- gene=284, id=131 removed by soft count = 274\n",
      "--- gene=284, id=132 removed by soft count = 65\n",
      "--- gene=284, id=219 removed by soft count = 156\n",
      "--- gene=284, id=260 removed by soft count = 502\n",
      "--- number of probes: 400 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 285\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=285, id=86 removed by soft count = 134\n",
      "--- gene=285, id=109 removed by soft count = 63\n",
      "--- gene=285, id=340 removed by soft count = 75\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 286\n",
      "--- gene=286, id=128 removed by soft count = 295\n",
      "--- gene=286, id=154 removed by soft count = 238\n",
      "--- gene=286, id=204 removed by soft count = 125\n",
      "--- gene=286, id=205 removed by soft count = 85\n",
      "--- gene=286, id=209 removed by soft count = 1352\n",
      "--- gene=286, id=212 removed by soft count = 681\n",
      "--- gene=286, id=266 removed by soft count = 179\n",
      "--- gene=286, id=269 removed by soft count = 118\n",
      "--- gene=286, id=382 removed by soft count = 114\n",
      "--- gene=286, id=384 removed by soft count = 195\n",
      "--- gene=286, id=389 removed by soft count = 82\n",
      "--- gene=286, id=391 removed by soft count = 118\n",
      "--- number of probes: 400 , kept by blast: 388 , if remove dups: 388\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 287\n",
      "--- gene=287, id=232 removed by soft count = 61\n",
      "--- gene=287, id=329 removed by soft count = 55\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 288\n",
      "--- gene=288, id=71 removed by soft count = 847\n",
      "--- gene=288, id=379 removed by soft count = 419\n",
      "--- gene=288, id=398 removed by soft count = 181\n",
      "--- gene=288, id=399 removed by soft count = 150\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 289\n",
      "--- gene=289, id=1 removed by soft count = 179\n",
      "--- gene=289, id=93 removed by soft count = 146\n",
      "--- gene=289, id=95 removed by soft count = 139\n",
      "--- gene=289, id=98 removed by soft count = 111\n",
      "--- gene=289, id=99 removed by soft count = 1974\n",
      "--- gene=289, id=163 removed by soft count = 125\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 290\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 291\n",
      "--- gene=291, id=1 removed by soft count = 352\n",
      "--- gene=291, id=30 removed by soft count = 359\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 292\n",
      "--- gene=292, id=80 removed by soft count = 51\n",
      "--- gene=292, id=142 removed by soft count = 61\n",
      "--- gene=292, id=193 removed by soft count = 66\n",
      "--- gene=292, id=227 removed by soft count = 1329\n",
      "--- gene=292, id=364 removed by soft count = 83\n",
      "--- gene=292, id=365 removed by soft count = 1070\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 293\n",
      "--- gene=293, id=156 removed by soft count = 121\n",
      "--- gene=293, id=399 removed by soft count = 127\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 295\n",
      "--- gene=295, id=53 removed by soft count = 62\n",
      "--- gene=295, id=191 removed by soft count = 95\n",
      "--- gene=295, id=262 removed by soft count = 134\n",
      "--- gene=295, id=347 removed by soft count = 54\n",
      "--- gene=295, id=358 removed by soft count = 58\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 296\n",
      "--- gene=296, id=132 removed by soft count = 5649\n",
      "--- gene=296, id=276 removed by soft count = 410\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 297\n",
      "--- gene=297, id=118 removed by soft count = 85\n",
      "--- gene=297, id=281 removed by soft count = 982\n",
      "--- gene=297, id=288 removed by soft count = 57\n",
      "--- gene=297, id=392 removed by soft count = 78\n",
      "--- gene=297, id=393 removed by soft count = 1609\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 298\n",
      "--- gene=298, id=258 removed by soft count = 87\n",
      "--- gene=298, id=260 removed by soft count = 131\n",
      "--- gene=298, id=325 removed by soft count = 108\n",
      "--- gene=298, id=326 removed by soft count = 55\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 299\n",
      "--- gene=299, id=33 removed by soft count = 56\n",
      "--- gene=299, id=187 removed by soft count = 150\n",
      "--- gene=299, id=197 removed by soft count = 109\n",
      "--- gene=299, id=198 removed by soft count = 347\n",
      "--- gene=299, id=199 removed by soft count = 1655\n",
      "--- gene=299, id=213 removed by soft count = 896\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 300\n",
      "--- gene=300, id=204 removed by soft count = 418\n",
      "--- gene=300, id=296 removed by soft count = 129\n",
      "--- gene=300, id=322 removed by soft count = 186\n",
      "--- gene=300, id=360 removed by soft count = 104\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 301\n",
      "--- gene=301, id=3 removed by soft count = 294\n",
      "--- gene=301, id=4 removed by soft count = 523\n",
      "--- gene=301, id=8 removed by soft count = 703\n",
      "--- gene=301, id=10 removed by soft count = 76\n",
      "--- gene=301, id=12 removed by soft count = 858\n",
      "--- gene=301, id=13 removed by soft count = 764\n",
      "--- gene=301, id=14 removed by soft count = 757\n",
      "--- gene=301, id=15 removed by soft count = 735\n",
      "--- gene=301, id=18 removed by soft count = 349\n",
      "--- gene=301, id=24 removed by soft count = 597\n",
      "--- gene=301, id=30 removed by soft count = 806\n",
      "--- gene=301, id=31 removed by soft count = 812\n",
      "--- gene=301, id=32 removed by soft count = 706\n",
      "--- gene=301, id=43 removed by soft count = 77\n",
      "--- gene=301, id=58 removed by soft count = 134\n",
      "--- gene=301, id=106 removed by soft count = 500\n",
      "--- gene=301, id=107 removed by soft count = 169\n",
      "--- gene=301, id=108 removed by soft count = 179\n",
      "--- gene=301, id=115 removed by soft count = 232\n",
      "--- gene=301, id=116 removed by soft count = 198\n",
      "--- gene=301, id=117 removed by soft count = 62\n",
      "--- gene=301, id=316 removed by soft count = 89\n",
      "--- gene=301, id=321 removed by soft count = 124\n",
      "--- gene=301, id=330 removed by soft count = 69\n",
      "--- number of probes: 400 , kept by blast: 376 , if remove dups: 376\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 302\n",
      "--- gene=302, id=47 removed by soft count = 57\n",
      "--- gene=302, id=72 removed by soft count = 131\n",
      "--- gene=302, id=274 removed by soft count = 282\n",
      "--- gene=302, id=322 removed by soft count = 2704\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 303\n",
      "--- gene=303, id=74 removed by soft count = 66\n",
      "--- gene=303, id=100 removed by soft count = 58\n",
      "--- gene=303, id=380 removed by soft count = 317\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 304\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=304, id=276 removed by soft count = 1095\n",
      "--- gene=304, id=304 removed by soft count = 67\n",
      "--- gene=304, id=392 removed by soft count = 57\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 305\n",
      "--- gene=305, id=266 removed by soft count = 74\n",
      "--- gene=305, id=281 removed by soft count = 54\n",
      "--- number of probes: 399 , kept by blast: 397 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 306\n",
      "--- gene=306, id=31 removed by soft count = 66\n",
      "--- gene=306, id=329 removed by soft count = 2103\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 307\n",
      "--- gene=307, id=59 removed by soft count = 102\n",
      "--- gene=307, id=60 removed by soft count = 98\n",
      "--- gene=307, id=79 removed by soft count = 97\n",
      "--- gene=307, id=127 removed by soft count = 2372\n",
      "--- gene=307, id=288 removed by soft count = 193\n",
      "--- gene=307, id=289 removed by soft count = 94\n",
      "--- gene=307, id=290 removed by soft count = 64\n",
      "--- gene=307, id=291 removed by soft count = 62\n",
      "--- gene=307, id=342 removed by soft count = 138\n",
      "--- number of probes: 400 , kept by blast: 391 , if remove dups: 391\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 308\n",
      "--- gene=308, id=31 removed by soft count = 65\n",
      "--- gene=308, id=375 removed by soft count = 161\n",
      "--- gene=308, id=376 removed by soft count = 148\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 309\n",
      "--- gene=309, id=53 removed by soft count = 203\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 310\n",
      "--- gene=310, id=279 removed by soft count = 56\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 311\n",
      "--- gene=311, id=73 removed by soft count = 74\n",
      "--- gene=311, id=78 removed by soft count = 128\n",
      "--- gene=311, id=81 removed by soft count = 70\n",
      "--- gene=311, id=128 removed by soft count = 99\n",
      "--- gene=311, id=131 removed by soft count = 169\n",
      "--- gene=311, id=178 removed by soft count = 51\n",
      "--- gene=311, id=182 removed by soft count = 291\n",
      "--- gene=311, id=183 removed by soft count = 127\n",
      "--- gene=311, id=184 removed by soft count = 289\n",
      "--- number of probes: 400 , kept by blast: 391 , if remove dups: 391\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 312\n",
      "--- gene=312, id=0 removed by soft count = 1796\n",
      "--- gene=312, id=12 removed by soft count = 74\n",
      "--- gene=312, id=340 removed by soft count = 58\n",
      "--- gene=312, id=372 removed by soft count = 58\n",
      "--- gene=312, id=373 removed by soft count = 193\n",
      "--- gene=312, id=375 removed by soft count = 57\n",
      "--- number of probes: 390 , kept by blast: 384 , if remove dups: 384\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 313\n",
      "--- gene=313, id=40 removed by soft count = 63\n",
      "--- gene=313, id=110 removed by soft count = 135\n",
      "--- gene=313, id=380 removed by soft count = 403\n",
      "--- number of probes: 396 , kept by blast: 393 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 314\n",
      "--- gene=314, id=339 removed by soft count = 94\n",
      "--- gene=314, id=343 removed by soft count = 12557\n",
      "--- gene=314, id=344 removed by soft count = 123\n",
      "--- gene=314, id=345 removed by soft count = 150\n",
      "--- gene=314, id=346 removed by soft count = 82\n",
      "--- gene=314, id=347 removed by soft count = 163\n",
      "--- gene=314, id=348 removed by soft count = 85\n",
      "--- gene=314, id=356 removed by soft count = 64\n",
      "--- gene=314, id=357 removed by soft count = 395\n",
      "--- gene=314, id=365 removed by soft count = 69\n",
      "--- gene=314, id=368 removed by soft count = 113\n",
      "--- number of probes: 400 , kept by blast: 389 , if remove dups: 389\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 315\n",
      "--- gene=315, id=209 removed by soft count = 86\n",
      "--- gene=315, id=211 removed by soft count = 172\n",
      "--- gene=315, id=213 removed by soft count = 385\n",
      "--- gene=315, id=326 removed by soft count = 266\n",
      "--- gene=315, id=351 removed by soft count = 92\n",
      "--- gene=315, id=357 removed by soft count = 253\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 316\n",
      "--- gene=316, id=278 removed by soft count = 87\n",
      "--- gene=316, id=283 removed by soft count = 147\n",
      "--- gene=316, id=284 removed by soft count = 108\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 317\n",
      "--- gene=317, id=7 removed by soft count = 85\n",
      "--- gene=317, id=35 removed by soft count = 154\n",
      "--- gene=317, id=136 removed by soft count = 62\n",
      "--- gene=317, id=301 removed by soft count = 230\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 318\n",
      "--- gene=318, id=94 removed by soft count = 121\n",
      "--- gene=318, id=96 removed by soft count = 239\n",
      "--- gene=318, id=290 removed by soft count = 258\n",
      "--- gene=318, id=339 removed by soft count = 763\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 319\n",
      "--- gene=319, id=30 removed by soft count = 430\n",
      "--- gene=319, id=56 removed by soft count = 101\n",
      "--- gene=319, id=159 removed by soft count = 84\n",
      "--- gene=319, id=184 removed by soft count = 122\n",
      "--- gene=319, id=191 removed by soft count = 53\n",
      "--- gene=319, id=236 removed by soft count = 275\n",
      "--- gene=319, id=255 removed by soft count = 57\n",
      "--- gene=319, id=299 removed by soft count = 92\n",
      "--- gene=319, id=307 removed by soft count = 332\n",
      "--- number of probes: 400 , kept by blast: 391 , if remove dups: 391\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 320\n",
      "--- gene=320, id=64 removed by soft count = 330\n",
      "--- gene=320, id=65 removed by soft count = 79\n",
      "--- gene=320, id=80 removed by soft count = 251\n",
      "--- gene=320, id=121 removed by soft count = 226\n",
      "--- gene=320, id=150 removed by soft count = 609\n",
      "--- gene=320, id=177 removed by soft count = 61\n",
      "--- gene=320, id=251 removed by soft count = 354\n",
      "--- gene=320, id=281 removed by soft count = 1319\n",
      "--- number of probes: 400 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 321\n",
      "--- gene=321, id=60 removed by soft count = 62\n",
      "--- gene=321, id=251 removed by soft count = 106\n",
      "--- gene=321, id=299 removed by soft count = 68\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 322\n",
      "--- gene=322, id=13 removed by soft count = 84\n",
      "--- gene=322, id=104 removed by soft count = 61\n",
      "--- gene=322, id=134 removed by soft count = 88\n",
      "--- gene=322, id=292 removed by soft count = 271\n",
      "--- gene=322, id=339 removed by soft count = 93\n",
      "--- gene=322, id=364 removed by soft count = 93\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=322, id=385 removed by soft count = 102\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 324\n",
      "--- gene=324, id=50 removed by soft count = 157\n",
      "--- gene=324, id=106 removed by soft count = 59\n",
      "--- gene=324, id=123 removed by soft count = 78\n",
      "--- gene=324, id=199 removed by soft count = 341\n",
      "--- gene=324, id=305 removed by hard count = 3\n",
      "--- gene=324, id=321 removed by soft count = 102\n",
      "--- gene=324, id=395 removed by soft count = 76\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 325\n",
      "--- gene=325, id=255 removed by soft count = 52\n",
      "--- gene=325, id=266 removed by soft count = 118\n",
      "--- gene=325, id=356 removed by soft count = 65\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 326\n",
      "--- gene=326, id=132 removed by soft count = 1948\n",
      "--- gene=326, id=172 removed by soft count = 176\n",
      "--- gene=326, id=202 removed by soft count = 72\n",
      "--- gene=326, id=314 removed by soft count = 64\n",
      "--- gene=326, id=322 removed by soft count = 77\n",
      "--- gene=326, id=341 removed by soft count = 167\n",
      "--- gene=326, id=377 removed by soft count = 75\n",
      "--- gene=326, id=387 removed by soft count = 117\n",
      "--- number of probes: 400 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 327\n",
      "--- gene=327, id=131 removed by soft count = 118\n",
      "--- gene=327, id=254 removed by soft count = 133\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 328\n",
      "--- gene=328, id=65 removed by soft count = 370\n",
      "--- gene=328, id=138 removed by soft count = 267\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 329\n",
      "--- gene=329, id=91 removed by soft count = 57\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 330\n",
      "--- gene=330, id=85 removed by soft count = 101\n",
      "--- gene=330, id=95 removed by soft count = 68\n",
      "--- gene=330, id=97 removed by soft count = 560\n",
      "--- gene=330, id=98 removed by soft count = 113\n",
      "--- gene=330, id=139 removed by soft count = 84\n",
      "--- gene=330, id=163 removed by soft count = 66\n",
      "--- gene=330, id=196 removed by soft count = 79\n",
      "--- gene=330, id=270 removed by soft count = 51\n",
      "--- gene=330, id=272 removed by soft count = 76\n",
      "--- gene=330, id=299 removed by soft count = 165\n",
      "--- gene=330, id=304 removed by soft count = 60\n",
      "--- gene=330, id=330 removed by soft count = 502\n",
      "--- gene=330, id=382 removed by soft count = 56\n",
      "--- gene=330, id=392 removed by soft count = 141\n",
      "--- gene=330, id=393 removed by soft count = 77\n",
      "--- gene=330, id=395 removed by soft count = 313\n",
      "--- gene=330, id=398 removed by soft count = 97\n",
      "--- number of probes: 400 , kept by blast: 383 , if remove dups: 383\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 332\n",
      "--- gene=332, id=1 removed by soft count = 113\n",
      "--- gene=332, id=77 removed by soft count = 51\n",
      "--- gene=332, id=144 removed by soft count = 55\n",
      "--- gene=332, id=172 removed by soft count = 77\n",
      "--- gene=332, id=274 removed by soft count = 238\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 333\n",
      "--- gene=333, id=34 removed by soft count = 72\n",
      "--- gene=333, id=73 removed by soft count = 70\n",
      "--- gene=333, id=74 removed by soft count = 133\n",
      "--- gene=333, id=75 removed by soft count = 548\n",
      "--- gene=333, id=296 removed by soft count = 105\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 334\n",
      "--- gene=334, id=223 removed by soft count = 117\n",
      "--- gene=334, id=370 removed by soft count = 131\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 335\n",
      "--- gene=335, id=2 removed by soft count = 80\n",
      "--- gene=335, id=3 removed by soft count = 137\n",
      "--- gene=335, id=4 removed by soft count = 143\n",
      "--- gene=335, id=19 removed by soft count = 110\n",
      "--- gene=335, id=170 removed by soft count = 3363\n",
      "--- gene=335, id=362 removed by soft count = 94\n",
      "--- gene=335, id=363 removed by soft count = 60\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 336\n",
      "--- gene=336, id=306 removed by soft count = 104\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 337\n",
      "--- gene=337, id=303 removed by soft count = 176\n",
      "--- gene=337, id=352 removed by soft count = 389\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 338\n",
      "--- gene=338, id=25 removed by soft count = 51\n",
      "--- gene=338, id=310 removed by soft count = 113\n",
      "--- gene=338, id=314 removed by soft count = 149\n",
      "--- gene=338, id=327 removed by soft count = 497\n",
      "--- gene=338, id=393 removed by soft count = 70\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 339\n",
      "--- gene=339, id=58 removed by soft count = 192\n",
      "--- gene=339, id=59 removed by soft count = 58\n",
      "--- gene=339, id=88 removed by soft count = 355\n",
      "--- gene=339, id=236 removed by soft count = 684\n",
      "--- gene=339, id=237 removed by soft count = 64\n",
      "--- gene=339, id=238 removed by soft count = 206\n",
      "--- gene=339, id=303 removed by soft count = 59\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 340\n",
      "--- gene=340, id=184 removed by soft count = 295\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 341\n",
      "--- gene=341, id=78 removed by soft count = 160\n",
      "--- gene=341, id=105 removed by soft count = 599\n",
      "--- gene=341, id=272 removed by soft count = 401\n",
      "--- gene=341, id=388 removed by soft count = 84\n",
      "--- gene=341, id=399 removed by soft count = 398\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 342\n",
      "--- gene=342, id=86 removed by soft count = 76\n",
      "--- gene=342, id=132 removed by soft count = 85\n",
      "--- gene=342, id=383 removed by soft count = 45491\n",
      "--- gene=342, id=394 removed by soft count = 172\n",
      "--- gene=342, id=395 removed by soft count = 85\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 343\n",
      "--- gene=343, id=0 removed by soft count = 230\n",
      "--- gene=343, id=278 removed by soft count = 51\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 399 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 344\n",
      "--- gene=344, id=25 removed by soft count = 86\n",
      "--- gene=344, id=26 removed by soft count = 99\n",
      "--- gene=344, id=317 removed by soft count = 91\n",
      "--- gene=344, id=374 removed by soft count = 427\n",
      "--- gene=344, id=388 removed by soft count = 55\n",
      "--- gene=344, id=390 removed by soft count = 219\n",
      "--- gene=344, id=395 removed by soft count = 133\n",
      "--- gene=344, id=398 removed by soft count = 118\n",
      "--- number of probes: 400 , kept by blast: 392 , if remove dups: 391\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 345\n",
      "--- gene=345, id=124 removed by soft count = 57\n",
      "--- gene=345, id=291 removed by soft count = 54\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 347\n",
      "--- gene=347, id=63 removed by soft count = 143\n",
      "--- gene=347, id=321 removed by soft count = 272\n",
      "--- gene=347, id=344 removed by soft count = 57\n",
      "--- number of probes: 399 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 348\n",
      "--- gene=348, id=111 removed by soft count = 177\n",
      "--- gene=348, id=112 removed by soft count = 69\n",
      "--- gene=348, id=155 removed by soft count = 288\n",
      "--- gene=348, id=156 removed by soft count = 175\n",
      "--- gene=348, id=161 removed by soft count = 55\n",
      "--- gene=348, id=165 removed by soft count = 58\n",
      "--- gene=348, id=168 removed by soft count = 322\n",
      "--- gene=348, id=169 removed by soft count = 65\n",
      "--- number of probes: 400 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 349\n",
      "--- gene=349, id=20 removed by soft count = 4117\n",
      "--- gene=349, id=21 removed by soft count = 228\n",
      "--- gene=349, id=308 removed by soft count = 2741\n",
      "--- number of probes: 396 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 350\n",
      "--- gene=350, id=325 removed by soft count = 688\n",
      "--- gene=350, id=382 removed by soft count = 113\n",
      "--- gene=350, id=384 removed by soft count = 75\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 351\n",
      "--- gene=351, id=58 removed by soft count = 162\n",
      "--- gene=351, id=80 removed by soft count = 92\n",
      "--- gene=351, id=82 removed by soft count = 70\n",
      "--- gene=351, id=83 removed by soft count = 71\n",
      "--- gene=351, id=333 removed by soft count = 62\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 352\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 353\n",
      "--- gene=353, id=299 removed by soft count = 898\n",
      "--- gene=353, id=344 removed by soft count = 132\n",
      "--- gene=353, id=379 removed by soft count = 60\n",
      "--- gene=353, id=380 removed by soft count = 77\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 354\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 355\n",
      "--- gene=355, id=237 removed by soft count = 140\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 356\n",
      "--- gene=356, id=34 removed by soft count = 81\n",
      "--- gene=356, id=80 removed by soft count = 57\n",
      "--- gene=356, id=115 removed by hard count = 5\n",
      "--- gene=356, id=119 removed by soft count = 715\n",
      "--- gene=356, id=120 removed by soft count = 139\n",
      "--- gene=356, id=126 removed by soft count = 132\n",
      "--- gene=356, id=178 removed by soft count = 57\n",
      "--- gene=356, id=374 removed by soft count = 115\n",
      "--- gene=356, id=391 removed by soft count = 254\n",
      "--- gene=356, id=392 removed by soft count = 307\n",
      "--- gene=356, id=393 removed by soft count = 439\n",
      "--- gene=356, id=394 removed by soft count = 115\n",
      "--- gene=356, id=395 removed by soft count = 551\n",
      "--- gene=356, id=396 removed by soft count = 823\n",
      "--- number of probes: 400 , kept by blast: 386 , if remove dups: 386\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 357\n",
      "--- gene=357, id=370 removed by soft count = 2577\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 358\n",
      "--- gene=358, id=127 removed by soft count = 63\n",
      "--- gene=358, id=147 removed by soft count = 61\n",
      "--- gene=358, id=148 removed by soft count = 327\n",
      "--- gene=358, id=284 removed by soft count = 160\n",
      "--- gene=358, id=287 removed by soft count = 90\n",
      "--- gene=358, id=342 removed by soft count = 172\n",
      "--- gene=358, id=343 removed by soft count = 740\n",
      "--- gene=358, id=344 removed by soft count = 11409\n",
      "--- number of probes: 400 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 359\n",
      "--- gene=359, id=25 removed by soft count = 170\n",
      "--- gene=359, id=46 removed by soft count = 66\n",
      "--- gene=359, id=60 removed by soft count = 61\n",
      "--- gene=359, id=62 removed by soft count = 127\n",
      "--- gene=359, id=64 removed by soft count = 132\n",
      "--- gene=359, id=127 removed by soft count = 82\n",
      "--- gene=359, id=189 removed by soft count = 130\n",
      "--- gene=359, id=192 removed by soft count = 490\n",
      "--- gene=359, id=203 removed by soft count = 886\n",
      "--- gene=359, id=217 removed by soft count = 205\n",
      "--- number of probes: 400 , kept by blast: 390 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 360\n",
      "--- gene=360, id=153 removed by soft count = 83\n",
      "--- gene=360, id=159 removed by soft count = 63\n",
      "--- gene=360, id=250 removed by soft count = 447\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 361\n",
      "--- gene=361, id=1 removed by soft count = 65\n",
      "--- gene=361, id=132 removed by soft count = 58\n",
      "--- gene=361, id=359 removed by soft count = 194\n",
      "--- gene=361, id=372 removed by soft count = 68\n",
      "--- gene=361, id=387 removed by soft count = 159\n",
      "--- gene=361, id=395 removed by soft count = 97\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 362\n",
      "--- gene=362, id=21 removed by soft count = 56\n",
      "--- gene=362, id=45 removed by soft count = 2720\n",
      "--- gene=362, id=120 removed by soft count = 716\n",
      "--- gene=362, id=124 removed by soft count = 217\n",
      "--- gene=362, id=141 removed by soft count = 178\n",
      "--- gene=362, id=192 removed by soft count = 147\n",
      "--- gene=362, id=198 removed by soft count = 985\n",
      "--- number of probes: 320 , kept by blast: 313 , if remove dups: 313\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 363\n",
      "--- gene=363, id=69 removed by soft count = 63\n",
      "--- gene=363, id=132 removed by soft count = 119\n",
      "--- gene=363, id=158 removed by soft count = 165\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=363, id=159 removed by soft count = 90\n",
      "--- gene=363, id=258 removed by soft count = 55\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 364\n",
      "--- gene=364, id=110 removed by soft count = 104\n",
      "--- gene=364, id=254 removed by soft count = 76\n",
      "--- gene=364, id=269 removed by soft count = 98\n",
      "--- number of probes: 376 , kept by blast: 373 , if remove dups: 373\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 365\n",
      "--- gene=365, id=12 removed by soft count = 646\n",
      "--- gene=365, id=31 removed by soft count = 101\n",
      "--- gene=365, id=49 removed by soft count = 153\n",
      "--- gene=365, id=51 removed by soft count = 222\n",
      "--- gene=365, id=122 removed by soft count = 109\n",
      "--- gene=365, id=131 removed by soft count = 63\n",
      "--- gene=365, id=135 removed by soft count = 72\n",
      "--- gene=365, id=204 removed by soft count = 83\n",
      "--- gene=365, id=208 removed by soft count = 103\n",
      "--- gene=365, id=209 removed by soft count = 201\n",
      "--- gene=365, id=210 removed by soft count = 52\n",
      "--- gene=365, id=211 removed by soft count = 84\n",
      "--- gene=365, id=212 removed by soft count = 401\n",
      "--- gene=365, id=229 removed by soft count = 183\n",
      "--- gene=365, id=231 removed by soft count = 59\n",
      "--- gene=365, id=270 removed by soft count = 53\n",
      "--- gene=365, id=271 removed by soft count = 170\n",
      "--- gene=365, id=272 removed by soft count = 68\n",
      "--- gene=365, id=278 removed by soft count = 68\n",
      "--- gene=365, id=281 removed by soft count = 51\n",
      "--- gene=365, id=282 removed by soft count = 309\n",
      "--- gene=365, id=294 removed by soft count = 1062\n",
      "--- gene=365, id=306 removed by soft count = 150\n",
      "--- gene=365, id=341 removed by soft count = 197\n",
      "--- gene=365, id=342 removed by soft count = 93\n",
      "--- gene=365, id=370 removed by soft count = 55\n",
      "--- gene=365, id=382 removed by soft count = 57\n",
      "--- number of probes: 400 , kept by blast: 373 , if remove dups: 373\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 366\n",
      "--- gene=366, id=1 removed by soft count = 213\n",
      "--- gene=366, id=59 removed by soft count = 95\n",
      "--- gene=366, id=62 removed by soft count = 71\n",
      "--- gene=366, id=143 removed by soft count = 226\n",
      "--- gene=366, id=249 removed by hard count = 9\n",
      "--- gene=366, id=286 removed by soft count = 59\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 367\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 368\n",
      "--- gene=368, id=44 removed by soft count = 113\n",
      "--- gene=368, id=140 removed by soft count = 78\n",
      "--- gene=368, id=160 removed by soft count = 257\n",
      "--- gene=368, id=226 removed by soft count = 80\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 369\n",
      "--- gene=369, id=152 removed by soft count = 58\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 370\n",
      "--- gene=370, id=47 removed by soft count = 113\n",
      "--- gene=370, id=127 removed by soft count = 98\n",
      "--- gene=370, id=129 removed by soft count = 85\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 371\n",
      "--- gene=371, id=172 removed by soft count = 405\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 372\n",
      "--- gene=372, id=163 removed by soft count = 58\n",
      "--- gene=372, id=307 removed by soft count = 250\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 373\n",
      "--- gene=373, id=125 removed by soft count = 188\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 374\n",
      "--- gene=374, id=49 removed by soft count = 1014\n",
      "--- gene=374, id=73 removed by soft count = 52\n",
      "--- gene=374, id=123 removed by soft count = 104\n",
      "--- gene=374, id=238 removed by soft count = 573\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 375\n",
      "--- gene=375, id=52 removed by soft count = 63\n",
      "--- gene=375, id=114 removed by soft count = 236\n",
      "--- gene=375, id=162 removed by soft count = 242\n",
      "--- gene=375, id=268 removed by soft count = 81\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 376\n",
      "--- gene=376, id=42 removed by soft count = 89\n",
      "--- gene=376, id=49 removed by soft count = 100\n",
      "--- gene=376, id=287 removed by soft count = 114\n",
      "--- gene=376, id=394 removed by soft count = 68\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 377\n",
      "--- gene=377, id=176 removed by soft count = 71\n",
      "--- gene=377, id=184 removed by soft count = 66\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 378\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 379\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 380\n",
      "--- gene=380, id=394 removed by soft count = 86\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 381\n",
      "--- gene=381, id=60 removed by soft count = 76\n",
      "--- gene=381, id=72 removed by soft count = 95\n",
      "--- gene=381, id=73 removed by soft count = 224\n",
      "--- gene=381, id=294 removed by soft count = 52\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 382\n",
      "--- gene=382, id=5 removed by soft count = 57\n",
      "--- gene=382, id=146 removed by soft count = 185\n",
      "--- gene=382, id=209 removed by soft count = 69\n",
      "--- number of probes: 390 , kept by blast: 387 , if remove dups: 387\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 383\n",
      "--- gene=383, id=40 removed by soft count = 111\n",
      "--- gene=383, id=42 removed by soft count = 190\n",
      "--- gene=383, id=94 removed by soft count = 55\n",
      "--- gene=383, id=110 removed by soft count = 68\n",
      "--- gene=383, id=167 removed by soft count = 98\n",
      "--- gene=383, id=284 removed by soft count = 131\n",
      "--- gene=383, id=288 removed by soft count = 133\n",
      "--- gene=383, id=289 removed by soft count = 73\n",
      "--- gene=383, id=290 removed by soft count = 202\n",
      "--- gene=383, id=298 removed by soft count = 69\n",
      "--- gene=383, id=302 removed by soft count = 773\n",
      "--- gene=383, id=315 removed by soft count = 147\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=383, id=318 removed by soft count = 205\n",
      "--- gene=383, id=328 removed by soft count = 208\n",
      "--- gene=383, id=358 removed by soft count = 60\n",
      "--- gene=383, id=359 removed by soft count = 487\n",
      "--- gene=383, id=365 removed by soft count = 469\n",
      "--- gene=383, id=366 removed by soft count = 414\n",
      "--- gene=383, id=384 removed by soft count = 75\n",
      "--- gene=383, id=385 removed by soft count = 236\n",
      "--- gene=383, id=389 removed by soft count = 79\n",
      "--- gene=383, id=390 removed by soft count = 101\n",
      "--- number of probes: 400 , kept by blast: 378 , if remove dups: 378\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 384\n",
      "--- gene=384, id=22 removed by soft count = 88\n",
      "--- gene=384, id=117 removed by soft count = 220\n",
      "--- gene=384, id=126 removed by soft count = 172\n",
      "--- gene=384, id=224 removed by soft count = 55\n",
      "--- gene=384, id=269 removed by soft count = 200\n",
      "--- gene=384, id=323 removed by soft count = 88\n",
      "--- gene=384, id=324 removed by soft count = 66\n",
      "--- gene=384, id=327 removed by soft count = 51\n",
      "--- gene=384, id=374 removed by soft count = 56\n",
      "--- gene=384, id=375 removed by soft count = 226\n",
      "--- gene=384, id=376 removed by soft count = 100\n",
      "--- gene=384, id=385 removed by soft count = 58\n",
      "--- number of probes: 400 , kept by blast: 388 , if remove dups: 388\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 385\n",
      "--- gene=385, id=78 removed by soft count = 240\n",
      "--- gene=385, id=130 removed by soft count = 57\n",
      "--- gene=385, id=373 removed by soft count = 61\n",
      "--- gene=385, id=386 removed by soft count = 64\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 386\n",
      "--- gene=386, id=48 removed by soft count = 80\n",
      "--- gene=386, id=222 removed by soft count = 2586\n",
      "--- gene=386, id=223 removed by soft count = 70\n",
      "--- gene=386, id=224 removed by soft count = 92\n",
      "--- gene=386, id=225 removed by soft count = 623\n",
      "--- gene=386, id=228 removed by soft count = 176\n",
      "--- gene=386, id=242 removed by soft count = 131\n",
      "--- gene=386, id=282 removed by soft count = 64\n",
      "--- gene=386, id=389 removed by soft count = 56\n",
      "--- gene=386, id=393 removed by soft count = 133\n",
      "--- number of probes: 400 , kept by blast: 390 , if remove dups: 389\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 387\n",
      "--- gene=387, id=116 removed by soft count = 164\n",
      "--- number of probes: 398 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 388\n",
      "--- gene=388, id=90 removed by soft count = 2264\n",
      "--- gene=388, id=96 removed by soft count = 1464\n",
      "--- gene=388, id=214 removed by soft count = 89\n",
      "--- gene=388, id=265 removed by soft count = 51\n",
      "--- gene=388, id=366 removed by soft count = 56\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 389\n",
      "--- gene=389, id=172 removed by soft count = 119\n",
      "--- gene=389, id=174 removed by soft count = 288\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 390\n",
      "--- gene=390, id=56 removed by soft count = 2494\n",
      "--- gene=390, id=219 removed by soft count = 67\n",
      "--- gene=390, id=263 removed by soft count = 84\n",
      "--- gene=390, id=264 removed by soft count = 132\n",
      "--- gene=390, id=337 removed by soft count = 199\n",
      "--- number of probes: 362 , kept by blast: 357 , if remove dups: 357\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 391\n",
      "--- gene=391, id=186 removed by soft count = 250\n",
      "--- gene=391, id=223 removed by soft count = 53\n",
      "--- gene=391, id=378 removed by soft count = 177\n",
      "--- gene=391, id=391 removed by soft count = 51\n",
      "--- number of probes: 399 , kept by blast: 395 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 392\n",
      "--- gene=392, id=46 removed by soft count = 187\n",
      "--- gene=392, id=145 removed by soft count = 63\n",
      "--- gene=392, id=166 removed by soft count = 185\n",
      "--- gene=392, id=175 removed by soft count = 2672\n",
      "--- gene=392, id=313 removed by soft count = 62\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 393\n",
      "--- gene=393, id=52 removed by soft count = 618\n",
      "--- gene=393, id=54 removed by soft count = 109\n",
      "--- gene=393, id=264 removed by soft count = 1027\n",
      "--- gene=393, id=299 removed by soft count = 99\n",
      "--- gene=393, id=340 removed by soft count = 142\n",
      "--- gene=393, id=385 removed by soft count = 79\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 394\n",
      "--- gene=394, id=8 removed by soft count = 706\n",
      "--- gene=394, id=159 removed by soft count = 102\n",
      "--- gene=394, id=279 removed by soft count = 86\n",
      "--- gene=394, id=288 removed by soft count = 58\n",
      "--- gene=394, id=289 removed by soft count = 95\n",
      "--- gene=394, id=291 removed by soft count = 329\n",
      "--- gene=394, id=293 removed by soft count = 184\n",
      "--- gene=394, id=297 removed by soft count = 75\n",
      "--- gene=394, id=301 removed by soft count = 72\n",
      "--- gene=394, id=341 removed by soft count = 76\n",
      "--- gene=394, id=363 removed by soft count = 105\n",
      "--- gene=394, id=369 removed by soft count = 547\n",
      "--- gene=394, id=370 removed by soft count = 1033\n",
      "--- gene=394, id=371 removed by soft count = 129\n",
      "--- gene=394, id=372 removed by soft count = 76\n",
      "--- gene=394, id=377 removed by soft count = 163\n",
      "--- gene=394, id=385 removed by soft count = 103\n",
      "--- gene=394, id=386 removed by soft count = 112\n",
      "--- gene=394, id=387 removed by soft count = 73\n",
      "--- gene=394, id=388 removed by soft count = 74\n",
      "--- gene=394, id=389 removed by soft count = 77\n",
      "--- gene=394, id=390 removed by soft count = 67\n",
      "--- gene=394, id=391 removed by soft count = 73\n",
      "--- gene=394, id=392 removed by soft count = 74\n",
      "--- gene=394, id=393 removed by soft count = 78\n",
      "--- gene=394, id=394 removed by soft count = 77\n",
      "--- gene=394, id=395 removed by soft count = 76\n",
      "--- gene=394, id=396 removed by soft count = 75\n",
      "--- gene=394, id=397 removed by soft count = 83\n",
      "--- gene=394, id=398 removed by soft count = 92\n",
      "--- gene=394, id=399 removed by soft count = 59\n",
      "--- number of probes: 400 , kept by blast: 369 , if remove dups: 369\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 395\n",
      "--- gene=395, id=21 removed by soft count = 190\n",
      "--- gene=395, id=270 removed by soft count = 65\n",
      "--- gene=395, id=315 removed by soft count = 219\n",
      "--- gene=395, id=368 removed by soft count = 109\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 396\n",
      "--- gene=396, id=2 removed by soft count = 123\n",
      "--- gene=396, id=3 removed by soft count = 131\n",
      "--- gene=396, id=45 removed by soft count = 231\n",
      "--- gene=396, id=47 removed by soft count = 159\n",
      "--- gene=396, id=48 removed by soft count = 447\n",
      "--- gene=396, id=62 removed by soft count = 68\n",
      "--- gene=396, id=178 removed by hard count = 4\n",
      "--- gene=396, id=301 removed by soft count = 373\n",
      "--- gene=396, id=383 removed by soft count = 53\n",
      "--- gene=396, id=394 removed by soft count = 54\n",
      "--- number of probes: 400 , kept by blast: 390 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 397\n",
      "--- gene=397, id=84 removed by soft count = 101\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=397, id=208 removed by soft count = 143\n",
      "--- gene=397, id=329 removed by soft count = 462\n",
      "--- gene=397, id=386 removed by soft count = 370\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 398\n",
      "--- gene=398, id=170 removed by soft count = 127\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 399\n",
      "--- gene=399, id=279 removed by soft count = 63\n",
      "--- gene=399, id=363 removed by soft count = 58\n",
      "--- gene=399, id=393 removed by soft count = 299\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 400\n",
      "--- gene=400, id=229 removed by soft count = 94\n",
      "--- gene=400, id=255 removed by soft count = 76\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 401\n",
      "--- gene=401, id=7 removed by soft count = 174\n",
      "--- gene=401, id=8 removed by soft count = 1234\n",
      "--- gene=401, id=9 removed by soft count = 84\n",
      "--- gene=401, id=12 removed by soft count = 64\n",
      "--- gene=401, id=16 removed by soft count = 192\n",
      "--- gene=401, id=18 removed by soft count = 238\n",
      "--- gene=401, id=27 removed by soft count = 1327\n",
      "--- gene=401, id=28 removed by soft count = 53\n",
      "--- gene=401, id=29 removed by soft count = 70\n",
      "--- gene=401, id=30 removed by soft count = 59\n",
      "--- gene=401, id=32 removed by soft count = 72\n",
      "--- gene=401, id=116 removed by soft count = 55\n",
      "--- gene=401, id=117 removed by soft count = 225\n",
      "--- gene=401, id=215 removed by soft count = 257\n",
      "--- gene=401, id=219 removed by soft count = 74\n",
      "--- gene=401, id=228 removed by soft count = 57\n",
      "--- gene=401, id=313 removed by soft count = 101\n",
      "--- gene=401, id=345 removed by soft count = 56\n",
      "--- gene=401, id=395 removed by soft count = 63\n",
      "--- number of probes: 400 , kept by blast: 381 , if remove dups: 381\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 402\n",
      "--- number of probes: 73 , kept by blast: 73 , if remove dups: 73\n",
      "-- number of probes kept for this region: 73\n",
      "-- checking probes in region: 403\n",
      "--- gene=403, id=255 removed by soft count = 65\n",
      "--- gene=403, id=311 removed by soft count = 63\n",
      "--- gene=403, id=382 removed by soft count = 1978\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 404\n",
      "--- gene=404, id=0 removed by soft count = 515\n",
      "--- gene=404, id=203 removed by soft count = 126\n",
      "--- gene=404, id=204 removed by soft count = 108\n",
      "--- gene=404, id=206 removed by soft count = 118\n",
      "--- gene=404, id=250 removed by soft count = 58\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 405\n",
      "--- gene=405, id=166 removed by soft count = 1742\n",
      "--- gene=405, id=209 removed by hard count = 3\n",
      "--- gene=405, id=314 removed by soft count = 75\n",
      "--- gene=405, id=315 removed by soft count = 79\n",
      "--- gene=405, id=365 removed by soft count = 140\n",
      "--- number of probes: 398 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 406\n",
      "--- gene=406, id=79 removed by soft count = 58\n",
      "--- gene=406, id=204 removed by soft count = 146\n",
      "--- gene=406, id=282 removed by soft count = 430\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 407\n",
      "--- gene=407, id=233 removed by soft count = 65\n",
      "--- gene=407, id=248 removed by soft count = 565\n",
      "--- gene=407, id=340 removed by soft count = 59\n",
      "--- gene=407, id=341 removed by soft count = 93\n",
      "--- gene=407, id=379 removed by soft count = 139\n",
      "--- gene=407, id=397 removed by soft count = 62\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 408\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 409\n",
      "--- gene=409, id=12 removed by soft count = 102\n",
      "--- gene=409, id=367 removed by soft count = 197\n",
      "--- gene=409, id=390 removed by soft count = 104\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 410\n",
      "--- gene=410, id=54 removed by soft count = 248\n",
      "--- gene=410, id=142 removed by soft count = 182\n",
      "--- gene=410, id=190 removed by soft count = 431\n",
      "--- gene=410, id=191 removed by soft count = 1307\n",
      "--- gene=410, id=287 removed by soft count = 243\n",
      "--- gene=410, id=366 removed by soft count = 241\n",
      "--- gene=410, id=369 removed by soft count = 137\n",
      "--- gene=410, id=371 removed by soft count = 56\n",
      "--- number of probes: 400 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 411\n",
      "--- gene=411, id=118 removed by soft count = 102\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 412\n",
      "--- gene=412, id=53 removed by soft count = 84\n",
      "--- gene=412, id=54 removed by soft count = 92\n",
      "--- gene=412, id=55 removed by soft count = 81\n",
      "--- gene=412, id=108 removed by soft count = 125\n",
      "--- gene=412, id=161 removed by soft count = 66\n",
      "--- gene=412, id=162 removed by soft count = 73\n",
      "--- gene=412, id=180 removed by soft count = 111\n",
      "--- gene=412, id=226 removed by soft count = 93\n",
      "--- gene=412, id=243 removed by soft count = 274\n",
      "--- gene=412, id=244 removed by soft count = 147\n",
      "--- number of probes: 400 , kept by blast: 390 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 413\n",
      "--- gene=413, id=79 removed by soft count = 77\n",
      "--- gene=413, id=352 removed by soft count = 825\n",
      "--- number of probes: 391 , kept by blast: 389 , if remove dups: 389\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 414\n",
      "--- gene=414, id=193 removed by soft count = 53\n",
      "--- gene=414, id=204 removed by soft count = 164\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 415\n",
      "--- gene=415, id=40 removed by soft count = 513\n",
      "--- gene=415, id=45 removed by soft count = 557\n",
      "--- gene=415, id=46 removed by soft count = 110\n",
      "--- gene=415, id=49 removed by soft count = 69\n",
      "--- gene=415, id=174 removed by soft count = 138\n",
      "--- gene=415, id=176 removed by soft count = 65\n",
      "--- gene=415, id=180 removed by soft count = 63\n",
      "--- gene=415, id=184 removed by soft count = 59\n",
      "--- gene=415, id=185 removed by soft count = 287\n",
      "--- gene=415, id=187 removed by soft count = 146\n",
      "--- gene=415, id=191 removed by soft count = 354\n",
      "--- gene=415, id=192 removed by soft count = 213\n",
      "--- gene=415, id=193 removed by soft count = 241\n",
      "--- gene=415, id=194 removed by soft count = 154\n",
      "--- gene=415, id=196 removed by soft count = 67\n",
      "--- gene=415, id=198 removed by soft count = 62\n",
      "--- gene=415, id=261 removed by soft count = 55\n",
      "--- gene=415, id=271 removed by soft count = 1216\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=415, id=276 removed by soft count = 391\n",
      "--- gene=415, id=279 removed by soft count = 716\n",
      "--- gene=415, id=280 removed by soft count = 159\n",
      "--- number of probes: 400 , kept by blast: 379 , if remove dups: 378\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 416\n",
      "--- gene=416, id=64 removed by soft count = 173\n",
      "--- gene=416, id=70 removed by soft count = 2049\n",
      "--- gene=416, id=73 removed by soft count = 79\n",
      "--- gene=416, id=75 removed by soft count = 864\n",
      "--- gene=416, id=123 removed by soft count = 1054\n",
      "--- gene=416, id=160 removed by hard count = 0\n",
      "--- gene=416, id=275 removed by soft count = 171\n",
      "--- gene=416, id=298 removed by soft count = 463\n",
      "--- number of probes: 400 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 417\n",
      "--- gene=417, id=182 removed by soft count = 69\n",
      "--- gene=417, id=207 removed by soft count = 306\n",
      "--- gene=417, id=291 removed by soft count = 61\n",
      "--- gene=417, id=292 removed by soft count = 2116\n",
      "--- number of probes: 399 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 418\n",
      "--- gene=418, id=306 removed by soft count = 119\n",
      "--- number of probes: 378 , kept by blast: 377 , if remove dups: 377\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 419\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 420\n",
      "--- gene=420, id=9 removed by soft count = 77\n",
      "--- gene=420, id=89 removed by soft count = 224\n",
      "--- gene=420, id=145 removed by soft count = 64\n",
      "--- gene=420, id=219 removed by soft count = 94\n",
      "--- gene=420, id=250 removed by soft count = 266\n",
      "--- gene=420, id=328 removed by soft count = 70\n",
      "--- number of probes: 391 , kept by blast: 385 , if remove dups: 385\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 421\n",
      "--- gene=421, id=120 removed by soft count = 60\n",
      "--- gene=421, id=139 removed by soft count = 69\n",
      "--- gene=421, id=249 removed by soft count = 80\n",
      "--- gene=421, id=311 removed by soft count = 60\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 422\n",
      "--- gene=422, id=22 removed by soft count = 114\n",
      "--- gene=422, id=24 removed by soft count = 123\n",
      "--- gene=422, id=37 removed by soft count = 63\n",
      "--- gene=422, id=38 removed by soft count = 802\n",
      "--- gene=422, id=39 removed by soft count = 149\n",
      "--- gene=422, id=40 removed by soft count = 736\n",
      "--- gene=422, id=41 removed by soft count = 889\n",
      "--- gene=422, id=42 removed by soft count = 914\n",
      "--- gene=422, id=364 removed by soft count = 317\n",
      "--- gene=422, id=382 removed by soft count = 197\n",
      "--- number of probes: 400 , kept by blast: 390 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 423\n",
      "--- gene=423, id=118 removed by soft count = 222\n",
      "--- gene=423, id=177 removed by soft count = 368\n",
      "--- gene=423, id=178 removed by soft count = 2740\n",
      "--- gene=423, id=183 removed by soft count = 82\n",
      "--- gene=423, id=184 removed by soft count = 53\n",
      "--- gene=423, id=185 removed by soft count = 55\n",
      "--- gene=423, id=191 removed by soft count = 124\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 424\n",
      "--- gene=424, id=213 removed by soft count = 131\n",
      "--- gene=424, id=251 removed by soft count = 106\n",
      "--- gene=424, id=252 removed by soft count = 128\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 425\n",
      "--- gene=425, id=0 removed by soft count = 237\n",
      "--- gene=425, id=182 removed by soft count = 122\n",
      "--- gene=425, id=186 removed by soft count = 72\n",
      "--- gene=425, id=311 removed by soft count = 139\n",
      "--- gene=425, id=313 removed by soft count = 1147\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 426\n",
      "--- gene=426, id=119 removed by soft count = 77\n",
      "--- gene=426, id=150 removed by soft count = 86\n",
      "--- gene=426, id=151 removed by soft count = 155\n",
      "--- gene=426, id=152 removed by soft count = 338\n",
      "--- gene=426, id=153 removed by soft count = 471\n",
      "--- gene=426, id=154 removed by soft count = 163\n",
      "--- gene=426, id=155 removed by soft count = 179\n",
      "--- gene=426, id=156 removed by soft count = 240\n",
      "--- gene=426, id=157 removed by soft count = 354\n",
      "--- gene=426, id=158 removed by soft count = 325\n",
      "--- gene=426, id=161 removed by soft count = 449\n",
      "--- gene=426, id=163 removed by soft count = 85\n",
      "--- gene=426, id=246 removed by soft count = 103\n",
      "--- gene=426, id=247 removed by soft count = 218\n",
      "--- gene=426, id=257 removed by soft count = 131\n",
      "--- gene=426, id=258 removed by soft count = 433\n",
      "--- gene=426, id=271 removed by soft count = 66\n",
      "--- gene=426, id=274 removed by soft count = 60\n",
      "--- gene=426, id=289 removed by soft count = 308\n",
      "--- gene=426, id=291 removed by soft count = 783\n",
      "--- gene=426, id=292 removed by soft count = 188\n",
      "--- gene=426, id=309 removed by soft count = 119\n",
      "--- gene=426, id=310 removed by soft count = 160\n",
      "--- gene=426, id=312 removed by soft count = 962\n",
      "--- number of probes: 313 , kept by blast: 289 , if remove dups: 289\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 427\n",
      "--- gene=427, id=79 removed by soft count = 70\n",
      "--- gene=427, id=85 removed by soft count = 211\n",
      "--- gene=427, id=166 removed by soft count = 67\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 428\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 429\n",
      "--- gene=429, id=9 removed by soft count = 105\n",
      "--- gene=429, id=11 removed by soft count = 153\n",
      "--- gene=429, id=12 removed by soft count = 78\n",
      "--- gene=429, id=182 removed by soft count = 95\n",
      "--- gene=429, id=198 removed by soft count = 235\n",
      "--- gene=429, id=208 removed by soft count = 209\n",
      "--- gene=429, id=250 removed by soft count = 382\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 430\n",
      "--- gene=430, id=14 removed by soft count = 138\n",
      "--- gene=430, id=105 removed by soft count = 68\n",
      "--- gene=430, id=114 removed by soft count = 71\n",
      "--- gene=430, id=168 removed by soft count = 66\n",
      "--- gene=430, id=169 removed by soft count = 264\n",
      "--- gene=430, id=207 removed by soft count = 168\n",
      "--- gene=430, id=250 removed by soft count = 7101\n",
      "--- gene=430, id=257 removed by soft count = 62\n",
      "--- gene=430, id=267 removed by soft count = 64\n",
      "--- gene=430, id=276 removed by soft count = 76\n",
      "--- gene=430, id=280 removed by soft count = 799\n",
      "--- gene=430, id=281 removed by soft count = 731\n",
      "--- gene=430, id=286 removed by soft count = 818\n",
      "--- gene=430, id=287 removed by soft count = 767\n",
      "--- gene=430, id=292 removed by soft count = 307\n",
      "--- gene=430, id=394 removed by soft count = 64\n",
      "--- gene=430, id=396 removed by soft count = 80\n",
      "--- number of probes: 400 , kept by blast: 383 , if remove dups: 383\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 431\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 432\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 433\n",
      "--- gene=433, id=42 removed by soft count = 4061\n",
      "--- gene=433, id=315 removed by soft count = 756\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 434\n",
      "--- gene=434, id=339 removed by soft count = 633\n",
      "--- gene=434, id=340 removed by soft count = 140\n",
      "--- gene=434, id=367 removed by soft count = 68\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 435\n",
      "--- gene=435, id=49 removed by soft count = 55\n",
      "--- gene=435, id=77 removed by soft count = 527\n",
      "--- gene=435, id=399 removed by soft count = 222\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 436\n",
      "--- gene=436, id=297 removed by soft count = 103\n",
      "--- gene=436, id=298 removed by soft count = 192\n",
      "--- number of probes: 398 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 437\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 438\n",
      "--- gene=438, id=3 removed by soft count = 149\n",
      "--- gene=438, id=17 removed by soft count = 62\n",
      "--- number of probes: 385 , kept by blast: 383 , if remove dups: 383\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 439\n",
      "--- gene=439, id=62 removed by soft count = 433\n",
      "--- gene=439, id=302 removed by soft count = 60\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 440\n",
      "--- gene=440, id=5 removed by soft count = 64\n",
      "--- gene=440, id=29 removed by soft count = 99\n",
      "--- gene=440, id=30 removed by soft count = 5353\n",
      "--- gene=440, id=307 removed by soft count = 58\n",
      "--- gene=440, id=397 removed by soft count = 60\n",
      "--- number of probes: 398 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 441\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 442\n",
      "--- gene=442, id=1 removed by soft count = 1433\n",
      "--- gene=442, id=173 removed by soft count = 65\n",
      "--- gene=442, id=362 removed by soft count = 100\n",
      "--- gene=442, id=369 removed by soft count = 99\n",
      "--- gene=442, id=389 removed by soft count = 422\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 444\n",
      "--- gene=444, id=73 removed by soft count = 1015\n",
      "--- gene=444, id=78 removed by soft count = 88\n",
      "--- gene=444, id=140 removed by soft count = 97\n",
      "--- gene=444, id=159 removed by soft count = 209\n",
      "--- gene=444, id=214 removed by soft count = 64\n",
      "--- gene=444, id=219 removed by soft count = 67\n",
      "--- gene=444, id=359 removed by soft count = 125\n",
      "--- number of probes: 393 , kept by blast: 386 , if remove dups: 386\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 445\n",
      "--- gene=445, id=16 removed by soft count = 196\n",
      "--- gene=445, id=73 removed by soft count = 175\n",
      "--- gene=445, id=150 removed by soft count = 4703\n",
      "--- gene=445, id=164 removed by soft count = 153\n",
      "--- gene=445, id=252 removed by soft count = 115\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 446\n",
      "--- gene=446, id=212 removed by hard count = 6\n",
      "--- gene=446, id=387 removed by soft count = 157\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 447\n",
      "--- gene=447, id=0 removed by soft count = 129\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 448\n",
      "--- gene=448, id=140 removed by soft count = 84\n",
      "--- gene=448, id=172 removed by soft count = 74\n",
      "--- gene=448, id=217 removed by soft count = 340\n",
      "--- gene=448, id=382 removed by soft count = 269\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 449\n",
      "--- gene=449, id=105 removed by soft count = 179\n",
      "--- gene=449, id=169 removed by soft count = 111\n",
      "--- gene=449, id=312 removed by soft count = 144\n",
      "--- gene=449, id=314 removed by soft count = 90\n",
      "--- gene=449, id=356 removed by soft count = 64\n",
      "--- gene=449, id=357 removed by soft count = 155\n",
      "--- gene=449, id=367 removed by soft count = 54\n",
      "--- gene=449, id=386 removed by soft count = 130\n",
      "--- number of probes: 400 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 450\n",
      "--- gene=450, id=42 removed by soft count = 74\n",
      "--- gene=450, id=189 removed by soft count = 96\n",
      "--- gene=450, id=199 removed by soft count = 166\n",
      "--- gene=450, id=349 removed by soft count = 1841\n",
      "--- gene=450, id=350 removed by soft count = 71\n",
      "--- gene=450, id=368 removed by soft count = 107\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 451\n",
      "--- gene=451, id=123 removed by soft count = 371\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 452\n",
      "--- gene=452, id=264 removed by soft count = 91\n",
      "--- gene=452, id=271 removed by soft count = 161\n",
      "--- gene=452, id=272 removed by soft count = 286\n",
      "--- gene=452, id=273 removed by soft count = 209\n",
      "--- gene=452, id=274 removed by soft count = 87\n",
      "--- gene=452, id=275 removed by soft count = 222\n",
      "--- gene=452, id=297 removed by soft count = 91\n",
      "--- gene=452, id=345 removed by soft count = 59\n",
      "--- gene=452, id=353 removed by soft count = 58\n",
      "--- gene=452, id=355 removed by soft count = 127\n",
      "--- number of probes: 400 , kept by blast: 390 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 453\n",
      "--- gene=453, id=92 removed by soft count = 69\n",
      "--- gene=453, id=100 removed by soft count = 208\n",
      "--- gene=453, id=124 removed by soft count = 112\n",
      "--- gene=453, id=132 removed by soft count = 91\n",
      "--- gene=453, id=133 removed by soft count = 104\n",
      "--- gene=453, id=134 removed by soft count = 131\n",
      "--- gene=453, id=167 removed by soft count = 56\n",
      "--- gene=453, id=171 removed by soft count = 124\n",
      "--- gene=453, id=274 removed by soft count = 1853\n",
      "--- gene=453, id=301 removed by soft count = 104\n",
      "--- number of probes: 400 , kept by blast: 390 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 454\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=454, id=10 removed by soft count = 101\n",
      "--- gene=454, id=12 removed by soft count = 302\n",
      "--- gene=454, id=286 removed by soft count = 75\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 455\n",
      "--- gene=455, id=37 removed by soft count = 54\n",
      "--- gene=455, id=83 removed by soft count = 175\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 456\n",
      "--- gene=456, id=35 removed by soft count = 125\n",
      "--- gene=456, id=370 removed by soft count = 77\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 457\n",
      "--- gene=457, id=2 removed by soft count = 4900\n",
      "--- gene=457, id=3 removed by soft count = 71\n",
      "--- gene=457, id=13 removed by soft count = 62\n",
      "--- gene=457, id=14 removed by soft count = 544\n",
      "--- gene=457, id=15 removed by soft count = 272\n",
      "--- gene=457, id=16 removed by soft count = 168\n",
      "--- gene=457, id=148 removed by soft count = 64\n",
      "--- gene=457, id=153 removed by soft count = 108\n",
      "--- gene=457, id=318 removed by soft count = 730\n",
      "--- gene=457, id=367 removed by soft count = 95\n",
      "--- number of probes: 400 , kept by blast: 390 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 458\n",
      "--- gene=458, id=181 removed by soft count = 68\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 459\n",
      "--- gene=459, id=20 removed by soft count = 199\n",
      "--- gene=459, id=24 removed by soft count = 283\n",
      "--- gene=459, id=26 removed by soft count = 127\n",
      "--- gene=459, id=27 removed by soft count = 300\n",
      "--- gene=459, id=62 removed by soft count = 110\n",
      "--- gene=459, id=63 removed by soft count = 71\n",
      "--- gene=459, id=98 removed by soft count = 72\n",
      "--- gene=459, id=225 removed by soft count = 244\n",
      "--- gene=459, id=367 removed by soft count = 76\n",
      "--- number of probes: 400 , kept by blast: 391 , if remove dups: 391\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 460\n",
      "--- gene=460, id=10 removed by soft count = 500\n",
      "--- gene=460, id=308 removed by soft count = 224\n",
      "--- number of probes: 399 , kept by blast: 397 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 461\n",
      "--- gene=461, id=1 removed by soft count = 105\n",
      "--- gene=461, id=117 removed by soft count = 97\n",
      "--- gene=461, id=310 removed by soft count = 367\n",
      "--- gene=461, id=325 removed by soft count = 56\n",
      "--- gene=461, id=328 removed by soft count = 62\n",
      "--- gene=461, id=329 removed by soft count = 228\n",
      "--- gene=461, id=331 removed by soft count = 70\n",
      "--- gene=461, id=377 removed by soft count = 478\n",
      "--- number of probes: 400 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 462\n",
      "--- gene=462, id=93 removed by soft count = 71\n",
      "--- gene=462, id=103 removed by soft count = 109\n",
      "--- gene=462, id=163 removed by soft count = 283\n",
      "--- gene=462, id=200 removed by soft count = 316\n",
      "--- gene=462, id=398 removed by soft count = 65\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 463\n",
      "--- gene=463, id=55 removed by soft count = 126\n",
      "--- gene=463, id=130 removed by soft count = 194\n",
      "--- gene=463, id=131 removed by soft count = 62\n",
      "--- gene=463, id=132 removed by soft count = 117\n",
      "--- gene=463, id=181 removed by soft count = 68\n",
      "--- gene=463, id=192 removed by soft count = 778\n",
      "--- gene=463, id=222 removed by soft count = 79\n",
      "--- gene=463, id=303 removed by soft count = 102\n",
      "--- number of probes: 400 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 464\n",
      "--- gene=464, id=52 removed by soft count = 842\n",
      "--- gene=464, id=275 removed by soft count = 165\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 465\n",
      "--- gene=465, id=82 removed by soft count = 398\n",
      "--- gene=465, id=118 removed by soft count = 69\n",
      "--- gene=465, id=215 removed by soft count = 132\n",
      "--- gene=465, id=238 removed by soft count = 715\n",
      "--- number of probes: 398 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 466\n",
      "--- gene=466, id=205 removed by hard count = 3\n",
      "--- number of probes: 397 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 467\n",
      "--- gene=467, id=46 removed by soft count = 245\n",
      "--- gene=467, id=266 removed by soft count = 113\n",
      "--- gene=467, id=320 removed by soft count = 541\n",
      "--- gene=467, id=321 removed by soft count = 229\n",
      "--- gene=467, id=322 removed by soft count = 5110\n",
      "--- gene=467, id=323 removed by soft count = 544\n",
      "--- gene=467, id=324 removed by soft count = 56\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 468\n",
      "--- gene=468, id=4 removed by soft count = 63\n",
      "--- gene=468, id=115 removed by soft count = 275\n",
      "--- gene=468, id=125 removed by soft count = 51\n",
      "--- gene=468, id=146 removed by soft count = 484\n",
      "--- gene=468, id=147 removed by soft count = 232\n",
      "--- gene=468, id=255 removed by soft count = 100\n",
      "--- gene=468, id=268 removed by soft count = 168\n",
      "--- gene=468, id=274 removed by soft count = 127\n",
      "--- gene=468, id=337 removed by soft count = 115\n",
      "--- gene=468, id=338 removed by soft count = 1644\n",
      "--- gene=468, id=339 removed by soft count = 406\n",
      "--- gene=468, id=395 removed by soft count = 125\n",
      "--- gene=468, id=397 removed by soft count = 72\n",
      "--- gene=468, id=399 removed by soft count = 69\n",
      "--- number of probes: 400 , kept by blast: 386 , if remove dups: 385\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 469\n",
      "--- gene=469, id=289 removed by soft count = 94\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 470\n",
      "--- gene=470, id=205 removed by soft count = 501\n",
      "--- gene=470, id=341 removed by soft count = 256\n",
      "--- gene=470, id=395 removed by soft count = 184\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 471\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 472\n",
      "--- gene=472, id=188 removed by soft count = 236\n",
      "--- gene=472, id=189 removed by soft count = 224\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 473\n",
      "--- gene=473, id=109 removed by soft count = 63\n",
      "--- gene=473, id=111 removed by soft count = 103\n",
      "--- gene=473, id=157 removed by soft count = 298\n",
      "--- gene=473, id=233 removed by soft count = 61\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=473, id=329 removed by soft count = 101\n",
      "--- number of probes: 399 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 474\n",
      "--- gene=474, id=339 removed by soft count = 721\n",
      "--- gene=474, id=356 removed by soft count = 212\n",
      "--- gene=474, id=384 removed by soft count = 56\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 475\n",
      "--- gene=475, id=80 removed by soft count = 60\n",
      "--- gene=475, id=124 removed by soft count = 220\n",
      "--- gene=475, id=343 removed by soft count = 566\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 476\n",
      "--- gene=476, id=74 removed by soft count = 99\n",
      "--- gene=476, id=75 removed by soft count = 70\n",
      "--- gene=476, id=79 removed by soft count = 1903\n",
      "--- gene=476, id=173 removed by soft count = 176\n",
      "--- gene=476, id=350 removed by soft count = 8670\n",
      "--- gene=476, id=384 removed by soft count = 127\n",
      "--- gene=476, id=386 removed by soft count = 53\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 477\n",
      "--- gene=477, id=23 removed by soft count = 63\n",
      "--- gene=477, id=24 removed by soft count = 65\n",
      "--- gene=477, id=25 removed by soft count = 223\n",
      "--- gene=477, id=235 removed by soft count = 163\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 478\n",
      "--- gene=478, id=326 removed by soft count = 867\n",
      "--- gene=478, id=332 removed by soft count = 66\n",
      "--- gene=478, id=346 removed by soft count = 74\n",
      "--- gene=478, id=348 removed by soft count = 332\n",
      "--- gene=478, id=380 removed by soft count = 199\n",
      "--- gene=478, id=381 removed by soft count = 61\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 479\n",
      "--- gene=479, id=3 removed by soft count = 67\n",
      "--- gene=479, id=138 removed by soft count = 76\n",
      "--- gene=479, id=210 removed by soft count = 62\n",
      "--- gene=479, id=213 removed by soft count = 709\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 480\n",
      "--- gene=480, id=63 removed by soft count = 157\n",
      "--- gene=480, id=140 removed by soft count = 79\n",
      "--- gene=480, id=307 removed by soft count = 69\n",
      "--- gene=480, id=321 removed by soft count = 137\n",
      "--- gene=480, id=334 removed by soft count = 59\n",
      "--- gene=480, id=346 removed by soft count = 153\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 481\n",
      "--- gene=481, id=10 removed by soft count = 52\n",
      "--- gene=481, id=158 removed by soft count = 143\n",
      "--- gene=481, id=159 removed by soft count = 251\n",
      "--- gene=481, id=216 removed by soft count = 103\n",
      "--- gene=481, id=384 removed by soft count = 125\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 482\n",
      "--- gene=482, id=29 removed by soft count = 86\n",
      "--- gene=482, id=82 removed by soft count = 60\n",
      "--- gene=482, id=248 removed by soft count = 220\n",
      "--- gene=482, id=298 removed by soft count = 643\n",
      "--- gene=482, id=351 removed by soft count = 278\n",
      "--- gene=482, id=371 removed by soft count = 58\n",
      "--- gene=482, id=384 removed by soft count = 83\n",
      "--- gene=482, id=385 removed by soft count = 122\n",
      "--- gene=482, id=386 removed by soft count = 73\n",
      "--- gene=482, id=387 removed by soft count = 107\n",
      "--- number of probes: 400 , kept by blast: 390 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 483\n",
      "--- gene=483, id=97 removed by soft count = 66\n",
      "--- gene=483, id=107 removed by soft count = 63\n",
      "--- gene=483, id=208 removed by soft count = 111\n",
      "--- gene=483, id=209 removed by soft count = 151\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 484\n",
      "--- gene=484, id=264 removed by soft count = 63\n",
      "--- gene=484, id=347 removed by soft count = 262\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 485\n",
      "--- gene=485, id=158 removed by soft count = 147\n",
      "--- gene=485, id=253 removed by soft count = 68\n",
      "--- gene=485, id=295 removed by soft count = 244\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 486\n",
      "--- gene=486, id=5 removed by soft count = 77\n",
      "--- gene=486, id=16 removed by soft count = 52\n",
      "--- gene=486, id=39 removed by soft count = 411\n",
      "--- gene=486, id=92 removed by soft count = 63\n",
      "--- gene=486, id=197 removed by soft count = 189\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 487\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 488\n",
      "--- gene=488, id=122 removed by soft count = 240\n",
      "--- gene=488, id=328 removed by soft count = 137\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 489\n",
      "--- gene=489, id=80 removed by soft count = 299\n",
      "--- gene=489, id=160 removed by soft count = 60\n",
      "--- gene=489, id=191 removed by soft count = 61\n",
      "--- gene=489, id=254 removed by soft count = 117\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 491\n",
      "--- gene=491, id=45 removed by soft count = 60\n",
      "--- gene=491, id=46 removed by soft count = 99\n",
      "--- gene=491, id=49 removed by soft count = 56\n",
      "--- gene=491, id=52 removed by soft count = 173\n",
      "--- gene=491, id=276 removed by soft count = 78\n",
      "--- gene=491, id=310 removed by soft count = 51\n",
      "--- gene=491, id=313 removed by soft count = 164\n",
      "--- gene=491, id=315 removed by soft count = 284\n",
      "--- gene=491, id=317 removed by soft count = 418\n",
      "--- gene=491, id=371 removed by soft count = 70\n",
      "--- gene=491, id=381 removed by soft count = 93\n",
      "--- gene=491, id=383 removed by soft count = 1923\n",
      "--- number of probes: 400 , kept by blast: 388 , if remove dups: 388\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 492\n",
      "--- gene=492, id=0 removed by soft count = 245\n",
      "--- gene=492, id=145 removed by soft count = 258\n",
      "--- gene=492, id=148 removed by soft count = 149\n",
      "--- gene=492, id=149 removed by soft count = 290\n",
      "--- gene=492, id=151 removed by soft count = 941\n",
      "--- gene=492, id=152 removed by soft count = 13387\n",
      "--- gene=492, id=153 removed by soft count = 1991\n",
      "--- gene=492, id=325 removed by soft count = 130\n",
      "--- number of probes: 400 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 493\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=493, id=232 removed by soft count = 109\n",
      "--- gene=493, id=264 removed by soft count = 63\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 494\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 495\n",
      "--- gene=495, id=2 removed by soft count = 89\n",
      "--- gene=495, id=71 removed by soft count = 139\n",
      "--- gene=495, id=265 removed by soft count = 52\n",
      "--- gene=495, id=329 removed by soft count = 63\n",
      "--- gene=495, id=382 removed by soft count = 56\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 496\n",
      "--- gene=496, id=137 removed by soft count = 187\n",
      "--- gene=496, id=322 removed by soft count = 175\n",
      "--- gene=496, id=344 removed by soft count = 86\n",
      "--- gene=496, id=345 removed by soft count = 16892\n",
      "--- gene=496, id=347 removed by soft count = 51\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 497\n",
      "--- gene=497, id=7 removed by soft count = 186\n",
      "--- gene=497, id=143 removed by soft count = 874\n",
      "--- gene=497, id=269 removed by soft count = 74\n",
      "--- gene=497, id=342 removed by soft count = 85\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 498\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 499\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 500\n",
      "--- gene=500, id=377 removed by soft count = 57\n",
      "--- gene=500, id=382 removed by soft count = 72\n",
      "--- gene=500, id=383 removed by soft count = 123\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 501\n",
      "--- gene=501, id=137 removed by soft count = 225\n",
      "--- gene=501, id=184 removed by soft count = 217\n",
      "--- gene=501, id=185 removed by soft count = 74\n",
      "--- gene=501, id=188 removed by soft count = 163\n",
      "--- gene=501, id=193 removed by soft count = 3911\n",
      "--- gene=501, id=194 removed by soft count = 108\n",
      "--- gene=501, id=196 removed by soft count = 60\n",
      "--- gene=501, id=198 removed by soft count = 106\n",
      "--- gene=501, id=199 removed by soft count = 96\n",
      "--- gene=501, id=200 removed by soft count = 58\n",
      "--- gene=501, id=257 removed by soft count = 137\n",
      "--- gene=501, id=344 removed by soft count = 677\n",
      "--- gene=501, id=355 removed by soft count = 1234\n",
      "--- number of probes: 397 , kept by blast: 384 , if remove dups: 384\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 502\n",
      "--- gene=502, id=33 removed by soft count = 55\n",
      "--- gene=502, id=34 removed by soft count = 385\n",
      "--- gene=502, id=106 removed by soft count = 259\n",
      "--- gene=502, id=178 removed by soft count = 67\n",
      "--- gene=502, id=338 removed by soft count = 582\n",
      "--- gene=502, id=374 removed by soft count = 99\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 503\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 504\n",
      "--- gene=504, id=21 removed by soft count = 207\n",
      "--- gene=504, id=23 removed by soft count = 110\n",
      "--- gene=504, id=25 removed by soft count = 169\n",
      "--- gene=504, id=190 removed by soft count = 125\n",
      "--- gene=504, id=266 removed by soft count = 163\n",
      "--- gene=504, id=305 removed by soft count = 57\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 505\n",
      "--- gene=505, id=152 removed by soft count = 100\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 506\n",
      "--- gene=506, id=40 removed by soft count = 108\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 507\n",
      "--- gene=507, id=50 removed by soft count = 96\n",
      "--- gene=507, id=65 removed by soft count = 77\n",
      "--- gene=507, id=123 removed by soft count = 71\n",
      "--- gene=507, id=247 removed by soft count = 82\n",
      "--- gene=507, id=302 removed by soft count = 575\n",
      "--- gene=507, id=308 removed by soft count = 73\n",
      "--- gene=507, id=359 removed by soft count = 341\n",
      "--- gene=507, id=360 removed by soft count = 167\n",
      "--- gene=507, id=361 removed by soft count = 735\n",
      "--- gene=507, id=362 removed by soft count = 82\n",
      "--- gene=507, id=363 removed by soft count = 207\n",
      "--- gene=507, id=370 removed by soft count = 396\n",
      "--- gene=507, id=391 removed by soft count = 105\n",
      "--- number of probes: 400 , kept by blast: 387 , if remove dups: 387\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 508\n",
      "--- gene=508, id=373 removed by soft count = 51\n",
      "--- gene=508, id=384 removed by soft count = 185\n",
      "--- gene=508, id=385 removed by soft count = 836\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 509\n",
      "--- gene=509, id=72 removed by soft count = 85\n",
      "--- gene=509, id=85 removed by soft count = 141\n",
      "--- gene=509, id=147 removed by soft count = 244\n",
      "--- gene=509, id=148 removed by soft count = 649\n",
      "--- gene=509, id=149 removed by soft count = 144\n",
      "--- gene=509, id=215 removed by soft count = 243\n",
      "--- gene=509, id=263 removed by soft count = 106\n",
      "--- number of probes: 396 , kept by blast: 389 , if remove dups: 388\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 510\n",
      "--- gene=510, id=260 removed by soft count = 214\n",
      "--- gene=510, id=261 removed by soft count = 63\n",
      "--- gene=510, id=287 removed by soft count = 105\n",
      "--- gene=510, id=291 removed by soft count = 381\n",
      "--- gene=510, id=304 removed by soft count = 238\n",
      "--- gene=510, id=305 removed by soft count = 155\n",
      "--- gene=510, id=317 removed by soft count = 87\n",
      "--- gene=510, id=320 removed by soft count = 69\n",
      "--- number of probes: 400 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 511\n",
      "--- gene=511, id=105 removed by soft count = 146\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 512\n",
      "--- gene=512, id=50 removed by soft count = 1030\n",
      "--- gene=512, id=60 removed by soft count = 122\n",
      "--- gene=512, id=127 removed by soft count = 753\n",
      "--- gene=512, id=162 removed by soft count = 116\n",
      "--- gene=512, id=164 removed by soft count = 178\n",
      "--- gene=512, id=222 removed by soft count = 90\n",
      "--- gene=512, id=305 removed by soft count = 3975\n",
      "--- gene=512, id=306 removed by soft count = 692\n",
      "--- number of probes: 398 , kept by blast: 390 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 513\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=513, id=171 removed by soft count = 67\n",
      "--- gene=513, id=194 removed by soft count = 83\n",
      "--- gene=513, id=196 removed by soft count = 56\n",
      "--- gene=513, id=290 removed by soft count = 361\n",
      "--- gene=513, id=312 removed by soft count = 59\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 514\n",
      "--- gene=514, id=0 removed by soft count = 102\n",
      "--- gene=514, id=174 removed by soft count = 120\n",
      "--- gene=514, id=269 removed by soft count = 124\n",
      "--- gene=514, id=271 removed by soft count = 75\n",
      "--- gene=514, id=273 removed by soft count = 83\n",
      "--- gene=514, id=281 removed by soft count = 52\n",
      "--- gene=514, id=287 removed by soft count = 69\n",
      "--- gene=514, id=295 removed by soft count = 52\n",
      "--- gene=514, id=296 removed by soft count = 212\n",
      "--- gene=514, id=309 removed by soft count = 257\n",
      "--- gene=514, id=342 removed by soft count = 115\n",
      "--- gene=514, id=399 removed by soft count = 1514\n",
      "--- number of probes: 400 , kept by blast: 388 , if remove dups: 387\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 515\n",
      "--- gene=515, id=286 removed by soft count = 67\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 516\n",
      "--- gene=516, id=80 removed by soft count = 972\n",
      "--- gene=516, id=209 removed by soft count = 1771\n",
      "--- gene=516, id=263 removed by soft count = 637\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 517\n",
      "--- gene=517, id=72 removed by soft count = 144\n",
      "--- gene=517, id=97 removed by soft count = 92\n",
      "--- gene=517, id=131 removed by soft count = 192\n",
      "--- gene=517, id=215 removed by soft count = 93\n",
      "--- gene=517, id=244 removed by soft count = 58\n",
      "--- gene=517, id=260 removed by soft count = 85\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 518\n",
      "--- gene=518, id=16 removed by soft count = 72\n",
      "--- gene=518, id=27 removed by soft count = 59\n",
      "--- gene=518, id=143 removed by soft count = 230\n",
      "--- gene=518, id=148 removed by soft count = 79\n",
      "--- gene=518, id=352 removed by soft count = 118\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 519\n",
      "--- gene=519, id=59 removed by soft count = 169\n",
      "--- gene=519, id=70 removed by soft count = 90\n",
      "--- gene=519, id=91 removed by soft count = 54\n",
      "--- gene=519, id=94 removed by soft count = 285\n",
      "--- gene=519, id=122 removed by soft count = 63\n",
      "--- gene=519, id=123 removed by soft count = 117\n",
      "--- gene=519, id=124 removed by soft count = 252\n",
      "--- gene=519, id=167 removed by soft count = 62\n",
      "--- gene=519, id=267 removed by soft count = 606\n",
      "--- number of probes: 400 , kept by blast: 391 , if remove dups: 389\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 520\n",
      "--- gene=520, id=212 removed by soft count = 102\n",
      "--- gene=520, id=383 removed by soft count = 113\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 521\n",
      "--- gene=521, id=2 removed by soft count = 66\n",
      "--- gene=521, id=9 removed by soft count = 77\n",
      "--- gene=521, id=10 removed by soft count = 53\n",
      "--- gene=521, id=11 removed by soft count = 53\n",
      "--- gene=521, id=14 removed by soft count = 75\n",
      "--- gene=521, id=127 removed by hard count = 3\n",
      "--- gene=521, id=152 removed by soft count = 139\n",
      "--- gene=521, id=184 removed by soft count = 103\n",
      "--- number of probes: 398 , kept by blast: 390 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 523\n",
      "--- gene=523, id=29 removed by soft count = 62\n",
      "--- gene=523, id=30 removed by soft count = 80\n",
      "--- gene=523, id=91 removed by soft count = 296\n",
      "--- gene=523, id=98 removed by soft count = 63\n",
      "--- gene=523, id=102 removed by soft count = 54\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 524\n",
      "--- gene=524, id=141 removed by soft count = 154\n",
      "--- number of probes: 399 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 525\n",
      "--- gene=525, id=14 removed by soft count = 59\n",
      "--- gene=525, id=305 removed by soft count = 938\n",
      "--- gene=525, id=307 removed by soft count = 165\n",
      "--- gene=525, id=311 removed by soft count = 65\n",
      "--- gene=525, id=345 removed by soft count = 54\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 526\n",
      "--- gene=526, id=20 removed by soft count = 129\n",
      "--- gene=526, id=47 removed by soft count = 60\n",
      "--- gene=526, id=139 removed by soft count = 65\n",
      "--- gene=526, id=140 removed by soft count = 51\n",
      "--- number of probes: 391 , kept by blast: 387 , if remove dups: 387\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 527\n",
      "--- gene=527, id=31 removed by soft count = 911\n",
      "--- gene=527, id=32 removed by soft count = 114\n",
      "--- gene=527, id=33 removed by soft count = 165\n",
      "--- gene=527, id=34 removed by soft count = 885\n",
      "--- gene=527, id=35 removed by soft count = 230\n",
      "--- gene=527, id=36 removed by soft count = 1679\n",
      "--- gene=527, id=37 removed by soft count = 141\n",
      "--- gene=527, id=57 removed by soft count = 83\n",
      "--- gene=527, id=111 removed by soft count = 72\n",
      "--- gene=527, id=138 removed by soft count = 222\n",
      "--- gene=527, id=251 removed by soft count = 288\n",
      "--- gene=527, id=252 removed by soft count = 85\n",
      "--- gene=527, id=282 removed by soft count = 62\n",
      "--- gene=527, id=340 removed by soft count = 92\n",
      "--- gene=527, id=349 removed by soft count = 317\n",
      "--- gene=527, id=368 removed by soft count = 51\n",
      "--- gene=527, id=369 removed by soft count = 145\n",
      "--- gene=527, id=370 removed by soft count = 159\n",
      "--- gene=527, id=374 removed by soft count = 78\n",
      "--- gene=527, id=399 removed by soft count = 52\n",
      "--- number of probes: 400 , kept by blast: 380 , if remove dups: 380\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 528\n",
      "--- gene=528, id=47 removed by soft count = 85\n",
      "--- gene=528, id=104 removed by soft count = 98\n",
      "--- gene=528, id=207 removed by soft count = 64\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 529\n",
      "--- gene=529, id=3 removed by soft count = 106\n",
      "--- gene=529, id=34 removed by soft count = 72\n",
      "--- gene=529, id=71 removed by soft count = 72\n",
      "--- gene=529, id=155 removed by soft count = 54\n",
      "--- gene=529, id=344 removed by soft count = 2796\n",
      "--- gene=529, id=353 removed by soft count = 349\n",
      "--- gene=529, id=354 removed by soft count = 141\n",
      "--- gene=529, id=368 removed by soft count = 170\n",
      "--- gene=529, id=387 removed by soft count = 62\n",
      "--- number of probes: 400 , kept by blast: 391 , if remove dups: 391\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 530\n",
      "--- gene=530, id=227 removed by soft count = 89\n",
      "--- gene=530, id=228 removed by soft count = 70\n",
      "--- gene=530, id=354 removed by soft count = 225\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 531\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=531, id=307 removed by soft count = 82\n",
      "--- gene=531, id=329 removed by soft count = 92\n",
      "--- gene=531, id=357 removed by soft count = 81\n",
      "--- gene=531, id=386 removed by soft count = 328\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 532\n",
      "--- number of probes: 395 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 533\n",
      "--- gene=533, id=18 removed by soft count = 55\n",
      "--- gene=533, id=24 removed by soft count = 92\n",
      "--- gene=533, id=61 removed by soft count = 98\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 534\n",
      "--- gene=534, id=193 removed by soft count = 55\n",
      "--- gene=534, id=364 removed by soft count = 401\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 535\n",
      "--- gene=535, id=202 removed by soft count = 108\n",
      "--- gene=535, id=310 removed by soft count = 101\n",
      "--- gene=535, id=381 removed by soft count = 595\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 536\n",
      "--- gene=536, id=177 removed by soft count = 137\n",
      "--- gene=536, id=209 removed by soft count = 152\n",
      "--- gene=536, id=216 removed by soft count = 53\n",
      "--- gene=536, id=384 removed by soft count = 334\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 537\n",
      "--- gene=537, id=71 removed by soft count = 282\n",
      "--- gene=537, id=73 removed by soft count = 450\n",
      "--- gene=537, id=134 removed by soft count = 98\n",
      "--- gene=537, id=136 removed by soft count = 159\n",
      "--- gene=537, id=172 removed by soft count = 228\n",
      "--- gene=537, id=261 removed by soft count = 104\n",
      "--- gene=537, id=279 removed by soft count = 135\n",
      "--- gene=537, id=373 removed by soft count = 80\n",
      "--- gene=537, id=381 removed by soft count = 137\n",
      "--- gene=537, id=384 removed by soft count = 52\n",
      "--- number of probes: 400 , kept by blast: 390 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 538\n",
      "--- gene=538, id=39 removed by soft count = 250\n",
      "--- gene=538, id=66 removed by soft count = 60\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 539\n",
      "--- gene=539, id=37 removed by soft count = 81\n",
      "--- gene=539, id=38 removed by soft count = 528\n",
      "--- gene=539, id=39 removed by soft count = 96\n",
      "--- gene=539, id=98 removed by soft count = 131\n",
      "--- gene=539, id=99 removed by soft count = 157\n",
      "--- gene=539, id=104 removed by soft count = 75\n",
      "--- gene=539, id=115 removed by soft count = 398\n",
      "--- gene=539, id=116 removed by soft count = 55\n",
      "--- gene=539, id=130 removed by soft count = 402\n",
      "--- gene=539, id=132 removed by soft count = 68\n",
      "--- gene=539, id=133 removed by soft count = 183\n",
      "--- gene=539, id=134 removed by soft count = 72\n",
      "--- gene=539, id=135 removed by soft count = 51\n",
      "--- gene=539, id=136 removed by soft count = 230\n",
      "--- gene=539, id=141 removed by soft count = 74\n",
      "--- gene=539, id=161 removed by soft count = 1667\n",
      "--- gene=539, id=181 removed by soft count = 103\n",
      "--- gene=539, id=205 removed by soft count = 56\n",
      "--- gene=539, id=232 removed by soft count = 115\n",
      "--- gene=539, id=243 removed by soft count = 95\n",
      "--- number of probes: 268 , kept by blast: 248 , if remove dups: 248\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 540\n",
      "--- gene=540, id=33 removed by soft count = 52\n",
      "--- gene=540, id=370 removed by soft count = 119\n",
      "--- gene=540, id=385 removed by soft count = 83\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 541\n",
      "--- gene=541, id=55 removed by soft count = 179\n",
      "--- gene=541, id=75 removed by hard count = 0\n",
      "--- gene=541, id=139 removed by soft count = 56\n",
      "--- gene=541, id=276 removed by soft count = 98\n",
      "--- gene=541, id=398 removed by soft count = 322\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 542\n",
      "--- gene=542, id=54 removed by soft count = 914\n",
      "--- gene=542, id=55 removed by soft count = 742\n",
      "--- gene=542, id=159 removed by soft count = 1330\n",
      "--- gene=542, id=160 removed by soft count = 254\n",
      "--- gene=542, id=161 removed by soft count = 55\n",
      "--- gene=542, id=164 removed by soft count = 59\n",
      "--- gene=542, id=200 removed by soft count = 54\n",
      "--- gene=542, id=201 removed by soft count = 2486\n",
      "--- gene=542, id=220 removed by soft count = 484\n",
      "--- gene=542, id=235 removed by soft count = 131\n",
      "--- gene=542, id=266 removed by soft count = 81\n",
      "--- gene=542, id=290 removed by soft count = 416\n",
      "--- gene=542, id=371 removed by soft count = 121\n",
      "--- number of probes: 400 , kept by blast: 387 , if remove dups: 387\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 543\n",
      "--- gene=543, id=85 removed by soft count = 76\n",
      "--- gene=543, id=172 removed by soft count = 126\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 544\n",
      "--- gene=544, id=78 removed by soft count = 84\n",
      "--- gene=544, id=132 removed by soft count = 1055\n",
      "--- gene=544, id=325 removed by soft count = 52\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 545\n",
      "--- gene=545, id=152 removed by soft count = 91\n",
      "--- gene=545, id=175 removed by soft count = 110\n",
      "--- gene=545, id=324 removed by soft count = 74\n",
      "--- gene=545, id=331 removed by soft count = 795\n",
      "--- gene=545, id=350 removed by soft count = 66\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 546\n",
      "--- gene=546, id=0 removed by soft count = 70\n",
      "--- gene=546, id=208 removed by soft count = 65\n",
      "--- gene=546, id=251 removed by soft count = 149\n",
      "--- gene=546, id=268 removed by soft count = 61\n",
      "--- gene=546, id=397 removed by soft count = 87\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 547\n",
      "--- gene=547, id=20 removed by soft count = 52\n",
      "--- gene=547, id=21 removed by soft count = 52\n",
      "--- gene=547, id=50 removed by hard count = 3\n",
      "--- gene=547, id=289 removed by soft count = 190\n",
      "--- gene=547, id=290 removed by soft count = 52\n",
      "--- number of probes: 373 , kept by blast: 368 , if remove dups: 368\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 548\n",
      "--- gene=548, id=77 removed by soft count = 344\n",
      "--- gene=548, id=87 removed by soft count = 221\n",
      "--- gene=548, id=88 removed by soft count = 687\n",
      "--- gene=548, id=338 removed by soft count = 67\n",
      "--- gene=548, id=350 removed by soft count = 98\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 549\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=549, id=45 removed by soft count = 53\n",
      "--- gene=549, id=144 removed by soft count = 5372\n",
      "--- gene=549, id=145 removed by soft count = 138\n",
      "--- gene=549, id=146 removed by soft count = 73\n",
      "--- gene=549, id=170 removed by soft count = 61\n",
      "--- gene=549, id=180 removed by soft count = 292\n",
      "--- gene=549, id=234 removed by soft count = 82\n",
      "--- gene=549, id=321 removed by soft count = 235\n",
      "--- gene=549, id=390 removed by soft count = 405\n",
      "--- number of probes: 400 , kept by blast: 391 , if remove dups: 391\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 550\n",
      "--- gene=550, id=0 removed by soft count = 4337\n",
      "--- gene=550, id=45 removed by soft count = 52\n",
      "--- gene=550, id=49 removed by soft count = 135\n",
      "--- gene=550, id=148 removed by soft count = 170\n",
      "--- gene=550, id=149 removed by soft count = 235\n",
      "--- gene=550, id=150 removed by soft count = 59\n",
      "--- gene=550, id=277 removed by soft count = 199\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 551\n",
      "--- gene=551, id=76 removed by soft count = 420\n",
      "--- gene=551, id=103 removed by soft count = 192\n",
      "--- gene=551, id=109 removed by soft count = 200\n",
      "--- gene=551, id=110 removed by soft count = 62\n",
      "--- gene=551, id=166 removed by soft count = 63\n",
      "--- gene=551, id=167 removed by soft count = 2144\n",
      "--- gene=551, id=169 removed by soft count = 3841\n",
      "--- gene=551, id=316 removed by soft count = 60\n",
      "--- gene=551, id=317 removed by soft count = 118\n",
      "--- gene=551, id=360 removed by soft count = 102\n",
      "--- gene=551, id=361 removed by soft count = 89\n",
      "--- number of probes: 398 , kept by blast: 387 , if remove dups: 387\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 552\n",
      "--- gene=552, id=262 removed by soft count = 156\n",
      "--- number of probes: 388 , kept by blast: 387 , if remove dups: 387\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 553\n",
      "--- gene=553, id=41 removed by soft count = 95\n",
      "--- gene=553, id=81 removed by soft count = 118\n",
      "--- gene=553, id=172 removed by soft count = 405\n",
      "--- gene=553, id=224 removed by soft count = 61\n",
      "--- gene=553, id=225 removed by soft count = 67\n",
      "--- gene=553, id=245 removed by soft count = 235\n",
      "--- gene=553, id=246 removed by soft count = 1250\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 554\n",
      "--- gene=554, id=14 removed by soft count = 67\n",
      "--- gene=554, id=371 removed by soft count = 84\n",
      "--- gene=554, id=375 removed by soft count = 57\n",
      "--- number of probes: 381 , kept by blast: 378 , if remove dups: 375\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 555\n",
      "--- gene=555, id=16 removed by soft count = 54\n",
      "--- gene=555, id=17 removed by soft count = 102\n",
      "--- gene=555, id=208 removed by soft count = 180\n",
      "--- gene=555, id=209 removed by soft count = 190\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 556\n",
      "--- gene=556, id=41 removed by soft count = 622\n",
      "--- gene=556, id=285 removed by soft count = 350\n",
      "--- gene=556, id=288 removed by soft count = 261\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 557\n",
      "--- gene=557, id=276 removed by soft count = 65\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 558\n",
      "--- gene=558, id=21 removed by soft count = 62\n",
      "--- gene=558, id=22 removed by soft count = 360\n",
      "--- gene=558, id=23 removed by soft count = 576\n",
      "--- gene=558, id=358 removed by soft count = 86\n",
      "--- gene=558, id=377 removed by soft count = 296\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 559\n",
      "--- gene=559, id=70 removed by soft count = 91\n",
      "--- gene=559, id=124 removed by soft count = 99\n",
      "--- gene=559, id=142 removed by soft count = 736\n",
      "--- gene=559, id=205 removed by soft count = 60\n",
      "--- gene=559, id=206 removed by soft count = 174\n",
      "--- gene=559, id=248 removed by soft count = 65\n",
      "--- gene=559, id=252 removed by soft count = 57\n",
      "--- number of probes: 398 , kept by blast: 391 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 560\n",
      "--- gene=560, id=0 removed by soft count = 225\n",
      "--- gene=560, id=74 removed by soft count = 92\n",
      "--- gene=560, id=75 removed by soft count = 93\n",
      "--- gene=560, id=76 removed by soft count = 91\n",
      "--- gene=560, id=77 removed by hard count = 4\n",
      "--- gene=560, id=78 removed by soft count = 92\n",
      "--- gene=560, id=112 removed by hard count = 4\n",
      "--- gene=560, id=154 removed by soft count = 66\n",
      "--- gene=560, id=220 removed by soft count = 145\n",
      "--- gene=560, id=224 removed by soft count = 264\n",
      "--- gene=560, id=372 removed by soft count = 52\n",
      "--- number of probes: 400 , kept by blast: 389 , if remove dups: 389\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 561\n",
      "--- gene=561, id=35 removed by soft count = 55\n",
      "--- gene=561, id=375 removed by soft count = 298\n",
      "--- gene=561, id=376 removed by soft count = 66\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 562\n",
      "--- gene=562, id=0 removed by soft count = 149\n",
      "--- gene=562, id=136 removed by soft count = 201\n",
      "--- gene=562, id=138 removed by soft count = 63\n",
      "--- gene=562, id=208 removed by soft count = 111\n",
      "--- gene=562, id=304 removed by soft count = 321\n",
      "--- gene=562, id=360 removed by soft count = 649\n",
      "--- gene=562, id=396 removed by soft count = 358\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 563\n",
      "--- gene=563, id=114 removed by soft count = 75\n",
      "--- gene=563, id=181 removed by soft count = 77\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 564\n",
      "--- gene=564, id=15 removed by soft count = 164\n",
      "--- gene=564, id=44 removed by soft count = 256\n",
      "--- gene=564, id=48 removed by soft count = 412\n",
      "--- gene=564, id=49 removed by soft count = 58\n",
      "--- gene=564, id=394 removed by soft count = 215\n",
      "--- gene=564, id=396 removed by soft count = 1778\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 565\n",
      "--- gene=565, id=19 removed by soft count = 1686\n",
      "--- gene=565, id=22 removed by soft count = 70\n",
      "--- gene=565, id=305 removed by soft count = 84\n",
      "--- gene=565, id=387 removed by soft count = 58\n",
      "--- gene=565, id=399 removed by soft count = 1309\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 566\n",
      "--- gene=566, id=85 removed by soft count = 558\n",
      "--- gene=566, id=184 removed by soft count = 96\n",
      "--- gene=566, id=345 removed by soft count = 1211\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 567\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=567, id=74 removed by soft count = 170\n",
      "--- gene=567, id=75 removed by soft count = 1074\n",
      "--- gene=567, id=76 removed by soft count = 114\n",
      "--- gene=567, id=77 removed by soft count = 62\n",
      "--- gene=567, id=386 removed by soft count = 330\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 568\n",
      "--- gene=568, id=66 removed by soft count = 76\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 569\n",
      "--- gene=569, id=61 removed by soft count = 219\n",
      "--- gene=569, id=68 removed by soft count = 410\n",
      "--- gene=569, id=185 removed by soft count = 70\n",
      "--- gene=569, id=249 removed by soft count = 178\n",
      "--- gene=569, id=290 removed by soft count = 104\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 570\n",
      "--- gene=570, id=170 removed by soft count = 6926\n",
      "--- gene=570, id=171 removed by soft count = 72\n",
      "--- gene=570, id=172 removed by soft count = 129\n",
      "--- gene=570, id=305 removed by soft count = 387\n",
      "--- gene=570, id=318 removed by soft count = 64\n",
      "--- gene=570, id=351 removed by soft count = 70\n",
      "--- gene=570, id=369 removed by soft count = 55\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 571\n",
      "--- gene=571, id=116 removed by soft count = 407\n",
      "--- gene=571, id=290 removed by soft count = 128\n",
      "--- gene=571, id=342 removed by soft count = 219\n",
      "--- gene=571, id=397 removed by soft count = 58\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 572\n",
      "--- gene=572, id=82 removed by soft count = 83\n",
      "--- gene=572, id=84 removed by soft count = 107\n",
      "--- gene=572, id=190 removed by soft count = 270\n",
      "--- gene=572, id=198 removed by soft count = 100\n",
      "--- gene=572, id=199 removed by soft count = 78\n",
      "--- gene=572, id=200 removed by soft count = 81\n",
      "--- gene=572, id=201 removed by soft count = 198\n",
      "--- gene=572, id=336 removed by soft count = 563\n",
      "--- number of probes: 400 , kept by blast: 392 , if remove dups: 391\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 573\n",
      "--- gene=573, id=302 removed by soft count = 121\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 574\n",
      "--- gene=574, id=0 removed by soft count = 578\n",
      "--- gene=574, id=3 removed by soft count = 9641\n",
      "--- gene=574, id=37 removed by soft count = 126\n",
      "--- gene=574, id=90 removed by soft count = 52\n",
      "--- gene=574, id=105 removed by soft count = 99\n",
      "--- gene=574, id=130 removed by hard count = 3\n",
      "--- gene=574, id=159 removed by soft count = 130\n",
      "--- gene=574, id=253 removed by soft count = 448\n",
      "--- gene=574, id=325 removed by soft count = 194\n",
      "--- gene=574, id=379 removed by soft count = 956\n",
      "--- gene=574, id=381 removed by soft count = 66\n",
      "--- number of probes: 400 , kept by blast: 389 , if remove dups: 389\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 575\n",
      "--- gene=575, id=1 removed by soft count = 115\n",
      "--- gene=575, id=50 removed by soft count = 70\n",
      "--- gene=575, id=167 removed by soft count = 121\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 576\n",
      "--- gene=576, id=17 removed by soft count = 181\n",
      "--- gene=576, id=70 removed by soft count = 122\n",
      "--- gene=576, id=73 removed by soft count = 76\n",
      "--- gene=576, id=164 removed by soft count = 63\n",
      "--- gene=576, id=297 removed by soft count = 224\n",
      "--- gene=576, id=367 removed by soft count = 149\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 577\n",
      "--- gene=577, id=42 removed by soft count = 198\n",
      "--- gene=577, id=44 removed by soft count = 141\n",
      "--- gene=577, id=100 removed by soft count = 64\n",
      "--- gene=577, id=101 removed by soft count = 86\n",
      "--- gene=577, id=102 removed by soft count = 55\n",
      "--- gene=577, id=105 removed by soft count = 105\n",
      "--- gene=577, id=168 removed by soft count = 486\n",
      "--- gene=577, id=206 removed by soft count = 99\n",
      "--- gene=577, id=349 removed by soft count = 69\n",
      "--- number of probes: 400 , kept by blast: 391 , if remove dups: 391\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 579\n",
      "--- gene=579, id=1 removed by soft count = 61\n",
      "--- gene=579, id=2 removed by soft count = 137\n",
      "--- gene=579, id=19 removed by soft count = 284\n",
      "--- gene=579, id=56 removed by soft count = 135\n",
      "--- gene=579, id=95 removed by soft count = 135\n",
      "--- gene=579, id=117 removed by soft count = 163\n",
      "--- gene=579, id=242 removed by soft count = 58\n",
      "--- gene=579, id=243 removed by soft count = 120\n",
      "--- gene=579, id=269 removed by soft count = 149\n",
      "--- gene=579, id=295 removed by soft count = 54\n",
      "--- number of probes: 400 , kept by blast: 390 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 580\n",
      "--- gene=580, id=117 removed by soft count = 139\n",
      "--- gene=580, id=168 removed by soft count = 58\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 581\n",
      "--- gene=581, id=43 removed by soft count = 88\n",
      "--- gene=581, id=81 removed by soft count = 130\n",
      "--- gene=581, id=82 removed by soft count = 192\n",
      "--- gene=581, id=203 removed by soft count = 83\n",
      "--- gene=581, id=241 removed by soft count = 301\n",
      "--- gene=581, id=294 removed by soft count = 1597\n",
      "--- gene=581, id=295 removed by soft count = 663\n",
      "--- gene=581, id=339 removed by soft count = 126\n",
      "--- number of probes: 398 , kept by blast: 390 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 582\n",
      "--- gene=582, id=187 removed by soft count = 69\n",
      "--- gene=582, id=318 removed by soft count = 136\n",
      "--- gene=582, id=320 removed by soft count = 195\n",
      "--- gene=582, id=377 removed by soft count = 128\n",
      "--- gene=582, id=391 removed by soft count = 60\n",
      "--- gene=582, id=393 removed by soft count = 66\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 583\n",
      "--- gene=583, id=61 removed by soft count = 55\n",
      "--- gene=583, id=63 removed by soft count = 213\n",
      "--- gene=583, id=104 removed by soft count = 73\n",
      "--- gene=583, id=106 removed by soft count = 129\n",
      "--- gene=583, id=111 removed by soft count = 111\n",
      "--- gene=583, id=113 removed by soft count = 116\n",
      "--- gene=583, id=116 removed by soft count = 62\n",
      "--- gene=583, id=117 removed by soft count = 53\n",
      "--- gene=583, id=138 removed by soft count = 51\n",
      "--- gene=583, id=139 removed by soft count = 368\n",
      "--- gene=583, id=305 removed by soft count = 243\n",
      "--- gene=583, id=307 removed by soft count = 92\n",
      "--- number of probes: 400 , kept by blast: 388 , if remove dups: 388\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 584\n",
      "--- gene=584, id=96 removed by soft count = 71\n",
      "--- gene=584, id=99 removed by soft count = 77\n",
      "--- gene=584, id=121 removed by soft count = 1618\n",
      "--- gene=584, id=123 removed by soft count = 301\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=584, id=163 removed by soft count = 610\n",
      "--- gene=584, id=165 removed by soft count = 64\n",
      "--- gene=584, id=212 removed by soft count = 97\n",
      "--- gene=584, id=213 removed by soft count = 214\n",
      "--- gene=584, id=215 removed by soft count = 52\n",
      "--- gene=584, id=217 removed by soft count = 151\n",
      "--- gene=584, id=218 removed by soft count = 72\n",
      "--- gene=584, id=222 removed by soft count = 53\n",
      "--- number of probes: 399 , kept by blast: 387 , if remove dups: 386\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 585\n",
      "--- gene=585, id=42 removed by soft count = 71\n",
      "--- gene=585, id=52 removed by soft count = 61\n",
      "--- gene=585, id=54 removed by soft count = 74\n",
      "--- gene=585, id=127 removed by soft count = 81\n",
      "--- number of probes: 399 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 586\n",
      "--- gene=586, id=89 removed by soft count = 91\n",
      "--- gene=586, id=384 removed by soft count = 100\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 587\n",
      "--- gene=587, id=46 removed by soft count = 861\n",
      "--- gene=587, id=47 removed by soft count = 184\n",
      "--- gene=587, id=48 removed by soft count = 129\n",
      "--- gene=587, id=206 removed by soft count = 118\n",
      "--- gene=587, id=232 removed by soft count = 64\n",
      "--- gene=587, id=316 removed by soft count = 83\n",
      "--- gene=587, id=350 removed by soft count = 75\n",
      "--- gene=587, id=351 removed by soft count = 83\n",
      "--- number of probes: 400 , kept by blast: 392 , if remove dups: 391\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 588\n",
      "--- gene=588, id=200 removed by soft count = 415\n",
      "--- gene=588, id=331 removed by soft count = 275\n",
      "--- gene=588, id=332 removed by soft count = 61\n",
      "--- gene=588, id=338 removed by soft count = 62\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 589\n",
      "--- gene=589, id=59 removed by soft count = 77\n",
      "--- gene=589, id=116 removed by soft count = 106\n",
      "--- gene=589, id=129 removed by soft count = 92\n",
      "--- gene=589, id=131 removed by soft count = 95\n",
      "--- gene=589, id=133 removed by soft count = 94\n",
      "--- gene=589, id=139 removed by soft count = 58\n",
      "--- gene=589, id=145 removed by soft count = 54\n",
      "--- gene=589, id=172 removed by soft count = 81\n",
      "--- gene=589, id=244 removed by soft count = 177\n",
      "--- gene=589, id=335 removed by soft count = 161\n",
      "--- number of probes: 400 , kept by blast: 390 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 590\n",
      "--- gene=590, id=19 removed by soft count = 81\n",
      "--- gene=590, id=59 removed by soft count = 81\n",
      "--- gene=590, id=76 removed by soft count = 138\n",
      "--- gene=590, id=235 removed by soft count = 70\n",
      "--- gene=590, id=371 removed by soft count = 165\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 591\n",
      "--- gene=591, id=112 removed by soft count = 76\n",
      "--- gene=591, id=163 removed by soft count = 54\n",
      "--- gene=591, id=204 removed by soft count = 261\n",
      "--- gene=591, id=252 removed by soft count = 60\n",
      "--- gene=591, id=262 removed by soft count = 80\n",
      "--- gene=591, id=303 removed by soft count = 73\n",
      "--- gene=591, id=345 removed by soft count = 53\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 592\n",
      "--- gene=592, id=11 removed by hard count = 0\n",
      "--- gene=592, id=33 removed by soft count = 62\n",
      "--- gene=592, id=136 removed by soft count = 76\n",
      "--- gene=592, id=225 removed by soft count = 55\n",
      "--- gene=592, id=287 removed by soft count = 62\n",
      "--- gene=592, id=340 removed by soft count = 811\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 593\n",
      "--- gene=593, id=4 removed by soft count = 69\n",
      "--- gene=593, id=5 removed by soft count = 133\n",
      "--- gene=593, id=67 removed by soft count = 51\n",
      "--- gene=593, id=213 removed by soft count = 97\n",
      "--- gene=593, id=259 removed by soft count = 205\n",
      "--- gene=593, id=276 removed by soft count = 82\n",
      "--- gene=593, id=277 removed by soft count = 91\n",
      "--- gene=593, id=393 removed by soft count = 83\n",
      "--- number of probes: 400 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 594\n",
      "--- gene=594, id=73 removed by soft count = 11693\n",
      "--- gene=594, id=161 removed by soft count = 131\n",
      "--- gene=594, id=163 removed by soft count = 23195\n",
      "--- gene=594, id=166 removed by soft count = 5676\n",
      "--- gene=594, id=201 removed by soft count = 110\n",
      "--- gene=594, id=235 removed by soft count = 51\n",
      "--- gene=594, id=275 removed by soft count = 64\n",
      "--- gene=594, id=377 removed by soft count = 62\n",
      "--- number of probes: 400 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 595\n",
      "--- gene=595, id=45 removed by soft count = 141\n",
      "--- gene=595, id=46 removed by soft count = 231\n",
      "--- gene=595, id=58 removed by soft count = 140\n",
      "--- gene=595, id=59 removed by soft count = 95\n",
      "--- gene=595, id=60 removed by soft count = 65\n",
      "--- gene=595, id=61 removed by soft count = 94\n",
      "--- gene=595, id=62 removed by soft count = 101\n",
      "--- gene=595, id=69 removed by soft count = 284\n",
      "--- gene=595, id=70 removed by soft count = 84\n",
      "--- gene=595, id=84 removed by soft count = 52\n",
      "--- gene=595, id=367 removed by soft count = 51\n",
      "--- number of probes: 400 , kept by blast: 389 , if remove dups: 389\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 596\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 597\n",
      "--- gene=597, id=111 removed by soft count = 143\n",
      "--- gene=597, id=195 removed by soft count = 328\n",
      "--- gene=597, id=259 removed by soft count = 167\n",
      "--- gene=597, id=314 removed by soft count = 145\n",
      "--- gene=597, id=315 removed by soft count = 87\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 598\n",
      "--- gene=598, id=166 removed by soft count = 111\n",
      "--- gene=598, id=212 removed by soft count = 102\n",
      "--- gene=598, id=305 removed by soft count = 1625\n",
      "--- gene=598, id=317 removed by soft count = 52\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 599\n",
      "--- gene=599, id=14 removed by soft count = 220\n",
      "--- gene=599, id=135 removed by soft count = 198\n",
      "--- gene=599, id=137 removed by soft count = 51\n",
      "--- gene=599, id=138 removed by soft count = 346\n",
      "--- gene=599, id=173 removed by soft count = 649\n",
      "--- gene=599, id=182 removed by soft count = 568\n",
      "--- gene=599, id=192 removed by soft count = 175\n",
      "--- gene=599, id=336 removed by soft count = 92\n",
      "--- number of probes: 400 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 600\n",
      "--- gene=600, id=113 removed by soft count = 90\n",
      "--- gene=600, id=114 removed by soft count = 88\n",
      "--- gene=600, id=154 removed by soft count = 71\n",
      "--- gene=600, id=157 removed by soft count = 112\n",
      "--- gene=600, id=161 removed by soft count = 112\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=600, id=172 removed by soft count = 81\n",
      "--- gene=600, id=179 removed by soft count = 112\n",
      "--- gene=600, id=302 removed by soft count = 120\n",
      "--- gene=600, id=303 removed by soft count = 269\n",
      "--- gene=600, id=350 removed by soft count = 103\n",
      "--- number of probes: 399 , kept by blast: 389 , if remove dups: 389\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 601\n",
      "--- gene=601, id=135 removed by soft count = 80\n",
      "--- gene=601, id=154 removed by soft count = 540\n",
      "--- gene=601, id=205 removed by soft count = 57\n",
      "--- gene=601, id=293 removed by soft count = 93\n",
      "--- gene=601, id=350 removed by soft count = 538\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 602\n",
      "--- gene=602, id=120 removed by soft count = 118\n",
      "--- gene=602, id=122 removed by soft count = 286\n",
      "--- gene=602, id=206 removed by soft count = 76\n",
      "--- gene=602, id=211 removed by soft count = 128\n",
      "--- gene=602, id=288 removed by soft count = 390\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 603\n",
      "--- gene=603, id=32 removed by soft count = 338\n",
      "--- gene=603, id=33 removed by soft count = 74\n",
      "--- gene=603, id=36 removed by soft count = 64\n",
      "--- gene=603, id=48 removed by soft count = 233\n",
      "--- gene=603, id=64 removed by soft count = 364\n",
      "--- gene=603, id=78 removed by soft count = 86\n",
      "--- gene=603, id=80 removed by soft count = 220\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 604\n",
      "--- gene=604, id=15 removed by soft count = 104\n",
      "--- gene=604, id=30 removed by soft count = 57\n",
      "--- gene=604, id=55 removed by soft count = 386\n",
      "--- gene=604, id=284 removed by soft count = 63\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 605\n",
      "--- gene=605, id=33 removed by soft count = 197\n",
      "--- gene=605, id=154 removed by soft count = 220\n",
      "--- gene=605, id=291 removed by hard count = 13\n",
      "--- gene=605, id=306 removed by soft count = 203\n",
      "--- gene=605, id=307 removed by soft count = 64\n",
      "--- gene=605, id=327 removed by soft count = 59\n",
      "--- gene=605, id=341 removed by soft count = 113\n",
      "--- gene=605, id=354 removed by soft count = 96\n",
      "--- number of probes: 400 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 606\n",
      "--- gene=606, id=18 removed by soft count = 290\n",
      "--- gene=606, id=19 removed by soft count = 133\n",
      "--- gene=606, id=104 removed by soft count = 101\n",
      "--- gene=606, id=105 removed by soft count = 296\n",
      "--- gene=606, id=107 removed by soft count = 207\n",
      "--- gene=606, id=108 removed by soft count = 61\n",
      "--- gene=606, id=126 removed by soft count = 62\n",
      "--- gene=606, id=393 removed by soft count = 53\n",
      "--- number of probes: 400 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 607\n",
      "--- gene=607, id=64 removed by soft count = 372\n",
      "--- gene=607, id=71 removed by soft count = 501\n",
      "--- gene=607, id=228 removed by soft count = 248\n",
      "--- gene=607, id=229 removed by soft count = 259\n",
      "--- gene=607, id=365 removed by soft count = 1133\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 608\n",
      "--- gene=608, id=122 removed by soft count = 167\n",
      "--- gene=608, id=261 removed by soft count = 64\n",
      "--- gene=608, id=270 removed by soft count = 100\n",
      "--- gene=608, id=275 removed by soft count = 210\n",
      "--- gene=608, id=277 removed by soft count = 144\n",
      "--- gene=608, id=278 removed by soft count = 236\n",
      "--- gene=608, id=346 removed by soft count = 92\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 609\n",
      "--- gene=609, id=50 removed by soft count = 1543\n",
      "--- gene=609, id=61 removed by soft count = 508\n",
      "--- gene=609, id=74 removed by soft count = 279\n",
      "--- gene=609, id=75 removed by soft count = 190\n",
      "--- gene=609, id=78 removed by soft count = 211\n",
      "--- gene=609, id=171 removed by soft count = 7229\n",
      "--- gene=609, id=302 removed by soft count = 102\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 391\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 610\n",
      "--- gene=610, id=20 removed by soft count = 132\n",
      "--- gene=610, id=32 removed by soft count = 632\n",
      "--- gene=610, id=53 removed by soft count = 240\n",
      "--- gene=610, id=55 removed by soft count = 68\n",
      "--- gene=610, id=60 removed by soft count = 787\n",
      "--- gene=610, id=62 removed by soft count = 180\n",
      "--- gene=610, id=63 removed by soft count = 983\n",
      "--- gene=610, id=64 removed by soft count = 858\n",
      "--- gene=610, id=67 removed by soft count = 178\n",
      "--- gene=610, id=73 removed by soft count = 740\n",
      "--- gene=610, id=91 removed by soft count = 81\n",
      "--- gene=610, id=298 removed by soft count = 313\n",
      "--- number of probes: 400 , kept by blast: 388 , if remove dups: 388\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 611\n",
      "--- gene=611, id=70 removed by soft count = 111\n",
      "--- gene=611, id=99 removed by soft count = 609\n",
      "--- gene=611, id=121 removed by soft count = 113\n",
      "--- gene=611, id=126 removed by soft count = 170\n",
      "--- gene=611, id=132 removed by soft count = 97\n",
      "--- gene=611, id=277 removed by soft count = 59\n",
      "--- gene=611, id=369 removed by soft count = 134\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 612\n",
      "--- gene=612, id=43 removed by soft count = 355\n",
      "--- gene=612, id=46 removed by soft count = 110\n",
      "--- gene=612, id=49 removed by soft count = 96\n",
      "--- gene=612, id=98 removed by soft count = 85\n",
      "--- gene=612, id=104 removed by soft count = 262\n",
      "--- gene=612, id=224 removed by soft count = 189\n",
      "--- gene=612, id=399 removed by soft count = 51\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 613\n",
      "--- gene=613, id=381 removed by soft count = 97\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 614\n",
      "--- gene=614, id=0 removed by soft count = 60\n",
      "--- gene=614, id=112 removed by soft count = 102\n",
      "--- gene=614, id=256 removed by soft count = 52\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 615\n",
      "--- gene=615, id=32 removed by soft count = 125\n",
      "--- gene=615, id=72 removed by soft count = 122\n",
      "--- gene=615, id=73 removed by soft count = 233\n",
      "--- gene=615, id=86 removed by soft count = 288\n",
      "--- gene=615, id=91 removed by soft count = 191\n",
      "--- gene=615, id=113 removed by soft count = 60\n",
      "--- gene=615, id=117 removed by soft count = 129\n",
      "--- gene=615, id=214 removed by soft count = 66\n",
      "--- gene=615, id=225 removed by soft count = 69\n",
      "--- gene=615, id=275 removed by soft count = 124\n",
      "--- gene=615, id=399 removed by soft count = 115\n",
      "--- number of probes: 400 , kept by blast: 389 , if remove dups: 389\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 616\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=616, id=196 removed by soft count = 54\n",
      "--- gene=616, id=257 removed by soft count = 81\n",
      "--- gene=616, id=260 removed by soft count = 1976\n",
      "--- gene=616, id=262 removed by soft count = 98\n",
      "--- gene=616, id=398 removed by soft count = 101\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 617\n",
      "--- gene=617, id=125 removed by soft count = 197\n",
      "--- gene=617, id=188 removed by soft count = 71\n",
      "--- gene=617, id=195 removed by soft count = 51\n",
      "--- gene=617, id=353 removed by soft count = 52\n",
      "--- gene=617, id=354 removed by soft count = 197\n",
      "--- gene=617, id=362 removed by soft count = 93\n",
      "--- number of probes: 399 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 618\n",
      "--- gene=618, id=198 removed by soft count = 4398\n",
      "--- gene=618, id=199 removed by soft count = 123\n",
      "--- gene=618, id=200 removed by soft count = 2878\n",
      "--- gene=618, id=202 removed by soft count = 142\n",
      "--- gene=618, id=267 removed by soft count = 889\n",
      "--- gene=618, id=383 removed by soft count = 164\n",
      "--- gene=618, id=385 removed by soft count = 100\n",
      "--- gene=618, id=386 removed by soft count = 70\n",
      "--- gene=618, id=393 removed by soft count = 283\n",
      "--- number of probes: 400 , kept by blast: 391 , if remove dups: 391\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 619\n",
      "--- gene=619, id=202 removed by soft count = 214\n",
      "--- gene=619, id=287 removed by soft count = 528\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 620\n",
      "--- gene=620, id=0 removed by soft count = 57\n",
      "--- gene=620, id=2 removed by soft count = 337\n",
      "--- gene=620, id=243 removed by soft count = 95\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 621\n",
      "--- gene=621, id=4 removed by soft count = 66\n",
      "--- gene=621, id=5 removed by soft count = 62\n",
      "--- gene=621, id=7 removed by soft count = 57\n",
      "--- gene=621, id=9 removed by soft count = 70\n",
      "--- gene=621, id=356 removed by soft count = 1637\n",
      "--- gene=621, id=358 removed by soft count = 81\n",
      "--- gene=621, id=359 removed by soft count = 137\n",
      "--- gene=621, id=360 removed by soft count = 106\n",
      "--- number of probes: 400 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 622\n",
      "--- gene=622, id=117 removed by soft count = 123\n",
      "--- gene=622, id=190 removed by hard count = 6\n",
      "--- gene=622, id=294 removed by soft count = 80\n",
      "--- gene=622, id=295 removed by soft count = 1590\n",
      "--- gene=622, id=339 removed by soft count = 61\n",
      "--- number of probes: 397 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 623\n",
      "--- gene=623, id=59 removed by soft count = 117\n",
      "--- gene=623, id=62 removed by soft count = 74\n",
      "--- gene=623, id=232 removed by soft count = 377\n",
      "--- gene=623, id=320 removed by soft count = 314\n",
      "--- gene=623, id=369 removed by soft count = 60\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 624\n",
      "--- gene=624, id=119 removed by soft count = 54\n",
      "--- gene=624, id=220 removed by soft count = 154\n",
      "--- gene=624, id=249 removed by soft count = 52\n",
      "--- gene=624, id=309 removed by soft count = 69\n",
      "--- gene=624, id=310 removed by soft count = 470\n",
      "--- gene=624, id=371 removed by soft count = 307\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 625\n",
      "--- gene=625, id=13 removed by soft count = 892\n",
      "--- gene=625, id=14 removed by soft count = 80\n",
      "--- gene=625, id=15 removed by soft count = 111\n",
      "--- gene=625, id=16 removed by soft count = 737\n",
      "--- gene=625, id=17 removed by soft count = 1378\n",
      "--- gene=625, id=18 removed by soft count = 4097\n",
      "--- gene=625, id=34 removed by soft count = 70\n",
      "--- gene=625, id=35 removed by soft count = 88\n",
      "--- gene=625, id=44 removed by soft count = 84\n",
      "--- gene=625, id=97 removed by soft count = 57\n",
      "--- gene=625, id=98 removed by soft count = 158\n",
      "--- gene=625, id=99 removed by soft count = 104\n",
      "--- gene=625, id=387 removed by soft count = 3646\n",
      "--- gene=625, id=391 removed by soft count = 1975\n",
      "--- gene=625, id=393 removed by soft count = 71\n",
      "--- gene=625, id=395 removed by soft count = 108\n",
      "--- number of probes: 400 , kept by blast: 384 , if remove dups: 384\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 626\n",
      "--- gene=626, id=47 removed by soft count = 127\n",
      "--- gene=626, id=49 removed by soft count = 58\n",
      "--- gene=626, id=56 removed by soft count = 51\n",
      "--- gene=626, id=83 removed by soft count = 103\n",
      "--- gene=626, id=112 removed by soft count = 151\n",
      "--- gene=626, id=130 removed by soft count = 53\n",
      "--- gene=626, id=146 removed by soft count = 228\n",
      "--- gene=626, id=158 removed by soft count = 117\n",
      "--- gene=626, id=162 removed by soft count = 71\n",
      "--- gene=626, id=163 removed by soft count = 92\n",
      "--- gene=626, id=172 removed by soft count = 70\n",
      "--- gene=626, id=185 removed by soft count = 225\n",
      "--- gene=626, id=205 removed by soft count = 229\n",
      "--- gene=626, id=213 removed by soft count = 382\n",
      "--- gene=626, id=328 removed by soft count = 381\n",
      "--- gene=626, id=330 removed by soft count = 81\n",
      "--- gene=626, id=369 removed by soft count = 157\n",
      "--- gene=626, id=386 removed by soft count = 68\n",
      "--- gene=626, id=393 removed by soft count = 113\n",
      "--- number of probes: 400 , kept by blast: 381 , if remove dups: 381\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 627\n",
      "--- gene=627, id=120 removed by soft count = 193\n",
      "--- gene=627, id=126 removed by soft count = 116\n",
      "--- gene=627, id=138 removed by soft count = 108\n",
      "--- gene=627, id=185 removed by soft count = 65\n",
      "--- gene=627, id=188 removed by soft count = 89\n",
      "--- gene=627, id=190 removed by soft count = 97\n",
      "--- gene=627, id=209 removed by soft count = 140\n",
      "--- gene=627, id=226 removed by soft count = 112\n",
      "--- gene=627, id=282 removed by soft count = 705\n",
      "--- gene=627, id=299 removed by soft count = 68\n",
      "--- number of probes: 400 , kept by blast: 390 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 628\n",
      "--- gene=628, id=137 removed by soft count = 116\n",
      "--- gene=628, id=203 removed by soft count = 52\n",
      "--- gene=628, id=239 removed by soft count = 188\n",
      "--- gene=628, id=240 removed by soft count = 73\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 629\n",
      "--- gene=629, id=0 removed by soft count = 316\n",
      "--- gene=629, id=115 removed by soft count = 54\n",
      "--- gene=629, id=298 removed by soft count = 104\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 630\n",
      "--- gene=630, id=37 removed by hard count = 3\n",
      "--- gene=630, id=55 removed by soft count = 258\n",
      "--- gene=630, id=69 removed by soft count = 1151\n",
      "--- gene=630, id=254 removed by soft count = 78\n",
      "--- gene=630, id=257 removed by soft count = 90\n",
      "--- gene=630, id=314 removed by soft count = 57\n",
      "--- gene=630, id=341 removed by soft count = 309\n",
      "--- number of probes: 397 , kept by blast: 390 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 631\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=631, id=34 removed by soft count = 164\n",
      "--- gene=631, id=236 removed by soft count = 172\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 632\n",
      "--- gene=632, id=28 removed by soft count = 90\n",
      "--- number of probes: 395 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 633\n",
      "--- gene=633, id=169 removed by soft count = 187\n",
      "--- gene=633, id=370 removed by soft count = 109\n",
      "--- gene=633, id=385 removed by soft count = 66\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 634\n",
      "--- gene=634, id=30 removed by soft count = 69\n",
      "--- gene=634, id=211 removed by soft count = 60\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 635\n",
      "--- gene=635, id=151 removed by soft count = 74\n",
      "--- gene=635, id=160 removed by soft count = 510\n",
      "--- gene=635, id=242 removed by soft count = 160\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 636\n",
      "--- gene=636, id=140 removed by soft count = 195\n",
      "--- gene=636, id=190 removed by soft count = 83\n",
      "--- gene=636, id=342 removed by soft count = 58\n",
      "--- gene=636, id=356 removed by soft count = 341\n",
      "--- gene=636, id=389 removed by soft count = 66\n",
      "--- gene=636, id=392 removed by soft count = 127\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 637\n",
      "--- gene=637, id=115 removed by soft count = 258\n",
      "--- gene=637, id=307 removed by soft count = 209\n",
      "--- gene=637, id=395 removed by soft count = 172\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 638\n",
      "--- gene=638, id=12 removed by soft count = 195\n",
      "--- gene=638, id=19 removed by soft count = 79\n",
      "--- gene=638, id=379 removed by soft count = 623\n",
      "--- gene=638, id=399 removed by soft count = 123\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 639\n",
      "--- gene=639, id=251 removed by soft count = 72\n",
      "--- gene=639, id=348 removed by soft count = 129\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 640\n",
      "--- gene=640, id=335 removed by soft count = 4019\n",
      "--- gene=640, id=393 removed by soft count = 248\n",
      "--- number of probes: 397 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 641\n",
      "--- gene=641, id=16 removed by soft count = 78\n",
      "--- gene=641, id=212 removed by soft count = 92\n",
      "--- gene=641, id=229 removed by soft count = 53\n",
      "--- gene=641, id=393 removed by soft count = 102\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 642\n",
      "--- gene=642, id=347 removed by soft count = 75\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 643\n",
      "--- gene=643, id=83 removed by soft count = 128\n",
      "--- gene=643, id=275 removed by soft count = 100\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 644\n",
      "--- gene=644, id=49 removed by soft count = 95\n",
      "--- gene=644, id=51 removed by soft count = 56\n",
      "--- gene=644, id=52 removed by soft count = 53\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 645\n",
      "--- gene=645, id=19 removed by soft count = 237\n",
      "--- gene=645, id=147 removed by soft count = 54\n",
      "--- gene=645, id=152 removed by soft count = 111\n",
      "--- gene=645, id=244 removed by soft count = 132\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 646\n",
      "--- gene=646, id=150 removed by soft count = 73\n",
      "--- gene=646, id=255 removed by soft count = 59\n",
      "--- gene=646, id=297 removed by soft count = 2775\n",
      "--- gene=646, id=299 removed by soft count = 54\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 647\n",
      "--- gene=647, id=1 removed by soft count = 132\n",
      "--- gene=647, id=3 removed by soft count = 60\n",
      "--- gene=647, id=38 removed by soft count = 52\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 648\n",
      "--- gene=648, id=4 removed by soft count = 144\n",
      "--- gene=648, id=347 removed by soft count = 321\n",
      "--- gene=648, id=348 removed by soft count = 66\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 649\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 650\n",
      "--- gene=650, id=14 removed by soft count = 3084\n",
      "--- gene=650, id=113 removed by soft count = 525\n",
      "--- gene=650, id=127 removed by soft count = 957\n",
      "--- gene=650, id=220 removed by soft count = 1859\n",
      "--- gene=650, id=222 removed by soft count = 151\n",
      "--- gene=650, id=223 removed by soft count = 311\n",
      "--- gene=650, id=227 removed by soft count = 57\n",
      "--- gene=650, id=322 removed by soft count = 300\n",
      "--- gene=650, id=323 removed by soft count = 449\n",
      "--- number of probes: 400 , kept by blast: 391 , if remove dups: 391\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 651\n",
      "--- gene=651, id=104 removed by soft count = 204\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 652\n",
      "--- gene=652, id=137 removed by soft count = 108\n",
      "--- gene=652, id=220 removed by soft count = 108\n",
      "--- gene=652, id=262 removed by soft count = 53\n",
      "--- gene=652, id=292 removed by soft count = 57\n",
      "--- gene=652, id=335 removed by soft count = 191\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 653\n",
      "--- gene=653, id=70 removed by soft count = 65\n",
      "--- gene=653, id=116 removed by soft count = 74\n",
      "--- gene=653, id=117 removed by soft count = 100\n",
      "--- gene=653, id=274 removed by soft count = 55\n",
      "--- gene=653, id=275 removed by soft count = 284\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 654\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=654, id=330 removed by soft count = 72\n",
      "--- gene=654, id=331 removed by soft count = 324\n",
      "--- gene=654, id=348 removed by soft count = 215\n",
      "--- gene=654, id=398 removed by soft count = 324\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 655\n",
      "--- gene=655, id=231 removed by soft count = 358\n",
      "--- gene=655, id=243 removed by soft count = 105\n",
      "--- gene=655, id=398 removed by soft count = 63\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 656\n",
      "--- gene=656, id=91 removed by soft count = 53\n",
      "--- gene=656, id=336 removed by soft count = 61\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 657\n",
      "--- gene=657, id=137 removed by soft count = 182\n",
      "--- gene=657, id=264 removed by soft count = 109\n",
      "--- gene=657, id=302 removed by soft count = 309\n",
      "--- gene=657, id=309 removed by soft count = 976\n",
      "--- gene=657, id=310 removed by soft count = 111\n",
      "--- gene=657, id=311 removed by soft count = 575\n",
      "--- gene=657, id=316 removed by soft count = 84\n",
      "--- gene=657, id=367 removed by soft count = 1638\n",
      "--- number of probes: 400 , kept by blast: 392 , if remove dups: 391\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 658\n",
      "--- gene=658, id=22 removed by soft count = 73\n",
      "--- gene=658, id=76 removed by soft count = 58\n",
      "--- gene=658, id=237 removed by soft count = 116\n",
      "--- gene=658, id=350 removed by soft count = 114\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 659\n",
      "--- gene=659, id=5 removed by soft count = 1014\n",
      "--- gene=659, id=135 removed by soft count = 750\n",
      "--- gene=659, id=315 removed by soft count = 59\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 660\n",
      "--- gene=660, id=77 removed by soft count = 212\n",
      "--- gene=660, id=163 removed by soft count = 1385\n",
      "--- gene=660, id=172 removed by soft count = 70\n",
      "--- gene=660, id=178 removed by soft count = 814\n",
      "--- gene=660, id=195 removed by soft count = 173\n",
      "--- gene=660, id=252 removed by soft count = 106\n",
      "--- gene=660, id=380 removed by soft count = 144\n",
      "--- gene=660, id=399 removed by soft count = 188\n",
      "--- number of probes: 400 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 661\n",
      "--- gene=661, id=363 removed by soft count = 76\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 662\n",
      "--- gene=662, id=21 removed by soft count = 76\n",
      "--- gene=662, id=75 removed by soft count = 59\n",
      "--- gene=662, id=77 removed by soft count = 123\n",
      "--- gene=662, id=126 removed by soft count = 97\n",
      "--- gene=662, id=197 removed by soft count = 94\n",
      "--- gene=662, id=212 removed by soft count = 156\n",
      "--- gene=662, id=237 removed by soft count = 61\n",
      "--- gene=662, id=249 removed by soft count = 73\n",
      "--- gene=662, id=250 removed by soft count = 58\n",
      "--- gene=662, id=251 removed by soft count = 103\n",
      "--- gene=662, id=341 removed by soft count = 86\n",
      "--- number of probes: 400 , kept by blast: 389 , if remove dups: 389\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 663\n",
      "--- gene=663, id=242 removed by soft count = 83\n",
      "--- gene=663, id=243 removed by soft count = 379\n",
      "--- gene=663, id=273 removed by soft count = 60\n",
      "--- gene=663, id=329 removed by soft count = 142\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 664\n",
      "--- gene=664, id=114 removed by soft count = 123\n",
      "--- gene=664, id=300 removed by soft count = 68\n",
      "--- gene=664, id=310 removed by soft count = 97\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 665\n",
      "--- gene=665, id=63 removed by soft count = 132\n",
      "--- gene=665, id=86 removed by soft count = 172\n",
      "--- gene=665, id=91 removed by soft count = 57\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 666\n",
      "--- gene=666, id=0 removed by soft count = 64\n",
      "--- gene=666, id=14 removed by soft count = 267\n",
      "--- gene=666, id=15 removed by soft count = 280\n",
      "--- gene=666, id=17 removed by soft count = 1037\n",
      "--- gene=666, id=18 removed by soft count = 72\n",
      "--- gene=666, id=19 removed by soft count = 321\n",
      "--- gene=666, id=20 removed by soft count = 369\n",
      "--- gene=666, id=23 removed by soft count = 88\n",
      "--- gene=666, id=140 removed by soft count = 413\n",
      "--- gene=666, id=141 removed by soft count = 59\n",
      "--- gene=666, id=197 removed by soft count = 76\n",
      "--- gene=666, id=344 removed by soft count = 116\n",
      "--- gene=666, id=373 removed by soft count = 225\n",
      "--- number of probes: 400 , kept by blast: 387 , if remove dups: 387\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 667\n",
      "--- gene=667, id=387 removed by soft count = 111\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 668\n",
      "--- gene=668, id=118 removed by soft count = 52\n",
      "--- gene=668, id=136 removed by soft count = 57\n",
      "--- gene=668, id=138 removed by soft count = 60\n",
      "--- gene=668, id=140 removed by soft count = 60\n",
      "--- gene=668, id=150 removed by soft count = 64\n",
      "--- gene=668, id=153 removed by soft count = 228\n",
      "--- gene=668, id=154 removed by soft count = 104\n",
      "--- gene=668, id=165 removed by soft count = 59\n",
      "--- gene=668, id=348 removed by soft count = 490\n",
      "--- number of probes: 400 , kept by blast: 391 , if remove dups: 387\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 669\n",
      "--- gene=669, id=47 removed by soft count = 93\n",
      "--- gene=669, id=182 removed by soft count = 138\n",
      "--- gene=669, id=221 removed by soft count = 53\n",
      "--- gene=669, id=223 removed by soft count = 210\n",
      "--- gene=669, id=224 removed by soft count = 227\n",
      "--- gene=669, id=225 removed by soft count = 353\n",
      "--- gene=669, id=258 removed by soft count = 95\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 670\n",
      "--- gene=670, id=153 removed by soft count = 160\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 671\n",
      "--- gene=671, id=230 removed by soft count = 101\n",
      "--- gene=671, id=265 removed by soft count = 61\n",
      "--- gene=671, id=343 removed by soft count = 56\n",
      "--- gene=671, id=370 removed by soft count = 286\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 672\n",
      "--- gene=672, id=240 removed by soft count = 60\n",
      "--- gene=672, id=254 removed by soft count = 1954\n",
      "--- gene=672, id=259 removed by soft count = 51\n",
      "--- gene=672, id=265 removed by soft count = 339\n",
      "--- gene=672, id=268 removed by soft count = 527\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=672, id=317 removed by soft count = 67\n",
      "--- gene=672, id=319 removed by soft count = 311\n",
      "--- gene=672, id=320 removed by soft count = 71\n",
      "--- gene=672, id=363 removed by soft count = 77\n",
      "--- gene=672, id=388 removed by soft count = 70\n",
      "--- number of probes: 398 , kept by blast: 388 , if remove dups: 388\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 673\n",
      "--- gene=673, id=137 removed by soft count = 205\n",
      "--- gene=673, id=277 removed by soft count = 85\n",
      "--- gene=673, id=345 removed by soft count = 95\n",
      "--- gene=673, id=346 removed by soft count = 106\n",
      "--- gene=673, id=347 removed by soft count = 110\n",
      "--- gene=673, id=384 removed by soft count = 1583\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 674\n",
      "--- gene=674, id=106 removed by soft count = 81\n",
      "--- gene=674, id=107 removed by soft count = 122\n",
      "--- gene=674, id=212 removed by soft count = 419\n",
      "--- gene=674, id=296 removed by soft count = 82\n",
      "--- gene=674, id=332 removed by soft count = 52\n",
      "--- gene=674, id=370 removed by soft count = 90\n",
      "--- gene=674, id=375 removed by soft count = 83\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 675\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 676\n",
      "--- gene=676, id=52 removed by soft count = 66\n",
      "--- gene=676, id=139 removed by soft count = 652\n",
      "--- gene=676, id=246 removed by soft count = 841\n",
      "--- gene=676, id=247 removed by soft count = 63\n",
      "--- gene=676, id=251 removed by soft count = 57\n",
      "--- gene=676, id=255 removed by soft count = 249\n",
      "--- gene=676, id=256 removed by soft count = 51\n",
      "--- gene=676, id=258 removed by soft count = 61\n",
      "--- gene=676, id=280 removed by soft count = 56\n",
      "--- gene=676, id=281 removed by soft count = 175\n",
      "--- gene=676, id=316 removed by soft count = 256\n",
      "--- number of probes: 400 , kept by blast: 389 , if remove dups: 389\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 677\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 678\n",
      "--- gene=678, id=97 removed by soft count = 295\n",
      "--- gene=678, id=98 removed by soft count = 7692\n",
      "--- gene=678, id=101 removed by soft count = 95\n",
      "--- gene=678, id=104 removed by soft count = 71\n",
      "--- gene=678, id=105 removed by soft count = 55\n",
      "--- gene=678, id=231 removed by soft count = 1317\n",
      "--- gene=678, id=245 removed by soft count = 435\n",
      "--- gene=678, id=258 removed by soft count = 141\n",
      "--- gene=678, id=259 removed by hard count = 0\n",
      "--- gene=678, id=260 removed by soft count = 125\n",
      "--- number of probes: 400 , kept by blast: 390 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 679\n",
      "--- gene=679, id=175 removed by soft count = 204\n",
      "--- gene=679, id=176 removed by soft count = 107\n",
      "--- gene=679, id=180 removed by soft count = 67\n",
      "--- gene=679, id=182 removed by soft count = 209\n",
      "--- gene=679, id=188 removed by soft count = 152\n",
      "--- gene=679, id=269 removed by soft count = 57\n",
      "--- gene=679, id=292 removed by hard count = 3\n",
      "--- gene=679, id=319 removed by soft count = 80\n",
      "--- gene=679, id=328 removed by soft count = 3851\n",
      "--- gene=679, id=357 removed by soft count = 282\n",
      "--- gene=679, id=378 removed by soft count = 53\n",
      "--- number of probes: 399 , kept by blast: 388 , if remove dups: 388\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 680\n",
      "--- gene=680, id=323 removed by soft count = 230\n",
      "--- gene=680, id=324 removed by soft count = 74\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 681\n",
      "--- gene=681, id=26 removed by soft count = 52\n",
      "--- gene=681, id=184 removed by soft count = 93\n",
      "--- gene=681, id=207 removed by soft count = 96\n",
      "--- gene=681, id=208 removed by soft count = 71\n",
      "--- gene=681, id=341 removed by soft count = 87\n",
      "--- gene=681, id=350 removed by soft count = 272\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 682\n",
      "--- gene=682, id=263 removed by soft count = 56\n",
      "--- gene=682, id=307 removed by soft count = 364\n",
      "--- gene=682, id=359 removed by soft count = 283\n",
      "--- gene=682, id=361 removed by soft count = 1643\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 683\n",
      "--- gene=683, id=98 removed by soft count = 56\n",
      "--- gene=683, id=121 removed by soft count = 182\n",
      "--- gene=683, id=122 removed by soft count = 1510\n",
      "--- gene=683, id=145 removed by soft count = 69\n",
      "--- gene=683, id=213 removed by soft count = 1020\n",
      "--- gene=683, id=281 removed by soft count = 1257\n",
      "--- gene=683, id=332 removed by soft count = 74\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 684\n",
      "--- gene=684, id=110 removed by soft count = 296\n",
      "--- gene=684, id=198 removed by soft count = 402\n",
      "--- gene=684, id=253 removed by soft count = 63\n",
      "--- gene=684, id=264 removed by soft count = 114\n",
      "--- gene=684, id=338 removed by soft count = 74\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 685\n",
      "--- gene=685, id=221 removed by soft count = 155\n",
      "--- gene=685, id=324 removed by soft count = 51\n",
      "--- number of probes: 399 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 687\n",
      "--- gene=687, id=39 removed by soft count = 66\n",
      "--- gene=687, id=304 removed by hard count = 3\n",
      "--- gene=687, id=317 removed by soft count = 86\n",
      "--- gene=687, id=377 removed by soft count = 817\n",
      "--- number of probes: 397 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 688\n",
      "--- gene=688, id=102 removed by soft count = 85\n",
      "--- gene=688, id=182 removed by soft count = 161\n",
      "--- gene=688, id=198 removed by soft count = 783\n",
      "--- gene=688, id=388 removed by soft count = 160\n",
      "--- gene=688, id=391 removed by soft count = 55\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 689\n",
      "--- gene=689, id=131 removed by soft count = 244\n",
      "--- gene=689, id=132 removed by soft count = 90\n",
      "--- gene=689, id=144 removed by soft count = 54\n",
      "--- gene=689, id=208 removed by soft count = 62\n",
      "--- gene=689, id=268 removed by soft count = 105\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 690\n",
      "--- gene=690, id=50 removed by soft count = 308\n",
      "--- gene=690, id=53 removed by soft count = 66\n",
      "--- gene=690, id=54 removed by soft count = 244\n",
      "--- gene=690, id=336 removed by soft count = 68\n",
      "--- gene=690, id=355 removed by hard count = 3\n",
      "--- gene=690, id=359 removed by soft count = 90\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 691\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 692\n",
      "--- gene=692, id=209 removed by soft count = 126\n",
      "--- gene=692, id=391 removed by soft count = 88\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 693\n",
      "--- gene=693, id=12 removed by soft count = 121\n",
      "--- gene=693, id=88 removed by soft count = 333\n",
      "--- gene=693, id=92 removed by soft count = 57\n",
      "--- gene=693, id=134 removed by soft count = 51\n",
      "--- gene=693, id=269 removed by soft count = 101\n",
      "--- gene=693, id=314 removed by soft count = 4205\n",
      "--- number of probes: 395 , kept by blast: 389 , if remove dups: 388\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 694\n",
      "--- gene=694, id=152 removed by soft count = 153\n",
      "--- gene=694, id=177 removed by soft count = 54\n",
      "--- gene=694, id=183 removed by soft count = 130\n",
      "--- gene=694, id=193 removed by soft count = 80\n",
      "--- gene=694, id=217 removed by soft count = 141\n",
      "--- gene=694, id=224 removed by soft count = 239\n",
      "--- gene=694, id=237 removed by soft count = 183\n",
      "--- gene=694, id=242 removed by soft count = 67\n",
      "--- number of probes: 400 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 695\n",
      "--- gene=695, id=10 removed by soft count = 93\n",
      "--- gene=695, id=238 removed by soft count = 1131\n",
      "--- gene=695, id=292 removed by soft count = 99\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 696\n",
      "--- gene=696, id=80 removed by soft count = 159\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 697\n",
      "--- gene=697, id=57 removed by soft count = 57\n",
      "--- gene=697, id=58 removed by soft count = 64\n",
      "--- gene=697, id=95 removed by soft count = 55\n",
      "--- gene=697, id=119 removed by soft count = 52\n",
      "--- gene=697, id=123 removed by soft count = 296\n",
      "--- gene=697, id=124 removed by soft count = 486\n",
      "--- gene=697, id=265 removed by soft count = 53\n",
      "--- gene=697, id=267 removed by soft count = 60\n",
      "--- number of probes: 400 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 698\n",
      "--- gene=698, id=310 removed by soft count = 57\n",
      "--- gene=698, id=337 removed by soft count = 527\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 699\n",
      "--- gene=699, id=189 removed by soft count = 114\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 700\n",
      "--- gene=700, id=117 removed by soft count = 75\n",
      "--- gene=700, id=368 removed by soft count = 148\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 701\n",
      "--- gene=701, id=239 removed by soft count = 255\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 702\n",
      "--- gene=702, id=2 removed by soft count = 827\n",
      "--- gene=702, id=165 removed by hard count = 3\n",
      "--- gene=702, id=168 removed by hard count = 3\n",
      "--- gene=702, id=173 removed by hard count = 3\n",
      "--- gene=702, id=176 removed by hard count = 4\n",
      "--- number of probes: 385 , kept by blast: 380 , if remove dups: 370\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 703\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 704\n",
      "--- gene=704, id=80 removed by soft count = 58\n",
      "--- gene=704, id=120 removed by soft count = 1988\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 705\n",
      "--- gene=705, id=284 removed by soft count = 335\n",
      "--- gene=705, id=393 removed by soft count = 199\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 706\n",
      "--- gene=706, id=59 removed by soft count = 300\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 707\n",
      "--- gene=707, id=7 removed by soft count = 174\n",
      "--- gene=707, id=11 removed by soft count = 111\n",
      "--- gene=707, id=12 removed by soft count = 106\n",
      "--- gene=707, id=61 removed by soft count = 67\n",
      "--- gene=707, id=64 removed by soft count = 142\n",
      "--- gene=707, id=85 removed by soft count = 581\n",
      "--- gene=707, id=383 removed by soft count = 128\n",
      "--- number of probes: 399 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 708\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 709\n",
      "--- gene=709, id=136 removed by soft count = 82\n",
      "--- gene=709, id=182 removed by soft count = 77\n",
      "--- gene=709, id=209 removed by soft count = 146\n",
      "--- gene=709, id=210 removed by soft count = 538\n",
      "--- gene=709, id=286 removed by soft count = 926\n",
      "--- gene=709, id=288 removed by soft count = 65\n",
      "--- gene=709, id=291 removed by soft count = 81\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 710\n",
      "--- gene=710, id=1 removed by soft count = 179\n",
      "--- gene=710, id=372 removed by soft count = 94\n",
      "--- gene=710, id=373 removed by soft count = 101\n",
      "--- gene=710, id=377 removed by soft count = 2369\n",
      "--- gene=710, id=378 removed by soft count = 177\n",
      "--- gene=710, id=397 removed by soft count = 207\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 711\n",
      "--- gene=711, id=153 removed by soft count = 53\n",
      "--- gene=711, id=157 removed by soft count = 79\n",
      "--- gene=711, id=187 removed by soft count = 285\n",
      "--- gene=711, id=188 removed by soft count = 105\n",
      "--- gene=711, id=189 removed by soft count = 484\n",
      "--- gene=711, id=195 removed by soft count = 152\n",
      "--- gene=711, id=196 removed by soft count = 184\n",
      "--- gene=711, id=206 removed by soft count = 171\n",
      "--- gene=711, id=207 removed by soft count = 251\n",
      "--- gene=711, id=209 removed by soft count = 149\n",
      "--- gene=711, id=311 removed by soft count = 231\n",
      "--- number of probes: 400 , kept by blast: 389 , if remove dups: 389\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 712\n",
      "--- gene=712, id=85 removed by soft count = 51\n",
      "--- gene=712, id=86 removed by soft count = 102\n",
      "--- gene=712, id=87 removed by soft count = 185\n",
      "--- gene=712, id=208 removed by soft count = 112\n",
      "--- gene=712, id=322 removed by soft count = 108\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=712, id=378 removed by soft count = 70\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 713\n",
      "--- gene=713, id=185 removed by soft count = 233\n",
      "--- gene=713, id=202 removed by soft count = 176\n",
      "--- gene=713, id=339 removed by soft count = 97\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 714\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 715\n",
      "--- gene=715, id=139 removed by soft count = 104\n",
      "--- gene=715, id=276 removed by soft count = 58\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 716\n",
      "--- gene=716, id=61 removed by soft count = 139\n",
      "--- gene=716, id=88 removed by soft count = 283\n",
      "--- gene=716, id=89 removed by soft count = 350\n",
      "--- gene=716, id=142 removed by hard count = 4\n",
      "--- gene=716, id=143 removed by hard count = 4\n",
      "--- gene=716, id=144 removed by hard count = 4\n",
      "--- gene=716, id=145 removed by hard count = 4\n",
      "--- gene=716, id=146 removed by hard count = 4\n",
      "--- gene=716, id=147 removed by hard count = 4\n",
      "--- gene=716, id=148 removed by hard count = 4\n",
      "--- gene=716, id=149 removed by hard count = 4\n",
      "--- gene=716, id=150 removed by hard count = 4\n",
      "--- gene=716, id=151 removed by hard count = 4\n",
      "--- gene=716, id=152 removed by hard count = 4\n",
      "--- gene=716, id=153 removed by hard count = 4\n",
      "--- gene=716, id=154 removed by hard count = 4\n",
      "--- gene=716, id=155 removed by hard count = 4\n",
      "--- gene=716, id=156 removed by hard count = 4\n",
      "--- gene=716, id=157 removed by hard count = 4\n",
      "--- gene=716, id=158 removed by hard count = 4\n",
      "--- gene=716, id=173 removed by soft count = 307\n",
      "--- gene=716, id=179 removed by soft count = 302\n",
      "--- gene=716, id=182 removed by soft count = 164\n",
      "--- gene=716, id=245 removed by soft count = 91\n",
      "--- gene=716, id=252 removed by soft count = 104\n",
      "--- gene=716, id=263 removed by soft count = 457\n",
      "--- gene=716, id=393 removed by soft count = 1363\n",
      "--- gene=716, id=396 removed by soft count = 57\n",
      "--- gene=716, id=397 removed by soft count = 462\n",
      "--- number of probes: 400 , kept by blast: 371 , if remove dups: 371\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 717\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 718\n",
      "--- gene=718, id=35 removed by soft count = 109\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 719\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 720\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 721\n",
      "--- gene=721, id=164 removed by soft count = 79\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 722\n",
      "--- gene=722, id=58 removed by soft count = 1787\n",
      "--- gene=722, id=349 removed by soft count = 418\n",
      "--- gene=722, id=350 removed by soft count = 125\n",
      "--- gene=722, id=351 removed by soft count = 179\n",
      "--- gene=722, id=373 removed by soft count = 63\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 723\n",
      "--- gene=723, id=214 removed by soft count = 80\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 724\n",
      "--- gene=724, id=43 removed by hard count = 0\n",
      "--- gene=724, id=154 removed by soft count = 276\n",
      "--- gene=724, id=161 removed by soft count = 56\n",
      "--- gene=724, id=222 removed by soft count = 138\n",
      "--- gene=724, id=243 removed by soft count = 125\n",
      "--- gene=724, id=265 removed by soft count = 91\n",
      "--- gene=724, id=268 removed by soft count = 428\n",
      "--- gene=724, id=370 removed by soft count = 55\n",
      "--- gene=724, id=381 removed by soft count = 290\n",
      "--- number of probes: 400 , kept by blast: 391 , if remove dups: 391\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 725\n",
      "--- gene=725, id=119 removed by soft count = 92\n",
      "--- gene=725, id=228 removed by soft count = 108\n",
      "--- gene=725, id=230 removed by soft count = 154\n",
      "--- gene=725, id=250 removed by soft count = 60\n",
      "--- gene=725, id=367 removed by soft count = 88\n",
      "--- gene=725, id=369 removed by soft count = 83\n",
      "--- gene=725, id=370 removed by soft count = 56\n",
      "--- gene=725, id=371 removed by soft count = 56\n",
      "--- gene=725, id=375 removed by soft count = 123\n",
      "--- gene=725, id=381 removed by soft count = 100\n",
      "--- gene=725, id=386 removed by soft count = 541\n",
      "--- gene=725, id=389 removed by soft count = 135\n",
      "--- gene=725, id=390 removed by soft count = 51\n",
      "--- number of probes: 400 , kept by blast: 387 , if remove dups: 387\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 726\n",
      "--- gene=726, id=121 removed by soft count = 53\n",
      "--- gene=726, id=161 removed by soft count = 52\n",
      "--- gene=726, id=162 removed by soft count = 64\n",
      "--- gene=726, id=170 removed by soft count = 58\n",
      "--- gene=726, id=174 removed by soft count = 157\n",
      "--- gene=726, id=176 removed by soft count = 74\n",
      "--- gene=726, id=177 removed by soft count = 61\n",
      "--- gene=726, id=178 removed by soft count = 221\n",
      "--- gene=726, id=181 removed by soft count = 218\n",
      "--- gene=726, id=182 removed by soft count = 145\n",
      "--- gene=726, id=266 removed by soft count = 77\n",
      "--- gene=726, id=269 removed by soft count = 135\n",
      "--- gene=726, id=309 removed by soft count = 91\n",
      "--- gene=726, id=311 removed by soft count = 58\n",
      "--- gene=726, id=312 removed by soft count = 772\n",
      "--- gene=726, id=317 removed by soft count = 745\n",
      "--- gene=726, id=319 removed by soft count = 822\n",
      "--- gene=726, id=320 removed by soft count = 130\n",
      "--- gene=726, id=321 removed by soft count = 677\n",
      "--- gene=726, id=323 removed by soft count = 62\n",
      "--- gene=726, id=324 removed by soft count = 833\n",
      "--- gene=726, id=325 removed by soft count = 822\n",
      "--- gene=726, id=329 removed by soft count = 782\n",
      "--- gene=726, id=330 removed by soft count = 770\n",
      "--- gene=726, id=331 removed by soft count = 726\n",
      "--- gene=726, id=335 removed by soft count = 119\n",
      "--- gene=726, id=336 removed by soft count = 840\n",
      "--- gene=726, id=337 removed by soft count = 854\n",
      "--- gene=726, id=339 removed by soft count = 980\n",
      "--- gene=726, id=348 removed by soft count = 733\n",
      "--- gene=726, id=375 removed by soft count = 149\n",
      "--- number of probes: 400 , kept by blast: 369 , if remove dups: 368\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 727\n",
      "--- gene=727, id=59 removed by soft count = 102\n",
      "--- gene=727, id=60 removed by soft count = 79\n",
      "--- gene=727, id=63 removed by soft count = 68\n",
      "--- gene=727, id=377 removed by soft count = 87\n",
      "--- gene=727, id=380 removed by soft count = 53\n",
      "--- number of probes: 389 , kept by blast: 384 , if remove dups: 383\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 728\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=728, id=63 removed by soft count = 88\n",
      "--- gene=728, id=153 removed by soft count = 155\n",
      "--- gene=728, id=328 removed by soft count = 74\n",
      "--- gene=728, id=334 removed by soft count = 149\n",
      "--- gene=728, id=335 removed by soft count = 168\n",
      "--- gene=728, id=386 removed by soft count = 1714\n",
      "--- gene=728, id=387 removed by soft count = 107\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 729\n",
      "--- gene=729, id=155 removed by soft count = 56\n",
      "--- gene=729, id=158 removed by soft count = 53\n",
      "--- gene=729, id=159 removed by soft count = 65\n",
      "--- gene=729, id=160 removed by soft count = 753\n",
      "--- gene=729, id=161 removed by soft count = 62\n",
      "--- gene=729, id=164 removed by soft count = 67\n",
      "--- gene=729, id=194 removed by soft count = 70\n",
      "--- gene=729, id=209 removed by soft count = 109\n",
      "--- gene=729, id=223 removed by hard count = 0\n",
      "--- number of probes: 400 , kept by blast: 391 , if remove dups: 391\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 730\n",
      "--- gene=730, id=30 removed by soft count = 112\n",
      "--- gene=730, id=50 removed by soft count = 51\n",
      "--- gene=730, id=70 removed by soft count = 68\n",
      "--- gene=730, id=77 removed by soft count = 58\n",
      "--- gene=730, id=93 removed by soft count = 70\n",
      "--- gene=730, id=129 removed by soft count = 229\n",
      "--- gene=730, id=130 removed by soft count = 149\n",
      "--- gene=730, id=365 removed by soft count = 65\n",
      "--- number of probes: 397 , kept by blast: 389 , if remove dups: 389\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 731\n",
      "--- gene=731, id=99 removed by soft count = 97\n",
      "--- gene=731, id=153 removed by soft count = 142\n",
      "--- gene=731, id=156 removed by soft count = 115\n",
      "--- gene=731, id=240 removed by soft count = 14713\n",
      "--- gene=731, id=252 removed by soft count = 129\n",
      "--- gene=731, id=259 removed by soft count = 239\n",
      "--- gene=731, id=260 removed by soft count = 138\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 732\n",
      "--- gene=732, id=11 removed by soft count = 56\n",
      "--- gene=732, id=12 removed by soft count = 61\n",
      "--- gene=732, id=14 removed by soft count = 86\n",
      "--- gene=732, id=15 removed by soft count = 419\n",
      "--- gene=732, id=16 removed by soft count = 1268\n",
      "--- gene=732, id=19 removed by soft count = 73\n",
      "--- gene=732, id=109 removed by soft count = 68\n",
      "--- gene=732, id=158 removed by soft count = 158\n",
      "--- gene=732, id=177 removed by soft count = 122\n",
      "--- number of probes: 391 , kept by blast: 382 , if remove dups: 382\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 733\n",
      "--- gene=733, id=184 removed by soft count = 64\n",
      "--- gene=733, id=228 removed by soft count = 64\n",
      "--- gene=733, id=299 removed by soft count = 74\n",
      "--- gene=733, id=300 removed by soft count = 62\n",
      "--- gene=733, id=301 removed by soft count = 54\n",
      "--- gene=733, id=302 removed by soft count = 133\n",
      "--- gene=733, id=303 removed by soft count = 118\n",
      "--- gene=733, id=364 removed by soft count = 55\n",
      "--- number of probes: 400 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 734\n",
      "--- gene=734, id=1 removed by soft count = 107\n",
      "--- gene=734, id=39 removed by soft count = 231\n",
      "--- gene=734, id=51 removed by soft count = 123\n",
      "--- gene=734, id=125 removed by soft count = 65\n",
      "--- gene=734, id=133 removed by soft count = 105\n",
      "--- gene=734, id=207 removed by soft count = 64\n",
      "--- gene=734, id=210 removed by soft count = 55\n",
      "--- gene=734, id=258 removed by soft count = 80\n",
      "--- gene=734, id=260 removed by soft count = 90\n",
      "--- gene=734, id=352 removed by soft count = 127\n",
      "--- gene=734, id=382 removed by soft count = 98\n",
      "--- gene=734, id=385 removed by soft count = 107\n",
      "--- number of probes: 400 , kept by blast: 388 , if remove dups: 388\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 735\n",
      "--- gene=735, id=22 removed by soft count = 2532\n",
      "--- gene=735, id=121 removed by soft count = 321\n",
      "--- gene=735, id=137 removed by soft count = 59\n",
      "--- gene=735, id=140 removed by soft count = 164\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 736\n",
      "--- gene=736, id=85 removed by soft count = 83\n",
      "--- gene=736, id=353 removed by soft count = 143\n",
      "--- gene=736, id=355 removed by soft count = 332\n",
      "--- gene=736, id=372 removed by soft count = 55\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 737\n",
      "--- gene=737, id=90 removed by soft count = 1085\n",
      "--- gene=737, id=131 removed by soft count = 351\n",
      "--- gene=737, id=142 removed by soft count = 385\n",
      "--- gene=737, id=143 removed by soft count = 218\n",
      "--- gene=737, id=145 removed by soft count = 53\n",
      "--- gene=737, id=148 removed by soft count = 116\n",
      "--- gene=737, id=154 removed by soft count = 222\n",
      "--- gene=737, id=159 removed by soft count = 108\n",
      "--- gene=737, id=160 removed by soft count = 187\n",
      "--- gene=737, id=162 removed by soft count = 127\n",
      "--- gene=737, id=184 removed by soft count = 79\n",
      "--- gene=737, id=185 removed by soft count = 96\n",
      "--- gene=737, id=192 removed by soft count = 983\n",
      "--- gene=737, id=292 removed by soft count = 121\n",
      "--- gene=737, id=337 removed by soft count = 58\n",
      "--- gene=737, id=362 removed by soft count = 74\n",
      "--- gene=737, id=390 removed by soft count = 154\n",
      "--- gene=737, id=391 removed by soft count = 2349\n",
      "--- gene=737, id=392 removed by soft count = 146\n",
      "--- number of probes: 400 , kept by blast: 381 , if remove dups: 381\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 738\n",
      "--- gene=738, id=8 removed by soft count = 52\n",
      "--- gene=738, id=136 removed by soft count = 60\n",
      "--- gene=738, id=137 removed by soft count = 107\n",
      "--- gene=738, id=195 removed by soft count = 70\n",
      "--- gene=738, id=364 removed by soft count = 54\n",
      "--- gene=738, id=381 removed by soft count = 89\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 739\n",
      "--- gene=739, id=50 removed by soft count = 222\n",
      "--- gene=739, id=101 removed by soft count = 62\n",
      "--- gene=739, id=114 removed by soft count = 68\n",
      "--- gene=739, id=153 removed by soft count = 56\n",
      "--- gene=739, id=202 removed by soft count = 90\n",
      "--- gene=739, id=216 removed by soft count = 214\n",
      "--- gene=739, id=217 removed by soft count = 112\n",
      "--- gene=739, id=230 removed by soft count = 89\n",
      "--- gene=739, id=235 removed by soft count = 320\n",
      "--- gene=739, id=285 removed by soft count = 57\n",
      "--- number of probes: 400 , kept by blast: 390 , if remove dups: 389\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 740\n",
      "--- gene=740, id=75 removed by soft count = 175\n",
      "--- gene=740, id=265 removed by hard count = 5\n",
      "--- gene=740, id=370 removed by soft count = 79\n",
      "--- number of probes: 388 , kept by blast: 385 , if remove dups: 385\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 741\n",
      "--- gene=741, id=63 removed by soft count = 201\n",
      "--- gene=741, id=115 removed by soft count = 112\n",
      "--- gene=741, id=205 removed by soft count = 138\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 742\n",
      "--- gene=742, id=129 removed by soft count = 102\n",
      "--- gene=742, id=131 removed by soft count = 229\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=742, id=214 removed by soft count = 59\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 743\n",
      "--- gene=743, id=317 removed by soft count = 68\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 744\n",
      "--- gene=744, id=20 removed by soft count = 313\n",
      "--- gene=744, id=21 removed by soft count = 413\n",
      "--- gene=744, id=101 removed by soft count = 55\n",
      "--- gene=744, id=105 removed by soft count = 94\n",
      "--- gene=744, id=106 removed by soft count = 1209\n",
      "--- gene=744, id=108 removed by soft count = 83\n",
      "--- gene=744, id=109 removed by soft count = 1809\n",
      "--- gene=744, id=122 removed by soft count = 151\n",
      "--- gene=744, id=125 removed by soft count = 95\n",
      "--- gene=744, id=195 removed by soft count = 90\n",
      "--- gene=744, id=196 removed by soft count = 78\n",
      "--- gene=744, id=198 removed by soft count = 73\n",
      "--- gene=744, id=223 removed by soft count = 145\n",
      "--- gene=744, id=234 removed by soft count = 100\n",
      "--- gene=744, id=250 removed by soft count = 1217\n",
      "--- gene=744, id=259 removed by soft count = 110\n",
      "--- gene=744, id=336 removed by soft count = 53\n",
      "--- number of probes: 400 , kept by blast: 383 , if remove dups: 383\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 745\n",
      "--- gene=745, id=53 removed by soft count = 171\n",
      "--- gene=745, id=185 removed by soft count = 282\n",
      "--- gene=745, id=240 removed by soft count = 128\n",
      "--- number of probes: 396 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 746\n",
      "--- gene=746, id=140 removed by soft count = 55\n",
      "--- gene=746, id=171 removed by soft count = 118\n",
      "--- gene=746, id=172 removed by soft count = 58\n",
      "--- gene=746, id=175 removed by soft count = 97\n",
      "--- gene=746, id=229 removed by soft count = 689\n",
      "--- gene=746, id=236 removed by soft count = 70\n",
      "--- number of probes: 399 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 747\n",
      "--- gene=747, id=108 removed by soft count = 80\n",
      "--- gene=747, id=200 removed by soft count = 57\n",
      "--- gene=747, id=229 removed by soft count = 386\n",
      "--- gene=747, id=230 removed by soft count = 130\n",
      "--- gene=747, id=261 removed by soft count = 78\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 748\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 749\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 750\n",
      "--- number of probes: 10 , kept by blast: 10 , if remove dups: 10\n",
      "-- number of probes kept for this region: 10\n",
      "-- checking probes in region: 1\n",
      "--- gene=1, id=14 removed by soft count = 85\n",
      "--- gene=1, id=16 removed by soft count = 781\n",
      "--- gene=1, id=17 removed by soft count = 139\n",
      "--- gene=1, id=182 removed by hard count = 4\n",
      "--- gene=1, id=210 removed by soft count = 109\n",
      "--- gene=1, id=308 removed by soft count = 918\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 2\n",
      "--- gene=2, id=52 removed by soft count = 230\n",
      "--- gene=2, id=109 removed by soft count = 721\n",
      "--- gene=2, id=315 removed by soft count = 56\n",
      "--- gene=2, id=360 removed by soft count = 59\n",
      "--- gene=2, id=373 removed by soft count = 183\n",
      "--- gene=2, id=374 removed by soft count = 419\n",
      "--- gene=2, id=393 removed by soft count = 61\n",
      "--- number of probes: 398 , kept by blast: 391 , if remove dups: 391\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 3\n",
      "--- gene=3, id=320 removed by soft count = 305\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 4\n",
      "--- gene=4, id=57 removed by soft count = 641\n",
      "--- gene=4, id=147 removed by hard count = 3\n",
      "--- gene=4, id=159 removed by soft count = 84\n",
      "--- gene=4, id=172 removed by soft count = 72\n",
      "--- gene=4, id=185 removed by soft count = 51\n",
      "--- gene=4, id=236 removed by soft count = 464\n",
      "--- gene=4, id=237 removed by soft count = 313\n",
      "--- gene=4, id=238 removed by soft count = 130\n",
      "--- gene=4, id=260 removed by soft count = 106\n",
      "--- gene=4, id=261 removed by soft count = 128\n",
      "--- gene=4, id=279 removed by soft count = 2855\n",
      "--- gene=4, id=299 removed by soft count = 88\n",
      "--- gene=4, id=345 removed by soft count = 481\n",
      "--- gene=4, id=346 removed by soft count = 72\n",
      "--- number of probes: 400 , kept by blast: 386 , if remove dups: 386\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 5\n",
      "--- gene=5, id=105 removed by soft count = 79\n",
      "--- gene=5, id=107 removed by soft count = 215\n",
      "--- gene=5, id=108 removed by soft count = 136\n",
      "--- gene=5, id=109 removed by soft count = 254\n",
      "--- gene=5, id=110 removed by soft count = 377\n",
      "--- gene=5, id=221 removed by soft count = 135\n",
      "--- gene=5, id=225 removed by soft count = 136\n",
      "--- gene=5, id=354 removed by soft count = 8427\n",
      "--- gene=5, id=363 removed by soft count = 91\n",
      "--- gene=5, id=369 removed by soft count = 62\n",
      "--- gene=5, id=377 removed by soft count = 55\n",
      "--- number of probes: 400 , kept by blast: 389 , if remove dups: 389\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 6\n",
      "--- gene=6, id=2 removed by soft count = 121\n",
      "--- gene=6, id=5 removed by soft count = 261\n",
      "--- gene=6, id=6 removed by soft count = 94\n",
      "--- gene=6, id=34 removed by soft count = 136\n",
      "--- gene=6, id=264 removed by soft count = 6325\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 7\n",
      "--- gene=7, id=25 removed by soft count = 53\n",
      "--- gene=7, id=50 removed by soft count = 184\n",
      "--- gene=7, id=59 removed by soft count = 99\n",
      "--- gene=7, id=106 removed by soft count = 65\n",
      "--- gene=7, id=107 removed by soft count = 478\n",
      "--- gene=7, id=109 removed by soft count = 60\n",
      "--- gene=7, id=134 removed by soft count = 89\n",
      "--- gene=7, id=135 removed by soft count = 696\n",
      "--- gene=7, id=136 removed by soft count = 83\n",
      "--- gene=7, id=171 removed by soft count = 102\n",
      "--- gene=7, id=180 removed by soft count = 117\n",
      "--- gene=7, id=184 removed by soft count = 185\n",
      "--- gene=7, id=187 removed by soft count = 87\n",
      "--- gene=7, id=191 removed by soft count = 101\n",
      "--- gene=7, id=227 removed by soft count = 103\n",
      "--- gene=7, id=233 removed by soft count = 288\n",
      "--- gene=7, id=234 removed by soft count = 100\n",
      "--- gene=7, id=239 removed by soft count = 2684\n",
      "--- gene=7, id=271 removed by soft count = 195\n",
      "--- gene=7, id=288 removed by soft count = 91\n",
      "--- gene=7, id=303 removed by soft count = 109\n",
      "--- gene=7, id=345 removed by soft count = 63\n",
      "--- number of probes: 400 , kept by blast: 378 , if remove dups: 378\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 8\n",
      "--- gene=8, id=188 removed by soft count = 62\n",
      "--- gene=8, id=327 removed by soft count = 73\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 9\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=9, id=1 removed by soft count = 1194\n",
      "--- gene=9, id=5 removed by soft count = 53\n",
      "--- gene=9, id=6 removed by soft count = 109\n",
      "--- gene=9, id=49 removed by soft count = 88\n",
      "--- gene=9, id=87 removed by soft count = 108\n",
      "--- gene=9, id=91 removed by soft count = 70\n",
      "--- gene=9, id=92 removed by soft count = 230\n",
      "--- gene=9, id=93 removed by soft count = 179\n",
      "--- gene=9, id=94 removed by soft count = 147\n",
      "--- gene=9, id=95 removed by soft count = 59\n",
      "--- gene=9, id=97 removed by soft count = 55\n",
      "--- gene=9, id=190 removed by soft count = 221\n",
      "--- gene=9, id=308 removed by soft count = 209\n",
      "--- gene=9, id=310 removed by soft count = 52\n",
      "--- gene=9, id=328 removed by soft count = 111\n",
      "--- number of probes: 400 , kept by blast: 385 , if remove dups: 385\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 10\n",
      "--- gene=10, id=123 removed by soft count = 441\n",
      "--- gene=10, id=205 removed by soft count = 76\n",
      "--- gene=10, id=363 removed by soft count = 11465\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 11\n",
      "--- gene=11, id=53 removed by soft count = 252\n",
      "--- gene=11, id=145 removed by soft count = 89\n",
      "--- gene=11, id=176 removed by soft count = 104\n",
      "--- gene=11, id=346 removed by soft count = 153\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 12\n",
      "--- gene=12, id=74 removed by soft count = 53\n",
      "--- gene=12, id=211 removed by soft count = 6681\n",
      "--- gene=12, id=217 removed by soft count = 197\n",
      "--- gene=12, id=218 removed by soft count = 215\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 13\n",
      "--- gene=13, id=196 removed by soft count = 286\n",
      "--- gene=13, id=197 removed by soft count = 61\n",
      "--- gene=13, id=303 removed by soft count = 227\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 14\n",
      "--- gene=14, id=298 removed by soft count = 192\n",
      "--- number of probes: 398 , kept by blast: 397 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 15\n",
      "--- gene=15, id=36 removed by soft count = 66\n",
      "--- gene=15, id=80 removed by soft count = 141\n",
      "--- gene=15, id=175 removed by soft count = 88\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 16\n",
      "--- gene=16, id=303 removed by soft count = 52\n",
      "--- gene=16, id=306 removed by soft count = 163\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 17\n",
      "--- gene=17, id=89 removed by soft count = 363\n",
      "--- gene=17, id=90 removed by soft count = 128\n",
      "--- gene=17, id=166 removed by soft count = 180\n",
      "--- gene=17, id=175 removed by soft count = 139\n",
      "--- gene=17, id=215 removed by soft count = 137\n",
      "--- gene=17, id=233 removed by soft count = 84\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 18\n",
      "--- gene=18, id=162 removed by soft count = 136\n",
      "--- gene=18, id=390 removed by soft count = 93\n",
      "--- gene=18, id=399 removed by soft count = 1849\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 19\n",
      "--- gene=19, id=206 removed by soft count = 110\n",
      "--- number of probes: 393 , kept by blast: 392 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 20\n",
      "--- gene=20, id=0 removed by soft count = 403\n",
      "--- gene=20, id=10 removed by soft count = 831\n",
      "--- gene=20, id=60 removed by soft count = 112\n",
      "--- gene=20, id=62 removed by soft count = 133\n",
      "--- gene=20, id=164 removed by soft count = 358\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 21\n",
      "--- gene=21, id=17 removed by soft count = 80\n",
      "--- gene=21, id=18 removed by soft count = 61\n",
      "--- gene=21, id=30 removed by soft count = 269\n",
      "--- gene=21, id=256 removed by soft count = 52\n",
      "--- gene=21, id=367 removed by soft count = 61\n",
      "--- gene=21, id=369 removed by soft count = 148\n",
      "--- gene=21, id=379 removed by soft count = 58\n",
      "--- gene=21, id=382 removed by soft count = 275\n",
      "--- gene=21, id=384 removed by soft count = 172\n",
      "--- gene=21, id=385 removed by soft count = 2085\n",
      "--- number of probes: 400 , kept by blast: 390 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 22\n",
      "--- gene=22, id=21 removed by soft count = 57\n",
      "--- gene=22, id=148 removed by soft count = 123\n",
      "--- gene=22, id=162 removed by soft count = 503\n",
      "--- gene=22, id=328 removed by soft count = 155\n",
      "--- gene=22, id=329 removed by soft count = 105\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 23\n",
      "--- gene=23, id=189 removed by soft count = 76\n",
      "--- gene=23, id=190 removed by soft count = 79\n",
      "--- gene=23, id=193 removed by soft count = 113\n",
      "--- gene=23, id=194 removed by soft count = 531\n",
      "--- gene=23, id=195 removed by soft count = 333\n",
      "--- gene=23, id=196 removed by soft count = 78\n",
      "--- gene=23, id=357 removed by soft count = 59\n",
      "--- gene=23, id=363 removed by soft count = 215\n",
      "--- gene=23, id=386 removed by soft count = 697\n",
      "--- gene=23, id=397 removed by soft count = 515\n",
      "--- gene=23, id=398 removed by soft count = 132\n",
      "--- gene=23, id=399 removed by soft count = 740\n",
      "--- number of probes: 400 , kept by blast: 388 , if remove dups: 388\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 24\n",
      "--- gene=24, id=29 removed by soft count = 80\n",
      "--- gene=24, id=85 removed by soft count = 937\n",
      "--- gene=24, id=232 removed by soft count = 211\n",
      "--- gene=24, id=235 removed by soft count = 181\n",
      "--- gene=24, id=236 removed by soft count = 64\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 25\n",
      "--- gene=25, id=257 removed by soft count = 53\n",
      "--- gene=25, id=300 removed by soft count = 66\n",
      "--- gene=25, id=399 removed by soft count = 74\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 26\n",
      "--- gene=26, id=354 removed by soft count = 409\n",
      "--- gene=26, id=378 removed by soft count = 95\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 27\n",
      "--- gene=27, id=17 removed by soft count = 213\n",
      "--- gene=27, id=19 removed by soft count = 107\n",
      "--- gene=27, id=24 removed by soft count = 85\n",
      "--- gene=27, id=28 removed by soft count = 136\n",
      "--- gene=27, id=30 removed by soft count = 94\n",
      "--- gene=27, id=50 removed by soft count = 8710\n",
      "--- gene=27, id=114 removed by soft count = 93\n",
      "--- gene=27, id=124 removed by soft count = 195\n",
      "--- gene=27, id=126 removed by soft count = 87\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=27, id=241 removed by soft count = 62\n",
      "--- gene=27, id=245 removed by soft count = 519\n",
      "--- gene=27, id=274 removed by soft count = 76\n",
      "--- gene=27, id=276 removed by soft count = 67\n",
      "--- gene=27, id=304 removed by soft count = 198\n",
      "--- gene=27, id=305 removed by soft count = 310\n",
      "--- gene=27, id=308 removed by soft count = 52\n",
      "--- gene=27, id=309 removed by soft count = 104\n",
      "--- number of probes: 400 , kept by blast: 383 , if remove dups: 383\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 28\n",
      "--- gene=28, id=0 removed by soft count = 219\n",
      "--- gene=28, id=191 removed by soft count = 299\n",
      "--- gene=28, id=228 removed by soft count = 56\n",
      "--- number of probes: 399 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 29\n",
      "--- gene=29, id=41 removed by soft count = 65\n",
      "--- gene=29, id=70 removed by soft count = 53\n",
      "--- gene=29, id=72 removed by soft count = 99\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 30\n",
      "--- gene=30, id=70 removed by soft count = 115\n",
      "--- gene=30, id=71 removed by soft count = 267\n",
      "--- gene=30, id=101 removed by soft count = 503\n",
      "--- gene=30, id=138 removed by soft count = 53\n",
      "--- gene=30, id=340 removed by soft count = 169\n",
      "--- gene=30, id=341 removed by soft count = 456\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 31\n",
      "--- gene=31, id=257 removed by soft count = 54\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 32\n",
      "--- gene=32, id=171 removed by soft count = 128\n",
      "--- gene=32, id=293 removed by soft count = 95\n",
      "--- gene=32, id=297 removed by soft count = 180\n",
      "--- gene=32, id=377 removed by soft count = 79\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 33\n",
      "--- gene=33, id=34 removed by soft count = 285\n",
      "--- gene=33, id=37 removed by soft count = 178\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 34\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 35\n",
      "--- gene=35, id=0 removed by soft count = 359\n",
      "--- gene=35, id=109 removed by soft count = 56\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 36\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 37\n",
      "--- gene=37, id=98 removed by soft count = 73\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 38\n",
      "--- gene=38, id=60 removed by soft count = 141\n",
      "--- gene=38, id=77 removed by soft count = 114\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 39\n",
      "--- gene=39, id=172 removed by soft count = 52\n",
      "--- gene=39, id=176 removed by soft count = 88\n",
      "--- gene=39, id=177 removed by soft count = 64\n",
      "--- gene=39, id=190 removed by soft count = 658\n",
      "--- gene=39, id=200 removed by soft count = 216\n",
      "--- gene=39, id=276 removed by soft count = 319\n",
      "--- gene=39, id=306 removed by soft count = 196\n",
      "--- gene=39, id=337 removed by soft count = 76\n",
      "--- number of probes: 400 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 40\n",
      "--- gene=40, id=103 removed by soft count = 322\n",
      "--- gene=40, id=138 removed by soft count = 75\n",
      "--- gene=40, id=153 removed by soft count = 154\n",
      "--- gene=40, id=172 removed by soft count = 61\n",
      "--- gene=40, id=223 removed by soft count = 109\n",
      "--- gene=40, id=237 removed by soft count = 126\n",
      "--- gene=40, id=298 removed by soft count = 85\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 41\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 42\n",
      "--- gene=42, id=83 removed by soft count = 174\n",
      "--- gene=42, id=113 removed by soft count = 79\n",
      "--- gene=42, id=116 removed by soft count = 53\n",
      "--- gene=42, id=117 removed by soft count = 89\n",
      "--- gene=42, id=356 removed by soft count = 100\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 43\n",
      "--- gene=43, id=28 removed by hard count = 5\n",
      "--- number of probes: 398 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 44\n",
      "--- gene=44, id=19 removed by soft count = 193\n",
      "--- gene=44, id=127 removed by soft count = 15314\n",
      "--- gene=44, id=128 removed by soft count = 289\n",
      "--- gene=44, id=171 removed by soft count = 65\n",
      "--- number of probes: 395 , kept by blast: 391 , if remove dups: 391\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 45\n",
      "--- number of probes: 399 , kept by blast: 399 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 46\n",
      "--- gene=46, id=33 removed by soft count = 55\n",
      "--- gene=46, id=38 removed by soft count = 107\n",
      "--- gene=46, id=44 removed by soft count = 60\n",
      "--- gene=46, id=45 removed by soft count = 79\n",
      "--- gene=46, id=65 removed by soft count = 568\n",
      "--- gene=46, id=72 removed by soft count = 196\n",
      "--- gene=46, id=322 removed by soft count = 51\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 47\n",
      "--- gene=47, id=200 removed by soft count = 284\n",
      "--- gene=47, id=306 removed by soft count = 164\n",
      "--- gene=47, id=336 removed by soft count = 85\n",
      "--- gene=47, id=337 removed by soft count = 5903\n",
      "--- gene=47, id=338 removed by soft count = 66\n",
      "--- gene=47, id=339 removed by soft count = 698\n",
      "--- gene=47, id=340 removed by soft count = 834\n",
      "--- gene=47, id=341 removed by soft count = 53\n",
      "--- gene=47, id=367 removed by soft count = 63\n",
      "--- number of probes: 400 , kept by blast: 391 , if remove dups: 391\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 48\n",
      "--- gene=48, id=41 removed by soft count = 77\n",
      "--- gene=48, id=141 removed by soft count = 76\n",
      "--- gene=48, id=169 removed by soft count = 71\n",
      "--- gene=48, id=200 removed by soft count = 89\n",
      "--- gene=48, id=251 removed by soft count = 86\n",
      "--- gene=48, id=311 removed by soft count = 279\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 49\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=49, id=17 removed by soft count = 56\n",
      "--- gene=49, id=18 removed by soft count = 118\n",
      "--- gene=49, id=99 removed by soft count = 143\n",
      "--- gene=49, id=108 removed by soft count = 107\n",
      "--- gene=49, id=206 removed by soft count = 801\n",
      "--- gene=49, id=264 removed by soft count = 107\n",
      "--- gene=49, id=392 removed by soft count = 159\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 50\n",
      "--- gene=50, id=18 removed by soft count = 146\n",
      "--- gene=50, id=51 removed by soft count = 155\n",
      "--- gene=50, id=82 removed by soft count = 171\n",
      "--- number of probes: 389 , kept by blast: 386 , if remove dups: 386\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 51\n",
      "--- gene=51, id=130 removed by soft count = 106\n",
      "--- gene=51, id=279 removed by soft count = 1472\n",
      "--- gene=51, id=335 removed by soft count = 103\n",
      "--- gene=51, id=399 removed by soft count = 478\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 52\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 53\n",
      "--- gene=53, id=179 removed by soft count = 1258\n",
      "--- gene=53, id=180 removed by soft count = 1888\n",
      "--- gene=53, id=197 removed by soft count = 58\n",
      "--- gene=53, id=243 removed by soft count = 209\n",
      "--- gene=53, id=250 removed by soft count = 800\n",
      "--- gene=53, id=264 removed by soft count = 60\n",
      "--- gene=53, id=319 removed by soft count = 77\n",
      "--- gene=53, id=320 removed by soft count = 1471\n",
      "--- gene=53, id=321 removed by soft count = 188\n",
      "--- gene=53, id=324 removed by soft count = 52\n",
      "--- gene=53, id=325 removed by soft count = 382\n",
      "--- number of probes: 400 , kept by blast: 389 , if remove dups: 389\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 54\n",
      "--- gene=54, id=10 removed by soft count = 65\n",
      "--- gene=54, id=31 removed by soft count = 51\n",
      "--- gene=54, id=178 removed by soft count = 298\n",
      "--- gene=54, id=196 removed by soft count = 257\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 55\n",
      "--- gene=55, id=318 removed by soft count = 79\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 56\n",
      "--- gene=56, id=244 removed by soft count = 75\n",
      "--- gene=56, id=329 removed by soft count = 121\n",
      "--- gene=56, id=358 removed by soft count = 473\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 57\n",
      "--- gene=57, id=138 removed by soft count = 51\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 58\n",
      "--- gene=58, id=11 removed by soft count = 152\n",
      "--- gene=58, id=32 removed by soft count = 139\n",
      "--- gene=58, id=117 removed by soft count = 393\n",
      "--- gene=58, id=118 removed by soft count = 232\n",
      "--- gene=58, id=202 removed by soft count = 61\n",
      "--- gene=58, id=205 removed by soft count = 112\n",
      "--- gene=58, id=207 removed by soft count = 124\n",
      "--- gene=58, id=216 removed by soft count = 165\n",
      "--- gene=58, id=217 removed by soft count = 533\n",
      "--- gene=58, id=219 removed by soft count = 67\n",
      "--- gene=58, id=248 removed by soft count = 146\n",
      "--- gene=58, id=250 removed by soft count = 111\n",
      "--- gene=58, id=256 removed by soft count = 122\n",
      "--- gene=58, id=257 removed by soft count = 142\n",
      "--- gene=58, id=337 removed by soft count = 420\n",
      "--- gene=58, id=338 removed by soft count = 3087\n",
      "--- number of probes: 400 , kept by blast: 384 , if remove dups: 384\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 59\n",
      "--- gene=59, id=205 removed by soft count = 219\n",
      "--- gene=59, id=256 removed by soft count = 106\n",
      "--- gene=59, id=259 removed by soft count = 178\n",
      "--- gene=59, id=263 removed by soft count = 1835\n",
      "--- gene=59, id=267 removed by soft count = 844\n",
      "--- gene=59, id=268 removed by soft count = 71\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 60\n",
      "--- gene=60, id=135 removed by soft count = 209\n",
      "--- gene=60, id=259 removed by soft count = 82\n",
      "--- gene=60, id=292 removed by soft count = 145\n",
      "--- gene=60, id=293 removed by soft count = 218\n",
      "--- gene=60, id=309 removed by soft count = 4382\n",
      "--- gene=60, id=324 removed by soft count = 88\n",
      "--- gene=60, id=325 removed by soft count = 163\n",
      "--- gene=60, id=332 removed by soft count = 60\n",
      "--- gene=60, id=372 removed by soft count = 2766\n",
      "--- gene=60, id=396 removed by soft count = 198\n",
      "--- number of probes: 400 , kept by blast: 390 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 61\n",
      "--- gene=61, id=135 removed by soft count = 213\n",
      "--- gene=61, id=358 removed by soft count = 139\n",
      "--- gene=61, id=359 removed by soft count = 160\n",
      "--- gene=61, id=360 removed by soft count = 146\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 62\n",
      "--- gene=62, id=198 removed by soft count = 352\n",
      "--- gene=62, id=204 removed by soft count = 119\n",
      "--- gene=62, id=224 removed by soft count = 345\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 63\n",
      "--- gene=63, id=52 removed by soft count = 3503\n",
      "--- gene=63, id=53 removed by soft count = 210\n",
      "--- gene=63, id=114 removed by soft count = 61\n",
      "--- gene=63, id=232 removed by soft count = 53\n",
      "--- gene=63, id=263 removed by soft count = 573\n",
      "--- gene=63, id=349 removed by soft count = 83\n",
      "--- gene=63, id=360 removed by soft count = 377\n",
      "--- gene=63, id=362 removed by soft count = 630\n",
      "--- gene=63, id=363 removed by soft count = 974\n",
      "--- number of probes: 400 , kept by blast: 391 , if remove dups: 391\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 64\n",
      "--- gene=64, id=160 removed by soft count = 241\n",
      "--- gene=64, id=258 removed by soft count = 436\n",
      "--- gene=64, id=259 removed by soft count = 369\n",
      "--- gene=64, id=266 removed by soft count = 25862\n",
      "--- gene=64, id=321 removed by soft count = 75\n",
      "--- gene=64, id=324 removed by soft count = 77\n",
      "--- gene=64, id=325 removed by soft count = 63\n",
      "--- gene=64, id=326 removed by soft count = 63\n",
      "--- gene=64, id=327 removed by soft count = 70\n",
      "--- gene=64, id=328 removed by soft count = 82\n",
      "--- gene=64, id=330 removed by soft count = 85\n",
      "--- gene=64, id=331 removed by soft count = 213\n",
      "--- gene=64, id=371 removed by soft count = 67\n",
      "--- number of probes: 400 , kept by blast: 387 , if remove dups: 387\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 65\n",
      "--- gene=65, id=4 removed by soft count = 810\n",
      "--- gene=65, id=5 removed by soft count = 721\n",
      "--- gene=65, id=8 removed by soft count = 390\n",
      "--- gene=65, id=11 removed by soft count = 677\n",
      "--- gene=65, id=14 removed by soft count = 156\n",
      "--- gene=65, id=54 removed by soft count = 61\n",
      "--- gene=65, id=73 removed by soft count = 224\n",
      "--- gene=65, id=159 removed by soft count = 54\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=65, id=266 removed by soft count = 156\n",
      "--- gene=65, id=268 removed by soft count = 184\n",
      "--- gene=65, id=269 removed by soft count = 278\n",
      "--- gene=65, id=276 removed by soft count = 55\n",
      "--- gene=65, id=278 removed by soft count = 330\n",
      "--- gene=65, id=279 removed by soft count = 2280\n",
      "--- gene=65, id=338 removed by soft count = 61\n",
      "--- gene=65, id=342 removed by soft count = 1355\n",
      "--- gene=65, id=344 removed by soft count = 70\n",
      "--- number of probes: 400 , kept by blast: 383 , if remove dups: 381\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 66\n",
      "--- gene=66, id=72 removed by soft count = 62\n",
      "--- gene=66, id=169 removed by soft count = 119\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 67\n",
      "--- gene=67, id=51 removed by soft count = 207\n",
      "--- gene=67, id=76 removed by soft count = 244\n",
      "--- gene=67, id=143 removed by soft count = 329\n",
      "--- gene=67, id=157 removed by soft count = 470\n",
      "--- gene=67, id=225 removed by soft count = 76\n",
      "--- gene=67, id=226 removed by soft count = 57\n",
      "--- gene=67, id=231 removed by soft count = 107\n",
      "--- gene=67, id=275 removed by soft count = 58\n",
      "--- gene=67, id=276 removed by soft count = 158\n",
      "--- gene=67, id=277 removed by soft count = 67\n",
      "--- gene=67, id=283 removed by soft count = 127\n",
      "--- gene=67, id=285 removed by soft count = 107\n",
      "--- gene=67, id=297 removed by soft count = 4541\n",
      "--- gene=67, id=319 removed by soft count = 571\n",
      "--- gene=67, id=324 removed by soft count = 98\n",
      "--- gene=67, id=373 removed by soft count = 144\n",
      "--- number of probes: 400 , kept by blast: 384 , if remove dups: 384\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 68\n",
      "--- gene=68, id=357 removed by soft count = 95\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 69\n",
      "--- gene=69, id=168 removed by soft count = 167\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 70\n",
      "--- gene=70, id=281 removed by soft count = 57\n",
      "--- number of probes: 398 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 71\n",
      "--- gene=71, id=111 removed by soft count = 52\n",
      "--- gene=71, id=255 removed by soft count = 56\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 72\n",
      "--- gene=72, id=225 removed by soft count = 171\n",
      "--- gene=72, id=227 removed by soft count = 98\n",
      "--- gene=72, id=231 removed by soft count = 121\n",
      "--- gene=72, id=232 removed by soft count = 138\n",
      "--- gene=72, id=233 removed by soft count = 173\n",
      "--- gene=72, id=243 removed by soft count = 69\n",
      "--- gene=72, id=244 removed by soft count = 75\n",
      "--- gene=72, id=245 removed by soft count = 119\n",
      "--- gene=72, id=367 removed by soft count = 53\n",
      "--- number of probes: 400 , kept by blast: 391 , if remove dups: 391\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 73\n",
      "--- gene=73, id=58 removed by soft count = 78\n",
      "--- gene=73, id=67 removed by soft count = 55\n",
      "--- gene=73, id=209 removed by soft count = 465\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 74\n",
      "--- gene=74, id=4 removed by soft count = 76\n",
      "--- gene=74, id=17 removed by soft count = 121\n",
      "--- gene=74, id=26 removed by soft count = 80\n",
      "--- gene=74, id=178 removed by soft count = 130\n",
      "--- gene=74, id=194 removed by soft count = 398\n",
      "--- gene=74, id=196 removed by soft count = 82\n",
      "--- gene=74, id=254 removed by soft count = 588\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 75\n",
      "--- gene=75, id=153 removed by soft count = 51\n",
      "--- gene=75, id=361 removed by soft count = 70\n",
      "--- gene=75, id=362 removed by soft count = 526\n",
      "--- number of probes: 396 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 76\n",
      "--- gene=76, id=187 removed by soft count = 219\n",
      "--- gene=76, id=215 removed by soft count = 58\n",
      "--- gene=76, id=371 removed by soft count = 70\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 77\n",
      "--- gene=77, id=67 removed by soft count = 344\n",
      "--- gene=77, id=68 removed by soft count = 183\n",
      "--- gene=77, id=181 removed by soft count = 83\n",
      "--- gene=77, id=182 removed by soft count = 76\n",
      "--- gene=77, id=388 removed by soft count = 196\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 78\n",
      "--- gene=78, id=220 removed by soft count = 53\n",
      "--- gene=78, id=318 removed by soft count = 91\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 79\n",
      "--- gene=79, id=80 removed by soft count = 83\n",
      "--- gene=79, id=81 removed by soft count = 57\n",
      "--- gene=79, id=139 removed by soft count = 58\n",
      "--- gene=79, id=358 removed by soft count = 66\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 80\n",
      "--- gene=80, id=23 removed by soft count = 1091\n",
      "--- gene=80, id=232 removed by soft count = 82\n",
      "--- gene=80, id=360 removed by soft count = 318\n",
      "--- gene=80, id=361 removed by soft count = 125\n",
      "--- gene=80, id=371 removed by soft count = 102\n",
      "--- gene=80, id=372 removed by soft count = 52\n",
      "--- gene=80, id=394 removed by soft count = 73\n",
      "--- gene=80, id=395 removed by soft count = 493\n",
      "--- number of probes: 400 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 81\n",
      "--- gene=81, id=103 removed by soft count = 127\n",
      "--- gene=81, id=196 removed by soft count = 11560\n",
      "--- gene=81, id=197 removed by soft count = 23179\n",
      "--- gene=81, id=198 removed by soft count = 1341\n",
      "--- gene=81, id=199 removed by soft count = 4028\n",
      "--- gene=81, id=314 removed by soft count = 76\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 82\n",
      "--- gene=82, id=20 removed by soft count = 87\n",
      "--- gene=82, id=48 removed by soft count = 1038\n",
      "--- gene=82, id=63 removed by soft count = 106\n",
      "--- gene=82, id=64 removed by soft count = 243\n",
      "--- gene=82, id=332 removed by soft count = 76\n",
      "--- gene=82, id=333 removed by soft count = 78\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 83\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 84\n",
      "--- gene=84, id=12 removed by soft count = 112\n",
      "--- gene=84, id=13 removed by soft count = 90\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=84, id=39 removed by soft count = 58\n",
      "--- gene=84, id=80 removed by soft count = 65\n",
      "--- gene=84, id=124 removed by soft count = 183\n",
      "--- gene=84, id=134 removed by soft count = 277\n",
      "--- gene=84, id=179 removed by soft count = 100\n",
      "--- gene=84, id=318 removed by soft count = 82\n",
      "--- gene=84, id=386 removed by soft count = 351\n",
      "--- gene=84, id=387 removed by soft count = 58\n",
      "--- gene=84, id=388 removed by soft count = 444\n",
      "--- gene=84, id=390 removed by soft count = 55\n",
      "--- gene=84, id=392 removed by soft count = 767\n",
      "--- gene=84, id=396 removed by soft count = 799\n",
      "--- gene=84, id=397 removed by soft count = 98\n",
      "--- number of probes: 400 , kept by blast: 385 , if remove dups: 383\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 85\n",
      "--- gene=85, id=263 removed by soft count = 74\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 86\n",
      "--- gene=86, id=0 removed by soft count = 188\n",
      "--- gene=86, id=2 removed by soft count = 7634\n",
      "--- gene=86, id=5 removed by soft count = 79\n",
      "--- gene=86, id=37 removed by soft count = 72\n",
      "--- gene=86, id=100 removed by soft count = 92\n",
      "--- gene=86, id=101 removed by soft count = 61\n",
      "--- gene=86, id=297 removed by soft count = 151\n",
      "--- gene=86, id=301 removed by soft count = 67\n",
      "--- number of probes: 400 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 87\n",
      "--- gene=87, id=265 removed by soft count = 63\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 88\n",
      "--- number of probes: 396 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 89\n",
      "--- gene=89, id=179 removed by soft count = 103\n",
      "--- gene=89, id=268 removed by soft count = 143\n",
      "--- gene=89, id=315 removed by soft count = 283\n",
      "--- gene=89, id=349 removed by soft count = 54\n",
      "--- number of probes: 396 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 90\n",
      "--- gene=90, id=63 removed by soft count = 342\n",
      "--- gene=90, id=382 removed by soft count = 151\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 91\n",
      "--- gene=91, id=0 removed by soft count = 655\n",
      "--- gene=91, id=189 removed by soft count = 148\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 92\n",
      "--- gene=92, id=1 removed by soft count = 95\n",
      "--- gene=92, id=27 removed by soft count = 58\n",
      "--- gene=92, id=58 removed by soft count = 52\n",
      "--- gene=92, id=112 removed by soft count = 216\n",
      "--- gene=92, id=126 removed by soft count = 148\n",
      "--- gene=92, id=129 removed by soft count = 71\n",
      "--- gene=92, id=130 removed by soft count = 315\n",
      "--- gene=92, id=144 removed by soft count = 232\n",
      "--- gene=92, id=365 removed by soft count = 352\n",
      "--- number of probes: 400 , kept by blast: 391 , if remove dups: 391\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 93\n",
      "--- gene=93, id=375 removed by soft count = 104\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 94\n",
      "--- gene=94, id=10 removed by soft count = 95\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 95\n",
      "--- gene=95, id=1 removed by soft count = 93\n",
      "--- gene=95, id=214 removed by soft count = 55\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 96\n",
      "--- gene=96, id=20 removed by soft count = 67\n",
      "--- gene=96, id=278 removed by soft count = 52\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 97\n",
      "--- gene=97, id=159 removed by soft count = 76\n",
      "--- gene=97, id=371 removed by soft count = 117\n",
      "--- gene=97, id=373 removed by soft count = 303\n",
      "--- gene=97, id=386 removed by soft count = 53\n",
      "--- gene=97, id=393 removed by soft count = 92\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 98\n",
      "--- gene=98, id=107 removed by soft count = 261\n",
      "--- gene=98, id=141 removed by soft count = 153\n",
      "--- gene=98, id=144 removed by soft count = 450\n",
      "--- gene=98, id=295 removed by soft count = 53\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 99\n",
      "--- gene=99, id=23 removed by soft count = 95\n",
      "--- gene=99, id=75 removed by soft count = 74\n",
      "--- gene=99, id=361 removed by soft count = 103\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 100\n",
      "--- gene=100, id=78 removed by soft count = 4718\n",
      "--- gene=100, id=79 removed by soft count = 226\n",
      "--- gene=100, id=80 removed by soft count = 410\n",
      "--- gene=100, id=82 removed by soft count = 64\n",
      "--- gene=100, id=96 removed by soft count = 74\n",
      "--- gene=100, id=98 removed by soft count = 65\n",
      "--- gene=100, id=116 removed by soft count = 96\n",
      "--- gene=100, id=164 removed by soft count = 487\n",
      "--- gene=100, id=294 removed by soft count = 109\n",
      "--- gene=100, id=328 removed by soft count = 178\n",
      "--- gene=100, id=329 removed by soft count = 6083\n",
      "--- gene=100, id=333 removed by soft count = 105\n",
      "--- gene=100, id=394 removed by soft count = 113\n",
      "--- number of probes: 400 , kept by blast: 387 , if remove dups: 385\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 101\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 102\n",
      "--- gene=102, id=7 removed by soft count = 377\n",
      "--- gene=102, id=241 removed by soft count = 69\n",
      "--- gene=102, id=269 removed by soft count = 254\n",
      "--- gene=102, id=305 removed by soft count = 335\n",
      "--- gene=102, id=332 removed by soft count = 103\n",
      "--- gene=102, id=342 removed by soft count = 64\n",
      "--- gene=102, id=375 removed by soft count = 204\n",
      "--- number of probes: 399 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 103\n",
      "--- gene=103, id=53 removed by soft count = 473\n",
      "--- gene=103, id=54 removed by soft count = 96\n",
      "--- gene=103, id=169 removed by soft count = 508\n",
      "--- gene=103, id=170 removed by soft count = 96\n",
      "--- gene=103, id=225 removed by soft count = 109\n",
      "--- gene=103, id=226 removed by soft count = 626\n",
      "--- gene=103, id=259 removed by soft count = 293\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 104\n",
      "--- gene=104, id=181 removed by soft count = 240\n",
      "--- gene=104, id=182 removed by soft count = 1002\n",
      "--- gene=104, id=183 removed by soft count = 144\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=104, id=184 removed by soft count = 443\n",
      "--- gene=104, id=242 removed by soft count = 259\n",
      "--- gene=104, id=268 removed by soft count = 114\n",
      "--- gene=104, id=298 removed by soft count = 69\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 105\n",
      "--- gene=105, id=133 removed by soft count = 583\n",
      "--- gene=105, id=134 removed by soft count = 146\n",
      "--- gene=105, id=375 removed by soft count = 65\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 106\n",
      "--- gene=106, id=46 removed by soft count = 101\n",
      "--- gene=106, id=143 removed by soft count = 69\n",
      "--- gene=106, id=145 removed by soft count = 70\n",
      "--- gene=106, id=146 removed by soft count = 84\n",
      "--- gene=106, id=160 removed by soft count = 59\n",
      "--- gene=106, id=161 removed by soft count = 152\n",
      "--- gene=106, id=163 removed by soft count = 60\n",
      "--- gene=106, id=164 removed by soft count = 460\n",
      "--- gene=106, id=165 removed by soft count = 61\n",
      "--- gene=106, id=236 removed by soft count = 88\n",
      "--- gene=106, id=237 removed by soft count = 75\n",
      "--- gene=106, id=248 removed by soft count = 222\n",
      "--- gene=106, id=280 removed by soft count = 225\n",
      "--- gene=106, id=281 removed by soft count = 201\n",
      "--- gene=106, id=282 removed by soft count = 81\n",
      "--- gene=106, id=361 removed by soft count = 123\n",
      "--- gene=106, id=364 removed by soft count = 72\n",
      "--- number of probes: 400 , kept by blast: 383 , if remove dups: 383\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 107\n",
      "--- gene=107, id=24 removed by soft count = 159\n",
      "--- gene=107, id=75 removed by soft count = 72\n",
      "--- gene=107, id=102 removed by soft count = 126\n",
      "--- gene=107, id=103 removed by soft count = 290\n",
      "--- gene=107, id=113 removed by soft count = 371\n",
      "--- gene=107, id=115 removed by soft count = 58\n",
      "--- gene=107, id=129 removed by soft count = 101\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 108\n",
      "--- gene=108, id=1 removed by soft count = 275\n",
      "--- gene=108, id=128 removed by soft count = 90\n",
      "--- gene=108, id=336 removed by soft count = 63\n",
      "--- gene=108, id=347 removed by soft count = 157\n",
      "--- gene=108, id=385 removed by soft count = 73\n",
      "--- gene=108, id=387 removed by soft count = 59\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 109\n",
      "--- gene=109, id=1 removed by soft count = 95\n",
      "--- gene=109, id=383 removed by soft count = 72\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 111\n",
      "--- gene=111, id=146 removed by soft count = 57\n",
      "--- gene=111, id=220 removed by soft count = 99\n",
      "--- gene=111, id=394 removed by soft count = 1174\n",
      "--- gene=111, id=397 removed by soft count = 119\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 112\n",
      "--- gene=112, id=146 removed by soft count = 5606\n",
      "--- gene=112, id=198 removed by soft count = 59\n",
      "--- gene=112, id=209 removed by soft count = 273\n",
      "--- gene=112, id=272 removed by soft count = 232\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 113\n",
      "--- gene=113, id=129 removed by soft count = 6416\n",
      "--- gene=113, id=160 removed by soft count = 205\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 114\n",
      "--- gene=114, id=14 removed by soft count = 537\n",
      "--- gene=114, id=15 removed by soft count = 81\n",
      "--- gene=114, id=68 removed by soft count = 111\n",
      "--- gene=114, id=158 removed by soft count = 101\n",
      "--- gene=114, id=159 removed by soft count = 87\n",
      "--- gene=114, id=160 removed by soft count = 55\n",
      "--- gene=114, id=181 removed by soft count = 183\n",
      "--- gene=114, id=372 removed by soft count = 59\n",
      "--- gene=114, id=373 removed by soft count = 178\n",
      "--- gene=114, id=380 removed by soft count = 149\n",
      "--- gene=114, id=390 removed by soft count = 404\n",
      "--- number of probes: 400 , kept by blast: 389 , if remove dups: 389\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 115\n",
      "--- gene=115, id=33 removed by hard count = 4\n",
      "--- gene=115, id=35 removed by hard count = 9\n",
      "--- gene=115, id=110 removed by soft count = 216\n",
      "--- gene=115, id=318 removed by soft count = 326\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 116\n",
      "--- gene=116, id=13 removed by soft count = 71\n",
      "--- gene=116, id=137 removed by soft count = 67\n",
      "--- gene=116, id=139 removed by soft count = 106\n",
      "--- gene=116, id=261 removed by soft count = 71\n",
      "--- gene=116, id=294 removed by soft count = 169\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 117\n",
      "--- gene=117, id=45 removed by soft count = 65\n",
      "--- gene=117, id=208 removed by soft count = 56\n",
      "--- gene=117, id=320 removed by soft count = 1042\n",
      "--- gene=117, id=334 removed by soft count = 68\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 118\n",
      "--- gene=118, id=84 removed by soft count = 52\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 119\n",
      "--- gene=119, id=0 removed by soft count = 300\n",
      "--- gene=119, id=43 removed by soft count = 135\n",
      "--- gene=119, id=269 removed by soft count = 67\n",
      "--- gene=119, id=272 removed by soft count = 147\n",
      "--- gene=119, id=290 removed by soft count = 83\n",
      "--- gene=119, id=293 removed by soft count = 67\n",
      "--- gene=119, id=373 removed by soft count = 86\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 120\n",
      "--- gene=120, id=95 removed by soft count = 70\n",
      "--- gene=120, id=99 removed by soft count = 238\n",
      "--- gene=120, id=103 removed by soft count = 400\n",
      "--- gene=120, id=109 removed by soft count = 390\n",
      "--- gene=120, id=128 removed by soft count = 516\n",
      "--- gene=120, id=129 removed by soft count = 153\n",
      "--- gene=120, id=131 removed by soft count = 84\n",
      "--- gene=120, id=132 removed by soft count = 305\n",
      "--- gene=120, id=221 removed by soft count = 121\n",
      "--- gene=120, id=222 removed by soft count = 98\n",
      "--- gene=120, id=268 removed by soft count = 136\n",
      "--- gene=120, id=270 removed by soft count = 111\n",
      "--- gene=120, id=273 removed by soft count = 366\n",
      "--- gene=120, id=274 removed by soft count = 242\n",
      "--- gene=120, id=362 removed by soft count = 1570\n",
      "--- gene=120, id=363 removed by soft count = 369\n",
      "--- gene=120, id=364 removed by soft count = 184\n",
      "--- gene=120, id=365 removed by soft count = 121\n",
      "--- gene=120, id=398 removed by soft count = 296\n",
      "--- number of probes: 400 , kept by blast: 381 , if remove dups: 381\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 121\n",
      "--- gene=121, id=36 removed by soft count = 60\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=121, id=138 removed by soft count = 142\n",
      "--- gene=121, id=139 removed by soft count = 134\n",
      "--- gene=121, id=143 removed by soft count = 131\n",
      "--- gene=121, id=145 removed by soft count = 737\n",
      "--- gene=121, id=146 removed by soft count = 537\n",
      "--- gene=121, id=147 removed by soft count = 219\n",
      "--- gene=121, id=152 removed by soft count = 65\n",
      "--- gene=121, id=344 removed by soft count = 250\n",
      "--- number of probes: 400 , kept by blast: 391 , if remove dups: 391\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 122\n",
      "--- gene=122, id=247 removed by soft count = 242\n",
      "--- gene=122, id=386 removed by soft count = 65\n",
      "--- number of probes: 399 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 123\n",
      "--- gene=123, id=73 removed by soft count = 318\n",
      "--- gene=123, id=78 removed by soft count = 103\n",
      "--- gene=123, id=134 removed by soft count = 56\n",
      "--- gene=123, id=297 removed by soft count = 162\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 124\n",
      "--- gene=124, id=43 removed by soft count = 116\n",
      "--- gene=124, id=45 removed by soft count = 167\n",
      "--- gene=124, id=59 removed by soft count = 51\n",
      "--- gene=124, id=83 removed by soft count = 236\n",
      "--- gene=124, id=90 removed by soft count = 209\n",
      "--- gene=124, id=165 removed by soft count = 60\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 125\n",
      "--- gene=125, id=0 removed by soft count = 738\n",
      "--- gene=125, id=2 removed by soft count = 72\n",
      "--- gene=125, id=52 removed by soft count = 786\n",
      "--- gene=125, id=57 removed by soft count = 731\n",
      "--- gene=125, id=58 removed by soft count = 771\n",
      "--- gene=125, id=61 removed by soft count = 822\n",
      "--- gene=125, id=62 removed by soft count = 419\n",
      "--- gene=125, id=64 removed by soft count = 818\n",
      "--- gene=125, id=71 removed by soft count = 79\n",
      "--- gene=125, id=75 removed by soft count = 815\n",
      "--- gene=125, id=85 removed by soft count = 779\n",
      "--- gene=125, id=88 removed by soft count = 875\n",
      "--- gene=125, id=89 removed by soft count = 927\n",
      "--- gene=125, id=101 removed by soft count = 86\n",
      "--- gene=125, id=152 removed by soft count = 67\n",
      "--- gene=125, id=157 removed by soft count = 876\n",
      "--- gene=125, id=266 removed by soft count = 65\n",
      "--- gene=125, id=326 removed by soft count = 227\n",
      "--- gene=125, id=355 removed by soft count = 74\n",
      "--- gene=125, id=364 removed by soft count = 954\n",
      "--- gene=125, id=369 removed by soft count = 56\n",
      "--- gene=125, id=373 removed by soft count = 78\n",
      "--- gene=125, id=381 removed by soft count = 101\n",
      "--- gene=125, id=384 removed by soft count = 110\n",
      "--- gene=125, id=387 removed by soft count = 254\n",
      "--- number of probes: 400 , kept by blast: 375 , if remove dups: 375\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 126\n",
      "--- gene=126, id=2 removed by soft count = 76\n",
      "--- gene=126, id=4 removed by soft count = 59\n",
      "--- gene=126, id=54 removed by soft count = 68\n",
      "--- gene=126, id=126 removed by soft count = 59\n",
      "--- gene=126, id=179 removed by soft count = 173\n",
      "--- gene=126, id=185 removed by soft count = 53\n",
      "--- gene=126, id=227 removed by soft count = 2623\n",
      "--- gene=126, id=267 removed by soft count = 1910\n",
      "--- gene=126, id=273 removed by soft count = 167\n",
      "--- gene=126, id=326 removed by soft count = 335\n",
      "--- number of probes: 400 , kept by blast: 390 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 127\n",
      "--- gene=127, id=67 removed by soft count = 231\n",
      "--- gene=127, id=99 removed by soft count = 89\n",
      "--- gene=127, id=158 removed by soft count = 149\n",
      "--- gene=127, id=190 removed by soft count = 82\n",
      "--- gene=127, id=208 removed by soft count = 157\n",
      "--- gene=127, id=350 removed by soft count = 54\n",
      "--- gene=127, id=352 removed by soft count = 95\n",
      "--- gene=127, id=353 removed by soft count = 192\n",
      "--- number of probes: 398 , kept by blast: 390 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 129\n",
      "--- gene=129, id=391 removed by soft count = 107\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 130\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 131\n",
      "--- gene=131, id=257 removed by soft count = 260\n",
      "--- gene=131, id=319 removed by hard count = 5\n",
      "--- number of probes: 396 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 132\n",
      "--- gene=132, id=88 removed by soft count = 97\n",
      "--- gene=132, id=121 removed by soft count = 81\n",
      "--- gene=132, id=181 removed by soft count = 391\n",
      "--- gene=132, id=196 removed by soft count = 699\n",
      "--- gene=132, id=198 removed by soft count = 178\n",
      "--- gene=132, id=217 removed by soft count = 109\n",
      "--- gene=132, id=241 removed by soft count = 120\n",
      "--- gene=132, id=256 removed by soft count = 128\n",
      "--- gene=132, id=257 removed by soft count = 2062\n",
      "--- gene=132, id=374 removed by soft count = 825\n",
      "--- number of probes: 400 , kept by blast: 390 , if remove dups: 389\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 133\n",
      "--- gene=133, id=24 removed by soft count = 187\n",
      "--- gene=133, id=41 removed by soft count = 63\n",
      "--- gene=133, id=44 removed by soft count = 66\n",
      "--- gene=133, id=46 removed by soft count = 77\n",
      "--- gene=133, id=53 removed by soft count = 93\n",
      "--- gene=133, id=54 removed by soft count = 568\n",
      "--- gene=133, id=59 removed by soft count = 60\n",
      "--- gene=133, id=247 removed by soft count = 143\n",
      "--- gene=133, id=248 removed by soft count = 162\n",
      "--- gene=133, id=287 removed by soft count = 147\n",
      "--- number of probes: 400 , kept by blast: 390 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 134\n",
      "--- gene=134, id=178 removed by soft count = 110\n",
      "--- gene=134, id=187 removed by soft count = 61\n",
      "--- gene=134, id=214 removed by soft count = 65\n",
      "--- gene=134, id=255 removed by soft count = 96\n",
      "--- gene=134, id=276 removed by soft count = 54\n",
      "--- gene=134, id=304 removed by soft count = 85\n",
      "--- gene=134, id=385 removed by soft count = 1775\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 135\n",
      "--- number of probes: 399 , kept by blast: 399 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 136\n",
      "--- gene=136, id=261 removed by soft count = 148\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 137\n",
      "--- gene=137, id=99 removed by soft count = 85\n",
      "--- gene=137, id=207 removed by soft count = 697\n",
      "--- gene=137, id=234 removed by soft count = 75\n",
      "--- gene=137, id=333 removed by soft count = 2082\n",
      "--- gene=137, id=334 removed by soft count = 85\n",
      "--- gene=137, id=335 removed by soft count = 69\n",
      "--- gene=137, id=337 removed by soft count = 88\n",
      "--- gene=137, id=338 removed by soft count = 3120\n",
      "--- gene=137, id=340 removed by soft count = 155\n",
      "--- number of probes: 400 , kept by blast: 391 , if remove dups: 391\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 138\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=138, id=0 removed by soft count = 51\n",
      "--- gene=138, id=57 removed by soft count = 114\n",
      "--- gene=138, id=191 removed by soft count = 604\n",
      "--- gene=138, id=203 removed by soft count = 178\n",
      "--- gene=138, id=250 removed by soft count = 61\n",
      "--- gene=138, id=257 removed by soft count = 209\n",
      "--- number of probes: 399 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 139\n",
      "--- gene=139, id=53 removed by soft count = 77\n",
      "--- gene=139, id=208 removed by soft count = 129\n",
      "--- gene=139, id=211 removed by soft count = 198\n",
      "--- gene=139, id=212 removed by soft count = 484\n",
      "--- gene=139, id=356 removed by soft count = 107\n",
      "--- gene=139, id=357 removed by soft count = 90\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 140\n",
      "--- gene=140, id=25 removed by soft count = 221\n",
      "--- gene=140, id=43 removed by hard count = 3\n",
      "--- gene=140, id=282 removed by soft count = 125\n",
      "--- gene=140, id=284 removed by soft count = 271\n",
      "--- gene=140, id=371 removed by soft count = 71\n",
      "--- number of probes: 398 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 141\n",
      "--- gene=141, id=81 removed by soft count = 186\n",
      "--- gene=141, id=82 removed by soft count = 85\n",
      "--- gene=141, id=283 removed by soft count = 53\n",
      "--- gene=141, id=364 removed by soft count = 16494\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 142\n",
      "--- gene=142, id=0 removed by soft count = 74\n",
      "--- gene=142, id=27 removed by soft count = 121\n",
      "--- gene=142, id=44 removed by soft count = 319\n",
      "--- gene=142, id=47 removed by soft count = 55\n",
      "--- gene=142, id=48 removed by soft count = 124\n",
      "--- gene=142, id=82 removed by soft count = 80\n",
      "--- gene=142, id=83 removed by soft count = 175\n",
      "--- gene=142, id=120 removed by soft count = 183\n",
      "--- gene=142, id=140 removed by soft count = 818\n",
      "--- gene=142, id=142 removed by soft count = 397\n",
      "--- gene=142, id=143 removed by soft count = 88\n",
      "--- gene=142, id=185 removed by soft count = 196\n",
      "--- gene=142, id=314 removed by soft count = 71\n",
      "--- gene=142, id=350 removed by soft count = 53\n",
      "--- number of probes: 400 , kept by blast: 386 , if remove dups: 386\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 143\n",
      "--- gene=143, id=42 removed by soft count = 112\n",
      "--- gene=143, id=45 removed by soft count = 129\n",
      "--- gene=143, id=149 removed by soft count = 74\n",
      "--- gene=143, id=200 removed by soft count = 74\n",
      "--- gene=143, id=275 removed by soft count = 320\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 144\n",
      "--- gene=144, id=76 removed by soft count = 272\n",
      "--- gene=144, id=78 removed by soft count = 194\n",
      "--- gene=144, id=222 removed by soft count = 80\n",
      "--- gene=144, id=226 removed by soft count = 81\n",
      "--- gene=144, id=325 removed by soft count = 257\n",
      "--- gene=144, id=334 removed by soft count = 150\n",
      "--- gene=144, id=397 removed by soft count = 232\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 145\n",
      "--- gene=145, id=5 removed by soft count = 328\n",
      "--- gene=145, id=8 removed by soft count = 156\n",
      "--- gene=145, id=10 removed by soft count = 144\n",
      "--- gene=145, id=28 removed by soft count = 347\n",
      "--- gene=145, id=30 removed by soft count = 258\n",
      "--- gene=145, id=56 removed by soft count = 101\n",
      "--- gene=145, id=186 removed by soft count = 163\n",
      "--- gene=145, id=231 removed by soft count = 5058\n",
      "--- gene=145, id=232 removed by soft count = 581\n",
      "--- gene=145, id=341 removed by soft count = 211\n",
      "--- number of probes: 400 , kept by blast: 390 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 146\n",
      "--- gene=146, id=17 removed by soft count = 64\n",
      "--- gene=146, id=167 removed by soft count = 56\n",
      "--- gene=146, id=242 removed by soft count = 609\n",
      "--- gene=146, id=300 removed by soft count = 569\n",
      "--- gene=146, id=328 removed by soft count = 70\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 147\n",
      "--- gene=147, id=43 removed by soft count = 72\n",
      "--- gene=147, id=55 removed by soft count = 129\n",
      "--- gene=147, id=138 removed by soft count = 170\n",
      "--- gene=147, id=181 removed by soft count = 64\n",
      "--- gene=147, id=182 removed by soft count = 129\n",
      "--- gene=147, id=184 removed by soft count = 77\n",
      "--- gene=147, id=211 removed by soft count = 61\n",
      "--- gene=147, id=213 removed by soft count = 140\n",
      "--- number of probes: 400 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 148\n",
      "--- gene=148, id=216 removed by soft count = 167\n",
      "--- gene=148, id=261 removed by soft count = 450\n",
      "--- gene=148, id=329 removed by soft count = 120\n",
      "--- gene=148, id=330 removed by soft count = 86\n",
      "--- gene=148, id=334 removed by soft count = 52\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 149\n",
      "--- gene=149, id=81 removed by soft count = 156\n",
      "--- gene=149, id=114 removed by soft count = 172\n",
      "--- gene=149, id=116 removed by soft count = 109\n",
      "--- gene=149, id=181 removed by soft count = 52\n",
      "--- gene=149, id=200 removed by soft count = 96\n",
      "--- gene=149, id=201 removed by soft count = 61\n",
      "--- gene=149, id=279 removed by soft count = 182\n",
      "--- gene=149, id=363 removed by soft count = 187\n",
      "--- number of probes: 384 , kept by blast: 376 , if remove dups: 376\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 150\n",
      "--- gene=150, id=49 removed by soft count = 105\n",
      "--- gene=150, id=198 removed by soft count = 75\n",
      "--- gene=150, id=199 removed by soft count = 295\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 151\n",
      "--- gene=151, id=348 removed by soft count = 79\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 152\n",
      "--- gene=152, id=0 removed by soft count = 249\n",
      "--- gene=152, id=10 removed by soft count = 62\n",
      "--- gene=152, id=11 removed by soft count = 75\n",
      "--- gene=152, id=110 removed by soft count = 82\n",
      "--- gene=152, id=111 removed by soft count = 557\n",
      "--- gene=152, id=119 removed by soft count = 83\n",
      "--- gene=152, id=169 removed by soft count = 59\n",
      "--- gene=152, id=240 removed by soft count = 58\n",
      "--- gene=152, id=241 removed by soft count = 145\n",
      "--- gene=152, id=242 removed by soft count = 75\n",
      "--- gene=152, id=395 removed by soft count = 76\n",
      "--- gene=152, id=399 removed by soft count = 79\n",
      "--- number of probes: 400 , kept by blast: 388 , if remove dups: 388\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 153\n",
      "--- gene=153, id=116 removed by soft count = 65\n",
      "--- gene=153, id=178 removed by soft count = 81\n",
      "--- gene=153, id=280 removed by hard count = 3\n",
      "--- gene=153, id=335 removed by soft count = 708\n",
      "--- gene=153, id=339 removed by soft count = 94\n",
      "--- gene=153, id=342 removed by soft count = 189\n",
      "--- gene=153, id=388 removed by soft count = 62\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 154\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=154, id=27 removed by soft count = 75\n",
      "--- gene=154, id=102 removed by soft count = 58\n",
      "--- gene=154, id=104 removed by soft count = 55\n",
      "--- gene=154, id=188 removed by soft count = 181\n",
      "--- gene=154, id=249 removed by soft count = 122\n",
      "--- gene=154, id=291 removed by soft count = 65\n",
      "--- gene=154, id=359 removed by soft count = 77\n",
      "--- number of probes: 394 , kept by blast: 387 , if remove dups: 387\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 155\n",
      "--- gene=155, id=92 removed by soft count = 61\n",
      "--- gene=155, id=141 removed by soft count = 64\n",
      "--- gene=155, id=143 removed by soft count = 325\n",
      "--- gene=155, id=219 removed by soft count = 151\n",
      "--- gene=155, id=220 removed by soft count = 101\n",
      "--- gene=155, id=244 removed by soft count = 103\n",
      "--- gene=155, id=287 removed by soft count = 82\n",
      "--- gene=155, id=288 removed by soft count = 55\n",
      "--- gene=155, id=385 removed by soft count = 500\n",
      "--- gene=155, id=397 removed by soft count = 358\n",
      "--- gene=155, id=398 removed by soft count = 119\n",
      "--- number of probes: 400 , kept by blast: 389 , if remove dups: 388\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 156\n",
      "--- gene=156, id=160 removed by soft count = 1686\n",
      "--- gene=156, id=187 removed by soft count = 111\n",
      "--- gene=156, id=290 removed by soft count = 55\n",
      "--- gene=156, id=377 removed by soft count = 182\n",
      "--- gene=156, id=379 removed by soft count = 265\n",
      "--- gene=156, id=382 removed by soft count = 753\n",
      "--- gene=156, id=398 removed by soft count = 190\n",
      "--- gene=156, id=399 removed by soft count = 1281\n",
      "--- number of probes: 400 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 157\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 158\n",
      "--- gene=158, id=135 removed by soft count = 57\n",
      "--- gene=158, id=271 removed by soft count = 147\n",
      "--- gene=158, id=286 removed by soft count = 138\n",
      "--- gene=158, id=381 removed by soft count = 170\n",
      "--- gene=158, id=383 removed by soft count = 78\n",
      "--- gene=158, id=394 removed by soft count = 368\n",
      "--- number of probes: 398 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 159\n",
      "--- gene=159, id=33 removed by soft count = 118\n",
      "--- gene=159, id=109 removed by hard count = 9\n",
      "--- gene=159, id=193 removed by soft count = 56\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 160\n",
      "--- gene=160, id=185 removed by soft count = 75\n",
      "--- gene=160, id=355 removed by soft count = 58\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 161\n",
      "--- gene=161, id=6 removed by soft count = 157\n",
      "--- gene=161, id=149 removed by soft count = 201\n",
      "--- gene=161, id=151 removed by soft count = 220\n",
      "--- gene=161, id=157 removed by soft count = 82\n",
      "--- gene=161, id=191 removed by soft count = 97\n",
      "--- gene=161, id=197 removed by soft count = 78\n",
      "--- gene=161, id=222 removed by soft count = 163\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 162\n",
      "--- gene=162, id=0 removed by soft count = 152\n",
      "--- gene=162, id=42 removed by soft count = 138\n",
      "--- gene=162, id=45 removed by soft count = 53\n",
      "--- gene=162, id=46 removed by soft count = 103\n",
      "--- gene=162, id=104 removed by soft count = 169\n",
      "--- gene=162, id=122 removed by soft count = 273\n",
      "--- gene=162, id=167 removed by soft count = 523\n",
      "--- gene=162, id=263 removed by soft count = 125\n",
      "--- gene=162, id=266 removed by soft count = 450\n",
      "--- gene=162, id=267 removed by soft count = 70\n",
      "--- gene=162, id=369 removed by soft count = 63\n",
      "--- gene=162, id=392 removed by soft count = 103\n",
      "--- gene=162, id=393 removed by soft count = 234\n",
      "--- gene=162, id=399 removed by soft count = 3669\n",
      "--- number of probes: 400 , kept by blast: 386 , if remove dups: 386\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 163\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 164\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 165\n",
      "--- gene=165, id=223 removed by soft count = 200\n",
      "--- gene=165, id=239 removed by soft count = 88\n",
      "--- gene=165, id=292 removed by soft count = 55\n",
      "--- gene=165, id=382 removed by soft count = 133\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 166\n",
      "--- gene=166, id=26 removed by soft count = 71\n",
      "--- gene=166, id=30 removed by soft count = 91\n",
      "--- gene=166, id=33 removed by soft count = 63\n",
      "--- number of probes: 398 , kept by blast: 395 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 168\n",
      "--- gene=168, id=164 removed by soft count = 51\n",
      "--- gene=168, id=174 removed by soft count = 112\n",
      "--- gene=168, id=326 removed by soft count = 63\n",
      "--- gene=168, id=364 removed by soft count = 135\n",
      "--- gene=168, id=366 removed by soft count = 125\n",
      "--- gene=168, id=368 removed by soft count = 64\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 169\n",
      "--- gene=169, id=17 removed by soft count = 986\n",
      "--- gene=169, id=18 removed by soft count = 119\n",
      "--- gene=169, id=19 removed by soft count = 98\n",
      "--- gene=169, id=112 removed by soft count = 53\n",
      "--- gene=169, id=126 removed by soft count = 69\n",
      "--- gene=169, id=139 removed by soft count = 177\n",
      "--- gene=169, id=194 removed by soft count = 74\n",
      "--- gene=169, id=252 removed by soft count = 53\n",
      "--- gene=169, id=259 removed by soft count = 62\n",
      "--- gene=169, id=363 removed by soft count = 100\n",
      "--- number of probes: 400 , kept by blast: 390 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 170\n",
      "--- gene=170, id=17 removed by soft count = 107\n",
      "--- gene=170, id=19 removed by soft count = 347\n",
      "--- gene=170, id=33 removed by soft count = 141\n",
      "--- gene=170, id=238 removed by soft count = 178\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 171\n",
      "--- gene=171, id=0 removed by soft count = 1007\n",
      "--- gene=171, id=173 removed by soft count = 87\n",
      "--- gene=171, id=228 removed by soft count = 155\n",
      "--- gene=171, id=320 removed by soft count = 555\n",
      "--- gene=171, id=332 removed by soft count = 51\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 172\n",
      "--- gene=172, id=17 removed by soft count = 252\n",
      "--- gene=172, id=18 removed by soft count = 96\n",
      "--- gene=172, id=62 removed by soft count = 56\n",
      "--- gene=172, id=64 removed by soft count = 113\n",
      "--- gene=172, id=68 removed by soft count = 98\n",
      "--- gene=172, id=112 removed by soft count = 72\n",
      "--- gene=172, id=125 removed by soft count = 3134\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=172, id=199 removed by soft count = 345\n",
      "--- gene=172, id=270 removed by soft count = 114\n",
      "--- gene=172, id=302 removed by soft count = 245\n",
      "--- gene=172, id=310 removed by soft count = 135\n",
      "--- gene=172, id=311 removed by soft count = 92\n",
      "--- gene=172, id=312 removed by soft count = 424\n",
      "--- number of probes: 400 , kept by blast: 387 , if remove dups: 387\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 173\n",
      "--- gene=173, id=5 removed by soft count = 137\n",
      "--- gene=173, id=6 removed by soft count = 126\n",
      "--- gene=173, id=8 removed by soft count = 117\n",
      "--- gene=173, id=123 removed by soft count = 162\n",
      "--- gene=173, id=256 removed by soft count = 242\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 174\n",
      "--- gene=174, id=296 removed by soft count = 386\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 175\n",
      "--- gene=175, id=36 removed by soft count = 63\n",
      "--- gene=175, id=120 removed by soft count = 158\n",
      "--- gene=175, id=121 removed by soft count = 77\n",
      "--- gene=175, id=281 removed by soft count = 98\n",
      "--- gene=175, id=284 removed by soft count = 51\n",
      "--- gene=175, id=287 removed by soft count = 73\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 176\n",
      "--- gene=176, id=70 removed by soft count = 160\n",
      "--- gene=176, id=148 removed by soft count = 574\n",
      "--- gene=176, id=151 removed by soft count = 249\n",
      "--- gene=176, id=152 removed by soft count = 1426\n",
      "--- gene=176, id=157 removed by soft count = 59\n",
      "--- gene=176, id=159 removed by soft count = 112\n",
      "--- gene=176, id=160 removed by soft count = 63\n",
      "--- gene=176, id=164 removed by soft count = 105\n",
      "--- gene=176, id=169 removed by soft count = 126\n",
      "--- gene=176, id=172 removed by soft count = 125\n",
      "--- gene=176, id=174 removed by soft count = 53\n",
      "--- gene=176, id=175 removed by soft count = 84\n",
      "--- gene=176, id=299 removed by soft count = 57\n",
      "--- gene=176, id=301 removed by soft count = 73\n",
      "--- gene=176, id=354 removed by soft count = 77\n",
      "--- gene=176, id=359 removed by soft count = 123\n",
      "--- gene=176, id=377 removed by soft count = 124\n",
      "--- gene=176, id=379 removed by soft count = 55\n",
      "--- gene=176, id=383 removed by soft count = 552\n",
      "--- gene=176, id=395 removed by soft count = 254\n",
      "--- number of probes: 400 , kept by blast: 380 , if remove dups: 380\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 177\n",
      "--- gene=177, id=135 removed by soft count = 124\n",
      "--- gene=177, id=226 removed by soft count = 62\n",
      "--- gene=177, id=230 removed by soft count = 60\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 178\n",
      "--- gene=178, id=360 removed by soft count = 295\n",
      "--- gene=178, id=361 removed by soft count = 105\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 179\n",
      "--- gene=179, id=105 removed by soft count = 97\n",
      "--- gene=179, id=107 removed by soft count = 77\n",
      "--- gene=179, id=246 removed by soft count = 108\n",
      "--- gene=179, id=247 removed by soft count = 91\n",
      "--- gene=179, id=291 removed by soft count = 128\n",
      "--- gene=179, id=369 removed by soft count = 287\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 180\n",
      "--- gene=180, id=83 removed by soft count = 168\n",
      "--- gene=180, id=245 removed by soft count = 94\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 181\n",
      "--- gene=181, id=27 removed by soft count = 215\n",
      "--- gene=181, id=138 removed by soft count = 59\n",
      "--- gene=181, id=221 removed by soft count = 57\n",
      "--- gene=181, id=222 removed by soft count = 141\n",
      "--- gene=181, id=223 removed by soft count = 113\n",
      "--- gene=181, id=254 removed by soft count = 51\n",
      "--- gene=181, id=255 removed by soft count = 408\n",
      "--- gene=181, id=264 removed by soft count = 65\n",
      "--- gene=181, id=265 removed by soft count = 138\n",
      "--- gene=181, id=266 removed by soft count = 475\n",
      "--- gene=181, id=267 removed by soft count = 76\n",
      "--- gene=181, id=281 removed by soft count = 81\n",
      "--- gene=181, id=359 removed by soft count = 164\n",
      "--- number of probes: 385 , kept by blast: 372 , if remove dups: 372\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 182\n",
      "--- gene=182, id=72 removed by soft count = 148\n",
      "--- gene=182, id=90 removed by soft count = 55\n",
      "--- gene=182, id=139 removed by soft count = 63\n",
      "--- gene=182, id=147 removed by soft count = 72\n",
      "--- gene=182, id=387 removed by soft count = 261\n",
      "--- gene=182, id=390 removed by soft count = 119\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 183\n",
      "--- gene=183, id=148 removed by soft count = 923\n",
      "--- gene=183, id=163 removed by soft count = 54\n",
      "--- gene=183, id=181 removed by soft count = 374\n",
      "--- gene=183, id=182 removed by soft count = 1214\n",
      "--- gene=183, id=197 removed by soft count = 944\n",
      "--- gene=183, id=324 removed by soft count = 112\n",
      "--- gene=183, id=332 removed by soft count = 55\n",
      "--- gene=183, id=342 removed by soft count = 53\n",
      "--- gene=183, id=348 removed by soft count = 214\n",
      "--- gene=183, id=375 removed by soft count = 84\n",
      "--- number of probes: 400 , kept by blast: 390 , if remove dups: 388\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 184\n",
      "--- gene=184, id=24 removed by soft count = 82\n",
      "--- gene=184, id=53 removed by soft count = 102\n",
      "--- gene=184, id=76 removed by soft count = 423\n",
      "--- gene=184, id=124 removed by soft count = 55\n",
      "--- gene=184, id=154 removed by soft count = 55\n",
      "--- gene=184, id=166 removed by soft count = 72\n",
      "--- gene=184, id=190 removed by soft count = 59\n",
      "--- gene=184, id=251 removed by soft count = 195\n",
      "--- gene=184, id=269 removed by soft count = 18768\n",
      "--- number of probes: 400 , kept by blast: 391 , if remove dups: 391\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 185\n",
      "--- gene=185, id=25 removed by hard count = 3\n",
      "--- gene=185, id=34 removed by soft count = 127\n",
      "--- gene=185, id=63 removed by soft count = 112\n",
      "--- gene=185, id=65 removed by soft count = 105\n",
      "--- gene=185, id=69 removed by soft count = 395\n",
      "--- gene=185, id=70 removed by soft count = 208\n",
      "--- gene=185, id=74 removed by soft count = 111\n",
      "--- gene=185, id=94 removed by hard count = 3\n",
      "--- gene=185, id=96 removed by hard count = 3\n",
      "--- gene=185, id=97 removed by hard count = 3\n",
      "--- gene=185, id=98 removed by hard count = 3\n",
      "--- gene=185, id=99 removed by hard count = 3\n",
      "--- gene=185, id=103 removed by hard count = 3\n",
      "--- gene=185, id=106 removed by hard count = 3\n",
      "--- gene=185, id=107 removed by hard count = 3\n",
      "--- gene=185, id=108 removed by hard count = 3\n",
      "--- gene=185, id=109 removed by hard count = 3\n",
      "--- gene=185, id=110 removed by hard count = 3\n",
      "--- number of probes: 114 , kept by blast: 96 , if remove dups: 17\n",
      "-- number of probes kept for this region: 96\n",
      "-- checking probes in region: 186\n",
      "--- gene=186, id=207 removed by soft count = 64\n",
      "--- gene=186, id=225 removed by soft count = 75\n",
      "--- gene=186, id=282 removed by soft count = 82\n",
      "--- gene=186, id=319 removed by soft count = 95\n",
      "--- gene=186, id=327 removed by soft count = 109\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=186, id=346 removed by soft count = 52\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 187\n",
      "--- gene=187, id=2 removed by soft count = 60\n",
      "--- gene=187, id=65 removed by soft count = 134\n",
      "--- gene=187, id=117 removed by soft count = 63\n",
      "--- gene=187, id=295 removed by soft count = 59\n",
      "--- gene=187, id=303 removed by soft count = 92\n",
      "--- gene=187, id=304 removed by soft count = 62\n",
      "--- gene=187, id=305 removed by soft count = 203\n",
      "--- gene=187, id=323 removed by soft count = 112\n",
      "--- gene=187, id=360 removed by soft count = 53\n",
      "--- gene=187, id=362 removed by soft count = 93\n",
      "--- gene=187, id=385 removed by soft count = 224\n",
      "--- number of probes: 400 , kept by blast: 389 , if remove dups: 389\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 188\n",
      "--- gene=188, id=93 removed by soft count = 238\n",
      "--- gene=188, id=94 removed by soft count = 341\n",
      "--- gene=188, id=139 removed by soft count = 80\n",
      "--- gene=188, id=388 removed by soft count = 125\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 189\n",
      "--- gene=189, id=31 removed by soft count = 231\n",
      "--- gene=189, id=190 removed by soft count = 303\n",
      "--- gene=189, id=235 removed by soft count = 296\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 190\n",
      "--- gene=190, id=100 removed by soft count = 147\n",
      "--- gene=190, id=104 removed by soft count = 539\n",
      "--- gene=190, id=124 removed by soft count = 83\n",
      "--- gene=190, id=154 removed by soft count = 202\n",
      "--- gene=190, id=164 removed by soft count = 3434\n",
      "--- gene=190, id=207 removed by soft count = 72\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 191\n",
      "--- gene=191, id=35 removed by soft count = 126\n",
      "--- gene=191, id=57 removed by soft count = 70\n",
      "--- gene=191, id=58 removed by soft count = 283\n",
      "--- gene=191, id=198 removed by soft count = 51\n",
      "--- gene=191, id=309 removed by soft count = 291\n",
      "--- gene=191, id=312 removed by soft count = 55\n",
      "--- gene=191, id=313 removed by soft count = 170\n",
      "--- gene=191, id=318 removed by soft count = 73\n",
      "--- number of probes: 400 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 192\n",
      "--- gene=192, id=81 removed by soft count = 57\n",
      "--- gene=192, id=126 removed by soft count = 415\n",
      "--- gene=192, id=149 removed by soft count = 73\n",
      "--- gene=192, id=309 removed by soft count = 77\n",
      "--- number of probes: 394 , kept by blast: 390 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 193\n",
      "--- gene=193, id=132 removed by soft count = 245\n",
      "--- gene=193, id=133 removed by soft count = 60\n",
      "--- gene=193, id=134 removed by soft count = 934\n",
      "--- gene=193, id=135 removed by soft count = 371\n",
      "--- gene=193, id=236 removed by soft count = 58\n",
      "--- gene=193, id=238 removed by soft count = 362\n",
      "--- gene=193, id=292 removed by soft count = 393\n",
      "--- gene=193, id=365 removed by soft count = 62\n",
      "--- gene=193, id=383 removed by soft count = 416\n",
      "--- number of probes: 400 , kept by blast: 391 , if remove dups: 391\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 194\n",
      "--- gene=194, id=314 removed by soft count = 300\n",
      "--- gene=194, id=321 removed by soft count = 592\n",
      "--- gene=194, id=322 removed by soft count = 57\n",
      "--- gene=194, id=327 removed by soft count = 60\n",
      "--- gene=194, id=329 removed by soft count = 354\n",
      "--- gene=194, id=330 removed by soft count = 53\n",
      "--- gene=194, id=332 removed by soft count = 121\n",
      "--- gene=194, id=333 removed by soft count = 951\n",
      "--- gene=194, id=335 removed by soft count = 270\n",
      "--- gene=194, id=354 removed by soft count = 108\n",
      "--- number of probes: 400 , kept by blast: 390 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 195\n",
      "--- gene=195, id=1 removed by soft count = 237\n",
      "--- gene=195, id=2 removed by soft count = 78\n",
      "--- gene=195, id=63 removed by soft count = 88\n",
      "--- gene=195, id=182 removed by soft count = 160\n",
      "--- gene=195, id=183 removed by soft count = 103\n",
      "--- gene=195, id=287 removed by soft count = 180\n",
      "--- gene=195, id=293 removed by soft count = 937\n",
      "--- gene=195, id=351 removed by soft count = 79\n",
      "--- gene=195, id=392 removed by soft count = 981\n",
      "--- number of probes: 400 , kept by blast: 391 , if remove dups: 391\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 196\n",
      "--- gene=196, id=103 removed by soft count = 398\n",
      "--- gene=196, id=104 removed by soft count = 1126\n",
      "--- gene=196, id=136 removed by soft count = 52\n",
      "--- gene=196, id=185 removed by soft count = 106\n",
      "--- gene=196, id=186 removed by soft count = 701\n",
      "--- gene=196, id=187 removed by soft count = 75\n",
      "--- gene=196, id=188 removed by soft count = 54\n",
      "--- gene=196, id=309 removed by soft count = 63\n",
      "--- gene=196, id=343 removed by soft count = 94\n",
      "--- gene=196, id=345 removed by soft count = 258\n",
      "--- gene=196, id=346 removed by soft count = 64\n",
      "--- gene=196, id=395 removed by soft count = 241\n",
      "--- gene=196, id=396 removed by soft count = 60\n",
      "--- gene=196, id=397 removed by soft count = 618\n",
      "--- number of probes: 400 , kept by blast: 386 , if remove dups: 386\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 197\n",
      "--- gene=197, id=0 removed by soft count = 146\n",
      "--- gene=197, id=10 removed by soft count = 10330\n",
      "--- gene=197, id=39 removed by soft count = 218\n",
      "--- gene=197, id=52 removed by soft count = 2869\n",
      "--- gene=197, id=73 removed by soft count = 83\n",
      "--- gene=197, id=90 removed by soft count = 197\n",
      "--- gene=197, id=91 removed by soft count = 207\n",
      "--- gene=197, id=92 removed by soft count = 741\n",
      "--- gene=197, id=146 removed by soft count = 156\n",
      "--- gene=197, id=187 removed by soft count = 62\n",
      "--- gene=197, id=395 removed by soft count = 76\n",
      "--- gene=197, id=396 removed by soft count = 232\n",
      "--- gene=197, id=397 removed by soft count = 170\n",
      "--- number of probes: 400 , kept by blast: 387 , if remove dups: 387\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 198\n",
      "--- gene=198, id=148 removed by soft count = 111\n",
      "--- gene=198, id=237 removed by soft count = 62\n",
      "--- gene=198, id=368 removed by soft count = 65\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 199\n",
      "--- gene=199, id=299 removed by soft count = 71\n",
      "--- gene=199, id=350 removed by soft count = 535\n",
      "--- gene=199, id=351 removed by soft count = 425\n",
      "--- gene=199, id=352 removed by soft count = 327\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 200\n",
      "--- gene=200, id=88 removed by soft count = 167\n",
      "--- gene=200, id=89 removed by soft count = 1384\n",
      "--- gene=200, id=182 removed by soft count = 85\n",
      "--- gene=200, id=233 removed by soft count = 120\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 201\n",
      "--- gene=201, id=296 removed by soft count = 148\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 202\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=202, id=62 removed by soft count = 58\n",
      "--- gene=202, id=63 removed by soft count = 51\n",
      "--- gene=202, id=64 removed by soft count = 54\n",
      "--- gene=202, id=65 removed by soft count = 66\n",
      "--- gene=202, id=228 removed by soft count = 99\n",
      "--- gene=202, id=329 removed by soft count = 1019\n",
      "--- gene=202, id=383 removed by soft count = 98\n",
      "--- number of probes: 392 , kept by blast: 385 , if remove dups: 385\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 203\n",
      "--- gene=203, id=38 removed by soft count = 3001\n",
      "--- gene=203, id=338 removed by soft count = 881\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 204\n",
      "--- gene=204, id=301 removed by soft count = 171\n",
      "--- number of probes: 399 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 205\n",
      "--- gene=205, id=0 removed by soft count = 2356\n",
      "--- gene=205, id=83 removed by soft count = 151\n",
      "--- gene=205, id=84 removed by soft count = 86\n",
      "--- gene=205, id=177 removed by soft count = 70\n",
      "--- gene=205, id=178 removed by soft count = 80\n",
      "--- gene=205, id=211 removed by soft count = 259\n",
      "--- gene=205, id=356 removed by soft count = 1741\n",
      "--- gene=205, id=370 removed by soft count = 90\n",
      "--- number of probes: 400 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 206\n",
      "--- gene=206, id=51 removed by soft count = 61\n",
      "--- gene=206, id=180 removed by soft count = 93\n",
      "--- gene=206, id=310 removed by soft count = 78\n",
      "--- gene=206, id=315 removed by soft count = 91\n",
      "--- gene=206, id=347 removed by soft count = 280\n",
      "--- number of probes: 399 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 207\n",
      "--- gene=207, id=35 removed by hard count = 0\n",
      "--- gene=207, id=137 removed by soft count = 512\n",
      "--- gene=207, id=169 removed by soft count = 75\n",
      "--- gene=207, id=250 removed by soft count = 87\n",
      "--- gene=207, id=257 removed by soft count = 129\n",
      "--- gene=207, id=258 removed by soft count = 113\n",
      "--- gene=207, id=259 removed by soft count = 132\n",
      "--- gene=207, id=278 removed by soft count = 587\n",
      "--- number of probes: 400 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 208\n",
      "--- gene=208, id=41 removed by soft count = 299\n",
      "--- gene=208, id=162 removed by soft count = 237\n",
      "--- gene=208, id=163 removed by soft count = 106\n",
      "--- gene=208, id=200 removed by soft count = 89\n",
      "--- gene=208, id=205 removed by soft count = 142\n",
      "--- number of probes: 398 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 209\n",
      "--- gene=209, id=386 removed by soft count = 58\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 210\n",
      "--- gene=210, id=9 removed by soft count = 328\n",
      "--- gene=210, id=105 removed by soft count = 138\n",
      "--- gene=210, id=227 removed by soft count = 80\n",
      "--- gene=210, id=258 removed by soft count = 51\n",
      "--- gene=210, id=269 removed by soft count = 94\n",
      "--- gene=210, id=322 removed by soft count = 101\n",
      "--- gene=210, id=324 removed by soft count = 86\n",
      "--- gene=210, id=379 removed by soft count = 1549\n",
      "--- number of probes: 397 , kept by blast: 389 , if remove dups: 389\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 211\n",
      "--- gene=211, id=229 removed by soft count = 170\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 212\n",
      "--- gene=212, id=0 removed by soft count = 111\n",
      "--- gene=212, id=2 removed by soft count = 153\n",
      "--- gene=212, id=5 removed by soft count = 588\n",
      "--- gene=212, id=6 removed by soft count = 171\n",
      "--- gene=212, id=7 removed by soft count = 1224\n",
      "--- gene=212, id=8 removed by soft count = 121\n",
      "--- gene=212, id=9 removed by soft count = 119\n",
      "--- gene=212, id=10 removed by soft count = 226\n",
      "--- gene=212, id=11 removed by soft count = 352\n",
      "--- gene=212, id=14 removed by soft count = 83\n",
      "--- gene=212, id=15 removed by soft count = 197\n",
      "--- gene=212, id=17 removed by soft count = 136\n",
      "--- gene=212, id=20 removed by soft count = 99\n",
      "--- gene=212, id=21 removed by soft count = 461\n",
      "--- gene=212, id=24 removed by soft count = 373\n",
      "--- gene=212, id=25 removed by soft count = 864\n",
      "--- gene=212, id=26 removed by soft count = 379\n",
      "--- gene=212, id=29 removed by soft count = 66\n",
      "--- gene=212, id=32 removed by soft count = 110\n",
      "--- gene=212, id=33 removed by soft count = 85\n",
      "--- gene=212, id=36 removed by soft count = 158\n",
      "--- gene=212, id=38 removed by soft count = 259\n",
      "--- gene=212, id=40 removed by soft count = 350\n",
      "--- gene=212, id=41 removed by soft count = 105\n",
      "--- gene=212, id=42 removed by soft count = 135\n",
      "--- gene=212, id=44 removed by soft count = 65\n",
      "--- gene=212, id=45 removed by soft count = 61\n",
      "--- gene=212, id=46 removed by soft count = 380\n",
      "--- gene=212, id=48 removed by soft count = 196\n",
      "--- gene=212, id=50 removed by soft count = 359\n",
      "--- gene=212, id=52 removed by soft count = 593\n",
      "--- gene=212, id=53 removed by soft count = 657\n",
      "--- gene=212, id=54 removed by soft count = 120\n",
      "--- gene=212, id=56 removed by soft count = 70\n",
      "--- gene=212, id=57 removed by soft count = 1281\n",
      "--- gene=212, id=58 removed by soft count = 219\n",
      "--- gene=212, id=59 removed by soft count = 53\n",
      "--- gene=212, id=61 removed by soft count = 157\n",
      "--- gene=212, id=64 removed by soft count = 1095\n",
      "--- gene=212, id=65 removed by soft count = 71\n",
      "--- gene=212, id=66 removed by soft count = 319\n",
      "--- gene=212, id=68 removed by soft count = 132\n",
      "--- gene=212, id=70 removed by soft count = 139\n",
      "--- gene=212, id=72 removed by soft count = 136\n",
      "--- gene=212, id=73 removed by soft count = 154\n",
      "--- gene=212, id=75 removed by soft count = 55\n",
      "--- gene=212, id=76 removed by soft count = 191\n",
      "--- gene=212, id=77 removed by soft count = 789\n",
      "--- gene=212, id=79 removed by soft count = 407\n",
      "--- gene=212, id=82 removed by soft count = 77\n",
      "--- gene=212, id=83 removed by soft count = 120\n",
      "--- gene=212, id=85 removed by soft count = 79\n",
      "--- gene=212, id=86 removed by soft count = 94\n",
      "--- gene=212, id=88 removed by soft count = 51\n",
      "--- gene=212, id=89 removed by soft count = 617\n",
      "--- gene=212, id=92 removed by soft count = 87\n",
      "--- gene=212, id=93 removed by soft count = 313\n",
      "--- gene=212, id=94 removed by soft count = 162\n",
      "--- gene=212, id=95 removed by soft count = 65\n",
      "--- gene=212, id=97 removed by soft count = 301\n",
      "--- gene=212, id=99 removed by soft count = 101\n",
      "--- gene=212, id=104 removed by soft count = 306\n",
      "--- gene=212, id=105 removed by soft count = 76\n",
      "--- gene=212, id=106 removed by soft count = 118\n",
      "--- gene=212, id=107 removed by soft count = 284\n",
      "--- gene=212, id=108 removed by soft count = 244\n",
      "--- gene=212, id=111 removed by soft count = 612\n",
      "--- gene=212, id=112 removed by soft count = 327\n",
      "--- gene=212, id=114 removed by soft count = 219\n",
      "--- gene=212, id=115 removed by soft count = 331\n",
      "--- gene=212, id=119 removed by soft count = 179\n",
      "--- gene=212, id=122 removed by soft count = 124\n",
      "--- gene=212, id=124 removed by soft count = 57\n",
      "--- gene=212, id=125 removed by soft count = 281\n",
      "--- gene=212, id=127 removed by soft count = 110\n",
      "--- gene=212, id=128 removed by soft count = 413\n",
      "--- gene=212, id=130 removed by soft count = 121\n",
      "--- gene=212, id=132 removed by soft count = 152\n",
      "--- gene=212, id=133 removed by soft count = 162\n",
      "--- gene=212, id=134 removed by soft count = 225\n",
      "--- gene=212, id=135 removed by soft count = 73\n",
      "--- gene=212, id=136 removed by soft count = 143\n",
      "--- gene=212, id=138 removed by soft count = 52\n",
      "--- gene=212, id=139 removed by soft count = 92\n",
      "--- gene=212, id=141 removed by soft count = 344\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=212, id=142 removed by soft count = 344\n",
      "--- gene=212, id=144 removed by soft count = 113\n",
      "--- gene=212, id=146 removed by soft count = 310\n",
      "--- gene=212, id=149 removed by soft count = 141\n",
      "--- gene=212, id=150 removed by soft count = 350\n",
      "--- gene=212, id=151 removed by soft count = 84\n",
      "--- gene=212, id=152 removed by soft count = 984\n",
      "--- gene=212, id=153 removed by soft count = 62\n",
      "--- gene=212, id=154 removed by soft count = 527\n",
      "--- gene=212, id=156 removed by soft count = 155\n",
      "--- gene=212, id=160 removed by soft count = 142\n",
      "--- gene=212, id=162 removed by soft count = 53\n",
      "--- gene=212, id=166 removed by soft count = 59\n",
      "--- gene=212, id=168 removed by soft count = 107\n",
      "--- gene=212, id=174 removed by soft count = 143\n",
      "--- gene=212, id=175 removed by soft count = 413\n",
      "--- gene=212, id=176 removed by soft count = 105\n",
      "--- gene=212, id=179 removed by soft count = 1587\n",
      "--- gene=212, id=180 removed by soft count = 204\n",
      "--- gene=212, id=181 removed by soft count = 144\n",
      "--- gene=212, id=183 removed by soft count = 303\n",
      "--- gene=212, id=186 removed by soft count = 89\n",
      "--- gene=212, id=187 removed by soft count = 52\n",
      "--- gene=212, id=189 removed by soft count = 320\n",
      "--- gene=212, id=191 removed by soft count = 213\n",
      "--- gene=212, id=192 removed by soft count = 71\n",
      "--- gene=212, id=193 removed by soft count = 262\n",
      "--- gene=212, id=195 removed by soft count = 642\n",
      "--- gene=212, id=196 removed by soft count = 73\n",
      "--- gene=212, id=200 removed by soft count = 2658\n",
      "--- gene=212, id=202 removed by soft count = 231\n",
      "--- gene=212, id=205 removed by soft count = 141\n",
      "--- number of probes: 206 , kept by blast: 89 , if remove dups: 87\n",
      "-- number of probes kept for this region: 89\n",
      "-- checking probes in region: 213\n",
      "--- gene=213, id=32 removed by soft count = 116\n",
      "--- gene=213, id=33 removed by soft count = 88\n",
      "--- gene=213, id=34 removed by soft count = 302\n",
      "--- gene=213, id=35 removed by soft count = 715\n",
      "--- gene=213, id=36 removed by soft count = 231\n",
      "--- gene=213, id=37 removed by soft count = 242\n",
      "--- gene=213, id=38 removed by soft count = 500\n",
      "--- gene=213, id=47 removed by soft count = 130\n",
      "--- gene=213, id=169 removed by soft count = 10136\n",
      "--- gene=213, id=237 removed by soft count = 773\n",
      "--- gene=213, id=253 removed by soft count = 69\n",
      "--- gene=213, id=254 removed by soft count = 710\n",
      "--- gene=213, id=282 removed by soft count = 353\n",
      "--- number of probes: 283 , kept by blast: 270 , if remove dups: 267\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 214\n",
      "--- gene=214, id=58 removed by soft count = 97\n",
      "--- gene=214, id=59 removed by soft count = 321\n",
      "--- gene=214, id=78 removed by soft count = 88\n",
      "--- gene=214, id=135 removed by soft count = 117\n",
      "--- gene=214, id=139 removed by soft count = 65\n",
      "--- gene=214, id=222 removed by soft count = 86\n",
      "--- gene=214, id=306 removed by soft count = 111\n",
      "--- gene=214, id=307 removed by soft count = 143\n",
      "--- gene=214, id=309 removed by soft count = 3602\n",
      "--- gene=214, id=311 removed by soft count = 63\n",
      "--- gene=214, id=312 removed by soft count = 64\n",
      "--- number of probes: 400 , kept by blast: 389 , if remove dups: 388\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 215\n",
      "--- gene=215, id=15 removed by soft count = 147\n",
      "--- gene=215, id=38 removed by soft count = 128\n",
      "--- gene=215, id=65 removed by soft count = 64\n",
      "--- gene=215, id=86 removed by soft count = 110\n",
      "--- gene=215, id=88 removed by soft count = 242\n",
      "--- gene=215, id=179 removed by soft count = 99\n",
      "--- gene=215, id=229 removed by soft count = 206\n",
      "--- gene=215, id=367 removed by soft count = 65\n",
      "--- gene=215, id=368 removed by soft count = 103\n",
      "--- gene=215, id=398 removed by soft count = 126\n",
      "--- gene=215, id=399 removed by soft count = 69\n",
      "--- number of probes: 400 , kept by blast: 389 , if remove dups: 389\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 216\n",
      "--- gene=216, id=125 removed by soft count = 416\n",
      "--- gene=216, id=230 removed by soft count = 93\n",
      "--- gene=216, id=322 removed by soft count = 63\n",
      "--- gene=216, id=323 removed by soft count = 97\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 217\n",
      "--- gene=217, id=47 removed by soft count = 2468\n",
      "--- gene=217, id=251 removed by soft count = 59\n",
      "--- gene=217, id=301 removed by soft count = 111\n",
      "--- gene=217, id=398 removed by soft count = 1370\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 218\n",
      "--- gene=218, id=281 removed by soft count = 154\n",
      "--- gene=218, id=326 removed by soft count = 101\n",
      "--- gene=218, id=371 removed by soft count = 73\n",
      "--- gene=218, id=380 removed by soft count = 76\n",
      "--- number of probes: 399 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 219\n",
      "--- gene=219, id=29 removed by soft count = 89\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 220\n",
      "--- gene=220, id=5 removed by soft count = 52\n",
      "--- gene=220, id=63 removed by soft count = 57\n",
      "--- gene=220, id=67 removed by soft count = 138\n",
      "--- gene=220, id=150 removed by soft count = 82\n",
      "--- gene=220, id=155 removed by soft count = 185\n",
      "--- gene=220, id=172 removed by soft count = 95\n",
      "--- gene=220, id=198 removed by soft count = 151\n",
      "--- gene=220, id=215 removed by soft count = 400\n",
      "--- gene=220, id=276 removed by soft count = 166\n",
      "--- gene=220, id=360 removed by soft count = 62\n",
      "--- gene=220, id=362 removed by soft count = 136\n",
      "--- gene=220, id=364 removed by soft count = 134\n",
      "--- number of probes: 400 , kept by blast: 388 , if remove dups: 388\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 221\n",
      "--- gene=221, id=6 removed by soft count = 62\n",
      "--- gene=221, id=78 removed by soft count = 121\n",
      "--- gene=221, id=319 removed by soft count = 113\n",
      "--- gene=221, id=321 removed by soft count = 217\n",
      "--- gene=221, id=379 removed by soft count = 256\n",
      "--- number of probes: 397 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 222\n",
      "--- gene=222, id=50 removed by soft count = 971\n",
      "--- gene=222, id=52 removed by soft count = 155\n",
      "--- gene=222, id=75 removed by soft count = 554\n",
      "--- gene=222, id=82 removed by soft count = 51\n",
      "--- gene=222, id=181 removed by soft count = 2355\n",
      "--- gene=222, id=246 removed by soft count = 83\n",
      "--- gene=222, id=271 removed by soft count = 108\n",
      "--- gene=222, id=309 removed by soft count = 110\n",
      "--- gene=222, id=313 removed by soft count = 23330\n",
      "--- gene=222, id=314 removed by soft count = 738\n",
      "--- gene=222, id=317 removed by soft count = 162\n",
      "--- gene=222, id=331 removed by soft count = 103\n",
      "--- gene=222, id=333 removed by soft count = 101\n",
      "--- number of probes: 400 , kept by blast: 387 , if remove dups: 387\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 223\n",
      "--- gene=223, id=21 removed by soft count = 51\n",
      "--- gene=223, id=102 removed by soft count = 68\n",
      "--- gene=223, id=120 removed by soft count = 66\n",
      "--- gene=223, id=303 removed by soft count = 116\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 224\n",
      "--- gene=224, id=27 removed by soft count = 61\n",
      "--- gene=224, id=72 removed by soft count = 1721\n",
      "--- gene=224, id=160 removed by soft count = 164\n",
      "--- gene=224, id=162 removed by soft count = 86\n",
      "--- gene=224, id=208 removed by soft count = 56\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=224, id=319 removed by soft count = 68\n",
      "--- gene=224, id=361 removed by soft count = 63\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 225\n",
      "--- gene=225, id=266 removed by soft count = 69\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 226\n",
      "--- gene=226, id=170 removed by soft count = 100\n",
      "--- gene=226, id=174 removed by soft count = 122\n",
      "--- gene=226, id=328 removed by soft count = 542\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 228\n",
      "--- gene=228, id=62 removed by soft count = 78\n",
      "--- gene=228, id=86 removed by soft count = 109\n",
      "--- gene=228, id=87 removed by soft count = 52\n",
      "--- gene=228, id=89 removed by soft count = 172\n",
      "--- gene=228, id=210 removed by soft count = 191\n",
      "--- gene=228, id=212 removed by soft count = 64\n",
      "--- gene=228, id=214 removed by soft count = 87\n",
      "--- gene=228, id=215 removed by soft count = 307\n",
      "--- gene=228, id=245 removed by soft count = 68\n",
      "--- gene=228, id=266 removed by soft count = 430\n",
      "--- number of probes: 400 , kept by blast: 390 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 229\n",
      "--- gene=229, id=86 removed by soft count = 52\n",
      "--- gene=229, id=203 removed by soft count = 136\n",
      "--- gene=229, id=204 removed by soft count = 99\n",
      "--- gene=229, id=205 removed by soft count = 205\n",
      "--- gene=229, id=286 removed by soft count = 60\n",
      "--- number of probes: 396 , kept by blast: 391 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 230\n",
      "--- gene=230, id=327 removed by soft count = 93\n",
      "--- gene=230, id=346 removed by soft count = 70\n",
      "--- gene=230, id=364 removed by soft count = 153\n",
      "--- gene=230, id=393 removed by soft count = 140\n",
      "--- gene=230, id=394 removed by soft count = 94\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 231\n",
      "--- gene=231, id=77 removed by soft count = 76\n",
      "--- gene=231, id=334 removed by soft count = 54\n",
      "--- gene=231, id=342 removed by soft count = 681\n",
      "--- gene=231, id=361 removed by soft count = 90\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 232\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 233\n",
      "--- gene=233, id=247 removed by soft count = 112\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 234\n",
      "--- gene=234, id=207 removed by soft count = 73\n",
      "--- gene=234, id=379 removed by soft count = 626\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 235\n",
      "--- gene=235, id=31 removed by soft count = 292\n",
      "--- gene=235, id=146 removed by soft count = 201\n",
      "--- gene=235, id=216 removed by soft count = 147\n",
      "--- gene=235, id=234 removed by soft count = 122\n",
      "--- gene=235, id=324 removed by soft count = 666\n",
      "--- gene=235, id=326 removed by soft count = 91\n",
      "--- gene=235, id=331 removed by soft count = 51\n",
      "--- number of probes: 398 , kept by blast: 391 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 236\n",
      "--- gene=236, id=109 removed by soft count = 62\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 237\n",
      "--- gene=237, id=38 removed by soft count = 78\n",
      "--- gene=237, id=137 removed by soft count = 753\n",
      "--- gene=237, id=142 removed by soft count = 589\n",
      "--- gene=237, id=148 removed by soft count = 103\n",
      "--- gene=237, id=151 removed by soft count = 69\n",
      "--- gene=237, id=210 removed by soft count = 60\n",
      "--- gene=237, id=259 removed by soft count = 103\n",
      "--- gene=237, id=370 removed by soft count = 64\n",
      "--- gene=237, id=388 removed by soft count = 128\n",
      "--- gene=237, id=390 removed by soft count = 224\n",
      "--- number of probes: 400 , kept by blast: 390 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 238\n",
      "--- gene=238, id=168 removed by soft count = 82\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 239\n",
      "--- gene=239, id=233 removed by soft count = 54\n",
      "--- gene=239, id=238 removed by soft count = 114\n",
      "--- gene=239, id=245 removed by soft count = 59\n",
      "--- gene=239, id=248 removed by soft count = 90\n",
      "--- gene=239, id=258 removed by soft count = 67\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 240\n",
      "--- gene=240, id=49 removed by soft count = 130\n",
      "--- gene=240, id=50 removed by soft count = 90\n",
      "--- gene=240, id=285 removed by soft count = 759\n",
      "--- gene=240, id=286 removed by soft count = 978\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 241\n",
      "--- gene=241, id=161 removed by soft count = 54\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 242\n",
      "--- gene=242, id=26 removed by soft count = 184\n",
      "--- gene=242, id=27 removed by soft count = 194\n",
      "--- gene=242, id=30 removed by soft count = 205\n",
      "--- gene=242, id=31 removed by soft count = 488\n",
      "--- gene=242, id=52 removed by soft count = 57\n",
      "--- gene=242, id=85 removed by soft count = 64\n",
      "--- gene=242, id=124 removed by soft count = 1428\n",
      "--- gene=242, id=139 removed by soft count = 84\n",
      "--- number of probes: 400 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 243\n",
      "--- gene=243, id=0 removed by soft count = 819\n",
      "--- gene=243, id=17 removed by soft count = 248\n",
      "--- gene=243, id=324 removed by soft count = 1036\n",
      "--- number of probes: 395 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 244\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 245\n",
      "--- gene=245, id=80 removed by soft count = 544\n",
      "--- gene=245, id=103 removed by soft count = 551\n",
      "--- gene=245, id=104 removed by soft count = 543\n",
      "--- gene=245, id=165 removed by soft count = 52\n",
      "--- gene=245, id=189 removed by soft count = 53\n",
      "--- gene=245, id=280 removed by soft count = 66\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 246\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 247\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=247, id=1 removed by hard count = 4\n",
      "--- gene=247, id=61 removed by soft count = 402\n",
      "--- gene=247, id=96 removed by soft count = 88\n",
      "--- gene=247, id=98 removed by soft count = 56\n",
      "--- gene=247, id=101 removed by soft count = 72\n",
      "--- gene=247, id=185 removed by soft count = 757\n",
      "--- gene=247, id=216 removed by soft count = 134\n",
      "--- gene=247, id=258 removed by soft count = 109\n",
      "--- gene=247, id=308 removed by soft count = 62\n",
      "--- gene=247, id=312 removed by soft count = 53\n",
      "--- number of probes: 400 , kept by blast: 390 , if remove dups: 389\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 248\n",
      "--- gene=248, id=20 removed by soft count = 186\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 249\n",
      "--- gene=249, id=133 removed by soft count = 536\n",
      "--- gene=249, id=227 removed by soft count = 774\n",
      "--- gene=249, id=309 removed by soft count = 372\n",
      "--- gene=249, id=361 removed by soft count = 619\n",
      "--- gene=249, id=391 removed by soft count = 69\n",
      "--- gene=249, id=392 removed by soft count = 70\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 250\n",
      "--- gene=250, id=16 removed by soft count = 137\n",
      "--- gene=250, id=21 removed by soft count = 129\n",
      "--- gene=250, id=32 removed by soft count = 71\n",
      "--- gene=250, id=33 removed by soft count = 77\n",
      "--- gene=250, id=70 removed by soft count = 54\n",
      "--- gene=250, id=238 removed by soft count = 51\n",
      "--- number of probes: 393 , kept by blast: 387 , if remove dups: 385\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 751\n",
      "--- number of probes: 10 , kept by blast: 10 , if remove dups: 10\n",
      "-- number of probes kept for this region: 10\n",
      "-- checking probes in region: 752\n",
      "--- gene=752, id=0 removed by soft count = 106\n",
      "--- gene=752, id=1 removed by soft count = 575\n",
      "--- gene=752, id=5 removed by soft count = 53\n",
      "--- gene=752, id=6 removed by soft count = 429\n",
      "--- gene=752, id=7 removed by soft count = 262\n",
      "--- gene=752, id=8 removed by soft count = 706\n",
      "--- gene=752, id=9 removed by soft count = 393\n",
      "--- gene=752, id=10 removed by soft count = 486\n",
      "--- gene=752, id=11 removed by soft count = 557\n",
      "--- gene=752, id=12 removed by hard count = 3\n",
      "--- gene=752, id=14 removed by soft count = 581\n",
      "--- gene=752, id=15 removed by soft count = 490\n",
      "--- gene=752, id=103 removed by soft count = 233\n",
      "--- number of probes: 176 , kept by blast: 163 , if remove dups: 162\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 758\n",
      "--- gene=758, id=0 removed by soft count = 62\n",
      "--- gene=758, id=90 removed by soft count = 60\n",
      "--- gene=758, id=91 removed by soft count = 169\n",
      "--- gene=758, id=171 removed by soft count = 494\n",
      "--- gene=758, id=193 removed by soft count = 55\n",
      "--- gene=758, id=194 removed by soft count = 134\n",
      "--- gene=758, id=324 removed by soft count = 634\n",
      "--- gene=758, id=339 removed by soft count = 313\n",
      "--- gene=758, id=375 removed by soft count = 56\n",
      "--- gene=758, id=397 removed by soft count = 85\n",
      "--- gene=758, id=398 removed by soft count = 298\n",
      "--- gene=758, id=399 removed by soft count = 113\n",
      "--- number of probes: 400 , kept by blast: 388 , if remove dups: 388\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 759\n",
      "--- gene=759, id=19 removed by soft count = 54\n",
      "--- gene=759, id=20 removed by soft count = 96\n",
      "--- gene=759, id=21 removed by soft count = 273\n",
      "--- gene=759, id=43 removed by soft count = 56\n",
      "--- gene=759, id=92 removed by soft count = 69\n",
      "--- gene=759, id=111 removed by soft count = 103\n",
      "--- gene=759, id=183 removed by soft count = 238\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 760\n",
      "--- gene=760, id=338 removed by soft count = 71\n",
      "--- gene=760, id=341 removed by soft count = 51\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 761\n",
      "--- gene=761, id=62 removed by soft count = 685\n",
      "--- gene=761, id=233 removed by soft count = 73\n",
      "--- gene=761, id=359 removed by soft count = 57\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 762\n",
      "--- gene=762, id=44 removed by soft count = 86\n",
      "--- gene=762, id=213 removed by soft count = 102\n",
      "--- gene=762, id=376 removed by soft count = 5710\n",
      "--- gene=762, id=377 removed by soft count = 64\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 763\n",
      "--- gene=763, id=159 removed by soft count = 127\n",
      "--- gene=763, id=233 removed by soft count = 152\n",
      "--- gene=763, id=235 removed by soft count = 609\n",
      "--- gene=763, id=240 removed by soft count = 82\n",
      "--- gene=763, id=241 removed by soft count = 245\n",
      "--- gene=763, id=242 removed by soft count = 200\n",
      "--- gene=763, id=316 removed by soft count = 116\n",
      "--- gene=763, id=325 removed by soft count = 121\n",
      "--- number of probes: 400 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 764\n",
      "--- gene=764, id=45 removed by soft count = 195\n",
      "--- gene=764, id=393 removed by soft count = 142\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 765\n",
      "--- gene=765, id=95 removed by soft count = 168\n",
      "--- gene=765, id=113 removed by soft count = 67\n",
      "--- gene=765, id=394 removed by soft count = 52\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 766\n",
      "--- gene=766, id=186 removed by soft count = 62\n",
      "--- gene=766, id=320 removed by soft count = 2281\n",
      "--- gene=766, id=392 removed by soft count = 62\n",
      "--- number of probes: 397 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 767\n",
      "--- gene=767, id=292 removed by soft count = 73\n",
      "--- gene=767, id=310 removed by soft count = 53\n",
      "--- gene=767, id=394 removed by soft count = 204\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 768\n",
      "--- gene=768, id=266 removed by soft count = 138\n",
      "--- gene=768, id=274 removed by soft count = 116\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 769\n",
      "--- gene=769, id=23 removed by soft count = 160\n",
      "--- gene=769, id=25 removed by soft count = 59\n",
      "--- gene=769, id=271 removed by soft count = 70\n",
      "--- gene=769, id=349 removed by soft count = 66\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 770\n",
      "--- gene=770, id=16 removed by soft count = 236\n",
      "--- gene=770, id=65 removed by soft count = 290\n",
      "--- gene=770, id=66 removed by soft count = 172\n",
      "--- gene=770, id=159 removed by soft count = 11270\n",
      "--- number of probes: 395 , kept by blast: 391 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 772\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=772, id=4 removed by soft count = 101\n",
      "--- gene=772, id=12 removed by soft count = 605\n",
      "--- gene=772, id=380 removed by soft count = 73\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 773\n",
      "--- gene=773, id=17 removed by soft count = 351\n",
      "--- gene=773, id=265 removed by soft count = 78\n",
      "--- gene=773, id=283 removed by soft count = 131\n",
      "--- gene=773, id=294 removed by soft count = 60\n",
      "--- gene=773, id=346 removed by soft count = 63\n",
      "--- gene=773, id=347 removed by soft count = 103\n",
      "--- number of probes: 393 , kept by blast: 387 , if remove dups: 387\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 774\n",
      "--- gene=774, id=339 removed by soft count = 140\n",
      "--- gene=774, id=340 removed by soft count = 228\n",
      "--- gene=774, id=341 removed by soft count = 112\n",
      "--- gene=774, id=342 removed by soft count = 149\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 775\n",
      "--- gene=775, id=174 removed by soft count = 62\n",
      "--- gene=775, id=175 removed by soft count = 205\n",
      "--- gene=775, id=297 removed by soft count = 1608\n",
      "--- gene=775, id=357 removed by soft count = 84\n",
      "--- gene=775, id=395 removed by soft count = 142\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 776\n",
      "--- number of probes: 62 , kept by blast: 62 , if remove dups: 62\n",
      "-- number of probes kept for this region: 62\n",
      "-- checking probes in region: 777\n",
      "--- gene=777, id=88 removed by soft count = 139\n",
      "--- gene=777, id=121 removed by soft count = 80\n",
      "--- gene=777, id=171 removed by soft count = 62\n",
      "--- gene=777, id=252 removed by soft count = 62\n",
      "--- gene=777, id=305 removed by hard count = 8\n",
      "--- gene=777, id=330 removed by soft count = 220\n",
      "--- number of probes: 395 , kept by blast: 389 , if remove dups: 389\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 778\n",
      "--- gene=778, id=73 removed by soft count = 136\n",
      "--- gene=778, id=81 removed by soft count = 167\n",
      "--- gene=778, id=162 removed by soft count = 61\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 779\n",
      "--- gene=779, id=4 removed by soft count = 649\n",
      "--- gene=779, id=81 removed by soft count = 79\n",
      "--- gene=779, id=82 removed by soft count = 99\n",
      "--- gene=779, id=83 removed by soft count = 83\n",
      "--- gene=779, id=84 removed by soft count = 170\n",
      "--- gene=779, id=113 removed by soft count = 57\n",
      "--- gene=779, id=117 removed by soft count = 88\n",
      "--- gene=779, id=120 removed by soft count = 56\n",
      "--- gene=779, id=121 removed by soft count = 58\n",
      "--- gene=779, id=155 removed by soft count = 556\n",
      "--- gene=779, id=209 removed by soft count = 91\n",
      "--- gene=779, id=307 removed by soft count = 108\n",
      "--- gene=779, id=358 removed by soft count = 63\n",
      "--- number of probes: 393 , kept by blast: 380 , if remove dups: 379\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 780\n",
      "--- gene=780, id=121 removed by soft count = 57\n",
      "--- gene=780, id=122 removed by soft count = 75\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 781\n",
      "--- gene=781, id=8 removed by soft count = 80\n",
      "--- gene=781, id=71 removed by soft count = 115\n",
      "--- gene=781, id=260 removed by soft count = 367\n",
      "--- gene=781, id=261 removed by soft count = 90\n",
      "--- gene=781, id=371 removed by soft count = 79\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 782\n",
      "--- gene=782, id=67 removed by soft count = 201\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 783\n",
      "--- gene=783, id=132 removed by soft count = 89\n",
      "--- gene=783, id=154 removed by soft count = 74\n",
      "--- gene=783, id=155 removed by soft count = 71\n",
      "--- gene=783, id=237 removed by soft count = 242\n",
      "--- gene=783, id=246 removed by soft count = 58\n",
      "--- gene=783, id=263 removed by soft count = 126\n",
      "--- gene=783, id=269 removed by soft count = 67\n",
      "--- gene=783, id=303 removed by soft count = 89\n",
      "--- gene=783, id=304 removed by soft count = 716\n",
      "--- gene=783, id=311 removed by soft count = 59\n",
      "--- gene=783, id=313 removed by soft count = 69\n",
      "--- gene=783, id=319 removed by soft count = 61\n",
      "--- number of probes: 400 , kept by blast: 388 , if remove dups: 388\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 784\n",
      "--- gene=784, id=57 removed by soft count = 361\n",
      "--- gene=784, id=83 removed by soft count = 214\n",
      "--- gene=784, id=86 removed by soft count = 102\n",
      "--- gene=784, id=227 removed by soft count = 1100\n",
      "--- gene=784, id=228 removed by soft count = 235\n",
      "--- gene=784, id=352 removed by soft count = 249\n",
      "--- gene=784, id=353 removed by soft count = 109\n",
      "--- gene=784, id=358 removed by soft count = 225\n",
      "--- gene=784, id=359 removed by soft count = 102\n",
      "--- gene=784, id=363 removed by soft count = 70\n",
      "--- gene=784, id=365 removed by soft count = 76\n",
      "--- gene=784, id=366 removed by soft count = 61\n",
      "--- gene=784, id=378 removed by soft count = 4503\n",
      "--- gene=784, id=389 removed by soft count = 187\n",
      "--- number of probes: 400 , kept by blast: 386 , if remove dups: 385\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 785\n",
      "--- gene=785, id=14 removed by soft count = 68\n",
      "--- gene=785, id=49 removed by soft count = 226\n",
      "--- gene=785, id=64 removed by soft count = 69\n",
      "--- gene=785, id=68 removed by soft count = 227\n",
      "--- gene=785, id=204 removed by soft count = 60\n",
      "--- gene=785, id=208 removed by soft count = 273\n",
      "--- gene=785, id=209 removed by soft count = 66\n",
      "--- gene=785, id=267 removed by soft count = 93\n",
      "--- gene=785, id=268 removed by soft count = 84\n",
      "--- gene=785, id=339 removed by soft count = 160\n",
      "--- number of probes: 400 , kept by blast: 390 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 786\n",
      "--- gene=786, id=90 removed by soft count = 76\n",
      "--- gene=786, id=151 removed by soft count = 122\n",
      "--- gene=786, id=234 removed by soft count = 240\n",
      "--- gene=786, id=389 removed by soft count = 82\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 787\n",
      "--- gene=787, id=12 removed by soft count = 403\n",
      "--- gene=787, id=40 removed by soft count = 116\n",
      "--- gene=787, id=348 removed by soft count = 973\n",
      "--- gene=787, id=349 removed by soft count = 138\n",
      "--- gene=787, id=350 removed by soft count = 162\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 788\n",
      "--- gene=788, id=75 removed by soft count = 115\n",
      "--- gene=788, id=76 removed by soft count = 599\n",
      "--- gene=788, id=170 removed by soft count = 79\n",
      "--- gene=788, id=312 removed by soft count = 54\n",
      "--- gene=788, id=313 removed by soft count = 249\n",
      "--- gene=788, id=348 removed by soft count = 53\n",
      "--- gene=788, id=357 removed by soft count = 122\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 789\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=789, id=339 removed by soft count = 366\n",
      "--- gene=789, id=341 removed by soft count = 711\n",
      "--- gene=789, id=342 removed by soft count = 2710\n",
      "--- gene=789, id=369 removed by soft count = 112\n",
      "--- gene=789, id=370 removed by hard count = 0\n",
      "--- gene=789, id=371 removed by hard count = 0\n",
      "--- gene=789, id=372 removed by hard count = 0\n",
      "--- gene=789, id=393 removed by soft count = 112\n",
      "--- number of probes: 398 , kept by blast: 390 , if remove dups: 389\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 790\n",
      "--- gene=790, id=51 removed by soft count = 52\n",
      "--- gene=790, id=297 removed by soft count = 129\n",
      "--- gene=790, id=327 removed by soft count = 497\n",
      "--- gene=790, id=358 removed by soft count = 75\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 791\n",
      "--- gene=791, id=296 removed by soft count = 70\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 792\n",
      "--- gene=792, id=59 removed by soft count = 188\n",
      "--- gene=792, id=114 removed by soft count = 841\n",
      "--- gene=792, id=186 removed by soft count = 259\n",
      "--- gene=792, id=246 removed by soft count = 378\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 793\n",
      "--- gene=793, id=56 removed by soft count = 208\n",
      "--- gene=793, id=119 removed by soft count = 139\n",
      "--- gene=793, id=357 removed by soft count = 63\n",
      "--- gene=793, id=358 removed by soft count = 67\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 794\n",
      "--- gene=794, id=5 removed by soft count = 102\n",
      "--- gene=794, id=35 removed by soft count = 251\n",
      "--- gene=794, id=96 removed by soft count = 76\n",
      "--- gene=794, id=169 removed by soft count = 100\n",
      "--- gene=794, id=225 removed by hard count = 7\n",
      "--- gene=794, id=233 removed by soft count = 70\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 795\n",
      "--- gene=795, id=89 removed by soft count = 53\n",
      "--- number of probes: 398 , kept by blast: 397 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 796\n",
      "--- gene=796, id=260 removed by soft count = 258\n",
      "--- gene=796, id=261 removed by soft count = 229\n",
      "--- gene=796, id=262 removed by soft count = 149\n",
      "--- gene=796, id=266 removed by soft count = 107\n",
      "--- gene=796, id=269 removed by soft count = 179\n",
      "--- gene=796, id=307 removed by soft count = 88\n",
      "--- gene=796, id=313 removed by soft count = 125\n",
      "--- gene=796, id=370 removed by soft count = 80\n",
      "--- gene=796, id=396 removed by soft count = 68\n",
      "--- number of probes: 400 , kept by blast: 391 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 797\n",
      "--- gene=797, id=1 removed by soft count = 71\n",
      "--- gene=797, id=4 removed by soft count = 57\n",
      "--- gene=797, id=42 removed by soft count = 64\n",
      "--- gene=797, id=351 removed by soft count = 135\n",
      "--- gene=797, id=352 removed by soft count = 81\n",
      "--- gene=797, id=354 removed by soft count = 355\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 798\n",
      "--- gene=798, id=2 removed by soft count = 73\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 799\n",
      "--- gene=799, id=27 removed by soft count = 260\n",
      "--- gene=799, id=29 removed by soft count = 119\n",
      "--- gene=799, id=210 removed by soft count = 102\n",
      "--- gene=799, id=214 removed by soft count = 488\n",
      "--- gene=799, id=306 removed by soft count = 361\n",
      "--- gene=799, id=370 removed by soft count = 124\n",
      "--- number of probes: 394 , kept by blast: 388 , if remove dups: 388\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 800\n",
      "--- gene=800, id=65 removed by soft count = 66\n",
      "--- gene=800, id=141 removed by soft count = 57\n",
      "--- gene=800, id=347 removed by soft count = 108\n",
      "--- gene=800, id=381 removed by soft count = 66\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 801\n",
      "--- gene=801, id=346 removed by soft count = 127\n",
      "--- gene=801, id=348 removed by soft count = 240\n",
      "--- gene=801, id=349 removed by soft count = 148\n",
      "--- gene=801, id=352 removed by soft count = 56\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 802\n",
      "--- gene=802, id=9 removed by soft count = 300\n",
      "--- gene=802, id=21 removed by soft count = 64\n",
      "--- gene=802, id=226 removed by soft count = 74\n",
      "--- gene=802, id=379 removed by soft count = 134\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 803\n",
      "--- gene=803, id=283 removed by soft count = 118\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 804\n",
      "--- gene=804, id=47 removed by soft count = 153\n",
      "--- gene=804, id=73 removed by soft count = 59\n",
      "--- gene=804, id=74 removed by soft count = 333\n",
      "--- gene=804, id=75 removed by soft count = 104\n",
      "--- gene=804, id=94 removed by soft count = 117\n",
      "--- gene=804, id=103 removed by soft count = 401\n",
      "--- gene=804, id=106 removed by soft count = 188\n",
      "--- gene=804, id=152 removed by soft count = 2427\n",
      "--- gene=804, id=153 removed by soft count = 171\n",
      "--- gene=804, id=170 removed by soft count = 135\n",
      "--- gene=804, id=221 removed by soft count = 75\n",
      "--- gene=804, id=237 removed by soft count = 313\n",
      "--- gene=804, id=267 removed by soft count = 1109\n",
      "--- gene=804, id=269 removed by soft count = 121\n",
      "--- gene=804, id=308 removed by soft count = 762\n",
      "--- number of probes: 400 , kept by blast: 385 , if remove dups: 385\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 805\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 806\n",
      "--- gene=806, id=26 removed by soft count = 872\n",
      "--- gene=806, id=44 removed by soft count = 192\n",
      "--- gene=806, id=92 removed by soft count = 318\n",
      "--- gene=806, id=299 removed by soft count = 598\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 807\n",
      "--- gene=807, id=1 removed by soft count = 65\n",
      "--- gene=807, id=2 removed by soft count = 130\n",
      "--- gene=807, id=12 removed by soft count = 214\n",
      "--- gene=807, id=44 removed by soft count = 72\n",
      "--- gene=807, id=205 removed by soft count = 641\n",
      "--- number of probes: 397 , kept by blast: 392 , if remove dups: 391\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 808\n",
      "--- gene=808, id=0 removed by soft count = 1150\n",
      "--- gene=808, id=1 removed by soft count = 115\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=808, id=59 removed by soft count = 184\n",
      "--- gene=808, id=60 removed by soft count = 260\n",
      "--- gene=808, id=61 removed by soft count = 62\n",
      "--- gene=808, id=62 removed by soft count = 104\n",
      "--- gene=808, id=64 removed by soft count = 77\n",
      "--- gene=808, id=127 removed by soft count = 738\n",
      "--- gene=808, id=128 removed by soft count = 65\n",
      "--- gene=808, id=129 removed by soft count = 67\n",
      "--- gene=808, id=130 removed by soft count = 68\n",
      "--- gene=808, id=184 removed by soft count = 311\n",
      "--- gene=808, id=201 removed by soft count = 63\n",
      "--- gene=808, id=203 removed by soft count = 135\n",
      "--- gene=808, id=211 removed by soft count = 51\n",
      "--- gene=808, id=260 removed by soft count = 57\n",
      "--- gene=808, id=305 removed by soft count = 169\n",
      "--- gene=808, id=356 removed by soft count = 292\n",
      "--- gene=808, id=357 removed by soft count = 222\n",
      "--- gene=808, id=358 removed by soft count = 60\n",
      "--- gene=808, id=399 removed by soft count = 91\n",
      "--- number of probes: 400 , kept by blast: 379 , if remove dups: 379\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 809\n",
      "--- gene=809, id=36 removed by soft count = 95\n",
      "--- gene=809, id=247 removed by soft count = 74\n",
      "--- gene=809, id=258 removed by soft count = 51\n",
      "--- gene=809, id=289 removed by soft count = 72\n",
      "--- number of probes: 397 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 810\n",
      "--- gene=810, id=120 removed by soft count = 253\n",
      "--- gene=810, id=198 removed by soft count = 93\n",
      "--- gene=810, id=201 removed by soft count = 93\n",
      "--- gene=810, id=227 removed by soft count = 135\n",
      "--- gene=810, id=229 removed by soft count = 85\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 811\n",
      "--- gene=811, id=147 removed by soft count = 69\n",
      "--- gene=811, id=152 removed by soft count = 336\n",
      "--- number of probes: 397 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 812\n",
      "--- gene=812, id=222 removed by soft count = 57\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 813\n",
      "--- gene=813, id=257 removed by soft count = 58\n",
      "--- number of probes: 396 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 814\n",
      "--- gene=814, id=211 removed by hard count = 9\n",
      "--- gene=814, id=293 removed by soft count = 103\n",
      "--- gene=814, id=294 removed by soft count = 222\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 815\n",
      "--- gene=815, id=6 removed by soft count = 213\n",
      "--- gene=815, id=84 removed by soft count = 13934\n",
      "--- gene=815, id=115 removed by soft count = 295\n",
      "--- gene=815, id=117 removed by soft count = 238\n",
      "--- gene=815, id=305 removed by soft count = 103\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 816\n",
      "--- gene=816, id=75 removed by soft count = 451\n",
      "--- gene=816, id=95 removed by soft count = 78\n",
      "--- gene=816, id=96 removed by soft count = 149\n",
      "--- gene=816, id=98 removed by soft count = 116\n",
      "--- gene=816, id=294 removed by soft count = 133\n",
      "--- gene=816, id=378 removed by soft count = 177\n",
      "--- gene=816, id=380 removed by soft count = 86\n",
      "--- gene=816, id=381 removed by soft count = 220\n",
      "--- gene=816, id=384 removed by soft count = 74\n",
      "--- number of probes: 400 , kept by blast: 391 , if remove dups: 391\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 817\n",
      "--- gene=817, id=16 removed by soft count = 56\n",
      "--- gene=817, id=35 removed by soft count = 129\n",
      "--- gene=817, id=50 removed by soft count = 1691\n",
      "--- gene=817, id=126 removed by soft count = 290\n",
      "--- gene=817, id=186 removed by soft count = 249\n",
      "--- gene=817, id=188 removed by soft count = 253\n",
      "--- gene=817, id=189 removed by soft count = 129\n",
      "--- gene=817, id=190 removed by soft count = 54\n",
      "--- gene=817, id=191 removed by soft count = 65\n",
      "--- gene=817, id=192 removed by soft count = 120\n",
      "--- gene=817, id=194 removed by soft count = 139\n",
      "--- gene=817, id=334 removed by soft count = 52\n",
      "--- gene=817, id=352 removed by soft count = 250\n",
      "--- gene=817, id=355 removed by soft count = 176\n",
      "--- gene=817, id=384 removed by soft count = 1573\n",
      "--- gene=817, id=385 removed by soft count = 97\n",
      "--- gene=817, id=386 removed by soft count = 134\n",
      "--- gene=817, id=387 removed by soft count = 198\n",
      "--- number of probes: 400 , kept by blast: 382 , if remove dups: 382\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 818\n",
      "--- gene=818, id=27 removed by soft count = 122\n",
      "--- gene=818, id=28 removed by soft count = 363\n",
      "--- gene=818, id=29 removed by soft count = 173\n",
      "--- gene=818, id=44 removed by soft count = 65\n",
      "--- gene=818, id=45 removed by soft count = 137\n",
      "--- gene=818, id=93 removed by soft count = 163\n",
      "--- gene=818, id=95 removed by soft count = 340\n",
      "--- gene=818, id=96 removed by soft count = 87\n",
      "--- gene=818, id=218 removed by soft count = 113\n",
      "--- number of probes: 400 , kept by blast: 391 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 819\n",
      "--- gene=819, id=139 removed by soft count = 52\n",
      "--- gene=819, id=336 removed by soft count = 56\n",
      "--- number of probes: 395 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 820\n",
      "--- gene=820, id=23 removed by soft count = 124\n",
      "--- gene=820, id=134 removed by soft count = 98\n",
      "--- gene=820, id=162 removed by soft count = 108\n",
      "--- gene=820, id=186 removed by soft count = 173\n",
      "--- gene=820, id=231 removed by soft count = 53\n",
      "--- gene=820, id=241 removed by soft count = 115\n",
      "--- gene=820, id=243 removed by soft count = 51\n",
      "--- gene=820, id=244 removed by soft count = 69\n",
      "--- gene=820, id=283 removed by soft count = 97\n",
      "--- gene=820, id=285 removed by soft count = 405\n",
      "--- gene=820, id=316 removed by soft count = 68\n",
      "--- gene=820, id=359 removed by soft count = 343\n",
      "--- number of probes: 398 , kept by blast: 386 , if remove dups: 386\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 821\n",
      "--- gene=821, id=76 removed by soft count = 172\n",
      "--- gene=821, id=77 removed by soft count = 196\n",
      "--- gene=821, id=80 removed by soft count = 62\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 822\n",
      "--- gene=822, id=2 removed by soft count = 459\n",
      "--- gene=822, id=28 removed by soft count = 158\n",
      "--- gene=822, id=45 removed by soft count = 165\n",
      "--- gene=822, id=61 removed by soft count = 129\n",
      "--- gene=822, id=385 removed by soft count = 62\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 823\n",
      "--- gene=823, id=10 removed by soft count = 64\n",
      "--- gene=823, id=88 removed by soft count = 109\n",
      "--- gene=823, id=103 removed by soft count = 252\n",
      "--- gene=823, id=116 removed by soft count = 62\n",
      "--- gene=823, id=277 removed by soft count = 123\n",
      "--- gene=823, id=330 removed by soft count = 54\n",
      "--- gene=823, id=335 removed by soft count = 94\n",
      "--- gene=823, id=341 removed by soft count = 73\n",
      "--- gene=823, id=343 removed by soft count = 189\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=823, id=362 removed by soft count = 88\n",
      "--- gene=823, id=363 removed by soft count = 309\n",
      "--- gene=823, id=368 removed by soft count = 72\n",
      "--- gene=823, id=370 removed by soft count = 277\n",
      "--- gene=823, id=371 removed by soft count = 84\n",
      "--- number of probes: 400 , kept by blast: 386 , if remove dups: 386\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 824\n",
      "--- gene=824, id=188 removed by soft count = 347\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 825\n",
      "--- gene=825, id=63 removed by soft count = 283\n",
      "--- gene=825, id=137 removed by soft count = 73\n",
      "--- gene=825, id=264 removed by soft count = 209\n",
      "--- gene=825, id=271 removed by soft count = 89\n",
      "--- gene=825, id=354 removed by soft count = 135\n",
      "--- gene=825, id=355 removed by soft count = 59\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 826\n",
      "--- gene=826, id=367 removed by soft count = 120\n",
      "--- gene=826, id=388 removed by soft count = 58\n",
      "--- gene=826, id=389 removed by soft count = 77\n",
      "--- gene=826, id=391 removed by soft count = 69\n",
      "--- number of probes: 398 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 827\n",
      "--- gene=827, id=93 removed by soft count = 94\n",
      "--- gene=827, id=102 removed by soft count = 62\n",
      "--- gene=827, id=110 removed by soft count = 735\n",
      "--- gene=827, id=139 removed by soft count = 170\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 828\n",
      "--- gene=828, id=0 removed by soft count = 37377\n",
      "--- gene=828, id=286 removed by soft count = 220\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 829\n",
      "--- gene=829, id=26 removed by soft count = 82\n",
      "--- gene=829, id=98 removed by soft count = 227\n",
      "--- gene=829, id=162 removed by soft count = 68\n",
      "--- gene=829, id=252 removed by soft count = 79\n",
      "--- gene=829, id=255 removed by soft count = 711\n",
      "--- gene=829, id=257 removed by soft count = 856\n",
      "--- gene=829, id=264 removed by soft count = 806\n",
      "--- gene=829, id=266 removed by soft count = 725\n",
      "--- gene=829, id=357 removed by soft count = 764\n",
      "--- gene=829, id=358 removed by soft count = 172\n",
      "--- gene=829, id=359 removed by soft count = 209\n",
      "--- gene=829, id=360 removed by soft count = 449\n",
      "--- number of probes: 400 , kept by blast: 388 , if remove dups: 388\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 830\n",
      "--- gene=830, id=214 removed by soft count = 55\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 831\n",
      "--- gene=831, id=88 removed by soft count = 100\n",
      "--- gene=831, id=89 removed by soft count = 307\n",
      "--- gene=831, id=150 removed by soft count = 225\n",
      "--- gene=831, id=257 removed by soft count = 62\n",
      "--- gene=831, id=260 removed by soft count = 57\n",
      "--- gene=831, id=287 removed by soft count = 118\n",
      "--- gene=831, id=295 removed by soft count = 151\n",
      "--- gene=831, id=339 removed by soft count = 109\n",
      "--- gene=831, id=344 removed by soft count = 156\n",
      "--- gene=831, id=350 removed by soft count = 63\n",
      "--- gene=831, id=399 removed by soft count = 315\n",
      "--- number of probes: 400 , kept by blast: 389 , if remove dups: 389\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 832\n",
      "--- gene=832, id=325 removed by soft count = 121\n",
      "--- gene=832, id=358 removed by soft count = 137\n",
      "--- gene=832, id=367 removed by soft count = 97\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 833\n",
      "--- gene=833, id=207 removed by soft count = 97\n",
      "--- gene=833, id=298 removed by soft count = 63\n",
      "--- gene=833, id=384 removed by soft count = 80\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 834\n",
      "--- gene=834, id=43 removed by soft count = 52\n",
      "--- gene=834, id=98 removed by soft count = 130\n",
      "--- gene=834, id=100 removed by soft count = 78\n",
      "--- gene=834, id=158 removed by soft count = 104\n",
      "--- gene=834, id=159 removed by soft count = 130\n",
      "--- gene=834, id=160 removed by soft count = 82\n",
      "--- gene=834, id=162 removed by soft count = 143\n",
      "--- gene=834, id=206 removed by soft count = 60\n",
      "--- gene=834, id=207 removed by soft count = 225\n",
      "--- gene=834, id=250 removed by soft count = 250\n",
      "--- gene=834, id=261 removed by soft count = 149\n",
      "--- gene=834, id=316 removed by soft count = 64\n",
      "--- gene=834, id=350 removed by soft count = 6427\n",
      "--- gene=834, id=355 removed by soft count = 332\n",
      "--- gene=834, id=388 removed by soft count = 72\n",
      "--- number of probes: 400 , kept by blast: 385 , if remove dups: 385\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 835\n",
      "--- gene=835, id=0 removed by soft count = 120\n",
      "--- gene=835, id=50 removed by soft count = 1649\n",
      "--- gene=835, id=83 removed by soft count = 59\n",
      "--- gene=835, id=93 removed by soft count = 92\n",
      "--- gene=835, id=212 removed by soft count = 267\n",
      "--- gene=835, id=229 removed by hard count = 5\n",
      "--- number of probes: 397 , kept by blast: 391 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 836\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 837\n",
      "--- gene=837, id=15 removed by soft count = 106\n",
      "--- gene=837, id=40 removed by soft count = 208\n",
      "--- gene=837, id=143 removed by soft count = 257\n",
      "--- gene=837, id=158 removed by soft count = 314\n",
      "--- gene=837, id=159 removed by soft count = 489\n",
      "--- gene=837, id=160 removed by soft count = 1815\n",
      "--- gene=837, id=165 removed by soft count = 599\n",
      "--- gene=837, id=247 removed by soft count = 180\n",
      "--- gene=837, id=366 removed by soft count = 71\n",
      "--- gene=837, id=381 removed by soft count = 74\n",
      "--- gene=837, id=385 removed by soft count = 147\n",
      "--- number of probes: 400 , kept by blast: 389 , if remove dups: 389\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 838\n",
      "--- gene=838, id=136 removed by soft count = 53\n",
      "--- gene=838, id=137 removed by soft count = 64\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 839\n",
      "--- gene=839, id=35 removed by soft count = 64\n",
      "--- gene=839, id=37 removed by soft count = 144\n",
      "--- gene=839, id=63 removed by hard count = 3\n",
      "--- gene=839, id=255 removed by soft count = 512\n",
      "--- gene=839, id=256 removed by soft count = 61\n",
      "--- gene=839, id=258 removed by soft count = 51\n",
      "--- number of probes: 399 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 840\n",
      "--- gene=840, id=235 removed by soft count = 54\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 841\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=841, id=91 removed by soft count = 54\n",
      "--- gene=841, id=247 removed by soft count = 225\n",
      "--- gene=841, id=259 removed by soft count = 63\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 842\n",
      "--- gene=842, id=53 removed by soft count = 361\n",
      "--- gene=842, id=54 removed by soft count = 286\n",
      "--- gene=842, id=72 removed by soft count = 103\n",
      "--- gene=842, id=82 removed by soft count = 319\n",
      "--- gene=842, id=228 removed by soft count = 81\n",
      "--- gene=842, id=345 removed by soft count = 202\n",
      "--- gene=842, id=346 removed by soft count = 121\n",
      "--- gene=842, id=347 removed by soft count = 245\n",
      "--- gene=842, id=348 removed by soft count = 192\n",
      "--- gene=842, id=349 removed by soft count = 172\n",
      "--- gene=842, id=350 removed by soft count = 162\n",
      "--- gene=842, id=351 removed by soft count = 215\n",
      "--- gene=842, id=354 removed by soft count = 142\n",
      "--- gene=842, id=373 removed by soft count = 229\n",
      "--- number of probes: 400 , kept by blast: 386 , if remove dups: 386\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 843\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 844\n",
      "--- gene=844, id=235 removed by soft count = 51\n",
      "--- gene=844, id=356 removed by soft count = 462\n",
      "--- gene=844, id=359 removed by soft count = 52\n",
      "--- gene=844, id=366 removed by soft count = 363\n",
      "--- gene=844, id=368 removed by soft count = 156\n",
      "--- gene=844, id=369 removed by soft count = 258\n",
      "--- gene=844, id=371 removed by soft count = 261\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 845\n",
      "--- gene=845, id=194 removed by soft count = 144\n",
      "--- gene=845, id=249 removed by soft count = 70\n",
      "--- gene=845, id=252 removed by soft count = 294\n",
      "--- gene=845, id=310 removed by soft count = 86\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 846\n",
      "--- gene=846, id=118 removed by soft count = 190\n",
      "--- gene=846, id=182 removed by soft count = 2317\n",
      "--- gene=846, id=191 removed by soft count = 84\n",
      "--- gene=846, id=342 removed by soft count = 779\n",
      "--- gene=846, id=370 removed by soft count = 151\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 847\n",
      "--- gene=847, id=7 removed by soft count = 93\n",
      "--- gene=847, id=48 removed by soft count = 68\n",
      "--- gene=847, id=127 removed by soft count = 19612\n",
      "--- gene=847, id=134 removed by soft count = 70\n",
      "--- gene=847, id=140 removed by soft count = 75\n",
      "--- gene=847, id=162 removed by soft count = 101\n",
      "--- gene=847, id=165 removed by soft count = 5882\n",
      "--- gene=847, id=166 removed by soft count = 242\n",
      "--- gene=847, id=171 removed by soft count = 73\n",
      "--- gene=847, id=175 removed by soft count = 375\n",
      "--- gene=847, id=178 removed by soft count = 795\n",
      "--- gene=847, id=179 removed by soft count = 773\n",
      "--- gene=847, id=180 removed by soft count = 690\n",
      "--- gene=847, id=206 removed by soft count = 52\n",
      "--- gene=847, id=314 removed by soft count = 92\n",
      "--- gene=847, id=319 removed by soft count = 158\n",
      "--- number of probes: 400 , kept by blast: 384 , if remove dups: 382\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 848\n",
      "--- gene=848, id=24 removed by soft count = 53\n",
      "--- gene=848, id=282 removed by soft count = 2537\n",
      "--- gene=848, id=293 removed by soft count = 75\n",
      "--- gene=848, id=327 removed by soft count = 121\n",
      "--- gene=848, id=360 removed by soft count = 223\n",
      "--- gene=848, id=361 removed by soft count = 129\n",
      "--- gene=848, id=376 removed by soft count = 53\n",
      "--- number of probes: 392 , kept by blast: 385 , if remove dups: 383\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 849\n",
      "--- gene=849, id=151 removed by soft count = 135\n",
      "--- gene=849, id=197 removed by soft count = 61\n",
      "--- gene=849, id=203 removed by soft count = 141\n",
      "--- gene=849, id=266 removed by soft count = 260\n",
      "--- gene=849, id=286 removed by soft count = 69\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 850\n",
      "--- gene=850, id=0 removed by soft count = 123\n",
      "--- gene=850, id=5 removed by soft count = 599\n",
      "--- gene=850, id=14 removed by soft count = 152\n",
      "--- gene=850, id=32 removed by soft count = 69\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 851\n",
      "--- gene=851, id=25 removed by soft count = 65\n",
      "--- gene=851, id=27 removed by soft count = 167\n",
      "--- gene=851, id=28 removed by soft count = 199\n",
      "--- gene=851, id=337 removed by soft count = 133\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 852\n",
      "--- gene=852, id=264 removed by soft count = 76\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 853\n",
      "--- gene=853, id=74 removed by soft count = 1436\n",
      "--- gene=853, id=75 removed by soft count = 462\n",
      "--- gene=853, id=76 removed by soft count = 1782\n",
      "--- gene=853, id=77 removed by soft count = 1017\n",
      "--- gene=853, id=211 removed by soft count = 53\n",
      "--- gene=853, id=231 removed by soft count = 63\n",
      "--- gene=853, id=280 removed by soft count = 90\n",
      "--- gene=853, id=334 removed by soft count = 508\n",
      "--- gene=853, id=335 removed by soft count = 144\n",
      "--- number of probes: 400 , kept by blast: 391 , if remove dups: 391\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 854\n",
      "--- gene=854, id=39 removed by soft count = 3831\n",
      "--- gene=854, id=41 removed by soft count = 944\n",
      "--- gene=854, id=42 removed by soft count = 759\n",
      "--- gene=854, id=46 removed by soft count = 69\n",
      "--- gene=854, id=47 removed by soft count = 221\n",
      "--- gene=854, id=66 removed by soft count = 249\n",
      "--- gene=854, id=68 removed by soft count = 273\n",
      "--- gene=854, id=70 removed by soft count = 181\n",
      "--- gene=854, id=120 removed by soft count = 116\n",
      "--- gene=854, id=288 removed by soft count = 104\n",
      "--- gene=854, id=341 removed by soft count = 285\n",
      "--- gene=854, id=346 removed by soft count = 112\n",
      "--- gene=854, id=347 removed by soft count = 56\n",
      "--- gene=854, id=358 removed by soft count = 144\n",
      "--- gene=854, id=359 removed by soft count = 58\n",
      "--- gene=854, id=370 removed by soft count = 90\n",
      "--- gene=854, id=375 removed by soft count = 84\n",
      "--- gene=854, id=376 removed by soft count = 58\n",
      "--- gene=854, id=397 removed by soft count = 3665\n",
      "--- number of probes: 400 , kept by blast: 381 , if remove dups: 381\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 855\n",
      "--- gene=855, id=1 removed by soft count = 55\n",
      "--- gene=855, id=2 removed by soft count = 131\n",
      "--- gene=855, id=75 removed by soft count = 62\n",
      "--- gene=855, id=77 removed by soft count = 118\n",
      "--- gene=855, id=382 removed by soft count = 90\n",
      "--- gene=855, id=392 removed by soft count = 88\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 856\n",
      "--- gene=856, id=113 removed by soft count = 68\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 857\n",
      "--- gene=857, id=269 removed by soft count = 101\n",
      "--- gene=857, id=270 removed by soft count = 54\n",
      "--- gene=857, id=271 removed by soft count = 97\n",
      "--- gene=857, id=365 removed by soft count = 141\n",
      "--- gene=857, id=376 removed by soft count = 59\n",
      "--- gene=857, id=381 removed by soft count = 126\n",
      "--- gene=857, id=382 removed by soft count = 57\n",
      "--- gene=857, id=385 removed by soft count = 560\n",
      "--- gene=857, id=386 removed by soft count = 94\n",
      "--- number of probes: 400 , kept by blast: 391 , if remove dups: 391\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 858\n",
      "--- gene=858, id=21 removed by soft count = 62\n",
      "--- gene=858, id=26 removed by soft count = 96\n",
      "--- gene=858, id=57 removed by soft count = 532\n",
      "--- gene=858, id=58 removed by soft count = 514\n",
      "--- gene=858, id=149 removed by soft count = 61\n",
      "--- gene=858, id=150 removed by soft count = 568\n",
      "--- gene=858, id=157 removed by soft count = 96\n",
      "--- gene=858, id=160 removed by soft count = 207\n",
      "--- gene=858, id=208 removed by soft count = 580\n",
      "--- gene=858, id=318 removed by soft count = 311\n",
      "--- gene=858, id=321 removed by soft count = 79\n",
      "--- gene=858, id=367 removed by soft count = 74\n",
      "--- gene=858, id=368 removed by soft count = 56\n",
      "--- number of probes: 400 , kept by blast: 387 , if remove dups: 387\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 859\n",
      "--- gene=859, id=51 removed by soft count = 204\n",
      "--- gene=859, id=278 removed by soft count = 90\n",
      "--- gene=859, id=279 removed by soft count = 52\n",
      "--- gene=859, id=280 removed by soft count = 170\n",
      "--- gene=859, id=281 removed by soft count = 76\n",
      "--- gene=859, id=288 removed by soft count = 80\n",
      "--- gene=859, id=290 removed by soft count = 1071\n",
      "--- gene=859, id=291 removed by soft count = 214\n",
      "--- gene=859, id=389 removed by soft count = 2282\n",
      "--- number of probes: 400 , kept by blast: 391 , if remove dups: 391\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 860\n",
      "--- gene=860, id=0 removed by soft count = 80\n",
      "--- gene=860, id=108 removed by soft count = 95\n",
      "--- gene=860, id=110 removed by soft count = 56\n",
      "--- gene=860, id=117 removed by soft count = 104\n",
      "--- gene=860, id=118 removed by soft count = 60\n",
      "--- gene=860, id=123 removed by soft count = 138\n",
      "--- gene=860, id=139 removed by soft count = 96\n",
      "--- gene=860, id=191 removed by soft count = 211\n",
      "--- gene=860, id=230 removed by soft count = 108\n",
      "--- gene=860, id=246 removed by hard count = 3\n",
      "--- number of probes: 399 , kept by blast: 389 , if remove dups: 387\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 861\n",
      "--- gene=861, id=264 removed by soft count = 66\n",
      "--- gene=861, id=265 removed by soft count = 103\n",
      "--- gene=861, id=317 removed by soft count = 131\n",
      "--- gene=861, id=318 removed by soft count = 60\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 862\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 863\n",
      "--- gene=863, id=127 removed by soft count = 118\n",
      "--- gene=863, id=204 removed by soft count = 524\n",
      "--- gene=863, id=223 removed by soft count = 72\n",
      "--- gene=863, id=288 removed by soft count = 212\n",
      "--- gene=863, id=303 removed by soft count = 110\n",
      "--- gene=863, id=370 removed by soft count = 71\n",
      "--- gene=863, id=375 removed by soft count = 408\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 864\n",
      "--- gene=864, id=16 removed by soft count = 224\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 866\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 867\n",
      "--- gene=867, id=7 removed by soft count = 1082\n",
      "--- gene=867, id=334 removed by soft count = 389\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 868\n",
      "--- gene=868, id=71 removed by soft count = 66\n",
      "--- gene=868, id=124 removed by soft count = 51\n",
      "--- gene=868, id=398 removed by soft count = 242\n",
      "--- gene=868, id=399 removed by soft count = 72\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 869\n",
      "--- gene=869, id=39 removed by soft count = 394\n",
      "--- gene=869, id=368 removed by soft count = 156\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 870\n",
      "--- gene=870, id=305 removed by hard count = 0\n",
      "--- gene=870, id=374 removed by soft count = 845\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 871\n",
      "--- gene=871, id=73 removed by soft count = 1255\n",
      "--- gene=871, id=74 removed by soft count = 352\n",
      "--- gene=871, id=86 removed by soft count = 548\n",
      "--- gene=871, id=107 removed by soft count = 309\n",
      "--- gene=871, id=110 removed by soft count = 61\n",
      "--- gene=871, id=220 removed by soft count = 96\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 872\n",
      "--- gene=872, id=215 removed by soft count = 317\n",
      "--- number of probes: 398 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 873\n",
      "--- gene=873, id=207 removed by soft count = 126\n",
      "--- gene=873, id=333 removed by soft count = 111\n",
      "--- gene=873, id=334 removed by soft count = 82\n",
      "--- gene=873, id=336 removed by soft count = 381\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 874\n",
      "--- gene=874, id=76 removed by soft count = 66\n",
      "--- gene=874, id=79 removed by soft count = 63\n",
      "--- gene=874, id=213 removed by soft count = 285\n",
      "--- gene=874, id=272 removed by soft count = 83\n",
      "--- gene=874, id=337 removed by soft count = 2089\n",
      "--- gene=874, id=338 removed by soft count = 188\n",
      "--- gene=874, id=343 removed by soft count = 85\n",
      "--- gene=874, id=344 removed by soft count = 251\n",
      "--- gene=874, id=345 removed by soft count = 52\n",
      "--- gene=874, id=347 removed by soft count = 84\n",
      "--- gene=874, id=357 removed by soft count = 78\n",
      "--- gene=874, id=358 removed by soft count = 58\n",
      "--- number of probes: 400 , kept by blast: 388 , if remove dups: 387\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 875\n",
      "--- gene=875, id=82 removed by soft count = 72\n",
      "--- gene=875, id=86 removed by soft count = 135\n",
      "--- gene=875, id=87 removed by soft count = 55\n",
      "--- gene=875, id=88 removed by soft count = 62\n",
      "--- gene=875, id=104 removed by soft count = 117\n",
      "--- gene=875, id=343 removed by soft count = 85\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 876\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=876, id=32 removed by soft count = 311\n",
      "--- gene=876, id=41 removed by soft count = 232\n",
      "--- gene=876, id=268 removed by soft count = 121\n",
      "--- gene=876, id=335 removed by soft count = 226\n",
      "--- gene=876, id=342 removed by soft count = 63\n",
      "--- number of probes: 399 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 877\n",
      "--- gene=877, id=61 removed by soft count = 94\n",
      "--- gene=877, id=65 removed by soft count = 73\n",
      "--- gene=877, id=326 removed by soft count = 181\n",
      "--- number of probes: 396 , kept by blast: 393 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 878\n",
      "--- gene=878, id=9 removed by soft count = 68\n",
      "--- gene=878, id=62 removed by soft count = 91\n",
      "--- gene=878, id=178 removed by soft count = 141\n",
      "--- gene=878, id=179 removed by soft count = 85\n",
      "--- gene=878, id=281 removed by soft count = 83\n",
      "--- gene=878, id=328 removed by soft count = 479\n",
      "--- gene=878, id=351 removed by soft count = 67\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 879\n",
      "--- gene=879, id=35 removed by soft count = 1760\n",
      "--- gene=879, id=85 removed by soft count = 116\n",
      "--- gene=879, id=140 removed by soft count = 140\n",
      "--- gene=879, id=201 removed by soft count = 52\n",
      "--- gene=879, id=290 removed by soft count = 53\n",
      "--- gene=879, id=343 removed by soft count = 56\n",
      "--- gene=879, id=366 removed by soft count = 80\n",
      "--- gene=879, id=367 removed by soft count = 127\n",
      "--- number of probes: 400 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 880\n",
      "--- gene=880, id=52 removed by soft count = 367\n",
      "--- gene=880, id=53 removed by soft count = 74\n",
      "--- gene=880, id=173 removed by soft count = 117\n",
      "--- gene=880, id=223 removed by soft count = 75\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 881\n",
      "--- gene=881, id=252 removed by soft count = 1953\n",
      "--- gene=881, id=253 removed by soft count = 278\n",
      "--- gene=881, id=254 removed by soft count = 53\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 882\n",
      "--- gene=882, id=14 removed by soft count = 57\n",
      "--- gene=882, id=361 removed by soft count = 75\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 883\n",
      "--- gene=883, id=155 removed by soft count = 2433\n",
      "--- gene=883, id=159 removed by soft count = 51\n",
      "--- gene=883, id=195 removed by soft count = 142\n",
      "--- gene=883, id=291 removed by soft count = 104\n",
      "--- gene=883, id=394 removed by soft count = 150\n",
      "--- gene=883, id=395 removed by soft count = 289\n",
      "--- number of probes: 399 , kept by blast: 393 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 884\n",
      "--- gene=884, id=103 removed by soft count = 860\n",
      "--- gene=884, id=162 removed by soft count = 120\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 885\n",
      "--- gene=885, id=36 removed by soft count = 524\n",
      "--- gene=885, id=98 removed by soft count = 393\n",
      "--- gene=885, id=129 removed by hard count = 5\n",
      "--- gene=885, id=190 removed by soft count = 130\n",
      "--- gene=885, id=393 removed by soft count = 247\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 886\n",
      "--- gene=886, id=178 removed by soft count = 104\n",
      "--- gene=886, id=341 removed by soft count = 102\n",
      "--- gene=886, id=344 removed by soft count = 122\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 887\n",
      "--- gene=887, id=24 removed by soft count = 245\n",
      "--- gene=887, id=27 removed by soft count = 177\n",
      "--- gene=887, id=28 removed by soft count = 209\n",
      "--- gene=887, id=29 removed by soft count = 131\n",
      "--- gene=887, id=30 removed by soft count = 120\n",
      "--- gene=887, id=31 removed by soft count = 217\n",
      "--- gene=887, id=33 removed by soft count = 109\n",
      "--- gene=887, id=35 removed by soft count = 288\n",
      "--- gene=887, id=36 removed by soft count = 351\n",
      "--- gene=887, id=37 removed by soft count = 358\n",
      "--- gene=887, id=38 removed by soft count = 101\n",
      "--- gene=887, id=173 removed by soft count = 150\n",
      "--- gene=887, id=174 removed by soft count = 132\n",
      "--- gene=887, id=187 removed by soft count = 53\n",
      "--- number of probes: 400 , kept by blast: 386 , if remove dups: 375\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 888\n",
      "--- gene=888, id=45 removed by soft count = 101\n",
      "--- gene=888, id=72 removed by soft count = 657\n",
      "--- gene=888, id=209 removed by soft count = 76\n",
      "--- gene=888, id=216 removed by soft count = 123\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 889\n",
      "--- gene=889, id=197 removed by soft count = 151\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 890\n",
      "--- gene=890, id=6 removed by soft count = 72\n",
      "--- gene=890, id=20 removed by soft count = 78\n",
      "--- gene=890, id=146 removed by soft count = 110\n",
      "--- gene=890, id=195 removed by soft count = 168\n",
      "--- gene=890, id=196 removed by soft count = 54\n",
      "--- gene=890, id=197 removed by soft count = 114\n",
      "--- gene=890, id=198 removed by soft count = 111\n",
      "--- gene=890, id=199 removed by soft count = 128\n",
      "--- gene=890, id=200 removed by soft count = 571\n",
      "--- gene=890, id=348 removed by soft count = 71\n",
      "--- gene=890, id=362 removed by soft count = 61\n",
      "--- number of probes: 400 , kept by blast: 389 , if remove dups: 388\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 891\n",
      "--- gene=891, id=111 removed by soft count = 77\n",
      "--- gene=891, id=150 removed by soft count = 301\n",
      "--- gene=891, id=152 removed by soft count = 237\n",
      "--- gene=891, id=170 removed by soft count = 62\n",
      "--- gene=891, id=396 removed by soft count = 757\n",
      "--- number of probes: 398 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 892\n",
      "--- gene=892, id=162 removed by soft count = 70\n",
      "--- gene=892, id=182 removed by soft count = 66\n",
      "--- gene=892, id=184 removed by soft count = 54\n",
      "--- gene=892, id=348 removed by soft count = 66\n",
      "--- gene=892, id=350 removed by soft count = 317\n",
      "--- gene=892, id=351 removed by soft count = 268\n",
      "--- gene=892, id=353 removed by soft count = 58\n",
      "--- gene=892, id=363 removed by soft count = 119\n",
      "--- gene=892, id=377 removed by soft count = 60\n",
      "--- number of probes: 400 , kept by blast: 391 , if remove dups: 391\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 893\n",
      "--- gene=893, id=51 removed by soft count = 79\n",
      "--- gene=893, id=197 removed by soft count = 54\n",
      "--- gene=893, id=373 removed by soft count = 131\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 894\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=894, id=54 removed by soft count = 56\n",
      "--- gene=894, id=64 removed by soft count = 74\n",
      "--- gene=894, id=261 removed by soft count = 61\n",
      "--- gene=894, id=268 removed by soft count = 114\n",
      "--- gene=894, id=363 removed by soft count = 247\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 895\n",
      "--- gene=895, id=12 removed by soft count = 53\n",
      "--- gene=895, id=14 removed by soft count = 61\n",
      "--- gene=895, id=16 removed by soft count = 115\n",
      "--- gene=895, id=17 removed by soft count = 186\n",
      "--- gene=895, id=60 removed by soft count = 54\n",
      "--- gene=895, id=75 removed by soft count = 83\n",
      "--- gene=895, id=295 removed by soft count = 64\n",
      "--- gene=895, id=296 removed by soft count = 206\n",
      "--- gene=895, id=298 removed by soft count = 60\n",
      "--- gene=895, id=325 removed by soft count = 125\n",
      "--- gene=895, id=328 removed by soft count = 61\n",
      "--- number of probes: 400 , kept by blast: 389 , if remove dups: 389\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 896\n",
      "--- gene=896, id=48 removed by soft count = 92\n",
      "--- gene=896, id=157 removed by soft count = 209\n",
      "--- gene=896, id=169 removed by soft count = 77\n",
      "--- gene=896, id=174 removed by soft count = 78\n",
      "--- gene=896, id=237 removed by soft count = 51\n",
      "--- gene=896, id=259 removed by soft count = 1270\n",
      "--- gene=896, id=297 removed by soft count = 66\n",
      "--- gene=896, id=358 removed by soft count = 88\n",
      "--- number of probes: 398 , kept by blast: 390 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 897\n",
      "--- gene=897, id=26 removed by soft count = 57\n",
      "--- gene=897, id=40 removed by soft count = 57\n",
      "--- gene=897, id=50 removed by soft count = 56\n",
      "--- gene=897, id=111 removed by soft count = 55\n",
      "--- gene=897, id=182 removed by soft count = 97\n",
      "--- gene=897, id=190 removed by soft count = 81\n",
      "--- gene=897, id=191 removed by soft count = 160\n",
      "--- gene=897, id=285 removed by soft count = 65\n",
      "--- gene=897, id=349 removed by soft count = 52\n",
      "--- gene=897, id=398 removed by soft count = 75\n",
      "--- number of probes: 400 , kept by blast: 390 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 898\n",
      "--- gene=898, id=160 removed by soft count = 81\n",
      "--- gene=898, id=162 removed by soft count = 70\n",
      "--- gene=898, id=364 removed by soft count = 72\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 899\n",
      "--- gene=899, id=10 removed by soft count = 71\n",
      "--- gene=899, id=151 removed by soft count = 73\n",
      "--- gene=899, id=225 removed by soft count = 135\n",
      "--- gene=899, id=322 removed by soft count = 89\n",
      "--- gene=899, id=336 removed by soft count = 135\n",
      "--- gene=899, id=379 removed by soft count = 113\n",
      "--- gene=899, id=382 removed by soft count = 72\n",
      "--- gene=899, id=384 removed by soft count = 63\n",
      "--- gene=899, id=394 removed by soft count = 58\n",
      "--- number of probes: 400 , kept by blast: 391 , if remove dups: 391\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 900\n",
      "--- gene=900, id=94 removed by soft count = 118\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 901\n",
      "--- gene=901, id=40 removed by soft count = 56\n",
      "--- gene=901, id=97 removed by soft count = 243\n",
      "--- gene=901, id=101 removed by hard count = 3\n",
      "--- gene=901, id=102 removed by hard count = 3\n",
      "--- gene=901, id=103 removed by hard count = 3\n",
      "--- gene=901, id=104 removed by hard count = 3\n",
      "--- gene=901, id=105 removed by hard count = 3\n",
      "--- gene=901, id=106 removed by hard count = 3\n",
      "--- gene=901, id=107 removed by hard count = 3\n",
      "--- gene=901, id=108 removed by hard count = 3\n",
      "--- gene=901, id=109 removed by hard count = 3\n",
      "--- gene=901, id=110 removed by hard count = 3\n",
      "--- gene=901, id=111 removed by hard count = 3\n",
      "--- gene=901, id=112 removed by hard count = 3\n",
      "--- gene=901, id=113 removed by hard count = 3\n",
      "--- gene=901, id=114 removed by hard count = 3\n",
      "--- gene=901, id=115 removed by hard count = 3\n",
      "--- gene=901, id=116 removed by hard count = 3\n",
      "--- gene=901, id=117 removed by hard count = 3\n",
      "--- gene=901, id=118 removed by hard count = 3\n",
      "--- gene=901, id=119 removed by hard count = 3\n",
      "--- gene=901, id=120 removed by hard count = 3\n",
      "--- gene=901, id=121 removed by hard count = 3\n",
      "--- gene=901, id=122 removed by hard count = 3\n",
      "--- gene=901, id=123 removed by hard count = 3\n",
      "--- gene=901, id=124 removed by hard count = 3\n",
      "--- gene=901, id=125 removed by hard count = 3\n",
      "--- gene=901, id=126 removed by hard count = 3\n",
      "--- gene=901, id=127 removed by hard count = 3\n",
      "--- gene=901, id=128 removed by hard count = 3\n",
      "--- gene=901, id=129 removed by hard count = 3\n",
      "--- gene=901, id=130 removed by hard count = 3\n",
      "--- gene=901, id=131 removed by hard count = 3\n",
      "--- gene=901, id=132 removed by hard count = 3\n",
      "--- gene=901, id=133 removed by hard count = 3\n",
      "--- gene=901, id=298 removed by soft count = 707\n",
      "--- number of probes: 400 , kept by blast: 364 , if remove dups: 362\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 902\n",
      "--- gene=902, id=69 removed by soft count = 51\n",
      "--- gene=902, id=70 removed by soft count = 74\n",
      "--- gene=902, id=75 removed by soft count = 58\n",
      "--- gene=902, id=79 removed by soft count = 60\n",
      "--- gene=902, id=321 removed by soft count = 106\n",
      "--- gene=902, id=376 removed by soft count = 320\n",
      "--- gene=902, id=377 removed by soft count = 264\n",
      "--- gene=902, id=378 removed by soft count = 332\n",
      "--- number of probes: 381 , kept by blast: 373 , if remove dups: 373\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 903\n",
      "--- gene=903, id=26 removed by soft count = 110\n",
      "--- gene=903, id=165 removed by soft count = 624\n",
      "--- gene=903, id=226 removed by soft count = 58\n",
      "--- gene=903, id=227 removed by soft count = 60\n",
      "--- gene=903, id=228 removed by soft count = 147\n",
      "--- gene=903, id=229 removed by soft count = 60\n",
      "--- gene=903, id=363 removed by soft count = 117\n",
      "--- gene=903, id=386 removed by soft count = 52\n",
      "--- number of probes: 395 , kept by blast: 387 , if remove dups: 387\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 904\n",
      "--- gene=904, id=61 removed by soft count = 79\n",
      "--- gene=904, id=76 removed by soft count = 122\n",
      "--- gene=904, id=95 removed by soft count = 55\n",
      "--- gene=904, id=167 removed by soft count = 103\n",
      "--- gene=904, id=169 removed by soft count = 61\n",
      "--- gene=904, id=170 removed by soft count = 393\n",
      "--- gene=904, id=186 removed by soft count = 53\n",
      "--- gene=904, id=201 removed by soft count = 242\n",
      "--- gene=904, id=217 removed by soft count = 113\n",
      "--- gene=904, id=219 removed by soft count = 994\n",
      "--- gene=904, id=221 removed by soft count = 616\n",
      "--- gene=904, id=360 removed by soft count = 97\n",
      "--- gene=904, id=395 removed by soft count = 82\n",
      "--- gene=904, id=397 removed by soft count = 54\n",
      "--- number of probes: 400 , kept by blast: 386 , if remove dups: 386\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 905\n",
      "--- gene=905, id=93 removed by soft count = 712\n",
      "--- number of probes: 399 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 906\n",
      "--- gene=906, id=386 removed by soft count = 4618\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 907\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=907, id=220 removed by soft count = 58\n",
      "--- gene=907, id=345 removed by soft count = 110\n",
      "--- number of probes: 399 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 908\n",
      "--- gene=908, id=329 removed by soft count = 275\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 909\n",
      "--- gene=909, id=1 removed by soft count = 155\n",
      "--- gene=909, id=146 removed by soft count = 150\n",
      "--- gene=909, id=255 removed by soft count = 9713\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 910\n",
      "--- gene=910, id=2 removed by soft count = 287\n",
      "--- gene=910, id=182 removed by soft count = 55\n",
      "--- gene=910, id=202 removed by soft count = 103\n",
      "--- gene=910, id=213 removed by soft count = 77\n",
      "--- gene=910, id=229 removed by soft count = 240\n",
      "--- gene=910, id=269 removed by soft count = 99\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 911\n",
      "--- gene=911, id=101 removed by soft count = 102\n",
      "--- gene=911, id=102 removed by soft count = 104\n",
      "--- gene=911, id=149 removed by soft count = 117\n",
      "--- gene=911, id=191 removed by soft count = 600\n",
      "--- gene=911, id=194 removed by soft count = 117\n",
      "--- gene=911, id=383 removed by soft count = 149\n",
      "--- gene=911, id=384 removed by soft count = 449\n",
      "--- gene=911, id=398 removed by soft count = 1049\n",
      "--- gene=911, id=399 removed by soft count = 51\n",
      "--- number of probes: 400 , kept by blast: 391 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 912\n",
      "--- gene=912, id=49 removed by soft count = 113\n",
      "--- gene=912, id=50 removed by soft count = 106\n",
      "--- gene=912, id=66 removed by soft count = 51\n",
      "--- gene=912, id=71 removed by soft count = 136\n",
      "--- gene=912, id=79 removed by soft count = 149\n",
      "--- gene=912, id=80 removed by soft count = 73\n",
      "--- gene=912, id=81 removed by soft count = 60\n",
      "--- gene=912, id=84 removed by soft count = 61\n",
      "--- gene=912, id=87 removed by soft count = 95\n",
      "--- gene=912, id=91 removed by soft count = 57\n",
      "--- gene=912, id=92 removed by soft count = 78\n",
      "--- gene=912, id=196 removed by hard count = 3\n",
      "--- gene=912, id=237 removed by soft count = 81\n",
      "--- gene=912, id=287 removed by soft count = 77\n",
      "--- gene=912, id=300 removed by soft count = 195\n",
      "--- gene=912, id=331 removed by soft count = 212\n",
      "--- gene=912, id=377 removed by soft count = 5915\n",
      "--- gene=912, id=378 removed by soft count = 174\n",
      "--- gene=912, id=379 removed by soft count = 249\n",
      "--- number of probes: 398 , kept by blast: 379 , if remove dups: 379\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 913\n",
      "--- gene=913, id=35 removed by soft count = 94\n",
      "--- gene=913, id=37 removed by soft count = 80\n",
      "--- gene=913, id=38 removed by soft count = 72\n",
      "--- gene=913, id=159 removed by soft count = 247\n",
      "--- gene=913, id=162 removed by soft count = 78\n",
      "--- gene=913, id=164 removed by soft count = 133\n",
      "--- gene=913, id=189 removed by soft count = 61\n",
      "--- gene=913, id=338 removed by hard count = 3\n",
      "--- number of probes: 398 , kept by blast: 390 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 914\n",
      "--- gene=914, id=27 removed by soft count = 249\n",
      "--- gene=914, id=29 removed by soft count = 55\n",
      "--- gene=914, id=30 removed by soft count = 127\n",
      "--- gene=914, id=31 removed by soft count = 151\n",
      "--- gene=914, id=38 removed by soft count = 102\n",
      "--- gene=914, id=115 removed by soft count = 74\n",
      "--- gene=914, id=116 removed by soft count = 56\n",
      "--- gene=914, id=164 removed by soft count = 254\n",
      "--- gene=914, id=171 removed by soft count = 283\n",
      "--- gene=914, id=317 removed by soft count = 61\n",
      "--- gene=914, id=318 removed by soft count = 96\n",
      "--- number of probes: 400 , kept by blast: 389 , if remove dups: 389\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 915\n",
      "--- number of probes: 396 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 916\n",
      "--- gene=916, id=82 removed by soft count = 145\n",
      "--- gene=916, id=84 removed by soft count = 241\n",
      "--- gene=916, id=210 removed by soft count = 61\n",
      "--- gene=916, id=213 removed by soft count = 951\n",
      "--- gene=916, id=215 removed by soft count = 1015\n",
      "--- gene=916, id=380 removed by soft count = 94\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 917\n",
      "--- gene=917, id=30 removed by soft count = 217\n",
      "--- gene=917, id=217 removed by soft count = 1025\n",
      "--- gene=917, id=219 removed by soft count = 51\n",
      "--- gene=917, id=221 removed by soft count = 120\n",
      "--- gene=917, id=231 removed by soft count = 94\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 918\n",
      "--- gene=918, id=164 removed by soft count = 87\n",
      "--- gene=918, id=165 removed by soft count = 480\n",
      "--- gene=918, id=167 removed by soft count = 136\n",
      "--- gene=918, id=177 removed by soft count = 57\n",
      "--- gene=918, id=182 removed by soft count = 434\n",
      "--- gene=918, id=252 removed by soft count = 97\n",
      "--- gene=918, id=253 removed by soft count = 98\n",
      "--- gene=918, id=289 removed by soft count = 502\n",
      "--- number of probes: 396 , kept by blast: 388 , if remove dups: 388\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 919\n",
      "--- gene=919, id=372 removed by soft count = 87\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 920\n",
      "--- gene=920, id=14 removed by soft count = 219\n",
      "--- gene=920, id=99 removed by soft count = 65\n",
      "--- gene=920, id=100 removed by soft count = 11150\n",
      "--- gene=920, id=101 removed by soft count = 3916\n",
      "--- gene=920, id=110 removed by soft count = 112\n",
      "--- gene=920, id=225 removed by soft count = 2107\n",
      "--- gene=920, id=226 removed by soft count = 147\n",
      "--- gene=920, id=342 removed by soft count = 139\n",
      "--- gene=920, id=344 removed by soft count = 177\n",
      "--- gene=920, id=358 removed by soft count = 55\n",
      "--- gene=920, id=359 removed by soft count = 115\n",
      "--- gene=920, id=364 removed by soft count = 57\n",
      "--- gene=920, id=385 removed by soft count = 51\n",
      "--- number of probes: 400 , kept by blast: 387 , if remove dups: 387\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 921\n",
      "--- gene=921, id=145 removed by soft count = 52\n",
      "--- gene=921, id=149 removed by soft count = 275\n",
      "--- number of probes: 396 , kept by blast: 394 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 922\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 923\n",
      "--- gene=923, id=160 removed by soft count = 510\n",
      "--- gene=923, id=240 removed by soft count = 62\n",
      "--- gene=923, id=247 removed by soft count = 220\n",
      "--- gene=923, id=248 removed by soft count = 1114\n",
      "--- gene=923, id=350 removed by soft count = 7561\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 925\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=925, id=150 removed by soft count = 68\n",
      "--- gene=925, id=237 removed by soft count = 187\n",
      "--- gene=925, id=250 removed by soft count = 298\n",
      "--- gene=925, id=346 removed by soft count = 156\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 926\n",
      "--- gene=926, id=35 removed by soft count = 53\n",
      "--- gene=926, id=40 removed by soft count = 1573\n",
      "--- gene=926, id=90 removed by soft count = 51\n",
      "--- gene=926, id=92 removed by soft count = 199\n",
      "--- gene=926, id=98 removed by soft count = 65\n",
      "--- gene=926, id=177 removed by soft count = 59\n",
      "--- gene=926, id=201 removed by soft count = 55\n",
      "--- gene=926, id=224 removed by soft count = 682\n",
      "--- gene=926, id=323 removed by soft count = 86\n",
      "--- gene=926, id=359 removed by soft count = 170\n",
      "--- gene=926, id=360 removed by soft count = 102\n",
      "--- number of probes: 400 , kept by blast: 389 , if remove dups: 389\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 927\n",
      "--- gene=927, id=20 removed by soft count = 387\n",
      "--- gene=927, id=21 removed by soft count = 116\n",
      "--- gene=927, id=122 removed by soft count = 109\n",
      "--- gene=927, id=176 removed by soft count = 143\n",
      "--- gene=927, id=233 removed by soft count = 55\n",
      "--- gene=927, id=281 removed by soft count = 178\n",
      "--- gene=927, id=330 removed by soft count = 291\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 928\n",
      "--- gene=928, id=32 removed by soft count = 56\n",
      "--- gene=928, id=134 removed by soft count = 149\n",
      "--- gene=928, id=141 removed by soft count = 252\n",
      "--- gene=928, id=156 removed by soft count = 81\n",
      "--- gene=928, id=182 removed by soft count = 61\n",
      "--- gene=928, id=249 removed by soft count = 1574\n",
      "--- gene=928, id=306 removed by soft count = 751\n",
      "--- gene=928, id=317 removed by soft count = 115\n",
      "--- gene=928, id=323 removed by soft count = 84\n",
      "--- number of probes: 400 , kept by blast: 391 , if remove dups: 387\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 929\n",
      "--- gene=929, id=24 removed by soft count = 267\n",
      "--- gene=929, id=112 removed by soft count = 85\n",
      "--- gene=929, id=113 removed by soft count = 99\n",
      "--- gene=929, id=114 removed by soft count = 232\n",
      "--- gene=929, id=116 removed by soft count = 69\n",
      "--- gene=929, id=371 removed by soft count = 146\n",
      "--- gene=929, id=392 removed by soft count = 285\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 930\n",
      "--- gene=930, id=14 removed by soft count = 62\n",
      "--- gene=930, id=94 removed by soft count = 209\n",
      "--- gene=930, id=158 removed by soft count = 99\n",
      "--- gene=930, id=159 removed by soft count = 82\n",
      "--- gene=930, id=382 removed by soft count = 172\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 931\n",
      "--- gene=931, id=39 removed by soft count = 68\n",
      "--- gene=931, id=40 removed by soft count = 77\n",
      "--- gene=931, id=98 removed by soft count = 189\n",
      "--- gene=931, id=99 removed by soft count = 158\n",
      "--- gene=931, id=100 removed by soft count = 77\n",
      "--- gene=931, id=101 removed by soft count = 328\n",
      "--- gene=931, id=123 removed by soft count = 52\n",
      "--- gene=931, id=202 removed by soft count = 58\n",
      "--- gene=931, id=268 removed by soft count = 63\n",
      "--- gene=931, id=345 removed by soft count = 91\n",
      "--- number of probes: 400 , kept by blast: 390 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 932\n",
      "--- gene=932, id=182 removed by soft count = 197\n",
      "--- gene=932, id=314 removed by soft count = 68\n",
      "--- gene=932, id=373 removed by soft count = 124\n",
      "--- gene=932, id=376 removed by soft count = 143\n",
      "--- gene=932, id=377 removed by soft count = 139\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 933\n",
      "--- gene=933, id=23 removed by soft count = 223\n",
      "--- gene=933, id=221 removed by soft count = 667\n",
      "--- gene=933, id=294 removed by soft count = 457\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 934\n",
      "--- gene=934, id=256 removed by soft count = 1759\n",
      "--- gene=934, id=266 removed by soft count = 63\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 935\n",
      "--- gene=935, id=130 removed by soft count = 99\n",
      "--- gene=935, id=131 removed by soft count = 274\n",
      "--- gene=935, id=241 removed by soft count = 89\n",
      "--- gene=935, id=340 removed by soft count = 254\n",
      "--- gene=935, id=349 removed by soft count = 110\n",
      "--- gene=935, id=362 removed by soft count = 75\n",
      "--- gene=935, id=371 removed by soft count = 65\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 936\n",
      "--- gene=936, id=10 removed by soft count = 65\n",
      "--- gene=936, id=219 removed by soft count = 126\n",
      "--- gene=936, id=352 removed by soft count = 1023\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 937\n",
      "--- gene=937, id=183 removed by soft count = 147\n",
      "--- gene=937, id=254 removed by soft count = 57\n",
      "--- gene=937, id=264 removed by soft count = 53\n",
      "--- gene=937, id=265 removed by soft count = 51\n",
      "--- gene=937, id=266 removed by soft count = 53\n",
      "--- gene=937, id=267 removed by soft count = 52\n",
      "--- gene=937, id=268 removed by soft count = 55\n",
      "--- number of probes: 397 , kept by blast: 390 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 938\n",
      "--- gene=938, id=243 removed by soft count = 9898\n",
      "--- gene=938, id=358 removed by soft count = 128\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 939\n",
      "--- gene=939, id=26 removed by soft count = 75\n",
      "--- gene=939, id=201 removed by soft count = 225\n",
      "--- gene=939, id=358 removed by soft count = 259\n",
      "--- gene=939, id=359 removed by soft count = 1369\n",
      "--- gene=939, id=360 removed by soft count = 338\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 940\n",
      "--- gene=940, id=28 removed by soft count = 89\n",
      "--- gene=940, id=57 removed by soft count = 91\n",
      "--- gene=940, id=70 removed by soft count = 179\n",
      "--- gene=940, id=105 removed by soft count = 97\n",
      "--- gene=940, id=106 removed by soft count = 70\n",
      "--- gene=940, id=107 removed by soft count = 110\n",
      "--- gene=940, id=108 removed by soft count = 316\n",
      "--- gene=940, id=109 removed by soft count = 2151\n",
      "--- gene=940, id=120 removed by soft count = 89\n",
      "--- gene=940, id=256 removed by soft count = 305\n",
      "--- number of probes: 400 , kept by blast: 390 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 941\n",
      "--- gene=941, id=45 removed by soft count = 62\n",
      "--- gene=941, id=102 removed by soft count = 121\n",
      "--- gene=941, id=151 removed by soft count = 165\n",
      "--- gene=941, id=152 removed by soft count = 80\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=941, id=249 removed by soft count = 155\n",
      "--- number of probes: 398 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 942\n",
      "--- gene=942, id=37 removed by soft count = 350\n",
      "--- gene=942, id=161 removed by soft count = 406\n",
      "--- gene=942, id=235 removed by soft count = 616\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 943\n",
      "--- gene=943, id=4 removed by soft count = 56\n",
      "--- gene=943, id=6 removed by soft count = 461\n",
      "--- gene=943, id=9 removed by soft count = 99\n",
      "--- gene=943, id=10 removed by soft count = 152\n",
      "--- gene=943, id=16 removed by soft count = 58\n",
      "--- gene=943, id=144 removed by soft count = 695\n",
      "--- gene=943, id=145 removed by soft count = 152\n",
      "--- gene=943, id=224 removed by soft count = 98\n",
      "--- gene=943, id=227 removed by soft count = 96\n",
      "--- gene=943, id=239 removed by soft count = 122\n",
      "--- gene=943, id=278 removed by soft count = 299\n",
      "--- gene=943, id=282 removed by soft count = 52\n",
      "--- gene=943, id=283 removed by soft count = 170\n",
      "--- gene=943, id=296 removed by soft count = 182\n",
      "--- gene=943, id=389 removed by soft count = 60\n",
      "--- number of probes: 400 , kept by blast: 385 , if remove dups: 384\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 944\n",
      "--- gene=944, id=164 removed by soft count = 114\n",
      "--- gene=944, id=248 removed by soft count = 61\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 945\n",
      "--- gene=945, id=166 removed by soft count = 109\n",
      "--- gene=945, id=169 removed by soft count = 74\n",
      "--- gene=945, id=191 removed by soft count = 187\n",
      "--- gene=945, id=227 removed by soft count = 86\n",
      "--- gene=945, id=243 removed by soft count = 55\n",
      "--- gene=945, id=288 removed by soft count = 79\n",
      "--- gene=945, id=318 removed by soft count = 85\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 946\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 947\n",
      "--- gene=947, id=49 removed by soft count = 122\n",
      "--- gene=947, id=236 removed by soft count = 166\n",
      "--- gene=947, id=237 removed by soft count = 73\n",
      "--- gene=947, id=361 removed by soft count = 72\n",
      "--- gene=947, id=372 removed by soft count = 126\n",
      "--- number of probes: 386 , kept by blast: 381 , if remove dups: 381\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 948\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 949\n",
      "--- gene=949, id=0 removed by soft count = 13143\n",
      "--- gene=949, id=16 removed by soft count = 182\n",
      "--- gene=949, id=68 removed by soft count = 2543\n",
      "--- gene=949, id=121 removed by soft count = 1860\n",
      "--- gene=949, id=158 removed by soft count = 14410\n",
      "--- gene=949, id=161 removed by soft count = 390\n",
      "--- gene=949, id=164 removed by soft count = 102\n",
      "--- gene=949, id=199 removed by soft count = 81\n",
      "--- gene=949, id=295 removed by soft count = 253\n",
      "--- gene=949, id=393 removed by soft count = 107\n",
      "--- number of probes: 400 , kept by blast: 390 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 950\n",
      "--- gene=950, id=108 removed by soft count = 60\n",
      "--- gene=950, id=200 removed by soft count = 52\n",
      "--- gene=950, id=263 removed by soft count = 425\n",
      "--- gene=950, id=267 removed by soft count = 123\n",
      "--- gene=950, id=298 removed by soft count = 52\n",
      "--- gene=950, id=329 removed by soft count = 166\n",
      "--- gene=950, id=332 removed by soft count = 97\n",
      "--- number of probes: 400 , kept by blast: 393 , if remove dups: 393\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 951\n",
      "--- gene=951, id=103 removed by soft count = 109\n",
      "--- gene=951, id=156 removed by soft count = 161\n",
      "--- gene=951, id=191 removed by soft count = 76\n",
      "--- gene=951, id=220 removed by soft count = 153\n",
      "--- gene=951, id=291 removed by soft count = 57\n",
      "--- gene=951, id=318 removed by soft count = 60\n",
      "--- gene=951, id=321 removed by soft count = 74\n",
      "--- gene=951, id=323 removed by soft count = 65\n",
      "--- number of probes: 400 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 952\n",
      "--- gene=952, id=70 removed by soft count = 1006\n",
      "--- gene=952, id=71 removed by soft count = 175\n",
      "--- number of probes: 394 , kept by blast: 392 , if remove dups: 391\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 953\n",
      "--- gene=953, id=50 removed by soft count = 133\n",
      "--- gene=953, id=93 removed by soft count = 140\n",
      "--- gene=953, id=247 removed by soft count = 733\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 954\n",
      "--- gene=954, id=223 removed by soft count = 131\n",
      "--- gene=954, id=302 removed by soft count = 645\n",
      "--- gene=954, id=366 removed by hard count = 0\n",
      "--- number of probes: 397 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 955\n",
      "--- gene=955, id=3 removed by soft count = 104\n",
      "--- gene=955, id=5 removed by soft count = 159\n",
      "--- gene=955, id=6 removed by soft count = 130\n",
      "--- gene=955, id=9 removed by soft count = 526\n",
      "--- gene=955, id=19 removed by soft count = 163\n",
      "--- gene=955, id=20 removed by soft count = 62\n",
      "--- gene=955, id=31 removed by soft count = 144\n",
      "--- gene=955, id=53 removed by soft count = 62\n",
      "--- gene=955, id=61 removed by soft count = 105\n",
      "--- gene=955, id=80 removed by soft count = 74\n",
      "--- gene=955, id=81 removed by soft count = 84\n",
      "--- gene=955, id=114 removed by soft count = 166\n",
      "--- gene=955, id=117 removed by soft count = 301\n",
      "--- gene=955, id=118 removed by soft count = 57\n",
      "--- gene=955, id=140 removed by soft count = 227\n",
      "--- gene=955, id=142 removed by soft count = 497\n",
      "--- gene=955, id=202 removed by soft count = 55\n",
      "--- gene=955, id=238 removed by soft count = 792\n",
      "--- gene=955, id=239 removed by soft count = 786\n",
      "--- gene=955, id=246 removed by soft count = 56\n",
      "--- gene=955, id=247 removed by soft count = 61\n",
      "--- gene=955, id=250 removed by soft count = 783\n",
      "--- gene=955, id=253 removed by soft count = 81\n",
      "--- number of probes: 400 , kept by blast: 377 , if remove dups: 377\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 956\n",
      "--- gene=956, id=5 removed by soft count = 165\n",
      "--- gene=956, id=12 removed by soft count = 51\n",
      "--- gene=956, id=234 removed by soft count = 68\n",
      "--- number of probes: 395 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 957\n",
      "--- gene=957, id=13 removed by soft count = 373\n",
      "--- gene=957, id=30 removed by soft count = 145\n",
      "--- gene=957, id=35 removed by soft count = 125\n",
      "--- gene=957, id=58 removed by soft count = 89\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 958\n",
      "--- gene=958, id=310 removed by soft count = 278\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 959\n",
      "--- gene=959, id=0 removed by soft count = 97\n",
      "--- gene=959, id=85 removed by soft count = 146\n",
      "--- gene=959, id=304 removed by soft count = 66\n",
      "--- gene=959, id=372 removed by soft count = 62\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 960\n",
      "--- gene=960, id=40 removed by soft count = 62\n",
      "--- gene=960, id=134 removed by soft count = 115\n",
      "--- gene=960, id=135 removed by soft count = 95\n",
      "--- gene=960, id=287 removed by soft count = 51\n",
      "--- gene=960, id=307 removed by soft count = 92\n",
      "--- number of probes: 399 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 961\n",
      "--- gene=961, id=37 removed by soft count = 348\n",
      "--- gene=961, id=46 removed by soft count = 349\n",
      "--- gene=961, id=47 removed by soft count = 76\n",
      "--- gene=961, id=48 removed by soft count = 293\n",
      "--- gene=961, id=134 removed by hard count = 4\n",
      "--- gene=961, id=325 removed by soft count = 70\n",
      "--- gene=961, id=326 removed by soft count = 74\n",
      "--- gene=961, id=327 removed by soft count = 63\n",
      "--- number of probes: 400 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 962\n",
      "--- gene=962, id=179 removed by soft count = 116\n",
      "--- gene=962, id=336 removed by soft count = 146\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 963\n",
      "--- gene=963, id=215 removed by soft count = 107\n",
      "--- gene=963, id=225 removed by soft count = 215\n",
      "--- gene=963, id=233 removed by soft count = 180\n",
      "--- gene=963, id=235 removed by soft count = 103\n",
      "--- gene=963, id=238 removed by soft count = 56\n",
      "--- gene=963, id=241 removed by soft count = 57\n",
      "--- gene=963, id=244 removed by soft count = 142\n",
      "--- gene=963, id=246 removed by soft count = 86\n",
      "--- gene=963, id=250 removed by soft count = 77\n",
      "--- gene=963, id=252 removed by soft count = 131\n",
      "--- gene=963, id=253 removed by soft count = 135\n",
      "--- gene=963, id=258 removed by soft count = 92\n",
      "--- gene=963, id=266 removed by soft count = 108\n",
      "--- gene=963, id=379 removed by soft count = 60\n",
      "--- number of probes: 400 , kept by blast: 386 , if remove dups: 386\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 964\n",
      "--- gene=964, id=155 removed by soft count = 107\n",
      "--- gene=964, id=384 removed by soft count = 137\n",
      "--- gene=964, id=386 removed by soft count = 92\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 965\n",
      "--- gene=965, id=12 removed by soft count = 66\n",
      "--- gene=965, id=99 removed by soft count = 59\n",
      "--- gene=965, id=105 removed by soft count = 54\n",
      "--- gene=965, id=197 removed by soft count = 62\n",
      "--- gene=965, id=203 removed by soft count = 53\n",
      "--- gene=965, id=204 removed by soft count = 57\n",
      "--- gene=965, id=343 removed by soft count = 88\n",
      "--- gene=965, id=374 removed by soft count = 1298\n",
      "--- number of probes: 400 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 966\n",
      "--- gene=966, id=0 removed by soft count = 514\n",
      "--- gene=966, id=178 removed by soft count = 83\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 967\n",
      "--- gene=967, id=103 removed by soft count = 118\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 968\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 969\n",
      "--- gene=969, id=15 removed by soft count = 1793\n",
      "--- gene=969, id=16 removed by soft count = 301\n",
      "--- gene=969, id=75 removed by soft count = 53\n",
      "--- gene=969, id=100 removed by soft count = 82\n",
      "--- gene=969, id=186 removed by soft count = 134\n",
      "--- gene=969, id=237 removed by soft count = 220\n",
      "--- gene=969, id=238 removed by soft count = 331\n",
      "--- gene=969, id=250 removed by soft count = 55\n",
      "--- gene=969, id=292 removed by soft count = 394\n",
      "--- gene=969, id=294 removed by soft count = 80\n",
      "--- gene=969, id=299 removed by soft count = 187\n",
      "--- number of probes: 400 , kept by blast: 389 , if remove dups: 389\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 970\n",
      "--- gene=970, id=319 removed by soft count = 83\n",
      "--- gene=970, id=323 removed by soft count = 61\n",
      "--- gene=970, id=324 removed by soft count = 85\n",
      "--- gene=970, id=326 removed by soft count = 79\n",
      "--- gene=970, id=370 removed by soft count = 66\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 971\n",
      "--- gene=971, id=74 removed by soft count = 68\n",
      "--- gene=971, id=86 removed by soft count = 278\n",
      "--- gene=971, id=87 removed by soft count = 1002\n",
      "--- gene=971, id=385 removed by soft count = 75\n",
      "--- gene=971, id=389 removed by soft count = 71\n",
      "--- number of probes: 400 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 972\n",
      "--- gene=972, id=146 removed by soft count = 116\n",
      "--- gene=972, id=148 removed by soft count = 55\n",
      "--- gene=972, id=150 removed by soft count = 193\n",
      "--- gene=972, id=162 removed by soft count = 125\n",
      "--- gene=972, id=163 removed by soft count = 355\n",
      "--- gene=972, id=165 removed by soft count = 51\n",
      "--- gene=972, id=266 removed by soft count = 54\n",
      "--- gene=972, id=279 removed by soft count = 174\n",
      "--- number of probes: 400 , kept by blast: 392 , if remove dups: 392\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 973\n",
      "--- gene=973, id=201 removed by soft count = 89\n",
      "--- gene=973, id=233 removed by soft count = 497\n",
      "--- gene=973, id=250 removed by soft count = 643\n",
      "--- gene=973, id=279 removed by soft count = 502\n",
      "--- gene=973, id=338 removed by soft count = 60\n",
      "--- gene=973, id=339 removed by soft count = 60\n",
      "--- gene=973, id=360 removed by soft count = 64\n",
      "--- gene=973, id=361 removed by soft count = 130\n",
      "--- gene=973, id=363 removed by soft count = 87\n",
      "--- number of probes: 400 , kept by blast: 391 , if remove dups: 389\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 974\n",
      "--- gene=974, id=18 removed by soft count = 70\n",
      "--- gene=974, id=21 removed by soft count = 1180\n",
      "--- gene=974, id=22 removed by soft count = 1144\n",
      "--- gene=974, id=23 removed by soft count = 66\n",
      "--- gene=974, id=25 removed by soft count = 118\n",
      "--- gene=974, id=26 removed by soft count = 187\n",
      "--- gene=974, id=27 removed by soft count = 2558\n",
      "--- gene=974, id=28 removed by soft count = 86\n",
      "--- gene=974, id=76 removed by soft count = 196\n",
      "--- gene=974, id=78 removed by soft count = 139\n",
      "--- gene=974, id=186 removed by soft count = 74\n",
      "--- gene=974, id=221 removed by soft count = 56\n",
      "--- gene=974, id=319 removed by soft count = 146\n",
      "--- number of probes: 400 , kept by blast: 387 , if remove dups: 387\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 975\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=975, id=177 removed by soft count = 121\n",
      "--- gene=975, id=179 removed by soft count = 108\n",
      "--- gene=975, id=188 removed by soft count = 51\n",
      "--- gene=975, id=191 removed by soft count = 84\n",
      "--- gene=975, id=192 removed by soft count = 200\n",
      "--- gene=975, id=314 removed by soft count = 66\n",
      "--- number of probes: 400 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 976\n",
      "--- gene=976, id=54 removed by soft count = 1605\n",
      "--- gene=976, id=114 removed by soft count = 367\n",
      "--- gene=976, id=115 removed by soft count = 186\n",
      "--- gene=976, id=183 removed by soft count = 209\n",
      "--- number of probes: 400 , kept by blast: 396 , if remove dups: 396\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 977\n",
      "--- gene=977, id=66 removed by soft count = 57\n",
      "--- gene=977, id=160 removed by soft count = 71\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 978\n",
      "--- gene=978, id=12 removed by soft count = 126\n",
      "--- gene=978, id=44 removed by soft count = 253\n",
      "--- gene=978, id=329 removed by soft count = 54\n",
      "--- gene=978, id=383 removed by soft count = 277\n",
      "--- number of probes: 394 , kept by blast: 390 , if remove dups: 390\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 979\n",
      "--- gene=979, id=83 removed by soft count = 303\n",
      "--- gene=979, id=86 removed by soft count = 200\n",
      "--- gene=979, id=101 removed by soft count = 56\n",
      "--- gene=979, id=269 removed by soft count = 99\n",
      "--- number of probes: 399 , kept by blast: 395 , if remove dups: 395\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 980\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 981\n",
      "--- gene=981, id=382 removed by soft count = 148\n",
      "--- gene=981, id=387 removed by soft count = 486\n",
      "--- gene=981, id=393 removed by soft count = 171\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 982\n",
      "--- gene=982, id=191 removed by soft count = 61\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 983\n",
      "--- gene=983, id=58 removed by soft count = 84\n",
      "--- gene=983, id=94 removed by soft count = 70\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 984\n",
      "--- gene=984, id=113 removed by soft count = 213\n",
      "--- gene=984, id=117 removed by soft count = 868\n",
      "--- gene=984, id=353 removed by soft count = 51\n",
      "--- gene=984, id=391 removed by soft count = 176\n",
      "--- number of probes: 398 , kept by blast: 394 , if remove dups: 394\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 985\n",
      "--- gene=985, id=17 removed by soft count = 349\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 986\n",
      "--- number of probes: 400 , kept by blast: 400 , if remove dups: 400\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 987\n",
      "--- gene=987, id=139 removed by soft count = 59\n",
      "--- number of probes: 400 , kept by blast: 399 , if remove dups: 399\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 988\n",
      "--- gene=988, id=248 removed by soft count = 67\n",
      "--- gene=988, id=347 removed by soft count = 88\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 989\n",
      "--- gene=989, id=122 removed by soft count = 406\n",
      "--- gene=989, id=226 removed by soft count = 74\n",
      "--- number of probes: 400 , kept by blast: 398 , if remove dups: 398\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 990\n",
      "--- gene=990, id=111 removed by soft count = 293\n",
      "--- gene=990, id=290 removed by soft count = 26642\n",
      "--- gene=990, id=382 removed by soft count = 122\n",
      "--- number of probes: 400 , kept by blast: 397 , if remove dups: 397\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 991\n",
      "--- gene=991, id=49 removed by soft count = 73\n",
      "--- gene=991, id=59 removed by soft count = 207\n",
      "--- gene=991, id=125 removed by soft count = 222\n",
      "--- number of probes: 127 , kept by blast: 124 , if remove dups: 124\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "- summarize\n",
      "-- region: 251 , number of probes: 120\n",
      "-- region: 252 , number of probes: 120\n",
      "-- region: 253 , number of probes: 120\n",
      "-- region: 255 , number of probes: 120\n",
      "-- region: 256 , number of probes: 120\n",
      "-- region: 257 , number of probes: 120\n",
      "-- region: 258 , number of probes: 120\n",
      "-- region: 259 , number of probes: 120\n",
      "-- region: 260 , number of probes: 120\n",
      "-- region: 261 , number of probes: 120\n",
      "-- region: 262 , number of probes: 120\n",
      "-- region: 263 , number of probes: 120\n",
      "-- region: 264 , number of probes: 120\n",
      "-- region: 265 , number of probes: 120\n",
      "-- region: 266 , number of probes: 120\n",
      "-- region: 267 , number of probes: 120\n",
      "-- region: 268 , number of probes: 120\n",
      "-- region: 269 , number of probes: 120\n",
      "-- region: 270 , number of probes: 120\n",
      "-- region: 271 , number of probes: 120\n",
      "-- region: 272 , number of probes: 120\n",
      "-- region: 273 , number of probes: 120\n",
      "-- region: 274 , number of probes: 120\n",
      "-- region: 275 , number of probes: 120\n",
      "-- region: 277 , number of probes: 120\n",
      "-- region: 278 , number of probes: 120\n",
      "-- region: 279 , number of probes: 120\n",
      "-- region: 280 , number of probes: 120\n",
      "-- region: 281 , number of probes: 120\n",
      "-- region: 282 , number of probes: 120\n",
      "-- region: 283 , number of probes: 120\n",
      "-- region: 284 , number of probes: 120\n",
      "-- region: 285 , number of probes: 120\n",
      "-- region: 286 , number of probes: 120\n",
      "-- region: 287 , number of probes: 120\n",
      "-- region: 288 , number of probes: 120\n",
      "-- region: 289 , number of probes: 120\n",
      "-- region: 290 , number of probes: 120\n",
      "-- region: 291 , number of probes: 120\n",
      "-- region: 292 , number of probes: 120\n",
      "-- region: 293 , number of probes: 120\n",
      "-- region: 295 , number of probes: 120\n",
      "-- region: 296 , number of probes: 120\n",
      "-- region: 297 , number of probes: 120\n",
      "-- region: 298 , number of probes: 120\n",
      "-- region: 299 , number of probes: 120\n",
      "-- region: 300 , number of probes: 120\n",
      "-- region: 301 , number of probes: 120\n",
      "-- region: 302 , number of probes: 120\n",
      "-- region: 303 , number of probes: 120\n",
      "-- region: 304 , number of probes: 120\n",
      "-- region: 305 , number of probes: 120\n",
      "-- region: 306 , number of probes: 120\n",
      "-- region: 307 , number of probes: 120\n",
      "-- region: 308 , number of probes: 120\n",
      "-- region: 309 , number of probes: 120\n",
      "-- region: 310 , number of probes: 120\n",
      "-- region: 311 , number of probes: 120\n",
      "-- region: 312 , number of probes: 120\n",
      "-- region: 313 , number of probes: 120\n",
      "-- region: 314 , number of probes: 120\n",
      "-- region: 315 , number of probes: 120\n",
      "-- region: 316 , number of probes: 120\n",
      "-- region: 317 , number of probes: 120\n",
      "-- region: 318 , number of probes: 120\n",
      "-- region: 319 , number of probes: 120\n",
      "-- region: 320 , number of probes: 120\n",
      "-- region: 321 , number of probes: 120\n",
      "-- region: 322 , number of probes: 120\n",
      "-- region: 324 , number of probes: 120\n",
      "-- region: 325 , number of probes: 120\n",
      "-- region: 326 , number of probes: 120\n",
      "-- region: 327 , number of probes: 120\n",
      "-- region: 328 , number of probes: 120\n",
      "-- region: 329 , number of probes: 120\n",
      "-- region: 330 , number of probes: 120\n",
      "-- region: 332 , number of probes: 120\n",
      "-- region: 333 , number of probes: 120\n",
      "-- region: 334 , number of probes: 120\n",
      "-- region: 335 , number of probes: 120\n",
      "-- region: 336 , number of probes: 120\n",
      "-- region: 337 , number of probes: 120\n",
      "-- region: 338 , number of probes: 120\n",
      "-- region: 339 , number of probes: 120\n",
      "-- region: 340 , number of probes: 120\n",
      "-- region: 341 , number of probes: 120\n",
      "-- region: 342 , number of probes: 120\n",
      "-- region: 343 , number of probes: 120\n",
      "-- region: 344 , number of probes: 120\n",
      "-- region: 345 , number of probes: 120\n",
      "-- region: 347 , number of probes: 120\n",
      "-- region: 348 , number of probes: 120\n",
      "-- region: 349 , number of probes: 120\n",
      "-- region: 350 , number of probes: 120\n",
      "-- region: 351 , number of probes: 120\n",
      "-- region: 352 , number of probes: 120\n",
      "-- region: 353 , number of probes: 120\n",
      "-- region: 354 , number of probes: 120\n",
      "-- region: 355 , number of probes: 120\n",
      "-- region: 356 , number of probes: 120\n",
      "-- region: 357 , number of probes: 120\n",
      "-- region: 358 , number of probes: 120\n",
      "-- region: 359 , number of probes: 120\n",
      "-- region: 360 , number of probes: 120\n",
      "-- region: 361 , number of probes: 120\n",
      "-- region: 362 , number of probes: 120\n",
      "-- region: 363 , number of probes: 120\n",
      "-- region: 364 , number of probes: 120\n",
      "-- region: 365 , number of probes: 120\n",
      "-- region: 366 , number of probes: 120\n",
      "-- region: 367 , number of probes: 120\n",
      "-- region: 368 , number of probes: 120\n",
      "-- region: 369 , number of probes: 120\n",
      "-- region: 370 , number of probes: 120\n",
      "-- region: 371 , number of probes: 120\n",
      "-- region: 372 , number of probes: 120\n",
      "-- region: 373 , number of probes: 120\n",
      "-- region: 374 , number of probes: 120\n",
      "-- region: 375 , number of probes: 120\n",
      "-- region: 376 , number of probes: 120\n",
      "-- region: 377 , number of probes: 120\n",
      "-- region: 378 , number of probes: 120\n",
      "-- region: 379 , number of probes: 120\n",
      "-- region: 380 , number of probes: 120\n",
      "-- region: 381 , number of probes: 120\n",
      "-- region: 382 , number of probes: 120\n",
      "-- region: 383 , number of probes: 120\n",
      "-- region: 384 , number of probes: 120\n",
      "-- region: 385 , number of probes: 120\n",
      "-- region: 386 , number of probes: 120\n",
      "-- region: 387 , number of probes: 120\n",
      "-- region: 388 , number of probes: 120\n",
      "-- region: 389 , number of probes: 120\n",
      "-- region: 390 , number of probes: 120\n",
      "-- region: 391 , number of probes: 120\n",
      "-- region: 392 , number of probes: 120\n",
      "-- region: 393 , number of probes: 120\n",
      "-- region: 394 , number of probes: 120\n",
      "-- region: 395 , number of probes: 120\n",
      "-- region: 396 , number of probes: 120\n",
      "-- region: 397 , number of probes: 120\n",
      "-- region: 398 , number of probes: 120\n",
      "-- region: 399 , number of probes: 120\n",
      "-- region: 400 , number of probes: 120\n",
      "-- region: 401 , number of probes: 120\n",
      "-- region: 402 , number of probes: 73\n",
      "-- region: 403 , number of probes: 120\n",
      "-- region: 404 , number of probes: 120\n",
      "-- region: 405 , number of probes: 120\n",
      "-- region: 406 , number of probes: 120\n",
      "-- region: 407 , number of probes: 120\n",
      "-- region: 408 , number of probes: 120\n",
      "-- region: 409 , number of probes: 120\n",
      "-- region: 410 , number of probes: 120\n",
      "-- region: 411 , number of probes: 120\n",
      "-- region: 412 , number of probes: 120\n",
      "-- region: 413 , number of probes: 120\n",
      "-- region: 414 , number of probes: 120\n",
      "-- region: 415 , number of probes: 120\n",
      "-- region: 416 , number of probes: 120\n",
      "-- region: 417 , number of probes: 120\n",
      "-- region: 418 , number of probes: 120\n",
      "-- region: 419 , number of probes: 120\n",
      "-- region: 420 , number of probes: 120\n",
      "-- region: 421 , number of probes: 120\n",
      "-- region: 422 , number of probes: 120\n",
      "-- region: 423 , number of probes: 120\n",
      "-- region: 424 , number of probes: 120\n",
      "-- region: 425 , number of probes: 120\n",
      "-- region: 426 , number of probes: 120\n",
      "-- region: 427 , number of probes: 120\n",
      "-- region: 428 , number of probes: 120\n",
      "-- region: 429 , number of probes: 120\n",
      "-- region: 430 , number of probes: 120\n",
      "-- region: 431 , number of probes: 120\n",
      "-- region: 432 , number of probes: 120\n",
      "-- region: 433 , number of probes: 120\n",
      "-- region: 434 , number of probes: 120\n",
      "-- region: 435 , number of probes: 120\n",
      "-- region: 436 , number of probes: 120\n",
      "-- region: 437 , number of probes: 120\n",
      "-- region: 438 , number of probes: 120\n",
      "-- region: 439 , number of probes: 120\n",
      "-- region: 440 , number of probes: 120\n",
      "-- region: 441 , number of probes: 120\n",
      "-- region: 442 , number of probes: 120\n",
      "-- region: 444 , number of probes: 120\n",
      "-- region: 445 , number of probes: 120\n",
      "-- region: 446 , number of probes: 120\n",
      "-- region: 447 , number of probes: 120\n",
      "-- region: 448 , number of probes: 120\n",
      "-- region: 449 , number of probes: 120\n",
      "-- region: 450 , number of probes: 120\n",
      "-- region: 451 , number of probes: 120\n",
      "-- region: 452 , number of probes: 120\n",
      "-- region: 453 , number of probes: 120\n",
      "-- region: 454 , number of probes: 120\n",
      "-- region: 455 , number of probes: 120\n",
      "-- region: 456 , number of probes: 120\n",
      "-- region: 457 , number of probes: 120\n",
      "-- region: 458 , number of probes: 120\n",
      "-- region: 459 , number of probes: 120\n",
      "-- region: 460 , number of probes: 120\n",
      "-- region: 461 , number of probes: 120\n",
      "-- region: 462 , number of probes: 120\n",
      "-- region: 463 , number of probes: 120\n",
      "-- region: 464 , number of probes: 120\n",
      "-- region: 465 , number of probes: 120\n",
      "-- region: 466 , number of probes: 120\n",
      "-- region: 467 , number of probes: 120\n",
      "-- region: 468 , number of probes: 120\n",
      "-- region: 469 , number of probes: 120\n",
      "-- region: 470 , number of probes: 120\n",
      "-- region: 471 , number of probes: 120\n",
      "-- region: 472 , number of probes: 120\n",
      "-- region: 473 , number of probes: 120\n",
      "-- region: 474 , number of probes: 120\n",
      "-- region: 475 , number of probes: 120\n",
      "-- region: 476 , number of probes: 120\n",
      "-- region: 477 , number of probes: 120\n",
      "-- region: 478 , number of probes: 120\n",
      "-- region: 479 , number of probes: 120\n",
      "-- region: 480 , number of probes: 120\n",
      "-- region: 481 , number of probes: 120\n",
      "-- region: 482 , number of probes: 120\n",
      "-- region: 483 , number of probes: 120\n",
      "-- region: 484 , number of probes: 120\n",
      "-- region: 485 , number of probes: 120\n",
      "-- region: 486 , number of probes: 120\n",
      "-- region: 487 , number of probes: 120\n",
      "-- region: 488 , number of probes: 120\n",
      "-- region: 489 , number of probes: 120\n",
      "-- region: 491 , number of probes: 120\n",
      "-- region: 492 , number of probes: 120\n",
      "-- region: 493 , number of probes: 120\n",
      "-- region: 494 , number of probes: 120\n",
      "-- region: 495 , number of probes: 120\n",
      "-- region: 496 , number of probes: 120\n",
      "-- region: 497 , number of probes: 120\n",
      "-- region: 498 , number of probes: 120\n",
      "-- region: 499 , number of probes: 120\n",
      "-- region: 500 , number of probes: 120\n",
      "-- region: 501 , number of probes: 120\n",
      "-- region: 502 , number of probes: 120\n",
      "-- region: 503 , number of probes: 120\n",
      "-- region: 504 , number of probes: 120\n",
      "-- region: 505 , number of probes: 120\n",
      "-- region: 506 , number of probes: 120\n",
      "-- region: 507 , number of probes: 120\n",
      "-- region: 508 , number of probes: 120\n",
      "-- region: 509 , number of probes: 120\n",
      "-- region: 510 , number of probes: 120\n",
      "-- region: 511 , number of probes: 120\n",
      "-- region: 512 , number of probes: 120\n",
      "-- region: 513 , number of probes: 120\n",
      "-- region: 514 , number of probes: 120\n",
      "-- region: 515 , number of probes: 120\n",
      "-- region: 516 , number of probes: 120\n",
      "-- region: 517 , number of probes: 120\n",
      "-- region: 518 , number of probes: 120\n",
      "-- region: 519 , number of probes: 120\n",
      "-- region: 520 , number of probes: 120\n",
      "-- region: 521 , number of probes: 120\n",
      "-- region: 523 , number of probes: 120\n",
      "-- region: 524 , number of probes: 120\n",
      "-- region: 525 , number of probes: 120\n",
      "-- region: 526 , number of probes: 120\n",
      "-- region: 527 , number of probes: 120\n",
      "-- region: 528 , number of probes: 120\n",
      "-- region: 529 , number of probes: 120\n",
      "-- region: 530 , number of probes: 120\n",
      "-- region: 531 , number of probes: 120\n",
      "-- region: 532 , number of probes: 120\n",
      "-- region: 533 , number of probes: 120\n",
      "-- region: 534 , number of probes: 120\n",
      "-- region: 535 , number of probes: 120\n",
      "-- region: 536 , number of probes: 120\n",
      "-- region: 537 , number of probes: 120\n",
      "-- region: 538 , number of probes: 120\n",
      "-- region: 539 , number of probes: 120\n",
      "-- region: 540 , number of probes: 120\n",
      "-- region: 541 , number of probes: 120\n",
      "-- region: 542 , number of probes: 120\n",
      "-- region: 543 , number of probes: 120\n",
      "-- region: 544 , number of probes: 120\n",
      "-- region: 545 , number of probes: 120\n",
      "-- region: 546 , number of probes: 120\n",
      "-- region: 547 , number of probes: 120\n",
      "-- region: 548 , number of probes: 120\n",
      "-- region: 549 , number of probes: 120\n",
      "-- region: 550 , number of probes: 120\n",
      "-- region: 551 , number of probes: 120\n",
      "-- region: 552 , number of probes: 120\n",
      "-- region: 553 , number of probes: 120\n",
      "-- region: 554 , number of probes: 120\n",
      "-- region: 555 , number of probes: 120\n",
      "-- region: 556 , number of probes: 120\n",
      "-- region: 557 , number of probes: 120\n",
      "-- region: 558 , number of probes: 120\n",
      "-- region: 559 , number of probes: 120\n",
      "-- region: 560 , number of probes: 120\n",
      "-- region: 561 , number of probes: 120\n",
      "-- region: 562 , number of probes: 120\n",
      "-- region: 563 , number of probes: 120\n",
      "-- region: 564 , number of probes: 120\n",
      "-- region: 565 , number of probes: 120\n",
      "-- region: 566 , number of probes: 120\n",
      "-- region: 567 , number of probes: 120\n",
      "-- region: 568 , number of probes: 120\n",
      "-- region: 569 , number of probes: 120\n",
      "-- region: 570 , number of probes: 120\n",
      "-- region: 571 , number of probes: 120\n",
      "-- region: 572 , number of probes: 120\n",
      "-- region: 573 , number of probes: 120\n",
      "-- region: 574 , number of probes: 120\n",
      "-- region: 575 , number of probes: 120\n",
      "-- region: 576 , number of probes: 120\n",
      "-- region: 577 , number of probes: 120\n",
      "-- region: 579 , number of probes: 120\n",
      "-- region: 580 , number of probes: 120\n",
      "-- region: 581 , number of probes: 120\n",
      "-- region: 582 , number of probes: 120\n",
      "-- region: 583 , number of probes: 120\n",
      "-- region: 584 , number of probes: 120\n",
      "-- region: 585 , number of probes: 120\n",
      "-- region: 586 , number of probes: 120\n",
      "-- region: 587 , number of probes: 120\n",
      "-- region: 588 , number of probes: 120\n",
      "-- region: 589 , number of probes: 120\n",
      "-- region: 590 , number of probes: 120\n",
      "-- region: 591 , number of probes: 120\n",
      "-- region: 592 , number of probes: 120\n",
      "-- region: 593 , number of probes: 120\n",
      "-- region: 594 , number of probes: 120\n",
      "-- region: 595 , number of probes: 120\n",
      "-- region: 596 , number of probes: 120\n",
      "-- region: 597 , number of probes: 120\n",
      "-- region: 598 , number of probes: 120\n",
      "-- region: 599 , number of probes: 120\n",
      "-- region: 600 , number of probes: 120\n",
      "-- region: 601 , number of probes: 120\n",
      "-- region: 602 , number of probes: 120\n",
      "-- region: 603 , number of probes: 120\n",
      "-- region: 604 , number of probes: 120\n",
      "-- region: 605 , number of probes: 120\n",
      "-- region: 606 , number of probes: 120\n",
      "-- region: 607 , number of probes: 120\n",
      "-- region: 608 , number of probes: 120\n",
      "-- region: 609 , number of probes: 120\n",
      "-- region: 610 , number of probes: 120\n",
      "-- region: 611 , number of probes: 120\n",
      "-- region: 612 , number of probes: 120\n",
      "-- region: 613 , number of probes: 120\n",
      "-- region: 614 , number of probes: 120\n",
      "-- region: 615 , number of probes: 120\n",
      "-- region: 616 , number of probes: 120\n",
      "-- region: 617 , number of probes: 120\n",
      "-- region: 618 , number of probes: 120\n",
      "-- region: 619 , number of probes: 120\n",
      "-- region: 620 , number of probes: 120\n",
      "-- region: 621 , number of probes: 120\n",
      "-- region: 622 , number of probes: 120\n",
      "-- region: 623 , number of probes: 120\n",
      "-- region: 624 , number of probes: 120\n",
      "-- region: 625 , number of probes: 120\n",
      "-- region: 626 , number of probes: 120\n",
      "-- region: 627 , number of probes: 120\n",
      "-- region: 628 , number of probes: 120\n",
      "-- region: 629 , number of probes: 120\n",
      "-- region: 630 , number of probes: 120\n",
      "-- region: 631 , number of probes: 120\n",
      "-- region: 632 , number of probes: 120\n",
      "-- region: 633 , number of probes: 120\n",
      "-- region: 634 , number of probes: 120\n",
      "-- region: 635 , number of probes: 120\n",
      "-- region: 636 , number of probes: 120\n",
      "-- region: 637 , number of probes: 120\n",
      "-- region: 638 , number of probes: 120\n",
      "-- region: 639 , number of probes: 120\n",
      "-- region: 640 , number of probes: 120\n",
      "-- region: 641 , number of probes: 120\n",
      "-- region: 642 , number of probes: 120\n",
      "-- region: 643 , number of probes: 120\n",
      "-- region: 644 , number of probes: 120\n",
      "-- region: 645 , number of probes: 120\n",
      "-- region: 646 , number of probes: 120\n",
      "-- region: 647 , number of probes: 120\n",
      "-- region: 648 , number of probes: 120\n",
      "-- region: 649 , number of probes: 120\n",
      "-- region: 650 , number of probes: 120\n",
      "-- region: 651 , number of probes: 120\n",
      "-- region: 652 , number of probes: 120\n",
      "-- region: 653 , number of probes: 120\n",
      "-- region: 654 , number of probes: 120\n",
      "-- region: 655 , number of probes: 120\n",
      "-- region: 656 , number of probes: 120\n",
      "-- region: 657 , number of probes: 120\n",
      "-- region: 658 , number of probes: 120\n",
      "-- region: 659 , number of probes: 120\n",
      "-- region: 660 , number of probes: 120\n",
      "-- region: 661 , number of probes: 120\n",
      "-- region: 662 , number of probes: 120\n",
      "-- region: 663 , number of probes: 120\n",
      "-- region: 664 , number of probes: 120\n",
      "-- region: 665 , number of probes: 120\n",
      "-- region: 666 , number of probes: 120\n",
      "-- region: 667 , number of probes: 120\n",
      "-- region: 668 , number of probes: 120\n",
      "-- region: 669 , number of probes: 120\n",
      "-- region: 670 , number of probes: 120\n",
      "-- region: 671 , number of probes: 120\n",
      "-- region: 672 , number of probes: 120\n",
      "-- region: 673 , number of probes: 120\n",
      "-- region: 674 , number of probes: 120\n",
      "-- region: 675 , number of probes: 120\n",
      "-- region: 676 , number of probes: 120\n",
      "-- region: 677 , number of probes: 120\n",
      "-- region: 678 , number of probes: 120\n",
      "-- region: 679 , number of probes: 120\n",
      "-- region: 680 , number of probes: 120\n",
      "-- region: 681 , number of probes: 120\n",
      "-- region: 682 , number of probes: 120\n",
      "-- region: 683 , number of probes: 120\n",
      "-- region: 684 , number of probes: 120\n",
      "-- region: 685 , number of probes: 120\n",
      "-- region: 687 , number of probes: 120\n",
      "-- region: 688 , number of probes: 120\n",
      "-- region: 689 , number of probes: 120\n",
      "-- region: 690 , number of probes: 120\n",
      "-- region: 691 , number of probes: 120\n",
      "-- region: 692 , number of probes: 120\n",
      "-- region: 693 , number of probes: 120\n",
      "-- region: 694 , number of probes: 120\n",
      "-- region: 695 , number of probes: 120\n",
      "-- region: 696 , number of probes: 120\n",
      "-- region: 697 , number of probes: 120\n",
      "-- region: 698 , number of probes: 120\n",
      "-- region: 699 , number of probes: 120\n",
      "-- region: 700 , number of probes: 120\n",
      "-- region: 701 , number of probes: 120\n",
      "-- region: 702 , number of probes: 120\n",
      "-- region: 703 , number of probes: 120\n",
      "-- region: 704 , number of probes: 120\n",
      "-- region: 705 , number of probes: 120\n",
      "-- region: 706 , number of probes: 120\n",
      "-- region: 707 , number of probes: 120\n",
      "-- region: 708 , number of probes: 120\n",
      "-- region: 709 , number of probes: 120\n",
      "-- region: 710 , number of probes: 120\n",
      "-- region: 711 , number of probes: 120\n",
      "-- region: 712 , number of probes: 120\n",
      "-- region: 713 , number of probes: 120\n",
      "-- region: 714 , number of probes: 120\n",
      "-- region: 715 , number of probes: 120\n",
      "-- region: 716 , number of probes: 120\n",
      "-- region: 717 , number of probes: 120\n",
      "-- region: 718 , number of probes: 120\n",
      "-- region: 719 , number of probes: 120\n",
      "-- region: 720 , number of probes: 120\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- region: 721 , number of probes: 120\n",
      "-- region: 722 , number of probes: 120\n",
      "-- region: 723 , number of probes: 120\n",
      "-- region: 724 , number of probes: 120\n",
      "-- region: 725 , number of probes: 120\n",
      "-- region: 726 , number of probes: 120\n",
      "-- region: 727 , number of probes: 120\n",
      "-- region: 728 , number of probes: 120\n",
      "-- region: 729 , number of probes: 120\n",
      "-- region: 730 , number of probes: 120\n",
      "-- region: 731 , number of probes: 120\n",
      "-- region: 732 , number of probes: 120\n",
      "-- region: 733 , number of probes: 120\n",
      "-- region: 734 , number of probes: 120\n",
      "-- region: 735 , number of probes: 120\n",
      "-- region: 736 , number of probes: 120\n",
      "-- region: 737 , number of probes: 120\n",
      "-- region: 738 , number of probes: 120\n",
      "-- region: 739 , number of probes: 120\n",
      "-- region: 740 , number of probes: 120\n",
      "-- region: 741 , number of probes: 120\n",
      "-- region: 742 , number of probes: 120\n",
      "-- region: 743 , number of probes: 120\n",
      "-- region: 744 , number of probes: 120\n",
      "-- region: 745 , number of probes: 120\n",
      "-- region: 746 , number of probes: 120\n",
      "-- region: 747 , number of probes: 120\n",
      "-- region: 748 , number of probes: 120\n",
      "-- region: 749 , number of probes: 120\n",
      "-- region: 750 , number of probes: 10\n",
      "-- region: 1 , number of probes: 120\n",
      "-- region: 2 , number of probes: 120\n",
      "-- region: 3 , number of probes: 120\n",
      "-- region: 4 , number of probes: 120\n",
      "-- region: 5 , number of probes: 120\n",
      "-- region: 6 , number of probes: 120\n",
      "-- region: 7 , number of probes: 120\n",
      "-- region: 8 , number of probes: 120\n",
      "-- region: 9 , number of probes: 120\n",
      "-- region: 10 , number of probes: 120\n",
      "-- region: 11 , number of probes: 120\n",
      "-- region: 12 , number of probes: 120\n",
      "-- region: 13 , number of probes: 120\n",
      "-- region: 14 , number of probes: 120\n",
      "-- region: 15 , number of probes: 120\n",
      "-- region: 16 , number of probes: 120\n",
      "-- region: 17 , number of probes: 120\n",
      "-- region: 18 , number of probes: 120\n",
      "-- region: 19 , number of probes: 120\n",
      "-- region: 20 , number of probes: 120\n",
      "-- region: 21 , number of probes: 120\n",
      "-- region: 22 , number of probes: 120\n",
      "-- region: 23 , number of probes: 120\n",
      "-- region: 24 , number of probes: 120\n",
      "-- region: 25 , number of probes: 120\n",
      "-- region: 26 , number of probes: 120\n",
      "-- region: 27 , number of probes: 120\n",
      "-- region: 28 , number of probes: 120\n",
      "-- region: 29 , number of probes: 120\n",
      "-- region: 30 , number of probes: 120\n",
      "-- region: 31 , number of probes: 120\n",
      "-- region: 32 , number of probes: 120\n",
      "-- region: 33 , number of probes: 120\n",
      "-- region: 34 , number of probes: 120\n",
      "-- region: 35 , number of probes: 120\n",
      "-- region: 36 , number of probes: 120\n",
      "-- region: 37 , number of probes: 120\n",
      "-- region: 38 , number of probes: 120\n",
      "-- region: 39 , number of probes: 120\n",
      "-- region: 40 , number of probes: 120\n",
      "-- region: 41 , number of probes: 120\n",
      "-- region: 42 , number of probes: 120\n",
      "-- region: 43 , number of probes: 120\n",
      "-- region: 44 , number of probes: 120\n",
      "-- region: 45 , number of probes: 120\n",
      "-- region: 46 , number of probes: 120\n",
      "-- region: 47 , number of probes: 120\n",
      "-- region: 48 , number of probes: 120\n",
      "-- region: 49 , number of probes: 120\n",
      "-- region: 50 , number of probes: 120\n",
      "-- region: 51 , number of probes: 120\n",
      "-- region: 52 , number of probes: 120\n",
      "-- region: 53 , number of probes: 120\n",
      "-- region: 54 , number of probes: 120\n",
      "-- region: 55 , number of probes: 120\n",
      "-- region: 56 , number of probes: 120\n",
      "-- region: 57 , number of probes: 120\n",
      "-- region: 58 , number of probes: 120\n",
      "-- region: 59 , number of probes: 120\n",
      "-- region: 60 , number of probes: 120\n",
      "-- region: 61 , number of probes: 120\n",
      "-- region: 62 , number of probes: 120\n",
      "-- region: 63 , number of probes: 120\n",
      "-- region: 64 , number of probes: 120\n",
      "-- region: 65 , number of probes: 120\n",
      "-- region: 66 , number of probes: 120\n",
      "-- region: 67 , number of probes: 120\n",
      "-- region: 68 , number of probes: 120\n",
      "-- region: 69 , number of probes: 120\n",
      "-- region: 70 , number of probes: 120\n",
      "-- region: 71 , number of probes: 120\n",
      "-- region: 72 , number of probes: 120\n",
      "-- region: 73 , number of probes: 120\n",
      "-- region: 74 , number of probes: 120\n",
      "-- region: 75 , number of probes: 120\n",
      "-- region: 76 , number of probes: 120\n",
      "-- region: 77 , number of probes: 120\n",
      "-- region: 78 , number of probes: 120\n",
      "-- region: 79 , number of probes: 120\n",
      "-- region: 80 , number of probes: 120\n",
      "-- region: 81 , number of probes: 120\n",
      "-- region: 82 , number of probes: 120\n",
      "-- region: 83 , number of probes: 120\n",
      "-- region: 84 , number of probes: 120\n",
      "-- region: 85 , number of probes: 120\n",
      "-- region: 86 , number of probes: 120\n",
      "-- region: 87 , number of probes: 120\n",
      "-- region: 88 , number of probes: 120\n",
      "-- region: 89 , number of probes: 120\n",
      "-- region: 90 , number of probes: 120\n",
      "-- region: 91 , number of probes: 120\n",
      "-- region: 92 , number of probes: 120\n",
      "-- region: 93 , number of probes: 120\n",
      "-- region: 94 , number of probes: 120\n",
      "-- region: 95 , number of probes: 120\n",
      "-- region: 96 , number of probes: 120\n",
      "-- region: 97 , number of probes: 120\n",
      "-- region: 98 , number of probes: 120\n",
      "-- region: 99 , number of probes: 120\n",
      "-- region: 100 , number of probes: 120\n",
      "-- region: 101 , number of probes: 120\n",
      "-- region: 102 , number of probes: 120\n",
      "-- region: 103 , number of probes: 120\n",
      "-- region: 104 , number of probes: 120\n",
      "-- region: 105 , number of probes: 120\n",
      "-- region: 106 , number of probes: 120\n",
      "-- region: 107 , number of probes: 120\n",
      "-- region: 108 , number of probes: 120\n",
      "-- region: 109 , number of probes: 120\n",
      "-- region: 111 , number of probes: 120\n",
      "-- region: 112 , number of probes: 120\n",
      "-- region: 113 , number of probes: 120\n",
      "-- region: 114 , number of probes: 120\n",
      "-- region: 115 , number of probes: 120\n",
      "-- region: 116 , number of probes: 120\n",
      "-- region: 117 , number of probes: 120\n",
      "-- region: 118 , number of probes: 120\n",
      "-- region: 119 , number of probes: 120\n",
      "-- region: 120 , number of probes: 120\n",
      "-- region: 121 , number of probes: 120\n",
      "-- region: 122 , number of probes: 120\n",
      "-- region: 123 , number of probes: 120\n",
      "-- region: 124 , number of probes: 120\n",
      "-- region: 125 , number of probes: 120\n",
      "-- region: 126 , number of probes: 120\n",
      "-- region: 127 , number of probes: 120\n",
      "-- region: 129 , number of probes: 120\n",
      "-- region: 130 , number of probes: 120\n",
      "-- region: 131 , number of probes: 120\n",
      "-- region: 132 , number of probes: 120\n",
      "-- region: 133 , number of probes: 120\n",
      "-- region: 134 , number of probes: 120\n",
      "-- region: 135 , number of probes: 120\n",
      "-- region: 136 , number of probes: 120\n",
      "-- region: 137 , number of probes: 120\n",
      "-- region: 138 , number of probes: 120\n",
      "-- region: 139 , number of probes: 120\n",
      "-- region: 140 , number of probes: 120\n",
      "-- region: 141 , number of probes: 120\n",
      "-- region: 142 , number of probes: 120\n",
      "-- region: 143 , number of probes: 120\n",
      "-- region: 144 , number of probes: 120\n",
      "-- region: 145 , number of probes: 120\n",
      "-- region: 146 , number of probes: 120\n",
      "-- region: 147 , number of probes: 120\n",
      "-- region: 148 , number of probes: 120\n",
      "-- region: 149 , number of probes: 120\n",
      "-- region: 150 , number of probes: 120\n",
      "-- region: 151 , number of probes: 120\n",
      "-- region: 152 , number of probes: 120\n",
      "-- region: 153 , number of probes: 120\n",
      "-- region: 154 , number of probes: 120\n",
      "-- region: 155 , number of probes: 120\n",
      "-- region: 156 , number of probes: 120\n",
      "-- region: 157 , number of probes: 120\n",
      "-- region: 158 , number of probes: 120\n",
      "-- region: 159 , number of probes: 120\n",
      "-- region: 160 , number of probes: 120\n",
      "-- region: 161 , number of probes: 120\n",
      "-- region: 162 , number of probes: 120\n",
      "-- region: 163 , number of probes: 120\n",
      "-- region: 164 , number of probes: 120\n",
      "-- region: 165 , number of probes: 120\n",
      "-- region: 166 , number of probes: 120\n",
      "-- region: 168 , number of probes: 120\n",
      "-- region: 169 , number of probes: 120\n",
      "-- region: 170 , number of probes: 120\n",
      "-- region: 171 , number of probes: 120\n",
      "-- region: 172 , number of probes: 120\n",
      "-- region: 173 , number of probes: 120\n",
      "-- region: 174 , number of probes: 120\n",
      "-- region: 175 , number of probes: 120\n",
      "-- region: 176 , number of probes: 120\n",
      "-- region: 177 , number of probes: 120\n",
      "-- region: 178 , number of probes: 120\n",
      "-- region: 179 , number of probes: 120\n",
      "-- region: 180 , number of probes: 120\n",
      "-- region: 181 , number of probes: 120\n",
      "-- region: 182 , number of probes: 120\n",
      "-- region: 183 , number of probes: 120\n",
      "-- region: 184 , number of probes: 120\n",
      "-- region: 185 , number of probes: 96\n",
      "-- region: 186 , number of probes: 120\n",
      "-- region: 187 , number of probes: 120\n",
      "-- region: 188 , number of probes: 120\n",
      "-- region: 189 , number of probes: 120\n",
      "-- region: 190 , number of probes: 120\n",
      "-- region: 191 , number of probes: 120\n",
      "-- region: 192 , number of probes: 120\n",
      "-- region: 193 , number of probes: 120\n",
      "-- region: 194 , number of probes: 120\n",
      "-- region: 195 , number of probes: 120\n",
      "-- region: 196 , number of probes: 120\n",
      "-- region: 197 , number of probes: 120\n",
      "-- region: 198 , number of probes: 120\n",
      "-- region: 199 , number of probes: 120\n",
      "-- region: 200 , number of probes: 120\n",
      "-- region: 201 , number of probes: 120\n",
      "-- region: 202 , number of probes: 120\n",
      "-- region: 203 , number of probes: 120\n",
      "-- region: 204 , number of probes: 120\n",
      "-- region: 205 , number of probes: 120\n",
      "-- region: 206 , number of probes: 120\n",
      "-- region: 207 , number of probes: 120\n",
      "-- region: 208 , number of probes: 120\n",
      "-- region: 209 , number of probes: 120\n",
      "-- region: 210 , number of probes: 120\n",
      "-- region: 211 , number of probes: 120\n",
      "-- region: 212 , number of probes: 89\n",
      "-- region: 213 , number of probes: 120\n",
      "-- region: 214 , number of probes: 120\n",
      "-- region: 215 , number of probes: 120\n",
      "-- region: 216 , number of probes: 120\n",
      "-- region: 217 , number of probes: 120\n",
      "-- region: 218 , number of probes: 120\n",
      "-- region: 219 , number of probes: 120\n",
      "-- region: 220 , number of probes: 120\n",
      "-- region: 221 , number of probes: 120\n",
      "-- region: 222 , number of probes: 120\n",
      "-- region: 223 , number of probes: 120\n",
      "-- region: 224 , number of probes: 120\n",
      "-- region: 225 , number of probes: 120\n",
      "-- region: 226 , number of probes: 120\n",
      "-- region: 228 , number of probes: 120\n",
      "-- region: 229 , number of probes: 120\n",
      "-- region: 230 , number of probes: 120\n",
      "-- region: 231 , number of probes: 120\n",
      "-- region: 232 , number of probes: 120\n",
      "-- region: 233 , number of probes: 120\n",
      "-- region: 234 , number of probes: 120\n",
      "-- region: 235 , number of probes: 120\n",
      "-- region: 236 , number of probes: 120\n",
      "-- region: 237 , number of probes: 120\n",
      "-- region: 238 , number of probes: 120\n",
      "-- region: 239 , number of probes: 120\n",
      "-- region: 240 , number of probes: 120\n",
      "-- region: 241 , number of probes: 120\n",
      "-- region: 242 , number of probes: 120\n",
      "-- region: 243 , number of probes: 120\n",
      "-- region: 244 , number of probes: 120\n",
      "-- region: 245 , number of probes: 120\n",
      "-- region: 246 , number of probes: 120\n",
      "-- region: 247 , number of probes: 120\n",
      "-- region: 248 , number of probes: 120\n",
      "-- region: 249 , number of probes: 120\n",
      "-- region: 250 , number of probes: 120\n",
      "-- region: 751 , number of probes: 10\n",
      "-- region: 752 , number of probes: 120\n",
      "-- region: 758 , number of probes: 120\n",
      "-- region: 759 , number of probes: 120\n",
      "-- region: 760 , number of probes: 120\n",
      "-- region: 761 , number of probes: 120\n",
      "-- region: 762 , number of probes: 120\n",
      "-- region: 763 , number of probes: 120\n",
      "-- region: 764 , number of probes: 120\n",
      "-- region: 765 , number of probes: 120\n",
      "-- region: 766 , number of probes: 120\n",
      "-- region: 767 , number of probes: 120\n",
      "-- region: 768 , number of probes: 120\n",
      "-- region: 769 , number of probes: 120\n",
      "-- region: 770 , number of probes: 120\n",
      "-- region: 772 , number of probes: 120\n",
      "-- region: 773 , number of probes: 120\n",
      "-- region: 774 , number of probes: 120\n",
      "-- region: 775 , number of probes: 120\n",
      "-- region: 776 , number of probes: 62\n",
      "-- region: 777 , number of probes: 120\n",
      "-- region: 778 , number of probes: 120\n",
      "-- region: 779 , number of probes: 120\n",
      "-- region: 780 , number of probes: 120\n",
      "-- region: 781 , number of probes: 120\n",
      "-- region: 782 , number of probes: 120\n",
      "-- region: 783 , number of probes: 120\n",
      "-- region: 784 , number of probes: 120\n",
      "-- region: 785 , number of probes: 120\n",
      "-- region: 786 , number of probes: 120\n",
      "-- region: 787 , number of probes: 120\n",
      "-- region: 788 , number of probes: 120\n",
      "-- region: 789 , number of probes: 120\n",
      "-- region: 790 , number of probes: 120\n",
      "-- region: 791 , number of probes: 120\n",
      "-- region: 792 , number of probes: 120\n",
      "-- region: 793 , number of probes: 120\n",
      "-- region: 794 , number of probes: 120\n",
      "-- region: 795 , number of probes: 120\n",
      "-- region: 796 , number of probes: 120\n",
      "-- region: 797 , number of probes: 120\n",
      "-- region: 798 , number of probes: 120\n",
      "-- region: 799 , number of probes: 120\n",
      "-- region: 800 , number of probes: 120\n",
      "-- region: 801 , number of probes: 120\n",
      "-- region: 802 , number of probes: 120\n",
      "-- region: 803 , number of probes: 120\n",
      "-- region: 804 , number of probes: 120\n",
      "-- region: 805 , number of probes: 120\n",
      "-- region: 806 , number of probes: 120\n",
      "-- region: 807 , number of probes: 120\n",
      "-- region: 808 , number of probes: 120\n",
      "-- region: 809 , number of probes: 120\n",
      "-- region: 810 , number of probes: 120\n",
      "-- region: 811 , number of probes: 120\n",
      "-- region: 812 , number of probes: 120\n",
      "-- region: 813 , number of probes: 120\n",
      "-- region: 814 , number of probes: 120\n",
      "-- region: 815 , number of probes: 120\n",
      "-- region: 816 , number of probes: 120\n",
      "-- region: 817 , number of probes: 120\n",
      "-- region: 818 , number of probes: 120\n",
      "-- region: 819 , number of probes: 120\n",
      "-- region: 820 , number of probes: 120\n",
      "-- region: 821 , number of probes: 120\n",
      "-- region: 822 , number of probes: 120\n",
      "-- region: 823 , number of probes: 120\n",
      "-- region: 824 , number of probes: 120\n",
      "-- region: 825 , number of probes: 120\n",
      "-- region: 826 , number of probes: 120\n",
      "-- region: 827 , number of probes: 120\n",
      "-- region: 828 , number of probes: 120\n",
      "-- region: 829 , number of probes: 120\n",
      "-- region: 830 , number of probes: 120\n",
      "-- region: 831 , number of probes: 120\n",
      "-- region: 832 , number of probes: 120\n",
      "-- region: 833 , number of probes: 120\n",
      "-- region: 834 , number of probes: 120\n",
      "-- region: 835 , number of probes: 120\n",
      "-- region: 836 , number of probes: 120\n",
      "-- region: 837 , number of probes: 120\n",
      "-- region: 838 , number of probes: 120\n",
      "-- region: 839 , number of probes: 120\n",
      "-- region: 840 , number of probes: 120\n",
      "-- region: 841 , number of probes: 120\n",
      "-- region: 842 , number of probes: 120\n",
      "-- region: 843 , number of probes: 120\n",
      "-- region: 844 , number of probes: 120\n",
      "-- region: 845 , number of probes: 120\n",
      "-- region: 846 , number of probes: 120\n",
      "-- region: 847 , number of probes: 120\n",
      "-- region: 848 , number of probes: 120\n",
      "-- region: 849 , number of probes: 120\n",
      "-- region: 850 , number of probes: 120\n",
      "-- region: 851 , number of probes: 120\n",
      "-- region: 852 , number of probes: 120\n",
      "-- region: 853 , number of probes: 120\n",
      "-- region: 854 , number of probes: 120\n",
      "-- region: 855 , number of probes: 120\n",
      "-- region: 856 , number of probes: 120\n",
      "-- region: 857 , number of probes: 120\n",
      "-- region: 858 , number of probes: 120\n",
      "-- region: 859 , number of probes: 120\n",
      "-- region: 860 , number of probes: 120\n",
      "-- region: 861 , number of probes: 120\n",
      "-- region: 862 , number of probes: 120\n",
      "-- region: 863 , number of probes: 120\n",
      "-- region: 864 , number of probes: 120\n",
      "-- region: 866 , number of probes: 120\n",
      "-- region: 867 , number of probes: 120\n",
      "-- region: 868 , number of probes: 120\n",
      "-- region: 869 , number of probes: 120\n",
      "-- region: 870 , number of probes: 120\n",
      "-- region: 871 , number of probes: 120\n",
      "-- region: 872 , number of probes: 120\n",
      "-- region: 873 , number of probes: 120\n",
      "-- region: 874 , number of probes: 120\n",
      "-- region: 875 , number of probes: 120\n",
      "-- region: 876 , number of probes: 120\n",
      "-- region: 877 , number of probes: 120\n",
      "-- region: 878 , number of probes: 120\n",
      "-- region: 879 , number of probes: 120\n",
      "-- region: 880 , number of probes: 120\n",
      "-- region: 881 , number of probes: 120\n",
      "-- region: 882 , number of probes: 120\n",
      "-- region: 883 , number of probes: 120\n",
      "-- region: 884 , number of probes: 120\n",
      "-- region: 885 , number of probes: 120\n",
      "-- region: 886 , number of probes: 120\n",
      "-- region: 887 , number of probes: 120\n",
      "-- region: 888 , number of probes: 120\n",
      "-- region: 889 , number of probes: 120\n",
      "-- region: 890 , number of probes: 120\n",
      "-- region: 891 , number of probes: 120\n",
      "-- region: 892 , number of probes: 120\n",
      "-- region: 893 , number of probes: 120\n",
      "-- region: 894 , number of probes: 120\n",
      "-- region: 895 , number of probes: 120\n",
      "-- region: 896 , number of probes: 120\n",
      "-- region: 897 , number of probes: 120\n",
      "-- region: 898 , number of probes: 120\n",
      "-- region: 899 , number of probes: 120\n",
      "-- region: 900 , number of probes: 120\n",
      "-- region: 901 , number of probes: 120\n",
      "-- region: 902 , number of probes: 120\n",
      "-- region: 903 , number of probes: 120\n",
      "-- region: 904 , number of probes: 120\n",
      "-- region: 905 , number of probes: 120\n",
      "-- region: 906 , number of probes: 120\n",
      "-- region: 907 , number of probes: 120\n",
      "-- region: 908 , number of probes: 120\n",
      "-- region: 909 , number of probes: 120\n",
      "-- region: 910 , number of probes: 120\n",
      "-- region: 911 , number of probes: 120\n",
      "-- region: 912 , number of probes: 120\n",
      "-- region: 913 , number of probes: 120\n",
      "-- region: 914 , number of probes: 120\n",
      "-- region: 915 , number of probes: 120\n",
      "-- region: 916 , number of probes: 120\n",
      "-- region: 917 , number of probes: 120\n",
      "-- region: 918 , number of probes: 120\n",
      "-- region: 919 , number of probes: 120\n",
      "-- region: 920 , number of probes: 120\n",
      "-- region: 921 , number of probes: 120\n",
      "-- region: 922 , number of probes: 120\n",
      "-- region: 923 , number of probes: 120\n",
      "-- region: 925 , number of probes: 120\n",
      "-- region: 926 , number of probes: 120\n",
      "-- region: 927 , number of probes: 120\n",
      "-- region: 928 , number of probes: 120\n",
      "-- region: 929 , number of probes: 120\n",
      "-- region: 930 , number of probes: 120\n",
      "-- region: 931 , number of probes: 120\n",
      "-- region: 932 , number of probes: 120\n",
      "-- region: 933 , number of probes: 120\n",
      "-- region: 934 , number of probes: 120\n",
      "-- region: 935 , number of probes: 120\n",
      "-- region: 936 , number of probes: 120\n",
      "-- region: 937 , number of probes: 120\n",
      "-- region: 938 , number of probes: 120\n",
      "-- region: 939 , number of probes: 120\n",
      "-- region: 940 , number of probes: 120\n",
      "-- region: 941 , number of probes: 120\n",
      "-- region: 942 , number of probes: 120\n",
      "-- region: 943 , number of probes: 120\n",
      "-- region: 944 , number of probes: 120\n",
      "-- region: 945 , number of probes: 120\n",
      "-- region: 946 , number of probes: 120\n",
      "-- region: 947 , number of probes: 120\n",
      "-- region: 948 , number of probes: 120\n",
      "-- region: 949 , number of probes: 120\n",
      "-- region: 950 , number of probes: 120\n",
      "-- region: 951 , number of probes: 120\n",
      "-- region: 952 , number of probes: 120\n",
      "-- region: 953 , number of probes: 120\n",
      "-- region: 954 , number of probes: 120\n",
      "-- region: 955 , number of probes: 120\n",
      "-- region: 956 , number of probes: 120\n",
      "-- region: 957 , number of probes: 120\n",
      "-- region: 958 , number of probes: 120\n",
      "-- region: 959 , number of probes: 120\n",
      "-- region: 960 , number of probes: 120\n",
      "-- region: 961 , number of probes: 120\n",
      "-- region: 962 , number of probes: 120\n",
      "-- region: 963 , number of probes: 120\n",
      "-- region: 964 , number of probes: 120\n",
      "-- region: 965 , number of probes: 120\n",
      "-- region: 966 , number of probes: 120\n",
      "-- region: 967 , number of probes: 120\n",
      "-- region: 968 , number of probes: 120\n",
      "-- region: 969 , number of probes: 120\n",
      "-- region: 970 , number of probes: 120\n",
      "-- region: 971 , number of probes: 120\n",
      "-- region: 972 , number of probes: 120\n",
      "-- region: 973 , number of probes: 120\n",
      "-- region: 974 , number of probes: 120\n",
      "-- region: 975 , number of probes: 120\n",
      "-- region: 976 , number of probes: 120\n",
      "-- region: 977 , number of probes: 120\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- region: 978 , number of probes: 120\n",
      "-- region: 979 , number of probes: 120\n",
      "-- region: 980 , number of probes: 120\n",
      "-- region: 981 , number of probes: 120\n",
      "-- region: 982 , number of probes: 120\n",
      "-- region: 983 , number of probes: 120\n",
      "-- region: 984 , number of probes: 120\n",
      "-- region: 985 , number of probes: 120\n",
      "-- region: 986 , number of probes: 120\n",
      "-- region: 987 , number of probes: 120\n",
      "-- region: 988 , number of probes: 120\n",
      "-- region: 989 , number of probes: 120\n",
      "-- region: 990 , number of probes: 120\n",
      "-- region: 991 , number of probes: 120\n",
      "- Number of probes kept: 115780\n",
      "- Saving to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\.\\blast_full_probes.fasta\n"
     ]
    }
   ],
   "source": [
    "# screen\n",
    "import ImageAnalysis3.library_tools.quality_check as check\n",
    "reload(check)\n",
    "\n",
    "# the max number to keep for each region\n",
    "num_probes_per_region = 120\n",
    "\n",
    "# the screen step also keeps probes close to the center of each region\n",
    "kept_pbs, blast_keep_dic, hard_count_list, soft_count_list = check.Screening_Probes_by_Blast(library_folder, \n",
    "                                                                                             num_probes_per_region,\n",
    "                                                                                             hard_thres=target_len,\n",
    "                                                                                             soft_count_th=50,\n",
    "                                                                                             )"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Below were not used for this library"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [],
   "source": [
    "# split probes\n",
    "final_pb_filename = os.path.join(library_folder, 'blast_full_probes.fasta')\n",
    "# load full probes\n",
    "kept_pb_records = []\n",
    "with open(final_pb_filename, 'r') as handle:\n",
    "    for record in SeqIO.parse(handle, \"fasta\"):\n",
    "        kept_pb_records.append(record)\n",
    "\n",
    "kept_pb_dict = library_tools.quality_check.split_probe_by_gene(kept_pb_records)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [],
   "source": [
    "chr_2_region_names = pickle.load(open(os.path.join(library_folder, 'chr_2_region_names.pkl'), 'rb'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "dict_keys(['251', '252', '253', '255', '256', '257', '258', '259', '260', '261', '262', '263', '264', '265', '266', '267', '268', '269', '270', '271', '272', '273', '274', '275', '277', '278', '279', '280', '281', '282', '283', '284', '285', '286', '287', '288', '289', '290', '291', '292', '293', '295', '296', '297', '298', '299', '300', '301', '302', '303', '304', '305', '306', '307', '308', '309', '310', '311', '312', '313', '314', '315', '316', '317', '318', '319', '320', '321', '322', '324', '325', '326', '327', '328', '329', '330', '332', '333', '334', '335', '336', '337', '338', '339', '340', '341', '342', '343', '344', '345', '347', '348', '349', '350', '351', '352', '353', '354', '355', '356', '357', '358', '359', '360', '361', '362', '363', '364', '365', '366', '367', '368', '369', '370', '371', '372', '373', '374', '375', '376', '377', '378', '379', '380', '381', '382', '383', '384', '385', '386', '387', '388', '389', '390', '391', '392', '393', '394', '395', '396', '397', '398', '399', '400', '401', '402', '403', '404', '405', '406', '407', '408', '409', '410', '411', '412', '413', '414', '415', '416', '417', '418', '419', '420', '421', '422', '423', '424', '425', '426', '427', '428', '429', '430', '431', '432', '433', '434', '435', '436', '437', '438', '439', '440', '441', '442', '444', '445', '446', '447', '448', '449', '450', '451', '452', '453', '454', '455', '456', '457', '458', '459', '460', '461', '462', '463', '464', '465', '466', '467', '468', '469', '470', '471', '472', '473', '474', '475', '476', '477', '478', '479', '480', '481', '482', '483', '484', '485', '486', '487', '488', '489', '491', '492', '493', '494', '495', '496', '497', '498', '499', '500', '501', '502', '503', '504', '505', '506', '507', '508', '509', '510', '511', '512', '513', '514', '515', '516', '517', '518', '519', '520', '521', '523', '524', '525', '526', '527', '528', '529', '530', '531', '532', '533', '534', '535', '536', '537', '538', '539', '540', '541', '542', '543', '544', '545', '546', '547', '548', '549', '550', '551', '552', '553', '554', '555', '556', '557', '558', '559', '560', '561', '562', '563', '564', '565', '566', '567', '568', '569', '570', '571', '572', '573', '574', '575', '576', '577', '579', '580', '581', '582', '583', '584', '585', '586', '587', '588', '589', '590', '591', '592', '593', '594', '595', '596', '597', '598', '599', '600', '601', '602', '603', '604', '605', '606', '607', '608', '609', '610', '611', '612', '613', '614', '615', '616', '617', '618', '619', '620', '621', '622', '623', '624', '625', '626', '627', '628', '629', '630', '631', '632', '633', '634', '635', '636', '637', '638', '639', '640', '641', '642', '643', '644', '645', '646', '647', '648', '649', '650', '651', '652', '653', '654', '655', '656', '657', '658', '659', '660', '661', '662', '663', '664', '665', '666', '667', '668', '669', '670', '671', '672', '673', '674', '675', '676', '677', '678', '679', '680', '681', '682', '683', '684', '685', '687', '688', '689', '690', '691', '692', '693', '694', '695', '696', '697', '698', '699', '700', '701', '702', '703', '704', '705', '706', '707', '708', '709', '710', '711', '712', '713', '714', '715', '716', '717', '718', '719', '720', '721', '722', '723', '724', '725', '726', '727', '728', '729', '730', '731', '732', '733', '734', '735', '736', '737', '738', '739', '740', '741', '742', '743', '744', '745', '746', '747', '748', '749', '750', '1', '2', '3', '4', '5', '6', '7', '8', '9', '10', '11', '12', '13', '14', '15', '16', '17', '18', '19', '20', '21', '22', '23', '24', '25', '26', '27', '28', '29', '30', '31', '32', '33', '34', '35', '36', '37', '38', '39', '40', '41', '42', '43', '44', '45', '46', '47', '48', '49', '50', '51', '52', '53', '54', '55', '56', '57', '58', '59', '60', '61', '62', '63', '64', '65', '66', '67', '68', '69', '70', '71', '72', '73', '74', '75', '76', '77', '78', '79', '80', '81', '82', '83', '84', '85', '86', '87', '88', '89', '90', '91', '92', '93', '94', '95', '96', '97', '98', '99', '100', '101', '102', '103', '104', '105', '106', '107', '108', '109', '111', '112', '113', '114', '115', '116', '117', '118', '119', '120', '121', '122', '123', '124', '125', '126', '127', '129', '130', '131', '132', '133', '134', '135', '136', '137', '138', '139', '140', '141', '142', '143', '144', '145', '146', '147', '148', '149', '150', '151', '152', '153', '154', '155', '156', '157', '158', '159', '160', '161', '162', '163', '164', '165', '166', '168', '169', '170', '171', '172', '173', '174', '175', '176', '177', '178', '179', '180', '181', '182', '183', '184', '185', '186', '187', '188', '189', '190', '191', '192', '193', '194', '195', '196', '197', '198', '199', '200', '201', '202', '203', '204', '205', '206', '207', '208', '209', '210', '211', '212', '213', '214', '215', '216', '217', '218', '219', '220', '221', '222', '223', '224', '225', '226', '228', '229', '230', '231', '232', '233', '234', '235', '236', '237', '238', '239', '240', '241', '242', '243', '244', '245', '246', '247', '248', '249', '250', '751', '752', '758', '759', '760', '761', '762', '763', '764', '765', '766', '767', '768', '769', '770', '772', '773', '774', '775', '776', '777', '778', '779', '780', '781', '782', '783', '784', '785', '786', '787', '788', '789', '790', '791', '792', '793', '794', '795', '796', '797', '798', '799', '800', '801', '802', '803', '804', '805', '806', '807', '808', '809', '810', '811', '812', '813', '814', '815', '816', '817', '818', '819', '820', '821', '822', '823', '824', '825', '826', '827', '828', '829', '830', '831', '832', '833', '834', '835', '836', '837', '838', '839', '840', '841', '842', '843', '844', '845', '846', '847', '848', '849', '850', '851', '852', '853', '854', '855', '856', '857', '858', '859', '860', '861', '862', '863', '864', '866', '867', '868', '869', '870', '871', '872', '873', '874', '875', '876', '877', '878', '879', '880', '881', '882', '883', '884', '885', '886', '887', '888', '889', '890', '891', '892', '893', '894', '895', '896', '897', '898', '899', '900', '901', '902', '903', '904', '905', '906', '907', '908', '909', '910', '911', '912', '913', '914', '915', '916', '917', '918', '919', '920', '921', '922', '923', '925', '926', '927', '928', '929', '930', '931', '932', '933', '934', '935', '936', '937', '938', '939', '940', '941', '942', '943', '944', '945', '946', '947', '948', '949', '950', '951', '952', '953', '954', '955', '956', '957', '958', '959', '960', '961', '962', '963', '964', '965', '966', '967', '968', '969', '970', '971', '972', '973', '974', '975', '976', '977', '978', '979', '980', '981', '982', '983', '984', '985', '986', '987', '988', '989', '990', '991'])"
      ]
     },
     "execution_count": 29,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "kept_pb_dict.keys()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
